<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/26/2020.03.21.990770.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.21.990770v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Potent human neutralizing antibodies elicited by SARS-CoV-2 infection" />
<meta name="DC.Identifier" content="10.1101/2020.03.21.990770" />
<meta name="DC.Date" content="2020-03-26" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The pandemic caused by emerging coronavirus SARS-CoV-2 presents a serious global public health emergency in urgent need of prophylactic and therapeutic interventions. SARS-CoV-2 cellular entry depends on binding between the viral Spike protein receptor-binding domain (RBD) and the angiotensin converting enzyme 2 (ACE2) target cell receptor. Here, we report on the isolation and characterization of 206 RBD-specific monoclonal antibodies (mAbs) derived from single B cells of eight SARS-CoV-2 infected individuals. These mAbs come from diverse families of antibody heavy and light chains without apparent enrichment for particular families in the repertoire. In samples from one patient selected for further analyses, we found coexistence of germline and germline divergent clones. Both clone types demonstrated impressive binding and neutralizing activity against pseudovirus and live SARS-CoV-2. However, the antibody neutralizing potency is determined by competition with ACE2 receptor for RBD binding. Surprisingly, none of the SARS-CoV-2 antibodies nor the infected plasma cross-reacted with RBDs from either SARS-CoV or MERS-CoV although substantial plasma cross-reactivity to the trimeric Spike proteins from SARS-CoV and MERS-CoV was found. These results suggest that antibody response to RBDs is viral species-specific while that cross-recognition target regions outside the RBD. The specificity and neutralizing characteristics of this plasma cross-reactivity requires further investigation. Nevertheless, the diverse and potent neutralizing antibodies identified here are promising candidates for prophylactic and therapeutic SARS-CoV-2 interventions." />
<meta name="DC.Contributor" content="Bin Ju" />
<meta name="DC.Contributor" content="Qi Zhang" />
<meta name="DC.Contributor" content="Xiangyang Ge" />
<meta name="DC.Contributor" content="Ruoke Wang" />
<meta name="DC.Contributor" content="Jiazhen Yu" />
<meta name="DC.Contributor" content="Sisi Shan" />
<meta name="DC.Contributor" content="Bing Zhou" />
<meta name="DC.Contributor" content="Shuo Song" />
<meta name="DC.Contributor" content="Xian Tang" />
<meta name="DC.Contributor" content="Jinfang Yu" />
<meta name="DC.Contributor" content="Jiwan Ge" />
<meta name="DC.Contributor" content="Jun Lan" />
<meta name="DC.Contributor" content="Jing Yuan" />
<meta name="DC.Contributor" content="Haiyan Wang" />
<meta name="DC.Contributor" content="Juanjuan Zhao" />
<meta name="DC.Contributor" content="Shuye Zhang" />
<meta name="DC.Contributor" content="Youchun Wang" />
<meta name="DC.Contributor" content="Xuanling Shi" />
<meta name="DC.Contributor" content="Lei Liu" />
<meta name="DC.Contributor" content="Xinquan Wang" />
<meta name="DC.Contributor" content="Zheng Zhang" />
<meta name="DC.Contributor" content="Linqi Zhang" />
<meta name="article:published_time" content="2020-03-26" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Potent human neutralizing antibodies elicited by SARS-CoV-2 infection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The pandemic caused by emerging coronavirus SARS-CoV-2 presents a serious global public health emergency in urgent need of prophylactic and therapeutic interventions. SARS CoV-2 cellular entry depends on binding between the viral Spike protein receptor-binding domain (RBD) and the angiotensin converting enzyme 2 (ACE2) target cell receptor. Here, we report on the isolation and characterization of 206 RBD-specific monoclonal antibodies (mAbs) derived from single B cells of eight SARS-CoV-2 infected individuals. These mAbs come from diverse families of antibody heavy and light chains without apparent enrichment for particular families in the repertoire. In samples from one patient selected for further analyses, we found coexistence of germline and germline divergent clones. Both clone types demonstrated impressive binding and neutralizing activity against pseudovirus and live SARS-CoV-2. However, the antibody neutralizing potency is determined by competition with ACE2 receptor for RBD binding. Surprisingly, none of the SARS CoV 2 antibodies nor the infected plasma cross-reacted with RBDs from either SARS CoV or MERS CoV although substantial plasma cross reactivity to the trimeric Spike proteins from SARS-CoV and MERS-CoV was found. These results suggest that antibody response to RBDs is viral species-specific while that cross-recognition target regions outside the RBD. The specificity and neutralizing characteristics of this plasma cross-reactivity requires further investigation. Nevertheless, the diverse and potent neutralizing antibodies identified here are promising candidates for prophylactic and therapeutic SARS-CoV-2 interventions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.21.990770v2" />
<meta name="citation_id" content="2020.03.21.990770v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.21.990770" />
<meta name="citation_num_pages" content="42" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.21.990770" />
<meta name="citation_author" content="Bin Ju" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author_institution" content="The Second Affiliated Hospital, School of Medicine, Southern University ofMN Science and Technology" />
<meta name="citation_author" content="Qi Zhang" />
<meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University" />
<meta name="citation_author" content="Xiangyang Ge" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author" content="Ruoke Wang" />
<meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University" />
<meta name="citation_author" content="Jiazhen Yu" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author" content="Sisi Shan" />
<meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University" />
<meta name="citation_author" content="Bing Zhou" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author" content="Shuo Song" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author" content="Xian Tang" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author" content="Jinfang Yu" />
<meta name="citation_author_institution" content="The Ministry of Education Key Laboratory of Protein Science, Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Collaborative Innovation Center for Biotherapy, School of Life Sciences, Tsinghua University" />
<meta name="citation_author" content="Jiwan Ge" />
<meta name="citation_author_institution" content="The Ministry of Education Key Laboratory of Protein Science, Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Collaborative Innovation Center for Biotherapy, School of Life Sciences, Tsinghua University" />
<meta name="citation_author" content="Jun Lan" />
<meta name="citation_author_institution" content="The Ministry of Education Key Laboratory of Protein Science, Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Collaborative Innovation Center for Biotherapy, School of Life Sciences, Tsinghua University" />
<meta name="citation_author" content="Jing Yuan" />
<meta name="citation_author_institution" content="Department for Infectious Diseases, Shenzhen Third People’s Hospital" />
<meta name="citation_author" content="Haiyan Wang" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author" content="Juanjuan Zhao" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author_institution" content="The Second Affiliated Hospital, School of Medicine, Southern University ofMN Science and Technology" />
<meta name="citation_author" content="Shuye Zhang" />
<meta name="citation_author_institution" content="Shanghai Public Health Clinical Center and Institute of Biomedical Sciences, Fudan University" />
<meta name="citation_author" content="Youchun Wang" />
<meta name="citation_author_institution" content="Division of HIV/AIDS and Sex-Transmitted Virus Vaccines, National Institutes for Food and Drug Control" />
<meta name="citation_author" content="Xuanling Shi" />
<meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University" />
<meta name="citation_author" content="Lei Liu" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author_institution" content="The Second Affiliated Hospital, School of Medicine, Southern University ofMN Science and Technology" />
<meta name="citation_author" content="Xinquan Wang" />
<meta name="citation_author_institution" content="The Ministry of Education Key Laboratory of Protein Science, Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Collaborative Innovation Center for Biotherapy, School of Life Sciences, Tsinghua University" />
<meta name="citation_author" content="Zheng Zhang" />
<meta name="citation_author_institution" content="Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital" />
<meta name="citation_author_institution" content="The Second Affiliated Hospital, School of Medicine, Southern University ofMN Science and Technology" />
<meta name="citation_author_email" content="zhangzheng1975@aliyun.com" />
<meta name="citation_author_email" content="zhanglinqi@tsinghua.edu.cn" />
<meta name="citation_author" content="Linqi Zhang" />
<meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University" />
<meta name="citation_author_email" content="zhangzheng1975@aliyun.com" />
<meta name="citation_author_email" content="zhanglinqi@tsinghua.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4931-509X" />
<meta name="citation_reference" content="Li, Q. et al. Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia. N Engl J Med, 10.1056/NEJMoa2001316, doi: 10.1056/NEJMoa2001316 (2020)." />
<meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579, 270–273, doi: 10.1038/s41586-020-2012-7 (2020)." />
<meta name="citation_reference" content="Zhu, N. et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 382, 727–733, doi: 10.1056/NEJMoa2001017 (2020)." />
<meta name="citation_reference" content="Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature 579, 265–269, doi: 10.1038/s41586-020-2008-3 (2020)." />
<meta name="citation_reference" content="Chan, J. F.-W. et al. A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet 395, 514–523, doi: 10.1016/S0140-6736(20)30154-9 (2020)." />
<meta name="citation_reference" content="Guan, W.-J. et al. Clinical Characteristics of Coronavirus Disease 2019 in China. N Engl J Med, 10.1056/NEJMoa2002032, doi: 10.1056/NEJMoa2002032 (2020)." />
<meta name="citation_reference" content="Huang, C. et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395, 497–506, doi: 10.1016/S0140-6736(20)30183-5 (2020)." />
<meta name="citation_reference" content="Wang, D. et al. Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China. JAMA, e201585, doi: 10.1001/jama.2020.1585 (2020)." />
<meta name="citation_reference" content="Chinazzi, M. et al. The effect of travel restrictions on the spread of the 2019 novel coronavirus (COVID-19) outbreak. Science, eaba9757, doi: 10.1126/science.aba9757 (2020)." />
<meta name="citation_reference" content="Lu, R. et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet 395, 565–574, doi: 10.1016/S0140-6736(20)30251-8 (2020)." />
<meta name="citation_reference" content="Wu, A. et al. Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. Cell Host Microbe, S1931-3128(1920)30072-X, doi: 10.1016/j.chom.2020.02.001 (2020)." />
<meta name="citation_reference" content="Ge, X.-Y. et al. Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538, doi: 10.1038/nature12711 (2013)." />
<meta name="citation_reference" content="Hoffmann, M. et al. SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell, S0092-8674(0020)30229-30224, doi: 10.1016/j.cell.2020.02.052 (2020)." />
<meta name="citation_reference" content="Walls, A. C. et al. Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell, S0092-8674(0020)30262-30262, doi: 10.1016/j.cell.2020.02.058 (2020)." />
<meta name="citation_reference" content="Du, L. et al. The spike protein of SARS-CoV--a target for vaccine and therapeutic development. Nat Rev Microbiol 7, 226–236, doi: 10.1038/nrmicro2090 (2009)." />
<meta name="citation_reference" content="Li, F. Structure, Function, and Evolution of Coronavirus Spike Proteins. Annu Rev Virol 3, 237–261, doi: 10.1146/annurev-virology-110615-042301 (2016)." />
<meta name="citation_reference" content="Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science, eabb2507, doi: 10.1126/science.abb2507 (2020)." />
<meta name="citation_reference" content="Gui, M. et al. Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell Res 27, 119–129, doi: 10.1038/cr.2016.152 (2017)." />
<meta name="citation_reference" content="Song, W., Gui, M., Wang, X. &amp; Xiang, Y. Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS Pathog 14, e1007236–e1007236, doi: 10.1371/journal.ppat.1007236 (2018)." />
<meta name="citation_reference" content="Kirchdoerfer, R. N. et al. Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis. Sci Rep 8, 15701–15701, doi: 10.1038/s41598-018-34171-7 (2018)." />
<meta name="citation_reference" content="Yuan, Y. et al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nat Commun 8, 15092–15092, doi: 10.1038/ncomms15092 (2017)." />
<meta name="citation_reference" content="Wan, Y., Shang, J., Graham, R., Baric, R. S. &amp; Li, F. Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol, JVI.00127-00120, doi: 10.1128/JVI.00127-20 (2020)." />
<meta name="citation_reference" content="Kruse, R. L. Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan, China. F1000Res 9, 72–72, doi: 10.12688/f1000research.22211.2 (2020)." />
<meta name="citation_reference" content="Li, W. et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454, doi: 10.1038/nature02145 (2003)." />
<meta name="citation_reference" content="Hamming, I. et al. The emerging role of ACE2 in physiology and disease. J Pathol 212, 1–11, doi: 10.1002/path.2162 (2007)." />
<meta name="citation_reference" content="Xu, J. et al. Antibodies and vaccines against Middle East respiratory syndrome coronavirus. Emerg Microbes Infect 8, 841–856, doi: 10.1080/22221751.2019.1624482 (2019)." />
<meta name="citation_reference" content="Sanders, R. W. et al. Stabilization of the soluble, cleaved, trimeric form of the envelope glycoprotein complex of human immunodeficiency virus type 1. J Virol 76, 8875–8889, doi: 10.1128/jvi.76.17.8875-8889.2002 (2002)." />
<meta name="citation_reference" content="Kong, L. et al. Key gp120 Glycans Pose Roadblocks to the Rapid Development of VRC01-Class Antibodies in an HIV-1-Infected Chinese Donor. Immunity 44, 939–950, doi: 10.1016/j.immuni.2016.03.006 (2016)." />
<meta name="citation_reference" content="Liao, H.-X. et al. High-throughput isolation of immunoglobulin genes from single human B cells and expression as monoclonal antibodies. J Virol Methods 158, 171–179, doi: 10.1016/j.jviromet.2009.02.014 (2009)." />
<meta name="citation_reference" content="Yu, L. et al. Delineating antibody recognition against Zika virus during natural infection. JCI Insight 2, e93042, doi: 10.1172/jci.insight.93042 (2017)." />
<meta name="citation_reference" content="Corti, D. &amp; Lanzavecchia, A. Broadly neutralizing antiviral antibodies. Annu Rev Immunol 31, 705–742, doi: 10.1146/annurev-immunol-032712-095916 (2013)." />
<meta name="citation_reference" content="Stettler, K. et al. Specificity, cross-reactivity, and function of antibodies elicited by Zika virus infection. Science 353, 823–826, doi: 10.1126/science.aaf8505 (2016)." />
<meta name="citation_reference" content="Scheid, J. F. et al. Broad diversity of neutralizing antibodies isolated from memory B cells in HIV-infected individuals. Nature 458, 636–640, doi: 10.1038/nature07930 (2009)." />
<meta name="citation_reference" content="Wu, X. et al. Focused evolution of HIV-1 neutralizing antibodies revealed by structures and deep sequencing. Science 333, 1593–1602, doi: 10.1126/science.1207532 (2011)." />
<meta name="citation_reference" content="Liao, H.-X. et al. Co-evolution of a broadly neutralizing HIV-1 antibody and founder virus. Nature 496, 469–476, doi: 10.1038/nature12053 (2013)." />
<meta name="citation_reference" content="Tian, X. et al. Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. Emerg Microbes Infect 9, 382–385, doi: 10.1080/22221751.2020.1729069 (2020)." />
<meta name="citation_reference" content="Wang, N. et al. Structure of MERS-CoV spike receptor-binding domain complexed with human receptor DPP4. Cell Res 23, 986–993, doi: 10.1038/cr.2013.92 (2013)." />
<meta name="citation_reference" content="Jiang, L. et al. Potent neutralization of MERS-CoV by human neutralizing monoclonal antibodies to the viral spike glycoprotein. Sci Transl Med 6, 234ra259–234ra259, doi: 10.1126/scitranslmed.3008140 (2014)." />
<meta name="citation_reference" content="Zhang, S. et al. Structural Definition of a Unique Neutralization Epitope on the Receptor-Binding Domain of MERS-CoV Spike Glycoprotein. Cell Rep 24, 441–452, doi: 10.1016/j.celrep.2018.06.041 (2018)." />
<meta name="citation_reference" content="Wu, X. et al. Rational design of envelope identifies broadly neutralizing human monoclonal antibodies to HIV-1. Science 329, 856–861, doi: 10.1126/science.1187659 (2010)." />
<meta name="citation_reference" content="Tiller, T. et al. Efficient generation of monoclonal antibodies from single human B cells by single cell RT-PCR and expression vector cloning. J Immunol Methods 329, 112–124, doi: 10.1016/j.jim.2007.09.017 (2008)." />
<meta name="citation_reference" content="Zhu, Z. et al. Potent cross-reactive neutralization of SARS coronavirus isolates by human monoclonal antibodies. Proc Natl Acad Sci U S A 104, 12123–12128, doi: 10.1073/pnas.0701000104 (2007)." />
<meta name="citation_reference" content="Niu, P. et al. Ultrapotent Human Neutralizing Antibody Repertoires Against Middle East Respiratory Syndrome Coronavirus From a Recovered Patient. J Infect Dis 218, 1249–1260, doi: 10.1093/infdis/jiy311 (2018)." />
<meta name="citation_reference" content="Jia, W. et al. Single intranasal immunization with chimpanzee adenovirus-based vaccine induces sustained and protective immunity against MERS-CoV infection. Emerg Microbes Infect 8, 760–772, doi: 10.1080/22221751.2019.1620083 (2019)." />
<meta name="citation_reference" content="Zhang, L. et al. Antibody responses against SARS coronavirus are correlated with disease outcome of infected individuals. J Med Virol 78, 1–8, doi: 10.1002/jmv.20499 (2006)." />
<meta name="citation_reference" content="Zhang, Q. et al. Potent neutralizing monoclonal antibodies against Ebola virus infection. Sci Rep 6, 25856–25856, doi: 10.1038/srep25856 (2016)." />
<meta name="twitter:title" content="Potent human neutralizing antibodies elicited by SARS-CoV-2 infection" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The pandemic caused by emerging coronavirus SARS-CoV-2 presents a serious global public health emergency in urgent need of prophylactic and therapeutic interventions. SARS-CoV-2 cellular entry depends on binding between the viral Spike protein receptor-binding domain (RBD) and the angiotensin converting enzyme 2 (ACE2) target cell receptor. Here, we report on the isolation and characterization of 206 RBD-specific monoclonal antibodies (mAbs) derived from single B cells of eight SARS-CoV-2 infected individuals. These mAbs come from diverse families of antibody heavy and light chains without apparent enrichment for particular families in the repertoire. In samples from one patient selected for further analyses, we found coexistence of germline and germline divergent clones. Both clone types demonstrated impressive binding and neutralizing activity against pseudovirus and live SARS-CoV-2. However, the antibody neutralizing potency is determined by competition with ACE2 receptor for RBD binding. Surprisingly, none of the SARS-CoV-2 antibodies nor the infected plasma cross-reacted with RBDs from either SARS-CoV or MERS-CoV although substantial plasma cross-reactivity to the trimeric Spike proteins from SARS-CoV and MERS-CoV was found. These results suggest that antibody response to RBDs is viral species-specific while that cross-recognition target regions outside the RBD. The specificity and neutralizing characteristics of this plasma cross-reactivity requires further investigation. Nevertheless, the diverse and potent neutralizing antibodies identified here are promising candidates for prophylactic and therapeutic SARS-CoV-2 interventions." />
<meta name="og-title" property="og:title" content="Potent human neutralizing antibodies elicited by SARS-CoV-2 infection" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The pandemic caused by emerging coronavirus SARS-CoV-2 presents a serious global public health emergency in urgent need of prophylactic and therapeutic interventions. SARS-CoV-2 cellular entry depends on binding between the viral Spike protein receptor-binding domain (RBD) and the angiotensin converting enzyme 2 (ACE2) target cell receptor. Here, we report on the isolation and characterization of 206 RBD-specific monoclonal antibodies (mAbs) derived from single B cells of eight SARS-CoV-2 infected individuals. These mAbs come from diverse families of antibody heavy and light chains without apparent enrichment for particular families in the repertoire. In samples from one patient selected for further analyses, we found coexistence of germline and germline divergent clones. Both clone types demonstrated impressive binding and neutralizing activity against pseudovirus and live SARS-CoV-2. However, the antibody neutralizing potency is determined by competition with ACE2 receptor for RBD binding. Surprisingly, none of the SARS-CoV-2 antibodies nor the infected plasma cross-reacted with RBDs from either SARS-CoV or MERS-CoV although substantial plasma cross-reactivity to the trimeric Spike proteins from SARS-CoV and MERS-CoV was found. These results suggest that antibody response to RBDs is viral species-specific while that cross-recognition target regions outside the RBD. The specificity and neutralizing characteristics of this plasma cross-reactivity requires further investigation. Nevertheless, the diverse and potent neutralizing antibodies identified here are promising candidates for prophylactic and therapeutic SARS-CoV-2 interventions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-26" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.21.990770v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.21.990770v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.21.990770v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1212177" />
    <title>Potent human neutralizing antibodies elicited by SARS-CoV-2 infection | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CzfpZSB46IeoUPAG_vS47P9uXcvsnpt8HlokZK_S1Mo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1212177","apath":"\/biorxiv\/early\/2020\/03\/26\/2020.03.21.990770.atom","pisa":"biorxiv;2020.03.21.990770v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.21.990770v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.21.990770v2","title":"Potent human neutralizing antibodies elicited by SARS-CoV-2 infection","identifier":"node\/1212177"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vXO9d0Lj8sWTpOPr0pkyi8GtIuZGja7SKY4hrLxqOshe0Eu9THqgtSx4vB5yeNfRdLE4dXj5UAds0wNLJGmrRYcL+fEDK+oQpBpLpHOefj8KkfF4QJoU4+9xi9usolSXopKGYilblafHuPbezzoQGs7cUu8rSV3VLki0AqEswIYVi7oQMl6i6Jb1AKuBG71asS0UXRs08qdrYUS6toFGwvmAiLenTDfP1euYHre+mw6CD7NnSAeRXCjAnG3Y4U\\\\\\\/vCOUdXv2Yg+kWnvu+gXM1LRrmtGWIWvZ5RPsTtcZ7m5CvpCI8munuSkOEEw+V+ZVaaCE7G2fQHWRGJzkYT7Kag9x65\\\\\\\/Ly2IR8XkeF+CjaSYMSMSMtlrwSAiznqTHq\\\\\\\/duQ2Z8\\\\\\\/968hL9HxA+JcvlsiuXc0c1TXrHF4KxtJI3K4XfodRPW3ScffNgFPxBJwyDTZtLSM69XAPLEOsytCanl7l2nsSlwg1ffxHtXypYXUJa8vQmiG0IRBDWZzH9dVow8wXCPfQhF9xElmleUaaMrWwjZ9ZnNqyP87Tztidig6Ui2VQ6CXsbzTlxcbixNMj57XrTCFlW1etYXTEr9a9pGl0G5RLGZzH6GUqSQHq3F3Wax8tB4mGGP2VbERYPzgD9mjYlIB6TOAifW0hwcfWFHh9oqRfWVhwKyXAEE8ivXHB6h0jg5fGg5BJMrSclVmA4ZBo7Nnf8vVMwQXqydnCUCEqc8YJShaV8LwHHiW8fbReh3\\\\\\\/yJzAf3\\\\\\\/gzJ2Pkoa9IOsGQkCH4X0G50EeJvhisifZdxe\\\\\\\/ToUQU2JQIASPF9e+hobs5zPgOe0jlxgELrurK4xmzgQcOL2+1LkM8yc2AsfWs4c9JUOVTt2KAcCqKdqTAFFnI8RYIDRNuiY3GoHUrA8xZl6URqIt1XcRlP2VZOwrSPSlNuw9w5l+ZJ84odyRRgb2YcQZy+srv22gIDH96MofbwGPnmqdMtmfaKGkZj4uLyg2Uzyy8ACfLCQiO6ZWS7r4xsyH0m1uiI3ePNju2DCt70TAkuv7eP3Hqk6qrNwEdFUOcqKhaWuIsgNhnjJIvE1kudVS1zfG2QyYwFylCy6ZhasSFXAWMu1N+vyZ5glb2\\\\\\\/1XBwleLG24nfbGvwVlbqg7bp5qb4suo9XcRN\\\\\\\/j2ou5RWKzJM9Tkrs852IkvtO8DClXUTAI0tIfayvpfMFcqiy35ZPCmWssqqn8MAaN3mKaHj7jlFosrbPA0OsUVJWtPJCronyzLppJECBUUFsMqcRHoQJNeddFCmJpVNuPXm1CyorcomoULVyxI5CjGYeoF7qAoCs6Ac07k1w\\\\\\\/todjqAD7NfnNP8Js6GFBAMIfNAFe\\\u0022,\\\u0022iv\\\u0022:\\\u0022VIxHqWkMAydaQDpSCqQ18Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00229de5e6d1c0e2783222ecc44651048cfc490a2870f745828c2b7b0d388e5623f8\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.21.990770v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1212177 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.21.990770v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FfwkQjSO1UeJGudJUFsodiF0NGfhpFxIADIU7jnodPw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1212177" id="node1212177" data-pisa="biorxiv;2020.03.21.990770v2" data-pisa-master="biorxiv;2020.03.21.990770" data-apath="/biorxiv/early/2020/03/26/2020.03.21.990770.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Potent human neutralizing antibodies elicited by SARS-CoV-2 infection</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Bin</span> <span class="nlm-surname">Ju</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Qi</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiangyang</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ruoke</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jiazhen</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sisi</span> <span class="nlm-surname">Shan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Bing</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Shuo</span> <span class="nlm-surname">Song</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xian</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jinfang</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Jiwan</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Jun</span> <span class="nlm-surname">Lan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Yuan</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Haiyan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Juanjuan</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Shuye</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Youchun</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Xuanling</span> <span class="nlm-surname">Shi</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Lei</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Xinquan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Zheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="21"><a href="http://orcid.org/0000-0003-4931-509X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Linqi</span> <span class="nlm-surname">Zhang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.21.990770 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1212177" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bin Ju </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>The Second Affiliated Hospital, School of Medicine, Southern University ofMN Science and Technology</span>, Shenzhen 518055, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bin%2BJu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ju%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABin%2BJu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Qi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University</span>, Beijing 100084, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiangyang Ge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiangyang%2BGe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ge%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiangyang%2BGe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ruoke Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University</span>, Beijing 100084, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ruoke%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARuoke%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jiazhen Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiazhen%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiazhen%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sisi Shan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University</span>, Beijing 100084, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sisi%2BShan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shan%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASisi%2BShan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Bing Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bing%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABing%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Shuo Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuo%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShuo%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Xian Tang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xian%2BTang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXian%2BTang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jinfang Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>The Ministry of Education Key Laboratory of Protein Science, Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Collaborative Innovation Center for Biotherapy, School of Life Sciences, Tsinghua University</span>, 100084 Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinfang%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinfang%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jiwan Ge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>The Ministry of Education Key Laboratory of Protein Science, Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Collaborative Innovation Center for Biotherapy, School of Life Sciences, Tsinghua University</span>, 100084 Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiwan%2BGe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ge%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiwan%2BGe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Jun Lan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>The Ministry of Education Key Laboratory of Protein Science, Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Collaborative Innovation Center for Biotherapy, School of Life Sciences, Tsinghua University</span>, 100084 Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jun%2BLan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJun%2BLan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Jing Yuan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department for Infectious Diseases, Shenzhen Third People’s Hospital</span>, Shenzhen, Guangdong Province 518112, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BYuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yuan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BYuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Haiyan Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haiyan%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaiyan%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Juanjuan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>The Second Affiliated Hospital, School of Medicine, Southern University ofMN Science and Technology</span>, Shenzhen 518055, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juanjuan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuanjuan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Shuye Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Shanghai Public Health Clinical Center and Institute of Biomedical Sciences, Fudan University</span>, Shanghai 201508, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuye%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShuye%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Youchun Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Division of HIV/AIDS and Sex-Transmitted Virus Vaccines, National Institutes for Food and Drug Control</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youchun%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYouchun%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Xuanling Shi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University</span>, Beijing 100084, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuanling%2BShi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shi%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuanling%2BShi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Lei Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>The Second Affiliated Hospital, School of Medicine, Southern University ofMN Science and Technology</span>, Shenzhen 518055, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lei%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALei%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Xinquan Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>The Ministry of Education Key Laboratory of Protein Science, Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Collaborative Innovation Center for Biotherapy, School of Life Sciences, Tsinghua University</span>, 100084 Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinquan%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinquan%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Zheng Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Hepatology, National Clinical Research Center for Infectious Disease, Shenzhen Third People’s Hospital</span>, Shenzhen 518112, Guangdong Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>The Second Affiliated Hospital, School of Medicine, Southern University ofMN Science and Technology</span>, Shenzhen 518055, Guangdong Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zheng%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZheng%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangzheng1975@aliyun.com" class="" data-icon-position="" data-hide-link-title="0">zhangzheng1975@aliyun.com</a>
<a href="mailto:zhanglinqi@tsinghua.edu.cn" class="" data-icon-position="" data-hide-link-title="0">zhanglinqi@tsinghua.edu.cn</a></span></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Linqi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center for Global Health and Infectious Diseases, Comprehensive AIDS Research Center, and Beijing Advanced Innovation Center for Structural Biology, School of Medicine, Tsinghua University</span>, Beijing 100084, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linqi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALinqi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4931-509X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Linqi Zhang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhangzheng1975@aliyun.com" class="" data-icon-position="" data-hide-link-title="0">zhangzheng1975@aliyun.com</a>
<a href="mailto:zhanglinqi@tsinghua.edu.cn" class="" data-icon-position="" data-hide-link-title="0">zhanglinqi@tsinghua.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.21.990770v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1212177" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1212177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.21.990770v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1212177" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1212177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.21.990770v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1212177" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1212177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.21.990770v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1212177" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1212177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.21.990770v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1212177" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1212177/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">The pandemic caused by emerging coronavirus SARS-CoV-2 presents a serious global public health emergency in urgent need of prophylactic and therapeutic interventions. SARS-CoV-2 cellular entry depends on binding between the viral Spike protein receptor-binding domain (RBD) and the angiotensin converting enzyme 2 (ACE2) target cell receptor. Here, we report on the isolation and characterization of 206 RBD-specific monoclonal antibodies (mAbs) derived from single B cells of eight SARS-CoV-2 infected individuals. These mAbs come from diverse families of antibody heavy and light chains without apparent enrichment for particular families in the repertoire. In samples from one patient selected for further analyses, we found coexistence of germline and germline divergent clones. Both clone types demonstrated impressive binding and neutralizing activity against pseudovirus and live SARS-CoV-2. However, the antibody neutralizing potency is determined by competition with ACE2 receptor for RBD binding. Surprisingly, none of the SARS-CoV-2 antibodies nor the infected plasma cross-reacted with RBDs from either SARS-CoV or MERS-CoV although substantial plasma cross-reactivity to the trimeric Spike proteins from SARS-CoV and MERS-CoV was found. These results suggest that antibody response to RBDs is viral species-specific while that cross-recognition target regions outside the RBD. The specificity and neutralizing characteristics of this plasma cross-reactivity requires further investigation. Nevertheless, the diverse and potent neutralizing antibodies identified here are promising candidates for prophylactic and therapeutic SARS-CoV-2 interventions.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">The source of the recent Coronavirus Disease 2019 (COVID-19) outbreak in Wuhan, China is a novel pathogenic coronavirus, SARS-CoV-2 <sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. Its unique pathogenesis and rapid international transmission poses a serious global health emergency <sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>-<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>. SARS-CoV-2 belongs to the betacoronavirus family and shares substantial genetic and functional similarity with other pathogenic human betacoronaviruses, including Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome Coronavirus (MERS-CoV) <sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>-<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>. The virus is believed to have originated in bats, although the source and animal reservoirs of SARS-CoV-2 remain uncertain <sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>-<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. No SARS-CoV-2-specific treatments or vaccine are currently available but several antiviral drugs including remdesivir are being investigated clinically.</p><p id="p-6">SARS-CoV-2 utilizes an envelope homotrimeric Spike glycoprotein (S) to interact with cellular receptor ACE2 <sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. Binding with ACE2 triggers a cascade of cell membrane fusion events for viral entry. Each S protomer consists of two subunits: a globular S1 domain at the N-terminal region, and the membrane-proximal S2 and transmembrane domains. Determinants of host range and cellular tropism are found in the RBD within the S1 domain, while mediators of membrane fusion have been identified within the S2 domain <sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. We and others have recently determined the high-resolution structure of SARS-CoV-2 RBD bound to the N-terminal peptidase domain of ACE2 <sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. The overall ACE2-binding mechanism is virtually the same between SARS-CoV-2 and SARS-CoV RBDs, indicating convergent ACE2-binding evolution between these two viruses <sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>-<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. This suggests that disruption of the RBD and ACE2 interaction would block SARS-CoV-2 entry into the target cell. Indeed, a few such disruptive agents targeted to ACE2 have been shown to inhibit SARS-CoV infection <sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. However, given the important physiological roles of ACE2 <em>in viv</em>o <sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>, these agents may have undesired side effects. Anti-RBD antibodies, on the other hand, are therefore more favorable. Furthermore, SARS-CoV or MERS-CoV RBD-based vaccine studies in experimental animals have also shown strong polyclonal antibody responses that inhibit viral entry <sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Such critical proof-of-concept findings indicate that anti-RBD antibodies should be able to effectively block SARS-CoV-2 entry. Here, we report on the isolation and characterization of 206 RBD-specific mAbs derived from single B cells of eight SARS-CoV-2 infected individuals. Using bioinformatic and biologic characterization, we identified several mAbs with potent neutralizing activity against pseudovirus and live SARS-CoV-2. However, no cross-activity between RBDs from SARS-CoV or MERS-CoV was found, suggesting that the RBD-based antibody response is viral species-specific. The potent neutralizing antibodies identified here are promising candidates for prophylactic and therapeutic SARS-CoV-2 interventions.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Plasma and B cell responses specific to SARS-CoV-2</h3><p id="p-7">We collected cross-sectional and longitudinal blood samples from eight SARS-CoV-2-infected subjects during the early outbreak in Shenzhen (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table S1</a>). Samples were named by patient number and either A, B, or C depending on collection sequence. Six patients (P#1 through P#4, P#8, and P#16) had Wuhan travel history and the remaining two (P#5 and P#22) had direct contact with those from Wuhan. P#1 through P#5 is a family cluster with the first documented case of human-to-human transmission of SARS-CoV-2 in Shenzhen <sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup>. All subjects recovered and were discharged from the hospital except for P#1 who succumbed to disease despite intensive intervention. To analyze antibody binding, serial plasma dilutions were applied to enzyme-linked immunosorbent assay (ELISA) plates coated with either recombinant RBD or trimeric Spike derived from SARS-CoV-2, SARS-CoV, and MERS-CoV or recombinant NP from SARS-CoV-2. Binding activity was visualized using anti-human IgG secondary antibodies at an optical density (OD) of 450nm. Varying degrees of binding were found across individuals and among samples from the same individual. Samples from P#1, P#2, P#5, and P#16 demonstrated higher binding to both SARS-CoV-2 RBD and NP than the rest (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Three sequential plasma samples collected from P#2 over nine days during early infection showed similar binding to SARS-CoV-2 RBD and NP and remained relative stable over the course of the infection. To our surprise, virtually no cross-reactivity between SARS-CoV RBD and MERS-CoV RBD was detected (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>), despite strong recognition by the positive control antibodies (data not shown). However, strong cross-reactivity was detected against trimeric Spikes from SARS-CoV and MERS-CoV in both ELISA (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>) and cell-surface staining (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure S1</a>). All samples except P#4A demonstrated significant levels of cross-binding to SARS-CoV trimeric Spike while only those from P#1, P#2 and P#4B cross recognized MERS-CoV trimeric Spike (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). None of the plasma samples were reactive to HIV-1 envelope trimer derived from strain BG505 <sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. The same plasma samples were also evaluated for neutralization of pseudoviruses bearing the Spike proteins of either SARS-CoV-2, SARS-CoV, or MERS-CoV. Consistent with the antibody binding results, varying degrees of neutralizing activities against SARS-CoV-2 were found across individuals (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>). However, cross-neutralizing against SARS-CoV and MERS-CoV is rather minimal as all plasma samples tested, including healthy control plasma, had negligible levels of neutralization (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). No detectable neutralization was found for any plasma sample against the pseudovirus control bearing the HIV-1 envelope MG04 (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). Taken together, these results suggest that RBDs from SARS-CoV-2, SARS-CoV, and MERS-CoV are likely to be immunologically distinct despite substantial sequence and structural similarities <sup><a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup>. Thus, regions beyond RBDs likely contribute to the observed cross-reactivity against SARS-CoV and MERS-CoV Spike protein.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analyses of plasma and B cell responses specific to SARS-CoV-2. Serial dilutions of plasma samples were analyzed for binding to the (A) RBDs or (B) trimeric Spikes of SARS-CoV-2, SARS-CoV and MERS-CoV by ELISA and (C) for neutralizing activity against pseudoviruses bearing envelope glycoprotein of SARS-CoV-2, SARS-CoV and MERS-CoV. Binding to SARS-CoV-2 NP protein was also evaluated (A). All results were derived from at least two independent experiments. (D) Gating strategy for analysis and isolation of RBD-specific memory B cells and (E) their representation among the total and memory subpopulation of B cells in the eight study subjects. Samples were named as either A, B, or C depending on collection sequence. FSC-W, forward scatter width; FSC-A, forward scatter area; and SSC-A side scatter area." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Analyses of plasma and B cell responses specific to SARS-CoV-2.&lt;/span&gt; Serial dilutions of plasma samples were analyzed for binding to the (A) RBDs or (B) trimeric Spikes of SARS-CoV-2, SARS-CoV and MERS-CoV by ELISA and (C) for neutralizing activity against pseudoviruses bearing envelope glycoprotein of SARS-CoV-2, SARS-CoV and MERS-CoV. Binding to SARS-CoV-2 NP protein was also evaluated (A). All results were derived from at least two independent experiments. (D) Gating strategy for analysis and isolation of RBD-specific memory B cells and (E) their representation among the total and memory subpopulation of B cells in the eight study subjects. Samples were named as either A, B, or C depending on collection sequence. FSC-W, forward scatter width; FSC-A, forward scatter area; and SSC-A side scatter area.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F1.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F1.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Analyses of plasma and B cell responses specific to SARS-CoV-2.</span><p id="p-8" class="first-child">Serial dilutions of plasma samples were analyzed for binding to the (A) RBDs or (B) trimeric Spikes of SARS-CoV-2, SARS-CoV and MERS-CoV by ELISA and (C) for neutralizing activity against pseudoviruses bearing envelope glycoprotein of SARS-CoV-2, SARS-CoV and MERS-CoV. Binding to SARS-CoV-2 NP protein was also evaluated (A). All results were derived from at least two independent experiments. (D) Gating strategy for analysis and isolation of RBD-specific memory B cells and (E) their representation among the total and memory subpopulation of B cells in the eight study subjects. Samples were named as either A, B, or C depending on collection sequence. FSC-W, forward scatter width; FSC-A, forward scatter area; and SSC-A side scatter area.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Flow cytometry with a range of gating strategies was used to study SARS-CoV-2-specific B cell responses and identity B cells recognizing fluorescent-labeled RBD probes (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1D</a> and <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S2</a>). As shown in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1E</a>, the RBD-specific B cells constitute about 0.005-0.065% among the total B cell population and 0.023-0.329% among the memory subpopulations. The number of RBD-specific B cells are relatively higher in P#2, P#5, P#16, and P#22 (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1E</a>), which appeared to correlate well with binding activity of corresponding plasma samples to SARS-CoV-2 RBD and trimeric Spike protein (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1A</a> and <a id="xref-fig-1-12" class="xref-fig" href="#F1">1B</a>). However, sample P#1A demonstrated the lowest RBD-specific B cell response despite high-level plasma binding. As P#1 was the only patient succumb to disease, it is uncertain whether this dichotomy of high plasma binding activity and low levels of RBD-specific B cells is a surrogate marker of rapid disease progression. This phenomenon needs study in a larger population of samples.</p></div><div id="sec-4" class="subsection"><h3>Single B cell antibody cloning and heavy chain repertoire analyses</h3><p id="p-10">We further isolated RBD-binding B cells into single cell suspension for cloning and evaluation of the mAb response (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1D</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure S2</a>). Immunoglobulin heavy and light chains were amplified by RT-PCR using nested primers. The amplified products were cloned into linear expression cassettes to produce full IgG1 antibodies as previously described <sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. The number of B cell clones varied from 10 to 106 among the subjects and each clone has been differentially represented (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S3</a>). Individual IgGs were produced by transfection of linear expression cassettes and tested for SARS-CoV-2 RBD reactivity by ELISA. On average, fifty-eight percent of the antibody clones were reactive, although great variability was found among different individuals (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure S3</a>). Out of 358 antibodies, we obtained 206 that bound to SARS-CoV-2 RBD with 165 distinct sequences (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table S2</a>). These 206 antibodies demonstrated significant differences in binding activity. For example, a large number of antibodies from samples P#2B, P#2C, P#4A, P4#B, P#5A, P#16A, and P#22A had OD 450 values well over 4.0, while none of those from sample P#1A exceeded 4.0. There were too few antibodies from P#3A and P#8A to make meaningful evaluations (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure S3</a>). Furthermore, samples from different study subjects also demonstrated substantial differences in heavy chain variable gene (VH) usage (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). For instance, P#1 samples are dominated by VH3-53, 3-13, and 1-69 which constituted approximately 21.4%, 14.3%, and 14.3% of the entire VH repertoire, respectively. Samples from P#2 and P#5 are more diverse in distribution and frequency of their VH usage. However, no single or group of VH families stood out among study subjects, suggesting patients have immunologically distinct responses to SARS-CoV-2 infection. This hypothesis is supported by the phylogenetic analysis of all 206 VH sequences superimposed with their corresponding binding activities as presented in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>. The high-binding clusters (light orange circle: 80% of clusters with OD 450 &gt; 3) were widely distributed across multiple heavy chain families. In fact, majority of the high-binding antibodies were derived by clonal expansion of specific VH families in P#2, P#4, and P#5. Similarly, the middle-(60-80% of clusters with OD 450 &gt; 3) and low- (&lt; 60% cluster with OD 450 &gt; 3) binding clusters were also widely distributed and each consisted of disproportionally represented VH gene families.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heavy chain repertoires of SARS-CoV-2 RBD-specific antibodies analyzed (A) by individual subject or (B) across the eight subjects. (A) Distribution and frequency of heavy chain variable (VH) genes usage in each subject shown along the horizontal bar. The same color scheme is used for each VH family across all study subjects. The VHs that dominate across isolated antibodies are indicated by actual frequencies in their respective color boxes. The number of RBD-binding antibodies versus total antibodies isolated are shown on the right. (B) Clustering of VH genes and their association with ELISA binding activity across the eight subjects. Unrooted phylogenetic tree depicting the genetic relationships among all VH genes of the RBD-binding antibodies. Branch lengths are drawn to scale so that sequence relatedness can be readily assessed. Sequences from the same study subject are shown in the same color at the branch tips. Colored circles represent the proportion (light orange, &gt; 80%; light yellow, 60%-80%; light green &lt; 60%) of VH clusters that bind to SARS-CoV-2 RBD with OD 450 values larger than 3. The VH gene families for the highest binding clusters are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Heavy chain repertoires of SARS-CoV-2 RBD-specific antibodies analyzed (A) by individual subject or (B) across the eight subjects.&lt;/span&gt; (A) Distribution and frequency of heavy chain variable (VH) genes usage in each subject shown along the horizontal bar. The same color scheme is used for each VH family across all study subjects. The VHs that dominate across isolated antibodies are indicated by actual frequencies in their respective color boxes. The number of RBD-binding antibodies versus total antibodies isolated are shown on the right. (B) Clustering of VH genes and their association with ELISA binding activity across the eight subjects. Unrooted phylogenetic tree depicting the genetic relationships among all VH genes of the RBD-binding antibodies. Branch lengths are drawn to scale so that sequence relatedness can be readily assessed. Sequences from the same study subject are shown in the same color at the branch tips. Colored circles represent the proportion (light orange, &gt; 80%; light yellow, 60%-80%; light green &lt; 60%) of VH clusters that bind to SARS-CoV-2 RBD with OD 450 values larger than 3. The VH gene families for the highest binding clusters are shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F2.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F2.medium.gif" width="328" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Heavy chain repertoires of SARS-CoV-2 RBD-specific antibodies analyzed (A) by individual subject or (B) across the eight subjects.</span><p id="p-11" class="first-child">(A) Distribution and frequency of heavy chain variable (VH) genes usage in each subject shown along the horizontal bar. The same color scheme is used for each VH family across all study subjects. The VHs that dominate across isolated antibodies are indicated by actual frequencies in their respective color boxes. The number of RBD-binding antibodies versus total antibodies isolated are shown on the right. (B) Clustering of VH genes and their association with ELISA binding activity across the eight subjects. Unrooted phylogenetic tree depicting the genetic relationships among all VH genes of the RBD-binding antibodies. Branch lengths are drawn to scale so that sequence relatedness can be readily assessed. Sequences from the same study subject are shown in the same color at the branch tips. Colored circles represent the proportion (light orange, &gt; 80%; light yellow, 60%-80%; light green &lt; 60%) of VH clusters that bind to SARS-CoV-2 RBD with OD 450 values larger than 3. The VH gene families for the highest binding clusters are shown.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Broad diversity and clonal expansion of antibodies in the repertoire</h3><p id="p-12">As P#2 showed a large number of RBD-binding antibodies and was the only patient with three sequential blood samples, we conducted more detailed characterization of P#2 antibodies. Among a total of 69 antibodies from P#2, the majority (59%) were scattered across various branches and the remaining (41%) were clonally expanded into three major clusters (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Antibodies from the three time points (A, B, C) do not appear to group together but rather interdigitate among themselves, suggesting they are highly related during early infection. Three clones were significantly enriched and each constituted between 12-14% of the entire tested repertoire (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). Their heavy-chain variable regions belong to the VH1-2*06, VH3-48*02, and VH3-9*01 families. The corresponding light-chain kappa (Igk) belongs to 2-40*01/2D-40*01, 3-20*01, and light-chain lambda (Igl) to 2-14*02 with the respective joining segment kappa 4 (Jk4), Jk5 and joining segment lambda 1 (Jl1) (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table S2</a>). More importantly, these clonally expanded antibodies were identified in all three samples indicating that they are strongly selected for during infection. When comparing their representation within each cluster, VH1-2*06 and VH3-9*01 appeared to increase from approximately 33 to 45%, whereas VH3-48*02 decreased from 33 to 9% over the three time points, although the number of clones was too small for statistical significance. Interestingly, the somatic hypermutation (SHM) or germline divergence for VH1-2*06 was 0% and this cluster persisted during the study period. However, the SHM for VH3-48*02 reached as high as 9.6% and for VH3-9*01 reached 3.8% compared to the overall average of 2.2% ± 3.3 % among the 69 VH sequences. Furthermore, the CDR3 length for VH1-2*06, VH3-48*02, and VH3-9*01 was 19aa, 16aa, and 23aa, respectively, compared with the overall average of 16 ± 4aa among the 69 VH sequences. Close examination of the longest CDR3 from the VH3-9*01 cluster revealed richness in tyrosine, indicating potential hydrogen bonding and hydrophobic interactions with the surrounding residues. These results shed light on the clonal expansion and broad diversity of RBD-specific antibodies during early infection and their potential role in controlling SARS-CoV-2 infection.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Clonal expansion of specific heavy and light chain families in the P#2 antibody repertoire. (A) Phylogenetic analysis of VH (left) and VL (right) genes for all RBD-binding antibodies. Clonal expanded VH and VL clusters are paired and highlighted in three different colors. Branch lengths are drawn to scale so that sequence relatedness can be readily assessed. (B) Clonal expansion in relation to members of other VH and VL families based on somatic hypermutations (SHM) and CDR3 loop lengths. For the pie charts of VH (left) and VL (right) genes, the radii represent the CDR3 loop length and the color scale indicates the degree of SHM. Heavy and light chain repertoires for each antibody are shown along the pie circles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Clonal expansion of specific heavy and light chain families in the P#2 antibody repertoire.&lt;/span&gt; (A) Phylogenetic analysis of VH (left) and VL (right) genes for all RBD-binding antibodies. Clonal expanded VH and VL clusters are paired and highlighted in three different colors. Branch lengths are drawn to scale so that sequence relatedness can be readily assessed. (B) Clonal expansion in relation to members of other VH and VL families based on somatic hypermutations (SHM) and CDR3 loop lengths. For the pie charts of VH (left) and VL (right) genes, the radii represent the CDR3 loop length and the color scale indicates the degree of SHM. Heavy and light chain repertoires for each antibody are shown along the pie circles.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F3.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F3.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Clonal expansion of specific heavy and light chain families in the P#2 antibody repertoire.</span><p id="p-13" class="first-child">(A) Phylogenetic analysis of VH (left) and VL (right) genes for all RBD-binding antibodies. Clonal expanded VH and VL clusters are paired and highlighted in three different colors. Branch lengths are drawn to scale so that sequence relatedness can be readily assessed. (B) Clonal expansion in relation to members of other VH and VL families based on somatic hypermutations (SHM) and CDR3 loop lengths. For the pie charts of VH (left) and VL (right) genes, the radii represent the CDR3 loop length and the color scale indicates the degree of SHM. Heavy and light chain repertoires for each antibody are shown along the pie circles.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Binding and neutralizing properties of selected antibodies</h3><p id="p-14">We selected 13 of the 69 P#2 antibodies sequences based on their representation and distribution on the phylogenetic tree (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, starred). Five P#1A antibody clones were used as controls. Surface plasmon resonance (SPR) with SARS-CoV-2 RBD showed that P#2 antibodies had dissociation constants (Kd) ranging from 10<sup>−8</sup> to 10<sup>−9</sup> M while those from P#1 ranged from not detectable to 10<sup>−9</sup> M (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S4</a>). SHM did not appear to correlate with Kd; some germline clones with 0% divergence in both VH and VL genes (P2A-1A10, P2B-2G4, P2C-1A3, and P2C-1E1) had Kd values comparable to clones with higher levels of SHM. The Kd of representative clones (P2A-1A8, P2A-1A10, and P2A-1B3) from the three clonally expanded clusters fell into a similar range, suggesting that their expansion may not be driven by affinity maturation. Next, we measured each antibody for competition with ACE2 for binding to the SARS-CoV-2 RBD. Specifically, the RBD was covalently immobilized on a CM5 sensor chip and first saturated by antibody and then flowed through with soluble ACE2. Competing capacity of each antibody was measured as percent reduction in ACE2 binding with the RBD (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure S5</a>). As shown in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, the evaluated antibodies demonstrated various competing capacity with ACE2. The most powerful were P2C-1F11 and P2B-2F6, which reduced ACE2 binding about 99.2% and 98.5%, respectively. Two of the three representative antibodies from the clonal expanded clusters (P2A-1A10 and P2A-1B3) had slightly over 80% and 90% reduction, respectively. The third representative (P2A-1A8) only showed 57% reduction. Many antibodies had only limited competing power with ACE2 despite impressive Kd values, suggesting binding affinity is not predictive of ACE2 competing capacity. Control antibodies from P#1 demonstrated even lower competing power with ACE2. Surprisingly, none of the antibodies tested demonstrated cross-binding with SARS-CoV and MERS-CoV RBD except P1A-1C7 (Kd=4.85μM), for which only limited cross reactivity with SARS-CoV RBD was detected (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure S4</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1218629/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1218629/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1218629" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Binding capacity, neutralizing activity, and heavy chain gene family analysis of 18 monoclonal Abs isolated from Patient #1 and Patient #2.</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">We next studied antibody neutralizing activities against pseudoviruses bearing the Spike protein of SARS-CoV-2. Consistent with the competing capacity findings, neutralizing activity varied considerably with IC<sub>50</sub> values ranging from 0.03 to &gt; 50 μg/ml (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4B</a>, 4A and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). P2C-1F11 and P2B-2F6 were the most potent, followed by P2C-1A3 and P2C-1C10. Overall, ACE2 competing capacity correlated well with the neutralizing activities, although this correlation was not exact in some instances. Notably, no cross-neutralization was found either against pseudoviruses bearing the Spike of SARS-CoV or MERS-CoV (data not shown) nor with cell-surface staining of trimeric SARS-CoV and MERS-CoV Spike (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure S6</a>). Furthermore, we selected P2C-1F11, P2B-2F6, and P2C-1A3 for neutralizing activity analyses against live SARS-CoV-2. Consistent with their respective pseudovirus assay findings, P2C-1F11 and P2B-2F6 demonstrated potent neutralization activity while that of P2C-1A3 was somewhat lower, although it needs to be noted that CPE assay is not particularly quantitative (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>). Lastly, we determined whether these antibodies compete for similar epitopes on the SARS-CoV-2 RBD. We selected a total of six antibodies with ACE2 competitive capacities of at least 70% and analyzed them in a pairwise competition fashion using SPR. As shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure S7</a>, variable degrees of competition were found among the pairs of antibodies. P2C-1A3, for instance, was competitive against all antibodies tested with reduction capacity ranging from 52 to 76. P2C-1F11, on the other hand, was less competitive with other antibodies and in particular, only minimally competitive with P2C-1C10. P2B-2F6, another potent neutralizing antibody, was broadly competitive with all antibodies tested. These results indicate that the antibodies analyzed recognized both overlapping and distinct epitopes. Different mAbs may therefore exert their neutralizing activity through different mechanisms.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1218616/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1218616/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1218616" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">Epitope mapping of mAbs through competitive binding to SARS-CoV-2 RBD</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Antibody neutralization analyzed by pseudovirus and live SARS-CoV-2. (A) Quality control of antibody through ELISA analysis prior to neutralization assay. A serial dilution of each antibody was evaluated against SARS-CoV-2 RBD coated on the ELISA plate and their binding activity was recorded at an optical density (OD) of 450nm and 630nm. (B-C) Antibody neutralization analyzed by pseudovirus (B) or live SARS-CoV-2 (C). A serial dilution of each antibody was tested against pseudovirus while two dilutions against live SARS-CoV-2. Cytopathic effects (CPE) were observed daily and recorded on Day 2 post-exposure. Selected antibodies and their concentrations tested are indicated at the upper left corner." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Antibody neutralization analyzed by pseudovirus and live SARS-CoV-2.&lt;/span&gt; (A) Quality control of antibody through ELISA analysis prior to neutralization assay. A serial dilution of each antibody was evaluated against SARS-CoV-2 RBD coated on the ELISA plate and their binding activity was recorded at an optical density (OD) of 450nm and 630nm. (B-C) Antibody neutralization analyzed by pseudovirus (B) or live SARS-CoV-2 (C). A serial dilution of each antibody was tested against pseudovirus while two dilutions against live SARS-CoV-2. Cytopathic effects (CPE) were observed daily and recorded on Day 2 post-exposure. Selected antibodies and their concentrations tested are indicated at the upper left corner.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F4.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F4.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Antibody neutralization analyzed by pseudovirus and live SARS-CoV-2.</span><p id="p-16" class="first-child">(A) Quality control of antibody through ELISA analysis prior to neutralization assay. A serial dilution of each antibody was evaluated against SARS-CoV-2 RBD coated on the ELISA plate and their binding activity was recorded at an optical density (OD) of 450nm and 630nm. (B-C) Antibody neutralization analyzed by pseudovirus (B) or live SARS-CoV-2 (C). A serial dilution of each antibody was tested against pseudovirus while two dilutions against live SARS-CoV-2. Cytopathic effects (CPE) were observed daily and recorded on Day 2 post-exposure. Selected antibodies and their concentrations tested are indicated at the upper left corner.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-17">We characterized antibody responses in eight COVID-19 patients and isolated 206 mAbs specific to the SARS-CoV-2 RBD. Bioinformatic and biologic characterization indicates that these antibodies are derived from broad and diverse families of antibody heavy and light chains. Each individual appears to have unique pattern of distribution in the antibody repertoire without apparent preferences for particular antibody families. Each antibody clone is also differentially represented. In P#2, for whom additional analyses were conducted, we found substantial variability in the distribution and frequency of each antibody family. Some clones were identified only once whereas others underwent high degrees of clonal expansion. Some clones were virtually identical to their germline ancestors while others became more divergent during the infection period. The CDR3 length also varied among the different clones. These differences at the genetic levels corresponded with their binding and neutralizing activities. Binding affinity (Kd) fell in the range of 10<sup>−8</sup> to 10<sup>−9</sup> M, equivalent to many antibodies identified during acute infections <sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>-<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> but significantly lower than those identified during chronic HIV-1 infections <sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>-<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. However, binding affinity alone does not predict neutralizing activity. Competition with the receptor ACE2 governs antibody potency, although some degree of discrepancy does exist. In particular, the most potent antibodies, P2C-1F11 and P2B-2F6, out-competed ACE2 with close to 100% efficiency, indicating that blocking the RBD and ACE2 interaction is a useful surrogate for antibody neutralization. Among the antibodies tested, substantial variations in competition for similar RBD epitopes or regions were also found. The most potent antibody, P2C-1F11, did not seem target the same epitope as the relatively moderate antibody P2C-1C10. Thus, these two antibodies could be combined for synergistic antiviral effect. As we continue to screen more antibodies from P#2 and other study subjects, more potent and diverse antibodies are expected to be identified. These antibodies will serve as the best candidates for the development of prophylactic and therapeutic intervention against COVID-19 infection.</p><p id="p-18">Most surprising in this study was the absence of antibody cross-reactivity with RBDs from SARS-CoV and MERS-CoV. Based on the sequential and structural similarities of RBDs from SARS-CoV-2 and SARS-CoV, we predicted some degree of cross-binding and even cross-neutralization between the two viruses. However, species-specific RBD responses in SARS-CoV-2 patients do suggest that RBDs from SARS-CoV-2 and SARS-CoV are immunologically distinct. If so, antibodies and vaccines must target each viral species differently in order to achieve maximum efficacy in protecting the host from infection. Our finding somewhat resolves the question of why many previously isolated SARS-CoV antibodies failed to cross-neutralize SARS-CoV-2 despite detectable levels of binding with Spike of SARS-CoV-2<sup>36</sup>. The absence of cross-recognition between RBDs was also apparent at the plasma level. Although strong binding to SARS-CoV-2 RBD was identified, plasma samples from the study subjects failed to demonstrate appreciable cross-reactivity with either SARS-CoV or MERS-CoV RBD, highlighting the immunological distinctions among the RBDs from the three viruses. However, substantial cross-reactivity were found when the same plasma samples were applied to the trimeric Spike proteins of SARS-CoV and MERS-CoV, although this was higher with the former than the latter. This indicates that such cross-reactivity likely occurs in regions outside the RBD. Determining whether this cross-reactive response has any neutralizing or protection capacity against infection would require further investigation. Finally, despite successfully isolating and characterizing a large of number mAbs against SARS-CoV-2, we cannot draw any firm correlation between antibody response and disease status at this time. In particular, the three severe cases (P#1, P#2, and P#5) appear to have relatively higher plasma binding and neutralizing activities against SARS-CoV-2 than those with relative mild symptoms. A larger number of patients must be studied to elucidate the drivers and impact of associations between antibody response and disease progression, which will provide pivotal reference for our antibody-based intervention as well as vaccine development.</p></div><div class="section" id="sec-8"><h2 class="">Materials and Methods</h2><div id="sec-9" class="subsection"><h3>Study approval</h3><p id="p-19">This study received approval from the Research Ethics Committee of Shenzhen Third People’s Hospital, China (approval number: 2020-084). The Research Ethics Committee waived the requirement informed consent before the study started because of the urgent need to collect epidemiological and clinical data. We analyzed all the data anonymously.</p></div><div id="sec-10" class="subsection"><h3>Patients and blood samples</h3><p id="p-20">The study enrolled a total of eight patients aged 10 to 66 years old infected with SARS-CoV-2 in January 2020 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table S1</a>). A plasma sample from a healthy control was also included. Of these eight patients, six (P#1 through P#4, P#8, and P#16) had Wuhan exposure history through personal visit and two had direct contact with individuals from Wuhan. Four subjects (P#1 through P#4) were part of a family cluster (P#1 through P#5) infected while visiting Wuhan and subsequently transmitted infection to P#5 after returning to Shenzhen <sup><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a></sup>. All patients were hospitalized at Shenzhen Third People’s Hospital, the designated city hospital for treatment of COVID-19 infected patients, three to nine days after symptom onset. All patients presented with fever, fatigue, and dry cough and three (P#1, P#2 and P#5) developed severe pneumonia. Four patients (P#1, P#2, P#5, and P#22) were 60 years or older, of which three (P#1, P#2, and P#22) had underlying disease such as hypertension. SARS-CoV-2 infection status was verified by RT-PCR of nasopharyngeal swab and throat swab specimens. No patient had detectable influenza A, B, respiratory syncytial virus (RSV), or adenovirus co-infections. Chest computed tomographic scans showed varying degrees of bilateral lung patchy shadows or opacity. All patients received antiviral and corticosteroid treatments, recovered and were discharged except for P#1, who succumbed to disease in hospital. Single (P#1, P#3, P#5, P#8, P#16, and P#22) or sequential (P#2 and P#4) blood samples were collected during hospitalization and follow-up visits and separated into plasma and peripheral blood mononuclear cells (PBMCs) by Ficoll-Hypaque gradient (GE Healthcare) centrifugation. All plasma samples were heat-inactivated at 56 °C for 1h before being stored at -80 °C. PBMCs were maintained in freezing media and stored in liquid nitrogen until use.</p></div><div id="sec-11" class="subsection"><h3>Recombinant RBDs and trimeric Spike from SARS-CoV-2, SARS-CoV, and MERS-CoV and receptor ACE2</h3><p id="p-21">Recombinant RBDs and trimeric Spike for MERS-CoV, SARS-CoV, and SARS-CoV-2 and the N-terminal peptidase domain of human ACE2 (residues Ser19-Asp615) were expressed using the Bac-to-Bac baculovirus system (Invitrogen) as previously described <sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>-<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>. SARS-CoV-2 RBD (residues Arg319-Phe541) containing the gp67 secretion signal peptide and a C-terminal 6×His tag was inserted into pFastBac-Dual vectors (Invitrogen) and transformed into DH10Bac component cells. The bacmid was extracted and further transfected into Sf9 cells using Cellfectin II Reagents (Invitrogen). The recombinant viruses were harvested from the transfected supernatant and amplified to generate high-titer virus stock. Viruses were then used to infect Hi5 cells for RBD and trimeric Spike expression. Secreted RBD and trimeric Spike were harvested from the supernatant and purified by gel filtration chromatography as previously reported <sup><a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>-<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup>.</p></div><div id="sec-12" class="subsection"><h3>ELISA analysis of plasma and antibody binding to RBD, trimeric Spike, and NP proteins</h3><p id="p-22">The recombinant RBDs and trimeric Spike derived from SARS-CoV-2, SARS-CoV and MERS-CoV and the SARS-CoV-2 NP protein (Sino Biological, Beijing) were diluted to final concentrations of 0.5 μg/ml or 2μg/ml, then coated onto 96-well plates and incubated at 4°C overnight. Samples were washed with PBS-T (PBS containing 0.05% Tween 20) and blocked with blocking buffer (PBS containing 5% skim milk and 2% BSA) at RT for 1h. Either serially diluted plasma samples or isolated mAbs were added the plates and incubated at 37°C for 1h. Wells were then incubated with secondary anti-human IgG labeled with HRP (ZSGB-BIO, Beijing) and TMB substrate (Kinghawk, Beijing) and optical density (OD) was measured by a spectrophotometer at 450nm and 630nm. The serially diluted plasma from healthy individuals or mAbs against SARS-CoV, MERS-CoV or HIV-1 were used as controls.</p></div><div id="sec-13" class="subsection"><h3>Isolation of RBD-specific single B cells by FACS</h3><p id="p-23">RBD-specific single B cells were sorted as previously described <sup><a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>. In brief, PBMCs from infected individuals were collected and incubated with an antibody and RBD cocktail for identification of RBD-specific B cells. The cocktail consisted of CD19-PE-Cy7, CD3-Pacific Blue, CD8-Pacific Blue, CD14-Pacific Blue, CD27-APC-H7, IgG-FITC (BD Biosciences) and the recombinant RBD-Strep or RBD-His described above. Three consecutive staining steps were conducted. The first was a LIVE/DEAD Fixable Dead Cell Stain Kit (Invitrogen) in 50µl phosphate-buffered saline (PBS) applied at RT for 20 minutes to exclude dead cells. The second utilized an antibody and RBD cocktail for an additional 30min at 4 °C. The third staining at 4 °C for 30min involved either: Streptavidin-APC (eBioscience) and/or Streptavidin-PE (BD Biosciences) to target the Strep tag of RBD, or anti-his-APC and anti-his-PE antibodies (Abcam) to target the His tag of RBD. The stained cells were washed and resuspended in PBS before being strained through a 70μm cell mesh (BD Biosciences). RBD-specific single B cells were gated as CD19+CD3-CD8-CD14-IgG+RBD+ and sorted into 96-well PCR plates containing 20μl of lysis buffer (5 μl of 5 x first strand buffer, 0.5 μl of RNase out, 1.25 μl of 0.1 M DTT (Invitrogen) per well and 0.0625 μl of Igepal (Sigma). Plates were then snap-frozen on dry ice and stored at -80 °C until RT reaction.</p></div><div id="sec-14" class="subsection"><h3>Single B cell PCR, cloning and expression of mAbs</h3><p id="p-24">The IgG heavy and light chain variable genes were amplified by nested PCR and cloned into linear expression cassettes or expression vectors to produce full IgG1 antibodies as previously described <sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. Specifically, all second round PCR primers containing tag sequences were used to produce the linear Ig expression cassettes by overlapping PCR. Separate primer pairs containing the specific restriction enzyme cutting sites (heavy chain, 5’-AgeI/3’-SalI; kappa chain, 5’-AgeI/3’-BsiWI; and lambda chain, 5’-AgeI/3’-XhoI) were used to amplify the cloned PCR products. The PCR products were purified and cloned into the backbone of antibody expression vectors containing the constant regions of human IgG1. Overlapping PCR products of paired heavy and light chain expression cassettes were co-transfected into 293T cells (ATCC) grown in 24-well plates. Antigen-specific ELISA was used to detect the binding capacity of transfected culture supernatants to SARS-CoV-2 RBD. Monoclonal antibodies were produced by transient transfection of 293F cells (Life Technologies) with equal amounts of paired heavy and light chain plasmids. Antibodies in the culture supernatant was purified by affinity chromatography using Protein A beads columns (National Engineering Research Center for Biotechnology, Beijing) according to the manufacturer’s protocol. Concentrations were determined by BCA Protein Assay Kits (Thermo Scientific). SARS-CoV, MERS-CoV, and HIV-1 mAbs were also included as controls. SARS-CoV antibodies (S230 and m396) previously isolated by others <sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> were synthesized and sequences verified before expression in 293T cells and purification by protein A chromatography. MERS-CoV antibodies (Mab-GD33) were derived from previously reported <sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>. HIV-1 antibody VRC01 was a broadly neutralizing antibody directly isolated from a patient targeting the CD4 binding site of envelope glycoprotein <sup><a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a></sup>.</p></div><div id="sec-15" class="subsection"><h3>Antibody binding kinetics, epitope mapping, and competition with receptor ACE2 measured by SPR</h3><p id="p-25">The binding kinetics and affinity of mAbs to SARS-CoV-2 RBD were analyzed by SPR (Biacore T200, GE Healthcare). Specifically, purified RBDs were covalently immobilized to a CM5 sensor chip via amine groups in 10mM sodium acetate buffer (pH 5.0) for a final RU around 250. SPR assays were run at a flow rate of 30ml/min in HEPES buffer. The sensograms were fit in a 1:1 binding model with BIA Evaluation software (GE Healthcare). For epitope mapping, two different antibodies were sequentially injected and monitored for binding activity to determine whether the two mAbs recognized separate or closely-situated epitopes. To determine competition with the human ACE2 peptidase domain, SARS-CoV-2 RBD was immobilized to a CM5 sensor chip via amine group for a final RU around 250. Antibodies (1 μM) were injected onto the chip until binding steady-state was reached. ACE2 (2 μM), which was produced and purified as above, was then injected for 60 seconds. Blocking efficacy was determined by comparison of response units with and without prior antibody incubation.</p></div><div id="sec-16" class="subsection"><h3>Analysis of plasma and antibody binding to cell surface expressed trimeric Spike protein</h3><p id="p-26">HEK 293T cells were transfected with expression plasmid encoding the full length spike of SARS-CoV-2, SARS-CoV or MERS-CoV and incubated at 37 °C for 36 h. The cells were digested with trypsin and distributed into 96 well plates for the individual staining. Cells were washed twice with 200µl staining buffer (PBS with 2% heated-inactivated FBS) between each following steps. The cells were stained at room temperature for 30 minutes in 100 μl staining buffer with 1:100 dilutions of plasma or 20 μg/ml monoclonal antibodies. The cells were then stained with PE labeled anti-human IgG Fc secondary antibody (Biolegend) at a 1:20 dilution in 50 μl staining buffer at room temperature for 30 minutes. Finally, the cells were re-suspended and analyzed with FACS Calibur instrument (BD Biosciences, USA) and FlowJo 10 software (FlowJo, USA). HEK 293T cells without transfection were also stained as background control. S230 and m396 targeting the RBD of SARS-CoV spike <sup><a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a></sup> and Mab-GD33 targeting the RBD of MERS-CoV spike <sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a></sup> were used as positive primary antibody controls, while VRC01 targeting HIV-1 env <sup><a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">40</a></sup> was used as an irrelevant primary antibody control.</p></div><div id="sec-17" class="subsection"><h3>Neutralization activity of mAbs against pseudovirus and live SARS-CoV-2</h3><p id="p-27">SARS-CoV-2, SARS-CoV and MERS-CoV pseudovirus were generated by co-transfection of human immunodeficiency virus backbones expressing firefly luciferase (pNL43R-E-luciferase) and pcDNA3.1 (Invitrogen) expression vectors encoding the respective S proteins into 293T cells (ATCC) <sup><a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a>,<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>,<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. Viral supernatants were collected 48 h later. Viral titers were measured as luciferase activity in relative light units (Bright-Glo Luciferase Assay Vector System, Promega Biosciences). Control envelope glycoproteins derived from human immunodeficiency virus (HIV)-1 and their corresponding pseudoviruses were produced in the same manner. Control mAbs included VRC01 against HIV-1 <sup><a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">40</a></sup>; S230 and m396 against SARS-CoV <sup><a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a></sup>; and Merb-GD33 against MERS-CoV <sup><a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a></sup>. Neutralization assays were performed by incubating pseudoviruses with serial dilutions of purified mAbs at 37°C for 1h. Huh7 cells (ATCC) (approximately 1.5 × 10<sup>4</sup> per well) were added in duplicate to the virus-antibody mixture. Half-maximal inhibitory concentrations (IC<sub>50</sub>) of the evaluated mAbs were determined by luciferase activity 48h after exposure to virus-antibody mixture using GraphPad Prism 6 (GraphPad Software Inc.).</p><p id="p-28">All experiments involving live SARS-CoV-2 followed approved Biosafety Level 3 laboratory standard operating procedures. Neutralization assays against live SARS-CoV-2 were conducted using a clinical isolate (Beta/Shenzhen/SZTH-003/2020, EPI_ISL_406594 at GISAID) previously obtained from a nasopharyngeal swab of P#3. The isolate was amplified in Vero cell lines to make working stocks of the virus (1 × 10<sup>5</sup> PFU/ml). To analyze the mAb neutralizing activities, Vero E6 cells were seeded at 10<sup>4</sup>/well in 96-well culture plates and cultured at 37 °C to form a monolayer. Serial dilutions of mAbs were mixed separately with 100 PFU of SARS-CoV-2, incubated at 37 °C for 1 h, and added to the monolayer of Vero E6 cells in duplicates. Cells either unexposed to the virus or mixed with 100 PFU SARS-CoV-2 were used as negative (uninfected) and positive (infected) controls, respectively. Cytopathic effects (CPE) were observed daily and recorded on Day 2 post-exposure.</p></div><div id="sec-18" class="subsection"><h3>Gene family usage and phylogenetic analysis of mAbs</h3><p id="p-29">The program IMGT/V-QUEST (<a href="http://www.imgt.org/IMGT_vquest/vquest">http://www.imgt.org/IMGT_vquest/vquest</a>) was used to analyze germline gene, germline divergence or degree of somatic hypermutation (SHM), the framework region (FR) and the loop length of the complementarity determining region 3 (CDR3) for each antibody clone. The IgG heavy and light chain variable genes were aligned using Clustal W in the BioEdit sequence analysis package (<a href="https://bioedit.software.informer.com/7.2/">https://bioedit.software.informer.com/7.2/</a>). Phylogenetic analyses were performed by the Maximum Likelihood method using MEGA X (Molecular Evolutionary Genetics Analysis across computing platforms). Several forms of the phylogenetic trees are presented for clarity.</p></div><div id="sec-19" class="subsection"><h3>Antibody production</h3><p id="p-30">The production of antibodies was conducted as previously described <sup><a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>. The genes encoding the heavy and light chains of isolated antibodies were separately cloned into expression vectors containing IgG1 constant regions and the vectors were transiently transfected into HEK293T or 293F cells using polyethylenimine (PEI) (Sigma). After 72h, the antibodies secreted into the supernatant were collected and captured by protein A Sepharose (GE Healthcare). The bound antibodies were eluted and further purified by gel-filtration chromatography using a Superdex 200 High Performance column (GE Healthcare). The purified antibodies were either used in binding and neutralizing assays.</p></div></div><div class="section" id="sec-20"><h2 class="">Author contributions</h2><p id="p-32">LZ, ZZ, LL and SZ conceived and designed the study. BJ and QZ performed most of the experiments together with assistance from XG, RW, JY, SS, BJ, SS, and XS. XT performed live SARS-CoV-2 neutralization assay. JY, JG, JL, XW provided assistance in RBD and trimeric Spike protein production. JY and LL played critical roles in recruitment and clinical management of the study subjects. HW and JZ are in charge of sample collection and processing. YW provides additional pseudovirus assay for measuring neutralizing activity against SARS-CoV-2. BJ, QZ, ZZ and LZ had full access to data in the study, generated figures and tables, and take responsibility for the integrity and accuracy of the data presentation. LZ and ZZ wrote the manuscript. All authors reviewed and approved the final version of the manuscript.</p></div><div class="section" id="sec-21"><h2 class="">Data availability statements</h2><p id="p-33">We are happy to share reagents and information presented in this study upon request.</p></div><div class="section" id="sec-22"><h2 class="">Conflict of interests</h2><p id="p-34">We declare no competing interest.</p></div><div class="section" id="sec-23"><h2 class="">Figure Legends</h2><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of plasma binding to cell surface expressed trimeric Spike protein. HEK 293T cells transfected with expression plasmid encoding the full length spike of SARS-CoV-2, SARS-CoV or MERS-CoV were incubated with 1:100 dilutions of plasma from the study subjects. The cells were then stained with PE labeled anti-human IgG Fc secondary antibody and analyzed by FACS. Positive control antibodies include S230 and m396 targeting the RBD of SARS-CoV Spike, and Mab-GD33 targeting the RBD of MERS-CoV Spike. VRC01 is negative control antibody targeting HIV-1 envelope glycoprotein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Analysis of plasma binding to cell surface expressed trimeric Spike protein.&lt;/span&gt; HEK 293T cells transfected with expression plasmid encoding the full length spike of SARS-CoV-2, SARS-CoV or MERS-CoV were incubated with 1:100 dilutions of plasma from the study subjects. The cells were then stained with PE labeled anti-human IgG Fc secondary antibody and analyzed by FACS. Positive control antibodies include S230 and m396 targeting the RBD of SARS-CoV Spike, and Mab-GD33 targeting the RBD of MERS-CoV Spike. VRC01 is negative control antibody targeting HIV-1 envelope glycoprotein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F5.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F5.medium.gif" width="404" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1.</span> <span class="caption-title">Analysis of plasma binding to cell surface expressed trimeric Spike protein.</span><p id="p-35" class="first-child">HEK 293T cells transfected with expression plasmid encoding the full length spike of SARS-CoV-2, SARS-CoV or MERS-CoV were incubated with 1:100 dilutions of plasma from the study subjects. The cells were then stained with PE labeled anti-human IgG Fc secondary antibody and analyzed by FACS. Positive control antibodies include S230 and m396 targeting the RBD of SARS-CoV Spike, and Mab-GD33 targeting the RBD of MERS-CoV Spike. VRC01 is negative control antibody targeting HIV-1 envelope glycoprotein.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RBD-specific memory B cells analyzed and isolated through FACS. The recombinant RBD was labeled with either a Strep or His tag and used alone or in combination to identify and isolate RBD-specific single B cells through staining with the Streptavidin-APC and/or Streptavidin-PE, or anti-His-APC and anti-His-PE antibodies. B cells to be isolated are highlighted in boxes or ovals. Samples were named as either A, B, or C depending on collection sequence. FSC-W, forward scatter width; FSC-A, forward scatter area; and SSC-A side scatter area." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;RBD-specific memory B cells analyzed and isolated through FACS.&lt;/span&gt; The recombinant RBD was labeled with either a Strep or His tag and used alone or in combination to identify and isolate RBD-specific single B cells through staining with the Streptavidin-APC and/or Streptavidin-PE, or anti-His-APC and anti-His-PE antibodies. B cells to be isolated are highlighted in boxes or ovals. Samples were named as either A, B, or C depending on collection sequence. FSC-W, forward scatter width; FSC-A, forward scatter area; and SSC-A side scatter area.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F6.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F6.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.</span> <span class="caption-title">RBD-specific memory B cells analyzed and isolated through FACS.</span><p id="p-36" class="first-child">The recombinant RBD was labeled with either a Strep or His tag and used alone or in combination to identify and isolate RBD-specific single B cells through staining with the Streptavidin-APC and/or Streptavidin-PE, or anti-His-APC and anti-His-PE antibodies. B cells to be isolated are highlighted in boxes or ovals. Samples were named as either A, B, or C depending on collection sequence. FSC-W, forward scatter width; FSC-A, forward scatter area; and SSC-A side scatter area.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ELISA screening of SARS-CoV-2 RBD-specific antibodies in the supernatant of transfected cells. The study subjects and the date of sampling are indicated on the top. Samples were named as either A, B, or C depending on collection sequence. Antibodies tested for each sample are aligned in one vertical column whenever possible. For each evaluated antibody, at least two independent measurements were performed and are presented adjacently on the same row. Binding activities were assessed by OD 450 and indicated by the color scheme on the right. Negatives (no binding activity) are shown in gray for OD 450 values less than 0.1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;ELISA screening of SARS-CoV-2 RBD-specific antibodies in the supernatant of transfected cells.&lt;/span&gt; The study subjects and the date of sampling are indicated on the top. Samples were named as either A, B, or C depending on collection sequence. Antibodies tested for each sample are aligned in one vertical column whenever possible. For each evaluated antibody, at least two independent measurements were performed and are presented adjacently on the same row. Binding activities were assessed by OD 450 and indicated by the color scheme on the right. Negatives (no binding activity) are shown in gray for OD 450 values less than 0.1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F7.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F7.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3.</span> <span class="caption-title">ELISA screening of SARS-CoV-2 RBD-specific antibodies in the supernatant of transfected cells.</span><p id="p-37" class="first-child">The study subjects and the date of sampling are indicated on the top. Samples were named as either A, B, or C depending on collection sequence. Antibodies tested for each sample are aligned in one vertical column whenever possible. For each evaluated antibody, at least two independent measurements were performed and are presented adjacently on the same row. Binding activities were assessed by OD 450 and indicated by the color scheme on the right. Negatives (no binding activity) are shown in gray for OD 450 values less than 0.1.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Binding kinetics of isolated mAbs with SARS-CoV-2 RBD measured by SPR. The purified soluble SARS-CoV-2 RBD were covalently immobilized onto a CM5 sensor chip followed by injection of individual antibody at four or five different concentrations. The black lines indicate the experimentally derived curves while the red lines represent fitted curves based on the experimental data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Binding kinetics of isolated mAbs with SARS-CoV-2 RBD measured by SPR.&lt;/span&gt; The purified soluble SARS-CoV-2 RBD were covalently immobilized onto a CM5 sensor chip followed by injection of individual antibody at four or five different concentrations. The black lines indicate the experimentally derived curves while the red lines represent fitted curves based on the experimental data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F8.medium.gif" width="349" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F8.medium.gif" width="349" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4.</span> <span class="caption-title">Binding kinetics of isolated mAbs with SARS-CoV-2 RBD measured by SPR.</span><p id="p-38" class="first-child">The purified soluble SARS-CoV-2 RBD were covalently immobilized onto a CM5 sensor chip followed by injection of individual antibody at four or five different concentrations. The black lines indicate the experimentally derived curves while the red lines represent fitted curves based on the experimental data.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Antibody and ACE2 competition for binding to SARS-CoV-2 RBD measured by SPR. The sensorgrams show distinct binding patterns of ACE2 to SARS-CoV-2 RBD with (red curve) or without (black curve) prior incubation with each testing antibody. The competition capacity of each antibody is indicated by the level of reduction in response unit of ACE2 comparing with or without prior antibody incubation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Antibody and ACE2 competition for binding to SARS-CoV-2 RBD measured by SPR.&lt;/span&gt; The sensorgrams show distinct binding patterns of ACE2 to SARS-CoV-2 RBD with (red curve) or without (black curve) prior incubation with each testing antibody. The competition capacity of each antibody is indicated by the level of reduction in response unit of ACE2 comparing with or without prior antibody incubation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F9.medium.gif" width="373" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F9.medium.gif" width="373" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5.</span> <span class="caption-title">Antibody and ACE2 competition for binding to SARS-CoV-2 RBD measured by SPR.</span><p id="p-39" class="first-child">The sensorgrams show distinct binding patterns of ACE2 to SARS-CoV-2 RBD with (red curve) or without (black curve) prior incubation with each testing antibody. The competition capacity of each antibody is indicated by the level of reduction in response unit of ACE2 comparing with or without prior antibody incubation.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of antibody binding to cell surface expressed trimeric Spike protein. HEK 293T cells transfected with expression plasmid encoding the full length spike of SARS-CoV-2, SARS-CoV or MERS-CoV were incubated with 20ug/ml testing antibodies. The cells were then stained with PE labeled anti-human IgG Fc secondary antibody and analyzed by FACS. Positive control antibodies include S230 and m396 targeting the RBD of SARS-CoV Spike, and Mab-GD33 targeting the RBD of MERS-CoV Spike. VRC01 is the negative control antibody targeting HIV-1 envelope glycoprotein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Analysis of antibody binding to cell surface expressed trimeric Spike protein.&lt;/span&gt; HEK 293T cells transfected with expression plasmid encoding the full length spike of SARS-CoV-2, SARS-CoV or MERS-CoV were incubated with 20ug/ml testing antibodies. The cells were then stained with PE labeled anti-human IgG Fc secondary antibody and analyzed by FACS. Positive control antibodies include S230 and m396 targeting the RBD of SARS-CoV Spike, and Mab-GD33 targeting the RBD of MERS-CoV Spike. VRC01 is the negative control antibody targeting HIV-1 envelope glycoprotein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F10.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F10.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6.</span> <span class="caption-title">Analysis of antibody binding to cell surface expressed trimeric Spike protein.</span><p id="p-40" class="first-child">HEK 293T cells transfected with expression plasmid encoding the full length spike of SARS-CoV-2, SARS-CoV or MERS-CoV were incubated with 20ug/ml testing antibodies. The cells were then stained with PE labeled anti-human IgG Fc secondary antibody and analyzed by FACS. Positive control antibodies include S230 and m396 targeting the RBD of SARS-CoV Spike, and Mab-GD33 targeting the RBD of MERS-CoV Spike. VRC01 is the negative control antibody targeting HIV-1 envelope glycoprotein.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Epitope mapping through competitive binding measured by SPR. The sensorgrams show distinct binding patterns when pairs of testing antibodies were sequentially applied to the purified SARS-CoV-2 RBD covalently immobilized onto a CM5 sensor chip. The level of reduction in response unit comparing with or without prior antibody incubation is the key criteria for determining the two mAbs recognize the separate or closely situated epitopes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1803230040" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Epitope mapping through competitive binding measured by SPR.&lt;/span&gt; The sensorgrams show distinct binding patterns when pairs of testing antibodies were sequentially applied to the purified SARS-CoV-2 RBD covalently immobilized onto a CM5 sensor chip. The level of reduction in response unit comparing with or without prior antibody incubation is the key criteria for determining the two mAbs recognize the separate or closely situated epitopes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F11.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F11.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.03.21.990770/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7.</span> <span class="caption-title">Epitope mapping through competitive binding measured by SPR.</span><p id="p-41" class="first-child">The sensorgrams show distinct binding patterns when pairs of testing antibodies were sequentially applied to the purified SARS-CoV-2 RBD covalently immobilized onto a CM5 sensor chip. The level of reduction in response unit comparing with or without prior antibody incubation is the key criteria for determining the two mAbs recognize the separate or closely situated epitopes.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1218624/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1218624/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1218624" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1</span> <span class="caption-title">Clinical characterization of the study subjects</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1218618/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1218618/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S2.</span> <span class="caption-title">Gene family analysis of monoclonal antibodies.</span><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">We acknowledge the work and contribution of all the health providers from Shenzhen Third People’s Hospital. We also thank patients for their active participation. This study was supported by Bill &amp; Melinda Gates Foundation, the Science and Technology Innovation Committee of Shenzhen Municipality (202002073000002), and by Tsinghua University Initiative Scientific Research Program (20201080053). This work is also partially supported by the National Natural Science Foundation Award (81530065), Beijing Municipal Science and Technology Commission (171100000517-001 and -003), Beijing Advanced Innovation Center for Structural Biology at Tsinghua University, the National Key Plan for Scientific Research and Development of China (grant number 2016YFD0500307), Tencent Foundation, Shuidi Foundation, and TH Capital. The funders had no role in study design, data collection, data analysis, data interpretation, or writing of the report.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-4">Figure 4 revised.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.1" data-doi="10.1056/NEJMoa2001316"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Q.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, 10.1056/NEJMoa2001316<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001316</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001316%26rft_id%253Dinfo%253Apmid%252F31995857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31995857&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.2" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">270</span>–<span class="cit-lpage">273</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.3" data-doi="10.1056/NEJMoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-vol">382</span>, <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001017</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252F31978945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31978945&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.4" data-doi="10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.5" data-doi="10.1016/S0140-6736(20)30154-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">J. F.-W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">514</span>–<span class="cit-lpage">523</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30154-9</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930154-9%26rft_id%253Dinfo%253Apmid%252F31986261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30154-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31986261&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.6" data-doi="10.1056/NEJMoa2002032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">W.-J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical Characteristics of Coronavirus Disease 2019 in China</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, 10.1056/NEJMoa2002032<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2002032</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2002032%26rft_id%253Dinfo%253Apmid%252F32109013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2002032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32109013&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.7" data-doi="10.1016/S0140-6736(20)30183-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">497</span>–<span class="cit-lpage">506</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30183-5</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252F31986264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31986264&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.8" data-doi="10.1001/jama.2020.1585"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">JAMA</abbr>, <span class="cit-fpage">e201585</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1001/jama.2020.1585</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2020.1585%26rft_id%253Dinfo%253Apmid%252F32031570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.2020.1585&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32031570&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.9" data-doi="10.1126/science.aba9757"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chinazzi</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The effect of travel restrictions on the spread of the 2019 novel coronavirus (COVID-19) outbreak</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-fpage">eaba9757</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aba9757</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aba9757%26rft_id%253Dinfo%253Apmid%252F32144116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiJzY2llbmNlLmFiYTk3NTd2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI2LzIwMjAuMDMuMjEuOTkwNzcwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.10" data-doi="10.1016/S0140-6736(20)30251-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">565</span>–<span class="cit-lpage">574</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30251-8</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930251-8%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30251-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.11" data-doi="10.1016/j.chom.2020.02.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>, S1931-3128(1920)30072-X<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2020.02.001</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2020.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2020.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.12" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X.-Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">503</span>, <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature12711</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.13" data-doi="10.1016/j.cell.2020.02.052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, S0092-8674(0020)30229-30224<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2020.02.052</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.02.052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.02.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.14" data-doi="10.1016/j.cell.2020.02.058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, S0092-8674(0020)30262-30262<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2020.02.058</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.02.058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.02.058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.15" data-doi="10.1038/nrmicro2090"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The spike protein of SARS-CoV--a target for vaccine and therapeutic development</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">226</span>–<span class="cit-lpage">236</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrmicro2090</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDu%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D226%26rft.epage%253D236%26rft.atitle%253DThe%2Bspike%2Bprotein%2Bof%2BSARS-CoV--a%2Btarget%2Bfor%2Bvaccine%2Band%2Btherapeutic%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2090%26rft_id%253Dinfo%253Apmid%252F19198616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19198616&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.16" data-doi="10.1146/annurev-virology-110615-042301"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">237</span>–<span class="cit-lpage">261</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-virology-110615-042301</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D3%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-110615-042301%26rft_id%253Dinfo%253Apmid%252F27578435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-virology-110615-042301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27578435&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.17" data-doi="10.1126/science.abb2507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science, eabb2507</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.abb2507</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%252C%2Beabb2507%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4My8xMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjYvMjAyMC4wMy4yMS45OTA3NzAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.18" data-doi="10.1038/cr.2016.152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gui</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">129</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/cr.2016.152</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D27%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.152%26rft_id%253Dinfo%253Apmid%252F28008928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2016.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28008928&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.19" data-doi="10.1371/journal.ppat.1007236"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gui</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">e1007236</span>–<span class="cit-lpage">e1007236</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1007236</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007236%26rft_id%253Dinfo%253Apmid%252F30102747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30102747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.20" data-doi="10.1038/s41598-018-36918-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>,  <span class="cit-name-given-names">R. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">15701</span>–<span class="cit-lpage">15701</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-018-34171-7</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D15701%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-36918-8%26rft_id%253Dinfo%253Apmid%252F30356097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-36918-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30356097&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.21" data-doi="10.1038/ncomms15092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">15092</span>–<span class="cit-lpage">15092</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/ncomms15092</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D15092%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15092%26rft_id%253Dinfo%253Apmid%252F28393837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28393837&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.22" data-doi="10.1128/JVI.00127-20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, JVI.00127-00120<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00127-20</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00127-20%26rft_id%253Dinfo%253Apmid%252F31996437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjE0OiI5NC83L2UwMDEyNy0yMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI2LzIwMjAuMDMuMjEuOTkwNzcwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.23" data-doi="10.12688/f1000research.22211.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kruse</span>,  <span class="cit-name-given-names">R. L.</span></span> <span class="cit-article-title">Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">F1000Res</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">72</span>–<span class="cit-lpage">72</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.12688/f1000research.22211.2</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000Res%26rft.volume%253D9%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.22211.2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.22211.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.24" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature02145</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.25" data-doi="10.1002/path.2162"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hamming</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The emerging role of ACE2 in physiology and disease</span>. <abbr class="cit-jnl-abbrev">J Pathol</abbr> <span class="cit-vol">212</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/path.2162</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bpathology%26rft.stitle%253DJ%2BPathol%26rft.aulast%253DHamming%26rft.auinit1%253DI.%26rft.volume%253D212%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D11%26rft.atitle%253DThe%2Bemerging%2Brole%2Bof%2BACE2%2Bin%2Bphysiology%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpath.2162%26rft_id%253Dinfo%253Apmid%252F17464936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/path.2162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17464936&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246415200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.26" data-doi="10.1080/22221751.2019.1624482"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Antibodies and vaccines against Middle East respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">841</span>–<span class="cit-lpage">856</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1080/22221751.2019.1624482</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D8%26rft.spage%253D841%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2019.1624482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/22221751.2019.1624482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.27" data-doi="10.1128/JVI.76.17.8875-8889.2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanders</span>,  <span class="cit-name-given-names">R. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Stabilization of the soluble, cleaved, trimeric form of the envelope glycoprotein complex of human immunodeficiency virus type 1</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">76</span>, <span class="cit-fpage">8875</span>–<span class="cit-lpage">8889</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.76.17.8875-8889.2002</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSanders%26rft.auinit1%253DR.%2BW.%26rft.volume%253D76%26rft.issue%253D17%26rft.spage%253D8875%26rft.epage%253D8889%26rft.atitle%253DStabilization%2Bof%2Bthe%2BSoluble%252C%2BCleaved%252C%2BTrimeric%2BForm%2Bof%2Bthe%2BEnvelope%2BGlycoprotein%2BComplex%2Bof%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.76.17.8875-8889.2002%26rft_id%253Dinfo%253Apmid%252F12163607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ni8xNy84ODc1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjYvMjAyMC4wMy4yMS45OTA3NzAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.28" data-doi="10.1016/j.immuni.2016.03.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Key gp120 Glycans Pose Roadblocks to the Rapid Development of VRC01-Class Antibodies in an HIV-1-Infected Chinese Donor</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">939</span>–<span class="cit-lpage">950</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.immuni.2016.03.006</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D44%26rft.spage%253D939%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2016.03.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2016.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.29" data-doi="10.1016/j.jviromet.2009.02.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">H.-X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">High-throughput isolation of immunoglobulin genes from single human B cells and expression as monoclonal antibodies</span>. <abbr class="cit-jnl-abbrev">J Virol Methods</abbr> <span class="cit-vol">158</span>, <span class="cit-fpage">171</span>–<span class="cit-lpage">179</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jviromet.2009.02.014</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.stitle%253DJ%2BVirol%2BMethods%26rft.aulast%253DLiao%26rft.auinit1%253DH.%2BX.%26rft.volume%253D158%26rft.issue%253D1-2%26rft.spage%253D171%26rft.epage%253D179%26rft.atitle%253DHigh-throughput%2Bisolation%2Bof%2Bimmunoglobulin%2Bgenes%2Bfrom%2Bsingle%2Bhuman%2BB%2Bcells%2Band%2Bexpression%2Bas%2Bmonoclonal%2Bantibodies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2009.02.014%26rft_id%253Dinfo%253Apmid%252F19428587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2009.02.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19428587&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266349200028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.30" data-doi="10.1172/jci.insight.93042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Delineating antibody recognition against Zika virus during natural infection</span>. <abbr class="cit-jnl-abbrev">JCI Insight</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">e93042</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1172/jci.insight.93042</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJCI%2BInsight%26rft_id%253Dinfo%253Adoi%252F10.1172%252Fjci.insight.93042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/jci.insight.93042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.31" data-doi="10.1146/annurev-immunol-032712-095916"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corti</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Broadly neutralizing antiviral antibodies</span>. <abbr class="cit-jnl-abbrev">Annu Rev Immunol</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">705</span>–<span class="cit-lpage">742</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-immunol-032712-095916</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BImmunol%26rft.volume%253D31%26rft.spage%253D705%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-immunol-032712-095916%26rft_id%253Dinfo%253Apmid%252F23330954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-immunol-032712-095916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23330954&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321861600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.32" data-doi="10.1126/science.aaf8505"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stettler</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Specificity, cross-reactivity, and function of antibodies elicited by Zika virus infection</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>, <span class="cit-fpage">823</span>–<span class="cit-lpage">826</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aaf8505</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf8505%26rft_id%253Dinfo%253Apmid%252F27417494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTMvNjMwMS84MjMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNi8yMDIwLjAzLjIxLjk5MDc3MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.33" data-doi="10.1038/nature07930"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scheid</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Broad diversity of neutralizing antibodies isolated from memory B cells in HIV-infected individuals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>, <span class="cit-fpage">636</span>–<span class="cit-lpage">640</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature07930</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DScheid%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D458%26rft.issue%253D7238%26rft.spage%253D636%26rft.epage%253D640%26rft.atitle%253DBroad%2Bdiversity%2Bof%2Bneutralizing%2Bantibodies%2Bisolated%2Bfrom%2Bmemory%2BB%2Bcells%2Bin%2BHIV-infected%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07930%26rft_id%253Dinfo%253Apmid%252F19287373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19287373&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264796200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.34" data-doi="10.1126/science.1207532"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Focused evolution of HIV-1 neutralizing antibodies revealed by structures and deep sequencing</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">333</span>, <span class="cit-fpage">1593</span>–<span class="cit-lpage">1602</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1207532</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1207532%26rft_id%253Dinfo%253Apmid%252F21835983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzMvNjA0OS8xNTkzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjYvMjAyMC4wMy4yMS45OTA3NzAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.35" data-doi="10.1038/nature12053"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">H.-X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Co-evolution of a broadly neutralizing HIV-1 antibody and founder virus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">496</span>, <span class="cit-fpage">469</span>–<span class="cit-lpage">476</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature12053</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12053%26rft_id%253Dinfo%253Apmid%252F23552890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23552890&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.21.990770v2.36" data-doi="10.1080/22221751.2020.1729069"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">382</span>–<span class="cit-lpage">385</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1080/22221751.2020.1729069</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D382%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2020.1729069%26rft_id%253Dinfo%253Apmid%252F32065055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/22221751.2020.1729069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32065055&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.37" data-doi="10.1038/cr.2013.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure of MERS-CoV spike receptor-binding domain complexed with human receptor DPP4</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">986</span>–<span class="cit-lpage">993</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/cr.2013.92</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2013.92%26rft_id%253Dinfo%253Apmid%252F23835475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2013.92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23835475&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.38" data-doi="10.1126/scitranslmed.3008140"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Potent neutralization of MERS-CoV by human neutralizing monoclonal antibodies to the viral spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">Sci Transl Med</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">234ra259</span>–<span class="cit-lpage">234ra259</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/scitranslmed.3008140</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BTransl%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.3008140%26rft_id%253Dinfo%253Apmid%252F24778414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjEzOiI2LzIzNC8yMzRyYTU5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjYvMjAyMC4wMy4yMS45OTA3NzAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.39" data-doi="10.1016/j.celrep.2018.06.041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural Definition of a Unique Neutralization Epitope on the Receptor-Binding Domain of MERS-CoV Spike Glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">441</span>–<span class="cit-lpage">452</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.celrep.2018.06.041</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D24%26rft.spage%253D441%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2018.06.041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2018.06.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.40" data-doi="10.1126/science.1187659"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Rational design of envelope identifies broadly neutralizing human monoclonal antibodies to HIV-1</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>, <span class="cit-fpage">856</span>–<span class="cit-lpage">861</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1187659</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1187659%26rft_id%253Dinfo%253Apmid%252F20616233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5My84NTYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNi8yMDIwLjAzLjIxLjk5MDc3MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.41" data-doi="10.1016/j.jim.2007.09.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tiller</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Efficient generation of monoclonal antibodies from single human B cells by single cell RT-PCR and expression vector cloning</span>. <abbr class="cit-jnl-abbrev">J Immunol Methods</abbr> <span class="cit-vol">329</span>, <span class="cit-fpage">112</span>–<span class="cit-lpage">124</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jim.2007.09.017</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bimmunological%2Bmethods%26rft.stitle%253DJ%2BImmunol%2BMethods%26rft.aulast%253DTiller%26rft.auinit1%253DT.%26rft.volume%253D329%26rft.issue%253D1-2%26rft.spage%253D112%26rft.epage%253D124%26rft.atitle%253DEfficient%2Bgeneration%2Bof%2Bmonoclonal%2Bantibodies%2Bfrom%2Bsingle%2Bhuman%2BB%2Bcells%2Bby%2Bsingle%2Bcell%2BRT-PCR%2Band%2Bexpression%2Bvector%2Bcloning.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jim.2007.09.017%26rft_id%253Dinfo%253Apmid%252F17996249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jim.2007.09.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17996249&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253124100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.42" data-doi="10.1073/pnas.0701000104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Potent cross-reactive neutralization of SARS coronavirus isolates by human monoclonal antibodies</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">104</span>, <span class="cit-fpage">12123</span>–<span class="cit-lpage">12128</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0701000104</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701000104%26rft_id%253Dinfo%253Apmid%252F17620608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI5LzEyMTIzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjYvMjAyMC4wMy4yMS45OTA3NzAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.43" data-doi="10.1093/infdis/jiy311"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Ultrapotent Human Neutralizing Antibody Repertoires Against Middle East Respiratory Syndrome Coronavirus From a Recovered Patient</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-vol">218</span>, <span class="cit-fpage">1249</span>–<span class="cit-lpage">1260</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/infdis/jiy311</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft.volume%253D218%26rft.spage%253D1249%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjiy311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jiy311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.44" data-doi="10.1080/22221751.2019.1620083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Single intranasal immunization with chimpanzee adenovirus-based vaccine induces sustained and protective immunity against MERS-CoV infection</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">760</span>–<span class="cit-lpage">772</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1080/22221751.2019.1620083</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D8%26rft.spage%253D760%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2019.1620083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/22221751.2019.1620083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.45" data-doi="10.1002/jmv.20499"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Antibody responses against SARS coronavirus are correlated with disease outcome of infected individuals</span>. <abbr class="cit-jnl-abbrev">J Med Virol</abbr> <span class="cit-vol">78</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.20499</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedical%2Bvirology%26rft.stitle%253DJ%2BMed%2BVirol%26rft.aulast%253DZhang%26rft.auinit1%253DL.%26rft.volume%253D78%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D8%26rft.atitle%253DAntibody%2Bresponses%2Bagainst%2BSARS%2Bcoronavirus%2Bare%2Bcorrelated%2Bwith%2Bdisease%2Boutcome%2Bof%2Binfected%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.20499%26rft_id%253Dinfo%253Apmid%252F16299724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.20499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16299724&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.21.990770v2.46" data-doi="10.1038/srep25856"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Q.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Potent neutralizing monoclonal antibodies against Ebola virus infection</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">25856</span>–<span class="cit-lpage">25856</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/srep25856</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D25856%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep25856%26rft_id%253Dinfo%253Apmid%252F27181584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep25856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27181584&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.03.21.990770.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.21.990770v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.25.998963v2" title="Cryo-EM structure of coronavirus-HKU1 haemagglutinin esterase reveals architectural changes arising from prolonged circulation in humans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.25.008078v1" title="A dynamic 6,000-year genetic history of Eurasia’s Eastern Steppe" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 26, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.21.990770v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.21.990770v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.21.990770v2" class="active" data-icon-position="" data-hide-link-title="0">Potent human neutralizing antibodies elicited by SARS-CoV-2 infection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1212177" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gRtvXJz3nUk408jPI7D4YUYJAst0tTbiDxj-27b78-g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1212177" id="node1212177--2" data-pisa="biorxiv;2020.03.21.990770v2" data-pisa-master="biorxiv;2020.03.21.990770" data-seqnum="1212177" data-apath="/biorxiv/early/2020/03/26/2020.03.21.990770.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Potent human neutralizing antibodies elicited by SARS-CoV-2 infection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Bin</span> <span class="nlm-surname">Ju</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Qi</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiangyang</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ruoke</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jiazhen</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sisi</span> <span class="nlm-surname">Shan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Bing</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Shuo</span> <span class="nlm-surname">Song</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xian</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jinfang</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Jiwan</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Jun</span> <span class="nlm-surname">Lan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Yuan</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Haiyan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Juanjuan</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Shuye</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Youchun</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Xuanling</span> <span class="nlm-surname">Shi</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Lei</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Xinquan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Zheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Linqi</span> <span class="nlm-surname">Zhang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.21.990770; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.21.990770 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;title=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;title=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;text=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;title=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;t=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;title=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;title=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1212177" id="node1212177--3" data-pisa="biorxiv;2020.03.21.990770v2" data-pisa-master="biorxiv;2020.03.21.990770" data-seqnum="1212177" data-apath="/biorxiv/early/2020/03/26/2020.03.21.990770.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Potent human neutralizing antibodies elicited by SARS-CoV-2 infection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Bin</span> <span class="nlm-surname">Ju</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Qi</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiangyang</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ruoke</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jiazhen</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sisi</span> <span class="nlm-surname">Shan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Bing</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Shuo</span> <span class="nlm-surname">Song</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xian</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jinfang</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Jiwan</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Jun</span> <span class="nlm-surname">Lan</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Yuan</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Haiyan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Juanjuan</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Shuye</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Youchun</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Xuanling</span> <span class="nlm-surname">Shi</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Lei</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Xinquan</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Zheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Linqi</span> <span class="nlm-surname">Zhang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.21.990770; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.21.990770 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Ju%2C+Bin&amp;rft.au=Zhang%2C+Qi&amp;rft.au=Ge%2C+Xiangyang&amp;rft.au=Wang%2C+Ruoke&amp;rft.au=Yu%2C+Jiazhen&amp;rft.au=Shan%2C+Sisi&amp;rft.au=Zhou%2C+Bing&amp;rft.au=Song%2C+Shuo&amp;rft.au=Tang%2C+Xian&amp;rft.au=Yu%2C+Jinfang&amp;rft.au=Ge%2C+Jiwan&amp;rft.au=Lan%2C+Jun&amp;rft.au=Yuan%2C+Jing&amp;rft.au=Wang%2C+Haiyan&amp;rft.au=Zhao%2C+Juanjuan&amp;rft.au=Zhang%2C+Shuye&amp;rft.au=Wang%2C+Youchun&amp;rft.au=Shi%2C+Xuanling&amp;rft.au=Liu%2C+Lei&amp;rft.au=Wang%2C+Xinquan&amp;rft.au=Zhang%2C+Zheng&amp;rft.au=Zhang%2C+Linqi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1212177/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1212177/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1212177/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1212177/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1212177/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1212177/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1212177/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1212177/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1212177/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1212177/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1212177/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1212177/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;count=horizontal&amp;via=&amp;text=Potent%20human%20neutralizing%20antibodies%20elicited%20by%20SARS-CoV-2%20infection&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.21.990770v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.21.990770v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
