<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/26/2020.02.15.951012.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.15.951012v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Solubility-Weighted Index: fast and accurate prediction of protein solubility"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.15.951012"/>
  <meta name="DC.Date" content="2020-03-26"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Motivation Recombinant protein production is a widely used technique in the biotechnology and biomedical industries, yet only a quarter of target proteins are soluble and can therefore be purified. Results We have discovered that global structural flexibility, which can be modeled by normalised B-factors, accurately predicts the solubility of 12,216 recombinant proteins expressed in Escherichia coli . We have optimised B-factors, and derived a new set of values for solubility scoring that further improves prediction accuracy. We call this new predictor the ‘Solubility-Weighted Index’ (SWI). Importantly, SWI outperforms many existing protein solubility prediction tools. Furthermore, we have developed ‘SoDoPE’ (Soluble Domain for Protein Expression), a web interface that allows users to choose a protein region of interest for predicting and maximising both protein expression and solubility. Availability The SoDoPE web server and source code are freely available at &lt;;https://tisigner.com/sodope&gt;; and &lt;;https://github.com/Gardner-BinfLab/TISIGNER-ReactJS&gt;;, respectively. The code and data for reproducing our analysis can be found at &lt;;https://github.com/Gardner-BinfLab/SoDoPE_paper2020&gt;;."/>
  <meta name="DC.Contributor" content="Bikash K. Bhandari"/>
  <meta name="DC.Contributor" content="Paul P. Gardner"/>
  <meta name="DC.Contributor" content="Chun Shen Lim"/>
  <meta name="article:published_time" content="2020-03-26"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Solubility-Weighted Index: fast and accurate prediction of protein solubility"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Motivation: Recombinant protein production is a widely used technique in the biotechnology and biomedical industries, yet only a quarter of target proteins are soluble and can therefore be purified. Results: We have discovered that global structural flexibility, which can be modeled by normalised B-factors, accurately predicts the solubility of 12,216 recombinant proteins expressed in &lt;;i&gt;;Escherichia coli&lt;;/i&gt;;. We have optimised B-factors, and derived a new set of values for solubility scoring that further improves prediction accuracy. We call this new predictor the 9Solubility-Weighted Index9 (SWI). Importantly, SWI outperforms many existing protein solubility prediction tools. Furthermore, we have developed 9SoDoPE9 (Soluble Domain for Protein Expression), a web interface that allows users to choose a protein region of interest for predicting and maximising both protein expression and solubility.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.15.951012v2"/>
  <meta name="citation_id" content="2020.02.15.951012v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.15.951012v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.15.951012v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.15.951012v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.15.951012"/>
  <meta name="citation_num_pages" content="19"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.15.951012"/>
  <meta name="citation_author" content="Bikash K. Bhandari"/>
  <meta name="citation_author_institution" content="Department of Biochemistry, School of Biomedical Sciences, University of Otago"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8698-5258"/>
  <meta name="citation_author" content="Paul P. Gardner"/>
  <meta name="citation_author_institution" content="Department of Biochemistry, School of Biomedical Sciences, University of Otago"/>
  <meta name="citation_author_institution" content="Biomolecular Interaction Centre, University of Canterbury"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7808-1213"/>
  <meta name="citation_author" content="Chun Shen Lim"/>
  <meta name="citation_author_institution" content="Department of Biochemistry, School of Biomedical Sciences, University of Otago"/>
  <meta name="citation_author_email" content="chunshen.lim@otago.ac.nz"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7015-0125"/>
  <meta name="citation_reference" content="Acton, Thomas B., Kristin C. Gunsalus, Rong Xiao, Li Chung Ma, James Aramini, Michael C. Baran, Yi-Wen Chiang, et al. 2005. “Robotic Cloning and Protein Production Platform of the Northeast Structural Genomics Consortium.” Methods in Enzymology 394: 210–43."/>
  <meta name="citation_reference" content="Agostini, Federico, Davide Cirillo, Carmen Maria Livi, Riccardo Delli Ponti, and Gian Gaetano Tartaglia. 2014. “ccSOL Omics: A Webserver for Solubility Prediction of Endogenous and Heterologous Expression in Escherichia Coli.” Bioinformatics 30 (20): 2975–77."/>
  <meta name="citation_reference" content="Aslund, F., and J. Beckwith. 1999. “The Thioredoxin Superfamily: Redundancy, Specificity, and Gray-Area Genomics.” Journal of Bacteriology 181 (5): 1375–79."/>
  <meta name="citation_reference" content="Bhandari, Bikash K., Chun Shen Lim, and Paul P. Gardner. 2019. “Highly Accessible Translation Initiation Sites Are Predictive of Successful Heterologous Protein Expression.” bioRxiv. https://doi.org/10.1101/726752."/>
  <meta name="citation_reference" content="Bhaskaran, R., and P. K. Ponnuswamy. 1988. “Positional Flexibilities of Amino Acid Residues in Globular Proteins.” International Journal of Peptide and Protein Research. https://doi.org/10.1111/j.1399-3011.1988.tb01258.x."/>
  <meta name="citation_reference" content="Caswell, Thomas A., Michael Droettboom, John Hunter, Eric Firing, Antony Lee, David Stansby, Elliott Sales de Andrade, et al. 2018. Matplotlib/matplotlib v3.0.2 (version 3.0.2). https://doi.org/10.5281/zenodo.1482099."/>
  <meta name="citation_reference" content="Chan, Wen-Ching, Po-Huang Liang, Yan-Ping Shih, Ueng-Cheng Yang, Wen-Chang Lin, and Chun-Nan Hsu. 2010. “Learning to Predict Expression Efficacy of Vectors in Recombinant Protein Production.” BMC Bioinformatics 11 Suppl 1 (January): S21."/>
  <meta name="citation_reference" content="Chen, Li, Rose Oughtred, Helen M. Berman, and John Westbrook. 2004. “TargetDB: A Target Registration Database for Structural Genomics Projects.” Bioinformatics 20 (16): 2860–62."/>
  <meta name="citation_reference" content="Chiti, Fabrizio, Massimo Stefani, Niccolò Taddei, Giampietro Ramponi, and Christopher M. Dobson. 2003. “Rationalization of the Effects of Mutations on Peptide and Protein Aggregation Rates.” Nature 424 (6950): 805–8."/>
  <meta name="citation_reference" content="Cock, Peter J. A., Tiago Antao, Jeffrey T. Chang, Brad A. Chapman, Cymon J. Cox, Andrew Dalke, Iddo Friedberg, et al. 2009. “Biopython: Freely Available Python Tools for Computational Molecular Biology and Bioinformatics.” Bioinformatics 25 (11): 1422–23."/>
  <meta name="citation_reference" content="Costa, Sofia, André Almeida, António Castro, and Lucília Domingues. 2014. “Fusion Tags for Protein Solubility, Purification and Immunogenicity in Escherichia Coli: The Novel Fh8 System.” Frontiers in Microbiology 5 (February): 63."/>
  <meta name="citation_reference" content="Craveur, Pierrick, Agnel P. Joseph, Jeremy Esque, Tarun J. Narwani, Floriane Noël, Nicolas Shinada, Matthieu Goguet, et al. 2015. “Protein Flexibility in the Light of Structural Alphabets.” Frontiers in Molecular Biosciences 2 (May): 20."/>
  <meta name="citation_reference" content="Davis, G. D., C. Elisee, D. M. Newham, and R. G. Harrison. 1999. “New Fusion Protein Systems Designed to Give Soluble Expression in Escherichia Coli.” Biotechnology and Bioengineering 65 (4): 382–88."/>
  <meta name="citation_reference" content="Diaz, Armando A., Emanuele Tomba, Reese Lennarson, Rex Richard, Miguel J. Bagajewicz, and Roger G. Harrison. 2010. “Prediction of Protein Solubility in Escherichia Coli Using Logistic Regression.” Biotechnology and Bioengineering 105 (2): 374–83."/>
  <meta name="citation_reference" content="Edgar, Robert C. 2010. “Search and Clustering Orders of Magnitude Faster than BLAST.” Bioinformatics 26 (19): 2460–61."/>
  <meta name="citation_reference" content="Esposito, Dominic, and Deb K. Chatterjee. 2006. “Enhancement of Soluble Protein Expression through the Use of Fusion Tags.” Current Opinion in Biotechnology 17 (4): 353–58."/>
  <meta name="citation_reference" content="Família, Carlos, Sarah R. Dennison, Alexandre Quintas, and David A. Phoenix. 2015. “Prediction of Peptide and Protein Propensity for Amyloid Formation.” PloS One 10 (8): e0134679."/>
  <meta name="citation_reference" content="Habibi, Narjeskhatoon, Siti Z. Mohd Hashim, Alireza Norouzi, and Mohammed Razip Samian. 2014. “A Review of Machine Learning Methods to Predict the Solubility of Overexpressed Recombinant Proteins in Escherichia Coli.” BMC Bioinformatics 15 (May): 134."/>
  <meta name="citation_reference" content="Han, Xi, Wenbo Ning, Xiaoqiang Ma, Xiaonan Wang, and Kang Zhou. 2019. “Improve Protein Solubility and Activity Based on Machine Learning Models.” bioRxiv. https://doi.org/10.1101/817890."/>
  <meta name="citation_reference" content="Harrison, R. G. 2000. “Expression of Soluble Heterologous Proteins via Fusion with NusA Protein.” Innovations 11: 4–7."/>
  <meta name="citation_reference" content="Hebditch, Max, M. Alejandro Carballo-Amador, Spyros Charonis, Robin Curtis, and Jim Warwicker. 2017. “Protein-Sol: A Web Tool for Predicting Protein Solubility from Sequence.” Bioinformatics 33 (19): 3098–3100."/>
  <meta name="citation_reference" content="Heckmann, David, Colton J. Lloyd, Nathan Mih, Yuanchi Ha, Daniel C. Zielinski, Zachary B. Haiman, Abdelmoneim Amer Desouki, Martin J. Lercher, and Bernhard O. Palsson. 2018. “Machine Learning Applied to Enzyme Turnover Numbers Reveals Protein Structural Correlates and Improves Metabolic Models.” Nature Communications 9 (1): 5252."/>
  <meta name="citation_reference" content="Hirose, Shuichi, and Tamotsu Noguchi. 2013. “ESPRESSO: A System for Estimating Protein Expression and Solubility in Protein Expression Systems.” Proteomics 13 (9): 1444–56."/>
  <meta name="citation_reference" content="Hou, Qingzhen, Raphaël Bourgeas, Fabrizio Pucci, and Marianne Rooman. 2018. “Computational Analysis of the Amino Acid Interactions That Promote or Decrease Protein Solubility.” Scientific Reports, https://doi.org/10.1038/s41598-018-32988-w."/>
  <meta name="citation_reference" content="Hou, Qingzhen, Jean-Marc Kwasigroch, Marianne Rooman, and Fabrizio Pucci. 2019. “SOLart: A Structure-Based Method to Predict Protein Solubility and Aggregation.” Bioinformatics, October. https://doi.org/10.1093/bioinformatics/btz773."/>
  <meta name="citation_reference" content="Huang, Hui-Ling, Phasit Charoenkwan, Te-Fen Kao, Hua-Chin Lee, Fang-Lin Chang, Wen-Lin Huang, Shinn-Jang Ho, Li-Sun Shu, Wen-Liang Chen, and Shinn-Ying Ho. 2012. “Prediction and Analysis of Protein Solubility Using a Novel Scoring Card Method with Dipeptide Composition.” BMC Bioinformatics 13 Suppl 17 (December): S3."/>
  <meta name="citation_reference" content="Idicula-Thomas, Susan, and Petety V. Balaji. 2005. “Understanding the Relationship between the Primary Structure of Proteins and Its Propensity to Be Soluble on Overexpression in Escherichia Coli.” Protein Science: A Publication of the Protein Society 14 (3): 582–92."/>
  <meta name="citation_reference" content="Jia, Baolei, and Che Ok Jeon. 2016. “High-Throughput Recombinant Protein Expression in Escherichia Coli: Current Status and Future Perspectives.” Open Biology 6 (8). https://doi.org/10.1098/rsob.160196."/>
  <meta name="citation_reference" content="Karplus, P. A., and G. E. Schulz. 1985. “Prediction of Chain Flexibility in Proteins.” Die Naturwissenschaften 72 (4): 212–13."/>
  <meta name="citation_reference" content="Khurana, Sameer, Reda Rawi, Khalid Kunji, Gwo-Yu Chuang, Halima Bensmail, and Raghvendra Mall. 2018. “DeepSol: A Deep Learning Framework for Sequence-Based Protein Solubility Prediction.” Bioinformatics 34 (15): 2605–13."/>
  <meta name="citation_reference" content="Kramer, Ryan M., Varad R. Shende, Nicole Motl, C. Nick Pace, and J. Martin Scholtz. 2012. “Toward a Molecular Understanding of Protein Solubility: Increased Negative Surface Charge Correlates with Increased Solubility.” Biophysical Journal. https://doi.org/10.1016/j.bpj.2012.01.060."/>
  <meta name="citation_reference" content="Kuriata, Aleksander, Valentin Iglesias, Jordi Pujols, Mateusz Kurcinski, Sebastian Kmiecik, and Salvador Ventura. 2019. “Aggrescan3D (A3D) 2.0: Prediction and Engineering of Protein Solubility.” Nucleic Acids Research 47 (W1): W300–307."/>
  <meta name="citation_reference" content="Kyte, J., and R. F. Doolittle. 1982. “A Simple Method for Displaying the Hydropathic Character of a Protein.” Journal of Molecular Biology 157 (1): 105–32."/>
  <meta name="citation_reference" content="Lebendiker, Mario, and Tsafi Danieli. 2014. “Production of Prone-to-Aggregate Proteins.” FEBS Letters 588 (2): 236–46."/>
  <meta name="citation_reference" content="Levy, E. D., S. De, and S. A. Teichmann. 2012. “Cellular Crowding Imposes Global Constraints on the Chemistry and Evolution of Proteomes.” Proceedings of the National Academy of Sciences. https://doi.org/10.1073/pnas.1209312109."/>
  <meta name="citation_reference" content="Ma, Jianpeng. 2005. “Usefulness and Limitations of Normal Mode Analysis in Modeling Dynamics of Biomolecular Complexes.” Structure 13 (3): 373–80."/>
  <meta name="citation_reference" content="Marra, Marco A., Steven J. M. Jones, Caroline R. Astell, Robert A. Holt, Angela Brooks-Wilson, Yaron S. N. Butterfield, Jaswinder Khattra, et al. 2003. “The Genome Sequence of the SARS-Associated Coronavirus.” Science 300 (5624): 1399–1404."/>
  <meta name="citation_reference" content="McKinney, Wes. 2010. “Data Structures for Statistical Computing in Python.” In Proceedings of the 9th Python in Science Conference, 51–56."/>
  <meta name="citation_reference" content="Millman, K. J., and M. Aivazis. 2011. “Python for Scientists and Engineers.” Computing in Science Engineering 13 (2): 9–12."/>
  <meta name="citation_reference" content="Natan, Eviatar, Tamaki Endoh, Liora Haim-Vilmovsky, Tilman Flock, Guilhem Chalancon, Jonathan T. S. Hopper, Bálint Kintses, et al. 2018. “Cotranslational Protein Assembly Imposes Evolutionary Constraints on Homomeric Proteins.” Nature Structural &amp;; Molecular Biology 25 (3): 279–88."/>
  <meta name="citation_reference" content="Nelder, J. A., and R. Mead. 1965. “A Simplex Method for Function Minimization.” Computer Journal 7 (4): 308–13."/>
  <meta name="citation_reference" content="Niwa, Tatsuya, Bei-Wen Ying, Katsuyo Saito, Wenzhen Jin, Shoji Takada, Takuya Ueda, and Hideki Taguchi. 2009. “Bimodal Protein Solubility Distribution Revealed by an Aggregation Analysis of the Entire Ensemble of Escherichia Coli Proteins.” Proceedings of the National Academy of Sciences of the United States of America 106 (11): 4201–6."/>
  <meta name="citation_reference" content="Oliphant, T. E. 2007. “Python for Scientific Computing.” Computing in Science Engineering 9 (3): 10–20."/>
  <meta name="citation_reference" content="Pedregosa, Fabian, Gaël Varoquaux, Alexandre Gramfort, Vincent Michel, Bertrand Thirion, Olivier Grisel, Mathieu Blondel, et al. 2011. “Scikit-Learn: Machine Learning in Python.” Journal of Machine Learning Research: JMLR 12 (Oct): 2825–30."/>
  <meta name="citation_reference" content="Potter, Simon C., Aurélien Luciani, Sean R. Eddy, Youngmi Park, Rodrigo Lopez, and Robert D. Finn. 2018. “HMMER Web Server: 2018 Update.” Nucleic Acids Research 46 (W1): W200–204."/>
  <meta name="citation_reference" content="Radivojac, P. 2004. “Protein Flexibility and Intrinsic Disorder.” Protein Science. https://doi.org/10.1110/ps.03128904."/>
  <meta name="citation_reference" content="Ragone, R., F. Facchiano, A. Facchiano, A. M. Facchiano, and G. Colonna. 1989. “Flexibility Plot of Proteins.” “Protein Engineering, Design and Selection.” https://doi.org/10.1093/protein/2.7.497."/>
  <meta name="citation_reference" content="Rawi, Reda, Raghvendra Mall, Khalid Kunji, Chen-Hsiang Shen, Peter D. Kwong, and Gwo-Yu Chuang. 2018. “PaRSnIP: Sequence-Based Protein Solubility Prediction Using Gradient Boosting Machine.” Bioinformatics. https://doi.org/10.1093/bioinformatics/btx662."/>
  <meta name="citation_reference" content="Rosano, Germán L., and Eduardo A. Ceccarelli. 2014. “Recombinant Protein Expression in Escherichia Coli: Advances and Challenges.” Frontiers in Microbiology 5 (April): 172."/>
  <meta name="citation_reference" content="Schlessinger, Avner, and Burkhard Rost. 2005. “Protein Flexibility and Rigidity Predicted from Sequence.” Proteins 61 (1): 115–26."/>
  <meta name="citation_reference" content="Seabold, Skipper, and Josef Perktold. 2010. “Statsmodels: Econometric and Statistical Modeling with Python.” In Proceedings of the 9th Python in Science Conference. http://conference.scipy.org/proceedings/scipy2010/pdfs/seabold.pdf."/>
  <meta name="citation_reference" content="Seiler, Catherine Y., Jin G. Park, Amit Sharma, Preston Hunter, Padmini Surapaneni, Casey Sedillo, James Field, et al. 2014. “DNASU Plasmid and PSI:Biology-Materials Repositories: Resources to Accelerate Biological Research.” Nucleic Acids Research 42 (Database issue): D1253–60."/>
  <meta name="citation_reference" content="Smith, David K., Predrag Radivojac, Zoran Obradovic, A. Keith Dunker, and Guang Zhu. 2003. “Improved Amino Acid Flexibility Parameters.” Protein Science: A Publication of the Protein Society 12 (5): 1060–72."/>
  <meta name="citation_reference" content="Sormanni, Pietro, Leanne Amery, Sofia Ekizoglou, Michele Vendruscolo, and Bojana Popovic. 2017. “Rapid and Accurate in Silico Solubility Screening of a Monoclonal Antibody Library.” Scientific Reports 7 (1): 8200."/>
  <meta name="citation_reference" content="Sormanni, Pietro, Francesco A. Aprile, and Michele Vendruscolo. 2015. “The CamSol Method of Rational Design of Protein Mutants with Enhanced Solubility.” Journal of Molecular Biology, https://doi.org/10.1016/j.jmb.2014.09.026."/>
  <meta name="citation_reference" content="Stewart, E. J., F. Aslund, and J. Beckwith. 1998. “Disulfide Bond Formation in the Escherichia Coli Cytoplasm: An in Vivo Role Reversal for the Thioredoxins.” The EMBO Journal 17 (19): 5543–50."/>
  <meta name="citation_reference" content="Tartaglia, Gian Gaetano, Andrea Cavalli, Riccardo Pellarin, and Amedeo Caflisch. 2004. “The Role of Aromaticity, Exposed Surface, and Dipole Moment in Determining Protein Aggregation Rates.” Protein Science: A Publication of the Protein Society 13 (7): 1939."/>
  <meta name="citation_reference" content="Teague, Simon J. 2003. “Implications of Protein Flexibility for Drug Discovery.” Nature Reviews. Drug Discovery 2 (7): 527–41."/>
  <meta name="citation_reference" content="Trevino, Saul R., J. Martin Scholtz, and C. Nick Pace. 2007. “Amino Acid Contribution to Protein Solubility: Asp, Glu, and Ser Contribute More Favorably than the Other Hydrophilic Amino Acids in RNase Sa.” Journal of Molecular Biology, https://doi.org/10.1016/j.jmb.2006.10.026."/>
  <meta name="citation_reference" content="Tsumoto, Kouhei, Daisuke Ejima, Izumi Kumagai, and Tsutomu Arakawa. 2003. “Practical Considerations in Refolding Proteins from Inclusion Bodies.” Protein Expression and Purification 28 (1): 1–8."/>
  <meta name="citation_reference" content="Vihinen, Mauno. 1987. “Relationship of Protein Flexibility to Thermostability.” “Protein Engineering, Design and Selection.” https://doi.org/10.1093/protein/1.6.477."/>
  <meta name="citation_reference" content="Vihinen, M., E. Torkkila, and P. Riikonen. 1994. “Accuracy of Protein Flexibility Predictions.” Proteins 19 (2): 141–49."/>
  <meta name="citation_reference" content="Waldo, Geoffrey S. 2003. “Genetic Screens and Directed Evolution for Protein Solubility.” Current Opinion in Chemical Biology 7 (1): 33–38."/>
  <meta name="citation_reference" content="Walt, Stéfan van der, S. Chris Colbert, and Gaël Varoquaux. 2011. “The NumPy Array: A Structure for Efficient Numerical Computation.” Computing in Science &amp;; Engineering 13 (2): 22–30."/>
  <meta name="citation_reference" content="Wang, Xiaoling, Tapan K. Das, Satish K. Singh, and Sandeep Kumar. 2009. “Potential Aggregation Prone Regions in Biotherapeutics: A Survey of Commercial Monoclonal Antibodies.” mAbs 1 (3): 254–67."/>
  <meta name="citation_reference" content="Warwicker, Jim, Spyros Charonis, and Robin A. Curtis. 2014. “Lysine and Arginine Content of Proteins: Computational Analysis Suggests a New Tool for Solubility Design.” Molecular Pharmaceutics 11 (1): 294–303."/>
  <meta name="citation_reference" content="Waskom, Michael, Olga Botvinnik, Paul Hobson, John B. Cole, Yaroslav Halchenko, Stephan Hoyer, Alistair Miles, et al. 2014. “Seaborn: v0.5.0 (November 2014),” November. https://doi.org/10.5281/zenodo.12710."/>
  <meta name="citation_reference" content="Wilkinson, D. L., and R. G. Harrison. 1991. “Predicting the Solubility of Recombinant Proteins in Escherichia Coli.” Bio/technology 9 (5): 443–48."/>
  <meta name="citation_reference" content="Wu, Fan, Su Zhao, Bin Yu, Yan-Mei Chen, Wen Wang, Yi Hu, Zhi-Gang Song, et al. 2020. “Complete Genome Characterisation of a Novel Coronavirus Associated with Severe Human Respiratory Disease in Wuhan, China.” bioRxiv, https://doi.org/10.1101/2020.01.24.919183."/>
  <meta name="citation_reference" content="Wu, Zachary, S. B. Jennifer Kan, Russell D. Lewis, Bruce J. Wittmann, and Frances H. Arnold. 2019. “Machine Learning-Assisted Directed Protein Evolution with Combinatorial Libraries.” Proceedings of the National Academy of Sciences of the United States of America 116 (18): 8852–58."/>
  <meta name="citation_reference" content="Xiao, Nan, Dong-Sheng Cao, Min-Feng Zhu, and Qing-Song Xu. 2015. “protr/ProtrWeb: R Package and Web Server for Generating Various Numerical Representation Schemes of Protein Sequences.” Bioinformatics 31 (11): 1857–59."/>
  <meta name="citation_reference" content="Xiao, Rong, Stephen Anderson, James Aramini, Rachel Belote, William A. Buchwald, Colleen Ciccosanti, Ken Conover, et al. 2010. “The High-Throughput Protein Sample Production Platform of the Northeast Structural Genomics Consortium.” Journal of Structural Biology 172 (1): 21–33."/>
  <meta name="citation_reference" content="Yang, Kevin K., Zachary Wu, and Frances H. Arnold. 2019. “Machine-Learning-Guided Directed Evolution for Protein Engineering.” Nature Methods 16 (8): 687–94."/>
  <meta name="citation_reference" content="Yin, Hui, Yi-Zhou Li, and Meng-Long Li. 2011. “On the Relation between Residue Flexibility and Residue Interactions in Proteins.” Protein and Peptide Letters 18 (5): 450–56."/>
  <meta name="citation_reference" content="Yuan, Zheng, Timothy L. Bailey, and Rohan D. Teasdale. 2005. “Prediction of Protein B-Factor Profiles.” Proteins: Structure, Function, and Bioinformatics. https://doi.org/10.1002/prot.20375."/>
  <meta name="twitter:title" content="Solubility-Weighted Index: fast and accurate prediction of protein solubility"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Motivation Recombinant protein production is a widely used technique in the biotechnology and biomedical industries, yet only a quarter of target proteins are soluble and can therefore be purified. Results We have discovered that global structural flexibility, which can be modeled by normalised B-factors, accurately predicts the solubility of 12,216 recombinant proteins expressed in Escherichia coli . We have optimised B-factors, and derived a new set of values for solubility scoring that further improves prediction accuracy. We call this new predictor the ‘Solubility-Weighted Index’ (SWI). Importantly, SWI outperforms many existing protein solubility prediction tools. Furthermore, we have developed ‘SoDoPE’ (Soluble Domain for Protein Expression), a web interface that allows users to choose a protein region of interest for predicting and maximising both protein expression and solubility. Availability The SoDoPE web server and source code are freely available at &lt;;https://tisigner.com/sodope&gt;; and &lt;;https://github.com/Gardner-BinfLab/TISIGNER-ReactJS&gt;;, respectively. The code and data for reproducing our analysis can be found at &lt;;https://github.com/Gardner-BinfLab/SoDoPE_paper2020&gt;;."/>
  <meta name="og-title" property="og:title" content="Solubility-Weighted Index: fast and accurate prediction of protein solubility"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.15.951012v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Motivation Recombinant protein production is a widely used technique in the biotechnology and biomedical industries, yet only a quarter of target proteins are soluble and can therefore be purified. Results We have discovered that global structural flexibility, which can be modeled by normalised B-factors, accurately predicts the solubility of 12,216 recombinant proteins expressed in Escherichia coli . We have optimised B-factors, and derived a new set of values for solubility scoring that further improves prediction accuracy. We call this new predictor the ‘Solubility-Weighted Index’ (SWI). Importantly, SWI outperforms many existing protein solubility prediction tools. Furthermore, we have developed ‘SoDoPE’ (Soluble Domain for Protein Expression), a web interface that allows users to choose a protein region of interest for predicting and maximising both protein expression and solubility. Availability The SoDoPE web server and source code are freely available at &lt;;https://tisigner.com/sodope&gt;; and &lt;;https://github.com/Gardner-BinfLab/TISIGNER-ReactJS&gt;;, respectively. The code and data for reproducing our analysis can be found at &lt;;https://github.com/Gardner-BinfLab/SoDoPE_paper2020&gt;;."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-26"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Solubility-Weighted Index: fast and accurate prediction of protein solubility | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1212171 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <div id="sec-1" class="subsection">
     <p id="p-2">
      <strong>Motivation</strong> Recombinant protein production is a widely used technique in the biotechnology and biomedical industries, yet only a quarter of target proteins are soluble and can therefore be purified.
     </p>
    </div>
    <div id="sec-2" class="subsection">
     <p id="p-3">
      <strong>Results</strong> We have discovered that global structural flexibility, which can be modeled by normalised B-factors, accurately predicts the solubility of 12,216 recombinant proteins expressed in 
      <em>Escherichia coli</em>. We have optimised B-factors, and derived a new set of values for solubility scoring that further improves prediction accuracy. We call this new predictor the ‘Solubility-Weighted Index’ (SWI). Importantly, SWI outperforms many existing protein solubility prediction tools. Furthermore, we have developed ‘SoDoPE’ (Soluble Domain for Protein Expression), a web interface that allows users to choose a protein region of interest for predicting and maximising both protein expression and solubility.
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <p id="p-4">
      <strong>Availability</strong> The SoDoPE web server and source code are freely available at 
      <a href="https://tisigner.com/sodope">https://tisigner.com/sodope</a> and 
      <a href="https://github.com/Gardner-BinfLab/TISIGNER-ReactJS">https://github.com/Gardner-BinfLab/TISIGNER-ReactJS</a>, respectively. The code and data for reproducing our analysis can be found at 
      <a href="https://github.com/Gardner-BinfLab/SoDoPE_paper2020">https://github.com/Gardner-BinfLab/SoDoPE_paper2020</a>.
     </p>
    </div>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-7">High levels of protein expression and solubility are two major requirements of successful recombinant protein production (
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Esposito and Chatterjee 2006</a>). However, recombinant protein production is a challenging process. Almost half of recombinant proteins fail to be expressed and half of the successfully expressed proteins are insoluble (
     <a href="http://targetdb.rcsb.org/metrics/">http://targetdb.rcsb.org/metrics/</a>). These failures hamper protein research, with particular implications for structural, functional and pharmaceutical studies that require soluble and concentrated protein solutions (
     <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kramer et al. 2012</a>; 
     <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hou et al. 2018</a>). Therefore, solubility prediction and protein engineering for enhanced solubility is an active area of research. Notable protein engineering approaches include mutagenesis, truncation (i.e., expression of partial protein sequences), or fusion with a solubility-enhancing tag (
     <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Waldo 2003</a>; 
     <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Esposito and Chatterjee 2006</a>; 
     <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Trevino, Martin Scholtz, and Nick Pace 2007</a>; 
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chan et al. 2010</a>; 
     <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kramer et al. 2012</a>; 
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Costa et al. 2014</a>).
    </p>
    <p id="p-8">Protein solubility, at least in part, depends upon extrinsic factors such as ionic strength, temperature and pH, as well as intrinsic factors—the physicochemical properties of the protein sequence and structure, including molecular weight, amino acid composition, hydrophobicity, aromaticity, isoelectric point, structural propensities and the polarity of surface residues (
     <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wilkinson and Harrison 1991</a>; 
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chiti et al. 2003</a>; 
     <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Tartaglia et al. 2004</a>; 
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Diaz et al. 2010</a>). Many solubility prediction tools have been developed around these features using statistical models (e.g., linear and logistic regression) or other machine learning models (e.g., support vector machines and neural networks) (
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hirose and Noguchi 2013</a>; 
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Habibi et al. 2014</a>; 
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hebditch et al. 2017</a>; 
     <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sormanni et al. 2017</a>; 
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Heckmann et al. 2018</a>; Z. 
     <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wu et al. 2019</a>; 
     <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yang, Wu, and Arnold 2019</a>).
    </p>
    <p id="p-9">In this study, we investigated the experimental outcomes of 12,216 recombinant proteins expressed in 
     <em>Escherichia coli</em> from the ‘Protein Structure Initiative:Biology’ (PSI:Biology) (
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen et al. 2004</a>; 
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Acton et al. 2005</a>). We showed that protein structural flexibility is more accurate than other protein sequence properties in predicting solubility (
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Craveur et al. 2015</a>; M. 
     <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vihinen, Torkkila, and Riikonen 1994</a>). Flexibility is a standard feature that appears to have been overlooked in previous solubility prediction attempts. On this basis, we derived a set of 20 values for the standard amino acid residues and used them to predict solubility. We call this new predictor the ‘Solubility-Weighted Index’ (SWI). SWI is a powerful predictor of solubility, and a good proxy for global structural flexibility. In addition, SWI outperforms many existing 
     <em>de novo</em> protein solubility prediction tools.
    </p>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">RESULTS</h2>
    <div id="sec-6" class="subsection">
     <h3>Global structural flexibility performs well at predicting protein solubility</h3>
     <p id="p-10">We sought to understand what makes a protein soluble, and develop a fast and accurate approach for solubility prediction. To determine which protein sequence properties accurately predict protein solubility, we analysed 12,216 target proteins from over 196 species that were expressed in 
      <em>E. coli</em> (the PSI:Biology dataset; see Supplementary Fig S1 and Table S1A) (
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chen et al. 2004</a>; 
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Acton et al. 2005</a>). These proteins were expressed either with a C-terminal or N-terminal polyhistidine fusion tag (pET21_NESG and pET15_NESG expression vectors, N=8,780 and 3,436, respectively). They were previously curated and labeled as ‘Protein_Soluble’ or ‘Tested_Not_Soluble’ (
      <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Seiler et al. 2014</a>), based on the soluble analysis of cell lysate using SDS-PAGE (R. 
      <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Xiao et al. 2010</a>). A total of 8,238 recombinant proteins were found to be soluble, in which 6,432 of them belong to the pET21_NESG dataset. Both the expression system and solubility analysis method are commonly used (
      <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Costa et al. 2014</a>). Therefore, this collection of data captures a broad range of protein solubility issues.
     </p>
     <p id="p-11">We evaluated nine standard and 9,920 miscellaneous protein sequence properties using the Biopython’s ProtParam module and ‘protr’ R package, respectively (
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cock et al. 2009</a>; N. 
      <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Xiao et al. 2015</a>). For example, the standard properties include the Grand Average of Hydropathy (GRAVY), secondary structure propensities, protein structural flexibility etc., whereas miscellaneous properties include amino acid composition, autocorrelation, etc. Strikingly, protein structural flexibility outperformed other features in solubility prediction [Area Under the ROC Curve (AUC) = 0.67; 
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1</a>, Supplementary Fig S2 and Table S2].
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Global structural flexibility outperforms the other standard protein sequence properties in protein solubility prediction. ROC analysis of the standard protein sequence features for predicting the solubility of 12,216 recombinant proteins expressed in E. coli (the PSI:Biology dataset). AUC scores (perfect = 1.00, random = 0.50) are shown in parentheses. The ROC curves are shown in two separate panels for clarity. Dashed lines denote the performance of random classifiers. See also Supplementary Fig S2 and Table S2. AUC, Area Under the ROC Curve; GRAVY, Grand Average of Hydropathy; PSI:Biology, Protein Structure Initiative:Biology; ROC, Receiver Operating Characteristic." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-40772057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Global structural flexibility outperforms the other standard protein sequence properties in protein solubility prediction.&lt;;/span&gt;; ROC analysis of the standard protein sequence features for predicting the solubility of 12,216 recombinant proteins expressed in &lt;;em&gt;;E. coli&lt;;/em&gt;; (the PSI:Biology dataset). AUC scores (perfect = 1.00, random = 0.50) are shown in parentheses. The ROC curves are shown in two separate panels for clarity. Dashed lines denote the performance of random classifiers. See also Supplementary Fig S2 and Table S2. AUC, Area Under the ROC Curve; GRAVY, Grand Average of Hydropathy; PSI:Biology, Protein Structure Initiative:Biology; ROC, Receiver Operating Characteristic.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F1.medium.gif" width="440" height="193"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F1.medium.gif" width="440" height="193"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 1.</span>
       <span class="caption-title">Global structural flexibility outperforms the other standard protein sequence properties in protein solubility prediction.</span>
       <p id="p-12" class="first-child">ROC analysis of the standard protein sequence features for predicting the solubility of 12,216 recombinant proteins expressed in 
        <em>E. coli</em> (the PSI:Biology dataset). AUC scores (perfect = 1.00, random = 0.50) are shown in parentheses. The ROC curves are shown in two separate panels for clarity. Dashed lines denote the performance of random classifiers. See also Supplementary Fig S2 and Table S2. AUC, Area Under the ROC Curve; GRAVY, Grand Average of Hydropathy; PSI:Biology, Protein Structure Initiative:Biology; ROC, Receiver Operating Characteristic.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-7" class="subsection">
     <h3>The Solubility-Weighted Index (SWI) is an improved predictor of solubility</h3>
     <p id="p-13">Protein structural flexibility, in particular, the flexibility of local regions, is often associated with function (
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Craveur et al. 2015</a>). The calculation of flexibility is usually performed by assigning a set of 20 normalised B-factors—a measure of vibration of C-alpha atoms (see Supplementary Notes)—to a protein sequence and averaging the values by a sliding window approach (
      <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Ragone et al. 1989</a>; 
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Karplus and Schulz 1985</a>; M. 
      <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Vihinen, Torkkila, and Riikonen 1994</a>; 
      <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Smith et al. 2003</a>). We reasoned that such sliding window approach can be approximated by a more straightforward arithmetic mean for calculating global structural flexibility (see Supplementary Notes). We determined the correlation between flexibility (Vihinen 
      <em>et al.’</em> s sliding window approach as implemented in Biopython) and solubility scores calculated as follows: 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/26/2020.02.15.951012/embed/graphic-2.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/26/2020.02.15.951012/embed/graphic-2.gif"/>
        </noscript>
       </span>
      </span> where 
      <em>B
       <sub>i</sub>
      </em> is the normalised B-factor of the amino acid residue at the position 
      <em>i</em>, and 
      <em>L</em> is the sequence length. We obtained a strong correlation for the PSI:Biology dataset (Spearman’s rho = 0.98, P-value below machine’s underflow level). Therefore, we reasoned that the sliding window approach is not necessary for our purpose.
     </p>
     <p id="p-14">We applied this arithmetic mean approach (i.e., sequence composition scoring) to the PSI:Biology dataset and compared four sets of previously published, normalised B-factors (
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bhaskaran and Ponnuswamy 1988</a>; 
      <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Ragone et al. 1989</a>; M. 
      <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Vihinen, Torkkila, and Riikonen 1994</a>; 
      <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Smith et al. 2003</a>) Among these sets of B-factors, sequence composition scoring using the most recently published set of normalised B-factors produced the highest AUC score (Supplementary Fig S3, AUC = 0.66).
     </p>
     <p id="p-15">To improve the prediction accuracy of solubility, we iteratively refined the weights of amino acid residues using the Nelder-Mead optimisation algorithm (
      <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nelder and Mead 1965</a>). To avoid testing and training on similar sequences, we generated 10 cross-validation sets with a maximised heterogeneity between these subsets (i.e. no similar sequences between subsets). We first clustered all 12,216 PSI:Biology protein sequences using a 40% similarity threshold using USEARCH to produce 5,050 clusters with remote similarity (see Methods and Supplementary Fig S4). The clusters were grouped into 10 cross-validation sets of approximately 1,200 sequences each manually. We did not select a representative sequence for each cluster as about 12% of clusters contain a mix of soluble and insoluble proteins (Supplementary Fig S4C). More importantly, to address the issues of sequence similarity and imbalanced classes, we performed 1,000 bootstrap resamplings for each cross-validation step (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A</a> and Supplementary Fig S5). We calculated the solubility scores using the optimised weights as 
      <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> and the AUC scores for each cross-validation step. Our training and test AUC scores were 0.72 ± 0.00 and 0.71 ± 0.01, respectively, showing an improvement over flexibility in solubility prediction (mean ± standard deviation; 
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2B</a> and Supplementary Table S3).
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Derivation of the Solubility-Weighted Index (SWI). (A) Flow chart shows an iterative refinement of the most recently published set of normalised B-factors for solubility prediction (Smith et al. 2003). The solubility score of a protein sequence was calculated using a sequence composition scoring approach (Equation 1, using optimised weights W, instead of normalised B-factors B). These scores were used to compute the AUC scores for training and test datasets. (B) Training and test performance of solubility prediction using optimised weights for 20 amino acid residues in a 10-fold cross-validation (mean AUC [[#xB1]] standard deviation). Related data and figures are available as Supplementary Table S3 and Supplementary Fig S4 and S5. (C) Comparison between the 20 initial and final weights for amino acid residues. The final weights are derived from the arithmetic mean of the optimised weights from cross-validation. These weights are used to calculate SWI, the solubility score of a protein sequence, in the subsequent analyses. Filled circles, which represent amino acid residues, are colored by hydrophobicity (Kyte and Doolittle 1982). Solid black circles denote aromatic amino acid residues phenylalanine (F), tyrosine (Y), tryptophan (W). Dotted diagonal line represents no change in weight. See also Supplementary Table S4 and Fig S4. AUC, Area Under the ROC Curve; ROC, Receiver Operating Characteristic; W, arithmetic mean of the weights of an amino acid residue optimised from 1,000 bootstrap samples in a cross-validation step." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-40772057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Derivation of the Solubility-Weighted Index (SWI).&lt;;/span&gt;; &lt;;strong&gt;;(A)&lt;;/strong&gt;; Flow chart shows an iterative refinement of the most recently published set of normalised B-factors for solubility prediction (Smith et al. 2003). The solubility score of a protein sequence was calculated using a sequence composition scoring approach (Equation 1, using optimised weights &lt;;em&gt;;W&lt;;/em&gt;;, instead of normalised B-factors &lt;;em&gt;;B&lt;;/em&gt;;). These scores were used to compute the AUC scores for training and test datasets. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Training and test performance of solubility prediction using optimised weights for 20 amino acid residues in a 10-fold cross-validation (mean AUC [[#xB1]] standard deviation). Related data and figures are available as Supplementary Table S3 and Supplementary Fig S4 and S5. &lt;;strong&gt;;(C)&lt;;/strong&gt;; Comparison between the 20 initial and final weights for amino acid residues. The final weights are derived from the arithmetic mean of the optimised weights from cross-validation. These weights are used to calculate SWI, the solubility score of a protein sequence, in the subsequent analyses. Filled circles, which represent amino acid residues, are colored by hydrophobicity (Kyte and Doolittle 1982). Solid black circles denote aromatic amino acid residues phenylalanine (F), tyrosine (Y), tryptophan (W). Dotted diagonal line represents no change in weight. See also Supplementary Table S4 and Fig S4. AUC, Area Under the ROC Curve; ROC, Receiver Operating Characteristic; &lt;;em&gt;;W&lt;;/em&gt;;, arithmetic mean of the weights of an amino acid residue optimised from 1,000 bootstrap samples in a cross-validation step.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F2.medium.gif" width="440" height="362"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F2.medium.gif" width="440" height="362"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 2.</span>
       <span class="caption-title">Derivation of the Solubility-Weighted Index (SWI).</span>
       <p id="p-16" class="first-child">
        <strong>(A)</strong> Flow chart shows an iterative refinement of the most recently published set of normalised B-factors for solubility prediction (
        <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Smith et al. 2003</a>). The solubility score of a protein sequence was calculated using a sequence composition scoring approach (
        <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>, using optimised weights 
        <em>W</em>, instead of normalised B-factors 
        <em>B</em>). These scores were used to compute the AUC scores for training and test datasets. 
        <strong>(B)</strong> Training and test performance of solubility prediction using optimised weights for 20 amino acid residues in a 10-fold cross-validation (mean AUC ± standard deviation). Related data and figures are available as Supplementary Table S3 and Supplementary Fig S4 and S5. 
        <strong>(C)</strong> Comparison between the 20 initial and final weights for amino acid residues. The final weights are derived from the arithmetic mean of the optimised weights from cross-validation. These weights are used to calculate SWI, the solubility score of a protein sequence, in the subsequent analyses. Filled circles, which represent amino acid residues, are colored by hydrophobicity (
        <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kyte and Doolittle 1982</a>). Solid black circles denote aromatic amino acid residues phenylalanine (F), tyrosine (Y), tryptophan (W). Dotted diagonal line represents no change in weight. See also Supplementary Table S4 and Fig S4. AUC, Area Under the ROC Curve; ROC, Receiver Operating Characteristic; 
        <em>W</em>, arithmetic mean of the weights of an amino acid residue optimised from 1,000 bootstrap samples in a cross-validation step.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-17">The final weights were derived from the arithmetic means of the weights for individual amino acid residues obtained cross-validation (Supplementary Table S4). We observed over a 20% change on the weights for cysteine (C) and histidine (H) residues (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2C</a> and Supplementary Table S4). These results are in agreement with the contributions of cysteine and histidine residues as shown in Supplementary Fig S2B. We call the solubility score of a protein sequence calculated using the final weights the Solubility-Weighted Index (SWI).
     </p>
     <p id="p-18">To validate the cross-validation results, we used a dataset independent of the PSI:Biology data known as eSOL (
      <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Niwa et al. 2009</a>). This dataset consists of the solubility percentages of 
      <em>E. coli</em> proteins determined using an 
      <em>E. coli</em> cell-free system (N = 3,198). Our solubility scoring using the final weights showed a significant improved correlation with 
      <em>E. coli</em> protein solubility over the initial weights (Smith 
      <em>et al.’</em> s normalised B-factors) [Spearman’s rho of 0.50 (P = 9.46 × 10
      <sup>-206</sup>) versus 0.40 (P = 4.57 × 10
      <sup>-120</sup>)]. We repeated the correlation analysis by removing extra amino acid residues including His-tags from the eSOL sequences (MRGSHHHHHHTDPALRA and GLCGR at the N- and C-termini, respectively). This artificial dataset was created based on the assumption that His-tags have little effect on solubility. We observed a slight decrease in correlation for this artificial dataset (Spearman’s rho = 0.47, P= 3.67 × 10-176), which may be due to the effects of His-tag in solubility and/or the limitation(s) of our approach that may overfit to His-tag fusion proteins.
     </p>
     <p id="p-19">We performed Spearman’s correlation analysis for both the PSI:Biology and eSOL datasets. SWI shows the strongest correlation with solubility compared to the standard and 9,920 protein sequence properties (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3</a> and Supplementary Fig S2, respectively). SWI also strongly correlates with flexibility, suggesting that SWI is also a good proxy for global structural flexibility.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SWI strongly correlates with solubility. (A) Correlation matrix plot of the solubility of recombinant proteins expressed in E. coli and their standard protein sequence properties and SWI. These recombinant proteins are the PSI:Biology targets (N = 12,216) with a binary solubility status of [[#x2018]]Protein_Soluble[[#x2019]] or [[#x2018]]Tested_Not_Soluble[[#x2019]]. Related data is available as Supplementary Table S5. (B) Correlation matrix plot of the solubility percentages of E. coli proteins and their standard protein sequence properties and SWI. The solubility percentages were previously determined using an E. coli cell-free system (eSOL, N = 3,198). Related data is available as Supplementary Table S6. GRAVY, Grand Average of Hydropathy;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-40772057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;SWI strongly correlates with solubility.&lt;;/span&gt;; &lt;;strong&gt;;(A)&lt;;/strong&gt;; Correlation matrix plot of the solubility of recombinant proteins expressed in &lt;;em&gt;;E. coli&lt;;/em&gt;; and their standard protein sequence properties and SWI. These recombinant proteins are the PSI:Biology targets (N = 12,216) with a binary solubility status of [[#x2018]]Protein_Soluble[[#x2019]] or [[#x2018]]Tested_Not_Soluble[[#x2019]]. Related data is available as Supplementary Table S5. &lt;;strong&gt;;(B)&lt;;/strong&gt;; Correlation matrix plot of the solubility percentages of &lt;;em&gt;;E. coli&lt;;/em&gt;; proteins and their standard protein sequence properties and SWI. The solubility percentages were previously determined using an &lt;;em&gt;;E. coli&lt;;/em&gt;; cell-free system (eSOL, N = 3,198). Related data is available as Supplementary Table S6. GRAVY, Grand Average of Hydropathy;&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F3.medium.gif" width="440" height="232"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F3.medium.gif" width="440" height="232"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 3.</span>
       <span class="caption-title">SWI strongly correlates with solubility.</span>
       <p id="p-20" class="first-child">
        <strong>(A)</strong> Correlation matrix plot of the solubility of recombinant proteins expressed in 
        <em>E. coli</em> and their standard protein sequence properties and SWI. These recombinant proteins are the PSI:Biology targets (N = 12,216) with a binary solubility status of ‘Protein_Soluble’ or ‘Tested_Not_Soluble’. Related data is available as Supplementary Table S5. 
        <strong>(B)</strong> Correlation matrix plot of the solubility percentages of 
        <em>E. coli</em> proteins and their standard protein sequence properties and SWI. The solubility percentages were previously determined using an 
        <em>E. coli</em> cell-free system (eSOL, N = 3,198). Related data is available as Supplementary Table S6. GRAVY, Grand Average of Hydropathy;
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-21">PSI:Biology, Protein Structure Initiative:Biology; R
      <sub>s</sub>, Spearman’s rho; SWI, Solubility-Weighted Index.
     </p>
     <p id="p-22">We asked whether protein solubility can be predicted by surface amino acid residues. To address this question, we examined a previously published dataset for the protein surface ‘stickiness’ of 397 
      <em>E. coli</em> proteins (
      <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Levy, De, and Teichmann 2012</a>). This dataset has the annotation for surface residues based on previously solved protein crystal structures. We observed little correlation between the protein surface ‘stickiness’ and the solubility data from eSOL (Spearman’s rho = 0.05, P = 0.34, N = 348; Supplementary Fig S6A). Next, we evaluated if amino acid composition scoring using surface residues is sufficient, optimising only the weights of surface residues should achieve similar or better results than SWI. As above, we iteratively refined the weights of surface residues using the Nelder-Mead optimisation algorithm. The method was initialised with Smith 
      <em>et al.’</em> s normalised B-factors and a maximised correlation coefficient was the target. However, a low correlation was obtained upon convergence (Spearman’s rho = 0.18, P = 7.20 × 10
      <sup>-4</sup>; Supplementary Fig S6B). In contrast, the SWI of the full-length sequences has a much stronger correlation with solubility (Spearman’s rho = 0.46, P = 2.97 × 10
      <sup>-19</sup>; Supplementary Fig S6C). These results suggest that the full-length of sequences contributes to protein solubility, not just surface residues, in which solubility is modulated by cotranslational folding (
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Natan et al. 2018</a>).
     </p>
     <p id="p-23">To understand the properties of soluble and insoluble proteins, we determined the enrichment of amino acid residues in the PSI:Biology targets relative to the eSOL sequences (see Methods). We observed that the PSI:Biology targets are enriched in charged residues lysine (K), glutamate (E) and aspartate (D), and depleted in aromatic residues tryptophan (W), albeit to a lesser extend for insoluble proteins (Supplementary Fig S7A). As expected, cysteine residues (C) are enriched in the PSI:Biology insoluble proteins, supporting previous findings that cysteine residues contribute to poor solubility in the 
      <em>E. coli</em> expression system (
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Diaz et al. 2010</a>; 
      <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Wilkinson and Harrison 1991</a>).
     </p>
     <p id="p-24">In addition, we compared the SWI of random sequences with the PSI:Biology and eSOL sequences. We included an analysis of random sequences to confirm whether SWI can distinguish between biological and random sequences. We found that the SWI scores of soluble proteins are higher than those of insoluble proteins (Supplementary Fig S7B), and that true biological sequences also tend to have higher SWI scores than random sequences, highlighting a potential evolutionary selection for solubility.</p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>SWI outperforms many protein solubility prediction tools</h3>
     <p id="p-25">To confirm the usefulness of SWI in solubility prediction, we compared it with the existing tools Protein-Sol (
      <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hebditch et al. 2017</a>), CamSol v2.1 (
      <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sormanni, Aprile, and Vendruscolo 2015</a>; 
      <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Sormanni et al. 2017</a>), PaRSnIP (
      <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rawi et al. 2018</a>), DeepSol v0.3 (
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Khurana et al. 2018</a>), the Wilkinson-Harrison model (
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Davis et al. 1999</a>; 
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Harrison 2000</a>; 
      <a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Wilkinson and Harrison 1991</a>), and ccSOL omics (
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Agostini et al. 2014</a>). We did not include the specialised tools that model protein structural information such as surface geometry, surface charges and solvent accessibility because these tools require prior knowledge of protein tertiary structure. For example, Aggrescan3D and SOLart accept only PDB files that can be downloaded from the Protein Data Bank or produced using a homology modeling program (
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kuriata et al. 2019</a>; 
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hou et al. 2019</a>). SWI outperforms other tools except for Protein-Sol in predicting 
      <em>E. coli</em> protein solubility (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, 
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4A</a>). Our SWI C program is also the fastest solubility prediction algorithm (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, 
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4B</a> and Supplementary Table S7).
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1218798/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/1218798/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-26" class="first-child">Comparison of protein solubility prediction methods and software.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SWI outperforms existing protein solubility prediction tools. (A) Prediction accuracy of solubility prediction tools using the above cross-validation sets (Fig 2A). For SWI, the test AUC scores were calculated from a 10-fold cross-validation (i.e., a boxplot representation of Fig 2B). For other tools, the test AUC scores were calculated directly from the individual subsets used for cross-validation. No cross-validations were done. CamSol and ccSOL omics are only available as web servers (no fill colors). (B) Wall time of protein solubility prediction tools per sequence (log scale). All command line tools were run three times using 10 sequences selected from the PSI:Biology and eSOL datasets. Related data is available as Supplementary Table S7. AUC, Area Under the ROC Curve; PSI:Biology, Protein Structure Initiative:Biology; ROC, Receiver Operating Characteristic; SWI, Solubility-Weighted Index; s, seconds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-40772057" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;SWI outperforms existing protein solubility prediction tools.&lt;;/span&gt;; &lt;;strong&gt;;(A)&lt;;/strong&gt;; Prediction accuracy of solubility prediction tools using the above cross-validation sets (Fig 2A). For SWI, the test AUC scores were calculated from a 10-fold cross-validation (i.e., a boxplot representation of Fig 2B). For other tools, the test AUC scores were calculated directly from the individual subsets used for cross-validation. No cross-validations were done. CamSol and ccSOL omics are only available as web servers (no fill colors). &lt;;strong&gt;;(B)&lt;;/strong&gt;; Wall time of protein solubility prediction tools per sequence (log scale). All command line tools were run three times using 10 sequences selected from the PSI:Biology and eSOL datasets. Related data is available as Supplementary Table S7. AUC, Area Under the ROC Curve; PSI:Biology, Protein Structure Initiative:Biology; ROC, Receiver Operating Characteristic; SWI, Solubility-Weighted Index; s, seconds.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F4.medium.gif" width="440" height="201"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F4.medium.gif" width="440" height="201"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/26/2020.02.15.951012/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 4.</span>
       <span class="caption-title">SWI outperforms existing protein solubility prediction tools.</span>
       <p id="p-27" class="first-child">
        <strong>(A)</strong> Prediction accuracy of solubility prediction tools using the above cross-validation sets (
        <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2A</a>). For SWI, the test AUC scores were calculated from a 10-fold cross-validation (i.e., a boxplot representation of 
        <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig 2B</a>). For other tools, the test AUC scores were calculated directly from the individual subsets used for cross-validation. No cross-validations were done. CamSol and ccSOL omics are only available as web servers (no fill colors). 
        <strong>(B)</strong> Wall time of protein solubility prediction tools per sequence (log scale). All command line tools were run three times using 10 sequences selected from the PSI:Biology and eSOL datasets. Related data is available as Supplementary Table S7. AUC, Area Under the ROC Curve; PSI:Biology, Protein Structure Initiative:Biology; ROC, Receiver Operating Characteristic; SWI, Solubility-Weighted Index; s, seconds.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-28">To demonstrate a use case for SWI, we developed the Soluble Domain for Protein Expression (SoDoPE) web server (see Methods and 
      <a href="https://tisigner.com/sodope">https://tisigner.com/sodope</a>). Upon sequence submission, the SoDoPE web server enables users to navigate the protein sequence and its domains for predicting and maximising protein expression and solubility.
     </p>
    </div>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">DISCUSSION</h2>
    <p id="p-29">Protein structural flexibility has been associated with conformal variations, functions, thermal stability, ligand binding and disordered regions (
     <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Mauno Vihinen 1987</a>; 
     <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Teague 2003</a>; 
     <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ma 2005</a>; 
     <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Radivojac 2004</a>; 
     <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Schlessinger and Rost 2005</a>; 
     <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Yuan, Bailey, and Teasdale 2005</a>; 
     <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Yin, Li, and Li 2011</a>). However, the use of flexibility in solubility prediction has been overlooked although their relationship has previously been noted (
     <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tsumoto et al. 2003</a>). In this study, we have shown that flexibility strongly correlates with solubility (
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3</a>). Based on the normalised B-factors used to compute flexibility, we have derived a new position and length independent weights to score the solubility of a given protein sequence (i.e., sequence composition based score). We call this protein solubility score as SWI.
    </p>
    <p id="p-30">Upon further inspection, we observe some interesting properties in SWI. SWI anti-correlates with helix propensity, GRAVY, aromaticity and isoelectric point (
     <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig 2C</a> and 
     <a id="xref-fig-3-3" class="xref-fig" href="#F3">3</a>), suggesting that SWI incorporates the key propensities affecting solubility. Amino acid residues with a lower aromaticity or hydrophilic are known to improve protein solubility (
     <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Trevino, Martin Scholtz, and Nick Pace 2007</a>; 
     <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Niwa et al. 2009</a>; 
     <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Kramer et al. 2012</a>; 
     <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Warwicker, Charonis, and Curtis 2014</a>; 
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Han et al. 2019</a>; 
     <a id="xref-ref-68-4" class="xref-bibr" href="#ref-68">Wilkinson and Harrison 1991</a>). Consistent with previous studies, the charged residues aspartate (D), glutamate (E) and lysine (K) are associated with high solubility, whereas the aromatic residues phenylalanine (F), tryptophan (W) and tyrosine (Y) are associated with low solubility (
     <a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig 2C</a> and Supplementary Fig S7A). Cysteine residue (C) has the lowest weight probably because disulfide bonds couldn’t be properly formed in the 
     <em>E. coli</em> expression hosts (
     <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stewart, Aslund, and Beckwith 1998</a>; 
     <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rosano and Ceccarelli 2014</a>; 
     <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jia and Jeon 2016</a>; 
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aslund and Beckwith 1999</a>). The weights are likely different if the solubility analysis was done using the reductase-deficient, 
     <em>E. coli</em> Origami host strains, or eukaryotic hosts.
    </p>
    <p id="p-31">Higher helix propensity has been reported to increase solubility (
     <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Idicula-Thomas and Balaji 2005</a>; 
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huang et al. 2012</a>). However, our analysis has shown that helical and turn propensities anti-correlate with solubility, whereas sheet propensity lacks correlation with solubility, suggesting that disordered regions may tend to be more soluble (
     <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig 3</a>). In accordance with these, SWI has stronger negative correlations with helix and turn propensities. These findings also suggest that protein solubility can be largely explained by overall amino acid composition, not just the surface amino acid residues. This idea aligns with our understanding that protein solubility and folding are closely linked, and folding occurs cotranscriptionally, a complex process that is driven various intrinsic and extrinsic factors (
     <a id="xref-ref-68-5" class="xref-bibr" href="#ref-68">Wilkinson and Harrison 1991</a>; 
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chiti et al. 2003</a>; 
     <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Tartaglia et al. 2004</a>; 
     <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Diaz et al. 2010</a>). However, it is unclear why sheet propensity has little contribution to solubility because β-sheets have been shown to link closely with protein aggregation (
     <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Idicula-Thomas and Balaji 2005</a>).
    </p>
    <p id="p-32">We conclude that SWI is a well-balanced index that is derived from a simple sequence composition scoring method. To demonstrate the usefulness of SWI, we developed a web server called SoDoPE (Soluble Domain for Protein Expression; 
     <a href="https://tisigner.com/sodope">https://tisigner.com/sodope</a>). SoDoPE calculates the probability of solubility of a user-selected region based on SWI, which can either be a full-length or a partial sequence (see Methods and Supplementary Table S8). This implementation is based on our observation that some protein domains tend to be more soluble than the others. To demonstrate this point, we have analysed three commercial monoclonal antibodies and the severe acute respiratory syndrome coronavirus proteomes (SARS-CoV and SARS-CoV-2) (
     <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wang et al. 2009</a>; 
     <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Marra et al. 2003</a>; F. 
     <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wu et al. 2020</a>) (Supplementary Fig S8 and S9). These soluble domains may enhance protein solubility as a whole. SoDoPE also provides options for solubility prediction at the presence of solubility fusion tags. Similarly, solubility tags may act as soluble ‘protein domains’ that can outweigh the aggregation propensity of insoluble proteins. However, some soluble fusion proteins may become insoluble after proteolytic cleavage of solubility tags (
     <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lebendiker and Danieli 2014</a>). In addition, SoDoPE is integrated with TIsigner, a gene optimisation web service for protein expression. This pipeline provides a holistic approach to improve the outcome of recombinant protein expression.
    </p>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">METHODS</h2>
    <div id="sec-11" class="subsection">
     <h3>Protein sequence properties</h3>
     <p id="p-33">The standard protein sequence properties were calculated using the Bio.SeqUtils.ProtParam module of Biopython v1.73 (
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cock et al. 2009</a>). All miscellaneous protein sequence properties were computed using the R package protr v1.6-2 (N. 
      <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Xiao et al. 2015</a>).
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Protein solubility prediction</h3>
     <p id="p-34">We used the standard and miscellaneous protein sequence properties to predict the solubility of the PSI:Biology and eSOL targets (N=12,216 and 3,198, respectively) (
      <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Seiler et al. 2014</a>; 
      <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Niwa et al. 2009</a>). For method comparison, we chose the protein solubility prediction tools that are scalable (
      <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Default configurations were used for running the command line tools.
     </p>
     <p id="p-35">To benchmark the wall time of solubility prediction tools, we selected 10 sequences that span a large range of lengths from the PSI:Biology and eSOL datasets (from 36 to 2389 residues). All the tools were run and timed using a single process without using GPUs on a high performance computer [
      <kbd>/usr/bin/time -f ‘%E’ &lt;;command&gt;;</kbd>; CentOS Linux 7 (Core) operating system, 72 cores in 2× Broadwell nodes (E5-2695v4, 2.1 GHz, dual socket 18 cores per socket), 528 GiB memory]. Single sequence fasta files were used as input files.
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>SWI</h3>
     <p id="p-36">To improve protein solubility prediction, we optimised the most recently published set of normalised B-factors using the PSI:Biology dataset (
      <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Smith et al. 2003</a>) (
      <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig 2</a>). To avoid including homologous sequences in the test and training sets, we clustered the PSI:Biology targets using USEARCH v11.0.667, 32-bit (
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Edgar 2010</a>). His-tag sequences were removed from all sequences before clustering to avoid false cluster inclusions. We obtained 5,050 clusters using the parameters: 
      <kbd>-cluster_fast &lt;;input_fiie&gt;; -id 0.4 -msaout &lt;;output_fiie&gt;; -threads 4</kbd>. These clusters were divided into 10 subsets with approximately 1,200 sequences per subsets manually. The subsequent steps were done with His-tag sequences. We used Smith 
      <em>et al.’</em> s normalised B-factors as the initial weights to maximise AUC using these 10 subsets with a 10-fold cross-validation. Since AUC is non-differentiable, we used the Nelder-Mead optimisation method (implemented in SciPy v1.2.0), which is a derivative-free, heuristic, simplex-based optimisation (
      <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Oliphant 2007</a>; 
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Millman and Aivazis 2011</a>; 
      <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Nelder and Mead 1965</a>). For each step in cross-validation, we used 1,000 bootstrap resamplings containing 1,000 soluble and 1,000 insoluble proteins. Optimisation was carried out for each sample, giving 1,000 sets of weights. The arithmetic mean of these weights was used to determine the training and test AUC for the cross-validation step (
      <a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig 2A</a>).
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Bit score</h3>
     <p id="p-37">To examine the enrichment of amino acid residues in soluble and insoluble proteins, we compute the bit scores for each amino acid residue in the PSI:Biology soluble and insoluble groups (Supplementary Fig S7A), we normalised the count of each residue (
      <em>x</em>) in each group by the total number of residues in that group. We used the normalised count of amino acid residues using the eSOL 
      <em>E. coli</em> sequences as the background. The bit score of residue (
      <em>x</em>) for soluble or insoluble group is then given by the following equation: 
      <span class="disp-formula" id="disp-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/26/2020.02.15.951012/embed/graphic-8.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/26/2020.02.15.951012/embed/graphic-8.gif"/>
        </noscript>
       </span>
      </span> where 
      <em>f
       <sub>i</sub>
      </em>(
      <em>x</em>) is the normalised count of residue (
      <em>x</em>) in the PSI:Biology soluble or insoluble group and 
      <em>f
       <sub>eSOL</sub>
      </em>(
      <em>x</em>) is the normalised count in the eSOL sequences.
     </p>
     <p id="p-38">For a control, random protein sequences were generated by incrementing the length of sequence, starting from a length of 50 residues to 6,000 residues with a step size of 50 residues. A hundred random sequences were generated for each length, giving a total of 12,000 unique random sequences.</p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>The SoDoPE web server</h3>
     <p id="p-39">To estimate the probability of solubility using SWI, we fitted the following logistic regression to the PSI:Biology dataset: 
      <span class="disp-formula" id="disp-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/26/2020.02.15.951012/embed/graphic-9.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/26/2020.02.15.951012/embed/graphic-9.gif"/>
        </noscript>
       </span>
      </span> where, 
      <em>x</em> is the SWI of a given protein sequence, 
      <em>a</em> = 81.05812 and 
      <em>b</em> = −62.7775. The P-value of log-likelihood ratio test was less than machine precision. 
      <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a> can be used to predict the solubility of a protein sequence given that the protein is successfully expressed in 
      <em>E. coli</em> (Supplementary Table S8).
     </p>
     <p id="p-40">On this basis, we developed a solubility prediction webservice called the Soluble Domain for Protein Expression (SoDoPE). Our web server accepts either a nucleotide or amino acid sequence. Upon sequence submission, a query is sent to the HMMER web server to annotate protein domains (
      <a href="https://www.ebi.ac.uk/Tools/hmmer/">https://www.ebi.ac.uk/Tools/hmmer/</a>) (
      <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Potter et al. 2018</a>). Once the protein domains are identified, users can choose a domain or any custom region (including full-length sequence) to examine the probability of solubility, flexibility and GRAVY. This functionality enables protein biochemists to plan their experiments and opt for the domains or regions with high probability of solubility. Furthermore, we implemented a simulated annealing algorithm that maximised the probability of solubility for a given region by generating a list of regions with extended boundaries. Users can also predict the improvement in solubility by selecting a commonly used solubility tag or a custom tag.
     </p>
     <p id="p-41">We linked SoDoPE with TIsigner, which is our existing web server for maximising the accessibility of translation initiation sites (
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bhandari, Lim, and Gardner 2019</a>). This pipeline allows users to predict and optimise both protein expression and solubility for a gene of interest. The SoDoPE web server is freely available at 
      <a href="https://tisigner.com/sodope">https://tisigner.com/sodope</a>.
     </p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Statistical analysis</h3>
     <p id="p-42">Data analysis was done using Pandas v0.25.3 (
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">McKinney 2010</a>), scikit-learn v0.20.2 (
      <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pedregosa et al. 2011</a>), numpy v1.16.2 (
      <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">van der Walt, Colbert, and Varoquaux 2011</a>) and statsmodel v0.10.1(
      <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Seabold and Perktold 2010</a>). Plots were generated using Matplotlib v3.0.2 (
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Caswell et al. 2018</a>) and Seaborn v0.9.0 (
      <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Waskom et al. 2014</a>).
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Code and data availability</h3>
     <p id="p-43">Jupyter notebook of our analysis can be found at 
      <a href="https://github.com/Gardner-BinfLab/SoDoPE_paper_2020">https://github.com/Gardner-BinfLab/SoDoPE_paper_2020</a>. The source code for our solubility prediction server (SoDoPE) can be found at 
      <a href="https://github.com/Gardner-BinfLab/TISIGNER-ReactJS">https://github.com/Gardner-BinfLab/TISIGNER-ReactJS</a>.
     </p>
    </div>
   </div>
   <div class="section" id="sec-19">
    <h2 class="">AUTHOR CONTRIBUTIONS</h2>
    <p id="p-45">C.S.L. conceived the work; B.K.B. and C.S.L. analysed the data and C.S.L. contributed flexibility analysis; B.K.B. and P.P.G formulated SWI; B.K.B. developed the SoDoPE web server; B.K.B., P.P.G. and C.S.L. wrote the manuscript.</p>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">COMPETING INTERESTS</h2>
    <p id="p-46">The authors declare no competing interests.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGEMENTS</h2>
    <p id="p-44">We thank New Zealand eScience Infrastructure for providing a high performance computing platform. We are grateful to Harry Biggs for proofreading our manuscript and providing feedback for the web server. This work was supported by the Ministry of Business, Innovation and Employment, New Zealand (MBIE grant: UOOX1709).</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-1">
      <p id="p-5">Added the solubility analysis of SARS-CoV and SARS-CoV-2 proteins in Supplementary Fig S9.</p>
     </li>
     <li class="fn-dataset fn-group-external-links" id="fn-2">
      <p id="p-6">
       <a href="https://github.com/Gardner-BinfLab/SoDoPE_paper_2020">https://github.com/Gardner-BinfLab/SoDoPE_paper_2020</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.1" data-doi="10.1016/S0076-6879(05)94008-1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Acton</span>, 
          <span class="cit-name-given-names">Thomas B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Kristin C.</span>
          <span class="cit-name-surname">Gunsalus</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Rong</span>
          <span class="cit-name-surname">Xiao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Li Chung</span>
          <span class="cit-name-surname">Ma</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">James</span>
          <span class="cit-name-surname">Aramini</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Michael C.</span>
          <span class="cit-name-surname">Baran</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yi-Wen</span>
          <span class="cit-name-surname">Chiang</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2005</span>. “
         <span class="cit-article-title">Robotic Cloning and Protein Production Platform of the Northeast Structural Genomics Consortium</span>.” 
         <abbr class="cit-jnl-abbrev">Methods in Enzymology</abbr>
         <span class="cit-vol">394</span>: 
         <span class="cit-fpage">210</span>–
         <span class="cit-lpage">43</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DActon%26rft.auinit1%253DT.%2BB.%26rft.volume%253D394%26rft.spage%253D210%26rft.epage%253D243%26rft.atitle%253DRobotic%2Bcloning%2Band%2BProtein%2BProduction%2BPlatform%2Bof%2Bthe%2BNortheast%2BStructural%2BGenomics%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252805%252994008-1%26rft_id%253Dinfo%253Apmid%252F15808222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0076-6879(05)94008-1&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15808222&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000228718700008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.2" data-doi="10.1093/bioinformatics/btu420">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Agostini</span>, 
          <span class="cit-name-given-names">Federico</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Davide</span>
          <span class="cit-name-surname">Cirillo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Carmen Maria</span>
          <span class="cit-name-surname">Livi</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Riccardo Delli</span>
          <span class="cit-name-surname">Ponti</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Gian Gaetano</span>
          <span class="cit-name-surname">Tartaglia</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. “
         <span class="cit-article-title">ccSOL Omics: A Webserver for Solubility Prediction of Endogenous and Heterologous Expression in Escherichia Coli</span>.” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">30</span> (
         <span class="cit-issue">20</span>): 
         <span class="cit-fpage">2975</span>–
         <span class="cit-lpage">77</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu420%26rft_id%253Dinfo%253Apmid%252F24990610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu420&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24990610&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aslund</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Beckwith</span>
         </span>. 
         <span class="cit-pub-date">1999</span>. “
         <span class="cit-article-title">The Thioredoxin Superfamily: Redundancy, Specificity, and Gray-Area Genomics</span>.” 
         <abbr class="cit-jnl-abbrev">Journal of Bacteriology</abbr>
         <span class="cit-vol">181</span> (
         <span class="cit-issue">5</span>): 
         <span class="cit-fpage">1375</span>–
         <span class="cit-lpage">79</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DAslund%26rft.auinit1%253DF.%26rft.volume%253D181%26rft.issue%253D5%26rft.spage%253D1375%26rft.epage%253D1379%26rft.atitle%253DThe%2BThioredoxin%2BSuperfamily%253A%2BRedundancy%252C%2BSpecificity%252C%2Band%2BGray-Area%2BGenomics%26rft_id%253Dinfo%253Apmid%252F10049365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4MS81LzEzNzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNi8yMDIwLjAyLjE1Ljk1MTAxMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bhandari</span>, 
          <span class="cit-name-given-names">Bikash K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Chun Shen</span>
          <span class="cit-name-surname">Lim</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Paul P.</span>
          <span class="cit-name-surname">Gardner</span>
         </span>. 
         <span class="cit-pub-date">2019</span>. “
         <span class="cit-article-title">Highly Accessible Translation Initiation Sites Are Predictive of Successful Heterologous Protein Expression</span>.” 
         <span class="cit-source">bioRxiv</span>. 
         <a href="https://doi.org/10.1101/726752">https://doi.org/10.1101/726752</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bhaskaran</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. K.</span>
          <span class="cit-name-surname">Ponnuswamy</span>
         </span>. 
         <span class="cit-pub-date">1988</span>. “
         <span class="cit-article-title">Positional Flexibilities of Amino Acid Residues in Globular Proteins</span>.” 
         <abbr class="cit-jnl-abbrev">International Journal of Peptide and Protein Research</abbr>. 
         <a href="https://doi.org/10.1111/j.1399-3011.1988.tb01258.x">https://doi.org/10.1111/j.1399-3011.1988.tb01258.x</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Caswell</span>, 
          <span class="cit-name-given-names">Thomas A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Michael</span>
          <span class="cit-name-surname">Droettboom</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">John</span>
          <span class="cit-name-surname">Hunter</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Eric</span>
          <span class="cit-name-surname">Firing</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Antony</span>
          <span class="cit-name-surname">Lee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">David</span>
          <span class="cit-name-surname">Stansby</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Elliott Sales</span>
          <span class="cit-name-surname">de Andrade</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2018</span>. 
         <abbr class="cit-jnl-abbrev">Matplotlib/matplotlib v3.0.2</abbr> (version 3.0.2). 
         <a href="https://doi.org/10.5281/zenodo.1482099">https://doi.org/10.5281/zenodo.1482099</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">Wen-Ching</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Po-Huang</span>
          <span class="cit-name-surname">Liang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yan-Ping</span>
          <span class="cit-name-surname">Shih</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Ueng-Cheng</span>
          <span class="cit-name-surname">Yang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Wen-Chang</span>
          <span class="cit-name-surname">Lin</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Chun-Nan</span>
          <span class="cit-name-surname">Hsu</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. “
         <span class="cit-article-title">Learning to Predict Expression Efficacy of Vectors in Recombinant Protein Production</span>.” 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>
         <span class="cit-vol">11</span> Suppl 
         <span class="cit-issue">1</span> (
         <span class="cit-month">January</span>): 
         <span class="cit-fpage">S21</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D11%26rft.spage%253DS21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.8" data-doi="10.1093/bioinformatics/bth300">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Rose</span>
          <span class="cit-name-surname">Oughtred</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Helen M.</span>
          <span class="cit-name-surname">Berman</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">John</span>
          <span class="cit-name-surname">Westbrook</span>
         </span>. 
         <span class="cit-pub-date">2004</span>. “
         <span class="cit-article-title">TargetDB: A Target Registration Database for Structural Genomics Projects</span>.” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">20</span> (
         <span class="cit-issue">16</span>): 
         <span class="cit-fpage">2860</span>–
         <span class="cit-lpage">62</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth300%26rft_id%253Dinfo%253Apmid%252F15130928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth300&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15130928&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000225250100050&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.9" data-doi="10.1038/nature01891">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chiti</span>, 
          <span class="cit-name-given-names">Fabrizio</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Massimo</span>
          <span class="cit-name-surname">Stefani</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Niccolò</span>
          <span class="cit-name-surname">Taddei</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Giampietro</span>
          <span class="cit-name-surname">Ramponi</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Christopher M.</span>
          <span class="cit-name-surname">Dobson</span>
         </span>. 
         <span class="cit-pub-date">2003</span>. “
         <span class="cit-article-title">Rationalization of the Effects of Mutations on Peptide and Protein Aggregation Rates</span>.” 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">424</span> (
         <span class="cit-issue">6950</span>): 
         <span class="cit-fpage">805</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChiti%26rft.auinit1%253DF.%26rft.volume%253D424%26rft.issue%253D6950%26rft.spage%253D805%26rft.epage%253D808%26rft.atitle%253DRationalization%2Bof%2Bthe%2Beffects%2Bof%2Bmutations%2Bon%2Bpeptide%2Band%2Bprotein%2Baggregation%2Brates.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01891%26rft_id%253Dinfo%253Apmid%252F12917692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature01891&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12917692&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000184733900047&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.10" data-doi="10.1093/bioinformatics/btp163">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cock</span>, 
          <span class="cit-name-given-names">Peter J. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Tiago</span>
          <span class="cit-name-surname">Antao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jeffrey T.</span>
          <span class="cit-name-surname">Chang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Brad A.</span>
          <span class="cit-name-surname">Chapman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Cymon J.</span>
          <span class="cit-name-surname">Cox</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Andrew</span>
          <span class="cit-name-surname">Dalke</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Iddo</span>
          <span class="cit-name-surname">Friedberg</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2009</span>. “
         <span class="cit-article-title">Biopython: Freely Available Python Tools for Computational Molecular Biology and Bioinformatics</span>.” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span> (
         <span class="cit-issue">11</span>): 
         <span class="cit-fpage">1422</span>–
         <span class="cit-lpage">23</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp163%26rft_id%253Dinfo%253Apmid%252F19304878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp163&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19304878&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000266109500013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Costa</span>, 
          <span class="cit-name-given-names">Sofia</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">André</span>
          <span class="cit-name-surname">Almeida</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">António</span>
          <span class="cit-name-surname">Castro</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Lucília</span>
          <span class="cit-name-surname">Domingues</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. “
         <span class="cit-article-title">Fusion Tags for Protein Solubility, Purification and Immunogenicity in Escherichia Coli: The Novel Fh8 System</span>.” 
         <abbr class="cit-jnl-abbrev">Frontiers in Microbiology</abbr>
         <span class="cit-vol">5</span> (
         <span class="cit-month">February</span>): 
         <span class="cit-fpage">63</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BMicrobiology%26rft.volume%253D5%26rft.spage%253D63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Craveur</span>, 
          <span class="cit-name-given-names">Pierrick</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Agnel P.</span>
          <span class="cit-name-surname">Joseph</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jeremy</span>
          <span class="cit-name-surname">Esque</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Tarun J.</span>
          <span class="cit-name-surname">Narwani</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Floriane</span>
          <span class="cit-name-surname">Noël</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Nicolas</span>
          <span class="cit-name-surname">Shinada</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Matthieu</span>
          <span class="cit-name-surname">Goguet</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2015</span>. “
         <span class="cit-article-title">Protein Flexibility in the Light of Structural Alphabets</span>.” 
         <abbr class="cit-jnl-abbrev">Frontiers in Molecular Biosciences</abbr>
         <span class="cit-vol">2</span> (
         <span class="cit-month">May</span>): 
         <span class="cit-fpage">20</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BMolecular%2BBiosciences%26rft.volume%253D2%26rft.spage%253D20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.13" data-doi="10.1002/(SICI)1097-0290(19991120)65:4&lt;;382::AID-BIT2&gt;;3.0.CO;2-I">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Davis</span>, 
          <span class="cit-name-given-names">G. D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Elisee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. M.</span>
          <span class="cit-name-surname">Newham</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. G.</span>
          <span class="cit-name-surname">Harrison</span>
         </span>. 
         <span class="cit-pub-date">1999</span>. “
         <span class="cit-article-title">New Fusion Protein Systems Designed to Give Soluble Expression in Escherichia Coli</span>.” 
         <abbr class="cit-jnl-abbrev">Biotechnology and Bioengineering</abbr>
         <span class="cit-vol">65</span> (
         <span class="cit-issue">4</span>): 
         <span class="cit-fpage">382</span>–
         <span class="cit-lpage">88</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Band%2BBioengineering%26rft.stitle%253DBiotechnology%2Band%2BBioengineering%26rft.aulast%253DDavis%26rft.auinit1%253DG.%2BD.%26rft.volume%253D65%26rft.issue%253D4%26rft.spage%253D382%26rft.epage%253D388%26rft.atitle%253DNew%2Bfusion%2Bprotein%2Bsystems%2Bdesigned%2Bto%2Bgive%2Bsoluble%2Bexpression%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0290%252819991120%252965%253A4%253C382%253A%253AAID-BIT2%253E3.0.CO%253B2-I%26rft_id%253Dinfo%253Apmid%252F10506413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0290(19991120)65:4&lt;;382::AID-BIT2&gt;;3.0.CO;2-I&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10506413&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000083078500002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.14" data-doi="10.1002/bit.22537">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Diaz</span>, 
          <span class="cit-name-given-names">Armando A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Emanuele</span>
          <span class="cit-name-surname">Tomba</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Reese</span>
          <span class="cit-name-surname">Lennarson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Rex</span>
          <span class="cit-name-surname">Richard</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Miguel J.</span>
          <span class="cit-name-surname">Bagajewicz</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Roger G.</span>
          <span class="cit-name-surname">Harrison</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. “
         <span class="cit-article-title">Prediction of Protein Solubility in Escherichia Coli Using Logistic Regression</span>.” 
         <abbr class="cit-jnl-abbrev">Biotechnology and Bioengineering</abbr>
         <span class="cit-vol">105</span> (
         <span class="cit-issue">2</span>): 
         <span class="cit-fpage">374</span>–
         <span class="cit-lpage">83</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Band%2BBioengineering%26rft.stitle%253DBiotechnology%2Band%2BBioengineering%26rft.aulast%253DDiaz%26rft.auinit1%253DA.%2BA.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D374%26rft.epage%253D383%26rft.atitle%253DPrediction%2Bof%2Bprotein%2Bsolubility%2Bin%2BEscherichia%2Bcoli%2Busing%2Blogistic%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbit.22537%26rft_id%253Dinfo%253Apmid%252F19739095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/bit.22537&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19739095&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000273715500016&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.15" data-doi="10.1093/bioinformatics/btq461">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Edgar</span>, 
          <span class="cit-name-given-names">Robert C.</span>
         </span>
         <span class="cit-pub-date">2010</span>. “
         <span class="cit-article-title">Search and Clustering Orders of Magnitude Faster than BLAST</span>.” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">26</span> (
         <span class="cit-issue">19</span>): 
         <span class="cit-fpage">2460</span>–
         <span class="cit-lpage">61</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq461%26rft_id%253Dinfo%253Apmid%252F20709691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq461&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20709691&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000282170000016&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.16" data-doi="10.1016/j.copbio.2006.06.003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Esposito</span>, 
          <span class="cit-name-given-names">Dominic</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Deb K.</span>
          <span class="cit-name-surname">Chatterjee</span>
         </span>. 
         <span class="cit-pub-date">2006</span>. “
         <span class="cit-article-title">Enhancement of Soluble Protein Expression through the Use of Fusion Tags</span>.” 
         <abbr class="cit-jnl-abbrev">Current Opinion in Biotechnology</abbr>
         <span class="cit-vol">17</span> (
         <span class="cit-issue">4</span>): 
         <span class="cit-fpage">353</span>–
         <span class="cit-lpage">58</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DEsposito%26rft.auinit1%253DD.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D353%26rft.epage%253D358%26rft.atitle%253DEnhancement%2Bof%2Bsoluble%2Bprotein%2Bexpression%2Bthrough%2Bthe%2Buse%2Bof%2Bfusion%2Btags.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2006.06.003%26rft_id%253Dinfo%253Apmid%252F16781139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.copbio.2006.06.003&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16781139&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000240023300005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.15.951012v2.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Família</span>, 
          <span class="cit-name-given-names">Carlos</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Sarah R.</span>
          <span class="cit-name-surname">Dennison</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Alexandre</span>
          <span class="cit-name-surname">Quintas</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">David A.</span>
          <span class="cit-name-surname">Phoenix</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. “
         <span class="cit-article-title">Prediction of Peptide and Protein Propensity for Amyloid Formation</span>.” 
         <abbr class="cit-jnl-abbrev">PloS One</abbr>
         <span class="cit-vol">10</span> (
         <span class="cit-issue">8</span>): 
         <span class="cit-fpage">e0134679</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPloS%2BOne%26rft.volume%253D10%26rft.spage%253De0134679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Habibi</span>, 
          <span class="cit-name-given-names">Narjeskhatoon</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Siti Z. Mohd</span>
          <span class="cit-name-surname">Hashim</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Alireza</span>
          <span class="cit-name-surname">Norouzi</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Mohammed Razip</span>
          <span class="cit-name-surname">Samian</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. “
         <span class="cit-article-title">A Review of Machine Learning Methods to Predict the Solubility of Overexpressed Recombinant Proteins in Escherichia Coli</span>.” 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>
         <span class="cit-vol">15</span> (
         <span class="cit-month">May</span>): 
         <span class="cit-fpage">134</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">Xi</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Wenbo</span>
          <span class="cit-name-surname">Ning</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Xiaoqiang</span>
          <span class="cit-name-surname">Ma</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Xiaonan</span>
          <span class="cit-name-surname">Wang</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Kang</span>
          <span class="cit-name-surname">Zhou</span>
         </span>. 
         <span class="cit-pub-date">2019</span>. “
         <span class="cit-article-title">Improve Protein Solubility and Activity Based on Machine Learning Models</span>.” 
         <span class="cit-source">bioRxiv</span>. 
         <a href="https://doi.org/10.1101/817890">https://doi.org/10.1101/817890</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Harrison</span>, 
          <span class="cit-name-given-names">R. G.</span>
         </span>
         <span class="cit-pub-date">2000</span>. “
         <span class="cit-article-title">Expression of Soluble Heterologous Proteins via Fusion with NusA Protein</span>.” 
         <abbr class="cit-jnl-abbrev">Innovations</abbr>
         <span class="cit-vol">11</span>: 
         <span class="cit-fpage">4</span>–
         <span class="cit-lpage">7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInnovations%26rft.volume%253D11%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hebditch</span>, 
          <span class="cit-name-given-names">Max</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Alejandro Carballo-Amador</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Spyros</span>
          <span class="cit-name-surname">Charonis</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Robin</span>
          <span class="cit-name-surname">Curtis</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jim</span>
          <span class="cit-name-surname">Warwicker</span>
         </span>. 
         <span class="cit-pub-date">2017</span>. “
         <span class="cit-article-title">Protein-Sol: A Web Tool for Predicting Protein Solubility from Sequence</span>.” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">33</span> (
         <span class="cit-issue">19</span>): 
         <span class="cit-fpage">3098</span>–
         <span class="cit-lpage">3100</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D3098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Heckmann</span>, 
          <span class="cit-name-given-names">David</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Colton J.</span>
          <span class="cit-name-surname">Lloyd</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Nathan</span>
          <span class="cit-name-surname">Mih</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yuanchi</span>
          <span class="cit-name-surname">Ha</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Daniel C.</span>
          <span class="cit-name-surname">Zielinski</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Zachary B.</span>
          <span class="cit-name-surname">Haiman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Abdelmoneim Amer</span>
          <span class="cit-name-surname">Desouki</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Martin J.</span>
          <span class="cit-name-surname">Lercher</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bernhard O.</span>
          <span class="cit-name-surname">Palsson</span>
         </span>. 
         <span class="cit-pub-date">2018</span>. “
         <span class="cit-article-title">Machine Learning Applied to Enzyme Turnover Numbers Reveals Protein Structural Correlates and Improves Metabolic Models</span>.” 
         <abbr class="cit-jnl-abbrev">Nature Communications</abbr>
         <span class="cit-vol">9</span> (
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">5252</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D5252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.23" data-doi="10.1002/pmic.201200175">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hirose</span>, 
          <span class="cit-name-given-names">Shuichi</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Tamotsu</span>
          <span class="cit-name-surname">Noguchi</span>
         </span>. 
         <span class="cit-pub-date">2013</span>. “
         <span class="cit-article-title">ESPRESSO: A System for Estimating Protein Expression and Solubility in Protein Expression Systems</span>.” 
         <abbr class="cit-jnl-abbrev">Proteomics</abbr>
         <span class="cit-vol">13</span> (
         <span class="cit-issue">9</span>): 
         <span class="cit-fpage">1444</span>–
         <span class="cit-lpage">56</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.volume%253D13%26rft.spage%253D1444%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201200175%26rft_id%253Dinfo%253Apmid%252F23436767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/pmic.201200175&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23436767&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hou</span>, 
          <span class="cit-name-given-names">Qingzhen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Raphaël</span>
          <span class="cit-name-surname">Bourgeas</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Fabrizio</span>
          <span class="cit-name-surname">Pucci</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Marianne</span>
          <span class="cit-name-surname">Rooman</span>
         </span>. 
         <span class="cit-pub-date">2018</span>. “
         <span class="cit-article-title">Computational Analysis of the Amino Acid Interactions That Promote or Decrease Protein Solubility</span>.” 
         <span class="cit-source">Scientific Reports</span>, 
         <a href="https://doi.org/10.1038/s41598-018-32988-w">https://doi.org/10.1038/s41598-018-32988-w</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hou</span>, 
          <span class="cit-name-given-names">Qingzhen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jean-Marc</span>
          <span class="cit-name-surname">Kwasigroch</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Marianne</span>
          <span class="cit-name-surname">Rooman</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Fabrizio</span>
          <span class="cit-name-surname">Pucci</span>
         </span>. 
         <span class="cit-pub-date">2019</span>. “
         <span class="cit-article-title">SOLart: A Structure-Based Method to Predict Protein Solubility and Aggregation</span>.” 
         <span class="cit-source">Bioinformatics</span>, 
         <span class="cit-month">October</span>. 
         <a href="https://doi.org/10.1093/bioinformatics/btz773">https://doi.org/10.1093/bioinformatics/btz773</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.26" data-doi="10.1186/1471&amp;;ndash;2105&amp;;ndash;13-S1-S3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">Hui-Ling</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Phasit</span>
          <span class="cit-name-surname">Charoenkwan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Te-Fen</span>
          <span class="cit-name-surname">Kao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Hua-Chin</span>
          <span class="cit-name-surname">Lee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Fang-Lin</span>
          <span class="cit-name-surname">Chang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Wen-Lin</span>
          <span class="cit-name-surname">Huang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Shinn-Jang</span>
          <span class="cit-name-surname">Ho</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Li-Sun</span>
          <span class="cit-name-surname">Shu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Wen-Liang</span>
          <span class="cit-name-surname">Chen</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Shinn-Ying</span>
          <span class="cit-name-surname">Ho</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. “
         <span class="cit-article-title">Prediction and Analysis of Protein Solubility Using a Novel Scoring Card Method with Dipeptide Composition</span>.” 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>
         <span class="cit-vol">13</span> Suppl 
         <span class="cit-issue">17</span> (
         <span class="cit-month">December</span>): 
         <span class="cit-fpage">S3</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D13%26rft.spage%253DS3%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471%2526ndash%253B2105%2526ndash%253B13-S1-S3%26rft_id%253Dinfo%253Apmid%252F22373359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471&amp;;ndash;2105&amp;;ndash;13-S1-S3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22373359&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Idicula-Thomas</span>, 
          <span class="cit-name-given-names">Susan</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Petety V.</span>
          <span class="cit-name-surname">Balaji</span>
         </span>. 
         <span class="cit-pub-date">2005</span>. “
         <span class="cit-article-title">Understanding the Relationship between the Primary Structure of Proteins and Its Propensity to Be Soluble on Overexpression in Escherichia Coli</span>.” 
         <abbr class="cit-jnl-abbrev">Protein Science: A Publication of the Protein Society</abbr>
         <span class="cit-vol">14</span> (
         <span class="cit-issue">3</span>): 
         <span class="cit-fpage">582</span>–
         <span class="cit-lpage">92</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%253A%2BA%2BPublication%2Bof%2Bthe%2BProtein%2BSociety%26rft.volume%253D14%26rft.spage%253D582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jia</span>, 
          <span class="cit-name-given-names">Baolei</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Che Ok</span>
          <span class="cit-name-surname">Jeon</span>
         </span>. 
         <span class="cit-pub-date">2016</span>. “
         <span class="cit-article-title">High-Throughput Recombinant Protein Expression in Escherichia Coli: Current Status and Future Perspectives</span>.” 
         <abbr class="cit-jnl-abbrev">Open Biology</abbr>
         <span class="cit-vol">6</span> (
         <span class="cit-issue">8</span>). 
         <a href="https://doi.org/10.1098/rsob.160196">https://doi.org/10.1098/rsob.160196</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.29" data-doi="10.1007/BF01195768">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Karplus</span>, 
          <span class="cit-name-given-names">P. A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. E.</span>
          <span class="cit-name-surname">Schulz</span>
         </span>. 
         <span class="cit-pub-date">1985</span>. “
         <span class="cit-article-title">Prediction of Chain Flexibility in Proteins</span>.” 
         <abbr class="cit-jnl-abbrev">Die Naturwissenschaften</abbr>
         <span class="cit-vol">72</span> (
         <span class="cit-issue">4</span>): 
         <span class="cit-fpage">212</span>–
         <span class="cit-lpage">13</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDie%2BNaturwissenschaften%26rft.volume%253D72%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01195768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/BF01195768&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1985AJQ6200010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Khurana</span>, 
          <span class="cit-name-given-names">Sameer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Reda</span>
          <span class="cit-name-surname">Rawi</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Khalid</span>
          <span class="cit-name-surname">Kunji</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Gwo-Yu</span>
          <span class="cit-name-surname">Chuang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Halima</span>
          <span class="cit-name-surname">Bensmail</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Raghvendra</span>
          <span class="cit-name-surname">Mall</span>
         </span>. 
         <span class="cit-pub-date">2018</span>. “
         <span class="cit-article-title">DeepSol: A Deep Learning Framework for Sequence-Based Protein Solubility Prediction</span>.” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">34</span> (
         <span class="cit-issue">15</span>): 
         <span class="cit-fpage">2605</span>–
         <span class="cit-lpage">13</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D2605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kramer</span>, 
          <span class="cit-name-given-names">Ryan M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Varad R.</span>
          <span class="cit-name-surname">Shende</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nicole Motl</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-auth">
          <span class="cit-name-given-names">Nick</span>
          <span class="cit-name-surname">Pace</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. Martin</span>
          <span class="cit-name-surname">Scholtz</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. “
         <span class="cit-article-title">Toward a Molecular Understanding of Protein Solubility: Increased Negative Surface Charge Correlates with Increased Solubility</span>.” 
         <span class="cit-source">Biophysical Journal</span>. 
         <a href="https://doi.org/10.1016/j.bpj.2012.01.060">https://doi.org/10.1016/j.bpj.2012.01.060</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kuriata</span>, 
          <span class="cit-name-given-names">Aleksander</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Valentin</span>
          <span class="cit-name-surname">Iglesias</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jordi</span>
          <span class="cit-name-surname">Pujols</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Mateusz</span>
          <span class="cit-name-surname">Kurcinski</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Sebastian</span>
          <span class="cit-name-surname">Kmiecik</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Salvador</span>
          <span class="cit-name-surname">Ventura</span>
         </span>. 
         <span class="cit-pub-date">2019</span>. “
         <span class="cit-article-title">Aggrescan3D (A3D) 2.0: Prediction and Engineering of Protein Solubility</span>.” 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-vol">47</span> (
         <span class="cit-issue">W1</span>): 
         <span class="cit-fpage">W300</span>–
         <span class="cit-lpage">307</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D47%26rft.spage%253DW300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.33" data-doi="10.1016/0022-2836(82)90515-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kyte</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. F.</span>
          <span class="cit-name-surname">Doolittle</span>
         </span>. 
         <span class="cit-pub-date">1982</span>. “
         <span class="cit-article-title">A Simple Method for Displaying the Hydropathic Character of a Protein</span>.” 
         <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr>
         <span class="cit-vol">157</span> (
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">105</span>–
         <span class="cit-lpage">32</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKyte%26rft.auinit1%253DJ.%26rft.volume%253D157%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D132%26rft.atitle%253DA%2Bsimple%2Bmethod%2Bfor%2Bdisplaying%2Bthe%2Bhydropathic%2Bcharacter%2Bof%2Ba%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252882%252990515-0%26rft_id%253Dinfo%253Apmid%252F7108955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0022-2836(82)90515-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=7108955&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1982NQ28000006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.34" data-doi="10.1016/j.febslet.2013.10.044">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lebendiker</span>, 
          <span class="cit-name-given-names">Mario</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Tsafi</span>
          <span class="cit-name-surname">Danieli</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. “
         <span class="cit-article-title">Production of Prone-to-Aggregate Proteins</span>.” 
         <abbr class="cit-jnl-abbrev">FEBS Letters</abbr>
         <span class="cit-vol">588</span> (
         <span class="cit-issue">2</span>): 
         <span class="cit-fpage">236</span>–
         <span class="cit-lpage">46</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.volume%253D588%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2013.10.044%26rft_id%253Dinfo%253Apmid%252F24211444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.febslet.2013.10.044&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24211444&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000329503100006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Levy</span>, 
          <span class="cit-name-given-names">E. D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">De</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. A.</span>
          <span class="cit-name-surname">Teichmann</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. “
         <span class="cit-article-title">Cellular Crowding Imposes Global Constraints on the Chemistry and Evolution of Proteomes</span>.” 
         <span class="cit-source">Proceedings of the National Academy of Sciences</span>. 
         <a href="https://doi.org/10.1073/pnas.1209312109">https://doi.org/10.1073/pnas.1209312109</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.36" data-doi="10.1016/j.str.2005.02.002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>, 
          <span class="cit-name-given-names">Jianpeng</span>
         </span>. 
         <span class="cit-pub-date">2005</span>. “
         <span class="cit-article-title">Usefulness and Limitations of Normal Mode Analysis in Modeling Dynamics of Biomolecular Complexes</span>.” 
         <abbr class="cit-jnl-abbrev">Structure</abbr>
         <span class="cit-vol">13</span> (
         <span class="cit-issue">3</span>): 
         <span class="cit-fpage">373</span>–
         <span class="cit-lpage">80</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DMa%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D373%26rft.epage%253D380%26rft.atitle%253DUsefulness%2Band%2Blimitations%2Bof%2Bnormal%2Bmode%2Banalysis%2Bin%2Bmodeling%2Bdynamics%2Bof%2Bbiomolecular%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2005.02.002%26rft_id%253Dinfo%253Apmid%252F15766538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.str.2005.02.002&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15766538&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.37" data-doi="10.1126/science.1085953">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Marra</span>, 
          <span class="cit-name-given-names">Marco A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Steven J. M.</span>
          <span class="cit-name-surname">Jones</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Caroline R.</span>
          <span class="cit-name-surname">Astell</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Robert A.</span>
          <span class="cit-name-surname">Holt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Angela</span>
          <span class="cit-name-surname">Brooks-Wilson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yaron S. N.</span>
          <span class="cit-name-surname">Butterfield</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jaswinder</span>
          <span class="cit-name-surname">Khattra</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2003</span>. “
         <span class="cit-article-title">The Genome Sequence of the SARS-Associated Coronavirus</span>.” 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">300</span> (
         <span class="cit-issue">5624</span>): 
         <span class="cit-fpage">1399</span>–
         <span class="cit-lpage">1404</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085953%26rft_id%253Dinfo%253Apmid%252F12730501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNC8xMzk5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjYvMjAyMC4wMi4xNS45NTEwMTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-2020.02.15.951012v2.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McKinney</span>, 
          <span class="cit-name-given-names">Wes</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. “
         <span class="cit-article-title">Data Structures for Statistical Computing in Python</span>.” In 
         <span class="cit-conf-name">Proceedings of the 9th Python in Science Conference</span>, 
         <span class="cit-fpage">51</span>–
         <span class="cit-lpage">56</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.39" data-doi="10.1109/mcse.2011.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Millman</span>, 
          <span class="cit-name-given-names">K. J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Aivazis</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. “
         <span class="cit-article-title">Python for Scientists and Engineers</span>.” 
         <abbr class="cit-jnl-abbrev">Computing in Science Engineering</abbr>
         <span class="cit-vol">13</span> (
         <span class="cit-issue">2</span>): 
         <span class="cit-fpage">9</span>–
         <span class="cit-lpage">12</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DComputing%2Bin%2BScience%2BEngineering%26rft.volume%253D13%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1109%252Fmcse.2011.36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1109/mcse.2011.36&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.40">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Natan</span>, 
          <span class="cit-name-given-names">Eviatar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Tamaki</span>
          <span class="cit-name-surname">Endoh</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Liora</span>
          <span class="cit-name-surname">Haim-Vilmovsky</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Tilman</span>
          <span class="cit-name-surname">Flock</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Guilhem</span>
          <span class="cit-name-surname">Chalancon</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jonathan T. S.</span>
          <span class="cit-name-surname">Hopper</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bálint</span>
          <span class="cit-name-surname">Kintses</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2018</span>. “
         <span class="cit-article-title">Cotranslational Protein Assembly Imposes Evolutionary Constraints on Homomeric Proteins</span>.” 
         <abbr class="cit-jnl-abbrev">Nature Structural &amp;; Molecular Biology</abbr>
         <span class="cit-vol">25</span> (
         <span class="cit-issue">3</span>): 
         <span class="cit-fpage">279</span>–
         <span class="cit-lpage">88</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BStructural%2B%2526%2BMolecular%2BBiology%26rft.volume%253D25%26rft.spage%253D279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.41" data-doi="10.1093/comjnl/7.4.308">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nelder</span>, 
          <span class="cit-name-given-names">J. A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Mead</span>
         </span>. 
         <span class="cit-pub-date">1965</span>. “
         <span class="cit-article-title">A Simplex Method for Function Minimization</span>.” 
         <abbr class="cit-jnl-abbrev">Computer Journal</abbr>
         <span class="cit-vol">7</span> (
         <span class="cit-issue">4</span>): 
         <span class="cit-fpage">308</span>–
         <span class="cit-lpage">13</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BComputer%2BJournal%26rft.stitle%253DThe%2BComputer%2BJournal%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D308%26rft.epage%253D313%26rft.atitle%253DA%2BSimplex%2BMethod%2Bfor%2BFunction%2BMinimization%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcomjnl%252F7.4.308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/comjnl/7.4.308&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1965CLA3200015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.42" data-doi="10.1073/pnas.0811922106">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Niwa</span>, 
          <span class="cit-name-given-names">Tatsuya</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bei-Wen</span>
          <span class="cit-name-surname">Ying</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Katsuyo</span>
          <span class="cit-name-surname">Saito</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Wenzhen</span>
          <span class="cit-name-surname">Jin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Shoji</span>
          <span class="cit-name-surname">Takada</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Takuya</span>
          <span class="cit-name-surname">Ueda</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Hideki</span>
          <span class="cit-name-surname">Taguchi</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. “
         <span class="cit-article-title">Bimodal Protein Solubility Distribution Revealed by an Aggregation Analysis of the Entire Ensemble of Escherichia Coli Proteins</span>.” 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>
         <span class="cit-vol">106</span> (
         <span class="cit-issue">11</span>): 
         <span class="cit-fpage">4201</span>–
         <span class="cit-lpage">6</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0811922106%26rft_id%253Dinfo%253Apmid%252F19251648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzExLzQyMDEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNi8yMDIwLjAyLjE1Ljk1MTAxMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.43" data-doi="10.1109/MCSE.2007.58.">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Oliphant</span>, 
          <span class="cit-name-given-names">T. E.</span>
         </span>
         <span class="cit-pub-date">2007</span>. “
         <span class="cit-article-title">Python for Scientific Computing</span>.” 
         <abbr class="cit-jnl-abbrev">Computing in Science Engineering</abbr>
         <span class="cit-vol">9</span> (
         <span class="cit-issue">3</span>): 
         <span class="cit-fpage">10</span>–
         <span class="cit-lpage">20</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DComputing%2Bin%2BScience%2BEngineering%26rft.volume%253D9%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2007.58.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1109/MCSE.2007.58.&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.44">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pedregosa</span>, 
          <span class="cit-name-given-names">Fabian</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Gaël</span>
          <span class="cit-name-surname">Varoquaux</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Alexandre</span>
          <span class="cit-name-surname">Gramfort</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Vincent</span>
          <span class="cit-name-surname">Michel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bertrand</span>
          <span class="cit-name-surname">Thirion</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Olivier</span>
          <span class="cit-name-surname">Grisel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Mathieu</span>
          <span class="cit-name-surname">Blondel</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2011</span>. “
         <span class="cit-article-title">Scikit-Learn: Machine Learning in Python</span>.” 
         <abbr class="cit-jnl-abbrev">Journal of Machine Learning Research: JMLR</abbr>
         <span class="cit-vol">12</span> (
         <span class="cit-month">Oct</span>): 
         <span class="cit-fpage">2825</span>–
         <span class="cit-lpage">30</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMachine%2BLearning%2BResearch%253A%2BJMLR%26rft.volume%253D12%26rft.spage%253D2825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.45" data-doi="10.1093/nar/gky448">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Potter</span>, 
          <span class="cit-name-given-names">Simon C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Aurélien</span>
          <span class="cit-name-surname">Luciani</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Sean R.</span>
          <span class="cit-name-surname">Eddy</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Youngmi</span>
          <span class="cit-name-surname">Park</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Rodrigo</span>
          <span class="cit-name-surname">Lopez</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Robert D.</span>
          <span class="cit-name-surname">Finn</span>
         </span>. 
         <span class="cit-pub-date">2018</span>. “
         <span class="cit-article-title">HMMER Web Server: 2018 Update</span>.” 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-vol">46</span> (
         <span class="cit-issue">W1</span>): 
         <span class="cit-fpage">W200</span>–
         <span class="cit-lpage">204</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D46%26rft.spage%253DW200%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky448&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Radivojac</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-pub-date">2004</span>. “
         <span class="cit-article-title">Protein Flexibility and Intrinsic Disorder</span>.” 
         <span class="cit-source">Protein Science</span>. 
         <a href="https://doi.org/10.1110/ps.03128904">https://doi.org/10.1110/ps.03128904</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.47">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ragone</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Facchiano</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Facchiano</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. M.</span>
          <span class="cit-name-surname">Facchiano</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Colonna</span>
         </span>. 
         <span class="cit-pub-date">1989</span>. “
         <span class="cit-article-title">Flexibility Plot of Proteins</span>.” 
         <span class="cit-source">“Protein Engineering, Design and Selection</span>.” 
         <a href="https://doi.org/10.1093/protein/2.7.497">https://doi.org/10.1093/protein/2.7.497</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.48">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rawi</span>, 
          <span class="cit-name-given-names">Reda</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Raghvendra</span>
          <span class="cit-name-surname">Mall</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Khalid</span>
          <span class="cit-name-surname">Kunji</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Chen-Hsiang</span>
          <span class="cit-name-surname">Shen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peter</span> D 
          <span class="cit-name-given-names">. Kwong</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Gwo-Yu</span>
          <span class="cit-name-surname">Chuang</span>
         </span>. 
         <span class="cit-pub-date">2018</span>. “
         <span class="cit-article-title">PaRSnIP: Sequence-Based Protein Solubility Prediction Using Gradient Boosting Machine</span>.” 
         <span class="cit-source">Bioinformatics</span>. 
         <a href="https://doi.org/10.1093/bioinformatics/btx662">https://doi.org/10.1093/bioinformatics/btx662</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.49">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rosano</span>, 
          <span class="cit-name-given-names">Germán L.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Eduardo A.</span>
          <span class="cit-name-surname">Ceccarelli</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. “
         <span class="cit-article-title">Recombinant Protein Expression in Escherichia Coli: Advances and Challenges</span>.” 
         <abbr class="cit-jnl-abbrev">Frontiers in Microbiology</abbr>
         <span class="cit-vol">5</span> (
         <span class="cit-month">April</span>): 
         <span class="cit-fpage">172</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BMicrobiology%26rft.volume%253D5%26rft.spage%253D172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.50" data-doi="10.1002/prot.20587">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schlessinger</span>, 
          <span class="cit-name-given-names">Avner</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Burkhard</span>
          <span class="cit-name-surname">Rost</span>
         </span>. 
         <span class="cit-pub-date">2005</span>. “
         <span class="cit-article-title">Protein Flexibility and Rigidity Predicted from Sequence</span>.” 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>
         <span class="cit-vol">61</span> (
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">115</span>–
         <span class="cit-lpage">26</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DSchlessinger%26rft.auinit1%253DA.%26rft.volume%253D61%26rft.issue%253D1%26rft.spage%253D115%26rft.epage%253D126%26rft.atitle%253DProtein%2Bflexibility%2Band%2Brigidity%2Bpredicted%2Bfrom%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.20587%26rft_id%253Dinfo%253Apmid%252F16080156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/prot.20587&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16080156&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000231683500011&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference in text" id="ref-51">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.51">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Seabold</span>, 
          <span class="cit-name-given-names">Skipper</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Josef</span>
          <span class="cit-name-surname">Perktold</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. “
         <span class="cit-article-title">Statsmodels: Econometric and Statistical Modeling with Python</span>.” In 
         <span class="cit-source">Proceedings of the 9th Python in Science Conference</span>. 
         <a href="http://conference.scipy.org/proceedings/scipy2010/pdfs/seabold.pdf">http://conference.scipy.org/proceedings/scipy2010/pdfs/seabold.pdf</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference in text" id="ref-52">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.52" data-doi="10.1093/nar/gkt1060">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Seiler</span>, 
          <span class="cit-name-given-names">Catherine Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Jin G.</span>
          <span class="cit-name-surname">Park</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Amit</span>
          <span class="cit-name-surname">Sharma</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Preston</span>
          <span class="cit-name-surname">Hunter</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Padmini</span>
          <span class="cit-name-surname">Surapaneni</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Casey</span>
          <span class="cit-name-surname">Sedillo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">James</span>
          <span class="cit-name-surname">Field</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2014</span>. “
         <span class="cit-article-title">DNASU Plasmid and PSI:Biology-Materials Repositories: Resources to Accelerate Biological Research</span>.” 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-vol">42</span> (
         <span class="cit-issue">Database issue</span>): 
         <span class="cit-fpage">D1253</span>–
         <span class="cit-lpage">60</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1060%26rft_id%253Dinfo%253Apmid%252F24225319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkt1060&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24225319&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000331139800183&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference in text" id="ref-53">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.53">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>, 
          <span class="cit-name-given-names">David K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Predrag</span>
          <span class="cit-name-surname">Radivojac</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Zoran</span>
          <span class="cit-name-surname">Obradovic</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. Keith</span>
          <span class="cit-name-surname">Dunker</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Guang</span>
          <span class="cit-name-surname">Zhu</span>
         </span>. 
         <span class="cit-pub-date">2003</span>. “
         <span class="cit-article-title">Improved Amino Acid Flexibility Parameters</span>.” 
         <abbr class="cit-jnl-abbrev">Protein Science: A Publication of the Protein Society</abbr>
         <span class="cit-vol">12</span> (
         <span class="cit-issue">5</span>): 
         <span class="cit-fpage">1060</span>–
         <span class="cit-lpage">72</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%253A%2BA%2BPublication%2Bof%2Bthe%2BProtein%2BSociety%26rft.volume%253D12%26rft.spage%253D1060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference in text" id="ref-54">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.54">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sormanni</span>, 
          <span class="cit-name-given-names">Pietro</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Leanne</span>
          <span class="cit-name-surname">Amery</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Sofia</span>
          <span class="cit-name-surname">Ekizoglou</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Michele</span>
          <span class="cit-name-surname">Vendruscolo</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bojana</span>
          <span class="cit-name-surname">Popovic</span>
         </span>. 
         <span class="cit-pub-date">2017</span>. “
         <span class="cit-article-title">Rapid and Accurate in Silico Solubility Screening of a Monoclonal Antibody Library</span>.” 
         <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>
         <span class="cit-vol">7</span> (
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">8200</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D7%26rft.spage%253D8200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference in text" id="ref-55">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.55">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sormanni</span>, 
          <span class="cit-name-given-names">Pietro</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Francesco A.</span>
          <span class="cit-name-surname">Aprile</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Michele</span>
          <span class="cit-name-surname">Vendruscolo</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. “
         <span class="cit-article-title">The CamSol Method of Rational Design of Protein Mutants with Enhanced Solubility</span>.” 
         <span class="cit-source">Journal of Molecular Biology</span>, 
         <a href="https://doi.org/10.1016/j.jmb.2014.09.026">https://doi.org/10.1016/j.jmb.2014.09.026</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference in text" id="ref-56">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.56" data-doi="10.1093/emboj/17.19.5543">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Stewart</span>, 
          <span class="cit-name-given-names">E. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Aslund</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Beckwith</span>
         </span>. 
         <span class="cit-pub-date">1998</span>. “
         <span class="cit-article-title">Disulfide Bond Formation in the Escherichia Coli Cytoplasm: An in Vivo Role Reversal for the Thioredoxins</span>.” 
         <abbr class="cit-jnl-abbrev">The EMBO Journal</abbr>
         <span class="cit-vol">17</span> (
         <span class="cit-issue">19</span>): 
         <span class="cit-fpage">5543</span>–
         <span class="cit-lpage">50</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DStewart%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D17%26rft.issue%253D19%26rft.spage%253D5543%26rft.epage%253D5550%26rft.atitle%253DDisulfide%2Bbond%2Bformation%2Bin%2Bthe%2BEscherichia%2Bcoli%2Bcytoplasm%253A%2Ban%2Bin%2Bvivo%2Brole%2Breversal%2Bfor%2Bthe%2Bthioredoxins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.19.5543%26rft_id%253Dinfo%253Apmid%252F9755155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTcvMTkvNTU0MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI2LzIwMjAuMDIuMTUuOTUxMDEyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference in text" id="ref-57">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.57">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tartaglia</span>, 
          <span class="cit-name-given-names">Gian Gaetano</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Andrea</span>
          <span class="cit-name-surname">Cavalli</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Riccardo</span>
          <span class="cit-name-surname">Pellarin</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Amedeo</span>
          <span class="cit-name-surname">Caflisch</span>
         </span>. 
         <span class="cit-pub-date">2004</span>. “
         <span class="cit-article-title">The Role of Aromaticity, Exposed Surface, and Dipole Moment in Determining Protein Aggregation Rates</span>.” 
         <abbr class="cit-jnl-abbrev">Protein Science: A Publication of the Protein Society</abbr>
         <span class="cit-vol">13</span> (
         <span class="cit-issue">7</span>): 
         <span class="cit-fpage">1939</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%253A%2BA%2BPublication%2Bof%2Bthe%2BProtein%2BSociety%26rft.volume%253D13%26rft.spage%253D1939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference in text" id="ref-58">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.58" data-doi="10.1038/nrd1129">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Teague</span>, 
          <span class="cit-name-given-names">Simon J.</span>
         </span>
         <span class="cit-pub-date">2003</span>. “
         <span class="cit-article-title">Implications of Protein Flexibility for Drug Discovery</span>.” 
         <abbr class="cit-jnl-abbrev">Nature Reviews. Drug Discovery</abbr>
         <span class="cit-vol">2</span> (
         <span class="cit-issue">7</span>): 
         <span class="cit-fpage">527</span>–
         <span class="cit-lpage">41</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BDrug%2Bdiscovery%26rft.stitle%253DNat%2BRev%2BDrug%2BDiscov%26rft.aulast%253DTeague%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D527%26rft.epage%253D541%26rft.atitle%253DImplications%2Bof%2Bprotein%2Bflexibility%2Bfor%2Bdrug%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd1129%26rft_id%253Dinfo%253Apmid%252F12838268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrd1129&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12838268&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000183713400015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference in text" id="ref-59">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.59">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Trevino</span>, 
          <span class="cit-name-given-names">Saul R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Martin Scholtz</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Nick Pace</span>
         </span>. 
         <span class="cit-pub-date">2007</span>. “
         <span class="cit-article-title">Amino Acid Contribution to Protein Solubility: Asp, Glu, and Ser Contribute More Favorably than the Other Hydrophilic Amino Acids in RNase Sa</span>.” 
         <span class="cit-source">Journal of Molecular Biology</span>, 
         <a href="https://doi.org/10.1016/j.jmb.2006.10.026">https://doi.org/10.1016/j.jmb.2006.10.026</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference in text" id="ref-60">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.60" data-doi="10.1016/S1046-5928(02)00641-1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tsumoto</span>, 
          <span class="cit-name-given-names">Kouhei</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Daisuke</span>
          <span class="cit-name-surname">Ejima</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Izumi</span>
          <span class="cit-name-surname">Kumagai</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Tsutomu</span>
          <span class="cit-name-surname">Arakawa</span>
         </span>. 
         <span class="cit-pub-date">2003</span>. “
         <span class="cit-article-title">Practical Considerations in Refolding Proteins from Inclusion Bodies</span>.” 
         <abbr class="cit-jnl-abbrev">Protein Expression and Purification</abbr>
         <span class="cit-vol">28</span> (
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2Bexpression%2Band%2Bpurification%26rft.stitle%253DProtein%2BExpr%2BPurif%26rft.aulast%253DTsumoto%26rft.auinit1%253DK.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D8%26rft.atitle%253DPractical%2Bconsiderations%2Bin%2Brefolding%2Bproteins%2Bfrom%2Binclusion%2Bbodies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-5928%252802%252900641-1%26rft_id%253Dinfo%253Apmid%252F12651100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S1046-5928(02)00641-1&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12651100&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000182063100001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference in text" id="ref-61">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.61">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vihinen</span>, 
          <span class="cit-name-given-names">Mauno</span>
         </span>. 
         <span class="cit-pub-date">1987</span>. “
         <span class="cit-article-title">Relationship of Protein Flexibility to Thermostability</span>.” 
         <span class="cit-source">“Protein Engineering, Design and Selection</span>.” 
         <a href="https://doi.org/10.1093/protein/1.6.477">https://doi.org/10.1093/protein/1.6.477</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference in text" id="ref-62">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.62" data-doi="10.1002/prot.340190207">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vihinen</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Torkkila</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Riikonen</span>
         </span>. 
         <span class="cit-pub-date">1994</span>. “
         <span class="cit-article-title">Accuracy of Protein Flexibility Predictions</span>.” 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>
         <span class="cit-vol">19</span> (
         <span class="cit-issue">2</span>): 
         <span class="cit-fpage">141</span>–
         <span class="cit-lpage">49</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DVihinen%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D141%26rft.epage%253D149%26rft.atitle%253DAccuracy%2Bof%2Bprotein%2Bflexibility%2Bpredictions.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.340190207%26rft_id%253Dinfo%253Apmid%252F8090708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/prot.340190207&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=8090708&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1994NR17900006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference in text" id="ref-63">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.63" data-doi="10.1016/S1367-5931(02)00017-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waldo</span>, 
          <span class="cit-name-given-names">Geoffrey S.</span>
         </span>
         <span class="cit-pub-date">2003</span>. “
         <span class="cit-article-title">Genetic Screens and Directed Evolution for Protein Solubility</span>.” 
         <abbr class="cit-jnl-abbrev">Current Opinion in Chemical Biology</abbr>
         <span class="cit-vol">7</span> (
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">33</span>–
         <span class="cit-lpage">38</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bchemical%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BChem%2BBiol%26rft.aulast%253DWaldo%26rft.auinit1%253DG.%2BS.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DGenetic%2Bscreens%2Band%2Bdirected%2Bevolution%2Bfor%2Bprotein%2Bsolubility.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1367-5931%252802%252900017-0%26rft_id%253Dinfo%253Apmid%252F12547424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S1367-5931(02)00017-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12547424&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000180868900006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference in text" id="ref-64">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.64">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walt</span>, 
          <span class="cit-name-given-names">Stéfan van der</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Chris Colbert</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Gaël</span>
          <span class="cit-name-surname">Varoquaux</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. “
         <span class="cit-article-title">The NumPy Array: A Structure for Efficient Numerical Computation</span>.” 
         <abbr class="cit-jnl-abbrev">Computing in Science &amp;; Engineering</abbr>
         <span class="cit-vol">13</span> (
         <span class="cit-issue">2</span>): 
         <span class="cit-fpage">22</span>–
         <span class="cit-lpage">30</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DComputing%2Bin%2BScience%2B%2526%2BEngineering%26rft.volume%253D13%26rft.spage%253D22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference in text" id="ref-65">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.65" data-doi="10.4161/mabs.1.3.8035">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Xiaoling</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Tapan K.</span>
          <span class="cit-name-surname">Das</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Satish K.</span>
          <span class="cit-name-surname">Singh</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Sandeep</span>
          <span class="cit-name-surname">Kumar</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. “
         <span class="cit-article-title">Potential Aggregation Prone Regions in Biotherapeutics: A Survey of Commercial Monoclonal Antibodies</span>.” 
         <abbr class="cit-jnl-abbrev">mAbs</abbr>
         <span class="cit-vol">1</span> (
         <span class="cit-issue">3</span>): 
         <span class="cit-fpage">254</span>–
         <span class="cit-lpage">67</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DmAbs%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D254%26rft.epage%253D267%26rft.atitle%253DPotential%2Baggregation%2Bprone%2Bregions%2Bin%2Bbiotherapeutics%253A%2BA%2Bsurvey%2Bof%2Bcommercial%2Bmonoclonal%2Bantibodies.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fmabs.1.3.8035%26rft_id%253Dinfo%253Apmid%252F20065649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.4161/mabs.1.3.8035&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20065649&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000271898000008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference in text" id="ref-66">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.66">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Warwicker</span>, 
          <span class="cit-name-given-names">Jim</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Spyros</span>
          <span class="cit-name-surname">Charonis</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Robin A.</span>
          <span class="cit-name-surname">Curtis</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. “
         <span class="cit-article-title">Lysine and Arginine Content of Proteins: Computational Analysis Suggests a New Tool for Solubility Design</span>.” 
         <abbr class="cit-jnl-abbrev">Molecular Pharmaceutics</abbr>
         <span class="cit-vol">11</span> (
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">294</span>–
         <span class="cit-lpage">303</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BPharmaceutics%26rft.volume%253D11%26rft.spage%253D294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference in text" id="ref-67">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.67">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waskom</span>, 
          <span class="cit-name-given-names">Michael</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Olga</span>
          <span class="cit-name-surname">Botvinnik</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Paul</span>
          <span class="cit-name-surname">Hobson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">John B.</span>
          <span class="cit-name-surname">Cole</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yaroslav</span>
          <span class="cit-name-surname">Halchenko</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Stephan</span>
          <span class="cit-name-surname">Hoyer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Alistair</span>
          <span class="cit-name-surname">Miles</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2014</span>. “
         <span class="cit-article-title">Seaborn: v0.5.0</span> (
         <span class="cit-month">November</span> 2014),” November. 
         <a href="https://doi.org/10.5281/zenodo.12710">https://doi.org/10.5281/zenodo.12710</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference in text" id="ref-68">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.68" data-doi="10.1038/nbt0591-443">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wilkinson</span>, 
          <span class="cit-name-given-names">D. L.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. G.</span>
          <span class="cit-name-surname">Harrison</span>
         </span>. 
         <span class="cit-pub-date">1991</span>. “
         <span class="cit-article-title">Predicting the Solubility of Recombinant Proteins in Escherichia Coli</span>.” 
         <abbr class="cit-jnl-abbrev">Bio/technology</abbr>
         <span class="cit-vol">9</span> (
         <span class="cit-issue">5</span>): 
         <span class="cit-fpage">443</span>–
         <span class="cit-lpage">48</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBio%252Ftechnology%26rft.volume%253D9%26rft.spage%253D443%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0591-443%26rft_id%253Dinfo%253Apmid%252F1367308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt0591-443&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=1367308&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1993MD87400001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference in text" id="ref-69">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.69">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Fan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Su</span>
          <span class="cit-name-surname">Zhao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bin</span>
          <span class="cit-name-surname">Yu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yan-Mei</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Wen</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yi</span>
          <span class="cit-name-surname">Hu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Zhi-Gang</span>
          <span class="cit-name-surname">Song</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2020</span>. “
         <span class="cit-article-title">Complete Genome Characterisation of a Novel Coronavirus Associated with Severe Human Respiratory Disease in Wuhan, China</span>.” 
         <span class="cit-source">bioRxiv</span>, 
         <a href="https://doi.org/10.1101/2020.01.24.919183">https://doi.org/10.1101/2020.01.24.919183</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference in text" id="ref-70">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.70" data-doi="10.1073/pnas.1901979116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Zachary</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. B.</span>
          <span class="cit-name-surname">Jennifer Kan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Russell D.</span>
          <span class="cit-name-surname">Lewis</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Bruce J.</span>
          <span class="cit-name-surname">Wittmann</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Frances H.</span>
          <span class="cit-name-surname">Arnold</span>
         </span>. 
         <span class="cit-pub-date">2019</span>. “
         <span class="cit-article-title">Machine Learning-Assisted Directed Protein Evolution with Combinatorial Libraries</span>.” 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>
         <span class="cit-vol">116</span> (
         <span class="cit-issue">18</span>): 
         <span class="cit-fpage">8852</span>–
         <span class="cit-lpage">58</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1901979116%26rft_id%253Dinfo%253Apmid%252F30979809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE2LzE4Lzg4NTIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNi8yMDIwLjAyLjE1Ljk1MTAxMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference in text" id="ref-71">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.71" data-doi="10.1093/bioinformatics/btv042">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">Nan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Dong-Sheng</span>
          <span class="cit-name-surname">Cao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Min-Feng</span>
          <span class="cit-name-surname">Zhu</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Qing-Song</span>
          <span class="cit-name-surname">Xu</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. “
         <span class="cit-article-title">protr/ProtrWeb: R Package and Web Server for Generating Various Numerical Representation Schemes of Protein Sequences</span>.” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">31</span> (
         <span class="cit-issue">11</span>): 
         <span class="cit-fpage">1857</span>–
         <span class="cit-lpage">59</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv042%26rft_id%253Dinfo%253Apmid%252F25619996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv042&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25619996&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference in text" id="ref-72">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.72" data-doi="10.1016/j.jsb.2010.07.011">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">Rong</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Stephen</span>
          <span class="cit-name-surname">Anderson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">James</span>
          <span class="cit-name-surname">Aramini</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Rachel</span>
          <span class="cit-name-surname">Belote</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">William A.</span>
          <span class="cit-name-surname">Buchwald</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Colleen</span>
          <span class="cit-name-surname">Ciccosanti</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Ken</span>
          <span class="cit-name-surname">Conover</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2010</span>. “
         <span class="cit-article-title">The High-Throughput Protein Sample Production Platform of the Northeast Structural Genomics Consortium</span>.” 
         <abbr class="cit-jnl-abbrev">Journal of Structural Biology</abbr>
         <span class="cit-vol">172</span> (
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">21</span>–
         <span class="cit-lpage">33</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DXiao%26rft.auinit1%253DR.%26rft.volume%253D172%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D33%26rft.atitle%253DThe%2Bhigh-throughput%2Bprotein%2Bsample%2Bproduction%2Bplatform%2Bof%2Bthe%2BNortheast%2BStructural%2BGenomics%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2010.07.011%26rft_id%253Dinfo%253Apmid%252F20688167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jsb.2010.07.011&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20688167&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference in text" id="ref-73">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.73">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">Kevin K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Zachary</span>
          <span class="cit-name-surname">Wu</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Frances H.</span>
          <span class="cit-name-surname">Arnold</span>
         </span>. 
         <span class="cit-pub-date">2019</span>. “
         <span class="cit-article-title">Machine-Learning-Guided Directed Evolution for Protein Engineering</span>.” 
         <abbr class="cit-jnl-abbrev">Nature Methods</abbr>
         <span class="cit-vol">16</span> (
         <span class="cit-issue">8</span>): 
         <span class="cit-fpage">687</span>–
         <span class="cit-lpage">94</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D16%26rft.spage%253D687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference in text" id="ref-74">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.15.951012v2.74">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yin</span>, 
          <span class="cit-name-given-names">Hui</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Yi-Zhou</span>
          <span class="cit-name-surname">Li</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Meng-Long</span>
          <span class="cit-name-surname">Li</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. “
         <span class="cit-article-title">On the Relation between Residue Flexibility and Residue Interactions in Proteins</span>.” 
         <abbr class="cit-jnl-abbrev">Protein and Peptide Letters</abbr>
         <span class="cit-vol">18</span> (
         <span class="cit-issue">5</span>): 
         <span class="cit-fpage">450</span>–
         <span class="cit-lpage">56</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2Band%2Bpeptide%2Bletters%26rft.stitle%253DProtein%2BPept%2BLett%26rft.aulast%253DYin%26rft.auinit1%253DH.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D450%26rft.epage%253D456%26rft.atitle%253DOn%2Bthe%2Brelation%2Bbetween%2Bresidue%2Bflexibility%2Band%2Bresidue%2Binteractions%2Bin%2Bproteins.%26rft_id%253Dinfo%253Apmid%252F21171945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=21171945&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F26%2F2020.02.15.951012.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference in text" id="ref-75">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.15.951012v2.75">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>, 
          <span class="cit-name-given-names">Zheng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Timothy L.</span>
          <span class="cit-name-surname">Bailey</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Rohan D.</span>
          <span class="cit-name-surname">Teasdale</span>
         </span>. 
         <span class="cit-pub-date">2005</span>. “
         <span class="cit-article-title">Prediction of Protein B-Factor Profiles</span>.” 
         <span class="cit-source">Proteins: Structure, Function, and Bioinformatics</span>. 
         <a href="https://doi.org/10.1002/prot.20375">https://doi.org/10.1002/prot.20375</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
