<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/16/2020.01.12.902452.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.12.902452v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Programmable low-cost DNA-based platform for viral RNA detection" />
<meta name="DC.Identifier" content="10.1101/2020.01.12.902452" />
<meta name="DC.Date" content="2020-01-16" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Viral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ∼24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact." />
<meta name="DC.Contributor" content="Lifeng Zhou" />
<meta name="DC.Contributor" content="Arun Richard Chandrasekaran" />
<meta name="DC.Contributor" content="Jibin Abraham Punnoose" />
<meta name="DC.Contributor" content="Gaston Bonenfant" />
<meta name="DC.Contributor" content="Stephon Charles" />
<meta name="DC.Contributor" content="Oksana Levchenko" />
<meta name="DC.Contributor" content="Pheonah Badu" />
<meta name="DC.Contributor" content="Cassandra Cavaliere" />
<meta name="DC.Contributor" content="Cara T. Pager" />
<meta name="DC.Contributor" content="Ken Halvorsen" />
<meta name="article:published_time" content="2020-01-16" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Programmable low-cost DNA-based platform for viral RNA detection" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Viral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ∼24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.12.902452v1" />
<meta name="citation_id" content="2020.01.12.902452v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.12.902452v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.12.902452v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.12.902452v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.12.902452" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.12.902452" />
<meta name="citation_author" content="Lifeng Zhou" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author" content="Arun Richard Chandrasekaran" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6757-5464" />
<meta name="citation_author" content="Jibin Abraham Punnoose" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2367-6874" />
<meta name="citation_author" content="Gaston Bonenfant" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author_institution" content="Department of Biology, University at Albany, State University of New York" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0081-7470" />
<meta name="citation_author" content="Stephon Charles" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author_institution" content="Department of Biology, University at Albany, State University of New York" />
<meta name="citation_author" content="Oksana Levchenko" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author" content="Pheonah Badu" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author_institution" content="Department of Biology, University at Albany, State University of New York" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7406-4248" />
<meta name="citation_author" content="Cassandra Cavaliere" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author_institution" content="Department of Biology, University at Albany, State University of New York" />
<meta name="citation_author" content="Cara T. Pager" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author_institution" content="Department of Biology, University at Albany, State University of New York" />
<meta name="citation_author_email" content="ctpager@albany.edu" />
<meta name="citation_author_email" content="khalvorsen@albany.edu" />
<meta name="citation_author" content="Ken Halvorsen" />
<meta name="citation_author_institution" content="The RNA Institute, University at Albany, State University of New York" />
<meta name="citation_author_email" content="ctpager@albany.edu" />
<meta name="citation_author_email" content="khalvorsen@albany.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2578-1339" />
<meta name="citation_reference" content="Smith, K. F., Sax, D. F., Gaines, S. D., Guernier, V. &amp; Guégan, J.-F. Globalization of human infectious disease. Ecology 88, 1903–1910 (2007)." />
<meta name="citation_reference" content="Sanjuán, R., Nebot, M. R., Chirico, N., Mansky, L. M. &amp; Belshaw, R. Viral Mutation Rates. Journal of Virology 84, 9733–9748 (2010)." />
<meta name="citation_reference" content="Steinhauer, D. A. &amp; Holland, J. J. Rapid Evolution of RNA Viruses. Annual Review of Microbiology 41, 409–431 (1987)." />
<meta name="citation_reference" content="Marston, H. D., Folkers, G. K., Morens, D. M. &amp; Fauci, A. S. Emerging Viral Diseases: Confronting Threats with New Technologies. Science Translational Medicine 6, 253ps10–253ps10 (2014)." />
<meta name="citation_reference" content="Jerome, K. R. Lennette’s Laboratory Diagnosis of Viral Infections. (CRC Press, 2016)." />
<meta name="citation_reference" content="Corman, V. M. et al. Assay optimization for molecular detection of Zika virus. Bull World Health Organ 94, 880–892 (2016)." />
<meta name="citation_reference" content="Musso, D. &amp; Gubler, D. J. Zika Virus. Clinical Microbiology Reviews 29, 487–524 (2016)." />
<meta name="citation_reference" content="Clark, D. J., Tyson, J., Sails, A. D., Krishna, S. &amp; Staines, H. M. The current landscape of nucleic acid tests for filovirus detection. Journal of Clinical Virology 103, 27–36 (2018)." />
<meta name="citation_reference" content="James, K. Immunoserology of infectious diseases. Clinical Microbiology Reviews 3, 132–152 (1990)." />
<meta name="citation_reference" content="Petersen, L. R., Jamieson, D. J., Powers, A. M. &amp; Honein, M. A. Zika Virus. New England Journal of Medicine 374, 1552–1563 (2016)." />
<meta name="citation_reference" content="Müller, B. et al. How safe is safe: new human immunodeficiency virus Type 1 variants missed by nucleic acid testing. Transfusion 53, 2422–2430 (2013)." />
<meta name="citation_reference" content="Dejnirattisai, W. et al. Dengue virus sero-cross-reactivity drives antibody-dependent enhancement of infection with zika virus. Nature Immunology 17, 1102–1108 (2016)." />
<meta name="citation_reference" content="Woods, C. R. False-Positive Results for Immunoglobulin M Serologic Results: Explanations and Examples. J Pediatric Infect Dis Soc 2, 87–90 (2013)." />
<meta name="citation_reference" content="Lazarus, J. V. et al. Too many people with viral hepatitis are diagnosed late — with dire consequences. Nat Rev Gastroenterol Hepatol 16, 451–452 (2019)." />
<meta name="citation_reference" content="Wang, S. et al. Advances in addressing technical challenges of point-of-care diagnostics in resource-limited settings. Expert Review of Molecular Diagnostics 16, 449–459 (2016)." />
<meta name="citation_reference" content="Draz, M. S. &amp; Shafiee, H. Applications of gold nanoparticles in virus detection. Theranostics 8, 1985–2017 (2018)." />
<meta name="citation_reference" content="Balcioglu, M. et al. Rapid Visual Screening and Programmable Subtype Classification of Ebola Virus Biomarkers. Advanced Healthcare Materials 6, 1600739 (2017)." />
<meta name="citation_reference" content="Afsahi, S. et al. Novel graphene-based biosensor for early detection of Zika virus infection. Biosensors and Bioelectronics 100, 85–88 (2018)." />
<meta name="citation_reference" content="Kim, J. W., Kim, S., Jang, Y., Lim, K. &amp; Lee, W. H. Attomolar detection of virus by liquid coplanargate graphene transistor on plastic. Nanotechnology 30, 345502 (2019)." />
<meta name="citation_reference" content="Gootenberg, J. S. et al. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science eaam9321 (2017) doi: 10.1126/science.aam9321." />
<meta name="citation_reference" content="Pardee, K. et al. Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components. Cell 165, 1255–1266 (2016)." />
<meta name="citation_reference" content="Strich, J. R. &amp; Chertow, D. S. CRISPR-Cas Biology and Its Application to Infectious Diseases. Journal of Clinical Microbiology 57, (2019)." />
<meta name="citation_reference" content="Halvorsen, K., Schaak, D. &amp; Wong, W. P. Nanoengineering a single-molecule mechanical switch using DNA self-assembly. Nanotechnology 22, 494005 (2011)." />
<meta name="citation_reference" content="Koussa, M. A., Halvorsen, K., Ward, A. &amp; Wong, W. P. DNA nanoswitches: a quantitative platform for gel-based biomolecular interaction analysis. Nature Methods 12, 123 (2015)." />
<meta name="citation_reference" content="Barrett, A. D. T. Current status of Zika vaccine development: Zika vaccines advance into clinical evaluation. npj Vaccines 3, 24 (2018)." />
<meta name="citation_reference" content="Chandrasekaran, A. R., Zavala, J. &amp; Halvorsen, K. Programmable DNA Nanoswitches for Detection of Nucleic Acid Sequences. ACS Sens. 1, 120–123 (2016)." />
<meta name="citation_reference" content="Chandrasekaran, A. R. et al. Cellular microRNA detection with miRacles: microRNA-activated conditional looping of engineered switches. Science Advances 5, eaau9443 (2019)." />
<meta name="citation_reference" content="Lanciotti, R. S. et al. Genetic and Serologic Properties of Zika Virus Associated with an Epidemic, Yap State, Micronesia, 2007. Emerg Infect Dis 14, 1232–1239 (2008)." />
<meta name="citation_reference" content="Halvorsen, K., Kizer, M. E., Wang, X., Chandrasekaran, A. R. &amp; Basanta-Sanchez, M. Shear Dependent LC Purification of an Engineered DNA Nanoswitch and Implications for DNA Origami. Anal. Chem. 89, 5673–5677 (2017)." />
<meta name="citation_reference" content="Shan, C. et al. An Infectious cDNA Clone of Zika Virus to Study Viral Virulence, Mosquito Transmission, and Antiviral Inhibitors. Cell Host &amp; Microbe 19, 891–900 (2016)." />
<meta name="citation_reference" content="Huber, R. G. et al. Structure mapping of dengue and Zika viruses reveals functional long-range interactions. Nat Commun 10, 1–13 (2019)." />
<meta name="citation_reference" content="Li, P. et al. Integrative Analysis of Zika Virus Genome RNA Structure Reveals Critical Determinants of Viral Infectivity. Cell Host &amp; Microbe 24, 875-886.e5 (2018)." />
<meta name="citation_reference" content="Gourinat, A.-C., O’Connor, O., Calvez, E., Goarant, C. &amp; Dupont-Rouzeyrol, M. Detection of Zika virus in urine. Emerging Infect. Dis. 21, 84–86 (2015)." />
<meta name="citation_reference" content="Paixão, E. S., Teixeira, M. G. &amp; Rodrigues, L. C. Zika, chikungunya and dengue: the causes and threats of new and re-emerging arboviral diseases. BMJ Global Health 3, (2018)." />
<meta name="citation_reference" content="Ramos-Castañeda, J. et al. Dengue in Latin America: Systematic Review of Molecular Epidemiological Trends. PLoS Negl Trop Dis 11, (2017)." />
<meta name="citation_reference" content="Haddow, A. D. et al. Genetic Characterization of Zika Virus Strains: Geographic Expansion of the Asian Lineage. PLoS Negl Trop Dis 6, (2012)." />
<meta name="citation_reference" content="Faye, O. et al. Molecular Evolution of Zika Virus during Its Emergence in the 20th Century. PLoS Negl Trop Dis 8, (2014)." />
<meta name="citation_reference" content="Bonenfant, G. et al. Zika Virus Subverts Stress Granules to Promote and Restrict Viral Gene Expression. Journal of Virology (2019) doi: 10.1128/JVI.00520-19." />
<meta name="citation_reference" content="Paz-Bailey, G. et al. Persistence of Zika Virus in Body Fluids — Final Report. New England Journal of Medicine 379, 1234–1243 (2018)." />
<meta name="citation_reference" content="Hansen, C. H., Yang, D., Koussa, M. A. &amp; Wong, W. P. Nanoswitch-linked immunosorbent assay (NLISA) for fast, sensitive, and specific protein detection. PNAS 201708148 (2017) doi: 10.1073/pnas.1708148114." />
<meta name="citation_reference" content="Gabrielle, M. E. et al. Nucleic acid sequence-based amplification (NASBA) for the identification of mycobacteria. Microbiology, 139, 2423–2429 (1993)." />
<meta name="citation_reference" content="Compton, J. Nucleic acid sequence-based amplification. Nature 350, 91–92 (1991)." />
<meta name="citation_reference" content="Seeman, N. C. DNA in a material world. Nature 421, 427–431 (2003)." />
<meta name="citation_reference" content="Hong, F., Zhang, F., Liu, Y. &amp; Yan, H. DNA Origami: Scaffolds for Creating Higher Order Structures. Chem. Rev. 117, 12584–12640 (2017)." />
<meta name="citation_reference" content="Xiao, M. et al. Rationally Engineered Nucleic Acid Architectures for Biosensing Applications. Chem. Rev. (2019) doi: 10.1021/acs.chemrev.9b00121." />
<meta name="citation_reference" content="Zhou, L., Marras, A. E., Huang, C.-M., Castro, C. E. &amp; Su, H.-J. Paper Origami-Inspired Design and Actuation of DNA Nanomachines with Complex Motions. Small 14, e1802580 (2018)." />
<meta name="citation_reference" content="Kuzuya, A., Sakai, Y., Yamazaki, T., Xu, Y. &amp; Komiyama, M. Nanomechanical DNA origami ‘single-molecule beacons’ directly imaged by atomic force microscopy. Nature Communications 2, 449 (2011)." />
<meta name="citation_reference" content="Funck, T., Nicoli, F., Kuzyk, A. &amp; Liedl, T. Sensing Picomolar Concentrations of RNA Using Switchable Plasmonic Chirality. Angewandte Chemie 130, 13683–13686 (2018)." />
<meta name="citation_reference" content="Douglas, S. M., Bachelet, I. &amp; Church, G. M. A Logic-Gated Nanorobot for Targeted Transport of Molecular Payloads. Science 335, 831–834 (2012)." />
<meta name="citation_reference" content="Modi, S. et al. A DNA nanomachine that maps spatial and temporal pH changes inside living cells. Nature Nanotechnology 4, 325 (2009)." />
<meta name="citation_reference" content="Jungmann, R. et al. Multiplexed 3D cellular super-resolution imaging with DNA-PAINT and Exchange-PAINT. Nature methods 11, 313–318 (2014)." />
<meta name="citation_reference" content="Berardi, M. J., Shih, W. M., Harrison, S. C. &amp; Chou, J. J. Mitochondrial uncoupling protein 2 structure determined by NMR molecular fragment searching. Nature 476, 109–113 (2011)." />
<meta name="citation_reference" content="Martin, T. G. et al. Design of a molecular support for cryo-EM structure determination. PNAS 113, E7456–E7463 (2016)." />
<meta name="citation_reference" content="Leung, K., Chakraborty, K., Saminathan, A. &amp; Krishnan, Y. A DNA nanomachine chemically resolves lysosomes in live cells. Nature Nanotechnology 1 (2018) doi: 10.1038/s41565-018-0318-5." />
<meta name="citation_reference" content="Modi, S., Nizak, C., Surana, S., Halder, S. &amp; Krishnan, Y. Two DNA nanomachines map pH changes along intersecting endocytic pathways inside the same cell. Nature Nanotech 8, 459–467 (2013)." />
<meta name="citation_reference" content="Chakraborty, K., Leung, K. &amp; Krishnan, Y. High lumenal chloride in the lysosome is critical for lysosome function. eLife 6, e28862 (2017)." />
<meta name="citation_reference" content="Gootenberg, J. S. et al. Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6. Science 360, 439–444 (2018)." />
<meta name="citation_reference" content="Theel, E. S. &amp; Hata, D. J. Diagnostic Testing for Zika Virus: a Postoutbreak Update. Journal of Clinical Microbiology 56, (2018)." />
<meta name="citation_reference" content="Whitesides, G. The frugal way: The promise of cost-conscious science. The Economist: The World in 2012 154 (2011)." />
<meta name="citation_reference" content="Reardon, S. Frugal science gets DIY diagnostics to world’s poorest. New Scientist 219, 20–21 (2013)." />
<meta name="citation_reference" content="Bhamla, M. S. et al. Hand-powered ultralow-cost paper centrifuge. Nature Biomedical Engineering 1, 0009 (2017)." />
<meta name="citation_reference" content="Wong, G., Wong, I., Chan, K., Hsieh, Y. &amp; Wong, S. A Rapid and Low-Cost PCR Thermal Cycler for Low Resource Settings. PLOS ONE 10, e0131701 (2015)." />
<meta name="citation_reference" content="Kinney, R. M. et al. Construction of Infectious cDNA Clones for Dengue 2 Virus: Strain 16681 and Its Attenuated Vaccine Derivative, Strain PDK-53. Virology 230, 300–308 (1997)." />
<meta name="citation_reference" content="Schneider, C. A., Rasband, W. S. &amp; Eliceiri, K. W. NIH Image to ImageJ: 25 years of image analysis. Nat Methods 9, 671–675 (2012)." />
<meta name="twitter:title" content="Programmable low-cost DNA-based platform for viral RNA detection" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Viral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ∼24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact." />
<meta name="og-title" property="og:title" content="Programmable low-cost DNA-based platform for viral RNA detection" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.12.902452v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Viral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ∼24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-16" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.12.902452v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.12.902452v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.12.902452v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.12.902452v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1107550" />
    <title>Programmable low-cost DNA-based platform for viral RNA detection | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"F9H8JtYKTvHDetZcWtRYLoeXCxQIeBtPKNhDMFCyUiE","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1107550","apath":"\/biorxiv\/early\/2020\/01\/16\/2020.01.12.902452.atom","pisa":"biorxiv;2020.01.12.902452v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.12.902452v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.12.902452v1","title":"Programmable low-cost DNA-based platform for viral RNA detection","identifier":"node\/1107550"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xpILljWbfUgSCvTu+CqExF3OPnixd95QqJHzfRRxCFtwSUyk9zqReOE3h8VuhbxW7\\\\\\\/dvqYIuFnLL9cB1fyUA0ZXbKmJVPdCkWjvo7GRCrT8J7SsDWdDNaxseshHF2N9ojSXg96jz7FLNd4EskH5LsLFQF4uKEtx7Rn2ASKt5Le+gVEtXqGpzfcZIjhLRxqWPQAwN4yp0Y\\\\\\\/GgNZdQzZP74rLGzjlypCqME5r98HC0ODgqpIoyq2d41vsBtogl4gFeguyZZNFm7CMxEq+\\\\\\\/kX\\\\\\\/btp3ceskzeNGxP7XeTOfzZBjfSPzD03q8gW\\\\\\\/cbGpKiIizInQBBm4wKTw62H+0IKTTdGyT\\\\\\\/Fe7nAYBgG2WOWUaw+oo65lavQ3clfN1DQlxCg8Vm4ircoZoVq0oNMAmNzGRmn+iOgY7vlI6iPyq6Kx2fBN53jzqv0ytNRPz9RlIhDitHULPv6mDcE9d8CJdHfLqvVL7cRLZAsqBgO6btekmi7\\\\\\\/VRBmZ456JhaFNNEpK7shBxNZo+mG3O\\\\\\\/yAVejpStqfs+TpCHlNgACpSOR9moR3fX0hMeDblXmUqfEAM12FHhbdizuWfdLZEp50EGUqf1vnKchXU5f7mpWPdnfOW+OmzsPFq5WiI9p1G5mSyCtJOf8NfEXL7l3K98n6BUoz8BYZ1J8vd1OMVcP80O\\\\\\\/VTPjH\\\\\\\/k7yNRM6cdXDfas38ZdVM5Y9hLSHZVT29gkKlz+Elyiri1oVrQaBzIhnjA4U4HD1rYDbg3C7+vvHR8KCDgI+y7ViB+qubRC3lVwaBGxyrpMO0oOmbPveFHr+CLT+dA2C8J6a0QQsA\\\\\\\/1NkJ9t0X5vauDZJbXCo6FxWbicSEYV\\\\\\\/Oezv8C3h+IJ4xxlS9MBihLKUqq8w9XvIB94bRe9gyB+3Jo0NCVdJFoePHYPXOFaErXMdlaNbcXzPTQqw2mnJYYtX4Y1617x469Nim3odQgFSUTjznoJoOQyuuIDCVFNUuhq5uzXVEQKoXVCQTvROEzTexGF6um1N3muhhCkAWF6we6u0i7S\\\\\\\/qHMtp0IKTGcrNeFh\\\\\\\/NgfWTrSaD6AVa4iSE17xYpX88sJlFZGEgDBYY2RzjxfHkn35b\\\\\\\/WWRqFuoZ1djxt52ayNrzzTZdpv5DFDmltMP123yUz5OVrpOPxVCLp1hDtj7pU7RSPjAWnLPx0EOiYy2l4XT3CbjkhkyrcilTQ7PqLCCRpqp93+Q2lL8YCepXEF7MLcaSWIfhKFdvLaKqaeock5yVaWnhSsqmH20i0IbOiWRgV0cuS1qsCPoMgP8ekbKENWq1UW\\\\\\\/86DTjA9C8Jj34Jh38yi1RIBeWaxw2tLI1LkPVBL6ocLdRQR9i9M1k\\\u0022,\\\u0022iv\\\u0022:\\\u0022Eda3YMfrRS4OtBMC9mTQzQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022a56749272c828ba67842bc51b9d014659330fe6ae5580d46deb826dd43f22308\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.12.902452v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1107550 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.12.902452v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Vxri04KpEAZjdHkiB_c-KMCEvDQJiNEodz9S-gOLui8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107550" id="node1107550" data-pisa="biorxiv;2020.01.12.902452v1" data-pisa-master="biorxiv;2020.01.12.902452" data-apath="/biorxiv/early/2020/01/16/2020.01.12.902452.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Programmable low-cost DNA-based platform for viral RNA detection</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Lifeng</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6757-5464" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Arun Richard</span> <span class="nlm-surname">Chandrasekaran</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2367-6874" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jibin Abraham</span> <span class="nlm-surname">Punnoose</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0081-7470" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gaston</span> <span class="nlm-surname">Bonenfant</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Stephon</span> <span class="nlm-surname">Charles</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Oksana</span> <span class="nlm-surname">Levchenko</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-7406-4248" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Pheonah</span> <span class="nlm-surname">Badu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Cassandra</span> <span class="nlm-surname">Cavaliere</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Cara T.</span> <span class="nlm-surname">Pager</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-2578-1339" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ken</span> <span class="nlm-surname">Halvorsen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.12.902452 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1107550" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lifeng Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lifeng%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALifeng%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Arun Richard Chandrasekaran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arun%2BRichard%2BChandrasekaran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chandrasekaran%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArun%2BRichard%2BChandrasekaran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6757-5464" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arun Richard Chandrasekaran</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jibin Abraham Punnoose </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jibin%2BAbraham%2BPunnoose%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Punnoose%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJibin%2BAbraham%2BPunnoose%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2367-6874" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jibin Abraham Punnoose</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gaston Bonenfant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gaston%2BBonenfant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonenfant%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGaston%2BBonenfant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0081-7470" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gaston Bonenfant</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Stephon Charles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephon%2BCharles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Charles%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephon%2BCharles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Oksana Levchenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oksana%2BLevchenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Levchenko%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOksana%2BLevchenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Pheonah Badu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pheonah%2BBadu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Badu%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APheonah%2BBadu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7406-4248" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pheonah Badu</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Cassandra Cavaliere </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cassandra%2BCavaliere%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavaliere%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACassandra%2BCavaliere%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cara T. Pager </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cara%2BT.%2BPager%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pager%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACara%2BT.%2BPager%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ctpager@albany.edu" class="" data-icon-position="" data-hide-link-title="0">ctpager@albany.edu</a>
<a href="mailto:khalvorsen@albany.edu" class="" data-icon-position="" data-hide-link-title="0">khalvorsen@albany.edu</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ken Halvorsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>The RNA Institute, University at Albany, State University of New York</span>, Albany, NY 12222, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ken%2BHalvorsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halvorsen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKen%2BHalvorsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2578-1339" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ken Halvorsen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ctpager@albany.edu" class="" data-icon-position="" data-hide-link-title="0">ctpager@albany.edu</a>
<a href="mailto:khalvorsen@albany.edu" class="" data-icon-position="" data-hide-link-title="0">khalvorsen@albany.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.12.902452v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1107550" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1107550/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.12.902452v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1107550" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1107550/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.12.902452v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1107550" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1107550/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.12.902452v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1107550" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1107550/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.12.902452v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1107550" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1107550/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.12.902452v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1107550" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1107550/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Viral detection is critical for controlling disease spread and progression. Recent emerging threats including the Zika and Ebola virus outbreaks highlight the cost and difficulty in responding rapidly. In low-resource areas, a key obstacle is quick and accurate detection of viruses near the point of care. To address these challenges, we develop a platform for low-cost and rapid detection of viral RNA with DNA nanoswitches designed to mechanically reconfigure in response to specific viruses. Using Zika virus as a model system, we show non-enzymatic detection of viral RNA to the attomole level, with selective and multiplexed detection between related viruses and viral strains. For clinical-level sensitivity in biological fluids, we paired the assay with a sample preparation step using either RNA extraction or isothermal pre-amplification. Our assay can be performed with minimal or no lab infrastructure, and is readily adaptable (with ∼24-hour development time) to detect other viruses. Given this versatility, we expect that further development and field implementation will improve our ability to detect emergent viral threats and ultimately limit their impact.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Newly emerging or re-emerging viruses pose significant challenges to health care systems, particularly as globalization has contributed to the rampant spread of these viruses.<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> RNA viruses are frequently the cause of sweeping outbreaks due to the naturally high mutation rates during replication that lead to rapid viral evolution.<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup> Examples of this include the annual influenza outbreak, as well as more recent threats from Ebola virus, henipavirus, severe acute respiratory syndrome (SARS)-associated coronavirus, and Zika virus (ZIKV). Technological advancements in structural biology and genomics have been important for identifying viruses, and for advancing fundamental viral research and antiviral therapeutics.<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> However, clinical methods for robust, low-cost and rapid detection of viral infections optimized for use in low-resource areas and point-of-care (POC) mode remain a major challenge for emergent viruses.</p><p id="p-5">Detection of RNA viruses in the clinical setting is typically performed using either immunological detection based on enzyme-linked immunosorbent assay (ELISA) to detect IgM antibodies or Nucleic acid testing (NAT) based on a reverse transcription polymerase chain reaction (RT-PCR) assay to detect viral RNA.<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup> Diagnosing RNA viruses is made challenging by several factors including a limited time window for detection, low or varying viral load, cross-reactive IgM antibodies, and laboratory resources. The detection time windows can vary widely from as short as several days to as long as several months,<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup> and molecular detection techniques are usually most reliable if performed within the first two weeks of the disease.<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup> Depending on the timing of testing relative to infection, even highly sensitive NAT assays may still produce false negative or positive results.<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup> On the other hand, results from IgM serology tests often cannot distinguish related viruses or different strains of the same virus due to cross-reactivity of IgM antibodies, thus leading to false positive results.<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> These detection challenges are further exacerbated when outbreaks occur in low resource settings where infrastructure for these lab-intensive tests can be lacking, accelerating the spread of disease.<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup></p><p id="p-6">In response to some of these challenges, new techniques are being developed to detect emerging viruses. Among these are methods that adopt nanoparticles,<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup> graphene-based biosensors,<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup> and CRISPR-based methods,<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>–<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup> to name a few. Many of these proposed strategies, although based on cutting-edge technology, require multiple reactions or signal transformation steps that may make them less suitable for a resource-limited POC setting. Here, we addressed these biosensing challenges by developing an assay that uses programmable DNA nanoswitches<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> for detection of viral RNA at clinically relevant levels. We validate our strategy using ZIKV as a proof-of-concept system due to its high global health relevance and continued threat due to its re-emerging mosquito-borne nature. Although ZIKV is typically associated with mild symptoms, it has been linked to devastating birth defects associated with intrauterine infections and the appearance of Guillian-Barré syndrome in adults.<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup> Moreover, despite significant advances in understanding the molecular biology of ZIKV, there is still a lack of antiviral drugs and vaccines, making robust detection of ZIKV vital to controlling the spread of the disease and implementing early treatments.<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup></p><p id="p-7">Our strategy for detecting the presence of viral RNA is based on using DNA nanoswitches that have been designed to undergo a conformational change (from linear to looped) upon binding a target viral RNA sequence. Using ZIKV as an example, the presence of the viral RNA would be indicated by shifted migration of the looped nanoswitch by gel electrophoresis (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a></strong>). Importantly, the system is designed to use common nucleic acid staining of the nanoswitch itself that can intercalate thousands of dye molecules to provide an inherently strong signal. Previously, we demonstrated sensitive and specific detection of DNA oligonucleotides<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup> and microRNAs (∼22 nucleotides long)<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> using this approach. Applied here to viral RNA detection, we solved the challenges of detecting a long viral RNA (&gt;10,000 nucleotides), improving the signal for long RNAs with a new signal multiplication strategy, and developing workflows for measuring viral loads in biological and mock clinical samples. We showed how multiplexing can be used to detect multiple viruses simultaneously from a single sample and demonstrated high specificity even between closely-related strains of Zika. This work enables direct detection of ZIKV RNA without amplification and paves the way toward a low-cost point-of-care assay for detection of ZIKV and other viruses.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of viral RNA using DNA nanoswitches. (A) Schematic of the DNA nanoswitch and its detection of a viral RNA sequence. (B) Schematic of the fragmentation of viral RNA and subsequent detection by the DNA nanoswitch. (C) Fragmentation analysis of ZIKV RNA that was fragmented at 94 &#xB0;C for 1, 3, 6, and 9 minutes respectively. (D) Proof-of-concept showing detection of a target region chosen from the literature.28 (E) Schematic of the design of multiple nanoswitches for detection with the signal multiplication strategy. (F) Validation of the signal multiplication strategy: the detection signal was increased for a fixed pool of DNA targets when using multiple targeting nanoswitches. (G) Detection sensitivity of the pooled nanoswitches for ZIKV RNA. Error bars represent standard deviation from triplicate experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-784549621" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Detection of viral RNA using DNA nanoswitches.&lt;/span&gt; (&lt;strong&gt;A&lt;/strong&gt;) Schematic of the DNA nanoswitch and its detection of a viral RNA sequence. (&lt;strong&gt;B&lt;/strong&gt;) Schematic of the fragmentation of viral RNA and subsequent detection by the DNA nanoswitch. &lt;strong&gt;(C)&lt;/strong&gt; Fragmentation analysis of ZIKV RNA that was fragmented at 94 &#xB0;C for 1, 3, 6, and 9 minutes respectively. (&lt;strong&gt;D&lt;/strong&gt;) Proof-of-concept showing detection of a target region chosen from the literature.&lt;sup&gt;28&lt;/sup&gt; (&lt;strong&gt;E&lt;/strong&gt;) Schematic of the design of multiple nanoswitches for detection with the signal multiplication strategy. (&lt;strong&gt;F&lt;/strong&gt;) Validation of the signal multiplication strategy: the detection signal was increased for a fixed pool of DNA targets when using multiple targeting nanoswitches. (&lt;strong&gt;G&lt;/strong&gt;) Detection sensitivity of the pooled nanoswitches for ZIKV RNA. Error bars represent standard deviation from triplicate experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F1.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F1.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">Detection of viral RNA using DNA nanoswitches.</span><p id="p-8" class="first-child">(<strong>A</strong>) Schematic of the DNA nanoswitch and its detection of a viral RNA sequence. (<strong>B</strong>) Schematic of the fragmentation of viral RNA and subsequent detection by the DNA nanoswitch. <strong>(C)</strong> Fragmentation analysis of ZIKV RNA that was fragmented at 94 °C for 1, 3, 6, and 9 minutes respectively. (<strong>D</strong>) Proof-of-concept showing detection of a target region chosen from the literature.<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> (<strong>E</strong>) Schematic of the design of multiple nanoswitches for detection with the signal multiplication strategy. (<strong>F</strong>) Validation of the signal multiplication strategy: the detection signal was increased for a fixed pool of DNA targets when using multiple targeting nanoswitches. (<strong>G</strong>) Detection sensitivity of the pooled nanoswitches for ZIKV RNA. Error bars represent standard deviation from triplicate experiments.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-9">As a first proof of concept for detecting ZIKV, we designed DNA nanoswitches to target an already validated sequence in the ZIKV genome that has been used to bind primers in qPCR<sup><a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a></sup> (note: all oligo sequences are specified in <strong>Tables S1 to S9</strong>). We made the DNA nanoswitches by hybridizing single-stranded DNA (ssDNA) oligos to linearized single-stranded M13mp18 (M13) genomic DNA in a thermal annealing ramp for 1 hour<sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup> and then purified them by high-performance liquid chromatography (HPLC).<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup> For our initial detection target, we <em>in vitro</em> transcribed RNA from the pFLZIKV infectious plasmid containing the full length genome of the Cambodia ZIKV isolate (FSS13025) (<strong>Fig. S1)</strong>.<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup> Previous results have shown robust nanoswitch detection of small DNA and RNA sequences (20-30 nucleotides), however, the strong secondaty structures present in long viral RNA was expected to interfere with our detection.<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> To overcome this, we used a chemical fragmentation method to segment the RNA into small pieces that are mostly shorter than 200 nt (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B-C</a> and S2</strong>). By incubating with our nanoswitch in an annealing temperature ramp, we showed successful detection of the fragmented viral RNAs by gel electrophoresis, thus validating our approach (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1D</a></strong>).</p><p id="p-10">Having shown successful detection of ZIKV RNA using a single target sequence, we recognized that we could exploit the large genome size (∼11,000 nucleotides) to increase our detection signal through multiple targets. Once the long viral RNA is fragmented, the number of available target sequences increases dramatically. Since our detection signal is proportional to the number of looped nanoswitches, a nanoswitch mixture for different target sequences within the viral genome is expected to provide an increased signal. To test this, we developed an algorithm for choosing multiple sequence regions in the viral genome that can be targeted by the nanoswitches (<strong>Note S1</strong>). First, we chose the default target length as 30 nucleotides based on results from screening nanoswitches with different detection arm lengths (<strong>Fig. S3</strong>). Then, the algorithm selectively excluded target sequences that could form stable secondary structures (<strong>Fig. S4</strong>) and cross-binding with nanoswitch backbone oligos (<strong>Fig. S5</strong>), and enforced GC content and uniqueness of sequences. Based on these criteria, we chose 18 target regions along the entire ZIKV RNA for testing and designed the nanoswitches. To facilitate use of our Matlab-based software, we have built a graphical user interface (<strong>Fig. S6</strong>) and made it freely available (<strong>File S1</strong>).</p><p id="p-11">We then validated quality and function of each nanoswitch in the panel of 18 nanoswitches. All nanoswitches performed well with a molar excess of positive DNA controls, although they showed more signal variation with fragmented ZIKV RNA (<strong>Fig. S7</strong>). We ranked the nanoswitches from strongest to weakest signal and made a series of equimolar nanoswitch mixtures. Using these mixtures, we validated our inherent signal multiplication strategy using a low concentration pool of equimolar DNA fragments to mimic the fragmented RNA. We observed that our detection signal increased steadily up to around 12 nanoswitches (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1E, F</a></strong>), and then plateaued above that value. This plateau was not unexpected considering that the largest mixtures added lower performing nanoswitches that may contribute less to the overall sample. Since there was no significant change in performance between 12 and 18, we continued with using the 18 nanoswitches mix for our follow up experiments.</p><p id="p-12">High sensitivity is one of the key requirements for viral detection. Clinical levels of ZIKV RNA in body fluids of infected patients are often in the femtomolar range,<sup><a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> making amplification a prerequisite for most detection approaches. Based on our earlier observation that DNA nanoswitches can detect microRNAs (∼22 nucleotides) at the sub-picomolar scale<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></sup> without amplification, we wanted to assess the sensitivity of our approach for ZIKV RNA detection. We reacted the DNA nanoswitch mixture with different amounts of fragmented RNA in a 12-hour annealing temperature ramp from 40 °C to 25 °C. The results showed visible detection for ZIKV RNA as low as 12.5 pg (∼3.5 attomole) (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1G</a> and S8</strong>) in a 10 µL reaction volume. Consistent with <strong><a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1F</a></strong>, the approach based on using a nanoswitch mix outperformed the highest performing nanoswitch used as a single agent, which had visible detection to about 50 pg (∼14 attomole) (<strong>Fig. S9</strong>).</p><p id="p-13">Another key requirement for a clinical viral detection assay is specificity. Since ZIKV and Dengue virus (DENV) have overlapping geographical distributions and clinical symptoms, infection with either virus may result in clinical misdiagnosis.<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup> Serological diagnostic assays are known to show antibody cross-reactivity between the two viruses, and DENV has some similarity to ZIKV in its envelope protein<sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup> and genome sequence.<sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a></sup> To test the specificity of our approach, we designed a similar panel of nanoswitches to detect DENV (<strong>Fig. S10</strong>). Using the pooled nanoswitches specific for ZIKV and DENV, we mixed each set with <em>in vitro</em> transcribed RNA from each virus and found perfect specificity, with each assay only detecting its correct target RNA <strong>(<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>).</strong> Using the programmability of the nanoswitch, we further demonstrated a multiplexed system for simultaneous detection of ZIKV and DENV. In this case we modified the DENV responsive nanoswitches to form a smaller loop size (<strong>Fig. S11</strong>), causing two distinct detection bands to migrate to different positions in the gel. Specifically, ZIKV RNA-nanoswitch complex migrated slower/higher in the gel, while the complex of DENV RNA and the nanoswitch migrated faster/lower in the gel (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a></strong>). Therefore, in a single reaction our nanoswitch showed differential and specific detection of ZIKV and DENV RNA. By programming different loop sizes for different targets, this assay can be expanded for up to five viral targets.<sup><a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a></sup></p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA nanoswitches specifically and differentially detect RNA from two different flaviviruses and between two highly similar ZIKV isolates. (A) ZIKV nanoswitches specifically detect ZIKV RNA but not DENV RNA, and vice versa. (B) Multiplexed detection of ZIKV and DENV RNA. (C) Diagram for the culture and RNA extraction of ZIKV Cambodia and Uganda strains. (D) Specificity test of Cambodia and Uganda strains of ZIKV RNA. * denotes a band of contaminating cellular DNA following RNA isolation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-784549621" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;DNA nanoswitches specifically and differentially detect RNA from two different flaviviruses and between two highly similar ZIKV isolates.&lt;/span&gt; (&lt;strong&gt;A&lt;/strong&gt;) ZIKV nanoswitches specifically detect ZIKV RNA but not DENV RNA, and &lt;em&gt;vice versa&lt;/em&gt;. (&lt;strong&gt;B&lt;/strong&gt;) Multiplexed detection of ZIKV and DENV RNA. (&lt;strong&gt;C&lt;/strong&gt;) Diagram for the culture and RNA extraction of ZIKV Cambodia and Uganda strains. (&lt;strong&gt;D&lt;/strong&gt;) Specificity test of Cambodia and Uganda strains of ZIKV RNA. * denotes a band of contaminating cellular DNA following RNA isolation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F2.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F2.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <span class="caption-title">DNA nanoswitches specifically and differentially detect RNA from two different flaviviruses and between two highly similar ZIKV isolates.</span><p id="p-14" class="first-child">(<strong>A</strong>) ZIKV nanoswitches specifically detect ZIKV RNA but not DENV RNA, and <em>vice versa</em>. (<strong>B</strong>) Multiplexed detection of ZIKV and DENV RNA. (<strong>C</strong>) Diagram for the culture and RNA extraction of ZIKV Cambodia and Uganda strains. (<strong>D</strong>) Specificity test of Cambodia and Uganda strains of ZIKV RNA. * denotes a band of contaminating cellular DNA following RNA isolation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">In addition to possible misdiagnosis between different viruses, there is an additional challenge in determining the specific strain of a virus. For example, in Latin America four different DENV serotypes are known to be present and co-circulate, where misdiagnosis of the infecting strain can have significant implications for treatment options.<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup> Thus being able to accurately identify a circulating strain of virus broadly impacts medical care, surveillance and vector control.<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> ZIKV was first identified in Uganda in 1947 before spreading to Asia and the Americas, and ZIKV strains (classified within African or Asian lineage) share significant sequence homology.<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup> To investigate if our assay can distinguish between the Asian and African lineages, we tested our nanoswitches against two ZIKV strains which have a ∼89% sequence homology, namely the FSS13025 isolated from Cambodia and the MR766 strain isolated from Uganda. In designing the ZIKV strain-specific nanoswitches, we identified five target regions that each have a 5-6 nucleotide difference (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2C</a> and S12</strong>). To achieve better discernment of the detection signal, the nanoswitches for the Uganda strain were designed to form a smaller loop-size than those designed for Cambodia. Next, a human hepatocellular carcinoma cell line (Huh7) was infected with either the Cambodian or Ugandan ZIKV strain. Infected cells were processed to extract total RNA, which was then fragmented and incubated with nanoswitches to probe for viral RNA from either the ZIKV Cambodia or ZIKV Uganda infected cells. The results showed that our assay was able to discriminate between two strains of the same virus even with high genetic similarities (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2D</a>, S12</strong>).</p><p id="p-16">Further applying our technique to detect ZIKV RNA in biological samples, we either mock-infected or infected Huh7 cells with the Cambodia ZIKV strain at a multiplicity of infection of 1 and extracted RNA from the ZIKV infected cells at 1-, 2- and 3-days post-infection.<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup> The nanoswitch assay detected ZIKV viral RNA from the infected cells but not the mock infected cells (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figs. 3A, 3B</a> &amp; S13</strong>). Our detection result shows that the copies of ZIKV RNA within infected cells steadily increased upon the infection and plateaued at 2- and 3-days post-infection (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3C</a></strong>). These data demonstrate that our assay can detect ZIKV RNA in infected cell lines, and in contrast to typical RT-PCR assays without amplification of the viral RNA.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA nanoswitches directly detected ZIKV RNA from infected human liver cells (A) RNA isolated from mock-infected Huh7 cells at 1, 2, and 3 days post infection show no ZIKV detection. (B) RNA isolated from Zika-infected Huh7 cells at 1, 2, and 3 days post infection shows increasing ZIKV detection over time, with red arrows denoting detection bands. *denotes a band of contaminating cellular DNA following RNA extraction (C) Quantification of nanoswitch detection signal, with error bars representing standard deviation from triplicate experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-784549621" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;DNA nanoswitches directly detected ZIKV RNA from infected human liver cells&lt;/span&gt; (&lt;strong&gt;A&lt;/strong&gt;) RNA isolated from mock-infected Huh7 cells at 1, 2, and 3 days post infection show no ZIKV detection. (&lt;strong&gt;B&lt;/strong&gt;) RNA isolated from Zika-infected Huh7 cells at 1, 2, and 3 days post infection shows increasing ZIKV detection over time, with red arrows denoting detection bands. *denotes a band of contaminating cellular DNA following RNA extraction (&lt;strong&gt;C&lt;/strong&gt;) Quantification of nanoswitch detection signal, with error bars representing standard deviation from triplicate experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F3.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F3.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">DNA nanoswitches directly detected ZIKV RNA from infected human liver cells</span><p id="p-17" class="first-child">(<strong>A</strong>) RNA isolated from mock-infected Huh7 cells at 1, 2, and 3 days post infection show no ZIKV detection. (<strong>B</strong>) RNA isolated from Zika-infected Huh7 cells at 1, 2, and 3 days post infection shows increasing ZIKV detection over time, with red arrows denoting detection bands. *denotes a band of contaminating cellular DNA following RNA extraction (<strong>C</strong>) Quantification of nanoswitch detection signal, with error bars representing standard deviation from triplicate experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Moving toward clinical applications, we aimed to demonstrate detection of relevant levels of ZIKV RNA from biological fluids. ZIKV is present in the serum, urine, and other body fluids of infected patients.<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> The viral loads can vary dramatically between individuals, body fluid, and post infection time,<sup><a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a></sup> but are frequently in the subfemtomolar to femtomolar range, with ZIKV in human urine reported as high as 220 × 10<sup>6</sup> copies/ml (365 fM).<sup><a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a></sup> While our nanoswitch sensitivity for <em>in vitro</em> transcribed viral RNA in buffer approaches clinically relevant concentrations, detection from body fluids is further challenged by varying viral loads and by body fluids that can reduce the performance of the nanoswitches due to physiological conditions and nuclease activity.<sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup> To overcome these potential difficulties, we investigated two independent solutions: 1) adding a preprocessing step to extract RNA from body fluids such as urine, or 2) adding an isothermal pre-amplification step. In the first approach, we spiked a clinically-relevant amount of <em>in vitro</em> transcribed ZIKV RNA into human urine and processed viral RNA extraction using a commercial RNA extraction kit. We then mixed the extracted RNA with the nanoswitches and demonstrated non-enzymatic, clinical level detection of the RNA at 0.28 pM (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A-B</a> &amp; S14</strong>). In the second approach, we demonstrated that our detection can be coupled with other amplification approaches such as nucleic acid sequence-based amplification (NASBA).<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>,<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> NASBA combines multiple enzymes and primers to achieve RNA amplification in a one-pot isothermal reaction (<strong>Fig. S15A</strong>). First, we showed feasibility of the amplification of ZIKV RNA by NASBA in water, followed by nanoswitch detection (<strong>Fig. S15</strong>). To mimic clinical samples, we spiked infectious ZIKV particles into either PBS or 10% human urine at clinical-levels (1.49 fM to 0.03 fM). From these samples our assay detected ZIKV RNA in ∼5 hours (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4C-E</a> and S15</strong>). We went one step further and showed that our assay can be performed using a commercially available bufferless gel cartridge (ThermoFisher E-gel) and imaged on a small and potentially portable gel reader (<strong>Fig. S16</strong>). With the help of NASBA amplification, the detection ability of our method has about 10,000-fold increase, from ∼350 fM (<strong><a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1G</a></strong>) to 0.03 fM (<strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4D, E</a></strong>) and the detection time was reduced from ∼13 hours to ∼5 hours (<strong>Fig. S17</strong>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prior extraction or pre-amplification of target RNA facilitates detection of ZIKV RNA at clinically relevant levels in urine. (A) Schematic of DNA nanoswitch detection of ZIKV RNA extracted from urine by first isolating in vitro transcribed target RNA using a commercially available viral RNA extraction kit. (B) Positive identification of ZIKV RNA in spiked urine. (C) Procedure of detecting ZIKV RNA from virus particles in cell culture media spiked into PBS/urine based on NASBA. (D) Positive detection of ZIKV RNA from infectious virus in PBS. (E) Positive detection of ZIKV RNA from infectious virus in human urine samples. Error bars represent standard deviation from triplicate experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-784549621" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Prior extraction or pre-amplification of target RNA facilitates detection of ZIKV RNA at clinically relevant levels in urine.&lt;/span&gt; (&lt;strong&gt;A&lt;/strong&gt;) Schematic of DNA nanoswitch detection of ZIKV RNA extracted from urine by first isolating &lt;em&gt;in vitro&lt;/em&gt; transcribed target RNA using a commercially available viral RNA extraction kit. (B) Positive identification of ZIKV RNA in spiked urine. (C) Procedure of detecting ZIKV RNA from virus particles in cell culture media spiked into PBS/urine based on NASBA. (D) Positive detection of ZIKV RNA from infectious virus in PBS. (E) Positive detection of ZIKV RNA from infectious virus in human urine samples. Error bars represent standard deviation from triplicate experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F4.medium.gif" width="348" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F4.medium.gif" width="348" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.12.902452/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <span class="caption-title">Prior extraction or pre-amplification of target RNA facilitates detection of ZIKV RNA at clinically relevant levels in urine.</span><p id="p-19" class="first-child">(<strong>A</strong>) Schematic of DNA nanoswitch detection of ZIKV RNA extracted from urine by first isolating <em>in vitro</em> transcribed target RNA using a commercially available viral RNA extraction kit. (B) Positive identification of ZIKV RNA in spiked urine. (C) Procedure of detecting ZIKV RNA from virus particles in cell culture media spiked into PBS/urine based on NASBA. (D) Positive detection of ZIKV RNA from infectious virus in PBS. (E) Positive detection of ZIKV RNA from infectious virus in human urine samples. Error bars represent standard deviation from triplicate experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Taken together, we demonstrate that programmable DNA nanoswitches can be developed into a robust viral RNA detection platform. The platform has key advantages over existing methodologies in terms of selectivity and specificity, as shown in our experiments with ZIKV and closely related DENV, as well as two closely related ZIKV strains. Moreover, DNA nanoswitch viral RNA detection strategy has femtomolar detection limit without an RNA amplification step, and attomolar detection limit when used with amplification. These limits are within a clinically relevant range and therefore our DNA nanoswitch assay together with the bufferless gel cartridge presents a putative diagnostic assay for clinical detection of ZIKV in low resource areas without significant laboratory infrastructure.</p></div><div class="section" id="sec-3"><h2 class="">Discussion</h2><p id="p-21">The functionality of our DNA nanoswitches is largely enabled by DNA nanotechnology, which has become a well-established field that uses DNA as a functional material to fabricate nanostructures.<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup> Biosensing is a particularly promising application of DNA nanotechnology,<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup> and reconfigurable DNA devices<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup> have been demonstrated for the detection of DNA,<sup><a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a></sup> RNA,<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup> proteins,<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup> and pH.<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup> However, most designs are complex and require laborious readout with advanced microscopy that reduces their practicality. A few approaches have overcome this practicality hurdle to provide widely useful solutions to problems in biological imaging (e.g. DNA-PAINT in super-resolution microscopy<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup> and DNA scaffolds for NMR<sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup> and cryo-EM<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>) and biosensing (e.g. detection of lysosomal disorders<sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup> and mapping cellular endocytic pathways<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>). Our DNA nanoswitches take a reductionist approach, resulting in assays that are robust and sensitive, yet simple to adapt and don’t require multiple steps or expensive equipment. With this work, we add viral detection to the existing suite of DNA nanoswitch assays that already includes protein<sup><a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a></sup> and microRNA<sup><a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">27</a></sup> detection.</p><p id="p-22">Our simple DNA nanoswitch-based assay for detection of viral RNA overcomes some limitations of currently available methods for clinical detection of RNA viruses in resource-limited areas. These challenges include 1) robust detection without enzymes or equipment, 2) maintaining low-cost and simplicity, and 3) providing specificity and versatility.</p><p id="p-23">The intrinsically high signal of our nanoswitches is enhanced here with a new “target multiplication” strategy where we use viral RNA fragmentation to multiply the number of targets, and thus increase the signal intensity. Using this approach, we reached near-clinical levels of detection in urine without the use of enzyme-mediated amplification strategies. This is of significance because enzymes can be key drivers of assay cost and complexity due to requirements including cold storage/transportation, special buffers and reagents, and strict operating temperatures. These factors make enzymatic assays difficult for field use or for use in low resource areas without modern lab infrastructure. Despite these challenges, most currently available techniques rely on enzyme triggered amplification.<sup><a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup> For our assay, we demonstrated compatibility and dramatic signal improvement with an optional enzymatic pre-amplification step <strong>(<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4C-4E</a>)</strong>. However, we believe that further improvements should enable complete coverage of the clinical range without enzymes. A 30 mL sample of urine from a ZIKV-infected patient would contain from 10<sup>5</sup> to 10<sup>9</sup> copies of viral RNA,<sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup> theoretically surpassing our current detection limit. Efficient sample preparation using a viral RNA extraction kit <strong>(<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4A-4B</a>)</strong>, for example, could facilitate use with our DNA nanoswitch assay.</p><p id="p-24">Two key features of our approach are simplicity and low cost. Our DNA nanoswitches align with the goals of “frugal science” movement, where cost and accessibility to new technologies are valued alongside typical performance metrics.<sup><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup> Our nanoswitches cost around 1 penny per reaction and can be stored dry at room temperature for at least one month, and can be delivered globally without transportation or biosafety concerns. The assay consists of few steps and can be performed in a matter of hours with limited laboratory needs (<strong>Fig. S17</strong>). Our assay uses a readout by gel electrophoresis, which is relatively inexpensive and already part of the workflow in many labs, which is comparatively simpler than many nanotechnology-based assays involving multiple incubation and wash steps. Improvements to the signal readout could potentially help make this approach even more lab independent. Successful detection with a commercially available bufferless gel system <strong>(Fig. S16)</strong> takes us a step closer to enabling field deployment of our assay, and sample preparation could be aided by other frugal science approaches such as the “paperfuge” <sup><a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a></sup> and low-cost thermal cycler.<sup><a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup></p><p id="p-25">The programmability of our system makes it versatile for a wide variety of viruses, which can be detected with high specificity as we showed for ZIKV and DENV <strong>(<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2A-2B</a>)</strong> and for different strains of ZIKV <strong>(<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2C-2D</a>)</strong>, even in a multiplexed fashion. Here we have focused on single-stranded RNA viruses but assays for other RNA or DNA viruses could likely be developed similarly. The fast construction and purification processes can facilitate rapid production of DNA nanoswitches to detect an emerging viral threat, potentially in as little as 1-2 days from knowledge of the target sequences, limited mostly by oligo synthesis turnaround time (<strong>Fig. S17</strong>). Due to the low cost of the test, our assay could also be useful for monitoring viral progression over time in patients, or for testing potentially infected insects or animals. Therefore, with future optimization towards point-of-care clinical applications in resource-limiting environments, the platform we describe here has a potential to improve accuracy and ease of diagnosis in humans, non-human vectors and other animals. Ultimately this can enhance our ability to control spread of infection and more rapidly respond to emerging viral threats, working toward a reduced death toll and economic burden.</p></div><div class="section" id="sec-5"><h2 class="">Funding</h2><p id="p-27">Research reported in this publication was supported by the NIH under award R35GM124720 to K. H. and awards R01GM123050 &amp; R21AI133617 to C.P. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH.</p></div><div class="section" id="sec-6"><h2 class="">Author contributions</h2><p id="p-28">L.Z., K.H., and C.T.P. designed the experiments. L.Z., A.R.C., J.A.P., S.C., O.L., C.C., and K.H. performed the nanoswitch experiments. G.B., P.B., and C.T.P. carried out the cell culture and total RNA extraction from infected cells. L.Z. developed the tool in Matlab and wrote the first draft of the manuscript. L.Z., A.R.C., J.A.P., C.T.P., and K.H. co-wrote later drafts of the manuscript.</p></div><div class="section COI-statement" id="sec-7"><h2 class="">Competing interests</h2><p id="p-29">K.H. and A.R.C. have intellectual property related to DNA nanoswitches. All other authors declare that they have no competing interests.</p></div><div class="section" id="sec-8"><h2 class="">Data and materials availability</h2><p id="p-30">All data needed to evaluate the conclusions in the paper are present in the paper and/or in the Supplementary Materials. Additional data related to this paper may be requested from the authors.</p></div><div class="section" id="sec-9"><h2 class="">Materials and Methods (Supplement)</h2><div class="section" id="sec-10"><h3>Construction and purification of nanoswitches</h3><p id="p-31">Oligonucleotides were purchased from Integrated DNA Technologies (IDT) with standard desalting, and the full sequences of all strands is listed in <strong>Supporting information Table S1 to S9</strong>. Nanoswitches were constructed as described previously.<sup><a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">27</a></sup> A genomic single-stranded DNA (New England Biolabs M13mp18) was linearized using targeted cleavage with BtsCI restriction enzyme. The linearized ssDNA was then mixed with a molar excess of an oligonucleotide mixture containing backbone oligos and detectors, and annealed from 90°C to 20°C at 1 °C min<sup>−1</sup> in a T100™ Thermal Cycler (Bio-Rad, USA). Following construction, the nanoswitches were purified using liquid chromatography (LC) purification<sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></sup> to remove excess oligonucleotides. The concentration of purified nanoswitches were determined by measuring A260 absorbance with a Thermo Scientific NanoDrop 2000.</p></div><div class="section" id="sec-11"><h3>In vitro transcription (IVT) of viral RNA</h3><p id="p-32">Plasmids containing the full-length ZIKV (Cambodia FSS13025 strain; pFLZIKV) and DENV-2 (strain 16681, pD2/IC-30P) cDNAs were gifts from Dr. Pei-Yong Shi (University of Texas Medical Branch) and Dr. Claire Huang (Centers for Disease Control), respectively.<sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup> pFLZIKV was linearized with ClaI (New England Biolabs, NEB), and pD2/IC-30P was linearized with XbaI (NEB). Digested plasmids were extracted with phenol:chloroform:isoamyl alcohol and then precipitated. Linearized plasmids were <em>in vitro</em> transcribed (Thermo Fisher Scientific) and the resulting viral RNA cleaned by MEGAclear Transcription Clean-Up Kit (Thermo Fisher Scientific). We followed the protocols of these two kits except that we did not heat the purification column in the elution step of the viral RNA because we noticed that high temperature can result in degradation of the viral RNA.</p></div><div class="section" id="sec-12"><h3>Viral RNA fragmentation test</h3><p id="p-33">Viral RNA was fragmented by using 10×Fragmentation buffer (NEB) and the recommended protocol. Briefly, the ZIKV RNA obtained from <em>in vitro</em> transcription was mixed with fragmentation buffer (1× final) and then incubated at 94°C in a thermal cycler for 1, 3, 6, or 9 min. Then RNA fragmentation analyzer (Agilent, model 5003) was used to quantify the length distribution of RNA fragments by using the DNF-471 Standard Sensitivity RNA Analysis Kit (<strong><a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2C</a></strong>).</p></div><div class="section" id="sec-13"><h3>Cell culture, ZIKV infections and total RNA extraction</h3><p id="p-34">Human hepatocarcinoma (Huh7) cells were maintained in Dulbecco’s modified Eagle’s medium (DMEM) (Life Technologies) supplemented with 10% fetal bovine serum (FBS; VWR Life Science Seradigm), 10 mM non-essential amino acids (NEAA; Life Technologies), and 5 mM L-glutamine (Life Technologies). Cells were passaged once every three days and maintained at 37°C with 5% CO<sub>2</sub>. Twenty-four hours prior to infection Huh7 cells were seeded into tissue culture plates. The following day, one plate was counted. The other two plates were used for mock- and ZIKV-infection, where cells were infected at a multiplicity of infection of one. The original Cambodia and Uganda (MR766) ZIKV stocks were a generous gift from Dr. Brett Lindenbach (Yale School of Medicine). RNA from mock- and ZIKV-infected cells was harvested by removing media from the cells was removed and then washing the cell monolayer washed once with ice-cold PBS. Hereafter, each tissue culture plate was lysed in 1 mL TRIzol (Invitrogen) and total RNA extracted per the manufacturer’s instructions. For experiments using ZIKV infectious particles in PBS/urine, cell culture media from mock- and ZIKV-infected cells was collected and stored at −80 °C. The number of infectious particles was determined by plaques assay, as described previously.<sup><a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a></sup></p></div><div class="section" id="sec-14"><h3>DNA nanoswitch detection</h3><p id="p-35">The total detection sample volume was 10 µl with 10 mM MgCl2, 1×PBS, 1.5-2 µl nanoswitch at 0.1 nM. The detection incubation was finished in a thermal cycler with a thermal annealing from 40 °C to 25 °C at −1 °C min<sup>−1</sup> or room temperature (e.g. the NASBA related detections). Before loading to the gel, the sample was stained by GelRed (Biotium Inc.) at 1×concentration (3.3×GelRed for total RNA related detection) and mixed with 2 µl 6× Blue loading dye.</p></div><div class="section" id="sec-15"><h3>Viral RNA detection sensitivity test</h3><p id="p-36">For the experiments of <strong><a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1G</a></strong> and <strong>Fig. S8</strong>, first, all nanoswitches were purified by LC and then their concentrations were determined by measuring A260 absorbance with a Thermo Scientific NanoDrop 2000. Nanoswitch mixtures were made by mixing nanoswitches in equimolar concentrations. The detection reaction volume is 10 µl with nanoswitch (0.1 nM), MgCl<sub>2</sub> (10mM), 1×PBS and blocking oligos (200 nM). The blocking oligos are short oligos (14 nucleotides) that can prevent the binding of target RNA to the inner surface of plastic tubes.<sup><a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">27</a></sup> The detection incubation was conducted in a thermal cycler with a thermal annealing from 40 °C to 25 °C for about 12 hours.</p></div><div class="section" id="sec-16"><h3>Detection of viral RNA from total RNA</h3><p id="p-37">First, 500 ng total RNA extracted from uninfected/infected cells was fragmented at 94°C for 9 minutes in 1x fragmentation buffer. Fragmented total RNA was then mixed with nanoswitches (1 nM), MgCl<sub>2</sub> (10 mM) and PBS (1×), and the mixture was made up to 10 µl with nuclease-free water. Samples were then incubated in a thermal annealing ramp from 40 °C to 25 °C at 1 °C min<sup>−1</sup>. After the incubation, samples were stained with GelRed (Biotium Inc.) at 3.3× concentration and incubated at room temperature for 30 min. Before loading the gel, 2 µl of 6x blue loading dye was mixed with each sample, and 10 µl sample was loaded to each well. Samples were run in a 0.8% agarose gel at 65-75 V for about 70-90 minutes in the cold room.</p></div><div class="section" id="sec-17"><h3>Detect viral RNA extracted from urine</h3><p id="p-38">For the detection of viral RNA extracted from urine, we first spiked the ZIKV RNA with blocking oligos (200 nM) into 200 µl human urine to mimic the clinic sample. Then, we used Quick-RNA Viral Kit (Zymo research) to extract the viral RNA from it. Different amount of ZIKV RNA were tested (<strong><a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4A</a></strong>). Here, the amount of human urine can be scaled up as needed according to the protocol of the kit. Finally, the viral RNA was eluted from the filter column by using 15 µl nuclease-free water. Then 5 µl extracted RNA was fragmented at 94°C for 9 minutes by using 0.2× fragmentation buffer (NEB) before conducting the nanoswitch detection. Here, we lowered the use of fragmentation buffer in the consideration of the small amount of RNA in the extracted sample as we noticed that too much fragmentation buffer could destroy the DNA nanoswitches (data not shown).</p></div><div class="section" id="sec-18"><h3>NASBA</h3><p id="p-39">First, we employed the classic NASBA protocol<sup><a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a></sup> to prove the concept (<strong>Fig. S15</strong>). The 25 µl one-pot reaction contained 3 µl RNA sample at various concentrations, 0.4 µM forward and reverse primers, 8 U AMV Reverse Transcriptase, 50 U T7 RNA Polymerase, 0.1 U RNase H, 40 U RNase Inhibitor (NEB, Murine), 2 mM NTP mix, 1 mM rNTP mix, 12 mM MgCl<sub>2</sub>, 40 mM Tris-HCl, 42 mM KCl, 5 mM Dithiotreitol (DTT), 15%(v/v) dimethyl sulfoxide. The primers were chosen from reference <sup><a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21</a></sup>. The sample was incubated at 41 °C for 2 hours in the thermal cycler and followed by heating at 94 °C for 10 minutes to deactivate all enzymes. Three microliters of the sample were used in the following DNA nanoswitch detection assay in PCR tubes with 10 µl final volumes. After mixing with the DNA nanoswitch and reaction buffer, the mix was incubated at room temperature for two hours. GelRed (Biotium Inc.) at 1×concentration was added to the detection samples before loading to the 0.8% agarose gel. The gel was run at room temperature for 45 minutes at 75 volts.</p><p id="p-40">For the ZIKV related NASBA experiments, first we spiked the ZIKV infectious particles (started at 1.96 fM of particles) into PBS or human urine. Then samples were diluted into different concentrations 1.49, 0.33, and 0.03 fM with 1xPBS or 10% human urine. Subsequently, blocking oligo (200 nM) was added and the viral RNA was released by heating the samples at 94 °C for 3 minutes. For the human urine samples, RNase Inhibitor (NEB, Murine) was also added at concentration of 2 U/µl before heating. After cooling down at room temperature, 0.5 µl RNase Inhibitor at 40 U/ul (NEB, Murine) was added to 10 µl human urine sample to protect the viral RNA. Total volume of each NASBA reaction was scaled down to 6 µl which contains 1.25 µl Enzyme COCKTAIL (NEC 1-24), 2 µl 3× buffer (NECB-24), 0.48 µl NTPs mix at 25 mM, 0.3 μl dNTPs mix at 20 mM, 0.2 µl two primers mix at 10 uM, 0.2 μl RNase Inhibitor with 40 U/ul (NEB, Murine) and 1.57 µl sample with viral RNA. The sample was incubated at 41 °C for 2 hours in the thermal cycler and followed by heating at 94 °C for 10 minutes to deactivate all enzymes. Then 1 µl of the NASBA sample was used in the following DNA nanoswitch detection assay in PCR tubes with 10 µl final volumes. The assay was finished by incubating at room temperature for two hours. After mixing with GelRed (Biotium Inc.) at 1×concentration and 2 µl 6× blue loading dye, the detection samples were loaded to the 25 ml 0.8% agarose gel which was run in 0.5×Tris-Borate-EDTA (TBE) buffer at 75 V at room temperature for 45 minutes.</p></div><div class="section" id="sec-19"><h3>Gel imaging and analysis</h3><p id="p-41">The detection samples were run in 0.8% agarose gels unless otherwise noted, cast from molecular biology grade agarose (Fisher BioReagents) dissolved in 0.5× TBE buffer. Typical running conditions were 75 V for 45 to 70 minutes at room temperature or cold room. Samples were mixed with a Ficoll based blue loading dye prior to loading. Imaging was completed on a Bio-Rad Gel Doc XR+ imager with different exposure time based on the brightness of the detection bands. The detection efficiency was analyzed using included Image Lab software (<strong><a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1F</a></strong>). The profiles of detection bands were obtained in ImageJ <sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup> and then their integrated intensity were obtained by using the peak analysis function in Origin (OriginLab Corporation), such as the data presented in <strong><a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1G, 3C</a> and <a id="xref-fig-4-7" class="xref-fig" href="#F4">4B</a></strong>. Detailed analysis procedure can be found in our previous publications<sup><a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">27</a></sup>. For the E-gel related experiments, we used Invitrogen E-gel agarose system (Thermo Fisher Scientific) and its precast agarose gel (1.0%, SYBR stained). Ten µl of nanoswitch detection sample was loaded to each lane and the gel was run at 48 volts for 1 hour at room temperature. Since the E-gel system does not allow user control of the voltage, we used an external power supplier connected with the negative and positive electrodes of the precast agarose gel to supply 48 volts.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-26">We thank Andy Berglund, Carl Shotwell, and Jared Richardson for facilitating use of the RNA fragment analyzer; Darren Yang and Wesley P. Wong for assistance and suggestion on adapting the E-gel system with the nanoswitches; Gabriele Fuchs and Bijan K. Dey for providing comments and suggestions on the experiments and manuscript.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.1" data-doi="10.1890/06-1052.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">K. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sax</span>,  <span class="cit-name-given-names">D. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaines</span>,  <span class="cit-name-given-names">S. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guernier</span>,  <span class="cit-name-given-names">V.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Guégan</span>,  <span class="cit-name-given-names">J.-F.</span></span> <span class="cit-article-title">Globalization of human infectious disease</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">1903</span>–<span class="cit-lpage">1910</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DSmith%26rft.auinit1%253DK.%2BF.%26rft.volume%253D88%26rft.issue%253D8%26rft.spage%253D1903%26rft.epage%253D1910%26rft.atitle%253DGlobalization%2Bof%2Bhuman%2Binfectious%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F06-1052.1%26rft_id%253Dinfo%253Apmid%252F17824419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/06-1052.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17824419&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248620500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.2" data-doi="10.1128/JVI.00694-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nebot</span>,  <span class="cit-name-given-names">M. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chirico</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mansky</span>,  <span class="cit-name-given-names">L. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Belshaw</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Viral Mutation Rates</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">9733</span>–<span class="cit-lpage">9748</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00694-10%26rft_id%253Dinfo%253Apmid%252F20660197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xOS85NzMzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xMi45MDI0NTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.3" data-doi="10.1146/annurev.mi.41.100187.002205"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Steinhauer</span>,  <span class="cit-name-given-names">D. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-article-title">Rapid Evolution of RNA Viruses</span>. <abbr class="cit-jnl-abbrev">Annual Review of Microbiology</abbr> <span class="cit-vol">41</span>, <span class="cit-fpage">409</span>–<span class="cit-lpage">431</span> (<span class="cit-pub-date">1987</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DSteinhauer%26rft.auinit1%253DD.%26rft.volume%253D41%26rft.issue%253D1%26rft.spage%253D409%26rft.epage%253D433%26rft.atitle%253DRapid%2BEvolution%2BOf%2BRNA%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.mi.41.100187.002205%26rft_id%253Dinfo%253Apmid%252F3318675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.mi.41.100187.002205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3318675&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K194900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.4" data-doi="10.1126/scitranslmed.3009872"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marston</span>,  <span class="cit-name-given-names">H. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Folkers</span>,  <span class="cit-name-given-names">G. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morens</span>,  <span class="cit-name-given-names">D. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fauci</span>,  <span class="cit-name-given-names">A. S.</span></span> <span class="cit-article-title">Emerging Viral Diseases: Confronting Threats with New Technologies</span>. <abbr class="cit-jnl-abbrev">Science Translational Medicine</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">253ps10</span>–<span class="cit-lpage">253ps10</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DMarston%26rft.auinit1%253DH.%2BD.%26rft.volume%253D6%26rft.issue%253D253%26rft.spage%253D253ps10%26rft.epage%253D253ps10%26rft.atitle%253DEmerging%2BViral%2BDiseases%253A%2BConfronting%2BThreats%2Bwith%2BNew%2BTechnologies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.3009872%26rft_id%253Dinfo%253Apmid%252F25210060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjEzOiI2LzI1My8yNTNwczEwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xMi45MDI0NTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.12.902452v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jerome</span>,  <span class="cit-name-given-names">K. R.</span></span> <span class="cit-source">Lennette’s Laboratory Diagnosis of Viral Infections</span>. (<span class="cit-publ-name">CRC Press</span>, <span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>,  <span class="cit-name-given-names">V. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Assay optimization for molecular detection of Zika virus</span>. <abbr class="cit-jnl-abbrev">Bull World Health Organ</abbr> <span class="cit-vol">94</span>, <span class="cit-fpage">880</span>–<span class="cit-lpage">892</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.volume%253D94%26rft.spage%253D880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.7" data-doi="10.1128/CMR.00072-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Musso</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gubler</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-article-title">Zika Virus</span>. <abbr class="cit-jnl-abbrev">Clinical Microbiology Reviews</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">487</span>–<span class="cit-lpage">524</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DMusso%26rft.auinit1%253DD.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D487%26rft.epage%253D524%26rft.atitle%253DZika%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00072-15%26rft_id%253Dinfo%253Apmid%252F27029595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjI5LzMvNDg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xMi45MDI0NTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.8" data-doi="10.1016/j.jcv.2018.03.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tyson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sails</span>,  <span class="cit-name-given-names">A. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krishna</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Staines</span>,  <span class="cit-name-given-names">H. M.</span></span> <span class="cit-article-title">The current landscape of nucleic acid tests for filovirus detection</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Virology</abbr> <span class="cit-vol">103</span>, <span class="cit-fpage">27</span>–<span class="cit-lpage">36</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BVirology%26rft.volume%253D103%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcv.2018.03.005%26rft_id%253Dinfo%253Apmid%252F29625392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jcv.2018.03.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29625392&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.9" data-doi="10.1128/CMR.3.2.132"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">James</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Immunoserology of infectious diseases</span>. <abbr class="cit-jnl-abbrev">Clinical Microbiology Reviews</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">132</span>–<span class="cit-lpage">152</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DJames%26rft.auinit1%253DK.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D132%26rft.epage%253D152%26rft.atitle%253DImmunoserology%2Bof%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.3.2.132%26rft_id%253Dinfo%253Apmid%252F2187592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjc6IjMvMi8xMzIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjEyLjkwMjQ1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.10" data-doi="10.1056/NEJMra1602113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petersen</span>,  <span class="cit-name-given-names">L. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jamieson</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Powers</span>,  <span class="cit-name-given-names">A. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Honein</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-article-title">Zika Virus</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr> <span class="cit-vol">374</span>, <span class="cit-fpage">1552</span>–<span class="cit-lpage">1563</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D374%26rft.spage%253D1552%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMra1602113%26rft_id%253Dinfo%253Apmid%252F27028561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMra1602113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27028561&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">How safe is safe: new human immunodeficiency virus Type 1 variants missed by nucleic acid testing</span>. <abbr class="cit-jnl-abbrev">Transfusion</abbr> <span class="cit-vol">53</span>, <span class="cit-fpage">2422</span>–<span class="cit-lpage">2430</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransfusion%26rft.volume%253D53%26rft.spage%253D2422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.12" data-doi="10.1038/ni.3515"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dejnirattisai</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dengue virus sero-cross-reactivity drives antibody-dependent enhancement of infection with zika virus</span>. <abbr class="cit-jnl-abbrev">Nature Immunology</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">1102</span>–<span class="cit-lpage">1108</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BImmunology%26rft.volume%253D17%26rft.spage%253D1102%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.3515%26rft_id%253Dinfo%253Apmid%252F27339099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.3515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27339099&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.13" data-doi="10.1093/jpids/pis133"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woods</span>,  <span class="cit-name-given-names">C. R.</span></span> <span class="cit-article-title">False-Positive Results for Immunoglobulin M Serologic Results: Explanations and Examples</span>. <abbr class="cit-jnl-abbrev">J Pediatric Infect Dis Soc</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">87</span>–<span class="cit-lpage">90</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BPediatric%2BInfect%2BDis%2BSoc%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjpids%252Fpis133%26rft_id%253Dinfo%253Apmid%252F26619450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jpids/pis133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26619450&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.14" data-doi="10.1038/s41575-019-0177-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lazarus</span>,  <span class="cit-name-given-names">J. V.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Too many people with viral hepatitis are diagnosed late — with dire consequences</span>. <abbr class="cit-jnl-abbrev">Nat Rev Gastroenterol Hepatol</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">451</span>–<span class="cit-lpage">452</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGastroenterol%2BHepatol%26rft.volume%253D16%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41575-019-0177-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41575-019-0177-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.15" data-doi="10.1586/14737159.2016.1142877"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Advances in addressing technical challenges of point-of-care diagnostics in resource-limited settings</span>. <abbr class="cit-jnl-abbrev">Expert Review of Molecular Diagnostics</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">449</span>–<span class="cit-lpage">459</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExpert%2BReview%2Bof%2BMolecular%2BDiagnostics%26rft.volume%253D16%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1586%252F14737159.2016.1142877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1586/14737159.2016.1142877&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Draz</span>,  <span class="cit-name-given-names">M. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shafiee</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Applications of gold nanoparticles in virus detection</span>. <abbr class="cit-jnl-abbrev">Theranostics</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1985</span>–<span class="cit-lpage">2017</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheranostics%26rft.volume%253D8%26rft.spage%253D1985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Balcioglu</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid Visual Screening and Programmable Subtype Classification of Ebola Virus Biomarkers</span>. <abbr class="cit-jnl-abbrev">Advanced Healthcare Materials</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">1600739</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvanced%2BHealthcare%2BMaterials%26rft.volume%253D6%26rft.spage%253D1600739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.18" data-doi="10.1016/j.bios.2017.08.051"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Afsahi</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Novel graphene-based biosensor for early detection of Zika virus infection</span>. <abbr class="cit-jnl-abbrev">Biosensors and Bioelectronics</abbr> <span class="cit-vol">100</span>, <span class="cit-fpage">85</span>–<span class="cit-lpage">88</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiosensors%2Band%2BBioelectronics%26rft.volume%253D100%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bios.2017.08.051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bios.2017.08.051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">J. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">W. H.</span></span> <span class="cit-article-title">Attomolar detection of virus by liquid coplanargate graphene transistor on plastic</span>. <abbr class="cit-jnl-abbrev">Nanotechnology</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">345502</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNanotechnology%26rft.volume%253D30%26rft.spage%253D345502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.20" data-doi="10.1126/science.aam9321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gootenberg</span>,  <span class="cit-name-given-names">J. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nucleic acid detection with CRISPR-Cas13a/C2c2</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-fpage">eaam9321</span> (<span class="cit-pub-date">2017</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aam9321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam9321%26rft_id%253Dinfo%253Apmid%252F28408723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTYvNjMzNi80MzgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjEyLjkwMjQ1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.21" data-doi="10.1016/j.cell.2016.04.059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pardee</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">165</span>, <span class="cit-fpage">1255</span>–<span class="cit-lpage">1266</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D165%26rft.spage%253D1255%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.04.059%26rft_id%253Dinfo%253Apmid%252F27160350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.04.059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27160350&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Strich</span>,  <span class="cit-name-given-names">J. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chertow</span>,  <span class="cit-name-given-names">D. S.</span></span> <span class="cit-article-title">CRISPR-Cas Biology and Its Application to Infectious Diseases</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr> <span class="cit-vol">57</span>, (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.23" data-doi="10.1088/0957-4484/22/49/494005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halvorsen</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schaak</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">W. P.</span></span> <span class="cit-article-title">Nanoengineering a single-molecule mechanical switch using DNA self-assembly</span>. <abbr class="cit-jnl-abbrev">Nanotechnology</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">494005</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNanotechnology%26rft.aulast%253DHalvorsen%26rft.auinit1%253DK.%26rft.volume%253D22%26rft.issue%253D49%26rft.spage%253D494005%26rft.epage%253D494005%26rft.atitle%253DNanoengineering%2Ba%2Bsingle-molecule%2Bmechanical%2Bswitch%2Busing%2BDNA%2Bself-assembly.%26rft_id%253Dinfo%253Adoi%252F10.1088%252F0957-4484%252F22%252F49%252F494005%26rft_id%253Dinfo%253Apmid%252F22101354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1088/0957-4484/22/49/494005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22101354&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koussa</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Halvorsen</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">W. P.</span></span> <span class="cit-article-title">DNA nanoswitches: a quantitative platform for gel-based biomolecular interaction analysis</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">123</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D12%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">A. D. T.</span></span> <span class="cit-article-title">Current status of Zika vaccine development: Zika vaccines advance into clinical evaluation</span>. <abbr class="cit-jnl-abbrev">npj Vaccines</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">24</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Dnpj%2BVaccines%26rft.volume%253D3%26rft.spage%253D24%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chandrasekaran</span>,  <span class="cit-name-given-names">A. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zavala</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Halvorsen</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Programmable DNA Nanoswitches for Detection of Nucleic Acid Sequences</span>. <abbr class="cit-jnl-abbrev">ACS Sens</abbr>. <span class="cit-vol">1</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">123</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSens%26rft.volume%253D1%26rft.spage%253D120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.27" data-doi="10.1126/sciadv.aau9443"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chandrasekaran</span>,  <span class="cit-name-given-names">A. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cellular microRNA detection with miRacles: microRNA-activated conditional looping of engineered switches</span>. <abbr class="cit-jnl-abbrev">Science Advances</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">eaau9443</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DChandrasekaran%26rft.auinit1%253DA.%2BR.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253Deaau9443%26rft.epage%253Deaau9443%26rft.atitle%253DCellular%2BmicroRNA%2Bdetection%2Bwith%2BmiRacles%253A%2BmicroRNA-%2Bactivated%2Bconditional%2Blooping%2Bof%2Bengineered%2Bswitches%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aau9443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNS8zL2VhYXU5NDQzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xMi45MDI0NTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.28" data-doi="10.3201/eid1408.080287"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lanciotti</span>,  <span class="cit-name-given-names">R. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genetic and Serologic Properties of Zika Virus Associated with an Epidemic, Yap State, Micronesia, 2007</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1232</span>–<span class="cit-lpage">1239</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DLanciotti%26rft.auinit1%253DR.%2BS.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D1232%26rft.epage%253D1239%26rft.atitle%253DGenetic%2Band%2Bserologic%2Bproperties%2Bof%2BZika%2Bvirus%2Bassociated%2Bwith%2Ban%2Bepidemic%252C%2BYap%2BState%252C%2BMicronesia%252C%2B2007.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1408.080287%26rft_id%253Dinfo%253Apmid%252F18680646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1408.080287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18680646&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258216900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halvorsen</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kizer</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandrasekaran</span>,  <span class="cit-name-given-names">A. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Basanta-Sanchez</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Shear Dependent LC Purification of an Engineered DNA Nanoswitch and Implications for DNA Origami</span>. <abbr class="cit-jnl-abbrev">Anal. Chem</abbr>. <span class="cit-vol">89</span>, <span class="cit-fpage">5673</span>–<span class="cit-lpage">5677</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal.%2BChem%26rft.volume%253D89%26rft.spage%253D5673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.30" data-doi="10.1016/j.chom.2016.05.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shan</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">An Infectious cDNA Clone of Zika Virus to Study Viral Virulence, Mosquito Transmission, and Antiviral Inhibitors</span>. <abbr class="cit-jnl-abbrev">Cell Host &amp; Microbe</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">891</span>–<span class="cit-lpage">900</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2B%2526%2BMicrobe%26rft.volume%253D19%26rft.spage%253D891%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2016.05.004%26rft_id%253Dinfo%253Apmid%252F27198478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2016.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27198478&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.31" data-doi="10.1038/s41467-019-09078-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huber</span>,  <span class="cit-name-given-names">R. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure mapping of dengue and Zika viruses reveals functional long-range interactions</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-09078-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-019-09078-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Integrative Analysis of Zika Virus Genome RNA Structure Reveals Critical Determinants of Viral Infectivity</span>. <abbr class="cit-jnl-abbrev">Cell Host &amp; Microbe</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">875</span>-<span class="cit-lpage">886</span>.e5 (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2B%2526%2BMicrobe%26rft.volume%253D24%26rft.spage%253D875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.33" data-doi="10.3201/eid2101.140894"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gourinat</span>,  <span class="cit-name-given-names">A.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Connor</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvez</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goarant</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dupont-Rouzeyrol</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Detection of Zika virus in urine</span>. <abbr class="cit-jnl-abbrev">Emerging Infect. Dis</abbr>. <span class="cit-vol">21</span>, <span class="cit-fpage">84</span>–<span class="cit-lpage">86</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfect.%2BDis%26rft.volume%253D21%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2101.140894%26rft_id%253Dinfo%253Apmid%252F25530324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2101.140894&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25530324&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paixão</span>,  <span class="cit-name-given-names">E. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teixeira</span>,  <span class="cit-name-given-names">M. G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rodrigues</span>,  <span class="cit-name-given-names">L. C.</span></span> <span class="cit-article-title">Zika, chikungunya and dengue: the causes and threats of new and re-emerging arboviral diseases</span>. <abbr class="cit-jnl-abbrev">BMJ Global Health</abbr> <span class="cit-vol">3</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramos-Castañeda</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dengue in Latin America: Systematic Review of Molecular Epidemiological Trends</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr> <span class="cit-vol">11</span>, (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haddow</span>,  <span class="cit-name-given-names">A. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genetic Characterization of Zika Virus Strains: Geographic Expansion of the Asian Lineage</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr> <span class="cit-vol">6</span>, (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faye</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular Evolution of Zika Virus during Its Emergence in the 20th Century</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr> <span class="cit-vol">8</span>, (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.38" data-doi="10.1128/JVI.00520-19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonenfant</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Zika Virus Subverts Stress Granules to Promote and Restrict Viral Gene Expression</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> (<span class="cit-pub-date">2019</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00520-19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00520-19%26rft_id%253Dinfo%253Apmid%252F30944179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjE1OiI5My8xMi9lMDA1MjAtMTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjEyLjkwMjQ1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paz-Bailey</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Persistence of Zika Virus in Body Fluids — Final Report</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr> <span class="cit-vol">379</span>, <span class="cit-fpage">1234</span>–<span class="cit-lpage">1243</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D379%26rft.spage%253D1234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.40" data-doi="10.1073/pnas.1708148114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hansen</span>,  <span class="cit-name-given-names">C. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koussa</span>,  <span class="cit-name-given-names">M. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">W. P.</span></span> <span class="cit-article-title">Nanoswitch-linked immunosorbent assay (NLISA) for fast, sensitive, and specific protein detection</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr> <span class="cit-vol">201708148</span> (<span class="cit-pub-date">2017</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1708148114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1708148114%26rft_id%253Dinfo%253Apmid%252F28893984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzM5LzEwMzY3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xMi45MDI0NTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.41" data-doi="10.1099/00221287-139-10-2423"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gabrielle</span>,  <span class="cit-name-given-names">M. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nucleic acid sequence-based amplification (NASBA) for the identification of mycobacteria</span>. <abbr class="cit-jnl-abbrev">Microbiology</abbr>, <span class="cit-vol">139</span>, <span class="cit-fpage">2423</span>–<span class="cit-lpage">2429</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253Dvan%2Bder%2BVliet%26rft.auinit1%253DG.%2BM.%26rft.volume%253D139%26rft.issue%253D10%26rft.spage%253D2423%26rft.epage%253D2429%26rft.atitle%253DNucleic%2Bacid%2Bsequence-based%2Bamplification%2B%2528NASBA%2529%2Bfor%2Bthe%2Bidentification%2Bof%2Bmycobacteria.%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-139-10-2423%26rft_id%253Dinfo%253Apmid%252F8254312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-139-10-2423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8254312&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.42" data-doi="10.1038/350091a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Compton</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Nucleic acid sequence-based amplification</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">350</span>, <span class="cit-fpage">91</span>–<span class="cit-lpage">92</span> (<span class="cit-pub-date">1991</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCompton%26rft.auinit1%253DJ.%26rft.volume%253D350%26rft.issue%253D6313%26rft.spage%253D91%26rft.epage%253D92%26rft.atitle%253DNucleic%2Bacid%2Bsequence-based%2Bamplification.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F350091a0%26rft_id%253Dinfo%253Apmid%252F1706072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/350091a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1706072&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FA69300075&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.43" data-doi="10.1038/nature01406"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seeman</span>,  <span class="cit-name-given-names">N. C.</span></span> <span class="cit-article-title">DNA in a material world</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>, <span class="cit-fpage">427</span>–<span class="cit-lpage">431</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSeeman%26rft.auinit1%253DN.%2BC.%26rft.volume%253D421%26rft.issue%253D6921%26rft.spage%253D427%26rft.epage%253D431%26rft.atitle%253DDNA%2Bin%2Ba%2Bmaterial%2Bworld.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01406%26rft_id%253Dinfo%253Apmid%252F12540916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12540916&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180533000057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.44" data-doi="10.1021/acs.chemrev.6b00825"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">DNA Origami: Scaffolds for Creating Higher Order Structures</span>. <abbr class="cit-jnl-abbrev">Chem. Rev</abbr>. <span class="cit-vol">117</span>, <span class="cit-fpage">12584</span>–<span class="cit-lpage">12640</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem.%2BRev%26rft.volume%253D117%26rft.spage%253D12584%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.chemrev.6b00825%26rft_id%253Dinfo%253Apmid%252F28605177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.chemrev.6b00825&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28605177&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.45" data-doi="10.1021/acs.chemrev.9b00121"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Rationally Engineered Nucleic Acid Architectures for Biosensing Applications</span>. <abbr class="cit-jnl-abbrev">Chem. Rev</abbr>. (<span class="cit-pub-date">2019</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/acs.chemrev.9b00121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem.%2BRev%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.chemrev.9b00121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.chemrev.9b00121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marras</span>,  <span class="cit-name-given-names">A. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castro</span>,  <span class="cit-name-given-names">C. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">H.-J.</span></span> <span class="cit-article-title">Paper Origami-Inspired Design and Actuation of DNA Nanomachines with Complex Motions</span>. <abbr class="cit-jnl-abbrev">Small</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">e1802580</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSmall%26rft.volume%253D14%26rft.spage%253De1802580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuzuya</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakai</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamazaki</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Komiyama</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Nanomechanical DNA origami ‘single-molecule beacons’ directly imaged by atomic force microscopy</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">449</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D2%26rft.spage%253D449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Funck</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicoli</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuzyk</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Liedl</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Sensing Picomolar Concentrations of RNA Using Switchable Plasmonic Chirality</span>. <abbr class="cit-jnl-abbrev">Angewandte Chemie</abbr> <span class="cit-vol">130</span>, <span class="cit-fpage">13683</span>–<span class="cit-lpage">13686</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAngewandte%2BChemie%26rft.volume%253D130%26rft.spage%253D13683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.49" data-doi="10.1126/science.1214081"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Douglas</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bachelet</span>,  <span class="cit-name-given-names">I.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Church</span>,  <span class="cit-name-given-names">G. M.</span></span> <span class="cit-article-title">A Logic-Gated Nanorobot for Targeted Transport of Molecular Payloads</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>, <span class="cit-fpage">831</span>–<span class="cit-lpage">834</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDouglas%26rft.auinit1%253DS.%2BM.%26rft.volume%253D335%26rft.issue%253D6070%26rft.spage%253D831%26rft.epage%253D834%26rft.atitle%253DA%2BLogic-Gated%2BNanorobot%2Bfor%2BTargeted%2BTransport%2Bof%2BMolecular%2BPayloads%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1214081%26rft_id%253Dinfo%253Apmid%252F22344439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA3MC84MzEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjEyLjkwMjQ1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Modi</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A DNA nanomachine that maps spatial and temporal pH changes inside living cells</span>. <abbr class="cit-jnl-abbrev">Nature Nanotechnology</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">325</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BNanotechnology%26rft.volume%253D4%26rft.spage%253D325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jungmann</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiplexed 3D cellular super-resolution imaging with DNA-PAINT and Exchange-PAINT</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">313</span>–<span class="cit-lpage">318</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D11%26rft.spage%253D313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.52" data-doi="10.1038/nature10257"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berardi</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shih</span>,  <span class="cit-name-given-names">W. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Chou</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-article-title">Mitochondrial uncoupling protein 2 structure determined by NMR molecular fragment searching</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">476</span>, <span class="cit-fpage">109</span>–<span class="cit-lpage">113</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBerardi%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D476%26rft.issue%253D7358%26rft.spage%253D109%26rft.epage%253D113%26rft.atitle%253DMitochondrial%2Buncoupling%2Bprotein%2B2%2Bstructure%2Bdetermined%2Bby%2BNMR%2Bmolecular%2Bfragment%2Bsearching.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10257%26rft_id%253Dinfo%253Apmid%252F21785437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21785437&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293447300040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.53" data-doi="10.1073/pnas.1612720113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">T. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Design of a molecular support for cryo-EM structure determination</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr> <span class="cit-vol">113</span>, <span class="cit-fpage">E7456</span>–<span class="cit-lpage">E7463</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1612720113%26rft_id%253Dinfo%253Apmid%252F27821763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzQ3L0U3NDU2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xMi45MDI0NTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.54" data-doi="10.1038/s41565-018-0318-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saminathan</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Krishnan</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">A DNA nanomachine chemically resolves lysosomes in live cells</span>. <abbr class="cit-jnl-abbrev">Nature Nanotechnology</abbr> <span class="cit-vol">1</span> (<span class="cit-pub-date">2018</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41565-018-0318-5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BNanotechnology%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41565-018-0318-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41565-018-0318-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Modi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nizak</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Surana</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Halder</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Krishnan</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Two DNA nanomachines map pH changes along intersecting endocytic pathways inside the same cell</span>. <abbr class="cit-jnl-abbrev">Nature Nanotech</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">459</span>–<span class="cit-lpage">467</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BNanotech%26rft.volume%253D8%26rft.spage%253D459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">56.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.12.902452v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Krishnan</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">High lumenal chloride in the lysosome is critical for lysosome function</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e28862</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253De28862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.57" data-doi="10.1126/science.aaq0179"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gootenberg</span>,  <span class="cit-name-given-names">J. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">360</span>, <span class="cit-fpage">439</span>–<span class="cit-lpage">444</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaq0179%26rft_id%253Dinfo%253Apmid%252F29449508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjAvNjM4Ny80MzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjEyLjkwMjQ1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Theel</span>,  <span class="cit-name-given-names">E. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hata</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-article-title">Diagnostic Testing for Zika Virus: a Postoutbreak Update</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr> <span class="cit-vol">56</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Whitesides</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">The frugal way: The promise of cost-conscious science</span>. <abbr class="cit-jnl-abbrev">The Economist: The World in 2012</abbr> <span class="cit-vol">154</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reardon</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Frugal science gets DIY diagnostics to world’s poorest</span>. <abbr class="cit-jnl-abbrev">New Scientist</abbr> <span class="cit-vol">219</span>, <span class="cit-fpage">20</span>–<span class="cit-lpage">21</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BScientist%26rft.volume%253D219%26rft.spage%253D20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhamla</span>,  <span class="cit-name-given-names">M. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Hand-powered ultralow-cost paper centrifuge</span>. <abbr class="cit-jnl-abbrev">Nature Biomedical Engineering</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">0009</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiomedical%2BEngineering%26rft.volume%253D1%26rft.spage%253D0009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsieh</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">A Rapid and Low-Cost PCR Thermal Cycler for Low Resource Settings</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e0131701</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D10%26rft.spage%253De0131701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.63" data-doi="10.1006/viro.1997.8500"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kinney</span>,  <span class="cit-name-given-names">R. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Construction of Infectious cDNA Clones for Dengue 2 Virus: Strain 16681 and Its Attenuated Vaccine Derivative, Strain PDK-53</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">230</span>, <span class="cit-fpage">300</span>–<span class="cit-lpage">308</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DKinney%26rft.auinit1%253DR.%2BM.%26rft.volume%253D230%26rft.issue%253D2%26rft.spage%253D300%26rft.epage%253D308%26rft.atitle%253DConstruction%2Bof%2Binfectious%2BcDNA%2Bclones%2Bfor%2Bdengue%2B2%2Bvirus%253A%2Bstrain%2B16681%2Band%2Bits%2Battenuated%2Bvaccine%2Bderivative%252C%2Bstrain%2BPDK-53.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1997.8500%26rft_id%253Dinfo%253Apmid%252F9143286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1997.8500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9143286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WV20400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.12.902452v1.64" data-doi="10.1038/nmeth.2089"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasband</span>,  <span class="cit-name-given-names">W. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Eliceiri</span>,  <span class="cit-name-given-names">K. W.</span></span> <span class="cit-article-title">NIH Image to ImageJ: 25 years of image analysis</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">671</span>–<span class="cit-lpage">675</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneider%26rft.auinit1%253DC.%2BA.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D671%26rft.epage%253D675%26rft.atitle%253DNIH%2BImage%2Bto%2BImageJ%253A%2B25%2Byears%2Bof%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2089%26rft_id%253Dinfo%253Apmid%252F22930834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22930834&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.12.902452.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305942200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.12.902452v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.12.902452v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.16.907394v1" title="Modular lipid nanoparticle platform technology for siRNA and lipophilic prodrug delivery" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 16, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.12.902452v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.01.12.902452v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.12.902452v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.12.902452v1" class="active" data-icon-position="" data-hide-link-title="0">Programmable low-cost DNA-based platform for viral RNA detection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1107550" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1zyiLbfmHUGqw871TQRJfx3By58Izx4JKPvqZ0tXOmE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107550" id="node1107550--2" data-pisa="biorxiv;2020.01.12.902452v1" data-pisa-master="biorxiv;2020.01.12.902452" data-seqnum="1107550" data-apath="/biorxiv/early/2020/01/16/2020.01.12.902452.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Programmable low-cost DNA-based platform for viral RNA detection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Lifeng</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Arun Richard</span> <span class="nlm-surname">Chandrasekaran</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jibin Abraham</span> <span class="nlm-surname">Punnoose</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Gaston</span> <span class="nlm-surname">Bonenfant</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Stephon</span> <span class="nlm-surname">Charles</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Oksana</span> <span class="nlm-surname">Levchenko</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Pheonah</span> <span class="nlm-surname">Badu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Cassandra</span> <span class="nlm-surname">Cavaliere</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Cara T.</span> <span class="nlm-surname">Pager</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Ken</span> <span class="nlm-surname">Halvorsen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.12.902452; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.12.902452 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.12.902452v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;title=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;title=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;text=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;title=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;t=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;title=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;title=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107550" id="node1107550--3" data-pisa="biorxiv;2020.01.12.902452v1" data-pisa-master="biorxiv;2020.01.12.902452" data-seqnum="1107550" data-apath="/biorxiv/early/2020/01/16/2020.01.12.902452.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Programmable low-cost DNA-based platform for viral RNA detection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Lifeng</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Arun Richard</span> <span class="nlm-surname">Chandrasekaran</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jibin Abraham</span> <span class="nlm-surname">Punnoose</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Gaston</span> <span class="nlm-surname">Bonenfant</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Stephon</span> <span class="nlm-surname">Charles</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Oksana</span> <span class="nlm-surname">Levchenko</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Pheonah</span> <span class="nlm-surname">Badu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Cassandra</span> <span class="nlm-surname">Cavaliere</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Cara T.</span> <span class="nlm-surname">Pager</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Ken</span> <span class="nlm-surname">Halvorsen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.12.902452; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.12.902452 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Zhou%2C+Lifeng&amp;rft.au=Chandrasekaran%2C+Arun+Richard&amp;rft.au=Punnoose%2C+Jibin+Abraham&amp;rft.au=Bonenfant%2C+Gaston&amp;rft.au=Charles%2C+Stephon&amp;rft.au=Levchenko%2C+Oksana&amp;rft.au=Badu%2C+Pheonah&amp;rft.au=Cavaliere%2C+Cassandra&amp;rft.au=Pager%2C+Cara+T.&amp;rft.au=Halvorsen%2C+Ken"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1107550/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1107550/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1107550/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1107550/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1107550/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1107550/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1107550/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1107550/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1107550/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1107550/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1107550/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1107550/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;count=horizontal&amp;via=&amp;text=Programmable%20low-cost%20DNA-based%20platform%20for%20viral%20RNA%20detection&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.12.902452v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.12.902452v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioengineering" class="highlight" data-icon-position="" data-hide-link-title="0">Bioengineering<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
