<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Front Immunol</span><span class="iso-abbrev" title="iso-abbrev">Front Immunol</span><span class="publisher-id" title="publisher-id">Front. Immunol.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Immunology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1664-3224</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6783508">6783508</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fimmu.2019.02340">10.3389/fimmu.2019.02340</a></span><div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span>: <span class="subject" title="subject">Editorial</span></div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Editorial: The Past and the Future of Human Immunity Under Viral Evolutionary Pressure</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurst</span><span class="given-names" tagx="given-names" title="given-names">Tara Patricia</span></span><a href="#aff1"><sup>1</sup></a><a href="#c001"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/458423/overview" /></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magiorkinis</span><span class="given-names" tagx="given-names" title="given-names">Gkikas</span></span><a href="#aff2"><sup>2</sup></a><a href="#c002"><sup>*</sup></a><span class="uri" title="uri"><a href="http://loop.frontiersin.org/people/451207/overview" /></span></span></div><span class="citation_author_institution" id="aff1">[], <sup>1</sup><span class="institution" title="institution">Department of Life Sciences, School of Health Sciences, Birmingham City University</span><span class="addr-line" title="addr-line">Birmingham</span><span class="country" tagx="country" title="country">United Kingdom</span></span><span class="citation_author_institution" id="aff2">[], <sup>2</sup><span class="institution" title="institution">Department of Hygiene, Epidemiology and Medical Statistics, Medical School, National and Kapodistrian University of Athens</span><span class="addr-line" title="addr-line">Athens</span><span class="country" tagx="country" title="country">Greece</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited and reviewed by: Ian Marriott, University of North Carolina at Charlotte, United States</p> </div> <div class="corresp" title="corresp">
*Correspondence: Tara Patricia Hurst <span class="email" tagx="email" title="email">tara.hurst@bcu.ac.uk</span></div> <div class="corresp" title="corresp">
Gkikas Magiorkinis <span class="email" tagx="email" title="email">gkikasmag@gmail.com</span></div> <div class="fn-type-other" title="other"> <p>This article was submitted to Microbial Immunology, a section of the journal Frontiers in Immunology</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-10-10</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">2340</span><span class="history" title="history"><span class="received" title="received">received: 2019-8-16</span><span class="accepted" title="accepted">accepted: 2019-9-17</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Hurst and Magiorkinis</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</span></span></div><span class="related-article" tagx="related-article" title="related-article">Editorial on the Research Topic <span class="article-title" tagx="article-title" title="article-title">The Past and the Future of Human Immunity Under Viral Evolutionary Pressure</span></span><div class="kwd-group"> <div class="kwd" title="kwd">
human viruses</div> <div class="kwd" title="kwd">
endogenous retrovirus</div> <div class="kwd" title="kwd">
immunity</div> <div class="kwd" title="kwd">
evolution</div> <div class="kwd" title="kwd">
disease</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> </div> </div> <div class="body" title="body"> <p>"<i>Nothing in biology makes sense except in the light of evolution</i>�? was argued by Theodosius Dobzhansky in 1973. After more than 100 years since Charles Darwin's <i>Origin of Species</i>, it was increasingly realized that the theory of evolution can help us disentangle biological mechanisms. Here we asked contributors to see their antiviral research interests "through the looking glass�? of evolution and to consider how evolutionary theory may help us understand human antiviral immunity. Thus, our special topic explored the scope of the evolution of human antiviral immunity, whether this included short-term effects or long-term impact. We specifically asked contributors to provide thought-provoking manuscripts that could fall within the scope of co-evolution of viruses and the human immune response. They submitted articles discussing the ongoing dynamics of co-existing with ancient endogenous retroviruses, as well as the pressures exerted by extant exogenous viruses such as influenza and human immunodeficiency virus (HIV-1). Our understanding of antiviral immunity was broadened by papers exploring mechanisms such as T cell function and RNA structural components.</p> <p>In a Hypothesis and Theory article, <a href="https://doi.org/10.3389/fimmu.2019.00197">Marchi et al.</a> explore the effects of persistent infection on T cell responses by analyzing gene expression profiles and gene networks in publicly-available data sets. Importantly, they identified T-box 21 (Tbx21)-mediated gene expression as a hallmark of inflation but not exhaustion of memory CD8+ T cells. This finding has potential to shape our understanding of the genes that define memory T-cell populations. Further, this shows how different viral infections can influence either exhaustion or inflation. This highlights the adaptability of the immune response to viral infection but also how infection could alter gene expression to shift T cell phenotypes.</p> <p>In one of two mini-reviews in the topic, <a href="https://doi.org/10.3389/fimmu.2018.02097">Smyth et al.</a> discuss the contribution of RNA structure to antiviral immunity. RNA is much more than an intermediary between DNA and protein; it can adopt complex three-dimensional structures that may be altered by single nucleotide mutations, with consequent effects on its functions. Thus, since RNA structures are critical to the replication of many viruses, they are also substrates upon which evolution can act. This can include non-coding RNAs produced during replication, as well as genomic RNA and transcripts. Viral RNA is sensed by different pattern recognition receptors (PRRs) of the innate immune response, such as Toll-like receptor 3 (TLR3) and retinoic acid inducible gene I (RIG-I). There is thus pressure on viral RNA to evade immune detection such as by nucleotide and codon usage bias (adenosine by HIV-1), the presence of specific secondary structures that restrict transcription to low levels, or the maintenance of single-stranded RNA regions to avoid PRR detection. In turn, viral RNA structures have influenced the human immune system, such as the evolution of the human leukocyte antigen (HLA) locus.</p> <p>The co-evolution of influenza virus and host immunity is discussed in the second mini-review (<a href="https://doi.org/10.3389/fimmu.2018.02017">Voskarides et al.</a>). The authors argue that this is an example of antagonistic evolution, with influenza virus a predator and the human host as prey. This conflict manifests in the genetic variability of Influenza viruses, a result of the absence of proof-reading by the RNA polymerase, that continually challenges the host immune system. In response, the human host has genetic major histocompatibility complex (MHC) diversity which facilitates T cell-mediated immunity to diverse pathogens.</p> <p>APOBEC3G (apolipoprotein B mRNA editing enzyme catalytic polypeptide-like 3G, A3G) is a cellular cytidine deaminase that mutates retroviral genomes, thereby being a key host cell mechanism to debilitate viruses that embed in the genome. In an original article herein, the authors suggest that the action of A3G extends to adaptive immunity, suggesting that HIV-1 cytotoxic T cell (CTL) escape mutants are generated through A3G activity (<a href="https://doi.org/10.3389/fimmu.2018.03032">Borzooee et al.</a>). The authors use a bioinformatics approach to simulate A3G mutations and show that HIV-1 mutants are biased toward cytotoxic T cell (CTL) escape mutants. These results point toward an evolutionary adaptation of the HIV-1 genome that essentially "hijacks�? human antiretroviral activity to counteract T-cell immunity.</p> <p>A cluster of papers within the topic discuss the role of endogenous retroviruses (ERV) in the immune response. This includes an up-to-date review on the shaping of the immune response by human ERVs (HERVs) (<a href="https://doi.org/10.3389/fimmu.2018.02039">Grandi and Tramontano</a>). Further, four original research articles explore diverse aspects of ERVs and immunity. Firstly, ERV sequences within the genome are a potential source of genetic diversity that are known to be co-opted by the host. This has been shown most recently to include HERV-H sequences contributing to stem cell identity. <a href="https://doi.org/10.3389/fimmu.2019.01339">Gemmell et al.</a> explore the nature of HERV-H loci and their transcription further, showing positive correlations between particular long-terminal repeats (LTRs) and cell types (type II LTRs and embryonic cells).</p> <p>Secondly, <a href="https://doi.org/10.3389/fmicb.2018.01577">Turnbull and Douville</a> analyzed HERV-K protease sequences within the human genome. They show that transcription of HERV-K proteases varies according to the functional motifs, with two motifs (DTGAD, DTGVD) being more abundant among all HERV-K integrations, but also more active, compared to the less common motifs, in diseases such as amyotrophic lateral sclerosis and breast cancer. These results are important not only for understanding the interactions among HERV-K integrations evolutionarily, but also to appreciate the diversity of enzymatic targets when designing anti-HERV based treatments.</p> <p>Thirdly, the effect of the histone deactylase (HDAC) inhibitor vorinostat on HERV expression in CD4+ T cells was analyzed using RNA-Seq (<a href="https://doi.org/10.3389/fimmu.2018.00603">White et al.</a>). Vorinostat is one latency reversing agent being examined in HIV-1 shock and kill strategies. The study found over 2000 HERV elements were modulated by vorinostat, notably the downregulation of ERVL and upregulation of HERV-9 elements, the latter of which was confirmed by digital droplet PCR. This study identified three HERV-9 members (LTR12 elements) that were upregulated by vorinostat in a dose-dependent manner. The relevance of this finding lies in the transcriptional regulation of certain genes by LTR12 elements, including the pro-apoptotic genes TP3 and TNFRSF10B (<a href="https://doi.org/10.3389/fimmu.2018.00603">White et al.</a>). In agreement with our previous work using RT-qPCR, members of the HERV-K (HML-2), HERV-W and HERV-FRD families were not found to be upregulated by vorinostat (<a href="#B1">1</a>). Notably the study by <a href="https://doi.org/10.3389/fimmu.2018.00603">White et al.</a> unlike that of Hurst et al., used uninfected, primary CD4+ T cells to avoid confounding effects of HIV-1 infection.</p> <p>Finally, ERVs can be upregulated in response to acute exogenous viral infections and this may contribute to the antiviral response. In the paper by <a href="https://doi.org/10.3389/fimmu.2019.00901">Maze et al.</a> the expression of <i>Papio cynocephalus</i> Endogenous Retrovirus (PcEV) was found to be upregulated in macaque plasma and tissues (peripheral blood mononuclear cells (PBMCs) and spleen) in response to simian immunodeficiency virus (SIV) infection. Further, upregulation of the interferon (IFN)-stimulated gene (ISG), STAT1, an important marker of the early antiviral response, correlated with the increased PcEV RNA levels (<a href="https://doi.org/10.3389/fimmu.2019.00901">Maze et al.</a>). The authors posit a role for ERVs in the activation of the innate immune response.</p> <p>The effect of immunity-virus interface at the population level was examined by <a href="https://doi.org/10.3389/fmicb.2018.01825">Williams et al.</a> where they present the perplexing phenomenon of a "pocket�? of uninfected individuals in a connected network of recently HIV-1-infected individuals. This raises the question of how a cluster of connected HIV-1 exposed individuals could remain uninfected within an HIV-1 outbreak. The authors discuss potential explanations including the possibility that long-term infections act as a "firewall�? in spreading the infections to the full-population. The latter could be explained by the existence of an epidemic equilibrium of co-existence where human-HIV co-evolution would be acting in the absence of treatment.</p> <p>The evolutionary gameplay underlying the pathogenesis of hepatitis B virus's e antigen (HBVeAg) is reviewed and explored by <a href="https://doi.org/10.3389/fmicb.2018.02521">Kramvis et al.</a> HBVeAg, as the authors explain, is a tolerogen, an antigen that increases the tolerance of HBV infection by the human host. There is significant variability in HBeAg seroconversion and transmission efficiency among genotypes, which the authors suggest can be explained as a result of short-sighted evolution. The virus-host interplay of HBV is further explored by <a href="https://doi.org/10.3389/fimmu.2018.01561">Lumley et al.</a> on the effects of CD8+ T cell responses. The authors review the evidence that CD8+ T cells play an important role in the chronicity of HBV infection as well as the disease outcomes such as cirrhosis and hepatocellular carcinoma (<a href="https://doi.org/10.3389/fimmu.2018.01561">Lumley et al.</a>). The role of interferon-α receptor-1 (IFNAR1) promoter polymorphisms on HBV infection are explored by <a href="https://doi.org/10.3389/fimmu.2018.00777">Karamitros et al.</a> with respect to the development of hepatocellular carcinoma. The authors show that the variable tandem repeat [VNTR: −77(GT)n] has a significant impact in the transcription profile of IFNAR1 and this may be relevant in cancer pathophysiology.</p> <p>Finally, <a href="https://doi.org/10.3389/fimmu.2018.01640">Beaver et al.</a>, have reviewed evidence of how Zika virus (ZIKV) evolution can explain the observed patterns of pathogenesis. The authors review evidence showing that the immune responses against African isolates differ from those observed against Asian ZIKV lineages, and discuss how this could explain observed pathogenic differences among ZIKV strains.</p> <p>From ancient germline infections to modern epidemics, outbreaks or emerging infections, our contributors explored the evolutionary interplay of human immunity and viruses at the molecular, individual, or population level. They offered thought-provoking hypotheses for further exploration, reviewed publications, and provided novel results in the field of human antiviral immunity field. This shows the power of integrating evolutionary theory with research of antiviral immunity and we hope that our special topic will further promote evolutionary thinking across fields of human pathogen research.</p> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>TH and GM wrote the draft and edited the final version of the manuscript.</p> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurst</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pace</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzourakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klenerman</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frater</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span>. <span class="mixed-article-title" title="mixed-article-title">Human endogenous retrovirus (HERV) expression is not induced by treatment with the histone deacetylase (HDAC) inhibitors in cellular models of HIV-1 latency</span>. <span class="source" tagx="source" title="source">Retrovirology.</span> (<span class="year" tagx="year" title="year">2016</span>) <span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s12977-016-0242-4">10.1186/s12977-016-0242-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26852322">26852322</a></span></span></li> </ul> </div> </div>  </body></html>