<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/06/19/676155.atom"/>
  <meta name="HW.pisa" content="biorxiv;676155v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Metagenomic Nanopore sequencing of influenza virus direct from clinical respiratory samples"/>
  <meta name="DC.Identifier" content="10.1101/676155"/>
  <meta name="DC.Date" content="2019-06-19"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Influenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential of a diagnostic test for influenza which also provides insights on transmission, evolution and drug resistance, and simultaneously detects other viruses. We therefore set out to apply Oxford Nanopore Technology to metagenomic sequencing of respiratory samples. We generated influenza reads down to a limit of detection of 102-103 genome copies/ml in pooled samples, observing a strong relationship between the viral titre and the proportion of influenza reads (p = 4.7×10-5). Applying our methods to clinical throat swabs, we generated influenza reads for 27/27 samples with high-to-mid viral titres (Cycle threshold (Ct) values &lt;;30) and 6/13 samples with low viral titres (Ct values 30-40). No false positive reads were generated from 10 influenza-negative samples. Thus Nanopore sequencing operated with 83% sensitivity (95% CI 67-93%) and 100% specificity (95% CI 69-100%) compared to the current diagnostic standard. Coverage of full length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza titres, we were able to reconstruct &gt;;99% complete sequence for all eight gene segments. We also detected Human Coronavirus and generated a near complete Human Metapneumovirus genome from clinical samples. While further optimisation is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza and other respiratory viruses."/>
  <meta name="DC.Contributor" content="Kuiama Lewandowski"/>
  <meta name="DC.Contributor" content="Yifei Xu"/>
  <meta name="DC.Contributor" content="Steven T. Pullan"/>
  <meta name="DC.Contributor" content="Sheila F. Lumley"/>
  <meta name="DC.Contributor" content="Dona Foster"/>
  <meta name="DC.Contributor" content="Nicholas Sanderson"/>
  <meta name="DC.Contributor" content="Alison Vaughan"/>
  <meta name="DC.Contributor" content="Marcus Morgan"/>
  <meta name="DC.Contributor" content="Nicole Bright"/>
  <meta name="DC.Contributor" content="James Kavanagh"/>
  <meta name="DC.Contributor" content="Richard Vipond"/>
  <meta name="DC.Contributor" content="Miles Carroll"/>
  <meta name="DC.Contributor" content="Anthony C. Marriott"/>
  <meta name="DC.Contributor" content="Karen E Gooch"/>
  <meta name="DC.Contributor" content="Monique Andersson"/>
  <meta name="DC.Contributor" content="Katie Jeffery"/>
  <meta name="DC.Contributor" content="Timothy EA Peto"/>
  <meta name="DC.Contributor" content="Derrick W. Crook"/>
  <meta name="DC.Contributor" content="A Sarah Walker"/>
  <meta name="DC.Contributor" content="Philippa C. Matthews"/>
  <meta name="article:published_time" content="2019-06-19"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Metagenomic Nanopore sequencing of influenza virus direct from clinical respiratory samples"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Influenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential of a diagnostic test for influenza which also provides insights on transmission, evolution and drug resistance, and simultaneously detects other viruses. We therefore set out to apply Oxford Nanopore Technology to metagenomic sequencing of respiratory samples. We generated influenza reads down to a limit of detection of 10&lt;;sup&gt;;2&lt;;/sup&gt;;-10&lt;;sup&gt;;3&lt;;/sup&gt;; genome copies/ml in pooled samples, observing a strong relationship between the viral titre and the proportion of influenza reads (p = 4.7×10&lt;;sup&gt;;-5&lt;;/sup&gt;;). Applying our methods to clinical throat swabs, we generated influenza reads for 27/27 samples with high-to-mid viral titres (Cycle threshold (Ct) values &amp;;lt;30) and 6/13 samples with low viral titres (Ct values 30-40). No false positive reads were generated from 10 influenza-negative samples. Thus Nanopore sequencing operated with 83% sensitivity (95% CI 67-93%) and 100% specificity (95% CI 69-100%) compared to the current diagnostic standard. Coverage of full length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza titres, we were able to reconstruct &amp;;gt;99% complete sequence for all eight gene segments. We also detected Human Coronavirus and generated a near complete Human Metapneumovirus genome from clinical samples. While further optimisation is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza and other respiratory viruses.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;676155v1"/>
  <meta name="citation_id" content="676155v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/676155v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/676155v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/676155v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155.full.pdf"/>
  <meta name="citation_doi" content="10.1101/676155"/>
  <meta name="citation_num_pages" content="36"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="676155"/>
  <meta name="citation_author" content="Kuiama Lewandowski"/>
  <meta name="citation_author_institution" content="Public Health England, National infection Service"/>
  <meta name="citation_author" content="Yifei Xu"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Steven T. Pullan"/>
  <meta name="citation_author_institution" content="Public Health England, National infection Service"/>
  <meta name="citation_author_email" content="steven.pullan@phe.gov.uk"/>
  <meta name="citation_author" content="Sheila F. Lumley"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Microbiology, Oxford University Hospitals NHS Foundation Trust, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Dona Foster"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Oxford NIHR BRC, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Nicholas Sanderson"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Oxford NIHR BRC, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Alison Vaughan"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Oxford NIHR BRC, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Marcus Morgan"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Microbiology, Oxford University Hospitals NHS Foundation Trust, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Nicole Bright"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author" content="James Kavanagh"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Richard Vipond"/>
  <meta name="citation_author_institution" content="Public Health England, National infection Service"/>
  <meta name="citation_author" content="Miles Carroll"/>
  <meta name="citation_author_institution" content="Public Health England, National infection Service"/>
  <meta name="citation_author" content="Anthony C. Marriott"/>
  <meta name="citation_author_institution" content="Public Health England, National infection Service"/>
  <meta name="citation_author" content="Karen E Gooch"/>
  <meta name="citation_author_institution" content="Public Health England, National infection Service"/>
  <meta name="citation_author" content="Monique Andersson"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Microbiology, Oxford University Hospitals NHS Foundation Trust, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Katie Jeffery"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Microbiology, Oxford University Hospitals NHS Foundation Trust, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Timothy EA Peto"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Microbiology, Oxford University Hospitals NHS Foundation Trust, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Oxford NIHR BRC, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Derrick W. Crook"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Microbiology, Oxford University Hospitals NHS Foundation Trust, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Oxford NIHR BRC, John Radcliffe Hospital"/>
  <meta name="citation_author" content="A Sarah Walker"/>
  <meta name="citation_author_institution" content="Nuffield Department of Medicine, University of Oxford, John Radcliffe Hospital"/>
  <meta name="citation_author" content="Philippa C. Matthews"/>
  <meta name="citation_author_institution" content="Department of Infectious Diseases and Microbiology, Oxford University Hospitals NHS Foundation Trust, John Radcliffe Hospital"/>
  <meta name="citation_author_institution" content="Peter Medawar Building for Pathogen Research, Nuffield Department of Medicine, University of Oxford"/>
  <meta name="citation_author_institution" content="Oxford NIHR BRC, John Radcliffe Hospital"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4036-4269"/>
  <meta name="citation_reference" content="Krammer F, Smith GJD, Fouchier RAM, Peiris M, Kedzierska K, Doherty PC, et al. Influenza. Nat Rev Dis Primers. 2018;4: 3."/>
  <meta name="citation_reference" content="Joseph U, Su YCF, Vijaykrishna D, Smith GJD. The ecology and adaptive evolution of influenza A interspecies transmission. Influenza Other Respi Viruses. 2017;11: 74–84."/>
  <meta name="citation_reference" content="Humphreys M. The influenza of 1918: Evolutionary perspectives in a historical context. Evol Med Public Health. 2018;2018: 219–229."/>
  <meta name="citation_reference" content="Smith GJD, Vijaykrishna D, Bahl J, Lycett SJ, Worobey M, Pybus OG, et al. Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza A epidemic. Nature. 2009;459: 1122–1125."/>
  <meta name="citation_reference" content="Sutton TC. The Pandemic Threat of Emerging H5 and H7 Avian Influenza Viruses. Viruses. 2018;10. doi:10.3390/v10090461"/>
  <meta name="citation_reference" content="Xiang D, Pu Z, Luo T, Guo F, Li X, Shen X, et al. Evolutionary dynamics of avian influenza A H7N9 virus across five waves in mainland China, 2013-2017. J Infect. 2018;77: 205–211."/>
  <meta name="citation_reference" content="Ten health issues WHO will tackle this year [Internet]. [cited 22 Jan 2019]. Available: https://www.who.int/emergencies/ten-threats-to-global-health-in-2019"/>
  <meta name="citation_reference" content="WHO | Burden of disease. World Health Organization; 2018; Available: https://www.who.int/influenza/surveillance_monitoring/bod/en/"/>
  <meta name="citation_reference" content="Uyeki TM, Bernstein HH, Bradley JS, Englund JA, File TM Jr, Fry AM, et al. Clinical Practice Guidelines by the Infectious Diseases Society of America: 2018 Update on Diagnosis, Treatment, Chemoprophylaxis, and Institutional Outbreak Management of Seasonal Influenza. Clin Infect Dis. 2018; doi:10.1093/cid/ciy866"/>
  <meta name="citation_reference" content="Iuliano AD, Roguski KM, Chang HH, Muscatello DJ, Palekar R, Tempia S, et al. Estimates of global seasonal influenza-associated respiratory mortality: a modelling study. Lancet. 2018;391: 1285–1300."/>
  <meta name="citation_reference" content="WHO | Burden of disease. World Health Organization; 2018; Available: https://www.who.int/influenza/surveillance_monitoring/bod/en/"/>
  <meta name="citation_reference" content="Morris DH, Gostic KM, Pompei S, Bedford T, Łuksza M, Neher RA, et al. Predictive Modeling of Influenza Shows the Promise of Applied Evolutionary Biology. Trends Microbiol. 2018;26: 102–118."/>
  <meta name="citation_reference" content="Blanco N, Eisenberg MC, Stillwell T, Foxman B. What Transmission Precautions Best Control Influenza Spread in a Hospital? Am J Epidemiol. 2016;183: 1045–1054."/>
  <meta name="citation_reference" content="Sansone M, Wiman Å, Karlberg ML, Brytting M, Bohlin L, Andersson L-M, et al. Molecular characterization of a nosocomial outbreak of influenza B virus in an acute care hospital setting. J Hosp Infect. 2019;101: 30–37."/>
  <meta name="citation_reference" content="Ito Y. Clinical Diagnosis of Influenza. Methods Mol Biol. 2018;1836: 23–31."/>
  <meta name="citation_reference" content="Green DA, StGeorge K. Rapid Antigen Tests for Influenza: Rationale and Significance of the FDA Reclassification. J Clin Microbiol. 2018;56. doi:10.1128/JCM.00711-18"/>
  <meta name="citation_reference" content="Benirschke RC, McElvania E, Thomson R, Kaul K, Das S. Clinical Impact of Rapid Point-of-Care Polymerase Chain Reaction Influenza Testing In An Urgent Care Setting: A Single Center Study. J Clin Microbiol. 2019; doi:10.1128/JCM.01281-18"/>
  <meta name="citation_reference" content="Schmidt RLJ, Simon A, Popow-Kraupp T, Laggner A, Haslacher H, Fritzer-Szekeres M, et al. A novel PCR-based point-of-care method facilitates rapid, efficient, and sensitive diagnosis of influenza virus infection. Clin Microbiol Infect. 2018; doi:10.1016/j.cmi.2018.12.017"/>
  <meta name="citation_reference" content="Votintseva AA, Bradley P, Pankhurst L, Del Ojo. Elias C, Loose M, Nilgiriwala K, et al. Same-Day Diagnostic and Surveillance Data for Tuberculosis via Whole-Genome Sequencing of Direct Respiratory Samples. J Clin Microbiol. 2017;55: 1285–1298."/>
  <meta name="citation_reference" content="Yan Y, Jia X-J, Wang H-H, Fu X-F, Ji J-M, He P-Y, et al. Dynamic quantification of avian influenza H7N9(A) virus in a human infection during clinical treatment using droplet digital PCR. J Virol Methods. 2016;234: 22–27."/>
  <meta name="citation_reference" content="Kafetzopoulou LE, Efthymiadis K, Lewandowski K, Crook A, Carter D, Osborne J, et al. Assessment of metagenomic Nanopore and Illumina sequencing for recovering whole genome sequences of chikungunya and dengue viruses directly from clinical samples. Euro Surveill. 2018;23. doi:10.2807/1560-7917.ES.2018.23.50.1800228"/>
  <meta name="citation_reference" content="Xu Y, Lewandowski K, Lumley S, Pullan S, Vipond R, Carroll M, et al. Detection of Viral Pathogens With Multiplex Nanopore MinION Sequencing: Be Careful With Cross-Talk. Front Microbiol. 2018;9: 2225."/>
  <meta name="citation_reference" content="Karamitros T, van Wilgenburg B, Wills M, Klenerman P, Magiorkinis G. Nanopore sequencing and full genome de novo assembly of human cytomegalovirus TB40/E reveals clonal diversity and structural variations. BMC Genomics. 2018;19: 577."/>
  <meta name="citation_reference" content="Greninger AL, Naccache SN, Federman S, Yu G, Mbala P, Bres V, et al. Rapid metagenomic identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis. Genome Med. 2015;7: 99."/>
  <meta name="citation_reference" content="McNaughton AL, Roberts HE, Bonsall D, de Cesare M, Mokaya J, Lumley SF, et al. Illumina and Nanopore methods for whole genome sequencing of hepatitis B virus (HBV) [Internet]. bioRxiv. 2018. p. 470633. doi:10.1101/470633"/>
  <meta name="citation_reference" content="Keller MW, Rambo-Martin BL, Wilson MM, Ridenour CA, Shepard SS, Stark TJ, et al. Direct RNA Sequencing of the Coding Complete Influenza A Virus Genome. Sci Rep. 2018;8: 14408."/>
  <meta name="citation_reference" content="Eckert SE, Chan JZ.M, Houniet D, PATHSEEK consortium, Breuer J, Speight G. Enrichment by hybridisation of long DNA fragments for Nanopore sequencing. Microb Genom. 2016;2: e000087."/>
  <meta name="citation_reference" content="Rambo-Martin BL, Keller MW, Wilson MM, Nolting JM, Anderson TK, Vincent AL, et al. Mitigating Pandemic Risk with Influenza A Virus Field Surveillance at a Swine-Human Interface [Internet]. bioRxiv. 2019. p. 585588. doi:10.1101/585588"/>
  <meta name="citation_reference" content="Xu Y, Lewandowski K, Lumley S, Sanderson ND, Vaughan A, Vipond RT, et al. Nanopore metagenomic sequencing of full length human metapneumovirus (HMPV) within a unique sub-lineage [Internet]. bioRxiv. 2018. p. 496687. doi:10.1101/496687"/>
  <meta name="citation_reference" content="Gooch KE, Marriott AC, Ryan KA, Yeates P, Slack GS, Brown PJ, et al. Heterosubtypic cross-protection correlates with cross-reactive interferon-gamma-secreting lymphocytes in the ferret model of influenza. Sci Rep. 2019;9: 2617."/>
  <meta name="citation_reference" content="Meinel DM, Heinzinger S, Eberle U, Ackermann N, Schönberger K, Sing A. Whole genome sequencing identifies influenza A H3N2 transmission and offers superior resolution to classical typing methods. Infection. 2018;46: 69–76."/>
  <meta name="citation_reference" content="Rambaut A, Pybus OG, Nelson MI, Viboud C, Taubenberger JK, Holmes EC. The genomic and epidemiological dynamics of human influenza A virus. Nature. 2008;453: 615–619."/>
  <meta name="citation_reference" content="Thompson CP, Lourenço J, Walters AA, Obolski U, Edmans M, Palmer DS, et al. A naturally protective epitope of limited variability as an influenza vaccine target. Nat Commun. 2018;9: 3859."/>
  <meta name="citation_reference" content="Kafetzopoulou LE, Pullan ST, Lemey P, Suchard MA, Ehichioya DU, Pahlmann M, et al. Metagenomic sequencing at the epicenter of the Nigeria 2018 Lassa fever outbreak. Science. 2019;363: 74–77."/>
  <meta name="citation_reference" content="Dudas G, Bedford T. The ability of single genes vs full genomes to resolve time and space in outbreak analysis [Internet]. bioRxiv. 2019. p. 582957. doi:10.1101/582957"/>
  <meta name="citation_reference" content="Miller S, Naccache SN, Samayoa E, Messacar K, Arevalo S, Federman S, et al. Laboratory validation of a clinical metagenomic sequencing assay for pathogen detection in cerebrospinal fluid. Genome Res. 2019;29: 831–842."/>
  <meta name="citation_reference" content="CDC. Influenza (flu) including seasonal, avian, swine, pandemic, and other. In: Centers for Disease Control and Prevention [Internet]. 29 Mar 2019 [cited 2 Apr 2019]. Available: https://www.cdc.gov/flu"/>
  <meta name="citation_reference" content="Lewandowski K, Bell A, Miles R, Carne S, Wooldridge D, Manso C, et al. The Effect of Nucleic Acid Extraction Platforms and Sample Storage on the Integrity of Viral RNA for Use in Whole Genome Sequencing. J Mol Diagn. 2017;19: 303–312."/>
  <meta name="citation_reference" content="Marriott AC, Dove BK, Whittaker CJ, Bruce C, Ryan KA, Bean TJ, et al. Low dose influenza virus challenge in the ferret leads to increased virus shedding and greater sensitivity to oseltamivir. PLoS One. 2014;9: e94090."/>
  <meta name="citation_reference" content="Squires RB, Noronha J, Hunt V, García-Sastre A, Macken C, Baumgarth N, et al. Influenza research database: an integrated bioinformatics resource for influenza research and surveillance. Influenza Other Respi Viruses. 2012;6: 404–416."/>
  <meta name="twitter:title" content="Metagenomic Nanopore sequencing of influenza virus direct from clinical respiratory samples"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Influenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential of a diagnostic test for influenza which also provides insights on transmission, evolution and drug resistance, and simultaneously detects other viruses. We therefore set out to apply Oxford Nanopore Technology to metagenomic sequencing of respiratory samples. We generated influenza reads down to a limit of detection of 102-103 genome copies/ml in pooled samples, observing a strong relationship between the viral titre and the proportion of influenza reads (p = 4.7×10-5). Applying our methods to clinical throat swabs, we generated influenza reads for 27/27 samples with high-to-mid viral titres (Cycle threshold (Ct) values &lt;;30) and 6/13 samples with low viral titres (Ct values 30-40). No false positive reads were generated from 10 influenza-negative samples. Thus Nanopore sequencing operated with 83% sensitivity (95% CI 67-93%) and 100% specificity (95% CI 69-100%) compared to the current diagnostic standard. Coverage of full length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza titres, we were able to reconstruct &gt;;99% complete sequence for all eight gene segments. We also detected Human Coronavirus and generated a near complete Human Metapneumovirus genome from clinical samples. While further optimisation is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza and other respiratory viruses."/>
  <meta name="og-title" property="og:title" content="Metagenomic Nanopore sequencing of influenza virus direct from clinical respiratory samples"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/676155v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Influenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential of a diagnostic test for influenza which also provides insights on transmission, evolution and drug resistance, and simultaneously detects other viruses. We therefore set out to apply Oxford Nanopore Technology to metagenomic sequencing of respiratory samples. We generated influenza reads down to a limit of detection of 102-103 genome copies/ml in pooled samples, observing a strong relationship between the viral titre and the proportion of influenza reads (p = 4.7×10-5). Applying our methods to clinical throat swabs, we generated influenza reads for 27/27 samples with high-to-mid viral titres (Cycle threshold (Ct) values &lt;;30) and 6/13 samples with low viral titres (Ct values 30-40). No false positive reads were generated from 10 influenza-negative samples. Thus Nanopore sequencing operated with 83% sensitivity (95% CI 67-93%) and 100% specificity (95% CI 69-100%) compared to the current diagnostic standard. Coverage of full length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza titres, we were able to reconstruct &gt;;99% complete sequence for all eight gene segments. We also detected Human Coronavirus and generated a near complete Human Metapneumovirus genome from clinical samples. While further optimisation is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza and other respiratory viruses."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-06-19"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Metagenomic Nanopore sequencing of influenza virus direct from clinical respiratory samples | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-782509 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Influenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential of a diagnostic test for influenza which also provides insights on transmission, evolution and drug resistance, and simultaneously detects other viruses. We therefore set out to apply Oxford Nanopore Technology to metagenomic sequencing of respiratory samples. We generated influenza reads down to a limit of detection of 10
     <sup>2</sup>-10
     <sup>3</sup> genome copies/ml in pooled samples, observing a strong relationship between the viral titre and the proportion of influenza reads (p = 4.7×10
     <sup>-5</sup>). Applying our methods to clinical throat swabs, we generated influenza reads for 27/27 samples with high-to-mid viral titres (Cycle threshold (Ct) values &lt;;30) and 6/13 samples with low viral titres (Ct values 30-40). No false positive reads were generated from 10 influenza-negative samples. Thus Nanopore sequencing operated with 83% sensitivity (95% CI 67-93%) and 100% specificity (95% CI 69-100%) compared to the current diagnostic standard. Coverage of full length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza titres, we were able to reconstruct &gt;;99% complete sequence for all eight gene segments. We also detected Human Coronavirus and generated a near complete Human Metapneumovirus genome from clinical samples. While further optimisation is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza and other respiratory viruses.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">Influenza A is an RNA Orthomyxovirus of approximately 13Kb length, with an eight-segment genome. It is typically classified on the basis of Haemagglutinin (HA) and Neuraminidase (NA), of which there are 16 and 9 main variants respectively [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Genetic reassortment underpins the potential for transmission between different host species [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>], and for the evolution of highly pathogenic variants [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>–
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], recognised in the WHO list of ‘ten threats to global health’ [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. Seasonal influenza causes an estimated 650,000 deaths globally each year, and H3N2 alone kills 35,000 each year in the USA [
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Certain groups are particularly at risk, including older adults, infants and young children, pregnant women, those with underlying lung disease, and the immunocompromised [
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. The burden of disease disproportionately affects low/middle income settings [
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Influenza diagnostics and surveillance are fundamental to identify the emergence of novel strains, to improve prediction of potential epidemics and pandemics [
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>], and to inform vaccine strategy [
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Diagnostic data facilitate real-time surveillance, can underpin infection control interventions [
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>] and inform the prescription of Neuraminidase Inhibitors (NAI) [
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>].
    </p>
    <p id="p-5">Currently, most clinical diagnostic tests for influenza depend on detecting viral antigen, or PCR amplification of viral nucleic acid derived from respiratory samples [
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. These two approaches offer trade-offs in benefits: antigen tests (including point-of-care tests (POCT)) are typically rapid but low sensitivity [
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>–
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], while PCR is more time-consuming but more sensitive [
     <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>]. Irrespective of test used, most clinical diagnostic facilities report a non-quantitative (binary) diagnostic result, and the data generated for influenza have limited capacity to inform insights into epidemiological linkage, vaccine efficacy or anti-viral susceptibility. Given this, there is an aspiration to generate new diagnostic tests that combine speed (incorporating potential for POCT [
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]), sensitivity, detection of co-infection [
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], and generation of quantitative or semi-quantitative data that can be used to identify drug resistance and reconstruct phylogeny to inform surveillance, public health strategy, and vaccine design.
    </p>
    <p id="p-6">The application of Oxford Nanopore Technologies (ONT; 
     <a href="https://nanoporetech.com/">https://nanoporetech.com/</a>) to generate full-length influenza sequences from clinical respiratory samples can address these challenges. ONT offers a ‘third-generation’, portable, real-time approach to generating long-read sequence data, with demonstrated success across a range of viruses [
     <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>,
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>–
     <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. To date, Nanopore sequencing of influenza has been reported using high titre virus from an 
     <em>in vitro</em> culture system, producing full length genome sequences through direct RNA sequencing [
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] or targeted enrichment by either hybridisation of cDNA [
     <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>] or influenza-specific PCR amplification [
     <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>].
    </p>
    <p id="p-7">We therefore aimed to optimise a metagenomic protocol for detecting influenza viruses directly from clinical samples using Nanopore sequencing. We determine its sensitivity compared to existing diagnostic methods and its accuracy compared to short-read (Illumina) sequencing, using clinical samples from hospital patients during an influenza season, and samples from a laboratory controlled infection in ferrets.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Method optimisation to increase proportion of viral reads derived from throat swabs</h3>
     <p id="p-8">We first sequenced five influenza A positive and five influenza-negative throat swabs, each spiked with Hazara virus control at 10
      <sup>4</sup> genome copies/ml. Using the SISPA approach [
      <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>] followed by Nanopore sequencing, we produced metagenomic data dominated by reads that were bacterial in origin, with extremely few viral reads detected. Passing the sample through a 0.4 μm filter prior to nucleic acid extraction increased the detection of viral reads by several orders of magnitude (
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig S1</a>). Filtration is relatively expensive, so we also assessed the alternative approach of adding a rapid centrifugation step to pellet bacterial and human cells, followed by nucleic acid extraction from the supernatant. We used a pooled set of influenza A positive samples (concentration 10
      <sup>6</sup> genome copies/ml), to provide a large enough sample to assess reproducibility, with the Hazara control spiked in at 10
      <sup>4</sup> genome copies/ml. Enrichment for influenza and Hazara was similar for filtration vs centrifugation, based on reads mapping to the viral genome (
      <a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig S2</a>). As centrifugation is simpler and cheaper, we selected this approach for all further testing.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Method optimisation to reduce time for cDNA synthesis</h3>
     <p id="p-9">Synthesis of tagged, randomly primed cDNA and its subsequent amplification via SISPA [
      <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21</a>] required lengthy reverse transcription and PCR steps (1h and 3h 45min) respectively. Optimising these stages upgraded the reverse transcriptase from SuperScriptIII to SuperScriptIV (ThermoFisher), reduced incubation time to 10min (processing time reduction of 50min) and reduced the PCR extension time within each cycle from 5min to 2min (1h 30min processing time reduction). Comparing this final method with our original protocol, using triplicate extractions from the pooled set of influenza A positive samples, demonstrated no significant loss in performance in the more rapid protocol (
      <a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig S3</a>) and we adopted this approach as our routine protocol.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Consistent retrieval of Hazara virus by Nanopore sequencing</h3>
     <p id="p-10">Starting with an influenza A positive sample pool (10
      <sup>6</sup> genome copies/ml), we made three volumetric dilution series using three independent influenza-negative pools (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1</a>). The total quantity of cDNA after preparation for sequencing was consistently higher in all samples using negative pool-3 as diluent (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A</a>), indicating the presence of a higher concentration of non- viral RNA within pool-3. This is likely due to host cell lysis or higher bacterial presence, and demonstrates the variable nature of throat swab samples.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Schematic to show processing protocol through clinical and research pipelines for influenza diagnosis. (A) Clinical sample collection (orange), clinical diagnostic testing (yellow), sample processing and sequencing using Oxford Nanopore Technologies (blue), processing of sequence data (purple). (B) Outline of pooled influenza-positive samples into influenza-negative background, to generate varying titres of influenza virus (from zero up to 106 genome copies/ml), undertaken in triplicate, and spiked with a standard titre of Hazara virus control at 104 genome copies/ml)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570647831" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Schematic to show processing protocol through clinical and research pipelines for influenza diagnosis.&lt;;/span&gt;; (A) Clinical sample collection (orange), clinical diagnostic testing (yellow), sample processing and sequencing using Oxford Nanopore Technologies (blue), processing of sequence data (purple). (B) Outline of pooled influenza-positive samples into influenza-negative background, to generate varying titres of influenza virus (from zero up to 10&lt;;sup&gt;;6&lt;;/sup&gt;; genome copies/ml), undertaken in triplicate, and spiked with a standard titre of Hazara virus control at 10&lt;;sup&gt;;4&lt;;/sup&gt;; genome copies/ml).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F1.medium.gif" width="440" height="264"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F1.medium.gif" width="440" height="264"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1:</span>
       <span class="caption-title">Schematic to show processing protocol through clinical and research pipelines for influenza diagnosis.</span>
       <p id="p-11" class="first-child">(A) Clinical sample collection (orange), clinical diagnostic testing (yellow), sample processing and sequencing using Oxford Nanopore Technologies (blue), processing of sequence data (purple). (B) Outline of pooled influenza-positive samples into influenza-negative background, to generate varying titres of influenza virus (from zero up to 10
        <sup>6</sup> genome copies/ml), undertaken in triplicate, and spiked with a standard titre of Hazara virus control at 10
        <sup>4</sup> genome copies/ml).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Characteristics of three pools of influenza-negative throat swabs, and Nanopore sequence results following spiking with influenza A. (A) Total concentration of cDNA produced per pooled sample following amplification by the SISPA reaction, grouped by dilution series. The 106 sample in each pool is the original, undiluted material, represented by the black filled bars. Samples diluted to influenza titres of 104, 103 and 102 contain more cDNA due to higher background material (bacterial/human) present in the diluent. Dilution series 1 and 2 contain comparable amounts of background material; dilution series 3 contains substantially more background. (B) Viral reads generated by Nanopore sequencing of samples with different titres of influenza A, and a consistent titre of Hazara virus (104 genome copies/ml). Graphs show reads per million of total reads mapping to influenza A or Hazara virus genomes, across the three individual dilution series. Note logarithmic scale on y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570647831" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Characteristics of three pools of influenza-negative throat swabs, and Nanopore sequence results following spiking with influenza A.&lt;;/span&gt;; (A) Total concentration of cDNA produced per pooled sample following amplification by the SISPA reaction, grouped by dilution series. The 10&lt;;sup&gt;;6&lt;;/sup&gt;; sample in each pool is the original, undiluted material, represented by the black filled bars. Samples diluted to influenza titres of 10&lt;;sup&gt;;4&lt;;/sup&gt;;, 10&lt;;sup&gt;;3&lt;;/sup&gt;; and 10&lt;;sup&gt;;2&lt;;/sup&gt;; contain more cDNA due to higher background material (bacterial/human) present in the diluent. Dilution series 1 and 2 contain comparable amounts of background material; dilution series 3 contains substantially more background. (B) Viral reads generated by Nanopore sequencing of samples with different titres of influenza A, and a consistent titre of Hazara virus (10&lt;;sup&gt;;4&lt;;/sup&gt;; genome copies/ml). Graphs show reads per million of total reads mapping to influenza A or Hazara virus genomes, across the three individual dilution series. Note logarithmic scale on y-axis.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-2.medium.gif" width="440" height="324"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-2.medium.gif" width="440" height="324"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Characteristics of three pools of influenza-negative throat swabs, and Nanopore sequence results following spiking with influenza A. (A) Total concentration of cDNA produced per pooled sample following amplification by the SISPA reaction, grouped by dilution series. The 106 sample in each pool is the original, undiluted material, represented by the black filled bars. Samples diluted to influenza titres of 104, 103 and 102 contain more cDNA due to higher background material (bacterial/human) present in the diluent. Dilution series 1 and 2 contain comparable amounts of background material; dilution series 3 contains substantially more background. (B) Viral reads generated by Nanopore sequencing of samples with different titres of influenza A, and a consistent titre of Hazara virus (104 genome copies/ml). Graphs show reads per million of total reads mapping to influenza A or Hazara virus genomes, across the three individual dilution series. Note logarithmic scale on y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570647831" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Characteristics of three pools of influenza-negative throat swabs, and Nanopore sequence results following spiking with influenza A.&lt;;/span&gt;; (A) Total concentration of cDNA produced per pooled sample following amplification by the SISPA reaction, grouped by dilution series. The 10&lt;;sup&gt;;6&lt;;/sup&gt;; sample in each pool is the original, undiluted material, represented by the black filled bars. Samples diluted to influenza titres of 10&lt;;sup&gt;;4&lt;;/sup&gt;;, 10&lt;;sup&gt;;3&lt;;/sup&gt;; and 10&lt;;sup&gt;;2&lt;;/sup&gt;; contain more cDNA due to higher background material (bacterial/human) present in the diluent. Dilution series 1 and 2 contain comparable amounts of background material; dilution series 3 contains substantially more background. (B) Viral reads generated by Nanopore sequencing of samples with different titres of influenza A, and a consistent titre of Hazara virus (10&lt;;sup&gt;;4&lt;;/sup&gt;; genome copies/ml). Graphs show reads per million of total reads mapping to influenza A or Hazara virus genomes, across the three individual dilution series. Note logarithmic scale on y-axis.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-3.medium.gif" width="440" height="316"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-3.medium.gif" width="440" height="316"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F2/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
       </div>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Characteristics of three pools of influenza-negative throat swabs, and Nanopore sequence results following spiking with influenza A.</span>
       <p id="p-12" class="first-child">(A) Total concentration of cDNA produced per pooled sample following amplification by the SISPA reaction, grouped by dilution series. The 10
        <sup>6</sup> sample in each pool is the original, undiluted material, represented by the black filled bars. Samples diluted to influenza titres of 10
        <sup>4</sup>, 10
        <sup>3</sup> and 10
        <sup>2</sup> contain more cDNA due to higher background material (bacterial/human) present in the diluent. Dilution series 1 and 2 contain comparable amounts of background material; dilution series 3 contains substantially more background. (B) Viral reads generated by Nanopore sequencing of samples with different titres of influenza A, and a consistent titre of Hazara virus (10
        <sup>4</sup> genome copies/ml). Graphs show reads per million of total reads mapping to influenza A or Hazara virus genomes, across the three individual dilution series. Note logarithmic scale on y-axis.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-13">We consistently retrieved Hazara virus reads from all three dilution series by Nanopore sequencing, independently of influenza virus titre in the sample (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2B</a>). Sequencing from dilution series 1 and 2 gave a consistent proportion of total reads mapping to the Hazara genome, across dilutions and between the first two pools with mean values per pool of 1.4×10
      <sup>3</sup> ±660 RPM (reads per million (of total reads) ± standard deviation) and 1.2×10
      <sup>3</sup> ±350 RPM respectively. The pool-3 dilution series generated 260 ±340 RPM Hazara reads across samples, and showed a decreasing trend associated with increased dilution factor, as increasingly more non-viral RNA was introduced from this high-background pool.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Limit of influenza virus detection by Nanopore from pooled samples</h3>
     <p id="p-14">Nanopore sequencing of the triplicate SISPA preparations of the influenza A positive pool produced mean 5.3×10
      <sup>4</sup> ±3.6×10
      <sup>4</sup> RPM mapping to the influenza A genome (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2B</a>). Across the dilution series, the proportion of influenza reads was strongly associated with influenza virus titre (p-value = 4.7×10
      <sup>-5</sup>), but was also influenced by which negative pool was used for dilution, consistent with the pattern observed for the Hazara control. Sequencing the negative controls (pools with no influenza virus spike) generated no reads mapping to influenza. At influenza virus titres &lt;;10
      <sup>3</sup> copies/ml, influenza reads were inconsistently detected across the samples (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2B</a>), suggesting the limit of detection is between 10
      <sup>2</sup>-10
      <sup>3</sup> influenza copies/ml.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Retrieval and reconstruction of complete influenza genomes from pooled/spiked samples</h3>
     <p id="p-15">For the Hazara virus control (10
      <sup>4</sup> genome copies/ml spike), genome coverage was 81.4-99.4% (at 1x depth) for pools 1 and 2. Coverage in the high-background pool 3 was more variable (21.5-96.5%; 
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3A</a>). Influenza A genome coverage at 10
      <sup>6</sup> copies/ml was ≥99.3% for each segment in all samples (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3A</a>). At 10
      <sup>4</sup> genome copies/ml of influenza, mean 1x coverage per segment was 90.3% for pools 1 and 2, but was substantially reduced in the high-background pool 3 to 5.7% (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3A</a>). At influenza titres &lt;;10
      <sup>4</sup> copies/ ml, coverage was highly variable across genome segments. However, when present at 10
      <sup>3</sup> copies/ml, 2/3 pools had sufficient data for correct subtyping as H3N2 (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/785988/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/785988/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/785988" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">Summary of results from Nanopore sequencing based on pooled samples with varying titres of Influenza A, and a consistent titre of Hazara virus control.</span>
       <p id="p-16" class="first-child">Each dilution is undertaken in triplicate (shown as 3 dilution pools).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Coverage of influenza and Hazara genome segments achieved by Nanopore sequencing from pooled samples (A) Data from three dilution series of pooled influenza positive samples, diluted with three separate negative sample pools to generate different titres of influenza virus. Each individual dilution was spiked with Hazara at 104 genome copies/ml. Proportion of genome covered at 1x depth is shown for each of the eight influenza genome segments (encoding: PB2 - polymerase subunit, PB1 - polymerase subunit, PA - polymerase acidic protein, HA - hemagglutinin, NP- nucleocapsid protein, NA - neuraminidase, M - matrix proteins, NS - nonstructural proteins) across the three dilution series. Coverage of the Hazara genome is plotted as total of all three genome segments for simplicity. (B) Representative coverage plot of influenza A genome segments from the dilution series 1 sample at 104 influenza copies per ml." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570647831" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Coverage of influenza and Hazara genome segments achieved by Nanopore sequencing from pooled samples&lt;;/span&gt;; (A) Data from three dilution series of pooled influenza positive samples, diluted with three separate negative sample pools to generate different titres of influenza virus. Each individual dilution was spiked with Hazara at 10&lt;;sup&gt;;4&lt;;/sup&gt;; genome copies/ml. Proportion of genome covered at 1x depth is shown for each of the eight influenza genome segments (encoding: PB2 - polymerase subunit, PB1 - polymerase subunit, PA - polymerase acidic protein, HA - hemagglutinin, NP- nucleocapsid protein, NA - neuraminidase, M - matrix proteins, NS - nonstructural proteins) across the three dilution series. Coverage of the Hazara genome is plotted as total of all three genome segments for simplicity. (B) Representative coverage plot of influenza A genome segments from the dilution series 1 sample at 10&lt;;sup&gt;;4&lt;;/sup&gt;; influenza copies per ml.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-5.medium.gif" width="440" height="321"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-5.medium.gif" width="440" height="321"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Coverage of influenza and Hazara genome segments achieved by Nanopore sequencing from pooled samples (A) Data from three dilution series of pooled influenza positive samples, diluted with three separate negative sample pools to generate different titres of influenza virus. Each individual dilution was spiked with Hazara at 104 genome copies/ml. Proportion of genome covered at 1x depth is shown for each of the eight influenza genome segments (encoding: PB2 - polymerase subunit, PB1 - polymerase subunit, PA - polymerase acidic protein, HA - hemagglutinin, NP- nucleocapsid protein, NA - neuraminidase, M - matrix proteins, NS - nonstructural proteins) across the three dilution series. Coverage of the Hazara genome is plotted as total of all three genome segments for simplicity. (B) Representative coverage plot of influenza A genome segments from the dilution series 1 sample at 104 influenza copies per ml." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570647831" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Coverage of influenza and Hazara genome segments achieved by Nanopore sequencing from pooled samples&lt;;/span&gt;; (A) Data from three dilution series of pooled influenza positive samples, diluted with three separate negative sample pools to generate different titres of influenza virus. Each individual dilution was spiked with Hazara at 10&lt;;sup&gt;;4&lt;;/sup&gt;; genome copies/ml. Proportion of genome covered at 1x depth is shown for each of the eight influenza genome segments (encoding: PB2 - polymerase subunit, PB1 - polymerase subunit, PA - polymerase acidic protein, HA - hemagglutinin, NP- nucleocapsid protein, NA - neuraminidase, M - matrix proteins, NS - nonstructural proteins) across the three dilution series. Coverage of the Hazara genome is plotted as total of all three genome segments for simplicity. (B) Representative coverage plot of influenza A genome segments from the dilution series 1 sample at 10&lt;;sup&gt;;4&lt;;/sup&gt;; influenza copies per ml.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-6.medium.gif" width="440" height="229"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-6.medium.gif" width="440" height="229"/>
           </noscript>
          </span>
         </a>
         <ul class="highwire-figure-links inline">
          <li class="download-fig first">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
          </li>
          <li class="new-tab last">
           <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F3/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
          </li>
         </ul>
        </div>
       </div>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">Coverage of influenza and Hazara genome segments achieved by Nanopore sequencing from pooled samples</span>
       <p id="p-17" class="first-child">(A) Data from three dilution series of pooled influenza positive samples, diluted with three separate negative sample pools to generate different titres of influenza virus. Each individual dilution was spiked with Hazara at 10
        <sup>4</sup> genome copies/ml. Proportion of genome covered at 1x depth is shown for each of the eight influenza genome segments (encoding: PB2 - polymerase subunit, PB1 - polymerase subunit, PA - polymerase acidic protein, HA - hemagglutinin, NP- nucleocapsid protein, NA - neuraminidase, M - matrix proteins, NS - nonstructural proteins) across the three dilution series. Coverage of the Hazara genome is plotted as total of all three genome segments for simplicity. (B) Representative coverage plot of influenza A genome segments from the dilution series 1 sample at 10
        <sup>4</sup> influenza copies per ml.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Influenza detection from individual clinical samples</h3>
     <p id="p-18">Having demonstrated our ability to retrieve influenza sequences from pooled influenza-positive material diluted with negative samples, we next applied our methods to individual anonymised clinical samples, 40 testing influenza-positive and 10 influenza-negative in the clinical diagnostic laboratory. Data yield varied between flow cells (range 2.5×10
      <sup>6</sup> - 13.2×10
      <sup>6</sup> reads from up to 12 multiplexed samples). Within flow cells, barcode performance was inconsistent when using a stringent, dual barcode, demultiplexing method [
      <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>]. From each clinical sample, the range of total reads generated was 1.0×10
      <sup>5</sup> to 2.4×10
      <sup>6</sup> (median 3.8×10
      <sup>5</sup>) (Table S1).
     </p>
     <p id="p-19">Reads mapping to either influenza A or B genomes were present in all 27 samples with Cycle threshold (Ct) &lt;;30 (range 6 to 274,955 reads). At Ct &gt;;30, 6/13 samples generated influenza reads (range 6 to 92,057), (difference between sensitivity at a Ct threshold of 30, p&lt;;0.0001, 
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4</a>) The highest Ct value at which any influenza reads were detected was Ct 36.8 (Sample 37; 17 reads influenza A). No reads classified as influenza were obtained from sequencing the ten GeneXpert-negative samples (Table S1). Based on this small dataset, sensitivity is 83% and specificity 100% (95% CI 67-93% and 69-100%, respectively). There was a strong correlation between Ct value and both reads per sample classified as influenza (R-squared =0.604) and number of influenza reads per million reads (R-squared =0.623) (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4</a>).
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Total and proportion of influenza reads derived by Nanopore sequencing of individual samples across a range of Ct values. Ct values were derived by testing using GeneXpert (Cepheid) in a clinical diagnostic laboratory. (A) Correlation between Ct value and total number of influenza reads generated. R2= 0.604, p-value = 2.47e-08. (B) Correlation between Ct value and number of influenza reads per million reads. R2= 0.623, p-value = 1.07e-08." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570647831" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Total and proportion of influenza reads derived by Nanopore sequencing of individual samples across a range of Ct values.&lt;;/span&gt;; Ct values were derived by testing using GeneXpert (Cepheid) in a clinical diagnostic laboratory. (A) Correlation between Ct value and total number of influenza reads generated. R&lt;;sup&gt;;2&lt;;/sup&gt;;= 0.604, p-value = 2.47e-08. (B) Correlation between Ct value and number of influenza reads per million reads. R&lt;;sup&gt;;2&lt;;/sup&gt;;= 0.623, p-value = 1.07e-08.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F4.medium.gif" width="440" height="204"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F4.medium.gif" width="440" height="204"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4:</span>
       <span class="caption-title">Total and proportion of influenza reads derived by Nanopore sequencing of individual samples across a range of Ct values.</span>
       <p id="p-20" class="first-child">Ct values were derived by testing using GeneXpert (Cepheid) in a clinical diagnostic laboratory. (A) Correlation between Ct value and total number of influenza reads generated. R
        <sup>2</sup>= 0.604, p-value = 2.47e-08. (B) Correlation between Ct value and number of influenza reads per million reads. R
        <sup>2</sup>= 0.623, p-value = 1.07e-08.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Detection of Hazara virus internal control</h3>
     <p id="p-21">Detection of the control virus (Hazara at 10
      <sup>4</sup> genome copies/ml) was highly variable, demonstrating that levels of background non-target RNA are a major source of inter-sample variation. Hazara reads per sample ranged from 0 to 13.5×10
      <sup>3</sup> (0-3.5×10
      <sup>4</sup> RPM) with a median of 70 (160 RPM) and mean of 706 (1.7×10
      <sup>3</sup> RPM) (Table S1). Four (8%) of 50 samples generated no detectable Hazara reads, two with high numbers of influenza reads (Sample 6: Ct 18.4, 1.5×10
      <sup>4</sup> influenza A reads and Sample 1: Ct 13.5, 1.5×10
      <sup>5</sup> influenza B reads) acting to dilute the control signal. The other two samples contained no detectable influenza reads (Sample 34: Ct 35.9, Sample 46, Flu negative). The lack of control detection therefore indicates a loss of assay sensitivity due to high levels of background nucleic acid present in some samples.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Comparison with Illumina sequencing</h3>
     <p id="p-22">A subset of 15 samples were selected from across the viral titre range and re-sequenced on an Illumina MiSeq. Proportions of reads generated that mapped to the influenza genome were similar across the two sequencing technologies (
      <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure S4</a>).
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Influenza phylogeny and typing</h3>
     <p id="p-23">We reconstructed the phylogeny using consensus sequences for the HA gene (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5</a>). This demonstrates closely related sequences, as expected within one location in a single influenza season, and complete concordance between the sequences obtained by Nanopore vs Illumina for the subset of isolates sequenced by both technologies.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic trees of consensus Influenza HA gene derived by Nanopore and Illumina sequencing. Maximum likelihood tree generated using 500 bootstrap replicates in RAxML v8.2.10 software. Bootstrap values &gt;;70 are shown. Scale bar shows substitutions per site. Red and blue indicate sequences derived from Nanopore and illumina sequencing, respectively. References sequences are shown in black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570647831" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Phylogenetic trees of consensus Influenza HA gene derived by Nanopore and Illumina sequencing.&lt;;/span&gt;; Maximum likelihood tree generated using 500 bootstrap replicates in RAxML v8.2.10 software. Bootstrap values &gt;;70 are shown. Scale bar shows substitutions per site. Red and blue indicate sequences derived from Nanopore and illumina sequencing, respectively. References sequences are shown in black.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F5.medium.gif" width="440" height="264"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F5.medium.gif" width="440" height="264"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5:</span>
       <span class="caption-title">Phylogenetic trees of consensus Influenza HA gene derived by Nanopore and Illumina sequencing.</span>
       <p id="p-24" class="first-child">Maximum likelihood tree generated using 500 bootstrap replicates in RAxML v8.2.10 software. Bootstrap values &gt;;70 are shown. Scale bar shows substitutions per site. Red and blue indicate sequences derived from Nanopore and illumina sequencing, respectively. References sequences are shown in black.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Detection of other RNA viruses in clinical samples</h3>
     <p id="p-25">Within the 50 clinical samples sequenced, we found limited evidence of other RNA viruses. Sample 6 produced 109 reads mapping to Human Coronavirus in addition to &gt;;1.5×10
      <sup>4</sup> influenza A reads, suggesting co-infection. We also derived &gt;;4.0×10
      <sup>4</sup> reads from Human Metapneumovirus from an influenza-negative sample, providing a near complete (99.8%) genome from one sample (
      <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure S1</a>, Sample I), further detailed here [
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>].
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Animal time-course study</h3>
     <p id="p-26">Finally, we used samples collected from a previous animal experiment [
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>] to test the reproducibility of our methods across a time-course model of influenza A infection (three ferrets pre-infection (day -3) and then at days 1, 2, 3 and 5 following laboratory infection with influenza A). The proportion of viral reads present at each timepoint was highly congruent with viral titre (titre shown in 
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6A</a> and sequencing reads in 
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig 6B</a>). Consensus genome sequences were generated from days two, three and five and were 100% concordant with Illumina-derived consensus sequences from the same cDNA.
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Time course experiment showing influenza A infection in three laboratory ferrets. Infection was introduced at day 0. Samples were collected three days prior to infection, and at days 1, 3 and 5 post infection. (A) Influenza titre (log scale) and (B) proportion of total nanopore reads (linear scale) mapping to influenza A from metagenomic sequencing of ferret nasal washes taken pre and post influenza challenge." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1570647831" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Time course experiment showing influenza A infection in three laboratory ferrets.&lt;;/span&gt;; Infection was introduced at day 0. Samples were collected three days prior to infection, and at days 1, 3 and 5 post infection. (A) Influenza titre (log scale) and (B) proportion of total nanopore reads (linear scale) mapping to influenza A from metagenomic sequencing of ferret nasal washes taken pre and post influenza challenge.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F6.medium.gif" width="440" height="332"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F6.medium.gif" width="440" height="332"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/06/19/676155/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6:</span>
       <span class="caption-title">Time course experiment showing influenza A infection in three laboratory ferrets.</span>
       <p id="p-27" class="first-child">Infection was introduced at day 0. Samples were collected three days prior to infection, and at days 1, 3 and 5 post infection. (A) Influenza titre (log scale) and (B) proportion of total nanopore reads (linear scale) mapping to influenza A from metagenomic sequencing of ferret nasal washes taken pre and post influenza challenge.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">DISCUSSION</h2>
    <p id="p-28">To our knowledge, this is the first report of successfully applying metagenomic Nanopore sequencing directly to respiratory samples to detect influenza virus and generate influenza sequences. The approach demonstrates excellent specificity; sensitivity varies by viral titre, but is comparable to existing laboratory diagnostic tests for Ct values &lt;;30. Our optimised protocol depletes human and bacterial nucleic acids, and reduces the time from sample to sequence. This method has the potential to be further optimised and validated to improve sensitivity for influenza, identify other RNA viruses, detect drug-resistance mutations, and provide insights into quasispecies diversity [
     <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>,
     <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. At a population level, this sequence-based diagnostic data can, in addition, provide phylo-epidemiological reconstruction, insights into transmission events, potential for estimating vaccine efficacy [
     <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>], and approaches for public health intervention [
     <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>].
    </p>
    <p id="p-29">Whole genome viral sequencing can accurately trace outbreaks across time and space [
     <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. The metagenomic method employed here produced &gt;;90% complete genomes for 17/27 samples with a Ct value ≤30, demonstrating the ability of metagenomics to produce sufficient data for influenza diagnostics and genome characterisation, whilst also detecting and sequencing other common RNA viruses.
    </p>
    <p id="p-30">Despite time reductions in wet laboratory processing, this method requires further modification to simplify and accelerate the protocol if it is to become viable as a near-to-patient test. High error rates are a recognised concern in Nanopore sequence data, and cross-barcode contamination can create challenges when low and high titre samples are batched together [
     <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>]. To avoid these problems, we batched samples according to Ct value, and applied stringent barcode demultiplexing criteria; however, this reduces the total data available for analysis [
     <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>]. For future primary diagnostic use it would be preferable to sequence samples individually using a lower throughput flow cell e.g. ONT Flongle. Careful optimisation of laboratory and bioinformatic methods are required to resolve individual sequence polymorphisms, particularly for drug resistance alleles.
    </p>
    <p id="p-31">IDSA guidelines [
     <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">9</a>] recommend nasal/nasopharyngeal specimens for influenza diagnosis, but throat swabs are easier to collect in clinical practice and therefore accounted for the majority of our diagnostic samples. Further work is needed to investigate the sensitivity and specificity of our protocol for a wider array of respiratory sample types (also including bronchoalveolar lavage, sputum and saliva), which may yield different degrees of contaminating bacterial and/or human reads. Loss of assay sensitivity due to the presence of high-level background DNA from either host or bacterial origin is a fundamental issue for metagenomic approaches, even in cell free sample types such as cerebrospinal-fluid [
     <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. This challenge is exacerbated in throat swabs, as seen in our data. Our use of hazara virus as an internal positive control allows us to identify those sample in which sensitivity has dropped below 10
     <sup>4</sup> viral genome copies per ml. In our test set 8% of samples showed insufficient sensitivity for hazara virus, however half of these contained a high titre of influenza, so only 4% were true sensitivity failures. This figure is in line with the reported 6% failure rate due to high background for RNA virus detection from a clinically validated metagenomic sequencing assay for pathogen detection in cerebrospinal-fluid [
     <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>]
    </p>
    <p id="p-32">At the higher Ct values in our clinical samples (Ct 30-40), the sensitivity of Nanopore sequencing was reduced compared to the current PCR-based test (Cepheid). Further optimisation will be required to maximise the diagnostic yield from this group of samples, without sacrificing specificity.</p>
    <p id="p-33">The correlation between Ct value and Nanopore reads confirms semi-quantitative output. Using samples from the ferret influenza model, collected under standardized laboratory conditions, demonstrated excellent reproducibility of viral read proportions at a given viral titre across biological replicates. However, we observed heterogeneity in output between clinical samples as well as between Nanopore flowcells, suggesting that the current platform is not yet sufficiently reliable for reproducibly generating quantitative data. In addition, the detection of positive controls can be impaired in high background samples also creating further uncontrolled variability in the test yield.</p>
    <p id="p-34">Future application of this method will involve real-time laboratory testing of respiratory samples, running the platform head-to-head with existing clinical diagnostics to further assess sensitivity and specificity, and using influenza sequence data to investigate transmission events. Identifying instances of nosocomial transmission may shed light on healthcare-acquired infection, thus helping to improve infection control practice. Assessment of diversity within deep sequence datasets provides an opportunity to investigate the relationship between within-host polymorphisms and clinical outcomes. Long-read sequences confer the potential advantage of identifying viral haplotypes and ascertaining the extent to which significant polymorphisms are transmitted together or independently [
     <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. We have shown that the method is robust for the identification of commonly circulating influenza strains in human populations, but further investigation is required to ascertain the extent to which it performs reliably in other (avian, animal) strains.
    </p>
    <p id="p-35">In summary, our methods show promise for generating influenza virus sequences directly from respiratory samples. The ‘pathogen agnostic’ metagenomic sequencing approach offers an opportunity for simultaneous testing for a wide range of potential pathogens, providing a faster route to optimum treatment and contributing to antimicrobial stewardship. Longer term, this approach has promise as a routine laboratory test, providing data to inform treatment, vaccine design and deployment, infection control policies, and surveillance.</p>
   </div>
   <div class="section" id="sec-15">
    <h2 class="">Methods</h2>
    <div id="sec-16" class="subsection">
     <h3>Study cohort and sample collection</h3>
     <p id="p-36">We collected respiratory samples from the clinical microbiology laboratory at Oxford University Hospitals NHS Foundation Trust, a large tertiary referral teaching hospital in the South East of England. We worked with anonymised residual material from throat and nose swabs generated as a result of routine clinical investigations, between January and May 2018. Samples were collected using a sterile polyester swab inoculated into 1-3 ml of sterile viral transport medium (VTM), using a standard approach as described on the CDC website [
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. Workflow is shown in 
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1</a>.
     </p>
     <p id="p-37">During the study, respiratory samples submitted to the clinical diagnostic laboratory were routinely tested by a PCR-based test using GeneXpert (Cepheid) to detect influenza A and B, and respiratory syncytial virus (RSV). Samples from patients in designated high-risk locations (haematology, oncology and critical care) were tested using Biofire FilmArray (Biomérieux) to detect an expanded panel of bacterial and viral pathogens. Quantitative data (Cycle threshold, Ct) were generated by the GeneXpert, and we used the Influenza Ct value to estimate viral titre in clinical samples. Using GeneXpert, up to 40 PCR cycles are performed before a sample is called negative (ie positives have Ct &lt;;40). Quantification was not available for Biofire results.</p>
     <p>For methodological assessment, we focused on four categories of samples: </p>
     <ul class="list-unord " id="list-1">
      <li id="list-item-1">
       <p id="p-39">Positive pool: we pooled 19 throat swab samples that had tested positive for influenza A in the clinical diagnostic laboratory to provide a large enough sample to assess reproducibility (
        <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig 1B</a>);
       </p>
      </li>
      <li id="list-item-2">
       <p id="p-40">Negative pools: we generated three pools of throat swab samples that had tested negative for influenza (consisting of 24, 38 and 38 individual samples respectively) (
        <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1B</a>);
       </p>
      </li>
      <li id="list-item-3">
       <p id="p-41">Individual positive samples: we included 40 individual samples (35 throat swabs and 5 nasal swabs) that had tested positive for influenza A or B, selected to represent the widest range of GeneXpert Ct values (13.5 to 39.3; valid test result range 12 to 40).</p>
      </li>
      <li id="list-item-4">
       <p id="p-42">Individual negative samples: we selected 10 individual throat swab samples that were influenza negative.</p>
      </li>
     </ul>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Quantification of viral RNA in samples</h3>
     <p id="p-43">We quantified viral titres in Hazara virus stocks and pooled influenza A positive throat swabs by qRT-PCR using previously described assays and standards [
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>].
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Optimisation of methods</h3>
     <p>Prior to establishing the protocol detailed in full below, we assessed the impact of two possible optimisation steps, as follows: </p>
     <ol class="list-romanlower " id="list-2">
      <li id="list-item-5">
       <p id="p-45">Centrifugation vs filtration: We investigated two approaches to depleting human/bacterial nucleic acid from our samples, filtration of the raw sample via a 0.4μm filter (Sartorius) before further processing, versus a hard spin (16,000 xg for 2 min). cDNA libraries for this comparison were produced as described previously [
        <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">21</a>].
       </p>
      </li>
      <li id="list-item-6">
       <p id="p-46">Reduced time cDNA synthesis: To assess the possibility of time-saving in the cDNA synthesis steps, we compared performance of the previously described protocol [
        <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">21</a>] to a modified version with two alterations, first using SuperScript IV (ThermoFisher) in place of Superscript III (ThermoFisher) for reverse transcription with incubation time reduced from 60min to 10min at 42°C, and secondly reducing the cDNA amplification PCR extension cycling time from 5min to 2min.
       </p>
      </li>
     </ol>
    </div>
    <div id="sec-19" class="subsection">
     <h3>Positive control</h3>
     <p id="p-47">Prior to nucleic acid extraction, each sample was spiked with Hazara virus virions to a final concentration of 10
      <sup>4</sup> genome copies per ml as a positive internal control. This is an enveloped negative-stranded RNA virus (genus Orthonairovirus, order Bunyavirales) with a tri-segmented genome of 11980, 4575 and 1677 nucleotides in length (Genbank 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=KP406723&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom">KP406723</a>-
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=KP406725&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom">KP406725</a>). It is non-pathogenic in humans, and would therefore not be anticipated to arise in any of our clinical samples. Cultured virions from an SW13 cell line were provided by the National Collection of Pathogenic Viruses (NCPV), catalogue no. 0408084v).
     </p>
    </div>
    <div id="sec-20" class="subsection">
     <h3>Nucleic acid extraction</h3>
     <p id="p-48">Samples were centrifuged at 16,000 xg for 2min. Supernatant was eluted, without disturbing pelleted material, and used in nucleic acid extraction. Total nucleic acid was extracted from 100 μl of supernatant using the QIAamp viral RNA kit (Qiagen) eluting in 50ul of H
      <sub>2</sub>O, followed by a DNase treatment with TURBO DNase (Thermo Fisher Scientific) at 37·C for 30min. RNA was purified and concentrated to 6μl using the RNA Clean &amp;; Concentrator™-5 kit (Zymo Research) following manufacturer’s instructions. Randomly amplified cDNA was prepared for each sample using a Sequence Independent Single Primer (SISPA) approach, adapted from our previously described workflow [
      <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">21</a>], based on the Round A/B methodology [
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. For reverse transcription, 4μl of RNA and 1 μl of Primer A (5’-GTTTCCCACTGGAGGATA-N9-3’, 40 pmol/μl) [
      <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>] were mixed and incubated for 5min at 65°C, then cooled to room temperature. First strand synthesis was performed by the addition of 2μl SuperScriptIV First-strand Buffer, 1μL 12.5mM dNTPs, 0. 5μL 0.1 M DTT, 1μl H
      <sub>2</sub>O, and 0.5μL SuperScriptIV (ThermoFisher) before incubation for l0min at 42°C. Second strand synthesis was performed by addition of 1μl Sequenase Buffer, 3.85μl H
      <sub>2</sub>O and 0.15μl Sequenase (Affymetrix) prior to incubation for 8min at 37°C, followed by addition of 0.45μl Sequenase Dilution Buffer, 0.15μl Sequenase and a further incubation at 37°C for 8min. Amplification of cDNA was performed in triplicate using 5μl of the reaction as input to a 50μl AccuTaq LA reaction (Sigma) according to manufacturer’s instructions, using 1μl Primer B (5’-GTTTCCCACTGGAGGATA-3’) [
      <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>], with PCR cycling conditions of: 98°C for 30s; 30 cycles of 94°C for 15s, 50°C for 20s, and 68°C for 2min, followed by 68°C for 10min. Amplified cDNA was pooled from the triplicate reactions and purified using a 1:1 ratio of AMPure XP beads (Beckman Coulter, Brea, CA) and quantified by Qubit High Sensitivity dsDNA kit (Thermo Fisher), both according to manufacturer’s instructions.
     </p>
    </div>
    <div id="sec-21" class="subsection">
     <h3>Nanopore Library Preparation and Sequencing</h3>
     <p id="p-49">Multiplex sequencing libraries were prepared using 250ng of cDNA from up to 12 samples as input to the SQK-LSK108 or SQK-LSK109 Kit, barcoded individually using the EXP-NBD103 Native barcodes (Oxford Nanopore Technologies) and a modified “One-pot” protocol (Quick 2018). Libraries were sequenced on FLO-MIN106 flow cells on the MinION Mk1b or GridION device (Oxford Nanopore Technologies), with sequencing proceeding for 48h. Samples were batched according to the GeneXpert Ct value (see supplementary data file 1).</p>
    </div>
    <div id="sec-22" class="subsection">
     <h3>Illumina methods</h3>
     <p id="p-50">Nextera XT V2 kit (Illumina) sequencing libraries were prepared using 1.5ng of amplified cDNA as per manufacturer’s instructions and sequenced on a 2×150 bp-paired end Illumina MiSeq run, by Genomics Services Development Unit, Public Health England. Read mapping and data analysis as described previously [
      <a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">21</a>]
     </p>
    </div>
    <div id="sec-23" class="subsection">
     <h3>Bioinformatic analysis</h3>
     <p id="p-51">Nanopore reads were basecalled using Guppy (Oxford Nanopore Technology, Oxford, UK). Output basecalled fastq files were demultiplexed using Porechop v0.2.3 (
      <a href="https://github.com/rrwick/Porechop">https://github.com/rrwick/Porechop</a>). Reads were first taxonomically classified against RefSeq database using Centrifuge v1.0.3. Reads were then mapped against the reference sequence selected from Centrifuge report using Minimap2 v2.9. A draft consensus sequence was generated by using a majority voting approach to determine the nucleotide at each position. The resulting draft consensus sequences were BLASTed against a flu sequence database that included &gt;;2,000 H1N1 and H3N2 seasonal influenza sequences between 2018 and 2019, and downloaded from the Influenza Research Database [
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Reads were again mapped against the reference sequence using Minimap2 v2.9 and the number of mapped reads were calculated using samtools v1.5 and Pysam (
      <a href="https://github.com/pysam-developers/pysam">https://github.com/pysam-developers/pysam</a>). The subtype of the influenza A virus derived from each clinical sample was determined by the subtype of HA and NA reference sequence. Consensus sequence was built using Nanopolish v0.11.0 and margin_cons.py script (
      <a href="https://github.com/zibraproject/zika-pipeline">https://github.com/zibraproject/zika-pipeline</a>).
     </p>
    </div>
    <div id="sec-24" class="subsection">
     <h3>Ferret study</h3>
     <p id="p-52">We applied our sequencing approach to residual samples collected in a previous time-course experiment undertaken in a controlled laboratory environment [
      <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>]. We tested ferret nasal saline wash samples from three independent animals over an eight day time course, from three days prior to first exposure with influenza H1N1pdm09 and at days 1, 2, 3 and 5 post-infection. Sampling and plaque assays of viral titre were described previously [
      <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>].
     </p>
    </div>
    <div id="sec-25" class="subsection">
     <h3>Ethics approval</h3>
     <p id="p-53">The study of anonymised discarded clinical samples was approved by the London - Queen Square Research Ethics Committee (17/LO/1420). Ferret samples were residual samples from an existing study [
      <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">30</a>], for which the project license was reviewed by the local Animal Welfare and Ethics Review Board of Public Health England (Porton) and was subsequently granted by the Home Office.
     </p>
    </div>
    <div id="sec-26" class="subsection">
     <h3>Data availability</h3>
     <p id="p-54">Following removal of human reads, our sequence data have been uploaded to the European Bioinformatics Institute 
      <a href="https://www.ebi.ac.uk/">https://www.ebi.ac.uk/</a>, project reference PRJEB32861.
     </p>
    </div>
    <div id="sec-27" class="subsection">
     <h3>Funding</h3>
     <p id="p-55">The study was funded by the NIHR Oxford Biomedical Research Centre. Computation used the Oxford Biomedical Research Computing (BMRC) facility, a joint development between the Wellcome Centre for Human Genetics and the Big Data Institute supported by Health Data Research UK and the NIHR Oxford Biomedical Research Centre. The views expressed in this publication are those of the authors and not necessarily those of the NHS, the National Institute for Health Research, the Department of Health or Public Health England, PCM is funded by the Wellcome Trust (grant ref 110110). DWC, TEAP and ASW are NIHR Senior Investigators.</p>
    </div>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.1" data-doi="10.1038/s41572-018-0002-y">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Krammer</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>
          <span class="cit-name-given-names">GJD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fouchier</span>
          <span class="cit-name-given-names">RAM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peiris</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kedzierska</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Doherty</span>
          <span class="cit-name-given-names">PC</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Influenza</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Dis Primers</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">4</span>: 
         <span class="cit-fpage">3</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BDis%2BPrimers%26rft.volume%253D4%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41572-018-0002-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41572-018-0002-y&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Joseph</span>
          <span class="cit-name-given-names">U</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Su</span>
          <span class="cit-name-given-names">YCF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vijaykrishna</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>
          <span class="cit-name-given-names">GJD</span>
         </span>. 
         <span class="cit-article-title">The ecology and adaptive evolution of influenza A interspecies transmission</span>. 
         <abbr class="cit-jnl-abbrev">Influenza Other Respi Viruses</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">11</span>: 
         <span class="cit-fpage">74</span>–
         <span class="cit-lpage">84</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInfluenza%2BOther%2BRespi%2BViruses%26rft.volume%253D11%26rft.spage%253D74%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Humphreys</span>
          <span class="cit-name-given-names">M</span>
         </span>. 
         <span class="cit-article-title">The influenza of 1918: Evolutionary perspectives in a historical context</span>. 
         <abbr class="cit-jnl-abbrev">Evol Med Public Health</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">2018</span>: 
         <span class="cit-fpage">219</span>–
         <span class="cit-lpage">229</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEvol%2BMed%2BPublic%2BHealth%26rft.volume%253D2018%26rft.spage%253D219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.4" data-doi="10.1038/nature08182">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>
          <span class="cit-name-given-names">GJD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vijaykrishna</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bahl</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lycett</span>
          <span class="cit-name-given-names">SJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Worobey</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pybus</span>
          <span class="cit-name-given-names">OG</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza A epidemic</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">2009</span>;
         <span class="cit-vol">459</span>: 
         <span class="cit-fpage">1122</span>–
         <span class="cit-lpage">1125</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSmith%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D459%26rft.issue%253D7250%26rft.spage%253D1122%26rft.epage%253D1125%26rft.atitle%253DOrigins%2Band%2Bevolutionary%2Bgenomics%2Bof%2Bthe%2B2009%2Bswine-origin%2BH1N1%2Binfluenza%2BA%2Bepidemic.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08182%26rft_id%253Dinfo%253Apmid%252F19516283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature08182&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19516283&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000267636700042&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-676155v1.5" data-doi="10.3390/v10090461">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sutton</span>
          <span class="cit-name-given-names">TC</span>
         </span>. 
         <span class="cit-article-title">The Pandemic Threat of Emerging H5 and H7 Avian Influenza Viruses</span>. 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">10</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.3390/v10090461</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DViruses%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv10090461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3390/v10090461&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xiang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pu</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shen</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Evolutionary dynamics of avian influenza A H7N9 virus across five waves in mainland China, 2013-2017</span>. 
         <abbr class="cit-jnl-abbrev">J Infect</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">77</span>: 
         <span class="cit-fpage">205</span>–
         <span class="cit-lpage">211</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%26rft.volume%253D77%26rft.spage%253D205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-676155v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-article-title">Ten health issues WHO will tackle this year [Internet]</span>. [
         <span class="cit-date-in-citation">cited 22 Jan 2019</span>]. Available: 
         <a href="https://www.who.int/emergencies/ten-threats-to-global-health-in-2019">https://www.who.int/emergencies/ten-threats-to-global-health-in-2019</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-web" id="cit-676155v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span> | 
         <span class="cit-article-title">Burden of disease</span>. 
         <span class="cit-source">World Health Organization</span>; 
         <span class="cit-pub-date">2018</span>; Available: 
         <a href="https://www.who.int/influenza/surveillance_monitoring/bod/en/">https://www.who.int/influenza/surveillance_monitoring/bod/en/</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.9" data-doi="10.1093/cid/ciy866">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Uyeki</span>
          <span class="cit-name-given-names">TM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bernstein</span>
          <span class="cit-name-given-names">HH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bradley</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Englund</span>
          <span class="cit-name-given-names">JA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">File</span>
          <span class="cit-name-given-names">TM</span>
          <span class="cit-name-suffix">Jr</span>.
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fry</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical Practice Guidelines by the Infectious Diseases Society of America: 2018 Update on Diagnosis, Treatment, Chemoprophylaxis, and Institutional Outbreak Management of Seasonal Influenza</span>. 
         <abbr class="cit-jnl-abbrev">Clin Infect Dis</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/cid/ciy866</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciy866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/cid/ciy866&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.10" data-doi="10.1016/S0140-6736(17)33293-2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Iuliano</span>
          <span class="cit-name-given-names">AD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roguski</span>
          <span class="cit-name-given-names">KM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>
          <span class="cit-name-given-names">HH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Muscatello</span>
          <span class="cit-name-given-names">DJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palekar</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tempia</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Estimates of global seasonal influenza-associated respiratory mortality: a modelling study</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">391</span>: 
         <span class="cit-fpage">1285</span>–
         <span class="cit-lpage">1300</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D391%26rft.spage%253D1285%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252817%252933293-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(17)33293-2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-web" id="cit-676155v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span> | 
         <span class="cit-article-title">Burden of disease</span>. 
         <span class="cit-source">World Health Organization</span>; 
         <span class="cit-pub-date">2018</span>; Available: 
         <a href="https://www.who.int/influenza/surveillance_monitoring/bod/en/">https://www.who.int/influenza/surveillance_monitoring/bod/en/</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.12" data-doi="10.1016/j.tim.2017.09.004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Morris</span>
          <span class="cit-name-given-names">DH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gostic</span>
          <span class="cit-name-given-names">KM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pompei</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bedford</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Łuksza</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Neher</span>
          <span class="cit-name-given-names">RA</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Predictive Modeling of Influenza Shows the Promise of Applied Evolutionary Biology</span>. 
         <abbr class="cit-jnl-abbrev">Trends Microbiol</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">26</span>: 
         <span class="cit-fpage">102</span>–
         <span class="cit-lpage">118</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol%26rft.volume%253D26%26rft.spage%253D102%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2017.09.004%26rft_id%253Dinfo%253Apmid%252F29097090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tim.2017.09.004&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29097090&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.13" data-doi="10.1093/aje/kwv293">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Blanco</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eisenberg</span>
          <span class="cit-name-given-names">MC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stillwell</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Foxman</span>
          <span class="cit-name-given-names">B</span>
         </span>. 
         <span class="cit-article-title">What Transmission Precautions Best Control Influenza Spread in a Hospital?</span>
         <abbr class="cit-jnl-abbrev">Am J Epidemiol</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">183</span>: 
         <span class="cit-fpage">1045</span>–
         <span class="cit-lpage">1054</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAm%2BJ%2BEpidemiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faje%252Fkwv293%26rft_id%253Dinfo%253Apmid%252F27188950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/aje/kwv293&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27188950&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sansone</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wiman</span>
          <span class="cit-name-given-names">Å</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Karlberg</span>
          <span class="cit-name-given-names">ML</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brytting</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bohlin</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Andersson</span>
          <span class="cit-name-given-names">L-M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Molecular characterization of a nosocomial outbreak of influenza B virus in an acute care hospital setting</span>. 
         <abbr class="cit-jnl-abbrev">J Hosp Infect</abbr>. 
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">101</span>: 
         <span class="cit-fpage">30</span>–
         <span class="cit-lpage">37</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BHosp%2BInfect%26rft.volume%253D101%26rft.spage%253D30%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ito</span>
          <span class="cit-name-given-names">Y</span>
         </span>. 
         <span class="cit-article-title">Clinical Diagnosis of Influenza</span>. 
         <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">1836</span>: 
         <span class="cit-fpage">23</span>–
         <span class="cit-lpage">31</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1836%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.16" data-doi="10.1128/JCM.00711-18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Green</span>
          <span class="cit-name-given-names">DA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">StGeorge</span>
          <span class="cit-name-given-names">K</span>
         </span>. 
         <span class="cit-article-title">Rapid Antigen Tests for Influenza: Rationale and Significance of the FDA Reclassification</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">56</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JCM.00711-18</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00711-18%26rft_id%253Dinfo%253Apmid%252F29899007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjE1OiI1Ni8xMC9lMDA3MTEtMTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNi8xOS82NzYxNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-676155v1.17" data-doi="10.1128/JCM.01281-18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Benirschke</span>
          <span class="cit-name-given-names">RC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McElvania</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thomson</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kaul</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Das</span>
          <span class="cit-name-given-names">S</span>
         </span>. 
         <span class="cit-article-title">Clinical Impact of Rapid Point-of-Care Polymerase Chain Reaction Influenza Testing In An Urgent Care Setting: A Single Center Study</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. 
         <span class="cit-pub-date">2019</span>;
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JCM.01281-18</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.01281-18%26rft_id%253Dinfo%253Apmid%252F30602445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjE0OiI1Ny8zL2UwMTI4MS0xOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE5LzY3NjE1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.18" data-doi="10.1016/j.cmi.2018.12.017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schmidt</span>
          <span class="cit-name-given-names">RLJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simon</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Popow-Kraupp</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Laggner</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haslacher</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fritzer-Szekeres</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A novel PCR-based point-of-care method facilitates rapid, efficient, and sensitive diagnosis of influenza virus infection</span>. 
         <abbr class="cit-jnl-abbrev">Clin Microbiol Infect</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.cmi.2018.12.017</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClin%2BMicrobiol%2BInfect%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmi.2018.12.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cmi.2018.12.017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.19" data-doi="10.1128/JCM.02483-16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Votintseva</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bradley</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pankhurst</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Del</span>
          <span class="cit-name-given-names">Ojo</span>
         </span>. 
         <span class="cit-auth">
          <span class="cit-name-surname">Elias</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Loose</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nilgiriwala</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Same-Day Diagnostic and Surveillance Data for Tuberculosis via Whole-Genome Sequencing of Direct Respiratory Samples</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">55</span>: 
         <span class="cit-fpage">1285</span>–
         <span class="cit-lpage">1298</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.02483-16%26rft_id%253Dinfo%253Apmid%252F28275074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjU1LzUvMTI4NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE5LzY3NjE1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jia</span>
          <span class="cit-name-given-names">X-J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">H-H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fu</span>
          <span class="cit-name-given-names">X-F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ji</span>
          <span class="cit-name-given-names">J-M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">He</span>
          <span class="cit-name-given-names">P-Y</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Dynamic quantification of avian influenza H7N9(A) virus in a human infection during clinical treatment using droplet digital PCR</span>. 
         <abbr class="cit-jnl-abbrev">J Virol Methods</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">234</span>: 
         <span class="cit-fpage">22</span>–
         <span class="cit-lpage">27</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%2BMethods%26rft.volume%253D234%26rft.spage%253D22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.21" data-doi="10.2807/1560-7917.ES.2018.23.50.1800228">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kafetzopoulou</span>
          <span class="cit-name-given-names">LE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Efthymiadis</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lewandowski</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Crook</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carter</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Osborne</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Assessment of metagenomic Nanopore and Illumina sequencing for recovering whole genome sequences of chikungunya and dengue viruses directly from clinical samples</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">23</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.2807/1560-7917.ES.2018.23.50.1800228</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuro%2BSurveill%26rft.volume%253D23%26rft.spage%253D1800228%26rft_id%253Dinfo%253Adoi%252F10.2807%252F1560-7917.ES.2018.23.50.1800228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2807/1560-7917.ES.2018.23.50.1800228&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lewandowski</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lumley</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pullan</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vipond</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carroll</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Detection of Viral Pathogens With Multiplex Nanopore MinION Sequencing: Be Careful With Cross-Talk</span>. 
         <abbr class="cit-jnl-abbrev">Front Microbiol</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">2225</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront%2BMicrobiol%26rft.volume%253D9%26rft.spage%253D2225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Karamitros</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van Wilgenburg</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wills</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Klenerman</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Magiorkinis</span>
          <span class="cit-name-given-names">G</span>
         </span>. 
         <span class="cit-article-title">Nanopore sequencing and full genome de novo assembly of human cytomegalovirus TB40/E reveals clonal diversity and structural variations</span>. 
         <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">19</span>: 
         <span class="cit-fpage">577</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.24" data-doi="10.1186/s13073-015-0220-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Naccache</span>
          <span class="cit-name-given-names">SN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Federman</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mbala</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bres</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Rapid metagenomic identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis</span>. 
         <abbr class="cit-jnl-abbrev">Genome Med</abbr>. 
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">7</span>: 
         <span class="cit-fpage">99</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BMed%26rft.volume%253D7%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13073-015-0220-9%26rft_id%253Dinfo%253Apmid%252F26416663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s13073-015-0220-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26416663&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.25" data-doi="10.1101/470633">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McNaughton</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>
          <span class="cit-name-given-names">HE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bonsall</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Cesare</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mokaya</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lumley</span>
          <span class="cit-name-given-names">SF</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Illumina and Nanopore methods for whole genome sequencing of hepatitis B virus (HBV) [Internet]</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. 
         <span class="cit-pub-date">2018</span>. p. 
         <span class="cit-fpage">470633</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1101/470633</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F470633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0NzA2MzN2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE5LzY3NjE1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.26" data-doi="10.1038/s41598-018-32615-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Keller</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambo-Martin</span>
          <span class="cit-name-given-names">BL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilson</span>
          <span class="cit-name-given-names">MM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ridenour</span>
          <span class="cit-name-given-names">CA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shepard</span>
          <span class="cit-name-given-names">SS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stark</span>
          <span class="cit-name-given-names">TJ</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Direct RNA Sequencing of the Coding Complete Influenza A Virus Genome</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">8</span>: 
         <span class="cit-fpage">14408</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D14408%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-32615-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41598-018-32615-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Eckert</span>
          <span class="cit-name-given-names">SE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">JZ.M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Houniet</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth cit-collab">PATHSEEK consortium</span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Breuer</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Speight</span>
          <span class="cit-name-given-names">G</span>
         </span>. 
         <span class="cit-article-title">Enrichment by hybridisation of long DNA fragments for Nanopore sequencing</span>. 
         <abbr class="cit-jnl-abbrev">Microb Genom</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">2</span>: 
         <span class="cit-fpage">e000087</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrob%2BGenom%26rft.volume%253D2%26rft.spage%253D000087e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.28" data-doi="10.1101/585588">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rambo-Martin</span>
          <span class="cit-name-given-names">BL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Keller</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilson</span>
          <span class="cit-name-given-names">MM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nolting</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>
          <span class="cit-name-given-names">TK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vincent</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Mitigating Pandemic Risk with Influenza A Virus Field Surveillance at a Swine-Human Interface [Internet]</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. 
         <span class="cit-pub-date">2019</span>. p. 
         <span class="cit-fpage">585588</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1101/585588</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F585588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1ODU1ODh2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE5LzY3NjE1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.29" data-doi="10.1101/496687">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lewandowski</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lumley</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sanderson</span>
          <span class="cit-name-given-names">ND</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vaughan</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vipond</span>
          <span class="cit-name-given-names">RT</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Nanopore metagenomic sequencing of full length human metapneumovirus (HMPV) within a unique sub-lineage [Internet]</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. 
         <span class="cit-pub-date">2018</span>. p. 
         <span class="cit-fpage">496687</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1101/496687</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F496687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0OTY2ODd2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE5LzY3NjE1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gooch</span>
          <span class="cit-name-given-names">KE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marriott</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ryan</span>
          <span class="cit-name-given-names">KA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yeates</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Slack</span>
          <span class="cit-name-given-names">GS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brown</span>
          <span class="cit-name-given-names">PJ</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Heterosubtypic cross-protection correlates with cross-reactive interferon-gamma-secreting lymphocytes in the ferret model of influenza</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>. 
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">2617</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D2617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Meinel</span>
          <span class="cit-name-given-names">DM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Heinzinger</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eberle</span>
          <span class="cit-name-given-names">U</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ackermann</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schönberger</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sing</span>
          <span class="cit-name-given-names">A</span>
         </span>. 
         <span class="cit-article-title">Whole genome sequencing identifies influenza A H3N2 transmission and offers superior resolution to classical typing methods</span>. 
         <abbr class="cit-jnl-abbrev">Infection</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">46</span>: 
         <span class="cit-fpage">69</span>–
         <span class="cit-lpage">76</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInfection%26rft.volume%253D46%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.32" data-doi="10.1038/nature06945">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pybus</span>
          <span class="cit-name-given-names">OG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nelson</span>
          <span class="cit-name-given-names">MI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Viboud</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Taubenberger</span>
          <span class="cit-name-given-names">JK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">EC</span>
         </span>. 
         <span class="cit-article-title">The genomic and epidemiological dynamics of human influenza A virus</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">2008</span>;
         <span class="cit-vol">453</span>: 
         <span class="cit-fpage">615</span>–
         <span class="cit-lpage">619</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRambaut%26rft.auinit1%253DA.%26rft.volume%253D453%26rft.issue%253D7195%26rft.spage%253D615%26rft.epage%253D619%26rft.atitle%253DThe%2Bgenomic%2Band%2Bepidemiological%2Bdynamics%2Bof%2Bhuman%2Binfluenza%2BA%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06945%26rft_id%253Dinfo%253Apmid%252F18418375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature06945&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18418375&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000256185200035&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thompson</span>
          <span class="cit-name-given-names">CP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lourenço</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Walters</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Obolski</span>
          <span class="cit-name-given-names">U</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Edmans</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palmer</span>
          <span class="cit-name-given-names">DS</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A naturally protective epitope of limited variability as an influenza vaccine target</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. 
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">3859</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D3859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.34" data-doi="10.1126/science.aau9343">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kafetzopoulou</span>
          <span class="cit-name-given-names">LE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pullan</span>
          <span class="cit-name-given-names">ST</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ehichioya</span>
          <span class="cit-name-given-names">DU</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pahlmann</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Metagenomic sequencing at the epicenter of the Nigeria 2018 Lassa fever outbreak</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>. 
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">363</span>: 
         <span class="cit-fpage">74</span>–
         <span class="cit-lpage">77</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKafetzopoulou%26rft.auinit1%253DL.%2BE.%26rft.volume%253D363%26rft.issue%253D6422%26rft.spage%253D74%26rft.epage%253D77%26rft.atitle%253DMetagenomic%2Bsequencing%2Bat%2Bthe%2Bepicenter%2Bof%2Bthe%2BNigeria%2B2018%2BLassa%2Bfever%2Boutbreak%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aau9343%26rft_id%253Dinfo%253Apmid%252F30606844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNjMvNjQyMi83NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE5LzY3NjE1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.35" data-doi="10.1101/582957">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dudas</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bedford</span>
          <span class="cit-name-given-names">T</span>
         </span>. 
         <span class="cit-article-title">The ability of single genes vs full genomes to resolve time and space in outbreak analysis [Internet]</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. 
         <span class="cit-pub-date">2019</span>. p. 
         <span class="cit-fpage">582957</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1101/582957</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F582957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1ODI5NTd2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA2LzE5LzY3NjE1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.36" data-doi="10.1101/gr.238170.118">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Naccache</span>
          <span class="cit-name-given-names">SN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Samayoa</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Messacar</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Arevalo</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Federman</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Laboratory validation of a clinical metagenomic sequencing assay for pathogen detection in cerebrospinal fluid</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res</abbr>. 
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">29</span>: 
         <span class="cit-fpage">831</span>–
         <span class="cit-lpage">842</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.238170.118%26rft_id%253Dinfo%253Apmid%252F30992304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI5LzUvODMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDYvMTkvNjc2MTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-web" id="cit-676155v1.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">CDC</span>. 
         <span class="cit-article-title">Influenza (flu) including seasonal, avian, swine, pandemic, and other</span>. In: 
         <span class="cit-source">Centers for Disease Control and Prevention [Internet]</span>. 
         <span class="cit-day">29</span>
         <span class="cit-month">Mar</span>
         <span class="cit-pub-date">2019</span> [
         <span class="cit-date-in-citation">cited 2 Apr 2019</span>]. Available: 
         <a href="https://www.cdc.gov/flu">https://www.cdc.gov/flu</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lewandowski</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bell</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Miles</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carne</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wooldridge</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Manso</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The Effect of Nucleic Acid Extraction Platforms and Sample Storage on the Integrity of Viral RNA for Use in Whole Genome Sequencing</span>. 
         <abbr class="cit-jnl-abbrev">J Mol Diagn</abbr>. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">19</span>: 
         <span class="cit-fpage">303</span>–
         <span class="cit-lpage">312</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BDiagn%26rft.volume%253D19%26rft.spage%253D303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.39" data-doi="10.1371/journal.pone.0094090">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Marriott</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dove</span>
          <span class="cit-name-given-names">BK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Whittaker</span>
          <span class="cit-name-given-names">CJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bruce</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ryan</span>
          <span class="cit-name-given-names">KA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bean</span>
          <span class="cit-name-given-names">TJ</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Low dose influenza virus challenge in the ferret leads to increased virus shedding and greater sensitivity to oseltamivir</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>. 
         <span class="cit-pub-date">2014</span>;
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">e94090</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De94090%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0094090%26rft_id%253Dinfo%253Apmid%252F24709834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0094090&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24709834&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-676155v1.40" data-doi="10.1111/j.1750-2659.2011.00331.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Squires</span>
          <span class="cit-name-given-names">RB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Noronha</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hunt</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">García-Sastre</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Macken</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baumgarth</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Influenza research database: an integrated bioinformatics resource for influenza research and surveillance</span>. 
         <abbr class="cit-jnl-abbrev">Influenza Other Respi Viruses</abbr>. 
         <span class="cit-pub-date">2012</span>;
         <span class="cit-vol">6</span>: 
         <span class="cit-fpage">404</span>–
         <span class="cit-lpage">416</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInfluenza%2BOther%2BRespi%2BViruses%26rft.volume%253D6%26rft.spage%253D404%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1750-2659.2011.00331.x%26rft_id%253Dinfo%253Apmid%252F22260278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1750-2659.2011.00331.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22260278&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F06%2F19%2F676155.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000310270300021&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
