<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">mBio</span><span class="iso-abbrev" title="iso-abbrev">mBio</span><span class="hwp" title="hwp">mbio</span><span class="pmc" title="pmc">mbio</span><span class="publisher-id" title="publisher-id">mBio</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">mBio</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2150-7511</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Society for Microbiology</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">1752 N St., N.W., Washington, DC</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7157529">7157529</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/32209693">32209693</a></span><span class="publisher-id" title="publisher-id">mBio03068-19</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1128/mBio.03068-19">10.1128/mBio.03068-19</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Clinical Science and Epidemiology</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Localized Hypermutation is the Major Driver of Meningococcal Genetic Variability during Persistent Asymptomatic Carriage</div> <div class="alt-title" title="alt-title">
Localized Hypermutation in Meningococcal Carriage</div> <div class="alt-title" title="alt-title">
Green et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-2836-2041</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">Luke R.</span></span><a href="#aff1"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Rubaiawi</span><span class="given-names" tagx="given-names" title="given-names">Ali A.</span></span><a href="#aff1"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Al-Maeni</span><span class="given-names" tagx="given-names" title="given-names">Mohammad A. R. M.</span></span><a href="#aff1"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">Odile B.</span></span><a href="#aff2"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blades</span><span class="given-names" tagx="given-names" title="given-names">Matthew</span></span><a href="#aff3"><sup>c</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldfield</span><span class="given-names" tagx="given-names" title="given-names">Neil J.</span></span><a href="#aff4"><sup>d</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">David P. J.</span></span><a href="#aff4"><sup>d</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">Martin C. J.</span></span><a href="#aff2"><sup>b</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-5564-2145</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">Christopher D.</span></span><a href="#aff1"><sup>a</sup></a></span><span class="citation_author_institution" id="aff1">[a], <span class="addr-line" title="addr-line">Department of Genetics and Genome Biology, University of Leicester, Leicester, United Kingdom</span></span><span class="citation_author_institution" id="aff2">[b], <span class="addr-line" title="addr-line">Department of Zoology, University of Oxford, Oxford, United Kingdom</span></span><span class="citation_author_institution" id="aff3">[c], <span class="addr-line" title="addr-line">BBASH, University of Leicester, Leicester, United Kingdom</span></span><span class="citation_author_institution" id="aff4">[d], <span class="addr-line" title="addr-line">School of Life Sciences, University of Nottingham, Nottingham, United Kingdom</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pizza</span><span class="given-names" tagx="given-names" title="given-names">Mariagrazia</span></span><span class="role" tagx="role" title="role">Invited Editor</span><span class="citation_author_institution" id="">[], </span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rappuoli</span><span class="given-names" tagx="given-names" title="given-names">Rino</span></span><span class="role" tagx="role" title="role">Editor</span><span class="citation_author_institution" id="">[], </span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
 Address correspondence to Christopher D. Bayliss, <span class="email" tagx="email" title="email">cdb2@le.ac.uk</span>.</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-3-3</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e03068-19</span><span class="history" title="history"><span class="received" title="received">received: 2019-11-25</span><span class="accepted" title="accepted">accepted: 2020-2-24</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div><a class="self-uri" href="mBio.03068-19.pdf" title="self-uri">mBio.03068-19.pdf</a><div class="abstract" title="abstract"> <p>Many bacterial pathogens coexist with host organisms, rarely causing disease while adapting to host responses. <span class="named-content" title="named-content">Neisseria meningitidis</span>, a major cause of meningitis and septicemia, is a frequent persistent colonizer of asymptomatic teenagers/young adults. To assess how genetic variation contributes to host persistence, whole-genome sequencing and hypermutable sequence analyses were performed on multiple isolates obtained from students naturally colonized with meningococci. High frequencies of gene transfer were observed, occurring in 16% of carriers and affecting 51% of all nonhypermutable variable genes. Comparative analyses showed that hypermutable sequences were the major mechanism of variation, causing 2-fold more changes in gene function than other mechanisms. Genetic variation was focused on genes affecting the outer membrane, with directional changes in proteins responsible for bacterial adhesion to host surfaces. This comprehensive examination of genetic plasticity in individual hosts provides a significant new platform for rationale design of approaches to prevent the spread of this pathogen.</p> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
ABSTRACT</div> <p>Host persistence of bacteria is facilitated by mutational and recombinatorial processes that counteract loss of genetic variation during transmission and selection from evolving host responses. Genetic variation was investigated during persistent asymptomatic carriage of <span class="named-content" title="named-content">Neisseria meningitidis</span>. Interrogation of whole-genome sequences for paired isolates from 25 carriers showed that <i>de novo</i> mutations were infrequent, while horizontal gene transfer occurred in 16% of carriers. Examination of multiple isolates per time point enabled separation of sporadic and transient allelic variation from directional variation. A comprehensive comparative analysis of directional allelic variation with hypermutation of simple sequence repeats and hyperrecombination of class 1 type IV pilus genes detected an average of seven events per carrier and 2:1 bias for changes due to localized hypermutation. Directional genetic variation was focused on the outer membrane with 69% of events occurring in genes encoding enzymatic modifiers of surface structures or outer membrane proteins. Multiple carriers exhibited directional and opposed switching of allelic variants of the surface-located Opa proteins that enables continuous expression of these adhesins alongside antigenic variation. A trend for switching from PilC1 to PilC2 expression was detected, indicating selection for specific alterations in the activities of the type IV pilus, whereas phase variation of restriction modification (RM) systems, as well as associated phasevarions, was infrequent. We conclude that asymptomatic meningococcal carriage on mucosal surfaces is facilitated by frequent localized hypermutation and horizontal gene transfer affecting genes encoding surface modifiers such that optimization of adhesive functions occurs alongside escape of immune responses by antigenic variation.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
KEYWORDS</div> <div class="kwd" title="kwd"><i>Neisseria meningitidis</i></div> <div class="kwd" title="kwd">
horizontal gene transfer</div> <div class="kwd" title="kwd">
localized hypermutation</div> <div class="kwd" title="kwd">
meningitis</div> <div class="kwd" title="kwd">
meningococcus</div> <div class="kwd" title="kwd">
phase variation</div> <div class="kwd" title="kwd">
whole-genome sequence</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Wellcome Trust</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/100004440</span></div> </div><span class="award-id" tagx="award-id" title="award-id">104992/Z/14/Z</span><span class="award-id" tagx="award-id" title="award-id">087622/Z/08/A</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">Martin C. J.</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">UK Research and Innovation | Medical Research Council (MRC)</span><span class="institution-id" title="institution-id">https://doi.org/10.13039/501100000265</span></div> </div><span class="award-id" tagx="award-id" title="award-id">MR/M020193/1</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">Christopher D.</span></span></span></div> </div> <div class="counts" title="counts"><span class="count" tagx="count" title="count" /><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="cover-date" value="March/April 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Persistence of colonizing organisms for extended periods in individual hosts perpetuates transmission and contributes to the spread of a diverse spectrum of protozoan, prokaryotic, and viral pathogens and commensals (<a href="#B1">1</a><a href="#B2">–</a><a href="#B3">3</a>). Persistence is antagonized by innate and adaptive immune responses of the host, causing frequent reductions in population size of the persisting population, and can lead to evasion of immune responses by rapid evolution of adaptive variants (<a href="#B4">4</a>). Reductions in population size due to nonselective bottlenecks or selection of fitness variants depletes the persisting population of genetic variation and reduces the capacity to respond to further selective pressures. This restricted genetic diversity is counteracted by <i>de novo</i> mutation, horizontal gene transfer (HGT), and localized hypermutation (LH), a descriptor of evolved mechanisms for rapid generation of genetic variation such as site-specific recombination, homologous recombination between multicopy allelic sequences, and simple sequence repeats (<a href="#B5">5</a>, <a href="#B6">6</a>). Understanding the relative contributions of these processes to "within-host�? evolution is important for discerning the key host-pathogen interactions that facilitate host persistence of bacterial pathogens and immune evasion.</p> <p>Multiple repetitive sequences and high levels of genetic variability are notable attributes of <span class="named-content" title="named-content">Neisseria meningitidis</span> (<a href="#B7">7</a>, <a href="#B8">8</a>). In many high-income countries, asymptomatic carriage of this species in the upper respiratory tract is prevalent in adolescents, with a peak at 19 years of age and rates of 30% to 60% in some subpopulations (<a href="#B9">9</a>). Disease rates are low (∼1 case/100,000), with infections mainly occurring in infants, except in sub-Saharan Africa, where attack rates during epidemics can reach 1 case/1,000. Lengths of host persistence of particular meningococcal clones range from a few weeks to &amp;gt;12 months (<a href="#B10">10</a>). Natural transformation, facilitated by a <i>Neisseria</i>-specific 12-mer DNA uptake sequence (DUS), is the major determinant of interlocus or genome variation by HGT and is estimated to exceed <i>de novo</i> mutation by ∼10-fold (<a href="#B11">11</a>). The ∼2-Mbp genomes of <span class="named-content" title="named-content">N. meningitidis</span> contain ∼1 DUS/1,000 nucleotides but are also replete with other repetitive elements and multicopy genes (<a href="#B8">8</a>). The latter includes four <i>opa</i> genes that are dispersed around the genome and loci containing arrays of complete and partial <i>maf</i> genes. The type IV pilus exemplifies a recombinatorial mechanism of LH with a G quadraduplex initiating high frequencies of recombination (gene conversion) between <i>pilE</i>, the expression locus for the major pilin, and multiple silent <i>pilS</i> genes encoding partial copies of this protein (<a href="#B12">12</a>). Simple sequence repeats (SSR) are also highly prevalent in meningococcal genomes, with an average of 45 tracts per genome (<a href="#B13">13</a>). These tracts are hypermutable due to a high propensity for slipped-strand mispairing during DNA replication and mediate changes in gene expression due to the SSRs being located in the reading frames or promoters of genes (<a href="#B7">7</a>, <a href="#B14">14</a>). This process, referred to as phase variation (PV), can generate high levels of genetic variation through combinatorial switches in the expression of multiple genes. Thus, <span class="named-content" title="named-content">N. meningitidis</span> has multiple mechanisms for rapid generation of genetic variation that are thought to support the adaptation and evolution of meningococci during within-host persistence of individual clones.</p> <p>Herd immunity increases the protection and cost effectiveness of meningococcal immunization programs by reducing circulation of disease-causing strains. Recent introduction of a conjugate MenACWY vaccine into UK teenagers and university entrants aims at reducing infections by cutting circulation of hyperinvasive MenW strains (<a href="#B15">15</a>). Similar ideas have been proposed for reducing circulation of MenB strains by immunization of teenagers and young adults with recombinant protein-based vaccines (<a href="#B16">16</a>). However, utilization of vaccines in the major carriage population may perturb the meningococcal population structure and drive evolution of vaccine escape mutants, as observed with pneumococcal immunization programs (<a href="#B17">17</a>, <a href="#B18">18</a>). To investigate the potential for within-host strain evolution, we have studied <i>de novo</i> mutation, recombination (gene conversion and HGT), and LH of meningococci during persistent meningococcal carriage in university students. This exploration of natural carriage provides the first comprehensive evidence that LH is the major driver of within-host evolution of this bacterial species and that directional genetic variation is biased toward genes affecting surface antigens.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
RESULTS</div> <div class="lowlevelsofgenicvariationoccurduringpersistentcarriageofmeningococci." title="sec"> <div class="title" tagx="title" title="title">
Low levels of genic variation occur during persistent carriage of meningococci.</div> <p>Although asymptomatic carriage is the natural state for <span class="named-content" title="named-content">N. meningitidis</span>, there is a limited understanding of the levels of genetic variation associated with persistence in individual carriers. High-coverage genome sequences were generated using Illumina HiSeq for two isolates of the same strain from each of 25 carriers and representing 1 to 6 months asymptomatic carriage of a range of <span class="named-content" title="named-content">N. meningitidis</span> clonal complexes (<a href="#fig1">Fig. 1</a>). Allelic variation within the coding regions of single-copy genes was detected in an average of five variable genes per carrier with an average genic variation rate per month of carriage of 7 × 10<sup>−4</sup> (i.e., a meningococcus of ∼2,000 genes will accumulate 1.4 variable genes for each month of carriage) (<a href="#tab1">Table 1</a>). Only one difference (single nucleotide or indel) was detected in the majority of variable genes (69% [95/138]), suggestive of <i>de novo</i> mutation (<a href="#tab1">Table 1</a>). However, the two isolates from carrier V54 contained one nucleotide allelic differences in 24 genes, indicative of multiple recombination events (estimated to between 19 and 24) with a highly homologous meningococcus. If carrier V54 is excluded, then the <i>de novo</i> mutation rate is 1.02 mutations/genome/month of carriage, of which 19% was due to indels (18/94). These single nucleotide changes exhibited a 3:1 bias toward nonsynonymous changes, suggestive of either diversifying selection or insufficient time for purifying selection to remove mutations with reduced fitness.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 1</span></a></div>   <p>Core genome phylogenetic tree for 25 pairs of short- and long-term persistent carriage isolates. Persistent carriage isolates were chosen from volunteers, all first-year students that were part of a longitudinal carriage study at the University of Nottingham. For 25 volunteers, one isolate was chosen from the first and last time point where persistent carriage was observed and subject to WGS by Illumina HiSeq. Analysis of the PV states of 14 SSR-containing genes was also performed on multiple isolates for 21 volunteers. (A) Sampling times, the distribution of carriers with respect to long-term (Lo; 5 to 6 months) and short-term (Sh; 1 to 3 months) carriage, and the number of carriers whose isolates were subject to WGS (genomic) and/or PV analysis. (B) Phylogenetic tree derived using a subset of the <i>Neisseria</i> core genes (<i>n</i> = 215) and iTOL from whole-genome sequences generated on an Illumina HiSeq platform. The specific volunteers (see <a href="#tab1">Table 1</a> for isolate names for each volunteer) in each group are: Lo1, V51, V58, V59, V73, V86, V88, V117, V128, V176, V185, V188, and V222; Lo2, V69, V82, and V96; Sh3, V43 and V93; Sh4, V52, V54, V64, V114, V124, V134, V138, and V199; Sh5, V113, and V115. Individual branches of the tree are labeled with the clonal complex (cc) and genogroup (capsule type). The volunteer for each isolate is also indicated (there is a single label where two or more isolates are on adjacent branches).</p>   </div> <div class="table-wrap_UNKNOWN" id="tab1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 1</span> <p>Allelic variation during persistent meningococcal carriage</p>  <div class="alternatives" title="alternatives"> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div>  <thead> <tr> <th rowspan="2" colspan="1">Clonal complex</th> <th rowspan="2" colspan="1">Volunteer no. (mo of carriage)<a href="#ngtab1.1"><sup><i>a</i></sup></a></th> <th rowspan="2" colspan="1">Isolates (genomes)<a href="#ngtab1.2"><sup><i>b</i></sup></a></th> <th colspan="2" rowspan="1">No. of genes <hr /> </th> <th rowspan="2" colspan="1">Mutability/mo of carriage (×10<sup>−4</sup>)</th> <th colspan="2" rowspan="1">No. of genes <hr /> </th> </tr> <tr> <th rowspan="1" colspan="1">Total<a href="#ngtab1.3"><sup><i>c</i></sup></a></th> <th rowspan="1" colspan="1">Variable<a href="#ngtab1.4"><sup><i>d</i></sup></a></th> <th rowspan="1" colspan="1">With 1 difference (indel)</th> <th rowspan="1" colspan="1">With &amp;gt;1 difference</th> </tr> </thead> <tbody> <tr> <td rowspan="8" colspan="1">cc174</td> <td rowspan="1" colspan="1">V51 (6)</td> <td rowspan="1" colspan="1">N51.1 (28248), N424.1 (28274)</td> <td rowspan="1" colspan="1">1,854</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">2.7</td> <td rowspan="1" colspan="1">3 (0)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V43 (2)</td> <td rowspan="1" colspan="1">N241.1 (28247), N349.1 (28273)</td> <td rowspan="1" colspan="1">1,845</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">2.7</td> <td rowspan="1" colspan="1">1 (1)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V52 (3)</td> <td rowspan="1" colspan="1">N52.1 (28249), N342.1 (28275)</td> <td rowspan="1" colspan="1">1,856</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V58 (6)</td> <td rowspan="1" colspan="1">N58.1 (28251), N429.1 (28277)</td> <td rowspan="1" colspan="1">1,846</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V59 (6)</td> <td rowspan="1" colspan="1">N59.1 (28252), N438.1 (28278)</td> <td rowspan="1" colspan="1">1,855</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">3.6</td> <td rowspan="1" colspan="1">4 (0)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V88 (6)</td> <td rowspan="1" colspan="1">N88.1 (28253), N449.1 (28279)</td> <td rowspan="1" colspan="1">1,860</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">4.5</td> <td rowspan="1" colspan="1">5 (2)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V138 (3)</td> <td rowspan="1" colspan="1">N138.1 (28254), N331.1 (28280)</td> <td rowspan="1" colspan="1">1,865</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">7.1</td> <td rowspan="1" colspan="1">4 (0)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V54 (3)</td> <td rowspan="1" colspan="1">N54.1 (28250), N343.1 (28276)</td> <td rowspan="1" colspan="1">1,840</td> <td rowspan="1" colspan="1">24</td> <td rowspan="1" colspan="1">43.4</td> <td rowspan="1" colspan="1">24 (2)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="4" colspan="1">cc167</td> <td rowspan="1" colspan="1">V64 (3)</td> <td rowspan="1" colspan="1">N64.1 (28255), N348.1 (28281)</td> <td rowspan="1" colspan="1">1,852</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">3.6</td> <td rowspan="1" colspan="1">2 (1)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V117 (6)</td> <td rowspan="1" colspan="1">N117.1 (28256), N417.1 (28282)</td> <td rowspan="1" colspan="1">1,850</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">3.6</td> <td rowspan="1" colspan="1">4 (1)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V124 (3)</td> <td rowspan="1" colspan="1">N124.1 (28257), N336.1 (28283)</td> <td rowspan="1" colspan="1">1,863</td> <td rowspan="1" colspan="1">4</td> <td rowspan="1" colspan="1">7.2</td> <td rowspan="1" colspan="1">4 (1)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V128 (6)</td> <td rowspan="1" colspan="1">N128.1 (28258), N420.1 (28284)</td> <td rowspan="1" colspan="1">1,846</td> <td rowspan="1" colspan="1">34</td> <td rowspan="1" colspan="1">30.7</td> <td rowspan="1" colspan="1">9 (2)</td> <td rowspan="1" colspan="1">25</td> </tr> <tr> <td rowspan="5" colspan="1">cc23</td> <td rowspan="1" colspan="1">V69 (5)</td> <td rowspan="1" colspan="1">N258.1 (28259), N431.1 (28285)</td> <td rowspan="1" colspan="1">1,820</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V93 (2)</td> <td rowspan="1" colspan="1">N264.1 (28260), N359.1 (28286)</td> <td rowspan="1" colspan="1">1,816</td> <td rowspan="1" colspan="1">6</td> <td rowspan="1" colspan="1">16.5</td> <td rowspan="1" colspan="1">6 (1)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V96 (5)</td> <td rowspan="1" colspan="1">N259.1 (28261), N445.1 (28287)</td> <td rowspan="1" colspan="1">1,833</td> <td rowspan="1" colspan="1">7</td> <td rowspan="1" colspan="1">7.6</td> <td rowspan="1" colspan="1">7 (3)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V222 (6)</td> <td rowspan="1" colspan="1">N222.1 (28262), N459.1 (28288)</td> <td rowspan="1" colspan="1">1,732</td> <td rowspan="1" colspan="1">8</td> <td rowspan="1" colspan="1">7.7</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">7</td> </tr> <tr> <td rowspan="1" colspan="1">V188 (6)</td> <td rowspan="1" colspan="1">N188.1 (28269), N462.1 (28295)</td> <td rowspan="1" colspan="1">1,863</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="3" colspan="1">cc60</td> <td rowspan="1" colspan="1">V114 (3)</td> <td rowspan="1" colspan="1">N114.1 (28263), N330.1 (28289)</td> <td rowspan="1" colspan="1">1,797</td> <td rowspan="1" colspan="1">8</td> <td rowspan="1" colspan="1">14.8</td> <td rowspan="1" colspan="1">7 (1)</td> <td rowspan="1" colspan="1">1</td> </tr> <tr> <td rowspan="1" colspan="1">V134 (3)</td> <td rowspan="1" colspan="1">N134.1 (28264), N333.1 (28290)</td> <td rowspan="1" colspan="1">1,791</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">3.8</td> <td rowspan="1" colspan="1">1 (0)</td> <td rowspan="1" colspan="1">1</td> </tr> <tr> <td rowspan="1" colspan="1">V185 (6)</td> <td rowspan="1" colspan="1">N185.1 (28265), N185.2 (28266), N456.1 (28291), N456.2 (28292)</td> <td rowspan="1" colspan="1">1,823</td> <td rowspan="1" colspan="1">5</td> <td rowspan="1" colspan="1">4.6</td> <td rowspan="1" colspan="1">4 (1)</td> <td rowspan="1" colspan="1">1</td> </tr> <tr> <td rowspan="2" colspan="1">cc1157</td> <td rowspan="1" colspan="1">V73 (6)</td> <td rowspan="1" colspan="1">N73.1 (28268), N450.1 (28294)</td> <td rowspan="1" colspan="1">1,838</td> <td rowspan="1" colspan="1">3</td> <td rowspan="1" colspan="1">2.7</td> <td rowspan="1" colspan="1">3 (0)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">V82 (5)</td> <td rowspan="1" colspan="1">N262.1 (28267), N446.1 (28293)</td> <td rowspan="1" colspan="1">1,843</td> <td rowspan="1" colspan="1">10</td> <td rowspan="1" colspan="1">9.0</td> <td rowspan="1" colspan="1">2 (0)</td> <td rowspan="1" colspan="1">8</td> </tr> <tr> <td rowspan="1" colspan="1">cc269</td> <td rowspan="1" colspan="1">V199 (3)</td> <td rowspan="1" colspan="1">N199.1 (28270), N378.1 (28296)</td> <td rowspan="1" colspan="1">1,947</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">1.7</td> <td rowspan="1" colspan="1">1 (1)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1">cc32</td> <td rowspan="1" colspan="1">V176 (6)</td> <td rowspan="1" colspan="1">N176.1 (28271), N408.1 (28297)</td> <td rowspan="1" colspan="1">1,974</td> <td rowspan="1" colspan="1">1</td> <td rowspan="1" colspan="1">0.8</td> <td rowspan="1" colspan="1">1 (1)</td> <td rowspan="1" colspan="1">0</td> </tr> <tr> <td rowspan="1" colspan="1" /> <td rowspan="1" colspan="1">V86 (6)</td> <td rowspan="1" colspan="1">N86.1 (28272), N447.1 (28298)</td> <td rowspan="1" colspan="1">1,794</td> <td rowspan="1" colspan="1">2</td> <td rowspan="1" colspan="1">1.9</td> <td rowspan="1" colspan="1">2 (0)</td> <td rowspan="1" colspan="1">0</td> </tr> </tbody> </table> <div class="graphic_UNKNOWN" /> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">a</span><p>Note that each pair of isolates is of the same sequence type and that carriage spanned between 2 and 6 months.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">b</span><p>Isolate number (number allocated to the genome sequence of that isolate in the PubMLST/neisseria database).</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">c</span><p>Total number of analyzed genes excluding truncated genes (average of 41/genome) and duplicated or multicopy genes (e.g., insertion elements [IS], <i>pilE</i>/<i>pilS</i>, and multigene families).</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">d</span><p>Gene sequences having one or more nucleotide differences between the two paired isolates.</p> </div> </div> </div> <p>We also observed a relatively high occurrence of HGT with three carriers (V82, V128, and V222) containing 93% (40/43) (<a href="#tab1">Table 1</a>) of the variable genes with &amp;gt;1 difference per coding region, suggesting that HGT between meningococci of different clonal complexes (ccs) occurred in 12% of carriers (3/25). The putative number of HGT events/carrier for V82, V128, and V222 ranges between 3 and 10 with an average recombination patch size of 2,905 bp (see <a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>). Another strain (N292.1, ST823, cc198) was isolated at an intermediate time point from carrier V128 and shown to be the donor for HGT into the cc167 isolates (N128.1 and N420.1) from this carrier (see <a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>). For V82, all recombination fragments had 100% homology to sequences from N64.1 (present in carrier V64), indicating that a cocolonizing cc167 strain was the most likely donor strain to the cc1157/MenE strains of this carrier (see <a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>). For V222, the recombination fragment carrying <i>tbp</i> sequences had very high homology to sequences from cc23 isolates obtained from carriers in 2015 and 2016 or patients in 2011 and 2012 (<a href="#B19">19</a>, <a href="#B20">20</a>), whereas homologous sequences in other strains could not be found for the other fragments (see <a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>). The average synonymous-to-nonsynonymous mutation (dS/dN) ratio for the genes with multiple allelic differences was 10.1, indicating that sequences transferred by HGT have been subject to purifying selection in the donor genomes.</p> </div> <div class="nodifferencesbetweenfunctionalgroupsintheextentofallelicvariation." title="sec"> <div class="title" tagx="title" title="title">
No differences between functional groups in the extent of allelic variation.</div> <p>No significant differences were detected in nucleotide diversity for variable genes with nonsynonymous allelic variation among five broad functional groups (see <a href="#figS1">Fig. S1</a> in the supplemental material). There was a significant deviation in the numbers of variable membrane-associated genes for all carriers and for mutation-only carriers (<a href="#tab2">Table 2</a>), suggesting that the mutations were occurring by chance within genes of greater than the average length for this functional group. There was also a lower level of significance for mutation only for genes involved in DNA metabolism. A statistically significant association was detected for indels in genes with membrane-associated functions (8/17; <i>P</i> value &amp;lt; 0.0001) with three carriers having identical 15-nucleotide deletions in <i>porB</i>, encoding a major outer membrane protein (OMP). Many indels were large (9 to 33 bp), suggestive of evolved indel-susceptible sequences in these genes. No significant differences were detected in genic variability between short (1 to 3 months) versus long (&amp;gt;3 months) persistent carriage; however, there was a significant difference for the four isolate pairs with putative HGT versus the other 21 strain pairs (<a href="#figS1">Fig. S1</a>), indicating that within-host HGT, but not host persistence <i>per se</i>, is the main driver of allelic variation. A high proportion (39% [14/36]) of the HGT events were associated with allelic variation in genes encoding an OMP or a surface-modifying enzyme (e.g., pilin glycosyl transferase), indicating a bias in selection for fixation of HGT events involving modifiers of surface-exposed molecules.</p> <div class="table-wrap_UNKNOWN" id="tab2" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 2</span> <p>Distribution of variable genes with nonsynonymous allelic variation by function<a href="#ngtab2.1"><sup><i>a</i></sup></a></p>  <div class="alternatives" title="alternatives"> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div>  <thead> <tr> <th rowspan="2" colspan="1">Functional group (no. in class per genome)</th> <th colspan="2" rowspan="1">No. (%) of variable genes<a href="#ngtab2.2"><sup><i>b</i></sup></a><hr /> </th> <th colspan="3" rowspan="1">Nucleotide diversity (×10<sup>−5</sup>)<a href="#ngtab2.2"><sup><i>b</i></sup></a><sup>,</sup><a href="#ngtab2.4"><sup><i>d</i></sup></a><hr /> </th> </tr> <tr> <th rowspan="1" colspan="1">All carriers</th> <th rowspan="1" colspan="1">Without HGT<a href="#ngtab2.3"><sup><i>c</i></sup></a></th> <th rowspan="1" colspan="1">All carriers</th> <th rowspan="1" colspan="1">Without HGT</th> <th rowspan="1" colspan="1">HGT only</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Membrane associated (180)</td> <td rowspan="1" colspan="1">25 (0.56)<a href="#ngtab2.5"><sup><i>e</i></sup></a></td> <td rowspan="1" colspan="1">12 (0.32)<a href="#ngtab2.5"><sup><i>e</i></sup></a></td> <td rowspan="1" colspan="1">5.1<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">0.19<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">27.5<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> </tr> <tr> <td rowspan="1" colspan="1">Metabolic functions (663)</td> <td rowspan="1" colspan="1">31 (0.19)<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">12 (0.07)<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">2.0<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">0.12<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">11.9<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> </tr> <tr> <td rowspan="1" colspan="1">Regulation (325)</td> <td rowspan="1" colspan="1">8 (0.1)<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">3 (0.04)<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">2.0<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">0.08<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">11.8<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> </tr> <tr> <td rowspan="1" colspan="1">DNA metabolism (104)</td> <td rowspan="1" colspan="1">7 (0.27)<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">6 (0.23)<a href="#ngtab2.7"><sup><i>g</i></sup></a></td> <td rowspan="1" colspan="1">0.2<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">0.16<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">0.1<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> </tr> <tr> <td rowspan="1" colspan="1">Unknown function (855)</td> <td rowspan="1" colspan="1">39 (0.18)<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">13 (0.06)<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">3.1<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">0.22<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> <td rowspan="1" colspan="1">13.8<a href="#ngtab2.6"><sup><i>f</i></sup></a></td> </tr> <tr> <td rowspan="1" colspan="1">Total (2,127)</td> <td rowspan="1" colspan="1">110 (0.21)</td> <td rowspan="1" colspan="1">46 (0.09)</td> <td rowspan="1" colspan="1">2.3</td> <td rowspan="1" colspan="1">0.15</td> <td rowspan="1" colspan="1">12.0</td> </tr> </tbody> </table> <div class="graphic_UNKNOWN" /> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">a</span><p>A list of variable genes with nonsynonymous changes (i.e., with an altered amino acid sequence) was derived by analysis of alignments of derived amino acid sequence for each variable gene detected in the 25 pairs of persistent meningococcal isolates (<a href="#tab1">Table 1</a>). These genes were classified into one of five functional groups based on known functions or a homology-based analysis. Variable genes were also separated by occurrence in a carrier with or without high levels of HGT, with these classifications predicted to separate genes whose variation occurred by recombination from mutation.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">b</span><p>The denominator for calculating the percentage of variable genes was determined by multiplying the total number of genes in the class by the number of carriers.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">c</span><p>Numbers of carriers in each group: all carriers, <i>n</i> = 25; without HGT, <i>n</i> = 21; HGT only, <i>n</i> = 4 (V54, V82, V128, and V222).</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">d</span><p>Nucleotide diversity was calculated by using the average diversity for all genes to estimate the number of variants in a gene of 1,000 bp and then dividing this number by the total length of genes in a class (calculated by multiplying the number of genes by 1,000).</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">e</span><p><i>P</i>, &amp;lt;0.001; chi-square test of observed versus expected.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">f</span><p>Not significant by a chi-square test of observed versus expected for the number of variable genes or in one-way analysis of variance (ANOVA) for nucleotide diversity.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">g</span><p><i>P</i>, &amp;lt;0.01, chi-square test of observed versus expected.</p> </div> </div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.2">10.1128/mBio.03068-19.2</a><span class="label" tagx="label" title="label">FIG S1</span><p>Comparison of the nucleotide diversity of genes with nonsynonymous and synonymous allelic diversity between paired meningococcal carriage isolates representing 2 to 6 months persistent asymptomatic carriage in one individual. Variable genes were identified by comparison of genic sequences. Genes were examined for differences in amino acid sequences and separated into those with 1 or more nonsynonymous differences (A) or only synonymous differences (B). Genes were then separated into one of four functional gene classes. Nucleotide diversity was calculated by dividing the numbers of differences in a gene by the total gene length, and an average was calculated for all variable genes in that class. This average number was then adjusted to account for the number of genes in each functional class and number of carriers analyzed. The average nucleotide diversity is presented for various classifications: all, all 25 pairs of isolates; all (no HGT), 21 carriers where no horizontal gene transfer was observed; all (only HGT), the four carriers were HGT was observed; short, 10 carriers representing 2 to 3 months persistent carriage; long, 15 carriers representing 5 to 6 months persistent carriage. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">FIG S1, PDF file, 0.2 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> </div> <div class="allelicvariationwithinameningococcalcarriagepopulation." title="sec"> <div class="title" tagx="title" title="title">
Allelic variation within a meningococcal carriage population.</div> <p>To examine within-time-point allelic variation, whole-genome sequencing (WGS) was performed on 9 to 10 isolates/time point for carrier V59. Allelic variation was detected in 16 genes, of which seven occurred in only one isolate (see <a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>). A network diagram shows how partial sweeps of 90% (i.e., 9/10 colonies) were observed for variants in two genes (<i>ddlB</i> and <i>mutS</i>) in the fourth time point and sweeps of 60 to 70% for three other genes in the third and fourth time points, while one genotype, or near-identical variant thereof, persisted across all 6 months of carriage (<a href="#fig2">Fig. 2A</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 2</span></a></div>   <p>Whole-genome variation during persistent carriage. Allelic variation in coding regions was derived from the whole-genome sequences of 12 to 39 isolates per carrier for six carriers. The distribution of the allelic variation and relatedness of isolates was visualized using GrapeTree and manual manipulation of branch positions. Isolates are color coded for time point of isolation (<a href="#fig1">Fig. 1</a>): red, first; blue, second; yellow, third; green, fourth. Genes subject to variation between each node are indicated using gene names or NEIS numbers (see <a href="https://pubmlst.org/neisseria/">https://pubmlst.org/neisseria/</a>) with alternate types of variation in the same gene indicated by a low dash and number. The clonal complexes of isolates were either cc174 (V51 and V59) or cc23 (V69, V93, V96, and V222).</p>   </div> <p>WGS on multiple colonies for five other carriers detected similar levels of allelic variation, with two showing stable persistence of an identical core-genome genotype for 5 to 6 months of carriage and transient appearance of variants (<a href="#fig2">Fig. 2B</a> to <a href="#fig2">F</a>). Two carriers (V96 and V222) exhibited evidence of directional changes in multiple genes (<a href="#fig2">Fig. 2E</a> and <a href="#fig2">F</a>). For V96, there was deletion of two genes (<i>NEIS1464</i> and <i>NEIS1469</i>) encoding the lactoferrin binding protein combined with allelic changes in <i>pilF</i>, followed by subsequent, concomitant single-nucleotide changes in five other genes with a variety of functions. In V222, there were three HGT events involving multiple genes, of which two were linked to variations in OMPs (i.e., OpaJ and TbpB, part of the transferrin binding protein). Allele-specific PCR on a further 12 variable genes for eight carriers (see <a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>) identified seven variant alleles occurring in only one isolate and four variant alleles in all isolates (<i>n</i> = 6) for either an intermediate or final time point. Examination of five putative HGT events in V128 indicated that only three were associated with sweeps for all isolates of the final time point. Finally, the 15-bp deletion in PorB produced near complete sweeps in all three carriers where this mutation was observed after 3 months of persistent carriage.</p> <p>These results indicated that allelic variation was split into four forms (<a href="#tab3">Table 3</a>): (i) sporadic, i.e., occurring in only 1 tested isolate of 6 or more, 43% (35/82 variable genes); (ii) transient, i.e., occurring at an intermediate time point or only in a subset of isolates, 17% (14/82); (iii) putative fixation, i.e., occurring in all isolates of the final observed time point, 27% (22/82); and (iv) potential fixation, i.e., the majority of isolates of the final two time points, 13% (11/82). Of the latter category, three encode OMPs (PorB, LbpA, and LbpB), four are on a recombination fragment associated with allelic variation of an Opa protein in strain V222, and another encodes a pilin modifier (PilF); thus, 73% (8/11) of the "fixed�? variation causes alterations in surface molecules, suggesting that adaptive variation is focused on surface structures during persistent meningococcal carriage.</p> <div class="table-wrap_UNKNOWN" id="tab3" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 3</span> <p>Analysis of the types and distribution of allelic variation observed for multiple meningococcal isolates and multiple time points during persistent carriage</p>  <div class="alternatives" title="alternatives"> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div>  <thead> <tr> <th rowspan="1" colspan="1">Allelic variation type<a href="#ngtab3.1"><sup><i>a</i></sup></a></th> <th rowspan="1" colspan="1">Incidence</th> <th rowspan="1" colspan="1">Variable genes (carriers)</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Sporadic</td> <td rowspan="1" colspan="1">35</td> <td rowspan="1" colspan="1"><i>NEIS0212</i> (V51), <i>pilQ</i> (V51), <i>pglA</i> (V51), <i>uvrA</i> (V54), <i>gltX</i> (V59), <i>rpoC</i> (V59), <i>NMB0830</i> (V59), <i>NMB1114</i> (V59), <i>NMB122</i>0 (V59), <i>sspA</i> (V59), <i>polA</i> (V59) <i>mutS</i> (V69), <i>NEIS1978</i> (V69), NEIS1339 (V93), NEIS0202 (V93), NEIS1009 (V93), <i>pabBC</i> (V93), <i>NEIS1623</i> (V93), <i>NEIS0202</i> (V96), <i>secG</i> (V96), <i>iga</i> (V96) <i>mafB1</i> (V96), <i>NEIS1640</i> (V96), <i>pldA</i> (V96), <i>NEIS1708</i> (V96), <i>hrpA</i> (V96), <i>yccA</i> (V96), <i>NMB0669</i> (V114), <i>ligA</i> (V114), <i>NMB0409</i> (V117), <i>NMB1288</i> (V138), <i>NMB0424</i> (V138), <i>pilG</i> (V138), <i>tspA</i> (V222), <i>NEIS2119</i> (V222)</td> </tr> <tr> <td rowspan="1" colspan="1">Transient</td> <td rowspan="1" colspan="1">14</td> <td rowspan="1" colspan="1"><i>pilU</i> (V59, V96), <i>NEIS007</i>1 (V59), <i>iga</i> (V59, V96), <i>clpA</i> (V59), <i>NMB0846</i> (V59), <i>ilvL</i> (V59), <i>MTFMT</i> (V96), <i>mutS</i> (V96), <i>NMB1485</i> (V114), <i>NMB0114</i> (V128), <i>suhB</i> (V128), <i>NMB0274</i> (V138)</td> </tr> <tr> <td rowspan="1" colspan="1">Putative fixation</td> <td rowspan="1" colspan="1">22</td> <td rowspan="1" colspan="1"><i>argC</i> (V51), <i>argF</i> (V54), <i>ddl</i> (V59), <i>NEIS0498</i> (V59), <i>mutS</i> (V59), <i>porB</i> (V64, V124), <i>NEIS0176</i> (V93), <i>hemD</i> (V93), <i>NEIS1555</i> (V93), <i>rplM</i> (V93), <i>NEIS0796</i> (V96), <i>NEIS1329</i> (V96), <i>tspA</i> (V96), <i>NEIS1978</i> (V96), <i>hemK</i> (V96), <i>NEIS2450</i> (V96), <i>NMB0920</i> (V114), <i>dnaB</i> (V117), <i>NMB0439</i> (V128), <i>aspC</i> (V128), <i>tbpB</i> (V222)</td> </tr> <tr> <td rowspan="1" colspan="1">Potential fixation</td> <td rowspan="1" colspan="1">11</td> <td rowspan="1" colspan="1"><i>lbpA</i> (V96), <i>lbpB</i> (V96), <i>pilF</i> (V96), <i>porB</i> (V117), <i>pip</i> (V222), <i>bamC</i> (V222), <i>dapA</i> (V222), <i>NEIS0908</i> (V222), <i>thiC</i> (V222), <i>NEIS2022</i> (V222), <i>NEIS2522</i> (V222)</td> </tr> </tbody> </table> <div class="graphic_UNKNOWN" /> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">a</span><p>Sporadic, occurring in only 1 tested isolate of 6 or more; transient, occurring at an intermediate time point or only in a subset of isolates; putative fixation, occurring in all isolates of the final time point; potential fixation, occurring in the majority of isolates of the final two time points.</p> </div> </div> </div> </div> <div class="extensiveintragenomicrecombinationoccursinthemajorpilinsubunitbutnotthemulticopyopagenes." title="sec"> <div class="title" tagx="title" title="title">
Extensive intragenomic recombination occurs in the major pilin subunit but not the multicopy <i>opa</i> genes.</div> <p>Variation in the hypervariable <i>pilE</i> gene was examined for multiple carriers by either bioinformatic comparisons of WGS data or sequencing of PCR products encompassing the <i>pilE</i> gene. No allelic variation was observed in <i>pilE</i> of cc174 isolates due to separation of the <i>pilE</i> gene and G quartet into different genomic regions. All other strains exhibited allelic variation in <i>pilE</i>, even after only 1 month of carriage (see <a href="#tabS1">Table S1</a>). Analysis of the nucleotide and amino acid sequences of multiple isolates per time point from two carriers (V117 and V222) indicated that every time point contained multiple PilE alleles and that complete switches in the PilE type between two time points were frequent (<a href="#fig3">Fig. 3</a>; see also <a href="#figS2">Fig. S2</a>). These data clearly demonstrate that allelic variation of PilE occurs at very high frequencies during persistent meningococcal carriage, as previously observed for gonococcal infections (<a href="#B21">21</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 3</span></a></div>   <p>High-level temporal and spatial allelic variation in the meningococcal PilE protein deduced from <i>pilE</i> genes. An alignment of the amino acid sequences of the PilE protein for 24 isolates from carrier V222 are shown. Sequences were derived by Sanger sequencing of PCR products spanning the <i>pilE</i> gene. Isolates were derived from four different time points representing up to 6 months of carriage, with 6 isolates per time point (<a href="#fig1">Fig. 1</a>).</p>   </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.1">10.1128/mBio.03068-19.1</a><span class="label" tagx="label" title="label">TABLE S1</span><p>Allelic variation in PilE amino acid sequences for longitudinal carriage isolates. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">Table S1, DOCX file, 0.03 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.3">10.1128/mBio.03068-19.3</a><span class="label" tagx="label" title="label">FIG S2</span><p>Allelic variation in the derived PilE amino acid sequences of multiple meningococcal isolates from a single carrier. The <i>pilE</i> gene was amplified from 19 meningococcal carriage isolates obtained from volunteer V117. The isolates were obtained at the first (2, N117), second (6, N284), third (5, N332), and fourth (6, N417) time points and represent initial, 1, 3, and 6 months persistent carriage. Gene sequences were translated and aligned using Clustal Omega. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">FIG S2, PDF file, 2.1 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> <p>Intragenomic recombination in the multicopy <i>opa</i> genes is presumed to occur by RecA-mediated recombination (<a href="#B22">22</a>). Genomic sequences were utilized to locate and derive specific primers for each <i>opa</i> gene. Four Opa-containing loci (termed <i>opaA</i>, <i>opaB</i>, <i>opaD</i>, and <i>opaJ</i>, herein) (<a href="#tab4">Table 4</a>) were found in every isolate, with multiple isolates having one or more identical Opa alleles (A. A. Al-Rubaiawi, unpublished data). Only one pair of <i>opa</i> loci in one isolate had undergone intragenomic recombination (A. A. Al-Rubaiawi, unpublished), while another had undergone gene conversion (V222) (A. A. Al-Rubaiawi, unpublished).</p> <div class="table-wrap_UNKNOWN" id="tab4" orientation="portrait" position="float"><span class="label" tagx="label" title="label">TABLE 4</span> <p>PV gene modules</p>  <div class="alternatives" title="alternatives"> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" span="1" /></div>  <thead> <tr> <th rowspan="1" colspan="1">Module</th> <th rowspan="1" colspan="1">Genes</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">LOS</td> <td rowspan="1" colspan="1"><i>lgtG</i> (<i>NEIS2011</i>), <i>NMB1255</i><a href="#ngtab4.1"><sup><i>a</i></sup></a></td> </tr> <tr> <td rowspan="1" colspan="1">Pilin modification</td> <td rowspan="1" colspan="1"><i>pglA</i> (<i>NEIS0213</i>), <i>pglE</i> (<i>NEIS0568</i>), <i>pglH</i> (<i>NEIS0400</i>), <i>pglI</i> (<i>NEIS0380</i>)</td> </tr> <tr> <td rowspan="1" colspan="1">Pilin modulation</td> <td rowspan="1" colspan="1"><i>pilC1</i> (<i>NEIS0371</i>)<i>, pilC2</i> (<i>NEIS0033</i>)</td> </tr> <tr> <td rowspan="1" colspan="1">Multicopy OMP-Opa</td> <td rowspan="1" colspan="1"><i>opaA</i> (<i>NMB0442</i>), <i>opaB</i> (<i>NMB1636</i>), <i>opaD</i> (<i>NMB1465</i>), <i>opaJ</i> (<i>NMB0926</i>)</td> </tr> <tr> <td rowspan="1" colspan="1">Single-copy OMPs<a href="#ngtab4.2"><sup><i>b</i></sup></a></td> <td rowspan="1" colspan="1"><i>fetA</i> (<i>NEIS1963</i>), <i>hmbR</i> (<i>NEIS1586</i>), <i>hpuA</i> (<i>NEIS1946</i>), <i>mspA</i> (<i>NEIS1974</i>), <i>nadA</i> (<i>NEIS1969</i>), <i>nalP</i>, <i>opc</i> (<i>NEIS2198</i>), <i>porA</i> (<i>NEIS1364</i>)</td> </tr> <tr> <td rowspan="1" colspan="1">RM</td> <td rowspan="1" colspan="1"><i>modA</i> (<i>NEIS1310</i>), <i>modB</i> (<i>NEIS1194</i>)</td> </tr> </tbody> </table> <div class="graphic_UNKNOWN" /> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">a</span><p>NMB, gene identifier for the MC58 genome sequence.</p> </div> <div class="fn-type-other" title="other"><span class="label" tagx="label" title="label">b</span><p>Data reported previously by Alamro et al. (<a href="#B23">23</a>).</p> </div> </div> </div> </div> <div class="variablefrequenciesofswitchinginpilin,los,andrmphasevariationmodulesduringpersistentmeningococcalcarriage." title="sec"> <div class="title" tagx="title" title="title">
Variable frequencies of switching in pilin, LOS, and RM phase variation modules during persistent meningococcal carriage.</div> <p>Previous analyses had detected frequent PV of eight single-copy (sc) phase-variable OMPs (<a href="#B23">23</a>, <a href="#B24">24</a>). These studies were extended to another 14 phase-variable genes in five PV modules whose PV is controlled by SSRs located within the reading frame (<a href="#tab4">Table 4</a>). These phase-variable genes were selected as major components of the <span class="named-content" title="named-content">N. meningitidis</span> core phasome (<a href="#B13">13</a>) and as genes with a known or high propensity to undergo PV (i.e., we excluded genes with only 3 to 4 repeats). The colocated <i>lgt</i> genes (e.g., <i>lgtA</i>, <i>lgtB</i>, etc. [<a href="#B25">25</a>]) were not analyzed due to problems with designing repeat-specific primers for these highly homologous genes.</p> <p>The pilin modulation module consists of two highly homologous <i>pilC</i> genes whose PV is mediated by poly(C) tracts located close to the ATG initiation codon (<a href="#B26">26</a>) and encode large 110-kDa proteins that modulate type IV pilus surface expression (<a href="#B27">27</a>). Using unique PCR primers for each <i>pilC</i> gene, repeat numbers were found to range from 8 to 15 with a modular length of 10 for both genes (see <a href="#dataS1">Data File S1</a>), while the average mutation rate was 0.085 mutations/gene/month. Unexpectedly, some cc174 strains had a second poly(C) tract of 10 to 14 repeats located in the central portion of <i>pilC1</i>. Following conversion of repeat numbers into predicted expression states, we observed that 62% and 24% of isolates express one or two PilC proteins, respectively, with a significant increase in expression during persistent carriage in cc174 isolates but stable or decreasing expression in other ccs (<a href="#fig4">Fig. 4A</a>). An important trend was for switching toward PilC2-only expression states in four of the lineages as a function of persistent carriage (see <a href="#figS3">Fig. S3</a>). Notably, five carriers exhibited some degree of opposing switching of the two <i>pilC</i> genes, while there was an overall switching rate of 31% for this module (i.e., 13/42 <i>pilC</i> genes exhibited an 80% to 100% switch in expression state) (<a href="#figS3">Fig. S3</a>). Variation within many of the time points, as measured using a Shannon factor-based diversity score, was between 0.2 and 0.7, suggesting that these changes are due to a mix of random drift and selection (<a href="#figS3">Fig. S3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 4</span></a></div>   <p>Combinatorial changes in PV modules during persistent meningococcal carriage. Repeat numbers were derived for each phase-variable gene by fragment analysis of PCR products spanning the relevant repeat tract. Expression states were derived from the repeat numbers and associations between repeat number and translational state ("on�? and "off�? coded as 1 and 0, respectively). Individual phasotypic scores for the initial and final point of observed carriage for each carrier were derived from analysis of ∼6 isolates per time point. Average phasotypic scores and standard deviations were derived from the following numbers of carriers for each clonal complex: 8, cc174; 3, cc167: 4, cc23; 5, cc60; and 1, cc32. Statistical differences between the initial and final times were determined using a Wilcoxon rank sum test in Prism. The PV modules consisted of the following genes: (A) pilin modulation, <i>pilC1</i> and <i>pilC2</i>; (B) pilin glycosylation, <i>pglA</i>, <i>pglE</i>, <i>pglH</i>, and <i>pglI</i>; (C) LOS, <i>lgtG</i> and <i>NMB1255</i>; (D) restriction modification, <i>modA</i> and <i>modB</i>. Note that the maximum phasotypic score occurs when all genes within a module are in an "on�? state (thus a module of four phase-variable genes has a maximum score of 4).</p>   </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.4">10.1128/mBio.03068-19.4</a><span class="label" tagx="label" title="label">FIG S3</span><p>Phasotypes for the pilin modulation module. (A) Proportions of isolates containing different combinations of expression of the <i>pilC1</i> and <i>pilC2</i> phase-variable genes (1 and 0 indicate "on�? and "off�? states, respectively). Percentages were calculated from total number of isolates analyzed at each time point: cc174, 47, 39, 37, 23; cc167, 14, 18, 18, 6; cc23, 6, 24, 18, 18; cc60, 30, 30, 12, 6; cc32, 6, 6, 6, 6. (B) Change in expression for <i>pilC1</i> and <i>pilC2</i> in each carrier between the initial and final time point. Values of 1 and 0, all observed isolates in "on�? or "off�? state, respectively. (C) Shannon diversity scores. Diversity scores were calculated from the combined expression scores of the <i>pilC1</i> and <i>pilC2</i> genes for multiple isolates per time point; 1, all isolates were different from each other; 0, all isolates had the same score. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">FIG S3, PDF file, 0.4 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> <p>For the pilin glycosylation module, we analyzed four phase-variable genes (<i>pglA</i>, <i>pglE</i>, <i>pglI</i>, and <i>pglH</i>) whose products vary glycans on the type IV pilus and other surface proteins between three different glycan extensions with and without acetylation (<a href="#B28">28</a>). Initiating glycan residues were predicted to vary, as isolates have either a <i>pglB</i> or <i>pglB2</i> gene (see reference <a href="#B29">29</a>) (<a href="#fig5">Fig. 5</a>). PV is controlled by a heptanucleotide repeat in <i>pglE</i> and by poly(G) tracts in the other genes. Phasotypes and linked phenotypes, derived from analysis of expression states, were confirmed for a subset of isolates by immunoblotting with glycan-specific antibodies (see <a href="#figS4">Fig. S4</a>). Comparative analyses indicated that all 16 possible phasotypes were detected in the eight carriers of cc174 isolates but only 2 to 7 phasotypes for the 13 carriers of cc23, cc167, cc60, and cc32 isolates, with the 1-1-0-0 and 1-1-0-1 phasotypes dominating in the early time points (<a href="#fig5">Fig. 5</a>). This diversity in cc174 isolates was due to differences between carriers and time points, as diversity within individual time points was low (see <a href="#figS5">Fig. S5</a>). Persistent carriage was associated with differential patterns of behavior; combinatorial <i>pgl</i> phasotypic scores exhibited significant increases in cc174 isolates but decreases in cc23 and cc60 isolates (<a href="#fig4">Fig. 4B</a>). The cc174 phenotypic pattern was complex (<a href="#fig5">Fig. 5</a>); thus, Gal-Gal-containing phasotypes rose over the first 3 months from 12% to 42% before falling back to 8% after a further 3 months of carriage. This fall coincided with a switch to glycoforms lacking Gal moieties (0-1-0-0; 46% isolates) or having a Glu extension (0-1-1-1 and 0-1-0-1; 24%). An explanation of this pattern is that the diGal motif initially masks an antigenic epitope and then becomes the main antigenic target within this clonal complex. Notably, the Gal-Gal extension produced by the combined action of <i>pglA</i> and <i>pglE</i> (phasotypes 1-1-1-1, 1-1-1-0, 1-0-1-1, and 1-0-1-0) was rarely observed, occurring in 0% to 6% of isolates across the four time points for the cc167/cc23/cc60/cc32 isolates (<a href="#fig5">Fig. 5</a>). These findings suggest that there is selection against this extended glycan form in most carriers.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 5</span></a></div>   <p>Putative phenotypic effects of phase-variable changes in the pilin glycosylation module. The longitudinal changes in the proportions of carriage isolates associated with the putative glycan structures of each phasotype are shown. The four-gene phasotype represents the expression states of the <i>pglA</i>, <i>pglI</i>, <i>pglE</i>, and <i>pglH</i> genes, the products of which are responsible for the addition of the first galactose, an acetyl group, a second galactose, and a glucose moiety, respectively, to the basal sugar. Addition of the second galactose by PglE requires prior addition of the first galactose by PglA, while PglH competes with PglA for addition of glucose to the same acceptor position. The basal sugar, GATDH or DATDH, is determined by PglB2 or PglB, respectively, and differs among clonal complexes. The percentage values were calculated by determining the number of isolates across multiple carriers with a specific phasotype and dividing by the total number of analyzed isolates for each time point. Boxes are colored according to the heat map shown by the bar between the lower panels and ranges from white (0% of isolates) to bright red (100% of isolates). Note that the cc60 and cc32 meningococci lacked the <i>pglH</i> gene, resulting in a three-gene phasotype for these lineages. The total numbers of isolates analyzed for each clonal complex for the four time points were as follows: cc174 (48, 40, 37, 24); cc167 (14, 14, 18, 6); cc23 (6, 24, 18, 18); cc60 (30, 30, 12, 6); and cc32 (6, 6, 6, 1).</p>   </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.5">10.1128/mBio.03068-19.5</a><span class="label" tagx="label" title="label">FIG S4</span><p>Western blot to determine the expression states of a subset of persistent carriage isolates. Whole-cell lysates were prepared from 15 meningococcal isolates obtained as part of this study. Lysates were subject to separation on a PAGE gel and then transferred to a nitrocellulose membrane. These membranes were probed with monoclonal antibodies specific for a range of different glycan extensions. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">FIG S4, PDF file, 0.4 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.6">10.1128/mBio.03068-19.6</a><span class="label" tagx="label" title="label">FIG S5</span><p>Shannon diversity scores for each time point for the pilin glycosylation module. The genes in this module and method for calculation of expression score is indicated in <a href="#fig5">Fig. 5</a>. The diversity was calculated as described for <a href="#figS3">Fig. S3</a>. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">FIG S5, PDF file, 0.1 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> <p>For the LOS PV module, we focused on two genes, <i>lgtG</i> and <i>NMB1255</i>. Repeat tracts ranged between 7 and 16 for these genes (<a href="#dataS1">Data File S1</a>). The cc23 and cc60 isolates exhibited higher levels of "on�? expression states than other isolates (see <a href="#figS6">Fig. S6</a>). A significant decrease in "on�? expression states of these genes in cc174 and cc60 carriers is indicative of selection against certain immunotypes (<a href="#fig4">Fig. 4</a>). PV of <i>NMB1255</i> was higher than for <i>lgtG</i>, while diversity within time points was generally low, suggesting that these LOS epitopes are subject to infrequent but strongly selective shifts in expression (see <a href="#figS6">Fig. S6</a>).</p> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.7">10.1128/mBio.03068-19.7</a><span class="label" tagx="label" title="label">FIG S6</span><p>Phasotypes for the LOS module (<i>lgtG</i> and <i>NMB1255</i>). (A) Percentages were calculated as described for <a href="#figS3">Fig. S3</a> for the following numbers of isolates: cc174, 48, 40, 37, 23; cc167, 14, 18, 18, 15; cc23, 6, 24, 18, 18; cc60, 30, 30, 12, 6; cc32, 6, 6, 6, 6. (B) Switching states for each volunteer for the LOS module. See the legend for <a href="#figS3">Fig. S3</a> for method for determination of expression states. (C) Shannon diversity scores for each time point. See the legend for <a href="#figS3">Fig. S3</a> for how diversity scores were derived. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">FIG S6, PDF file, 0.2 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> <p>An unexpected finding for PV was of phase-variable restriction modification (RM) systems (<a href="#B14">14</a>, <a href="#B30">30</a>). The cc174 isolates contained phase-variable <i>modA</i> and <i>modB</i> genes, while only the <i>modA</i> gene of cc167 and cc23 isolates contained an SSR. Both tetra- and pentanucleotide SSRs were present and contained 3 to 37 repeats (<a href="#dataS1">Data File S1</a>). Phase-variable <i>mod</i> genes were in "off�? states in most isolates, with only cc174 isolates exhibiting a significant shift in expression (<a href="#fig4">Fig. 4D</a>). However, a complete shift in expression of four <i>mod</i> genes (4/31 [13%]) (see <a href="#figS7">Fig. S7</a>) is suggestive of strong but infrequent selection for switches in expression of these genes. Conversely, high levels of heterogeneity in expression states (i.e., a Shannon diversity score of &amp;gt;0.4) were observed at multiple time points for several carriers (<a href="#figS7">Fig. S7</a>). Heterogeneity may arise due to selection for mixed populations or as result of high mutability in SSRs combined with nonspecific bottlenecks or mutational drift.</p> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.8">10.1128/mBio.03068-19.8</a><span class="label" tagx="label" title="label">FIG S7</span><p>Phasotypes for the Mod module (<i>modA</i> and <i>modB</i>). (A) Percentages were calculated as described for <a href="#figS3">Fig. S3</a> for the following numbers of isolates: cc174, 48, 40, 37, 24; cc167, 14, 18, 18, 6; cc23, 6, 24, 18, 18; cc60, 30, 30, 12, 6; cc32, 6, 6, 6, 5. (B) Switching states for each volunteer for the Mod module. See the legend for <a href="#figS3">Fig. S3</a> for how expression states were determined. Note that the cc167 and cc23 strains did not contain the <i>modB</i> gene. (C) Shannon diversity scores. See the legend for <a href="#figS3">Fig. S3</a> for how diversity scores were derived. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">FIG S7, PDF file, 0.2 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> </div> <div class="persistentmeningococcalcarriageisassociatedwithfrequentdirectionalswitchinginthemulticopyopagenes." title="sec"> <div class="title" tagx="title" title="title">
Persistent meningococcal carriage is associated with frequent directional switching in the multicopy <i>opa</i> genes.</div> <p>Pentanucleotide repeats of five or more were found in 83% (63/76) of <i>opa</i> genes (<a href="#dataS1">Data File S1</a>). Multiplex PCRs/GeneScan assays were designed to detect variation in SSRs of all four Opa proteins (A. A. Al-Rubaiawi, unpublished). The average Opa mutation rate per month of carriage was 0.11 with variability increasing with repeat number: 0.007 for ≤5, 0.09 for 6 to 8, and 0.16 for ≥9 (<a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>).</p> <p>Genome sequence analysis and resequencing of each SSR/flanking sequence enabled association of repeat number to expression state and derivation of four-gene Opa phasotypes (see <a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>). Variable patterns were detected between Opa alleles in a run of C and T residues located between the ATG and SSR plus insertion/deletion of a codon on either side of the repeat tract. Expression states for the phasotypes were confirmed by Western blotting using an Opa antiserum specific for a conserved domain (see <a href="#figS8">Fig. S8</a>). The phasotypes and individual gene expression states were examined for carriage-associated patterns. Expression of a single Opa was the dominant pattern occurring in 66% (260/392) of isolates (<a href="#fig6">Fig. 6A</a>). Contrastingly, only 16% of isolates did not express any Opa proteins, and no isolates exhibited expression of all four Opa proteins (<a href="#fig6">Fig. 6A</a>). Examination of phasotypic scores for four-gene Opa phasotypes indicated that expression of a single gene was maintained during persistent carriage (<a href="#fig6">Fig. 6B</a>). However, there was significant switching from low to high and high to low average Opa expression states (<a href="#fig6">Fig. 6C</a>). Furthermore, opposing switches in expression of individual Opa genes (i.e., "on�? to "off�? in one locus and "off�? to "on�? in another locus) were observed in 63% (12/19) of carriers. Two examples of opposing switching are for V222 wherein OpaJ switches "on�? while OpaB switches "off�? over 6 months carriage (<a href="#fig6">Fig. 6D</a>), whereas in V51 OpaA and OpaJ switch "off�? while OpaB switches "on�? (<a href="#fig6">Fig. 6E</a>). These overall and individual PV patterns are indicative of immune selection against an Opa variant(s) expressed by initial colonizing isolates accompanied by selection for expression of alternate functionally equivalent Opa alleles in order to maintain an adhesion phenotype.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 6</span></a></div>   <p>Directional and opposing PV of Opa proteins during longitudinal meningococcal carriage. The expression states of the Opa proteins were derived from analyses of gene sequences and pentanucleotide repeat numbers for all four loci of each isolate. (A) Percentages of 392 isolates from 19 carriers that had an "on�? expression state in none ("off�?) to one or more loci. (B) Average phasotypic scores for the Opa module for the initial and final time points of observed carriage for these carriers (note that the maximum score is 4; error bars are standard deviations). (C) Change in phasotypic scores for individual <i>opa</i> genes across multiple carriers for loci starting with a high level of isolates in either an "on�? or "off�? state. Loci were split by the overall expression state in the initial time point for each carrier into three categories: (i) 50% or more of isolates in the "on�? state (Opa-high, 24 loci) (top); (ii) &amp;lt;50% of isolates in the "on�? state (Opa-low, black line/open circles, 52 loci) (bottom); (iii) &amp;lt;50% of isolates in the "on�? state but excluding carriers where there was no change in expression between the initial and final time points (Opa-low, dotted line/open squares, 21 loci). (D and E) Changes in expression of individual Opa loci for multiple time points of two carriers. Phasotypic scores are the averages from multiple isolates per time point. Blue line, OpaA; orange line, OpaB; green line, OpaD; yellow line, OpaJ.</p>   </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.9">10.1128/mBio.03068-19.9</a><span class="label" tagx="label" title="label">FIG S8</span><p>Confirmation of <i>opa</i> gene expression states by Western blotting. Proteins from whole-cell lysates of carriage isolates of isolates with different repeat numbers and predicted expression states for the <i>opa</i> genes were separated by electrophoresis on 12% SDS-PAGE gels and transferred to polyvinylidene difluoride (PVDF) membranes. These membranes were probed with primary antibodies as follows: rabbit polyclonal anti-Opa antibodies, mouse monoclonal anti-PorA antibodies (cc174, P1.16; cc60, P1.2; cc167, P1.5), and mouse polyclonal anti-fHbp V1 antibodies. Numbers on the Opa blots refer to the repeat number of the four <i>opa</i> genes in the order <i>opaA</i>-<i>opaB</i>-<i>opaD</i>-<i>opaJ</i>, with red and black indicating genes in the phase "on�? and "off�? states, respectively. (A) cc174 isolates: lane 1, N59.3; 2, N54.1; 3, N369.1; 4, N88.1; 5, N352.2; 6, N424.1; 7, N354.1; 8, N51.1; 9, N138.1; 10, N272.1; 11, N253.1. (B) cc60 isolates: lane 1, N114.1; 2, N333.1; 3, N330.1; 4, N113.1; 5, 134.2. (C) cc167 isolates: lane 1, N332.2; 2, N348.1; 3, N117.1; 4, N284.1. (D) cc23 isolates: lane 1, N309.1; 2, N222.1; 3, N459.6; 4, N370.4; 5, N370.5. Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">FIG S8, TIF file, 0.8 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.1128/mBio.03068-19.10">10.1128/mBio.03068-19.10</a><span class="label" tagx="label" title="label">DATA FILE S1</span><p>This file contains lists of the phase variation and GeneScan data for all isolates for pilin modulation, pilin glycosylation, restriction modification, and Opa modules of phase-variable genes. The following information is listed for every gene for each isolate: size of PCR product as determined by GeneScan, derived repeat number (as determined from GeneScan relative to a product of known repeat number), and derived expression state (as determined from a bioinformatics analysis of the open reading frame or relative to other isolates of known expression state). Two additional items of information are provided for the Opa genes: size of G tract and size of A tract (these two tracts are adjacent to the repetitive sequence and influence the expression state). Download <span class="inline-supplementary-material" tagx="inline-supplementary-material" title="inline-supplementary-material">DATA SET S1, PDF file, 2.4 MB</span>.</p> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Green et al.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This content is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div> </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
DISCUSSION</div> <p>Mutators, genetic exchange, and LH have evolved in several bacterial species in response to specific pressures encountered during persistence in or transmission between host organisms (<a href="#B5">5</a>, <a href="#B31">31</a>, <a href="#B32">32</a>). To examine the relative contributions of different mutational processes to asymptomatic carriage, we analyzed genetic variation in multiple meningococcal isolates obtained from persistently colonized asymptomatic carriers. Samples were from a University of Nottingham carriage study and hence are representative of natural carriage of this species in the age group that is associated with the highest rates of asymptomatic carriage. Our sampling was performed ∼6 weeks after the start of the university term, such that most of the carriers were likely to have acquired carriage within that time period (note that carriers V69, V93, and V96 were negative for carriage at time point one) and have developed or were developing an immune response to surface antigens (see Alamro et al. [<a href="#B23">23</a>] for data on the immune responses and further discussion of acquisition times). Thus, our study is likely to be representative of the adaptive events that occur a few days after initial colonization of carriers by meningococci and, more importantly, when these organisms are subject to adaptive immune responses.</p> <div class="examinationofmultipleisolatesdistinguishesbetweensporadicandnonsporadicgeneticvariation." title="sec"> <div class="title" tagx="title" title="title">
Examination of multiple isolates distinguishes between sporadic and nonsporadic genetic variation.</div> <p>Meningococcal epidemiology has focused on measuring genetic variation over periods of months and years. However, this variation arises from processes occurring in individual carriers, which is poorly characterized. Recent studies by Pandey et al. (<a href="#B33">33</a>) and Borud et al. (<a href="#B34">34</a>) detected evidence of transient and fixed genetic variants during persistent neisserial carriage but were limited by having only one isolate per time point. Using samples from a 2008–2009 meningococcal carriage study wherein nasopharyngeal swabs were deliberately plated to yield multiple single colonies (see Alamro et al. [<a href="#B23">23</a>]), we performed both WGS and allele-specific PCR on 6 to 10 isolates/time point from multiple sequential time points from several volunteers. We found that genetic variants can be classified into one of four classes (sporadic, transient, putative fixation, and potential fixation) and that WGS of four isolates/time point differentiates sporadic from transient/fixed variants. Furthermore, transient variants were separable from fixed variants with three time points, while detection of fixation was enhanced by four time points. Thus, we recommend collection of ≥4 isolates for each of ≥4 time points for future longitudinal carriage studies.</p> </div> <div class="howfrequentaremutationandhgtduringpersistentmeningococcalcarriage?" title="sec"> <div class="title" tagx="title" title="title">
How frequent are mutation and HGT during persistent meningococcal carriage?</div> <p>While meningococcal mutation rates are low, high rates of turnover of colonizing populations are expected to provide ample time for mutation. Excluding SSR, our WGS analysis indicated an average of 1.02 mutations/genome/month of carriage (i.e., 6.1 × 10<sup>−6</sup> mutations/site/year). Comparable rates were obtained by Lamelas et al. (<a href="#B35">35</a>) for serogroup A meningococci (3.1 × 10<sup>−6</sup> substitutions/site/year), from analysis of 100 isolates representing an 8-year study in a single Ghanaian district, and Pandey et al. (<a href="#B33">33</a>) for <span class="named-content" title="named-content">Neisseria lactamica</span> (2.2 × 10<sup>−6</sup>; 27 volunteers) and <span class="named-content" title="named-content">N. meningitidis</span> (9.3 × 10<sup>−7</sup>; 7 volunteers), using longitudinal isolates from a volunteer study. Meningococcal mutation rates for rifampin resistance are 2.6 × 10<sup>−9</sup> mutations/division (<a href="#B36">36</a>), equivalent to 2.2 × 10<sup>−5</sup> mutations/site/year. The 4-fold differences between these rates suggests that stabilizing selection limits mutation accumulation. However, we observed that nonsynonymous mutations exceeded synonymous mutations. This indicates that stabilizing selection is weak over short periods of persistent carriage and that nonselective bottlenecks, due to expansion after nonspecific physical reductions in population size, could be a key factor in limiting accumulation of variation.</p> <p>HGT is a major driver of genetic variation in meningococcal populations. We found HGT in 16% of carriers, with 36 recombination events encompassing 51% of all nonhypermutable variable genes. This observation provides definitive evidence from individual infections that natural transformation between two phylogenetically divergent strains generates a high proportion of genetic variation in this bacterial species. This estimate of HGT may be low, as we avoided the 36% of carriers from the 2008–2009 study in which sequential carriage of divergent isolates was detected (<a href="#B45">45</a>).</p> </div> <div class="fixationofgeneticvariantsoccursatsimilarratesduringlong-andshort-termhostpersistence." title="sec"> <div class="title" tagx="title" title="title">
Fixation of genetic variants occurs at similar rates during long- and short-term host persistence.</div> <p>Fixed genetic variation is an indicator of selection. We estimated the extent of fixed functional variation during persistent carriage by focusing on nonsynonymous genic variation and PV. Events classified as putative or potential fixation (see Results for a definition) from this study were combined with those of Alamro et al. on scOMP PV (<a href="#B23">23</a>). We detected a total of 140 fixed events for the 19 carriers wherein all four modes of genetic variation were examined (71 PVs, 11 antigenic PilE variations, 7 indels, 29 single-nucleotide variants, 22 HGTs). All PV data were derived from analyses of multiple colonies while only 53% (31/58) of the indel/mutation/HGT class and 18% of the PilE variation was from multiple colonies. <a href="#fig7">Figure 7</a> shows the distribution of genetic events across these 19 carriers plus 6 carriers where only mutation and HGT were examined. Nine carriers exhibited ≤4 events, while the maximum is 23 (4 PVs and 19 HGTs) for V54 (where fixation has not been thoroughly tested) or 12 (5 LHs, 1 indel, and 6 mutations) for V96 and 12 (4 LHs, 3 HGTs) for V222 (where fixation was examined in detail). Overall, we detected an average of 7 genetic events/carrier (note that one recombination event affects multiple genes) with no difference between long- and short-term carriage. These results indicate that meningococcal populations can persist in their hosts with a minimal amount of adaption and that host persistence <i>per se</i> is not a strong driver of genetic variability.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 7</span></a></div>   <p>Relative contributions of LH, mutation, and horizontal gene transfer to functional variation during persistent meningococcal carriage. These diagrams depict the relative amounts of different types of genetic variation occurring during natural asymptomatic carriage of meningococci in 23 university students. Each spoke represents the proportion and type of variation relative to a different but homogeneous starting population, indicated by the black circle. We show only genetic variation that alters the amino acid sequence or expression state of genes and has been putatively or probably fixed in the population (note that fixation was determined by comparison of multiple colonies and time points for all PV and the majority of mutation and HGT events). Each rectangle represents either a single gene or a recombination fragment containing one or more variable genes. The rectangles are colored coded to represent different types of recombination events: dark blue, PV; light blue, antigenic variation of the PilE protein; orange, indel; yellow, mutation; green, recombination due to horizontal gene transfer. The blue lines separate events affecting outer membrane proteins or structures on the outside from other functional groups on the inside. (A) Long-term carriage of 5 to 6 months; (B) short-term carriage of 1 to 3 months; (C) carriers where only mutation and HGT has been assessed.</p>   </div> </div> <div class="lhisthemajordriverofgeneticvariationandmostvariationaffectstheoutermembrane." title="sec"> <div class="title" tagx="title" title="title">
LH is the major driver of genetic variation and most variation affects the outer membrane.</div> <p><a href="#fig7">Figure 7</a> also depicts the relative contributions of the four mechanisms to within-host fixed genetic variation. Seven carriers exhibited variation only by LH or an indel. The average contribution of LH (i.e., PV plus PilE antigenic variation) to fixed genetic variation in each of the 19 carriers was 69% and across all carriers was 59% (82/140; <i>P</i> &amp;lt; 0.0001 assuming 2.25% [45] of loci in a genome of 2,000 genes are subject to LH and an equal potential for all genes to undergo directional variation), with again, no significant difference between long- and short-term carriage. These results highlight the critical contribution of LH to within host adaptation.</p> <p>Another important observation is that 69% of fixed events (96/140; <i>P</i> &amp;lt; 0.0001 assuming 8.5% [180] genes in a genome of 2,127 genes affect the outer membrane) affect the outer membrane, with 6 carriers having fixed variation only in outer membrane-associated genes (<a href="#fig7">Fig. 7</a>). While PV was the major driver of this variation, high variability in OMPs was also observed for non-PV indels and HGT, suggesting that the strong diversifying selection acting on these surface antigens requires significant alterations to an epitope as opposed to single amino acid alterations. Comparable hot spots of recombination and variation in capsule biosynthesis, pilin-associated, <i>pgl</i>, and OMP-encoding genes have been reported in several comparative population studies of meningococcal isolates (<a href="#B34">34</a>, <a href="#B35">35</a>, <a href="#B37">37</a><a href="#B38">–</a><a href="#B39">39</a>). Major changes in antigenic epitopes during persistent carriage are presumed to facilitate evasion of a polyclonal immune response against surface antigens (<a href="#B40">40</a><a href="#B41">–</a><a href="#B42">42</a>).</p> </div> <div class="directionalpvoftheopaadhesinsandpilcproteins." title="sec"> <div class="title" tagx="title" title="title">
Directional PV of the Opa adhesins and PilC proteins.</div> <p>A key observation was for switching between antigenically divergent Opa alleles during persistent carriage combined with continuous expression of one or more Opa proteins. This result indicates that both maintenance of Opa adhesive functions and antigenic variation are required for persistence on mucosal surfaces. Previously, high turnover of gonococcal Opa variants was observed in experimental human challenge experiments, but without clear evidence of directional switching other than selection against Opa variants with specificity for CEACAM3, the neutrophil decoy receptor (<a href="#B43">43</a><a href="#B44">–</a><a href="#B45">45</a>). Our findings suggest that variant-specific antibody responses are frequently elicited during carriage and drive PV-mediated loss of expression of initially dominant Opa variants. Similarly, the PilC proteins modulate meningococcal adhesive properties, with evidence for differential binding of PilC1- and PilC2-expressing variants to different eukaryotic cell types (<a href="#B27">27</a>). Our finding of frequent switches from PilC1 to PilC2 expression during persistent carriage suggests that the tissue tropism alters during persistent carriage. This process may be driven by immune responses to PilC1 or adaptation to other alterations in the microenvironment. Finally, we observed limited PV of the RM systems, suggesting that selection for alterations in expression of phasevarions, as controlled by PV of RM-mediated methylation (<a href="#B46">46</a><a href="#B47">–</a><a href="#B48">48</a>), is infrequent during persistent carriage. These observations are consistent with the idea that PV acts to maintain a range of adhesive functions while enabling frequent antigenic variation and hence a high potential for evasion of antigen-specific antibody responses.</p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusion.</div> <p>We have demonstrated that HGT and LH are the major drivers of within-host adaptive evolution during natural carriage of a bacterial organism. Conversely <i>de novo</i> mutations were infrequent and may only contribute significant effects on longer evolutionary timescales. Future studies are required to ascertain the phenotypic effects of observed genetic variation and to correlate these effects with variations in the host niche, including antigen-specific antibody responses. Our study provides a strong baseline for understanding how within-host genetic variation contributes to the evolution of a bacterial commensal pathogen and to the selective pressures driving evolution of HGT and LH.</p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
MATERIALS AND METHODS</div> <div class="bacterialisolates." title="sec"> <div class="title" tagx="title" title="title">
Bacterial isolates.</div> <p>Meningococcal isolates were obtained from a carriage study performed at the University of Nottingham between November 2008 and May 2009 as described previously (<a href="#B23">23</a>, <a href="#B49">49</a>). The study was approved by the Nottingham University Medical School Ethics Committee, and written informed consent was obtained from all volunteers.</p> </div> <div class="next-generationsequencing." title="sec"> <div class="title" tagx="title" title="title">
Next-generation sequencing.</div> <p>Genomic DNA was extracted according to the manufacturer’s instructions with either the DNeasy blood and tissue kit (Qiagen) or the Wizard Genomic DNA Purification kit (Promega). Index-tagged Illumina sequencing libraries were generated, multiplexed, and sequenced on Illumina HiSeq 2500 machines to generate paired end sequences. Short sequence reads were trimmed with Trimmomatic v0.32 (<a href="#B50">50</a>) and assembled with SPAdes v3.9.0 (<a href="#B51">51</a>).</p> </div> <div class="allelicvariationandhomologousrecombinationanalysis." title="sec"> <div class="title" tagx="title" title="title">
Allelic variation and homologous recombination analysis.</div> <p>The <span class="named-content" title="named-content">N. meningitidis</span> cgMLST v1.0 scheme within the Genome Comparator tool of the Bacterial Isolate Genome Sequence Database (BIGSdb) platform (<a href="#B52">52</a>) was used to analyze 1,605 loci for allelic variation. After removal of incomplete loci, phylogenetic networks were derived from allelic variation and visualized using GrapeTree (<a href="#B53">53</a>). Variable loci were also aligned with Clustal Omega (<a href="#B54">54</a>) and compared manually. Variability was classified as follows: point mutation, a single nucleotide substitution in a gene; indel, presence or absence of a single or contiguous base pairs within a gene; recombination (either gene conversion or HGT), multiple substitutions (&amp;gt;1) or indels within a gene. Recombination patches were detected by extracting and aligning recombinant allelic sequences for a specific gene of a pair of isolates along with the flanking sequences using tools within BIGSdb. A BLAST analysis was performed with the recombinant allele against subsets of isolates in BIGSdb (e.g., other isolates from the same carrier, all isolates collected in the 2008–2009 carriage study or all UK isolates of the same clonal complex with similar isolation times). The gene and flanking sequences were extracted from the genomes of putative donor isolates with the highest match (usually between 95% and 100% similarity). The contiguous sequences of each pair of recombinant alleles and, where possible, the donor genomes were aligned in order to detect the putative start and end points for intergenic recombination. This process was iterative, with flanking sequences being extracted and aligned until the donor sequence no longer matched the recombinant sequence.</p> </div> <div class="enumerationofssrrepeatnumbers." title="sec"> <div class="title" tagx="title" title="title">
Enumeration of SSR repeat numbers.</div> <p>Each SSR was subject to PCR amplification and sequencing using specific primers. Amplicons produced using fluorescently labeled primers were subject to GeneScan fragment size analysis on an ABI3730 autosequencer and compared to either GeneScan 500LIZ or GeneScan 600LIZ size standards (Applied Biosystems). Product sizes were analyzed using PeakScanner v1.0 (Applied Biosystems) and compared to controls of known size. A subset of SSRs were analyzed by dideoxy sequencing and alignment with BioEdit (v7.2.5) to confirm repeat numbers. Two or more labeled products were produced when SSRs contained ≥9 mononucleotide repeats due to slippage during PCR amplification. If the ratio of the areas of the primary and secondary peaks was ≥1.2, then repeat number was determined from the primary peak; if &amp;lt;1.2, then the sample was either reported as mixed or, when occurring across multiple samples, the larger peak was selected as the determinant of repeat number.</p> </div> <div class="westernblotting." title="sec"> <div class="title" tagx="title" title="title">
Western blotting.</div> <p>Meningococcal isolates were grown overnight on brain heart infusion (BHI) agar with 5% horse blood at 37°C with 5% CO<sub>2</sub>. Lysates were prepared from colonies by heating suspensions at 65°C for 15 min in SDS sample loading buffer. Gel electrophoresis of bacterial cell lysates (50 μg) was performed on 10% polyacrylamide gels before transfer to a polyvinylidene difluoride membrane. Glycan-specific monoclonal antibodies (npg1, npg2, and npg3) or a polyclonal antiserum (pDAb2) followed by an alkaline phosphatase-conjugated secondary antibody or an alkaline phosphatase-conjugated succinylated wheat germ agglutinin (sWGA) lectin were used to detect specific protein glycosylation states (<a href="#B28">28</a>).</p> </div> <div class="statistics." title="sec"> <div class="title" tagx="title" title="title">
Statistics.</div> <p>Chi-square tests were performed using an online calculator in GraphPad (<a href="https://www.graphpad.com/quickcalcs/">https://www.graphpad.com/quickcalcs/</a>).</p> </div> <div class="dataavailability." title="sec"> <div class="title" tagx="title" title="title">
Data availability.</div> <p>All gene alignments and analyses are available at <a href="https://leicester.figshare.com/s/24e2782f537d2c77af60">https://leicester.figshare.com/s/24e2782f537d2c77af60</a>. Sequence reads were deposited in the European Nucleotide Archive (ENA), while assembled contigs were deposited into the <a href="https://pubmlst.org/neisseria/">https://pubmlst.org/neisseria/</a> database and are accessible through the <a href="https://pubmlst.org/">https://pubmlst.org/</a> website using the BIGSdb platform (<a href="#B52">52</a>). The identification (ID) numbers for the isolates in this database are as follows: 28247 to 28298 for the paired isolates from the 25 carriers, 28299 to 28338 for the 39 isolates from carrier V59, and 53889 to 53967 for the multiple isolates from five other carriers.</p> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-other" title="other"> <p><b>Citation</b> Green LR, Al-Rubaiawi AA, Al-Maeni MARM, Harrison OB, Blades M, Oldfield NJ, Turner DPJ, Maiden MCJ, Bayliss CD. 2020. Localized hypermutation is the major driver of meningococcal genetic variability during persistent asymptomatic carriage. mBio 11:e03068-19. <a href="https://doi.org/10.1128/mBio.03068-19">https://doi.org/10.1128/mBio.03068-19</a>.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGMENTS</div> <p>We thank Michael Koomey and Bente Borud for provision of reagents for assessing <i>pgl</i> phenotypes and for performing Western blotting for confirmation of <i>pgl</i> expression states. We thank Darryl Hill for provision of an anti-Opa antiserum. We thank multiple undergraduate and master students for contributions to experimental data sets. We thank the student volunteers who provided samples through the underpinning carriage study in 2008 to 2009 at the University of Nottingham, UK.</p> <p>This work was supported by a Medical Research Council grant MR/M020193/1 to C.D.B., Higher Commission for Educational Development in Iraq (PhD funding) for A.A.A.-R. and M.A.R.M.A.-M., and Wellcome Trust grants 104992/Z/14/Z and 087622/Z/08/A to M.C.J.M. The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</p> <p>This study made use of the <i>Neisseria</i> PubMLST website (<a href="https://pubmlst.org/neisseria/">https://pubmlst.org/neisseria/</a>) hosted by the University of Oxford and developed with funding from the Wellcome Trust and European Union (<a href="#B55">55</a>).</p> <p>The work was conceived and planned by C.D.B., L.R.G., N.J.O., D.P.J.T., and M.C.J.M. Genome sequencing was facilitated by O.B.H. and M.C.J.M. Experimental work was performed by L.R.G. and A.A.A.-R. Bioinformatics was performed by M.A.R.M.A.-M., O.B.H., M.B., L.R.G., and C.D.B. The manuscript was written by C.D.B. and L.R.G. All authors commented on the final draft.</p> <p>C.D.B. reports funding from or interactions with GlaxoSmithKline, Sanofi Pasteur MSD, Pfizer, and Roche, outside the submitted work. All other authors declare no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="B1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Althouse</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanley</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The tortoise or the hare? Impacts of within-host dynamics on transmission success of arthropod-borne viruses</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span><span class="volume" tagx="volume" title="volume">370</span>:<span class="fpage" tagx="fpage" title="fpage">20140299</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2014.0299">10.1098/rstb.2014.0299</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26150665">26150665</a></span></span></li> <li tag="ref"><a name="B2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipsitch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moxon</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Virulence and transmissibility of pathogens: what is the relationship?</span><span class="source" tagx="source" title="source">Trends Microbiol</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">31</span>–<span class="lpage" tagx="lpage" title="lpage">37</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S0966-842X(97)81772-6">10.1016/S0966-842X(97)81772-6</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9025233">9025233</a></span></span></li> <li tag="ref"><a name="B3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monack</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title"><i>Salmonella</i> persistence and transmission strategies</span>. <span class="source" tagx="source" title="source">Curr Opin Microbiol</span><span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">100</span>–<span class="lpage" tagx="lpage" title="lpage">107</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.mib.2011.10.013">10.1016/j.mib.2011.10.013</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22137596">22137596</a></span></span></li> <li tag="ref"><a name="B4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Georgieva</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buckee</span><span class="given-names" tagx="given-names" title="given-names">CO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipsitch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Models of immune selection for multi-locus antigenic diversity of pathogens</span>. <span class="source" tagx="source" title="source">Nat Rev Immunol</span><span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">55</span>–<span class="lpage" tagx="lpage" title="lpage">62</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/s41577-018-0092-5">10.1038/s41577-018-0092-5</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30479379">30479379</a></span></span></li> <li tag="ref"><a name="B5" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moxon</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rainey</span><span class="given-names" tagx="given-names" title="given-names">PB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenski</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Adaptive evolution of highly mutable loci in pathogenic bacteria</span>. <span class="source" tagx="source" title="source">Curr Biol</span><span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">24</span>–<span class="lpage" tagx="lpage" title="lpage">33</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/s0960-9822(00)00005-1">10.1016/s0960-9822(00)00005-1</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7922307">7922307</a></span></span></li> <li tag="ref"><a name="B6" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Woude</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bäumler</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Phase and antigenic variation in bacteria</span>. <span class="source" tagx="source" title="source">Clin Microbiol Rev</span><span class="volume" tagx="volume" title="volume">17</span>:<span class="fpage" tagx="fpage" title="fpage">581</span>–<span class="lpage" tagx="lpage" title="lpage">611</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/CMR.17.3.581-611.2004">10.1128/CMR.17.3.581-611.2004</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15258095">15258095</a></span></span></li> <li tag="ref"><a name="B7" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Field</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moxon</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">The simple sequence contingency loci of <i>Haemophilus influenzae</i> and <i>Neisseria meningitidis</i></span>. <span class="source" tagx="source" title="source">J Clin Invest</span><span class="volume" tagx="volume" title="volume">107</span>:<span class="fpage" tagx="fpage" title="fpage">657</span>–<span class="lpage" tagx="lpage" title="lpage">662</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1172/JCI12557">10.1172/JCI12557</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11254662">11254662</a></span></span></li> <li tag="ref"><a name="B8" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tettelin</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkhill</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frosch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Genome flexibility in <i>Neisseria meningitidis</i></span>. <span class="source" tagx="source" title="source">Vaccine</span><span class="volume" tagx="volume" title="volume">27</span><span class="issue" tagx="issue" title="issue">Suppl 2</span>:<span class="fpage" tagx="fpage" title="fpage">B103</span>–<span class="lpage" tagx="lpage" title="lpage">B111</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2009.04.064">10.1016/j.vaccine.2009.04.064</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19477564">19477564</a></span></span></li> <li tag="ref"><a name="B9" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christensen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hickman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trotter</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Meningococcal carriage by age: a systematic review and meta-analysis</span>. <span class="source" tagx="source" title="source">Lancet Infect Dis</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">853</span>–<span class="lpage" tagx="lpage" title="lpage">861</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(10)70251-6">10.1016/S1473-3099(10)70251-6</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21075057">21075057</a></span></span></li> <li tag="ref"><a name="B10" /><span class="label" tagx="label" title="label">10.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caugant</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Meningococcal carriage and disease–population biology and evolution</span>. <span class="source" tagx="source" title="source">Vaccine</span><span class="volume" tagx="volume" title="volume">27</span><span class="issue" tagx="issue" title="issue">Suppl 2</span>:<span class="fpage" tagx="fpage" title="fpage">B64</span>–<span class="lpage" tagx="lpage" title="lpage">B70</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2009.04.061">10.1016/j.vaccine.2009.04.061</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19464092">19464092</a></span></span></li> <li tag="ref"><a name="B11" /><span class="label" tagx="label" title="label">11.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolley</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kriz</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McVean</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">The influence of mutation, recombination, population history, and selection on patterns of genetic diversity in <i>Neisseria meningitidis</i></span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">562</span>–<span class="lpage" tagx="lpage" title="lpage">569</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msi041">10.1093/molbev/msi041</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15537808">15537808</a></span></span></li> <li tag="ref"><a name="B12" /><span class="label" tagx="label" title="label">12.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cahoon</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seifert</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Transcription of a <i>cis</i>-acting, noncoding, small RNA is required for pilin antigenic variation in <i>Neisseria gonorrhoeae</i></span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003074</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003074">10.1371/journal.ppat.1003074</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23349628">23349628</a></span></span></li> <li tag="ref"><a name="B13" /><span class="label" tagx="label" title="label">13.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wanford</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aidley</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Phasome analysis of pathogenic and commensal <i>Neisseria</i> species expands the known repertoire of phase variable genes, and highlights common adaptive strategies</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">13</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e0196675</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0196675">10.1371/journal.pone.0196675</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29763438">29763438</a></span></span></li> <li tag="ref"><a name="B14" /><span class="label" tagx="label" title="label">14.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moxon</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hood</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Bacterial contingency loci: the role of simple sequence DNA repeats in bacterial adaptation</span>. <span class="source" tagx="source" title="source">Annu Rev Genet</span><span class="volume" tagx="volume" title="volume">40</span>:<span class="fpage" tagx="fpage" title="fpage">307</span>–<span class="lpage" tagx="lpage" title="lpage">333</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.genet.40.110405.090442">10.1146/annurev.genet.40.110405.090442</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17094739">17094739</a></span></span></li> <li tag="ref"><a name="B15" /><span class="label" tagx="label" title="label">15.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saliba</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borrow</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramsay</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ladhani</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Targeted vaccination of teenagers following continued rapid endemic expansion of a single meningococcal group W clone (sequence type 11 clonal complex), United Kingdom 2015</span>. <span class="source" tagx="source" title="source">Euro Surveill</span><span class="volume" tagx="volume" title="volume">20</span>:<span class="fpage" tagx="fpage" title="fpage">21188</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.2807/1560-7917.es2015.20.28.21188">10.2807/1560-7917.es2015.20.28.21188</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26212140">26212140</a></span></span></li> <li tag="ref"><a name="B16" /><span class="label" tagx="label" title="label">16.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilkins</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Snape</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Emerging clinical experience with vaccines against group B meningococcal disease</span>. <span class="source" tagx="source" title="source">Vaccine</span><span class="volume" tagx="volume" title="volume">36</span>:<span class="fpage" tagx="fpage" title="fpage">5470</span>–<span class="lpage" tagx="lpage" title="lpage">5476</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2017.07.056">10.1016/j.vaccine.2017.07.056</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28778616">28778616</a></span></span></li> <li tag="ref"><a name="B17" /><span class="label" tagx="label" title="label">17.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brueggemann</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pai</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crook</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beall</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Vaccine escape recombinants emerge after pneumococcal vaccination in the United States</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">3</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e168</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.0030168">10.1371/journal.ppat.0030168</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18020702">18020702</a></span></span></li> <li tag="ref"><a name="B18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamng'ona</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hinds</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bar-Zeev</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gould</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaguza</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Msefula</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cornick</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulohoma</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bentley</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">French</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heyderman</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Everett</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">High multiple carriage and emergence of <i>Streptococcus pneumoniae</i> vaccine serotype variants in Malawian children</span>. <span class="source" tagx="source" title="source">BMC Infect Dis</span><span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">234</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1186/s12879-015-0980-2">10.1186/s12879-015-0980-2</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26088623">26088623</a></span></span></li> <li tag="ref"><a name="B19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucidarme</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newbold</span><span class="given-names" tagx="given-names" title="given-names">LS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ure</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brehony</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">OB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bray</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolley</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bratcher</span><span class="given-names" tagx="given-names" title="given-names">HB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkhill</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borrow</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Genomic epidemiology of age-associated meningococcal lineages in national surveillance: an observational cohort study</span>. <span class="source" tagx="source" title="source">Lancet Infect Dis</span><span class="volume" tagx="volume" title="volume">15</span>:<span class="fpage" tagx="fpage" title="fpage">1420</span>–<span class="lpage" tagx="lpage" title="lpage">1428</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(15)00267-4">10.1016/S1473-3099(15)00267-4</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26515523">26515523</a></span></span></li> <li tag="ref"><a name="B20" /><span class="label" tagx="label" title="label">20.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldfield</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">OB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">MCJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ala'Aldeen</span><span class="given-names" tagx="given-names" title="given-names">DAA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">DPJ</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Genomic analysis of serogroup Y <i>Neisseria meningitidis</i> isolates reveals extensive similarities between carriage-associated and disease-associated organisms</span>. <span class="source" tagx="source" title="source">J Infect Dis</span><span class="volume" tagx="volume" title="volume">213</span>:<span class="fpage" tagx="fpage" title="fpage">1777</span>–<span class="lpage" tagx="lpage" title="lpage">1785</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiw008">10.1093/infdis/jiw008</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26747709">26747709</a></span></span></li> <li tag="ref"><a name="B21" /><span class="label" tagx="label" title="label">21.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seifert</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jerse</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cannon</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Multiple gonococcal pilin antigenic variants are produced during experimental human infections</span>. <span class="source" tagx="source" title="source">J Clin Invest</span><span class="volume" tagx="volume" title="volume">93</span>:<span class="fpage" tagx="fpage" title="fpage">2744</span>–<span class="lpage" tagx="lpage" title="lpage">2749</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1172/JCI117290">10.1172/JCI117290</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7911129">7911129</a></span></span></li> <li tag="ref"><a name="B22" /><span class="label" tagx="label" title="label">22.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bilek</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ison</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spratt</span><span class="given-names" tagx="given-names" title="given-names">BG</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Relative contributions of recombination and mutation to the diversification of the <i>opa</i> gene repertoire of <i>Neisseria gonorrhoeae</i></span>. <span class="source" tagx="source" title="source">J Bacteriol</span><span class="volume" tagx="volume" title="volume">191</span>:<span class="fpage" tagx="fpage" title="fpage">1878</span>–<span class="lpage" tagx="lpage" title="lpage">1890</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.01518-08">10.1128/JB.01518-08</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19114493">19114493</a></span></span></li> <li tag="ref"><a name="B23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alamro</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bidmos</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldfield</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newton</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aidley</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Care</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mattick</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">DPJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neal</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ala'aldeen</span><span class="given-names" tagx="given-names" title="given-names">DAA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feavers</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borrow</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Phase variation mediates reductions in expression of surface proteins during persistent meningococcal carriage</span>. <span class="source" tagx="source" title="source">Infect Immun</span><span class="volume" tagx="volume" title="volume">82</span>:<span class="fpage" tagx="fpage" title="fpage">2472</span>–<span class="lpage" tagx="lpage" title="lpage">2484</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/IAI.01521-14">10.1128/IAI.01521-14</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24686058">24686058</a></span></span></li> <li tag="ref"><a name="B24" /><span class="label" tagx="label" title="label">24.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldfield</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bidmos</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alamro</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neal</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">DPJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ala'aldeen</span><span class="given-names" tagx="given-names" title="given-names">DAA</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Prevalence and phase variable expression status of two autotransporters, NalP and MspA, in carriage and disease isolates of <i>Neisseria meningitidis</i></span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">8</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e69746</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0069746">10.1371/journal.pone.0069746</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23936091">23936091</a></span></span></li> <li tag="ref"><a name="B25" /><span class="label" tagx="label" title="label">25.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berrington</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srikhanta</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuipers</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Ley</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peak</span><span class="given-names" tagx="given-names" title="given-names">IR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jennings</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Phase variation in meningococcal lipooligosaccharide biosynthesis genes</span>. <span class="source" tagx="source" title="source">FEMS Immunol Med Microbiol</span><span class="volume" tagx="volume" title="volume">34</span>:<span class="fpage" tagx="fpage" title="fpage">267</span>–<span class="lpage" tagx="lpage" title="lpage">275</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1574-695X.2002.tb00633.x">10.1111/j.1574-695X.2002.tb00633.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12443826">12443826</a></span></span></li> <li tag="ref"><a name="B26" /><span class="label" tagx="label" title="label">26.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonsson</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyberg</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Normark</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1991</span><span class="mixed-article-title" title="mixed-article-title">Phase variation of gonococcal pili by frameshift mutation in pilC, a novel gene for pilus assembly</span>. <span class="source" tagx="source" title="source">EMBO J</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">477</span>–<span class="lpage" tagx="lpage" title="lpage">488</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1002/j.1460-2075.1991.tb07970.x">10.1002/j.1460-2075.1991.tb07970.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1671354">1671354</a></span></span></li> <li tag="ref"><a name="B27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morand</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drab</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajalingam</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nassif</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">TF</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title"><i>Neisseria meningitidis</i> differentially controls host cell motility through PilC1 and PilC2 components of type IV pili</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">4</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e6834</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0006834">10.1371/journal.pone.0006834</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19718432">19718432</a></span></span></li> <li tag="ref"><a name="B28" /><span class="label" tagx="label" title="label">28.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borud</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aas</span><span class="given-names" tagx="given-names" title="given-names">FE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vik</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winther-Larsen</span><span class="given-names" tagx="given-names" title="given-names">HC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egge-Jacobsen</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koomey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Genetic, structural, and antigenic analyses of glycan diversity in the <i>O</i>-linked protein glycosylation systems of human <i>Neisseria</i> species</span>. <span class="source" tagx="source" title="source">J Bacteriol</span><span class="volume" tagx="volume" title="volume">192</span>:<span class="fpage" tagx="fpage" title="fpage">2816</span>–<span class="lpage" tagx="lpage" title="lpage">2829</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.00101-10">10.1128/JB.00101-10</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20363948">20363948</a></span></span></li> <li tag="ref"><a name="B29" /><span class="label" tagx="label" title="label">29.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viburiene</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vik</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koomey</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borud</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Allelic variation in a simple sequence repeat element of neisserial <i>pglB2</i> and its consequences for protein expression and protein glycosylation</span>. <span class="source" tagx="source" title="source">J Bacteriol</span><span class="volume" tagx="volume" title="volume">195</span>:<span class="fpage" tagx="fpage" title="fpage">3476</span>–<span class="lpage" tagx="lpage" title="lpage">3485</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.00276-13">10.1128/JB.00276-13</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23729645">23729645</a></span></span></li> <li tag="ref"><a name="B30" /><span class="label" tagx="label" title="label">30.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hood</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deadman</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jennings</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bisercic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleischmann</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venter</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moxon</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">DNA repeats identify novel virulence genes in <i>Haemophilus influenzae</i></span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">93</span>:<span class="fpage" tagx="fpage" title="fpage">11121</span>–<span class="lpage" tagx="lpage" title="lpage">11125</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.93.20.11121">10.1073/pnas.93.20.11121</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8855319">8855319</a></span></span></li> <li tag="ref"><a name="B31" /><span class="label" tagx="label" title="label">31.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrick</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenski</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Genome dynamics during experimental evolution</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span><span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">827</span>–<span class="lpage" tagx="lpage" title="lpage">839</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/nrg3564">10.1038/nrg3564</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24166031">24166031</a></span></span></li> <li tag="ref"><a name="B32" /><span class="label" tagx="label" title="label">32.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Determinants of phase variation rate and the fitness implications of differing rates for bacterial pathogens and commensals</span>. <span class="source" tagx="source" title="source">FEMS Microbiol Rev</span><span class="volume" tagx="volume" title="volume">33</span>:<span class="fpage" tagx="fpage" title="fpage">504</span>–<span class="lpage" tagx="lpage" title="lpage">520</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1574-6976.2009.00162.x">10.1111/j.1574-6976.2009.00162.x</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19222587">19222587</a></span></span></li> <li tag="ref"><a name="B33" /><span class="label" tagx="label" title="label">33.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleary</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laver</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorringe</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deasy</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dale</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Didelot</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Read</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Microevolution of <i>Neisseria lactamica</i> during nasopharyngeal colonisation induced by controlled human infection</span>. <span class="source" tagx="source" title="source">Nat Commun</span><span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">4753</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-07235-5">10.1038/s41467-018-07235-5</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30420631">30420631</a></span></span></li> <li tag="ref"><a name="B34" /><span class="label" tagx="label" title="label">34.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borud</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barnes</span><span class="given-names" tagx="given-names" title="given-names">GK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brynildsrud</span><span class="given-names" tagx="given-names" title="given-names">OB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fritzsonn</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caugant</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Genotypic and phenotypic characterization of the <i>O</i>-linked protein glycosylation system reveals high glycan diversity in paired meningococcal carriage isolates</span>. <span class="source" tagx="source" title="source">J Bacteriol</span><span class="volume" tagx="volume" title="volume">200</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e00794-17</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/JB.00794-17">10.1128/JB.00794-17</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29555702">29555702</a></span></span></li> <li tag="ref"><a name="B35" /><span class="label" tagx="label" title="label">35.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamelas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roltgen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dangy</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauser</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kingsley</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Connor</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sie</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodgson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dougan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkhill</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bentley</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pluschke</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Emergence of a new epidemic <i>Neisseria meningitidis</i> serogroup A clone in the African meningitis belt: high-resolution picture of genomic changes that mediate immune evasion</span>. <span class="source" tagx="source" title="source">mBio</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e01974-14</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.01974-14">10.1128/mBio.01974-14</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25336458">25336458</a></span></span></li> <li tag="ref"><a name="B36" /><span class="label" tagx="label" title="label">36.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Popovic</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stojiljkovic</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Mutator clones of <i>Neisseria meningitidis</i> in epidemic serogroup A disease</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">99</span>:<span class="fpage" tagx="fpage" title="fpage">6103</span>–<span class="lpage" tagx="lpage" title="lpage">6107</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.092568699">10.1073/pnas.092568699</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11983903">11983903</a></span></span></li> <li tag="ref"><a name="B37" /><span class="label" tagx="label" title="label">37.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krauland</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunning Hotopp</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daugherty</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsh</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messonnier</span><span class="given-names" tagx="given-names" title="given-names">NE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayer</span><span class="given-names" tagx="given-names" title="given-names">LW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tettelin</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Whole genome sequencing to investigate the emergence of clonal complex 23 <i>Neisseria meningitidis</i> serogroup Y disease in the United States</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">7</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e35699</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0035699">10.1371/journal.pone.0035699</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22558202">22558202</a></span></span></li> <li tag="ref"><a name="B38" /><span class="label" tagx="label" title="label">38.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamelas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hauser</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dangy</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamid</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roltgen</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdul Sater</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodgson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sie</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Junghanss</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkhill</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bentley</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pluschke</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Emergence and genomic diversification of a virulent serogroup W:ST-2881(CC175) <i>Neisseria meningitidis</i> clone in the African meningitis belt</span>. <span class="source" tagx="source" title="source">Microb Genom</span><span class="volume" tagx="volume" title="volume">3</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e000120</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1099/mgen.0.000120">10.1099/mgen.0.000120</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29026659">29026659</a></span></span></li> <li tag="ref"><a name="B39" /><span class="label" tagx="label" title="label">39.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toros</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedberg</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Unemo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacobsson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olcen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fredlund</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bratcher</span><span class="given-names" tagx="given-names" title="given-names">HB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolley</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molling</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Genome-based characterization of emergent invasive <i>Neisseria meningitidis</i> serogroup Y isolates in Sweden from 1995 to 2012</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="volume" tagx="volume" title="volume">53</span>:<span class="fpage" tagx="fpage" title="fpage">2154</span>–<span class="lpage" tagx="lpage" title="lpage">2162</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.03524-14">10.1128/JCM.03524-14</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25926489">25926489</a></span></span></li> <li tag="ref"><a name="B40" /><span class="label" tagx="label" title="label">40.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horton</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vidarsson</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Virji</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heyderman</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">IgA1 antibodies specific for outer membrane protein PorA modulate the interaction between <i>Neisseria meningitidis</i> and the epithelium</span>. <span class="source" tagx="source" title="source">Microb Pathog</span><span class="volume" tagx="volume" title="volume">46</span>:<span class="fpage" tagx="fpage" title="fpage">253</span>–<span class="lpage" tagx="lpage" title="lpage">260</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1016/j.micpath.2009.01.009">10.1016/j.micpath.2009.01.009</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19486642">19486642</a></span></span></li> <li tag="ref"><a name="B41" /><span class="label" tagx="label" title="label">41.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordens</span><span class="given-names" tagx="given-names" title="given-names">JZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christodoulides</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heckels</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Development of immunity to serogroup B meningococci during carriage of <i>Neisseria meningitidis</i> in a cohort of university students</span>. <span class="source" tagx="source" title="source">Infect Immun</span><span class="volume" tagx="volume" title="volume">72</span>:<span class="fpage" tagx="fpage" title="fpage">6503</span>–<span class="lpage" tagx="lpage" title="lpage">6510</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/IAI.72.11.6503-6510.2004">10.1128/IAI.72.11.6503-6510.2004</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15501781">15501781</a></span></span></li> <li tag="ref"><a name="B42" /><span class="label" tagx="label" title="label">42.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Litt</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beddek</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comanducci</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandiford</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevens</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ison</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pizza</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rappuoli</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kroll</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Putative vaccine antigens from <i>Neisseria meningitidis</i> recognized by serum antibodies of young children convalescing after meningococcal disease</span>. <span class="source" tagx="source" title="source">J Infect Dis</span><span class="volume" tagx="volume" title="volume">190</span>:<span class="fpage" tagx="fpage" title="fpage">1488</span>–<span class="lpage" tagx="lpage" title="lpage">1497</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1086/424464">10.1086/424464</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15378443">15378443</a></span></span></li> <li tag="ref"><a name="B43" /><span class="label" tagx="label" title="label">43.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jerse</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drown</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whicker</span><span class="given-names" tagx="given-names" title="given-names">LG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isbey</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seifert</span><span class="given-names" tagx="given-names" title="given-names">HS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cannon</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Multiple gonococcal opacity proteins are expressed during experimental urethral infection in the male</span>. <span class="source" tagx="source" title="source">J Exp Med</span><span class="volume" tagx="volume" title="volume">179</span>:<span class="fpage" tagx="fpage" title="fpage">911</span>–<span class="lpage" tagx="lpage" title="lpage">920</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1084/jem.179.3.911">10.1084/jem.179.3.911</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8113683">8113683</a></span></span></li> <li tag="ref"><a name="B44" /><span class="label" tagx="label" title="label">44.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sintsova</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacDonald</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaul</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Virji</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray-Owen</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Selection for a CEACAM receptor-specific binding phenotype during <i>Neisseria gonorrhoeae</i> infection of the human genital tract</span>. <span class="source" tagx="source" title="source">Infect Immun</span><span class="volume" tagx="volume" title="volume">83</span>:<span class="fpage" tagx="fpage" title="fpage">1372</span>–<span class="lpage" tagx="lpage" title="lpage">1383</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/IAI.03123-14">10.1128/IAI.03123-14</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25605771">25605771</a></span></span></li> <li tag="ref"><a name="B45" /><span class="label" tagx="label" title="label">45.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hobbs</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sparling</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shafer</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deal</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jerse</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Experimental gonococcal infection in male volunteers: cumulative experience with <i>Neisseria gonorrhoeae</i> strains FA1090 and MS11mkC</span>. <span class="source" tagx="source" title="source">Front Microbiol</span><span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">123</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2011.00123">10.3389/fmicb.2011.00123</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21734909">21734909</a></span></span></li> <li tag="ref"><a name="B46" /><span class="label" tagx="label" title="label">46.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srikhanta</span><span class="given-names" tagx="given-names" title="given-names">YN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dowideit</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falsetta</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrison</span><span class="given-names" tagx="given-names" title="given-names">OB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fox</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seib</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maguire</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grimmond</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Apicella</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jennings</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Phasevarions mediate random switching of gene expression in pathogenic <i>Neisseria</i></span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000400</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1000400">10.1371/journal.ppat.1000400</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19390608">19390608</a></span></span></li> <li tag="ref"><a name="B47" /><span class="label" tagx="label" title="label">47.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srikhanta</span><span class="given-names" tagx="given-names" title="given-names">YN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fox</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jennings</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">The phasevarion: phase variation of type III DNA methyltransferases controls coordinated switching in multiple genes</span>. <span class="source" tagx="source" title="source">Nat Rev Microbiol</span><span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">196</span>–<span class="lpage" tagx="lpage" title="lpage">206</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro2283">10.1038/nrmicro2283</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20140025">20140025</a></span></span></li> <li tag="ref"><a name="B48" /><span class="label" tagx="label" title="label">48.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seib</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jen</span><span class="given-names" tagx="given-names" title="given-names">FE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jennings</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Phase variation of DNA methyltransferases and the regulation of virulence and immune evasion in the pathogenic <i>Neisseria</i></span>. <span class="source" tagx="source" title="source">Pathog Dis</span><span class="volume" tagx="volume" title="volume">75</span>:<span class="fpage" tagx="fpage" title="fpage">ftx080</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/femspd/ftx080">10.1093/femspd/ftx080</a></span>.</span></li> <li tag="ref"><a name="B49" /><span class="label" tagx="label" title="label">49.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bidmos</span><span class="given-names" tagx="given-names" title="given-names">FA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neal</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oldfield</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">DPJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ala'Aldeen</span><span class="given-names" tagx="given-names" title="given-names">DAA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayliss</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Persistence, replacement, and rapid clonal expansion of meningococcal carriage isolates in a 2008 university student cohort</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="volume" tagx="volume" title="volume">49</span>:<span class="fpage" tagx="fpage" title="fpage">506</span>–<span class="lpage" tagx="lpage" title="lpage">512</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.01322-10">10.1128/JCM.01322-10</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21123536">21123536</a></span></span></li> <li tag="ref"><a name="B50" /><span class="label" tagx="label" title="label">50.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolger</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usadel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">30</span>:<span class="fpage" tagx="fpage" title="fpage">2114</span>–<span class="lpage" tagx="lpage" title="lpage">2120</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu170">10.1093/bioinformatics/btu170</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404">24695404</a></span></span></li> <li tag="ref"><a name="B51" /><span class="label" tagx="label" title="label">51.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bankevich</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nurk</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antipov</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gurevich</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dvorkin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulikov</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lesin</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikolenko</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pham</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prjibelski</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyshkin</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sirotkin</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vyahhi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesler</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alekseyev</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pevzner</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</span>. <span class="source" tagx="source" title="source">J Comput Biol</span><span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">455</span>–<span class="lpage" tagx="lpage" title="lpage">477</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1089/cmb.2012.0021">10.1089/cmb.2012.0021</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22506599">22506599</a></span></span></li> <li tag="ref"><a name="B52" /><span class="label" tagx="label" title="label">52.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolley</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">BIGSdb: scalable analysis of bacterial genome variation at the population level</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">595</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-11-595">10.1186/1471-2105-11-595</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21143983">21143983</a></span></span></li> <li tag="ref"><a name="B53" /><span class="label" tagx="label" title="label">53.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alikhan</span><span class="given-names" tagx="given-names" title="given-names">NF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sergeant</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luhmann</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francisco</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrico</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Achtman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">GrapeTree: visualization of core genomic relationships among 100,000 bacterial pathogens</span>. <span class="source" tagx="source" title="source">Genome Res</span><span class="volume" tagx="volume" title="volume">28</span>:<span class="fpage" tagx="fpage" title="fpage">1395</span>–<span class="lpage" tagx="lpage" title="lpage">1404</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.232397.117">10.1101/gr.232397.117</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30049790">30049790</a></span></span></li> <li tag="ref"><a name="B54" /><span class="label" tagx="label" title="label">54.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sievers</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilm</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dineen</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karplus</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McWilliam</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Remmert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soding</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgins</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <span class="source" tagx="source" title="source">Mol Syst Biol</span><span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">539</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/msb.2011.75">10.1038/msb.2011.75</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21988835">21988835</a></span></span></li> <li tag="ref"><a name="B55" /><span class="label" tagx="label" title="label">55.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jolley</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bray</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiden</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Open-access bacterial population genomics: BIGSdb software, the PubMLST.org website and their applications</span>. <span class="source" tagx="source" title="source">Wellcome Open Res</span><span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">124</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.12688/wellcomeopenres.14826.1">10.12688/wellcomeopenres.14826.1</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30345391">30345391</a></span></span></li> </ul> </div> </div>  </body></html>