<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/embed/graphic-6.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/13/2020.02.12.946772.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.12.946772v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.12.946772"/>
  <meta name="DC.Date" content="2020-02-13"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Motivation Accurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm. Results We trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours. Availability The repository of this project will be published. Contact dongsi{at}uw.edu Supplementary information Supplementary data will be available at Bioinformatics online."/>
  <meta name="DC.Contributor" content="Jonas Pfab"/>
  <meta name="DC.Contributor" content="Dong Si"/>
  <meta name="article:published_time" content="2020-02-13"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;h3&gt;;Motivation&lt;;/h3&gt;; &lt;;p&gt;;Accurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm.&lt;;/p&gt;;&lt;;h3&gt;;Results&lt;;/h3&gt;; &lt;;p&gt;;We trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours.&lt;;/p&gt;;&lt;;h3&gt;;Availability&lt;;/h3&gt;; &lt;;p&gt;;The repository of this project will be published.&lt;;/p&gt;;&lt;;h3&gt;;Contact&lt;;/h3&gt;; &lt;;p&gt;;dongsi@uw.edu&lt;;/p&gt;;&lt;;h3&gt;;Supplementary information&lt;;/h3&gt;; &lt;;p&gt;;Supplementary data will be available at &lt;;i&gt;;Bioinformatics&lt;;/i&gt;; online.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.12.946772v1"/>
  <meta name="citation_id" content="2020.02.12.946772v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.12.946772v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.12.946772v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.12.946772v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.12.946772"/>
  <meta name="citation_num_pages" content="8"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.12.946772"/>
  <meta name="citation_author" content="Jonas Pfab"/>
  <meta name="citation_author_institution" content="Division of Computing and Software Systems, University of Washington"/>
  <meta name="citation_author" content="Dong Si"/>
  <meta name="citation_author_institution" content="Division of Computing and Software Systems, University of Washington"/>
  <meta name="citation_author_email" content="dongsi@uw.edu"/>
  <meta name="citation_reference" content="C. I. Branden and J. Tooze, Introduction to Protein Structure. Garland Science, 2012."/>
  <meta name="citation_reference" content="citation_title=Advances in protein chemistry;citation_journal_abbrev=Adv Protein Chem;citation_author=J. S. Richardson;citation_title=The anatomy and taxonomy of protein structure.;citation_pages=167-339;citation_volume=34;citation_year=1981;citation_pmid=7020376;citation_doi=10.1016/S0065-3233(08)60520-3"/>
  <meta name="citation_reference" content="“Wuhan coronavirus 2019-nCoV - what we can find out on a structural bioinformatics level,” Innophore, 23-Jan-2020. [Online]. Available: https://innophore.com/2019-ncov/. [Accessed: 30-Jan-2020]."/>
  <meta name="citation_reference" content="X. Xu et al., “Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission,” Sci. CHINA Life Sci., Jan. 2020, doi: 10.1007/s11427-020-1637-5."/>
  <meta name="citation_reference" content="L. E. Gralinski and V. D. Menachery, “Return of the Coronavirus: 2019-nCoV,” Viruses, vol. 12, no. 2, p. 135, Feb. 2020, doi: 10.3390/v12020135."/>
  <meta name="citation_reference" content="G. Corradin, V. Villard, and A. V. Kajava, “Protein Structure Based Strategies for Antigen Discovery and Vaccine Development Against Malaria and Other Pathogens,” 2007. [Online]. Available: https://www.ingentaconnect.com/content/ben/emiddt/2007/00000007/00000004/art00004. [Accessed: 14-Jan-2020]."/>
  <meta name="citation_reference" content="“Application of the Three-Dimensional Structures of Protein Target Molecules in Structure-Based Drug Design | Journal of Medicinal Chemistry.” [Online]. Available: https://pubs.acs.org/doi/pdf/10.1021/jm00034a001. [Accessed: 07-Jul-2019]."/>
  <meta name="citation_reference" content="D. F. Hunt, J. R. Yates, J. Shabanowitz, S. Winston, and C. R. Hauer, “Protein sequencing by tandem mass spectrometry,” Proc. Natl. Acad. Sci., vol. 83, no. 17, pp. 6233–6237, Sep. 1986, doi: 10.1073/pnas.83.17.6233."/>
  <meta name="citation_reference" content="J. Garnier, J.-F. Gibrat, and B. Robson, “[32] GOR method for predicting protein secondary structure from amino acid sequence,” in Methods in Enzymology, vol. 266, Academic Press, 1996, pp. 540–553."/>
  <meta name="citation_reference" content="F. Eisenhaber, B. Persson, and P. Argos, “Protein Structure Prediction: Recognition of Primary, Secondary, and Tertiary Structural Features from Amino Acid Sequence,” Crit. Rev. Biochem. Mol. Biol., vol. 30, no. 1, pp. 1–94, Jan. 1995, doi: 10.3109/10409239509085139."/>
  <meta name="citation_reference" content="A. W. Senior et al., “Improved protein structure prediction using potentials from deep learning,” Nature, pp. 1–5, Jan. 2020, doi: 10.1038/s41586-019-1923-7."/>
  <meta name="citation_reference" content="B. Contreras-Moreira, I. Ezkurdia, M. L. Tress, and A. Valencia, “Empirical limits for template-based protein structure prediction: the CASP5 example,” FEBS Lett., vol. 579, no. 5, pp. 1203–1207, 2005, doi: 10.1016/j.febslet.2005.01.014."/>
  <meta name="citation_reference" content="E. Nogales, “Cryo-EM,” Curr. Biol., vol. 28, no. 19, pp. R1127–R1128, Oct. 2018, doi: 10.1016/j.cub.2018.07.016."/>
  <meta name="citation_reference" content="A. Merk et al., “Breaking Cryo-EM Resolution Barriers to Facilitate Drug Discovery,” Cell, vol. 165, no. 7, pp. 1698–1707, Jun. 2016, doi: 10.1016/j.cell.2016.05.040."/>
  <meta name="citation_reference" content="K. Zhang et al., “Cryo-EM structures of Helicobacter pylori vacuolating cytotoxin A oligomeric assemblies at near-atomic resolution,” Proc. Natl. Acad. Sci., vol. 116, no. 14, pp. 6800–6805, Apr. 2019, doi: 10.1073/pnas.1821959116."/>
  <meta name="citation_reference" content="P. D. Adams et al., “PHENIX: a comprehensive Python-based system for macromolecular structure solution,” Acta Crystallogr. D Biol. Crystallogr., vol. 66, no. 2, pp. 213–221, Feb. 2010, doi: 10.1107/S0907444909052925."/>
  <meta name="citation_reference" content="C. A. Rohl, C. E. M. Strauss, K. M. S. Misura, and D. Baker, “Protein Structure Prediction Using Rosetta,” in Methods in Enzymology, vol. 383, Academic Press, 2004, pp. 66–93."/>
  <meta name="citation_reference" content="S. A. Moritz et al., “Cascaded-CNN: Deep Learning to Predict Protein Backbone Structure from High-Resolution Cryo-EM Density Maps,” bioRxiv, p. 572990, Mar. 2019, doi: 10.1101/572990."/>
  <meta name="citation_reference" content="“EMDataResource.” [Online]. Available: https://www.emdataresource.org/. [Accessed: 10-Jan-2020]."/>
  <meta name="citation_reference" content="R. P. D. Bank, “RCSB PDB: Homepage.” [Online]. Available: https://www.rcsb.org/. [Accessed: 10-Jan-2020]."/>
  <meta name="citation_reference" content="“Traveling Salesman Problem | OR-Tools,” Google Developers. [Online]. Available: https://developers.google.com/optimization/routing/tsp. [Accessed: 22-Jan-2020]."/>
  <meta name="citation_reference" content="“Introduction to Peptides and Proteins,” CRC Press. [Online]. Available: https://www.crcpress.com/Introduction-to-Peptides-and-Proteins/Langel-Cravatt-Graslund-von-Heijne-Zorko-Land-Niessen/p/book/9780367384876. [Accessed: 28-Jan-2020]."/>
  <meta name="twitter:title" content="DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Motivation Accurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm. Results We trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours. Availability The repository of this project will be published. Contact dongsi{at}uw.edu Supplementary information Supplementary data will be available at Bioinformatics online."/>
  <meta name="og-title" property="og:title" content="DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.12.946772v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Motivation Accurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm. Results We trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours. Availability The repository of this project will be published. Contact dongsi{at}uw.edu Supplementary information Supplementary data will be available at Bioinformatics online."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-13"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1149483 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <div id="sec-1" class="subsection">
     <p id="p-2">
      <strong>Motivation</strong> Accurately determining the atomic structure of proteins represents a fundamental problem in the field of structural bioinformatics. A solution would be significant as protein structure information could be utilized in the medical field, e.g. in the development of vaccines for new viruses. This paper focuses on predicting the protein structure based on 3D images of the proteins captured through cryogenic electron microscopes (cryo-EM). A fully automated computationally efficient protein structure prediction method would be particularly beneficial in the field of cryo-EM as the technology allows researchers to photograph multiple large protein complexes in a single study, which means that a fast prediction method could allow for a high throughput of derived protein structures. We present a deep learning approach, DeepTracer, for predicting locations of the backbone atoms, secondary structure elements, and the amino acid types. In order to connect the predicted amino acids into chains, we applied a modified traveling salesman algorithm.
     </p>
    </div>
    <div id="sec-2" class="subsection">
     <p id="p-3">
      <strong>Results</strong> We trained our deep learning model on experimental cryo-EM density maps and tested it on a set of 50 density maps. We found that our new approach predicted protein structures with an average RMSD value of 1.18 and a coverage of 87.5%. Furthermore, we detected secondary structure information for 87.2% of amino acids correctly. We also showed preliminarily that 25.2% of amino acid types could be predicted directly from the 3D cryo-EM density map, considering 20 different types in total. Finally, we noted that the prediction runtime of DeepTracer is significantly improved compared to other methods. It predicts a large protein complex structure of more than 30,000 amino acids in only 2 hours.
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <p id="p-4">
      <strong>Availability</strong> The repository of this project will be published.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <p id="p-5">
      <strong>Contact</strong>
      <span class="em-link">
       <span class="em-addr">dongsi{at}uw.edu</span>
      </span>
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <p id="p-6">
      <strong>Supplementary information</strong> Supplementary data will be available at 
      <em>Bioinformatics</em> online.
     </p>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">1 Introduction</h2>
    <p id="p-7">Proteins represent an essential building block in all organisms. They perform crucial functions such as DNA replication, responding to stimuli, and many more [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Each protein consists of a sequence of the same 20 kinds of amino acids, while all amino acids are made up of a carbon, carbon-alpha (Cα), and nitrogen atom, as well as a side chain which determines the amino acid type and is connected to the Cα atom. The functionality of a protein is determined through its structure, defined by the three-dimensional (3D) arrangement of the amino acids that make up the protein [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>].
    </p>
    <p id="p-8">As a consequence, knowledge about the structure of a protein helps to gain a deeper understanding about its behavior. This can be particularly useful, e.g., in the development of new vaccines and drugs for new viruses such as Novel Coronavirus (2019-nCoV) [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]–[
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], as the surface proteins of a virus determine how it will interact with a person [
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. Therefore, it is fundamentally important to be able to accurately predict the 3D structure of a protein.
    </p>
    <p id="p-9">There are two main approaches to predict protein structures: homology modeling and de novo structure prediction. They differ in such that homology modeling requires structural knowledge about a homologous protein for its prediction while de novo methods do not. This limits the application range of homology modeling, since homologues proteins must be obtained first before making predictions, which is why we aim at the more challenging de novo structure prediction. Traditionally, research on de novo protein structure prediction was focused on inferring the 3D structure through the protein’s amino acid sequence [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. This approach is beneficial as the determination of the sequence is relatively straightforward. Methods that follow this approach either use geometric calculations to estimate the protein structure [
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], [
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], or they use deep learning techniques, such as Google’s AlphaFold [
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. While these methodologies have led to impressive results, their accuracy is naturally limited as the amino acid sequence does not completely contain the protein’s structural information, especially for very large protein complexes [
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>].
    </p>
    <p id="p-10">Throughout the last decade, technological advancements in cryogenic electron microscopy (cryo-EM) have provided a new approach to solve the structure determination problem. Cryo-EM allows researchers to capture 3D images of large protein complexes by cooling them to cryogenic temperatures without the need of a costly crystallization of the protein [
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. In recent years the resolution of these images has increased to a near-atomic level which led to an exponential gain in popularity of the technology [
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. A 3D image captured through cryo-EM is an electron density map (see 
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1</a>). It represents the volume of the protein complex through a 3D grid of voxels. Each voxel stores the electron density value, meaning the probability that an electron is present, for its location. Latest developments in the field of cryo-EM have allowed researchers to capture many high-resolution density maps from large protein complexes in the course of a single study [
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. This makes the ability to predict protein structures from cryo-EM maps fully automatically even more significant as it allows for a very high throughput of protein structures derived from cryo-EM maps.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="EMD-6272 (rotavirus VP6) Density Map (left) and its corresponding solved structure PDB-3j9s depicted in ribbon (center) and backbone view (right). " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;EMD-6272 (rotavirus VP6) Density Map (left) and its corresponding solved structure PDB-3j9s depicted in ribbon (center) and backbone view (right).&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F1.medium.gif" width="440" height="261"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F1.medium.gif" width="440" height="261"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig 1.</span>
      <span class="caption-title">EMD-6272 (rotavirus VP6) Density Map (left) and its corresponding solved structure PDB-3j9s depicted in ribbon (center) and backbone view (right).</span>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">The structure of a protein can be estimated from its density map, intuitively by placing the amino acids in such way that the resulting structure fits well into the density map. While this sounds like a simple task it becomes very challenging due to lower local resolutions as well as noise created through errors in the cryo-EM. Currently, there are several existing methods that perform protein structure predictions based on cryo-EM data such as Phenix and Rosetta [
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>], [
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. They both use conventional algorithms to place and connect the amino acids of the protein. However, they often struggle to identify amino acids as such and, therefore, tend to have a low coverage on large protein complex [
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. Furthermore, their long runtime is a problem. The prediction of small density maps can take up to several days rendering the method practically useless for the prediction of large proteins. The C-CNN method, another protein structure prediction application which utilizes deep learning techniques, achieves better results in terms of coverage [
     <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]. However, it requires some manual input and the runtime could be further improved. Additionally, it predicts the location of Cα atoms without information about their amino acid type. Therefore, we present the DeepTracer, a new fully automated method for the prediction of protein structures from cryo-EM density maps, with the goal of improving the runtime by applying a 3D U-Net in combination with a modified travelling salesman algorithm and having the ability of recognizing the amino acid type from 3D cryo-EM density maps.
    </p>
    <p id="p-12">The functionality of our new DeepTracer is explained in the Methods section. In the Results section, the DeepTracer is applied on a test set of 30 density maps in order to assess and compare its accuracy with other methods. The implications of the project and possible future work is examined in Discussion. Finally, we recapture our findings in the Conclusion section.</p>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">2 Methods</h2>
    <p id="p-13">The full prediction pipeline of the DeepTracer includes many different steps. In this section we will look at the major ones and explain how they create a protein structure based of a density map. We begin by looking at the 3D U-Net which predicts confidence maps that show where e.g. atoms might be or where which secondary structure is likely to be present. Next, we will examine how the DeepTracer uses this information to place the actual amino acids, as well as how it connects them into chains to form the protein structure. Finally, we discuss how secondary structure information can be used to refine predicted α-helices.</p>
    <div id="sec-8" class="subsection">
     <h3>A. 3D U-Net Deep Learning Model</h3>
     <p id="p-14">The deep learning model represents the central entity in the structure prediction. Its job is to predict not only the locations of amino acids but also secondary structure positions and amino acid types. To explain its realization, we start by looking at its architecture. Then, we move on to the data collection and preprocessing steps and finally describe the model’s training process.</p>
     <p id="p-15">The U-Net model gets its name from the U-shape of its architecture which is optimized for segmentation tasks on medical images [
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. The detailed architecture of the model can be seen in 
      <strong>
       <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2</a>
      </strong>. The cryo-EM density maps are fed to the 64
      <sup>3</sup> input layer. The output layer has the same 64
      <sup>3</sup> shape with N different channels. The number of channels depends hereby on what the U-Net is predicting (e.g. the secondary structure U-Net has three channels for loops, sheets, and helices).
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Architecture of the U-Net as used by the DeepTracer. The blue boxes show the output maps of the different layers where the dimensions of the maps are depicted on the left and the number of channels is depicted on top." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Architecture of the U-Net as used by the DeepTracer.&lt;;/span&gt;; The blue boxes show the output maps of the different layers where the dimensions of the maps are depicted on the left and the number of channels is depicted on top.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F2.medium.gif" width="440" height="361"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F2.medium.gif" width="440" height="361"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 2.</span>
       <span class="caption-title">Architecture of the U-Net as used by the DeepTracer.</span>
       <p id="p-16" class="first-child">The blue boxes show the output maps of the different layers where the dimensions of the maps are depicted on the left and the number of channels is depicted on top.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-17">As mentioned above, the U-Net should predict multiple things, including the amino acid positions, secondary structure elements, as well as the amino acid types. Therefore, the model consists of three parallel U-Nets as shown in 
      <strong>
       <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3</a>
      </strong>. The first one is responsible for predicting the amino acid positions. As the Cα atom of each amino acid is central to its position we can reduce this task to only predicting the Cα atoms of the protein structure. Additionally, this U-Net also predicts the backbone location of the protein, which is defined as the position of all Cα, C, and N atoms of the structure. This prediction will later on be useful in order to connect the amino acids into chains. Thus, the output of the Cα atoms U-Net has two channels. The second U-Net is responsible for predicting the secondary structure elements and, therefore, its output has three channels, one for each structural element (α-helix, loop, and sheet). Finally, the third U-Net is tasked to predict the amino acid types of the protein structure. Since there are 20 different types of amino acids occurring in nature its output has the same number of channels. In 
      <strong>
       <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4</a>
      </strong> we can see an example output for each of the U-Nets with the density map from 
      <strong>
       <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1</a>
      </strong> as input.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Architecture of the deep learning model, which includes three parallel U-Nets. The blue boxes show the input and output maps of the model, with their dimensions noted to the left and the number of channels marked below." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Architecture of the deep learning model, which includes three parallel U-Nets.&lt;;/span&gt;; The blue boxes show the input and output maps of the model, with their dimensions noted to the left and the number of channels marked below.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F3.medium.gif" width="440" height="342"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F3.medium.gif" width="440" height="342"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 3.</span>
       <span class="caption-title">Architecture of the deep learning model, which includes three parallel U-Nets.</span>
       <p id="p-18" class="first-child">The blue boxes show the input and output maps of the model, with their dimensions noted to the left and the number of channels marked below.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Example output confidence maps of the model for the EMD-6272 density map. (A) and (B) are both the output of the C[[#x3B1]] atom U-Net and show the C[[#x3B1]] atom locations (A) as well as the backbone of the protein (B). (C) shows the output of the secondary structure U-Net with a-helices in yellow, loops in purple, and sheets in red. In (D) we can see the output of the amino acid type U-Net where each color represents a different type of amino acid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Example output confidence maps of the model for the EMD-6272 density map.&lt;;/span&gt;; (A) and (B) are both the output of the C[[#x3B1]] atom U-Net and show the C[[#x3B1]] atom locations (A) as well as the backbone of the protein (B). (C) shows the output of the secondary structure U-Net with a-helices in yellow, loops in purple, and sheets in red. In (D) we can see the output of the amino acid type U-Net where each color represents a different type of amino acid.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F4.medium.gif" width="440" height="254"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F4.medium.gif" width="440" height="254"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 4.</span>
       <span class="caption-title">Example output confidence maps of the model for the EMD-6272 density map.</span>
       <p id="p-19" class="first-child">(A) and (B) are both the output of the Cα atom U-Net and show the Cα atom locations (A) as well as the backbone of the protein (B). (C) shows the output of the secondary structure U-Net with a-helices in yellow, loops in purple, and sheets in red. In (D) we can see the output of the amino acid type U-Net where each color represents a different type of amino acid.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-20">Now that we have looked at the architecture of the model, we can continue with the data collection. In order for the deep learning model to learn common noises and errors present in cryo-EM density maps we decided to train the U-Net using experimental data rather than density maps simulated from solved structures such as [
      <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>]. We downloaded the density maps from the EMDataResource website [
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] in combination with their solved protein structures which function as the ground truth for the training process. We obtained the solved structures from the RCSB Protein Data Bank [
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. Since this paper is focused on high resolution maps, we only used density maps with a resolution of 4Å or better. In total we used 411 different density maps to build the training and validation sets.
     </p>
     <p id="p-21">The preprocessing steps can be split into preparing the cryo-EM density maps and creating the different labels based on the solved structures. For the experimental density maps we have to make sure that the voxel size equals one, meaning that every voxel measures 1Å. Furthermore, we need to adjust the dimensions of the map to the 64
      <sup>3</sup> input dimensions of the model as seen in 
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2</a>. This is done by splitting the maps into several 64
      <sup>3</sup> big cubes. In order to avoid errors at the borders of the cubes we overlay them partly such that we only use the 50
      <sup>3</sup> center voxels of each cube to reconstruct the output maps to the original dimensions of the density map. By splitting the 411 density maps that we collected in this manner, we retrieve 16,070 training examples. To create the labels that the model can be trained on, we use the solved structure to create masks by filtering out the corresponding atoms and setting all voxels within a certain distance to those atoms to one. For the Cα mask we filter only the Cα atoms of the protein structure and set all directly neighboring voxels around them to one. The backbone mask is created similarly except that we use all Cα, C, and N atoms. To generate the secondary structure masks, we filter out all atoms for each structural element and set all voxels within a distance of two around each filtered atom to one. Finally, to create the amino acid type masks we filter out all atoms for each amino acid type and set all neighboring voxels to one for each atom. In total this gives us 25 different masks for each training example (Cα mask, backbone mask, three secondary structure masks, and 20 amino acid type masks).
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>B. Placing Cα atoms</h3>
     <p id="p-22">As mentioned above, the determining factor for the position of an amino acid is the position of its Cα atom. In order to place the Cα atoms in the 3D space we use the Cα atoms confidence map as well as the backbone confidence map. The backbone map is used to identify different chains and the Cα atoms map to find the locations of the Cα atom.</p>
     <p id="p-23">Before we place any Cα atoms, we try to identify separate chains in the protein structure. This will help us later when connecting the atoms and improves the runtime of the connection process. In order to discern the chains, we identify connected areas of voxels that have a confidence value of 0.5 or higher. An example of the chains identified from a backbone prediction can be seen in 
      <strong>
       <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5</a>
      </strong>. We can now mask all other confidence maps with each chain area that we identified in order to continue the prediction independently for all chains.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Backbone prediction of EMD-0478 density map (human TPC2 channel) (human TPC2 channel) with identified chains marked in different colors. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Backbone prediction of EMD-0478 density map (human TPC2 channel) (human TPC2 channel) with identified chains marked in different colors.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F5.medium.gif" width="440" height="384"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F5.medium.gif" width="440" height="384"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 5.</span>
       <span class="caption-title">Backbone prediction of EMD-0478 density map (human TPC2 channel) (human TPC2 channel) with identified chains marked in different colors.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-24">Next, we place the actual Cα atoms in two simple steps, using the masked Cα confidence maps. First, we find the initial atom locations by calculating local minima in the confidence map within a neighborhood of one voxel and a minimum value of 0.6. The value of 0.6 was chosen after some testing as it produced the most accurate results. This gives us a list of voxel indices. However, as atoms can have floating point coordinates, we try to improve this initial location by calculating the center of mass for 3
      <sup>3</sup> cubes around each local minimum. The resulting values are used as the coordinates for the Cα atoms of the protein structure.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>C. Tracing Backbone</h3>
     <p id="p-25">In this prediction step, we use the predicted Cα atoms and connect them into chains using a travelling salesman algorithm (TSP) [
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. However, instead of using the Euclidean distances between atoms to measure the total length of the path, we use a custom confidence score that expresses how confident we are that two atoms share a connection. The task of the TSP algorithm is then to connect the atoms into chains such that they maximize the sum of all confidence scores.
     </p>
     <p id="p-26">The calculation of the confidence score between two atoms takes two factors into consideration: The Euclidean distance between them, as well as the average confidence of voxels that lay in between the atoms in the backbone prediction. Let 
      <em>p</em>(
      <em>x</em>, 
      <em>μ</em>, 
      <em>σ</em>) be the normal probability density function at 
      <em>x</em> with mean 
      <em>μ</em> and standard deviation 
      <em>σ</em>, 
      <em>d</em>
      <sub>
       <em>a,b</em>
      </sub> the Euclidean distance between atoms a and b, and 
      <em>b</em>
      <sub>
       <em>a,b</em>
      </sub> the average backbone confidence between atoms a and b. We define the confidence score as shown in 
      <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation 1</a>. The constant 3.8 was chosen as the mean for the distances between two atoms as this corresponds to the average distance found in nature. The standard deviation parameters of 5 and 0.3 where selected based on test runs. They were intentionally chosen higher than in the distribution found in solved structures, in order to cut some slack for prediction inaccuracies. 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/13/2020.02.12.946772/embed/graphic-6.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/13/2020.02.12.946772/embed/graphic-6.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-27">In order to apply the TSP algorithm, we also need to specify a start/end point. However, we do not know yet at which atom the chain will start and end. Therefore, we add a new atom that is connected to every other atom with a confidence of 1. This atom is then specified as the start/end and later on removed from the actual chain. An example of the application of the TSP on a list of Cα atoms can be seen in 
      <strong>
       <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6</a>
      </strong>.
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Predicted C[[#x3B1]] atoms for EMD-4054 density map (major capsid protein) (major capsid protein) in blue before (left) and after (right) the backbone tracing step compared to the true structure in pink. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Predicted C[[#x3B1]] atoms for EMD-4054 density map (major capsid protein) (major capsid protein) in blue before (left) and after (right) the backbone tracing step compared to the true structure in pink.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F6.medium.gif" width="440" height="194"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F6.medium.gif" width="440" height="194"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 6.</span>
       <span class="caption-title">Predicted Cα atoms for EMD-4054 density map (major capsid protein) (major capsid protein) in blue before (left) and after (right) the backbone tracing step compared to the true structure in pink.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-11" class="subsection">
     <h3>D. Helix Refinement</h3>
     <p id="p-28">The last prediction step is responsible for refining modelled α-helices. Here for, we can exploit the fact that the shape of an α-helix has a general definition which is valid across proteins [
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. Since the U-Net predicts the confidence of secondary structure elements, as shown in 
      <strong>
       <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4</a>
      </strong>, we know which amino acids belong to an α-helix based on the confidence of their region in space. We combine this knowledge of α-helix locations and their shape attributes in order to adjust the appropriate Cα atoms to better fit the shape of a natural α-helix structure.
     </p>
     <p id="p-29">For an α-helix which centers around the z-axis, we can use 
      <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> to model its shape where the variables 
      <em>s</em> and 
      <em>r</em> represent the initial shift and rotation of the helix. The values 
      <em>2.11</em> and 
      <em>1.149</em> are constants that define the radius and pitch of the helix to best match those of an α-helix. 
      <span class="disp-formula" id="disp-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/13/2020.02.12.946772/embed/graphic-8.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/13/2020.02.12.946772/embed/graphic-8.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-30">
      <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> however, cannot be used to describe an α-helix which does not center around the z-axis or whose shape is not a straight cylinder. Since this is the case for most α-helices, it is necessary to adjust the equation in such way that it will address these issues. With the aim of doing so, we first locate the screw axis, the center line around which the helix winds itself, for each α-helix. This is achieved by calculating the centroid of consecutive intervals of the α-helix and then connecting them to approximate the true curve. An example of an α-helix and its calculated screw axis can be seen in 
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig 7</a>.
     </p>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="[[#x3B1]]-Helix extracted from the backbone prediction of the EMD-8515 density map in tan color and its screw axis in teal color. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;[[#x3B1]]-Helix extracted from the backbone prediction of the EMD-8515 density map in tan color and its screw axis in teal color.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F7.medium.gif" width="440" height="95"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F7.medium.gif" width="440" height="95"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 7.</span>
       <span class="caption-title">α-Helix extracted from the backbone prediction of the EMD-8515 density map in tan color and its screw axis in teal color.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-31">Now that we know the location and shape of the screw axis for the α-helix, we need to incorporate this information into 
      <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a>. This is achieved by interpreting 
      <em>t</em> as the distance that we travelled on the screw axis and use the unit direction vector of the screw axis at a certain point 
      <em>t</em> as the new z axis. Next, we can find the new y-axis by calculating the cross product of the x-axis and the new z-axis and then normalizing it. Finally, we can get the new x-axis by calculating the cross product of the new z and y-axis and normalizing it again. By concatenating the three new axes we can get a rotation matrix 
      <strong>RM</strong> with which we can calculate the point of the α-helix for any value 
      <em>t</em> as shown in 
      <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>. 
      <span class="disp-formula" id="disp-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/13/2020.02.12.946772/embed/graphic-10.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/13/2020.02.12.946772/embed/graphic-10.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-32">Now, we need to know the values 
      <em>t</em> at which we have to insert Cα atoms. Since we know that an α-helix has a rise of 1.5Å per residue [
      <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>] we can increase 
      <em>t</em> in steps of 1.5 and add a new Cα atom at α-helix(
      <em>t</em>).
     </p>
     <p id="p-33">In the final step we minimize the average distance from the Cα atoms of the refined α-helix to the Cα atoms of the original prediction. This is done by applying a minimization algorithm
      <sup>
       <a id="xref-fn-1-1" class="xref-fn" href="#fn-1">1</a>
      </sup> over the variables 
      <em>s</em> and 
      <em>r</em> to try different initial shifts and rotations. The final results of the α-helix refinement step are shown in 
      <strong>
       <a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig 8</a>
      </strong>.
     </p>
     <div id="F8" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="[[#x3B1]]-Helix extracted from the prediction of the EMD-8515 density map. (Top) Original prediction before the helix-refinement step. (Center) [[#x3B1]]-Helix after the refinement. (Bottom) Direct comparison of original prediction in tan color, refined prediction colored in teal, and the solved structure (PDB-5u70) in pink color." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;[[#x3B1]]-Helix extracted from the prediction of the EMD-8515 density map.&lt;;/span&gt;; (Top) Original prediction before the helix-refinement step. (Center) [[#x3B1]]-Helix after the refinement. (Bottom) Direct comparison of original prediction in tan color, refined prediction colored in teal, and the solved structure (PDB-5u70) in pink color.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F8.medium.gif" width="440" height="262"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F8.medium.gif" width="440" height="262"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 8." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig 8.</span>
       <span class="caption-title">α-Helix extracted from the prediction of the EMD-8515 density map.</span>
       <p id="p-34" class="first-child">(Top) Original prediction before the helix-refinement step. (Center) α-Helix after the refinement. (Bottom) Direct comparison of original prediction in tan color, refined prediction colored in teal, and the solved structure (PDB-5u70) in pink color.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Results</h2>
    <p id="p-35">In order to gain a better understanding about the performance of the DeepTracer we will apply it to a test set of 30 experimental density maps and evaluate the predictions using different metrics. Additionally, we will compare its performance against Phenix, as well as the C-CNN backbone prediction method.</p>
    <p id="p-36">We measure the accuracy of a predicted protein structure by comparing it to its corresponding solved structure using the following metrics. First, the root-mean-square deviation (RMSD) value describes the average distance between atoms of the predicted and solved protein structure. This gives us an idea how close the predicted atoms are to their correct counter parts. However, this does not give us any information about the percentage of atoms that we have correctly identified. For this purpose, we calculate the coverage value, which expresses the percentage of atoms in the solved structure that lay within 3Å of a predicted atom. Finally, we evaluate the false-positive percentage which is the number of atoms that were predicted further than 3Å from any atom in the solved structure, divided by the total number of predicted atoms. In addition to the metrics that we use to compare the other methods to the DeepTracer, we also calculate the secondary structure and amino acid type accuracy. We cannot evaluate these for the other methods as they utilize the U-Net confidence maps rather than the predicted protein structure. The secondary structure accuracy is defined as the percentage of atoms from the solved structure whose secondary structure type equals the one predicted by the U-Net for its location. The amino acid type accuracy is calculated in the same way. In 
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, we can see the test results for all 30 density maps.
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1152518/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1152518/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1152518" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1.</span>
      <p id="p-37" class="first-child">Prediction results of the Phenix, C-CNN, and DeepTracer methods for 30 experimental density map.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-38">In order to get a better idea of the implications of the results we take a closer look at the protein structures predicted by the DeepTracer and Phenix for the EMD-6272 density map (rotavirus VP6) and compare them to the solved structure (see 
     <strong>
      <a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig 9</a>
     </strong>). We can see in the highlighted areas that the DeepTracer incorrectly connected amino acids in two cases. However, all predicted amino acids are still connected as a single chain. In comparison to that the prediction of Phenix is fragmented into multiple chains. Furthermore, amino acids appear to be missing from the prediction explaining the coverage difference of 8.1% (see 
     <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The predicted secondary structure elements in the Phenix prediction, particularly for sheets, appear also much less accurate than the ones made by the DeepTracer. Another noteworthy observation, although not a fair comparison as Phenix predicts more than just the backbone structure, is that the prediction runtime of Phenix for the EMD-6272 map was 1 hour and 45 minutes while the runtime of the DeepTracer for the same map was 40 seconds.
    </p>
    <div id="F9" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Solved PDB-3j9s protein structure (blue) and predictions by the DeepTracer (green) and Phenix (yellow) for EMD-6272 density map (rotavirus VP6). Highlighted areas show where the DeepTracer connected the amino acids incorrectly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Solved PDB-3j9s protein structure (blue) and predictions by the DeepTracer (green) and Phenix (yellow) for EMD-6272 density map (rotavirus VP6).&lt;;/span&gt;; Highlighted areas show where the DeepTracer connected the amino acids incorrectly.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F9.medium.gif" width="440" height="299"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig 9." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F9.medium.gif" width="440" height="299"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 9." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig 9.</span>
      <span class="caption-title">Solved PDB-3j9s protein structure (blue) and predictions by the DeepTracer (green) and Phenix (yellow) for EMD-6272 density map (rotavirus VP6).</span>
      <p id="p-39" class="first-child">Highlighted areas show where the DeepTracer connected the amino acids incorrectly.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-40">As mentioned in the introduction, a central goal of the DeepTracer was to improve the runtime of protein structure predictions. This makes it possible to predict ever larger density maps which allows for a more versatile application of the method. In order to compare the runtime of the DeepTracer with the C-CNN method we measure the execution time of each method for the prediction of multiple density maps. A scatter plot of the measured times is depicted in 
     <a id="xref-fig-10-1" class="xref-fig" href="#F10">Fig 10</a>.
    </p>
    <div id="F10" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Scatter plot of prediction runtimes (y-axis) of the DeepTracer (blue) and the C-CNN method (red) for various density maps against the number of predicted atoms (x-axis). In order to see the runtimes more clearly, we use a logarithmic scale with a base of 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Scatter plot of prediction runtimes (y-axis) of the DeepTracer (blue) and the C-CNN method (red) for various density maps against the number of predicted atoms (x-axis).&lt;;/span&gt;; In order to see the runtimes more clearly, we use a logarithmic scale with a base of 4.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig 10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F10.medium.gif" width="440" height="287"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig 10." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F10.medium.gif" width="440" height="287"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 10." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig 10.</span>
      <span class="caption-title">Scatter plot of prediction runtimes (y-axis) of the DeepTracer (blue) and the C-CNN method (red) for various density maps against the number of predicted atoms (x-axis).</span>
      <p id="p-41" class="first-child">In order to see the runtimes more clearly, we use a logarithmic scale with a base of 4.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-42">In order to evaluate the runtime of the DeepTracer for larger proteins we predicted the EMD-4903 density map (echovirus 1 intact particle). The number of predicted atoms for this map was 32,446. The execution of the end-to-end prediction pipeline of the DeepTracer took only two hours. As a point of comparison, the expected runtime of the C-CNN method for the same density map is more than 5 days assuming the trend in 
     <a id="xref-fig-11-1" class="xref-fig" href="#F11">Fig 11</a> continues for larger proteins. The original density map next to the prediction of the DeepTracer can be seen in 
     <a id="xref-fig-11-2" class="xref-fig" href="#F11">Fig 11</a>.
    </p>
    <div id="F11" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F11.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Density map (left) of EMD-4903 (echovirus 1 intact particle) and its predicted protein structure (right), containing more than 30,000 atoms. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Density map (left) of EMD-4903 (echovirus 1 intact particle) and its predicted protein structure (right), containing more than 30,000 atoms.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig 11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F11.medium.gif" width="440" height="352"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig 11." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F11.medium.gif" width="440" height="352"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 11." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig 11.</span>
      <span class="caption-title">Density map (left) of EMD-4903 (echovirus 1 intact particle) and its predicted protein structure (right), containing more than 30,000 atoms.</span>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-43">In addition to the location and secondary structure of amino acids, the DeepTracer also predicts their types. As reported in 
     <strong>
      <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>
     </strong> we assigned the correct type out of the 20 possible amino acid types with an accuracy of 25.2%. In order to get a better idea of what that result means we look at a specific structure prediction. For the EMD-6272 density map, we predicted the correct amino acid type for 31.49% of amino acids. Note that this is predicted solely from 3D map without using any genetic information of the protein. In 
     <strong>
      <a id="xref-fig-12-1" class="xref-fig" href="#F12">Fig 12</a>
     </strong> we can see an extract of the amino acid sequence as predicted by the DeepTracer, compared to the sequence of the solved structure. The sequence was created by walking along the predicted chain of amino acids and noting down the type of each amino acid. The predicted amino acids from 3D could potentially be utilized for the reference of future full-atom structure prediction.
    </p>
    <div id="F12" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F12.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Extract of the amino acid sequence predicted by the DeepTracer for the EMD-6272 density map (top) compared to the sequence of the solved structure PDB-3j9s (bottom). Correctly predicted amino acids are highlighted in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1726861249" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Extract of the amino acid sequence predicted by the DeepTracer for the EMD-6272 density map (top) compared to the sequence of the solved structure PDB-3j9s (bottom).&lt;;/span&gt;; Correctly predicted amino acids are highlighted in red.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig 12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F12.medium.gif" width="440" height="268"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig 12." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F12.medium.gif" width="440" height="268"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 12." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.12.946772/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig 12.</span>
      <span class="caption-title">Extract of the amino acid sequence predicted by the DeepTracer for the EMD-6272 density map (top) compared to the sequence of the solved structure PDB-3j9s (bottom).</span>
      <p id="p-44" class="first-child">Correctly predicted amino acids are highlighted in red.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-13">
    <h2 class="">3 Discussion</h2>
    <p id="p-45">The final goal of the DeepTracer is to accurately predict the complete protein structure from high resolution cryo-EM density maps. In this section we evaluate how close we are to this goal, based on the results presented in the previous section, as well as discuss what further work is necessary.</p>
    <p id="p-46">When looking at the results reported in 
     <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, we can note a couple of things. First, we see that the percentage of false positives predicted by the DeepTracer has increased by 1.9% compared to the C-CNN method. This can be explained through the manual thresholding which reduces noises in the experimental density maps and is utilized in the C-CNN method. However, considering that this step introduces manual effort to the prediction process, this is an acceptable concession as our new method is fully automated. Next, we see that the coverage percentage of the DeepTracer beats Phenix by over 20%. However, it still lacks behind the C-CNN method by 3%. On the contrary, Phenix performs best for RMSD, beating the DeepTracer by 0.08 and the C-CNN method by 0.09. If we compare the DeepTracer to Phenix based on these two metrics, we can see that the coverage improvements significantly outweigh the small deterioration in RMSD. However, compared to the C-CNN method there is no significant difference regarding the RMSD and coverage. As mentioned in the introduction, we wanted to focus not only on the prediction accuracy, but also on the runtime of the predictions. And here we can discern immense differences. For the prediction of smaller structures. the C-CNN method took at least 20 minutes while the DeepTracer ran small predictions in a matter of seconds. However, the bigger implications lay in the prediction of larger proteins. For around 6000 atoms the C-CNN method took over 24 hours to finish, whereas the DeepTracer only needed around 20 minutes, for some even less than 5 minutes. Particularly, the trendline is interesting as it gives us an idea of how the methods will behave for very large proteins. The C-CNN trendline seems almost linear above 2000 predicted proteins and the runtime seems to quadruple when 6000 predicted atoms are reached. If this trend continues then we could expect a 75-fold runtime increase in order to predict 100,000 atoms. The runtimes of DeepTracer, however, seem to remain flat once a certain number of atoms is reached which would mean that very large proteins could be predicted within a reasonable runtime. This can be explained by the chain identification step that is applied before the prediction of any atom. While larger proteins consist of more chains than smaller ones, the number of atoms in the chains does not necessarily grow. As each chain is processed separately this means that if the chain lengths remain constant and only the number of chains gets larger the runtime grows linearly with the number of chains.
    </p>
    <p id="p-47">In addition to the false-positive percentage, the coverage, and the RMSD we also calculated the secondary structure and amino acid type accuracy for the DeepTracer using the confidence maps predicted by U-Net. We can see that the secondary structure prediction performs well with an average accuracy of 81.72%. We also show that deep learning could discern different types of amino acids based on the density maps. Particularly, as this is only the preliminary stage of predicting amino acid type directly from 3D cryo-EM density map. It gives promise for future improvements through a more elaborate training of the U-Net with more data. Furthermore, we plan to include the amino acid sequence as an input parameter, so that we can map the predicted sequence with the input sequence and refine our prediction. For this sequence mapping work, an amino acid type prediction accuracy of around 30% from 3D could be already helpful. Particularly, as we can see from 
     <strong>
      <a id="xref-fig-12-2" class="xref-fig" href="#F12">Fig 12</a>
     </strong> that the U-Net tends to correctly predict the type of consecutive amino acids, likely due to high local resolutions. As it is very unlikely that this happens by chance, we can use this information to map the input sequence to our 3D backbone atom and amino acids prediction.
    </p>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">4 Conclusion</h2>
    <p id="p-48">In this paper, we presented the DeepTracer, a fully automated tool which can accurately predict backbone protein structures based on 3D cryo-EM density maps. In terms of accuracy, we were able to outperform the established structure prediction method Phenix by over 20% in structure coverage with only a slight RMSD increase of 0.08. Compared to the deep learning-based C-CNN method we achieved a slightly better RMSD by a margin of 0.01 with a slightly decreased coverage of 3%. In contrast with the C-CNN method, this fully automated method requires no manually set parameters. And the vastly reduced runtime allows us to predict protein structures within minutes instead of days. This is significant, as it allows the researchers to apply the DeepTracer on very large density maps and expect results maximally within a couple of hours. Furthermore, we achieved preliminary results in the amino acid type prediction from 3D with an accuracy of 25.2%. This gives promise to the incorporation of amino acid sequence into the prediction pipeline. Since the predicted amino acids from 3D could potentially be utilized for the reference of future full-atom structure prediction. In the future, we expect to use the amino acid type information to extend the functionality of the DeepTracer such that it can predict the protein full atom structure, including the side chains of the amino acids.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn" id="fn-1">
      <p id="p-49">
       <a class="rev-xref" href="#xref-fn-1-1">↵</a>
       <span class="fn-label">1</span>
       <a href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html">https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">[1].</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.12.946772v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C. I.</span>
          <span class="cit-name-surname">Branden</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Tooze</span>
         </span>, 
         <span class="cit-source">Introduction to Protein Structure</span>. 
         <span class="cit-publ-name">Garland Science</span>, 
         <span class="cit-pub-date">2012</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[2].</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.12.946772v1.2" data-doi="10.1016/S0065-3233(08)60520-3">
       <div class="cit-metadata">
        <ol class="duplicate">
         <li>
          <span class="cit-ed">
           <span class="cit-name-given-names">C. B.</span>
           <span class="cit-name-surname">Anfinsen</span>
          </span>, 
         </li>
         <li>
          <span class="cit-ed">
           <span class="cit-name-given-names">J. T.</span>
           <span class="cit-name-surname">Edsall</span>
          </span>, and 
         </li>
         <li>
          <span class="cit-ed">
           <span class="cit-name-given-names">F. M.</span>
           <span class="cit-name-surname">Richards</span>
          </span>
         </li>
        </ol>
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J. S.</span>
          <span class="cit-name-surname">Richardson</span>
         </span>, “
         <span class="cit-chapter-title">The Anatomy and Taxonomy of Protein Structure</span>,” in 
         <span class="cit-source">Advances in Protein Chemistry</span>, vol. 
         <span class="cit-vol">34</span>, 
         <span class="cit-ed">
          <span class="cit-name-given-names">C. B.</span>
          <span class="cit-name-surname">Anfinsen</span>
         </span>, 
         <span class="cit-ed">
          <span class="cit-name-given-names">J. T.</span>
          <span class="cit-name-surname">Edsall</span>
         </span>, and 
         <span class="cit-ed">
          <span class="cit-name-given-names">F. M.</span>
          <span class="cit-name-surname">Richards</span>
         </span>, Eds. 
         <span class="cit-publ-name">Academic Press</span>, 
         <span class="cit-pub-date">1981</span>, pp. 
         <span class="cit-fpage">167</span>–
         <span class="cit-lpage">339</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bprotein%2Bchemistry%26rft.stitle%253DAdv%2BProtein%2BChem%26rft.aulast%253DRichardson%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D34%26rft.spage%253D167%26rft.epage%253D339%26rft.atitle%253DThe%2Banatomy%2Band%2Btaxonomy%2Bof%2Bprotein%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3233%252808%252960520-3%26rft_id%253Dinfo%253Apmid%252F7020376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0065-3233(08)60520-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=7020376&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.946772.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[3].</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.3">
       <div class="cit-metadata">
        <cite>“
         <span class="cit-article-title">Wuhan coronavirus 2019-nCoV - what we can find out on a structural bioinformatics level</span>,” 
         <abbr class="cit-jnl-abbrev">Innophore</abbr>, 
         <span class="cit-day">23</span>-
         <span class="cit-month">Jan</span>-
         <span class="cit-pub-date">2020</span>. [Online]. Available: 
         <a href="https://innophore.com/2019-ncov/">https://innophore.com/2019-ncov/</a>. [Accessed: 
         <span class="cit-date-in-citation">30-Jan-2020</span>].
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[4].</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.12.946772v1.4" data-doi="10.1007/s11427-020-1637-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Xu</span>
         </span>
         <span class="cit-etal">et al.</span>, “
         <span class="cit-article-title">Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission</span>,” 
         <abbr class="cit-jnl-abbrev">Sci. CHINA Life Sci.</abbr>, 
         <span class="cit-month">Jan</span>. 
         <span class="cit-pub-date">2020</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1007/s11427-020-1637-5</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci.%2BCHINA%2BLife%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11427-020-1637-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s11427-020-1637-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[5].</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.5" data-doi="10.3390/v12020135">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">L. E.</span>
          <span class="cit-name-surname">Gralinski</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">V. D.</span>
          <span class="cit-name-surname">Menachery</span>
         </span>, “
         <span class="cit-article-title">Return of the Coronavirus: 2019-nCoV</span>,” 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>, vol. 
         <span class="cit-vol">12</span>, no. 
         <span class="cit-issue">2</span>, p. 
         <span class="cit-fpage">135</span>, 
         <span class="cit-month">Feb</span>. 
         <span class="cit-pub-date">2020</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.3390/v12020135</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D12%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv12020135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3390/v12020135&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[6].</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.12.946772v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Corradin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">V.</span>
          <span class="cit-name-surname">Villard</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. V.</span>
          <span class="cit-name-surname">Kajava</span>
         </span>, “
         <span class="cit-article-title">Protein Structure Based Strategies for Antigen Discovery and Vaccine Development Against Malaria and Other Pathogens</span>,” 
         <span class="cit-pub-date">2007</span>. [Online]. Available: 
         <a href="https://www.ingentaconnect.com/content/ben/emiddt/2007/00000007/00000004/art00004">https://www.ingentaconnect.com/content/ben/emiddt/2007/00000007/00000004/art00004</a>. [Accessed: 
         <span class="cit-date-in-citation">14-Jan-2020</span>].
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[7].</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.12.946772v1.7">
       <div class="cit-metadata">
        <cite>“
         <span class="cit-article-title">Application of the Three-Dimensional Structures of Protein Target Molecules in Structure-Based Drug Design | Journal of Medicinal Chemistry</span>.” [Online]. Available: 
         <a href="https://pubs.acs.org/doi/pdf/10.1021/jm00034a001">https://pubs.acs.org/doi/pdf/10.1021/jm00034a001</a>. [Accessed: 
         <span class="cit-date-in-citation">07-Jul-2019</span>].
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[8].</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.8" data-doi="10.1073/pnas.83.17.6233">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D. F.</span>
          <span class="cit-name-surname">Hunt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. R.</span>
          <span class="cit-name-surname">Yates</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Shabanowitz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Winston</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. R.</span>
          <span class="cit-name-surname">Hauer</span>
         </span>, “
         <span class="cit-article-title">Protein sequencing by tandem mass spectrometry</span>,” 
         <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr>, vol. 
         <span class="cit-vol">83</span>, no. 
         <span class="cit-issue">17</span>, pp. 
         <span class="cit-fpage">6233</span>–
         <span class="cit-lpage">6237</span>, 
         <span class="cit-month">Sep</span>. 
         <span class="cit-pub-date">1986</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.83.17.6233</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHunt%26rft.auinit1%253DD.%2BF.%26rft.volume%253D83%26rft.issue%253D17%26rft.spage%253D6233%26rft.epage%253D6237%26rft.atitle%253DProtein%2Bsequencing%2Bby%2Btandem%2Bmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.83.17.6233%26rft_id%253Dinfo%253Apmid%252F3462691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODMvMTcvNjIzMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzEzLzIwMjAuMDIuMTIuOTQ2NzcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[9].</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.12.946772v1.9" data-doi="10.1016/S0076-6879(96)66034-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Garnier</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-F.</span>
          <span class="cit-name-surname">Gibrat</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Robson</span>
         </span>, “
         <span class="cit-chapter-title">[32] GOR method for predicting protein secondary structure from amino acid sequence</span>,” in 
         <span class="cit-source">Methods in Enzymology</span>, vol. 
         <span class="cit-vol">266</span>, 
         <span class="cit-publ-name">Academic Press</span>, 
         <span class="cit-pub-date">1996</span>, pp. 
         <span class="cit-fpage">540</span>–
         <span class="cit-lpage">553</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DGarnier%26rft.auinit1%253DJ.%26rft.volume%253D266%26rft.spage%253D540%26rft.epage%253D553%26rft.atitle%253DGOR%2Bmethod%2Bfor%2Bpredicting%2Bprotein%2Bsecondary%2Bstructure%2Bfrom%2Bamino%2Bacid%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252896%252966034-0%26rft_id%253Dinfo%253Apmid%252F8743705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0076-6879(96)66034-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=8743705&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.946772.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1996BF77C00032&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[10].</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.10" data-doi="10.3109/10409239509085139">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Eisenhaber</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Persson</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Argos</span>
         </span>, “
         <span class="cit-article-title">Protein Structure Prediction: Recognition of Primary, Secondary, and Tertiary Structural Features from Amino Acid Sequence</span>,” 
         <abbr class="cit-jnl-abbrev">Crit. Rev. Biochem. Mol. Biol.</abbr>, vol. 
         <span class="cit-vol">30</span>, no. 
         <span class="cit-issue">1</span>, pp. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">94</span>, 
         <span class="cit-month">Jan</span>. 
         <span class="cit-pub-date">1995</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.3109/10409239509085139</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCritical%2BReviews%2Bin%2BBiochemistry%2Band%2BMolecular%2BBiology%26rft.stitle%253DCRIT%2BREV%2BBIOCHEM%2BMOL%2BBIOL%26rft.aulast%253DEisenhaber%26rft.auinit1%253DF%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D94%26rft.atitle%253DProtein%2Bstructure%2Bprediction%253A%2Brecognition%2Bof%2Bprimary%252C%2Bsecondary%252C%2Band%2Btertiary%2Bstructural%2Bfeatures%2Bfrom%2Bamino%2Bacid%2Bsequence%26rft_id%253Dinfo%253Adoi%252F10.3109%252F10409239509085139%26rft_id%253Dinfo%253Apmid%252F7587278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3109/10409239509085139&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=7587278&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.946772.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1995QK25500001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[11].</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.11" data-doi="10.1038/s41586-019-1923-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. W.</span>
          <span class="cit-name-surname">Senior</span>
         </span>
         <span class="cit-etal">et al.</span>, “
         <span class="cit-article-title">Improved protein structure prediction using potentials from deep learning</span>,” 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, pp. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">5</span>, 
         <span class="cit-month">Jan</span>. 
         <span class="cit-pub-date">2020</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-019-1923-7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1923-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-019-1923-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[12].</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.12" data-doi="10.1016/j.febslet.2005.01.014">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Contreras-Moreira</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">I.</span>
          <span class="cit-name-surname">Ezkurdia</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. L.</span>
          <span class="cit-name-surname">Tress</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Valencia</span>
         </span>, “
         <span class="cit-article-title">Empirical limits for template-based protein structure prediction: the CASP5 example</span>,” 
         <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr>, vol. 
         <span class="cit-vol">579</span>, no. 
         <span class="cit-issue">5</span>, pp. 
         <span class="cit-fpage">1203</span>–
         <span class="cit-lpage">1207</span>, 
         <span class="cit-pub-date">2005</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.febslet.2005.01.014</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DContreras-Moreira%26rft.auinit1%253DB.%26rft.volume%253D579%26rft.issue%253D5%26rft.spage%253D1203%26rft.epage%253D1207%26rft.atitle%253DEmpirical%2Blimits%2Bfor%2Btemplate-based%2Bprotein%2Bstructure%2Bprediction%253A%2Bthe%2BCASP5%2Bexample.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2005.01.014%26rft_id%253Dinfo%253Apmid%252F15710414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.febslet.2005.01.014&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15710414&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.946772.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[13].</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.13" data-doi="10.1016/j.cub.2018.07.016">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Nogales</span>
         </span>, “
         <span class="cit-article-title">Cryo-EM</span>,” 
         <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr>, vol. 
         <span class="cit-vol">28</span>, no. 
         <span class="cit-issue">19</span>, pp. 
         <span class="cit-fpage">R1127</span>–
         <span class="cit-lpage">R1128</span>, 
         <span class="cit-month">Oct</span>. 
         <span class="cit-pub-date">2018</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.cub.2018.07.016</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D28%26rft.spage%253DR1127%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2018.07.016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cub.2018.07.016&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[14].</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.14" data-doi="10.1016/j.cell.2016.05.040">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Merk</span>
         </span>
         <span class="cit-etal">et al.</span>, “
         <span class="cit-article-title">Breaking Cryo-EM Resolution Barriers to Facilitate Drug Discovery</span>,” 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, vol. 
         <span class="cit-vol">165</span>, no. 
         <span class="cit-issue">7</span>, pp. 
         <span class="cit-fpage">1698</span>–
         <span class="cit-lpage">1707</span>, 
         <span class="cit-month">Jun</span>. 
         <span class="cit-pub-date">2016</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2016.05.040</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D165%26rft.spage%253D1698%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.040%26rft_id%253Dinfo%253Apmid%252F27238019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.040&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27238019&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.946772.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[15].</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.15" data-doi="10.1073/pnas.1821959116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>
         <span class="cit-etal">et al.</span>, “
         <span class="cit-article-title">Cryo-EM structures of Helicobacter pylori vacuolating cytotoxin A oligomeric assemblies at near-atomic resolution</span>,” 
         <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr>, vol. 
         <span class="cit-vol">116</span>, no. 
         <span class="cit-issue">14</span>, pp. 
         <span class="cit-fpage">6800</span>–
         <span class="cit-lpage">6805</span>, 
         <span class="cit-month">Apr</span>. 
         <span class="cit-pub-date">2019</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1821959116</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1821959116%26rft_id%253Dinfo%253Apmid%252F30894496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE2LzE0LzY4MDAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xMy8yMDIwLjAyLjEyLjk0Njc3Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[16].</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.16" data-doi="10.1107/S0907444909052925">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P. D.</span>
          <span class="cit-name-surname">Adams</span>
         </span>
         <span class="cit-etal">et al.</span>, “
         <span class="cit-article-title">
          <em>PHENIX</em>: a comprehensive Python-based system for macromolecular structure solution
         </span>,” 
         <abbr class="cit-jnl-abbrev">Acta Crystallogr. D Biol. Crystallogr.</abbr>, vol. 
         <span class="cit-vol">66</span>, no. 
         <span class="cit-issue">2</span>, pp. 
         <span class="cit-fpage">213</span>–
         <span class="cit-lpage">221</span>, 
         <span class="cit-month">Feb</span>. 
         <span class="cit-pub-date">2010</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1107/S0907444909052925</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DAdams%26rft.auinit1%253DP.%2BD.%26rft.volume%253D66%26rft.issue%253DPt%2B2%26rft.spage%253D213%26rft.epage%253D221%26rft.atitle%253DPHENIX%253A%2Ba%2Bcomprehensive%2BPython-based%2Bsystem%2Bfor%2Bmacromolecular%2Bstructure%2Bsolution.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444909052925%26rft_id%253Dinfo%253Apmid%252F20124702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S0907444909052925&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20124702&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.946772.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000273820800013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[17].</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.12.946772v1.17" data-doi="10.1016/S0076-6879(04)83004-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C. A.</span>
          <span class="cit-name-surname">Rohl</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. E. M.</span>
          <span class="cit-name-surname">Strauss</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K. M. S.</span>
          <span class="cit-name-surname">Misura</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Baker</span>
         </span>, “
         <span class="cit-chapter-title">Protein Structure Prediction Using Rosetta</span>,” in 
         <span class="cit-source">Methods in Enzymology</span>, vol. 
         <span class="cit-vol">383</span>, 
         <span class="cit-publ-name">Academic Press</span>, 
         <span class="cit-pub-date">2004</span>, pp. 
         <span class="cit-fpage">66</span>–
         <span class="cit-lpage">93</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DRohl%26rft.auinit1%253DC.%2BA.%26rft.volume%253D383%26rft.spage%253D66%26rft.epage%253D93%26rft.atitle%253DProtein%2Bstructure%2Bprediction%2Busing%2BRosetta.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252804%252983004-0%26rft_id%253Dinfo%253Apmid%252F15063647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0076-6879(04)83004-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15063647&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.12.946772.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000221514500004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[18].</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.18" data-doi="10.1101/572990">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S. A.</span>
          <span class="cit-name-surname">Moritz</span>
         </span>
         <span class="cit-etal">et al.</span>, “
         <span class="cit-article-title">Cascaded-CNN: Deep Learning to Predict Protein Backbone Structure from High-Resolution Cryo-EM Density Maps</span>,” 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, p. 
         <span class="cit-fpage">572990</span>, 
         <span class="cit-month">Mar</span>. 
         <span class="cit-pub-date">2019</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1101/572990</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F572990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI1NzI5OTB2MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzEzLzIwMjAuMDIuMTIuOTQ2NzcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[19].</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Ronneberger</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Fischer</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Brox</span>
         </span>, “
         <span class="cit-article-title">U-Net: Convolutional Networks for Biomedical Image Segmentation</span>,”
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span>
         <span class="cit-pub-id-scheme">ArXiv</span>
         <span class="cit-pub-id cit-pub-id-arxiv">150504597</span> Cs, 
         <span class="cit-month">May</span>
         <span class="cit-pub-date">2015</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[20].</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.12.946772v1.20">
       <div class="cit-metadata">
        <cite>“
         <span class="cit-article-title">EMDataResource</span>.” [Online]. Available: 
         <a href="https://www.emdataresource.org/">https://www.emdataresource.org/</a>. [Accessed: 
         <span class="cit-date-in-citation">10-Jan-2020</span>].
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[21].</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.12.946772v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R. P. D.</span>
          <span class="cit-name-surname">Bank</span>
         </span>, “
         <span class="cit-article-title">RCSB PDB: Homepage</span>.” [Online]. Available: 
         <a href="https://www.rcsb.org/">https://www.rcsb.org/</a>. [Accessed: 
         <span class="cit-date-in-citation">10-Jan-2020</span>].
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[22].</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.22">
       <div class="cit-metadata">
        <cite>“
         <span class="cit-article-title">Traveling Salesman Problem | OR-Tools</span>,” 
         <abbr class="cit-jnl-abbrev">Google Developers</abbr>. [Online]. Available: 
         <a href="https://developers.google.com/optimization/routing/tsp">https://developers.google.com/optimization/routing/tsp</a>. [Accessed: 
         <span class="cit-date-in-citation">22-Jan-2020</span>].
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[23].</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.12.946772v1.23">
       <div class="cit-metadata">
        <cite>“
         <span class="cit-chapter-title">Introduction to Peptides and Proteins</span>,” 
         <span class="cit-publ-name">CRC Press</span>. [Online]. Available: 
         <a href="https://www.crcpress.com/Introduction-to-Peptides-and-Proteins/Langel-Cravatt-Graslund-von-Heijne-Zorko-Land-Niessen/p/book/9780367384876">https://www.crcpress.com/Introduction-to-Peptides-and-Proteins/Langel-Cravatt-Graslund-von-Heijne-Zorko-Land-Niessen/p/book/9780367384876</a>. [Accessed: 
         <span class="cit-date-in-citation">28-Jan-2020</span>].
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
