<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Med Virol</span><span class="iso-abbrev" title="iso-abbrev">J. Med. Virol</span><span class="doi" title="doi">10.1002/(ISSN)1096-9071</span><span class="publisher-id" title="publisher-id">JMV</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Medical Virology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0146-6615</span><span class="issn-epub" tagx="issn" title="issn-epub">1096-9071</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7228365">7228365</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/jmv.25777">10.1002/jmv.25777</a></span><span class="publisher-id" title="publisher-id">JMV25777</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Letter to the Editor</span>: <span class="subject" title="subject">Letter to the Editor</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Might the many positive COVID19 subjects in Italy have been caused by resident batâ€�derived zoonotic Î²â€�coronaviruses instead of the Wuhan (China) outbreak?</div> <div class="alt-title" title="alt-title">
LETTER TO THE EDITOR</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chirumbolo</span><span class="given-names" tagx="given-names" title="given-names">Salvatore</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1789-8307</span><a href="#jmv25777-aff-0001"><sup>1</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">salvatore.chirumbolo@univr.it</span></span></span></div><span class="citation_author_institution" id="jmv25777-aff-0001">[1], <span class="named-content" title="named-content">Department of Neurosciences Biomedicine and Movement Sciences</span><span class="institution" title="institution">University of Verona</span><span class="city" tagx="city" title="city">Verona</span><span class="country" tagx="country" title="country">Italy</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Correspondence</b> Dr Salvatore Chirumbolo, Department of Neurosciences, Biomedicine and Movement Sciences, University of Verona, Strada Le Grazie 9 37134 Verona, Italy. Email <span class="email" tagx="email" title="email">salvatore.chirumbolo@univr.it</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-3-3</span></span><span class="elocation-id" tagx="elocation-id" title="elocation-id">10.1002/jmv.25777</span><span class="history" title="history"><span class="received" title="received">received: 2020-3-03</span><span class="accepted" title="accepted">accepted: 2020-3-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is being made freely available through PubMed Central as part of the COVID-19 public health emergency response. It can be used for unrestricted research re-use and analysis in any form or by any means with acknowledgement of the original source, for the duration of the public health emergency.</span></span></div><a class="self-uri" href="file:JMV-9999-na.pdf" title="self-uri">file:JMV-9999-na.pdf</a><div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="edited-state" value="corrected-proof" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_JATSPMC version:5.8.0 mode:remove_FC converted:16.04.2020" /> </div> </div> </div> <div class="body" title="body"> <p><span class="named-content" title="named-content">To the Editor,</span></p> <div class="opening-section" title="opening-section"> <p>In a recent article by Lai et al,<a href="#jmv25777-bib-0001"><sup>1</sup></a> published on the <i>Journal of Medical Virology</i>, the authors attempted a mathematical reconstruction of the evolutionary dynamics of the new coronavirus severe acute respiratory syndrome coronavirus 2 (SARSâ€�CoV2) outbreak occurred in Wuhan (China), by analyzing 52 SARSâ€�CoV2 genomes provided at GISAID on 4 February 2020. This evaluation is fundamental to make authorities aware about spreading characteristics of SARSâ€�CoV2 in the Italian population and to earn insightful clues about the presumptive hypothesis that the current viral spreading in Italy surely comes from a Wuhanâ€�borne genotype and/or a Chinese outbreak. The same authors concluded that the SARSâ€�CoV2 in Italy might be present at least since September and October 2019, much before the claimed Wuhan outbreak.<a href="#jmv25777-bib-0001"><sup>1</sup></a> According to the World Health Organization evaluation, SARSâ€�CoV2 outbreaks in Europe occurred much before in Germany and France respect to Italy. Therefore, it might be presumed that a significant proportion of Italians were infected by SARSâ€�CoV2 in times greatly preceding the Government dispositions upon the cases enumeration. The daily differences between cases in the highest emergence period, that is, 1 to 9 March 2020, plotted a linear rather than an exponential trend. As <i>x</i> increases (1 day each), <i>y</i> values (cases number) increases by the same amount (1.2 or +20%) (Shapiroâ€�Wilk's exp test <i>P</i>â€‰=â€‰.442857). This possibly suggests that rhinopharyngeal swabs are catching homogeneous clusters of cases from dating back homogeneously, normally distributed preinfected population, depending also on an established maximal number of analyzed swabs for the day.</p> <p>Questions may be raised, therefore, if, taking into account the genomic distance (or similarity) and the RNAâ€�virus mutation rate, the â€œItalianâ€� SARSâ€�CoV2 might be the evolutionary balanced genotype (or strain) from a resident zoonotic spillover.</p> <p>The authors reported their conclusions by taking into account coalescent analysis and a birthâ€�death method to estimate the SARSâ€�CoV2 ancestor and establish unequivocally the origin of pandemics.<a href="#jmv25777-bib-0001"><sup>1</sup></a> Despite the trivial observation that a common reader of these issues may raise that is, that genetic divergence cannot be confused with a phylogenetic difference, RNA viruses mutation rate and their maintenance in the population structure or the balancing selection must be debated with great caution.<a href="#jmv25777-bib-0002"><sup>2</sup></a> It is well known that, particularly for viral pathogens, their ecological (genetic) and evolutionary dynamics may occur in the same timescale where potential cross interactions represent a fundamental hallmark. As regards RNA viruses, such as coronaviruses, the nucleotide mutation rate is at least a million times higher respect to a vertebrate, frequently recombine their genomes, between animals and humans, and are therefore highly pushed by evolutionary selection to adapt and survive in the animalâ€�human coexisting population.<a href="#jmv25777-bib-0003"><sup>3</sup></a> This should suggest that the crossâ€�talk between evolution and epidemiology is closely intertwined, causing that the maintenance of an onward transmission might be crucially associated with a continuous viral adaptation.</p> <p>Interestingly, the possible comparison of different isolates with the Italian strain (Gene Bank MT008023.1) of only 322 base pair coding the highly conserved M (E1) protein, allows retrieving very high similarity in the early timescale, due to the highly conserved sequence.</p> <p>Probably, the most stringent topic about SARSâ€�CoV2 should regard virulence, an issue that is strictly dependent on the population health condition and urbanization, more than on the viral mutation speed. Furthermore, the complete sequence diversity of SARS and the Middle East respiratory syndrome (MERS) viruses with further human Î²â€�coronaviruses at the S protein, is raising people the anguished question of why the current pandemic alarmed warning did not occur also for SARS, as a completely new emerging virus. One possible answer was forwarded taking into account that the SARSâ€�CoV virus in 2003 outbreak came simply from domestic animals.</p> <p>The issue of human coronaviruses virulence was recently addressed for SARSâ€�CoV and MERSâ€�CoV and some reports have outlined the role of coronavirus E protein in triggering an inflammatory response, cytokine storm, and/or inhibition of the innate immunity with dampening Th1 interferonâ€�Î³ signaling.<a href="#jmv25777-bib-0004"><sup>4</sup></a> In this context, alarming claims may appear completely justified, despite the fact that for SARS pandemics was not announced an alarming concern as doing currently. In this respect, authorities should inform the social collectivity about the correct patientsâ€™ stratification, how many hospitalized because of COVID19 and how many deaths because of COVID19.</p> <p>SARSâ€�CoV2 genome shares about 70% similarity with SARS coronavirus<a href="#jmv25777-bib-0005"><sup>5</sup></a> but probably, its completely new emergence via the mechanism of jumping species, is not a novelty in the biology of human <i>Betaâ€�coronaviridae</i>. TableÂ <a href="#jmv25777-tbl-0001">1</a> shows the Watermanâ€�Eggert score and similarity of main zoonotic coronavirus genomes and the SARSâ€�Cov2 Italian genomic sequence (Gene Bank MT_008022.1) of about 322 nucleotides, from a BLAST evaluation on 60 different genome sequences. The similarity is higher than 85%, a similarity quite similar for previous human SARS outbreaks, for example, the 2004â€�SARS epidemic virus (Gene Bank NC_004718.3), with a similarity of 84% but not with MERS coronavirus (Gene Bank NC_019843.3), with 56.9% of similarity.</p> <div class="table-wrap_UNKNOWN" id="jmv25777-tbl-0001" content-type="Table" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>Comparison between the Italian SARSâ€�CoV2 sequence MT008022.1 with zoonotic spillover SARS CoVs genomics</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <thead valign="bottom"> <tr style="border-bottom:solid 1px #000000" valign="bottom"> <th valign="bottom" rowspan="1" colspan="1"> <div class="sc_UNKNOWN">
g</div>ene bank <div class="sc_UNKNOWN">
id</div> </th> <th valign="bottom" rowspan="1" colspan="1"> <div class="sc_UNKNOWN">
n</div>ame</th> <th valign="bottom" rowspan="1" colspan="1"> <div class="sc_UNKNOWN">
w</div>aterman <div class="sc_UNKNOWN">
â€�e</div>ggert score</th> <th valign="bottom" rowspan="1" colspan="1"> <div class="sc_UNKNOWN">
s</div>imilarity</th> <th valign="bottom" rowspan="1" colspan="1"> <div class="sc_UNKNOWN">
n</div>otes</th> </tr> </thead> <tbody valign="top"> <tr> <td valign="top" rowspan="1" colspan="1">DQ071615.1</td> <td valign="top" rowspan="1" colspan="1">Bat Rp3 SARS coronavirus</td> <td valign="top" rowspan="1" colspan="1">1177; 176.2 bits; E(1) &amp;lt;8.9â€‰Ã—â€‰10<sup>âˆ’47</sup></td> <td valign="top" rowspan="1" colspan="1">85.3%</td> <td valign="top" rowspan="1" colspan="1">85.3% Identity (85.3% similar) in 320â€‰nt overlap (26â€‰532â€�26â€‰851:3â€�322)</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">JX993988.1</td> <td valign="top" rowspan="1" colspan="1">Bat coronavirus Cp/Yunnan2011</td> <td valign="top" rowspan="1" colspan="1">1199; 179.1 bits; E(1) &amp;lt;1.2â€‰Ã—â€‰10<sup>âˆ’47</sup></td> <td valign="top" rowspan="1" colspan="1">86.2%</td> <td valign="top" rowspan="1" colspan="1">86.2% Identity (86.2% similar) in 319â€‰nt overlap (26â€‰437â€�26â€‰755:3â€�321)</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MG772933.1</td> <td valign="top" rowspan="1" colspan="1">Bat SARSâ€�like coronavirus isolate batâ€�SLâ€�CoVZC45</td> <td valign="top" rowspan="1" colspan="1">1412; 221.9 bits; E(1) &amp;lt;1.5â€‰Ã—â€‰10<sup>âˆ’60</sup></td> <td valign="top" rowspan="1" colspan="1">93.2%</td> <td valign="top" rowspan="1" colspan="1">93.2% Identity (93.2% similar) in 322â€‰nt overlap (26â€‰610â€�26â€‰931:1â€�322)</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MG772934.1</td> <td valign="top" rowspan="1" colspan="1">Bat SARSâ€�like coronavirus isolate batâ€�SLâ€�CoVZXC21</td> <td valign="top" rowspan="1" colspan="1">1412; 222.4 bits; E(1) &amp;lt;1.1â€‰Ã—â€‰10<sup>âˆ’60</sup></td> <td valign="top" rowspan="1" colspan="1">93.2%</td> <td valign="top" rowspan="1" colspan="1">93.2% Identity (93.2% similar) in 322â€‰nt overlap (26â€‰541â€�26â€‰862:1â€�322)</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MN996532.1</td> <td valign="top" rowspan="1" colspan="1">Bat coronavirus RaTG13</td> <td valign="top" rowspan="1" colspan="1">1466; 214.0 bits; E(1) &amp;lt;3.7â€‰Ã—â€‰10<sup>âˆ’58</sup></td> <td valign="top" rowspan="1" colspan="1">95%</td> <td valign="top" rowspan="1" colspan="1">95.0% Identity (95.0% similar) in 322â€‰nt overlap (26â€‰672â€�26â€‰993:1â€�322)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><i>Note</i>: Italy SARSâ€�CoV2 isolateâ€‰=â€‰322â€‰nt. Similarity with Wuhan seafood market pneumonia virus isolates Wuhan HUâ€�1â€‰=â€‰100% (Lalign Software version 2.1.30 (12 May 2010) Readseq version 2.1.30 (12 May 2010).</p> </div> <div class="fn-type-" title=""> <p>Abbreviations: nt, nucleotide; SARSâ€�CoV2, severe acute respiratory syndrome coronavirus 2.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd.</span>, </span><span class="license" title="license"><span class="license-p" title="license-p">This article is being made freely available through PubMed Central as part of the COVID-19 public health emergency response. It can be used for unrestricted research re-use and analysis in any form or by any means with acknowledgement of the original source, for the duration of the public health emergency.</span></span></div> </div> <p>Previous reports have outlined that the genetic difference in the Spike protein sequence, between the most frequent human coronavirus genotypes such as HCoVâ€�NL63, HCoVâ€�229E and HCoVâ€�OC43, and SARS or MERS coronaviruses, is practically absolute (100%), so describing new emerging species in the human coronavirus outbreaks.<a href="#jmv25777-bib-0006"><sup>6</sup></a> This suggests that SARS had the same genetic novelty without immune protection in the population as COVID19 to date, though with a much lesser extent in the emergence claim.</p> <p>The low kinship with other human coronavirus outbreaks, such as MERS, suggests that human coronaviruses rapidly evolve in the local population and rapidly disappear, probably via evolutionary adaptation, via an immune pressure. Briefly speaking and despite the fact that SARS HCoV was a newly emerging virus likewise SARSâ€�CoV2, the COVID19 causing agent, when the outbreak occurred in 2002 to 2003, worldwide alarming and warnings widespread in lesser extent respect to the current SARSâ€�CoV2 claim.<a href="#jmv25777-bib-0007"><sup>7</sup></a> Actually, the genomic evolution of the Î²â€�coronavirus family in Italy may come from zoonotic spillover and a wide interplay vectors/human hosts.<a href="#jmv25777-bib-0008"><sup>8</sup></a> While Liu et al<a href="#jmv25777-bib-0008"><sup>8</sup></a> suggested turtles as intermediate vectors of the SARSâ€�CoV2, Li et al,<a href="#jmv25777-bib-0009"><sup>9</sup></a> dismiss the hypothesis of pangolin as a possible intermediate, mainly focusing on bats. We do not know if in Italy a wide genetic subfamily of Î²â€�coronaviruses coexisted in the country due to zoonotic spillover and population spreading caused by the very frequent human exchanges with also Chinese people from endemic areas. This issue has not been thoroughly investigated so far, because of the urgent need to diagnose infected individuals just following the Wuhan outbreak, but it might have an important consequence also in political decisions.</p> <p>The crowded debate burst on the epidemiological causes of the SARSâ€�CoV2 outbreak in Italy never addressed the possibility that, also in a European country, the animalâ€�mediated intermediate transmission may have a role in the positive SARSâ€�CoV2 diagnosed individuals so far. Interestingly, the paper by Lai et al,<a href="#jmv25777-bib-0001"><sup>1</sup></a> discussed the hypothesis that a COVID19â€�like coronavirus causative agent, might have existed in Italy quite earlier than the Wuhan outbreak, probably since September or October 2019. The question if we are currently â€œmappingâ€� a family of SARSâ€�CoV2â€�corelated genotypes derived from preâ€�existing human autochthonous Î²â€�coronavirus, is particularly concerning. In Italy, the Lombardia region, which accounts for the highest number of SARSâ€�CoV2 positive subjects, has also the highest percentage of Chinese resident people (&amp;gt;23%), according to recent ISTAT data in 2019. Interestingly, Tuscany (Toscana) region, with Prato, accounting on about 18% Chinese individuals, appeared to have a very low number of people purported to have contracted the SARSâ€�CoV2.<a href="#jmv25777-bib-0010"><sup>10</sup></a></p> <p>Besides China, a spillover of zoonotic pathogens from wildlife was reported also in Italy from bats.<a href="#jmv25777-bib-0011"><sup>11</sup></a> In Italy, the association between Î²â€�coronavirus species, closely akin to SARSâ€�CoV, and bats (<i>Rhinolophus ferrumequinum</i> L), has been recently reported.<a href="#jmv25777-bib-0009"><sup>9</sup></a>, <a href="#jmv25777-bib-0012"><sup>12</sup></a> Lelli et al<a href="#jmv25777-bib-0013"><sup>13</sup></a> reported a phylogenetic investigation about the origin of the many different coronavirus genotypes in Italy from wild bats and identified two alphaâ€�coronaviruses from Kuhl's pipistrelle (<i>Pipistrellus kuhlii</i>), three clade 2b Î²â€�coronaviruses from lesser horseshoe bats (<i>Rhinolophus hipposideros</i>), and 10 clade 2c Î²â€�coronaviruses from Kuhl's pipistrelle, common noctule (<i>Nyctalus noctula</i>), and Savi's pipistrelle (<i>Hypsugo savii</i>), mainly in the Italian Regions Lombardia and Emiliaâ€�Romagna, where major SARSâ€�CoV2 focuses are present.<a href="#jmv25777-bib-0013"><sup>13</sup></a> The group of Moreno et al,<a href="#jmv25777-bib-0014"><sup>14</sup></a> described two Î²â€�coronaviruses closely associated with MERSâ€�CoV and present in very common bat species, that is, <i>P. kuhlii</i> and <i>H. savii</i>), widespread in the Northern regions of Italy. This study fills a substantive gap in the knowledge of batâ€�CoV ecology in Italy and extends the current knowledge of clade 2c Î²â€�coronaviruses with new sequences obtained from bats that have not been previously described as hosts of these viruses. According to the authors, the mean evolutionary rate was estimated as 2.15â€‰Ã—â€‰10<sup>âˆ’6</sup> substitutions/site, corresponding to a value of 7.8â€‰Ã—â€‰10<sup>âˆ’4</sup> substitutions/year, so retrieving an estimated tMRCA dating back to 10 September.<a href="#jmv25777-bib-0001"><sup>1</sup></a> The conclusions reported by the authors showed that the evolutionary rate of the novel SARSâ€�CoV2 is online with the SARS and MERS ones.<a href="#jmv25777-bib-0001"><sup>1</sup></a></p> <p>Zhang and colleagues<a href="#jmv25777-bib-0015"><sup>15</sup></a> have investigated the adaptive evolution of the S (Spike) protein in SARSâ€�CoV and SARSâ€�CoVâ€�like viruses in animals, reaching the conclusion that both the positive selected pressure and the positively selected target site contributes in the adaptive evolution of the S protein from animals to humans, evidence that supports the suggestion to analyze this issue to comprehend the coronavirus spillover in Northern Italy during the current SARSâ€�CoV2 outbreak.<a href="#jmv25777-bib-0015"><sup>15</sup></a></p> <p>Bats are wellâ€�known reservoirs of mammals coronaviruses, including humans. Therefore, a keen survey about the genetic variants and phylogeny of SARSâ€�related coronaviruses in Italy is mandatory to shed light on the very recent SARSâ€�CoV2 in Italy from spillover zoonosis. This should enlight many still debated concerns.</p> <p>Moreover, the current literature on the field is still scanty about the clinical aspects of COVID19 in Italian patients. So far, evidence was reported only for individuals having held direct contact with Wuhan.<a href="#jmv25777-bib-0016"><sup>16</sup></a> Death rates from pulmonary infectious agents account for about 11% to 14%, usually involving people aged 75 to 90 years, therefore the current estimation of about 2.83% to 3.6% of deaths caused by COVID19, may appear limited respect to the big claim burst on the SARSâ€�CoV2 infection. Furthermore, the biggest concern currently forwarded by media regards the SARSâ€�CoV2 impact on the resuscitation units of the many health care structures in Italy. This may appear controversial with the number of patients with confirmed pulmonary pathology undergoing hospitalization. During the 2002â€�SARS epidemics evidence that transmission may occur during cardiopulmonary resuscitation was reported but did not raise particular concern.<a href="#jmv25777-bib-0017"><sup>17</sup></a> Therefore, the Italian authorities engaged to face at the SARSâ€�CoV2 outbreak should inform citizens also about the stratified number of hospitalized people in the resuscitation units, to have a cleared overview of the SARSâ€�CoV2 epidemics in our country.</p> <p>In 2018, the total number of resuscitation units (intensive care and cardiovascular arterial units) were estimated to be 69â€‰921â€‰+â€‰148â€‰124 (a total of 218â€‰045 hospitalized cases, ie, 597.38 cases per day), which means about 30 cases/day/region and 600 subjects in those units in a time range of about 20 days. Data from the Health Ministry were sensitively lower (351) on 5 March 2020 and showed a very modest upâ€�growth only very recently (877 on 10 March 2020), which might explain the big concern about health care services.</p> <p>Much better communication and the elucidation about the actual origin of the SARSâ€�CoV2 outbreak in Italy, by deepening the evolutionary phylogeny of the virus, may give fundamental insights also about the security and safety dispositions held by the institutional authorities and politics.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="jmv25777-bib-0001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bergna</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Acciarri</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Galli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zehender</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="mixed-article-title" title="mixed-article-title">Early phylogenetic estimate of the effective reproduction number of SARSâ€�CoVâ€�2</span> [published online ahead of print February 25, 2020]. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25723">10.1002/jmv.25723</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kozak</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bocianowski</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liersch</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Genetic divergence is not the same as phenotypic divergence</span>. <span class="source" tagx="source" title="source">Mol Breed</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">277</span>â€�<span class="lpage" tagx="lpage" title="lpage">280</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21841910">21841910</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary analysis of the dynamics of viral infectious disease</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">540</span>â€�<span class="lpage" tagx="lpage" title="lpage">550</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19564871">19564871</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">De Diego</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nietoâ€�Torres</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jimenezâ€�GuardeÃ±o</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Coronavirus virulence genes with main focus on SARSâ€�CoV envelope gene</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">194</span>:<span class="fpage" tagx="fpage" title="fpage">124</span>â€�<span class="lpage" tagx="lpage" title="lpage">137</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25093995">25093995</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Genomic characterization and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">395</span>(<span class="issue" tagx="issue" title="issue">10224</span>):<span class="fpage" tagx="fpage" title="fpage">565</span>â€�<span class="lpage" tagx="lpage" title="lpage">574</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32007145">32007145</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Menachery</span><span class="given-names" tagx="given-names" title="given-names">VD</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Graham</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Jumping speciesâ€�a mechanism for coronavirus persistence and survival</span>. <span class="source" tagx="source" title="source">Curr Opin Virol</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">23</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>â€�<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28214731">28214731</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">The SARS epidemics and its aftermath in China: a political perspective. Learning from SARS: Preparing for the next disease outbreak, workshop summary</span> In: <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knobler</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span>, <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahmoud</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span>, <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span>, eds. <span class="source" tagx="source" title="source">Institute of Medicine (US) Forum on Microbial Threats</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington (DC)</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">National Academies Press (US)</span>; <span class="year" tagx="year" title="year">2004</span>.</span></li> <li tag="ref"><a name="jmv25777-bib-0008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Composition and divergence of coronavirus spike proteins and host ACE2 receptors predict potential intermediate hosts of SARSâ€�CoVâ€�2</span> [published online ahead of print February 6, 2020]. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25726">10.1002/jmv.25726</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zai</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Evolutionary history, potential intermediate animal host, and crossâ€�species analyses of SARSâ€�CoVâ€�2</span> [published online ahead of print February 27, 2020]. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25731">10.1002/jmv.25731</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Baldassar</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Johanson</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McAuliffe</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bressan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="source" tagx="source" title="source">Chinese Migration to Europe: Prato, Italy and Beyond</span>. <span class="volume" tagx="volume" title="volume">2015</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Palgrave McMillan</span>.</span></li> <li tag="ref"><a name="jmv25777-bib-0011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rizzo</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Edenborough</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Toffoli</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Coronavirus and paramyxovirus in bats from Northwest Italy</span>. <span class="source" tagx="source" title="source">BMC Vet Res</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">396</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29273042">29273042</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Balboni</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Palladini</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bogliani</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Battilani</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Detection of a virus related to betacoronaviruses in Italian greater horseshoe bats</span>. <span class="source" tagx="source" title="source">Epidemiol Infect</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">139</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">216</span>â€�<span class="lpage" tagx="lpage" title="lpage">219</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20478089">20478089</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lelli</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Papetti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sabelli</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rosti</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moreno</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Boniotti</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span>. <span class="mixed-article-title" title="mixed-article-title">Detection of coronaviruses in bats of various species in Italy</span>. <span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">2679</span>â€�<span class="lpage" tagx="lpage" title="lpage">2689</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24184965">24184965</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moreno</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lelli</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">de Sabato</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Detection and full genome characterization of two beta CoV viruses related to Middle East respiratory syndrome from bats in Italy</span>. <span class="source" tagx="source" title="source">Virol J</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">239</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12985-017-0907-1">10.1186/s12985-017-0907-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29258555">29258555</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Adaptive evolution of the spike gene of SARS coronavirus: changes in positively selected sites in different epidemic groups</span>. <span class="source" tagx="source" title="source">BMC Microbiol</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">4</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">88</span>.</span></li> <li tag="ref"><a name="jmv25777-bib-0016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Albarello</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Pianura</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Di Stefano</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">2019â€�novel Coronavirus severe adult respiratory distress syndrome in two cases in Italy: an uncommon radiological presentation</span>. <span class="source" tagx="source" title="source">Int J Infect Dis</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">93</span>:<span class="fpage" tagx="fpage" title="fpage">192</span>â€�<span class="lpage" tagx="lpage" title="lpage">197</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32112966">32112966</a></span></span></li> <li tag="ref"><a name="jmv25777-bib-0017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Christian</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Loutfy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McDonald</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Possible SARS coronavirus transmission during cardiopulmonary resuscitation</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">287</span>â€�<span class="lpage" tagx="lpage" title="lpage">293</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15030699">15030699</a></span></span></li> </ul> </div> </div>  </body></html>