<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/17/2019.12.16.875872.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.16.875872v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses" />
<meta name="DC.Identifier" content="10.1101/2019.12.16.875872" />
<meta name="DC.Date" content="2019-12-17" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="RNA viruses rapidly mutate, which can result in increased virulence, increased escape from vaccine protection, and false negative detection results. Targeted detection methods have a limited ability to detect unknown viruses and often provide insufficient data to detect coinfections or identify antigenic variants. Random, deep sequencing is a method that can more fully detect and characterize RNA viruses and is often coupled with molecular techniques or culture methods for viral enrichment. Viral culture coupled with third-generation sequencing were tested for the ability to detect and characterize RNA viruses. Cultures of bovine viral diarrhea virus, canine distemper virus, epizootic hemorrhagic disease virus, infectious bronchitis virus, two influenza A viruses, and porcine respiratory and reproductive syndrome virus were sequenced on the MinION platform using a random, reverse primer in a strand-switching reaction, coupled with PCR-based barcoding. Reads were taxonomically classified and used for reference-based sequence building using a stock personal computer. This method accurately detected and identified complete coding sequence genomes with a minimum of 20× coverage depth for all seven viruses, including a sample containing two viruses. Each lineage-typing region had at least 26× coverage depth for all viruses. Furthermore, analyzing the canine distemper virus sample through a pipeline devoid of canine distemper virus reference sequences modeled the ability of this protocol to detect unknown viruses. These results show the ability of this technique to detect and characterize dsRNA, negative- and positive-sense ssRNA, nonsegmented, and segmented RNA viruses." />
<meta name="DC.Contributor" content="Kelsey T. Young" />
<meta name="DC.Contributor" content="Kevin K. Lahmers" />
<meta name="DC.Contributor" content="Holly S. Sellers" />
<meta name="DC.Contributor" content="David E. Stallknecht" />
<meta name="DC.Contributor" content="Rebecca L. Poulson" />
<meta name="DC.Contributor" content="Jerry T. Saliki" />
<meta name="DC.Contributor" content="S. Mark Tompkins" />
<meta name="DC.Contributor" content="Ian Padykula" />
<meta name="DC.Contributor" content="Chris Siepker" />
<meta name="DC.Contributor" content="Elizabeth W. Howerth" />
<meta name="DC.Contributor" content="Michelle Todd" />
<meta name="DC.Contributor" content="James B. Stanton" />
<meta name="article:published_time" content="2019-12-17" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;RNA viruses rapidly mutate, which can result in increased virulence, increased escape from vaccine protection, and false negative detection results. Targeted detection methods have a limited ability to detect unknown viruses and often provide insufficient data to detect coinfections or identify antigenic variants. Random, deep sequencing is a method that can more fully detect and characterize RNA viruses and is often coupled with molecular techniques or culture methods for viral enrichment. Viral culture coupled with third-generation sequencing were tested for the ability to detect and characterize RNA viruses. Cultures of bovine viral diarrhea virus, canine distemper virus, epizootic hemorrhagic disease virus, infectious bronchitis virus, two influenza A viruses, and porcine respiratory and reproductive syndrome virus were sequenced on the MinION platform using a random, reverse primer in a strand-switching reaction, coupled with PCR-based barcoding. Reads were taxonomically classified and used for reference-based sequence building using a stock personal computer. This method accurately detected and identified complete coding sequence genomes with a minimum of 20× coverage depth for all seven viruses, including a sample containing two viruses. Each lineage-typing region had at least 26× coverage depth for all viruses. Furthermore, analyzing the canine distemper virus sample through a pipeline devoid of canine distemper virus reference sequences modeled the ability of this protocol to detect unknown viruses. These results show the ability of this technique to detect and characterize dsRNA, negative- and positive-sense ssRNA, nonsegmented, and segmented RNA viruses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.16.875872v2" />
<meta name="citation_id" content="2019.12.16.875872v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.16.875872v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.16.875872v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.16.875872v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.16.875872" />
<meta name="citation_num_pages" content="37" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.16.875872" />
<meta name="citation_author" content="Kelsey T. Young" />
<meta name="citation_author_institution" content="Department of Pathology, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1163-567X" />
<meta name="citation_author" content="Kevin K. Lahmers" />
<meta name="citation_author_institution" content="Department of Biomedical Sciences &amp; Pathobiology, VA-MD College of Veterinary Medicine, Virginia Tech University" />
<meta name="citation_author" content="Holly S. Sellers" />
<meta name="citation_author_institution" content="Poultry Diagnostic and Research Center, Department of Population Health, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author" content="David E. Stallknecht" />
<meta name="citation_author_institution" content="Southeastern Cooperative Wildlife Disease Study Department of Population Health, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author" content="Rebecca L. Poulson" />
<meta name="citation_author_institution" content="Southeastern Cooperative Wildlife Disease Study Department of Population Health, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author" content="Jerry T. Saliki" />
<meta name="citation_author_institution" content="Athens Veterinary Diagnostic Laboratory, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author" content="S. Mark Tompkins" />
<meta name="citation_author_institution" content="Center for Vaccines and Immunology, Department of Infectious Diseases, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author" content="Ian Padykula" />
<meta name="citation_author_institution" content="Center for Vaccines and Immunology, Department of Infectious Diseases, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author" content="Chris Siepker" />
<meta name="citation_author_institution" content="Department of Pathology, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author" content="Elizabeth W. Howerth" />
<meta name="citation_author_institution" content="Department of Pathology, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author" content="Michelle Todd" />
<meta name="citation_author_institution" content="Department of Biomedical Sciences &amp; Pathobiology, VA-MD College of Veterinary Medicine, Virginia Tech University" />
<meta name="citation_author" content="James B. Stanton" />
<meta name="citation_author_institution" content="Department of Pathology, College of Veterinary Medicine, University of Georgia" />
<meta name="citation_author_email" content="jbs@uga.edu" />
<meta name="citation_reference" content="Sartore S, et al. The effects of control measures on the economic burden associated with epidemics of avian influenza in Italy. Poult Sci 2010;89:1115–1121." />
<meta name="citation_reference" content="Butt SL, et al. Rapid virulence prediction and identification of Newcastle disease virus genotypes using third-generation sequencing. Virol J 2018;15:179." />
<meta name="citation_reference" content="Clavijo A, et al. An improved Real-Time Polymerase Chain Reaction for the Simultaneous Detection of All Serotypes of Epizootic Hemorrhagic Disease Virus. J Vet Diagn Invest 2010;22:588–593." />
<meta name="citation_reference" content="Cavanagh, D and Naqi SA. Infectious bronchitis. In: Diseases of Poultry. 11th ed. Ames, Iowa: Iowa State Univer Pr, 2003:101–119." />
<meta name="citation_reference" content="Zhang J, et al. High-throughput whole genome sequencing of Porcine reproductive and respiratory syndrome virus from cell culture materials and clinical specimens using next-generation sequencing technology. J Vet Diagn Invest 2016;29:41–50." />
<meta name="citation_reference" content="Woolhouse MEJ, et al. Assessing the Epidemic Potential of RNA and DNA Viruses. Emerg Inf Dis 2016;22:2037–2044." />
<meta name="citation_reference" content="Rubio L, et al. Genetic variability and evolutionary dynamics of viruses of the family Closteroviridae. Front. Microbiol. 2013;4:151–151." />
<meta name="citation_reference" content="Read SJ, et al. Molecular techniques for clinical diagnostic virology. J Clin Pathol 2000;53:502–506." />
<meta name="citation_reference" content="Chan RWY, et al. Tissue tropism of swine influenza viruses and reassortants in ex vivo cultures of the human respiratory tract and conjunctiva. J Virol 2011;85:11581–11587." />
<meta name="citation_reference" content="Flint, J. Synthesis of RNA from RNA templates. In: Principles of Virology. 4th ed. Washington, DC: ASM Press, 2015: 157–184." />
<meta name="citation_reference" content="Gilchrist CA, et al. Whole-genome sequencing in outbreak analysis. Clin Microbiol Rev 2015;28:541–563." />
<meta name="citation_reference" content="Cobo F. Application of molecular diagnostic techniques for viral testing. Open Virol J 2012;6:104–114." />
<meta name="citation_reference" content="Gullett JC, et al. Quantitative Nucleic Acid Amplification Methods for Viral Infections. Clin Chem 2015;61:72–78." />
<meta name="citation_reference" content="Ranjan R, et al. Analysis of the microbiome: Advantages of whole genome shotgun versus 16S amplicon sequencing. Biochem Biophys Res Commun 2016;469:967–977." />
<meta name="citation_reference" content="Kasibhatla, SM, et al. Analysis of next-generation sequencing data in virology-Opportunities and challenges. IntechOpen 2016; DOI: 10.5772/61610." />
<meta name="citation_reference" content="Head SR, et al. Library construction for next-generation sequencing: overviews and challenges. BioTechniques 2014;56:61–64" />
<meta name="citation_reference" content="Marston DA, et al. Next generation sequencing of viral RNA genomes. BMC Genomics 2013;14:444." />
<meta name="citation_reference" content="Quick J, et al. Real-time, portable genome sequencing for Ebola surveillance. Nature 2016;530:228–232." />
<meta name="citation_reference" content="Greninger AL, et al. Rapid metagenomic identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis. Genome Med 2015;7:99." />
<meta name="citation_reference" content="Kilianski A, et al. Use of Unamplified RNA/cDNA-Hybrid Nanopore Sequencing for Rapid Detection and Characterization of RNA Viruses. Emerg Infect Dis 2016;22:1448–1451." />
<meta name="citation_reference" content="Beato MS, et al. Identification and genetic characterization of bovine enterovirus by combination of two next generation sequencing platforms. J Virol Methods 2018;260:21–25." />
<meta name="citation_reference" content="Peserico A, et al. Diagnosis and characterization of canine distemper virus through sequencing by MinION nanopore technology. Sci Rep 2019;9:1714–1714." />
<meta name="citation_reference" content="Mahlum CE, et al. Detection of bovine viral diarrhea virus by TaqMan® reverse transcription polymerase chain reaction. J Vet Diagn Invest 2002;14:120–125." />
<meta name="citation_reference" content="Kim D, et al. Centrifuge: rapid and sensitive classification of metagenomic sequences. Genome Res 2016;26:1721–1729." />
<meta name="citation_reference" content="Butt SL, et al. Real-time, MinION-based, amplicon sequencing for lineage typing of infectious bronchitis virus from upper respiratory samples. J Vet Diagn Invest 2019. Accepted." />
<meta name="citation_reference" content="Yeşilbağ K, et al. Variability and Global Distribution of Subgenotypes of Bovine Viral Diarrhea Virus. Viruses 2017;9:128." />
<meta name="citation_reference" content="Piewbang C, et al. Genetic and evolutionary analysis of a new Asia-4 lineage and naturally recombinant canine distemper virus strains from Thailand. Sci Rep 2019;9:3198." />
<meta name="citation_reference" content="Brown-Joseph T, et al. Identification and characterization of epizootic hemorrhagic disease virus serotype 6 in cattle co-infected with bluetongue virus in Trinidad, West Indies. Vet Microbiol 2019;229:1–6." />
<meta name="citation_reference" content="Valastro V, et al. S1 gene-based phylogeny of infectious bronchitis virus: An attempt to harmonize virus classification. Infect Genet and Evol 2016;39:349–364." />
<meta name="citation_reference" content="Shi W, et al. A complete analysis of HA and NA genes of influenza A viruses. PloS One 2010;5:e14454–e14454." />
<meta name="citation_reference" content="Kang H, et al. Geographic distribution and molecular analysis of porcine reproductive and respiratory syndrome viruses circulating in swine farms in the Republic of Korea between 2013 and 2016. BMC Vet Res 2018;14:160." />
<meta name="citation_reference" content="O’Donnell CR, et al. Error analysis of idealized nanopore sequencing. Electrophoresis 2013;34:2137–2144." />
<meta name="citation_reference" content="Breitwieser, FP, et al. Pavian: interactive analysis of metagenomics data or microbiome studies and pathogen identification. Bioinformatics 2019." />
<meta name="citation_reference" content="Thompson JD, et al. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res 1994;22:4673–4680." />
<meta name="citation_reference" content="Tamura K. Estimation of the number of nucleotide substitutions when there are strong transition-transversion and G+C-content biases. Mol Biol Evol 1992;9:678–687." />
<meta name="citation_reference" content="Kumar S, et al. MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms. Mol Biol Evol 2018;35:1547–1549." />
<meta name="citation_reference" content="Vilček Š, et al. Pestiviruses isolated from pigs, cattle and sheep can be allocated into at least three genogroups using polymerase chain reaction and restriction endonuclease analysis. Arch of Virol 1994;136:309–323." />
<meta name="citation_reference" content="Martella V, et al. Genotyping canine distemper virus (CDV) by a hemi-nested multiplex PCR provides a rapid approach for investigation of CDV outbreaks. Vet Microbiol 2007;122:32–42." />
<meta name="citation_reference" content="Sun F, et al. Molecular Typing of Epizootic Hemorrhagic Disease Virus Serotypes by One-Step Multiplex RT-PCR. J Wildl Dis 2014;50:639–644." />
<meta name="citation_reference" content="J Woolhouse ME, et al. RNA Viruses: A case study of the biology of emerging infectious diseases. Microbiol Spectr 2013;1:10.1128/microbiolspec.OH-0001-2012." />
<meta name="citation_reference" content="Ginocchio CC, et al. Current best practices for respiratory virus testing. J Clin Microbiol 2011;49:S44–S48." />
<meta name="citation_reference" content="Wostenberg DJ, et al. Evidence of two cocirculating canine distemper virus strains in mesocarnivores from northern Colorado, USA. J. of Wildlife Diseases 2018;54:534–543." />
<meta name="citation_reference" content="Ahasan MS, et al. Complete genome sequence of Mobuck Virus isolated from a Florida white-tailed deer (Odocoileus virginianus). Microbiol Resour Announc 2019;8:e01324–01318." />
<meta name="citation_reference" content="citation_title=In: Advances in Virus Research;citation_author=K. Pfeffermann;citation_pages=75-98;citation_volume=100;citation_year=2018" />
<meta name="citation_reference" content="Jain M, et al. Improved data analysis for the MinION nanopore sequencer. Nat Methods 2015;12:351–356." />
<meta name="citation_reference" content="McNaughton AL, et al. Illumina and Nanopore methods for whole genome sequencing of hepatitis B virus (HBV). Sci Rep 2019;9:7081." />
<meta name="citation_reference" content="Jain M, et al. MinION Analysis and Reference Consortium: Phase 2 data release and analysis of R9.0 chemistry. F1000Res 2017;6:760–760." />
<meta name="citation_reference" content="Magi A, et al. Characterization of MinION nanopore data for resequencing analyses. Brief Bioinform 2016;18:940–953." />
<meta name="citation_reference" content="Goldstein EJ, et al. Integrating patient and whole-genome sequencing data to provide insights into the epidemiology of seasonal influenza A (H3N2) viruses. Microb Genom 2018;4:e000137." />
<meta name="citation_reference" content="Pallen MJ, et al. High-throughput sequencing and clinical microbiology: progress, opportunities and challenges. Curr Opin in Microbiol 2010;13:625–631." />
<meta name="citation_reference" content="Runckel C, et al. Identification and manipulation of the molecular determinants influencing poliovirus recombination. PLOS Pathog 2013;9:e1003164." />
<meta name="citation_reference" content="Su S, et al. Epidemiology, genetic recombination, and pathogenesis of coronaviruses. Trends in Microbiol 2016;24:490–502." />
<meta name="citation_reference" content="Park E-S, et al. Identification of a natural recombination in the F and H genes of feline morbillivirus. Virology 2014;468–470:524–531." />
<meta name="twitter:title" content="Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="RNA viruses rapidly mutate, which can result in increased virulence, increased escape from vaccine protection, and false negative detection results. Targeted detection methods have a limited ability to detect unknown viruses and often provide insufficient data to detect coinfections or identify antigenic variants. Random, deep sequencing is a method that can more fully detect and characterize RNA viruses and is often coupled with molecular techniques or culture methods for viral enrichment. Viral culture coupled with third-generation sequencing were tested for the ability to detect and characterize RNA viruses. Cultures of bovine viral diarrhea virus, canine distemper virus, epizootic hemorrhagic disease virus, infectious bronchitis virus, two influenza A viruses, and porcine respiratory and reproductive syndrome virus were sequenced on the MinION platform using a random, reverse primer in a strand-switching reaction, coupled with PCR-based barcoding. Reads were taxonomically classified and used for reference-based sequence building using a stock personal computer. This method accurately detected and identified complete coding sequence genomes with a minimum of 20× coverage depth for all seven viruses, including a sample containing two viruses. Each lineage-typing region had at least 26× coverage depth for all viruses. Furthermore, analyzing the canine distemper virus sample through a pipeline devoid of canine distemper virus reference sequences modeled the ability of this protocol to detect unknown viruses. These results show the ability of this technique to detect and characterize dsRNA, negative- and positive-sense ssRNA, nonsegmented, and segmented RNA viruses." />
<meta name="og-title" property="og:title" content="Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.16.875872v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="RNA viruses rapidly mutate, which can result in increased virulence, increased escape from vaccine protection, and false negative detection results. Targeted detection methods have a limited ability to detect unknown viruses and often provide insufficient data to detect coinfections or identify antigenic variants. Random, deep sequencing is a method that can more fully detect and characterize RNA viruses and is often coupled with molecular techniques or culture methods for viral enrichment. Viral culture coupled with third-generation sequencing were tested for the ability to detect and characterize RNA viruses. Cultures of bovine viral diarrhea virus, canine distemper virus, epizootic hemorrhagic disease virus, infectious bronchitis virus, two influenza A viruses, and porcine respiratory and reproductive syndrome virus were sequenced on the MinION platform using a random, reverse primer in a strand-switching reaction, coupled with PCR-based barcoding. Reads were taxonomically classified and used for reference-based sequence building using a stock personal computer. This method accurately detected and identified complete coding sequence genomes with a minimum of 20× coverage depth for all seven viruses, including a sample containing two viruses. Each lineage-typing region had at least 26× coverage depth for all viruses. Furthermore, analyzing the canine distemper virus sample through a pipeline devoid of canine distemper virus reference sequences modeled the ability of this protocol to detect unknown viruses. These results show the ability of this technique to detect and characterize dsRNA, negative- and positive-sense ssRNA, nonsegmented, and segmented RNA viruses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-17" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.16.875872v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.16.875872v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.16.875872v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.16.875872v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1042888" />
    <title>Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Iv2IXNEtP89etJP7txrtEbFK7eZ7qX81-ylgxmph0ho","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2019.12.16.875872v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2019.12.16.875872v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.16.875872v2"}],"nid":"1042888","apath":"\/biorxiv\/early\/2019\/12\/17\/2019.12.16.875872.atom","pisa":"biorxiv;2019.12.16.875872v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.16.875872v2","title":"Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses","identifier":"node\/1042888"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IzJtBxNoVc72sFY1j5mUXC8TuBJacdjtSIkrkTT\\\\\\\/QL3tiMcZFaIHRytM8UAIUyNR7XsHdDi2KencMtLJLGWoNcn8YM+4rqSUUZJBOeAxcS7m0Ik37geSvfGZyCsl0vLmzwSItBLf+sg5Okuqk8KdXek\\\\\\\/cVTOGD1NVBagg1A7CbrTvFI95IPHZV69ieWGUiyEDJcaQLAJ8awK6D4uakmK2drT\\\\\\\/644688y5KT1jjRMVmrkOyBnyrUhySINWus8I6wtYxPgbSwXS02RO5b087HDVH0uRV6kAoxgBriCENfAyeWhiffxdc0sIZmayoIRaawKsNrmQ5CGq3nkXfWYqLsfGUA9PC8kH0msKesCD9nLA6IWrMmM23DR9YLHCsSrSLHMXxFlSjnn3MTjiZHOjuV+OYUX3TilG6rlyhZd9VoBs3q8roWqBDuQkjfRhuyTZG2sx5OrFJmytw2SzgrkR\\\\\\\/mOJ8yv2eU8grITYkQCUDSBomBdAXGKs99tyOmS2Z9NO1llGH3x\\\\\\\/PgI\\\\\\\/j6OBp8yNv4edEBwt8uj\\\\\\\/4rSpgtquOtBWwEXHSQCTu0YSyCdzyS2YoW4XlNEjV+RUFHKZxHRQ37gt7BA\\\\\\\/ujoGhQOTz8zasc1zfylIt4L6as+\\\\\\\/A\\\\\\\/ozsKlcpFvfXPSA5Pk2JADarpIrnl84s+ZUde\\\\\\\/K7ewaxysFWbFStiRjtbQRNpzKRcRpeoaDFGqSGoaeI+fvStcur9Ip\\\\\\\/AEeHZSwZpBniI44EWA3DFew6VtF3P8V0XxlQURVYOeRmSG\\\\\\\/y1\\\\\\\/kMlBxTbVvVzQ\\\\\\\/qkI5MfNHrcx7m5YXJAmJsdB5vxOWrQHnfWWjgmjDC2MIOidrdHELBJl4sjHzl4dbeBd5owfbXBIjfdoJNOY4YcEm6L9jKc2\\\\\\\/o67RXLXusvlZD6U6LGqUCggwzOlB9I6v66dE9LXfmOzxZ7l5R6AI8CNLHncC\\\\\\\/HE3+a2JDnfmYT\\\\\\\/FYCIZ8jf49EM8GkTMZ1256ixwETmkiKiU4roiAfLwAgCLGBQAoQdJK5eA1MUHrsTSxsyV39bdPhlBZ4rFWGA75Ojd6IAleqXUpHtuI7RI55cunTGh\\\\\\\/xM9rA6VAlS\\\\\\\/HQu0MlV8CWk9owSWU5bDqm8vuDYlOq4j14tOfF2KYCVmWjD79boAhKmWwysR+KhdFhXfLZuKO\\\\\\\/S\\\\\\\/UvoEYpWpigyrP2x9lZMDPq3HsQuvTNSjMofQQSeRSIqSP9FuZeAw29vBKAdusQJt17HhHnDRVLp7X4TVLxax4H+ncdM5qigdGjVenG1ZZYwWH+TrB30BODI\\\\\\\/0pJrN0vhi\\\\\\\/l2CMbRjEYu\\\\\\\/Qt8quLcBSvenxbSuDDCDYr+zrl+GGqn9+4yIXp4vssnmCJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022jNsgDunSzwzPQAycZoDMfQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022653025cb89e05c820aae4e9b9d2535e408048b7d633c3fe04c82c6b4b5eba909\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.16.875872v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1042888 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.16.875872v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-B0aTlCXJt6ERHkRUnZEHpoaDN4z9xeuo3uk6SNHYCIU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1042888" id="node1042888" data-pisa="biorxiv;2019.12.16.875872v2" data-pisa-master="biorxiv;2019.12.16.875872" data-apath="/biorxiv/early/2019/12/17/2019.12.16.875872.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1163-567X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kelsey T.</span> <span class="nlm-surname">Young</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Kevin K.</span> <span class="nlm-surname">Lahmers</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Holly S.</span> <span class="nlm-surname">Sellers</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">David E.</span> <span class="nlm-surname">Stallknecht</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Rebecca L.</span> <span class="nlm-surname">Poulson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jerry T.</span> <span class="nlm-surname">Saliki</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">S. Mark</span> <span class="nlm-surname">Tompkins</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Ian</span> <span class="nlm-surname">Padykula</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Chris</span> <span class="nlm-surname">Siepker</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Elizabeth W.</span> <span class="nlm-surname">Howerth</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Michelle</span> <span class="nlm-surname">Todd</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">James B.</span> <span class="nlm-surname">Stanton</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.16.875872 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1042888" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kelsey T. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Pathology, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsey%2BT.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20KT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelsey%2BT.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1163-567X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelsey T. Young</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kevin K. Lahmers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Biomedical Sciences &#38; Pathobiology, VA-MD College of Veterinary Medicine, Virginia Tech University</span>, Blacksburg, VA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BK.%2BLahmers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lahmers%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BK.%2BLahmers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Holly S. Sellers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Poultry Diagnostic and Research Center, Department of Population Health, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Holly%2BS.%2BSellers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sellers%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHolly%2BS.%2BSellers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">David E. Stallknecht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Southeastern Cooperative Wildlife Disease Study Department of Population Health, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BE.%2BStallknecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stallknecht%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BE.%2BStallknecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rebecca L. Poulson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Southeastern Cooperative Wildlife Disease Study Department of Population Health, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BL.%2BPoulson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poulson%20RL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BL.%2BPoulson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jerry T. Saliki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Athens Veterinary Diagnostic Laboratory, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jerry%2BT.%2BSaliki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saliki%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJerry%2BT.%2BSaliki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">S. Mark Tompkins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Center for Vaccines and Immunology, Department of Infectious Diseases, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S.%2BMark%2BTompkins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tompkins%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS.%2BMark%2BTompkins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ian Padykula </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Center for Vaccines and Immunology, Department of Infectious Diseases, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BPadykula%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Padykula%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BPadykula%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Chris Siepker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Pathology, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BSiepker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Siepker%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BSiepker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Elizabeth W. Howerth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Pathology, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BW.%2BHowerth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howerth%20EW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BW.%2BHowerth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Michelle Todd </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Biomedical Sciences &#38; Pathobiology, VA-MD College of Veterinary Medicine, Virginia Tech University</span>, Blacksburg, VA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BTodd%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Todd%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichelle%2BTodd%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">James B. Stanton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Pathology, College of Veterinary Medicine, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BB.%2BStanton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stanton%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BB.%2BStanton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jbs@uga.edu" class="" data-icon-position="" data-hide-link-title="0">jbs@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.16.875872v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1042888" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1042888/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.16.875872v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1042888" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1042888/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.16.875872v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1042888" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1042888/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.16.875872v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1042888" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1042888/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.16.875872v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1042888" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1042888/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.16.875872v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1042888" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1042888/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">RNA viruses rapidly mutate, which can result in increased virulence, increased escape from vaccine protection, and false negative detection results. Targeted detection methods have a limited ability to detect unknown viruses and often provide insufficient data to detect coinfections or identify antigenic variants. Random, deep sequencing is a method that can more fully detect and characterize RNA viruses and is often coupled with molecular techniques or culture methods for viral enrichment. Viral culture coupled with third-generation sequencing were tested for the ability to detect and characterize RNA viruses. Cultures of bovine viral diarrhea virus, canine distemper virus, epizootic hemorrhagic disease virus, infectious bronchitis virus, two influenza A viruses, and porcine respiratory and reproductive syndrome virus were sequenced on the MinION platform using a random, reverse primer in a strand-switching reaction, coupled with PCR-based barcoding. Reads were taxonomically classified and used for reference-based sequence building using a stock personal computer. This method accurately detected and identified complete coding sequence genomes with a minimum of 20× coverage depth for all seven viruses, including a sample containing two viruses. Each lineage-typing region had at least 26× coverage depth for all viruses. Furthermore, analyzing the canine distemper virus sample through a pipeline devoid of canine distemper virus reference sequences modeled the ability of this protocol to detect unknown viruses. These results show the ability of this technique to detect and characterize dsRNA, negative- and positive-sense ssRNA, nonsegmented, and segmented RNA viruses.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">RNA viruses are common etiological agents of animal diseases. Many of these diseases, such as influenza,<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> Newcastle disease (ND),<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup> epizootic hemorrhagic disease (EHD),<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup> infectious bronchitis (IB),<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> and porcine respiratory and reproductive syndrome (PRRS)<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup> are global economic and/or health burdens for domestic and wild animal populations. Moreover, RNA viruses account for most emerging diseases due to the swift production of genetic variants, enabling rapid evolution for adapting to environments and hosts.<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup> The low fidelity of the viral-encoded RNA-dependent RNA polymerases significantly contributes to the genetic diversity, causing mutations up to a million times higher compared to cells.<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup> Additionally, genetically related viral species with segmented genomes (e.g., reoviruses and orthomyxoviruses) can reassort genomic segments, resulting in increased genetic variation. Enhanced virulence, resistance to vaccines, and ability for novel tissue tropism can also occur due to nucleotide mutations, and reassortment events.<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>-<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup> Therefore, there is a need for rapid detection of variants through whole genome sequencing for proper diagnosis, treatment, control, and prevention.<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup></p><p id="p-4">The molecular detection of RNA viruses is traditionally done using methods such as polymerase chain reaction (PCR), real-time PCR (rtPCR), cloning, and <em>in situ</em> hybridization (ISH).<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> However, these methods use targeted approaches that require prior knowledge of the viral genome for detection and are inefficient for the discovery of novel viruses, mixed infections, and identifying whole genomes. The advent of next generation sequencing (NGS) has permitted new techniques that circumvent some issues with targeted RNA sequencing. Platforms like Illumina allow untargeted deep sequencing for novel virus detection; but can be expensive and labor intensive, particularly for whole genome sequencing.<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup> Major drawbacks of these platforms is the generation of large volumes of raw data and short reads which require high performance computers and extensive computational analysis.<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup> Another limitation of untargeted RNA sequencing is the low relative abundance of viral RNA compared to host cellular RNA, which may require depletion of ribosomal RNA (rRNA) and/or enrichment of viral RNA to obtain an ample number of reads for viral strain detection.<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup> For these reasons, using NGS for accurate, whole viral genome sequencing remains challenging.</p><p id="p-5">The long read sequencing technology provided by MinION sequencing (Oxford Nanopore Technologies (ONT), Oxford, UK) has enabled rapid, inexpensive, high-throughput, whole-genome sequencing of viruses.<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup> MinION-based viral metagenomics studies have accurately sequenced and identified whole genomes of chikungunya virus, Zaire ebolavirus, hepatitis C virus, Venezuelan equine encephalitis virus, and Zika virus using untargeted approaches by targeting poly-A sites and directly sequencing RNA (dRNA-seq) or using primer-extension pre-amplification method (Round A/B); however, both of these methods resulted in low quality reads and poor depth of coverage across the viral genome.<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> Other methods such as sequence independent primer amplification (SISPA) have been used with MinION to obtain whole genome sequences for bovine enterovirus from culture and canine distemper virus from the brain of an affected dog.<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup> While virus targeting can be accomplished with the previously mentioned rRNA depletion or sequence targeting, it is also possible to couple this newest sequencing technology with classical virus culture for viral enrichment. The aim of this study was to develop a method to simultaneously detect and characterize various RNA viruses from culture by using a randomly primed, strand-switching approach and sequencing on MinION. Viruses were selected to include a double-stranded RNA virus (epizootic hemorrhagic disease virus 2 [EHDV-2; family <em>Reoviridae</em>, genus <em>Orbivirus</em>]), positive-stranded viruses (infectious bronchitis virus [IBV; family <em>Coronaviridae</em>, genus <em>Gammacoronavirus</em>, species <em>Avian coronavirus</em>], porcine reproductive and respiratory syndrome virus [PRRSV; family <em>Arteriviridae</em>, genus <em>Betaarterivirus</em>, species <em>Betaartervirus suid 2</em>], and bovine viral diarrhea virus [BVDV; family <em>Flaviviridae</em>, genus <em>Pestivirus</em>, species <em>Pestivirus B</em>]), and negative-stranded viruses (canine distemper virus [CDV; family <em>Paramyxoviridae</em>, genus <em>Morbillivirus</em>, species <em>Canine morbillivirus</em>] and influenza A viruses [IAV; family <em>Orthomyxoviridae</em>, genus <em>Alphainfluenzavirus</em>, species <em>Influenza A virus</em>] isolated from a dog and from a pig). Segmented (EHDV-2 and IAV) and unsegmented genomes (BVDV, CDV, and PRRSV) were also represented in this study. This approach provides rapid, complete genome coding sequences (CDS) for unknown RNA viruses in culture fluids and demonstrates the utility of the random hexamer-based strand-switching primer for MinION library synthesis.</p></div><div class="section" id="sec-2"><h2 class="">Materials and methods</h2><div id="sec-3" class="subsection"><h3>Samples</h3><p id="p-6">The EHDV-2 isolate was propagated on cattle pulmonary artery endothelial cells (CPAE) from spleen/lung tissue from a white-tailed deer (<em>Odocoileus virginianus</em>) collected in Georgia, USA in 2016 at the Southeastern Cooperative Wildlife Disease Study at the University of Georgia (UGA); CPAE cells are also persistently infected with BVDV. The CDV sample was isolated from the brain of an infant, female raccoon (<em>Procyon lotor</em>) from Kentucky in 2018 using African green monkey kidney cells expressing canine signaling lymphocytic activation molecule (Vero-Dog SLAM cell line) in the Athens Veterinary Diagnostic Laboratory (AVDL). The canine-origin IAV sample was collected from a nasal swab of a 7-year-old, male, boxer dog in 2015 from Georgia and was cultured in embryonated chicken eggs at the Poultry Diagnostic and Research Center (PDRC), UGA. The Center for Vaccines and Immunology, UGA provided the swine-origin influenza sample, which was isolated in 2019 from a 6-month-old, female, Hampshire-cross pig from Georgia after testing positive for IAV by immunohistochemistry and PCR. Swine IAV from lung homogenate was propagated on Madin-Darby canine kidney (MDCK) cells. The IBV sample (Mass vaccine) was cultured in embryonated chicken eggs and supplied by the PDRC. Isolate VR2385 of PRRSV was cultured on MARC-145 cells at the Veterinary Diagnostic Laboratory, Iowa State University. Reverse-transcription real-time PCR (RT-rtPCR) was conducted for BVDV, EHDV-2,<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup> CDV, canine-origin influenza, and IBV (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The CDV (AVDL), canine IAV (PDRC), and IBV (PDRC) RT-rtPCR assays were done using in-house diagnostic methods. For BVDV, (RT-rtPCR) was performed using a modified protocol<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> with SuperScript® III First-Strand Synthesis SuperMix for qRT-PCR (Invitrogen, Carlsbad, CA) and amplified in a CFX96™ Touch Real-Time PCR Detection System (Bio-Rad Laboratories, Inc, Hercules, CA) thermal cycler.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1051753/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1051753/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1051753" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-7" class="first-child">Sample background information and reverse-transcription real-time PCR (RT-rtPCR) results.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Total RNA extraction</h3><p id="p-8">All RNA extraction to sequencing steps for all viruses, except for PRRSV and a CDV replicate, were conducted at UGA. Library preparation and sequencing of PRRSV and a CDV replicate were conducted at Virginia Tech University. Total RNA for CDV, EHDV-2, BVDV, swine IAV, and canine IAV was extracted using 1 ml of culture supernatant using Trizol® LS Reagent (ThermoFisher Scientific, Waltham, MA) following manufacturer’s protocol. Total RNA was eluted in 88.5 µl of nuclease-free water (Qiagen, Hilden, Germany). DNase treatment was performed using RNase-Free DNase Set (Qiagen) and then purified using RNeasy® MinElute® Cleanup Kit (Qiagen) per manufacturer’s instructions. Total RNA for IBV was extracted using the QIAamp® Viral RNA Mini Kit (Qiagen) following manufacturer’s protocol. At Virginia Tech, RNA was extracted from PRRSV and one replicate of CDV using the QIAamp Viral RNA Mini Kit (Qiagen) according to the manufacturer’s instructions. Concentrations were measured using the Qubit® RNA HS Assay Kit (ThermoFisher Scientific) on a Qubit® 3.0 Fluorometer.</p></div><div id="sec-5" class="subsection"><h3>Strand-switching cDNA synthesis</h3><p id="p-9">Strand-Switching cDNA synthesis for MinION sequencing was completed by modifying the 1D PCR barcoding cDNA (SQK-LSK108) protocol from ONT. Reverse transcription was performed by combining 8 µl of total RNA, 2 µl of 1 µM PCR-RH-RT primer (5’ - /5Phos/<span class="underline">ACTTGCCTGTCGCTCTATCTTC</span>NNNNNN-3’; synthesized by Integrated DNA Technologies [IDT], Coralville, IA, with standard desalting, ONT adapter sequence is underlined), and 1 µl of 10 mM dNTPs. The reaction mixture was incubated at 65°C for 5 minutes, then snap cooled in an ice-water slurry for 1 minute. Then 4 µl of 5× RT buffer, 1 µl of 100mM DTT, 1 µl of 40 U/µl RNaseOUT™ (Invitrogen), and 2 µl of 10 µM strand-switching oligo (PCR_Sw_mod_3G) (5’ - TTTCTGTTGGTGCTGATATTGCTGCCATTACGGCCmGmGmG-3’; ONT provided sequence, synthesized by IDT with HPLC purification) were added and incubated at 42°C for 2 minutes. SuperScript™ IV Reverse Transcriptase (Invitrogen) was added at a volume of 1 µl and the reaction was incubated at the following conditions: 30 minutes at 50°C, 10 minutes at 42°C, and 10 minutes at 80°C. The cDNA was purified by KAPA Pure Beads (Kapa Biosystems, Wilmington, MA) or with AMPure XP beads (Beckman Coulter, Indianapolis, IN) at 0.7× beads: solution ratio.</p></div><div id="sec-6" class="subsection"><h3>Barcoding PCR</h3><p id="p-10">The reverse-transcribed cDNA was amplified following ONT’s 1D PCR barcoding cDNA (SQK-LSK108) protocol using the PCR Barcoding Expansion 1-12 (EXP-PBC001) kit (ONT) and LongAmp Taq 2× Master Mix (New England Biolabs, MA) with the following thermocycling conditions: 95°C for 3 minutes; 18 cycles of 95°C for 15 seconds, 62°C for 15 seconds, 65°C for 22 minutes; 65°C for 23 minutes. The barcoded, amplified DNA was bead purified at a 0.8× ratio.</p></div><div id="sec-7" class="subsection"><h3>MinION library preparation and sequencing</h3><p id="p-11">After the barcoding PCR, two to three samples were pooled by equal volume to make a total volume of 47 µl. MinION libraries were prepared from the pooled barcoded amplicons using the Ligation Sequencing Kit (SQK-LSK109) (ONT) following the 1D amplicon/cDNA by Ligation per ONT’s instructions. Briefly, the pooled samples were end prepped with NEBNext FFPE Repair Mix (New England Biolabs) and NEBNext End repair/dA-tailing Module (New England Biolabs) and bead purified at 1.0× bead volume. Then, sequencing adapters (ONT) were ligated onto the end-prepped library using NEBNext Quick T4 DNA Ligase (New England Biolabs) and bead purified (0.4× beads ratio) with the Long Fragment Buffer (LFB) (ONT). The final libraries were combined with Sequencing Buffer (SQB) and Loading Beads (LB) per ONT’s instructions and were sequenced on used or new FLO-MIN106 R9.4 flow cells (ONT), with the exception of the PRRSV and replicated CDV library, which were sequenced on a FLO-MIN107 R9.5, with the MinION Mk1b sequencer. Per the pre-run quality control check for previously used flow cells, a minimum of 1,000 active pores were required for sequencing. To maximize data generation from single flow cells, used flow cells were nuclease flushed, before or after sequencing, by adding a mixture of 10 µl DNase I and 190 µl DNase I Reaction Buffer (New England Biolabs) to the priming port and incubated for 30 minutes at room temperature. Sequencing was initiated by MinKNOW v.18.12.4–19.05.0 (ONT) using the 48-hour sequencing protocol with the live basecalling option turned off. During the run, available FAST5 files were processed as below to estimate number and length of viral reads to estimate sequencing time required to obtain sufficient genome coverage.</p></div><div id="sec-8" class="subsection"><h3>Pre-processing of raw MinION sequence data</h3><p id="p-12">The FAST5 files produced from sequencing for IBV, CDV, swine-origin and canine-origin influenza, and EHDV-2 and BVDV libraries were processed using an in-house script that sequentially basecalled, demultiplexed, and trimmed adapters on a Macbook Pro (3.1 GHz Intel Core i7, 8GB) using OS X El Capitan v.10.11.6. In summary, reads were basecalled using the CPU version of Guppy v.2.1.3 (ONT) by defining the appropriate configurations for the flowcell (FLO-MIN106) and kit (SQK-LSK109) used for sequencing and library preparation. Calibration strand detection and filtering was enabled with the --calib_detect parameter. The basecalled fastq files with a qscore ≥7 were sorted into a “pass” folder using the --qscore-filtering parameter and used for further analysis. Then, reads were demultiplexed using Porechop v.0.2.4 (<a href="https://github.com/rrwick/Porechop">https://github.com/rrwick/Porechop</a>) based on barcodes (-b output_dir) with -- require_two_barcodes setting enabled, 1,000,000 reads were aligned to all known adapter sets (--check_reads 1000000), adapters with 99% identity were trimmed from read ends (--adapter_threshold 99), and chimeric reads with middle adapters were removed.</p><p id="p-13">The raw FAST5 files produced for PRRSV and the CDV replicate from Virginia Tech were basecalled using the GPU version of Guppy v.3.1.5 on a Dell Precision 3060 Tower (Intel Core i7, 31.3GB, NVIDIA GeForce GTX 1080) using Ubuntu 16.04 LTS. Guppy was initiated using the same parameters as above with the flowcell configuration defined as “FLO-MIN107”. The basecalled reads were then trimmed and demultiplexed using Porechop with the same parameters described previously.</p></div><div id="sec-9" class="subsection"><h3>Virus classification and lineage typing</h3><p id="p-14">After basecalling, demultiplexing, and adapter trimming, reads were classified with Centrifuge v.1.0.4<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> by the lowest taxonomical rank with default parameters with the addition of allowing 50 assignments (or hits) for each read (-k 50) using a custom index built for each library based on the propagation system. Custom indices were constructed using an exhaustive search for complete genome sequences of all possible viruses infecting vertebrates downloaded from NCBI’s nucleotide database, as of 15 March 2019, that included 80,550 viral reference genomes (Supplementary Table 1). Low-complexity regions in sequences were masked using dustmasker (NCBI C++ Toolkit, <a href="https://ncbi.github.io/cxx-toolkit">https://ncbi.github.io/cxx-toolkit</a>). Dustmasked whole genomes of the species of cell line (see below), and/or the bovine genome (GCF_002263795.1_ARS-UCD1.2), to account for fetal bovine serum used in cell culture, were obtained using <kbd>centrifuge-download</kbd> and concatenated with the dustmasked vertebrate virus sequences to classify reads from host(s) cellular RNA. Using the concatenated files, indices were built by <kbd>centrifuge-build</kbd> with default parameters. The indices were then used to cluster reads based on short alignments to respective host and viral sequences to determine the presence and abundance of viral species in each sample using the standard out file. Reads to unexpected viral species were clustered and compared to the Genbank database using web-based nucleotide Basic Local Alignment Tool (BLASTn) (<a href="https://blast.ncbi.nlm.nih.gov">https://blast.ncbi.nlm.nih.gov</a>) with default settings. Percentage of viral reads and host reads were determined by removing duplicate reads and then dividing the number of reads clustered by the number of total reads after demultiplexing.</p><p id="p-15">Once the viral species was identified, custom lineage-typing Centrifuge indices were also built to further classify beyond viral species and assess for the possibility of mixed infections of the same virus species, as previously described for IBV.<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup> Briefly, for each main lineage per virus, one complete sequence of the lineage-typing region was selected and then combined with the respective genome of the cell line. Indices were constructed using the following: 20 N-terminal protease fragment (<em>N</em><sup><em>pro</em></sup>) sequences for BVDV<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup> with the bovine genome (GCF_002263795.1_ARS-UCD1.2), 13 hemagglutinin (<em>H</em>) gene sequences for CDV<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> with the African green monkey genome (GCF_000409795_Chlorocebus_sabeus_1.1), nine <em>VP2</em> sequences for EHDV<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> with the bovine genome (GCF_002263795.1_ARS-UCD1.2), 32 spike 1 (<em>S1</em>) sequences for IBV<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup> with the chicken genome (GCF_000002315.4_Gallus_gallus-5.0), 18 hemagglutinin (<em>HA</em>) sequences and 11 neuraminidase (<em>NA</em>) sequences for IAV<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup> with the canine genome (GCF_000002285.3_CanFam3.1) or chicken genome (GCF_000002315.6_GRCg6a), and 18 <em>ORF5</em> sequences for PRRSV<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> with the swine genome (GCF_000003025.6_Sscrofa11.1) (Supplementary Table 2). All sequences were dustmasked, assigned a unique taxonomy identification number, with the exception of the IAV sequences, and indices were built with default settings using <kbd>centrifuge-build</kbd>. Viral lineage typing was evaluated by aligning the individual trimmed and barcoded FASTQ files produced from Porechop to the respective custom lineage-typing indexes with Centrifuge, and reads were clustered by taxonomy identifications that represent potential lineage types in the sample. Each cluster of reads, representing potentially different lineages, were then used for reference based consensus building using Geneious v.11.1.3 (Biomatters, Aukland, New Zealand) to build a consensus for each lineage using the “Map to Reference” tool with medium sensitivity, re-iterating up to five times, and using the same reference sequences chosen for generating the custom Centrifuge lineage-typing databases. Consensus sequences derived for each potential lineage were analyzed using BLASTn with default settings. Results from BLASTn were sorted by query coverage and subject coverage. Subjects with the highest query, subject coverages, and identical bit-scores were considered the “top hit(s)” to identify the virus and lineage in each cluster. A coinfection was defined as different clusters of reads creating consensus sequences that matched to different lineages and reads could be parsed accordingly for final consensus building. If no coinfection was detected, all viral reads were used for consensus building.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1051754/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1051754/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1051754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-16" class="first-child">Reverse-transcription PCR (RT-PCR) primer information for Sanger sequencing.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Viral genome consensus sequence generation</h3><p id="p-17">After viruses were lineage typed for each sample, all reads that aligned to respective viral species from Centrifuge output were imported into Geneious for whole-genome, reference-based consensus building with the “Map to Reference” tool, after removing duplicate reads, using the same methods from above. Consensus sequence building was done with the requirement of at least 20× coverage at each base. References used for mapping were selected from using the “top hit” result in BLASTn from the lineage-typing analysis. Due to the known errors within long homopolymer regions during MinION sequencing,<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> consensus sequences were manually inspected and homopolymer areas that resulted in frameshifts were manually edited. The CDS regions were extracted from each edited consensus sequence and analyzed using BLASTn. For viruses with gapped genomes (e.g., CDV, IBV, and PRRSV), complete CDS and non-coding intergenic regions were extracted and analyzed with BLASTn. The “top hit” for each consensus sequence was determined by using the same criteria from above. The CDS regions for lineage-typing regions for each virus were also extracted and analyzed with BLASTn. The best “top hit”, percent pairwise identity, mean, minimum, maximum, and standard deviation of base coverage, and percentage of genome coverage were evaluated for each viral genome and lineage-typing region consensus sequences with the statistics tool in Geneious.</p></div><div id="sec-11" class="subsection"><h3>Simulating novel virus identification</h3><p id="p-18">In addition to the above method, sequencing data from the CDV sample were analyzed in a manner that simulated novel virus identification by constructing an all vertebrate virus, custom Centrifuge index with the omission of all canine morbillivirus sequences downloaded from NCBI as of 12 April 2019 (Supplementary Table 1). The index was assembled as described previously with African green monkey genome. Trimmed and demultiplexed reads from the CDV sample were classified with Centrifuge using parameters described above. Reads were grouped by taxonomy ID and exported into Geneious to build consensus sequences as described previously. The consensus was analyzed with BLASTn by excluding canine morbillivirus (taxid: 11232) in the search set. For comparison, Kraken-style reports were created using <kbd>centrifuge-kreport</kbd> from the Centrifuge output files from the CDV-absent pipeline and the CDV-present pipeline from above. Output files were visualized using Pavian v.1.0.<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> To discern the phylogeny of the “novel” virus among other pathogens in the <em>Morbillivirus</em> genus, the final genome CDS and phosphoprotein (<em>P</em>) gene CDS consensus sequences were aligned with 18 complete genome CDS and <em>P</em> gene CDS acquired from GenBank using the neighbor-joining algorithm in <em>ClustalW</em><sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup> with default settings. Phylogenetic trees for complete genome CDS and <em>P</em> gene CDS were inferred using the Maximum Likelihood statistical method based on the Tamura 3-parameter substitution model<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup> with bootstrap values calculated at 1000 replicates in MEGA X.<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup></p></div><div id="sec-12" class="subsection"><h3>Sanger sequencing for validation</h3><p id="p-19">Sanger sequencing was performed for the 5’ UTR region of BVDV, <em>H</em> gene of CDV, and <em>VP2</em> segment of EHDV-2 to confirm the identity of isolates and to compare MinION consensus sequences for the lineage-typing regions. Briefly, using 8 µl of total RNA from each sample, cDNA was synthesized using SuperScript® III First-Strand Synthesis System for RT-PCR (Invitrogen) with random hexamers (50 ng/µl) following manufacturer’s instructions. The cDNA from each sample was amplified using DreamTaq Green PCR Master Mix (2×) (ThermoFisher Scientific) following manufacturer’s protocol with 10 µM of each respective primer (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) targeting partial sequences of the lineage-typing regions and 1 µl of cDNA. Thermocycling conditions for PCR amplification of the 5’ UTR region for BVDV are as follows: 95°C for 5 minutes; 34 cycles of 95°C for 30 seconds, 58°C for 30 seconds, 72°C for 30 seconds; 72°C for 5 minutes. Thermocycling conditions for PCR amplification of the <em>H</em> gene for CDV are as follows: 95°C for 5 minutes; 40 cycles of 95°C for 30 seconds, 50°C for 30 seconds, 72°C for 1.5 minutes; 72°C for 10 minutes. Thermocycling conditions for PCR amplification of the <em>VP2</em> segment for EHDV are as follows: 95°C for 3 minutes; 40 cycles of 95°C for 30 seconds, 57°C for 30 seconds, 72°C for 45 seconds; 72°C for 5 minutes. Electrophoresis with a 1.0% agarose gel was performed to confirm PCR products. Amplicons were then purified using the QIAquick PCR Purification Kit (Qiagen) following manufacturer’s protocol and eluted in 30 µl nuclease-free water (Qiagen). Final concentration and purity were measured using NanoDrop™ 2000 Spectrophotometer (ThermoFisher Scientific). The purified PCR products and 5 µM of each primer (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) were submitted to GENEWIZ (South Plainfield, NJ) for bidirectional Sanger sequencing.</p></div><div id="sec-13" class="subsection"><h3>Sanger sequencing analysis and pairwise identity with MinION consensus sequences</h3><p id="p-20">Results from Sanger sequencing were analyzed with Geneious. Using the chromatograms, low quality regions (≥Q40) from the 5□ and 3□ ends were trimmed and ambiguous bases were manually edited. Primers were trimmed before aligning forward and reverse sequences from each sample using Geneious Alignment with default settings. The consensus sequence was compared using BLASTn and selecting the best “top hit” with the same criteria described above for MinION data.</p><p id="p-21">Partial lineage-typing sequences from Sanger sequencing for BVDV, CDV, and EHDV were compared with the full-length lineage-typing consensus sequences from MinION using Geneious Alignment with default settings. Pairwise identity of the alignment was calculated with Geneious.</p></div></div><div class="section" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>MinION sequencing, viral classification, and lineage typing</h3><p id="p-22">Libraries were prepared for seven cultured samples for seven different viruses by using random hexamer primed, strand-switching for reverse transcription, PCR-based barcoding, and pooled sequencing using MinION. Sequencing occurred for a minimum of about 2 hrs and 45 min and a maximum of about 47 hrs and 37 min to obtain 567,780–6,984,000 raw reads. Total sequencing time varied between runs based on the time of day the run was started, the number of viral reads detected early in the sequencing run, and the ability to re-use a flow cell (i.e., if the flow cell was not to be reused, then it was often sequenced to near exhaustion). After the raw reads were basecalled, demultiplexed, and trimmed, 7,523–1,173,058 reads were assigned to barcodes of interest and used for viral classification (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1051748/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1051748/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1051748" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-23" class="first-child">Sequencing data from Porechop and Centrifuge.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Barcoded reads for each of the seven libraries were aligned to custom-built indices with Centrifuge that detected seven different viruses belonging to three main types of RNA viral genomes: BVDV (positive-sense ssRNA), CDV (negative-sense ssRNA), EHDV-2 (dsRNA), IBV (positive-sense ssRNA), canine-origin IAV (negative-sense ssRNA), swine-origin IAV (negative-sense ssRNA), and PRRSV (positive-sense ssRNA). Additionally, three of the viruses, EHDV-2, canine-origin influenza, and swine-origin influenza, are comprised of segmented genomes. The randomly primed strand-switching method was also able to classify a sample containing EHDV-2 and BVDV. All viruses detected were categorized with viral reads representing 0.97–63.3% of all reads sequenced for each sample with percentage of host reads ranging from 2.5 to 80.4% (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Reads to unexpected viral species were analyzed with BLASTn and were determined to be short alignments to various host, often ribosomal, sequences.</p><p id="p-25">Viral sequences were parsed and further classified into lineage types using custom built lineage-typing indices for Centrifuge. Reads were clustered based on the lineage-typed alignment and consensus sequences were built using Geneious. While the Centrifuge alignments suggested the possibility of 2–19 lineages per virus isolate, only one lineage per species was detected in each isolate after consensus building of each potential lineage (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1051749/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1051749/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1051749" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4.</span> <p id="p-26" class="first-child">Consensus results for coding sequences (CDS) of lineage-typing regions for each virus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">The complete CDS for each virus’s lineage-typing region had at least 33× depth of coverage with a mean range of 105.1–22,968.0 (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). The BLASTn comparison for the genotyping regions resulted in 96.49–100.0% pairwise identity with each “top hit” (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Using the <em>N</em><sup><em>pro</em></sup> region, the lineage of BVDV was determined as genotype BVDV-2a with 96.23% identity with McCart_C strain. Sequences from both libraries of CDV were genotyped as America II with 96.49% identity of the <em>H</em> gene to A75/17. The genotype for EHDV-2 was determined as EHDV-2w with 99.90% identity to the OV215 strain using the <em>VP2</em> segment. IBV was determined to be genotype 1-lineage 4, with 100.0% identity of the <em>S1</em> gene to the Ma5 strain. The <em>HA</em> and <em>NA</em> segments of the canine-origin IAV showed 100.0% and 99.79% identity, respectively, to an H3N2 subtype circulating in dogs in North America. The porcine-origin influenza was determined to be an H1N2 subtype similar to other North American porcine-origin IAVs, with 99.65% identity to the <em>HA</em> segment and 99.79% identity to the <em>NA</em> segment. The <em>ORF5</em> for PRRSV had 99.67% identity to VR2385 and was determined as a Type 2, lineage 5.</p></div><div id="sec-16" class="subsection"><h3>Viral genome consensus sequence evaluation</h3><p id="p-28">For all seven viruses, complete genome CDS were acquired with a minimum of 26× depth of each base, with the exception of the replicated CDV with 90.2% genome coverage at 20× depth (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Additionally, complete CDS for all segments of three segmented viruses, EHDV-2 (dsRNA), swine influenza, and canine influenza (negative-sense ssRNA), were obtained with a minimum of 43× depth of coverage and an average of 99.89% identity across all segments. The mean depth for each virus was 72.1–28,014.8. The complete genome CDS consensus sequences had a high pairwise identity of 97.37–100.0% to their respective “top hit” when using BLASTn to compare sequences with GenBank databases (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). The complete CDS consensus sequences for BVDV, EHDV, and canine IAV were deposited in GenBank under the following accession numbers: BVDV = <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN824468&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom">MN824468</a>; EHDV= <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN824457&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom">MN824457</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN824466&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom">MN824466</a>; canine IAV = <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN812282&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom">MN812282</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN812289.&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom">MN812289.</a> The complete CDS with intergenic regions for CDV was also deposited in Genbank as accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN824467&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom">MN824467</a>.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1051752/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1051752/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5.</span> <p id="p-29" class="first-child">Consensus results of whole genome coding sequences (CDS) for each virus.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Simulating novel virus identification</h3><p id="p-30">Under simulated conditions in which CDV would be an unknown virus, the Centrifuge standard out file for CDV analyzed showed that approximately 500 reads aligned to a morbillivirus, with 239 reads aligned to phocine distemper virus (PDV; family <em>Paramyxoviridae</em>, genus <em>Morbillivirus</em>, species <em>Phocine morbillivirus</em>) and fewer reads (&lt;70) aligning to other more distantly related morbilliviruses: feline morbilliviruses (species <em>Feline morbillivirus</em>), rinderpest virus (species <em>Rinderpest morbillivirus</em>), peste-des-petits-ruminants virus (species <em>Small ruminant morbillivirus</em>), cetacean morbillivirus (species <em>Cetacean morbillivirus</em>), and measles virus (species <em>Measles morbillivirus</em>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). In contrast, when CDV was included in the Centrifuge database, approximately 4,000 reads aligned to CDV, which was the only morbillivirus detected (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>). In the CDV-absent analysis, reads scattered across different morbilliviruses suggesting that the actual species was absent from the database but is most similar to PDV. A total of 251 (12 reads aligned to 3 or more sequences, resulting in these reads not being counted in the Centrifuge standard out file) reads were classified as PDV and were exported to Geneious for reference-based consensus building by aligning to PDV/Wadden_Sea.NLD/1988 (accession: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_028249&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom">NC_028249</a>). The consensus sequence was analyzed with BLASTn by excluding canine morbillivirus in the search set which resulted in a 78.71% identity with PDV/Wadden_Sea.NLD/1988 (accession: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KC802221.1&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom">KC802221.1</a>) with 78.0% query coverage. Furthermore, phylogenetic analysis of the whole genome CDS and <em>P</em> gene CDS consensus of the dubbed “novel” sequence clustered with PDV but shows sequence divergence suggestive of a species similar to, but different from, PDV (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sankey diagram displaying the top ten nodes per taxonomic classification level. Centrifuge v.1.0.424 Kraken-style reports from analyzing sequences from canine distemper virus (CDV) with (a) pipeline containing CDV sequences and (b) novel virus simulation using a pipeline devoid of CDV sequences in custom Centrifuge indices were visualized with Pavian v.1.0.33 Number of reads assigned to taxonomic classification correspond with the width of the flow and are listed above each node. Reads hitting to the bovine and African green monkey genomes were excluded. Abbreviations are: D = domain; F = family; G = genus; P=phylum; S = species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-216968235" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sankey diagram displaying the top ten nodes per taxonomic classification level. Centrifuge v.1.0.4&lt;sup&gt;24&lt;/sup&gt; Kraken-style reports from analyzing sequences from canine distemper virus (CDV) with (a) pipeline containing CDV sequences and (b) novel virus simulation using a pipeline devoid of CDV sequences in custom Centrifuge indices were visualized with Pavian v.1.0.&lt;sup&gt;33&lt;/sup&gt; Number of reads assigned to taxonomic classification correspond with the width of the flow and are listed above each node. Reads hitting to the bovine and African green monkey genomes were excluded. Abbreviations are: D = domain; F = family; G = genus; P=phylum; S = species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F1.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F1.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-31" class="first-child">Sankey diagram displaying the top ten nodes per taxonomic classification level. Centrifuge v.1.0.4<sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup> Kraken-style reports from analyzing sequences from canine distemper virus (CDV) with (a) pipeline containing CDV sequences and (b) novel virus simulation using a pipeline devoid of CDV sequences in custom Centrifuge indices were visualized with Pavian v.1.0.<sup><a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a></sup> Number of reads assigned to taxonomic classification correspond with the width of the flow and are listed above each node. Reads hitting to the bovine and African green monkey genomes were excluded. Abbreviations are: D = domain; F = family; G = genus; P=phylum; S = species.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic analysis of morbilliviruses of (a), whole genome CDS and (b), P gene CDS after a data analysis of CDV sequences using a pipeline devoid of CDV reads to simulate novel virus detection and identification. Each sequence is designated by species abbreviation and GenBank accession number. The sequence denoted with an asterisk was generated from the data analysis pipeline with MinION sequencing. The evolutionary history was inferred by using the Maximum Likelihood method and Tamura 3-parameter model.35 These analyses involved 19 nucleotide sequences and a total of 14,578 positions (a) or 1,460 positions (b) in the final dataset. All positions containing gaps and missing data were eliminated. Evolutionary analyses were conducted in MEGA X.36 Abbreviations are: CeMV = cetacean morbillivirus; FeMV = feline morbillivirus; MeV = measles virus; PDV = phocine distemper virus; PPRV = peste-des-pestits-ruminants virus; RPV = rinderpest virus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-216968235" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic analysis of morbilliviruses of (a), whole genome CDS and (b), &lt;em&gt;P&lt;/em&gt; gene CDS after a data analysis of CDV sequences using a pipeline devoid of CDV reads to simulate novel virus detection and identification. Each sequence is designated by species abbreviation and GenBank accession number. The sequence denoted with an asterisk was generated from the data analysis pipeline with MinION sequencing. The evolutionary history was inferred by using the Maximum Likelihood method and Tamura 3-parameter model.&lt;sup&gt;35&lt;/sup&gt; These analyses involved 19 nucleotide sequences and a total of 14,578 positions (a) or 1,460 positions (b) in the final dataset. All positions containing gaps and missing data were eliminated. Evolutionary analyses were conducted in MEGA X.&lt;sup&gt;36&lt;/sup&gt; Abbreviations are: CeMV = cetacean morbillivirus; FeMV = feline morbillivirus; MeV = measles virus; PDV = phocine distemper virus; PPRV = peste-des-pestits-ruminants virus; RPV = rinderpest virus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F2.medium.gif" width="440" height="187"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F2.medium.gif" width="440" height="187"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/17/2019.12.16.875872/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-32" class="first-child">Phylogenetic analysis of morbilliviruses of (a), whole genome CDS and (b), <em>P</em> gene CDS after a data analysis of CDV sequences using a pipeline devoid of CDV reads to simulate novel virus detection and identification. Each sequence is designated by species abbreviation and GenBank accession number. The sequence denoted with an asterisk was generated from the data analysis pipeline with MinION sequencing. The evolutionary history was inferred by using the Maximum Likelihood method and Tamura 3-parameter model.<sup><a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a></sup> These analyses involved 19 nucleotide sequences and a total of 14,578 positions (a) or 1,460 positions (b) in the final dataset. All positions containing gaps and missing data were eliminated. Evolutionary analyses were conducted in MEGA X.<sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a></sup> Abbreviations are: CeMV = cetacean morbillivirus; FeMV = feline morbillivirus; MeV = measles virus; PDV = phocine distemper virus; PPRV = peste-des-pestits-ruminants virus; RPV = rinderpest virus.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Sanger Sequencing and pairwise identity with MinION</h3><p id="p-33">Sanger sequencing targeting partial sequences of the lineage-typing regions for BVDV, CDV, and EHDV was used to confirm lineage types in the samples (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). For BVDV, primers targeting a partial sequence of the 5□ UTR were used and resulted in a “top hit” to BVDV-2 isolate 95-1501. Sequencing of the <em>H</em> gene for CDV resulted in a “top hit” to CDV isolate THA/VG. The <em>VP2</em> segment was targeted for EHDV and resulted in a “top hit” to EHDV-2 isolate OV617. The consensus sequences from Sanger and MinION sequencing had 100.0% pairwise identities for all three viruses (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). MinION consensus sequences compared to Sanger for BVDV and EHDV had identical “top hits”. For CDV, MinION sequencing had a “top hit” to CDV A75/15 with 96.49% identity but the shorter 923 bp fragment from Sanger had a top alignment with CDV THA/VG with 96.53% identity (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>). The Sanger sequence is based on smaller fragments of the lineage-typing region and, therefore, the BLASTn-based pairwise identities cannot be directly compared between MinION and Sanger sequences.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1051750/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1051750/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1051750" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6.</span> <p id="p-34" class="first-child">Sanger and MinION sequencing consensus pairwise identity results.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-19"><h2 class="">Discussion</h2><p id="p-35">A large proportion of emerging diseases are caused by RNA viruses<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup> and, due to their mutability, rapid detection is needed; however, this can be hindered due to the series of PCR panels used for identification of unknown viruses and inefficient for the discovery of coinfections.<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup> Deep sequencing-based approaches using viral nucleic acid enrichment methods have been described to address this issue, including targeted and untargeted library preparations, like SISPA.<sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup> The methodology in this study demonstrates the application of culture-based viral enrichment followed by random, strand-switching MinION sequencing for accurately detecting and characterizing RNA viruses. RNA viruses with varying genome compositions (single stranded [positive- and negative-sense], double stranded, and segmented) were used to demonstrate the ability of untargeted strand-switching to obtain complete CDS of genotyping regions and whole genomes with viral culture enrichment methods. Moreover, two viruses (EHDV-2 and BVDV) were detected from a single sample, illustrating the utility of the random sequencing approach. Lastly, data analysis for one sample (CDV) was treated as a novel virus, highlighting the feasibility of this method to identify a new or poorly characterized virus.</p><p id="p-36">This random sequencing approach proved to be robust across various RNA viruses in obtaining full length CDS of the complete genome after using an unbiased, fast aligner to identify the likely virus, followed by reference-based consensus building to identify the lineage type. With at least 26× depth of coverage across the genome for all viruses, whole genome CDS had 96.99–100.0% identity to the top BLASTn hits. Complete CDS for all segments of three segmented viruses, EHDV-2 (dsRNA), swine influenza, and canine influenza (negative-sense ssRNA), were obtained with a minimum of 43× depth of coverage and an average of 99.89% identity across all segments. The best “top hit” for each segment does not match across all segments to the same isolate, which may be due to reassortment events for segmented genomes<sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup> and the unavailable sequence data for all segments for some isolates in NCBI.</p><p id="p-37">The alignments of the whole genome CDS and the lineage-typing regions were similar and consistent with the origin of the samples in this study. The identification of the EHDV as an EHDV-2w with highest similarity to EHDVs circulating in white-tailed deer in the southeastern United States in 2017 is consistent with the collection of this sample from a white-tailed deer in Georgia in 2016. The influenza sample of canine origin was determined as a canine H3N2 strain, matching the typing completed as part of the original diagnostic case workup. The full sequencing provided by this method was able to confirm that the isolate in this study was most similar to canine H3N2 viruses circulating in the southeastern United States in 2015, consistent with the time and geographic location in which this sample was collected. Lineage typing of the swine-origin influenza virus categorized it as an H1N2 most similar to an H1N2 IAV isolated from a swine in 2018 from Ohio. It is possible that the slightly lower percent identity for the swine-origin influenza is due to a relative paucity of sequence data available for 2019 swine-origin IAV at this time. The CDV America II lineage, the classification of CDV in this study, is a common lineage found in North American wildlife<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> and is consistent with the collection of this sample from a raccoon in Kentucky. The sequences for IBV and PRRSV were highly similar to the known sequences of those isolates.</p><p id="p-38">This protocol was also tested on two scenarios. The first was the detection of two viruses in a single culture system, demonstrating its ability to simultaneously obtain accurate, complete CDS for rapid detection and characterization of viruses in coinfected samples, comparable to other studies using advanced sequencing for analysis of cultured viruses.<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup> The second was the data analysis under the simulating conditions of CDV being a novel virus. This resulted in read alignments that spanned across the <em>Morbillivirus</em> genus. Excluding the background hits, the largest proportion of reads hit to PDV and consensus building with these reads had a low pairwise identity (78.71%), suggesting it was not a PDV. The phylogenetic divergence gives evidence that the virus in the sample belongs to the <em>Morbillivirus</em> genus and is phylogenetically related to PDV. If CDV was truly an unknown virus, the Centrifuge output would have suggested the identification of a novel or divergent virus that is most similar to PDV sequences, consistent with the known close genetic relationship between CDV and PDV.<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup> Furthermore, in the event of an unknown viral etiology and due to the multiplexing capabilities of MinION sequencing, DNA library approaches can be applied and sequenced concurrently with the random strand-switching library to identify RNA or DNA virus in the sample.<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup></p><p id="p-39">Sanger sequencing was performed on partial lineage-typing sequences for BVDV, CDV, and EHDV and compared with the full-length lineage-typing regions obtained from MinION sequencing, resulting in 100.0% pairwise identity. The 100% pairwise identity between the Sanger and MinION sequences is consistent with previous results using MinION sequencing,<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup> that demonstrated the ability to obtain accurate MinION sequences by increasing depth of coverage.<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup> Sanger sequencing and BLASTn results for the 5□ UTR for BVDV and <em>VP2</em> for EHDV matched the MinION lineage-typing region BLASTn results. For the <em>H</em> gene for CDV, Sanger sequencing had a top BLASTn alignment with isolate THA/VG with 96.53%; however, the results did show 96.0% identity with isolate A75/15, comparable to MinION sequencing result of 96.49% identity to isolate A75/15. It is noteworthy that aligning the Sanger and MinION results for the <em>H</em> gene in CDV showed 100.0% pairwise identity. The differences in BLASTn top alignments are attributed to the shorter sequence from Sanger sequencing, causing a slightly lower sequence specificity. Longer fragments, particularly whole genome sequencing, have shown improved resolution for genotyping analyses.<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup> Additionally, the limitations of partial sequences used in Sanger sequencing are illustrated by the inability of Sanger sequencing to identify a single best hit for EHDV-2 and CDV, which resulted in multiple hits with identical BLASTn scores; whereas, the MinION-based method obtained complete CDS of the <em>VP2</em> segment and <em>H</em> gene allowing for a single best hit for these viruses. In addition to more accurate classification, whole genome sequencing of viruses is advantageous in providing more data for detection, epidemiology, genotyping and phylogenetic analysis compared to partial and/or targeted sequences obtained from other classical sequencing methods. The relatively limited data provided by targeted sequencing may require further sequencing and expenses to obtain additional biological information. In particular, different genetic sequences are used for phylogenetic analysis and genotyping. Some viruses also have multiple genotyping regions, such as BVDV, EHDV, and influenza.<sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup> Furthermore, recombination events are often difficult to identify with partial sequences and are important in investigating host range, virulence, and vaccine evasion.<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>-<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup> Thus, routine, whole genome sequencing of viruses gives quick turnaround data for various analyses and could provide more comprehensive databases needed to understand viral evolution.</p><p id="p-40">Random, deep sequencing for many high-throughput sequencing platforms has some caveats such as false hits. As one example, illustrated in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, some reads were classified as lassa mammarenavirus, various herpesviruses, and others. Endogenous retroviruses and some DNA viruses were also frequently found to be present after annotating sequences. After clustering and aligning these reads to GenBank using BLASTn, it was determined that the reads were short sequences that matched to various host sequences. Reference sequences of the host and propagation systems of the cultured viruses were included in the Centrifuge indices to reduce these misalignments; however, these genomes are not as well described as other commonly studied organisms and may not represent the full sequence diversity of the host genome. Users can adjust alignment settings to help reduce the number of false hits, but as with any test, increasing the specificity of the test will negatively impact sensitivity. Similar to other fields (e.g., background lesions in pathology, growth of nonpathogenic bacteria in bacterial cultures), confirmatory tests may be required and analysis of deep sequencing results requires evaluation by a trained user, one experienced with bioinformatic methodology and with knowledge of veterinary infectious diseases.</p><p id="p-41">While rapid, cost-effective whole-genome sequencing may be useful in a diagnostic setting, ease of use and robustness across laboratories is key for deployment. For this reason, libraries for CDV and PRRSV were prepared at different laboratories. Both libraries for CDV had similar results; however, the differences can possibly be attributed to the use of a sequencing kit intended for 1D sequencing with a FLO-MIN107 (R9.5) flowcell that is typically used for 1D<sup>2</sup> sequencing. The repercussions of the 1D kit with a 1D<sup>2</sup> flowcell combination are not well known but it is of interest to note that trimming the basecalled files resulted in 1,691,792 reads removed due to middle adapters, significantly decreasing the total number of reads assigned to each barcode.</p><p id="p-42">The present study provides promising results for quick identification of unknown cultured RNA viruses by using MinION sequencing with a previously untested random approach. Future studies to compare the varying random methods of deep sequencing are required to determine the most efficient methods. As with other deep sequencing methods,<sup><a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a></sup> MinION-based sequencing will also likely be used to metagenomically detect viruses directly from clinical samples (e.g., serum, swabs, tissues), and work is ongoing to investigate this usage.</p><p id="p-43">MinION sequencing is a cost-effective way to multiplex samples and achieve long reads for more accurate genome consensus building compared to other short read sequencing technologies. Overall, the addition of full-genome sequencing to more routine diagnostic use will increase the available knowledge regarding sequence diversity and allow for improved tracking of viruses and a better understanding of the genetic determinants of viral pathogenesis.</p></div><div class="section" id="sec-21"><h2 class="">Declaration of conflicting interests</h2><p id="p-45">The authors declared no potential conflicts of interest with respect to the research, authorship, and/or publication of this article.</p></div><div class="section" id="sec-22"><h2 class="">Funding</h2><p id="p-46">The project described was supported by Agriculture and Food Research Initiative Competitive Grant no. 2018-67015-28306 from the USDA National Institute of Food and Agriculture. This project has been funded in part with funds from the National Institute of Allergy and Infectious Diseases, National Institutes of Health, Department of Health and Human Services, under Center of Excellence of Influenza Research and Surveillance (CEIRS) under contract number HHSN272201400004C (SMT).</p></div><div class="section" id="sec-23"><h2 class="">Supplementary material</h2><p id="p-47">Supplementary material for this article is available online.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">The authors acknowledge Vanessa Gauthiersloan, Samantha Day and Erich Linnemann from the Poultry Diagnostic &amp; Research Center, University of Georgia, Clara Kienzle-Dean from the Southeastern Cooperative Wildlife Disease Study, University of Georgia, and Pablo Pinyero from the Veterinary Diagnostic Lab, Iowa State University for their technical help.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.1" data-doi="10.3382/ps.2009-00556"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sartore</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The effects of control measures on the economic burden associated with epidemics of avian influenza in Italy</span>. <abbr class="cit-jnl-abbrev">Poult Sci</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">89</span>:<span class="cit-fpage">1115</span>–<span class="cit-lpage">1121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPoult%2BSci%26rft_id%253Dinfo%253Adoi%252F10.3382%252Fps.2009-00556%26rft_id%253Dinfo%253Apmid%252F20460656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3382/ps.2009-00556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20460656&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Butt</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid virulence prediction and identification of Newcastle disease virus genotypes using third-generation sequencing</span>. <abbr class="cit-jnl-abbrev">Virol J</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">15</span>:<span class="cit-fpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol%2BJ%26rft.volume%253D15%26rft.spage%253D179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.3" data-doi="10.1177/104063871002200414"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clavijo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">An improved Real-Time Polymerase Chain Reaction for the Simultaneous Detection of All Serotypes of Epizootic Hemorrhagic Disease Virus</span>. <abbr class="cit-jnl-abbrev">J Vet Diagn Invest</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">22</span>:<span class="cit-fpage">588</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVeterinary%2BDiagnostic%2BInvestigation%26rft.stitle%253DJournal%2Bof%2BVeterinary%2BDiagnostic%2BInvestigation%26rft.aulast%253DClavijo%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D4%26rft.spage%253D588%26rft.epage%253D593%26rft.atitle%253DAn%2Bimproved%2BReal-Time%2BPolymerase%2BChain%2BReaction%2Bfor%2Bthe%2BSimultaneous%2BDetection%2Bof%2BAll%2BSerotypes%2Bof%2BEpizootic%2BHemorrhagic%2BDisease%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1177%252F104063871002200414%26rft_id%253Dinfo%253Apmid%252F20622230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/104063871002200414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20622230&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.16.875872v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>,  <span class="cit-name-given-names">D</span></span> and <span class="cit-auth"><span class="cit-name-surname">Naqi</span>  <span class="cit-name-given-names">SA</span></span>. <span class="cit-chapter-title">Infectious bronchitis</span>. <span class="cit-source">In: Diseases of Poultry</span>. <span class="cit-edition">11th ed.</span> <span class="cit-publ-loc">Ames, Iowa</span>: <span class="cit-publ-name">Iowa State Univer Pr</span>, <span class="cit-pub-date">2003</span>:<span class="cit-fpage">101</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">High-throughput whole genome sequencing of Porcine reproductive and respiratory syndrome virus from cell culture materials and clinical specimens using next-generation sequencing technology</span>. <abbr class="cit-jnl-abbrev">J Vet Diagn Invest</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">29</span>:<span class="cit-fpage">41</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVet%2BDiagn%2BInvest%26rft.volume%253D29%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.6" data-doi="10.3201/eid2212.160123"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woolhouse</span>  <span class="cit-name-given-names">MEJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Assessing the Epidemic Potential of RNA and DNA Viruses</span>. <abbr class="cit-jnl-abbrev">Emerg Inf Dis</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">22</span>:<span class="cit-fpage">2037</span>–<span class="cit-lpage">2044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInf%2BDis%26rft.volume%253D22%26rft.spage%253D2037%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2212.160123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2212.160123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rubio</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genetic variability and evolutionary dynamics of viruses of the family Closteroviridae</span>. <abbr class="cit-jnl-abbrev">Front. Microbiol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">4</span>:<span class="cit-fpage">151</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol%26rft.volume%253D4%26rft.spage%253D151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.8" data-doi="10.1136/jcp.53.7.502"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular techniques for clinical diagnostic virology</span>. <abbr class="cit-jnl-abbrev">J Clin Pathol</abbr> <span class="cit-pub-date">2000</span>;<span class="cit-vol">53</span>:<span class="cit-fpage">502</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BPathology%26rft.stitle%253DJ.%2BClin.%2BPathol.%26rft.aulast%253DRead%26rft.auinit1%253DS.%2BJ%26rft.volume%253D53%26rft.issue%253D7%26rft.spage%253D502%26rft.epage%253D506%26rft.atitle%253DMolecular%2Btechniques%2Bfor%2Bclinical%2Bdiagnostic%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fjcp.53.7.502%26rft_id%253Dinfo%253Apmid%252F10961172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6OToiamNsaW5wYXRoIjtzOjU6InJlc2lkIjtzOjg6IjUzLzcvNTAyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTcvMjAxOS4xMi4xNi44NzU4NzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.9" data-doi="10.1128/JVI.05662-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">RWY</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Tissue tropism of swine influenza viruses and reassortants in ex vivo cultures of the human respiratory tract and conjunctiva</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">85</span>:<span class="cit-fpage">11581</span>–<span class="cit-lpage">11587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05662-11%26rft_id%253Dinfo%253Apmid%252F21880750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8yMi8xMTU4MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE3LzIwMTkuMTIuMTYuODc1ODcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.16.875872v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Flint</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-chapter-title">Synthesis of RNA from RNA templates</span>. <span class="cit-source">In: Principles of Virology</span>. <span class="cit-edition">4th ed.</span> <span class="cit-publ-loc">Washington, DC</span>: <span class="cit-publ-name">ASM Press</span>, <span class="cit-pub-date">2015</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.11" data-doi="10.1128/CMR.00075-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilchrist</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Whole-genome sequencing in outbreak analysis</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Rev</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">28</span>:<span class="cit-fpage">541</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DGilchrist%26rft.auinit1%253DC.%2BA.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D541%26rft.epage%253D563%26rft.atitle%253DWhole-Genome%2BSequencing%2Bin%2BOutbreak%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00075-13%26rft_id%253Dinfo%253Apmid%252F25876885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjI4LzMvNTQxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTcvMjAxOS4xMi4xNi44NzU4NzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cobo</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Application of molecular diagnostic techniques for viral testing</span>. <abbr class="cit-jnl-abbrev">Open Virol J</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">6</span>:<span class="cit-fpage">104</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2BVirol%2BJ%26rft.volume%253D6%26rft.spage%253D104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.13" data-doi="10.1373/clinchem.2014.223289"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gullett</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Quantitative Nucleic Acid Amplification Methods for Viral Infections</span>. <abbr class="cit-jnl-abbrev">Clin Chem</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">61</span>:<span class="cit-fpage">72</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1373%252Fclinchem.2014.223289%26rft_id%253Dinfo%253Apmid%252F25403817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiY2xpbmNoZW0iO3M6NToicmVzaWQiO3M6NzoiNjEvMS83MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzE3LzIwMTkuMTIuMTYuODc1ODcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.14" data-doi="10.1016/j.bbrc.2015.12.083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Analysis of the microbiome: Advantages of whole genome shotgun versus 16S amplicon sequencing</span>. <abbr class="cit-jnl-abbrev">Biochem Biophys Res Commun</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">469</span>:<span class="cit-fpage">967</span>–<span class="cit-lpage">977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.volume%253D469%26rft.spage%253D967%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2015.12.083%26rft_id%253Dinfo%253Apmid%252F26718401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2015.12.083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26718401&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.15" data-doi="10.5772/61610"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kasibhatla</span>,  <span class="cit-name-given-names">SM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Analysis of next-generation sequencing data in virology-Opportunities and challenges</span>. <abbr class="cit-jnl-abbrev">IntechOpen</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.5772/61610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntechOpen%26rft_id%253Dinfo%253Adoi%252F10.5772%252F61610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5772/61610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.16" data-doi="10.2144/000114133"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Head</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Library construction for next-generation sequencing: overviews and challenges</span>. <abbr class="cit-jnl-abbrev">BioTechniques</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">56</span>:<span class="cit-fpage">61</span>–<span class="cit-lpage">64</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.volume%253D56%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000114133%26rft_id%253Dinfo%253Apmid%252F24502796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000114133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24502796&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331491800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.17" data-doi="10.1186/1471-2164-14-444"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Next generation sequencing of viral RNA genomes</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">14</span>:<span class="cit-fpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D444%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-444%26rft_id%253Dinfo%253Apmid%252F23822119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23822119&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.18" data-doi="10.1038/nature16996"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quick</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Real-time, portable genome sequencing for Ebola surveillance</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">530</span>:<span class="cit-fpage">228</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D530%26rft.spage%253D228%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16996%26rft_id%253Dinfo%253Apmid%252F26840485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16996&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26840485&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.19" data-doi="10.1186/s13073-015-0220-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greninger</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid metagenomic identification of viral pathogens in clinical samples by real-time nanopore sequencing analysis</span>. <abbr class="cit-jnl-abbrev">Genome Med</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">7</span>:<span class="cit-fpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed%26rft.volume%253D7%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13073-015-0220-9%26rft_id%253Dinfo%253Apmid%252F26416663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13073-015-0220-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26416663&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.20" data-doi="10.3201/eid2208.160270"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kilianski</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Use of Unamplified RNA/cDNA-Hybrid Nanopore Sequencing for Rapid Detection and Characterization of RNA Viruses</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">22</span>:<span class="cit-fpage">1448</span>–<span class="cit-lpage">1451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D22%26rft.spage%253D1448%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2208.160270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2208.160270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beato</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification and genetic characterization of bovine enterovirus by combination of two next generation sequencing platforms</span>. <abbr class="cit-jnl-abbrev">J Virol Methods</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">260</span>:<span class="cit-fpage">21</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%2BMethods%26rft.volume%253D260%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peserico</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Diagnosis and characterization of canine distemper virus through sequencing by MinION nanopore technology</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">1714</span>–<span class="cit-lpage">1714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D1714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.23" data-doi="10.1177/104063870201400205"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mahlum</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Detection of bovine viral diarrhea virus by TaqMan® reverse transcription polymerase chain reaction</span>. <abbr class="cit-jnl-abbrev">J Vet Diagn Invest</abbr> <span class="cit-pub-date">2002</span>;<span class="cit-vol">14</span>:<span class="cit-fpage">120</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVeterinary%2BDiagnostic%2BInvestigation%26rft.stitle%253DJournal%2Bof%2BVeterinary%2BDiagnostic%2BInvestigation%26rft.aulast%253DMahlum%26rft.auinit1%253DC.%2BE.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D120%26rft.epage%253D125%26rft.atitle%253DDetection%2Bof%2BBovine%2BViral%2BDiarrhea%2BVirus%2Bby%2BTaqMan%2528R%2529%2BReverse%2BTranscription%2BPolymerase%2BChain%2BReaction%26rft_id%253Dinfo%253Adoi%252F10.1177%252F104063870201400205%26rft_id%253Dinfo%253Apmid%252F11939332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/104063870201400205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11939332&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174669600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.24" data-doi="10.1101/gr.210641.116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Centrifuge: rapid and sensitive classification of metagenomic sequences</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">26</span>:<span class="cit-fpage">1721</span>–<span class="cit-lpage">1729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.210641.116%26rft_id%253Dinfo%253Apmid%252F27852649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi8xNzIxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTcvMjAxOS4xMi4xNi44NzU4NzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Butt</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Real-time, MinION-based, amplicon sequencing for lineage typing of infectious bronchitis virus from upper respiratory samples</span>. <abbr class="cit-jnl-abbrev">J Vet Diagn Invest</abbr> <span class="cit-pub-date">2019</span>. Accepted.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.26" data-doi="10.3390/v9060128"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeşilbağ</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Variability and Global Distribution of Subgenotypes of Bovine Viral Diarrhea Virus</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D9%26rft.spage%253D128%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv9060128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v9060128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Piewbang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genetic and evolutionary analysis of a new Asia-4 lineage and naturally recombinant canine distemper virus strains from Thailand</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">3198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D3198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown-Joseph</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification and characterization of epizootic hemorrhagic disease virus serotype 6 in cattle co-infected with bluetongue virus in Trinidad, West Indies</span>. <abbr class="cit-jnl-abbrev">Vet Microbiol</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">229</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVet%2BMicrobiol%26rft.volume%253D229%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Valastro</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">S1 gene-based phylogeny of infectious bronchitis virus: An attempt to harmonize virus classification</span>. <abbr class="cit-jnl-abbrev">Infect Genet and Evol</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">39</span>:<span class="cit-fpage">349</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect%2BGenet%2Band%2BEvol%26rft.volume%253D39%26rft.spage%253D349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.30" data-doi="10.1371/journal.pone.0014454"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A complete analysis of HA and NA genes of influenza A viruses</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">5</span>:<span class="cit-fpage">e14454</span>–<span class="cit-lpage">e14454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DShi%26rft.auinit1%253DW.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De14454%26rft.epage%253De14454%26rft.atitle%253DA%2Bcomplete%2Banalysis%2Bof%2BHA%2Band%2BNA%2Bgenes%2Bof%2Binfluenza%2BA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0014454%26rft_id%253Dinfo%253Apmid%252F21209922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0014454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21209922&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Geographic distribution and molecular analysis of porcine reproductive and respiratory syndrome viruses circulating in swine farms in the Republic of Korea between 2013 and 2016</span>. <abbr class="cit-jnl-abbrev">BMC Vet Res</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">14</span>:<span class="cit-fpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BVet%2BRes%26rft.volume%253D14%26rft.spage%253D160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.32" data-doi="10.1002/elps.201300174."><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Error analysis of idealized nanopore sequencing</span>. <abbr class="cit-jnl-abbrev">Electrophoresis</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">34</span>:<span class="cit-fpage">2137</span>–<span class="cit-lpage">2144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.volume%253D34%26rft.spage%253D2137%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felps.201300174.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/elps.201300174.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Breitwieser</span>,  <span class="cit-name-given-names">FP</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Pavian: interactive analysis of metagenomics data or microbiome studies and pathogen identification</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.34" data-doi="10.1093/nar/22.22.4673"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">1994</span>;<span class="cit-vol">22</span>:<span class="cit-fpage">4673</span>–<span class="cit-lpage">4680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.22.4673%26rft_id%253Dinfo%253Apmid%252F7984417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/22.22.4673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7984417&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PU19900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span> <abbr class="cit-jnl-abbrev">Estimation of the number of nucleotide substitutions when there are strong transition-transversion and G+C-content biases. Mol Biol Evol</abbr> <span class="cit-pub-date">1992</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">678</span>–<span class="cit-lpage">687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEstimation%2Bof%2Bthe%2Bnumber%2Bof%2Bnucleotide%2Bsubstitutions%2Bwhen%2Bthere%2Bare%2Bstrong%2Btransition-transversion%2Band%2BG%252BC-content%2Bbiases.%2BMol%2BBiol%2BEvol%26rft.volume%253D9%26rft.spage%253D678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.36" data-doi="10.1093/molbev/msy096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">35</span>:<span class="cit-fpage">1547</span>–<span class="cit-lpage">1549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D35%26rft.spage%253D1547%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy096%26rft_id%253Dinfo%253Apmid%252F29722887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29722887&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.16.875872v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vilček</span>  <span class="cit-name-given-names">Š</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Pestiviruses isolated from pigs, cattle and sheep can be allocated into at least three genogroups using polymerase chain reaction and restriction endonuclease analysis</span>. <abbr class="cit-jnl-abbrev">Arch of Virol</abbr> <span class="cit-pub-date">1994</span>;<span class="cit-vol">136</span>:<span class="cit-fpage">309</span>–<span class="cit-lpage">323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2Bof%2BVirol%26rft.volume%253D136%26rft.spage%253D309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.16.875872v2.38" data-doi="10.1016/j.vetmic.2007.01.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martella</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genotyping canine distemper virus (CDV) by a hemi-nested multiplex PCR provides a rapid approach for investigation of CDV outbreaks</span>. <abbr class="cit-jnl-abbrev">Vet Microbiol</abbr> <span class="cit-pub-date">2007</span>;<span class="cit-vol">122</span>:<span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DMartella%26rft.auinit1%253DV.%26rft.volume%253D122%26rft.issue%253D1-2%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DGenotyping%2Bcanine%2Bdistemper%2Bvirus%2B%2528CDV%2529%2Bby%2Ba%2Bhemi-nested%2Bmultiplex%2BPCR%2Bprovides%2Ba%2Brapid%2Bapproach%2Bfor%2Binvestigation%2Bof%2BCDV%2Boutbreaks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2007.01.005%26rft_id%253Dinfo%253Apmid%252F17275219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2007.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17275219&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246546100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.16.875872v2.39" data-doi="10.7589/2013-11-302"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular Typing of Epizootic Hemorrhagic Disease Virus Serotypes by One-Step Multiplex RT-PCR</span>. <abbr class="cit-jnl-abbrev">J Wildl Dis</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">50</span>:<span class="cit-fpage">639</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BWildl%2BDis%26rft.volume%253D50%26rft.spage%253D639%26rft_id%253Dinfo%253Adoi%252F10.7589%252F2013-11-302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7589/2013-11-302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.40" data-doi="10.1128/microbiolspec.OH-0001-2012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">J Woolhouse</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">RNA Viruses: A case study of the biology of emerging infectious diseases</span>. <abbr class="cit-jnl-abbrev">Microbiol Spectr</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">1</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">:</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/microbiolspec.OH-0001-2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol%2BSpectr%26rft.volume%253D1%26rft.spage%253DOH00012012%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fmicrobiolspec.OH-0001-2012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/microbiolspec.OH-0001-2012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.41" data-doi="10.1128/JCM.00698-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ginocchio</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Current best practices for respiratory virus testing</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">49</span>:<span class="cit-fpage">S44</span>–<span class="cit-lpage">S48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DGinocchio%26rft.auinit1%253DC.%2BC.%26rft.volume%253D49%26rft.issue%253D9_Supplement%26rft.spage%253DS44%26rft.epage%253DS48%26rft.atitle%253DCurrent%2BBest%2BPractices%2Bfor%2BRespiratory%2BVirus%2BTesting%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00698-11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjE5OiI0OS85X1N1cHBsZW1lbnQvUzQ0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTcvMjAxOS4xMi4xNi44NzU4NzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wostenberg</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Evidence of two cocirculating canine distemper virus strains in mesocarnivores from northern Colorado, USA</span>. <abbr class="cit-jnl-abbrev">J. of Wildlife Diseases</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">54</span>:<span class="cit-fpage">534</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2Bof%2BWildlife%2BDiseases%26rft.volume%253D54%26rft.spage%253D534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahasan</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Complete genome sequence of Mobuck Virus isolated from a Florida white-tailed deer (<em>Odocoileus virginianus</em>)</span>. <abbr class="cit-jnl-abbrev">Microbiol Resour Announc</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">8</span>:<span class="cit-fpage">e01324</span>–<span class="cit-lpage">01318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol%2BResour%2BAnnounc%26rft.volume%253D8%26rft.spage%253De01324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.16.875872v2.44"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Kielian</span>  <span class="cit-name-given-names">M</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Mettenleiter</span>  <span class="cit-name-given-names">TC</span></span> and </li><li><span class="cit-ed"><span class="cit-name-surname">Roossinck</span>  <span class="cit-name-given-names">MJ</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Pfeffermann</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-chapter-title">Chapter Four - Morbillivirus Pathogenesis and Virus–Host Interactions</span>. <span class="cit-source">In: Advances in Virus Research</span>. eds. <span class="cit-ed"><span class="cit-name-surname">Kielian</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-ed"><span class="cit-name-surname">Mettenleiter</span>  <span class="cit-name-given-names">TC</span></span> and <span class="cit-ed"><span class="cit-name-surname">Roossinck</span>  <span class="cit-name-given-names">MJ</span></span>, Vol. <span class="cit-vol">100</span>: <span class="cit-publ-name">Academic Press</span>, <span class="cit-pub-date">2018</span>:<span class="cit-fpage">75</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIn%253A%2BAdvances%2Bin%2BVirus%2BResearch%26rft.volume%253D100%26rft.spage%253D75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.45" data-doi="10.1038/nmeth.3290"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Improved data analysis for the MinION nanopore sequencer</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">12</span>:<span class="cit-fpage">351</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D12%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3290%26rft_id%253Dinfo%253Apmid%252F25686389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25686389&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McNaughton</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Illumina and Nanopore methods for whole genome sequencing of hepatitis B virus (HBV)</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-pub-date">2019</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">7081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D7081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">MinION Analysis and Reference Consortium: Phase 2 data release and analysis of R9.0 chemistry</span>. <abbr class="cit-jnl-abbrev">F1000Res</abbr> <span class="cit-pub-date">2017</span>;<span class="cit-vol">6</span>:<span class="cit-fpage">760</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000Res%26rft.volume%253D6%26rft.spage%253D760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Magi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterization of MinION nanopore data for resequencing analyses</span>. <abbr class="cit-jnl-abbrev">Brief Bioinform</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">18</span>:<span class="cit-fpage">940</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BBioinform%26rft.volume%253D18%26rft.spage%253D940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Integrating patient and whole-genome sequencing data to provide insights into the epidemiology of seasonal influenza A (H3N2) viruses</span>. <abbr class="cit-jnl-abbrev">Microb Genom</abbr> <span class="cit-pub-date">2018</span>;<span class="cit-vol">4</span>:<span class="cit-fpage">e000137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob%2BGenom%26rft.volume%253D4%26rft.spage%253D000137e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pallen</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">High-throughput sequencing and clinical microbiology: progress, opportunities and challenges</span>. <abbr class="cit-jnl-abbrev">Curr Opin in Microbiol</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">13</span>:<span class="cit-fpage">625</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2Bin%2BMicrobiol%26rft.volume%253D13%26rft.spage%253D625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.51" data-doi="10.1371/journal.ppat.1003164"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Runckel</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification and manipulation of the molecular determinants influencing poliovirus recombination</span>. <abbr class="cit-jnl-abbrev">PLOS Pathog</abbr> <span class="cit-pub-date">2013</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">e1003164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003164%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003164%26rft_id%253Dinfo%253Apmid%252F23408891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23408891&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">52.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.16.875872v2.52" data-doi="10.1016/j.tim.2016.03.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span>. <abbr class="cit-jnl-abbrev">Trends in Microbiol</abbr> <span class="cit-pub-date">2016</span>;<span class="cit-vol">24</span>:<span class="cit-fpage">490</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BMicrobiol%26rft.volume%253D24%26rft.spage%253D490%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2016.03.003%26rft_id%253Dinfo%253Apmid%252F27012512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2016.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27012512&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F17%2F2019.12.16.875872.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.16.875872v2.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">E-S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of a natural recombination in the F and H genes of feline morbillivirus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2014</span>;<span class="cit-vol">468–470</span>:<span class="cit-fpage">524</span>–<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D468470%26rft.spage%253D524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.16.875872v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.16.875872v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/608505v3" title="FYN tyrosine kinase, a downstream target of receptor tyrosine kinases, modulates anti-glioma immune responses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 17, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.16.875872v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2019.12.16.875872v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.16.875872v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.16.875872v2" class="active" data-icon-position="" data-hide-link-title="0">Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1042888" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tkbEUdU-NC1dLAgkPlXfNTRHUxdsWWDJhxgWHyVVkKs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1042888" id="node1042888--2" data-pisa="biorxiv;2019.12.16.875872v2" data-pisa-master="biorxiv;2019.12.16.875872" data-seqnum="1042888" data-apath="/biorxiv/early/2019/12/17/2019.12.16.875872.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Kelsey T.</span> <span class="nlm-surname">Young</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Kevin K.</span> <span class="nlm-surname">Lahmers</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Holly S.</span> <span class="nlm-surname">Sellers</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">David E.</span> <span class="nlm-surname">Stallknecht</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Rebecca L.</span> <span class="nlm-surname">Poulson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jerry T.</span> <span class="nlm-surname">Saliki</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">S. Mark</span> <span class="nlm-surname">Tompkins</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Ian</span> <span class="nlm-surname">Padykula</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Chris</span> <span class="nlm-surname">Siepker</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Elizabeth W.</span> <span class="nlm-surname">Howerth</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Michelle</span> <span class="nlm-surname">Todd</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">James B.</span> <span class="nlm-surname">Stanton</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.16.875872; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.16.875872 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.16.875872v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;title=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;title=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;text=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;title=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;t=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;title=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;title=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1042888" id="node1042888--3" data-pisa="biorxiv;2019.12.16.875872v2" data-pisa-master="biorxiv;2019.12.16.875872" data-seqnum="1042888" data-apath="/biorxiv/early/2019/12/17/2019.12.16.875872.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Randomly primed, strand-switching MinION-based sequencing for the detection and characterization of cultured RNA viruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Kelsey T.</span> <span class="nlm-surname">Young</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Kevin K.</span> <span class="nlm-surname">Lahmers</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Holly S.</span> <span class="nlm-surname">Sellers</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">David E.</span> <span class="nlm-surname">Stallknecht</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Rebecca L.</span> <span class="nlm-surname">Poulson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jerry T.</span> <span class="nlm-surname">Saliki</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">S. Mark</span> <span class="nlm-surname">Tompkins</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Ian</span> <span class="nlm-surname">Padykula</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Chris</span> <span class="nlm-surname">Siepker</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Elizabeth W.</span> <span class="nlm-surname">Howerth</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Michelle</span> <span class="nlm-surname">Todd</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">James B.</span> <span class="nlm-surname">Stanton</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.16.875872; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.16.875872 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Young%2C+Kelsey+T.&amp;rft.au=Lahmers%2C+Kevin+K.&amp;rft.au=Sellers%2C+Holly+S.&amp;rft.au=Stallknecht%2C+David+E.&amp;rft.au=Poulson%2C+Rebecca+L.&amp;rft.au=Saliki%2C+Jerry+T.&amp;rft.au=Tompkins%2C+S.+Mark&amp;rft.au=Padykula%2C+Ian&amp;rft.au=Siepker%2C+Chris&amp;rft.au=Howerth%2C+Elizabeth+W.&amp;rft.au=Todd%2C+Michelle&amp;rft.au=Stanton%2C+James+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1042888/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1042888/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1042888/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1042888/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1042888/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1042888/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1042888/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1042888/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1042888/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1042888/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1042888/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1042888/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;count=horizontal&amp;via=&amp;text=Randomly%20primed%2C%20strand-switching%20MinION-based%20sequencing%20for%20the%20detection%20and%20characterization%20of%20cultured%20RNA%20viruses&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.16.875872v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.16.875872v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/pathology" class="highlight" data-icon-position="" data-hide-link-title="0">Pathology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
