<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Virus Evol</span><span class="iso-abbrev" title="iso-abbrev">Virus Evol</span><span class="publisher-id" title="publisher-id">vevolu</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Virus Evolution</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2057-1577</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7069216">7069216</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/ve/veaa011">10.1093/ve/veaa011</a></span><span class="publisher-id" title="publisher-id">veaa011</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Public health in genetic spaces: a statistical framework to optimize cluster-based outbreak detection</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chato</span><span class="given-names" tagx="given-names" title="given-names">Connor</span></span><a href="#veaa011-aff1">v1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalish</span><span class="given-names" tagx="given-names" title="given-names">Marcia L</span></span><a href="#veaa011-aff2">v2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3779-154X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">Art F Y</span></span><a href="#veaa011-aff1">v1</a><a href="#veaa011-aff3">v3</a><a href="#veaa011-aff4">v4</a><a href="#veaa011-cor1" /></span></div><span class="citation_author_institution" id="veaa011-aff1">[v1], <span class="institution" title="institution">Department of Pathology and Laboratory Medicine, Western University</span><span class="country" tagx="country" title="country">Canada</span></span><span class="citation_author_institution" id="veaa011-aff2">[v2], <span class="institution" title="institution">Division of Infectious Diseases, Department of Medicine, Vanderbilt University School of Medicine</span><span class="country" tagx="country" title="country">USA</span></span><span class="citation_author_institution" id="veaa011-aff3">[v3], <span class="institution" title="institution">Department of Applied Mathematics, Western University, Middlesex College MC255</span><span class="country" tagx="country" title="country">Canada</span></span><span class="citation_author_institution" id="veaa011-aff4">[v4], <span class="institution" title="institution">Department of Microbiology and Immunology, Western University</span><span class="country" tagx="country" title="country">Canada</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Corresponding author: E-mail: <span class="email" tagx="email" title="email">apoon42@uwo.ca</span></div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-1</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-3-3</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-3-3</span></span><span class="volume" tagx="volume" title="volume">6</span><span class="issue" tagx="issue" title="issue">1</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">veaa011</span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="veaa011.pdf" title="self-uri">veaa011.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Genetic clustering is a popular method for characterizing variation in transmission rates for rapidly evolving viruses, and could potentially be used to detect outbreaks in ‘near real time’. However, the statistical properties of clustering are poorly understood in this context, and there are no objective guidelines for setting clustering criteria. Here, we develop a new statistical framework to optimize a genetic clustering method based on the ability to forecast new cases. We analysed the pairwise Tamura-Nei (TN93) genetic distances for anonymized HIV-1 subtype B <i>pol</i> sequences from Seattle (<i>n </i>=<i> </i>1,653) and Middle Tennessee, USA (<i>n </i>=<i> </i>2,779), and northern Alberta, Canada (<i>n </i>=<i> </i>809). Under varying TN93 thresholds, we fit two models to the distributions of new cases relative to clusters of known cases: 1, a null model that assumes cluster growth is strictly proportional to cluster size, i.e. no variation in transmission rates among individuals; and 2, a weighted model that incorporates individual-level covariates, such as recency of diagnosis. The optimal threshold maximizes the difference in information loss between models, where covariates are used most effectively. Optimal TN93 thresholds varied substantially between data sets, e.g. 0.0104 in Alberta and 0.016 in Seattle and Tennessee, such that the optimum for one population would potentially misdirect prevention efforts in another. For a given population, the range of thresholds where the weighted model conferred greater predictive accuracy tended to be narrow (±0.005 units), and the optimal threshold tended to be stable over time. Our framework also indicated that variation in the recency of HIV diagnosis among clusters was significantly more predictive of new cases than sample collection dates (ΔAIC &amp;gt; 50). These results suggest that one cannot rely on historical precedence or convention to configure genetic clustering methods for public health applications, especially when translating methods between settings of low-level and generalized epidemics. Our framework not only enables investigators to calibrate a clustering method to a specific public health setting, but also provides a variable selection procedure to evaluate different predictive models of cluster growth.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
molecular epidemiology</div> <div class="kwd" title="kwd">
genetic clustering</div> <div class="kwd" title="kwd">
modifiable areal unit problem</div> <div class="kwd" title="kwd">
virus evolution</div> <div class="kwd" title="kwd">
HIV prevention</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Project Grant from the Canadian Institutes of Health Research</span></div> </div><span class="award-id" tagx="award-id" title="award-id">PJT-156178</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Administrative Supplement to a Center Core Grant from the Tennessee Center for AIDS Research</span></div> </div><span class="award-id" tagx="award-id" title="award-id">P30-AI110527</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution" title="institution">Dr Frederick Winnett Luney Graduate Research Award from the Department of Pathology at Western University, Canada</span></div> </div> </div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="1.background" title="sec"> <div class="title" tagx="title" title="title">
1. Background</div> <p>Spatiotemporal clustering is a fundamental public health methodology for the detection of infectious disease outbreaks (<a href="#veaa011-B36">Robertson et al. 2010</a>). The colocalization of cases in space and time can reveal the existence of a common source, and cases within a cluster tend to be related by recent transmission events. For example, an automated time–space clustering method (<a href="#veaa011-B19">Kulldorff et al. 2005</a>) was demonstrated to retrospectively detect outbreaks of nosocomial bacterial infection in a US-based hospital, including the outbreaks that were detected contemporaneously by the hospital’s pre-existing infection control programme (<a href="#veaa011-B15">Huang et al. 2010</a>). At a broader spatial scale, the same clustering method was recently used to identify outbreaks of severe acute respiratory infections over a five-year period, using case data from a network of hospitals in Uganda (<a href="#veaa011-B6">Cummings et al. 2019</a>). Early detection of a cluster represents a potential opportunity for a targeted public health response to prevent additional cases. Time–space clustering may be less effective, however, for pathogens that can establish a chronic infection with a long asymptomatic period (e.g. <i>Mycobacterium tuberculosis</i>, hepatitis C virus, or human immunodeficiency virus type 1; HIV-1) where the transmission event may have occurred months or years before diagnosis. Furthermore, pathogens with a relatively low per-act transmission rate present difficulties for time–space clustering because a single exposure in a specific location is unlikely to result in transmission. Under these circumstances, the spread of an epidemic is more likely to be shaped by a social network of repeated contacts between individuals, rather than shared venues.</p> <p>For many infectious diseases, the molecular evolution of the pathogen is sufficiently rapid that genetic differences can accumulate between related infections on a similar time scale as disease transmission. Consequently, it can be effective to cluster cases in a high-dimensional <i>genetic space</i> in addition to clustering in physical space and time. In these studies, a case of infection is represented by a pathogen-derived molecular sequence that maps to some point in genetic space, and it may be associated with subject-level metadata such as the diagnosis date or treatment history. Clustering infections by their evolutionary relatedness is a popular method to identify and characterize subgroups with potentially elevated transmission rates. For example, pairs of sequences can be clustered if the number of genetic differences between them falls below some threshold. The resulting clusters are often visualized as a network or undirected graph, where each node (vertex) represents an individual case of infection, and each edge connecting vertices indicate that the sequences of the corresponding cases are within a threshold genetic distance of each other. Sampling a group of cases that are nearly genetically identical implies that they are related through an unknown number of recent and rapid transmission events. A substantial number of genetic clustering studies have focused on the molecular epidemiology of HIV-1 (<a href="#veaa011-B30">Poon 2016</a>; <a href="#veaa011-B37">Rose et al. 2017</a><a href="#veaa011-B34">Ragonnet-Cronin et al. 2018</a>; <a href="#veaa011-B3">Billock et al. 2019</a>). Under current global treatment and prevention guidelines (<a href="#veaa011-B20">Levi et al. 2016</a>), greater proportions of HIV cases are being diagnosed, and new diagnoses are more frequently screened for drug resistance by genetic sequencing prior to initiating antiretroviral treatment. As a result, public health organizations are beginning to use genetic clustering methods in ‘near real-time’ to identify ongoing HIV-1 outbreaks (<a href="#veaa011-B32">Poon et al. 2016</a>; <a href="#veaa011-B11">Gonsalves and Crawford 2018</a>), to reconstruct the risk factors and aetiology (<a href="#veaa011-B9">Dennis et al. 2012</a>; <a href="#veaa011-B31">Poon et al. 2015</a>), and to prioritize groups for prevention initiatives such as access to pre-exposure prophylaxis (PrEP; <a href="#veaa011-B41">Volz et al. 2018</a>).</p> <p>A fundamental challenge in the use of genetic clustering to identify potential outbreaks is that these methods usually require the specification of one or more clustering criteria (<a href="#veaa011-B30">Poon 2016</a>; <a href="#veaa011-B13">Hassan <i>et al.</i> 2017</a>). Many HIV-1 studies that employ pairwise clustering use similar genetic distance thresholds, where a genetic distance is a measure that adjusts for the possibility of multiple substitutions at the same nucleotide. For instance, a recent review (<a href="#veaa011-B13">Hassan <i>et al.</i> 2017</a>) of 105 articles defining HIV-1 clusters found that 1.5 per cent was the most commonly used threshold among 52 studies that employed a genetic distance. In contrast, the United States Centers for Disease Control and Prevention (US-CDC) currently recommends a stricter pairwise distance threshold of 0.5 per cent (National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention 2018) for the purpose of identifying clusters with the most recent and rapid growth. In some cases the selected threshold is informed by the expected divergence between HIV-1 sequences sampled longitudinally from the same patient (<a href="#veaa011-B31">Poon <i>et al.</i> 2015</a>; <a href="#veaa011-B44">Wertheim <i>et al.</i> 2017</a>)—however, this empirical distribution can vary substantially among subjects (<a href="#veaa011-B35">Ratmann <i>et al.</i> 2016</a>) and may be influenced by the extent of clinical follow-up. In other cases, the threshold is based on the expected number of substitutions between two epidemiologically unrelated individuals in the same sample space (∼5% for the USA; <a href="#veaa011-B2">Aldous <i>et al.</i> 2012</a>; <a href="#veaa011-B27">Oster <i>et al.</i> 2018</a>), which makes the optimal threshold sensitive to regional variation in HIV prevalence and population density. On the other hand, population studies in Botswana (<a href="#veaa011-B24">Novitsky <i>et al.</i> 2014</a>), South Africa (<a href="#veaa011-B8">De Oliveira <i>et al.</i> 2017</a>), and South America (<a href="#veaa011-B17">Junqueira <i>et al.</i> 2016</a>) have used substantially higher distance thresholds (10%, 4.5%, and 4.5%, respectively) that imply the optimal thresholds may vary substantially among settings and HIV-1 subtypes (<a href="#veaa011-B14">Hemelaar 2013</a>). Furthermore, simulation-based studies (<a href="#veaa011-B24">Novitsky <i>et al.</i> 2014</a>; <a href="#veaa011-B30">Poon 2016</a>; <a href="#veaa011-B7">Dasgupta <i>et al.</i> 2019</a>) have demonstrated that clustering is highly sensitive to the sampled proportion of the infected population. Given the known differences in the empirical distributions of HIV-1 genetic distances among populations, as well as the significant global disparities in prevalence and access to testing and treatment, it is urgently necessary to establish an objective, quantitative framework for optimizing a clustering method to the target population.</p> <p>Here we propose that the most useful approach to select clustering criteria is to base this decision on our ability to predict where the next cases will occur. A high, permissive clustering threshold tends to result in a single cluster that comprises the majority of known cases. The next cases are proportionately more likely to connect to this cluster simply because it is large, but its size will also average out the individual- and group-level attributes that are informative for predicting the next cases. Put another way, a single large cluster is not likely to confer a public health benefit because it is akin to prioritizing the entire population. Conversely, setting a low, strict clustering threshold results in a large number of small clusters. This increases the variation of attributes among clusters, resolving greater information. As cluster sizes continue to decline with progressively lower thresholds, however, the variation in attributes among clusters is less associated with the emergence of new cases—in other words, the distribution of new cases among clusters becomes increasingly random. This trade-off is analogous to the modifiable areal unit problem (MAUP), a concept in spatial statistics first fully conceptualized by <a href="#veaa011-B25">Openshaw and Taylor (1979)</a>. Areal units are derived from a partition of a geographic range by drawing boundaries that separate households or neighbourhoods. The MAUP formally recognizes the inconsistency of statistical associations with changing boundaries. For example, aggregating units into larger spatial units, such as cities or countries, can prevent an investigator from detecting a strong association between water quality and gastrointestinal illness (<a href="#veaa011-B38">Swift, Liu, and Uber 2008</a>).</p> <p>To address the MAUP in the context of genetic clustering and public health, we develop an information criterion-based framework inspired by work from <a href="#veaa011-B22">Nakaya (2000)</a>. The objective of our framework is to identify the clustering criteria that maximizes the information content of the resulting clusters for forecasting where the next cases will occur. We evaluate our approach on anonymized HIV-1 sequence data from three populations, and demonstrate how this framework can also be used to select between predictive models of cluster growth that utilize different cluster attributes. Furthermore, we examine the problems associated with the selection of clustering criteria or the application of criteria from one population to another, and evaluate the stability of information-optimized criteria for a given population over time.</p> </div> <div class="2.methods" title="sec"> <div class="title" tagx="title" title="title">
2. Methods</div> <div class="2.1datacollectionandprocessing" title="sec"> <div class="title" tagx="title" title="title">
2.1 Data collection and processing</div> <p>From the public GenBank database (<a href="https://www.ncbi.nlm.nih.gov/genbank">https://www.ncbi.nlm.nih.gov/genbank</a>, last accessed 2 April 2019), we obtained <i>n </i>=<i> </i>809 anonymized HIV-1 <i>pol</i> sequences sampled from northern Alberta, Canada, between 2007 and 2013 (<a href="#veaa011-B42">Vrancken <i>et al.</i> 2017</a>), and <i>n </i>=<i> </i>1,653 sequences collected from Seattle, USA, between 2000 and 2013 (<a href="#veaa011-B45">Wolf <i>et al.</i> 2017</a>). In addition, <i>n </i>=<i> </i>2,779 HIV-1 <i>pol</i> sequences predominantly from the middle Tennessee region of the USA were collected by the Vanderbilt Comprehensive Care Clinic in Nashville between 2001 and 2015. These data were linked to patient records to extract limited data (e.g. year of HIV-1 diagnosis) and anonymized before being made available for this study; further details are available in <a href="#veaa011-B10">Dennis <i>et al.</i> (2018)</a>. All sequence data were annotated with years of sample collection. We used the pre-existing HIV subtype annotations from the sequence records to filter each data set for non-B subtypes, and excluded repeated samples from the same individual. Next, we filtered each data set to remove any sequences with a proportion of ambiguous nucleotides above 5 per cent, which affected 1 sequence from each of the northern Alberta and Seattle data sets, and 163 sequences from Tennessee. Given the relatively small number of sequences collected in part of 2013 for the Seattle data set (<i>n </i>=<i> </i>35, <a href="#veaa011-F1">Fig. 1</a>), we excluded this year to maintain a consistent sampling rate. We retrieved the sample collection dates for Seattle and North Alberta by querying GenBank with the respective accession numbers and extracting this information from the XML stream returned from the server using the BioPython module (<a href="#veaa011-B5">Cock <i>et al.</i> 2009</a>) in Python. Next, we used an open-source implementation of the <a href="#veaa011-B39">Tamura and Nei (1993)</a> genetic distance in C++ (TN93 version 1.0.6, <a href="https://github.com/veg/tn93">https://github.com/veg/tn93</a>, last accessed 5 September 2018) for each data set to compute these distances between all pairs of sequences. All other options for the TN93 analyses were set to the default values.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>(top) Distribution of sample collection years for the Seattle (blue), northern Alberta (orange), and Middle Tennessee (red) data sets. Absent bars indicate that no sampling was carried out in the respective years, and does not reflect an absence of cases. (bottom) Histograms of Tamura-Nei (TN93) genetic distances among pairwise comparisons of HIV-1 sequences. The height of each bin has been rescaled to reflect the total number of pairwise comparisons, for which the majority were censored from the data. An expanded section of the barplots in the range (0, 0.03) is provided as a figure inset to clarify differences among the distributions.</p>   </div> </div> <div class="2.2definingclusters" title="sec"> <div class="title" tagx="title" title="title">
2.2 Defining clusters</div> <p>Using a custom script in the R programming language, we generated an undirected graph <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
E</div> <div class="mo_UNKNOWN">
)</div> </div></div></span> from the TN93 output of each data set, where the set of vertices <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
V</div></div></span> represents individual cases (assuming one sequence per case) connected by edges in the set <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
E</div></div></span>. Every edge between vertices <i>v</i> and <i>u</i>, denoted <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
E</div> </div></div></span>, was weighted with the TN93 distance between the respective sequences, which we denote by the edge attribute <i>d</i>(<i>v</i>, <i>u</i>). In addition, each vertex <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
V</div> </div></div></span> carries a temporal attribute <i>t</i>(<i>v</i>), which may represent the year of diagnosis or sample collection. Note that we are not limited to analysing dates at the level of years and can utilize more precise time intervals, e.g. quarters or months, given the availability of these data. For a given clustering threshold <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
max</div> </div> </div> </div></div></span>, we obtained a spanning subgraph <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
)</div> </div></div></span> from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
G</div></div></span> that results from filtering the complete list of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
/</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> edge weights, such that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
{</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
E</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
≤</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div></div></span>. The subset of sequences with the most recent collection time-point was specified as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
{</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div></div></span>, such that the total number of new cases is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
|</div> </div></div></span>. In other words, sampling time cuts <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
V</div></div></span> into disjoint vertex sets <i>V</i> and <i>V</i><sup>c</sup>, where <i>V</i><sup>c</sup> is the complement of <i>V</i> (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
∪</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
V</div> </div></div></span>) and contains all known cases over time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span>. Later it will be useful to refer to the subset of edges in <i>E</i><sub>d</sub> that connect a vertex in <i>V</i><sup>c</sup> and a vertex in <i>V</i>, which we denote as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
{</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
}</div> </div></div></span>. The set of edges in <i>E</i> can also be interpreted as edges in a bipartite subgraph comprising parts <i>V</i><sup>c</sup> and <i>V</i>.</p> <p>A clustering method defines a partition on the known cases <i>V</i><sup>c</sup> into a set of clusters <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
…</div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
}</div> </div></div></span> such that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
∩</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
∅</div> </div></div></span> for all <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
≠</div> <div class="mi_UNKNOWN">
j</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
≤</div> <div class="mi_UNKNOWN">
n</div> </div></div></span>; and such that the union of all clusters recovers the entire set: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∪</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span>. Note that this definition does not strictly require the existence of edges, which we use to represent genetic similarity, but can be adapted to any method that defines a partition on the database of known cases. For our analysis, clusters were defined as the connected components of <i>V</i><sup>c</sup>, meaning any pair of vertices within the same cluster (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span>) are connected by at least one path (a sequence of edges), any pair of vertices in different clusters are not connected by any path, and single cases can count as their own cluster of size one.</p> </div> <div class="2.3modellinggrowth" title="sec"> <div class="title" tagx="title" title="title">
2.3 Modelling growth</div> <p>We define total cluster growth <i>R</i> as the number of new cases in <i>V</i> adjacent (connected by an edge) to any known case in <i>V</i><sup>c</sup>, where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
≤</div> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
|</div> </div></div></span>. To resolve the event that a new vertex in <i>V</i> is adjacent to vertices in more than one cluster, we reduced the subset of edges between <i>V</i> and <i>V</i><sup>c</sup>, maintaining only the edges with minimum weight per vertex in <i>V</i>. If more than one edge to a given vertex <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
V</div> </div></div></span> had exactly the same minimum weight, then we selected one edge at random.</p> <p>We formulated two predictive models to generate estimates of growth for the <i>i</i>th cluster <i>C<sub>i</sub></i>, which we denote respectively as the null model and the weighted model. The null model requires less information by postulating that each cluster is expected to grow in proportion to its current size, prior to the addition of new cases, as a fraction of the entire population of known cases. For example, a cluster that comprises half of all known cases is predicted to accumulate half of new cases that are adjacent to any cluster. Expressed as a Poisson regression model, the expected growth of <i>C<sub>i</sub></i> given total cluster growth <i>R</i> is given by: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mtext_UNKNOWN">
exp</div> <div class="mo_UNKNOWN">
 </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
|</div> </div> </div> <div class="mi_UNKNOWN">
R</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></div>where we use boldface <b>E</b> to denote the expectation (and distinguish it from our edge set notation <i>E</i>), and <i>R</i><sub>0</sub> is given the subscript 0 to indicate it is the predicted growth under the null model. Thus the null model does not use any individual-level attributes to predict cluster growth—it is a naive model that assumes that the allocation of cluster-adjacent new cases in <i>R</i> is not influenced by any characteristics of those clusters other than the ‘space’ they occupy. In contrast, the weighted model assigns individual-level weights <i>w</i>(<i>v</i>) to every vertex in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span>. Expressed as a Poisson regression model, the expected growth of cluster <i>C<sub>i</sub></i> under the weighted model is written: <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mtext_UNKNOWN">
exp</div> <div class="mo_UNKNOWN">
 </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
+</div> <div class="mo_UNKNOWN">
β</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div>where <i>α</i> and <i>β</i> are parameters to be estimated by regression. Note that <a href="#E2">Equation (2)</a> reduces to (1) when <i>w</i>(<i>v</i>) = 1 for all <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span>, <i>α</i>  =  0 and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
/</div> <div class="mo_UNKNOWN">
|</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
|</div> </div></div></span>. There are two advantages to using individual-level weights in <a href="#E2">Equation (2)</a> rather than cluster-level weights: first, individual-level weights are independent of clustering thresholds, so it is sufficient to calculate weights one time only; second, individual-level weights confer a greater degree of precision for measuring effects on edge densities that would become averaged out in clusters.<p /> <p>For our demonstration, we weighted individual cases by their recency of sample collection or diagnosis, measured as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> </div></div></span>. The predictive weight <i>w</i> of a known case of a given age <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> relative to <i>t</i><sub>max</sub> was based on the expected rate of adjacency (edge density) between sets of known cases separated by the same time lag. Thus, we needed to calculate the edge densities for all bipartite graphs <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
(</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
j</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div></div></span> where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
{</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
}</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>. For compatibility with our definition of cluster growth, we removed bipartite edges from <i>E<sub>ij</sub></i> so that the maximum degree size for any vertex <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
j</div> <div class="mi_UNKNOWN">
c</div> </div> </div></div></span> was 1, where the remaining edge minimized the edge weight <i>w</i>(<i>u</i>, <i>v</i>) for all <i>u</i> given <i>v</i>. We use <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> to denote this reduced set of bipartite edges. This had the effect of reducing the maximum possible number of bipartite edges in <i>K<sub>ij</sub></i> from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
|</div> <div class="mo_UNKNOWN">
|</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
|</div> </div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
|</div> </div></div></span>. We refer to the set of all bipartite graphs for a given time lag as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
{</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
}</div> </div></div></span>. Thus, the expected edge density <i>�?</i> given <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> is: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
�?</div> <div class="mo_UNKNOWN">
|</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
K</div> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
|</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
|</div> <div class="mo_UNKNOWN">
/</div> <div class="mo_UNKNOWN">
|</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
V</div> </div> <div class="mi_UNKNOWN">
j</div> <div class="mi_UNKNOWN">
c</div> </div> <div class="mo_UNKNOWN">
|</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
min</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div></div></div>where the denominator adjusts for the number of bipartite graphs with time lag <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>. For this model, we assume that the edges in <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
E</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span> are independent and identically distributed binary outcomes. Furthermore, we expect the probability of this outcome to decay with increasing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>. Hence we used logistic regression to estimate <i>�?</i> as a function of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>: <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
log</div> <div class="mo_UNKNOWN">
 </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> <div class="mo_UNKNOWN">
^</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
′</div> <div class="mo_UNKNOWN">
+</div> <div class="mo_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
′</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
′</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
′</div> </div></div></span> are parameters to be estimated by regression. Using logistic regression enabled us to measure the effect of case recency at the level of individuals. The simplest use of this information would be to set the weight of a known case to its predicted edge density given its time lag <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> relative to the most recent year; i.e. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div></div></span>. To summarize, we identified edges that minimized the genetic distance between known cases separated by a given time lag (<a href="#E3">Equation 3</a>) and fit a logistic model to these edge densities as a function of time lag (<a href="#E4">Equation 4</a>) so we can predict the adjacency of new cases to a cluster of known cases given their net recency.<p /> <p>The weighted model can be extended to employ a linear combination of additional individual-level attributes (e.g. plasma viral load) and/or graph attributes that are parameterized from bipartite subgraphs on <i>V</i><sup>c</sup>. For example, we added an additional measure, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
deg</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> </div></div></span>, that represents the average degree of known cases from the same time point as <i>v</i>. A high mean degree in the graph for a given threshold <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> can reveal specific time points with an unusually dense concentration of cases in genetic space, which may be caused by a period of increased sampling effort or a past outbreak. Without making some adjustment, this period can have a disproportionate influence on the association of edge density on <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> as estimated by <a href="#E4">Equation (4)</a>. Thus we also evaluated a weighted model substituting the following weighting formula into <a href="#E2">Equation (2)</a>: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
β</div> </div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
β</div> </div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
deg</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> </div></div></div>where the coefficient <i>β</i> has been brought into the summation over individual known cases. In this model, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
deg</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
)</div> </div></div></span> is a graph-level attribute that controls for the confounding effect of variation in degree size among years of diagnosis or collection. We have released R scripts and examples for calculating generalized Akaike information criterion (GAIC) profiles under a permissive free software license at <a href="https://github.com/PoonLab/MountainPlot">https://github.com/PoonLab/MountainPlot</a> (last accessed 20 January 2020).<p /> </div> <div class="2.4evaluatingclusterthresholds" title="sec"> <div class="title" tagx="title" title="title">
2.4 Evaluating cluster thresholds</div> <p>For each data set, we segregated all HIV-1 sequences that were sampled in the most recent year as new cases comprising the set <i>V</i>. Next, we extracted the observed cluster growth outcomes <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> </div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div></div></span> and individual case weights <i>w</i>(<i>v</i>) at fifty-one different cluster-defining distance thresholds, ranging from <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.04</div> </div></div></span> in steps of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
8</div> <div class="mo_UNKNOWN">
×</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
4</div> </div> </div> </div></div></span>. To clarify, we used the same threshold <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> to evaluate cluster growth (the occurrence of new cases in clusters) as was used to generate the clusters of known cases. We fit the null and weighted models described by <a href="#E1">Equations (1)</a> and <a href="#E2">(2)</a> to the resulting distributions of cluster growth. The Akaike information criterion (AIC), which penalizes likelihood for the number of model parameters, was recorded for each regression model (<a href="#veaa011-B1">Akaike 1998</a>). The ‘generalized AIC’ (GAIC) is simply the difference in AIC between models, and has been proposed as a key quantity for resolving the MAUP (<a href="#veaa011-B22">Nakaya 2000</a>). Cut-offs with a negative GAIC indicate that the weighted model explains the data more effectively than the null model, and the magnitude of GAIC quantifies that difference in effectiveness. We define the optimal distance threshold as the value <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> associated with the lowest (most negative) GAIC. The GAIC was evaluated for the weighted models using either dates of sample collection or diagnosis to compute <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> for the Middle Tennessee data set for which both dates were available. Since some dates were missing data, we normalized the number of new cases to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
|</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
125</div> </div></div></span> for both analyses to ensure that growth rates were comparable. After this step, the total number of cases was reduced to <i>n </i>=<i> </i>2,015 and <i>n </i>=<i> </i>2,588 for the diagnostic and sample collection analyses, respectively. We note that the resulting weighted models are not being compared directly; instead, they are compared to the null models for their respective data sets.</p> <p>We repeated the cluster threshold evaluation on progressively censored subsets of the Seattle and Tennessee data to evaluate the consistency of the GAIC-optimized thresholds over time. This was accomplished by removing cases from the most recent year to a maximum of four years, and obtaining the GAIC measurements for all values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> for the remaining data at each step. Because of the limited size and temporal range of the northern Alberta data set, we did not use it for this sensitivity analysis. We also re-ran the experiment on random subsets of cases from the complete Seattle and Tennessee data sets, creating a total of ninety subsets; thirty subsets at three different proportions of the original total, 80 per cent, 60 per cent, and 40 per cet. For each resampling proportion, we obtained the kernel density for the optimal cut-off location over the thirty replicate samples and the GAIC measurements obtained by a smoothed function from all ninety resampled GAIC runs.</p> </div> </div> <div class="3.results" title="sec"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1studypopulations" title="sec"> <div class="title" tagx="title" title="title">
3.1 Study populations</div> <p>A total of <i>n </i>=<i> </i>5,010 HIV-1 sequences and sample collection dates were obtained from published studies in Seattle (<i>n </i>=<i> </i>1,591; <a href="#veaa011-B45">Wolf <i>et al.</i> 2017</a>), northern Alberta (<i>n </i>=<i> </i>803; <a href="#veaa011-B42">Vrancken <i>et al.</i> 2017</a>), and Middle Tennessee (<i>n </i>=<i> </i>2,616; <a href="#veaa011-B10">Dennis <i>et al.</i> 2018</a>), respectively. In addition, dates of HIV-1 diagnosis were available for a total of 2,527 cases in the Tennessee data set. The distributions of sample collection dates are summarized in <a href="#veaa011-F1">Fig. 1</a> and the direct comparison between diagnostic and collection year distributions for the Tennessee data set can be found in <a href="#sup1">Supplementary Fig. S1</a>. The lowest mean sampling rate was obtained in northern Alberta, with 114.7 cases per year, compared to 122.4 and 174.4 cases/year for Seattle and Tennessee, respectively. Cases from the final years of sampling were omitted to adjust for studies being terminated before the end of the calendar year. For instance, there were only thirty-five cases sampled in 2013 in the Seattle data set, where sample collection was ended on March 2013 (<a href="#veaa011-B45">Wolf <i>et al.</i> 2017</a>)—since the cases in the final year were reserved to evaluate the predictive models, an artificially low sample size would have a disproportionate influence on model validation. Hence, we proceeded with 110 cases collected in 2012, 110 cases from 2013, and 153 cases from 2015 for Seattle, northern Alberta, and Tennessee, respectively. We refer to the cases sampled in these final years as ‘new cases’, and those sampled in the preceding years as ‘known cases’.</p> <p>For each data set, we calculated the TN93 (<a href="#veaa011-B39">Tamura and Nei 1993</a>) genetic distance for every pair of sequences. This distance, which adjusts for differences in the mean rates among nucleotide transversions and the two types of transitions, is the basis for clustering in the HIV-TRACE programme (<a href="#veaa011-B29">Pond <i>et al.</i> 2018</a>) that is employed by the US-CDC for public health surveillance (<a href="#veaa011-B27">Oster <i>et al.</i> 2018</a>). Although the northern Alberta data set comprised a smaller number of sequences, the lower tail of its TN93 distribution contained relatively higher numbers of pairwise distances than the other two data sets (<a href="#veaa011-F1">Fig. 1</a>). For instance, the TN93 distance at the 1 per cent quantile was 0.013 in northern Alberta, while the same quantile was roughly twice this distance for Seattle (0.026) and Tennessee (0.023). Overall, these distributions were significantly different (Kruskal–Wallis test, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
10</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
15</div> </div> </div> </div></div></span>).</p> </div> <div class="3.2adjacencyofcasesdecayswithtimelag" title="sec"> <div class="title" tagx="title" title="title">
3.2 Adjacency of cases decays with time lag</div> <p>We generated a sequence of graphs at varying TN93 distance thresholds for each data set, where each vertex represents a known case (sampled or diagnosed prior to the final year) and an edge indicates that the corresponding pairwise distance is below the threshold—in graph theory, the cases are said to be ‘adjacent’. Thus, each distance threshold defines a different partition of known cases into clusters, where a cluster may consist of only a single known case. Our objective is to determine which threshold results in the most information-rich partition of known cases for predicting where new cases will arise. As we will demonstrate below, there is no information value in either extreme of a single giant cluster or the complete atomization of cases into singular clusters. To quantify the information loss associated with different partitions, we compared two predictive models. First, we fit a null model that assumes the probability that a new case appears in a cluster (i.e. cluster growth) is only influenced by the number of known cases in the cluster, i.e. the cluster size. This is equivalent to assuming that every known case is equally likely to be adjacent (connected by an edge) to the new case. Second, we fit a weighted model where the probability of cluster growth is predicted by some linear combination of individual-level attributes among the known cases in the cluster.</p> <p>For example, we hypothesize that the probability that a new case is adjacent to a known case declines with an increasing time lag between their respective sample collection dates. To investigate this effect, we plotted the observed densities of edges at a threshold <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.04</div> </div></div></span> between sets of known cases sampled in different years (<a href="#veaa011-F2">Fig. 2</a>). These plots confirm that edge densities decline significantly with increasing time lag (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>), which we measured by fitting binomial regression models (<a href="#E4">Equation 4</a>). Specifically, the estimated effect of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> on the log-odds of a bipartite edge was –0.42 (95% CI <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.45</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.39</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN"> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
year</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div></div></span> for the Seattle data and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.40</div> <div class="mo_UNKNOWN"> </div> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.48</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.32</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN"> </div> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
year</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div></div></span> for northern Alberta. The coefficient of determination for the respective models was <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.70</div> </div></div></span> and 0.58. For the Tennessee data set, the effect of time lag was lower than the other data sets (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.19</div> <div class="mo_UNKNOWN"> </div> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.21</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.18</div> <div class="mo_UNKNOWN">
)</div> </div></div></span> year<sup>–</sup><sup>1</sup>; <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.41</div> </div></div></span>). In general, lowering <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> reduced the observed bipartite edge densities as fewer edge weights passed the threshold. Nevertheless, the negative associations between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> and the log-odds of bipartite edges were robust to varying <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> (<a href="#sup1">Supplementary Fig. S2</a>). These results supported the use of ‘case recency’ (the time lag between sample collection dates) as an individual-level predictor of a new case joining a cluster of known cases.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Decay in bipartite edge densities with increasing time lags. Each point represents a bipartite graph at a given time lag (difference in years, <i>x</i>-axis), coloured with respect to the data set (see inset legend). We added random noise along the horizontal axis to separate overlapping points. The log-transformed <i>y</i>-axis represents the frequency of edges between cases in different time points below the threshold <i>d</i><sub>max</sub> = 0.04 (<a href="#E3">Equation 3</a>). (left) Decay of edge densities with respect to dates of sample collection. Each trend line summarizes the binomial regression model (<a href="#E4">Equation 4</a>) for each data set. (right) Decay of edge densities with respect to dates of HIV diagnosis in Middle Tennessee. Each point was scaled in proportion to the mean degree size among known cases in the earlier time point. Bipartite graphs without any edges are represented by × marks at the arbitrary value of 0.005, since zero cannot be displayed on a log-transformed axis. The dashed line indicates the fit of the binomial regression to these data, while the solid line indicates the fit of an extended model (<a href="#E5">Equation 5</a>) that controls for variation in the mean degree size.</p>   </div> <p>Since dates of HIV-1 diagnosis were also available for the Tennessee data set, we applied the same binomial regression to known cases separated by their years of HIV diagnosis rather than by sample collection (<a href="#veaa011-F2">Fig. 2</a>, right). We noted that the decline in adjacency rates with time lag was distorted by unusually high edge densities involving cases diagnosed in 1982, 1984 and 1986, which resulted in a much smaller but significant effect of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
0.067</div> <div class="mo_UNKNOWN"> </div> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
95</div> <div class="mo_UNKNOWN">
%</div> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
.</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
.</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
0.074</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
0.060</div> <div class="mo_UNKNOWN">
)</div> </div></div></span> year<sup>–</sup><sup>1</sup>; <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.016</div> </div></div></span>). This motivated the use of an extended binomial regression model (<a href="#E5">Equation 5</a>) to control for variation in degree size among years of diagnosis for known cases. We found that controlling for variation in degree sizes among years substantially improved the fit of the regression model to the time lags in diagnosis dates (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> </div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.37</div> <div class="mo_UNKNOWN">
,</div> <div class="mo_UNKNOWN">
 </div> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AIC</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
351.4</div> </div></div></span>). Moreover, this extended model conferred significantly improved fits to sample collection dates for all three data sets (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mi_UNKNOWN">
AIC</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
7.5</div> </div></div></span>, –12.8 and –171.6 for Seattle, northern Alberta and Middle Tennessee, respectively). Therefore, we used the extended binomial regression model for our subsequent analyses to predict the distribution of new cases among clusters.</p> </div> <div class="3.3trade-offbetweencasecoverageandclusterinformation" title="sec"> <div class="title" tagx="title" title="title">
3.3 Trade-off between case coverage and cluster information</div> <p><a href="#veaa011-F3">Figure 3</a> illustrates the effect of relaxing the threshold <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> on the number of new cases that are adjacent to one or more known cases, which we denote by <i>R</i>. A new case that is not adjacent to any known case cannot be anticipated by any clustering method. Although these results do not inform us about our ability to forecast cluster growth, i.e. which cluster of known cases is more likely to accumulate new cases, they do characterize the effect size of the TN93 distance cut-off for perceived cluster growth. When <i>R</i> approaches the total number of new cases (denoted as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
|</div> <div class="mi_UNKNOWN">
V</div> <div class="mo_UNKNOWN">
|</div> </div></div></span>, where <i>V</i> is the set of vertices in the final year) we say that the clusters have a high <i>case coverage</i>. As expected, decreasing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> reduced <i>R</i> as a progressively greater number of edges were excluded, which would limit the utility of clustering for public health surveillance. The accumulation of <i>R</i> with increasing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> was slightly slower for Seattle than the other data sets (<a href="#veaa011-F3">Fig. 3</a>, left), which was not anticipated by our comparisons of the overall distributions of TN93 distances among data sets (<a href="#veaa011-F1">Fig. 1</a>, bottom). In addition, <a href="#veaa011-F3">Fig. 3</a> (left) summarizes how the number of clusters with at least one edge to a new case (the number of active clusters) initially tracks the accumulation in <i>R</i> with increasing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span>. Thus <i>R</i> sets an upper limit to the number of active clusters; these numbers can only be equal if each new case is uniquely adjacent to its own cluster, as we observed for the Seattle data set for cut-offs below <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.0072</div> </div></div></span> or the Tennessee data at cut-offs below <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.008</div> </div></div></span>. As <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> continues to increase, the number of active clusters peaks and begins to decline towards 1. This outcome reflects the gradual accretion of cases into a single giant cluster. Substituting year of diagnosis for sample collection dates in the Tennessee data resulted in a slight reduction in <i>R</i> and a modest increase in the number of active clusters, which we attribute to a more uniform distribution of new cases across clusters.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Distribution of new cases among clusters as a function of the Tamura-Nei (TN93) distance clustering threshold (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span>, <i>x</i>-axis). The solid lines represent the total number (<i>R</i>) of new cases adjacent to clusters of known cases. The points correspond to the number of clusters of known cases with edges to new cases, which we refer to as ‘active’ clusters. (left) This plot summarizes the trends obtained when cases are stratified by year of sample collection. Trends in <i>R</i> and numbers of active clusters are coloured with respect to data set (see inset legend). (right) This plot contrasts the trends obtained from the Tennessee data set when cases were stratified by year of sample collection (lighter red) versus the year of diagnosis (darker red). Note that the collection date trend is not identical to the trend in the left plot because we downsampled cases to match the availability of diagnosis dates.</p>   </div> </div> <div class="3.4obtaininggaic" title="sec"> <div class="title" tagx="title" title="title">
3.4 Obtaining GAIC</div> <p>The results in the preceding section imply that there exists an intermediate value of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> that optimizes the trade-off between case coverage and the number of active clusters, where both quantities have a significant impact on the information content of clusters for public health. We propose that the best criterion for optimizing a clustering method is our ability to predict where the next cases will occur among the resulting clusters. Specifically, we adapted the GAIC (<a href="#veaa011-B22">Nakaya 2000</a>) to select the optimal threshold. Our implementation of the GAIC is a comparison between two Poisson regression models, where the count outcomes are the number of new cases adjacent to each cluster. In the null model, we assume that the rate parameter is proportional to cluster size as a fraction of all known cases (<a href="#E1">Equation 1</a>), assuming no variation among individual known cases. In the weighted model, the rate parameter is the total weight of known cases in the <i>i</i>th cluster, where each weight can be calculated from a linear combination of individual- or group-level attributes. For our analysis, we weighted cases by their predicted edge densities from the extended binomial model (<a href="#E5">Equation 5</a>).</p> <p><a href="#veaa011-F4">Figure 4</a> (left) summarizes the distributions of GAIC for varying <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> for each data set. We observed that GAIC tended to be near zero for relaxed thresholds (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
≥</div> <div class="mn_UNKNOWN">
0.03</div> </div></div></span>), which indicated that the ability of the weighted model to forecast new cases was indistinguishable from the null model. At these high thresholds, the majority of known cases tended to become grouped into a single large cluster, thereby homogenizing any individual-level variation that could be used by the weighted model to predict the distribution of new cases. The minimum (most negative) GAIC was obtained <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.0104</div> </div></div></span> for northern Alberta and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.0160</div> </div></div></span> for both Seattle and Tennessee. These minima identified the optimal thresholds that maximized the difference in information loss between the weighted and null models. As we continued to decrease <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> past these optima, the GAIC trends returned to the zero line and even increase sporadically into positive values where the weighted model was <i>worse</i> than the null model. At these low values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span>, the disintegration of clusters into large number of singletons disrupts the covariation between individual attributes and the distribution of new cases. Furthermore, lowering <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> also leads to a reduction of case coverage as shown in <a href="#veaa011-F3">Fig. 3</a>. At the respective optimal thresholds, less than half of new cases were adjacent to clusters of known cases (38.2% for Seattle, 49.1% for Northern Alberta, and 41.8% for Tennessee).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <p>Profiles of GAIC relative to the distance cut-off <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span>. This is calculated as the difference in AIC between the weighted and null Poisson models. A negative GAIC value indicates that the weighted model is more effective in predicting the distribution of new cases among the clusters defined at that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span>. (left) Profiles for the three complete data sets using collection dates. The global GAIC minima for each data set are labelled with the corresponding TN93 distance cut-off values. We note that GAIC values become more erratic with decreasing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> as the distribution of new cases among smaller clusters becomes increasingly sparse and stochastic. (right) Profiles for the Tennessee data sets using sample collection or diagnosis dates, excluding cases with missing diagnostic dates for direct comparison.</p>   </div> <p>The GAIC also provides a framework for variable selection. For instance, the GAIC obtained for the Tennessee data when cases were stratified by year of diagnosis was substantially lower than the values obtained with sample collection dates over wide range of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span> (<a href="#veaa011-F4">Fig. 4</a>, right). The optimal threshold identified by the minimum GAIC coincided for both sets of dates (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> </div></div></span>); however, the GAIC for diagnosis dates was substantially more negative (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
Δ</div> <div class="mtext_UNKNOWN">
GAIC</div> <div class="mo_UNKNOWN">
=</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
58.3</div> </div></div></span>), indicating a more effective use of cluster information. For larger values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
0.02</div> </div></div></span>, the stratification of cases by either set of dates was irrelevant as they collapsed into a single giant cluster, such that both GAIC trends converged to zero. For smaller values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
0.01</div> </div></div></span>, both trends became more erratic with the reduced adjacency of new cases and increasingly stochastic composition of smaller clusters.</p> <p>The graphs for these optimal cut-offs are summarized in <a href="#veaa011-F5">Fig. 5</a>. In the Seattle graph, the largest cluster (Se1) comprising thirty-four known cases was adjacent to two new cases. The sample collection years associated with this cluster range from 2000 to 2012 with a mean of 2006.8 (interquartile range, IQR = 2006–2009). In contrast, the second largest cluster (Se2) comprised only ten known cases that were sampled more recently with a mean of 2009.6 (2008–2011), and accumulated six new cases. Similarly, the largest cluster in the northern Alberta graph (NA1) comprised twenty-two cases of which none were new, with a mean sample collection year of 2009.2 years (2008–2011). This contrasts a smaller cluster of ten known cases (NA2), of which five were collected in 2012 (mean 2010.5, IQR = 2009–2012), that gained twelve new cases in 2013. Finally, we observed a large cluster (Tn1) of seventy-two known cases in the Tennessee data set with only one new case and a mean sample collection year of 2007.6 (IQR = 2005–2010). In contrast, the cluster labelled Tn2 comprised thirty-nine known cases with a mean collection year of 2012 (IQR 2010, 2013) and three new cases. These simple examples illustrate the effect of optimizing the clustering threshold on the covariation of new case counts and the recency of known cases among clusters. As predicted by our model analysis, the effect of variation in case recency among clusters on the distribution of new cases is even clearer for the graph depicting clusters of cases stratified by year of diagnosis rather than sample collection (<a href="#sup1">Supplementary Fig. S3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <p>Network visualizations of the graphs from the Middle Tennessee data (red), Seattle (blue), and northern Alberta (orange) data sets obtained at the respective GAIC-optimized distance thresholds, rendered using an implementation of the <a href="#veaa011-B18">Kamada and Kawai (1989)</a> algorithm. Vertices that corresponded to new cases were coloured in a darker shade, and the width of each vertex was scaled to the sample collection year with more recent cases drawn at a larger size. The vertices with zero degrees (singletons) were not drawn for clarity. Clusters with the largest number of new cases and/or the largest number of cases overall are highlighted for discussion in the text.</p>   </div> </div> <div class="3.5robustnessofgaicoptimization" title="sec"> <div class="title" tagx="title" title="title">
3.5 Robustness of GAIC optimization</div> <p>The difference in optimal clustering thresholds between northern Alberta and the other sites implies that a globally optimal threshold does not exist. However, variation in thresholds may also be a stochastic outcome due to incomplete sampling. To measure the effect of sampling variation, we repeated our GAIC analysis on random sub-samples of the Seattle and Tennessee data sets to 40 per cent, 60 per cent, and 80 per cent of the data (thirty replicates each). The comparably low number of cases per year in the northern Alberta data set precluded sub-sampling. The results for the Seattle data set are summarized in <a href="#sup1">Supplementary Fig. S4</a>. As expected, we observed shallower GAIC trends and more variable GAIC-optimized thresholds with decreasing sample size. However, the optimal thresholds remained clustered around the original value <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.016</div> </div></div></span>, with only four (13%) replicates with optima below <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
0.015</div> </div></div></span> at 40 per cent sub-sampling (<i>n </i>=<i> </i>636, comparable to the northern Alberta sample size of <i>n </i>=<i> </i>803). We obtained similar results on random sub-samples of the Tennessee data set, except that the optima were more robust for the data stratified by diagnosis dates.</p> <p>Finally, to assess the stability of GAIC-optimized thresholds over time, we generated additional sub-samples by progressively right censoring the Seattle and Tennessee data sets. If the information content of pairwise TN93 clusters is stable over time, then we should expect the optimized threshold from fitting models to ‘new cases’ in a given year should be similar to the optimized threshold with an additional year of case data. <a href="#veaa011-F6">Figure 6</a> summarizes our results for the Tennessee data set where the four most recent years were progressively censored by dates of sample collection (left) and diagnosis (right), respectively. Results for the complete Tennessee data set (including cases with missing diagnosis dates) and the Seattle data set are provided in the <a href="#sup1">Supplementary Fig. S7</a>. In general, we observed that the optimal threshold identified by the minimum GAIC was relatively stable over time. Although the exact thresholds varied slightly, the optimal threshold from the previous year tended to confer a GAIC similar to the threshold estimated from that year. In our analysis of the Seattle data set, however, the optimal distance thresholds fluctuated between two local minima around <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.005</div> </div></div></span> and 0.015. For instance, in the subset where cases were limited to 2011 (censoring cases sampled in 2012) we observed three local minima in the GAIC (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.0024</div> </div></div></span>, 0.0072, and 0.0144), including one that was close to the global minimum of the previous year. These results suggest that the larger database of cases sampled in Middle Tennessee confer increased robustness to sampling variation, although we cannot rule out site-specific effects.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <p>GAIC plotted against the cut-off threshold for five progressively right-censored subgraphs with respect to sample collection (left) and diagnosis (right) for the Middle Tennessee data set. The minimum GAIC for each plot is indicated by a circular mark. Arrows and dashed lines indicate where the GAIC-optimized threshold from the previous result would carry over to the subsequent year.</p>   </div> </div> </div> <div class="4.discussion" title="sec"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>Our results demonstrate that an apparently small difference in pairwise genetic distances—for instance, between 0.5 per cent and 1.5 per cent—can make the difference between accurate forecasting of new cases among clusters and becoming misled by stochastic noise. Specifically, both cut-offs cited above are routinely used as customary settings in pairwise genetic distance clustering studies of the same HIV-1 subtype in the same country (<a href="#veaa011-B26">Oster <i>et al.</i> 2015</a>; <a href="#veaa011-B45">Wolf <i>et al.</i> 2017</a>; <a href="#veaa011-B23">National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention, 2018</a>). To investigate the sensitivity of clustering thresholds, we have examined three data sets of anonymized HIV-1 subtype B <i>pol</i> sequences that were collected in different regions of North America within similar time frames. All GAIC-optimized cut-offs are located in the left tails of the respective empirical distributions of pairwise distances, with no clear demarcation that might motivate the selection <i>a priori</i> of one cut-off over another (<a href="#veaa011-F1">Fig. 1</a>). However, our information-based criterion reveals a stark difference between these cut-offs when we evaluate the ability of genetic clusters to ‘forecast’ the occurrence of new cases (<a href="#veaa011-F4">Fig. 4</a>). This discordance is a result of a trade-off between the coverage and predictive value of spatial information that is encapsulated by the MAUP (<a href="#veaa011-B22">Nakaya 2000</a>). As we relax the clustering threshold, instances of cluster growth become more frequent such that aggregate effects (viz. case recency and mean degree) can be distinguished against the background of stochastic effects. However, the variation in growth rates among clusters eventually becomes homogenized as they are collapsed into a single giant cluster at increasingly high thresholds. These two driving forces can be differentiated in their approach to an optimum, with poor case coverage from strict thresholds resulting in erratic changes in forecasting information, and higher case coverage with relaxed thresholds resulting in a smoother gain of information. If the database is small relative to the population or the number of new cases per year is low, the global minimum in the GAIC profile may become obscured by sampling variation (<a href="#sup1">Supplementary Fig. S4</a>), especially when the true optimal threshold is low. Resampling experiments (<a href="http://creativecommons.org/licenses/by/4.0/">Supplementary Figs S4–S6</a>) can help ameliorate the effects of sampling variation and identify situations where sample size may be a concern.</p> <p>Unlike most instances of the MAUP that arise in spatial epidemiology (<a href="#veaa011-B22">Nakaya 2000</a>; <a href="#veaa011-B38">Swift, Liu, and Uber 2008</a>; <a href="#veaa011-B28">Parenteau and Sawada 2011</a>; <a href="#veaa011-B12">Haddawy <i>et al.</i> 2018</a>), our outcome variable (the number of new cases per genetic cluster) is directly dependent on the same parameters that reshape the partition of the spatial distribution of covarates into units. This dual dependency results in an asymptotically increasing model likelihood with increasing distance thresholds, plateauing at the point where all known cases were assigned to the same giant cluster, such that any new cases are effectively guaranteed to be adjacent to this cluster. We addressed this unique problem by formulating a null model where the predicted growth of a cluster was directly proportional to its relative size in the number of known cases. Hence, this null model provided a useful baseline that controlled for the proportionate effect of the largest cluster with increasing cut-offs, thereby enabling us to focus on the predictive value of variation in covariates among clusters.</p> <p>We selected a relatively simple clustering method in widespread use (pairwise TN93 distance clustering by components; <a href="#veaa011-B2">Aldous <i>et al.</i> 2012</a>) to demonstrate our new framework for evaluating clusters, which is based on the GAIC proposed by <a href="#veaa011-B22">Nakaya (2000)</a>. Pairwise clustering has been widely adopted in health jurisdictions around the world, including the US-CDC (<a href="#veaa011-B27">Oster <i>et al.</i> 2018</a>), in part due to the growing popularity of the HIV-TRACE software package that employs TN93 (<a href="#veaa011-B29">Pond <i>et al.</i> 2018</a>). However, we hypothesize that it should be feasible to use this framework to evaluate potentially any clustering method that defines a partition on the database of known cases. For example, if we require some minimum bootstrap support value to define clusters as subtrees extracted from the total phylogenetic tree (<a href="#veaa011-B33">Ragonnet-Cronin <i>et al.</i> 2013</a>), then this bootstrap support threshold can represent a second dimension (in addition to a branch length threshold) to locate the minimal GAIC in combination with a distance threshold. We have also demonstrated that the GAIC can be repurposed for model selection where different linear combinations of predictor variables, such as the mean degree size in a given year of diagnosis (<a href="#veaa011-F4">Fig. 4</a>), are evaluated within the Poisson regression model. <a href="#veaa011-B3">Billock <i>et al.</i> (2019)</a> recently employed a similar model selection approach for pairwise TN93 clusters, although their analysis pre-specified a fixed clustering threshold of 1.5 per cent. Thus, if sufficient metadata are available then one can use the GAIC to select more accurate predictive models of cluster growth while adjusting the clustering criteria, so the models can be evaluated at their best performance.</p> <p>Collection dates in units of years are most frequently available as sample metadata in association with published HIV-1 genetic sequences. We had a strong <i>a priori</i> expectation for an association between new case adjacency and known case recency that we subsequently confirmed from these data (<a href="#veaa011-F2">Fig. 2</a>). On the other hand, we recognize that samples may be collected well after the start of a new infection, due to the long asymptomatic period of HIV-1 infection and social barriers to HIV testing (<a href="#veaa011-B21">Mahajan <i>et al.</i> 2008</a>). Even when using diagnostic dates, there is potential conflation of recent transmission with late diagnosis. Although estimated dates of infection (e.g. the midpoint between the last HIV seronegative and first seropositive visit dates) will tend to be closer to the actual date of infection, the necessary information for accurate estimates are not routinely available in a public health context. We furthermore recognize that more precise dates of sample collection would likely confer greater prediction accuracy. Indeed, the granularity of time in the context of genetic cluster analysis represents an extension of MAUP that is known as the modifiable temporal unit problem (<a href="#veaa011-B4">Cheng and Adepeju 2014</a>). While reducing the length of time intervals may produce more timely predictions, e.g. new cases in the next three months instead of the next year, the accuracy of prediction will erode with progressively shorter intervals. Finally, we propose that an informative assessment on the potential value of genetic clustering for public health would be to compare the GAIC of the genetic clustering method against the value obtained from the prioritization of groups by public health experts. However, the confidential information comprising the latter case is not likely to be found in the public domain.</p> <p>An important caveat to our approach is that the expected probability of an edge between specific known and new cases is very small. Consequently, our method requires a substantial number of new cases to parameterize models of the variation in edge densities among clusters and, ultimately, to discriminate between the null and weighted models. (Note that the number of cases sampled in a given year does not correspond to the annual incidence.) Given the results summarized in <a href="#veaa011-F6">Fig. 6</a>, if those requirements are not met, the minimal GAIC that results from the theoretical MAUP trade-off may be masked by the initial noise of a graph with low case coverage and the merging of clusters with increasing thresholds. The results that we obtained with the smallest data set (Northern Alberta) and our sub-sampling experiments (<a href="http://creativecommons.org/licenses/by/4.0/">Supplementary Figs S4–S6</a>) imply that averaging about 50–100 sampled cases per year over a 6-year period should be adequate for the relatively simple models evaluated here. In addition, our comparison of sample collection versus diagnosis dates (<a href="#sup1">Supplementary Figs S5 and S6</a>) implies that the minimum sample size requirement should diminish with the addition of covariates into the weighted model that have strong associations with the distribution of new cases.</p> <p>Genetic clustering is used increasingly for near real-time monitoring of clinical populations for the purpose of guiding public health activities (<a href="#veaa011-B32">Poon <i>et al.</i> 2016</a>; <a href="#veaa011-B37">Rose <i>et al.</i> 2017</a>; <a href="#veaa011-B27">Oster <i>et al.</i> 2018</a>; <a href="#veaa011-B3">Billock <i>et al.</i> 2019</a>). Our method is not specific to HIV-1, although the proliferation of clustering methods in HIV molecular epidemiology—driven by the abundance of genetic sequence data and the relatively rapid rate of evolution and low transmission rate of the virus—does make this approach particularly applicable to this virus. Similar pairwise distance clustering methods, for instance, have been used for <i>Mycobacterium tuberculosis</i> (<a href="#veaa011-B43">Walker <i>et al.</i> 2013</a>) and hepatitis C virus (<a href="#veaa011-B16">Jacka <i>et al.</i> 2014</a>) to infer epidemiological characteristics from molecular sequence variation. In these cases, it may be necessary to rescale the step size/range of clustering thresholds to the expected distribution of genetic distances in order to locate the minimum GAIC. Furthermore, variation in population density and overall case prevalence among regions, as well as the proportion of diagnosed cases, will influence the GAIC. For instance, the visible difference in minimum GAIC we observed between data sets from Northern Alberta and the two locations in the USA was likely driven by differences in population density and sampling fraction. Applications of clustering for HIV-1 prevention have tended to be developed in the context of low-level epidemics, e.g. North America and western Europe, making it challenging to translate these methods to generalized epidemics in other countries without a robust statistical framework for calibrating methods. No matter what pathogen or location is the focus of investigation, it is imperative that we become more critical of clustering methods (<a href="#veaa011-B40">Volz <i>et al.</i> 2012</a>; <a href="#veaa011-B30">Poon 2016</a>). Improperly calibrated clustering methods may otherwise prioritize false positives, diverting limited public health resources away from subpopulations where the immediate need for prevention and treatment services was greatest.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">veaa011_Supplementary_Data</span><div class="media" title="media"><a href="veaa011_supplementary_data.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>An early version of this work was presented at the 26th International HIV Dynamics &amp;amp; Evolution meeting in Cascais, Portugal. We thank the two anonymous reviewers for their insightful commentary, John R. J. Thompson for a helpful discussion on applications of spatial statistics to model forest fire risk, which motivated us to look beyond autoregressive models; and Dr Tomoki Nakaya for encouraging feedback at an early stage of this study.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>Data available at GenBank accession numbers <a href="KY034691">KY034691</a>–<a href="KY037792">KY037792</a>, <a href="KU190031">KU190031</a>–<a href="KU190839">KU190839</a> ,and <a href="MH352627">MH352627</a>–<a href="MH355541">MH355541</a>.</p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>This study was supported by a Project Grant from the Canadian Institutes of Health Research (PJT-156178) and by an Administrative Supplement to a Center Core Grant from the Tennessee Center for AIDS Research (P30-AI110527). CC was supported by a Dr Frederick Winnett Luney Graduate Research Award from the Department of Pathology at Western University, Canada.</p> <p><b>Conflict of interest:</b> None declared.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="veaa011-B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akaike</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">1998</span>) ‘<span class="chapter-title" tagx="chapter-title" title="chapter-title">Information Theory and an Extension of the Maximum Likelihood Principle’,</span> in Parzen, E., Tanabe, K., and Kitagawa, G. (eds.) <span class="source" tagx="source" title="source">Selected Papers of Hirotugu Akaike</span>, pp. <span class="fpage" tagx="fpage" title="fpage">199</span>–<span class="lpage" tagx="lpage" title="lpage">213</span>. New York, NY: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>.</span></li> <li tag="ref"><a name="veaa011-B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aldous</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Characterizing HIV Transmission Networks across the United</span> States’, <span class="source" tagx="source" title="source">Clinical Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">1135</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22784872">22784872</a></span></span></li> <li tag="ref"><a name="veaa011-B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Billock</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2019</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Prediction of HIV Transmission Cluster Growth with Statewide Surveillance Data</span>’, <span class="source" tagx="source" title="source">Journal of Acquired Immune Deficiency Syndromes</span>, <span class="volume" tagx="volume" title="volume">80</span>: <span class="fpage" tagx="fpage" title="fpage">152</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30422907">30422907</a></span></span></li> <li tag="ref"><a name="veaa011-B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adepeju</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">‘Modifiable Temporal Unit Problem (MTUP) and Its Effect on Space–Time Cluster Detection’</span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e100465</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24971885">24971885</a></span></span></li> <li tag="ref"><a name="veaa011-B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cock</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Biopython: Freely Available Python Tools for Computational Molecular Biology and Bioinformatics</span>’, <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1422</span>–<span class="lpage" tagx="lpage" title="lpage">3</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19304878">19304878</a></span></span></li> <li tag="ref"><a name="veaa011-B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">M. J.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2019</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Precision Surveillance for Viral Respiratory Pathogens: Virome Capture Sequencing for the Detection and Genomic Characterization of Severe Acute Respiratory Infection in Uganda</span>’, <span class="source" tagx="source" title="source">Clinical Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">68</span>: <span class="fpage" tagx="fpage" title="fpage">1118</span>–<span class="lpage" tagx="lpage" title="lpage">25</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30099510">30099510</a></span></span></li> <li tag="ref"><a name="veaa011-B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dasgupta</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2019</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Estimating Effects of HIV Sequencing Data Completeness on Transmission Network Patterns and Detection of Growing HIV Transmission Clusters</span>’, <span class="source" tagx="source" title="source">AIDS Research and Human Retroviruses</span>, <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">368</span>–<span class="lpage" tagx="lpage" title="lpage">75</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30403157">30403157</a></span></span></li> <li tag="ref"><a name="veaa011-B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Oliveira</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Transmission Networks and Risk of HIV Infection in KwaZulu-Natal, South Africa: A Community-Wide Phylogenetic Study</span>’, <span class="source" tagx="source" title="source">The Lancet HIV</span>, <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">e41</span>–<span class="lpage" tagx="lpage" title="lpage">50</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27914874">27914874</a></span></span></li> <li tag="ref"><a name="veaa011-B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dennis</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Phylogenetic Insights into Regional HIV Transmission</span>’, <span class="source" tagx="source" title="source">AIDS</span>, <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">1813</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22739398">22739398</a></span></span></li> <li tag="ref"><a name="veaa011-B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dennis</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">HIV-1 Transmission Clustering and Phylodynamics Highlight the Important Role of Young Men Who Have Sex with Men</span>’, <span class="source" tagx="source" title="source">AIDS Research and Human Retroviruses</span>, <span class="volume" tagx="volume" title="volume">34</span>: <span class="fpage" tagx="fpage" title="fpage">879</span>–<span class="lpage" tagx="lpage" title="lpage">88</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30027754">30027754</a></span></span></li> <li tag="ref"><a name="veaa011-B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonsalves</span><span class="given-names" tagx="given-names" title="given-names">G. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawford</span><span class="given-names" tagx="given-names" title="given-names">F. W.</span></span></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Dynamics of the HIV Outbreak and Response in Scott County, in, USA, 2011–15: A Modelling Study</span>’, <span class="source" tagx="source" title="source">The Lancet HIV</span>, <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e569</span>–<span class="lpage" tagx="lpage" title="lpage">77</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30220531">30220531</a></span></span></li> <li tag="ref"><a name="veaa011-B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haddawy</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Complexity-Based Spatial Hierarchical Clustering for Malaria Prediction</span>’, <span class="source" tagx="source" title="source">Journal of Healthcare Informatics Research</span>, <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">423</span>–<span class="lpage" tagx="lpage" title="lpage">47</span>.</span></li> <li tag="ref"><a name="veaa011-B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hassan</span><span class="given-names" tagx="given-names" title="given-names">A. S.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Defining HIV-1 Transmission Clusters Based on Sequence Data</span>’, <span class="source" tagx="source" title="source">AIDS</span>, <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">1211</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28353537">28353537</a></span></span></li> <li tag="ref"><a name="veaa011-B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hemelaar</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Implications of HIV Diversity for the HIV-1 Pandemic</span>’, <span class="source" tagx="source" title="source">Journal of Infection</span>, <span class="volume" tagx="volume" title="volume">66</span>: <span class="fpage" tagx="fpage" title="fpage">391</span>–<span class="lpage" tagx="lpage" title="lpage">400</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23103289">23103289</a></span></span></li> <li tag="ref"><a name="veaa011-B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S. S.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2010</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Automated Detection of Infectious Disease Outbreaks in Hospitals: A Retrospective Cohort Study</span>’, <span class="source" tagx="source" title="source">PLoS Medicine</span>, <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1000238</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20186274">20186274</a></span></span></li> <li tag="ref"><a name="veaa011-B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacka</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Phylogenetic Clustering of Hepatitis C Virus among People Who Inject Drugs in Vancouver, Canada</span>’, <span class="source" tagx="source" title="source">Hepatology</span>, <span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">1571</span>–<span class="lpage" tagx="lpage" title="lpage">80</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25042607">25042607</a></span></span></li> <li tag="ref"><a name="veaa011-B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Junqueira</span><span class="given-names" tagx="given-names" title="given-names">D. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Short-Term Dynamic and Local Epidemiological Trends in the South American HIV-1b Epidemic</span>’, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">e0156712</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27258369">27258369</a></span></span></li> <li tag="ref"><a name="veaa011-B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamada</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawai</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">1989</span>) ‘<span class="mixed-article-title" title="mixed-article-title">An Algorithm for Drawing General Undirected Graphs</span>’, <span class="source" tagx="source" title="source">Information Processing Letters</span>, <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">7</span>–<span class="lpage" tagx="lpage" title="lpage">15</span>.</span></li> <li tag="ref"><a name="veaa011-B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kulldorff</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2005</span>) ‘<span class="mixed-article-title" title="mixed-article-title">A Space–Time Permutation Scan Statistic for Disease Outbreak Detection</span>’, <span class="source" tagx="source" title="source">PLoS Medicine</span>, <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">e59</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15719066">15719066</a></span></span></li> <li tag="ref"><a name="veaa011-B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Can the UNAIDS 90-90-90 Target Be Achieved? A Systematic Analysis of National HIV Treatment Cascades</span>’, <span class="source" tagx="source" title="source">BMJ Global Health</span>, <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">e000010</span>.</span></li> <li tag="ref"><a name="veaa011-B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahajan</span><span class="given-names" tagx="given-names" title="given-names">A. P.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2008</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Stigma in the HIV/AIDS Epidemic: A Review of the Literature and Recommendations for the Way Forward</span>’, <span class="source" tagx="source" title="source">AIDS</span>, <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">S67</span>.</span></li> <li tag="ref"><a name="veaa011-B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakaya</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2000</span>) ‘<span class="mixed-article-title" title="mixed-article-title">An Information Statistical Approach to the Modifiable Areal Unit Problem in Incidence Rate Maps</span>’, <span class="source" tagx="source" title="source">Environment and Planning A: Economy and Space</span>, <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">91</span>–<span class="lpage" tagx="lpage" title="lpage">109</span>.</span></li> <li tag="ref"><a name="veaa011-B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">National Center for HIV/AIDS, Viral Hepatitis, STD, and TB Prevention (<span class="year" tagx="year" title="year">2018</span>) <i>Detecting and Responding to HIV Transmission Clusters: A Guide for Health Departments</i> &amp;lt;<a href="https://www.cdc.gov/hiv/pdf/funding/announcements/ps18-1802/CDC-HIV-PS18-1802-AttachmentE-Detecting-Investigating-and-Responding-to-HIV-Transmission-Clusters.pdf">https://www.cdc.gov/hiv/pdf/funding/announcements/ps18-1802/CDC-HIV-PS18-1802-AttachmentE-Detecting-Investigating-and-Responding-to-HIV-Transmission-Clusters.pdf</a>&amp;gt; accessed 20 Jan. 2020.</span></li> <li tag="ref"><a name="veaa011-B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Novitsky</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Impact of Sampling Density on the Extent of HIV Clustering</span>’, <span class="source" tagx="source" title="source">AIDS Research and Human Retroviruses</span>, <span class="volume" tagx="volume" title="volume">30</span>: <span class="fpage" tagx="fpage" title="fpage">1226</span>–<span class="lpage" tagx="lpage" title="lpage">35</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25275430">25275430</a></span></span></li> <li tag="ref"><a name="veaa011-B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Openshaw</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="year" tagx="year" title="year">1979</span> ‘<span class="chapter-title" tagx="chapter-title" title="chapter-title">A Million or So Correlation Coefficients: Three Experiments on the Modifiable Areal Unit Problem’,</span> in <span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wrigley</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span> (ed.) <span class="source" tagx="source" title="source">Statistical Applications in the Spatial Sciences</span>, pp. <span class="fpage" tagx="fpage" title="fpage">127</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Pion</span>.</span></li> <li tag="ref"><a name="veaa011-B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oster</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Using Molecular HIV Surveillance Data to Understand Transmission between Subpopulations in the United States</span>’, <span class="source" tagx="source" title="source">Journal of Acquired Immune Deficiency Syndromes</span>, <span class="volume" tagx="volume" title="volume">70</span>: <span class="fpage" tagx="fpage" title="fpage">444</span>–<span class="lpage" tagx="lpage" title="lpage">51</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26302431">26302431</a></span></span></li> <li tag="ref"><a name="veaa011-B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oster</span><span class="given-names" tagx="given-names" title="given-names">A. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Identifying Clusters of Recent and Rapid HIV Transmission through Analysis of Molecular Surveillance Data</span>’, <span class="source" tagx="source" title="source">Journal of Acquired Immune Deficiency Syndromes</span>, <span class="volume" tagx="volume" title="volume">79</span>: <span class="fpage" tagx="fpage" title="fpage">543</span>–<span class="lpage" tagx="lpage" title="lpage">50</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30222659">30222659</a></span></span></li> <li tag="ref"><a name="veaa011-B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parenteau</span><span class="given-names" tagx="given-names" title="given-names">M.-P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawada</span><span class="given-names" tagx="given-names" title="given-names">M. C.</span></span></span> (<span class="year" tagx="year" title="year">2011</span>) ‘<span class="mixed-article-title" title="mixed-article-title">The Modifiable Areal Unit Problem (MAUP) in the Relationship between Exposure to NO<sub>2</sub> and Respiratory Health</span>’, <span class="source" tagx="source" title="source">International Journal of Health Geographics</span>, <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">58</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22040001">22040001</a></span></span></li> <li tag="ref"><a name="veaa011-B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pond</span><span class="given-names" tagx="given-names" title="given-names">S. L. K.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">HIV-TRACE (TRAnsmission Cluster Engine): A Tool for Large Scale Molecular Epidemiology of HIV-1 and Other Rapidly Evolving Pathogens</span>’, <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>, <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">1812</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29401317">29401317</a></span></span></li> <li tag="ref"><a name="veaa011-B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">A. F.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Impacts and Shortcomings of Genetic Clustering Methods for Infectious Disease Outbreaks</span>’, <span class="source" tagx="source" title="source">Virus Evolution</span>, <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">vew031</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28058111">28058111</a></span></span></li> <li tag="ref"><a name="veaa011-B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">A. F.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) ‘<span class="mixed-article-title" title="mixed-article-title">The Impact of Clinical, Demographic and Risk Factors on Rates of HIV Transmission: A Population-Based Phylogenetic Analysis in British Columbia, Canada</span>’, <span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">211</span>: <span class="fpage" tagx="fpage" title="fpage">926</span>–<span class="lpage" tagx="lpage" title="lpage">35</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25312037">25312037</a></span></span></li> <li tag="ref"><a name="veaa011-B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">A. F.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Near Real-Time Monitoring of HIV Transmission Hotspots from Routine HIV Genotyping: An Implementation Case Study</span>’, <span class="source" tagx="source" title="source">The Lancet HIV</span>, <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">e231</span>–<span class="lpage" tagx="lpage" title="lpage">38</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27126490">27126490</a></span></span></li> <li tag="ref"><a name="veaa011-B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragonnet-Cronin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Automated Analysis of Phylogenetic Clusters</span>’, <span class="source" tagx="source" title="source">BMC Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">317</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24191891">24191891</a></span></span></li> <li tag="ref"><a name="veaa011-B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragonnet-Cronin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Recent and Rapid Transmission of HIV among People Who Inject Drugs in Scotland Revealed through Phylogenetic Analysis</span>’, <span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">217</span>: <span class="fpage" tagx="fpage" title="fpage">1875</span>–<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29546333">29546333</a></span></span></li> <li tag="ref"><a name="veaa011-B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratmann</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Sources of HIV Infection among Men Having Sex with Men and Implications for Prevention</span>’, <span class="source" tagx="source" title="source">Science Translational Medicine</span>, <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">320ra2</span>.</span></li> <li tag="ref"><a name="veaa011-B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robertson</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2010</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Review of Methods for Space–Time Disease Surveillance</span>’, <span class="source" tagx="source" title="source">Spatial and Spatio-Temporal Epidemiology</span>, <span class="volume" tagx="volume" title="volume">1</span>: <span class="fpage" tagx="fpage" title="fpage">105</span>–<span class="lpage" tagx="lpage" title="lpage">16</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22749467">22749467</a></span></span></li> <li tag="ref"><a name="veaa011-B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Identifying Transmission Clusters with Cluster Picker and HIV-TRACE</span>’, <span class="source" tagx="source" title="source">AIDS Research and Human Retroviruses</span>, <span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">211</span>–<span class="lpage" tagx="lpage" title="lpage">18</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27824249">27824249</a></span></span></li> <li tag="ref"><a name="veaa011-B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swift</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uber</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Reducing MAUP Bias of Correlation Statistics between Water Quality and GI Illness</span>’, <span class="source" tagx="source" title="source">Computers, Environment and Urban Systems</span>, <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">134</span>–<span class="lpage" tagx="lpage" title="lpage">48</span>.</span></li> <li tag="ref"><a name="veaa011-B39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">1993</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Estimation of the Number of Nucleotide Substitutions in the Control Region of Mitochondrial DNA in Humans and Chimpanzees</span>’, <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>, <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">512</span>–<span class="lpage" tagx="lpage" title="lpage">26</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8336541">8336541</a></span></span></li> <li tag="ref"><a name="veaa011-B40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Simple Epidemiological Dynamics Explain Phylogenetic Clustering of HIV from Patients with Recent Infection</span>’, <span class="source" tagx="source" title="source">PLoS Computational Biology</span>, <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e1002552</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22761556">22761556</a></span></span></li> <li tag="ref"><a name="veaa011-B41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2018</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Molecular Epidemiology of HIV-1 Subtype B Reveals Heterogeneous Transmission Risk: Implications for Intervention and Control</span>’, <span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">217</span>: <span class="fpage" tagx="fpage" title="fpage">1522</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29506269">29506269</a></span></span></li> <li tag="ref"><a name="veaa011-B42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vrancken</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">The Multi-Faceted Dynamics of HIV-1 Transmission in Northern Alberta: A Combined Analysis of Virus Genetic and Public Health Data</span>’, <span class="source" tagx="source" title="source">Infection, Genetics and Evolution</span>, <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">100</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>.</span></li> <li tag="ref"><a name="veaa011-B43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">T. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Whole-Genome Sequencing to Delineate <i>Mycobacterium tuberculosis</i> Outbreaks: A Retrospective Observational Study</span>’, <span class="source" tagx="source" title="source">The Lancet Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">137</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23158499">23158499</a></span></span></li> <li tag="ref"><a name="veaa011-B44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wertheim</span><span class="given-names" tagx="given-names" title="given-names">J. O.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Social and Genetic Networks of HIV-1 Transmission in New York City</span>’, <span class="source" tagx="source" title="source">PLoS Pathogens</span>, <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">e1006000</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28068413">28068413</a></span></span></li> <li tag="ref"><a name="veaa011-B45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolf</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2017</span>) ‘<span class="mixed-article-title" title="mixed-article-title">Phylogenetic Evidence of HIV-1 Transmission between Adult and Adolescent Men Who Have Sex with Men</span>’, <span class="source" tagx="source" title="source">AIDS Research and Human Retroviruses</span>, <span class="volume" tagx="volume" title="volume">33</span>: <span class="fpage" tagx="fpage" title="fpage">318</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27762596">27762596</a></span></span></li> </ul> </div> </div>  </body></html>