<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/28/2020.03.26.010322.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.26.010322v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Site-specific analysis of the SARS-CoV-2 glycan shield" />
<meta name="DC.Identifier" content="10.1101/2020.03.26.010322" />
<meta name="DC.Date" content="2020-03-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The emergence of the betacoronavirus, SARS-CoV-2 that causes COVID-19, represents a significant threat to global human health. Vaccine development is focused on the principal target of the humoral immune response, the spike (S) glycoprotein, that mediates cell entry and membrane fusion. SARS-CoV-2 S gene encodes 22 N-linked glycan sequons per protomer, which likely play a role in immune evasion and occluding immunogenic protein epitopes. Here, using a site-specific mass spectrometric approach, we reveal the glycan structures on a recombinant SARS-CoV-2 S immunogen. This analysis enables mapping of the glycan-processing states across the trimeric viral spike. We show how SARS-CoV-2 S glycans differ from typical host glycan processing, which may have implications in viral pathobiology and vaccine design." />
<meta name="DC.Contributor" content="Yasunori Watanabe" />
<meta name="DC.Contributor" content="Joel D. Allen" />
<meta name="DC.Contributor" content="Daniel Wrapp" />
<meta name="DC.Contributor" content="Jason S. McLellan" />
<meta name="DC.Contributor" content="Max Crispin" />
<meta name="article:published_time" content="2020-03-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Site-specific analysis of the SARS-CoV-2 glycan shield" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The emergence of the betacoronavirus, SARS-CoV-2 that causes COVID-19, represents a significant threat to global human health. Vaccine development is focused on the principal target of the humoral immune response, the spike (S) glycoprotein, that mediates cell entry and membrane fusion. SARS-CoV-2 S gene encodes 22 N-linked glycan sequons per protomer, which likely play a role in immune evasion and occluding immunogenic protein epitopes. Here, using a site-specific mass spectrometric approach, we reveal the glycan structures on a recombinant SARS-CoV-2 S immunogen. This analysis enables mapping of the glycan-processing states across the viral spike. We show how SARS-CoV-2 S glycans differ from typical host glycan processing, which may have implications in viral pathobiology and vaccine design.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.26.010322v1" />
<meta name="citation_id" content="2020.03.26.010322v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.26.010322" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.26.010322" />
<meta name="citation_author" content="Yasunori Watanabe" />
<meta name="citation_author_institution" content="School of Biological Sciences, University of Southampton" />
<meta name="citation_author_institution" content="Oxford Glycobiology Institute, Department of Biochemistry, University of Oxford" />
<meta name="citation_author_institution" content="Division of Structural Biology, University of Oxford, Wellcome Centre for Human Genetics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7952-1268" />
<meta name="citation_author" content="Joel D. Allen" />
<meta name="citation_author_institution" content="School of Biological Sciences, University of Southampton" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2547-968X" />
<meta name="citation_author" content="Daniel Wrapp" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin" />
<meta name="citation_author" content="Jason S. McLellan" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin" />
<meta name="citation_author" content="Max Crispin" />
<meta name="citation_author_institution" content="School of Biological Sciences, University of Southampton" />
<meta name="citation_author_email" content="max.crispin@soton.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1072-2694" />
<meta name="citation_reference" content="Huang, C. et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 6736, 1–10 (2020)." />
<meta name="citation_reference" content="Yang, X. et al. Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study. Lancet Respir. Med. (2020). doi:10.1016/S2213-2600(20)30079-5" />
<meta name="citation_reference" content="Letko, M., Marzi, A. &amp; Munster, V. Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses. Nat. Microbiol. 1–8 (2020). doi:10.1038/s41564-020-0688-y" />
<meta name="citation_reference" content="Watanabe, Y. et al. Vulnerabilities in coronavirus glycan shields despite extensive glycosylation. bioRxiv 2020.02.20.957472 (2020). doi:10.1101/2020.02.20.957472" />
<meta name="citation_reference" content="Watanabe, Y., Bowden, T. A., Wilson, I. A. &amp; Crispin, M. Exploitation of glycosylation in enveloped virus pathobiology. Biochim. Biophys. Acta 1863, 1480–1497 (2019)." />
<meta name="citation_reference" content="Dalziel, M., Crispin, M., Scanlan, C. N., Zitzmann, N. &amp; Dwek, R. A. Emerging principles for the therapeutic exploitation of glycosylation. Science 343, 1235681 (2014)." />
<meta name="citation_reference" content="Scanlan, C. N., Offer, J., Zitzmann, N. &amp; Dwek, R. A. Exploiting the defensive sugars of HIV-1 for drug and vaccine design. Nature 446, 1038–1045 (2007)." />
<meta name="citation_reference" content="Walls, A. C. et al. Unexpected receptor functional mimicry elucidates activation of coronavirus fusion. Cell 176, 1026-1–39.e5 (2019)." />
<meta name="citation_reference" content="Yang, T. J. et al. Cryo-EM analysis of a feline coronavirus spike protein reveals a unique structure and camouflaging glycans. Proc. Natl. Acad. Sci. U. S. A. 117, 1438–1446 (2020)." />
<meta name="citation_reference" content="Safety and Immunogenicity Study of 2019-nCoV Vaccine (mRNA-1273) to Prevent SARS-CoV-2 Infection - Full Text View - ClinicalTrials.gov. Available at: https://clinicaltrials.gov/ct2/show/NCT04283461?term=mrna-1273&amp;draw=2&amp;rank=1. (Accessed: 25th March 2020)" />
<meta name="citation_reference" content="Stertz, S. et al. The intracellular sites of early replication and budding of SARS-coronavirus. Virology 361, 304–315 (2007)." />
<meta name="citation_reference" content="Fehr, A. R. &amp; Perlman, S. Coronaviruses: An overview of their replication and pathogenesis. in Coronaviruses: Methods and Protocols pp 1–23 (Springer New York, 2015)." />
<meta name="citation_reference" content="Ng, M. L., Tan, S. H., See, E. E., Ooi, E. E. &amp; Ling, A. E. Proliferative growth of SARS coronavirus in Vero E6 cells. J. Gen. Virol. 84, 3291–3303 (2003)." />
<meta name="citation_reference" content="Venkatagopalan, P., Daskalova, S. M., Lopez, L. A., Dolezal, K. A. &amp; Hogue, B. G. Coronavirus envelope (E) protein remains at the site of assembly. Virology 478, 75–85 (2015)." />
<meta name="citation_reference" content="Ritchie, G. et al. Identification of N-linked carbohydrates from severe acute respiratory syndrome (SARS) spike glycoprotein. Virology 399, 257–69 (2010)." />
<meta name="citation_reference" content="Behrens, A.-J. et al. Molecular architecture of the cleavage-dependent mannose patch on a soluble HIV-1 envelope glycoprotein trimer. J. Virol. 91, e01894–16 (2017)." />
<meta name="citation_reference" content="Tokatlian, T. et al. Innate immune recognition of glycans targets HIV nanoparticle immunogens to germinal centers. Science 363, 649–654 (2019)." />
<meta name="citation_reference" content="Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (2020). doi:10.1126/science.abb2507" />
<meta name="citation_reference" content="Walls, A. C. et al. Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell (2020). doi:10.1016/j.cell.2020.02.058" />
<meta name="citation_reference" content="Struwe, W. B. et al. Site-specific glycosylation of virion-derived HIV-1 Env Is mimicked by a soluble trimeric immunogen. Cell Rep. 24, 1958–1966.e5 (2018)." />
<meta name="citation_reference" content="Watanabe, Y. et al. Structure of the Lassa virus glycan shield provides a model for immunological resistance. Proc. Natl. Acad. Sci. 115, 7320–7325 (2018)." />
<meta name="citation_reference" content="Behrens, A.-J. et al. Composition and antigenic effects of individual glycan sites of a trimeric HIV-1 envelope glycoprotein. Cell Rep. 14, 2695–2706 (2016)." />
<meta name="citation_reference" content="Andrabi, R. et al. The chimpanzee SIV envelope trimer: structure and deployment as an HIV vaccine template. Cell Rep. 27, 2426–2441.e6 (2019)." />
<meta name="citation_reference" content="Pallesen, J. et al. Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen. Proc. Natl. Acad. Sci. U. S. A. 114, E7348–E7357 (2017)." />
<meta name="citation_reference" content="Pritchard, L. et al. Structural constraints determine the glycosylation of HIV-1 envelope trimers. Cell Rep. 11, 1604–1613 (2015)." />
<meta name="citation_reference" content="Panico, M. et al. Mapping the complete glycoproteome of virion-derived HIV-1 gp120 provides insights into broadly neutralizing antibody binding. Sci. Rep. 6, 32956 (2016)." />
<meta name="citation_reference" content="Stumpo, K. A. &amp; Reinhold, V. N. The N-glycome of human plasma. J. Proteome Res. 9, 4823–4830 (2010)." />
<meta name="citation_reference" content="Knežević, A. et al. Variability, heritability and environmental determinants of human plasma n-glycome. J. Proteome Res. 8, 694–701 (2009)." />
<meta name="citation_reference" content="Shivatare, V. S. et al. Unprecedented role of hybrid N-Glycans as ligands for HIV-1 broadly neutralizing antibodies. J. Am. Chem. Soc. 140, (2018)." />
<meta name="citation_reference" content="Bianchi, M. et al. Electron-Microscopy-Based Epitope Mapping Defines Specificities of Polyclonal Antibodies Elicited during HIV-1 BG505 Envelope Trimer Immunization. Immunity 49, 288–300.e8 (2018)." />
<meta name="citation_reference" content="Jardine, J. et al. Rational HIV immunogen design to target specific germline B cell receptors. Science 340, 711–716 (2013)." />
<meta name="citation_reference" content="Wei, C.-J. et al. Cross-neutralization of 1918 and 2009 Influenza viruses: role of glycans in viral evolution and vaccine design. Sci. Transl. Med. 2, 24ra21 (2010)." />
<meta name="citation_reference" content="Xu, R. et al. Structural basis of preexisting immunity to the 2009 H1N1 pandemic influenza virus. Science 328, 357–360 (2010)." />
<meta name="citation_reference" content="Wei, X. et al. Antibody neutralization and escape by HIV-1. Nature 422, 307–312 (2003)." />
<meta name="citation_reference" content="Zhang, M. et al. Tracking global patterns of N-linked glycosylation site variation in highly variable viral glycoproteins: HIV, SIV, and HCV envelopes and influenza hemagglutinin. Glycobiology 14, 1229–1246 (2004)." />
<meta name="citation_reference" content="Stewart-Jones, G. B. E. et al. Trimeric HIV-1-Env structures define glycan shields from Clades A, B, and G. Cell 165, 813–26 (2016)." />
<meta name="citation_reference" content="Cao, L. et al. Differential processing of HIV envelope glycans on the virus and soluble recombinant trimer. Nat. Commun. 9, 3693 (2018)." />
<meta name="citation_reference" content="Pritchard, L. K., Harvey, D. J., Bonomelli, C., Crispin, M. &amp; Doores, K. J. Cell- and protein-directed glycosylation of native cleaved HIV-1 envelope. J. Virol. 89, 8932–44 (2015)." />
<meta name="citation_reference" content="Grant, T., Rohou, A. &amp; Grigorieff, N. CisTEM, user-friendly software for single-particle image processing. eLife 7, (2018)." />
<meta name="citation_reference" content="Emsley, P. &amp; Crispin, M. Structural analysis of glycoproteins: Building N-linked glycans with coot. Acta Crystallogr. Sect. D Struct. Biol. 74, 256–263 (2018)." />
<meta name="twitter:title" content="Site-specific analysis of the SARS-CoV-2 glycan shield" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The emergence of the betacoronavirus, SARS-CoV-2 that causes COVID-19, represents a significant threat to global human health. Vaccine development is focused on the principal target of the humoral immune response, the spike (S) glycoprotein, that mediates cell entry and membrane fusion. SARS-CoV-2 S gene encodes 22 N-linked glycan sequons per protomer, which likely play a role in immune evasion and occluding immunogenic protein epitopes. Here, using a site-specific mass spectrometric approach, we reveal the glycan structures on a recombinant SARS-CoV-2 S immunogen. This analysis enables mapping of the glycan-processing states across the trimeric viral spike. We show how SARS-CoV-2 S glycans differ from typical host glycan processing, which may have implications in viral pathobiology and vaccine design." />
<meta name="og-title" property="og:title" content="Site-specific analysis of the SARS-CoV-2 glycan shield" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The emergence of the betacoronavirus, SARS-CoV-2 that causes COVID-19, represents a significant threat to global human health. Vaccine development is focused on the principal target of the humoral immune response, the spike (S) glycoprotein, that mediates cell entry and membrane fusion. SARS-CoV-2 S gene encodes 22 N-linked glycan sequons per protomer, which likely play a role in immune evasion and occluding immunogenic protein epitopes. Here, using a site-specific mass spectrometric approach, we reveal the glycan structures on a recombinant SARS-CoV-2 S immunogen. This analysis enables mapping of the glycan-processing states across the trimeric viral spike. We show how SARS-CoV-2 S glycans differ from typical host glycan processing, which may have implications in viral pathobiology and vaccine design." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.26.010322v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.26.010322v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.26.010322v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1215896" />
    <title>Site-specific analysis of the SARS-CoV-2 glycan shield | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"F6D3XVWWwdIrVBPoELKh3rGouMQnvduguBNGTBYtwg4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1215896","apath":"\/biorxiv\/early\/2020\/03\/28\/2020.03.26.010322.atom","pisa":"biorxiv;2020.03.26.010322v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.26.010322v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.26.010322v1","title":"Site-specific analysis of the SARS-CoV-2 glycan shield","identifier":"node\/1215896"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+WuqHM8snPozf5xZwfMYA4DKOQvvsjYnml4\\\\\\\/eBftuXy4QTDMBKUqnK1RWc21eUBXRXVltoOzmmAFaFHXDyq6QLDdDEucexAa9pGTlH7zgurigS3bhfRfF9gyk2q14zubtDfx4zdUJ\\\\\\\/cTaOoAQrIErHATFmnVIPvjfhlh5xokb0VtJ28boXBEJUyHStLDhtmc6RE4yK\\\\\\\/e2K5PemVsQgmmTgoIsGR14Kkgm3YcFBxnwEMp0H7vuEtdcHoJonQM4u7yCdP4HOfIkeBVxHvFL5xqvSxfPUeWsg90tEzIAq5FXEbZ7zTaMqx3VnT5lA7+NQJm4ZrwMKapDG5r4QYHBRBPwRS17U3xa5lvaoHV4ZWf0fWlVxs\\\\\\\/aKC8rwkrWIaLxHyn166D7D3JOcKxy\\\\\\\/WhN54UD2yIzjJ+PuMs3rpyR15yE1hcn5Woh9Afmt7MGx7X2mASr45LX9kJkgTSQGY4Sx6sx0wendJHRm6OUXBFqJnbg4XnPElNwvrS7zApnn3dH9R+bgLL+nnxjrtzy84MnMgRh+0QhQwrHIpBNwUA+Azkab0QKBur9TyPIHD32el3vH7ve8myCCaSTFFKsgdiE9r5uOkA4JWjS\\\\\\\/HaEPUtK\\\\\\\/0xtbxLV2xpKboMsCKHzffCtgAY\\\\\\\/Fi1ynM+mtHOZkq732nIRmHJ4d275FD9vTHFcJbTI8IW7iNXhmawuS44fj1bxCXJodli8\\\\\\\/3aO4naRAvPytBmg0E6lqCzD12wwjucIKSLGm0SvU7wKM6ysJxERbIwznQsFhrLAPPzj4Qj9KxY762+64p6aaxw6UfOX4sy56FjxuESfctXGtdpN4i8xL+3bRKfZYrds9GxOLvRyXE6iugDDfOYQbhIvz4jsBeuJbdzUeMRXaNZl8ThRawIqtaMVWgpahoh2PCLMq7qQ0b8pD\\\\\\\/fOvrSQ1bamEtrr+\\\\\\\/DAsOscrNdgY3S9b7\\\\\\\/MCxj20oMJHzfmSf\\\\\\\/yrhTP+Q8rsl3zIAV1tkxTyy\\\\\\\/Ejsn2IZ\\\\\\\/3D1uUBCH3tbbVkLQpiZ\\\\\\\/+4uJwTR9H9s9+hAGRnQS+yzh3xfjpGIOkrLBvzyhUeqOVEEMa6ZSOjOjj7tGY6NgnCd86MCc5UDquthvL3V9Q2HbZ9lQxZSUomaE3JbwZL5cK6bWjUxScGScONKYQD6q1D4uGQeu3NmekUuI\\\\\\\/cX7xXezu5XIVHzBlvMqcO2+6CTJqUn198MyOgGWCrUrsM7pfo+9m5zEt6lCURWqOKDWEpxWOUyBqqZnSIQm5epkS6N8JO1XQWr5eb4a602\\\\\\\/wBfBwmXVbi\\\\\\\/74+uinDtBUn8fSJXmVeqaf6t5NClylBmtEaNTlCFJGkngO1kx8IUlp7tLuuxIqO6u\\\u0022,\\\u0022iv\\\u0022:\\\u0022s9XZcRq0mdafqElPGs20RA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022d915e3c2bf371b1445335030c7c9c2524e9d524c6a2fcdf64b07f3eebeeff15e\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.26.010322v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1215896 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.26.010322v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-g6MMRk_QSjQ5IWqa4FTEsgK8wSONOemp_4ElK52Kgt4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215896" id="node1215896" data-pisa="biorxiv;2020.03.26.010322v1" data-pisa-master="biorxiv;2020.03.26.010322" data-apath="/biorxiv/early/2020/03/28/2020.03.26.010322.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Site-specific analysis of the SARS-CoV-2 glycan shield</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7952-1268" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Yasunori</span> <span class="nlm-surname">Watanabe</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2547-968X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Joel D.</span> <span class="nlm-surname">Allen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1072-2694" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Max</span> <span class="nlm-surname">Crispin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.26.010322 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1215896" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yasunori Watanabe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Biological Sciences, University of Southampton</span>, Southampton, SO17 1BJ, <span class='nlm-country'>UK</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Oxford Glycobiology Institute, Department of Biochemistry, University of Oxford</span>, South Parks Road, Oxford, OX1 3QU, <span class='nlm-country'>UK</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Division of Structural Biology, University of Oxford, Wellcome Centre for Human Genetics</span>, Oxford, OX3 7BN, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yasunori%2BWatanabe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Watanabe%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYasunori%2BWatanabe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7952-1268" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yasunori Watanabe</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joel D. Allen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Biological Sciences, University of Southampton</span>, Southampton, SO17 1BJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BD.%2BAllen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allen%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoel%2BD.%2BAllen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2547-968X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joel D. Allen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Wrapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Molecular Biosciences, The University of Texas at Austin</span>, Austin, TX 78712, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BWrapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wrapp%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BWrapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jason S. McLellan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Molecular Biosciences, The University of Texas at Austin</span>, Austin, TX 78712, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BS.%2BMcLellan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLellan%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BS.%2BMcLellan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Max Crispin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Biological Sciences, University of Southampton</span>, Southampton, SO17 1BJ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Max%2BCrispin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crispin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMax%2BCrispin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1072-2694" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Max Crispin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:max.crispin@soton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">max.crispin@soton.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.26.010322v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1215896" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1215896/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.010322v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1215896" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1215896/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.010322v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1215896" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1215896/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.010322v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1215896" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1215896/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.010322v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1215896" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1215896/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.26.010322v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1215896" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1215896/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">The emergence of the betacoronavirus, SARS-CoV-2 that causes COVID-19, represents a significant threat to global human health. Vaccine development is focused on the principal target of the humoral immune response, the spike (S) glycoprotein, that mediates cell entry and membrane fusion. SARS-CoV-2 S gene encodes 22 N-linked glycan sequons per protomer, which likely play a role in immune evasion and occluding immunogenic protein epitopes. Here, using a site-specific mass spectrometric approach, we reveal the glycan structures on a recombinant SARS-CoV-2 S immunogen. This analysis enables mapping of the glycan-processing states across the trimeric viral spike. We show how SARS-CoV-2 S glycans differ from typical host glycan processing, which may have implications in viral pathobiology and vaccine design.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), the causative pathogen of COVID-19<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>, induces fever, severe respiratory illness and pneumonia. SARS-CoV-2 utilizes an extensively glycosylated spike (S) protein that protrudes from the viral surface to bind to angiotensin-converting enzyme 2 (ACE2), the host cell receptor, to mediate cell entry<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. The S protein is a trimeric class I fusion protein that is composed of two functional subunits responsible for receptor binding (S1 subunit) and membrane fusion (S2 subunit). Remarkably, the surface of the virally encoded envelope spike is dominated by an array of host-derived glycans with each trimer displaying 66 N-linked glycosylation sites. This extensive glycosylation has important implications for vaccine design.</p><p id="p-5">As obligate parasites, many viruses exploit host-cell machinery to glycosylate their own proteins. Numerous viral envelope proteins, including HIV-1 envelope (Env), influenza hemagglutinin (HA) and Lassa virus glycoprotein complex (GPC), possess genetically encoded N-linked glycan sequons (N-X-S/T motifs, where X is any amino acid except proline). Viral glycosylation has wide-ranging roles in viral pathobiology, including mediating protein folding and stability, and shaping viral tropism. The genetically encoded sequons can be under significant selective pressure as a mechanism for immune evasion by shielding specific epitopes from antibody neutralization. However, we note the currently reported low mutation rate of SARS-CoV-2, and as yet that there have been no observed mutations to N-linked glycosylation sites<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. Surfaces with an unusually high density of glycans can also enable immune recognition<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. The role of glycosylation in immune evasion by camouflaging immunogenic protein epitopes has been well studied for other coronaviruses<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>.</p><p id="p-6">As the principal antigen presented on the surface of SARS-CoV-2 virions, the S protein is a key target in vaccine design efforts. It is apparent that the viral spike will be targeted by the full assortment of vaccine strategies from nucleic-acid based approaches<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>, whereby the viral protein is expressed <em>in vivo</em>, to recombinant strategies whereby viral glycoproteins are delivered with appropriate adjuvants. Such strategies aim to elicit neutralizing adaptive immunity with an emphasis on achieving an antibody response at the sites of viral entry.</p><p id="p-7">Understanding the glycosylation of recombinant viral spikes can both reveal fundamental features of viral biology and can guide vaccine design strategies and manufacturing. As glycans are enzymatically elaborated in the Golgi apparatus, some features of processed, so-called complex-type, glycosylation will necessarily be influenced by the producer cell-line. However, the presence of glycans typical of the early stages of the secretory pathway on otherwise mature glycoproteins often closely relate to fundamental features of viral spike architecture. High viral glycan density, or extensive packing of the glycan to the protein surface, can impair the normal glycan maturation pathway by steric interference of host enzymes. Under-processed oligomannose- and hybrid-type glycans are found on the majority of viral envelope proteins such as HIV-1 Env, influenza HA and Lassa virus GPC<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup>. These viral glycoproteins traffic through the secretory system and the glycosylation processing of recombinant material often closely captures the glycan maturation state of the virion. This can be particularly important for viruses such as HIV-1, where viral glycans can also be targeted by neutralizing antibodies. Coronaviruses have been reported to form virions by budding into the lumen of endoplasmic reticulum-Golgi intermediate compartments (ERGIC)<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>–<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. However observations of hybrid- and complex-type glycans on virally derived material suggests that the viral glycoproteins are subjected to Golgi resident processing enzymes<sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>.</p><p id="p-8">As impaired glycan maturation can be a sensitive reporter of local viral protein architecture<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>, detailed site-specific analysis has emerged as an indicator of native-like architecture and is increasingly used to compare different immunogens and in the monitoring of manufacturing processes. Importantly, in addition to these structural insights, the presence of oligomannose-type glycans on viral spike-based immunogens has also been shown to enhance trafficking of glycoprotein to germinal centers via interaction with lectins such as mannose-binding lectin<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. It is therefore of considerable importance to understand the glycosylation of recombinant mimetics of the virus spike.</p><p id="p-9">Here, we apply mass spectrometry to understand both the site-specific N-linked glycan composition and the degree of sequon occupancy on a soluble, native-like SARS-CoV-2 S protein. The native-like folding of trimeric recombinant material has been recently revealed by detailed structural analysis by cryo-electron microscopy<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. We have previously validated our glycopeptide analysis methodology and applied this approach to the study of a range of other viral glycoprotein immunogens<sup><a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>–<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>, which enables cross-viral comparisons of glycosylation to be made. We report here the site-specific glycosylation at each of the 22 N-linked glycan sites found on the SARS-CoV-2 S protomer. As observed on other viral glycoproteins, there is an elevation in oligomannose- and hybrid-type glycans compared to host-derived glycoproteins, although compared to many other viruses there are still a large population of complex-type glycans displayed across the trimer surface. We also report that for each of the 22 glycan sites the occupancy is nearly fully complete, which means that any epitopes shielded from the immune system on the virus will also likely be shielded on the immunogen. Site-specific glycan analysis of SARS-CoV-2 immunogens will help guide vaccine design and manufacturing.</p></div><div class="section" id="sec-2"><h2 class="">Results and discussion</h2><div id="sec-3" class="subsection"><h3>Localized impairment to SARS-CoV-2 S glycan maturation</h3><p id="p-10">To resolve the site-specific glycosylation of SARS-CoV-2 S protein and visualize the distribution of glycoforms across the protein surface, we expressed and purified recombinant soluble material in an identical manner to that which was used to obtain the high-resolution cryo-electron microscopy (cryo-EM) structure, albeit without glycan processing blockade using kifunensine<sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup>. This soluble recombinant variant of the S protein contains all 22 glycans on the SARS-CoV-2 S protein (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a></strong>). Stabilization of the trimeric prefusion structure was achieved using the “2P” stabilizing mutations<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> at residues 986 and 987 in addition to a C-terminal trimerization motif. This ensures that the quaternary structure remains intact during glycan processing, as in the case of HIV Env mimetics, this is known to influence glycosylation of certain sites<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. Prior to analysis, supernatant containing the recombinant SARS-CoV-2 was purified using a C-terminal StrepTag followed by size-exclusion chromatography to ensure only native-like trimeric SARS-CoV-2 S protein is analyzed (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a></strong>). The trimeric conformation of the purified material was validated using negative stain electron microscopy (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a></strong>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression and validation of SARS-CoV-2 S glycoprotein. (A) Schematic representation of SARS-CoV-2 S glycoprotein. The positions of N-linked glycosylation sequons (N-X-S/T, where X&#x2260;P) are shown as branches. Protein domains are illustrated: N-terminal domain (NTD), receptor-binding domain (RBD), fusion peptide (FP), heptad repeat 1 (HR1), central helix (CH), connector domain (CD), and transmembrane domain (TM). (B) SDS-PAGE analysis of SARS-CoV-2 S protein. Lane 1: filtered supernatant from transfected cells; lane 2: flowthrough from StrepTactin resin; lane 3: wash from StrepTactin resin; lane 4: elution from StrepTactin resin. (C) Negative-stain EM 2D class averages of the SARS-CoV-2 S protein. 2D class averages of the SARS-CoV-2 S protein are shown, confirming that the protein adopts the trimeric prefusion conformation matching the material used to determine the structure18." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523994860" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Expression and validation of SARS-CoV-2 S glycoprotein.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Schematic representation of SARS-CoV-2 S glycoprotein. The positions of N-linked glycosylation sequons (N-X-S/T, where X&#x2260;P) are shown as branches. Protein domains are illustrated: N-terminal domain (NTD), receptor-binding domain (RBD), fusion peptide (FP), heptad repeat 1 (HR1), central helix (CH), connector domain (CD), and transmembrane domain (TM). &lt;strong&gt;(B)&lt;/strong&gt; SDS-PAGE analysis of SARS-CoV-2 S protein. Lane 1: filtered supernatant from transfected cells; lane 2: flowthrough from StrepTactin resin; lane 3: wash from StrepTactin resin; lane 4: elution from StrepTactin resin. &lt;strong&gt;(C)&lt;/strong&gt; Negative-stain EM 2D class averages of the SARS-CoV-2 S protein. 2D class averages of the SARS-CoV-2 S protein are shown, confirming that the protein adopts the trimeric prefusion conformation matching the material used to determine the structure&lt;sup&gt;18&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F1.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F1.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Expression and validation of SARS-CoV-2 S glycoprotein.</span><p id="p-11" class="first-child"><strong>(A)</strong> Schematic representation of SARS-CoV-2 S glycoprotein. The positions of N-linked glycosylation sequons (N-X-S/T, where X≠P) are shown as branches. Protein domains are illustrated: N-terminal domain (NTD), receptor-binding domain (RBD), fusion peptide (FP), heptad repeat 1 (HR1), central helix (CH), connector domain (CD), and transmembrane domain (TM). <strong>(B)</strong> SDS-PAGE analysis of SARS-CoV-2 S protein. Lane 1: filtered supernatant from transfected cells; lane 2: flowthrough from StrepTactin resin; lane 3: wash from StrepTactin resin; lane 4: elution from StrepTactin resin. <strong>(C)</strong> Negative-stain EM 2D class averages of the SARS-CoV-2 S protein. 2D class averages of the SARS-CoV-2 S protein are shown, confirming that the protein adopts the trimeric prefusion conformation matching the material used to determine the structure<sup><a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a></sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Trypsin, chymotrypsin, and alpha-lytic protease were employed to generate three glycopeptide samples. These proteases cleave at different sites and were selected in order to generate glycopeptides that contain a single N-linked glycan sequon. The glycopeptide pools were analyzed by LC-MS, with high-energy collision-induced dissociation (HCD) fragmentation, and the glycan compositions at each site were determined for all 22 N-linked glycan sites on the SARS-CoV-2 S protein (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a></strong>). A diverse range of glycan compositions were observed across the different glycosylation sites. In order to convey the main processing features at each site, the abundances of each glycan are summed into oligomannose-, hybrid- and complex-type glycosylation. In addition, the diverse signals arising from heterogeneous complex-type glycosylation are simplified by the summation of glycan intensities into a more limited range of structural categories.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Site-specific N-linked glycosylation of SARS-CoV-2 S glycoprotein. The schematic illustrates the color code for the principal glycan types that can arise along the maturation pathway from oligomannose-, hybrid-to complex-type glycans. The graphs summarize quantitative mass spectrometric analysis of the glycan population present at individual N-linked glycosylation sites. The bar graphs represent the quantities of each glycan group with oligomannose-type glycan series (M9 to M5; Man9GlcNAc2 to Man5GlcNAc2) (green), afucosylated and fucosylated hybrid glycans (Hybrid &amp; F Hybrid) (dashed pink), and complex glycans grouped according to the number of antennae and presence of core fucosylation (A1 to FA4) (pink). Left to right; least processed to most processed. The pie charts summarize the quantification of these glycans. Glycan sites are colored according to oligomannose-type glycan content with the glycan sites labelled in green (80&#x2212;100%), orange (30&#x2212;79%) and pink (0&#x2212;29%)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523994860" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Site-specific N-linked glycosylation of SARS-CoV-2 S glycoprotein.&lt;/span&gt; The schematic illustrates the color code for the principal glycan types that can arise along the maturation pathway from oligomannose-, hybrid-to complex-type glycans. The graphs summarize quantitative mass spectrometric analysis of the glycan population present at individual N-linked glycosylation sites. The bar graphs represent the quantities of each glycan group with oligomannose-type glycan series (M9 to M5; Man&lt;sub&gt;9&lt;/sub&gt;GlcNAc&lt;sub&gt;2&lt;/sub&gt; to Man&lt;sub&gt;5&lt;/sub&gt;GlcNAc&lt;sub&gt;2&lt;/sub&gt;) (green), afucosylated and fucosylated hybrid glycans (Hybrid &amp; F Hybrid) (dashed pink), and complex glycans grouped according to the number of antennae and presence of core fucosylation (A1 to FA4) (pink). Left to right; least processed to most processed. The pie charts summarize the quantification of these glycans. Glycan sites are colored according to oligomannose-type glycan content with the glycan sites labelled in green (80&#x2212;100%), orange (30&#x2212;79%) and pink (0&#x2212;29%).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F2.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F2.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Site-specific N-linked glycosylation of SARS-CoV-2 S glycoprotein.</span><p id="p-13" class="first-child">The schematic illustrates the color code for the principal glycan types that can arise along the maturation pathway from oligomannose-, hybrid-to complex-type glycans. The graphs summarize quantitative mass spectrometric analysis of the glycan population present at individual N-linked glycosylation sites. The bar graphs represent the quantities of each glycan group with oligomannose-type glycan series (M9 to M5; Man<sub>9</sub>GlcNAc<sub>2</sub> to Man<sub>5</sub>GlcNAc<sub>2</sub>) (green), afucosylated and fucosylated hybrid glycans (Hybrid &amp; F Hybrid) (dashed pink), and complex glycans grouped according to the number of antennae and presence of core fucosylation (A1 to FA4) (pink). Left to right; least processed to most processed. The pie charts summarize the quantification of these glycans. Glycan sites are colored according to oligomannose-type glycan content with the glycan sites labelled in green (80−100%), orange (30−79%) and pink (0−29%).</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">There are three sites on SARS-CoV-2 that are predominantly oligomannose-type: N234, N709 and N801. The predominant structure observed at each site, with the exception of N234, is Man5GlcNAc2, which demonstrates that these sites are largely accessible to a α1,2-mannosidases but are poor substrates for GlcNAcT-I, which is the gateway enzyme in the formation of hybrid- and complex-type glycans in the Golgi apparatus. The stage at which processing is impeded is a signature related to the density and presentation of glycans on the viral spike. For example, the more densely glycosylated spikes of HIV-1 Env and Lassa virus GPC give rise to numerous sites dominated by Man<sub>9</sub>GlcNAc<sub>2</sub><sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>–<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>.</p><p id="p-15">Interestingly, there are several sites which possess significant populations of hybrid-type glycans, most notably at N657. This phenomenon has been observed on other viral glycoproteins, such as HIV-1 Env<sup><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a></sup>, and these structures are not particularly prevalent on mammalian glycoproteins<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. Such hybrid-type glycans have been shown to be targeted by anti-HIV antibodies<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup> and could also be important for immunogen trafficking since they have mannose-terminating moieties<sup><a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup>.</p><p id="p-16">A mixture of oligomannose-type glycans and complex-type glycans can be found at sites N61, N122, N165, N603, N657, N717 and N1074 (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a></strong>). Of the 22 sites on the S protein, 10 contain significant populations of oligomannose-type glycans, highlighting how the processing of the SARS-CoV-2 S glycans is divergent from host glycoproteins. The remaining 12 sites are dominated by processed, complex-type glycans. The predominant category of complex-type glycans observed on the S protein are fucosylated biantennary structures, very similar to those abundant on mammalian glycoproteins.</p><p id="p-17">A further feature that can be interrogated using this methodology is the extent of unoccupancy of glycosylation sites, where a sequon is present but a glycan has not been attached. In HIV immunogen research, the holes generated by unoccupied glycan sites have been shown to be immunogenic and potentially give rise to distracting epitopes<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. While unoccupied glycosylation sites were detected on SARS-CoV-2, when quantified they were revealed to form a very minor component of the total peptide pool. The efficiency of glycan site occupancy of the recombinant SARS-CoV-2 immunogen versus the counterpart for HIV may well arise due to the somewhat larger spacing of sites along the polypeptide chain. The high occupancy of N-linked glycan sequons of SARS-CoV-2 indicates that recombinant immunogens will not require further optimization to enhance site occupancy.</p></div><div id="sec-4" class="subsection"><h3>Fully glycosylated model of the SARS-CoV-2 spike</h3><p id="p-18">Using the cryo-EM structure of the trimeric SARS-CoV-2 S protein (PDB ID 6VSB)<sup><a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">18</a></sup>, we generated a model to map the glycosylation status of the coronavirus spike mimetic onto the experimentally determined 3D structure (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a></strong>). This combined mass spectrometric and cryo-EM analysis reveals how the N-linked glycans occlude distinct regions across the surface of the SARS-CoV-2 spike. Glycans were modelled onto each site using predominant glycans observed (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a></strong>) and colored according to the prevalence of oligomannose-type glycans.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure-based mapping of SARS-CoV-2 S N-linked glycans. The modelling of experimentally observed glycosylation site compositions is illustrated on the prefusion structure of trimeric SARS-CoV-2 S glycoprotein (PDB ID 6VSB)18, with one RBD in the &#x201C;up&#x201D; conformation and the other two RBDs in the &#x201C;down&#x201D; conformation. The glycans are colored according to oligomannose content as defined by the key. ACE2 receptor binding sites are highlighted in light blue. The S1 and S2 subunits are rendered with translucent surface representation, colored light and dark grey, respectively. Note that the flexible loops on which N74 and N149 glycan sites reside are represented as dashed lines with glycan sites on the loops mapped at approximate regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523994860" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Structure-based mapping of SARS-CoV-2 S N-linked glycans.&lt;/span&gt; The modelling of experimentally observed glycosylation site compositions is illustrated on the prefusion structure of trimeric SARS-CoV-2 S glycoprotein (PDB ID 6VSB)&lt;sup&gt;18&lt;/sup&gt;, with one RBD in the &#x201C;up&#x201D; conformation and the other two RBDs in the &#x201C;down&#x201D; conformation. The glycans are colored according to oligomannose content as defined by the key. ACE2 receptor binding sites are highlighted in light blue. The S1 and S2 subunits are rendered with translucent surface representation, colored light and dark grey, respectively. Note that the flexible loops on which N74 and N149 glycan sites reside are represented as dashed lines with glycan sites on the loops mapped at approximate regions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F3.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F3.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Structure-based mapping of SARS-CoV-2 S N-linked glycans.</span><p id="p-19" class="first-child">The modelling of experimentally observed glycosylation site compositions is illustrated on the prefusion structure of trimeric SARS-CoV-2 S glycoprotein (PDB ID 6VSB)<sup><a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">18</a></sup>, with one RBD in the “up” conformation and the other two RBDs in the “down” conformation. The glycans are colored according to oligomannose content as defined by the key. ACE2 receptor binding sites are highlighted in light blue. The S1 and S2 subunits are rendered with translucent surface representation, colored light and dark grey, respectively. Note that the flexible loops on which N74 and N149 glycan sites reside are represented as dashed lines with glycan sites on the loops mapped at approximate regions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Shielding of the receptor binding sites on the SARS-CoV-2 spike by proximal glycosylation sites (N165, N234, N343) can be observed, especially when the receptor binding domain is in the “down” conformation. The shielding of receptor binding sites by glycans is a common feature of viral glycoproteins and has been observed for SARS-CoV S<sup><a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a></sup>, HIV-1 Env<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>, influenza HA<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>, and LASV GPC<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. Given the functional constraints of receptor binding sites and the subsequent low mutation rates of these residues, it is likely that there has been selective pressure to utilize N-linked glycans as a method to camouflage one of the most conserved and potentially vulnerable areas of their respective glycoproteins<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>.</p><p id="p-21">It is interesting to note the absence of a specific glycan cluster that is responsible for the presence of the oligomannose-type glycans but rather there is a dispersion of these glycans across both the S1 and S2 subunits. This is in significant contrast to other viral glycoproteins, for example the density of glycans clusters in HIV have even enabled structural classification of their different modes of interaction<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. In SARS-CoV-2 the oligomannose-type structures are probably protected, to some extent, by the protein component, as exemplified by the N234 glycan which is partially sandwiched between the N-terminal and receptor-binding domains (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a></strong>).</p><p id="p-22">We characterized the N-linked glycans on extended loop structures that were not resolved in the cryo-EM maps<sup><a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">18</a></sup> (N74 and N149). These were determined to be complex-type glycans, consistent with the inherent flexibility of these regions and resulting accessibility of these residues to glycan processing enzymes.</p><p id="p-23">In addition to the site-specific glycosylation of the SARS-CoV-2 S protein it is also important to consider overall trends in glycosylation across the glycoprotein. The averaged compositions across all 22 glycan sites reveals that the two most common type of N-glycans on the protein are Man<sub>5</sub>GlcNAc<sub>2</sub> (M5) and fucosylated biantennary (FA2/FA1B) glycans (<strong>Sup. Fig. 1</strong>). Oligomannose-type glycans comprise 32% of the total glycan pool, with hybrid-type and complex-type glycans comprising 7% and 62%, respectively (<strong>Sup. Fig. 1</strong>). Despite the potential impact of different local protein structure on glycan processing, the overall glycosylation of SARS-CoV-2 is comparable with SARS-CoV-1 S protein and other coronavirus S proteins<sup><a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup>. We have previously reported that a recombinant SARS-CoV-1 S mimetic also contained 32% oligomannose-type glycans showing a remarkable conservation in glycan processing across these coronaviruses. Whilst the oligomannose-type glycan content is well above that observed on typical host glycoproteins, it is significantly lower than is found on other viral glycoproteins. For example, one of the most densely glycosylated viral spike proteins is HIV-1 Env, which contains ~60% oligomannose-type glycans<sup><a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>. This suggests that SARS-CoV-2 S protein is less densely glycosylated and that the glycans form much less of a shield compared with other viral glycoproteins including HIV Env and LASV GPC, which may be beneficial for the elicitation of potent neutralizing antibodies.</p><p id="p-24">In addition to oligomannose-type glycans, the processing of complex-type glycans is an important consideration in immunogen engineering. Across the 22 N-linked glycosylation sites, 16% of the glycans contain at least one sialic acid residue and 48% are fucosylated (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a></strong>). These data suggest high levels of fucosylation but low levels of sialylation, considering complex and hybrid type glycans make up 69% of the total glycans of SARS-CoV-2. Understanding the distribution of glycan modifications across the viral spike illustrates the differential susceptibility to different processing enzymes while the absolute levels can be heavily influenced by the cellular expression system utilized. We have previously demonstrated for HIV-1 Env glycosylation that the processing of complex-type glycans is driven by the producer cell but that the levels of oligomannose-type glycans were largely independent of the expression system and is much more closely related to the protein structure and glycan density<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Glycosylated model of SARS-CoV-2 S glycoprotein highlighting different abundances of glycan modifications. The modelling of experimentally observed glycans, illustrated on the prefusion structure of trimeric SARS-CoV-2 S glycoprotein (PDB ID 6VSB)18, are highlighted according abundances of (A) mannosylation (Man5-9GlcNAc2), (B) fucosylation of the protein-proximal GlcNAc residue and (C) terminal sialylation. S1 and S2 subunits are colored light grey and dark grey, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-523994860" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Glycosylated model of SARS-CoV-2 S glycoprotein highlighting different abundances of glycan modifications.&lt;/span&gt; The modelling of experimentally observed glycans, illustrated on the prefusion structure of trimeric SARS-CoV-2 S glycoprotein (PDB ID 6VSB)&lt;sup&gt;18&lt;/sup&gt;, are highlighted according abundances of &lt;strong&gt;(A)&lt;/strong&gt; mannosylation (Man&lt;sub&gt;5-9&lt;/sub&gt;GlcNAc&lt;sub&gt;2&lt;/sub&gt;), &lt;strong&gt;(B)&lt;/strong&gt; fucosylation of the protein-proximal GlcNAc residue and &lt;strong&gt;(C)&lt;/strong&gt; terminal sialylation. S1 and S2 subunits are colored light grey and dark grey, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F4.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F4.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.26.010322/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Glycosylated model of SARS-CoV-2 S glycoprotein highlighting different abundances of glycan modifications.</span><p id="p-25" class="first-child">The modelling of experimentally observed glycans, illustrated on the prefusion structure of trimeric SARS-CoV-2 S glycoprotein (PDB ID 6VSB)<sup><a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">18</a></sup>, are highlighted according abundances of <strong>(A)</strong> mannosylation (Man<sub>5-9</sub>GlcNAc<sub>2</sub>), <strong>(B)</strong> fucosylation of the protein-proximal GlcNAc residue and <strong>(C)</strong> terminal sialylation. S1 and S2 subunits are colored light grey and dark grey, respectively.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-5"><h2 class="">Perspectives</h2><p id="p-26">Our glycosylation analysis of SARS-CoV-2 offers a detailed benchmark of site-specific glycan signatures characteristic of a natively folded trimeric spike. As an increasing number of glycoprotein-based vaccine candidates are being developed, their detailed glycan analysis offers a route for comparing immunogen integrity and will also be important to monitor as manufacturing processes are scaled for clinical use. Glycosylation will therefore also be an important measure of antigen quality in the manufacture of serological testing kits, particularly as some S protein fragments may offer advantages in terms of production yield but lack effective glycan mimicry of the natively folded trimeric spike. The lower levels of mannose-terminating glycans on SARS-CoV-2 compared to many other viral spikes may indicate that glycan engineering should be considered in the scenario where first-generation glycoprotein-based vaccine candidates be poorly immunogenic. Finally, with the advent of nucleotide-based vaccines it will be important to understand how those delivery mechanisms impact immunogen processing and presentation.</p></div><div class="section" id="sec-6"><h2 class="">Materials and Methods</h2><div id="sec-7" class="subsection"><h3>Protein expression and purification</h3><p id="p-27">To express the prefusion S ectodomain, a gene encoding residues 1−1208 of SARS-CoV-2 S (GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom">MN908947</a>) with proline substitutions at residues 986 and 987, a “GSAS” substitution at the furin cleavage site (residues 682–685), a C-terminal T4 fibritin trimerization motif, an HRV3C protease cleavage site, a TwinStrepTag and an 8XHisTag was synthesized and cloned into the mammalian expression vector pαH. This expression vector was used to transiently transfect FreeStyle293F cells (Thermo Fisher) using polyethylenimine. Protein was purified from filtered cell supernatants using StrepTactin resin (IBA) before being subjected to additional purification by size-exclusion chromatography using a Superose 6 10/300 column (GE Healthcare) in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN<sub>3</sub>.</p></div><div id="sec-8" class="subsection"><h3>Negative-stain electron microscopy and 2D class averaging</h3><p id="p-28">Purified SARS-CoV-2 spike was diluted to a concentration of 0.04 mg/mL using 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN<sub>3</sub> before being applied to a plasma cleaned CF400-Cu grid (Electron Microscopy Sciences). Protein was then stained using methylamine tungstate (Nanoprobes) before being allowed to dry at room temperature for 15 minutes. This grid was imaged in a Talos TEM (Thermo Fisher Scientific) equipped with a Ceta 16M detector. Micrographs were collected using TIA v4.14 software at a nominal magnification of 92,000×, corresponding to a calibrated pixel size of 1.63 Å/pix. CTF estimation, particle picking and 2D class averaging were performed using <em>cis</em>TEM<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>.</p></div><div id="sec-9" class="subsection"><h3>Glycopeptide analysis by mass spectrometry</h3><p id="p-29">Three 30 μg aliquots of SARS-CoV-2 S protein were denatured for 1h in 50 mM Tris/HCl, pH 8.0 containing 6 M of urea and 5 mM dithiothreitol (DTT). Next, the S protein were reduced and alkylated by adding 20 mM iodoacetamide (IAA) and incubated for 1h in the dark, followed by a 1h incubation with 20 mM DTT to eliminate residual IAA. The alkylated Env proteins were buffer-exchanged into 50 mM Tris/HCl, pH 8.0 using Vivaspin columns (3 kDa) and digested separately overnight using trypsin chymotrypsin or alpha lytic protease (Mass Spectrometry Grade, Promega) at a ratio of 1:30 (w/w). The next day, the peptides were dried and extracted using C18 Zip-tip (MerckMilipore). The peptides were dried again, re-suspended in 0.1% formic acid and analyzed by nanoLC-ESI MS with an Easy-nLC 1200 (Thermo Fisher Scientific) system coupled to a Fusion mass spectrometer (Thermo Fisher Scientific) using higher energy collision-induced dissociation (HCD) fragmentation. Peptides were separated using an EasySpray PepMap RSLC C18 column (75 μm × 75 cm). A trapping column (PepMap 100 C18 3μM 75μM x 2cm) was used in line with the LC prior to separation with the analytical column. The LC conditions were as follows: 275 minute linear gradient consisting of 0-32% acetonitrile in 0.1% formic acid over 240 minutes followed by 35 minutes of 80% acetonitrile in 0.1% formic acid. The flow rate was set to 200 nL/min. The spray voltage was set to 2.7 kV and the temperature of the heated capillary was set to 40 °C. The ion transfer tube temperature was set to 275 °C. The scan range was 400−1600 m/z. The HCD collision energy was set to 50%, appropriate for fragmentation of glycopeptide ions. Precursor and fragment detection were performed using an Orbitrap at a resolution MS<sup>1</sup>= 100,000. MS<sup>2</sup>= 30,000. The AGC target for MS<sup>1</sup>=4e<sup>5</sup> and MS<sup>2</sup>=5e<sup>4</sup> and injection time: MS<sup>1</sup>=50ms MS<sup>2</sup>=54ms.</p><p id="p-30">Glycopeptide fragmentation data were extracted from the raw file using Byonic™ (Version 3.5) and Byologic™ software (Version 3.5; Protein Metrics Inc.). The glycopeptide fragmentation data were evaluated manually for each glycopeptide; the peptide was scored as true-positive when the correct b and y fragment ions were observed along with oxonium ions corresponding to the glycan identified. The MS data was searched using the Protein Metrics 305 N-glycan library. The relative amounts of each glycan at each site as well as the unoccupied proportion were determined by comparing the extracted chromatographic areas for different glycotypes with an identical peptide sequence. All charge states for a single glycopeptide were summed. The precursor mass tolerance was set at 4 ppm and 10 ppm for fragments. A 1% false discovery rate (FDR) was applied. The relative amounts of each glycan at each site as well as the unoccupied proportion were determined by comparing the extracted ion chromatographic areas for different glycopeptides with an identical peptide sequence. Glycans were categorized according to the composition detected. HexNAc(2)Hex(9−5) was classified as M9 to M5. HexNAc(3)Hex(5−6)X was classified as Hybrid with HexNAc(3)Fuc(1)X classified as Fhybrid. Complex-type glycans were classified according to the number of processed antenna and fucosylation. If all of the following compositions have a fucose they are assigned into the FA categories. HexNAc(3)Hex(3-4)X is assigned as A1, HexNAc(4)X is A2/A1B, HexNAc(5)X is A3/A2B, and HexNAc(6)X is A4/A3B. As this fragmentation method does not provide linkage information compositional isomers are group, so for example a triantennary glycan contains HexNAc 5 but so does a biantennary glycans with a bisect. Any glycan containing at least one sialic acid was counted as sialylated.</p></div><div id="sec-10" class="subsection"><h3>Model construction</h3><p id="p-31">Structural models of N-linked glycan presentation on SARS-CoV-2 were created using electron microscopy structures (PDB ID: 6VSB) along with complex-, hybrid-, and oligomannose-type N-linked glycans (PDB ID 4BYH, 4B7I, and 2WAH). The most dominant glycoform presented at each site was modelled on to the N-linked carbohydrate attachment sites in Coot<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-32">We thank M. Dixon and M. Gowland-Pryde for supporting our work on this project during the difficulties arising from the pandemic, and G. Ould for critical reading of the manuscript. This work was funded by the International AIDS Vaccine Initiative, Bill and Melinda Gates Foundation through the Collaboration for AIDS Discovery (OPP1084519 to M.C., and 1196345 to M.C.), the NIAID (R01-AI127521 to J.S.M) and the Scripps Consortium for HIV Vaccine Development (CHAVD) (AI144462 to M.C.).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">6736</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D6736%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.2" data-doi="10.1016/S2213-2600(20)30079-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study</span>. <abbr class="cit-jnl-abbrev">Lancet Respir. Med.</abbr> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S2213-2600(20)30079-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BRespir.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS2213-2600%252820%252930079-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S2213-2600(20)30079-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.3" data-doi="10.1038/s41564-020-0688-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Letko</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marzi</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Munster</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses</span>. <abbr class="cit-jnl-abbrev">Nat. Microbiol.</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41564-020-0688-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41564-020-0688-y%26rft_id%253Dinfo%253Apmid%252F32094589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41564-020-0688-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32094589&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.4" data-doi="10.1101/2020.02.20.957472"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Vulnerabilities in coronavirus glycan shields despite extensive glycosylation</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">2020.02.20.957472</span> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.20.957472</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.20.957472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMi4yMC45NTc0NzJ2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjYuMDEwMzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowden</span>,  <span class="cit-name-given-names">T. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">I. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Crispin</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Exploitation of glycosylation in enveloped virus pathobiology</span>. <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1863</span>, <span class="cit-fpage">1480</span>–<span class="cit-lpage">1497</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1863%26rft.spage%253D1480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.26.010322v1.6" data-doi="10.1126/science.1235681"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dalziel</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crispin</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scanlan</span>,  <span class="cit-name-given-names">C. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zitzmann</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dwek</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-article-title">Emerging principles for the therapeutic exploitation of glycosylation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>, <span class="cit-fpage">1235681</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDalziel%26rft.auinit1%253DM.%26rft.volume%253D343%26rft.issue%253D6166%26rft.spage%253D1235681%26rft.epage%253D1235681%26rft.atitle%253DEmerging%2BPrinciples%2Bfor%2Bthe%2BTherapeutic%2BExploitation%2Bof%2BGlycosylation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1235681%26rft_id%253Dinfo%253Apmid%252F24385630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDMvNjE2Ni8xMjM1NjgxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAzMjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.7" data-doi="10.1038/nature05818"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scanlan</span>,  <span class="cit-name-given-names">C. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Offer</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zitzmann</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dwek</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-article-title">Exploiting the defensive sugars of HIV-1 for drug and vaccine design</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">446</span>, <span class="cit-fpage">1038</span>–<span class="cit-lpage">1045</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DScanlan%26rft.auinit1%253DC.%2BN.%26rft.volume%253D446%26rft.issue%253D7139%26rft.spage%253D1038%26rft.epage%253D1045%26rft.atitle%253DExploiting%2Bthe%2Bdefensive%2Bsugars%2Bof%2BHIV-1%2Bfor%2Bdrug%2Band%2Bvaccine%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05818%26rft_id%253Dinfo%253Apmid%252F17460665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05818&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17460665&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245950400041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Unexpected receptor functional mimicry elucidates activation of coronavirus fusion</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">1026-1</span>–<span class="cit-lpage">39.e5</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.9" data-doi="10.1073/pnas.1908898117"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">T. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM analysis of a feline coronavirus spike protein reveals a unique structure and camouflaging glycans</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A.</abbr> <span class="cit-vol">117</span>, <span class="cit-fpage">1438</span>–<span class="cit-lpage">1446</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1908898117%26rft_id%253Dinfo%253Apmid%252F31900356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE3LzMvMTQzOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjYuMDEwMzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.10"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Safety and Immunogenicity Study of 2019-nCoV Vaccine (mRNA-1273) to Prevent SARS-CoV-2 Infection - Full Text View - ClinicalTrials.gov</span>. Available at: <a href="https://clinicaltrials.gov/ct2/show/NCT04283461?term=mrna-1273&amp;draw=2&amp;rank=1">https://clinicaltrials.gov/ct2/show/NCT04283461?term=mrna-1273&amp;draw=2&amp;rank=1</a>. (Accessed: <span class="cit-date-in-citation">25th March 2020</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.11" data-doi="10.1016/j.virol.2006.11.027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stertz</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The intracellular sites of early replication and budding of SARS-coronavirus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">361</span>, <span class="cit-fpage">304</span>–<span class="cit-lpage">315</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DStertz%26rft.auinit1%253DS.%26rft.volume%253D361%26rft.issue%253D2%26rft.spage%253D304%26rft.epage%253D315%26rft.atitle%253DThe%2Bintracellular%2Bsites%2Bof%2Bearly%2Breplication%2Band%2Bbudding%2Bof%2BSARS-coronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2006.11.027%26rft_id%253Dinfo%253Apmid%252F17210170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2006.11.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17210170&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-book no-rev-xref" id="cit-2020.03.26.010322v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">A. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-chapter-title">Coronaviruses: An overview of their replication and pathogenesis</span>. in <span class="cit-source">Coronaviruses: Methods and Protocols</span> pp <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span> (<span class="cit-publ-name">Springer</span> <span class="cit-publ-loc">New York</span>, <span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.26.010322v1.13" data-doi="10.1099/vir.0.19505-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">M. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">S. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">See</span>,  <span class="cit-name-given-names">E. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ooi</span>,  <span class="cit-name-given-names">E. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ling</span>,  <span class="cit-name-given-names">A. E.</span></span> <span class="cit-article-title">Proliferative growth of SARS coronavirus in Vero E6 cells</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">3291</span>–<span class="cit-lpage">3303</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DNg%26rft.auinit1%253DM.-L.%26rft.volume%253D84%26rft.issue%253D12%26rft.spage%253D3291%26rft.epage%253D3303%26rft.atitle%253DProliferative%2Bgrowth%2Bof%2BSARS%2Bcoronavirus%2Bin%2BVero%2BE6%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19505-0%26rft_id%253Dinfo%253Apmid%252F14645910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.19505-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14645910&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220465300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Venkatagopalan</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daskalova</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolezal</span>,  <span class="cit-name-given-names">K. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hogue</span>,  <span class="cit-name-given-names">B. G.</span></span> <span class="cit-article-title">Coronavirus envelope (E) protein remains at the site of assembly</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">478</span>, <span class="cit-fpage">75</span>–<span class="cit-lpage">85</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D478%26rft.spage%253D75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of N-linked carbohydrates from severe acute respiratory syndrome (SARS) spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">399</span>, <span class="cit-fpage">257</span>–<span class="cit-lpage">69</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D399%26rft.spage%253D257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Behrens</span>,  <span class="cit-name-given-names">A.-J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular architecture of the cleavage-dependent mannose patch on a soluble HIV-1 envelope glycoprotein trimer</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">91</span>, <span class="cit-fpage">e01894</span>–<span class="cit-lpage">16</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft.volume%253D91%26rft.spage%253De01894%26rft_id%253Dinfo%253Apmid%252F27807235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=27807235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.17" data-doi="10.1126/science.aat9120"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tokatlian</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Innate immune recognition of glycans targets HIV nanoparticle immunogens to germinal centers</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">363</span>, <span class="cit-fpage">649</span>–<span class="cit-lpage">654</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aat9120%26rft_id%253Dinfo%253Apmid%252F30573546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjMvNjQyNy82NDkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI2LjAxMDMyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.18" data-doi="10.1126/science.abb2507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.abb2507</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4My8xMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAzMjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.19" data-doi="10.1016/j.cell.2020.02.058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2020.02.058</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.02.058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.02.058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.20" data-doi="10.1016/j.celrep.2018.07.080"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Struwe</span>,  <span class="cit-name-given-names">W. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Site-specific glycosylation of virion-derived HIV-1 Env Is mimicked by a soluble trimeric immunogen</span>. <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1958</span>–<span class="cit-lpage">1966.e5</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D24%26rft.spage%253D1958%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2018.07.080%26rft_id%253Dinfo%253Apmid%252F30134158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2018.07.080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30134158&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.21" data-doi="10.1073/pnas.1803990115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structure of the Lassa virus glycan shield provides a model for immunological resistance</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr> <span class="cit-vol">115</span>, <span class="cit-fpage">7320</span>–<span class="cit-lpage">7325</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1803990115%26rft_id%253Dinfo%253Apmid%252F29941589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzI4LzczMjAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI2LjAxMDMyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.22" data-doi="10.1016/j.celrep.2016.02.058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Behrens</span>,  <span class="cit-name-given-names">A.-J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Composition and antigenic effects of individual glycan sites of a trimeric HIV-1 envelope glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">2695</span>–<span class="cit-lpage">2706</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D14%26rft.spage%253D2695%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2016.02.058%26rft_id%253Dinfo%253Apmid%252F26972002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2016.02.058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26972002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andrabi</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The chimpanzee SIV envelope trimer: structure and deployment as an HIV vaccine template</span>. <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">2426</span>–<span class="cit-lpage">2441.e6</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D27%26rft.spage%253D2426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.24" data-doi="10.1073/pnas.1707304114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pallesen</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A.</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">E7348</span>–<span class="cit-lpage">E7357</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1707304114%26rft_id%253Dinfo%253Apmid%252F28807998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzM1L0U3MzQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAzMjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.25" data-doi="10.1016/j.celrep.2015.05.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural constraints determine the glycosylation of HIV-1 envelope trimers</span>. <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">1604</span>–<span class="cit-lpage">1613</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D11%26rft.spage%253D1604%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.05.017%26rft_id%253Dinfo%253Apmid%252F26051934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.05.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26051934&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.26" data-doi="10.1038/srep32956"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Panico</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mapping the complete glycoproteome of virion-derived HIV-1 gp120 provides insights into broadly neutralizing antibody binding</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">32956</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D32956%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep32956%26rft_id%253Dinfo%253Apmid%252F27604319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep32956&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27604319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.27" data-doi="10.1021/pr100528k"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stumpo</span>,  <span class="cit-name-given-names">K. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reinhold</span>,  <span class="cit-name-given-names">V. N.</span></span> <span class="cit-article-title">The N-glycome of human plasma</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res.</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">4823</span>–<span class="cit-lpage">4830</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DStumpo%26rft.auinit1%253DK.%2BA.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D4823%26rft.epage%253D4830%26rft.atitle%253DThe%2BN-glycome%2Bof%2Bhuman%2Bplasma.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr100528k%26rft_id%253Dinfo%253Apmid%252F20690605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr100528k&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20690605&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281443700045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.28" data-doi="10.1021/pr800737u"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Knežević</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Variability, heritability and environmental determinants of human plasma n-glycome</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res.</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">694</span>–<span class="cit-lpage">701</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DKnez%25CC%258Cevic%25CC%2581%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D694%26rft.epage%253D701%26rft.atitle%253DVariability%252C%2Bheritability%2Band%2Benvironmental%2Bdeterminants%2Bof%2Bhuman%2Bplasma%2BN-glycome.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr800737u%26rft_id%253Dinfo%253Apmid%252F19035662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr800737u&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19035662&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263193300030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shivatare</span>,  <span class="cit-name-given-names">V. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Unprecedented role of hybrid N-Glycans as ligands for HIV-1 broadly neutralizing antibodies</span>. <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc.</abbr> <span class="cit-vol">140</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.30" data-doi="10.1016/j.immuni.2018.07.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bianchi</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Electron-Microscopy-Based Epitope Mapping Defines Specificities of Polyclonal Antibodies Elicited during HIV-1 BG505 Envelope Trimer Immunization</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">49</span>, <span class="cit-fpage">288</span>–<span class="cit-lpage">300.e8</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D49%26rft.spage%253D288%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2018.07.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2018.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.31" data-doi="10.1126/science.1234150"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jardine</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Rational HIV immunogen design to target specific germline B cell receptors</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">340</span>, <span class="cit-fpage">711</span>–<span class="cit-lpage">716</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1234150%26rft_id%253Dinfo%253Apmid%252F23539181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDAvNjEzMy83MTEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI2LjAxMDMyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.32" data-doi="10.1126/scitranslmed.3000799"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">C.-J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cross-neutralization of 1918 and 2009 Influenza viruses: role of glycans in viral evolution and vaccine design</span>. <abbr class="cit-jnl-abbrev">Sci. Transl. Med.</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">24ra21</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BTranslational%2BMedicine%26rft.stitle%253DSci%2BTransl%2BMed%26rft.aulast%253DWei%26rft.auinit1%253DC.-J.%26rft.volume%253D2%26rft.issue%253D24%26rft.spage%253D24ra21%26rft.epage%253D24ra21%26rft.atitle%253DCross-Neutralization%2Bof%2B1918%2Band%2B2009%2BInfluenza%2BViruses%253A%2BRole%2Bof%2BGlycans%2Bin%2BViral%2BEvolution%2Band%2BVaccine%2BDesign%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscitranslmed.3000799%26rft_id%253Dinfo%253Apmid%252F20375007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InNjaXRyYW5zbWVkIjtzOjU6InJlc2lkIjtzOjExOiIyLzI0LzI0cmEyMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjYuMDEwMzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.33" data-doi="10.1126/science.1186430"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural basis of preexisting immunity to the 2009 H1N1 pandemic influenza virus</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>, <span class="cit-fpage">357</span>–<span class="cit-lpage">360</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1186430%26rft_id%253Dinfo%253Apmid%252F20339031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjgvNTk3Ni8zNTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI2LjAxMDMyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.34" data-doi="10.1038/nature01470"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Antibody neutralization and escape by HIV-1</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>, <span class="cit-fpage">307</span>–<span class="cit-lpage">312</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWei%26rft.auinit1%253DX.%26rft.volume%253D422%26rft.issue%253D6929%26rft.spage%253D307%26rft.epage%253D312%26rft.atitle%253DAntibody%2Bneutralization%2Band%2Bescape%2Bby%2BHIV-1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01470%26rft_id%253Dinfo%253Apmid%252F12646921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12646921&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181637300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.35" data-doi="10.1093/glycob/cwh106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Tracking global patterns of N-linked glycosylation site variation in highly variable viral glycoproteins: HIV, SIV, and HCV envelopes and influenza hemagglutinin</span>. <abbr class="cit-jnl-abbrev">Glycobiology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">1229</span>–<span class="cit-lpage">1246</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycobiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fglycob%252Fcwh106%26rft_id%253Dinfo%253Apmid%252F15175256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/glycob/cwh106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15175256&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225104200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.36" data-doi="10.1016/j.cell.2016.04.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stewart-Jones</span>,  <span class="cit-name-given-names">G. B. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Trimeric HIV-1-Env structures define glycan shields from Clades A, B, and G</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">165</span>, <span class="cit-fpage">813</span>–<span class="cit-lpage">26</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D165%26rft.spage%253D813%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.04.010%26rft_id%253Dinfo%253Apmid%252F27114034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.04.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27114034&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.26.010322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential processing of HIV envelope glycans on the virus and soluble recombinant trimer</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">3693</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D3693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.38" data-doi="10.1128/JVI.01190-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">L. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harvey</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonomelli</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crispin</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Doores</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-article-title">Cell- and protein-directed glycosylation of native cleaved HIV-1 envelope</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">8932</span>–<span class="cit-lpage">44</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01190-15%26rft_id%253Dinfo%253Apmid%252F26085151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xNy84OTMyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNi4wMTAzMjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohou</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Grigorieff</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">CisTEM, user-friendly software for single-particle image processing</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">7</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.010322v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emsley</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Crispin</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Structural analysis of glycoproteins: Building N-linked glycans with coot</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr. Sect. D Struct. Biol.</abbr> <span class="cit-vol">74</span>, <span class="cit-fpage">256</span>–<span class="cit-lpage">263</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallogr.%2BSect.%2BD%2BStruct.%2BBiol.%26rft.volume%253D74%26rft.spage%253D256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.26.010322v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.25.996348v1" title="Structure-Based Design, Synthesis and Biological Evaluation of Peptidomimetic Aldehydes as a Novel Series of Antiviral Drug Candidates Targeting the SARS-CoV-2 Main Protease" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.26.009605v1" title="Re-analysis of SARS-CoV-2 infected host cell proteomics time-course data by impact pathway analysis and network analysis. A potential link with inflammatory response" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 28, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.26.010322v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.26.010322v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.26.010322v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.26.010322v1" class="active" data-icon-position="" data-hide-link-title="0">Site-specific analysis of the SARS-CoV-2 glycan shield</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1215896" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Rti05HXsxRGH0r7fNQbAMHlz5KaQgYY8gF3k2-0rkzA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215896" id="node1215896--2" data-pisa="biorxiv;2020.03.26.010322v1" data-pisa-master="biorxiv;2020.03.26.010322" data-seqnum="1215896" data-apath="/biorxiv/early/2020/03/28/2020.03.26.010322.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Site-specific analysis of the SARS-CoV-2 glycan shield</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yasunori</span> <span class="nlm-surname">Watanabe</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joel D.</span> <span class="nlm-surname">Allen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Max</span> <span class="nlm-surname">Crispin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.26.010322; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.26.010322 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;title=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;title=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;text=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;title=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;t=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;title=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;title=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215896" id="node1215896--3" data-pisa="biorxiv;2020.03.26.010322v1" data-pisa-master="biorxiv;2020.03.26.010322" data-seqnum="1215896" data-apath="/biorxiv/early/2020/03/28/2020.03.26.010322.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Site-specific analysis of the SARS-CoV-2 glycan shield</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yasunori</span> <span class="nlm-surname">Watanabe</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joel D.</span> <span class="nlm-surname">Allen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Max</span> <span class="nlm-surname">Crispin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.26.010322; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.26.010322 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Watanabe%2C+Yasunori&amp;rft.au=Allen%2C+Joel+D.&amp;rft.au=Wrapp%2C+Daniel&amp;rft.au=McLellan%2C+Jason+S.&amp;rft.au=Crispin%2C+Max"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1215896/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1215896/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1215896/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1215896/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1215896/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1215896/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1215896/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1215896/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1215896/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1215896/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1215896/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1215896/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;count=horizontal&amp;via=&amp;text=Site-specific%20analysis%20of%20the%20SARS-CoV-2%20glycan%20shield&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.010322v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.26.010322v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
