<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Viruses</span><span class="iso-abbrev" title="iso-abbrev">Viruses</span><span class="publisher-id" title="publisher-id">viruses</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Viruses</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1999-4915</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7150801">7150801</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/v12030287">10.3390/v12030287</a></span><span class="publisher-id" title="publisher-id">viruses-12-00287</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
New Viral Sequences Identified in the Flavescence Dorée Phytoplasma Vector <i>Scaphoideus titanus</i></div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ottati</span><span class="given-names" tagx="given-names" title="given-names">Sara</span></span><a href="#af1-viruses-12-00287">1</a><a href="#af2-viruses-12-00287">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiapello</span><span class="given-names" tagx="given-names" title="given-names">Marco</span></span><a href="#af2-viruses-12-00287">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galetto</span><span class="given-names" tagx="given-names" title="given-names">Luciana</span></span><a href="#af2-viruses-12-00287">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-3945-2752</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosco</span><span class="given-names" tagx="given-names" title="given-names">Domenico</span></span><a href="#af1-viruses-12-00287">1</a><a href="#af2-viruses-12-00287">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-1084-6048</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzachì</span><span class="given-names" tagx="given-names" title="given-names">Cristina</span></span><a href="#af2-viruses-12-00287">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-9345-5407</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abbà</span><span class="given-names" tagx="given-names" title="given-names">Simona</span></span><a href="#af2-viruses-12-00287">2</a><a href="#c1-viruses-12-00287">*</a></span></div><span class="citation_author_institution" id="af1-viruses-12-00287">[1], <span class="email" tagx="email" title="email">sara.ottati@unito.it</span><span class="email" tagx="email" title="email">domenico.bosco@unito.it</span></span><span class="citation_author_institution" id="af2-viruses-12-00287">[2], <span class="email" tagx="email" title="email">marco.chiapello@ipsp.cnr.it</span><span class="email" tagx="email" title="email">luciana.galetto@ipsp.cnr.it</span><span class="email" tagx="email" title="email">cristina.marzachi@ipsp.cnr.it</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">simona.abba@ipsp.cnr.it</span>; Tel.: +39-011-397-7925</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-3-3</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-3</span></span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">287</span><span class="history" title="history"><span class="received" title="received">received: 2020-2-10</span><span class="accepted" title="accepted">accepted: 2020-3-04</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p><b>(1) Background:</b> The leafhopper Scaphoideus titanus is the primary vector of Flavescence dorée phytoplasma (FDp) in European vineyards. Flavescence dorée is one of the most severely damaging diseases of Vitis vinifera and, consequently, a major threat to grape and wine production in several European countries. Control measures are compulsory, but they mainly involve large-scale insecticide treatments, with detrimental impacts on the environment. One possible solution is to exploit the largely unexplored genetic diversity of viruses infecting S. titanus as highly specific and environmentally benign tools for biological control. <b>(2) Methods:</b> A metatranscriptomic approach was adopted to identify viruses that may infect individuals caught in the wild in both its native (United States) and invasive (Europe) areas. Reverse transcription PCR was used to confirm their presence in RNA pools and explore their prevalence. <b>(3) Results:</b> We described nine new RNA viruses, including members of "Picorna-Calici�?, "Permutotetra�?, "Bunya-Arena�?, "Reo�?, "Partiti-Picobirna�?, "Luteo-Sobemo�? and "Toti-Chryso�? clades. A marked difference in the diversity and abundance of the viral species was observed between the USA population and the European ones. <b>(4) Conclusions:</b> This work represents the first survey to assess the viral community of a phytoplasma insect vector. The possibility to exploit these naturally occurring viruses as specific and targeted biocontrol agents of <i>S. titanus</i> could be the answer to increasing demand for a more sustainable viticulture.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Flavescence dorée</div> <div class="kwd" title="kwd"><i>Scaphoideus titanus</i></div> <div class="kwd" title="kwd">
insect viruses</div> <div class="kwd" title="kwd">
metatranscriptomics</div> <div class="kwd" title="kwd"><i>Vitis vinifera</i></div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p><i>Scaphoideus titanus</i> Ball. (Hemiptera: Cicadellidae) is a phloem-feeding nearctic leafhopper that was probably introduced accidentally from North America to Europe in the late 19<sup>th</sup> century, in the attempt to find a solution against downy mildew and phylloxera epidemics occurring in Europe at that time [<a href="#B1-viruses-12-00287">1</a>]. Genetic characterization of American and European <i>S. titanus</i> populations revealed that the latters are likely to descend from a single introduction from the United States [<a href="#B2-viruses-12-00287">2</a>]. In its original range, <i>S. titanus</i> was mostly captured in open woodlands, on wild <i>Vitis</i> species and only rarely on cultivated grapevines [<a href="#B3-viruses-12-00287">3</a>,<a href="#B4-viruses-12-00287">4</a>]. In Europe, by contrast, it has been reported mainly on <i>Vitis vinifera</i>, although it is known to thrive in abandoned vineyards and wild grapevines [<a href="#B5-viruses-12-00287">5</a>,<a href="#B6-viruses-12-00287">6</a>,<a href="#B7-viruses-12-00287">7</a>,<a href="#B8-viruses-12-00287">8</a>]. <i>S. titanus</i> is the primary vector of Flavescence dorée phytoplasma (FDp) in vineyards, and it has so far, spread to many European grape-growing countries, such as Italy [<a href="#B9-viruses-12-00287">9</a>], France [<a href="#B10-viruses-12-00287">10</a>], Hungary [<a href="#B11-viruses-12-00287">11</a>] and Switzerland [<a href="#B12-viruses-12-00287">12</a>]. Phytoplasmas are obligate bacterial pathogens, confined mainly in the phloem tissue. Upon feeding on infected plants, sap-sucking insects, such as <i>S. titanus</i>, can passively acquire them and transmit the pathogen to healthy plants [<a href="#B13-viruses-12-00287">13</a>]. FDp is a quarantine pest of grapevine in the European Union. It typically causes color aberrations and downward-rolled margins in foliage, lack of lignification in shoots, reduced fruit setting, flower withering, bunch wilting and berry drop. In the end, infected plants either die or recover from the disease, but with a significant reduction in productivity [<a href="#B14-viruses-12-00287">14</a>]. The overall economy of wine-producing regions is not only affected by yield losses caused by this disease, but also by the expensive compulsory containment measures [<a href="#B15-viruses-12-00287">15</a>,<a href="#B16-viruses-12-00287">16</a>,<a href="#B17-viruses-12-00287">17</a>]. The only currently available control strategies focus on the reduction of vector populations in vineyards, and include uprooting of symptomatic grapevines, compulsory insecticide applications with specific timing, removal of wild vines that may act as refuges for the insect, and hot water treatments of rootstocks, scions or grafted cuttings to kill deposited eggs [<a href="#B18-viruses-12-00287">18</a>]. At the same time, concerns are mounting especially over the effects of pesticides on the environment, non-target organisms and human health. Consequently, it is essential to develop more sustainable alternatives to the use of chemicals. A possible option would be to exploit the largely unexplored genetic diversity of viruses infecting <i>S. titanus</i> for developing highly specific and environmentally benign tools for biological control.</p> <p>Baculoviruses have been successfully used as safe and selective bioinsecticides to control mainly Lepidoptera, since their first commercial use in 1975 [<a href="#B19-viruses-12-00287">19</a>]. Nevertheless, the use of baculoviruses is still limited by their narrow host range, which exclude major plant pests such as those belonging to the order Hemiptera. Only two members of the families <i>Parvoviridae</i> and <i>Nudiviridae</i> were used so far for biological control of insect pests other than Lepidoptera: Cockroach densonucleosis virus against <i>Periplaneta fuliginosa</i> (Blattodea) [<a href="#B20-viruses-12-00287">20</a>] and Oryctes virus against <i>Oryctes rhinoceros</i> (Coleoptera) [<a href="#B21-viruses-12-00287">21</a>].</p> <p>In this work we applied RNA-Seq to describe putative viruses that may infect wild-caught <i>S. titanus</i> populations in both its invasive (Europe) and native (United States) geographical areas. Comparisons between the two sampling areas allowed for the description of two completely different viral profiles, characterized by a reduction in the number of potential enemies in the European populations. Nine new complete or near-complete viral genomes were identified, representing seven virus families. An iflavirus identified in Europe, in particular, represented a potential candidate for future isolation and pathogenicity assays or for the construction of virus-induced gene silencing (VIGS) vectors to transiently knock-down insect genes involved in phytoplasma acquisition/transmission.</p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.insectcollection" title="sec"> <div class="title" tagx="title" title="title">
2.1. Insect Collection</div> <p><i>S. titanus</i> specimens used for RNA-seq were collected from the field with a sweeping net in 15 European sites and in one USA site during summer 2018 (<a href="#app1-viruses-12-00287">Supplementary Materials S1</a>). Sampling locations, GPS coordinates in decimal degrees, collection dates and number of sampled insects are reported in <a href="#viruses-12-00287-t001">Table 1</a>. All insects, except those collected in Piedmont, were immediately stored and shipped in RNA<i>later</i> (Thermo Fisher Scientific Inc., Waltham, MA, USA).</p> <p>After the virome assembly, four specimens were used for verifying the possible integration of some viral sequences into the insect genome. They were collected from the field with yellow sticky traps left for two days in highly infested vineyards (Montà and Dogliani, Piedmont, Italy) (<a href="#viruses-12-00287-t001">Table 1</a>) localized in the same geographical area from which insects were sampled to construct one of the two Italian libraries for RNA sequencing.</p> </div> <div class="2.2.dnaextraction,rnaextraction,librarypreparationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
2.2. DNA Extraction, RNA Extraction, Library Preparation and Sequencing</div> <p>Adult insects were frozen with liquid nitrogen, crushed with a micropestle in sterile tubes, and then subjected to either DNA or RNA extraction. DNA was extracted according to the method described by Marzachì et al. [<a href="#B22-viruses-12-00287">22</a>]. For RNA extraction, samples were homogenized in TRI Reagent (Zymo Research, Irvine, CA, USA) and centrifuged for 1 min at 12,000 g at 4°C to remove cell debris. RNA was extracted from supernatants with Direct-zol RNA Mini Prep kit (Zymo Research, Irvine, CA, USA), following the manufacturer’s protocol with DNAse treatment. Concentration, purity, and quality of DNA and RNA extractions were estimated using a Nanodrop spectrophotometer.</p> <p>RNA extracts were pooled for library construction according to the country of origin of the insects: St_FR library for France, St_CH library for Switzerland, St_HU library for Hungary, and St_USA for United States of America (<a href="#viruses-12-00287-t001">Table 1</a>, <a href="#app1-viruses-12-00287">Supplementary Materials S1</a>). The Italian samples were grouped in two libraries: specimens collected in Piedmont, Lombardy and Veneto were pooled to construct the St_IT1 library, whereas the remaining insects from Friuli-Venezia Giulia and Abruzzo were pooled in the St_IT2 library (<a href="#viruses-12-00287-t001">Table 1</a>, <a href="#app1-viruses-12-00287">Supplementary Materials S1</a>).</p> <p>Library preparation and sequencing were conducted by Macrogen Inc. (Seoul, South Korea). The Ribo-Zero Gold Kit (Human/Mouse/Rat) (Illumina Inc., San Diego, CA, USA) was used to deplete ribosomal RNA. cDNA libraries were prepared using a TruSeq Stranded Total RNA Library Prep Kit (v2) (Illumina Inc. San Diego, CA, USA) and were sequenced from both ends (100 bp) using the NovaSeq System.</p> </div> <div class="2.3.bioinformaticanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.3. Bioinformatic Analysis</div> <p>The pre-assembly steps were performed using the suite of bioinformatic tools called BBTools v38.70 [<a href="#B23-viruses-12-00287">23</a>]. Raw paired-end files were processed for removal of Illumina adaptor sequences and artifacts, and for length and ribosomal filtering via kmer matching using BBDuk. To avoid unwanted environmental contaminants, the remaining reads were further analyzed to remove reads matching to human, mouse and dog sequences, using BBMap. Finally, BBMerge and BBNorm were used for merging overlapping paired reads and normalizing read coverage before assembling with Trinity v2.6.6 [<a href="#B24-viruses-12-00287">24</a>]. Some of the resulting sequences were further assembled by CAP3 v3 (overlap length cutoff = 65; overlap percent identity cutoff N = 90) [<a href="#B25-viruses-12-00287">25</a>]. DIAMOND v0.9.24.125 [<a href="#B26-viruses-12-00287">26</a>] with an E-value cut-off of 0.0001 was used to perform a blastx search for matches to viral genomes in the NCBI non redundant protein database (version October 2018). Putative viral sequences were further checked and analyzed manually, in an effort to exclude as much as possible viruses associated with insect food (especially grapevine), gut microflora or possible parasites. Such filtering was based on similarities with known plant, fungi and nematode viruses, and on the number of reads mapping on each viral transcript, assuming that the more common the virus was, the more likely it was infecting <i>S. titanus</i>. Bowtie2 [<a href="#B27-viruses-12-00287">27</a>] was used to map reads against the reference transcripts with parameters that selected only reads that mapped in pairs with the right orientation (--no-mixed –no-discordant). Reads mapping onto viral genomes were expressed as RPKM (Reads Per Kilobase of transcript per Million mapped reads).</p> <p>The bioinformatic command lines and parameters used in RNA-seq data analyses are provided as <a href="#app1-viruses-12-00287">Supplementary Materials S2</a>.</p> <p>Viral sequences have been deposited in the GenBank database under the accession numbers MN982379-MN982405.</p> </div> <div class="2.4.phylogenetictrees" title="sec"> <div class="title" tagx="title" title="title">
2.4. Phylogenetic Trees</div> <p>Phylogenetic relationships were inferred on the basis of putative RNA-dependent RNA polymerases (RdRps), as they shared conserved domains across all RNA viruses. All the phylogenies described in this manuscript used clades, nomenclature and sequences provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. RdRp amino acid sequences of the newly discovered viruses were trimmed to include only conserved domains and aligned using MAFFT v7 [<a href="#B29-viruses-12-00287">29</a>] with the E-INS-i algorithm. Phylogenetic trees were then generated using the maximum likelihood approach (ML) implemented in IQ-TREE [<a href="#B30-viruses-12-00287">30</a>] with default parameters.</p> </div> <div class="2.5.pcrandrt-pcr" title="sec"> <div class="title" tagx="title" title="title">
2.5. PCR and RT-PCR</div> <p>In order to independently confirm the RNA-seq results, reverse transcriptase PCR (RT-PCR) was used to obtain short amplicons from at least one segment (in the case of multipartite viruses) of the newly identified viruses.</p> <p>For each RNA pool used for library construction, cDNA was synthesized from total RNA (200 ng) using the High Capacity cDNA reverse transcription kit (Thermo Fisher Scientific Inc., Waltham, MA, USA). Specific primers were designed for all the newly identified viruses. RT-PCR was also conducted to determine the prevalence of the European viruses in the original unpooled RNA samples. The absence of contaminating genomic DNA was verified, including in the PCR step samples without the reverse transcription step.</p> <p>Additional primers were designed on putative endogenous viral elements (EVEs) and used in PCR (without reverse transcriptions) on <i>S. titanus</i> genomic DNA. All primer sequences and amplification conditions used in this work are listed in <a href="#app1-viruses-12-00287">Supplementary Materials S3</a>. The resulting amplicons were validated by Sanger sequencing at BMR Genomics (Padua, Italy).</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
3. Results</div> <p>We characterized the total transcriptome of six <i>S. titanus</i> pools: five from European locations (St_IT1, St_IT2, St_HU, St_FR and St_CH) and one from a USA location (St_USA). RNA sequencing resulted in 110 million to 130 million reads per library, which were independently assembled into contigs and compared to NCBI protein non redundant database for virus discovery. These transcriptome data allowed us to identify nine new viral RNA genomes associated to <i>S. titanus</i>: two in the European libraries and seven in the USA one (<a href="#viruses-12-00287-t002">Table 2</a>). Five of the viruses described here are expected to have a positive-sense single-stranded RNA (+ssRNA) genome, one a negative-sense single-stranded RNA (−ssRNA) genome and three a double-stranded RNA (dsRNA) genome. Only complete or near complete viral genomes were considered in this computation, i.e., genomes comprising the complete coding potential that should be expected from the existing virus taxa they are most closely related to. None of these genomes featured premature stop codons or repeat sequences, so they unlikely represented endogenous viral elements (EVEs), namely integrations of DNA and nonretroviral RNA viruses into the host genome. According to these strict criteria, the St_HU library was the only one in which we could not identify any sequence matching to viruses deposited into public databases.</p> <p>The presence of viral genomes in the original RNA samples was confirmed by RT-PCR coupled with Sanger sequencing. The newly identified viruses were tentatively named according to their host (<i>S. titanus</i>) and phylogenetic relationships.</p> <div class="3.1.single-strandedpositive-sensernaviruses" title="sec"> <div class="title" tagx="title" title="title">
3.1. Single-Stranded Positive-Sense RNA Viruses</div> <p>Three contigs, two from St_IT1 and St_FR libraries and one from the St_USA library, showed the highest similarity (83% amino acid identity, 51% query coverage) with the polyprotein of Graminella nigrifrons virus 1, which was assigned to the family <i>Iflaviridae</i> (<a href="#viruses-12-00287-t002">Table 2</a>).</p> <p>According to the International Committee on Taxonomy of Viruses (ICTV) species demarcation criteria for the genus Iflavirus [<a href="#B31-viruses-12-00287">31</a>], isolates/strains belong to the same iflavirus species if the sequence identity at the amino acid level, between their coat proteins, is above 90%. On this basis, the two European genomes significantly differed from the known iflavirus sequences, but not from each other, so they were likely to represent a single new iflavirus species, hereafter named Scaphoideus titanus iflavirus 1. The USA iflavirus showed a percentage of amino acid identity below 90% if compared to other known iflaviruses and Scaphoideus titanus iflavirus 1. Therefore, it could represent another new iflavirus species, from here on named Scaphoideus titanus iflavirus 2.</p> <p>The genome of iflaviruses is a monopartite +ssRNA genome, which encodes a large polyprotein auto-catalytically cleaved into structural and nonstructural component peptides: starting from the N-terminus, capsid proteins, followed by three non-structural proteins, an RNA helicase, a 3C-like cysteine protease, and an RdRp. A Pfam search revealed the presence of these peptides in both Scaphoideus titanus iflaviruses (<a href="#app1-viruses-12-00287">Supplementary Materials S4</a>). The two new RdRps were aligned to those of viruses within the "Iflaviridae-Secoviridae related cluster�? [<a href="#B28-viruses-12-00287">28</a>] with the addition of the first three best hits retrieved from the blastx analysis (GenBank accessions: APD68841.1, YP_009553259.1 and YP_009129265.1). The phylogenetic analysis confirmed that both sequences were closely related to iflaviruses identified from other hosts of the family Cicadellidae (<i>Graminella nigrifrons</i>, <i>Psammotettix alienus</i> and <i>Euscelidius variegatus</i>) (<a href="#viruses-12-00287-f001">Figure 1</a>A, <a href="#app1-viruses-12-00287">Supplementary Materials S5</a>).</p> <p>Four contigs from the St_USA library were tentatively assigned to two sobemo-like viruses, hereafter referred to as Scaphoideus titanus sobemo-like virus 1 and 2 (<a href="#viruses-12-00287-t002">Table 2</a>). The two RdRps showed the highest similarity with those of Hubei sobemo-like virus 26 and Hubei sobemo-like virus 24, respectively. Despite the fact that most sobemo-like viruses described by Shi et al. [<a href="#B28-viruses-12-00287">28</a>] have a +ssRNA monopartite genome, we could reasonably hypothesize that each sobemo-like genome identified in <i>S. titanus</i> has a bipartite genome: the longer fragment (around 3000 bp) encodes two non-overlapping ORFs coding for a trypsin-like serine protease and an RdRp; the second shorter genomic fragment (around 1500 bp) codes for a putative capsid protein (<a href="#app1-viruses-12-00287">Supplementary Materials S4</a>). Such hypothesis was based on similar depth of coverage across the viral segments, and, above all, on the presence of the same heptanucleotide at the termini of the two segments assigned to Scaphoideus titanus sobemo-like virus 1. Consequently, the other two sequences most likely belonged to Scaphoideus titanus sobemo-like virus 2. The correct two-by-two association between long and short segments was also confirmed by RT-PCR: one insect was positive only for one couple of segments and another insect was positive only for the other couple (data not shown). Notably, both hypothetical capsid proteins were similar to a hypothetical protein identified in a nodavirus (Naganuma virus). The two newly identified sobemo-like viruses appeared to belong to two different lineages within the large clade that grouped viruses identified from a variety of arthropods (<a href="#viruses-12-00287-f001">Figure 1</a>B, <a href="#app1-viruses-12-00287">Supplementary Materials S6</a>).</p> <p>A contig from the St_USA library showed the highest similarity (39% amino acid identity, 39% query coverage) with Hubei permutotetra-like virus 9 (<a href="#viruses-12-00287-t002">Table 2</a>). Despite the blastx result, the genome organization of this virus (<a href="#viruses-12-00287-f002">Figure 2</a>A) was more similar to Hubei permutotetra-like virus 8 (GenBank accession: NC_033216.1). Both genomes, in fact, contained three coding sequences on the same strand, which simultaneously overlapped for a short region. In our case, the first and the longest ORF, encoding the RdRp, overlapped at the 3′ terminus for 106 nucleotides with both 5′ termini of the other two ORFs, which putatively coded for the capsid proteins. The latter two were found in a +1 and -1 reading frame compared to the RdRp coding sequence. Similarly to all permutotetraviruses, the RdRp motif C was located upstream of motif A, forming a non-canonical C-A-B arrangement. On the basis of these characteristics, hereafter we referred to this virus as Scaphoideus titanus permutotetra-like virus 1. This virus was distantly related to previously described members of the family <i>Permutotetraviridae</i> and fell within a weakly supported clade (bootstrap &amp;lt; 70%), which grouped viruses isolated from insects, including the abovementioned Hubei permutotetra-like virus 9 (<a href="#viruses-12-00287-f001">Figure 1</a>C, <a href="#app1-viruses-12-00287">Supplementary Materials S7</a>).</p> </div> <div class="3.2.single-strandednegative-sensernavirus" title="sec"> <div class="title" tagx="title" title="title">
3.2. Single-Stranded Negative-Sense RNA Virus</div> <p>On the basis of the blastx results, three contigs from the St_USA library were identified as the three putative segments of a bunya-like virus, hereafter named Scaphoideus titanus bunya-like virus 1 (<a href="#viruses-12-00287-t002">Table 2</a>). <i>Bunyaviridae</i> have tripartite genomes consisting of a large segment (L), which encodes the RdRp, a medium (M) segment, which encodes the viral glycoprotein precursor, and a small (S) segment, which encodes the nucleoprotein (<a href="#app1-viruses-12-00287">Supplementary Materials S4</a>). In the case of Scaphoideus titanus bunya-like virus 1, it was possible to identify the same terminal complementary heptanucleotide in segments M and S, but not in the L segment. Despite that, the RdRp-coding sequence that we putatively assigned to this bunya-like virus was the only one found in this library that showed significant similarities with RdRps of known bunyaviruses. In addition, the phylogenetic analysis unambiguously (100% bootstrap support) placed it within the <i>Phasmaviridae</i> cluster, a family of insect-infecting viruses in the order <i>Bunyavirales</i> (<a href="#viruses-12-00287-f003">Figure 3</a>A, <a href="#app1-viruses-12-00287">Supplementary Materials S8</a>).</p> </div> <div class="3.3.double-strandedrnaviruses" title="sec"> <div class="title" tagx="title" title="title">
3.3. Double-Stranded RNA Viruses</div> <p>The deduced amino acid sequences of three contigs from St_IT2, St_FR and St_CH libraries were identical and their closest relative identified by blastx (42% amino acid identity, 37% query coverage) was a dsRNA virus named Spissistilus festinus virus 1 (SpFV1) (<a href="#viruses-12-00287-t002">Table 2</a>). According to the ICTV, less than 50% sequence identity at the protein level generally reflects a species difference, so the newly identified virus was named Scaphoideus titanus toti-like virus 1.</p> <p>SpFV1 has a monopartite dsRNA genome that shares similarities at the amino acid level with a variety of dsRNA viruses, some unclassified and some classified in the <i>Totiviridae</i> or <i>Chrysoviridae</i> families [<a href="#B32-viruses-12-00287">32</a>]. Similarly to SpFV1 and its sister taxon Circulifer tenellus virus 1 (CiTV1) [<a href="#B32-viruses-12-00287">32</a>], the genome organization of Scaphoideus titanus toti-like virus 1 presented a 5′-leader sequence followed by an ORF encoding a proline-alanine rich protein (PArp), a 3′ proximal RdRp ORF, and finally a short 3′-untranslated region (<a href="#viruses-12-00287-f002">Figure 2</a>B). The presence of a putative 7nt-frameshift site (AAACCCU) near the PArp ORF stop codon and a downstream stretch of 85 nucleotides that may form a pseudoknot suggested that, similarly to SpFV1 and CiTV1, the expression of the RdRp could be the result of a programmed -1 ribosomal frameshifting (<a href="#viruses-12-00287-f002">Figure 2</a>B). Such genomic organization is very similar to that of members of the genus <i>Totivirus</i>. Despite that, Scaphoideus titanus toti-like virus 1, as well as SpFV1 and CiTV1, fell into a well-supported clade distinct from the one that included viruses belonging to the family <i>Totiviridae</i> (<a href="#viruses-12-00287-f003">Figure 3</a>B, <a href="#app1-viruses-12-00287">Supplementary Materials S9</a>). Besides Scaphoideus titanus toti-like virus 1, this small clade included other eight viruses isolated from members of the class Insecta and two viruses identified in plants, Cucurbit yellows-associated virus [<a href="#B33-viruses-12-00287">33</a>] and Persimmon latent virus [<a href="#B34-viruses-12-00287">34</a>].</p> <p>Eleven sequences from the St_USA library were identified as putative segments of a reovirus, hereafter named Scaphoideus titanus reo-like virus 1 (<a href="#viruses-12-00287-t002">Table 2</a>, <a href="#app1-viruses-12-00287">Supplementary Materials S4</a>). Initially, only nine segments were assigned to this virus. The putative missing segments were then manually searched using the genome of Homalodisca vitripennis reovirus (GenBank accessions: FJ497789.1 – FJ497800.1) [<a href="#B35-viruses-12-00287">35</a>] as bait. The targeted analysis succeeded in finding segments 9 and 11. The fact that both formed chimeric sequences with long insect transcripts was probably the reason for the failure to identify them by automatic blastx analysis. Segment 12 of Homalodisca vitripennis reovirus did not find any significant match with any of the contigs within the St_USA library. The phylogenetic analysis placed Scaphoideus titanus reo-like virus 1 in the phytoreovirus clade, which included Homalodisca vitripennis reovirus and two plant-infecting viruses (<a href="#viruses-12-00287-f003">Figure 3</a>C, <a href="#app1-viruses-12-00287">Supplementary Materials S10</a>). The phytoreovirus clade was, in its turn, phylogenetically close to two reo-like viruses isolated from insects.</p> <p>In the St_USA library we also identified a partiti-like virus, hereafter named Scaphoideus titanus-associated partiti-like virus 1 (<a href="#viruses-12-00287-t002">Table 2</a>, <a href="#app1-viruses-12-00287">Supplementary Materials S4</a>). A blastx analysis, in fact, revealed the presence of two contigs with similarity to a hypothetical protein (61% amino acid identity, 35% query coverage) and an RdRp (87% amino acid identity, 44% query coverage), both belonging to a partiti-like virus identified in a nematode, named Wuhan large pig roundworm virus 1. The same nine nucleotides were identified at the termini of both segments, confirming they were likely to be part of the same viral genome. Interestingly, Scaphoideus titanus-associated partiti-like virus 1 fell in a large heterogeneous clade that included also deltapartitiviruses (plant-infecting viruses), as well as a group of four viruses identified in arthropods (two from the subphylum Crustacea and two from the class Insecta) and two identified in nematodes (<a href="#viruses-12-00287-f003">Figure 3</a>D, <a href="#app1-viruses-12-00287">Supplementary Materials S11</a>).</p> </div> <div class="3.4.virusprevalence" title="sec"> <div class="title" tagx="title" title="title">
3.4. Virus Prevalence</div> <p>RT-PCR was used to define the prevalence of the two viruses discovered in the European libraries. The same RNA extracts used for the construction of RNA-seq libraries were used unpooled to diagnose the presence of Scaphoideus titanus iflavirus 1 and Scaphoideus titanus toti-like virus 1, even in those European samples that, according to the bioinformatic analysis, were negative to one or both viruses (<a href="#viruses-12-00287-t001">Table 1</a>). We observed a perfect concordance between the presence/absence of the two viruses in the five libraries and the RT-PCR results (<a href="#viruses-12-00287-t001">Table 1</a> and <a href="#viruses-12-00287-t002">Table 2</a>). Regarding the prevalence, results divided by sampling countries are reported in <a href="#viruses-12-00287-f004">Figure 4</a>. Scaphoideus titanus iflavirus 1 was found only in Italy and in France populations with extremely low frequencies (3% at most). Scaphoideus titanus toti-like virus 1 was the most geographically widespread virus, as it was found in six out of the eight European sampling locations with variable percentages: 24% of the insects sampled in France and 13% of those collected in Switzerland. The percentage of insects negative for both viruses also rather fluctuated, being 100% in samples collected in Hungary and 74% observed in the ones from France.</p> </div> <div class="3.5.endogenousviralelements" title="sec"> <div class="title" tagx="title" title="title">
3.5. Endogenous Viral Elements</div> <p>Some contigs from the European and USA libraries found significant matches with known viruses, but sequences were partial and/or contained multiple stop codons, so they were considered as potential EVEs. This was the case of sequences disclosing similarities to: three fragments of RdRps from viruses belonging to the order <i>Mononegavirales</i>, a glycoprotein from a phlebovirus (order <i>Bunyavirales</i>), an RdRp from a reovirus, multiple non-structural proteins (NS1) and capsid proteins (VP1) from members of the family <i>Parvoviridae</i>.</p> <p>Given that <i>S. titanus</i> genome is not available, specific primers were designed on one sequence for each phylogenetic group that is characterized by an RNA genome without DNA intermediates, i.e., mononegaviruses, reoviruses and bunyaviruses (<a href="#app1-viruses-12-00287">Supplementary Materials S3 and S12</a>). PCR analyses on total DNA extracted from <i>S. titanus</i> revealed that they were indeed all endogenised into the host genome (data not shown). The potential integration of sequences similar to parvovirus genomes could not be verified by PCR, as they are DNA viruses and a PCR amplification could be due to either, an integration into the host genome or simply to the presence of the virus within the insect.</p> <p>Additionally, multiple fragments with similarities to the PArp from Spissistilus festinus virus 1 were found in each European library, but, unlike the EVEs mentioned above, they were characterized by ORFs that spanned the whole sequence without any premature stop codon. Notably, none of these sequences could be assigned to the previously described genome of Scaphoideus titanus toti-like virus 1. It was possible to reconstruct a large part of the 3′ PArp coding sequence only by using transcripts from all the European libraries together, but still the deduced protein was partial. In addition, a specific search to identify the missing RdRp failed. The fragmentation of the PArp coding sequence observed in each library and the lack of the corresponding RdRp represented clues in favor of a possible integration of these sequences into the insect genome. Specific primers designed on one of these fragments (<a href="#app1-viruses-12-00287">Supplementary Materials S3 and S12</a>), which was found in two out of the five European libraries, revealed that at least this part of the viral genome was indeed integrated into the insect genome (data not shown).</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>Pioneered by studies of bacteriophages in the marine environment [<a href="#B36-viruses-12-00287">36</a>], undirected high-throughput sequencing is becoming the easiest and cheapest procedure for the detection and genetic characterization of new viruses or virus variants. Metagenomic and metatranscriptomic studies have been radically changing virology, re-shaping virus lineages and revealing an unexpected richness of genome sizes and structures [<a href="#B37-viruses-12-00287">37</a>]. Although such astonishingly abundant sequences are linked to little or no biological information, there is unanimity in the assumption that genome sequences can be used to support the existence of a free/replicating virus [<a href="#B38-viruses-12-00287">38</a>]. In our work we chose a metatranscriptomic approach to characterize the virome of <i>S. titanus</i>, primarily because arthropods demonstrated to harbor an unprecedented diversity of RNA viruses [<a href="#B28-viruses-12-00287">28</a>]. Secondly, transcriptome profiling can provide additional information, such as the presence of RNAs possibly expressed by DNA viruses as well as a rough quantification of the viral load within the host.</p> <p>Virus-like sequences often derive from mixed virus populations and, consequently, there is the risk of assembling chimeric or artefactual genomes, neglecting sequences from viruses with segmented and multipartite genomes, and assigning transcribed virus-derived sequences integrated into the host genome to actively replicating viruses [<a href="#B37-viruses-12-00287">37</a>]. Stringent criteria were adopted in this work to minimize, as far as possible, the bioinformatic pitfalls that may lead to mis-assemblies and misinterpretation of the newly discovered genomes. First, we only considered contigs that encoded complete, or near complete, viral genomes with intact open reading frames to avoid EVEs. Additionally, in the case of multipartite viruses, we looked for the presence of the same nucleotides at the termini of each segment to ensure the correct genome assignment. The low and variable prevalence of the two newly identified viruses in the European populations of <i>S. titanus</i> constituted a further clue in favor of true replicating viruses. On the opposite, the presence in all or almost all libraries of some viral sequences with premature stop codons supported the hypothesis that they were potential EVEs. Some of these were unambiguously demonstrated to be integrated into <i>S. titanus</i> genome and expressed as RNAs.</p> <p>Both metatranscriptomics and metagenomics rely solely on inferred homology to known viruses to identify putative viral sequences, as viruses do not share any specific genetic marker. Such an approach, however, precludes the discovery of viruses that lack closely related sequences. Consequently, we could not exclude that our libraries contained more viruses than the nine described in this work. Another issue raised by such sequencing studies is whether the newly discovered viruses are true infectious agents of the targeted host. Field populations of insects can, in fact, harbour viruses that infect the insect itself, viruses of microbes or pathogens associated with the insect, and viruses associated with ingested materials [<a href="#B39-viruses-12-00287">39</a>]. Viruses infecting insect microflora, in particular, could also be potential candidates for insect biocontrol (e.g., bacteriophages attacking primary bacteria symbionts), but their exploitation would require the knowledge of the exact composition of <i>S. titanus</i> microbioma, which is so far fairly unexplored. All samples used in this work were caught in the wild, so the chance that some of the sequenced microorganisms were associated with insects only as environmental contaminants was very high. Therefore, we preferred to exclude bacteriophages and known plant viruses (e.g., Grapevine asteroid mosaic associated virus found in St_CH library) and retained only those viruses that are closely related to other insect-infecting viruses. The main ambiguity was represented by Scaphoideus titanus-associated partiti-like virus 1. After the taxonomic reorganization of <i>Partitiviridae</i>, the family has now five genera: <i>Alphapartitivirus</i> and <i>Betapartitivirus</i> (fungi and plant viruses), <i>Gammapartitivirus</i> (fungi viruses) <i>Deltapartitivirus</i> (plant viruses) and <i>Cryspovirus</i> (protozoa viruses) [<a href="#B40-viruses-12-00287">40</a>]. Scaphoideus titanus-associated partiti-like virus 1 clustered with two viruses identified in nematodes and formed a distinct clade only distantly related to <i>Deltapartitivirus,</i> the phylogenetically closest genus among those officially recognized by the ICTV. This result was consistent with phylogenetic analyses presented in previous works [<a href="#B41-viruses-12-00287">41</a>,<a href="#B42-viruses-12-00287">42</a>], especially in the manuscript by Shi et al. [<a href="#B28-viruses-12-00287">28</a>], which remarked that in the <i>Partitiviridae</i> family it is possible to resolve well-supported clades that appear to be specific to invertebrate hosts. Despite that, conclusive evidence that a partiti-like virus could infect insects has never been reported in literature, so we preferred the indication "associated virus�?.</p> <p><i>Scaphoideus titanus</i> is native to North America and invasive in Europe. Sampling in Europe was quite straightforward, because this species is abundant in vineyards showing symptoms of the Flavescence dorée disease. On the contrary, in the USA, the presence of <i>S. titanus</i> is not related to the manifestation of specific symptoms in the vegetation. Sampling in the USA is further complicated by the fact that <i>S. titanus</i> is usually found in unmanaged open woodlands and in mixed populations with other morphologically similar species, making its correct identification quite problematic, while sampling [<a href="#B43-viruses-12-00287">43</a>]. Despite the low number of insects caught in the USA, the most striking result of this investigation was the marked difference in the diversity and abundance of the viral species identified in the two geographical areas. Seven viral species were discovered from three individuals collected in the USA, whereas only two viral species, Scaphoideus titanus iflavirus 1 and Scaphoideus titanus toti-like virus 1, were collectively identified in the 214 individuals that were analyzed from various European sites. Additionally, 91% of the insects collected throughout Europe appeared to be free from both viruses. The Enemy Release Hypothesis (ERH) could be a possible explanation of such a dramatic shift in the number of viruses putatively infecting <i>S. titanus.</i> ERH predicts that a species will be successful in a new habitat as a result of a drop in overall number of natural enemies [<a href="#B44-viruses-12-00287">44</a>], e.g., predators, parasites and pathogens, including viruses. This hypothesis has been widely applied to invasive species, such as plant or animal pests in new habitats, even if the outcomes of these studies are highly variable and sometimes even contradictory [<a href="#B45-viruses-12-00287">45</a>]. A similar change in the profile of insect viruses associated with invasion of new habitats was also observed in <i>Drosophila suzukii</i>, a dipteran pest of soft fruits first isolated in Japan at the beginning of the 20<sup>th</sup> century and now spread worldwide [<a href="#B46-viruses-12-00287">46</a>].</p> <p>Although the inventory of viruses identified in <i>S. titanus</i> does not include any member of the family <i>Baculoviridae,</i> the species identified in this work have the potential to be used as natural enemies to control the insect vector. Unfortunately, most of them were identified in the St_USA library only and will not be given priority as potential biocontrol agents for the European <i>S. titanus</i> populations, in order to avoid issues related to the introduction of alien viral species. As far as it concerns the remaining two species, Scaphoideus titanus iflavirus 1 and Scaphoideus titanus toti-like virus 1, experimental investigations, such as isolation and pathogenicity assays, are required before proposing them as components of integrated pest management strategies. Viruses may not be necessarily lethal to their host, but just cause a reduction in its survival and fecundity, which in our case may decrease the overall insect density in vineyards and thereby Flavescence dorée incidence. Besides targeting insect vectors directly, they might also interfere with FDp acquisition and transmission. Studies on <i>Aedes aegypti</i>, in fact, demonstrated the existence of reciprocal interactions among insect antiviral, anti-bacterial, and anti-parasite immune responses that involve the Toll immune signaling pathway [<a href="#B47-viruses-12-00287">47</a>,<a href="#B48-viruses-12-00287">48</a>].</p> <p>In addition to wild-type viruses, biotechnologically manipulated viruses in the form of VIGS could be used to interfere with the insect immune system or manipulate the expression of insect genes that are hypothesized to be associated with the acquisition and transmission mechanisms of phytoplasmas. From this perspective, Scaphoideus titanus iflavirus 1 could be considered a good candidate for the construction of infectious clones. Indeed, an appropriate strategy to deliver virus-based biocontrol agents should be developed, especially for piercing-sucking insects like <i>S. titanus</i>. In-field application of RNA silencing techniques in phloem-feeding insects poses, in fact, a great technical challenge, since the delivery should preferably occur through vascular tissues. In this regard, double-stranded RNAs (dsRNAs) have been successfully delivered to psyllids and leafhoppers by root drench and trunk injections of grapevine and Citrus plants [<a href="#B49-viruses-12-00287">49</a>]. Moreover, topical application of dsRNAs has been used as an alternative way to induce gene silencing in the Asian citrus psyllid, <i>Diaphorina citri</i> [<a href="#B50-viruses-12-00287">50</a>]. Both approaches might also be explored for VIGS delivery.</p> <p>The majority of meta-omics studies about the virosphere of insect vectors has been conducted mostly on species transmitting diseases of medical and veterinary importance, such as <i>Anopheles</i> [<a href="#B51-viruses-12-00287">51</a>,<a href="#B52-viruses-12-00287">52</a>]<i>, Aedes</i> [<a href="#B53-viruses-12-00287">53</a>,<a href="#B54-viruses-12-00287">54</a>] and <i>Culex</i> [<a href="#B42-viruses-12-00287">42</a>,<a href="#B55-viruses-12-00287">55</a>]. By contrast, investigations on viruses that may infect insect vectors of plant pathogens are scarce. Metagenomics study of <i>Bemisia tabaci,</i> vector of several plant viruses, resulted in the identification of many RNA viruses, some of which are likely to infect the whiteflies themselves [<a href="#B56-viruses-12-00287">56</a>]. New putative viruses that may have the potential to be used as biocontrol agents have been discovered in <i>D. citri</i>, the natural vector of the causal agent of Huanglongbing (HLB) [<a href="#B57-viruses-12-00287">57</a>]. Novel dicistrovirus-like sequences were also identified during a metagenomics project of another vector for plant-infecting viruses, <i>Aphis fabae</i> [<a href="#B58-viruses-12-00287">58</a>]. The nine new viral genomes described in this work probably do not depict the total diversity of viruses that may infect <i>S. titanus</i>, as it is known that the composition of insect viromes is influenced by a variety of factors, such as diet, stage of development, geographical location [<a href="#B59-viruses-12-00287">59</a>,<a href="#B60-viruses-12-00287">60</a>]. Yet, we provide first insights into the unexplored viral community of this vector, which is responsible for the spread of FDp in European vineyards. The possibility of exploiting these naturally occurring viruses, as specific and targeted biocontrol agents of <i>S. titanus,</i> would be a possible answer to the increasing demand for a more sustainable viticulture.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors thank Nicola Mori (University of Padova, Italy), Valeria Trivellone (Illinois Natural History Survey, USA), Xavier Foissac, Sylvie Malembich-Maher, Delphine Desque (INRA, France), Christian Linder (Agroscope, Switzerland), Francesco Pavan (University of Udine, Italy), Ibolya Ember (The Hungarian National Blood Transfusion Service, Hungary), Matteo Ripamonti (University of Turin, Italy) for the maintencance of the <i>Scaphoideus titanus</i> rearing in Turin, Massimo Turina (CNR-IPSP, Turin, Italy) for critical reading of the manuscript.</p> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p><b>Supplementary Materials S1</b> Geographical maps illustrating the sampling locations. Maps were generated by R (version 3.4.4) packages "ggplot2�? and "tidyverse�?. <b>Supplementary Materials S2</b> Bioinformatic command lines and parameters used in RNAseq data analysis. <i><b>Supplementary Materials S3</b></i> List of primers used in this work. <b>Supplementary Materials S4</b> Structures of the newly identified viral genomes Structures of Scaphoideus titanus permutotetra-like virus 1 and Scaphoideus titanus toti-like virus 1 are illustrated in <a href="#viruses-12-00287-f002">Figure 2</a>. In this figure we specifically illustrated the iflavirus assembled from the St_IT1 library. Boxes represent boundaries of ORFs. Conserved domains and families identified by Pfam (<span class="uri" title="uri"><a href="https://pfam.xfam.org/">https://pfam.xfam.org/</a></span>) are reported within the polyproteins of the two iflaviruses: Waikav: Waikavirus capsid protein 1, Rhv: picornavirus capsid protein, CRPV: CRPV capsid protein like, 3C: 3C-like cysteine protease, RdRp: RNA-dependent RNA polymerase. List of of other abbreviations used in the figure: Hyp prt: hypothetical protein, Capsid prt: capsid protein, trypsin Ser protease: trypsin-like serine protease, NS prt: non-structural protein. <b>Supplementary Materials S5</b> Maximum likelihood phylogeny of Scaphoideus titanus iflavirus 1 and Scaphoideus titanus iflavirus 2 within the "Iflaviridae-Secoviridae related cluster�?. Sequence names and codes are those provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. The scale bar indicates the evolutionary distance for amino acid substitutions per site. The newly identified viruses are shown in red. <b>Supplementary Materials S6</b> Maximum likelihood phylogeny of Scaphoideus titanus sobemo-like virus 1 and Scaphoideus titanus sobemo-like virus 2 within the "Luteo-Sobemo�? clade. Sequence names and codes are those provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. The scale bar indicates the evolutionary distance for amino acid substitutions per site. The newly identified viruses are shown in red. <b>Supplementary Materials S7</b> Maximum likelihood phylogeny of Scaphoideus titanus permutotetra-like virus 1 within the "Permutotetra�? clade. Sequence names and codes are those provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. The scale bar indicates the evolutionary distance for amino acid substitutions per site. The newly identified virus is shown in red. <b>Supplementary Materials S8</b> Maximum likelihood phylogeny of Scaphoideus titanus bunya-like virus 1 within the "Bunya-Arena�? clade. Sequence names and codes are those provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. The scale bar indicates the evolutionary distance for amino acid substitutions per site. The newly identified virus is shown in red. <b>Supplementary Materials S9</b> Maximum likelihood phylogeny of Scaphoideus titanus toti-like virus 1 within the "Toti-Chryso�? clade. Sequence names and codes are those provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. The scale bar indicates the evolutionary distance for amino acid substitutions per site. The newly identified virus is shown in red. <b>Supplementary Materials S10</b> Maximum likelihood phylogeny of Scaphoideus titanus reo-like virus 1 within the "Reo�? clade. Sequence names and codes are those provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. The scale bar indicates the evolutionary distance for amino acid substitutions per site. The newly identified virus is shown in red. <b>Supplementary Materials S11</b> Maximum likelihood phylogeny of Scaphoideus titanus associated partiti-like virus 1 within the "Partiti-Picobirna�? clade. Sequence names and codes are those provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. The scale bar indicates the evolutionary distance for amino acid substitutions per site. The newly identified virus is shown in red. <b>Supplementary Materials S12</b> Sequences of potential endogenous viral elements (EVEs) found in the European RNA-Seq libraries. Each sequence header reports the internal code assigned to the sequence, the best blastx hit and the library code the sequence was identified from. Binding positions of the primers used in PCR to demonstrate the integration into <i>S. titanus</i> genome (see Supplementary Materials S3) are highlighted in yellow.</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="viruses-12-00287-s001.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization, S.A., C.M. and L.G.; methodology, S.O. and S.A.; formal analysis, S.O. and S.A.; investigation, S.O., S.A. and L.G.; software, M.C. and S.A.; writing—original draft preparation, S.A.; writing—review and editing, S.O., L.G., M.C., C.M. and D.B.; visualization, M.C., S.A.; supervision, C.M. and D.B.; resources, C.M.; project administration, C.M. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This project has received funding from the European Union’s Horizon 2020 research and innovation programme under grant agreement No 773567.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-viruses-12-00287" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carton</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sorensen</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Une coopération exemplaire entre entomologistes enançais et américains pendant la crise du Phylloxera en France (1868–1895)</span><span class="source" tagx="source" title="source">Ann. Société Entomol. Fr. NS</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">103</span><span class="lpage" tagx="lpage" title="lpage">125</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00379271.2007.10697500">10.1080/00379271.2007.10697500</a></span></span></li> <li tag="ref"><a name="B2-viruses-12-00287" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papura</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burban</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Helden</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giresse</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nusillard</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guillemaud</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerdelhué</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Microsatellite and Mitochondrial Data Provide Evidence for a Single Major Introduction for the Neartic Leafhopper Scaphoideus titanus in Europe</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e36882</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0036882">10.1371/journal.pone.0036882</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22629338">22629338</a></span></span></li> <li tag="ref"><a name="B3-viruses-12-00287" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maixner</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pearson</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boudon-Padieu</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caudwell</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Scaphoideus titanus, a possible vector of grapevine yellows in New York</span><span class="source" tagx="source" title="source">Plant Dis.</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">408</span><span class="lpage" tagx="lpage" title="lpage">413</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/PD-77-0408">10.1094/PD-77-0408</a></span></span></li> <li tag="ref"><a name="B4-viruses-12-00287" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schvester</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moutous</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carle</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Scaphoideus littoralis Ball (Homopt. Jassidae) cicadelle vectrice de la Flavescence dorée de la vigne</span><span class="source" tagx="source" title="source">Rev. Zool. Agric. Appl.</span><span class="year" tagx="year" title="year">1962</span><span class="volume" tagx="volume" title="volume">61</span><span class="fpage" tagx="fpage" title="fpage">118</span><span class="lpage" tagx="lpage" title="lpage">131</span></span></li> <li tag="ref"><a name="B5-viruses-12-00287" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krnjajić</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitrović</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cvrković</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jović</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petrović</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forte</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Angelini</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toševski</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Occurrence and distribution of Scaphoideus titanus in multiple outbreaks of "flavescence dorée�? in Serbia</span><span class="source" tagx="source" title="source">Bull. Insectol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">197</span><span class="lpage" tagx="lpage" title="lpage">198</span></span></li> <li tag="ref"><a name="B6-viruses-12-00287" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lessio</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tota</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alma</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tracking the dispersion of Scaphoideus titanus Ball (Hemiptera: Cicadellidae) from wild to cultivated grapevine: Use of a novel mark-capture technique</span><span class="source" tagx="source" title="source">Bull. Entomol. Res.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">104</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0007485314000030">10.1017/S0007485314000030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23614942">23614942</a></span></span></li> <li tag="ref"><a name="B7-viruses-12-00287" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pavan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mori</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bigot</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zandigiacomo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Border effect in spatial distribution of Flavescence dorée affected grapevines and outside source of Scaphoideus titanus vectors</span><span class="source" tagx="source" title="source">Bull. Insectol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">281</span><span class="lpage" tagx="lpage" title="lpage">290</span></span></li> <li tag="ref"><a name="B8-viruses-12-00287" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pegoraro</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ripamonti</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abbà</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beal</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraudo</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veratti</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malembic-Maher</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salar</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosco</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic diversity of Flavescence Dorée phytoplasmas at the vineyard scale</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">85</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e03123-18</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.03123-18">10.1128/AEM.03123-18</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30877117">30877117</a></span></span></li> <li tag="ref"><a name="B9-viruses-12-00287" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martini</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murari</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mori</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertaccini</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification and epidemic distribution of two flavescence dorée—related phytoplasmas in Veneto (Italy)</span><span class="source" tagx="source" title="source">Plant Dis.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">925</span><span class="lpage" tagx="lpage" title="lpage">930</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/PDIS.1999.83.10.925">10.1094/PDIS.1999.83.10.925</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30841074">30841074</a></span></span></li> <li tag="ref"><a name="B10-viruses-12-00287" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caudwell</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="source" tagx="source" title="source">Deux Années D’études sur la Flavescence Dorée Nouvelle Maladie Grave de la Vigne</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Institut National de la Recherche Agronomique</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Paris, France</span><span class="year" tagx="year" title="year">1957</span></span></li> <li tag="ref"><a name="B11-viruses-12-00287" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kriston</span><span class="given-names" tagx="given-names" title="given-names">É.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krizbai</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szabó</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bujdosó</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orosz</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dancsházy</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szőnyegi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">First occurrence of grapevine flavescence doree in Hungary</span><span class="source" tagx="source" title="source">Növényvédelem</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">433</span><span class="lpage" tagx="lpage" title="lpage">438</span></span></li> <li tag="ref"><a name="B12-viruses-12-00287" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jermini</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gusberti</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schaub</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linder</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gugerli</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schärer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emery</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Flavescence dorée and Scaphoideus titanus: Distribution and control in Switzerland</span><span class="source" tagx="source" title="source">IOBCWPRS Bull.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">36</span><span class="fpage" tagx="fpage" title="fpage">107</span><span class="lpage" tagx="lpage" title="lpage">111</span></span></li> <li tag="ref"><a name="B13-viruses-12-00287" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weintraub</span><span class="given-names" tagx="given-names" title="given-names">P.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beanland</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Insect vectors of phytoplasmas</span><span class="source" tagx="source" title="source">Annu. Rev. Entomol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">111</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.ento.51.110104.151039">10.1146/annurev.ento.51.110104.151039</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16332205">16332205</a></span></span></li> <li tag="ref"><a name="B14-viruses-12-00287" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morone</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boveri</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giosuè</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gotta</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossi</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scapin</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzachì</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology of Flavescence Dorée in Vineyards in Northwestern Italy</span><span class="source" tagx="source" title="source">Phytopathology</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">1422</span><span class="lpage" tagx="lpage" title="lpage">1427</span><span class="pub-id"><a href="https://dx.doi.org/10.1094/PHYTO-97-11-1422">10.1094/PHYTO-97-11-1422</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18943511">18943511</a></span></span></li> <li tag="ref"><a name="B15-viruses-12-00287" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belli</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bianco</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conti</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Grapevine yellows in Italy: Past, present and future</span><span class="source" tagx="source" title="source">J. Plant Pathol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">303</span><span class="lpage" tagx="lpage" title="lpage">326</span></span></li> <li tag="ref"><a name="B16-viruses-12-00287" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chuche</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiéry</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biology and ecology of the Flavescence dorée vector Scaphoideus titanus: A review</span><span class="source" tagx="source" title="source">Agron. Sustain. Dev.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">381</span><span class="lpage" tagx="lpage" title="lpage">403</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s13593-014-0208-7">10.1007/s13593-014-0208-7</a></span></span></li> <li tag="ref"><a name="B17-viruses-12-00287" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steffek</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reisenzein</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeisner</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of the pest risk from Grapevine flavescence dorée phytoplasma to Austrian viticulture</span><span class="source" tagx="source" title="source">EPPO Bull.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">191</span><span class="lpage" tagx="lpage" title="lpage">203</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2338.2007.01102.x">10.1111/j.1365-2338.2007.01102.x</a></span></span></li> <li tag="ref"><a name="B18-viruses-12-00287" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: EFSA Panel on Plant Health</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeger</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bragard</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caffier</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Candresse</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chatzivassiliou</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehnen-Schmutz</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilioli</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaques Miret</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacLeod</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Risk to plant health of Flavescence dorée for the EU territory</span><span class="source" tagx="source" title="source">EFSA J.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e04603</span></span></li> <li tag="ref"><a name="B19-viruses-12-00287" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ignoffo</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Entomopathogens as insecticides</span><span class="source" tagx="source" title="source">Environ. Lett.</span><span class="year" tagx="year" title="year">1975</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">23</span><span class="lpage" tagx="lpage" title="lpage">40</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/00139307509435834">10.1080/00139307509435834</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1091480">1091480</a></span></span></li> <li tag="ref"><a name="B20-viruses-12-00287" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergoin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tijssen</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Biological and molecular properties of densoviruses and their use in protein expression and biological control</span><span class="source" tagx="source" title="source">The Insect Viruses</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Plenum Publishing, Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span><span class="year" tagx="year" title="year">1998</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">169</span></span></li> <li tag="ref"><a name="B21-viruses-12-00287" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">T.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawford</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glare</span><span class="given-names" tagx="given-names" title="given-names">T.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Oryctes virus—Time for a new look at a useful biocontrol agent</span><span class="source" tagx="source" title="source">J. Invertebr. Pathol.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">94</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jip.2005.03.009">10.1016/j.jip.2005.03.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16039310">16039310</a></span></span></li> <li tag="ref"><a name="B22-viruses-12-00287" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzachì</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veratti</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosco</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Direct PCR detection of phytoplasmas in experimentally infected insects</span><span class="source" tagx="source" title="source">Ann. Appl. Biol.</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">133</span><span class="fpage" tagx="fpage" title="fpage">45</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1744-7348.1998.tb05801.x">10.1111/j.1744-7348.1998.tb05801.x</a></span></span></li> <li tag="ref"><a name="B23-viruses-12-00287" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="mixed-article-title" title="mixed-article-title">BBMap</span><span class="comment">Available online: <a href="Sourceforge.net/projects/bbmap/">Sourceforge.net/projects/bbmap/</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 11 March 2019)</span></span></li> <li tag="ref"><a name="B24-viruses-12-00287" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabherr</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haas</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yassour</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">J.Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amit</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adiconis</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raychowdhury</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span><span class="source" tagx="source" title="source">Nat. Biotechnol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">644</span><span class="lpage" tagx="lpage" title="lpage">652</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.1883">10.1038/nbt.1883</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a></span></span></li> <li tag="ref"><a name="B25-viruses-12-00287" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">CAP3: A DNA sequence assembly program</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">868</span><span class="lpage" tagx="lpage" title="lpage">877</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.9.9.868">10.1101/gr.9.9.868</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846">10508846</a></span></span></li> <li tag="ref"><a name="B26-viruses-12-00287" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchfink</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huson</span><span class="given-names" tagx="given-names" title="given-names">D.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast and sensitive protein alignment using DIAMOND</span><span class="source" tagx="source" title="source">Nat. Methods</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">59</span><span class="lpage" tagx="lpage" title="lpage">60</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.3176">10.1038/nmeth.3176</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25402007">25402007</a></span></span></li> <li tag="ref"><a name="B27-viruses-12-00287" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langmead</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with Bowtie 2</span><span class="source" tagx="source" title="source">Nat. Methods</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a></span></span></li> <li tag="ref"><a name="B28-viruses-12-00287" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">X.-D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">J.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">C.-X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">X.-C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">J.-P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eden</span><span class="given-names" tagx="given-names" title="given-names">J.-S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Redefining the invertebrate RNA virosphere</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">540</span><span class="fpage" tagx="fpage" title="fpage">539</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature20167">10.1038/nature20167</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27880757">27880757</a></span></span></li> <li tag="ref"><a name="B29-viruses-12-00287" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Standley</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">MAFFT multiple sequence alignment software version 7: Improvements in performance and usability</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">772</span><span class="lpage" tagx="lpage" title="lpage">780</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a></span></span></li> <li tag="ref"><a name="B30-viruses-12-00287" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trifinopoulos</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">L.-T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Haeseler</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minh</span><span class="given-names" tagx="given-names" title="given-names">B.Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">W-IQ-TREE: A fast online phylogenetic tool for maximum likelihood analysis</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">W232</span><span class="lpage" tagx="lpage" title="lpage">W235</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw256">10.1093/nar/gkw256</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27084950">27084950</a></span></span></li> <li tag="ref"><a name="B31-viruses-12-00287" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lefkowitz</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carstens</span><span class="given-names" tagx="given-names" title="given-names">E.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Iflaviridae</span><span class="source" tagx="source" title="source">Virus Taxonomy: Ninth Report of the International Committee on Taxonomy of Viruses</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lefkowitz</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carstens</span><span class="given-names" tagx="given-names" title="given-names">E.B.</span></span></span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Diego, CA, USA</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">848</span></span></li> <li tag="ref"><a name="B32-viruses-12-00287" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spear</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sisterson</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yokomi</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stenger</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Plant-feeding insects harbor double-stranded RNA viruses encoding a novel proline-alanine rich protein and a polymerase distantly related to that of fungal viruses</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">404</span><span class="fpage" tagx="fpage" title="fpage">304</span><span class="lpage" tagx="lpage" title="lpage">311</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2010.05.015">10.1016/j.virol.2010.05.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20541786">20541786</a></span></span></li> <li tag="ref"><a name="B33-viruses-12-00287" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffin</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coutts</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Relationships among Trialeurodes vaporariorum—Transmitted Yellowing Viruses from Europe and North America</span><span class="source" tagx="source" title="source">J. Phytopathol.</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">143</span><span class="fpage" tagx="fpage" title="fpage">375</span><span class="lpage" tagx="lpage" title="lpage">380</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1439-0434.1995.tb00278.x">10.1111/j.1439-0434.1995.tb00278.x</a></span></span></li> <li tag="ref"><a name="B34-viruses-12-00287" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ito</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzaki</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakano</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic characterization of novel putative rhabdovirus and dsRNA virus from Japanese persimmon</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">94</span><span class="fpage" tagx="fpage" title="fpage">1917</span><span class="lpage" tagx="lpage" title="lpage">1921</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.054445-0">10.1099/vir.0.054445-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23704275">23704275</a></span></span></li> <li tag="ref"><a name="B35-viruses-12-00287" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stenger</span><span class="given-names" tagx="given-names" title="given-names">D.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sisterson</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krugner</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Backus</span><span class="given-names" tagx="given-names" title="given-names">E.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">W.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new Phytoreovirus infecting the glassy-winged sharpshooter (Homalodisca vitripennis)</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">386</span><span class="fpage" tagx="fpage" title="fpage">469</span><span class="lpage" tagx="lpage" title="lpage">477</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2009.01.037">10.1016/j.virol.2009.01.037</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19239972">19239972</a></span></span></li> <li tag="ref"><a name="B36-viruses-12-00287" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitbart</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salamon</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andresen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahaffy</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Segall</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mead</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azam</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohwer</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic analysis of uncultured marine viral communities</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">14250</span><span class="lpage" tagx="lpage" title="lpage">14255</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.202488399">10.1073/pnas.202488399</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12384570">12384570</a></span></span></li> <li tag="ref"><a name="B37-viruses-12-00287" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Expansion of the metazoan virosphere: Progress, pitfalls, and prospects</span><span class="source" tagx="source" title="source">Curr. Opin. Virol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">17</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2018.08.008">10.1016/j.coviro.2018.08.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30237139">30237139</a></span></span></li> <li tag="ref"><a name="B38-viruses-12-00287" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benkő</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitbart</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brister</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carstens</span><span class="given-names" tagx="given-names" title="given-names">E.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davison</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delwart</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorbalenya</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrach</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Consensus statement: Virus taxonomy in the age of metagenomics</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">161</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro.2016.177">10.1038/nrmicro.2016.177</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28134265">28134265</a></span></span></li> <li tag="ref"><a name="B39-viruses-12-00287" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonning</span><span class="given-names" tagx="given-names" title="given-names">B.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Insect Virome: Opportunities and Challenges</span><span class="source" tagx="source" title="source">Curr. Issues Mol. Biol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">12</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31167953">31167953</a></span></span></li> <li tag="ref"><a name="B40-viruses-12-00287" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vainio</span><span class="given-names" tagx="given-names" title="given-names">E.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiba</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghabrial</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maiss</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roossinck</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabanadzovic</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nibert</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ictv Report Consortium ICTV Virus Taxonomy Profile: Partitiviridae</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">17</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/jgv.0.000985">10.1099/jgv.0.000985</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29214972">29214972</a></span></span></li> <li tag="ref"><a name="B41-viruses-12-00287" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lara Pinto</span><span class="given-names" tagx="given-names" title="given-names">A.Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Santos de Carvalho</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Melo</span><span class="given-names" tagx="given-names" title="given-names">F.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribeiro</span><span class="given-names" tagx="given-names" title="given-names">A.L.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morais Ribeiro</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dezengrini Slhessarenko</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Novel viruses in salivary glands of mosquitoes from sylvatic Cerrado, Midwestern Brazil</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0187429</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0187429">10.1371/journal.pone.0187429</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29117239">29117239</a></span></span></li> <li tag="ref"><a name="B42-viruses-12-00287" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pettersson</span><span class="given-names" tagx="given-names" title="given-names">H.-O.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eden</span><span class="given-names" tagx="given-names" title="given-names">J.-S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">C.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hesson</span><span class="given-names" tagx="given-names" title="given-names">C.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Meta-Transcriptomic Comparison of the RNA Viromes of the Mosquito Vectors Culex pipiens and Culex torrentium in Northern Europe</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1033</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v11111033">10.3390/v11111033</a></span></span></li> <li tag="ref"><a name="B43-viruses-12-00287" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trivellone</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="citation_author_institution" id="">[], </span></span><span class="source" tagx="source" title="source">Personal Communication</span><span class="year" tagx="year" title="year">2018</span></span></li> <li tag="ref"><a name="B44-viruses-12-00287" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keane</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crawley</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exotic plant invasions and the enemy release hypothesis</span><span class="source" tagx="source" title="source">Trends Ecol. Evol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">164</span><span class="lpage" tagx="lpage" title="lpage">170</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-5347(02)02499-0">10.1016/S0169-5347(02)02499-0</a></span></span></li> <li tag="ref"><a name="B45-viruses-12-00287" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colautti</span><span class="given-names" tagx="given-names" title="given-names">R.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ricciardi</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grigorovich</span><span class="given-names" tagx="given-names" title="given-names">I.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacIsaac</span><span class="given-names" tagx="given-names" title="given-names">H.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Is invasion success explained by the enemy release hypothesis?</span><span class="source" tagx="source" title="source">Ecol. Lett.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">721</span><span class="lpage" tagx="lpage" title="lpage">733</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1461-0248.2004.00616.x">10.1111/j.1461-0248.2004.00616.x</a></span></span></li> <li tag="ref"><a name="B46-viruses-12-00287" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medd</span><span class="given-names" tagx="given-names" title="given-names">N.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fellous</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waldron</span><span class="given-names" tagx="given-names" title="given-names">F.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xuéreb</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakai</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cross</span><span class="given-names" tagx="given-names" title="given-names">J.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The virome of Drosophila suzukii, an invasive pest of soft fruit</span><span class="source" tagx="source" title="source">Virus Evol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vey009">10.1093/ve/vey009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29644097">29644097</a></span></span></li> <li tag="ref"><a name="B47-viruses-12-00287" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramirez</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Souza-Neto</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cosme</span><span class="given-names" tagx="given-names" title="given-names">R.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rovira</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ortiz</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pascale</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimopoulos</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reciprocal tripartite interactions between the Aedes aegypti midgut microbiota, innate immune system and dengue virus influences vector competence</span><span class="source" tagx="source" title="source">PLoS Negl. Trop. Dis.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1561</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0001561">10.1371/journal.pntd.0001561</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22413032">22413032</a></span></span></li> <li tag="ref"><a name="B48-viruses-12-00287" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramirez</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimopoulos</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Aedes aegypti toll pathway controls dengue virus infection</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000098</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1000098">10.1371/journal.ppat.1000098</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18604274">18604274</a></span></span></li> <li tag="ref"><a name="B49-viruses-12-00287" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glick</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paldi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bextine</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Advances in RNA interference: dsRNA Treatment in Trees and Grapevines for Insect Pest Suppression</span><span class="source" tagx="source" title="source">Southwest. Entomol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">87</span><span class="pub-id"><a href="https://dx.doi.org/10.3958/059.037.0110">10.3958/059.037.0110</a></span></span></li> <li tag="ref"><a name="B50-viruses-12-00287" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Killiny</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hajeri</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tiwari</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gowda</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stelinski</span><span class="given-names" tagx="given-names" title="given-names">L.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Double-Stranded RNA Uptake through Topical Application, Mediates Silencing of Five CYP4 Genes and Suppresses Insecticide Resistance in Diaphorina citri</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span></span></li> <li tag="ref"><a name="B51-viruses-12-00287" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cook</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">B.Y.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bass</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moureau</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McAlister</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Culverwell</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glücksman</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">T.D.K.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Novel Virus Discovery and Genome Reconstruction from Field RNA Samples Reveals Highly Divergent Viruses in Dipteran Hosts</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e80720</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0080720">10.1371/journal.pone.0080720</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24260463">24260463</a></span></span></li> <li tag="ref"><a name="B52-viruses-12-00287" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fauver</span><span class="given-names" tagx="given-names" title="given-names">J.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grubaugh</span><span class="given-names" tagx="given-names" title="given-names">N.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krajacich</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weger-Lucarelli</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lakin</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fakoli</span><span class="given-names" tagx="given-names" title="given-names">L.S.</span><span class="suffix" tagx="suffix" title="suffix">III</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolay</span><span class="given-names" tagx="given-names" title="given-names">F.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diclaro</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span><span class="suffix" tagx="suffix" title="suffix">II</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dabiré</span><span class="given-names" tagx="given-names" title="given-names">K.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foy</span><span class="given-names" tagx="given-names" title="given-names">B.D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">West African Anopheles gambiae mosquitoes harbor a taxonomically diverse virome including new insect-specific flaviviruses, mononegaviruses, and totiviruses</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">498</span><span class="fpage" tagx="fpage" title="fpage">288</span><span class="lpage" tagx="lpage" title="lpage">299</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2016.07.031">10.1016/j.virol.2016.07.031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27639161">27639161</a></span></span></li> <li tag="ref"><a name="B53-viruses-12-00287" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zakrzewski</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rašić</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darbro</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poo</span><span class="given-names" tagx="given-names" title="given-names">Y.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Filipović</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parry</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asgari</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devine</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suhrbier</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mapping the virome in wild-caught Aedes aegypti from Cairns and Bangkok</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">4690</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-22945-y">10.1038/s41598-018-22945-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29549363">29549363</a></span></span></li> <li tag="ref"><a name="B54-viruses-12-00287" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asad</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khromykh</span><span class="given-names" tagx="given-names" title="given-names">A.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asgari</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cell fusing agent virus and dengue virus mutually interact in Aedes aegypti cell lines</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">6935</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-07279-5">10.1038/s41598-017-07279-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28761113">28761113</a></span></span></li> <li tag="ref"><a name="B55-viruses-12-00287" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadeghi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altan</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barker</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffey</span><span class="given-names" tagx="given-names" title="given-names">L.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delwart</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Virome of &amp;gt; 12 thousand Culex mosquitoes from throughout California</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">523</span><span class="fpage" tagx="fpage" title="fpage">74</span><span class="lpage" tagx="lpage" title="lpage">88</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2018.07.029">10.1016/j.virol.2018.07.029</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30098450">30098450</a></span></span></li> <li tag="ref"><a name="B56-viruses-12-00287" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosario</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capobianco</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">T.F.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breitbart</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polston</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RNA viral metagenome of whiteflies leads to the discovery and characterization of a whitefly-transmitted carlavirus in North America</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e86748</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0086748">10.1371/journal.pone.0086748</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24466220">24466220</a></span></span></li> <li tag="ref"><a name="B57-viruses-12-00287" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nouri</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salem</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nigg</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falk</span><span class="given-names" tagx="given-names" title="given-names">B.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diverse Array of New Viral Sequences Identified in Worldwide Populations of the Asian Citrus Psyllid (Diaphorina citri) Using Viral Metagenomics</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">2434</span><span class="lpage" tagx="lpage" title="lpage">2445</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02793-15">10.1128/JVI.02793-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26676774">26676774</a></span></span></li> <li tag="ref"><a name="B58-viruses-12-00287" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wamonje</span><span class="given-names" tagx="given-names" title="given-names">F.O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michuki</span><span class="given-names" tagx="given-names" title="given-names">G.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braidwood</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Njuguna</span><span class="given-names" tagx="given-names" title="given-names">J.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Musembi Mutuku</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Djikeng</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harvey</span><span class="given-names" tagx="given-names" title="given-names">J.J.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carr</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viral metagenomics of aphids present in bean and maize plots on mixed-use farms in Kenya reveals the presence of three dicistroviruses including a novel Big Sioux River virus-like dicistrovirus</span><span class="source" tagx="source" title="source">Virol. J.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">188</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12985-017-0854-x">10.1186/s12985-017-0854-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28969654">28969654</a></span></span></li> <li tag="ref"><a name="B59-viruses-12-00287" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Atoni</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karungu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waruhiu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zohaib</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obanda</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agwanda</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mutua</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Metagenomic virome analysis of Culex mosquitoes from Kenya and China</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">30</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v10010030">10.3390/v10010030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29329230">29329230</a></span></span></li> <li tag="ref"><a name="B60-viruses-12-00287" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nanfack-Minkeu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitri</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bischoff</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belda</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casademont</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vernick</span><span class="given-names" tagx="given-names" title="given-names">K.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Interaction of RNA viruses of the natural virome with the African malaria vector, Anopheles coluzzii</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">6319</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-019-42825-3">10.1038/s41598-019-42825-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31004099">31004099</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Maximum likelihood phylogeny of +ssRNA viruses. Putative viruses that may infect <i>S. titanus</i> are in bold. All the phylogenies described in this manuscript used virus sequences and nomenclature as provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. Tree (<b>A</b>): Scaphoideus titanus iflavirus 1 and Scaphoideus titanus iflavirus 2 within the "Iflaviridae-Secoviridae related cluster�?. GenBank accession numbers of the viruses included in the phylogenetic analysis are as follows: Euscelidius variegatus virus 1, APD68841.1; Psammotettix alienus iflavirus 1, YP_009553259.1; Graminella nigrifrons virus 1, YP_009129265.1; Hubei tetragnatha maxillosa virus 2, APG77446.1; Moku virus, YP_009305421.1; Slow bee paralysis virus, YP_003622540; Hubei odonate virus 4, YP_009337760.1; Hubei picorna-like virus 28, YP_009337284.1. Tree (<b>B</b>): Scaphoideus titanus sobemo-like virus 1 and Scaphoideus titanus sobemo-like virus 2 within the "Luteo-Sobemo�? clade. GenBank accession numbers of the viruses included in the phylogenetic analysis are as follows: Hubei sobemo-like virus 24, YP_009330073.1; Shuangao sobemo-like virus 1, KX882824.1; Sanxia sobemo-like virus 4, YP_009336857.1; Hubei sobemo-like virus 31, YP_009330060.1; Hubei tetragnatha maxillosa virus 6, APG75951.1; Hubei sobemo-like virus 33, YP_009330069.1; Sanxia sobemo-like virus 5, YP_009336914.1; Hubei sobemo-like virus 27, YP_009330071.1; Hubei diptera virus 13, YP_009337911.1; Hubei sobemo-like virus 28, YP_009330115.1; Hubei sobemo-like virus 26, YP_009329970.1; Hubei sobemo-like virus 21, APG75719.1; Hubei diptera virus 12, YP_009337873.1; Sanxia sobemo-like virus 3, YP_009336906.1; Hubei sobemo-like virus 20, YP_009330033.1; Hubei sobemo-like virus 18, APG75949.1; Hubei sobemo-like virus 19, YP_009329966.1. Tree (<b>C</b>): Scaphoideus titanus permutotetra-like virus 1 within the "Permutotetra�? clade. GenBank accession numbers of the viruses included in the phylogenetic analysis are as follows: Hubei permutotetra-like virus 4, YP_009337378.1; Aphis glycines virus 1, YP_009179352_; Hubei permutotetra-like virus 7, YP_009337364.1; Daeseongdong virus 2, YP_009182196; Hubei_permutotetra-like virus 11, YP_009337276.1; Drosophila A virus, YP_003038595; Hubei permutotetra-like virus 6, YP_009337318.1; Hubei permutotetra-like virus 5, YP_009337742.1; Hubei permutotetra-like virus 3, YP_009337623.1; Newfield virus, AKH40283; Wuhan house centipede virus 9, APG76963.1; Hubei permutotetra-like virus 1, YP_009337300.1; Hubei permutotetra-like virus 2, APG76964.1; Sanxia_permutotetra-like virus 1, YP_009337650.1; Shuangao permutotetra-like virus 1, KX883439.1; Sanxia water strider virus 19, YP_009337667.1; Hubei permutotetra-like virus 9, YP_009337778.1; Hubei permutotetra-like virus 8, YP_009337703.1.Only bootstrap values higher than 70 are shown. Complete trees are provided in <a href="#app1-viruses-12-00287">Supplementary Materials S5–S7</a>. All viruses included in this figure were isolated from members of the class Insecta, with the exceptions of those with silhouetted animals on the right. Silhouettes were downloaded from <span class="uri" title="uri"><a href="http://www.iconarchive.com/">http://www.iconarchive.com/</a></span>. The scale bars indicate the evolutionary distance expressed as amino acid substitutions per site.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Structures of Scaphoideus titanus permutotetra-like virus 1 and Scaphoideus titanus toti-like virus 1. (<b>A</b>) The organization of the genome of Scaphoideus titanus permutotetra-like virus 1; (<b>B</b>) the organization of the genomes of Scaphoideus titanus toti-like virus 1 found in the St_IT2, St_FR and St_CH libraries is the same, but the 5′ and 3′ UTRs show different lengths. In this figure we specifically illustrated the genome assembled from the St_IT2 library. Boxes represent boundaries of ORFs. RdRp: RNA-dependent RNA polymerase. Hyp: hypothetical protein. Capsid prt: capsid protein. PArp: proline-alanine rich protein.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Maximum likelihood phylogeny of -ssRNA and dsRNA viruses. Putative viruses that may infect <i>S. titanus</i> are in bold. All the phylogenies described in this manuscript used virus sequences and nomenclature as provided by Shi et al. [<a href="#B28-viruses-12-00287">28</a>]. Tree (<b>A</b>): Scaphoideus titanus bunya-like virus 1 within the "Bunya-Arena�? clade; GenBank accession numbers of the viruses included in the phylogenetic analysis are as follows: Hubei bunya-like virus 9, APG79278.1; Shuangao Insect Virus 2, YP_009664561.1; Wuhan Mosquito Virus 2, YP_009305135.1; Kigluaik phantom virus, AIA24559.1; Wuhan mosquito virus 1, YP_009305130.1; Hubei diptera virus 7, APG79296.1; Hubei diptera virus 6, APG79294.1; Hubei odonate virus 8, APG79264.1; Hubei odonate virus 9, YP_009329871.1; Wuchang Cockroach Virus 1, YP_009304995.1; Ganda bee virus, APT68154.1; Hubei bunya-like virus, 8 APG79291.1. The collapsed branched named "<i>Phasmaviridae�?</i> grouped five sequences: Jonchet virus, AKN56871; Hubei bunya-like virus 10, APG79271.1; Wuhan Insect virus 2, YP_009270651.1; Sanxia Water Strider Virus 2, AJG39245.1; Ferak virus, AKN56888. Tree (<b>B</b>): Scaphoideus titanus toti-like virus 1 within the "Toti-Chryso�? clade. GenBank accession numbers of the viruses included in the phylogenetic analysis are as follows: Cucurbit yellows-associated virus, CAA63099.2; Spissistilus festinus virus 1, YP_003800001.1; Circulifer tenellus virus 1, YP_003800003.1; Barrymore virus, QED21514.1; Persimmon latent virus, YP_009025166.1. Tree (<b>C</b>): Scaphoideus titanus reo-like virus 1 within the "Reo�? clade. GenBank accession numbers of the viruses included in the phylogenetic analysis are as follows: Hubei reo-like virus 11, APG79151.1; Hubei reo-like virus 10, APG79149.1; Rice gall dwarf virus, YP_001111373; Rice dwarf virus, NP_620544; Homalodisca vitripennis reovirus, YP_002790884. Tree (<b>D</b>): Scaphoideus titanus associated partiti-like virus 1 within the "Partiti-Picobirna�? clade. GenBank accession numbers of the viruses included in the phylogenetic analysis are as follows: Beihai partiti-like virus 11, APG78185.1; Hubei partiti-like virus 57, APG78229.1; Changjiang partiti-like virus 1, APG78201.1; Hubei partiti-like virus 56, APG78242.1; Wuhan large pig roundworm virus 1, YP_009329866.1; Xinzhou partiti-like virus 1, YP_009329867.1; Ustilaginoidea virens partitivirus 2, YP_008327312; Hubei partiti-like virus 55, APG78299.1; Hubei partiti-like virus 53, APG78297.1; Hubei partiti-like virus 54, APG78298.1. The collapsed branched named "<i>Deltapartitivirus�?</i> grouped six sequences: Fig cryptic virus, YP_004429258; Rose cryptic virus 1, YP_001686786; Raphanus sativus cryptic virus 2, YP_001686783; Raphanus sativus cryptic virus 3, YP_002364401; Hubei partiti-like virus 58, APG78223.1; Persimmon cryptic virus, YP_006390091. The collapsed branched named "<i>Gammapartitivirus�?</i> grouped five sequences: Fusarium solani virus 1, NP_624350.1; Penicillium stoloniferum virus S, YP_052856.2; Gremmeniella abietina RNA virus MS1, NP_659027.1; Botryotinia fuckeliana partitivirus 1, YP_001686789.1; Discula destructiva virus 1, NP_116716.1. Only bootstrap values higher than 70 are shown. Complete trees are provided in <a href="#app1-viruses-12-00287">Supplementary Materialss S8, S9, S10, and S11</a>.All viruses included in this figure were isolated from members of the class Insecta, with the exceptions of those with silhouetted animals, fungi or plants on the right. Silhouettes were downloaded from <span class="uri" title="uri"><a href="http://www.iconarchive.com/">http://www.iconarchive.com/</a></span>. The scale bars indicate the evolutionary distance expressed as amino acid substitutions per site.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Virus clades and clusters identified in Europe and in the USA. Each sunburst reports prevalence of the newly identified virus in the 5 sampling countries and assignment to viral clades and clusters, according to Shi et al. [<a href="#B28-viruses-12-00287">28</a>] (see also <a href="#viruses-12-00287-t001">Table 1</a>). Sunbursts were generated by R (version 3.4.4) packages "cowplot�? and "tidyverse�?.</p>   </div> <div class="table-wrap_UNKNOWN" id="viruses-12-00287-t001" orientation="portrait" position="float">
viruses-12-00287-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>Sampling information, RNA-seq pooling strategies and prevalence of Scaphoideus titanus iflavirus 1 and Scaphoideus titanus toti-like virus 1 in the European <i>S. titanus</i> populations.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">RNA-seq<br /><br />LibrariesID</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1"><b>Country</b></th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">State/Region</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Sampling Sites</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Latitude (Decimal Degrees)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Longitude (Decimal Degrees)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Sampling Date<br /><br />(yyyy/mm/dd)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Number of Sampled Individuals</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Positive to Scaphoideus titanus Iflavirus 1</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Positive to Scaphoideus titanustoti-Like Virus 1</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_HU</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Hungary</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Pest</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Monor</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">47.354861</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">19.472177</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-07-12</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">20</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_HU</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Hungary</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Pest</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Gomba</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">47.349622</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">19.505863</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-07-12</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">18</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_FR</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">France</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Burgundy</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Corgoloin</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">47.095667</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">4.908233</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2018-07-06</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">20</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">8</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_FR</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">France</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Dordogne</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Saint-Nexans</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">44.472132</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0.345279</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2018-07-10</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">18</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_IT1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Piedmont</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Asti</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">44.921854</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8.195758</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-07-31</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_IT1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Piedmont</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Montà</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">44.807434</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.965787</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-07-31</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">12</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_IT1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Piedmont</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Portacomaro</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">44.962898</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8.26072</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-08-03</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">16</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_IT1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Piedmont</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Cisterna</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">44.82591</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8.008371</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-08-03</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_IT1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Lombardy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Ome</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">45.6302972</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">10.129867</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-07-11</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_IT1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Veneto</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Verona</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">45.4227833</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">11.025431</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-08-08</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">8</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_IT2</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Friuli-Venezia Giulia</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Togliano</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">46.1129556</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">13.411311</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2018-07-30</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">22</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_IT2</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Friuli-Venezia Giulia</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Savorgnano al Torre</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">46.161786</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">13.286864</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2018-07-31</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">24</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_IT2</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Abruzzo</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Vacri</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">42.290333</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">14.223861</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2018-07-16</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">12</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_CH</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Switzerland</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Canton Vaud</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Gland</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">46.42616</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">6.28217</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-08-09</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">16</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">4</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_CH</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Switzerland</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Canton Valais</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Pramagnon</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">46.24987</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.45371</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-08-09</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">16</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>214</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>2</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>18</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Total</b></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Illinois</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Muncie</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">40.0893</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">-87.8323</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2018-06-11</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">3</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td rowspan="3" align="center" valign="middle" style="border-bottom:solid thin" colspan="1"><b>Insects for DNA extraction<sup>1</sup></b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Country</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>State/Region</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Sampling Sites</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Latitude (Decimal Degrees)</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Longitude (Decimal Degrees)</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Sampling Date</b><br /><br /><b>(yyyy/mm/dd)</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><b>Number of Sampled Individuals</b></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Piedmont</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Montà</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">44.807434</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.965787</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-07-31</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Italy</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Piedmont</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Dogliani</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">44.5362837</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7.9312913</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2018-08-20</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1" /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>1</sup> The second section of the table reports sampling information about insects used to demonstrate the integration of some viral sequences into the insect genome.</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" id="viruses-12-00287-t002" orientation="portrait" position="float">
viruses-12-00287-t002_Table 2<span class="label" tagx="label" title="label">Table 2</span> <p>List of the sequences assigned to putative insect-infecting viruses found in each RNA-seq library. Column 9 reports the kind (+ssRNA, -ssRNA and dsRNA) and structure (monopartite/multipartite) of the viral genomes identified in this work. The identification of segments in some multipartite genomes is also indicated.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Library ID</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Accession Numbers</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Sequence Length (bp)</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Best Blastx Hit Description</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">% Identity</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">% Query Coverage</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">RPKM</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Tentative Name</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Genome</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_IT1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MN982380</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">10729</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">YP_009129265.1 polyprotein [Graminella nigrifrons virus 1] </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">51</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">83</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.01</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Scaphoideus titanus iflavirus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">+ssRNA (monopartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">St_IT2</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">MN982382 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">7847</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">YP_003800001.1 RNA-directed RNA polymerase partial [Spissistilus festinus virus 1] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">42</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">37</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">0.002</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">Scaphoideus titanus toti-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#EEEEEE" rowspan="1" colspan="1">dsRNA (monopartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_CH</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MN982383 </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7839</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">YP_003800001.1 RNA-directed RNA polymerase partial [Spissistilus festinus virus 1] </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">42</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">37</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.032</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Scaphoideus titanus toti-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">dsRNA (monopartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_FR</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982379 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">10693</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">YP_009129265.1 polyprotein [Graminella nigrifrons virus 1] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">51</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">83</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0.004</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus iflavirus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">+ssRNA (monopartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_FR</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MN982381 </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7871</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">YP_003800001.1 RNA-directed RNA polymerase partial [Spissistilus festinus virus 1] </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">42</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">37</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.023</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Scaphoideus titanus toti-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">dsRNA (monopartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982384 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">4361</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">API65464.1 glycoprotein precursor [Sanxia Water Strider Virus 2] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">25</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">12</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1.016</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus bunya-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">−ssRNA (tripartite): segment M</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982385 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1458</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">APG79297.1 putative nucleoprotein [Hubei diptera virus 7] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">27</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">43</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0.9</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus bunya-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">−ssRNA (tripartite): segment S</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982386 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">7740</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">APT68154.1 RNA-dependent RNA polymerase [Ganda bee virus]</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">33</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">74</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0.359</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus bunya-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">−ssRNA (tripartite): segment L</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MN982387 </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">10519</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">YP_009129265.1 polyprotein [Graminella nigrifrons virus 1] </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">51</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">85</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.045</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Scaphoideus titanus iflavirus 2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">+ssRNA (monopartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982388 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1455</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">AWA82259.1 hypothetical protein, partial [Naganuma virus]</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">38</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">91</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0.145</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus sobemo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">+ssRNA (bipartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982389 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">3213</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">YP_009329970.1 hypothetical protein [Hubei sobemo-like virus 26] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">60</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">99</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0.103</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus sobemo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">+ssRNA (bipartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MN982390 </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1550</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">AWA82259.1 hypothetical protein partial [Naganuma virus]</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">36</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">77</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.231</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Scaphoideus titanus sobemo-like virus 2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">+ssRNA (bipartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MN982391 </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">3100</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">YP_009330073.1 hypothetical protein 2 [Hubei sobemo-like virus 24] </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">46</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">97</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.172</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Scaphoideus titanus sobemo-like virus 2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">+ssRNA (bipartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982393 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1303</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">YP_009329884.1 hypothetical protein [Wuhan large pig roundworm virus 1] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">35</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">61</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0.227</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus-associated partiti-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (bipartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982392 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1558</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">AVV63192.1 RNA-directed RNA polymerase [Wuhan large pig roundworm virus]</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">44</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">87</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">0.272</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus-associated partiti-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (bipartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">MN982394 </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">5239</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">YP_009337778.1 RdRp [Hubei permutotetra-like virus 9] </td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">39</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">39</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.764</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Scaphoideus titanus permutotetra-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">+ssRNA (monopartite)</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982397 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1240</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">ADN64742.1 non-structural protein [Homalodisca vitripennis reo-like virus] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">79</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">58</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2.378</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 10</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982398 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1867</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">ADN64768.1 minor core protein [Homalodisca vitripennis reo-like virus]</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">83</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">42</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1.146</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 7</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982399 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2937</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">ADN64783.1 minor core protein [Homalodisca vitripennis reo-like virus]</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">52</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">49</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2.661</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 5</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982395 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">4957</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">ADN64816.1 RNA-directed RNA polymerase [Homalodisca vitripennis reo-like virus] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">87</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">61</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">5.053</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 1</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982400 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">3560</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">YP_002790885.1 RNA-binding protein [Homalodisca vitripennis reo-like virus] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">96</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">49</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">5.444</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 2</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982401 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1416</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Q85451.1 RecName: Full=Outer capsid protein P8</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">62</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">88</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1.679</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 8</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982402 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">3179</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">ADN64799.1 major core protein [Homalodisca vitripennis reo-like virus]</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">95</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">55</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1.901</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 3</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982403 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">2648</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">YP_002790887.1 zinc-finger protein [Homalodisca vitripennis reo-like virus] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">73</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">30</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">3.571</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 4</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982404 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1778</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">YP_002790889.1 non-structural protein [Homalodisca vitripennis reo-like virus] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">78</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">33</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1.196</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 6</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982396 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1179</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">YP_002790892.1 non-structural protein [Homalodisca vitripennis reo-like virus] </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">68</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">47</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">1.802</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 9</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">St_USA</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">MN982405 </td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">798</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">YP_002790894.1 non-structural protein [Homalodisca vitripennis reo-like virus]</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">48</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">45</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">4.192</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">Scaphoideus titanus reo-like virus 1</td> <td align="center" valign="middle" style="border-bottom:solid thin;background:#DDDDDD" rowspan="1" colspan="1">dsRNA (multipartite): segment 11</td> </tr> </tbody> </table> </div> </div>  </body></html>