<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/30/2020.03.25.006213.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.25.006213v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution" />
<meta name="DC.Identifier" content="10.1101/2020.03.25.006213" />
<meta name="DC.Date" content="2020-03-30" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The Coronavirus disease-19 (COVID19) that started in Wuhan, China in December 2019 has spread worldwide emerging as a global pandemic. The severe respiratory pneumonia caused by the novel SARS-CoV-2 has so far claimed more than 14,500 lives and has impacted human lives worldwide. Development of universal vaccines against the novel SARS-CoV-2 holds utmost urgency to control COVID19 pandemic that appears to be more severe than any of the previous outbreaks of severe acute respiratory syndrome (SARS) and Middle-East respiratory syndrome (MERS). However, as the novel SARS-CoV-2 displays high transmission rates, the underlying severity hidden in the SARS-CoV2 genomes is required to be fully understood. We studied the complete genomes of 95 strains of SARS-CoV-2 reported from different geographical regions worldwide to uncover the pattern of spread of the novel SARS-CoV-2 across the globe. We show that there is no direct transmission pattern of the virus among neighbouring countries suggesting that the outbreak is a result of travel of infected humans to different countries. We revealed unique single nucleotide polymorphisms (SNPs) in nsp13, nsp14, nsp15, nsp16 (present in ORF1b polyprotein region) and S-Protein within 10 viral isolates from USA. These viral proteins are involved in RNA replication and processing, indicating highly evolved strains of the novel SARS-CoV-2 circulating in the population of USA than in other countries. Furthermore, we found an isolate from USA (MT188341) to carry frameshift mutation between positions 2540 and 2570 of nsp16 which functions as mRNA cap-1 methyltransferase (2′-O-MTase). Thus, we reason that the replicative machinery of the novel SARS-CoV-2 is fast evolving to evade host challenges and survival. These mutations are needed to be considered, otherwise it will be difficult to develop effective treatment strategies. The two proteins also had dN/dS values approaching 1-ORF1ab polyprotein (dN/dS= 0.996, 0.575) and S protein (dN/dS= 0.88) and might confer selective advantage to the virus. Through the construction of SARS-CoV-2-human interactome, we further reveal multiple host proteins (PHB, PPP1CA, TGF-beta, JACK1, JACK2, SOCS3,STAT3, JAK1-2, SMAD3, BCL2, CAV1 &amp; SPECC1) which are manipulated by the virus proteins (nsp2, PL-PRO, N-protein, ORF7a, M-S-ORF3a complex, nsp7-nsp8-nsp9-RdRp complex) for mediating host immune mechanism for its survival." />
<meta name="DC.Contributor" content="Roshan Kumar" />
<meta name="DC.Contributor" content="Helianthous Verma" />
<meta name="DC.Contributor" content="Nirjara Singhvi" />
<meta name="DC.Contributor" content="Utkarsh Sood" />
<meta name="DC.Contributor" content="Vipin Gupta" />
<meta name="DC.Contributor" content="Mona Singh" />
<meta name="DC.Contributor" content="Rashmi Kumari" />
<meta name="DC.Contributor" content="Princy Hira" />
<meta name="DC.Contributor" content="Shekhar Nagar" />
<meta name="DC.Contributor" content="Chandni Talwar" />
<meta name="DC.Contributor" content="Namita Nayyar" />
<meta name="DC.Contributor" content="Shailly Anand" />
<meta name="DC.Contributor" content="Charu Dogra Rawat" />
<meta name="DC.Contributor" content="Mansi Verma" />
<meta name="DC.Contributor" content="Ram Krishan Negi" />
<meta name="DC.Contributor" content="Yogendra Singh" />
<meta name="DC.Contributor" content="Rup Lal" />
<meta name="article:published_time" content="2020-03-30" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Coronavirus disease -19 (COVID19) that started in Wuhan, China in December 2019 has spread worldwide emerging as a global pandemic. The severe respiratory pneumonia caused by the novel SARS-CoV-2 has so far claimed more than 14,500 lives and has impacted human lives worldwide. Development of universal vaccines against the novel SARS-CoV-2 holds utmost urgency to control COVID19 pandemic that appears to be more severe than any of the previous outbreaks of severe acute respiratory syndrome (SARS) and Middle-East respiratory syndrome (MERS). However, as the novel SARS-CoV-2 displays high transmission rates, the underlying severity hidden in the SARS-CoV2 genomes is required to be fully understood. We studied the complete genomes of 95 strains of SARS-CoV-2 reported from different geographical regions worldwide to uncover the pattern of spread of the novel SARS-CoV-2 across the globe. We show that there is no direct transmission pattern of the virus among neighbouring countries suggesting that the outbreak is a result of travel of infected humans to different countries. We revealed unique single nucleotide polymorphisms (SNPs) in nsp13, nsp14, nsp15, nsp16 (present in ORF1b polyprotein region) and S-Protein within 10 viral isolates from USA. These viral proteins are involved in RNA replication and processing, indicating highly evolved strains of the novel SARS-CoV-2 circulating in the population of USA than in other countries. Furthermore, we found an isolate from USA (MT188341) to carry frameshift mutation between positions 2540 and 2570 of nsp16 which functions as mRNA cap-1 methyltransferase (2′-O-MTase). Thus, we reason that the replicative machinery of the novel SARS-CoV-2 is fast evolving to evade host challenges and survival. These mutations are needed to be considered, otherwise it will be difficult to develop effective treatment strategies. The two proteins also had dN/dS values approaching 1- ORF1ab polyprotein (dN/dS= 0.996, 0.575) and S protein (dN/dS= 0.88) and might confer selective advantage to the virus. Through the construction of SARS-CoV-2-human interactome, we further reveal multiple host proteins (PHB, PPP1CA, TGF-beta, JACK1, JACK2, SOCS3,STAT3, JAK1-2,  SMAD3, BCL2, CAV1 &amp;amp; SPECC1)  which are manipulated by the virus proteins (nsp2, PL-PRO, N-protein, ORF7a, M-S-ORF3a complex, nsp7-nsp8-nsp9-RdRp complex) for mediating host immune mechanism for its survival.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.25.006213v1" />
<meta name="citation_id" content="2020.03.25.006213v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.006213v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.006213v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.006213v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.25.006213" />
<meta name="citation_num_pages" content="40" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.25.006213" />
<meta name="citation_author" content="Roshan Kumar" />
<meta name="citation_author_institution" content="P.G. Department of Zoology, Magadh University" />
<meta name="citation_author" content="Helianthous Verma" />
<meta name="citation_author_institution" content="Department of Zoology, Ramjas College, University of Delhi" />
<meta name="citation_author" content="Nirjara Singhvi" />
<meta name="citation_author_institution" content="Department of Zoology, University of Delhi" />
<meta name="citation_author" content="Utkarsh Sood" />
<meta name="citation_author_institution" content="The Energy and Resources Institute" />
<meta name="citation_author" content="Vipin Gupta" />
<meta name="citation_author_institution" content="PhiXGen Private Limited, Gurugram" />
<meta name="citation_author" content="Mona Singh" />
<meta name="citation_author_institution" content="PhiXGen Private Limited, Gurugram" />
<meta name="citation_author" content="Rashmi Kumari" />
<meta name="citation_author_institution" content="Department of Zoology, College of Commerce, Arts &amp; Science, Patliputra University" />
<meta name="citation_author" content="Princy Hira" />
<meta name="citation_author_institution" content="Department of Zoology, Maitreyi College, University of Delhi" />
<meta name="citation_author" content="Shekhar Nagar" />
<meta name="citation_author_institution" content="Department of Zoology, University of Delhi" />
<meta name="citation_author" content="Chandni Talwar" />
<meta name="citation_author_institution" content="Department of Zoology, University of Delhi" />
<meta name="citation_author" content="Namita Nayyar" />
<meta name="citation_author_institution" content="Department of Zoology, Sri Venkateswara College, University of Delhi" />
<meta name="citation_author" content="Shailly Anand" />
<meta name="citation_author_institution" content="Department of Zoology, Deen Dayal Upadhyaya College, University of Delhi" />
<meta name="citation_author" content="Charu Dogra Rawat" />
<meta name="citation_author_institution" content="Department of Zoology, Ramjas College, University of Delhi" />
<meta name="citation_author" content="Mansi Verma" />
<meta name="citation_author_institution" content="Department of Zoology, Sri Venkateswara College, University of Delhi" />
<meta name="citation_author" content="Ram Krishan Negi" />
<meta name="citation_author_institution" content="Department of Zoology, University of Delhi" />
<meta name="citation_author" content="Yogendra Singh" />
<meta name="citation_author_institution" content="Department of Zoology, University of Delhi" />
<meta name="citation_author" content="Rup Lal" />
<meta name="citation_author_institution" content="The Energy and Resources Institute" />
<meta name="citation_author_email" content="ruplal@gmail.com" />
<meta name="citation_reference" content="Altschul, S.F., Gish, W., Miller, W., Myers, E.W., and Lipman, D.J. (1990). Basic Local Alignment Search Tool. J Mol Biol 215, 403–410." />
<meta name="citation_reference" content="Athmer, J., Fehr, A.R., Grunewald, M., Smith, E.C., Denison, M.R., and Perlman, S. (2017). In situ tagged nsp15 reveals interactions with coronavirus replication/transcription complex-associated proteins. mBio 8, e02320–16." />
<meta name="citation_reference" content="Barcena, M., Oostergetel, G.T., Bartelink, W., Faas, F.G., Verkleij, A., Rottier, P.J., Koster, A.J., and Bosch, B.J. (2009). Cryo-electron tomography of mouse hepatitis virus: Insights into the structure of the coronavirion. Proc Natl Acad Sci U S A 106, 582–587." />
<meta name="citation_reference" content="Becares, M., Pascual-Iglesias, A., Nogales, A., Sola, I., Enjuanes, L., and Zuñiga, S. (2016). Mutagenesis of coronavirus nsp14 reveals its potential role in modulation of the innate immune response. J Virol 90, 5399–5414." />
<meta name="citation_reference" content="Brian, D.A., and Baric, R.S. (2005). Coronavirus genome structure and replication. Curr Top Microbiol Immunol 287, 1–30." />
<meta name="citation_reference" content="Chen, Y., Su, C., Ke, M., Jin, X., Xu, L., Zhang, Z., Wu, A., Sun, Y., Yang, Z., Tien, P., Ahola, T., Liang, Y., Liu, X., and Guo, D. (2011). Biochemical and structural insights into the mechanisms of SARS coronavirus RNA ribose 2’-O-methylation by nsp16/nsp10 protein complex. PLoS Pathog 7, e1002294." />
<meta name="citation_reference" content="Collins, A.R., Knobler, R.L., Powell, H., and Buchmeier, M.J. (1982). Monoclonal antibodies to murine hepatitis virus-4 (strain JHM) define the viral glycoprotein responsible for attachment and cell--cell fusion. Virology 119, 358–371." />
<meta name="citation_reference" content="Cook, H.V., Doncheva, N.T., Szklarczyk, D., von Mering, C., and Jensen, L.J. (2018). Viruses.STRING: A Virus-Host Protein-Protein Interaction Database. Viruses 10, 519." />
<meta name="citation_reference" content="Cornillez-Ty, C.T., Liao, L., Yates, J.R. 3rd Kuhn, P., and Buchmeier, M.J. (2009). Severe acute respiratory syndrome coronavirus nonstructural protein 2 interacts with a host protein complex involved in mitochondrial biogenesis and intracellular signaling. J Virol 83, 10314–10318." />
<meta name="citation_reference" content="de Haan, C.A., Smeets, M., Vernooij, F., Vennema, H., and Rottier, P.J. (1999). Mapping of the coronavirus membrane protein domains involved in interaction with the spike protein. J Virol 73, 7441–7452." />
<meta name="citation_reference" content="Edgar, E. (2004). MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 32:1792–7." />
<meta name="citation_reference" content="Fehr, A.R., and Perlman, S. (2015). Coronaviruses: an overview of their replication and pathogenesis. Methods Mol Biol 1282, 1–23." />
<meta name="citation_reference" content="Fielding, B.C., Gunalan, V., Tan, T.H., Chou, C.F., Shen, S., Khan, S., Lim, S.G., Hong, W., and Tan, Y.J. (2006). Severe acute respiratory syndrome coronavirus protein 7a interacts with hSGT. Biochem Biophys Res Commun 343,:1201–8." />
<meta name="citation_reference" content="Fung, T.S., and Liu, D.X. (2019). Human Coronavirus: Host-Pathogen Interaction. Annu Rev Microbiol 73, 529–557." />
<meta name="citation_reference" content="Gorbalenya, A.E., Baker, S.C., Baric, R.S., de Groot, R.J., Drosten, C., Gulyaeva, A.A., Haagmans, B.L., Lauber, C., Leontovich, A.M., Neuman, B.W., Penzar, D., Perlman, S., et al. (2020). The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. Nat Microbiol 5, 536–544." />
<meta name="citation_reference" content="Gurevich, A., Saveliev, V., Vyahhi, N., and Tesler, G. (2013). QUAST: quality assessment tool for genome assemblies. Bioinformatics 29, 1072–1075." />
<meta name="citation_reference" content="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/" />
<meta name="citation_reference" content="Huang, L., Shi, Y., Gong, B., Jiang, L., Liu, X., Yang, J., Tang, J., You, C., Jiang, Q., Long, B., Zeng, T., Luo, M., Zeng, F., Zeng, F., Wang, S., Yang, X., and Yang, Z. (2020). Blood single cell immune profiling reveals the interferon-MAPK pathway mediated adaptive immune response for COVID-19. BMJ, doi: https://doi.org/10.1101/2020.03.15.20033472." />
<meta name="citation_reference" content="Huang, Y., Wang, X., Li, X., Ren, L., Zhao, J., Hu, Y., Zhang, L., Fan, G., Xu, J., Gu, X., Cheng, Z., Yu, T., Xia, J., Wei, Y., Wu, W., Xie, X., Yin, W., Li, H., Liu, M., Xiao, Y., Gao, H., Guo, L., Xie, J., Wang, G., Jiang, R., Gao, Z., Jin, Q., Wang, J., and Cao, B. (2020). Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395, 497–506." />
<meta name="citation_reference" content="Jang, K.J., Jeong, S., Kang, D.Y., Sp, N., Yang, Y.M., and Kim, D.E. (2020). A high ATP concentration enhances the cooperative translocation of the SARS coronavirus helicase nsP13 in the unwinding of duplex RNA. Sci Rep 10, 1–13." />
<meta name="citation_reference" content="Josset, L., Menachery, V.D., Gralinski, L.E., Agnihothram, S., Sova, P., Carter, V.S., Yount, B.L., Graham, R.L., Baric, R.S., and Katze, M.G. (2013). Cell host response to infection with novel human coronavirus EMC predicts potential antivirals and important differences with SARS coronavirus. mBio 4, e00165–13." />
<meta name="citation_reference" content="Kanehisa, M., and Goto, S. (2000). KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28, 27–30." />
<meta name="citation_reference" content="Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., and Tanabe, M. (2016). KEGG as a reference resource for gene and protein annotation. Nucleic Acids Res. 44, D457–D462." />
<meta name="citation_reference" content="Kerrien, S., Aranda, B., Breuza, L., Bridge, A., Broackes-Carter, F., Chen, C., Duesbury, M., Dumousseau, M., Feuermann, M., Hinz, U., et al. (2013). The IntAct molecular interaction database in 2012. Nucleic Acids Res 41, D43–D47." />
<meta name="citation_reference" content="Klumperman, J., Locker, J.K., Meijer, A., Horzinek, M.C., Geuze, H.J., and Rottier, P.J. (1994). Coronavirus M proteins accumulate in the Golgi complex beyond the site of virion budding. J Virol 68, 6523–6534." />
<meta name="citation_reference" content="Kosakovsky Pond, S.L. and Frost, S.D. (2005). Not So Different After All: A Comparison of methods for detecting amino acid sites under selection. Mol Biol Evol 22, 1208–1222." />
<meta name="citation_reference" content="Kryazhimskiy, S., and Plotkin, J.B. (2008). The Population Genetics of dN/dS. PLoS Genet 4, e1000304." />
<meta name="citation_reference" content="Kumar, S., Stecher, G., and Tamura, K. (2016). MEGA7: Molecular Evolutionary Genetics Analysis version 7.0 for bigger datasets. Mol Biol Evol 33, 1870–1874." />
<meta name="citation_reference" content="Lau, S.K., Woo, P.C., Li, K.S., Huang, Y., Tsoi, H.W., Wong, B.H., Wong, S.S., Leung, S.Y., Chan, K.H., and Yuen, K.Y. (2005). Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats. Proc Natl Acad Sci U S A 102, 14040–14045." />
<meta name="citation_reference" content="Li, F. (2016). Structure, function, and evolution of coronavirus spike proteins. Annu Rev Virol 3, 237–261." />
<meta name="citation_reference" content="Li, S.W., Wang, C.Y., Jou, Y.J., Jou, Y.J., Tang, T.C., Huang, S.H., Wan, L., Lin, Y.J., and Lin, C.W. (2016). SARS coronavirus papain-like protease induces Egr-1-dependent up-regulation of TGF-β1 via ROS/p38 MAPK/STAT3 pathway. Sci Rep 6, 25754." />
<meta name="citation_reference" content="Lu, R., Zhao, X., Li, J., Niu, P., Yang, B., Wu, H., Wang, W., Ma, X., Zhan, F., Wang, L., Hu, T., Zhou, H., Hu, Z. et al. (2020). Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancelet 395, 565–574." />
<meta name="citation_reference" content="McBride, R., van Zyl, M., and Fielding, B.C. (2014). The coronavirus nucleocapsid is a multifunctional protein. Viruses 6, 2991–3018." />
<meta name="citation_reference" content="Letunic, I., and Bork, P. (2016). Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees. Nucleic Acids Res 44, W242–245." />
<meta name="citation_reference" content="Maglott, D., Ostell, J., Pruitt, K.D., and Tatusova, T. (2005). Entrez Gene: gene-centered information at NCBI. Nucleic Acids Res. 33, D54–D58." />
<meta name="citation_reference" content="Mergia, A. (2017). The Role of Caveolin 1 in HIV Infection and Pathogenesis. Viruses 9, 129." />
<meta name="citation_reference" content="Meyer, B., Muller, M.A., Corman, V.M., Reusken, C.B., Ritz, D., Godeke, G.J., Lattwein, E., Kallies, S., Siemens, A., van Beek, J., Drexler, J.F., Muth, D., Bosch, B.J., Wernery, U., Koopmans, M.P., Wernery, R., and Drosten, C. (2014). Antibodies against MERS coronavirus in dromedary camels, United Arab Emirates, 2003 and 2013. Emerg Infect Dis 20, 552–559." />
<meta name="citation_reference" content="Nakamura, Y., and Tomii, K. (2018). Parallelization of MAFFT for large-scale multiple sequence alignments. Bioinformatics 34, 2490–2492." />
<meta name="citation_reference" content="Neuman, B.W., Adair, B.D., Yoshioka, C., Quispe, J.D., Orca, G., Kuhn, P., Milligan, R.A., Yeager, M., and Buchmeier, M.J. (2006). Supramolecular architecture of severe acute respiratory syndrome coronavirus revealed by electron cryomicroscopy. J Virol 80,7918–7928." />
<meta name="citation_reference" content="Neuman, B.W., Kiss, G., Kunding, A.H., Bhella, D., Baksh, M.F., Connelly, S., Droese, B., Klaus, J.P., Makino, S., Sawicki, S.G., Siddell, S.G., Stamou, D.G., Wilson, I.A., Kuhn, P., and Buchmeier, M.J. (2011). A structural analysis of M protein in coronavirus assembly and morphology. J Struct Biol 174, 11–22." />
<meta name="citation_reference" content="Page, A.J., Cummins, C.A., Hunt, M., Wong, V.K., Reuter, S., Holden, M.T., Fookes, M., Falush, D., Keane, J.A., and Parkhill, J. (2015). Roary: rapid large-scale prokaryote pan genome analysis. Bioinformatics 31, 3691–3693." />
<meta name="citation_reference" content="Pond, SL., Frost, S.D., and Muse, S.V. (2005). HyPhy: hypothesis testing using phylogenies. Bioinformatics. 21: 676–9." />
<meta name="citation_reference" content="Prompetchara, E., Ketloy, C., and Palaga, T. (2020). Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic. Asian Pac J Allergy Immunol 38, 1–9." />
<meta name="citation_reference" content="Qin, H., Wang, L., Feng, T., Elson, C.O., Niyongere, S.A., Lee, A.J., Reynolds, S.L., Weaver, C.T., Roarty, K., Serra, R., Benveniste, E.N., and Cong, Y. (2009). TGF-beta promotes Th17 cell development through inhibition of SOCS3. J Immunol 183, 97–105." />
<meta name="citation_reference" content="Ren. L.L., Wang, Y.M., Wu, Z.Q., Xiang, Z.C., Guo, L., Xu, T., Jiang, Y.Z., Xiong, Y., Li, Y.J., Li, X.W., Li, H., Fan, G.H., Gu, X.Y., Xiao, Y., Gao, H., Xu, J.Y., Yang, F., Wang, X.M., Wu, C., Chen, L., Liu, Y.W., Liu, B., Yang, J., Wang, X.R., Dong, J., Li, L., Huang, C.L., Zhao, J.P., Hu, Y., Cheng, Z.S., Liu, L.L., Qian, Z.H., Qin, C., Jin, Q., Cao, B., and Wang, J.W. (2020). Identification of a novel coronavirus causing severe pneumonia in human: a descriptive study. Chin Med J (Engl). doi: 10.1097/CM9.0000000000000722." />
<meta name="citation_reference" content="Ruch, T.R., and Machamer, C.E. (2012). The coronavirus E protein: assembly and beyond. Viruses 4, 363–382." />
<meta name="citation_reference" content="Sah, R., Alfonso, J., Rodriguez-Morales Jha, R., Daniel, K.W., Chu, H.G., Peiris, M., Bastola, A., Lal, B.K., Ojha, H.C., Rabaan, A.A., Zambrano, L.I., Costello, A., Morita, K., Pandey, B.D., and Poon, L.L. M. (2020). Complete genome sequence of a 2019 novel coronavirus (SARS-CoV-2) strain isolated in Nepal. Microbiol Res Announce 9, e00169–20." />
<meta name="citation_reference" content="Seemann, T. (2014). Prokka: rapid prokaryotic genome annotation. Bioinformatics 30, 2068–2069." />
<meta name="citation_reference" content="Shannon, P., Markiel, A., Ozier, O., Baliga, N.S., Wang, J.T., Ramage, D., Amin, N., et al. (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res 13, 2498–2504." />
<meta name="citation_reference" content="Snijder, E.J., Decroly, E., and Ziebuhr, J. (2016). The non-structural proteins directing coronavirus RNA synthesis and processing. Adv Virus Res 96, 59–126." />
<meta name="citation_reference" content="Stano, M., Beke, G., and Klucar, L. (2016). viruSITE—integrated database for viral genomics. Database 2, article ID baw162; doi:10.1093/database/baw162." />
<meta name="citation_reference" content="Szklarczyk, D., Franceschini, A., Wyder, S., Forslund, K., Heller, D., Huerta-Cepas, J., Simonovic, M., et al. (2015). STRING v10: protein-protein interaction networks, integrated over the tree of life. Nucleic Acids Res 43, D447–452." />
<meta name="citation_reference" content="Tamura, K., and Nei, M. (1993). Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees. Mol Biol Evol 10, 512–526." />
<meta name="citation_reference" content="Tang, Q., Song, Y., Shi, M., Cheng, Y., Zhang, W., Xia, X.Q. (2015). Inferring the hosts of coronavirus using dual statistical models based on nucleotide composition. Sci Rep 5, 17155." />
<meta name="citation_reference" content="Tan, Y.X., Timothy, H.P. Tan Marvin, J.-R., Lee Tham, P.Y., Gunalan, V., Druce, J., Birch, C., Catton, M., Fu, N.Y., Yu, V.C., and Tan, Y.J. (2007). Induction of apoptosis by the severe acute respiratory syndrome coronavirus 7a protein is dependent on its interaction with the Bcl-XL protein. J Virol 81, 6346–6355." />
<meta name="citation_reference" content="Tatsuta, T., Model, K., and Langer, T. (2005). Formation of membrane-bound ring complexes by prohibitins in mitochondria. Mol Biol Cell 16, 248–259." />
<meta name="citation_reference" content="Treangen, T.J., Ondov, B.D., Koren, S. and Phillippy, A.M. (2014). The Harvest suite for rapid core-genome alignment and visualization of thousands of intraspecific microbial genomes. Genome Biol 15, 524." />
<meta name="citation_reference" content="Tyrrell, D.A., and Bynoe, M.L. (1966). Cultivation of viruses from a high proportion of patients with colds. Lancet 1, 76–77." />
<meta name="citation_reference" content="UniProt Consortium. (2007). The universal protein resource (UniProt). Nucleic Acids Res 36, D190–195." />
<meta name="citation_reference" content="von Grotthuss, M., Wyrwicz, L.S., and Rychlewski, L. (2003). mRNA cap-1 methyltransferase in the SARS genome. Cell 113, 701–702." />
<meta name="citation_reference" content="Wall, A.C., Park, Y.J., Tortorici, M.A., Wall, A., McGuire, A.T., and Vessler, D. (2020). Structure, function, and antigenicity of the SARS-CoV-2 spike glycoprotein. Cell 180, 1–12." />
<meta name="citation_reference" content="Wang, Q., Zhang, Y., Wu, L., Niu, S., Song, C., Zhang, Z., Lu, G.,Qiao, C., Hu, Y., Yuen, K.Y., Wang, Q., Zhou, H., Yan, J., and Qi, J. (2020). Structural and functional basis of SARS-CoV-2 entry by using human ACE2. Cell doi: 10.1016/j.cell.2020.03.045." />
<meta name="citation_reference" content="Weaver, S., Shank, S.D., Spielman, S.J., Li, M., Muse, S.V., Kosakovsky Pond S.L. (2018). Datamonkey 2.0: A Modern Web Application for Characterizing Selective and Other Evolutionary Processes. Mol Biol Evol. 35: 773–777." />
<meta name="citation_reference" content="Woo, P.C., Lau, S.K., Lam, C.S., Lau, C.C., Tsang, A.K., Lau, J.H., Bai, R., Teng, J.L., Tsang, C.C., Wang, M., Zheng, B.J., Chan, K.H., and Yuen, K.Y. (2012). Discovery of seven novel Mammalian and avian coronaviruses in the genus deltacoronavirus supports bat coronaviruses as the gene source of alphacoronavirus and betacoronavirus and avian coronaviruses as the gene source of gammacoronavirus and deltacoronavirus. J Virol 86, 3995–4008." />
<meta name="citation_reference" content="Wu, A., Peng, Y., Huang, B., Huang, B., Ding, X., Wang, X., Niu, P., Meng, J., Zhu, Z.Z., Wang, J., Sheng, J., Quan, L., Xia, Z., Tan, W., Cheng, G., and Jiang, T. (2020). Genome composition and divergence of the novel coronavirus (2019-nCoV) originating in China. Cell Host Microbe 27, 325–328." />
<meta name="citation_reference" content="Zhao, X., Nicholls, J.M., and Chen, Y. (2008). Sars-cov nucleocapsid protein interacts with smad3 and modulates TGF-β signaling. J Biol Chem. 283: 3272–80." />
<meta name="citation_reference" content="Xu, Z., Shi, L., Wang, Y., Zhang, J., Huang, L., Zhang, C., Liu, S., Zhao, P., Liu, H., Zhu, L., Tai, Y., Bai, C., Gao, T., Song, J., Xia, P., Dong, J., Zhao, J., and Wang, F.S. (2020). Pathological findings of COVID-19 associated with acute respiratory distress syndrome. Lancet Respir Med doi: 10.1016/S2213-2600(20)30076-X." />
<meta name="citation_reference" content="Yuan, X., Li, J., Shan, Y., Yang, Z., Zhao, Z., Chen, B., Yao, Z., Dong, B., Wang, S., Chen, J., and Chong, Y. (2005). subcellular localization and membrane association of SARS-CoV 3a protein. Virus Res 109, 191–202." />
<meta name="citation_reference" content="Zhang, S. (2018). The role of transforming growth factor β in T helper 17 differentiation. Immunology. 155: 24–35." />
<meta name="citation_reference" content="Zhao, X., Nicholls, J.M., and Chen, Y.G. (2008). Sars-cov nucleocapsid protein interacts with smad3 and modulates TGF-β signaling. J Biol Chem 8, 3272–3280." />
<meta name="citation_reference" content="Zhou, Z., Alikhan, N.F., Sergeant, M.J., Luhmann, N., Vaz, C., Francisco, A.P., Carrico, J.A., and Achtman, M. (2018). GrapeTree: visualization of core genomic relationships among 100,000 bacterial pathogens. Genome Res 28, 1395–1404." />
<meta name="twitter:title" content="Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The Coronavirus disease-19 (COVID19) that started in Wuhan, China in December 2019 has spread worldwide emerging as a global pandemic. The severe respiratory pneumonia caused by the novel SARS-CoV-2 has so far claimed more than 14,500 lives and has impacted human lives worldwide. Development of universal vaccines against the novel SARS-CoV-2 holds utmost urgency to control COVID19 pandemic that appears to be more severe than any of the previous outbreaks of severe acute respiratory syndrome (SARS) and Middle-East respiratory syndrome (MERS). However, as the novel SARS-CoV-2 displays high transmission rates, the underlying severity hidden in the SARS-CoV2 genomes is required to be fully understood. We studied the complete genomes of 95 strains of SARS-CoV-2 reported from different geographical regions worldwide to uncover the pattern of spread of the novel SARS-CoV-2 across the globe. We show that there is no direct transmission pattern of the virus among neighbouring countries suggesting that the outbreak is a result of travel of infected humans to different countries. We revealed unique single nucleotide polymorphisms (SNPs) in nsp13, nsp14, nsp15, nsp16 (present in ORF1b polyprotein region) and S-Protein within 10 viral isolates from USA. These viral proteins are involved in RNA replication and processing, indicating highly evolved strains of the novel SARS-CoV-2 circulating in the population of USA than in other countries. Furthermore, we found an isolate from USA (MT188341) to carry frameshift mutation between positions 2540 and 2570 of nsp16 which functions as mRNA cap-1 methyltransferase (2′-O-MTase). Thus, we reason that the replicative machinery of the novel SARS-CoV-2 is fast evolving to evade host challenges and survival. These mutations are needed to be considered, otherwise it will be difficult to develop effective treatment strategies. The two proteins also had dN/dS values approaching 1-ORF1ab polyprotein (dN/dS= 0.996, 0.575) and S protein (dN/dS= 0.88) and might confer selective advantage to the virus. Through the construction of SARS-CoV-2-human interactome, we further reveal multiple host proteins (PHB, PPP1CA, TGF-beta, JACK1, JACK2, SOCS3,STAT3, JAK1-2, SMAD3, BCL2, CAV1 &amp; SPECC1) which are manipulated by the virus proteins (nsp2, PL-PRO, N-protein, ORF7a, M-S-ORF3a complex, nsp7-nsp8-nsp9-RdRp complex) for mediating host immune mechanism for its survival." />
<meta name="og-title" property="og:title" content="Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.25.006213v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The Coronavirus disease-19 (COVID19) that started in Wuhan, China in December 2019 has spread worldwide emerging as a global pandemic. The severe respiratory pneumonia caused by the novel SARS-CoV-2 has so far claimed more than 14,500 lives and has impacted human lives worldwide. Development of universal vaccines against the novel SARS-CoV-2 holds utmost urgency to control COVID19 pandemic that appears to be more severe than any of the previous outbreaks of severe acute respiratory syndrome (SARS) and Middle-East respiratory syndrome (MERS). However, as the novel SARS-CoV-2 displays high transmission rates, the underlying severity hidden in the SARS-CoV2 genomes is required to be fully understood. We studied the complete genomes of 95 strains of SARS-CoV-2 reported from different geographical regions worldwide to uncover the pattern of spread of the novel SARS-CoV-2 across the globe. We show that there is no direct transmission pattern of the virus among neighbouring countries suggesting that the outbreak is a result of travel of infected humans to different countries. We revealed unique single nucleotide polymorphisms (SNPs) in nsp13, nsp14, nsp15, nsp16 (present in ORF1b polyprotein region) and S-Protein within 10 viral isolates from USA. These viral proteins are involved in RNA replication and processing, indicating highly evolved strains of the novel SARS-CoV-2 circulating in the population of USA than in other countries. Furthermore, we found an isolate from USA (MT188341) to carry frameshift mutation between positions 2540 and 2570 of nsp16 which functions as mRNA cap-1 methyltransferase (2′-O-MTase). Thus, we reason that the replicative machinery of the novel SARS-CoV-2 is fast evolving to evade host challenges and survival. These mutations are needed to be considered, otherwise it will be difficult to develop effective treatment strategies. The two proteins also had dN/dS values approaching 1-ORF1ab polyprotein (dN/dS= 0.996, 0.575) and S protein (dN/dS= 0.88) and might confer selective advantage to the virus. Through the construction of SARS-CoV-2-human interactome, we further reveal multiple host proteins (PHB, PPP1CA, TGF-beta, JACK1, JACK2, SOCS3,STAT3, JAK1-2, SMAD3, BCL2, CAV1 &amp; SPECC1) which are manipulated by the virus proteins (nsp2, PL-PRO, N-protein, ORF7a, M-S-ORF3a complex, nsp7-nsp8-nsp9-RdRp complex) for mediating host immune mechanism for its survival." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-30" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.25.006213v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.25.006213v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.25.006213v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.25.006213v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1217854" />
    <title>Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"PXl7sK1PKY1bPsMvnK3KjaU5x7w0b43D0m5A10I6EAU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1217854","apath":"\/biorxiv\/early\/2020\/03\/30\/2020.03.25.006213.atom","pisa":"biorxiv;2020.03.25.006213v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.25.006213v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.25.006213v1","title":"Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution","identifier":"node\/1217854"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225u7XKE3K4lOpzzLo8fXj0CQM2iGbPSbtHnNtMUr\\\\\\\/thnml+ZgxzuaYmOjK60gEnM47U2UVI+XSLTB6kXyW+BkBvMgFbc8iYSi\\\\\\\/UyIXH60Xf3S8XGDMrVX39o2HX2dHXVBIvzYMJkx6c82lfDiqhfnO2VPtYClW4b\\\\\\\/4UJX0JfN0lraY+ayV7FR\\\\\\\/HLW0QY+hW10C1Jxxe6m8SXye\\\\\\\/NQC5PpFe9fqClWiniGS80ZME3dLuMIPfdqHSGaiQIghWFK7n6FFsJ6GJiHlxeXgjqKZEJQwmiEn+kz9sCRYRbT9RB2ZomglU3XeAM1WCf6qGkrF0RimlOnsX5+y1oOpDrFXhKFeOZ0RTMxsHzvUkH6Tp7IHKbjp0vx61IYZ09CIiP5QeGJEfZvH+70K+68tWuZAlfRjv5YWqbiuUBbHfUpiiXwhupJICDcQlPF2Vmyoolz2H01zv6jTjTXGpMU9oZbMNc22UBQEIY\\\\\\\/\\\\\\\/t9asQETvt2byBeaG2cm\\\\\\\/TkNFqUYYkaNeGCbFP9DfDpdywVZgdBR6FFAVkrGJYrJxLtbriuurx24chVctvMc+urp9tPlYHcgooHdrmmFeq+54fBnIfx1sWknP\\\\\\\/pkX7qC4vkyfSAuSJOP5jP2uRBVETLlfn7RoybjV3WVSSlt7+5x+6SoLyyTGzZ2\\\\\\\/KVpOEsDTAIhukgSJyWp4snyo31va+R7gNfOidjSY24znTdwPMEVx65Lj66jNbLfC+4+aklioIPvtspY9g\\\\\\\/D7DJONBbKxK8XAr8RJ3eubJKzy9EbnYwOSWUqzuGcxMP4wC7BTpIxAImDPThLtbZCqtWUm42MhNXXAdGpjf+btiRiLpAD8Cba5Mst4TojXb9dcDdeaqgpqh+S1WH\\\\\\\/bIlfY7g\\\\\\\/OfziPaBcNTeSfvz4tZGGlOTUhSKqQRRaYAXTWF2xXAR33IHGat0NxTRWk9cn+Z3pdBXWD7MhyxrHE8xo8q+BUleG5WnLqzB3D7mfNpwVwClf1r2qHS8rF5ieNMtMbPBqxem5NiWcGcCAMRxamNceBGfeQXsQkC4Q1xpqBg0CN4TtyYGc63\\\\\\\/AJUjRk9H9+8M4oF1B3opDuc+fc14mVePnrPyBr5kJQ3ClpEl5w6P\\\\\\\/YN6n+qfDZC7Ra5bw7D3WLv5OZMQduPf2Uw2Kw8av9w8O+WEzYmzdwEHtmSHIpooaVu85NgzXko1DbwcnrgrRZyJxQKNzsh63udvZpEhdp4oXz4KeJAiF3PSsqs0VNXTTzGT8HYjYf7UvbzUOozGJr\\\\\\\/eHwVBmsAMCaVTu58A0AHjshwtNpka5tU9wGV6NcrPxFBnw1VcvIAhlDS1U8iwfQ6ibWOhasiQjihFgL8Wc2kf6\\\u0022,\\\u0022iv\\\u0022:\\\u0022fJccd9wLIhLBY7j9\\\\\\\/ckqWw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f6c30aa5571a451e10959d84703a8f1a325b1e4ae78b476d7c42a8a4ea734042\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.25.006213v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1217854 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.25.006213v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1a_mGZvTxuqmj7idqoPC8QI9iinBTZ63yT6VoGNUC5k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1217854" id="node1217854" data-pisa="biorxiv;2020.03.25.006213v1" data-pisa-master="biorxiv;2020.03.25.006213" data-apath="/biorxiv/early/2020/03/30/2020.03.25.006213.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Roshan</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Helianthous</span> <span class="nlm-surname">Verma</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Nirjara</span> <span class="nlm-surname">Singhvi</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Utkarsh</span> <span class="nlm-surname">Sood</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Vipin</span> <span class="nlm-surname">Gupta</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mona</span> <span class="nlm-surname">Singh</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rashmi</span> <span class="nlm-surname">Kumari</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Princy</span> <span class="nlm-surname">Hira</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Shekhar</span> <span class="nlm-surname">Nagar</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Chandni</span> <span class="nlm-surname">Talwar</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Namita</span> <span class="nlm-surname">Nayyar</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Shailly</span> <span class="nlm-surname">Anand</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Charu Dogra</span> <span class="nlm-surname">Rawat</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Mansi</span> <span class="nlm-surname">Verma</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Ram Krishan</span> <span class="nlm-surname">Negi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Yogendra</span> <span class="nlm-surname">Singh</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Rup</span> <span class="nlm-surname">Lal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.25.006213 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1217854" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Roshan Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>P.G. Department of Zoology, Magadh University</span>, Bodh Gaya, Bihar-824234, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roshan%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoshan%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Helianthous Verma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Zoology, Ramjas College, University of Delhi</span>, New Delhi-110007, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helianthous%2BVerma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verma%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelianthous%2BVerma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nirjara Singhvi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Zoology, University of Delhi</span>, New Delhi-110007, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nirjara%2BSinghvi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singhvi%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANirjara%2BSinghvi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Utkarsh Sood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>The Energy and Resources Institute</span>, Darbari Seth Block, IHC Complex, Lodhi Road, New Delhi-110003, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Utkarsh%2BSood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sood%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUtkarsh%2BSood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vipin Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>PhiXGen Private Limited, Gurugram</span>, Haryana 122001, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vipin%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVipin%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mona Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>PhiXGen Private Limited, Gurugram</span>, Haryana 122001, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mona%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMona%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rashmi Kumari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Zoology, College of Commerce, Arts &#38; Science, Patliputra University</span>, Patna, Bihar-800020, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rashmi%2BKumari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumari%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARashmi%2BKumari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Princy Hira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Department of Zoology, Maitreyi College, University of Delhi</span>, New Delhi-110007, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Princy%2BHira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hira%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrincy%2BHira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Shekhar Nagar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Zoology, University of Delhi</span>, New Delhi-110007, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shekhar%2BNagar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nagar%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShekhar%2BNagar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Chandni Talwar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Zoology, University of Delhi</span>, New Delhi-110007, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chandni%2BTalwar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Talwar%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChandni%2BTalwar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Namita Nayyar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Zoology, Sri Venkateswara College, University of Delhi</span>, New Delhi-110021, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Namita%2BNayyar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nayyar%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANamita%2BNayyar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Shailly Anand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Department of Zoology, Deen Dayal Upadhyaya College, University of Delhi</span>, New Delhi-110078, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shailly%2BAnand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anand%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShailly%2BAnand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Charu Dogra Rawat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Zoology, Ramjas College, University of Delhi</span>, New Delhi-110007, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charu%2BDogra%2BRawat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rawat%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharu%2BDogra%2BRawat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Mansi Verma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Zoology, Sri Venkateswara College, University of Delhi</span>, New Delhi-110021, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mansi%2BVerma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Verma%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMansi%2BVerma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Ram Krishan Negi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Zoology, University of Delhi</span>, New Delhi-110007, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ram%2BKrishan%2BNegi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Negi%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARam%2BKrishan%2BNegi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Yogendra Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Zoology, University of Delhi</span>, New Delhi-110007, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yogendra%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYogendra%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Rup Lal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>The Energy and Resources Institute</span>, Darbari Seth Block, IHC Complex, Lodhi Road, New Delhi-110003, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rup%2BLal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lal%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARup%2BLal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ruplal@gmail.com" class="" data-icon-position="" data-hide-link-title="0">ruplal@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.25.006213v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1217854" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1217854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.006213v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1217854" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1217854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.006213v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1217854" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1217854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.25.006213v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1217854" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1217854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.25.006213v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1217854" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1217854/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The Coronavirus disease-19 (COVID19) that started in Wuhan, China in December 2019 has spread worldwide emerging as a global pandemic. The severe respiratory pneumonia caused by the novel SARS-CoV-2 has so far claimed more than 14,500 lives and has impacted human lives worldwide. Development of universal vaccines against the novel SARS-CoV-2 holds utmost urgency to control COVID19 pandemic that appears to be more severe than any of the previous outbreaks of severe acute respiratory syndrome (SARS) and Middle-East respiratory syndrome (MERS). However, as the novel SARS-CoV-2 displays high transmission rates, the underlying severity hidden in the SARS-CoV2 genomes is required to be fully understood. We studied the complete genomes of 95 strains of SARS-CoV-2 reported from different geographical regions worldwide to uncover the pattern of spread of the novel SARS-CoV-2 across the globe. We show that there is no direct transmission pattern of the virus among neighbouring countries suggesting that the outbreak is a result of travel of infected humans to different countries. We revealed unique single nucleotide polymorphisms (SNPs) in nsp13, nsp14, nsp15, nsp16 (present in ORF1b polyprotein region) and S-Protein within 10 viral isolates from USA. These viral proteins are involved in RNA replication and processing, indicating highly evolved strains of the novel SARS-CoV-2 circulating in the population of USA than in other countries. Furthermore, we found an isolate from USA (MT188341) to carry frameshift mutation between positions 2540 and 2570 of nsp16 which functions as mRNA cap-1 methyltransferase (2′-O-MTase). Thus, we reason that the replicative machinery of the novel SARS-CoV-2 is fast evolving to evade host challenges and survival. These mutations are needed to be considered, otherwise it will be difficult to develop effective treatment strategies. The two proteins also had dN/dS values approaching 1-ORF1ab polyprotein (dN/dS= 0.996, 0.575) and S protein (dN/dS= 0.88) and might confer selective advantage to the virus. Through the construction of SARS-CoV-2-human interactome, we further reveal multiple host proteins (PHB, PPP1CA, TGF-beta, JACK1, JACK2, SOCS3,STAT3, JAK1-2, SMAD3, BCL2, CAV1 &amp; SPECC1) which are manipulated by the virus proteins (nsp2, PL-PRO, N-protein, ORF7a, M-S-ORF3a complex, nsp7-nsp8-nsp9-RdRp complex) for mediating host immune mechanism for its survival.</p></div><div class="section" id="sec-1"><h2 class="">Background</h2><p id="p-3">Since the current outbreak of pandemic coronavirus disease 19 (COVID-19) caused by Severe Acute Respiratory Syndrome-related Coronavirus (SARS-CoV-2), the assessment of biogeographical pattern of SARS-CoV-2 isolates and the mutations at nucleotide and protein level is of high interest to many research groups (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wang <em>et al.</em>, 2020</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wall <em>et al.</em>, 2020</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wu <em>et al.</em>, 2020</a>). Coronaviruses (CoVs), members of <em>Coronaviridae</em> family, order <em>Nidovirales</em>, have been known as human pathogens from last six decades (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tyrrell and Bynoe,1966</a>). Their target is not just limited to humans, but also to other mammals and birds (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Woo <em>et al.</em>, 2012</a>). Coronaviruses have been classified under alpha, beta, gamma and delta-coronavirus groups (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li, 2016</a>) in which former two are known to infect mammals while latter two primarily infect bird species (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Tang <em>et al.</em>, 2015</a>). Illness in humans varies from common cold to respiratory and gastrointestinal distress of varying intensities. In the past, more severe forms caused major outbreaks that include Severe Acute Respiratory Syndrome (SARS-CoV) (outbreak in 2003, China) and Middle East Respiratory Syndrome (MERS-CoV) (outbreak in 2012, Middle East) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fehr and Perlman, 2015</a>). Bats are known to host coronaviruses acting as their natural reservoirs which may be transmitted to humans through an intermediate host. SARS-CoV and MERS-CoV were transmitted from intermediate hosts, palm civets and camel, respectively (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lau <em>et al.</em>, 2005</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mayer <em>et al.</em>, 2014</a>). It is not, however, yet clear which animal served as the intermediate host for transmission of SARS-CoV-2 transmission from bats to humans which is most likely suggested to be a warm-blooded vertebrate (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em>, 2020</a>).</p><p id="p-4">The inherently high recombination frequency and mutation rates of coronavirus genomes allows for their easy transmission from different intermediate hosts. Structurally, they are positive-sense single stranded RNA (ssRNA) virions with characteristic spikes projecting from the surface of capsid coating (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Newman <em>et al.</em>, 2006</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barcena <em>et al.</em>, 2009</a>). Their genome is nearly 27 to 31 Kb long, largest among the RNA viruses, with 5’cap and 3’ polyA tail, for translation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brian and Baric, 2005</a>). Their spherical capsid and spikes give them crown-like appearance due to which they were named as ‘corona’, meaning ‘crown’ or ‘halo’ in <em>Latin</em>. Coronavirus consists four main proteins, spike (S), membrane (M), envelope (E) and nucleocapsid (N). The spike (~150 kDa) mediates its attachment to host receptor proteins (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Colins <em>et al.</em>, 1982</a>). Membrane protein (~25-30 kDa) attaches with nucleocapsid and maintains curvature of virus membrane (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Neumann <em>et al.</em>, 2011</a>). E protein (8-12 kDa) is responsible for the pathogenesis of the virus as it eases assembly and release of virion particles and also has ion channel activity as integral membrane protein (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ruch and Machamer, 2012</a>). N-protein, the fourth protein, helps in packaging of virus particles into capsids and promotes replicase-transcriptase complex (RTC) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">McBride <em>et al.</em>, 2014</a>).</p><p id="p-5">Recently, in December 2019, the outbreak of novel beta-coronavirus (2019-nCoV) or SARS-CoV-2 in Wuhan, China has shown devastating effects worldwide (<a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/</a>). World Health Organisation (WHO) has declared COVID19, the disease caused by the novel SARS-CoV-2 a pandemic, affecting more than 186 countries and territories where China has most reported cases 81,601 and Italy has highest mortality rate 9.26% (59,138 infected individuals, 5476 deaths) (WHO situation report-63). As on date (March 24, 2020), more than 0.3 million individuals have been infected by SARS-CoV-2 and nearly 14,510 have died worldwide. Virtually all human lives have been impacted with no foreseeable end of the pandemic. A recent study on ten novel coronavirus strains by Lu <em>et al.</em>, suggested that SARS-CoV-2 is sufficiently diverged from SARS-CoV (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lu <em>et al.</em>, 2020</a>). SARS-CoV-2 is assumed to have originated from bats, which serve as reservoir host of the virus (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lu <em>et al.</em>, 2020</a>). Other studies have also reported the genome composition and divergence patterns of SARS-CoV-2 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sah <em>et al.</em>, 2020</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Wu <em>et al.</em>, 2020</a>). However, no study has yet explained the biogeographical pattern of this emerging pathogen. In this study, we selected 95 strains of SARS-CoV-2, isolated from 11 different countries to understand the transmission patterns, evolution and pathogenesis of the virus. Using core genome and Single Nucleotide Polymorphism (SNP) based phylogeny, we attempted to uncover any existence of a transmission pattern of the virus across the affected countries, which was not known earlier. We analysed the ORFs of the isolates to reveal unique point and frameshift mutations in the isolates from the USA. In addition, we analysed the gene/protein mutations in these novel strains and estimated the direction of selection to decipher their evolutionary divergence rate. Further, we also established the interactome of SARS-CoV-2 with the human host proteins to predict functional implications of the viral infection host cells. The results obtained from the analyses indicate high severity of SARS-CoV-2 isolates with inherent capability of unique mutations and the evolving viral replication strategies to adapt to human hosts.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Selection of genomes and annotation</h3><p id="p-6">Sequences of different strains were downloaded from NCBI database <a href="https://www.ncbi.nlm.nih.gov/genbank/sars-cov-2-seqs/">https://www.ncbi.nlm.nih.gov/genbank/sars-cov-2-seqs/</a> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). A total of 97 genomes were downloaded on March 19, 2020 and based on quality assessment two genomes with multiple Ns were removed from the study. Further the genomes were annotated using Prokka (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Seemann, 2014</a>). A manually annotated reference database was generated using the Genbank file of Severe acute respiratory syndrome coronavirus 2 isolate-SARS-CoV-2/SH01/human/2020/CHN (Accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MT121215&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom">MT121215</a>) and open reading frames (ORFs) were predicted against the formatted database using prokka (-gcode 1) (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Seemann, 2014</a>). Further the GC content information were generated using QUAST standalone tool (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gurevich <em>et al.</em>, 2013</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1225317/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1225317/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-7" class="first-child">General genomic attributes of SARS-CoV-2 strains.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Analysis of natural selection</h3><p id="p-8">To determine the evolutionary pressure on viral proteins, dN/dS values were calculated for 9 ORFs of all strains. The orthologous gene clusters were aligned using MUSCLE v3.8 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Edgar <em>et al.</em>, 2004</a>) and further processed for removing stop codons using HyPhy v2.2.4 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pond <em>et al.</em>, 2005</a>). Single-Likelihood Ancestor Counting (SLAC) method in Datamonkey v2.0 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Weaver <em>et al.</em>, 2018</a>) (<a href="http://www.datamonkey.org/slac">http://www.datamonkey.org/slac</a>) was used to calculate dN/dS value for each orthologous gene cluster. The dN/dS values were plotted in R (R Development Core Team, 2015).</p></div><div id="sec-5" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-9">In order to infer the phylogeny, the core gene alignment was generated using MAFFT (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nakamura <em>et al</em>., 2018</a>) present within the Roary Package (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Page <em>et al</em>., 2015</a>). Further, the phylogeny was inferred using the Maximum Likelihood method based and Tamura-Nei model (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tamura and Nei, 1993</a>) in MEGAX (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kumar <em>et al</em>., 2016</a>) and visualized in interactive Tree of Life (iTOL) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Letunic and Bork, 2016</a>) and GrapeTree (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zhou <em>et al</em>., 2018</a>).</p><p id="p-10">In order to determine the single nucleotide polymorphism (SNP), whole-genome alignments were made using libMUSCLE aligner. For this, we used SARS-CoV-2/SH01/human/2020/CHN (Accession no. <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MT121215&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom">MT121215</a>) as the reference. As only genomes within a specified MUMI distance threshold are recruited, we used option-c to force include all the strains and for output, it produces a core-genome alignment, variant calls and a phylogeny based on Single nucleotide polymorphisms. The SNPs were further visualized in Gingr, a dynamic visual platform (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Treangen <em>et al</em>., 2014</a>). Further, the tree was visualized in interactive Tree of Life (iTOL) (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Letunic and Bork, 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>SARS-CoV-2 protein annotation and host-pathogenic interactions</h3><p id="p-11">SARS-CoV-2/SH01/human/2020/CHN virus genome having accession no. <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MT121215.1&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom">MT121215.1</a> was used for protein-protein network analysis. Since, none of the SARS-CoV-2 genomes are updated in any protein database, we first annotated the genes using BLASTp tool (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em>, 1990</a>). The similarity searches were performed against SARS-CoV isolate Tor2 having accession no. <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AY274119&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom">AY274119</a> selected from NCBI at default parameters. The annotated SARS-CoV-2 proteins were mapped against virSITE (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Stano <em>et al.</em>, 2016</a>) and interaction databases such as STRING Virus v10.5 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cook <em>et al.</em>, 2018</a>) and IntRact (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kerrien <em>et al.</em>, 2013</a>) for predicting their interaction against host proteins. These proteins were either the direct targets of HCoV proteins or were involved in critical pathways of HCoV infection identified by multiple experimental sources. To build a comprehensive list of human PPIs, we assembled data from a total of 18 bioinformatics and systems biology databases with five types of experimental evidence: (i) binary PPIs tested by high-throughput yeast two-hybrid (Y2H) systems; (ii) binary, physical PPIs from protein 3D structures; (iii) kinase-substrate interactions by literature-derived low-throughput or high-throughput experiments; (iv) signalling network by literature-derived low-throughput experiments; and (v) literature-curated PPIs identified by affinity purification followed by mass spectrometry (AP-MS), Y2H, or by literature-derived low (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Szklarczyk <em>et al</em>., 2018</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cook <em>et al</em>., 2018</a>).</p><p id="p-12">Filtered proteins (confidence value: 0.7) were mapped to their Entrez ID (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Maglott <em>et al.</em>, 2005</a>) based on the NCBI database used for interactome analysis. HPI were stimulated using Cytoscape v.3.7.2 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Shannon <em>et al.</em>, 2003</a>).</p></div><div id="sec-7" class="subsection"><h3>Functional enrichment analysis</h3><p id="p-13">Next, functional studies were performed using the Kyoto Encyclopedia of Genes and Genomes (KEGG) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kanehisa and Goto, 2000</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kanehisa <em>et al.</em>, 2016</a>) and Gene Ontology (GO) enrichment analyses using UniPROT database (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">UniProt Consortium, 2007</a>) to evaluate the biological relevance and functional pathways of the HCoV-associated proteins. All functional analyses were performed using STRINGenrichment and STRINGify, plugin of Cytoscape v.3.7.2 (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Shannon <em>et al.</em>, 2003</a>). Network analysis was performed by tool NetworkAnalyzer, plugin of Cytoscape with orthogonal layout.</p></div></div><div class="section" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>General genomic attributes of SARS-CoV-2</h3><p id="p-14">In this study, we analysed a total of 95 SARS-CoV-2 strains isolated between December 2019-March 2020 from 11 different countries namely USA (n=52), China (n=30), Japan (n=3), India (n=2), Taiwan (n=2) and one each from Australia, Brazil, Italy, Nepal, South Korea and Sweden (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). A total of 68 strains were isolated from either oronasopharynges or lungs, while two of them were isolated from faeces suggesting both respiratory and gastrointestinal connection of SARS-CoV-2 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). No information of the source of isolation of the remaining isolates is available. The average genome size and GC content was found to be 29879 ± 26.6 bp and 37.99 ± 0.018%, respectively. All these isolates were found to harbour 9 open reading frames coding for ORF1a (13218 bp) and ORF1b (7788 bp) polyproteins, surface glycoprotein or S-protein (3822 bp), ORF3a protein (828 bp), membrane glycoprotein or M-protein (669 bp), ORF6 protein (186 bp), ORF7a protein (366 bp), ORF8 protein (366 bp), and nucleocapsid phosphoprotein or N-protein (1260 bp) which is in consensus with a recently published study (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ren <em>et al</em>., 2020</a>). The ORF1a harbours 12 non-structural protein (nsp) namely nsp1, nsp2, nsp3 (papain-like protease or PLpro domain), nsp4, nsp5 (3C-like protease or 3CLpro), nsp6, nsp7, nsp8, nsp9, nsp10, nsp11and nsp12 (RNA-dependent RNA polymerase or RdRp) (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Ren <em>et al.</em>, 2020</a>). Similarly, ORF1b contains four putative nsp’s namely nsp13 (helicase or Hel), nsp14 (3′-to-5′ exoribonuclease or ExoN), nsp15 and nsp16.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution pattern of COVID-19 cases across the globe and the number of isolates (in circle) included in the present study." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1760362176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The distribution pattern of COVID-19 cases across the globe and the number of isolates (in circle) included in the present study.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F1.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F1.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-15" class="first-child">The distribution pattern of COVID-19 cases across the globe and the number of isolates (in circle) included in the present study.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Phylogenomic analysis: defining evolutionary relatedness</h3><p id="p-16">Our analysis revealed that strains of human infecting SARS-CoV-2 are novel and highly identical (&gt;99.9%). A recent study established the closest neighbour of SARS-CoV-2 as SARSr-CoV-RaTG13, a bat coronavirus (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gorbaleneya <em>et al.</em>, 2020</a>). As COVID19 transits from epidemic to pandemic due to extremely contagious nature of the SARS-CoV-2, it was interesting to draw the relation between strains and their geographical locations. In this study, we employed two methods to delineate phylogenomic relatedness of the isolates: core genome (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A &amp; C</a>) and single nucleotide polymorphisms (SNPs) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Phylogenies obtained were annotated with country of isolation of each strain (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a> &amp; <a id="xref-fig-2-4" class="xref-fig" href="#F2">B</a>). The phylogenetic clustering was found majorly concordant by both core-genome (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>) and SNP based methods (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). The strains formed a monophyletic clade, in which MT093571.1 (South Korea) and MT039890.1 (Sweden) were most diverged. Focusing on the edge-connection between the neighbouring countries from where the transmission is more likely to occur, we noted a strain from Taiwan (MT066176) closely clustered with another from China (MT121215.1). With exception to these two strains, we did not find any connection between strains of neighbouring countries. Thus, most strains belonging to the same country clustered distantly from each other and showed relatedness with strains isolated from distant geographical locations (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a> &amp; <a id="xref-fig-2-7" class="xref-fig" href="#F2">B</a>). For instance, a SARS-CoV-2 strain isolated from Nepal (MT072688) clustered with a strain from USA (MT039888). Also, strains from Wuhan (LR757998 and LR757995), where the virus was originated, showed highest identity with USA as well as China strains; strains from India, MT012098 and MT050493 clustered closely with China and USA strains, respectively (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a> &amp; <a id="xref-fig-2-9" class="xref-fig" href="#F2">B</a>). Similarly, Australian strain (MT007544) showed close clustering with USA strain (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2A</a> &amp; <a id="xref-fig-2-11" class="xref-fig" href="#F2">B</a>) and one strain from Taiwan (MT066175) clustered nearly with China strain (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2B</a>). Isolates from Italy (MT012098) and Brazil (MT126808) clustered with different USA strains (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2A</a> &amp; <a id="xref-fig-2-14" class="xref-fig" href="#F2">B</a>). Notably, isolates from same country or geographical location formed a mosaic pattern of phylogenetic placements of countries’ isolates. For viral transmission, contact between the individuals is also an important factor, supposedly due to which the spread of identical strains across the border of neighbouring countries is more likely. But we obtained a pattern where Indian strains showed highest similarity with USA and China strains, Australian strains with USA strains, Italy and Brazil strains with USA strains among others. This depicts the viral spread across different communities. However, as genomes of SARS-CoV-2 were available mostly from USA and China, sampling biasness is evident in analysed dataset as available on NCBI. Thus, it is plausible for strains from other countries to show most similarity with strains from these two countries. In near future as more and more genome sequences will become available from different geographical locations, more accurate patterns of their relatedness across the globe will become available</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Core genome based phylogenetic analysis of SARS-CoV-2 isolates using the Maximum Likelihood method based on the Tamura-Nei model. The analysis involved 95 SARS-CoV-2 sequences with a total of 28451 nucleotide positions. Bootstrap values more than 70% are shown on branches as blue dots with sizes corresponding to the bootstrap values. The coloured circle represents the country of origin of each isolate. The two isolates from Wuhan are marked separately on the outside of the ring. B) SNP based phylogeny of SARS-CoV-2 isolates. Highly similar genomes of coronaviruses were taken as input by Parsnp. Whole-genome alignments were made using libMUSCLE aligner using the annotated genome of MT121215 strain as reference. Parsnp identifies the maximal unique matches (MUMs) among the query genomes provided in a single directory. As only genomes within a specified MUMI distance threshold are recruited, option-c to force include all the strains was used. The output phylogeny based on Single nucleotide polymorphisms was obtained following variant calling on core-genome alignment. C) The minimum spanning tree generated using Maximum Likelihood method and Tamura-Nei model showing the genetic relationships of SARS-CoV-2 isolates with their geographical distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1760362176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A) Core genome based phylogenetic analysis of SARS-CoV-2 isolates using the Maximum Likelihood method based on the Tamura-Nei model. The analysis involved 95 SARS-CoV-2 sequences with a total of 28451 nucleotide positions. Bootstrap values more than 70% are shown on branches as blue dots with sizes corresponding to the bootstrap values. The coloured circle represents the country of origin of each isolate. The two isolates from Wuhan are marked separately on the outside of the ring. B) SNP based phylogeny of SARS-CoV-2 isolates. Highly similar genomes of coronaviruses were taken as input by Parsnp. Whole-genome alignments were made using libMUSCLE aligner using the annotated genome of MT121215 strain as reference. Parsnp identifies the maximal unique matches (MUMs) among the query genomes provided in a single directory. As only genomes within a specified MUMI distance threshold are recruited, option-c to force include all the strains was used. The output phylogeny based on Single nucleotide polymorphisms was obtained following variant calling on core-genome alignment. C) The minimum spanning tree generated using Maximum Likelihood method and Tamura-Nei model showing the genetic relationships of SARS-CoV-2 isolates with their geographical distribution.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F2.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F2.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-17" class="first-child">A) Core genome based phylogenetic analysis of SARS-CoV-2 isolates using the Maximum Likelihood method based on the Tamura-Nei model. The analysis involved 95 SARS-CoV-2 sequences with a total of 28451 nucleotide positions. Bootstrap values more than 70% are shown on branches as blue dots with sizes corresponding to the bootstrap values. The coloured circle represents the country of origin of each isolate. The two isolates from Wuhan are marked separately on the outside of the ring. B) SNP based phylogeny of SARS-CoV-2 isolates. Highly similar genomes of coronaviruses were taken as input by Parsnp. Whole-genome alignments were made using libMUSCLE aligner using the annotated genome of MT121215 strain as reference. Parsnp identifies the maximal unique matches (MUMs) among the query genomes provided in a single directory. As only genomes within a specified MUMI distance threshold are recruited, option-c to force include all the strains was used. The output phylogeny based on Single nucleotide polymorphisms was obtained following variant calling on core-genome alignment. C) The minimum spanning tree generated using Maximum Likelihood method and Tamura-Nei model showing the genetic relationships of SARS-CoV-2 isolates with their geographical distribution.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>SNPs in the SARS-CoV-2 genomes</h3><p id="p-18">SNPs in all predicted ORFs in each genome were analysed using SARS-CoV-2/SH01/human/2020/CHN as a reference. SNPs were determined using maximum unique matches between the genomes of coronavirus, we observed that the strains isolated from USA (MT188341; MN985325; MT020881; MT020880; MT163719; MT163718; MT163717; MT152824; MT163720; MT188339) are the most evolved and they carry set of unique point mutations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) in nsp13, nsp14, nsp15, nsp16 (present in orf1b polyprotein region) and S-Protein. All the mutated proteins are non-structural proteins (NSP) functionally involved in forming viral replication-transcription complexes (RTC) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Snijder <em>et al.</em>, 2016</a>). For instance, non-structural protein 13 (nsp13), belongs to helicase superfamily 1 and is putatively involved in viral RNA replication through RNA-DNA duplex unwinding (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jang <em>et al.</em>, 2020</a>) whereas nsp14 and nsp15 are exoribonuclease and endoribonuclease, respectively (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Becares <em>et al.</em>, 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Athmer <em>et al.</em>, 2017</a>). nsp16 functions as a mRNA cap-1 methyltransferase (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">von Grotthuss <em>et al.</em>, 2003</a>). All these proteins containing SNPs at several positions (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) indicate that viral machinery for its RNA replication and processing is utmost evolved in strains from USA as compared to the other countries. Further we analysed the SNPs at protein level and interestingly in ORF1b protein, there were amino acid substitutions at P1327L, Y1364C and S2540F in USA isolates. One isolate namely USA0/MN1-MDH1/2020 (MT188341) carried frameshift mutation between positions 2540 and 2570 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), which might affect the functioning of nsp16 (2′-O-MTase). As the proteins involved in viral replication are evolving rapidly, this highlights the need to consider these mutants in order to develop the treatment strategies.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple sequence alignment of ORF1b protein showing amino acid substitutions at three positions: P1327L, Y1364C and S2540F. The isolate USA/MN1-MDH1/2020 (MT188341) showed the frameshift mutation between positions 2540 and 2570." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1760362176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Multiple sequence alignment of ORF1b protein showing amino acid substitutions at three positions: P1327L, Y1364C and S2540F. The isolate USA/MN1-MDH1/2020 (MT188341) showed the frameshift mutation between positions 2540 and 2570.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F3.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F3.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-19" class="first-child">Multiple sequence alignment of ORF1b protein showing amino acid substitutions at three positions: P1327L, Y1364C and S2540F. The isolate USA/MN1-MDH1/2020 (MT188341) showed the frameshift mutation between positions 2540 and 2570.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1225324/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1225324/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1225324" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-20" class="first-child">Major mutations present in different isolates of SARS-CoV-2 at different locations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Direction of selection</h3><p id="p-21">Our analysis revealed that ORF8 (121 a.a.) (dN/dS= 35.8) along with ORF3a (275 bp) (dN/dS= 8.95) showed highest dN/dS values among the nine ORFs thus, have much greater number of non-synonymous substitutions than the synonymous substitution (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4D</a>). Values of dN/dS ≫1 are indicative of strong divergent lineage (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kryazhimskiy <em>et. al.</em>, 2008</a>). Thus, both of these proteins are evolving under high selection pressure and are highly divergent ORFs across strains. Two other proteins, ORF1ab polyprotein (dN/dS= 0.996, 0.575) and S protein (dN/dS= 0.88) might confer selective advantage with host challenges and survival. The dN/dS rates nearly 1 and greater than 1 suggests that the strains are coping up with the challenges <em>i.e.</em>, immune responses and inhibitory environment of host cells (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Pond <em>et al</em>., 2005</a>). The other gene clusters namely M-protein and orf1a polyprotein did not possess at least three unique sequences necessary for the analysis, hence, should be similar across the strains. The two genes ORF1ab polyprotein encodes for protein translation and post translation modification found to be evolved which actively translates, enhance the multiplication and facilitates growth of virus inside the host. Similarly, the S protein which helps in the entry of virus to the host cells by surpassing the cell membrane found to be accelerated towards positive selection confirming the successful ability of enzyme to initiate the infection. Another positive diversifying gene N protein encodes for nucleocapsid formation which protects the genetic material of virus form host immune responses such as cellular proteases. Overall, the data represent that the growth and multiplication related genes are highly evolving. The other proteins with dN/dS values equal to zero suggesting a conserved repertoire of genes.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) SARS-CoV-2 -Host interactome analysis. Sub-setnetwork highlighting SARS-CoV-2 and host nodes targeting each other. In total, nine direct interactions were observed (shown with red arrows). (B) Circular genome map of SARS-CoV-2 with genome size of 29.8 Kb compared with that of SARS-CoV generated using CGView. The ruler for genome size is shown as innermost ring where Kbp stands for kilo base pairs. Concentric circles from inside to outside denote: SARS-CoV genome (used as reference), G + C content, G + C skew, predicted ORFs in SARS-CoV-2 genome and annotated CDS in SARS-CoV-2 genome. Gaps in alignment are shown in white. The positive and negative deviation from mean G + C content and G + C skew are represented with outward and inward peaks respectively. (C) SARS-CoV 2 and Host interactome generated using STRING Virus interaction database v10.5. Both interacting and non-interacting viral proteins are shown. (D) Estimation of purifying natural selection pressure in nine coding sequences of SARS-CoV-2. dN/dS values are plotted as a function of dS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1760362176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) SARS-CoV-2 -Host interactome analysis. Sub-setnetwork highlighting SARS-CoV-2 and host nodes targeting each other. In total, nine direct interactions were observed (shown with red arrows). (B) Circular genome map of SARS-CoV-2 with genome size of 29.8 Kb compared with that of SARS-CoV generated using CGView. The ruler for genome size is shown as innermost ring where Kbp stands for kilo base pairs. Concentric circles from inside to outside denote: SARS-CoV genome (used as reference), G + C content, G + C skew, predicted ORFs in SARS-CoV-2 genome and annotated CDS in SARS-CoV-2 genome. Gaps in alignment are shown in white. The positive and negative deviation from mean G + C content and G + C skew are represented with outward and inward peaks respectively. (C) SARS-CoV 2 and Host interactome generated using STRING Virus interaction database v10.5. Both interacting and non-interacting viral proteins are shown. (D) Estimation of purifying natural selection pressure in nine coding sequences of SARS-CoV-2. dN/dS values are plotted as a function of dS.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F4.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F4.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.25.006213/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-22" class="first-child">(A) SARS-CoV-2 -Host interactome analysis. Sub-setnetwork highlighting SARS-CoV-2 and host nodes targeting each other. In total, nine direct interactions were observed (shown with red arrows). (B) Circular genome map of SARS-CoV-2 with genome size of 29.8 Kb compared with that of SARS-CoV generated using CGView. The ruler for genome size is shown as innermost ring where Kbp stands for kilo base pairs. Concentric circles from inside to outside denote: SARS-CoV genome (used as reference), G + C content, G + C skew, predicted ORFs in SARS-CoV-2 genome and annotated CDS in SARS-CoV-2 genome. Gaps in alignment are shown in white. The positive and negative deviation from mean G + C content and G + C skew are represented with outward and inward peaks respectively. (C) SARS-CoV 2 and Host interactome generated using STRING Virus interaction database v10.5. Both interacting and non-interacting viral proteins are shown. (D) Estimation of purifying natural selection pressure in nine coding sequences of SARS-CoV-2. dN/dS values are plotted as a function of dS.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>SARS CoV-2-Host interactome unveils immunopathogenicity of COVID-19</h3><p id="p-23">Although the primary mode of infection is human to human transmission through close contact, which occurs via spraying of nasal droplets from the infected person, yet the primary site of infection and pathogenesis of SAR-CoV-2 is still not clear and under investigation. To explore the role of SARS-CoV-2 proteins in host immune evasion, the SARSCoV-2 proteins were mapped over host proteome database (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). We identified a total 28 proteins from host proteome forming close association with 25 viral proteins present in 9 ORFs of SARS-CoV-2 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>). The network was trimmed in Cytoscape v3.7.2 where only interacting proteins were selected. Only 12 viral proteins were found to interact with host proteins (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). Detailed analysis of interactome highlighted 9 host proteins in direct association with 6 viral proteins. Further, the network was analysed for identification of regulatory hubs based on degree analysis. We identified Mitogen activated protein kinase 1 (MAPK1) and AKT proteins as major hubs forming 24 and 21 interactions in the network respectively, highlighting their crucial role in pathogenesis. Recently, Huang <em>et al</em>, demonstrated the role of Mitogen activated protein kinase (MAPK) in COVID-19 mediated blood immune responses in infected patients (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Huang <em>et al.</em>, 2020</a>) and showed that MAPK activation certainly plays a major defence mechanism.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1225322/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1225322/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <p id="p-24" class="first-child">Description of SARS-CoV2 proteins and its similarity in comparison to SARS-CoV used for PPI prediction.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Gene Ontology based functional annotation studies predicted the role of direct interactions of several viral proteins with host proteins. One such protein is non-structural protein2 (nsp2) which directly interacts with host Prohbitin (PHB), a known regulator of cell proliferation and maintains functional integrity of mitochondria (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Tatsuta <em>et al</em>., 2005</a>). SARS-CoV nsp2 is also known for its interaction with host PHB1 and PHB2 (Cromwell T <em>et al.</em>, 2009). Nsp2 is a methyltransferase like domain which is known to mediate mRNA cap 2’-O-ribose methylation to the 5’-cap structure of viral mRNAs. This N7-methylguanosine cap is required for the action of nsp16 (2’-O-methyltransferase) and nsp10 complex (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al</em>., 2011</a>). This 5’-capping of viral RNA plays a crucial role in escape of virus from innate immunity recognition (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chen <em>et al</em>., 2011</a>). Hence, nsp2-is responsible for modulating host cell survival strategies by altering host cell environment (Cromwell <em>et al</em>., 2009). Based on network predicted we propose nsp16/nsp10 interface as a better drug target for anti-coronavirus drugs corresponding to the prediction made by Cormwell and group (2011).</p><p id="p-26">Similarly, the viral protein Papain-like proteinase (PL-PRO) which has deubiquitinase and deISGylating activity is responsible for cleaving viral polyprotein into 3 mature proteins which are essential for viral replication. Our study showed that PL-PRO directly interacts with PPP1CA which is a protein phosphatase that associates with over 200 regulatory host proteins to form highly specific holoenzymes. PL-PRO is also found to interact with TGFβ which is a beta transforming growth factor and promotes T-helper 17 cells (Th17) and regulatory T-cells (T<sub>reg</sub>) differentiation (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Zhang <em>et al</em>., 2007</a>). Reports have shown the PL-PRO induced upregulation of TGFβ in human promonocytes via MAPK pathway result in pro-fibrotic responses (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al</em>., 2016</a>). This reflect that viral PL-PRO antagonises innate immune system and is directly involved in the pathogenicity of SARS-CoV 2 induced pulmonary fibrosis (tu sing fung, 2019; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Li <em>et al</em>., 2016</a>). Many COVID-19 patients develop acute respiratory distress syndrome (ADRS) which leads to pulmonary edema and lung failure (Xu Z <em>et al</em>., 2020, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Huang <em>et al</em>., 2019</a>). These symptoms are because of cytokine storm manifesting elevated levels of pro-inflammatory cytokines like IL6, IFNγ, IL17, IL1β etc (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Huang <em>et al</em>., 2019</a>). These results are in agreement with our prediction where we found IL6 as an interacting partner. Our study also showed JAK1/2 as an interacting partner which is known for IFNγ signalling. It is well known that TGFβ along with IL6 and STAT3 promotes Th17 differentiation by inhibiting SOCS3. Th17 is a source of IL17, which is commonly found in serum samples of COVID19 patients. Hence, our interactome is supported from these findings where we found SOCS3, STAT3, JAK1/2 as an interacting partner. The results suggested that proinflammatory cytokine storm is one of the reasons for SARS-CoV-2 mediated immune-pathogenicity.</p><p id="p-27">In next cycle of physical events the viral protein NC (nucleoprotein), which is a major structural part of SARV family associates with the genomic RNA to form a flexible, helical nucleocapsid. Interaction of this protein with SMAD3 leads to inhibition of apoptosis of SARS-CoV infected lung cells (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhao <em>et al</em>, 2008</a>), which is a successful strategy of immune evasion by virus. More complex and multiple associations of ORF7a viral protein which is a non-structural protein and known as growth factor for SARS family viruses, directly captures BCL2L1 which is a potent regulator of apoptosis. <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Tan et al (2007)</a> have shown that SARS-CoV ORF7a protein induces apoptosis by interacting with Bcl X<sub>L</sub> protein which is a responsible for lymphopenia, an abnormality found in SARS-CoV infected patients (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Tan <em>et al.</em>, 2007</a>). Another target of viral ORF7a protein is SGTA (Small glutamine-rich tetratricopeptide repeat) which is an ATPase regulator and promotes viral encapsulation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fielding 2006</a>). Subordinate viral proteins M (Membrane), S (Glycoprotein) and ORF3a (viroporin) were found to interact with each other. This interaction is important for viral cell formation and budding (<em>de <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Haan et al., 1999</a></em>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Klumperman <em>et al</em>., 1994</a>). Studies have shown the localization of ORF3a protein in Golgi apparatus of SARS-CoV infected patients along with M protein and responsible for viral budding and cell injury (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Yuan <em>et al</em>., 2005</a>). ORF3a protein also targets the functioning of CAV1 (Caveolin 1), caveolae protein, act as a scaffolding protein within caveolar membranes. CAV1 have been reported to be involved in viral replication, persistence, and the potential role in pathogenesis in HIV infection also (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mergia, 2017</a>). Thus, ORF3a interaction will upregulate viral replication thus playing very crucial role in pathogenesis. Multiple methyltransferase assembly viral proteins (nsp7, nsp8, nsp9, RdRp) which are nuclear structural proteins were observed to target the SPECC1 proteins and linked with cytokinesis and spindle formations during division. Thus, major viral assembly also targets the proteins linked with immunity and cell division. Taken together, we estimated that SARS-CoV-2 manipulate multiple host proteins for its own survival while, their interaction is also a reason for immunopathogenicity.</p></div></div><div class="section" id="sec-14"><h2 class="">Conclusions</h2><p id="p-28">As COVID19 continues to impact virtually all human lives worldwide due to its extremely contagious nature, it has spiked the interest of scientific community all over the world to understand better the pathogenesis of the novel SARS-CoV-2. In this study, the analysis was performed on the genomes of the novel SARS-CoV-2 isolates recently reported from different countries to viral pathogenesis. With the limited data so far available we observed no direct transmission pattern of the novel SARS-CoV-2 in the neighbouring countries through our analyses of the phylogenomic relatedness of geographical isolates. The isolates from same locations were distant phylogenetically, for instance, isolates from USA and China. Thus, there appears to be a mosaic pattern of transmission indicative of the result of infected human travel across different countries. As COVID19 transited from epidemic to pandemic within a short time, it does not look surprising from the genome structures of the viral isolates. The genomes of six isolates, specifically from USA, were found to harbour unique amino acid SNPs and showed amino acid substitutions in ORF1b protein and S-protein, while one of them also harboured a frameshift mutation. This is suggestive of the severity of the mutating viral genomes within the population of USA. These proteins are directly involved in formation of viral replication-transcription complexes (RTC). Therefore, we argue that the novel SARS-CoV-2 has fast evolving replicative machinery and that it is urgent to consider these mutants in order to develop strategies for COVID19 treatment. The ORF1ab polyprotein protein and S-protein were also found to have dN/dS values approaching to 1 and thus might confer selective advantage to evade host responsive mechanisms. The construction of SARS-CoV-2-human interactome revealed that its pathogenicity is mediated by surge in pro-inflammatory cytokine. It is predicted that major immune-pathogenicity mechanism by SARS-CoV-2 includes the host cell environment alteration by disintegration by signal transduction pathways and immunity evasion by several protection mechanisms. The mode of entry of this virus by S-proteins inside host cell is still unclear but it might be similar to SARS CoV-1 like viruses. Lastly, we believe that COVID-19 is being transmitted from human to human, but as more data accumulate the picture will be more clear, as these virus spread beyond the imagination of scientific community.</p></div><div class="section" id="sec-15"><h2 class="">Authors Contribution</h2><p id="p-29">RL, RK, VG conceived the study and designed the study. RK, HV, NS, US, VG, MS, SN, PH executed the analysis and prepared figures. RK, HV, RS, NS, US, VG, MS, SN, PH, CT, NN, SA, CDR, MV wrote the manuscript with contributions from all authors. YS and RKN provided time to time guidance.</p></div><div class="section COI-statement" id="sec-16"><h2 class="">Conflict of Interest</h2><p id="p-30">Authors declare no conflict of Interest</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-31">RK acknowledges Magadh University, Bodh Gaya for providing support. RL and US also acknowledge The National Academy of Sciences, India, for support under the NASI‐Senior Scientist Platinum Jubilee Fellowship Scheme. NS, SN, CT acknowledge Council of Scientific and Industrial Research, New Delhi (CSIR) for doctoral fellowships. HV would like to thank Ramjas College, University of Delhi, Delhi for providing support. VG and MS acknowledge Phixgen Pvt. Ltd. and NMCG for research fellowship. PH would like to thank Maitrey College, University of Delhi, Delhi for providing support.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">E.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lipman</span>,  <span class="cit-name-given-names">D.J.</span></span> (<span class="cit-pub-date">1990</span>). <span class="cit-article-title">Basic Local Alignment Search Tool</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">215</span>, <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Athmer</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grunewald</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>,  <span class="cit-name-given-names">M.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">In situ tagged nsp15 reveals interactions with coronavirus replication/transcription complex-associated proteins</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e02320</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D8%26rft.spage%253De02320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.3" data-doi="10.1073/pnas.0805270106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barcena</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oostergetel</span>,  <span class="cit-name-given-names">G.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bartelink</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faas</span>,  <span class="cit-name-given-names">F.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verkleij</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>,  <span class="cit-name-given-names">P.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koster</span>,  <span class="cit-name-given-names">A.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bosch</span>,  <span class="cit-name-given-names">B.J.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Cryo-electron tomography of mouse hepatitis virus: Insights into the structure of the coronavirion</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">106</span>, <span class="cit-fpage">582</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0805270106%26rft_id%253Dinfo%253Apmid%252F19124777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDYvMi81ODIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI1LjAwNjIxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.4" data-doi="10.1128/JVI.03259-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Becares</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pascual-Iglesias</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nogales</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sola</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zuñiga</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Mutagenesis of coronavirus nsp14 reveals its potential role in modulation of the innate immune response</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>, <span class="cit-fpage">5399</span>–<span class="cit-lpage">5414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03259-15%26rft_id%253Dinfo%253Apmid%252F27009949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xMS81Mzk5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yNS4wMDYyMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.5" data-doi="10.1007/3-540-26765-4_1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brian</span>,  <span class="cit-name-given-names">D.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Coronavirus genome structure and replication</span>. <abbr class="cit-jnl-abbrev">Curr Top Microbiol Immunol</abbr> <span class="cit-vol">287</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bmicrobiology%2Band%2Bimmunology%26rft.stitle%253DCurr%2BTop%2BMicrobiol%2BImmunol%26rft.aulast%253DBrian%26rft.auinit1%253DD.%2BA.%26rft.volume%253D287%26rft.spage%253D1%26rft.epage%253D30%26rft.atitle%253DCoronavirus%2Bgenome%2Bstructure%2Band%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F3-540-26765-4_1%26rft_id%253Dinfo%253Apmid%252F15609507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/3-540-26765-4_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15609507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227919900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.6" data-doi="10.1371/journal.ppat.1002294"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ke</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tien</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahola</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Biochemical and structural insights into the mechanisms of SARS coronavirus RNA ribose 2’-O-methylation by nsp16/nsp10 protein complex</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e1002294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002294%26rft.epage%253De1002294%26rft.atitle%253DBiochemical%2Band%2Bstructural%2Binsights%2Binto%2Bthe%2Bmechanisms%2Bof%2BSARS%2Bcoronavirus%2BRNA%2Bribose%2B2%2527-O-methylation%2Bby%2Bnsp16%252Fnsp10%2Bprotein%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002294%26rft_id%253Dinfo%253Apmid%252F22022266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22022266&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.7" data-doi="10.1016/0042-6822(82)90095-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Collins</span>,  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knobler</span>,  <span class="cit-name-given-names">R.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Powell</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>,  <span class="cit-name-given-names">M.J.</span></span> (<span class="cit-pub-date">1982</span>). <span class="cit-article-title">Monoclonal antibodies to murine hepatitis virus-4 (strain JHM) define the viral glycoprotein responsible for attachment and cell--cell fusion</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">119</span>, <span class="cit-fpage">358</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DCollins%26rft.auinit1%253DA.%2BR.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D358%26rft.epage%253D371%26rft.atitle%253DMonoclonal%2Bantibodies%2Bto%2Bmurine%2Bhepatitis%2Bvirus-4%2B%2528strain%2BJHM%2529%2Bdefine%2Bthe%2Bviral%2Bglycoprotein%2Bresponsible%2Bfor%2Battachment%2Band%2Bcell--cell%2Bfusion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252882%252990095-2%26rft_id%253Dinfo%253Apmid%252F6281979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(82)90095-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6281979&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982NQ95700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cook</span>,  <span class="cit-name-given-names">H.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doncheva</span>,  <span class="cit-name-given-names">N.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Mering</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">L.J.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Viruses.STRING: A Virus-Host Protein-Protein Interaction Database</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253D519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.9" data-doi="10.1128/JVI.00842-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cornillez-Ty</span>,  <span class="cit-name-given-names">C.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yates</span>,  <span class="cit-name-given-names">J.R.</span>   <span class="cit-name-suffix">3rd</span>.</span> <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>,  <span class="cit-name-given-names">M.J.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Severe acute respiratory syndrome coronavirus nonstructural protein 2 interacts with a host protein complex involved in mitochondrial biogenesis and intracellular signaling</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">83</span>, <span class="cit-fpage">10314</span>–<span class="cit-lpage">10318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00842-09%26rft_id%253Dinfo%253Apmid%252F19640993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8xOS8xMDMxNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMwLzIwMjAuMDMuMjUuMDA2MjEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.10"><div class="cit-metadata"><cite>de <span class="cit-auth"><span class="cit-name-surname">Haan</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smeets</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vernooij</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vennema</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rottier</span>,  <span class="cit-name-given-names">P.J.</span></span> (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Mapping of the coronavirus membrane protein domains involved in interaction with the spike protein</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">73</span>, <span class="cit-fpage">7441</span>–<span class="cit-lpage">7452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253Dde%2BHaan%26rft.auinit1%253DC.%2BA.%2BM.%26rft.volume%253D73%26rft.issue%253D9%26rft.spage%253D7441%26rft.epage%253D7452%26rft.atitle%253DMapping%2Bof%2Bthe%2BCoronavirus%2BMembrane%2BProtein%2BDomains%2BInvolved%2Bin%2BInteraction%2Bwith%2Bthe%2BSpike%2BProtein%26rft_id%253Dinfo%253Apmid%252F10438834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjczLzkvNzQ0MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMwLzIwMjAuMDMuMjUuMDA2MjEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.11" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>:<span class="cit-fpage">1792</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.12" data-doi="10.1007/978-1-4939-2438-7_1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">A.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Coronaviruses: an overview of their replication and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr> <span class="cit-vol">1282</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1282%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-2438-7_1%26rft_id%253Dinfo%253Apmid%252F25720466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-2438-7_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25720466&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.13" data-doi="10.1016/j.bbrc.2006.03.091"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fielding</span>,  <span class="cit-name-given-names">B.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunalan</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">T.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>,  <span class="cit-name-given-names">C.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">S.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">Y.J.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Severe acute respiratory syndrome coronavirus protein 7a interacts with hSGT</span>. <abbr class="cit-jnl-abbrev">Biochem Biophys Res Commun</abbr> <span class="cit-vol">343</span>,:<span class="cit-fpage">1201</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DFielding%26rft.auinit1%253DB.%2BC.%26rft.volume%253D343%26rft.issue%253D4%26rft.spage%253D1201%26rft.epage%253D1208%26rft.atitle%253DSevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Bprotein%2B7a%2Binteracts%2Bwith%2BhSGT.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2006.03.091%26rft_id%253Dinfo%253Apmid%252F16580632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2006.03.091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16580632&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fung</span>,  <span class="cit-name-given-names">T.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">D.X.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Human Coronavirus: Host-Pathogen Interaction</span>. <abbr class="cit-jnl-abbrev">Annu Rev Microbiol</abbr> <span class="cit-vol">73</span>, <span class="cit-fpage">529</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BMicrobiol%26rft.volume%253D73%26rft.spage%253D529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulyaeva</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haagmans</span>,  <span class="cit-name-given-names">B.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauber</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leontovich</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuman</span>,  <span class="cit-name-given-names">B.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Penzar</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Nat Microbiol</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">536</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D5%26rft.spage%253D536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.16" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saveliev</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vyahhi</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tesler</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.03.25.006213v1.17"><div class="cit-metadata unstructured"><a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">You</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Long</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Blood single cell immune profiling reveals the interferon-MAPK pathway mediated adaptive immune response for COVID-19</span>. <abbr class="cit-jnl-abbrev">BMJ</abbr>, doi: <a href="https://doi.org/10.1101/2020.03.15.20033472">https://doi.org/10.1101/2020.03.15.20033472</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.19" data-doi="10.1016/S0140-6736(20)30183-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">497</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.20" data-doi="10.1038/s41598-020-59121-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jang</span>,  <span class="cit-name-given-names">K.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeong</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kang</span>,  <span class="cit-name-given-names">D.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sp</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.E.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">A high ATP concentration enhances the cooperative translocation of the SARS coronavirus helicase nsP13 in the unwinding of duplex RNA</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-020-59121-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-020-59121-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.21" data-doi="10.1128/mBio.00165-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Josset</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gralinski</span>,  <span class="cit-name-given-names">L.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agnihothram</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sova</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carter</span>,  <span class="cit-name-given-names">V.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yount</span>,  <span class="cit-name-given-names">B.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Katze</span>,  <span class="cit-name-given-names">M.G.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Cell host response to infection with novel human coronavirus EMC predicts potential antivirals and important differences with SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e00165</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D4%26rft.spage%253De00165%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00165-13%26rft_id%253Dinfo%253Apmid%252F23631916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00165-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23631916&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.22" data-doi="10.1093/nar/28.1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goto</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">KEGG: kyoto encyclopedia of genes and genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">27</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKanehisa%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D30%26rft.atitle%253DKEGG%253A%2BKyoto%2BEncyclopedia%2Bof%2BGenes%2Band%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.27%26rft_id%253Dinfo%253Apmid%252F10592173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592173&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.23" data-doi="10.1093/nar/gkv1070"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawashima</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furumichi</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tanabe</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">KEGG as a reference resource for gene and protein annotation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">D457</span>–<span class="cit-lpage">D462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1070%26rft_id%253Dinfo%253Apmid%252F26476454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26476454&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.24" data-doi="10.1093/nar/gks1068"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kerrien</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aranda</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breuza</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bridge</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Broackes-Carter</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duesbury</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dumousseau</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feuermann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hinz</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">The IntAct molecular interaction database in 2012</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">41</span>, <span class="cit-fpage">D43</span>–<span class="cit-lpage">D47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1068%26rft_id%253Dinfo%253Apmid%252F23161681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23161681&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.25" data-doi="10.1128/JVI.68.10.6523-6534.1994"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klumperman</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Locker</span>,  <span class="cit-name-given-names">J.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meijer</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>,  <span class="cit-name-given-names">M.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geuze</span>,  <span class="cit-name-given-names">H.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rottier</span>,  <span class="cit-name-given-names">P.J.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">Coronavirus M proteins accumulate in the Golgi complex beyond the site of virion budding</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">68</span>, <span class="cit-fpage">6523</span>–<span class="cit-lpage">6534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKlumperman%26rft.auinit1%253DJ%26rft.volume%253D68%26rft.issue%253D10%26rft.spage%253D6523%26rft.epage%253D6534%26rft.atitle%253DCoronavirus%2BM%2Bproteins%2Baccumulate%2Bin%2Bthe%2BGolgi%2Bcomplex%2Bbeyond%2Bthe%2Bsite%2Bof%2Bvirion%2Bbudding.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.68.10.6523-6534.1994%26rft_id%253Dinfo%253Apmid%252F8083990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI2OC8xMC82NTIzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yNS4wMDYyMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.26" data-doi="10.1093/molbev/msi105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>,  <span class="cit-name-given-names">S.L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Frost</span>,  <span class="cit-name-given-names">S.D.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Not So Different After All: A Comparison of methods for detecting amino acid sites under selection</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">1208</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi105%26rft_id%253Dinfo%253Apmid%252F15703242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15703242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228700200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.27" data-doi="10.1371/journal.pgen.1000304"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kryazhimskiy</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Plotkin</span>,  <span class="cit-name-given-names">J.B.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">The Population Genetics of dN/dS</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e1000304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKryazhimskiy%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000304%26rft.epage%253De1000304%26rft.atitle%253DThe%2Bpopulation%2Bgenetics%2Bof%2BdN%252FdS.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000304%26rft_id%253Dinfo%253Apmid%252F19081788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000304&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19081788&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.28" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tamura</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis version 7.0 for bigger datasets</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.29" data-doi="10.1073/pnas.0506735102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">S.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">P.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsoi</span>,  <span class="cit-name-given-names">H.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">B.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">S.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">K.H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.Y.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">14040</span>–<span class="cit-lpage">14045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0506735102%26rft_id%253Dinfo%253Apmid%252F16169905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzM5LzE0MDQwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yNS4wMDYyMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.30" data-doi="10.1146/annurev-virology-110615-042301"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Structure, function, and evolution of coronavirus spike proteins</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">237</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D3%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-110615-042301%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-virology-110615-042301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.31" data-doi="10.1038/srep25754"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jou</span>,  <span class="cit-name-given-names">Y.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jou</span>,  <span class="cit-name-given-names">Y.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">T.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Y.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">C.W.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">SARS coronavirus papain-like protease induces Egr-1-dependent up-regulation of TGF-β1 via ROS/p38 MAPK/STAT3 pathway</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">25754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D25754%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep25754%26rft_id%253Dinfo%253Apmid%252F27173006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep25754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27173006&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhan</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Z.</span></span>. <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancelet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">565</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancelet%26rft.volume%253D395%26rft.spage%253D565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.33" data-doi="10.3390/v6082991"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McBride</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Zyl</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fielding</span>,  <span class="cit-name-given-names">B.C.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">The coronavirus nucleocapsid is a multifunctional protein</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">2991</span>–<span class="cit-lpage">3018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D6%26rft.spage%253D2991%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv6082991%26rft_id%253Dinfo%253Apmid%252F25105276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v6082991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25105276&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.34" data-doi="10.1093/nar/gkw290"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Letunic</span>,  <span class="cit-name-given-names">I.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bork</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">W242</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw290%26rft_id%253Dinfo%253Apmid%252F27095192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27095192&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.35" data-doi="10.1093/nar/gni052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maglott</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ostell</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pruitt</span>,  <span class="cit-name-given-names">K.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tatusova</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Entrez Gene: gene-centered information at NCBI</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">D54</span>–<span class="cit-lpage">D58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMaglott%26rft.auinit1%253DD.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD54%26rft.epage%253DD58%26rft.atitle%253DEntrez%2BGene%253A%2Bgene-centered%2Binformation%2Bat%2BNCBI%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgni052%26rft_id%253Dinfo%253Apmid%252F15608257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gni052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15608257&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226524300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mergia</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">The Role of Caveolin 1 in HIV Infection and Pathogenesis</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D9%26rft.spage%253D129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.37" data-doi="10.3201/eid2004.131746"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>,  <span class="cit-name-given-names">V.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reusken</span>,  <span class="cit-name-given-names">C.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ritz</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Godeke</span>,  <span class="cit-name-given-names">G.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lattwein</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kallies</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siemens</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Beek</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drexler</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muth</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosch</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wernery</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koopmans</span>,  <span class="cit-name-given-names">M.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wernery</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Antibodies against MERS coronavirus in dromedary camels, United Arab Emirates, 2003 and 2013</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">552</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2004.131746%26rft_id%253Dinfo%253Apmid%252F24655412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2004.131746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24655412&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.38" data-doi="10.1093/bioinformatics/bty121"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tomii</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Parallelization of MAFFT for large-scale multiple sequence alignments</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">2490</span>–<span class="cit-lpage">2492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D2490%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty121%26rft_id%253Dinfo%253Apmid%252F29506019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29506019&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.39" data-doi="10.1128/JVI.00645-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neuman</span>,  <span class="cit-name-given-names">B.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adair</span>,  <span class="cit-name-given-names">B.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoshioka</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quispe</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orca</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milligan</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeager</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>,  <span class="cit-name-given-names">M.J.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Supramolecular architecture of severe acute respiratory syndrome coronavirus revealed by electron cryomicroscopy</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">80</span>,<span class="cit-fpage">7918</span>–<span class="cit-lpage">7928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DNeuman%26rft.auinit1%253DB.%2BW.%26rft.volume%253D80%26rft.issue%253D16%26rft.spage%253D7918%26rft.epage%253D7928%26rft.atitle%253DSupramolecular%2BArchitecture%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BRevealed%2Bby%2BElectron%2BCryomicroscopy%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00645-06%26rft_id%253Dinfo%253Apmid%252F16873249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xNi83OTE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yNS4wMDYyMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.40" data-doi="10.1016/j.jsb.2010.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neuman</span>,  <span class="cit-name-given-names">B.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiss</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kunding</span>,  <span class="cit-name-given-names">A.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhella</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baksh</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Connelly</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Droese</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klaus</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sawicki</span>,  <span class="cit-name-given-names">S.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>,  <span class="cit-name-given-names">S.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stamou</span>,  <span class="cit-name-given-names">D.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">I.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Buchmeier</span>,  <span class="cit-name-given-names">M.J.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">A structural analysis of M protein in coronavirus assembly and morphology</span>. <abbr class="cit-jnl-abbrev">J Struct Biol</abbr> <span class="cit-vol">174</span>, <span class="cit-fpage">11</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DNeuman%26rft.auinit1%253DB.%2BW.%26rft.volume%253D174%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D22%26rft.atitle%253DA%2Bstructural%2Banalysis%2Bof%2BM%2Bprotein%2Bin%2Bcoronavirus%2Bassembly%2Band%2Bmorphology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2010.11.021%26rft_id%253Dinfo%253Apmid%252F21130884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2010.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21130884&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.41" data-doi="10.1093/bioinformatics/btv421"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Page</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cummins</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hunt</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">V.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reuter</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holden</span>,  <span class="cit-name-given-names">M.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fookes</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falush</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keane</span>,  <span class="cit-name-given-names">J.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Parkhill</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Roary: rapid large-scale prokaryote pan genome analysis</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">3691</span>–<span class="cit-lpage">3693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv421%26rft_id%253Dinfo%253Apmid%252F26198102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26198102&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.42" data-doi="10.1093/bioinformatics/bti079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pond</span>,  <span class="cit-name-given-names">SL.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>,  <span class="cit-name-given-names">S.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Muse</span>,  <span class="cit-name-given-names">S.V.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">HyPhy: hypothesis testing using phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-vol">21</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti079%26rft_id%253Dinfo%253Apmid%252F15509596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15509596&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227241200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prompetchara</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ketloy</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Palaga</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Immune responses in COVID-19 and potential vaccines: Lessons learned from SARS and MERS epidemic</span>. <abbr class="cit-jnl-abbrev">Asian Pac J Allergy Immunol</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsian%2BPac%2BJ%2BAllergy%2BImmunol%26rft.volume%253D38%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">44.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.44" data-doi="10.4049/jimmunol.0801986"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qin</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elson</span>,  <span class="cit-name-given-names">C.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niyongere</span>,  <span class="cit-name-given-names">S.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reynolds</span>,  <span class="cit-name-given-names">S.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weaver</span>,  <span class="cit-name-given-names">C.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roarty</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Serra</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benveniste</span>,  <span class="cit-name-given-names">E.N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cong</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">TGF-beta promotes Th17 cell development through inhibition of SOCS3</span>. <abbr class="cit-jnl-abbrev">J Immunol</abbr> <span class="cit-vol">183</span>, <span class="cit-fpage">97</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BImmunol%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.0801986%26rft_id%253Dinfo%253Apmid%252F19535626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6ODoiMTgzLzEvOTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI1LjAwNjIxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.45" data-doi="10.1097/CM9.0000000000000722"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>.  <span class="cit-name-given-names">L.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Z.Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">Z.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">Y.Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiong</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">G.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">X.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">Z.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">L.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qian</span>,  <span class="cit-name-given-names">Z.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.W.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Identification of a novel coronavirus causing severe pneumonia in human: a descriptive study</span>. <abbr class="cit-jnl-abbrev">Chin Med J (Engl)</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1097/CM9.0000000000000722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChin%2BMed%2BJ%2B%2528Engl%2529%26rft_id%253Dinfo%253Adoi%252F10.1097%252FCM9.0000000000000722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/CM9.0000000000000722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.46" data-doi="10.3390/v4030363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruch</span>,  <span class="cit-name-given-names">T.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Machamer</span>,  <span class="cit-name-given-names">C.E.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">The coronavirus E protein: assembly and beyond</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">363</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DRuch%26rft.auinit1%253DT.%2BR.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D382%26rft.atitle%253DThe%2Bcoronavirus%2BE%2Bprotein%253A%2Bassembly%2Band%2Bbeyond.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4030363%26rft_id%253Dinfo%253Apmid%252F22590676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4030363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22590676&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sah</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alfonso</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez-Morales Jha</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daniel</span>,  <span class="cit-name-given-names">K.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">H.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiris</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bastola</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lal</span>,  <span class="cit-name-given-names">B.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ojha</span>,  <span class="cit-name-given-names">H.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabaan</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zambrano</span>,  <span class="cit-name-given-names">L.I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Costello</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morita</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pandey</span>,  <span class="cit-name-given-names">B.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">L.L. M.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Complete genome sequence of a 2019 novel coronavirus (SARS-CoV-2) strain isolated in Nepal</span>. <abbr class="cit-jnl-abbrev">Microbiol Res Announce</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e00169</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol%2BRes%2BAnnounce%26rft.volume%253D9%26rft.spage%253De00169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.48" data-doi="10.1093/bioinformatics/btu153"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seemann</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Prokka: rapid prokaryotic genome annotation</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">2068</span>–<span class="cit-lpage">2069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu153%26rft_id%253Dinfo%253Apmid%252F24642063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24642063&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339814300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.49" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shannon</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markiel</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozier</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baliga</span>,  <span class="cit-name-given-names">N.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramage</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amin</span>,  <span class="cit-name-given-names">N.</span></span>, et al. (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yNS4wMDYyMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.50" data-doi="10.1016/bs.aivir.2016.08.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">The non-structural proteins directing coronavirus RNA synthesis and processing</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">96</span>, <span class="cit-fpage">59</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D96%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2016.08.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2016.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.51" data-doi="10.1093/database/baw162"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stano</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beke</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Klucar</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">viruSITE—integrated database for viral genomics</span>. <abbr class="cit-jnl-abbrev">Database 2, article ID baw162</abbr>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/database/baw162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDatabase%2B2%252C%2Barticle%2BID%2Bbaw162%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbaw162%26rft_id%253Dinfo%253Apmid%252F28025349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/database/baw162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28025349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.52" data-doi="10.1093/nar/gku1003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franceschini</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wyder</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forslund</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heller</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simonovic</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">STRING v10: protein-protein interaction networks, integrated over the tree of life</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>, <span class="cit-fpage">D447</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1003%26rft_id%253Dinfo%253Apmid%252F25352553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25352553&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.53" data-doi="10.1093/oxfordjournals.molbev.a040023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tamura</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nei</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1993</span>). <span class="cit-article-title">Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">512</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DTamura%26rft.auinit1%253DK%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D512%26rft.epage%253D526%26rft.atitle%253DEstimation%2Bof%2Bthe%2Bnumber%2Bof%2Bnucleotide%2Bsubstitutions%2Bin%2Bthe%2Bcontrol%2Bregion%2Bof%2Bmitochondrial%2BDNA%2Bin%2Bhumans%2Band%2Bchimpanzees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a040023%26rft_id%253Dinfo%253Apmid%252F8336541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a040023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8336541&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LD11400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">X.Q.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Inferring the hosts of coronavirus using dual statistical models based on nucleotide composition</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">17155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D5%26rft.spage%253D17155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.55" data-doi="10.1128/JVI.00090-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">Y.X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Timothy</span>,  <span class="cit-name-given-names">H.P.</span></span> <span class="cit-auth"><span class="cit-name-surname">Tan Marvin</span>,  <span class="cit-name-given-names">J.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee Tham</span>,  <span class="cit-name-given-names">P.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunalan</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Druce</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birch</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Catton</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">N.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">V.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">Y.J.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Induction of apoptosis by the severe acute respiratory syndrome coronavirus 7a protein is dependent on its interaction with the Bcl-XL protein</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">6346</span>–<span class="cit-lpage">6355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00090-07%26rft_id%253Dinfo%253Apmid%252F17428862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MS8xMi82MzQ2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yNS4wMDYyMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.56" data-doi="10.1091/mbc.E04-09-0807"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tatsuta</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Model</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Langer</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Formation of membrane-bound ring complexes by prohibitins in mitochondria</span>. <abbr class="cit-jnl-abbrev">Mol Biol Cell</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">248</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E04-09-0807%26rft_id%253Dinfo%253Apmid%252F15525670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE2LzEvMjQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yNS4wMDYyMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.57" data-doi="10.1186/s13059-014-0524-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Treangen</span>,  <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ondov</span>,  <span class="cit-name-given-names">B.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koren</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Phillippy</span>,  <span class="cit-name-given-names">A.M.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">The Harvest suite for rapid core-genome alignment and visualization of thousands of intraspecific microbial genomes</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0524-x%26rft_id%253Dinfo%253Apmid%252F25410596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0524-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25410596&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tyrrell</span>,  <span class="cit-name-given-names">D.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bynoe</span>,  <span class="cit-name-given-names">M.L.</span></span> (<span class="cit-pub-date">1966</span>). <span class="cit-article-title">Cultivation of viruses from a high proportion of patients with colds</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">76</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DTyrrell%26rft.auinit1%253DD.%2BA.%26rft.volume%253D1%26rft.issue%253D7428%26rft.spage%253D76%26rft.epage%253D77%26rft.atitle%253DCultivation%2Bof%2Bviruses%2Bfrom%2Ba%2Bhigh%2Bproportion%2Bof%2Bpatients%2Bwith%2Bcolds.%26rft_id%253Dinfo%253Apmid%252F4158999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4158999&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19667156900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.59" data-doi="10.1093/nar/gkm895"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">UniProt Consortium</span>. (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">The universal protein resource (UniProt)</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">36</span>, <span class="cit-fpage">D190</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D36%26rft.spage%253DD190%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm895%26rft_id%253Dinfo%253Apmid%252F18045787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18045787&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252545400035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.60" data-doi="10.1016/S0092-8674(03)00424-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">von Grotthuss</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wyrwicz</span>,  <span class="cit-name-given-names">L.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rychlewski</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">mRNA cap-1 methyltransferase in the SARS genome</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">113</span>, <span class="cit-fpage">701</span>–<span class="cit-lpage">702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253Dvon%2BGrotthuss%26rft.auinit1%253DM.%26rft.volume%253D113%26rft.issue%253D6%26rft.spage%253D701%26rft.epage%253D702%26rft.atitle%253DmRNA%2Bcap-1%2Bmethyltransferase%2Bin%2Bthe%2BSARS%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252803%252900424-0%26rft_id%253Dinfo%253Apmid%252F12809601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(03)00424-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12809601&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183514400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wall</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wall</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGuire</span>,  <span class="cit-name-given-names">A.T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Vessler</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Structure, function, and antigenicity of the SARS-CoV-2 spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">180</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D180%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.62" data-doi="10.1016/j.cell.2020.03.045"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">G.</span></span>,<span class="cit-auth"><span class="cit-name-surname">Qiao</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Structural and functional basis of SARS-CoV-2 entry by using human ACE2</span>. <abbr class="cit-jnl-abbrev">Cell</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2020.03.045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.03.045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.03.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weaver</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shank</span>,  <span class="cit-name-given-names">S.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spielman</span>,  <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muse</span>,  <span class="cit-name-given-names">S.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>  <span class="cit-name-given-names">S.L.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Datamonkey 2.0: A Modern Web Application for Characterizing Selective and Other Evolutionary Processes</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>. <span class="cit-vol">35</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D35%26rft.spage%253D773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.64" data-doi="10.1128/JVI.06540-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">P.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">S.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">C.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsang</span>,  <span class="cit-name-given-names">A.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">J.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bai</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teng</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsang</span>,  <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">K.H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.Y.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Discovery of seven novel Mammalian and avian coronaviruses in the genus deltacoronavirus supports bat coronaviruses as the gene source of alphacoronavirus and betacoronavirus and avian coronaviruses as the gene source of gammacoronavirus and deltacoronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">86</span>, <span class="cit-fpage">3995</span>–<span class="cit-lpage">4008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06540-11%26rft_id%253Dinfo%253Apmid%252F22278237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzcvMzk5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMwLzIwMjAuMDMuMjUuMDA2MjEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Z.Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sheng</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Genome composition and divergence of the novel coronavirus (2019-nCoV) originating in China</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">325</span>–<span class="cit-lpage">328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D27%26rft.spage%253D325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.66" data-doi="10.1074/jbc.M708033200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholls</span>,  <span class="cit-name-given-names">J.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Sars-cov nucleocapsid protein interacts with smad3 and modulates TGF-β signaling</span>. <abbr class="cit-jnl-abbrev">J Biol Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">3272</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M708033200%26rft_id%253Dinfo%253Apmid%252F18055455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODMvNi8zMjcyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yNS4wMDYyMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">67.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.67" data-doi="10.1016/S2213-2600(20)30076-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tai</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bai</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">F.S.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Pathological findings of COVID-19 associated with acute respiratory distress syndrome</span>. <abbr class="cit-jnl-abbrev">Lancet Respir Med</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S2213-2600(20)30076-X</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BRespir%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS2213-2600%252820%252930076-X%26rft_id%253Dinfo%253Apmid%252F32085846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S2213-2600(20)30076-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32085846&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.68" data-doi="10.1016/j.virusres.2005.01.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yao</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chong</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">subcellular localization and membrane association of SARS-CoV 3a protein</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">109</span>, <span class="cit-fpage">191</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DYuan%26rft.auinit1%253DX.%26rft.volume%253D109%26rft.issue%253D2%26rft.spage%253D191%26rft.epage%253D202%26rft.atitle%253DSubcellular%2Blocalization%2Band%2Bmembrane%2Bassociation%2Bof%2BSARS-CoV%2B3a%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2005.01.001%26rft_id%253Dinfo%253Apmid%252F15763150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2005.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15763150&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.25.006213.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228095800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">69.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">The role of transforming growth factor <em>β</em> in T helper 17 differentiation</span>. <abbr class="cit-jnl-abbrev">Immunology</abbr>. <span class="cit-vol">155</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%26rft.volume%253D155%26rft.spage%253D24%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>.  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">E.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">X.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2020</span>). <abbr class="cit-jnl-abbrev">Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1</abbr>.<span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv:</span><span class="cit-pub-id cit-pub-id-arxiv">2002.03173</span>[q-bio.GN].</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">71.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.25.006213v1.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholls</span>,  <span class="cit-name-given-names">J.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.G.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Sars-cov nucleocapsid protein interacts with smad3 and modulates TGF-β signaling</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">3272</span>–<span class="cit-lpage">3280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft.volume%253D8%26rft.spage%253D3272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.25.006213v1.72" data-doi="10.1101/gr.232397.117"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alikhan</span>,  <span class="cit-name-given-names">N.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sergeant</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luhmann</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaz</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Francisco</span>,  <span class="cit-name-given-names">A.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrico</span>,  <span class="cit-name-given-names">J.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Achtman</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">GrapeTree: visualization of core genomic relationships among 100,000 bacterial pathogens</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">1395</span>–<span class="cit-lpage">1404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.232397.117%26rft_id%253Dinfo%253Apmid%252F30049790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI4LzkvMTM5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMwLzIwMjAuMDMuMjUuMDA2MjEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.25.006213v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.25.006213v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.27.012633v1" title="SCITO-seq: single-cell combinatorial indexed cytometry sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.03.974220v2" title="Improving replicability in single-cell RNA-Seq cell type discovery with Dune" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 30, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.25.006213v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.25.006213v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.25.006213v1" class="active" data-icon-position="" data-hide-link-title="0">Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1217854" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-RMy5hhz4TstlCGtYrC5g1TWPbILswkxPAy5bODMQFCs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1217854" id="node1217854--2" data-pisa="biorxiv;2020.03.25.006213v1" data-pisa-master="biorxiv;2020.03.25.006213" data-seqnum="1217854" data-apath="/biorxiv/early/2020/03/30/2020.03.25.006213.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Roshan</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Helianthous</span> <span class="nlm-surname">Verma</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Nirjara</span> <span class="nlm-surname">Singhvi</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Utkarsh</span> <span class="nlm-surname">Sood</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Vipin</span> <span class="nlm-surname">Gupta</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mona</span> <span class="nlm-surname">Singh</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rashmi</span> <span class="nlm-surname">Kumari</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Princy</span> <span class="nlm-surname">Hira</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Shekhar</span> <span class="nlm-surname">Nagar</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Chandni</span> <span class="nlm-surname">Talwar</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Namita</span> <span class="nlm-surname">Nayyar</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Shailly</span> <span class="nlm-surname">Anand</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Charu Dogra</span> <span class="nlm-surname">Rawat</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Mansi</span> <span class="nlm-surname">Verma</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Ram Krishan</span> <span class="nlm-surname">Negi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Yogendra</span> <span class="nlm-surname">Singh</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Rup</span> <span class="nlm-surname">Lal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.25.006213; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.25.006213 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.25.006213v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;title=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;title=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;text=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;title=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;t=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;title=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;title=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1217854" id="node1217854--3" data-pisa="biorxiv;2020.03.25.006213v1" data-pisa-master="biorxiv;2020.03.25.006213" data-seqnum="1217854" data-apath="/biorxiv/early/2020/03/30/2020.03.25.006213.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Comparative Genomic Analysis of Rapidly Evolving SARS CoV-2 Viruses Reveal Mosaic Pattern of Phylogeographical Distribution</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Roshan</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Helianthous</span> <span class="nlm-surname">Verma</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Nirjara</span> <span class="nlm-surname">Singhvi</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Utkarsh</span> <span class="nlm-surname">Sood</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Vipin</span> <span class="nlm-surname">Gupta</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mona</span> <span class="nlm-surname">Singh</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Rashmi</span> <span class="nlm-surname">Kumari</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Princy</span> <span class="nlm-surname">Hira</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Shekhar</span> <span class="nlm-surname">Nagar</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Chandni</span> <span class="nlm-surname">Talwar</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Namita</span> <span class="nlm-surname">Nayyar</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Shailly</span> <span class="nlm-surname">Anand</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Charu Dogra</span> <span class="nlm-surname">Rawat</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Mansi</span> <span class="nlm-surname">Verma</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Ram Krishan</span> <span class="nlm-surname">Negi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Yogendra</span> <span class="nlm-surname">Singh</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Rup</span> <span class="nlm-surname">Lal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.25.006213; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.25.006213 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Kumar%2C+Roshan&amp;rft.au=Verma%2C+Helianthous&amp;rft.au=Singhvi%2C+Nirjara&amp;rft.au=Sood%2C+Utkarsh&amp;rft.au=Gupta%2C+Vipin&amp;rft.au=Singh%2C+Mona&amp;rft.au=Kumari%2C+Rashmi&amp;rft.au=Hira%2C+Princy&amp;rft.au=Nagar%2C+Shekhar&amp;rft.au=Talwar%2C+Chandni&amp;rft.au=Nayyar%2C+Namita&amp;rft.au=Anand%2C+Shailly&amp;rft.au=Rawat%2C+Charu+Dogra&amp;rft.au=Verma%2C+Mansi&amp;rft.au=Negi%2C+Ram+Krishan&amp;rft.au=Singh%2C+Yogendra&amp;rft.au=Lal%2C+Rup"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1217854/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1217854/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1217854/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1217854/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1217854/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1217854/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1217854/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1217854/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1217854/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1217854/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1217854/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1217854/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;count=horizontal&amp;via=&amp;text=Comparative%20Genomic%20Analysis%20of%20Rapidly%20Evolving%20SARS%20CoV-2%20Viruses%20Reveal%20Mosaic%20Pattern%20of%20Phylogeographical%20Distribution&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.25.006213v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.25.006213v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
