<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/15/2020.02.11.944462.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.11.944462v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.11.944462"/>
  <meta name="DC.Date" content="2020-02-15"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 Å-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis."/>
  <meta name="DC.Contributor" content="Daniel Wrapp"/>
  <meta name="DC.Contributor" content="Nianshuang Wang"/>
  <meta name="DC.Contributor" content="Kizzmekia S. Corbett"/>
  <meta name="DC.Contributor" content="Jory A. Goldsmith"/>
  <meta name="DC.Contributor" content="Ching-Lin Hsieh"/>
  <meta name="DC.Contributor" content="Olubukola Abiona"/>
  <meta name="DC.Contributor" content="Barney S. Graham"/>
  <meta name="DC.Contributor" content="Jason S. McLellan"/>
  <meta name="article:published_time" content="2020-02-15"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;The outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 Å-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.11.944462v1"/>
  <meta name="citation_id" content="2020.02.11.944462v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.11.944462v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.11.944462v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.11.944462v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.11.944462"/>
  <meta name="citation_num_pages" content="30"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.11.944462"/>
  <meta name="citation_author" content="Daniel Wrapp"/>
  <meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin"/>
  <meta name="citation_author" content="Nianshuang Wang"/>
  <meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin"/>
  <meta name="citation_author" content="Kizzmekia S. Corbett"/>
  <meta name="citation_author_institution" content="Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health"/>
  <meta name="citation_author" content="Jory A. Goldsmith"/>
  <meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin"/>
  <meta name="citation_author" content="Ching-Lin Hsieh"/>
  <meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin"/>
  <meta name="citation_author" content="Olubukola Abiona"/>
  <meta name="citation_author_institution" content="Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health"/>
  <meta name="citation_author" content="Barney S. Graham"/>
  <meta name="citation_author_institution" content="Vaccine Research Center, National Institute of Allergy and Infectious Diseases, National Institutes of Health"/>
  <meta name="citation_author" content="Jason S. McLellan"/>
  <meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin"/>
  <meta name="citation_author_email" content="jmclellan@austin.utexas.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3991-542X"/>
  <meta name="citation_reference" content="J. F. Chan et al., A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. Lancet, (2020)."/>
  <meta name="citation_reference" content="C. Huang et al., Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet, (2020)."/>
  <meta name="citation_reference" content="R. Lu et al., Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet, (2020)."/>
  <meta name="citation_reference" content="F. Wu et al., A new coronavirus associated with human respiratory disease in China. Nature, (2020)."/>
  <meta name="citation_reference" content="N. Chen et al., Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. Lancet, (2020)."/>
  <meta name="citation_reference" content="Q. Li et al., Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia. N Engl J Med, (2020)."/>
  <meta name="citation_reference" content="F. Li, Structure, Function, and Evolution of Coronavirus Spike Proteins. Annu Rev Virol 3, 237–261 (2016)."/>
  <meta name="citation_reference" content="B. J. Bosch, R. van der Zee, C. A. de Haan, P. J. Rottier, The coronavirus spike protein is a class I virus fusion protein: structural and functional characterization of the fusion core complex. J Virol 77, 8801–8811 (2003)."/>
  <meta name="citation_reference" content="A. C. Walls et al., Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion. Proc Natl Acad Sci U S A 114, 11157–11162 (2017)."/>
  <meta name="citation_reference" content="M. Gui et al., Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell Res 27, 119–129 (2017)."/>
  <meta name="citation_reference" content="J. Pallesen et al., Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen. Proc Natl Acad Sci U S A 114, E7348–E7357 (2017)."/>
  <meta name="citation_reference" content="A. C. Walls et al., Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion. Cell 176, 1026–1039 e1015 (2019)."/>
  <meta name="citation_reference" content="Y. Yuan et al., Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nat Commun 8, 15092 (2017)."/>
  <meta name="citation_reference" content="R. N. Kirchdoerfer et al., Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis. Sci Rep 8, 15701 (2018)."/>
  <meta name="citation_reference" content="A. Punjani, J. L. Rubinstein, D. J. Fleet, M. A. Brubaker, cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Nat Methods 14, 290–296 (2017)."/>
  <meta name="citation_reference" content="D. Wrapp, J. S. McLellan, The 3.1-Angstrom Cryo-electron Microscopy Structure of the Porcine Epidemic Diarrhea Virus Spike Protein in the Prefusion Conformation. J Virol 93, (2019)."/>
  <meta name="citation_reference" content="A. C. Walls et al., Glycan shield and epitope masking of a coronavirus spike protein observed by cryo-electron microscopy. Nat Struct Mol Biol 23, 899–905 (2016)."/>
  <meta name="citation_reference" content="R. N. Kirchdoerfer et al., Pre-fusion structure of a human coronavirus spike protein. Nature 531, 118–121 (2016)."/>
  <meta name="citation_reference" content="B. J. Bosch, W. Bartelink, P. J. Rottier, Cathepsin L functionally cleaves the severe acute respiratory syndrome coronavirus class I fusion protein upstream of rather than adjacent to the fusion peptide. J Virol 82, 8887–8890 (2008)."/>
  <meta name="citation_reference" content="I. Glowacka et al., Evidence that TMPRSS2 activates the severe acute respiratory syndrome coronavirus spike protein for membrane fusion and reduces viral control by the humoral immune response. J Virol 85, 4122–4134 (2011)."/>
  <meta name="citation_reference" content="W. Li et al., Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454 (2003)."/>
  <meta name="citation_reference" content="S. Belouzard, V. C. Chu, G. R. Whittaker, Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites. Proc Natl Acad Sci U S A 106, 5871–5876 (2009)."/>
  <meta name="citation_reference" content="J. Chen et al., Structure of the hemagglutinin precursor cleavage site, a determinant of influenza pathogenicity and the origin of the labile conformation. Cell 95, 409–417 (1998)."/>
  <meta name="citation_reference" content="M. K.-W. Hoffmann H.; Krüger, N.; Müller, M.; Drosten, C.; Pöhlmann, S., The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells. BioRXiv, (2020)."/>
  <meta name="citation_reference" content="Y. Wan, J. Shang, R. Graham, R. S. Baric, F. Li, Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol, (2020)."/>
  <meta name="citation_reference" content="P. Zhou et al., A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, (2020)."/>
  <meta name="citation_reference" content="W. Song, M. Gui, X. Wang, Y. Xiang, Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS Pathog 14, e1007236 (2018)."/>
  <meta name="citation_reference" content="W. C. Hwang et al., Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R. J Biol Chem 281, 34610–34616 (2006)."/>
  <meta name="citation_reference" content="P. Prabakaran et al., Structure of severe acute respiratory syndrome coronavirus receptor-binding domain complexed with neutralizing antibody. J Biol Chem 281, 15829–15836 (2006)."/>
  <meta name="citation_reference" content="X. L. Tian C.; Huang, A.; Xia, S.; Lu, S.; Shi, Z.; Lu, L.; Jiang, S.; Yang, Z.; Wu, Y.; Ying, T., Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. BioRXiv, (2020)."/>
  <meta name="citation_reference" content="B. Carragher et al., Leginon: an automated system for acquisition of images from vitreous ice specimens. J Struct Biol 132, 33–45 (2000)."/>
  <meta name="citation_reference" content="D. Tegunov, P. Cramer, Real-time cryo-electron microscopy data preprocessing with Warp. Nat Methods 16, 1146–1152 (2019)."/>
  <meta name="citation_reference" content="E. Ramirez-Aportela et al., Automatic local resolution-based sharpening of cryo-EM maps. Bioinformatics, (2019)."/>
  <meta name="citation_reference" content="P. D. Adams et al., PHENIX: building new software for automated crystallographic structure determination. Acta Crystallogr D Biol Crystallogr 58, 1948–1954 (2002)."/>
  <meta name="citation_reference" content="T. I. Croll, ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps. Acta Crystallogr D Struct Biol 74, 519–530 (2018)."/>
  <meta name="citation_reference" content="P. Emsley, K. Cowtan, Coot: model-building tools for molecular graphics. Acta Crystallogr D Biol Crystallogr 60, 2126–2132 (2004)."/>
  <meta name="citation_reference" content="A. Morin et al., Collaboration gets the most out of software. Elife 2, e01456 (2013)."/>
  <meta name="citation_reference" content="T. Grant, A. Rohou, N. Grigorieff, cisTEM, user-friendly software for single-particle image processing. Elife 7, (2018)."/>
  <meta name="twitter:title" content="Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 Å-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis."/>
  <meta name="og-title" property="og:title" content="Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.11.944462v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 Å-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-15"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Cryo-EM Structure of the 2019-nCoV Spike in the Prefusion Conformation | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1153112 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">The outbreak of a novel betacoronavirus (2019-nCov) represents a pandemic threat that has been declared a public health emergency of international concern. The CoV spike (S) glycoprotein is a key target for urgently needed vaccines, therapeutic antibodies, and diagnostics. To facilitate medical countermeasure (MCM) development we determined a 3.5 Å-resolution cryo-EM structure of the 2019-nCoV S trimer in the prefusion conformation. The predominant state of the trimer has one of the three receptor-binding domains (RBDs) rotated up in a receptor-accessible conformation. We also show biophysical and structural evidence that the 2019-nCoV S binds ACE2 with higher affinity than SARS-CoV S. Additionally we tested several published SARS-CoV RBD-specific monoclonal antibodies and found that they do not have appreciable binding to nCoV-2019 S, suggesting antibody cross-reactivity may be limited between the two virus RBDs. The atomic-resolution structure of 2019-nCoV S should enable rapid development and evaluation of MCMs to address the ongoing public health crisis.</p>
   </div>
   <div id="sec-1" class="subsection">
    <p id="p-4">The novel coronavirus 2019-nCoV has recently emerged as a human pathogen in the city of Wuhan in China’s Hubei province, causing fever, severe respiratory illness and pneumonia (
     <em>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, 
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </em>). According to the World Health Organization on February 10
     <sup>th</sup>, 2020, there have been over 40,000 confirmed cases globally, leading to at least 900 deaths. The new pathogen was rapidly shown to be a novel member of the betacoronavirus genus that is closely related to several bat coronaviruses as well as severe acute respiratory syndrome coronavirus (SARS-CoV) (
     <em>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, 
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </em>). Compared to SARS-CoV, 2019-nCoV appears to be more readily transmitted from human-to-human, spreading to multiple continents and leading to the WHO declaration of a Public Health Emergency of International Concern (PHEIC) (
     <em>
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, 
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, 
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </em>).
    </p>
    <p id="p-5">2019-nCoV makes use of a densely glycosylated, homotrimeric class I fusion spike (S) protein to gain entry into host cells. The S protein exists in a metastable prefusion conformation that undergoes a dramatic structural rearrangement to fuse the viral membrane with the host cell membrane (
     <em>
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, 
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </em>). This process is triggered by binding of the S1 subunit to a host-cell receptor, which destabilizes the prefusion trimer, resulting in shedding of the S1 subunit and transition of the S2 subunit to a highly stable postfusion conformation (
     <em>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
     </em>). In order to engage a host-cell receptor, the receptor-binding domain (RBD) of S1 undergoes hinge-like conformational movements that transiently hide or expose the determinants of receptor binding. These two states are referred to as the “down” conformation and the “up” conformation, where “down” corresponds to the receptor-inaccessible state and “up” corresponds to the receptor-accessible state, which is thought to be less stable (
     <em>
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </em>). Due to the indispensable function of the S protein it represents a vulnerable target for antibody-mediated neutralization, and characterization of the prefusion S structure would provide atomic-level information to guide vaccine design and development.
    </p>
    <p id="p-6">Based on the reported genome sequence of 2019-nCoV(
     <em>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </em>), we expressed ectodomain residues 1−1208 of 2019-nCoV S (
     <strong>
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, 
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Supplementary Figure 1</a>
     </strong>), adding two stabilizing proline mutations in the C-terminal S2 fusion machinery based on a previous stabilization strategy which proved highly effective for betacoronavirus S proteins (
     <em>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, 
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </em>). We obtained roughly 0.5 mg/L of the recombinant prefusion-stabilized S ectodomain from FreeStyle 293 cells, and the protein was purified to homogeneity by affinity chromatography and size-exclusion chromatography (
     <strong>
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Supplementary Figure 1</a>
     </strong>). Cryo-EM grids were prepared using this purified, fully glycosylated S protein and preliminary screening revealed a high particle density with little aggregation near the edges of the holes.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structure of 2019-nCoV S in the prefusion conformation. (A) Schematic of 2019-nCoV S primary structure, colored by domain. Domains that were excluded from the ectodomain expression construct or could not be visualized in the final map are colored white. SS= signal sequence, NTD= N-terminal domain, RBD= receptor-binding domain, SD1= subdomain 1, SD2= subdomain 2, S1/S2= S1/S2 protease cleavage site, S2[[#x2032]]= S2[[#x2032]] protease cleavage site, FP= fusion peptide, HR1= heptad repeat 1, CH= central helix, CD= connector domain, HR2= heptad repeat 2, TM= transmembrane domain, CT= cytoplasmic tail. Arrows denote protease cleavage sites. (B) Select 2D class averages of the particles that were used to calculate the 2019-nCoV S reconstruction (left). Side and top views of the prefusion structure of the 2019-nCoV S protein with a single RBD in the [[#x201C]]up[[#x201D]] conformation (right). The two RBD [[#x201C]]down[[#x201D]] protomers are shown as cryo-EM density in either white or gray and the RBD [[#x201C]]up[[#x201D]] protomer is shown in ribbons, colored corresponding to the schematic in Fig 1A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Structure of 2019-nCoV S in the prefusion conformation.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Schematic of 2019-nCoV S primary structure, colored by domain. Domains that were excluded from the ectodomain expression construct or could not be visualized in the final map are colored white. SS= signal sequence, NTD= N-terminal domain, RBD= receptor-binding domain, SD1= subdomain 1, SD2= subdomain 2, S1/S2= S1/S2 protease cleavage site, S2[[#x2032]]= S2[[#x2032]] protease cleavage site, FP= fusion peptide, HR1= heptad repeat 1, CH= central helix, CD= connector domain, HR2= heptad repeat 2, TM= transmembrane domain, CT= cytoplasmic tail. Arrows denote protease cleavage sites. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Select 2D class averages of the particles that were used to calculate the 2019-nCoV S reconstruction (&lt;;em&gt;;left&lt;;/em&gt;;). Side and top views of the prefusion structure of the 2019-nCoV S protein with a single RBD in the [[#x201C]]up[[#x201D]] conformation (&lt;;em&gt;;right&lt;;/em&gt;;). The two RBD [[#x201C]]down[[#x201D]] protomers are shown as cryo-EM density in either white or gray and the RBD [[#x201C]]up[[#x201D]] protomer is shown in ribbons, colored corresponding to the schematic in &lt;;strong&gt;;Fig 1A&lt;;/strong&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F1.medium.gif" width="440" height="320"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F1.medium.gif" width="440" height="320"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <span class="caption-title">Structure of 2019-nCoV S in the prefusion conformation.</span>
      <p id="p-7" class="first-child">(
       <strong>A</strong>) Schematic of 2019-nCoV S primary structure, colored by domain. Domains that were excluded from the ectodomain expression construct or could not be visualized in the final map are colored white. SS= signal sequence, NTD= N-terminal domain, RBD= receptor-binding domain, SD1= subdomain 1, SD2= subdomain 2, S1/S2= S1/S2 protease cleavage site, S2′= S2′ protease cleavage site, FP= fusion peptide, HR1= heptad repeat 1, CH= central helix, CD= connector domain, HR2= heptad repeat 2, TM= transmembrane domain, CT= cytoplasmic tail. Arrows denote protease cleavage sites. (
       <strong>B</strong>) Select 2D class averages of the particles that were used to calculate the 2019-nCoV S reconstruction (
       <em>left</em>). Side and top views of the prefusion structure of the 2019-nCoV S protein with a single RBD in the “up” conformation (
       <em>right</em>). The two RBD “down” protomers are shown as cryo-EM density in either white or gray and the RBD “up” protomer is shown in ribbons, colored corresponding to the schematic in 
       <strong>
        <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1A</a>
       </strong>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-8">After collecting and processing 3,207 micrograph movies, we obtained a 3.5 Å-resolution 3D reconstruction of an asymmetrical trimer in which a single RBD was observed in the “up” conformation. (
     <strong>
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>, 
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Supplementary Figure 2</a>
     </strong>). Due to the small size of the RBD (∼21 kDa), the asymmetry of this conformation was not readily apparent until 
     <em>ab initio</em> 3D reconstruction and 3D classification were performed (
     <strong>
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>, 
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplementary Figure 3</a>
     </strong>). By using the 3D variability feature in cryoSPARC v2 (
     <em>
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
     </em>), we were able to observe breathing of the S1 subunits as the RBD underwent a hinge-like movement, which likely contributed to the relatively poor local resolution of S1 compared to the more stable S2 subunit (
     <strong>Supplementary Movies 1 and 2</strong>). This seemingly stochastic RBD movement has been captured during structural characterization of the closely related betacoronaviruses SARS-CoV and MERS-CoV, as well as the more distantly related alphacoronavirus porcine epidemic diarrhea virus (PEDV)(
     <em>
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, 
      <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, 
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, 
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
     </em>). The observation of this phenomenon in 2019-nCoV S suggests that it shares the same mechanism of triggering that is thought to be conserved among the 
     <em>Coronaviridae</em>, wherein receptor-binding to exposed RBDs leads to an unstable 3 RBD-up conformation that results in shedding of S1 and refolding of S2 (
     <em>
      <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, 
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </em>).
    </p>
    <p id="p-9">Because the S2 subunit appeared to be a symmetric trimer, we performed a 3D refinement imposing C3 symmetry, resulting in a 3.2 Å-resolution map, with excellent density for the S2 subunit. Using both maps we built the vast majority of the 2019-nCoV S ectodomain, including glycans at 44 of the 66 
     <em>N</em>-linked glycosylation sites per trimer (
     <strong>
      <a id="xref-fig-8-1" class="xref-fig" href="#F8">Supplementary Figure 4</a>
     </strong>). Our final model spans S residues 27–1146, with several flexible loops omitted. Like all previously reported coronavirus S ectodomain structures, the density for 2019-nCoV S begins to fade after the connector domain (CD), reflecting the flexibility of the heptad repeat 2 (HR2) domain in the prefusion conformation (
     <strong>
      <a id="xref-fig-8-2" class="xref-fig" href="#F8">Supplementary Figure 4A</a>
     </strong>) (
     <em>
      <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>, 
      <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>-
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
     </em>).
    </p>
    <p id="p-10">The overall structure of 2019-nCoV S resembles that of SARS-CoV S, with a root mean square deviation (RMSD) of 3.8 Å over 959 Cα atoms. The largest discrepancy between these two structures is a conformational difference between the positions of the RBDs in their respective “down” conformations (
     <strong>
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>
     </strong>). Whereas the SARS-CoV RBD in the “down” conformation packs tightly against the N-terminal domain (NTD) of the neighboring protomer, the 2019-nCoV RBD in the “down” conformation is angled closer to the central cavity of the homotrimer. Despite this observed conformational difference, when the individual structural domains of 2019-nCoV S are aligned to their counterparts from SARS-CoV S, they reflect the high degree of structural homology between the two proteins, with the NTDs, RBDs, subdomains 1 and 2 (SD1 and SD2) and S2 subunits yielding RMSD values of 2.6 Å, 3.0 Å, 2.7 Å and 2.0 Å, respectively (
     <strong>
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>
     </strong>).
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structural comparison between 2019-nCoV S and SARS-CoV S. (A) A single RBD [[#x201C]]down[[#x201D]] monomer of 2019-nCoV S is shown in ribbons, colored according to Figure 1. A monomer of SARS-CoV S is also shown in ribbons, colored white (PDB ID: 6CRZ). (B) The following structural domains from 2019-nCoV S have been aligned to their counterparts from SARS-CoV S; NTD (top left), RBD (top right), SD1 and SD2, (bottom left) and S2 (bottom right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Structural comparison between 2019-nCoV S and SARS-CoV S.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) A single RBD [[#x201C]]down[[#x201D]] monomer of 2019-nCoV S is shown in ribbons, colored according to &lt;;strong&gt;;Figure 1&lt;;/strong&gt;;. A monomer of SARS-CoV S is also shown in ribbons, colored white (PDB ID: 6CRZ). (&lt;;strong&gt;;B&lt;;/strong&gt;;) The following structural domains from 2019-nCoV S have been aligned to their counterparts from SARS-CoV S; NTD (&lt;;em&gt;;top left&lt;;/em&gt;;), RBD (&lt;;em&gt;;top right&lt;;/em&gt;;), SD1 and SD2, (&lt;;em&gt;;bottom left&lt;;/em&gt;;) and S2 (&lt;;em&gt;;bottom right&lt;;/em&gt;;).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F2.medium.gif" width="440" height="188"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F2.medium.gif" width="440" height="188"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <span class="caption-title">Structural comparison between 2019-nCoV S and SARS-CoV S.</span>
      <p id="p-11" class="first-child">(
       <strong>A</strong>) A single RBD “down” monomer of 2019-nCoV S is shown in ribbons, colored according to 
       <strong>
        <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>
       </strong>. A monomer of SARS-CoV S is also shown in ribbons, colored white (PDB ID: 6CRZ). (
       <strong>B</strong>) The following structural domains from 2019-nCoV S have been aligned to their counterparts from SARS-CoV S; NTD (
       <em>top left</em>), RBD (
       <em>top right</em>), SD1 and SD2, (
       <em>bottom left</em>) and S2 (
       <em>bottom right</em>).
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-12">2019-nCoV S shares roughly 96% sequence identity with the S protein from the bat coronavirus RaTG13, with the most notable variation arising from an insertion in the S1/S2 protease cleavage site that results in an “RRAR” furin recognition site in 2019-nCoV, rather than the single arginine in SARS-CoV (
     <strong>
      <a id="xref-fig-9-1" class="xref-fig" href="#F9">Supplementary Figure 5</a>
     </strong>) (
     <em>
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>-
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
     </em>). A similar phenomenon has been observed for influenza viruses, where amino acid insertions that create a polybasic furin site in a related position in influenza hemagglutinin proteins are often found in highly virulent avian and human influenza viruses (
     <em>
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>
     </em>). In addition to this insertion of residues in the S1/S2 junction, 29 variant residues exist between 2019-nCoV S and RaTG13 S, with 17 of these positions mapping to the RBD (
     <strong>
      <a id="xref-fig-9-2" class="xref-fig" href="#F9">Supplementary Figures 5</a> and 
      <a id="xref-fig-10-1" class="xref-fig" href="#F10">6</a>
     </strong>). We also analyzed the 61 available 2019-nCoV S sequences in GISAID and found that there were only 9 amino acid substitutions among all deposited sequences. Most of these substitutions are relatively conservative and they are not expected to have a dramatic effect on the structure or function of the 2019-nCoV S protein (
     <strong>
      <a id="xref-fig-10-2" class="xref-fig" href="#F10">Supplementary Figure 6</a>
     </strong>).
    </p>
    <p id="p-13">Recent reports demonstrating that 2019-nCoV S and SARS-CoV S share the same functional host-cell receptor—angiotensin-converting enzyme 2 (ACE2) (
     <em>
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, 
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>-
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>
     </em>)—prompted us to quantify the kinetics mediating this interaction via surface plasmon resonance (SPR). Surprisingly, ACE2 bound to 2019-nCoV S ectodomain with ∼15 nM affinity, which is approximately 10- to 20-fold higher affinity than ACE2 binding to SARS-CoV S (
     <strong>
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, 
      <a id="xref-fig-11-1" class="xref-fig" href="#F11">Supplementary Figure 7</a>
     </strong>) (
     <em>
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>
     </em>). We also formed a complex of ACE2 bound to the 2019-nCoV S ectodomain and observed it by negative-stain EM, where it strongly resembled the complex formed between SARS-CoV S and ACE2, which has been observed at high-resolution by cryo-EM (
     <strong>
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>
     </strong>) (
     <em>
      <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>, 
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>
     </em>). The high affinity of 2019-nCoV S for human ACE2 may contribute to the apparent ease with which 2019-nCoV can spread from human-to-human(
     <em>
      <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>
     </em>), however additional studies are needed to investigate this possibility.
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="2019-nCoV S binds human ACE2 with high affinity. (A) SPR sensorgram showing the binding kinetics for human ACE2 and immobilized 2019-nCoV S. Data are shown as black lines and the best fit of the data to a 1:1 binding model is shown in red. (B) Negative-stain EM 2D class averages of 2019-nCoV S bound by ACE2. Averages have been rotated so that ACE2 is positioned above the 2019-nCoV S protein with respect to the viral membrane. A cartoon depicting the ACE2-bound 2019-nCoV S protein is shown (right) with ACE2 in blue and S protein monomers colored tan, pink and green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;2019-nCoV S binds human ACE2 with high affinity.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) SPR sensorgram showing the binding kinetics for human ACE2 and immobilized 2019-nCoV S. Data are shown as black lines and the best fit of the data to a 1:1 binding model is shown in red. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Negative-stain EM 2D class averages of 2019-nCoV S bound by ACE2. Averages have been rotated so that ACE2 is positioned above the 2019-nCoV S protein with respect to the viral membrane. A cartoon depicting the ACE2-bound 2019-nCoV S protein is shown (&lt;;em&gt;;right&lt;;/em&gt;;) with ACE2 in blue and S protein monomers colored tan, pink and green.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F3.medium.gif" width="440" height="144"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F3.medium.gif" width="440" height="144"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 3.</span>
      <span class="caption-title">2019-nCoV S binds human ACE2 with high affinity.</span>
      <p id="p-14" class="first-child">(
       <strong>A</strong>) SPR sensorgram showing the binding kinetics for human ACE2 and immobilized 2019-nCoV S. Data are shown as black lines and the best fit of the data to a 1:1 binding model is shown in red. (
       <strong>B</strong>) Negative-stain EM 2D class averages of 2019-nCoV S bound by ACE2. Averages have been rotated so that ACE2 is positioned above the 2019-nCoV S protein with respect to the viral membrane. A cartoon depicting the ACE2-bound 2019-nCoV S protein is shown (
       <em>right</em>) with ACE2 in blue and S protein monomers colored tan, pink and green.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-15">The overall structural homology and shared receptor usage between SARS-CoV S and 2019-nCoV S prompted us to test published SARS-CoV RBD-directed monoclonal antibodies (mAbs) for cross-reactivity to the 2019-nCoV RBD (
     <strong>
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>
     </strong>). A 2019-nCoV RBD-SD1 fragment (S residues 319–591) was recombinantly expressed, and appropriate folding of this construct was validated by measuring ACE2 binding using biolayer interferometry (BLI) (
     <strong>
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>
     </strong>). Cross-reactivity of the SARS-CoV RBD-directed mAbs S230, m396 and 80R was then evaluated by BLI (
     <em>
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, 
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>-
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>
     </em>). Despite the relatively high degree of structural homology between the 2019-nCoV RBD and the SARS-CoV RBD, no binding to the 2019-nCoV RBD could be detected for any of the three mAbs at the concentration tested (1 μM) (
     <strong>
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>, 
      <a id="xref-fig-12-1" class="xref-fig" href="#F12">Supplementary Figure 8</a>
     </strong>). Although the epitopes of these three antibodies represent a relatively small percentage of the surface area of the 2019-nCoV RBD, the lack of observed binding suggests that SARS-directed mAbs will not necessarily be cross-reactive and that future antibody isolation and therapeutic design efforts will benefit from using 2019-nCoV S proteins as probes.
    </p>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Antigenicity of the 2019-nCoV RBD. (A) The SARS-CoV RBD is shown as a white molecular surface (PDB ID: 2AJF), with residues that vary in the 2019-nCoV RBD colored red. The ACE2 binding site is outlined with a black dotted line. (B) A biolayer interferometry sensorgram that shows binding to ACE2 by the 2019-nCoV RBD-SD1. Binding data are shown as a black line and the best fit of the data to a 1:1 binding model is shown in red. (C) Biolayer interferometry to measure cross-reactivity of the SARS-CoV RBD-directed antibodies S230, m396 and 80R. Sensortips with immobilized antibodies were dipped into wells containing 2019-nCoV RBD-SD1 and the resulting data are shown as a black line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Antigenicity of the 2019-nCoV RBD.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) The SARS-CoV RBD is shown as a white molecular surface (PDB ID: 2AJF), with residues that vary in the 2019-nCoV RBD colored red. The ACE2 binding site is outlined with a black dotted line. (&lt;;strong&gt;;B&lt;;/strong&gt;;) A biolayer interferometry sensorgram that shows binding to ACE2 by the 2019-nCoV RBD-SD1. Binding data are shown as a black line and the best fit of the data to a 1:1 binding model is shown in red. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Biolayer interferometry to measure cross-reactivity of the SARS-CoV RBD-directed antibodies S230, m396 and 80R. Sensortips with immobilized antibodies were dipped into wells containing 2019-nCoV RBD-SD1 and the resulting data are shown as a black line.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F4.medium.gif" width="440" height="243"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F4.medium.gif" width="440" height="243"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 4.</span>
      <span class="caption-title">Antigenicity of the 2019-nCoV RBD.</span>
      <p id="p-16" class="first-child">(
       <strong>A</strong>) The SARS-CoV RBD is shown as a white molecular surface (PDB ID: 2AJF), with residues that vary in the 2019-nCoV RBD colored red. The ACE2 binding site is outlined with a black dotted line. (
       <strong>B</strong>) A biolayer interferometry sensorgram that shows binding to ACE2 by the 2019-nCoV RBD-SD1. Binding data are shown as a black line and the best fit of the data to a 1:1 binding model is shown in red. (
       <strong>C</strong>) Biolayer interferometry to measure cross-reactivity of the SARS-CoV RBD-directed antibodies S230, m396 and 80R. Sensortips with immobilized antibodies were dipped into wells containing 2019-nCoV RBD-SD1 and the resulting data are shown as a black line.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-17">The rapid global spread of 2019-nCoV, prompting the PHEIC declaration by WHO signals the urgent need for coronavirus vaccines and therapeutics. Knowing the atomic-level structure of the spike will support precision vaccine design and discovery of antivirals, facilitating medical countermeasure development.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Protein expression and purification</h3>
     <p id="p-18">To express the prefusion S ectodomain, a gene encoding residues 1−1208 of 2019-nCoV S (GenBank: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN908947&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom">MN908947</a>) with proline substitutions at residues 986 and 987, a “GSAS” substitution at the furin cleavage site (residues 682–685), a C-terminal T4 fibritin trimerization motif, an HRV3C protease cleavage site, a TwinStrepTag and an 8XHisTag was synthesized and cloned into the mammalian expression vector pαH. To express the 2019-nCoV RBD-SD1, residues 319−591 of 2019-nCoV S were cloned upstream of a C-terminal HRV3C protease cleavage site, a monomeric Fc tag and an 8XHisTag. Similarly, to express the SARS-CoV RBD-SD1, residues 306−577 of SARS-CoV S (Urbani strain) were cloned upstream of a C-terminal HRV3C protease cleavage site, a monomeric Fc tag and an 8XHisTag. Lastly, a plasmid encoding residues 1−615 of human ACE2 with a C-terminal HRV3C protease cleavage site, a TwinStrepTag and an 8XHisTag was generated.
     </p>
     <p id="p-19">These expression vectors were used to transiently transfect FreeStyle293F cells (Thermo Fischer) using polyethylenimine. Protein was purified from filtered cell supernatants using either StrepTactin resin (IBA) or Protein A resin (Pierce) before being subjected to additional purification by size-exclusion chromatography using either a Superose 6 10/300 column (GE Healthcare) or a Superdex 200 10/300 Increase column (GE Healthcare) in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN
      <sub>3</sub>. ACE2 and the 2019-nCoV RBD-SD1 were incubated with 10% (wt/wt) HRV3C protease for 2 hours at room temperature. Cleaved protein was then passed over either NiNTA resin (ACE2) or Protein A and NiNTA resins (2019-nCoV RBD) to remove cleaved tags and His-tagged protease before being run over a Superdex 200 10/300 Increase column in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN
      <sub>3</sub>.
     </p>
     <p id="p-20">Plasmids encoding the heavy and light chains of S230, 80R and m396 IgG were transiently transfected into Expi293 (Thermo Fischer) using polyethylenimine. Antibodies were purified from cell supernatants using Protein A resin before being used for biolayer interferometry.</p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Cryo-EM sample preparation and data collection</h3>
     <p id="p-21">Purified 2019-nCoV S was diluted to a concentration of 0.35 mg/mL in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN
      <sub>3</sub>. 3 uL of protein was deposited on a CF-1.2/1.3 grid that had been plasma cleaned for 30 seconds in a Solarus 950 plasma cleaner (Gatan) with a 4:1 ratio of O
      <sub>2</sub>/H
      <sub>2</sub>. Excess protein was blotted away for 6 seconds before being plunge frozen into liquid ethane using a Vitrobot Mark IV (Thermo Scientific). Frozen grids were imaged in a Titan Krios (Thermo Scientific) equipped with a K3 detector (Gatan). Movies were collected using Leginon 
      <em>(31)</em> at a magnification of x22,500, corresponding to a calibrated pixel size of 1.047 Å/pixel. A full description of the cryo-EM data collection parameters can be found in 
      <strong>
       <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplementary Table 1</a>
      </strong>.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Cryo-EM data processing</h3>
     <p id="p-22">Motion correction, CTF-estimation and non-templated particle picking were performed in Warp (
      <em>
       <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>
      </em>). Extracted particles were imported into cryoSPARC v2.12.4 (
      <em>
       <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>
      </em>) for 2D classification, 3D classification and non-uniform 3D refinement. The C1 RBD “up” reconstruction was sharpened in cryoSPARC, and the 3D reconstruction with C3 symmetry was subjected to local B-factor sharpening using LocalDeBlur (
      <em>
       <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>
      </em>). Models were built in Coot, before being iteratively refined in both Phenix and ISOLDE (
      <em>
       <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>-
       <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>
      </em>). Some of the data processing and refinement software was curated by SBGrid (
      <em>
       <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>
      </em>). The full cryo-EM data processing workflow is described in 
      <strong>
       <a id="xref-fig-7-2" class="xref-fig" href="#F7">Supplementary Figure 3</a>
      </strong> and the model refinement statistics can be found in 
      <strong>
       <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Supplementary Table 1</a>
      </strong>.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Surface plasmon resonance</h3>
     <p id="p-23">His-tagged 2019-nCoV S was immobilized to an NiNTA sensorchip (GE Healthcare) to a level of ∼800 response units (RUs) using a Biacore X100 (GE Healthcare) and a running buffer composed of 10 mM HEPES pH 8.0, 150 mM NaCl and 0.05% Tween 20. Serial dilutions of purified and untagged ACE2 were injected ranging in concentration from 250 to 15.6 nM. The resulting data were fit to a 1:1 binding model using Biacore Evaluation Software (GE Healthcare). His-tagged SARS-CoV RBD-SD1 was immobilized to an NiNTA sensorchip to a level of ∼350 RUs using a Biacore X100 and the same running buffer listed above. Serial dilutions of purified and untagged ACE2 were injected ranging in concentration from 500 to 31.3 nM. The resulting data were fit to a 1:1 binding model using Biacore Evaluation Software.</p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Negative stain EM</h3>
     <p id="p-24">Purified 2019-nCoV S was diluted to a concentration of 0.032 mg/mL in 2 mM Tris pH 8.0, 200 mM NaCl and 0.02% NaN
      <sub>3</sub>. Diluted S protein was mixed with a 1.5-fold molar excess of ACE2 and the mixture was incubated on ice for 1 minute before 4.8 uL of the protein mixture was deposited on a CF400-Cu grid (Electron Microscopy Sciences) before being stained with methylamine tungstate (Nanoprobes). This grid was imaged in an FEI Talos TEM (Thermo Scientific) equipped with a Ceta 16M detector. Micrographs were collected manually using TIA v4.14 software at a magnification of x92,000, corresponding to a pixel size of 1.63 Å/pixel. CTF estimation, particle picking and 2D class averaging were performed in 
      <em>cis</em>TEM (
      <em>
       <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>
      </em>).
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Biolayer interferometry</h3>
     <p id="p-25">Fc-tagged 2019-nCoV RBD-SD1 was immobilized to an anti-human capture (AHC) sensortip (FortéBio) using an Octet RED96e (FortéBio). The sensortip was then dipped into 100 nM ACE2 to measure association before being dipped into a well containing only running buffer composed of 10 mM HEPES pH 7.5, 150 mM NaCl, 3 mM EDTA, 0.05% Tween 20 and 1 mg/mL bovine serum albumin to measure dissociation. Data were reference subtracted and fit to a 1:1 binding model using Octet Data Analysis Software v11.1 (FortéBio).</p>
     <p id="p-26">S230, 80R and m396 IgGs were immobilized to AHC sensortips to a response level of ∼0.8 nm and dipped into wells containing 1 μM untagged 2019-nCoV RBD-SD1 before being dipped into wells containing only running buffer to measure dissociation. Data were reference-subtracted and aligned to a baseline after IgG capture using Octet Data Analysis software v11.1. An analogous experiment was performed under identical conditions by dipping AHC sensor tips loaded with S230, 80R or m396 IgG into untagged SARS-CoV RBD-SD1. Data were reference-subtracted, aligned to a baseline after IgG capture and fit to a 1:1 binding model using Octet Data Analysis software v11.1.</p>
    </div>
   </div>
   <div id="sec-10" class="subsection">
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="2019-nCoV S expression and purification. (A) SDS-PAGE analysis of the 2019-nCoV S protein. Lane 1: molecular weight ladder, with relevant bands labeled in kilodaltons (left); lane 2: filtered supernatant from transfected cells; lane 3: supernatant after passing through StrepTactin resin; lane 4: wash of StrepTactin resin; lane 5: elution from StrepTactin resin. The band corresponding to 2019-nCoV S is denoted with a black arrow. (B) Size-exclusion chromatogram of the affinity-purified 2019-nCoV S protein. Data from a Superose 6 10/300 column are shown in red. The elution volume of a 670 kilodalton molecular weight standard is shown as a black dotted line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;2019-nCoV S expression and purification.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) SDS-PAGE analysis of the 2019-nCoV S protein. Lane 1: molecular weight ladder, with relevant bands labeled in kilodaltons (&lt;;em&gt;;left&lt;;/em&gt;;); lane 2: filtered supernatant from transfected cells; lane 3: supernatant after passing through StrepTactin resin; lane 4: wash of StrepTactin resin; lane 5: elution from StrepTactin resin. The band corresponding to 2019-nCoV S is denoted with a black arrow. (&lt;;strong&gt;;B&lt;;/strong&gt;;) Size-exclusion chromatogram of the affinity-purified 2019-nCoV S protein. Data from a Superose 6 10/300 column are shown in red. The elution volume of a 670 kilodalton molecular weight standard is shown as a black dotted line.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F5.medium.gif" width="440" height="226"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F5.medium.gif" width="440" height="226"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 1.</span>
      <span class="caption-title">2019-nCoV S expression and purification.</span>
      <p id="p-28" class="first-child">(
       <strong>A</strong>) SDS-PAGE analysis of the 2019-nCoV S protein. Lane 1: molecular weight ladder, with relevant bands labeled in kilodaltons (
       <em>left</em>); lane 2: filtered supernatant from transfected cells; lane 3: supernatant after passing through StrepTactin resin; lane 4: wash of StrepTactin resin; lane 5: elution from StrepTactin resin. The band corresponding to 2019-nCoV S is denoted with a black arrow. (
       <strong>B</strong>) Size-exclusion chromatogram of the affinity-purified 2019-nCoV S protein. Data from a Superose 6 10/300 column are shown in red. The elution volume of a 670 kilodalton molecular weight standard is shown as a black dotted line.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Cryo-EM structure validation. (A) FSC curves (top) and the viewing direction distribution plot (bottom) for 2019-nCoV S with a single RBD [[#x201C]]up[[#x201D]]. (B) FSC curves (top) and the viewing direction distribution plot (bottom) for the 2019-nCoV S processed with C3 symmetry. (C) The cryo-EM density of the 2019-nCoV S with a single RBD [[#x201C]]up[[#x201D]] is shown, colored according to local resolution. (D) Density from S2 of the C3-refined 2019-nCoV S structure. Residues are shown as sticks, colored according to Figure 1A with oxygen atoms colored red, nitrogens colored blue and sulfurs colored yellow. The cryo-EM density map is shown as a gray mesh." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Cryo-EM structure validation.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) FSC curves (&lt;;em&gt;;top&lt;;/em&gt;;) and the viewing direction distribution plot (&lt;;em&gt;;bottom&lt;;/em&gt;;) for 2019-nCoV S with a single RBD [[#x201C]]up[[#x201D]]. (&lt;;strong&gt;;B&lt;;/strong&gt;;) FSC curves (&lt;;em&gt;;top&lt;;/em&gt;;) and the viewing direction distribution plot (&lt;;em&gt;;bottom&lt;;/em&gt;;) for the 2019-nCoV S processed with C3 symmetry. (&lt;;strong&gt;;C&lt;;/strong&gt;;) The cryo-EM density of the 2019-nCoV S with a single RBD [[#x201C]]up[[#x201D]] is shown, colored according to local resolution. (&lt;;strong&gt;;D&lt;;/strong&gt;;) Density from S2 of the C3-refined 2019-nCoV S structure. Residues are shown as sticks, colored according to &lt;;strong&gt;;Figure 1A&lt;;/strong&gt;; with oxygen atoms colored red, nitrogens colored blue and sulfurs colored yellow. The cryo-EM density map is shown as a gray mesh.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F6.medium.gif" width="440" height="285"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F6.medium.gif" width="440" height="285"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 2.</span>
      <span class="caption-title">Cryo-EM structure validation.</span>
      <p id="p-29" class="first-child">(
       <strong>A</strong>) FSC curves (
       <em>top</em>) and the viewing direction distribution plot (
       <em>bottom</em>) for 2019-nCoV S with a single RBD “up”. (
       <strong>B</strong>) FSC curves (
       <em>top</em>) and the viewing direction distribution plot (
       <em>bottom</em>) for the 2019-nCoV S processed with C3 symmetry. (
       <strong>C</strong>) The cryo-EM density of the 2019-nCoV S with a single RBD “up” is shown, colored according to local resolution. (
       <strong>D</strong>) Density from S2 of the C3-refined 2019-nCoV S structure. Residues are shown as sticks, colored according to 
       <strong>
        <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>
       </strong> with oxygen atoms colored red, nitrogens colored blue and sulfurs colored yellow. The cryo-EM density map is shown as a gray mesh.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Cryo-EM data processing workflow. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Cryo-EM data processing workflow.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F7.medium.gif" width="425" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F7.medium.gif" width="425" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 3.</span>
      <span class="caption-title">Cryo-EM data processing workflow.</span>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F8" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Cryo-EM map and N-linked glycosylation sites. (A) The unsharpened cryo-EM density map for the C3-processed 2019-nCoV S is shown as a transparent molecular surface, with a single protomer fit into the map shown in ribbons and colored according to Figure 1A. Some S2 density that corresponds to N-linked glycans is labeled. (B) The 2019-nCoV S trimer is shown as a molecular surface with each protomer colored green, pink or tan. Asparagine residues that correspond to N-linked glycosylation sites are shown as red spheres." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Cryo-EM map and &lt;;em&gt;;N&lt;;/em&gt;;-linked glycosylation sites.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) The unsharpened cryo-EM density map for the C3-processed 2019-nCoV S is shown as a transparent molecular surface, with a single protomer fit into the map shown in ribbons and colored according to &lt;;strong&gt;;Figure 1A&lt;;/strong&gt;;. Some S2 density that corresponds to &lt;;em&gt;;N&lt;;/em&gt;;-linked glycans is labeled. (&lt;;strong&gt;;B&lt;;/strong&gt;;) The 2019-nCoV S trimer is shown as a molecular surface with each protomer colored green, pink or tan. Asparagine residues that correspond to &lt;;em&gt;;N&lt;;/em&gt;;-linked glycosylation sites are shown as red spheres.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F8.medium.gif" width="440" height="227"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F8.medium.gif" width="440" height="227"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 4.</span>
      <span class="caption-title">Cryo-EM map and 
       <em>N</em>-linked glycosylation sites.
      </span>
      <p id="p-30" class="first-child">(
       <strong>A</strong>) The unsharpened cryo-EM density map for the C3-processed 2019-nCoV S is shown as a transparent molecular surface, with a single protomer fit into the map shown in ribbons and colored according to 
       <strong>
        <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a>
       </strong>. Some S2 density that corresponds to 
       <em>N</em>-linked glycans is labeled. (
       <strong>B</strong>) The 2019-nCoV S trimer is shown as a molecular surface with each protomer colored green, pink or tan. Asparagine residues that correspond to 
       <em>N</em>-linked glycosylation sites are shown as red spheres.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F9" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sequence alignment of 2019-nCoV S, SARS-CoV S and RaTG13 S. Identical residues are denoted by an [[#x201C]]*[[#x201D]] beneath the consensus position. Structural domains are colored according to Figure 1A." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Sequence alignment of 2019-nCoV S, SARS-CoV S and RaTG13 S.&lt;;/span&gt;; Identical residues are denoted by an [[#x201C]]*[[#x201D]] beneath the consensus position. Structural domains are colored according to &lt;;strong&gt;;Figure 1A&lt;;/strong&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F9.medium.gif" width="331" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F9.medium.gif" width="331" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 5.</span>
      <span class="caption-title">Sequence alignment of 2019-nCoV S, SARS-CoV S and RaTG13 S.</span>
      <p id="p-31" class="first-child">Identical residues are denoted by an “*” beneath the consensus position. Structural domains are colored according to 
       <strong>
        <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>
       </strong>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F10" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sequence variability between RaTG13 S and 2019-nCoV S clinical isolates. (A) Table shows residues in the 2019-nCoV S protein that vary in RaTG13, grouped by structural domain. (B) A single monomer of the 2019-nCoV S protein is shown in ribbons, colored green. RaTG13 variant residues are shown as blue spheres. (C) Table shows variations in the 2019-nCoV S sequence based on 61 clinical isolates and the domains wherein these variations occur. (D) A single monomer of the 2019-nCoV S protein is shown in ribbons, colored green. Variant residues are shown as gold spheres." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Sequence variability between RaTG13 S and 2019-nCoV S clinical isolates.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) Table shows residues in the 2019-nCoV S protein that vary in RaTG13, grouped by structural domain. (&lt;;strong&gt;;B&lt;;/strong&gt;;) A single monomer of the 2019-nCoV S protein is shown in ribbons, colored green. RaTG13 variant residues are shown as blue spheres. (&lt;;strong&gt;;C&lt;;/strong&gt;;) Table shows variations in the 2019-nCoV S sequence based on 61 clinical isolates and the domains wherein these variations occur. (&lt;;strong&gt;;D&lt;;/strong&gt;;) A single monomer of the 2019-nCoV S protein is shown in ribbons, colored green. Variant residues are shown as gold spheres.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F10.medium.gif" width="336" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F10.medium.gif" width="336" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 6.</span>
      <span class="caption-title">Sequence variability between RaTG13 S and 2019-nCoV S clinical isolates.</span>
      <p id="p-32" class="first-child">(
       <strong>A</strong>) Table shows residues in the 2019-nCoV S protein that vary in RaTG13, grouped by structural domain. (
       <strong>B</strong>) A single monomer of the 2019-nCoV S protein is shown in ribbons, colored green. RaTG13 variant residues are shown as blue spheres. (
       <strong>C</strong>) Table shows variations in the 2019-nCoV S sequence based on 61 clinical isolates and the domains wherein these variations occur. (
       <strong>D</strong>) A single monomer of the 2019-nCoV S protein is shown in ribbons, colored green. Variant residues are shown as gold spheres.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F11" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F11.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SARS-CoV RBD-SD1 binding to human ACE2. An SPR sensorgram is shown, displaying the binding between soluble human ACE2 and immobilized SARS-CoV RBD-SD1. The data are shown as black lines and the best fit of the data to a 1:1 binding model is shown in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;SARS-CoV RBD-SD1 binding to human ACE2.&lt;;/span&gt;; An SPR sensorgram is shown, displaying the binding between soluble human ACE2 and immobilized SARS-CoV RBD-SD1. The data are shown as black lines and the best fit of the data to a 1:1 binding model is shown in red.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F11.medium.gif" width="440" height="218"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F11.medium.gif" width="440" height="218"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 7.</span>
      <span class="caption-title">SARS-CoV RBD-SD1 binding to human ACE2.</span>
      <p id="p-33" class="first-child">An SPR sensorgram is shown, displaying the binding between soluble human ACE2 and immobilized SARS-CoV RBD-SD1. The data are shown as black lines and the best fit of the data to a 1:1 binding model is shown in red.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F12" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F12.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SARS-CoV RBD-directed antibody validation. The monoclonal antibodies that were tested for cross-reactivity to the 2019-nCoV RBD-SD1 were also tested for binding to the SARS-CoV S RBD-SD1 as a positive control. Binding data are shown as a black line and the best fit of the data to a 1:1 binding model is shown in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-394013990" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;SARS-CoV RBD-directed antibody validation.&lt;;/span&gt;; The monoclonal antibodies that were tested for cross-reactivity to the 2019-nCoV RBD-SD1 were also tested for binding to the SARS-CoV S RBD-SD1 as a positive control. Binding data are shown as a black line and the best fit of the data to a 1:1 binding model is shown in red.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F12.medium.gif" width="440" height="147"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F12.medium.gif" width="440" height="147"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/15/2020.02.11.944462/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 8.</span>
      <span class="caption-title">SARS-CoV RBD-directed antibody validation.</span>
      <p id="p-34" class="first-child">The monoclonal antibodies that were tested for cross-reactivity to the 2019-nCoV RBD-SD1 were also tested for binding to the SARS-CoV S RBD-SD1 as a positive control. Binding data are shown as a black line and the best fit of the data to a 1:1 binding model is shown in red.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1155079/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1155079/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1155079" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Supplementary Table 1.</span>
      <span class="caption-title">Cryo-EM data collection and refinement statistics.</span>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-35">
     <strong>Supplementary Movie 1.</strong> CryoSPARC 3D variability analysis. 2019-nCoV S trimer viewed from the side, along the viral membrane.
    </p>
    <p id="p-36">
     <strong>Supplementary Movie 2.</strong> CryoSPARC 3D variability analysis. 2019-nCoV S trimer viewed from the top, toward the viral membrane.
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-27">We thank Dr. John Ludes-Meyers for assistance with cell transfection and the rest of the members of the McLellan laboratory for critical reading of the manuscript. We would also like to thank Dr. Aguang Dai from the Sauer Structural Biology Laboratory at the University of Texas at Austin for his assistance with microscope alignment. This work was supported by an NIH/NIAID grant awarded to J.S.M. (R01-AI127521). The Sauer Structural Biology Laboratory is supported by the University of Texas College of Natural Sciences and by award RR160023 of the Cancer Prevention and Research Institute of Texas (CPRIT).</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2>References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J. F.</span>
          <span class="cit-name-surname">Chan</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Huang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Lu</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Wu</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Chen</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Q.</span>
          <span class="cit-name-surname">Li</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. 
         <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">237</span>–
         <span class="cit-lpage">261</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D3%26rft.spage%253D237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.8" data-doi="10.1128/JVI.77.16.8801-8811.2003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B. J.</span>
          <span class="cit-name-surname">Bosch</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">van der Zee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. A.</span>
          <span class="cit-name-surname">de Haan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. J.</span>
          <span class="cit-name-surname">Rottier</span>
         </span>, 
         <span class="cit-article-title">The coronavirus spike protein is a class I virus fusion protein: structural and functional characterization of the fusion core complex</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">77</span>, 
         <span class="cit-fpage">8801</span>–
         <span class="cit-lpage">8811</span> (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBosch%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D77%26rft.issue%253D16%26rft.spage%253D8801%26rft.epage%253D8811%26rft.atitle%253DThe%2BCoronavirus%2BSpike%2BProtein%2BIs%2Ba%2BClass%2BI%2BVirus%2BFusion%2BProtein%253A%2BStructural%2Band%2BFunctional%2BCharacterization%2Bof%2Bthe%2BFusion%2BCore%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.16.8801-8811.2003%26rft_id%253Dinfo%253Apmid%252F12885899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ny8xNi84ODAxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTUvMjAyMC4wMi4xMS45NDQ0NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.9" data-doi="10.1073/pnas.1708727114">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. C.</span>
          <span class="cit-name-surname">Walls</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">114</span>, 
         <span class="cit-fpage">11157</span>–
         <span class="cit-lpage">11162</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1708727114%26rft_id%253Dinfo%253Apmid%252F29073020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQyLzExMTU3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTUvMjAyMC4wMi4xMS45NDQ0NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.10" data-doi="10.1038/cr.2016.152">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Gui</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">Cell Res</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">119</span>–
         <span class="cit-lpage">129</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D27%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/cr.2016.152&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.11" data-doi="10.1073/pnas.1707304114">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Pallesen</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">114</span>, 
         <span class="cit-fpage">E7348</span>–
         <span class="cit-lpage">E7357</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1707304114%26rft_id%253Dinfo%253Apmid%252F28807998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzM1L0U3MzQ4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTUvMjAyMC4wMi4xMS45NDQ0NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.12" data-doi="10.1016/j.cell.2018.12.028">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. C.</span>
          <span class="cit-name-surname">Walls</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-vol">176</span>, 
         <span class="cit-fpage">1026</span>–
         <span class="cit-lpage">1039 e1015</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.13" data-doi="10.1038/ncomms15092">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Yuan</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">15092</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D15092%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ncomms15092&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.14" data-doi="10.1038/s41598-018-36918-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R. N.</span>
          <span class="cit-name-surname">Kirchdoerfer</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">15701</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D8%26rft.spage%253D15701%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-36918-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41598-018-36918-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.15" data-doi="10.1038/nmeth.4169">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Punjani</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. L.</span>
          <span class="cit-name-surname">Rubinstein</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. J.</span>
          <span class="cit-name-surname">Fleet</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. A.</span>
          <span class="cit-name-surname">Brubaker</span>
         </span>, 
         <span class="cit-article-title">cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">290</span>–
         <span class="cit-lpage">296</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D290%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4169%26rft_id%253Dinfo%253Apmid%252F28165473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.4169&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28165473&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Wrapp</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. S.</span>
          <span class="cit-name-surname">McLellan</span>
         </span>, 
         <span class="cit-article-title">The 3.1-Angstrom Cryo-electron Microscopy Structure of the Porcine Epidemic Diarrhea Virus Spike Protein in the Prefusion Conformation</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">93</span>, (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.11.944462v1.17" data-doi="10.1038/nsmb.3293">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. C.</span>
          <span class="cit-name-surname">Walls</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Glycan shield and epitope masking of a coronavirus spike protein observed by cryo-electron microscopy</span>. 
         <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr>
         <span class="cit-vol">23</span>, 
         <span class="cit-fpage">899</span>–
         <span class="cit-lpage">905</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BStruct%2BMol%2BBiol%26rft.volume%253D23%26rft.spage%253D899%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nsmb.3293&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.18" data-doi="10.1038/nature17200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R. N.</span>
          <span class="cit-name-surname">Kirchdoerfer</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Pre-fusion structure of a human coronavirus spike protein</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">531</span>, 
         <span class="cit-fpage">118</span>–
         <span class="cit-lpage">121</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D531%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17200%26rft_id%253Dinfo%253Apmid%252F26935699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature17200&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26935699&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.19" data-doi="10.1128/JVI.00415-08">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B. J.</span>
          <span class="cit-name-surname">Bosch</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Bartelink</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. J.</span>
          <span class="cit-name-surname">Rottier</span>
         </span>, 
         <span class="cit-article-title">Cathepsin L functionally cleaves the severe acute respiratory syndrome coronavirus class I fusion protein upstream of rather than adjacent to the fusion peptide</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">82</span>, 
         <span class="cit-fpage">8887</span>–
         <span class="cit-lpage">8890</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00415-08%26rft_id%253Dinfo%253Apmid%252F18562523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNy84ODg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMTUvMjAyMC4wMi4xMS45NDQ0NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.11.944462v1.20" data-doi="10.1128/JVI.02232-10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">I.</span>
          <span class="cit-name-surname">Glowacka</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Evidence that TMPRSS2 activates the severe acute respiratory syndrome coronavirus spike protein for membrane fusion and reduces viral control by the humoral immune response</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">85</span>, 
         <span class="cit-fpage">4122</span>–
         <span class="cit-lpage">4134</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02232-10%26rft_id%253Dinfo%253Apmid%252F21325420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg1LzkvNDEyMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzE1LzIwMjAuMDIuMTEuOTQ0NDYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.21" data-doi="10.1038/nature02145">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Li</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">426</span>, 
         <span class="cit-fpage">450</span>–
         <span class="cit-lpage">454</span> (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14647384&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.22" data-doi="10.1073/pnas.0809524106">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Belouzard</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">V. C.</span>
          <span class="cit-name-surname">Chu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. R.</span>
          <span class="cit-name-surname">Whittaker</span>
         </span>, 
         <span class="cit-article-title">Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">106</span>, 
         <span class="cit-fpage">5871</span>–
         <span class="cit-lpage">5876</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0809524106%26rft_id%253Dinfo%253Apmid%252F19321428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzE0LzU4NzEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNS8yMDIwLjAyLjExLjk0NDQ2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.23" data-doi="10.1016/S0092-8674(00)81771-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Chen</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Structure of the hemagglutinin precursor cleavage site, a determinant of influenza pathogenicity and the origin of the labile conformation</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-vol">95</span>, 
         <span class="cit-fpage">409</span>–
         <span class="cit-lpage">417</span> (
         <span class="cit-pub-date">1998</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChen%26rft.auinit1%253DJ.%26rft.volume%253D95%26rft.issue%253D3%26rft.spage%253D409%26rft.epage%253D417%26rft.atitle%253DStructure%2Bof%2Bthe%2Bhemagglutinin%2Bprecursor%2Bcleavage%2Bsite%252C%2Ba%2Bdeterminant%2Bof%2Binfluenza%2Bpathogenicity%2Band%2Bthe%2Borigin%2Bof%2Bthe%2Blabile%2Bconformation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981771-7%26rft_id%253Dinfo%253Apmid%252F9814710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81771-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9814710&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000076789600013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M. K.-W</span>. 
          <span class="cit-name-surname">Hoffmann H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Krüger</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Müller</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Drosten</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pöhlmann</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-article-title">The novel coronavirus 2019 (2019-nCoV) uses the SARS-coronavirus receptor ACE2 and the cellular protease TMPRSS2 for entry into target cells</span>. 
         <abbr class="cit-jnl-abbrev">BioRXiv</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.11.944462v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Wan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Shang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Graham</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. S.</span>
          <span class="cit-name-surname">Baric</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Zhou</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.27" data-doi="10.1371/journal.ppat.1007236">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Song</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Gui</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Xiang</span>
         </span>, 
         <span class="cit-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">e1007236</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007236&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.28" data-doi="10.1074/jbc.M603275200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">W. C.</span>
          <span class="cit-name-surname">Hwang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R</span>. 
         <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>
         <span class="cit-vol">281</span>, 
         <span class="cit-fpage">34610</span>–
         <span class="cit-lpage">34616</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M603275200%26rft_id%253Dinfo%253Apmid%252F16954221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDUvMzQ2MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNS8yMDIwLjAyLjExLjk0NDQ2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.11.944462v1.29" data-doi="10.1074/jbc.M600697200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Prabakaran</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Structure of severe acute respiratory syndrome coronavirus receptor-binding domain complexed with neutralizing antibody</span>. 
         <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>
         <span class="cit-vol">281</span>, 
         <span class="cit-fpage">15829</span>–
         <span class="cit-lpage">15836</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M600697200%26rft_id%253Dinfo%253Apmid%252F16597622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjMvMTU4MjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8xNS8yMDIwLjAyLjExLjk0NDQ2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">X. L.</span>
          <span class="cit-name-surname">Tian C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ying</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. 
         <abbr class="cit-jnl-abbrev">BioRXiv</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.11.944462v1.31" data-doi="10.1006/jsbi.2000.4314">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Carragher</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Leginon: an automated system for acquisition of images from vitreous ice specimens</span>. 
         <abbr class="cit-jnl-abbrev">J Struct Biol</abbr>
         <span class="cit-vol">132</span>, 
         <span class="cit-fpage">33</span>–
         <span class="cit-lpage">45</span> (
         <span class="cit-pub-date">2000</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DCarragher%26rft.auinit1%253DB.%26rft.volume%253D132%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D45%26rft.atitle%253DLeginon%253A%2Ban%2Bautomated%2Bsystem%2Bfor%2Bacquisition%2Bof%2Bimages%2Bfrom%2Bvitreous%2Bice%2Bspecimens.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjsbi.2000.4314%26rft_id%253Dinfo%253Apmid%252F11121305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/jsbi.2000.4314&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11121305&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000166379300004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Tegunov</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Cramer</span>
         </span>, 
         <span class="cit-article-title">Real-time cryo-electron microscopy data preprocessing with Warp</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>
         <span class="cit-vol">16</span>, 
         <span class="cit-fpage">1146</span>–
         <span class="cit-lpage">1152</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D16%26rft.spage%253D1146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Ramirez-Aportela</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Automatic local resolution-based sharpening of cryo-EM maps</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.34" data-doi="10.1107/S0907444902016657">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P. D.</span>
          <span class="cit-name-surname">Adams</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">PHENIX: building new software for automated crystallographic structure determination</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallogr D Biol Crystallogr</abbr>
         <span class="cit-vol">58</span>, 
         <span class="cit-fpage">1948</span>–
         <span class="cit-lpage">1954</span> (
         <span class="cit-pub-date">2002</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DAdams%26rft.auinit1%253DP.%2BD.%26rft.volume%253D58%26rft.issue%253DPt%2B11%26rft.spage%253D1948%26rft.epage%253D1954%26rft.atitle%253DPHENIX%253A%2Bbuilding%2Bnew%2Bsoftware%2Bfor%2Bautomated%2Bcrystallographic%2Bstructure%2Bdetermination.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444902016657%26rft_id%253Dinfo%253Apmid%252F12393927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S0907444902016657&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12393927&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000178745000009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.11.944462v1.35" data-doi="10.1107/S2059798318002425">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">T. I.</span>
          <span class="cit-name-surname">Croll</span>
         </span>, 
         <span class="cit-article-title">ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallogr D Struct Biol</abbr>
         <span class="cit-vol">74</span>, 
         <span class="cit-fpage">519</span>–
         <span class="cit-lpage">530</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallogr%2BD%2BStruct%2BBiol%26rft.volume%253D74%26rft.spage%253D519%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS2059798318002425%26rft_id%253Dinfo%253Apmid%252F29872003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S2059798318002425&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29872003&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.36" data-doi="10.1107/S0907444904019158">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Emsley</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Cowtan</span>
         </span>, 
         <span class="cit-article-title">Coot: model-building tools for molecular graphics</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallogr D Biol Crystallogr</abbr>
         <span class="cit-vol">60</span>, 
         <span class="cit-fpage">2126</span>–
         <span class="cit-lpage">2132</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DEmsley%26rft.auinit1%253DP.%26rft.volume%253D60%26rft.issue%253DPt%2B12%2BPt%2B1%26rft.spage%253D2126%26rft.epage%253D2132%26rft.atitle%253DCoot%253A%2Bmodel-building%2Btools%2Bfor%2Bmolecular%2Bgraphics.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444904019158%26rft_id%253Dinfo%253Apmid%252F15572765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S0907444904019158&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15572765&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000225360500002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.37" data-doi="10.7554/eLife.01456">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Morin</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Collaboration gets the most out of software</span>. 
         <abbr class="cit-jnl-abbrev">Elife</abbr>
         <span class="cit-vol">2</span>, 
         <span class="cit-fpage">e01456</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D2%26rft.issue%253D0%26rft.spage%253De01456%26rft.epage%253De01456%26rft.atitle%253DCollaboration%2Bgets%2Bthe%2Bmost%2Bout%2Bof%2Bsoftware%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.01456%26rft_id%253Dinfo%253Apmid%252F24040512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.7554/eLife.01456&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24040512&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F15%2F2020.02.11.944462.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.11.944462v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Grant</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Rohou</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Grigorieff</span>
         </span>, 
         <span class="cit-article-title">cisTEM, user-friendly software for single-particle image processing</span>. 
         <abbr class="cit-jnl-abbrev">Elife</abbr>
         <span class="cit-vol">7</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
