<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5027562">5027562</a></span><span class="pii" title="pii">srep32251</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/srep32251">10.1038/srep32251</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
A Direct Comparison of Two Densely Sampled HIV Epidemics: The UK and Switzerland</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragonnet-Cronin</span><span class="given-names" tagx="given-names" title="given-names">Manon L.</span></span><a href="#c1">a</a><a href="#a1">1</a><a href="#n1">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shilaih</span><span class="given-names" tagx="given-names" title="given-names">Mohaned</span></span><a href="#a2">2</a><a href="#a3">3</a><a href="#n1">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Günthard</span><span class="given-names" tagx="given-names" title="given-names">Huldrych F.</span></span><a href="#a2">2</a><a href="#a3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodcroft</span><span class="given-names" tagx="given-names" title="given-names">Emma B.</span></span><a href="#a1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Böni</span><span class="given-names" tagx="given-names" title="given-names">Jürg</span></span><a href="#a3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fearnhill</span><span class="given-names" tagx="given-names" title="given-names">Esther</span></span><a href="#a4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunn</span><span class="given-names" tagx="given-names" title="given-names">David</span></span><a href="#a4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerly</span><span class="given-names" tagx="given-names" title="given-names">Sabine</span></span><a href="#a5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimkait</span><span class="given-names" tagx="given-names" title="given-names">Thomas</span></span><a href="#a6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aubert</span><span class="given-names" tagx="given-names" title="given-names">Vincent</span></span><a href="#a7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Wan-Lin</span></span><a href="#a2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">Alison E.</span></span><a href="#a8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lycett</span><span class="given-names" tagx="given-names" title="given-names">Samantha J.</span></span><a href="#a1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">Roger</span></span><a href="#a2">2</a><a href="#a3">3</a><a href="#n1">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">Andrew J. Leigh</span></span><a href="#a1">1</a><a href="#n1">*</a></span><span class="citation_author_institution" id="a1">[1], <span class="institution" title="institution">University of Edinburgh</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="a2">[2], <span class="institution" title="institution">Division of Infectious Diseases and Hospital Epidemiology, University Hospital Zurich, University of Zurich</span><span class="country" tagx="country" title="country">Switzerland</span></span><span class="citation_author_institution" id="a3">[3], <span class="institution" title="institution">Institute of Medical Virology, University of Zurich</span><span class="country" tagx="country" title="country">Switzerland</span></span><span class="citation_author_institution" id="a4">[4], <span class="institution" title="institution">MRC Clinical Trials Unit</span><span class="country" tagx="country" title="country">UK</span></span><span class="citation_author_institution" id="a5">[5], <span class="institution" title="institution">Laboratory of Virology and AIDS Center, Geneva University Hospital</span><span class="country" tagx="country" title="country">Switzerland</span></span><span class="citation_author_institution" id="a6">[6], <span class="institution" title="institution">Department Biomedicine-Petersplatz, University of Basel</span><span class="country" tagx="country" title="country">Switzerland</span></span><span class="citation_author_institution" id="a7">[7], <span class="institution" title="institution">Division of Immunology and Allergy, Centre Hospitalier Universitaire Vaudois and University of Lausanne</span><span class="country" tagx="country" title="country">Switzerland</span></span><span class="citation_author_institution" id="a8">[8], <span class="institution" title="institution">Public Health England</span><span class="country" tagx="country" title="country">UK</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">a</span><span class="email" tagx="email" title="email">Manon.Ragonnet@ed.ac.uk</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p>These authors contributed equally to this work.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-09-09</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span></span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">32251</span><span class="history" title="history"><span class="received" title="received">received: 2016-05-19</span><span class="accepted" title="accepted">accepted: 2016-07-27</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">The Author(s)</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span></span><span class="license" title="license"><span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a></span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Phylogenetic clustering approaches can elucidate HIV transmission dynamics. Comparisons across countries are essential for evaluating public health policies. Here, we used a standardised approach to compare the UK HIV Drug Resistance Database and the Swiss HIV Cohort Study while maintaining data-protection requirements. Clusters were identified in subtype A1, B and C <i>pol</i> phylogenies. We generated degree distributions for each risk group and compared distributions between countries using Kolmogorov-Smirnov (KS) tests, Degree Distribution Quantification and Comparison (DDQC) and bootstrapping. We used logistic regression to predict cluster membership based on country, sampling date, risk group, ethnicity and sex. We analysed &amp;gt;8,000 Swiss and &amp;gt;30,000 UK subtype B sequences. At 4.5% genetic distance, the UK was more clustered and MSM and heterosexual degree distributions differed significantly by the KS test. The KS test is sensitive to variation in network scale, and jackknifing the UK MSM dataset to the size of the Swiss dataset removed the difference. Only heterosexuals varied based on the DDQC, due to UK male heterosexuals who clustered exclusively with MSM. Their removal eliminated this difference. In conclusion, the UK and Swiss HIV epidemics have similar underlying dynamics and observed differences in clustering are mainly due to different population sizes.</p> </div> </div> </div> <div class="body" title="body"> <p>Despite huge advances in prevention and treatment, the global burden of HIV continues to rise and novel approaches are required to eliminate the virus. Phylogenetic analyses of HIV genetic sequences enable detailed reconstructions of historical and current HIV transmission. As such, phylogenetics can help identify populations at risk and better tailor public health interventions.</p> <p>The UK and Switzerland have two of the most extensive HIV sequence databases in the world: the UK HIV Drug Resistance Database (UK HIV RDB) and the Swiss HIV Cohort Study (SHCS), which contain <i>pol</i> sequences from at least 60% of diagnosed patients within each country. These databases have allowed Switzerland and the UK to lead the way in population-level HIV research, including HIV phylogenetics. Phylogenetic analyses of HIV in these densely sampled epidemics have been used to estimate epidemic parameters including times between transmissions according to risk group<a href="#b1">1</a><a href="#b2">2</a>, crossover between risk groups<a href="#b3">3</a><a href="#b4">4</a><a href="#b5">5</a> and the relative contributions of local versus imported infections<a href="#b5">5</a>.</p> <p>HIV was introduced into UK men who have sex with men (MSM) around 1980<a href="#b6">6</a>, probably from the USA<a href="#b7">7</a>. Until the early 1990 s the epidemic was dominated by the transmission of subtype B among MSM and people who inject drugs (PWID). Six large reconstructed MSM transmission chains reflect independent introductions. The MSM epidemic displayed exponential growth in the 1980 s then stabilised in the 1990 s but continues to be highly clustered<a href="#b8">8</a>. In the 1990 s, cases among heterosexuals became more common<a href="#b9">9</a> and non-B subtypes increased<a href="#b10">10</a><a href="#b11">11</a>. Within MSM clusters, 25% of transmissions occur within 6 months of infection<a href="#b2">2</a>. Heterosexuals in the UK display far less clustering and slower epidemic dynamics: only 2% of transmissions occur within 6 months of infection<a href="#b1">1</a>. Currently, 100,000 people are living with HIV (0.15% prevalence), one quarter of whom are unaware of their infection<a href="#b12">12</a>. Highly Active Antiretroviral Therapy (HAART) became available to residents in 1996 and to all in 2012. HIV positive people on successful treatment in the UK have normal life expectancy<a href="#b13">13</a>.</p> <p>Switzerland had the highest HIV prevalence in Europe in the 1980 s<a href="#b14">14</a>. HIV initially spread among MSM and PWID<a href="#b15">15</a> with heterosexual transmission starting to play a role after the mid-1980 s. The number of new diagnoses declined in the 1990 s, owing to needle exchange, heightened awareness, wide-scale testing, and the introduction of HAART in 1996. The number of new HIV diagnoses in Switzerland has fluctuated since 2000 with no clear time trends. The MSM and PWID epidemics display limited overlap. The heterosexual subtype B epidemic appears to be reseeded by PWID, migration, and (to a limited extent) MSM, with the importance of PWID in driving new infections decreasing over time<a href="#b3">3</a><a href="#b5">5</a><a href="#b16">16</a> (less than 3 PWID infections per year in the last 5 years<a href="#b17">17</a>). Meanwhile only 25% of non-B infections arise within Swiss-specific clusters, indicating a growing role for immigration<a href="#b5">5</a>.</p> <p>One important focus of phylogenetic analyses is on clusters: groups of sequences more related to each other than to the rest of the tree. Clustered sequences represent epidemiologically linked infections with short durations of time between transmissions and thus clusters represent the leading edge of the epidemic. High clustering of sequences within a country indicates rapid transmission and a bias towards within-country transmission.</p> <p>Despite the two countries sharing similar epidemic histories, separate analyses have suggested different structures of the two countries’ epidemics, including distinct proportions of clustered sequences. In the UK, 24%, 40% and 22% of patients infected with HIV-1 subtypes B<sup>17</sup>, A1 and C<sup>12</sup>, respectively, cluster, whereas the numbers for Switzerland are 55%, 21% and 16%<a href="#b3">3</a><a href="#b5">5</a>. Clearly these differences arise in part due to distinct cluster definitions: the SHCS defines a cluster as ≥10 sequences supported by &amp;gt;80% bootstrap<a href="#b3">3</a>, while the UK studies defined clusters by a genetic distance (GD) ≤4.5% and bootstrap ≥90%<a href="#b2">2</a>. Variable cluster definitions are a common problem in the literature. Bootstraps from 70% to 99% are used, in combination with GD from 1.5% to 4.5%. Another reason for the disparity could be differences in sampling procedures. However it is also possible that the contact and transmission processes between Switzerland and the UK differ. Given these observed differences, it is unclear to what extent findings from one country can be applied to the other, and more importantly whether results from either can be extended to other less densely sampled European epidemics.</p> <p>Because access to data from national cohorts is subject to restrictions, and because analyses have been conducted according to in-house bioinformatics pipelines, the differences between the two epidemics have never been elucidated. Here, we present an analysis conducted in parallel on the two epidemics using a standardised approach. We hypothesised that because of its geography, the Swiss epidemic might be more integrated into the European epidemic and less clustered because of unsampled links in Swiss transmission chains. Using the same cluster definition, we compared the cluster distributions (the number and sizes of clusters) of the two countries as an indicator of underlying epidemic dynamics. We determined whether clustering was affected by risk group and ethnicity and compared the degree distributions (the number of linked partners) of heterosexual, MSM and PWID to test whether differences between countries were down to any specific risk group. Finally, we tested whether the UK and Swiss epidemics intermingled with the same foreign countries through sequence analysis.</p> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="baselinedemographics" title="sec"> <div class="title" tagx="title" title="title">
Baseline demographics</div> <p>HIV <i>pol</i> sequences were retrieved from the SHCS drug resistance database<a href="#b18">18</a> (SHCS DRDB, 2014) and from the UK HIV RDB (sequences up to mid-2013). Closely related sequences were obtained from the Los Alamos National Laboratory (LANL) database as described in the Methods. In the final analyses, the Swiss datasets contained 1,374 subtype A1 sequences (435 Swiss, 939 LANL), 15,043 B (8,390 Swiss, 6,653 LANL) and 1,571 C (419 Swiss, 1,152 LANL). The UK datasets comprised 4,421 A1 (2,507 UK, 1,914 LANL), 38,863 B (31,450 UK, 7,413 LANL) and 22,027 C (15,815 UK, 6,212 LANL). The Swiss epidemic comprised more PWID, fewer heterosexuals and more patients of White ethnicity (<a href="#t1">Table 1</a>). These differences were in part a result of the different subtype composition across the two countries, but even within subtype B there were notable differences, with proportionally more cases among heterosexuals and PWID in Switzerland.</p> <p>In both countries individuals for whom sequence and epidemiological data were available broadly matched the characteristics of the HIV diagnosed population as a whole, in terms of risk group, sex, ethnicity and age distribution<a href="#b19">19</a><a href="#b20">20</a>. Swiss sequence dates go back to 1995 owing to retrospective sequencing of samples from the SHCS Bio-bank<a href="#b21">21</a>.</p> </div> <div class="differenceinclustering" title="sec"> <div class="title" tagx="title" title="title">
Difference in clustering</div> <p>Maximum likelihood phylogenetic trees were constructed and clusters were selected<a href="#b22">22</a> at a range of genetic distance (GD) and bootstrap thresholds. A significantly larger proportion of UK subtype B sequences clustered in the uncorrected analysis at all thresholds (<a href="#S1">Supplementary Table 1</a>, <a href="#f1">Fig. 1</a>). Concordantly, the UK was more clustered in the univariate analysis (<a href="#t2">Table 2</a>) with odds for being in a cluster 4 times higher at 4.5% GD. At 4.5% GD, subtypes C and A1 were also more clustered in the UK, but there was no significant difference at 1.5% GD. The subsequent analysis focused mainly on subtype B.</p> <p>Because of the difference in sampling time distributions and demographics between the two subtype B datasets, we considered the logistic regression adjusting for those variables. More recent samples were much more likely to cluster than older samples (<a href="#t3">Table 3</a>) and when the model was adjusted for sample date, the Swiss epidemic was more clustered than the UK epidemic at 1.5% GD. At 4.5% GD, clustering remained higher for the UK but the strength of the association was halved compared with the univariate model. The effect of risk group and ethnicity on clustering was consistent across the two countries, with MSM in both countries showing the highest propensity for clustering.</p> </div> <div class="degreedistributions" title="sec"> <div class="title" tagx="title" title="title">
Degree Distributions</div> <p>Degree distributions for each country and risk group were generated based on cluster size distributions and compositions. For example a cluster containing 3 heterosexuals is equivalent to 3 heterosexuals each with degree 2. Statistical frameworks exist to formally compare degree distributions<a href="#b23">23</a><a href="#b24">24</a><a href="#b25">25</a> and include bootstrapping to simulate the effect of the sampling process and test the robustness of conclusions. Degree distributions for the populations as a whole and for each risk group were compared using the Kolmogorov-Smirnov (KS) tests<a href="#b24">24</a> and the Degree Distribution Quantification and Comparison (DDQC) algorithm<a href="#b23">23</a>. Based on the KS test, there was no difference between the two countries at 1.5% GD. At 4.5% GD, distributions differed significantly for HET, MSM and the population as a whole, but not for PWID (<a href="#S1">Supplementary Table 2</a>). The difference appeared to be driven by the longer tail of the UK distributions (<a href="#f2">Fig. 2</a>), indicating the existence of larger clusters in the UK. The UK epidemic thus comprised not only a higher proportion of sequences in clusters, but also clusters were larger.</p> <p>Because the KS test is sensitive to network size, we applied the DDQC, which is robust to differences in scale. The DDQC measures the distance between networks based on features extracted from their degree distributions. However, it does not indicate whether distances calculated are significant or not. In order to generate null distributions, the UK and Swiss degree distributions were compared to themselves through bootstrapping. The UK and Swiss degree distributions were each bootstrapped 100 times to simulate the effect of sampling (see Methods) and the DDQC distance calculated between the true data and each bootstrap replicate (<a href="#f3">Fig. 3</a>). Between country DDQC values were considered significant if they exceeded the 95% percentile of the within country DDQC values (one-sided test). The DDQC distance was higher than expected only for heterosexuals at 4.5% GD (95<sup>th</sup> percentile DDQC = 0.97, observed HET DDQC = 1.22).</p> <p>We hypothesised that the difference highlighted by the KS test at 4.5% might be the result of a difference in scale between the two epidemics. The UK population (and the HIV+ population) is much larger than that of Switzerland and so the pool of partners available is bigger. To examine the effect of epidemic size on clustering and degree distributions, we down-sampled the UK subtype B datasets to match the size of the Swiss datasets. In parallel, the Swiss datasets were bootstrap sampled with replacement. When these equal-sized resampled datasets were compared, the UK and Swiss degree distributions overlapped for the population as a whole and for the MSM population, but not for heterosexuals (<a href="#f4">Fig. 4</a>).</p> <p>In the true and the jack-knife sampled UK heterosexual population, we observed male heterosexuals with high degree (&amp;gt;20) not present in the Swiss data (<a href="#f4">Fig. 4</a>). The largest exclusively heterosexual cluster in the UK comprised 27 individuals (bootstrap = 0.9, GD = 4.5%); all heterosexuals with higher degree were in clusters dominated by MSM. When we dropped heterosexuals with degree &amp;gt;26 from the UK sample (125 individuals out of a total of 1556 heterosexuals), the DDQC distance between the two networks fell within its null distribution (DDQC = 0.17, <a href="#f3">Fig. 3</a>).</p> </div> <div class="crossbordertransmission" title="sec"> <div class="title" tagx="title" title="title">
Cross border transmission</div> <p>We investigated intermingling between national and foreign sequences by removing the 80% national criterion. We used a tight GD threshold (1.5%, 70% bootstrap) to capture close transmission partners. At this threshold, Swiss sequences clustered with 162 non-Swiss sequences and UK sequences clustered with 353 non-UK sequences. For Switzerland, Western European countries provided over 75% of the links. For the UK, 50% of close links were with other European countries and 20% originated from other Anglophone countries: Australia, Canada and the USA (<a href="#S1">Supplementary Figure 1</a>).</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The aim of this study was to compare epidemic dynamics between the two most densely sampled HIV epidemics, the UK and Switzerland, while adhering to data governance procedures and privacy protection requirements. We found that the fraction of sequences in transmission clusters was similar between the UK and Switzerland for a strict GD threshold (1.5%) but that they differed at a more relaxed GD threshold (4.5%). This suggests that the two epidemics resemble each other at a micro-level but differ at a macro-level. Because a statistical framework for comparing cluster distributions directly is lacking, we generated degree distributions based on cluster sizes and compared them through formal statistical tests: the KS test, the DDQC and bootstrapping. Based on the KS test, there were differences between the UK and Swiss subtype B degree distributions at 4.5% GD. However, downsampling the UK dataset to the size of the Swiss dataset rendered this difference insignificant in MSM and the population as a whole, but not heterosexuals (<a href="#f4">Fig. 4</a>). In parallel, only heterosexuals showed a significant difference based on the DDQC test, which corrects for network size.</p> <p>The degree distribution of UK heterosexuals had a long tail representing male heterosexuals clustered exclusively with MSM. Previous UK analyses have demonstrated that a proportion of self-reported male heterosexuals are likely to have been infected through sex with men<a href="#b26">26</a>, which appears to be the case here. When those high degree heterosexuals were removed from the dataset, the UK and Swiss no longer differed. Male heterosexuals who have sex with men are likely to also have sex with women and provide a bridge between MSM and heterosexual epidemics. This is a likely route for the spread of non-B subtypes among MSM in the UK<a href="#b4">4</a>. More detailed analyses of the Swiss epidemic have found little overlap between the MSM and HET epidemics<a href="#b3">3</a>; however, the Swiss heterosexual with the highest degree was similarly part of a HET/MSM cluster comprising 36 individuals, while the largest exclusively heterosexual cluster contained only 9. In fact, 47% of UK and 38% of Swiss heterosexuals were in HET/MSM clusters. The present analysis cannot determine whether bridging is more common in the UK or whether risk group classification is assessed more thoroughly in Switzerland.</p> <p>There was more overlap between PWID and heterosexuals in Switzerland than in the UK (23% vs 12%), but the difference in PWID degree distributions was not significant. Although this could be due to sample size, the stemming of the heterosexual epidemic through PWID in Switzerland is likely to be an old process<a href="#b3">3</a> while the bridging between HET and MSM in the UK is ongoing<a href="#b4">4</a><a href="#b26">26</a>.</p> <p>Our findings were consistent across bootstrap thresholds. At 1.5% GD, we found no difference between the degree distributions of the Swiss and UK epidemics. At tight thresholds mostly pairs and recently infected patients are captured and these groupings are similar across the two countries. At 4.5% GD, the UK was more clustered and so the UK HIV RDB is more likely to capture larger transmission chains. However, the downsampled UK epidemic degree distribution overlapped with the Swiss degree distribution. While the proportion of clustered individuals in the UK is higher, the difference is seemingly due to the UK greater epidemic size rather than because of differences in contact or transmission processes. Both countries are similarly integrated into global unsampled epidemics, and this study underlines the importance of HIV public health interventions at the European and global levels.</p> <p>Transmission between European countries has been analysed in more depth elsewhere<a href="#b27">27</a>. In agreement with that analysis, we found Spain to be a major mixing partner for both Switzerland and the UK. Germany and the Czech Republic were also identified as significant. We found increased linkage between the UK and other Anglophone countries. In Switzerland strong segregation has been observed between German and French-speaking regions<a href="#b3">3</a> and this language-dependency of HIV transmission warrants investigation at the global scale.</p> <p>Both countries have noted the subtype diversification of their respective epidemics<a href="#b9">9</a><a href="#b28">28</a>, yet the difference in size between the UK and Swiss subtype A1 and C datasets (18,000 vs 900, respectively) rendered a comparison meaningless. In Switzerland fewer than 25% of non-B infections were acquired in the country<a href="#b5">5</a>, whereas in the UK over 50% of infections in individuals born abroad are thought to have occurred in the UK<a href="#b11">11</a>. Local non-B non-heterosexual transmission appears far more extensive in the UK<a href="#b4">4</a>.</p> <p>Although degree distributions are a blunt tool for elucidating the dynamics of an epidemic<a href="#b29">29</a>, they allowed us to apply statistically robust methods to compare the two epidemics without the need for exchanging sensitive data. Sequences from national databases were never exchanged and while this precluded a combined phylogenetic analysis of UK and Swiss sequences, one of the strengths of the study stems from undertaking such an analysis without compromising patient privacy. A second issue, the distributions of sample dates differing between the two cohorts, arose because the SHCS has conducted extensive retrospective sequencing on patients diagnosed early on in the epidemic and for whom samples had been stored. The SHCS coverage of older samples explains in part the lower clustering observed in Switzerland. However, clustering remained significantly higher in the UK at 4.5% GD after sample date was adjusted for. Thirdly, international comparisons were based on the LANL database which is in essence a large-scale convenience sample and not necessarily representative. We suggest the apparently important contribution of the Czech Republic to both epidemics may arise from recent submission of large numbers of sequences from that country.</p> <p>In conclusion, we showed that apparent major differences in clustering patterns between the UK and Switzerland subtype B epidemics can be explained for the most part by differences in size and sampling time. This is the first study leveraging the vast amounts of data available in multiple national HIV databases. We made use of data without breaching data governance procedures and highlighted that transmission trends in these two countries are driven by similar underlying factors. From a methodological perspective, our study highlights the importance of using the same cluster-detection algorithm and correcting for demographic factors when comparing clustering patterns across settings.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="data" title="sec"> <div class="title" tagx="title" title="title">
Data</div> <div class="switzerland" title="sec"> <div class="title" tagx="title" title="title">
Switzerland</div> <p>9,232 HIV <i>pol</i> sequences were retrieved from the SHCS DRDB. The SHCS DRDB aggregates all HIV resistance tests for patients of the SHCS. SmartGene is responsible for data storage and management (<a href="http://www.smartgene.com">http://www.smartgene.com</a>). The DRDB is part of the SHCS, which is an ongoing national clinical cohort of HIV patients aged 16 and above with biannual follow up (<a href="http://www.shcs.ch">http://www.shcs.ch</a>)<a href="#b20">20</a>. Sequences were assigned subtypes using REGA<a href="#b30">30</a><a href="#b31">31</a>; subtypes B (91%), A1 (5%), and C (5%) were analysed. The SHCS has been approved by the ethics committees of all participating institutions, and written, informed consent has been obtained from participants.</p> </div> <div class="uk" title="sec"> <div class="title" tagx="title" title="title">
UK</div> <p>63,065 HIV <i>pol</i> sequences were obtained from the UK HIV RDB (<a href="http://www.hivrdb.org">http://www.hivrdb.org</a>). Sequences cover protease and 900 bases of reverse transcriptase. After age, sex, risk group and ethnicity were attached, the data were anonymised and delinked. Subtypes were assigned using SCUEAL<a href="#b32">32</a> and REGA<a href="#b30">30</a><a href="#b31">31</a>. The three most common subtypes were B (48.6%), C (24.4%), and A1 (4.0%). Ethical approval was given by the London Multicentre Research Ethics Committee (MREC/01/2/10; 5 April 2001).</p> <p>As submission of the UK and Swiss sequence datasets to public databases would permit transmission network identification and thus risk breaching patient confidentiality, we have followed earlier practice<a href="#b3">3</a><a href="#b33">33</a>. A random sample of 10% of each subtype and country has been submitted to Genbank (accession numbers available in <a href="#S1">supplementary material</a>).</p> </div> <div class="backgroundsequences" title="sec"> <div class="title" tagx="title" title="title">
Background sequences</div> <p>All <i>pol</i> (HXB2 positions 2253–3870) sequences of HIV subtype A1, B, and C longer than 900 bases were retrieved from LANL (January 2014). To limit the size of alignments, the ten closest sequences to each of the local (UK and Swiss) sequences were selected using Viroblast<a href="#b34">34</a>. For this step, UK sequences were removed from LANL alignments before the UK Viroblast run, and Swiss LANL sequences were removed before the Swiss run.</p> <p>Only the earliest available sequence for each individual was used. All sequences were stripped of 44 sites associated with drug resistance based on the 2013 International AIDS Society list<a href="#b35">35</a>.</p> </div> </div> <div class="treebuildingandclusterpicking" title="sec"> <div class="title" tagx="title" title="title">
Tree Building and Cluster Picking</div> <p>Duplicate sequences were removed. Maximum likelihood phylogenetic trees were constructed for each country and subtype separately (six trees in total) using FastTree v2.0<a href="#b36">36</a> with 100 bootstraps. Initially clusters were selected for further analysis if they were supported by bootstrap thresholds of 70%, 80%, 90% and 95% and maximum GD of 1.5% or 4.5% (8 thresholds total)<a href="#b22">22</a>. Of the initially identified clusters, those in the Swiss trees were further selected to contain at least 80% SHCS sequences, and clusters in the UK trees at least 80% UK sequences. In a separate analysis, we examined all clusters with at least one UK or Swiss sequence (within the respective datasets) to investigate mixing between national and foreign sequences. The automated pipeline included analysis with the Cluster Picker and Cluster Matcher<a href="#b22">22</a> as well as processing through python and R scripts (available upon request). The Cluster Picker was upgraded to recognise IUPAC nucleotide ambiguity codes as matches (version available upon request from the authors), increasing clustering by around 15% in both datasets.</p> <p>From the Cluster Picker and Cluster Matcher output files, we generated degree distributions (the number of links for each node). As files contained risk group composition for each cluster, it was possible to break down degree distribution by risk group. Nodes were sampled with replacement from the network with information on their cluster membership. Nodes sampled with the same cluster membership were linked together in each bootstrapped network, so that clusters sometimes increased in size, sometimes decreased in size or otherwise disappeared, and degree distribution was re-estimated each time. Jack-knife resampling where the number of nodes sampled was smaller than the full network size was also performed.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>The number of sequences clustering at different thresholds between the two epidemics was compared using Fisher’s exact test with Bonferroni correction (24 comparisons across clustering thresholds and subtypes). Degree distributions were compared using the KS test<a href="#b24">24</a> and the DDQC algorithm<a href="#b23">23</a>. The KS test is a nonparametric test which compares the cumulative distribution of two samples to estimate whether they have been drawn from the same distribution and is frequently used to compare degree distributions<a href="#b25">25</a>. The DDQC was developed specifically to compare degree distributions and corrects for differences in population size while the KS test does not. The DDQC extracts a vector of eight values from degree distributions for comparison. In brief, the range of node degrees is divided into eight regions based on the minimum, maximum, mean and standard deviation of the degree distribution. The probability of the degree of any node being contained within each interval is calculated. The distance between two networks is the sum of the absolute differences for each of the eight features extracted.</p> <p>The UK subtype B dataset used here was 3.75 times larger than the Swiss dataset; the UK MSM dataset was 5.7 times larger and the UK heterosexual dataset was 1.3 times larger (<a href="#t1">Table 1</a>). To investigate the effect of the difference in size of the pool of possible infectors, the UK dataset was jack-knife sampled to the size of the Swiss dataset. One hundred jack-knife replicates were generated, and in each replicate the degree distribution was re-estimated based on the links present in the sample.</p> <p>A logistic regression model was used to characterise the factors influencing clustering in the two countries. The model was applied with cluster membership as the outcome variable and with the country of origin (UK or Switzerland) as the main exposure variable. Sampling dates, risk group, sex, and ethnicity were adjusted for. Statistical analyses were conducted in R<a href="#b37">37</a>.</p> </div> </div> <div class="additionalinformation" title="sec"> <div class="title" tagx="title" title="title">
Additional Information</div> <p><b>How to cite this article</b>: Ragonnet-Cronin, M. L. <i>et al</i>. A Direct Comparison of Two Densely Sampled HIV Epidemics: The UK and Switzerland. <i>Sci. Rep.</i><b>6</b>, 32251; doi: 10.1038/srep32251 (2016).</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Information</div>  <div class="media" title="media"><a href="srep32251-s1.pdf">LINK</a></div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Anna van Weringh for reviewing the manuscript. We wish to acknowledge the contributions and support of the members of the Swiss HIV Cohort Study and the UK HIV Drug Resistance Database. Members of the Swiss HIV Cohort study and of the UK HIV Drug Resistance Database are provided as <a href="#S1">supplementary information</a>. This work was supported through the PANGEA-HIV Consortium with support provided by the Bill &amp;amp; Melinda Gates Foundation and by NIH GM110749, by the Wellcome Trust (SJL, Grant number 092807) and the Biotechnology and Biological Science Research Council (EH and MRC, Grant number BB/F017030/1). The UK HIV Drug Resistance Database is supported by the Medical Research Council (grant number G0900274) and is partly funded by the Department of Health; the views expressed in the publication are those of the authors and not necessarily those of the Department of Health. Additional support for the UK HIV RDB is provided by Boehringer Ingelheim, Bristol-Myers Squibb, Gilead, Tibotec (a division of Janssen-Cilag) and Roche. The Swiss HIV Cohort Study is supported by the Swiss National Science Foundation (SNF grant #33CS30-134277) and the SHCS Drug Resistance Database was supported by SHCS projects #470, 528, 569, 683, 752 the SHCS Research Foundation, the Swiss National Science Foundation (grant #-159868 (to HFG), the European Community’s Seventh Framework Program (grant FP7/2007–2013), under the Collaborative HIV and Anti-HIV Drug-resistance Network (CHAIN; grant 223131, to HFG), by the Yvonne-Jacob foundation, and by a further research grant of the Union Bank of Switzerland, in the name of an anonymous donor to HFG, an unrestricted research grant from Gilead, Switzerland to the SHCS research foundation, and by the University of Zurich’s Clinical research Priority Program (CRPP) "Viral infectious diseases: Zurich Primary HIV Infection Study�? (to HFG). RK was supported by SNF #PZ00P3-142411. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="b1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular phylodynamics of the heterosexual HIV epidemic in the United Kingdom</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="volume" tagx="volume" title="volume">5</span>(9), <span class="fpage" tagx="fpage" title="fpage">e1000590</span> (<span class="year" tagx="year" title="year">2009</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19779560">19779560</a></span></span></li> <li tag="ref"><a name="b2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pozniak</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leigh Brown</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Episodic sexual transmission of HIV revealed by molecular phylodynamics</span>. <span class="source" tagx="source" title="source">PLoS. Med.</span><span class="volume" tagx="volume" title="volume">5</span>(3), <span class="fpage" tagx="fpage" title="fpage">e50</span> (<span class="year" tagx="year" title="year">2008</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18351795">18351795</a></span></span></li> <li tag="ref"><a name="b3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">R. D.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology reveals long-term changes in HIV type 1 subtype B transmission in Switzerland</span>. <span class="source" tagx="source" title="source">J Infect Dis</span><span class="volume" tagx="volume" title="volume">201</span>(10), <span class="fpage" tagx="fpage" title="fpage">1488</span> (<span class="year" tagx="year" title="year">2010</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20384495">20384495</a></span></span></li> <li tag="ref"><a name="b4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragonnet-Cronin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Transmission of Non-B HIV Subtypes in the United Kingdom Is Increasingly Driven by Large Non-Heterosexual Transmission Clusters</span>. <span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">213</span>(9), <span class="fpage" tagx="fpage" title="fpage">1410</span> (<span class="year" tagx="year" title="year">2016</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26704616">26704616</a></span></span></li> <li tag="ref"><a name="b5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Wyl</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">The role of migration and domestic transmission in the spread of HIV-1 non-B subtypes in Switzerland</span>. <span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">204</span>(7), <span class="fpage" tagx="fpage" title="fpage">1095</span> (<span class="year" tagx="year" title="year">2011</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21881125">21881125</a></span></span></li> <li tag="ref"><a name="b6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinching</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Studies of cellular immunity in male homosexuals in London</span>. <span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">2</span>(8342), <span class="fpage" tagx="fpage" title="fpage">126</span> (<span class="year" tagx="year" title="year">1983</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6134980">6134980</a></span></span></li> <li tag="ref"><a name="b7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomson</span><span class="given-names" tagx="given-names" title="given-names">M. M.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Najera</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="mixed-article-title" title="mixed-article-title">Increasing HIV-1 genetic diversity in Europe</span>. <span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">196</span>(8), <span class="fpage" tagx="fpage" title="fpage">1120</span> (<span class="year" tagx="year" title="year">2007</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17955428">17955428</a></span></span></li> <li tag="ref"><a name="b8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hue</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pillay</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clewley</span><span class="given-names" tagx="given-names" title="given-names">J. P.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O. G.</span></span><span class="mixed-article-title" title="mixed-article-title">Genetic analysis reveals the complex structure of HIV-1 transmission within defined risk groups</span>. <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="volume" tagx="volume" title="volume">102</span>(12), <span class="fpage" tagx="fpage" title="fpage">4425</span> (<span class="year" tagx="year" title="year">2005</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15767575">15767575</a></span></span></li> <li tag="ref"><a name="b9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">The UK Collaborative Group on HIV Drug Resistance The increasing genetic diversity of HIV-1 in the UK, 2002–2010. <span class="source" tagx="source" title="source">AIDS</span><span class="volume" tagx="volume" title="volume">28</span>(5), <span class="fpage" tagx="fpage" title="fpage">773</span> (<span class="year" tagx="year" title="year">2014</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24257094">24257094</a></span></span></li> <li tag="ref"><a name="b10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aggarwal</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Evidence for onward transmission of HIV-1 non-B subtype strains in the United Kingdom</span>. <span class="source" tagx="source" title="source">J. Acquir. Immune. Defic. Syndr.</span><span class="volume" tagx="volume" title="volume">41</span>(2), <span class="fpage" tagx="fpage" title="fpage">201</span> (<span class="year" tagx="year" title="year">2006</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16394853">16394853</a></span></span></li> <li tag="ref"><a name="b11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rice</span><span class="given-names" tagx="given-names" title="given-names">B. D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elford</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delpech</span><span class="given-names" tagx="given-names" title="given-names">V. C.</span></span><span class="mixed-article-title" title="mixed-article-title">A new method to assign country of HIV infection among heterosexuals born abroad and diagnosed with HIV</span>. <span class="source" tagx="source" title="source">AIDS</span><span class="volume" tagx="volume" title="volume">26</span>(15), <span class="fpage" tagx="fpage" title="fpage">1961</span> (<span class="year" tagx="year" title="year">2012</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22781226">22781226</a></span></span></li> <li tag="ref"><a name="b12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span>. HIV in the United Kingdom: 2014 Report. [Report] (2014).</span></li> <li tag="ref"><a name="b13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Impact on life expectancy of HIV-1 positive individuals of CD4+ cell count and viral load response to antiretroviral therapy</span>. <span class="source" tagx="source" title="source">AIDS</span><span class="volume" tagx="volume" title="volume">28</span>(8), <span class="fpage" tagx="fpage" title="fpage">1193</span> (<span class="year" tagx="year" title="year">2014</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24556869">24556869</a></span></span></li> <li tag="ref"><a name="b14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">EuroHIV: European Centre for the Epidemiological Monitoring of AIDS &amp;amp; UNAIDS. HIV/AIDS Surveillance in Europe [Report] (1999).</span></li> <li tag="ref"><a name="b15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Csete</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grob</span><span class="given-names" tagx="given-names" title="given-names">P. J.</span></span><span class="mixed-article-title" title="mixed-article-title">Switzerland, HIV and the power of pragmatism: lessons for drug policy development</span>. <span class="source" tagx="source" title="source">Int. J. Drug Policy</span><span class="volume" tagx="volume" title="volume">23</span>(1), <span class="fpage" tagx="fpage" title="fpage">82</span> (<span class="year" tagx="year" title="year">2012</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21852097">21852097</a></span></span></li> <li tag="ref"><a name="b16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Estimating the basic reproductive number from viral sequence data</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">29</span>(1), <span class="fpage" tagx="fpage" title="fpage">347</span> (<span class="year" tagx="year" title="year">2012</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21890480">21890480</a></span></span></li> <li tag="ref"><a name="b17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Nationales Programm HIV und ande-re sexuell übertragbare Infektionen 2011–2017 (NPHS). HIV- und STI-Fallzahlen 2012: Berichterstattung, Analysen und Trends [Report] (2013).</span></li> <li tag="ref"><a name="b18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Wyl</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Emergence of HIV-1 drug resistance in previously untreated patients initiating combination antiretroviral treatment: a comparison of different regimen types</span>. <span class="source" tagx="source" title="source">Arch. Intern. Med.</span><span class="volume" tagx="volume" title="volume">167</span>(16), <span class="fpage" tagx="fpage" title="fpage">1782</span> (<span class="year" tagx="year" title="year">2007</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846398">17846398</a></span></span></li> <li tag="ref"><a name="b19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aghaizu</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">A. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nardone</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gill</span><span class="given-names" tagx="given-names" title="given-names">O. N.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delpech</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span> HIV in the United Kingdom: 2013 Report. [Report] (2013).</span></li> <li tag="ref"><a name="b20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoeni-Affolter</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Cohort profile: the Swiss HIV Cohort study</span>. <span class="source" tagx="source" title="source">Int. J. Epidemiol.</span><span class="volume" tagx="volume" title="volume">39</span>(5), <span class="fpage" tagx="fpage" title="fpage">1179</span> (<span class="year" tagx="year" title="year">2010</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19948780">19948780</a></span></span></li> <li tag="ref"><a name="b21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">W. L.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Assessing the Paradox Between Transmitted and Acquired HIV Type 1 Drug Resistance Mutations in the Swiss HIV Cohort Study From 1998 to 2012</span>. <span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">212</span>(1), <span class="fpage" tagx="fpage" title="fpage">28</span> (<span class="year" tagx="year" title="year">2015</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25576600">25576600</a></span></span></li> <li tag="ref"><a name="b22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragonnet-Cronin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Automated analysis of phylogenetic clusters</span>. <span class="source" tagx="source" title="source">BMC. Bioinformatics</span><span class="volume" tagx="volume" title="volume">14</span>(1), <span class="fpage" tagx="fpage" title="fpage">317</span> (<span class="year" tagx="year" title="year">2013</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24191891">24191891</a></span></span></li> <li tag="ref"><a name="b23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aliakbary</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Habibi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Movaghar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Feature Extraction from Degree Distribution for Comparison and Analysis of Complex Networks</span>. <span class="source" tagx="source" title="source">Comput. J.</span><span class="volume" tagx="volume" title="volume">12</span> (<span class="year" tagx="year" title="year">2015</span>).</span></li> <li tag="ref"><a name="b24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massey</span><span class="given-names" tagx="given-names" title="given-names">F. J.</span></span><span class="mixed-article-title" title="mixed-article-title">The Kolmogorov-Smirnov Test for Goodness of Fit</span>. <span class="source" tagx="source" title="source">Journal of the American Statistical Association</span><span class="volume" tagx="volume" title="volume">46</span>(253), <span class="fpage" tagx="fpage" title="fpage">68</span> (<span class="year" tagx="year" title="year">1951</span>).</span></li> <li tag="ref"><a name="b25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Latapy</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Soria</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Deciding on the type of the degree distribution of a graph from traceroute-like measurements</span>. <span class="source" tagx="source" title="source">International Journal of Computer Networks &amp;amp; Communications</span><span class="volume" tagx="volume" title="volume">4</span>(3), <span class="fpage" tagx="fpage" title="fpage">151</span> (<span class="year" tagx="year" title="year">2012</span>).</span></li> <li tag="ref"><a name="b26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hue</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analyses reveal HIV-1 infections between men misclassified as heterosexual transmissions</span>. <span class="source" tagx="source" title="source">AIDS</span><span class="volume" tagx="volume" title="volume">28</span>(13), <span class="fpage" tagx="fpage" title="fpage">1967</span> (<span class="year" tagx="year" title="year">2014</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24991999">24991999</a></span></span></li> <li tag="ref"><a name="b27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraskevis</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Tracing the HIV-1 subtype B mobility in Europe: a phylogeographic approach</span>. <span class="source" tagx="source" title="source">Retrovirology</span>. <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">49</span> (<span class="year" tagx="year" title="year">2009</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19457244">19457244</a></span></span></li> <li tag="ref"><a name="b28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neogi</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Temporal trends in the Swedish HIV-1 epidemic: increase in non-B subtypes and recombinant forms over three decades</span>. <span class="source" tagx="source" title="source">PLoS. One.</span><span class="volume" tagx="volume" title="volume">9</span>(6), <span class="fpage" tagx="fpage" title="fpage">e99390</span> (<span class="year" tagx="year" title="year">2014</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24922326">24922326</a></span></span></li> <li tag="ref"><a name="b29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamilton</span><span class="given-names" tagx="given-names" title="given-names">D. T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handcock</span><span class="given-names" tagx="given-names" title="given-names">M. S.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">Degree distributions in sexual networks: a framework for evaluating evidence</span>. <span class="source" tagx="source" title="source">Sex Transm. Dis.</span><span class="volume" tagx="volume" title="volume">35</span>(1), <span class="fpage" tagx="fpage" title="fpage">30</span> (<span class="year" tagx="year" title="year">2008</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18217224">18217224</a></span></span></li> <li tag="ref"><a name="b30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alcantara</span><span class="given-names" tagx="given-names" title="given-names">L. C.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">A standardized framework for accurate, high-throughput genotyping of recombinant and non-recombinant viral sequences</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="volume" tagx="volume" title="volume">37</span>, (Web Server issue), <span class="fpage" tagx="fpage" title="fpage">W634</span>–<span class="lpage" tagx="lpage" title="lpage">W642</span> (<span class="year" tagx="year" title="year">2009</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19483099">19483099</a></span></span></li> <li tag="ref"><a name="b31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">An automated genotyping system for analysis of HIV-1 and other microbial sequences</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="volume" tagx="volume" title="volume">21</span>(19), <span class="fpage" tagx="fpage" title="fpage">3797</span> (<span class="year" tagx="year" title="year">2005</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16076886">16076886</a></span></span></li> <li tag="ref"><a name="b32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosakovsky Pond</span><span class="given-names" tagx="given-names" title="given-names">S. L.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">An evolutionary model-based algorithm for accurate phylogenetic breakpoint mapping and subtype prediction in HIV-1</span>. <span class="source" tagx="source" title="source">PLoS. Comput. Biol.</span><span class="volume" tagx="volume" title="volume">5</span>(11), <span class="fpage" tagx="fpage" title="fpage">e1000581</span> (<span class="year" tagx="year" title="year">2009</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19956739">19956739</a></span></span></li> <li tag="ref"><a name="b33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leigh Brown</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Transmission network parameters estimated from HIV sequences for a nationwide epidemic</span>. <span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">204</span>(9), <span class="fpage" tagx="fpage" title="fpage">1463</span> (<span class="year" tagx="year" title="year">2011</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21921202">21921202</a></span></span></li> <li tag="ref"><a name="b34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nickle</span><span class="given-names" tagx="given-names" title="given-names">D. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Learn</span><span class="given-names" tagx="given-names" title="given-names">G. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maust</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mullins</span><span class="given-names" tagx="given-names" title="given-names">J. I.</span></span><span class="mixed-article-title" title="mixed-article-title">ViroBLAST: a stand-alone BLAST web server for flexible queries of multiple databases and user’s datasets</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="volume" tagx="volume" title="volume">23</span>(17), <span class="fpage" tagx="fpage" title="fpage">2334</span> (<span class="year" tagx="year" title="year">2007</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586542">17586542</a></span></span></li> <li tag="ref"><a name="b35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">V. A.</span></span><span class="etal" title="etal"><i>et al.</i></span>. <span class="mixed-article-title" title="mixed-article-title">Update of the drug resistance mutations in HIV-1: March 2013. <i>Top</i></span>. <span class="source" tagx="source" title="source">Antivir. Med.</span><span class="volume" tagx="volume" title="volume">21</span>(1), <span class="fpage" tagx="fpage" title="fpage">6</span> (<span class="year" tagx="year" title="year">2013</span>).</span></li> <li tag="ref"><a name="b36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">M. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehal</span><span class="given-names" tagx="given-names" title="given-names">P. S.</span></span> &amp;amp; <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkin</span><span class="given-names" tagx="given-names" title="given-names">A. P.</span></span><span class="mixed-article-title" title="mixed-article-title">FastTree 2–approximately maximum-likelihood trees for large alignments</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">5</span>(3), <span class="fpage" tagx="fpage" title="fpage">e9490</span> (<span class="year" tagx="year" title="year">2010</span>).<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a></span></span></li> <li tag="ref"><a name="b37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">R Development Core Team. <span class="source" tagx="source" title="source">R: A language and environment for statistical computing</span> (R Foundation for Statistical Computing, Vienna, Austria, <span class="year" tagx="year" title="year">2011</span>).</span></li> </ul> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Author Contributions</b> Conceived and designed the experiments: M.L.R.-C., M.S., E.B.H., R.K. and A.J.L.B. Performed the experiments: M.L.R.-C. and M.S. Analysed the data: M.L.R.-C., M.S., R.K. and A.J.L.B. Contributed reagents/materials/analysis tools: H.G., E.F., E.B.H., S.Y., S.L., D.D., V.D., A.J.L.B., V.A., T.K. and W.L.-Y. Wrote the paper: M.L.R.-C., M.S., R.K., H.G., A.J.L.B. All authors reviewed the paper.</p> </div> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Proportion of UK (pink) and Swiss (blue) sequences in clusters at different genetic distance (1.5% and 4.5%) and bootstrap (70%, 80%, 90%, 95%) thresholds.</div> <p>CH - Switzerland.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Degree distributions of the UK (pink) and Swiss (blue) subtype B epidemics.</div> <p>MSM - men who have sex with men, HET - heterosexuals. Cluster definition was: genetic distance = 4.5% and bootstrap = 90%. Note that the proportion of individuals of each degree is shown rather than the absolute number of individuals of each degree. The number of clustered individuals was much larger in the UK than in Switzerland.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
DDQC distances within and between countries.</div> <p>DDQC – Degree Distribution Quantification and Comparison, CH - Switzerland, MSM - men who have sex with men, HET - heterosexuals. PWID – people who inject drugs. Cluster definition was: genetic distance = 4.5% and bootstrap = 90%. In order to generate null distributions of the expected values for the DDQC, we bootstrapped the Swiss and UK distributions and calculated DDQC values comparing the original datasets to their bootstrap samples (in blue and pink). The top of the coloured bars represent the mean distance of within country comparisons and the whiskers represent the 95% percentiles. The DDQC distance was then calculated between the UK and Swiss degree distributions (black triangles). The distance between countries was considered significant if it exceeded the 95% percentile from the simulated values, which was the case only for HET at 4.5% GD (indicated by *). When we removed heterosexuals who were likely to have been infected through sex with men from the UK dataset, the DDQC distance between the UK and Swiss HET degree distributions fell within the simulated null distribution (orange triangle).</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
Jack-knife and bootstrap sampled degree distributions of the UK (pink) and Swiss (blue) epidemics.</div> <p>UK subtype B degree distribution for men who have sex with men (MSM), heterosexuals (HET) and the population as a whole (all) were jack-knife sampled 100 times to match the size of the Swiss epidemics (in light pink). The Swiss epidemic was bootstrapped 100 times to its full size (in light blue). Degree distributions are shown on a double-logged scale. Samples overlapped for MSM and the dataset as a whole, but not for HET. Where Swiss replicates cannot be seen they are covered by the UK replicates.</p>   </div> <div class="table-wrap_UNKNOWN" position="float" id="t1"><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Baseline demographics of the two datasets.</div>  <table frame="hsides" rules="groups" border="1"> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" align="left" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div>  <thead valign="bottom"> <tr> <th rowspan="3" align="left" valign="top" charoff="50">Sex</th> <th rowspan="3" align="left" valign="top" charoff="50">Female</th> <th colspan="3" align="center" valign="top" charoff="50">UK <hr /> </th> <th colspan="3" align="center" valign="top" charoff="50">CH <hr /> </th> </tr> <tr> <th align="center" valign="top" charoff="50">A1</th> <th align="center" valign="top" charoff="50">B</th> <th align="center" valign="top" charoff="50">C</th> <th align="center" valign="top" charoff="50">A1</th> <th align="center" valign="top" charoff="50">B</th> <th align="center" valign="top" charoff="50">C</th> </tr> <tr> <th align="center" valign="top" charoff="50">1442 (57.5%)</th> <th align="center" valign="top" charoff="50">2327 (7.4%)</th> <th align="center" valign="top" charoff="50">3869 (62.2%)</th> <th align="center" valign="top" charoff="50">269 (62%)</th> <th align="center" valign="top" charoff="50">1812 (22%)</th> <th align="center" valign="top" charoff="50">237 (57%)</th> </tr> </thead> <tbody valign="top"> <tr> <td rowspan="4" align="left" valign="middle" charoff="50">Risk</td> <td align="left" valign="top" charoff="50">MSM</td> <td align="center" valign="top" charoff="50">170 (6.8%)</td> <td align="center" valign="top" charoff="50">22157 (70.4%)</td> <td align="center" valign="top" charoff="50">660 (4.2%)</td> <td align="center" valign="top" charoff="50">21 (5%)</td> <td align="center" valign="top" charoff="50">3914 (47%)</td> <td align="center" valign="top" charoff="50">25 (6%)</td> </tr> <tr> <td align="left" valign="top" charoff="50">HET</td> <td align="center" valign="top" charoff="50">1718 (68.5%)</td> <td align="center" valign="top" charoff="50">3399 (10.8%)</td> <td align="center" valign="top" charoff="50">11893 (75.2%)</td> <td align="center" valign="top" charoff="50">366 (84%)</td> <td align="center" valign="top" charoff="50">2626 (31%)</td> <td align="center" valign="top" charoff="50">357 (85%)</td> </tr> <tr> <td align="left" valign="top" charoff="50">PWID</td> <td align="center" valign="top" charoff="50">92 (3.7%)</td> <td align="center" valign="top" charoff="50">873 (2.8%)</td> <td align="center" valign="top" charoff="50">128 (0.8%)</td> <td align="center" valign="top" charoff="50">12 (3%)</td> <td align="center" valign="top" charoff="50">1553 (19%)</td> <td align="center" valign="top" charoff="50">9 (2%)</td> </tr> <tr> <td align="left" valign="top" charoff="50">Other/NA</td> <td align="center" valign="top" charoff="50">527 (21%)</td> <td align="center" valign="top" charoff="50">5021 (16%)</td> <td align="center" valign="top" charoff="50">3134 (19.8%)</td> <td align="center" valign="top" charoff="50">36 (8%)</td> <td align="center" valign="top" charoff="50">287 (3%)</td> <td align="center" valign="top" charoff="50">28 (7%)</td> </tr> <tr> <td align="left" valign="top" charoff="50">Ethnicity</td> <td align="left" valign="top" charoff="50">White</td> <td align="center" valign="top" charoff="50">494 (19.7%)</td> <td align="center" valign="top" charoff="50">22724 (72.3%)</td> <td align="center" valign="top" charoff="50">1724 (10.9%)</td> <td align="center" valign="top" charoff="50">194 (45%)</td> <td align="center" valign="top" charoff="50">7333 (87%)</td> <td align="center" valign="top" charoff="50">118 (28%)</td> </tr> <tr> <td align="left" valign="top" charoff="50">Total</td> <td align="left" valign="top" charoff="50"> </td> <td align="center" valign="top" charoff="50">2507</td> <td align="center" valign="top" charoff="50">31450</td> <td align="center" valign="top" charoff="50">15815</td> <td align="center" valign="top" charoff="50">435</td> <td align="center" valign="top" charoff="50">8390</td> <td align="center" valign="top" charoff="50">419</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" position="float" id="t2"><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Unadjusted logistic regression for the comparison of the degree of patients clustering between the UK and Switzerland (Subtype B).</div>  <table frame="hsides" rules="groups" border="1"> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" align="left" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div>  <thead valign="bottom"> <tr> <th align="left" valign="top" charoff="50">Bootstrap</th> <th align="center" valign="top" charoff="50">Genetic Distance</th> <th align="center" valign="top" charoff="50">Covariates</th> <th align="center" valign="top" charoff="50">OR</th> <th align="center" valign="top" charoff="50">2.5%</th> <th align="center" valign="top" charoff="50">97.5%</th> </tr> </thead> <tbody valign="top"> <tr> <td align="left" valign="top" charoff="50">0.7</td> <td align="center" valign="top" charoff="50">0.015</td> <td align="center" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">1.316</td> <td align="center" valign="top" charoff="50">1.243</td> <td align="center" valign="top" charoff="50">1.394</td> </tr> <tr> <td align="left" valign="top" charoff="50">0.8</td> <td align="center" valign="top" charoff="50">0.015</td> <td align="center" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">1.335</td> <td align="center" valign="top" charoff="50">1.258</td> <td align="center" valign="top" charoff="50">1.418</td> </tr> <tr> <td align="left" valign="top" charoff="50">0.9</td> <td align="center" valign="top" charoff="50">0.015</td> <td align="center" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">1.396</td> <td align="center" valign="top" charoff="50">1.308</td> <td align="center" valign="top" charoff="50">1.492</td> </tr> <tr> <td align="left" valign="top" charoff="50">0.95</td> <td align="center" valign="top" charoff="50">0.015</td> <td align="center" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">1.444</td> <td align="center" valign="top" charoff="50">1.345</td> <td align="center" valign="top" charoff="50">1.551</td> </tr> <tr> <td align="left" valign="top" charoff="50">0.7</td> <td align="center" valign="top" charoff="50">0.045</td> <td align="center" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">5.001</td> <td align="center" valign="top" charoff="50">4.745</td> <td align="center" valign="top" charoff="50">5.272</td> </tr> <tr> <td align="left" valign="top" charoff="50">0.8</td> <td align="center" valign="top" charoff="50">0.045</td> <td align="center" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">4.119</td> <td align="center" valign="top" charoff="50">3.904</td> <td align="center" valign="top" charoff="50">4.347</td> </tr> <tr> <td align="left" valign="top" charoff="50">0.9</td> <td align="center" valign="top" charoff="50">0.045</td> <td align="center" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">3.942</td> <td align="center" valign="top" charoff="50">3.722</td> <td align="center" valign="top" charoff="50">4.176</td> </tr> <tr> <td align="left" valign="top" charoff="50">0.95</td> <td align="center" valign="top" charoff="50">0.045</td> <td align="center" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">3.176</td> <td align="center" valign="top" charoff="50">2.986</td> <td align="center" valign="top" charoff="50">3.381</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>OR: odds-ratio.</p> </div> </div> </div> <div class="table-wrap_UNKNOWN" position="float" id="t3"><span class="label" tagx="label" title="label">Table 3</span> <div class="title" tagx="title" title="title">
Logistic regression for the comparison of the degree of patients clustering between the UK and Switzerland, corrected for sample year (Subtype B).</div>  <table frame="hsides" rules="groups" border="1"> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" align="left" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="center" /></div>  <thead valign="bottom"> <tr> <th align="left" valign="top" charoff="50">Bootstrap</th> <th align="center" valign="top" charoff="50">Genetic Distance</th> <th align="left" valign="top" charoff="50">Covariates</th> <th align="center" valign="top" charoff="50">OR</th> <th align="center" valign="top" charoff="50">2.5%</th> <th align="center" valign="top" charoff="50">97.5%</th> </tr> </thead> <tbody valign="top"> <tr> <td rowspan="2" align="left" valign="middle" charoff="50">0.7</td> <td rowspan="2" align="center" valign="middle" charoff="50">0.015</td> <td align="left" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">0.66</td> <td align="center" valign="top" charoff="50">0.617</td> <td align="center" valign="top" charoff="50">0.705</td> </tr> <tr> <td align="left" valign="top" charoff="50">Sample year</td> <td align="center" valign="top" charoff="50">1.138</td> <td align="center" valign="top" charoff="50">1.131</td> <td align="center" valign="top" charoff="50">1.145</td> </tr> <tr> <td rowspan="2" align="left" valign="middle" charoff="50">0.8</td> <td rowspan="2" align="center" valign="middle" charoff="50">0.015</td> <td align="left" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">0.665</td> <td align="center" valign="top" charoff="50">0.621</td> <td align="center" valign="top" charoff="50">0.713</td> </tr> <tr> <td align="left" valign="top" charoff="50">Sample year</td> <td align="center" valign="top" charoff="50">1.142</td> <td align="center" valign="top" charoff="50">1.134</td> <td align="center" valign="top" charoff="50">1.149</td> </tr> <tr> <td rowspan="2" align="left" valign="middle" charoff="50">0.9</td> <td rowspan="2" align="center" valign="middle" charoff="50">0.015</td> <td align="left" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">0.694</td> <td align="center" valign="top" charoff="50">0.645</td> <td align="center" valign="top" charoff="50">0.749</td> </tr> <tr> <td align="left" valign="top" charoff="50">Sample year</td> <td align="center" valign="top" charoff="50">1.146</td> <td align="center" valign="top" charoff="50">1.138</td> <td align="center" valign="top" charoff="50">1.154</td> </tr> <tr> <td rowspan="2" align="left" valign="middle" charoff="50">0.95</td> <td rowspan="2" align="center" valign="middle" charoff="50">0.015</td> <td align="left" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">0.706</td> <td align="center" valign="top" charoff="50">0.651</td> <td align="center" valign="top" charoff="50">0.765</td> </tr> <tr> <td align="left" valign="top" charoff="50">Sample year</td> <td align="center" valign="top" charoff="50">1.153</td> <td align="center" valign="top" charoff="50">1.144</td> <td align="center" valign="top" charoff="50">1.162</td> </tr> <tr> <td rowspan="2" align="left" valign="middle" charoff="50">0.7</td> <td rowspan="2" align="center" valign="middle" charoff="50">0.045</td> <td align="left" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">2.692</td> <td align="center" valign="top" charoff="50">2.538</td> <td align="center" valign="top" charoff="50">2.856</td> </tr> <tr> <td align="left" valign="top" charoff="50">Sample year</td> <td align="center" valign="top" charoff="50">1.134</td> <td align="center" valign="top" charoff="50">1.128</td> <td align="center" valign="top" charoff="50">1.14</td> </tr> <tr> <td rowspan="2" align="left" valign="middle" charoff="50">0.8</td> <td rowspan="2" align="center" valign="middle" charoff="50">0.045</td> <td align="left" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">2.12</td> <td align="center" valign="top" charoff="50">1.996</td> <td align="center" valign="top" charoff="50">2.252</td> </tr> <tr> <td align="left" valign="top" charoff="50">Sample year</td> <td align="center" valign="top" charoff="50">1.145</td> <td align="center" valign="top" charoff="50">1.139</td> <td align="center" valign="top" charoff="50">1.151</td> </tr> <tr> <td rowspan="2" align="left" valign="middle" charoff="50">0.9</td> <td rowspan="2" align="center" valign="middle" charoff="50">0.045</td> <td align="left" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">1.968</td> <td align="center" valign="top" charoff="50">1.846</td> <td align="center" valign="top" charoff="50">2.099</td> </tr> <tr> <td align="left" valign="top" charoff="50">Sample year</td> <td align="center" valign="top" charoff="50">1.156</td> <td align="center" valign="top" charoff="50">1.15</td> <td align="center" valign="top" charoff="50">1.163</td> </tr> <tr> <td rowspan="2" align="left" valign="middle" charoff="50">0.95</td> <td rowspan="2" align="center" valign="middle" charoff="50">0.045</td> <td align="left" valign="top" charoff="50">UK</td> <td align="center" valign="top" charoff="50">1.551</td> <td align="center" valign="top" charoff="50">1.448</td> <td align="center" valign="top" charoff="50">1.662</td> </tr> <tr> <td align="left" valign="top" charoff="50">Sample year</td> <td align="center" valign="top" charoff="50">1.161</td> <td align="center" valign="top" charoff="50">1.154</td> <td align="center" valign="top" charoff="50">1.168</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>OR: odds-ratio.</p> </div> </div> </div> </div>  </body></html>