<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">eLife</span><span class="iso-abbrev" title="iso-abbrev">Elife</span><span class="publisher-id" title="publisher-id">eLife</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">eLife</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6491035">6491035</a></span><span class="publisher-id" title="publisher-id">46440</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.46440">10.7554/eLife.46440</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Evolutionary Biology</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Host-pathogen coevolution increases genetic variation in susceptibility to infection</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duxbury</span><span class="given-names" tagx="given-names" title="given-names">Elizabeth ML</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5733-3645</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#equal-contrib1">†</a><a href="#fund1" /><a href="#fund3" /><a href="#con1" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">Jonathan P</span></span><a href="#aff1">1</a><a href="#equal-contrib1">†</a><a href="#fund1" /><a href="#fund3" /><a href="#con2" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maria Vespasiani</span><span class="given-names" tagx="given-names" title="given-names">Davide</span></span><a href="#aff1">1</a><a href="#con3" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thüringer</span><span class="given-names" tagx="given-names" title="given-names">Yannik</span></span><a href="#aff1">1</a><a href="#con4" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tolosana</span><span class="given-names" tagx="given-names" title="given-names">Ignacio</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-3766-8296</span><a href="#aff1">1</a><a href="#con5" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">Sophia CL</span></span><a href="#aff1">1</a><a href="#con6" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tagliaferri</span><span class="given-names" tagx="given-names" title="given-names">Lucia</span></span><a href="#aff1">1</a><a href="#con7" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamacioglu</span><span class="given-names" tagx="given-names" title="given-names">Altug</span></span><a href="#aff1">1</a><a href="#con8" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindsley</span><span class="given-names" tagx="given-names" title="given-names">Imogen</span></span><a href="#aff1">1</a><a href="#con9" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Love</span><span class="given-names" tagx="given-names" title="given-names">Luca</span></span><a href="#aff1">1</a><a href="#con10" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Unckless</span><span class="given-names" tagx="given-names" title="given-names">Robert L</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-8586-7137</span><a href="#aff3">3</a><a href="#con11" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">Francis M</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-7470-8157</span><span class="email" tagx="email" title="email">fmj1001@cam.ac.uk</span><a href="#aff1">1</a><a href="#fund1" /><a href="#fund3" /><a href="#con12" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">Ben</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6936-1697</span><span class="email" tagx="email" title="email">b.longdon2@exeter.ac.uk</span><a href="#aff1">1</a><a href="#aff4">4</a><a href="#fund1" /><a href="#fund2" /><a href="#fund3" /><a href="#fund4" /><a href="#con13" /><a href="#conf1" /><a href="#dataset1" /><a href="#dataset2" /><a href="#dataset3" /></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Department of Genetics</span><span class="institution" title="institution">University of Cambridge</span><span class="addr-line" title="addr-line">Cambridge</span><span class="country" tagx="country" title="country">United Kingdom</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">School of Biological Sciences</span><span class="institution" title="institution">University of East Anglia</span><span class="addr-line" title="addr-line">Norwich</span><span class="country" tagx="country" title="country">United Kingdom</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">Department of Molecular Biosciences</span><span class="institution" title="institution">University of Kansas</span><span class="addr-line" title="addr-line">Lawrence</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Centre for Ecology and Conservation, Biosciences</span><span class="institution" title="institution">University of Exeter (Penryn Campus)</span><span class="addr-line" title="addr-line">Cornwall</span><span class="country" tagx="country" title="country">United Kingdom</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemaître</span><span class="given-names" tagx="given-names" title="given-names">Bruno</span></span><span class="role" tagx="role" title="role">Reviewing Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">École Polytechnique Fédérale de Lausanne</span><span class="country" tagx="country" title="country">Switzerland</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tautz</span><span class="given-names" tagx="given-names" title="given-names">Diethard</span></span><span class="role" tagx="role" title="role">Senior Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Max-Planck Institute for Evolutionary Biology</span><span class="country" tagx="country" title="country">Germany</span></span></span></div> <div class="author-notes" title="author-notes"> <div class="fn-type-con" title="con"><span class="label" tagx="label" title="label">†</span><p>These authors contributed equally to this work.</p> </div> </div><span class="pub-date-" title="pub-date-">: <span>2019-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e46440</span><span class="history" title="history"><span class="received" title="received">received: 2019-3-03</span><span class="accepted" title="accepted">accepted: 2019-4-07</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Duxbury et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license">[license_ref] http://creativecommons.org/licenses/by/4.0/<span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</span></span></div><a class="self-uri" href="elife-46440.pdf" title="self-uri">elife-46440.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>It is common to find considerable genetic variation in susceptibility to infection in natural populations. We have investigated whether natural selection increases this variation by testing whether host populations show more genetic variation in susceptibility to pathogens that they naturally encounter than novel pathogens. In a large cross-infection experiment involving four species of <i>Drosophila</i> and four host-specific viruses, we always found greater genetic variation in susceptibility to viruses that had coevolved with their host. We went on to examine the genetic architecture of resistance in one host species, finding that there are more major-effect genetic variants in coevolved host-pathogen interactions. We conclude that selection by pathogens has increased genetic variation in host susceptibility, and much of this effect is caused by the occurrence of major-effect resistance polymorphisms within populations.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd"><i>Drosophila</i></div> <div class="kwd" title="kwd">
viruses</div> <div class="kwd" title="kwd">
sigma virus</div> <div class="kwd" title="kwd">
coevolution</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd"><i>D. melanogaster</i></div> <div class="kwd" title="kwd">
Virus</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100008668</span><span class="institution" title="institution">NERC Environmental Bioinformatics Centre</span></div> </div><span class="award-id" tagx="award-id" title="award-id">NE/L004232/1</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duxbury</span><span class="given-names" tagx="given-names" title="given-names">Elizabeth ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">Jonathan P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">Francis M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">Ben</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100004440</span><span class="institution" title="institution">Wellcome</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Sir Henry Dale Fellowship (Grant Number 109356/Z/15/Z)</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">Ben</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100010663</span><span class="institution" title="institution">H2020 European Research Council</span></div> </div><span class="award-id" tagx="award-id" title="award-id">281668</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duxbury</span><span class="given-names" tagx="given-names" title="given-names">Elizabeth ML</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100000288</span><span class="institution" title="institution">Royal Society</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Sir Henry Dale Fellowship (Grant Number 109356/Z/15/Z)</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">Ben</span></span></span></div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Author impact statement" value="A history of coevolution increases genetic variation in the susceptibility of Drosophila to viruses, largely by introducing major-effect resistance polymorphisms into populations." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>From bacteria to plants and insects to humans, it is common to find considerable genetic variation in susceptibility to infection in natural populations (<a href="#bib20">Chapman and Hill, 2012</a>; <a href="#bib6">Bangham et al., 2008a</a>; <a href="#bib36">Hammond-Kosack and Jones, 1997</a>; <a href="#bib45">Lazzaro et al., 2004</a>). This variation in susceptibility can determine the impact of disease on health and economic output (<a href="#bib24">Cooke and Hill, 2001</a>; <a href="#bib43">King and Lively, 2012</a>; <a href="#bib1">Alonso-Blanco and Méndez-Vigo, 2014</a>; <a href="#bib17">Burgner et al., 2006</a>). In nature and breeding programs, it determines the ability of populations to evolve resistance to infection. Insect populations, like those of other organisms, typically contain considerable genetic variation in susceptibility to infection (<a href="#bib6">Bangham et al., 2008a</a>; <a href="#bib45">Lazzaro et al., 2004</a>; <a href="#bib75">Tinsley et al., 2006</a>; <a href="#bib66">Obbard and Dudas, 2014</a>), and provide a convenient laboratory model in which to investigate basic questions about how this variation is maintained (<a href="#bib60">Magwire et al., 2012</a>). Within vector species like mosquitoes, resistant genotypes are less likely to transmit pathogens, and this has the potential to reduce disease in vertebrate populations (<a href="#bib9">Beerntsen et al., 2000</a>). Where pathogens are contributing the decline of beneficial species like pollinators, high levels of genetic variation may allow populations to recover (<a href="#bib61">Maori et al., 2007</a>). Understanding the origins of genetic variation in susceptibility is therefore a fundamental question in infectious disease biology.</p> <p>As pathogens are harmful, natural selection is expected to favour resistant host genotypes. Directional selection on standing genetic variation will drive alleles to fixation, removing variants from the population (<a href="#bib25">Falconer, 1960</a>; <a href="#bib26">Falconer and Mackay, 1996</a>; <a href="#bib11">Blows and Hoffmann, 2005</a>). However, as directional selection also increases the frequency of new mutations that change the trait in the direction of selection, at equilibrium it is expected to have no effect on levels of standing genetic variation (relative to mutation-drift balance; <a href="#bib37">Hill, 1982</a>). However, selection mediated by pathogens may be different. Coevolution with pathogens can result in the maintenance of both resistant and susceptible alleles by negative frequency dependent selection (<a href="#bib84">Woolhouse et al., 2002</a>; <a href="#bib35">Haldane, 1949</a>). Similarly, when infection prevalence exhibits geographical or temporal variation, selection can maintain genetic variation, especially if pleiotropic costs to resistance provide an advantage to susceptible individuals when infection is rare (<a href="#bib64">Nuismer et al., 2003</a>; <a href="#bib74">Thompson, 1999</a>; <a href="#bib44">Koskella, 2018</a>). Even when there is simple directional selection on alleles that increase resistance, the direction of selection by pathogens may frequently change so populations may not be at equilibrium. If selection favours rare alleles – such as new mutations – directional selection can transiently increase genetic variation during their spread through the population (<a href="#bib8">Barton and Turelli, 1987</a>; <a href="#bib5">Bangham et al., 2007</a>; <a href="#bib59">Magwire et al., 2011</a>).</p> <p>As part of a whole genome association study, we have previously estimated levels of genetic variation in the susceptibility of <i>D. melanogaster</i> to four different viruses (<a href="#bib60">Magwire et al., 2012</a>). We found that there was more genetic variation in susceptibility to the two viruses that were isolated from <i>D. melanogaster</i> than the two viruses from other insect species. Furthermore, in each of these naturally coevolved host-pathogen associations we detected a single major-effect polymorphism affecting resistance. This led us to propose that coevolution had increased genetic variation in susceptibility due to the presence of major-effect resistance polymorphisms. However, this conclusion remains anecdotal. First, aside from two sigma viruses, the viruses we used were mostly very distantly related, so their biology may differ for many reasons. Second, our association study had low statistical power, so conclusions about the genetics were based on just two genes. Finally, and most importantly, the link between coevolutionary history and genetic variation is based on a single host and four viruses, and so could arise by chance. For this reason, here we return to this question and formally test whether a history of coevolution alters the amount and nature of genetic variation.</p> <p>To examine how selection by a pathogen affects levels of genetic variation we used a natural host-virus system; <i>Drosophila</i> and sigma viruses (<a href="#bib57">Longdon and Jiggins, 2012</a>; <a href="#bib52">Longdon et al., 2012</a>). Sigma viruses are a clade of insect RNA viruses with negative-sense genomes in the family Rhabdoviridae (<a href="#bib56">Longdon et al., 2017</a>; <a href="#bib54">Longdon et al., 2015a</a>; <a href="#bib48">Longdon et al., 2010</a>; <a href="#bib49">Longdon et al., 2011a</a>). They are vertically transmitted through eggs and sperm, and each sigma virus infects a single host species, simplifying studies of coevolution (<a href="#bib56">Longdon et al., 2017</a>; <a href="#bib49">Longdon et al., 2011a</a>). In <i>Drosophila melanogaster</i>, despite the virus causing little adult mortality, infection reduces host fitness by approximately 25% (<a href="#bib82">Wilfert and Jiggins, 2013</a>; <a href="#bib85">Yampolsky et al., 1999</a>). As prevalence in wild populations is typically around 10% (<a href="#bib82">Wilfert and Jiggins, 2013</a>), there is the necessary selective pressure for resistance to evolve (<a href="#bib59">Magwire et al., 2011</a>; <a href="#bib18">Cao et al., 2016</a>). In <i>D. melanogaster</i>, three major-effect resistance alleles have been identified (<a href="#bib6">Bangham et al., 2008a</a>; <a href="#bib60">Magwire et al., 2012</a>; <a href="#bib5">Bangham et al., 2007</a>; <a href="#bib59">Magwire et al., 2011</a>; <a href="#bib18">Cao et al., 2016</a>; <a href="#bib7">Bangham et al., 2008b</a>; <a href="#bib78">Wayne et al., 1996</a>). There has been a recent sweep of <i>D. melanogaster</i> sigma virus (DMelSV) genotypes that are able to overcome one of these host resistance genes (<a href="#bib29">Fleuriet, 1988</a>; <a href="#bib80">Wilfert and Jiggins, 2010a</a>; <a href="#bib83">Wilfert and Jiggins, 2014</a>). Given the power of <i>Drosophila</i> genetics, this system is an excellent model of a coevolutionary arms race between hosts and pathogens.</p> <p>Sigma viruses offer a novel way to test how coevolution with a pathogen alters the amount of genetic variation in host susceptibility. As sigma viruses are vertically transmitted, we can be certain of which hosts and viruses are naturally coevolving and which are not. In this study we have used four species of <i>Drosophila</i> (<i>D. affinis, D. immigrans, D. melanogaster</i> and <i>D. obscura</i>) that shared a common ancestor approximately 40 million years ago (<a href="#bib65">Obbard et al., 2012</a>; <a href="#bib73">Tamura et al., 2004</a>), and their natural sigma viruses (DAffSV, DImmSV, DMelSV and DObsSV, which have amino acid identities of &amp;lt;55% in the most conserved gene) (<a href="#bib54">Longdon et al., 2015a</a>; <a href="#bib48">Longdon et al., 2010</a>; <a href="#bib50">Longdon et al., 2011b</a>). Despite their vertical mode of transmission, the phylogenies of the viruses and their hosts are incongruent, suggesting they have jumped between host species during their evolution (<a href="#bib50">Longdon et al., 2011b</a>). To test whether selection by viruses increases the amount of genetic variation in host susceptibility we have compared the viral load of endemic viruses that naturally infect each of the four host species to non-endemic viruses. We then examined how selection by these viruses has altered the genetic architecture of resistance by mapping loci that confer resistance to endemic and non-endemic viruses in <i>D. melanogaster</i>.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="geneticvariationinsusceptibilitytoinfectionisgreatestincoevolvedhost-virusassociations" title="sec"> <div class="title" tagx="title" title="title">
Genetic variation in susceptibility to infection is greatest in coevolved host-virus associations</div> <p>To test whether selection by viruses has increased genetic variation in susceptibility to infection, we compared endemic host-virus associations with novel associations that have no history of coevolution. We used four different species of <i>Drosophila,</i> each of which is naturally host to a different sigma virus (<a href="#bib57">Longdon and Jiggins, 2012</a>; <a href="#bib52">Longdon et al., 2012</a>; <a href="#bib56">Longdon et al., 2017</a>; <a href="#bib48">Longdon et al., 2010</a>; <a href="#bib49">Longdon et al., 2011a</a>). We collected four species from the wild and created genetically diverse populations in the laboratory. Using flies from these populations we crossed single males to single females to create full-sib families. The progeny of these crosses were then injected with either the virus isolated from that species, or a virus isolated from one of the other species (<a href="#fig1">Figure 1</a>). Studies on DMelSV have shown that loci that reduce loads when the virus is injected also reduce infection rates in both the lab and field (<a href="#bib6">Bangham et al., 2008a</a>; <a href="#bib52">Longdon et al., 2012</a>; <a href="#bib15">Bregliano, 1970</a>; <a href="#bib16">Brun and Plus, 1980</a>; <a href="#bib67">Ohanessian-Guillemain, 1963</a>; <a href="#bib81">Wilfert and Jiggins, 2010b</a>). As infection is costly, this is expected to increase host fitness. Fifteen days post infection we extracted RNA from the flies and measured viral load by quantitative RT-PCR. The differences between the viral load of different families allowed us to estimate the genetic variance (<i>V<sub>G</sub></i>) in viral load – a measure of how much viral resistance varies in the population due to genetic as opposed to environmental causes. In total we infected 52,592 flies and measured the viral load in 4295 biological replicates (a vial containing a mean of 12 flies) across 1436 full-sib families (details of sample sizes in Additional Methods).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.46440.002"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Experimental design and phylogenies.</div> <p>(<b>A</b>) Four species of <i>Drosophila</i> were independently infected both with a sigma virus with which they are naturally infected with in nature (red) and two viruses that naturally infect another species (black). (<b>B</b>) Phylogenies of the sigma viruses (inferred using the <i>L</i> gene) and their <i>Drosophila</i> hosts (inferred using <i>COI, COII, 28S rDNA, Adh, SOD, Amyrel</i> and <i>RpL32</i> genes), redrawn from <a href="#bib54">Longdon et al. (2015a)</a> and <a href="#bib55">Longdon et al. (2015b)</a>. Scale bars represent substitutions per site under a relaxed clock model. Posterior supports for nodes are all &amp;gt;0.99.</p>   </div> <p>Within populations of all four species, we found significantly greater genetic variance in susceptibility to the sigma virus that naturally infects that species compared to viruses from other species (<a href="#fig2">Figure 2</a>; <a href="#fig2s1">Figure 2—figure supplement 1</a>; <a href="#supp1">Supplementary file 1</a> and <a href="#supp2">2</a>). These different variances reflect considerable differences in the mean viral load between families (<a href="#fig2">Figure 2A</a>). For example, when families of <i>D. obscura</i> in the 2<sup>nd</sup> and 98<sup>th</sup> percentile were compared, there was a 1294 fold difference between the viral loads of the coevolved virus in the families (<a href="#fig2">Figure 2A</a>). In contrast, for the non-coevolved viruses there was a 27 fold difference in DMelSV loads and a 19 fold difference in DAffSV loads (see <a href="#fig2">Figure 2A</a> for statistics). The data in <a href="#fig2">Figure 2</a> is zero-centred to allow comparison of the variances, but across the four species, there was no consistent difference between the mean viral load in coevolved versus non-coevolved associations (i.e. the coevolved virus does not always replicate to higher levels suggesting this is not an artefact of simply replicating poorly in a host – see <a href="#fig2s1">Figure 2—figure supplement 1</a>). Additionally, there was no correlation between the genetic variance in viral load and the mean viral load (<a href="#fig2s1">Figure 2—figure supplement 1</a>, Spearman’s correlation: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div></div></span>= -0.38, S=296, <i>P</i>=0.22).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.46440.003"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <div class="title" tagx="title" title="title">
Genetic variation in susceptibility to coevolved and non-coevolved viruses.</div> <p>The viral load was measured 15 days post infection by quantitative RT-PCR relative to a <i>Drosophila</i> reference gene (<i>RpL32</i>). (<b>A</b>) The points show model prediction family means from our GLM and are centred on zero. The number of families in each panel was down-sampled so the same number of families is shown for each virus. Coevolved host-virus associations are in red. (<b>B</b>) The genetic variance in log<sub>2 </sub>viral load was estimated from the between family variance assuming that all genetic variance is additive. The bars are 95% credible intervals. Posterior probabilities for significantly different genetic variances are shown in grey (see <a href="#supp1">Supplementary file 1</a> and <a href="#supp2">2</a>).</p>   </div> </div> <div class="major-effectgeneticvariantsthatareknowntoprovideresistancetodmelsvdonotprotectagainstotherviruses" title="sec"> <div class="title" tagx="title" title="title">
Major-effect genetic variants that are known to provide resistance to DMelSV do not protect against other viruses</div> <p>To examine whether the genetic basis of resistance to coevolved and non-coevolved viruses was different we estimated the genetic correlations (<i>r<sub>g</sub></i>) in their viral loads. These represent the proportion of genetic variance in viral load between pairs of viruses that shares the same genetic causes. In <i>D. melanogaster</i> these were 0.40 for DMelSV-DAffSV, (95% CIs: 0.20, 0.61) and 0.25 for DMelSV-DObsSV (95% CIs: −0.01,0.47). Similar results were obtained using the <i>D. melanogaster</i> mapping population described below (<a href="#supp3">Supplementary file 3</a>). In the other species our estimates sometimes had wide credible intervals, but the genetic correlations between coevolved and non-coevolved viruses were mostly below 0.5 (<a href="#supp3">Supplementary file 3</a>; note correlations are frequently low between pairs of non-endemic viruses too). Therefore if natural selection increases genetic variation in susceptibility to a natural pathogen, there is expected to be a smaller effect on non-coevolved viruses.</p> <p>In <i>D. melanogaster</i> a substantial proportion of the genetic variance in susceptibility to DMelSV is explained by major-effect variants in the genes <i>CHKov1</i> and <i>p62</i> (also known as <i>Ref(2)P</i>) (<a href="#bib60">Magwire et al., 2012</a>; <a href="#bib5">Bangham et al., 2007</a>; <a href="#bib59">Magwire et al., 2011</a>; <a href="#bib78">Wayne et al., 1996</a>; <a href="#bib23">Contamine et al., 1989</a>). The resistant allele in each of these genes has arisen recently by mutation and been driven up in frequency by natural selection, presumably due to the presence of DMelSV in natural populations (<a href="#bib5">Bangham et al., 2007</a>; <a href="#bib59">Magwire et al., 2011</a>). Therefore, if these genetic variants confer resistance to DMelSV but not the other sigma viruses, then this may explain the differences in genetic variance that we observed.</p> <p>To examine whether <i>CHKov1</i> or <i>p62</i> contributed to the differences in genetic variance we observed in <i>D. melanogaster</i>, we genotyped the parents of the full sib families for variants that confer resistance (<a href="#supp4">Supplementary file 4</a>). Assuming the effects of the resistant alleles are additive, we estimated that the load of the coevolved virus DMelSV was more than halved in homozygous <i>CHKov1</i> resistant flies compared to susceptible flies (reduction in log<sub>2 </sub>viral load = 1.2, 95% CI = 0.6, 1.8). In contrast we found no significant effect of this gene on loads of the non-coevolved viruses (DAffSV = −0.2, 95% CI = 0.2,–0.8; DObsSV = −0.4, 95% CI = −0.04, 1.0). The resistant allele of <i>p62</i> was present at such a low frequency (1.5%) in the population that we lacked statistical power to investigate its effects.</p> <p>To confirm these results we infected 1869 flies from 32 inbred <i>D. melanogaster</i> lines (<a href="#bib58">Mackay et al., 2012</a>) that had known <i>CHKov1</i> or <i>p62</i> genotypes. The effect of these genes was greater on the naturally occurring virus than the viruses from other species (<a href="#fig3">Figure 3</a>; effect of genotype on DMelSV load: <i>F<sub>2,28</sub></i> = 13.2, p=0.00001; DAffSV: <i>F<sub>2,29</sub></i> = 4.9, p=0.01; DObsSV: <i>F<sub>2,29</sub></i> = 5.7, p=0.01).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.46440.005"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <div class="title" tagx="title" title="title">
Viral load in <i>D. melanogaster</i> lines carrying different alleles of <i>CHKov1</i> and <i>p62.</i></div> <p>Each point is the viral load of a separate inbred fly line carrying the resistant (Res) or susceptible (Sus) allele of <i>P62</i> or <i>CHKov1.</i> Horizontal bars are medians. Viral load was measured 15 days post infection by quantitative RT-PCR relative to a <i>Drosophila</i> reference gene (<i>RpL32</i>).</p>   </div> </div> <div class="thereareagreaternumberofmajor-effectvariantsincoevolvedhost-virusassociations" title="sec"> <div class="title" tagx="title" title="title">
There are a greater number of major-effect variants in coevolved host-virus associations</div> <p>To investigate how coevolution shapes the genetics of resistance, we mapped loci controlling resistance using a <i>D. melanogaster</i> advanced intercross population (the DSPR panel [<a href="#bib42">King et al., 2012</a>]). This population samples genetic variation in a small number of genotypes from around the world (the experiments above sampled many genotypes from a single location). It was founded by allowing two sets of 8 inbred founder lines to interbreed for 50 generations, then creating recombinant inbred lines (RILs) whose genomes are a fine-scale mosaic of the original founder genomes. We used 377 RILs from these populations, which have up to 15 alleles of each gene (one founder line is shared between the two populations). We infected 15,916 flies across 1362 biological replicates (a vial containing a mean of 12 flies) with DMelSV, DAffSV or DObsSV and measured viral load as above (see Materials and methods).</p> <p>We first estimated the genetic variance in viral load within our mapping population. The results recapitulated what we had found above in a natural population of flies — there was considerably more genetic variation in susceptibility to the coevolved virus than the non-coevolved viruses (<a href="#fig4">Figure 4A</a>, filled circles). Therefore, our earlier result from a single population holds when sampling flies from across six continents, although the magnitude of the effect is considerably greater in this mapping population.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.46440.006"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <div class="title" tagx="title" title="title">
The genetic architecture of resistance to coevolved and non-coevolved viruses in <i>D. melanogaster</i>.</div> <p>(<b>A</b>) The genetic variance in viral load within the mapping population (filled circles). The open circles are estimates of the genetic variance after accounting for the effects of the QTL in panel C. Error bars are 95% credible intervals. (<b>B</b>) QTL affecting viral load. The horizontal line shows a genome-wide significance threshold of p&amp;lt;0.05 that was obtained by permutation of Logarithm of odds (LOD) scores. (<b>C</b>) The effect of the seven QTL detected on the load of the three viruses. Only QTL that remained were significant following multiple regression with all the loci are shown. The coevolved virus is shown in red.</p>   </div> <p>To examine the genetic basis of virus resistance, we looked for associations between genotype and viral load across the genome (<a href="#fig4">Figure 4B</a>). In the coevolved association (DMelSV) we identified seven QTL associated with resistance, compared to one that affects DObsSV and none affecting DAffSV (<a href="#supp5">Supplementary file 5</a>; this excludes one DMelSV QTL on the X chromosome that did not remain significant after accounting for the other QTL). The QTL affecting DObsSV also has a significant effect on DMelSV. One of the QTL corresponded to <i>p62</i> (2L 53 cM). The susceptible allele of <i>CHKov1</i> was not present in the fly lines assayed.</p> <p>To examine the effect that the QTL have on viral load, we first split the founder alleles into a resistant class and a susceptible class (see Materials and methods) and then estimated the difference in viral load between the functionally distinct alleles. Six of the seven QTL resulted in greater reductions in the load of the coevolved virus (DMelSV) than the viruses isolated from other species (<a href="#fig4">Figure 4C</a>). There were only two cases where there was substantial cross-resistance to multiple viruses—3R 49 cM confers strong resistance to DMelSV and weak resistance to DObsSV, while 3L 46 cM confers weak resistance to DMelSV and strong resistance to DObsSV.</p> <p>Together, this modest number of loci with substantial effects on resistance explains most of the high genetic variance in resistance to the coevolved virus (<a href="#fig4">Figure 4A</a>, filled versus open circles). Individually, resistant alleles cause an approximate 3–7 fold reduction in viral load (<a href="#fig4">Figure 4C</a>), and together they explain 59% of the genetic variance in susceptibility to DMelSV, 77% for DObsSV and 3% for DAffSV (<a href="#fig4">Figure 4A</a>, filled versus open circles). However, even after accounting for these genes there remains a significantly higher genetic variance in the viral load of the coevolved virus (<a href="#fig4">Figure 4A</a>, open circles, non-overlapping 95% CI).</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We have found greater genetic variation in susceptibility to viruses that naturally infect <i>Drosophila</i> compared to viruses that do not, suggesting that selection by these pathogens has acted to increase the amount of genetic variation in susceptibility. This effect was largely caused by a modest number of major-effect genes that explain over half of the genetic variance in resistance.</p> <p>As the genetic variants in the genes <i>p62 (ref(2)P)</i> and <i>CHKov1</i> that confer resistance to DMelSV have been identified, this has previously allowed us to use patterns of DNA sequence variation to infer how selection has acted on resistance in <i>D. melanogaster</i>. In both these genes the resistant alleles have arisen relatively recently by mutation and natural selection has pushed them rapidly up in frequency, leaving a characteristic signature of elevated linkage disequilibrium and low genetic diversity around the variant causing resistance (<a href="#bib5">Bangham et al., 2007</a>; <a href="#bib59">Magwire et al., 2011</a>; <a href="#bib78">Wayne et al., 1996</a>). There is no indication of negative frequency dependent selection, and these polymorphisms appear to have arisen from partial selective sweeps (<a href="#bib5">Bangham et al., 2007</a>; <a href="#bib59">Magwire et al., 2011</a>).</p> <p>The most parsimonious explanation of these observations is that there has been directional selection favouring resistance alleles (although this type of data cannot rule out negative frequency dependent selection as is predicted by models of coevolution). At equilibrium, directional selection on a trait is not expected to affect its genetic variance (relative to a population under mutation-drift balance; <a href="#bib37">Hill, 1982</a>). However, the genetic variance will transiently increase if the variants under selection are initially at low frequency (<a href="#bib8">Barton and Turelli, 1987</a>), as was the case for both <i>p62</i> and <i>CHKov1</i> (<a href="#bib5">Bangham et al., 2007</a>; <a href="#bib59">Magwire et al., 2011</a>). A particular feature of pathogens is that the direction of selection is likely to continually change as new pathogens appear in populations or existing pathogens evolve to overcome host defences. For example, in France and Germany in the 1980s, DMelSV evolved to largely overcome the effects of the resistant allele of <i>p62</i> (<a href="#bib30">Fleuriet and Periquet, 1993</a>; <a href="#bib31">Fleuriet and Sperlich, 1992</a>). Similarly, DImmSV has swept through European populations of <i>D. immigrans</i> in the last ~16 years and DObsSV through UK populations of <i>D. obscura</i> in the last ~11 years (<a href="#bib56">Longdon et al., 2017</a>; <a href="#bib49">Longdon et al., 2011a</a>). If selection by pathogens continually changes and resistance evolves from new mutations, then this may cause a sustained increase in genetic variance in susceptibility to infection.</p> <p>A key question is whether the increased genetic variation that we see in coevolved <i>Drosophila</i>–sigma virus interactions will hold for coevolved pathogens more generally. Theory suggests that a critical factor determining levels of genetic variation is whether resistance is costly to evolve, as this can result in the maintenance of variation by negative frequency dependent selection (<a href="#bib3">Antonovics and Thrall, 1994</a>; <a href="#bib12">Boots et al., 2014</a>). In humans this has been proposed as an explanation of why there is less genetic variation in susceptibility to pathogens that are effectively controlled by the adaptive immune response, as these resistance mechanisms may be less costly (<a href="#bib4">Baker and Antonovics, 2012</a>). However, it seems unlikely that virus resistance in <i>Drosophila</i> is costly, as experiments have failed to detect costs of DCV resistance (<a href="#bib27">Faria et al., 2015</a>) despite costs of parasitoid and bacterial resistance being repeatedly detected (<a href="#bib62">McGonigle et al., 2017</a>; <a href="#bib63">McKean et al., 2008</a>; <a href="#bib87">Ye et al., 2009</a>). Sigma viruses are also extreme host specialists, so evolutionary changes in resistance will tend to alter pathogen prevalence and so the strength of selection. These epidemiological feedbacks are predicted to frequently increase genetic diversity (<a href="#bib12">Boots et al., 2014</a>; <a href="#bib10">Best et al., 2009</a>; <a href="#bib13">Boots and Haraguchi, 1999</a>). However, in <i>D. melanogaster</i> we see large amounts of genetic variation in susceptibility to viruses that have a broader host range than sigma viruses (DCV and Kallithea virus that infect <i>D. melanogaster</i> and <i>D. simulans</i> in the wild) (<a href="#bib60">Magwire et al., 2012</a>; <a href="#bib70">Palmer et al., 2018</a>; <a href="#bib21">Christian, 1987</a>; <a href="#bib79">Webster et al., 2015</a>). Therefore, it seems unlikely that our conclusions will be a quirk of the sigma virus system. In an analogous study of rust resistance in wild flax plants, sympatric (putatively coevolved) populations had fewer partial resistances than allopatric populations, suggesting more major gene effects, even though overall there was somewhat less genetic variation in susceptibility to sympatric fungal pathogens (<a href="#bib2">Antonovics et al., 2011</a>).</p> <p>Quantitative traits are typically controlled by a very large number of genetic variants, each of which tends to have a very small effect (<a href="#bib72">Shi et al., 2016</a>; <a href="#bib86">Yang et al., 2010</a>). However, susceptibility to sigma viruses has a simpler genetic basis, with seven polymorphisms explaining over half the genetic variance. This confirms our previous work in <i>D. melanogaster</i> showing a simple genetic basis of virus resistance (<a href="#bib60">Magwire et al., 2012</a>; <a href="#bib22">Cogni et al., 2016</a>). As these genetic variants mostly only affect the naturally occurring pathogen of <i>D. melanogaster,</i> our results suggest that not only is selection by pathogens increasing the genetic variance but it is also altering the genetic architecture of resistance by introducing major-effect variants into the population. One explanation for this observation is that most quantitative traits are under stabilising selection, so major effect variants will tend to be deleterious and removed by selection (<a href="#bib32">Gibson, 2018</a>). In contrast, selection by pathogens likely changes through time and populations may be far from their optimal level of resistance. If this is the case, Fisher’s geometric model predicts that major effect variants will be favoured by directional selection (<a href="#bib28">Fisher, 1930</a>). Alternatively, the coevolution of hosts and parasites can favour discrete susceptible and resistant hosts (<a href="#bib12">Boots et al., 2014</a>), and at the genetic level this may result in major-effect variants (although this theory does not explicitly address the underlying genetics). The simple genetics may also be driven by mutation—for many traits major-effect mutations that increase fitness may be extremely rare, but this may not be the case for virus resistance. For example a single (loss of function) mutation may prevent a virus binding to a host receptor or utilising other parts of the host cellular machinery, and so confer strong resistance.</p> <p>Regardless of its causes, it may be common that susceptibility to infectious disease has a simple genetic basis. In humans, <a href="#bib39">Hill (2012)</a> advocated the view that susceptibility to infectious disease is qualitatively different from other traits and has a much simpler genetic basis (<a href="#bib39">Hill, 2012</a>). In <i>Drosophila,</i> resistance to DCV and parasitoid wasps both have a simple genetic basis (<a href="#bib60">Magwire et al., 2012</a>; <a href="#bib19">Cao et al., 2017</a>; <a href="#bib68">Orr and Irving, 1997</a>). In plants, major-effect polymorphisms in R genes are commonplace (<a href="#bib36">Hammond-Kosack and Jones, 1997</a>) and in a plant-fungi system genotype-by-genotype interactions explain a larger proportion of the total variance in sympatric (more coevolved) associations (<a href="#bib2">Antonovics et al., 2011</a>). In contrast, studies of bacterial resistance in <i>Drosophila</i> have typically used pathogens that are unlikely to have any history of coevolution, and have found a polygenic basis to resistance (<a href="#bib14">Bou Sleiman et al., 2015</a>; <a href="#bib40">Hotson and Schneider, 2015</a>; <a href="#bib41">Howick and Lazzaro, 2017</a>; <a href="#bib77">Wang et al., 2017</a>). In these studies the polymorphism with the largest effect was found against the only natural <i>D. melanogaster</i> pathogen tested (a polymorphism in <i>Diptericin</i> detected using <i>Providencia rettgeri</i> infection), anecdotally supporting the patterns seen here (<a href="#bib41">Howick and Lazzaro, 2017</a>; <a href="#bib76">Unckless et al., 2015</a>).</p> <p>A major source of emerging infectious disease is pathogens jumping into novel hosts where they have no co-evolutionary history (<a href="#bib53">Longdon et al., 2014</a>; <a href="#bib71">Parrish et al., 2008</a>). Our results suggest that when a pathogen infects a novel host species, there may be far less genetic variation in susceptibility among individuals than is normally the case. This may create a ‘monoculture effect’ (<a href="#bib43">King and Lively, 2012</a>; <a href="#bib46">Lively, 2010</a>; <a href="#bib69">Ostfeld and Keesing, 2012</a>), which could leave populations vulnerable to epidemics of pathogens that have previously circulated in other host species. Longer term, low levels of pre-standing genetic variation may slow down the rate at which the new host can evolve resistance to a new pathogen.</p> <p>In conclusion, we have demonstrated that selection by pathogens has increased the amount of genetic variation in host susceptibility. We find resistance has a simple underlying genetic architecture and is largely controlled by major effect resistance loci.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="virusextractionandinfection" title="sec"> <div class="title" tagx="title" title="title">
Virus extraction and infection</div> <p>We extracted the sigma viruses DAffSV, DImmSV, DMelSV and DObsSV from infected stocks of <i>D. affinis</i> (line: NC10), <i>D. immigrans</i> (line: DA2), <i>D. melanogaster</i> (line: E320 Ex) and <i>D. obscura</i> (line: 10A) respectively (<a href="#bib56">Longdon et al., 2017</a>; <a href="#bib54">Longdon et al., 2015a</a>; <a href="#bib48">Longdon et al., 2010</a>; <a href="#bib49">Longdon et al., 2011a</a>; <a href="#bib16">Brun and Plus, 1980</a>). These infected lines were collected from the wild between 2007–2012 (all from the UK, bar <i>D.affinis</i> which was collected in the USA). Infected fly stocks were checked for infection by exposing the flies to 100% CO<sub>2</sub> at 12°C for 15mins then paralysed flies were collected 30mins later. The DImmSV infected line does not show CO<sub>2</sub> sensitivity and so was confirmed to have a high level of infection using RT-PCR. Infected flies were frozen at −80°C and later homogenised in ringers solution (2.5 μl per fly) and centrifuged at 13,000 g for 10 min at 4°C. The supernatant was collected, 2% v/v FBS was added then virus solutions were aliquoted and stored at −80°C.</p> </div> <div class="experimentaldesign" title="sec"> <div class="title" tagx="title" title="title">
Experimental design</div> <p>We set up a common garden experiment to measure genetic variation in susceptibility to natural and non-natural viruses across four host species. In a fractional factorial experiment each species was infected it with its own virus, as well as two viruses that do not infect that host species (see <a href="#fig1">Figure 1</a>). All of the viruses replicate in all hosts (with the exception of DImmSV in <i>D. melanogaster</i> that showed limited evidence of replication – this combination was not used in the experiment). All fly stocks used (see additional methods for stock details) were tested for existing sigma virus infection using RT-PCR over two generations. For all species we collected flies from the wild and we used a full-sib mating design. The progeny of these crosses were infected by injecting them with 69 nl of the viruses intrathoracically and measuring viral RNA loads 15 days post infection, as in <a href="#bib51">Longdon et al. (2011c)</a>. This time point was selected as RNA viral load tends to plateau from around day 15 post infection and there is no mortality from infection in this period. The specifics for each host species, including sample sizes, are detailed in the additional methods.</p> </div> <div class="knownresistancegenesind.melanogaster" title="sec"> <div class="title" tagx="title" title="title">
Known resistance genes in <i>D. melanogaster</i></div> <p>We genotyped parents of each <i>D. melanogaster</i> full sib family from the experiment above for two resistance alleles that are known to confer protection against DMelSV; <i>p62</i> (<i>Ref(2)P)</i> and <i>CHKov1</i>. We genotyped parental flies using PCR assays that produce different sized products depending on whether flies carry resistant or susceptible alleles. Information on these PCRs and primer sequences can be found in <a href="#supp4">Supplementary file 4</a>. We then calculated the number of resistance alleles in each family by summing the number of alleles from both mothers and fathers. We produced genotype information for 230 of the 255 families.</p> <p>Another resistance allele has been identified in the gene <i>Ge-1</i> (<a href="#bib18">Cao et al., 2016</a>). However, this allele has been found to occur at a low frequency in wild populations. We genotyped 184 parental flies from our experiment (parental flies for some families could not be collected) and found the resistant allele was not present, suggesting it is rare or absent.</p> <p>We further examined the effect of alleles known to affect susceptibility to DMelSV on all three viruses. Firstly, we infected 32 lines from the Drosophila Genetic Reference Panel (DGRP) (<a href="#bib58">Mackay et al., 2012</a>) that were susceptible for both <i>p62</i> and <i>CHKov1</i> (<i>n</i> = 11 lines), were resistant for <i>CHKov1</i> only (<i>n</i> = 13 lines), or were resistant for both genes (<i>n</i> = 8 lines) with DMelSV, DAffSV and DObsSV. No lines in the panel were resistant for <i>p62</i> and susceptible for <i>CHKov1</i>. We infected a mean of 18 flies per line (range = 3–22).</p> </div> <div class="mappingresistancegenesind.melanogaster" title="sec"> <div class="title" tagx="title" title="title">
Mapping resistance genes in <i>D. melanogaster</i></div> <p>We used 377 DSPR lines (154 from panel A and 223 from panel B, <a href="http://FlyRILs.org">http://FlyRILs.org</a> [<a href="#bib42">King et al., 2012</a>; <a href="#bib47">Long et al., 2014</a>]), kindly provided by S.J. Macdonald, University of Kansas) to carry out a Quantitative Trait Locus (QTL) study to examine the genetic basis of resistance to DAffSV, DMelSV and DObsSV in <i>D. melanogaster</i>.</p> <p>Three females and three males from each DSPR line were placed into yeasted cornmeal vials and allowed to lay for 3–4 days, at 25°C. Male offspring were collected at 0–4 days post-eclosion, and placed at 18°C for 4–6 days. Flies were then injected with DMelSV, DAffSV or DObsSV as described above. Injected males were maintained on unyeasted cornmeal at 18°C, and frozen on day 15 post-infection as above.</p> <p>Injections were carried out over 13 weeks. Each day of injection a mean of 47 unique lines (range 20–60) and 51 replicate vials were injected with 1–3 different viruses. In total we assayed 377 DSPR lines (108 lines had two biological replicates). Each replicate vial contained a mean of 12 flies (range 1–22). In total, 15,916 flies were injected across both panels of DSPR lines. We injected 319 lines with all three viruses, 38 with 2 viruses and 20 with one virus. The order of injection of lines and of viruses, was randomised across injection days. Independent biological replicates were injected on different days. Panel A and Panel B lines were assayed in two overlapping blocks.</p> </div> <div class="measuringviralload" title="sec"> <div class="title" tagx="title" title="title">
Measuring viral load</div> <p>We measured the change in RNA viral load using qRT-PCR. The viral RNA load was expressed relative to the endogenous control housekeeping gene <i>RpL32</i> (<i>Rp49</i>). RNA was extracted from flies homogenised in Trizol and reverse transcribed with GoScript reverse transcriptase (Promega) and random hexamer primers, and then diluted 1:10 with nuclease free water. The qRT-PCR was performed on an Applied Biosystems StepOnePlus system using Sensifast Hi-Rox Sybr kit (Bioline) with the following PCR cycle: 95°C for 2 min followed by 40 cycles of: 95°C for 5 s followed by 60°C for 30 s. Two qRT-PCR reactions (technical replicates) were carried out per sample with both the viral and endogenous control primers. Each qRT-PCR plate contained three standard samples, and all experimental samples were split across plates in a blocked design. A linear model was used to correct the cycle threshold (Ct) values for differences between qRT-PCR plates. Primer sequences are in <a href="#supp6">Supplementary file 6</a>.</p> <p>To estimate viral load, we calculated ΔCt as the difference between the qRT-PCR cycle thresholds of the virus and the endogenous control. Viral load calculated without using the endogenous control is strongly correlated to ΔCt for all species.</p> </div> <div class="statisticalanalysisfull-sibexperiments" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis full-sib experiments</div> <p>We used a linear mixed model to examine the amount of genetic variation in susceptibility to the different viruses. We used a trivariate model with the load of the three viruses as the response variable. For each species the model was structured as: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
v</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
f</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
u</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div> </div></div></div> <p /> <p>Where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div></div></span> is the log<sub>2 </sub>viral load of the <i>i<sup>th</sup></i> biological replicate of full-sib family <i>f</i> infected with virus <i>v.</i><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> </div></div></span> are the fixed effects, with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> </div> </div> </div> </div></div></span> being the mean viral load of each virus. <i>u</i> are the random effects for full-sib families (<i>f</i>) and for the day of injection (<i>d</i>), <i>e</i> are the residuals. By assuming that all the genetic variation in the population is additive (<a href="#bib38">Hill et al., 2008</a>), we estimated the genetic variance (<i>V<sub>G</sub></i>) of the viral load as twice the between-family variance (<a href="#bib25">Falconer, 1960</a>; <a href="#bib26">Falconer and Mackay, 1996</a>). Both empirical data and theory suggest additive genetic variation makes up large proportion of the total genetic variance (<a href="#bib38">Hill et al., 2008</a>).</p> <p>In addition, for <i>D. melanogaster</i> we ran a further model that included the additional fixed effects <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
v</div> </div> </div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
:</div> <div class="mi_UNKNOWN">
v</div> </div> </div> </div> </div></div></span> that are the linear effects of the <i>CHKov1</i> and <i>p62</i> (<i>Ref(2)P)</i> resistance alleles. We assumed these genes had additive effects, and modelled their effects simply as the proportion of resistant alleles in a family (if one parent was heterozygous and the other homozygous susceptible, the value is 0.25).</p> <p>The model was fitted using the MCMCglmm package in R (<a href="#bib34">Hadfield, 2010</a>). The random effects (and residuals) are assumed to be multivariate normal with zero mean and covariance structure <b>V</b> ⊗ <b>I. I</b> is an identity matrix, and <b>V</b> a matrix of estimated variances and covariances. For the random effects <b>V</b> is a 3 × 3 covariance matrix describing the variances for each virus and the covariances between them. The off-diagonal elements of <b>V</b> for the residual were set to zero because the covariances between traits at these levels are not estimable by design.</p> <p>Diffuse independent normal priors were placed on the fixed effects (means of zero and variances of 10<sup>8</sup>). Parameter expanded priors were placed on the covariance matrices resulting in scaled multivariate <i>F</i> distributions which have the property that the marginal distributions for the variances are scaled (by 1000) <i>F</i><sub>1, 1</sub>. The exceptions were the residual variances for which an inverse-gamma prior was used with shape and scale equal to 0.001. The MCMC chain was ran for 130 million iterations with a burn-in of 30 million iterations and a thinning interval of 100,000.</p> <p>We confirmed the results were not sensitive to the choice of prior by also fitting models with inverse-Wishart and flat priors for the variance covariance matrices (described in <a href="#bib51">Longdon et al., 2011c</a>), as well as fitting the models by REML in ASReml R (<a href="#bib33">Gilmour et al., 2002</a>). These analyses all gave qualitatively similar results (data not shown).</p> </div> <div class="statisticalanalysisofqtlexperiment" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis of QTL experiment</div> <p>Based on genotyping data, the probability that each Recombinant Inbred Line (RIL) in the DSPR panel was derived from each of the eight founder lines has been estimated at 10 kB intervals across the genome (<a href="#bib42">King et al., 2012</a>). To identify QTL affecting viral load we first calculated the mean viral load (ΔCt) across the biological replicates of each RIL. We then regressed the mean viral load against the eight genotype probabilities and calculated logarithm of odds (LOD) scores using the DSPRqtl package in R (<a href="#bib42">King et al., 2012</a>). These LOD scores were calculated separately for DSPR Panel A and Panel B, and then summed at each genomic location. To obtain a significance threshold, we permuted our mean viral load estimates across the RILs within each panel, repeated the analysis above and recorded the highest LOD score across the entire genome. This process was repeated 1000 times to obtain a null distribution of the maximum LOD score.</p> <p>To estimate the effect of each QTL we assumed that there was a single genetic variant affecting viral load, so the founder alleles could be assigned to two functionally distinct allelic classes. First, we regressed the mean viral load against the genotype probabilities (as described above), resulting in estimates of the mean viral load of each founder allele in the dataset. The two DSPR panels had one founder line (line 8) in common. For QTL where the line eight allele was present in both panels, this analysis included data from both panels, and ‘panel’ was included as a fixed effect in the analysis. When this was not the case, we analysed only data from the panel where the QTL was most significant. We then ranked the founder alleles by viral load estimate, and split this ranked list into all possible groups of two alleles. For each split, the genotype probabilities in the first group of founder alleles were summed. We then regressed mean viral load against each of these combined genotype probabilities. The regression model with the highest likelihood was taken as the most likely classification into allelic classes. The effect size of the QTL was then estimated from this model.</p> <p>To estimate the genetic variance in viral load within the DSPR panels we modified the model described in <a href="#equ1">Equation 1</a> as follows. <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> </div> </div> </div></div></span> is the log<sub>2 </sub>viral load of the <i>i<sup>th</sup></i> biological replicate of each RIL <i>f</i> infected with virus <i>v.</i> There was a single fixed effect, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> </div> </div></div></span>, of the panel the line is from. <i>u</i> is the random effect for each RIL (<i>f</i>). The day of injection (<i>d</i>) was omitted. As all the RILs are homozygous, we estimated the genetic variance in viral load (<i>V<sub>G</sub></i>) as half the between-RIL variance. This assumes all the genetic variation is additive.</p> <p>To estimate the proportion of the genetic variance that is explained by the QTL we identified, we repeated this analysis but included the 7 QTL we identified as fixed effects in the model. Each QTL was included by estimating the probability that each line carried the resistant allele of the QTL and adding this as a fixed effect to the model. The between-RIL variance then allowed us to estimate the genetic variance in viral load after removing the effects of the QTL.</p> </div> <div class="additionalmethods" title="sec"> <div class="title" tagx="title" title="title">
Additional methods</div> <p>All lines were screened for their retrospective sigma virus over two generations by RT-PCR, and infected isofemale lines discarded prior to the experiment.</p> </div> <div class="drosophilamelanogaster" title="sec"> <div class="title" tagx="title" title="title">
Drosophila melanogaster</div> <p>We created an outcrossed population by combining 150 isofemale lines of <i>D. melanogaster</i> (collected in Accra, Ghana (5.593,–0.188) in 2014) in a population cage. The population was maintained throughout the experiment with a large population size (~1500–2000 flies), with eggs collected from the population cage used to set up each subsequent generation. All rearing was carried out on cornmeal medium (recipe below) sprinkled with live yeast (‘yeasted’) at 25°C.</p> <p>Virgin flies were collected daily from bottles set up at a controlled egg density. Full-sib families were set up using crosses of single male and female virgins placed in the same vial and aged for 3 days. Each of these families was tipped onto fresh food daily for 5 days to create replicate vials. After 5 days the adult flies were frozen for later genotyping. 12 days after laying, male offspring were collected from each replicate vial and split into two vials of cornmeal medium without any yeast on the surface (‘unyeasted’) and placed at 18°C.</p> <p>After 5 days these flies were injected with 69 nl of virus extract intra-abdominally using a Nanoject II micro-injector (Drummond scientific). Injected flies were kept at 18°C and tipped onto fresh unyeasted cornmeal every 5 days, before being homogenised in Trizol (Invitrogen) and frozen at −80°C on day 15 post injection for later RNA extraction and qRT-PCR.</p> <p>Injections were carried out over 25 overlapping blocks. Each block consisted of 10 families, and each day, two vials per family were injected with two different viruses. Each replicate vial contained a mean of 14 flies (range 3–28 flies). In total we measured 255 families over 1567 biological replicates. We aimed to carry out a minimum of 2 replicates of each virus per family, but where possible we carried out 3 or four replicates (92 virus-family combinations had one replicate, 555 had two replicates, 82 had 3 replicates and 25 had four replicates.). 248 families had replicates for all three viruses. Blocks were staggered to overlap with at least 20 families being infected on any one day. The order families were injected in was randomised, and the order the different viruses were injected was blocked across days.</p> </div> <div class="d. immigrans" title="sec"> <div class="title" tagx="title" title="title">
D. immigrans</div> <p>92 <i>D. immigrans</i> lines were collected from Madingley, Cambridge, UK (52.225, 0.043) in 2012 and 2015. Full-sib families were set up using crosses between the 92 isofemale lines of <i>Drosophila immigrans</i>. Flies were reared on malt food (recipe below) at 18°C. Crosses were between different isofemale lines (i.e. excluding reciprocal crosses) and maximising the number of lines used. Families were established from 2 to 4 day old single female and male virgin flies placed in the same vial for 7 days. These crosses were tipped onto fresh food every 7 days to generate replicate vials of each family. Eclosed males were collected 27–34 days after initial egg laying and injected with DImmSV, DMelSV or DObsSV 1–3 days post-collection, then maintained and frozen on day 15 post-infection as above.</p> <p>Injections were carried out over 18 overlapping blocks. Each block consisted of an average of 19 families and 46 replicate vials. Each replicate vial contained a mean of 14 flies (range: 4–26). In total we assayed 341 families over 812 biological replicates. We aimed to have a minimum of 2 replicates per virus per family (235 virus-family combinations had one replicate, 270 had two replicates, 11 had 3 replicates and 1 had four replicates). 140 families had replicates across two different viruses and 18 families had replicates for all three viruses. Blocks were staggered to overlap with a mean of 39 families being infected on any one day. The order families were injected in was randomised, and the order the different viruses were injected was blocked across days.</p> </div> <div class="d. affinis" title="sec"> <div class="title" tagx="title" title="title">
D. affinis</div> <p> <div class="table-wrap_UNKNOWN" id="inlinetable1" orientation="portrait" position="anchor"> </div></p><table frame="hsides" rules="groups"> <thead> <tr> <th valign="top" rowspan="1" colspan="1">Site</th> <th valign="top" rowspan="1" colspan="1">N</th> </tr> </thead> <tbody> <tr> <td valign="top" rowspan="1" colspan="1">Athens, Georgia, USA, (33.946,–83.384) in 2012</td> <td valign="top" rowspan="1" colspan="1">13</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Great Smokey Mountain National Park, Gatlinburg, USA (35.698,–83.613) in 2015</td> <td valign="top" rowspan="1" colspan="1">23</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Rochester, New York, USA (43.135,–77.599) in 2012</td> <td valign="top" rowspan="1" colspan="1">4</td> </tr> </tbody> </table>  <p /> <p>Full-sib families were set up using crosses between 40 isofemale lines of <i>Drosophila affinis</i> (see above for collection details) collected in the U.S. Flies were reared on malt food (recipe below) at 18°C. Crosses were between different isofemale lines (i.e. excluding reciprocal crosses) and maximising the number of lines used. Families were established from 6 day old single female and male virgin flies placed in the same vial for 7 days. These crosses were tipped onto fresh food every 7 days to generate replicate vials of each family. Eclosed males were collected 35–42 days after initial egg laying and then injected with DAffSV, DImmSV or DMelSV 1–3 days post-collection, then maintained and frozen on day 15 post-infection as above.</p> <p>Injections were carried out over 27 overlapping blocks. Each block consisted of an average of 19 families and 28 replicate vials. Each replicate vial contained a mean of 11 flies (range: 3–23). In total we assayed 520 families over 1003 biological replicates. We aimed to have a minimum of 2 replicates per virus per family (336 virus-family combinations had one replicate, 286 had 2 replicates and 30 had three replicates). 109 families had replicates across two different viruses and 12 families had replicates for all three viruses. Blocks were staggered to overlap with a mean of 23 families being infected on any one day. The order families were injected in was randomised, and the order the different viruses were injected was blocked across days.</p> </div> <div class="d. obscura" title="sec"> <div class="title" tagx="title" title="title">
D. obscura</div> <p> <div class="table-wrap_UNKNOWN" id="inlinetable2" orientation="portrait" position="anchor"> </div></p><table frame="hsides" rules="groups"> <thead> <tr> <th valign="top" rowspan="1" colspan="1">Site</th> <th valign="top" rowspan="1" colspan="1">N</th> </tr> </thead> <tbody> <tr> <td valign="top" rowspan="1" colspan="1">Derbyshire Site A, UK (52.978,–1.440) in 2012</td> <td valign="top" rowspan="1" colspan="1">4</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Derbyshire Site C, UK (52.903,–1.374), in 2012</td> <td valign="top" rowspan="1" colspan="1">1</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Les Gorges du Chambon, France (45.622, 0.555) in 2012</td> <td valign="top" rowspan="1" colspan="1">1</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Madingley, Cambridge, UK (52.226, 0.046) in 2014</td> <td valign="top" rowspan="1" colspan="1">15</td> </tr> </tbody> </table>  <p /> <p><i>D. obscura</i> were collected in the United Kingdom and France (see above). Males and females were separated, and females were placed in vials to establish isofemale lines. Full-sib families were set up using crosses between 21 isofemale lines of <i>Drosophila obscura</i> collected in the UK. Flies were reared on banana food (recipe below) at 18°C. Crosses were between different isofemale lines (i.e. excluding reciprocal crosses) and maximising the number of lines used. Families were established from 6 day old single female and male virgin flies placed in the same vial for 7 days. These crosses were tipped onto fresh food every 7 days to generate replicate vials of each family. Eclosed males were collected 35–42 days after initial egg laying and then injected with DAffSV, DMelSV, or DObsSV 1–3 days post-collection, then maintained and frozen on day 15 post-infection as above.</p> <p>Injections were carried out over 25 overlapping blocks. Each block consisted of a mean of 16 families with a mean of 76 vials being injected each day for 12 days. Each replicate vial contained a mean of 8 flies (range: 1–15). In total we assayed 320 families over 913 biological replicates. We aimed to have a minimum of 2 replicates per virus per family (126 virus-family combinations had one replicate, 314 had 2 replicates and 49 had 3 replicates and 3 had four replicates). 94 families had replicates across two different viruses and 39 families had replicates for all three viruses. Blocks were staggered to overlap with at least 10 families being infected on any one day. The order families were injected in was randomised, and the order the different viruses were injected was blocked across days.</p> </div> <div class="samplesizeestimation" title="sec"> <div class="title" tagx="title" title="title">
Sample size estimation</div> <p>The number of full-sib families required for estimating genetic variance in susceptibility was determined by simulation using previous estimates of genetic variation to DMelSV in <i>D. melanogaster</i> (<a href="#bib60">Magwire et al., 2012</a>). After carrying out the full-sib experiment in <i>D. melanogaster</i>, we then down-sampled this data to calculate the minimum number of families required to provide accurate estimates for the other species. Sample sizes for the DSPR experiment were based on previous data (<a href="#bib27">Faria et al., 2015</a>).</p> </div> <div class="foodrecipes" title="sec"> <div class="title" tagx="title" title="title">
Food recipes</div> <p>Banana:</p> <ul> <li> <p>Mixture 1</p> <ul> <li> <p>1000 ml water</p> </li> <li> <p>30 g yeast</p> </li> <li> <p>10 g agar</p> </li> </ul> </li> <li> <p>Mixture 2</p> <ul> <li> <p>20 ml Nipagin</p> </li> <li> <p>150 g pureed banana</p> </li> <li> <p>50 g corn syrup</p> </li> <li> <p>30 g malt powder</p> </li> </ul> </li> </ul> <p>Bring mixture one to the boil for 3–4 min, whisk constantly. Add to Mixture two to Mixture 1. Whisk constantly and simmer for 5 min.</p> <p>Cornmeal:</p> <ul> <li> <p>1200 ml water</p> </li> <li> <p>13 g agar</p> </li> <li> <p>105 g dextrose</p> </li> <li> <p>105 g maize</p> </li> <li> <p>23 g yeast</p> </li> </ul> <p>Combine and bring to a boil for 5mins, cool to 70°C before adding 35 ml Nipagin (10%)</p> <p>Malt:</p> <ul> <li> <p>1000 ml water</p> </li> <li> <p>10 g agar</p> </li> <li> <p>60 g semolina</p> </li> <li> <p>20 g yeast</p> </li> <li> <p>80 g malt extract</p> </li> </ul> <p>Combine and bring to a boil for 5mins, cool to 70°C and then add 14 ml Nipagin (10%) and 5 ml propionic acid.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>Datasets and R code for estimating the amount of genetic variation in susceptibility <a href="https://doi.org/10.6084/m9.figshare.6743339">https://doi.org/10.6084/m9.figshare.6743339</a></p> <p>DGRP dataset <a href="https://doi.org/10.6084/m9.figshare.6743354">https://doi.org/10.6084/m9.figshare.6743354</a></p> <p>DSPR dataset and R code <a href="https://doi.org/10.6084/m9.figshare.7195751">https://doi.org/10.6084/m9.figshare.7195751</a></p> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100008668</span><span class="institution" title="institution">NERC Environmental Bioinformatics Centre</span></div> </div><span class="award-id" tagx="award-id" title="award-id">NE/L004232/1</span> to Elizabeth ML Duxbury, Jonathan P Day, Francis M Jiggins, Ben Longdon.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100004440</span><span class="institution" title="institution">Wellcome</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Sir Henry Dale Fellowship (Grant Number 109356/Z/15/Z)</span> to Ben Longdon.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100010663</span><span class="institution" title="institution">H2020 European Research Council</span></div> </div><span class="award-id" tagx="award-id" title="award-id">281668</span> to Elizabeth ML Duxbury.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100000288</span><span class="institution" title="institution">Royal Society</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Sir Henry Dale Fellowship (Grant Number 109356/Z/15/Z)</span> to Ben Longdon.<p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Many thanks to: Alastair Wilson and Jarrod Hadfield for useful advice and discussion; Stuart Macdonald for providing DSPR lines, Trudy Mackay for providing the DGRP fly lines and Kelly Dyer with help collecting <i>D. affinis</i>; Darren Obbard for providing photographs of <i>Drosophila</i> species; Camille Bonneaud, Katherine Roberts, Ryan Imrie and the Unckless lab group for constructive comments on this work. Many thanks to Brian Lazzaro, Janis Antonovics, Bruno Lemaître and one anonymous reviewer for constructive comments that greatly improved the manuscript.</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Competing interests</b></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests declared.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Author contributions</b></div> <div class="fn-type-con" title="con"> <p>Data curation, Formal analysis, Supervision, Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Data curation, Formal analysis, Validation, Investigation, Visualization, Methodology, Writing—original draft, Project administration, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Investigation, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Resources, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Data curation, Formal analysis, Supervision, Funding acquisition, Validation, Investigation, Visualization, Methodology, Writing—original draft, Project administration, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Data curation, Formal analysis, Supervision, Funding acquisition, Validation, Investigation, Visualization, Methodology, Writing—original draft, Project administration, Writing—review and editing.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.46440.007">10.7554/eLife.46440.007</a><span class="label" tagx="label" title="label">Supplementary file 1.</span> <div class="title" tagx="title" title="title">
Credible intervals of the differences between estimates of genetic variance in susceptibility across host species and viruses.</div> <p>The natural virus for each host is in red and bold. Genetic variances are estimated from the among-family variances in viral load. 95% CIs show differences in estimates of genetic variation for different host-virus combinations, intervals that do not cross zero represent statistically significant differences.</p>  <div class="media" title="media"><a href="elife-46440-supp1.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.46440.008">10.7554/eLife.46440.008</a><span class="label" tagx="label" title="label">Supplementary file 2.</span> <div class="title" tagx="title" title="title">
Estimates of genetic variation for each host virus combination.</div> <p>The natural virus for each host is in red and bold. Genetic variances are estimated from the among-family variances in viral load.</p>  <div class="media" title="media"><a href="elife-46440-supp2.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.46440.009">10.7554/eLife.46440.009</a><span class="label" tagx="label" title="label">Supplementary file 3.</span> <div class="title" tagx="title" title="title">
Table S3 Genetic correlations (<i>r<sub>g</sub></i>) in viral loads across host species after infection by coevolved and non-coevolved viruses.</div> <p>The coevolved (natural) virus for each host is in red and bold. Models ran using REML gave similar estimates.</p>  <div class="media" title="media"><a href="elife-46440-supp3.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.46440.010">10.7554/eLife.46440.010</a><span class="label" tagx="label" title="label">Supplementary file 4.</span> <div class="title" tagx="title" title="title">
Table S4 Primers for genotyping<i>D. melanogaster</i>resistance genes<i>Ge-1, p62 (Ref(2)P) and CHKov-1.</i></div> <p>PCRs were carried out using a touchdown PCR cycle (95°C 30sec, 62°C (-1°C per cycle) 30sec, 72°C 1min; for 10x cycles followed by; 95°C 30sec, 52°C 30sec, 72°C 1min; for a further 25x cycles).</p>  <div class="media" title="media"><a href="elife-46440-supp4.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.46440.011">10.7554/eLife.46440.011</a><span class="label" tagx="label" title="label">Supplementary file 5.</span> <div class="title" tagx="title" title="title">
Table S5 QTL and their locations.</div>  <div class="media" title="media"><a href="elife-46440-supp5.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.46440.012">10.7554/eLife.46440.012</a><span class="label" tagx="label" title="label">Supplementary file 6.</span> <div class="title" tagx="title" title="title">
Table S6 Primers for qRT-PCR (5’-3’).</div> <p>RpL32 primers overlap an intron-exon boundary. Sigma virus primers cross gene boundaries except for DImmSV that amplifies the L gene</p>  <div class="media" title="media"><a href="elife-46440-supp6.docx">LINK</a></div> </div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.46440.013">10.7554/eLife.46440.013</a><span class="label" tagx="label" title="label">Transparent reporting form</span><div class="media" title="media"><a href="elife-46440-transrepform.pdf">LINK</a></div> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>All data generated or analysed during this study are available at Figshare: Datasets and R code for estimating the amount of genetic variation in susceptibility <a href="https://doi.org/10.6084/m9.figshare.6743339">https://doi.org/10.6084/m9.figshare.6743339</a>; DGRP dataset <a href="https://doi.org/10.6084/m9.figshare.6743354">https://doi.org/10.6084/m9.figshare.6743354</a>; DSPR dataset and R code <a href="https://doi.org/10.6084/m9.figshare.7195751">https://doi.org/10.6084/m9.figshare.7195751</a>.</p> <p>The following datasets were generated:</p> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elizabeth</span><span class="given-names" tagx="given-names" title="given-names">ML Duxbury</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonathan</span><span class="given-names" tagx="given-names" title="given-names">P Day</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davide</span><span class="given-names" tagx="given-names" title="given-names">Maria Vespasiani</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yannik</span><span class="given-names" tagx="given-names" title="given-names">Thüringer</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ignacio</span><span class="given-names" tagx="given-names" title="given-names">Tolosana</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sophia</span><span class="given-names" tagx="given-names" title="given-names">CL Smith</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucia</span><span class="given-names" tagx="given-names" title="given-names">Tagliaferri</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altug</span><span class="given-names" tagx="given-names" title="given-names">Kamacioglu</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imogen</span><span class="given-names" tagx="given-names" title="given-names">Lindsley</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luca</span><span class="given-names" tagx="given-names" title="given-names">Love</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robert</span><span class="given-names" tagx="given-names" title="given-names">L Unckless</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francis</span><span class="given-names" tagx="given-names" title="given-names">M Jiggins</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben</span><span class="given-names" tagx="given-names" title="given-names">Longdon</span></span></span><span class="year" tagx="year" title="year">2019</span>[data-title] Susceptibility of different Drosophila melanogaster DGRP lines to three sigma viruses<span class="source" tagx="source" title="source">Figshare</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.6743354">10.6084/m9.figshare.6743354</a></span></span></p> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elizabeth</span><span class="given-names" tagx="given-names" title="given-names">ML Duxbury</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jonathan</span><span class="given-names" tagx="given-names" title="given-names">P Day</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davide</span><span class="given-names" tagx="given-names" title="given-names">Maria Vespasiani</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yannik</span><span class="given-names" tagx="given-names" title="given-names">Thüringer</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ignacio</span><span class="given-names" tagx="given-names" title="given-names">Tolosana</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sophia</span><span class="given-names" tagx="given-names" title="given-names">CL Smith</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucia</span><span class="given-names" tagx="given-names" title="given-names">Tagliaferri</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altug</span><span class="given-names" tagx="given-names" title="given-names">Kamacioglu</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imogen</span><span class="given-names" tagx="given-names" title="given-names">Lindsley</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luca</span><span class="given-names" tagx="given-names" title="given-names">Love</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robert</span><span class="given-names" tagx="given-names" title="given-names">L Unckless</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francis</span><span class="given-names" tagx="given-names" title="given-names">M Jiggins</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben</span><span class="given-names" tagx="given-names" title="given-names">Longdon</span></span></span><span class="year" tagx="year" title="year">2019</span>[data-title] Natural selection by pathogens increases genetic variation in susceptibility to infection<span class="source" tagx="source" title="source">Figshare</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.6743339">10.6084/m9.figshare.6743339</a></span></span></p> <p><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elizabeth</span><span class="given-names" tagx="given-names" title="given-names">ML Duxbury</span></span></span><span class="year" tagx="year" title="year">2019</span>[data-title] Susceptibility of Drosophila melanogaster DSPR lines to three sigma viruses<span class="source" tagx="source" title="source">Figshare</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.7195751">10.6084/m9.figshare.7195751</a></span></span></p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso-Blanco</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Méndez-Vigo</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Genetic architecture of naturally occurring quantitative traits in plants: an updated synthesis</span><span class="source" tagx="source" title="source">Current Opinion in Plant Biology</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">37</span><span class="lpage" tagx="lpage" title="lpage">43</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pbi.2014.01.002">10.1016/j.pbi.2014.01.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24565952">24565952</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonovics</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thrall</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burdon</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laine</span><span class="given-names" tagx="given-names" title="given-names">A-L</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Partial resistance in the linum-melampsora host-pathogen system: does partial resistance make the red queen run slower?</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">512</span><span class="lpage" tagx="lpage" title="lpage">522</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.2010.01146.x">10.1111/j.1558-5646.2010.01146.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21029078">21029078</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonovics</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thrall</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span></span><span class="year" tagx="year" title="year">1994</span><span class="mixed-article-title" title="mixed-article-title">Cost of resistance and the maintenance of Genetic-Polymorphism in Host-Pathogen systems</span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B-Biological Sciences</span><span class="volume" tagx="volume" title="volume">257</span><span class="fpage" tagx="fpage" title="fpage">105</span><span class="lpage" tagx="lpage" title="lpage">110</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.1994.0101">10.1098/rspb.1994.0101</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antonovics</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary determinants of genetic variation in susceptibility to infectious diseases in humans</span><span class="source" tagx="source" title="source">PLOS ONE</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e29089</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0029089">10.1371/journal.pone.0029089</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22242158">22242158</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bangham</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">K-W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haddrill</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">The age and evolution of an antiviral resistance mutation in <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">274</span><span class="fpage" tagx="fpage" title="fpage">2027</span><span class="lpage" tagx="lpage" title="lpage">2034</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2007.0611">10.1098/rspb.2007.0611</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bangham</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2008a</span><span class="mixed-article-title" title="mixed-article-title">Genetic variation affecting host-parasite interactions: different genes affect different aspects of sigma virus replication and transmission in <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">178</span><span class="fpage" tagx="fpage" title="fpage">2191</span><span class="lpage" tagx="lpage" title="lpage">2199</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.107.085449">10.1534/genetics.107.085449</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18430944">18430944</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bangham</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knott</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2008b</span><span class="mixed-article-title" title="mixed-article-title">Genetic variation affecting host-parasite interactions: major-effect quantitative trait loci affect the transmission of sigma virus in <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">3800</span><span class="lpage" tagx="lpage" title="lpage">3807</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-294X.2008.03873.x">10.1111/j.1365-294X.2008.03873.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18665899">18665899</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barton</span><span class="given-names" tagx="given-names" title="given-names">NH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turelli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">Adaptive landscapes, genetic distance and the evolution of quantitative characters</span><span class="source" tagx="source" title="source">Genetical Research</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">173</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0016672300026951">10.1017/S0016672300026951</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3596236">3596236</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beerntsen</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christensen</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Genetics of mosquito vector competence</span><span class="source" tagx="source" title="source">Microbiology and Molecular Biology Reviews</span><span class="volume" tagx="volume" title="volume">64</span><span class="fpage" tagx="fpage" title="fpage">115</span><span class="lpage" tagx="lpage" title="lpage">137</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MMBR.64.1.115-137.2000">10.1128/MMBR.64.1.115-137.2000</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10704476">10704476</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Best</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">The implications of coevolutionary dynamics to host-parasite interactions</span><span class="source" tagx="source" title="source">The American Naturalist</span><span class="volume" tagx="volume" title="volume">173</span><span class="fpage" tagx="fpage" title="fpage">779</span><span class="lpage" tagx="lpage" title="lpage">791</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/598494">10.1086/598494</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19374557">19374557</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blows</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">A reassessment of genetic limits to evolutionary change</span><span class="source" tagx="source" title="source">Ecology</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">1371</span><span class="lpage" tagx="lpage" title="lpage">1384</span><span class="pub-id"><a href="https://dx.doi.org/10.1890/04-1209">10.1890/04-1209</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">White</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Best</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowers</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">How specificity and epidemiology drive the coevolution of static trait diversity in hosts and parasites</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">1594</span><span class="lpage" tagx="lpage" title="lpage">1606</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/evo.12393">10.1111/evo.12393</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24593303">24593303</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boots</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haraguchi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">The evolution of costly resistance in Host-Parasite systems</span><span class="source" tagx="source" title="source">The American Naturalist</span><span class="volume" tagx="volume" title="volume">153</span><span class="fpage" tagx="fpage" title="fpage">359</span><span class="lpage" tagx="lpage" title="lpage">370</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/303181">10.1086/303181</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29586625">29586625</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bou Sleiman</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osman</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Massouras</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemaitre</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deplancke</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Genetic, molecular and physiological basis of variation in <i>Drosophila</i> gut immunocompetence</span><span class="source" tagx="source" title="source">Nature Communications</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">7829</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms8829">10.1038/ncomms8829</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26213329">26213329</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bregliano</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="year" tagx="year" title="year">1970</span><span class="mixed-article-title" title="mixed-article-title">Study of infection of germ line in female <i>Drosophila</i> infected with sigma virus. 2. evidence of a correspondance between ovarian cysts with increased virus yield and stabilized progeny</span><span class="source" tagx="source" title="source">Annales De L Institut Pasteur</span><span class="volume" tagx="volume" title="volume">119</span><span class="fpage" tagx="fpage" title="fpage">685</span><span class="lpage" tagx="lpage" title="lpage">704</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5533948">5533948</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brun</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plus</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">1980</span><span class="chapter-title" tagx="chapter-title" title="chapter-title">The viruses of <i>Drosophila</i></span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashburner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">T. R. F</span></span></span><span class="source" tagx="source" title="source">The Genetics and Biology of Drosophila</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span><span class="fpage" tagx="fpage" title="fpage">625</span><span class="lpage" tagx="lpage" title="lpage">702</span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burgner</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jamieson</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blackwell</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Genetic susceptibility to infectious diseases: big is beautiful, but will bigger be even better?</span><span class="source" tagx="source" title="source">The Lancet Infectious Diseases</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">653</span><span class="lpage" tagx="lpage" title="lpage">663</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(06)70601-6">10.1016/S1473-3099(06)70601-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17008174">17008174</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magwire</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayer</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">A polymorphism in the processing body component Ge-1 controls resistance to a naturally occurring rhabdovirus in <i>Drosophila</i></span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1005387</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1005387">10.1371/journal.ppat.1005387</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26799957">26799957</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cogni</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barbier</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Complex coding and regulatory polymorphisms in a restriction factor determine the susceptibility of <i>Drosophila</i> to Viral Infection</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">206</span><span class="fpage" tagx="fpage" title="fpage">2159</span><span class="lpage" tagx="lpage" title="lpage">2173</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.117.201970">10.1534/genetics.117.201970</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28630113">28630113</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chapman</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Human genetic susceptibility to infectious disease</span><span class="source" tagx="source" title="source">Nature Reviews Genetics</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">175</span><span class="lpage" tagx="lpage" title="lpage">188</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrg3114">10.1038/nrg3114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22310894">22310894</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christian</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">Studies of Drosophila C and a Viruses in Australian Populations of Drosophila Melanogaster</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Australian National University</span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cogni</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bridson</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">The genetic architecture of resistance to virus infection in <i>Drosophila</i></span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">5228</span><span class="lpage" tagx="lpage" title="lpage">5241</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.13769">10.1111/mec.13769</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27460507">27460507</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Contamine</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petitjean</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashburner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1989</span><span class="mixed-article-title" title="mixed-article-title">Genetic resistance to viral infection: the molecular cloning of a <i>Drosophila</i> gene that restricts infection by the rhabdovirus sigma</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">123</span><span class="fpage" tagx="fpage" title="fpage">525</span><span class="lpage" tagx="lpage" title="lpage">533</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2557263">2557263</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooke</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Genetics of susceptibility to human infectious disease</span><span class="source" tagx="source" title="source">Nature Reviews Genetics</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">967</span><span class="lpage" tagx="lpage" title="lpage">977</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/35103577">10.1038/35103577</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11733749">11733749</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falconer</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span></span><span class="year" tagx="year" title="year">1960</span><span class="source" tagx="source" title="source">Introduction to Quantitative Genetics</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Edinburgh/London</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oliver &amp;amp; Boyd</span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falconer</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackay</span><span class="given-names" tagx="given-names" title="given-names">TFC</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="source" tagx="source" title="source">Introduction to Quantitative Genetics</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Essex, UK</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Longmans Green, Harlow</span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">VG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martins</span><span class="given-names" tagx="given-names" title="given-names">NE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teixeira</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sucena</span><span class="given-names" tagx="given-names" title="given-names">Élio</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magalhães</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Evolution of <i>Drosophila</i> resistance against different pathogens and infection routes entails no detectable maintenance costs</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">2799</span><span class="lpage" tagx="lpage" title="lpage">2809</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/evo.12782">10.1111/evo.12782</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26496003">26496003</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisher</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">1930</span><span class="source" tagx="source" title="source">The Genetical Theory of Natural Selection</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford, UK</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Clarendon Press</span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleuriet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1988</span><span class="mixed-article-title" title="mixed-article-title">Maintenance of a hereditary virus: the Sigma-Virus in populations of its host, <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4613-1043-3_1">10.1007/978-1-4613-1043-3_1</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleuriet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Periquet</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title">Evolution of the <i>Drosophila melanogaster</i>-sigma virus system in natural populations from Languedoc (southern France)</span><span class="source" tagx="source" title="source">Archives of Virology</span><span class="volume" tagx="volume" title="volume">129</span><span class="fpage" tagx="fpage" title="fpage">131</span><span class="lpage" tagx="lpage" title="lpage">143</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF01316890">10.1007/BF01316890</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8470948">8470948</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleuriet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sperlich</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="mixed-article-title" title="mixed-article-title">Evolution of the <i>Drosophila melanogaster</i>-sigma virus system in a natural population from tübingen</span><span class="source" tagx="source" title="source">TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">186</span><span class="lpage" tagx="lpage" title="lpage">189</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF00222858">10.1007/BF00222858</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24197303">24197303</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibson</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Population genetics and GWAS: a primer</span><span class="source" tagx="source" title="source">PLOS Biology</span><span class="volume" tagx="volume" title="volume">16</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e2005485</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.2005485">10.1371/journal.pbio.2005485</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29547618">29547618</a></span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilmour</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gogel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cullis</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">WElham</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2002</span>[data-title] <i>ASReml User Guide Release 1.0</i><span class="source" tagx="source" title="source">Vsni</span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">MCMC methods for Multi-Response generalized Linear mixed models: the MCMCglmm <i>R</i> package</span><span class="source" tagx="source" title="source">Journal of Statistical Software</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="https://dx.doi.org/10.18637/jss.v033.i02">10.18637/jss.v033.i02</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20808728">20808728</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haldane</span><span class="given-names" tagx="given-names" title="given-names">JBS</span></span></span><span class="year" tagx="year" title="year">1949</span><span class="mixed-article-title" title="mixed-article-title">Disease and evolution</span><span class="source" tagx="source" title="source">Ric Sci Suppl</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">68</span><span class="lpage" tagx="lpage" title="lpage">76</span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hammond-Kosack</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Plant disease resistance genes</span><span class="source" tagx="source" title="source">Annual Review of Plant Physiology and Plant Molecular Biology</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">575</span><span class="lpage" tagx="lpage" title="lpage">607</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.arplant.48.1.575">10.1146/annurev.arplant.48.1.575</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15012275">15012275</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">Rates of change in quantitative traits from fixation of new mutations</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">142</span><span class="lpage" tagx="lpage" title="lpage">145</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.79.1.142">10.1073/pnas.79.1.142</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6948296">6948296</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">WG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goddard</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Visscher</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Data and theory point to mainly additive genetic variance for complex traits</span><span class="source" tagx="source" title="source">PLOS Genetics</span><span class="volume" tagx="volume" title="volume">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000008</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1000008">10.1371/journal.pgen.1000008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18454194">18454194</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">AVS</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Evolution, revolution and heresy in the genetics of infectious disease susceptibility</span><span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">367</span><span class="fpage" tagx="fpage" title="fpage">840</span><span class="lpage" tagx="lpage" title="lpage">849</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2011.0275">10.1098/rstb.2011.0275</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hotson</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title"><i>Drosophila melanogaster</i> natural variation affects growth dynamics of infecting listeria monocytogenes</span><span class="source" tagx="source" title="source">G3: Genes|Genomes|Genetics</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">2593</span><span class="lpage" tagx="lpage" title="lpage">2600</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/g3.115.022558">10.1534/g3.115.022558</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26438294">26438294</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howick</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazzaro</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The genetic architecture of defence as resistance to and tolerance of bacterial infection in <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">1533</span><span class="lpage" tagx="lpage" title="lpage">1546</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.14017">10.1111/mec.14017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28099780">28099780</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macdonald</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Properties and power of the <i>Drosophila</i> synthetic population resource for the routine dissection of complex traits</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">191</span><span class="fpage" tagx="fpage" title="fpage">935</span><span class="lpage" tagx="lpage" title="lpage">949</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.112.138537">10.1534/genetics.112.138537</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22505626">22505626</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lively</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Does genetic diversity limit disease spread in natural host populations?</span><span class="source" tagx="source" title="source">Heredity</span><span class="volume" tagx="volume" title="volume">109</span><span class="fpage" tagx="fpage" title="fpage">199</span><span class="lpage" tagx="lpage" title="lpage">203</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/hdy.2012.33">10.1038/hdy.2012.33</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22713998">22713998</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koskella</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Resistance gained, resistance lost: an explanation for host-parasite coexistence</span><span class="source" tagx="source" title="source">PLOS Biology</span><span class="volume" tagx="volume" title="volume">16</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e3000013</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.3000013">10.1371/journal.pbio.3000013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30248103">30248103</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazzaro</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sceurman</span><span class="given-names" tagx="given-names" title="given-names">BK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Genetic basis of natural variation in <i>D. melanogaster</i> antibacterial immunity</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">303</span><span class="fpage" tagx="fpage" title="fpage">1873</span><span class="lpage" tagx="lpage" title="lpage">1876</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1092447">10.1126/science.1092447</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15031506">15031506</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lively</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">The effect of host genetic diversity on disease spread</span><span class="source" tagx="source" title="source">The American Naturalist</span><span class="volume" tagx="volume" title="volume">175</span><span class="fpage" tagx="fpage" title="fpage">E149</span><span class="lpage" tagx="lpage" title="lpage">E152</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/652430">10.1086/652430</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20388005">20388005</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macdonald</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">EG</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Dissecting complex traits using the <i>Drosophila</i> synthetic population resource</span><span class="source" tagx="source" title="source">Trends in Genetics</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">488</span><span class="lpage" tagx="lpage" title="lpage">495</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tig.2014.07.009">10.1016/j.tig.2014.07.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25175100">25175100</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Sigma viruses from three species of <i>Drosophila</i> form a major new clade in the rhabdovirus phylogeny</span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">277</span><span class="fpage" tagx="fpage" title="fpage">35</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2009.1472">10.1098/rspb.2009.1472</a></span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilfert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2011a</span><span class="mixed-article-title" title="mixed-article-title">Rhabdoviruses in two species of <i>Drosophila</i>: vertical transmission and a recent sweep</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">188</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">150</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.111.127696">10.1534/genetics.111.127696</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21339477">21339477</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilfert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osei-Poku</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cagney</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2011b</span><span class="mixed-article-title" title="mixed-article-title">Host-switching by a vertically transmitted rhabdovirus in <i>Drosophila</i></span><span class="source" tagx="source" title="source">Biology Letters</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">747</span><span class="lpage" tagx="lpage" title="lpage">750</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsbl.2011.0160">10.1098/rsbl.2011.0160</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21450721">21450721</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2011c</span><span class="mixed-article-title" title="mixed-article-title">Host phylogeny determines viral persistence and replication in novel hosts</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1002260</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002260">10.1371/journal.ppat.1002260</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21966271">21966271</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilfert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">The Sigma Viruses of <i>Drosophila</i></span><span class="source" tagx="source" title="source">Norfolk, UK, Caister Academic Press</span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockhurst</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">The evolution and genetics of virus host shifts</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004395</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004395">10.1371/journal.ppat.1004395</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25375777">25375777</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">GG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2015a</span><span class="mixed-article-title" title="mixed-article-title">The evolution, diversity, and host associations of rhabdoviruses</span><span class="source" tagx="source" title="source">Virus Evolution</span><span class="volume" tagx="volume" title="volume">1</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">vev014</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vev014">10.1093/ve/vev014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774286">27774286</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGonigle</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cogni</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2015b</span><span class="mixed-article-title" title="mixed-article-title">The causes and consequences of changes in virulence following pathogen host shifts</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004728</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004728">10.1371/journal.ppat.1004728</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25774803">25774803</a></span></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulz</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leftwich</span><span class="given-names" tagx="given-names" title="given-names">PT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breuker</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilfert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">SCL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGonigle</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houslay</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">LI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Livraghi</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evans</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friend</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chapman</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vontas</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kambouraki</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Vertically transmitted rhabdoviruses are found across three insect families and have dynamic interactions with their hosts</span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">284</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">20162381</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2016.2381">10.1098/rspb.2016.2381</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Vertically transmitted viral endosymbionts of insects: do sigma viruses walk alone?</span><span class="source" tagx="source" title="source">Proceedings of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">279</span><span class="fpage" tagx="fpage" title="fpage">3889</span><span class="lpage" tagx="lpage" title="lpage">3898</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2012.1208">10.1098/rspb.2012.1208</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22859592">22859592</a></span></span></li> <li tag="ref"><a name="bib58" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackay</span><span class="given-names" tagx="given-names" title="given-names">TF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barbadilla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayroles</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casillas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magwire</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cridland</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anholt</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrón</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bess</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blankenburg</span><span class="given-names" tagx="given-names" title="given-names">KP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carbone</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castellano</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaboub</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duncan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Javaid</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jayaseelan</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jhangiani</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jordan</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lara</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawrence</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Librado</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linheiro</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyman</span><span class="given-names" tagx="given-names" title="given-names">RF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackey</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munidasa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muzny</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nazareth</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Newsham</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perales</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pu</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ràmia</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reid</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rollmann</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozas</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saada</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turlapati</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Worley</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">YQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamamoto</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergman</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thornton</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mittelman</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">The <i>Drosophila melanogaster</i> genetic reference panel</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">482</span><span class="fpage" tagx="fpage" title="fpage">173</span><span class="lpage" tagx="lpage" title="lpage">178</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature10811">10.1038/nature10811</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22318601">22318601</a></span></span></li> <li tag="ref"><a name="bib59" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magwire</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayer</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Successive increases in the resistance of <i>Drosophila</i> to viral infection through a transposon insertion followed by a duplication</span><span class="source" tagx="source" title="source">PLOS Genetics</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1002337</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1002337">10.1371/journal.pgen.1002337</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22028673">22028673</a></span></span></li> <li tag="ref"><a name="bib60" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magwire</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fabian</span><span class="given-names" tagx="given-names" title="given-names">DK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schweyen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayer</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Genome-wide association studies reveal a simple genetic basis of resistance to naturally coevolving viruses in <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">PLOS Genetics</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003057</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1003057">10.1371/journal.pgen.1003057</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23166512">23166512</a></span></span></li> <li tag="ref"><a name="bib61" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maori</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanne</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sela</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Reciprocal sequence exchange between non-retro viruses and hosts leading to the appearance of new host phenotypes</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">368</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">218</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2006.11.038">10.1016/j.virol.2006.11.038</a></span></span></li> <li tag="ref"><a name="bib62" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGonigle</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leitão</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ommeslag</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Parallel and costly changes to cellular immunity underlie the evolution of parasitoid resistance in three <i>Drosophila</i> species</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">13</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1006683</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006683">10.1371/journal.ppat.1006683</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29049362">29049362</a></span></span></li> <li tag="ref"><a name="bib63" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKean</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yourth</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazzaro</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The evolutionary costs of immunological maintenance and deployment</span><span class="source" tagx="source" title="source">BMC Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">76</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-8-76">10.1186/1471-2148-8-76</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18315877">18315877</a></span></span></li> <li tag="ref"><a name="bib64" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nuismer</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomulkiewicz</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Coevolution between hosts and parasites with partially overlapping geographic ranges</span><span class="source" tagx="source" title="source">Journal of Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">1337</span><span class="lpage" tagx="lpage" title="lpage">1345</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1420-9101.2003.00609.x">10.1046/j.1420-9101.2003.00609.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14640425">14640425</a></span></span></li> <li tag="ref"><a name="bib65" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maclennan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Grady</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Estimating divergence dates and substitution rates in the <i>Drosophila</i> phylogeny</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">3459</span><span class="lpage" tagx="lpage" title="lpage">3473</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mss150">10.1093/molbev/mss150</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22683811">22683811</a></span></span></li> <li tag="ref"><a name="bib66" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudas</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">The genetics of host-virus coevolution in invertebrates</span><span class="source" tagx="source" title="source">Current Opinion in Virology</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">73</span><span class="lpage" tagx="lpage" title="lpage">78</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2014.07.002">10.1016/j.coviro.2014.07.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25063907">25063907</a></span></span></li> <li tag="ref"><a name="bib67" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohanessian-Guillemain</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1963</span><span class="mixed-article-title" title="mixed-article-title">Etude de facteurs genetiques controlant les relations Du virus sigma et de la drosophile son hote</span><span class="source" tagx="source" title="source">Annales De Genetique</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">G1</span><span class="lpage" tagx="lpage" title="lpage">G64</span></span></li> <li tag="ref"><a name="bib68" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orr</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irving</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">The genetics of adaptation: the genetic basis of resistance to wasp parasitism in <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">1877</span><span class="lpage" tagx="lpage" title="lpage">1885</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.1997.tb05110.x">10.1111/j.1558-5646.1997.tb05110.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28565120">28565120</a></span></span></li> <li tag="ref"><a name="bib69" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostfeld</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keesing</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Effects of host diversity on infectious disease</span><span class="source" tagx="source" title="source">Annual Review of Ecology, Evolution, and Systematics</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">182</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-ecolsys-102710-145022">10.1146/annurev-ecolsys-102710-145022</a></span></span></li> <li tag="ref"><a name="bib70" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">WH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medd</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beard</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Isolation of a natural DNA virus of <i>Drosophila melanogaster</i>, and characterisation of host resistance and immune responses</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007050</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1007050">10.1371/journal.ppat.1007050</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29864164">29864164</a></span></span></li> <li tag="ref"><a name="bib71" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parrish</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morens</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calisher</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laughlin</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saif</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daszak</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Cross-species virus transmission and the emergence of new epidemic diseases</span><span class="source" tagx="source" title="source">Microbiology and Molecular Biology Reviews</span><span class="volume" tagx="volume" title="volume">72</span><span class="fpage" tagx="fpage" title="fpage">457</span><span class="lpage" tagx="lpage" title="lpage">470</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/MMBR.00004-08">10.1128/MMBR.00004-08</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18772285">18772285</a></span></span></li> <li tag="ref"><a name="bib72" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kichaev</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasaniuc</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Contrasting the genetic architecture of 30 complex traits from summary association data</span><span class="source" tagx="source" title="source">The American Journal of Human Genetics</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">139</span><span class="lpage" tagx="lpage" title="lpage">153</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ajhg.2016.05.013">10.1016/j.ajhg.2016.05.013</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27346688">27346688</a></span></span></li> <li tag="ref"><a name="bib73" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramanian</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Temporal patterns of fruit fly (<i>Drosophila</i>) evolution revealed by mutation clocks</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">36</span><span class="lpage" tagx="lpage" title="lpage">44</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msg236">10.1093/molbev/msg236</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12949132">12949132</a></span></span></li> <li tag="ref"><a name="bib74" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Specific hypotheses on the geographic mosaic of coevolution</span><span class="source" tagx="source" title="source">The American Naturalist</span><span class="volume" tagx="volume" title="volume">153</span><span class="fpage" tagx="fpage" title="fpage">S1</span><span class="lpage" tagx="lpage" title="lpage">S14</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/303208">10.1086/303208</a></span></span></li> <li tag="ref"><a name="bib75" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tinsley</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanford</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Genetic variation in <i>Drosophila melanogaster</i> pathogen susceptibility</span><span class="source" tagx="source" title="source">Parasitology</span><span class="volume" tagx="volume" title="volume">132</span><span class="fpage" tagx="fpage" title="fpage">767</span><span class="lpage" tagx="lpage" title="lpage">773</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0031182006009929">10.1017/S0031182006009929</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16497252">16497252</a></span></span></li> <li tag="ref"><a name="bib76" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Unckless</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rottschaefer</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazzaro</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The complex contributions of genetics and nutrition to immunity in <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">PLOS Genetics</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1005030</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pgen.1005030">10.1371/journal.pgen.1005030</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25764027">25764027</a></span></span></li> <li tag="ref"><a name="bib77" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">St Leger</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">The genetic basis for variation in resistance to infection in the <i>Drosophila melanogaster</i> genetic reference panel</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">13</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1006260</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006260">10.1371/journal.ppat.1006260</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28257468">28257468</a></span></span></li> <li tag="ref"><a name="bib78" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wayne</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Contamine</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kreitman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Molecular population genetics of ref(2)P, a locus which confers viral resistance in <i>Drosophila</i></span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">191</span><span class="lpage" tagx="lpage" title="lpage">199</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a025555">10.1093/oxfordjournals.molbev.a025555</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8583891">8583891</a></span></span></li> <li tag="ref"><a name="bib79" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waldron</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robertson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crowson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrari</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quintana</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brouqui</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bayne</span><span class="given-names" tagx="given-names" title="given-names">EH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buck</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lazzaro</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akorli</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haddrill</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">The discovery, distribution, and evolution of viruses associated with <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">PLOS Biology</span><span class="volume" tagx="volume" title="volume">13</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1002210</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.1002210">10.1371/journal.pbio.1002210</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26172158">26172158</a></span></span></li> <li tag="ref"><a name="bib80" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilfert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2010a</span><span class="mixed-article-title" title="mixed-article-title">Host-parasite coevolution: genetic variation in a virus population and the interaction with a host gene</span><span class="source" tagx="source" title="source">Journal of Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1447</span><span class="lpage" tagx="lpage" title="lpage">1455</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1420-9101.2010.02002.x">10.1111/j.1420-9101.2010.02002.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20456575">20456575</a></span></span></li> <li tag="ref"><a name="bib81" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilfert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2010b</span><span class="mixed-article-title" title="mixed-article-title">Disease association mapping in <i>Drosophila</i> can be replicated in the wild</span><span class="source" tagx="source" title="source">Biology Letters</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">666</span><span class="lpage" tagx="lpage" title="lpage">668</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsbl.2010.0329">10.1098/rsbl.2010.0329</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20444760">20444760</a></span></span></li> <li tag="ref"><a name="bib82" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilfert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">The dynamics of reciprocal selective sweeps of host resistance and a parasite counter-adaptation in <i>Drosophila</i></span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">761</span><span class="lpage" tagx="lpage" title="lpage">773</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1558-5646.2012.01832.x">10.1111/j.1558-5646.2012.01832.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23461326">23461326</a></span></span></li> <li tag="ref"><a name="bib83" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilfert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Flies on the move: an inherited virus mirrors <i>Drosophila melanogaster</i>'s elusive ecology and demography</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">2093</span><span class="lpage" tagx="lpage" title="lpage">2104</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.12709">10.1111/mec.12709</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24597631">24597631</a></span></span></li> <li tag="ref"><a name="bib84" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domingo</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charlesworth</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Biological and biomedical implications of the co-evolution of pathogens and their hosts</span><span class="source" tagx="source" title="source">Nature Genetics</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">569</span><span class="lpage" tagx="lpage" title="lpage">577</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng1202-569">10.1038/ng1202-569</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12457190">12457190</a></span></span></li> <li tag="ref"><a name="bib85" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yampolsky</span><span class="given-names" tagx="given-names" title="given-names">LY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shabalina</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kondrashov</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Rapid accumulation of a vertically transmitted parasite triggered by relaxation of natural selection among hosts</span><span class="source" tagx="source" title="source">Evolutionary Ecology Research</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">581</span><span class="lpage" tagx="lpage" title="lpage">589</span></span></li> <li tag="ref"><a name="bib86" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benyamin</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McEvoy</span><span class="given-names" tagx="given-names" title="given-names">BP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gordon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henders</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nyholt</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madden</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heath</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">NG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montgomery</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goddard</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Visscher</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Common SNPs explain a large proportion of the heritability for human height</span><span class="source" tagx="source" title="source">Nature Genetics</span><span class="volume" tagx="volume" title="volume">42</span><span class="fpage" tagx="fpage" title="fpage">565</span><span class="lpage" tagx="lpage" title="lpage">569</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng.608">10.1038/ng.608</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20562875">20562875</a></span></span></li> <li tag="ref"><a name="bib87" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chenoweth</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGraw</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Effective but costly, evolved mechanisms of defense against a virulent opportunistic pathogen in <i>Drosophila melanogaster</i></span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">5</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1000385">10.1371/journal.ppat.1000385</a></span></span></li> </ul> </div> </div>  </body></html>