<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Pathog</span><span class="iso-abbrev" title="iso-abbrev">PLoS Pathog</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plospath</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Pathogens</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7366</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5897010">5897010</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.ppat.1006951">10.1371/journal.ppat.1006951</a></span><span class="publisher-id" title="publisher-id">PPATHOGENS-D-18-00070</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span>: <span class="subject" title="subject">Microbial Genetics</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Replication</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span>: <span class="subject" title="subject">Microbial Mutation</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Processes</span>: <span class="subject" title="subject">Parallel Evolution</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span>: <span class="subject" title="subject">Viral Pathogens</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span>: <span class="subject" title="subject">Viral Pathogens</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span>: <span class="subject" title="subject">Viral Pathogens</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Experimental Organism Systems</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Model Organisms</span>: <span class="subject" title="subject">Drosophila Melanogaster</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Model Organisms</span>: <span class="subject" title="subject">Drosophila Melanogaster</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Experimental Organism Systems</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Models</span>: <span class="subject" title="subject">Drosophila Melanogaster</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Eukaryota</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animals</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Invertebrates</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Arthropoda</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Insects</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Drosophila</span>: <span class="subject" title="subject">Drosophila Melanogaster</span></div> </div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span>: <span class="subject" title="subject">Evolutionary Genetics</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Genomic Libraries</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Genomic Libraries</span></div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Host shifts result in parallel genetic changes when viruses evolve in closely related species</div> <div class="alt-title" title="alt-title">
Virus evolution following host shifts</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6936-1697</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">Ben</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">Jonathan P.</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6138-9134</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alves</span><span class="given-names" tagx="given-names" title="given-names">Joel M.</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">Sophia C. L.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houslay</span><span class="given-names" tagx="given-names" title="given-names">Thomas M.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-8390-2867</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGonigle</span><span class="given-names" tagx="given-names" title="given-names">John E.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tagliaferri</span><span class="given-names" tagx="given-names" title="given-names">Lucia</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-7470-8157</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">Francis M.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff002"><sup>2</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Biosciences, College of Life &amp;amp; Environmental Sciences, University of Exeter, Penryn Campus, Penryn, United Kingdom</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Department of Genetics, University of Cambridge, Cambridge, United Kingdom</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">CIBIO, Centro de Investigação em Biodiversidade e Recursos Genéticos, InBIO Laboratório Associado, Universidade do Porto, Vairão, Portugal</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lauring</span><span class="given-names" tagx="given-names" title="given-names">Adam S.</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">University of Michigan, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">b.longdon2@exeter.ac.uk</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018-4</span></span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1006951</span><span class="history" title="history"><span class="received" title="received">received: 2018-1-15</span><span class="accepted" title="accepted">accepted: 2018-2-27</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Longdon et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="ppat.1006951.pdf" title="self-uri">ppat.1006951.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Host shifts, where a pathogen invades and establishes in a new host species, are a major source of emerging infectious diseases. They frequently occur between related host species and often rely on the pathogen evolving adaptations that increase their fitness in the novel host species. To investigate genetic changes in novel hosts, we experimentally evolved replicate lineages of an RNA virus (Drosophila C Virus) in 19 different species of Drosophilidae and deep sequenced the viral genomes. We found a strong pattern of parallel evolution, where viral lineages from the same host were genetically more similar to each other than to lineages from other host species. When we compared viruses that had evolved in different host species, we found that parallel genetic changes were more likely to occur if the two host species were closely related. This suggests that when a virus adapts to one host it might also become better adapted to closely related host species. This may explain in part why host shifts tend to occur between related species, and may mean that when a new pathogen appears in a given species, closely related species may become vulnerable to the new disease.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>Host shifts, where a pathogen jumps from one host species to another, are a major source of infectious disease. Hosts shifts are more likely to occur between related host species and often rely on the pathogen evolving adaptations that increase their fitness in the novel host. Here we have investigated how viruses evolve in different host species, by experimentally evolving replicate lineages of an RNA virus in 19 different host species that shared a common ancestor 40 million years ago. We then deep sequenced the genomes of these viruses to examine the genetic changes that have occurred in different host species that vary in their relatedness. We found that parallel mutations–that are indicative of selection–were significantly more likely to occur within viral lineages from the same host, and between viruses evolved in closely related species. This suggests that a mutation that may adapt a virus to a given host, may also adapt it to closely related host species.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
BL and FMJ are supported by a Natural Environment Research Council grant (NE/L004232/1 <a href="http://www.nerc.ac.uk/">http://www.nerc.ac.uk/</a>) and by an European Research Council grant (281668, DrosophilaInfection, <a href="http://erc.europa.eu/">http://erc.europa.eu/</a>). JMA was supported by a grant from the Portuguese Ministério da Ciência, Tecnologia e Ensino Superior (SFRH/BD/72381/2010). BL is supported by a Sir Henry Dale Fellowship jointly funded by the Wellcome Trust and the Royal Society (Grant Number 109356/Z/15/Z). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="Sequence data (fastq files) are available in the NCBI SRA (Accession: SRP119720). BAM files, data and R scripts for analysis in the main text are available from the NERC data repository (https://doi.org/10.5285/4434a27d-5288-4f2e-88ac-4b1372e4d073)." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>Sequence data (fastq files) are available in the NCBI SRA (Accession: SRP119720). BAM files, data and R scripts for analysis in the main text are available from the NERC data repository (<a href="https://doi.org/10.5285/4434a27d-5288-4f2e-88ac-4b1372e4d073">https://doi.org/10.5285/4434a27d-5288-4f2e-88ac-4b1372e4d073</a>).</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Host shifts–where a pathogen jumps into and establishes in a new host species–are a major source of emerging infectious diseases. RNA viruses seem particularly prone to host shift [<a href="#ppat.1006951.ref001">1</a>–<a href="#ppat.1006951.ref004">4</a>], with HIV, Ebola virus and SARS coronavirus all having been acquired by humans from other host species [<a href="#ppat.1006951.ref005">5</a>–<a href="#ppat.1006951.ref007">7</a>]. Whilst some pathogens may be pre-adapted to a novel host, there are increasing numbers of examples demonstrating that adaptation to the new host occurs following a host shift [<a href="#ppat.1006951.ref008">8</a>, <a href="#ppat.1006951.ref009">9</a>]. These adaptations may allow a pathogen to enter host cells, increase replication rates, avoid or suppress the host immune response, or optimise virulence or transmission [<a href="#ppat.1006951.ref010">10</a>, <a href="#ppat.1006951.ref011">11</a>]. For example, in the 2013–2016 Ebola virus epidemic in West Africa, a mutation in the viral glycoprotein gene that arose early in the outbreak and rose to high frequency was found to increase infectivity in human cells and decrease infectivity in bats, which are thought to be the source of Ebola virus [<a href="#ppat.1006951.ref012">12</a>, <a href="#ppat.1006951.ref013">13</a>]. Likewise, a switch of a parvovirus from cats to dogs resulted in mutations in the virus capsid that allowed the virus to bind to cell receptors in dogs, but resulted in the virus losing its ability to infect cats [<a href="#ppat.1006951.ref014">14</a>, <a href="#ppat.1006951.ref015">15</a>].</p> <p>In some instances adaptation to a novel host relies on specific mutations that arise repeatedly whenever a pathogen switches to a given host. For example, in the jump of HIV-1 from chimps to humans, codon 30 of the <i>gag</i> gene has undergone a change that increases virus replication in humans, and this has occurred independently in all three HIV-1 lineages [<a href="#ppat.1006951.ref005">5</a>, <a href="#ppat.1006951.ref016">16</a>]. Similarly, five parallel mutations have been observed in the two independent epidemics of SARS coronavirus following its jump from palm civets into humans [<a href="#ppat.1006951.ref017">17</a>]. Similar patterns have been seen in experimental evolution studies, where parallel genetic changes occur repeatedly when replicate viral lineages adapt to a new host species in the lab. For example, when Vesicular Stomatitis Virus was passaged in human or dog cells, the virus evolved parallel mutations when evolved on the same cell type [<a href="#ppat.1006951.ref018">18</a>]. Likewise, a study passaging Tobacco Etch Potyvirus on four plant species found parallel mutations occurred only when the virus infected the same host species [<a href="#ppat.1006951.ref019">19</a>]. These parallel mutations provide compelling evidence that these genetic changes are adaptive, with the same mutations evolving independently in response to natural selection [<a href="#ppat.1006951.ref020">20</a>]. These studies have only used a limited number of hosts, and so do not provide information on how viral evolution occurs across a wide phylogenetic breadth of host species.</p> <p>The host phylogeny is important for determining a pathogens ability to infect a novel host, with pathogens tending to replicate most efficiently when they infect a novel host that is closely related to their original host [<a href="#ppat.1006951.ref002">2</a>, <a href="#ppat.1006951.ref021">21</a>–<a href="#ppat.1006951.ref034">34</a>]. Here, we asked whether viruses acquire the same genetic changes when evolving in the same and closely related host species. We experimentally evolved replicate lineages of an RNA virus called Drosophila C Virus (DCV; Discistroviridae) in 19 species of Drosophilidae that vary in their relatedness and shared a common ancestor approximately 40 million years ago [<a href="#ppat.1006951.ref035">35</a>, <a href="#ppat.1006951.ref036">36</a>]. We then sequenced the genomes of the evolved viral lineages and tested whether the same genetic changes arose when the virus was evolved in closely related host species.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="parallelgeneticchangesoccurindcvlineagesthathaveevolvedinthesamehostspecies" title="sec"> <div class="title" tagx="title" title="title">
Parallel genetic changes occur in DCV lineages that have evolved in the same host species</div> <p>To examine how viruses evolve in different host species we serially passaged DCV in 19 species of Drosophilidae. In total we infected 22,095 adult flies and generated 173 independent replicate lineages (6–10 per host species). We deep sequenced the evolved virus genomes to generate over 740,000 300bp sequence reads from each viral lineage. Out of 8989 sites, 584 contained a SNP with a derived allele frequency &amp;gt;0.05 in at least one viral lineage, and 84 of these were tri-allelic. None of these variants were found at an appreciable frequency in five sequencing libraries produced from the ancestral virus, indicating that they had spread though populations during the experiment (<a href="#ppat.1006951.g001">Fig 1</a>). In multiple cases these variants had nearly reached fixation (<a href="#ppat.1006951.g001">Fig 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1006951.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
The frequency of SNPs in viral lineages that have evolved in different host species.</div> <p>Each row represents an independent viral lineage. Viruses that evolved in different host species are separated by black horizontal lines. Each column represents a polymorphic site in the DCV genome, and only sites where the derived allele frequency &amp;gt;0.05 in at least two lineages are shown. The intensity of shading represents the derived allele frequency. Sites where there are three alleles have the two derived allele frequencies pooled for illustrative purposes. Sites with SNP frequencies that are significantly correlated among lineages from the same host species are shown by red stars at the bottom the column (permutation test; <i>p</i>&amp;lt;0.05). Open reading frames (ORFs) and viral proteins based on predicted polyprotein cleavage sites [<a href="#ppat.1006951.ref038">38</a>–<a href="#ppat.1006951.ref042">42</a>] are below the x axis. Information on the distribution of mutations across the genome and whether they are synonymous or non-synonymous can be found in the supplementary results. Sites with missing data are shown in white. The phylogeny was inferred under a relaxed molecular clock [<a href="#ppat.1006951.ref033">33</a>, <a href="#ppat.1006951.ref043">43</a>] and the scale axis represents the approximate age since divergence in millions of years (my) based on estimates from: [<a href="#ppat.1006951.ref035">35</a>, <a href="#ppat.1006951.ref036">36</a>].</p>   </div> <p>We next examined whether the same genetic changes occur in parallel when different populations encounter the same host species. Of the 584 SNPs, 102 had derived allele frequencies &amp;gt;0.05 in at least two viral lineages, and some had risen to high frequencies in multiple lineages (<a href="#ppat.1006951.g001">Fig 1</a>). We estimated the genetic differentiation between viral lineages by calculating <i>F</i><sub>ST</sub>. We found that viral lineages that had evolved within the same host were genetically more similar to each other than to lineages from other host species (<a href="#ppat.1006951.g002">Fig 2</a>; <i>P</i>&amp;lt;0.001). Furthermore, we found no evidence of differences in substitution biases in the different host species (Fisher Exact Test: <i>p</i> = 0.14; see <a href="#sec006">methods</a>), suggesting that this pattern is not driven by changes in the types of mutations in different host species.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1006951.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Viral lineages from the same host species were genetically more similar to each other than to lineages from different host species.</div> <p>The mean pairwise <i>F</i><sub><i>ST</i></sub> between all possible pairs of viral lineages from the same host species was calculated. The red line shows the observed value. The grey bars are the null distribution of this statistic obtained by permuting the viral lineages across host species 1000 times.</p>   </div> <p>To examine the genetic basis of parallel evolution, we individually tested whether each SNP in the DCV genome showed a signature of parallel evolution among viral lineages passaged in the same host species (i.e. we repeated the analysis in <a href="#ppat.1006951.g002">Fig 2</a> for each SNP). We identified 56 polymorphic sites with a significant signal of parallel evolution within the same host species (<i>P</i>&amp;lt;0.05; significantly parallel sites are shown with a red asterisk in <a href="#ppat.1006951.g001">Fig 1</a>; the false discovery rate is estimated to be 17% [<a href="#ppat.1006951.ref037">37</a>]).</p> </div> <div class="virusesincloselyrelatedhostsaregeneticallymoresimilar" title="sec"> <div class="title" tagx="title" title="title">
Viruses in closely related hosts are genetically more similar</div> <p>We investigated if viruses passaged through closely related hosts showed evidence of parallel genetic changes. We calculated <i>F</i><sub>ST</sub> between all possible pairs of viral lineages that had evolved in different host species. We found that viral lineages from closely related hosts were more similar to each other than viral lineages from more distantly related hosts (<a href="#ppat.1006951.g003">Fig 3A</a>). This is reflected in a significant positive relationship between virus <i>F</i><sub>ST</sub> and host genetic distance (<a href="#ppat.1006951.g003">Fig 3B</a>, Permutation test: <i>r</i> = 0.15, <i>P</i> = 0.002). We lacked the statistical power to identify the specific SNPs that are causing the signature of parallel evolution in <a href="#ppat.1006951.g003">Fig 3</a> (false discovery rate &amp;gt;0.49 for all SNPs).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1006951.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Viral lineages from more closely related host species are genetically more similar.</div> <p>(A) The correlation between the genetic differentiation of viral lineages and the genetic distance between the species they have evolved in. Linear regression line is shown in red. Genetic distances were scaled so that the distance from the root to the tip of the tree was one. (B) Pearson’s correlation coefficient (<i>r</i>) of <i>F</i><sub><i>ST</i></sub> between pairs of viral lineage and the genetic distance between the host species they evolved in. The observed value is in red and the grey bars are the null distribution obtained by permutation.</p>   </div> <p>Two of the most striking examples of parallel evolution in related species are in <i>Scaptodrosophila pattersoni</i> and <i>S</i>. <i>lebanonensis</i>, which show two high frequency parallel mutations. These are a synonymous mutation in the 2C replicase protein at position 1901 and a triallelic non-synonymous mutation in a viral capsid protein at position 8072. However, the wider pattern of parallel evolution is not driven by these two examples, as the results remained significant after viruses that had evolved in these two species were removed from the dataset (within species parallelism: <i>P</i>&amp;lt;0.001; between species parallelism: <i>P</i> = 0.013).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>When a pathogen infects a novel host species, it finds itself in a new environment to which it must adapt [<a href="#ppat.1006951.ref004">4</a>, <a href="#ppat.1006951.ref008">8</a>, <a href="#ppat.1006951.ref010">10</a>, <a href="#ppat.1006951.ref044">44</a>]. When DCV was passaged through different species of Drosophilidae, we found the same genetic changes arose repeatedly in replicate viral lineages in the same host species. Such repeatable parallel genetic changes to the same host environment are compelling evidence that these changes are adaptive [<a href="#ppat.1006951.ref020">20</a>]. We then examined whether these same genetic changes might occur in closely related host species, as these are likely to present a similar environment for the virus. We found that viruses evolved in closely related hosts were more similar to each other than viruses that evolved in more distantly related species. Therefore, mutations that evolve in one host species frequently arise when the virus infects closely related hosts. This finding of parallel genetic changes in closely related host species suggests that when a virus adapts to one host it might also become better adapted to closely related host species.</p> <p>Phylogenetic patterns of host adaptation may in part explain why pathogens tend to be more likely to jump between closely related host species. This pattern is seen in nature, where host shifts tend to occur most frequently between closely related hosts, and in laboratory cross-infection studies, where viruses tend to replicate more rapidly when the new host is related to the pathogens natural host [<a href="#ppat.1006951.ref002">2</a>, <a href="#ppat.1006951.ref021">21</a>–<a href="#ppat.1006951.ref034">34</a>]. For example, in a large cross-infection experiment involving Drosophila sigma viruses (Rhabdoviridae) isolated from different species of <i>Drosophila</i>, the viruses tended to replicate most efficiently in species closely related to their natural hosts [<a href="#ppat.1006951.ref034">34</a>]. This suggests that these viruses had acquired adaptations to their host species that benefitted them when they infected closely related species. Our results demonstrate that this pattern is apparent at the level of specific nucleotides, and can arise very shortly after a host shift.</p> <p>While the susceptibility of a novel host is correlated to its relatedness to the pathogens’ original host, it is also common to find exceptions to this pattern. This is seen both in nature when pathogens shift between very distant hosts [<a href="#ppat.1006951.ref045">45</a>, <a href="#ppat.1006951.ref046">46</a>], and in laboratory cross-infection experiments [<a href="#ppat.1006951.ref033">33</a>, <a href="#ppat.1006951.ref034">34</a>]. This pattern is also seen in our data where we also observe parallel genetic changes occurring between more distantly related hosts. For example, a mutation at position 8072 was not only near fixation in most of the lineages infecting two closely related species, but also occurred at a high frequency in replicate lineages in a phylogenetically distant host (<a href="#ppat.1006951.g001">Fig 1</a>).</p> <p>The function of these mutations is unknown, but in other systems adaptations after host shifts have been found to enhance the ability of the virus to bind to host receptors [<a href="#ppat.1006951.ref011">11</a>], increase replication rates [<a href="#ppat.1006951.ref016">16</a>] or avoid the host immune response [<a href="#ppat.1006951.ref008">8</a>, <a href="#ppat.1006951.ref010">10</a>, <a href="#ppat.1006951.ref047">47</a>]. Of the high frequency significant SNPs (shown in <a href="#ppat.1006951.g001">Fig 1</a> with red asterisk) nine occur in the non-structural proteins (ORF1: RNA dependant RNA polymerase, the putative protease and helicase proteins), and eleven occur in the capsid proteins (ORF2). Interestingly, none were in DCV-1A, which suppresses the host antiviral RNAi defences [<a href="#ppat.1006951.ref048">48</a>]. It will be of interest to examine the functions of the parallel mutations we detected, and characterise phenotypically how they affect viral infectivity and replication.</p> <p>One mutation rose to a high frequency across all the host species (<a href="#ppat.1006951.g001">Fig 1</a>, position 214 in the 3’ un-translated region). This is unlikely to be an error in the genome sequencing, as it did not occur when we sequenced the ancestral virus. This may have been due to natural selection favouring this change in all species, perhaps because there was a strongly deleterious mutation at this site in the virus we cloned or due to the virus going from cell culture to being passaged <i>in vivo</i>.</p> <p>Previous studies have elegantly demonstrated parallel evolution following host shifts (eg [<a href="#ppat.1006951.ref018">18</a>]). However, these are often in cell culture, and so do not reflect the heterogeneity of tissue and cell types in whole animals that occur in studies <i>in vivo</i> (although see [<a href="#ppat.1006951.ref049">49</a>] that suggests otherwise). The complex nature of different tissue types <i>in vivo</i> coupled with a limited number of generations may explain why some parallel SNPs have remained at a low frequency in this study. Following a host shift, viruses must sometimes acquire specific mutations that allow them to be transmitted in their new host [<a href="#ppat.1006951.ref009">9</a>, <a href="#ppat.1006951.ref010">10</a>]. As we artificially inoculated the virus, this aspect of adaptation to a new host is missing from our study.</p> <p>In conclusion, we have found that host relatedness can be important in determining how viruses evolve when they find themselves in a new host. This study suggests that while some genetic changes will be found only in specific hosts, we frequently see the same changes occurring in closely related host species. These phylogenetic patterns suggest that mutations that adapt a virus to one host may also adapt it to closely related host species. Therefore, there may be a knock-on effect, where a host shift leaves closely related species vulnerable to the new disease.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="virusproduction" title="sec"> <div class="title" tagx="title" title="title">
Virus production</div> <p>DCV is a positive sense RNA virus in the family Discistroviridae that was isolated from <i>D</i>. <i>melanogaster</i>, which it naturally infects in the wild [<a href="#ppat.1006951.ref050">50</a>, <a href="#ppat.1006951.ref051">51</a>]. To minimise the amount of genetic variation in the DCV isolate we used to initiate the experimental evolution study, we aimed to isolate single infectious clones of DCV using a serial dilution procedure. DCV was produced in Schneider’s Drosophila line 2 (DL2) cells [<a href="#ppat.1006951.ref052">52</a>] as described in [<a href="#ppat.1006951.ref053">53</a>]. Cells were cultured at 25°C in Schneider’s Drosophila Medium with 10% Fetal Bovine Serum, 100 U/ml penicillin and 100 μg/ml streptomycin (all Invitrogen, UK). The DCV strain used was isolated from <i>D</i>. <i>melanogaster</i> collected in Charolles, France [<a href="#ppat.1006951.ref054">54</a>]. DL2 cells were seeded into two 96-well tissue culture plates at approximately 10<sup>4</sup> cells in 100 μl of media per well. Cells were allowed to adhere to the plates by incubating at 25°C for five hours or over-night. Serial 1:1 dilutions of DCV were made in complete Schneider’s media, giving a range of final dilutions from 1:10<sup>8</sup>–1:4x10<sup>14</sup>. 100 μl of these dilutions were then added to the cells and incubated for 7 days, 8 replicates were made for each DCV dilution. Each well was then examined for DCV infection of the DL2 cells, and a well was scored as positive for DCV infection if clear cytopathic effects were present in the majority of the cells. The media was taken from the wells with the greatest dilution factor that were scored as infected with DCV and stored at -80°C. This processes was then repeated using the DCV samples from the first dilution series. One clone, B6A, was selected for amplification and grown in cell culture as described above. Media containing DCV was removed and centrifuged at 3000 x g for 5 minutes at 4°C to pellet any remaining cell debris, before being aliquoted and stored at -80°C. The Tissue Culture Infective Dose 50 (TCID<sub>50</sub>) of the DCV was 6.32 x 10<sup>9</sup> infectious particles per ml using the Reed-Muench end-point method [<a href="#ppat.1006951.ref055">55</a>].</p> </div> <div class="inoculatingflyspecies" title="sec"> <div class="title" tagx="title" title="title">
Inoculating fly species</div> <p>We passaged the virus through 19 species of Drosophilidae, with 6–10 independent replicate passages for each species. We selected species from across the phylogeny (that shared a common ancestor approximately 40 million years ago [<a href="#ppat.1006951.ref035">35</a>, <a href="#ppat.1006951.ref036">36</a>]), but included clades of closely related species that recently shared common ancestors less than 5 million years ago (<a href="#ppat.1006951.g001">Fig 1</a>). All fly stocks were reared at 22°C. Stocks of each fly species were kept in 250ml bottles at staggered ages. Flies were collected and sexed, and males were placed on cornmeal medium for 4 days before inoculation. Details of the fly stocks used can be found in the supplementary materials.</p> <p>4–11 day old males were infected with DCV using a 0.0125 mm diameter stainless steel needle (26002–10, Fine Science Tools, CA, USA) dipped in DCV solution. For the first passage this was the cloned DCV isolate in cell culture supernatant (described above), and then subsequently was the virus extracted from the previous passage (described below). The needle was pricked into the pleural suture on the thorax of flies, towards the midcoxa. Each replicate was infected using a new needle and strict general cleaning procedures were used to minimise any risk of cross-contamination between replicates. Species were collected and inoculated in a randomised order each passage. Flies were then placed into vials of cornmeal medium and kept at 22°C and 70% relative humidity. Flies were snap frozen in liquid nitrogen 3 days post-infection, homogenised in Ringer’s solution (2.5μl per fly) and then centrifuged at 12,000g for 10 mins at 4°C. The resulting supernatant was removed and frozen at -80°C to be used for infecting flies in the subsequent passage. The remaining homogenate was preserved in Trizol reagent (Invitrogen) and stored at -80°C for RNA extraction. The 3 day viral incubation period was chosen based on time course and pilot data showing that viral load reaches a maximum at approximately 3 days post-infection. This process was repeated for 10 passages for all species, except <i>D</i>. <i>montana</i> where only 8 passages were carried out due to the fly stocks failing to reproduce. Each lineage was injected into a mean of 11 flies at each passage (range 4–18). Experimental evolution studies in different tissue types have seen clear signals of adaptation in 100 virus generations [<a href="#ppat.1006951.ref018">18</a>]. Based on log<sub>2</sub> change in RNA viral load we estimate that we have passaged DCV for approximately 100–200 generations.</p> </div> <div class="sequencing" title="sec"> <div class="title" tagx="title" title="title">
Sequencing</div> <p>After passaging the virus, we sequenced evolved viral lineages from 19 host species, with a mean of 9 independent replicate lineages of the virus per species (range 6–10 replicates). cDNA was synthesised using Invitrogen Superscript III reverse-transcriptase with random hexamer primers (25°C 5mins, 50°C 50mins, 70°C 15mins). The genome of the evolved viruses, along with the initial DCV ancestor (x5) were then amplified using Q5 high fidelity polymerase (NEB) in nine overlapping PCR reactions (see <a href="#ppat.1006951.s001">supplementary Table S2</a> for PCR primers and cycle conditions). Primers covered position 62-9050bp (8989bp) of the Genbank refseq (NC_001834.1) giving 97% coverage of the genome. PCRs of individual genomes were pooled and purified with Ampure XP beads (Agencourt). Individual Nextera XT libraries (Illumina) were prepared for each viral lineage. In total we sequenced 173 DCV pooled amplicon libraries on an Illumina MiSeq (Cambridge Genomic Service) v3 for 600 cycles to give 300bp paired-end reads.</p> </div> <div class="bioinformaticsandvariantcalling" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatics and variant calling</div> <p>FastQC, version 0.11.2 [<a href="#ppat.1006951.ref056">56</a>] was used to assess read quality and primer contamination. Trimmomatic, version 0.32 [<a href="#ppat.1006951.ref057">57</a>] was used to removed low quality bases and adaptor sequences, using the following options: MINLEN = 30 (Drop the read if it is below 30 base pairs), TRAILING = 15 (cut bases of the end of the read if below a threshold quality of 15), SLIDINGWINDOW = 4:20 (perform a sliding window trimming, cutting once the average quality within a 4bp window falls below a threshold of 20), and ILLUMINACLIP = TruSeq3-PE.fa:2:20:10:1:true (remove adapter contamination; the values correspond in order to: input fasta file with adapter sequences to be matched, seed mismatches, palindrome clip threshold, simple clip threshold, minimum adapter length and logical value to keep both reads in case of read-through being detected in paired reads by palindrome mode).</p> <p>To generate a reference ancestral Drosophila C Virus sequence we amplified the ancestral starting virus by PCR as above. PCR products were treated with exonuclease 1 and Antarctic phosphatase to remove unused PCR primers and dNTPs and then sequenced directly using BigDye reagents (ABI) on an ABI 3730 capillary sequencer in both directions (Source Bioscience, Cambridge, UK). Sequences were edited in Sequencher (version 4.8; Gene Codes), and were manually checked for errors. Fastq reads were independently aligned to this reference sequence (Genbank accession: MG570143) using BWA-MEM, version 0.7.10 {Li, 2009 #1605} with default options with exception of the parameter–M, which marks shorter split hits as secondary. 99.5% of reads had mapping phred quality scores of &amp;gt;60. The generated SAM files were converted to their binary format (BAM) and sorted by their leftmost coordinates with SAMtools, version 0.1.19 (website: <a href="http://samtools.sourceforge.net/">http://samtools.sourceforge.net/</a>) [<a href="#ppat.1006951.ref058">58</a>]. Read Group information (RG) was added to the BAM files using the module AddOrReplaceReadGroups from Picard Tools, version 1.126 (<a href="https://broadinstitute.github.io/picard">https://broadinstitute.github.io/picard</a>).</p> <p>The variant calling was then performed for each individual BAM using UnifiedGenotyper tool from GATK, version 3.3.0. As we were interested in calling low frequency variants in our viruses, we assumed a ploidy level of 100 (-sample_ploidy:100). The other parameters were set to their defaults except—stand_call_conf:30 (minimum phred-scaled confidence threshold at which variants should be called) and—downsample_to_coverage:1000 (down-sample each sample to 1000X coverage)</p> </div> <div class="hostphylogeny" title="sec"> <div class="title" tagx="title" title="title">
Host phylogeny</div> <p>We used a trimmed version of a phylogeny produced previously [<a href="#ppat.1006951.ref033">33</a>]. This time-based tree (where the distance from the root to the tip is equal for all taxa) was inferred using seven genes with a relaxed molecular clock model in BEAST (v1.8.0) [<a href="#ppat.1006951.ref043">43</a>, <a href="#ppat.1006951.ref059">59</a>]. The tree was pruned to the 19 species used using the Ape package in R [<a href="#ppat.1006951.ref060">60</a>, <a href="#ppat.1006951.ref061">61</a>].</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>We examined the frequency of alternate alleles (single nucleotide polymorphisms: SNPs) in five ancestral virus replicates (aliquots of the same virus stock that was used to found the evolved lineages). SNPs in these ancestral viruses may represent pre-standing genetic variation, or may be sequencing errors. We found the mean SNP frequency was 0.000923 and the highest frequency of any SNP was 0.043 across the ancestral viruses. We therefore included a SNP in our analyses if its frequency was &amp;gt;0.05 in any of the evolved viral lineages. For all analyses we included all three alleles at triallelic sites.</p> </div> <div class="parallelevolutionwithinspecies" title="sec"> <div class="title" tagx="title" title="title">
Parallel evolution within species</div> <p>As a measure of genetic differentiation we estimated <i>F</i><sub><i>ST</i></sub> between all the virus lineages based on the heterozygosity (<i>H</i>) of the SNPs we called [<a href="#ppat.1006951.ref062">62</a>]: </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1006951.e001g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
T</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mi_UNKNOWN">
b</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mi_UNKNOWN">
w</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
H</div> <div class="mi_UNKNOWN">
b</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(Eq 1)</span></div> where <i>H</i><sub><i>b</i></sub> is the mean number of differences between pairs of sequence reads sampled from the two different lineages. <i>H</i><sub><i>w</i></sub> is mean number of differences between sequence reads sampled from within each lineage. <i>H</i><sub><i>b</i></sub> and <i>H</i><sub><i>w</i></sub> were calculated separately for each polymorphic site, and the mean across sites used in Eq (<a href="#ppat.1006951.e001">1</a>). <i>H</i><sub><i>w</i></sub> was calculated separately for the two lineages being compared, and the unweighted mean used in Eq (<a href="#ppat.1006951.e001">1</a>).<p /> <p>To examine whether there had been parallel evolution among viral lineages that had evolved within the same fly species, we calculated the mean <i>F</i><sub><i>ST</i></sub> between lineages that had evolved in the same fly species, and compared this to the mean <i>F</i><sub><i>ST</i></sub> between lineages that had evolved in different fly species. We tested whether this difference was statistically significant using a permutation test. The fly species labels were randomly reassigned to the viral lineages, and we calculated the mean <i>F</i><sub><i>ST</i></sub> between lineages that had evolved in the same fly species. This was repeated 1000 times to generate a null distribution of the test statistic, and this was then compared to the observed value.</p> <p>To identify individual SNPs with a signature of parallel evolution within species, we repeated this procedure separately for each SNP.</p> </div> <div class="parallelevolutionbetweenspecies" title="sec"> <div class="title" tagx="title" title="title">
Parallel evolution between species</div> <p>We next examined whether viral lineages that had evolved in different fly species tended to be more similar if the fly species were more closely related. Considering all pairs of viral lineages from different host species, we correlated pairwise <i>F</i><sub><i>ST</i></sub> with the genetic distance between the fly species. To test the significance of this correlation, we permuted the fly species over the <i>Drosophila</i> phylogeny and recalculated the Pearson correlation coefficient. This was repeated 1000 times to generate a null distribution of the test statistic, and this was then compared to the observed value. To identify individual SNPs whose frequencies were correlated with the genetic distance between hosts we repeated this procedure separately for each SNP.</p> <p>We confirmed there was no relationship between rates of molecular evolution (SNP frequency) and either genetic distance from the host DCV was isolated from (<i>D</i>. <i>melanogaster</i>) or estimated viral population size (see <a href="#ppat.1006951.s001">supplementary S1 and S2 Figs</a>) using generalised linear mixed models that include the phylogeny as a random effect in the MCMCglmm package in R [<a href="#ppat.1006951.ref063">63</a>] as described previously [<a href="#ppat.1006951.ref034">34</a>]. We also examined the distribution of SNPs and whether they were synonymous or non-synonymous (see <a href="#ppat.1006951.s001">supplementary results</a>).</p> <p>To test whether there were systematic differences in the types of mutations occurring in the different host species, we classified all the SNPs into the six possible types (A/G, A/T, A/C, G/T, G/C and C/T). We then counted the number of times each type of SNP arose in each host species at a frequency above 5% and in at least one biological replicate (SNPs in multiple biological replicates were only counted once). This resulted in a contingency table with 6 columns and 19 rows. We tested for differences between the species in the relative frequency of the 6 SNP types by simulation [<a href="#ppat.1006951.ref064">64</a>].</p> </div> <div class="accessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Accession numbers</div> <p>Sequence data (fastq files) are available in the NCBI SRA (Accession: SRP119720). BAM files, data and R scripts for analysis in the main text are available from the NERC data repository (<a href="https://doi.org/10.5285/4434a27d-5288-4f2e-88ac-4b1372e4d073">https://doi.org/10.5285/4434a27d-5288-4f2e-88ac-4b1372e4d073</a>).</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Supporting information</span> <div class="title" tagx="title" title="title">
Supplementary methods, results, tables and figures.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1006951.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>Thanks to the Drosophila species stock centre for providing fly stocks and four anonymous reviewers for constructive comments.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ppat.1006951.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleaveland</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laurenson</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Diseases of humans and their domestic mammals: pathogen characteristics, host range and the risk of emergence</span>. <span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society of London Series B-Biological Sciences</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">356</span>(<span class="issue" tagx="issue" title="issue">1411</span>):<span class="fpage" tagx="fpage" title="fpage">991</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. PubMed PMID: WOS:000170315900003.</span></li> <li tag="ref"><a name="ppat.1006951.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davies</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pedersen</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogeny and geography predict pathogen community similarity in wild primates and humans</span>. <span class="source" tagx="source" title="source">Proceedings of the Royal Society B-Biological Sciences</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">275</span>(<span class="issue" tagx="issue" title="issue">1643</span>):<span class="fpage" tagx="fpage" title="fpage">1695</span>–<span class="lpage" tagx="lpage" title="lpage">701</span>. <span class="comment">doi: <a href="https://doi.org/10.1098/rspb.2008.0284">10.1098/rspb.2008.0284</a></span> PubMed PMID: ISI:000256387500014. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18445561">18445561</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Latham</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>. <span class="mixed-article-title" title="mixed-article-title">Risk factors for human disease emergence</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">356</span>(<span class="issue" tagx="issue" title="issue">1411</span>):<span class="fpage" tagx="fpage" title="fpage">983</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. Epub 2001/08/23. <span class="comment">doi: <a href="https://doi.org/10.1098/rstb.2001.0888">10.1098/rstb.2001.0888</a></span> ; PubMed Central PMCID: PMC1088493.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11516376">11516376</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antia</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">Emerging pathogens: the epidemiology and evolution of species jumps</span>. <span class="source" tagx="source" title="source">Trends Ecol Evol</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">238</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>. Epub 2006/05/17. doi: S0169-5347(05)00038-8 [pii] <span class="comment">doi: <a href="https://doi.org/10.1016/j.tree.2005.02.009">10.1016/j.tree.2005.02.009</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701375">16701375</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">BH</span></span>. <span class="mixed-article-title" title="mixed-article-title">The evolution of HIV-1 and the origin of AIDS</span>. <span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society B-Biological Sciences</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">365</span>(<span class="issue" tagx="issue" title="issue">1552</span>):<span class="fpage" tagx="fpage" title="fpage">2487</span>–<span class="lpage" tagx="lpage" title="lpage">94</span>. <span class="comment">doi: <a href="https://doi.org/10.1098/rstb.2010.0031">10.1098/rstb.2010.0031</a></span> PubMed PMID: WOS:000280097000008. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20643738">20643738</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leroy</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumulungui</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pourrut</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rouquet</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hassanin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yaba</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Fruit bats as reservoirs of Ebola virus</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">438</span>(<span class="issue" tagx="issue" title="issue">7068</span>):<span class="fpage" tagx="fpage" title="fpage">575</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/438575a">10.1038/438575a</a></span> PubMed PMID: WOS:000233593100030. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16319873">16319873</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">ZL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ren</span><span class="given-names" tagx="given-names" title="given-names">WZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Epstein</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">310</span>(<span class="issue" tagx="issue" title="issue">5748</span>):<span class="fpage" tagx="fpage" title="fpage">676</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="comment">doi: <a href="https://doi.org/10.1126/science.1118391">10.1126/science.1118391</a></span> PubMed PMID: WOS:000232997700042. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16195424">16195424</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parrish</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morens</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calisher</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Cross-species virus transmission and the emergence of new epidemic diseases</span>. <span class="source" tagx="source" title="source">Microbiology and Molecular Biology Reviews</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">72</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">457</span>–<span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="comment">doi: <a href="https://doi.org/10.1128/MMBR.00004-08">10.1128/MMBR.00004-08</a></span> PubMed PMID: WOS:000258951200004. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18772285">18772285</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fonville</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The potential for respiratory droplet-transmissible A/H5N1 influenza virus to evolve in a mammalian host</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">336</span>(<span class="issue" tagx="issue" title="issue">6088</span>):<span class="fpage" tagx="fpage" title="fpage">1541</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. Epub 2012/06/23. <span class="comment">doi: <a href="https://doi.org/10.1126/science.1222526">10.1126/science.1222526</a></span> ; PubMed Central PMCID: PMC3426314.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22723414">22723414</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brockhurst</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>. <span class="mixed-article-title" title="mixed-article-title">The Evolution and Genetics of Virus Host Shifts</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">e1004395</span> Epub 2014/11/07. <span class="comment">doi: <a href="https://doi.org/10.1371/journal.ppat.1004395">10.1371/journal.ppat.1004395</a></span> ; PubMed Central PMCID: PMC4223060.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25375777">25375777</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parrish</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">The origins of new pandemic viruses: the acquisition of new host ranges by canine parvovirus and influenza A viruses</span>. <span class="source" tagx="source" title="source">Annual review of microbiology</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">59</span>:<span class="fpage" tagx="fpage" title="fpage">553</span>–<span class="lpage" tagx="lpage" title="lpage">86</span>. Epub 2005/09/13. <span class="comment">doi: <a href="https://doi.org/10.1146/annurev.micro.59.030804.121059">10.1146/annurev.micro.59.030804.121059</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16153179">16153179</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diehl</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grubaugh</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvalho</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kyawe</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Ebola Virus Glycoprotein with Increased Infectivity Dominated the 2013–2016</span><span class="source" tagx="source" title="source">Epidemic. Cell</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">167</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">1088</span>–<span class="lpage" tagx="lpage" title="lpage">98 e6</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.cell.2016.10.014">10.1016/j.cell.2016.10.014</a></span> ; PubMed Central PMCID: PMCPMC5115602.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27814506">27814506</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urbanowicz</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McClure</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakuntabhai</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sall</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobinger</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Human Adaptation of Ebola Virus during the West African Outbreak</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">167</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">1079</span>–<span class="lpage" tagx="lpage" title="lpage">87 e5</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.cell.2016.10.013">10.1016/j.cell.2016.10.013</a></span> ; PubMed Central PMCID: PMCPMC5101188.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27814505">27814505</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shackelton</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parrish</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Truyen</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>. <span class="mixed-article-title" title="mixed-article-title">High rate of viral evolution associated with the emergence of carnivore parvovirus</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">102</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">379</span>–<span class="lpage" tagx="lpage" title="lpage">84</span>. Epub 2005/01/01. <span class="comment">doi: <a href="https://doi.org/10.1073/pnas.0406765102">10.1073/pnas.0406765102</a></span> ; PubMed Central PMCID: PMC544290.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15626758">15626758</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Truyen</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evermann</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vieler</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parrish</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Evolution of canine parvovirus involved loss and gain of feline host range</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">1996</span>;<span class="volume" tagx="volume" title="volume">215</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">186</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. Epub 1996/01/15. <span class="comment">doi: <a href="https://doi.org/10.1006/viro.1996.0021">10.1006/viro.1996.0021</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8560765">8560765</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wain</span><span class="given-names" tagx="given-names" title="given-names">LV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailes</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bibollet-Ruche</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decker</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keele</span><span class="given-names" tagx="given-names" title="given-names">BF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Heuverswyn</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Adaptation of HIV-1 to its human host</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">1853</span>–<span class="lpage" tagx="lpage" title="lpage">60</span>. Epub 2007/06/05. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/msm110">10.1093/molbev/msm110</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17545188">17545188</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fontanet</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology of SARS-associated coronavirus, Beijing</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1420</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. Epub 2005/10/19. <span class="comment">doi: <a href="https://doi.org/10.3201/eid1109.040773">10.3201/eid1109.040773</a></span> ; PubMed Central PMCID: PMC3310602.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16229772">16229772</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Remold</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">PE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Evolutionary genomics of host adaptation in vesicular stomatitis virus</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1138</span>–<span class="lpage" tagx="lpage" title="lpage">47</span>. Epub 2008/03/21. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/msn059">10.1093/molbev/msn059</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18353798">18353798</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedhomme</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lafforgue</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elena</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span>. <span class="mixed-article-title" title="mixed-article-title">Multihost experimental evolution of a plant RNA virus reveals local adaptation and host-specific mutations</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">1481</span>–<span class="lpage" tagx="lpage" title="lpage">92</span>. Epub 2012/02/10. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/msr314">10.1093/molbev/msr314</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22319146">22319146</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bollback</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>. <span class="mixed-article-title" title="mixed-article-title">Parallel Genetic Evolution Within and Between Bacteriophage Species of Varying Degrees of Divergence</span>. <span class="source" tagx="source" title="source">Genetics</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">181</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">225</span>–<span class="lpage" tagx="lpage" title="lpage">34</span>. <span class="comment">doi: <a href="https://doi.org/10.1534/genetics.107.085225">10.1534/genetics.107.085225</a></span> PubMed PMID: WOS:000262595500021. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19001294">19001294</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streicker</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turmelle</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vonhof</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuzmin</span><span class="given-names" tagx="given-names" title="given-names">IV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCracken</span><span class="given-names" tagx="given-names" title="given-names">GF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rupprecht</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Host Phylogeny Constrains Cross-Species Emergence and Establishment of Rabies Virus in Bats</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">329</span>(<span class="issue" tagx="issue" title="issue">5992</span>):<span class="fpage" tagx="fpage" title="fpage">676</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="comment">doi: <a href="https://doi.org/10.1126/science.1188836">10.1126/science.1188836</a></span> PubMed PMID: WOS:000280602700037. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20689015">20689015</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streicker</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">Simultaneously reconstructing viral cross-species transmission history and identifying the underlying constraints</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">368</span>(<span class="issue" tagx="issue" title="issue">1614</span>):<span class="fpage" tagx="fpage" title="fpage">20120196</span> Epub 2013/02/06. <span class="comment">doi: <a href="https://doi.org/10.1098/rstb.2012.0196">10.1098/rstb.2012.0196</a></span> ; PubMed Central PMCID: PMC3678322.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382420">23382420</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Griffin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Omotayo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nunn</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fryxell</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogenetic host specificity and understanding parasite sharing in primates</span>. <span class="source" tagx="source" title="source">Ecol Lett</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1370</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. Epub 2012/08/24. <span class="comment">doi: <a href="https://doi.org/10.1111/j.1461-0248.2012.01858.x">10.1111/j.1461-0248.2012.01858.x</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22913776">22913776</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waxman</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinert</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Inferring host range dynamics from comparative data: the protozoan parasites of new world monkeys</span>. <span class="source" tagx="source" title="source">Am Nat</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">184</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">65</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>. Epub 2014/06/13. <span class="comment">doi: <a href="https://doi.org/10.1086/676589">10.1086/676589</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24921601">24921601</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bininda-Emonds</span><span class="given-names" tagx="given-names" title="given-names">ORP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gittleman</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altizer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogenetically related and ecologically similar carnivores harbor similar parasite assemblages</span>. <span class="source" tagx="source" title="source">Journal of Animal Ecology</span>. <span class="year" tagx="year" title="year">2013</span>:n/a-n/a. <span class="comment">doi: <a href="https://doi.org/10.1111/1365-2656.12160">10.1111/1365-2656.12160</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24289314">24289314</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krasnov</span><span class="given-names" tagx="given-names" title="given-names">BR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poulin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakagawa</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">A Tale of Two Phylogenies: Comparative Analyses of Ecological Interactions</span>. <span class="source" tagx="source" title="source">The American Naturalist</span>. <span class="year" tagx="year" title="year">2014</span>;0(0):000. <span class="comment">doi: <a href="https://doi.org/10.1086/674445">10.1086/674445</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24464193">24464193</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramsden</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charleston</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Hantavirus evolution in relation to its rodent and insectivore hosts: no evidence for codivergence</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">143</span>–<span class="lpage" tagx="lpage" title="lpage">53</span>. Epub 2008/10/17. doi: msn234 [pii] <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/msn234">10.1093/molbev/msn234</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18922760">18922760</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vienne</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hood</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giraud</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogenetic determinants of potential host shifts in fungal pathogens</span>. <span class="source" tagx="source" title="source">Journal of Evolutionary Biology</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">22</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">2532</span>–<span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="comment">doi: <a href="https://doi.org/10.1111/j.1420-9101.2009.01878.x">10.1111/j.1420-9101.2009.01878.x</a></span> PubMed PMID: WOS:000271785800019. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19878406">19878406</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">GS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webb</span><span class="given-names" tagx="given-names" title="given-names">CO</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogenetic signal in plant pathogen-host range</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">104</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">4979</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>. <span class="comment">doi: <a href="https://doi.org/10.1073/pnas.0607968104">10.1073/pnas.0607968104</a></span> PubMed PMID: WOS:000245256700040. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17360396">17360396</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tinsley</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Majerus</span><span class="given-names" tagx="given-names" title="given-names">MEN</span></span>. <span class="mixed-article-title" title="mixed-article-title">Small steps or giant leaps for male-killers? Phylogenetic constraints to male-killer host shifts</span>. <span class="source" tagx="source" title="source">Bmc Evolutionary Biology</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">7</span><span class="comment">doi: <a href="https://doi.org/10.1186/1471-2148-7-238">10.1186/1471-2148-7-238</a></span> PubMed PMID: WOS:000252786000001. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18047670">18047670</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman-Huertas</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moreau</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stahlhut</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Werren</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Specialization and geographic isolation among Wolbachia symbionts from ants and lycaenid butterflies</span>. <span class="source" tagx="source" title="source">Evolution</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">63</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">624</span>–<span class="lpage" tagx="lpage" title="lpage">40</span>. Epub 2008/12/05. <span class="comment">doi: <a href="https://doi.org/10.1111/j.1558-5646.2008.00579.x">10.1111/j.1558-5646.2008.00579.x</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19054050">19054050</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaenike</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Infection success in novel hosts: An experimental and phylogenetic study of Drosophila-parasitic nematodes</span>. <span class="source" tagx="source" title="source">Evolution</span>. <span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">57</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">544</span>–<span class="lpage" tagx="lpage" title="lpage">57</span>. PubMed PMID: WOS:000182193800010. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12703944">12703944</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Day</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGonigle</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cogni</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Causes and Consequences of Changes in Virulence following Pathogen Host Shifts</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">e1004728</span> Epub 2015/03/17. <span class="comment">doi: <a href="https://doi.org/10.1371/journal.ppat.1004728">10.1371/journal.ppat.1004728</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25774803">25774803</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Host phylogeny determines viral persistence and replication in novel hosts</span>. <span class="source" tagx="source" title="source">PLoS Pathogens</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">7</span>((<span class="issue" tagx="issue" title="issue">9</span>)):<span class="fpage" tagx="fpage" title="fpage">e1002260</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.ppat.1002260">10.1371/journal.ppat.1002260</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21966271">21966271</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obbard</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maclennan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">K-W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O’Grady</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Estimating divergence dates and substitution rates in the Drosophila phylogeny</span>. <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">3459</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/mss150">10.1093/molbev/mss150</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22683811">22683811</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subramanian</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Temporal patterns of fruit fly (Drosophila) evolution revealed by mutation clocks</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">36</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>. Epub 2003/09/02. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/msg236">10.1093/molbev/msg236</a></span> [pii]. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12949132">12949132</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storey</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>. <span class="mixed-article-title" title="mixed-article-title">A direct approach to false discovery rates</span>. <span class="source" tagx="source" title="source">J Roy Stat Soc B</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">64</span>:<span class="fpage" tagx="fpage" title="fpage">479</span>–<span class="lpage" tagx="lpage" title="lpage">98</span>. doi: Unsp 1369-7412/02/64479 <span class="comment">doi: <a href="https://doi.org/10.1111/1467-9868.00346">10.1111/1467-9868.00346</a></span> PubMed PMID: WOS:000177425500009.</span></li> <li tag="ref"><a name="ppat.1006951.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jan</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Divergent IRES elements in invertebrates</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">119</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">16</span>–<span class="lpage" tagx="lpage" title="lpage">28</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.virusres.2005.10.011">10.1016/j.virusres.2005.10.011</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16307820">16307820</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">KN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christian</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span>. <span class="mixed-article-title" title="mixed-article-title">The novel genome organization of the insect picorna-like virus Drosophila C virus suggests this virus belongs to a previously undescribed virus family</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">79</span> (<span class="issue" tagx="issue" title="issue">Pt 1</span>):<span class="fpage" tagx="fpage" title="fpage">191</span>–<span class="lpage" tagx="lpage" title="lpage">203</span>. Epub 1998/02/14. <span class="comment">doi: <a href="https://doi.org/10.1099/0022-1317-79-1-191">10.1099/0022-1317-79-1-191</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9460942">9460942</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakashima</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakamura</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Cleavage sites of the "P3 region" in the nonstructural polyprotein precursor of a dicistrovirus</span>. <span class="source" tagx="source" title="source">Arch Virol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">153</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">1955</span>–<span class="lpage" tagx="lpage" title="lpage">60</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s00705-008-0208-5">10.1007/s00705-008-0208-5</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18810573">18810573</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakashima</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uchiumi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Functional analysis of structural motifs in dicistroviruses</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">139</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">137</span>–<span class="lpage" tagx="lpage" title="lpage">47</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.virusres.2008.06.006">10.1016/j.virusres.2008.06.006</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18621089">18621089</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">UniProtKB [cited 2017]. Available from: <a href="http://www.uniprot.org/uniprot/O36966">http://www.uniprot.org/uniprot/O36966</a>.</span></li> <li tag="ref"><a name="ppat.1006951.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">1969</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. Epub 2012/03/01. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/mss075">10.1093/molbev/mss075</a></span> ; PubMed Central PMCID: PMC3408070.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748">22367748</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gowtage-Sequeria</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Host range and emerging and reemerging pathogens</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1842</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. Epub 2006/02/21. <span class="comment">doi: <a href="https://doi.org/10.3201/eid1112.050997">10.3201/eid1112.050997</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16485468">16485468</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webby</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Emergence of influenza A viruses</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">356</span>(<span class="issue" tagx="issue" title="issue">1416</span>):<span class="fpage" tagx="fpage" title="fpage">1817</span>–<span class="lpage" tagx="lpage" title="lpage">28</span>. Epub 2002/01/10. <span class="comment">doi: <a href="https://doi.org/10.1098/rstb.2001.0997">10.1098/rstb.2001.0997</a></span> ; PubMed Central PMCID: PMC1088557.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11779380">11779380</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinert</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitzgerald</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular dating of human-to-bovid host jumps by Staphylococcus aureus reveals an association with the spread of domestication</span>. <span class="source" tagx="source" title="source">Biol Lett</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">829</span>–<span class="lpage" tagx="lpage" title="lpage">32</span>. Epub 2012/05/26. <span class="comment">doi: <a href="https://doi.org/10.1098/rsbl.2012.0290">10.1098/rsbl.2012.0290</a></span> ; PubMed Central PMCID: PMC3440972.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22628096">22628096</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauter</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schindler</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Specht</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Landford</span><span class="given-names" tagx="given-names" title="given-names">WN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munch</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Tetherin-Driven Adaptation of Vpu and Nef Function and the Evolution of Pandemic and Nonpandemic HIV-1 Strains</span>. <span class="source" tagx="source" title="source">Cell Host &amp;amp; Microbe</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">409</span>–<span class="lpage" tagx="lpage" title="lpage">21</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/J.Chom.2009.10.004">10.1016/J.Chom.2009.10.004</a></span> PubMed PMID: ISI:000272539700006. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19917496">19917496</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Rij</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saleh</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berry</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Houk</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antoniewski</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The RNA silencing endonuclease Argonaute 2 mediates specific antiviral immunity in Drosophila melanogaster</span>. <span class="source" tagx="source" title="source">Genes &amp;amp; development</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">21</span>):<span class="fpage" tagx="fpage" title="fpage">2985</span>–<span class="lpage" tagx="lpage" title="lpage">95</span>. PubMed PMID: ISI:000241767900009.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17079687">17079687</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allison</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kohler</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ortega</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoover</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grove</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Host-Specific Parvovirus Evolution in Nature Is Recapitulated by In Vitro Adaptation to Different Carnivore Species</span>. <span class="source" tagx="source" title="source">Plos Pathogens</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">11</span>). doi: ARTN e1004475 <span class="comment">doi: <a href="https://doi.org/10.1371/journal.ppat.1004475">10.1371/journal.ppat.1004475</a></span> PubMed PMID: WOS:000345515800014. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25375184">25375184</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christian</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span>. <span class="source" tagx="source" title="source">Studies of Drosophila C and A viruses in Australian populations of Drosophila melanogaster</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Australian National University</span>; <span class="year" tagx="year" title="year">1987</span>.</span></li> <li tag="ref"><a name="ppat.1006951.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waldron</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robertson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crowson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferrai</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quintana</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The discovery, distribution and evolution of viruses associated with Drosophila melanogaster</span>. <span class="source" tagx="source" title="source">PLOS Biology</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">7</span>): <span class="fpage" tagx="fpage" title="fpage">e1002210</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pbio.1002210">10.1371/journal.pbio.1002210</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26172158">26172158</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teixeira</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferreira</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashburner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">The Bacterial Symbiont Wolbachia Induces Resistance to RNA Viral Infections in Drosophila melanogaster</span>. <span class="source" tagx="source" title="source">Plos Biology</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">2753</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>. <span class="comment">doi: <a href="https://doi.org/10.1371/journal.pbio.1000002">10.1371/journal.pbio.1000002</a></span> PubMed PMID: ISI:000261913700016. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19222304">19222304</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiggins</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Previous Exposure to an RNA Virus Does Not Protect against Subsequent Infection in Drosophila melanogaster</span>. <span class="source" tagx="source" title="source">Plos One</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">e73833</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0073833">10.1371/journal.pone.0073833</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24040086">24040086</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jousset</span><span class="given-names" tagx="given-names" title="given-names">FX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plus</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croizier</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">[Existence in Drosophila of 2 groups of picornavirus with different biological and serological properties]</span>. <span class="source" tagx="source" title="source">C R Acad Sci Hebd Seances Acad Sci D</span>. <span class="year" tagx="year" title="year">1972</span>;<span class="volume" tagx="volume" title="volume">275</span>(<span class="issue" tagx="issue" title="issue">25</span>):<span class="fpage" tagx="fpage" title="fpage">3043</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. Epub 1972/12/18. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4631976">4631976</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reed</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muench</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="mixed-article-title" title="mixed-article-title">A simple method of estimating fifty per cent endpoints</span>. <span class="source" tagx="source" title="source">The American Journal of Hygiene</span>. <span class="year" tagx="year" title="year">1938</span>;<span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">493</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>.</span></li> <li tag="ref"><a name="ppat.1006951.ref056" /><span class="label" tagx="label" title="label">56</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Andrews S. FastQC: a quality control tool for high throughput sequence data. Available at: <a href="http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc.2010">http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc.2010</a>.</span></li> <li tag="ref"><a name="ppat.1006951.ref057" /><span class="label" tagx="label" title="label">57</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolger</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usadel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">15</span>):<span class="fpage" tagx="fpage" title="fpage">2114</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btu170">10.1093/bioinformatics/btu170</a></span> ; PubMed Central PMCID: PMCPMC4103590.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404">24695404</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref058" /><span class="label" tagx="label" title="label">58</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handsaker</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wysoker</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fennell</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Homer</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Sequence Alignment/Map format and SAMtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">16</span>):<span class="fpage" tagx="fpage" title="fpage">2078</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a></span> ; PubMed Central PMCID: PMCPMC2723002.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref059" /><span class="label" tagx="label" title="label">59</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <span class="source" tagx="source" title="source">BMC Evolutionary Biology</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">214</span>. doi: Artn 214 <span class="comment">doi: <a href="https://doi.org/10.1186/1471-2148-7-214">10.1186/1471-2148-7-214</a></span> PubMed PMID: ISI:000253468300001. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref060" /><span class="label" tagx="label" title="label">60</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Team RDC. R: a language and environment for statistical computing. V 2.4. 2006.</span></li> <li tag="ref"><a name="ppat.1006951.ref061" /><span class="label" tagx="label" title="label">61</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paradis</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claude</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strimmer</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>. <span class="mixed-article-title" title="mixed-article-title">APE: Analyses of Phylogenetics and Evolution in R language</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">289</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14734327">14734327</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref062" /><span class="label" tagx="label" title="label">62</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudson</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatkin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maddison</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span>. <span class="mixed-article-title" title="mixed-article-title">Estimation of levels of gene flow from DNA sequence data</span>. <span class="source" tagx="source" title="source">Genetics</span>. <span class="year" tagx="year" title="year">1992</span>;<span class="volume" tagx="volume" title="volume">132</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">583</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. Epub 1992/10/01. ; PubMed Central PMCID: PMC1205159.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1427045">1427045</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref063" /><span class="label" tagx="label" title="label">63</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>. <span class="mixed-article-title" title="mixed-article-title">MCMC Methods for Multi-Response Generalized Linear Mixed Models: The MCMCglmm R Package</span>. <span class="source" tagx="source" title="source">Journal of Statistical Software</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">33</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>. PubMed PMID: WOS:000275203300001.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20808728">20808728</a></span></span></li> <li tag="ref"><a name="ppat.1006951.ref064" /><span class="label" tagx="label" title="label">64</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patefield</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Algorithm AS 159: An Efficient Method of Generating Random R × C Tables with Given Row and Column Totals</span>. <span class="source" tagx="source" title="source">Journal of the Royal Statistical Society Series C (Applied Statistics)</span>. <span class="year" tagx="year" title="year">1981</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">91</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="comment">doi: <a href="https://doi.org/10.2307/2346669">10.2307/2346669</a></span></span></li> </ul> </div> </div>  </body></html>