<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Viruses</span><span class="iso-abbrev" title="iso-abbrev">Viruses</span><span class="publisher-id" title="publisher-id">viruses</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Viruses</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1999-4915</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/v11050482">10.3390/v11050482</a></span><span class="publisher-id" title="publisher-id">viruses-11-00482</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Limited Sustained Local Transmission of HIV-1 CRF01_AE in New South Wales, Australia</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Giallonardo</span><span class="given-names" tagx="given-names" title="given-names">Francesca</span></span><a href="#af1-viruses-11-00482">1</a><a href="#c1-viruses-11-00482">*</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-3583-9490</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinto</span><span class="given-names" tagx="given-names" title="given-names">Angie N.</span></span><a href="#af1-viruses-11-00482">1</a><a href="#af2-viruses-11-00482">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-2678-0645</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keen</span><span class="given-names" tagx="given-names" title="given-names">Phillip</span></span><a href="#af1-viruses-11-00482">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaik</span><span class="given-names" tagx="given-names" title="given-names">Ansari</span></span><a href="#af1-viruses-11-00482">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrera</span><span class="given-names" tagx="given-names" title="given-names">Alex</span></span><a href="#af3-viruses-11-00482">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salem</span><span class="given-names" tagx="given-names" title="given-names">Hanan</span></span><a href="#af4-viruses-11-00482">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Telfer</span><span class="given-names" tagx="given-names" title="given-names">Barbara</span></span><a href="#af5-viruses-11-00482">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">Craig</span></span><a href="#af6-viruses-11-00482">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">Karen</span></span><a href="#af7-viruses-11-00482">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selvey</span><span class="given-names" tagx="given-names" title="given-names">Christine</span></span><a href="#af5-viruses-11-00482">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holden</span><span class="given-names" tagx="given-names" title="given-names">Joanne</span></span><a href="#af8-viruses-11-00482">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bachmann</span><span class="given-names" tagx="given-names" title="given-names">Nadine</span></span><a href="#af9-viruses-11-00482">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">Frederick J.</span></span><a href="#af4-viruses-11-00482">4</a><a href="#af10-viruses-11-00482">10</a><a href="#af11-viruses-11-00482">11</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwyer</span><span class="given-names" tagx="given-names" title="given-names">Dominic E.</span></span><a href="#af12-viruses-11-00482">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DuchÃªne</span><span class="given-names" tagx="given-names" title="given-names">SebastiÃ¡n</span></span><a href="#af13-viruses-11-00482">13</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-9596-3552</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">Edward C.</span></span><a href="#af11-viruses-11-00482">11</a><a href="#af14-viruses-11-00482">14</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grulich</span><span class="given-names" tagx="given-names" title="given-names">Andrew E.</span></span><a href="#af1-viruses-11-00482">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelleher</span><span class="given-names" tagx="given-names" title="given-names">Anthony D.</span></span><a href="#af1-viruses-11-00482">1</a><span class="on-behalf-of" tagx="on-behalf-of" title="on-behalf-of">on behalf of the NSW HIV Prevention Partnership Project</span></span></div><span class="citation_author_institution" id="af1-viruses-11-00482">[1], <span class="email" tagx="email" title="email">apinto@kirby.unsw.edu.au</span><span class="email" tagx="email" title="email">pkeen@kirby.unsw.edu.au</span><span class="email" tagx="email" title="email">ashaik@kirby.unsw.edu.au</span><span class="email" tagx="email" title="email">agrulich@kirby.unsw.edu.au</span><span class="email" tagx="email" title="email">akelleher@kirby.unsw.edu.au</span></span><span class="citation_author_institution" id="af2-viruses-11-00482">[2], </span><span class="citation_author_institution" id="af3-viruses-11-00482">[3], <span class="email" tagx="email" title="email">alex.carrera@svha.org.au</span></span><span class="citation_author_institution" id="af4-viruses-11-00482">[4], <span class="email" tagx="email" title="email">hanan.salem@health.nsw.gov.au</span><span class="email" tagx="email" title="email">frederick.lee@health.nsw.gov.au</span></span><span class="citation_author_institution" id="af5-viruses-11-00482">[5], <span class="email" tagx="email" title="email">barbara.telfer@health.nsw.gov.au</span><span class="email" tagx="email" title="email">christine.selvey@health.nsw.gov.au</span></span><span class="citation_author_institution" id="af6-viruses-11-00482">[6], <span class="email" tagx="email" title="email">craigc@positivelife.org.au</span></span><span class="citation_author_institution" id="af7-viruses-11-00482">[7], <span class="email" tagx="email" title="email">kprice@acon.org.au</span></span><span class="citation_author_institution" id="af8-viruses-11-00482">[8], <span class="email" tagx="email" title="email">jo.holden@health.nsw.gov.au</span></span><span class="citation_author_institution" id="af9-viruses-11-00482">[9], <span class="email" tagx="email" title="email">nadine.bachmann2@usz.ch</span></span><span class="citation_author_institution" id="af10-viruses-11-00482">[10], </span><span class="citation_author_institution" id="af11-viruses-11-00482">[11], <span class="email" tagx="email" title="email">edward.holmes@sydney.edu.au</span></span><span class="citation_author_institution" id="af12-viruses-11-00482">[12], <span class="email" tagx="email" title="email">dominic.dwyer@sydney.edu.au</span></span><span class="citation_author_institution" id="af13-viruses-11-00482">[13], <span class="email" tagx="email" title="email">sebastian.duchene@unimelb.edu.au</span></span><span class="citation_author_institution" id="af14-viruses-11-00482">[14], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">fdigiallonardo@kirby.unsw.edu.au</span>; Tel.: +61-9385-0465</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-5-5</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-5</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">482</span><span class="history" title="history"><span class="received" title="received">received: 2019-4-12</span><span class="accepted" title="accepted">accepted: 2019-5-24</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Australiaâ€™s response to the human immunodeficiency virus type 1 (HIV-1) pandemic led to effective control of HIV transmission and one of the worldâ€™s lowest HIV incidence ratesâ€"0.14%. Although there has been a recent decline in new HIV diagnoses in New South Wales (NSW), the most populous state in Australia, there has been a concomitant increase with non-B subtype infections, particularly for the HIV-1 circulating recombinant form CRF01_AE. This aforementioned CRF01_AE sampled in NSW, were combined with those sampled globally to identify NSW-specific viral clades. The population growth of these clades was assessed in two-year period intervals from 2009 to 2017. Overall, 109 NSW-specific clades were identified, most comprising pairs of sequences; however, five large clades comprising â‰¥10 sequences were also found. Forty-four clades grew over time with one or two sequences added to each in different two-year periods. Importantly, while 10 of these clades have seemingly discontinued, the remaining 34 were still active in 2016/2017. Seven such clades each comprised â‰¥10 sequences, and are representative of individual sub-epidemics in NSW. Thus, although the majority of new CRF01_AE infections were associated with small clades that rarely establish ongoing chains of local transmission, individual sub-epidemics are present and should be closely monitored.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
HIV-1 CRF01_AE</div> <div class="kwd" title="kwd">
transmission cluster</div> <div class="kwd" title="kwd">
evolution</div> <div class="kwd" title="kwd">
cluster growth</div> <div class="kwd" title="kwd">
sub-epidemic</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>The human immunodeficiency virus type 1 (HIV-1) is still a major burden worldwide with approximately 36.9 million individuals infected with HIV globally (WHO). The Joint United Nations Programme on HIV/AIDS (UNAIDS) set the 90/90/90 targets with the aim that by 2020 90% of all people living with HIV-1 know their status, 90% of these will be receiving antiretroviral therapy, and 90% of these will have an undetectable plasma viral load [<a href="#B1-viruses-11-00482">1</a>]. The latter is of particular importance, as infected people that are virally suppressed do not develop the acquired immunodeficiency syndrome (AIDS) [<a href="#B2-viruses-11-00482">2</a>], and cannot transmit the virus further while on treatment [<a href="#B3-viruses-11-00482">3</a>]. Australia has an estimated HIV-1 prevalence of 0.14%, with approximately 1000 new infections annually [<a href="#B4-viruses-11-00482">4</a>]. New South Wales (NSW) is the most populous state in Australia, and has the highest HIV-1 burden, with approximately 10,100 people living with HIV-1 [<a href="#B5-viruses-11-00482">5</a>]. Importantly, NSW achieved the 90/90/90 targets in 2016 [<a href="#B5-viruses-11-00482">5</a>,<a href="#B6-viruses-11-00482">6</a>], and has experienced the first decline in new infections in ten years [<a href="#B7-viruses-11-00482">7</a>]. However, this decline was only observed in Australian-born men-who-have-sex-with-men (MSM), and an increase in new infections was observed in overseas-born MSM [<a href="#B6-viruses-11-00482">6</a>], indicating that there are differences in how different risk groups have been impacted by public health interventions.</p> <p>HIV-1 circulates globally as different genetically defined subtypes, although the subtype prevalence varies across different countries; subtype C is most prevalent in Africa (and globally), the circulating recombinant form 01_AE (CRF01_AE) is most prevalent in Southeast Asia, and subtype B is most prevalent in the USA, Europe, and Australia [<a href="#B8-viruses-11-00482">8</a>]. This is of current importance because the HIV-1 epidemic in Australia is experiencing a shift in subtype incidence with a major increase in non-B infections, as an estimated 20% of all new infections in Australia were of non-B origin in 2006, and that increased to 38% in 2016 [<a href="#B9-viruses-11-00482">9</a>]. CRF01_AE is of particular interest for Australia due to the close proximity to Southeast Asia, with both travel and migration between these regions representing a conduit for HIV-1 importation [<a href="#B10-viruses-11-00482">10</a>]. Yet, little is known about the geographic origins and epidemiological characteristics of CRF01_AE in Australia.</p> <p>To better understand the phylodynamics of CRF01_AE in NSW we (i) identified independent transmission clades, (ii) estimated clade growth over time, and (iii) assessed the impact of transmission clade growth on the NSW epidemic.</p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.ethics" title="sec"> <div class="title" tagx="title" title="title">
2.1. Ethics</div> <p>This study was approved by the NSW Population and Health Services Research Ethics Committee and the ACON Research Ethics Review Committee (RERC) [AU RED Reference: HREC/15/CIHS/38, Cancer Institute NSW reference number: 2015/08/605]. The HIV/AIDS Legal Centre was consulted for legal advice on data anonymity.</p> </div> <div class="2.2.hiv-1sequencedata" title="sec"> <div class="title" tagx="title" title="title">
2.2. HIV-1 Sequence Data</div> <p>Protease and reverse transcriptase sequences sampled between 2004 and 2017 were obtained from the three laboratories performing HIV genotypic drug resistance testing for HIV-1 in NSW; St. Vincentâ€™s HIV State Reference Laboratory, New South Wales Health Pathology-Royal Prince Alfred Hospital, and New South Wales Health Pathology Institute of Clinical Pathology and Medical Research (ICPMR). </p> <p>The final data set was created by the Center for Health Record Linkage (CHeReL) by linking the individual sequence identifiers obtained from the laboratories performing resistance testing to HIV notification case identifiers from the NSW HIV registry. Patient data always remained anonymized. The earliest available sequence for each patient was used for the analyses to avoid including multiple sequences from the same patient (<i>n</i> = 7000). HIV-1 subtype was determined using the REGA HIV-1 Subtyping Tool Version 3.0 [<a href="#B11-viruses-11-00482">11</a>]. </p> </div> <div class="2.3.phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.3. Phylogenetic Analysis</div> <p>A maximum likelihood tree including 83 HIV-1 reference sequences from the REGA tool and the 7000 NSW sequences was estimated using FastTreeMP v2.1.10 [<a href="#B12-viruses-11-00482">12</a>] via the CIPRES Science Gateway [<a href="#B13-viruses-11-00482">13</a>], implementing a GTR+Î" model of nucleotide substitution, and accounting for invariable sites, and NNI branch-swapping was implemented for topology search. Sequences forming a monophyletic CRF01_AE group (<i>n</i> = 758) were used for subsequent analyses. All HIV-1 sequences were retrieved from the Los Alamos HIV database (<span class="uri" title="uri"><a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a></span>) and used for comparison. NSW protease and reverse transcriptase nucleotide sequences were blasted against the global database, and the best 50 hits for each NSW sequence were retrieved from the global database [<a href="#B14-viruses-11-00482">14</a>]. Accordingly, a total of 2807 global sequences were used as background sequence data; three HIV-1 subtype G sequences (accession numbers DQ168576 and EU786670) were added as an outgroup to root the phylogeny (<a href="#app1-viruses-11-00482">Table S1</a>). Pairwise alignment was performed in Mafft, implementing the L-INS-I algorithm [<a href="#B15-viruses-11-00482">15</a>]. The sequences from the Los Alamos HIV database, as well as the sequences obtained from the three reference laboratories in NSW differ slightly in their length and genome position. As a consequence, the alignment was reduced to the nucleotide positions 2253â€"3384, based on the reference sequence HXB2 (accession number K03455.1). The alignment was visually inspected in Geneious 11.1.3 (<span class="uri" title="uri"><a href="https://www.geneious.com">https://www.geneious.com</a></span>), and sequences that were too short were removed as well as codons associated with drug resistance mutations (according to reference sequence HXB2; accession number K03455.1) [<a href="#B16-viruses-11-00482">16</a>]. The final alignment comprised 3510 sequences of 1132 nucleotides in length, of which 702 were from NSW. Next, the NSW CRF01_AE sequences were split into four groups according to sampling year, with two new years of sampling added successively: (i) 2004â€"2009 (<i>n</i> = 149), (ii) 2004â€"2011 (<i>n</i> = 262), (iii) 2004â€"2013 (<i>n</i> = 387), (iv) 2004â€"2015 (<i>n</i> = 542), and (v) 2004â€"2017 (<i>n</i> = 702), the latter representing the complete data set. A maximum likelihood (ML) tree was estimated, employing FastTreeMP for each sub-data set, and using the same background data as described above. An approximate Shimodaira-Hasegawa test was implemented in PhyML v3.1. to test for node support [<a href="#B17-viruses-11-00482">17</a>,<a href="#B18-viruses-11-00482">18</a>]. Phylogenetic trees were visualized in FigTree v1.4.3. The high sampling density in this study and in potential subsequent studies enables the identification of complete transmission networks which would compromise patient privacy. Thus, a random subset of 10% of sequence data (<i>n</i> = 70) from the 702 CRF01_AE samples used in this study are available via NCBI under the accession numbers MK941065-MK941134 (<a href="#app1-viruses-11-00482">Table S1</a>).</p> </div> <div class="2.4.cladeidentification" title="sec"> <div class="title" tagx="title" title="title">
2.4. Clade Identification</div> <p>Clade identification was performed using R v3.4.3 [<a href="#B19-viruses-11-00482">19</a>], and a NSW-specific HIV-1 clade was defined as a node on the CRF01_AE phylogeny in which at least 80% of all tips were NSW sequences collected in this study [<a href="#B14-viruses-11-00482">14</a>,<a href="#B20-viruses-11-00482">20</a>,<a href="#B21-viruses-11-00482">21</a>]. ClusterPicker v1.2.5. [<a href="#B22-viruses-11-00482">22</a>] was used to identify potential active transmission clusters with that where characterized by a node support of 0.9 or higher and a genetic distance of 1.5% or lower [<a href="#B23-viruses-11-00482">23</a>,<a href="#B24-viruses-11-00482">24</a>].</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
3. Results and Discussion</div> <div class="3.1.increaseinnon-bhiv-1subtypeinnsw" title="sec"> <div class="title" tagx="title" title="title">
3.1. Increase in non-B HIV-1 Subtype in NSW</div> <p>Virus subtype was determined and changes in subtype incidence were measured over time for NSW HIV-1 sequences sampled between 2004â€"2017 (<i>n</i> = 7000). The proportion of non-B subtype sequences increased over time: Non-B subtypes comprised an average of 15% of all samples during 2004â€"2008, an average of 28% of all samples between 2009â€"2013, and 42% between 2014â€"2017 (<a href="#viruses-11-00482-f001">Figure 1</a>). Notably, CRF01_AE became the most prevalent non-B subtype and second most prevalent subtype overall in NSW, i.e., approximately 17% of all samples between 2014â€"2017 were CRF01_AE.</p> </div> <div class="3.2.identificationoflargensw-specificclades" title="sec"> <div class="title" tagx="title" title="title">
3.2. Identification of Large NSW-Specific Clades</div> <p>Virus sequences were available from 2004 onwards. Although sequence numbers were low in the first three years, they increased steadily between 2007 and 2017, with an average of 62 sequences available for each year after 2007 (<a href="#viruses-11-00482-f002">Figure 2</a>). These data were combined with a background data set consisting of global sequences to identify NSW-specific independent clades (<a href="#app1-viruses-11-00482">Figure S1</a>). For the complete data set including all 702 NSW sequences, a total of 109 NSW-specific clades were identified, comprising 463 sequences, representing 66% of all NSW sequences in this study. A total of 50% (<i>n</i> = 54) of these clades were sequence pairs, with no evidence of additional onwards transmission. However, 355 sequences fell within 55 clades characterized by three or more sequences, and were distributed as follows: 19 clades contained three sequences, nine clades contained four sequences, 13 clades containing five sequences, three clades contained six and eight sequences each, two clades contained nine and ten sequences each, and four clades contained more than 10 sequences, i.e., 14, 16, 19, and 68 sequences, respectively. This clade size pattern is typical for HIV-1 phylogenies, with the majority of clades being small (two sequences), and only a handful of larger clades representing sub-epidemics [<a href="#B25-viruses-11-00482">25</a>,<a href="#B26-viruses-11-00482">26</a>]. This pattern also strongly supports multiple independent virus entries, i.e., individual sub-epidemics, of CRF01_AE in NSW, which arrived in NSW either via acquisition outside the state due to travel, or migration to NSW. </p> <p>We next mapped clade durationâ€"that is, the time between the earliest and oldest sequences present in a cladeâ€"onto a time scale (<a href="#viruses-11-00482-f002">Figure 2</a>). Notably, while some small clades with two to four sequences were sampled within a short time frame, other small clades contain sequences sampled up to ten years apart. In fact, of the 100 clades identified, 51 spanned a time frame of three to 12 years, and 44 clades were sampled within one year. Thus, clade number and size appear to be growing over time.</p> <p>Notably, the background data set contained 361 sequences sampled in Australia; however, only two of these formed a cluster with the NSW sequences, such that the contribution of inter-state transmission within Australia is seemingly limited, which is in concordance with a previous study showing that there is only limited clustering between sequences from different states [<a href="#B27-viruses-11-00482">27</a>].</p> </div> <div class="3.3.cladegrowthisslowovertime" title="sec"> <div class="title" tagx="title" title="title">
3.3. Clade Growth is Slow over Time</div> <p>To assess the rate of clade growth, NSW sequences were split into four different data subsets according to their year of sampling and combined with the global sequences. The first sub-data set comprised of 149 NSW sequences sampled between 2004â€"2009. A wider time range was chosen in this case to enable a solid starting point for clade identification. Two additional sampling years were added for each subsequent sub-data set. NSW-specific clades were identified for each sub-data set. Clades where the most recent sequence was sampled before 2016 were classified as temporarily discontinued, although these may include more recent undiagnosed cases (11% of all HIV-1 infected people living in Australia are thought to be undiagnosed [<a href="#B9-viruses-11-00482">9</a>]). Ten such discontinued clades were found (<a href="#viruses-11-00482-f003">Figure 3</a>A). Notably, 34 clades were still active in 2017 (ongoing clades) and have been steadily growing over time, with an addition of one or two sequences for each two-year period. This includes larger clades found comprising more than ten sequences, and including the largest clade with 68 sequences, all of which represent independent sub-epidemics. This is consistent with a previous study showing that clades grow slowly and irregularly [<a href="#B28-viruses-11-00482">28</a>]. </p> <p>Within the starting data set with sequences up to 2009, approximately 42% of the sequences represented singletons, and 58% were part of a clade (<a href="#viruses-11-00482-f003">Figure 3</a>B). Notably, this was the highest ratio of singleton sequences as the proportion of singleton sequences decreased over time in the following sub-data sets from 38% in the 2011 sub-data set to 24% in the 2017 data set. </p> <p>In contrast, the proportion of sequences forming a new cluster increased initially from 15% in the 2011 sub-data set to 17% in the 2013 sub-data set, but then decreased again to 14% in the complete data set (<a href="#viruses-11-00482-f003">Figure 3</a>B). This could simply be associated with an increase in the extent of sampling (HIV-1 testing). However, the number of new HIV notifications actually decreased in NSW from 332 in 2011, to 317 in 2016, and 313 notifications in 2017 [<a href="#B6-viruses-11-00482">6</a>]. Hence, while existing clades are growing more slowly, new clades are arising, such that public health interventions to stop virus transmission may be impacting on different populations unevenly. Indeed, epidemiological differences have been observed between HIV-1 positive individuals born in Australia, and those born overseas; for example a higher proportion of late diagnoses were observed in overseas born individuals [<a href="#B9-viruses-11-00482">9</a>]. However, from the 702 NSW CRF01_AE sequences used in this study, only 51% had available meta data including transmission risk and country of birth, so that no correlation between demographic factor and clade size and duration could be performed. </p> </div> <div class="3.4.potentialactivetransmissionnetworks" title="sec"> <div class="title" tagx="title" title="title">
3.4. Potential Active Transmission Networks </div> <p>Finally, potential direct transmission events were investigated by implementing a distance threshold of 1.5% and a node support of 99%. Sequences fulfilling these parameters form a transmission network, representing either direct virus transmission between individuals (i.e., a transmitter-recipient pair), or virus transmission to two recipients from the same transmitter. Sixteen singleton sequences had a high similarity to one or more of the global sequences, representing a potential transmitter-recipient pair between a sequence from overseas and a sequence sampled in NSW, although the directionality of transmission could not be established. In addition, four NSW-specific clades had a low genetic distance with one or more sequences from the global data set, indicating that transmission between sequences from different countries is not uncommon (<a href="#app1-viruses-11-00482">Figure S2</a>). </p> <p>Forty NSW-specific sequence pairs had a low genetic distance and high node support, indicative of a transmission network. Similarly, 13 networks with three sequences, four networks with five sequences, and one network with four, six, and eight sequences, respectively, were also identified. One of these has been active for at least six years, which is indicative of active clade growth, and which could be prevented by focused public health interventions. Notably, in one of the networks all three sequences share the drug resistance mutations K101E, Y181C and G190S, which can lead to reduced susceptibility to the non-nucleoside analog reverse transcriptase inhibitor drug class [<a href="#B16-viruses-11-00482">16</a>]. </p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
4. Conclusions</div> <p>Our study has revealed the depth of active HIV-1 transmission for the CRF01_AE subtype in NSW, Australia. Importantly, the marked increase in CRF01_AE infections might be associated with an increase of small local transmission networks forming local sub-epidemics within the state. The slow growth in clade size and increase in clade number over time identified here highlights the likely effectiveness of interventions facilitating early diagnosis and contact-tracing.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The investigators on the NSW HIV Prevention Partnership Project are Andrew Grulich, Christine Selvey, Rebecca Guy, Garrett Prestage, Iryna Zablotska, Jo Holden, Tim Duck, Craig Cooper, Karen Price, Martin Holt, John de Wit, John Kaldor, Anthony Kelleher, and David Wilson. The project steering committee includes these investigators and Bill Whittaker, Phillip Keen, Prital Patel, Barbara Telfer, Steven Nigro, Cherie Power, and Nick Rose. The late Alan Brotherton, the late Adjunct Levinia Crooks, and the late David A. Cooper were also investigators on the project grant. The authors also want to acknowledge the numerous investigators of contributing studies of this cohort. We also thank Roger D. Kouyos, Huldrych F. GÃ¼nthard, and Karin J. Metzner for their insightful comments.</p> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The following are available online at <span class="uri" title="uri"><a href="https://www.mdpi.com/1999-4915/11/5/482/s1">https://www.mdpi.com/1999-4915/11/5/482/s1</a></span>, Figure S1: Global phylogeny of CRF 01_AE; Figure S2: Potential active transmission networks; Table S1: Accession numbers.</p> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="viruses-11-00482-s001.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization, F.D.G., A.N.P., C.C., K.P., J.H., E.C.H. and A.D.K.; Data curation, A.N.P., A.S., A.C., H.S., B.T., C.S. and A.D.K.; Funding acquisition, F.D.G., A.E.G. and A.D.K.; Methodology, F.D.G., A.N.P., N.B., and S.D.; Supervision, A.N.P. and A.D.K.; Visualization, F.D.G.; Writingâ€"original draft, F.D.G., A.N.P., P.K., F.J.L., D.E.D., E.C.H. and A.D.K.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was supported by the Swiss National Science Foundation (P300PA_174462 to F.D.G.), the Practitioner Fellowship, the NHMRC Postgraduate Scholarship (APP1074467 to A.N.P.), and the Australian Laureate Fellowship (FL170100022 to E.C.H.). This analysis was further supported by funding from NHMRC Partnership (APP1092852) and Practitioner Fellowship (APP117907) Grants, the NSW Ministry of Health, and UNSW Sydney. The Kirby Institute receives funding from the Australian Government Department of Health and is affiliated with the Faculty of Medicine, UNSW Sydney. No pharmaceutical industry grants were received for this study.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>F.J.L. has received educational grants and sponsorship from ViiV Healthcare, Gilead Sciences and MSD, speaker fees from AstraZeneca, MSD and the Australasian Society for HIV, Viral Hepatitis and Sexual Health Medicine (ASHM), and provided consultancy and served on advisory boards for Viiv Healthcare, MSD and Calimmune Australia. All other authors report no conflict of interest. </p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-viruses-11-00482" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: UNAIDS</span></span><span class="source" tagx="source" title="source">90-90-90 An Ambitious Treatment Target to Help End the AIDS Epidemic</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">UNAIDS</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva, Switzerland</span><span class="year" tagx="year" title="year">2014</span></span></li> <li tag="ref"><a name="B2-viruses-11-00482" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Insight Start Study Group</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lundgren</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babiker</span><span class="given-names" tagx="given-names" title="given-names">A.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gordin</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emery</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grund</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avihingsanon</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fatkenheuer</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Initiation of antiretroviral therapy in early asymptomatic HIV infection</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">373</span><span class="fpage" tagx="fpage" title="fpage">795</span><span class="lpage" tagx="lpage" title="lpage">807</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26192873">26192873</a></span></span></li> <li tag="ref"><a name="B3-viruses-11-00482" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vernazza</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirschel</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernasconi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flepp</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Les personnes sÃ©ropositives ne souffrant dâ€™aucune autre MST et suivant un traitement antirÃ©troviral efficace ne transmettent pas le VIH par voie sexuelle</span><span class="source" tagx="source" title="source">Bulletin des MÃ©decins Suisses</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">165</span><span class="lpage" tagx="lpage" title="lpage">169</span></span></li> <li tag="ref"><a name="B4-viruses-11-00482" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Kirby Institute</span></span><span class="source" tagx="source" title="source">Bloodborne Viral and Sexually Transmissible Infections in Aboriginal and Torres Strait Islander People: Annual Surveillance Report 2018</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Kirby Institute, UNSW Sydney</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sydney, Australia</span><span class="year" tagx="year" title="year">2018</span></span></li> <li tag="ref"><a name="B5-viruses-11-00482" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keen</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gray</span><span class="given-names" tagx="given-names" title="given-names">R.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Telfer</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">H.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whittaker</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holden</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelleher</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The 2016 HIV diagnosis and care cascade in New South Wales, Australia: Meeting the UNAIDS 90-90-90 targets</span><span class="source" tagx="source" title="source">J. Int. AIDS Soc.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">e25109</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jia2.25109">10.1002/jia2.25109</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29676000">29676000</a></span></span></li> <li tag="ref"><a name="B6-viruses-11-00482" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: NSW Ministry of Health</span></span><span class="source" tagx="source" title="source">NSW HIV Strategy 2016â€"2020 Quarter 4 and Annual 2018 Data Report</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Centre for Population Health</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">North Parramatta, NSW, Australia</span><span class="year" tagx="year" title="year">2018</span></span></li> <li tag="ref"><a name="B7-viruses-11-00482" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grulich</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guy</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amin</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selvey</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holden</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">H.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zablotska</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whittaker</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Population-level effectiveness of rapid, targeted, high-coverage roll-out of HIV pre-exposure prophylaxis in men who have sex with men: The EPIC-NSW prospective cohort study</span><span class="source" tagx="source" title="source">Lancet HIV</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e629</span><span class="lpage" tagx="lpage" title="lpage">e637</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S2352-3018(18)30215-7">10.1016/S2352-3018(18)30215-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30343026">30343026</a></span></span></li> <li tag="ref"><a name="B8-viruses-11-00482" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hemelaar</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The origin and diversity of the HIV-1 pandemic</span><span class="source" tagx="source" title="source">Trends Mol. Med.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">182</span><span class="lpage" tagx="lpage" title="lpage">192</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molmed.2011.12.001">10.1016/j.molmed.2011.12.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22240486">22240486</a></span></span></li> <li tag="ref"><a name="B9-viruses-11-00482" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: Kirby Institute</span></span><span class="source" tagx="source" title="source">HIV, Viral Hepatitis and Sexually Transmissible Infections in Australia: Annual Surveillance Report 2018</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Kirby Institute, UNSW Sydney</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sydney, Australia</span><span class="year" tagx="year" title="year">2018</span></span></li> <li tag="ref"><a name="B10-viruses-11-00482" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herring</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">Y.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratnamohan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunningham</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saksena</span><span class="given-names" tagx="given-names" title="given-names">N.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwyer</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Segregation of human immunodeficiency virus type 1 subtypes by risk factor in Australia</span><span class="source" tagx="source" title="source">J. Clin. Microbiol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">4600</span><span class="lpage" tagx="lpage" title="lpage">4604</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.41.10.4600-4604.2003">10.1128/JCM.41.10.4600-4604.2003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14532189">14532189</a></span></span></li> <li tag="ref"><a name="B11-viruses-11-00482" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="mixed-article-title" title="mixed-article-title">REGA HIV-1 Subtyping Tool v3.0</span><span class="comment">Available online: <a href="http://dbpartners.stanford.edu:8080/RegaSubtyping/stanford-hiv/typingtool/">http://dbpartners.stanford.edu:8080/RegaSubtyping/stanford-hiv/typingtool/</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 1 February 2019)</span></span></li> <li tag="ref"><a name="B12-viruses-11-00482" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">M.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehal</span><span class="given-names" tagx="given-names" title="given-names">P.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkin</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">FastTree: Computing large minimum evolution trees with profiles instead of a distance matrix</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">1641</span><span class="lpage" tagx="lpage" title="lpage">1650</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msp077">10.1093/molbev/msp077</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19377059">19377059</a></span></span></li> <li tag="ref"><a name="B13-viruses-11-00482" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfeiffer</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwartz</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Creating the CIPRES Science Gateway for inference of large phylogenetic trees</span><span class="source" tagx="source" title="source">Proceedings of the Gateway Computing Environments Workshop (GCE)</span><span class="conf-loc" tagx="conf-loc" title="conf-loc">New Orleans, LA, USA</span><span class="conf-date" tagx="conf-date" title="conf-date">14 November 2010</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span></span></li> <li tag="ref"><a name="B14-viruses-11-00482" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turk</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bachmann</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadelka</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerly</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aubert</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimkait</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Battegay</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernasconi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calmy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Assessing the danger of self-sustained HIV epidemics in heterosexuals by population based phylogenetic cluster analysis</span><span class="source" tagx="source" title="source">eLife</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">6</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.28721">10.7554/eLife.28721</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28895527">28895527</a></span></span></li> <li tag="ref"><a name="B15-viruses-11-00482" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuraku</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zmasek</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishimura</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">aLeaves facilitates on-demand exploration of metazoan gene family trees on MAFFT sequence alignment server with enhanced interactivity</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">W22</span><span class="lpage" tagx="lpage" title="lpage">W28</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkt389">10.1093/nar/gkt389</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23677614">23677614</a></span></span></li> <li tag="ref"><a name="B16-viruses-11-00482" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wensing</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calvez</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gunthard</span><span class="given-names" tagx="given-names" title="given-names">H.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">V.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paredes</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pillay</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shafer</span><span class="given-names" tagx="given-names" title="given-names">R.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richman</span><span class="given-names" tagx="given-names" title="given-names">D.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">2017 update of the drug resistance mutations in HIV-1</span><span class="source" tagx="source" title="source">Top. Antivir. Med.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">132</span><span class="lpage" tagx="lpage" title="lpage">133</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28208121">28208121</a></span></span></li> <li tag="ref"><a name="B17-viruses-11-00482" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guindon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dufayard</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lefort</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anisimova</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hordijk</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gascuel</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: Assessing the performance of PhyML 3.0</span><span class="source" tagx="source" title="source">Syst. Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">307</span><span class="lpage" tagx="lpage" title="lpage">321</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syq010">10.1093/sysbio/syq010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638">20525638</a></span></span></li> <li tag="ref"><a name="B18-viruses-11-00482" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anisimova</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gascuel</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Approximate likelihood ratio test for branchs: A fast, accurate and powerful alternative</span><span class="source" tagx="source" title="source">Syst. Biol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">539</span><span class="lpage" tagx="lpage" title="lpage">552</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/10635150600755453">10.1080/10635150600755453</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16785212">16785212</a></span></span></li> <li tag="ref"><a name="B19-viruses-11-00482" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: R Core Team</span></span><span class="source" tagx="source" title="source">R: A Language and Environment for Statistical Computing</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">R Foundation for Statistical Computing</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Vienna, Austria</span><span class="year" tagx="year" title="year">2017</span></span></li> <li tag="ref"><a name="B20-viruses-11-00482" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kusejko</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kadelka</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzel</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Battegay</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernasconi</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calmy</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavassini</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerly</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inferring the age difference in HIV transmission pairs by applying phylogenetic methods on the HIV transmission network of the Swiss HIV Cohort Study</span><span class="source" tagx="source" title="source">Virus Evol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">4</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vey024">10.1093/ve/vey024</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30250751">30250751</a></span></span></li> <li tag="ref"><a name="B21-viruses-11-00482" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Wyl</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerly</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boni</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taffe</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burgisser</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimkait</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirschel</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology reveals long-term changes in HIV type 1 subtype B transmission in Switzerland</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">201</span><span class="fpage" tagx="fpage" title="fpage">1488</span><span class="lpage" tagx="lpage" title="lpage">1497</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/651951">10.1086/651951</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20384495">20384495</a></span></span></li> <li tag="ref"><a name="B22-viruses-11-00482" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ragonnet-Cronin</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodcroft</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hue</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fearnhill</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delpech</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">A.J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lycett</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="collab'">collab: UK HIV Drug Resistance Database</span></span><span class="mixed-article-title" title="mixed-article-title">Automated analysis of phylogenetic clusters</span><span class="source" tagx="source" title="source">BMC Bioinform.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">317</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-14-317">10.1186/1471-2105-14-317</a></span></span></li> <li tag="ref"><a name="B23-viruses-11-00482" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Junqueira</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sibisi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilkinson</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Factors influencing HIV-1 phylogenetic clustering</span><span class="source" tagx="source" title="source">Curr. Opin. HIV AIDS</span><span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/COH.0000000000000540">10.1097/COH.0000000000000540</a></span></span></li> <li tag="ref"><a name="B24-viruses-11-00482" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberle</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Joos</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rusert</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campbell</span><span class="given-names" tagx="given-names" title="given-names">N.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beauparlant</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuster</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weber</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schenkel</span><span class="given-names" tagx="given-names" title="given-names">C.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scherrer</span><span class="given-names" tagx="given-names" title="given-names">A.U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magnus</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Tracing HIV-1 transmission: Envelope traits of HIV-1 transmitter and recipient pairs</span><span class="source" tagx="source" title="source">Retrovirology</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">62</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12977-016-0299-0">10.1186/s12977-016-0299-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27595568">27595568</a></span></span></li> <li tag="ref"><a name="B25-viruses-11-00482" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasylyeva</span><span class="given-names" tagx="given-names" title="given-names">T.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">du Plessis</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pineda-Pena</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhnert</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandamme</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomes</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Camacho</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abecasis</span><span class="given-names" tagx="given-names" title="given-names">A.B.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Tracing the impact of public health interventions on HIV-1 transmission in Portugal using molecular epidemiology</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="year" tagx="year" title="year">2019</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiz085">10.1093/infdis/jiz085</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30805610">30805610</a></span></span></li> <li tag="ref"><a name="B26-viruses-11-00482" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chibo</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birch</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Increasing diversity of human immunodeficiency virus type 1 subtypes circulating in Australia</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retrovir.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">578</span><span class="lpage" tagx="lpage" title="lpage">583</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/aid.2011.0200">10.1089/aid.2011.0200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22077905">22077905</a></span></span></li> <li tag="ref"><a name="B27-viruses-11-00482" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castley</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sawleshwarkar</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varma</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herring</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thapa</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dwyer</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chibo</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawke</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratcliff</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A national study of the molecular epidemiology of HIV-1 in Australia 2005â€"2012</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0170601</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0170601">10.1371/journal.pone.0170601</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28489920">28489920</a></span></span></li> <li tag="ref"><a name="B28-viruses-11-00482" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chibo</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaye</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birch</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">HIV transmissions during seroconversion contribute significantly to new infections in men who have sex with men in Australia</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retrovir.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">460</span><span class="lpage" tagx="lpage" title="lpage">464</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/aid.2011.0137">10.1089/aid.2011.0137</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21806486">21806486</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Changes in HIV-1 subtypes and circulating recombinant form 01_AE (CRF01_AE)_demographics over time. The proportion of subtypes is mapped across different years. Subtype B = yellow, subtype CRF01_AE = olive, subtype C = dark cyan, other non-recombinant subtypes = steel, and recombinants = blue.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Sampling size changes over time. Number of sequences mapped across different years (bottom). New South Wales (NSW)-specific clade duration is shown by plotting the duration from the oldest to the most recent sequence sampled within a clade (top). Clades are colored according to clade size that represents the number of sequences within each clade.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Discontinued and active ongoing NSW-specific clades. (<b>A</b>) Clade size for different clades is shown across the five sub-data sets. Discontinued clades are shown in orange with time period for the most recent sequence sampled shown with a red circle, sustained ongoing clades in blue, all other clades in black. Clade size for the analyzed time frame is indicated in the legend. (<b>B</b>) The number of singleton sequences is shown for the five different sub-data sets (dark blue). For each sub-data set new sequences belonging to an existing clade (blue) from the previous sub-data set, and new sequences forming a new clade (grey), are also shown. The x-axis defines the sequence sampling date grouped into the two-year time periods investigated.</p>   </div> </div>  </body></html>