<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/28/2020.03.27.009480.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.27.009480v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies" />
<meta name="DC.Identifier" content="10.1101/2020.03.27.009480" />
<meta name="DC.Date" content="2020-03-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 - COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics.

We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annotation of the identified SNVs but also of all currently circulating strain variations revealed colocalization of intra-host but also strain specific SNVs with primers and probes currently used in molecular diagnostics assays. Moreover, we de-novo assembled the viral genome, in order to isolate and validate intra-host structural variations and recombination breakpoints. The bioinformatics analysis disclosed genomic rearrangements over poly-A / poly-U regions located in ORF1ab and spike (S) gene, including a potential recombination hot-spot within S gene.

Our results highlight the intra-host genomic diversity and plasticity of SARS-CoV-2, pointing out genomic regions that are prone to alterations. The isolated SNVs and genomic rearrangements, reflect the intra-patient capacity of the polymorphic quasispecies, which may arise rapidly during the outbreak, allowing immunological escape of the virus, offering resistance to anti-viral drugs and affecting the sensitivity of the molecular diagnostics assays." />
<meta name="DC.Contributor" content="Timokratis Karamitros" />
<meta name="DC.Contributor" content="Gethsimani Papadopoulou" />
<meta name="DC.Contributor" content="Maria Bousali" />
<meta name="DC.Contributor" content="Anastasios Mexias" />
<meta name="DC.Contributor" content="Sotiris Tsiodras" />
<meta name="DC.Contributor" content="Andreas Mentis" />
<meta name="article:published_time" content="2020-03-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 -COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics. We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annotation of the identified SNVs but also of all currently circulating strain variations revealed colocalization of intra-host but also strain specific SNVs with primers and probes currently used in molecular diagnostics assays. Moreover, we de-novo assembled the viral genome, in order to isolate and validate intra-host structural variations and recombination breakpoints. The bioinformatics analysis disclosed genomic rearrangements over poly-A / poly-U regions located in ORF1ab and spike (S) gene, including a potential recombination hot-spot within S gene. Our results highlight the intra-host genomic diversity and plasticity of SARS-CoV-2, pointing out genomic regions that are prone to alterations. The isolated SNVs and genomic rearrangements, reflect the intra-patient capacity of the polymorphic quasispecies, which may arise rapidly during the outbreak, allowing immunological escape of the virus, offering resistance to anti-viral drugs and affecting the sensitivity of the molecular diagnostics assays.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.27.009480v1" />
<meta name="citation_id" content="2020.03.27.009480v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.009480v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.009480v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.009480v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.27.009480" />
<meta name="citation_num_pages" content="25" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.27.009480" />
<meta name="citation_author" content="Timokratis Karamitros" />
<meta name="citation_author_institution" content="Unit of Bioinformatics and Applied Genomics, Department of Microbiology, Hellenic Pasteur Institute" />
<meta name="citation_author_email" content="tkaram@pasteur.gr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0841-9159" />
<meta name="citation_author" content="Gethsimani Papadopoulou" />
<meta name="citation_author_institution" content="Unit of Bioinformatics and Applied Genomics, Department of Microbiology, Hellenic Pasteur Institute" />
<meta name="citation_author" content="Maria Bousali" />
<meta name="citation_author_institution" content="Unit of Bioinformatics and Applied Genomics, Department of Microbiology, Hellenic Pasteur Institute" />
<meta name="citation_author" content="Anastasios Mexias" />
<meta name="citation_author_institution" content="Unit of Bioinformatics and Applied Genomics, Department of Microbiology, Hellenic Pasteur Institute" />
<meta name="citation_author" content="Sotiris Tsiodras" />
<meta name="citation_author_institution" content="4th Academic Department of Medicine, National and Kapodistrian University of Athens, Medical School" />
<meta name="citation_author" content="Andreas Mentis" />
<meta name="citation_author_institution" content="Public Health Laboratories, Department of Microbiology, Hellenic Pasteur Institute" />
<meta name="citation_reference" content="V. M. Corman, D. Muth, D. Niemeyer, and C. Drosten, “Hosts and Sources of Endemic Human Coronaviruses,” vol. 100, pp. 163–188, 2018, doi: 10.1016/bs.aivir.2018.01.001." />
<meta name="citation_reference" content="A. R. Fehr and S. Perlman, “HHS Public Access,” pp. 1–23, 2016, doi: 10.1007/978-1-4939-2438-7." />
<meta name="citation_reference" content="D. Vijaykrishna, G. J. D. Smith, J. X. Zhang, J. S. M. Peiris, H. Chen, and Y. Guan, “Evolutionary Insights into the Ecology of Coronaviruses,” J. Virol., vol. 81, no. 8, pp. 4012–4020, 2007, doi: 10.1128/jvi.02605-06." />
<meta name="citation_reference" content="C. I. Paules, “Coronavirus Infections - More Than Just the Common Cold,” vol. 2520, pp. 3–4, 2020, doi: 10.1007/82." />
<meta name="citation_reference" content="F. Li, W. Li, M. Farzan, and S. C. Harrison, “Structural biology: Structure of SARS coronavirus spike receptor-binding domain complexed with receptor,” Science (80-.)., vol. 309, no. 5742, pp. 1864–1868, 2005, doi: 10.1126/science.1116480." />
<meta name="citation_reference" content="J. Cui, “Origin and evolution of pathogenic coronaviruses,” Nat. Rev. Microbiol., vol. 17, no. March, pp. 181–192, 2019, doi: 10.1038/s41579-018-0118-9." />
<meta name="citation_reference" content="J. H. Epstein et al., “Bats Are Natural Reservoirs of SARS-Like Coronaviruses,” Science (80-.)., vol. 310, no. 5748, pp. 676–679, 2005, doi: 10.1109/NEMS.2006.334722." />
<meta name="citation_reference" content="“Novel coronavirus 2019;-nCoV: early estimation of epidemiological parameters and epidemic predictions,” vol. 2020, no. January, 2020." />
<meta name="citation_reference" content="J. F. Chan et al., “Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan,” vol. 1751, 2020, doi: 10.1080/22221751.2020.1719902." />
<meta name="citation_reference" content="B. Li et al., “Rapid Reversion of Sequence Polymorphisms Dominates Early Human Immunodeficiency Virus Type 1 Evolution,” J. Virol., vol. 81, no. 1, pp. 193–201, 2007, doi: 10.1128/jvi.01231-06." />
<meta name="citation_reference" content="A. Beloukas et al., “Assessment of phylogenetic sensitivity for reconstructing HIV-1 epidemiological relationships,” Virus Res., vol. 166, no. 1–2, pp. 54–60, 2012." />
<meta name="citation_reference" content="M. Vignuzzi, J. K. Stone, J. J. Arnold, C. E. Cameron, and R. Andino, “Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population,” Nature, vol. 439, no. 7074, pp. 344–348, 2006, doi: 10.1038/nature04388." />
<meta name="citation_reference" content="T. Karamitros et al., “The interferon receptor-1 promoter polymorphisms affect the outcome of Caucasians with HB eAg-negative chronic HBV infection,” Liver Int., vol. 35, no. 12, pp. 2506–2513, 2015." />
<meta name="citation_reference" content="R. Malley, K. Trzcinski, A. Srivastava, C. M. Thompson, P. W. Anderson, and M. Lipsitch, “CD4+ T cells mediate antibody-independent acquired immunity to pneumococcal colonization,” Proc. Natl. Acad. Sci. U. S. A., vol. 102, no. 13, pp. 4848–4853, 2005, doi: 10.1073/pnas.0501254102." />
<meta name="citation_reference" content="M. Lipsitch and J. J. O’Hagan, “Patterns of antigenic diversity and the mechanisms that maintain them,” J. R. Soc. Interface, vol. 4, no. 16, pp. 787–802, 2007, doi: 10.1098/rsif.2007.0229." />
<meta name="citation_reference" content="C. Y. Yong, H. K. Ong, S. K. Yeap, K. L. Ho, and W. S. Tan, “Recent Advances in the Vaccine Development Against Middle East Respiratory Syndrome-Coronavirus,” Front. Microbiol., vol. 10, no. August, pp. 1–18, 2019, doi: 10.3389/fmicb.2019.01781." />
<meta name="citation_reference" content="R. J. Nicolas W. Cortes-Penfield, Barbara W. Trautner, “A decade after SARS: Strategies to control emerging coronaviruses,” Physiol. Behav., vol. 176, no. 5, pp. 139–148, 2017, doi: 10.1016/j.physbeh.2017.03.040." />
<meta name="citation_reference" content="B. Langmead and S. L. Salzberg, “Fast gapped-read alignment with Bowtie 2,” Nat. Methods, vol. 9, no. 4, pp. 357–359, Apr. 2012, doi: 10.1038/nmeth.1923." />
<meta name="citation_reference" content="J. T. Robinson, H. ThorvaldsdÃóttir, A. M. Wenger, A. Zehir, and J. P. Mesirov, “Variant review with the integrative genomics viewer,” Cancer Research, vol. 77, no. 21. American Association for Cancer Research Inc., pp. e31-e34, 01-Nov-2017, doi: 10.1158/0008-5472.CAN-17-0337." />
<meta name="citation_reference" content="H. Li et al., “The Sequence Alignment/Map format and SAMtools,” Bioinformatics, vol. 25, no. 16, pp. 2078–2079, Aug. 2009, doi: 10.1093/bioinformatics/btp352." />
<meta name="citation_reference" content="A. Wilm et al., “LoFreq: A sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets,” Nucleic Acids Res., vol. 40, no. 22, pp. 11189–11201, Dec. 2012, doi: 10.1093/nar/gks918." />
<meta name="citation_reference" content="P. Cingolani et al., “A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3,” Fly (Austin)., vol. 6, no. 2, pp. 80–92, 2012, doi: 10.4161/fly.19695." />
<meta name="citation_reference" content="P. Cingolani et al., “Using Drosophila melanogaster as a model for genotoxic chemical mutational studies with a new program, SnpSift,” Front. Genet., vol. 3, no. MAR, 2012, doi: 10.3389/fgene.2012.00035." />
<meta name="citation_reference" content="V. M. Corman et al., “Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR,” Euro Surveill., vol. 25, no. 3, Jan. 2020, doi: 10.2807/1560-7917.ES.2020.25.3.2000045." />
<meta name="citation_reference" content="A. Bankevich et al., “SPAdes: A new genome assembly algorithm and its applications to single-cell sequencing,” J. Comput. Biol., vol. 19, no. 5, pp. 455–477, May 2012, doi: 10.1089/cmb.2012.0021." />
<meta name="citation_reference" content="S. F. Altschul, W. Gish, W. Miller, E. W. Myers, and D. J. Lipman, “Basic local alignment search tool,” J. Mol. Biol., vol. 215, no. 3, pp. 403–410, Oct. 1990, doi: 10.1016/S0022-2836(05)80360-2." />
<meta name="citation_reference" content="A. R. Gruber, R. Lorenz, S. H. Bernhart, R. Neuböck, and I. L. Hofacker, “The Vienna RNA websuite,” Nucleic Acids Res., vol. 36, no. Web Server issue, p. W70, 2008, doi: 10.1093/nar/gkn188." />
<meta name="citation_reference" content="M. Cotten et al., “Full-genome deep sequencing and phylogenetic analysis of novel human betacoronavirus,” Emerg. Infect. Dis., vol. 19, no. 5, pp. 736–742, May 2013, doi: 10.3201/eid1905.130057." />
<meta name="citation_reference" content="B. L. Haagmans, A. C. Andeweg, and A. D. M. E. Osterhaus, “The Application of Genomics to Emerging Zoonotic Viral Diseases,” PLoS Pathog., vol. 5, no. 10, p. e1000557, Oct. 2009, doi: 10.1371/journal.ppat.1000557." />
<meta name="citation_reference" content="E.-G. Kostaki et al., “Unravelling the history of hepatitis B virus genotypes A and D infection using a full-genome phylogenetic and phylogeographic approach,” Elife, vol. 7, p. e36709, 2018." />
<meta name="citation_reference" content="D. N. Cooper and N. P. Group., Nature encyclopedia of the human genome. London; New York: Nature Pub. Group, 2003." />
<meta name="citation_reference" content="J. Aaskov, K. Buzacott, H. M. Thu, K. Lowry, and E. C. Holmes, “Long-term transmission of defective RNA viruses in humans and Aedes mosquitoes,” Science (80-.)., vol. 311, no. 5758, pp. 236–238, Jan. 2006, doi: 10.1126/science.1115030." />
<meta name="citation_reference" content="P. Farci et al., “The outcome of acute hepatitis C predicted by the evolution of the viral quasispecies,” Science (80-.)., vol. 288, no. 5464, pp. 339–344, Apr. 2000, doi: 10.1126/science.288.5464.339." />
<meta name="citation_reference" content="E. Karamichali et al., “HCV defective genomes promote persistent infection by modulating the viral life cycle,” Front. Microbiol., vol. 9, p. 2942, 2018." />
<meta name="citation_reference" content="G. Magiorkinis et al., “HIV-1 epidemic in Russia: an evolutionary epidemiology analysis,” Lancet, vol. 383, p. S71, 2014." />
<meta name="citation_reference" content="D. Paraskevis et al., “Molecular investigation of HIV-1 cross-group transmissions during an outbreak among people who inject drugs (2011--2014) in Athens, Greece,” Infect. Genet. Evol., vol. 62, pp. 11–16, 2018." />
<meta name="citation_reference" content="G. Magiorkinis et al., “An innovative study design to assess the community effect of interventions to mitigate HIV epidemics using transmission-chain phylodynamics,” Am. J. Epidemiol., vol. 187, no. 12, pp. 2615–2622, 2018." />
<meta name="citation_reference" content="J. Y. Noh et al., “Simultaneous detection of severe acute respiratory syndrome, Middle East respiratory syndrome, and related bat coronaviruses by real-time reverse transcription PCR,” Arch. Virol., vol. 162, no. 6, pp. 1617–1623, Jun. 2017, doi: 10.1007/s00705-017-3281-9." />
<meta name="citation_reference" content="T. Karamitros et al., “A contaminant-free assessment of Endogenous Retroviral RNA in human plasma,” Nat. Sci. Reports, vol. 6, p. 33598, 2016." />
<meta name="citation_reference" content="E. C. Holmes, The evolution and emergence of RNA viruses. Oxford University Press, 2009." />
<meta name="citation_reference" content="E.-G. Kostaki et al., “Spatiotemporal characteristics of the HIV-1 CRF02_AG/CRF63_02A1 epidemic in Russia and Central Asia,” AIDS Res. Hum. Retroviruses, vol. 34, no. 5, pp. 415–420, 2018." />
<meta name="citation_reference" content="M. M. C. Lai, “RNA recombination in animal and plant viruses,” Microbiological Reviews, vol. 56, no. 1. pp. 61–79, 1992, doi: 10.1128/mmbr.56.1.61-79.1992." />
<meta name="citation_reference" content="R. S. Baric, K. Fu, M. C. Schaad, and S. A. Stohlman, “Establishing a genetic recombination map for murine coronavirus strain A59 complementation groups,” Virology, vol. 177, no. 2, pp. 646–656, Aug. 1990, doi: 10.1016/0042-6822(90)90530-5." />
<meta name="citation_reference" content="S. K. P. Lau et al., “Molecular Epidemiology of Human Coronavirus OC43 Reveals Evolution of Different Genotypes over Time and Recent Emergence of a Novel Genotype due to Natural Recombination,” J. Virol., vol. 85, no. 21, pp. 11325–11337, Nov. 2011, doi: 10.1128/jvi.05512-11." />
<meta name="citation_reference" content="K. Pyrc et al., “Mosaic Structure of Human Coronavirus NL63, One Thousand Years of Evolution,” J. Mol. Biol., vol. 364, no. 5, pp. 964–973, Dec. 2006, doi: 10.1016/j.jmb.2006.09.074." />
<meta name="citation_reference" content="C.-C. Hon et al., “Evidence of the Recombinant Origin of a Bat Severe Acute Respiratory Syndrome (SARS)-Like Coronavirus and Its Implications on the Direct Ancestor of SARS Coronavirus,” J. Virol., vol. 82, no. 4, pp. 1819–1826, Feb. 2008, doi: 10.1128/jvi.01926-07." />
<meta name="citation_reference" content="J. S. M. Sabir et al., “Co-circulation of three camel coronavirus species and recombination of MERS-CoVs in Saudi Arabia,” Science (80-.)., vol. 351, no. 6268, pp. 81–84, Jan. 2016, doi: 10.1126/science.aac8608." />
<meta name="citation_reference" content="G. Lu, Q. Wang, and G. F. Gao, “Bat-to-human: Spike features determining ‘host jump’ of coronaviruses SARS-CoV, MERS-CoV, and beyond,” Trends Microbiol., vol. 23, no. 8, pp. 468–478, 2015, doi: 10.1016/j.tim.2015.06.003." />
<meta name="citation_reference" content="S. Agnihothram et al., “Evaluation of serologic and antigenic relationships between middle eastern respiratory syndrome coronavirus and other coronaviruses to develop vaccine platforms for the rapid response to emerging coronaviruses,” J. Infect. Dis., vol. 209, no. 7, pp. 995–1006, 2014, doi: 10.1093/infdis/jit609." />
<meta name="citation_reference" content="H. Bisht et al., “Severe acute respiratory syndrome coronavirus spike protein expressed by attenuated vaccinia virus protectively immunizes mice,” Proc. Natl. Acad. Sci. U. S. A., vol. 101, no. 17, pp. 6641–6646, Apr. 2004, doi: 10.1073/pnas.0401939101." />
<meta name="citation_reference" content="L. Enjuanes, M. L. DeDiego, E. Ãlvarez, D. Deming, T. Sheahan, and R. Baric, “Vaccines to prevent severe acute respiratory syndrome coronavirus-induced disease,” Virus Res., vol. 133, no. 1, pp. 45–62, Apr. 2008, doi: 10.1016/j.virusres.2007.01.021." />
<meta name="citation_reference" content="L. Jiaming et al., “The recombinant N-terminal domain of spike proteins is a potential vaccine against Middle East respiratory syndrome coronavirus (MERS-CoV) infection,” Vaccine, vol. 35, no. 1, pp. 10–18, Jan. 2017, doi: 10.1016/j.vaccine.2016.11.064." />
<meta name="twitter:title" content="SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 - COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics.

We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annotation of the identified SNVs but also of all currently circulating strain variations revealed colocalization of intra-host but also strain specific SNVs with primers and probes currently used in molecular diagnostics assays. Moreover, we de-novo assembled the viral genome, in order to isolate and validate intra-host structural variations and recombination breakpoints. The bioinformatics analysis disclosed genomic rearrangements over poly-A / poly-U regions located in ORF1ab and spike (S) gene, including a potential recombination hot-spot within S gene.

Our results highlight the intra-host genomic diversity and plasticity of SARS-CoV-2, pointing out genomic regions that are prone to alterations. The isolated SNVs and genomic rearrangements, reflect the intra-patient capacity of the polymorphic quasispecies, which may arise rapidly during the outbreak, allowing immunological escape of the virus, offering resistance to anti-viral drugs and affecting the sensitivity of the molecular diagnostics assays." />
<meta name="og-title" property="og:title" content="SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.009480v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 - COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics.

We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annotation of the identified SNVs but also of all currently circulating strain variations revealed colocalization of intra-host but also strain specific SNVs with primers and probes currently used in molecular diagnostics assays. Moreover, we de-novo assembled the viral genome, in order to isolate and validate intra-host structural variations and recombination breakpoints. The bioinformatics analysis disclosed genomic rearrangements over poly-A / poly-U regions located in ORF1ab and spike (S) gene, including a potential recombination hot-spot within S gene.

Our results highlight the intra-host genomic diversity and plasticity of SARS-CoV-2, pointing out genomic regions that are prone to alterations. The isolated SNVs and genomic rearrangements, reflect the intra-patient capacity of the polymorphic quasispecies, which may arise rapidly during the outbreak, allowing immunological escape of the virus, offering resistance to anti-viral drugs and affecting the sensitivity of the molecular diagnostics assays." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.27.009480v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.27.009480v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.27.009480v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.27.009480v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1215890" />
    <title>SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CCzgBud6h4XwjOtxX_st5ozJXRI_9ltSQEeif3yiSGg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1215890","apath":"\/biorxiv\/early\/2020\/03\/28\/2020.03.27.009480.atom","pisa":"biorxiv;2020.03.27.009480v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.27.009480v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.27.009480v1","title":"SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies","identifier":"node\/1215890"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r0iVsGMrhXM0vBgSBOZokM9Ka2OkijrPGGtGom5NINVi9Sv02hZNp304ubBm+wVE6T4HUgZwPs9WAzHJwhov1U96wBtXXYgcGnz7eM8UG0mV\\\\\\\/T4\\\\\\\/+MswB3KxjcW\\\\\\\/BZJODuVR\\\\\\\/ryWA6Yuf5LVhIjop9WCLeBmwWozratTnRD\\\\\\\/tZ5ry6bZKQl8+OGtZAsNm22FOAuH1ElHccJohuuXQO\\\\\\\/Y3+my3BWQYkVEm9sQrBDYwsdiaFjVTWZZT5h9Xupmd4Qrltd8G03DywK1fTXks1Z1Wl\\\\\\\/M8TcFkH0a7pzrdww6gLJzJUQLYj5thgEvIABooehcJYAxmH0O5qMUL6AnXCgCNXjnJ27cqjVMPZJyxBQmf4A47hBUrFJ8Ukx3EW3o\\\\\\\/WW86\\\\\\\/glTPEICuEOq22bSg\\\\\\\/F2wVJ7lL7RLaTVVXf+XGp9jweTfEdzF\\\\\\\/Yf9H+lV8Dfi4C9km6GPiZxmDkMsrOokhfasqXWmdht6yPM4lNOb+tlZh8fS2GmZjfBCKXlRnvfO0GYwKVprgUS8cX+cvFVoT966YCEONx\\\\\\\/J01ClEtT1gRZZxThrsWgU+kHhvv2OasLZCEpVhtcxgF3IfV0nEGHP8x82l8bj8BN1E0N5OviQwJxTex5lnwlvrtkbp9hppqWmcJF3LEiOExGoTPdQ8Ub3ju6dL3N9pgrKz++\\\\\\\/h7xfkO27L5kDMhEXKTOb\\\\\\\/XceTniCahOC8zHDrg6+jap1JXwfqGxggBIqkt5vHHPF9FkYbOzvxgJSGXBxpsi2rtaTRxgmQufA27IkKV8EuLl4MSuZqfRj74oA3bC4A55E0pClwp\\\\\\\/D32gjnI1AWGFbAynaGwTCTJelKsuY5H988ktr232VFL9M1xePWnmU3CjnZSPeOlqtbDItuokg581mJ09+nvsAR2dPCl7BXj4gFxitDtW6LK4ws8hjkbD8ORXaXGYxGIXs2EDqF0t4XU40SxBlpuu3fiJvtqoSjdeiJSPQHh\\\\\\\/ao6Sul6WGhZyQtxcAn+AuCcQgPK5gyHcTgNaMltX3KnVgkdF1Cl27lps8nGSfzKNhQBn61UIfnAbvVlhcs835DZzsOmcqhLEWP5SenTRoY\\\\\\\/YOxOoizfM74qMHujvn\\\\\\\/GLT2AAjudiHrsJYg8SMrlV58uTrtKyvGOJEV0vgQ7mDbralRo2u5uMZ\\\\\\\/p5BWIpbCoCbb+mj2pfaXwm4uVdgduYisqzH+R5mxYKlc3K5B5IgpBb5FJkA2Cm+zoueZEsSfWe9LJ4X4SMF3fBjNWsWQGVYIkRGhGoDeL19dq8B9PueMdH9lQ0MDAUbQD\\\\\\\/\\\\\\\/MFeZIQSY7n3jV7yuIelFlFqTymAbvv\\\\\\\/yuf38fAQrl+oOcMKKaZKb1l\\\u0022,\\\u0022iv\\\u0022:\\\u00228iuDACFya3yI8EpoDPjh8A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00223b3f90bcb412eb4bd7d3aaab92804b3693f8f32e043e0cc8ec3cb8f420340645\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.27.009480v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1215890 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.27.009480v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Amq1lQA9tFhd5d04TnJ-1pFwquj7jlRK9f6hzZZqM8k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215890" id="node1215890" data-pisa="biorxiv;2020.03.27.009480v1" data-pisa-master="biorxiv;2020.03.27.009480" data-apath="/biorxiv/early/2020/03/28/2020.03.27.009480.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0841-9159" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Timokratis</span> <span class="nlm-surname">Karamitros</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gethsimani</span> <span class="nlm-surname">Papadopoulou</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Bousali</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Anastasios</span> <span class="nlm-surname">Mexias</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Sotiris</span> <span class="nlm-surname">Tsiodras</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Mentis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.27.009480 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1215890" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Timokratis Karamitros </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Unit of Bioinformatics and Applied Genomics, Department of Microbiology, Hellenic Pasteur Institute</span>, Athens, <span class='nlm-country'>Greece</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timokratis%2BKaramitros%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Karamitros%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimokratis%2BKaramitros%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0841-9159" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timokratis Karamitros</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tkaram@pasteur.gr" class="" data-icon-position="" data-hide-link-title="0">tkaram@pasteur.gr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gethsimani Papadopoulou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Unit of Bioinformatics and Applied Genomics, Department of Microbiology, Hellenic Pasteur Institute</span>, Athens, <span class='nlm-country'>Greece</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gethsimani%2BPapadopoulou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Papadopoulou%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGethsimani%2BPapadopoulou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maria Bousali </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Unit of Bioinformatics and Applied Genomics, Department of Microbiology, Hellenic Pasteur Institute</span>, Athens, <span class='nlm-country'>Greece</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BBousali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bousali%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BBousali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Anastasios Mexias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Unit of Bioinformatics and Applied Genomics, Department of Microbiology, Hellenic Pasteur Institute</span>, Athens, <span class='nlm-country'>Greece</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasios%2BMexias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mexias%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasios%2BMexias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sotiris Tsiodras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-sup'>th</span>, Athens, <span class='nlm-country'>Greece</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sotiris%2BTsiodras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsiodras%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASotiris%2BTsiodras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andreas Mentis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Public Health Laboratories, Department of Microbiology, Hellenic Pasteur Institute</span>, Athens, <span class='nlm-country'>Greece</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BMentis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mentis%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BMentis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.27.009480v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1215890" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1215890/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.27.009480v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1215890" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1215890/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.27.009480v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1215890" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1215890/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.27.009480v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1215890" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1215890/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.27.009480v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1215890" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1215890/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.27.009480v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1215890" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1215890/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">In December 2019, an outbreak of atypical pneumonia (Coronavirus disease 2019 - COVID-19) associated with a novel coronavirus (SARS-CoV-2) was reported in Wuhan city, Hubei province, China. The outbreak was traced to a seafood wholesale market and human to human transmission was confirmed. The rapid spread and the death toll of the new epidemic warrants immediate intervention. The intra-host genomic variability of SARS-CoV-2 plays a pivotal role in the development of effective antiviral agents and vaccines, but also in the design of accurate diagnostics.</p><p id="p-3">We analyzed NGS data derived from clinical samples of three Chinese patients infected with SARS-CoV-2, in order to identify small- and large-scale intra-host variations in the viral genome. We identified tens of low- or higher-frequency single nucleotide variations (SNVs) with variable density across the viral genome, affecting 7 out of 10 protein-coding viral genes. The majority of these SNVs corresponded to missense changes. The annotation of the identified SNVs but also of all currently circulating strain variations revealed colocalization of intra-host but also strain specific SNVs with primers and probes currently used in molecular diagnostics assays. Moreover, we de-novo assembled the viral genome, in order to isolate and validate intra-host structural variations and recombination breakpoints. The bioinformatics analysis disclosed genomic rearrangements over poly-A / poly-U regions located in ORF1ab and spike (S) gene, including a potential recombination hot-spot within S gene.</p><p id="p-4">Our results highlight the intra-host genomic diversity and plasticity of SARS-CoV-2, pointing out genomic regions that are prone to alterations. The isolated SNVs and genomic rearrangements, reflect the intra-patient capacity of the polymorphic quasispecies, which may arise rapidly during the outbreak, allowing immunological escape of the virus, offering resistance to anti-viral drugs and affecting the sensitivity of the molecular diagnostics assays.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-5">Coronaviruses (CoVs), considered to be the largest group of viruses, belong to the <em>Nidovirales</em> order, <em>Coronaviridae</em> family and <em>Coronavirinae</em> subfamily, which is further subdivided into four genera, the alpha- and betacoronaviruses, which infect mammalian species and gamma- and deltacoronaviruses infecting mainly birds [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>], [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Small mammals (mice, dogs, cats) serve as reservoirs for HCoVs, with significant diversity seen in bats, which are considered to be primordial hosts of HCoVs [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. On the contrary, peridomestic animals are usually intermediate hosts, who enable long-term establishment of endemicity of the viruses, facilitating mutations and recombination events [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>], [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>].</p><p id="p-6">Until 2002, minor consideration was given to HCoVs, as they were associated with mild-to-severe disease phenotypes in immunocompetent people [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>]–[<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. In 2002, the beginning of severe acute respiratory syndrome (SARS) outbreak took place [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. In 2005, after the discovery of SARS-CoV-related viruses in horseshoe bats (<em>Rhinolophus</em>), palm civets were suggested as intermediate hosts, and bats as primordial hosts of the virus [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>], [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. In 2012, the emerging Middle East respiratory syndrome coronavirus (MERS-CoV) caused an outbreak in Saudi Arabia, which affected both camels and humans (44% mortality).</p><p id="p-7">On December 31<sup>st</sup> – 2019, a novel Coronavirus (SARS-CoV-2) was first reported from the city of Wuhan, Hubei province in China, causing severe infection of the respiratory tract in humans, after the identification of a group of similar cases of patients with pneumonia of unknown etiology [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>] (<a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">https://www.who.int/emergencies/diseases/novel-coronavirus-2019</a>). Similarly to SARS, epidemiological links between the majority of 2019-nCoV cases and Huanan South China Seafood Market, a live-animal market, have been reported. A total of 76,775 confirmed cases of “Coronavirus Disease 2019” (COVID-19) were reported up to February 21<sup>st</sup> 2020, from which 2,247 died and 18,855 recovered. Notably, 75,447 of the confirmed cases were reported in China (<a href="https://www.gisaid.org/epiflu-applications/global-cases-betacov/">https://www.gisaid.org/epiflu-applications/global-cases-betacov/</a>).</p><p id="p-8">The size of the ssRNA genome of SARS-CoV-2 is 29,891 nucleotides, it encodes 9860 amino acids and is characterized by nucleotide identity of ~ 89% with bat SARS-related-CoV SL-ZXC21 and ~ 82% with human SARS-CoVs BJ01 2003 and Tor2 [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. CoVs are enveloped positive-sense RNA viruses, which are characterized by a very large non-segmented RNA genome (26 to 32kb length), ready to be translated [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>], [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>]. The genes arrangement on the SARS-CoV-2 genome is: 5′UTR-replicase (ORF1/ab) -Spike (S) -ORF3a -Envelope (E) - Membrane (M) -ORF6 -ORF7a -ORF8 -Nucleocapsid (N) ORF10 −3′UTR [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>]. The main difference between SARS-CoV-2 and SARS-CoV is in ORF3b, ORF8 and Spike.</p><p id="p-9">Intra host variability of pathogenic viruses and bacteria represents a significant barrier in the control of infectious diseases. In viral infections, this variation emerges from genomic phenomena taking place during error-prone replication, ending up to multiple circulating quasispecies of low or higher frequency [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. These variants, in combination with the genetic profile of the host, can potentially influence the natural history of the infection, the viral phenotype, but also the sensitivity of molecular and serological diagnostics assays [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Importantly, intra-host genomic variability leads to antigenic variability, which is of higher importance, especially for pathogens that fail to elicit long-lasting immunity in their hosts, and remains a major contributor to the complexity of vaccine design [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>], [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. To date, there are no clinically approved vaccines available for protection of general population from SARS- and MERS-CoV infections as there is no effective vaccine to induce robust cell mediated and humoral immune responses [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>], [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>].</p><p id="p-10">Here, we explore intra-host genomic variants and low-frequency polymorphic quasispecies in Next Generation Sequencing (NGS) data derived from patients infected by SARS-CoV-2. Our analyses provide insights into the intra-patient pool of viral genomes, identify the frequency levels of rare variants and highlight variable genomic regions and a potential recombination hot-spot within S gene. Intra-host genomic variability is critical for the development of novel drugs and vaccines, which are of urgent necessity, towards the containment of this newly emerging epidemic.</p></div><div class="section" id="sec-2"><h2 class="">MATERIALS AND METHODS</h2><p id="p-11">In this study we analysed NGS data derived from clinical specimens (oral swabs) from three Chinese patients infected by SARS-CoV-2 (SRA projects PRJNA601736 and PRJNA603194). We aligned the raw read data on reference strain MN975262.1 using bowtie2 [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], after quality check with FastQC (<a href="https://www.bioinformatics.bbsrc.ac.uk/projects/fastqc">www.bioinformatics.bbsrc.ac.uk/projects/fastqc</a>). The resulting alignments were visualized with the <em>Integrated Genomics Viewer</em> (IGV) [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. After removing PCR duplicates, SNVs were called with a Bonferroni-corrected <em>P</em>-value threshold of 0.05 using <em>samtools</em> [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] and <em>LoFreq</em> [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. Variants supported by absolute read concordance (&gt;98%) were filtered-out from intra-host variant frequency calculations. We annotated the variations to the reference strain using snpEff [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], SNVs effects were further filtered with snpSift [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>] and we estimated the average mutation rate per gene across the viral genome using R scripts. We compared the localization of the intra-host SNVs with all available SNVs observed at population level up to February 18<sup>th</sup> 2020 (retrieved from <a href="https://www.GISAID.org">www.GISAID.org</a>). We also compared all intra-host and population level SNPs with all primers and probes coordinates to investigate for potential interferences with currently available molecular diagnostic assays [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] (<a href="https://www.who.int/docs/default-source/coronaviruse/peiris-protocol-16-1-20.pdf">www.who.int/docs/default-source/coronaviruse/peiris-protocol-16-1-20.pdf</a>).</p><p id="p-12">To investigate intra-host genomic rearrangements, we performed <em>de novo</em> assembly of the SARS-CoV-2 genomes using Spades [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>], and the resulting contigs were analyzed with BLAST [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] and confirmed by remapping of the raw reads. Smaller contigs (&lt;200 bp) were elongated where possible, after pair-wise realignment of the corresponding mapped reads. Basic computations and visualizations we implemented in R programming language R version 3.6.2, using in-house scripts. The secondary structures of the genomic regions surrounding the recombination breakpoints was predicted using RNAfold [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>].</p></div><div class="section" id="sec-3"><h2 class="">RESULTS</h2><p id="p-13">The mapping assembly of the viral genome was almost complete for all samples. The genome coverage and the average read depth across the genome was 99.99% and 133.5x for sample SRR10903401, 99.99% and 522.5x for sample SRR10903402, and 99.94%, and 598.2x for sample SRR10971381, respectively. The alignment statistics for all samples are summarized in <strong>Suppl.Table 1</strong>.</p><p id="p-14">In all cases we isolated the same 5 SNVs with 98-100% read concordance, thus in total divergence with the reference strain (MN975262.1), which were excluded from the downstream analysis. For sample SRR10903401 we isolated 34 lower frequency SNVs in total. Off these, 33 were present with frequencies ranking between 2 and 15%, while only one was present in 40% of the intra-host viral population. The sequencing depth, which is also evaluated during the SNV calling by the LoFreq algorithm, ranked between 39x and 290x at the corresponding SNV positions. The sequencing depth of sample SRR10903402 at the polymorphic positions was substantially higher (103x – 1137x), allowing the isolation of 55 SNVs with frequencies distributed between 0.9% and 14%. The depth over the polymorphic positions of sample SRR10971381 was between 159x – 1872x, allowing the isolation of 10 intra-host SNVs, with frequencies 1.1% - 6.8% (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1.A</a>, Suppl.Table 2</strong>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Intra &#x2013; host SNVs: (A) Intra host SNV frequency vs read depth in the corresponding alignment position (B) SNVs overlaps between the samples analysed (C) Intra-host SNVs frequency vs. variant type &#x2013; synonymous, missense, monsense (low, moderate, high impact respectively). (D) Intra-host SNVs frequency vs. all seven genes affected (ORF1ab, S, ORF3a, ORF6, ORF7a, ORF8, N). Average values are in red rhombs. (E) Density histogram of intra-host SNVs across the SARS-CoV-2 genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942535232" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Intra &#x2013; host SNVs: (A) Intra host SNV frequency vs read depth in the corresponding alignment position (B) SNVs overlaps between the samples analysed (C) Intra-host SNVs frequency vs. variant type &#x2013; synonymous, missense, monsense (low, moderate, high impact respectively). (D) Intra-host SNVs frequency vs. all seven genes affected (ORF1ab, S, ORF3a, ORF6, ORF7a, ORF8, N). Average values are in red rhombs. (E) Density histogram of intra-host SNVs across the SARS-CoV-2 genome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F1.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F1.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-15" class="first-child">Intra – host SNVs: (A) Intra host SNV frequency vs read depth in the corresponding alignment position (B) SNVs overlaps between the samples analysed (C) Intra-host SNVs frequency vs. variant type – synonymous, missense, monsense (low, moderate, high impact respectively). (D) Intra-host SNVs frequency vs. all seven genes affected (ORF1ab, S, ORF3a, ORF6, ORF7a, ORF8, N). Average values are in red rhombs. (E) Density histogram of intra-host SNVs across the SARS-CoV-2 genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Intra-host variants were distributed across 7 out of the 10 protein-coding genes of the viral genome, namely ORF1ab, S, ORF3a, ORF6, ORF7a, ORF8 and N. After normalising for the gene length (variants / kb-gene-length – “v/kbgl”), the higher density was observed in the small ORF6 (16.21 v/kbgl), followed by ORF8 (8.21 v/kbgl), N (4.76 v/kbgl), S (4.18 v/kbgl), ORF1ab (3.47 v/kbgl), ORF7a (2.73 v/kbgl) and ORF3a (1.21 v/kbgl). Interestingly, the majority of the SNPs corresponded to missense changes (leading to amino-acid change) compared to synonymous changes (72 vs. 29 respectively, ratio 2.48:1) (<strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong>). The average intra-host variant frequency did not differ substantially either between missense and synonymous polymorphisms (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1.C</a></strong>), neither between their hosting genes (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1.D</a></strong>). We did not detect any small-scale insertions or deletions in the samples (<strong>Suppl. Table 2</strong>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1223151/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1223151/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1223151" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <span class="caption-title">Impact of Intra-host SNVs on viral genes</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">The comparison of all SNVs (intra-host and population level) with the genomic targets of the molecular diagnostics assays, revealed colocalizations of three intra-host SNVs and 2 isolate-specific SNVs with primers and probes currently in use. In detail, intra-host SNVs colocalized with the probe of RdRP_SARSr reaction (15,474 T &gt; G), with the reverse primer of HKU-N reaction (28,971 A &gt; G) and with the probe of 2019-nCoV-N2 reaction (29,095 T &gt; C). More importantly, two SNVs belonging to isolates Wuhan/IVD-HB-04/2020 and Chongqing/YC01/2020, colocalized with the forward primer of 2019-nCoV-N1 reaction (28,291 C &gt; T) and the probe of 2019nCoV-N2 reaction (29,200 C &gt; T), respectively (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a></strong>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Truncated map of SARS-CoV-2 genome illustrating a subset of intra-host (blue lines) and globally collected, strain-specific SNVs (orange lines) with respect to the genomic targets of molecular diagnostics assays (red arrows &#x2013; primers, red bars - probes). Three intra-host variants (orange triangles), and two strain specific variants (Wuhan/IVD-HB-04/2020 and Chongqing/YC01/2020 - red triangles), are colocalized with RdRP_SARSr probe (15,474 T &gt; G), HKU-N primer (28,971 A &gt; G) and 2019-nCoV-N2 probe (29,095 T &gt; C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942535232" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Truncated map of SARS-CoV-2 genome illustrating a subset of intra-host (blue lines) and globally collected, strain-specific SNVs (orange lines) with respect to the genomic targets of molecular diagnostics assays (red arrows &#x2013; primers, red bars - probes). Three intra-host variants (orange triangles), and two strain specific variants (Wuhan/IVD-HB-04/2020 and Chongqing/YC01/2020 - red triangles), are colocalized with RdRP_SARSr probe (15,474 T &gt; G), HKU-N primer (28,971 A &gt; G) and 2019-nCoV-N2 probe (29,095 T &gt; C).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F2.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F2.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-18" class="first-child">Truncated map of SARS-CoV-2 genome illustrating a subset of intra-host (blue lines) and globally collected, strain-specific SNVs (orange lines) with respect to the genomic targets of molecular diagnostics assays (red arrows – primers, red bars - probes). Three intra-host variants (orange triangles), and two strain specific variants (Wuhan/IVD-HB-04/2020 and Chongqing/YC01/2020 - red triangles), are colocalized with RdRP_SARSr probe (15,474 T &gt; G), HKU-N primer (28,971 A &gt; G) and 2019-nCoV-N2 probe (29,095 T &gt; C).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The <em>de novo</em> assembly of the viral genomes revealed intra-host genomic rearrangements. For samples SRR10903401 and SRR10903402, these large-scale structural events were systematically observed over poly-A / poly-U-rich genomic regions, located in ORF1ab and S genes. In all cases, similar or identical strings of nucleotides in close proximity appear to have served as seeds for homologous recombination events. All rearrangements were validated by remapping of the raw reads on the corresponding <em>de novo</em> assembled contigs, setting a threshold of at least 5 supporting reads of high mapping quality (&gt;40) in each case. For sample SRR10903401 we isolated three inversions/misassemblies in ORF1ab (<strong>Suppl. Figure 2</strong>) and one inversion/misassembly in S gene (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3-A</a></strong>). Notably, we were able to validate the same inversion in S gene for sample SRR10903402 as well (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3-B</a></strong>). Apart from 2 inversions in ORF1ab supported by only 2 reads each (not passing the validation threshold), there were no further large-scale intra-host events observed for sample SRR10903402. Similarly, we identified one inversion/misassembly in sample SRR10971381 that was supported by only one read. The alignment coordinates of all rearrangement-supporting contigs with respect to the reference strain are presented in (<strong><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a></strong>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recombination events in S gene. Samples SRR10903401 (A) and SRR10903402 (B). Alignments of the de novo assembled contigs with respect to the reference genome (MN 975262). Donor &#x2013; acceptor palindrome sequences are indicated in green bars. Raw, non-duplicated NGS reads, validating the recombination event, are represented below the corresponding contig. (C): Prediction of the secondary structure of the genomic region spanning the rearrangement breakpoint (100 bases upstream and 100 bases downstream). The corresponding donor-acceptor sequences, exposed in internal loops, are indicated in green bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1942535232" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Recombination events in S gene. Samples SRR10903401 (A) and SRR10903402 (B). Alignments of the de novo assembled contigs with respect to the reference genome (MN 975262). Donor &#x2013; acceptor palindrome sequences are indicated in green bars. Raw, non-duplicated NGS reads, validating the recombination event, are represented below the corresponding contig. (C): Prediction of the secondary structure of the genomic region spanning the rearrangement breakpoint (100 bases upstream and 100 bases downstream). The corresponding donor-acceptor sequences, exposed in internal loops, are indicated in green bars.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F3.medium.gif" width="380" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F3.medium.gif" width="380" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.009480/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-20" class="first-child">Recombination events in S gene. Samples SRR10903401 (A) and SRR10903402 (B). Alignments of the de novo assembled contigs with respect to the reference genome (MN 975262). Donor – acceptor palindrome sequences are indicated in green bars. Raw, non-duplicated NGS reads, validating the recombination event, are represented below the corresponding contig. (C): Prediction of the secondary structure of the genomic region spanning the rearrangement breakpoint (100 bases upstream and 100 bases downstream). The corresponding donor-acceptor sequences, exposed in internal loops, are indicated in green bars.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1223154/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1223154/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1223154" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <span class="caption-title">Alignment characteristics of de novo assembled contigs</span><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-4"><h2 class="">DISCUSSION</h2><p id="p-21">The rapid spread and the death toll of the new SARS-CoV-2 epidemic warrants the immediate identification / development of effective antiviral agents and vaccines, but also the design of accurate diagnostics. The intra- and inter-patient variability of the viral genome plays a pivotal role in all the abovementioned efforts, since it affects the compatibility of molecular diagnostics but also impairs the effectiveness of the vaccines and the serological assays by altering the antigenicity of the virus. Intra-host low-frequency variants are also the main source of resistance to anti-viral drugs.</p><p id="p-22">Bioinformatics analysis of NGS data allows the generation of the consensus sequence of a viral genome from the of majority nucleotides at each position but also the identification of non-consensus nucleotides, enabling the exploration of intra-host variability but also its consequences on intra-host viral evolution [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]–[<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. All samples analysed in this study were probably infected by the same viral strain since they shared the same set of consensus SNVs. However, apart from 3 intra-host SNVs that were common between SRR10903401 and SRR10903402, there was no other overlap observed between the low frequency variants of each sample (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1-B</a></strong>). This indicates that these variations have been occurred in a rather random fashion and are not subject of selective pressures, which is also supported by the fact that the missense mutations were systematically more, compared to the synonymous mutations. On the other hand, missense substitutions are more common in loci involving pathogen resistance, indicating positive selection [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. The analysed viral RNA might have been originated from functional/packed virions, but also from unpacked viral genomes, which are unable to replicate and infect other host cells. Even if a viral genome is unable to replicate independently, its abundant presence in the pool of viral quasispecies implies some functionality regarding the intra-host evolution and adaptation. For example, defective viral genomes might affect infection dynamics such as viral persistence but also the natural history of an infection [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]–[<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. At the same time, these variants may arise rapidly during an outbreak and can be used for tracking the transmission chains and the spaciotemporal characteristics of the epidemic [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]–[<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. Studies involving large number of samples and in-vitro experiments on SARS-CoV-2 viral isolates are needed, in order to conclude whether these variations are advantageous or come with a fitness cost for the virus.</p><p id="p-23">SNVs and quasispecies that are observed at low frequency could represent viral variations of low impact on the functionality of the genome. However, their abundance is largely affected by the population size and the epidemic characteristics. For example, a neutral substitution in a region that represents a primer target for a molecular diagnostic assay can drift to fixation rather quickly in a rapidly spreading virus, jeopardizing the sensitivity of the assay [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>], [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. Here, we highlight three intra-host but also two fixed variants that colocalized with primers or probes of real-time PCR diagnostics assays that are currently in use (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a></strong>). Since the alignment of these oligos with their genomic targets is directly linked to the performance of the corresponding diagnostic assays, the community should pay extra attention in the evaluation of these potentially emerging variations and be alerted, in case redesigning of these oligos is needed.</p><p id="p-24">As it is well documented, recombination events lead to substantial changes in genetic diversity of RNA viruses [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>], [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. In CoVs, discontinuous RNA synthesis is commonly observed, resulting in high frequencies of homologous recombination [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>], which can be up to 25% across the entire CoV genome [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. For pathogenic HCoVs genomic rearrangements are frequently reported during the course of epidemic outbreaks, such as HCoV-OC43 [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>], and HCoV-NL63 [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], SARS-CoV [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>] [<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>] and MERS-CoV [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. We have isolated intra-host genomic rearrangements, located in poly-A and poly-U enriched palindrome regions across the SARS-CoV-2 genome (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, Suppl. Figure 1</strong>). We have validated the majority of these events by visual inspection of the alignments. We conclude that these rearrangements do not represent artifacts derived from the NGS library preparation (e.g. PCR crosstalk artifacts), especially since all the supporting reads were not duplicated and, in some cases, differed in polymorphic positions (<strong>Suppl. Figure 2</strong>).</p><p id="p-25">Recombination processes involving S gene particularly, have been reported for SARS- and SARS-like CoV but also for HCoV-OC43. In the case of sister species HCoV-NL63 and HCoV-229E, recombination breakpoints are located near 3’- and 5’-end of the gene [<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>] [<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>]. S is a trimeric protein, which is cleaved into two subunits, the globular N-terminal S1 and the C-terminal S2 [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. The S1 subunit consists of a signal peptide and the NT and receptor binding (RB) domains, with the latter sharing only 40% amino acid identity with other SARS-related CoVs. Our analysis revealed that similarly to other genomic regions, the S1 subunit hosts many low-frequency SNVs, characterized by higher density compared to the rest of the S gene sequence (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1-E</a></strong>). The S2 subunit is highly conserved, with 99% identity compared to human SARS-CoV and two bat SARS-like CoVs [<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>]. The S2 subunit consists of two fusion peptides (FP, IFP), followed by two heptad repeats (HR 1 and 2), the pretransmembrane domain (PTM), the transmembrane and the cytoplasmic domain (TM, CP) [<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>]. In S gene, the same rearrangement event has taken place in two samples analyzed in this study. This observation highlights a potential recombination hot-spot in S gene. The rearrangement that was common between the two samples of this study is located in nt24,000 of the 2019-nCoV genome, which corresponds to the ~200nt linking region between the fusion peptides FP and IFP (aa 812-813). Examining closely the secondary structure of the RNA genome around the breakpoints, we suggest a model where the palindromes 5’-UGGUUUU-3’ and 5’-AAAACCAA-3’, have served as donor-acceptor sequences during the recombination event, since they are both exposed in the single-stranded internal loops formed in a highly structured RNA pseudoknot (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3-C</a></strong>). The RB domain of the S protein has been tested as a potential immunogen as it contains neutralization epitopes which appear to have a role in the induction of neutralizing antibodies [<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>], [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. It should be mentioned though that S protein of SARS-CoV is the most divergent in all strains infecting humans [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>], [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>], as in both C and N-terminal domains variations arise rapidly, allowing immunological escape [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]. Our findings support that apart from these variations, the N-terminal region also hosts a recombination hot-spot, which together with the rest of the observed rearrangements, indicates the genomic instability of SARS-CoV-2 over poly-A and poly-U regions.</p></div><div class="section" id="sec-6"><h2 class="">Authors’ contributions</h2><p id="p-26">TK: Conceptualization; Data curation; Formal analysis; Methodology; Supervision; Validation; Visualization; Writing - original draft; Writing - review &amp; editing. GP: Data curation; Formal analysis; Writing - original draft; Writing - review &amp; editing. MB: Visualization; Writing - review &amp; editing. AM, ST and AM: Writing - original draft; Writing - review &amp; editing.</p></div><div class="section" id="sec-7"><h2 class="">Ethics approval and consent to participate</h2><p id="p-27">Not applicable.</p></div><div class="section COI-statement" id="sec-8"><h2 class="">Declarations of interest</h2><p id="p-28">None.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.1" data-doi="10.1016/bs.aivir.2018.01.001"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Corman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Muth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Niemeyer</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, “<abbr class="cit-jnl-abbrev">Hosts and Sources of Endemic Human Coronaviruses</abbr>,” vol. <span class="cit-vol">100</span>, pp. <span class="cit-fpage">163</span>–<span class="cit-lpage">188</span>, <span class="cit-pub-date">2018</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/bs.aivir.2018.01.001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHosts%2Band%2BSources%2Bof%2BEndemic%2BHuman%2BCoronaviruses%26rft.volume%253D100%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2018.01.001%26rft_id%253Dinfo%253Apmid%252F29551135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2018.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29551135&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.2" data-doi="10.1007/978-1-4939-2438-7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Fehr</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Perlman</span></span>, “<abbr class="cit-jnl-abbrev">HHS Public Access</abbr>,” pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>, <span class="cit-pub-date">2016</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-4939-2438-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHHS%2BPublic%2BAccess%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-2438-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-2438-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.3" data-doi="10.1128/JVI.02605-06"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Vijaykrishna</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. J. D.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. X.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S. M.</span> <span class="cit-name-surname">Peiris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guan</span></span>, “<span class="cit-article-title">Evolutionary Insights into the Ecology of Coronaviruses</span>,” <abbr class="cit-jnl-abbrev">J. Virol.</abbr>, vol. <span class="cit-vol">81</span>, no. <span class="cit-issue">8</span>, pp. <span class="cit-fpage">4012</span>–<span class="cit-lpage">4020</span>, <span class="cit-pub-date">2007</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.02605-06</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02605-06%26rft_id%253Dinfo%253Apmid%252F17267506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzgvNDAxMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjcuMDA5NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.4" data-doi="10.1007/82"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. I.</span> <span class="cit-name-surname">Paules</span></span>, “<abbr class="cit-jnl-abbrev">Coronavirus Infections - More Than Just the Common Cold</abbr>,” vol. <span class="cit-vol">2520</span>, pp. <span class="cit-fpage">3</span>–<span class="cit-lpage">4</span>, <span class="cit-pub-date">2020</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1007/82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCoronavirus%2BInfections%2B-%2BMore%2BThan%2BJust%2Bthe%2BCommon%2BCold%26rft_id%253Dinfo%253Adoi%252F10.1007%252F82%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/82&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.5" data-doi="10.1126/science.1116480"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Farzan</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Harrison</span></span>, “<span class="cit-article-title">Structural biology: Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</span>,” <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>., vol. <span class="cit-vol">309</span>, no. <span class="cit-issue">5742</span>, pp. <span class="cit-fpage">1864</span>–<span class="cit-lpage">1868</span>, <span class="cit-pub-date">2005</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1116480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNy4wMDk0ODAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.6" data-doi="10.1038/s41579-018-0118-9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cui</span></span>, “<span class="cit-article-title">Origin and evolution of pathogenic coronaviruses</span>,” <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr>, vol. <span class="cit-vol">17</span>, no. <span class="cit-issue">March</span>, pp. <span class="cit-fpage">181</span>–<span class="cit-lpage">192</span>, <span class="cit-pub-date">2019</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41579-018-0118-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol.%26rft.volume%253D17%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41579-018-0118-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41579-018-0118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.7" data-doi="10.1126/science.1118391"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Epstein</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Bats Are Natural Reservoirs of SARS-Like Coronaviruses</span>,” <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>., vol. <span class="cit-vol">310</span>, no. <span class="cit-issue">5748</span>, pp. <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span>, <span class="cit-pub-date">2005</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1109/NEMS.2006.334722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%252880-.%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI3LjAwOTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.8"><div class="cit-metadata"><cite>“<span class="cit-article-title">Novel coronavirus 2019;-nCoV: early estimation of epidemiological parameters and epidemic predictions</span>,” vol. <span class="cit-vol">2020</span>, no. <span class="cit-issue">January</span>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.9" data-doi="10.1080/22221751.2020.1719902"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Chan</span></span> <span class="cit-etal">et al.</span>, “<abbr class="cit-jnl-abbrev">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</abbr>,” vol. <span class="cit-vol">1751</span>, <span class="cit-pub-date">2020</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1080/22221751.2020.1719902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomic%2Bcharacterization%2Bof%2Bthe%2B2019%2Bnovel%2Bhuman-pathogenic%2Bcoronavirus%2Bisolated%2Bfrom%2Ba%2Bpatient%2Bwith%2Batypical%2Bpneumonia%2Bafter%2Bvisiting%2BWuhan%26rft.volume%253D9%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2020.1719902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/22221751.2020.1719902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.10" data-doi="10.1128/JVI.01231-06"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Rapid Reversion of Sequence Polymorphisms Dominates Early Human Immunodeficiency Virus Type 1 Evolution</span>,” <abbr class="cit-jnl-abbrev">J. Virol.</abbr>, vol. <span class="cit-vol">81</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">193</span>–<span class="cit-lpage">201</span>, <span class="cit-pub-date">2007</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01231-06</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01231-06%26rft_id%253Dinfo%253Apmid%252F17065207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjgxLzEvMTkzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNy4wMDk0ODAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Beloukas</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Assessment of phylogenetic sensitivity for reconstructing HIV-1 epidemiological relationships</span>,” <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, vol. <span class="cit-vol">166</span>, no. <span class="cit-issue">1–2</span>, pp. <span class="cit-fpage">54</span>–<span class="cit-lpage">60</span>, <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DBeloukas%26rft.auinit1%253DA.%26rft.volume%253D166%26rft.issue%253D1-2%26rft.spage%253D54%26rft.epage%253D60%26rft.atitle%253DAssessment%2Bof%2Bphylogenetic%2Bsensitivity%2Bfor%2Breconstructing%2BHIV-1%2Bepidemiological%2Brelationships.%26rft_id%253Dinfo%253Apmid%252F22429620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22429620&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.12" data-doi="10.1038/nature04388"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Vignuzzi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Stone</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Arnold</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Cameron</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Andino</span></span>, “<span class="cit-article-title">Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population</span>,” <abbr class="cit-jnl-abbrev">Nature</abbr>, vol. <span class="cit-vol">439</span>, no. <span class="cit-issue">7074</span>, pp. <span class="cit-fpage">344</span>–<span class="cit-lpage">348</span>, <span class="cit-pub-date">2006</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature04388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVignuzzi%26rft.auinit1%253DM.%26rft.volume%253D439%26rft.issue%253D7074%26rft.spage%253D344%26rft.epage%253D348%26rft.atitle%253DQuasispecies%2Bdiversity%2Bdetermines%2Bpathogenesis%2Bthrough%2Bcooperative%2Binteractions%2Bin%2Ba%2Bviral%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04388%26rft_id%253Dinfo%253Apmid%252F16327776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16327776&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234682100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.13"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Karamitros</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">The interferon receptor-1 promoter polymorphisms affect the outcome of Caucasians with HB eAg-negative chronic HBV infection</span>,” <abbr class="cit-jnl-abbrev">Liver Int.</abbr>, vol. <span class="cit-vol">35</span>, no. <span class="cit-issue">12</span>, pp. <span class="cit-fpage">2506</span>–<span class="cit-lpage">2513</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLiver%2BInt.%26rft.volume%253D35%26rft.spage%253D2506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.14" data-doi="10.1073/pnas.0501254102"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Malley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Trzcinski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Srivastava</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Thompson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Anderson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lipsitch</span></span>, “<span class="cit-article-title">CD4+ T cells mediate antibody-independent acquired immunity to pneumococcal colonization</span>,” <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>., vol. <span class="cit-vol">102</span>, no. <span class="cit-issue">13</span>, pp. <span class="cit-fpage">4848</span>–<span class="cit-lpage">4853</span>, <span class="cit-pub-date">2005</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0501254102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0501254102%26rft_id%253Dinfo%253Apmid%252F15781870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzEzLzQ4NDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI3LjAwOTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.15" data-doi="10.1098/rsif.2007.0229"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lipsitch</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">O’Hagan</span></span>, “<span class="cit-article-title">Patterns of antigenic diversity and the mechanisms that maintain them</span>,” <abbr class="cit-jnl-abbrev">J. R. Soc. Interface</abbr>, vol. <span class="cit-vol">4</span>, no. <span class="cit-issue">16</span>, pp. <span class="cit-fpage">787</span>–<span class="cit-lpage">802</span>, <span class="cit-pub-date">2007</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1098/rsif.2007.0229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BThe%2BRoyal%2BSociety%2BInterface%26rft.stitle%253DJ%2BR%2BSoc%2BInterface%26rft.aulast%253DLipsitch%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D16%26rft.spage%253D787%26rft.epage%253D802%26rft.atitle%253DPatterns%2Bof%2Bantigenic%2Bdiversity%2Band%2Bthe%2Bmechanisms%2Bthat%2Bmaintain%2Bthem%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsif.2007.0229%26rft_id%253Dinfo%253Apmid%252F17426010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsif.2007.0229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17426010&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249422100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.16" data-doi="10.3389/fmicb.2019.01347"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. Y.</span> <span class="cit-name-surname">Yong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. K.</span> <span class="cit-name-surname">Ong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Yeap</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Ho</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Tan</span></span>, “<span class="cit-article-title">Recent Advances in the Vaccine Development Against Middle East Respiratory Syndrome-Coronavirus</span>,” <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr>, vol. <span class="cit-vol">10</span>, no. <span class="cit-issue">August</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>, <span class="cit-pub-date">2019</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3389/fmicb.2019.01781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2019.01347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2019.01347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.17" data-doi="10.1016/j.physbeh.2017.03.040"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Nicolas</span></span> <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Cortes-Penfield</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Barbara W.</span> <span class="cit-name-surname">Trautner</span></span>, “<span class="cit-article-title">A decade after SARS: Strategies to control emerging coronaviruses</span>,” <abbr class="cit-jnl-abbrev">Physiol. Behav.</abbr>, vol. <span class="cit-vol">176</span>, no. <span class="cit-issue">5</span>, pp. <span class="cit-fpage">139</span>–<span class="cit-lpage">148</span>, <span class="cit-pub-date">2017</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.physbeh.2017.03.040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BBehav.%26rft.volume%253D176%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.physbeh.2017.03.040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.physbeh.2017.03.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.18" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Langmead</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span>, “<span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>,” <abbr class="cit-jnl-abbrev">Nat. Methods</abbr>, vol. <span class="cit-vol">9</span>, no. <span class="cit-issue">4</span>, pp. <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>, <span class="cit-month">Apr</span>. <span class="cit-pub-date">2012</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.1923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.19" data-doi="10.1158/0008-5472.CAN-17-0337"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Robinson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">ThorvaldsdÃóttir</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Wenger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zehir</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mesirov</span></span>, “<span class="cit-article-title">Variant review with the integrative genomics viewer</span>,” <abbr class="cit-jnl-abbrev">Cancer Research</abbr>, vol. <span class="cit-vol">77</span>, no. <span class="cit-issue">21</span>. <span class="cit-auth cit-collab">American Association for Cancer Research Inc</span>., pp. <span class="cit-fpage">e31</span>-<span class="cit-lpage">e34</span>, <span class="cit-day">01</span>-<span class="cit-month">Nov</span>-<span class="cit-pub-date">2017</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1158/0008-5472.CAN-17-0337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D77%26rft.issue%253D21%26rft.spage%253De31%26rft.epage%253De34%26rft.atitle%253DVariant%2BReview%2Bwith%2Bthe%2BIntegrative%2BGenomics%2BViewer%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-17-0337%26rft_id%253Dinfo%253Apmid%252F29092934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjk6Ijc3LzIxL2UzMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjcuMDA5NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.20" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>,” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. <span class="cit-vol">25</span>, no. <span class="cit-issue">16</span>, pp. <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>, <span class="cit-month">Aug</span>. <span class="cit-pub-date">2009</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btp352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.21" data-doi="10.1093/nar/gks918"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wilm</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">LoFreq: A sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets</span>,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, vol. <span class="cit-vol">40</span>, no. <span class="cit-issue">22</span>, pp. <span class="cit-fpage">11189</span>–<span class="cit-lpage">11201</span>, <span class="cit-month">Dec</span>. <span class="cit-pub-date">2012</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gks918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks918%26rft_id%253Dinfo%253Apmid%252F23066108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks918&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23066108&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313414800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.22" data-doi="10.4161/fly.19695"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Cingolani</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</span>,” <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr>., vol. <span class="cit-vol">6</span>, no. <span class="cit-issue">2</span>, pp. <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>, <span class="cit-pub-date">2012</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.4161/fly.19695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFly%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D80%26rft.epage%253D92%26rft.atitle%253DA%2Bprogram%2Bfor%2Bannotating%2Band%2Bpredicting%2Bthe%2Beffects%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%252C%2BSnpEff%253A%2BSNPs%2Bin%2Bthe%2Bgenome%2Bof%2BDrosophila%2Bmelanogaster%2Bstrain%2Bw1118%253B%2Biso-2%253B%2Biso-3.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305965500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.23" data-doi="10.3389/fgene.2012.00035"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Cingolani</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Using Drosophila melanogaster as a model for genotoxic chemical mutational studies with a new program, SnpSift</span>,” <abbr class="cit-jnl-abbrev">Front. Genet.</abbr>, vol. <span class="cit-vol">3</span>, no. <span class="cit-issue">MAR</span>, <span class="cit-pub-date">2012</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3389/fgene.2012.00035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DUsing%2BDrosophila%2Bmelanogaster%2Bas%2Ba%2BModel%2Bfor%2BGenotoxic%2BChemical%2BMutational%2BStudies%2Bwith%2Ba%2BNew%2BProgram%252C%2BSnpSift.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2012.00035%26rft_id%253Dinfo%253Apmid%252F22435069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2012.00035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22435069&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.24" data-doi="10.2807/1560-7917.ES.2020.25.3.2000045"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Corman</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>,” <abbr class="cit-jnl-abbrev">Euro Surveill.</abbr>, vol. <span class="cit-vol">25</span>, no. <span class="cit-issue">3</span>, <span class="cit-month">Jan</span>. <span class="cit-pub-date">2020</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.2807/1560-7917.ES.2020.25.3.2000045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuro%2BSurveill.%26rft_id%253Dinfo%253Adoi%252F10.2807%252F1560-7917.ES.2020.25.3.2000045%26rft_id%253Dinfo%253Apmid%252F31992387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2807/1560-7917.ES.2020.25.3.2000045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31992387&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.25" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bankevich</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">SPAdes: A new genome assembly algorithm and its applications to single-cell sequencing</span>,” <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr>, vol. <span class="cit-vol">19</span>, no. <span class="cit-issue">5</span>, pp. <span class="cit-fpage">455</span>–<span class="cit-lpage">477</span>, <span class="cit-month">May</span> <span class="cit-pub-date">2012</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1089/cmb.2012.0021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.26" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Altschul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gish</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Myers</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Lipman</span></span>, “<span class="cit-article-title">Basic local alignment search tool</span>,” <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, vol. <span class="cit-vol">215</span>, no. <span class="cit-issue">3</span>, pp. <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>, <span class="cit-month">Oct</span>. <span class="cit-pub-date">1990</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0022-2836(05)80360-2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.27" data-doi="10.1093/nar/gkn188"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Gruber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lorenz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Bernhart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Neuböck</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">I. L.</span> <span class="cit-name-surname">Hofacker</span></span>, “<span class="cit-article-title">The Vienna RNA websuite</span>,” <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, vol. <span class="cit-vol">36</span>, no. <span class="cit-issue">Web Server</span> issue, p. <span class="cit-fpage">W70</span>, <span class="cit-pub-date">2008</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkn188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn188%26rft_id%253Dinfo%253Apmid%252F18424795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18424795&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258142300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.28" data-doi="10.3201/eid1905.130057"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cotten</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Full-genome deep sequencing and phylogenetic analysis of novel human betacoronavirus</span>,” <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis.</abbr>, vol. <span class="cit-vol">19</span>, no. <span class="cit-issue">5</span>, pp. <span class="cit-fpage">736</span>–<span class="cit-lpage">742</span>, <span class="cit-month">May</span> <span class="cit-pub-date">2013</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid1905.130057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg.%2BInfect.%2BDis.%26rft.volume%253D19%26rft.spage%253D736%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1905.130057%26rft_id%253Dinfo%253Apmid%252F23693015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1905.130057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23693015&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[29].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.27.009480v1.29" data-doi="10.1371/journal.ppat.1000557"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Haagmans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Andeweg</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. D. M. E.</span> <span class="cit-name-surname">Osterhaus</span></span>, “<span class="cit-article-title">The Application of Genomics to Emerging Zoonotic Viral Diseases</span>,” <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr>, vol. <span class="cit-vol">5</span>, no. <span class="cit-issue">10</span>, p. <span class="cit-fpage">e1000557</span>, <span class="cit-month">Oct</span>. <span class="cit-pub-date">2009</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1000557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DHaagmans%26rft.auinit1%253DB.%2BL.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000557%26rft.epage%253De1000557%26rft.atitle%253DThe%2Bapplication%2Bof%2Bgenomics%2Bto%2Bemerging%2Bzoonotic%2Bviral%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000557%26rft_id%253Dinfo%253Apmid%252F19855817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19855817&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.-G.</span> <span class="cit-name-surname">Kostaki</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Unravelling the history of hepatitis B virus genotypes A and D infection using a full-genome phylogenetic and phylogeographic approach</span>,” <abbr class="cit-jnl-abbrev">Elife</abbr>, vol. <span class="cit-vol">7</span>, p. <span class="cit-fpage">e36709</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D7%26rft.spage%253De36709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.27.009480v1.31"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. N.</span> <span class="cit-name-surname">Cooper</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">N. P.</span> <span class="cit-name-surname">Group</span></span>., <span class="cit-source">Nature encyclopedia of the human genome.</span> <span class="cit-publ-loc">London; New York</span>: <span class="cit-publ-name">Nature Pub. Group</span>, <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.32" data-doi="10.1126/science.1115030"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Aaskov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Buzacott</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Thu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lowry</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Holmes</span></span>, “<span class="cit-article-title">Long-term transmission of defective RNA viruses in humans and Aedes mosquitoes</span>,” <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>., vol. <span class="cit-vol">311</span>, no. <span class="cit-issue">5758</span>, pp. <span class="cit-fpage">236</span>–<span class="cit-lpage">238</span>, <span class="cit-month">Jan</span>. <span class="cit-pub-date">2006</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1115030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAaskov%26rft.auinit1%253DJ.%26rft.volume%253D311%26rft.issue%253D5758%26rft.spage%253D236%26rft.epage%253D238%26rft.atitle%253DLong-Term%2BTransmission%2Bof%2BDefective%2BRNA%2BViruses%2Bin%2BHumans%2Band%2BAedes%2BMosquitoes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1115030%26rft_id%253Dinfo%253Apmid%252F16410525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTEvNTc1OC8yMzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI3LjAwOTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[33].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.27.009480v1.33" data-doi="10.1126/science.288.5464.339"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Farci</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">The outcome of acute hepatitis C predicted by the evolution of the viral quasispecies</span>,” <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>., vol. <span class="cit-vol">288</span>, no. <span class="cit-issue">5464</span>, pp. <span class="cit-fpage">339</span>–<span class="cit-lpage">344</span>, <span class="cit-month">Apr</span>. <span class="cit-pub-date">2000</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.288.5464.339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFarci%26rft.auinit1%253DP.%26rft.volume%253D288%26rft.issue%253D5464%26rft.spage%253D339%26rft.epage%253D344%26rft.atitle%253DThe%2BOutcome%2Bof%2BAcute%2BHepatitis%2BC%2BPredicted%2Bby%2Bthe%2BEvolution%2Bof%2Bthe%2BViral%2BQuasispecies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5464.339%26rft_id%253Dinfo%253Apmid%252F10764648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvNTQ2NC8zMzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI3LjAwOTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.34" data-doi="10.3389/fmicb.2018.02942"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Karamichali</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">HCV defective genomes promote persistent infection by modulating the viral life cycle</span>,” <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr>, vol. <span class="cit-vol">9</span>, p. <span class="cit-fpage">2942</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D9%26rft.spage%253D2942%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2018.02942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2018.02942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.35"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Magiorkinis</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">HIV-1 epidemic in Russia: an evolutionary epidemiology analysis</span>,” <abbr class="cit-jnl-abbrev">Lancet</abbr>, vol. <span class="cit-vol">383</span>, p. <span class="cit-fpage">S71</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D383%26rft.spage%253DS71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[36].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.27.009480v1.36"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Paraskevis</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Molecular investigation of HIV-1 cross-group transmissions during an outbreak among people who inject drugs (2011--2014) in Athens, Greece</span>,” <abbr class="cit-jnl-abbrev">Infect. Genet. Evol.</abbr>, vol. <span class="cit-vol">62</span>, pp. <span class="cit-fpage">11</span>–<span class="cit-lpage">16</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol.%26rft.volume%253D62%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.37"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Magiorkinis</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">An innovative study design to assess the community effect of interventions to mitigate HIV epidemics using transmission-chain phylodynamics</span>,” <abbr class="cit-jnl-abbrev">Am. J. Epidemiol.</abbr>, vol. <span class="cit-vol">187</span>, no. <span class="cit-issue">12</span>, pp. <span class="cit-fpage">2615</span>–<span class="cit-lpage">2622</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BEpidemiol.%26rft.volume%253D187%26rft.spage%253D2615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.38" data-doi="10.1007/s00705-017-3281-9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. Y.</span> <span class="cit-name-surname">Noh</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Simultaneous detection of severe acute respiratory syndrome, Middle East respiratory syndrome, and related bat coronaviruses by real-time reverse transcription PCR</span>,” <abbr class="cit-jnl-abbrev">Arch. Virol.</abbr>, vol. <span class="cit-vol">162</span>, no. <span class="cit-issue">6</span>, pp. <span class="cit-fpage">1617</span>–<span class="cit-lpage">1623</span>, <span class="cit-month">Jun</span>. <span class="cit-pub-date">2017</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00705-017-3281-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch.%2BVirol.%26rft.volume%253D162%26rft.spage%253D1617%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-017-3281-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-017-3281-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.39"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Karamitros</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">A contaminant-free assessment of Endogenous Retroviral RNA in human plasma</span>,” <abbr class="cit-jnl-abbrev">Nat. Sci. Reports</abbr>, vol. <span class="cit-vol">6</span>, p. <span class="cit-fpage">33598</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BSci.%2BReports%26rft.volume%253D6%26rft.spage%253D33598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.40"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Holmes</span></span>, <abbr class="cit-jnl-abbrev">The evolution and emergence of RNA viruses</abbr>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-pub-date">2009</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.41"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.-G.</span> <span class="cit-name-surname">Kostaki</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Spatiotemporal characteristics of the HIV-1 CRF02_AG/CRF63_02A1 epidemic in Russia and Central Asia</span>,” <abbr class="cit-jnl-abbrev">AIDS Res. Hum. Retroviruses</abbr>, vol. <span class="cit-vol">34</span>, no. <span class="cit-issue">5</span>, pp. <span class="cit-fpage">415</span>–<span class="cit-lpage">420</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2BRes.%2BHum.%2BRetroviruses%26rft.volume%253D34%26rft.spage%253D415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.42" data-doi="10.1128/mmbr.56.1.61-79.1992"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. M. C.</span> <span class="cit-name-surname">Lai</span></span>, “<span class="cit-article-title">RNA recombination in animal and plant viruses</span>,” <abbr class="cit-jnl-abbrev">Microbiological Reviews</abbr>, vol. <span class="cit-vol">56</span>, no. <span class="cit-issue">1</span>. pp. <span class="cit-fpage">61</span>–<span class="cit-lpage">79</span>, <span class="cit-pub-date">1992</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/mmbr.56.1.61-79.1992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DLai%26rft.auinit1%253DM%2BM%26rft.volume%253D56%26rft.issue%253D1%26rft.spage%253D61%26rft.epage%253D79%26rft.atitle%253DRNA%2Brecombination%2Bin%2Banimal%2Band%2Bplant%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fmmbr.56.1.61-79.1992%26rft_id%253Dinfo%253Apmid%252F1579113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo3OiI1Ni8xLzYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yNy4wMDk0ODAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.43" data-doi="10.1016/0042-6822(90)90530-5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Baric</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Fu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Schaad</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Stohlman</span></span>, “<span class="cit-article-title">Establishing a genetic recombination map for murine coronavirus strain A59 complementation groups</span>,” <abbr class="cit-jnl-abbrev">Virology</abbr>, vol. <span class="cit-vol">177</span>, no. <span class="cit-issue">2</span>, pp. <span class="cit-fpage">646</span>–<span class="cit-lpage">656</span>, <span class="cit-month">Aug</span>. <span class="cit-pub-date">1990</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/0042-6822(90)90530-5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBaric%26rft.auinit1%253DR.%2BS.%26rft.volume%253D177%26rft.issue%253D2%26rft.spage%253D646%26rft.epage%253D656%26rft.atitle%253DEstablishing%2Ba%2Bgenetic%2Brecombination%2Bmap%2Bfor%2Bmurine%2Bcoronavirus%2Bstrain%2BA59%2Bcomplementation%2Bgroups.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252890%252990530-5%26rft_id%253Dinfo%253Apmid%252F2164728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(90)90530-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2164728&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.44" data-doi="10.1128/JVI.05512-11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. K. P.</span> <span class="cit-name-surname">Lau</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Molecular Epidemiology of Human Coronavirus OC43 Reveals Evolution of Different Genotypes over Time and Recent Emergence of a Novel Genotype due to Natural Recombination</span>,” <abbr class="cit-jnl-abbrev">J. Virol.</abbr>, vol. <span class="cit-vol">85</span>, no. <span class="cit-issue">21</span>, pp. <span class="cit-fpage">11325</span>–<span class="cit-lpage">11337</span>, <span class="cit-month">Nov</span>. <span class="cit-pub-date">2011</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.05512-11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05512-11%26rft_id%253Dinfo%253Apmid%252F21849456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8yMS8xMTMyNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjcuMDA5NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.45" data-doi="10.1016/j.jmb.2006.09.074"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Pyrc</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Mosaic Structure of Human Coronavirus NL63, One Thousand Years of Evolution</span>,” <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, vol. <span class="cit-vol">364</span>, no. <span class="cit-issue">5</span>, pp. <span class="cit-fpage">964</span>–<span class="cit-lpage">973</span>, <span class="cit-month">Dec</span>. <span class="cit-pub-date">2006</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2006.09.074</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPyrc%26rft.auinit1%253DK.%26rft.volume%253D364%26rft.issue%253D5%26rft.spage%253D964%26rft.epage%253D973%26rft.atitle%253DMosaic%2Bstructure%2Bof%2Bhuman%2Bcoronavirus%2BNL63%252C%2Bone%2Bthousand%2Byears%2Bof%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2006.09.074%26rft_id%253Dinfo%253Apmid%252F17054987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2006.09.074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17054987&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.46" data-doi="10.1128/JVI.01926-07"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.-C.</span> <span class="cit-name-surname">Hon</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Evidence of the Recombinant Origin of a Bat Severe Acute Respiratory Syndrome (SARS)-Like Coronavirus and Its Implications on the Direct Ancestor of SARS Coronavirus</span>,” <abbr class="cit-jnl-abbrev">J. Virol.</abbr>, vol. <span class="cit-vol">82</span>, no. <span class="cit-issue">4</span>, pp. <span class="cit-fpage">1819</span>–<span class="cit-lpage">1826</span>, <span class="cit-month">Feb</span>. <span class="cit-pub-date">2008</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01926-07</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01926-07%26rft_id%253Dinfo%253Apmid%252F18057240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzQvMTgxOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjcuMDA5NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.47" data-doi="10.1126/science.aac8608"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. S. M.</span> <span class="cit-name-surname">Sabir</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Co-circulation of three camel coronavirus species and recombination of MERS-CoVs in Saudi Arabia</span>,” <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>., vol. <span class="cit-vol">351</span>, no. <span class="cit-issue">6268</span>, pp. <span class="cit-fpage">81</span>–<span class="cit-lpage">84</span>, <span class="cit-month">Jan</span>. <span class="cit-pub-date">2016</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aac8608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%252880-.%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac8608%26rft_id%253Dinfo%253Apmid%252F26678874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTEvNjI2OC84MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjcuMDA5NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[48].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.48" data-doi="10.1016/j.tim.2015.06.003"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Wang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G. F.</span> <span class="cit-name-surname">Gao</span></span>, “<span class="cit-article-title">Bat-to-human: Spike features determining ‘host jump’ of coronaviruses SARS-CoV, MERS-CoV, and beyond</span>,” <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr>, vol. <span class="cit-vol">23</span>, no. <span class="cit-issue">8</span>, pp. <span class="cit-fpage">468</span>–<span class="cit-lpage">478</span>, <span class="cit-pub-date">2015</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tim.2015.06.003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol.%26rft.volume%253D23%26rft.spage%253D468%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2015.06.003%26rft_id%253Dinfo%253Apmid%252F26206723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2015.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26206723&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.49" data-doi="10.1093/infdis/jit609"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Agnihothram</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Evaluation of serologic and antigenic relationships between middle eastern respiratory syndrome coronavirus and other coronaviruses to develop vaccine platforms for the rapid response to emerging coronaviruses</span>,” <abbr class="cit-jnl-abbrev">J. Infect. Dis.</abbr>, vol. <span class="cit-vol">209</span>, no. <span class="cit-issue">7</span>, pp. <span class="cit-fpage">995</span>–<span class="cit-lpage">1006</span>, <span class="cit-pub-date">2014</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/infdis/jit609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjit609%26rft_id%253Dinfo%253Apmid%252F24253287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jit609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24253287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.50" data-doi="10.1073/pnas.0401939101"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bisht</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">Severe acute respiratory syndrome coronavirus spike protein expressed by attenuated vaccinia virus protectively immunizes mice</span>,” <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>., vol. <span class="cit-vol">101</span>, no. <span class="cit-issue">17</span>, pp. <span class="cit-fpage">6641</span>–<span class="cit-lpage">6646</span>, <span class="cit-month">Apr</span>. <span class="cit-pub-date">2004</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0401939101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401939101%26rft_id%253Dinfo%253Apmid%252F15096611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE3LzY2NDEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjI3LjAwOTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[51].</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.51" data-doi="10.1016/j.virusres.2007.01.021"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Enjuanes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">DeDiego</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ãlvarez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Deming</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sheahan</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Baric</span></span>, “<span class="cit-article-title">Vaccines to prevent severe acute respiratory syndrome coronavirus-induced disease</span>,” <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, vol. <span class="cit-vol">133</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">45</span>–<span class="cit-lpage">62</span>, <span class="cit-month">Apr</span>. <span class="cit-pub-date">2008</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2007.01.021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DEnjuanes%26rft.auinit1%253DL.%26rft.volume%253D133%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D62%26rft.atitle%253DVaccines%2Bto%2Bprevent%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus-induced%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2007.01.021%26rft_id%253Dinfo%253Apmid%252F17416434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2007.01.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17416434&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.009480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[52].</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.27.009480v1.52" data-doi="10.1016/j.vaccine.2016.11.064"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Jiaming</span></span> <span class="cit-etal">et al.</span>, “<span class="cit-article-title">The recombinant N-terminal domain of spike proteins is a potential vaccine against Middle East respiratory syndrome coronavirus (MERS-CoV) infection</span>,” <abbr class="cit-jnl-abbrev">Vaccine</abbr>, vol. <span class="cit-vol">35</span>, no. <span class="cit-issue">1</span>, pp. <span class="cit-fpage">10</span>–<span class="cit-lpage">18</span>, <span class="cit-month">Jan</span>. <span class="cit-pub-date">2017</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.vaccine.2016.11.064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D35%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2016.11.064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2016.11.064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.27.009480v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.27.009480v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.27.012013v1" title="SARS-CoV-2 and ORF3a: Non-Synonymous Mutations and Polyproline Regions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.25.008904v1" title="Modeling of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Proteins by Machine Learning and Physics-Based Refinement" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 28, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.27.009480v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.27.009480v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.27.009480v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.27.009480v1" class="active" data-icon-position="" data-hide-link-title="0">SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1215890" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9e5XhJl8MTDN79e3aRdD6QDUgSozks6nQKZEBUJf9lw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215890" id="node1215890--2" data-pisa="biorxiv;2020.03.27.009480v1" data-pisa-master="biorxiv;2020.03.27.009480" data-seqnum="1215890" data-apath="/biorxiv/early/2020/03/28/2020.03.27.009480.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Timokratis</span> <span class="nlm-surname">Karamitros</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gethsimani</span> <span class="nlm-surname">Papadopoulou</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Bousali</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Anastasios</span> <span class="nlm-surname">Mexias</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Sotiris</span> <span class="nlm-surname">Tsiodras</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Mentis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.27.009480; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.27.009480 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.27.009480v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;title=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;title=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;text=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;title=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;t=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;title=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;title=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215890" id="node1215890--3" data-pisa="biorxiv;2020.03.27.009480v1" data-pisa-master="biorxiv;2020.03.27.009480" data-seqnum="1215890" data-apath="/biorxiv/early/2020/03/28/2020.03.27.009480.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">SARS-CoV-2 exhibits intra-host genomic plasticity and low-frequency polymorphic quasispecies</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Timokratis</span> <span class="nlm-surname">Karamitros</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gethsimani</span> <span class="nlm-surname">Papadopoulou</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Bousali</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Anastasios</span> <span class="nlm-surname">Mexias</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Sotiris</span> <span class="nlm-surname">Tsiodras</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Mentis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.27.009480; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.27.009480 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Karamitros%2C+Timokratis&amp;rft.au=Papadopoulou%2C+Gethsimani&amp;rft.au=Bousali%2C+Maria&amp;rft.au=Mexias%2C+Anastasios&amp;rft.au=Tsiodras%2C+Sotiris&amp;rft.au=Mentis%2C+Andreas"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1215890/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1215890/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1215890/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1215890/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1215890/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1215890/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1215890/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1215890/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1215890/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1215890/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1215890/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1215890/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;count=horizontal&amp;via=&amp;text=SARS-CoV-2%20exhibits%20intra-host%20genomic%20plasticity%20and%20low-frequency%20polymorphic%20quasispecies&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.27.009480v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.27.009480v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
