<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/> 
  <meta name="type" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/10/2020.04.03.024075.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.03.024075v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Cells and gene expression programs in the adult human heart"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.03.024075"/>
  <meta name="DC.Date" content="2020-04-10"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="Â© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Cardiovascular disease is the leading cause of death worldwide. Advanced insights into disease mechanisms and strategies to improve therapeutic opportunities require deeper understanding of the molecular processes of the normal heart. Knowledge of the full repertoire of cardiac cells and their gene expression profiles is a fundamental first step in this endeavor. Here, using large-scale single cell and nuclei transcriptomic profiling together with state-of-the-art analytical techniques, we characterise the adult human heart cellular landscape covering six anatomical cardiac regions (left and right atria and ventricles, apex and interventricular septum). Our results highlight the cellular heterogeneity of cardiomyocytes, pericytes and fibroblasts, revealing distinct subsets in the atria and ventricles indicative of diverse developmental origins and specialized properties. Further we define the complexity of the cardiac vascular network which includes clusters of arterial, capillary, venous, lymphatic endothelial cells and an atrial-enriched population. By comparing cardiac cells to skeletal muscle and kidney, we identify cardiac tissue resident macrophage subsets with transcriptional signatures indicative of both inflammatory and reparative phenotypes. Further, inference of cell-cell interactions highlight a macrophage-fibroblast-cardiomyocyte network that differs between atria and ventricles, and compared to skeletal muscle. We expect this reference human cardiac cell atlas to advance mechanistic studies of heart homeostasis and disease. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Monika Litvinukova"/>
  <meta name="DC.Contributor" content="Carlos Talavera-Lopez"/>
  <meta name="DC.Contributor" content="Henrike Maatz"/>
  <meta name="DC.Contributor" content="Daniel Reichart"/>
  <meta name="DC.Contributor" content="Catherine L. Worth"/>
  <meta name="DC.Contributor" content="Eric L. Lindberg"/>
  <meta name="DC.Contributor" content="Masatoshi Kanda"/>
  <meta name="DC.Contributor" content="Krzysztof Polanski"/>
  <meta name="DC.Contributor" content="Eirini S. Fasouli"/>
  <meta name="DC.Contributor" content="Sara Samari"/>
  <meta name="DC.Contributor" content="Kenny Roberts"/>
  <meta name="DC.Contributor" content="Elizabeth Tuck"/>
  <meta name="DC.Contributor" content="Matthias Heinig"/>
  <meta name="DC.Contributor" content="Daniel DeLaughter"/>
  <meta name="DC.Contributor" content="Barbara McDonough"/>
  <meta name="DC.Contributor" content="Hiroko Wakimoto"/>
  <meta name="DC.Contributor" content="Joshua M. Gorham"/>
  <meta name="DC.Contributor" content="Emily Nadelmann"/>
  <meta name="DC.Contributor" content="Krishnaa T. Mahbubani"/>
  <meta name="DC.Contributor" content="Kourosh Saeb-Parsy"/>
  <meta name="DC.Contributor" content="Giannino Patone"/>
  <meta name="DC.Contributor" content="Joseph J Boyle"/>
  <meta name="DC.Contributor" content="Hongbo Zhang"/>
  <meta name="DC.Contributor" content="Hao Zhang"/>
  <meta name="DC.Contributor" content="Anissa Viveiros"/>
  <meta name="DC.Contributor" content="Gavin Oudit"/>
  <meta name="DC.Contributor" content="Omer Bayraktar"/>
  <meta name="DC.Contributor" content="J. G. Seidman"/>
  <meta name="DC.Contributor" content="Christine Seidman"/>
  <meta name="DC.Contributor" content="Michela Noseda"/>
  <meta name="DC.Contributor" content="Norbert Hubner"/>
  <meta name="DC.Contributor" content="Sarah A. Teichmann"/>
  <meta name="article:published_time" content="2020-04-10"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Cells and gene expression programs in the adult human heart"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Cardiovascular disease is the leading cause of death worldwide. Advanced insights into disease mechanisms and strategies to improve therapeutic opportunities require deeper understanding of the molecular processes of the normal heart. Knowledge of the full repertoire of cardiac cells and their gene expression profiles is a fundamental first step in this endeavor. Here, using large-scale single cell and nuclei transcriptomic profiling together with state-of-the-art analytical techniques, we characterise the adult human heart cellular landscape covering six anatomical cardiac regions (left and right atria and ventricles, apex and interventricular septum). Our results highlight the cellular heterogeneity of cardiomyocytes, pericytes and fibroblasts, revealing distinct subsets in the atria and ventricles indicative of diverse developmental origins and specialized properties. Further we define the complexity of the cardiac vascular network which includes clusters of arterial, capillary, venous, lymphatic endothelial cells and an atrial-enriched population. By comparing cardiac cells to skeletal muscle and kidney, we identify cardiac tissue resident macrophage subsets with transcriptional signatures indicative of both inflammatory and reparative phenotypes. Further, inference of cell-cell interactions highlight a macrophage-fibroblast-cardiomyocyte network that differs between atria and ventricles, and compared to skeletal muscle. We expect this reference human cardiac cell atlas to advance mechanistic studies of heart homeostasis and disease.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.03.024075v2"/>
  <meta name="citation_id" content="2020.04.03.024075v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.024075v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.024075v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.024075v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.03.024075.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.03.024075"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.03.024075"/>
  <meta name="citation_author" content="Monika Litvinukova"/>
  <meta name="citation_author_institution" content="Max Delbruck Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany.;"/>
  <meta name="citation_author_email" content="monika.litvinukova@mdc-berlin.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6713-3647"/>
  <meta name="citation_author" content="Carlos Talavera-Lopez"/>
  <meta name="citation_author_institution" content="Wellcome Sanger Institute;"/>
  <meta name="citation_author_email" content="ct5@sanger.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8590-2393"/>
  <meta name="citation_author" content="Henrike Maatz"/>
  <meta name="citation_author_institution" content="Max Delbruck Center for Molecular Medicine in the Helmholtz Association (MDC);"/>
  <meta name="citation_author_email" content="h.maatz@mdc-berlin.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9232-6272"/>
  <meta name="citation_author" content="Daniel Reichart"/>
  <meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, United States.;"/>
  <meta name="citation_author_email" content="daniel_reichart@hms.harvard.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8559-5888"/>
  <meta name="citation_author" content="Catherine L. Worth"/>
  <meta name="citation_author_institution" content="Max Delbruck Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany.;"/>
  <meta name="citation_author_email" content="catherine.sargent@mdc-berlin.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1796-9606"/>
  <meta name="citation_author" content="Eric L. Lindberg"/>
  <meta name="citation_author_institution" content="Max Delbruck Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany.;"/>
  <meta name="citation_author_email" content="eric.lindberg@mdc-berlin.de"/>
  <meta name="citation_author" content="Masatoshi Kanda"/>
  <meta name="citation_author_institution" content="Max Delbruck Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany.;"/>
  <meta name="citation_author_email" content="masatoshi.kanda@mdc-berlin.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3468-3586"/>
  <meta name="citation_author" content="Krzysztof Polanski"/>
  <meta name="citation_author_institution" content="Wellcome Sanger Institute;"/>
  <meta name="citation_author_email" content="kp9@sanger.ac.uk"/>
  <meta name="citation_author" content="Eirini S. Fasouli"/>
  <meta name="citation_author_institution" content="Wellcome Sanger Institute;"/>
  <meta name="citation_author_email" content="ef5@sanger.ac.uk"/>
  <meta name="citation_author" content="Sara Samari"/>
  <meta name="citation_author_institution" content="National Heart and Lung Institute, Imperial College London, London, United Kingdom.;"/>
  <meta name="citation_author_email" content="sara.samari13@imperial.ac.uk"/>
  <meta name="citation_author" content="Kenny Roberts"/>
  <meta name="citation_author_institution" content="Wellcome Sanger Institute;"/>
  <meta name="citation_author_email" content="kr19@sanger.ac.uk"/>
  <meta name="citation_author" content="Elizabeth Tuck"/>
  <meta name="citation_author_institution" content="Wellcome Sanger Institute;"/>
  <meta name="citation_author_email" content="et2@sanger.ac.uk"/>
  <meta name="citation_author" content="Matthias Heinig"/>
  <meta name="citation_author_institution" content="Helmholtz Zentrum Munchen;"/>
  <meta name="citation_author_email" content="matthias.heinig@helmholtz-muenchen.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5612-1720"/>
  <meta name="citation_author" content="Daniel DeLaughter"/>
  <meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, United States.;"/>
  <meta name="citation_author_email" content="daniel_delaughter@hms.harvard.edu"/>
  <meta name="citation_author" content="Barbara McDonough"/>
  <meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, United States.;"/>
  <meta name="citation_author_email" content="mcdonough@genetics.med.harvard.edu"/>
  <meta name="citation_author" content="Hiroko Wakimoto"/>
  <meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, United States.;"/>
  <meta name="citation_author_email" content="hwakimoto@genetics.med.harvard.edu"/>
  <meta name="citation_author" content="Joshua M. Gorham"/>
  <meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, United States.;"/>
  <meta name="citation_author_email" content="jgorham@genetics.med.harvard.edu"/>
  <meta name="citation_author" content="Emily Nadelmann"/>
  <meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, United States.;"/>
  <meta name="citation_author_email" content="emily_nadelmann@hms.harvard.edu"/>
  <meta name="citation_author" content="Krishnaa T. Mahbubani"/>
  <meta name="citation_author_institution" content="Department of Surgery, University of Cambridge, NIHR Cambridge Biomedical Centre, and Cambridge Biorepository for Translational Medicine, Cambridge, United Kingdom.;"/>
  <meta name="citation_author_email" content="ktam2@cam.ac.uk"/>
  <meta name="citation_author" content="Kourosh Saeb-Parsy"/>
  <meta name="citation_author_institution" content="Department of Surgery, University of Cambridge, NIHR Cambridge Biomedical Centre, and Cambridge Biorepository for Translational Medicine, Cambridge, United Kingdom.;"/>
  <meta name="citation_author_email" content="ks10014@cam.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0633-3696"/>
  <meta name="citation_author" content="Giannino Patone"/>
  <meta name="citation_author_institution" content="Max Delbruck Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany.;"/>
  <meta name="citation_author_email" content="giannino.patone@mdc-berlin.de"/>
  <meta name="citation_author" content="Joseph J Boyle"/>
  <meta name="citation_author_institution" content="Imperial College London;"/>
  <meta name="citation_author_email" content="joseph.boyle@imperial.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4337-3320"/>
  <meta name="citation_author" content="Hongbo Zhang"/>
  <meta name="citation_author_institution" content="Department of Histology and Embryology of Zhongshan School of Medicine, Sun-Yat Sen University, Guangzhou, China.;"/>
  <meta name="citation_author_email" content="hz3@sanger.ac.uk"/>
  <meta name="citation_author" content="Hao Zhang"/>
  <meta name="citation_author_institution" content="Division of Cardiology Faculty of Medicine and Dentistry, University of Alberta, Edmonton, Canada.;"/>
  <meta name="citation_author_email" content="hzhang10@ualberta.ca"/>
  <meta name="citation_author" content="Anissa Viveiros"/>
  <meta name="citation_author_institution" content="Division of Cardiology Faculty of Medicine and Dentistry, University of Alberta, Edmonton, Canada.;"/>
  <meta name="citation_author_email" content="aviveiro@ualberta.ca"/>
  <meta name="citation_author" content="Gavin Oudit"/>
  <meta name="citation_author_institution" content="Division of Cardiology Faculty of Medicine and Dentistry, University of Alberta, Edmonton, Canada.;"/>
  <meta name="citation_author_email" content="gavin.oudit@ualberta.ca"/>
  <meta name="citation_author" content="Omer Bayraktar"/>
  <meta name="citation_author_institution" content="Wellcome Sanger Institute;"/>
  <meta name="citation_author_email" content="ob5@sanger.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6055-277X"/>
  <meta name="citation_author" content="J. G. Seidman"/>
  <meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, United States.;"/>
  <meta name="citation_author_email" content="seidman@genetics.med.harvard.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9082-3566"/>
  <meta name="citation_author" content="Christine Seidman"/>
  <meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA, United States.;"/>
  <meta name="citation_author_email" content="cseidman@genetics.med.harvard.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6380-1209"/>
  <meta name="citation_author" content="Michela Noseda"/>
  <meta name="citation_author_institution" content="National Heart and Lung Institute, Imperial College London, London, United Kingdom.;"/>
  <meta name="citation_author_email" content="m.noseda@imperial.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9553-5029"/>
  <meta name="citation_author" content="Norbert Hubner"/>
  <meta name="citation_author_institution" content="Max-Delbruck-Center for Molecular Medicine"/>
  <meta name="citation_author_email" content="nhuebner@mdc-berlin.de"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1218-6223"/>
  <meta name="citation_author" content="Sarah A. Teichmann"/>
  <meta name="citation_author_institution" content="Wellcome Sanger Institute;"/>
  <meta name="citation_author_email" content="st9@sanger.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6294-6366"/>
  <meta name="twitter:title" content="Cells and gene expression programs in the adult human heart"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Cardiovascular disease is the leading cause of death worldwide. Advanced insights into disease mechanisms and strategies to improve therapeutic opportunities require deeper understanding of the molecular processes of the normal heart. Knowledge of the full repertoire of cardiac cells and their gene expression profiles is a fundamental first step in this endeavor. Here, using large-scale single cell and nuclei transcriptomic profiling together with state-of-the-art analytical techniques, we characterise the adult human heart cellular landscape covering six anatomical cardiac regions (left and right atria and ventricles, apex and interventricular septum). Our results highlight the cellular heterogeneity of cardiomyocytes, pericytes and fibroblasts, revealing distinct subsets in the atria and ventricles indicative of diverse developmental origins and specialized properties. Further we define the complexity of the cardiac vascular network which includes clusters of arterial, capillary, venous, lymphatic endothelial cells and an atrial-enriched population. By comparing cardiac cells to skeletal muscle and kidney, we identify cardiac tissue resident macrophage subsets with transcriptional signatures indicative of both inflammatory and reparative phenotypes. Further, inference of cell-cell interactions highlight a macrophage-fibroblast-cardiomyocyte network that differs between atria and ventricles, and compared to skeletal muscle. We expect this reference human cardiac cell atlas to advance mechanistic studies of heart homeostasis and disease. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Cells and gene expression programs in the adult human heart"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.024075v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Cardiovascular disease is the leading cause of death worldwide. Advanced insights into disease mechanisms and strategies to improve therapeutic opportunities require deeper understanding of the molecular processes of the normal heart. Knowledge of the full repertoire of cardiac cells and their gene expression profiles is a fundamental first step in this endeavor. Here, using large-scale single cell and nuclei transcriptomic profiling together with state-of-the-art analytical techniques, we characterise the adult human heart cellular landscape covering six anatomical cardiac regions (left and right atria and ventricles, apex and interventricular septum). Our results highlight the cellular heterogeneity of cardiomyocytes, pericytes and fibroblasts, revealing distinct subsets in the atria and ventricles indicative of diverse developmental origins and specialized properties. Further we define the complexity of the cardiac vascular network which includes clusters of arterial, capillary, venous, lymphatic endothelial cells and an atrial-enriched population. By comparing cardiac cells to skeletal muscle and kidney, we identify cardiac tissue resident macrophage subsets with transcriptional signatures indicative of both inflammatory and reparative phenotypes. Further, inference of cell-cell interactions highlight a macrophage-fibroblast-cardiomyocyte network that differs between atria and ventricles, and compared to skeletal muscle. We expect this reference human cardiac cell atlas to advance mechanistic studies of heart homeostasis and disease. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-10"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Cells and gene expression programs in the adult human heart | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1239213 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-category-new-results"> 
  <div class="article abstract-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-2">Cardiovascular disease is the leading cause of death worldwide. Advanced insights into disease mechanisms and strategies to improve therapeutic opportunities require deeper understanding of the molecular processes of the normal heart. Knowledge of the full repertoire of cardiac cells and their gene expression profiles is a fundamental first step in this endeavor. Here, using large-scale single cell and nuclei transcriptomic profiling together with state-of-the-art analytical techniques, we characterise the adult human heart cellular landscape covering six anatomical cardiac regions (left and right atria and ventricles, apex and interventricular septum). Our results highlight the cellular heterogeneity of cardiomyocytes, pericytes and fibroblasts, revealing distinct subsets in the atria and ventricles indicative of diverse developmental origins and specialized properties. Further we define the complexity of the cardiac vascular network which includes clusters of arterial, capillary, venous, lymphatic endothelial cells and an atrial-enriched population. By comparing cardiac cells to skeletal muscle and kidney, we identify cardiac tissue resident macrophage subsets with transcriptional signatures indicative of both inflammatory and reparative phenotypes. Further, inference of cell-cell interactions highlight a macrophage-fibroblast-cardiomyocyte network that differs between atria and ventricles, and compared to skeletal muscle. We expect this reference human cardiac cell atlas to advance mechanistic studies of heart homeostasis and disease.</p>
   </div>
   <h3>Competing Interest Statement</h3>
   <p id="p-3">The authors have declared no competing interest.</p>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-dataset fn-group-external-links" id="fn-1">
      <p id="p-4">
       <a href="https://www.heartcellatlas.org/">https://www.heartcellatlas.org/</a>
      </p>
     </li>
    </ul>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
