<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2015/09/28/027722.atom" />
<meta name="HW.pisa" content="biorxiv;027722v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="On the causes of evolutionary transition:transversion bias" />
<meta name="DC.Identifier" content="10.1101/027722" />
<meta name="DC.Date" content="2015-09-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2015, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the “conservative transitions” hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias." />
<meta name="DC.Contributor" content="Arlin Stoltzfus" />
<meta name="DC.Contributor" content="Ryan W. Norris" />
<meta name="article:published_time" content="2015-09-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="On the causes of evolutionary transition:transversion bias" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the “conservative transitions” hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2015/01/01" />
<meta name="citation_mjid" content="biorxiv;027722v1" />
<meta name="citation_id" content="027722v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/027722v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/027722v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/027722v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722.full.pdf" />
<meta name="citation_doi" content="10.1101/027722" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="027722" />
<meta name="citation_author" content="Arlin Stoltzfus" />
<meta name="citation_author_institution" content="Institute for Bioscience and Biotechnology Research" />
<meta name="citation_author_email" content="arlin.stoltzfus@nist.gov" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0963-1357" />
<meta name="citation_author" content="Ryan W. Norris" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0963-1357" />
<meta name="citation_reference" content="Acevedo A, Brodsky L, Andino R. 2014. Mutational and fitness landscapes of an RNA virus revealed through population sequencing. Nature 505: 686–690." />
<meta name="citation_reference" content="Bloom JD. 2014. An experimentally determined evolutionary model dramatically improves phylogenetic fit. Mol Biol Evol 31: 1956–1978." />
<meta name="citation_reference" content="Carrasco P, Daros JA, Agudelo-Romero P, Elena SF. 2007. A real-time RT-PCR assay for quantifying the fitness of tobacco etch virus in competition experiments. J Virol Methods 139: 181–188." />
<meta name="citation_reference" content="Di Giulio M. 2001. The origin of the genetic code cannot be studied using measurements based on the PAM matrix because this matrix reflects the code itself, making any such analyses tautologous. J Theor Biol 208: 141–144." />
<meta name="citation_reference" content="Domingo-Calap P, Cuevas JM, Sanjuan R. 2009. The fitness effects of random mutations in single-stranded DNA and RNA bacteriophages. PLoS Genet 5:e1000742." />
<meta name="citation_reference" content="Eyre-Walker A, Keightley PD. 2007. The distribution of fitness effects of new mutations. Nat Rev Genet 8: 610–618." />
<meta name="citation_reference" content="Ferris MT, Joyce P, Burch CL. 2007. High frequency of mutations that expand the host range of an RNA virus. Genetics 176: 1013–1022." />
<meta name="citation_reference" content="Firnberg E, Labonte JW, Gray JJ, Ostermeier M. 2014. A Comprehensive, High-Resolution Map of a Gene’s Fitness Landscape. Mol Biol Evol 31: 1581–1592." />
<meta name="citation_reference" content="Fitch WM. 1967. Evidence Suggesting a Non-random Character to Nucleotide Replacements in Naturally Occurring Mutations. J Mol Biol 26: 499–507." />
<meta name="citation_reference" content="Grantham R. 1974. Amino acid difference formula to help explain protein evolution. Science 185: 862–864." />
<meta name="citation_reference" content="Hanley JA, McNeil BJ. 1982. The meaning and use of the area under a receiver operating characteristic (ROC) curve. Radiology 143: 29–36." />
<meta name="citation_reference" content="Heider D. 2012. Interpolation of amino acid sequences. Version 1.3.1: University of Duisburg-Essen, Germany." />
<meta name="citation_reference" content="Jacquier H, Birgy A, Le Nagard H, Mechulam Y, Schmitt E, Glodt J, Bercot B, Petit E, Poulain J, Barnaud G, et al. 2013. Capturing the mutational landscape of the betalactamase TEM-1. Proc Natl Acad Sci U S A 110: 13067–13072." />
<meta name="citation_reference" content="Jiang C, Zhao Z. 2006. Mutational spectrum in the recent human genome inferred by single nucleotide polymorphisms. Genomics 88: 527–534." />
<meta name="citation_reference" content="Kawashima S, Kanehisa M. 2000. AAindex: amino acid index database. Nucleic Acids Res 28: 374." />
<meta name="citation_reference" content="Keller I, Bensasson D, Nichols RA. 2007. Transition-transversion bias is not universal: a counter example from grasshopper pseudogenes. PLoS Genet 3:e22." />
<meta name="citation_reference" content="Kleina LG, Miller JH. 1990. Genetic studies of the lac repressor. XIII. Extensive amino acid replacements generated by the use of natural and synthetic nonsense suppressors. J Mol Biol 212: 295–318." />
<meta name="citation_reference" content="Kumar S. 1996. Patterns of nucleotide substitution in mitochondrial protein coding genes of vertebrates. Genetics 143: 537–548." />
<meta name="citation_reference" content="Li W-H. 1997. Molecular Evolution. Sunderland, Mass.: Sinauer." />
<meta name="citation_reference" content="Li W-H, Luo C-C, Wu C-I. 1985. Evolution of DNA Sequences. In: MacIntyre RJ, editor. Molecular Evolutionary Genetics. New York: Plenum." />
<meta name="citation_reference" content="Lynch M. 2010. Rate, molecular spectrum, and consequences of human mutation. Proceedings of the National Academy of Sciences of the United States of America 107: 961–968." />
<meta name="citation_reference" content="MacLean RC, Perron GG, Gardner A. 2010. Diminishing returns from beneficial mutations and pervasive epistasis shape the fitness landscape for rifampicin resistance in Pseudomonas aeruginosa. Genetics 186: 1345–1354." />
<meta name="citation_reference" content="McCandlish DM, Stoltzfus A. 2014. Modeling evolution using the probability of fixation: history and implications. Quarterly Review of Biology 89: 225–252." />
<meta name="citation_reference" content="Miller CR, Joyce P, Wichman HA. 2011. Mutational effects and population dynamics during viral adaptation challenge current models. Genetics 187: 185–202." />
<meta name="citation_reference" content="Miyata T, Miyazawa S, Yasunaga T. 1979. Two types of amino acid substitutions in protein evolution. J Mol Evol 12: 219–236." />
<meta name="citation_reference" content="Morton BR, Bi IV, McMullen MD, Gaut BS. 2006. Variation in mutation dynamics across the maize genome as a function of regional and flanking base composition. Genetics 172: 569–577." />
<meta name="citation_reference" content="Nei M. 2013. Mutation-Driven Evolution: Oxford University Press." />
<meta name="citation_reference" content="Peris JB, Davis P, Cuevas JM, Nebot MR, Sanjuan R. 2010. Distribution of fitness effects caused by single-nucleotide substitutions in bacteriophage f1. Genetics 185: 603–609." />
<meta name="citation_reference" content="Petrov DA, Hartl DL. 1999. Patterns of nucleotide substitution in Drosophila and mammalian genomes. Proc Natl Acad Sci U S A 96: 1475–1479." />
<meta name="citation_reference" content="Rihn SJ, Hughes J, Wilson SJ, Bieniasz PD. 2015. Uneven Genetic Robustness of HIV-1 Integrase. J Virol 89: 552–567." />
<meta name="citation_reference" content="Rihn SJ, Wilson SJ, Loman NJ, Alim M, Bakker SE, Bhella D, Gifford RJ, Rixon FJ, Bieniasz PD. 2013. Extreme genetic fragility of the HIV-1 capsid. PLoS Pathog 9:e1003461." />
<meta name="citation_reference" content="Roscoe BP, Thayer KM, Zeldovich KB, Fushman D, Bolon DN. 2013. Analyses of the effects of all ubiquitin point mutants on yeast growth rate. J Mol Biol 425: 1363–1377." />
<meta name="citation_reference" content="Rosenberg MS, Subramanian S, Kumar S. 2003. Patterns of transitional mutation biases within and among mammalian genomes. Mol Biol Evol 20: 988–993." />
<meta name="citation_reference" content="Sanjuan R, Moya A, Elena SF. 2004. The distribution of fitness effects caused by single-nucleotide substitutions in an RNA virus. Proc Natl Acad Sci U S A 101: 8396–8401." />
<meta name="citation_reference" content="Schaaper RM, Dunn RL. 1991. Spontaneous mutation in the Escherichia coli lacI gene. Genetics 129: 317–326." />
<meta name="citation_reference" content="Schenk MF, Szendro IG, Krug J, de Visser JA. 2012. Quantifying the adaptive potential of an antibiotic resistance enzyme. PLoS Genet 8:e1002783." />
<meta name="citation_reference" content="Schrider DR, Houle D, Lynch M, Hahn MW. 2013. Rates and genomic consequences of spontaneous mutational events in Drosophila melanogaster. Genetics 194: 937–954." />
<meta name="citation_reference" content="Sinsheimer JS, Lake JA, Little RJ. 1997. Inference for phylogenies under a hybrid parsimony method: evolutionary-symmetric transversion parsimony. Biometrics 53: 23–38." />
<meta name="citation_reference" content="Stoltzfus A. 2006. Mutation-Biased Adaptation in a Protein NK Model. Mol Biol Evol 23: 1852–1862." />
<meta name="citation_reference" content="Tang H, Wyckoff GJ, Lu J, Wu CI. 2004. A universal evolutionary index for amino acid changes. Mol Biol Evol 21: 1548–1556." />
<meta name="citation_reference" content="Thyagarajan B, Bloom JD. 2014. The inherent mutational tolerance and antigenic evolvability of influenza hemagglutinin. Elife:e03300." />
<meta name="citation_reference" content="Vogel F. 1972. Non-randomness of base replacement in point mutation. J Mol Evol 1: 334–367." />
<meta name="citation_reference" content="Vogel F, Kopun M. 1977. Higher frequencies of transitions among point mutations. J Mol Evol 9: 159–180." />
<meta name="citation_reference" content="Wakeley J. 1996. The excess of transitions among nucleotide substitutions: new methods of estimating transition bias underscore its significance. TREE 11: 158–162." />
<meta name="citation_reference" content="Wu NC, Young AP, Al-Mawsawi LQ, Olson CA, Feng J, Qi H, Chen SH, Lu IH, Lin CY, Chin RG, et al. 2014. High-throughput profiling of influenza A virus hemagglutinin gene at single-nucleotide resolution. Sci Rep 4: 4942." />
<meta name="citation_reference" content="Xia X. 1998. The rate heterogeneity of nonsynonymous substitutions in mammalian mitochondrial genes. Mol Biol Evol 15: 336–344." />
<meta name="citation_reference" content="Yampolsky LY, Stoltzfus A. 2001. Bias in the introduction of variation as an orienting factor in evolution. Evol Dev 3: 73–83." />
<meta name="citation_reference" content="Yampolsky LY, Stoltzfus A. 2005. The exchangeability of amino acids in proteins. Genetics 170: 1459–1472." />
<meta name="citation_reference" content="Zhang J. 2000. Rates of conservative and radical nonsynonymous nucleotide substitutions in mammalian nuclear genes. J Mol Evol 50: 56–68." />
<meta name="citation_reference" content="Zhao Z, Li H, Wu X, Zhong Y, Zhang K, Zhang YP, Boerwinkle E, Fu YX. 2004. Moderate mutation rate in the SARS coronavirus genome and its implications. BMC Evol Biol 4: 21." />
<meta name="citation_reference" content="Zhu YO, Siegal ML, Hall DW, Petrov DA. 2014. Precise estimates of mutation rate and spectrum in yeast. Proc Natl Acad Sci USA 111:E2310–2318." />
<meta name="twitter:title" content="On the causes of evolutionary transition:transversion bias" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the “conservative transitions” hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias." />
<meta name="og-title" property="og:title" content="On the causes of evolutionary transition:transversion bias" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/027722v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the “conservative transitions” hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2015-09-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/027722v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/027722v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/027722v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/027722v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/7417" />
    <title>On the causes of evolutionary transition:transversion bias | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gCnpQYCy6zf9e0rATaxcAEL2DgbKUUq1JPIbAgnj1Q0","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"7417","apath":"\/biorxiv\/early\/2015\/09\/28\/027722.atom","pisa":"biorxiv;027722v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;027722\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;027722\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;027722v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;027722\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;027722\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/027722v1","title":"On the causes of evolutionary transition:transversion bias","identifier":"node\/7417"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+iyTwJ55SEfkd+SohwAtsJemPnPhGaZ6VonFH7vBCnVhTrCGrmZdSKAEEm6TAbsd5KuiSUl3rnZ+GWrHLhPAm8cTmrNhxYovcqnWPFovAvH87ixhGWxRA+IGpyj5+KMINCSMAjvlPRJ61JSuygeXgA0kGZLwR6hiP7WzDI4kraJ1eSKqpXXceKZERdGBvNRPAuMVztGB249UUkOGaQZuZqn+KSJqYjWqkXi+pEhay31dH5fd7ltzyZsTsqT2IK7VWl8mxfMrpxG2ZYtDAyeccl5J22z6KA2+xuozk3GOBf3TYvX9BC8TL7n1+VCklXWNN4N5VayFO9LUI73Y2moXL4TzMMWjRPpgs153B4qle4J+\\\\\\\/+8e+f3k8K9ImYpzVs1uJ\\\\\\\/2WuE4t6+PquvtuKeoO3tUB1XeeEUnZ6iDIa0MDq+7NjOTxupq3StTY8UIpo4AQMx\\\\\\\/X9Nui9kNRHcFkYKbeUccbgtOHjszIAKQgCSBw8hMQZ4HH5pAnZetx9Tne5FXFypPv1Dxck5kHdn0igvVp\\\\\\\/rpbB1HfFeYKPWHw\\\\\\\/rLOajbYzsDPrFeKh9MiOghUuW76MQG2pTexIIlld2dUQG93emtYcwtqsJXjvpHcEMB2cZvMRX5943PmJ0ILrbLaNCtrDgmOZWDUwMwFijbuxRTfgN56rSGdJnTnKg5SMK2tsQeMfnghVd+hAr8GlwYD1Zntser4BD3c2Llx0Tvll9HjTAjGUfRj8WawhJZl2bXFMNu\\\\\\\/AxFaiHfrWxaJD4kkO\\\\\\\/HzwgVrn9Oj3673XyK4mPPM5wLJgojX46yl6jh5hH8BcvyrpoBpDMpffduFCyD2AAKxW8uIibX25zyw1FXxGdUXofwkoBGYrCuACpl8kN4s6RKOFsApVSMPI5Op+G4LvfaaWHn7gWikEuW0umPiQNMmGrwqG6280xlPapzXtn4lbZkXpEqw5uTLR8OtjCnuHwEQ6of2aftS0Xb1QYVOIXjSR6+OhzgnIYB3JY6L3L1YbJHXOQa+VYASVoaIh4TVTctdKgcsrhXihYI\\\\\\\/rCU6wGlsTGQFS8wl4yjsJChDvVFkzX0RGhvG+CHLt8MG68y9KWHgxKU0b8LXuDsdyt6yWFIusMb3k07uWrcSx8S00vIZ64SEsLsHXCxspMUNZeAUV0bI8AToaaTZvh3S2rRy4l7L0yuLV7UjHrMCdkaMb3Qz48uyYQ0fb0bSdnv2vFDLi6uH1V3yHwp5H8x+i5pwRIjTioYeJxcVgY7fbFaHdBRbkkSjqSkn8N3q5JxBYdqy1aEZWLW8u6QUhvFJnZIXv7xuf2BttAnpIiREyLOeD2S5ylI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022MPwRw3VIe6JsaQTg0PBEBw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222cd8c0a02dbcfb542242f633a2f14a2eb63d257e6a9d524f5e72e36c0b11b5aa\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/027722v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-7417 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/027722v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mQuqNRTWnGFHT6WmgYA07WxLm_eHtLQ2ZSnwfKsZ_dg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="7417" id="node7417" data-pisa="biorxiv;027722v1" data-pisa-master="biorxiv;027722" data-apath="/biorxiv/early/2015/09/28/027722.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">On the causes of evolutionary transition:transversion bias</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0963-1357" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Arlin</span> <span class="nlm-surname">Stoltzfus</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0963-1357" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ryan W.</span> <span class="nlm-surname">Norris</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/027722 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node7417" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Arlin Stoltzfus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute for Bioscience and Biotechnology Research</span>, 9600 Gudelsky Dr., Rockville, <span class='nlm-country'>MD</span>, 20850
</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arlin%2BStoltzfus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoltzfus%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArlin%2BStoltzfus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0963-1357" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arlin Stoltzfus</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:arlin.stoltzfus@nist.gov" class="" data-icon-position="" data-hide-link-title="0">arlin.stoltzfus@nist.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ryan W. Norris </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BW.%2BNorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norris%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARyan%2BW.%2BNorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0963-1357" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ryan W. Norris</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/027722v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:7417" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:7417/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/027722v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:7417" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:7417/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/027722v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:7417" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:7417/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/027722v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:7417" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:7417/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/027722v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:7417" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:7417/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/027722v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:7417" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:7417/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the “conservative transitions” hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Of the 12 types of changes from one nucleotide to another, 8 are “transversions” between a purine (A or G) and a pyrimidine (C or T), and the other 4 are “transitions”. Early protein comparisons showed that related proteins often differ by transitions more than expected by chance (e.g., <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fitch 1967</a>; sources cited in <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vogel 1972</a>). By the 1980's, this “transition bias” was well known (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li, et al. 1985</a>). By the 1990s, systematists had noted effects on phylogeny inference (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wakeley 1996</a>), and methods were revised to give more weight to transversion differences (e.g., <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sinsheimer, et al. 1997</a>).</p><p id="p-5">In many early works, this bias is presented as a ratio of differences, which makes the expected ratio a complex function of the degree of sequence divergence. As the use of rate models became routine in comparative sequence analysis, the phenomenon of transition bias was redefined as a bias in instantaneous rates, relative to a null model of equal rates. Because every nucleotide site (e.g., a G site) may experience 1 type of transition (G→A) at rate α, and 2 types of transversion (G→C, G→T) at rate β, the aggregate rate ratio of transitions to transversions has a null expectation of R = α/(2β) = 0.5. In some contexts, the ratio is expressed differently as κ = α/β = 1. When considering amino acid changes, it is more relevant to compare the 116 possible transitions and 276 possible transversions that change a codon so as to encode a different amino acid (assuming the canonical genetic code), leading to a null expectation of R = 116 α / (276 β) = 0.42 α/β. Thus, the observation of roughly equal numbers of inferred transitions and transversions in classic works (e.g., <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vogel and Kopun 1977</a>), or in the extensive analysis of mammalian genes in <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li (1997</a>, Table 7.2), indicates a bias of over 2-fold. <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kumar (1996)</a> estimates 2-fold to 5-fold rate biases in vertebrate mitochondrial genes (excluding 3<sup>rd</sup> positions). Other estimates may be found in work cited by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rosenberg, et al. (2003)</a>, but there is not (to our knowledge) a systematic contemporary review of this issue.</p><p id="p-6">The causes of the observed bias have not been resolved. The hypothesis of a mutational cause— a transition:transversion bias in mutation— was promoted early by <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Vogel (1972</a>; see also <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Vogel and Kopun 1977</a>). This hypothesis was bolstered when DNA sequence comparisons revealed that a transition bias is observed in introns and other non-coding regions (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li, et al. 1985</a>), suggesting a cause that (like mutation) acts at the level of DNA, across the entire genome.</p><p id="p-7">The alternative hypothesis that natural selection favors amino acid replacements via transitions is also common, and is argued on the grounds that transitions are “less severe with respect to the chemical properties of the original and mutant amino acids” (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Rosenberg, et al. 2003</a>) or “tend to cause changes that conserve the chemical properties of amino acids” (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Wakeley 1996</a>), or that “the biochemical difference in the protein product tends to be greater for transversions” (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Keller, et al. 2007</a>).</p><p id="p-8">For purposes of evaluation, we can break down either the mutational hypothesis or the selective hypothesis into (1) a claim that there is an underlying bias (mutational or selective) favoring transitions, and (2) a claim that this bias accounts for the observed evolutionary bias. For the mutational hypothesis, the existence of an underlying bias is indicated in direct studies of mutation (e.g., <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schaaper and Dunn 1991</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lynch 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schrider, et al. 2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhu, et al. 2014</a>), and by many indirect estimates based on the asumption of neutral sequence divergence (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Petrov and Hartl 1999</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Rosenberg, et al. 2003</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhao, et al. 2004</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jiang and Zhao 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Morton, et al. 2006</a>), though <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Keller, et al. (2007)</a> report a lack of bias in grasshoppers. The bias typically is 2-fold to 4-fold over null expectations. In theory, a bias in mutation of magnitude B can cause a B-fold effect on the rate of evolution (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yampolsky and Stoltzfus 2001</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">McCandlish and Stoltzfus 2014</a>). That is, the observed magnitude of mutation bias appears to be sufficient, in principle, to account for the observed evolutionary bias.</p><p id="p-9">For the selective hypothesis, arguments to the effect that transitions are more conservative typically invoke a biochemical factor (or a composite such as the Grantham index) that correlates with patterns of evolutionary divergence, and is found to be more conserved by transitions than by transversions (e.g., <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Vogel and Kopun 1977</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang 2000</a>). This form of argument suffers from a logical circularity: if mutation shapes patterns of evolutionary amino acid replacement, then biochemical factors chosen for their ability to make sense of evolutionary patterns are not independent of mutation.</p><p id="p-10">Presumably no biochemical factor, nor any simple combination of factors, fully captures the effects of replacements in complex proteins operating in a complex milieu. Indeed, the use of biochemical surrogates would seem unnecessary, given the availability of more direct measurements. Systematic laboratory studies of the effects of amino acid replacements in proteins have been carried out for 25 years (e.g., <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kleina and Miller 1990</a>). Whereas early studies summarized by <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yampolsky and Stoltzfus (2005)</a> typically reported crude measures of biochemical or growth effects (e.g., a 2-valued scale of “−” and “+”), a number of more recent studies report a continuous measure of fitness for each mutant (e.g., <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sanjuan, et al. 2004</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carrasco, et al. 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Domingo-Calap, et al. 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Peris, et al. 2010</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jacquier, et al. 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Roscoe, et al. 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Acevedo, et al. 2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bloom 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Firnberg, et al. 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Thyagarajan and Bloom 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wu, et al. 2014</a>). Such studies provide direct evidence on the relative conservativeness of transitions and transversions that change amino acids.</p><p id="p-11">Here we focus on whether direct measurements of fitness support the conservative transitions hypothesis, based on a collection of 8 studies comprising measured fitness values for 544 transitions and 695 transversions that change an amino acid. We assess the power of each study by comparing mutant fitnesses for each type of replacement (e.g., Ser to Pro) with a cross-validation predictor and with 2 existing measures of amino acid exchangeability called EX (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Yampolsky and Stoltzfus 2005</a>) and U (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tang, et al. 2004</a>). We find that, for every mutation study, even the smallest, there is a significant correlation with one or more of these predictors; half of the studies show a highly significant correlation (P &lt; 0.001). More importantly, for most studies, measured fitness values correlate significantly with a conservative-vs-radical distinction based on EX or U. Specifically, a replacement designated as “conservative” has a 65 % (EX) or 64 % (U) chance of being more fit than a “radical” replacement.</p><p id="p-12">However, the same studies typically do <strong>not</strong> show significant conservativeness of transitions. In the combined data, a transition has a 53 % chance (CI, 50 % to 56 %) of being more fit than a transversion, only slightly above the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical predictors, and is not large enough to explain the several-fold bias toward transition replacements observed in evolutionary studies. The mutation-bias hypothesis, though not proven, remains an obvious possibility, while the selective hypothesis would seem untenable.</p><div id="sec-2" class="subsection"><h3>Results</h3><p id="p-13">The literature search described in Materials and Methods (see also Supplementary Material) resulted in the 8 data sets in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, each of which provides measures of fitness based on individual growth or paired growth (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Sanjuan, et al. 2004</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Carrasco, et al. 2007</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Domingo-Calap, et al. 2009</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Peris, et al. 2010</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Jacquier, et al. 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rihn, et al. 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rihn, et al. 2015</a>). We will refer to these 8 data sets as 8 studies, although they correspond to 7 publications, one of which (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Domingo-Calap, et al. 2009</a>) reports separate mutant fitness distributions for 2 different phages. Because measures of growth from different studies are not scaled in the same way, we convert fitnesses to within-study quantiles, e.g., the median fitness in a study is assigned a quantile of 0.5, and the fitness at the 95<sup>th</sup> percentile is assigned a quantile of 0.95.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/190504/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/190504/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/190504" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-14" class="first-child">Power of 8 studies to discern generic effects of exchangeability.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Correlation of mutant fitnesses with amino acid exchangeability</h3><p id="p-15">To assess the power of mutation studies individually and collectively, we correlate observed mutant fitness quantiles with expected values from three independent predictors: the EX matrix (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Yampolsky and Stoltzfus 2005</a>), the U matrix of <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Tang, et al (2004)</a>, and a cross-validation predictor. The cross-validation predictor applied to a given target study is constructed from all <strong>other</strong> studies (i.e., excluding the target study), and is simply a matrix of mean quantiles for each type of replacement (e.g., Ala to Val).</p><p id="p-16">EX and U are used on the grounds of being powerful and mutationally unbiased predictors, whereas various biochemical predictors are less powerful (as will become apparent below), and various evolution-based measures other than U (e.g., PAM, BLOSUM), though perhaps powerful, cannot be used, because they are not known to be free of the mutational effects that we wish to exclude. The EX matrix, based on a meta-analysis of early mutation studies (which reported phenotypes other than fitness), was designed specifically to serve as a mutationally unbiased measure of exchangeability in models that separate selection from mutation. In a comparative evaluation, EX was shown to be as powerful, or more powerful, than a representative sample of other predictors (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Yampolsky and Stoltzfus 2005</a>). The “universal evolutionary index” or U matrix of <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Tang, et al (2004)</a> is based on modeling evolution of thousands of genes, using a method designed to separate codon-level mutational effects from protein-level effects. It purports to be a measure of evolutionary acceptability that scales directly with the rate of evolution.</p><p id="p-17">The results of using EX, U and a cross-validation predictor (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) indicate that even small studies of mutant fitnesses have considerable power to reveal generic effects of amino acid exchangeability. For instance, for the study of 135 HIV capsid mutants by Rihn, et al., there is a significant correlation between the fitness reported for a mutant and the predictor for the relevant replacement type (e.g., Val to Ala), whether the predictor is EX, U, or a cross-validation predictor based on the other studies. This shows, not only that individual studies are powerful, but that there is a consistency across studies: although most effects of an amino acid replacement in a protein are very context-dependent (which is why the R<sup>2</sup> values are small), generic effects of exchangeability are seen across sites and proteins.</p></div><div id="sec-4" class="subsection"><h3>Ability to distinguish conservative from radical replacements</h3><p id="p-18">The conservative transitions hypothesis proposes that transitions collectively are more conservative than transversions. How well do mutant fitness studies distinguish conservative replacements from radical ones? We construct two versions of this distinction, EX<sub>B</sub> and U<sub>B</sub> (the “B” indicates a binary distinction, as opposed to a continuous measure), simply by designating higher-exchangeability replacements as “conservative”, and the remainder as “radical”.</p><p id="p-19"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows how well studies of mutant fitness distinguish conservative from radical replacements, and how well they distinguish transitions from transversions. The measure of effect-size used here is the chance that a mutant designated as “conservative” is more fit than a randomly chosen “radical” mutant. This statistic is not affected by the relative sizes of the 2 classes; its range is from 0 to 1, with a null expectation of 0.5; higher values indicate that nominally “conservative” changes are indeed conservative. We call this measure AUC because it has the same meaning as the area under a ROC (receiver-operating characteristic) curve for a binary classifier. That is, as pointed out by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hanley and McNeil (1982)</a>, the AUC for a binary classifier is equivalent to the chance that a randomly chosen positive instance will be ranked higher than a randomly chosen negative instance (see Methods).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/190277/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/190277/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/190277" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-20" class="first-child">Power of 8 studies to detect various binary distinctions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Even small studies have significant power to distinguish conservative from radical substitutions based on EX<sub>B</sub> and U<sub>B</sub>. In the combined data set, the AUC is 0.65 for EX<sub>B</sub> and 0.64 for U<sub>B</sub>. That is, a conservative replacement according to EX<sub>B</sub> has a 65 % chance of being more fit than a randomly drawn radical replacement.</p><p id="p-22">However, the same studies typically do not distinguish transitions from transversions. Only one study shows a marginally significant result (p = 0.019 for the largest study). The combined results for the entire set of 1239 replacements are shown at the bottom of <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. For the combined data, the AUC is 0.53, with a 95 % confidence interval of 0.50 to 0.56 (based on 400 bootstrap replicates).</p><p id="p-23">One might object that this approach is framed incorrectly, in that it uses the entire distribution of mutational effects, whereas the distribution of changes fixed in evolution is obviously weighted toward more modest effects, because natural selection removes the most damaging ones whether they are transitions or transversions. If the changes actually accepted in evolution are mostly in the top 50 %, or the top 5 %, of the fitness distribution, then this is the fraction that should be examined most closely to test the conservative transitions hypothesis.</p><p id="p-24">The effect of testing for a ti:tv effect at successively higher thresholds of fitness is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. In fact, the AUC does not go up if we filter out the low end, but stays close to 0.5.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The relative conservativeness of transitions for distributions of mutant effects truncated at the low end of fitness. The advantage of transitions (AUC) is shown as a function of threshold quantile for left-truncated data, e.g., the AUC value for x = 0.2 is computed without the bottom 20 % of the distribution. Under the conservative transitions hypothesis, one might expect that, even if there is no advantage over the entire distribution, an advantage will appear at the high end. In fact, this is not observed. As mentioned in the text, AUC = 0.53 for the complete set of data, corresponding to a truncation threshold of 0, i.e., no truncation. As the threshold increases, AUC decreases (rather than increases), though the differences are insignificant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991900145" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The relative conservativeness of transitions for distributions of mutant effects truncated at the low end of fitness. The advantage of transitions (AUC) is shown as a function of threshold quantile for left-truncated data, e.g., the AUC value for x = 0.2 is computed without the bottom 20 % of the distribution. Under the conservative transitions hypothesis, one might expect that, even if there is no advantage over the entire distribution, an advantage will appear at the high end. In fact, this is not observed. As mentioned in the text, AUC = 0.53 for the complete set of data, corresponding to a truncation threshold of 0, i.e., no truncation. As the threshold increases, AUC decreases (rather than increases), though the differences are insignificant.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F1.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F1.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-25" class="first-child">The relative conservativeness of transitions for distributions of mutant effects truncated at the low end of fitness. The advantage of transitions (AUC) is shown as a function of threshold quantile for left-truncated data, e.g., the AUC value for x = 0.2 is computed without the bottom 20 % of the distribution. Under the conservative transitions hypothesis, one might expect that, even if there is no advantage over the entire distribution, an advantage will appear at the high end. In fact, this is not observed. As mentioned in the text, AUC = 0.53 for the complete set of data, corresponding to a truncation threshold of 0, i.e., no truncation. As the threshold increases, AUC decreases (rather than increases), though the differences are insignificant.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Another way to explore the upper end of the fitness distribution is to consider studies of mutational effects that focus on <strong>beneficial</strong> mutations (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ferris, et al. 2007</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">MacLean, et al. 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Miller, et al. 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schenk, et al. 2012</a>). These studies are small, with only 15 to 38 mutants, and have little power to distinguish effects of exchangeability. For the combined set of 111 beneficial mutants shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, the AUC for the conservative transitions hypothesis is 0.40 (95 % CI, 0.28 to 0.51), suggesting that perhaps beneficial transitions are not more, but less fit than beneficial transversions.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/190602/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/190602/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/190602" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-27" class="first-child">Relative advantage of transitions in 4 studies of beneficial mutations</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Gauging the evolutionary effect size</h3><p id="p-28">As mentioned above, the conservative transitions hypothesis has 2 parts, a claim that transitions are conservative, and a claim that this conservativeness accounts for an evolutionary pattern. The present set of studies suggests that transitions are more conservative, but only slightly. How important could an effect of this size be?</p><p id="p-29">One way to ask this question is to compare the ti:tv distinction to various biochemical distinctions. Any quantitative property of an amino acid can be used to create a conservative-vs-radical distinction, e.g., for a measure of the polarity of each amino acid, the “conservative” changes will be the ones with the least change in polarity. The AAindex database (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kawashima and Kanehisa 2000</a>) has data on nearly 250 biochemical factors (see Methods). The random sample of 25 factors from AAIndex shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> indicates that biochemical predictors typically are (1) considerably more powerful than the ti:tv distinction, and (2) considerably less powerful than EX and U (to view the full set of predictors, see Supplementary Material, section 3).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative power of conservative-radical distinctions based on some biochemical factors. Binary predictors based on 25 randomly chosen biochemical factors from the AAIndex database were applied to the prediction of mutant fitnesses in mutation-scanning experiments (for the full set of predictors, see Supplementary Material, section 3). The AUC is the chance that a randomly chosen mutant designated as &#x201C;conservative&#x201D; by the predictor has a higher fitness than one designated as &#x201C;radical&#x201D;. Most predictors are more powerful than the ti:tv distinction (AUC = 0.53)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991900145" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative power of conservative-radical distinctions based on some biochemical factors. Binary predictors based on 25 randomly chosen biochemical factors from the AAIndex database were applied to the prediction of mutant fitnesses in mutation-scanning experiments (for the full set of predictors, see Supplementary Material, section 3). The AUC is the chance that a randomly chosen mutant designated as &#x201C;conservative&#x201D; by the predictor has a higher fitness than one designated as &#x201C;radical&#x201D;. Most predictors are more powerful than the ti:tv distinction (AUC = 0.53).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F2.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F2.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-30" class="first-child">Relative power of conservative-radical distinctions based on some biochemical factors. Binary predictors based on 25 randomly chosen biochemical factors from the AAIndex database were applied to the prediction of mutant fitnesses in mutation-scanning experiments (for the full set of predictors, see Supplementary Material, section 3). The AUC is the chance that a randomly chosen mutant designated as “conservative” by the predictor has a higher fitness than one designated as “radical”. Most predictors are more powerful than the ti:tv distinction (AUC = 0.53).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Yet, natural selection has the ability to amplify small differences into major effects. Perhaps a difference with an effect size of AUC = 0.53 might translate into a several-fold bias in terms of evolutionary acceptance.</p><p id="p-32">How do these two relate to each other? The U matrix illustrates this relationship, because values of U scale with evolutionary rates, and U<sub>B</sub> has a known power as a conservative-radical distinction, namely AUC = 0.64. The ratio of U values for conservative replacements relative to radical ones is 2.7. That is, conservative replacements as defined by U<sub>B</sub> are 2.7-fold more likely to be accepted in evolution than radical ones.</p><p id="p-33">This pair of values, AUC = 0.64 and evolutionary bias = 2.7, represents one point in the relationship between evolutionary acceptability and classification power for mutant fitness effects. There is another point where AUC = 0.5 (no power) and evolutionary bias = 1 (no effect). We can fill in the relationship further by randomizing U<sub>B</sub>, as shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. The results show that, when about 75 % of the values are randomized, U<sub>B</sub> has an AUC of 0.53, equal to that of the transition:transversion distinction. This corresponds to an evolutionary bias of 1.3. The confidence interval of AUC from 0.50 to 0.56 for the transition:transversion distinction corresponds to the interval of 1.0 to 1.6 in evolutionary bias. That is, the expected evolutionary effect of the transition:transverion bias is a 1.3-fold bias, with a confidence interval from 1.0 (no effect) to 1.6. This makes it unlikely that selection plays the major role in causing the evolutionary transition:transversion bias, which typically is several-fold favoring transitions.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between power to predict mutant fitnesses and evolutionary effect size. AUC and evolutionary acceptance ratio are shown for increasingly randomized verions of UB. For the unrandomized UB, the power in predicting mutational al effects is AUC = 0.64, and this corresponds to an evolutionary acceptance ratio of 2.7 for conservative versus radical replacements. To estimate the evolutionary acceptance ratio for more modest values of AUC, we can weaken UB by randomly re-assigning &#x201C;conservative&#x201D; or &#x201C;radical&#x201D; labels to an increasingly large fraction of replacement types (200 replicates at each level of randomization). The AUC of 0.53 is reached at about 75 75 % randomization, where the evolutionary effect size is 1.3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991900145" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between power to predict mutant fitnesses and evolutionary effect size. AUC and evolutionary acceptance ratio are shown for increasingly randomized verions of U&lt;sub&gt;B&lt;/sub&gt;. For the unrandomized U&lt;sub&gt;B&lt;/sub&gt;, the power in predicting mutational al effects is AUC = 0.64, and this corresponds to an evolutionary acceptance ratio of 2.7 for conservative versus radical replacements. To estimate the evolutionary acceptance ratio for more modest values of AUC, we can weaken U&lt;sub&gt;B&lt;/sub&gt; by randomly re-assigning &#x201C;conservative&#x201D; or &#x201C;radical&#x201D; labels to an increasingly large fraction of replacement types (200 replicates at each level of randomization). The AUC of 0.53 is reached at about 75 75 % randomization, where the evolutionary effect size is 1.3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F3.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F3.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-34" class="first-child">Relationship between power to predict mutant fitnesses and evolutionary effect size. AUC and evolutionary acceptance ratio are shown for increasingly randomized verions of U<sub>B</sub>. For the unrandomized U<sub>B</sub>, the power in predicting mutational al effects is AUC = 0.64, and this corresponds to an evolutionary acceptance ratio of 2.7 for conservative versus radical replacements. To estimate the evolutionary acceptance ratio for more modest values of AUC, we can weaken U<sub>B</sub> by randomly re-assigning “conservative” or “radical” labels to an increasingly large fraction of replacement types (200 replicates at each level of randomization). The AUC of 0.53 is reached at about 75 75 % randomization, where the evolutionary effect size is 1.3.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Discussion</h3><p id="p-35">Based on a collection of 8 studies that report fitnesses for replacement mutations, we have assessed the prospects for the hypothesis that the conservativeness of replacements via transition accounts for their increased frequency in evolution. Even small studies reveal predictable patterns of amino acid exchangeability, and most have sufficient power to distinguish a binary conservative-vs-radical distinction. However, the same studies typically do not show significant conservativeness of transitions. Overall, the chance of a transition mutation being more fit than a transversion is 53 % (95 % CI, 50 % to 56 %). This effect size is not large compared to that of most biochemical predictors, and is not large enough to explain the several-fold bias toward transition replacements observed in evolutionary studies.</p><p id="p-36">The finding that the conservativeness of transitions is a rather weak effect increases the prospects for the alternative mutational explanation, in which the rate at which new alleles are introduced by transition mutations is several-fold higher than for transversions, and this bias predisposes evolutionary change to happen via transitions.</p><p id="p-37">Though this idea may be familiar, it relates to a rather substantial and unresolved issue in evolutionary genetics, which is the extent to which evolution in nature happens in the “gene pool” regime imagined by the architects of the Modern Synthesis, in the kind of mutation-driven regime imagined by early mutationists and later molecular evolutionists, or something in between (see <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">McCandlish and Stoltzfus 2014</a>). For many, years the mutationist view has suffered from an association with neutral evolution (e.g., the two remain conflated in <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nei 2013</a>). This may explain the ongoing popularity of the conservative transitions hypothesis: a mutation bias is the obvious explanation for transition bias in the evolution of introns and other non-coding sequences, but was not accepted for protein sequences, which are assumed to be “under selection” and thus not subject to such biases. Yet it is clear theoretically that mutation-biased adaptation is possible and, more generally, that mutation and selection can both contribute to orientation or direction in evolution (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Yampolsky and Stoltzfus 2001</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Stoltzfus 2006</a>).</p><p id="p-38">The results presented here also prompt the question of how the lore that transitions are conservative was estabished. In a survey of the literature, we found that, when the alleged conservativeness of transitions is attributed to a source, the source is often <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zhang (2000)</a>, or early works such as <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Fitch (1967)</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Grantham (1974)</a> or <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Vogel and Kopun (1977)</a>. Grantham does not directly address this issue, but a genetic-code-based calculation shows that the mean Grantham distance for transition-mediated replacements is lower than that for transversions, e.g., as indicated in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> of <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xia, et al (1998)</a>. The study by Vogel and Kopun is often cited as evidence for the conservative transitions hypothesis, because they present a calculation that, for 3 different biochemical measures, suggests that transitions are more conservative. Nevertheless, Vogel and Kopun themselves favored a mutational explanation (see hypothesis 3 on p. 179), arguing that the effect size is too small for conservativeness of transitions to account for the evolutionary bias.</p><p id="p-39">These prior studies are inconclusive for 2 very general reasons. The first is that none reports an effect size sufficient to account for the evolutionary bias. For instance, <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Zhang's (2000)</a> analysis of 3 possible conservative:radical distinctions finds that the distinction based on <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Miyata, et al (1979)</a> yields the largest evolutionary effect size, which is a 2-fold effect, i.e., radical replacements are roughly half as likely to accumulate, relative to null expectations. However, though the effect of conservativeness is strong, the link reported between transitions and conservativeness is weak. According to <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Zhang (2000)</a>, the chance that a transition is conservative by Miyata's measure is 35 %, compared to 33 % for transversions, a proportional difference of only 6 % (i.e., 2 / 33 = 0.06). Miyata-conservativeness may be a 2-fold evolutionary effect, but if transitions are only 6 % more Miyata-conservative than transversions, the overall bias will be far less than 2-fold.</p><p id="p-40">Second, none of these works escapes the kind of logical circularity pointed out by di <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Giulio (2001</a>; see also <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Yampolsky and Stoltzfus 2005</a>), in which a measure of evolutionary tendencies (e.g., PAM, BLOSUM) is invoked to argue for effects of selection rather than mutation, ignoring the fact that the pattern of evolution is influenced (to an unknown degree) by mutational effects. This is an indirect form of the Panglossian fallacy, i.e., it is formally a fallacy of arguing that transitions are more adaptive simply because they happen more often, without inquiring into why they happen more often.</p><p id="p-41">The fallacy is not avoided by invoking biochemical factors. The popular composite indices of “biochemical” distance constructed by <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Grantham (1974)</a> and <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Miyata, et al (1979)</a> are based on choosing biochemical factors that fit well with observed evolutionary patterns from earlier protein comparisons. Likewise, all 3 “biochemical” measures used by <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Vogel and Kopun (1977)</a> are based on fitting to protein comparisons. The problem with this approach is suggested by <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, which shows the conservativeness of transitions for a random sample of the biochemical indices in the AAindex database (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Kawashima and Kanehisa 2000</a>). Roughly half make transitions seem conservative, and the other half make them seem radical.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The relative advantage of transitions as indicated by a random sample of biochemical factors. Each biochemical attribute of an amino acid is converted to a pairwise similarity measure, so that each possible amino acid replacement has a similarity score. Here the AUC is the chance that a replacement due to a randomly chosen transition (from the pool of actual mutants from the 8 studies) has a higher similarity score (for the given biochemical attribute) than a randomly chosen transversion. The resulting distribution indicates that many biochemical factors make transitions seem more conservative (AUC &gt; 0.5), but a roughly equal number make transversions seem more conservative (AUC &lt; 0.5). For the full set of predictors, see Supplementary Material, section 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-991900145" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The relative advantage of transitions as indicated by a random sample of biochemical factors. Each biochemical attribute of an amino acid is converted to a pairwise similarity measure, so that each possible amino acid replacement has a similarity score. Here the AUC is the chance that a replacement due to a randomly chosen transition (from the pool of actual mutants from the 8 studies) has a higher similarity score (for the given biochemical attribute) than a randomly chosen transversion. The resulting distribution indicates that many biochemical factors make transitions seem more conservative (AUC &gt; 0.5), but a roughly equal number make transversions seem more conservative (AUC &lt; 0.5). For the full set of predictors, see Supplementary Material, section 3.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F4.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F4.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/09/28/027722/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-42" class="first-child">The relative advantage of transitions as indicated by a random sample of biochemical factors. Each biochemical attribute of an amino acid is converted to a pairwise similarity measure, so that each possible amino acid replacement has a similarity score. Here the AUC is the chance that a replacement due to a randomly chosen transition (from the pool of actual mutants from the 8 studies) has a higher similarity score (for the given biochemical attribute) than a randomly chosen transversion. The resulting distribution indicates that many biochemical factors make transitions seem more conservative (AUC &gt; 0.5), but a roughly equal number make transversions seem more conservative (AUC &lt; 0.5). For the full set of predictors, see Supplementary Material, section 3.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">As shown earlier (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), this is not because biochemical indices are generally poor predictors of exchangeability. Instead, among many moderately powerful predictors, there are ones that make transitions seem favorable, and others that make transversions seem favorable. Thus, converting evolutionary patterns into biochemical descriptors before re-applying them to the analysis of evolutionary patterns does not allow one to escape a logical circularity: if early molecular evolutionists had found that transversions dominate evolution, they could have rationalized this pattern by appeal to biochemistry just as easily as they rationalized the observed dominance of transitions.</p></div></div><div class="section" id="sec-7"><h2 class="">Materials and Methods</h2><div id="sec-8" class="subsection"><h3>Identification of studies and data sets for inclusion</h3><p id="p-44">An initial core set of studies (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Sanjuan, et al. 2004</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Carrasco, et al. 2007</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Roscoe, et al. 2013</a>) was expanded by including other work cited by these studies. Then this set was expanded further by open-ended searches based on keywords or by tracking citations. In general, no text-based search does a good job of recovering mutation-scanning studies of the desired type. Narrow searches (e.g., “distribution of mutational effects”) implicate only a fraction of true positives and did little to expand the core set of studies; broad searches (e.g., “mutation” plus “fitness”) implicate so many false positives that they are impractical and were abandoned. Most relevant studies cite the pioneering work of <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Sanjuan (2004)</a> or the seminal review by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Eyre-Walker and Keightley (2007)</a>. Candidate studies identified in this manner were screened for appropriateness, ultimately resulting in the 8 studies listed in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. The search covered literature published through December, 2014 and does not include more recent studies.</p><p id="p-45">As noted in Materials and Methods, we restricted our attention to studies with (1) a size of at least 20 replacement mutants; (2) measures of growth (fitness) rather than simply biochemical activities; and (3) a random or arbitrary set of mutants. Most excluded studies of mutational effects have only a few mutants, or they report effects on binding or activity (but not on fitness), or they are focused on achieving particular outcomes rather than exploring a random set of variants, or they use deep sequencing to identify and quantify mutants, an approach that introduces uncontrolled nucleotide biases (Supplementary Material, section 2).</p></div><div id="sec-9" class="subsection"><h3>Processing and management of mutation data</h3><p id="p-46">Starting from raw data tables supplied by authors (either directly, or via published supplements), all further processing and analysis steps were encoded in scripts. For each study used here, there is a an R-Markdown (Rmd) file that (when executed in an appropriate environment, such as RStudio) describes and executes the steps (e.g., cleaning, re-coding, sequence integration) to convert input data into a standard tabular form in which there is a single row describing each mutant and its effects. The figures and tables in this paper are generated by further Rmd scripts that operate on the standardized input data.</p></div><div id="sec-10" class="subsection"><h3>Other data sources</h3><p id="p-47">Values of U are from <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Tang, et al. (2004)</a>, and values of EX are from published supplements. Biochemical indices from the AAIndex database were accessed via the Interpol package (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Heider 2012</a>) and custom R code.</p><p id="p-48">Note that, although AAindex lists 533 biochemical indices, less than half are pure biochemical indices. The others are based on some method of counting occurrences in naturally evolved proteins, e.g., frequency with which an amino is found in a helix. Because the distribution of an amino acid in an evolving set of natural proteins will depend on the distributions of its closest mutational neighbors, such measures are not mutationally unbiased. They were removed using a custom list of name exclusion patterns (“[fF]requenc”, “[pP]reference”, “[cC]ompositi”, “[pP]ropensit”, “[dD]istribution”, “[iI]nformation”, “[wW]eights”, “[oO]ccurrence”, “[Pp]roportion”, “probability”, “mutability”, “Geisow”, “Janin”), resulting in a set of 247 indices.</p></div><div id="sec-11" class="subsection"><h3>Tests of power and effect</h3><p id="p-49">The results presented here rely mainly on standard statistical procedures. When P values are reported in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> for a linear predictor, this is from the t-test in the built-in linear model (lm) function in R. When P values are reported for binary predictors in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">3</a>, this is based on the Wilcoxon-Mann-Whitney test as implemented in the “wilcox.test” function of the R “stats” package, using a one-sided test. When confidence intervals are given on an AUC value, this is based on re-sampling using 400 bootstrap replicates.</p><p id="p-50">The only unfamiliar methods involve the use of binary predictors. To convert a biochemical index C to a binary distinction, we first convert it to a pairwise similarity by the formula Sij = 1 − abs(C<sub>i</sub> − C<sub>j</sub>) / max, where max is the maximum absolute difference. Converting a continuous measure of similarity into a binary measure is a simple matter of assigning all values above a particular quantile to the “conservative” class, and the rest to the “radical” class. To ensure that a constructed predictor is comparable to the transition:transversion distinction, the threshold is chosen so that the conservative class is the same size as the transition class in the data to be tested.</p><p id="p-51">As explained above, we can define a measure of effect-size with intuitive properties that we designate as AUC, based on an application of ROC analysis that may not be obvious. In ROC analysis of a binary classifier, each instance has a binary state (e.g., disease vs. non-disease), and the classifier makes a ranking of instances and predicts the binary state based on a threshold. The ROC curve plots the true-positive rate against the false-negative rate, and the area under this curve is equivalent to the chance that a randomly chosen positive instance is ranked higher than a randomly chosen negative one (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hanley and McNeil 1982</a>). If we treat the fitness study as the classifier that supplies a ranking for each mutant, and the conservative-radical distinction as the binary state of a mutant, then the AUC is the chance that a mutant of a nominally conservative type has a higher fitness than a randomly chosen mutant of a nominally radical type. The relationship of AUC to the Wilcoxon-Mann-Whitney test is explained by Manley (1982). Calculating AUC from the test statistic is an algebraic conversion based on the formula AUC = (pairs – WMW_statistic(x, y)) / pairs, where x and y are vectors representing the two samples, and pairs = length(x) * length(y). This formula applies specifically to wilcox.test in the R “stats” package (some other implementations define the test statistic in a different way).</p><p id="p-52">Note that converting fitnesses to within-study quantiles allows us to compare studies, and allows us to combine data for across-study tests. The use of quantiles rather than absolute fitnesses does not have any effect on a within-study AUC or Wilcoxon-Mann-Whitney test, which is non-parametric.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-53">We thank Greg Babbitt and David McCandlish for comments, and Ashley Acevedo, Rafael Sanjuan and Nicholas Wu for help in acquiring and interpreting data. The identification of any specific commercial products is for the purpose of specifying a protocol, and does not imply a recommendation or endorsement by the National Institute of Standards and Technology.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> Department of Evolution, Ecology and Organismal Biology, The Ohio State University, Lima, OH 45804</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.1" data-doi="10.1038/nature12861"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Acevedo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Mutational and fitness landscapes of an RNA virus revealed through population sequencing</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D505%26rft.spage%253D686%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12861%26rft_id%253Dinfo%253Apmid%252F24284629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24284629&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330321000042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.2" data-doi="10.1093/molbev/msu173"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Bloom</span> <span class="cit-name-surname">JD.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">An experimentally determined evolutionary model dramatically improves phylogenetic fit</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1956</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu173%26rft_id%253Dinfo%253Apmid%252F24859245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24859245&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339927800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.3" data-doi="10.1016/j.jviromet.2006.09.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carrasco</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daros</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agudelo-Romero</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">A real-time RT-PCR assay for quantifying the fitness of tobacco etch virus in competition experiments</span>. <abbr class="cit-jnl-abbrev">J Virol Methods</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.stitle%253DJ%2BVirol%2BMethods%26rft.aulast%253DCarrasco%26rft.auinit1%253DP.%26rft.volume%253D139%26rft.issue%253D2%26rft.spage%253D181%26rft.epage%253D188%26rft.atitle%253DA%2Breal-time%2BRT-PCR%2Bassay%2Bfor%2Bquantifying%2Bthe%2Bfitness%2Bof%2Btobacco%2Betch%2Bvirus%2Bin%2Bcompetition%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2006.09.020%26rft_id%253Dinfo%253Apmid%252F17092574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2006.09.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17092574&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244090300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.4" data-doi="10.1006/jtbi.2000.2206"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Di Giulio</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2001</span>. <span class="cit-article-title">The origin of the genetic code cannot be studied using measurements based on the PAM matrix because this matrix reflects the code itself, making any such analyses tautologous</span>. <abbr class="cit-jnl-abbrev">J Theor Biol</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DDi%2BGiulio%26rft.auinit1%253DM.%26rft.volume%253D208%26rft.issue%253D2%26rft.spage%253D141%26rft.epage%253D144%26rft.atitle%253DThe%2Borigin%2Bof%2Bthe%2Bgenetic%2Bcode%2Bcannot%2Bbe%2Bstudied%2Busing%2Bmeasurements%2Bbased%2Bon%2Bthe%2BPAM%2Bmatrix%2Bbecause%2Bthis%2Bmatrix%2Breflects%2Bthe%2Bcode%2Bitself%252C%2Bmaking%2Bany%2Bsuch%2Banalyses%2Btautologous.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.2000.2206%26rft_id%253Dinfo%253Apmid%252F11162059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.2000.2206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11162059&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166507000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.5" data-doi="10.1371/journal.pgen.1000742"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Domingo-Calap</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuan</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The fitness effects of random mutations in single-stranded DNA and RNA bacteriophages</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">e1000742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDomingo-Calap%26rft.auinit1%253DP.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De1000742%26rft.epage%253De1000742%26rft.atitle%253DThe%2Bfitness%2Beffects%2Bof%2Brandom%2Bmutations%2Bin%2Bsingle-stranded%2BDNA%2Band%2BRNA%2Bbacteriophages.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000742%26rft_id%253Dinfo%253Apmid%252F19956760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19956760&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.6" data-doi="10.1038/nrg2146"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">PD.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">The distribution of fitness effects of new mutations</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D610%26rft.epage%253D618%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bfitness%2Beffects%2Bof%2Bnew%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2146%26rft_id%253Dinfo%253Apmid%252F17637733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637733&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248170700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.7" data-doi="10.1534/genetics.106.064634"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">MT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burch</span>  <span class="cit-name-given-names">CL.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">High frequency of mutations that expand the host range of an RNA virus</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064634%26rft_id%253Dinfo%253Apmid%252F17409090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8yLzEwMTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wOS8yOC8wMjc3MjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.8" data-doi="10.1093/molbev/msu081"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firnberg</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Labonte</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ostermeier</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">A Comprehensive, High-Resolution Map of a Gene’s Fitness Landscape</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu081%26rft_id%253Dinfo%253Apmid%252F24567513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567513&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000337067400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.9" data-doi="10.1016/0022-2836(67)90317-8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Fitch</span> <span class="cit-name-surname">WM.</span></span> <span class="cit-pub-date">1967</span>. <span class="cit-article-title">Evidence Suggesting a Non-random Character to Nucleotide Replacements in Naturally Occurring Mutations</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DFitch%26rft.auinit1%253DW.%2BM.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D499%26rft.epage%253D507%26rft.atitle%253DEvidence%2Bsuggesting%2Ba%2Bnon-random%2Bcharacter%2Bto%2Bnucleotide%2Breplacements%2Bin%2Bnaturally%2Boccurring%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252867%252990317-8%26rft_id%253Dinfo%253Apmid%252F6029740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(67)90317-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6029740&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19679646100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.10" data-doi="10.1126/science.185.4154.862"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grantham</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">1974</span>. <span class="cit-article-title">Amino acid difference formula to help explain protein evolution</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">862</span>–<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGrantham%26rft.auinit1%253DR.%26rft.volume%253D185%26rft.issue%253D4154%26rft.spage%253D862%26rft.epage%253D864%26rft.atitle%253DAmino%2BAcid%2BDifference%2BFormula%2Bto%2BHelp%2BExplain%2BProtein%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.185.4154.862%26rft_id%253Dinfo%253Apmid%252F4843792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIxODUvNDE1NC84NjIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wOS8yOC8wMjc3MjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.11" data-doi="10.1148/radiology.143.1.7063747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hanley</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">BJ.</span></span> <span class="cit-pub-date">1982</span>. <span class="cit-article-title">The meaning and use of the area under a receiver operating characteristic (ROC) curve</span>. <abbr class="cit-jnl-abbrev">Radiology</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRadiology%26rft.stitle%253DRadiology%26rft.aulast%253DHanley%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D143%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D36%26rft.atitle%253DThe%2Bmeaning%2Band%2Buse%2Bof%2Bthe%2Barea%2Bunder%2Ba%2Breceiver%2Boperating%2Bcharacteristic%2B%2528ROC%2529%2Bcurve%26rft_id%253Dinfo%253Adoi%252F10.1148%252Fradiology.143.1.7063747%26rft_id%253Dinfo%253Apmid%252F7063747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1148/radiology.143.1.7063747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7063747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982NG95400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-027722v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heider</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-chapter-title">Interpolation of amino acid sequences</span>. Version 1.3.1: <span class="cit-publ-loc">University of Duisburg-Essen</span>, <span class="cit-publ-name">Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.13" data-doi="10.1073/pnas.1215206110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacquier</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birgy</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Le Nagard</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mechulam</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glodt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bercot</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poulain</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barnaud</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Capturing the mutational landscape of the betalactamase TEM-1</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">13067</span>–<span class="cit-lpage">13072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215206110%26rft_id%253Dinfo%253Apmid%252F23878237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzMyLzEzMDY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.14" data-doi="10.1016/j.ygeno.2006.06.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Mutational spectrum in the recent human genome inferred by single nucleotide polymorphisms</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">527</span>–<span class="cit-lpage">534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DJiang%26rft.auinit1%253DC.%26rft.volume%253D88%26rft.issue%253D5%26rft.spage%253D527%26rft.epage%253D534%26rft.atitle%253DMutational%2Bspectrum%2Bin%2Bthe%2Brecent%2Bhuman%2Bgenome%2Binferred%2Bby%2Bsingle%2Bnucleotide%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2006.06.003%26rft_id%253Dinfo%253Apmid%252F16860534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2006.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16860534&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241563400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.15" data-doi="10.1093/nar/28.1.374"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">AAindex: amino acid index database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKawashima%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D374%26rft.epage%253D374%26rft.atitle%253DAAindex%253A%2BAmino%2BAcid%2Bindex%2Bdatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.374%26rft_id%253Dinfo%253Apmid%252F10592278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592278&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300112&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.16" data-doi="10.1371/journal.pgen.0030022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">RA.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Transition-transversion bias is not universal: a counter example from grasshopper pseudogenes</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">e22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKeller%26rft.auinit1%253DI.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253De22%26rft.epage%253De22%26rft.atitle%253DTransition-transversion%2Bbias%2Bis%2Bnot%2Buniversal%253A%2Ba%2Bcounter%2Bexample%2Bfrom%2Bgrasshopper%2Bpseudogenes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030022%26rft_id%253Dinfo%253Apmid%252F17274688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17274688&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.17" data-doi="10.1016/0022-2836(90)90126-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kleina</span>  <span class="cit-name-given-names">LG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">JH.</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Genetic studies of the lac repressor. XIII. Extensive amino acid replacements generated by the use of natural and synthetic nonsense suppressors</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKleina%26rft.auinit1%253DL.%2BG.%26rft.volume%253D212%26rft.issue%253D2%26rft.spage%253D295%26rft.epage%253D318%26rft.atitle%253DGenetic%2Bstudies%2Bof%2Bthe%2Blac%2Brepressor.%2BXIII.%2BExtensive%2Bamino%2Bacid%2Breplacements%2Bgenerated%2Bby%2Bthe%2Buse%2Bof%2Bnatural%2Band%2Bsynthetic%2Bnonsense%2Bsuppressors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252890%252990126-7%26rft_id%253Dinfo%253Apmid%252F2157024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(90)90126-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2157024&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CX59100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">1996</span>. <span class="cit-article-title">Patterns of nucleotide substitution in mitochondrial protein coding genes of vertebrates</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D143%26rft.issue%253D1%26rft.spage%253D537%26rft.epage%253D548%26rft.atitle%253DPatterns%2Bof%2BNucleotide%2BSubstitution%2Bin%2BMitochondrial%2BProtein%2BCoding%2BGenes%2Bof%2BVertebrates%26rft_id%253Dinfo%253Apmid%252F8722802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQzLzEvNTM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-027722v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W-H</span></span>. <span class="cit-pub-date">1997</span>. <span class="cit-source">Molecular Evolution</span>. <span class="cit-publ-loc">Sunderland, Mass</span>.: <span class="cit-publ-name">Sinauer</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-027722v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">C-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C-I</span></span>. <span class="cit-pub-date">1985</span>. <span class="cit-chapter-title">Evolution of DNA Sequences</span>. In: <span class="cit-auth"><span class="cit-name-surname">MacIntyre</span>  <span class="cit-name-given-names">RJ</span></span>, editor. <span class="cit-source">Molecular Evolutionary Genetics</span>. <span class="cit-publ-loc">New York</span>: <span class="cit-publ-name">Plenum</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.21" data-doi="10.1073/pnas.0912629107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Rate, molecular spectrum, and consequences of human mutation</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0912629107%26rft_id%253Dinfo%253Apmid%252F20080596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMy85NjEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wOS8yOC8wMjc3MjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.22" data-doi="10.1534/genetics.110.123083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">MacLean</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perron</span>  <span class="cit-name-given-names">GG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Diminishing returns from beneficial mutations and pervasive epistasis shape the fitness landscape for rifampicin resistance in Pseudomonas aeruginosa</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1345</span>–<span class="cit-lpage">1354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.123083%26rft_id%253Dinfo%253Apmid%252F20876562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni80LzEzNDUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wOS8yOC8wMjc3MjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.23" data-doi="10.1086/677571"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McCandlish</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoltzfus</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Modeling evolution using the probability of fixation: history and implications</span>. <abbr class="cit-jnl-abbrev">Quarterly Review of Biology</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuarterly%2BReview%2Bof%2BBiology%26rft.volume%253D89%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1086%252F677571%26rft_id%253Dinfo%253Apmid%252F25195318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/677571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25195318&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.24" data-doi="10.1534/genetics.110.121400"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">HA.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Mutational effects and population dynamics during viral adaptation challenge current models</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121400%26rft_id%253Dinfo%253Apmid%252F21041559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzEvMTg1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.25" data-doi="10.1007/BF01732340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyazawa</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yasunaga</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">1979</span>. <span class="cit-article-title">Two types of amino acid substitutions in protein evolution</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMiyata%26rft.auinit1%253DT.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D219%26rft.epage%253D236%26rft.atitle%253DTwo%2Btypes%2Bof%2Bamino%2Bacid%2Bsubstitutions%2Bin%2Bprotein%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01732340%26rft_id%253Dinfo%253Apmid%252F439147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01732340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=439147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GQ04700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.26" data-doi="10.1534/genetics.105.049916"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">IV</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">BS.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Variation in mutation dynamics across the maize genome as a function of regional and flanking base composition</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049916%26rft_id%253Dinfo%253Apmid%252F16219784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNTY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-027722v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-publ-loc">Mutation-Driven Evolution</span>: <span class="cit-publ-name">Oxford University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.28" data-doi="10.1534/genetics.110.115162"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peris</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nebot</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuan</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Distribution of fitness effects caused by single-nucleotide substitutions in bacteriophage f1</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115162%26rft_id%253Dinfo%253Apmid%252F20382832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNjAzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.29" data-doi="10.1073/pnas.96.4.1475"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">DL.</span></span> <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Patterns of nucleotide substitution in Drosophila and mammalian genomes</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1475</span>–<span class="cit-lpage">1479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPetrov%26rft.auinit1%253DD.%2BA.%26rft.volume%253D96%26rft.issue%253D4%26rft.spage%253D1475%26rft.epage%253D1479%26rft.atitle%253DPatterns%2Bof%2Bnucleotide%2Bsubstitution%2Bin%2BDrosophila%2Band%2Bmammalian%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.4.1475%26rft_id%253Dinfo%253Apmid%252F9990048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni80LzE0NzUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wOS8yOC8wMjc3MjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.30" data-doi="10.1128/JVI.02451-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rihn</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Uneven Genetic Robustness of HIV-1 Integrase</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02451-14%26rft_id%253Dinfo%253Apmid%252F25339768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg5LzEvNTUyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.31" data-doi="10.1371/journal.ppat.1003461"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rihn</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alim</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhella</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rixon</span>  <span class="cit-name-given-names">FJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Extreme genetic fragility of the HIV-1 capsid</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">e1003461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003461%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003461%26rft_id%253Dinfo%253Apmid%252F23818857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23818857&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.32" data-doi="10.1016/j.jmb.2013.01.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roscoe</span>  <span class="cit-name-given-names">BP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thayer</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeldovich</span>  <span class="cit-name-given-names">KB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fushman</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bolon</span>  <span class="cit-name-given-names">DN.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Analyses of the effects of all ubiquitin point mutants on yeast growth rate</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">1363</span>–<span class="cit-lpage">1377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BBiol%26rft.volume%253D425%26rft.spage%253D1363%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2013.01.032%26rft_id%253Dinfo%253Apmid%252F23376099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2013.01.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23376099&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.33" data-doi="10.1093/molbev/msg113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Patterns of transitional mutation biases within and among mammalian genomes</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg113%26rft_id%253Dinfo%253Apmid%252F12716982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12716982&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183138500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.34" data-doi="10.1073/pnas.0400146101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuan</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">The distribution of fitness effects caused by single-nucleotide substitutions in an RNA virus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">8396</span>–<span class="cit-lpage">8401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0400146101%26rft_id%253Dinfo%253Apmid%252F15159545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzIyLzgzOTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wOS8yOC8wMjc3MjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schaaper</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">RL.</span></span> <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Spontaneous mutation in the Escherichia coli lacI gene</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchaaper%26rft.auinit1%253DR.%2BM.%26rft.volume%253D129%26rft.issue%253D2%26rft.spage%253D317%26rft.epage%253D326%26rft.atitle%253DSpontaneous%2Bmutation%2Bin%2Bthe%2BEscherichia%2Bcoli%2BlacI%2Bgene.%26rft_id%253Dinfo%253Apmid%252F1660424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzIvMzE3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.36" data-doi="10.1371/journal.pgen.1002783"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schenk</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szendro</span>  <span class="cit-name-given-names">IG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krug</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Visser</span>  <span class="cit-name-given-names">JA.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Quantifying the adaptive potential of an antibiotic resistance enzyme</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">e1002783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSchenk%26rft.auinit1%253DM.%2BF.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002783%26rft.epage%253De1002783%26rft.atitle%253DQuantifying%2Bthe%2Badaptive%2Bpotential%2Bof%2Ban%2Bantibiotic%2Bresistance%2Benzyme.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002783%26rft_id%253Dinfo%253Apmid%252F22761587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002783&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22761587&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.37" data-doi="10.1534/genetics.113.151670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">MW.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Rates and genomic consequences of spontaneous mutational events in Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151670%26rft_id%253Dinfo%253Apmid%252F23733788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.38" data-doi="10.2307/2533094"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lake</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-pub-date">1997</span>. <span class="cit-article-title">Inference for phylogenies under a hybrid parsimony method: evolutionary-symmetric transversion parsimony</span>. <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DSinsheimer%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D38%26rft.atitle%253DInference%2Bfor%2Bphylogenies%2Bunder%2Ba%2Bhybrid%2Bparsimony%2Bmethod%253A%2Bevolutionary-symmetric%2Btransversion%2Bparsimony.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533094%26rft_id%253Dinfo%253Apmid%252F9147592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9147592&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.39" data-doi="10.1093/molbev/msl064"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stoltzfus</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Mutation-Biased Adaptation in a Protein NK Model</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1852</span>–<span class="cit-lpage">1862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl064%26rft_id%253Dinfo%253Apmid%252F16857856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16857856&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240586600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.40" data-doi="10.1093/molbev/msh158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wyckoff</span>  <span class="cit-name-given-names">GJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">CI.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">A universal evolutionary index for amino acid changes</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1548</span>–<span class="cit-lpage">1556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh158%26rft_id%253Dinfo%253Apmid%252F15140949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msh158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15140949&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222961400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thyagarajan</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">JD.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">The inherent mutational tolerance and antigenic evolvability of influenza hemagglutinin</span>. <abbr class="cit-jnl-abbrev">Elife</abbr>:<span class="cit-fpage">e03300</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.42" data-doi="10.1007/BF01653962"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-pub-date">1972</span>. <span class="cit-article-title">Non-randomness of base replacement in point mutation</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DVogel%26rft.auinit1%253DF.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D334%26rft.epage%253D367%26rft.atitle%253DNon-randomness%2Bof%2Bbase%2Breplacement%2Bin%2Bpoint%2Bmutation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01653962%26rft_id%253Dinfo%253Apmid%252F4681234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01653962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4681234&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.43" data-doi="10.1007/BF01732746"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kopun</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">1977</span>. <span class="cit-article-title">Higher frequencies of transitions among point mutations</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DVogel%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D180%26rft.atitle%253DHigher%2Bfrequencies%2Bof%2Btransitions%2Bamong%2Bpoint%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01732746%26rft_id%253Dinfo%253Apmid%252F864721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01732746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=864721&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">1996</span>. <span class="cit-article-title">The excess of transitions among nucleotide substitutions: new methods of estimating transition bias underscore its significance</span>. <abbr class="cit-jnl-abbrev">TREE</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTREE%26rft.volume%253D11%26rft.spage%253D158%26rft_id%253Dinfo%253Apmid%252F21237791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21237791&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.45" data-doi="10.1038/srep04942"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">AP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Mawsawi</span>  <span class="cit-name-given-names">LQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">IH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">CY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">High-throughput profiling of influenza A virus hemagglutinin gene at single-nucleotide resolution</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">4942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D4%26rft.spage%253D4942%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep04942%26rft_id%253Dinfo%253Apmid%252F24820965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep04942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24820965&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.46" data-doi="10.1093/oxfordjournals.molbev.a025930"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">1998</span>. <span class="cit-article-title">The rate heterogeneity of nonsynonymous substitutions in mammalian mitochondrial genes</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DXia%26rft.auinit1%253DX%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D336%26rft.epage%253D344%26rft.atitle%253DThe%2Brate%2Bheterogeneity%2Bof%2Bnonsynonymous%2Bsubstitutions%2Bin%2Bmammalian%2Bmitochondrial%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025930%26rft_id%253Dinfo%253Apmid%252F9501500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a025930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9501500&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072361600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.47" data-doi="10.1046/j.1525-142x.2001.003002073.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yampolsky</span>  <span class="cit-name-given-names">LY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoltzfus</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Bias in the introduction of variation as an orienting factor in evolution</span>. <abbr class="cit-jnl-abbrev">Evol Dev</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DYampolsky%26rft.auinit1%253DL.%2BY.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D73%26rft.epage%253D83%26rft.atitle%253DBias%2Bin%2Bthe%2Bintroduction%2Bof%2Bvariation%2Bas%2Ban%2Borienting%2Bfactor%2Bin%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1525-142x.2001.003002073.x%26rft_id%253Dinfo%253Apmid%252F11341676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1525-142x.2001.003002073.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11341676&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168050300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.48" data-doi="10.1534/genetics.104.039107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yampolsky</span>  <span class="cit-name-given-names">LY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoltzfus</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">The exchangeability of amino acids in proteins</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1459</span>–<span class="cit-lpage">1472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039107%26rft_id%253Dinfo%253Apmid%252F15944362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC80LzE0NTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wOS8yOC8wMjc3MjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Rates of conservative and radical nonsynonymous nucleotide substitutions in mammalian nuclear genes</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D50%26rft.issue%253D1%26rft.spage%253D56%26rft.epage%253D68%26rft.atitle%253DRates%2Bof%2Bconservative%2Band%2Bradical%2Bnonsynonymous%2Bnucleotide%2Bsubstitutions%2Bin%2Bmammalian%2Bnuclear%2Bgenes.%26rft_id%253Dinfo%253Apmid%252F10654260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10654260&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088446900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.50" data-doi="10.1186/1471-2148-4-21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">YP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boerwinkle</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">YX.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Moderate mutation rate in the SARS coronavirus genome and its implications</span>. <abbr class="cit-jnl-abbrev">BMC Evol Biol</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DZhao%26rft.auinit1%253DZ.%26rft.volume%253D4%26rft.spage%253D21%26rft.epage%253D21%26rft.atitle%253DModerate%2Bmutation%2Brate%2Bin%2Bthe%2BSARS%2Bcoronavirus%2Bgenome%2Band%2Bits%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-4-21%26rft_id%253Dinfo%253Apmid%252F15222897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-4-21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15222897&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F09%2F28%2F027722.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-027722v1.51" data-doi="10.1073/pnas.1323011111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">YO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">DA.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Precise estimates of mutation rate and spectrum in yeast</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">111</span>:<span class="cit-fpage">E2310</span>–<span class="cit-lpage">2318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323011111%26rft_id%253Dinfo%253Apmid%252F24847077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzIyL0UyMzEwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDkvMjgvMDI3NzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/027722v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F027722v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/027730v1" title="Investigation of the cellular reprogramming phenomenon referred to as stimulus-triggered acquisition of pluripotency (STAP)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/023523v2" title="Estimating directed connectivity from cortical recordings and reconstructed sources" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;September 28, 2015.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/027722v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/027722v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/027722v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/027722v1" class="active" data-icon-position="" data-hide-link-title="0">On the causes of evolutionary transition:transversion bias</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/7417" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Bt2IvnNJx9743u9zy_zkIt3O-MZnqj89Cz_Khfes3aM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="7417" id="node7417--2" data-pisa="biorxiv;027722v1" data-pisa-master="biorxiv;027722" data-seqnum="7417" data-apath="/biorxiv/early/2015/09/28/027722.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">On the causes of evolutionary transition:transversion bias</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Arlin</span> <span class="nlm-surname">Stoltzfus</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ryan W.</span> <span class="nlm-surname">Norris</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">027722; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/027722 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/027722v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;title=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;title=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;text=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;title=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;t=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;title=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;title=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="7417" id="node7417--3" data-pisa="biorxiv;027722v1" data-pisa-master="biorxiv;027722" data-seqnum="7417" data-apath="/biorxiv/early/2015/09/28/027722.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">On the causes of evolutionary transition:transversion bias</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Arlin</span> <span class="nlm-surname">Stoltzfus</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ryan W.</span> <span class="nlm-surname">Norris</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">027722; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/027722 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias&amp;rft.volume&amp;rft.issue&amp;rft.date=2015-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Stoltzfus%2C+Arlin&amp;rft.au=Norris%2C+Ryan+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/7417/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/7417/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/7417/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/7417/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/7417/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/7417/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/7417/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/7417/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/7417/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/7417/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/7417/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/7417/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;count=horizontal&amp;via=&amp;text=On%20the%20causes%20of%20evolutionary%20transition%3Atransversion%20bias&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/027722v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/027722v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/027722v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
