<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/05/07/315366.atom" />
<meta name="HW.pisa" content="biorxiv;315366v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding" />
<meta name="DC.Identifier" content="10.1101/315366" />
<meta name="DC.Date" content="2018-05-07" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="One unexplored aspect of HIV-1 genetic architecture is how codon choice influences population diversity and evolvability. Here we compared the development of HIV-1 resistance to protease inhibitors (PIs) between wild-type (WT) virus and a synthetic virus (MAX) carrying a codon-pair re-engineered protease sequence including 38 (13%) synonymous mutations. WT and MAX viruses showed indistinguishable replication in MT-4 cells or PBMCs. Both viruses were subjected to serial passages in MT-4 cells with selective pressure from the PIs atazanavir (ATV) and darunavir (DRV). After 32 successive passages, both the WT and MAX viruses developed phenotypic resistance to PIs (IC50 14.6 ± 5.3 and 21.2 ± 9 nM for ATV, and 5. 9 ± 1.0 and 9.3 ± 1.9 for DRV, respectively). Ultra-deep sequence clonal analysis revealed that both viruses harbored previously described resistance mutations to ATV and DRV. However, the WT and MAX virus proteases showed different resistance variant repertoires, with the G16E and V77I substitutions observed only in WT, and the L33F, S37P, G48L, Q58E/K, and L89I substitutions detected only in MAX. Remarkably, G48L and L89I are rarely found in vivo in PI-treated patients. The MAX virus showed significantly higher nucleotide and amino acid diversity of the propagated viruses with and without PIs ( P &lt; 0.0001), suggesting higher selective pressure for change in this recoded virus. Our results indicate that HIV-1 protease position in sequence space delineates the evolution of its mutant spectra. Nevertheless, the investigated synonymously recoded variant showed mutational robustness and evolvability similar to the WT virus.

IMPORTANCE Large-scale synonymous recoding of virus genomes is a new tool for exploring various aspects of virus biology. Synonymous virus genome recoding can be used to investigate how a virus’s position in sequence space defines its mutant spectrum, evolutionary trajectory, and pathogenesis. In this study, we evaluated how synonymous recoding of the human immunodeficiency virus type 1 (HIV-1) protease impacts the development of protease inhibitor (PI) resistance. HIV-1 protease is a main target of current antiretroviral therapies. Our present results demonstrate that the wild-type (WT) virus and the virus with the recoded protease exhibited different patterns of resistance mutations after PI treatment. Nevertheless, the developed PI resistance phenotype was indistinguishable between the recoded virus and the WT virus, suggesting that the synonymously recoded protease HIV-1 and the WT protease virus were equally robust and evolvable." />
<meta name="DC.Contributor" content="Maria Nevot" />
<meta name="DC.Contributor" content="Ana Jordan-Paiz" />
<meta name="DC.Contributor" content="Glòria Martrus" />
<meta name="DC.Contributor" content="Cristina Andrés" />
<meta name="DC.Contributor" content="Damir García-Cehic" />
<meta name="DC.Contributor" content="Josep Gregori" />
<meta name="DC.Contributor" content="Sandra Franco" />
<meta name="DC.Contributor" content="Josep Quer" />
<meta name="DC.Contributor" content="Miguel Angel Martinez" />
<meta name="article:published_time" content="2018-05-07" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;One unexplored aspect of HIV-1 genetic architecture is how codon choice influences population diversity and evolvability. Here we compared the development of HIV-1 resistance to protease inhibitors (PIs) between wild-type (WT) virus and a synthetic virus (MAX) carrying a codon-pair re-engineered protease sequence including 38 (13%) synonymous mutations. WT and MAX viruses showed indistinguishable replication in MT-4 cells or PBMCs. Both viruses were subjected to serial passages in MT-4 cells with selective pressure from the PIs atazanavir (ATV) and darunavir (DRV). After 32 successive passages, both the WT and MAX viruses developed phenotypic resistance to PIs (IC&lt;sub&gt;50&lt;/sub&gt; 14.6 ± 5.3 and 21.2 ± 9 nM for ATV, and 5. 9 ± 1.0 and 9.3 ± 1.9 for DRV, respectively). Ultra-deep sequence clonal analysis revealed that both viruses harbored previously described resistance mutations to ATV and DRV. However, the WT and MAX virus proteases showed different resistance variant repertoires, with the G16E and V77I substitutions observed only in WT, and the L33F, S37P, G48L, Q58E/K, and L89I substitutions detected only in MAX. Remarkably, G48L and L89I are rarely found &lt;i&gt;in vivo&lt;/i&gt; in PI-treated patients. The MAX virus showed significantly higher nucleotide and amino acid diversity of the propagated viruses with and without PIs (&lt;i&gt;P&lt;/i&gt; &amp;lt; 0.0001), suggesting higher selective pressure for change in this recoded virus. Our results indicate that HIV-1 protease position in sequence space delineates the evolution of its mutant spectra. Nevertheless, the investigated synonymously recoded variant showed mutational robustness and evolvability similar to the WT virus.&lt;/p&gt;&lt;h3&gt;IMPORTANCE&lt;/h3&gt;
&lt;p&gt;Large-scale synonymous recoding of virus genomes is a new tool for exploring various aspects of virus biology. Synonymous virus genome recoding can be used to investigate how a virus’s position in sequence space defines its mutant spectrum, evolutionary trajectory, and pathogenesis. In this study, we evaluated how synonymous recoding of the human immunodeficiency virus type 1 (HIV-1) protease impacts the development of protease inhibitor (PI) resistance. HIV-1 protease is a main target of current antiretroviral therapies. Our present results demonstrate that the wild-type (WT) virus and the virus with the recoded protease exhibited different patterns of resistance mutations after PI treatment. Nevertheless, the developed PI resistance phenotype was indistinguishable between the recoded virus and the WT virus, suggesting that the synonymously recoded protease HIV-1 and the WT protease virus were equally robust and evolvable.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;315366v1" />
<meta name="citation_id" content="315366v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/315366v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/315366v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/315366v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366.full.pdf" />
<meta name="citation_doi" content="10.1101/315366" />
<meta name="citation_num_pages" content="33" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="315366" />
<meta name="citation_author" content="Maria Nevot" />
<meta name="citation_author_institution" content="IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)" />
<meta name="citation_author" content="Ana Jordan-Paiz" />
<meta name="citation_author" content="Glòria Martrus" />
<meta name="citation_author_institution" content="IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)" />
<meta name="citation_author_institution" content="Heinrich-Pette-Institute, Leibniz Institute for Experimental Virology" />
<meta name="citation_author" content="Cristina Andrés" />
<meta name="citation_author_institution" content="IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)" />
<meta name="citation_author_institution" content="Virology Unit, Microbiology Department, Hospital Universitari Vall d’Hebron, Vall d’Hebron Research Institute, Universitat Autònoma de Barcelona (UAB)" />
<meta name="citation_author" content="Damir García-Cehic" />
<meta name="citation_author_institution" content="Liver Unit, Liver Disease Laboratory-Viral Hepatitis, Internal Medicine Department, Vall d’Hebron Institut Recerca (VHIR)-Hospital Universitari Vall d’Hebron (HUVH)" />
<meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red (CIBER) de Enfermedades Hepáticas y Digestivas (CIBERehd) del Instituto de Salud Carlos III" />
<meta name="citation_author" content="Josep Gregori" />
<meta name="citation_author_institution" content="Liver Unit, Liver Disease Laboratory-Viral Hepatitis, Internal Medicine Department, Vall d’Hebron Institut Recerca (VHIR)-Hospital Universitari Vall d’Hebron (HUVH)" />
<meta name="citation_author_institution" content="Roche Diagnostics SL, Sant Cugat del Vallès" />
<meta name="citation_author" content="Sandra Franco" />
<meta name="citation_author_institution" content="IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)" />
<meta name="citation_author" content="Josep Quer" />
<meta name="citation_author_institution" content="Liver Unit, Liver Disease Laboratory-Viral Hepatitis, Internal Medicine Department, Vall d’Hebron Institut Recerca (VHIR)-Hospital Universitari Vall d’Hebron (HUVH)" />
<meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red (CIBER) de Enfermedades Hepáticas y Digestivas (CIBERehd) del Instituto de Salud Carlos III" />
<meta name="citation_author_institution" content="Universitat Autònoma de Barcelona" />
<meta name="citation_author" content="Miguel Angel Martinez" />
<meta name="citation_author_institution" content="IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)" />
<meta name="citation_author_email" content="mmartinez@irsicaixa.es" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6681-4950" />
<meta name="citation_reference" content="Hunt RC, Simhadri VL, Iandoli M, Sauna ZE, Kimchi-Sarfaty C. 2014. Exposing synonymous mutations. Trends in genetics: TIG 30:308–321." />
<meta name="citation_reference" content="Plotkin JB, Kudla G. 2011. Synonymous but not the same: the causes and consequences of codon bias. Nature reviews. Genetics 12:32–42." />
<meta name="citation_reference" content="Martinez MA, Jordan-Paiz A, Franco S, Nevot M. 2016. Synonymous Virus Genome Recoding as a Tool to Impact Viral Fitness. Trends in microbiology 24:134–147." />
<meta name="citation_reference" content="Burns CC, Campagnoli R, Shaw J, Vincent A, Jorba J, Kew O. 2009. Genetic inactivation of poliovirus infectivity by increasing the frequencies of CpG and UpA dinucleotides within and across synonymous capsid region codons. Journal of virology 83:9957–9969." />
<meta name="citation_reference" content="Burns CC, Shaw J, Campagnoli R, Jorba J, Vincent A, Quay J, Kew O. 2006. Modulation of poliovirus replicative fitness in HeLa cells by deoptimization of synonymous codon usage in the capsid region. Journal of virology 80:3259–3272." />
<meta name="citation_reference" content="Coleman JR, Papamichail D, Skiena S, Futcher B, Wimmer E, Mueller S. 2008. Virus attenuation by genome-scale changes in codon pair bias. Science 320:1784–1787." />
<meta name="citation_reference" content="Mueller S, Papamichail D, Coleman JR, Skiena S, Wimmer E. 2006. Reduction of the rate of poliovirus protein synthesis through large-scale codon deoptimization causes attenuation of viral virulence by lowering specific infectivity. Journal of virology 80:9687–9696." />
<meta name="citation_reference" content="Mueller S, Coleman JR, Papamichail D, Ward CB, Nimnual A, Futcher B, Skiena S, Wimmer E. 2010. Live attenuated influenza virus vaccines by computer-aided rational design. Nat Biotechnol 28:723–726." />
<meta name="citation_reference" content="Yang C, Skiena S, Futcher B, Mueller S, Wimmer E. 2013. Deliberate reduction of hemagglutinin and neuraminidase expression of influenza virus leads to an ultraprotective live vaccine in mice. Proceedings of the National Academy of Sciences of the United States of America 110:9481–9486." />
<meta name="citation_reference" content="Martrus G, Nevot M, Andres C, Clotet B, Martinez MA. 2013. Changes in codon-pair bias of human immunodeficiency virus type 1 have profound effects on virus replication in cell culture. Retrovirology 10:78." />
<meta name="citation_reference" content="Takata MA, Goncalves-Carneiro D, Zang TM, Soll SJ, York A, Blanco-Melo D, Bieniasz PD. 2017. CG dinucleotide suppression enables antiviral defence targeting non-self RNA. Nature 550:124–127." />
<meta name="citation_reference" content="Takata MA, Soll SJ, Emery A, Blanco-Melo D, Swanstrom R, Bieniasz PD. 2018. Global synonymous mutagenesis identifies cis-acting RNA elements that regulate HIV-1 splicing and replication. PLoS pathogens 14:e1006824." />
<meta name="citation_reference" content="Vabret N, Bailly-Bechet M, Lepelley A, Najburg V, Schwartz O, Verrier B, Tangy F. 2014. Large-scale nucleotide optimization of simian immunodeficiency virus reduces its capacity to stimulate type I interferon in vitro. Journal of virology 88:4161–4172." />
<meta name="citation_reference" content="Nougairede A, De Fabritus L, Aubry F, Gould EA, Holmes EC, de Lamballerie X. 2013. Random codon re-encoding induces stable reduction of replicative fitness of Chikungunya virus in primate and mosquito cells. PLoS pathogens 9:e1003172." />
<meta name="citation_reference" content="Le Nouen C, Brock LG, Luongo C, McCarty T, Yang L, Mehedi M, Wimmer E, Mueller S, Collins PL, Buchholz UJ, DiNapoli JM. 2014. Attenuation of human respiratory syncytial virus by genome-scale codon-pair deoptimization. Proceedings of the National Academy of Sciences of the United States of America 111:13169–13174." />
<meta name="citation_reference" content="Le Nouen C, McCarty T, Brown M, Smith ML, Lleras R, Dolan MA, Mehedi M, Yang L, Luongo C, Liang B, Munir S, DiNapoli JM, Mueller S, Wimmer E, Collins PL, Buchholz UJ. 2017. Genetic stability of genome-scale deoptimized RNA virus vaccine candidates under selective pressure. Proceedings of the National Academy of Sciences of the United States of America 114:E386–E395." />
<meta name="citation_reference" content="Meng J, Lee S, Hotard AL, Moore ML. 2014. Refining the balance of attenuation and immunogenicity of respiratory syncytial virus by targeted codon deoptimization of virulence genes. mBio 5:e01704–01714." />
<meta name="citation_reference" content="Ni YY, Zhao Z, Opriessnig T, Subramaniam S, Zhou L, Cao D, Cao Q, Yang H, Meng XJ. 2014. Computer-aided codon-pairs deoptimization of the major envelope GP5 gene attenuates porcine reproductive and respiratory syndrome virus. Virology 450-451:132–139." />
<meta name="citation_reference" content="Atkinson NJ, Witteveldt J, Evans DJ, Simmonds P. 2014. The influence of CpG and UpA dinucleotide frequencies on RNA virus replication and characterization of the innate cellular pathways underlying virus attenuation and enhanced replication. Nucleic acids research 42:4527–4545." />
<meta name="citation_reference" content="Tulloch F, Atkinson NJ, Evans DJ, Ryan MD, Simmonds P. 2014. RNA virus attenuation by codon pair deoptimisation is an artefact of increases in CpG/UpA dinucleotide frequencies. eLife 3:e04531." />
<meta name="citation_reference" content="de Fabritus L, Nougairede A, Aubry F, Gould EA, de Lamballerie X. 2015. Attenuation of tick-borne encephalitis virus using large-scale random codon re-encoding. PLoS pathogens 11:e1004738." />
<meta name="citation_reference" content="Shen SH, Stauft CB, Gorbatsevych O, Song Y, Ward CB, Yurovsky A, Mueller S, Futcher B, Wimmer E. 2015. Large-scale recoding of an arbovirus genome to rebalance its insect versus mammalian preference. Proceedings of the National Academy of Sciences of the United States of America 112:4749–4754." />
<meta name="citation_reference" content="Sitaraman V, Hearing P, Ward CB, Gnatenko DV, Wimmer E, Mueller S, Skiena S, Bahou WF. 2011. Computationally designed adeno-associated virus (AAV) Rep 78 is efficiently maintained within an adenovirus vector. Proceedings of the National Academy of Sciences of the United States of America 108:14294–14299." />
<meta name="citation_reference" content="Cladel NM, Budgeon LR, Hu J, Balogh KK, Christensen ND. 2013. Synonymous codon changes in the oncogenes of the cottontail rabbit papillomavirus lead to increased oncogenicity and immunogenicity of the virus. Virology 438:70–83." />
<meta name="citation_reference" content="Dolan PT, Whitfield ZJ, Andino R. 2018. Mapping the Evolutionary Potential of RNA Viruses. Cell host &amp; microbe 23:435–446." />
<meta name="citation_reference" content="Andino R, Domingo E. 2015. Viral quasispecies. Virology 479-480:46–51." />
<meta name="citation_reference" content="Mas A, Lopez-Galindez C, Cacho I, Gomez J, Martinez MA. 2010. Unfinished stories on viral quasispecies and Darwinian views of evolution. J Mol Biol 397:865–877." />
<meta name="citation_reference" content="Song Y, Liu Y, Ward CB, Mueller S, Futcher B, Skiena S, Paul AV, Wimmer E. 2012. Identification of two functionally redundant RNA elements in the coding sequence of poliovirus using computer-generated design. Proceedings of the National Academy of Sciences of the United States of America 109:14301–14307." />
<meta name="citation_reference" content="Li M, Kao E, Gao X, Sandig H, Limmer K, Pavon-Eternod M, Jones TE, Landry S, Pan T, Weitzman MD, David M. 2012. Codon-usage-based inhibition of HIV protein synthesis by human schlafen 11. Nature 491:125–128." />
<meta name="citation_reference" content="Vabret N, Bhardwaj N, Greenbaum BD. 2017. Sequence-Specific Sensing of Nucleic Acids. Trends in immunology 38:53–65." />
<meta name="citation_reference" content="Shin YC, Bischof GF, Lauer WA, Desrosiers RC. 2015. Importance of codon usage for the temporal regulation of viral gene expression. Proceedings of the National Academy of Sciences of the United States of America 112:14030–14035." />
<meta name="citation_reference" content="Lauring AS, Acevedo A, Cooper SB, Andino R. 2012. Codon usage determines the mutational robustness, evolutionary capacity, and virulence of an RNA virus. Cell host &amp; microbe 12:623–632." />
<meta name="citation_reference" content="Lauring AS, Frydman J, Andino R. 2013. The role of mutational robustness in RNA virus evolution. Nat Rev Microbiol 11:327–336." />
<meta name="citation_reference" content="Eigen M. 1971. Selforganization of matter and the evolution of biological macromolecules. Naturwissenschaften 58:465–523." />
<meta name="citation_reference" content="Domingo E, Sabo D, Taniguchi T, Weissmann C. 1978. Nucleotide sequence heterogeneity of an RNA phage population. Cell 13:735–744." />
<meta name="citation_reference" content="van der Kuyl AC, Berkhout B. 2012. The biased nucleotide composition of the HIV genome: a constant factor in a highly variable virus. Retrovirology 9:92." />
<meta name="citation_reference" content="Gifford RJ, Liu TF, Rhee SY, Kiuchi M, Hue S, Pillay D, Shafer RW. 2009. The calibrated population resistance tool: standardized genotypic estimation of transmitted HIV-1 drug resistance. Bioinformatics 25:1197–1198." />
<meta name="citation_reference" content="Wensing AM, Calvez V, Gunthard HF, Johnson VA, Paredes R, Pillay D, Shafer RW, Richman DD. 2017. 2017 Update of the Drug Resistance Mutations in HIV-1. Topics in antiviral medicine 24:132–133." />
<meta name="citation_reference" content="Parera M, Martinez MA. 2014. Strong epistatic interactions within a single protein. Molecular biology and evolution 31:1546–1553." />
<meta name="citation_reference" content="Shafer RW, Schapiro JM. 2008. HIV-1 drug resistance mutations: an updated framework for the second decade of HAART. AIDS reviews 10:67–84." />
<meta name="citation_reference" content="Flynn WF, Haldane A, Torbett BE, Levy RM. 2017. Inference of Epistatic Effects Leading to Entrenchment and Drug Resistance in HIV-1 Protease. Molecular biology and evolution 34:1291–1306." />
<meta name="citation_reference" content="Cambray G, Mazel D. 2008. Synonymous genes explore different evolutionary landscapes. PLoS Genet 4:e1000256." />
<meta name="citation_reference" content="Maschera B, Furfine E, Blair ED. 1995. Analysis of resistance to human immunodeficiency virus type 1 protease inhibitors by using matched bacterial expression and proviral infection vectors. Journal of virology 69:5431–5436." />
<meta name="citation_reference" content="Parera M, Clotet B, Martinez MA. 2004. Genetic screen for monitoring severe acute respiratory syndrome coronavirus 3C-like protease. Journal of virology 78:14057–14061." />
<meta name="citation_reference" content="Pauwels R, Balzarini J, Baba M, Snoeck R, Schols D, Herdewijn P, Desmyter J, De Clercq E. 1988. Rapid and automated tetrazolium-based colorimetric assay for the detection of anti-HIV compounds. Journal of virological methods 20:309–321." />
<meta name="citation_reference" content="Betancor G, Alvarez M, Marcelli B, Andres C, Martinez MA, Menendez-Arias L. 2015. Effects of HIV-1 reverse transcriptase connection subdomain mutations on polypurine tract removal and initiation of (+)-strand DNA synthesis. Nucleic acids research 43:2259–2270." />
<meta name="citation_reference" content="Capel E, Martrus G, Parera M, Clotet B, Martinez MA. 2012. Evolution of the human immunodeficiency virus type 1 protease: effects on viral replication capacity and protease robustness. J Gen Virol 93:2625–2634." />
<meta name="citation_reference" content="Fernandez G, Clotet B, Martinez MA. 2007. Fitness landscape of human immunodeficiency virus type 1 protease quasispecies. Journal of virology 81:2485–2496." />
<meta name="citation_reference" content="Magoc T, Salzberg SL. 2011. FLASH: fast length adjustment of short reads to improve genome assemblies. Bioinformatics 27:2957–2963." />
<meta name="citation_reference" content="Gregori J, Esteban JI, Cubero M, Garcia-Cehic D, Perales C, Casillas R, Alvarez-Tejado M, Rodriguez-Frias F, Guardia J, Domingo E, Quer J. 2013. Ultra-deep pyrosequencing (UDPS) data treatment to study amplicon HCV minor variants. PloS one 8:e83361." />
<meta name="citation_reference" content="Paradis E, Claude J, Strimmer K. 2004. APE: Analyses of Phylogenetics and Evolution in R language. Bioinformatics 20:289–290." />
<meta name="citation_reference" content="Charif D, Thioulouse J, Lobry JR, Perriere G. 2005. Online synonymous codon usage analyses with the ade4 and seqinR packages. Bioinformatics 21:545–547." />
<meta name="citation_reference" content="Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. 2013. MEGA6: Molecular Evolutionary Genetics Analysis version 6.0. Molecular biology and evolution 30:2725–2729." />
<meta name="citation_reference" content="Wolinsky SM, Kunstman KJ, Safrit JT, Koup RA, Neumann AU, Korber BT. 1996. Response: HIV-1 Evolution and Disease Progression. Science 274:1010–1011." />
<meta name="twitter:title" content="HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="One unexplored aspect of HIV-1 genetic architecture is how codon choice influences population diversity and evolvability. Here we compared the development of HIV-1 resistance to protease inhibitors (PIs) between wild-type (WT) virus and a synthetic virus (MAX) carrying a codon-pair re-engineered protease sequence including 38 (13%) synonymous mutations. WT and MAX viruses showed indistinguishable replication in MT-4 cells or PBMCs. Both viruses were subjected to serial passages in MT-4 cells with selective pressure from the PIs atazanavir (ATV) and darunavir (DRV). After 32 successive passages, both the WT and MAX viruses developed phenotypic resistance to PIs (IC50 14.6 ± 5.3 and 21.2 ± 9 nM for ATV, and 5. 9 ± 1.0 and 9.3 ± 1.9 for DRV, respectively). Ultra-deep sequence clonal analysis revealed that both viruses harbored previously described resistance mutations to ATV and DRV. However, the WT and MAX virus proteases showed different resistance variant repertoires, with the G16E and V77I substitutions observed only in WT, and the L33F, S37P, G48L, Q58E/K, and L89I substitutions detected only in MAX. Remarkably, G48L and L89I are rarely found in vivo in PI-treated patients. The MAX virus showed significantly higher nucleotide and amino acid diversity of the propagated viruses with and without PIs ( P &lt; 0.0001), suggesting higher selective pressure for change in this recoded virus. Our results indicate that HIV-1 protease position in sequence space delineates the evolution of its mutant spectra. Nevertheless, the investigated synonymously recoded variant showed mutational robustness and evolvability similar to the WT virus.

IMPORTANCE Large-scale synonymous recoding of virus genomes is a new tool for exploring various aspects of virus biology. Synonymous virus genome recoding can be used to investigate how a virus’s position in sequence space defines its mutant spectrum, evolutionary trajectory, and pathogenesis. In this study, we evaluated how synonymous recoding of the human immunodeficiency virus type 1 (HIV-1) protease impacts the development of protease inhibitor (PI) resistance. HIV-1 protease is a main target of current antiretroviral therapies. Our present results demonstrate that the wild-type (WT) virus and the virus with the recoded protease exhibited different patterns of resistance mutations after PI treatment. Nevertheless, the developed PI resistance phenotype was indistinguishable between the recoded virus and the WT virus, suggesting that the synonymously recoded protease HIV-1 and the WT protease virus were equally robust and evolvable." />
<meta name="og-title" property="og:title" content="HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/315366v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="One unexplored aspect of HIV-1 genetic architecture is how codon choice influences population diversity and evolvability. Here we compared the development of HIV-1 resistance to protease inhibitors (PIs) between wild-type (WT) virus and a synthetic virus (MAX) carrying a codon-pair re-engineered protease sequence including 38 (13%) synonymous mutations. WT and MAX viruses showed indistinguishable replication in MT-4 cells or PBMCs. Both viruses were subjected to serial passages in MT-4 cells with selective pressure from the PIs atazanavir (ATV) and darunavir (DRV). After 32 successive passages, both the WT and MAX viruses developed phenotypic resistance to PIs (IC50 14.6 ± 5.3 and 21.2 ± 9 nM for ATV, and 5. 9 ± 1.0 and 9.3 ± 1.9 for DRV, respectively). Ultra-deep sequence clonal analysis revealed that both viruses harbored previously described resistance mutations to ATV and DRV. However, the WT and MAX virus proteases showed different resistance variant repertoires, with the G16E and V77I substitutions observed only in WT, and the L33F, S37P, G48L, Q58E/K, and L89I substitutions detected only in MAX. Remarkably, G48L and L89I are rarely found in vivo in PI-treated patients. The MAX virus showed significantly higher nucleotide and amino acid diversity of the propagated viruses with and without PIs ( P &lt; 0.0001), suggesting higher selective pressure for change in this recoded virus. Our results indicate that HIV-1 protease position in sequence space delineates the evolution of its mutant spectra. Nevertheless, the investigated synonymously recoded variant showed mutational robustness and evolvability similar to the WT virus.

IMPORTANCE Large-scale synonymous recoding of virus genomes is a new tool for exploring various aspects of virus biology. Synonymous virus genome recoding can be used to investigate how a virus’s position in sequence space defines its mutant spectrum, evolutionary trajectory, and pathogenesis. In this study, we evaluated how synonymous recoding of the human immunodeficiency virus type 1 (HIV-1) protease impacts the development of protease inhibitor (PI) resistance. HIV-1 protease is a main target of current antiretroviral therapies. Our present results demonstrate that the wild-type (WT) virus and the virus with the recoded protease exhibited different patterns of resistance mutations after PI treatment. Nevertheless, the developed PI resistance phenotype was indistinguishable between the recoded virus and the WT virus, suggesting that the synonymously recoded protease HIV-1 and the WT protease virus were equally robust and evolvable." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-05-07" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/315366v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/315366v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/315366v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/315366v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/96967" />
    <title>HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VMgZ-gExnV4WDK6oMvZZHeaj0alsfnKAsV4h4tmrX2E","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"96967","apath":"\/biorxiv\/early\/2018\/05\/07\/315366.atom","pisa":"biorxiv;315366v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;315366v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/315366v1","title":"HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding","identifier":"node\/96967"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VFWms1t\\\\\\\/9sq59yFuOUrPnIFHWwN+QQRCGfNkSSwHx1yLzU0niIzYtNB69OV\\\\\\\/yibtWR25VJ5DbtTotjuUP5vyUzJvQbgXXuBmcZNyywmfvUhjZQCMu\\\\\\\/fhNEpFHWUEi\\\\\\\/41PRyuZb8RtB\\\\\\\/+ZshOAizA8g6reJc9bFoq0kMQ3PB3EPYH6IlWF\\\\\\\/LbUmWXPCCfmNFwhc219Orr4AFGOKjwVXF1eZsn301M7ClxEb+wwrOu4LD8TRILG3yaXjvdw+YvhLHJWWZAo1CwoehzdsusEvK3Vc3vpWJcsFvu3eRQbCrYiQl7hvZxwGKz5ZBHi8OrpIR\\\\\\\/3toj\\\\\\\/Cw5UgFZXi46jrLbR4471OgoeCTA\\\\\\\/3Tg8KPkDmy9rfo9YP3ZlTVyxrA0KeqtVAYIbY3zKeXwCVi1GoLsG98FFXCgakmtD\\\\\\\/melE0NUXu4t\\\\\\\/K3+NMlbzSS5H\\\\\\\/BPXaAUUru\\\\\\\/PssJNi4Qg7uA2M+ffiWahsHIHj1v44cLbogH8mrUJa9uE4xdIAg4FXgM+Wx56Xvicoa3zxTfimgVcc5STQ2eIiNAjIfywSUxdFmOIYKIXKYdbMwNts6w7vU8eSWwLgxo2ZsqiXfCMjzIt\\\\\\\/dUn++Elkhd8oo4shxsXaTU6dAtYt8E8i3qdfRHUipjUS7E7G5iv7ZCSAEvLx3oU+PF+91KqY2bzoyHGjXxVkJFUy6n6n1zh1ozHtZCsdH+gkMSp4oMuZ+beZJ4\\\\\\\/3Dfp5Ns\\\\\\\/1PjMuqu5S\\\\\\\/B0ztFkCsfDGsL95mIEX0MCmH5TVtjrzMSc1Z47jIOfuV3Jw3nYsf79OR6RIdjGvJBj6gd9wAD0IEbptH26bUdf\\\\\\\/rN1xPYpXpdj9tKN5PRVgJ6KB7tNZjkFFOe7x5eZ5ED\\\\\\\/pwO70nLSk2\\\\\\\/JOizBBcsseB0wkGWoPhECyH0QGKyvrafLKIhmM5X38ODC69Evs36sl8x6Zn4UWrkvypzfol2QL3msYGMcMtbAXAgoNKMy9dV4czcySPc0hC5yVTiC1NbT7bCiyG0j\\\\\\\/t5d8c+pZWrtYFjf3\\\\\\\/ms6Qtc5bBdAHDHZPyx6FnVqqDlVffkdvWa4iAA2qLWEnzmJRQ1s8b9tggSqpnQXAhMQgW\\\\\\\/y2Xe8Bz\\\\\\\/7Zqt0Ekw29bYarmdqOJPT0Xzri95ntUJVFPafg8iJ6Z9MXm+UEcE91AU2U2\\\\\\\/0ttpDU3nzYogPR\\\\\\\/zghh5numpZqm9xC2K\\\\\\\/pFfd6JL2SkxTX0HVDx7\\\\\\\/RSDvge1hkHhSZssRsGSDLBGWUvuG6url+lQrMkU7XJH4CcyAFhDtwn8CfXnriMffrC+ueAJjvS4TOoZ83lGFhrMjCAtVf8rueosmczau4eRKg0hU=\\\u0022,\\\u0022iv\\\u0022:\\\u00229l7h31BbwGNJGVk81j5xqQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224805b77e2fda1b63282ee03c73f0f81c2be5aeb3b0f9748eb2141b3d24c0fc59\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/315366v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-96967 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/315366v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2UVFQNuSsRaYQGYKxEnRldEoUsNBQqlzUo8h2Zxo-N0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="96967" id="node96967" data-pisa="biorxiv;315366v1" data-pisa-master="biorxiv;315366" data-apath="/biorxiv/early/2018/05/07/315366.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Nevot</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ana</span> <span class="nlm-surname">Jordan-Paiz</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Glòria</span> <span class="nlm-surname">Martrus</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Cristina</span> <span class="nlm-surname">Andrés</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Damir</span> <span class="nlm-surname">García-Cehic</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Josep</span> <span class="nlm-surname">Gregori</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sandra</span> <span class="nlm-surname">Franco</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Josep</span> <span class="nlm-surname">Quer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-6681-4950" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Miguel Angel</span> <span class="nlm-surname">Martinez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/315366 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node96967" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maria Nevot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)</span>, Badalona, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BNevot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nevot%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BNevot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ana Jordan-Paiz </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BJordan-Paiz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jordan-Paiz%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BJordan-Paiz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Glòria Martrus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)</span>, Badalona, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span>Current address: <span class='nlm-institution'>Heinrich-Pette-Institute, Leibniz Institute for Experimental Virology</span>, Hamburg, <span class='nlm-country'>Germany</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gl%C3%B2ria%2BMartrus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martrus%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGl%25C3%25B2ria%2BMartrus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Cristina Andrés </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)</span>, Badalona, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span>Current address: <span class='nlm-institution'>Virology Unit, Microbiology Department, Hospital Universitari Vall d’Hebron, Vall d’Hebron Research Institute, Universitat Autònoma de Barcelona (UAB)</span>, Barcelona, <span class='nlm-country'>Spain</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cristina%2BAndr%C3%A9s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Andr%C3%A9s%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACristina%2BAndr%25C3%25A9s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Damir García-Cehic </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Liver Unit, Liver Disease Laboratory-Viral Hepatitis, Internal Medicine Department, Vall d’Hebron Institut Recerca (VHIR)-Hospital Universitari Vall d’Hebron (HUVH)</span>, Barcelona, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Centro de Investigación Biomédica en Red (CIBER) de Enfermedades Hepáticas y Digestivas (CIBERehd) del Instituto de Salud Carlos III</span>, Madrid, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damir%2BGarc%C3%ADa-Cehic%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa-Cehic%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADamir%2BGarc%25C3%25ADa-Cehic%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Josep Gregori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Liver Unit, Liver Disease Laboratory-Viral Hepatitis, Internal Medicine Department, Vall d’Hebron Institut Recerca (VHIR)-Hospital Universitari Vall d’Hebron (HUVH)</span>, Barcelona, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Roche Diagnostics SL, Sant Cugat del Vallès</span>, Barcelona, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josep%2BGregori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gregori%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosep%2BGregori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sandra Franco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)</span>, Badalona, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BFranco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franco%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BFranco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Josep Quer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Liver Unit, Liver Disease Laboratory-Viral Hepatitis, Internal Medicine Department, Vall d’Hebron Institut Recerca (VHIR)-Hospital Universitari Vall d’Hebron (HUVH)</span>, Barcelona, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Centro de Investigación Biomédica en Red (CIBER) de Enfermedades Hepáticas y Digestivas (CIBERehd) del Instituto de Salud Carlos III</span>, Madrid, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Universitat Autònoma de Barcelona</span>, Bellaterra, Barcelona, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josep%2BQuer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Quer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosep%2BQuer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Miguel Angel Martinez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>IrsiCaixa, Hospital Universitari Germans Trias i Pujol, Universitat Autònoma de Barcelona (UAB)</span>, Badalona, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BAngel%2BMartinez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martinez%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiguel%2BAngel%2BMartinez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6681-4950" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miguel Angel Martinez</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mmartinez@irsicaixa.es" class="" data-icon-position="" data-hide-link-title="0">mmartinez@irsicaixa.es</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/315366v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:96967" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:96967/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/315366v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:96967" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:96967/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/315366v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:96967" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:96967/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/315366v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:96967" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:96967/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/315366v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:96967" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:96967/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">One unexplored aspect of HIV-1 genetic architecture is how codon choice influences population diversity and evolvability. Here we compared the development of HIV-1 resistance to protease inhibitors (PIs) between wild-type (WT) virus and a synthetic virus (MAX) carrying a codon-pair re-engineered protease sequence including 38 (13%) synonymous mutations. WT and MAX viruses showed indistinguishable replication in MT-4 cells or PBMCs. Both viruses were subjected to serial passages in MT-4 cells with selective pressure from the PIs atazanavir (ATV) and darunavir (DRV). After 32 successive passages, both the WT and MAX viruses developed phenotypic resistance to PIs (IC<sub>50</sub> 14.6 ± 5.3 and 21.2 ± 9 nM for ATV, and 5. 9 ± 1.0 and 9.3 ± 1.9 for DRV, respectively). Ultra-deep sequence clonal analysis revealed that both viruses harbored previously described resistance mutations to ATV and DRV. However, the WT and MAX virus proteases showed different resistance variant repertoires, with the G16E and V77I substitutions observed only in WT, and the L33F, S37P, G48L, Q58E/K, and L89I substitutions detected only in MAX. Remarkably, G48L and L89I are rarely found <em>in vivo</em> in PI-treated patients. The MAX virus showed significantly higher nucleotide and amino acid diversity of the propagated viruses with and without PIs (<em>P</em> &lt; 0.0001), suggesting higher selective pressure for change in this recoded virus. Our results indicate that HIV-1 protease position in sequence space delineates the evolution of its mutant spectra. Nevertheless, the investigated synonymously recoded variant showed mutational robustness and evolvability similar to the WT virus.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>IMPORTANCE</strong> Large-scale synonymous recoding of virus genomes is a new tool for exploring various aspects of virus biology. Synonymous virus genome recoding can be used to investigate how a virus’s position in sequence space defines its mutant spectrum, evolutionary trajectory, and pathogenesis. In this study, we evaluated how synonymous recoding of the human immunodeficiency virus type 1 (HIV-1) protease impacts the development of protease inhibitor (PI) resistance. HIV-1 protease is a main target of current antiretroviral therapies. Our present results demonstrate that the wild-type (WT) virus and the virus with the recoded protease exhibited different patterns of resistance mutations after PI treatment. Nevertheless, the developed PI resistance phenotype was indistinguishable between the recoded virus and the WT virus, suggesting that the synonymously recoded protease HIV-1 and the WT protease virus were equally robust and evolvable.</p></div></div><div class="section" id="sec-2"><h2 class="">INTRODUCTION</h2><p id="p-4">Alterations in a DNA or mRNA sequence that do not change the protein amino acid sequence are called synonymous mutations. Although they do not influence the resulting protein sequence, synonymous mutations can still substantially affect cellular processes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). Notably, synonymous virus genome recoding can impact viral replication capacity and fitness (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>), reportedly leading to attenuation of multiple RNA and DNA viruses, including poliovirus (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>-<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>), influenza virus (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>), HIV-1 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>), SIV (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>), Chikungunya virus (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>), human respiratory syncytial virus (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>-<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>), porcine reproductive and respiratory syndrome virus (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>), echovirus 7 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>), tick-borne encephalitis virus (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>), vesicular stomatitis virus, dengue virus (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>), adeno-associated virus (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>), and papillomavirus (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>).</p><p id="p-5">Synonymous virus genome recoding is being investigated as a new strategy for generating novel live-attenuated vaccine candidates. This method is promising because the amino acid coding is completely unaffected, thereby avoiding the potential generation of new and undesirable biological properties. Moreover, synonymous virus genome recoding involves the introduction of hundreds or thousands of nucleotide substitutions, which minimizes the risk of phenotypic reversion via point mutations or through recombination with homologous sequences in circulating strains. This is particularly important with regards to RNA viruses, since viral RNA polymerases lack error-correction mechanisms (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>-<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). The high genetic variability of RNA viruses is a critical limitation when designing novel antiviral strategies.</p><p id="p-6">The usefulness of synonymous virus genome recoding goes beyond the generation of new live attenuated vaccines. This method has also been used to identify specific RNA structures required for virus replication (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>), virus genome cis-inhibitory signal sequences important for complex viral functions (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>), and novel antiviral mechanisms within the innate immune response (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>), as well as to resolve the importance of codon usage in the temporal regulation of viral gene expression (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). In one interesting example, synonymous virus genome recoding was used to demonstrate that a synonymous position in sequence space can impact poliovirus evolvability and pathogenesis (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>).</p><p id="p-7">Like other RNA viruses, human immunodeficiency virus type 1 (HIV-1) populations comprise a closely related mutant spectra or mutant clouds termed viral quasispecies (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Mutant cloud composition can impact virus evolvability, fitness, and virulence (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>-<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>). One unexplored aspect of HIV-1 genetic architecture is how codon choice influences population diversity and evolvability. It is presently unclear whether HIV-1 sequences have evolved to optimize both protein coding and DNA/RNA sequences. The HIV-1 genome exhibits a particularly striking bias towards enrichment of A-rich codons, which may be a selectable trait (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>) and affect innate immune recognition (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>). Similarly, synonymous codon usage can temporally regulate expressions of structural gene products of the simian immunodeficiency virus (SIV) (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>) and regulate HIV-1 splicing and replication (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>).</p><p id="p-8">Here we aimed to explore whether synonymous sequence space influences the development of protease inhibitor (PI) resistance, and to thus determine whether HIV-1 evolvability is influenced by the natural position of a protease in sequence space. To this end, we compared the development of HIV-1 resistance to the protease inhibitors atazanavir (ATV) and darunavir (DRV) between wild-type HIV-1 (WT) and synthetic HIV-1 carrying a synonymously recoded protease sequence (MAX).</p></div><div class="section" id="sec-3"><h2 class="">RESULTS</h2><p id="p-9">WT HIV-1 was compared with the MAX variant that carried a protease gene including 38 synonymous mutations (13% of the protease sequence) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). These 38 synonymous substitutions were scattered throughout the protease coding region, excluding the first 40 amino-terminal nucleotides that overlap with the carboxy-terminal of the gag p6 reading frame. These 38 substitutions were chosen in order to improve protease gene codon pair bias (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>) without modifying its codon bias or folding free energy (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>). The WT and MAX viruses have identical consensus amino acid sequences,-and, as we have previously demonstrated (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>), this MAX variant and the WT virus show indistinguishable replication in MT-4 cells (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>) or PBMCs (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HIV-1 protease nucleotide sequences of the wild-type (WT) virus that corresponds to the HIV-1 HXB2 strain (http://www.hiv.lanl.gov), and of the synthetic MAX variant that was generated by PCR combining three overlapping synthetic DNA oligonucleotides as previously described (10). No substitution was introduced in the first 40 protease amino-terminal nucleotides that overlap with the carboxy-terminal of the gag p6 reading frame." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1851797889" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;HIV-1 protease nucleotide sequences of the wild-type (WT) virus that corresponds to the HIV-1 HXB2 strain (http://www.hiv.lanl.gov), and of the synthetic MAX variant that was generated by PCR combining three overlapping synthetic DNA oligonucleotides as previously described (10). No substitution was introduced in the first 40 protease amino-terminal nucleotides that overlap with the carboxy-terminal of the gag p6 reading frame.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F1.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 1" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F1.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">FIG 1</span> <p id="p-10" class="first-child">HIV-1 protease nucleotide sequences of the wild-type (WT) virus that corresponds to the HIV-1 HXB2 strain (<a href="http://www.hiv.lanl.gov">http://www.hiv.lanl.gov</a>), and of the synthetic MAX variant that was generated by PCR combining three overlapping synthetic DNA oligonucleotides as previously described (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">10</a>). No substitution was introduced in the first 40 protease amino-terminal nucleotides that overlap with the carboxy-terminal of the gag p6 reading frame.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Replication kinetic assay of wild-type HIV-1 (WTp1) and the recoded MAX protease variant (MAXp1) in MT-4 cells. HIV-1 antigen p24 concentrations in culture supernatants were measured on days 0-4 in the absence of drug or days 0&#x2013;6 in the presence of drug. For each virus, the slope of the plot provides an estimate of the viral replication capacity. Bars show the slope of the p24 antigen production from each virus after infection of MT-4 cells. Comparison between WTp1 (HXB2) and the mutant MAX recoded viruses are shown, as well as between the corresponding WT and MAX virus variants (i.e.. S37P, G48L, Q58E, I84V and I89L). The significance of the difference between slopes was calculated using an unpaired t-test with Whech&#x2019;s correction in GraphPrism v. 7 software. (A) Kinetic assays performed in the absence of drug. All slopes values were statistically tested against the WT value. Only WTG48L and MAXG48L displayed a lower replication capacity than the WT. (B) Kinetic assays performed in the presence of 20nM atazanavir (ATV). All slopes values were statistically tested against the WT value. WTG48L, MAXG48L and WTQ58E displayed a higher replication capacity than the WT in the presence of ATV. Values represent the mean &#xB1; standard deviation (SD) from at least three independent experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1851797889" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Replication kinetic assay of wild-type HIV-1 (WTp1) and the recoded MAX protease variant (MAXp1) in MT-4 cells. HIV-1 antigen p24 concentrations in culture supernatants were measured on days 0-4 in the absence of drug or days 0&#x2013;6 in the presence of drug. For each virus, the slope of the plot provides an estimate of the viral replication capacity. Bars show the slope of the p24 antigen production from each virus after infection of MT-4 cells. Comparison between WTp1 (HXB2) and the mutant MAX recoded viruses are shown, as well as between the corresponding WT and MAX virus variants (i.e.. S37P, G48L, Q58E, I84V and I89L). The significance of the difference between slopes was calculated using an unpaired t-test with Whech&#x2019;s correction in GraphPrism v. 7 software. &lt;strong&gt;(A)&lt;/strong&gt; Kinetic assays performed in the absence of drug. All slopes values were statistically tested against the WT value. Only WTG48L and MAXG48L displayed a lower replication capacity than the WT. &lt;strong&gt;(B)&lt;/strong&gt; Kinetic assays performed in the presence of 20nM atazanavir (ATV). All slopes values were statistically tested against the WT value. WTG48L, MAXG48L and WTQ58E displayed a higher replication capacity than the WT in the presence of ATV. Values represent the mean &#xB1; standard deviation (SD) from at least three independent experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F2.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 2" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F2.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIG 2</span> <p id="p-11" class="first-child">Replication kinetic assay of wild-type HIV-1 (WTp1) and the recoded MAX protease variant (MAXp1) in MT-4 cells. HIV-1 antigen p24 concentrations in culture supernatants were measured on days 0-4 in the absence of drug or days 0–6 in the presence of drug. For each virus, the slope of the plot provides an estimate of the viral replication capacity. Bars show the slope of the p24 antigen production from each virus after infection of MT-4 cells. Comparison between WTp1 (HXB2) and the mutant MAX recoded viruses are shown, as well as between the corresponding WT and MAX virus variants (i.e.. S37P, G48L, Q58E, I84V and I89L). The significance of the difference between slopes was calculated using an unpaired t-test with Whech’s correction in GraphPrism v. 7 software. <strong>(A)</strong> Kinetic assays performed in the absence of drug. All slopes values were statistically tested against the WT value. Only WTG48L and MAXG48L displayed a lower replication capacity than the WT. <strong>(B)</strong> Kinetic assays performed in the presence of 20nM atazanavir (ATV). All slopes values were statistically tested against the WT value. WTG48L, MAXG48L and WTQ58E displayed a higher replication capacity than the WT in the presence of ATV. Values represent the mean ± standard deviation (SD) from at least three independent experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">We subjected the WT and MAX viruses to selective pressure from two PIs: ATV and DRV. The viruses were propagated in duplicate in MT-4 cells over 32 serial passages (128 days of culture) without drugs or with increasing concentrations of ATV or DRV. The starting PI concentrations were near the half maximal inhibitory concentration (IC<sub>50</sub>) for the WT HXB2 virus (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">10</a>). Before the passages, both viruses showed similar IC<sub>50</sub> values for ATV and DRV (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">10</a>). The WTp32 and MAXp32 viruses still showed comparable IC<sub>50</sub> values after 32 serial passages in the presence or absence of PIs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). WTp32 and MAXp32, respectively, showed 5- fold and 13-fold increases in IC<sub>50</sub> for ATV, and 6-fold and 10-fold increases in IC<sub>50</sub> for DRV (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Although MAXp32 displayed a higher resistance to ATV and DRV than WTp32 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), these differences were not significant (P = 0.4816 and P = 0.3451, respectively). These assays demonstrated that the MAX variant virus did not show impaired capacity to develop phenotypic resistance to PIs.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486312/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/486312/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486312" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 1</span> <p id="p-13" class="first-child">Susceptibility of HIV-1 carrying WT or MAX proteases to atazanavir (ATV) or darunavir (DRV)</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">After the 32 cell passages in the presence of ATV or DRV, virus RNA was recovered. This RNA was RT-PCR amplified and ultra-deep sequenced, and we compared the frequencies of resistant mutations. For each of the two studied viruses and the two tested drugs, we sequenced between 1.9 × 10<sup>7</sup> and 4.1 × 10<sup>7</sup> individual protease nucleotides (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Sequence clonal analysis revealed no resistance-associated substitutions in viruses propagated without drugs (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). On the other hand, both WT and MAX viruses propagated in the presence of PIs developed previously described resistance mutations to ATV and DRV (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Moreover, the resistance variant repertoire differed between the MAX and WT viruses (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Specifically, the G16E substitution was observed only in the WT protease virus propagated with ATV or DRV. Notably, the WT protease required only a transition to develop this substitution, whereas the MAX protease would need two substitutions, a transition, and a transversion. Additionally, the L33F, G48L, Q58E/K, and I89L substitutions were detected only in the recoded MAX protease. Other accompanying substitutions were also detected only in the MAX virus (e.g., E21K, H69Y, and T91S), mainly in the MAX virus (e.g., L10F), or only in the WT virus (e.g., L23I, P39Q, and V77I). Interestingly, some resistance mutations selected by the MAX protease virus (i.e., G48L and I89L), are extremely rare non-polymorphic substitutions <em>in vivo</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>). This finding indicates that the MAX protease may explore a different sequence space than that of the WT protease. Similar to G16E, the I89L mutation requires two substitutions in the WT background and only one substitution in the MAX background. However, there were no obvious reasons for the preferential emergence of K45I, G48L, Q58E and I84V in the MAX background. Substitutions L10F, G16E, L33F, Q58E/K, V77I, I84V, and I89L have been previously associated with PIs resistance (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>). In contrast, E21K, L23I, P39Q, G48L, H69Y, and T91S have not been described as associated to PIs resistance.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/485937/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/485937/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/485937" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 2</span> <p id="p-15" class="first-child">Substitutions associated with protease inhibitor resistance detected after MT-4 cell passages in the presence of atazanavir (ATV), darunavir (DRV), or no drug (ND)</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486299/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/486299/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486299" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 3</span> <p id="p-16" class="first-child">Susceptibility of HIV-1 carrying WT or MAX protease variants to atazanavir (ATV).</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">To explore the favored emergence in the presence of ATV of G48L, Q58E, I84V and I89L in the MAX background and to determine the effect of these substitutions in the WT background, we used site-directed mutagenesis to introduce this these mutations in both the WT and MAX protease backgrounds. The substitution S37P, preferentially selected in the MAX background when the virus was propagated in the absence of drug (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), was also included in this analysis. We found that the five generated mutants displayed comparable IC<sub>50</sub> values to ATV in both backgrounds, WT and MAX (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). We also determined the replication capacity of these two ten mutant viruses in MT-4 cells with and without ATV (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). Similar to the IC50 results, the five tested variants exhibited similar replication capacities in both backgrounds, WT and MAX, in either absence or presence of 20nM ATV (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a>). Only WTG48L and MAXG48L showed a lower replication capacity in the absence of drug. Likewise, only substitutions at positions G48L and Q58E conferred an advantage when viruses were propagated in the presence of 20nM ATV. These results demonstrated that the G48L, Q58E, I84V and I89L substitutions were not intrinsically prohibited in the WT protease background, and that other factors must explain their low <em>in vivo</em> frequency.</p><p id="p-18">The amino acid mutant repertoire also differed between the WT and MAX viruses when they were propagated without drugs (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Only one variant, D30N, was detected in both virus populations. We do not know whether the observed variants are adaptive or neutral mutations. Regardless, completely different mutant spectra were detected in these two viruses.</p><p id="p-19">We performed a maximum likelihood phylogenetic reconstruction of all WT and MAX unique amino acid variants that were recovered after 32 MT-4 cell passages in the presence of ATV or DRV. Remarkably, the results showed that the two viruses, which shared an identical starting amino acid sequence, followed different evolutionary trajectories (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). Upon visual inspection of these phylogenetic trees, it was also apparent that the MAX protease generated higher amino acid variant diversity (see below).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum likelihood phylogram of wild-type (WT) and MAX unique HIV-1 protease amino acid variants selected after 32 passages in MT-4 cells and in the presence of (A) atazanavir (ATV) or (B) darunavir (DRV). Phylogenetic reconstruction was generated using a Jones-Taylor Thornton (JTT) model as implemented in the MEGA6 software package. Both phylogenetic trees showed that the WT and MAX viruses, which shared an identical starting amino acid sequence, followed different evolutionary trajectories. Blue and red labels correspond to WT and MAX variants, respectively. Green labels represent the starting HXB2 protease amino acid sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1851797889" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Maximum likelihood phylogram of wild-type (WT) and MAX unique HIV-1 protease amino acid variants selected after 32 passages in MT-4 cells and in the presence of &lt;strong&gt;(A)&lt;/strong&gt; atazanavir (ATV) or &lt;strong&gt;(B)&lt;/strong&gt; darunavir (DRV). Phylogenetic reconstruction was generated using a Jones-Taylor Thornton (JTT) model as implemented in the MEGA6 software package. Both phylogenetic trees showed that the WT and MAX viruses, which shared an identical starting amino acid sequence, followed different evolutionary trajectories. Blue and red labels correspond to WT and MAX variants, respectively. Green labels represent the starting HXB2 protease amino acid sequence.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F3.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="FIG 3" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F3.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/07/315366/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIG 3</span> <p id="p-20" class="first-child">Maximum likelihood phylogram of wild-type (WT) and MAX unique HIV-1 protease amino acid variants selected after 32 passages in MT-4 cells and in the presence of <strong>(A)</strong> atazanavir (ATV) or <strong>(B)</strong> darunavir (DRV). Phylogenetic reconstruction was generated using a Jones-Taylor Thornton (JTT) model as implemented in the MEGA6 software package. Both phylogenetic trees showed that the WT and MAX viruses, which shared an identical starting amino acid sequence, followed different evolutionary trajectories. Blue and red labels correspond to WT and MAX variants, respectively. Green labels represent the starting HXB2 protease amino acid sequence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">We next compared the overall population nucleotide diversity of the WT and MAX proteases after 32 passages in the absence or presence of ATV or DRV (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Overall, nucleotide sequence diversity was significantly higher in MAX populations propagated with ATV, 0.00946 ± 0.00005 vs. 0.00517 ± 0.00004 (<em>P</em> &lt; 0.0001), but not meaningful differences were observed in the presence of DRV, 0.00490 ± 0.00004 vs. 0.00482 ± 0.00003. As expected, in the presence of a PI, we detected a higher number of nonsynonymous substitutions than synonymous substitutions in both the WT and MAX virus proteases. However, the MAX populations always displayed a significantly higher diversity (<em>P</em> &lt; 0.0001) of either synonymous or nonsynonymous mutations (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Diversity was also strikingly higher in MAX populations when viruses were propagated in the absence of drug (0.00095 ± 0.00001 vs. 0.00057 ± 0.00001, <em>P</em> &lt; 0.0001) (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Interestingly, in the absence of drugs, the MAX population showed significantly higher nonsynonymous diversity (0.00117 ± 0.00001 vs. 0.00061 ± 0.00001, <em>P</em> &lt; 0.0001) but not synonymous diversity (0.000310 ± 0.00002 vs. 0.00053 ± 0.00002, <em>P</em> &lt; 0.0001). This suggested that the WT and MAX viruses are subjected to different selective forces in the absence of pressure from a PI. Compared to WT, the MAX populations also showed higher Shannon’s entropy values, another parameter for measuring genetic population diversity (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). Notably, after 32 passages in cell culture, we observed no significant reversions of the starting synonymous substitutions introduced in the MAX protease, either in the presence or absence of PIs. Overall, our results demonstrated that MAX viruses displayed higher population genetic diversity after 32 passages in cell culture in the presence or absence of PIs.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486043/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/486043/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486043" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 4</span> <p id="p-22" class="first-child">Summary of population metrics</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-4"><h2 class="">DISCUSSION</h2><p id="p-23">Previous research shows that codon usage can determine the mutational robustness, evolutionary capacity, and virulence of poliovirus (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>). Earlier results indicate that polioviruses with synonymously mutated capsids were less mutationally robust and displayed an attenuated phenotype in an animal model. However, that study did not focus on how synonymous mutations might affect the development of escape mutations to overcome specific selection pressure targeting a precise virus gene. Here we tested the extent to which a synonymously recoded HIV-1 protease reacted to the specific selective pressure of a PI. Our present study also explored the evolvability of a retrovirus which, in contrast to other RNA viruses, integrates into the host cell genome such that viral proteins are translated from mRNAs using host cellular machinery.</p><p id="p-24">We found that the WT and MAX protease viruses displayed different patterns of resistance mutations after PI treatment. These findings extend those of Lauring et al. (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>), confirming that synonymously recoded and WT HIV-1 proteases occupy different sequence spaces. We further demonstrated that although the MAX and WT proteases occupied different sequence spaces, they still showed similar development of phenotypic resistance to PIs. These findings indicate that the recoded protease did not attenuate the virus’ capability to develop PI resistance, strongly suggesting that the MAX protease was as robust as the WT protease with regards to this trait. To our knowledge, this is the first study to investigate the evolvability of a synonymously recoded virus enzyme. Even if the resistance selection was conducted in MT4 cells and the result probably would be qualitatively different in primary cells, our results build on and augment the convincing evidence that recoded proteins occupy a different sequence space.</p><p id="p-25">In some instances, the different mutant repertoire within the MAX protease background can be easily explained by proximity within the corresponding sequence space (e.g., G16E and L89I). However, with regards to other mutations, the explanation for the difference is not readily apparent (e.g., S37P, G48L, Q58E and I84V). In particular, the G48L substitution is very rarely selected <em>in vivo</em> in patients undergoing PI therapy (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>). However, when the above substitutions (S37P, G48L, Q58E, I84V or L89I) were introduced in the WT sequence they showed parallel replication capacities to those observed with a MAX background. We can speculate that the introduced synonymous substitutions affected neighboring residues (e.g., RNA structure). However, it must be noted that the MAX and WT proteases have similar RNA folding free energy (<a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">10</a>).</p><p id="p-26">One plausible explanation is based on epistatic interactions between protease amino acid substitutions. Epistasis is a phenomenon by which a mutation’s impact on protein stability or fitness depends on the genetic background in which it is acquired (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). Complex mutational patterns often arise during the development of resistance to HIV-1 protease inhibitors. More therapy-associated mutations accumulate under PI therapy than under all other types of antiretroviral therapy. Moreover, among patients experiencing therapy failure, the majority of <em>in vivo</em> drug-experienced protease sequences include over four mutations associated with PI therapy (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). Recent findings suggest that the consequences of acquiring primary HIV-1 protease resistance mutations depend on epistatic interactions with the sequence background (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). In our study, over 80% of the MAX protease clones harboring the G48L mutation also had the I89L substitution. As mentioned above, I89L requires two nucleotide substitutions in the WT background and only one in the MAX background. In either case, our results strongly suggest that the MAX protease’s sequence position affects its genotypic PI resistance profile. Synonymous codons differ in their propensity to mutate and, as previously suggested (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>), this differential access to protein sequence space may affect adaptive pathways.</p><p id="p-27">Another intriguing finding of our study is that the MAX virus showed higher population diversity in the recoded and targeted gene following propagation in both the absence and presence of PIs. Again, we can speculate that although the MAX virus shows high fitness in tissue culture, it is subjected to greater pressure to changing or reverting to a WT synonymous background. However, remarkably, we detected almost no reversions of the synonymous substitutions introduced in the MAX protease following propagation in the presence or absence of PIs. One limitation of our study is that we investigated only one virus enzyme or protein. Further studies should include other virus proteins and other selective pressures (e.g., neutralizing antibodies and cellular virus restriction factors).</p><p id="p-28">It has been suggested that RNA virus synonymous recoding can be used to push a virus to a sequence space region having a low density of neutral mutations (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">32</a>). Such a lack of access to neutral substitutions could potentially reduce the virus’ capacity to generate fit progeny and adaptability to the host’s selective pressures, such that this method might serve as a new strategy for development of attenuated vaccines. Our present data suggest that this approach must be developed cautiously, and support a need to evaluate the long-term stability of synonymously recoded viruses and to carefully test individual candidates.</p></div><div class="section" id="sec-5"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-6" class="subsection"><h3>Cell line and viruses</h3><p id="p-29">MT-4 cells were obtained from the National Institutes of Health (NIH) AIDS Research and Reference Reagent Program, and were grown in Roswell Park Memorial Institute (RPMI) 1640 L-glutamine medium supplemented with 10% heat-inactivated fetal bovine serum (FBS) (Gibco). The utilized WT virus corresponded to the HIV-1 HXB2 strain (<a href="http://www.hiv.lanl.gov">http://www.hiv.lanl.gov</a>) (Genbank accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=K03455&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom">K03455</a>). The synthetic MAX HIV-1 protease was generated by PCR amplification with a combination of three overlapping synthetic DNA oligonucleotides, as previously described (<a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">10</a>). In MT-4 cells, the MAX protease PCR product was recombined with a protease-deleted HXB2 infectious clone that had been previously linearized with BstE II (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>). The protease PCR oligonucleotides used to reconstruct the full-length protease have been previously described (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>). WT and MAX S37P, G48L, Q58E, I84V and I89L mutants were generated by site-directed mutagenesis using overlap extension PCR with mutated oligonucleotides as previously described (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). Again, the mutant protease PCR products were recombined with the protease-deleted HXB2 infectious clone in MT-4 cells. Cell culture supernatants were harvested at 3, 5, and 7 days post-transfection when the HIV-1 p24 antigen concentration surpassed 500 ng/ml as measured by the Genscreen HIV-1 Ag assay (Bio-Rad). Virus titration was performed in MT-4 cells, and values were expressed as tissue culture dose for 50% infectivity (TCID<sub>50</sub>) as previously described (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>).</p></div><div id="sec-7" class="subsection"><h3>Replication capacity assays</h3><p id="p-30">Viral replication kinetics were analyzed by infecting 1 × 10<sup>6</sup> MT-4 cells with 200 TCID<sub>50</sub> (MOI of 0.0002). The infected cells were incubated for 4 h at 37°C and 5% CO<sub>2</sub>, washed twice with phosphate-buffered saline (PBS), and then resuspended in RPMI medium supplemented with 10% FBS. To quantify viral replication, we measured the HIV-1 capsid p24 antigen concentration in 200-μl aliquots of supernatant collected every 24 h for 4–6 days. Growth kinetics were analyzed by fitting a linear model to the log-transformed p24 data during the exponential growth phase using maximum likelihood methods as previously described (<a id="xref-ref-10-10" class="xref-bibr" href="#ref-10">10</a>).</p></div><div id="sec-8" class="subsection"><h3>HIV-1 drug susceptibility tests</h3><p id="p-31">ATV and DRV were obtained from the NIH AIDS Research and Reference Reagent Program. Following virus propagation and titration, we used a tetrazolium-based colorimetric method to determine the HIV-1 drug susceptibility (IC<sub>50</sub>) to ATV and DRV in MT-4 cells using a MOI of 0.003, as previously described (<a id="xref-ref-10-11" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>).</p></div><div id="sec-9" class="subsection"><h3>Selection of ATV- and DRV-resistant viruses</h3><p id="p-32">WT and MAX viruses were added at an MOI of 0.01 to 1 × 10<sup>6</sup> MT-4 cells, and the cells were maintained as described above. After 4 days, we transferred one-tenth of the culture, including cells and supernatant, into 1 × 10<sup>6</sup> fresh MT-4 cells. All virus passages were performed in duplicate. Virus production was monitored by measurements of p24 antigen. The starting concentrations were 4 nM ATV and 3 nM DRV. Through the passages, the drug concentration was increased until reaching 40 nM ATV and 25 nM DRV. In parallel, both viruses were also propagated without either drug.</p><p id="p-33">At passages 1 and 32, 140-μl aliquots of culture supernatant were collected, from which we isolated WT and MAX viral genomic RNA using the QIAamp Viral RNA Kit (QIAGEN). This purified viral RNA was then reverse transcribed and PCR amplified using the SuperScript III First-Strand Synthesis System for RT-PCR (Invitrogen) and 10 pmol of the corresponding protease oligonucleotides, which are described elsewhere (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>). Details of this protocol were previously reported (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>). These PCR products were the starting material for performing ultra-deep sequencing.</p></div><div id="sec-10" class="subsection"><h3>Ultra-deep sequencing</h3><p id="p-34">Massive parallel sequencing was performed in the MiSeq (Illumina) platform. Libraries of 558-nt DNA fragments were ligated to the Illumina adapters using the KAPA HyperPrep Kit (Roche #07962347001), SeqCap Adapter Kit A (Roche #07141530001), and SeqCap Adapter Kit B (Roche #07141548001). The products were purified using KAPA Pure Beads (Roche #07983280001). All libraries were quantified using the Qubit<sup>®</sup> dsDNA HS Assay Kit (ThermoFisher #Q32854) and a Qubit Fluorometer (ThermoFisher #Q33216), and were qualified using the Agilent DNA 1000 Kit (Agilent #5067-1504) and a bioanalyzer (Agilent #G2939BA). Sequencing was performed using the Illumina MiSeq<sup>®</sup> Reagent Kit v3 (600 cycle) (Illumina #MS-102-3003) following manufacturer’s protocol. Sequencing and paired-end analysis were performed to obtain robust fastq data for bioinformatics analysis. We obtained a mean of 50,000 sequences (reads) per amplicon and per patient sample.</p><p id="p-35">Fastq files by index and pool were obtained from MiSeq and submitted to FLASH (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>). The 2 × 300 paired-end reads were overlapped to reconstruct the amplicons, with the minimum number of overlapping nucleotides set to 20, and the maximum number of overlapping mismatches set to 10%. The subsequent analysis was performed as previously described (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>). Briefly, fastq files were demultiplexed using amplicon oligonucleotides, and oligonucleotides were trimmed at both ends. Each amplicon and strand read was pairwise aligned with respect to the reference WT sequence, insertions were removed, and deletions were repaired if fewer than three gaps were produced. Reads with multiple indeterminations were removed, while reads having a single indetermination were repaired as per the reference sequence. Filtered and repaired reads were collapsed into haplotypes with corresponding frequencies. Haplotypes with abundances below 0.1% or that were unique to the forward or reverse strands were removed. Haplotypes common to the forward and reverse strands and with abundances of ≥0.1%, were considered consensus haplotypes, and their frequencies were summed. In the final step to remove artifacts, consensus haplotypes with abundances below 0.5% were filtered out. All computations were performed in the R language and platform, using in-house developed scripts as well as the packages Biostrings (R package 2.24.1, 2012), Ape (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>), and Seqinr (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>).</p><p id="p-36">Virus population genetic diversity (p-distance) was determined using the MEGA6 software package (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>). To determine possible selective pressures, the MEGA6 software package was used to calculate the proportion of synonymous substitutions per potential synonymous sites, and the proportion of nonsynonymous substitutions per potential nonsynonymous sites. Shannon’s entropy values were calculated as Sn = −Σ<sub>i</sub> (p<sub>i</sub>ln p<sub>i</sub>)/ln N, where N is the total number of analyzed sequences and p<sub>i</sub> is the frequency of each sequence in the viral quasispecies. Sn values vary from 0 (no complexity) to 1 (maximum complexity) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). The phylogenetic reconstructions were also performed using the MEGA6 software package.</p></div><div id="sec-11" class="subsection"><h3>Statistical analysis</h3><p id="p-37">Virus population diversity was compared by unpaired t-test using GraphPad Prism version 7 for Windows. The significance of the difference between replication kinetic slopes and IC<sub>50</sub>s was calculated using an unpaired t-test with Whech’s correction as implemented in GraphPad Prism.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-38">This study was supported by the Spanish Ministry of Economy and Competitiveness (SAF2016-75277-R) by Instituto de Salud Carlos III (PI16/00337), and cofinanced by the European Regional Development Fund (ERDF). MN was supported by the Instituto de Salud Carlos III through the Spanish AIDS network (RD16/0025/0041). AJ-P was supported by a contract of the Spanish Ministry of Economy and Competitiveness (BES-2014-069931).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simhadri</span>  <span class="cit-name-given-names">VL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iandoli</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauna</span>  <span class="cit-name-given-names">ZE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kimchi-Sarfaty</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Exposing synonymous mutations</span>. <abbr class="cit-jnl-abbrev">Trends in genetics: TIG</abbr> <span class="cit-vol">30</span>:<span class="cit-fpage">308</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%253A%2BTIG%26rft.volume%253D30%26rft.spage%253D308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias. Nature reviews</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D12%26rft.spage%253D32%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.3" data-doi="10.1016/j.tim.2015.11.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jordan-Paiz</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nevot</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Synonymous Virus Genome Recoding as a Tool to Impact Viral Fitness</span>. <abbr class="cit-jnl-abbrev">Trends in microbiology</abbr> <span class="cit-vol">24</span>:<span class="cit-fpage">134</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.volume%253D24%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2015.11.002%26rft_id%253Dinfo%253Apmid%252F26646373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2015.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26646373&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.4" data-doi="10.1128/JVI.00508-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campagnoli</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jorba</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kew</span>  <span class="cit-name-given-names">O</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Genetic inactivation of poliovirus infectivity by increasing the frequencies of CpG and UpA dinucleotides within and across synonymous capsid region codons</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">83</span>:<span class="cit-fpage">9957</span>–<span class="cit-lpage">9969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00508-09%26rft_id%253Dinfo%253Apmid%252F19605476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4My8xOS85OTU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDcvMzE1MzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-315366v1.5" data-doi="10.1128/JVI.80.7.3259-3272.2006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campagnoli</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jorba</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quay</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kew</span>  <span class="cit-name-given-names">O</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Modulation of poliovirus replicative fitness in HeLa cells by deoptimization of synonymous codon usage in the capsid region</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">80</span>:<span class="cit-fpage">3259</span>–<span class="cit-lpage">3272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBurns%26rft.auinit1%253DC.%2BC.%26rft.volume%253D80%26rft.issue%253D7%26rft.spage%253D3259%26rft.epage%253D3272%26rft.atitle%253DModulation%2Bof%2BPoliovirus%2BReplicative%2BFitness%2Bin%2BHeLa%2BCells%2Bby%2BDeoptimization%2Bof%2BSynonymous%2BCodon%2BUsage%2Bin%2Bthe%2BCapsid%2BRegion%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.80.7.3259-3272.2006%26rft_id%253Dinfo%253Apmid%252F16537593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgwLzcvMzI1OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzA3LzMxNTM2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.6" data-doi="10.1126/science.1155761"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Virus attenuation by genome-scale changes in codon pair bias</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>:<span class="cit-fpage">1784</span>–<span class="cit-lpage">1787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColeman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D320%26rft.issue%253D5884%26rft.spage%253D1784%26rft.epage%253D1787%26rft.atitle%253DVirus%2BAttenuation%2Bby%2BGenome-Scale%2BChanges%2Bin%2BCodon%2BPair%2BBias%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1155761%26rft_id%253Dinfo%253Apmid%252F18583614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4NC8xNzg0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDcvMzE1MzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.7" data-doi="10.1128/JVI.00738-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Reduction of the rate of poliovirus protein synthesis through large-scale codon deoptimization causes attenuation of viral virulence by lowering specific infectivity</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">80</span>:<span class="cit-fpage">9687</span>–<span class="cit-lpage">9696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMueller%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D19%26rft.spage%253D9687%26rft.epage%253D9696%26rft.atitle%253DReduction%2Bof%2Bthe%2BRate%2Bof%2BPoliovirus%2BProtein%2BSynthesis%2Bthrough%2BLarge-Scale%2BCodon%2BDeoptimization%2BCauses%2BAttenuation%2Bof%2BViral%2BVirulence%2Bby%2BLowering%2BSpecific%2BInfectivity%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00738-06%26rft_id%253Dinfo%253Apmid%252F16973573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xOS85Njg3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDcvMzE1MzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.8" data-doi="10.1038/nbt.1636"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nimnual</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Live attenuated influenza virus vaccines by computer-aided rational design</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-vol">28</span>:<span class="cit-fpage">723</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMueller%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D723%26rft.epage%253D726%26rft.atitle%253DLive%2Battenuated%2Binfluenza%2Bvirus%2Bvaccines%2Bby%2Bcomputer-aided%2Brational%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1636%26rft_id%253Dinfo%253Apmid%252F20543832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1636&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20543832&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279723900029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.9" data-doi="10.1073/pnas.1307473110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Deliberate reduction of hemagglutinin and neuraminidase expression of influenza virus leads to an ultraprotective live vaccine in mice</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">110</span>:<span class="cit-fpage">9481</span>–<span class="cit-lpage">9486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1307473110%26rft_id%253Dinfo%253Apmid%252F23690603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIzLzk0ODEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8wNy8zMTUzNjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.10" data-doi="10.1186/1742-4690-10-78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martrus</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nevot</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andres</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clotet</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Changes in codon-pair bias of human immunodeficiency virus type 1 have profound effects on virus replication in cell culture</span>. <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRetrovirology%26rft.volume%253D10%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-4690-10-78%26rft_id%253Dinfo%253Apmid%252F23885919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-4690-10-78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23885919&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.11" data-doi="10.1038/nature24039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takata</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goncalves-Carneiro</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zang</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soll</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">York</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanco-Melo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">CG dinucleotide suppression enables antiviral defence targeting non-self RNA</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">550</span>:<span class="cit-fpage">124</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D550%26rft.spage%253D124%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24039%26rft_id%253Dinfo%253Apmid%252F28953888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28953888&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.12" data-doi="10.1371/journal.ppat.1006824"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takata</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soll</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanco-Melo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swanstrom</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>  <span class="cit-name-given-names">PD</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Global synonymous mutagenesis identifies cis-acting RNA elements that regulate HIV-1 splicing and replication</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">e1006824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D14%26rft.spage%253De1006824%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006824&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.13" data-doi="10.1128/JVI.03223-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vabret</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailly-Bechet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lepelley</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Najburg</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verrier</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tangy</span>  <span class="cit-name-given-names">F</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Large-scale nucleotide optimization of simian immunodeficiency virus reduces its capacity to stimulate type I interferon in vitro</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">88</span>:<span class="cit-fpage">4161</span>–<span class="cit-lpage">4172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03223-13%26rft_id%253Dinfo%253Apmid%252F24478441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzgvNDE2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzA3LzMxNTM2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.14" data-doi="10.1371/journal.ppat.1003172"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nougairede</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Fabritus</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aubry</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Lamballerie</span>  <span class="cit-name-given-names">X</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Random codon re-encoding induces stable reduction of replicative fitness of Chikungunya virus in primate and mosquito cells</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">e1003172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D9%26rft.spage%253De1003172%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003172%26rft_id%253Dinfo%253Apmid%252F23436995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23436995&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.15" data-doi="10.1073/pnas.1411290111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Nouen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">LG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luongo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mehedi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchholz</span>  <span class="cit-name-given-names">UJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">DiNapoli</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Attenuation of human respiratory syncytial virus by genome-scale codon-pair deoptimization</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">111</span>:<span class="cit-fpage">13169</span>–<span class="cit-lpage">13174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1411290111%26rft_id%253Dinfo%253Apmid%252F25157129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM2LzEzMTY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDcvMzE1MzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-315366v1.16" data-doi="10.1073/pnas.1619242114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Nouen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lleras</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mehedi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luongo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munir</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">DiNapoli</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchholz</span>  <span class="cit-name-given-names">UJ</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Genetic stability of genome-scale deoptimized RNA virus vaccine candidates under selective pressure</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">114</span>:<span class="cit-fpage">E386</span>–<span class="cit-lpage">E395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1619242114%26rft_id%253Dinfo%253Apmid%252F28049853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE0LzMvRTM4NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzA3LzMxNTM2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.17" data-doi="10.1128/mBio.01704-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hotard</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">ML</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Refining the balance of attenuation and immunogenicity of respiratory syncytial virus by targeted codon deoptimization of virulence genes</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">e01704</span>–<span class="cit-lpage">01714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D5%26rft.spage%253De01704%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01704-14%26rft_id%253Dinfo%253Apmid%252F25249281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01704-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25249281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.18" data-doi="10.1016/j.virol.2013.12.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">YY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Opriessnig</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subramaniam</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">XJ</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Computer-aided codon-pairs deoptimization of the major envelope GP5 gene attenuates porcine reproductive and respiratory syndrome virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">450-451</span>:<span class="cit-fpage">132</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D450451%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2013.12.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2013.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.19" data-doi="10.1093/nar/gku075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witteveldt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">The influence of CpG and UpA dinucleotide frequencies on RNA virus replication and characterization of the innate cellular pathways underlying virus attenuation and enhanced replication</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">42</span>:<span class="cit-fpage">4527</span>–<span class="cit-lpage">4545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku075%26rft_id%253Dinfo%253Apmid%252F24470146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24470146&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.20" data-doi="10.7554/eLife.04531"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tulloch</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">RNA virus attenuation by codon pair deoptimisation is an artefact of increases in CpG/UpA dinucleotide frequencies</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">e04531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.04531%26rft_id%253Dinfo%253Apmid%252F25490153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.04531&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25490153&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.21" data-doi="10.1371/journal.ppat.1004738"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Fabritus</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nougairede</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aubry</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Lamballerie</span>  <span class="cit-name-given-names">X</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Attenuation of tick-borne encephalitis virus using large-scale random codon re-encoding</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">e1004738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D11%26rft.spage%253De1004738%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004738%26rft_id%253Dinfo%253Apmid%252F25734338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25734338&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.22" data-doi="10.1073/pnas.1502864112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stauft</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbatsevych</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yurovsky</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Large-scale recoding of an arbovirus genome to rebalance its insect versus mammalian preference</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">112</span>:<span class="cit-fpage">4749</span>–<span class="cit-lpage">4754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1502864112%26rft_id%253Dinfo%253Apmid%252F25825721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE1LzQ3NDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8wNy8zMTUzNjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.23" data-doi="10.1073/pnas.1102883108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sitaraman</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hearing</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gnatenko</span>  <span class="cit-name-given-names">DV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bahou</span>  <span class="cit-name-given-names">WF</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Computationally designed adeno-associated virus (AAV) Rep 78 is efficiently maintained within an adenovirus vector</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">108</span>:<span class="cit-fpage">14294</span>–<span class="cit-lpage">14299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102883108%26rft_id%253Dinfo%253Apmid%252F21844368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM0LzE0Mjk0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDcvMzE1MzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cladel</span>  <span class="cit-name-given-names">NM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Budgeon</span>  <span class="cit-name-given-names">LR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balogh</span>  <span class="cit-name-given-names">KK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">ND</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Synonymous codon changes in the oncogenes of the cottontail rabbit papillomavirus lead to increased oncogenicity and immunogenicity of the virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">438</span>:<span class="cit-fpage">70</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D438%26rft.spage%253D70%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.25" data-doi="10.1016/j.chom.2018.03.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitfield</span>  <span class="cit-name-given-names">ZJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Mapping the Evolutionary Potential of RNA Viruses</span>. <abbr class="cit-jnl-abbrev">Cell host &amp; microbe</abbr> <span class="cit-vol">23</span>:<span class="cit-fpage">435</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bhost%2B%2526%2Bmicrobe%26rft.volume%253D23%26rft.spage%253D435%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2018.03.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2018.03.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-315366v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Viral quasispecies</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">479-480</span>:<span class="cit-fpage">46</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D479480%26rft.spage%253D46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.27" data-doi="10.1016/j.jmb.2010.02.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mas</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez-Galindez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cacho</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Unfinished stories on viral quasispecies and Darwinian views of evolution</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">397</span>:<span class="cit-fpage">865</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMas%26rft.auinit1%253DA.%26rft.volume%253D397%26rft.issue%253D4%26rft.spage%253D865%26rft.epage%253D877%26rft.atitle%253DUnfinished%2Bstories%2Bon%2Bviral%2Bquasispecies%2Band%2BDarwinian%2Bviews%2Bof%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2010.02.005%26rft_id%253Dinfo%253Apmid%252F20152841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2010.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20152841&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.28" data-doi="10.1073/pnas.1211484109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Identification of two functionally redundant RNA elements in the coding sequence of poliovirus using computer-generated design</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">109</span>:<span class="cit-fpage">14301</span>–<span class="cit-lpage">14307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1211484109%26rft_id%253Dinfo%253Apmid%252F22886087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM2LzE0MzAxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDcvMzE1MzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.29" data-doi="10.1038/nature11433"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandig</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Limmer</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavon-Eternod</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">TE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weitzman</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Codon-usage-based inhibition of HIV protein synthesis by human schlafen 11</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>:<span class="cit-fpage">125</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DM.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D125%26rft.epage%253D128%26rft.atitle%253DCodon-usage-based%2Binhibition%2Bof%2BHIV%2Bprotein%2Bsynthesis%2Bby%2Bhuman%2Bschlafen%2B11.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11433%26rft_id%253Dinfo%253Apmid%252F23000900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23000900&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.30" data-doi="10.1016/j.it.2016.10.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vabret</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhardwaj</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>  <span class="cit-name-given-names">BD</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Sequence-Specific Sensing of Nucleic Acids</span>. <abbr class="cit-jnl-abbrev">Trends in immunology</abbr> <span class="cit-vol">38</span>:<span class="cit-fpage">53</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bimmunology%26rft.volume%253D38%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.it.2016.10.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.it.2016.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.31" data-doi="10.1073/pnas.1515387112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">YC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bischof</span>  <span class="cit-name-given-names">GF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauer</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desrosiers</span>  <span class="cit-name-given-names">RC</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Importance of codon usage for the temporal regulation of viral gene expression</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">112</span>:<span class="cit-fpage">14030</span>–<span class="cit-lpage">14035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1515387112%26rft_id%253Dinfo%253Apmid%252F26504241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzQ1LzE0MDMwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDcvMzE1MzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.32" data-doi="10.1016/j.chom.2012.10.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauring</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acevedo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">SB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Codon usage determines the mutational robustness, evolutionary capacity, and virulence of an RNA virus</span>. <abbr class="cit-jnl-abbrev">Cell host &amp; microbe</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">623</span>–<span class="cit-lpage">632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DLauring%26rft.auinit1%253DA.%2BS.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D623%26rft.epage%253D632%26rft.atitle%253DCodon%2Busage%2Bdetermines%2Bthe%2Bmutational%2Brobustness%252C%2Bevolutionary%2Bcapacity%252C%2Band%2Bvirulence%2Bof%2Ban%2BRNA%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2012.10.008%26rft_id%253Dinfo%253Apmid%252F23159052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2012.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23159052&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311765100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.33" data-doi="10.1038/nrmicro3003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauring</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">The role of mutational robustness in RNA virus evolution</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">327</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BMicrobiol%26rft.volume%253D11%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro3003%26rft_id%253Dinfo%253Apmid%252F23524517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro3003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23524517&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.34" data-doi="10.1007/BF00623322"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eigen</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">1971</span>. <span class="cit-article-title">Selforganization of matter and the evolution of biological macromolecules</span>. <abbr class="cit-jnl-abbrev">Naturwissenschaften</abbr> <span class="cit-vol">58</span>:<span class="cit-fpage">465</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNaturwissenschaften%26rft.stitle%253DNaturwissenschaften%26rft.aulast%253DEigen%26rft.auinit1%253DM.%26rft.volume%253D58%26rft.issue%253D10%26rft.spage%253D465%26rft.epage%253D523%26rft.atitle%253DSelforganization%2Bof%2Bmatter%2Band%2Bthe%2Bevolution%2Bof%2Bbiological%2Bmacromolecules.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00623322%26rft_id%253Dinfo%253Apmid%252F4942363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00623322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4942363&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1971K783800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.35" data-doi="10.1016/0092-8674(78)90223-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taniguchi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weissmann</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-pub-date">1978</span>. <span class="cit-article-title">Nucleotide sequence heterogeneity of an RNA phage population</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">735</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDomingo%26rft.auinit1%253DE.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D735%26rft.epage%253D744%26rft.atitle%253DNucleotide%2Bsequence%2Bheterogeneity%2Bof%2Ban%2BRNA%2Bphage%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252878%252990223-4%26rft_id%253Dinfo%253Apmid%252F657273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(78)90223-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=657273&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978EW03300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.36" data-doi="10.1186/1742-4690-9-92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van der Kuyl</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berkhout</span>  <span class="cit-name-given-names">B</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">The biased nucleotide composition of the HIV genome: a constant factor in a highly variable virus</span>. <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRetrovirology%26rft.aulast%253Dvan%2Bder%2BKuyl%26rft.auinit1%253DA.%2BC.%26rft.volume%253D9%26rft.spage%253D92%26rft.epage%253D92%26rft.atitle%253DThe%2Bbiased%2Bnucleotide%2Bcomposition%2Bof%2Bthe%2BHIV%2Bgenome%253A%2Ba%2Bconstant%2Bfactor%2Bin%2Ba%2Bhighly%2Bvariable%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-4690-9-92%26rft_id%253Dinfo%253Apmid%252F23131071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-4690-9-92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23131071&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.37" data-doi="10.1093/bioinformatics/btp134"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">TF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rhee</span>  <span class="cit-name-given-names">SY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiuchi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hue</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pillay</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">RW</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The calibrated population resistance tool: standardized genotypic estimation of transmitted HIV-1 drug resistance</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">1197</span>–<span class="cit-lpage">1198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp134%26rft_id%253Dinfo%253Apmid%252F19304876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19304876&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265523300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wensing</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvez</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gunthard</span>  <span class="cit-name-given-names">HF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">VA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paredes</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pillay</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richman</span>  <span class="cit-name-given-names">DD</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">2017 Update of the Drug Resistance Mutations in HIV-1</span>. <abbr class="cit-jnl-abbrev">Topics in antiviral medicine</abbr> <span class="cit-vol">24</span>:<span class="cit-fpage">132</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTopics%2Bin%2Bantiviral%2Bmedicine%26rft.volume%253D24%26rft.spage%253D132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.39" data-doi="10.1093/molbev/msu113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parera</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Strong epistatic interactions within a single protein</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">31</span>:<span class="cit-fpage">1546</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu113%26rft_id%253Dinfo%253Apmid%252F24682281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24682281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000337067400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schapiro</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">HIV-1 drug resistance mutations: an updated framework for the second decade of HAART</span>. <abbr class="cit-jnl-abbrev">AIDS reviews</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">67</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2Breviews%26rft.stitle%253DAIDS%2BRev%26rft.aulast%253DShafer%26rft.auinit1%253DR.%2BW.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D67%26rft.epage%253D84%26rft.atitle%253DHIV-1%2Bdrug%2Bresistance%2Bmutations%253A%2Ban%2Bupdated%2Bframework%2Bfor%2Bthe%2Bsecond%2Bdecade%2Bof%2BHAART.%26rft_id%253Dinfo%253Apmid%252F18615118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18615118&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257941100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.41" data-doi="10.1093/molbev/msx095"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Flynn</span>  <span class="cit-name-given-names">WF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torbett</span>  <span class="cit-name-given-names">BE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">RM</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Inference of Epistatic Effects Leading to Entrenchment and Drug Resistance in HIV-1 Protease</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">34</span>:<span class="cit-fpage">1291</span>–<span class="cit-lpage">1306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft.volume%253D34%26rft.spage%253D1291%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.42" data-doi="10.1371/journal.pgen.1000256"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cambray</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mazel</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Synonymous genes explore different evolutionary landscapes</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">e1000256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCambray%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De1000256%26rft.epage%253De1000256%26rft.atitle%253DSynonymous%2Bgenes%2Bexplore%2Bdifferent%2Bevolutionary%2Blandscapes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000256%26rft_id%253Dinfo%253Apmid%252F19008944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19008944&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maschera</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furfine</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">ED</span></span>. <span class="cit-pub-date">1995</span>. <span class="cit-article-title">Analysis of resistance to human immunodeficiency virus type 1 protease inhibitors by using matched bacterial expression and proviral infection vectors</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">69</span>:<span class="cit-fpage">5431</span>–<span class="cit-lpage">5436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMaschera%26rft.auinit1%253DB.%26rft.volume%253D69%26rft.issue%253D9%26rft.spage%253D5431%26rft.epage%253D5436%26rft.atitle%253DAnalysis%2Bof%2Bresistance%2Bto%2Bhuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1%2Bprotease%2Binhibitors%2Bby%2Busing%2Bmatched%2Bbacterial%2Bexpression%2Band%2Bproviral%2Binfection%2Bvectors%26rft_id%253Dinfo%253Apmid%252F7636988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY5LzkvNTQzMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzA3LzMxNTM2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.44" data-doi="10.1128/JVI.78.24.14057-14061.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parera</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clotet</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Genetic screen for monitoring severe acute respiratory syndrome coronavirus 3C-like protease</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">78</span>:<span class="cit-fpage">14057</span>–<span class="cit-lpage">14061</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DParera%26rft.auinit1%253DM.%26rft.volume%253D78%26rft.issue%253D24%26rft.spage%253D14057%26rft.epage%253D14061%26rft.atitle%253DGenetic%2BScreen%2Bfor%2BMonitoring%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2B3C-Like%2BProtease%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.24.14057-14061.2004%26rft_id%253Dinfo%253Apmid%252F15564515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OC8yNC8xNDA1NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzA3LzMxNTM2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.45" data-doi="10.1016/0166-0934(88)90134-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pauwels</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balzarini</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baba</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snoeck</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schols</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herdewijn</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desmyter</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Clercq</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">1988</span>. <span class="cit-article-title">Rapid and automated tetrazolium-based colorimetric assay for the detection of anti-HIV compounds</span>. <abbr class="cit-jnl-abbrev">Journal of virological methods</abbr> <span class="cit-vol">20</span>:<span class="cit-fpage">309</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.stitle%253DJ%2BVirol%2BMethods%26rft.aulast%253DPauwels%26rft.auinit1%253DR.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D309%26rft.epage%253D321%26rft.atitle%253DRapid%2Band%2Bautomated%2Btetrazolium-based%2Bcolorimetric%2Bassay%2Bfor%2Bthe%2Bdetection%2Bof%2Banti-HIV%2Bcompounds.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0166-0934%252888%252990134-6%26rft_id%253Dinfo%253Apmid%252F2460479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0166-0934(88)90134-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2460479&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q217400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.46" data-doi="10.1093/nar/gkv077"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Betancor</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marcelli</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andres</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menendez-Arias</span>  <span class="cit-name-given-names">L</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Effects of HIV-1 reverse transcriptase connection subdomain mutations on polypurine tract removal and initiation of (+)-strand DNA synthesis</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">43</span>:<span class="cit-fpage">2259</span>–<span class="cit-lpage">2270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv077%26rft_id%253Dinfo%253Apmid%252F25662223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25662223&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.47" data-doi="10.1099/vir.0.045492-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Capel</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martrus</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parera</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clotet</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Evolution of the human immunodeficiency virus type 1 protease: effects on viral replication capacity and protease robustness</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">93</span>:<span class="cit-fpage">2625</span>–<span class="cit-lpage">2634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.045492-0%26rft_id%253Dinfo%253Apmid%252F22933665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.045492-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22933665&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.48" data-doi="10.1128/JVI.01594-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clotet</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Fitness landscape of human immunodeficiency virus type 1 protease quasispecies</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">81</span>:<span class="cit-fpage">2485</span>–<span class="cit-lpage">2496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01594-06%26rft_id%253Dinfo%253Apmid%252F17151104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzUvMjQ4NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA1LzA3LzMxNTM2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.49" data-doi="10.1093/bioinformatics/btr507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Magoc</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">FLASH: fast length adjustment of short reads to improve genome assemblies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>:<span class="cit-fpage">2957</span>–<span class="cit-lpage">2963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr507%26rft_id%253Dinfo%253Apmid%252F21903629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903629&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.50" data-doi="10.1371/journal.pone.0083361"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gregori</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Esteban</span>  <span class="cit-name-given-names">JI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cubero</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia-Cehic</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perales</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casillas</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez-Tejado</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez-Frias</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guardia</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quer</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Ultra-deep pyrosequencing (UDPS) data treatment to study amplicon HCV minor variants</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">e83361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D8%26rft.spage%253De83361%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0083361%26rft_id%253Dinfo%253Apmid%252F24391758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0083361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24391758&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.51" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claude</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">APE: Analyses of Phylogenetics and Evolution in R language</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>:<span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14734327&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188389700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.52" data-doi="10.1093/bioinformatics/bti037"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Charif</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thioulouse</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lobry</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perriere</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Online synonymous codon usage analyses with the ade4 and seqinR packages</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>:<span class="cit-fpage">545</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti037%26rft_id%253Dinfo%253Apmid%252F15374859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15374859&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227127000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.53" data-doi="10.1093/molbev/mst197"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">MEGA6: Molecular Evolutionary Genetics Analysis version 6.0</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">30</span>:<span class="cit-fpage">2725</span>–<span class="cit-lpage">2729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst197%26rft_id%253Dinfo%253Apmid%252F24132122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24132122&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F07%2F315366.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327793000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-315366v1.54" data-doi="10.1126/science.274.5289.1010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wolinsky</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kunstman</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Safrit</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koup</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">AU</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korber</span>  <span class="cit-name-given-names">BT</span></span>. <span class="cit-pub-date">1996</span>. <span class="cit-article-title">Response: HIV-1 Evolution and Disease Progression</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>:<span class="cit-fpage">1010</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWolinsky%26rft.auinit1%253DS.%2BM.%26rft.volume%253D274%26rft.issue%253D5289%26rft.spage%253D1010%26rft.epage%253D1011%26rft.atitle%253DResponse%253A%2BHIV-1%2BEvolution%2Band%2BDisease%2BProgression%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5289.1010%26rft_id%253Dinfo%253Apmid%252F17798610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzQvNTI4OS8xMDEwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDcvMzE1MzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/315366v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F315366v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/315382v1" title="Antibiotic-induced dysbiosis predicts mortality in an animal model of Clostridium difficile infection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 07, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/315366v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/315366v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/315366v1" class="active" data-icon-position="" data-hide-link-title="0">HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/96967" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zDVgSH1NLo0GfL_oTPLSXvZn5Tto6uapaOcY15HsdXI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="96967" id="node96967--2" data-pisa="biorxiv;315366v1" data-pisa-master="biorxiv;315366" data-seqnum="96967" data-apath="/biorxiv/early/2018/05/07/315366.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Nevot</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ana</span> <span class="nlm-surname">Jordan-Paiz</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Glòria</span> <span class="nlm-surname">Martrus</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Cristina</span> <span class="nlm-surname">Andrés</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Damir</span> <span class="nlm-surname">García-Cehic</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Josep</span> <span class="nlm-surname">Gregori</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sandra</span> <span class="nlm-surname">Franco</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Josep</span> <span class="nlm-surname">Quer</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Miguel Angel</span> <span class="nlm-surname">Martinez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">315366; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/315366 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/315366v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;title=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;title=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;text=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;title=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;t=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;title=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;title=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="96967" id="node96967--3" data-pisa="biorxiv;315366v1" data-pisa-master="biorxiv;315366" data-seqnum="96967" data-apath="/biorxiv/early/2018/05/07/315366.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">HIV-1 Protease Evolvability is Affected by Synonymous Nucleotide Recoding</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Nevot</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ana</span> <span class="nlm-surname">Jordan-Paiz</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Glòria</span> <span class="nlm-surname">Martrus</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Cristina</span> <span class="nlm-surname">Andrés</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Damir</span> <span class="nlm-surname">García-Cehic</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Josep</span> <span class="nlm-surname">Gregori</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sandra</span> <span class="nlm-surname">Franco</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Josep</span> <span class="nlm-surname">Quer</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Miguel Angel</span> <span class="nlm-surname">Martinez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">315366; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/315366 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Nevot%2C+Maria&amp;rft.au=Jordan-Paiz%2C+Ana&amp;rft.au=Martrus%2C+Gl%C3%B2ria&amp;rft.au=Andr%C3%A9s%2C+Cristina&amp;rft.au=Garc%C3%ADa-Cehic%2C+Damir&amp;rft.au=Gregori%2C+Josep&amp;rft.au=Franco%2C+Sandra&amp;rft.au=Quer%2C+Josep&amp;rft.au=Martinez%2C+Miguel+Angel"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/96967/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/96967/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/96967/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/96967/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/96967/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/96967/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/96967/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/96967/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/96967/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/96967/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/96967/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/96967/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;count=horizontal&amp;via=&amp;text=HIV-1%20Protease%20Evolvability%20is%20Affected%20by%20Synonymous%20Nucleotide%20Recoding&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/315366v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/315366v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/315366v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
