<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5599654">5599654</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28912478">28912478</a></span><span class="publisher-id" title="publisher-id">10286</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-017-10286-1">10.1038/s41598-017-10286-1</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
The molecular epidemiology of HIV-1 in the Comunidad Valenciana (Spain): analysis of transmission clusters</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patiño-Galindo</span><span class="given-names" tagx="given-names" title="given-names">Juan �?ngel</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torres-Puente</span><span class="given-names" tagx="given-names" title="given-names">Manoli</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bracho</span><span class="given-names" tagx="given-names" title="given-names">María Alma</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alastrué</span><span class="given-names" tagx="given-names" title="given-names">Ignacio</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juan</span><span class="given-names" tagx="given-names" title="given-names">Amparo</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Navarro</span><span class="given-names" tagx="given-names" title="given-names">David</span></span><a href="#Aff4">4</a><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galindo</span><span class="given-names" tagx="given-names" title="given-names">María José</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ocete</span><span class="given-names" tagx="given-names" title="given-names">Dolores</span></span><a href="#Aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ortega</span><span class="given-names" tagx="given-names" title="given-names">Enrique</span></span><a href="#Aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gimeno</span><span class="given-names" tagx="given-names" title="given-names">Concepción</span></span><a href="#Aff5">5</a><a href="#Aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belda</span><span class="given-names" tagx="given-names" title="given-names">Josefina</span></span><a href="#Aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domínguez</span><span class="given-names" tagx="given-names" title="given-names">Victoria</span></span><a href="#Aff8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moreno</span><span class="given-names" tagx="given-names" title="given-names">Rosario</span></span><a href="#Aff8">8</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0879-5798</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">González-Candelas</span><span class="given-names" tagx="given-names" title="given-names">Fernando</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">fernando.gonzalez@uv.es</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="citation_author_institution" id="Aff1">[1], </span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0000 9314 1427</span><span class="institution-id" title="institution-id">grid.413448.e</span><span class="institution" title="institution">CIBER of Epidemiology and Public Health, </span><span class="institution" title="institution">Instituto de Salud Carlos III, </span></div></span><span class="citation_author_institution" id="Aff3">[3], </span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.411308.f</span><span class="institution" title="institution">Hospital Clínico Universitario, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 2173 938X</span><span class="institution-id" title="institution-id">grid.5338.d</span><span class="institution" title="institution">Dpto. Microbiología, </span><span class="institution" title="institution">Universidad de Valencia, </span></div></span><span class="citation_author_institution" id="Aff6">[6], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 1770 977X</span><span class="institution-id" title="institution-id">grid.106023.6</span><span class="institution" title="institution">Consorcio Hospital General Universitario, </span></div></span><span class="citation_author_institution" id="Aff7">[7], </span><span class="citation_author_institution" id="Aff8">[8], </span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-9-9</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">11584</span><span class="history" title="history"><span class="received" title="received">received: 2017-4-12</span><span class="accepted" title="accepted">accepted: 2017-8-7</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">HIV infections are still a very serious concern for public heath worldwide. We have applied molecular evolution methods to study the HIV-1 epidemics in the Comunidad Valenciana (CV, Spain) from a public health surveillance perspective. For this, we analysed 1804 HIV-1 sequences comprising protease and reverse transcriptase (PR/RT) coding regions, sampled between 2004 and 2014. These sequences were subtyped and subjected to phylogenetic analyses in order to detect transmission clusters. In addition, univariate and multinomial comparisons were performed to detect epidemiological differences between HIV-1 subtypes, and risk groups. The HIV epidemic in the CV is dominated by subtype B infections among local men who have sex with men (MSM). 270 transmission clusters were identified (&amp;gt;57% of the dataset), 12 of which included ≥10 patients; 11 of subtype B (9 affecting MSMs) and one (n = 21) of CRF14, affecting predominately intravenous drug users (IDUs). Dated phylogenies revealed these large clusters to have originated from the mid-80s to the early 00 s. Subtype B is more likely to form transmission clusters than non-B variants and MSMs to cluster than other risk groups. Multinomial analyses revealed an association between non-B variants, which are not established in the local population yet, and different foreign groups.</p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2017" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Of the four phylogenetic groups which comprise HIV-1 (M, N, O, P), group M is the causal agent of the AIDS pandemic<sup><a href="#CR1">1</a>,<a href="#CR2">2</a></sup>. The latest UNAIDS/WHO report<sup><a href="#CR3">3</a></sup> estimated in almost 37 million the number of persons infected with HIV globally, with approximately 2.1 million new infections in 2015.</p> <p id="Par3">Although the rate of new HIV diagnosis has stabilized from the early 2000s in the European Union and European Economic Area (EU/EEA), transmissions among men who have sex with men (MSM) have experienced a sustained increase, thus following a different trend to other risk groups<sup><a href="#CR4">4</a></sup>. This is evident in Spain: during the late 90 s, most HIV new diagnoses were associated to intravenous drug use (IDU). However, in 2013, 51% of the infections occurred among MSM<sup><a href="#CR5">5</a>,<a href="#CR6">6</a></sup>. The increasing incidence among MSM is remarkably high in the age range 20–35 years<sup><a href="#CR6">6</a></sup>. HIV infections in Spain also affect the foreign population disproportionally: in 2012, 35% of the new diagnosis corresponded to immigrants or persons of foreign origin<sup><a href="#CR6">6</a></sup>.</p> <p id="Par4">There exist nine subtypes (denoted as A, B, C, D, F, G, H, J and K) and at least 61 circulating recombinant forms (CRFs) within HIV-1 group M<sup><a href="#CR7">7</a></sup>. There are differences among HIV-1 variants in several biological features. For instance, some subtypes and CRFs are associated to a faster progression to AIDS than others<sup><a href="#CR8">8</a>,<a href="#CR9">9</a></sup>. Genetic and antigenic differences among HIV-1 subtypes and CRFs are also a challenge for the development of an effective HIV-1 vaccine<sup><a href="#CR10">10</a></sup>.</p> <p id="Par5">Worldwide, the most prevalent HIV-1 subtype is C, accounting for around 50% of all cases. However, the HIV epidemic in Europe, particularly among MSM, is mainly driven by subtype B<sup><a href="#CR11">11</a></sup>, with frequent reported transmission clusters affecting this group<sup><a href="#CR12">12</a>–<a href="#CR20">20</a></sup>. However, there is evidence for an increased introduction of non-B subtypes<sup><a href="#CR11">11</a></sup>. For instance, in a sample of 206 patients from Spain, Abecasis and colleagues<sup><a href="#CR11">11</a></sup> found that CRF02_AG was the second most prevalent HIV-1 variant after subtype B (prevalence &amp;lt;2%).</p> <p id="Par6">The Comunidad Valenciana (CV) is the fourth largest region in Spain (~5 million inhabitants), representing &amp;gt;10% of the total population. Genotypic tests of resistance to antiviral drugs are performed routinely for the design of individualized antiretroviral treatments. Here, we have analysed more than 1,800 HIV-1 <i>pol</i> sequences obtained between 2004 and 2014 from different patients in the CV. We have used molecular phylogenetic analyses to complement HIV surveillance tasks of the Public Health Directorate of the CV Regional Government. Specifically, we have used this information to infer the distribution of HIV-1 subtypes, to analyse the introductions (and further local expansion) of this virus in the CV, to identify the emergence of new viral variants in this region, and also to analyse which risk groups are currently more vulnerable to HIV infection. The results obtained from this work may be useful in establishing and reinforcing preventive measures in specific target groups.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par7">Of the 1804 sequences analysed, 1512 (83.8%) belonged to subtype B. Among the 292 non-B sequences, the most prevalent HIV-1 variant was CRF02_AG (n = 66, overall prevalence = 3.7%), followed by subtypes A1, F1 (both n = 34, 1.9%), CRF14_BG (n = 28, 1.6%) and subtype G (n = 20, 1.1%). Other variants (n = 110) were present with a prevalence lower than 1.0%. Considering those patients for whom epidemiological information was available, 83.04% (930/1120) were male vs 16.96% (190/1120) female; 67.12% (637/949) were native from Spain (determined by place of birth) vs 32.87% (312/949) immigrants or of foreign origin; 66.84% (637/953) were MSM, 21.41% (204/953) heterosexual (HT) and 11.54% (110/953) IDUs. One patient was infected vertically and other one was haemophiliac. The mean age was 34.82 years (range 0 to 76) (Table <a href="#Tab1">1</a>). MSMs were more likely to be Spaniards than non-MSM (Fisher’s Exact test, FET: p-value = 5.5 × 10<sup>−5</sup>, odds-ratio, OR = 1.84 (1.35–2.48). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Distribution of HIV cases in the dataset (n = 1804) classified by viral subtype, gender, nationality, risk group, age and clustering status.</p>  <table frame="hsides" rules="groups"> <tbody> <tr> <td /> <td><b>B (n</b> = <b>1512)</b></td> <td><b>A1 (n</b> = <b>34)</b></td> <td><b>F1 (n</b> = <b>34)</b></td> <td><b>G (n</b> = <b>20)</b></td> <td><b>CRF02_AG (n</b> = <b>66)</b></td> <td><b>CRF14_BG (n</b> = <b>28)</b></td> <td><b>Others (n</b> = <b>110)*</b></td> <td><b>Total (n</b> = <b>1804)</b></td> </tr> <tr> <td colspan="9"><b>Gender</b></td> </tr> <tr> <td>Male</td> <td>818</td> <td>10</td> <td>13</td> <td>5</td> <td>24</td> <td>11</td> <td>49</td> <td>930</td> </tr> <tr> <td>Female</td> <td>115</td> <td>6</td> <td>6</td> <td>8</td> <td>25</td> <td>6</td> <td>24</td> <td>190</td> </tr> <tr> <td>UNK</td> <td>579</td> <td>18</td> <td>15</td> <td>7</td> <td>17</td> <td>11</td> <td>37</td> <td>684</td> </tr> <tr> <td colspan="9"><b>Geographical origin</b></td> </tr> <tr> <td>Spain</td> <td>579</td> <td>6</td> <td>5</td> <td>3</td> <td>11</td> <td>8</td> <td>25</td> <td>637</td> </tr> <tr> <td>W. Europe and N. America</td> <td>27</td> <td>0</td> <td>0</td> <td>0</td> <td>1</td> <td>0</td> <td>1</td> <td>29</td> </tr> <tr> <td>Eastern Europe</td> <td>17</td> <td>6</td> <td>5</td> <td>0</td> <td>0</td> <td>5</td> <td>6</td> <td>39</td> </tr> <tr> <td>Africa and M. East</td> <td>13</td> <td>3</td> <td>3</td> <td>10</td> <td>26</td> <td>0</td> <td>12</td> <td>67</td> </tr> <tr> <td>Latin America</td> <td>141</td> <td>1</td> <td>3</td> <td>0</td> <td>7</td> <td>3</td> <td>19</td> <td>174</td> </tr> <tr> <td>Others</td> <td>1</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>2</td> <td>3</td> </tr> <tr> <td>UNK</td> <td>734</td> <td>18</td> <td>18</td> <td>7</td> <td>21</td> <td>12</td> <td>45</td> <td>855</td> </tr> <tr> <td colspan="9"><b>Risk group</b></td> </tr> <tr> <td>HT</td> <td>124</td> <td>8</td> <td>6</td> <td>9</td> <td>29</td> <td>2</td> <td>26</td> <td>204</td> </tr> <tr> <td>MSM</td> <td>586</td> <td>3</td> <td>6</td> <td>0</td> <td>11</td> <td>1</td> <td>30</td> <td>637</td> </tr> <tr> <td>IDU</td> <td>85</td> <td>3</td> <td>0</td> <td>2</td> <td>3</td> <td>12</td> <td>5</td> <td>110</td> </tr> <tr> <td>Other</td> <td>2</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>2</td> </tr> <tr> <td>UNK</td> <td>715</td> <td>20</td> <td>22</td> <td>9</td> <td>23</td> <td>13</td> <td>49</td> <td>851</td> </tr> <tr> <td colspan="9"><b>Clustering</b></td> </tr> <tr> <td>No cluster</td> <td>622</td> <td>22</td> <td>24</td> <td>13</td> <td>32</td> <td>5</td> <td>49</td> <td>767</td> </tr> <tr> <td>Small cluster (2–3)</td> <td>371</td> <td>12</td> <td>5</td> <td>7</td> <td>25</td> <td>2</td> <td>39</td> <td>461</td> </tr> <tr> <td>Medium cluster (4–9)</td> <td>240</td> <td>0</td> <td>5</td> <td>0</td> <td>9</td> <td>0</td> <td>22</td> <td>276</td> </tr> <tr> <td>Large cluster (&amp;gt; = &amp;gt;=10)</td> <td>279</td> <td>0</td> <td>0</td> <td>0</td> <td>0</td> <td>21</td> <td>0</td> <td>300</td> </tr> <tr> <td><b>Age: mean (min</b>–<b>max)</b></td> <td>35.25 (14–76)</td> <td>32.8 (19–56)</td> <td>33.5 (18–62)</td> <td>28.9 (21–41)</td> <td>30.6 (19–49)</td> <td>38.11 (22–65)</td> <td>33.2 (0–62)</td> <td>34.82 (0–76)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>*The "Others�? subset includes 10 subtype C, 1 subtype D, 13 CRF19_cpx, 12 CRF12_BF, 9 CRF06_cpx, 7 CRF47_BF and 58 other (mostly unassigned) recombinant sequences.</p> </div>  <p /> <p id="Par8">Phylogenetic analyses revealed the existence of 270 transmission clusters, with sizes ranging from 2 to 111 patients (Fig. <a href="#Fig1">1A</a>). In total, 1039 patients from the dataset were included in a transmission cluster (57.5%), with 302 patients (16.7%) included in large clusters of 10 or more patients (Table <a href="#Tab1">1</a>; Fig. <a href="#Fig1">1A</a>). Among the 892 patients clustering in transmission groups of subtype B, 407 (45.6%) were MSM, 56 (6.3%) HTs, 38 (4.2%) IDUs and 391 (43.8%) of unknown transmission route. On the other hand, of the 147 patients clustering in non-B clusters 29 (19.7%) were HTs, 29 (19.7%) MSM, 15 (10.2%) IDUs and (50.3%) 74 of unknown transmission route (Fig. <a href="#Fig1">1B</a>). Among the transmission clusters that could be classified by the most frequent risk of transmission (≥50%) for the CV patients included in them, the most prevalent transmission clusters of subtype B were those classified as MSM (n = 99/219, 45.2%), followed by IDUs (n = 18/219, 8.2%) and HTs (n = 15/219, 6.8%). However, for non-B transmission clusters, ﻿thos﻿e of HTs were the most frequent ones (n = 16/51, 31.4%), followed by MSM (n = 14/51, 27.5%) and IDUs (n = 1/51, 2.0%). 87 of the 219 (39.7%) subtype B clusters and 20 of 51 (39.2%) non-B clusters could not be classified based on a majority risk of transmission (Fig. <a href="#Fig1">1C</a>). Subtype B sequences were more likely to be part of a transmission cluster than non-B sequences (FET: p-value = 0.007, OR = 1.42, 95% CI = 1.09–1.84). Also, MSM were more likely to be part of a transmission cluster than other risk groups (FET: p-value = 1.68 × 10<sup>−4</sup>, OR = 1.56, 95% CI = 1.23–1.99). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>(<b>A</b>) Distribution of sizes (log10 scale) of the 270 transmission clusters found in the CV (2004–2014) through phylogenetic analysis. Block letters indicate the 12 clusters that were analysed with BEAST. (<b>B</b>) Total number of patients for each risk group included in transmission clusters (n = 1039). (<b>C</b>) Number of transmission clusters depending on the risk group in which they were classified. *&amp;gt;1/4 patients shared a known risk group, but they were not enough to classify the cluster.</p>   </div> <p /> <p id="Par9">Twelve large transmission clusters, which included at least 10 patients from the CV (Fig. <a href="#Fig2">2</a>), were detected: 11 were from subtype B and included a total of 281 patients (Fig. <a href="#Fig2">2A</a>). One cluster of 21 patients corresponded to CRF14_BG (Cluster K, Fig. <a href="#Fig2">2B</a>). Patients infected with subtype B were more likely to form large transmission clusters than those infected with non-B variants (FET: p-value = 3.4 × 10<sup>−7</sup>, OR = 2.94, 95% CI = 1.84–4.92). Nine of the large clusters were classified as MSM; the other three included mostly IDUs, although in a proportion lower than 50% (Table <a href="#Tab2">2</a>). Dated phylogenies of these clusters (Supplementary Figs <a href="#MOESM1">1</a> to <a href="#MOESM1">12</a>) revealed that they originated between 1984 (cluster G, unclassified transmission route) and 2005 (cluster B, MSM). Only transmission clusters that included MSM as the main transmission risk were originated since 2000 (Table <a href="#Tab2">2</a>; Supplementary Figs <a href="#MOESM1">1</a> to <a href="#MOESM1">12</a>). Bagplots (bivariate representations of boxplots<sup><a href="#CR21">21</a></sup>) representing tree height (the time elapsed between tMRCA and the sampling date of the last sequence) and evolutionary rate estimates from the posterior distribution of each large cluster are shown in Fig. <a href="#Fig3">3</a>. A significant, negative correlation between median tree height and substitution rate estimates of these large clusters was obtained (R = −0.70, p-value = 0.013). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Maximum likelihood trees with the 12 largest transmission clusters highlighted in red. (<b>A</b>) Subtype B tree (clusters A-J, L). (<b>B</b>) Subtype G and CRF14_BG tree (cluster K).</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> <p>Size (n, number of patients), risk group, range of sampling dates and root-to-tip vs sampling date correlation coefficient for each transmission cluster (R), and estimates of their tMRCAs (median) and substitution rates as obtained with BEAST under the best-fitting demographic model (EXPO: exponential growth; BSP: Bayesian Skyline Plot; LOG: logarithmic growth).</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Transmission cluster</th> <th>n</th> <th>risk group</th> <th>range #</th> <th>R</th> <th>Model</th> <th>tMRCA (95% HPD)#</th> <th>Substitution rate (95% HPD)*</th> </tr> </thead> <tbody> <tr> <td>A</td> <td>10</td> <td>MSM</td> <td>2004–2011</td> <td>(−0.08)</td> <td>EXPO</td> <td>1984.7 (1971.2–1994.5)</td> <td>0.0014 (0.0009–0.0021)</td> </tr> <tr> <td>B</td> <td>13</td> <td>MSM</td> <td>2007–2013</td> <td>0.61</td> <td>BSP</td> <td>2005.9 (2003.8–2006.9)</td> <td>0.0043 (0.0016–0.0087)</td> </tr> <tr> <td>C</td> <td>15</td> <td>UNK/IDU</td> <td>2004–2014</td> <td>(−0.18)</td> <td>BSP</td> <td>1988.0 (1969.0–1999.2)</td> <td>0.0015 (0.0007–0.0025)</td> </tr> <tr> <td>D</td> <td>16</td> <td>MSM</td> <td>2011- 2013</td> <td>(−0.05)</td> <td>LOG</td> <td>1994.9 (1982.4–2006.3)</td> <td>0.0022 (0.0012–0.0034)</td> </tr> <tr> <td>E</td> <td>18</td> <td>UNK/MSM</td> <td>2010–2014</td> <td>0.25</td> <td>EXPO</td> <td>2002.2 (1995.9–2006.8)</td> <td>0.0018 (0.0011–0.0027)</td> </tr> <tr> <td>F</td> <td>18</td> <td>MSM</td> <td>2008–2013</td> <td>0.40</td> <td>LOG</td> <td>1999.3 (1989.6–2005.0)</td> <td>0.0013 (0.0005–0.0023)</td> </tr> <tr> <td>G</td> <td>19</td> <td>UNK</td> <td>2004–2013</td> <td>(−0.05)</td> <td>EXPO</td> <td>1984.3 (1966.9–1997.0)</td> <td>0.0011 (0.0006–0.0019)</td> </tr> <tr> <td>H</td> <td>19</td> <td>MSM</td> <td>2008–2014</td> <td>0.58</td> <td>BSP</td> <td>2002.1 (1995.9–2006.2)</td> <td>0.0019 (0.0010. 0.0029)</td> </tr> <tr> <td>I</td> <td>21</td> <td>MSM</td> <td>2004–2012</td> <td>0.58</td> <td>LOG</td> <td>1994.6 (1982.2–2001.3)</td> <td>0.0012 (0.0005–0.0020)</td> </tr> <tr> <td>J</td> <td>21</td> <td>MSM</td> <td>2004–2013</td> <td>0.76</td> <td>LOG</td> <td>1999.5 (1993.2–2003.4)</td> <td>0.0023 (0.0010–0.0045)</td> </tr> <tr> <td>K</td> <td>21</td> <td>UNK/IDU</td> <td>2004–2014</td> <td>0.62</td> <td>BSP</td> <td>1990.6 (1979.6–1998.1)</td> <td>0.0010 (0.0005–0.0016)</td> </tr> <tr> <td>L</td> <td>111</td> <td>MSM</td> <td>2006–2014</td> <td>0.61</td> <td>EXPO</td> <td>2001.8 (1998.6–2004.8)</td> <td>0.0028 (0.0022–0.0035)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>#</sup>Time measured in years. *Substitutions per site and year.</p> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Bagplots representing tree height and evolutionary rate estimates, obtained from the posterior distribution of the 12 largest transmission clusters detected (<b>A</b> to <b>L</b>).</p>   </div> <p /> <p id="Par10">Despite most patients infected with subtype B were Spanish natives (579 Spaniards vs 199 foreigners), a majority of those infected with non-B variants were foreigners (58 vs 113; FET: p-value &amp;lt; 2.2 × 10<sup>−16</sup>, OR = 5.66, 95% CI = 3.91–8.23). This difference was also observed when considering only patients belonging to a transmission cluster (subtype B: 378 vs 114; non-B: 31 vs 47; FET: p-value = 1.8 × 10<sup>−10</sup>; OR = 5.01, 95% CI = 2.96–8.57) (Supplementary Table <a href="#MOESM1">S1</a>).</p> <p id="Par11">A total of 88 subtype B, and 9 non-B transmission clusters did not include patients of foreign origin. These subtype B clusters included 178 Spanish patients and 122 of unknown origin, and the non-B clusters included 13 Spanish patients and 14 of unknown origin. A total of 49 subtype B, and 7 non-B transmission clusters were formed by both Spanish and non-Spanish patients. Altogether, these mixed subtype B clusters included 200 Spanish, 87 non-Spanish (mostly from Latin America) and 113 of unknown origin, and the mixed non-B clusters included 18 Spanish, 10 non-Spanish (mostly from Latin America or Eastern Europe) and 13 of unknown origin. All large transmission clusters but one (cluster G, for which there was not information available in 17 of its 19 patients) presented this characteristic. Finally, a total of 16 subtype B and 25 non-B transmission clusters included only patients of foreign origin. These subtype B clusters included 27 foreign patients (mostly from Latin America) and 16 of unknown origin, and the non-B clusters included 37 foreign patients (mostly from Africa/Middle East and Latin America) and 20 of unknown origin (Supplementary Table <a href="#MOESM1">S1</a>).</p> <p id="Par12">In addition to the patients’ origin, other variables considered for the multinomial analysis presented differences between subtypes. So, subtype B sequences were disproportionally more present in males compared to females (FET: p-value &amp;lt; 2.2 × 10<sup>−16</sup>, OR = 4.75, 95% CI = 3.29–6.86), in MSM with respect to HTs and IDUs (p-value &amp;lt; 2.2 × 10<sup>−16</sup>, OR = 7.4, 95% CI = 4.87–11.31 and p-value = 2 × 10<sup>−5</sup>, OR = 3.37, 95% CI = 1.90–5.89, respectively), and in IDUs compared to HTs (p-value = 0.0038, OR = 2.19, 95% CI = 1.26–3.88). There were also differences between subtypes with respect to age distribution (Kruskal-Wallis test: P = 0.0013, Chi<sup>2</sup> = 21.8, df = 6), with CRF02_AG patients being significantly younger than B patients (Games-Howell post-hoc test result for this comparison: p = 0.001, t = 4.37, df = 58). Given that no significant differences were found between subtypes in the distribution of collection dates (P &amp;gt; 0.70), this variable was excluded from the multinomial analysis.</p> <p id="Par13">Multinomial analyses were performed using a subset of 906 patients for whom there was information for all the variables considered: sex, risk group, country of origin, clustering status and age. We considered as baseline the most frequent category for each variable (subtype B, Spaniard, male, MSM, age between 21 and 29 years and not clustering), and a significant model was obtained (McFadden R<sup>2</sup> = 0.32, Chi<sup>2</sup> = 421.34, p-value &amp;lt; 2.2 × 10<sup>−16)</sup>). We checked that this subset was representative of the global set (n = 1804) by means of FETs. The p-values obtained for all the analysed variants were &amp;gt;0.05, including the distribution of patients from the different cluster-size categories (no cluster, small cluster, medium cluster and large cluster) that would result if only sequences included in the multinomial analysis had been taken into account for cluster detection, (FET: P &amp;gt; 0.10; Supplementary Table <a href="#MOESM1">S2</a>). This dataset was also genetically representative of the whole dataset (Supplementary Fig. <a href="#MOESM1">S13</a>). The multivariate model shows that the chances of being infected with all non-B groups increased in foreign patients, coming from Eastern Europe (A1, F1, 14_BG and the pooled, rare, variants), Africa and the Middle East (F1, G, 02_AG and the rare variants) and Latin America (rare variants), with all p-values &amp;lt; 0.01, all ORs &amp;gt;3.0. The likelihood of being infected with CRF14_BG also increased in patients older than 50 years (p-value = 0.029, OR = 36.2, 95% CI = 1.45–904.0), in IDUs (p-value = 9.44 × 10<sup>−5</sup>, OR = 257, 95% CI = 15.8–4160) and in those patients forming large transmission clusters (p-value = 1.6 × 10<sup>−4</sup>, OR = 36.9, 95% CI = 5.67–240) (Table <a href="#Tab3">3</a>). <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Results of the multinomial analysis (only significant associations between each HIV variant and the categories compared at each variable are shown).</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th colspan="9">Coefficients</th> </tr> <tr> <th /> <th>Estimate</th> <th>Std error</th> <th>t-value</th> <th>Pr(&amp;gt;|t|)</th> <th /> <th>OR</th> <th>2.50%</th> <th>97.50%</th> </tr> </thead> <tbody> <tr> <td>14_BG:Age(&amp;gt;50)</td> <td>3.59</td> <td>1.64</td> <td>2.188</td> <td>0.028696</td> <td>*</td> <td>36.2</td> <td>1.45</td> <td>904</td> </tr> <tr> <td>14_BG:RISK(HT)</td> <td>2.39</td> <td>1.43</td> <td>1.674</td> <td>0.094265</td> <td>$</td> <td>11.0</td> <td>0.664</td> <td>181</td> </tr> <tr> <td>02_AG:RISK(HT)</td> <td>1.07</td> <td>0.579</td> <td>1.850</td> <td>0.064773</td> <td>$</td> <td>2.91</td> <td>0.937</td> <td>9.07</td> </tr> <tr> <td>14_BG:RISK(UDI)</td> <td>5.55</td> <td>1.42</td> <td>3.905</td> <td>9.44E-05</td> <td>***</td> <td>257.0</td> <td>15.8</td> <td>4160</td> </tr> <tr> <td>14_BG:Eastern Europe</td> <td>3.29</td> <td>1.09</td> <td>3.023</td> <td>0.002502</td> <td>**</td> <td>26.9</td> <td>3.18</td> <td>227</td> </tr> <tr> <td>F:Eastern Europe</td> <td>3.51</td> <td>1.08</td> <td>3.250</td> <td>0.001155</td> <td>**</td> <td>33.5</td> <td>4.03</td> <td>279</td> </tr> <tr> <td>OTHERS: Eastern Europe</td> <td>2.01</td> <td>0.583</td> <td>3.448</td> <td>0.000566</td> <td>***</td> <td>7.47</td> <td>2.38</td> <td>23.5</td> </tr> <tr> <td>A1:Eastern Europe</td> <td>2.78</td> <td>0.795</td> <td>3.493</td> <td>0.000478</td> <td>***</td> <td>16.1</td> <td>3.38</td> <td>76.3</td> </tr> <tr> <td>F1:Africa and Middle East</td> <td>3.50</td> <td>1.05</td> <td>3.353</td> <td>0.000799</td> <td>***</td> <td>33.3</td> <td>4.29</td> <td>258</td> </tr> <tr> <td>G:Africa and Middle East</td> <td>3.78</td> <td>1.17</td> <td>3.236</td> <td>0.001218</td> <td>**</td> <td>43.7</td> <td>4.43</td> <td>431</td> </tr> <tr> <td>OTHERS: Africa and Middle East</td> <td>2.77</td> <td>0.554</td> <td>5.009</td> <td>5.48E-07</td> <td>***</td> <td>16.0</td> <td>5.41</td> <td>47.4</td> </tr> <tr> <td>A1:Africa and Middle East</td> <td>1.92</td> <td>0.993</td> <td>1.932</td> <td>0.053351</td> <td>$</td> <td>6.81</td> <td>0.973</td> <td>47.7</td> </tr> <tr> <td>02_AG:Africa and Middle East</td> <td>3.76</td> <td>0.580</td> <td>6.478</td> <td>9.28E-11</td> <td>***</td> <td>42.9</td> <td>13.8</td> <td>134</td> </tr> <tr> <td>14_BG:Latin America</td> <td>2.71</td> <td>1.38</td> <td>1.955</td> <td>0.050547</td> <td>$</td> <td>15.0</td> <td>0.994</td> <td>225</td> </tr> <tr> <td>OTHERS:Latin America</td> <td>1.19</td> <td>0.352</td> <td>3.391</td> <td>0.000696</td> <td>**</td> <td>3.30</td> <td>1.66</td> <td>6.58</td> </tr> <tr> <td>14_BG:ClusterLARGE</td> <td>3.61</td> <td>0.955</td> <td>3.776</td> <td>0.000159</td> <td>***</td> <td>36.9</td> <td>5.67</td> <td>240</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>***P &amp;lt; 0.001; **0001 &amp;lt; P &amp;lt; 0.01; *0.01 &amp;lt; P &amp;lt; 0.05; $0.05 &amp;lt; P &amp;lt; 0.</p> </div>  <p /> <p id="Par14">Pairwise genetic distances (GD) between individuals from a given transmission cluster ranged between 0.00 substitutions/site (s/s), in clusters of sizes of 2, 3, 5, 7, 8, 15, 16, 18 and 111 individuals, and 0.073 s/s, in a cluster of 6 individuals. For most pairwise comparisons, GD was lower than 0.045 s/s, regardless the size of the transmission cluster (Fig. <a href="#Fig4">4</a>; Supplementary Table <a href="#MOESM1">S3</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Boxplots representing mean pairwise genetic distances (s/s) between individuals within transmission clusters of different sizes, as estimated with the TN93 + GAMMA (4 CAT) model. Green and red bars represent the minimum and maximum pairwise distances, respectively, between a pair of individuals from a given cluster size.</p>   </div> <p /> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par15">We have studied the HIV epidemic in the Comunidad Valenciana by analysing, with molecular and evolutionary tools, 1804 sequences obtained between 2004 and 2014 from different patients. Our results indicate that the HIV epidemic in the CV is dominated by HIV-1 subtype B infections among local MSM. However, non-B infections represented an important number of cases, with a prevalence higher than 15%, being CRF02_AG the most prevalent non-B variant (prevalence = 3.7%), in agreement with previous estimates for Spain<sup><a href="#CR11">11</a>,<a href="#CR22">22</a></sup>.</p> <p id="Par16">Overall, the detection of transmission clusters demonstrates the importance of the domestic spread of HIV-1 in the CV. Most patients from the whole dataset (&amp;gt;57%) were included in local transmission clusters, of sizes ranging from 2 to 111 individuals. Local transmission was especially important among MSM, who were more likely to be included in a transmission cluster, as well as for Spanish natives, than other risk groups. Considering the 12 largest clusters (size ≥10 patients), this transmission risk was the most frequent in 9 of them. Excluding transmission cluster A (MSM, tMRCA = 1984.7), MSM clusters were of more recent origin as estimated using Bayesian coalescent analyses, especially clusters B (n = 13), E (n = 18), H (n = 19) and L (n = 111), which originated after year 2000. Previous analyses in the Spanish regions of Madrid<sup><a href="#CR23">23</a></sup> and the Basque Country<sup><a href="#CR24">24</a></sup> detected lower proportions of clustering patients (18 and 27% of their analysed sequences, respectively). These results suggest that the importance of local transmissions may not be the same in different Spanish regions. In addition, they also found evidences for an increased vulnerability to HIV of the Spanish MSM community in recent years. A more detailed analysis of cluster L is provided elsewhere<sup><a href="#CR25">25</a></sup>.</p> <p id="Par17">Immigrants were disproportionally represented in the dataset (almost 1/3 of the patients were of non-Spanish origin), reflecting their higher vulnerability to HIV infection. Multinomial analyses evidenced the significant association between all non-B groups analysed and different foreign populations: Eastern Europe (subtypes A1, F1, CRf14_BG and the rare variants), Africa and the Middle East (subtypes F1, G, CRF02_AG and rare variants) and Latin America (rare variants). These associations were in agreement with the geographical distributions of these variants<sup><a href="#CR26">26</a>–<a href="#CR28">28</a></sup>. These results, along with the fact that most non-B patients, either clustering or not, were of non-Spanish origin (CRF14_BG was the only exception) suggest that along the analysed time-span non-B HIV variants were not well established among Spanish locals in this region. This is exemplified by the fact that almost half of the non-B transmission clusters did not include any Spanish patient. Furthermore, non-B patients clustered in a significantly lower proportion than patients infected with subtype B, especially when considering clusters with at least 10 patients, thus displaying significantly lower local transmission efficiency. Previous molecular epidemiology studies of HIV-1 in Western Europe also reported non-B variants to be associated with the migrant population<sup><a href="#CR11">11</a></sup>, mainly affecting HTs. In the Spanish region of Madrid, where non-B variants have been found with a prevalence of up to 37%<sup><a href="#CR23">23</a></sup>, more than 70% of non-B sequences were found to belong to non-Spanish patients<sup><a href="#CR29">29</a></sup>. Other works in different Western European countries have reported differences in the prevalence of non-B variants in the migrant population. Similarly to our results, in countries such as Switzerland (estimated prevalence of non-B HIV sequences = 21%<sup><a href="#CR30">30</a></sup>) or the Belgian-Luxemburg population (prevalence &amp;gt;45%<sup><a href="#CR11">11</a>,<a href="#CR31">31</a></sup>) nearly 70% of non-B variants have been associated to patients of migrant origin<sup><a href="#CR30">30</a>,<a href="#CR31">31</a></sup>. These results contrast with those from Portugal, the Western European country where subtype B displays the lowest prevalence<sup><a href="#CR11">11</a></sup>. An study in the Portuguese region of Minho (close to the border with Spain) found non-B variants to have a prevalence of 73%, and only 10% of these sequences belonged to non-Portuguese patients<sup><a href="#CR32">32</a></sup>, thus evidencing that non-B infections are already established among locals.</p> <p id="Par18">Although 11 of the 12 largest transmission clusters were of subtype B, one of the largest clusters found (n = 21, median tMRCA = 1990.6) corresponded to the CRF14_BG, and included a high number of IDUs from Spain. CRF14 has been associated to a predominance of CXCR4 tropism, which usually leads to faster AIDS onset<sup><a href="#CR33">33</a>,<a href="#CR34">34</a></sup>. Previous phylogeographic analyses have suggested that CRF14_BG originated in the Iberian Peninsula<sup><a href="#CR33">33</a></sup>, and its prevalence is increasing in some Eastern European countries, such as Romania, boosted by migration between these countries and Spain<sup><a href="#CR35">35</a></sup>. Similarly to Romania, this CRF has been found to be common among IDUs in Greece, where a large transmission network of CRF14_BG of recent origin (tMRCA = 2009) has been reported to affect more than 70 people, and which origin has been linked to strains from South-western Europe<sup><a href="#CR36">36</a></sup>.</p> <p id="Par19">We also detected two other transmission clusters of smaller size (n = 5 and n = 4) affecting Spanish-native MSMs of another highly pathogenic variant (CRF19_cpx), which we previously reported as the first evidence of expansion of this variant outside Cuba<sup><a href="#CR37">37</a></sup>. Despite the prevalence of these CRFs remaining low in the CV, the effective expansion of these highly pathogenic HIV variants, evidenced by the detection of transmission clusters, is of especial interest because it may hamper the control of the local HIV epidemic, especially among vulnerable populations such as IDUs or MSMs.</p> <p id="Par20">The significant association between non-B variants and immigrants reported in this work and in previous molecular epidemiology analyses of HIV-1 in Madrid, Spain<sup><a href="#CR23">23</a>,<a href="#CR29">29</a></sup>, as well as the aforementioned examples of emergence of CRFs associated with migratory waves, suggest that high migration and tourism rates in some Spanish regions may contribute to the high genetic diversity of their HIV-1 epidemics. Thus, although our results suggest that non-B HIV groups are not well established in the local population, with only a few of them including patients of both Spanish and non-Spanish origin, this situation could change soon without intensified HIV prevention campaigns focused on vulnerable groups, including migrants, IDUs and MSM.</p> <p id="Par21">Although the large number of sequences for which no epidemiological information was available could be a potential limitation of our study, the results reported are representative of the sampled population and represent one of the most comprehensive analysis of the HIV pandemics in Spain<sup><a href="#CR22">22</a>,<a href="#CR23">23</a>,<a href="#CR29">29</a>,<a href="#CR38">38</a></sup>.</p> <p id="Par22">We have found a significant, negative correlation between tree height and evolutionary rate estimates obtained when comparing the 12 largest clusters. Several publications have addressed this time-dependency on the evolutionary rate (TDRP), that is, virus lineages estimated to have a recent origin yield higher estimated evolutionary rates than older viruses<sup><a href="#CR39">39</a>–<a href="#CR41">41</a></sup>. In our study the most plausible explanation for this phenomenon is the overestimation of evolutionary rates in the most recent clusters, caused by the presence of deleterious mutations over which purifying selection has not had time to act. This phenomenon might be potentiated by the bottlenecks that occur at viral transmission, which might originate a transient accumulation of deleterious mutations. Consequently, these results suggest that TDRP is an important factor to consider in molecular epidemiology, even when datasets are obtained from the same population and at short timescales (in this case 10 years, from 2004 to 2014).</p> <p id="Par23">It is also important to mention that many works on the molecular epidemiology of HIV-1 remove resistance-associated positions from the analysis, in order to prevent spurious clustering by convergent evolution. In our analyses, we did not remove those positions because their impact on the detection of transmission clusters has been demonstrated to be irrelevant<sup><a href="#CR42">42</a></sup>. Furthermore, a great majority of our sequences were known to derive from treatment-naïve patients (only 66 patients were known to have been treated).</p> <p id="Par24">Additionally, although the identification or detection of transmission clusters is commonly based only on phylogenetic cluster support, it is also often based on criteria that combine phylogenetic support, usually high bootstrap values, with genetic distance (GD) thresholds<sup><a href="#CR43">43</a></sup>. In this work, we did not consider any GD threshold value as criterion to define transmission clusters. This is because the rationale for genetic distance cut-offs is rarely provided, with very different thresholds being reported in the literature<sup><a href="#CR44">44</a></sup>, and larger GD values are expected for older transmission events. In this way, choosing distance-based thresholds can lead to underestimating transmission clusters, especially those that span long time periods<sup><a href="#CR43">43</a></sup>, as it is the case of our dataset. Nevertheless, a great majority of GDs estimated from the pairwise comparisons performed here were lower than 0.045 s/s. Thus, almost all the transmission clusters reported here would be characterized by both high phylogenetic support and low GD.</p> <p id="Par25">In conclusion, our results provide evidence that the HIV-1 epidemic in the CV is dominated by subtype B, especially among local MSMs. Although there was an important number of non-B cases, they occurred mostly among immigrants. This suggests that non-B infections are not well established in the local population. However, the detection of transmission clusters of non-B variants associated to a higher pathogenicity and affecting Spanish patients, urges to increase efforts on HIV testing and prevention campaigns to prevent their further expansion.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="dataset" title="sec"> <div class="title" tagx="title" title="title">
Dataset</div> <p id="Par26">A total of 1804 PR/RT sequences were obtained from newly HIV diagnosed people at six different hospitals and two HIV counselling and testing centres (CIPS) from the three provinces in the CV between 2004 and 2014. The sequences comprised the complete PR and the first 1005 nucleotides (335 amino acids) of the RT (1302 nt in total), and were obtained and subtyped as detailed in Patiño-Galindo <i>et al</i>.<sup><a href="#CR25">25</a></sup>.</p> </div> <div class="detectionoflocaltransmissionclusters" title="sec"> <div class="title" tagx="title" title="title">
Detection of local transmission clusters</div> <p id="Par27">Detection of transmission clusters, was performed following the same two-step procedure detailed in Patiño-Galindo <i>et al</i>.<sup><a href="#CR25">25</a></sup>. Briefly, in the first step, we obtained independent phylogenetic trees for each HIV-1 subtype and CRF detected, which included reference sequences retrieved from the Los Alamos HIV database spanning the analysed PR/RT region (n = 1787 subtype B, 2097 A1, 720 F1, 854 G + CRF14_BG and 2157 CRF02_AG, as well as 5720 sequences from all the HIV-1 subtypes/CRFs that were rare in the CV dataset). The initial trees were reconstructed with FastTree 2.1 in order to detect potential transmission clusters, defined as clades with SH-like support ≥0.70, and containing ≥90% sequences from the CV dataset<sup><a href="#CR17">17</a></sup>.</p> <p id="Par28">In the second step, these potential clusters were confirmed with ML phylogenies obtained with PhyML 3.0<sup><a href="#CR45">45</a></sup> in which the following additional reference sequences were included: (i) the 10 sequences with the highest similarity to each CV sequence from the potential clusters, as retrieved with a BLASTN search at the NCBI server (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>); (ii) 854 subtype B, 77 A1, 74 F1, 173 G + CRF14_BG, 101 CRF02_AG as well as 482 reference sequences from the other HIV-1 subtypes/CRFs. Only those potential clusters which contained more than 90% of sequences from the CV and grouped in the ML tree with aLRT support ≥0.98 were considered as confirmed. Clusters were then classified depending on the major risk of transmission (≥50%) for the corresponding patients from the CV.</p> </div> <div class="datedphylogenies" title="sec"> <div class="title" tagx="title" title="title">
Dated phylogenies</div> <p id="Par29">Dated phylogenies of large transmission clusters (those containing at least 10 patients from the CV<sup><a href="#CR46">46</a></sup>) were obtained with BEAST v1.8.1<sup><a href="#CR47">47</a></sup>, similarly as in Patiño-Galindo <i>et al</i>.<sup><a href="#CR25">25</a></sup>. In those clusters with low root-to-tip divergence vs sampling date correlation (R &amp;lt; 0.4), a log-normal prior (median = 0.0025 per site and year, s/s/y, 95% HPD upper limit = 0.0035 s/s/y)<sup><a href="#CR17">17</a></sup> was placed on the ucld.mean parameter.</p> </div> <div class="statisticalanalyses" title="sec"> <div class="title" tagx="title" title="title">
Statistical analyses</div> <p id="Par30">A multinomial logistic regression analysis was performed in order to identify the main predictors of HIV-1 subtype/CRF group distribution, considering relevant epidemiologic variables: country of origin, sex, risk group, collection date, age and clustering status. Due to the lack of epidemiological data for many sequences from the dataset, only 906 of the 1804 sequences were included in this analysis. Seven groups of different HIV-1 subtypes and CRFs were used: A1 (n = 14), B (n = 752), F1 (n = 13), G (n = 11), 02_AG (n = 42) and 14_BG (n = 15). All HIV-1 variants with fewer than 10 patients sampled were pooled as "Other variants�? (n = 59). Prior to the multinomial analysis, univariate analyses (Fisher’s Exact Tests) were performed for the aforementioned variables, in order to exclude from the multinomial analysis those with non-significant p-values. Only the variable "collection date�? (p-value &amp;gt; 0.70) was excluded. In the multinomial analysis, the most representative category of each variable was used as "baseline category�? (Subtype B, Spaniard, male, MSM, age between 21 and 29 years and not clustering; Supplementary Table <a href="#MOESM1">S4</a>). All the statistical analyses were performed using R<sup><a href="#CR48">48</a></sup>. The mlogit R package<sup><a href="#CR49">49</a></sup> was used for the multinomial analysis.</p> </div> <div class="estimationofpairwisegeneticdistances" title="sec"> <div class="title" tagx="title" title="title">
Estimation of pairwise genetic distances</div> <p id="Par31">Pairwise genetic distances (GD) between individuals within transmission clusters were estimated, as substitutions per site (s/s), with the TN93 + Γ(4CAT) model, using Perl and R scripts (available upon request) dependent on the ape R package<sup><a href="#CR50">50</a></sup>.</p> </div> <div class="sequenceinformation" title="sec"> <div class="title" tagx="title" title="title">
Sequence information</div> <p id="Par32">GenBank accession numbers for the CV sequences used in this study are HF567872-HF567912 and MF403205–MF404967.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2017_10286_MOESM1_ESM.pdf">LINK</a> <p>Supplementary Material</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Electronic supplementary material</b></p> <p><b>Supplementary information</b> accompanies this paper at doi:10.1038/s41598-017-10286-1 </p> </div> <div class="fn-type-" title=""> <p><b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This work was partly supported by project BFU2014-58656-R from Ministerio de Economía y Competitividad (Spanish Government) and project PROMETEO/2016/122 from Generalitat Valenciana to F.G.C., project CP07/00078 from Ministerio de Sanidad y Consumo (Spanish Government) to M.A.B., and Gilead Science S.L. J.A.P.G. was recipient of a fellowship (FPU-AP2010-0561) from Ministerio de Educación﻿,﻿ Cultura y Deporte, Spanish Government. The funders played no role on the design, analysis and conclusions from this work. We thank the CRIVIH for their support of this investigation.</p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>J.A.P.G., M.T.P., M.A.B. and F.G.C. designed the study. M.T.P. and M.A.B. performed the experiments. J.A.P.G. and F.G.C. analysed the data. M.T.P., M.A.B., I.A., A.J., D.N., M.J.G., D.O., E.O., C.G., J.B., V.D., R.M. contributed data. J.A.P.G. and F.G.C. wrote the paper. All authors reviewed the manuscript.</p> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing Interests</div> <p id="Par33">The authors declare that they have no competing interests.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hahn</span><span class="given-names" tagx="given-names" title="given-names">BH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Cock</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span></span><span class="mixed-article-title" title="mixed-article-title">AIDS as a zoonosis: scientific and public health implications</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">287</span><span class="fpage" tagx="fpage" title="fpage">607</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.287.5453.607">10.1126/science.287.5453.607</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10649986">10649986</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plantier</span><span class="given-names" tagx="given-names" title="given-names">J-C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A new human immunodeficiency virus derived from gorillas</span><span class="source" tagx="source" title="source">Nat. Med.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">871</span><span class="lpage" tagx="lpage" title="lpage">2</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nm.2016">10.1038/nm.2016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19648927">19648927</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">UNAIDS. Fact sheet – Latest global and regional statistics on the status of the AIDS epidemic. (2016).</span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">ECDC. Thematic report: Men who have sex with men. Monitoring implementation of the Dublin Declaration on Partnership to fight HIV/AIDS in Europe and Central Asia: 2012 progress report. (2013).</span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">UNAIDS (Joint United Nations Programme on HIV/AIDS). Report on the global HIV/AIDS epidemic. 1–226 (2002).</span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">DGSP. Vigilancia epidemiológica del VIH/SIDA enEspaña: Sistema de información sobre nuevos diagnósticos del VIH y registro nacional de casos de SIDA. (2014).</span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kuiken, C. L. <i>et al</i>. <i>HIV Sequence Compendium 2012</i> (2012).</span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaleebu</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Effect of human immunodeficiency virus (HIV) type 1 envelope subtypes A and D on disease progression in a large cohort of HIV-1-positive persons in Uganda</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">185</span><span class="fpage" tagx="fpage" title="fpage">1244</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/340130">10.1086/340130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12001041">12001041</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouri</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">CRF19_cpx is an Evolutionary fit HIV-1 Variant Strongly Associated With Rapid Progression to AIDS in Cuba</span><span class="source" tagx="source" title="source">EBioMedicine</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">244</span><span class="lpage" tagx="lpage" title="lpage">54</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ebiom.2015.01.015">10.1016/j.ebiom.2015.01.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26137563">26137563</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nickle</span><span class="given-names" tagx="given-names" title="given-names">DC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Coping with viral diversity in HIV vaccine design</span><span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e75</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.0030075">10.1371/journal.pcbi.0030075</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17465674">17465674</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abecasis</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">HIV-1 subtype distribution and its demographic determinants in newly diagnosed patients in Europe suggest highly compartmentalized epidemics</span><span class="source" tagx="source" title="source">Retrovirology</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1742-4690-10-7">10.1186/1742-4690-10-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23317093">23317093</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kouyos</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology reveals long-term changes in HIV type 1 subtype B transmission in Switzerland</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">201</span><span class="fpage" tagx="fpage" title="fpage">1488</span><span class="lpage" tagx="lpage" title="lpage">97</span><span class="pub-id"><a href="https://dx.doi.org/10.1086/651951">10.1086/651951</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20384495">20384495</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pozniak</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leigh Brown</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Episodic sexual transmission of HIV revealed by molecular phylodynamics</span><span class="source" tagx="source" title="source">PLoS Med.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e50</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pmed.0050050">10.1371/journal.pmed.0050050</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18351795">18351795</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leigh Brown</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Transmission network parameters estimated from HIV sequences for a nationwide epidemic</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">204</span><span class="fpage" tagx="fpage" title="fpage">1463</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jir550">10.1093/infdis/jir550</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21921202">21921202</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cuevas</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">HIV-1 transmission cluster with T215D revertant mutation among newly diagnosed patients from the Basque Country, Spain</span><span class="source" tagx="source" title="source">J. Acquir. Immune Defic. Syndr.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">99</span><span class="lpage" tagx="lpage" title="lpage">103</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/QAI.0b013e318199063e">10.1097/QAI.0b013e318199063e</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19282784">19282784</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zehender</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Population dynamics of HIV-1 subtype B in a cohort of men-having-sex-with-men in Rome, Italy</span><span class="source" tagx="source" title="source">J. Acquir. Immune Defic. Syndr.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">156</span><span class="lpage" tagx="lpage" title="lpage">60</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/QAI.0b013e3181eb3002">10.1097/QAI.0b013e3181eb3002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20703157">20703157</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hué</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pillay</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clewley</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic analysis reveals the complex structure of HIV-1 transmission within defined risk groups</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">102</span><span class="fpage" tagx="fpage" title="fpage">4425</span><span class="lpage" tagx="lpage" title="lpage">4429</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0407534102">10.1073/pnas.0407534102</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15767575">15767575</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bezemer, D. <i>et al</i>. Dispersion of the HIV-1 Epidemic in Men Who Have Sex with Men in the Netherlands: A Combined Mathematical Model and Phylogenetic Analysis. <i>PLoS Med</i>. <b>12</b>, e1001898; discussion e1001898 (2015).</span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magiorkinis</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The global spread of HIV-1 subtype B epidemic</span><span class="source" tagx="source" title="source">Infect. Genet. Evol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">179</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2016.05.041">10.1016/j.meegid.2016.05.041</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27262355">27262355</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beloukas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology of HIV-1 infection in Europe: An overview</span><span class="source" tagx="source" title="source">Infect. Genet. Evol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">180</span><span class="lpage" tagx="lpage" title="lpage">189</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2016.06.033">10.1016/j.meegid.2016.06.033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27321440">27321440</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rousseeuw</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruts</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tukey</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Bagplot: A Bivariate Boxplot</span><span class="source" tagx="source" title="source">Am. Stat.</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">382</span><span class="lpage" tagx="lpage" title="lpage">387</span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yebra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Most HIV type 1 non-B infections in the Spanish cohort of antiretroviral treatment-naïve HIV-infected patients (CoRIS) are due to recombinant viruses</span><span class="source" tagx="source" title="source">J. Clin. Microbiol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">407</span><span class="lpage" tagx="lpage" title="lpage">13</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.05798-11">10.1128/JCM.05798-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22162552">22162552</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yebra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holguín</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pillay</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hué</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic and demographic characterization of HIV-1 transmission in Madrid, Spain</span><span class="source" tagx="source" title="source">Infect. Genet. Evol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">232</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2012.12.006">10.1016/j.meegid.2012.12.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23291408">23291408</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patiño-Galindo</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Transmission dynamics of HIV-1 subtype B in the Basque Country, Spain</span><span class="source" tagx="source" title="source">Infect. Genet. Evol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2016.02.028">10.1016/j.meegid.2016.02.028</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26921800">26921800</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patiño-Galindo</span><span class="given-names" tagx="given-names" title="given-names">J�?</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a large, fast-expanding HIV-1 subtype B transmission cluster among MSM in Valencia, Spain</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">e0171062</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0171062">10.1371/journal.pone.0171062</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28152089">28152089</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hemelaar</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">The origin and diversity of the HIV-1 pandemic</span><span class="source" tagx="source" title="source">Trends Mol. Med.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">182</span><span class="lpage" tagx="lpage" title="lpage">92</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molmed.2011.12.001">10.1016/j.molmed.2011.12.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22240486">22240486</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buonaguro</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tornesello</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buonaguro</span><span class="given-names" tagx="given-names" title="given-names">FM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Human immunodeficiency virus type 1 subtype distribution in the worldwide epidemic: pathogenetic and therapeutic implications</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">10209</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00872-07">10.1128/JVI.00872-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17634242">17634242</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bello</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afonso</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgado</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylodynamics of HIV-1 subtype F1 in Angola, Brazil and Romania</span><span class="source" tagx="source" title="source">Infect. Genet. Evol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">1079</span><span class="lpage" tagx="lpage" title="lpage">86</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2012.03.014">10.1016/j.meegid.2012.03.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22484759">22484759</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">González-Alba</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular surveillance of HIV-1 in Madrid, Spain: a phylogeographic analysis</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">10755</span><span class="lpage" tagx="lpage" title="lpage">63</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00454-11">10.1128/JVI.00454-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21795343">21795343</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Wyl</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The role of migration and domestic transmission in the spread of HIV-1 non-B subtypes in Switzerland</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">204</span><span class="fpage" tagx="fpage" title="fpage">1095</span><span class="lpage" tagx="lpage" title="lpage">103</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jir491">10.1093/infdis/jir491</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21881125">21881125</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dauwe</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characteristics and spread to the native population of HIV-1 non-B subtypes in two European countries with high migration rate</span><span class="source" tagx="source" title="source">BMC Infect. Dis.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">524</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12879-015-1217-0">10.1186/s12879-015-1217-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26572861">26572861</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvalho</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of a local HIV-1 epidemic in portugal highlights established transmission of non-B and non-G subtypes</span><span class="source" tagx="source" title="source">J. Clin. Microbiol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">1506</span><span class="lpage" tagx="lpage" title="lpage">14</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.03611-14">10.1128/JCM.03611-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25694526">25694526</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bártolo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Origin and Epidemiological History of HIV-1 CRF14_BG</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e24130</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0024130">10.1371/journal.pone.0024130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21969855">21969855</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pérez-�?lvarez</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Predominance of CXCR4 tropism in HIV-1 CRF14_BG strains from newly diagnosed infections</span><span class="source" tagx="source" title="source">J. Antimicrob. Chemother.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">246</span><span class="lpage" tagx="lpage" title="lpage">53</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/jac/dkt305">10.1093/jac/dkt305</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23900735">23900735</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niculescu</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Recent HIV-1 Outbreak Among Intravenous Drug Users in Romania: Evidence for Cocirculation of CRF14_BG and Subtype F1 Strains</span><span class="source" tagx="source" title="source">AIDS Res. Hum. Retroviruses</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">488</span><span class="lpage" tagx="lpage" title="lpage">95</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/aid.2014.0189">10.1089/aid.2014.0189</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25369079">25369079</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraskevis</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Economic recession and emergence of an HIV-1 outbreak among drug injectors in Athens metropolitan area: a longitudinal study</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e78941</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0078941">10.1371/journal.pone.0078941</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24265730">24265730</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patiño Galindo</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Expansion of the CRF19_cpx Variant in Spain</span><span class="source" tagx="source" title="source">J. Clin. Virol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">146</span><span class="lpage" tagx="lpage" title="lpage">149</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jcv.2015.06.094">10.1016/j.jcv.2015.06.094</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26209397">26209397</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yebra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Different trends of transmitted HIV-1 drug resistance in Madrid, Spain, among risk groups in the last decade</span><span class="source" tagx="source" title="source">Arch. Virol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">159</span><span class="fpage" tagx="fpage" title="fpage">1079</span><span class="lpage" tagx="lpage" title="lpage">87</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00705-013-1933-y">10.1007/s00705-013-1933-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24297490">24297490</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">SYW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Skyline-plot methods for estimating demographic history from nucleotide sequences</span><span class="source" tagx="source" title="source">Mol. Ecol. Resour.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">423</span><span class="lpage" tagx="lpage" title="lpage">434</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1755-0998.2011.02988.x">10.1111/j.1755-0998.2011.02988.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21481200">21481200</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aiewsakun</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzourakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Time dependency of foamy virus evolutionary rate estimates</span><span class="source" tagx="source" title="source">BMC Evol. Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12862-015-0408-z">10.1186/s12862-015-0408-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26111824">26111824</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aiewsakun</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzourakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Time-Dependent Rate Phenomenon in Viruses</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">7184</span><span class="lpage" tagx="lpage" title="lpage">95</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00593-16">10.1128/JVI.00593-16</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27252529">27252529</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hué</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clewley</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cane</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pillay</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">HIV-1 pol gene variation is sufficient for reconstruction of transmissions in the era of antiretroviral therapy</span><span class="source" tagx="source" title="source">AIDS</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">719</span><span class="lpage" tagx="lpage" title="lpage">28</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/00002030-200403260-00002">10.1097/00002030-200403260-00002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15075506">15075506</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hassan</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanders</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Albert</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Esbjörnsson</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Defining HIV-1 transmission clusters based on sequence data</span><span class="source" tagx="source" title="source">AIDS</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">1211</span><span class="lpage" tagx="lpage" title="lpage">1222</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/QAD.0000000000001470">10.1097/QAD.0000000000001470</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28353537">28353537</a></span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grabowski</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Redd</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular tools for studying HIV transmission in sexual networks</span><span class="source" tagx="source" title="source">Curr. Opin. HIV AIDS</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">126</span><span class="lpage" tagx="lpage" title="lpage">33</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/COH.0000000000000040">10.1097/COH.0000000000000040</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24384502">24384502</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guindon</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</span><span class="source" tagx="source" title="source">Syst. Biol.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">59</span><span class="fpage" tagx="fpage" title="fpage">307</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/sysbio/syq010">10.1093/sysbio/syq010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638">20525638</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunar</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Bridging epidemiology with population genetics in a low incidence MSM-driven HIV-1 subtype B epidemic in Central Europe</span><span class="source" tagx="source" title="source">BMC Infect. Dis.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">65</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12879-015-0802-6">10.1186/s12879-015-0802-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25887543">25887543</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span><span class="source" tagx="source" title="source">BMC Evol. Biol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">214</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-7-214">10.1186/1471-2148-7-214</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">R Core Team. <i>R: A language and environment for statistical computing</i>. (R Foundation for Statistical computing, 2014).</span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Croissant, Y. <i>Package mlogit</i>. (2015).</span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Paradis, E. <i>et al</i>. Package ‘ape’ (2017).</span></li> </ul> </div> </div>  </body></html>