<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/04/11/299776.atom" />
<meta name="HW.pisa" content="biorxiv;299776v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A planarian nidovirus expands the limits of RNA genome size" />
<meta name="DC.Identifier" content="10.1101/299776" />
<meta name="DC.Date" content="2018-04-11" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication (albeit within a permissive environment). A 33.5-kb nidovirus has been considered close to the upper size limit for such entities; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the transition from primordial RNP to contemporary DNA-RNP-based life. Whether or not RNA viruses represent transitional steps on the road to DNA-based life, studies of larger RNA viruses advance our understanding of size constraints on RNP entities. For example, emergence of the largest previously known RNA genomes (20-34 kb in positive-stranded nidoviruses, including coronaviruses) is associated with a proofreading exoribonuclease encoded in the nidoviral open reading frame 1b (ORF1b). However, apparent constraints on the size of ORF1b, which encodes this and other key replicative enzymes, have been hypothesized to limit further expansion of viral RNA genomes. Here, we characterize a novel nidovirus (planarian secretory cell nidovirus; PSCNV) whose disproportionately large ORF1b-like region, and overall 41.1 kb genome, substantially extend the presumed limits on RNA genome size. This genome encodes a predicted 13,556-aa polyprotein in an unconventional single ORF, yet retains canonical nidoviral genome organization and expression, and key replicative domains. Our evolutionary analysis suggests that PSCNV diverged early from multi-ORF nidoviruses, and subsequently acquired additional genes, including those typical of large DNA viruses or hosts. PSCNV’s greatly expanded genome, proteomic complexity, and unique features – impressive in themselves – attest to the likelihood of still-larger RNA genomes awaiting discovery.

Significance Statement RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication. The upper genome size for such entities was assumed to be &lt;35 kb; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the proposed evolution of contemporary DNA-RNP-based life from primordial RNP entities. Here, we describe a nidovirus from planarians, whose 41.1 kb genome is 23% larger than the largest known of RNA virus. The planarian secretory cell nidovirus has broken apparent constraints on the size of the genomic subregion that encodes core replication machinery, and has acquired genes not previously observed in RNA viruses. This virus challenges and advances our understanding of the limits to RNA genome size." />
<meta name="DC.Contributor" content="Amir Saberi" />
<meta name="DC.Contributor" content="Anastasia A. Gulyaeva" />
<meta name="DC.Contributor" content="John L. Brubacher" />
<meta name="DC.Contributor" content="Phillip A. Newmark" />
<meta name="DC.Contributor" content="Alexander E. Gorbalenya" />
<meta name="article:published_time" content="2018-04-11" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A planarian nidovirus expands the limits of RNA genome size" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication (albeit within a permissive environment). A 33.5-kb nidovirus has been considered close to the upper size limit for such entities; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the transition from primordial RNP to contemporary DNA-RNP-based life. Whether or not RNA viruses represent transitional steps on the road to DNA-based life, studies of larger RNA viruses advance our understanding of size constraints on RNP entities. For example, emergence of the largest previously known RNA genomes (20-34 kb in positive-stranded nidoviruses, including coronaviruses) is associated with a proofreading exoribonuclease encoded in the nidoviral open reading frame 1b (ORF1b). However, apparent constraints on the size of ORF1b, which encodes this and other key replicative enzymes, have been hypothesized to limit further expansion of viral RNA genomes. Here, we characterize a novel nidovirus (planarian secretory cell nidovirus; PSCNV) whose disproportionately large ORF1b-like region, and overall 41.1 kb genome, substantially extend the presumed limits on RNA genome size. This genome encodes a predicted 13,556-aa polyprotein in an unconventional single ORF, yet retains canonical nidoviral genome organization and expression, and key replicative domains. Our evolutionary analysis suggests that PSCNV diverged early from multi-ORF nidoviruses, and subsequently acquired additional genes, including those typical of large DNA viruses or hosts. PSCNV’s greatly expanded genome, proteomic complexity, and unique features – impressive in themselves – attest to the likelihood of still-larger RNA genomes awaiting discovery.&lt;/p&gt;&lt;h3&gt;Significance Statement&lt;/h3&gt;
&lt;p&gt;RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication. The upper genome size for such entities was assumed to be &amp;lt;35 kb; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the proposed evolution of contemporary DNA-RNP-based life from primordial RNP entities. Here, we describe a nidovirus from planarians, whose 41.1 kb genome is 23% larger than the largest known of RNA virus. The planarian secretory cell nidovirus has broken apparent constraints on the size of the genomic subregion that encodes core replication machinery, and has acquired genes not previously observed in RNA viruses. This virus challenges and advances our understanding of the limits to RNA genome size.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;299776v1" />
<meta name="citation_id" content="299776v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/299776v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/299776v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/299776v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776.full.pdf" />
<meta name="citation_doi" content="10.1101/299776" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="299776" />
<meta name="citation_author" content="Amir Saberi" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign" />
<meta name="citation_author" content="Anastasia A. Gulyaeva" />
<meta name="citation_author_institution" content="Department of Medical Microbiology, Leiden University Medical Center" />
<meta name="citation_author" content="John L. Brubacher" />
<meta name="citation_author_institution" content="Department of Biology, Canadian Mennonite University" />
<meta name="citation_author" content="Phillip A. Newmark" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign" />
<meta name="citation_author_email" content="pnewmark@morgridge.org" />
<meta name="citation_author_email" content="A.E.Gorbalenya@lumc.nl" />
<meta name="citation_author" content="Alexander E. Gorbalenya" />
<meta name="citation_author_institution" content="Department of Medical Microbiology, Leiden University Medical Center" />
<meta name="citation_author_institution" content="Faculty of Bioengineering and Bioinformatics, Lomonosov Moscow State University" />
<meta name="citation_author_email" content="pnewmark@morgridge.org" />
<meta name="citation_author_email" content="A.E.Gorbalenya@lumc.nl" />
<meta name="citation_reference" content="Joyce GF (2002) The antiquity of RNA-based evolution. Nature 418(6894):214–221." />
<meta name="citation_reference" content="Leipe DD, Aravind L, &amp; Koonin EV (1999) Did DNA replication evolve twice independently? Nucleic Acids Res 27(17):3389–3401." />
<meta name="citation_reference" content="Poole AM &amp; Logan DT (2005) Modern mRNA proofreading and repair: clues that the last universal common ancestor possessed an RNA genome? Mol Biol Evol 22(6):1444–1455." />
<meta name="citation_reference" content="Xavier JC, Patil KR, &amp; Rocha I (2014) Systems biology perspectives on minimal and simpler cells. Microbiol Mol Biol Rev 78(3):487–509." />
<meta name="citation_reference" content="Li S, Guo W, Dewey CN, &amp; Greaser ML (2013) Rbm20 regulates titin alternative splicing as a splicing repressor. Nucleic Acids Res 41(4):2659–2672." />
<meta name="citation_reference" content="Gorbalenya AE (1992) Host-related sequences in RNA viral genomes. Seminars in Virology 3:359–371." />
<meta name="citation_reference" content="Koonin EV &amp; Dolja VV (2013) A virocentric perspective on the evolution of life. Curr Opin Virol 3(5):546–557." />
<meta name="citation_reference" content="Forterre P (2006) Three RNA cells for ribosomal lineages and three DNA viruses to replicate their genomes: a hypothesis for the origin of cellular domain. Proc Natl Acad Sci U S A 103(10):3669–3674." />
<meta name="citation_reference" content="Steinhauer DA, Domingo E, &amp; Holland JJ (1992) Lack of evidence for proofreading mechanisms associated with an RNA virus polymerase. Gene 122(2):281–288." />
<meta name="citation_reference" content="den Boon JA, et al. (1991) Equine arteritis virus is not a togavirus but belongs to the coronaviruslike superfamily. J Virol 65(6):2910–2920." />
<meta name="citation_reference" content="Stenglein MD, et al. (2014) Ball python nidovirus: a candidate etiologic agent for severe respiratory disease in Python regius. MBio 5(5):e01484–01414." />
<meta name="citation_reference" content="Gorbalenya AE, Enjuanes L, Ziebuhr J, &amp; Snijder EJ (2006) Nidovirales: evolving the largest RNA virus genome. Virus Res 117(1):17–37." />
<meta name="citation_reference" content="Minskaia E, et al. (2006) Discovery of an RNA virus 3’-&gt;5’ exoribonuclease that is critically involved in coronavirus RNA synthesis. Proc Natl Acad Sci U S A 103(13):5108–5113." />
<meta name="citation_reference" content="Denison MR, Graham RL, Donaldson EF, Eckerle LD, &amp; Baric RS (2011) Coronaviruses: an RNA proofreading machine regulates replication fidelity and diversity. RNA Biol 8(2):270–279." />
<meta name="citation_reference" content="Nga PT, et al. (2011) Discovery of the first insect nidovirus, a missing evolutionary link in the emergence of the largest RNA virus genomes. PLoS. Pathog 7(9):e1002215." />
<meta name="citation_reference" content="Ma Y, et al. (2015) Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex. Proc. Natl. Acad. Sci. U. S. A 112(30):9436–9441." />
<meta name="citation_reference" content="Ferron F, et al. (2018) Structural and molecular basis of mismatch correction and ribavirin excision from coronavirus RNA. Proc Natl Acad Sci U S A 115(2):E162–E171." />
<meta name="citation_reference" content="Shi M, et al. (2016) Redefining the invertebrate RNA virosphere. Nature 540:539–543." />
<meta name="citation_reference" content="Shi M, et al. (2018) The evolutionary history of vertebrate RNA viruses. Nature, in press." />
<meta name="citation_reference" content="Raoult D, et al. (2004) The 1.2-megabase genome sequence of Mimivirus. Science 306(5700):1344–1350." />
<meta name="citation_reference" content="Philippe N, et al. (2013) Pandoraviruses: amoeba viruses with genomes up to 2.5 Mb reaching that of parasitic eukaryotes. Science 341(6143):281–286." />
<meta name="citation_reference" content="Schulz F, et al. (2017) Giant viruses with an expanded complement of translation system components. Science 356(6333):82–85." />
<meta name="citation_reference" content="Lauber C, et al. (2013) The footprint of genome architecture in the largest genome expansion in RNA viruses. PLoS. Pathog 9(7):e1003500." />
<meta name="citation_reference" content="Saberi A, Jamal A, Beets I, Schoofs L, &amp; Newmark PA (2016) GPCRs Direct Germline Development and Somatic Gonad Function in Planarians. PLoS Biol 14(5):e1002457." />
<meta name="citation_reference" content="Newmark PA &amp; Sanchez Alvarado A (2002) Not your father’s planarian: a classic model enters the era of functional genomics. Nat Rev Genet 3(3):210–219." />
<meta name="citation_reference" content="Zayas RM, et al. (2005) The planarian Schmidtea mediterranea as a model for epigenetic germ cell specification: analysis of ESTs from the hermaphroditic strain. Proc Natl Acad Sci U S A 102(51):18491–18496." />
<meta name="citation_reference" content="Grohme MA, et al. (2018) The genome of Schmidtea mediterranea and the evolution of core cellular mechanisms. Nature 554(7690):56–61." />
<meta name="citation_reference" content="Brandl H, et al. (2016) PlanMine--a mineable resource of planarian biology and biodiversity. Nucleic Acids Res 44(D1):D764–773." />
<meta name="citation_reference" content="Lehmann KC, et al. (2015) Discovery of an essential nucleotidylating activity associated with a newly delineated conserved domain in the RNA polymerase-containing protein of all nidoviruses. Nucleic Acids Res 43(17):8416–8434." />
<meta name="citation_reference" content="Deng Z, et al. (2014) Structural basis for the regulatory function of a complex zinc-binding domain in a replicative arterivirus helicase resembling a nonsense-mediated mRNA decay helicase. Nucleic Acids Res 42(5):3464–3477." />
<meta name="citation_reference" content="Ziebuhr J, Snijder EJ, &amp; Gorbalenya AE (2000) Virus-encoded proteinases and proteolytic processing in the Nidovirales. J. Gen. Virol 81(Pt 4):853–879." />
<meta name="citation_reference" content="Ziebuhr J, Bayer S, Cowley JA, &amp; Gorbalenya AE (2003) The 3C-like proteinase of an invertebrate nidovirus links coronavirus and potyvirus homologs. J. Virol 77(2):1415–1426." />
<meta name="citation_reference" content="Smits SL, Snijder EJ, &amp; de Groot RJ (2006) Characterization of a torovirus main proteinase. J. Virol 80(8):4157–4167." />
<meta name="citation_reference" content="Ulferts R, Mettenleiter TC, &amp; Ziebuhr J (2011) Characterization of Bafinivirus main protease autoprocessing activities. J. Virol 85(3):1348–1359." />
<meta name="citation_reference" content="Blanck S, et al. (2014) Characterization of an alphamesonivirus 3C-like protease defines a special group of nidovirus main proteases. J. Virol 88(23):13747–13758." />
<meta name="citation_reference" content="Perlman S &amp; Netland J (2009) Coronaviruses post-SARS: update on replication and pathogenesis. Nat Rev Microbiol 7(6):439–450." />
<meta name="citation_reference" content="Rauceo JM, et al. (2006) Threonine-rich repeats increase fibronectin binding in the Candida albicans adhesin Als5p. Eukaryot Cell 5(10):1664–1673." />
<meta name="citation_reference" content="Hevia A, et al. (2013) An extracellular Serine/Threonine-rich protein from Lactobacillus plantarum NCIMB 8826 is a novel aggregation-promoting factor with affinity to mucin. Appl Environ Microbiol 79(19):6059–6066." />
<meta name="citation_reference" content="Camus-Bouclainville C, et al. (2004) A virulence factor of myxoma virus colocalizes with NF-kappaB in the nucleus and interferes with inflammation. J Virol 78(5):2510–2516." />
<meta name="citation_reference" content="Krey T, et al. (2012) Crystal structure of the pestivirus envelope glycoprotein E(rns) and mechanistic analysis of its ribonuclease activity. Structure 20(5):862–873." />
<meta name="citation_reference" content="Park B KY (2012) Immunosuppression induced by expression of a viral RNase enhances susceptibility of Plutella xylostella to microbial pesticides. Insect Science 19(1):47–54." />
<meta name="citation_reference" content="Zayas RM, Cebria F, Guo T, Feng J, &amp; Newmark PA (2010) The use of lectins as markers for differentiated secretory cells in planarians. Dev Dyn 239(11):2888–2897." />
<meta name="citation_reference" content="Ortego J, Ceriani JE, Patino C, Plana J, &amp; Enjuanes L (2007) Absence of E protein arrests transmissible gastroenteritis coronavirus maturation in the secretory pathway. Virology 368(2):296–308." />
<meta name="citation_reference" content="Thuy NT, et al. (2013) A new nidovirus (NamDinh virus NDiV): Its ultrastructural characterization in the C6/36 mosquito cell line. Virology 444(1-2):337–342." />
<meta name="citation_reference" content="Knoops K, et al. (2008) SARS-coronavirus replication is supported by a reticulovesicular network of modified endoplasmic reticulum. PLoS Biol 6(9):e226." />
<meta name="citation_reference" content="Firth AE &amp; Brierley I (2012) Non-canonical translation in RNA viruses. J Gen Virol 93(Pt 7):1385–1409." />
<meta name="citation_reference" content="Napthine S, et al. (2017) Protein-directed ribosomal frameshifting temporally regulates gene expression. Nat Commun 8:15582." />
<meta name="citation_reference" content="Thomas G (2002) Furin at the cutting edge: from protein traffic to embryogenesis and disease. Nat Rev Mol Cell Biol 3(10):753–766." />
<meta name="citation_reference" content="Pasternak AO, Spaan WJ, &amp; Snijder EJ (2006) Nidovirus transcription: how to make sense…? J. Gen. Virol 87(Pt 6):1403–1421." />
<meta name="citation_reference" content="Enjuanes L, Almazan F, Sola I, &amp; Zuniga S (2006) Biochemical aspects of coronavirus replication and virus-host interaction. Annu Rev Microbiol 60:211–230." />
<meta name="citation_reference" content="Marzano SY, et al. (2016) Identification of Diverse Mycoviruses through Metatranscriptomics Characterization of the Viromes of Five Major Fungal Plant Pathogens. J Virol 90(15):6846–6863." />
<meta name="citation_reference" content="Fauver JR, et al. (2016) West African Anopheles gambiae mosquitoes harbor a taxonomically diverse virome including new insect-specific flaviviruses, mononegaviruses, and totiviruses. Virology 498:288–299." />
<meta name="citation_reference" content="Dolja VV &amp; Koonin EV (2018) Metagenomics reshapes the concepts of RNA virus evolution by revealing extensive horizontal virus transfer. Virus Res 244:36–52." />
<meta name="citation_reference" content="Zhang YZ, Shi M, &amp; Holmes EC (2018) Using Metagenomics to Characterize an Expanding Virosphere. Cell 172(6):1168–1172." />
<meta name="citation_reference" content="Boursnell ME, et al. (1987) Completion of the sequence of the genome of the coronavirus avian infectious bronchitis virus. J Gen Virol 68 (Pt 1):57–77." />
<meta name="citation_reference" content="Altschul SF, Gish W, Miller W, Myers EW, &amp; Lipman DJ (1990) Basic local alignment search tool. J Mol Biol 215(3):403–410." />
<meta name="citation_reference" content="Silvester N, et al. (2017) The European Nucleotide Archive in 2017. Nucleic Acids Res." />
<meta name="citation_reference" content="Langmead B &amp; Salzberg SL (2012) Fast gapped-read alignment with Bowtie 2. Nat Methods 9(4):357–359." />
<meta name="citation_reference" content="Li H, et al. (2009) The Sequence Alignment/Map format and SAMtools. Bioinformatics 25(16):2078–2079." />
<meta name="citation_reference" content="Li H (2011) A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics 27(21):2987–2993." />
<meta name="citation_reference" content="King RS &amp; Newmark PA (2013) In situ hybridization protocol for enhanced detection of gene expression in the planarian Schmidtea mediterranea. BMC Dev Biol 13:8." />
<meta name="citation_reference" content="Brubacher JL, Vieira AP, &amp; Newmark PA (2014) Preparation of the planarian Schmidtea mediterranea for high-resolution histology and transmission electron microscopy. Nat Protoc 9(3):661–673." />
<meta name="citation_reference" content="Venable JH &amp; Coggeshall R (1965) A Simplified Lead Citrate Stain for Use in Electron Microscopy. J Cell Biol 25:407–408." />
<meta name="citation_reference" content="Robb SM, Gotting K, Ross E, &amp; Sanchez AA (2015) SmedGD 2.0: The Schmidtea mediterranea genome database. Genesis 53(8):535–546." />
<meta name="citation_reference" content="Andreeva A, et al. (2008) Data growth and its impact on the SCOP database: new developments. Nucleic Acids Res 36(Database issue):D419–425." />
<meta name="citation_reference" content="Berman HM, et al. (2000) The Protein Data Bank. Nucleic Acids Res 28(1):235–242." />
<meta name="citation_reference" content="Finn RD, et al. (2014) Pfam: the protein families database. Nucleic Acids Res 42(Database issue):D222–D230." />
<meta name="citation_reference" content="UniProt Consortium (2015) UniProt: a hub for protein information. Nucleic Acids Res 43(Database issue):D204–D212." />
<meta name="citation_reference" content="Adams MJ, et al. (2016) Ratification vote on taxonomic proposals to the International Committee on Taxonomy of Viruses (2016). Arch Virol 161:2921–2949." />
<meta name="citation_reference" content="Brister JR, Ako-Adjei D, Bao Y, &amp; Blinkova O (2015) NCBI viral genomes resource. Nucleic Acids Res 43(Database issue):D571–577." />
<meta name="citation_reference" content="Benson DA, et al. (2017) GenBank. Nucleic Acids Res 45(D1):D37–D42." />
<meta name="citation_reference" content="O’Leary NA, et al. (2016) Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Nucleic Acids Res 44(D1):D733–745." />
<meta name="citation_reference" content="Linding R, et al. (2003) Protein disorder prediction: implications for structural proteomics. Structure 11(11):1453–1459." />
<meta name="citation_reference" content="Drozdetskiy A, Cole C, Procter J, &amp; Barton GJ (2015) JPred4: a protein secondary structure prediction server. Nucleic Acids Res 43(W1):W389–394." />
<meta name="citation_reference" content="Petersen TN, Brunak S, von HG, &amp; Nielsen H (2011) SignalP 4.0: discriminating signal peptides from transmembrane regions. Nat. Methods 8(10):785–786." />
<meta name="citation_reference" content="Duckert P, Brunak S, &amp; Blom N (2004) Prediction of proprotein convertase cleavage sites. Protein Eng Des Sel 17(1):107–112." />
<meta name="citation_reference" content="Gorbalenya AE, et al. (2010) Practical application of bioinformatics by the multidisciplinary VIZIER consortium. Antiviral Res 87(2):95–110." />
<meta name="citation_reference" content="Eddy SR (2009) A new generation of homology search tools based on probabilistic inference. Genome Inform 23(1):205–211." />
<meta name="citation_reference" content="Soding J (2005) Protein homology detection by HMM-HMM comparison. Bioinformatics 21(7):951–960." />
<meta name="citation_reference" content="Zuker M (2003) Mfold web server for nucleic acid folding and hybridization prediction. Nucleic Acids Res 31(13):3406–3415." />
<meta name="citation_reference" content="Janssen S &amp; Giegerich R (2015) The RNA shapes studio. Bioinformatics 31(3):423–425." />
<meta name="citation_reference" content="Drummond AJ, Suchard MA, Xie D, &amp; Rambaut A (2012) Bayesian phylogenetics with BEAUti and the BEAST 1.7. Mol. Biol. Evol 29(8):1969–1973." />
<meta name="citation_reference" content="Darriba D, Taboada GL, Doallo R, &amp; Posada D (2011) ProtTest 3: fast selection of best-fit models of protein evolution. Bioinformatics 27(8):1164–1165." />
<meta name="citation_reference" content="Gulyaeva A, et al. (2017) Domain organization and evolution of the highly divergent 5’ coding region of genomes of arteriviruses including the novel possum nidovirus. J Virol 91(6)." />
<meta name="citation_reference" content="Pagel M, Meade A, &amp; Barker D (2004) Bayesian estimation of ancestral character states on phylogenies. Syst. Biol 53(5):673–684." />
<meta name="citation_reference" content="Kass RE &amp; Raftery AE (1995) Bayes Factors. J Am Stat Assoc 90(430):773–795." />
<meta name="citation_reference" content="Gouet P, Robert X, &amp; Courcelle E (2003) ESPript/ENDscript: Extracting and rendering sequence and 3D information from atomic structures of proteins. Nucleic Acids Res 31(13):3320–3323." />
<meta name="citation_reference" content="Heled J &amp; Bouckaert RR (2013) Looking for trees in the forest: summary tree from posterior samples. BMC. Evol. Biol 13:221." />
<meta name="citation_reference" content="R Core Team (2013) R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, Vienna, Austria)." />
<meta name="twitter:title" content="A planarian nidovirus expands the limits of RNA genome size" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication (albeit within a permissive environment). A 33.5-kb nidovirus has been considered close to the upper size limit for such entities; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the transition from primordial RNP to contemporary DNA-RNP-based life. Whether or not RNA viruses represent transitional steps on the road to DNA-based life, studies of larger RNA viruses advance our understanding of size constraints on RNP entities. For example, emergence of the largest previously known RNA genomes (20-34 kb in positive-stranded nidoviruses, including coronaviruses) is associated with a proofreading exoribonuclease encoded in the nidoviral open reading frame 1b (ORF1b). However, apparent constraints on the size of ORF1b, which encodes this and other key replicative enzymes, have been hypothesized to limit further expansion of viral RNA genomes. Here, we characterize a novel nidovirus (planarian secretory cell nidovirus; PSCNV) whose disproportionately large ORF1b-like region, and overall 41.1 kb genome, substantially extend the presumed limits on RNA genome size. This genome encodes a predicted 13,556-aa polyprotein in an unconventional single ORF, yet retains canonical nidoviral genome organization and expression, and key replicative domains. Our evolutionary analysis suggests that PSCNV diverged early from multi-ORF nidoviruses, and subsequently acquired additional genes, including those typical of large DNA viruses or hosts. PSCNV’s greatly expanded genome, proteomic complexity, and unique features – impressive in themselves – attest to the likelihood of still-larger RNA genomes awaiting discovery.

Significance Statement RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication. The upper genome size for such entities was assumed to be &lt;35 kb; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the proposed evolution of contemporary DNA-RNP-based life from primordial RNP entities. Here, we describe a nidovirus from planarians, whose 41.1 kb genome is 23% larger than the largest known of RNA virus. The planarian secretory cell nidovirus has broken apparent constraints on the size of the genomic subregion that encodes core replication machinery, and has acquired genes not previously observed in RNA viruses. This virus challenges and advances our understanding of the limits to RNA genome size." />
<meta name="og-title" property="og:title" content="A planarian nidovirus expands the limits of RNA genome size" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/299776v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication (albeit within a permissive environment). A 33.5-kb nidovirus has been considered close to the upper size limit for such entities; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the transition from primordial RNP to contemporary DNA-RNP-based life. Whether or not RNA viruses represent transitional steps on the road to DNA-based life, studies of larger RNA viruses advance our understanding of size constraints on RNP entities. For example, emergence of the largest previously known RNA genomes (20-34 kb in positive-stranded nidoviruses, including coronaviruses) is associated with a proofreading exoribonuclease encoded in the nidoviral open reading frame 1b (ORF1b). However, apparent constraints on the size of ORF1b, which encodes this and other key replicative enzymes, have been hypothesized to limit further expansion of viral RNA genomes. Here, we characterize a novel nidovirus (planarian secretory cell nidovirus; PSCNV) whose disproportionately large ORF1b-like region, and overall 41.1 kb genome, substantially extend the presumed limits on RNA genome size. This genome encodes a predicted 13,556-aa polyprotein in an unconventional single ORF, yet retains canonical nidoviral genome organization and expression, and key replicative domains. Our evolutionary analysis suggests that PSCNV diverged early from multi-ORF nidoviruses, and subsequently acquired additional genes, including those typical of large DNA viruses or hosts. PSCNV’s greatly expanded genome, proteomic complexity, and unique features – impressive in themselves – attest to the likelihood of still-larger RNA genomes awaiting discovery.

Significance Statement RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication. The upper genome size for such entities was assumed to be &lt;35 kb; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the proposed evolution of contemporary DNA-RNP-based life from primordial RNP entities. Here, we describe a nidovirus from planarians, whose 41.1 kb genome is 23% larger than the largest known of RNA virus. The planarian secretory cell nidovirus has broken apparent constraints on the size of the genomic subregion that encodes core replication machinery, and has acquired genes not previously observed in RNA viruses. This virus challenges and advances our understanding of the limits to RNA genome size." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-04-11" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/299776v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/299776v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/299776v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/299776v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/92053" />
    <title>A planarian nidovirus expands the limits of RNA genome size | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"p7Ah2IaHnWcH6qGocP_CwaPjT_k2iyRNeVFlFbyL5eU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"92053","apath":"\/biorxiv\/early\/2018\/04\/11\/299776.atom","pisa":"biorxiv;299776v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;299776\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;299776\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;299776v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;299776\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;299776\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/299776v1","title":"A planarian nidovirus expands the limits of RNA genome size","identifier":"node\/92053"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jW5MEHDNgyjohCJAJj66MFyPgMNQ5T0+0Poj13BOxPX\\\\\\\/89ccqJ+IfIi\\\\\\\/Hjn8gBLa3ZaUYPpO2n989WRwbdlPhkz7\\\\\\\/lG2JQgV33VNt+i1SISmkV6t4qpCUc9OtlpQxrVOq1DXtPRhRiTZMkjcHwVffvIL8V\\\\\\\/VE4tI3ySuh21ltMMUe0StYdph6uqPO3hlR+Z\\\\\\\/vEQwsoXA5tLZA7BVWWRMJkC6+4AJQ1HQVlVELefTvreK0eqSOJM7BaiSQVdNd7KUtmgMIeY9PqayfuOCujG0EswTuRmuG9GOwO8R1\\\\\\\/0gIALTpJULa\\\\\\\/HjdMDN0OOCnj30dwsoPnfzLBmPcsLm2AeakcvWC9pmZqWE1c2Fece9iJnb5PKwZcdbtqan6IMtrj5zRiGNfpTDIMNao2tWrdeh95q2dAfbRqvDJA1TYFX3Wn\\\\\\\/IRxlc+UgWZtommhSR8UzBZLnLUoFZJjNHifqzq4GA9MKJN3z7TCq4DjdSWEOkjmA2xLtVjN35uabmHEQmhj\\\\\\\/9bkRWe5VMPAVxnC1hG7foBfzTjh+DKNgP3F0f9CEqLQ0bEhcyjxGWSJOReoztxuAISmaO7vtZdg5ycnXslRDwgsEJ+qS1SeyYVdkKMhNCOneu6YsDwr4SyeEWm\\\\\\\/BaexB8TUwb53+bFL1LCp7yEu0VCo44bqSIxRS2OP855CvK6RAfoH4ywzy4aIv1GYqvb+yM0rKLvtkeqbnH8b0Weu3rh\\\\\\\/0bIQU1MCEbKMlP2XBlqSV1kJ4xI\\\\\\\/YEY9gcL4gJAMH9wleV8VLtJEcJ9rcpgpMdAy3kP6DHxa1JFR4mREqckNBELBMFsi+8lP2h4mp8v8e7OARMu98LxTMLs+351qUSTWTXwhGO+xCbCG5YLwKt+\\\\\\\/H3pKbuvCqn1TfAbc0LkPuQOOVL4UpmOELwEAETd9cTj9j2NfdnZrlZci\\\\\\\/yjddZov9jmXxzu1t2fJRZsW1OoFUpmnn\\\\\\\/nVqC24rE1QKbXFSiVCkfRu6zt6GQ+7MirZTDcNxeN4JH844MFuDhng0262+LefLPuPaG2AP2lL5hjkgPnwkU6QZShACqWWAQFjRenNet7PyVqRu67NF4UnJxKvmQpeEUamHHx4dfNc\\\\\\\/syXSNC5FAgeLxPUgHDhO4eIhtyeyAIEwVqIiTCQPaDNvE4iMDHVGHPU9WQNAuD1ieLpmNNfAashM0m0kSR2gyIll+8QdNO+VkbrkbFBtWEpPGtlYMamkD77b3Q2LBpGJRzXkKp\\\\\\\/8WBlF9rJJmS3XKLwTE3dC0I\\\\\\\/nHVXvL0wj2NOxZwR+x7tbUbUQAKemljkVzDmQymj96rLNbpNWRuV4luDLTLm0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/1a8f9mCOBuG25RY2HV8\\\\\\\/g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f9d8866808479cd89adb7a15f877e93efa6c06847c001ddf903e8245365817fb\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/299776v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-92053 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/299776v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MT65ghokqZ3Wcos_8c_6XEsUCOzWjlXcAhpjkRX2V1w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="92053" id="node92053" data-pisa="biorxiv;299776v1" data-pisa-master="biorxiv;299776" data-apath="/biorxiv/early/2018/04/11/299776.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A planarian nidovirus expands the limits of RNA genome size</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Amir</span> <span class="nlm-surname">Saberi</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anastasia A.</span> <span class="nlm-surname">Gulyaeva</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">John L.</span> <span class="nlm-surname">Brubacher</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Phillip A.</span> <span class="nlm-surname">Newmark</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Alexander E.</span> <span class="nlm-surname">Gorbalenya</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/299776 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node92053" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amir Saberi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Howard Hughes Medical Institute, Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign</span>, Urbana, Illinois, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amir%2BSaberi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saberi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmir%2BSaberi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anastasia A. Gulyaeva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Medical Microbiology, Leiden University Medical Center</span>, Leiden, <span class='nlm-country'>The Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anastasia%2BA.%2BGulyaeva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gulyaeva%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnastasia%2BA.%2BGulyaeva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John L. Brubacher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Department of Biology, Canadian Mennonite University</span>, Winnipeg, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BL.%2BBrubacher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brubacher%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BL.%2BBrubacher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Phillip A. Newmark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Howard Hughes Medical Institute, Department of Cell and Developmental Biology, University of Illinois at Urbana-Champaign</span>, Urbana, Illinois, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Phillip%2BA.%2BNewmark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Newmark%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhillip%2BA.%2BNewmark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pnewmark@morgridge.org" class="" data-icon-position="" data-hide-link-title="0">pnewmark@morgridge.org</a>
<a href="mailto:A.E.Gorbalenya@lumc.nl" class="" data-icon-position="" data-hide-link-title="0">A.E.Gorbalenya@lumc.nl</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alexander E. Gorbalenya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Department of Medical Microbiology, Leiden University Medical Center</span>, Leiden, <span class='nlm-country'>The Netherlands</span></div><div class='author-affiliation'><span class='nlm-sup'>d</span><span class='nlm-institution'>Faculty of Bioengineering and Bioinformatics, Lomonosov Moscow State University</span>, Moscow, <span class='nlm-country'>Russia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BGorbalenya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gorbalenya%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BE.%2BGorbalenya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pnewmark@morgridge.org" class="" data-icon-position="" data-hide-link-title="0">pnewmark@morgridge.org</a>
<a href="mailto:A.E.Gorbalenya@lumc.nl" class="" data-icon-position="" data-hide-link-title="0">A.E.Gorbalenya@lumc.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/299776v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:92053" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:92053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/299776v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:92053" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:92053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/299776v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:92053" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:92053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/299776v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:92053" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:92053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/299776v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:92053" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:92053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/299776v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:92053" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:92053/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-5">RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication (albeit within a permissive environment). A 33.5-kb nidovirus has been considered close to the upper size limit for such entities; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the transition from primordial RNP to contemporary DNA-RNP-based life. Whether or not RNA viruses represent transitional steps on the road to DNA-based life, studies of larger RNA viruses advance our understanding of size constraints on RNP entities. For example, emergence of the largest previously known RNA genomes (20-34 kb in positive-stranded nidoviruses, including coronaviruses) is associated with a proofreading exoribonuclease encoded in the nidoviral open reading frame 1b (ORF1b). However, apparent constraints on the size of ORF1b, which encodes this and other key replicative enzymes, have been hypothesized to limit further expansion of viral RNA genomes. Here, we characterize a novel nidovirus (planarian secretory cell nidovirus; PSCNV) whose disproportionately large ORF1b-like region, and overall 41.1 kb genome, substantially extend the presumed limits on RNA genome size. This genome encodes a predicted 13,556-aa polyprotein in an unconventional single ORF, yet retains canonical nidoviral genome organization and expression, and key replicative domains. Our evolutionary analysis suggests that PSCNV diverged early from multi-ORF nidoviruses, and subsequently acquired additional genes, including those typical of large DNA viruses or hosts. PSCNV’s greatly expanded genome, proteomic complexity, and unique features – impressive in themselves – attest to the likelihood of still-larger RNA genomes awaiting discovery.</p><div id="sec-1" class="subsection"><p id="p-6"><strong>Significance Statement</strong> RNA viruses are the only known RNA-protein (RNP) entities capable of autonomous replication. The upper genome size for such entities was assumed to be &lt;35 kb; conversely, the minimal cellular DNA genome is ~200 kb. This large difference presents a daunting gap for the proposed evolution of contemporary DNA-RNP-based life from primordial RNP entities. Here, we describe a nidovirus from planarians, whose 41.1 kb genome is 23% larger than the largest known of RNA virus. The planarian secretory cell nidovirus has broken apparent constraints on the size of the genomic subregion that encodes core replication machinery, and has acquired genes not previously observed in RNA viruses. This virus challenges and advances our understanding of the limits to RNA genome size.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-7">Radiation of primitive life over its initial two billion years of evolution was likely accompanied by genome expansion and a proposed transition from RNA-based through RNA-protein to DNA-based life (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). The feasibility of an autonomous ancient RNA genome and the mechanisms underlying such fateful transitions are challenging to reconstruct. It is especially unclear whether such RNA entities ever evolved genomes close to the 100–300 kilobases (kb) range (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>) of the “minimal” reconstructed cellular DNA genome (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). This range overlaps with the upper size limit of nuclear pre-mRNAs (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>); however, pre-mRNAs are incapable of self-replication, the defining property of primordial genomic RNAs.</p><p id="p-8">RNA viruses may uniquely illuminate the evolutionary constraints on RNA genome size, whether or not they descended directly from primitive RNA-based entities (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>–<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). The average-sized RNA virus genome is around 10 kb, due to low-fidelity RNA replication without proofreading (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>) However, positive-stranded RNA genomes of viruses of the order <em>Nidovirales</em>, which includes pathogenic coronaviruses, range from 12.7 to 33.5 kb, the largest known RNA genome (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>, Table S1). These viruses may have evolved genomes larger than 20 kb after acquiring a proofreading exoribonuclease (ExoN) that improved replication fidelity (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>–<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). Despite this notable innovation, over the last 20 years of virus discovery, including unbiased metagenomics of invertebrate and vertebrate RNA viruses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>), the largest-known RNA viral genome has only increased ~10% in size – a mere fraction of the nearly ten-fold increase observed for DNA viruses (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>–<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1A</a>). Apparently, genome sizes of large nidoviruses are tightly constrained by the interplay of genetic and host-specific factors, with the size of open reading frame 1b (ORF1b), encoding key replicative enzymes, emerging as a major barrier for further genome expansion (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). Hence, ~34 kb is considered the de facto limit of RNA virus genome size (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome and proteome of PSCNV. (A) Timeline of discovery of largest RNA and DNA virus genomes (1982&#x2013;2017). PV, poliovirus. (B) Genome sizes of nidoviruses encoding an ExoN domain or not. Percentage indicates the difference between sizes of PSCNV and the next-largest entity (also in D,G). (C) ORFs and conserved protein domains in PSCNV and nidoviruses. ORF1a frame is set as zero. TM, transmembrane domain (TM helices shown by black bars above TM domains); ANK, Ankyrin; RNase T2, ribonuclease T2 homolog; for other domain acronyms, see text. (D) PSCNV polyprotein vs RNA virus protein sizes. (E) Closest cellular homologs of PSCNV ANK are ranked by similarity (left, above the broken baseline) and depicted through phylogeny (right; PP, posterior probability of clades) along with protein domain architecture: S. med, Schmidtea mediterranea; D. lac, Dendrocoelum lacteum; RHD, Rel homology DNA-binding domain. (F) Phylogeny and ORF number of PSCNV, nido-, and astroviruses (outgroup). (G) Sizes of three nidovirus ORF regions. (H) Size increase of three genome regions in PSCNV (grey bars) relative to the increase expected if all regions expanded evenly (broken line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-414990573" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Genome and proteome of PSCNV.&lt;/span&gt; (&lt;em&gt;&lt;strong&gt;A&lt;/strong&gt;&lt;/em&gt;) Timeline of discovery of largest RNA and DNA virus genomes (1982&#x2013;2017). PV, poliovirus. (&lt;em&gt;&lt;strong&gt;B&lt;/strong&gt;&lt;/em&gt;) Genome sizes of nidoviruses encoding an ExoN domain or not. Percentage indicates the difference between sizes of PSCNV and the next-largest entity (also in &lt;em&gt;D&lt;/em&gt;,&lt;em&gt;G&lt;/em&gt;). (&lt;em&gt;&lt;strong&gt;C&lt;/strong&gt;&lt;/em&gt;) ORFs and conserved protein domains in PSCNV and nidoviruses. ORF1a frame is set as zero. TM, transmembrane domain (TM helices shown by black bars above TM domains); ANK, Ankyrin; RNase T2, ribonuclease T2 homolog; for other domain acronyms, see text. (&lt;em&gt;&lt;strong&gt;D&lt;/strong&gt;&lt;/em&gt;) PSCNV polyprotein vs RNA virus protein sizes. (&lt;em&gt;&lt;strong&gt;E&lt;/strong&gt;&lt;/em&gt;) Closest cellular homologs of PSCNV ANK are ranked by similarity (left, above the broken baseline) and depicted through phylogeny (right; PP, posterior probability of clades) along with protein domain architecture: &lt;em&gt;S. med&lt;/em&gt;, &lt;em&gt;Schmidtea mediterranea&lt;/em&gt;; &lt;em&gt;D. lac&lt;/em&gt;, &lt;em&gt;Dendrocoelum lacteum&lt;/em&gt;; RHD, Rel homology DNA-binding domain. (&lt;em&gt;&lt;strong&gt;F&lt;/strong&gt;&lt;/em&gt;) Phylogeny and ORF number of PSCNV, nido-, and astroviruses (outgroup). (&lt;em&gt;&lt;strong&gt;G&lt;/strong&gt;&lt;/em&gt;) Sizes of three nidovirus ORF regions. (&lt;em&gt;&lt;strong&gt;H&lt;/strong&gt;&lt;/em&gt;) Size increase of three genome regions in PSCNV (grey bars) relative to the increase expected if all regions expanded evenly (broken line).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F1.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F1.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">Genome and proteome of PSCNV.</span><p id="p-9" class="first-child">(<em><strong>A</strong></em>) Timeline of discovery of largest RNA and DNA virus genomes (1982–2017). PV, poliovirus. (<em><strong>B</strong></em>) Genome sizes of nidoviruses encoding an ExoN domain or not. Percentage indicates the difference between sizes of PSCNV and the next-largest entity (also in <em>D</em>,<em>G</em>). (<em><strong>C</strong></em>) ORFs and conserved protein domains in PSCNV and nidoviruses. ORF1a frame is set as zero. TM, transmembrane domain (TM helices shown by black bars above TM domains); ANK, Ankyrin; RNase T2, ribonuclease T2 homolog; for other domain acronyms, see text. (<em><strong>D</strong></em>) PSCNV polyprotein vs RNA virus protein sizes. (<em><strong>E</strong></em>) Closest cellular homologs of PSCNV ANK are ranked by similarity (left, above the broken baseline) and depicted through phylogeny (right; PP, posterior probability of clades) along with protein domain architecture: <em>S. med</em>, <em>Schmidtea mediterranea</em>; <em>D. lac</em>, <em>Dendrocoelum lacteum</em>; RHD, Rel homology DNA-binding domain. (<em><strong>F</strong></em>) Phylogeny and ORF number of PSCNV, nido-, and astroviruses (outgroup). (<em><strong>G</strong></em>) Sizes of three nidovirus ORF regions. (<em><strong>H</strong></em>) Size increase of three genome regions in PSCNV (grey bars) relative to the increase expected if all regions expanded evenly (broken line).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-3"><h2 class="">Results and discussion</h2><div id="sec-4" class="subsection"><h3>Identification of a large RNA virus from planarians</h3><p id="p-10">To examine whether this limit applies beyond the currently recognized ~3000 RNA virus species that infect just hundreds of host species, further sampling of virus diversity is required. To this end, we analyzed <em>de novo</em> transcriptomes of the planarian <em>Schmidtea mediterranea</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>), in which viruses remain virtually unknown, and which exists as two reproductively distinct strains: sexuals that reproduce as simultaneous hermaphrodites and asexuals that reproduce via transverse fission (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). We determined the complete genome sequence of a putative nidovirus corresponding to one particularly large contig (SI Text, Table S2, Fig. S1). Excluding the polyadenylated tail, the fully assembled genome contains 41,103 nucleotides (nt) and encodes a single 40,671-nt ORF that is flanked by a 128-nt 5’-UTR and a 304-nt 3’-UTR (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B,C</a>). The presence of this RNA in vivo was confirmed by RT-PCR amplification of overlapping regions of 4–7 kb, 5’-RACE, and multiple EST clones (Fig. S1) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). These sequences could not be amplified from <em>S. mediterranea</em> genomic DNA, nor could they be found in the genome (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>); thus, they appear to derive from an exogenous source. Based on several criteria (see below), we assigned this RNA sequence to the genome of a virus we dubbed Planarian Secretory Cell Nidovirus (PSCNV). PSCNV circulates in both sexual and asexual host strains: analysis of RNA-seq data from three laboratories revealed its very low (&lt;0.1%), albeit predominantly non-synonymous variation (SI Text, Tables S3,4). Analysis of <em>Planaria torva</em> transcriptome identified three contigs of the cumulative ~9400 nt size, whose regions displayed from 25% to 48% aa identity with PSCNV, indicative of at least one additional putative nidovirus (SI Text, Fig. S2) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>).</p></div><div id="sec-5" class="subsection"><h3>The PSCNV proteome reveals a unique nidovirus</h3><p id="p-11">The genome and proteome of PSCNV are by far the largest yet reported for an RNA virus. Its RNA genome is ~25% larger than that of the next-largest known RNA virus (also a nidovirus), which is separated by a comparable margin from the first nidovirus genome sequenced 30 years ago (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1A</a>). The size of the predicted PSCNV polyprotein (13,556 amino acids, aa) is 58–67% larger than the largest known RNA virus proteins produced from a single ORF (8,572 aa) or multiple ORFs through frameshifting (8,108 aa) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1D</a>).</p><p id="p-12">We delineated at least twenty domains in the PSCNV polyprotein using a complex multistage procedure that combined different analyses within a probabilistic framework (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1C</a>; SI Text, Table S5, and Figs. S3–S16). These domains include distant homologs of all ten domains characteristic of invertebrate nidoviruses (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>) and established with utmost confidence: seven ORF1a/1b enzymes (3C-like protease, 3CLpro; nidovirus RdRp-associated nucleotidyltransferase, NiRAN; RNA-dependent RNA polymerase, RdRp; superfamily 1 helicase, HEL1; exoribonuclease of DEDDh subfamily, ExoN; and SAM-dependent N7- and 2’-O-methyltransferases, N-MT and O-MT, respectively) and three non-enzymatic domains (transmembrane domain 2 and 3, TM2 and TM3, respectively; and zinc-binding domain, ZBD) (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1C</a>). These domains are syntenic in PSCNV and nidoviruses, with the conservation of NiRAN and ZBD in PSCNV being of particular significance, since both domains are the only known genetic markers of the order <em>Nidovirales</em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). Thus, these findings further support the phylogenetic affinity of PSCNV and nidoviruses (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1C</a>), which we quantified in separate analyses (see below). Accordingly, three functional regions corresponding to the ORF1a, ORF1b, and 3’ORFs of nidoviruses could be identified tentatively in the PSCNV genome, with three characteristic nidovirus domains mapping to the ORF1a-like region and seven domains to the ORF1b-like region.</p><p id="p-13">The enzymatic domains contain most essential catalytic residues, suggesting that they retain canonical functions, although notable deviations were also observed (Figs. S7–S14). Unique features of PSCNV’s putative enzymes include: a) 3CLpro encodes a valine (Val) residue in the position commonly occupied by a His residue in the putative substrate-binding pocket (GXV vs G/YXH) that controls the P1 substrate preference for Glu or glutamine (Gln) residues (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>–<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>); b) a Ser residue of the nidovirus-specific SDD signature of RdRp (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>) is replaced by glycine (Gly) – a characteristic of ssRNA+ viruses other than nidoviruses; c) the invariant aspartate (Asp), along with the highly conserved Ser/Thr, of motif B<sub>N</sub> of the NiRAN (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>) are substituted with Thr and Asn, respectively; d) four Cys/His residues coordinating a Zn<sup>2+</sup> in the active site of ExoN (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>) are replaced in PSCNV, which, like non-nidoviral homologs, apparently lacks this Zn-finger.</p><p id="p-14">Conserved sequence features reveal that, as in other nidoviruses (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>), the PSCNV 3’ORFs-like region may encode major structural proteins of enveloped virions, including the transmembrane glycoproteins known as S and M in coronaviruses, and nucleocapsid (N) protein (SI Text, Fig. S4). This region also appears to encode several notable domains: a ~130 aa region with Thr and Ser accounting for 70.7% of residues (Fig. S4); an Ankyrin domain (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figs. 1E</a> and S16) and two fibronectin type II domains (FN2a and FN2b; Fig. S15), unique for RNA and all viruses, respectively. The Thr-rich and Thr/Ser-rich regions have been implicated in mediating adherence of fungal and bacterial extracellular (glyco) proteins to various substrates (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>), suggesting a similar role in PSCNV. The Ankyrin domain might counteract host immune responses by suppressing the NF-kB signaling pathway, as observed in poxviruses (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>), while FN2a/b could facilitate virus dissemination (SI Text).</p><p id="p-15">Computational analysis suggests that PSCNV might have acquired the Ankyrin domain from a planarian host relatively late in evolution (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1E</a>), while the origins of the FN2 domains remain uncertain. We also identified a ribonuclease T2 homolog upstream of the putative 3CLpro in the ORF1a-like region (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1C</a>), in which all active-site residues are conserved (Fig. S6). Pestivirus and polydnavirus ribonuclease T2 homologs modulate cell toxicity and immunity (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>).</p></div><div id="sec-6" class="subsection"><h3>PSCNV likely derived from multi-ORF nidoviruses and expanded disproportionately in the ORF1b-like region</h3><p id="p-16">To learn about the evolution of PSCNV, we inferred the rooted phylogeny of PSCNV based on the RdRp protein domain of nidoviruses and two astroviruses (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Fig. 1F</a>; SI Text). PSCNV predominantly clustered with other invertebrate nidoviruses in the Bayesian sample, basal to either mesoni- and roniviruses (54.7% of the trees), roniviruses (20.6%), or mesoniviruses (13.4%). Ancestral state reconstruction analysis indicated that the most recent common ancestor of nidoviruses contained multiple ORFs (Log Bayes Factor 6.06) (Fig. S17). Thus, the single-ORF organization of PSCNV likely emerged through loss of signals separating the three ORF regions of ancestral nidoviruses.</p><p id="p-17">Compared to known nidoviruses, PSCNV genome size increased unevenly in the three main regions (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Fig. 1G,H</a>, SI Text, Table S6). The size increase of the ORF1b-like region was an extraordinary 9-fold higher than would be expected from the extremely low variation in the size of this region among other nidoviruses normalized to genome size variation. This observation implies that PSCNV broke the tightly constrained ORF1b size barrier (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>), but expanded modestly in the ORF1a-like and 3’ORFs-like regions. This pattern fits the wavelike dynamic model of genome expansion in nidoviruses (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>). Accordingly, extant nidovirus genomes of different sizes reached particular points on a common trajectory of genome expansion dominated by consecutive increases of ORF1b, ORF1a, and 3’ORFs. A complete cycle of genome expansion was reported to encompass the 12.7-31.7 kb range, with the expansion of ORF1b expected to lead a second cycle beyond that size range. The sizes of PSCNV’s three main regions support this model, and show that nidoviruses were able to start a second cycle of genome expansion, whose feasibility has remained uncertain so far. This observation makes plausible the existence of other yet-undiscovered nidoviruses that have moved further along this genome-expanding trajectory.</p></div><div id="sec-7" class="subsection"><h3>PSCNV infects the secretory cells of planarians</h3><p id="p-18">We next examined PSCNV infection in planarian tissues by whole-mount in situ hybridization (ISH). PSCNV RNA was detected abundantly in cells of the secretory system in both sexuals and asexuals (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). Fluorescent ISH revealed viral RNA in gland cell projections that form secretory canals (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a>). Notably, viral RNA was detected largely in ventral cells (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2C</a>) whose localization corresponds to mucus-secreting cells producing the slime upon which planarians glide, and use to immobilize prey (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression of PSCNV RNA in planarians. (A) PSCNV RNA (blue) detected in asexual (left) and sexual S. mediterranea by whole-mount ISH. (B) Fluorescent ISH showing PSCNV expression in a sexual planarian. Insets show higher magnification of areas indicated by boxes. Top two insets are confocal projections. Secretory cell projections to lateral body edges are indicated by arrowheads. (C) Tiled confocal projections of PSCNV expression in a cross-section. Cells expressing PSCNV are ventrally located (arrowheads). Gut (&#x201C;g&#x201D;) and pharynx (&#x201C;ph&#x201D;) are indicated. DAPI (blue) labels nuclei." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-414990573" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Expression of PSCNV RNA in planarians.&lt;/span&gt; (&lt;em&gt;&lt;strong&gt;A&lt;/strong&gt;&lt;/em&gt;) PSCNV RNA (blue) detected in asexual (left) and sexual &lt;em&gt;S. mediterranea&lt;/em&gt; by whole-mount ISH. (&lt;em&gt;&lt;strong&gt;B&lt;/strong&gt;&lt;/em&gt;) Fluorescent ISH showing PSCNV expression in a sexual planarian. Insets show higher magnification of areas indicated by boxes. Top two insets are confocal projections. Secretory cell projections to lateral body edges are indicated by arrowheads. (&lt;em&gt;&lt;strong&gt;C&lt;/strong&gt;&lt;/em&gt;) Tiled confocal projections of PSCNV expression in a cross-section. Cells expressing PSCNV are ventrally located (arrowheads). Gut (&#x201C;g&#x201D;) and pharynx (&#x201C;ph&#x201D;) are indicated. DAPI (blue) labels nuclei.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F2.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F2.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <span class="caption-title">Expression of PSCNV RNA in planarians.</span><p id="p-19" class="first-child">(<em><strong>A</strong></em>) PSCNV RNA (blue) detected in asexual (left) and sexual <em>S. mediterranea</em> by whole-mount ISH. (<em><strong>B</strong></em>) Fluorescent ISH showing PSCNV expression in a sexual planarian. Insets show higher magnification of areas indicated by boxes. Top two insets are confocal projections. Secretory cell projections to lateral body edges are indicated by arrowheads. (<em><strong>C</strong></em>) Tiled confocal projections of PSCNV expression in a cross-section. Cells expressing PSCNV are ventrally located (arrowheads). Gut (“g”) and pharynx (“ph”) are indicated. DAPI (blue) labels nuclei.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">We then analyzed planarians by electron microscopy (EM) for the presence of viral structures. Membrane-bound compartments with 90–150 nm spherical-to-oblong particles resembling nidoviral nucleocapsids (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>) were found in the cytoplasm of mucus-secreting cells, which are abundant in planarian parenchyma. These sub-epidermal gland cells are notable for their extensive rough endoplasmic reticulum and long projections into the ventral epithelium, through which they secrete mucus (Fig. S18). These cells provide an ideal environment for nidoviral replication – which co-opts host membranes to produce viral replication complexes (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>). Putative viral particles were found both in deep regions of these cells, and in their trans-epidermal projections (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>–C). The latter location suggests a route for viral transmission. Notably, particles in sub-epidermal layers have a “hazy” appearance and are embedded in a relatively electron-dense matrix (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3D</a>). In contrast, particles closer to the apical surface of the epidermis appear as relatively discrete structures, standing out against electron-lucent surrounding material (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3E</a>). The size, ultrastructure, and host-cell locations are all consistent with these structures being nidoviral nucleocapsids (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Putative PSCNV particles revealed by electron microscopy. (A) Adjacent histological section to orient EM images. Black rectangle corresponds to location of (B), a low-magnification EM view to provide context. White rectangle corresponds to location of (C), in which putative viral particles enclosed within membrane sacs are indicated by arrowheads. White rectangle in (C) and square in (B) indicate positions of higher magnification views shown in (D) and (E), respectively &#x2013; each illustrating several viral particles within a membrane sac. In top-left of (B), note the mucus granules adjacent to virus-laden sacs (see also Fig. S18). Scale bars as indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-414990573" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Putative PSCNV particles revealed by electron microscopy.&lt;/span&gt; (&lt;em&gt;&lt;strong&gt;A&lt;/strong&gt;&lt;/em&gt;) Adjacent histological section to orient EM images. Black rectangle corresponds to location of (&lt;em&gt;&lt;strong&gt;B&lt;/strong&gt;&lt;/em&gt;), a low-magnification EM view to provide context. White rectangle corresponds to location of (&lt;em&gt;&lt;strong&gt;C&lt;/strong&gt;&lt;/em&gt;), in which putative viral particles enclosed within membrane sacs are indicated by arrowheads. White rectangle in (&lt;em&gt;C&lt;/em&gt;) and square in (&lt;em&gt;B&lt;/em&gt;) indicate positions of higher magnification views shown in (&lt;em&gt;&lt;strong&gt;D&lt;/strong&gt;&lt;/em&gt;) and (&lt;em&gt;&lt;strong&gt;E&lt;/strong&gt;&lt;/em&gt;), respectively &#x2013; each illustrating several viral particles within a membrane sac. In top-left of (&lt;em&gt;B&lt;/em&gt;), note the mucus granules adjacent to virus-laden sacs (see also Fig. S18). Scale bars as indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F3.medium.gif" width="301" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F3.medium.gif" width="301" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">Putative PSCNV particles revealed by electron microscopy.</span><p id="p-21" class="first-child">(<em><strong>A</strong></em>) Adjacent histological section to orient EM images. Black rectangle corresponds to location of (<em><strong>B</strong></em>), a low-magnification EM view to provide context. White rectangle corresponds to location of (<em><strong>C</strong></em>), in which putative viral particles enclosed within membrane sacs are indicated by arrowheads. White rectangle in (<em>C</em>) and square in (<em>B</em>) indicate positions of higher magnification views shown in (<em><strong>D</strong></em>) and (<em><strong>E</strong></em>), respectively – each illustrating several viral particles within a membrane sac. In top-left of (<em>B</em>), note the mucus granules adjacent to virus-laden sacs (see also Fig. S18). Scale bars as indicated.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Potential mechanisms for varying stoichiometries of viral proteins encoded by a single-ORF genome</h3><p id="p-22">Virus reproduction requires different viral protein stoichiometries at distinct replicative cycle stages, a challenge for a single-ORF genome theoretically producing equimolar quantities of encoded polypeptides. For example, the putative structural proteins encoded by 3’ORFs are needed in much larger quantities than replicative enzymes at the end of nidovirus infection, and vice versa. We analysed the PSCNV genome and its transcript expression to resolve this apparent paradox.</p><p id="p-23">Nidoviruses use −1 programmed ribosomal frameshifting (PRF) at the ORF1a/1b border to attenuate the synthesis of major replicative enzymes upon genome translation (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>). We located a potential −1 PRF signal in the canonical position in the PSCNV genome (SI Text). This signal could potentially attenuate in-frame translation downstream of the ORF1a-like region (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>) similar to a mechanism used by encephalomyocarditis virus (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome translation and transcription. (A) Comparison of replicase ORFs translation mechanism in nidoviruses (left) and PSCNV (right, hypothetical). On the top, RNA structure of the PRF sites is presented: slippery sequence, pink; pseudoknot, blue. (B) Mean depth of RNA-seq coverage along the PSCNV genome (approximated by exponential regression in ORF1b-like and 3&#x2019;ORFs-like regions) calculated based on five datasets used to assemble the transcriptomes in which PSCNV was found (24), position of oligonucleotide repeats (leader and body TRSs) in the genome, and their alignment with subgenomic (sg) mRNA 5&#x2019;-terminus established by 5&#x2019;-RACE (nucleotide mismatches between sg mRNA and TRSs are shown on grey background). (C) Predicted secondary structure of TRSs. TRSs are highlighted in green, region upstream of bTRS, interacting with its 5&#x2019;-terminus in yellow, asterisks indicate mismatching nucleotides of TRSs. (D) Model of discontinuous RNA synthesis mediated by TRSs and their secondary structure. Genome is shown by solid line and nascent minus strand by dashed line. Color code matches that of panel C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-414990573" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Genome translation and transcription.&lt;/span&gt; (&lt;em&gt;&lt;strong&gt;A&lt;/strong&gt;&lt;/em&gt;) Comparison of replicase ORFs translation mechanism in nidoviruses (left) and PSCNV (right, hypothetical). On the top, RNA structure of the PRF sites is presented: slippery sequence, pink; pseudoknot, blue. (&lt;em&gt;&lt;strong&gt;B&lt;/strong&gt;&lt;/em&gt;) Mean depth of RNA-seq coverage along the PSCNV genome (approximated by exponential regression in ORF1b-like and 3&#x2019;ORFs-like regions) calculated based on five datasets used to assemble the transcriptomes in which PSCNV was found (24), position of oligonucleotide repeats (leader and body TRSs) in the genome, and their alignment with subgenomic (sg) mRNA 5&#x2019;-terminus established by 5&#x2019;-RACE (nucleotide mismatches between sg mRNA and TRSs are shown on grey background). (&lt;em&gt;&lt;strong&gt;C&lt;/strong&gt;&lt;/em&gt;) Predicted secondary structure of TRSs. TRSs are highlighted in green, region upstream of bTRS, interacting with its 5&#x2019;-terminus in yellow, asterisks indicate mismatching nucleotides of TRSs. (&lt;em&gt;&lt;strong&gt;D&lt;/strong&gt;&lt;/em&gt;) Model of discontinuous RNA synthesis mediated by TRSs and their secondary structure. Genome is shown by solid line and nascent minus strand by dashed line. Color code matches that of panel &lt;em&gt;C&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F4.medium.gif" width="284" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F4.medium.gif" width="284" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/11/299776/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <span class="caption-title">Genome translation and transcription.</span><p id="p-24" class="first-child">(<em><strong>A</strong></em>) Comparison of replicase ORFs translation mechanism in nidoviruses (left) and PSCNV (right, hypothetical). On the top, RNA structure of the PRF sites is presented: slippery sequence, pink; pseudoknot, blue. (<em><strong>B</strong></em>) Mean depth of RNA-seq coverage along the PSCNV genome (approximated by exponential regression in ORF1b-like and 3’ORFs-like regions) calculated based on five datasets used to assemble the transcriptomes in which PSCNV was found (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>), position of oligonucleotide repeats (leader and body TRSs) in the genome, and their alignment with subgenomic (sg) mRNA 5’-terminus established by 5’-RACE (nucleotide mismatches between sg mRNA and TRSs are shown on grey background). (<em><strong>C</strong></em>) Predicted secondary structure of TRSs. TRSs are highlighted in green, region upstream of bTRS, interacting with its 5’-terminus in yellow, asterisks indicate mismatching nucleotides of TRSs. (<em><strong>D</strong></em>) Model of discontinuous RNA synthesis mediated by TRSs and their secondary structure. Genome is shown by solid line and nascent minus strand by dashed line. Color code matches that of panel <em>C</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The PSCNV putative 3CLpro domain is expected (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>) to release itself and all downstream polyprotein components up to the border between the ORF1b- and 3’ORFs-like regions (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Fig. 1C</a>). The polyprotein region processed by 3CLpro, and its flanking regions, are of comparable sizes (5006 vs 4400 and 4150 aa), suggesting further posttranslational processing. Indeed, we have identified a number of candidate cleavage sites for host furin-like proteases (Fig. S4), commonly involved in processing viral glycoprotein precursors (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>) (SI Text).</p><p id="p-26">We obtained several lines of evidence for upregulated transcription of the 3’ORFs-like region (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>–D; SI Text). The PSCNV genome contains a pair of 59 and 57-nt imperfect repeats with 86% identity starting at positions 3 and 28,389, respectively (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4B</a>); their positioning immediately upstream of ORF1a-like and at the border between ORF1b- and 3’ORFs-like regions resembles that of leader and body transcription-regulating sequences (l- and bTRSs) in other nidoviruses (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>). Accordingly, we found a sharp (~3-fold) change in RNA-seq read coverage at the bTRS. We identified an RNA species (by 5’-RACE), whose first two and following 57 nucleotides matched the genomic 5’-terminal AU followed by the bTRS (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4B</a>), suggesting the presence of a subgenomic mRNA produced by discontinuous transcription. Furthermore, the bTRS potentially forms strong complementary interactions with its upstream and downstream sequences (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4C</a>). When unwound, the upstream sequence can form a duplex with the lTRS, bringing together the two spatially separated genomic regions, consistent with a model of discontinuous subgenomic RNA synthesis (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4D</a>) resembling that of other nidoviruses (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>).</p></div></div><div class="section" id="sec-9"><h2 class="">Conclusions</h2><p id="p-27">The advent of metagenomics and transcriptomics have greatly accelerated the pace of virus discovery, leading to studies reporting genome sequences of dozens to thousands of new RNA viruses in poorly characterized hosts (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>). These developments have substantially advanced our appreciation of RNA virus diversity, and improved our understanding of the mechanisms of its generation (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). Notwithstanding that sea change, the largest known RNA genomes continue to belong to nidoviruses, as has been the case for 30 years since the first coronavirus genome of 27 kb was sequenced (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>) (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Fig. 1A</a>). The unbiased, transcriptomics-based discovery of PSCNV in planarians reinforces the status of nidoviruses as relative giants among RNA viruses, while demonstrating that RNA genomes may be substantially larger than previously understood. Further, genomic and proteomic characteristics of PSCNV defy the perceived central role of multiple ORFs in the life cycle and evolution of nidoviruses. Contrary to conventional wisdom, single-ORF genome expression may involve subgenomic mRNA synthesis, while functional constraints linked to the synteny of key replicative enzymes may be the hallmark characteristics of nidoviruses. Since the ORF1b-size barrier can be overcome and genome organization changed, nidoviruses of yet-to-be-sampled hosts might prove to have evolved even larger RNA genomes than that reported here, further decreasing the gap between virus RNA and host DNA genome sizes.</p></div><div class="section" id="sec-10"><h2 class="">Materials and Methods</h2><p id="p-28">All Materials and Methods are described in Supplementary Information in detail.</p><div id="sec-11" class="subsection"><h3>PSCNV genome and its variants in <em>S. mediterranea</em> RNA-seq data</h3><p id="p-29">PSCNV genome was assembled based on contigs identified in two <em>de novo</em> transcriptomes of <em>S. mediterranea</em> (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>) by comparison with genome of human coronavirus OC43 (KY014282.1) using tblastx (BLAST+ v2.2.29 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>)); termini of the assembly were completed by contigs from the two transcriptomes and planarian EST clones (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>), identified by similarity to the incomplete assembly using blastn, and 5’-RACE amplicons. Reads from planarian RNA-seq datasets: in-house ones used to assemble the two transcriptomes, and the ones available from EBI ENA (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>), were mapped to the PSCNV genome sequence by either CLC Genomics Workbench 7, or Bowtie2 version 2.1.0 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>); read counts and coverage were estimated using SAMtools 0.1.19 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>), genome sequence variants were called by BCFtools 1.4 (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>).</p></div><div id="sec-12" class="subsection"><h3>Reverse transcription, PCR, and 5’-RACE</h3><p id="p-30">Freshly prepared RNA from mature sexual planarians was used for cDNA synthesis (iScript, Bio-Rad) or 5’-RACE (RLM-RACE, Ambion) according to manufacturer instructions. Large overlapping amplicons across the PSCNV genome (primers in Table S2) were amplified by standard Phusion<sup>®</sup> High-Fidelity DNA polymerase reaction with 65ºC primer annealing temperature and 10 min extension steps.</p></div><div id="sec-13" class="subsection"><h3>In situ hybridization</h3><p id="p-31">Colorimetric and fluorescent in situ hybridizations were done following published methods (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>). Digoxigenin (DIG)-labelled PSCNV probes were generated by antisense transcription of the planarian EST clone PL06016B2F06 (GenBank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=DN313906.1&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom">DN313906.1</a>) (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>). Following color development, all samples were cleared in 80% (v/v) glycerol and imaged on a Leica M205A microscope (colorimetric) or a Carl Zeiss LSM710 confocal microscope (fluorescent).</p></div><div id="sec-14" class="subsection"><h3>Histology and Transmission Electron Microscopy</h3><p id="p-32">Sexual and asexual planarians originating from the Newmark laboratory were fixed and processed for epoxy (Epon-Araldite) embedding as previously described (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>). For light-microscopic histology, 0.5 µm sections were stained with 1% (w/v) toluidine blue O in 1% (w/v) borax for 30 s at 100°C, and imaged on a Zeiss Axio Observer. For transmission electron microscopy, 50–70 nm sections were collected on copper grids, stained with lead citrate (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>) and imaged with a AMT 1600 M CCD camera on a Hitachi H-7000 STEM at 75 kV.</p></div><div id="sec-15" class="subsection"><h3>Genome and Protein databases</h3><p id="p-33">For various analyses we used the following databases: PlanMine (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>), Smed Unigene (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>), scop70_1.75, pdb70_06Sep14 and pfamA_28.0 supplemented with profiles of conserved nidovirus domains (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>–<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>), Uniprot (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>), list of genome sequences representing nidovirus species (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>), NCBI Viral Genomes Resource (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>), GenBank (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>) and RefSeq (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>).</p></div><div id="sec-16" class="subsection"><h3>Computational protein and RNA sequence analyses</h3><p id="p-34">Virus protein sequences were analyzed to predict disordered regions (DisEMBL 1.5 (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>)), transmembrane regions (TMHMM v.2.0), secondary structure (Jpred4 (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>)), signal peptides (SignalP 4.1 (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>)), N-glycosylation sites (NetNGlyc 1.0) and furin cleavage sites (ProP 1.0 (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>)). To identify sites enriched with amino acid residue, distribution of each residue along polyprotein sequence was assessed using permutation test executed with a custom R script. Generation of multiple sequence alignments of RNA virus proteins was facilitated by Viralis platform (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>). Protein homology profile-based analyses were assisted with HMMER 3.1 (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>), and HH-suite 2.0.16 (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>). To predict RNA secondary structure and PRF sites we used Mfold web server (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>) and KnotInFrame (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>), respectively. Blastn (BLAST+ v2.2.29) (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a>) was used to identify RNA repeats.</p></div><div id="sec-17" class="subsection"><h3>Evolutionary analyses</h3><p id="p-35">Phylogeny was reconstructed by Bayesian approach using a set of tools including BEAST 1.8.2 package (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>) and ProtTest 3.4 (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>) as described in (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>). BayesTraits V2 (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>) was used to perform ancestral state reconstruction. Preference for a state at a node was considered statistically significant only if Log BF exceeded 2 (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>).</p></div><div id="sec-18" class="subsection"><h3>Visualization of results</h3><p id="p-36">Protein alignments were visualized with the help of ESPript 2.1 (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>). To visualize Bayesian samples of trees, DensiTree.v2.2.1 was used (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>). R was used for visualization (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a>).</p></div></div><div class="section" id="sec-19"><h2 class="">Data availability</h2><p id="p-37">Contigs used to assemble PSCNV genome were deposited to GenBank (accession nos. BK010447-BK010449).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We thank Paul Ahlquist for helping initiate this collaboration and Johan den Boon, Andrey M. Leontovich, Dmitry V. Samborskiy, and Igor A. Sidorov for discussions and assistance. This work was supported by NIH R01 HD043403 to P.A.N. and by EU Horizon2020 EVAg 653316 project and LUMC MoBiLe program to A.E.G.; P.A.N. is an investigator of the Howard Hughes Medical Institute and A.E.G. was Leiden University Fund Professor at the time of this study.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.1" data-doi="10.1038/418214a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">GF</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">The antiquity of RNA-based evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>(<span class="cit-issue">6894</span>):<span class="cit-fpage">214</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJoyce%26rft.auinit1%253DG.%2BF.%26rft.volume%253D418%26rft.issue%253D6894%26rft.spage%253D214%26rft.epage%253D221%26rft.atitle%253DThe%2Bantiquity%2Bof%2BRNA-based%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F418214a%26rft_id%253Dinfo%253Apmid%252F12110897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/418214a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12110897&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176710400049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.2" data-doi="10.1093/nar/27.17.3389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leipe</span>  <span class="cit-name-given-names">DD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aravind</span>  <span class="cit-name-given-names">L</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Did DNA replication evolve twice independently?</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">3389</span>–<span class="cit-lpage">3401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.17.3389%26rft_id%253Dinfo%253Apmid%252F10446225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10446225&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082507200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.3" data-doi="10.1093/molbev/msi132"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">AM</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Logan</span>  <span class="cit-name-given-names">DT</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Modern mRNA proofreading and repair: clues that the last universal common ancestor possessed an RNA genome?</span> <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1444</span>–<span class="cit-lpage">1455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi132%26rft_id%253Dinfo%253Apmid%252F15774424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15774424&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229279100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.4" data-doi="10.1128/MMBR.00050-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patil</span>  <span class="cit-name-given-names">KR</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Systems biology perspectives on minimal and simpler cells</span>. <abbr class="cit-jnl-abbrev">Microbiol Mol Biol Rev</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">487</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DXavier%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D78%26rft.issue%253D3%26rft.spage%253D487%26rft.epage%253D509%26rft.atitle%253DSystems%2BBiology%2BPerspectives%2Bon%2BMinimal%2Band%2BSimpler%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00050-13%26rft_id%253Dinfo%253Apmid%252F25184563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3OC8zLzQ4NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.5" data-doi="10.1093/nar/gks1362"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dewey</span>  <span class="cit-name-given-names">CN</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Greaser</span>  <span class="cit-name-given-names">ML</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Rbm20 regulates titin alternative splicing as a splicing repressor</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">2659</span>–<span class="cit-lpage">2672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1362%26rft_id%253Dinfo%253Apmid%252F23307558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23307558&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318062000053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Host-related sequences in RNA viral genomes</span>. <abbr class="cit-jnl-abbrev">Seminars in Virology</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">359</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2BVirology%26rft.volume%253D3%26rft.spage%253D359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-299776v1.7" data-doi="10.1016/j.coviro.2013.06.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">A virocentric perspective on the evolution of life</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">546</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D3%26rft.spage%253D546%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2013.06.008%26rft_id%253Dinfo%253Apmid%252F23850169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2013.06.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23850169&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.8" data-doi="10.1073/pnas.0510333103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forterre</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Three RNA cells for ribosomal lineages and three DNA viruses to replicate their genomes: a hypothesis for the origin of cellular domain</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">103</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">3669</span>–<span class="cit-lpage">3674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0510333103%26rft_id%253Dinfo%253Apmid%252F16505372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzEwLzM2NjkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xMS8yOTk3NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.9" data-doi="10.1016/0378-1119(92)90216-C"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Steinhauer</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">JJ</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Lack of evidence for proofreading mechanisms associated with an RNA virus polymerase</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">122</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">281</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DSteinhauer%26rft.auinit1%253DD.%2BA.%26rft.volume%253D122%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D288%26rft.atitle%253DLack%2Bof%2Bevidence%2Bfor%2Bproofreading%2Bmechanisms%2Bassociated%2Bwith%2Ban%2BRNA%2Bvirus%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252892%252990216-C%26rft_id%253Dinfo%253Apmid%252F1336756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(92)90216-C&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1336756&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KG73200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">den Boon</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">Equine arteritis virus is not a togavirus but belongs to the coronaviruslike superfamily</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">65</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">2910</span>–<span class="cit-lpage">2920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253Dden%2BBoon%26rft.auinit1%253DJ%2BA%26rft.volume%253D65%26rft.issue%253D6%26rft.spage%253D2910%26rft.epage%253D2920%26rft.atitle%253DEquine%2Barteritis%2Bvirus%2Bis%2Bnot%2Ba%2Btogavirus%2Bbut%2Bbelongs%2Bto%2Bthe%2Bcoronaviruslike%2Bsuperfamily.%26rft_id%253Dinfo%253Apmid%252F1851863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY1LzYvMjkxMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.11" data-doi="10.1128/mBio.01484-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stenglein</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Ball python nidovirus: a candidate etiologic agent for severe respiratory disease in Python regius</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">e01484</span>–<span class="cit-lpage">01414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D5%26rft.spage%253De01484%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01484-14%26rft_id%253Dinfo%253Apmid%252F25205093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01484-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25205093&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.12" data-doi="10.1016/j.virusres.2006.01.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Nidovirales: evolving the largest RNA virus genome</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">117</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">17</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D37%26rft.atitle%253DNidovirales%253A%2Bevolving%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.01.017%26rft_id%253Dinfo%253Apmid%252F16503362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16503362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236746000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-299776v1.13" data-doi="10.1073/pnas.0508200103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minskaia</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Discovery of an RNA virus 3’-&gt;5’ exoribonuclease that is critically involved in coronavirus RNA synthesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">103</span>(<span class="cit-issue">13</span>):<span class="cit-fpage">5108</span>–<span class="cit-lpage">5113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508200103%26rft_id%253Dinfo%253Apmid%252F16549795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzEzLzUxMDgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xMS8yOTk3NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-299776v1.14" data-doi="10.4161/rna.8.2.15013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">LD</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Coronaviruses: an RNA proofreading machine regulates replication fidelity and diversity</span>. <abbr class="cit-jnl-abbrev">RNA Biol</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">270</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRNA%2BBiol%26rft.aulast%253DDenison%26rft.auinit1%253DM.%2BR.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D270%26rft.epage%253D279%26rft.atitle%253DCoronaviruses%253A%2Ban%2BRNA%2Bproofreading%2Bmachine%2Bregulates%2Breplication%2Bfidelity%2Band%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.8.2.15013%26rft_id%253Dinfo%253Apmid%252F21593585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.8.2.15013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21593585&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289934300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.15" data-doi="10.1371/journal.ppat.1002215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nga</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Discovery of the first insect nidovirus, a missing evolutionary link in the emergence of the largest RNA virus genomes</span>. <abbr class="cit-jnl-abbrev">PLoS. Pathog</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">e1002215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DNga%26rft.auinit1%253DP.%2BT.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De1002215%26rft.epage%253De1002215%26rft.atitle%253DDiscovery%2Bof%2Bthe%2Bfirst%2Binsect%2Bnidovirus%252C%2Ba%2Bmissing%2Bevolutionary%2Blink%2Bin%2Bthe%2Bemergence%2Bof%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002215%26rft_id%253Dinfo%253Apmid%252F21931546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21931546&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.16" data-doi="10.1073/pnas.1508686112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr> <span class="cit-vol">112</span>(<span class="cit-issue">30</span>):<span class="cit-fpage">9436</span>–<span class="cit-lpage">9441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1508686112%26rft_id%253Dinfo%253Apmid%252F26159422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzMwLzk0MzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xMS8yOTk3NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.17" data-doi="10.1073/pnas.1718806115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferron</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Structural and molecular basis of mismatch correction and ribavirin excision from coronavirus RNA</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">115</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">E162</span>–<span class="cit-lpage">E171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1718806115%26rft_id%253Dinfo%253Apmid%252F29279395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE1LzIvRTE2MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.18" data-doi="10.1038/nature20167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Redefining the invertebrate RNA virosphere</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">540</span>:<span class="cit-fpage">539</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D540%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature20167%26rft_id%253Dinfo%253Apmid%252F27880757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature20167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27880757&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">The evolutionary history of vertebrate RNA viruses</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, in press.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.20" data-doi="10.1126/science.1101485"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raoult</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">The 1.2-megabase genome sequence of Mimivirus</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">306</span>(<span class="cit-issue">5700</span>):<span class="cit-fpage">1344</span>–<span class="cit-lpage">1350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1101485%26rft_id%253Dinfo%253Apmid%252F15486256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDYvNTcwMC8xMzQ0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTEvMjk5Nzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-299776v1.21" data-doi="10.1126/science.1239181"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Pandoraviruses: amoeba viruses with genomes up to 2.5 Mb reaching that of parasitic eukaryotes</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>(<span class="cit-issue">6143</span>):<span class="cit-fpage">281</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPhilippe%26rft.auinit1%253DN.%26rft.volume%253D341%26rft.issue%253D6143%26rft.spage%253D281%26rft.epage%253D286%26rft.atitle%253DPandoraviruses%253A%2BAmoeba%2BViruses%2Bwith%2BGenomes%2BUp%2Bto%2B2.5%2BMb%2BReaching%2BThat%2Bof%2BParasitic%2BEukaryotes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1239181%26rft_id%253Dinfo%253Apmid%252F23869018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0My8yODEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xMS8yOTk3NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.22" data-doi="10.1126/science.aal4657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Giant viruses with an expanded complement of translation system components</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">356</span>(<span class="cit-issue">6333</span>):<span class="cit-fpage">82</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchulz%26rft.auinit1%253DF.%26rft.volume%253D356%26rft.issue%253D6333%26rft.spage%253D82%26rft.epage%253D85%26rft.atitle%253DGiant%2Bviruses%2Bwith%2Ban%2Bexpanded%2Bcomplement%2Bof%2Btranslation%2Bsystem%2Bcomponents%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aal4657%26rft_id%253Dinfo%253Apmid%252F28386012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTYvNjMzMy84MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.23" data-doi="10.1371/journal.ppat.1003500"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauber</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">The footprint of genome architecture in the largest genome expansion in RNA viruses</span>. <abbr class="cit-jnl-abbrev">PLoS. Pathog</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e1003500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS.%2BPathog%26rft.volume%253D9%26rft.spage%253De1003500%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003500%26rft_id%253Dinfo%253Apmid%252F23874204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874204&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.24" data-doi="10/1371/journal.pbio.1002457"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saberi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jamal</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beets</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoofs</span>  <span class="cit-name-given-names">L</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Newmark</span>  <span class="cit-name-given-names">PA</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">GPCRs Direct Germline Development and Somatic Gonad Function in Planarians</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">e1002457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D14%26rft.spage%253De1002457%26rft_id%253Dinfo%253Adoi%252F10%252F1371%252Fjournal.pbio.1002457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10/1371/journal.pbio.1002457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.25" data-doi="10.1038/nrg759"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Newmark</span>  <span class="cit-name-given-names">PA</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">Alvarado A</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Not your father’s planarian: a classic model enters the era of functional genomics</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">210</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNewmark%26rft.auinit1%253DP.%2BA.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D210%26rft.epage%253D219%26rft.atitle%253DNot%2Byour%2Bfather%2527s%2Bplanarian%253A%2Ba%2Bclassic%2Bmodel%2Benters%2Bthe%2Bera%2Bof%2Bfunctional%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg759%26rft_id%253Dinfo%253Apmid%252F11972158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11972158&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174159100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.26" data-doi="10.1073/pnas.0509507102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zayas</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">The planarian Schmidtea mediterranea as a model for epigenetic germ cell specification: analysis of ESTs from the hermaphroditic strain</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">102</span>(<span class="cit-issue">51</span>):<span class="cit-fpage">18491</span>–<span class="cit-lpage">18496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0509507102%26rft_id%253Dinfo%253Apmid%252F16344473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzUxLzE4NDkxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTEvMjk5Nzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grohme</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">The genome of Schmidtea mediterranea and the evolution of core cellular mechanisms</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">554</span>(<span class="cit-issue">7690</span>):<span class="cit-fpage">56</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D554%26rft.spage%253D56%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.28" data-doi="10.1093/nar/gkv1148"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brandl</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">PlanMine--a mineable resource of planarian biology and biodiversity</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D764</span>–<span class="cit-lpage">773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1148%26rft_id%253Dinfo%253Apmid%252F26578570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26578570&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.29" data-doi="10.1093/nar/gkv838"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">KC</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Discovery of an essential nucleotidylating activity associated with a newly delineated conserved domain in the RNA polymerase-containing protein of all nidoviruses</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">8416</span>–<span class="cit-lpage">8434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv838%26rft_id%253Dinfo%253Apmid%252F26304538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26304538&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.30" data-doi="10.1093/nar/gkt1310"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Structural basis for the regulatory function of a complex zinc-binding domain in a replicative arterivirus helicase resembling a nonsense-mediated mRNA decay helicase</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">3464</span>–<span class="cit-lpage">3477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1310%26rft_id%253Dinfo%253Apmid%252F24369429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24369429&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.31" data-doi="10.1099/0022-1317-81-4-853"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Virus-encoded proteinases and proteolytic processing in the Nidovirales</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">Pt 4</span>):<span class="cit-fpage">853</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DZiebuhr%26rft.auinit1%253DJ.%26rft.volume%253D81%26rft.issue%253D4%26rft.spage%253D853%26rft.epage%253D879%26rft.atitle%253DVirus-encoded%2Bproteinases%2Band%2Bproteolytic%2Bprocessing%2Bin%2Bthe%2BNidovirales%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-81-4-853%26rft_id%253Dinfo%253Apmid%252F10725411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-81-4-853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10725411&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086341400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-299776v1.32" data-doi="10.1128/JVI.77.2.1415-1426.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowley</span>  <span class="cit-name-given-names">JA</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">The 3C-like proteinase of an invertebrate nidovirus links coronavirus and potyvirus homologs</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr> <span class="cit-vol">77</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">1415</span>–<span class="cit-lpage">1426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DZiebuhr%26rft.auinit1%253DJ.%26rft.volume%253D77%26rft.issue%253D2%26rft.spage%253D1415%26rft.epage%253D1426%26rft.atitle%253DThe%2B3C-Like%2BProteinase%2Bof%2Ban%2BInvertebrate%2BNidovirus%2BLinks%2BCoronavirus%2Band%2BPotyvirus%2BHomologs%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.2.1415-1426.2003%26rft_id%253Dinfo%253Apmid%252F12502857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc3LzIvMTQxNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-299776v1.33" data-doi="10.1128/JVI.80.8.4157-4167.2006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smits</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Characterization of a torovirus main proteinase</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr> <span class="cit-vol">80</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">4157</span>–<span class="cit-lpage">4167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSmits%26rft.auinit1%253DS.%2BL.%26rft.volume%253D80%26rft.issue%253D8%26rft.spage%253D4157%26rft.epage%253D4167%26rft.atitle%253DCharacterization%2Bof%2Ba%2BTorovirus%2BMain%2BProteinase%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.80.8.4157-4167.2006%26rft_id%253Dinfo%253Apmid%252F16571831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgwLzgvNDE1NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-299776v1.34" data-doi="10.1128/JVI.01716-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ulferts</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mettenleiter</span>  <span class="cit-name-given-names">TC</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Characterization of Bafinivirus main protease autoprocessing activities</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr> <span class="cit-vol">85</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">1348</span>–<span class="cit-lpage">1359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01716-10%26rft_id%253Dinfo%253Apmid%252F21068254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg1LzMvMTM0OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.35" data-doi="10.1128/JVI.02040-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blanck</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Characterization of an alphamesonivirus 3C-like protease defines a special group of nidovirus main proteases</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr> <span class="cit-vol">88</span>(<span class="cit-issue">23</span>):<span class="cit-fpage">13747</span>–<span class="cit-lpage">13758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02040-14%26rft_id%253Dinfo%253Apmid%252F25231310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8yMy8xMzc0NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.36" data-doi="10.1038/nrmicro2147"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Netland</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Coronaviruses post-SARS: update on replication and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">439</span>–<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DPerlman%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D439%26rft.epage%253D450%26rft.atitle%253DCoronaviruses%2Bpost-SARS%253A%2Bupdate%2Bon%2Breplication%2Band%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2147%26rft_id%253Dinfo%253Apmid%252F19430490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19430490&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.37" data-doi="10.1128/EC.00120-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rauceo</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Threonine-rich repeats increase fibronectin binding in the Candida albicans adhesin Als5p</span>. <abbr class="cit-jnl-abbrev">Eukaryot Cell</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">1664</span>–<span class="cit-lpage">1673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00120-06%26rft_id%253Dinfo%253Apmid%252F16936142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI1LzEwLzE2NjQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xMS8yOTk3NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.38" data-doi="10.1128/AEM.01657-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hevia</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">An extracellular Serine/Threonine-rich protein from Lactobacillus plantarum NCIMB 8826 is a novel aggregation-promoting factor with affinity to mucin</span>. <abbr class="cit-jnl-abbrev">Appl Environ Microbiol</abbr> <span class="cit-vol">79</span>(<span class="cit-issue">19</span>):<span class="cit-fpage">6059</span>–<span class="cit-lpage">6066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl%2BEnviron%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01657-13%26rft_id%253Dinfo%253Apmid%252F23892754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3OS8xOS82MDU5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTEvMjk5Nzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.39" data-doi="10.1128/JVI.78.5.2510-2516.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Camus-Bouclainville</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">A virulence factor of myxoma virus colocalizes with NF-kappaB in the nucleus and interferes with inflammation</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">2510</span>–<span class="cit-lpage">2516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DCamus-Bouclainville%26rft.auinit1%253DC.%26rft.volume%253D78%26rft.issue%253D5%26rft.spage%253D2510%26rft.epage%253D2516%26rft.atitle%253DA%2BVirulence%2BFactor%2Bof%2BMyxoma%2BVirus%2BColocalizes%2Bwith%2BNF-%257Bkappa%257DB%2Bin%2Bthe%2BNucleus%2Band%2BInterferes%2Bwith%2BInflammation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.5.2510-2516.2004%26rft_id%253Dinfo%253Apmid%252F14963153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc4LzUvMjUxMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzExLzI5OTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.40" data-doi="10.1016/j.str.2012.03.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krey</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Crystal structure of the pestivirus envelope glycoprotein E(rns) and mechanistic analysis of its ribonuclease activity</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">862</span>–<span class="cit-lpage">873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DKrey%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D862%26rft.epage%253D873%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bpestivirus%2Benvelope%2Bglycoprotein%2BE%2528rns%2529%2Band%2Bmechanistic%2Banalysis%2Bof%2Bits%2Bribonuclease%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2012.03.018%26rft_id%253Dinfo%253Apmid%252F22579253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2012.03.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22579253&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park B</span>  <span class="cit-name-given-names">KY</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Immunosuppression induced by expression of a viral RNase enhances susceptibility of Plutella xylostella to microbial pesticides</span>. <abbr class="cit-jnl-abbrev">Insect Science</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">47</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2BScience%26rft.volume%253D19%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.42" data-doi="10.1002/dvdy.22427"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zayas</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cebria</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">J</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Newmark</span>  <span class="cit-name-given-names">PA</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">The use of lectins as markers for differentiated secretory cells in planarians</span>. <abbr class="cit-jnl-abbrev">Dev Dyn</abbr> <span class="cit-vol">239</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">2888</span>–<span class="cit-lpage">2897</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bdynamics%2B%253A%2B%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BAssociation%2Bof%2BAnatomists%26rft.stitle%253DDev%2BDyn%26rft.aulast%253DZayas%26rft.auinit1%253DR.%2BM.%26rft.volume%253D239%26rft.issue%253D11%26rft.spage%253D2888%26rft.epage%253D2897%26rft.atitle%253DThe%2Buse%2Bof%2Blectins%2Bas%2Bmarkers%2Bfor%2Bdifferentiated%2Bsecretory%2Bcells%2Bin%2Bplanarians.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvdy.22427%26rft_id%253Dinfo%253Apmid%252F20865784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvdy.22427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20865784&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.43" data-doi="10.1016/j.virol.2007.05.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ortego</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ceriani</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patino</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plana</span>  <span class="cit-name-given-names">J</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Absence of E protein arrests transmissible gastroenteritis coronavirus maturation in the secretory pathway</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">368</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">296</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DOrtego%26rft.auinit1%253DJ.%26rft.volume%253D368%26rft.issue%253D2%26rft.spage%253D296%26rft.epage%253D308%26rft.atitle%253DAbsence%2Bof%2BE%2Bprotein%2Barrests%2Btransmissible%2Bgastroenteritis%2Bcoronavirus%2Bmaturation%2Bin%2Bthe%2Bsecretory%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2007.05.032%26rft_id%253Dinfo%253Apmid%252F17692883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2007.05.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17692883&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thuy</span>  <span class="cit-name-given-names">NT</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">A new nidovirus (NamDinh virus NDiV): Its ultrastructural characterization in the C6/36 mosquito cell line</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">444</span>(<span class="cit-issue">1-2</span>):<span class="cit-fpage">337</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D444%26rft.spage%253D337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.45" data-doi="10.1371/journal.pbio.0060226"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Knoops</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">SARS-coronavirus replication is supported by a reticulovesicular network of modified endoplasmic reticulum</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">e226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DKnoops%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De226%26rft.epage%253De226%26rft.atitle%253DSARS-coronavirus%2Breplication%2Bis%2Bsupported%2Bby%2Ba%2Breticulovesicular%2Bnetwork%2Bof%2Bmodified%2Bendoplasmic%2Breticulum.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060226%26rft_id%253Dinfo%253Apmid%252F18798692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18798692&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.46" data-doi="10.1099/vir.0.042499-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>  <span class="cit-name-given-names">AE</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brierley</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Non-canonical translation in RNA viruses</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">93</span>(<span class="cit-issue">Pt 7</span>):<span class="cit-fpage">1385</span>–<span class="cit-lpage">1409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.042499-0%26rft_id%253Dinfo%253Apmid%252F22535777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.042499-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22535777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306348900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.47" data-doi="10.1038/ncomms15582"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Napthine</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Protein-directed ribosomal frameshifting temporally regulates gene expression</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">15582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D15582%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15582%26rft_id%253Dinfo%253Apmid%252F28593994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28593994&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.48" data-doi="10.1038/nrm934"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Furin at the cutting edge: from protein traffic to embryogenesis and disease</span>. <abbr class="cit-jnl-abbrev">Nat Rev Mol Cell Biol</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">753</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DThomas%26rft.auinit1%253DG.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253D753%26rft.epage%253D766%26rft.atitle%253DFurin%2Bat%2Bthe%2Bcutting%2Bedge%253A%2Bfrom%2Bprotein%2Btraffic%2Bto%2Bembryogenesis%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm934%26rft_id%253Dinfo%253Apmid%252F12360192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12360192&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178319800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.49" data-doi="10.1099/vir.0.81611-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pasternak</span>  <span class="cit-name-given-names">AO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJ</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Nidovirus transcription: how to make sense…?</span> <abbr class="cit-jnl-abbrev">J. Gen. Virol</abbr> <span class="cit-vol">87</span>(<span class="cit-issue">Pt 6</span>):<span class="cit-fpage">1403</span>–<span class="cit-lpage">1421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DPasternak%26rft.auinit1%253DA.%2BO.%26rft.volume%253D87%26rft.issue%253D6%26rft.spage%253D1403%26rft.epage%253D1421%26rft.atitle%253DNidovirus%2Btranscription%253A%2Bhow%2Bto%2Bmake%2Bsense...%253F%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.81611-0%26rft_id%253Dinfo%253Apmid%252F16690906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.81611-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16690906&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238167400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.50" data-doi="10.1146/annurev.micro.60.080805.142157"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almazan</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sola</span>  <span class="cit-name-given-names">I</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Zuniga</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Biochemical aspects of coronavirus replication and virus-host interaction</span>. <abbr class="cit-jnl-abbrev">Annu Rev Microbiol</abbr> <span class="cit-vol">60</span>:<span class="cit-fpage">211</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DEnjuanes%26rft.auinit1%253DL.%26rft.volume%253D60%26rft.spage%253D211%26rft.epage%253D230%26rft.atitle%253DBiochemical%2Baspects%2Bof%2Bcoronavirus%2Breplication%2Band%2Bvirus-host%2Binteraction.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.60.080805.142157%26rft_id%253Dinfo%253Apmid%252F16712436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.60.080805.142157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16712436&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242032500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.51" data-doi="10.1128/JVI.00357-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marzano</span>  <span class="cit-name-given-names">SY</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Identification of Diverse Mycoviruses through Metatranscriptomics Characterization of the Viromes of Five Major Fungal Plant Pathogens</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">6846</span>–<span class="cit-lpage">6863</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00357-16%26rft_id%253Dinfo%253Apmid%252F27194764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNS82ODQ2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTEvMjk5Nzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fauver</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">West African Anopheles gambiae mosquitoes harbor a taxonomically diverse virome including new insect-specific flaviviruses, mononegaviruses, and totiviruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">498</span>:<span class="cit-fpage">288</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D498%26rft.spage%253D288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.53" data-doi="10.1016/j.virusres.2017.10.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolja</span>  <span class="cit-name-given-names">VV</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Metagenomics reshapes the concepts of RNA virus evolution by revealing extensive horizontal virus transfer</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">244</span>:<span class="cit-fpage">36</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D244%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.10.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.10.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.54" data-doi="10.1016/j.cell.2018.02.043"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">YZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Using Metagenomics to Characterize an Expanding Virosphere</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">172</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1168</span>–<span class="cit-lpage">1172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D172%26rft.spage%253D1168%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.02.043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.02.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.55" data-doi="10.1099/0022-1317-68-1-57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boursnell</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Completion of the sequence of the genome of the coronavirus avian infectious bronchitis virus</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">68</span> (<span class="cit-issue">Pt 1</span>):<span class="cit-fpage">57</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DBoursnell%26rft.auinit1%253DM.%2BE.%2BG.%26rft.volume%253D68%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D77%26rft.atitle%253DCompletion%2Bof%2Bthe%2BSequence%2Bof%2Bthe%2BGenome%2Bof%2Bthe%2BCoronavirus%2BAvian%2BInfectious%2BBronchitis%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-68-1-57%26rft_id%253Dinfo%253Apmid%252F3027249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-68-1-57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3027249&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987F681400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.56" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">EW</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">DJ</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">215</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silvester</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">The European Nucleotide Archive in 2017</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.58" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.59" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">16</span>):<span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.60" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">21</span>):<span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.61" data-doi="10.1186/1471-213X-13-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">RS</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Newmark</span>  <span class="cit-name-given-names">PA</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">In situ hybridization protocol for enhanced detection of gene expression in the planarian Schmidtea mediterranea</span>. <abbr class="cit-jnl-abbrev">BMC Dev Biol</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BDev%2BBiol%26rft.volume%253D13%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-213X-13-8%26rft_id%253Dinfo%253Apmid%252F23497040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-213X-13-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23497040&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.62" data-doi="10.1038/nprot.2014.041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brubacher</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">AP</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Newmark</span>  <span class="cit-name-given-names">PA</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Preparation of the planarian Schmidtea mediterranea for high-resolution histology and transmission electron microscopy</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">661</span>–<span class="cit-lpage">673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D9%26rft.spage%253D661%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2014.041%26rft_id%253Dinfo%253Apmid%252F24556788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2014.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24556788&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.63" data-doi="10.1083/jcb.25.2.407"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Venable</span>  <span class="cit-name-given-names">JH</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Coggeshall</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">1965</span>) <span class="cit-article-title">A Simplified Lead Citrate Stain for Use in Electron Microscopy</span>. <abbr class="cit-jnl-abbrev">J Cell Biol</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">407</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DVenable%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D25%26rft.issue%253D2%26rft.spage%253D407%26rft.epage%253D408%26rft.atitle%253DA%2BSIMPLIFIED%2BLEAD%2BCITRATE%2BSTAIN%2BFOR%2BUSE%2BIN%2BELECTRON%2BMICROSCOPY%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.25.2.407%26rft_id%253Dinfo%253Apmid%252F14287192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJqY2IiO3M6NToicmVzaWQiO3M6ODoiMjUvMi80MDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xMS8yOTk3NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.64" data-doi="10.1002/dvg.22872"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robb</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gotting</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">E</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">AA</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">SmedGD 2.0: The Schmidtea mediterranea genome database</span>. <abbr class="cit-jnl-abbrev">Genesis</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">535</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenesis%26rft.volume%253D53%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.22872%26rft_id%253Dinfo%253Apmid%252F26138588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.22872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26138588&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.65" data-doi="10.1093/nar/gkm993"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andreeva</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Data growth and its impact on the SCOP database: new developments</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">36</span>(<span class="cit-issue">Database issue</span>):<span class="cit-fpage">D419</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm993%26rft_id%253Dinfo%253Apmid%252F18000004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm993&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18000004&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252545400076&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">66.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-299776v1.66" data-doi="10.1093/nar/28.1.235"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">The Protein Data Bank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBerman%26rft.auinit1%253DH.%2BM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DThe%2BProtein%2BData%2BBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.235%26rft_id%253Dinfo%253Apmid%252F10592235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.67" data-doi="10.1093/nar/gkt1223"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Pfam: the protein families database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">Database issue</span>):<span class="cit-fpage">D222</span>–<span class="cit-lpage">D230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1223%26rft_id%253Dinfo%253Apmid%252F24288371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24288371&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.68" data-doi="10.1093/nar/gku989"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">UniProt Consortium</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">UniProt: a hub for protein information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>):<span class="cit-fpage">D204</span>–<span class="cit-lpage">D212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku989%26rft_id%253Dinfo%253Apmid%252F25348405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25348405&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.69" data-doi="10.1007/s00705-016-2977-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Ratification vote on taxonomic proposals to the International Committee on Taxonomy of Viruses (2016)</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">161</span>:<span class="cit-fpage">2921</span>–<span class="cit-lpage">2949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D161%26rft.spage%253D2921%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-016-2977-6%26rft_id%253Dinfo%253Apmid%252F27424026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-016-2977-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27424026&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.70" data-doi="10.1093/nar/gku1207"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brister</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ako-Adjei</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Y</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Blinkova</span>  <span class="cit-name-given-names">O</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">NCBI viral genomes resource</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>):<span class="cit-fpage">D571</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1207%26rft_id%253Dinfo%253Apmid%252F25428358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25428358&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.71" data-doi="10.1093/nar/gkw1070"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">GenBank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D37</span>–<span class="cit-lpage">D42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1070%26rft_id%253Dinfo%253Apmid%252F27899564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899564&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.72" data-doi="10.1093/nar/gkv1189"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D733</span>–<span class="cit-lpage">745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1189%26rft_id%253Dinfo%253Apmid%252F26553804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26553804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.73" data-doi="10.1016/j.str.2003.10.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Linding</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Protein disorder prediction: implications for structural proteomics</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">1453</span>–<span class="cit-lpage">1459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DLinding%26rft.auinit1%253DR.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1453%26rft.epage%253D1459%26rft.atitle%253DProtein%2Bdisorder%2Bprediction%253A%2Bimplications%2Bfor%2Bstructural%2Bproteomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2003.10.002%26rft_id%253Dinfo%253Apmid%252F14604535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2003.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14604535&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.74" data-doi="10.1093/nar/gkv332"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drozdetskiy</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Procter</span>  <span class="cit-name-given-names">J</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">GJ</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">JPred4: a protein secondary structure prediction server</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">W1</span>):<span class="cit-fpage">W389</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv332%26rft_id%253Dinfo%253Apmid%252F25883141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv332&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25883141&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.75" data-doi="10.1038/nmeth.1701"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">TN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">von</span>  <span class="cit-name-given-names">HG</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">785</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPetersen%26rft.auinit1%253DT.%2BN.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D785%26rft.epage%253D786%26rft.atitle%253DSignalP%2B4.0%253A%2Bdiscriminating%2Bsignal%2Bpeptides%2Bfrom%2Btransmembrane%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1701%26rft_id%253Dinfo%253Apmid%252F21959131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21959131&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.76" data-doi="10.1093/protein/gzh013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duckert</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Blom</span>  <span class="cit-name-given-names">N</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Prediction of proprotein convertase cleavage sites</span>. <abbr class="cit-jnl-abbrev">Protein Eng Des Sel</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">107</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%2BDesign%2Band%2BSelection%26rft.stitle%253DProtein%2BEng%2BDes%2BSel%26rft.aulast%253DDuckert%26rft.auinit1%253DP.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D112%26rft.atitle%253DPrediction%2Bof%2Bproprotein%2Bconvertase%2Bcleavage%2Bsites%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252Fgzh013%26rft_id%253Dinfo%253Apmid%252F14985543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/protein/gzh013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14985543&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220905100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.77" data-doi="10.1016/j.antiviral.2010.02.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Practical application of bioinformatics by the multidisciplinary VIZIER consortium</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr> <span class="cit-vol">87</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">95</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D87%26rft.issue%253D2%26rft.spage%253D95%26rft.epage%253D110%26rft.atitle%253DPractical%2Bapplication%2Bof%2Bbioinformatics%2Bby%2Bthe%2Bmultidisciplinary%2BVIZIER%2Bconsortium.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2010.02.005%26rft_id%253Dinfo%253Apmid%252F20153379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2010.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20153379&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.78" data-doi="10.1142/9781848165632_0019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">SR</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">A new generation of homology search tools based on probabilistic inference</span>. <abbr class="cit-jnl-abbrev">Genome Inform</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">205</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BInform%26rft.volume%253D23%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1142%252F9781848165632_0019%26rft_id%253Dinfo%253Apmid%252F20180275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/9781848165632_0019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20180275&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.79" data-doi="10.1093/bioinformatics/bti125"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soding</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Protein homology detection by HMM-HMM comparison</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">951</span>–<span class="cit-lpage">960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti125%26rft_id%253Dinfo%253Apmid%252F15531603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15531603&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227977800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.80" data-doi="10.1093/nar/gkg595"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zuker</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Mfold web server for nucleic acid folding and hybridization prediction</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">13</span>):<span class="cit-fpage">3406</span>–<span class="cit-lpage">3415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZuker%26rft.auinit1%253DM.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3406%26rft.epage%253D3415%26rft.atitle%253DMfold%2Bweb%2Bserver%2Bfor%2Bnucleic%2Bacid%2Bfolding%2Band%2Bhybridization%2Bprediction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg595%26rft_id%253Dinfo%253Apmid%252F12824337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824337&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.81" data-doi="10.1093/bioinformatics/btu649"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">S</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Giegerich</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">The RNA shapes studio</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">423</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu649%26rft_id%253Dinfo%253Apmid%252F25273103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25273103&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.82" data-doi="10.1093/molbev/mss075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1969</span>–<span class="cit-lpage">1973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss075%26rft_id%253Dinfo%253Apmid%252F22367748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22367748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307171300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.83" data-doi="10.1093/bioinformatics/btr088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">GL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">ProtTest 3: fast selection of best-fit models of protein evolution</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1164</span>–<span class="cit-lpage">1165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr088%26rft_id%253Dinfo%253Apmid%252F21335321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21335321&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289301600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.84"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gulyaeva</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Domain organization and evolution of the highly divergent 5’ coding region of genomes of arteriviruses including the novel possum nidovirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">6</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.85" data-doi="10.1080/10635150490522232"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pagel</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meade</span>  <span class="cit-name-given-names">A</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Bayesian estimation of ancestral character states on phylogenies</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">673</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150490522232%26rft_id%253Dinfo%253Apmid%252F15545248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150490522232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15545248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224960100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.86" data-doi="10.2307/2291091"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">RE</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">AE</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">Bayes Factors</span>. <abbr class="cit-jnl-abbrev">J Am Stat Assoc</abbr> <span class="cit-vol">90</span>(<span class="cit-issue">430</span>):<span class="cit-fpage">773</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAm%2BStat%2BAssoc%26rft.volume%253D90%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291091%26rft_id%253Dinfo%253Apmid%252F10783804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2291091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10783804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RA10400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.87" data-doi="10.1093/nar/gkg556"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gouet</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">X</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Courcelle</span>  <span class="cit-name-given-names">E</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">ESPript/ENDscript: Extracting and rendering sequence and 3D information from atomic structures of proteins</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">13</span>):<span class="cit-fpage">3320</span>–<span class="cit-lpage">3323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGouet%26rft.auinit1%253DP.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3320%26rft.epage%253D3323%26rft.atitle%253DESPript%252FENDscript%253A%2Bextracting%2Band%2Brendering%2Bsequence%2Band%2B3D%2Binformation%2Bfrom%2Batomic%2Bstructures%2Bof%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg556%26rft_id%253Dinfo%253Apmid%252F12824317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824317&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-299776v1.88" data-doi="10.1186/1471-2148-13-221"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">RR</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Looking for trees in the forest: summary tree from posterior samples</span>. <abbr class="cit-jnl-abbrev">BMC. Evol. Biol</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC.%2BEvol.%2BBiol%26rft.volume%253D13%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-221%26rft_id%253Dinfo%253Apmid%252F24093883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24093883&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F11%2F299776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-book" id="cit-299776v1.89"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span> (<span class="cit-pub-date">2013</span>) <span class="cit-source">R: A Language and Environment for Statistical Computing</span> (<span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/299776v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F299776v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/299875v1" title="TraceSpecks: a software for automated idealization of noisy patch-clamp and imaging data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 11, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/299776v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/299776v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/299776v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/299776v1" class="active" data-icon-position="" data-hide-link-title="0">A planarian nidovirus expands the limits of RNA genome size</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/92053" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iRN5tFqrVK0aYZcu3PDJE19tTJeGQdZAe4i-swFzIQo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="92053" id="node92053--2" data-pisa="biorxiv;299776v1" data-pisa-master="biorxiv;299776" data-seqnum="92053" data-apath="/biorxiv/early/2018/04/11/299776.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A planarian nidovirus expands the limits of RNA genome size</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Amir</span> <span class="nlm-surname">Saberi</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anastasia A.</span> <span class="nlm-surname">Gulyaeva</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">John L.</span> <span class="nlm-surname">Brubacher</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Phillip A.</span> <span class="nlm-surname">Newmark</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Alexander E.</span> <span class="nlm-surname">Gorbalenya</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">299776; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/299776 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/299776v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;title=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;title=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;text=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;title=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;t=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;title=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;title=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="92053" id="node92053--3" data-pisa="biorxiv;299776v1" data-pisa-master="biorxiv;299776" data-seqnum="92053" data-apath="/biorxiv/early/2018/04/11/299776.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A planarian nidovirus expands the limits of RNA genome size</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Amir</span> <span class="nlm-surname">Saberi</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anastasia A.</span> <span class="nlm-surname">Gulyaeva</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">John L.</span> <span class="nlm-surname">Brubacher</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Phillip A.</span> <span class="nlm-surname">Newmark</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Alexander E.</span> <span class="nlm-surname">Gorbalenya</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">299776; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/299776 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Saberi%2C+Amir&amp;rft.au=Gulyaeva%2C+Anastasia+A.&amp;rft.au=Brubacher%2C+John+L.&amp;rft.au=Newmark%2C+Phillip+A.&amp;rft.au=Gorbalenya%2C+Alexander+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/92053/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/92053/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/92053/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/92053/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/92053/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/92053/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/92053/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/92053/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/92053/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/92053/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/92053/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/92053/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;count=horizontal&amp;via=&amp;text=A%20planarian%20nidovirus%20expands%20the%20limits%20of%20RNA%20genome%20size&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/299776v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/299776v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/299776v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
