<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2014/01/10/001727.atom" />
<meta name="HW.pisa" content="biorxiv;001727v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="SIANN: Strain Identification by Alignment to Near Neighbors" />
<meta name="DC.Identifier" content="10.1101/001727" />
<meta name="DC.Date" content="2014-01-10" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2014, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 3.0 Unported), CC BY-NC-ND 3.0, as described at http://creativecommons.org/licenses/by-nc-nd/3.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results." />
<meta name="DC.Contributor" content="Samuel S. Minot" />
<meta name="DC.Contributor" content="Stephen D. Turner" />
<meta name="DC.Contributor" content="Krista L. Ternus" />
<meta name="DC.Contributor" content="Dana R. Kadavy" />
<meta name="article:published_time" content="2014-01-10" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="SIANN: Strain Identification by Alignment to Near Neighbors" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2014/01/01" />
<meta name="citation_mjid" content="biorxiv;001727v1" />
<meta name="citation_id" content="001727v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/001727v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/001727v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/001727v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727.full.pdf" />
<meta name="citation_doi" content="10.1101/001727" />
<meta name="citation_num_pages" content="47" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="001727" />
<meta name="citation_author" content="Samuel S. Minot" />
<meta name="citation_author" content="Stephen D. Turner" />
<meta name="citation_author" content="Krista L. Ternus" />
<meta name="citation_author" content="Dana R. Kadavy" />
<meta name="citation_reference" content="Ames SK, Hysom DA, Gardner SN, Lloyd GS, Gokhale MB, and Allen JE (2013). Scalable metagenomic taxonomy classification using a reference genome database. Bioinformatics, 29(18):2253–60. doi: 10.1093/bioinformatics/btt389." />
<meta name="citation_reference" content="Bazinet AL &amp; Cummings MP (2012). A comparative evaluation of sequence classification programs. BMC Bioinformatics. 13: 92. doi: 10.1186/1471-2105-13-92" />
<meta name="citation_reference" content="Berendzen J, Bruno WJ, Cohn JD, Hengartner NW, Kuske CR, McMahon BH, Wolinsky MA and Xie G (2012). Rapid phylogenetic and functional classification of short genomic fragments with signature peptides. BMC Research Notes, 5:460 doi:10.1186/1756-0500-5-460" />
<meta name="citation_reference" content="Brady A and Salzberg SL (2009). Phymm and PhymmBL: metagenomic phylogenetic classification with interpolated Markov models. Nature Methods, 6(9):673–6. doi: 10.1038/nmeth.1358." />
<meta name="citation_reference" content="Huson DH, Mitra S, Weber N, Ruscheweyh H, and Schuster SC (2011). Integrative analysis of environmental sequences using MEGAN4. Genome Research, 21:1552–1560." />
<meta name="citation_reference" content="Korbel JO, Abyzov A, Mu XJ, Carriero N, Cayting P, Zhang Z, Snyder M and Gerstein MB (2009). PEMer: a computational framework with simulation-based error models for inferring genomic structural variants from massive paired-end sequencing data. Genome Biology, 10:R23 doi:10.1186/gb-2009-10-2-r23" />
<meta name="citation_reference" content="Langmead B, Salzberg S. Fast gapped-read alignment with Bowtie 2. Nature Methods. 2012, 9:357–359." />
<meta name="citation_reference" content="Liu B, Gibbons T, Ghodsi M, Treangen T, and Pop M (2011). Accurate and fast estimation of taxonomic profiles from metagenomic shotgun sequences. BMC Genomics, 12(Suppl 2):S4" />
<meta name="citation_reference" content="Richter DC, Ott F, Auch AF, Schmid R, Huson DH (2008). MetaSim: a sequencing simulator for genomics and metagenomics.PLoS One, 3(10):e3373. doi: 10.1371/journal.pone.0003373" />
<meta name="citation_reference" content="Schloissnig S, Arumugam M, Sunagawa S, Mitreva M, Tap J, Zhu A, Waller A, Mende DR, Kultima JR, Martin J, Kota K, Sunyaev SR, Weinstock GM, Bork P (2013). Genomic variation landscape of the human gut microbiome. Nature, 493(7430):45–50. doi: 10.1038/nature11711" />
<meta name="citation_reference" content="Segata N, Boernigen D, Tickle TL, Morgan XC, Garrett WS, Huttenhower C (2013). Computational meta’omics for microbial community studies. Molecular Systems Biology, 14(9):666. doi: 10.1038/msb.2013.22" />
<meta name="citation_reference" content="Segata N, Waldron L, Ballarini A, Narasimhan V, Jousson O, and Huttenhower C (2012). Metagenomic microbial community profiling using unique clade-specific marker genes. Nature Methods, 9(8):811–4. doi: 10.1038/nmeth.2066." />
<meta name="citation_reference" content="Sunagawa S, Mende DR, Zeller G, Izquierdo-Carrasco F, Berger SA, Kultima JR, Coelho LP, Arumugam M, Tap J, Nielsen HB, Rasmussen S, Brunak S, Pedersen O, Guarner F, de Vos WM, Wang J, Li J, Doré J, Ehrlich SD, Stamatakis A &amp; Bork P (2013). Metagenomic species profiling using universal phylogenetic marker genes. Nature Methods, 10, 1196–1199 (2013) doi:10.1038/nmeth.2693" />
<meta name="citation_reference" content="Wood DE and Salzberg SL. Ultrafast metagenomic sequence classification using exact alignments. In submission." />
<meta name="citation_reference" content="Wu GD, Chen J, Hoffmann C, Bittinger K, Chen YY, Keilbaugh SA, Bewtra M, Knights D, Walters WA, Knight R, Sinha R, Gilroy E, Gupta K, Baldassano R, Nessel L, Li H, Bushman FD, Lewis JD (2011). Linking long-term dietary patterns with gut microbial enterotypes. Science. 334(6052):105–8. doi: 10.1126/science.1208344." />
<meta name="twitter:title" content="SIANN: Strain Identification by Alignment to Near Neighbors" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results." />
<meta name="og-title" property="og:title" content="SIANN: Strain Identification by Alignment to Near Neighbors" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/001727v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2014-01-10" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/001727v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/001727v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/001727v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/001727v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/443" />
    <title>SIANN: Strain Identification by Alignment to Near Neighbors | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RTpA9-mnjpkPkhXkDn93hF2POT5E4dkI8yd3EaEY3A4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"443","apath":"\/biorxiv\/early\/2014\/01\/10\/001727.atom","pisa":"biorxiv;001727v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;001727v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/001727v1","title":"SIANN: Strain Identification by Alignment to Near Neighbors","identifier":"node\/443"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221HamJPa4RnQfLMxez62fIHMip4otaaS4flT3hQpKaH6dpv+6JTIAlZKxA0lFWBlh9edlFAOvLtiO\\\\\\\/Z4HSSIRr8Gqxdkiq6ifpo9uzMqjM7nguwI2uZz4ePlCUGcN06dzAp4FGzIzlg4Xm\\\\\\\/ItvHh6ZR7EwzvRExgVXjjM5L0NeUGvcpguNGENqq6cVFB3tZZSJHTXHvbx6zFKKl\\\\\\\/SAS4bCNjaLzTRZ1Vaj2EX9vmxKdrHH8Ud0PFtIEVh37vA9G5u+q3ZoZj4bhsz+5Ge+3Txu2c\\\\\\\/K7k04jJxRA7vozozKp\\\\\\\/KXdrR0kyGonVEU9VoDom01\\\\\\\/\\\\\\\/30RwQt7VQYGq7nQaW57hh79iatFKpdSUgRh3Zo21oxeEEn+NZtIHLNB5CUuH8QnPGA0Fo\\\\\\\/uE+YvS3\\\\\\\/4TEn\\\\\\\/4gyF8vNgANzvShI59ub4vkxkDvCGqnt88FmMZ1QggR32uYvIguVBywDg8tvi18Nrv6MFLveFkW8x47NsKzzSJEAWvjeXeoBexqrlalvdQAXIckM2uaUKKrJ1sMlOX0OFCOwmE\\\\\\\/Sui7vnac9XDhJp3GfusAw9xH+cJuCqaCDg6q+3p9hDvAX7+faOsss7pU1aErNVdezt3MCCyYCvWc275mboSxUApHTBitmHkHfRXvBKEevRdxRV4CyTciiyz19jOA3ENg6xAIsxwQagkmup9CFlW+gtWntiDE3kzkCGw\\\\\\\/td7YFU+csT+0ML23Z5\\\\\\\/KAwxaq1+HlmQztkcToGoZqIdshkxd9Dch8HSqpEBor4BrCgYPVsOfl\\\\\\\/WdirzhzqG27vIWNn15ogNHdDmVvADegqAS552+eUEMv4aZCaB2q8s9YsTxuTbMnCEWzGZUGeTwbmcbd2yo7QE8UM+PNsuk7en1j3VPEoNTX4OBsC4knADlE9tiBBdIdNHHIZAysR19Pvw9FFO4h7VbUM9n5El5ISqkSxLr6j7DJxKJa0SRFp9IJXO8uM5Y0x5rQt3oGuo2mkYAF1BISedu6Teqt5jR8GnjvhJiX1Uk\\\\\\\/bRBpwTy2\\\\\\\/Q+eSemR1QTyb7qVu7M2vcI72HXRwRmsSisYBFLz3IIX5KA5upo5T2TdyBy1zURPzeGz+MBBI4rYI18IerTHMaEvIU4bA40iWydst79Al7V41Oj7LZ37cR4DQVcl7Cu4\\\\\\\/ETac9bEvnJJJXOefkVSarpvdXE4t03cJHxwV\\\\\\\/mynGcWZClY9cn4sAa+tGX7Eaz6gYwqvymfc3jgvcYW7kPf+XqAnaF2nOc2rjwP00FrGW+mq1aI98YsNSHB9G2V5GHIZDVl9nUIb6ZSw0c7DNZjSLMCSmPyugNhuhDvI3zKi4ClIM=\\\u0022,\\\u0022iv\\\u0022:\\\u00223iujjQBHK15d\\\\\\\/IiNcEDp0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f34249411d9fc6926f43950c6dbb0bea48c73c955c0049021cd1fb1c27ee327b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/001727v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-443 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/001727v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KfLFAqPF8NsNjEedczZYeu08jiPzd5ewEuZQuiRNa2w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="443" id="node443" data-pisa="biorxiv;001727v1" data-pisa-master="biorxiv;001727" data-apath="/biorxiv/early/2014/01/10/001727.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">SIANN: Strain Identification by Alignment to Near Neighbors</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Samuel S.</span> <span class="nlm-surname">Minot</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Stephen D.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Krista L.</span> <span class="nlm-surname">Ternus</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Dana R.</span> <span class="nlm-surname">Kadavy</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/001727 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node443" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Samuel S. Minot </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BS.%2BMinot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minot%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASamuel%2BS.%2BMinot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephen D. Turner </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BD.%2BTurner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turner%20SD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BD.%2BTurner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Krista L. Ternus </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Krista%2BL.%2BTernus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ternus%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKrista%2BL.%2BTernus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Dana R. Kadavy </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dana%2BR.%2BKadavy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kadavy%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADana%2BR.%2BKadavy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/001727v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:443" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/001727v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:443" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/001727v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:443" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/001727v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:443" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/001727v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:443" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:443/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Next-generation sequencing is increasingly being used to study samples composed of mixtures of organisms, such as in clinical applications where the presence of a pathogen at very low abundance may be highly important. We present an analytical method (SIANN: Strain Identification by Alignment to Near Neighbors) specifically designed to rapidly detect a set of target organisms in mixed samples that achieves a high degree of species- and strain-specificity by aligning short sequence reads to the genomes of near neighbor organisms, as well as that of the target. Empirical benchmarking alongside the current state-of-the-art methods shows an extremely high Positive Predictive Value, even at very low abundances of the target organism in a mixed sample. SIANN is available as an Illumina BaseSpace app, as well as through Signature Science, LLC. SIANN results are presented in a streamlined report designed to be comprehensible to the non-specialist user, providing a powerful tool for rapid species detection in a mixed sample. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">There are many different methods that characterize the mixture of organisms present within a metagenomic dataset. Such datasets are generated when a complex environmental sample is processed by a “next-generation” high-throughput genome sequencing protocol, and they consist of large numbers of short nucleotide sequences. Each sequence represents a small fragment of a randomly selected genome from the very large collection of genomes present in the source sample. Those sequences indicate the presence of one organism or another according to their similarity to a set of known reference genomes. While a given sequence may be unique to one species, it also may be found in diverse organisms across the tree of life. Therefore, one analytical challenge (among many) is to take that collection of sequences (likely numbering in the millions) and accurately determine what species are present in the sample. Here we describe a novel method (SIANN: Strain Identification by Alignment to Near Neighbors) that is specifically designed to rapidly detect a set of targeted organisms from a metagenomic dataset by aligning reads to genomic regions that are unique at the strain or species level.</p><p id="p-4">The analytical question motivating a particular piece of metagenomic bioinformatic analysis may vary widely by user and sample type (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Segata, et al., 2013</a>). For example, the function of the human gut microbiome may depend on the relative abundance of hundreds of species of bacteria and the types of metabolic genes they contain (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Wu, et al., 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Schloissnig, et al., 2013</a>). In contrast, the clinical treatment of a patient may depend on whether or not a particular virus, or a consortium of co-infecting pathogens, is/are detected in their blood. It is this second class of presence/absence questions that SIANN is designed to address. SIANN is appropriate for situations in which a user wants to know whether a particular organism or set of organisms is present in a sample, but isn’t interested in the functions encoded in their genomes, the relative abundance of each organism, or any other more in-depth analysis.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Approach</h3><p id="p-5">Metagenomic classification methods are based on a wide variety of theoretical underpinnings. The basic varieties include alignment of reads to various nucleotide databases or exact matching to nucleotide or protein signature sequences (or <em>kmers</em>). A representative set of recent methods are described in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> (also see <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bazinet &amp; Cummings 2012</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/159384/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/159384/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/159384" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-6" class="first-child">Summary of methods for metagenomic classification.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Overall, these methods are designed to either classify individual reads to, and/or predict the total abundance of, clades (<em>e.g.</em> genus or species) across the entire tree of life. They generally require reference databases that are very large and/or require a large amount of processing to generate. The gap SIANN is designed to fill is when the entire tree of life is irrelevant, and only predefined subsets of organisms need to be detected. For an underlying method we chose read alignment to diagnostic genomic regions because the algorithms for read alignment are highly parallelizable and have been optimized heavily by the community at large (the current implementation of SIANN uses bowtie2 [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Langmead &amp; Salzberg, 2012</a>] for the alignment function, but can be adapted to any alignment algorithm). This approach is distinct from using clade-specific marker genes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Segata, et al., 2012</a>) because unique regions that are larger, smaller, or outside of genes can also be used. Furthermore, this approach supports the rapid construction of custom databases using reference genome sets that require only minimal user-supplied structure.</p><p id="p-8">To understand the principle at work, consider a set of reads that have been aligned to the genomes of several strains belonging to two species. Some regions of those genomes are species-specific, some are strain-specific, and some are shared (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). When a set of reads is aligned to those genomes such that each read is placed in as many locations as it has a match (at a reasonably stringent threshold), visual inspection of the distribution of reads yields an intuitive understanding of the true source organism as Species I/Strain B (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>). If Strain B were not present in the reference database, it would still be clear that the organism was an unknown strain of Species I.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) For a group of strains belonging to two different species, some regions may be unique to each species (region 1), while other regions may be unique to strains within each species (regions 2 and 3). B) A set of reads are aligned to these genomes, and the ones that align in a species- or strain-specific manner are identified by the combination of genomes to which they align. In this example, Strain B of Species I is the organism identified." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-918112716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A) For a group of strains belonging to two different species, some regions may be unique to each species (region 1), while other regions may be unique to strains within each species (regions 2 and 3). B) A set of reads are aligned to these genomes, and the ones that align in a species- or strain-specific manner are identified by the combination of genomes to which they align. In this example, Strain B of Species I is the organism identified.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F1.medium.gif" width="440" height="143"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F1.medium.gif" width="440" height="143"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-9" class="first-child">A) For a group of strains belonging to two different species, some regions may be unique to each species (region 1), while other regions may be unique to strains within each species (regions 2 and 3). B) A set of reads are aligned to these genomes, and the ones that align in a species- or strain-specific manner are identified by the combination of genomes to which they align. In this example, Strain B of Species I is the organism identified.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">The unique identification of a species or strain is quantified by the proportion of the genome that is determined to be species- or strain-specific (defined as reads that are aligned to regions that are species- or strain-specific). Each species and strain is then assigned a numerical measure of the proportion that is covered by these diagnostic reads, and that proportional measure is compared to the ideal case, where sequences from a single organism (generated <em>in silico</em>) are aligned against the database in an identical manner. After that normalization factor is applied, the resulting score indicates whether the source sample contained any of the organisms in the reference database. The analysis is conducted independently on both the species and the strain level, so that if the true strain is not present in the database, the species of origin will still be identified. While many methods consider the complete taxonomic tree and assign reads to the least common ancestor, SIANN considers only two taxonomic levels: species and strain, throwing out anything that is not unique at one of those levels and thus obviating many of the confounding factors introduced by manually curated taxonomies.</p><p id="p-11">The example shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1b</a> indicates that species-specific reads are identified as reads that align to one species (Species I, in that case) but not the other. If Species II were not present in the example shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1b</a>, a much larger number of reads would be assigned as “species-specific,” when in fact those regions are shared with other species. Therefore, the ability of this method to identify strain- and species-specific sequences is a direct function of the inclusion of near neighbors in the reference database. This characteristic is shared among many classification algorithms, but it is of particular note for this method when users have an opportunity to construct their own database. In order to detect a target species with a high degree of specificity (reducing false positives), it is necessary to include other related species in the reference database. Only by parallel alignment to those near neighbors can the redundant sequences be separated from the species-specific ones. For example, in order to detect <em>Bacillus anthracis</em> in a sample, it would be necessary to include other species of <em>Bacilli</em> in the reference database so that the presence of <em>B. cereus</em> or <em>B. thuringiensis</em> in a sample does not lead to a false call for <em>B. anthracis</em>.</p><p id="p-12">The nomenclature of genus, species, and strain is potentially problematic because it does not correspond to a consistent degree of evolutionary distance or genomic distinctiveness. The ability to distinguish two organisms by any method using genomic sequence data is proportional to the amount of each genome that is shared or unique. One might assume that any two organisms of the same species will have a relatively predictable amount of shared genomic identity. However, some pairs of organisms from the same species may have less in common than other pairs of organisms from different species or even genera. This ambiguity impacts SIANN in two ways. If two organisms have very little genomic sequence to distinguish them, the sensitivity of SIANN to detect either one will diminish (the rate of false negatives will increase as the likelihood of sequencing unique regions decreases). Conversely, if an organism is extremely dissimilar to the near neighbors selected for the database, the specificity with which SIANN detects that organism will decline (the rate of false positives will increase as the number of related genomes available in the database decreases). For example, if a database contained only <em>E. coli</em> and <em>B.anthracis</em>, a sample containing <em>B. cereus</em> would be misidentified as contraining <em>B. anthracis.</em> In the intended use case, a database targeting <em>B. anthracis</em> would contain <em>B. cereus</em> and a number of other near neighbors to prevent that kind of misidentification. It would be convenient to say that an ideal database can be made by calculating the ideal genetic distance between all references and then finding an ideal set of organisms to make up that database, but the behavior of any database will be governed by the particular genomes of the organisms it encounters in the wild. Because not all organisms evolve in the same manner (differences in mutation rate, horizontal gene transfer, recombination, etc), the suitability of a database and method to detect a given organism can only be determined by thorough validation and benchmarking, as well as updating the reference database as needed. Users of SIANN may construct their own custom databases to include newly identified genomes or specific subsets of genomes that best suit their research interests.</p><p>Steps to construct a custom database:
</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-14">Select a set of target organisms</p></li><li id="list-item-2"><p id="p-15">Gather a set of genome sequences for those target organisms as well as a matched set of near neighbors</p></li><li id="list-item-3"><p>Using those reference genome sequences as an input, SIANN will:
</p><ol class="list-alphalower " id="list-2"><li id="list-item-4"><p id="p-17">Construct a reference index for alignment</p></li><li id="list-item-5"><p id="p-18">Simulate a set of reads from each genome</p></li><li id="list-item-6"><p id="p-19">Align each of those simulated read sets to all of the reference genomes</p></li><li id="list-item-7"><p id="p-20">Calculate the proportion of each reference genome that is strain- or species-specific</p></li><li id="list-item-8"><p id="p-21">[If two organisms do not have a minimal amount of unique sequences that exceeds the rate of sequencing error, SIANN asks that all but one of those organisms are removed from the database to eliminate redundancy. Note that the user can provide a single representative genome with multiple strain names so that the redundant strain names are not lost.]</p></li></ol></li></ol><p id="p-22">The files contained within each SIANN database are a compressed genomic index and a list containing the proportion of each reference genome that was found to be strain- or species-specific during database construction.</p><p>To run SIANN:
</p><ol class="list-ord " id="list-3"><li id="list-item-9"><p id="p-24">Select a pre-made SIANN database and a set of sequences to be analyzed, and</p></li><li id="list-item-10"><p>SIANN will:
</p><ol class="list-alphalower " id="list-4"><li id="list-item-11"><p id="p-26">Align each of the reads against the reference genomes</p></li><li id="list-item-12"><p id="p-27">Calculate the proportion of each reference genome that is strain- or species-specific within those reads</p></li><li id="list-item-13"><p id="p-28">Compare that proportion to the simulated ideal case generated during database creation</p></li><li id="list-item-14"><p id="p-29">Calculate the probability that the given results could be generated by random chance</p></li><li id="list-item-15"><p id="p-30">Report the normalized proportion and non-parametric statistic of likelihood for each strain and species in the reference database. The normalized proportion of the genome covered by strain- or species- specific reads is the primary statistic reported by this tool.</p></li></ol></li></ol></div><div id="sec-4" class="subsection"><h3>Benchmarking</h3><p id="p-31">The performance of SIANN (version 1.6) was tested in comparison to the following state-of-the-art metagenomic classification programs: LMAT (version 1.2), MetaPhlAn (version 1.7.7), and Kraken (version 0.9.1b). All of the programs in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> were investigated for this effort, and three were chosen based on their ability to run on our high-performance computing cluster with an execution time and memory requirement that would be suitable to a clinical lab. Each program was run on a set of 600 simulated datasets generated by MetaSim (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Richter, et al., 2008</a>). Each dataset consisted of 15,000,000 reads (100bp single-ended) with Illumina-simulated error (fourth-degree polynomial) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Korbel, et al., 2009</a>). The 600 datasets were broken into 12 sets of 50 replicates. Each of the 12 sets contained organisms at different levels of abundance as shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Organisms were specifically chosen in pairs so that the ability to distinguish these near neighbors could be determined. The abundances were staggered at 4-fold intervals so that a wide range could be evaluated. All known species of near neighbors for each of the 12 target organisms were included in the reference database used by SIANN for this benchmarking (“Target Pathogen Database”) and are shown in <a id="xref-app-1-1" class="xref-app" href="#app-1">Appendix 1</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/159196/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/159196/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/159196" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">The abundance of each target organism in each set of simulated datasets. Each set is indicated by the number in the top row, and was generated with 50 replicates.</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Each program outputs a distinct measure. Kraken and LMAT both count the reads assigned to each taxon, MetaPhlAn calculates the abundance, and SIANN outputs a measure of the proportion of diagnostic genomic regions present. To put these measures on an even footing, we empirically calculated the false positive rate for each method over all 600 samples, at each possible measure of output. Because each dataset is made up of known organisms, any result can be classified as true or false. Therefore, for any possible result (say, 513 reads classified by LMAT or 1.6% abundance assigned by MetaPhlAn), one can calculate the proportion of calls with at least the same amount of support that were correct (True Positives/[True Positives + False Positives]), over all of the 600 datasets. That measure is commonly given as Positive Predictive Value (PPV). For each program, the results can be translated from the raw value into a PPV that is based on this empirical measure of error. The key item of interest is the PPV value for the results that we know to be true positives, the defined spike organisms. Another way of describing this approach is to say that the results of each program have been normalized to the false positive error rate that was empirically observed. If another set of samples were generated, the PPV vs. raw value curve (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) would likely fall differently, but in this case it gives us a means of comparing a diverse set of methods against the same ground truth. If method 1 detects an organism with a higher PPV than method 2 does, it means that method 1 has fewer false positives in the range that it reports true positives, which is the definition of utility in this setting.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship of reported value for each program (horizontal axis, log scale) to the empirically-determined Positive Predictive Value (PPV), shown on the vertical axis. While the exact values depend on the test data used, the general values at significant cutoff values (0.8, 0.9, 0.95 PPV) remain relatively constant across different datasets (data not shown)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-918112716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship of reported value for each program (horizontal axis, log scale) to the empirically-determined Positive Predictive Value (PPV), shown on the vertical axis. While the exact values depend on the test data used, the general values at significant cutoff values (0.8, 0.9, 0.95 PPV) remain relatively constant across different datasets (data not shown).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F2.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F2.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-33" class="first-child">Relationship of reported value for each program (horizontal axis, log scale) to the empirically-determined Positive Predictive Value (PPV), shown on the vertical axis. While the exact values depend on the test data used, the general values at significant cutoff values (0.8, 0.9, 0.95 PPV) remain relatively constant across different datasets (data not shown).</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">For each method, PPV was calculated as a function of raw output value. Briefly, this was done by compiling the output for all 600 samples, labeling each result as false or true based on the sample set that it came from, and then calculating (at each possible value of output) what the proportion of TP/[TP + FP] was for results with at least that level of raw output. Some simplification steps were taken, such as focusing on the species-level assignments (for comparison with methods that do not perform strain assignment), and only taking the top hit for each species from each dataset. Custom R and BASH scripts were used for the data compilation and analysis.</p></div></div><div class="section" id="sec-5"><h2 class="">Results</h2><p id="p-35">The relationship of raw output value to PPV is shown for each of the four methods in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. The point at which PPV is very close to 1 (where 95% of results are true positives) is ∼41,000 reads for Kraken, ∼2,800 reads for LMAT, ∼38% abundance for MetaPhlAn, and 0.21 for SIANN. For SIANN this means that having 38% of the species-unique genome covered by reads resulted in the vast majority of calls being accurate.</p><p id="p-36">For read-assignment methods (such as LMAT and Kraken), manual inspection of the results may yield a different understanding of confidence than is presented here, or in any automated analysis. For example, while each read that is assigned by LMAT and Kraken fall above a certain cutoff for species-specificity, some individual reads may be much more specific than others. One could identify a read that aligns to a single species of bacteria with 100% accuracy over its 300bp length, with the next closest match being only 90% similar. It is extremely unlikely that a 300bp exact match would arise due to random chance, and so the user could say with confidence that the organism of interest is found within the sequence data (not considering contamination, horizontal gene transfer, etc). However, such an approach is not currently implemented in an automated method, and many of the steps needed to make that assertion are performed manually by a domain expert, including alignment to near neighbors and ensuring that the read does not fall within a transposon, plasmid, etc. Therefore, while one could say that a single read is all that is needed to state with high PPV that an organism is present, the amount of reads assigned in an automated manner needed to achieve that level of PPV will number in the thousands (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2</a>).</p><p id="p-37">The next phase of benchmarking was to determine how many raw input reads were needed to achieve the threshold for high PPV. To demonstrate this we plotted the known abundance of each spike organism against the PPV value generated by each method (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Each point (an organism at a known level of abundance) is comprised of a maximum of 50 replicates, where the diameter of each point increases with an increasing number of replicates. For demonstration purposes we are showing two pairs of bacteria and three viruses. Recall that for each of the pairs of bacteria (and the two poxviruses) any sample containing one did not contain the other (as shown in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The empty boxes result from the organisms not being called at any abundance. For MetaPhlAn, that is a result of no viruses being included in the version of the reference database available for this analysis. Kraken assigned no reads to Hanta virus because viral RNA genomes were not included in this version of the reference database (personal communication with D. Wood). This emphasizes the point that a) the ability to create custom databases targeting organisms of interest can be valuable, and b) the performance of any method must be benchmarked against each potential target of interest.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Positive Predictive Value (PPV, vertical axis) is shown for each organism (boxes on right), at each level of known abundance (horizontal axis, see Table 2), and each program (boxes at top), across a maximum of 50 replicates (indicated by the size of each point). Note that the reference database for MetaPhlAn does not include viruses, and the reference database for Kraken does not include RNA viruses" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-918112716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The Positive Predictive Value (PPV, vertical axis) is shown for each organism (boxes on right), at each level of known abundance (horizontal axis, see Table 2), and each program (boxes at top), across a maximum of 50 replicates (indicated by the size of each point). Note that the reference database for MetaPhlAn does not include viruses, and the reference database for Kraken does not include RNA viruses&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F3.medium.gif" width="426" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F3.medium.gif" width="426" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2014/01/10/001727/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-38" class="first-child">The Positive Predictive Value (PPV, vertical axis) is shown for each organism (boxes on right), at each level of known abundance (horizontal axis, see <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), and each program (boxes at top), across a maximum of 50 replicates (indicated by the size of each point). Note that the reference database for MetaPhlAn does not include viruses, and the reference database for Kraken does not include RNA viruses</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">All methods were able identify the bulk of organisms in their databases at high abundances (75% and 18%, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), however performance varied considerably at lower abundances and depended on the particular organism and method used. SIANN detected each organism at high confidence, even at levels as low as 0.3% and 0.07% of the total.</p></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-40">The process of detecting trace amounts of a specific organism in a complex mixture of DNA is challenging enough for an expert, but that pales in comparison to the difficulty of accomplishing the same certainty of detection in an automated manner. The results presented here show that SIANN rapidly detects the presence of a given set of organisms with a high degree of specificity and sensitivity. For example, at the 95% confidence (PPV) cutoff of 0.2, SIANN reliably detects all of the organisms tested here at as low as 0.3% abundance. This strong performance is likely due to the fact that SIANN is able to use a method (read alignment to whole genomes) that would be far too computationally costly if it were applied to the entire collection of known genomes. By focusing on a set of (customizable) target organisms and their near neighbors, SIANN can operate quickly and with low computational requirements while delivering highly accurate results.</p><p id="p-41">SIANN is available on Illumina’s BaseSpace (<a href="http://www.basespace.illumina.com">www.basespace.illumina.com</a>) as a NativeApp, with the database tested here (<a id="xref-app-1-2" class="xref-app" href="#app-1">Appendix 1</a>), as well as a database made from the NCBI representative set of prokaryotic genomes (<a href="ftp://ftp.ncbi.nlm.nih.gov/genomes/genome_reports/">ftp://ftp.ncbi.nlm.nih.gov/genomes/genome_reports/</a>) (<a id="xref-app-2-1" class="xref-app" href="#app-2">Appendix 2</a>) and the complete set of NCBI viral genomes (<a href="ftp://ftp.ncbi.nlm.nih.gov/refseq/release/viral/">ftp://ftp.ncbi.nlm.nih.gov/refseq/release/viral/</a>) (<a id="xref-app-3-1" class="xref-app" href="#app-3">Appendix 3</a>).</p><p id="p-42">BaseSpace was chosen as an appropriate release platform because while the entire set of software and dependencies can be deployed by the user from within a graphical user interface, the actual computation takes place in a controlled ‘cloud’ environment. Such a distribution strategy obviates the need to satisfy the multiple software or OS dependencies that often arises with academic computational methods. Results for SIANN are compiled into a report format, showing both the organisms that surpass 95% confidence, as well as the closest strain match for each species. The default view masks the raw data output, so that the results are human-readable and do not present extraneous information. While the code for execution and database-construction on a users system is available from Signature Science, LLC, additional databases on the BaseSpace platform can be made available upon request.</p><p id="p-43">There is a neverending list of questions that one could ask of metagenomic sequencing data generated from important samples. Instead of answering them all, we demonstrate a technique with a very narrow focus that is able to report with a high degree of confidence whether a given set of organisms is present in a sample. These results are presented to the user in a comprehensible format, and accessible on a commonly-used web platform. The world of bioinformatics will continue to progress and develop more sophisticated tools for metagenomic analysis, and we hope that the utility of SIANN will convince others to package and benchmark their tools in a way that they can be used with confidence by the larger public, as well as the research community.</p></div><h2 class="">Appendices</h2><div class="section app" id="app-1"><h2 class="">Appendix 1: Target Pathogen Database</h2><p id="p-44">Arenaviridae Arenavirus Golden-Gate-virus</p><p id="p-45">Arenaviridae Arenavirus Lujo-virus</p><p id="p-46">Arenaviridae Flexal-virus segment-L</p><p id="p-47">Arenaviridae New-world-arenaviruses Allpahuayo-virus</p><p id="p-48">Arenaviridae New-world-arenaviruses Chapare-virus</p><p id="p-49">Arenaviridae New-world-arenaviruses Guanarito-virus</p><p id="p-50">Arenaviridae New-world-arenaviruses Junin-virus</p><p id="p-51">Arenaviridae New-world-arenaviruses Machupo-virus</p><p id="p-52">Arenaviridae New-world-arenaviruses Sabia-virus</p><p id="p-53">Arenaviridae New-world-arenaviruses Tacaribe-virus</p><p id="p-54">Arenaviridae New-world-arenaviruses Whitewater-Arroyo-virus</p><p id="p-55">Arenaviridae Old-world-arenaviruses Ippy-virus</p><p id="p-56">Arenaviridae Old-world-arenaviruses Lassa-virus</p><p id="p-57">Arenaviridae Old-world-arenaviruses Mopeia-virus-AN20410</p><p id="p-58">Asfarviridae African-swine-fever-virus Benin-971-pathogenic-isolate</p><p id="p-59">Asfarviridae African-Swine-Fever-Virus</p><p id="p-60">Bacillus anthracis A2012 Bant 02 1</p><p id="p-61">Bacillus anthracis Ames Ancestor</p><p id="p-62">Bacillus anthracis Sterne</p><p id="p-63">Bacillus cereus 03BB102</p><p id="p-64">Bacillus cereus AH187</p><p id="p-65">Bacillus cereus AH820</p><p id="p-66">Bacillus cereus ATCC 14579</p><p id="p-67">Bacillus cereus B4264</p><p id="p-68">Bacillus cereus F65185</p><p id="p-69">Bacillus cytotoxicus NVH 391-98</p><p id="p-70">Bacillus mycoides DSM 2048</p><p id="p-71">Bacillus mycoides Rock1-4</p><p id="p-72">Bacillus thuringiensis BMB171</p><p id="p-73">Bacillus thuringiensis Bt407</p><p id="p-74">Bacillus thuringiensis HD-771</p><p id="p-75">Bacillus thuringiensis serovar chinensis CT-43</p><p id="p-76">Bacillus thuringiensis serovar konkukian 97-27</p><p id="p-77">Brucella abortus A13334</p><p id="p-78">Brucella ceti B1 94</p><p id="p-79">Brucella ceti M13 05 1 supercont1 22</p><p id="p-80">Brucella melitensis ATCC 23457</p><p id="p-81">Brucella melitensis bv 1 str 16M</p><p id="p-82">Brucella ovis ATCC 25840</p><p id="p-83">Brucella suis 1330</p><p id="p-84">Brucella suis ATCC 23445</p><p id="p-85">Bunyaviridae Akabane-virus segment-M</p><p id="p-86">Bunyaviridae Hantavirus Andes-virus</p><p id="p-87">Bunyaviridae Hantavirus Dobrava-Belgrade-virus-strain-DOBV-Ano-Poroia-Afl9-1999</p><p id="p-88">Bunyaviridae Hantavirus Hantaan-virus</p><p id="p-89">Bunyaviridae Hantavirus Puumala-virus</p><p id="p-90">Bunyaviridae Hantavirus Seoul-virus-strain-Seoul-80-39-clone-1</p><p id="p-91">Bunyaviridae Hantavirus Sin-Nombre-virus</p><p id="p-92">Bunyaviridae Hantavirus Thottapalayam-virus</p><p id="p-93">Bunyaviridae Hantavirus Tula-virus</p><p id="p-94">Bunyaviridae Nairovirus Crimean-Congo-hemorrhagic-fever-virus</p><p id="p-95">Bunyaviridae Nairovirus Dugbe-virus</p><p id="p-96">Bunyaviridae Phlebovirus Rift-Valley-fever-virus</p><p id="p-97">Burkholderia cenocepacia HI2424</p><p id="p-98">Burkholderia cenocepacia J2315</p><p id="p-99">Burkholderia cenocepacia MC0-3</p><p id="p-100">Burkholderia cepacia GG4</p><p id="p-101">Burkholderia gladioli BSR3</p><p id="p-102">Burkholderia glumae BGR1</p><p id="p-103">Burkholderia mallei 2002721280</p><p id="p-104">Burkholderia mallei ATCC 10399</p><p id="p-105">Burkholderia mallei NCTC 10247</p><p id="p-106">Burkholderia mallei SAVP1</p><p id="p-107">Burkholderia multivorans ATCC 17616</p><p id="p-108">Burkholderia oklahomensis C6786</p><p id="p-109">Burkholderia oklahomensis EO147</p><p id="p-110">Burkholderia pseudomallei 1026b</p><p id="p-111">Burkholderia pseudomallei 1106a</p><p id="p-112">Burkholderia pseudomallei 1710b</p><p id="p-113">Burkholderia pseudomallei 668</p><p id="p-114">Burkholderia pseudomallei BPC006</p><p id="p-115">Burkholderia pseudomallei K96243</p><p id="p-116">Burkholderia pyrrocinia CH-67</p><p id="p-117">Burkholderia thailandensis ATCC 700388</p><p id="p-118">Burkholderia thailandensis E264</p><p id="p-119">Burkholderia thailandensis MSMB121</p><p id="p-120">Campylobacter coli JV20 Campylobacter fetus subsp fetus 82-40</p><p id="p-121">Campylobacter jejuni RM1221</p><p id="p-122">Campylobacter jejuni subsp doylei 26997</p><p id="p-123">Campylobacter jejuni subsp jejuni 81-176</p><p id="p-124">Campylobacter jejuni subsp jejuni NCTC 11168 ATCC 700819</p><p id="p-125">Campylobacter upsaliensis JV21</p><p id="p-126">Clostridium acetobutylicum DSM 1731</p><p id="p-127">Clostridium botulinum A str ATCC 3502</p><p id="p-128">Clostridium botulinum BKT015925</p><p id="p-129">Clostridium botulinum B str Eklund 17B</p><p id="p-130">Clostridium botulinum E3 str Alaska E43</p><p id="p-131">Clostridium botulinum F str 230613</p><p id="p-132">Clostridium botulinum H04402 065</p><p id="p-133">Clostridium difficile 2007855</p><p id="p-134">Clostridium difficile 630</p><p id="p-135">Clostridium difficile BI1</p><p id="p-136">Clostridium difficile BI9</p><p id="p-137">Clostridium perfringens ATCC 13124</p><p id="p-138">Clostridium perfringens SM101</p><p id="p-139">Clostridium perfringens str 13</p><p id="p-140">Clostridium symbiosum WAL-14163</p><p id="p-141">Clostridium symbiosum WAL-14673</p><p id="p-142">Clostridium tetani E88</p><p id="p-143">Clostridium thermocellum ATCC 27405</p><p id="p-144">Clostridium thermocellum DSM 1313</p><p id="p-145">Clostridium tunisiense TJ C661</p><p id="p-146">Clostridium ultunense Esp</p><p id="p-147">Coccidioides immitis H5384</p><p id="p-148">Coccidioides immitis RMSCC 2394</p><p id="p-149">Coccidioides immitis RS</p><p id="p-150">Coccidioides posadasii C735 delta SOWgp</p><p id="p-151">Coccidioides posadasii CPA 0001</p><p id="p-152">Coccidioides posadasii CPA 0020</p><p id="p-153">Coronaviridae Alphacoronavirus Bat-coronavirus-HKU2</p><p id="p-154">Coronaviridae Alphacoronavirus Feline-infectious-peritonitis-virus</p><p id="p-155">Coronaviridae Alphacoronavirus Human-coronavirus-229E</p><p id="p-156">Coronaviridae Alphacoronavirus TGEV-Purdue-P115</p><p id="p-157">Coronaviridae Bafinivirus White-bream-virus</p><p id="p-158">Coronaviridae Betacoronavirus Bovine-coronavirus</p><p id="p-159">Coronaviridae Betacoronavirus Murine-hepatitis-virus-strain-A59</p><p id="p-160">Coronaviridae Betacoronavirus Murine-hepatitis-virus-strain-JHM</p><p id="p-161">Coronaviridae Betacoronavirus SARS-coronavirus</p><p id="p-162">Coronaviridae Coronavirinae Munia-coronavirus-HKU13-3514</p><p id="p-163">Coronaviridae Gammacoronavirus Avian-infectious-bronchitis-virus</p><p id="p-164">Coronaviridae Gammacoronavirus Turkey-coronavirus</p><p id="p-165">Coronaviridae Torovirus Breda-virus</p><p id="p-166">Coronaviridae unclassified-coronaviruses Bat-coronavirus-BM48-31-BGR-2008</p><p id="p-167">Coronaviridae unclassified-coronaviruses Bovine-respiratory-coronavirus-AH187</p><p id="p-168">Coronaviridae unclassified-coronaviruses Human-enteric-coronavirus-strain-4408</p><p id="p-169">Coxiella burnetii CbuG Q212</p><p id="p-170">Coxiella burnetii Dugway 5J108 111</p><p id="p-171">Coxiella burnetii RSA 493 Diplorickettsia massiliensis 20B CS 1</p><p id="p-172">Ehrlichia canis str Jake Ehrlichia chaffeensis str Arkansas</p><p id="p-173">Ehrlichia chaffeensis str Sapulpa ctg90</p><p id="p-174">Ehrlichia ruminantium str Gardel</p><p id="p-175">Ehrlichia ruminantium str Welgevonden</p><p id="p-176">Escherichia coli APEC O1</p><p id="p-177">Escherichia coli BL21 DE3</p><p id="p-178">Escherichia coli B str REL606</p><p id="p-179">Escherichia coli ETEC H10407</p><p id="p-180">Escherichia coli O104H4 str 2011C-3493</p><p id="p-181">Escherichia coli O157H7 str EC4115</p><p id="p-182">Escherichia coli O157H7 str Sakai</p><p id="p-183">Escherichia coli O7K1 str CE10</p><p id="p-184">Escherichia coli O83H1 str NRG 857C</p><p id="p-185">Escherichia coli str K-12 substr MG1655</p><p id="p-186">Filoviridae Ebolavirus Bundibugyo-ebolavirus</p><p id="p-187">Filoviridae Ebolavirus Cote-dIvoire-ebolavirus</p><p id="p-188">Filoviridae Ebolavirus Ebola-virus-Mayinga-Zaire</p><p id="p-189">Filoviridae Ebolavirus Reston-ebolavirus</p><p id="p-190">Filoviridae Ebolavirus Sudan-ebolavirus</p><p id="p-191">Filoviridae Marburgvirus Lake-Victoria-marburgvirus-Musoke</p><p id="p-192">Flaviviridae Alkhurma-virus</p><p id="p-193">Flaviviridae Classical-swine-fever-virus</p><p id="p-194">Flaviviridae Dengue-virus 1</p><p id="p-195">Flaviviridae Dengue-virus 2</p><p id="p-196">Flaviviridae Dengue-virus 3</p><p id="p-197">Flaviviridae Dengue-virus 4</p><p id="p-198">Flaviviridae Japanese-encephalitis-virus genome</p><p id="p-199">Flaviviridae Karshi-virus</p><p id="p-200">Flaviviridae Langat-virus</p><p id="p-201">Flaviviridae Louping-ill-virus</p><p id="p-202">Flaviviridae Murray-Valley-encephalitis-virus</p><p id="p-203">Flaviviridae Omsk-hemorrhagic-fever-virus</p><p id="p-204">Flaviviridae Powassan-virus</p><p id="p-205">Flaviviridae St-Louis-encephalitis-virus</p><p id="p-206">Flaviviridae Tick-borne-encephalitis-virus</p><p id="p-207">Flaviviridae Usutu-virus</p><p id="p-208">Flaviviridae West-Nile-virus</p><p id="p-209">Flaviviridae Yellow-fever-virus</p><p id="p-210">Francisella cf novicida Fx1</p><p id="p-211">Francisella noatunensis subsp orientalis str Toba 04</p><p id="p-212">Francisella novicida U112</p><p id="p-213">Francisella philomiragia subsp philomiragia ATCC 25015</p><p id="p-214">Francisella philomiragia subsp philomiragia ATCC 25017</p><p id="p-215">Francisella tularensis subsp holarctica F92</p><p id="p-216">Francisella tularensis subsp mediasiatica FSC147</p><p id="p-217">Francisella tularensis subsp tularensis FSC198</p><p id="p-218">Herpesviridae Alcelaphine-herpesvirus 1</p><p id="p-219">Herpesviridae Macacine-herpesvirus 1</p><p id="p-220">Listeria fleischmannii LU2006-1 c88</p><p id="p-221">Listeria innocua Clip11262</p><p id="p-222">Listeria innocua FSL J1-023</p><p id="p-223">Listeria ivanovii FSL F6-596</p><p id="p-224">Listeria ivanovii subsp ivanovii PAM 55</p><p id="p-225">Listeria marthii FSL S4-120</p><p id="p-226">Listeria monocytogenes 07PF0776</p><p id="p-227">Listeria monocytogenes 08-5578</p><p id="p-228">Listeria monocytogenes 10403S</p><p id="p-229">Listeria monocytogenes ATCC 19117</p><p id="p-230">Listeria monocytogenes Finland 1998</p><p id="p-231">Listeria monocytogenes FSL R2-561</p><p id="p-232">Listeria monocytogenes La111</p><p id="p-233">Listeria seeligeri FSL N1-067</p><p id="p-234">Listeria seeligeri serovar 12b str SLCC3954</p><p id="p-235">Listeria welshimeri serovar 6b str SLCC5334</p><p id="p-236">Paramyxoviridae Avulavirus Newcastle-disease-virus-B1</p><p id="p-237">Paramyxoviridae Henipavirus Hendra-virus</p><p id="p-238">Paramyxoviridae Henipavirus Nipah-virus</p><p id="p-239">Paramyxoviridae Menangle-virus</p><p id="p-240">Paramyxoviridae Morbillivirus Measles-virus</p><p id="p-241">Paramyxoviridae Peste-des-petits-ruminants-virus</p><p id="p-242">Paramyxoviridae Respirovirus Human-parainfluenza-virus-1</p><p id="p-243">Paramyxoviridae Respirovirus Human-parainfluenza-virus-3</p><p id="p-244">Paramyxoviridae Respirovirus Sendai-virus</p><p id="p-245">Paramyxoviridae Rinderpest-virus strain-Kabete-O</p><p id="p-246">Paramyxoviridae Rubulavirus Human-parainfluenza-virus-2</p><p id="p-247">Paramyxoviridae Rubulavirus Human-parainfluenza-virus-4a</p><p id="p-248">Paramyxoviridae Rubulavirus Mumps-virus</p><p id="p-249">Picornaviridae Foot-and-mouth-disease-virus -type-O</p><p id="p-250">Picornaviridae Swine-vesicular-disease-virus strain-HK70</p><p id="p-251">Picornaviridae Swine-vesicular-disease-virus strain-NET192</p><p id="p-252">Poxviridae Avipoxvirus Fowlpox-virus</p><p id="p-253">Poxviridae Crocodylidpoxvirus Nile-crocodilepox-virus</p><p id="p-254">Poxviridae Goatpox-virus Pellor</p><p id="p-255">Poxviridae Leporipoxvirus Myxoma-virus</p><p id="p-256">Poxviridae Lumpy-skin-disease-virus NI-2490</p><p id="p-257">Poxviridae Molluscipoxvirus Molluscum-contagiosum-virus-subtype-1</p><p id="p-258">Poxviridae Orthopoxvirus Camelpox-virus</p><p id="p-259">Poxviridae Orthopoxvirus Cowpox-virus</p><p id="p-260">Poxviridae Orthopoxvirus Ectromelia-virus</p><p id="p-261">Poxviridae Orthopoxvirus Monkeypox-virus-Zaire-96-I-16</p><p id="p-262">Poxviridae Orthopoxvirus Taterapox-virus</p><p id="p-263">Poxviridae Orthopoxvirus Vaccinia-virus</p><p id="p-264">Poxviridae Orthopoxvirus Variola-virus</p><p id="p-265">Poxviridae Sheeppox-virus 17077-99</p><p id="p-266">Poxviridae Suipoxvirus Swinepox-virus</p><p id="p-267">Puccinia graminis f sp tritici CRL 75-36-700-3</p><p id="p-268">Ralstonia pickettii 12D</p><p id="p-269">Ralstonia pickettii 12J</p><p id="p-270">Ralstonia solanacearum CFBP2957</p><p id="p-271">Ralstonia solanacearum CMR15</p><p id="p-272">Ralstonia solanacearum GMI1000</p><p id="p-273">Rathayibacter toxicus DSM 7488</p><p id="p-274">Reoviridae African-horsesickness-virus segment-10</p><p id="p-275">Rhabdoviridae Vesicular-stomatitis-Indiana-virus</p><p id="p-276">Rhabdoviridae Vesicular-stomatitis-virus strain-NJ2075212NM</p><p id="p-277">Rickettsia bellii OSU 85-389</p><p id="p-278">Rickettsia conorii str Malish 7</p><p id="p-279">Rickettsia prowazekii str Breinl</p><p id="p-280">Rickettsia prowazekii str RpGvF24</p><p id="p-281">Rickettsia rickettsii str Arizona</p><p id="p-282">Rickettsia typhi str B9991CWPP</p><p id="p-283">Rickettsiella grylli gcontig 634</p><p id="p-284">Salmonella bongori N268-08</p><p id="p-285">Salmonella bongori NCTC 12419</p><p id="p-286">Salmonella enterica subsp arizonae serovar 62z4z23-str RSK2980</p><p id="p-287">Salmonella enterica subsp enterica serovar Dublin str CT 02021853</p><p id="p-288">Salmonella enterica subsp enterica serovar Newport str SL254</p><p id="p-289">Salmonella enterica subsp enterica serovar Paratyphi A str ATCC 9150</p><p id="p-290">Salmonella enterica subsp enterica serovar Typhimurium str LT2</p><p id="p-291">Salmonella enterica subsp enterica serovar Typhi str Ty2</p><p id="p-292">Shigella boydii CDC 3083-94</p><p id="p-293">Shigella boydii Sb227</p><p id="p-294">Shigella dysenteriae Sd197</p><p id="p-295">Shigella flexneri 2002017</p><p id="p-296">Shigella flexneri 2a str 2457T</p><p id="p-297">Shigella flexneri 2a str 301</p><p id="p-298">Shigella flexneri 5 str 8401</p><p id="p-299">Shigella sonnei 53G</p><p id="p-300">Shigella sonnei Ss046</p><p id="p-301">Staphylococcus arlettae CVD059 SARL c230</p><p id="p-302">Staphylococcus aureus 04-02981</p><p id="p-303">Staphylococcus aureus 08BA02176</p><p id="p-304">Staphylococcus aureus subsp aureus N315</p><p id="p-305">Staphylococcus aureus subsp aureus NCTC 8325</p><p id="p-306">Staphylococcus aureus subsp aureus TW20</p><p id="p-307">Staphylococcus capitis QN1 Contig63</p><p id="p-308">Staphylococcus capitis SK14</p><p id="p-309">Staphylococcus caprae C87</p><p id="p-310">Staphylococcus carnosus subsp carnosus TM300</p><p id="p-311">Staphylococcus epidermidis ATCC 12228</p><p id="p-312">Staphylococcus epidermidis RP62A</p><p id="p-313">Staphylococcus equorum subsp equorum Mu2</p><p id="p-314">Staphylococcus haemolyticus JCSC1435</p><p id="p-315">Staphylococcus hominis SK119</p><p id="p-316">Staphylococcus hominis subsp hominis C80</p><p id="p-317">Staphylococcus lugdunensis HKU09-01</p><p id="p-318">Staphylococcus lugdunensis N920143</p><p id="p-319">Togaviridae Alphavirus Barmah-Forest-virus Togaviridae Chikungunya-virus</p><p id="p-320">Togaviridae EEEV-complex Eastern-equine-encephalitis-virus</p><p id="p-321">Togaviridae Rubivirus Rubella-virus</p><p id="p-322">Togaviridae SFV-complex O-nyong-nyong-virus</p><p id="p-323">Togaviridae Venezuelan-equine-encephalitis-virus</p><p id="p-324">Togaviridae WEEV-complex Sindbis-virus</p><p id="p-325">Togaviridae Western-equine-encephalomyelitis-virus</p><p id="p-326">Xanthomonas albilineans GPE PC73</p><p id="p-327">Xanthomonas axonopodis Xac29-1</p><p id="p-328">Xanthomonas oryzae pv oryzae KACC 10331</p><p id="p-329">Xanthomonas oryzae pv oryzae MAFF 311018</p><p id="p-330">Xanthomonas oryzae pv oryzae PXO99A</p><p id="p-331">Xanthomonas vasicola pv vasculorum NCPPB 1326 scf 9767 4580</p><p id="p-332">Yersinia aldovae ATCC 35236</p><p id="p-333">Yersinia bercovieri ATCC 43970</p><p id="p-334">Yersinia enterocolitica IP 10393</p><p id="p-335">Yersinia enterocolitica IP2222</p><p id="p-336">Yersinia enterocolitica subsp enterocolitica 8081</p><p id="p-337">Yersinia enterocolitica subsp palearctica 105 5R</p><p id="p-338">Yersinia frederiksenii ATCC 33641</p><p id="p-339">Yersinia intermedia ATCC 29909</p><p id="p-340">Yersinia kristensenii ATCC 33638</p><p id="p-341">Yersinia mollaretii ATCC 43969</p><p id="p-342">Yersinia pestis A1122</p><p id="p-343">Yersinia pestis Antiqua</p><p id="p-344">Yersinia pestis KIM 10</p><p id="p-345">Yersinia pestis Pestoides F</p><p id="p-346">Yersinia pseudotuberculosis IP 31758</p><p id="p-347">Yersinia pseudotuberculosis IP 32953</p><p id="p-348">Yersinia pseudotuberculosis PB1</p><p id="p-349">Yersinia pseudotuberculosis YPIII</p><p id="p-350">Yersinia rohdei ATCC 43380</p><p id="p-351">Yersinia ruckeri ATCC 29473</p></div><div class="section app" id="app-2"><h2 class="">Appendix 2: Viral Database</h2><p id="p-352">Abaca bunchy top virus DNA-C</p><p id="p-353">Abaca bunchy top virus DNA-M</p><p id="p-354">Abaca bunchy top virus DNA-N</p><p id="p-355">Abaca bunchy top virus DNA-R</p><p id="p-356">Abaca bunchy top virus DNA-S</p><p id="p-357">Abaca bunchy top virus segment 2</p><p id="p-358">Abalone shriveling syndrome-associated virus</p><p id="p-359">Abutilon Brazil virus DNA A</p><p id="p-360">Abutilon Brazil virus DNA B</p><p id="p-361">Abutilon mosaic virus DNA A</p><p id="p-362">Abutilon mosaic virus DNA B</p><p id="p-363">Acanthocystis turfacea Chlorella virus 1</p><p id="p-364">Acheta domesticus densovirus</p><p id="p-365">Acholeplasma phage L2</p><p id="p-366">Acholeplasma phage MV-L1</p><p id="p-367">Acidianus bottle-shaped virus</p><p id="p-368">Acidianus filamentous virus 1</p><p id="p-369">Acidianus filamentous virus 2</p><p id="p-370">Acidianus filamentous virus 3</p><p id="p-371">Acidianus filamentous virus 6</p><p id="p-372">Acidianus filamentous virus 7</p><p id="p-373">Acidianus filamentous virus 8</p><p id="p-374">Acidianus filamentous virus 9</p><p id="p-375">Acidianus rod-shaped virus 1</p><p id="p-376">Acidianus spindle-shaped virus 1</p><p id="p-377">Acidianus two-tailed virus</p><p id="p-378">Actinomyces phage Av-1</p><p id="p-379">Actinoplanes phage phiAsp2</p><p id="p-380">Acyrthosiphon pisum bacteriophage APSE-1</p><p id="p-381">Adeno-associated virus-1</p><p id="p-382">Adeno-associated virus-2</p><p id="p-383">Adeno-associated virus-3</p><p id="p-384">Adeno-associated virus-4</p><p id="p-385">Adeno-associated virus 5</p><p id="p-386">Adeno-associated virus-7</p><p id="p-387">Adeno-associated virus-8</p><p id="p-388">Adoxophyes honmai NPV</p><p id="p-389">Adoxophyes orana granulovirus</p><p id="p-390">Adoxophyes orana nucleopolyhedrovirus</p><p id="p-391">Aedes aegypti densovirus</p><p id="p-392">Aedes albopictus densovirus</p><p id="p-393">Aedes taeniorhynchus iridescent virus</p><p id="p-394">Aeromonas phage 25</p><p id="p-395">Aeromonas phage 31</p><p id="p-396">Aeromonas phage 44RR2.8t</p><p id="p-397">Aeromonas phage Aeh1</p><p id="p-398">Aeromonas phage phiO18P african cassava mosaic virus DNA A african cassava mosaic virus DNA B</p><p id="p-399">African green monkey polyomavirus</p><p id="p-400">African swine fever virus</p><p id="p-401">Ageratum enation virus</p><p id="p-402">Ageratum leaf Cameroon betasatellite</p><p id="p-403">Ageratum leaf curl virus-G52</p><p id="p-404">Ageratum yellow vein China virus-associated DNA beta</p><p id="p-405">Ageratum yellow vein Chinavirus</p><p id="p-406">Ageratum yellow vein Hualian virus-TaiwanHsinchutom2003 DNA A</p><p id="p-407">Ageratum yellow vein Sri Lanka virus segment A</p><p id="p-408">Ageratum yellow vein Taiwan virus</p><p id="p-409">Ageratum yellow vein virusassociated DNA beta</p><p id="p-410">Ageratum yellow veinvirus</p><p id="p-411">Agrotis ipsilon multiple nucleopolyhedrovirus</p><p id="p-412">Agrotis segetum granulovirus</p><p id="p-413">Agrotis segetum nucleopolyhedrovirus</p><p id="p-414">Alcelaphine herpesvirus 1</p><p id="p-415">Aleutian mink disease virus</p><p id="p-416">Allamanda leaf curl virus DNA-A</p><p id="p-417">Alternanthera yellow vein virus DNA-A</p><p id="p-418">Alternanthera yellow vein virus satellite DNA beta</p><p id="p-419">Ambystoma tigrinum virus</p><p id="p-420">Amsacta moorei entomopoxvirus L</p><p id="p-421">Anguillid herpesvirus 1</p><p id="p-422">Anopheles gambiae densonucleosis virus</p><p id="p-423">Antheraea pernyi nucleopolyhedrovirus</p><p id="p-424">Anticarsia gemmatalis nucleopolyhedrovirus</p><p id="p-425">Archaeal BJ1 virus</p><p id="p-426">Ateline herpesvirus 3</p><p id="p-427">Autographa californica nucleopolyhedrovirus</p><p id="p-428">Avian adeno-associated virus</p><p id="p-429">ATCC VR-865</p><p id="p-430">Avian adeno-associated virus strain DA-1</p><p id="p-431">Avian endogenous retrovirus EAV-HP</p><p id="p-432">Azospirillum phage Cd</p><p id="p-433">Bacillus phage 0305phi8-36</p><p id="p-434">Bacillus phage AP50</p><p id="p-435">Bacillus phage B103</p><p id="p-436">Bacillus phage Bam35c</p><p id="p-437">Bacillus phage BCJA1c</p><p id="p-438">Bacillus phage Cherry</p><p id="p-439">Bacillus phage Fah</p><p id="p-440">Bacillus phage GA-1</p><p id="p-441">Bacillus phage Gamma</p><p id="p-442">Bacillus phage GIL16c</p><p id="p-443">Bacillus phage IEBH</p><p id="p-444">Bacillus phage phi105</p><p id="p-445">Bacillus phage phi29</p><p id="p-446">Bacillus phage SPBc2</p><p id="p-447">Bacillus phage SPO1</p><p id="p-448">Bacillus phage SPP1</p><p id="p-449">Bacillus phage TP21-L</p><p id="p-450">Bacillus phage WBeta</p><p id="p-451">Bacillus prophage phBC6A51</p><p id="p-452">Bacillus prophage phBC6A52</p><p id="p-453">Bacillus virus 1</p><p id="p-454">Bacteriophage Aaphi23</p><p id="p-455">Bacteriophage APSE-2</p><p id="p-456">Bacteriophage PSA</p><p id="p-457">Bacteriophage RB32</p><p id="p-458">Bacteroides phage B40-8</p><p id="p-459">Banana bunchy top virus DNA C</p><p id="p-460">Banana bunchy top virus DNA M</p><p id="p-461">Banana bunchy top virus DNA N</p><p id="p-462">Banana bunchy top virus DNA R</p><p id="p-463">Banana bunchy top virus DNA S</p><p id="p-464">Banana bunchy top virus DNA U3</p><p id="p-465">Banana streak GF virus</p><p id="p-466">Banana streak Mysore virus</p><p id="p-467">Banana streak OL virus</p><p id="p-468">Banana streak virus genome</p><p id="p-469">Banana streak virus strain Acuminata Vietnam</p><p id="p-470">Bandicoot papillomatosis carcinomatosis virus type 1</p><p id="p-471">Bandicoot papillomatosis carcinomatosis virus type 2</p><p id="p-472">Bat adeno-associated virus YNM</p><p id="p-473">Bdellovibrio phage phiMH2K</p><p id="p-474">Beak and feather disease virus</p><p id="p-475">Bean calico mosaic virus DNA A</p><p id="p-476">Bean calico mosaic virus DNA B</p><p id="p-477">Bean dwarf mosaic virus DNA A</p><p id="p-478">Bean dwarf mosaic virus DNA B</p><p id="p-479">Bean golden mosaic virus DNA A</p><p id="p-480">Bean golden mosaic virus DNA B</p><p id="p-481">Bean golden yellow mosaic virus DNA A</p><p id="p-482">Bean golden yellow mosaic virus DNA B</p><p id="p-483">Bean yellow dwarf virus putative genes V1</p><p id="p-484">Beet curly top Iran virus-K</p><p id="p-485">Beet curly top virus-California Logan</p><p id="p-486">Beet mild curly top virus-Worland4</p><p id="p-487">Beet severe curly top virus-Cfh</p><p id="p-488">Begomovirus-associated DNA II</p><p id="p-489">Begomovirus-associated DNA-III</p><p id="p-490">Bettongia penicillata papillomavirus 1</p><p id="p-491">Bhendi yellow vein</p><p id="p-492">Bhubhaneswar virus DNA-A</p><p id="p-493">Bhendi yellow vein Delhi virus 2004New Delhi DNA-A</p><p id="p-494">Bhendi yellow vein mosaic virus-associated DNA beta</p><p id="p-495">Bhendi yellow vein mosaic Virus</p><p id="p-496">Bitter gourd leaf curl disease-associated DNA beta</p><p id="p-497">BK polyomavirus</p><p id="p-498">Blainvillea yellow spot virus DNA-A</p><p id="p-499">Blainvillea yellow spot virus DNA-B</p><p id="p-500">Blattella germanica densovirus</p><p id="p-501">Blueberry red ringspot virus</p><p id="p-502">Bocavirus gorillaGBoV12009</p><p id="p-503">Bombyx mandarina nucleopolyhedrovirus</p><p id="p-504">Bombyx mori densovirus 5</p><p id="p-505">Bombyx mori NPV</p><p id="p-506">Bordetella phage BIP-1</p><p id="p-507">Bordetella phage BMP-1</p><p id="p-508">Bordetella phage BPP-1</p><p id="p-509">Bougainvillea spectabilis chlorotic vein-banding virus</p><p id="p-510">Bovine adeno-associated virus</p><p id="p-511">Bovine adenovirus A</p><p id="p-512">Bovine adenovirus B</p><p id="p-513">Bovine adenovirus D</p><p id="p-514">Bovine ephemeral fever virus</p><p id="p-515">Bovine foamy virus</p><p id="p-516">Bovine herpesvirus 1</p><p id="p-517">Bovine herpesvirus 4 long unique region</p><p id="p-518">Bovine herpesvirus 5</p><p id="p-519">Bovine papillomavirus-1</p><p id="p-520">Bovine papillomavirus 3</p><p id="p-521">Bovine papular stomatitis virus</p><p id="p-522">Bovine parvovirus 2</p><p id="p-523">Bovine Parvovirus</p><p id="p-524">Bovine polyomavirus</p><p id="p-525">Burkholderia ambifaria phage BcepF1</p><p id="p-526">Burkholderia phage Bcep 176</p><p id="p-527">Burkholderia phage Bcep1</p><p id="p-528">Burkholderia phage Bcep22</p><p id="p-529">Burkholderia phage Bcep43</p><p id="p-530">Burkholderia phage Bcep781</p><p id="p-531">Burkholderia phage BcepB1A</p><p id="p-532">Burkholderia phage BcepC6B</p><p id="p-533">Burkholderia phage BcepGomr</p><p id="p-534">Burkholderia phage BcepIL02</p><p id="p-535">Burkholderia phage BcepMu</p><p id="p-536">Burkholderia phage BcepNazgul</p><p id="p-537">Burkholderia phage BcepNY3</p><p id="p-538">Burkholderia phage KS10</p><p id="p-539">Burkholderia phage KS9</p><p id="p-540">Burkholderia phage phi1026b</p><p id="p-541">Burkholderia phage phi644-2 chromosome</p><p id="p-542">Burkholderia phage phiE12-2 chromosome</p><p id="p-543">Burkholderia phage phiE125</p><p id="p-544">Burkholderia phage phiE202 chromosome</p><p id="p-545">Burkholderia phage phiE255 chromosome</p><p id="p-546">Burkholderia prophage phi52237</p><p id="p-547">Cabbage leaf curl virus DNA A</p><p id="p-548">Cabbage leaf curl virus DNA B</p><p id="p-549">Cacao swollen shoot virus</p><p id="p-550">California sea lion anellovirus</p><p id="p-551">California sea lion polyomavirus 1</p><p id="p-552">Callitrichine herpesvirus 3 strain CJ0149</p><p id="p-553">Camelpox virus</p><p id="p-554">Campoletis sonorensis ichnovirus chromosome segment W</p><p id="p-555">Campoletis sonorensis ichnovirus segment B</p><p id="p-556">Campoletis sonorensis ichnovirus segment C</p><p id="p-557">Campoletis sonorensis ichnovirus segment D</p><p id="p-558">Campoletis sonorensis ichnovirus segment E</p><p id="p-559">Campoletis sonorensis ichnovirus segment F</p><p id="p-560">Campoletis sonorensis ichnovirus segmentG2</p><p id="p-561">Campoletis sonorensis ichnovirus segment G</p><p id="p-562">Campoletis sonorensis ichnovirus segment H</p><p id="p-563">Campoletis sonorensis ichnovirus segmentI2</p><p id="p-564">Campoletis sonorensis ichnovirus segment I</p><p id="p-565">Campoletis sonorensis ichnovirus segment J</p><p id="p-566">Campoletis sonorensis ichnovirus segment L</p><p id="p-567">Campoletis sonorensis ichnovirus segment M</p><p id="p-568">Campoletis sonorensis ichnovirus segment N</p><p id="p-569">Campoletis sonorensis ichnovirus segment O1</p><p id="p-570">Campoletis sonorensis ichnovirus segment P</p><p id="p-571">Campoletis sonorensis ichnovirus segment Q</p><p id="p-572">Campoletis sonorensis ichnovirus segment T</p><p id="p-573">Campoletis sonorensis ichnovirus segment U</p><p id="p-574">Campoletis sonorensis ichnovirus segment V</p><p id="p-575">Campoletis sonorensis ichnovirus segment Z</p><p id="p-576">Campoletis sonorensis ichnovirus superhelical segment A</p><p id="p-577">Campoletis sonorensis ichnovirus superhelical segment aprime</p><p id="p-578">Canary circovirus</p><p id="p-579">Canarypox virus</p><p id="p-580">Canine adenovirus Canine minute virus</p><p id="p-581">Canine oral papillomavirus</p><p id="p-582">Canine papillomavirus 2</p><p id="p-583">Canine papillomavirus 3</p><p id="p-584">Canine papillomavirus 4</p><p id="p-585">Canine papillomavirus 6</p><p id="p-586">Canine parvovirus</p><p id="p-587">Capra hircus papillomavirus type 1</p><p id="p-588">Cardiospermum yellow leaf curl virus satellite DNA beta</p><p id="p-589">Caretta caretta papillomavirus 1</p><p id="p-590">Carnation etched ring virus</p><p id="p-591">Casphalia extranea densovirus</p><p id="p-592">Cassava vein mosaic virus</p><p id="p-593">Cauliflower mosaic virus</p><p id="p-594">Cercopithecine herpesvirus 2</p><p id="p-595">Cercopithecine herpesvirus 5 strain 2715</p><p id="p-596">Cercopithecine herpesvirus 9</p><p id="p-597">Cestrum yellow leaf curling virus</p><p id="p-598">Chaetoceros salsugineum DNA virus</p><p id="p-599">Chayote yellow mosaic virus</p><p id="p-600">Chicken anemia virus</p><p id="p-601">Chickpea chlorotic dwarf virus</p><p id="p-602">Chilli leaf curl disease associated sequence virion</p><p id="p-603">Chilli leaf curl Multan alphasatellite</p><p id="p-604">Chilli leaf curl virus</p><p id="p-605">Chino del tomate virus DNA A</p><p id="p-606">Chino del tomate virus DNA B</p><p id="p-607">Chlamydia phage 3</p><p id="p-608">Chlamydia phage 4</p><p id="p-609">Chlamydia phage Chp1</p><p id="p-610">Chlamydia phage Chp2</p><p id="p-611">Chlamydia phage PhiCPG1</p><p id="p-612">Chloris striate mosaic virus</p><p id="p-613">Choristoneura fumiferana DEF MNPV</p><p id="p-614">Choristoneura fumiferana MNPV</p><p id="p-615">Choristoneura occidentalis granulovirus</p><p id="p-616">Chrysodeixis chalcites nucleopolyhedrovirus</p><p id="p-617">Circovirus-like genome BBC-A</p><p id="p-618">Circovirus-like genome CB-A</p><p id="p-619">Circovirus-like genome CB-B</p><p id="p-620">Circovirus-like genome RW-A</p><p id="p-621">Circovirus-like genome RW-B</p><p id="p-622">Circovirus-like genome RW-C</p><p id="p-623">Circovirus-like genome RW-D</p><p id="p-624">Circovirus-like genome RW-E</p><p id="p-625">Circovirus-like genome SAR-A</p><p id="p-626">Circovirus-like genome SAR-B</p><p id="p-627">Citrus psorosis virus RNA1</p><p id="p-628">Citrus psorosis virus RNA2</p><p id="p-629">Citrus psorosis virus RNA3</p><p id="p-630">Citrus yellow mosaic virus</p><p id="p-631">Clanis bilineata nucleopolyhedrosis virus</p><p id="p-632">Clavibacter phage CMP1</p><p id="p-633">Clerodendron yellow mosaic virus</p><p id="p-634">Clerodendrum golden mosaic</p><p id="p-635">China virus DNA A</p><p id="p-636">Clerodendrum golden mosaic</p><p id="p-637">China virus DNA B</p><p id="p-638">Clerodendrum golden mosaic virus DNA-A</p><p id="p-639">Clerodendrum golden mosaic virus DNA-B</p><p id="p-640">Clostridium phage 39-O</p><p id="p-641">Clostridium phage c-st</p><p id="p-642">Clostridium phage phi3626</p><p id="p-643">Clostridium phage phiC2</p><p id="p-644">Clostridium phage phi CD119</p><p id="p-645">Clostridium phage phiCD27</p><p id="p-646">Clostridium phage phiCTP1</p><p id="p-647">Coconut foliar decay virus</p><p id="p-648">Columbid circovirus</p><p id="p-649">Commelina yellow mottle virus</p><p id="p-650">Corchorus golden mosaic virus DNA-A</p><p id="p-651">Corchorus golden mosaic virus DNA-B</p><p id="p-652">Corchorus yellow spot virus DNA A</p><p id="p-653">Corchorus yellow spot virus DNA B</p><p id="p-654">Corchorus yellow vein virus-Hoa Binh DNA A</p><p id="p-655">Corchorus yellow vein virus-Hoa Binh DNA B</p><p id="p-656">Corynebacterium phage BFK20</p><p id="p-657">Corynebacterium phage P1201</p><p id="p-658">Cotesia congregata bracovirus segment Circle10</p><p id="p-659">Cotesia congregata bracovirus segment Circle11</p><p id="p-660">Cotesia congregata bracovirus segment Circle12</p><p id="p-661">Cotesia congregata bracovirus segment Circle13</p><p id="p-662">Cotesia congregata bracovirus segment Circle14</p><p id="p-663">Cotesia congregata bracovirus segment Circle15</p><p id="p-664">Cotesia congregata bracovirus segment Circle17</p><p id="p-665">Cotesia congregata bracovirus segment Circle19</p><p id="p-666">Cotesia congregata bracovirus segment circle1</p><p id="p-667">Cotesia congregata bracovirus segment Circle20</p><p id="p-668">Cotesia congregata bracovirus segment Circle21</p><p id="p-669">Cotesia congregata bracovirus segment Circle22</p><p id="p-670">Cotesia congregata bracovirus segment Circle23</p><p id="p-671">Cotesia congregata bracovirus segment Circle25</p><p id="p-672">Cotesia congregata bracovirus segment Circle26</p><p id="p-673">Cotesia congregata bracovirus segment circle2</p><p id="p-674">Cotesia congregata bracovirus segment Circle30</p><p id="p-675">Cotesia congregata bracovirus segment Circle31</p><p id="p-676">Cotesia congregata bracovirus segment Circle32</p><p id="p-677">Cotesia congregata bracovirus segment Circle33</p><p id="p-678">Cotesia congregata bracovirus segment Circle35</p><p id="p-679">Cotesia congregata bracovirus segment Circle36</p><p id="p-680">Cotesia congregata bracovirus segment circle3</p><p id="p-681">Cotesia congregata bracovirus segment Circle4</p><p id="p-682">Cotesia congregata bracovirus segment Circle5</p><p id="p-683">Cotesia congregata bracovirus segment Circle9</p><p id="p-684">Cotesia congregata virus segment Circle18</p><p id="p-685">Cotesia congregata virus segment Circle6</p><p id="p-686">Cotesia congregata virus segment Circle7</p><p id="p-687">Cotesia congregata virus segment Circle8</p><p id="p-688">Cotton leaf crumple geminivirus DNA B</p><p id="p-689">Cotton leaf crumple virus DNA A</p><p id="p-690">Cotton leaf curl Alabad virus</p><p id="p-691">Cotton leaf curl Bangalore virus-associated DNA beta</p><p id="p-692">Cotton leaf curl Bangalore virus segment A</p><p id="p-693">Cotton leaf curl Burewala alphasatellite</p><p id="p-694">Cotton leaf curl Burewala betasatellite</p><p id="p-695">Cotton leaf curl Burewala virus-IndiaVehari2004</p><p id="p-696">Cotton leaf curl Gezira alphasatellite</p><p id="p-697">Cotton leaf curl Gezira beta</p><p id="p-698">Cotton leaf curl Gezira Betasatellite extrachromosomal</p><p id="p-699">Cotton leaf curl Gezira virus</p><p id="p-700">Cotton leaf curl Kokhran virus</p><p id="p-701">Cotton leaf curl Multan Virus</p><p id="p-702">Cotton leaf curl Multan virus satellite DNA beta</p><p id="p-703">Cotton leaf curl Multan virus satellite U36-1</p><p id="p-704">Cotton leaf curl Rajasthan virus segment A</p><p id="p-705">Cotton leaf curl virus-associated DNA beta</p><p id="p-706">Cottontail rabbit papillomavirus</p><p id="p-707">Cowpea severe leaf curl-associated DNA beta</p><p id="p-708">Cowpox virus Crassocephalum yellow vein virus-Jinghong</p><p id="p-709">Croton yellow vein mosaic alphasatellite</p><p id="p-710">Croton yellow vein mosaic virus</p><p id="p-711">Croton yellow vein mosaic Virus satellite DNA beta</p><p id="p-712">Croton yellow vein virus Crow polyomavirus</p><p id="p-713">Cryptophlebia leucotreta granulovirus</p><p id="p-714">Cucurbita yellow vein virus-associated DNA beta</p><p id="p-715">Cucurbit leaf crumple virus DNA A</p><p id="p-716">Cucurbit leaf crumple virus DNA B</p><p id="p-717">Culex nigripalpus NPV</p><p id="p-718">Culex pipiens densovirus Cyanophage PSS2</p><p id="p-719">Cyanophage Syn5 Cycad leaf necrosis virus</p><p id="p-720">Cydia pomonella granulovirus</p><p id="p-721">Cyprinid herpesvirus 3</p><p id="p-722">Deer papillomavirus</p><p id="p-723">Deerpox virus W-1170-84</p><p id="p-724">Deerpox virus W-848-83</p><p id="p-725">Deftia phage phiW-14</p><p id="p-726">Dendrolimus punctatus densovirus</p><p id="p-727">Desmodium leaf distortion virus DNA A</p><p id="p-728">Desmodium leaf distortion virus DNA B</p><p id="p-729">Diadromus pulchellus ascovirus 4a</p><p id="p-730">Diatraea saccharalis densovirus</p><p id="p-731">Dicliptera yellow mottle virus DNA A</p><p id="p-732">Dicliptera yellow mottle virus DNA B</p><p id="p-733">Digitaria streak virus</p><p id="p-734">Dioscorea bacilliform virus</p><p id="p-735">Dolichos yellow mosaic virus</p><p id="p-736">Dracaena mottle virus</p><p id="p-737">Duck adenovirus A</p><p id="p-738">Duck circovirus</p><p id="p-739">Duck hepatitis B virus</p><p id="p-740">East African cassava mosaic Cameroon virus DNA A</p><p id="p-741">East African cassava mosaic Cameroon virus DNA B</p><p id="p-742">East African cassava mosaic Kenya virus DNA A</p><p id="p-743">East African cassava mosaic Kenya virus DNA B</p><p id="p-744">East African cassava mosaic virus DNA A</p><p id="p-745">East African cassava mosaic virus DNA B</p><p id="p-746">East African cassava mosaic Zanzibar virus DNA-A</p><p id="p-747">East African cassava mosaic Zanzibar virus DNA B</p><p id="p-748">Ecotropis obliqua NPV</p><p id="p-749">Ectocarpus siliculosus virus 1</p><p id="p-750">Ectromelia virus</p><p id="p-751">Emiliania huxleyi virus 86</p><p id="p-752">Emilia yellow vein virus-associated DNA beta</p><p id="p-753">Emilia yellow vein virus-Fz1</p><p id="p-754">Enterobacteria phage 13a</p><p id="p-755">Enterobacteria phage 933W</p><p id="p-756">Enterobacteria phage alpha3</p><p id="p-757">Enterobacteria phage BA14</p><p id="p-758">Enterobacteria phage BP-4795</p><p id="p-759">Enterobacteria phage EcoDS1</p><p id="p-760">Enterobacteria phage EPS7</p><p id="p-761">Enterobacteria phage epsilon15</p><p id="p-762">Enterobacteria phage ES18</p><p id="p-763">Enterobacteria phage Felix 01</p><p id="p-764">Enterobacteria phage Fels-2</p><p id="p-765">Enterobacteria phage G4</p><p id="p-766">Enterobacteria phage HK022</p><p id="p-767">Enterobacteria phage HK620</p><p id="p-768">Enterobacteria phage HK97</p><p id="p-769">Enterobacteria phage I2-2</p><p id="p-770">Enterobacteria phage ID18</p><p id="p-771">Enterobacteria phage ID2 MoscowID2001</p><p id="p-772">Enterobacteria phage If1</p><p id="p-773">Enterobacteria phage Ike</p><p id="p-774">Enterobacteria phage IME08</p><p id="p-775">Enterobacteria phage JK06</p><p id="p-776">Enterobacteria phage JS10</p><p id="p-777">Enterobacteria phage JS98</p><p id="p-778">Enterobacteria phage JSE</p><p id="p-779">Enterobacteria phage K1-5</p><p id="p-780">Enterobacteria phage K1E</p><p id="p-781">Enterobacteria phage K1F</p><p id="p-782">Enterobacteria phage lambda</p><p id="p-783">Enterobacteria phage LKA1</p><p id="p-784">Enterobacteria phage M13</p><p id="p-785">Enterobacteria phage Min27</p><p id="p-786">Enterobacteria phage Mu</p><p id="p-787">Enterobacteria phage N15</p><p id="p-788">Enterobacteria phage N4</p><p id="p-789">Enterobacteria phage P1</p><p id="p-790">Enterobacteria phage P22 virus</p><p id="p-791">Enterobacteria phage P2 virus</p><p id="p-792">Enterobacteria phage P4</p><p id="p-793">Enterobacteria phage Phi1</p><p id="p-794">Enterobacteria phage phiEco32</p><p id="p-795">Enterobacteria phage phiEcoM-GJ1</p><p id="p-796">Enterobacteria phage phiP27</p><p id="p-797">Enterobacteria phage phiX174 sensu lato</p><p id="p-798">Enterobacteria phage PRD1</p><p id="p-799">Enterobacteria phage PsP3</p><p id="p-800">Enterobacteria phage RB14</p><p id="p-801">Enterobacteria phage RB16</p><p id="p-802">Enterobacteria phage RB43</p><p id="p-803">Enterobacteria phage RB49</p><p id="p-804">Enterobacteria phage RB51</p><p id="p-805">Enterobacteria phage RB69</p><p id="p-806">Enterobacteria phage RTP</p><p id="p-807">Enterobacteria phage Sf6</p><p id="p-808">Enterobacteria phage SfV</p><p id="p-809">Enterobacteria phage SP6</p><p id="p-810">Enterobacteria phage SSL-2009a</p><p id="p-811">Enterobacteria phage ST104</p><p id="p-812">Enterobacteria phage St-1</p><p id="p-813">Enterobacteria phage ST64T</p><p id="p-814">Enterobacteria phage T1</p><p id="p-815">Enterobacteria phage T3</p><p id="p-816">Enterobacteria phage T4</p><p id="p-817">Enterobacteria phage T5</p><p id="p-818">Enterobacteria phage T7</p><p id="p-819">Enterobacteria phage TLS</p><p id="p-820">Enterobacteria phage VT2-Sakai</p><p id="p-821">Enterobacteria phage WA13</p><p id="p-822">Enterobacteria phage WV8</p><p id="p-823">Enterobacteria phage YYZ-2008</p><p id="p-824">Enterococcus faecalis 62 chromosome</p><p id="p-825">Enterococcus faecalis 62 plasmid EF62pA</p><p id="p-826">Enterococcus faecalis 62 plasmid EF62pB</p><p id="p-827">Enterococcus faecalis 62 plasmid EF62pC</p><p id="p-828">Enterococcus phage EF62phi</p><p id="p-829">Enterococcus phage EFAP-1</p><p id="p-830">Enterococcus phage phiEf11</p><p id="p-831">Enterococcus phage phiEF24C</p><p id="p-832">Enterococcus phage phiFL1A</p><p id="p-833">Enterococcus phage phiFL2A</p><p id="p-834">Enterococcus phage phiFL3A</p><p id="p-835">Enterococcus phage phiFL4A</p><p id="p-836">Enzootic nasal tumour virus of goats</p><p id="p-837">Epiphyas postvittana NPV</p><p id="p-838">Equid herpesvirus 1</p><p id="p-839">Equid herpesvirus 2</p><p id="p-840">Equid herpesvirus 4</p><p id="p-841">Equid herpesvirus 9</p><p id="p-842">Equine papillomavirus 2</p><p id="p-843">Eragrostis curvula streak virus</p><p id="p-844">Eragrostis streak virus</p><p id="p-845">Erectites yellow mosaic virus DNA-A</p><p id="p-846">Erectites yellow mosaic virus satellite DNA beta</p><p id="p-847">Erethizon dorsatum papillomavirus type 1</p><p id="p-848">Erinaceus europaeus papillomavirus</p><p id="p-849">Erwinia amylovora phage Era103</p><p id="p-850">Erwinia phage phiEa21-4</p><p id="p-851">Escherichia coli bacteriophage rv5</p><p id="p-852">Escherichia phage D108</p><p id="p-853">Escherichia phage phiV10</p><p id="p-854">Eupatorium vein clearing virus</p><p id="p-855">Eupatorium yellow vein associated DNA beta</p><p id="p-856">Eupatorium yellow vein virus</p><p id="p-857">Euphorbia leaf curl virus DNA A</p><p id="p-858">Euphorbia yellow mosaic virus DNA A</p><p id="p-859">Euphorbia yellow mosaic virus DNA B</p><p id="p-860">Euproctis pseudoconspersa nucleopolyhedrovirus</p><p id="p-861">European elk papillomavirus</p><p id="p-862">Faba bean necrotic stunt virus DNA C</p><p id="p-863">Faba bean necrotic stunt virus DNA M</p><p id="p-864">Faba bean necrotic stunt virus DNA N</p><p id="p-865">Faba bean necrotic stunt virus DNA R</p><p id="p-866">Faba bean necrotic stunt virus DNA S</p><p id="p-867">Faba bean necrotic stunt virus DNA U1</p><p id="p-868">Faba bean necrotic stunt virus DNA U2</p><p id="p-869">Faba bean necrotic stunt virus DNA U4</p><p id="p-870">Faba bean necrotic yellows virus DNA 10</p><p id="p-871">Faba bean necrotic yellows virus DNA-1</p><p id="p-872">Faba bean necrotic yellows virus DNA 2</p><p id="p-873">Faba bean necrotic yellows virus DNA 4</p><p id="p-874">Faba bean necrotic yellows virus DNA 5</p><p id="p-875">Faba bean necrotic yellows virus DNA 7</p><p id="p-876">Faba bean necrotic yellows virus DNA 8</p><p id="p-877">Faba bean necrotic yellows virus DNA 9</p><p id="p-878">Faba bean necrotic yellows Virus</p><p id="p-879">Feldmannia species virus Felid herpesvirus 1</p><p id="p-880">Felis domesticus papillomavirus type 1</p><p id="p-881">Fenneropenaeus chinensis hepatopancreatic densovirus</p><p id="p-882">Figwort mosaic virus</p><p id="p-883">Finch circovirus Finch polyomavirus</p><p id="p-884">Flavobacterium phage 11b</p><p id="p-885">Fowl adenovirus A</p><p id="p-886">Fowl adenovirus D</p><p id="p-887">Fowlpox virus</p><p id="p-888">Francolinus leucoscepus papillomavirus 1</p><p id="p-889">Fringilla coelebs papillomavirus</p><p id="p-890">Frog adenovirus 1</p><p id="p-891">Frog virus 3</p><p id="p-892">Galleria mellonella densovirus</p><p id="p-893">Gallid herpesvirus 1</p><p id="p-894">Gallid herpesvirus 2</p><p id="p-895">Gallid herpesvirus 3</p><p id="p-896">Gammapapillomavirus HPV127</p><p id="p-897">Geobacillus phage</p><p id="p-898">GBSV1 Geobacillus virus E2</p><p id="p-899">Glossina pallidipes salivary gland hypertrophy virus</p><p id="p-900">Glypta fumiferanae ichnovirus segmentA10</p><p id="p-901">Glypta fumiferanae ichnovirus segment A1</p><p id="p-902">Glypta fumiferanae ichnovirus segment A2</p><p id="p-903">Glypta fumiferanae ichnovirus segment A3</p><p id="p-904">Glypta fumiferanae ichnovirus segment A4</p><p id="p-905">Glypta fumiferanae ichnovirus segment A5</p><p id="p-906">Glypta fumiferanae ichnovirus segment A6</p><p id="p-907">Glypta fumiferanae ichnovirus segment A7</p><p id="p-908">Glypta fumiferanae ichnovirus segment A8</p><p id="p-909">Glypta fumiferanae ichnovirus segment A9</p><p id="p-910">Glypta fumiferanae ichnovirus segment B10</p><p id="p-911">Glypta fumiferanae ichnovirus segment B11</p><p id="p-912">Glypta fumiferanae ichnovirus segment B12</p><p id="p-913">Glypta fumiferanae ichnovirus segment B13</p><p id="p-914">Glypta fumiferanae ichnovirus segment B14</p><p id="p-915">Glypta fumiferanae ichnovirus segment B15</p><p id="p-916">Glypta fumiferanae ichnovirus segment B16</p><p id="p-917">Glypta fumiferanae ichnovirus segment B17</p><p id="p-918">Glypta fumiferanae ichnovirus segment B18</p><p id="p-919">Glypta fumiferanae ichnovirus segment B19</p><p id="p-920">Glypta fumiferanae ichnovirus segment-B1</p><p id="p-921">Glypta fumiferanae ichnovirus segment B20</p><p id="p-922">Glypta fumiferanae ichnovirus segment B21</p><p id="p-923">Glypta fumiferanae ichnovirus segment B22</p><p id="p-924">Glypta fumiferanae ichnovirus segment B23</p><p id="p-925">Glypta fumiferanae ichnovirus segment B24</p><p id="p-926">Glypta fumiferanae ichnovirus segment B25</p><p id="p-927">Glypta fumiferanae ichnovirus segment B26</p><p id="p-928">Glypta fumiferanae ichnovirus segment B27</p><p id="p-929">Glypta fumiferanae ichnovirus segment B28</p><p id="p-930">Glypta fumiferanae ichnovirus segment B29</p><p id="p-931">Glypta fumiferanae ichnovirus segment-B2</p><p id="p-932">Glypta fumiferanae ichnovirus segment B30</p><p id="p-933">Glypta fumiferanae ichnovirus segment B31</p><p id="p-934">Glypta fumiferanae ichnovirus segment B32</p><p id="p-935">Glypta fumiferanae ichnovirus segment B33</p><p id="p-936">Glypta fumiferanae ichnovirus segment B34</p><p id="p-937">Glypta fumiferanae ichnovirus segment B35</p><p id="p-938">Glypta fumiferanae ichnovirus segment B36</p><p id="p-939">Glypta fumiferanae ichnovirus segment B37</p><p id="p-940">Glypta fumiferanae ichnovirus segment B38</p><p id="p-941">Glypta fumiferanae ichnovirus segment B39</p><p id="p-942">Glypta fumiferanae ichnovirus segment-B3</p><p id="p-943">Glypta fumiferanae ichnovirus segment B40</p><p id="p-944">Glypta fumiferanae ichnovirus segment B41</p><p id="p-945">Glypta fumiferanae ichnovirus segment B42</p><p id="p-946">Glypta fumiferanae ichnovirus segment B43</p><p id="p-947">Glypta fumiferanae ichnovirus segment B44</p><p id="p-948">Glypta fumiferanae ichnovirus segment B45</p><p id="p-949">Glypta fumiferanae ichnovirus segment B46</p><p id="p-950">Glypta fumiferanae ichnovirus segment B47</p><p id="p-951">Glypta fumiferanae ichnovirus segment B48</p><p id="p-952">Glypta fumiferanae ichnovirus segment B49</p><p id="p-953">Glypta fumiferanae ichnovirus segment-B4</p><p id="p-954">Glypta fumiferanae ichnovirus segment B50</p><p id="p-955">Glypta fumiferanae ichnovirus segment B51</p><p id="p-956">Glypta fumiferanae ichnovirus segment B52</p><p id="p-957">Glypta fumiferanae ichnovirus segment B53</p><p id="p-958">Glypta fumiferanae ichnovirus segment B54</p><p id="p-959">Glypta fumiferanae ichnovirus segment B55</p><p id="p-960">Glypta fumiferanae ichnovirus segment B56</p><p id="p-961">Glypta fumiferanae ichnovirus segment B57</p><p id="p-962">Glypta fumiferanae ichnovirus segment B58</p><p id="p-963">Glypta fumiferanae ichnovirus segment B59</p><p id="p-964">Glypta fumiferanae ichnovirus segment-B5</p><p id="p-965">Glypta fumiferanae ichnovirus segment B60</p><p id="p-966">Glypta fumiferanae ichnovirus segment B61</p><p id="p-967">Glypta fumiferanae ichnovirus segment B62</p><p id="p-968">Glypta fumiferanae ichnovirus segment B63</p><p id="p-969">Glypta fumiferanae ichnovirus segment B64</p><p id="p-970">Glypta fumiferanae ichnovirus segment B65</p><p id="p-971">Glypta fumiferanae ichnovirus segment-B6</p><p id="p-972">Glypta fumiferanae ichnovirus segment B7</p><p id="p-973">Glypta fumiferanae ichnovirus segment B8</p><p id="p-974">Glypta fumiferanae ichnovirus segment B9</p><p id="p-975">Glypta fumiferanae ichnovirus segment C10</p><p id="p-976">Glypta fumiferanae ichnovirus segment C11</p><p id="p-977">Glypta fumiferanae ichnovirus segment C12</p><p id="p-978">Glypta fumiferanae ichnovirus segment C13</p><p id="p-979">Glypta fumiferanae ichnovirus segment C14</p><p id="p-980">Glypta fumiferanae ichnovirus segment C15</p><p id="p-981">Glypta fumiferanae ichnovirus segment C16</p><p id="p-982">Glypta fumiferanae ichnovirus segment C17</p><p id="p-983">Glypta fumiferanae ichnovirus segment C18</p><p id="p-984">Glypta fumiferanae ichnovirus segment C19</p><p id="p-985">Glypta fumiferanae ichnovirus segment-C1</p><p id="p-986">Glypta fumiferanae ichnovirus segment C20</p><p id="p-987">Glypta fumiferanae ichnovirus segment C21</p><p id="p-988">Glypta fumiferanae ichnovirus segment C22</p><p id="p-989">Glypta fumiferanae ichnovirus segment-C2</p><p id="p-990">Glypta fumiferanae ichnovirus segment C3</p><p id="p-991">Glypta fumiferanae ichnovirus segment C4</p><p id="p-992">Glypta fumiferanae ichnovirus segment C5</p><p id="p-993">Glypta fumiferanae ichnovirus segment C6</p><p id="p-994">Glypta fumiferanae ichnovirus segment C7</p><p id="p-995">Glypta fumiferanae ichnovirus segment C8</p><p id="p-996">Glypta fumiferanae ichnovirus segment C9</p><p id="p-997">Glypta fumiferanae ichnovirus segment D1</p><p id="p-998">Glypta fumiferanae ichnovirus segment D2</p><p id="p-999">Glypta fumiferanae ichnovirus segment D3</p><p id="p-1000">Glypta fumiferanae ichnovirus segment D4</p><p id="p-1001">Glypta fumiferanae ichnovirus segment D5</p><p id="p-1002">Glypta fumiferanae ichnovirus segment D6</p><p id="p-1003">Glypta fumiferanae ichnovirus segment D7</p><p id="p-1004">Glypta fumiferanae ichnovirus segment E1</p><p id="p-1005">Goatpox virus Pellor Goose circovirus</p><p id="p-1006">Goose hemorrhagic polyomavirus</p><p id="p-1007">Goose parvovirus</p><p id="p-1008">Gossypium darwinii symptomless alphasatellite DNA-alpha</p><p id="p-1009">Gossypium darwinii symptomless virus DNA-A</p><p id="p-1010">Gossypium davidsonii symptomless alphasatellite DNA-alpha-B</p><p id="p-1011">Gossypium mustilinum symptomless alphasatellite DNA-alpha-B</p><p id="p-1012">Gossypium punctatum mild leaf curl virus DNA A</p><p id="p-1013">Gossypium punctatum mild leaf curl virus DNA B</p><p id="p-1014">Ground squirrel hepatitis virus</p><p id="p-1015">Gryllus bimaculatus nudivirus Gull circovirus</p><p id="p-1016">Haemophilus phage HP1</p><p id="p-1017">Haemophilus phage HP2</p><p id="p-1018">Haloarcula hispanica pleomorphic virus 1</p><p id="p-1019">Haloarcula phage SH1</p><p id="p-1020">Halomonas phage phiHAP-1</p><p id="p-1021">Halorubrum phage HF2</p><p id="p-1022">Halorubrum pleomorphic virus 1</p><p id="p-1023">Halovirus HF1</p><p id="p-1024">Hamster polyomavirus</p><p id="p-1025">Helicoverpa armigera granulovirus</p><p id="p-1026">Helicoverpa armigera multiple nucleopolyhedrovirus</p><p id="p-1027">Helicoverpa armigera-NPV</p><p id="p-1028">Helicoverpa armigera NPV NNg1</p><p id="p-1029">Helicoverpa armigera nucleopolyhedrovirus G4</p><p id="p-1030">Helicoverpa zea SNPV</p><p id="p-1031">Heliothis virescens ascovirus 3e</p><p id="p-1032">Hepatitis B virus</p><p id="p-1033">Heron hepatitis B virus</p><p id="p-1034">His1 virus</p><p id="p-1035">His2 virus</p><p id="p-1036">Hollyhock leaf crumple virus</p><p id="p-1037">Honeysuckle yellow vein beta-JapanFukui2001</p><p id="p-1038">Honeysuckle yellow vein mosaic beta-JapanMiyizaki2001</p><p id="p-1039">Honeysuckle yellow vein mosaic disease associated satellite DNA beta-Ibaraki</p><p id="p-1040">Honeysuckle yellow vein mosaic Virus</p><p id="p-1041">Honeysuckle yellow vein mosaic virus-Kagoshima</p><p id="p-1042">Honeysuckle yellow vein mosaic virus satellite DNA beta</p><p id="p-1043">Honeysuckle yellow vein virus-UK1</p><p id="p-1044">Horsegram yellow mosaic Virus DNA B</p><p id="p-1045">Horsegram yellow mosaic virus</p><p id="p-1046">Horseradish curly top virus</p><p id="p-1047">Human adenovirus 54</p><p id="p-1048">Human adenovirus A</p><p id="p-1049">Human adenovirus B1</p><p id="p-1050">Human adenovirus B2</p><p id="p-1051">Human adenovirus C</p><p id="p-1052">Human adenovirus D</p><p id="p-1053">Human adenovirus E</p><p id="p-1054">Human adenovirus F</p><p id="p-1055">Human bocavirus 1</p><p id="p-1056">Human bocavirus 2</p><p id="p-1057">Human bocavirus 3</p><p id="p-1058">Human bocavirus 4</p><p id="p-1059">Human erythrovirus V9</p><p id="p-1060">Human herpesvirus 1</p><p id="p-1061">Human herpesvirus 2</p><p id="p-1062">Human herpesvirus 3</p><p id="p-1063">Human herpesvirus 4</p><p id="p-1064">Human herpesvirus 5 strain Merlin</p><p id="p-1065">Human herpesvirus 7</p><p id="p-1066">Human herpesvirus 8</p><p id="p-1067">Human papillomavirus-18</p><p id="p-1068">Human papillomavirus 1</p><p id="p-1069">Human papillomavirus-2</p><p id="p-1070">Human papillomavirus 54</p><p id="p-1071">Human papillomavirus-5</p><p id="p-1072">Human papillomavirus type 101</p><p id="p-1073">Human papillomavirus type 103</p><p id="p-1074">Human papillomavirus type 108</p><p id="p-1075">Human papillomavirus type-10</p><p id="p-1076">Human papillomavirus type 16</p><p id="p-1077">Human papillomavirus type 26</p><p id="p-1078">Human papillomavirus type 32</p><p id="p-1079">Human papillomavirus type 34</p><p id="p-1080">Human papillomavirus type 41</p><p id="p-1081">Human papillomavirus type 48</p><p id="p-1082">Human papillomavirus type 49</p><p id="p-1083">Human papillomavirus type-4</p><p id="p-1084">Human papillomavirus type 50</p><p id="p-1085">Human papillomavirus type 53</p><p id="p-1086">Human papillomavirus type 60</p><p id="p-1087">Human papillomavirus type 63</p><p id="p-1088">Human papillomavirus type 6b</p><p id="p-1089">Human papillomavirus type 7</p><p id="p-1090">Human papillomavirus type 88</p><p id="p-1091">Human papillomavirus type 90</p><p id="p-1092">Human papillomavirus type 92</p><p id="p-1093">Human papillomavirus type 96</p><p id="p-1094">Human papillomavirus type-9</p><p id="p-1095">Human parvovirus B19</p><p id="p-1096">Human T-lymphotropic virus 1</p><p id="p-1097">Human T-lymphotropic virus 4</p><p id="p-1098">Hyperthermophilic Archaeal Virus 1</p><p id="p-1099">Hyperthermophilic Archaeal Virus 2</p><p id="p-1100">Hyphantria cunea nucleopolyhedrovirus</p><p id="p-1101">Hyposoter fugitivus ichnovirus segment A1</p><p id="p-1102">Hyposoter fugitivus ichnovirus segment A2</p><p id="p-1103">Hyposoter fugitivus ichnovirus segment A3</p><p id="p-1104">Hyposoter fugitivus ichnovirus segment B10</p><p id="p-1105">Hyposoter fugitivus ichnovirus segment B11</p><p id="p-1106">Hyposoter fugitivus ichnovirus segment B12</p><p id="p-1107">Hyposoter fugitivus ichnovirus segment B13</p><p id="p-1108">Hyposoter fugitivus ichnovirus segment B14</p><p id="p-1109">Hyposoter fugitivus ichnovirus segment B15</p><p id="p-1110">Hyposoter fugitivus ichnovirus segment B16</p><p id="p-1111">Hyposoter fugitivus ichnovirus segment B17</p><p id="p-1112">Hyposoter fugitivus ichnovirus segment B18</p><p id="p-1113">Hyposoter fugitivus ichnovirus segment-B1</p><p id="p-1114">Hyposoter fugitivus ichnovirus segment B2</p><p id="p-1115">Hyposoter fugitivus ichnovirus segment B3</p><p id="p-1116">Hyposoter fugitivus ichnovirus segment B4</p><p id="p-1117">Hyposoter fugitivus ichnovirus segment B5</p><p id="p-1118">Hyposoter fugitivus ichnovirus segment B6</p><p id="p-1119">Hyposoter fugitivus ichnovirus segment B7</p><p id="p-1120">Hyposoter fugitivus ichnovirus segment B8</p><p id="p-1121">Hyposoter fugitivus ichnovirus segment B9</p><p id="p-1122">Hyposoter fugitivus ichnovirus segment C10</p><p id="p-1123">Hyposoter fugitivus ichnovirus segment C11</p><p id="p-1124">Hyposoter fugitivus ichnovirus segment C12</p><p id="p-1125">Hyposoter fugitivus ichnovirus segment C13</p><p id="p-1126">Hyposoter fugitivus ichnovirus segment C14</p><p id="p-1127">Hyposoter fugitivus ichnovirus segment C15</p><p id="p-1128">Hyposoter fugitivus ichnovirus segment C16</p><p id="p-1129">Hyposoter fugitivus ichnovirus segment C17</p><p id="p-1130">Hyposoter fugitivus ichnovirus segment C18</p><p id="p-1131">Hyposoter fugitivus ichnovirus segment C19</p><p id="p-1132">Hyposoter fugitivus ichnovirus segment-C1</p><p id="p-1133">Hyposoter fugitivus ichnovirus segment C20</p><p id="p-1134">Hyposoter fugitivus ichnovirus segment-C2</p><p id="p-1135">Hyposoter fugitivus ichnovirus segment C3</p><p id="p-1136">Hyposoter fugitivus ichnovirus segment C4</p><p id="p-1137">Hyposoter fugitivus ichnovirus segment C5</p><p id="p-1138">Hyposoter fugitivus ichnovirus segment C6</p><p id="p-1139">Hyposoter fugitivus ichnovirus segment C7</p><p id="p-1140">Hyposoter fugitivus ichnovirus segment C8</p><p id="p-1141">Hyposoter fugitivus ichnovirus segment C9</p><p id="p-1142">Hyposoter fugitivus ichnovirus segment D10</p><p id="p-1143">Hyposoter fugitivus ichnovirus segment D11</p><p id="p-1144">Hyposoter fugitivus ichnovirus segment D12</p><p id="p-1145">Hyposoter fugitivus ichnovirus segment-D1</p><p id="p-1146">Hyposoter fugitivus ichnovirus segment D2</p><p id="p-1147">Hyposoter fugitivus ichnovirus segment D3</p><p id="p-1148">Hyposoter fugitivus ichnovirus segment D4</p><p id="p-1149">Hyposoter fugitivus ichnovirus segment D5</p><p id="p-1150">Hyposoter fugitivus ichnovirus segment D6</p><p id="p-1151">Hyposoter fugitivus ichnovirus segment D7</p><p id="p-1152">Hyposoter fugitivus ichnovirus segment D8</p><p id="p-1153">Hyposoter fugitivus ichnovirus segment D9</p><p id="p-1154">Hyposoter fugitivus ichnovirus segment E1</p><p id="p-1155">Hyposoter fugitivus ichnovirus segment E2</p><p id="p-1156">Hyposoter fugitivus ichnovirus segment G1</p><p id="p-1157">Ictalurid herpesvirus 1 strain Auburn 1</p><p id="p-1158">Indian cassava mosaic virus DNA A</p><p id="p-1159">Indian cassava mosaic virus DNA B</p><p id="p-1160">Infectious hypodermal and hematopoietic necrosis virus</p><p id="p-1161">Infectious spleen and kidney necrosis virus</p><p id="p-1162">Invertebrate iridescent virus 6</p><p id="p-1163">Iodobacteriophage phiPLPE</p><p id="p-1164">Ipomoea yellow vein virus</p><p id="p-1165">Jatropha leaf curl virus DNA A</p><p id="p-1166">Jatropha yellow mosaic India virus DNA-A</p><p id="p-1167">JC polyomavirus</p><p id="p-1168">Junonia coenia densovirus</p><p id="p-1169">Kalanchoe top-spotting virus</p><p id="p-1170">Kenaf leaf curl virus DNA A</p><p id="p-1171">KI polyomavirus Stockholm 60</p><p id="p-1172">Klebsiella phage KP15</p><p id="p-1173">Klebsiella phage KP32</p><p id="p-1174">Klebsiella phage KP34</p><p id="p-1175">Klebsiella phage phiKO2</p><p id="p-1176">Kluyvera phage Kvp1</p><p id="p-1177">Kudzu mosaic virus DNA-A</p><p id="p-1178">Kudzu mosaic virus DNA-B</p><p id="p-1179">Lactobacillus johnsonii prophage Lj771</p><p id="p-1180">Lactobacillus phage A2</p><p id="p-1181">Lactobacillus phage KC5a</p><p id="p-1182">Lactobacillus phage Lb338-1</p><p id="p-1183">Lactobacillus phage Lc-Nu</p><p id="p-1184">Lactobacillus phage LL-H</p><p id="p-1185">Lactobacillus phage LP65</p><p id="p-1186">Lactobacillus phage Lrm1</p><p id="p-1187">Lactobacillus phage Lv-1</p><p id="p-1188">Lactobacillus phage phiAT3</p><p id="p-1189">Lactobacillus phage phig1e</p><p id="p-1190">Lactobacillus phage phiJL-1</p><p id="p-1191">Lactobacillus prophage Lj928</p><p id="p-1192">Lactobacillus prophage Lj965</p><p id="p-1193">Lactobacillus prophage phiadh</p><p id="p-1194">Lactococcus phage 1706</p><p id="p-1195">Lactococcus phage 4268</p><p id="p-1196">Lactococcus phage 712</p><p id="p-1197">Lactococcus phage asccphi28</p><p id="p-1198">Lactococcus phage bIBB29</p><p id="p-1199">Lactococcus phage bIL170</p><p id="p-1200">Lactococcus phage bIL67</p><p id="p-1201">Lactococcus phage BK5-T</p><p id="p-1202">Lactococcus phage c2</p><p id="p-1203">Lactococcus phage jj50</p><p id="p-1204">Lactococcus phage KSY1</p><p id="p-1205">Lactococcus phage P008</p><p id="p-1206">Lactococcus phage P087</p><p id="p-1207">Lactococcus phage phiLC3</p><p id="p-1208">Lactococcus phage Q54</p><p id="p-1209">Lactococcus phage r1t</p><p id="p-1210">Lactococcus phage sk1</p><p id="p-1211">Lactococcus phage TP901-1</p><p id="p-1212">Lactococcus phage Tuc2009</p><p id="p-1213">Lactococcus phage ul36</p><p id="p-1214">Lactococcus prophage bIL285</p><p id="p-1215">Lactococcus prophage bIL286</p><p id="p-1216">Lactococcus prophage bIL309</p><p id="p-1217">Lactococcus prophage bIL310</p><p id="p-1218">Lactococcus prophage bIL311</p><p id="p-1219">Lactococcus prophage bIL312</p><p id="p-1220">Lamium leaf distortion associated virus</p><p id="p-1221">Leucania separata nuclear polyhedrosis virus</p><p id="p-1222">Leucas zeylanica yellow vein virus satellite DNA beta</p><p id="p-1223">Lindernia anagallis yellow vein virus DNA-A</p><p id="p-1224">Lindernia anagallis yellow vein virus satellite DNA beta</p><p id="p-1225">Listeria phage A006</p><p id="p-1226">Listeria phage A118</p><p id="p-1227">Listeria phage A500</p><p id="p-1228">Listeria phage A511</p><p id="p-1229">Listeria phage B025</p><p id="p-1230">Listeria phage B054</p><p id="p-1231">Listeria phage P35</p><p id="p-1232">Listeria phage P40</p><p id="p-1233">Listonella phage phiHSIC</p><p id="p-1234">Loofa yellow mosaic virus DNA A</p><p id="p-1235">Loofa yellow mosaic virus DNA B</p><p id="p-1236">Lucky bamboo bacilliform virus</p><p id="p-1237">Ludwigia leaf distortion betasatellite IndiaAmadalavalasaHibiscus2 007</p><p id="p-1238">Ludwigia yellow vein virus-associated DNA beta</p><p id="p-1239">Ludwigia yellow vein virus DNA-A</p><p id="p-1240">Luffa begomovirus associated DNA beta</p><p id="p-1241">Luffa puckering and leaf distortion-associated DNA beta</p><p id="p-1242">LuIII virus</p><p id="p-1243">Lumpy skin disease virus NI-2490</p><p id="p-1244">Lymantria dispar MNPV</p><p id="p-1245">Lymantria xylina MNPV</p><p id="p-1246">Lymphocystis disease virus 1</p><p id="p-1247">Lymphocystis disease virus-isolate China</p><p id="p-1248">Macacine herpesvirus 1</p><p id="p-1249">Macacine herpesvirus 3</p><p id="p-1250">Macacine herpesvirus 4</p><p id="p-1251">Macacine herpesvirus 5</p><p id="p-1252">Macaque simian foamy virus</p><p id="p-1253">Macroptilium golden mosaic virus-</p><p id="p-1254">JamaicaWissadulaAugust Town DNA A</p><p id="p-1255">Macroptilium golden mosaic virus-</p><p id="p-1256">JamaicaWissadulaAugust Town DNA B</p><p id="p-1257">Macroptilium mosaic Puerto Rico virus DNA A</p><p id="p-1258">Macroptilium mosaic Puerto Rico virus DNA B</p><p id="p-1259">Macroptilium yellow mosaic Florida virus DNA A</p><p id="p-1260">Macroptilium yellow mosaic Florida virus DNA B</p><p id="p-1261">Macroptilium yellow mosaic virus DNA A</p><p id="p-1262">Macroptilium yellow mosaic virus DNA B</p><p id="p-1263">Maize streak virus-ASouth Africa</p><p id="p-1264">Malachra yellow vein mosaic virus-associated satellite DNA beta</p><p id="p-1265">Mal de Rio Cuarto virus segment 9</p><p id="p-1266">Malvastrum leaf curl Guangdong virus</p><p id="p-1267">Malvastrum leaf curl virus-associated defective DNA beta</p><p id="p-1268">Malvastrum leaf curl virus-G87</p><p id="p-1269">Malvastrum yellow mosaic virus-associated DNA 1</p><p id="p-1270">Malvastrum yellow mosaic virus DNA-A</p><p id="p-1271">Malvastrum yellow mosaic virus satellite DNA beta</p><p id="p-1272">Malvastrum yellow vein Baoshan virus DNA-A</p><p id="p-1273">Malvastrum yellow vein-virus</p><p id="p-1274">Malvastrum yellow vein virus satellite DNA beta</p><p id="p-1275">Malvastrum yellow vein Yunnan-virus</p><p id="p-1276">Malvastrum yellow vein Yunnan virus satellite DNA beta</p><p id="p-1277">Mamestra configurata NPV-A</p><p id="p-1278">Mamestra configurata NPV-B</p><p id="p-1279">Mannheimia phage phiMHaA1</p><p id="p-1280">Maruca vitrata MNPV</p><p id="p-1281">Mastomys coucha papillomavirus 2</p><p id="p-1282">Mastomys natalensis papillomavirus</p><p id="p-1283">Melanoplus sanguinipes entomopoxvirus</p><p id="p-1284">Meleagrid herpesvirus 1</p><p id="p-1285">Melon chlorotic leaf curl virus DNA A</p><p id="p-1286">Melon chlorotic mosaic virus-associated alphasatellite</p><p id="p-1287">Melon chlorotic mosaic virus DNA-A</p><p id="p-1288">Melon chlorotic mosaic virus DNA-B</p><p id="p-1289">Merkel cell polyomavirus</p><p id="p-1290">Merremia mosaic virus DNA A</p><p id="p-1291">Merremia mosaic virus DNA B</p><p id="p-1292">Mesta yellow vein mosaic Bahraich virus-</p><p id="p-1293">IndiaBahraich2007 DNA A</p><p id="p-1294">Mesta yellow vein mosaic virus-associated DNA beta</p><p id="p-1295">Mesta yellow vein mosaic virus DNA-A</p><p id="p-1296">Methanobacterium phage psiM2</p><p id="p-1297">Methanothermobacter prophage psiM100</p><p id="p-1298">Microbacterium phage Min1</p><p id="p-1299">Microcystis phage Ma-LMM01</p><p id="p-1300">Microplitis demolitor bracovirus segment A</p><p id="p-1301">Microplitis demolitor bracovirus segment B</p><p id="p-1302">Microplitis demolitor bracovirus segment C</p><p id="p-1303">Microplitis demolitor bracovirus segment D</p><p id="p-1304">Microplitis demolitor bracovirus segment E</p><p id="p-1305">Microplitis demolitor bracovirus segment F</p><p id="p-1306">Microplitis demolitor bracovirus segment G</p><p id="p-1307">Microplitis demolitor bracovirus segment H</p><p id="p-1308">Microplitis demolitor bracovirus segment I</p><p id="p-1309">Microplitis demolitor bracovirus segment J</p><p id="p-1310">Microplitis demolitor bracovirus segment K</p><p id="p-1311">Microplitis demolitor bracovirus segment L</p><p id="p-1312">Microplitis demolitor bracovirus segment M</p><p id="p-1313">Microplitis demolitor bracovirus segment N</p><p id="p-1314">Microplitis demolitor bracovirus segment O</p><p id="p-1315">Milk vetch dwarf virus segment 10</p><p id="p-1316">Milk vetch dwarf virus segment 11</p><p id="p-1317">Milk vetch dwarf virus segment-1</p><p id="p-1318">Milk vetch dwarf virus segment 2</p><p id="p-1319">Milk vetch dwarf virus segment 3</p><p id="p-1320">Milk vetch dwarf virus segment 4</p><p id="p-1321">Milk vetch dwarf virus segment 5</p><p id="p-1322">Milk vetch dwarf virus segment 6</p><p id="p-1323">Milk vetch dwarf virus segment 7</p><p id="p-1324">Milk vetch dwarf virus segment 8</p><p id="p-1325">Milk vetch dwarf virus segment 9</p><p id="p-1326">Mimosa yellow leaf curl virus-associated DNA 1</p><p id="p-1327">Mimosa yellow leaf curl virus DNA-A</p><p id="p-1328">Mimosa yellow leaf curl virus satellite DNA beta</p><p id="p-1329">Minute virus of mice</p><p id="p-1330">Mirabilis mosaic virus</p><p id="p-1331">Miscanthus streak virus-91</p><p id="p-1332">Molluscum contagiosum virus subtype 1</p><p id="p-1333">Monkeypox virus Zaire-96-I-16</p><p id="p-1334">Morganella phage MmP1</p><p id="p-1335">Mouse mammary tumor virus</p><p id="p-1336">Mouse parvovirus 1</p><p id="p-1337">Mouse parvovirus 2</p><p id="p-1338">Mouse parvovirus 3</p><p id="p-1339">Mouse parvovirus 4</p><p id="p-1340">Mouse parvovirus 5</p><p id="p-1341">Mulard duck circovirus</p><p id="p-1342">Mungbean yellow mosaic India virus DNA A</p><p id="p-1343">Mungbean yellow mosaic India virus DNA B</p><p id="p-1344">Mungbean yellow mosaic virus DNA A</p><p id="p-1345">Mungbean yellow mosaic virus DNA B</p><p id="p-1346">Murid herpesvirus 1</p><p id="p-1347">Murid herpesvirus 2</p><p id="p-1348">Murid herpesvirus 4</p><p id="p-1349">Murine adenovirus 3</p><p id="p-1350">Murine adenovirus A</p><p id="p-1351">Murine pneumotropic virus</p><p id="p-1352">Murine polyomavirus</p><p id="p-1353">Murine type C retrovirus</p><p id="p-1354">Musca domestica salivary gland hypertrophy virus</p><p id="p-1355">Muscovy duck circovirus</p><p id="p-1356">Muscovy duck parvovirus</p><p id="p-1357">Mus musculus papillomavirus type 1</p><p id="p-1358">Mycobacterium phage 244</p><p id="p-1359">Mycobacterium phage Adjutor</p><p id="p-1360">Mycobacterium phage Angel</p><p id="p-1361">Mycobacterium phage angelica</p><p id="p-1362">Mycobacterium phage Ardmore</p><p id="p-1363">Mycobacterium phage Barnyard</p><p id="p-1364">Mycobacterium phage Bethlehem</p><p id="p-1365">Mycobacterium phage Boomer</p><p id="p-1366">Mycobacterium phage BPs</p><p id="p-1367">Mycobacterium phage Brujita</p><p id="p-1368">Mycobacterium phage Butterscotch</p><p id="p-1369">Mycobacterium phage Bxb1</p><p id="p-1370">Mycobacterium phage Bxz1</p><p id="p-1371">Mycobacterium phage Bxz2</p><p id="p-1372">Mycobacterium phage Cali</p><p id="p-1373">Mycobacterium phage Catera</p><p id="p-1374">Mycobacterium phage Chah</p><p id="p-1375">Mycobacterium phage Che12</p><p id="p-1376">Mycobacterium phage Che8</p><p id="p-1377">Mycobacterium phage Che9c</p><p id="p-1378">Mycobacterium phage Che9d</p><p id="p-1379">Mycobacterium phage Cjw1</p><p id="p-1380">Mycobacterium phage Cooper</p><p id="p-1381">Mycobacterium phage Corndog</p><p id="p-1382">Mycobacterium phage CrimD</p><p id="p-1383">Mycobacterium phage D29</p><p id="p-1384">Mycobacterium phage DD5</p><p id="p-1385">Mycobacterium phage ET08</p><p id="p-1386">Mycobacterium phage Fruitloop</p><p id="p-1387">Mycobacterium phage Giles</p><p id="p-1388">Mycobacterium phage Gumball</p><p id="p-1389">Mycobacterium phage Halo</p><p id="p-1390">Mycobacterium phage Jasper</p><p id="p-1391">Mycobacterium phage KBG</p><p id="p-1392">Mycobacterium phage Konstantine</p><p id="p-1393">Mycobacterium phage Kostya</p><p id="p-1394">Mycobacterium phage L5</p><p id="p-1395">Mycobacterium phage LeBron</p><p id="p-1396">Mycobacterium phage Llij</p><p id="p-1397">Mycobacterium phage Lockley</p><p id="p-1398">Mycobacterium phage Myrna</p><p id="p-1399">Mycobacterium phage Nigel</p><p id="p-1400">Mycobacterium phage Omega</p><p id="p-1401">Mycobacterium phage Orion</p><p id="p-1402">Mycobacterium phage Pacc40</p><p id="p-1403">Mycobacterium phage PBI1</p><p id="p-1404">Mycobacterium phage Peaches</p><p id="p-1405">Mycobacterium phage PG1</p><p id="p-1406">Mycobacterium phage Phaedrus</p><p id="p-1407">Mycobacterium phage Phlyer</p><p id="p-1408">Mycobacterium phage Pipefish</p><p id="p-1409">Mycobacterium phage Plot</p><p id="p-1410">Mycobacterium phage PMC</p><p id="p-1411">Mycobacterium phage Porky</p><p id="p-1412">Mycobacterium phage Predator</p><p id="p-1413">Mycobacterium phage Pukovnik</p><p id="p-1414">Mycobacterium phage Qyrzula</p><p id="p-1415">Mycobacterium phage Ramsey</p><p id="p-1416">Mycobacterium phage Rizal</p><p id="p-1417">Mycobacterium phage Rosebush</p><p id="p-1418">Mycobacterium phage ScottMcG</p><p id="p-1419">Mycobacterium phage Solon</p><p id="p-1420">Mycobacterium phage Spud</p><p id="p-1421">Mycobacterium phage TM4</p><p id="p-1422">Mycobacterium phage Troll4</p><p id="p-1423">Mycobacterium phage Tweety</p><p id="p-1424">Mycobacterium phage U2</p><p id="p-1425">Mycobacterium phage Wildcat</p><p id="p-1426">Mycoplasma phage MAV1</p><p id="p-1427">Mycoplasma phage P1</p><p id="p-1428">Mycoplasma phage phiMFV1</p><p id="p-1429">Myotis polyomavirus VM-2008</p><p id="p-1430">Mythimna loreyi densovirus</p><p id="p-1431">Myxococcus phage Mx8</p><p id="p-1432">Myxoma virus</p><p id="p-1433">Myzus persicae densovirus</p><p id="p-1434">Nanovirus-like particle</p><p id="p-1435">Natrialba phage PhiCh1</p><p id="p-1436">Neodiprion abietis NPV</p><p id="p-1437">Neodiprion lecontei NPV</p><p id="p-1438">Neodiprion sertifer NPV Oat dwarf virus</p><p id="p-1439">OkLCV satDNA 10</p><p id="p-1440">Okra leaf curl disease associated DNA 1</p><p id="p-1441">Okra leaf curl Mali virus satellite DNA beta</p><p id="p-1442">Okra leaf curl virus-Cameroon</p><p id="p-1443">Okra mottle virus-Brazilokra DNA A</p><p id="p-1444">Okra mottle virus-Brazilokra DNA B</p><p id="p-1445">Okra yellow crinkle virus segment A</p><p id="p-1446">Okra yellow mosaic Mexico virus DNA A</p><p id="p-1447">Okra yellow mosaic Mexico virus DNA B</p><p id="p-1448">Okra yellow vein disease associated sequence virion</p><p id="p-1449">Okra yellow vein mosaic virus</p><p id="p-1450">Old World harvest mouse papillomavirus</p><p id="p-1451">Orangutan polyomavirus Orf virus</p><p id="p-1452">Orgyia leucostigma NPV</p><p id="p-1453">Orgyia pseudotsugata MNPV</p><p id="p-1454">Oryctes rhinoceros virus</p><p id="p-1455">Ostreid herpesvirus 1</p><p id="p-1456">Ostreococcus tauri virus 1</p><p id="p-1457">Ostreococcus virus OsV5</p><p id="p-1458">Ovine adenovirus A</p><p id="p-1459">Ovine adenovirus D</p><p id="p-1460">Ovine herpesvirus 2</p><p id="p-1461">Ovine papillomavirus-1</p><p id="p-1462">Panicum streak virus-Karino</p><p id="p-1463">Panine herpesvirus 2 strain Heberling</p><p id="p-1464">Papaya leaf curl China virus-G8</p><p id="p-1465">Papaya leaf curl China virus satellite DNA beta</p><p id="p-1466">Papaya leaf curl Guandong virus-GD2 DNA A</p><p id="p-1467">Papaya leaf curl virus-associated DNA beta</p><p id="p-1468">Papaya leaf curl-virus Papiine herpesvirus 2</p><p id="p-1469">Paramecium bursaria Chlorella virus 1</p><p id="p-1470">Paramecium bursaria Chlorella virus AR158</p><p id="p-1471">Paramecium bursaria Chlorella virus FR483</p><p id="p-1472">Paramecium bursaria Chlorella virus NY2A</p><p id="p-1473">Parvovirus H1</p><p id="p-1474">Passionfruit severe leaf distortion virus DNA-A</p><p id="p-1475">Passionfruit severe leaf distortion virus DNA-B</p><p id="p-1476">Pasteurella phage F108</p><p id="p-1477">Peanut chlorotic streak virus</p><p id="p-1478">Pedilanthus leaf curl virus-</p><p id="p-1479">Pedilanthus PakistanMultan2004</p><p id="p-1480">Pelargonium vein banding virus</p><p id="p-1481">Penaeus merguiensis densovirus</p><p id="p-1482">Penaeus monodon hepatopancreatic parvovirus</p><p id="p-1483">Pepper curly top virus</p><p id="p-1484">Pepper golden mosaic virus DNA A</p><p id="p-1485">Pepper golden mosaic virus DNA B</p><p id="p-1486">Pepper huasteco yellow vein virus DNA A</p><p id="p-1487">Pepper huasteco yellow vein virus DNA B</p><p id="p-1488">Pepper leaf curl Bangladesh virus segment A component</p><p id="p-1489">Pepper leaf curl virus DNA-A</p><p id="p-1490">Pepper leaf curl virus satellite DNA beta</p><p id="p-1491">Pepper leaf curl Yunnan virus satellite DNA beta</p><p id="p-1492">Pepper leaf curl Yunnan virus-YN323</p><p id="p-1493">Pepper yellow dwarf virus-New Mexico</p><p id="p-1494">Pepper yellow leaf curl Indonesia virus DNA-A</p><p id="p-1495">Pepper yellow leaf curl Indonesia virus DNA-B</p><p id="p-1496">Pepper yellow vein Mali virus</p><p id="p-1497">Periplaneta fuliginosa densovirus</p><p id="p-1498">Petunia vein clearing virus</p><p id="p-1499">Phage cdtI</p><p id="p-1500">Phage Gifsy-1</p><p id="p-1501">Phage Gifsy-2</p><p id="p-1502">Phage phiJL001</p><p id="p-1503">Phocoena spinipinnis papillomavirus</p><p id="p-1504">Phormidium phage Pf-WMP3</p><p id="p-1505">Phormidium phage Pf-WMP4</p><p id="p-1506">Phthorimaea operculella granulovirus</p><p id="p-1507">Pieris rapae granulovirus</p><p id="p-1508">Planaria asexual strain-specific virus-like element type 1 large DNA segment</p><p id="p-1509">Planaria asexual strain-specific virus-like element type 1 small DNA segment</p><p id="p-1510">Planococcus citri densovirus</p><p id="p-1511">Plutella xylostella granulovirus</p><p id="p-1512">Plutella xylostella multiple nucleopolyhedrovirus</p><p id="p-1513">Polyomavirus HPyV6</p><p id="p-1514">Polyomavirus HPyV7</p><p id="p-1515">Porcine adenovirus C</p><p id="p-1516">Porcine circovirus 1</p><p id="p-1517">Porcine circovirus 2</p><p id="p-1518">Porcine endogenous retrovirus E</p><p id="p-1519">Porcine parvovirus</p><p id="p-1520">Potato apical leaf curl disease-associated satellite DNA beta</p><p id="p-1521">Potato yellow mosaic</p><p id="p-1522">Panama virus DNA A</p><p id="p-1523">Potato yellow mosaic</p><p id="p-1524">Panama virus DNA B</p><p id="p-1525">Potato yellow mosaic Trinidad virus DNA A</p><p id="p-1526">Potato yellow mosaic Trinidad virus DNA B</p><p id="p-1527">Potato yellow mosaic virus DNA A</p><p id="p-1528">Potato yellow mosaic virus DNA B</p><p id="p-1529">Prochlorococcus phage P-SSM4</p><p id="p-1530">Propionibacterium phage B5</p><p id="p-1531">Propionibacterium phage PA6</p><p id="p-1532">Pseudaletia unipuncta granulovirus</p><p id="p-1533">Pseudoalteromonas phage PM2</p><p id="p-1534">Pseudocowpox virus</p><p id="p-1535">Pseudomonas phage 119X</p><p id="p-1536">Pseudomonas phage 14-1</p><p id="p-1537">Pseudomonas phage 201phi2-1</p><p id="p-1538">Pseudomonas phage 73</p><p id="p-1539">Pseudomonas phage B3</p><p id="p-1540">Pseudomonas phage D3112</p><p id="p-1541">Pseudomonas phage-D3</p><p id="p-1542">Pseudomonas phage DMS3</p><p id="p-1543">Pseudomonas phage EL</p><p id="p-1544">Pseudomonas phage F10</p><p id="p-1545">Pseudomonas phage F116</p><p id="p-1546">Pseudomonas phage F8</p><p id="p-1547">Pseudomonas phage gh-1</p><p id="p-1548">Pseudomonas phage LBL3</p><p id="p-1549">Pseudomonas phage LIT1</p><p id="p-1550">Pseudomonas phage LKD16</p><p id="p-1551">Pseudomonas phage LMA2</p><p id="p-1552">Pseudomonas phage LUZ19</p><p id="p-1553">Pseudomonas phage LUZ24</p><p id="p-1554">Pseudomonas phage LUZ7</p><p id="p-1555">Pseudomonas phage M6</p><p id="p-1556">Pseudomonas phage MP22</p><p id="p-1557">Pseudomonas phage MP29</p><p id="p-1558">Pseudomonas phage MP38</p><p id="p-1559">Pseudomonas phage PA11</p><p id="p-1560">Pseudomonas phage PAJU2</p><p id="p-1561">Pseudomonas phage PaP2</p><p id="p-1562">Pseudomonas phage PaP3</p><p id="p-1563">Pseudomonas phage PB1</p><p id="p-1564">Pseudomonas phage Pf1</p><p id="p-1565">Pseudomonas phage Pf3</p><p id="p-1566">Pseudomonas phage phi-2</p><p id="p-1567">Pseudomonas phage phiCTX</p><p id="p-1568">Pseudomonas phage phikF77</p><p id="p-1569">Pseudomonas phage phiKMV</p><p id="p-1570">Pseudomonas phage phiKZ</p><p id="p-1571">Pseudomonas phage PT2</p><p id="p-1572">Pseudomonas phage PT5</p><p id="p-1573">Pseudomonas phage SN</p><p id="p-1574">Pseudomonas phage YuA</p><p id="p-1575">Psittacid herpesvirus 1</p><p id="p-1576">Psittacus erithacus timneh papillomavirus</p><p id="p-1577">Pumpkin yellow mosaic Malaysia virus DNA A</p><p id="p-1578">Pyrobaculum spherical virus</p><p id="p-1579">Pyrococcus abyssi virus 1</p><p id="p-1580">Rabbit fibroma virus</p><p id="p-1581">Rabbit oral papillomavirus</p><p id="p-1582">Rachiplusia ou MNPV</p><p id="p-1583">Radish leaf curl virus satellite DNA beta</p><p id="p-1584">Radish leaf curl virus segment A</p><p id="p-1585">Ralstonia phage p12J</p><p id="p-1586">Ralstonia phage phiRSA1</p><p id="p-1587">Ralstonia phage RSB1</p><p id="p-1588">Ralstonia phage RSL1</p><p id="p-1589">Ralstonia phage RSM1</p><p id="p-1590">Ralstonia phage RSM3</p><p id="p-1591">Ralstonia phage RSS1</p><p id="p-1592">Ramie mosaic virus DNA-A</p><p id="p-1593">Ramie mosaic virus DNA-B</p><p id="p-1594">Ranid herpesvirus 1 strain McKinnell</p><p id="p-1595">Ranid herpesvirus 2 strain ATCC VR-568</p><p id="p-1596">Rauscher murine leukemia virus</p><p id="p-1597">Raven circovirus RD114 retrovirus</p><p id="p-1598">Reticuloendotheliosis virus</p><p id="p-1599">Rhesus monkey</p><p id="p-1600">Papillomavirus</p><p id="p-1601">Rhizobium phage 16-3</p><p id="p-1602">Rhodothermus phage RM378</p><p id="p-1603">Rhynchosia golden mosaic virus DNA A</p><p id="p-1604">Rhynchosia golden mosaic virus DNA B</p><p id="p-1605">Rhynchosia golden mosaic Yucatan virus DNA A</p><p id="p-1606">Rhynchosia golden mosaic Yucatan virus DNA B</p><p id="p-1607">Rice tungro bacilliform virus</p><p id="p-1608">Roseobacter phage SIO1</p><p id="p-1609">Roseophage DSS3P2</p><p id="p-1610">Roseophage EE36P1</p><p id="p-1611">Rosss goose hepatitis B virus</p><p id="p-1612">Rousettus aegyptiacus papillomavirus type 1</p><p id="p-1613">Rudbeckia flower distortion virus</p><p id="p-1614">Saccharum streak virus</p><p id="p-1615">Saimiriine herpesvirus 2</p><p id="p-1616">Salmonella enterica bacteriophage SE1</p><p id="p-1617">Salmonella phage c341</p><p id="p-1618">Salmonella phage E1</p><p id="p-1619">Salmonella phage epsilon34</p><p id="p-1620">Salmonella phage Fels-1</p><p id="p-1621">Salmonella phage phiSG-JL2</p><p id="p-1622">Salmonella phage SETP3</p><p id="p-1623">Salmonella phage ST64B</p><p id="p-1624">Sclerotinia sclerotiorum hypovirulence associated DNA virus 1</p><p id="p-1625">Sea turtle tornovirus 1</p><p id="p-1626">Senecio yellow mosaic virus</p><p id="p-1627">Sheeppox virus 17077-99</p><p id="p-1628">Sheldgoose hepatitis B virus</p><p id="p-1629">Shigella phage phiSboM-AG3</p><p id="p-1630">Shrimp white spot syndrome virus</p><p id="p-1631">Sida golden mosaic Costa Rica virus DNA A</p><p id="p-1632">Sida golden mosaic Costa Rica virus DNA B</p><p id="p-1633">Sida golden mosaic Florida virus-Malvastrum DNA-A</p><p id="p-1634">Sida golden mosaic Florida virus-Malvastrum DNA-B</p><p id="p-1635">Sida golden mosaic Honduras virus DNA A</p><p id="p-1636">Sida golden mosaic Honduras virus DNA B</p><p id="p-1637">Sida golden mosaic virus DNA-A</p><p id="p-1638">Sida golden mosaic virus DNA-B</p><p id="p-1639">Sida golden mottle virus DNA-A</p><p id="p-1640">Sida golden mottle virus DNA-B</p><p id="p-1641">Sida leaf curl virus-associated DNA 1</p><p id="p-1642">Sida leaf curl virus-associated DNA beta</p><p id="p-1643">Sida leaf curl-virus</p><p id="p-1644">Sida leaf curl virus satellite DNA beta</p><p id="p-1645">Sida micrantha mosaic virus segment A</p><p id="p-1646">Sida micrantha mosaic virus segment B</p><p id="p-1647">Sida mosaic Sinaloa virus DNA A</p><p id="p-1648">Sida mosaic Sinaloa virus DNA B</p><p id="p-1649">Sida mottle virus</p><p id="p-1650">Sida yellow mosaic virus-China-associated DNA beta DNA beta</p><p id="p-1651">Sida yellow mosaic-virus</p><p id="p-1652">Sida yellow mosaic Yucatan virus DNA A</p><p id="p-1653">Sida yellow mosaic Yucatan virus DNA B</p><p id="p-1654">Sida yellow vein disease associated DNA 1</p><p id="p-1655">Sida yellow vein Madurai virus</p><p id="p-1656">Sida yellow vein Vietnam virus-associated DNA 1</p><p id="p-1657">Sida yellow vein Vietnam virus DNA-A</p><p id="p-1658">Sida yellow vein Vietnam virus satellite DNA beta</p><p id="p-1659">Sida yellow vein virus DNA A</p><p id="p-1660">Sida yellow vein virus DNA B</p><p id="p-1661">Sida yellow vein virus satellite DNA beta</p><p id="p-1662">Siegesbeckia yellow vein Guangxi virus</p><p id="p-1663">Siegesbeckia yellow vein virus-GD13-associated DNA beta</p><p id="p-1664">Siegesbeckia yellow vein virus GD13</p><p id="p-1665">Simian adenovirus 1</p><p id="p-1666">Simian adenovirus 3</p><p id="p-1667">Simian foamy virus 3</p><p id="p-1668">Simian immunodeficiency virus SIV-mnd 2</p><p id="p-1669">Simian retrovirus 4</p><p id="p-1670">Simian T-cell lymphotropic virus 6</p><p id="p-1671">Simian T-lymphotropic virus 1</p><p id="p-1672">Simian T-lymphotropic virus 3</p><p id="p-1673">Simian virus 40</p><p id="p-1674">Singapore grouper iridovirus</p><p id="p-1675">Sinorhizobium phage PBC5</p><p id="p-1676">Small anellovirus 1</p><p id="p-1677">Small anellovirus 2</p><p id="p-1678">Snake parvovirus 1</p><p id="p-1679">Snow goose hepatitis B virus</p><p id="p-1680">Sodalis phage phiSG1</p><p id="p-1681">Sodalis phage SO-1</p><p id="p-1682">South African cassava mosaic virus DNA A</p><p id="p-1683">South African cassava mosaic virus DNA B</p><p id="p-1684">Soybean chlorotic blotch virus DNA A</p><p id="p-1685">Soybean chlorotic blotch virus DNA B</p><p id="p-1686">Soybean chlorotic mottle virus</p><p id="p-1687">Soybean crinkle leaf virus</p><p id="p-1688">Soybean mild mottle virus</p><p id="p-1689">Spilanthes yellow vein virus DNA-A</p><p id="p-1690">Spinach curly top virus</p><p id="p-1691">Spiroplasma kunkelii virus</p><p id="p-1692">SkV1 CR2-3x</p><p id="p-1693">Spiroplasma phage 1-C74</p><p id="p-1694">Spiroplasma phage 1-R8A2B</p><p id="p-1695">Spiroplasma phage 4</p><p id="p-1696">Spiroplasma phage SVTS2</p><p id="p-1697">Spodoptera exigua MNPV</p><p id="p-1698">Spodoptera frugiperda ascovirus 1a</p><p id="p-1699">Spodoptera frugiperda MNPV virus</p><p id="p-1700">Spodoptera litura granulovirus</p><p id="p-1701">Spodoptera litura NPV</p><p id="p-1702">Spodoptera litura nucleopolyhedrovirus II</p><p id="p-1703">Sputnik virophage</p><p id="p-1704">Squash leaf curl China virus-B DNA-A</p><p id="p-1705">Squash leaf curl China virus-B DNA B</p><p id="p-1706">Squash leaf curl Philippines virus segment A</p><p id="p-1707">Squash leaf curl Philippines virus segment B</p><p id="p-1708">Squash leaf curl virus A component DNA</p><p id="p-1709">Squash leaf curl virus B component DNA</p><p id="p-1710">Squash leaf curl Yunnan virus</p><p id="p-1711">Squash mild leaf curl virus-Imperial Valley DNA A</p><p id="p-1712">Squash mild leaf curl virus-Imperial Valley DNA B</p><p id="p-1713">Squash yellow mild mottle virus DNA B</p><p id="p-1714">Squirrel monkey polyomavirus</p><p id="p-1715">Sri Lankan cassava mosaic virus DNA A</p><p id="p-1716">Sri Lankan cassava mosaic virus DNA B</p><p id="p-1717">Stachytarpheta leaf curl virus</p><p id="p-1718">Staphylococcus phage 11</p><p id="p-1719">Staphylococcus phage 187</p><p id="p-1720">Staphylococcus phage 2638A</p><p id="p-1721">Staphylococcus phage 29</p><p id="p-1722">Staphylococcus phage 37</p><p id="p-1723">Staphylococcus phage 3A</p><p id="p-1724">Staphylococcus phage 42e</p><p id="p-1725">Staphylococcus phage 44AHJD</p><p id="p-1726">Staphylococcus phage 47</p><p id="p-1727">Staphylococcus phage 52A</p><p id="p-1728">Staphylococcus phage 53</p><p id="p-1729">Staphylococcus phage 55</p><p id="p-1730">Staphylococcus phage 66</p><p id="p-1731">Staphylococcus phage 69</p><p id="p-1732">Staphylococcus phage 71</p><p id="p-1733">Staphylococcus phage 77</p><p id="p-1734">Staphylococcus phage 80alpha</p><p id="p-1735">Staphylococcus phage 85</p><p id="p-1736">Staphylococcus phage 88</p><p id="p-1737">Staphylococcus phage 92</p><p id="p-1738">Staphylococcus phage 96</p><p id="p-1739">Staphylococcus phage CNPH82</p><p id="p-1740">Staphylococcus phage EW</p><p id="p-1741">Staphylococcus phage G1</p><p id="p-1742">Staphylococcus phage K</p><p id="p-1743">Staphylococcus phage P954</p><p id="p-1744">Staphylococcus phage PH15</p><p id="p-1745">Staphylococcus phage phi2958PVL</p><p id="p-1746">Staphylococcus phage phiETA2</p><p id="p-1747">Staphylococcus phage phiETA3</p><p id="p-1748">Staphylococcus phage-phiETA</p><p id="p-1749">Staphylococcus phage phiMR11</p><p id="p-1750">Staphylococcus phage phiMR25</p><p id="p-1751">Staphylococcus phage phiNM1</p><p id="p-1752">Staphylococcus phage phiNM3</p><p id="p-1753">Staphylococcus phage phiP68</p><p id="p-1754">Staphylococcus phage phiPVL108</p><p id="p-1755">Staphylococcus phage phiPVL-CN125</p><p id="p-1756">Staphylococcus phage phiSauS-IPLA35</p><p id="p-1757">Staphylococcus phage phiSauS-IPLA88</p><p id="p-1758">Staphylococcus phage phiSLT</p><p id="p-1759">Staphylococcus phage PT1028</p><p id="p-1760">Staphylococcus phage ROSA</p><p id="p-1761">Staphylococcus phage SAP-26</p><p id="p-1762">Staphylococcus phage SAP2</p><p id="p-1763">Staphylococcus phage Twort</p><p id="p-1764">Staphylococcus phage X2</p><p id="p-1765">Staphylococcus prophage phi 12</p><p id="p-1766">Staphylococcus prophage phi 13</p><p id="p-1767">Staphylococcus prophage phiN315</p><p id="p-1768">Staphylococcus prophage phiPV83</p><p id="p-1769">Staphylococcus prophage PVL</p><p id="p-1770">Staphylococcus prophage tp310-1</p><p id="p-1771">Staphylococcus prophage tp310-2</p><p id="p-1772">Staphylococcus prophage tp310-3</p><p id="p-1773">Starling circovirus</p><p id="p-1774">Stenotrophomonas phage phiSMA9</p><p id="p-1775">Stenotrophomonas phage S1</p><p id="p-1776">Strawberry vein banding virus</p><p id="p-1777">Streptococcus phage 2972</p><p id="p-1778">Streptococcus phage 5093</p><p id="p-1779">Streptococcus phage 7201</p><p id="p-1780">Streptococcus phage 858</p><p id="p-1781">Streptococcus phage Abc2</p><p id="p-1782">Streptococcus phage ALQ13.2</p><p id="p-1783">Streptococcus phage C1</p><p id="p-1784">Streptococcus phage Cp-1</p><p id="p-1785">Streptococcus phage DT1</p><p id="p-1786">Streptococcus phage M102</p><p id="p-1787">Streptococcus phage O1205</p><p id="p-1788">Streptococcus phage P9</p><p id="p-1789">Streptococcus phage PH10</p><p id="p-1790">Streptococcus phage PH15</p><p id="p-1791">Streptococcus phage phi3396</p><p id="p-1792">Streptococcus phage Sfi11</p><p id="p-1793">Streptococcus phage Sfi19</p><p id="p-1794">Streptococcus phage Sfi21</p><p id="p-1795">Streptococcus phage SM1</p><p id="p-1796">Streptococcus phage SMP</p><p id="p-1797">Streptococcus prophage 315.1</p><p id="p-1798">Streptococcus prophage 315.2</p><p id="p-1799">Streptococcus prophage 315.5</p><p id="p-1800">Streptococcus prophage 315.6</p><p id="p-1801">Streptococcus prophage EJ-1</p><p id="p-1802">Streptococcus prophage MM1</p><p id="p-1803">Streptococcus pyogenes phage 315.3</p><p id="p-1804">Streptomyces phage mu16</p><p id="p-1805">Streptomyces phage phiBT1</p><p id="p-1806">Streptomyces phage phiC31</p><p id="p-1807">Streptomyces phage phiSASD1</p><p id="p-1808">Streptomyces phage VWB Stx1 converting phage</p><p id="p-1809">Stx2-converting phage 1717</p><p id="p-1810">Stx2-converting phage 86</p><p id="p-1811">Stx2 converting phage-I</p><p id="p-1812">Stx2 converting phage II</p><p id="p-1813">Subterranean clover stunt virus DNA 1</p><p id="p-1814">Subterranean clover stunt virus DNA 2</p><p id="p-1815">Subterranean clover stunt virus DNA 3</p><p id="p-1816">Subterranean clover stunt virus DNA 4</p><p id="p-1817">Subterranean clover stunt virus DNA 5</p><p id="p-1818">Subterranean clover stunt virus DNA 6</p><p id="p-1819">Subterranean clover stunt virus DNA 7</p><p id="p-1820">Subterranean clover stunt virus DNA 8</p><p id="p-1821">Sugarcane bacilliform IM virus</p><p id="p-1822">Sugarcane bacilliform Mor virus</p><p id="p-1823">Sugarcane bacilliform virus</p><p id="p-1824">Sugarcane streak Egypt virus-Giza</p><p id="p-1825">Sugarcane streak Reunion virus</p><p id="p-1826">Sugarcane streak virus-Natal</p><p id="p-1827">Suid herpesvirus 1</p><p id="p-1828">Sulfolobus islandicus filamentous virus</p><p id="p-1829">Sulfolobus islandicus rod-shaped virus 1</p><p id="p-1830">Sulfolobus islandicus rod-shaped virus 2</p><p id="p-1831">Sulfolobus spindle-shaped virus 4</p><p id="p-1832">Sulfolobus spindle-shaped virus 5</p><p id="p-1833">Sulfolobus spindle-shaped virus 6</p><p id="p-1834">Sulfolobus spindle-shaped virus 7</p><p id="p-1835">Sulfolobus turreted icosahedral virus 2</p><p id="p-1836">Sulfolobus turreted icosahedral-virus</p><p id="p-1837">Sulfolobus virus 1</p><p id="p-1838">Sulfolobus virus 2</p><p id="p-1839">Sulfolobus virus Kamchatka 1</p><p id="p-1840">Sulfolobus virus Ragged Hills</p><p id="p-1841">Sulfolobus virus STSV1</p><p id="p-1842">Sunn hemp leaf distortion virus DNA-A</p><p id="p-1843">Sus scrofa papillomavirus type 1</p><p id="p-1844">Sweetpotato badnavirus B</p><p id="p-1845">Sweet potato leaf curl Bengal virus-IndiaWest Bengal2008 segment A</p><p id="p-1846">Sweet potato leaf curl Canary virus</p><p id="p-1847">Sweet potato leaf curl Georgia virus</p><p id="p-1848">Sweet potato leaf curl Lanzarote virus</p><p id="p-1849">Sweet potato leaf curl Spain virus</p><p id="p-1850">Sweet potato leaf curl virus</p><p id="p-1851">Swinepox virus Synechococcus phage P60</p><p id="p-1852">Synechococcus phage S-PM2</p><p id="p-1853">Synechococcus phage S-RSM4</p><p id="p-1854">Synechococcus phage syn9</p><p id="p-1855">Tanapox virus</p><p id="p-1856">Taro bacilliform virus</p><p id="p-1857">Taterapox virus</p><p id="p-1858">Thalassomonas phage BA3</p><p id="p-1859">Thermoproteus tenax spherical virus 1</p><p id="p-1860">Thermus phage IN93</p><p id="p-1861">Thermus phage P23-45</p><p id="p-1862">Thermus phage P23-77</p><p id="p-1863">Thermus phage P74-26</p><p id="p-1864">Thermus phage phiYS40</p><p id="p-1865">Tobacco curly shoot virus associated DNA 1</p><p id="p-1866">Tobacco curly shoot virus-associated DNA beta</p><p id="p-1867">Tobacco curly shoot-virus</p><p id="p-1868">Tobacco leaf curl disease associated sequence virion</p><p id="p-1869">Tobacco leaf curl Japan virus</p><p id="p-1870">Tobacco leaf curl Kochi virus</p><p id="p-1871">Tobacco leaf curl Thailand virus</p><p id="p-1872">Tobacco leaf curl virus-associated DNA beta</p><p id="p-1873">Tobacco leaf curl Yunnan virus associated DNA 1</p><p id="p-1874">Tobacco leaf curl Yunnan virus satellite DNA beta</p><p id="p-1875">Tobacco leaf curl Yunnan virus-Y136</p><p id="p-1876">Tobacco leaf curl Zimbabwe virus</p><p id="p-1877">Tobacco vein clearing virus</p><p id="p-1878">Tobacco yellow dwarf virus</p><p id="p-1879">Tomato begomovirus satellite DNA beta</p><p id="p-1880">Tomato chino La Paz virus segment A</p><p id="p-1881">Tomato chlorotic mottle virus DNA A</p><p id="p-1882">Tomato chlorotic mottle virus DNA B</p><p id="p-1883">Tomato common mosaic virus DNA-A</p><p id="p-1884">Tomato common mosaic virus DNA-B</p><p id="p-1885">Tomato curly stunt virus</p><p id="p-1886">Tomato golden mosaic virus DNA A</p><p id="p-1887">Tomato golden mosaic virus DNA B</p><p id="p-1888">Tomato golden mottle virus DNA A</p><p id="p-1889">Tomato golden mottle virus DNA B</p><p id="p-1890">Tomato leaf curl Arusha virus DNA-A</p><p id="p-1891">Tomato leaf curl Bangalore virus-Ban5 satellite DNA beta</p><p id="p-1892">Tomato leaf curl Bangalore-virus</p><p id="p-1893">Tomato leaf curl Bangladesh virus</p><p id="p-1894">Tomato leaf curl Cameroon virus-CameroonBueaOkra2008</p><p id="p-1895">Tomato leaf curl Cebu virus DNA-A</p><p id="p-1896">Tomato leaf curl China virus-G32</p><p id="p-1897">Tomato leaf curl Cotabato virus DNA-A</p><p id="p-1898">Tomato leaf curl Ghana virus segment A</p><p id="p-1899">Tomato leaf curl Guangdong virus DNA-A</p><p id="p-1900">Tomato leaf curl Guangxi virus</p><p id="p-1901">Tomato leaf curl Gujarat virus-Varanasi segment A</p><p id="p-1902">Tomato leaf curl Gujarat virus-Varanasi segment B</p><p id="p-1903">Tomato leaf curl Hainan virus</p><p id="p-1904">Tomato leaf curl Hsinchu virus-TaiwanHsinchu2005 DNA A</p><p id="p-1905">Tomato leaf curl Iran virus</p><p id="p-1906">Tomato leaf curl Java virus-Ageratum satellite DNA</p><p id="p-1907">Tomato leaf curl Java-virus</p><p id="p-1908">Tomato leaf curl Joydebpur beta virus</p><p id="p-1909">Tomato leaf curl Joydebpur virus DNA-A</p><p id="p-1910">Tomato leaf curl Karnataka virus-associated DNA beta DNA-A</p><p id="p-1911">Tomato leaf curl Karnataka-virus</p><p id="p-1912">Tomato leaf curl Kerala virus</p><p id="p-1913">Tomato leaf curl Kumasi virus segment A</p><p id="p-1914">Tomato leaf curl Laos virus</p><p id="p-1915">Tomato leaf curl Malaysia virus</p><p id="p-1916">Tomato leaf curl Mali virus</p><p id="p-1917">Tomato leaf curl Mayotte virus</p><p id="p-1918">Tomato leaf curl Mindanao virus DNA-A</p><p id="p-1919">Tomato leaf curl New Delhi virus-associated DNA beta</p><p id="p-1920">Tomato leaf curl New Delhi virus DNA A</p><p id="p-1921">Tomato leaf curl New Delhi virus DNA B</p><p id="p-1922">Tomato leaf curl Nigeria virus-Nigeria2006</p><p id="p-1923">Tomato leaf curl Pakistan virus associated DNA 1</p><p id="p-1924">Tomato leaf curl Pakistan virus segment A</p><p id="p-1925">Tomato leaf curl Palampur virus</p><p id="p-1926">Tomato leaf curl Patna virus DNA-A</p><p id="p-1927">Tomato leaf curl Philippines virus</p><p id="p-1928">Tomato leaf curl Philippine virus satellite DNA beta</p><p id="p-1929">Tomato leaf curl Pune virus</p><p id="p-1930">Tomato leaf curl Seychelles virus</p><p id="p-1931">Tomato leaf curl Sinaloa virus DNA A</p><p id="p-1932">Tomato leaf curl Sinaloa virus DNA B</p><p id="p-1933">Tomato leaf curl Sri Lanka virus</p><p id="p-1934">Tomato leaf curl Sudan virus-Gezira</p><p id="p-1935">Tomato leaf curl Sulawesi virus DNA-A</p><p id="p-1936">Tomato leaf curl Taiwan virus</p><p id="p-1937">Tomato leaf curl Togo virus-Togo2006</p><p id="p-1938">Tomato leaf curl Vietnam virus DNA A</p><p id="p-1939">Tomato leaf curl virus-associated DNA beta</p><p id="p-1940">Tomato leaf curl-virus</p><p id="p-1941">Tomato leaf curl virus-Pune-associated DNA beta DNA-A</p><p id="p-1942">Tomato mild mosaic virus DNA-A</p><p id="p-1943">Tomato mild mosaic virus DNA-B</p><p id="p-1944">Tomato mild yellow leaf curl Aragua virus DNA A</p><p id="p-1945">Tomato mild yellow leaf curl Aragua virus DNA B</p><p id="p-1946">Tomato mosaic Havana virus DNA A</p><p id="p-1947">Tomato mosaic Havana virus DNA B</p><p id="p-1948">Tomato mosaic leaf curl virus DNA A</p><p id="p-1949">Tomato mosaic leaf curl virus DNA B</p><p id="p-1950">Tomato mottle Taino virus DNA A</p><p id="p-1951">Tomato mottle Taino virus DNA B</p><p id="p-1952">Tomato mottle virus DNA A</p><p id="p-1953">Tomato mottle virus DNA B</p><p id="p-1954">Tomato pseudo-curly top virus</p><p id="p-1955">Tomato rugose mosaic virus DNA A</p><p id="p-1956">Tomato rugose mosaic virus DNA B</p><p id="p-1957">Tomato severe leaf curl virus</p><p id="p-1958">Tomato severe rugose virus DNA A</p><p id="p-1959">Tomato severe rugose virus DNA B</p><p id="p-1960">Tomato yellow dwarf disease associated satellite DNA beta-Kochi virus</p><p id="p-1961">Tomato yellow leaf curl China virus associated DNA 1</p><p id="p-1962">Tomato yellow leaf curl China-virus</p><p id="p-1963">Tomato yellow leaf curl Guangdong virus DNA-A</p><p id="p-1964">Tomato yellow leaf curl Indonesia virus-Lembang</p><p id="p-1965">Tomato yellow leaf curl Kanchanaburi virus DNA A</p><p id="p-1966">Tomato yellow leaf curl Kanchanaburi virus DNA B</p><p id="p-1967">Tomato yellow leaf curl Mali virus-associated DNA beta</p><p id="p-1968">Tomato yellow leaf curl Thailand virus associated DNA 1</p><p id="p-1969">Tomato yellow leaf curl Thailand virus DNA A</p><p id="p-1970">Tomato yellow leaf curl Thailand virus DNA B</p><p id="p-1971">Tomato yellow leaf curl Vietnam virus DNA-A</p><p id="p-1972">Tomato yellow leaf curl Vietnam virus satellite DNA beta</p><p id="p-1973">Tomato yellow leaf curl virus-associated DNA beta</p><p id="p-1974">Tomato yellow margin leaf curl virus DNA A</p><p id="p-1975">Tomato yellow margin leaf curl virus DNA B</p><p id="p-1976">Tomato yellow spot virus DNA-A</p><p id="p-1977">Tomato yellow spot virus DNA-B</p><p id="p-1978">Tomato yellow vein streak virus DNA-A</p><p id="p-1979">Tomato yellow vein streak virus DNA-B</p><p id="p-1980">Torque teno canis virus Torque teno douroucouli virus</p><p id="p-1981">Torque teno felis virus</p><p id="p-1982">Torque teno midi virus 1</p><p id="p-1983">Torque teno midi virus 2</p><p id="p-1984">Torque teno mini virus 1</p><p id="p-1985">Torque teno mini virus 2</p><p id="p-1986">Torque teno mini virus 3</p><p id="p-1987">Torque teno mini virus 4</p><p id="p-1988">Torque teno mini virus 5</p><p id="p-1989">Torque teno mini virus 6</p><p id="p-1990">Torque teno mini virus 7</p><p id="p-1991">Torque teno mini virus 8</p><p id="p-1992">Torque teno mini virus 9</p><p id="p-1993">Torque teno sus virus 1</p><p id="p-1994">Torque teno tamarin virus</p><p id="p-1995">Torque teno virus 10</p><p id="p-1996">Torque teno virus 12</p><p id="p-1997">Torque teno virus 14</p><p id="p-1998">Torque teno virus 15</p><p id="p-1999">Torque teno virus 16</p><p id="p-2000">Torque teno virus 19</p><p id="p-2001">Torque teno virus-1</p><p id="p-2002">Torque teno virus 25</p><p id="p-2003">Torque teno virus 26</p><p id="p-2004">Torque teno virus 27</p><p id="p-2005">Torque teno virus 28</p><p id="p-2006">Torque teno virus 3</p><p id="p-2007">Torque teno virus 4</p><p id="p-2008">Torque teno virus 6</p><p id="p-2009">Torque teno virus 7</p><p id="p-2010">Torque teno virus 8</p><p id="p-2011">Trichodysplasia spinulosa-associated polyomavirus</p><p id="p-2012">Trichoplusia ni ascovirus 2c</p><p id="p-2013">Trichoplusia ni SNPV</p><p id="p-2014">Tupaiid herpesvirus 1</p><p id="p-2015">Turkey adenovirus A</p><p id="p-2016">Turnip curly top virus</p><p id="p-2017">TYLCCNV-Y322 satellite DNA beta</p><p id="p-2018">Urochloa streak virus Vaccinia virus Variola virus</p><p id="p-2019">Velvet bean severe mosaic virus DNA A</p><p id="p-2020">Velvet bean severe mosaic virus DNA B</p><p id="p-2021">Vernonia yellow vein betasatellite</p><p id="p-2022">Vernonia yellow vein virus DNA-A</p><p id="p-2023">Vibrio phage fs1</p><p id="p-2024">Vibrio phage fs2</p><p id="p-2025">Vibrio phage K139</p><p id="p-2026">Vibrio phage kappa</p><p id="p-2027">Vibrio phage KSF-1phi</p><p id="p-2028">Vibrio phage KVP40</p><p id="p-2029">Vibrio phage N4</p><p id="p-2030">Vibrio phage VEJphi</p><p id="p-2031">Vibrio phage Vf12</p><p id="p-2032">Vibrio phage VfO3K6</p><p id="p-2033">Vibrio phage VfO4K68</p><p id="p-2034">Vibrio phage VGJphi</p><p id="p-2035">Vibrio phage VHML</p><p id="p-2036">Vibrio phage VP2</p><p id="p-2037">Vibriophage VP4</p><p id="p-2038">Vibrio phage VP5</p><p id="p-2039">Vibrio phage VP882</p><p id="p-2040">Vibrio phage VP93</p><p id="p-2041">Vibrio phage VpV262</p><p id="p-2042">Vibrio phage VSK</p><p id="p-2043">Watermelon chlorotic stunt virus DNA A</p><p id="p-2044">Watermelon chlorotic stunt virus DNA B</p><p id="p-2045">Wheat dwarf virus</p><p id="p-2046">Wissadula golden mosaic St Thomas Virus DNA A</p><p id="p-2047">Wissadula golden mosaic St Thomas Virus DNA B</p><p id="p-2048">Woodchuck hepatitis virus</p><p id="p-2049">WU Polyomavirus</p><p id="p-2050">Xanthomonas phage Cf1c</p><p id="p-2051">Xanthomonas phage OP1</p><p id="p-2052">Xanthomonas phage OP2</p><p id="p-2053">Xanthomonas phage phiL7</p><p id="p-2054">Xanthomonas phage Xop411</p><p id="p-2055">Xanthomonas phage Xp10</p><p id="p-2056">Xanthomonas phage Xp15</p><p id="p-2057">Xenopus laevis endogenous retrovirus Xen1</p><p id="p-2058">Xestia c-nigrum granulovirus</p><p id="p-2059">Xylella phage Xfas53</p><p id="p-2060">Yaba-like disease virus</p><p id="p-2061">Yaba monkey tumor virus</p><p id="p-2062">Yersinia pestis phage phiA1122</p><p id="p-2063">Yersinia phage Berlin</p><p id="p-2064">Yersinia phage L-413C</p><p id="p-2065">Yersinia phage phiYeO3-12</p><p id="p-2066">Yersinia phage PY54</p><p id="p-2067">Zinnia leaf curl disease associated sequence virion</p><p id="p-2068">Zinnia leaf curl virus-associated DNA beta</p></div><div class="section app" id="app-3"><h2 class="">Appendix 3: Bacterial Database</h2><p id="p-2069">Acaryochloris marina MBIC11017</p><p id="p-2070">Acetobacter pasteurianus IFO 3283 01</p><p id="p-2071">Acetohalobium arabaticum DSM 5501</p><p id="p-2072">Acholeplasma laidlawii PG 8A</p><p id="p-2073">Achromobacter xylosoxidans A8</p><p id="p-2074">Acidaminococcus fermentans DSM 20731</p><p id="p-2075">Acidaminococcus intestini RyC MR95</p><p id="p-2076">Acidianus hospitalis W1</p><p id="p-2077">Acidilobus saccharovorans 345 15</p><p id="p-2078">Acidimicrobium ferrooxidans DSM 10331</p><p id="p-2079">Acidiphilium cryptum JF 5</p><p id="p-2080">Acidiphilium multivorum AIU301</p><p id="p-2081">Acidithiobacillus caldus SM 1</p><p id="p-2082">Acidithiobacillus ferrivorans SS3</p><p id="p-2083">Acidithiobacillus ferrooxidans ATCC 23270</p><p id="p-2084">Acidobacterium capsulatum ATCC 51196</p><p id="p-2085">Acidothermus cellulolyticus 11B</p><p id="p-2086">Acidovorax avenae subsp avenae ATCC 19860</p><p id="p-2087">Acidovorax citrulli AAC00 1</p><p id="p-2088">Acidovorax ebreus TPSY</p><p id="p-2089">Acidovorax sp JS42</p><p id="p-2090">Aciduliprofundum boonei T469</p><p id="p-2091">Acinetobacter baumannii 1656 2</p><p id="p-2092">Acinetobacter calcoaceticus PHEA 2</p><p id="p-2093">Acinetobacter oleivorans DR1</p><p id="p-2094">Acinetobacter sp ADP1</p><p id="p-2095">Actinobacillus pleuropneumoniae serovar 5b str L20</p><p id="p-2096">Actinobacillus succinogenes 130Z</p><p id="p-2097">Actinoplanes sp SE50 110</p><p id="p-2098">Actinosynnema mirum DSM 43827</p><p id="p-2099">Aerococcus urinae ACS 120 V Col10a</p><p id="p-2100">Aeromonas hydrophila subsp hydrophila ATCC 7966</p><p id="p-2101">Aeromonas salmonicida subsp salmonicida A449</p><p id="p-2102">Aeromonas veronii B565</p><p id="p-2103">Aeropyrum pernix K1</p><p id="p-2104">Aggregatibacter actinomycetemcomitans D11S 1</p><p id="p-2105">Aggregatibacter aphrophilus NJ8700</p><p id="p-2106">Agrobacterium fabrum str C58</p><p id="p-2107">Agrobacterium radiobacter K84</p><p id="p-2108">Agrobacterium sp H13 3</p><p id="p-2109">Agrobacterium vitis S4</p><p id="p-2110">Akkermansia muciniphila ATCC BAA 835</p><p id="p-2111">Alcanivorax borkumensis SK2</p><p id="p-2112">Alicycliphilus denitrificans BC</p><p id="p-2113">Alicyclobacillus acidocaldarius subsp acidocaldarius DSM 446</p><p id="p-2114">Aliivibrio salmonicida LFI1238</p><p id="p-2115">Alkalilimnicola ehrlichii MLHE 1</p><p id="p-2116">Alkaliphilus metalliredigens QYMF</p><p id="p-2117">Alkaliphilus oremlandii OhILAs</p><p id="p-2118">Allochromatium vinosum DSM 180</p><p id="p-2119">Alteromonas macleodii str Deep ecotype</p><p id="p-2120">Alteromonas sp SN2</p><p id="p-2121">Aminobacterium colombiense DSM 12261</p><p id="p-2122">Ammonifex degensii KC4</p><p id="p-2123">Amycolatopsis mediterranei U32</p><p id="p-2124">Amycolicicoccus subflavus DQS3 9A1</p><p id="p-2125">Anabaena variabilis ATCC 29413</p><p id="p-2126">Anaerococcus prevotii DSM 20548</p><p id="p-2127">Anaerolinea thermophila UNI 1</p><p id="p-2128">Anaeromyxobacter dehalogenans 2CP 1</p><p id="p-2129">Anaeromyxobacter sp Fw109 5</p><p id="p-2130">Anaplasma centrale str Israel</p><p id="p-2131">Anaplasma marginale str Florida</p><p id="p-2132">Anaplasma phagocytophilum HZ</p><p id="p-2133">Anoxybacillus flavithermus WK1</p><p id="p-2134">Aquifex aeolicus VF5</p><p id="p-2135">Arcanobacterium haemolyticum DSM 20595</p><p id="p-2136">Archaeoglobus fulgidus DSM 4304</p><p id="p-2137">Archaeoglobus profundus DSM 5631</p><p id="p-2138">Archaeoglobus veneficus SNP6</p><p id="p-2139">Arcobacter butzleri RM4018</p><p id="p-2140">Arcobacter nitrofigilis DSM 7299</p><p id="p-2141">Arcobacter sp L</p><p id="p-2142">Aromatoleum aromaticum EbN1</p><p id="p-2143">Arthrobacter arilaitensis Re117</p><p id="p-2144">Arthrobacter aurescens TC1</p><p id="p-2145">Arthrobacter chlorophenolicus A6</p><p id="p-2146">Arthrobacter phenanthrenivorans Sphe3</p><p id="p-2147">Arthrobacter sp FB24</p><p id="p-2148">Aster yellows witches broom phytoplasma AYWB</p><p id="p-2149">Asticcacaulis excentricus CB 48</p><p id="p-2150">Atopobium parvulum DSM 20469</p><p id="p-2151">Azoarcus sp BH72</p><p id="p-2152">Azorhizobium caulinodans ORS 571</p><p id="p-2153">Azospirillum sp B510</p><p id="p-2154">Azotobacter vinelandii DJ Bacillus amyloliquefaciens DSM 7</p><p id="p-2155">Bacillus anthracis str Ames</p><p id="p-2156">Bacillus atrophaeus 1942</p><p id="p-2157">Bacillus cellulosilyticus DSM 2522</p><p id="p-2158">Bacillus cereus 03BB102</p><p id="p-2159">Bacillus clausii KSM K16</p><p id="p-2160">Bacillus coagulans 2 6</p><p id="p-2161">Bacillus cytotoxicus NVH 391 98</p><p id="p-2162">Bacillus halodurans C 125</p><p id="p-2163">Bacillus licheniformis DSM 13 ATCC 14580</p><p id="p-2164">Bacillus megaterium DSM 319</p><p id="p-2165">Bacillus mycoides DSM 2048</p><p id="p-2166">Bacillus pseudofirmus OF4</p><p id="p-2167">Bacillus pumilus SAFR 032</p><p id="p-2168">Bacillus selenitireducens MLS10</p><p id="p-2169">Bacillus subtilis BSn5</p><p id="p-2170">Bacillus thuringiensis serovar berliner ATCC 10792</p><p id="p-2171">Bacillus weihenstephanensis KBAB4</p><p id="p-2172">Bacteriovorax marinus SJ</p><p id="p-2173">Bacteroides fragilis YCH46</p><p id="p-2174">Bacteroides helcogenes P 36 108</p><p id="p-2175">Bacteroides salanitronis DSM 18170</p><p id="p-2176">Bacteroides thetaiotaomicron VPI 5482</p><p id="p-2177">Bacteroides vulgatus ATCC 8482</p><p id="p-2178">Bartonella bacilliformis KC583</p><p id="p-2179">Bartonella clarridgeiae 73</p><p id="p-2180">Bartonella grahamii as4aup</p><p id="p-2181">Bartonella henselae str Houston 1</p><p id="p-2182">Bartonella quintana str Toulouse</p><p id="p-2183">Bartonella tribocorum CIP 105476</p><p id="p-2184">Baumannia cicadellinicola str Hc Homalodisca coagulata</p><p id="p-2185">Bdellovibrio bacteriovorus HD100</p><p id="p-2186">Beijerinckia indica subsp indica ATCC 9039</p><p id="p-2187">Beutenbergia cavernae DSM 12333</p><p id="p-2188">Bifidobacterium adolescentis ATCC 15703</p><p id="p-2189">Bifidobacterium animalis subsp animalis ATCC 25527</p><p id="p-2190">Bifidobacterium bifidum S17</p><p id="p-2191">Bifidobacterium breve ACS 071 V Sch8b</p><p id="p-2192">Bifidobacterium dentium Bd1</p><p id="p-2193">Bifidobacterium longum DJO10A</p><p id="p-2194">Blattabacterium sp</p><p id="p-2195">Blattella germanica str Bge</p><p id="p-2196">Bordetella avium 197N</p><p id="p-2197">Bordetella bronchiseptica RB50</p><p id="p-2198">Bordetella parapertussis 12822</p><p id="p-2199">Bordetella pertussis CS</p><p id="p-2200">Bordetella petrii DSM 12804</p><p id="p-2201">Borrelia afzelii PKo</p><p id="p-2202">Borrelia bissettii DN127</p><p id="p-2203">Borrelia burgdorferi B31</p><p id="p-2204">Borrelia duttonii Ly</p><p id="p-2205">Borrelia garinii PBi</p><p id="p-2206">Borrelia hermsii DAH</p><p id="p-2207">Borrelia recurrentis A1</p><p id="p-2208">Borrelia turicatae 91E135</p><p id="p-2209">Brachybacterium faecium DSM 4810</p><p id="p-2210">Brachyspira hyodysenteriae WA1</p><p id="p-2211">Brachyspira intermedia PWS A</p><p id="p-2212">Brachyspira murdochii DSM 12563</p><p id="p-2213">Brachyspira pilosicoli 95 1000</p><p id="p-2214">Bradyrhizobium japonicum USDA 110</p><p id="p-2215">Bradyrhizobium sp BTAi1</p><p id="p-2216">Brevibacillus brevis NBRC 100599</p><p id="p-2217">Brevundimonas subvibrioides ATCC 15264</p><p id="p-2218">Brucella abortus A13334</p><p id="p-2219">Brucella canis ATCC 23365</p><p id="p-2220">Brucella melitensis bv 1 str 16M</p><p id="p-2221">Brucella microti CCM 4915</p><p id="p-2222">Brucella ovis ATCC 25840</p><p id="p-2223">Brucella pinnipedialis B2 94</p><p id="p-2224">Brucella suis 1330</p><p id="p-2225">Buchnera aphidicola str APS Acyrthosiphon pisum</p><p id="p-2226">Burkholderia ambifaria AMMD</p><p id="p-2227">Burkholderia cenocepacia HI2424</p><p id="p-2228">Burkholderia gladioli BSR3</p><p id="p-2229">Burkholderia glumae BGR1</p><p id="p-2230">Burkholderia mallei ATCC 23344</p><p id="p-2231">Burkholderia multivorans ATCC 17616</p><p id="p-2232">Burkholderia phymatum STM815</p><p id="p-2233">Burkholderia phytofirmans PsJN</p><p id="p-2234">Burkholderia pseudomallei 668</p><p id="p-2235">Burkholderia rhizoxinica HKI 454</p><p id="p-2236">Burkholderia sp 383</p><p id="p-2237">Burkholderia thailandensis E264</p><p id="p-2238">Burkholderia vietnamiensis G4</p><p id="p-2239">Burkholderia xenovorans LB400</p><p id="p-2240">Butyrivibrio proteoclasticus B316</p><p id="p-2241">Caldicellulosiruptor bescii DSM 6725</p><p id="p-2242">Caldicellulosiruptor hydrothermalis 108</p><p id="p-2243">Caldicellulosiruptor kristjanssonii 177R1B</p><p id="p-2244">Caldicellulosiruptor kronotskyensis 2002</p><p id="p-2245">Caldicellulosiruptor lactoaceticus 6A</p><p id="p-2246">Caldicellulosiruptor obsidiansis OB47</p><p id="p-2247">Caldicellulosiruptor owensensis OL</p><p id="p-2248">Caldicellulosiruptor saccharolyticus DSM 8903</p><p id="p-2249">Calditerrivibrio nitroreducens DSM 19672</p><p id="p-2250">Caldivirga maquilingensis IC 167</p><p id="p-2251">Campylobacter concisus 13826</p><p id="p-2252">Campylobacter curvus 52592</p><p id="p-2253">Campylobacter fetus subsp fetus 82 40</p><p id="p-2254">Campylobacter hominis ATCC BAA 381</p><p id="p-2255">Campylobacter jejuni subsp jejuni 81116</p><p id="p-2256">Campylobacter lari RM2100</p><p id="p-2257">Candidatus Accumulibacter phosphatis clade IIA str UW 1</p><p id="p-2258">Candidatus Amoebophilus asiaticus 5a2</p><p id="p-2259">Candidatus Arthromitus sp SFB mouse Japan</p><p id="p-2260">Candidatus Azobacteroides pseudotrichonymphae genomovar CFP2</p><p id="p-2261">Candidatus Blochmannia floridanus</p><p id="p-2262">Candidatus Carsonella ruddii PV</p><p id="p-2263">Candidatus Chloracidobacterium thermophilum B</p><p id="p-2264">Candidatus Desulforudis audaxviator MP104C</p><p id="p-2265">Candidatus Hamiltonella defensa 5AT Acyrthosiphon pisum</p><p id="p-2266">Candidatus Hodgkinia cicadicola Dsem</p><p id="p-2267">Candidatus Korarchaeum cryptofilum OPF8</p><p id="p-2268">Candidatus Koribacter versatilis Ellin345</p><p id="p-2269">Candidatus Liberibacter asiaticus str psy62</p><p id="p-2270">Candidatus Methylomirabilis oxyfera</p><p id="p-2271">Candidatus Midichloria mitochondrii IricVA</p><p id="p-2272">Candidatus Moranella endobia PCIT</p><p id="p-2273">Candidatus Nitrospira defluvii</p><p id="p-2274">Candidatus Pelagibacter sp IMCC9063</p><p id="p-2275">Candidatus Phytoplasma australiense</p><p id="p-2276">Candidatus Protochlamydia amoebophila UWE25</p><p id="p-2277">Candidatus Puniceispirillum marinum IMCC1322</p><p id="p-2278">Candidatus Riesia pediculicola USDA</p><p id="p-2279">Candidatus Ruthia magnifica str Cm</p><p id="p-2280">Calyptogena magnifica</p><p id="p-2281">Candidatus Solibacter usitatus Ellin6076</p><p id="p-2282">Candidatus Sulcia muelleri GWSS</p><p id="p-2283">Candidatus Tremblaya princeps PCIT</p><p id="p-2284">Candidatus Vesicomyosocius okutanii HA</p><p id="p-2285">Candidatus Zinderia insecticola CARI</p><p id="p-2286">Capnocytophaga canimorsus Cc5</p><p id="p-2287">Capnocytophaga ochracea DSM 7271</p><p id="p-2288">Carboxydothermus hydrogenoformans Z 2901</p><p id="p-2289">Carnobacterium sp 17 4</p><p id="p-2290">Catenulispora acidiphila DSM 44928</p><p id="p-2291">Caulobacter crescentus CB15</p><p id="p-2292">Caulobacter segnis ATCC 21756</p><p id="p-2293">Caulobacter sp K31</p><p id="p-2294">Cellulomonas fimi ATCC 484</p><p id="p-2295">Cellulomonas flavigena DSM 20109</p><p id="p-2296">Cellulophaga algicola DSM 14237</p><p id="p-2297">Cellulophaga lytica DSM 7489</p><p id="p-2298">Cellvibrio gilvus ATCC 13127</p><p id="p-2299">Cellvibrio japonicus Ueda107</p><p id="p-2300">Cenarchaeum symbiosum A</p><p id="p-2301">Chelativorans sp BNC1</p><p id="p-2302">Chitinophaga pinensis DSM 2588</p><p id="p-2303">Chlamydia muridarum Nigg</p><p id="p-2304">Chlamydia trachomatis 434 Bu</p><p id="p-2305">Chlamydophila abortus S26 3</p><p id="p-2306">Chlamydophila caviae GPIC</p><p id="p-2307">Chlamydophila felis Fe C 56</p><p id="p-2308">Chlamydophila pecorum E58</p><p id="p-2309">Chlamydophila pneumoniae CWL029</p><p id="p-2310">Chlamydophila psittaci 6BC</p><p id="p-2311">Chlorobaculum parvum NCIB 8327</p><p id="p-2312">Chlorobium chlorochromatii CaD3</p><p id="p-2313">Chlorobium limicola DSM 245</p><p id="p-2314">Chlorobium luteolum DSM 273</p><p id="p-2315">Chlorobium phaeobacteroides DSM 266</p><p id="p-2316">Chlorobium phaeovibrioides DSM 265</p><p id="p-2317">Chlorobium tepidum TLS</p><p id="p-2318">Chloroflexus aggregans DSM 9485</p><p id="p-2319">Chloroflexus aurantiacus J 10 fl</p><p id="p-2320">Chloroflexus sp Y 400 fl</p><p id="p-2321">Chloroherpeton thalassium ATCC 35110</p><p id="p-2322">Chromobacterium violaceum ATCC 12472</p><p id="p-2323">Chromohalobacter salexigens DSM 3043</p><p id="p-2324">Citrobacter koseri ATCC BAA 895</p><p id="p-2325">Citrobacter rodentium ICC168</p><p id="p-2326">Clavibacter michiganensis subsp michiganensis NCPPB 382</p><p id="p-2327">Clostridium acetobutylicum ATCC 824</p><p id="p-2328">Clostridium beijerinckii NCIMB 8052</p><p id="p-2329">Clostridium botulinum A str ATCC 3502</p><p id="p-2330">Clostridium cellulolyticum H10</p><p id="p-2331">Clostridium cellulovorans 743B</p><p id="p-2332">Clostridium difficile 630</p><p id="p-2333">Clostridium kluyveri DSM 555</p><p id="p-2334">Clostridium lentocellum DSM 5427</p><p id="p-2335">Clostridium ljungdahlii DSM 13528</p><p id="p-2336">Clostridium novyi NT</p><p id="p-2337">Clostridium perfringens ATCC 13124</p><p id="p-2338">Clostridium phytofermentans ISDg</p><p id="p-2339">Clostridium saccharolyticum WM1</p><p id="p-2340">Clostridium sp SY8519</p><p id="p-2341">Clostridium sticklandii DSM 519</p><p id="p-2342">Clostridium tetani E88</p><p id="p-2343">Clostridium thermocellum ATCC 27405</p><p id="p-2344">Collimonas fungivorans Ter331</p><p id="p-2345">Colwellia psychrerythraea 34H</p><p id="p-2346">Comamonas testosteroni CNB 2</p><p id="p-2347">Conexibacter woesei DSM 14684</p><p id="p-2348">Coprothermobacter proteolyticus DSM 5265</p><p id="p-2349">Coraliomargarita akajimensis DSM 45221</p><p id="p-2350">Coriobacterium glomerans PW2</p><p id="p-2351">Corynebacterium aurimucosum ATCC 700975</p><p id="p-2352">Corynebacterium diphtheriae NCTC 13129</p><p id="p-2353">Corynebacterium efficiens YS 314</p><p id="p-2354">Corynebacterium glutamicum ATCC 13032</p><p id="p-2355">Corynebacterium jeikeium K411</p><p id="p-2356">Corynebacterium kroppenstedtii DSM 44385</p><p id="p-2357">Corynebacterium pseudotuberculosis FRC41</p><p id="p-2358">Corynebacterium resistens DSM 45100</p><p id="p-2359">Corynebacterium ulcerans 809</p><p id="p-2360">Corynebacterium urealyticum DSM 7109</p><p id="p-2361">Corynebacterium variabile DSM 44702</p><p id="p-2362">Coxiella burnetii RSA 493</p><p id="p-2363">Croceibacter atlanticus HTCC2559</p><p id="p-2364">Cronobacter sakazakii ATCC BAA 894</p><p id="p-2365">Cronobacter turicensis z3032</p><p id="p-2366">Cryptobacterium curtum DSM 15641</p><p id="p-2367">Cupriavidus metallidurans CH34</p><p id="p-2368">Cupriavidus necator N 1</p><p id="p-2369">Cupriavidus taiwanensis LMG 19424</p><p id="p-2370">cyanobacterium UCYN A</p><p id="p-2371">Cyanothece sp ATCC 51142</p><p id="p-2372">Cyclobacterium marinum DSM 745</p><p id="p-2373">Cytophaga hutchinsonii ATCC 33406</p><p id="p-2374">Dechloromonas aromatica RCB</p><p id="p-2375">Deferribacter desulfuricans SSM1</p><p id="p-2376">Dehalococcoides ethenogenes 195</p><p id="p-2377">Dehalococcoides sp BAV1</p><p id="p-2378">Dehalogenimonas lykanthroporepellens BL DC 9</p><p id="p-2379">Deinococcus deserti VCD115 Deinococcus geothermalis DSM 11300</p><p id="p-2380">Deinococcus maricopensis DSM 21211</p><p id="p-2381">Deinococcus proteolyticus MRP</p><p id="p-2382">Deinococcus radiodurans R1</p><p id="p-2383">Delftia acidovorans SPH 1 Delftia sp Cs1 4</p><p id="p-2384">Denitrovibrio acetiphilus DSM 12809</p><p id="p-2385">Desulfarculus baarsii DSM 2075</p><p id="p-2386">Desulfatibacillum alkenivorans AK 01</p><p id="p-2387">Desulfitobacterium hafniense Y51</p><p id="p-2388">Desulfobacca acetoxidans DSM 11109</p><p id="p-2389">Desulfobacterium autotrophicum HRM2</p><p id="p-2390">Desulfobulbus propionicus DSM 2032</p><p id="p-2391">Desulfococcus oleovorans Hxd3</p><p id="p-2392">Desulfohalobium retbaense DSM 5692</p><p id="p-2393">Desulfomicrobium baculatum DSM 4028</p><p id="p-2394">Desulfotalea psychrophila LSv54</p><p id="p-2395">Desulfotomaculum acetoxidans DSM 771</p><p id="p-2396">Desulfotomaculum carboxydivorans CO 1 SRB</p><p id="p-2397">Desulfotomaculum kuznetsovii DSM 6115</p><p id="p-2398">Desulfotomaculum reducens MI 1</p><p id="p-2399">Desulfotomaculum ruminis DSM 2154</p><p id="p-2400">Desulfovibrio aespoeensis Aspo 2</p><p id="p-2401">Desulfovibrio africanus str Walvis Bay</p><p id="p-2402">Desulfovibrio alaskensis G20</p><p id="p-2403">Desulfovibrio desulfuricans ND132</p><p id="p-2404">Desulfovibrio magneticus RS 1</p><p id="p-2405">Desulfovibrio salexigens DSM 2638</p><p id="p-2406">Desulfovibrio vulgaris RCH1</p><p id="p-2407">Desulfurispirillum indicum S5</p><p id="p-2408">Desulfurivibrio alkaliphilus AHT2</p><p id="p-2409">Desulfurobacterium thermolithotrophum DSM 11699</p><p id="p-2410">Desulfurococcus kamchatkensis 1221n</p><p id="p-2411">Desulfurococcus mucosus DSM 2162</p><p id="p-2412">Dichelobacter nodosus VCS1703A</p><p id="p-2413">Dickeya dadantii 3937</p><p id="p-2414">Dickeya zeae Ech1591</p><p id="p-2415">Dictyoglomus thermophilum H 6 12</p><p id="p-2416">Dictyoglomus turgidum DSM 6724</p><p id="p-2417">Dinoroseobacter shibae DFL 12</p><p id="p-2418">Dyadobacter fermentans DSM 18053</p><p id="p-2419">Edwardsiella ictaluri 93 146</p><p id="p-2420">Edwardsiella tarda EIB202</p><p id="p-2421">Eggerthella lenta DSM 2243</p><p id="p-2422">Eggerthella sp YY7918</p><p id="p-2423">Ehrlichia canis str Jake</p><p id="p-2424">Ehrlichia chaffeensis str Arkansas</p><p id="p-2425">Ehrlichia ruminantium str Welgevonden</p><p id="p-2426">Elusimicrobium minutum Pei191</p><p id="p-2427">Enterobacter aerogenes KCTC 2190</p><p id="p-2428">Enterobacter asburiae LF7a</p><p id="p-2429">Enterobacter cloacae subsp cloacae ATCC 13047</p><p id="p-2430">Enterobacter sp 638 Enterococcus faecalis V583</p><p id="p-2431">Erwinia amylovora ATCC 49946</p><p id="p-2432">Erwinia billingiae Eb661</p><p id="p-2433">Erwinia pyrifoliae DSM 12163</p><p id="p-2434">Erwinia sp Ejp617</p><p id="p-2435">Erwinia tasmaniensis Et1 99</p><p id="p-2436">Erysipelothrix rhusiopathiae</p><p id="p-2437">Erythrobacter litoralis HTCC2594</p><p id="p-2438">Escherichia coli O157:H7 str Sakai</p><p id="p-2439">Escherichia fergusonii ATCC 35469</p><p id="p-2440">Ethanoligenens harbinense YUAN 3</p><p id="p-2441">Eubacterium eligens ATCC 27750</p><p id="p-2442">Eubacterium limosum KIST612</p><p id="p-2443">Eubacterium rectale ATCC 33656</p><p id="p-2444">Exiguobacterium sibiricum 25515</p><p id="p-2445">Exiguobacterium sp AT1b</p><p id="p-2446">Ferrimonas balearica DSM 9799</p><p id="p-2447">Ferroglobus placidus DSM 10642</p><p id="p-2448">Fervidobacterium nodosum Rt17 B1</p><p id="p-2449">Fibrobacter succinogenes subsp succinogenes S85</p><p id="p-2450">Filifactor alocis ATCC 35896</p><p id="p-2451">Finegoldia magna ATCC 29328</p><p id="p-2452">Flavobacteriaceae bacterium 3519 10</p><p id="p-2453">Flavobacterium branchiophilum FL 15</p><p id="p-2454">Flavobacterium columnare ATCC 49512</p><p id="p-2455">Flavobacterium johnsoniae UW101</p><p id="p-2456">Flavobacterium psychrophilum JIP02 86</p><p id="p-2457">Flexistipes sinusarabici DSM 4947</p><p id="p-2458">Fluviicola taffensis DSM 16823</p><p id="p-2459">Francisella novicida U112</p><p id="p-2460">Francisella philomiragia subsp philomiragia ATCC 25017</p><p id="p-2461">Francisella sp TX077308</p><p id="p-2462">Francisella tularensis subsp holarctica LVS</p><p id="p-2463">Frankia alni ACN14a</p><p id="p-2464">Frankia sp CcI3</p><p id="p-2465">Frankia symbiont of Datisca glomerata</p><p id="p-2466">Fusobacterium nucleatum subsp nucleatum ATCC 25586</p><p id="p-2467">Gallibacterium anatis UMN179</p><p id="p-2468">Gallionella capsiferriformans ES 2 gamma proteobacterium HdN1</p><p id="p-2469">Gardnerella vaginalis 409 05</p><p id="p-2470">Gemmatimonas aurantiaca T 27</p><p id="p-2471">Geobacillus kaustophilus HTA426</p><p id="p-2472">Geobacillus sp C56 T3</p><p id="p-2473">Geobacillus thermodenitrificans NG80 2</p><p id="p-2474">Geobacillus thermoglucosidasius C56 YS93</p><p id="p-2475">Geobacillus thermoleovorans CCB US3 UF5</p><p id="p-2476">Geobacter bemidjiensis Bem</p><p id="p-2477">Geobacter daltonii FRC 32</p><p id="p-2478">Geobacter lovleyi SZ</p><p id="p-2479">Geobacter metallireducens GS 15</p><p id="p-2480">Geobacter sp M18</p><p id="p-2481">Geobacter sulfurreducens PCA</p><p id="p-2482">Geobacter uraniireducens Rf4</p><p id="p-2483">Geodermatophilus obscurus DSM 43160</p><p id="p-2484">Glaciecola nitratireducens FR1064</p><p id="p-2485">Glaciecola sp 4H 3 7YE 5</p><p id="p-2486">Gloeobacter violaceus PCC 7421</p><p id="p-2487">Gluconacetobacter diazotrophicus PAl 5</p><p id="p-2488">Gluconacetobacter xylinus NBRC 3288</p><p id="p-2489">Gluconobacter oxydans 621H</p><p id="p-2490">Gordonia bronchialis DSM 43247</p><p id="p-2491">Gramella forsetii KT0803</p><p id="p-2492">Granulibacter bethesdensis CGDNIH1</p><p id="p-2493">Granulicella mallensis MP5ACTX8</p><p id="p-2494">Granulicella tundricola</p><p id="p-2495">Haemophilus ducreyi 35000HP</p><p id="p-2496">Haemophilus influenzae 10810</p><p id="p-2497">Haemophilus parainfluenzae T3T1</p><p id="p-2498">Haemophilus parasuis SH0165</p><p id="p-2499">Haemophilus somnus 129PT</p><p id="p-2500">Hahella chejuensis KCTC 2396</p><p id="p-2501">Halalkalicoccus jeotgali B3</p><p id="p-2502">Halanaerobium hydrogeniformans</p><p id="p-2503">Haliangium ochraceum DSM 14365</p><p id="p-2504">Haliscomenobacter hydrossis DSM 1100</p><p id="p-2505">Haloarcula hispanica ATCC 33960</p><p id="p-2506">Haloarcula marismortui ATCC 43049</p><p id="p-2507">Halobacterium sp NRC 1</p><p id="p-2508">Haloferax volcanii DS2</p><p id="p-2509">Halogeometricum borinquense DSM 11551</p><p id="p-2510">Halomicrobium mukohataei DSM 12286</p><p id="p-2511">Halomonas elongata DSM 2581</p><p id="p-2512">halophilic archaeon DL31</p><p id="p-2513">Halopiger xanaduensis SH 6</p><p id="p-2514">Haloquadratum walsbyi C23</p><p id="p-2515">Halorhabdus utahensis DSM 12940</p><p id="p-2516">Halorhodospira halophila SL1</p><p id="p-2517">Halorubrum lacusprofundi ATCC 49239</p><p id="p-2518">Haloterrigena turkmenica DSM 5511</p><p id="p-2519">Halothermothrix orenii H 168</p><p id="p-2520">Halothiobacillus neapolitanus c2</p><p id="p-2521">Helicobacter acinonychis str Sheeba</p><p id="p-2522">Helicobacter bizzozeronii CIII 1</p><p id="p-2523">Helicobacter felis ATCC 49179</p><p id="p-2524">Helicobacter hepaticus ATCC 51449</p><p id="p-2525">Helicobacter mustelae 12198</p><p id="p-2526">Helicobacter pylori 26695</p><p id="p-2527">Heliobacterium modesticaldum Ice1</p><p id="p-2528">Herbaspirillum seropedicae SmR1</p><p id="p-2529">Herminiimonas arsenicoxydans</p><p id="p-2530">Herpetosiphon aurantiacus DSM 785</p><p id="p-2531">Hippea maritima DSM 10411</p><p id="p-2532">Hirschia baltica ATCC 49814</p><p id="p-2533">Hydrogenobacter thermophilus TK 6</p><p id="p-2534">Hydrogenobaculum sp Y04AAS1</p><p id="p-2535">Hyperthermus butylicus DSM 5456</p><p id="p-2536">Hyphomicrobium denitrificans ATCC 51888</p><p id="p-2537">Hyphomicrobium sp</p><p id="p-2538">Hyphomonas neptunium ATCC 15444</p><p id="p-2539">Idiomarina loihiensis L2TR</p><p id="p-2540">Ignicoccus hospitalis KIN4 I</p><p id="p-2541">Ignisphaera aggregans DSM 17230</p><p id="p-2542">Ilyobacter polytropus DSM 2926</p><p id="p-2543">Intrasporangium calvum DSM 43043</p><p id="p-2544">Isoptericola variabilis 225</p><p id="p-2545">Isosphaera pallida ATCC 43644</p><p id="p-2546">Jannaschia sp CCS1</p><p id="p-2547">Janthinobacterium sp Marseille</p><p id="p-2548">Jonesia denitrificans DSM 20603</p><p id="p-2549">Kangiella koreensis DSM 16069</p><p id="p-2550">Ketogulonicigenium vulgare WSH 001</p><p id="p-2551">Kineococcus radiotolerans SRS30216</p><p id="p-2552">Kitasatospora setae KM 6054</p><p id="p-2553">Klebsiella oxytoca KCTC 1686</p><p id="p-2554">Klebsiella pneumoniae 342</p><p id="p-2555">Klebsiella variicola At 22</p><p id="p-2556">Kocuria rhizophila DC2201</p><p id="p-2557">Kosmotoga olearia TBF 1951</p><p id="p-2558">Kribbella flavida DSM 17836</p><p id="p-2559">Krokinobacter sp 4H 3 7 5</p><p id="p-2560">Kyrpidia tusciae DSM 2912</p><p id="p-2561">Kytococcus sedentarius DSM 20547</p><p id="p-2562">Lacinutrix sp 5H 3 7 4</p><p id="p-2563">Lactobacillus acidophilus NCFM</p><p id="p-2564">Lactobacillus amylovorus GRL 1112</p><p id="p-2565">Lactobacillus brevis ATCC 367</p><p id="p-2566">Lactobacillus buchneri NRRL B 30929</p><p id="p-2567">Lactobacillus casei ATCC 334</p><p id="p-2568">Lactobacillus crispatus ST1</p><p id="p-2569">Lactobacillus delbrueckii subsp bulgaricus ATCC 11842</p><p id="p-2570">Lactobacillus fermentum IFO 3956</p><p id="p-2571">Lactobacillus gasseri ATCC 33323</p><p id="p-2572">Lactobacillus helveticus DPC 4571</p><p id="p-2573">Lactobacillus johnsonii NCC 533</p><p id="p-2574">Lactobacillus kefiranofaciens ZW3</p><p id="p-2575">Lactobacillus plantarum subsp plantarum ST III</p><p id="p-2576">Lactobacillus reuteri DSM 20016</p><p id="p-2577">Lactobacillus rhamnosus ATCC 8530</p><p id="p-2578">Lactobacillus ruminis ATCC 27782</p><p id="p-2579">Lactobacillus sakei subsp sakei 23K</p><p id="p-2580">Lactobacillus salivarius UCC118</p><p id="p-2581">Lactobacillus sanfranciscensis TMW 11304</p><p id="p-2582">Lactococcus garvieae ATCC 49156</p><p id="p-2583">Lactococcus lactis subsp cremoris NZ9000</p><p id="p-2584">Laribacter hongkongensis HLHK9</p><p id="p-2585">Lawsonia intracellularis PHE MN1 00</p><p id="p-2586">Leadbetterella byssophila DSM 17132</p><p id="p-2587">Legionella longbeachae NSW150</p><p id="p-2588">Legionella pneumophila subsp pneumophila ATCC 43290</p><p id="p-2589">Leifsonia xyli subsp xyli str CTCB07</p><p id="p-2590">Leptospira biflexa serovar Patoc strain Patoc 1 Ames</p><p id="p-2591">Leptospira borgpetersenii serovar Hardjo bovis str L550</p><p id="p-2592">Leptospira interrogans serovar Copenhageni str Fiocruz L1 130</p><p id="p-2593">Leptothrix cholodnii SP 6</p><p id="p-2594">Leptotrichia buccalis C 1013 b</p><p id="p-2595">Leuconostoc citreum KM20</p><p id="p-2596">Leuconostoc gasicomitatum LMG 18811</p><p id="p-2597">Leuconostoc kimchii IMSNU 11154</p><p id="p-2598">Leuconostoc mesenteroides subsp mesenteroides ATCC 8293</p><p id="p-2599">Leuconostoc sp C2 Listeria innocua Clip11262 Listeria ivanovii</p><p id="p-2600">Listeria monocytogenes EGD e</p><p id="p-2601">Listeria seeligeri serovar 1 2b str SLCC3954</p><p id="p-2602">Listeria welshimeri serovar 6b str SLCC5334</p><p id="p-2603">Lysinibacillus sphaericus C3 41</p><p id="p-2604">Macrococcus caseolyticus JCSC5402</p><p id="p-2605">Magnetococcus marinus MC 1</p><p id="p-2606">Magnetospirillum magneticum AMB 1</p><p id="p-2607">Mahella australiensis 50 1 BON</p><p id="p-2608">Mannheimia succiniciproducens MBEL55E</p><p id="p-2609">Maribacter sp HTCC2170</p><p id="p-2610">Maricaulis maris MCS10</p><p id="p-2611">Marinithermus hydrothermalis DSM 14884</p><p id="p-2612">Marinobacter adhaerens HP15</p><p id="p-2613">Marinobacter aquaeolei VT8</p><p id="p-2614">Marinomonas mediterranea MMB 1</p><p id="p-2615">Marinomonas posidonica IVIA Po 181</p><p id="p-2616">Marinomonas sp MWYL1</p><p id="p-2617">Marivirga tractuosa DSM 4126</p><p id="p-2618">Megasphaera elsdenii</p><p id="p-2619">Meiothermus ruber DSM 1279</p><p id="p-2620">Meiothermus silvanus DSM 9946</p><p id="p-2621">Melissococcus plutonius ATCC 35311</p><p id="p-2622">Mesoplasma florum L1</p><p id="p-2623">Mesorhizobium ciceri biovar biserrulae WSM1271</p><p id="p-2624">Mesorhizobium loti MAFF303099</p><p id="p-2625">Mesorhizobium opportunistum WSM2075</p><p id="p-2626">Metallosphaera cuprina Ar 4</p><p id="p-2627">Metallosphaera sedula DSM 5348</p><p id="p-2628">Methanobacterium sp AL 21</p><p id="p-2629">Methanobrevibacter ruminantium M1</p><p id="p-2630">Methanobrevibacter smithii ATCC 35061</p><p id="p-2631">Methanocaldococcus fervens AG86</p><p id="p-2632">Methanocaldococcus infernus ME</p><p id="p-2633">Methanocaldococcus jannaschii DSM 2661</p><p id="p-2634">Methanocaldococcus sp FS406 22</p><p id="p-2635">Methanocaldococcus vulcanius M7</p><p id="p-2636">Methanocella arvoryzae MRE50</p><p id="p-2637">Methanocella paludicola SANAE</p><p id="p-2638">Methanococcoides burtonii DSM 6242</p><p id="p-2639">Methanococcus aeolicus Nankai 3</p><p id="p-2640">Methanococcus maripaludis S2</p><p id="p-2641">Methanococcus vannielii SB</p><p id="p-2642">Methanococcus voltae A3</p><p id="p-2643">Methanocorpusculum labreanum Z</p><p id="p-2644">Methanoculleus marisnigri JR1</p><p id="p-2645">Methanohalobium evestigatum Z 7303</p><p id="p-2646">Methanohalophilus mahii DSM 5219</p><p id="p-2647">Methanoplanus petrolearius DSM 11571</p><p id="p-2648">Methanopyrus kandleri AV19</p><p id="p-2649">Methanoregula boonei 6A8</p><p id="p-2650">Methanosaeta concilii GP6</p><p id="p-2651">Methanosaeta thermophila PT</p><p id="p-2652">Methanosalsum zhilinae DSM 4017</p><p id="p-2653">Methanosarcina acetivorans C2A</p><p id="p-2654">Methanosarcina barkeri str Fusaro</p><p id="p-2655">Methanosarcina mazei Go1</p><p id="p-2656">Methanosphaera stadtmanae DSM 3091</p><p id="p-2657">Methanosphaerula palustris E1 9c</p><p id="p-2658">Methanospirillum hungatei JF 1</p><p id="p-2659">Methanothermobacter marburgensis str Marburg</p><p id="p-2660">Methanothermobacter thermautotrophicus str Delta H</p><p id="p-2661">Methanothermococcus okinawensis IH1</p><p id="p-2662">Methanothermus fervidus DSM 2088</p><p id="p-2663">Methanotorris igneus Kol 5</p><p id="p-2664">Methylacidiphilum infernorum V4</p><p id="p-2665">Methylibium petroleiphilum PM1</p><p id="p-2666">Methylobacillus flagellatus KT</p><p id="p-2667">Methylobacterium chloromethanicum CM4</p><p id="p-2668">Methylobacterium extorquens AM1</p><p id="p-2669">Methylobacterium nodulans ORS 2060</p><p id="p-2670">Methylobacterium populi BJ001</p><p id="p-2671">Methylobacterium radiotolerans JCM 2831</p><p id="p-2672">Methylobacterium sp 4 46</p><p id="p-2673">Methylocella silvestris BL2</p><p id="p-2674">Methylococcus capsulatus str Bath</p><p id="p-2675">Methylomicrobium alcaliphilum</p><p id="p-2676">Methylomonas methanica MC09</p><p id="p-2677">Methylotenera mobilis JLW8</p><p id="p-2678">Methylotenera versatilis 301</p><p id="p-2679">Methylovorus glucosetrophus SIP3 4</p><p id="p-2680">Methylovorus sp MP688</p><p id="p-2681">Micavibrio aeruginosavorus ARL 13</p><p id="p-2682">Microbacterium testaceum StLB037</p><p id="p-2683">Micrococcus luteus NCTC 2665</p><p id="p-2684">Microcystis aeruginosa NIES 843</p><p id="p-2685">Microlunatus phosphovorus NM 1</p><p id="p-2686">Micromonospora aurantiaca ATCC 27029</p><p id="p-2687">Micromonospora sp L5</p><p id="p-2688">Mobiluncus curtisii ATCC 43063</p><p id="p-2689">Moorella thermoacetica ATCC 39073</p><p id="p-2690">Moraxella catarrhalis RH4</p><p id="p-2691">Muricauda ruestringensis DSM 13258</p><p id="p-2692">Mycobacterium abscessus ATCC 19977</p><p id="p-2693">Mycobacterium africanum GM041182</p><p id="p-2694">Mycobacterium avium 104</p><p id="p-2695">Mycobacterium bovis AF2122 97</p><p id="p-2696">Mycobacterium canettii CIPT 140010059</p><p id="p-2697">Mycobacterium gilvum PYR GCK</p><p id="p-2698">Mycobacterium leprae TN</p><p id="p-2699">Mycobacterium marinum M</p><p id="p-2700">Mycobacterium rhodesiae NBB3</p><p id="p-2701">Mycobacterium smegmatis str MC2 155</p><p id="p-2702">Mycobacterium sp JDM601</p><p id="p-2703">Mycobacterium tuberculosis H37Rv</p><p id="p-2704">Mycobacterium ulcerans Agy99</p><p id="p-2705">Mycobacterium vanbaalenii PYR 1</p><p id="p-2706">Mycoplasma agalactiae PG2</p><p id="p-2707">Mycoplasma arthritidis 158L3 1</p><p id="p-2708">Mycoplasma bovis PG45</p><p id="p-2709">Mycoplasma capricolum subsp capricolum ATCC 27343</p><p id="p-2710">Mycoplasma conjunctivae HRC 581</p><p id="p-2711">Mycoplasma crocodyli MP145</p><p id="p-2712">Mycoplasma gallisepticum str Rlow</p><p id="p-2713">Mycoplasma genitalium G37</p><p id="p-2714">Mycoplasma haemocanis str Illinois</p><p id="p-2715">Mycoplasma haemofelis str Langford 1</p><p id="p-2716">Mycoplasma hominis</p><p id="p-2717">Mycoplasma hyopneumoniae 232</p><p id="p-2718">Mycoplasma hyorhinis HUB 1</p><p id="p-2719">Mycoplasma leachii 99 014 6</p><p id="p-2720">Mycoplasma mobile 163K</p><p id="p-2721">Mycoplasma mycoides subsp mycoides SC str PG1</p><p id="p-2722">Mycoplasma penetrans HF 2</p><p id="p-2723">Mycoplasma pneumoniae M129</p><p id="p-2724">Mycoplasma pulmonis UAB CTIP</p><p id="p-2725">Mycoplasma putrefaciens KS1</p><p id="p-2726">Mycoplasma suis str Illinois</p><p id="p-2727">Mycoplasma synoviae 53</p><p id="p-2728">Myxococcus fulvus HW 1</p><p id="p-2729">Myxococcus xanthus DK 1622</p><p id="p-2730">Nakamurella multipartita DSM 44233</p><p id="p-2731">Nanoarchaeum equitans Kin4 M</p><p id="p-2732">Natranaerobius thermophilus JW NM WN LF</p><p id="p-2733">Natrialba magadii ATCC 43099</p><p id="p-2734">Natronomonas pharaonis DSM 2160</p><p id="p-2735">Nautilia profundicola AmH Neisseria gonorrhoeae FA 1090</p><p id="p-2736">Neisseria lactamica 020 06</p><p id="p-2737">Neisseria meningitidis M01 240149</p><p id="p-2738">Neorickettsia risticii str Illinois</p><p id="p-2739">Neorickettsia sennetsu str Miyayama</p><p id="p-2740">Niastella koreensis GR20 10</p><p id="p-2741">Nitratifractor salsuginis DSM 16511</p><p id="p-2742">Nitratiruptor sp SB155 2</p><p id="p-2743">Nitrobacter hamburgensis X14</p><p id="p-2744">Nitrobacter winogradskyi Nb 255</p><p id="p-2745">Nitrosococcus halophilus Nc4</p><p id="p-2746">Nitrosococcus oceani ATCC 19707</p><p id="p-2747">Nitrosococcus watsonii C 113</p><p id="p-2748">Nitrosomonas europaea ATCC 19718</p><p id="p-2749">Nitrosomonas eutropha C91</p><p id="p-2750">Nitrosomonas sp AL212</p><p id="p-2751">Nitrosopumilus maritimus SCM1</p><p id="p-2752">Nitrosospira multiformis ATCC 25196</p><p id="p-2753">Nocardia farcinica IFM 10152</p><p id="p-2754">Nocardioides sp JS614</p><p id="p-2755">Nocardiopsis dassonvillei subsp dassonvillei DSM 43111</p><p id="p-2756">Nostoc azollae 0708</p><p id="p-2757">Nostoc punctiforme PCC 73102</p><p id="p-2758">Nostoc sp PCC 7120</p><p id="p-2759">Novosphingobium aromaticivorans DSM 12444</p><p id="p-2760">Novosphingobium sp PP1Y Oceanithermus profundus DSM 14977</p><p id="p-2761">Oceanobacillus iheyensis HTE831</p><p id="p-2762">Ochrobactrum anthropi ATCC 49188</p><p id="p-2763">Odoribacter splanchnicus DSM 20712</p><p id="p-2764">Oenococcus oeni PSU 1</p><p id="p-2765">Oligotropha carboxidovorans OM5</p><p id="p-2766">Olsenella uli DSM 7084</p><p id="p-2767">Onion yellows phytoplasma OY M</p><p id="p-2768">Opitutus terrae PB90 1</p><p id="p-2769">Orientia tsutsugamushi str Boryong</p><p id="p-2770">Oscillibacter valericigenes</p><p id="p-2771">Owenweeksia hongkongensis DSM 17368</p><p id="p-2772">Paenibacillus mucilaginosus KNP414</p><p id="p-2773">Paenibacillus polymyxa E681</p><p id="p-2774">Paenibacillus sp JDR 2</p><p id="p-2775">Paenibacillus terrae HPL 003</p><p id="p-2776">Paludibacter propionicigenes WB4</p><p id="p-2777">Pantoea ananatis LMG 20103</p><p id="p-2778">Pantoea sp At 9b</p><p id="p-2779">Pantoea vagans C9 1</p><p id="p-2780">Parabacteroides distasonis ATCC 8503</p><p id="p-2781">Parachlamydia acanthamoebae UV 7</p><p id="p-2782">Paracoccus denitrificans PD1222</p><p id="p-2783">Parvibaculum lavamentivorans DS 1</p><p id="p-2784">Parvularcula bermudensis HTCC2503</p><p id="p-2785">Pasteurella multocida subsp multocida str Pm70</p><p id="p-2786">Pectobacterium atrosepticum SCRI1043</p><p id="p-2787">Pectobacterium carotovorum subsp carotovorum PC1</p><p id="p-2788">Pectobacterium wasabiae WPP163</p><p id="p-2789">Pediococcus pentosaceus ATCC 25745</p><p id="p-2790">Pedobacter heparinus DSM 2366</p><p id="p-2791">Pedobacter saltans DSM 12145</p><p id="p-2792">Pelagibacterium halotolerans B2</p><p id="p-2793">Pelobacter carbinolicus DSM 2380</p><p id="p-2794">Pelobacter propionicus DSM 2379</p><p id="p-2795">Pelodictyon phaeoclathratiforme BU 1</p><p id="p-2796">Pelotomaculum thermopropionicum SI</p><p id="p-2797">Persephonella marina EX H1</p><p id="p-2798">Petrotoga mobilis SJ95</p><p id="p-2799">Phenylobacterium zucineum HLK1</p><p id="p-2800">Photobacterium profundum SS9</p><p id="p-2801">Photorhabdus asymbiotica subsp asymbiotica ATCC 43949</p><p id="p-2802">Photorhabdus luminescens subsp laumondii TTO1</p><p id="p-2803">Picrophilus torridus DSM 9790</p><p id="p-2804">Pirellula staleyi DSM 6068</p><p id="p-2805">Planctomyces brasiliensis DSM 5305</p><p id="p-2806">Planctomyces limnophilus DSM 3776</p><p id="p-2807">Polaromonas naphthalenivorans CJ2</p><p id="p-2808">Polaromonas sp JS666</p><p id="p-2809">Polymorphum gilvum SL003B 26A1</p><p id="p-2810">Polynucleobacter necessarius subsp asymbioticus QLW P1DMWA 1</p><p id="p-2811">Porphyromonas asaccharolytica DSM 20707</p><p id="p-2812">Porphyromonas gingivalis W83</p><p id="p-2813">Prevotella denticola F0289</p><p id="p-2814">Prevotella melaninogenica ATCC 25845</p><p id="p-2815">Prevotella ruminicola 23</p><p id="p-2816">Prochlorococcus marinus str MIT 9215</p><p id="p-2817">Propionibacterium acnes 6609</p><p id="p-2818">Propionibacterium freudenreichii subsp shermanii CIRM BIA1</p><p id="p-2819">Prosthecochloris aestuarii DSM 271</p><p id="p-2820">Proteus mirabilis HI4320</p><p id="p-2821">Pseudoalteromonas atlantica T6c</p><p id="p-2822">Pseudoalteromonas haloplanktis TAC125</p><p id="p-2823">Pseudoalteromonas sp SM9913</p><p id="p-2824">Pseudogulbenkiania sp NH8B</p><p id="p-2825">Pseudomonas aeruginosa PAO1</p><p id="p-2826">Pseudomonas brassicacearum subsp brassicacearum NFM421</p><p id="p-2827">Pseudomonas entomophila L48</p><p id="p-2828">Pseudomonas fluorescens F113</p><p id="p-2829">Pseudomonas fulva 12 X</p><p id="p-2830">Pseudomonas mendocina ymp</p><p id="p-2831">Pseudomonas protegens Pf 5</p><p id="p-2832">Pseudomonas putida F1</p><p id="p-2833">Pseudomonas stutzeri A1501</p><p id="p-2834">Pseudomonas syringae pv phaseolicola 1448A</p><p id="p-2835">Pseudonocardia dioxanivorans CB1190</p><p id="p-2836">Pseudovibrio sp FO BEG1</p><p id="p-2837">Pseudoxanthomonas spadix BD a59</p><p id="p-2838">Pseudoxanthomonas suwonensis 11 1</p><p id="p-2839">Psychrobacter arcticus 273 4</p><p id="p-2840">Psychrobacter cryohalolentis K5</p><p id="p-2841">Psychrobacter sp PRwf 1</p><p id="p-2842">Psychromonas ingrahamii 37</p><p id="p-2843">Pusillimonas sp T7 7</p><p id="p-2844">Pyrobaculum aerophilum str IM2</p><p id="p-2845">Pyrobaculum arsenaticum DSM 13514</p><p id="p-2846">Pyrobaculum calidifontis JCM 11548</p><p id="p-2847">Pyrobaculum islandicum DSM 4184</p><p id="p-2848">Pyrobaculum neutrophilum V24Sta</p><p id="p-2849">Pyrobaculum sp 1860</p><p id="p-2850">Pyrococcus abyssi GE5</p><p id="p-2851">Pyrococcus furiosus DSM 3638</p><p id="p-2852">Pyrococcus horikoshii OT3</p><p id="p-2853">Pyrococcus sp NA2</p><p id="p-2854">Pyrococcus yayanosii CH1</p><p id="p-2855">Pyrolobus fumarii 1A</p><p id="p-2856">Rahnella sp Y9602</p><p id="p-2857">Ralstonia eutropha JMP134</p><p id="p-2858">Ralstonia pickettii 12D</p><p id="p-2859">Ralstonia solanacearum GMI1000</p><p id="p-2860">Ramlibacter tataouinensis TTB310</p><p id="p-2861">Renibacterium salmoninarum ATCC 33209</p><p id="p-2862">Rhizobium etli CFN 42</p><p id="p-2863">Rhizobium leguminosarum bv viciae 3841</p><p id="p-2864">Rhodobacter capsulatus SB 1003</p><p id="p-2865">Rhodobacter sphaeroides 241</p><p id="p-2866">Rhodococcus equi 103S</p><p id="p-2867">Rhodococcus erythropolis PR4</p><p id="p-2868">Rhodococcus jostii RHA1</p><p id="p-2869">Rhodococcus opacus B4</p><p id="p-2870">Rhodoferax ferrireducens T118</p><p id="p-2871">Rhodomicrobium vannielii ATCC 17100</p><p id="p-2872">Rhodopirellula baltica SH 1</p><p id="p-2873">Rhodopseudomonas palustris CGA009</p><p id="p-2874">Rhodospirillum centenum SW</p><p id="p-2875">Rhodospirillum rubrum ATCC 11170</p><p id="p-2876">Rhodothermus marinus DSM 4252</p><p id="p-2877">Rickettsia africae ESF 5</p><p id="p-2878">Rickettsia akari str Hartford</p><p id="p-2879">Rickettsia bellii RML369 C</p><p id="p-2880">Rickettsia canadensis str McKiel</p><p id="p-2881">Rickettsia conorii str Malish 7</p><p id="p-2882">Rickettsia felis URRWXCal2</p><p id="p-2883">Rickettsia heilongjiangensis 054</p><p id="p-2884">Rickettsia japonica YH</p><p id="p-2885">Rickettsia massiliae MTU5</p><p id="p-2886">Rickettsia peacockii str Rustic</p><p id="p-2887">Rickettsia prowazekii str Madrid E</p><p id="p-2888">Rickettsia rickettsii str Iowa</p><p id="p-2889">Rickettsia sibirica 246</p><p id="p-2890">Rickettsia slovaca 13 B</p><p id="p-2891">Rickettsia typhi str Wilmington</p><p id="p-2892">Riemerella anatipestifer ATCC 11845 DSM 15868</p><p id="p-2893">Robiginitalea biformata HTCC2501</p><p id="p-2894">Roseburia hominis A2 183</p><p id="p-2895">Roseiflexus castenholzii DSM 13941</p><p id="p-2896">Roseiflexus sp RS 1</p><p id="p-2897">Roseobacter denitrificans OCh 114</p><p id="p-2898">Roseobacter litoralis Och 149</p><p id="p-2899">Rothia dentocariosa ATCC 17931</p><p id="p-2900">Rothia mucilaginosa DY 18</p><p id="p-2901">Rubrobacter xylanophilus DSM 9941</p><p id="p-2902">Ruegeria pomeroyi DSS 3</p><p id="p-2903">Ruegeria sp TM1040</p><p id="p-2904">Ruminococcus albus 7</p><p id="p-2905">Runella slithyformis DSM 19594</p><p id="p-2906">Saccharomonospora viridis DSM 43017</p><p id="p-2907">Saccharophagus degradans 2 40</p><p id="p-2908">Saccharopolyspora erythraea NRRL 2338</p><p id="p-2909">Salinibacter ruber DSM 13855</p><p id="p-2910">Salinispora arenicola CNS 205</p><p id="p-2911">Salinispora tropica CNB 440</p><p id="p-2912">Salmonella bongori NCTC 12419</p><p id="p-2913">Salmonella enterica subsp enterica serovar Enteritidis str P125109</p><p id="p-2914">Sanguibacter keddieii DSM 10542</p><p id="p-2915">Sebaldella termitidis ATCC 33386</p><p id="p-2916">Segniliparus rotundus DSM 44985</p><p id="p-2917">Selenomonas sputigena ATCC 35185</p><p id="p-2918">Serratia plymuthica AS9</p><p id="p-2919">Serratia proteamaculans 568 Serratia sp AS12</p><p id="p-2920">Serratia symbiotica str Cinara cedri</p><p id="p-2921">Shewanella amazonensis SB2B</p><p id="p-2922">Shewanella baltica OS155</p><p id="p-2923">Shewanella denitrificans OS217</p><p id="p-2924">Shewanella frigidimarina NCIMB 400</p><p id="p-2925">Shewanella halifaxensis HAW EB4</p><p id="p-2926">Shewanella loihica PV 4</p><p id="p-2927">Shewanella oneidensis MR 1</p><p id="p-2928">Shewanella pealeana ATCC 700345</p><p id="p-2929">Shewanella piezotolerans WP3</p><p id="p-2930">Shewanella putrefaciens CN 32</p><p id="p-2931">Shewanella sediminis HAW EB3</p><p id="p-2932">Shewanella sp ANA 3</p><p id="p-2933">Shewanella violacea DSS12</p><p id="p-2934">Shewanella woodyi ATCC 51908</p><p id="p-2935">Shigella boydii Sb227</p><p id="p-2936">Shigella dysenteriae Sd197</p><p id="p-2937">Shigella flexneri 2a str 301</p><p id="p-2938">Shigella sonnei Ss046</p><p id="p-2939">Sideroxydans lithotrophicus ES 1</p><p id="p-2940">Simkania negevensis Z</p><p id="p-2941">Sinorhizobium fredii NGR234</p><p id="p-2942">Sinorhizobium medicae WSM419</p><p id="p-2943">Sinorhizobium meliloti 1021</p><p id="p-2944">Slackia heliotrinireducens DSM 20476</p><p id="p-2945">Sodalis glossinidius str morsitans</p><p id="p-2946">Sorangium cellulosum So ce56</p><p id="p-2947">Sphaerobacter thermophilus DSM 20745</p><p id="p-2948">Sphaerochaeta coccoides DSM 17374</p><p id="p-2949">Sphaerochaeta globus str Buddy</p><p id="p-2950">Sphaerochaeta pleomorpha str Grapes</p><p id="p-2951">Sphingobacterium sp 21</p><p id="p-2952">Sphingobium chlorophenolicum L 1</p><p id="p-2953">Sphingobium japonicum UT26S</p><p id="p-2954">Sphingobium sp SYK 6</p><p id="p-2955">Sphingomonas wittichii RW1</p><p id="p-2956">Sphingopyxis alaskensis RB2256</p><p id="p-2957">Spirochaeta caldaria DSM 7334</p><p id="p-2958">Spirochaeta smaragdinae DSM 11293</p><p id="p-2959">Spirochaeta thermophila DSM 6192</p><p id="p-2960">Spirosoma linguale DSM 74</p><p id="p-2961">Stackebrandtia nassauensis DSM 44728</p><p id="p-2962">Staphylococcus aureus subsp aureus JH1</p><p id="p-2963">Staphylococcus carnosus subsp carnosus TM300</p><p id="p-2964">Staphylococcus epidermidis ATCC 12228</p><p id="p-2965">Staphylococcus haemolyticus JCSC1435</p><p id="p-2966">Staphylococcus lugdunensis HKU09 01</p><p id="p-2967">Staphylococcus pseudintermedius HKU10 03</p><p id="p-2968">Staphylothermus hellenicus DSM 12710</p><p id="p-2969">Staphylothermus marinus F1</p><p id="p-2970">Starkeya novella DSM 506</p><p id="p-2971">Stenotrophomonas maltophilia K279a</p><p id="p-2972">Stigmatella aurantiaca DW4 3 1</p><p id="p-2973">Streptobacillus moniliformis DSM 12112</p><p id="p-2974">Streptococcus agalactiae 2603V R</p><p id="p-2975">Streptococcus dysgalactiae subsp equisimilis ATCC 12394</p><p id="p-2976">Streptococcus equi subspecies zooepidemicus</p><p id="p-2977">Streptococcus gallolyticus subsp gallolyticus ATCC BAA 2069</p><p id="p-2978">Streptococcus gordonii str Challis substr CH1</p><p id="p-2979">Streptococcus macedonicus ACA DC 198</p><p id="p-2980">Streptococcus mitis B6</p><p id="p-2981">Streptococcus mutans NN2025</p><p id="p-2982">Streptococcus oralis Uo5</p><p id="p-2983">Streptococcus parasanguinis ATCC 15912</p><p id="p-2984">Streptococcus parauberis KCTC 11537</p><p id="p-2985">Streptococcus pasteurianus ATCC 43144</p><p id="p-2986">Streptococcus pneumoniae ST556</p><p id="p-2987">Streptococcus pseudopneumoniae IS7493</p><p id="p-2988">Streptococcus pyogenes M1 GAS</p><p id="p-2989">Streptococcus salivarius JIM8780</p><p id="p-2990">Streptococcus sanguinis SK36</p><p id="p-2991">Streptococcus suis 05ZYH33</p><p id="p-2992">Streptococcus thermophilus CNRZ1066</p><p id="p-2993">Streptococcus uberis 0140J</p><p id="p-2994">Streptomyces avermitilis MA 4680</p><p id="p-2995">Streptomyces bingchenggensis BCW 1</p><p id="p-2996">Streptomyces cattleya NRRL 8057 DSM 46488</p><p id="p-2997">Streptomyces coelicolor A32</p><p id="p-2998">Streptomyces flavogriseus ATCC 33331</p><p id="p-2999">Streptomyces griseus subsp griseus NBRC 13350</p><p id="p-3000">Streptomyces scabiei 8722</p><p id="p-3001">Streptomyces sp SirexAA E</p><p id="p-3002">Streptomyces violaceusniger Tu 4113</p><p id="p-3003">Streptosporangium roseum DSM 43021</p><p id="p-3004">Sulfobacillus acidophilus DSM 10332</p><p id="p-3005">Sulfolobus acidocaldarius DSM 639</p><p id="p-3006">Sulfolobus islandicus M1425</p><p id="p-3007">Sulfolobus solfataricus P2</p><p id="p-3008">Sulfolobus tokodaii str 7</p><p id="p-3009">Sulfuricurvum kujiense DSM 16994</p><p id="p-3010">Sulfurihydrogenibium azorense Az Fu1</p><p id="p-3011">Sulfurihydrogenibium sp YO3AOP1</p><p id="p-3012">Sulfurimonas autotrophica DSM 16294</p><p id="p-3013">Sulfurimonas denitrificans DSM 1251</p><p id="p-3014">Sulfurospirillum deleyianum DSM 6946</p><p id="p-3015">Sulfurovum sp NBC37 1</p><p id="p-3016">Symbiobacterium thermophilum IAM 14863</p><p id="p-3017">Synechococcus elongatus PCC 6301</p><p id="p-3018">Synechococcus sp CC9605</p><p id="p-3019">Synechocystis sp PCC 6803</p><p id="p-3020">Syntrophobacter fumaroxidans MPOB</p><p id="p-3021">Syntrophobotulus glycolicus DSM 8271</p><p id="p-3022">Syntrophomonas wolfei subsp wolfei str Goettingen</p><p id="p-3023">Syntrophothermus lipocalidus DSM 12680</p><p id="p-3024">Syntrophus aciditrophicus SB</p><p id="p-3025">Tannerella forsythia ATCC 43037</p><p id="p-3026">Taylorella equigenitalis MCE9</p><p id="p-3027">Tepidanaerobacter acetatoxydans Re1</p><p id="p-3028">Teredinibacter turnerae T7901</p><p id="p-3029">Terriglobus saanensis SP1PR4</p><p id="p-3030">Tetragenococcus halophilus</p><p id="p-3031">Thauera sp MZ1T</p><p id="p-3032">Thermaerobacter marianensis DSM 12885</p><p id="p-3033">Thermanaerovibrio acidaminovorans DSM 6589</p><p id="p-3034">Thermincola potens JR</p><p id="p-3035">Thermoanaerobacter brockii subsp finnii Ako 1</p><p id="p-3036">Thermoanaerobacter italicus Ab9</p><p id="p-3037">Thermoanaerobacterium thermosaccharolyticum DSM 571</p><p id="p-3038">Thermoanaerobacterium xylanolyticum LX 11</p><p id="p-3039">Thermoanaerobacter mathranii subsp mathranii str A3</p><p id="p-3040">Thermoanaerobacter pseudethanolicus ATCC 33223</p><p id="p-3041">Thermoanaerobacter sp X513</p><p id="p-3042">Thermoanaerobacter tengcongensis MB4</p><p id="p-3043">Thermoanaerobacter wiegelii Rt8B1</p><p id="p-3044">Thermobaculum terrenum ATCC BAA 798</p><p id="p-3045">Thermobifida fusca YX</p><p id="p-3046">Thermobispora bispora DSM 43833</p><p id="p-3047">Thermococcus barophilus MP</p><p id="p-3048">Thermococcus gammatolerans EJ3</p><p id="p-3049">Thermococcus kodakarensis KOD1</p><p id="p-3050">Thermococcus onnurineus NA1</p><p id="p-3051">Thermococcus sibiricus MM 739</p><p id="p-3052">Thermococcus sp 4557</p><p id="p-3053">Thermocrinis albus DSM 14484</p><p id="p-3054">Thermodesulfatator indicus DSM 15286</p><p id="p-3055">Thermodesulfobacterium sp OPB45</p><p id="p-3056">Thermodesulfobium narugense DSM 14796</p><p id="p-3057">Thermodesulfovibrio yellowstonii DSM 11347</p><p id="p-3058">Thermofilum pendens Hrk 5</p><p id="p-3059">Thermomicrobium roseum DSM 5159</p><p id="p-3060">Thermomonospora curvata DSM 43183</p><p id="p-3061">Thermoplasma acidophilum DSM 1728</p><p id="p-3062">Thermoplasma volcanium GSS1</p><p id="p-3063">Thermoproteus uzoniensis 76820</p><p id="p-3064">Thermosediminibacter oceani DSM 16646</p><p id="p-3065">Thermosipho africanus TCF52B</p><p id="p-3066">Thermosipho melanesiensis BI429</p><p id="p-3067">Thermosphaera aggregans DSM 11486</p><p id="p-3068">Thermosynechococcus elongatus BP 1</p><p id="p-3069">Thermotoga lettingae TMO</p><p id="p-3070">Thermotoga maritima MSB8</p><p id="p-3071">Thermotoga naphthophila RKU 10</p><p id="p-3072">Thermotoga neapolitana DSM 4359</p><p id="p-3073">Thermotoga petrophila RKU 1</p><p id="p-3074">Thermotoga sp RQ2</p><p id="p-3075">Thermotoga thermarum DSM 5069</p><p id="p-3076">Thermovibrio ammonificans HB 1</p><p id="p-3077">Thermovirga lienii DSM 17291</p><p id="p-3078">Thermus scotoductus SA 01</p><p id="p-3079">Thermus sp CCB US3 UF1</p><p id="p-3080">Thermus thermophilus HB27</p><p id="p-3081">Thioalkalimicrobium cyclicum ALM1</p><p id="p-3082">Thioalkalivibrio sp K90mix</p><p id="p-3083">Thioalkalivibrio sulfidophilus HL EbGr7</p><p id="p-3084">Thiobacillus denitrificans ATCC 25259</p><p id="p-3085">Thiomicrospira crunogena XCL 2</p><p id="p-3086">Thiomonas intermedia K12</p><p id="p-3087">Tolumonas auensis DSM 9187</p><p id="p-3088">Treponema azotonutricium ZAS 9</p><p id="p-3089">Treponema brennaborense DSM 12168</p><p id="p-3090">Treponema denticola ATCC 35405</p><p id="p-3091">Treponema pallidum subsp pallidum str Nichols</p><p id="p-3092">Treponema paraluiscuniculi Cuniculi A</p><p id="p-3093">Treponema primitia ZAS 2</p><p id="p-3094">Treponema succinifaciens DSM 2489</p><p id="p-3095">Trichodesmium erythraeum IMS101</p><p id="p-3096">Tropheryma whipplei str Twist</p><p id="p-3097">Truepera radiovictrix DSM 17093</p><p id="p-3098">Tsukamurella paurometabola DSM 20162</p><p id="p-3099">Ureaplasma parvum serovar 3 str ATCC 27815</p><p id="p-3100">Ureaplasma urealyticum serovar 10 str ATCC 33699</p><p id="p-3101">Variovorax paradoxus S110</p><p id="p-3102">Veillonella parvula DSM 2008</p><p id="p-3103">Verminephrobacter eiseniae EF01 2</p><p id="p-3104">Verrucosispora maris AB 18 032</p><p id="p-3105">Vibrio anguillarum 775</p><p id="p-3106">Vibrio cholerae O1 biovar El Tor str N16961</p><p id="p-3107">Vibrio fischeri ES114</p><p id="p-3108">Vibrio furnissii NCTC 11218</p><p id="p-3109">Vibrio harveyi ATCC BAA 1116</p><p id="p-3110">Vibrio parahaemolyticus RIMD 2210633</p><p id="p-3111">Vibrio sp EJY3</p><p id="p-3112">Vibrio splendidus LGP32</p><p id="p-3113">Vibrio vulnificus CMCP6</p><p id="p-3114">Vulcanisaeta distributa DSM 14429</p><p id="p-3115">Vulcanisaeta moutnovskia 768 28</p><p id="p-3116">Waddlia chondrophila WSU 86 1044</p><p id="p-3117">Weeksella virosa DSM 16922</p><p id="p-3118">Weissella koreensis KACC 15510</p><p id="p-3119">Wigglesworthia glossinidia endosymbiont of Glossina brevipalpis</p><p id="p-3120">Wolbachia endosymbiont of Culex quinquefasciatus Pel Wolbachia sp wRi</p><p id="p-3121">Wolinella succinogenes DSM 1740</p><p id="p-3122">Xanthobacter autotrophicus Py2</p><p id="p-3123">Xanthomonas albilineans GPE PC73</p><p id="p-3124">Xanthomonas axonopodis pv citri str 306</p><p id="p-3125">Xanthomonas campestris pv campestris str 8004</p><p id="p-3126">Xanthomonas oryzae pv oryzae KACC 10331</p><p id="p-3127">Xenorhabdus bovienii SS 2004</p><p id="p-3128">Xenorhabdus nematophila ATCC 19061</p><p id="p-3129">Xylanimonas cellulosilytica DSM 15894</p><p id="p-3130">Xylella fastidiosa 9a5c</p><p id="p-3131">Yersinia enterocolitica subsp enterocolitica 8081</p><p id="p-3132">Yersinia pestis CO92</p><p id="p-3133">Yersinia pseudotuberculosis IP 32953</p><p id="p-3134">Zobellia galactanivorans</p><p id="p-3135">Zunongwangia profunda SM A87</p><p id="p-3136">Zymomonas mobilis subsp mobilis ATCC 10988</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-001727v1.1" data-doi="10.1093/bioinformatics/btt389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ames</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hysom</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gokhale</span>  <span class="cit-name-given-names">MB</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">JE</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Scalable metagenomic taxonomy classification using a reference genome database</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">29</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">2253</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btt389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt389%26rft_id%253Dinfo%253Apmid%252F23828782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23828782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-001727v1.2" data-doi="10.1186/1471-2105-13-92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bazinet</span>  <span class="cit-name-given-names">AL</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cummings</span>  <span class="cit-name-given-names">MP</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">A comparative evaluation of sequence classification programs</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. <span class="cit-vol">13</span>: <span class="cit-issue">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2105-13-92</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DBazinet%26rft.auinit1%253DA.%2BL.%26rft.volume%253D13%26rft.spage%253D92%26rft.epage%253D92%26rft.atitle%253DA%2Bcomparative%2Bevaluation%2Bof%2Bsequence%2Bclassification%2Bprograms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-92%26rft_id%253Dinfo%253Apmid%252F22574964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22574964&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-001727v1.3" data-doi="10.1186/1756-0500-5-460"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berendzen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruno</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cohn</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hengartner</span>  <span class="cit-name-given-names">NW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuske</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMahon</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolinsky</span>  <span class="cit-name-given-names">MA</span></span> and <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Rapid phylogenetic and functional classification of short genomic fragments with signature peptides</span>. <abbr class="cit-jnl-abbrev">BMC Research Notes</abbr>, <span class="cit-vol">5</span>:<span class="cit-issue">460</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1756-0500-5-460</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DBerendzen%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.spage%253D460%26rft.epage%253D460%26rft.atitle%253DRapid%2Bphylogenetic%2Band%2Bfunctional%2Bclassification%2Bof%2Bshort%2Bgenomic%2Bfragments%2Bwith%2Bsignature%2Bpeptides.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-5-460%26rft_id%253Dinfo%253Apmid%252F22925230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-5-460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22925230&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-001727v1.4" data-doi="10.1038/nmeth.1358"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">A</span></span> and <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Phymm and PhymmBL: metagenomic phylogenetic classification with interpolated Markov models</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">673</span>–<span class="cit-lpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.1358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBrady%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D673%26rft.epage%253D676%26rft.atitle%253DPhymm%2Band%2BPhymmBL%253A%2Bmetagenomic%2Bphylogenetic%2Bclassification%2Bwith%2Binterpolated%2BMarkov%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1358%26rft_id%253Dinfo%253Apmid%252F19648916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19648916&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-001727v1.5" data-doi="10.1101/gr.120618.111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitra</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruscheweyh</span>  <span class="cit-name-given-names">H</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">SC</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Integrative analysis of environmental sequences using MEGAN4</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr>, <span class="cit-vol">21</span>:<span class="cit-fpage">1552</span>–<span class="cit-lpage">1560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.120618.111%26rft_id%253Dinfo%253Apmid%252F21690186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzkvMTU1MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE0LzAxLzEwLzAwMTcyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-001727v1.6" data-doi="10.1186/gb-2009-10-2-r23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korbel</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abyzov</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">XJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carriero</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cayting</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">MB</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">PEMer: a computational framework with simulation-based error models for inferring genomic structural variants from massive paired-end sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, <span class="cit-vol">10</span>:<span class="cit-issue">R23</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/gb-2009-10-2-r23</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKorbel%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253DR23%26rft.epage%253DR23%26rft.atitle%253DPEMer%253A%2Ba%2Bcomputational%2Bframework%2Bwith%2Bsimulation-based%2Berror%2Bmodels%2Bfor%2Binferring%2Bgenomic%2Bstructural%2Bvariants%2Bfrom%2Bmassive%2Bpaired-end%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-2-r23%26rft_id%253Dinfo%253Apmid%252F19236709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-2-r23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19236709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-001727v1.7" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>. <span class="cit-pub-date">2012</span>, <span class="cit-vol">9</span>:<span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D9%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-001727v1.8" data-doi="10.1186/1471-2164-12-S2-S4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghodsi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Treangen</span>  <span class="cit-name-given-names">T</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Accurate and fast estimation of taxonomic profiles from metagenomic shotgun sequences</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">Suppl 2</span>):<span class="cit-fpage">S4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D12%26rft.spage%253DS4%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-S2-S4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-S2-S4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-001727v1.9" data-doi="10.1371/journal.pone.0003373"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ott</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auch</span>  <span class="cit-name-given-names">AF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">DH</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">MetaSim: a sequencing simulator for genomics and metagenomics</span>.<abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">e3373</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0003373</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRichter%26rft.auinit1%253DD.%2BC.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3373%26rft.epage%253De3373%26rft.atitle%253DMetaSim%253A%2Ba%2Bsequencing%2Bsimulator%2Bfor%2Bgenomics%2Band%2Bmetagenomics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003373%26rft_id%253Dinfo%253Apmid%252F18841204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18841204&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-001727v1.10" data-doi="10.1038/nature11711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schloissnig</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arumugam</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sunagawa</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitreva</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tap</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waller</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mende</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kultima</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kota</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Genomic variation landscape of the human gut microbiome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">493</span>(<span class="cit-issue">7430</span>):<span class="cit-fpage">45</span>–<span class="cit-lpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature11711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D493%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11711%26rft_id%253Dinfo%253Apmid%252F23222524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23222524&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312933800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-001727v1.11" data-doi="10.1038/msb.2013.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Segata</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boernigen</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tickle</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">XC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garrett</span>  <span class="cit-name-given-names">WS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huttenhower</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Computational meta’omics for microbial community studies</span>. <abbr class="cit-jnl-abbrev">Molecular Systems Biology</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/msb.2013.22</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.volume%253D14%26rft.spage%253D666%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2013.22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2013.22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-001727v1.12" data-doi="10.1038/nmeth.2066"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Segata</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waldron</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ballarini</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Narasimhan</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jousson</span>  <span class="cit-name-given-names">O</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Huttenhower</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Metagenomic microbial community profiling using unique clade-specific marker genes</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">811</span>–<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.2066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSegata%26rft.auinit1%253DN.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D811%26rft.epage%253D814%26rft.atitle%253DMetagenomic%2Bmicrobial%2Bcommunity%2Bprofiling%2Busing%2Bunique%2Bclade-specific%2Bmarker%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2066%26rft_id%253Dinfo%253Apmid%252F22688413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22688413&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307015700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-001727v1.13" data-doi="10.1038/nmeth.2693"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sunagawa</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mende</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeller</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Izquierdo-Carrasco</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kultima</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coelho</span>  <span class="cit-name-given-names">LP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arumugam</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tap</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">HB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guarner</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Vos</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doré</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehrlich</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Metagenomic species profiling using universal phylogenetic marker genes</span>. <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">1196</span>–<span class="cit-lpage">1199</span> (2013)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.2693</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D10%26rft.spage%253D1196%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2693%26rft_id%253Dinfo%253Apmid%252F24141494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24141494&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2014%2F01%2F10%2F001727.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327698100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-001727v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">DE</span></span> and <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Ultrafast metagenomic sequence classification using exact alignments</span>. In submission.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-001727v1.15" data-doi="10.1126/science.1208344"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">GD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bittinger</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">YY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keilbaugh</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bewtra</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knights</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilroy</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baldassano</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nessel</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bushman</span>  <span class="cit-name-given-names">FD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">JD</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Linking long-term dietary patterns with gut microbial enterotypes</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">334</span>(<span class="cit-issue">6052</span>):<span class="cit-fpage">105</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1208344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1208344%26rft_id%253Dinfo%253Apmid%252F21885731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzQvNjA1Mi8xMDUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNC8wMS8xMC8wMDE3MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/001727v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F001727v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/001743v1" title="Opposing Microtubule Motors Control Motility, Morphology, and Cargo Segregation During ER-to-Golgi Transport" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/000042v1" title="Routes for breaching and protecting genetic privacy" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 10, 2014.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/001727v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/001727v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/001727v1" class="active" data-icon-position="" data-hide-link-title="0">SIANN: Strain Identification by Alignment to Near Neighbors</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/443" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-PLqVn6e6wb9sVu8belvfiCUdgNc0VSkUpgx_Z4bTlgE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="443" id="node443--2" data-pisa="biorxiv;001727v1" data-pisa-master="biorxiv;001727" data-seqnum="443" data-apath="/biorxiv/early/2014/01/10/001727.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">SIANN: Strain Identification by Alignment to Near Neighbors</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Samuel S.</span> <span class="nlm-surname">Minot</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Stephen D.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Krista L.</span> <span class="nlm-surname">Ternus</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Dana R.</span> <span class="nlm-surname">Kadavy</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">001727; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/001727 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/001727v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;title=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;title=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;text=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;title=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;t=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;title=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;title=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="443" id="node443--3" data-pisa="biorxiv;001727v1" data-pisa-master="biorxiv;001727" data-seqnum="443" data-apath="/biorxiv/early/2014/01/10/001727.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">SIANN: Strain Identification by Alignment to Near Neighbors</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Samuel S.</span> <span class="nlm-surname">Minot</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Stephen D.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Krista L.</span> <span class="nlm-surname">Ternus</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Dana R.</span> <span class="nlm-surname">Kadavy</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">001727; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/001727 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors&amp;rft.volume&amp;rft.issue&amp;rft.date=2014-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Minot%2C+Samuel+S.&amp;rft.au=Turner%2C+Stephen+D.&amp;rft.au=Ternus%2C+Krista+L.&amp;rft.au=Kadavy%2C+Dana+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/443/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/443/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/443/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/443/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/443/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/443/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/443/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/443/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/443/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/443/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/443/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/443/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;count=horizontal&amp;via=&amp;text=SIANN%3A%20Strain%20Identification%20by%20Alignment%20to%20Near%20Neighbors&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/001727v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/001727v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/001727v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
