<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">eLife</span><span class="iso-abbrev" title="iso-abbrev">Elife</span><span class="publisher-id" title="publisher-id">eLife</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">eLife</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2050-084X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">eLife Sciences Publications, Ltd</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6731059">6731059</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31385805">31385805</a></span><span class="publisher-id" title="publisher-id">42496</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.7554/eLife.42496">10.7554/eLife.42496</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Evolutionary Biology</span>: <span class="subject" title="subject">Microbiology and Infectious Disease</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Dengue genetic divergence generates within-serotype antigenic variation, but serotypes dominate evolutionary dynamics</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">Sidney M</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-1933-6033</span><a href="#aff1">1</a><a href="#aff2">2</a><a href="#fund1" /><a href="#con1" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzelnick</span><span class="given-names" tagx="given-names" title="given-names">Leah</span></span><a href="#aff3">3</a><a href="#aff4">4</a><a href="#fund4" /><a href="#fund5" /><a href="#con2" /><a href="#conf1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">Trevor</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-4039-5794</span><span class="email" tagx="email" title="email">tbedford@fredhutch.org</span><a href="#aff1">1</a><a href="#fund2" /><a href="#fund3" /><a href="#con3" /><a href="#conf1" /></span><span class="citation_author_institution" id="aff1">[1], <span class="institution" title="institution">Vaccine and Infectious Disease Division</span><span class="institution" title="institution">Fred Hutchinson Cancer Research Center</span><span class="addr-line" title="addr-line">Seattle</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff2">[2], <span class="institution" title="institution">Molecular and Cell Biology Program</span><span class="institution" title="institution">University of Washington</span><span class="addr-line" title="addr-line">Seattle</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff3">[3], <span class="institution" title="institution">Division of Infectious Diseases and Vaccinology</span><span class="institution" title="institution">University of California, Berkeley</span><span class="addr-line" title="addr-line">Berkeley</span><span class="country" tagx="country" title="country">United States</span></span><span class="citation_author_institution" id="aff4">[4], <span class="institution" title="institution">Department of Biology</span><span class="institution" title="institution">University of Florida</span><span class="addr-line" title="addr-line">Gainesville</span><span class="country" tagx="country" title="country">United States</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">Neil M</span></span><span class="role" tagx="role" title="role">Reviewing Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Imperial College London</span><span class="country" tagx="country" title="country">United Kingdom</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tautz</span><span class="given-names" tagx="given-names" title="given-names">Diethard</span></span><span class="role" tagx="role" title="role">Senior Editor</span><span class="citation_author_institution" id="">[], <span class="institution" title="institution">Max-Planck Institute for Evolutionary Biology</span><span class="country" tagx="country" title="country">Germany</span></span></span></div><span class="pub-date-" title="pub-date-">: <span>2019-8-8</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e42496</span><span class="history" title="history"><span class="received" title="received">received: 2018-10-02</span><span class="accepted" title="accepted">accepted: 2019-8-05</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Bell et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license">[license_ref] http://creativecommons.org/licenses/by/4.0/<span class="license-p" title="license-p">This article is distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</span></span></div><a class="self-uri" href="elife-42496.pdf" title="self-uri">elife-42496.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Dengue virus (DENV) exists as four genetically distinct serotypes, each of which is historically assumed to be antigenically uniform. Recent analyses suggest that antigenic heterogeneity may exist within each serotype, but its source, extent and impact remain unclear. Here, we construct a sequence-based model to directly map antigenic change to underlying genetic divergence. We identify 49 specific substitutions and four colinear substitution clusters that robustly predict dengue antigenic relationships. We report moderate antigenic diversity within each serotype, resulting in genotype-specific patterns of heterotypic cross-neutralization. We also quantify the impact of antigenic variation on real-world DENV population dynamics, and find that serotype-level antigenic fitness is a dominant driver of dengue clade turnover. These results provide a more nuanced understanding of the relationship between dengue genetic and antigenic evolution, and quantify the effect of antigenic fitness on dengue evolutionary dynamics.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
dengue</div> <div class="kwd" title="kwd">
antigenic evolution</div> <div class="kwd" title="kwd">
viral fitness</div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Research organism</div> <div class="kwd" title="kwd">
Virus</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000001</span><span class="institution" title="institution">National Science Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DGE-1256082</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">Sidney M</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000875</span><span class="institution" title="institution">Pew Charitable Trusts</span></div> </div><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">Trevor</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000057</span><span class="institution" title="institution">National Institute of General Medical Sciences</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R35GM119774-01</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">Trevor</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01AI114703-01</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzelnick</span><span class="given-names" tagx="given-names" title="given-names">Leah</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">P01AI106695</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzelnick</span><span class="given-names" tagx="given-names" title="given-names">Leah</span></span></span></div> <div class="funding-statement" title="funding-statement">
The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Author impact statement" value="Each dengue serotype contains moderate antigenic diversity, and population immunity drives clade turnover in a hyperendemic population." /> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Dengue virus (DENV) is a mosquito-borne flavivirus which consists of four genetically distinct clades, canonically thought of as serotypes (DENV1 – DENV4) (<a href="#bib26">Lanciotti et al., 1997</a>). DENV circulates primarily in South America and Southeast Asia, infecting 400 million people annually. Primary DENV infection is more often mild and is thought to generate lifelong homotypic immunity and temporary heterotypic immunity, which typically wanes over 6 months to 2 years (<a href="#bib44">Sabin, 1952</a>; <a href="#bib41">Reich et al., 2013</a>; <a href="#bib22">Katzelnick et al., 2016</a>). Subsequent heterotypic secondary infection induces broad cross-protection, and symptomatic tertiary and quaternary cases are rare (<a href="#bib11">Gibbons et al., 2007</a>; <a href="#bib38">Olkowski et al., 2013</a>). However, a small subset of secondary infections are enhanced by non-neutralizing, cross-reactive antibodies, resulting in severe disease via antibody-dependent enhancement (ADE) (<a href="#bib15">Halstead, 1979</a>; <a href="#bib23">Katzelnick et al., 2017</a>; <a href="#bib47">Sangkawibha et al., 1984</a>; <a href="#bib46">Salje et al., 2018</a>). Approximately 1–3% of cases progress to severe dengue hemorrhagic fever, causing <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>9000 deaths each year (<a href="#bib5">Bhatt et al., 2013</a>; <a href="#bib49">Stanaway et al., 2016</a>) and relative risk of severe dengue from secondary heterotypic infection relative to primary infection is estimated to be <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mo_UNKNOWN">
∼</div></div></span>24 (<a href="#bib33">Mizumoto et al., 2014</a>). Thus, the antigenic relationships between dengue viruses — describing whether the immune response generated after primary infection results in protection or enhancement of secondary infection — are key drivers of DENV case outcomes and epidemic patterns.</p> <p>While each serotype is clearly genetically and antigenically distinct, it is not clear how subserotype clades of DENV interact antigenically. Each DENV serotype consists of broad genetic diversity (<a href="#fig1">Figure 1A</a>), including canonical clades termed ‘genotypes’ (<a href="#bib42">Rico-Hesse, 1990</a>; <a href="#bib52">Twiddy et al., 2003</a>). Specific genotypes have been associated with characteristically mild or severe disease, and heterogeneous neutralization titers suggest that the immune response to some genotypes is more cross-protective than others (<a href="#bib10">Gentry et al., 1982</a>; <a href="#bib43">Russell and Nisalak, 1967</a>). Until recently, it has been assumed that these intraserotype differences are minimally important compared to interserotype differences. However, empirical evidence has demonstrated that these genotype-specific differences can drive case outcomes and epidemic severity (reviewed in <a href="#bib16">Holmes and Twiddy, 2003</a>). For example, analysis of a longitudinal cohort study demonstrated that specific combinations of primary infection serotype and secondary infection genotype can mediate individual case outcomes (<a href="#bib37">OhAinle et al., 2011</a>). On a population scale, the DENV1-immune population of Iquitos, Peru, experienced either entirely asymptomatic or very severe epidemic seasons in response to two different genotypes of DENV2 (<a href="#bib24">Kochel et al., 2002</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.42496.002"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <div class="title" tagx="title" title="title">
Phylogeny of dengue virus sequences and normalized antigenic distances.</div> <p>(<b>A</b>) Maximum likelihood phylogeny of the <i>E</i> (envelope) gene from titered dengue viruses. Notably, each of the four serotypes contains substantial genetic diversity. (<b>B</b>) Pairwise antigenic distances were estimated by <a href="#bib21">Katzelnick et al. (2015)</a> using plaque reduction neutralization titers (PRNT50, see Materials and methods). Aggregated titer values are standardized such that the distance between autologous virus-serum pairs is 0, and each titer unit corresponds to a two-fold change in PRNT50 value. Light gray areas represent missing data. Larger values correspond to greater antigenic distance.</p>   </div> <p>One explanation for these and similar observations is that overlooked intraserotype antigenic variation contributes to these genotype-specific case outcomes and epidemic patterns. Recent efforts to antigenically characterize diverse DENV viruses suggests that each serotype may contain antigenic heterogeneity, but the source and impact of this heterogeneity is not clear (<a href="#bib21">Katzelnick et al., 2015</a>). Here, we characterize the evolutionary basis for observed antigenic heterogeneity among DENV clades. We also quantify the impact of within- and between-serotype antigenic variation on real-world DENV population dynamics.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="dengueneutralizationtiterdata" title="sec"> <div class="title" tagx="title" title="title">
Dengue neutralization titer data</div> <p>Antigenic distance between a pair of viruses <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> is experimentally quantified using neutralization titers, which measure how well serum drawn after infection with virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> is able to neutralize virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> in vitro (<a href="#bib43">Russell and Nisalak, 1967</a>). Throughout the following, we refer to serum raised against virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> as serum <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> for brevity. To measure the pairwise antigenic distances for a panel of diverse DENV viruses (<a href="#fig1">Figure 1</a>), <a href="#bib21">Katzelnick et al. (2015)</a> infected naive non-human primates (NHP) with each virus, drew sera at 3 months post-infection, and then titered this sera against a panel of test viruses. To compare patterns of cross-protection in NHP and humans, they also drew sera from 31 study participants 6 weeks after inoculation with a monovalent component of the NIH dengue vaccine candidate. This sera was also titered against a broad panel of DENV viruses. As originally reported, we find generally consistent patterns of neutralization between the NHP and human sera data; see <a href="#bib21">Katzelnick et al. (2015)</a> for a detailed comparison. In total, our dataset consists of 454 NHP sera titers spanning the breadth of DENV diversity, and 728 human sera titers providing deep coverage of a small subset of viruses.</p> <p>To normalize these measurements, we take the log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span> of each value, such that one antigenic unit corresponds to a two-fold drop in neutralization, and we define antigenic distance between autologous serum-virus pairs (i.e. virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and serum <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>) as zero. Normalized antigenic distance between virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and serum <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> is thus calculated as <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
log</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
log</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div></div></span>, such that a higher value of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div></div></span> indicates that serum <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> is less effective at neutralizing virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>, implying greater antigenic distance between viruses <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>. For brevity, these normalized titer values are hereafter referred to simply as log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span>(titers).</p> <p>The full dataset of standardized titer values is shown in <a href="#fig1">Figure 1B</a>. Here, we see that homotypic virus-serum pairs are more closely related antigenically than heterotypic pairs. However, we also observe large variance around this trend, both within and between serotypes. This suggests that treating each serotype as antigenically uniform potentially overlooks important antigenic heterogeneity across viruses within each serotype.</p> </div> <div class="dengueantigenicevolutioncorrespondstogeneticdivergence" title="sec"> <div class="title" tagx="title" title="title">
Dengue antigenic evolution corresponds to genetic divergence</div> <p>Titer measurements are prone to noise, and there is a limited amount of available titer data. If the antigenic heterogeneity observed in the raw data is truly the result of an underlying evolutionary process, we expect that differences in antigenic phenotype correspond to underlying mutations in surface proteins. Dengue has two surface proteins, prM (membrane) and E (envelope). While previous studies have identified epitopes on both prM and E, it is believed that antibodies involved in ADE primarily target prM, while neutralizing antibodies primarily target E (<a href="#bib7">de Alwis et al., 2014</a>). The assay used to generate this titer dataset captures neutralization, but does not capture the effects of ADE; we thus focus our analysis on the <i>E</i> gene.</p> <p>To fully map the relationship between DENV genetic and antigenic evolution, we adapt a substitution-based model originally developed for influenza (<a href="#bib35">Neher et al., 2016</a>). Conceptually, this model predicts titer values through three steps. First, we align <i>E</i> gene sequences from titered dengue viruses and catalog the amino acid mutations between each serum strain and test virus strain in our dataset. Next, we infer how much antigenic change is attributable to specific mutations by constructing a parsimonious model that links normalized antigenic distances to observed mutations. This assigns each mutation <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
m</div></div></span> an antigenic effect size, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div> <div class="mo_UNKNOWN">
≥</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>; forward and reverse mutations are assigned separate values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span>. With this in hand, we estimate the asymmetrical antigenic distance <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div></div></span> between all pairs of sera and test viruses by summing over <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> for all mutations observed between the serum and the test virus (Materials and methods, <a href="#equ2">Equation 2</a>).</p> <p>To learn these values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span>, we first split our dataset into training (random 90% of measurements) and test data (the remaining 10% of values). We take the training data and fit <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> for each mutation that is observed two or more times, subject to regularization as follows (also detailed in Materials and methods, <a href="#equ3">Equation 3</a>). Parsimoniously, we expect that antigenic change is more likely to be incurred by a few key mutations than by many mutations; correspondingly, our prior expectation of values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> is exponentially distributed such that most values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>. This is directly analogous to lasso regression to identify a few parameters with positive weights and set other parameters to 0 (<a href="#bib51">Tibshirani, 1996</a>). Additionally, some viruses have greater binding avidity, and some sera are more potent than others (<a href="#fig1s1">Figure 1—figure supplement 1</a>); these ‘row’ and ‘column’ effects, respectively, are normally distributed and are taken into account when training the model. The model uses convex optimization to learn the values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> that minimize the sum of squared errors (SSE) between observed and predicted titers in the training data. We thus learn model parameters from the training data, and then use those parameters to predict test data values. We assess model performance by comparing the predicted test titer values to the actual values, aggregated across 100-fold Monte Carlo cross validation.</p> <p>This model formulation is an effective tool for estimating antigenic relationships between viruses based on their genetic sequences. On average across cross-validation replicates, this model yields a root mean squared error (RMSE) of 0.75 when predicting titers relative to their true value (95% CI 0.74–0.77, RMSE), and explains 78% of the observed variation in neutralization titers overall (95% CI 0.77–0.79, Pearson <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
2</div> </div></div></span>). This is comparable to the model error from a cartography-based characterization of the same dataset (RMSE 0.65–0.8 log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span> titer units) (<a href="#bib21">Katzelnick et al., 2015</a>). Prediction error was comparable between human and non-human primate sera, indicating that these genetic determinants of antigenic phenotypes are not host species-specific (<a href="#fig2s2">Figure 2—figure supplement 2</a>).</p> <p>The 48 strains included in the titer dataset (as serum strains, test virus strains, or both) are 25.7% divergent on average (amino acid differences in <i>E</i>). Pairwise comparisons of all serum strains and test viruses yields 1534 unique mutations that are observed at least twice. Our parsimonious model attributes antigenic change to a total of 49 specific mutations and four colinear mutation clusters (each consisting of 2–6 co-occurring mutations) (<a href="#fig2">Figure 2</a>, <a href="#table1">Table 1</a>). Each of these mutations confers 0.01–2.11 (median 0.19) log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span> titer units of antigenic change; 27 mutations or mutation clusters have <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div> <div class="mo_UNKNOWN">
≥</div> <div class="mn_UNKNOWN">
0.2</div> </div></div></span>. These mutations span all domains of <i>E</i>, and most occur both between and within serotypes (<a href="#fig2">Figure 2</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.42496.004"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <div class="title" tagx="title" title="title">
Distribution and effect size of antigenic mutations.</div> <p>Each point represents one antigenically relevant mutation or colinear mutation cluster. Clustered mutations are connected with dashed lines with point size proportionate to cluster size (N = 2–6). The x axis indicates mutations’ position in <i>E</i>, relative to each functional domain as noted in (<b>B</b>). The y axis indicates antigenic effect size.</p>   </div> <div class="table-wrap_UNKNOWN" id="table1" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.42496.007">10.7554/eLife.42496.007</a><span class="label" tagx="label" title="label">Table 1.</span> <div class="title" tagx="title" title="title">
Antigenically relevant mutations.</div> <p>Each entry represents a mutation (or colinear cluster of mutations) inferred by the titer model to have a non-zero antigenic effect size <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> (shown in parentheses).</p>  <table frame="hsides" rules="groups"> <tbody> <tr valign="top"> <td rowspan="1" colspan="1">I6V, S29G, F90Y, T176P, V197I, L475M (0.71)</td> <td rowspan="1" colspan="1">D154E (0.03)</td> <td rowspan="1" colspan="1">T339I (0.65)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">A19T (0.02)</td> <td rowspan="1" colspan="1">K160V (0.03)</td> <td rowspan="1" colspan="1">V347A (0.28)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">N83K (0.34)</td> <td rowspan="1" colspan="1">E161T (0.22)</td> <td rowspan="1" colspan="1">I380V (0.45)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">A88K (0.08)</td> <td rowspan="1" colspan="1">A162I (0.19)</td> <td rowspan="1" colspan="1">V382A (0.04)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">A88Q (1.39)</td> <td rowspan="1" colspan="1">I162A (0.29)</td> <td rowspan="1" colspan="1">V382I (0.37)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">Y90F (0.43)</td> <td rowspan="1" colspan="1">I164V (0.01)</td> <td rowspan="1" colspan="1">N385K, V454I (0.12)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">V91I (0.35)</td> <td rowspan="1" colspan="1">T171S (0.23)</td> <td rowspan="1" colspan="1">D390S (0.12)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">K93R (0.20)</td> <td rowspan="1" colspan="1">V174E (0.46)</td> <td rowspan="1" colspan="1">N390S (0.18)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">V114I (0.90)</td> <td rowspan="1" colspan="1">E180T (0.43)</td> <td rowspan="1" colspan="1">V454T (0.25)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">L122S (0.03)</td> <td rowspan="1" colspan="1">N203E (0.04)</td> <td rowspan="1" colspan="1">V461F (0.01)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">S122L (0.07)</td> <td rowspan="1" colspan="1">N203K (0.08)</td> <td rowspan="1" colspan="1">F461I (0.03)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">N124K (0.10)</td> <td rowspan="1" colspan="1">D203N (0.14)</td> <td rowspan="1" colspan="1">I462V (0.10)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">V129I (0.01)</td> <td rowspan="1" colspan="1">E203N (1.09)</td> <td rowspan="1" colspan="1">L462I (0.16)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">I129V (0.21)</td> <td rowspan="1" colspan="1">E203D (1.34)</td> <td rowspan="1" colspan="1">V462L (2.11)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">I129A (0.23)</td> <td rowspan="1" colspan="1">I308V (0.18)</td> <td rowspan="1" colspan="1">T478S (0.10)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">Y132I (0.12)</td> <td rowspan="1" colspan="1">G330D (0.22)</td> <td rowspan="1" colspan="1">S478M (0.43)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">Y132P, R233Q (0.34)</td> <td rowspan="1" colspan="1">I335V, N355T, P364V(0.18)</td> <td rowspan="1" colspan="1">V484I (0.39)</td> </tr> <tr valign="top"> <td rowspan="1" colspan="1">I139V (0.05)</td> <td rowspan="1" colspan="1">L338E (0.43)</td> <td rowspan="1" colspan="1" /> </tr> </tbody> </table> </div> </div> <div class="eachserotypeofdenguecontainsmoderateantigenicheterogeneity" title="sec"> <div class="title" tagx="title" title="title">
Each serotype of dengue contains moderate antigenic heterogeneity</div> <p>By linking antigenic change to specific mutations, we are able to estimate unmeasured antigenic distances between any pair of viruses in the dataset based on their genetic differences. As an example, we estimated the antigenic distance between serum raised against each monovalent component of the NIH vaccine candidate and all other viruses in the dataset. As shown in <a href="#fig3">Figure 3</a>, vaccine-elicited antibodies result in strong homotypic neutralization, but heterotypic cross-neutralization varies widely between specific strains. This has important ramifications for vaccine design and trial evaluation.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.42496.008"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <div class="title" tagx="title" title="title">
Antigenic distance from NIH vaccine strains.</div> <p>By assigning a discrete increment of antigenic change to each mutation, we can estimate the asymmetrical antigenic distance between any serum strain and test virus strain based on their genetic differences. Here, we show the estimated antigenic distance between serum raised against each monovalent component of the NIH vaccine candidate (indicated as ‘X’) and each test virus in the tree.</p>   </div> <p>We also observe antigenic heterogeneity at the genotype level. On average, heterotypic genotypes are separated by 6.9 antigenic mutations (or colinear mutation clusters) and 2.18 log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span> titers. Homotypic genotypes are separated by a mean of 1.9 antigenic mutations, conferring a total of 0.30 log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span> titers of antigenic distance (<a href="#fig4">Figure 4</a>). Notably, the titer dataset spans the breadth of canonical DENV genotypes, but in most cases lacks the resolution to detect within-genotype antigenic diversity. We thus expect that these results represent a lower bound on the true extent of DENV intraserotype antigenic diversity.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.42496.009"><span class="label" tagx="label" title="label">Figure 4.</span></a></div>   <div class="title" tagx="title" title="title">
Titer distance by genotype.</div> <p>Values represent the mean interpolated antigenic distance between canonical dengue genotypes (in standardized log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span> titer units). Columns represent sera; rows represent test viruses.</p>   </div> <p>In summary, we have identified a small number of antigenically relevant mutations that explain most of the observed antigenic heterogeneity in dengue, as indicated by neutralization titers. These mutations occur both between and within serotypes, suggesting that dengue antigenic evolution is an ongoing, although gradual, process. This results in strain-specific and genotype-level antigenic variation, although the scale of this variation is small compared to serotype-level differences. From this, we conclude that there is antigenic variation within each serotype of DENV, and that this is driven by underlying genetic divergence.</p> </div> <div class="antigenicnoveltypredictsserotypesuccess" title="sec"> <div class="title" tagx="title" title="title">
Antigenic novelty predicts serotype success</div> <p>From the titer model, we find evidence that homotypic genotypes of DENV vary in their ability to escape antibody neutralization. However, antibody neutralization is only one of many factors that shape epidemic patterns. We investigate whether the observed antigenic diversity influences dengue population dynamics in the real world.</p> <p>The size of the viral population (i.e. prevalence, commonly analyzed using SIR models as reviewed in <a href="#bib29">Lourenço et al., 2018</a>) is determined by many complex factors, and reliable values for population prevalence are largely unavailable. Contrastingly, the composition of the viral population (i.e. the relative frequency of each viral clade currently circulating) can be estimated over time by examining historical sequence data (<a href="#bib27">Lee et al., 2018</a>; <a href="#bib35">Neher et al., 2016</a>), and is primarily driven by viral fitness (<a href="#bib3">Bedford et al., 2011</a>).</p> <p>In meaningfully antigenically diverse viral populations, antigenic novelty (relative to standing population immunity) contributes to viral fitness: as a given virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> circulates in a population, the proportion of the population that is susceptible to infection with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>–and other viruses antigenically similar to <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>–decreases over time as more people acquire immunity (<a href="#bib4">Bedford et al., 2012</a>; <a href="#bib31">Luksza and Lässig, 2014</a>). Antigenically novel viruses that are able to escape this population immunity are better able to infect hosts and sustain transmission chains, making them fitter than the previously circulating viruses (<a href="#bib57">Zhang et al., 2005</a>; <a href="#bib4">Bedford et al., 2012</a>; <a href="#bib13">Gupta et al., 1998</a>; <a href="#bib56">Wearing and Rohani, 2006</a>; <a href="#bib30">Lourenço and Recker, 2013</a>). Thus, if antigenic novelty constitutes a fitness advantage for DENV, then we would expect greater antigenic distance from recently circulating viruses to correlate with higher growth rates.</p> <p>To test this hypothesis, we examine the composition of the dengue virus population in Southeast Asia from 1970 to 2015. We estimate the relative population frequency of each DENV serotype at three month intervals, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> (<a href="#fig5">Figure 5A</a>), based on their observed relative abundance in the ‘slice’ of the phylogeny corresponding to each timepoint (N = 8644 viruses; see Materials and methods, <a href="#equ4 equ5">Equations 4-5</a>). While there is insufficient data to directly compare these estimated frequencies to regional case counts, we see good qualitative concordance between frequencies similarly estimated for Thailand and previously reported case counts from Bangkok (<a href="#fig5s1">Figure 5—figure supplement 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.42496.010"><span class="label" tagx="label" title="label">Figure 5.</span></a></div>   <div class="title" tagx="title" title="title">
Antigenic novelty predicts serotype success.</div> <p>(<b>A</b>) The relative frequency of each serotype, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div></div></span>, in Southeast Asia estimated every 3 months based on available sequence data. (<b>B</b>) Total fitness of each serotype. We calculate antigenic fitness for each serotype over time as its frequency-weighted antigenic distance from recently circulating viruses. We then add this to a time-invariant intrinsic fitness value to calculate total fitness. (<b>C</b>) DENV1 frequencies between 1994 and 1996 alongside model projection. At each timepoint <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span>, we blind the model to all empirical data from timepoints later than <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> and predict each serotype’s future trajectory based on its initial frequency, time-invariant intrinsic fitness, and antigenic fitness at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> (Materials and methods, <a href="#equ11">Equation 11</a>). We predict forward in 3-month increments for a total prediction period of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> years. At each increment, we use the current predicted frequency to adjust our estimates of antigenic fitness on a rolling basis (Materials and methods, <a href="#equ16">Equation 15</a>). (<b>D</b>) Predicted growth rates, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
g</div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div></div></span>, compared to empirically observed growth rates, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
g</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div></div></span>. Predicted and empirical growth rate of the example illustrated in (<b>C</b>) is shown in (<b>D</b>) as the blue point. Serotype growth versus decline is accurate (i.e. the predicted and actual growth rates are both <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> or both <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
1</div> </div></div></span>, all points outside the gray area) for 66% of predictions.</p>   </div> <div class="table-wrap_UNKNOWN" id="table2" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.42496.015">10.7554/eLife.42496.015</a><span class="label" tagx="label" title="label">Table 2.</span> <div class="title" tagx="title" title="title">
Optimized fitness model parameters for primary analysis.</div>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">Parameter</th> <th rowspan="1" colspan="1">Value</th> <th rowspan="1" colspan="1">Description</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
β</div></div></span></td> <td rowspan="1" colspan="1">1.02</td> <td rowspan="1" colspan="1">Slope of linear relationship between population immunity and viral fitness</td> </tr> <tr> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div></div></span></td> <td rowspan="1" colspan="1">0.83</td> <td rowspan="1" colspan="1">Proportion of titers waning each year since primary infection</td> </tr> <tr> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div></div></span></td> <td rowspan="1" colspan="1">0.76</td> <td rowspan="1" colspan="1">Slope of linear relationship between titers and probability of protection</td> </tr> <tr> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></td> <td rowspan="1" colspan="1">0.74</td> <td rowspan="1" colspan="1">Relative intrinsic fitness of DENV1</td> </tr> <tr> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></td> <td rowspan="1" colspan="1">0.84</td> <td rowspan="1" colspan="1">Relative intrinsic fitness of DENV2</td> </tr> <tr> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></td> <td rowspan="1" colspan="1">0.50</td> <td rowspan="1" colspan="1">Relative intrinsic fitness of DENV3</td> </tr> <tr> <td rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
4</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></td> <td rowspan="1" colspan="1">0.00</td> <td rowspan="1" colspan="1">Relative intrinsic fitness of DENV4 (fixed)</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="table3" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.42496.014">10.7554/eLife.42496.014</a><span class="label" tagx="label" title="label">Table 3.</span> <div class="title" tagx="title" title="title">
Fitness model performance comparisons.</div> <p>Here we compare the performance of the antigenically-informed fitness models to model performance under two null formulations. In the ‘equal’ null model, all clades are assigned equal fitness (i.e. antigenic and intrinsic fitness are set to 0, Equation 17; Equation 18). In the ‘intrinsic’ null model formulation, only the serotype-specific, time-invariant intrinsic fitness values contribute to clade fitness (i.e. antigenic fitness is set to 0, Equation 19; Equation 20). For both formulations of generalized waning, all other parameters were set to the values reported in <a href="#table2">Table 2</a> (optimized for RMSE). Parameters for heterotypic waning were optimized separately.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">Resolution</th> <th rowspan="1" colspan="1">Fitness model</th> <th rowspan="1" colspan="1">Waning</th> <th rowspan="1" colspan="1">RMSE</th> <th rowspan="1" colspan="1">Pearson <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
2</div> </div></div></span></th> <th rowspan="1" colspan="1">Accuracy</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Serotype</td> <td rowspan="1" colspan="1">Interserotype</td> <td rowspan="1" colspan="1">Generalized</td> <td rowspan="1" colspan="1">0.105</td> <td rowspan="1" colspan="1">0.547</td> <td rowspan="1" colspan="1">0.660</td> </tr> <tr> <td rowspan="1" colspan="1">Serotype</td> <td rowspan="1" colspan="1">Equal fitness null</td> <td rowspan="1" colspan="1">Generalized</td> <td rowspan="1" colspan="1">0.130</td> <td rowspan="1" colspan="1">0.000</td> <td rowspan="1" colspan="1">0.480</td> </tr> <tr> <td rowspan="1" colspan="1">Serotype</td> <td rowspan="1" colspan="1">Intrinsic fitness null</td> <td rowspan="1" colspan="1">Generalized</td> <td rowspan="1" colspan="1">0.140</td> <td rowspan="1" colspan="1">0.042</td> <td rowspan="1" colspan="1">0.510</td> </tr> <tr> <td rowspan="1" colspan="1">Genotype</td> <td rowspan="1" colspan="1">Interserotype</td> <td rowspan="1" colspan="1">Generalized</td> <td rowspan="1" colspan="1">0.062</td> <td rowspan="1" colspan="1">0.286</td> <td rowspan="1" colspan="1">0.666</td> </tr> <tr> <td rowspan="1" colspan="1">Genotype</td> <td rowspan="1" colspan="1">Intergenotype</td> <td rowspan="1" colspan="1">Generalized</td> <td rowspan="1" colspan="1">0.062</td> <td rowspan="1" colspan="1">0.254</td> <td rowspan="1" colspan="1">0.610</td> </tr> <tr> <td rowspan="1" colspan="1">Genotype</td> <td rowspan="1" colspan="1">Equal fitness null</td> <td rowspan="1" colspan="1">Generalized</td> <td rowspan="1" colspan="1">0.070</td> <td rowspan="1" colspan="1">0.000</td> <td rowspan="1" colspan="1">0.440</td> </tr> <tr> <td rowspan="1" colspan="1">Genotype</td> <td rowspan="1" colspan="1">Intrinsic fitness null</td> <td rowspan="1" colspan="1">Generalized</td> <td rowspan="1" colspan="1">0.072</td> <td rowspan="1" colspan="1">0.032</td> <td rowspan="1" colspan="1">0.530</td> </tr> <tr> <td rowspan="1" colspan="1">Serotype</td> <td rowspan="1" colspan="1">Interserotype</td> <td rowspan="1" colspan="1">Heterotypic</td> <td rowspan="1" colspan="1">0.109</td> <td rowspan="1" colspan="1">0.533</td> <td rowspan="1" colspan="1">0.666</td> </tr> <tr> <td rowspan="1" colspan="1">Genotype</td> <td rowspan="1" colspan="1">Interserotype</td> <td rowspan="1" colspan="1">Heterotypic</td> <td rowspan="1" colspan="1">0.063</td> <td rowspan="1" colspan="1">0.291</td> <td rowspan="1" colspan="1">0.661</td> </tr> <tr> <td rowspan="1" colspan="1">Genotype</td> <td rowspan="1" colspan="1">Intergenotype</td> <td rowspan="1" colspan="1">Heterotypic</td> <td rowspan="1" colspan="1">0.063</td> <td rowspan="1" colspan="1">0.203</td> <td rowspan="1" colspan="1">0.599</td> </tr> </tbody> </table> </div> <p>Fitter virus clades increase in frequency over time, such that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></span>. It follows that these clades have a growth rate—defined as the fold-change in frequency over time—greater than one: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
1</div> </div></div></span>. To isolate the extent to which antigenic fitness contributes to clade success and decline, we extend work by <a href="#bib31">Luksza and Lässig (2014)</a> to build a simple model that attempts to predict clade growth rates based on two variables: the antigenic fitness of the clade at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span>, and a time-invariant-free parameter representing the intrinsic fitness of the serotype the clade belongs to. We estimate the antigenic fitness of clade <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> as a function of its antigenic distance from each viral clade <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> that has circulated in the same population over the previous 2 years, weighted by the relative frequency of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> and adjusted for waning population immunity (<a href="#fig5">Figure 5B</a>; Materials and methods, <a href="#equ6 equ7 equ8 equ9 equ10">Equation 6-10</a>). Growth rates are estimated based on a 2-year sliding window (<a href="#fig5">Figure 5C</a>).</p> <p>This simple model explains 54.7% of the observed variation in serotype growth rates, and predicts serotype growth vs. decline correctly for 66.0% of predictions (<a href="#fig5">Figure 5D</a>). This suggests that antigenic fitness is a major driver of serotype population dynamics. This also demonstrates that this model captures key components of dengue population dynamics; examining the formulation of this model in more detail can yield insights into how antigenic relationships influence DENV population composition. The fitness model includes six free parameters that are optimized such that the model most accurately reproduces the observed fluctuations in DENV population composition (minimizing the RMSE of frequency predictions, see Materials and methods). We find that serotype fluctuations are consistent with a model wherein population immunity wanes linearly over time, with the probability of protection dropping by about 63% per year for the first 2 years after primary infection. This model assumes no fundamental difference between homotypic and heterotypic reinfection; rather, homotypic immunity is assumed to wane at the same rate as heterotypic immunity, but starts from a higher baseline of protection based on closer antigenic distances. We also find that these dynamics are best explained by intrinsic fitness that moderately varies by serotype (<a href="#table2">Table 2</a>); we are not aware of any literature that directly addresses this observation via competition experiments. However, intrinsic fitness alone is unable to predict serotype dynamics (<a href="#table3">Table 3</a>) and relative strength of antigenic fitness and intrinsic fitness are approximately matched in determining overall serotype fitness.</p> </div> <div class="antigenicnoveltyalsopartiallypredictsgenotypesuccess" title="sec"> <div class="title" tagx="title" title="title">
Antigenic novelty also partially predicts genotype success</div> <p>To estimate how well antigenic fitness predicts genotype dynamics, we used the same model to predict genotype success and decline. As before, fitness of genotype <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> is based on the intrinsic fitness of the serotype <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> belongs to, and the antigenic distance between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and each other genotype, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>, that has recently circulated (<a href="#equ6 equ7 equ8 equ9 equ10">Equation 6-10</a>). For genotypes, we can calculate antigenic distance between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> at either the serotype level or the genotype level. In the ‘interserotype model’, we treat each serotype as antigenically uniform, and assign the mean serotype-level antigenic distances to all pairs of constituent genotypes. In the ‘intergenotype model’, we incorporate the observed within-serotype heterogeneity, and use the mean genotype-level antigenic distances (as shown in <a href="#fig4">Figure 4</a>). If within-serotype antigenic heterogeneity contributes to genotype fitness, then we would expect estimates of antigenic fitness based on the ‘intergenotype model’ to better predict genotype growth rates.</p> <p>We find that antigenic fitness contributes to genotype turnover, although it explains less of the observed variation than for serotypes. As for serotypes, intrinsic fitness alone was unable to predict genotype turnover (<a href="#table3">Table 3</a>). When antigenic distance is estimated from the ‘interserotype model’, we find that our model of antigenic fitness explains approximately 28.6% of the observed variation in genotype growth rates, and correctly predicts genotype growth vs. decline 66.6% of the time (<a href="#fig6">Figure 6C</a>). Perhaps surprisingly, more precise estimates of antigenic distance between genotypes from the ‘intergenotype model’ does not improve our predictions of genotype success (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.254</div> </div></div></span>, 61.0% accuracy; <a href="#fig6">Figure 6D</a>, <a href="#table3">Table 3</a>). This suggests that although we find strong evidence that genotypes vary in their ability to escape neutralizing antibodies, these differences are subtle enough that they do not impact broad-scale regional dynamics over time.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.7554/eLife.42496.017"><span class="label" tagx="label" title="label">Figure 6.</span></a></div>   <div class="title" tagx="title" title="title">
Antigenic novelty partially predicts genotype success.</div> <p>(<b>A</b>) Relative frequencies of each canonical dengue genotype across Southeast Asia, estimated from available sequence data. (<b>B</b>) Antigenic fitness is calculated for each genotype as its frequency-weighted antigenic distance from recently circulating genotypes. We then add this to a time-invariant, serotype-specific intrinsic fitness value to calculate total fitness (shown here, arbitrary units). We assess antigenic distance at either the ‘intergenotype’ or the ‘interserotype’ resolution. In this panel, we show total fitness over time, incorporating estimates of antigenic fitness derived from the ‘intergenotype’ model. (<b>C, D</b>) Fitness estimates were used to predict clade growth rates over 2 years, compounding immunity every 3 months based on predicted frequency changes (Materials and methods <a href="#equ16">Equation 15</a>). Here, we compare observed vs. predicted growth rates for both formulations of the fitness model (using fitness derived from either ‘interserotype’ or ‘intergenotype’ antigenic distances). Growth versus decline was accurate (predicted and actual growth rates both <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
&amp;gt;</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> or both <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
1</div> </div></div></span>, points outside the gray shaded area) for 67% and 61% of predictions, respectively.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <div class="within-serotypeantigenicheterogeneity" title="sec"> <div class="title" tagx="title" title="title">
Within-serotype antigenic heterogeneity</div> <p>We show that mapping antigenic change to specific mutations and interpolating across the DENV alignment is able to explain a large majority of the observed variation in antigenic phenotypes, as measured by neutralization titers. We identify 49 specific mutations and 4 colinear mutation clusters that contribute to antigenic variation, of which 27 mutations or mutation clusters have an antigenic impact of 0.20 log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span> titers or greater. These mutations span all major domains of <i>E</i>, and occur both within and between serotypes. This demonstrates that DENV antigenic divergence is closely coupled to genetic divergence. We use these mutations to infer unmeasured antigenic relationships between viruses, revealing substantial within-serotype antigenic variation. For comparison, we reconstructed the ancestral sequence of each serotype and constrained the model to only permit antigenic change to be attributed to these serotype-level differences. While this interserotype-only model predicts titers to a reasonable degree, we find that it has higher error (RMSE = 0.86) than the full model which accounts for within-serotype heterogeneity (RMSE = 0.79; <a href="#table4">Table 4</a>). This supports and expands upon previous reports (<a href="#bib21">Katzelnick et al., 2015</a>; <a href="#bib8">Forshey et al., 2016</a>; <a href="#bib54">Waggoner et al., 2016</a>) that the null hypothesis of antigenically uniform serotypes is inconsistent with observed patterns of cross-protection and susceptibility.</p> <div class="table-wrap_UNKNOWN" id="table4" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.42496.018">10.7554/eLife.42496.018</a><span class="label" tagx="label" title="label">Table 4.</span> <div class="title" tagx="title" title="title">
Titer model performance comparisons.</div> <p>We compared performance across several different variations of the titer model. As described in <a href="#bib35">Neher et al. (2016)</a>, incremental antigenic change can be assigned to either amino acid substitutions (‘Substitution’ model) or to branches in the phylogeny (‘Tree’ model). For each of these models, we can constrain the model such that antigenic change is allowed to occur only between serotypes (‘Interserotype’) or between AND within serotypes (‘Full’). For the substitution model, we constrain the interserotype model by reconstructing the amino acid sequence of the most recent common ancestor for each serotype and allowing the model to assign antigenic change only to mutations between these ancestral sequences. For the tree model, we constrain the interserotype model by allowing the model to assign antigenic change only to branches in the phylogeny that lie between serotypes. We also assess the impact of the virus avidity and serum potency terms, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
a</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
b</div> </div></div></span>. For all models and metrics, we report the mean and 95% confidence interval across 100-fold Monte Carlo cross validation with random 90%:10%, training:test splits.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">Model</th> <th rowspan="1" colspan="1">Antigenic resolution</th> <th rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
a</div> </div></div></span>And <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
b</div> </div></div></span></th> <th rowspan="1" colspan="1">RMSE</th> <th rowspan="1" colspan="1">Pearson <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
2</div> </div></div></span></th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1">Substitution</td> <td rowspan="1" colspan="1">Full</td> <td rowspan="1" colspan="1">Yes</td> <td rowspan="1" colspan="1">0.75 (0.74–0.77)</td> <td rowspan="1" colspan="1">0.78 (0.77–0.79)</td> </tr> <tr> <td rowspan="1" colspan="1">Substitution</td> <td rowspan="1" colspan="1">Full</td> <td rowspan="1" colspan="1">No</td> <td rowspan="1" colspan="1">1.13 (1.11–1.16)</td> <td rowspan="1" colspan="1">0.50 (0.48–0.52)</td> </tr> <tr> <td rowspan="1" colspan="1">Substitution</td> <td rowspan="1" colspan="1">Interserotype</td> <td rowspan="1" colspan="1">Yes</td> <td rowspan="1" colspan="1">0.86 (0.85–0.88)</td> <td rowspan="1" colspan="1">0.72 (0.70–0.73)</td> </tr> <tr> <td rowspan="1" colspan="1">Substitution</td> <td rowspan="1" colspan="1">Interserotype</td> <td rowspan="1" colspan="1">No</td> <td rowspan="1" colspan="1">0.86 (0.84–0.87)</td> <td rowspan="1" colspan="1">0.71 (0.70–0.72)</td> </tr> <tr> <td rowspan="1" colspan="1">Tree</td> <td rowspan="1" colspan="1">Full</td> <td rowspan="1" colspan="1">Yes</td> <td rowspan="1" colspan="1">0.84 (0.83–0.86)</td> <td rowspan="1" colspan="1">0.72 (0.71–0.73)</td> </tr> <tr> <td rowspan="1" colspan="1">Tree</td> <td rowspan="1" colspan="1">Full</td> <td rowspan="1" colspan="1">No</td> <td rowspan="1" colspan="1">1.40 (1.38–1.42)</td> <td rowspan="1" colspan="1">0.24 (0.23–0.26)</td> </tr> <tr> <td rowspan="1" colspan="1">Tree</td> <td rowspan="1" colspan="1">Interserotype</td> <td rowspan="1" colspan="1">Yes</td> <td rowspan="1" colspan="1">0.87 (0.85–0.88)</td> <td rowspan="1" colspan="1">0.70 (0.69–0.71)</td> </tr> <tr> <td rowspan="1" colspan="1">Tree</td> <td rowspan="1" colspan="1">Interserotype</td> <td rowspan="1" colspan="1">No</td> <td rowspan="1" colspan="1">0.86 (0.84–0.88)</td> <td rowspan="1" colspan="1">0.72 (0.71–0.73)</td> </tr> </tbody> </table> </div> <p>Consistent with the relatively long timescale of dengue evolution, we observe many sites in the dengue phylogeny to have mutated multiple times. These represent instances of parallelism, reversion and homoplasy. For example, we observe that site 390 is consistently S in DENV1, N in DENV3 and H in DENV4, while DENV2 genotypes show a mixture of D, N and S (<a href="#fig2s1">Figure 2—figure supplement 1</a>). We estimate an antigenic impact of 0.18 log<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN" /> <div class="mn_UNKNOWN">
2</div> </div></div></span> titers of the N390S mutation. Our model predicts that the parallel N390S mutations in DENV1 and DENV2 Cosmopolitan makes these viruses slightly more antigenically similar rather than more antigenically distinct. Along these lines, we compared the ‘substitution’ model to a similar model formulation (termed the ‘tree’ model) which assigns <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> values to individual branches in the phylogeny, rather than to individual mutations, so that each branch with a positive <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> value increases antigenic distance between strains (<a href="#bib35">Neher et al., 2016</a>). As expected from the high degree of homoplasy across the dengue phylogeny, we observe that the ‘substitution’ model outperforms the ‘tree’ model in predicting titers in validation datasets (<a href="#table3">Table 3</a>).</p> <p>To investigate the impact of this observed variation, we examine patterns of neutralization in response to vaccination with each monovalent component of the NIH vaccine candidate (<a href="#fig3">Figure 3</a>). Here, we see that each monovalent component elicits broad homotypic protection, but levels of heterotypic cross protection vary widely between heterotypic genotypes. This is consistent with previous reports of genotype-specific interactions between standing population immunity and subsequent heterotypic epidemics as modulating epidemic severity (<a href="#bib37">OhAinle et al., 2011</a>; <a href="#bib24">Kochel et al., 2002</a>). We hypothesize that this observed within-serotype variation primarily effects heterotypic secondary infection outcomes, rather than modulating homotypic immunity. Although we note that <a href="#bib19">Juraska et al. (2018)</a> demonstrate that vaccine efficacy decreases with increasing amino acid divergence of breakthrough infections from the vaccine insert.</p> <p>Overall, we expect that these antigenic phenotypes represent a lower bound on the extent, magnitude, and nature of antigenic heterogeneity with DENV. Our current titer dataset spans the breadth of DENV diversity, but due to small sample size, it lacks the resolution to detect most sub-genotype antigenic variation. The appearance of the deep antigenic divergence of the four serotypes, and the more recent antigenic divergences within each serotype, suggest that DENV antigenic evolution is likely an ongoing, although gradual, process. We therefore expect that future studies with richer datasets will find additional antigenic variation within each genotype. This dataset also contains many left-censored titer values, where we know two viruses are at least <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
T</div></div></span> titer units apart, but do not know exactly how far apart. If we knew the true value of these censored titers, many of them would indicate larger antigenic distances than the reported values, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
T</div></div></span>, which are used to train the model. Thus, it is likely that our model systematically underestimates the magnitude of titer distances.</p> <p>Finally, antibody neutralization and escape (as measured by PRNT titers) is only one component of the immune response to DENV. Although analysis of a longitudinal cohort study shows that these neutralization titers correlate with protection from severe secondary infection, it is unclear how PRNT titers correspond to antibody-dependent enhancement (<a href="#bib22">Katzelnick et al., 2016</a>). It is also important to note that DENV case outcomes are partially mediated by interactions with innate and T-cell immunity, the effects of which are not captured in neutralization titers (<a href="#bib12">Green et al., 2014</a>). Overall, while richer datasets and the development of more holistic assays will be required in order to fully characterize the extent of DENV antigenic diversity, it is clear that the four-serotype model is insufficient to explain DENV antigenic evolution.</p> </div> <div class="viralcladedynamics" title="sec"> <div class="title" tagx="title" title="title">
Viral clade dynamics</div> <p>We use these inferred antigenic relationships to directly quantify the impact of antigenic fitness on DENV population composition. To do so, we measure serotype frequencies across Southeast Asia over time and construct a model to estimate how they will fluctuate (Materials and methods, <a href="#equ6 equ7 equ8 equ9 equ10 equ11 equ12 equ13 equ15 equ16 equ17">Equation 6-16</a>). This model places a fitness value on each serotype that derives from a constant intrinsic component alongside a time-dependent antigenic component. Antigenic fitness declines with population immunity, which is accumulated via the recent circulation of antigenically similar viruses. Our primary model parameterization assumes that both heterotypic and homotypic immunity wane linearly over time at the same rate, with homotypic immunity starting from a higher baseline of protection based on closer antigenic distances. We compared this to a secondary model parameterization with only heterotypic waning (see Materials and methods), under which we observe similar model performance (<a href="#table3">Table 3</a>).</p> <p>We find that antigenic fitness is able to explain much of the observed variation in serotype growth and decline (<a href="#fig5">Figure 5</a>). Forward simulations under the optimized parameter set display damped oscillations around the serotype-specific ‘set points’ determined by intrinsic fitnesses, but intrinsic fitness alone is unable to explain serotype fluctuations (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
R</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.04</div> </div></div></span>; <a href="#table3">Table 3</a>, <a href="#fig5s2">Figure 5—figure supplement 2</a>). This demonstrates that although intrinsic fitness plays an important role in dictating long-term dynamics, wherein particular serotypes tend to circulate at low frequency (e.g. DENV4) and others at high frequency (e.g. DENV1 and DENV2), antigenic fitness plays out on shorter-term time scales, dictating circulation over several subsequent years.</p> <p>We similarly use this model to quantify the effect of within-serotype antigenic variation on the success and decline of canonical DENV genotypes (<a href="#fig6">Figure 6</a>). As above, genotype antigenic fitness declines with population immunity. Here, we estimate population immunity based on antigenic distance from recently circulating genotypes, using distances that are either genotype-specific or based only on the serotype that each genotype belongs to. We then directly compare how strongly these coarser serotype-level versus specific genotype-level antigenic relationships impact DENV population dynamics. Overall, we find that antigenic fitness explains a moderate portion of the observed variation in genotype growth and decline. Surprisingly, however, we find that incorporating within-serotype antigenic differences does not improve our predictions (<a href="#fig6">Figure 6C–D</a>). We suggest two possible explanations for this observation.</p> <p>First, it may be that although genotypes are antigenically diverse, these differences do not influence large-scale regional dynamics over time. We may then hypothesize that within-serotype antigenic heterogeneity mediates disease severity, but does not influence infection or onward transmission. This hypothesis is consistent with the findings of <a href="#bib34">Nagao and Koelle (2008)</a>, who demonstrated that dengue epidemiological dynamics are compatible with a model wherein immunity confers protection against severe symptoms, but not asymptomatic infection. This is also consistent with <a href="#bib50">Ten Bosch et al. (2018)</a>’s findings that asymptomatic dengue infections contribute to onward transmission.</p> <p>Alternatively, this lack of signal could be methodologically explained if either (A) genotype-level frequency trajectories estimated from public data are overly noisy for this application or (B) our model of antigenic fitness based on PRNT assay data does not match reality, due to either PRNT assay data not well reflecting human immunity or due to our particular model formulation that parameterizes immunity from titer distances (<a href="#equ6 equ7 equ8 equ9 equ10">Equation 6-10</a>). In the present analysis, we are not able to firmly resolve these disparate possibilities.</p> <p>These observations are also subject to caveats imposed by the available data and model assumptions. Our estimates of antigenic fitness are informed by the antigenic distances inferred by the substitution model; thus, as above, we are unable to account for nuanced antigenic differences between sub-genotype clades of DENV due to limited titer data. We estimate DENV population composition over time based on available sequence data, pooled across all of Southeast Asia (Materials and methods, <a href="#equ4">Equation 4</a>). As the vast majority of cases of DENV are asymptomatic, sequenced viruses likely represent a biased sample of more severe cases from urban centers where patients are more likely to seek and access care. We also assume that Southeast Asia represents a closed viral population with homogeneous mixing. However, increasing globalization likely results in some amount of viral importation that is not accounted for in this model (<a href="#bib1">Allicock et al., 2012</a>). Finally, although Southeast Asia experiences hyperendemic DENV circulation, the majority of DENV transmissions are hyper-local (<a href="#bib45">Salje et al., 2017</a>), and viral populations across this broad region may not mix homogeneously each season. Thus, it is possible that these sub-serotype antigenic differences impact finer-scale population dynamics, but we lack the requisite data to examine this hypothesis.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>We find that within-serotype antigenic evolution helps explain observed patterns of cross-neutralization among dengue genotypes. We also find that serotype-level population immunity is a strong determinant of viral clade dynamics across Southeast Asia. As richer datasets become available, future studies that similarly combine viral genomics, functional antigenic characterization, and population modeling have great potential to improve our understanding of how DENV evolves antigenically and moves through populations.</p> </div> <div class="modelsharingandextensions" title="sec"> <div class="title" tagx="title" title="title">
Model sharing and extensions</div> <p>We have provided all source code, configuration files and datasets at https://github.com/blab/dengue-antigenic-dynamics (copy archived at https://github.com/elifesciences-publications/dengue-antigenic-dynamics), and wholeheartedly encourage other groups to adapt and extend this framework for further investigation of DENV antigenic evolution and population dynamics.</p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="data" title="sec"> <div class="title" tagx="title" title="title">
Data</div> <div class="titers" title="sec"> <div class="title" tagx="title" title="title">
Titers</div> <p>Antigenic distance between pairs of viruses <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> is experimentally measured using a neutralization titer, which measures how well serum drawn after infection with virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> is able to neutralize virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> in vitro (<a href="#bib43">Russell and Nisalak, 1967</a>). Briefly, two-fold serial dilutions of serum <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> are incubated with a fixed concentration of virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>. Titers represent the lowest serum concentration able to neutralize 50% of virus, and are reported as the inverse dilution. We used two publicly available plaque reduction neutralization titer (PRNT50) datasets generated by <a href="#bib21">Katzelnick et al. (2015)</a>. The primary dataset was generated by infecting each of 36 non-human primates with a unique strain of DENV. NHP sera was drawn after 12 weeks and titered against the panel of DENV viruses. The secondary dataset was generated by vaccinating 31 human trial participants with a monovalent component of the NIH DENV vaccine. Sera was drawn after 6 weeks and titered against the same panel of DENV viruses. As discussed in Katzelnick et al., these two datasets show similar patterns of antigenic relationships between DENV viruses. In total, our dataset includes 1182 measurements across 48 virus strains: 36 of these were used to generate serum, and 47 were used as test viruses.</p> </div> <div class="sequences" title="sec"> <div class="title" tagx="title" title="title">
Sequences</div> <p>For the titer model analysis, we used the full sequence of <i>E</i> (envelope) from the 48 strains in the titer dataset.</p> <p>For the clade frequencies analysis, we downloaded all dengue genome sequences available from the Los Alamos National Lab Hemorrhagic Fever Virus Database as of March 7, 2018, that contained at least the full coding sequence of <i>E</i> (envelope) (total N = 12,645) (<a href="#bib25">Kuiken et al., 2012</a>). We discarded sequences which were putative recombinants, duplicates, lab strains, or which lacked an annotated sampling location and/or sampling date. We selected all remaining virus strains that were annotated as a Southeast Asian isolate (total N = 8,644).</p> <p>For both datasets, we used the annotated reference dataset from <a href="#bib40">Pyke et al. (2016)</a> to assign sequences to canonical genotypes.</p> </div> </div> <div class="alignmentsandtrees" title="sec"> <div class="title" tagx="title" title="title">
Alignments and trees</div> <p>We used MAFFT v7.305b to align nucleotide <i>E</i> gene sequences for each strain before translating the aligned sequences (no frame-shift indels were present) (<a href="#bib20">Katoh and Standley, 2013</a>). All maximum likelihood phylogenies were constructed with IQ-TREE version 1.6.8 and the GTR + I + G15 nucleotide substitution model (<a href="#bib36">Nguyen et al., 2015</a>).</p> </div> <div class="titermodel" title="sec"> <div class="title" tagx="title" title="title">
Titer model</div> <p>We compute standardized antigenic distance between virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and serum <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> (denoted <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div></div></span>) from measured titers <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div></div></span> relative to autologous titers <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> </div> </div></div></span>, such that </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
log</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
log</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
T</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></div> <p /> <p>We then average normalized titers across individuals. To predict unmeasured titers, we employ the ‘substitution model’ from <a href="#bib35">Neher et al. (2016)</a> and implemented in Nextstrain (<a href="#bib14">Hadfield et al., 2018</a>), which assumes that antigenic evolution is driven by underlying genetic evolution.</p> <p>In the substitution model, observed titer drops are mapped to mutations between each serum and test virus strain after correcting for overall virus avidity, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
i</div> </div></div></span>, and serum potency, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
j</div> </div></div></span> (‘row’ and ‘column’ effects, respectively), so that </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(2)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
≈</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div></div></span> is the titer drop assigned to each mutation, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
m</div></div></span>, between serum <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
m</div></div></span> iterates over mutations. We randomly withhold 10% of titer measurements as a test set. We use the remaining 90% of titer measurements as a training set to learn values for virus avidity, serum potency, and mutation effects. As in <a href="#bib35">Neher et al. (2016)</a>, we formulate this as a convex optimization problem and solve for these parameter values to minimize the cost function <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(3)</span><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
=</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mo_UNKNOWN">
^</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="msup_UNKNOWN"> <div class="mo_UNKNOWN">
)</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
λ</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
κ</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
δ</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div></div></div> <p /> <p>We used <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
λ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
3.0</div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
κ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.6</div> </div></div></span>, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
δ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1.2</div> </div></div></span> to minimize test error. Respectively, these terms represent the squared training error; an L1 regularization term on mutation effects, such that most values of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
m</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></span>; and L2 regularization terms on virus avidities and serum potencies, such that they are normally distributed. These parameter values are then used to predict the antigenic distance between all pairs of viruses, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>. We assess performance by comparing predicted to known titer values in our test data set, and present test error (aggregated from 100-fold Monte Carlo cross-validation) throughout the manuscript.</p> </div> <div class="viralcladedynamics" title="sec"> <div class="title" tagx="title" title="title">
Viral clade dynamics</div> <div class="empiricalcladefrequencies" title="sec"> <div class="title" tagx="title" title="title">
Empirical clade frequencies</div> <p>As discussed in <a href="#bib35">Neher et al. (2016)</a> and <a href="#bib27">Lee et al. (2018)</a>, we estimate empirical clade frequencies from 1970 to 2015 based on observed relative abundances of each clade in the ‘slice’ of the phylogeny corresponding to each quarterly timepoint.</p> <p>Briefly, the frequency trajectory of each clade in the phylogeny is modeled according to a Brownian motion diffusion process discretized to 3-month intervals. Relative to a simple Brownian motion, the expectation includes an ‘inertia’ term that adds velocity to the diffusion and the variance includes a term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
x</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> to scale variance according to frequency following a Wright-Fisher population genetic process. This results in the diffusion process </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(4)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
�?��</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mi_UNKNOWN">
ϵ</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
x</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="unknown" title="mpadded" tag="mpadded"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div></div></div>with ‘volatility’ parameter <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> and inertia parameter <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ϵ</div></div></span>. The term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
x</div> </div></div></span> is the increment in the previous timestep, so that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
x</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div></div></span>. We used <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ϵ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0.7</div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
2.0</div> </div></div></span> to maximize fit to empirical trajectory behavior.<p /> <p>We also include an Bernoulli observation model for clade presence/absence among sampled viruses at timestep <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span>. This observation model follows </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(5)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
�?</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
V</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
�?</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∉</div> <div class="mi_UNKNOWN">
V</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∈</div> <div class="mi_UNKNOWN">
V</div> </div></div></span> represents the set of viruses that belong to the clade and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mo_UNKNOWN">
∉</div> <div class="mi_UNKNOWN">
V</div> </div></div></span> represents the set of viruses that do not belong to the clade. Each frequency trajectory is estimated by simultaneously maximizing the likelihood of the process model and the likelihood of the observation model via adjusting frequency trajectory <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mo_UNKNOWN">
→</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
…</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>.<p /> </div> <div class="populationimmunity" title="sec"> <div class="title" tagx="title" title="title">
Population immunity</div> <p>For antigenically diverse pathogens, antigenic novelty represents a fitness advantage (<a href="#bib28">Lipsitch and O'Hagan, 2007</a>). This means that viruses that are antigenically distinct from previously circulating viruses are able to access more susceptible hosts, allowing the antigenically novel lineage to expand. We adapt a simple deterministic model from <a href="#bib31">Luksza and Lässig (2014)</a> to directly quantify dengue antigenic novelty and its impact on viral fitness. We quantify population immunity to virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, as a function of which clades have recently circulated in the past <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
N</div></div></span> years, and how antigenically similar each of these clades is to virus <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span>, so that </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(6)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="munderover_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
N</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div></div></span> is the antigenic distance between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
i</div></div></span> and each non-overlapping clade <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
n</div></div></span> is the number of years since exposure, and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span> is the relative frequency of <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
j</div></div></span> at year <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
n</div> </div></div></span>. Waning immunity is modeled as a non-negative linear function of time following <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(7)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></div> <p /> <p>The relationship between antigenic distance and the probability of protection, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
C</div></div></span>, is also assumed to be non-negative and linear with slope <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
σ</div> </div></div></span>, such that </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(8)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
max</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></div> <p /> <p>In addition to this primary analysis, we conducted a secondary analysis with a different parameterization of immunity that removes waning of homotypic immunity while allowing waning of heterotypic immunity. In this case, we assume the relationship between antigenic distance and the probability of protection, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
C</div></div></span>, to be 50% at antigenic distance <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
/</div> <div class="mi_UNKNOWN">
σ</div> </div></div></span> and to wane based on years since infection <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
n</div></div></span> modified by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
het</div> </div></div></span> following </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(9)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
exp</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="unknown" title="mpadded" tag="mpadded"> <div class="mi_UNKNOWN">
σ</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="unknown" title="mpadded" tag="mpadded"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mi_UNKNOWN">
het</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mi_UNKNOWN">
n</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
D</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></div> <p /> <p>We model the effects of population immunity, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, on viral antigenic fitness, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span>, as </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(10)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> </div> <div class="mo_UNKNOWN">
-</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div></div></div>where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
β</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> </div></div></span> are fit parameters representing the slope of the linear relationship between immunity and fitness, and the intrinsic relative fitness of each serotype, respectively.<p /> </div> <div class="frequencypredictions" title="sec"> <div class="title" tagx="title" title="title">
Frequency predictions</div> <p>Similar to the model implemented in <a href="#bib31">Luksza and Lässig (2014)</a>, we estimate predicted clade frequencies at time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div></div></span> as </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(11)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> </div> </div> </div> </div></div></div>for short-term predictions (where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
1</div> </div></div></span> year).<p /> <p>We do not attempt to predict future frequencies for clades with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
&amp;lt;</div> <div class="mn_UNKNOWN">
0.05</div> </div></div></span>.</p> <p>For long-term predictions, we must account for immunity accrued at each intermediate timepoint between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div></div></span>. We divide the interval between <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div></div></span> into 3 month timepoints, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
[</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
…</div> <div class="mo_UNKNOWN">
,</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
U</div> </div> <div class="mo_UNKNOWN">
]</div> </div></div></span>, such that <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
U</div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
d</div> <div class="mi_UNKNOWN">
t</div> </div> </div></div></span>. We then compound immunity based on predicted clade frequencies at each intermediate timepoint following </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(12)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(13)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
…</div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(14)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
U</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
…</div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
U</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(15)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
U</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
∑</div> <div class="mi_UNKNOWN">
u</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
u</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> </div> </div> </div> </div> </div></div></div> <p /> <p>We then calculate clade growth rates, defined as the fold-change in relative clade frequency between time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span> and time <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div></div></span></p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(16)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></div> <p /> </div> <div class="nullmodels" title="sec"> <div class="title" tagx="title" title="title">
Null models</div> <p>To quantify the impact of antigenic fitness on DENV clade success, we compare our antigenically-informed model to two null models.</p> <p>Under the ‘equal fitness null’ model, all viruses have equal total fitness (antigenic and intrinsic fitness) at all timepoints </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(17)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
0</div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(18)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
q</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
u</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
l</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mn_UNKNOWN">
0</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></div> <p /> <p>Under the ‘intrinsic fitness null’ model, all viruses have equal antigenic fitness but serotype-specific intrinsic fitness at all timepoints </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(19)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
i</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
c</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> </div> </div></div></div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(20)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
c</div> </div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
�?�</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div></div></div> <p /> </div> <div class="modelperformanceassessmentandparameterfitting" title="sec"> <div class="title" tagx="title" title="title">
Model performance assessment and parameter fitting</div> <p>We assess predictive power as the root mean squared error between predicted and empirical clade frequencies. To assess both the final frequency predictions and the predicted clade trajectories, this RMSE includes error for each clade, for each starting timepoint <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
t</div></div></span>, and for each intermediate predicted timepoint <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
u</div> </div></div></span>.</p> <p>Our frequency prediction model has a total of six free parameters. We jointly fit these parameters to minimize RMSE of serotype frequency predictions via the Nelder-Mead algorithm as implemented in SciPy v.1.0.0 (<a href="#table2">Table 2</a>) (<a href="#bib18">Jones et al., 2001</a>; <a href="#bib9">Gao and Han, 2012</a>). We use <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> years of previous immunity that contribute to antigenic fitness and project <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
�?�</div> <div class="mi_UNKNOWN">
t</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
2</div> </div></div></span> years in the future when predicting clade frequencies.</p> </div> </div> <div class="simulations" title="sec"> <div class="title" tagx="title" title="title">
Simulations</div> <p>To ensure the model machinery functions correctly, we seeded a forward simulation of clade dynamics with 2 years of empirical frequencies and simulated predicted dynamics over the remainder of the time course (<a href="#fig5s3">Figure 5—figure supplement 3</a>). We then fit model parameters as described above, and obtained parameter values that well recover input values (<a href="#table5">Table 5</a>).</p> <div class="table-wrap_UNKNOWN" id="table5" orientation="portrait" position="float"><a href="https://dx.doi.org/10.7554/eLife.42496.016">10.7554/eLife.42496.016</a><span class="label" tagx="label" title="label">Table 5.</span> <div class="title" tagx="title" title="title">
Parameter recovery against simulated data.</div>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="1" colspan="1">Parameter</th> <th rowspan="1" colspan="1">Input value</th> <th rowspan="1" colspan="1">Optimized value</th> </tr> </thead> <tbody> <tr> <th rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
β</div></div></span></th> <td rowspan="1" colspan="1">3.25</td> <td rowspan="1" colspan="1">3.10</td> </tr> <tr> <th rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div></div></span></th> <td rowspan="1" colspan="1">0.55</td> <td rowspan="1" colspan="1">0.56</td> </tr> <tr> <th rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
σ</div></div></span></th> <td rowspan="1" colspan="1">2.35</td> <td rowspan="1" colspan="1">2.57</td> </tr> <tr> <th rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></th> <td rowspan="1" colspan="1">0.70</td> <td rowspan="1" colspan="1">0.72</td> </tr> <tr> <th rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></th> <td rowspan="1" colspan="1">0.85</td> <td rowspan="1" colspan="1">0.78</td> </tr> <tr> <th rowspan="1" colspan="1"><span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mn_UNKNOWN">
0</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
3</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></span></th> <td rowspan="1" colspan="1">0.40</td> <td rowspan="1" colspan="1">0.41</td> </tr> </tbody> </table> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data and software availability</div> <p>Sequence and titer data, as well as all source code used for analyses and figure generation, is publicly available at <a href="https://github.com/blab/dengue-antigenic-dynamics">https://github.com/blab/dengue-antigenic-dynamics (copy archived at https://github.com/elifesciences-publications/dengue-antigenic-dynamics)</a>. Our work relies upon many open source Python packages and software tools, including iPython (<a href="#bib39">Perez and Granger, 2007</a>), Matplotlib (<a href="#bib17">Hunter, 2007</a>), Seaborn (<a href="#bib55">Waskom, 2017</a>), Pandas (<a href="#bib32">McKinney, 2010</a>), CVXOPT (<a href="#bib2">Andersen et al., 2013</a>), NumPy (<a href="#bib53">van der Walt et al., 2011</a>; <a href="#bib9">Gao and Han, 2012</a>), Biopython (<a href="#bib6">Cock et al., 2009</a>), SciPy (<a href="#bib18">Jones et al., 2001</a>), Statsmodels (<a href="#bib48">Seabold and Perktold, 2010</a>), Nextstrain (<a href="#bib14">Hadfield et al., 2018</a>), MAFFT (<a href="#bib20">Katoh and Standley, 2013</a>), and IQ-TREE (<a href="#bib36">Nguyen et al., 2015</a>). Package versions are documented in the GitHub repository.</p> </div> </div> </div> <div class="back" title="back"> <div class="funding-information" title="funding-information"> <div class="title" tagx="title" title="title">
Funding Information</div> <p>This paper was supported by the following grants:</p> <ul> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000001</span><span class="institution" title="institution">National Science Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">DGE-1256082</span> to Sidney M Bell.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000875</span><span class="institution" title="institution">Pew Charitable Trusts</span></div> </div> to Trevor Bedford.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000057</span><span class="institution" title="institution">National Institute of General Medical Sciences</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R35GM119774-01</span> to Trevor Bedford.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">R01AI114703-01</span> to Leah Katzelnick.<p /> </li> <li> <p> </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000060</span><span class="institution" title="institution">National Institute of Allergy and Infectious Diseases</span></div> </div><span class="award-id" tagx="award-id" title="award-id">P01AI106695</span> to Leah Katzelnick.<p /> </li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Richard Neher, John Huddleston, Andrew Rambaut, Molly OhAinle, David Shaw, Paul Edlefsen, Michal Juraska, and all members of the Bedford Lab for useful discussion and advice. SB is a Graduate Research Fellow and is supported by NSF DGE-1256082. TB is a Pew Biomedical Scholar and is supported by NIH R35 GM119774-01. LK is supported by NIH awards R01AI114703-01 and P01AI106695. Our work depends on open data sharing and many open source software tools. We gratefully acknowledge the authors and developers who make our work possible.</p> </div> <div class="additional-information" title="additional-information"> <div class="title" tagx="title" title="title">
Additional information</div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Competing interests</b></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests declared.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="title" tagx="title" title="title"><b>Author contributions</b></div> <div class="fn-type-con" title="con"> <p>Data curation, Software, Formal analysis, Funding acquisition, Validation, Investigation, Visualization, Methodology, Writing—original draft, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Data curation, Methodology, Writing—review and editing.</p> </div> <div class="fn-type-con" title="con"> <p>Conceptualization, Resources, Software, Formal analysis, Supervision, Funding acquisition, Investigation, Methodology, Writing—review and editing.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Additional files</div> <div class="local-data" title="local-data"><a href="https://dx.doi.org/10.7554/eLife.42496.019">10.7554/eLife.42496.019</a><span class="label" tagx="label" title="label">Transparent reporting form</span><div class="media" title="media"><a href="elife-42496-transrepform.pdf">LINK</a></div> </div> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <p>All data, code, model implementations, analyses and figures are available via our online repository at github.com/blab/dengue-antigenic-dynamics (copy archived at<a href="https://github.com/elifesciences-publications/dengue-antigenic-dynamics">https://github.com/elifesciences-publications/dengue-antigenic-dynamics</a>).</p> <p>The following datasets were generated:</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allicock</span><span class="given-names" tagx="given-names" title="given-names">OM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatem</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bennett</span><span class="given-names" tagx="given-names" title="given-names">SN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mueller</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foster</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrington</span><span class="given-names" tagx="given-names" title="given-names">CV</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Phylogeography and population dynamics of dengue viruses in the americas</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">1533</span><span class="lpage" tagx="lpage" title="lpage">1543</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msr320">10.1093/molbev/msr320</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22319149">22319149</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dahl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandenberghe</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2013</span>[data-title] CVXOPT: a Python package for convex optimization<span class="source" tagx="source" title="source"> Abel Ee Ucla Edu/cvxopt</span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cobey</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pascual</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Strength and tempo of selection revealed in viral gene genealogies</span><span class="source" tagx="source" title="source">BMC Evolutionary Biology</span><span class="volume" tagx="volume" title="volume">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">220</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2148-11-220">10.1186/1471-2148-11-220</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21787390">21787390</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pascual</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Canalization of the evolutionary trajectory of the human influenza virus</span><span class="source" tagx="source" title="source">BMC Biology</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">38</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1741-7007-10-38">10.1186/1741-7007-10-38</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22546494">22546494</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhatt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gething</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brady</span><span class="given-names" tagx="given-names" title="given-names">OJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messina</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farlow</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moyes</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drake</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brownstein</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoen</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sankoh</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">George</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaenisch</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wint</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farrar</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hay</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">The global distribution and burden of dengue</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">496</span><span class="fpage" tagx="fpage" title="fpage">504</span><span class="lpage" tagx="lpage" title="lpage">507</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature12060">10.1038/nature12060</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23563266">23563266</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cock</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antao</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chapman</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalke</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedberg</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamelryck</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kauff</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilczynski</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Hoon</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Biopython: freely available Python tools for computational molecular biology and bioinformatics</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1422</span><span class="lpage" tagx="lpage" title="lpage">1423</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp163">10.1093/bioinformatics/btp163</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19304878">19304878</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Alwis</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmid</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crowe</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">WK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Silva</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Dengue viruses are enhanced by distinct populations of serotype cross-reactive antibodies in human immune sera</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1004386</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004386">10.1371/journal.ppat.1004386</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25275316">25275316</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forshey</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reiner</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olkowski</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espinoza</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vilcarromero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casanova</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wearing</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halsey</span><span class="given-names" tagx="given-names" title="given-names">ES</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kochel</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoddard</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Incomplete protection against dengue virus type 2 Re-infection in peru</span><span class="source" tagx="source" title="source">PLOS Neglected Tropical Diseases</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0004398</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0004398">10.1371/journal.pntd.0004398</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26848841">26848841</a></span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Implementing the Nelder-Mead simplex algorithm with adaptive parameters</span><span class="source" tagx="source" title="source">Computational Optimization and Applications</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">259</span><span class="lpage" tagx="lpage" title="lpage">277</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s10589-010-9329-3">10.1007/s10589-010-9329-3</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gentry</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henchal</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCown</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brandt</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalrymple</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">Identification of distinct antigenic determinants on dengue-2 virus using monoclonal antibodies</span><span class="source" tagx="source" title="source">The American Journal of Tropical Medicine and Hygiene</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">548</span><span class="lpage" tagx="lpage" title="lpage">555</span><span class="pub-id"><a href="https://dx.doi.org/10.4269/ajtmh.1982.31.548">10.4269/ajtmh.1982.31.548</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6177259">6177259</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">RV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalanarooj</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarman</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nisalak</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaughn</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endy</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mammen</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Srikiatkhachorn</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Analysis of repeat hospital admissions for dengue to estimate the frequency of third or fourth dengue infections resulting in admissions and dengue hemorrhagic fever, and serotype sequences</span><span class="source" tagx="source" title="source">The American Journal of Tropical Medicine and Hygiene</span><span class="volume" tagx="volume" title="volume">77</span><span class="fpage" tagx="fpage" title="fpage">910</span><span class="lpage" tagx="lpage" title="lpage">913</span><span class="pub-id"><a href="https://dx.doi.org/10.4269/ajtmh.2007.77.910">10.4269/ajtmh.2007.77.910</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17984352">17984352</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beatty</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadjilaou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">Innate immunity to dengue virus infection and subversion of antiviral responses</span><span class="source" tagx="source" title="source">Journal of Molecular Biology</span><span class="volume" tagx="volume" title="volume">426</span><span class="fpage" tagx="fpage" title="fpage">1148</span><span class="lpage" tagx="lpage" title="lpage">1160</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jmb.2013.11.023">10.1016/j.jmb.2013.11.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24316047">24316047</a></span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Chaos, persistence, and evolution of strain structure in antigenically diverse infectious agents</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">280</span><span class="fpage" tagx="fpage" title="fpage">912</span><span class="lpage" tagx="lpage" title="lpage">915</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.280.5365.912">10.1126/science.280.5365.912</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9572737">9572737</a></span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hadfield</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Megill</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huddleston</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Potter</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callender</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sagulenko</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neher</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Nextstrain: real-time tracking of pathogen evolution</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">4121</span><span class="lpage" tagx="lpage" title="lpage">4123</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bty407">10.1093/bioinformatics/bty407</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29790939">29790939</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halstead</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span></span><span class="year" tagx="year" title="year">1979</span><span class="mixed-article-title" title="mixed-article-title">In vivo enhancement of dengue virus infection in rhesus monkeys by passively transferred antibody</span><span class="source" tagx="source" title="source">Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">140</span><span class="fpage" tagx="fpage" title="fpage">527</span><span class="lpage" tagx="lpage" title="lpage">533</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/140.4.527">10.1093/infdis/140.4.527</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/117061">117061</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twiddy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">The origin, emergence and evolutionary genetics of dengue virus</span><span class="source" tagx="source" title="source">Infection, Genetics and Evolution</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">19</span><span class="lpage" tagx="lpage" title="lpage">28</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1567-1348(03)00004-2">10.1016/S1567-1348(03)00004-2</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Matplotlib: a 2D graphics environment</span><span class="source" tagx="source" title="source">Computing in Science &amp;amp; Engineering</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">90</span><span class="lpage" tagx="lpage" title="lpage">95</span><span class="pub-id"><a href="https://dx.doi.org/10.1109/MCSE.2007.55">10.1109/MCSE.2007.55</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliphant</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2001</span>[data-title] SciPy: Open source scientific tools for Python<a href="http://www.scipy.org/">http://www.scipy.org/</a></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Juraska</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magaret</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shao</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carpp</span><span class="given-names" tagx="given-names" title="given-names">LN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fiore-Gartland</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benkeser</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girerd-Chambaz</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langevin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frago</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guy</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duong Thi Hue</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edlefsen</span><span class="given-names" tagx="given-names" title="given-names">PT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">PB</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Viral genetic diversity and protective efficacy of a tetravalent dengue vaccine in two phase 3 trials</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">E8378</span><span class="lpage" tagx="lpage" title="lpage">E8387</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1714250115">10.1073/pnas.1714250115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30127007">30127007</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Standley</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">772</span><span class="lpage" tagx="lpage" title="lpage">780</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzelnick</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fonville</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gromowski</span><span class="given-names" tagx="given-names" title="given-names">GD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bustos Arriaga</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">James</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montoya</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">VanBlargan</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thu</span><span class="given-names" tagx="given-names" title="given-names">HM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pierson</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchy</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aaskov</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muñoz-Jordán</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasilakis</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">RV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesh</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitehead</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">Dengue viruses cluster antigenically but not as discrete serotypes</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">349</span><span class="fpage" tagx="fpage" title="fpage">1338</span><span class="lpage" tagx="lpage" title="lpage">1343</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aac5017">10.1126/science.aac5017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26383952">26383952</a></span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzelnick</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montoya</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gresh</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balmaseda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Neutralizing antibody titers against dengue virus correlate with protection from symptomatic infection in a longitudinal cohort</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">113</span><span class="fpage" tagx="fpage" title="fpage">728</span><span class="lpage" tagx="lpage" title="lpage">733</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1522136113">10.1073/pnas.1522136113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26729879">26729879</a></span></span></li> <li tag="ref"><a name="bib23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzelnick</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gresh</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halloran</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mercado</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gordon</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balmaseda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Antibody-dependent enhancement of severe dengue disease in humans</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">358</span><span class="fpage" tagx="fpage" title="fpage">929</span><span class="lpage" tagx="lpage" title="lpage">932</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aan6836">10.1126/science.aan6836</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29097492">29097492</a></span></span></li> <li tag="ref"><a name="bib24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kochel</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watts</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halstead</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayes</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espinoza</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felices</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caceda</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bautista</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montoya</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Douglas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Effect of dengue-1 antibodies on american dengue-2 viral infection and dengue haemorrhagic fever</span><span class="source" tagx="source" title="source">The Lancet</span><span class="volume" tagx="volume" title="volume">360</span><span class="fpage" tagx="fpage" title="fpage">310</span><span class="lpage" tagx="lpage" title="lpage">312</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(02)09522-3">10.1016/S0140-6736(02)09522-3</a></span></span></li> <li tag="ref"><a name="bib25" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuiken</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thurmond</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dimitrijevic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">The LANL hemorrhagic fever virus database, a new platform for analyzing biothreat viruses</span><span class="source" tagx="source" title="source">Nucleic Acids Research</span><span class="volume" tagx="volume" title="volume">40</span><span class="fpage" tagx="fpage" title="fpage">D587</span><span class="lpage" tagx="lpage" title="lpage">D592</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkr898">10.1093/nar/gkr898</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22064861">22064861</a></span></span></li> <li tag="ref"><a name="bib26" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanciotti</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubler</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trent</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Molecular evolution and phylogeny of dengue-4 viruses</span><span class="source" tagx="source" title="source">Journal of General Virology</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">2279</span><span class="lpage" tagx="lpage" title="lpage">2284</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-78-9-2279">10.1099/0022-1317-78-9-2279</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9292015">9292015</a></span></span></li> <li tag="ref"><a name="bib27" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huddleston</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doud</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hooper</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Deep mutational scanning of hemagglutinin helps predict evolutionary fates of human H3N2 influenza variants</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">115</span><span class="fpage" tagx="fpage" title="fpage">E8276</span><span class="lpage" tagx="lpage" title="lpage">E8285</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1806133115">10.1073/pnas.1806133115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30104379">30104379</a></span></span></li> <li tag="ref"><a name="bib28" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipsitch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Hagan</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Patterns of antigenic diversity and the mechanisms that maintain them</span><span class="source" tagx="source" title="source">Journal of the Royal Society Interface</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">787</span><span class="lpage" tagx="lpage" title="lpage">802</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2007.0229">10.1098/rsif.2007.0229</a></span></span></li> <li tag="ref"><a name="bib29" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lourenço</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tennant</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Recker</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Challenges in dengue research: a computational perspective</span><span class="source" tagx="source" title="source">Evolutionary Applications</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">516</span><span class="lpage" tagx="lpage" title="lpage">533</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/eva.12554">10.1111/eva.12554</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29636803">29636803</a></span></span></li> <li tag="ref"><a name="bib30" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lourenço</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Recker</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Natural, persistent oscillations in a spatial multi-strain disease system with application to dengue</span><span class="source" tagx="source" title="source">PLOS Computational Biology</span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1003308</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1003308">10.1371/journal.pcbi.1003308</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24204241">24204241</a></span></span></li> <li tag="ref"><a name="bib31" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luksza</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lässig</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">A predictive fitness model for influenza</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">507</span><span class="fpage" tagx="fpage" title="fpage">57</span><span class="lpage" tagx="lpage" title="lpage">61</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature13087">10.1038/nature13087</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24572367">24572367</a></span></span></li> <li tag="ref"><a name="bib32" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKinney</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Data structures for statistical computing in Python</span><span class="conf-name" tagx="conf-name" title="conf-name">Proceedings of the 9th Python in Science Conference</span><span class="fpage" tagx="fpage" title="fpage">51</span><span class="lpage" tagx="lpage" title="lpage">56</span></span></li> <li tag="ref"><a name="bib33" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mizumoto</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ejima</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamamoto</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishiura</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="year" tagx="year" title="year">2014</span><span class="mixed-article-title" title="mixed-article-title">On the risk of severe dengue during secondary infection: a systematic review coupled with mathematical modeling</span><span class="source" tagx="source" title="source">Journal of Vector Borne Diseases</span><span class="volume" tagx="volume" title="volume">51</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">153</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25253207">25253207</a></span></span></li> <li tag="ref"><a name="bib34" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koelle</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Decreases in dengue transmission may act to increase the incidence of dengue hemorrhagic fever</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">2238</span><span class="lpage" tagx="lpage" title="lpage">2243</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0709029105">10.1073/pnas.0709029105</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18250338">18250338</a></span></span></li> <li tag="ref"><a name="bib35" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neher</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daniels</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shraiman</span><span class="given-names" tagx="given-names" title="given-names">BI</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Prediction, dynamics, and visualization of antigenic phenotypes of seasonal influenza viruses</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">113</span><span class="fpage" tagx="fpage" title="fpage">E1701</span><span class="lpage" tagx="lpage" title="lpage">E1709</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1525578113">10.1073/pnas.1525578113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26951657">26951657</a></span></span></li> <li tag="ref"><a name="bib36" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">LT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Haeseler</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minh</span><span class="given-names" tagx="given-names" title="given-names">BQ</span></span></span><span class="year" tagx="year" title="year">2015</span><span class="mixed-article-title" title="mixed-article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">268</span><span class="lpage" tagx="lpage" title="lpage">274</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msu300">10.1093/molbev/msu300</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25371430">25371430</a></span></span></li> <li tag="ref"><a name="bib37" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">OhAinle</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balmaseda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macalalad</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tellez</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zody</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saborío</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nuñez</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lennon</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Birren</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gordon</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henn</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">Dynamics of dengue disease severity determined by the interplay between viral genetics and serotype-specific immunity</span><span class="source" tagx="source" title="source">Science Translational Medicine</span><span class="volume" tagx="volume" title="volume">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">114ra128</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/scitranslmed.3003084">10.1126/scitranslmed.3003084</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22190239">22190239</a></span></span></li> <li tag="ref"><a name="bib38" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olkowski</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forshey</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rocha</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vilcarromero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halsey</span><span class="given-names" tagx="given-names" title="given-names">ES</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kochel</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoddard</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Reduced risk of disease during postsecondary dengue virus infections</span><span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">208</span><span class="fpage" tagx="fpage" title="fpage">1026</span><span class="lpage" tagx="lpage" title="lpage">1033</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jit273">10.1093/infdis/jit273</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23776195">23776195</a></span></span></li> <li tag="ref"><a name="bib39" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perez</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Granger</span><span class="given-names" tagx="given-names" title="given-names">BE</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">IPython: a system for interactive scientific computing</span><span class="source" tagx="source" title="source">Computing in Science &amp;amp; Engineering</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">21</span><span class="lpage" tagx="lpage" title="lpage">29</span><span class="pub-id"><a href="https://dx.doi.org/10.1109/MCSE.2007.53">10.1109/MCSE.2007.53</a></span></span></li> <li tag="ref"><a name="bib40" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyke</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall-Mendelin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cameron</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hewitson</span><span class="given-names" tagx="given-names" title="given-names">GR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pukallus</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warrilow</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Hurk</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Highly divergent dengue virus type 1 genotype sets a new distance record</span><span class="source" tagx="source" title="source">Scientific Reports</span><span class="volume" tagx="volume" title="volume">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">22356</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/srep22356">10.1038/srep22356</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26924208">26924208</a></span></span></li> <li tag="ref"><a name="bib41" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reich</span><span class="given-names" tagx="given-names" title="given-names">NG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shrestha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohani</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lessler</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalayanarooj</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">I-K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">RV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">DAT</span></span></span><span class="year" tagx="year" title="year">2013</span><span class="mixed-article-title" title="mixed-article-title">Interactions between serotypes of dengue highlight epidemiological impact of cross-immunity</span><span class="source" tagx="source" title="source">Journal of the Royal Society Interface</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">20130414</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2013.0414">10.1098/rsif.2013.0414</a></span></span></li> <li tag="ref"><a name="bib42" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rico-Hesse</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1990</span><span class="mixed-article-title" title="mixed-article-title">Molecular evolution and distribution of dengue viruses type 1 and 2 in nature</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">174</span><span class="fpage" tagx="fpage" title="fpage">479</span><span class="lpage" tagx="lpage" title="lpage">493</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0042-6822(90)90102-W">10.1016/0042-6822(90)90102-W</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2129562">2129562</a></span></span></li> <li tag="ref"><a name="bib43" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nisalak</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1967</span><span class="mixed-article-title" title="mixed-article-title">Dengue virus identification by the plaque reduction neutralization test</span><span class="source" tagx="source" title="source">Journal of Immunology</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">291</span><span class="lpage" tagx="lpage" title="lpage">296</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4961907">4961907</a></span></span></li> <li tag="ref"><a name="bib44" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabin</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span></span><span class="year" tagx="year" title="year">1952</span><span class="mixed-article-title" title="mixed-article-title">Research on dengue during world war II</span><span class="source" tagx="source" title="source">The American Journal of Tropical Medicine and Hygiene</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">30</span><span class="lpage" tagx="lpage" title="lpage">50</span><span class="pub-id"><a href="https://dx.doi.org/10.4269/ajtmh.1952.1.30">10.4269/ajtmh.1952.1.30</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14903434">14903434</a></span></span></li> <li tag="ref"><a name="bib45" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salje</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lessler</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maljkovic Berry</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Melendrez</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endy</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalayanarooj</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">A-Nuegoonpipat</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chanama</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sangkijporn</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klungthong</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thaisomboonsuk</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nisalak</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">RV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iamsirithaworn</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macareo</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">IK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sangarsang</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarman</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Dengue diversity across spatial and temporal scales: local structure and the effect of host population size</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">355</span><span class="fpage" tagx="fpage" title="fpage">1302</span><span class="lpage" tagx="lpage" title="lpage">1306</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aaj9384">10.1126/science.aaj9384</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28336667">28336667</a></span></span></li> <li tag="ref"><a name="bib46" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salje</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cummings</span><span class="given-names" tagx="given-names" title="given-names">DAT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez-Barraquer</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzelnick</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lessler</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klungthong</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thaisomboonsuk</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nisalak</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weg</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellison</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macareo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yoon</span><span class="given-names" tagx="given-names" title="given-names">IK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jarman</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rothman</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endy</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cauchemez</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Reconstruction of antibody dynamics and infection histories to evaluate dengue risk</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">557</span><span class="fpage" tagx="fpage" title="fpage">719</span><span class="lpage" tagx="lpage" title="lpage">723</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-018-0157-4">10.1038/s41586-018-0157-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29795354">29795354</a></span></span></li> <li tag="ref"><a name="bib47" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sangkawibha</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rojanasuphot</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahandrik</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viriyapongse</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jatanasen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salitul</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phanthumachinda</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halstead</span><span class="given-names" tagx="given-names" title="given-names">SB</span></span></span><span class="year" tagx="year" title="year">1984</span><span class="mixed-article-title" title="mixed-article-title">Risk factors in dengue shock syndrome: a prospective epidemiologic study in Rayong, Thailand. I. the 1980 outbreak</span><span class="source" tagx="source" title="source">American Journal of Epidemiology</span><span class="volume" tagx="volume" title="volume">120</span><span class="fpage" tagx="fpage" title="fpage">653</span><span class="lpage" tagx="lpage" title="lpage">669</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/oxfordjournals.aje.a113932">10.1093/oxfordjournals.aje.a113932</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6496446">6496446</a></span></span></li> <li tag="ref"><a name="bib48" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seabold</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perktold</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Statsmodels: econometric and statistical modeling with Python</span><span class="conf-name" tagx="conf-name" title="conf-name">9th Python in Science Conference</span></span></li> <li tag="ref"><a name="bib49" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanaway</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shepard</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Undurraga</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halasa</span><span class="given-names" tagx="given-names" title="given-names">YA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffeng</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brady</span><span class="given-names" tagx="given-names" title="given-names">OJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hay</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedi</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bensenor</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castañeda-Orjuela</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chuang</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibney</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Memish</span><span class="given-names" tagx="given-names" title="given-names">ZA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rafay</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ukwaja</span><span class="given-names" tagx="given-names" title="given-names">KN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yonemoto</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">CJL</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">The global burden of dengue: an analysis from the global burden of disease study 2013</span><span class="source" tagx="source" title="source">The Lancet Infectious Diseases</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">712</span><span class="lpage" tagx="lpage" title="lpage">723</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(16)00026-8">10.1016/S1473-3099(16)00026-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26874619">26874619</a></span></span></li> <li tag="ref"><a name="bib50" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ten Bosch</span><span class="given-names" tagx="given-names" title="given-names">QA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clapham</span><span class="given-names" tagx="given-names" title="given-names">HE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambrechts</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duong</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchy</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Althouse</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lloyd</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waller</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitron</span><span class="given-names" tagx="given-names" title="given-names">U</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vazquez-Prokopec</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">TW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perkins</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Contributions from the silent majority dominate dengue virus transmission</span><span class="source" tagx="source" title="source">PLOS Pathogens</span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1006965</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006965">10.1371/journal.ppat.1006965</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29723307">29723307</a></span></span></li> <li tag="ref"><a name="bib51" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tibshirani</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="mixed-article-title" title="mixed-article-title">Regression shrinkage and selection via the lasso</span><span class="source" tagx="source" title="source">Journal of the Royal Statistical Society: Series B (Methodological)</span><span class="volume" tagx="volume" title="volume">58</span><span class="fpage" tagx="fpage" title="fpage">267</span><span class="lpage" tagx="lpage" title="lpage">288</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.2517-6161.1996.tb02080.x">10.1111/j.2517-6161.1996.tb02080.x</a></span></span></li> <li tag="ref"><a name="bib52" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twiddy</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Inferring the rate and time-scale of dengue virus evolution</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">122</span><span class="lpage" tagx="lpage" title="lpage">129</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msg010">10.1093/molbev/msg010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519914">12519914</a></span></span></li> <li tag="ref"><a name="bib53" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Walt</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colbert</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Varoquaux</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">2011</span><span class="mixed-article-title" title="mixed-article-title">The NumPy array: a structure for efficient numerical computation</span><span class="source" tagx="source" title="source">Computing in Science &amp;amp; Engineering</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">22</span><span class="lpage" tagx="lpage" title="lpage">30</span><span class="pub-id"><a href="https://dx.doi.org/10.1109/MCSE.2011.37">10.1109/MCSE.2011.37</a></span></span></li> <li tag="ref"><a name="bib54" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waggoner</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balmaseda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gresh</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sahoo</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montoya</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abeynayake</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuan</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinsky</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2016</span><span class="mixed-article-title" title="mixed-article-title">Homotypic dengue virus reinfections in nicaraguan children</span><span class="source" tagx="source" title="source">Journal of Infectious Diseases</span><span class="volume" tagx="volume" title="volume">214</span><span class="fpage" tagx="fpage" title="fpage">986</span><span class="lpage" tagx="lpage" title="lpage">993</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiw099">10.1093/infdis/jiw099</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26984144">26984144</a></span></span></li> <li tag="ref"><a name="bib55" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waskom</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="source" tagx="source" title="source">Seaborn Visualization Library</span><a href="http://seaborn.pydata.org/">http://seaborn.pydata.org/</a></span></li> <li tag="ref"><a name="bib56" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wearing</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohani</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Ecological and immunological determinants of dengue epidemics</span><span class="source" tagx="source" title="source">PNAS</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">11802</span><span class="lpage" tagx="lpage" title="lpage">11807</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0602960103">10.1073/pnas.0602960103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16868086">16868086</a></span></span></li> <li tag="ref"><a name="bib57" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mammen</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chinnawirotpisan</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klungthong</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodpradit</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monkongdee</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nimmannitya</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalayanarooj</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Clade replacements in dengue virus serotypes 1 and 3 are associated with changing serotype prevalence</span><span class="source" tagx="source" title="source">Journal of Virology</span><span class="volume" tagx="volume" title="volume">79</span><span class="fpage" tagx="fpage" title="fpage">15123</span><span class="lpage" tagx="lpage" title="lpage">15130</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.79.24.15123-15130.2005">10.1128/JVI.79.24.15123-15130.2005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16306584">16306584</a></span></span></li> </ul> </div> </div>  </body></html>