<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/18/2020.03.16.994152.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.16.994152v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Characterization of the SARS-CoV-2 Spike in an Early Prefusion Conformation"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.16.994152"/>
  <meta name="DC.Date" content="2020-03-18"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Pandemic coronavirus disease 2019 (COVID-19) is caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), for which there are no efficacious vaccines or therapeutics that are urgently needed. We expressed three versions of spike (S) proteins—receptor binding domain (RBD), S1 subunit and S ectodomain—in insect cells. RBD appears monomer in solutions, whereas S1 and S associate into homotrimer with substantial glycosylation. The three proteins confer excellent antigenicity with six convalescent COVID-19 patient sera. Cryo-electron microscopy (cryo-EM) analyses indicate that the SARS-CoV-2 S trimer dominate in a unique conformation distinguished from the classic prefusion conformation of coronaviruses by the upper S1 region at lower position ~15 Å proximal to viral membrane. Such conformation is proposed as an early prefusion state for the SARS-CoV-2 spike that may broaden the knowledge of coronavirus and facilitate vaccine development."/>
  <meta name="DC.Contributor" content="Tingting Li"/>
  <meta name="DC.Contributor" content="Qingbing Zheng"/>
  <meta name="DC.Contributor" content="Hai Yu"/>
  <meta name="DC.Contributor" content="Dinghui Wu"/>
  <meta name="DC.Contributor" content="Wenhui Xue"/>
  <meta name="DC.Contributor" content="Yuyun Zhang"/>
  <meta name="DC.Contributor" content="Xiaofen Huang"/>
  <meta name="DC.Contributor" content="Lizhi Zhou"/>
  <meta name="DC.Contributor" content="Zhigang Zhang"/>
  <meta name="DC.Contributor" content="Zhenghui Zha"/>
  <meta name="DC.Contributor" content="Tingting Chen"/>
  <meta name="DC.Contributor" content="Zhiping Wang"/>
  <meta name="DC.Contributor" content="Jie Chen"/>
  <meta name="DC.Contributor" content="Hui Sun"/>
  <meta name="DC.Contributor" content="Tingting Deng"/>
  <meta name="DC.Contributor" content="Yingbin Wang"/>
  <meta name="DC.Contributor" content="Yixin Chen"/>
  <meta name="DC.Contributor" content="Qinjian Zhao"/>
  <meta name="DC.Contributor" content="Jun Zhang"/>
  <meta name="DC.Contributor" content="Ying Gu"/>
  <meta name="DC.Contributor" content="Shaowei Li"/>
  <meta name="DC.Contributor" content="Ningshao Xia"/>
  <meta name="article:published_time" content="2020-03-18"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Characterization of the SARS-CoV-2 Spike in an Early Prefusion Conformation"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Pandemic coronavirus disease 2019 (COVID-19) is caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), for which there are no efficacious vaccines or therapeutics that are urgently needed. We expressed three versions of spike (S) proteins—receptor binding domain (RBD), S1 subunit and S ectodomain—in insect cells. RBD appears monomer in solutions, whereas S1 and S associate into homotrimer with substantial glycosylation. The three proteins confer excellent antigenicity with six convalescent COVID-19 patient sera. Cryo-electron microscopy (cryo-EM) analyses indicate that the SARS-CoV-2 S trimer dominate in a unique conformation distinguished from the classic prefusion conformation of coronaviruses by the upper S1 region at lower position ~15 Å proximal to viral membrane. Such conformation is proposed as an early prefusion state for the SARS-CoV-2 spike that may broaden the knowledge of coronavirus and facilitate vaccine development.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.16.994152v2"/>
  <meta name="citation_id" content="2020.03.16.994152v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.16.994152v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.16.994152v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.16.994152v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.16.994152"/>
  <meta name="citation_num_pages" content="31"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.16.994152"/>
  <meta name="citation_author" content="Tingting Li"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Qingbing Zheng"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Hai Yu"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Dinghui Wu"/>
  <meta name="citation_author_institution" content="Department of Pulmonary Medicine, The First Affiliated Hospital of Xiamen University"/>
  <meta name="citation_author" content="Wenhui Xue"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Yuyun Zhang"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Xiaofen Huang"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Lizhi Zhou"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Zhigang Zhang"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Zhenghui Zha"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Tingting Chen"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Zhiping Wang"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Jie Chen"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Hui Sun"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Tingting Deng"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Yingbin Wang"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Yixin Chen"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Qinjian Zhao"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Jun Zhang"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author" content="Ying Gu"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author_email" content="guying@xmu.edu.cn"/>
  <meta name="citation_author_email" content="shaowei@xmu.edu.cn"/>
  <meta name="citation_author_email" content="nsxia@xmu.edu.cn"/>
  <meta name="citation_author" content="Shaowei Li"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author_email" content="guying@xmu.edu.cn"/>
  <meta name="citation_author_email" content="shaowei@xmu.edu.cn"/>
  <meta name="citation_author_email" content="nsxia@xmu.edu.cn"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3374-1038"/>
  <meta name="citation_author" content="Ningshao Xia"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Molecular Vaccinology and Molecular Diagnostics, School of Life Sciences, School of Public Health, Xiamen University"/>
  <meta name="citation_author_institution" content="National Institute of Diagnostics and Vaccine Development in Infectious Disease, Xiamen University"/>
  <meta name="citation_author_email" content="guying@xmu.edu.cn"/>
  <meta name="citation_author_email" content="shaowei@xmu.edu.cn"/>
  <meta name="citation_author_email" content="nsxia@xmu.edu.cn"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0179-5266"/>
  <meta name="citation_reference" content="Chan JF, Kok KH, Zhu Z et al. Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan. Emerg Microbes Infect 2020; 9:221–236."/>
  <meta name="citation_reference" content="Zhu N, Zhang D, Wang W et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 2020; 382:727–733."/>
  <meta name="citation_reference" content="Wang D, Hu B, Hu C et al. Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China. JAMA 2020."/>
  <meta name="citation_reference" content="Zhou P, Yang XL, Wang XG et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 2020; 579:270–273."/>
  <meta name="citation_reference" content="Wan Y, Shang J, Graham R, Baric RS, Li F. Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol 2020."/>
  <meta name="citation_reference" content="Millet JK, Whittaker GR. Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis. Virus Research; 202:120–134."/>
  <meta name="citation_reference" content="Wrapp D, Wang N, Corbett KS et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (New York, NY) 2020."/>
  <meta name="citation_reference" content="Walls AC, Xiong X, Park Y-JJ et al. Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion. Cell 2019; 176:1026."/>
  <meta name="citation_reference" content="Yuan Y, Cao D, Zhang Y et al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nature communications 2017; 8:15092."/>
  <meta name="citation_reference" content="Walls AC, Park Y-JJ, Tortorici MA, Wall A, McGuire AT, Veesler D. Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell 2020."/>
  <meta name="citation_reference" content="Gui M, Song W, Zhou H et al. Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell research 2017; 27:119–129."/>
  <meta name="citation_reference" content="Lu S. Timely development of vaccines against SARS-CoV-2. Emerg Microbes Infect 2020; 9:542–544."/>
  <meta name="citation_reference" content="Sui J, Hwang WC, Perez S et al. Structural and functional bases for broad-spectrum neutralization of avian and human influenza A viruses. Nature structural &amp;; molecular biology 2009; 16:265–273."/>
  <meta name="citation_reference" content="Zhang X, Wei M, Pan H et al. Robust manufacturing and comprehensive characterization of recombinant hepatitis E virus-like particles in Hecolin(®). Vaccine 2014; 32:4039–4050."/>
  <meta name="citation_reference" content="Medina RA, Stertz S, Manicassamy B et al. Glycosylations in the globular head of the hemagglutinin protein modulate the virulence and antigenic properties of the H1N1 influenza viruses. Science translational medicine 2013; 5."/>
  <meta name="citation_reference" content="Liu W-CC, Jan J-TT, Huang Y-JJ, Chen T-HH, Wu S-CC. Unmasking stem-specific neutralizing epitopes by abolishing N-linked glycosylation sites of influenza hemagglutinin proteins for vaccine design. Journal of virology 2016."/>
  <meta name="citation_reference" content="Walls AC, Tortorici MA, Frenz B et al. Glycan shield and epitope masking of a coronavirus spike protein observed by cryo-electron microscopy. Nat Struct Mol Biol 2016; 23:899–905."/>
  <meta name="citation_reference" content="Yuan Y, Cao D, Zhang Y et al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nature communications 2017; 8:15092."/>
  <meta name="citation_reference" content="Shang J, Zheng Y, Yang Y et al. Cryo-EM structure of infectious bronchitis coronavirus spike protein reveals structural and functional evolution of coronavirus spike proteins. PLoS Pathog 2018; 14:e1007009."/>
  <meta name="citation_reference" content="Shang J, Zheng Y, Yang Y et al. Cryo-Electron Microscopy Structure of Porcine Deltacoronavirus Spike Protein in the Prefusion State. J Virol 2018; 92."/>
  <meta name="citation_reference" content="Song W, Gui M, Wang X, Xiang Y. Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS Pathog 2018; 14:e1007236."/>
  <meta name="citation_reference" content="Yan R, Zhang Y, Li Y, Xia L, Guo Y, Zhou Q. Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2. Science (New York, NY) 2020."/>
  <meta name="citation_reference" content="Zhao S, Musa SS, Lin Q et al. Estimating the Unreported Number of Novel Coronavirus (2019-nCoV) Cases in China in the First Half of January 2020: A Data-Driven Modelling Analysis of the Early Outbreak. Journal of clinical medicine 2020; 9."/>
  <meta name="citation_reference" content="Palomares LA, Srivastava IK, Ramírez OT, Cox MMJMJ. Glycobiotechnology of the Insect Cell-Baculovirus Expression System Technology. Advances in biochemical engineering/biotechnology 2018."/>
  <meta name="citation_reference" content="Wang SF, Tseng SP, Yen CH et al. Antibody-dependent SARS coronavirus infection is mediated by antibodies against spike proteins. Biochem Biophys Res Commun 2014; 451:208–214."/>
  <meta name="citation_reference" content="Wan Y, Shang J, Sun S et al. Molecular Mechanism for Antibody-Dependent Enhancement of Coronavirus Entry. J Virol 2020; 94."/>
  <meta name="citation_reference" content="Kam YW, Kien F, Roberts A et al. Antibodies against trimeric S glycoprotein protect hamsters against SARS-CoV challenge despite their capacity to mediate FcgammaRII-dependent entry into B cells in vitro. Vaccine 2007; 25:729–740."/>
  <meta name="citation_reference" content="Dejnirattisai W, Supasa P, Wongwiwat W et al. Dengue virus sero-cross-reactivity drives antibody-dependent enhancement of infection with zika virus. Nature immunology 2016; 17:1102–1108."/>
  <meta name="citation_reference" content="Rey FAA, Stiasny K, Vaney M-CC, Dellarole M, Heinz FX. The bright and the dark side of human antibody responses to flaviviruses: lessons for vaccine design. EMBO reports 2018; 19:206–224."/>
  <meta name="citation_reference" content="Ngwuta JO, Chen M, Modjarrad K et al. Prefusion F-specific antibodies determine the magnitude of RSV neutralizing activity in human sera. Science translational medicine 2015; 7."/>
  <meta name="citation_reference" content="Crank MC, Ruckwardt TJ, Chen M et al. A proof of concept for structure-based vaccine design targeting RSV in humans. Science 2019; 365:505–509."/>
  <meta name="citation_reference" content="McLellan JS, Chen M, Joyce MG et al. Structure-based design of a fusion glycoprotein vaccine for respiratory syncytial virus. Science (New York, NY) 2013; 342:592–598."/>
  <meta name="citation_reference" content="Li T, Zhang Z, Zhang Z et al. Characterization of native-like HIV-1 gp140 glycoprotein expressed in insect cells. Vaccine 2019; 37:1418–1427."/>
  <meta name="citation_reference" content="Li Z, Wang D, Gu Y et al. Crystal Structures of Two Immune Complexes Identify Determinants for Viral Infectivity and Type-Specific Neutralization of Human Papillomavirus. mBio 2017; 8."/>
  <meta name="citation_reference" content="Zheng SQ, Palovcak E, Armache J-P, Verba KA, Cheng Y, Agard DA. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nature methods 2017; 14:331."/>
  <meta name="citation_reference" content="Zhang K. Gctf: Real-time CTF determination and correction. Journal of structural biology 2016; 193:1–12."/>
  <meta name="citation_reference" content="Punjani A, Rubinstein JL, Fleet DJ, Brubaker MA. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Nat Methods 2017; 14:290–296."/>
  <meta name="citation_reference" content="Kucukelbir A, Sigworth FJ, Tagare HD. Quantifying the local resolution of cryo-EM density maps. Nature methods 2014; 11:63."/>
  <meta name="citation_reference" content="Pettersen EF, Goddard TD, Huang CC et al. UCSF Chimera—a visualization system for exploratory research and analysis. Journal of computational chemistry 2004; 25:1605–1612."/>
  <meta name="twitter:title" content="Characterization of the SARS-CoV-2 Spike in an Early Prefusion Conformation"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Pandemic coronavirus disease 2019 (COVID-19) is caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), for which there are no efficacious vaccines or therapeutics that are urgently needed. We expressed three versions of spike (S) proteins—receptor binding domain (RBD), S1 subunit and S ectodomain—in insect cells. RBD appears monomer in solutions, whereas S1 and S associate into homotrimer with substantial glycosylation. The three proteins confer excellent antigenicity with six convalescent COVID-19 patient sera. Cryo-electron microscopy (cryo-EM) analyses indicate that the SARS-CoV-2 S trimer dominate in a unique conformation distinguished from the classic prefusion conformation of coronaviruses by the upper S1 region at lower position ~15 Å proximal to viral membrane. Such conformation is proposed as an early prefusion state for the SARS-CoV-2 spike that may broaden the knowledge of coronavirus and facilitate vaccine development."/>
  <meta name="og-title" property="og:title" content="Characterization of the SARS-CoV-2 Spike in an Early Prefusion Conformation"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.16.994152v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Pandemic coronavirus disease 2019 (COVID-19) is caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), for which there are no efficacious vaccines or therapeutics that are urgently needed. We expressed three versions of spike (S) proteins—receptor binding domain (RBD), S1 subunit and S ectodomain—in insect cells. RBD appears monomer in solutions, whereas S1 and S associate into homotrimer with substantial glycosylation. The three proteins confer excellent antigenicity with six convalescent COVID-19 patient sera. Cryo-electron microscopy (cryo-EM) analyses indicate that the SARS-CoV-2 S trimer dominate in a unique conformation distinguished from the classic prefusion conformation of coronaviruses by the upper S1 region at lower position ~15 Å proximal to viral membrane. Such conformation is proposed as an early prefusion state for the SARS-CoV-2 spike that may broaden the knowledge of coronavirus and facilitate vaccine development."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-18"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Characterization of the SARS-CoV-2 Spike in an Early Prefusion Conformation | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1201627 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Pandemic coronavirus disease 2019 (COVID-19) is caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), for which there are no efficacious vaccines or therapeutics that are urgently needed. We expressed three versions of spike (S) proteins—receptor binding domain (RBD), S1 subunit and S ectodomain—in insect cells. RBD appears monomer in solutions, whereas S1 and S associate into homotrimer with substantial glycosylation. The three proteins confer excellent antigenicity with six convalescent COVID-19 patient sera. Cryo-electron microscopy (cryo-EM) analyses indicate that the SARS-CoV-2 S trimer dominate in a unique conformation distinguished from the classic prefusion conformation of coronaviruses by the upper S1 region at lower position ~15 Å proximal to viral membrane. Such conformation is proposed as an early prefusion state for the SARS-CoV-2 spike that may broaden the knowledge of coronavirus and facilitate vaccine development.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">The novel coronavirus grouped in betacoronavirus genus has become the third serious virus intruder to human in the coronaviridae, after sever acute respiratory syndrome coronaviruses (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV), recently named SARS-CoV-2. In the phylogenic tree of the coronaviruses, SARS-CoV-2 is genetically close to some bat coronavirus and SARS-CoV, however, with its origin undefined
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. SARS-CoV-2 causative disease “Coronavirus disease 2019” (abbreviated “COVID-19”) is characterized by high fever, dry cough, difficulty breathing and sever atypical pneumonia, which usually be confirmed by virus RNA positive or pulmonary computed tomography (CT) in clinical practice
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, 
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>. In terms of higher human-to-human transmissibility, SARS-CoV-2 has spread over 118 countries and areas, and led to over 125,288 confirmed cases worldwide and at least 4,614 deaths, as of March 12
     <sup>th</sup> 2020. The World Health Organization (WHO) has declared the SARS-CoV-2 epidemic as a pandemic of international concern and updates the COVID-19 situation every day.
    </p>
    <p id="p-5">SARS-CoV-2 is an enveloped, single and positive-stranded RNA virus encapsulated with a genome of ~30 kb. At least three membrane proteins including the surface spike protein (S), an integral membrane protein (M), a membrane protein (E). Like other coronaviruses, S is responsible for initiating the engagement to a specific cellular receptor angiotensin-converting enzyme 2 (ACE2) and mediating the cell-virus membrane fusion by the class I fusion mechanism
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, 
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>. Thus, S is the main target for neutralizing antibodies against viral infection and the core immunogen constituent of vaccine design. S is consisted of S1 and S2 subunits and the cleavage on S1/S2 boundary by protease during biosynthesis is prerequisite for coronaviruses cellular membrane fusion and subsequent infection
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>. SARS-CoV-2 evolves a 4-residue insertion (RRAR) as potential furin cleavage site rather than SARS-CoV and other bat coronaviruses, which may contribute to the higher transmissibility of this novel coronavirus
     <sup>
      <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>, 
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </sup>. Previous studies suggested the infection process of MERS-CoV
     <sup>
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup> and SARS
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
     </sup> viruses, where S trimer undergoes conformational transition from a prefusion conformation ready for ACE2 binding to a postfusion conformation for eventual virus-cell membrane fusion. Structure determination of SARS-CoV and MERS-CoV spike trimers captured a variety of scenarios in the prefusion conformation showing partial (one or two) receptor-binding domain (RBD) in the “up” conformation and the rest in the “down”, and all in either “up” or “down”. The conformation transition from “down” to “up” could expose the receptor binding site, and the subsequent receptor engagement would lead to a substantial conformation rearrangement of S trimer from prefusion conformation to postfusion. Two recent studies
     <sup>
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, 
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup> reported cryo-electron microscope (cryo-EM) structures of SARS-CoV-2 spike trimers in the prefusion conformation with 2 RBDs down and 1 RBD up. In the case of SARS-CoV, this conformational change during RBDs “down” to “up” was associated with the binding of receptor ACE2 as well as the recognition of neutralizing monoclonal antibodies 
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>.
    </p>
    <p id="p-6">A safe and efficacious vaccine is urgently needed to control and eliminate the SARS-CoV-2 infection. Various forms of vaccine candidates, mostly aiming to elicit neutralizing antibodies against S proteins, are under preclinical research or even subjected to clinical trials
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </sup>. Here, we cloned S ectodomain and its fragments RBD and S1 into recombinant baculovirus and expressed the proteins in insect cells. We found that S and S1 formed homotrimer in solutions and the three proteins reacted well with COVID-19 convalescent human sera. Cryo-EM analysis demonstrated the S trimer unexpectedly retains a unique conformation distinguished from the classic prefusion conformation of coronavirus spikes, that may represent an early state rather than the known prefusion conformation of S spike. These results might broaden the knowledge on coronavirus virology and provide another protective conformation of S trimer for structure-based vaccine design against SARS-CoV-2 infection and its causative COVID-19.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Construct design, expression and purification of SARS-CoV-2 S proteins</h3>
     <p id="p-7">To screen a potent immunogen for COVID-19 vaccine development, we designed three constructs—S ectodomain, S1 and RBD—for the SARS-CoV-2 Spike (S) protein expression by aligning the SARS-CoV-2 S gene (Genbank accession no. 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=NC_045512.2&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom">NC_045512.2</a>) to a SARS-CoV strain (Genbank accession no. 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=NC_004718&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom">NC_004718</a>) S gene sequence in terms of structure-defined domain profile of the SARS-CoV S protein (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). The gene of SARS-CoV-2 S ectodomain encoding amino acids (aa) 15-1,213 with removal of its original signal sequence was cloned to the downstream of the gp67 signal sequence in pAcgp67B plasmid vector (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig.1B</a>). and with its C-terminal addition of a thrombin cleavage site, a T4 trimerization foldon motif and his tag. The segments S1 (aa 15-680) and RBD (aa 319-541) were cloned similar to S ectodomain, keeping gp67 signal peptide and his tag to facilitate secretory outside cell and affinity purification, respectively, but without the thrombin site and T4 foldon (
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1A</a>). The three constructed plasmids were respectively co-transfected into Sf9 insect cells with v-cath/chiA gene deficient baculovirus DNA for the generation and amplification of recombinant baculovirus, which were then harnessed to infect Hive Five insect cells to eventually produce recombinant proteins, named S, S1 and RBD respectively.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Schematic map of the SARS-CoV-2 S protein constructs. (A) Linear representations of the S protein primary structure and construct design. NTD, N-terminal domain; RBD, receptor binding domain; SD1, subdomain 1; SD2, subdomain 2; HR1, heptad repeat 1; CH, central helix; CD, connector domain; HR2, heptad repeat 2; TM, transmembrane domain; CT, cytoplasmic tail; FD, T4 foldon motif. The predicted glycosylation sites are indicated above the domain bars. (B) Map of the cloning vector pAcgp67B. The interest genes were cloned to plasmid pAcgp67B at BamH I/Not I site to generate transfer vectors. (C) SDS-PAGE and western blotting of the Ni-NTA purified proteins. RBS, S1 and S were eluted by 250 mM imidazole. Anti-His antibody was used as detection antibody in western blotting. (D) Size-exclusion chromatogram of the second-step purification of the S protein. (E) SDS-PAGE of the four fractions harvested from the chromatography purification as shown in (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1328696874" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Schematic map of the SARS-CoV-2 S protein constructs.&lt;;/span&gt;; (A) Linear representations of the S protein primary structure and construct design. NTD, N-terminal domain; RBD, receptor binding domain; SD1, subdomain 1; SD2, subdomain 2; HR1, heptad repeat 1; CH, central helix; CD, connector domain; HR2, heptad repeat 2; TM, transmembrane domain; CT, cytoplasmic tail; FD, T4 foldon motif. The predicted glycosylation sites are indicated above the domain bars. (B) Map of the cloning vector pAcgp67B. The interest genes were cloned to plasmid pAcgp67B at BamH I/Not I site to generate transfer vectors. (C) SDS-PAGE and western blotting of the Ni-NTA purified proteins. RBS, S1 and S were eluted by 250 mM imidazole. Anti-His antibody was used as detection antibody in western blotting. (D) Size-exclusion chromatogram of the second-step purification of the S protein. (E) SDS-PAGE of the four fractions harvested from the chromatography purification as shown in (D).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F1.medium.gif" width="440" height="227"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F1.medium.gif" width="440" height="227"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">Schematic map of the SARS-CoV-2 S protein constructs.</span>
       <p id="p-8" class="first-child">(A) Linear representations of the S protein primary structure and construct design. NTD, N-terminal domain; RBD, receptor binding domain; SD1, subdomain 1; SD2, subdomain 2; HR1, heptad repeat 1; CH, central helix; CD, connector domain; HR2, heptad repeat 2; TM, transmembrane domain; CT, cytoplasmic tail; FD, T4 foldon motif. The predicted glycosylation sites are indicated above the domain bars. (B) Map of the cloning vector pAcgp67B. The interest genes were cloned to plasmid pAcgp67B at BamH I/Not I site to generate transfer vectors. (C) SDS-PAGE and western blotting of the Ni-NTA purified proteins. RBS, S1 and S were eluted by 250 mM imidazole. Anti-His antibody was used as detection antibody in western blotting. (D) Size-exclusion chromatogram of the second-step purification of the S protein. (E) SDS-PAGE of the four fractions harvested from the chromatography purification as shown in (D).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-9">The recombinant proteins were mostly soluble expressed and secreted into the culture medium. The centrifugation supernatants of cell culture went through metal affinity chromatography using Ni-NTA resin. S, S1 and RBD proteins were mainly eluted in a separation fractions under 250 mM imidazole elution, and resolved as molecular weight (m.w.) of ~180 kDa, 110 kDa and 35 kDa, respectively, in SDS-PAGE as indicated by a corresponding western blotting (WB) using anti-His antibody as detection antibody (
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a>). Interestingly, about one half S proteins were cleaved into S1 (identical migration site to S1 lane in 
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1C</a>) and S2 (about 80kDa developed in anti-His WB) possibly by innate furin of insect cell that was also found in other cases of enzymatic cleave while protein expression in insect cell, such as Flu HA 
      <sup>
       <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
      </sup>. The eluted S fraction was further polished by Superdex 200 to remove contaminative proteins (
      <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1D</a>). These peaks fractionated at retention volume 28mL, 36mL, 48mL, and 65mL, were further harvested and subjected to SDS-PAGE analysis. The results indicated that S proteins together with cleaved S1/S2 were resolved at peak 1 in size-exclusion chromatography (
      <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1D</a>) and showed a high purity of over 95% total S/S1/S2 in gel (
      <a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1E</a>). Overall, one-step Ni-NTA affinity chromatography produced RBD with 95% purity and a yield of 30 mg per L cell culture, S1 with about 90% purity and 10 mg per L yield, while further purification through a size-exclusion chromatography (SEC), the resultant S sample had over 95% purity regarding intact S and cleaved S1/S2, and was harvested in a yield of 1 mg per L cell culture. These data set up a start point for further optimization on expression and purification process of SARS-CoV-2 S immunogen candidates through insect baculovirus expression vector system (BEVS).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Physiochemical properties of SARS-CoV-2 S-RBD, S1 and S proteins</h3>
     <p id="p-10">We next investigated the physiochemical properties of the recombinant S protein and its fragments purified from insect cells, including association potential, thermal stability and glycosylation situation. Firstly, high pressure size-exclusion chromatography (HPSEC) and sedimentation velocity analytical ultracentrifugation (SV-AUC) analyses were carried out to measure the oligomerization potential of the three proteins in solution. RBD, S1 and S all showed single major peak in HPSEC profiles at elution volume of 9.0 mL, 5.5 and 5.3 mL, respectively (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a> and 
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a>). RBD, S1 and S were further verified by SV-AUC, where RBD sedimented as single species of 3.1S in c(s) profile, corresponding to apparent molecular weight 22 kDa (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2D</a>); S1 existed as a dominant species of 11.3 S (estimated as 277 kDa corresponding to S1 trimer) and a minor aggregate form of 20 S (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2E</a>); S and cleaved S1/S2 resolved as 15.2 S, equivalent to 577 kDa, approximately as the theoretical molecular weight of intact S trimer. The three proteins were further analyzed by differential scanning calorimetry (DSC) that was usually used to investigate the inner thermostability of macromolecules or their complexes
      <sup>
       <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
      </sup>. RBD and S1 showed one major peak at comparable thermal denaturation midpoints (Tm) of 46.0 °C and 45.5 °C, respectively (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3G</a> and 
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">3H</a>), whereas S sample showed two major peaks at Tm of 45.5°C (identical to Tm of S1) and 64.5°C (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3I</a>), which might reflect the coexistence of intact S and cleaved S1/S2.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Characterization of the purified RBD, S1 and S proteins. (A-C) HPSEC profiles of the purified RBD, S1 and S proteins; (D-F) AUC profiles of RBD, S1 and S proteins; (G-I) DSC profiles of RBD, S1 and S proteins. (J) Western blotting of three purified proteins treated with Endo H and PNGase F or untreated as control. Anti-His antibody was used as detection antibody in western blotting." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1328696874" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Characterization of the purified RBD, S1 and S proteins.&lt;;/span&gt;; (A-C) HPSEC profiles of the purified RBD, S1 and S proteins; (D-F) AUC profiles of RBD, S1 and S proteins; (G-I) DSC profiles of RBD, S1 and S proteins. (J) Western blotting of three purified proteins treated with Endo H and PNGase F or untreated as control. Anti-His antibody was used as detection antibody in western blotting.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F2.medium.gif" width="420" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F2.medium.gif" width="420" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Characterization of the purified RBD, S1 and S proteins.</span>
       <p id="p-11" class="first-child">(A-C) HPSEC profiles of the purified RBD, S1 and S proteins; (D-F) AUC profiles of RBD, S1 and S proteins; (G-I) DSC profiles of RBD, S1 and S proteins. (J) Western blotting of three purified proteins treated with Endo H and PNGase F or untreated as control. Anti-His antibody was used as detection antibody in western blotting.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Antigenicity of RBD, S1 and S proteins against convalescent sera. (A-F) The reactivity of the RBD, S1 and S proteins against six COVID-19 convalescent human sera (#1-#6) by western blotting (left panel) and ELISA (Right panel). (G, H) Results of two control sera. The gels used for western blotting were duplicates of the reducing SDS gels same as Fig. 1C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1328696874" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Antigenicity of RBD, S1 and S proteins against convalescent sera.&lt;;/span&gt;; (A-F) The reactivity of the RBD, S1 and S proteins against six COVID-19 convalescent human sera (#1-#6) by western blotting (left panel) and ELISA (Right panel). (G, H) Results of two control sera. The gels used for western blotting were duplicates of the reducing SDS gels same as Fig. 1C.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F3.medium.gif" width="381" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F3.medium.gif" width="381" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">Antigenicity of RBD, S1 and S proteins against convalescent sera.</span>
       <p id="p-12" class="first-child">(A-F) The reactivity of the RBD, S1 and S proteins against six COVID-19 convalescent human sera (#1-#6) by western blotting (left panel) and ELISA (Right panel). (G, H) Results of two control sera. The gels used for western blotting were duplicates of the reducing SDS gels same as 
        <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1C</a>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-13">On the other hand, we investigated the glycosylation extent of the three protein by enzymatic deglycosylation analysis. Endo H could unleash the chithobiose core of high mannose and some hybrid oligosaccharides from N-liked glycoproteins, therefore remove the extended branches of glycans and leave the one N acetylglucosamine (GlcNAc) on N-linked glycoproteins. While PNGase F would release N-linked glycan moieties between GlcNAc and ASN residues within a glycoprotein. It should be noted that glycosylation in insect cells is featured as terminal mannose glycans, unlike complex sialylated glycans in mammalian cells, and glycosylation is known to correlate the immunogenicity and broad-coverage protection of a glycoprotein immunogen
      <sup>
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, 
       <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
      </sup>. After the treatment of either Endo H or PNGase F, RBD showed no discernible decrease of molecular weight in SDS-PAGE/anti-His WB, S1 and S2 both demonstrated nearly ~10 kDa decrease, and the intact S exhibited substantial shrinkage in molecular weight of about ~20 kDa decrease (
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2J</a>). The analyses conclude that the glycosylation extent within S glycoprotein is RBD &lt;; S1 ~ S2, consistent to the predicted glycosylation profile of S polypeptide (
      <a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1A</a>).
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Reactivity of SARS-CoV-2 RBD, S1 and S proteins against convalescent COVID-19 human sera</h3>
     <p id="p-14">We next evaluated the antigenicity of the three versions of S proteins by WB and ELISA using a panel of six COVID-19 convalescent human sera, which was collected from COVID-19 patients after they recovered from the disease in the First Affiliated Hospital of Xiamen University. Eight reducing SDS gel duplicates of the one depicted in 
      <a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1C</a> were prepared for WB analysis using these six convalescent sera and two control sera from health human (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3A-3H</a>, left panel). As expected, intact S protein bands reacted well with all the six convalescent sera (
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3A-3H</a>, left panel). Unexpectedly, five of six sera showed no or very weak reactivities against RBD, only Serum #6 possessed RBD’s activity. Among the five sera with lower RBD-reactivity, Serum #2, #3 and #5 well recognized S1 and the cleaved S1 in lane S, suggesting these sera may specifically react with NTD of S1. S2 demonstrated reaction activity against all the six sera, like the intact S. No detectable reaction was observed in the control sera (
      <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3G</a> and 
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">3H</a>). Inconsistent to the WB results, RBD, S1 and S shared comparable reactivities against the convalescent sera in ELISA, although the sera per se presented varied reaction titers (represented as ET50) following the reaction sequence: Serum #5 &gt;; #2 &gt;; #3&gt;; #1 &gt;; # 6 &gt;; #4 (
      <a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3A-3H</a>, right panel).
     </p>
     <p id="p-15">Taken together, RBD, S1 and S proteins from insect cells maintain the native-like SARS-CoV-2 epitopes. These epitopes in native virion should be immunogenic in COVID-19 patients and capable of eliciting high antibody titer in the convalescent phase of SARS-CoV-2 infection. Among these epitopes, most RBD epitopes are strictly tertiary conformation-dependent sites that are damaged upon the mild denatured condition of reductant and SDS treatment, NTD within S1 bears some linear epitopes, whereas S2 part essentially has linear epitopes that are immunogenic in all COVID-19 patients (n=6).</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Cryo-EM structures of SARS-CoV-2 S proteins</h3>
     <p id="p-16">To examine the structure of the trimeric S ectodomain with native sequence, we prepared cryo-EM grids using the Ni-NTA purified S proteins and collected 1,513 electron micrograph movies. Most of motion-corrected micrographs demonstrated plenty of well-dispersed particles with an approximate size as the canonical coronavirus S trimer (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>). A total of 162,645 particles were picked out for multiple rounds of 2D classification, consequently, 37,147 particles grouped into top 10 classes, rendering typical feature of S trimer in prefusion conformation as recently reported
      <sup>
       <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>, 
       <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>
      </sup>, were selected for further analysis (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>). 3D reconstruction (applying 3-fold symmetry) yielded the density map of prefusion spike (S-pre) at resolution of 5.43 Å (
      <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3D</a> and Supplementary Fig. 1A).
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Cryo-EM structure of the SARS-CoV-2 S trimer. (A) Representative micrograph of frozen-hydrated SARS-CoV-2 S particles, Scale bar 50nm. (B, C) Ten and five selected class averages showing the particles along different orientations belonging to prefusion (B) and postfusion (C) S protein, respectively. (D) 5.43 [[#xC5]] density map of prefusion S trimer (S-pre) that is colored by protomer. (E) Structural comparison to the reported prefusion SARS-CoV-2 S trimer (EMD-21374, C3 symmetry, low-pass to 5.43[[#xC5]]) show a different conformation of S-pre (~15[[#xC5]] shorter in height). (F, G) Each domain of the model of prefusion SARS-CoV-2 S monomer (F) or trimer (G) (PDB no. 6VSB) were separately fitted in the density map of S-pre. (H) Schematic diagram shows conformational diversities between NTD and RBD of S-pre (NTD: red, RBD: orange) and reported prefusion S (NTD: light green, RBD: light purple)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1328696874" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Cryo-EM structure of the SARS-CoV-2 S trimer.&lt;;/span&gt;; (A) Representative micrograph of frozen-hydrated SARS-CoV-2 S particles, Scale bar 50nm. (B, C) Ten and five selected class averages showing the particles along different orientations belonging to prefusion (B) and postfusion (C) S protein, respectively. (D) 5.43 [[#xC5]] density map of prefusion S trimer (S-pre) that is colored by protomer. (E) Structural comparison to the reported prefusion SARS-CoV-2 S trimer (EMD-21374, C3 symmetry, low-pass to 5.43[[#xC5]]) show a different conformation of S-pre (~15[[#xC5]] shorter in height). (F, G) Each domain of the model of prefusion SARS-CoV-2 S monomer (F) or trimer (G) (PDB no. 6VSB) were separately fitted in the density map of S-pre. (H) Schematic diagram shows conformational diversities between NTD and RBD of S-pre (NTD: red, RBD: orange) and reported prefusion S (NTD: light green, RBD: light purple).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F4.medium.gif" width="386" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F4.medium.gif" width="386" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <span class="caption-title">Cryo-EM structure of the SARS-CoV-2 S trimer.</span>
       <p id="p-17" class="first-child">(A) Representative micrograph of frozen-hydrated SARS-CoV-2 S particles, Scale bar 50nm. (B, C) Ten and five selected class averages showing the particles along different orientations belonging to prefusion (B) and postfusion (C) S protein, respectively. (D) 5.43 Å density map of prefusion S trimer (S-pre) that is colored by protomer. (E) Structural comparison to the reported prefusion SARS-CoV-2 S trimer (EMD-21374, C3 symmetry, low-pass to 5.43Å) show a different conformation of S-pre (~15Å shorter in height). (F, G) Each domain of the model of prefusion SARS-CoV-2 S monomer (F) or trimer (G) (PDB no. 6VSB) were separately fitted in the density map of S-pre. (H) Schematic diagram shows conformational diversities between NTD and RBD of S-pre (NTD: red, RBD: orange) and reported prefusion S (NTD: light green, RBD: light purple).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-18">Structurally, three S monomers intertwine around each other and associate to homotrimers with 145 Å height seen from side-view and 160 Å diameter in top-view (
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4C</a> and 
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">4D</a>). We then recruited the recently reported cryo-EM map of S prefusion trimer (EMD-21374, at resolution of 3.17 Å, low pass to 5.43 Å prior to structural comparison) and compared our cryo-EM map at same resolution (
      <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4D</a>). It was worthy noted that the compared prefusion SARS-CoV-2 S trimer was engineered with site-directed mutations to stabilize prefusion conformation and expressed in 239F cells. The mutant included two stabilizing proline mutations at residues 986, 987 and a “GSAS” substitution at the furin cleavage site
      <sup>
       <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>
      </sup>. Surprisingly, the alignment demonstrated that the two cryo-EM structures share similar mushroom-shaped architecture in particular nearly identical at stalk moiety (S2 region), but our S-pre shows the cap part (S1 region) at ~15Å lower position than the reported S trimer in RBD-down prefusion conformation (
      <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4D</a>). Regarding to substantial mismatch at the density of 3 S1 subunits, we respectively fitted 5 individual domains (NTD, RBD, SD1, SD2 and S2) of the SARS-CoV-2 S structure (PDB code 6VSB) to our S-pre map. In the fitting map, NTD, RBD, SD2 and S2 could be well placed in the S-pre map, especially for the latter two, which reflects the aforementioned good match at the stalk of the mushroom-shape (Supplementary Fig. 2). However, there is no observable density between RBD and SD2 to accommodate an SD1 model (Supplementary Fig. 2), which suggests SD1 region is dramatically flexible in our S-pre structure (
      <a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4E</a> and 
      <a id="xref-fig-4-8" class="xref-fig" href="#F4">4F</a>). When the combined model of fitted NTD-RBD-SD2-S2 was superimposed to the original S protomer structure (PDB code 6VSB, Chain A, RBD in down conformation), both NTD and RBD in the original S obviously move and rotate up against our combined model (
      <a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4G</a>). The structural comparison demonstrated that the S-pre trimer retains a unique conformation different from the prefusion conformation of the two reported SARS-CoV-2 spike structures (PDB codes 6VSB and 6VXX).
     </p>
     <p id="p-19">We then compared the conformation of our S-pre structure with that of 21 deposited coronavirus S models. Six representative S structures
      <sup>
       <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>, 
       <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–
       <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
      </sup> from four known genus (α-, β-, γ- and δ-genus) in coronaviridae are respectively fitted to the S-pre map (Supplementary Fig. 3). Five S trimer structures of other coronaviruses share similar prefusion conformation with the reported SARS-CoV-2 S structure but substantially distinct with the unique conformation of our S-pre.
     </p>
     <p id="p-20">Apart from most particles classified as S-pre in our sample, 2D classifications also showed five classes of few particles (2,951) assuming an elongated rosette-shape assembly. These particles were further reconstructed and yielded a structure at lower resolution of 8.40 Å (Supplementary Fig. 1B) that could be considered as post-fusion spike (S-post), as the structure has similar shape but shorter length (~170 Å) as compared to the postfusion spike of SARS-CoV
      <sup>
       <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
      </sup> and the presumed one observed in native SARS-CoV-2 virion (BioRxiv, 
      <a href="https://doi.org/10.1101/2020.03.02.972927">https://doi.org/10.1101/2020.03.02.972927</a>) (Supplementary Fig. 4). Fitting the S-post map with the core region structure of SAR-CoV-2 S2 subunit in post-fusion conformation (PDB code 6LXT) indicated that our S-post exhibits roughly rod shape similar with the post-fusion structure (Supplementary Fig. 4).
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Conformational transition of SARS-CoV-2 spike from early prefusion to postfusion</h3>
     <p id="p-21">Briefly, we’ve obtained two conformations of SARS-CoV-2 spike from insect cells. The dominant one maintains the similar mushroom-shaped trimer as the previous models, while the S1 region substantially diverges. The other conformation essentially resembles the postfusion state. However, we could not find the classic prefusion conformation in our sample. We next tried to figure out at which stage the unique conformation occurs during the spike conformation change. The space relationship of NTD or RBD to S2 domain in the unique, RBD-down and RBD-up prefusion conformations (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>) was measured by a reference plant approximately parallel to the viral membrane. The plane is defined by the positions of three equivalent Cα atoms (residue 694 be used) from three S2 subunits of the trimer structures. Numerical data shows that (1) the NTD and RBD in the unique conformation retain the lowest position in the three prefusion conformations; (2) the NTD and RBD of RBD-down prefusion stretch upward 16.6° rotation/16 Å elevation, and 13.1°/18 Å, respectively, with respect to the unique conformation; (3) from RBD-down to RBD-up prefusion state, the RBD elevates 9 Å with an additional rotation whereas the NTD remaining nearly stationary (Supplementary Movie S1). The resultant “up” RBD is ready for ACE2 binding and the spike eventually is rearranged to postfusion state upon RBD-ACE2 interaction
      <sup>
       <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, 
       <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
      </sup> (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a>). The motion trend of NTD and RBD from prefusion to postfusion state in conformational transition is away from the viral membrane, suggesting the unique conformation may occur earlier than RBD-down prefusion conformation, named as “early prefusion conformation” (
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5</a>). This early prefusion conformation might exist in other coronaviruses as well.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SARS-CoV-2 S spike at early pre-, pre- and post-fusion states and the proposed timeline for conformation change. (A) Four conformations of SARS-CoV-2 S spike. For clarity, only one monomer at early prefusion, RBD down prefusion and RBD up prefusion conformation is shown in ribbon mode. The model fitted in our S-pre map is a combined model comprised of individually fitted NTD, RBD, SD2 and S2 domain, which were evicted from the model of prefusion S trimer (PDB no. 6VSB). A deposited postfusion core of SARS-CoV-2 S2 subunit (PDB no. 6LXT) is fitted to our S-post map. Domains are designated by different colors, red: NTD, orange: RBD, yellow: SD1, green: SD2 and cyan: S2. (B) Simplified schematic diagram of the S monomer interpreting the conformation change across different states." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1328696874" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;SARS-CoV-2 S spike at early pre-, pre- and post-fusion states and the proposed timeline for conformation change.&lt;;/span&gt;; (A) Four conformations of SARS-CoV-2 S spike. For clarity, only one monomer at early prefusion, RBD down prefusion and RBD up prefusion conformation is shown in ribbon mode. The model fitted in our S-pre map is a combined model comprised of individually fitted NTD, RBD, SD2 and S2 domain, which were evicted from the model of prefusion S trimer (PDB no. 6VSB). A deposited postfusion core of SARS-CoV-2 S2 subunit (PDB no. 6LXT) is fitted to our S-post map. Domains are designated by different colors, red: NTD, orange: RBD, yellow: SD1, green: SD2 and cyan: S2. (B) Simplified schematic diagram of the S monomer interpreting the conformation change across different states.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F5.medium.gif" width="440" height="345"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F5.medium.gif" width="440" height="345"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.03.16.994152/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <span class="caption-title">SARS-CoV-2 S spike at early pre-, pre- and post-fusion states and the proposed timeline for conformation change.</span>
       <p id="p-22" class="first-child">(A) Four conformations of SARS-CoV-2 S spike. For clarity, only one monomer at early prefusion, RBD down prefusion and RBD up prefusion conformation is shown in ribbon mode. The model fitted in our S-pre map is a combined model comprised of individually fitted NTD, RBD, SD2 and S2 domain, which were evicted from the model of prefusion S trimer (PDB no. 6VSB). A deposited postfusion core of SARS-CoV-2 S2 subunit (PDB no. 6LXT) is fitted to our S-post map. Domains are designated by different colors, red: NTD, orange: RBD, yellow: SD1, green: SD2 and cyan: S2. (B) Simplified schematic diagram of the S monomer interpreting the conformation change across different states.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-23">SARS-CoV-2 has crossed the species barrier and sweep over the planet by person-to-person transmission in an R0 ~2.56 rate
     <sup>
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>
     </sup>, first wave in China and the second wave booming outside China. WHO has declared the event as another pandemic infectious disease in human history, and the epidemiology of SARS-CoV-2 infection is still in data accumulation. Although the biology and virology of SARS-CoV-2 remain elusive, in terms of knowledge on other coronaviruses, the spikes decorating the SARS-CoV-2 virion play a critical role in viral attachment and entry to host cells. Cryo-EM structures of spikes in the prefusion conformation, and RBD-bound receptor ACE2 have indicated the engagement of SARS-CoV-2 to cellular membrane requires a serial of conformational change of RBDs. The change is presumed from the start point of 3 RBDs down in the prefusion conformation, then RBD(s) up for ACE2 binding, and eventually spike is rearranged to postfusion. In this study, we suggest that the SARS-CoV-2 spike may retain at more precedent state than the classic prefusion conformation that has been determined for other coronaviruses. This early prefusion conformation features that the cap of the mushroom-shaped spike constituted by three S1 subunits is more proximal to viral membrane by 15 Å than in the classic prefusion conformation.
    </p>
    <p id="p-24">The SARS-CoV-2 spike expressed in insect cells predominantly retains a unique early prefusion conformation, which was repeatable in at least three batches of samples and is ascribed to two possible reasons – native aa sequence used in the S ectodomain construct and over-expression in insect cells. There is about a half of S proteins undergoing cleavage on the S1/S2 boundary site in the purified samples both after the first Ni-NAT and the second SEC purification (
     <a id="xref-fig-1-12" class="xref-fig" href="#F1">Fig. 1C</a> and 
     <a id="xref-fig-1-13" class="xref-fig" href="#F1">1E</a>). Further analyses suggest the split between S1 and S2 likely takes no effect on the trimerization of S trimer. It is known that the insect cells can confer post-translation glycosylation for protein over-expression as mammalian cells despite the latter can produce more complex sialyation
     <sup>
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>
     </sup>, and thus provide an alternative way to generate glycoprotein in native conformation. Our results indicate that RBD, NTD and S2 domains of SARS-CoV-2 demonstrated different glycosylation extent in insect cells, however, RBD, S1 and S proteins comparably react well with six convalescent COVID-19 human sera albeit they differ in domain composition, polypeptide length and oligomerization.
    </p>
    <p id="p-25">There are numbers of SARS-CoV-2 vaccine candidates, including inactivated, vectored, recombinant and nucleotide vaccine forms, under preclinical research. Various versions of S proteins are the major targets for vaccine immunogen candidate. In addition to potent neutralizing antibody elicitation upon immunization, potential antibody-dependent disease enhancement (ADE) is the major concern for an efficacious SARS-CoV-2 vaccine. ADE has been found in the development of numbers of virus vaccine candidates, including respiratory syndrome virus (RSV), dengue fever 
     <sup>
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, 
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>
     </sup>, human immunodeficient virus (HIV), SARS-CoV, MERS-CoV 
     <sup>
      <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>–
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>
     </sup> and so on. It is believed that ADE is associated with non-neutralization epitope attribute and / or specific antibody isotype
     <sup>
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, 
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>
     </sup>, in which virus-bound antibody would promote the viral infection to immune cells through Fc fragment targeting γFc receptors on the cellular surface and enhance the disease severity. Therefore, the strategy of vaccine design against SARS-CoV-2 should include the consideration of antigen region selection, glycosylation number/extent and exactly presented prefusion conformation. The prefusion conformation needed to be maintained is exemplified by the case of RSV vaccine candidate in which F trimer in prefusion is much potent than postfusion
     <sup>
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>–
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>
     </sup>. Hence the early prefusion conformation proposed for SARS-CoV-2 spike should be drawn an attention for immunogen design as well as the prefusion one.
    </p>
    <p id="p-26">In conclusion, we obtain three kinds of S proteins showing excellent antigenicity and find an early prefusion conformation for SARS-CoV-2 spike. Nevertheless, the molecular level detail for such conformation and the underlying immunogenicity should be further investigated, and whether this conformation recapitulates the exact state of spike in native SARS-CoV-2 virion remains to be determined.</p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-10" class="subsection">
     <h3>Cloning, protein expression and purification</h3>
     <p id="p-27">The SARS-CoV-2 S gene (Genbank accession no. 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=NC_045512.2&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom">NC_045512.2</a>) was synthesized and cloned into a baculovirus shuttle vector pAcgp67B (BD Biosciences, CA, USA) using Gibson assembly. The S construct encoding aa 15-1,213 (numbered as original sequence), contains a thrombin site, a T4 foldon domain to assist in trimerization and a C-terminal 10-His tag for purification. For S1 construct contains gene encoding aa 15-680 followed by a 10-His tag. The RBD construct (aa 319-541) also contains 10-his tag to facilitate purification. In all three constructs, the natural signal peptide (aa 1-14 analyzed by SignalP tool) was replaced with a gp67 secretion signal peptide at N-terminus.
     </p>
     <p id="p-28">The expression and purification of proteins were performed as described previously
      <sup>
       <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>
      </sup>. All plasmids were co-transfected with linearized 2.0 DNA (deficient in 
      <em>v-cath/chiA</em>genes) (Expression Systems, CA, USA) into 
      <em>Sf9</em> insect cells (Thermo Fisher Scientific, MA, USA), according to the protocol provided by the manufacturer (Expression Systems). The transfection supernatant was harvested and amplified 2 times to obtain a high titer of the recombinant viruses. Hive Five cells (BTI-TN-5B1-4) (Thermo Fisher Scientific) were cultured in ESF921 medium (Expression Systems) and infected with recombinant virus at an multiplicity of infection (MOI) of 5 in the exponential growth phase (2× 106 cells/ml, 95% viability) at 28°C for 72 h. The culture media was centrifugated at 8,000 rpm for 20 min. Then the supernatant was dialyzed against phosphate-buffered saline (PBS), pH 7.4, and purified with Ni-sepharose fast flow 6 resin (GE Healthcare, Boston, USA) by the elution with 250 mM imidazole. The protein concentrations of the final purified samples were measured with Pierce™BCA Protein Assay Kit (Thermo Fisher Scientific).
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>SDS-PAGE and western blot</h3>
     <p id="p-29">Protein samples were mixed with loading buffer and boiled for 10 min, and subjected to sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). Equal amounts of proteins for each sample were loaded onto two SDS-PAGE gels, one for western blotting and one for Coomassie staining. The proteins were electrophoresed for 70 min at 80 V in a BioRad MINI-PROTEAN Tetra system (BioRad Laboratories, CA, USA), and the gel was stained with Coomassie Brilliant Blue R-250 (Bio-Rad) for 30 min at room temperature. For western blotting, separated proteins were transferred onto a nitrocellulose membrane (Whatman, Dassel, Germany) using a Trans-Blot Turbo transfer system (Bio-Rad). The membrane was blocked and then incubated for 1 h with an His-tag-specific mouse mAb antibody (Proteintech, Rosemont, USA) or human sera (1:500 dilution). Unbound antibody was removed by five 5-min washes and the membrane was incubated with alkaline phosphatase-conjugated goat anti-mouse secondary antibody or goat anti-human IgG secondary antibody (Abcam, Cambridge, UK). Membranes were washed again and then developed using SuperSignal ELISA Pico Chemiluminescent Substrate Kit (Thermo Fisher Scientific).</p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Enzyme-Linked Immunosorbent Assay (ELISA)</h3>
     <p id="p-30">Purified proteins were coated onto 96-well microtiter plates at 100 ng/well in PBS at 37°C for 4 h. The background was blocked with 1 × Enzyme dilution buffer (PBS + 0.25% casein + 1% gelatin + 0.05% proclin-300) at 37°C for 2 h. Sera were diluted started at 1:100 followed with three-fold serially dilution, and added to the wells (100 μl) and incubated at 37°C for 1 h. Horseradish peroxidase (HRP)-labeled mouse anti-human antibody (Abcam) was used as secondary antibody at 1:5,000 for 30 min. Wells were washed again and the reaction catalyzed using o-phenylenediamine (OPD) substrate at 37°C for 10 min. The OD450nm (reference, OD620nm) was measured on a microplate reader (TECAN, Männedorf, Switzerland), with a cut-off value 0.1. The Half effective titers (ET50) was calculated by sigmoid trend fitting using GraphPad Prism software.</p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Size-Exclusive Chromatography (SEC)</h3>
     <p id="p-31">Ni-NTA purified S proteins were further loaded into Superdex200 (GE Healthcare), the fractions were harvested and analyzed by SDS-PAGE. All high-purity RBD, S1 and S proteins were subjected to HPLC (Waters; Milford, MA) analysis using a TSK Gel G5000PWXL7.8 × 300 mm column (TOSOH, Tokyo, Japan) equilibrated in PBS, pH 7.4. The system flow rate was maintained at 0.5 mL/min and eluted proteins were detected at 280 nm.</p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Analytical Ultracentrifuge (AUC)</h3>
     <p id="p-32">The AUC assay was performed using a Beckman XL-Analytical ultracentrifuge (Beckman Coulter, Fullerton, CA), as described elsewhere
      <sup>
       <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>
      </sup>. The sedimentation velocity (SV) was carried out at 20°C with diluted proteins in PBS. The AN-60 Ti rotor speed was set to 20,000-30,000 rpm according to the molecular weight of the control proteins. Data was collected using SEDFIT computer software, kindly provided by Dr. P. C. Shuck (NIH, Bethesda, MA, USA). Multiple curves were fit to calculate the sedimentation coefficient (S) using continuous sedimentation coefficient distribution model [c(s)], and then the c(s) used to estimate protein molar mass.
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Differential scanning calorimetry (DSC)</h3>
     <p id="p-33">Differential scanning calorimetry (DSC) was carried out on the S proteins using a MicroCal VP-DSC instrument (GE Healthcare, MicroCal Products Group, Northampton, MA) as described previously
      <sup>
       <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>
      </sup>. In brief, all samples with a concentration of 0.2 mg/mL were measured at a heating rate of 1.5°C /min with the scan temperature ranging from 10°C to 90°C. The melting temperatures (Tm) were calculated using MicroCal Origin 7.0 (Origin-Lab Corp., Northampton, MA) software assuming a non-two-state unfolding model.
     </p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Endo-H and PNGase-F digestion</h3>
     <p id="p-34">The Endo-H (NEB) and PNGase-F (NEB) digestions were performed according to the protocol offered by instruction. In brief, the deglycosylation reactions were carried out using 10ug S proteins with 5uL of Endo H or PNGase F and incubated at 37°C overnights. The reactions were loaded in to SDS-PAGE and analyzed by Western blotting using anti-His as detecting anybody.</p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Cryo-EM sample preparation and data collection</h3>
     <p id="p-35">Aliquots (3 μL) of 0.5 mg/mL purified SARS-CoV-2 S protein were loaded onto glow-discharged (60 s at 20 mA) holey carbon Quantifoil grids (R1.2/1.3, 200 mesh, Quantifoil Micro Tools) using a Vitrobot Mark IV (ThermoFisher Scientific) at 100% humidity and 4°C. Data were acquired using the EPU software to control a FEI Tecnai F30 transmission electron microscope (ThermoFisher Scientific) operated at 300 kV. and equipped with a ThermoFisher Falcon-3 direct detector. Images were recorded in the 58-frame movie mode at a nominal magnification of 93,000X with a pixel size of 1.12 Å. The total electron dose was set to 46 e
      <sup>−</sup> Å
      <sup>−2</sup> and the exposure time was 1.5 s. 537 micrographs were collected with a defocus range comprised between 1.5 and 2.8 μm.
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Cryo-EM data processing</h3>
     <p id="p-36">Movie frame alignment and contrast transfer function estimation of each aligned micrograph were carried out with the programs Motioncor
      <sup>
       <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>
      </sup> and Gctf
      <sup>
       <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>
      </sup>. Particles were picked by the ‘Templete picker’ session of cryoSPARC v2
      <sup>
       <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>
      </sup>. Two rounds of reference-free 2D classification were performed and well-defined particle images were selected and non-uniform 3D refinement, 3D reconstruction with C3 symmetry were performed using cryoSPARC v2. The resolutions of the final maps were estimated on the basis of the gold-standard FSC curve with a cutoff at 0.143
      <sup>
       <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>
      </sup>. Density-map-based visualization and segmentation were performed with Chimera
      <sup>
       <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>
      </sup>.
     </p>
    </div>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">Financial Disclosure</h2>
    <p id="p-38">The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
   </div>
   <div class="section COI-statement" id="sec-21">
    <h2 class="">Competing Interest</h2>
    <p id="p-39">The authors have declared that no competing interests exist.</p>
   </div>
   <div class="section" id="sec-22">
    <h2 class="">Author Contributions</h2>
    <p id="p-40">Y.G, S.L. and N.X. designed the study. T.L., Q.Zheng., H.Y., D.W., W.X., Y.Z., X.H., L.Z., Z.Zhang., Z.Zhai., T.C., Z.W., J.C., H.S. and T.D. performed experiments. T.L., Q.Z., H.Y., Y.W., Y.C., Q.Zhao., J.Z., Y.G., S.L. and N.X. analyzed data. T.L., Q.Z., H.Y., Y.G., and S.L. wrote the manuscript. T.L., Q.Zheng., H.Y., D.W., W.X., Q.Zhao., J.Z. Y.G., S.L., and N.X. participated in discussion and interpretation of the results. All authors contributed to experimental design.</p>
   </div>
   <div class="section" id="sec-23">
    <h2 class="">Supplementary Information</h2>
    <p id="p-41">
     <strong>Supplementary Figure 1</strong>
    </p>
    <p id="p-42">
     <strong>Supplementary Figure 2</strong>
    </p>
    <p id="p-43">
     <strong>Supplementary Figure 3</strong>
    </p>
    <p id="p-44">
     <strong>Supplementary Figure 4</strong>
    </p>
    <p id="p-45">
     <strong>Supplementary Movie 1</strong>
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-37">This work was supported by grants from the National Natural Science Foundation (grant no. U1705283, 31670935, 81971932, 81991491, 31730029), and the Major Project of Fujian Provincial Science Foundation for COVID-19 Research (Grants 2020YZ014001).</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list">
     <li>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.1" data-doi="10.1080/22221751.2020.1719902">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">JF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kok</span>
          <span class="cit-name-given-names">KH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">Z</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr>
         <span class="cit-pub-date">2020</span>; 
         <span class="cit-vol">9</span>:
         <span class="cit-fpage">221</span>–
         <span class="cit-lpage">236</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2020.1719902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/22221751.2020.1719902&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-date">2020</span>; 
         <span class="cit-vol">382</span>:
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">733</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">C</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">JAMA</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.4" data-doi="10.1038/s41586-020-2012-7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">XL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">XG</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-date">2020</span>; 
         <span class="cit-vol">579</span>:
         <span class="cit-fpage">270</span>–
         <span class="cit-lpage">273</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=http://www.n&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graham</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baric</span>
          <span class="cit-name-given-names">RS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">F</span>
         </span>. 
         <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Millet</span>
          <span class="cit-name-given-names">JK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Whittaker</span>
          <span class="cit-name-given-names">GR</span>
         </span>. 
         <span class="cit-article-title">Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis</span>. 
         <abbr class="cit-jnl-abbrev">Virus Research</abbr>; 
         <span class="cit-vol">202</span>:
         <span class="cit-fpage">120</span>–
         <span class="cit-lpage">134</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wrapp</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Corbett</span>
          <span class="cit-name-given-names">KS</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. 
         <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.8" data-doi="10.1016/j.cell.2018.12.028">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walls</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiong</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>
          <span class="cit-name-given-names">Y-JJ</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-pub-date">2019</span>; 
         <span class="cit-vol">176</span>:
         <span class="cit-fpage">1026</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft_id%253Dinfo%253Apmid%252F30712865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30712865&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. 
         <abbr class="cit-jnl-abbrev">Nature communications</abbr>
         <span class="cit-pub-date">2017</span>; 
         <span class="cit-vol">8</span>:
         <span class="cit-fpage">15092</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D8%26rft.spage%253D15092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walls</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>
          <span class="cit-name-given-names">Y-JJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tortorici</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wall</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McGuire</span>
          <span class="cit-name-given-names">AT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Veesler</span>
          <span class="cit-name-given-names">D</span>
         </span>. 
         <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.11" data-doi="10.1038/cr.2016.152">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gui</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">H</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">Cell research</abbr>
         <span class="cit-pub-date">2017</span>; 
         <span class="cit-vol">27</span>:
         <span class="cit-fpage">119</span>–
         <span class="cit-lpage">129</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.volume%253D27%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.152%26rft_id%253Dinfo%253Apmid%252F28008928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/cr.2016.152&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28008928&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">S</span>
         </span>. 
         <span class="cit-article-title">Timely development of vaccines against SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr>
         <span class="cit-pub-date">2020</span>; 
         <span class="cit-vol">9</span>:
         <span class="cit-fpage">542</span>–
         <span class="cit-lpage">544</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sui</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hwang</span>
          <span class="cit-name-given-names">WC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Perez</span>
          <span class="cit-name-given-names">S</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structural and functional bases for broad-spectrum neutralization of avian and human influenza A viruses</span>. 
         <abbr class="cit-jnl-abbrev">Nature structural &amp;; molecular biology</abbr>
         <span class="cit-pub-date">2009</span>; 
         <span class="cit-vol">16</span>:
         <span class="cit-fpage">265</span>–
         <span class="cit-lpage">273</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.volume%253D16%26rft.spage%253D265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wei</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pan</span>
          <span class="cit-name-given-names">H</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Robust manufacturing and comprehensive characterization of recombinant hepatitis E virus-like particles in Hecolin(®)</span>. 
         <abbr class="cit-jnl-abbrev">Vaccine</abbr>
         <span class="cit-pub-date">2014</span>; 
         <span class="cit-vol">32</span>:
         <span class="cit-fpage">4039</span>–
         <span class="cit-lpage">4050</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D32%26rft.spage%253D4039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Medina</span>
          <span class="cit-name-given-names">RA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stertz</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Manicassamy</span>
          <span class="cit-name-given-names">B</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Glycosylations in the globular head of the hemagglutinin protein modulate the virulence and antigenic properties of the H1N1 influenza viruses</span>. 
         <abbr class="cit-jnl-abbrev">Science translational medicine</abbr>
         <span class="cit-pub-date">2013</span>; 
         <span class="cit-vol">5</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">W-CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jan</span>
          <span class="cit-name-given-names">J-TT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">Y-JJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">T-HH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">S-CC</span>
         </span>. 
         <span class="cit-article-title">Unmasking stem-specific neutralizing epitopes by abolishing N-linked glycosylation sites of influenza hemagglutinin proteins for vaccine design</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-pub-date">2016</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.17" data-doi="10.1038/nsmb.3293">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walls</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tortorici</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Frenz</span>
          <span class="cit-name-given-names">B</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Glycan shield and epitope masking of a coronavirus spike protein observed by cryo-electron microscopy</span>. 
         <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr>
         <span class="cit-pub-date">2016</span>; 
         <span class="cit-vol">23</span>:
         <span class="cit-fpage">899</span>–
         <span class="cit-lpage">905</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BStruct%2BMol%2BBiol%26rft.volume%253D23%26rft.spage%253D899%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3293%26rft_id%253Dinfo%253Apmid%252F27617430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nsmb.3293&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27617430&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.16.994152v2.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. 
         <abbr class="cit-jnl-abbrev">Nature communications</abbr>
         <span class="cit-pub-date">2017</span>; 
         <span class="cit-vol">8</span>:
         <span class="cit-fpage">15092</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D8%26rft.spage%253D15092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.16.994152v2.19" data-doi="10.1371/journal.ppat.1007009">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Y</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-EM structure of infectious bronchitis coronavirus spike protein reveals structural and functional evolution of coronavirus spike proteins</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-pub-date">2018</span>; 
         <span class="cit-vol">14</span>:
         <span class="cit-fpage">e1007009</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007009%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007009%26rft_id%253Dinfo%253Apmid%252F29684066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007009&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29684066&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Y</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-Electron Microscopy Structure of Porcine Deltacoronavirus Spike Protein in the Prefusion State</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-pub-date">2018</span>; 
         <span class="cit-vol">92</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.21" data-doi="10.1371/journal.ppat.1007236">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gui</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiang</span>
          <span class="cit-name-given-names">Y</span>
         </span>. 
         <span class="cit-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-pub-date">2018</span>; 
         <span class="cit-vol">14</span>:
         <span class="cit-fpage">e1007236</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1007236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007236%26rft_id%253Dinfo%253Apmid%252F30102747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007236&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30102747&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">Q</span>
         </span>. 
         <span class="cit-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span>. 
         <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr>
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Musa</span>
          <span class="cit-name-given-names">SS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">Q</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Estimating the Unreported Number of Novel Coronavirus (2019-nCoV) Cases in China in the First Half of January 2020: A Data-Driven Modelling Analysis of the Early Outbreak</span>. 
         <abbr class="cit-jnl-abbrev">Journal of clinical medicine</abbr>
         <span class="cit-pub-date">2020</span>; 
         <span class="cit-vol">9</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Palomares</span>
          <span class="cit-name-given-names">LA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Srivastava</span>
          <span class="cit-name-given-names">IK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ramírez</span>
          <span class="cit-name-given-names">OT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cox</span>
          <span class="cit-name-given-names">MMJMJ</span>
         </span>. 
         <span class="cit-article-title">Glycobiotechnology of the Insect Cell-Baculovirus Expression System Technology</span>. 
         <abbr class="cit-jnl-abbrev">Advances in biochemical engineering/biotechnology</abbr>
         <span class="cit-pub-date">2018</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.25" data-doi="10.1016/j.bbrc.2014.07.090">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">SF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tseng</span>
          <span class="cit-name-given-names">SP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yen</span>
          <span class="cit-name-given-names">CH</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Antibody-dependent SARS coronavirus infection is mediated by antibodies against spike proteins</span>. 
         <abbr class="cit-jnl-abbrev">Biochem Biophys Res Commun</abbr>
         <span class="cit-pub-date">2014</span>; 
         <span class="cit-vol">451</span>:
         <span class="cit-fpage">208</span>–
         <span class="cit-lpage">214</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.volume%253D451%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2014.07.090%26rft_id%253Dinfo%253Apmid%252F25073113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2014.07.090&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25073113&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>
          <span class="cit-name-given-names">S</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Molecular Mechanism for Antibody-Dependent Enhancement of Coronavirus Entry</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-pub-date">2020</span>; 
         <span class="cit-vol">94</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.27" data-doi="10.1016/j.vaccine.2006.08.011">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kam</span>
          <span class="cit-name-given-names">YW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kien</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>
          <span class="cit-name-given-names">A</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Antibodies against trimeric S glycoprotein protect hamsters against SARS-CoV challenge despite their capacity to mediate FcgammaRII-dependent entry into B cells in vitro</span>. 
         <abbr class="cit-jnl-abbrev">Vaccine</abbr>
         <span class="cit-pub-date">2007</span>; 
         <span class="cit-vol">25</span>:
         <span class="cit-fpage">729</span>–
         <span class="cit-lpage">740</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.stitle%253DVaccine%26rft.aulast%253DKam%26rft.auinit1%253DY.%2BW.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D729%26rft.epage%253D740%26rft.atitle%253DAntibodies%2Bagainst%2Btrimeric%2BS%2Bglycoprotein%2Bprotect%2Bhamsters%2Bagainst%2BSARS-CoV%2Bchallenge%2Bdespite%2Btheir%2Bcapacity%2Bto%2Bmediate%2BFcgammaRII-dependent%2Bentry%2Binto%2BB%2Bcells%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2006.08.011%26rft_id%253Dinfo%253Apmid%252F17049691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2006.08.011&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17049691&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.28" data-doi="10.1038/ni.3515">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dejnirattisai</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Supasa</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wongwiwat</span>
          <span class="cit-name-given-names">W</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Dengue virus sero-cross-reactivity drives antibody-dependent enhancement of infection with zika virus</span>. 
         <abbr class="cit-jnl-abbrev">Nature immunology</abbr>
         <span class="cit-pub-date">2016</span>; 
         <span class="cit-vol">17</span>:
         <span class="cit-fpage">1102</span>–
         <span class="cit-lpage">1108</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bimmunology%26rft.volume%253D17%26rft.spage%253D1102%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.3515%26rft_id%253Dinfo%253Apmid%252F27339099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ni.3515&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27339099&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.29" data-doi="10.15252/embr.201745302">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rey</span>
          <span class="cit-name-given-names">FAA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stiasny</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vaney</span>
          <span class="cit-name-given-names">M-CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dellarole</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Heinz</span>
          <span class="cit-name-given-names">FX</span>
         </span>. 
         <span class="cit-article-title">The bright and the dark side of human antibody responses to flaviviruses: lessons for vaccine design</span>. 
         <abbr class="cit-jnl-abbrev">EMBO reports</abbr>
         <span class="cit-pub-date">2018</span>; 
         <span class="cit-vol">19</span>:
         <span class="cit-fpage">206</span>–
         <span class="cit-lpage">224</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEMBO%2Breports%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembr.201745302%26rft_id%253Dinfo%253Apmid%252F29282215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6ODoiMTkvMi8yMDYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xOC8yMDIwLjAzLjE2Ljk5NDE1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ngwuta</span>
          <span class="cit-name-given-names">JO</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Modjarrad</span>
          <span class="cit-name-given-names">K</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Prefusion F-specific antibodies determine the magnitude of RSV neutralizing activity in human sera</span>. 
         <abbr class="cit-jnl-abbrev">Science translational medicine</abbr>
         <span class="cit-pub-date">2015</span>; 
         <span class="cit-vol">7</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.16.994152v2.31" data-doi="10.1126/science.aav9033">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Crank</span>
          <span class="cit-name-given-names">MC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruckwardt</span>
          <span class="cit-name-given-names">TJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">M</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A proof of concept for structure-based vaccine design targeting RSV in humans</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-pub-date">2019</span>; 
         <span class="cit-vol">365</span>:
         <span class="cit-fpage">505</span>–
         <span class="cit-lpage">509</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCrank%26rft.auinit1%253DM.%2BC.%26rft.volume%253D365%26rft.issue%253D6452%26rft.spage%253D505%26rft.epage%253D509%26rft.atitle%253DA%2Bproof%2Bof%2Bconcept%2Bfor%2Bstructure-based%2Bvaccine%2Bdesign%2Btargeting%2BRSV%2Bin%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aav9033%26rft_id%253Dinfo%253Apmid%252F31371616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjUvNjQ1Mi81MDUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xOC8yMDIwLjAzLjE2Ljk5NDE1Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McLellan</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joyce</span>
          <span class="cit-name-given-names">MG</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure-based design of a fusion glycoprotein vaccine for respiratory syncytial virus</span>. 
         <abbr class="cit-jnl-abbrev">Science (New York, NY)</abbr>
         <span class="cit-pub-date">2013</span>; 
         <span class="cit-vol">342</span>:
         <span class="cit-fpage">592</span>–
         <span class="cit-lpage">598</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BNY%2529%26rft.volume%253D342%26rft.spage%253D592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Z</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Characterization of native-like HIV-1 gp140 glycoprotein expressed in insect cells</span>. 
         <abbr class="cit-jnl-abbrev">Vaccine</abbr>
         <span class="cit-pub-date">2019</span>; 
         <span class="cit-vol">37</span>:
         <span class="cit-fpage">1418</span>–
         <span class="cit-lpage">1427</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D37%26rft.spage%253D1418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gu</span>
          <span class="cit-name-given-names">Y</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Crystal Structures of Two Immune Complexes Identify Determinants for Viral Infectivity and Type-Specific Neutralization of Human Papillomavirus</span>. 
         <abbr class="cit-jnl-abbrev">mBio</abbr>
         <span class="cit-pub-date">2017</span>; 
         <span class="cit-vol">8</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">SQ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palovcak</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Armache</span>
          <span class="cit-name-given-names">J-P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Verba</span>
          <span class="cit-name-given-names">KA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Agard</span>
          <span class="cit-name-given-names">DA</span>
         </span>. 
         <span class="cit-article-title">MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</span>. 
         <abbr class="cit-jnl-abbrev">Nature methods</abbr>
         <span class="cit-pub-date">2017</span>; 
         <span class="cit-vol">14</span>:
         <span class="cit-fpage">331</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D14%26rft.spage%253D331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.36" data-doi="10.1016/j.jsb.2015.11.003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">K</span>
         </span>. 
         <span class="cit-article-title">Gctf: Real-time CTF determination and correction</span>. 
         <abbr class="cit-jnl-abbrev">Journal of structural biology</abbr>
         <span class="cit-pub-date">2016</span>; 
         <span class="cit-vol">193</span>:
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">12</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.volume%253D193%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2015.11.003%26rft_id%253Dinfo%253Apmid%252F26592709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jsb.2015.11.003&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26592709&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.37" data-doi="10.1038/nmeth.4169">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Punjani</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rubinstein</span>
          <span class="cit-name-given-names">JL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fleet</span>
          <span class="cit-name-given-names">DJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brubaker</span>
          <span class="cit-name-given-names">MA</span>
         </span>. 
         <span class="cit-article-title">cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>
         <span class="cit-pub-date">2017</span>; 
         <span class="cit-vol">14</span>:
         <span class="cit-fpage">290</span>–
         <span class="cit-lpage">296</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D290%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4169%26rft_id%253Dinfo%253Apmid%252F28165473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.4169&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28165473&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kucukelbir</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sigworth</span>
          <span class="cit-name-given-names">FJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tagare</span>
          <span class="cit-name-given-names">HD</span>
         </span>. 
         <span class="cit-article-title">Quantifying the local resolution of cryo-EM density maps</span>. 
         <abbr class="cit-jnl-abbrev">Nature methods</abbr>
         <span class="cit-pub-date">2014</span>; 
         <span class="cit-vol">11</span>:
         <span class="cit-fpage">63</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D11%26rft.spage%253D63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.16.994152v2.39" data-doi="10.1002/jcc.20084">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pettersen</span>
          <span class="cit-name-given-names">EF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Goddard</span>
          <span class="cit-name-given-names">TD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">CC</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">UCSF Chimera—a visualization system for exploratory research and analysis</span>. 
         <abbr class="cit-jnl-abbrev">Journal of computational chemistry</abbr>
         <span class="cit-pub-date">2004</span>; 
         <span class="cit-vol">25</span>:
         <span class="cit-fpage">1605</span>–
         <span class="cit-lpage">1612</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15264254&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.03.16.994152.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000223379100005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
