<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/02/15/349001.atom" />
<meta name="HW.pisa" content="biorxiv;349001v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes" />
<meta name="DC.Identifier" content="10.1101/349001" />
<meta name="DC.Date" content="2019-02-15" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Defective viral genomes of the copy-back type (cbDVGs) are the primary initiators of the antiviral immune response during infection with respiratory syncytial virus (RSV) both in vitro and in vivo. However, the mechanism governing cbDVG generation remains unknown, thereby limiting our ability to manipulate cbDVG content in order to modulate the host response to infection. Here we report a specific genomic signal that mediates the generation of RSV cbDVGs. Using a customized bioinformatics tool, we identified regions in the RSV genome frequently used to generate cbDVGs during infection. We then created a minigenome system to validate the function of one of these sequences and to determine if specific nucleotides were essential for cbDVG generation at that position. Further, we created a recombinant virus that selectively produced a specific cbDVG based on variations introduced in this sequence. The identified sequence was also found as a common site for cbDVG generation during natural RSV infections, and common cbDVGs generated at this sequence were found among samples from various infected patients. These data demonstrate that sequences encoded in the viral genome are critical determinants of the location of cbDVG generation and, therefore, this is not a stochastic process. Most importantly, these findings open the possibility of genetically manipulating cbDVG formation to modulate infection outcome.

Author summary Copy-back defective viral genomes (cbDVGs) regulate infection and pathogenesis of Mononegavirales. cbDVG are believed to arise from random errors that occur during virus replication and the predominant hypothesis is that the viral polymerase is the main driver of cbDVG generation. Here we describe a specific genomic sequence in the RSV genome that is necessary for the generation of a large proportion of the cbDVG population present during infection. We identified specific nucleotides that when modified altered cbDVG generation at this position, and we created a recombinant virus that selectively produced cbDVGs based on mutations in this sequence. These data demonstrate that the generation of RSV cbDVGs is regulated by specific viral sequences and that these sequences can be manipulated to alter the content and quality of cbDVG generated during infection." />
<meta name="DC.Contributor" content="Yan Sun" />
<meta name="DC.Contributor" content="Eun Ji Kim" />
<meta name="DC.Contributor" content="Sébastien A. Felt" />
<meta name="DC.Contributor" content="Louis J. Taylor" />
<meta name="DC.Contributor" content="Divyansh Agarwal" />
<meta name="DC.Contributor" content="Gregory R. Grant" />
<meta name="DC.Contributor" content="Carolina B. López" />
<meta name="article:published_time" content="2019-02-15" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Defective viral genomes of the copy-back type (cbDVGs) are the primary initiators of the antiviral immune response during infection with respiratory syncytial virus (RSV) both &lt;i&gt;in vitro&lt;/i&gt; and &lt;i&gt;in vivo.&lt;/i&gt; However, the mechanism governing cbDVG generation remains unknown, thereby limiting our ability to manipulate cbDVG content in order to modulate the host response to infection. Here we report a specific genomic signal that mediates the generation of RSV cbDVGs. Using a customized bioinformatics tool, we identified regions in the RSV genome frequently used to generate cbDVGs during infection. We then created a minigenome system to validate the function of one of these sequences and to determine if specific nucleotides were essential for cbDVG generation at that position. Further, we created a recombinant virus that selectively produced a specific cbDVG based on variations introduced in this sequence. The identified sequence was also found as a common site for cbDVG generation during natural RSV infections, and common cbDVGs generated at this sequence were found among samples from various infected patients. These data demonstrate that sequences encoded in the viral genome are critical determinants of the location of cbDVG generation and, therefore, this is not a stochastic process. Most importantly, these findings open the possibility of genetically manipulating cbDVG formation to modulate infection outcome.&lt;/p&gt;&lt;h3&gt;Author summary&lt;/h3&gt;
&lt;p&gt;Copy-back defective viral genomes (cbDVGs) regulate infection and pathogenesis of &lt;i&gt;Mononegavirales.&lt;/i&gt; cbDVG are believed to arise from random errors that occur during virus replication and the predominant hypothesis is that the viral polymerase is the main driver of cbDVG generation. Here we describe a specific genomic sequence in the RSV genome that is necessary for the generation of a large proportion of the cbDVG population present during infection. We identified specific nucleotides that when modified altered cbDVG generation at this position, and we created a recombinant virus that selectively produced cbDVGs based on mutations in this sequence. These data demonstrate that the generation of RSV cbDVGs is regulated by specific viral sequences and that these sequences can be manipulated to alter the content and quality of cbDVG generated during infection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;349001v3" />
<meta name="citation_id" content="349001v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/349001v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/349001v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/349001v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001.full.pdf" />
<meta name="citation_doi" content="10.1101/349001" />
<meta name="citation_num_pages" content="19" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="349001" />
<meta name="citation_author" content="Yan Sun" />
<meta name="citation_author_institution" content="Department of Pathobiology, School of Veterinary Medicine, University of Pennsylvania" />
<meta name="citation_author" content="Eun Ji Kim" />
<meta name="citation_author_institution" content="Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania" />
<meta name="citation_author" content="Sébastien A. Felt" />
<meta name="citation_author_institution" content="Department of Pathobiology, School of Veterinary Medicine, University of Pennsylvania" />
<meta name="citation_author" content="Louis J. Taylor" />
<meta name="citation_author_institution" content="Department of Microbiology, Perelman School of Medicine, University of Pennsylvania" />
<meta name="citation_author" content="Divyansh Agarwal" />
<meta name="citation_author_institution" content="Department of Pathobiology, School of Veterinary Medicine, University of Pennsylvania" />
<meta name="citation_author" content="Gregory R. Grant" />
<meta name="citation_author_institution" content="Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania" />
<meta name="citation_author_institution" content="Department of Genetics, Perelman School of Medicine, University of Pennsylvania" />
<meta name="citation_author" content="Carolina B. López" />
<meta name="citation_author_institution" content="Department of Pathobiology, School of Veterinary Medicine, University of Pennsylvania" />
<meta name="citation_author_email" content="lopezca@upenn.edu" />
<meta name="citation_reference" content="Henle W, Henle G. INTERFERENCE OF INACTIVE VIRUS WITH THE PROPAGATION OF VIRUS OF INFLUENZA. Science. 1943;98(2534):87–9. Epub 1943/07/23. doi:10.1126/science.98.2534.87. PubMed PMID:17749157." />
<meta name="citation_reference" content="von MP. Propagation of the PR8 strain of influenza A virus in chick embryos. II. The formation of incomplete virus following inoculation of large doses of seed virus. Acta Pathol Microbiol Scand. 1951;28(3):278–93. PubMed PMID: 14856732." />
<meta name="citation_reference" content="Von Magnus P. Incomplete forms of influenza virus. Adv Virus Res. 1954;2:59–79. PubMed PMID:13228257." />
<meta name="citation_reference" content="Lopez CB. Defective viral genomes: critical danger signals of viral infections. J Virol. 2014;88(16):8720–3. doi:10.1128/JVI.00707-14. PubMed PMID:24872580; PubMed Central PMCID:PMCPMC4136278." />
<meta name="citation_reference" content="Weber M, Weber F. RIG-I-like receptors and negative-strand RNA viruses: RLRly bird catches some worms. Cytokine Growth Factor Rev. 2014;25(5):621–8. doi:10.1016/j.cytogfr.2014.05.004. PubMed PMID:24894317." />
<meta name="citation_reference" content="Sun Y, Jain D, Koziol-White cj, Genoyer E, Gilbert M, Tapia K, et al. Immunostimulatory Defective Viral Genomes from Respiratory Syncytial Virus Promote a Strong Innate Antiviral Response during Infection in Mice and Humans. PLoS Pathog. 2015;11(9):e1005122. doi:10.1371/journal.ppat.1005122." />
<meta name="citation_reference" content="Tapia K, Kim WK, Sun Y, Mercado-Lopez X, Dunay E, Wise M, et al. Defective viral genomes arising in vivo provide critical danger signals for the triggering of lung antiviral immunity. PLoS Pathog. 2013;9(10):e1003703. doi:10.1371/journal.ppat.1003703. PubMed PMID:24204261; PubMed Central PMCID:PMCPMC3814336." />
<meta name="citation_reference" content="Vasilijevic J, Zamarreno N, Oliveros JC, Rodriguez-Frandsen A, Gomez G, Rodriguez G, et al. Reduced accumulation of defective viral genomes contributes to severe outcome in influenza virus infected patients. PLoS Pathog. 2017;13(10):e1006650. Epub 2017/10/13. doi:10.1371/journal.ppat.1006650. PubMed PMID:29023600; PubMed Central PMCID:PMCPmc5638565." />
<meta name="citation_reference" content="Strahle L, Garcin D, Kolakofsky D. Sendai virus defective-interfering genomes and the activation of interferon-beta. Virology. 2006;351(1):101–11. doi:10.1016/j.virol.2006.03.022. PubMed PMID:16631220." />
<meta name="citation_reference" content="Frensing T. Defective interfering viruses and their impact on vaccines and viral vectors. Biotechnol J. 2015;10(5):681–9. doi:10.1002/biot.201400429. PubMed PMID:25728309." />
<meta name="citation_reference" content="Huang AS, Baltimore D. Defective viral particles and viral disease processes. Nature. 1970;226(5243):325–7. PubMed PMID:5439728." />
<meta name="citation_reference" content="Lazzarini RA, Keene JD, Schubert M. The origins of defective interfering particles of the negative-strand RNA viruses. Cell. 1981;26(2 Pt 2):145–54. PubMed PMID:7037195." />
<meta name="citation_reference" content="Pathak KB, Nagy PD. Defective Interfering RNAs: Foes of Viruses and Friends of Virologists. Viruses. 2009;1(3):895–919. doi:10.3390/v10308951. PubMed PMID:21994575; PubMed Central PMCID:PMCPMC3185524." />
<meta name="citation_reference" content="Janda JM, Davis AR, Nayak DP, De BK. Diversity and generation of defective interfering influenza virus particles. Virology. 1979;95(1):48–58. Epub 1979/05/01. PubMed PMID:442544." />
<meta name="citation_reference" content="Nayak DP, Chambers TM, Akkina RK. Defective-interfering (DI) RNAs of influenza viruses: origin, structure, expression, and interference. Curr Top Microbiol Immunol. 1985;114:103–51. PubMed PMID:3888540." />
<meta name="citation_reference" content="Kajigaya S, Arakawa H, Kuge S, Koi T, Imura N, Nomoto A. Isolation and characterization of defective-interfering particles of poliovirus Sabin 1 strain. Virology. 1985;142(2):307–16. Epub 1985/04/30. PubMed PMID:2997988." />
<meta name="citation_reference" content="Poirier EZ, Mounce BC, Rozen-Gagnon K, Hooikaas PJ, Stapleford KA, Moratorio G, et al. Low-Fidelity Polymerases of Alphaviruses Recombine at Higher Rates To Overproduce Defective Interfering Particles. J Virol. 2015;90(5):2446–54. doi:10.1128/JVI.02921-15. PubMed PMID:26676773; PubMed Central PMCID:PMCPMC4810721." />
<meta name="citation_reference" content="Xiao CT, Liu ZH, Yu XL, Ge M, Li RC, Xiao BR, et al. Identification of new defective interfering RNA species associated with porcine reproductive and respiratory syndrome virus infection. Virus Res. 2011;158(1-2):33–6. Epub 2011/03/10. doi:10.1016/j.virusres.2011.03.001. PubMed PMID:21385595." />
<meta name="citation_reference" content="Odagiri T, Tashiro M. Segment-specific noncoding sequences of the influenza virus genome RNA are involved in the specific competition between defective interfering RNA and its progenitor RNA segment at the virion assembly step. J Virol. 1997;71(3):2138–45. PubMed PMID:9032347; PubMed Central PMCID:PMCPMC191316." />
<meta name="citation_reference" content="Shingai M, Ebihara T, Begum NA, Kato A, Honma T, Matsumoto K, et al. Differential type I IFN-inducing abilities of wild-type versus vaccine strains of measles virus. J Immunol. 2007;179(9):6123–33. Epub 2007/10/20. doi:179/9/6123 [pii]. PubMed PMID:17947687." />
<meta name="citation_reference" content="Ho TH, Kew C, Lui PY, Chan CP, Satoh T, Akira S, et al. PACT- and RIG-I-Dependent Activation of Type I Interferon Production by a Defective Interfering RNA Derived from Measles Virus Vaccine. J Virol. 2016;90(3):1557–68. doi:10.1128/JVI.02161-15. PubMed PMID:26608320; PubMed Central PMCID:PMCPMC4719617." />
<meta name="citation_reference" content="Leppert M, Kort L, Kolakofsky D. Further characterization of Sendai virus DI-RNAs: a model for their generation. Cell. 1977;12(2):539–52. Epub 1977/10/01. doi:0092-8674(77)90130-1 [pii]. PubMed PMID:199355." />
<meta name="citation_reference" content="citation_title=Initiation Signal in Viral Gene Expression;citation_pages=151-207;citation_year=1981" />
<meta name="citation_reference" content="Fearns R, Collins PL. Role of the M2-1 transcription antitermination protein of respiratory syncytial virus in sequential transcription. J Virol. 1999;73(7):5852–64. PubMed PMID:10364337; PubMed Central PMCID:PMCPMC112646." />
<meta name="citation_reference" content="Shcherbo D, Murphy CS, Ermakova GV, Solovieva EA, Chepurnykh TV, Shcheglov AS, et al. Farred fluorescent tags for protein imaging in living tissues. Biochem J. 2009;418(3):567–74. doi:10.1042/BJ20081949. PubMed PMID:19143658; PubMed Central PMCID:PMCPMC2893397." />
<meta name="citation_reference" content="Pfaller CK, Mastorakos GM, Matchett WE, Ma X, Samuel CE, Cattaneo R. Measles Virus Defective Interfering RNAs Are Generated Frequently and Early in the Absence of C Protein and Can Be Destabilized by Adenosine Deaminase Acting on RNA-1-Like Hypermutations. J Virol. 2015;89(15):7735–47. Epub 2015/05/15. doi:10.1128/jvi.01017-15. PubMed PMID:25972541; PubMed Central PMCID:PMCPmc4505647." />
<meta name="citation_reference" content="Yoshida A, Kawabata R, Honda T, Sakai K, Ami Y, Sakaguchi T, et al. A Single Amino Acid Substitution within the Paramyxovirus Sendai Virus Nucleoprotein Is a Critical Determinant for Production of Interferon-Beta-Inducing Copyback-Type Defective Interfering Genomes. J Virol. 2018;92(5). doi:10.1128/JVI.02094-17. PubMed PMID:29237838; PubMed Central PMCID:PMCPMC5809723." />
<meta name="citation_reference" content="Fearns R, Plemper RK. Polymerases of paramyxoviruses and pneumoviruses. Virus Res. 2017;234:87–102. doi:10.1016/j.virusres.2017.01.008. PubMed PMID:28104450; PubMed Central PMCID:PMCPMC5476513." />
<meta name="citation_reference" content="Kuo L, Fearns R, Collins PL. Analysis of the gene start and gene end signals of human respiratory syncytial virus: quasi-templated initiation at position 1 of the encoded mRNA. J Virol. 1997;71(7):4944–53. PubMed PMID:9188557; PubMed Central PMCID:PMCPMC191725." />
<meta name="citation_reference" content="Calain P, Roux L. The rule of six, a basic feature for efficient replication of Sendai virus defective interfering RNA. J Virol. 1993;67(8):4822–30. Epub 1993/08/01. PubMed PMID:8392616; PubMed Central PMCID:PMC237869." />
<meta name="citation_reference" content="Kolakofsky D, Pelet T, Garcin D, Hausmann S, Curran J, Roux L. Paramyxovirus RNA synthesis and the requirement for hexamer genome length: the rule of six revisited. J Virol. 1998;72(2):891–9. PubMed PMID:9444980." />
<meta name="citation_reference" content="Tawar RG, Duquerroy S, Vonrhein C, Varela PF, Damier-Piolle L, Castagne N, et al. Crystal structure of a nucleocapsid-like nucleoprotein-RNA complex of respiratory syncytial virus. Science (New York, NY). 2009;326(5957):1279–83. doi:10.1126/science.1177634. PubMed PMID:19965480." />
<meta name="citation_reference" content="Samal SK, Collins PL. RNA replication by a respiratory syncytial virus RNA analog does not obey the rule of six and retains a nonviral trinucleotide extension at the leader end. J Virol. 1996;70(8):5075–82. PubMed PMID:8764015; PubMed Central PMCID:PMCPMC190462." />
<meta name="citation_reference" content="Fodor E, Mingay LJ, Crow M, Deng T, Brownlee GG. A single amino acid mutation in the PA subunit of the influenza virus RNA polymerase promotes the generation of defective interfering RNAs. J Virol. 2003;77(8):5017–20. PubMed PMID:12663810; PubMed Central PMCID:PMCPMC152145." />
<meta name="citation_reference" content="Odagiri T, Tobita K. Mutation in NS2, a nonstructural protein of influenza A virus, extragenically causes aberrant replication and expression of the PA gene and leads to generation of defective interfering particles. Proc Natl Acad Sci U S A. 1990;87(15):5988–92. PubMed PMID:2143025; PubMed Central PMCID:PMCPMC54455." />
<meta name="citation_reference" content="Kang CY, Weide LG, Tischfield JA. Suppression of vesicular stomatitis virus defective intefering particle generation by a function(s) associated with human chromosome 16. J Virol. 1981;40(3):946–52. PubMed PMID:6275129; PubMed Central PMCID:PMCPMC256708." />
<meta name="citation_reference" content="Whistler T, Bellini WJ, Rota PA. Generation of defective interfering particles by two vaccine strains of measles virus. Virology. 1996;220(2):480–4. PubMed PMID:8661398." />
<meta name="citation_reference" content="Kingsbury DW, Portner A. On the genesis of incomplete Sendai virions. Virology. 1970;42(4):872–9. Epub 1970/12/01. PubMed PMID:4321309." />
<meta name="citation_reference" content="Yount JS, Kraus TA, Horvath CM, Moran TM, Lopez CB. A novel role for viral-defective interfering particles in enhancing dendritic cell maturation. J Immunol. 2006;177(7):4503–13. PubMed PMID:16982887." />
<meta name="citation_reference" content="Sun Y, López CB. Preparation of Respiratory Syncytial Virus with High or Low Content of Defective Viral Particles and Their Purification from Viral Stocks. Bio-protocol. 2016;6(10). Epub 5/20/2016. doi:0.21769/BioProtoc.1820." />
<meta name="citation_reference" content="Hotard AL, Shaikh FY, Lee S, Yan D, Teng MN, Plemper RK, et al. A stabilized respiratory syncytial virus reverse genetics system amenable to recombination-mediated mutagenesis. Virology. 2012;434(1):129–36. doi:10.1016/j.virol.2012.09.022. PubMed PMID:23062737; PubMed Central PMCID:PMCPMC3492879." />
<meta name="citation_reference" content="Zhao L, Jha BK, Wu A, Elliott R, Ziebuhr J, Gorbalenya AE, et al. Antagonism of the interferoninduced OAS-RNase L pathway by murine coronavirus ns2 protein is required for virus replication and liver pathology. Cell Host Microbe. 2012;11(6):607–16. doi:10.1016/j.chom.2012.04.011. PubMed PMID:22704621; PubMed Central PMCID:PMCPMC3377938." />
<meta name="citation_reference" content="Xu J, Sun Y, Li Y, Ruthel G, Weiss SR, Raj A, et al. Replication defective viral genomes exploit a cellular pro-survival mechanism to establish paramyxovirus persistence. Nat Commun. 2017;8(1):799. doi:10.1038/s41467-017-00909-6. PubMed PMID:28986577; PubMed Central PMCID:PMCPMC5630589." />
<meta name="citation_reference" content="Wickham H. ggplot2: Elegant Graphics for Data Analysis. New York: Springer-Verlag; 2016." />
<meta name="citation_reference" content="Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9(4):357–9. doi:10.1038/nmeth.1923. PubMed PMID:22388286; PubMed Central PMCID:PMCPMC3322381." />
<meta name="twitter:title" content="A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Defective viral genomes of the copy-back type (cbDVGs) are the primary initiators of the antiviral immune response during infection with respiratory syncytial virus (RSV) both in vitro and in vivo. However, the mechanism governing cbDVG generation remains unknown, thereby limiting our ability to manipulate cbDVG content in order to modulate the host response to infection. Here we report a specific genomic signal that mediates the generation of RSV cbDVGs. Using a customized bioinformatics tool, we identified regions in the RSV genome frequently used to generate cbDVGs during infection. We then created a minigenome system to validate the function of one of these sequences and to determine if specific nucleotides were essential for cbDVG generation at that position. Further, we created a recombinant virus that selectively produced a specific cbDVG based on variations introduced in this sequence. The identified sequence was also found as a common site for cbDVG generation during natural RSV infections, and common cbDVGs generated at this sequence were found among samples from various infected patients. These data demonstrate that sequences encoded in the viral genome are critical determinants of the location of cbDVG generation and, therefore, this is not a stochastic process. Most importantly, these findings open the possibility of genetically manipulating cbDVG formation to modulate infection outcome.

Author summary Copy-back defective viral genomes (cbDVGs) regulate infection and pathogenesis of Mononegavirales. cbDVG are believed to arise from random errors that occur during virus replication and the predominant hypothesis is that the viral polymerase is the main driver of cbDVG generation. Here we describe a specific genomic sequence in the RSV genome that is necessary for the generation of a large proportion of the cbDVG population present during infection. We identified specific nucleotides that when modified altered cbDVG generation at this position, and we created a recombinant virus that selectively produced cbDVGs based on mutations in this sequence. These data demonstrate that the generation of RSV cbDVGs is regulated by specific viral sequences and that these sequences can be manipulated to alter the content and quality of cbDVG generated during infection." />
<meta name="og-title" property="og:title" content="A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/349001v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Defective viral genomes of the copy-back type (cbDVGs) are the primary initiators of the antiviral immune response during infection with respiratory syncytial virus (RSV) both in vitro and in vivo. However, the mechanism governing cbDVG generation remains unknown, thereby limiting our ability to manipulate cbDVG content in order to modulate the host response to infection. Here we report a specific genomic signal that mediates the generation of RSV cbDVGs. Using a customized bioinformatics tool, we identified regions in the RSV genome frequently used to generate cbDVGs during infection. We then created a minigenome system to validate the function of one of these sequences and to determine if specific nucleotides were essential for cbDVG generation at that position. Further, we created a recombinant virus that selectively produced a specific cbDVG based on variations introduced in this sequence. The identified sequence was also found as a common site for cbDVG generation during natural RSV infections, and common cbDVGs generated at this sequence were found among samples from various infected patients. These data demonstrate that sequences encoded in the viral genome are critical determinants of the location of cbDVG generation and, therefore, this is not a stochastic process. Most importantly, these findings open the possibility of genetically manipulating cbDVG formation to modulate infection outcome.

Author summary Copy-back defective viral genomes (cbDVGs) regulate infection and pathogenesis of Mononegavirales. cbDVG are believed to arise from random errors that occur during virus replication and the predominant hypothesis is that the viral polymerase is the main driver of cbDVG generation. Here we describe a specific genomic sequence in the RSV genome that is necessary for the generation of a large proportion of the cbDVG population present during infection. We identified specific nucleotides that when modified altered cbDVG generation at this position, and we created a recombinant virus that selectively produced cbDVGs based on mutations in this sequence. These data demonstrate that the generation of RSV cbDVGs is regulated by specific viral sequences and that these sequences can be manipulated to alter the content and quality of cbDVG generated during infection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-02-15" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/349001v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/349001v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/349001v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/349001v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/176341" />
    <title>A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"F0Qwi5xRZCQ32vf8eWFwPyqsANPBOFiqwYcEnIok_Zk","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"176341","apath":"\/biorxiv\/early\/2019\/02\/15\/349001.atom","pisa":"biorxiv;349001v3","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;349001\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;349001\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;349001v3"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;349001\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;349001\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/349001v3","title":"A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes","identifier":"node\/176341"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VsHD+0U37DwKEp4kUGKtMAe9SE8rEmW6ssxPgC8MIhptupMqB4bISBwO4KeJWXnpT\\\\\\\/ZAUKugtCdvW3C62Onyl43HFWOLzOxp5K7bC+99sqIwNop63sz5LHmE02fNRym7pU931B\\\\\\\/lcZNji0aoct4aoNCqclnBnzdfDnwvlidsGakZkvGa+gmgBerlmZIghavwIt5J5sSf1s2AaZvdfHErD5lKS7stb\\\\\\\/ZwBK8ZUrYkbQ00EdHkCX5kQvWcIhXgC6sYjNKg5KzkBz4YdLaGsW3mTFgfcO4HeC+Kxx9OpSIy6r1qTwnAM0oMkaB7azinEmlyH0rn9KqqO3zHCaqpMuHcuyI7awqCugVs8fwuu\\\\\\\/PIR+tH+MALwBwVtmkA3JLI7wcJej1cydruq3b+lAV2srFN2NO5jSFGBAkcV1dJoIxC0Mys13D4ixf\\\\\\\/o+sikp2Vot0WM45WS9EeTvDkbTZ\\\\\\\/leNF6WhZTrVyX9tGVIMlssvuNB+m3lGiTYd4iPNG7uJ3dQ6QznG+rsulVdMIZTZ6UjB9K0KvCFSP7WTMAtOjmMmgrfmwJKxh\\\\\\\/HIHl4BQF9RLwxepGXfiMl3dbwFEur05jqyoPFmdK5tvPdIUkQwY7ao14nTnu3wmu0AYFGdQCnJfSdYuPq\\\\\\\/nDKnV0vvt+hrItGt0B6nX45cLJNI1Lcl+OFvj\\\\\\\/pGGR+JsPxwXHeQEWmpxQrc2ea5Su8Uzs6uh5euEh8CJ2oNtzyizuyvJcAqn\\\\\\\/RaZMDQ7zt0Ho7poMtA5WOXNi03\\\\\\\/3Op2dpl0wfJzNSDLosQb5tG0iMhB9toMfzuYaNTqeMwEloelS8o+LOYJ\\\\\\\/VvBiXc5Dkp+wO0cW7RZU9M2PYwf+dkgKYCo8JTcRX32OIRVCdA8K4S2hihLG0II+LceIxtEg9sjQKh\\\\\\\/K6zXwVIpXEi\\\\\\\/T9NH+68\\\\\\\/ISUXWmAq3xLRqGApuB7ZIqKahyBPEUiim3v2h02tIg44DlOhr9xlxLdl9Di+WqXbjl1xn+Hxrkx5IzK8PC7TfJ2dPpnKmu9o1+i80PdRio8NBJZxetyJ3EuIggroviZ7JMWSERMx2ITTSVZ65d5lE9Hr3Rjjjsh72MIR\\\\\\\/I9nV3zxjU7Bjsl5ZYKhSKzToAcv\\\\\\\/4HqDDd2zHhD\\\\\\\/hpY7GmSCjJ9lu7hUUn1u+GItFp9jJWkrAZJYd6YXMHWYsQfHDMkTSUTRrDJ+yY0zOvIwO95LNE0aIJFLlgQKopS1O48cF0Od4DenjeeBnh1D+x+uU0rP1aA+oyXx3hhdrKxzUWZvDU5DD2\\\\\\\/lQ2oS2tv5s+wefuBvrE2StlJs6lfprPHzzMmri36h6\\\\\\\/TTe6VziE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022N6fyE3SlUxreZtc5Nm6mEQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e1110548c02b54be9114739ba748669dc0f9b573fd83fb81f9a446100b9725a2\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/349001v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-176341 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/349001v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fFwmiHS0syQj8pnywwgrSE1MOBKDXwYeyy18v6rVp2s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="176341" id="node176341" data-pisa="biorxiv;349001v3" data-pisa-master="biorxiv;349001" data-apath="/biorxiv/early/2019/02/15/349001.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yan</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Eun Ji</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Sébastien A.</span> <span class="nlm-surname">Felt</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Louis J.</span> <span class="nlm-surname">Taylor</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Divyansh</span> <span class="nlm-surname">Agarwal</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gregory R.</span> <span class="nlm-surname">Grant</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Carolina B.</span> <span class="nlm-surname">López</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/349001 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node176341" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yan Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Pathobiology, School of Veterinary Medicine, University of Pennsylvania</span>, Philadelphia, PA 19104, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yan%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYan%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eun Ji Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania</span>, Philadelphia, PA 19104, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eun%2BJi%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEun%2BJi%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sébastien A. Felt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Pathobiology, School of Veterinary Medicine, University of Pennsylvania</span>, Philadelphia, PA 19104, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9bastien%2BA.%2BFelt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Felt%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS%25C3%25A9bastien%2BA.%2BFelt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Louis J. Taylor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Microbiology, Perelman School of Medicine, University of Pennsylvania</span>, Philadelphia, PA 19104, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BJ.%2BTaylor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taylor%20LJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis%2BJ.%2BTaylor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Divyansh Agarwal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Pathobiology, School of Veterinary Medicine, University of Pennsylvania</span>, Philadelphia, PA 19104, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Divyansh%2BAgarwal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Agarwal%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADivyansh%2BAgarwal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gregory R. Grant </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania</span>, Philadelphia, PA 19104, <span class='nlm-country'>United States</span>.</div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Genetics, Perelman School of Medicine, University of Pennsylvania</span>, Philadelphia, PA 19104, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BR.%2BGrant%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grant%20GR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGregory%2BR.%2BGrant%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Carolina B. López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Pathobiology, School of Veterinary Medicine, University of Pennsylvania</span>, Philadelphia, PA 19104, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BB.%2BL%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=L%C3%B3pez%20CB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarolina%2BB.%2BL%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lopezca@upenn.edu" class="" data-icon-position="" data-hide-link-title="0">lopezca@upenn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/349001v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:176341" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:176341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/349001v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:176341" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:176341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/349001v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:176341" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:176341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/349001v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:176341" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:176341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/349001v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:176341" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:176341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/349001v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:176341" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:176341/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Defective viral genomes of the copy-back type (cbDVGs) are the primary initiators of the antiviral immune response during infection with respiratory syncytial virus (RSV) both <em>in vitro</em> and <em>in vivo.</em> However, the mechanism governing cbDVG generation remains unknown, thereby limiting our ability to manipulate cbDVG content in order to modulate the host response to infection. Here we report a specific genomic signal that mediates the generation of RSV cbDVGs. Using a customized bioinformatics tool, we identified regions in the RSV genome frequently used to generate cbDVGs during infection. We then created a minigenome system to validate the function of one of these sequences and to determine if specific nucleotides were essential for cbDVG generation at that position. Further, we created a recombinant virus that selectively produced a specific cbDVG based on variations introduced in this sequence. The identified sequence was also found as a common site for cbDVG generation during natural RSV infections, and common cbDVGs generated at this sequence were found among samples from various infected patients. These data demonstrate that sequences encoded in the viral genome are critical determinants of the location of cbDVG generation and, therefore, this is not a stochastic process. Most importantly, these findings open the possibility of genetically manipulating cbDVG formation to modulate infection outcome.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>Author summary</strong> Copy-back defective viral genomes (cbDVGs) regulate infection and pathogenesis of <em>Mononegavirales.</em> cbDVG are believed to arise from random errors that occur during virus replication and the predominant hypothesis is that the viral polymerase is the main driver of cbDVG generation. Here we describe a specific genomic sequence in the RSV genome that is necessary for the generation of a large proportion of the cbDVG population present during infection. We identified specific nucleotides that when modified altered cbDVG generation at this position, and we created a recombinant virus that selectively produced cbDVGs based on mutations in this sequence. These data demonstrate that the generation of RSV cbDVGs is regulated by specific viral sequences and that these sequences can be manipulated to alter the content and quality of cbDVG generated during infection.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-4">Defective viral genomes (DVGs), which are generated during the replication of most RNA viruses, potentiate the host innate immune response [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>] and attenuate the infection <em>in vitro</em> and <em>in vivo</em> [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>-<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Importantly, in naturally infected humans, the presence of DVGs correlates with enhanced antiviral immune responses during RSV infection [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>] and reduced disease severity in influenza virus infection [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Significant effort is currently invested in harnessing DVGs as antivirals due to their strong immunostimulatory activity and ability to interfere with the replication of the standard virus. However, despite over 50 years of appreciating their critical functions in multiple aspects of viral infections, the molecular mechanisms that drive DVG generation remain largely unknown. This lack of understanding hampers our ability to effectively harness DVGs for therapeutic purposes and limits our capacity to generate tools to elucidate their mechanism of action and impact during specific viral infections.</p><p id="p-5">There are two major types of DVGs: deletion and copy-back (cb) [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Both types are unable to complete a full replication cycle without the help of a co-infecting full-length virus [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>] and can be packaged to become part of the viral population [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Deletion DVGs, common in influenza virus and positive strand RNA viruses, retain the 3’ and 5’ ends of the viral genomes but carry an internal deletion [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>-<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. These types of DVGs are believed to arise from recombination events [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>] and can have a strong interfering activity over the standard virus [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. cbDVGs are common products of negative sense (ns)RNA virus replication, including Sendai virus, measles virus, and respiratory syncytial virus (RSV), and are the primary stimulators of the innate immune response during nsRNA virus infection [<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>], cbDVGs arise when the viral polymerase detaches from its template at a “break point” and resumes elongation at a downstream “rejoin point” by copying the 5’ end of the nascent daughter strand [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], This process results in the formation of a new junction sequence and a truncated genome flanked by reverse complementary ends [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. cbDVGs have long been thought to result from errors made by the RNA-dependent RNA polymerase (RdRps) during replication due to a combination of lack of proofreading activity and the presence of a polymerase with lower replication fidelity [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>], No pattern or specific sequences for the break and rejoin points of cdDVGs have been reported so far.</p><p id="p-6">Based on our consistent observations of discrete populations of cbDVG generated during RSV infections <em>in vitro</em> and <em>in vivo</em> [<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a>], we set out to test the hypothesis that the generation of cbDVGs is not completely stochastic but instead is regulated by a carefully orchestrated process. Upon identification of cbDVG populations generated during infection, we show that specific viral sequences within the viral genome are preferred sites for cbDVG generation and that these sequences are conserved across viral strains. Utilizing this knowledge, we generated a recombinant virus that produced a restricted set of cbDVGs, serving as strong evidence that specific sequences dictate where cbDVGs are generated. In addition, we demonstrate for the first time that common cbDVGs are generated independently in natural infections in humans, further supporting an orchestrated origin for cbDVGs.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>High sensitivity detection of cbDVG populations during infection</h3><p id="p-7">To acquire a comprehensive view of the population of cbDVGs generated during infection, we developed an algorithm to identify cbDVGs junction regions within RNA-seq datasets with high sensitivity. The principle of this Viral Opensource DVG Key Algorithm (VODKA) is illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1A</a>. In brief, the break point (T’) and the rejoin point (W) are far apart in the parental viral genome, but in cbDVGs they become continuous and form the new cbDVG junction sequence when the viral polymerase (Vp) is released from the template and rejoins in the nascent strand. Since cbDVG junction sequences are absent in the full-length viral genome, VODKA identifies the sequence reads that capture junction sequences and then filters out false-positives reads that fully align to the reference genome.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="VODKA identified hotspots for polymerase break and rejoin during cbDVG formation. (A) Schematic representation of cbDVG generation from negative strand viruses and general principle behind VODKA identification ofcbDVGs. The red dashed square marks the unique junction region that distinguishes DVGs from full-length viral genomes. This junction region was used by VODKA to identify cbDVGs. (B) A549 cells were mock infected or infected with SeV Cantell at an MOI of 1.5 TCID50/cell and harvested at 2, 6, 12, and 24h post infection followed by detection of cbDVGs by RT-PCR using primers SeV Dll and gSeV Dll (SI Table). (C) Alignment of SeV cbDVG junction reads obtained from VODKA to the last 3kb nucleotides of the SeV antisense genome (top) or the DVG-546junction sequence (bottom). Blue histogram shows a synopsis of total coverage at any given position of the last 3Kb of the SeV reference genome (top) or the DVG-546 junction sequence (bottom). The numbers on the right-side axis of the graphs represent the total number of reads at the position with the highest coverage. Colored nucleotides beneath the blue histogram represent the consensus sequences of cbDVG junction identified by VODKA. The size of the letter represents the degree of conservation at the site and if there were mutations at a certain position the most representative nucleotide was listed on top. Black nucleotides beneath colored nucleotides indicate the sequence of the DVG-546junction region obtained by Sanger sequencing. The grey boxed areas in the graphs mark the region after the rejoin point and the pink boxed area the DVG region before the break point. (D) A549 cells were infected with RSV stocks1-7. For stocks1-6, RNA was extracted from the supernatants of the infected cells at 48 h post infections, followed by RNA-seq and VODKA screening. D VG junction reads were then mapped to the antigenome of the RSVA2 reference strain. Blue histogram shows synopsis of total coverage at any given position of the last 3kb of the RSV reference genome. The number on the right side of the graph represents the total reads at the position with highest coverage. Based on the VODKA output, individual peaks were identified as break or rejoin regions indicated by red or black facing-down arrows, respectively. For stock7, RNA was extracted from infected cells at 24 h post infections, followed by DVG specific RT-PCR using primer Dl1 and Dl-R (see SI Table) that captures cbDVGs larger than 453 bp and with the break after Dl1 and the rejoin after Dl-R. Based on conventional cloning and Sanger sequencing ofPCR products, the break and rejoin points of cbDVGs detected in this stock were marked by red and black facing upwards arrows, respectively. Dashed lines indicate join sequences from individual cbDVGs in stock7. Light and dark grey shades represented the rejoin and break regions that were cloned into RSV minigenome backbone for further analysis in Fig 2A. Reference genome is shown at the bottom of the figure and the corresponding positions of DVG break/rejoin regions from all stocks, Break1, Rejoin1, and Trailer are also indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1210250752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;VODKA identified hotspots for polymerase break and rejoin during cbDVG formation.&lt;/span&gt; (A) Schematic representation of cbDVG generation from negative strand viruses and general principle behind VODKA identification ofcbDVGs. The red dashed square marks the unique junction region that distinguishes DVGs from full-length viral genomes. This junction region was used by VODKA to identify cbDVGs. (B) A549 cells were mock infected or infected with SeV Cantell at an MOI of 1.5 TCID&lt;sub&gt;50&lt;/sub&gt;/cell and harvested at 2, 6, 12, and 24h post infection followed by detection of cbDVGs by RT-PCR using primers SeV Dll and gSeV Dll (SI Table). (C) Alignment of SeV cbDVG junction reads obtained from VODKA to the last 3kb nucleotides of the SeV antisense genome (top) or the DVG-546junction sequence (bottom). Blue histogram shows a synopsis of total coverage at any given position of the last 3Kb of the SeV reference genome (top) or the DVG-546 junction sequence (bottom). The numbers on the right-side axis of the graphs represent the total number of reads at the position with the highest coverage. Colored nucleotides beneath the blue histogram represent the consensus sequences of cbDVG junction identified by VODKA. The size of the letter represents the degree of conservation at the site and if there were mutations at a certain position the most representative nucleotide was listed on top. Black nucleotides beneath colored nucleotides indicate the sequence of the DVG-546junction region obtained by Sanger sequencing. The grey boxed areas in the graphs mark the region after the rejoin point and the pink boxed area the DVG region before the break point. (D) A549 cells were infected with RSV stocks1-7. For stocks1-6, RNA was extracted from the supernatants of the infected cells at 48 h post infections, followed by RNA-seq and VODKA screening. D VG junction reads were then mapped to the antigenome of the RSVA2 reference strain. Blue histogram shows synopsis of total coverage at any given position of the last 3kb of the RSV reference genome. The number on the right side of the graph represents the total reads at the position with highest coverage. Based on the VODKA output, individual peaks were identified as break or rejoin regions indicated by red or black facing-down arrows, respectively. For stock7, RNA was extracted from infected cells at 24 h post infections, followed by DVG specific RT-PCR using primer Dl1 and Dl-R (see SI Table) that captures cbDVGs larger than 453 bp and with the break after Dl1 and the rejoin after Dl-R. Based on conventional cloning and Sanger sequencing ofPCR products, the break and rejoin points of cbDVGs detected in this stock were marked by red and black facing upwards arrows, respectively. Dashed lines indicate join sequences from individual cbDVGs in stock7. Light and dark grey shades represented the rejoin and break regions that were cloned into RSV minigenome backbone for further analysis in Fig 2A. Reference genome is shown at the bottom of the figure and the corresponding positions of DVG break/rejoin regions from all stocks, Break1, Rejoin1, and Trailer are also indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F1.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F1.medium.gif" width="440" height="428"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig 1.</span> <span class="caption-title">VODKA identified hotspots for polymerase break and rejoin during cbDVG formation.</span><p id="p-8" class="first-child">(A) Schematic representation of cbDVG generation from negative strand viruses and general principle behind VODKA identification ofcbDVGs. The red dashed square marks the unique junction region that distinguishes DVGs from full-length viral genomes. This junction region was used by VODKA to identify cbDVGs. (B) A549 cells were mock infected or infected with SeV Cantell at an MOI of 1.5 TCID<sub>50</sub>/cell and harvested at 2, 6, 12, and 24h post infection followed by detection of cbDVGs by RT-PCR using primers SeV Dll and gSeV Dll (SI Table). (C) Alignment of SeV cbDVG junction reads obtained from VODKA to the last 3kb nucleotides of the SeV antisense genome (top) or the DVG-546junction sequence (bottom). Blue histogram shows a synopsis of total coverage at any given position of the last 3Kb of the SeV reference genome (top) or the DVG-546 junction sequence (bottom). The numbers on the right-side axis of the graphs represent the total number of reads at the position with the highest coverage. Colored nucleotides beneath the blue histogram represent the consensus sequences of cbDVG junction identified by VODKA. The size of the letter represents the degree of conservation at the site and if there were mutations at a certain position the most representative nucleotide was listed on top. Black nucleotides beneath colored nucleotides indicate the sequence of the DVG-546junction region obtained by Sanger sequencing. The grey boxed areas in the graphs mark the region after the rejoin point and the pink boxed area the DVG region before the break point. (D) A549 cells were infected with RSV stocks1-7. For stocks1-6, RNA was extracted from the supernatants of the infected cells at 48 h post infections, followed by RNA-seq and VODKA screening. D VG junction reads were then mapped to the antigenome of the RSVA2 reference strain. Blue histogram shows synopsis of total coverage at any given position of the last 3kb of the RSV reference genome. The number on the right side of the graph represents the total reads at the position with highest coverage. Based on the VODKA output, individual peaks were identified as break or rejoin regions indicated by red or black facing-down arrows, respectively. For stock7, RNA was extracted from infected cells at 24 h post infections, followed by DVG specific RT-PCR using primer Dl1 and Dl-R (see SI Table) that captures cbDVGs larger than 453 bp and with the break after Dl1 and the rejoin after Dl-R. Based on conventional cloning and Sanger sequencing ofPCR products, the break and rejoin points of cbDVGs detected in this stock were marked by red and black facing upwards arrows, respectively. Dashed lines indicate join sequences from individual cbDVGs in stock7. Light and dark grey shades represented the rejoin and break regions that were cloned into RSV minigenome backbone for further analysis in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A</a>. Reference genome is shown at the bottom of the figure and the corresponding positions of DVG break/rejoin regions from all stocks, Break1, Rejoin1, and Trailer are also indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">We corroborated VODKA’s performance by testing for the presence of the highly dominant DVG-546 in samples from infections with SeV Cantell (Fig IB). Briefly, 98,543 out of the 98,626 (99.9%) cbDVG junction reads identified by VODKA from a RNA-seq data set obtained from SeV Cantell-infected cells mapped exactly to the known junction region of DVG-546 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1C</a>, bottom panel). By aligning the cbDVG junction reads to the SeV full-length antisense genome, we determined the location of the major break and rejoin regions (blue peaks in the upper panel of <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig 1C</a>). Each read that aligned to a break or rejoin region contained two portions, one of which fully aligned to the reference genome. The aligned reads in the break region (pink box, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1C</a>) and in the rejoin region (gray box, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig 1C</a>) corresponded to the DVG-546 sequence before the break point and after the rejoin point, respectively. The breakpoint for DVG-546 predicted by VODKA (14932±1_15292±1) exactly matched the one identified by Sanger sequencing (<strong>↑</strong> in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig 1C</a>), thereby establishing the efficiency and accuracy of VODKA in identifying cbDVG-specific sequences.</p><p id="p-10">We then used VODKA to identify the population of cbDVGs generated during RSV infection. RSV is a virus known to generate immunostimulatory cbDVGs in infected patients [<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">6</a>], and thus a subject of interest in our laboratory. We analyzed pooled RNA-seq datasets from six RSV-infected cell cultures. These cultures were infected with RSV generated from the same parental stock (strain A2) that was first depleted of DVGs and then passaged independently in different cell lines to generate six different stocks enriched for DVGs. The presence of cbDVGs in these infections was confirmed using a specific RT-PCR followed by Sanger sequencing. By aligning VODKA-identified cbDVG junctions to the RSV A2 reference antigenome, we observed 4 major break hotspots spanning over 1300 nucleotides (red down-facing arrows in Fig ID). In contrast, only 2 major rejoin hotspots were observed within a narrower region of 223 nucleotides in length at the 3’end of the viral genome (black down-facing arrows in Fig ID). Remarkably, the rejoin area with the highest peak included counts present in all six virus stocks. We then compared these break and rejoin hotspots to those generated in infections with a different stock of RSV enriched in cbDVGs (stock7) from which the major cbDVGs were identified upon Sanger sequencing of PCR amplicons. We observed that the cbDVG rejoin points from stock7 were located within the strongest rejoin hotspot, whereas its four break points were distributed more broadly across the genome (Fig ID). These results reveal strong hotspots for the polymerase rejoin during cbDVG formation and suggest a large degree of conservation of the RSV cbDVGs rejoin positions.</p></div><div id="sec-5" class="subsection"><h3>Identification of specific RSV genomic regions responsible for cbDVG generation</h3><p id="p-11">To determine if candidate hotspots were involved in cbDVG generation, we selected the region containing most break points (Break1, dark grey in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig 1</a>), or most rejoin points (Rejoin1+Trailer, light grey in Fig ID) for further testing using a minigenome system [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>], We constructed a RSV minigenome backbone (BKB) that included the reporter gene mKate2 for flow cytometry quantification of transcripts produced by the viral polymerase. In addition, the minigenome BKB included restriction enzyme sites to insert the selected Break1 and/or Rejoin1 regions (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2A</a>). The goal was to use this system to establish whether sequences in the candidate break and rejoin regions altered the polymerase elongation capacity, eventually leading to the generation of cbDVGs. As illustrated in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2B</a>, in this system mKate2 expression should only occur if the viral polymerase replicates the entire minigenome sequence from the trailer to the leader. Co-transfection of the minigenome construct with the four helper plasmids expressing the polymerase proteins (L, P, NP, and M2-1), resulted in mKate2 expression in 8-17% of the cells, whereas no mKate2 expression was detected in control transfections that lacked the viral polymerase (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2C</a>; -Vp). Constructs containing only the Rejoin1 sequence led to similar mKate2 expression as the BKB construct, whereas constructs containing Break1 caused a ∼30% reduction in mKate2 expression (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig 2D-2E</a>). We verified that the difference in mKate2 expression among transfections with different constructs was not due to variable transfection efficiency (S1A-S1C Fig), or cell death (S1D Fig). These results are consistent with the concept that during cbDVG generation, the viral polymerase falls off the template at the break region leading to a reduced amount of newly synthetized template available for mKate2 transcription.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Specific RSV genomic regions alter the viral polymerase elongation capacity. (A) Schematic representation of the RSV minigenome backbone construct. Going from 5&#x2019; to 3&#x2019;: T7 promoter, hammerhead ribozyme, complementary sense RSV leader sequence (LeC), NS1 gene start sequence, NS1 non-coding sequence, gene encoding mKate2 [25], L gene start sequence (GS), L gene end sequence (GE), restriction enzyme sites, complementary sense trailer sequence (TrC), hepatitis delta virus ribozyme, and T7 terminator. All constructs were generated in a pSH180 vector and included three restriction enzyme sites that were used for ligating Break1 and Rejoin1 as indicated. The length of all minigenome constructs after T7 promoter and before T7 terminator obeyed the rule of six. Red arrows indicate the primers that were used for detecting cbDVGs by PCR. One primer was within the viral trailer and the other primer was designed against the 3&#x2019; end of the break region. (B) Workflow of the RSV minigenome system. BSR-T7 cells were transfected with expression vectors encoding the polymerase genes M2-1, L, NP, P, and the minigenome construct to be tested. The minigenome was then transcribed by the T7 polymerase generating a template for amplification by the viral polymerase. If the paired junction region was not functional, the viral polymerase continued to elongate from 3&#x2019; TrC to the 5&#x2019; LeC, exposing the promoter in Lefor transcription leading to mKate2 expression. If break and rejoin sequences were functional, the viral polymerase recognized those signals resulting in dropping off from the template, generation of cbDVG-like fragments, and reduction ofmKate2 expression. Since DVG junctions are split into two non-continuous regions (break and rejoin) in the minigenome plasmid, detection of the end-to-end DVG junction regions via DI-RT-PCR is a second indication of a productive junction region. (C) BSR-T7 cells were transfected with BKB either with or without the polymerase expression plasmids. One representative plot ofmKate2 expression determined by flow cytometry is shown. (D-E) BSR-T7 cells were co-transfected with all 4 helper plasmids as well as either a BKB construct or a construct including the candidate signal sequences. mKate2 expression was measured by flow cytometry. The quantification of all repeats is shown as a percentage in D (F test P=0.0028). Representative flow plots are shown in E. *p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1210250752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Specific RSV genomic regions alter the viral polymerase elongation capacity.&lt;/span&gt; (A) Schematic representation of the RSV minigenome backbone construct. Going from 5&#x2019; to 3&#x2019;: T7 promoter, hammerhead ribozyme, complementary sense RSV leader sequence (LeC), NS1 gene start sequence, NS1 non-coding sequence, gene encoding mKate2 [25], L gene start sequence (GS), L gene end sequence (GE), restriction enzyme sites, complementary sense trailer sequence (TrC), hepatitis delta virus ribozyme, and T7 terminator. All constructs were generated in a pSH180 vector and included three restriction enzyme sites that were used for ligating Break1 and Rejoin1 as indicated. The length of all minigenome constructs after T7 promoter and before T7 terminator obeyed the rule of six. Red arrows indicate the primers that were used for detecting cbDVGs by PCR. One primer was within the viral trailer and the other primer was designed against the 3&#x2019; end of the break region. (B) Workflow of the RSV minigenome system. BSR-T7 cells were transfected with expression vectors encoding the polymerase genes M2-1, L, NP, P, and the minigenome construct to be tested. The minigenome was then transcribed by the T7 polymerase generating a template for amplification by the viral polymerase. If the paired junction region was not functional, the viral polymerase continued to elongate from 3&#x2019; TrC to the 5&#x2019; LeC, exposing the promoter in Lefor transcription leading to mKate2 expression. If break and rejoin sequences were functional, the viral polymerase recognized those signals resulting in dropping off from the template, generation of cbDVG-like fragments, and reduction ofmKate2 expression. Since DVG junctions are split into two non-continuous regions (break and rejoin) in the minigenome plasmid, detection of the end-to-end DVG junction regions via DI-RT-PCR is a second indication of a productive junction region. (C) BSR-T7 cells were transfected with BKB either with or without the polymerase expression plasmids. One representative plot ofmKate2 expression determined by flow cytometry is shown. (D-E) BSR-T7 cells were co-transfected with all 4 helper plasmids as well as either a BKB construct or a construct including the candidate signal sequences. mKate2 expression was measured by flow cytometry. The quantification of all repeats is shown as a percentage in D (F test P=0.0028). Representative flow plots are shown in E. *p&lt;0.05 by one-way ANOVA Bonferroni&#x2019;s multiple comparison test, n=8, mean&#xB1;EM, Bartlett&#x2019;s test is used for examination of equal variance. Flow gating strategy is shown in S1A Fig.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F2.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F2.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 2.</span> <span class="caption-title">Specific RSV genomic regions alter the viral polymerase elongation capacity.</span><p id="p-12" class="first-child">(A) Schematic representation of the RSV minigenome backbone construct. Going from 5’ to 3’: T7 promoter, hammerhead ribozyme, complementary sense RSV leader sequence (LeC), NS1 gene start sequence, NS1 non-coding sequence, gene encoding mKate2 [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>], L gene start sequence (GS), L gene end sequence (GE), restriction enzyme sites, complementary sense trailer sequence (TrC), hepatitis delta virus ribozyme, and T7 terminator. All constructs were generated in a pSH180 vector and included three restriction enzyme sites that were used for ligating Break1 and Rejoin1 as indicated. The length of all minigenome constructs after T7 promoter and before T7 terminator obeyed the rule of six. Red arrows indicate the primers that were used for detecting cbDVGs by PCR. One primer was within the viral trailer and the other primer was designed against the 3’ end of the break region. (B) Workflow of the RSV minigenome system. BSR-T7 cells were transfected with expression vectors encoding the polymerase genes M2-1, L, NP, P, and the minigenome construct to be tested. The minigenome was then transcribed by the T7 polymerase generating a template for amplification by the viral polymerase. If the paired junction region was not functional, the viral polymerase continued to elongate from 3’ TrC to the 5’ LeC, exposing the promoter in Lefor transcription leading to mKate2 expression. If break and rejoin sequences were functional, the viral polymerase recognized those signals resulting in dropping off from the template, generation of cbDVG-like fragments, and reduction ofmKate2 expression. Since DVG junctions are split into two non-continuous regions (break and rejoin) in the minigenome plasmid, detection of the end-to-end DVG junction regions via DI-RT-PCR is a second indication of a productive junction region. (C) BSR-T7 cells were transfected with BKB either with or without the polymerase expression plasmids. One representative plot ofmKate2 expression determined by flow cytometry is shown. (D-E) BSR-T7 cells were co-transfected with all 4 helper plasmids as well as either a BKB construct or a construct including the candidate signal sequences. mKate2 expression was measured by flow cytometry. The quantification of all repeats is shown as a percentage in D (F test P=0.0028). Representative flow plots are shown in E. *p&lt;0.05 by one-way ANOVA Bonferroni’s multiple comparison test, n=8, mean±EM, Bartlett’s test is used for examination of equal variance. Flow gating strategy is shown in S1A Fig.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">To formally assess whether candidate break and rejoin sequences lead to cbDVG formation, we cloned the designated Pair1 composed of Break1/Rejoin1 into the minigenome system. Upon transfection, we observed that the construct containing Pair1 led to a similar degree of mKate2 expression than the construct bearing Break1 alone (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3A-B</a>). We also observed two major amplicons (white arrows in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3C</a>), both of which were absent in cells transfected with the construct bearing Break1 alone. These two amplicons contained cbDVGs that were confirmed by conventional Sanger sequencing (S2A Fig). The individual break and rejoin points of these minigenome-generated cbDVGs are indicated in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3D</a>. Interestingly, the rejoin points clustered in close proximity to the rejoin points that we identified from <em>in vitro</em> infected cells. Taken together, these data demonstrate that RSV cbDVG rejoin points fall into a discrete region of the viral genome, which is critical for cbDVG generation.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of one conserved genomic region critical for cbDVG rejoin. (A-B) BSR-T7 cells were co-transfected with all 4 helper plasmids as well as BKB or constructs with candidate signal sequences. mKate2 expression was measured by flow cytometry. The quantification of all repeats is shown as a fold change expression over BKB in A (F test P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1210250752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Identification of one conserved genomic region critical for cbDVG rejoin.&lt;/span&gt; (A-B) BSR-T7 cells were co-transfected with all &lt;strong&gt;4&lt;/strong&gt; helper plasmids as well as BKB or constructs with candidate signal sequences. mKate2 expression was measured by flow cytometry. The quantification of all repeats is shown as a fold change expression over BKB in A (F test P&lt;0.0001). **p&lt;0.01, ***p&lt;0.001 by one-way ANOVA Bonferroni&#x2019;s multiple comparison test, n=8, mean&#xB1;EM, Bartlett&#x2019;s test is used for examination of equal variance. Flow gating strategy is shown in S1A Fig. Representative flow plots are shown in B. (C-D) cbDVG-like fragments formed in Pair1 were detected by RT-PCR using DI-F1 and DI-R primer sets (SI Table). Representative gel picture was shown in C. Bands labeled with a white arrow were confirmed to correspond to cbDVG by conventional sequencing. Their sequence is shown in S2A Fig. Location and the sequence length between the break and rejoin points are indicated in panel (D).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F3.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F3.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 3.</span> <span class="caption-title">Identification of one conserved genomic region critical for cbDVG rejoin.</span><p id="p-14" class="first-child">(A-B) BSR-T7 cells were co-transfected with all <strong>4</strong> helper plasmids as well as BKB or constructs with candidate signal sequences. mKate2 expression was measured by flow cytometry. The quantification of all repeats is shown as a fold change expression over BKB in A (F test P&lt;0.0001). **p&lt;0.01, ***p&lt;0.001 by one-way ANOVA Bonferroni’s multiple comparison test, n=8, mean±EM, Bartlett’s test is used for examination of equal variance. Flow gating strategy is shown in S1A Fig. Representative flow plots are shown in B. (C-D) cbDVG-like fragments formed in Pair1 were detected by RT-PCR using DI-F1 and DI-R primer sets (SI Table). Representative gel picture was shown in C. Bands labeled with a white arrow were confirmed to correspond to cbDVG by conventional sequencing. Their sequence is shown in S2A Fig. Location and the sequence length between the break and rejoin points are indicated in panel (D).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Specific nucleotide composition determines the position for cbDVGs rejoin</h3><p id="p-15">Since the late Rejoin1 + early Trailer region of the RSV genome was highly enriched with DVG rejoin points relative to other regions in the RSV genome, we then examined which specific features within this region impacted cbDVG generation. This region is within one of most A and U enriched areas of the RSV genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4A</a>), suggesting that nucleotide composition might play a role in directing cbDVG formation. To avoid affecting the L gene “end signal” and the genome trailer region, we chose to mutate six nucleotides at the beginning of this rejoin region (nucleotide positions 191-186 from the 3’ end of antigenome) to either all Us (named GC&gt;Us) or all GCs (named AU&gt;GCs). We then used RT-PCR (DI-1/DI-R primer set) to detect cbDVG-like fragments formed in the cells co-transfected with all Us or all GCs mutant constructs and polymerase-expressing plasmids, as described earlier. Mutant GC&gt;Us generated a dominant amplicon (Iane2, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4B</a>) that was absent in cells transfected with mutant AU&gt;GCs (Iane4, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig 4B</a>). From sequencing PCR products within the areas marked by asterisks in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig 4B</a>, we identified five distinct rejoin points from mutant AU&gt;GCs and three from mutant GC&gt;Us (<strong>↑</strong> in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig 4C</a>). Compared to WT Pair1, the mutant GC&gt;Us did not generate rejoin points proximal to the mutated region (grey area in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig 4C</a>), whereas the mutant AU&gt;GCs still produced cbDVGs-like fragments at the mutated area.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Specific nucleotide composition determines the position for cbDVGs rejoin. (A) AU-content (top, purple) over the entire genome and major rejoin region (inset, purple) of the RSV strain A2 antigenome. AU-content is plotted centered on a 40 base-pair sliding window in both plots. Blue ticks in the inset represent rejoin points from stock1-6 at each position starting at base 14700, with a proportionally darker color representing increased rejoin frequency at a given position. Grey box indicates the position ofRejoin1 and Trailer. (B) cbDVG-like fragments generated in mutant GC&gt;Us and mutant AU&gt;GCs were detected by RT-PCR using the DI-F1 and Dl-R primer set (S1 Table). Confirmed cbDVG-like fragments are indicated with asterisks next to the gel. (C) Schematic summary of all identified cbDVG rejoin points within positions 14998-15105 of the genome (RSVA2, reference genome NCBI KT992094.1) generated from WT Pair1 and Pair1 mutants minigenomes. This region includes gene end of L and the beginning of Trailer. Different colored arrows indicate rejoin points from cbDVGs detected by different primer sets as indicated in the figure. Primer sets are composed of the identical reverse primer, Dl-R, and one of either three different forward primers, DI-F1, DI-F2, and DI-F3. The numbers near the arrows indicated the number of times (&gt;1) the same cbDVG was observed at this location (out of three experimental repeats with 1-3 clones sequenced each time). The thick black line marks the sequence of rejoin region selected for mutation studies. The antigenomic position range of the mutations is indicated at the bottom of the figure and the actual RNA sequences at the mutation sites in WT and five mutants are indicated. The numbers on top of the WT RNA sequence indicate their positions from the 3&#x2019; end of the antigenome. Grey shade marks the area where mutants GC&gt;Us, C186U, and AU, did not have the rejoin points. (D) cbDVG fragments generated in mutant GC&gt;Us and mutant AU&gt;GCs were detected by DI-F2/DI-R and Dl-F3/DI-R through DVG specific RT-PCR. The DVG-like bands labeled by asterisks were confirmed to contain the same cbDVG-like fragment detected by DI-F1/DI-R in B (only band labeled by asterisk). (E) cbDVG fragments generated in mutants C188U, C186U, and AU were detected by DVG specific RT-PCR using the indicated primers. Confirmed cbDVG-like fragments are indicated with asterisks. Band labeled with an arrowhead was confirmed not to be a DVG product. Sequences of all the confirmed cbDVG-like fragments are shown in S2B-S2F Fig." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1210250752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Specific nucleotide composition determines the position for cbDVGs rejoin.&lt;/span&gt; (A) AU-content (top, purple) over the entire genome and major rejoin region (inset, purple) of the RSV strain A2 antigenome. AU-content is plotted centered on a 40 base-pair sliding window in both plots. Blue ticks in the inset represent rejoin points from stock1-6 at each position starting at base 14700, with a proportionally darker color representing increased rejoin frequency at a given position. Grey box indicates the position ofRejoin1 and Trailer. (B) cbDVG-like fragments generated in mutant GC&gt;Us and mutant AU&gt;GCs were detected by RT-PCR using the DI-F1 and Dl-R primer set (S1 Table). Confirmed cbDVG-like fragments are indicated with asterisks next to the gel. (C) Schematic summary of all identified cbDVG rejoin points within positions 14998-15105 of the genome (RSVA2, reference genome NCBI KT992094.1) generated from WT Pair1 and Pair1 mutants minigenomes. This region includes gene end of L and the beginning of Trailer. Different colored arrows indicate rejoin points from cbDVGs detected by different primer sets as indicated in the figure. Primer sets are composed of the identical reverse primer, Dl-R, and one of either three different forward primers, DI-F1, DI-F2, and DI-F3. The numbers near the arrows indicated the number of times (&gt;1) the same cbDVG was observed at this location (out of three experimental repeats with 1-3 clones sequenced each time). The thick black line marks the sequence of rejoin region selected for mutation studies. The antigenomic position range of the mutations is indicated at the bottom of the figure and the actual RNA sequences at the mutation sites in WT and five mutants are indicated. The numbers on top of the WT RNA sequence indicate their positions from the 3&#x2019; end of the antigenome. Grey shade marks the area where mutants GC&gt;Us, C186U, and AU, did not have the rejoin points. (D) cbDVG fragments generated in mutant GC&gt;Us and mutant AU&gt;GCs were detected by DI-F2/DI-R and Dl-F3/DI-R through DVG specific RT-PCR. The DVG-like bands labeled by asterisks were confirmed to contain the same cbDVG-like fragment detected by DI-F1/DI-R in B (only band labeled by asterisk). (E) cbDVG fragments generated in mutants C188U, C186U, and AU were detected by DVG specific RT-PCR using the indicated primers. Confirmed cbDVG-like fragments are indicated with asterisks. Band labeled with an arrowhead was confirmed not to be a DVG product. Sequences of all the confirmed cbDVG-like fragments are shown in S2B-S2F Fig.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F4.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F4.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 4.</span> <span class="caption-title">Specific nucleotide composition determines the position for cbDVGs rejoin.</span><p id="p-16" class="first-child">(A) AU-content (top, purple) over the entire genome and major rejoin region (inset, purple) of the RSV strain A2 antigenome. AU-content is plotted centered on a 40 base-pair sliding window in both plots. Blue ticks in the inset represent rejoin points from stock1-6 at each position starting at base 14700, with a proportionally darker color representing increased rejoin frequency at a given position. Grey box indicates the position ofRejoin1 and Trailer. (B) cbDVG-like fragments generated in mutant GC&gt;Us and mutant AU&gt;GCs were detected by RT-PCR using the DI-F1 and Dl-R primer set (S1 Table). Confirmed cbDVG-like fragments are indicated with asterisks next to the gel. (C) Schematic summary of all identified cbDVG rejoin points within positions 14998-15105 of the genome (RSVA2, reference genome NCBI KT992094.1) generated from WT Pair1 and Pair1 mutants minigenomes. This region includes gene end of L and the beginning of Trailer. Different colored arrows indicate rejoin points from cbDVGs detected by different primer sets as indicated in the figure. Primer sets are composed of the identical reverse primer, Dl-R, and one of either three different forward primers, DI-F1, DI-F2, and DI-F3. The numbers near the arrows indicated the number of times (&gt;1) the same cbDVG was observed at this location (out of three experimental repeats with 1-3 clones sequenced each time). The thick black line marks the sequence of rejoin region selected for mutation studies. The antigenomic position range of the mutations is indicated at the bottom of the figure and the actual RNA sequences at the mutation sites in WT and five mutants are indicated. The numbers on top of the WT RNA sequence indicate their positions from the 3’ end of the antigenome. Grey shade marks the area where mutants GC&gt;Us, C186U, and AU, did not have the rejoin points. (D) cbDVG fragments generated in mutant GC&gt;Us and mutant AU&gt;GCs were detected by DI-F2/DI-R and Dl-F3/DI-R through DVG specific RT-PCR. The DVG-like bands labeled by asterisks were confirmed to contain the same cbDVG-like fragment detected by DI-F1/DI-R in B (only band labeled by asterisk). (E) cbDVG fragments generated in mutants C188U, C186U, and AU were detected by DVG specific RT-PCR using the indicated primers. Confirmed cbDVG-like fragments are indicated with asterisks. Band labeled with an arrowhead was confirmed not to be a DVG product. Sequences of all the confirmed cbDVG-like fragments are shown in S2B-S2F Fig.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">To rule out bias due to primer location, we designed two additional forward primers to detect cbDVG-like fragments from the same samples. Rejoins detected with DI-F2 primers are identified with red arrows in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig 4C</a>, while rejoins detected with DI-F3 are indicated with blue arrows. Transfections with mutant GC&gt;Us led to one strong amplicon while not predominant amplicon was observed in transfections with mutant AU&gt;GCs (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig 4D</a>), agreeing with results obtained using the DI-F1 primer set. Sequencing confirmed that the strong amplicons produced by all three different primer sets in transfections with mutant GC&gt;Us were analogous cbDVG-like fragments and shared their break and rejoin points (sequence in S2 Fig, DVG 303bp). To examine if the observed lack of a predominant product resulting from mutant AU&gt;GCs was due to a general reduction of replication ability of the viral polymerase induced by mutations, we introduced the same mutations in the construct with Rejoin1 alone and examined mKate2 expression by flow cytometry. We found no significant differences between Rejoin1 and the two mutants, Rejoin1-GC&gt;Us and Rejoin1-AU&gt;GCs. Neither of these constructs reduced mKate2 expression compared to BKB (S3 Fig), suggesting that the function of the RSV minigenome system remained intact despite of the mutations. Altogether, these data suggest that a minimal content of GC nucleotides in the rejoin region determines if cbDVGs are produced at that particular genomic location.</p><p id="p-18">To determine if any of the two Cs within the mutated sequence was critical for cbDVG rejoining at this location, we performed a similar analysis using three new constructs: first C at position 188 or second C at position 186 from the 3’ end of antigenome mutated to U (named C188U or C186U, respectively), or both Cs mutated to Us (named AU). Transfection of the C186U, but not the C188U construct, resulted in one major DVG amplicon (indicated with an asterisk in <a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig 4E</a>; sequences in S2 Fig). The C186U construct rejoin points skipped the mutation area and concentrated in the early trailer region, similar to GC&gt;Us. This was confirmed by the two other primer sets. A strong band shown in lane C188U at high molecular weight (indicated with an arrowhead in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig 3E</a>) was determined to not correspond to a cbDVG by Sanger sequencing. The construct bearing the double mutation (AU) behaved similar to C186U in terms of the rejoin positions (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig 4C</a>). Thus, we found the second C at position 15037 (position 186 from 3’ trailer end of antigenome) to be critical for cbDVG generation.</p></div><div id="sec-7" class="subsection"><h3>A conserved rejoin region determines cbDVG formation during viral infection</h3><p id="p-19">To next establish whether rejoin1 impacts on cbDVG generation during viral infection, we created a mutant virus harboring mutations identical to the GOUs minigenome construct. This virus is herein identified as gRSV-FR-GC&gt;Us. The backbone of the recombinant RSV (Line 19) included the mKate2 gene and we used mKate2 expression to estimate its replication. As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5A</a>, cells infected with gRSV-FR-GC&gt;Us expressed the same level of mKate2 protein as cells infected with the WT reporter virus (gRSV-FR-WT) at 72 h post infection. Both viruses began to generate cbDVGs from P3 and the pattern of cbDVGs was maintained, and became stronger, in P5 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig 5B</a>). We verified that P5 gRSV-FR-GC&gt;Us still carried the mutations we introduced (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig 5C</a>). Interestingly, gRSV-FR-WT produced 4 major DVGs, whereas gRSV-FR-GC&gt;Us only generated one dominant cbDVG (asterisks in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig 5B</a>, confirmed sequence in S2G and S3H Fig), which is consistent with results from the minigenome system. The dominant cbDVG generated in cells infected with gRSV-FR-GC&gt;Us rejoined at the early trailer region and skipped the mutation site, similar to what was observed in the minigenome system. Cells infected with gRSV-FR-WT produced one cbDVGs that rejoined within the mutation site and other 3 cbDVGs that rejoined at the same region of the mutant virus (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Fig 4D</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A conserved rejoin region determines cbDVG formation during viral infection in tissue culture. (A) HEp2 cells were infected with gRSV-FR-WT-P3 and gRSV-FR-GC-&gt;Us-P3 at MOI of 1.5 and harvested at 72 h post infection and stained for aqua L/D, followed by flow cytometry for mKate2 expression as a proxy for virus replication. (B) cbDVG detection from P2, P3, P4, and P5 of gRSV-FR-WT and gRSV-FR-GC&gt;Us via DVG specific RT-PCR using Dl1/DI-R primer set in HEp2 cell. Confirmed cbDVG-like fragments are labeled by asterisks next to the gel. Different colors correspond to the color of double arrows in Fig 4D. Sequence of confirmed DVGs are listed in S2G and S2H Fig. (C) Confirmation of introduced mutations in gRSV-FR-GC-&gt;Us-P5 infected cells via Sanger sequencing. (D) Schematic summary of all identified cbDVG rejoin points within positions 14998-15105 (RSVA2, reference genome NCBI KT992094.1) generated during infections with gRSV-FR-WT, gRSV-FR-GC&gt;Us, and RSV stocksl-7. All DVGs were detected by DVG specific RT-PCR using the Dl1/DI-R primer set. RSV HD stock1-7 are strain A2 and gRSV-FR-WT and gRSV-FR-CG-&gt;Us are strain Linel9. The sequence difference between A2 and line 19 within common rejoin region is indicated by a red line. Colored double arrows indicated the rejoin points ofcbDVGs detected from viruses. Black parallel line marks the 6 nucleotides mutated in gRSV-FR-GC-&gt;Us." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1210250752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;A conserved rejoin region determines cbDVG formation during viral infection in tissue culture.&lt;/span&gt; (A) HEp2 cells were infected with gRSV-FR-WT-P3 and gRSV-FR-GC-&gt;Us-P3 at MOI of 1.5 and harvested at 72 h post infection and stained for aqua L/D, followed by flow cytometry for mKate2 expression as a proxy for virus replication. (B) cbDVG detection from P2, P3, P4, and P5 of gRSV-FR-WT and gRSV-FR-GC&gt;Us via DVG specific RT-PCR using Dl1/DI-R primer set in HEp2 cell. Confirmed cbDVG-like fragments are labeled by asterisks next to the gel. Different colors correspond to the color of double arrows in Fig 4D. Sequence of confirmed DVGs are listed in S2G and S2H Fig. (C) Confirmation of introduced mutations in gRSV-FR-GC-&gt;Us-P5 infected cells via Sanger sequencing. (D) Schematic summary of all identified cbDVG rejoin points within positions 14998-15105 (RSVA2, reference genome NCBI KT992094.1) generated during infections with gRSV-FR-WT, gRSV-FR-GC&gt;Us, and RSV stocksl-7. All DVGs were detected by DVG specific RT-PCR using the Dl1/DI-R primer set. RSV HD stock1-7 are strain A2 and gRSV-FR-WT and gRSV-FR-CG-&gt;Us are strain Linel9. The sequence difference between A2 and line 19 within common rejoin region is indicated by a red line. Colored double arrows indicated the rejoin points ofcbDVGs detected from viruses. Black parallel line marks the 6 nucleotides mutated in gRSV-FR-GC-&gt;Us.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F5.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F5.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 5.</span> <span class="caption-title">A conserved rejoin region determines cbDVG formation during viral infection in tissue culture.</span><p id="p-20" class="first-child">(A) HEp2 cells were infected with gRSV-FR-WT-P3 and gRSV-FR-GC-&gt;Us-P3 at MOI of 1.5 and harvested at 72 h post infection and stained for aqua L/D, followed by flow cytometry for mKate2 expression as a proxy for virus replication. (B) cbDVG detection from P2, P3, P4, and P5 of gRSV-FR-WT and gRSV-FR-GC&gt;Us via DVG specific RT-PCR using Dl1/DI-R primer set in HEp2 cell. Confirmed cbDVG-like fragments are labeled by asterisks next to the gel. Different colors correspond to the color of double arrows in <a id="xref-fig-4-12" class="xref-fig" href="#F4">Fig 4D</a>. Sequence of confirmed DVGs are listed in S2G and S2H Fig. (C) Confirmation of introduced mutations in gRSV-FR-GC-&gt;Us-P5 infected cells via Sanger sequencing. (D) Schematic summary of all identified cbDVG rejoin points within positions 14998-15105 (RSVA2, reference genome NCBI KT992094.1) generated during infections with gRSV-FR-WT, gRSV-FR-GC&gt;Us, and RSV stocksl-7. All DVGs were detected by DVG specific RT-PCR using the Dl1/DI-R primer set. RSV HD stock1-7 are strain A2 and gRSV-FR-WT and gRSV-FR-CG-&gt;Us are strain Linel9. The sequence difference between A2 and line 19 within common rejoin region is indicated by a red line. Colored double arrows indicated the rejoin points ofcbDVGs detected from viruses. Black parallel line marks the 6 nucleotides mutated in gRSV-FR-GC-&gt;Us.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">The majority of rejoin points found in infection with gRSV-FR-WT, which derived from RSV Line 19, located within the early trailer sequence, rather than around the mutation site as found in infection with RSVstocksl-7 derived from RSV line A2 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig 5D</a>). Alignment of both sequences revealed one natural mutation in RSV Line 19 that introduced three GCs right at the beginning of the trailer sequence, which are not present in RSV A2 (sequence indicated with a red horizontal line in <a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig 5D</a>). The increased GC content in this position in Line 19 likely explains why gRSV-FR-WT generates more cbDVGs at this location than RSV A2 stocksl-7. Regardless of this natural preference for rejoining in the early trailer, gRSV-FR-GC&gt;Us diminished the rejoin signal at the mutation site as no cbDVGs rejoin points were found at this location and resulted in less diversity of cbDVG generation compared to the WT virus. Overall, these data confirm that the common rejoin region sequence tested in the minigenome system determines cbDVG rejoining during RSV infection and that the GC content on this region critically determines the site of cbDVG rejoin.</p></div><div id="sec-8" class="subsection"><h3>Conserved cbDVGs form during natural RSV infections in humans</h3><p id="p-22">To examine whether the rejoin1 region was utilized during natural infections, we applied VODKA to RNA-seq datasets obtained from RSV-positive pediatric clinical samples. A total of 10 pediatric specimens were sequenced; 4 were classified as DVG-low and 6 as DVG-high based on semi-quantification following cbDVG PCR. VODKA outputs were aligned to the reference genome of an RSV strain A isolate (Reference genome NCBIKJ672447, 2012) and showed that, consistent with previous cbDVG-RT-PCR results, DVG-low samples (upper panel in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6A</a>) contained ∼8 fold less cbDVG junction reads than DVG-high patients (lower panel in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig 6A</a>). In addition, the coverage mapping showed the presence of multiple break and rejoin regions. Some of them were a mix of both break points and rejoin points (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig 6A</a>, read and black arrows). The rejoin points were particularly noteworthy because the majority of them clustered within one narrow AU-rich “Rejoin1+ Trailer” region (red ticks in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig 6B</a>) similar to that identified in <em>in vitro</em> infections (blue ticks in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig 6B</a>). According to the frequency of different cbDVG junction positions, we illustrated the top 6 major cbDVGs (one of them is a snap-back) in <a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig 6C</a> (details summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, Break_Rejoin position shown as T’_W). All of them were found in multiple patients (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Fig. 6C</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The most abundant DVG again had the rejoin point within the “Rejoin1+Trailer” region, despite of a higher diversity of rejoin points compared to <em>in vitro</em> infection, possibly due to the deeper depth of RNA-seq used to analyze clinical samples. Taken together, these results demonstrate that a conserved rejoin region drives the generation of most cbDVGs during RSV infection <em>in vitro</em> and <em>in vivo</em> and that identical RSV cbDVGs are generated in naturally infection of different individuals</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1075212/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1075212/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1075212" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Major DVG junctions identified by VODKA from RSV positive nasal secretions</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A conserved rejoin region determines cbDVG formation in infected human patients. (A) Four specimens from RSV A-positive pediatric patients with low contents of DVGs and 6 with high content of DVGs were selected for RNA-seq. All seq reads that passed the quality threshold were analyzed by VODKA to screen for cbDVG junctions. DVG junction reads were then aligned to the RSV reference genome NCBIKJ672447. Blue histogram shows a synopsis of total coverage at any given position of the RSV reference genome (illustrated by the y-axis). The number on the right side of the graph represents the total reads at the position with highest coverage. (B) AU-content (top, purple) within nucleotides 14700-15109 of the RSV strain A isolate KJ642247 (human samples) is plotted centered on a 40 base-pair sliding window in both plots. Grey box indicated the position ofRejoin1 and Trailer. Individual rejoin points identified by DVG junction reads from pediatric samples within this region are represented by red ticks underneath the graph. The rejoin points from in vitro infection of RSV stocks 1-6 are plotted using blue ticks. The darker color represented increased rejoin frequency at a given position. (C) Top 6 major DVGs identified from clinical samples are depicted by dashed lines that join their break points (red arrows) and rejoin points (black arrows). The thickness of the line represents the frequency of DVG reads at a given DVG as indicated. The number of patients that have the Particular DVG is indicated at the right of the rejoin point for each cbDVG." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1210250752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;A conserved rejoin region determines cbDVG formation in infected human patients.&lt;/span&gt; (A) Four specimens from RSV A-positive pediatric patients with low contents of DVGs and 6 with high content of DVGs were selected for RNA-seq. All seq reads that passed the quality threshold were analyzed by VODKA to screen for cbDVG junctions. DVG junction reads were then aligned to the RSV reference genome NCBIKJ672447. Blue histogram shows a synopsis of total coverage at any given position of the RSV reference genome (illustrated by the y-axis). The number on the right side of the graph represents the total reads at the position with highest coverage. (B) AU-content (top, purple) within nucleotides 14700-15109 of the RSV strain A isolate KJ642247 (human samples) is plotted centered on a 40 base-pair sliding window in both plots. Grey box indicated the position ofRejoin1 and Trailer. Individual rejoin points identified by DVG junction reads from pediatric samples within this region are represented by red ticks underneath the graph. The rejoin points from in vitro infection of RSV stocks 1-6 are plotted using blue ticks. The darker color represented increased rejoin frequency at a given position. (C) Top 6 major DVGs identified from clinical samples are depicted by dashed lines that join their break points (red arrows) and rejoin points (black arrows). The thickness of the line represents the frequency of DVG reads at a given DVG as indicated. The number of patients that have the Particular DVG is indicated at the right of the rejoin point for each cbDVG.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F6.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F6.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/15/349001/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 6.</span> <span class="caption-title">A conserved rejoin region determines cbDVG formation in infected human patients.</span><p id="p-23" class="first-child">(A) Four specimens from RSV A-positive pediatric patients with low contents of DVGs and 6 with high content of DVGs were selected for RNA-seq. All seq reads that passed the quality threshold were analyzed by VODKA to screen for cbDVG junctions. DVG junction reads were then aligned to the RSV reference genome NCBIKJ672447. Blue histogram shows a synopsis of total coverage at any given position of the RSV reference genome (illustrated by the y-axis). The number on the right side of the graph represents the total reads at the position with highest coverage. (B) AU-content (top, purple) within nucleotides 14700-15109 of the RSV strain A isolate KJ642247 (human samples) is plotted centered on a 40 base-pair sliding window in both plots. Grey box indicated the position ofRejoin1 and Trailer. Individual rejoin points identified by DVG junction reads from pediatric samples within this region are represented by red ticks underneath the graph. The rejoin points from in vitro infection of RSV stocks 1-6 are plotted using blue ticks. The darker color represented increased rejoin frequency at a given position. (C) Top 6 major DVGs identified from clinical samples are depicted by dashed lines that join their break points (red arrows) and rejoin points (black arrows). The thickness of the line represents the frequency of DVG reads at a given DVG as indicated. The number of patients that have the Particular DVG is indicated at the right of the rejoin point for each cbDVG.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-24">DVGs are critical regulators of viral replication and pathogenesis in multiple RNA virus infections, but the mechanisms modulating their generation are unknown. Historically, DVGs are thought to result from random errors introduced during replication by the viral polymerase. However, mounting evidence indicates that the generation of cbDVGs is not totally stochastic. Specifically, we show that during RSV infection discrete hotspots in the viral genome mark sites of the viral polymerase release and rejoin during cbDVG formation, both <em>in vitro</em> and during natural RSV infections in humans. Moreover, we show that the content of GC nucleotides within the major rejoin hotspot critically impacts the generation of cbDVGs at that position and we identified specific nucleotides that, when mutated, altered the ability of recombinant viruses to generate diverse species of DVGs. The identification of a specific sequence involved in cbDVG formation opens the unprecedented possibility of genetically manipulating the content of cbDVGs during infection. This possibility may significantly impact our ability to generate tools to further understand the role of these viral products in virus pathogenesis, as well as potentially manipulate the cbDVG content with antiviral and/or therapeutic purposes.</p><p id="p-25">In this study, we utilized a custom-designed algorithm, VODKA, to identify cbDVG in infections <em>in vitro</em> or <em>in vivo</em> from children naturally infected with RSV. VODKA outputs were consistent with previous results obtained using classic DVG-RT-PCR and demonstrated a higher sensitivity in the detection of cbDVGs both <em>in vitro</em> and in clinical samples. False-positive DVG junction reads were ruled out by screening all reads aligned to the host (reads from human transcriptome) using VODKA. This test resulted in a minimal number of hits compared to viral samples, adding to the evidence reported throughout this manuscript to support the specificity of cbDVG detection by VODKA. VODKA can successfully identify cbDVGs in a number of viruses, including SeV (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig 1</a>) and Ebola (not shown), offering a powerful tool for cbDVG detection in clinical samples.</p><p id="p-26">Based on our data, we conclude that the rejoin position significantly influences cbDVG generation. A minimum proportion of GC nucleotides are necessary for rejoining to occur at that particular region, and one nucleotide alone can influence the location of the DVG rejoin point implying that a strong rejoin signal likely needs an optimal number of GC nucleotides in specific locations. Importantly, the same differential distribution of cbDVG rejoin points was observed when we compared cbDVG generation from infections with RSV A2 and Linel9, which differ in their GC content at the beginning of the trailer region. In addition, our data suggest that rejoin sequences influence the function of break signals when inserted as pairs in the construct. Our data is in agreement with data from <em>in vitro</em> infections with measles virus lacking the C protein, where break points of cbDVGs were widely distributed along the genome, whereas the rejoin points were clustered in a narrow region close to 5’ end of the genome [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>].</p><p id="p-27">Further investigation into the molecular details of how the viral polymerase recognizes these signals may lead to important insights about the mechanism involved in RSV virus replication and the generation of cbDVGs. A lower density of nucleocapsid proteins (NPs) at certain genomic locations has been shown to result in increased cbDVG formation in SeV infection [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>], However, the mutations described to be responsible for low NP density were absent in our SeV stocks, suggesting that alternative mechanisms are likely involved. The usage of C nucleotides as a signal closely resembles the recognitions of “gene end” or “gene start” by the viral polymerase when working on transcription [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>] and it would be intriguing to evaluate if the mechanisms of cbDVG generation and viral RNA transcription are related. Another factor influencing DVG accumulation is their length, which is tightly related to the spatial structure of the viral RNPs. In paramyxoviruses, although it is thought that “only genomes with hexametric or heptametric lengths are efficiently replicated” [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], some cbDVGs generated <em>in vitro</em> do not obey this rule [<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. For RSV, we observed that a number of cbDVGs do not follow the rule of six or the rule of seven. Nonetheless, cbDVGs with certain length may have increased replication efficiency and thus an enhanced fitness advantage. Interestingly, in our minigenome system, although cbDVGs from Pair1 contained the expected rejoin point positions, break points frequently fell into a region further ahead of Break1, suggesting that the distance between the Break and Rejoin points may also play a role in determining where the break position is.</p><p id="p-28">In addition to genomic sequences, other factors, such as viral proteins, likely play an important role in DVG generation. For instance, influenza viruses harboring a high fidelity polymerase generate fewer deletion DVGs [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>], Mutations in non-structural protein 2 of influenza have also been shown to increase the <em>de novo</em> generation of DVGs by altering the fidelity of viral polymerase [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. Host factors may be essential contributors to DVG generation as well [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>]. For example, vesicular stomatitis virus produces a large amount of snap-back DVGs in most cell lines, except human-mouse somatic cell hybrids, and this cellular attribute was mapped to human chromosome 16 [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Similarly, infection with measles virus did not show <em>de novo</em> generation of DIPs in human WI-38 cells and SeV did not produce cbDVGs in chicken embryo lung cells [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. Despite the potential importance of these additional factors on DVG generation, the current work represents a major paradigm shift with the identification of sequences that regulate cbDVG formation.</p><p id="p-29">Remarkably, we found various common cbDVGs present in more one than one patient and at least one of those cbDVGs was also present in infections <em>in vitro.</em> These observations support a conserved origin for cbDVGs during infection and challenge the idea that DVGs occur as random product of virus replication. To date, all studies on DVG biology have been correlative in nature. This work opens up new areas of investigation and can ultimately allow us to manipulate the ability of viruses to produce DVGs as a powerful tool to study the role of DVGs in viral pathogenesis.</p></div><div class="section" id="sec-10"><h2 class="">Materials and methods</h2><div id="sec-11" class="subsection"><h3>Ethics statement</h3><p id="p-30">Studies of human samples were approved by University of Pennsylvania Institutional Review Board. The embryonated chicken eggs used in theses studies were 10 days old and were obtained from Charles River.</p></div><div id="sec-12" class="subsection"><h3>Cells and viruses</h3><p id="p-31">A549 cells (human type II alveolar cells, ATCC, #CRM-CCL185) and HEp2 cells (HeLa-derived human epithelial cells, ATCC, CCL23) were cultured at 7% C0<sub>2</sub> and 37°C with Dulbecco’s modified Eagle’s medium supplemented with 10% fetal bovine serum (FBS), 1 mM sodium pyruvate, 2 mM L-Glutamine, and 50 mg/ml gentamicin. BSR-T7 cells (Hamster kidney cells, BHK cells constitutively expressing the T7 polymerase, provided by Dr. Christopher Basler’s lab at Icahn School of Medicine) and were maintained in 10% FBS with 1 mg/ml Geneticin (Invitrogen). All cell lines were treated with mycoplasma removal agent (MP Biomedicals) and routinely tested for mycoplasma before use. Sendai virus Cantell stock (referred to as SeV HD, with high DVG particle content) was prepared in embryonated chicken eggs as described previously [<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. The SeV HD stock used in these experiments had a high infectious to total particle ratio of 500:15,000. RSV-HD stocks 1-7 (stock of RSV derived from strain A2, ATCC, #VR-1540 with a high content of cbDVGs) were prepared and characterized as described previously [<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>] in MAVS KO, STAT1 KO, and WT A549 cells, respectively. The cell lines were kindly provided by Dr. Susan Weiss (University of Pennsylvania).</p></div><div id="sec-13" class="subsection"><h3>Plasmids</h3><p id="p-32">Mammalian expression vectors for RSV N (NR-36462), P (NR-36463), M2-1 (NR-36464), and L (NR-36461) proteins, and the RSV reverse genetic backbone pSynkRSV-linel9F (rRSV-FR, NR-36460) were obtained from BEI Resources. Detailed information of the constructs can be found in reference [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>], The backbone plasmid of the RSV minigenome used for testing various DVG junction regions was constructed by cloning two regions of sequences amplified from pSynkRSV-linel9F into the pSH180 vector. The first region included a T7 promoter, a hammerhead ribozyme, RSV leader sequence, and genes encoding monomeric Katushka 2 (mKate2), while the second region included the RSV trailer sequence, a Hepatitis delta virus nbozyme and a T7 terminator. These regions were sequentially cloned into pslll80 vector using restriction enzyme pairs Spel/Sandl and Sandl/EcoRI, respectively. The potential cbDVG break and/or rejoin regions (positions in S1 Table) were then inserted between those two regions using restriction enzyme pairs Notl/Sandl and Sandl/Spal, respectively. A detailed scheme of the construct can be seen in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig 2A</a>. Pair1 and Rejoin1 mutations were introduced using the site-directed mutagenesis commercial kit QuickChange® II XL (Agilent, CA) according to the manufacture’s protocol. All primers used for cloning are listed in S1 Table. Mutations in reverse genetic backbone pSynkRSV-linel9F were generated by fusion PCR using primers in S1 Table as previously described [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>],</p></div><div id="sec-14" class="subsection"><h3>RNA extraction and DVG-RT-PCR</h3><p id="p-33">Total RNA was extracted using TRIzol or TRIzol LS (Invitrogen) according to the manufacturer’s specifications. For detection of RSV DVGs in RSV infection, 1-2 μg of isolated total RNA was reverse transcribed with the Dl1 primer using the SuperScript III reverse transcriptase (Invitrogen) without RNase H activity to avoid self-priming. Recombinant RNase H (Invitrogen) was later added to the reverse transcribed samples and incubated for 20 min at 37°C. DVGs were partially amplified using both Dl1 primer and Dl-R primer. The temperature cycle parameters used for the cbDVG-PCR in a BioRad C1000 Thermal Cycler were: 95°C for 10 min and 33-35 cycles of 95°C for 30 sec, 55°C for 30 sec and 72°C for 90 sec followed by a hold at 72°C for 5 min. Ramp rate of all steps was 3 degree/sec. Detailed method can be found in[<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">6</a>]. For detection of cbDVGs in the RSV minigenome system, extracted RNAs were treated with 2 μ1 TurboDNasel (Invitrogen) for 15 min at 37°C, followed by reverse transcription. Same procedures as above were utilized, except replacing Dl1 primer with DI-F1, DI-F2, and DI-F3 primers. These were then all paired with Dl-R reverse primer to amplify the different sizes of PCR products. Sequences of all primers are listed in S1 Table.</p></div><div id="sec-15" class="subsection"><h3>RT-qPCR</h3><p id="p-34">Total RNA (1 μ.g) was reversed transcribed using the high capacity RNA to cDNA kit from Applied Biosystems. cDNA was diluted to a concentration of 10 μg/μl and amplified with specific primers in the presence of SYBR green (Applied Biosystems). qPCR reactions were performed in triplicate using specific primers and the Power SYBR Green PCR Master Mixture (Applied Biosystems) in a Viia7 Applied Biosystems Light-cycler. Gene expression levels of RSV G were normalized to the <em>GAPDH</em> copy number. Sequences of primers used in these studies can be found in S1 Table.</p></div><div id="sec-16" class="subsection"><h3>RNA-Seq</h3><p id="p-35">RNA-Seq for SeV Cantell and RSV HD stocks 1-6 were performed as previously described [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. RNA was extracted using TRIzol reagent and was re-purified using the PicoPure™ RNA isolation kit (Thermo Fisher Scientific). RNA quality was assessed using the RNA Pico 6000 module on an Agilent Tapestation 2100 (Agilent Technologies) prior to cDNA library preparation. For SeV RNA-Seq dataset, total cDNA libraries were prepared starting from 75 ng (SeV Cantell) and 450 ng (RSV HD stocks) of extracted raw RNA using the lllumina TruSeq Stranded Total RNA LT kit with Ribo-Zero Gold, according to the manufacturer’s instructions. Samples were run on lllumina NextSeq 500 to generate 75 bp, single-end reads, resulting in 21-53 million reads per sample, with an average Q30 score &gt; 96.8%. For sequencing of samples from RSV-positive patients, including 4 DVG low patients and 6 DVG high patients, 100-450 ng of extracted raw RNA was used for preparation of cDNA library using the same kit as above. Samples were run on lllumina NextSeq 500 to generate 150bp, paired-end reads, resulting in 60-170 million reads/sample with average Q30 score &gt; 84.6%. To analyze genomic AU-content relative to DVG break and rejoin points, we calculated the percentage of A or U nucleotides over sliding windows of 40 bases using the Python programming language (Python Software Foundation, <a href="https://www.python.org/">https://www.python.org/</a>). We plotted AU-content and cbDVG rejoin points in R using the ggplot2 package [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>],</p></div><div id="sec-17" class="subsection"><h3>Viral Opensource DVG Key Algorithm (VODKA)</h3><p id="p-36">Based on our <em>in vitro</em> RSV experiments, we made the assumption that most cbDVGs are generated from the viral sequence near the 5’ end region of the genome (close to the Trailer sequence). Therefore, starting with the last 3kb of a reference viral genome, we built an index of potential DVG sequences by taking all possible combinations of two non-overlapping segments of L bases, where L is the read length. The segments are linked by reverse complementing the second segment (C-D) and adding the first segment (A-B) to it (S4 Fig). Sequenced reads are aligned to the potential DVGs using bowtie2 [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], and subsequently undergo two filtering steps. First, reads are removed unless they map across a breakpoint (A_C) with at least 15bp of mapped segment on each side. Second, the reads that map cleanly to the reference genome are filtered out. This pipeline gives the output read counts for each breakpoint (A_C). To be consistent with the structure of copy-back DVGs in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig 1A</a>, C is equivalent to breakpoint T’ and A is equivalent to rejoin point W. VODKA output reads were further aligned to reference viral genomes (RSV A2: NCBI accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KT992094.1&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom">KT992094.1</a>; RSV 2012 clinical isolate: NCBI accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KJ672447&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom">KJ672447</a>) or known SeV DVG-546 to identify the potential DVG junction regions using the Genious 7.0 software.</p></div><div id="sec-18" class="subsection"><h3>RSV minigenome and reverse genetics system</h3><p id="p-37">BHK cells constitutively expressing the T7 polymerase (BSR-T7 cells) were transfected with different minigenome constructs, gRSV-FR-WT, or gRSV-FR-GC-&gt;Us, as well as the sequence-optimized helper plasmids encoding N, P, M2-1, and L, all underT7 control as described previously [<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>]. Cells were incubated with transfection complex (total plasmid: lipofectamine=1:3.3) for 2 h at room temperature and then at 37°C for overnight using Opti-MEM as medium. The following morning, the medium was replaced with antibiotic free tissue culture medium containing 2% FBS. For minigenome experiments, cells were harvested at 48 h post-transfection for either RNA extraction or flow cytometry. For mutant virus production, cells were maintained and split every 2-3 days until see obvious CPEs. Then viruses were collected and blindly passaged in HEp2 cells three times to obtain P3. P3 was titrated and passaged two more times at MOI of 10 to generate P4 and P5.</p></div><div id="sec-19" class="subsection"><h3>Flow Cytometry</h3><p id="p-38">Transfected BSR-T7 cells were trypsinized 48 h post transfection and were either directly diluted in FACS buffer (PBS containing 20 mM EDTA) or stained with aqua L/D. Cells were washed twice in FACS buffer before flow cytometry analysis on an LSRFortessa (Becton Dickinson). Data analysis was performed using Flowjo version Legacy.</p></div><div id="sec-20" class="subsection"><h3>Statistical analysis</h3><p id="p-39">All statistical analyses were performed with GraphPad Prism version 5.0 (GraphPad Software, San Diego, CA) and <em>R v3.4.1.</em> A statistically significant difference was defined as a <em>p</em>-value &lt;0.05 by one-way analysis of variance (ANOVA) with a <em>post hoc</em> test to correct for multiple comparisons (based on specific data sets as indicated in each figure legend).</p></div><div id="sec-21" class="subsection"><h3>Code availability</h3><p id="p-40">The VODKA algorithm is open-source and available at: <a href="https://github.com/itmat/VODKA">https://github.com/itmat/VODKA</a>.</p></div><div id="sec-22" class="subsection"><h3>Data availability</h3><p id="p-41">All data are available upon request to the corresponding author. Raw RNA-Sequencing data of FISH-FACS sorted SeV infected cells and RSV infected samples have been deposited on the Gene Expression Omnibus (GEO) database for public access (SeV: GSE96774; RSV: GSE114948).</p></div></div><div class="section" id="sec-24"><h2 class="">Supporting information captions</h2><p id="p-44">S1 Fig. Identification of RSV genomic regions responsible for the generation of cbDVGs.</p><p id="p-45">S2 Fig. Sequences of cbDVGs identified from labeled bands in RSV minigenome system and viral infection.</p><p id="p-46">S3 Fig. GC mutations in the RSV minigenome did not alter viral polymerase processivity.</p><p id="p-47">S4 Fig. Working scheme of VODKA.</p><p id="p-48">S1 Table. Oligo primer list.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-42">This work was supported by the US National Institutes of Health National Institute of Allergy and Infectious Diseases (NIH AI083284 and AI137062 to C.B.L). We thank Dan Beiting for the support with RNA-seq and GEO submission.</p><p id="p-43">Conceived experiments: Y.S., and C.B.L.; Developed methodology: Y.S., E.J.K, D.A., and G.R.G; Performed experiments, collected and analyzed data: Y.S., E.J.K., L.J.T., and S.F.; Provided reagents and resources: G.R.G; Wrote the original draft: Y.S., D.A., and C.B.L.; Supervised research activities: C.B.L</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.1" data-doi="10.1126/science.98.2534.87"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Henle</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henle</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">INTERFERENCE OF INACTIVE VIRUS WITH THE PROPAGATION OF VIRUS OF INFLUENZA</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">1943</span>;<span class="cit-vol">98</span>(<span class="cit-issue">2534</span>):<span class="cit-fpage">87</span>–<span class="cit-lpage">9</span>. Epub 1943/07/23.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.98.2534.87</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">17749157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHENLE%26rft.auinit1%253DW.%26rft.volume%253D98%26rft.issue%253D2534%26rft.spage%253D87%26rft.epage%253D89%26rft.atitle%253DINTERFERENCE%2BOF%2BINACTIVE%2BVIRUS%2BWITH%2BTHE%2BPROPAGATION%2BOF%2BVIRUS%2BOF%2BINFLUENZA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.98.2534.87%26rft_id%253Dinfo%253Apmid%252F17749157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTA6Ijk4LzI1MzQvODciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMi8xNS8zNDkwMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-349001v3.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">von</span>  <span class="cit-name-given-names">MP</span></span>. <span class="cit-article-title">Propagation of the PR8 strain of influenza A virus in chick embryos. II. The formation of incomplete virus following inoculation of large doses of seed virus</span>. <abbr class="cit-jnl-abbrev">Acta Pathol Microbiol Scand</abbr>. <span class="cit-pub-date">1951</span>;<span class="cit-vol">28</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">278</span>–<span class="cit-lpage">93</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID: </span><span class="cit-pub-id cit-pub-id-pmid">14856732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2Bpathologica%2Bet%2Bmicrobiologica%2BScandinavica%26rft.stitle%253DActa%2BPathol%2BMicrobiol%2BScand%26rft.aulast%253Dvon%2BMAGNUS%26rft.auinit1%253DP.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D278%26rft.epage%253D293%26rft.atitle%253DPropagation%2Bof%2Bthe%2BPR8%2Bstrain%2Bof%2Binfluenza%2BA%2Bvirus%2Bin%2Bchick%2Bembryos.%2BII.%2BThe%2Bformation%2Bof%2Bincomplete%2Bvirus%2Bfollowing%2Binoculation%2Bof%2Blarge%2Bdoses%2Bof%2Bseed%2Bvirus.%26rft_id%253Dinfo%253Apmid%252F14856732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=14856732&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1951YC39500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-349001v3.3" data-doi="10.1016/S0065-3527(08)60529-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Von Magnus</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Incomplete forms of influenza virus</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr>. <span class="cit-pub-date">1954</span>;<span class="cit-vol">2</span>:<span class="cit-fpage">59</span>–<span class="cit-lpage">79</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">13228257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253DVON%2BMAGNUS%26rft.auinit1%253DP.%26rft.volume%253D2%26rft.spage%253D59%26rft.epage%253D79%26rft.atitle%253DIncomplete%2Bforms%2Bof%2Binfluenza%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252808%252960529-1%26rft_id%253Dinfo%253Apmid%252F13228257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-3527(08)60529-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13228257&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1954XS98500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.4" data-doi="10.1128/JVI.00707-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">CB</span></span>. <span class="cit-article-title">Defective viral genomes: critical danger signals of viral infections</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">88</span>(<span class="cit-issue">16</span>):<span class="cit-fpage">8720</span>–<span class="cit-lpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00707-14</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">24872580</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC4136278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00707-14%26rft_id%253Dinfo%253Apmid%252F24872580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OC8xNi84NzIwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDIvMTUvMzQ5MDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.5" data-doi="10.1016/j.cytogfr.2014.05.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">RIG-I-like receptors and negative-strand RNA viruses: RLRly bird catches some worms</span>. <abbr class="cit-jnl-abbrev">Cytokine Growth Factor Rev</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">25</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">621</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cytogfr.2014.05.004</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">24894317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytokine%2BGrowth%2BFactor%2BRev%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cytogfr.2014.05.004%26rft_id%253Dinfo%253Apmid%252F24894317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cytogfr.2014.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24894317&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.6" data-doi="10.1371/journal.ppat.1005122"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Koziol-White</span> <span class="cit-name-surname">cj</span></span>, <span class="cit-auth"><span class="cit-name-surname">Genoyer</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tapia</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Immunostimulatory Defective Viral Genomes from Respiratory Syncytial Virus Promote a Strong Innate Antiviral Response during Infection in Mice and Humans</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">11</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">e1005122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1005122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D11%26rft.spage%253De1005122%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005122%26rft_id%253Dinfo%253Apmid%252F26336095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26336095&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.7" data-doi="10.1371/journal.ppat.1003703"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tapia</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">WK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mercado-Lopez</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunay</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wise</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Defective viral genomes arising in vivo provide critical danger signals for the triggering of lung antiviral immunity</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">9</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">e1003703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1003703</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">24204261</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC3814336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003703%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003703%26rft_id%253Dinfo%253Apmid%252F24204261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204261&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.8" data-doi="10.1371/journal.ppat.1006650"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vasilijevic</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zamarreno</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oliveros</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez-Frandsen</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Reduced accumulation of defective viral genomes contributes to severe outcome in influenza virus infected patients</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">13</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">e1006650</span>. Epub 2017/10/13.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1006650</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">29023600</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPmc5638565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253De1006650%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006650%26rft_id%253Dinfo%253Apmid%252F29023600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29023600&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.9" data-doi="10.1016/j.virol.2006.03.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Strahle</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolakofsky</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Sendai virus defective-interfering genomes and the activation of interferon-beta</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">351</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">101</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2006.03.022</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">16631220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DStrahle%26rft.auinit1%253DL.%26rft.volume%253D351%26rft.issue%253D1%26rft.spage%253D101%26rft.epage%253D111%26rft.atitle%253DSendai%2Bvirus%2Bdefective-interfering%2Bgenomes%2Band%2Bthe%2Bactivation%2Bof%2Binterferon-beta.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2006.03.022%26rft_id%253Dinfo%253Apmid%252F16631220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2006.03.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16631220&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239319900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.10" data-doi="10.1002/biot.201400429"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frensing</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Defective interfering viruses and their impact on vaccines and viral vectors</span>. <abbr class="cit-jnl-abbrev">Biotechnol J</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">10</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">681</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/biot.201400429</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">25728309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnol%2BJ%26rft.volume%253D10%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbiot.201400429%26rft_id%253Dinfo%253Apmid%252F25728309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/biot.201400429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25728309&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.11" data-doi="10.1038/226325a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baltimore</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Defective viral particles and viral disease processes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">1970</span>;<span class="cit-vol">226</span>(<span class="cit-issue">5243</span>):<span class="cit-fpage">325</span>–<span class="cit-lpage">7</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">5439728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHuang%26rft.auinit1%253DA.%2BS.%26rft.volume%253D226%26rft.issue%253D5243%26rft.spage%253D325%26rft.epage%253D327%26rft.atitle%253DDefective%2Bviral%2Bparticles%2Band%2Bviral%2Bdisease%2Bprocesses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F226325a0%26rft_id%253Dinfo%253Apmid%252F5439728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/226325a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5439728&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.12" data-doi="10.1016/0092-8674(81)90298-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lazzarini</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keene</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">The origins of defective interfering particles of the negative-strand RNA viruses</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-pub-date">1981</span>;<span class="cit-vol">26</span>(<span class="cit-issue">2 Pt 2</span>):<span class="cit-fpage">145</span>–<span class="cit-lpage">54</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">7037195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLazzarini%26rft.auinit1%253DR.%2BA.%26rft.volume%253D26%26rft.issue%253D2%2BPt%2B2%26rft.spage%253D145%26rft.epage%253D154%26rft.atitle%253DThe%2Borigins%2Bof%2Bdefective%2Binterfering%2Bparticles%2Bof%2Bthe%2Bnegative-strand%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252881%252990298-1%26rft_id%253Dinfo%253Apmid%252F7037195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(81)90298-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7037195&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MM63400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.13" data-doi="10.3390/v1030895"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pathak</span>  <span class="cit-name-given-names">KB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">PD</span></span>. <span class="cit-article-title">Defective Interfering RNAs: Foes of Viruses and Friends of Virologists</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">1</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">895</span>–<span class="cit-lpage">919</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v10308951</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">21994575</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC3185524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DPathak%26rft.auinit1%253DK.%2BB.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D895%26rft.epage%253D919%26rft.atitle%253DDefective%2BInterfering%2BRNAs%253A%2BFoes%2Bof%2BViruses%2Band%2BFriends%2Bof%2BVirologists.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv1030895%26rft_id%253Dinfo%253Apmid%252F21994575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v1030895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21994575&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280413200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.14" data-doi="10.1016/0042-6822(79)90400-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janda</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nayak</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">De</span>  <span class="cit-name-given-names">BK</span></span>. <span class="cit-article-title">Diversity and generation of defective interfering influenza virus particles</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">1979</span>;<span class="cit-vol">95</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">48</span>–<span class="cit-lpage">58</span>. Epub 1979/05/01. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">442544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DJanda%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D48%26rft.epage%253D58%26rft.atitle%253DDiversity%2Band%2Bgeneration%2Bof%2Bdefective%2Binterfering%2Binfluenza%2Bvirus%2Bparticles.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252879%252990400-8%26rft_id%253Dinfo%253Apmid%252F442544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(79)90400-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=442544&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GV71000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-349001v3.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nayak</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akkina</span>  <span class="cit-name-given-names">RK</span></span>. <span class="cit-article-title">Defective-interfering (DI) RNAs of influenza viruses: origin, structure, expression, and interference</span>. <abbr class="cit-jnl-abbrev">Curr Top Microbiol Immunol</abbr>. <span class="cit-pub-date">1985</span>;<span class="cit-vol">114</span>:<span class="cit-fpage">103</span>–<span class="cit-lpage">51</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">3888540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bmicrobiology%2Band%2Bimmunology%26rft.stitle%253DCurr%2BTop%2BMicrobiol%2BImmunol%26rft.aulast%253DNayak%26rft.auinit1%253DD.%2BP.%26rft.volume%253D114%26rft.spage%253D103%26rft.epage%253D151%26rft.atitle%253DDefective-interfering%2B%2528DI%2529%2BRNAs%2Bof%2Binfluenza%2Bviruses%253A%2Borigin%252C%2Bstructure%252C%2Bexpression%252C%2Band%2Binterference.%26rft_id%253Dinfo%253Apmid%252F3888540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3888540&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AML2000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.16" data-doi="10.1016/0042-6822(85)90339-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kajigaya</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arakawa</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuge</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imura</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nomoto</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Isolation and characterization of defective-interfering particles of poliovirus Sabin 1 strain</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">1985</span>;<span class="cit-vol">142</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">307</span>–<span class="cit-lpage">16</span>. Epub 1985/04/30. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">2997988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DKajigaya%26rft.auinit1%253DS.%26rft.volume%253D142%26rft.issue%253D2%26rft.spage%253D307%26rft.epage%253D316%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Bdefective-interfering%2Bparticles%2Bof%2Bpoliovirus%2BSabin%2B1%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252885%252990339-3%26rft_id%253Dinfo%253Apmid%252F2997988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(85)90339-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2997988&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AEL3000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.17" data-doi="10.1128/JVI.02921-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poirier</span>  <span class="cit-name-given-names">EZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mounce</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozen-Gagnon</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hooikaas</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stapleford</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moratorio</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Low-Fidelity Polymerases of Alphaviruses Recombine at Higher Rates To Overproduce Defective Interfering Particles</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">90</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">2446</span>–<span class="cit-lpage">54</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02921-15</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">26676773</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC4810721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02921-15%26rft_id%253Dinfo%253Apmid%252F26676773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjkwLzUvMjQ0NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.18" data-doi="10.1016/j.virusres.2011.03.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">ZH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of new defective interfering RNA species associated with porcine reproductive and respiratory syndrome virus infection</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">158</span>(<span class="cit-issue">1-2</span>):<span class="cit-fpage">33</span>–<span class="cit-lpage">6</span>. Epub 2011/03/10.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2011.03.001</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">21385595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DXiao%26rft.auinit1%253DC.%2BT.%26rft.volume%253D158%26rft.issue%253D1-2%26rft.spage%253D33%26rft.epage%253D36%26rft.atitle%253DIdentification%2Bof%2Bnew%2Bdefective%2Binterfering%2BRNA%2Bspecies%2Bassociated%2Bwith%2Bporcine%2Breproductive%2Band%2Brespiratory%2Bsyndrome%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2011.03.001%26rft_id%253Dinfo%253Apmid%252F21385595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2011.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21385595&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Odagiri</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tashiro</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Segment-specific noncoding sequences of the influenza virus genome RNA are involved in the specific competition between defective interfering RNA and its progenitor RNA segment at the virion assembly step</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1997</span>;<span class="cit-vol">71</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">2138</span>–<span class="cit-lpage">45</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">9032347</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC191316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DOdagiri%26rft.auinit1%253DT.%26rft.volume%253D71%26rft.issue%253D3%26rft.spage%253D2138%26rft.epage%253D2145%26rft.atitle%253DSegment-specific%2Bnoncoding%2Bsequences%2Bof%2Bthe%2Binfluenza%2Bvirus%2Bgenome%2BRNA%2Bare%2Binvolved%2Bin%2Bthe%2Bspecific%2Bcompetition%2Bbetween%2Bdefective%2Binterfering%2BRNA%2Band%2Bits%2Bprogenitor%2BRNA%2Bsegment%2Bat%2Bthe%2Bvirion%2Bassembly%2Bstep%26rft_id%253Dinfo%253Apmid%252F9032347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcxLzMvMjEzOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.20" data-doi="10.4049/jimmunol.179.9.6123"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shingai</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebihara</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Honma</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential type I IFN-inducing abilities of wild-type versus vaccine strains of measles virus</span>. <abbr class="cit-jnl-abbrev">J Immunol</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">179</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">6123</span>–<span class="cit-lpage">33</span>. Epub 2007/10/20. doi:179/9/6123 [pii]. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">17947687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DShingai%26rft.auinit1%253DM.%26rft.volume%253D179%26rft.issue%253D9%26rft.spage%253D6123%26rft.epage%253D6133%26rft.atitle%253DDifferential%2BType%2BI%2BIFN-Inducing%2BAbilities%2Bof%2BWild-Type%2Bversus%2BVaccine%2BStrains%2Bof%2BMeasles%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.179.9.6123%26rft_id%253Dinfo%253Apmid%252F17947687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE3OS85LzYxMjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMi8xNS8zNDkwMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.21" data-doi="10.1128/JVI.02161-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kew</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lui</span>  <span class="cit-name-given-names">PY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">CP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akira</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">PACT- and RIG-I-Dependent Activation of Type I Interferon Production by a Defective Interfering RNA Derived from Measles Virus Vaccine</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">90</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">1557</span>–<span class="cit-lpage">68</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02161-15</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">26608320</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC4719617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02161-15%26rft_id%253Dinfo%253Apmid%252F26608320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjkwLzMvMTU1NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.22" data-doi="10.1016/0092-8674(77)90130-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leppert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kort</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolakofsky</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Further characterization of Sendai virus DI-RNAs: a model for their generation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-pub-date">1977</span>;<span class="cit-vol">12</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">539</span>–<span class="cit-lpage">52</span>. Epub 1977/10/01. doi:0092-8674(77)90130-1 [pii]. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">199355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLeppert%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D539%26rft.epage%253D552%26rft.atitle%253DFurther%2Bcharacterization%2Bof%2BSendai%2Bvirus%2BDI-RNAs%253A%2Ba%2Bmodel%2Bfor%2Btheir%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252877%252990130-1%26rft_id%253Dinfo%253Apmid%252F199355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(77)90130-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=199355&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977DY09600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-349001v3.23"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Shatkin</span>  <span class="cit-name-given-names">AJ</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Perrault</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-chapter-title">Origin and replicaition of defective interfering particles</span>. In: <span class="cit-ed"><span class="cit-name-surname">Shatkin</span>  <span class="cit-name-given-names">AJ</span></span>, editor. <span class="cit-source">Initiation Signal in Viral Gene Expression</span>. <span class="cit-publ-name">Berlin: Springer-Verlag</span>; <span class="cit-pub-date">1981</span>. p. <span class="cit-fpage">151</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fearns</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">PL</span></span>. <span class="cit-article-title">Role of the M2-1 transcription antitermination protein of respiratory syncytial virus in sequential transcription</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1999</span>;<span class="cit-vol">73</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">5852</span>–<span class="cit-lpage">64</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">10364337</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC112646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DFearns%26rft.auinit1%253DR.%26rft.volume%253D73%26rft.issue%253D7%26rft.spage%253D5852%26rft.epage%253D5864%26rft.atitle%253DRole%2Bof%2Bthe%2BM2-1%2BTranscription%2BAntitermination%2BProtein%2Bof%2BRespiratory%2BSyncytial%2BVirus%2Bin%2BSequential%2BTranscription%26rft_id%253Dinfo%253Apmid%252F10364337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjczLzcvNTg1MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.25" data-doi="10.1042/BJ20081949"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shcherbo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ermakova</span>  <span class="cit-name-given-names">GV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solovieva</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chepurnykh</span>  <span class="cit-name-given-names">TV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shcheglov</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Farred fluorescent tags for protein imaging in living tissues</span>. <abbr class="cit-jnl-abbrev">Biochem J</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">418</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">567</span>–<span class="cit-lpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1042/BJ20081949</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">19143658</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC2893397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem%2BJ%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20081949%26rft_id%253Dinfo%253Apmid%252F19143658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiNDE4LzMvNTY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDIvMTUvMzQ5MDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.26" data-doi="10.1128/JVI.01017-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pfaller</span>  <span class="cit-name-given-names">CK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mastorakos</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matchett</span>  <span class="cit-name-given-names">WE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samuel</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cattaneo</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Measles Virus Defective Interfering RNAs Are Generated Frequently and Early in the Absence of C Protein and Can Be Destabilized by Adenosine Deaminase Acting on RNA-1-Like Hypermutations</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">89</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">7735</span>–<span class="cit-lpage">47</span>. Epub 2015/05/15.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01017-15</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">25972541</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPmc4505647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01017-15%26rft_id%253Dinfo%253Apmid%252F25972541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xNS83NzM1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDIvMTUvMzQ5MDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.27" data-doi="10.1128/JVI.02094-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawabata</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ami</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakaguchi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A Single Amino Acid Substitution within the Paramyxovirus Sendai Virus Nucleoprotein Is a Critical Determinant for Production of Interferon-Beta-Inducing Copyback-Type Defective Interfering Genomes</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">92</span>(<span class="cit-issue">5</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02094-17</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">29237838</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC5809723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02094-17%26rft_id%253Dinfo%253Apmid%252F29237838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjE0OiI5Mi81L2UwMjA5NC0xNyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.28" data-doi="10.1016/j.virusres.2017.01.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fearns</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plemper</span>  <span class="cit-name-given-names">RK.</span></span> <span class="cit-article-title">Polymerases of paramyxoviruses and pneumoviruses</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">234</span>:<span class="cit-fpage">87</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2017.01.008</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">28104450</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC5476513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D234%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.01.008%26rft_id%253Dinfo%253Apmid%252F28104450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28104450&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fearns</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">PL</span></span>. <span class="cit-article-title">Analysis of the gene start and gene end signals of human respiratory syncytial virus: quasi-templated initiation at position 1 of the encoded mRNA</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1997</span>;<span class="cit-vol">71</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">4944</span>–<span class="cit-lpage">53</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">9188557</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC191725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKuo%26rft.auinit1%253DL.%26rft.volume%253D71%26rft.issue%253D7%26rft.spage%253D4944%26rft.epage%253D4953%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bgene%2Bstart%2Band%2Bgene%2Bend%2Bsignals%2Bof%2Bhuman%2Brespiratory%2Bsyncytial%2Bvirus%253A%2Bquasi-templated%2Binitiation%2Bat%2Bposition%2B1%2Bof%2Bthe%2Bencoded%2BmRNA%26rft_id%253Dinfo%253Apmid%252F9188557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcxLzcvNDk0NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calain</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">The rule of six, a basic feature for efficient replication of Sendai virus defective interfering RNA</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1993</span>;<span class="cit-vol">67</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">4822</span>–<span class="cit-lpage">30</span>. Epub 1993/08/01. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">8392616</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMC237869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DCalain%26rft.auinit1%253DP%26rft.volume%253D67%26rft.issue%253D8%26rft.spage%253D4822%26rft.epage%253D4830%26rft.atitle%253DThe%2Brule%2Bof%2Bsix%252C%2Ba%2Bbasic%2Bfeature%2Bfor%2Befficient%2Breplication%2Bof%2BSendai%2Bvirus%2Bdefective%2Binterfering%2BRNA.%26rft_id%253Dinfo%253Apmid%252F8392616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY3LzgvNDgyMiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kolakofsky</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pelet</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hausmann</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Curran</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Paramyxovirus RNA synthesis and the requirement for hexamer genome length: the rule of six revisited</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1998</span>;<span class="cit-vol">72</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">891</span>–<span class="cit-lpage">9</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">9444980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKolakofsky%26rft.auinit1%253DD.%26rft.volume%253D72%26rft.issue%253D2%26rft.spage%253D891%26rft.epage%253D899%26rft.atitle%253DParamyxovirus%2BRNA%2BSynthesis%2Band%2Bthe%2BRequirement%2Bfor%2BHexamer%2BGenome%2BLength%253A%2Bthe%2BRule%2Bof%2BSix%2BRevisited%26rft_id%253Dinfo%253Apmid%252F9444980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjcyLzIvODkxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDIvMTUvMzQ5MDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-349001v3.32" data-doi="10.1126/science.1177634"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tawar</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duquerroy</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vonrhein</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Varela</span>  <span class="cit-name-given-names">PF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Damier-Piolle</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castagne</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-chapter-title">Crystal structure of a nucleocapsid-like nucleoprotein-RNA complex of respiratory syncytial virus</span>. <span class="cit-source">Science</span> (<span class="cit-publ-loc">New York, NY</span>). <span class="cit-pub-date">2009</span>;<span class="cit-vol">326</span>(<span class="cit-issue">5957</span>):<span class="cit-fpage">1279</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1177634</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">19965480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTawar%26rft.auinit1%253DR.%2BG.%26rft.volume%253D326%26rft.issue%253D5957%26rft.spage%253D1279%26rft.epage%253D1283%26rft.atitle%253DCrystal%2BStructure%2Bof%2Ba%2BNucleocapsid-Like%2BNucleoprotein-RNA%2BComplex%2Bof%2BRespiratory%2BSyncytial%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1177634%26rft_id%253Dinfo%253Apmid%252F19965480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ny8xMjc5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDIvMTUvMzQ5MDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Samal</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">PL</span></span>. <span class="cit-article-title">RNA replication by a respiratory syncytial virus RNA analog does not obey the rule of six and retains a nonviral trinucleotide extension at the leader end</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1996</span>;<span class="cit-vol">70</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">5075</span>–<span class="cit-lpage">82</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">8764015</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC190462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSamal%26rft.auinit1%253DS.%2BK.%26rft.volume%253D70%26rft.issue%253D8%26rft.spage%253D5075%26rft.epage%253D5082%26rft.atitle%253DRNA%2Breplication%2Bby%2Ba%2Brespiratory%2Bsyncytial%2Bvirus%2BRNA%2Banalog%2Bdoes%2Bnot%2Bobey%2Bthe%2Brule%2Bof%2Bsix%2Band%2Bretains%2Ba%2Bnonviral%2Btrinucleotide%2Bextension%2Bat%2Bthe%2Bleader%2Bend.%26rft_id%253Dinfo%253Apmid%252F8764015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcwLzgvNTA3NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.34" data-doi="10.1128/JVI.77.8.5017-5020.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fodor</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mingay</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brownlee</span>  <span class="cit-name-given-names">GG</span></span>. <span class="cit-article-title">A single amino acid mutation in the PA subunit of the influenza virus RNA polymerase promotes the generation of defective interfering RNAs</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">77</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">5017</span>–<span class="cit-lpage">20</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">12663810</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC152145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DFodor%26rft.auinit1%253DE.%26rft.volume%253D77%26rft.issue%253D8%26rft.spage%253D5017%26rft.epage%253D5020%26rft.atitle%253DA%2BSingle%2BAmino%2BAcid%2BMutation%2Bin%2Bthe%2BPA%2BSubunit%2Bof%2Bthe%2BInfluenza%2BVirus%2BRNA%2BPolymerase%2BPromotes%2Bthe%2BGeneration%2Bof%2BDefective%2BInterfering%2BRNAs%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.8.5017-5020.2003%26rft_id%253Dinfo%253Apmid%252F12663810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc3LzgvNTAxNyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.35" data-doi="10.1073/pnas.87.15.5988"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Odagiri</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tobita</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Mutation in NS2, a nonstructural protein of influenza A virus, extragenically causes aberrant replication and expression of the PA gene and leads to generation of defective interfering particles</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">1990</span>;<span class="cit-vol">87</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">5988</span>–<span class="cit-lpage">92</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">2143025</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC54455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOdagiri%26rft.auinit1%253DT.%26rft.volume%253D87%26rft.issue%253D15%26rft.spage%253D5988%26rft.epage%253D5992%26rft.atitle%253DMutation%2Bin%2BNS2%252C%2Ba%2Bnonstructural%2Bprotein%2Bof%2Binfluenza%2BA%2Bvirus%252C%2Bextragenically%2Bcauses%2Baberrant%2Breplication%2Band%2Bexpression%2Bof%2Bthe%2BPA%2Bgene%2Band%2Bleads%2Bto%2Bgeneration%2Bof%2Bdefective%2Binterfering%2Bparticles.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.87.15.5988%26rft_id%253Dinfo%253Apmid%252F2143025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODcvMTUvNTk4OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAyLzE1LzM0OTAwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">CY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weide</span>  <span class="cit-name-given-names">LG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tischfield</span>  <span class="cit-name-given-names">JA</span></span>. <span class="cit-article-title">Suppression of vesicular stomatitis virus defective intefering particle generation by a function(s) associated with human chromosome 16</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1981</span>;<span class="cit-vol">40</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">946</span>–<span class="cit-lpage">52</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">6275129</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC256708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKang%26rft.auinit1%253DC.%2BY.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D946%26rft.epage%253D952%26rft.atitle%253DSuppression%2Bof%2BVesicular%2BStomatitis%2BVirus%2BDefective%2BIntefering%2BParticle%2BGeneration%2Bby%2Ba%2BFunction%2528s%2529%2BAssociated%2Bwith%2BHuman%2BChromosome%2B16%26rft_id%253Dinfo%253Apmid%252F6275129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjQwLzMvOTQ2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDIvMTUvMzQ5MDAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.37" data-doi="10.1006/viro.1996.0335"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Whistler</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellini</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rota</span>  <span class="cit-name-given-names">PA</span></span>. <span class="cit-article-title">Generation of defective interfering particles by two vaccine strains of measles virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">1996</span>;<span class="cit-vol">220</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">480</span>–<span class="cit-lpage">4</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">8661398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DWhistler%26rft.auinit1%253DT.%26rft.volume%253D220%26rft.issue%253D2%26rft.spage%253D480%26rft.epage%253D484%26rft.atitle%253DGeneration%2Bof%2Bdefective%2Binterfering%2Bparticles%2Bby%2Btwo%2Bvaccine%2Bstrains%2Bof%2Bmeasles%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1996.0335%26rft_id%253Dinfo%253Apmid%252F8661398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1996.0335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8661398&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.38" data-doi="10.1016/0042-6822(70)90336-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kingsbury</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Portner</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">On the genesis of incomplete Sendai virions</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">1970</span>;<span class="cit-vol">42</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">872</span>–<span class="cit-lpage">9</span>. Epub 1970/12/01. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">4321309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DKingsbury%26rft.auinit1%253DD.%2BW.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D872%26rft.epage%253D879%26rft.atitle%253DOn%2Bthe%2Bgenesis%2Bof%2Bincomplete%2BSendai%2Bvirions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252870%252990336-3%26rft_id%253Dinfo%253Apmid%252F4321309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(70)90336-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4321309&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.39" data-doi="10.4049/jimmunol.177.7.4503"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yount</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kraus</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">CB</span></span>. <span class="cit-article-title">A novel role for viral-defective interfering particles in enhancing dendritic cell maturation</span>. <abbr class="cit-jnl-abbrev">J Immunol</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">177</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">4503</span>–<span class="cit-lpage">13</span>. PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">16982887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DYount%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D177%26rft.issue%253D7%26rft.spage%253D4503%26rft.epage%253D4513%26rft.atitle%253DA%2BNovel%2BRole%2Bfor%2BViral-Defective%2BInterfering%2BParticles%2Bin%2BEnhancing%2BDendritic%2BCell%2BMaturation%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.177.7.4503%26rft_id%253Dinfo%253Apmid%252F16982887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny83LzQ1MDMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMi8xNS8zNDkwMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">López</span>  <span class="cit-name-given-names">CB</span></span>. <span class="cit-article-title">Preparation of Respiratory Syncytial Virus with High or Low Content of Defective Viral Particles and Their Purification from Viral Stocks</span>. <abbr class="cit-jnl-abbrev">Bio-protocol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">6</span>(<span class="cit-issue">10</span>). Epub 5/20/2016. doi:0.21769/BioProtoc.1820.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.41" data-doi="10.1016/j.virol.2012.09.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hotard</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaikh</span>  <span class="cit-name-given-names">FY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plemper</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A stabilized respiratory syncytial virus reverse genetics system amenable to recombination-mediated mutagenesis</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">434</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">129</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2012.09.022</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">23062737</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC3492879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirology%26rft.aulast%253DHotard%26rft.auinit1%253DA.%2BL.%26rft.volume%253D434%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D136%26rft.atitle%253DA%2Bstabilized%2Brespiratory%2Bsyncytial%2Bvirus%2Breverse%2Bgenetics%2Bsystem%2Bamenable%2Bto%2Brecombination-mediated%2Bmutagenesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2012.09.022%26rft_id%253Dinfo%253Apmid%252F23062737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2012.09.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23062737&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311183300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.42" data-doi="10.1016/j.chom.2012.04.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">BK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Antagonism of the interferoninduced OAS-RNase L pathway by murine coronavirus ns2 protein is required for virus replication and liver pathology</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">11</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">607</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2012.04.011</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">22704621</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC3377938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DZhao%26rft.auinit1%253DL.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D607%26rft.epage%253D616%26rft.atitle%253DAntagonism%2Bof%2Bthe%2Binterferon-induced%2BOAS-RNase%2BL%2Bpathway%2Bby%2Bmurine%2Bcoronavirus%2Bns2%2Bprotein%2Bis%2Brequired%2Bfor%2Bvirus%2Breplication%2Band%2Bliver%2Bpathology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2012.04.011%26rft_id%253Dinfo%253Apmid%252F22704621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2012.04.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22704621&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305821800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.43" data-doi="10.1038/s41467-017-00909-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruthel</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Replication defective viral genomes exploit a cellular pro-survival mechanism to establish paramyxovirus persistence</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">8</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">799</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41467-017-00909-6</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">28986577</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC5630589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D799%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-00909-6%26rft_id%253Dinfo%253Apmid%252F28986577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-00909-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28986577&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-349001v3.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-source">ggplot2: Elegant Graphics for Data Analysis</span>. <span class="cit-publ-loc">New York: Springer-Verlag</span>; <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-349001v3.45" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">9</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">357</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.1923</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID:</span><span class="cit-pub-id cit-pub-id-pmid">22388286</span><span class="cit-pub-id-sep cit-pub-id-pmid-sep">;</span> PubMed Central<span class="cit-pub-id-sep cit-pub-id-pmcid-sep"> </span><span class="cit-pub-id-scheme">PMCID:</span><span class="cit-pub-id cit-pub-id-pmcid">PMCPMC3322381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F02%2F15%2F349001.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/349001v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F349001v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/505487v2" title="Neural ensemble dynamics in dorsal motor cortex during speech in people with paralysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 15, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/349001v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/349001v3.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/349001v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/349001v3" class="active" data-icon-position="" data-hide-link-title="0">A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/176341" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_QSbbfaBm1Ov0p6OOPaXu1rGAHbkCm_mWjoop3MaPfY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="176341" id="node176341--2" data-pisa="biorxiv;349001v3" data-pisa-master="biorxiv;349001" data-seqnum="176341" data-apath="/biorxiv/early/2019/02/15/349001.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yan</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Eun Ji</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Sébastien A.</span> <span class="nlm-surname">Felt</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Louis J.</span> <span class="nlm-surname">Taylor</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Divyansh</span> <span class="nlm-surname">Agarwal</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gregory R.</span> <span class="nlm-surname">Grant</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Carolina B.</span> <span class="nlm-surname">López</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">349001; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/349001 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/349001v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;title=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;title=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;text=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;title=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;t=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;title=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;title=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="176341" id="node176341--3" data-pisa="biorxiv;349001v3" data-pisa-master="biorxiv;349001" data-seqnum="176341" data-apath="/biorxiv/early/2019/02/15/349001.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A specific sequence in the genome of respiratory syncytial virus regulates the generation of copy-back defective viral genomes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yan</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Eun Ji</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Sébastien A.</span> <span class="nlm-surname">Felt</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Louis J.</span> <span class="nlm-surname">Taylor</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Divyansh</span> <span class="nlm-surname">Agarwal</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gregory R.</span> <span class="nlm-surname">Grant</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Carolina B.</span> <span class="nlm-surname">López</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">349001; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/349001 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Sun%2C+Yan&amp;rft.au=Kim%2C+Eun+Ji&amp;rft.au=Felt%2C+S%C3%A9bastien+A.&amp;rft.au=Taylor%2C+Louis+J.&amp;rft.au=Agarwal%2C+Divyansh&amp;rft.au=Grant%2C+Gregory+R.&amp;rft.au=L%C3%B3pez%2C+Carolina+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/176341/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/176341/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/176341/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/176341/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/176341/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/176341/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/176341/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/176341/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/176341/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/176341/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/176341/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/176341/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;count=horizontal&amp;via=&amp;text=A%20specific%20sequence%20in%20the%20genome%20of%20respiratory%20syncytial%20virus%20regulates%20the%20generation%20of%20copy-back%20defective%20viral%20genomes&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/349001v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/349001v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/349001v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
