<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Med Virol</span><span class="iso-abbrev" title="iso-abbrev">J. Med. Virol</span><span class="doi" title="doi">10.1002/(ISSN)1096-9071</span><span class="publisher-id" title="publisher-id">JMV</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Medical Virology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0146-6615</span><span class="issn-epub" tagx="issn" title="issn-epub">1096-9071</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7166400">7166400</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/jmv.25688">10.1002/jmv.25688</a></span><span class="publisher-id" title="publisher-id">JMV25688</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Research Articles</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
The 2019â€�new coronavirus epidemic: Evidence for virus evolution</div> <div class="alt-title" title="alt-title">
BENVENUTO <div class="sc_UNKNOWN">
et al.</div> </div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benvenuto</span><span class="given-names" tagx="given-names" title="given-names">Domenico</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3833-2927</span><a href="#jmv25688-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giovanetti</span><span class="given-names" tagx="given-names" title="given-names">Marta</span></span><a href="#jmv25688-aff-0002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciccozzi</span><span class="given-names" tagx="given-names" title="given-names">Alessandra</span></span><a href="#jmv25688-aff-0001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spoto</span><span class="given-names" tagx="given-names" title="given-names">Silvia</span></span><a href="#jmv25688-aff-0003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Angeletti</span><span class="given-names" tagx="given-names" title="given-names">Silvia</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-7393-8732</span><a href="#jmv25688-aff-0004"><sup>4</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">s.angeletti@unicampus.it</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciccozzi</span><span class="given-names" tagx="given-names" title="given-names">Massimo</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3866-9239</span><a href="#jmv25688-aff-0002"><sup>2</sup></a></span></div><span class="citation_author_institution" id="jmv25688-aff-0001">[1], <span class="named-content" title="named-content">Unit of Medical Statistics and Molecular Epidemiology</span><span class="institution" title="institution">University Campus Bioâ€�Medico of Rome</span><span class="city" tagx="city" title="city">Rome</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="jmv25688-aff-0002">[2], <span class="named-content" title="named-content">LaboratÃ³rio de FlavivÃ­rus</span><span class="institution" title="institution">Instituto Oswaldo Cruz, FundaÃ§Ã£o Oswaldo Cruz</span><span class="city" tagx="city" title="city">Rio de Janeiro</span><span class="country" tagx="country" title="country">Brazil</span></span><span class="citation_author_institution" id="jmv25688-aff-0003">[3], <span class="named-content" title="named-content">Internal Medicine Unit</span><span class="institution" title="institution">University Campus Bioâ€�Medico of Rome</span><span class="city" tagx="city" title="city">Rome</span><span class="country" tagx="country" title="country">Italy</span></span><span class="citation_author_institution" id="jmv25688-aff-0004">[4], <span class="named-content" title="named-content">Unit of Clinical Laboratory Science</span><span class="institution" title="institution">University Campus Bioâ€�Medico of Rome</span><span class="city" tagx="city" title="city">Rome</span><span class="country" tagx="country" title="country">Italy</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Correspondence</b> Silvia Angeletti, Unit of Clinical Laboratory Science, University Campus Bioâ€�Medico of Rome, Rome 00128, Italy.<br /><br /> Email: <span class="email" tagx="email" title="email">s.angeletti@unicampus.it</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-2-2</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-4</span></span><span class="volume" tagx="volume" title="volume">92</span><span class="issue" tagx="issue" title="issue">4</span><span class="issue-id" tagx="issue-id" title="issue-id">10.1002/jmv.v92.4</span><span class="issue-title" tagx="issue-title" title="issue-title">2019 Novel Coronavirus Origin, Evolution, Disease, Biology and Epidemiology: Partâ€�I</span><span class="fpage" tagx="fpage" title="fpage">455</span><span class="lpage" tagx="lpage" title="lpage">459</span><span class="history" title="history"><span class="received" title="received">received: 2020-1-25</span><span class="accepted" title="accepted">accepted: 2020-1-27</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is being made freely available through PubMed Central as part of the COVID-19 public health emergency response. It can be used for unrestricted research re-use and analysis in any form or by any means with acknowledgement of the original source, for the duration of the public health emergency.</span></span></div><a class="self-uri" href="file:JMV-92-455.pdf" title="self-uri">file:JMV-92-455.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>There is a worldwide concern about the new coronavirus 2019â€�nCoV as a global public health threat. In this article, we provide a preliminary evolutionary and molecular epidemiological analysis of this new virus. A phylogenetic tree has been built using the 15 available whole genome sequences of 2019â€�nCoV, 12 whole genome sequences of 2019â€�nCoV, and 12 highly similar whole genome sequences available in gene bank (five from the severe acute respiratory syndrome, two from Middle East respiratory syndrome, and five from bat SARSâ€�like coronavirus). Fast unconstrained Bayesian approximation analysis shows that the nucleocapsid and the spike glycoprotein have some sites under positive pressure, whereas homology modeling revealed some molecular and structural differences between the viruses. The phylogenetic tree showed that 2019â€�nCoV significantly clustered with bat SARSâ€�like coronavirus sequence isolated in 2015, whereas structural analysis revealed mutation in Spike Glycoprotein and nucleocapsid protein. From these results, the new 2019â€�nCoV is distinct from SARS virus, probably trasmitted from bats after mutation conferring ability to infect humans.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Highlights</div> <p> </p><ul> <li> <p>An epidemic by a new Coronavirus, the 2019â€�nCoV is worrying worldwide.</p> </li> <li> <p>The epdiemic originated in Wuhan, China causing severe infections in some cases.</p> </li> <li> <p>The infection probably originated in bats.</p> </li> <li> <p>A mutation in the viral spike glycoprotein could have favured the crossâ€�sepcies tramsnimssion from bats to human.</p> </li> </ul> <p /> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
coronavirus</div> <div class="kwd" title="kwd">
epidemiology</div> <div class="kwd" title="kwd">
macromolecular design</div> <div class="kwd" title="kwd">
SARS coronavirus</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="cover-date" value="April 2020" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_JATSPMC version:5.8.0 mode:remove_FC converted:16.04.2020" /> </div> </div> <p content-type="self-citation"><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Benvenuto</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Giovanetti</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ciccozzi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Spoto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Angeletti</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ciccozzi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="mixed-article-title" title="mixed-article-title">The 2019â€�new coronavirus epidemic: Evidence for virus evolution</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">92</span>:<span class="fpage" tagx="fpage" title="fpage">455</span>â€"<span class="lpage" tagx="lpage" title="lpage">459</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25688">10.1002/jmv.25688</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31994738">31994738</a></span></span></p> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p>Silvia Angeletti and Massimo Ciccozzi contributed equally to this study.</p> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>The family <i>Coronaviridae</i> comprises a group of large, single, plusâ€�stranded RNA viruses isolated from several species, and it is previously known to cause the common cold and diarrheal illnesses in humans.<a href="#jmv25688-bib-0001"><sup>1</sup></a>, <a href="#jmv25688-bib-0002"><sup>2</sup></a> In 2003, a new coronavirus (severe acute respiratory syndrome coronavirus [SARSâ€�CoV]) was associated with the SARS outbreak.<a href="#jmv25688-bib-0001"><sup>1</sup></a>, <a href="#jmv25688-bib-0002"><sup>2</sup></a> Recently, a new coronavirus (2019â€�nCoV) has emerged in the region of Wuhan (China) as a cause of severe respiratory infection in humans. Since December 2019, different cases of pneumonia of unknown origin associated with permanence at the Wuhan market in China have been reported.<a href="#jmv25688-bib-0003"><sup>3</sup></a>, <a href="#jmv25688-bib-0004"><sup>4</sup></a> A new coronavirus, named 2019â€�nCoV, belonging to the <i>Orthocoronavirinae</i> subfamily, distinct from MERSâ€�CoV and SARSâ€�CoV, was described.<a href="#jmv25688-bib-0005"><sup>5</sup></a> To date, a total of 1975 pneumonia cases have been confirmed in China (the State Council Information Office in Beijing, capital of China, 26 January 2020).<a href="#jmv25688-bib-0006"><sup>6</sup></a>, <a href="#jmv25688-bib-0007"><sup>7</sup></a> Animal to human transmission is considered the origin of epidemics, as many patients declared to have visited a local fish and wild animal market in Wuhan in November. Quite recently, evidence has been gathered for the animal to the human and interhuman transmission of the virus.<a href="#jmv25688-bib-0007"><sup>7</sup></a>, <a href="#jmv25688-bib-0008"><sup>8</sup></a></p> <p>Although prompt diagnosis and patient isolation are the hallmarks for initial control of this new epidemic, molecular epidemiology, evolutionary models, and phylogenetic analysis can help estimate genetic variability and the evolutionary rate, which in turn have important implications for disease progression as well as for drug and vaccine development. In this short report, we provide a phylogenetic tree of the 2019â€�nCoV and identify sites of positive or negative selection pressure in distinct regions of the virus.</p> </div> <div class="materials-and-methods" title="materials-and-methods"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
MATERIAL AND METHODS</div> <p>The complete genomes of 15 2019â€�nCoV sequences have been downloaded from GISAID (<a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) and GenBank (<a href="http://www.ncbi.nlm.nih.gov/genbank/">http://www.ncbi.nlm.nih.gov/genbank/</a>). A dataset has been built using five highly similar sequences for SARS, two sequences for the Middle East respiratory syndrome (MERS), and five highly similar sequences for bat SARSâ€�like coronavirus. The percentage of similarity has been identified using a basic local alignment search tool (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>); eventually duplicated sequences have been excluded from the datasets. The dataset including 27 sequences has been aligned using multiple sequence alignment online tool<a href="#jmv25688-bib-0009"><sup>9</sup></a> and manually edited using BioEdit program v7.0.5.<a href="#jmv25688-bib-0010"><sup>10</sup></a></p> <p>Maximum likelihood (ML) methods were employed for the analyses because they allow for testing different phylogenetic hypotheses by calculating the probability of a given model of evolution generating the observed data and by comparing the probabilities of nested models by the likelihood ratio test. The bestâ€�fitting nucleotide substitution model was chosen by jModeltest software.<a href="#jmv25688-bib-0011"><sup>11</sup></a> ML tree was reconstructed using generalized timeâ€�reversible plus gamma distribution and invariant sites (+G+I) as an evolutionary model using MEGAâ€�X.<a href="#jmv25688-bib-0012"><sup>12</sup></a></p> <p>The adaptive evolution server (<a href="http://www.datamonkey.org/">http://www.datamonkey.org/</a>) was used to find eventual sites of positive or negative selection. For this purpose, the following test has been used: fast unconstrained Bayesian approximation (FUBAR).<a href="#jmv25688-bib-0013"><sup>13</sup></a> This test allowed us to infer the siteâ€�specific pervasive selection, the episodic diversifying selection across the region of interest, and to identify episodic selection at individual sites.<a href="#jmv25688-bib-0014"><sup>14</sup></a>Â The statistically significant positive or negative selection was based on <i>P</i> value less than .05.<a href="#jmv25688-bib-0014"><sup>14</sup></a></p> <p>Homology models have been built relying on the website SwissModel.<a href="#jmv25688-bib-0015"><sup>15</sup></a> Structural templates have been searched and validated using the software available within the SwissModel environment and HHPred.<a href="#jmv25688-bib-0016"><sup>16</sup></a> Homology models have been validated using the QMEAN tool.<a href="#jmv25688-bib-0017"><sup>17</sup></a> Threeâ€�dimensional structures have been analyzed and displayed using PyMOL.<a href="#jmv25688-bib-0018"><sup>18</sup></a> To map the structural variability of the N, E, S, and M regions of the virus and their sites under selection pressure, homology modeling has been applied to the sequence of 2019â€�nCoV.</p> </div> <div class="results" title="results"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
RESULTS</div> <p>The ML phylogenetic tree, performed on whole genome sequences, is represented in FigureÂ <a href="#jmv25688-fig-0001">1</a>. In the tree, MERS virus sequences formed a distinct clade (clade I) from Bat SARSâ€�like coronavirus, SARS virus, and the 2019â€�nCoV clustering together in clade II. This clade includes two different clusters: cluster IIa with Bat SARSâ€�like coronavirus and the 2019â€�nCoV sequences, and cluster IIb with the bat SARSâ€�like coronavirus and the SARS virus sequences. The 2019â€�nCoV is significantly and closely related only to the specific bat SARSâ€�like coronavirus isolated from <i>Rhinolophus sinicus</i> in 2015 in China (MG772934.1) (FigureÂ <a href="#jmv25688-fig-0001">1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Maximum likelihood tree of 2019â€�nCoV. Along the branches, an asterisk (*) indicates a statistical value from bootstrap (&amp;gt;97%)</p>   </div> <p>Regarding the FUBAR analysis performed on the N Region, significant (<i>P</i>â€‰<div class="â€‰.05)_UNKNOWN">
Pâ€‰<div class="â€‰.05)_UNKNOWN">
Pâ€‰<div class="â€‰.05)_UNKNOWN"> </div></div></div></p><p>On the S region, a significant (<i>P</i>â€‰<div class="â€‰.05)_UNKNOWN">
Pâ€‰<div class="â€‰.05)_UNKNOWN"> </div></div></p><p>No sites under positive selection sites have been found in the E and M regions.</p> <p>The N region of the 2019â€�nCoV homology model has been built using a SARS coronavirus nucleocapsid protein structure (2jw8.1) as the statistical test has shown that this is the most stable and similar model among the other possible structures (Ramachandran favored 90.52%, GMQE 0.17, QMEAN âˆ’2.43), whereas, for the S region, the 2019â€�nCoV homology model has been built using a SARS coronavirus spike glycoprotein (6acc.1) for the same reason (Ramachandran favored 87.87%, GMQE 0.83, QMEAN âˆ’3.14) (FigureÂ <a href="#jmv25688-fig-0003">3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Cartoon model of the structural superposition between the homology model of the 2019â€�nCoV in blue and the nucleocapsid protein of SARS coronavirus (PDB code 2jw8.1) in orange. The red circle highlights the presence of an alphaâ€�helix on the SARSâ€�CoV and the absence of the 2019â€�nCoV structure, and e positional difference of the betaâ€�sheets. SARS, severe acute respiratory syndrome</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Cartoon model of the structural superposition between the homology model of the 2019â€�nCoV in blue and the spike glycoprotein of SARS coronavirus (PDB code 6acc.1) in orange. The red circle highlights the presence of a variable region on the 2019â€�nCoV at the beginning of the protein, whereas the blue square highlights the presence of two betaâ€�sheets on the 2019â€�nCoV (401:KYR and 440:LND) that are not present on the SARSâ€�CoV structure; SARS, severe acute respiratory syndrome</p>   </div> <p>The further structural and molecular analysis of the Nucleocapsid region of the 2019â€�nCoV (MN908947.3) has highlighted that 2019â€�nCoV and the bat SARSâ€�like coronavirus (MG772934.) share the same aminoacidic sequence near the 309th position (SKQLQQ,) whereas the SARS reference genome has a different aminoacidic sequence (SRQLQN). The same results have been found in the 380th aminoacidic position (KADET for 2019â€�nCoV and bat SARSâ€�like coronavirus and KTDEA for the SARS reference genome). In particular, in this case, the 2019â€�nCoV has a polar amino acid, whereas the SARS has a nonpolar amino acid (FigureÂ <a href="#jmv25688-fig-0002">2</a>).</p> <p>Regarding the sites under positive selective pressure found on the spike glycoprotein, the results have shown that the 536th aminoacidic position in 2019â€�nCoV has an asparagine residue, whereas the bat SARSâ€�like coronavirus has a glutamine residue. The SARS virus, instead, has an aspartic acid residue. On the 644th aminoacidic position in 2019â€�nCoV sequence, there is a threonine residue, whereas the bat SARSâ€�like virus has a serine residue. The SARS virus, however, has an alanine residue.</p> </div> <div class="discussion" title="discussion"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
DISCUSSION</div> <p>The data reported above show that the new 2019â€�nCoV significantly clustered with a sequence from the bat SARSâ€�like coronavirus isolated in 2015. Moreover, in the phylogenetic tree, these two sequences are separated from the other bat SARSâ€�like coronavirus sequences, suggesting that this bat SARSâ€�like coronavirus is homologous and genetically more similar to the 2019â€�nCoV than to the other sequences of Bat SARSâ€�like coronavirus. This supports the hypothesis that the transmission chain began from the bat and reached the human. All other genomic sequences represented in the phylogenetic tree, also including SARS and MERS coronavirus, clustered separately, thus excluding the fact that the virus involved in the actual epidemic could belong to these subgenuses. The structural analysis of two important viral proteins, the nucleocapsid and the spikeâ€�like nucleoprotein (protein S), confirmed the significant similarity of the new coronavirus with the batâ€�like SARS coronavirus and its difference from SARS coronavirus.</p> <p>From the selective pressure and structural analysis, mutations of surface proteins, as the spike protein S, and of nucleocapsid N protein conferring stability to the viral particle have been shown. The viral spike protein is responsible for virus entry into the cell after by binding to a cell receptor and membrane fusion, two key steps in viral infection and pathogenesis. The N protein is a structural protein involved in virion assembly, playing a pivotal role in virus transcription and assembly efficiency. Mutation of these proteins could determine two important characteristics of the coronavirus isolated during the 2019â€�nCoV epidemic: a higher ability to infect and enhanced pathogenicity than the batâ€�like SARS coronavirus but lower pathogenicity than SARS coronavirus. These features can explain the 2019â€�nCoV zoonotic transmission and its initial lower severity than SARS epidemic. These results do not exclude the fact that further mutation due to positive selective pressure, led by the epidemic evolution, could favor an enhancement of pathogenicity and transmission of this novel virus.</p> <p>Recently, Ji et al<a href="#jmv25688-bib-0008"><sup>8</sup></a> described homologous recombination within the spike glycoprotein of 2019â€�nCoV favoring crossâ€�species transmission and suggested snake as probable virus reservoir for human infection because its resampling similarity codon usage bias is more similar to <i>Bungarus multicinctus</i> snake compared with other animals and humans. In a previous article, it has been proven that compositional properties, mutation pressure, natural selection, gene expression, and dinucleotides, affect the codon usage bias of <i>Bungarus</i> species.<a href="#jmv25688-bib-0019"><sup>19</sup></a></p> <p>These data, along with analysis from our study, enforcing bat origin of infection, could clarify the transmission dynamics of the 2019â€�nCoV, supporting infection control policy during the ongoing epidemic.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="jmv25688-bib-0001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">GÃ¼nther</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Preiser</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Identification of a novel coronavirus associated with severe acute respiratory syndrome</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">348</span>:<span class="fpage" tagx="fpage" title="fpage">1967</span>â€�<span class="lpage" tagx="lpage" title="lpage">1976</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12690091">12690091</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="mixed-article-title" title="mixed-article-title">Emerging coronaviruses: genome structure, replication, and pathogenesis</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25681">10.1002/jmv.25681</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">JFâ€�W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kok</span><span class="given-names" tagx="given-names" title="given-names">Kâ€�H</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating personâ€�toâ€�person transmission: a study of a family cluster</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30154-9">10.1016/S0140-6736(20)30154-9</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(20)30183-5">10.1016/S0140-6736(20)30183-5</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">A novel coronavirus from patients with pneumonia in China, 2019</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa2001017">10.1056/NEJMoa2001017</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Azhar</span><span class="given-names" tagx="given-names" title="given-names">EI</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Madani</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">The continuing 2019â€�nCoV epidemic threat of novel coronaviruses to global health â€� The latest 2019 novel coronavirus outbreak in Wuhan, China</span>. <span class="source" tagx="source" title="source">Int J Infect Dis</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">2020</span>(<span class="issue" tagx="issue" title="issue">91</span>):<span class="fpage" tagx="fpage" title="fpage">264</span>â€�<span class="lpage" tagx="lpage" title="lpage">266</span>.</span></li> <li tag="ref"><a name="jmv25688-bib-0007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stratton</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">YW</span></span>. <span class="mixed-article-title" title="mixed-article-title">Outbreak of pneumonia of unknown etiology in Wuhan China: the mystery and the miracle</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25678">10.1002/jmv.25678</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zai</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="mixed-article-title" title="mixed-article-title">Homologous recombination within the spike glycoprotein of the newly identified coronavirus may boost crossâ€�species transmission from snake to human</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25682">10.1002/jmv.25682</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Rozewicki</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yamada</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span>. <span class="mixed-article-title" title="mixed-article-title">MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization</span>. <span class="source" tagx="source" title="source">Brief Bioinform</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">1160</span>â€�<span class="lpage" tagx="lpage" title="lpage">1166</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bib/bbx108">10.1093/bib/bbx108</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28968734">28968734</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span>. <span class="mixed-article-title" title="mixed-article-title">BioEditA userâ€�friendly biological sequence alignment editor and analysis program for Windows 95/98/NT</span>. <span class="source" tagx="source" title="source">Nucleic Acids Symp Ser</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">41</span>:<span class="fpage" tagx="fpage" title="fpage">95</span>â€�<span class="lpage" tagx="lpage" title="lpage">98</span>.</span></li> <li tag="ref"><a name="jmv25688-bib-0011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Darriba</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Taboada</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Doallo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="mixed-article-title" title="mixed-article-title">jModelTest 2: more models, new heuristics and parallel computing</span>. <span class="source" tagx="source" title="source">Nat Methods</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">772</span>.</span></li> <li tag="ref"><a name="jmv25688-bib-0012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Knyaz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="mixed-article-title" title="mixed-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across computing platforms</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">35</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1547</span>â€�<span class="lpage" tagx="lpage" title="lpage">1549</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29722887">29722887</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Murrell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moola</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mabona</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">FUBAR: a fast, unconstrained bayesian approximation for inferring selection</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">1196</span>â€�<span class="lpage" tagx="lpage" title="lpage">1205</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23420840">23420840</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Murrell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wertheim</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Moola</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Weighill</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Scheffler</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kosakovsky Pond</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Detecting individual sites subject to episodic diversifying selection</span>. <span class="source" tagx="source" title="source">PLoS Genet</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="elocation-id" tagx="elocation-id" title="elocation-id">e1002764</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22807683">22807683</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Waterhouse</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bertoni</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bienert</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="mixed-article-title" title="mixed-article-title">SWISSâ€�MODEL: homology modelling of protein structures and complexes</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">46</span>(<span class="issue" tagx="issue" title="issue">W1</span>):<span class="fpage" tagx="fpage" title="fpage">W296</span>â€�<span class="lpage" tagx="lpage" title="lpage">W303</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29788355">29788355</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zimmermann</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Stephens</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nam</span><span class="given-names" tagx="given-names" title="given-names">SZ</span></span>. <span class="mixed-article-title" title="mixed-article-title">A completely reimplemented MPI Bioinformatics Toolkit with a new HHpred server at its core</span>. <span class="source" tagx="source" title="source">J Mol Biol</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">430</span>(<span class="issue" tagx="issue" title="issue">15</span>):<span class="fpage" tagx="fpage" title="fpage">2237</span>â€�<span class="lpage" tagx="lpage" title="lpage">2243</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29258817">29258817</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Benkert</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">KÃ¼nzli</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schwede</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="mixed-article-title" title="mixed-article-title">QMEAN server for protein model quality estimation</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">37</span>(<span class="issue" tagx="issue" title="issue">W</span>):<span class="fpage" tagx="fpage" title="fpage">W510</span>â€�<span class="lpage" tagx="lpage" title="lpage">W514</span>. 2009.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19429685">19429685</a></span></span></li> <li tag="ref"><a name="jmv25688-bib-0018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: SchrÃ¶dinger LLC</span> . <span class="mixed-article-title" title="mixed-article-title">The {PyMOL} Molecular Graphics System, Version~1.8</span>, <span class="year" tagx="year" title="year">2015</span></span></li> <li tag="ref"><a name="jmv25688-bib-0019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chakraborty</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nag</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Mazumder</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Uddin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="mixed-article-title" title="mixed-article-title">Codon usage pattern and prediction of gene expression level in Bungarus species</span>. <span class="source" tagx="source" title="source">Gene</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">604</span>:<span class="fpage" tagx="fpage" title="fpage">48</span>â€�<span class="lpage" tagx="lpage" title="lpage">60</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.gene.2016.11.023">10.1016/j.gene.2016.11.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27845207">27845207</a></span></span></li> </ul> </div> </div>  </body></html>