<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/04/860049.atom" />
<meta name="HW.pisa" content="biorxiv;860049v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Processing of the SARS-CoV pp1a/ab nsp7-10 region" />
<meta name="DC.Identifier" content="10.1101/860049" />
<meta name="DC.Date" content="2019-12-04" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Severe acute respiratory syndrome coronavirus (SARS-CoV) is the causative agent of a respiratory disease with a high case fatality rate. During the formation of the coronaviral replication/transcription complex (RTC), essential steps include processing of the conserved polyprotein nsp7-10 region by the main protease Mpro and subsequent complex formation of the released nsp’s. Here, we analyzed processing of the coronavirus nsp7-10 region using native mass spectrometry showing consumption of substrate, rise and fall of intermediate products and complexation. Importantly, there is a clear order of cleavage efficiencies, which is influenced by the polyprotein tertiary structure. Furthermore, the predominant product is an nsp7+8(2:2) hetero-tetramer with nsp8 scaffold. In conclusion, native MS, opposed to other methods, can expose the processing dynamics of viral polyproteins and the landscape of protein interactions in one set of experiments. Thereby, new insights into protein interactions, essential for generation of viral progeny, were provided, with relevance for development of antivirals." />
<meta name="DC.Contributor" content="Boris Krichel" />
<meta name="DC.Contributor" content="Sven Falke" />
<meta name="DC.Contributor" content="Rolf Hilgenfeld" />
<meta name="DC.Contributor" content="Lars Redecke" />
<meta name="DC.Contributor" content="Charlotte Uetrecht" />
<meta name="article:published_time" content="2019-12-04" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Processing of the SARS-CoV pp1a/ab nsp7-10 region" />
<meta name="citation_abstract" lang="en" content="1.1.&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Severe acute respiratory syndrome coronavirus (SARS-CoV) is the causative agent of a respiratory disease with a high case fatality rate. During the formation of the coronaviral replication/transcription complex (RTC), essential steps include processing of the conserved polyprotein nsp7-10 region by the main protease M&lt;sup&gt;pro&lt;/sup&gt; and subsequent complex formation of the released nsp’s. Here, we analyzed processing of the coronavirus nsp7-10 region using native mass spectrometry showing consumption of substrate, rise and fall of intermediate products and complexation. Importantly, there is a clear order of cleavage efficiencies, which is influenced by the polyprotein tertiary structure. Furthermore, the predominant product is an nsp7+8(2:2) hetero-tetramer with nsp8 scaffold. In conclusion, native MS, opposed to other methods, can expose the processing dynamics of viral polyproteins and the landscape of protein interactions in one set of experiments. Thereby, new insights into protein interactions, essential for generation of viral progeny, were provided, with relevance for development of antivirals.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;860049v2" />
<meta name="citation_id" content="860049v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/860049v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/860049v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/860049v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049.full.pdf" />
<meta name="citation_doi" content="10.1101/860049" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="860049" />
<meta name="citation_author" content="Boris Krichel" />
<meta name="citation_author_institution" content="Heinrich Pette Institute, Leibniz Institute for Experimental Virology" />
<meta name="citation_author" content="Sven Falke" />
<meta name="citation_author_institution" content="University of Hamburg, Institut für Biochemie und Molekularbiologie" />
<meta name="citation_author" content="Rolf Hilgenfeld" />
<meta name="citation_author_institution" content="University of Lübeck, Institute of Biochemistry, Center for Structural and Cell Biology in Medicine" />
<meta name="citation_author_institution" content="German Center for Infection Research (DZIF), Hamburg-Lübeck-Borstel-Riems Site, University of Lübeck" />
<meta name="citation_author" content="Lars Redecke" />
<meta name="citation_author_institution" content="University of Lübeck, Institute of Biochemistry, Center for Structural and Cell Biology in Medicine" />
<meta name="citation_author_institution" content="Deutsches Elektronen Synchrotron (DESY)" />
<meta name="citation_author" content="Charlotte Uetrecht" />
<meta name="citation_author_institution" content="Heinrich Pette Institute, Leibniz Institute for Experimental Virology" />
<meta name="citation_author_institution" content="European XFEL GmbH" />
<meta name="citation_author_email" content="charlotte.uetrecht@xfel.eu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1991-7922" />
<meta name="citation_reference" content="Drosten, C., et al., Identification of a novel coronavirus in patients with severe acute respiratory syndrome. New. Engl. J. Med., 2003. 348(20): p. 1967–1976." />
<meta name="citation_reference" content="Hilgenfeld, R. and M., Peiris, From SARS to MERS: 10 years of research on highly pathogenic human coronaviruses. Antiviral research, 2013. 100(1): p. 286–295." />
<meta name="citation_reference" content="Brierley, I., P., Digard, and S.C., Inglis, Characterization of an efficient coronavirus ribosomal frameshifting signal: requirement for an RNA pseudoknot. Cell, 1989. 57(4): p. 537–47." />
<meta name="citation_reference" content="Marra, M.A., et al., The Genome sequence of the SARS-associated coronavirus. Science, 2003. 300(5624): p. 1399–404." />
<meta name="citation_reference" content="Thiel, V., et al., Mechanisms and enzymes involved in SARS coronavirus genome expression. J. Gen. Virol., 2003. 84(Pt 9): p. 2305–15." />
<meta name="citation_reference" content="Snijder, E.J., et al., Unique and conserved features of genome and proteome of SARS-coronavirus, an early split-off from the coronavirus group 2 lineage. J Mol Biol, 2003. 331(5): p. 991–1004." />
<meta name="citation_reference" content="Gorbalenya, A.E., et al., Nidovirales: evolving the largest RNA virus genome. Virus Res., 2006. 117(1): p. 17–37." />
<meta name="citation_reference" content="Hsu, M.F., et al., Mechanism of the maturation process of SARS-CoV 3CL protease. J Biol Chem, 2005. 280(35): p. 31257–66." />
<meta name="citation_reference" content="Ziebuhr, J. and S.G., Siddell, Processing of the human coronavirus 229E replicase polyproteins by the virus-encoded 3C-like proteinase: identification of proteolytic products and cleavage sites common to pp1a and pp1ab. J Virol, 1999. 73(1): p. 177–85." />
<meta name="citation_reference" content="Anand, K., et al., Coronavirus main proteinase (3CLpro) structure: basis for design of anti-SARS drugs. Science, 2003. 300(5626): p. 1763–7." />
<meta name="citation_reference" content="Yang, H., et al., The crystal structures of severe acute respiratory syndrome virus main protease and its complex with an inhibitor. Proc. Natl. Acad. Sci. U.S.A., 2003. 100(23): p. 13190–13195." />
<meta name="citation_reference" content="Sawicki, S.G., et al., Functional and genetic analysis of coronavirus replicase-transcriptase proteins. PLoS Pathog, 2005. 1(4): p. e39." />
<meta name="citation_reference" content="Deming, D.J., et al., Processing of open reading frame 1a replicase proteins nsp7 to nsp10 in murine hepatitis virus strain A59 replication. J Virol, 2007. 81(19): p. 10280–91." />
<meta name="citation_reference" content="Fan, K., et al., Biosynthesis, purification, and substrate specificity of severe acute respiratory syndrome coronavirus 3C-like proteinase. J Biol Chem, 2004. 279(3): p. 1637–42." />
<meta name="citation_reference" content="Wu, A., et al., Prediction and biochemical analysis of putative cleavage sites of the 3C-like protease of Middle East respiratory syndrome coronavirus. Virus Res., 2015. 208: p. 56–65." />
<meta name="citation_reference" content="Hegyi, A. and J., Ziebuhr, Conservation of substrate specificities among coronavirus main proteases. J. Gen. Virol., 2002. 83(Pt 3): p. 595–9." />
<meta name="citation_reference" content="Falke, S., (Doctoral Thesis) Coronaviral Polyprotein Nsp7-10: Proteolytic Processing and Dynamic Interactions within the Transcriptase/Replicase Complex. 2014, Staats- und Universitätsbibliothek Hamburg." />
<meta name="citation_reference" content="Subissi, L., et al., One severe acute respiratory syndrome coronavirus protein complex integrates processive RNA polymerase and exonuclease activities. Proc. Natl. Acad. Sci. U.S.A., 2014. 111(37): p. E3900–E3909." />
<meta name="citation_reference" content="te Velthuis, A.J., S.H. van den Worm, and E.J., Snijder, The SARS-coronavirus nsp7+nsp8 complex is a unique multimeric RNA polymerase capable of both de novo initiation and primer extension. Nucleic Acids Res., 2012. 40(4): p. 1737–47." />
<meta name="citation_reference" content="Li, S., et al., New nsp8 isoform suggests mechanism for tuning viral RNA synthesis. Protein Cell, 2010. 1(2): p. 198–204." />
<meta name="citation_reference" content="Xiao, Y., et al., Nonstructural proteins 7 and 8 of feline coronavirus form a 2:1 heterotrimer that exhibits primer-independent RNA polymerase activity. J Virol, 2012. 86(8): p. 4444–54." />
<meta name="citation_reference" content="Kirchdoerfer, R.N. and A.B. Ward, Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors. Nat Commun, 2019. 10(1): p. 2342." />
<meta name="citation_reference" content="Chandler, S.A. and J.L. Benesch, Mass spectrometry beyond the native state. Curr Opin Chem Biol, 2018. 42: p. 130–137." />
<meta name="citation_reference" content="Dülfer, J., et al., Structural mass spectrometry goes viral. Adv. Virus Res., 2019. 105: p. 189–238." />
<meta name="citation_reference" content="Grum-Tokars, V., et al., Evaluating the 3C-like protease activity of SARS-Coronavirus: recommendations for standardized assays for drug discovery. Virus Res., 2008. 133(1): p. 63–73." />
<meta name="citation_reference" content="Huang, C., et al., 3C-like proteinase from SARS coronavirus catalyzes substrate hydrolysis by a general base mechanism. Biochemistry, 2004. 43(15): p. 4568–74." />
<meta name="citation_reference" content="Fan, K., et al., The substrate specificity of SARS coronavirus 3C-like proteinase. Biochem. Biophys. Res. Commun., 2005. 329(3): p. 934–40." />
<meta name="citation_reference" content="Chuck, C.-P., et al., Profiling of substrate specificity of SARS-CoV 3CLpro. PloS one, 2010. 5(10): p. e13197." />
<meta name="citation_reference" content="Ponnusamy, R., et al., Variable oligomerization modes in coronavirus non-structural protein 9. J Mol Biol, 2008. 383(5): p. 1081–96." />
<meta name="citation_reference" content="Zhai, Y., et al., Insights into SARS-CoV transcription and replication from the structure of the nsp7-nsp8 hexadecamer. Nat. Struct. Mol. Biol., 2005. 12(11): p. 980–6." />
<meta name="citation_reference" content="Xue, X., et al., Production of authentic SARS-CoV M pro with enhanced activity: application as a novel tag-cleavage endopeptidase for protein overproduction. Journal of molecular biology, 2007. 366(3): p. 965–975." />
<meta name="citation_reference" content="Liu, Y., et al., Use of a fluorescence plate reader for measuring kinetic parameters with inner filter effect correction. Analytical biochemistry, 1999. 267(2): p. 331–335." />
<meta name="citation_reference" content="van den Heuvel, R.H., et al., Improving the performance of a quadrupole time-of-flight instrument for macromolecular mass spectrometry. Analytical chemistry, 2006. 78(21): p. 7473–83." />
<meta name="citation_reference" content="Morgner, N. and C.V. Robinson, Massign: an assignment strategy for maximizing information from the mass spectra of heterogeneous protein assemblies. Analytical chemistry, 2012. 84(6): p. 2939–48." />
<meta name="twitter:title" content="Processing of the SARS-CoV pp1a/ab nsp7-10 region" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Severe acute respiratory syndrome coronavirus (SARS-CoV) is the causative agent of a respiratory disease with a high case fatality rate. During the formation of the coronaviral replication/transcription complex (RTC), essential steps include processing of the conserved polyprotein nsp7-10 region by the main protease Mpro and subsequent complex formation of the released nsp’s. Here, we analyzed processing of the coronavirus nsp7-10 region using native mass spectrometry showing consumption of substrate, rise and fall of intermediate products and complexation. Importantly, there is a clear order of cleavage efficiencies, which is influenced by the polyprotein tertiary structure. Furthermore, the predominant product is an nsp7+8(2:2) hetero-tetramer with nsp8 scaffold. In conclusion, native MS, opposed to other methods, can expose the processing dynamics of viral polyproteins and the landscape of protein interactions in one set of experiments. Thereby, new insights into protein interactions, essential for generation of viral progeny, were provided, with relevance for development of antivirals." />
<meta name="og-title" property="og:title" content="Processing of the SARS-CoV pp1a/ab nsp7-10 region" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/860049v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Severe acute respiratory syndrome coronavirus (SARS-CoV) is the causative agent of a respiratory disease with a high case fatality rate. During the formation of the coronaviral replication/transcription complex (RTC), essential steps include processing of the conserved polyprotein nsp7-10 region by the main protease Mpro and subsequent complex formation of the released nsp’s. Here, we analyzed processing of the coronavirus nsp7-10 region using native mass spectrometry showing consumption of substrate, rise and fall of intermediate products and complexation. Importantly, there is a clear order of cleavage efficiencies, which is influenced by the polyprotein tertiary structure. Furthermore, the predominant product is an nsp7+8(2:2) hetero-tetramer with nsp8 scaffold. In conclusion, native MS, opposed to other methods, can expose the processing dynamics of viral polyproteins and the landscape of protein interactions in one set of experiments. Thereby, new insights into protein interactions, essential for generation of viral progeny, were provided, with relevance for development of antivirals." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-04" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/860049v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/860049v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/860049v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/860049v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1020108" />
    <title>Processing of the SARS-CoV pp1a/ab nsp7-10 region | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"u7drtiinwPYT6DH_an92heqCTvReOLjMWS6sHWL6fbQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1020108","apath":"\/biorxiv\/early\/2019\/12\/04\/860049.atom","pisa":"biorxiv;860049v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;860049v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/860049v2","title":"Processing of the SARS-CoV pp1a\/ab nsp7-10 region","identifier":"node\/1020108"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Mh33D38NCI5WkRRf6Y5ibf9pR+xe50TpTKsVXvizf8njJhVkcVNboS4ESl54a2kAju8S\\\\\\\/JcV9Sx210w7OKcBd1jlrurKDPNWlNwD05Yf1KmLpBcb24MW5urXBcO9Qp\\\\\\\/Ge4QHB1t1IDVa3l7wRPm3rWrllbmhGVA6meidDZTgGhuMBTU6U2KTr3L00F7s7U0TA7aAjrduetCjnQOPdkid4TsPU+8J67lq8wpBrMAwr0ySqW4aiCShri5uqUx7+id8jQ+XibcOq1Qj9zBTBpPe2\\\\\\\/61fFS6Bcyf0dM6qL2I9W30+aGxQkosJH+uJgRG2WrWr9ef+Ld5CMsk8GDlqStRSHl8OvCsedSM\\\\\\\/7r7NwguHzx1KQklD3Av0MFcIPEcrIUMGr0zxGZxDyugFpbeose0JorGcXTcfKjLa++xhe7qwtOL1AsBQzolazcgdDqHF6U61yMAEx+rBZxKWf3G5ru1HSyfeELrmANMDODf65VvxYZLcyeaI\\\\\\\/ER64D4yIu09m9N9WjF8nTYVa+57gJbc5ToBhlhXbuFBe5yEIref40B4qDH8AwlvtxyvZYJE7\\\\\\\/Ug0gaPDUzmV\\\\\\\/w\\\\\\\/qI78kOzDhia8k0B2OIestiRl\\\\\\\/iE6vbYvpUfyk0lSJdsFTSWZpxBmWCi0CPJ9CELLJ0gdjiVbaZmHaqNKc5arLI1LQjiM4xtWQt3pumSiprADHqKfTnCQC9ZinRzgw1SSMNdwdLdCke4o3gp0MW4XxSa\\\\\\\/F7+p+K3txWIpVUOreZDuCs5jEDCAoOCyc1WeOHUREQNuRukKYlvVkMnPiP\\\\\\\/ctzFHdYg4DW3CW1UHcVE0\\\\\\\/AcB5SJCIasO9JXQM5N+LAolXsZGicmITjG9O8kwexsSBlYLljq\\\\\\\/VMX1vQxb9fcJ\\\\\\\/G28ShzfjvJb0365IOUoeQaNN0LtDc4OnwKCfChwCvsbcF0+QmE6tuQsLKUfQP\\\\\\\/pe67Kz2BEeEEw7UUPYSQM0VR5lbLVXKV5ULsFxfs88WRluOnr\\\\\\\/sKs3FAZj4VRDPmBWzGqA0aDa95CHe9OjRu\\\\\\\/KbDlhz+bzbrT8lWR2JwujZLGy09xmKNMXMXK5yIggM3MHKAhiabxZAy8DkAb3jgJPQfeyuyALpBqORT\\\\\\\/51dWmRfiUlISfHUZ3BzEvYlfYo5r3CaneLeOK\\\\\\\/\\\\\\\/VeGYuX5wLDHQJadIBHF4f1uRQLpV9enLfMgB1jJ+MVEtWWEWOCiMrXACLW2JHy8LgkoolEr1k8YNq0hRmSkBARK0pdPP050DgGo\\\\\\\/aZ2j3gLqORLUFz7qBT9y\\\\\\\/oB701PJJV+cFFTKeW5v8eSo00HDp30hVqFRj2J3LpChW\\\\\\\/zGXoPal1MmRpgNK6+PeqsdKxEjiPTp\\\u0022,\\\u0022iv\\\u0022:\\\u002233IyxFbvM0xhcoK11nUQGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022cb6794e798b52853813e39d8fdf4fdbed7bf381bec5a0ef773b379347e9e8628\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/860049v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1020108 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/860049v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CidIGQClisVpbtgXfEnDdU5ZJHoiVQhzoQqt_irFOfs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1020108" id="node1020108" data-pisa="biorxiv;860049v2" data-pisa-master="biorxiv;860049" data-apath="/biorxiv/early/2019/12/04/860049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Processing of the SARS-CoV pp1a/ab nsp7-10 region</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Krichel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sven</span> <span class="nlm-surname">Falke</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Rolf</span> <span class="nlm-surname">Hilgenfeld</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lars</span> <span class="nlm-surname">Redecke</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1991-7922" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Charlotte</span> <span class="nlm-surname">Uetrecht</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/860049 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1020108" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Boris Krichel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Heinrich Pette Institute, Leibniz Institute for Experimental Virology</span>, Martinistraße 52, 20251 Hamburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boris%2BKrichel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krichel%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoris%2BKrichel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sven Falke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>University of Hamburg, Institut für Biochemie und Molekularbiologie</span>, Martin-Luther-King-Platz 6, 20146 Hamburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sven%2BFalke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falke%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASven%2BFalke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rolf Hilgenfeld </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>University of Lübeck, Institute of Biochemistry, Center for Structural and Cell Biology in Medicine</span>, Ratzeburger Allee 160, 23562 Lübeck, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>German Center for Infection Research (DZIF), Hamburg-Lübeck-Borstel-Riems Site, University of Lübeck</span>, Ratzeburger Allee 160, 23562 Lübeck, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rolf%2BHilgenfeld%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hilgenfeld%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARolf%2BHilgenfeld%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lars Redecke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>University of Lübeck, Institute of Biochemistry, Center for Structural and Cell Biology in Medicine</span>, Ratzeburger Allee 160, 23562 Lübeck, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Deutsches Elektronen Synchrotron (DESY)</span>, Notkestraße 85, 22607 Hamburg, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BRedecke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Redecke%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BRedecke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Charlotte Uetrecht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Heinrich Pette Institute, Leibniz Institute for Experimental Virology</span>, Martinistraße 52, 20251 Hamburg, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>European XFEL GmbH</span>, Holzkoppel 4, 22869 Schenefeld, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BUetrecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Uetrecht%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharlotte%2BUetrecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1991-7922" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charlotte Uetrecht</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:charlotte.uetrecht@xfel.eu" class="" data-icon-position="" data-hide-link-title="0">charlotte.uetrecht@xfel.eu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/860049v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1020108" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1020108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/860049v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1020108" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1020108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/860049v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1020108" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1020108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/860049v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1020108" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1020108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/860049v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1020108" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1020108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/860049v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1020108" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1020108/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">1.1. Abstract</h2><p id="p-2">Severe acute respiratory syndrome coronavirus (SARS-CoV) is the causative agent of a respiratory disease with a high case fatality rate. During the formation of the coronaviral replication/transcription complex (RTC), essential steps include processing of the conserved polyprotein nsp7-10 region by the main protease M<sup>pro</sup> and subsequent complex formation of the released nsp’s. Here, we analyzed processing of the coronavirus nsp7-10 region using native mass spectrometry showing consumption of substrate, rise and fall of intermediate products and complexation. Importantly, there is a clear order of cleavage efficiencies, which is influenced by the polyprotein tertiary structure. Furthermore, the predominant product is an nsp7+8(2:2) hetero-tetramer with nsp8 scaffold. In conclusion, native MS, opposed to other methods, can expose the processing dynamics of viral polyproteins and the landscape of protein interactions in one set of experiments. Thereby, new insights into protein interactions, essential for generation of viral progeny, were provided, with relevance for development of antivirals.</p></div><div class="section" id="sec-1"><h2 class="">1.2. Introduction</h2><p id="p-4">The human pathogenic potential of zoonotic infections by coronaviruses was exposed by the discovery of severe acute respiratory syndrome coronavirus (SARS-CoV) as causative agent of the SARS epidemic in 2003 [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. The replicase gene ORF 1ab occupies two thirds of the CoV single-strand (+)-sense RNA genome. Initially, ORF1ab is directly translated into either replicase polyprotein pp1a (nsp1-11) or pp1ab (nsp1-16), depending on a ribosomal (−1)-frameshift [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. Subsequently, the polyproteins undergo proteolytic processing into 11 or 16 individual nsp’s. Eventually, the nsp’s take part in forming the RTC, a membrane-anchored, highly dynamic protein-RNA complex facilitating the replicative processes [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>].</p><p id="p-5">The main chymotrypsin-like protease (M<sup>pro</sup>; nsp5) facilitates processing of the C-terminal part of the polyprotein [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Following maturation by auto-cleavage, two protomers of M<sup>pro</sup> associate into one active dimeric unit that specifically targets the cleavage sites at the nsp inter-domain junctions of nsp5-11 or nsp5-16. These sites primarily contain LQ↓S or LQ↓A residues at the positions P2, P1 and P1′, but strictly conserved is only Q at P1 [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Being a possible drug target, structure and function of this protease were extensively investigated [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>].</p><p id="p-6">The timely coordinated processing of pp1a is crucial for replication [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. This was specifically shown for the processing order of the nsp7-10 region [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. In general, the order of processing is directly dependent on the substrate specificity of M<sup>pro</sup>, which has been tested in peptide-based biochemical assays [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. However, peptide assays interrogate merely the substrate specificity for the isolated amino-acid sequence but disregard the polyprotein's structural layout, i.e. overall conformation or accessibility.</p><p id="p-7">Upon processing of the nsp7-10 region by M<sup>pro</sup>, the four small proteins nsp7, nsp8, nsp9 and nsp10 are released [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. They can form functional complexes with CoV core enzymes and thereby stimulate replication. Most importantly, a complex of nsp7+8 acts as processivity factor for nsp12, the RNA-dependent RNA-polymerase (RdRp) [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. Particularly, various oligomeric states of nsp7+8 have been reported [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>–<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>].</p><p id="p-8">Detailed studies of nsp processing and complex formation require a method that follows these dynamic processes at a molecular level. Native mass spectrometry (MS) probes and detects protein complexes in mixtures, while keeping their non-covalent interactions intact through nano-electrospray ionization (nanoESI) [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Furthermore, collision-induced dissociation (CID) enables to decipher the stoichiometry and topology of natively sprayed protein complexes, by dissociating non-covalent interactions <em>in vacuo</em>. Thereby, it allows to reveal the landscape of protein species from a heterogeneous sample solution and to observe dynamic processes over time.</p><p id="p-9">Herein, we analyzed the processing of SARS-CoV nsp7-10 and complex formation of its sequentially released nsp’s. First, the substrate specificity of SARS-CoV M<sup>pro</sup> for the cleavage sites within nsp7-10 was determined in an assay with FRET-labelled peptides. Then, we explored native MS to reveal time-resolved nsp release and complex formation upon processing of a full-length SARS-CoV nsp7-10 substrate. Specifically, the analysis via CID-MS gave insights into a preferred complex of SARS-CoV nsp7+8.</p></div><div class="section" id="sec-2"><h2 class="">1.3. Results</h2><div id="sec-3" class="subsection"><h3>1.3.1. Specific substrate efficiencies from FRET peptide assay</h3><p id="p-10">Initially, the substrate specificity of SARS-CoV M<sup>pro</sup> for nsp7-10 inter-domain junction cleavage was studied by means of FRET peptide substrates (FPS’s). The substrates FPS7-8, FPS8-9 and FPS9-10 represented the cleavage sites nsp7-8, nsp8-9 and nsp9-10, respectively. Additionally, FPS4-5 served as a positive control analogous to the highly efficient N-terminal auto-cleavage site nsp4-5 of SARS-CoV M<sup>pro</sup> (nsp5). The FPS’s encompassed 12 amino-acid residues spanning positions P6 to P6’ around the cleavage site and were labelled with a fluorophore and a quencher at their N- and C-terminus, respectively. Substrate specificities were determined by monitoring the increasing fluorescence upon M<sup>pro</sup> cleavage of the FPS (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and Table S 4). As expected, the assay showed FPS4-5 to be most efficiently cleaved amongst the FPS’s tested here, with an apparent efficiency (K<sub>cat</sub>/K<sub>M</sub>) of 0.105±0.005 μM<sup>−1</sup>min<sup>−1</sup>. Compared to FPS4-5 (100 %), substrates FPS7-8 (17.1 %) and FPS9-10 (51.8 %) were less efficiently cleaved while FPS8-9 (0.1 %) was almost non-cleavable. The absolute and relative substrate efficiencies determined here agree with results of similar experiments [<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. Considering that short peptides represent merely the primary structure of the cleavage site, more native-like substrates were prepared and tested to verify these results obtained with the FPS’s.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Process curves of Mpro (0.5 &#x3BC;M) with different concentrations of FRET peptide substrates (FPS) in 50 mM HEPES, 10 % Glycerol pH 7.5. The slopes correspond to the apparent kcat/KM as given in the legend." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1902911622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Process curves of M&lt;sup&gt;pro&lt;/sup&gt; (0.5 &#x3BC;M) with different concentrations of FRET peptide substrates (FPS) in 50 mM HEPES, 10 % Glycerol pH 7.5. The slopes correspond to the apparent k&lt;sub&gt;cat&lt;/sub&gt;/K&lt;sub&gt;M&lt;/sub&gt; as given in the legend.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F1.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F1.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-11" class="first-child">Process curves of M<sup>pro</sup> (0.5 μM) with different concentrations of FRET peptide substrates (FPS) in 50 mM HEPES, 10 % Glycerol pH 7.5. The slopes correspond to the apparent k<sub>cat</sub>/K<sub>M</sub> as given in the legend.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Processing of nsp7-10</h3><p id="p-12">Next, we studied the substrate specificity of SARS-M<sup>pro</sup> for folded proteins, which exhibit the structural environment of the cleavage sites beyond the amino-acid sequence. Therefore, full-length SARS-CoV His-nsp7-10 was recombinantly produced in <em>E.coli</em> and purified. This protein construct had a non-cleavable N-terminal His tag to reduce number of cleavage product species and thereby simplified interpretation of the mass spectra.</p><p id="p-13">To monitor the processing dynamics, SARS-CoV His-nsp7-10 was cleaved by SARS-CoV M<sup>pro</sup> in a test tube and samples were subjected to native MS. Cleavage was performed in a nano ESI compatible ammonium acetate (NH<sub>4</sub>OAc) solution at pH 8.0, which proved equally suitable to maintain protease activity compared to buffers with NaCl (Figure S 1). Processing products were observed as peaks in the mass spectra and assigned to protein mass species (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Table S 3). When acquiring mass spectra at different times during processing, signals of three protein categories dominated successively, in the beginning full-length nsp7-10 substrate and M<sup>pro</sup> (~1h); subsequently, intermediate products (~6 h), representing partially cleaved substrate; and in the end mature products encompassing nsp monomers and protein complexes (~20 h) (Figure S 6). Covalent and non-covalent products were distinguished via CID-MS/MS.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS nsp7-10 processing in native MS. Peak assignment in native MS after 6h. Mass species, charge states and their m/z are labelled. For processing, 1.25 &#x3BC;M SARS-CoV Mpro and 12.5 &#x3BC;M SARS-CoV His-nsp7-10 (~1:10 ratio) were incubated at 4&#xB0;C in 250 mM NH4OAc, 1 mM DTT at pH 8.0. The components were mixed and after a defined incubation time, samples were injected into an electrospray capillary and mass spectra were acquired." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1902911622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SARS nsp7-10 processing in native MS. Peak assignment in native MS after 6h. Mass species, charge states and their &lt;em&gt;m/z&lt;/em&gt; are labelled. For processing, 1.25 &#x3BC;M SARS-CoV M&lt;sup&gt;pro&lt;/sup&gt; and 12.5 &#x3BC;M SARS-CoV His-nsp7-10 (~1:10 ratio) were incubated at 4&#xB0;C in 250 mM NH&lt;sub&gt;4&lt;/sub&gt;OAc, 1 mM DTT at pH 8.0. The components were mixed and after a defined incubation time, samples were injected into an electrospray capillary and mass spectra were acquired.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F2.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F2.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-14" class="first-child">SARS nsp7-10 processing in native MS. Peak assignment in native MS after 6h. Mass species, charge states and their <em>m/z</em> are labelled. For processing, 1.25 μM SARS-CoV M<sup>pro</sup> and 12.5 μM SARS-CoV His-nsp7-10 (~1:10 ratio) were incubated at 4°C in 250 mM NH<sub>4</sub>OAc, 1 mM DTT at pH 8.0. The components were mixed and after a defined incubation time, samples were injected into an electrospray capillary and mass spectra were acquired.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The order of reactions became evident from monitoring peak intensities over time. The relative signal intensity of M<sup>pro</sup> remained stable while nsp7-10 decreased (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3 A</a>), indicating that M<sup>pro</sup> depleted nsp7-10. At 20 h, no signal of nsp7-10 was detected anymore and therefore, cleavage was considered complete.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS-CoV nsp7-10 processing monitored by native MS: Signal over time of protease, substrate and products. (A) Substrate and enzyme. (B) Mass fractions over time show the order of nsp release. (C) Intermediate products. Error bars depict standard deviation (N=3). Time points (AVG&#xB1;SD, N=3): 0.25&#xB1;0.1 h, 1.2&#xB1;0.1 h, 3.3&#xB1;0.2 h, 5.3&#xB1;0.3 h, 6.4&#xB1;0.3 h&#xB1;0.5 h. (D) Schematic illustration of cleavage order and efficiency." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1902911622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SARS-CoV nsp7-10 processing monitored by native MS: Signal over time of protease, substrate and products. &lt;strong&gt;(A)&lt;/strong&gt; Substrate and enzyme. &lt;strong&gt;(B)&lt;/strong&gt; Mass fractions over time show the order of nsp release. &lt;strong&gt;(C)&lt;/strong&gt; Intermediate products. Error bars depict standard deviation (&lt;em&gt;N&lt;/em&gt;=3). Time points (AVG&#xB1;SD, &lt;em&gt;N&lt;/em&gt;=3): 0.25&#xB1;0.1 h, 1.2&#xB1;0.1 h, 3.3&#xB1;0.2 h, 5.3&#xB1;0.3 h, 6.4&#xB1;0.3 h&#xB1;0.5 h. &lt;strong&gt;(D)&lt;/strong&gt; Schematic illustration of cleavage order and efficiency.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F3.medium.gif" width="217" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F3.medium.gif" width="217" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-16" class="first-child">SARS-CoV nsp7-10 processing monitored by native MS: Signal over time of protease, substrate and products. <strong>(A)</strong> Substrate and enzyme. <strong>(B)</strong> Mass fractions over time show the order of nsp release. <strong>(C)</strong> Intermediate products. Error bars depict standard deviation (<em>N</em>=3). Time points (AVG±SD, <em>N</em>=3): 0.25±0.1 h, 1.2±0.1 h, 3.3±0.2 h, 5.3±0.3 h, 6.4±0.3 h±0.5 h. <strong>(D)</strong> Schematic illustration of cleavage order and efficiency.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">To obtain the sequence of nsp release, the ratio of the mature nsp monomers was followed over time. However, some non-covalent interactions of the nsp’s introduced a bias to the relative peak intensities among monomers (Figure S 5). To correct for this bias, signal intensities were converted into mass fractions (MFs), which include the relative signal of each released subunit over time (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S 2</a>). Thereby, MFs demonstrated a processing order of the nsp9-10 site first, the nsp8-9 site second, and the nsp7-8 site third (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3 B</a>). MFs were similar at the end-point of processing (~20 h) showing that the individual nsp’s were all cleaved off nsp7-10 (Figure S 5).</p><p id="p-18">To determine the relative cleavage efficiencies in the full-length substrate, we used the relative MFs between nsp10, nsp9, nsp8 and nsp7 at 3.3 h (100%, 66%, 36 % and 36 %, respectively). These values suggest decreasing cleavage efficiency from the C- to the N-terminus of SARS nsp7-10. Consistently, the cleavage pattern of the intermediate products revealed that nsp7-10 is predominantly cleaved to nsp7-9 and only then to nsp7-8 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3 C</a>). The time-course of processing of the SARS-CoV full-length substrate as determined here via native MS corresponded well with results from SDS-PAGE using the shorter SARS-CoV His-nsp7-9 substrate (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3 C</a>, Figure S 7). Additionally, shorter substrates were also analyzed in native MS, fully in line with the results obtained SARS-CoV nsp7-10 (Figure S 8).</p><p id="p-19">The results from FRET peptide assay and full-length protein cleavage are in conflict regarding the suggested cleavage order. This is particularly interesting for nsp8-9, because of its <em>Coronaviridae</em>-wide conserved NNE motif at P1’-P3’. Previously, it was suggested that this motif could confer poor substrate properties to the nsp8-9 cleavage site, which would result in a considerable influence on processing order and RTC assembly [<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>]. In fact, two observations made by other researchers provide indication for a peptide-specific effect: Fan et al. analyzed the secondary structure of nsp peptides and observed that nsp8-9, comprising the NNE motif and closely resembling the FPS8-9 tested here, had a higher propensity for α-helices than other nsp peptides [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Moreover, Chuck et al. reported that β-strands in substrates promote binding to M<sup>pro</sup> [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. The results shown in our work indicate that the structural layout of the cleavage site is different when comparing peptide to full-length protein, which plays a role in endowing the nsp8-9 cleavage site with decent cleavability. Since the full-length protein possesses the structural environment of the cleavage sites, we consider results with this substrate more relevant for studying the processing pattern of the CoV polyprotein.</p></div><div id="sec-5" class="subsection"><h3>1.3.3. Protein complex formation upon processing of nsp7-10</h3><p id="p-20">In parallel, we applied CID-MS/MS to identify non-covalent interactions. Thereby, covalent processing intermediates were distinguished from newly formed complexes, for instance nsp7-8 intermediate and nsp7+8(1:1) dimer overlap in mass spectra (Figure S 3). Most importantly, we identified an nsp7+8 (2:2) complex, undescribed before, and subject to detailed analysis later on. Here, this complex’ intensity in native MS remained low, most likely due to interference by the non-cleavable His-tag N-terminal of nsp7. Furthermore, homo-dimers of nsp7 and nsp9 were found (Figure S 4). The MFs of nsp9 monomers and dimers (AVG±SD, <em>N</em>=3; 75±11% and 25±2%, respectively) were determined, based on the intensity ratio of the assigned peaks. Although, the X-ray structure of nsp9 was reported as a dimer, its low abundance found here is consistent with a low dimer affinity in solution [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. So far, nsp7’s ability of dimerization in absence of other proteins has already been noted, but no function has been suggested [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] and therefore, more detailed information remains elusive. Many interactions between components of nsp7-10 have been reported in the past, however, nsp interactions other than those reported above were completely absent, despite the high sensitivity of native MS.</p></div><div id="sec-6" class="subsection"><h3>1.3.4. Complex formation of nsp7+8</h3><p id="p-21">Finally, the complex of nsp7+8 was investigated in detail. In order to promote interactions of nsp’s with authentic amino acid sequence, we engineered a SARS-CoV nsp7-9-His containing a cleavable His-tag at the C-terminus. Upon processing of this substrate by M<sup>pro</sup>, we tested the landscape of mass species via native MS. Peak envelopes were detected for the monomeric products nsp7, nsp8 and nsp9 as well as for M<sup>pro</sup> monomer and dimer. While nsp9 did not interact with any of the other subunits, even after cleavage of the tag, nsp7+8 formed a hetero-tetramer of nsp7+8(2:2) (62.2 kDa) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). This stoichiometry was predicted based on the molecular weights of the subunits (2×9.3 kDa + 2×21.8 kDa = 62.2 kDa). However, for further evidence, molecular ions of three different charge states of the putative nsp7+8 hetero-tetramer (4449 <em>m/z</em>, +14; 4153 <em>m/z</em>, +15; 2893 <em>m/z</em>, +16) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5 A</a>) were selected as precursor and subjected to CID-MS/MS. To successively induce subunit dissociation, we increased the collision voltage (CV) stepwise. Thereby, the overall dissociation pattern was exposed (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5 A</a> and Figure S 9), which confirmed the hetero-tetrameric (2:2) complex stoichiometry.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Upon cleavage of nsp7-9-His, a putative nsp7+8 (2:2) tetramer (red) emerged. Other assigned peaks correspond to Mpro (asterisk), nsp7 (yellow), nsp8 (green) and nsp9 (blue). For cleaving the precursor, 2 &#x3BC;M SARS-CoV Mpro was incubated with 14 &#x3BC;M SARS-CoV nsp7-9-His (ratio ~1:7) at 4&#xB0;C in 300 mM NH4OAc, 1 mM DTT at pH 8.0 overnight." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1902911622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Upon cleavage of nsp7-9-His, a putative nsp7+8 (2:2) tetramer (red) emerged. Other assigned peaks correspond to M&lt;sup&gt;pro&lt;/sup&gt; (asterisk), nsp7 (yellow), nsp8 (green) and nsp9 (blue). For cleaving the precursor, 2 &#x3BC;M SARS-CoV M&lt;sup&gt;pro&lt;/sup&gt; was incubated with 14 &#x3BC;M SARS-CoV nsp7-9-His (ratio ~1:7) at 4&#xB0;C in 300 mM NH&lt;sub&gt;4&lt;/sub&gt;OAc, 1 mM DTT at pH 8.0 overnight.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F4.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F4.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-22" class="first-child">Upon cleavage of nsp7-9-His, a putative nsp7+8 (2:2) tetramer (red) emerged. Other assigned peaks correspond to M<sup>pro</sup> (asterisk), nsp7 (yellow), nsp8 (green) and nsp9 (blue). For cleaving the precursor, 2 μM SARS-CoV M<sup>pro</sup> was incubated with 14 μM SARS-CoV nsp7-9-His (ratio ~1:7) at 4°C in 300 mM NH<sub>4</sub>OAc, 1 mM DTT at pH 8.0 overnight.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Product ion spectra showing two CID dissociation pathways of SARS-CoV nsp7+8 hetero-tetramer (+14, 4449 m/z) at (A) 50 V and (B) 80 V collisional energy. (C) Two conformers denoted as T1 and T2 extracted from a hexa-decameric SARS-CoV nsp7+8 (PDB 2G9T) [30] and both in line with the native MS results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1902911622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Product ion spectra showing two CID dissociation pathways of SARS-CoV nsp7+8 hetero-tetramer (+14, 4449 &lt;em&gt;m/z&lt;/em&gt;) at (A) 50 V and (B) 80 V collisional energy. (C) Two conformers denoted as T1 and T2 extracted from a hexa-decameric SARS-CoV nsp7+8 (PDB 2G9T) [30] and both in line with the native MS results.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F5.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F5.medium.gif" width="404" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/04/860049/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-23" class="first-child">Product ion spectra showing two CID dissociation pathways of SARS-CoV nsp7+8 hetero-tetramer (+14, 4449 <em>m/z</em>) at (A) 50 V and (B) 80 V collisional energy. (C) Two conformers denoted as T1 and T2 extracted from a hexa-decameric SARS-CoV nsp7+8 (PDB 2G9T) [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>] and both in line with the native MS results.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Two alternative dissociation pathways revealed the arrangement of subunits within the complex (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5 B</a>). The hetero-tetramer dissociated at lower activation energy into nsp7 and nsp7+8(1:2), and alternatively, at elevated collisional energy into nsp8 and nsp7+8(2:1). Additionally, the product ions nsp7+8(1:1) and nsp8(2) provided evidence that their subunits interacted in the complex. The alternative dissociation routes may have resulted from analogous energy absorption paths, either due to similar binding contribution of nsp7 and nsp8 in the complex or due to different gas phase conformers.</p></div></div><div class="section" id="sec-7"><h2 class="">1.4. Discussion</h2><p id="p-25">Recently, the structure of the SARS-CoV RdRp complex was reported as nsp(7+8)+8+12 ((1:1):1:1)[<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>]. Nevertheless, the quaternary structure in the absence of other proteins could be different and an important intermediate for either stability or subsequent interactions in solution. Although, molecular structures of isolated SARS-CoV nsp7+8 have been reported as dimer (1:1) containing N-terminally chopped nsp8 and as a hexa-decamer (8:8) [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>]. Up to date, a hetero-tetrameric interaction has not been shown.</p><p id="p-26">In terms of stoichiometry and subunit connectivity, only nsp7+8(2:2) subcomplexes of the hexadecamer resemble our results [<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>]. Specifically, in two conformers, T1 and T2 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5 C</a>), nsp8 builds a scaffold with head-to-tail interaction and nsp7 subunits sandwich the scaffold without self-interaction. All interfaces are mainly hydrophobic. However, three polar bonds within or in proximity of the nsp8 shaft domain facilitate binding to nsp7. Polar bonds more readily persist in the gas phase than hydrophobic interactions, which explains higher gas phase stability for nsp7+8(1:1) over nsp8(2). The total buried surface areas of T1 and T2 are virtually equal (Table S 5). Nevertheless, the nsp8(2) dimer scaffold in T2 is compact because a loop within the N-terminal α-helix allows bending and interacting with the other nsp8, whereas in T1 the α-helix extends to interact with another tetramer in the crystal lattice. Hence, T2 represents a more likely structure of nsp7+8(2:2) in solution.</p><p id="p-27">In summary, we performed a comprehensive analysis of specific processing and complex formation of a folded viral polyprotein with native MS. While it remains a challenge to perform quantitative MS from heterogeneous protein mixtures, the usage of mass fractions disentangled the mass species and allowed following the processing dynamics. Intriguingly, our results suggest that the processing pattern of SARS-CoV nsp7-10 could have evolved to tailor a remaining nsp7-8 intermediate product. This intermediate product may have independent enzymatic function [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>] but finally serves as an efficient precursor for nsp7+8(2:2) hetero-tetramer formation, as stimulant of the CoV RdRp [<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]. Extending our approach to processing of different CoV species and longer polyproteins will increase understanding of the formation of the functional CoV-RTC.</p></div><div class="section" id="sec-8"><h2 class="">1.5. Materials and methods</h2><div id="sec-9" class="subsection"><h3>1.5.1. Cloning and gene constructs</h3><p id="p-28">To generate inserts for expression plasmids for the CoV ORF1ab nsp7-10 region, DNA was amplified by PCR from commercially obtained cDNA (Eurofins scientific SE) and from DNA plasmids existing from earlier work [<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>]. To create ends for directed cloning, the PCR products were digested by Eco31I (BsaI) (Thermo Fisher Scientific) and ligated into IBA pASK33+ and pASK35+, encoding for C- and N-terminal His-tag, respectively. The expression plasmid for SARS-CoV M<sup>pro</sup> with authentic ends was created as described by Xue et al.[<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>].</p></div><div id="sec-10" class="subsection"><h3>1.5.2. Expression and purification</h3><p id="p-29">Main protease SARS-CoV M<sup>pro</sup> was produced with authentic ends [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>]. Briefly, the protein constructs had a GST-tag connected via an auto-cleavage site at the N-terminus and a His-tag connected via a PreScission protease cleavage site at the C-terminus. Protease was purified by affinity chromatography using Ni<sup>2+</sup>-NTA beads (Thermo Fisher Scientific) and dialysis into storage buffer (50 mM HEPES, 10 % glycerol pH 7.5). The His-tag was cleaved by dialysis into PreScission cleavage buffer (20 mM Tris-HCl, 150 mM NaCl, 1mM EDTA, 1 mM DTT at pH 7.0) while incubating with PreScission protease (protease to protein ratio 1:2000) (GE Healthcare) overnight. PreScission protease was removed by binding to GST-Sepharose (GE Healthcare) for 2 h and buffer exchange with centrifugal filter devices (10,000 MWCO, Amicon, Merck Millipore) into storage buffer. Purified M<sup>pro</sup> was frozen in liquid nitrogen and stored at −80°C.</p><p id="p-30">To produce proteins (Protein ID: P0C6X7) of different-length of the nsp7-10 region, BL21 Rosetta2 (Merck Millipore) were transformed, grown in culture flasks to 0.4-0.6 OD<sub>600</sub> in 0.3-1 L, then induced with 50 μM anhydrotetracycline and continued to grow at 20°C overnight. To pellet, cultures were centrifuged (6000 x <em>g</em> for 20 min) and cells were frozen at −20°C. To separate soluble nsp’s, pelleted cells were lysed in 1:5 (v/v) buffer B1 (40 mM phosphate buffer, 300 mM NaCl) with one freeze-thaw cycle, sonicated (micro tip, 70 % power, 6 times on 10 s, off 60 s; Branson digital sonifier SFX 150) and then centrifuged (20,000 × <em>g</em> for 45 min). The proteins were isolated with Ni<sup>2+</sup>-NTA beads (Thermo Fisher Scientific) in gravity flow columns (BioRad). First the beads were equilibrated with 20 column volumes (CV) B1 + 20 mM imidazole, then bound to nsp’s by incubation with crude extract for 60 min and finally to remove unspecifically bound proteins washed with 20 CV B1 +20 mM imidazole followed by 10 CV of B1 +50 mM imidazole. To elute the nsp’s, eight fractions of 0.5 CV B1 +300 mM imidazole were collected. Immediately after elution, the fractions were supplemented with 4 mM DTT. For quality analysis, SDS-PAGE was performed.</p></div><div id="sec-11" class="subsection"><h3>1.5.3. FRET peptide assays</h3><p id="p-31">For SARS-CoV M<sup>pro</sup> activity assays, Förster resonance energy transfer (FRET) peptide substrates (FPS) were commercially purchased (Eurogentec), designed as SARS-CoV cleavage site analogues of twelve amino-acid residues (P6-P6’) with a FRET pair labelling, namely the fluorophore (F) HiLyte488 at the N-terminus and the Quencher (Q) QXL520 at the C-terminus (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table S 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1022580/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1022580/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1022580" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S 1:</span> <span class="caption-title">FRET peptide substrates (FPS) purchased to determine kinetic parameters of Mpro.</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Initially, the freeze-dried FPS were dissolved in DMSO to 2 mM and stored at −20°C, protected from light. For sample preparation, the peptides were serially diluted in 50 mM HEPES, 10% (v/v) glycerol, 1 mM DTT, pH 7.5 and final concentrations were adjusted by measuring A<sub>280</sub> from five independent droplets, using ε = 73,000 mol<sup>−1</sup> cm<sup>−1</sup>, the absorption coefficient of the HiLyte488 fluorophore as given by the manufacturer.</p><p id="p-33">Assays were carried out using a 96-well plate reader (Infinite200, Tecan) with the following parameters: excitation wavelength 485 nm (bandwidth 9 nm), emission wavelength 535 nm (bandwidth 20 nm), gain 80, 10 flashes and 40 μs integration time. Further, flat bottom black polystyrol 96-well plates (Greiner) were used for fluorescence applications.</p><p id="p-34">Fluorescence coefficients k<sub>sx</sub> and inner filter effect of the peptides were determined for each substrate according to previous protocols [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>] (Figure S 2). The kinetic parameters were determined as suggested by Grum-Tokars et al. [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. Briefly, 20 μL SARS-CoV M<sup>pro</sup> (final conc. c<sub>Mpro</sub> of 0.5 μM) was mixed with 80 μL FRET peptide substrates of six different concentrations (final conc. c<sub>X,S</sub> of 2 μM, 1μM, 0.5 μM, 0.25 μM, 0.125 μM and 0 μM) and fluorescence (AFU) was monitored every 30 seconds for 5 minutes. To calculate kinetic parameters, triplicate measurements were performed.</p><p id="p-35">The fluorescence coefficients k<sub>sx</sub> were used to convert the initial increase of AFU into the initial reaction rate, which was then normalized by c<sub>Mpro</sub> to rate/[Enzyme]. For comparing substrate specificities, the rates/[Enzyme] were plotted against the substrate concentrations c<sub>X,S</sub> resulting in process curves. The slopes of the process curves correspond to the apparent k<sub>cat</sub>/K<sub>M</sub>.</p><p id="p-36">The influence of salt conditions on enzyme activity was determined by incubating 20 μL M<sup>pro</sup> (final concentration 0.1 μM) with 80 μM FRET peptide substrate FPS4-5 (final concentration 1 μM) in varying NH<sub>4</sub>OAc (10% Glycerol, pH7.5 and 50 mM, 165 mM, 280 mM, 395 mM, 510 mM, 625 mM, 740 mM, 855 mM, 970 mM, 1085 mM and 1200 mM) and NaCl (50 mM HEPES, 10% Glycerol, pH 7.5 and 0 mM, 120 mM, 240 mM, 360 mM, 480 mM, 600 mM, 720 mM, 840 mM, 960 mM, 1080 mM and 1200 mM). The fluorescence was followed every 60 seconds by monitoring (AFU), which initial slopes (AFU/s) were converted by the fluorescence coefficient k<sub>sx</sub> into the initial rate (Figure S 1).</p></div><div id="sec-12" class="subsection"><h3>1.5.4. SDS PAGE</h3><p id="p-37">SDS-PAGE was performed with a 4-12% gradient acrylamide Bis-tris gel with XT MES running buffer. For <em>in-vitro</em> processing, 3.2 μM SARS-CoV M<sup>pro</sup> was incubated with 13 μM SARS-CoV nsp7-9-His (ratio ~1:4) at 4°C in 20 mM phosphate buffer, 150 mM NaCl, 1 mM DTT at pH 8.0.</p></div><div id="sec-13" class="subsection"><h3>1.5.5. Native MS</h3><p id="p-38">To prepare samples for native MS measurements, they were buffer-exchanged into nano electrospray ionization (nanoESI)-compatible solution. Proteases M<sup>pro</sup> were buffer exchanged into 250-500 mM NH<sub>4</sub>OAc, 1 mM pH 8.0 by two cycles of centrifugal gel filtration (Biospin mini columns, 6,000 MWCO, Biorad). The nsp’s were buffer-exchanged into 250-500 mM NH<sub>4</sub>OAc, 1 mM DTT, pH 8.0 by five rounds of dilution and concentration in centrifugal filter units (Amicon, 10,000 MWCO, Merck Millipore).</p><p id="p-39">NanoESI capillaries were pulled in-house from borosilicate capillaries (1.2 mm outer diameter, 0.68 mm inner diameter, filament, World Precision Instruments) with a micropipette puller (P-1000, Sutter Instruments) using a squared box filament (2.5 × 2.5 mm, Sutter Instruments) in a two-step program. Subsequently, capillaries were gold-coated using a sputter coater (Q150R, Quorum Technologies) with 40 mA, 200 s, tooling factor 2.3 and end bleed vacuum of 8×10<sup>−2</sup> mbar argon.</p><p id="p-40">Native MS was performed at an nano-electrospray quadrupole time-of-flight (nanoESI-Q-TOF) instrument (Q-TOF2, Micromass/Waters, MS Vision) modified for higher masses [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. Samples were ionized in positive ion mode with voltages applied at the capillary of 1300-1500 V and at the cone of 130-135 V. The pressure in the source region was kept at 10 mbar throughout all native MS experiments. For purpose of desolvation and dissociation, the pressure in the collision cell was adjusted to 1.3-1.5×10<sup>−2</sup> mbar argon. Native mass spectra were obtained at accelerating voltage of 10-30 V while for CID-MS/MS these voltages were increased to 30-200 V. In ESI-MS overview spectra for nsp’s, the quadrupole profile was 1-10,000 <em>m/z</em>. In tandem MS, for precursor selection, LMres and HMres were adjusted at 10-30 V collisional voltage until a single peak was recorded, and then dissociation was induced.</p><p id="p-41">To calibrate raw data, CsI (25 mg/ml) spectra were acquired and calibration was carried out with MassLynx (Waters) software. Data were analysed using MassLynx (Waters), Massign (by Nina Morgner [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]). All determined masses are provided (Table S 3).</p><p id="p-42">To start the processing reactions, nsp’s and M<sup>pro</sup> were incubated (ratio ~1:4–1:10). Three independent reactions were started in parallel and incubated at 4°C. To acquire mass spectra at defined time-points, sample aliquots of 1-3 μL were taken by means of a microliter syringe (5 μL, Hamilton) with flexible fused silica tubing (Optronis) and loaded into in-house fabricated nanoESI capillaries, which were mounted on the nanoESI source, all within two minutes. Then, raw spectra were acquired in the first 300 scans (1 scan/sec).</p><p id="p-43">To analyze the data, the raw spectra were smoothed (2×5) in Masslynx 4.1 (Waters) and then nsp’s were assigned to peak series. For each assigned mass species, signal responses (SRs), using the combined relative intensities of peaks, were summarized and normalized. This was done independently for each spectrum. Finally, the average and standard deviation for SR and time-points was calculated from three independent spectra. Poorly resolved spectra were not included in the data analysis.</p><p id="p-44">To determine mass fractions (MF), the SRs of assigned peaks for monomers and complexes were combined (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table S 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1022584/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1022584/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1022584" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S 2:</span> <span class="caption-title">Formulas to determine mass fractions:</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">1.6. Acknowledgement</h2><p id="p-45">The Heinrich Pette Institute, Leibniz Institute for Experimental Virology is supported by the Free and Hanseatic City of Hamburg and the Federal Ministry of Health. C.U. and B.K. are supported by EU Horizon 2020 ERC StG-2017 759661. C.U. and B.K.․ also received funding through Leibniz Association through SAW-2014-HPI-4 grant. SF and LR thank the German Federal Ministry for Education and Research (BMBF) for funding (grants 01KX0806, 01KX0807, and 05K18FLA). RH acknowledges funding from the SILVER project of the European Commission (contract HEALTH-F3-2010-260644).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">methods moved to main text</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">1.8. References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.1" data-doi="10.1056/NEJMoa030747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Identification of a novel coronavirus in patients with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">New. Engl. J. Med.</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">348</span>(<span class="cit-issue">20</span>): p. <span class="cit-fpage">1967</span>–<span class="cit-lpage">1976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew.%2BEngl.%2BJ.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030747%26rft_id%253Dinfo%253Apmid%252F12690091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.2" data-doi="10.1016/j.antiviral.2013.08.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M.</span>, <span class="cit-name-surname">Peiris</span></span>, <span class="cit-article-title">From SARS to MERS: 10 years of research on highly pathogenic human coronaviruses</span>. <abbr class="cit-jnl-abbrev">Antiviral research</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">100</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">286</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.volume%253D100%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2013.08.015%26rft_id%253Dinfo%253Apmid%252F24012996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2013.08.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24012996&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328179300032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.3" data-doi="10.1016/0092-8674(89)90124-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span>, <span class="cit-name-surname">Digard</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.C.</span>, <span class="cit-name-surname">Inglis</span></span>, <span class="cit-article-title">Characterization of an efficient coronavirus ribosomal frameshifting signal: requirement for an RNA pseudoknot</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">1989</span>. <span class="cit-vol">57</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">537</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBrierley%26rft.auinit1%253DI.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D537%26rft.epage%253D547%26rft.atitle%253DCharacterization%2Bof%2Ban%2Befficient%2Bcoronavirus%2Bribosomal%2Bframeshifting%2Bsignal%253A%2Brequirement%2Bfor%2Ban%2BRNA%2Bpseudoknot.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990124-4%26rft_id%253Dinfo%253Apmid%252F2720781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90124-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2720781&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U743600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.4" data-doi="10.1126/science.1085953"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marra</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The Genome sequence of the SARS-associated coronavirus</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">300</span>(<span class="cit-issue">5624</span>): p. <span class="cit-fpage">1399</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085953%26rft_id%253Dinfo%253Apmid%252F12730501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNC8xMzk5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMDQvODYwMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-860049v2.5" data-doi="10.1099/vir.0.19424-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thiel</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Mechanisms and enzymes involved in SARS coronavirus genome expression</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">84</span>(<span class="cit-issue">Pt 9</span>): p. <span class="cit-fpage">2305</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DThiel%26rft.auinit1%253DV.%26rft.volume%253D84%26rft.issue%253D9%26rft.spage%253D2305%26rft.epage%253D2315%26rft.atitle%253DMechanisms%2Band%2Benzymes%2Binvolved%2Bin%2BSARS%2Bcoronavirus%2Bgenome%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19424-0%26rft_id%253Dinfo%253Apmid%252F12917450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.19424-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12917450&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184993100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-860049v2.6" data-doi="10.1016/S0022-2836(03)00865-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Unique and conserved features of genome and proteome of SARS-coronavirus, an early split-off from the coronavirus group 2 lineage</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">331</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">991</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSnijder%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D331%26rft.issue%253D5%26rft.spage%253D991%26rft.epage%253D1004%26rft.atitle%253DUnique%2Band%2Bconserved%2Bfeatures%2Bof%2Bgenome%2Band%2Bproteome%2Bof%2BSARS-coronavirus%252C%2Ban%2Bearly%2Bsplit-off%2Bfrom%2Bthe%2Bcoronavirus%2Bgroup%2B2%2Blineage.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252803%252900865-9%26rft_id%253Dinfo%253Apmid%252F12927536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(03)00865-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12927536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185036400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.7" data-doi="10.1016/j.virusres.2006.01.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Nidovirales: evolving the largest RNA virus genome</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, <span class="cit-pub-date">2006</span>. <span class="cit-vol">117</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">17</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D37%26rft.atitle%253DNidovirales%253A%2Bevolving%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.01.017%26rft_id%253Dinfo%253Apmid%252F16503362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16503362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236746000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.8" data-doi="10.1074/jbc.M502577200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hsu</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Mechanism of the maturation process of SARS-CoV 3CL protease</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">280</span>(<span class="cit-issue">35</span>): p. <span class="cit-fpage">31257</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M502577200%26rft_id%253Dinfo%253Apmid%252F15788388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMzUvMzEyNTciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8wNC84NjAwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S.G.</span>, <span class="cit-name-surname">Siddell</span></span>, <span class="cit-article-title">Processing of the human coronavirus 229E replicase polyproteins by the virus-encoded 3C-like proteinase: identification of proteolytic products and cleavage sites common to pp1a and pp1ab</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-pub-date">1999</span>. <span class="cit-vol">73</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">177</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DZiebuhr%26rft.auinit1%253DJ.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D177%26rft.epage%253D185%26rft.atitle%253DProcessing%2Bof%2Bthe%2BHuman%2BCoronavirus%2B229E%2BReplicase%2BPolyproteins%2Bby%2Bthe%2BVirus-Encoded%2B3C-Like%2BProteinase%253A%2BIdentification%2Bof%2BProteolytic%2BProducts%2Band%2BCleavage%2BSites%2BCommon%2Bto%2Bpp1a%2Band%2Bpp1ab%26rft_id%253Dinfo%253Apmid%252F9847320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjczLzEvMTc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMDQvODYwMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.10" data-doi="10.1126/science.1085658"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anand</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Coronavirus main proteinase (3CLpro) structure: basis for design of anti-SARS drugs</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">300</span>(<span class="cit-issue">5626</span>): p. <span class="cit-fpage">1763</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085658%26rft_id%253Dinfo%253Apmid%252F12746549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzYzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMDQvODYwMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.11" data-doi="10.1073/pnas.1835675100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The crystal structures of severe acute respiratory syndrome virus main protease and its complex with an inhibitor</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr>, <span class="cit-pub-date">2003</span>. <span class="cit-vol">100</span>(<span class="cit-issue">23</span>): p. <span class="cit-fpage">13190</span>–<span class="cit-lpage">13195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1835675100%26rft_id%253Dinfo%253Apmid%252F14585926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzIzLzEzMTkwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMDQvODYwMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.12" data-doi="10.1371/journal.ppat.0010039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sawicki</span>,  <span class="cit-name-given-names">S.G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Functional and genetic analysis of coronavirus replicase-transcriptase proteins</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">1</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">e39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DSawicki%26rft.auinit1%253DS.%2BG.%26rft.volume%253D1%26rft.issue%253D4%26rft.spage%253De39%26rft.epage%253De39%26rft.atitle%253DFunctional%2Band%2Bgenetic%2Banalysis%2Bof%2Bcoronavirus%2Breplicase-transcriptase%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.0010039%26rft_id%253Dinfo%253Apmid%252F16341254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.0010039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16341254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.13" data-doi="10.1128/JVI.00017-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deming</span>,  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Processing of open reading frame 1a replicase proteins nsp7 to nsp10 in murine hepatitis virus strain A59 replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">81</span>(<span class="cit-issue">19</span>): p. <span class="cit-fpage">10280</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00017-07%26rft_id%253Dinfo%253Apmid%252F17634238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8xOS8xMDI4MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzA0Lzg2MDA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.14" data-doi="10.1074/jbc.M310875200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Biosynthesis, purification, and substrate specificity of severe acute respiratory syndrome coronavirus 3C-like proteinase</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>, <span class="cit-pub-date">2004</span>. <span class="cit-vol">279</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">1637</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M310875200%26rft_id%253Dinfo%253Apmid%252F14561748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzkvMy8xNjM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMDQvODYwMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-860049v2.15" data-doi="10.1016/j.virusres.2015.05.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Prediction and biochemical analysis of putative cleavage sites of the 3C-like protease of Middle East respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">208</span>: p. <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D208%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2015.05.018%26rft_id%253Dinfo%253Apmid%252F26036787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2015.05.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26036787&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.16" data-doi="10.1099/0022-1317-83-3-595"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hegyi</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.</span>, <span class="cit-name-surname">Ziebuhr</span></span>, <span class="cit-article-title">Conservation of substrate specificities among coronavirus main proteases</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr>, <span class="cit-pub-date">2002</span>. <span class="cit-vol">83</span>(<span class="cit-issue">Pt 3</span>): p. <span class="cit-fpage">595</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DHegyi%26rft.auinit1%253DA.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D595%26rft.epage%253D599%26rft.atitle%253DConservation%2Bof%2Bsubstrate%2Bspecificities%2Bamong%2Bcoronavirus%2Bmain%2Bproteases%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-83-3-595%26rft_id%253Dinfo%253Apmid%252F11842254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-83-3-595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11842254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173983100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Falke</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-article-title">(Doctoral Thesis) Coronaviral Polyprotein Nsp7-10: Proteolytic Processing and Dynamic Interactions within the Transcriptase/Replicase Complex</span>. <span class="cit-pub-date">2014</span>, <abbr class="cit-jnl-abbrev">Staats- und Universitätsbibliothek Hamburg</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.18" data-doi="10.1073/pnas.1323705111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Subissi</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">One severe acute respiratory syndrome coronavirus protein complex integrates processive RNA polymerase and exonuclease activities</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">111</span>(<span class="cit-issue">37</span>): p. <span class="cit-fpage">E3900</span>–<span class="cit-lpage">E3909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323705111%26rft_id%253Dinfo%253Apmid%252F25197083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM3L0UzOTAwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMDQvODYwMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.19" data-doi="10.1093/nar/gkr893"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">te Velthuis</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.H.</span> <span class="cit-name-surname">van den Worm</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.J.</span>, <span class="cit-name-surname">Snijder</span></span>, <span class="cit-article-title">The SARS-coronavirus nsp7+nsp8 complex is a unique multimeric RNA polymerase capable of both de novo initiation and primer extension</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">40</span>(<span class="cit-issue">4</span>): p. <span class="cit-fpage">1737</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr893%26rft_id%253Dinfo%253Apmid%252F22039154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039154&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301069400034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.20" data-doi="10.1007/s13238-010-0028-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">New nsp8 isoform suggests mechanism for tuning viral RNA synthesis</span>. <abbr class="cit-jnl-abbrev">Protein Cell</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">1</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">198</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DProtein%2BCell%26rft.aulast%253DLi%26rft.auinit1%253DS.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D198%26rft.epage%253D204%26rft.atitle%253DNew%2Bnsp8%2Bisoform%2Bsuggests%2Bmechanism%2Bfor%2Btuning%2Bviral%2BRNA%2Bsynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13238-010-0028-8%26rft_id%253Dinfo%253Apmid%252F21203988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13238-010-0028-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21203988&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.21" data-doi="10.1128/JVI.06635-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Nonstructural proteins 7 and 8 of feline coronavirus form a 2:1 heterotrimer that exhibits primer-independent RNA polymerase activity</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">86</span>(<span class="cit-issue">8</span>): p. <span class="cit-fpage">4444</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06635-11%26rft_id%253Dinfo%253Apmid%252F22318142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzgvNDQ0NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzA0Lzg2MDA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>,  <span class="cit-name-given-names">R.N.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A.B.</span> <span class="cit-name-surname">Ward</span></span>, <span class="cit-article-title">Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">10</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">2342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D2342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.23" data-doi="10.1016/j.cbpa.2017.11.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chandler</span>,  <span class="cit-name-given-names">S.A.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.L.</span> <span class="cit-name-surname">Benesch</span></span>, <span class="cit-article-title">Mass spectrometry beyond the native state</span>. <abbr class="cit-jnl-abbrev">Curr Opin Chem Biol</abbr>, <span class="cit-pub-date">2018</span>. <span class="cit-vol">42</span>: p. <span class="cit-fpage">130</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BChem%2BBiol%26rft.volume%253D42%26rft.spage%253D130%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cbpa.2017.11.019%26rft_id%253Dinfo%253Apmid%252F29288996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cbpa.2017.11.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29288996&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dülfer</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Structural mass spectrometry goes viral</span>. <abbr class="cit-jnl-abbrev">Adv. Virus Res.</abbr>, <span class="cit-pub-date">2019</span>. <span class="cit-vol">105</span>: p. <span class="cit-fpage">189</span>–<span class="cit-lpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BVirus%2BRes.%26rft.volume%253D105%26rft.spage%253D189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.25" data-doi="10.1016/j.virusres.2007.02.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grum-Tokars</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evaluating the 3C-like protease activity of SARS-Coronavirus: recommendations for standardized assays for drug discovery</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">133</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGrum-Tokars%26rft.auinit1%253DV.%26rft.volume%253D133%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D73%26rft.atitle%253DEvaluating%2Bthe%2B3C-like%2Bprotease%2Bactivity%2Bof%2BSARS-Coronavirus%253A%2Brecommendations%2Bfor%2Bstandardized%2Bassays%2Bfor%2Bdrug%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2007.02.015%26rft_id%253Dinfo%253Apmid%252F17397958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2007.02.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17397958&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.26" data-doi="10.1021/bi036022q"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">3C-like proteinase from SARS coronavirus catalyzes substrate hydrolysis by a general base mechanism</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr>, <span class="cit-pub-date">2004</span>. <span class="cit-vol">43</span>(<span class="cit-issue">15</span>): p. <span class="cit-fpage">4568</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D43%26rft.spage%253D4568%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi036022q%26rft_id%253Dinfo%253Apmid%252F15078103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi036022q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15078103&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220812900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.27" data-doi="10.1016/j.bbrc.2005.02.061"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The substrate specificity of SARS coronavirus 3C-like proteinase</span>. <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">329</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">934</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DFan%26rft.auinit1%253DK.%26rft.volume%253D329%26rft.issue%253D3%26rft.spage%253D934%26rft.epage%253D940%26rft.atitle%253DThe%2Bsubstrate%2Bspecificity%2Bof%2BSARS%2Bcoronavirus%2B3C-like%2Bproteinase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2005.02.061%26rft_id%253Dinfo%253Apmid%252F15752746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2005.02.061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15752746&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.28" data-doi="10.1371/journal.pone.0013197"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chuck</span>,  <span class="cit-name-given-names">C.-P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Profiling of substrate specificity of SARS-CoV 3CLpro</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">5</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">e13197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChuck%26rft.auinit1%253DC.%2BP.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De13197%26rft.epage%253De13197%26rft.atitle%253DProfiling%2Bof%2Bsubstrate%2Bspecificity%2Bof%2BSARS-CoV%2B3CL.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0013197%26rft_id%253Dinfo%253Apmid%252F20949131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0013197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20949131&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.29" data-doi="10.1016/j.jmb.2008.07.071"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ponnusamy</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Variable oligomerization modes in coronavirus non-structural protein 9</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">383</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">1081</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPonnusamy%26rft.auinit1%253DR.%26rft.volume%253D383%26rft.issue%253D5%26rft.spage%253D1081%26rft.epage%253D1096%26rft.atitle%253DVariable%2Boligomerization%2Bmodes%2Bin%2Bcoronavirus%2Bnon-structural%2Bprotein%2B9.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2008.07.071%26rft_id%253Dinfo%253Apmid%252F18694760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2008.07.071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18694760&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.30" data-doi="10.1038/ng1622"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhai</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Insights into SARS-CoV transcription and replication from the structure of the nsp7-nsp8 hexadecamer</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">12</span>(<span class="cit-issue">11</span>): p. <span class="cit-fpage">980</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DSharon%26rft.auinit1%253DI.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D980%26rft.epage%253D986%26rft.atitle%253DInsights%2Binto%2BSARS-CoV%2Btranscription%2Band%2Breplication%2Bfrom%2Bthe%2Bstructure%2Bof%2Bthe%2Bnsp7-nsp8%2Bhexadecamer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1622%26rft_id%253Dinfo%253Apmid%252F16228002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16228002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233115100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.31" data-doi="10.1016/j.jmb.2006.11.073"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Production of authentic SARS-CoV M pro with enhanced activity: application as a novel tag-cleavage endopeptidase for protein overproduction</span>. <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">366</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">965</span>–<span class="cit-lpage">975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DXue%26rft.auinit1%253DX.%26rft.volume%253D366%26rft.issue%253D3%26rft.spage%253D965%26rft.epage%253D975%26rft.atitle%253DProduction%2Bof%2Bauthentic%2BSARS-CoV%2BM%2528pro%2529%2Bwith%2Benhanced%2Bactivity%253A%2Bapplication%2Bas%2Ba%2Bnovel%2Btag-cleavage%2Bendopeptidase%2Bfor%2Bprotein%2Boverproduction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2006.11.073%26rft_id%253Dinfo%253Apmid%252F17189639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2006.11.073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17189639&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.32" data-doi="10.1006/abio.1998.3014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Use of a fluorescence plate reader for measuring kinetic parameters with inner filter effect correction</span>. <abbr class="cit-jnl-abbrev">Analytical biochemistry</abbr>, <span class="cit-pub-date">1999</span>. <span class="cit-vol">267</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">331</span>–<span class="cit-lpage">335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DLiu%26rft.auinit1%253DY.%26rft.volume%253D267%26rft.issue%253D2%26rft.spage%253D331%26rft.epage%253D335%26rft.atitle%253DUse%2Bof%2Ba%2Bfluorescence%2Bplate%2Breader%2Bfor%2Bmeasuring%2Bkinetic%2Bparameters%2Bwith%2Binner%2Bfilter%2Beffect%2Bcorrection.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fabio.1998.3014%26rft_id%253Dinfo%253Apmid%252F10036138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/abio.1998.3014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10036138&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.33" data-doi="10.1021/ac061039a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van den Heuvel</span>,  <span class="cit-name-given-names">R.H.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Improving the performance of a quadrupole time-of-flight instrument for macromolecular mass spectrometry</span>. <abbr class="cit-jnl-abbrev">Analytical chemistry</abbr>, <span class="cit-pub-date">2006</span>. <span class="cit-vol">78</span>(<span class="cit-issue">21</span>): p. <span class="cit-fpage">7473</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253Dvan%2Bden%2BHeuvel%26rft.auinit1%253DR.%2BH.%26rft.volume%253D78%26rft.issue%253D21%26rft.spage%253D7473%26rft.epage%253D7483%26rft.atitle%253DImproving%2Bthe%2Bperformance%2Bof%2Ba%2Bquadrupole%2Btime-of-flight%2Binstrument%2Bfor%2Bmacromolecular%2Bmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac061039a%26rft_id%253Dinfo%253Apmid%252F17073415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac061039a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17073415&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-860049v2.34" data-doi="10.1021/ac300056a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morgner</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">C.V.</span> <span class="cit-name-surname">Robinson</span></span>, <span class="cit-article-title">Massign: an assignment strategy for maximizing information from the mass spectra of heterogeneous protein assemblies</span>. <abbr class="cit-jnl-abbrev">Analytical chemistry</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">84</span>(<span class="cit-issue">6</span>): p. <span class="cit-fpage">2939</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DMorgner%26rft.auinit1%253DN.%26rft.volume%253D84%26rft.issue%253D6%26rft.spage%253D2939%26rft.epage%253D2948%26rft.atitle%253DMassign%253A%2Ban%2Bassignment%2Bstrategy%2Bfor%2Bmaximizing%2Binformation%2Bfrom%2Bthe%2Bmass%2Bspectra%2Bof%2Bheterogeneous%2Bprotein%2Bassemblies.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac300056a%26rft_id%253Dinfo%253Apmid%252F22409725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac300056a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22409725&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F04%2F860049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/860049v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F860049v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/864017v1" title="MicroRNA function transitions from regulating developmental genes to transposable elements during the maturation of pollen" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 04, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/860049v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/860049v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/860049v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/860049v2" class="active" data-icon-position="" data-hide-link-title="0">Processing of the SARS-CoV pp1a/ab nsp7-10 region</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1020108" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_KOzAeMQGiW4tC4FbyLsLxa1ZgXECPKzjqJOxZ1Qs2s" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1020108" id="node1020108--2" data-pisa="biorxiv;860049v2" data-pisa-master="biorxiv;860049" data-seqnum="1020108" data-apath="/biorxiv/early/2019/12/04/860049.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Processing of the SARS-CoV pp1a/ab nsp7-10 region</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Krichel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sven</span> <span class="nlm-surname">Falke</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Rolf</span> <span class="nlm-surname">Hilgenfeld</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lars</span> <span class="nlm-surname">Redecke</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Charlotte</span> <span class="nlm-surname">Uetrecht</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">860049; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/860049 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/860049v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;title=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;title=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;text=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;title=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;t=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;title=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;title=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1020108" id="node1020108--3" data-pisa="biorxiv;860049v2" data-pisa-master="biorxiv;860049" data-seqnum="1020108" data-apath="/biorxiv/early/2019/12/04/860049.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Processing of the SARS-CoV pp1a/ab nsp7-10 region</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Krichel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sven</span> <span class="nlm-surname">Falke</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Rolf</span> <span class="nlm-surname">Hilgenfeld</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lars</span> <span class="nlm-surname">Redecke</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Charlotte</span> <span class="nlm-surname">Uetrecht</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">860049; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/860049 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Krichel%2C+Boris&amp;rft.au=Falke%2C+Sven&amp;rft.au=Hilgenfeld%2C+Rolf&amp;rft.au=Redecke%2C+Lars&amp;rft.au=Uetrecht%2C+Charlotte"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1020108/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1020108/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1020108/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1020108/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1020108/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1020108/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1020108/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1020108/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1020108/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1020108/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1020108/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1020108/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;count=horizontal&amp;via=&amp;text=Processing%20of%20the%20SARS-CoV%20pp1a/ab%20nsp7-10%20region&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/860049v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/860049v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/860049v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biophysics" class="highlight" data-icon-position="" data-hide-link-title="0">Biophysics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
