<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/04/05/597963.atom"/>
  <meta name="HW.pisa" content="biorxiv;597963v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Identification of viruses with the potential to infect human"/>
  <meta name="DC.Identifier" content="10.1101/597963"/>
  <meta name="DC.Date" content="2019-04-05"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The virus has caused much mortality and morbidity to humans, and still posed a serious threat to the global public health. The virome with the human-infection potential is far from complete. Novel viruses have been discovered at an unprecedented pace as the rapid development of viral metagenomics. However, there is still a lack of a method for rapidly identifying the virus with the human-infection potential. This study built several machine learning models for discriminating the human-infecting viruses from other viruses based on the frequency of k-mers in the viral genomic sequences. The k-nearest neighbor (KNN) model could predict the human-infecting virus with an accuracy of over 90%. Even for the KNN models built on the contigs as short as 1kb, they performed comparably to those built on the viral genomes, suggesting that the models could be used to identify the human-infecting virus from the viral metagenomic sequences. This work could help for discovery of novel human-infecting virus in metagenomics studies."/>
  <meta name="DC.Contributor" content="Zheng Zhang"/>
  <meta name="DC.Contributor" content="Zena Cai"/>
  <meta name="DC.Contributor" content="Zhiying Tan"/>
  <meta name="DC.Contributor" content="Congyu Lu"/>
  <meta name="DC.Contributor" content="Gaihua Zhang"/>
  <meta name="DC.Contributor" content="Yousong Peng"/>
  <meta name="article:published_time" content="2019-04-05"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Identification of viruses with the potential to infect human"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;The virus has caused much mortality and morbidity to humans, and still posed a serious threat to the global public health. The virome with the human-infection potential is far from complete. Novel viruses have been discovered at an unprecedented pace as the rapid development of viral metagenomics. However, there is still a lack of a method for rapidly identifying the virus with the human-infection potential. This study built several machine learning models for discriminating the human-infecting viruses from other viruses based on the frequency of k-mers in the viral genomic sequences. The k-nearest neighbor (KNN) model could predict the human-infecting virus with an accuracy of over 90%. Even for the KNN models built on the contigs as short as 1kb, they performed comparably to those built on the viral genomes, suggesting that the models could be used to identify the human-infecting virus from the viral metagenomic sequences. This work could help for discovery of novel human-infecting virus in metagenomics studies.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;597963v1"/>
  <meta name="citation_id" content="597963v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/597963v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/597963v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/597963v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963.full.pdf"/>
  <meta name="citation_doi" content="10.1101/597963"/>
  <meta name="citation_num_pages" content="11"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="597963"/>
  <meta name="citation_author" content="Zheng Zhang"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan University"/>
  <meta name="citation_author" content="Zena Cai"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan University"/>
  <meta name="citation_author" content="Zhiying Tan"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan University"/>
  <meta name="citation_author" content="Congyu Lu"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan University"/>
  <meta name="citation_author" content="Gaihua Zhang"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Hunan Normal University"/>
  <meta name="citation_author" content="Yousong Peng"/>
  <meta name="citation_author_institution" content="College of Biology, Hunan University"/>
  <meta name="citation_author_email" content="pys2013@hnu.edu.cn"/>
  <meta name="citation_reference" content="Ahlgren, N. A., et al. (2016). “Alignment-free oligonucleotide frequency dissimilarity measure improves prediction of hosts from metagenomically-derived viral sequences.” Nucleic acids research 45(1): 39–53."/>
  <meta name="citation_reference" content="Akondy, R. S., et al. (2009). “The yellow fever virus vaccine induces a broad and polyfunctional human memory CD8+ T cell response.” The Journal of Immunology 183(12): 7919–7930."/>
  <meta name="citation_reference" content="Alavandi, S. and M. Poornima (2012). “Viral metagenomics: a tool for virus discovery and diversity in aquaculture.” Indian Journal of Virology 23(2): 88–98."/>
  <meta name="citation_reference" content="Barandiaran, I. (1998). “The random subspace method for constructing decision forests.” IEEE Trans. Pattern Anal. Mach. Intell 20(8)."/>
  <meta name="citation_reference" content="Baseman, J. G. and L. A. Koutsky (2005). “The epidemiology of human papillomavirus infections.” Journal of clinical virology 32: 16–24."/>
  <meta name="citation_reference" content="Bolotin, A., et al. (2005). “Clustered regularly interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal origin.” Microbiology 151(8): 2551–2561."/>
  <meta name="citation_reference" content="Breban, R., et al. (2013). “Interhuman transmissibility of Middle East respiratory syndrome coronavirus: estimation of pandemic risk.” The Lancet 382(9893): 694-699."/>
  <meta name="citation_reference" content="Breiman, L. (1996). “Bagging predictors.” Machine learning 24(2): 123–140."/>
  <meta name="citation_reference" content="Carroll, D., et al. (2018). “The global virome project.” Science 359(6378): 872-874."/>
  <meta name="citation_reference" content="Chen, C., et al. (2004). “Using random forest to learn imbalanced data.” University of California, Berkeley 110: 1–12."/>
  <meta name="citation_reference" content="de Oliveira Garcia M. H. (2019). “Zika: the continuing threat.” Bull World Health Organ 97: 6–7."/>
  <meta name="citation_reference" content="Edwards, R. A., et al. (2015). “Computational approaches to predict bacteriophage–host relationships.” FEMS microbiology reviews 40(2): 258–272."/>
  <meta name="citation_reference" content="Gao, R., et al. (2013). “Human infection with a novel avian-origin influenza A (H7N9) virus.” New England Journal of Medicine 368(20): 1888–1897."/>
  <meta name="citation_reference" content="Li, H. and F. Sun (2018). “Comparative studies of alignment, alignment-free and SVM based approaches for predicting the hosts of viruses based on viral sequences.” Scientific reports 8(1): 10032."/>
  <meta name="citation_reference" content="Maganga, G. D., et al. (2014). “Ebola virus disease in the Democratic Republic of Congo.” New England Journal of Medicine 371(22): 2083–2091."/>
  <meta name="citation_reference" content="Mihara, T., et al. (2016). “Linking virus genomes with host taxonomy.” Viruses 8(3): 66."/>
  <meta name="citation_reference" content="Mlakar, J., et al. (2016). “Zika virus associated with microcephaly.” New England Journal of Medicine 374(10): 951–958."/>
  <meta name="citation_reference" content="Paez-Espino, D., et al. (2016). “Uncovering Earth’s virome.” Nature 536(7617): 425."/>
  <meta name="citation_reference" content="Pedregosa, F., et al. (2011). “Scikit-learn: Machine learning in Python.” Journal of machine learning research 12(Oct): 2825–2830."/>
  <meta name="citation_reference" content="Prevention, E. C. f. D. and Control (2016). Zika virus epidemic in the Americas: potential association with microcephaly and Guillain-Barré syndrome (first update), ECDC Stockholm."/>
  <meta name="citation_reference" content="Rosenberg, R. (2015). “Detecting the emergence of novel, zoonotic viruses pathogenic to humans.” Cellular and molecular life sciences 72(6): 1115–1125."/>
  <meta name="citation_reference" content="Woolhouse, M. and E. Gaunt (2007). “Ecological origins of novel human pathogens.” Critical reviews in microbiology 33(4): 231–242."/>
  <meta name="citation_reference" content="Xu, B., et al. (2017). “Predicting the host of influenza viruses based on the word vector.” PeerJ 5: e3579."/>
  <meta name="twitter:title" content="Identification of viruses with the potential to infect human"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The virus has caused much mortality and morbidity to humans, and still posed a serious threat to the global public health. The virome with the human-infection potential is far from complete. Novel viruses have been discovered at an unprecedented pace as the rapid development of viral metagenomics. However, there is still a lack of a method for rapidly identifying the virus with the human-infection potential. This study built several machine learning models for discriminating the human-infecting viruses from other viruses based on the frequency of k-mers in the viral genomic sequences. The k-nearest neighbor (KNN) model could predict the human-infecting virus with an accuracy of over 90%. Even for the KNN models built on the contigs as short as 1kb, they performed comparably to those built on the viral genomes, suggesting that the models could be used to identify the human-infecting virus from the viral metagenomic sequences. This work could help for discovery of novel human-infecting virus in metagenomics studies."/>
  <meta name="og-title" property="og:title" content="Identification of viruses with the potential to infect human"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/597963v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The virus has caused much mortality and morbidity to humans, and still posed a serious threat to the global public health. The virome with the human-infection potential is far from complete. Novel viruses have been discovered at an unprecedented pace as the rapid development of viral metagenomics. However, there is still a lack of a method for rapidly identifying the virus with the human-infection potential. This study built several machine learning models for discriminating the human-infecting viruses from other viruses based on the frequency of k-mers in the viral genomic sequences. The k-nearest neighbor (KNN) model could predict the human-infecting virus with an accuracy of over 90%. Even for the KNN models built on the contigs as short as 1kb, they performed comparably to those built on the viral genomes, suggesting that the models could be used to identify the human-infecting virus from the viral metagenomic sequences. This work could help for discovery of novel human-infecting virus in metagenomics studies."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-04-05"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Identification of viruses with the potential to infect human | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-496104 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">The virus has caused much mortality and morbidity to humans, and still posed a serious threat to the global public health. The virome with the human-infection potential is far from complete. Novel viruses have been discovered at an unprecedented pace as the rapid development of viral metagenomics. However, there is still a lack of a method for rapidly identifying the virus with the human-infection potential. This study built several machine learning models for discriminating the human-infecting viruses from other viruses based on the frequency of k-mers in the viral genomic sequences. The k-nearest neighbor (KNN) model could predict the human-infecting virus with an accuracy of over 90%. Even for the KNN models built on the contigs as short as 1kb, they performed comparably to those built on the viral genomes, suggesting that the models could be used to identify the human-infecting virus from the viral metagenomic sequences. This work could help for discovery of novel human-infecting virus in metagenomics studies.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">Viruses are the most abundant biological entities on Earth and exist in all habitats of the world (
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Paez-Espino, Eloe-Fadrosh et al. 2016</a>). They can infect all kinds of organisms ranging from the bacteria to animals, including the human. Humans are constantly exposed to a large diversity of viruses, and over two-thirds of all human pathogens belong to viruses (
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Woolhouse and Gaunt 2007</a>). The viruses have caused huge mortality and morbidity to the human society in human history, such as the devastating smallpox and Spanish flu. Although much progress have been made in prevention and control of viral infectious disease, recent serial outbreaks caused by the Middle East Respiratory Syndrome coronavirus (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Breban, Riou et al. 2013</a>), avian influenza H7N9 virus (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gao, Cao et al. 2013</a>), Ebola virus (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Maganga, Kapetshi et al. 2014</a>) and Zika virus (
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Mlakar, Korva et al. 2016</a>) indicate that the viruses still pose a serious threat to the global public health.
    </p>
    <p id="p-5">The virome which could infect humans is far from complete. Generally speaking, a new pathogen would not be identified until it caused epidemics or pandemics. Many viruses may remain undiscovered, even if they have already entered into human populations (
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Rosenberg 2015</a>). Traditional diagnostic methods such as polymerase chain reaction, immunological assays and pan-viral microarrays are limited in their ability to identify novel viruses with the potential to infect humans. Rapid development of viral metagenomic sequencing in recent years provides a powerful high-throughput and culture-independent means to identify new viruses, which leads to accumulation of new viruses at an unprecedented pace (
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alavandi and Poornima 2012</a>). The Global Virome Project (GVP), which was proposed and initiated in the beginning of 2018, estimated that there were over 1.67 million yet-to-be-discovered viruses in animal reservoirs, and between 631,000 to 827,000 of these unknown viruses have the capacity to infect humans (
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carroll, Daszak et al. 2018</a>). Therefore, it is in great need to develop rapid methods for identifying viruses with the potential to infect humans.
    </p>
    <p id="p-6">Two kinds of methods, the sequence alignment-based and alignment-free methods, have been developed to predict the host for viruses. For example, several methods based on k-mers, which were extracted from viral genomes (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahlgren, Ren et al. 2016</a>, 
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li and Sun 2018</a>), and several methods based on sequence blast, have been developed to predict the host of the phage (
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bolotin, Quinquis et al. 2005</a>, 
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edwards, McNair et al. 2015</a>). Some studies have also attempted to identify the human virus with these methods. For example, Xu et al. (
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Xu, Tan et al. 2017</a>) developed SVM models to predict the host of influenza viruses based on word vectors. However, all these studies have focused some type of viruses, such as coronavirus and influenza virus. The methods they developed were not suitable for identification of novel human-infecting viruses from the viral metagenomic sequences. Here, we attempted to build machine learning models for rapid identification of viruses with the potential to infect human.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Virus-host interactions and viral genomes</h3>
     <p id="p-7">The virus with specific hosts and the viral genomic sequences were obtained from the database of Virus-Host DB (available at 
      <a href="https://www.genome.jp/virushostdb/">https://www.genome.jp/virushostdb/</a>) on July 15, 2018 (
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mihara, Nishimura et al. 2016</a>). After removing the viroid and satellites, and the viruses with genomic sequence less than 1kb, a total of 9428 viruses were kept for further analysis, which included 1236 viruses infecting humans, which were defined as human-infecting viruses, and 8192 viruses infecting other species.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Machine learning models</h3>
     <p id="p-8">The machine learning models of k-nearest neighbor (KNN) (k=1), support vector machine (SVM) (using the linear kernel function), gaussian naive bayes classifier (GNBC), random forest (RF) (with default settings) and logistic regression (LR) (with default settings) were built with the default parameters using the package “scikit-learn” (version 0.20.2) (
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Pedregosa, Varoquaux et al. 2011</a>) in Python (version 3.6.2). Because the number of human-infecting viruses was much smaller than that of other viruses, the functions of “BalanceBaggingClassifier” (
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Breiman 1996</a>, 
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barandiaran 1998</a>) and “BalanceRandomForest” (
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen, Liaw et al. 2004</a>) in the package of “imbalanced-learn” (version 0.4.3) in Python were used to deal with the imbalance between the human-infecting virus and other viruses in the modeling, with the parameter of “n_estimators” set to be 100.
     </p>
     <p id="p-9">Ten-fold cross-validations were used to evaluate the predictive performances of the machine learning models, and were conducted using the functions of ‘StratifiedKFold’ in the package “scikit-learn” in Python. To measure the predictive performances of the machine learning models, the area under the receiver operating characteristics (ROC) curve (AUC), the accuracy, recall rate, specificity and predictive precision were calculated for each model.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Statistical analysis</h3>
     <p id="p-10">All the statistical analysis was conducted in R (version 3.5.0).</p>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Results</h2>
    <div id="sec-7" class="subsection">
     <h3>Taxonomy distribution of viruses</h3>
     <p id="p-11">A total of 9428 viruses were used in the study. Among them, 1236 viruses were reported to infect humans, which were defined as the human-infecting virus. They included 1043 viruses exclusively infecting humans and 193 viruses both infecting humans and other species. The human-infecting virus covered all groups of viruses in the Baltimore classification (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Over 60% of the human-infecting virus belonged to single-stranded RNA (ssRNA) viruses, which also had the highest ratio of human-infecting viruses (28%). The second largest component (25%) of the human-infecting virus was the double-stranded DNA (dsDNA) virus. On the level of family, the human-infecting virus comprised of viruses from 30 viral families. Among them, the families of Caliciviridae (ssRNA), Picornaviridae (ssRNA) and Papillomaviridae (dsDNA) were the three most abundant ones, which accounted for nearly 60% of the human-infecting virus.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Analysis of the taxonomy distribution of all viruses used. (A) Taxonomy distribution of the human-infecting and other viruses in the Baltimore classification. (B) Taxonomy distribution of the human-infecting virus by viral family." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1258266839" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Analysis of the taxonomy distribution of all viruses used. (A) Taxonomy distribution of the human-infecting and other viruses in the Baltimore classification. (B) Taxonomy distribution of the human-infecting virus by viral family.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F1.medium.gif" width="440" height="210"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F1.medium.gif" width="440" height="210"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-12" class="first-child">Analysis of the taxonomy distribution of all viruses used. (A) Taxonomy distribution of the human-infecting and other viruses in the Baltimore classification. (B) Taxonomy distribution of the human-infecting virus by viral family.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-13">Besides the human-infecting virus, there were a total of 8192 viruses which infect other species except the human, including the archaea, bacteria, fungi, plant, animal, and so on. They also covered all groups of viruses in the Baltimore classification (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>), and comprised of viruses from 91 viral families.
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Machine-learning models for identifying the human-infecting virus based on k-mer frequencies in the genome</h3>
     <p id="p-14">To discriminating the human-infecting virus from other viruses, several machine-learning models, including k-nearest neighbor (KNN), random forest (RF), gaussian naive bayes classifier (GNBC), support-vector machine (SVM) and logistic regression (LR), were built based on k-mer frequencies in the viral genome. K-mers of one to six nucleotides were used in the models to evaluating the influence of the k-mer length on the model performance (Table S1). 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> showed that the AUC of GNBC, SVM and LR models increased as the increase of the k-mer length from one to six. While for KNN and RF, the AUCs of them peaked at k-mer length of 4 and 3, respectively; then, they began to decrease as the increase of k-mer length. The AUCs of KNN and RF were much larger than those of other models, including GNBC, SVM and LR, at all length of k-mers, suggesting that the KNN and RF outperformed other models in discriminating the human-infecting virus from other viruses. The best performance was achieved on the KNN model with the k-mers of four nucleotides. The model not only had the best overall performance, with accuracy of 0.90 and AUC of 0.92, respectively, but also had the best ability of capturing the human-infecting virus, with recall rate of 0.94.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The AUCs for machine learning models with k-mer length ranging from one to six." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1258266839" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;The AUCs for machine learning models with k-mer length ranging from one to six.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F2.medium.gif" width="440" height="316"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F2.medium.gif" width="440" height="316"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-15" class="first-child">The AUCs for machine learning models with k-mer length ranging from one to six.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Identifying the human-infecting virus based on contigs of varying length</h3>
     <p id="p-16">In the metagenomics studies, contigs of varying length were assembled, which range from several hundreds to several thousands of nucleotides. Rapid determination of the human-infecting virus from metagenomic sequences is important for early warnings of newly emerging viruses. To mimic the metagenomic sequences, viral genomes were split into non-overlapping contigs of 500, 1,000, 3,000, 5,000 and 10,000 nucleotides long. The number of contigs of varying length in the human-infecting virus and other viruses were listed in Table S2.</p>
     <p id="p-17">Since the KNN model performed best in discriminating the human-infecting virus from other viruses, it was used to identify the human-infecting virus based on contigs of varying length. For the contigs of a given length, KNN models for discriminating the human-infecting virus and other viruses were built with k-mers of different size (Table S3). As was shown in 
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, for contigs of all length, the AUC of KNN models increased as the increase of k-mer length from one to four. The best performance was achieved for all models with k-mers of four nucleotides long. The longer the contigs were, the better the models performed in identifying the human-infecting virus (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The model built on the contigs of 10,000 nucleotides had an AUC of 0.96 and a recall rate of 0.99, suggesting the model could capture most human-infecting viruses. While on the contig of 500 nucleotides, the model only had an AUC of 0.86 and a recall rate of 0.88. Attention to note, for models built on the contigs of 1000 nucleotides or longer, they performed similarly to those built on the viral genome.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/518719/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/518719/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/518719" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-18" class="first-child">Best performance of each kind of machine learning model.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/518718/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/518718/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/518718" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-19" class="first-child">Best performance of KNN models built on contigs of varying length with k-mers of four nucleotides used in modeling.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The AUCs for KNN models built on contigs of varying length, with k-mer length ranging from one to five." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1258266839" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;The AUCs for KNN models built on contigs of varying length, with k-mer length ranging from one to five.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F3.medium.gif" width="440" height="298"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F3.medium.gif" width="440" height="298"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/04/05/597963/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <p id="p-20" class="first-child">The AUCs for KNN models built on contigs of varying length, with k-mer length ranging from one to five.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Discussion</h2>
    <p id="p-21">Identification of human-infecting viruses is important for early warnings of newly emerging viruses. As the rapid accumulation of viral metagenomic sequences, more and more novel viruses have been discovered. Unfortunately, there is currently a lack of rapid methods for identification of viruses with the potential to infect humans. As far as we know, this study for the first time attempted to discriminate the human-infecting virus from non-human-infecting virus in the perspective of virome. The KNN model could predict the human-infecting virus with high accuracy and sensitivity. Even for the KNN models built on the contigs as short as 1kb, they performed comparably to those built on the viral genomes. This suggested that the models built here could be used in metagenomics for identifying the viruses with the potential of infecting humans.</p>
    <p id="p-22">The human-infecting virus used here has different ability of infecting humans. Some of them are human viruses and have been circulating in human populations for a long time, such as the human papillomavirus (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baseman and Koutsky 2005</a>) and yellow fever virus (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akondy, Monson et al. 2009</a>); while some of them were zoonotic and only infect humans rarely, such as the avian influenza virus and coronavirus. As was shown in numerous studies, the zoonotic viruses are very likely to cause epidemics or even pandemics after adaptive mutations. The most recent example is the Zika virus. It rarely caused wide-spread epidemics in humans, even in highly enzootic areas, in the 20
     <sup>th</sup> century. However, a few mutations on the viral genome facilitated its rapid spread in human populations. The virus has caused epidemics in more than 50 countries and has infected more than one million people since 2014 (
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Prevention and Control 2016</a>, de 
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Oliveira Garcia 2019</a>). Taken together, it is difficult to distinguish between viruses with different extent of infecting humans. Therefore, the human-infecting virus with varying ability of infecting humans were considered equally in the modeling.
    </p>
    <p id="p-23">There are some limitations to this study. Firstly, the human-infecting virus is far from complete when compared to that estimated by the GVP. But they covered viruses of 30 families in all groups of the Baltimore classification, which was similar to that estimated by the GVP. Besides, the machine learning models had excellent performances in discriminating the human-infecting virus from other viruses. They could help much in discovering novel human-infecting viruses. infect humans after adaptive mutations. Secondly, the number of human-infecting virus is much smaller than that of other viruses. Such a large imbalance may hinder accurate modeling. Here, the under-sampling method was used to deal with the imbalance problem. The KNN model not only achieved excellent overall performances, but also had high sensitivity in identifying the human-infecting virus.</p>
    <p id="p-24">In conclusion, this study for the first time built the computational models to predict the human-infecting virus in the perspective of virome. The high accuracy and sensitivity of the KNN model built on the viral contigs of varying length suggests that the model could be used to identify the human-infecting virus from the viral metagenomic sequences. This work could help for discovery of novel human-infecting virus in metagenomics studies.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">Reference</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ahlgren</span>, 
          <span class="cit-name-given-names">N. A.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2016</span>). “
         <span class="cit-article-title">Alignment-free oligonucleotide frequency dissimilarity measure improves prediction of hosts from metagenomically-derived viral sequences</span>.” 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-vol">45</span>(
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">39</span>–
         <span class="cit-lpage">53</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft.volume%253D45%26rft.spage%253D39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Akondy</span>, 
          <span class="cit-name-given-names">R. S.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2009</span>). “
         <span class="cit-article-title">The yellow fever virus vaccine induces a broad and polyfunctional human memory CD8+ T cell response</span>.” 
         <abbr class="cit-jnl-abbrev">The Journal of Immunology</abbr>
         <span class="cit-vol">183</span>(
         <span class="cit-issue">12</span>): 
         <span class="cit-fpage">7919</span>–
         <span class="cit-lpage">7930</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.volume%253D183%26rft.spage%253D7919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Alavandi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Poornima</span>
         </span> (
         <span class="cit-pub-date">2012</span>). “
         <span class="cit-article-title">Viral metagenomics: a tool for virus discovery and diversity in aquaculture</span>.” 
         <abbr class="cit-jnl-abbrev">Indian Journal of Virology</abbr>
         <span class="cit-vol">23</span>(
         <span class="cit-issue">2</span>): 
         <span class="cit-fpage">88</span>–
         <span class="cit-lpage">98</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIndian%2BJournal%2Bof%2BVirology%26rft.volume%253D23%26rft.spage%253D88%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Barandiaran</span>, 
          <span class="cit-name-given-names">I.</span>
         </span> (
         <span class="cit-pub-date">1998</span>). “
         <span class="cit-article-title">The random subspace method for constructing decision forests</span>.” 
         <abbr class="cit-jnl-abbrev">IEEE Trans. Pattern Anal. Mach. Intell</abbr>
         <span class="cit-vol">20</span>(
         <span class="cit-issue">8</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.5" data-doi="10.1016/j.jcv.2004.12.008">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baseman</span>, 
          <span class="cit-name-given-names">J. G.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. A.</span>
          <span class="cit-name-surname">Koutsky</span>
         </span> (
         <span class="cit-pub-date">2005</span>). “
         <span class="cit-article-title">The epidemiology of human papillomavirus infections</span>.” 
         <abbr class="cit-jnl-abbrev">Journal of clinical virology</abbr>
         <span class="cit-vol">32</span>: 
         <span class="cit-fpage">16</span>–
         <span class="cit-lpage">24</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bvirology%26rft.volume%253D32%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcv.2004.12.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jcv.2004.12.008&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.6" data-doi="10.1099/mic.0.28048-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bolotin</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2005</span>). “
         <span class="cit-article-title">Clustered regularly interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal origin</span>.” 
         <abbr class="cit-jnl-abbrev">Microbiology</abbr>
         <span class="cit-vol">151</span>(
         <span class="cit-issue">8</span>): 
         <span class="cit-fpage">2551</span>–
         <span class="cit-lpage">2561</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DBolotin%26rft.auinit1%253DA.%26rft.volume%253D151%26rft.issue%253D8%26rft.spage%253D2551%26rft.epage%253D2561%26rft.atitle%253DClustered%2Bregularly%2Binterspaced%2Bshort%2Bpalindrome%2Brepeats%2B%2528CRISPRs%2529%2Bhave%2Bspacers%2Bof%2Bextrachromosomal%2Borigin%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.28048-0%26rft_id%253Dinfo%253Apmid%252F16079334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1099/mic.0.28048-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16079334&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F05%2F597963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000231459000006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Breban</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013</span>). “
         <span class="cit-article-title">Interhuman transmissibility of Middle East respiratory syndrome coronavirus: estimation of pandemic risk</span>.” 
         <abbr class="cit-jnl-abbrev">The Lancet</abbr>
         <span class="cit-vol">382</span>(
         <span class="cit-issue">9893</span>): 
         <span class="cit-fpage">694</span>-
         <span class="cit-lpage">699</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D382%26rft.spage%253D694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.8" data-doi="10.1023/A:1018054314350">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Breiman</span>, 
          <span class="cit-name-given-names">L.</span>
         </span> (
         <span class="cit-pub-date">1996</span>). “
         <span class="cit-article-title">Bagging predictors</span>.” 
         <abbr class="cit-jnl-abbrev">Machine learning</abbr>
         <span class="cit-vol">24</span>(
         <span class="cit-issue">2</span>): 
         <span class="cit-fpage">123</span>–
         <span class="cit-lpage">140</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMachine%2Blearning%26rft.volume%253D24%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1018054314350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1023/A:1018054314350&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1996UZ38000003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.9" data-doi="10.1126/science.aap7463">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Carroll</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>). “
         <span class="cit-article-title">The global virome project</span>.” 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">359</span>(
         <span class="cit-issue">6378</span>): 
         <span class="cit-fpage">872</span>-
         <span class="cit-lpage">874</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCarroll%26rft.auinit1%253DD.%26rft.volume%253D359%26rft.issue%253D6378%26rft.spage%253D872%26rft.epage%253D874%26rft.atitle%253DThe%2BGlobal%2BVirome%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aap7463%26rft_id%253Dinfo%253Apmid%252F29472471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTkvNjM3OC84NzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNC8wNS81OTc5NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2004</span>). “
         <span class="cit-article-title">Using random forest to learn imbalanced data</span>.” 
         <abbr class="cit-jnl-abbrev">University of California, Berkeley</abbr>
         <span class="cit-vol">110</span>: 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">12</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DUniversity%2Bof%2BCalifornia%252C%2BBerkeley%26rft.volume%253D110%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">de Oliveira Garcia M.</span>
          <span class="cit-name-given-names">H.</span>
         </span> (
         <span class="cit-pub-date">2019</span>). “
         <span class="cit-article-title">Zika: the continuing threat</span>.” 
         <abbr class="cit-jnl-abbrev">Bull World Health Organ</abbr>
         <span class="cit-vol">97</span>: 
         <span class="cit-fpage">6</span>–
         <span class="cit-lpage">7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.volume%253D97%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Edwards</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>). “
         <span class="cit-article-title">Computational approaches to predict bacteriophage–host relationships</span>.” 
         <abbr class="cit-jnl-abbrev">FEMS microbiology reviews</abbr>
         <span class="cit-vol">40</span>(
         <span class="cit-issue">2</span>): 
         <span class="cit-fpage">258</span>–
         <span class="cit-lpage">272</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFEMS%2Bmicrobiology%2Breviews%26rft.volume%253D40%26rft.spage%253D258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.13" data-doi="10.1056/NEJMoa1304459">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013</span>). “
         <span class="cit-article-title">Human infection with a novel avian-origin influenza A (H7N9) virus</span>.” 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>
         <span class="cit-vol">368</span>(
         <span class="cit-issue">20</span>): 
         <span class="cit-fpage">1888</span>–
         <span class="cit-lpage">1897</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D368%26rft.spage%253D1888%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1304459%26rft_id%253Dinfo%253Apmid%252F23577628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa1304459&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23577628&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F05%2F597963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000318896200008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Sun</span>
         </span> (
         <span class="cit-pub-date">2018</span>). “
         <span class="cit-article-title">Comparative studies of alignment, alignment-free and SVM based approaches for predicting the hosts of viruses based on viral sequences</span>.” 
         <abbr class="cit-jnl-abbrev">Scientific reports</abbr>
         <span class="cit-vol">8</span>(
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">10032</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D8%26rft.spage%253D10032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.15" data-doi="10.1056/NEJMoa1411099">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Maganga</span>, 
          <span class="cit-name-given-names">G. D.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2014</span>). “
         <span class="cit-article-title">Ebola virus disease in the Democratic Republic of Congo</span>.” 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>
         <span class="cit-vol">371</span>(
         <span class="cit-issue">22</span>): 
         <span class="cit-fpage">2083</span>–
         <span class="cit-lpage">2091</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D371%26rft.spage%253D2083%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1411099%26rft_id%253Dinfo%253Apmid%252F25317743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa1411099&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25317743&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F05%2F597963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.16" data-doi="10.3390/v8030066">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mihara</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2016</span>). “
         <span class="cit-article-title">Linking virus genomes with host taxonomy</span>.” 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>
         <span class="cit-vol">8</span>(
         <span class="cit-issue">3</span>): 
         <span class="cit-fpage">66</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D8%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv8030066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3390/v8030066&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.17" data-doi="10.1056/NEJMoa1600651">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mlakar</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2016</span>). “
         <span class="cit-article-title">Zika virus associated with microcephaly</span>.” 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>
         <span class="cit-vol">374</span>(
         <span class="cit-issue">10</span>): 
         <span class="cit-fpage">951</span>–
         <span class="cit-lpage">958</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D374%26rft.spage%253D951%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1600651%26rft_id%253Dinfo%253Apmid%252F26862926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa1600651&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26862926&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F05%2F597963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.18" data-doi="10.1038/nature19094">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Paez-Espino</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2016</span>). “
         <span class="cit-article-title">Uncovering Earth’s virome</span>.” 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">536</span>(
         <span class="cit-issue">7617</span>): 
         <span class="cit-fpage">425</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D425%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19094%26rft_id%253Dinfo%253Apmid%252F27533034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature19094&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27533034&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F05%2F597963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pedregosa</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2011</span>). “
         <span class="cit-article-title">Scikit-learn: Machine learning in Python</span>.” 
         <abbr class="cit-jnl-abbrev">Journal of machine learning research</abbr>
         <span class="cit-vol">12</span>(
         <span class="cit-month">Oct</span>): 
         <span class="cit-fpage">2825</span>–
         <span class="cit-lpage">2830</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmachine%2Blearning%2Bresearch%26rft.volume%253D12%26rft.spage%253D2825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=000298103200003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Prevention</span>, 
          <span class="cit-name-given-names">E. C. f. D.</span>
         </span> and Control (
         <span class="cit-pub-date">2016</span>). 
         <abbr class="cit-jnl-abbrev">Zika virus epidemic in the Americas: potential association with microcephaly and Guillain-Barré syndrome (first update), ECDC Stockholm</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rosenberg</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2015</span>). “
         <span class="cit-article-title">Detecting the emergence of novel, zoonotic viruses pathogenic to humans</span>.” 
         <abbr class="cit-jnl-abbrev">Cellular and molecular life sciences</abbr>
         <span class="cit-vol">72</span>(
         <span class="cit-issue">6</span>): 
         <span class="cit-fpage">1115</span>–
         <span class="cit-lpage">1125</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%26rft.volume%253D72%26rft.spage%253D1115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.22" data-doi="10.1080/10408410701647560">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Woolhouse</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Gaunt</span>
         </span> (
         <span class="cit-pub-date">2007</span>). “
         <span class="cit-article-title">Ecological origins of novel human pathogens</span>.” 
         <abbr class="cit-jnl-abbrev">Critical reviews in microbiology</abbr>
         <span class="cit-vol">33</span>(
         <span class="cit-issue">4</span>): 
         <span class="cit-fpage">231</span>–
         <span class="cit-lpage">242</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCritical%2Breviews%2Bin%2Bmicrobiology%26rft.stitle%253DCrit%2BRev%2BMicrobiol%26rft.aulast%253DWoolhouse%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D231%26rft.epage%253D242%26rft.atitle%253DEcological%2Borigins%2Bof%2Bnovel%2Bhuman%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10408410701647560%26rft_id%253Dinfo%253Apmid%252F18033594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/10408410701647560&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18033594&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F04%2F05%2F597963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000251710500001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-597963v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>). “
         <span class="cit-article-title">Predicting the host of influenza viruses based on the word vector</span>.” 
         <abbr class="cit-jnl-abbrev">PeerJ</abbr>
         <span class="cit-vol">5</span>: 
         <span class="cit-fpage">e3579</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D5%26rft.spage%253De3579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
