<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.04.06.028316.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.06.028316v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Extraction-free COVID-19 (SARS-CoV-2) diagnosis by RT-PCR to increase capacity for national testing programmes during a pandemic"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.06.028316"/>
  <meta name="DC.Date" content="2020-04-09"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) causes Coronavirus disease 2019 (COVID-19), a respiratory tract infection. The standard molecular diagnostic test is a multistep process involving viral RNA extraction and real-time quantitative reverse transcriptase PCR (qRT-PCR). Laboratories across the globe face constraints on equipment and reagents during the COVID-19 pandemic. We have developed a simplified qRT-PCR assay that removes the need for an RNA extraction process and can be run on a real-time thermal cycler. The assay uses custom primers and probes, and maintains diagnostic sensitivity within 98.0% compared to the assay run on a high-throughput, random-access automated platform, the Panther Fusion (Hologic). This assay can be used to increase capacity for COVID-19 testing for national programmes worldwide. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Paul R Grant"/>
  <meta name="DC.Contributor" content="Melanie A Turner"/>
  <meta name="DC.Contributor" content="Gee Yen Shin"/>
  <meta name="DC.Contributor" content="Eleni Nastouli"/>
  <meta name="DC.Contributor" content="Lisa J Levett"/>
  <meta name="article:published_time" content="2020-04-09"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Extraction-free COVID-19 (SARS-CoV-2) diagnosis by RT-PCR to increase capacity for national testing programmes during a pandemic"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) causes Coronavirus disease 2019 (COVID-19), a respiratory tract infection. The standard molecular diagnostic test is a multistep process involving viral RNA extraction and real-time quantitative reverse transcriptase PCR (qRT-PCR). Laboratories across the globe face constraints on equipment and reagents during the COVID-19 pandemic. We have developed a simplified qRT-PCR assay that removes the need for an RNA extraction process and can be run on a real-time thermal cycler. The assay uses custom primers and probes, and maintains diagnostic sensitivity within 98.0% compared to the assay run on a high-throughput, random-access automated platform, the Panther Fusion (Hologic). This assay can be used to increase capacity for COVID-19 testing for national programmes worldwide.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.06.028316v2"/>
  <meta name="citation_id" content="2020.04.06.028316v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028316v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028316v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028316v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.06.028316.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.06.028316"/>
  <meta name="citation_num_pages" content="6"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.06.028316"/>
  <meta name="citation_author" content="Paul R Grant"/>
  <meta name="citation_author_institution" content="Health Services Laboratories"/>
  <meta name="citation_author_email" content="paul.grant@hslpathology.com"/>
  <meta name="citation_author" content="Melanie A Turner"/>
  <meta name="citation_author_institution" content="Health Services Laboratories"/>
  <meta name="citation_author" content="Gee Yen Shin"/>
  <meta name="citation_author_institution" content="Health Services Laboratories"/>
  <meta name="citation_author_institution" content="University College London Hospitals NHS Trust"/>
  <meta name="citation_author" content="Eleni Nastouli"/>
  <meta name="citation_author_institution" content="University College London Hospitals NHS Trust"/>
  <meta name="citation_author" content="Lisa J Levett"/>
  <meta name="citation_author_institution" content="Health Services Laboratories"/>
  <meta name="citation_reference" content="Wu F, Zhao S, Yu B, Chen YM, Wang W, Song ZG, Hu Y, Tao ZW, Tian JH, Pei YY, Yuan ML, Zhang YL, Dai FH, Liu Y, Wang QM, Zheng JJ, Xu L, Holmes EC, Zhang YZ. A new coronavirus associated with human respiratory disease in China. Nature. 2020 Mar;579(7798):265–269. doi: 10.1038/s41586-020-2008-3. Epub 2020 Feb 3."/>
  <meta name="citation_reference" content="First data on stability and resistance of SARS coronavirus compiled by members of WHO laboratory network. https://www.who.int/csr/sars/survival_2003_05_04/en/#"/>
  <meta name="citation_reference" content="Emery SL, Erdman DD, Bowen MD, Newton BR, Winchell JM, Meyer RF, Tong S, Cook BT, Holloway BP, McCaustland KA, Rota PA, Bankamp B, Lowe LE, Ksiazek TG, Bellini WJ, Anderson LJ. Real-time reverse transcription-polymerase chain reaction assay for SARS-associated coronavirus. Emerg Infect Dis. 2004 Feb;10(2):311–6."/>
  <meta name="twitter:title" content="Extraction-free COVID-19 (SARS-CoV-2) diagnosis by RT-PCR to increase capacity for national testing programmes during a pandemic"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) causes Coronavirus disease 2019 (COVID-19), a respiratory tract infection. The standard molecular diagnostic test is a multistep process involving viral RNA extraction and real-time quantitative reverse transcriptase PCR (qRT-PCR). Laboratories across the globe face constraints on equipment and reagents during the COVID-19 pandemic. We have developed a simplified qRT-PCR assay that removes the need for an RNA extraction process and can be run on a real-time thermal cycler. The assay uses custom primers and probes, and maintains diagnostic sensitivity within 98.0% compared to the assay run on a high-throughput, random-access automated platform, the Panther Fusion (Hologic). This assay can be used to increase capacity for COVID-19 testing for national programmes worldwide. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Extraction-free COVID-19 (SARS-CoV-2) diagnosis by RT-PCR to increase capacity for national testing programmes during a pandemic"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028316v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) causes Coronavirus disease 2019 (COVID-19), a respiratory tract infection. The standard molecular diagnostic test is a multistep process involving viral RNA extraction and real-time quantitative reverse transcriptase PCR (qRT-PCR). Laboratories across the globe face constraints on equipment and reagents during the COVID-19 pandemic. We have developed a simplified qRT-PCR assay that removes the need for an RNA extraction process and can be run on a real-time thermal cycler. The assay uses custom primers and probes, and maintains diagnostic sensitivity within 98.0% compared to the assay run on a high-throughput, random-access automated platform, the Panther Fusion (Hologic). This assay can be used to increase capacity for COVID-19 testing for national programmes worldwide. ### Competing Interest Statement The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-09"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Extraction-free COVID-19 (SARS-CoV-2) diagnosis by RT-PCR to increase capacity for national testing programmes during a pandemic | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1236085 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) causes Coronavirus disease 2019 (COVID-19), a respiratory tract infection. The standard molecular diagnostic test is a multistep process involving viral RNA extraction and real-time quantitative reverse transcriptase PCR (qRT-PCR). Laboratories across the globe face constraints on equipment and reagents during the COVID-19 pandemic. We have developed a simplified qRT-PCR assay that removes the need for an RNA extraction process and can be run on a real-time thermal cycler. The assay uses custom primers and probes, and maintains diagnostic sensitivity within 98.0% compared to the assay run on a high-throughput, random-access automated platform, the Panther Fusion (Hologic). This assay can be used to increase capacity for COVID-19 testing for national programmes worldwide.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-5">Coronavirus disease 2019 (COVID-19) is a respiratory tract infection caused by a newly emergent coronavirus – Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) – which was first recognised in Wuhan, Hubei Province, China, in December 2019. Genetic sequencing of the virus suggests that SARS-CoV-2 is a betacoronavirus closely linked to SARS coronavirus 1 (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Wu et al.2020</a>).
    </p>
    <p id="p-6">The standard molecular diagnostic test for this virus is a multistep process involving viral RNA extraction and real-time quantitative reverse transcriptase PCR (qRT-PCR). Although many companies have produced PCR kits to amplify the viral RNA, RNA extraction at any scale in a diagnostic laboratory is performed on a limited number of automated platforms that require specific reagents and consumables. This has led to significant effort to build large laboratories with existing research equipment to increase testing capacity, and to extract RNA on more open platforms that enable non-specific reagents and plastics to be used.</p>
    <p id="p-7">The COVID-19 pandemic placed severe constraints on the availability of laboratory equipment, reagents and consumables required for molecular diagnostics in the UK and Europe. This delayed the ability to scale-up testing capacity required for healthcare and population screening.</p>
    <p id="p-8">At Health Services Laboratories (HSL), we developed a qRT-PCR that can be run on a high-throughput, random-access automated platform, the Panther Fusion (Hologic). Using the Open Access facility on this platform, custom primers and probes designed in-house can be added to a DNA/RNA extraction cartridge. In London, this qRT-PCR was used for large-scale testing of patients hospitalized with suspected COVID-19. However, the COVID-19 pandemic also led to these cartridges being in short supply.</p>
    <p id="p-9">Using the same primers and probes, we have now developed a qRT-PCR that can be run on a real-time thermal cycler without the need for an RNA extraction process. This qRT-PCR maintains sensitivity to within 98.0% of the assay run on the Panther Fusion.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Method</h2>
    <p id="p-10">A panel of SARS-CoV-2 positive and negative samples was used to compare the RNA extraction and RNA-extraction free methods.</p>
    <div id="sec-3" class="subsection">
     <h3>RNA extraction</h3>
     <p id="p-11">100 μL viral transport medium (VTM) from a swab was added to 100 μL Qiagen Lysis buffer containing guanidinum to inactivate the virus. This was then processed either on a QIAsymphony SP using the QIAsymphony DSP Virus/pathogen Mini kit and Complex 200 protocol, or the EZ1 Advanced XL using the EZ1 DSP virus kit. The elution volume was set to 60 μL, and 10 μL of the purified RNA was added to the PCR.</p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Direct sample transfer</h3>
     <p id="p-12">10 μL, 5 μL and 2 μL of sample expressed in viral transport medium was added directly to the PCR without any heating step, the plate was sealed and thermal cycling begun.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Sample heating prior to direct transfer</h3>
     <p id="p-13">50 μL of sample expressed in viral transport medium was added to a PCR tube and heated to 95 °C for 10 mins prior to loading into the PCR at 10ul, 5 μL and 2 μL.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>RT-PCR</h3>
     <p id="p-14">A 20 μL reaction containing 10 μL RNA, 5 μL 4x TaqMan Fast Virus 1-step Master Mix (Applied Biosystems) and 5 μL primer and probe mix as shown in 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Where VTM was added directly to the PCR at 5 μL or 2 μL, this was made up to 10 μL with RNase-Free water.
     </p>
     <p id="p-15">Cycling was performed at 56 °C for 15 min for reverse transcription, followed by 95 °C for 20 sec and then 45 cycles of 95 °C for 3 s, 60 °C for 30s using an Applied Biosystems QuantStudio 5 real-time PCR system (ThermoFisher Scientific).</p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1240461/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1240461/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1240461" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1:</span>
       <p id="p-16" class="first-child">Primer/Probe sequences and concentrations</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Panther Fusion Open Access Assay</h3>
     <p id="p-17">Panther Fusion PPR was made with the SARS-CoV-2 N gene primers and probe (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and Hologic internal RNA IC control primers and probe. A PPR tube for 40 reactions was made by adding 3 μL each N gene primer or probe (at 100 μM stock) and 8 μL of the RNA IC primer and probe (Supplied by Hologic). The salts were KCl at 100 mM (37.5 μL of 2M stock), MgCl
      <sub>2</sub> at 3mM (4.5 μL of 1M stock) and Tris (pH 8.0) at 8mM (12 μL of 1M stock). This primer probe salt mix was made up to 1200 μL with water and overlaid with 400 μL mineral oil (Hologic) in a 2ml PPR tube (Hologic). This assay was run on the Panther Fusion (Hologic) with and Open Access RNA/DNA enzyme cartridge.
     </p>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Results</h2>
    <p id="p-18">The RNA extraction method was compared to the direct addition of samples to the RT-PCR with and without prior heating.</p>
    <p id="p-19">When 10 μL of the heated or unheated sample was added to the PCR, no amplification was observed. Both the direct addition methods gave lower median Ct values than those added after heat treatment and were equivalent to the EZ1 (Qiagen) extraction (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.06.028316/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Median and interquartile range for Ct values obtained using different methods" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1922589864" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Median and interquartile range for Ct values obtained using different methods&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.06.028316/F1.medium.gif" width="440" height="256"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.06.028316/F1.medium.gif" width="440" height="256"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.06.028316/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.06.028316/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1:</span>
      <p id="p-20" class="first-child">Median and interquartile range for Ct values obtained using different methods</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-21">The lowest Ct values were achieved by adding 2 μL of the VTM direct to the PCR without any prior heating (median Ct value 25.74 vs 29.51 using EZ1 RNA). This method was selected for further analysis.</p>
    <p id="p-22">The direct addition of 2 μL sample to the PCR was compared to the standard method in use within the clinical laboratory using the Open access channel of the Panther Fusion. An overall accuracy of 98.8% was achieved compared to the Panther Fusion assay (see 
     <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).
    </p>
    <div id="T2" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1240460/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1240460/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1240460" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 2:</span>
      <p id="p-23" class="first-child">Diagnostic accuracy</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-24">The analytical sensitivity was compared by diluting a positive clinical sample to end point, and testing using the extracted RNA and adding VTM directly to the PCR. The results are shown in 
     <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>.
    </p>
    <div id="T3" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1240463/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1240463/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1240463" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 3:</span>
      <p id="p-25" class="first-child">Analytical sensitivity</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-26">There was no cross reactivity with conventional coronavirus types OC43, NL63, 229E, HKU1, and a number of other respiratory viruses such as influenza A and B, respiratory syncytial virus, parainfluenza, metapneumovirus, adenovirus, rhinovirus and enterovirus.</p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Conclusion</h2>
    <p id="p-27">Direct addition of samples to the qRT-PCR without extraction with a diagnostic sensitivity of 98.0%, specificity of 100% and accuracy of 98.8% compared to the method on the Panther Fusion. This simplifies the process for COVID-19 testing, and will enable increased capacity in diagnostic laboratories.</p>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Discussion</h2>
    <p id="p-28">Implementation of this method will enable laboratories to provide a COVID-19 testing service without the need for RNA extraction equipment, reagents and consumables. Turn-around times are similar to those of high-throughput RNA-based assays, and faster than a two-step RNA extraction and qRT-PCR. Capacity can be significantly increased without the extraction step but is dependent on the number of safety cabinets for swab processing and number of real time PCR thermal cyclers.</p>
    <p id="p-29">Heating at 56°C for 15 minutes causes SARS CoV (SARS coronavirus) to lose Infectivity (WHO, 2003). Health and safety assessments have been completed and the process has been deemed safe to perform with relevant precautions and safety practices. Samples can be processed in batches of 96, each batch takes 56 minutes to run the rtPCR on the thermal cycler, the rate limiting step being the swab processing. Lower numbers would be processed more rapidly, within an equivalent time to a point-of-care test. Standard swab processing can be automated to speed up the initial process on a large scale.</p>
    <p id="p-30">Many laboratories use real-time thermal cyclers, so this method can be used to increase national screening capacity without the need for other specialized equipment or RNA extraction reagents.</p>
    <p id="p-31">Applying an extraction-free PCR protocol as described here would avoid limitations on COVID-19 screening capacity in the UK and elsewhere caused by global PCR reagent supply constraints. We recommend this method is explored further by other medical laboratories using alternative PCR reagents to improve the resilience and capacity of virology laboratories during the pandemic. The sensitivity of the assay will be dependent upon the PCR efficiency, and so other PCR protocols will need to be carefully evaluated with this new approach.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-1">
      <p id="p-4">Added Panther Fusion SARS-CoV-2 method</p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028316v2.1" data-doi="10.1038/s41586-020-2008-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">YM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">ZG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tao</span>
          <span class="cit-name-given-names">ZW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>
          <span class="cit-name-given-names">JH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pei</span>
          <span class="cit-name-given-names">YY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">ML</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">YL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dai</span>
          <span class="cit-name-given-names">FH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">QM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">JJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">EC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">YZ</span>
         </span>. 
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">2020</span>
         <span class="cit-month">Mar</span>;
         <span class="cit-vol">579</span>(
         <span class="cit-issue">7798</span>):
         <span class="cit-fpage">265</span>–
         <span class="cit-lpage">269</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> Epub 2020 Feb 3.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32015508&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.06.028316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.04.06.028316v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-source">First data on stability and resistance of SARS coronavirus compiled by members of WHO laboratory network</span>. 
         <a href="https://www.who.int/csr/sars/survival_2003_05_04/en/#">https://www.who.int/csr/sars/survival_2003_05_04/en/#</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.028316v2.3" data-doi="10.3201/eid1002.030759">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Emery</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Erdman</span>
          <span class="cit-name-given-names">DD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bowen</span>
          <span class="cit-name-given-names">MD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Newton</span>
          <span class="cit-name-given-names">BR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Winchell</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meyer</span>
          <span class="cit-name-given-names">RF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tong</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cook</span>
          <span class="cit-name-given-names">BT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holloway</span>
          <span class="cit-name-given-names">BP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McCaustland</span>
          <span class="cit-name-given-names">KA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rota</span>
          <span class="cit-name-given-names">PA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bankamp</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lowe</span>
          <span class="cit-name-given-names">LE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ksiazek</span>
          <span class="cit-name-given-names">TG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bellini</span>
          <span class="cit-name-given-names">WJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>
          <span class="cit-name-given-names">LJ</span>
         </span>. 
         <span class="cit-article-title">Real-time reverse transcription-polymerase chain reaction assay for SARS-associated coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>. 
         <span class="cit-pub-date">2004</span>
         <span class="cit-month">Feb</span>;
         <span class="cit-vol">10</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">311</span>–
         <span class="cit-lpage">6</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DEmery%26rft.auinit1%253DS.%2BL.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D311%26rft.epage%253D316%26rft.atitle%253DReal-time%2Breverse%2Btranscription-polymerase%2Bchain%2Breaction%2Bassay%2Bfor%2BSARS-associated%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1002.030759%26rft_id%253Dinfo%253Apmid%252F15030703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid1002.030759&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15030703&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.06.028316.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000188867700028&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
