<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/10/09/798546.atom" />
<meta name="HW.pisa" content="biorxiv;798546v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly" />
<meta name="DC.Identifier" content="10.1101/798546" />
<meta name="DC.Date" content="2019-10-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Here we present a novel protocol for the construction of saturation single-site—and massive multi-site—mutant libraries of a bacteriophage. We segmented the ΦX174 genome into 14 non-toxic and non-replicative fragments compatible with golden gate assembly. We next used nicking mutagenesis with oligonucleotides prepared from unamplified oligo pools with individual segments as templates to prepare near-comprehensive single-site mutagenesis libraries of genes encoding the F capsid protein (421 amino acids scanned) and G spike protein (172 amino acids scanned). Libraries possessed greater than 99% of all 11,860 programmed mutations. Golden Gate cloning was then used to assemble the complete ΦX174 mutant genome and generate libraries of infective viruses. This protocol will enable reverse genetics experiments for studying viral evolution and, with some modifications, can be applied for engineering of therapeutically relevant bacteriophages with larger genomes." />
<meta name="DC.Contributor" content="Matthew S. Faber" />
<meta name="DC.Contributor" content="James T. Van Leuven" />
<meta name="DC.Contributor" content="Martina M. Ederer" />
<meta name="DC.Contributor" content="Yesol Sapozhnikov" />
<meta name="DC.Contributor" content="Zoë L. Wilson" />
<meta name="DC.Contributor" content="Holly A. Wichman" />
<meta name="DC.Contributor" content="Timothy A. Whitehead" />
<meta name="DC.Contributor" content="Craig R. Miller" />
<meta name="article:published_time" content="2019-10-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Here we present a novel protocol for the construction of saturation single-site—and massive multi-site—mutant libraries of a bacteriophage. We segmented the ΦX174 genome into 14 non-toxic and non-replicative fragments compatible with golden gate assembly. We next used nicking mutagenesis with oligonucleotides prepared from unamplified oligo pools with individual segments as templates to prepare near-comprehensive single-site mutagenesis libraries of genes encoding the F capsid protein (421 amino acids scanned) and G spike protein (172 amino acids scanned). Libraries possessed greater than 99% of all 11,860 programmed mutations. Golden Gate cloning was then used to assemble the complete ΦX174 mutant genome and generate libraries of infective viruses. This protocol will enable reverse genetics experiments for studying viral evolution and, with some modifications, can be applied for engineering of therapeutically relevant bacteriophages with larger genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;798546v1" />
<meta name="citation_id" content="798546v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/798546v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/798546v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/798546v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546.full.pdf" />
<meta name="citation_doi" content="10.1101/798546" />
<meta name="citation_num_pages" content="31" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="798546" />
<meta name="citation_author" content="Matthew S. Faber" />
<meta name="citation_author_institution" content="Dept. Biochemistry and Molecular Biology, Michigan State University" />
<meta name="citation_author" content="James T. Van Leuven" />
<meta name="citation_author_institution" content="Institute for Modeling Collaboration and Innovation, University of Idaho" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, University of Idaho" />
<meta name="citation_author_email" content="jvanleuven@uidaho.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2730-0287" />
<meta name="citation_author" content="Martina M. Ederer" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, University of Idaho" />
<meta name="citation_author" content="Yesol Sapozhnikov" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, University of Idaho" />
<meta name="citation_author" content="Zoë L. Wilson" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, University of Idaho" />
<meta name="citation_author" content="Holly A. Wichman" />
<meta name="citation_author_institution" content="Institute for Modeling Collaboration and Innovation, University of Idaho" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, University of Idaho" />
<meta name="citation_author" content="Timothy A. Whitehead" />
<meta name="citation_author_institution" content="Dept. of Chemical &amp; Biological Engineering; University of Colorado" />
<meta name="citation_author_institution" content="Dept. of Chemical Engineering &amp; Materials Science; Michigan State University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3177-1361" />
<meta name="citation_author" content="Craig R. Miller" />
<meta name="citation_author_institution" content="Institute for Modeling Collaboration and Innovation, University of Idaho" />
<meta name="citation_author_institution" content="Dept. of Biological Sciences, University of Idaho" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7920-4994" />
<meta name="citation_reference" content="Ando, H., Lemire, S., Pires, D.P., Lu, T.K. (2015) Engineering modular viral scaffolds for targeted bacterial population editing. Cell Systems 1:187–196." />
<meta name="citation_reference" content="Aoyama, A., Hamatake, R.K., Hayashi, M. (1981) Morphogenesis of ΦX174: in vitro synthesis of infectious phage from purified viral components. Proc. Natl. Aca. Sci. USA 12, 7285–7289." />
<meta name="citation_reference" content="Barrick, J.E., Lenski, R.E. (2013) Genome dynamics during experimental evolution. Nat Rev Genetics 14:827–839." />
<meta name="citation_reference" content="Bernal, R.A., Hafenstein, S., Esmeralda, R., Fane, B.A. (2004) Rossmann, M.G. The ΦX174 protein J mediates DNA packaging and viral attachment to host cells. J. Mol. Biol. 337, 1109–1122." />
<meta name="citation_reference" content="Bull, J.J., Badgett, M.R., Wichman, H.A., Huelsenbeck, J.P., Hillis, D.M., Gulati, A., Ho, C., and Molineux, I.J. (1997) Exceptional Convergent Evolution in a Virus. Genetics 147, 1497–1507." />
<meta name="citation_reference" content="Caudle, S.B., Miller, C.R., and Rokyta, D.R. (2014) Environment Determines Epistatic Patterns for a ssDNA Virus. Genetics 196, 267–279." />
<meta name="citation_reference" content="Dedrick, R.M., Guerrero-Bustamante, C.A., Garlena, R.A., Russell, D.A., Ford, K., Harris, K., Gilmour, K.C., Soothill, J., Jacobs-Sera, D., Schooley, R.T., Hatfull, G.F., Spencer, H. (2019) Engineered bacteriophages for treatment of a patient with a disseminated drug-resistant Mycobacterium abscessus. Nature Medicine 25, 730–733." />
<meta name="citation_reference" content="Dokland, T., McKenna, R., Ilag, L.L., Bowman, B.R., Incardona, N.L., Fane, B.A., Rossmann, M.G. (1997) Structure of a viral procapsid with molecular scaffolding. Nature 389, 308–313." />
<meta name="citation_reference" content="Donaldson, E.F., Yount, B., Sims, A.C., Burkett, S., Pickles, R.J., Baric, R.S. (2008) Systematic Assembly of a Full-Length Infectious Clone of Human Coronavirus NL63. J. Virol. 82, 11948–11957." />
<meta name="citation_reference" content="Doore, S.M., Fane, B.A. (2015) The kinetic and thermodynamic aftermath of horizontal gene transfer governs evolutionary recovery. Mol. Biol. Evo. 32, 2571–2584." />
<meta name="citation_reference" content="Doore, S.M., Schweers, N.J., Fane, B.A. (2017) Elevating fitness after a horizontal gene exchange in bacteriophage ΦX174. Virology 501, 25–34." />
<meta name="citation_reference" content="Doub, M.B., Lee, J.M., Bloom, J.D. (2018) How single mutations affect viral escape from broad and narrow antibodies to H1 influenza hemagglutinin. Nature Comm. 9, 1386." />
<meta name="citation_reference" content="Edwards, R.A, Rohwer, F. (2005) Viral metagenomics. Nature Reviews Micro. 3:504–509." />
<meta name="citation_reference" content="Faber, M.S., Wrenbeck, E.W., Azouz, L.R., Steiner, P.J., Whitehead, T.A. (2019) Impact of in vivo protein folding probability on local fitness landscapes. Mol. Biol. Evo. 10.1093/molbev/msz184" />
<meta name="citation_reference" content="Firnberg, E., Labonte, J.W., Gray, J.J., Ostermeier, M. (2014) A comprehensive, high resolution map of a gene’s fitness landscape. Mol. Biol. Evol. 31:1581–1592." />
<meta name="citation_reference" content="Fowler, D.M., Fields, S. (2014) Deep mutational scanning: a new stype of protein science. Nat. Methods 11:801–807." />
<meta name="citation_reference" content="Garamella, J., Marshall, R., Rustad, M., Noireaux, V. (2016) The all E. coli Toolbox 2.0: a platform for cell-free synthetic biology. ACS Synth. Biol. 5:344–355." />
<meta name="citation_reference" content="Hafenstein, S., Fane, B.A. (2002) ΦX174 genome-capsid interactions influence the biophysical properties of the virion: evidence for a scaffolding-like function for the genome during the final stages of morphogenesis. J. of Viro. 76, 5350–5356." />
<meta name="citation_reference" content="Hartman, E.C., Jakobson, C.M., Favor, A.H., Lobba, M.J., Álvarez-Benedicto, E., Francis, M.B., Tullman-Ercek, D. (2018) Quantitative characterization of all single amino acid variants of a viral capsid based drug delivery vehicle. Nature Comm. 9, 1385." />
<meta name="citation_reference" content="Holmes, E.C. (2013) What can we predict about viral evolution and emergence? Curr. Opin. in Vir. 3:180–184." />
<meta name="citation_reference" content="Jaschke, P.R., Lieberman, E.K., Rodriguez, J., Sierra, A., Endy, D. (2012) A fully decompressed synthetic bacteriophage ΦX174 genome assembled and archived in yeast. Virology 434:278–284." />
<meta name="citation_reference" content="Jaschke, P.R., Dotson, G.A., Hung, K., Liu, D., and Endy, D. (2018) Definitive demonstration by synthesis of genome annotation completeness. BioRxiv 455428." />
<meta name="citation_reference" content="Kiro, R., Shitrit, D., Qimron, U. (2014) Efficient engineering of a bacteriophage genome using the type I-E CRISPR-Cas system. RNA Biol. 11:42–44." />
<meta name="citation_reference" content="Klesmith, J.R., Hackel, B.J. (2018) Improved mutation function prediction via PACT: Protein Analysis and Classifier Toolkit. Bioinformatics 35(6):2707–2712." />
<meta name="citation_reference" content="Kowalsky, C.A., Klesmith, J.R., Stapleton, J.A., Kelly, V., Reichkitzer, N., Whitehead T.A. (2015) High-Resolution Sequence-Function Mapping of Full-Length Proteins. PLoS One 10:e0118193." />
<meta name="citation_reference" content="Kosuri, S., Church, G.M. (2014) Large-scale de novo DNA synthesis: technologies and applications. Nat. Methods 11, 499–507." />
<meta name="citation_reference" content="Lee, J.M., Huddleston, J., Doud, M.B., Hooper, K.A., Wu, N.C., Bedford, T., Bloom, J.D. (2018) Deep mutational scanning of hemagglutinin helps predict evolutionary fates of human H3N2 influenza variants. Proc. Natl. Aca. Sci. USA 115, E8276–E8285." />
<meta name="citation_reference" content="Lee, M.E., DeLoache, W.C., Cervantes, B., Dueber, J.E. (2015) A highly characterized yeast toolkit for modular, multipart assembly. ACS Synth. Biol. 4:975–986." />
<meta name="citation_reference" content="Lemay, M.L., Tremblay, D.M., Moineau, S. (2017) Genome engineering of virulent lactococcal phages using CRIPSR-Cas9. ACS Synth. Biol. 6:1351–1358." />
<meta name="citation_reference" content="Lieberman, T.D., Flett, K.B., Yelin, I., Martin, T.R., McAdam, A.J., Priebe, G.P., Kishony, R. (2014) Genetic variation of a bacterial pathogen within individuals with cystic fibrosis provides a record of selective pressures. Nature Gen 46:82–87." />
<meta name="citation_reference" content="McKenna, R., Xia, D., Willingmann, R., Ilag, L.L., Krishnaswamy, S., Rossmann, M.G., Olson, N.H., Baker, T.S., Incardona, N.L. (1992) Atomic structure of single-stranded DNA bacteriophage ΦX174 and its functional implications. Nature 355, 137–143." />
<meta name="citation_reference" content="Medina-Cucurella, A., Steiner, P.J., Faber, M.S., Beltrán, J., Borelli, A., Kirby, M.B., Cutler, S.R., Whitehead, T.A. (2019) User-defined single pot mutagenesis using unpurified oligo pools. Prot. Eng. Des. Sel. 32(1)41–45." />
<meta name="citation_reference" content="Neher, R.A. (2013) Genetic draft, selective interference, and population genetics of rapid adaptation. Annu. Rev. Ecol. Evol. Syst. 44:195–215." />
<meta name="citation_reference" content="Pires, D.P., Cleto, S., Sillankorva, S., Azeredo, J., Lu, T.K. (2016) Genetically engineered phages: a review of advances over the last decade. Microbiol. and Mol. Biol. Rev. 80:523–543." />
<meta name="citation_reference" content="Rokyta, D.R., Joyce, P., Caudle, S.B., and Wichman, H.A. (2005) An empirical test of the mutational landscape model of adaptation using a single-stranded DNA virus. Nat Genet. 37, 441–444." />
<meta name="citation_reference" content="Rokyta, D.R., Burch, C.L., Caudle, S.B., Wichman, H.A. (2006) Horizontal gene transfer and the evolution of microvirid coliphage genomes. J. of Bacter. 188, 1134–1142." />
<meta name="citation_reference" content="Rogers, S.O., Bendich, A.J. (1988) Extraction of DNA from plant tissues. Plant Mole Biol. Manual A6:1–10." />
<meta name="citation_reference" content="Sackman, A.M., and Rokyta, D.R. (2018) Additive Phenotypes Underlie Epistasis of Fitness Effects. Genetics 208, 339–348." />
<meta name="citation_reference" content="Sanger, F., Coulson, A.R., Friedmann, T., Air, G.M., Barrell, B.G., Brown, N.L., Fiddes, J.C., Hutchinson, C.A., Slocombe, P.M., Smith, M. (1978) The nucleotide sequence of bacteriophage phiX174. J. Mol. Biol. 125:225–246." />
<meta name="citation_reference" content="Schilling, T., Dietrich, S., Hoppert, M., Hertel, R. (2018) A CRISPR-Cas9-based toolkit for fast and precise in vivo genetic engineering of Bacillus subtilis phages. Viruses 10:241." />
<meta name="citation_reference" content="Smith, H.O., Hutchinson III, C.A., Pfannkoch, C., Venter, J.C. (2003) Generating a synthetic genome by whole genome assembly: ΦX174 bacteriophage from synthetic oligonucleotides. Proc. Natl. Aca. Sci. USA 100, 15440–15445." />
<meta name="citation_reference" content="Stockdale, S.R., Collins, B., Silvia, S., Douillard, F.P., Mahony, J., Cambillau, C., van Sinderen, D. (2015) Structure and assembly of TP901-1 Virion unveiled by mutagenesis. PLoS One 10, e0131676." />
<meta name="citation_reference" content="Uchiyama, A., Heiman, P., Fane, B.A. (2009) N-terminal deletions of the ΦX174 external scaffolding protein affect the timing and fidelity of assembly. Virology 386, 303–309." />
<meta name="citation_reference" content="van Dolleweerd, C.J., Kessans, S.A., Van de Bittner, K.C., Bustamante, L.Y., Bundela, R., Scott, B., Nicholson, M.J., Parker, E.J. (2018) MIDAS: a modular DNA assembly system for synthetic biology. ACS Synth Biol 7:1018–1029." />
<meta name="citation_reference" content="Wichman, H.A., and Brown, C.J. (2010) Experimental evolution of viruses: Microviridae as a model system. Phil. Trans. of the Royal Society B. 365, 2495–2501." />
<meta name="citation_reference" content="Yehl, K., Lemire, S., Yang, A.C., Ando, H., Mimee, M., Torres, M.D.T., de la Fuente-Nunez, C., and Lu, T.K. (2019). Engineering Phage Host-Range and Suppressing Bacterial Resistance through Phage Tail Fiber Mutagenesis. Cell 179, 459–469.e9. Genetics 170, 19-31." />
<meta name="twitter:title" content="Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Here we present a novel protocol for the construction of saturation single-site—and massive multi-site—mutant libraries of a bacteriophage. We segmented the ΦX174 genome into 14 non-toxic and non-replicative fragments compatible with golden gate assembly. We next used nicking mutagenesis with oligonucleotides prepared from unamplified oligo pools with individual segments as templates to prepare near-comprehensive single-site mutagenesis libraries of genes encoding the F capsid protein (421 amino acids scanned) and G spike protein (172 amino acids scanned). Libraries possessed greater than 99% of all 11,860 programmed mutations. Golden Gate cloning was then used to assemble the complete ΦX174 mutant genome and generate libraries of infective viruses. This protocol will enable reverse genetics experiments for studying viral evolution and, with some modifications, can be applied for engineering of therapeutically relevant bacteriophages with larger genomes." />
<meta name="og-title" property="og:title" content="Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/798546v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Here we present a novel protocol for the construction of saturation single-site—and massive multi-site—mutant libraries of a bacteriophage. We segmented the ΦX174 genome into 14 non-toxic and non-replicative fragments compatible with golden gate assembly. We next used nicking mutagenesis with oligonucleotides prepared from unamplified oligo pools with individual segments as templates to prepare near-comprehensive single-site mutagenesis libraries of genes encoding the F capsid protein (421 amino acids scanned) and G spike protein (172 amino acids scanned). Libraries possessed greater than 99% of all 11,860 programmed mutations. Golden Gate cloning was then used to assemble the complete ΦX174 mutant genome and generate libraries of infective viruses. This protocol will enable reverse genetics experiments for studying viral evolution and, with some modifications, can be applied for engineering of therapeutically relevant bacteriophages with larger genomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-10-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/798546v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/798546v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/798546v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/798546v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/942725" />
    <title>Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"BEpg_KFHlYNDOt1d3lyjUl-GjMj5pbJ6s0eVqI20xw4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"942725","apath":"\/biorxiv\/early\/2019\/10\/09\/798546.atom","pisa":"biorxiv;798546v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;798546v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/798546v1","title":"Saturation mutagenesis genome engineering of infective \u03a6X174 bacteriophage via unamplified oligo pools and golden gate assembly","identifier":"node\/942725"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aZ4s6yt3DcOpwjw40OOOuJpsncjLSdHknqm53CEOH1VE3u\\\\\\\/ao2PLuHgB+vn6OLSIJhMs6yXiMqy4xF1YVZPQyAGOTzWkguym3NWbJMYJ9ARhOp8yc63aiMeWm5ry8\\\\\\\/PWuIVetlhGkFDCnHQudbNYiN66XcShlMsu3IvJFbwKTlVYghfPRqzDq7n5zkSYaR8ZyY2W94DjR1KZ9UAgR01GxcI0Ruyd1cjz\\\\\\\/zHEN7lJI20xcNYueufowGgVygSry6gj6+UVz8g27cTJZHIVtVr1jWo\\\\\\\/epQjfcFQjlpbxfZH9zwP8HXAA1lqW6ra7LIT99mboqrRSCVcYfnDjLvXTHmp+OHoB6YrQQSr8jQJksSVZu5kWVI5mr2Lv\\\\\\\/TSU87p7LwcwuZNM73LRlEAS3NwiFY\\\\\\\/\\\\\\\/Xkg7s0CDrVW23GMj824yuXIeXHtmV1KI1OArrohIDYQnaBtoLmPv3Ncqk0cFIaK4ePYb0HOhI4n+k\\\\\\\/vIZozklAO3EOntEf3T+3MgGbn4uUd5lPtBItedq9mFe9H5YQB47tk5aIVO6vFqHAcJNqPT11vz4fNuFlFx9VXSeR61s\\\\\\\/relVbKUxMJ1ShSF30xNPhuupB1Z+ZdJEwA7BDDfGDEtN3Y1qS2l5a9nPlDricU66ZPRwayiGO\\\\\\\/IE03\\\\\\\/CGk4R5SkjBB3XmDsXLfyyN3T7n8S5kAxaNKh4BzfpSb\\\\\\\/0VyZ4JyR3Kjv0R6Wmma52JHB3f7XGh4L5BXP8YXww1BxQ+8EfSk62Ys08NDdaHeMuxQoVcshF0GRQIA3DgtvrORMkDOsGKiXBfBRKvAgurCsGS0112RTbc8h+yjwnm2J2Yola2onukoR\\\\\\\/k\\\\\\\/p6EfVBQzEw+dGH7pPQaApfxzvaXAJhDur8AGExbd7bCjnmJNrC9JSref0GFNR8vONEf3PBawpEUukiYlQJKBy9OztQEJqTgpQu9C5I5vpCo2lXNo2DjW2mI8C9nT+XzfG+yYjwVRrxALwKTZfDzPt3RfVcrExHs0rV7jzTXItyH8KCTf4xCT6SxG+lpdh+IiWS85Qz4jgKu\\\\\\\/fxgEF9XWTTfaSQlsrg6AQ\\\\\\\/FrABSWImEuy+UnBCfedqjYwxuvFsiCTlmQNPHk6Rf2vxLQexnb9SNGRhF8\\\\\\\/RDWuyFobK8GE0Efnpcr6lXy5MDndVoV0cs0Kw61FevM87Qusq2diyhtmyMe1yeWGvQvyvKRB+dzWXSwZaQUknkIjqPcO0ECSv4Hs6CiewTD2eIKnnoQKFysKYHFO+UUClozRJMlissua0xMlwtFnpR9q9EQE9ZSi3m\\\\\\\/BJH0xepyqz7hMNH0FvNwgqlvQpdMGhG9dw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022PFjILxkOecADYf0MEu0FeA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022379123dc2583447247fb1417b413f0d2487075246d454054f26bf1ee662646ba\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/798546v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-942725 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/798546v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JLung7xySgFUF6bD3tU9pYFKbfQbLh4MFP6t4Id7CW0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="942725" id="node942725" data-pisa="biorxiv;798546v1" data-pisa-master="biorxiv;798546" data-apath="/biorxiv/early/2019/10/09/798546.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew S.</span> <span class="nlm-surname">Faber</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2730-0287" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">James T.</span> <span class="nlm-surname">Van Leuven</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Martina M.</span> <span class="nlm-surname">Ederer</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yesol</span> <span class="nlm-surname">Sapozhnikov</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Zoë L.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Holly A.</span> <span class="nlm-surname">Wichman</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3177-1361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Timothy A.</span> <span class="nlm-surname">Whitehead</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-7920-4994" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Craig R.</span> <span class="nlm-surname">Miller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/798546 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node942725" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew S. Faber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Dept. Biochemistry and Molecular Biology, Michigan State University</span>, East Lansing MI, 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BS.%2BFaber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Faber%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BS.%2BFaber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James T. Van Leuven </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Modeling Collaboration and Innovation, University of Idaho</span>, Moscow ID, 83844</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Dept. of Biological Sciences, University of Idaho</span>, Moscow ID, 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BT.%2BVan%2BLeuven%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Leuven%20JT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BT.%2BVan%2BLeuven%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2730-0287" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for James T. Van Leuven</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jvanleuven@uidaho.edu" class="" data-icon-position="" data-hide-link-title="0">jvanleuven@uidaho.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martina M. Ederer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Dept. of Biological Sciences, University of Idaho</span>, Moscow ID, 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martina%2BM.%2BEderer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ederer%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartina%2BM.%2BEderer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yesol Sapozhnikov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Dept. of Biological Sciences, University of Idaho</span>, Moscow ID, 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yesol%2BSapozhnikov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sapozhnikov%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYesol%2BSapozhnikov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Zoë L. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Dept. of Biological Sciences, University of Idaho</span>, Moscow ID, 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zo%C3%AB%2BL.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20ZL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZo%25C3%25AB%2BL.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Holly A. Wichman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Modeling Collaboration and Innovation, University of Idaho</span>, Moscow ID, 83844</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Dept. of Biological Sciences, University of Idaho</span>, Moscow ID, 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Holly%2BA.%2BWichman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wichman%20HA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHolly%2BA.%2BWichman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Timothy A. Whitehead </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Dept. of Chemical &#38; Biological Engineering; University of Colorado</span>, Boulder, CO 80303</div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Dept. of Chemical Engineering &#38; Materials Science; Michigan State University</span>, East Lansing MI, 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BA.%2BWhitehead%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitehead%20TA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BA.%2BWhitehead%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3177-1361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy A. Whitehead</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Craig R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute for Modeling Collaboration and Innovation, University of Idaho</span>, Moscow ID, 83844</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Dept. of Biological Sciences, University of Idaho</span>, Moscow ID, 83844</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Craig%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACraig%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7920-4994" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Craig R. Miller</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/798546v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:942725" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:942725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/798546v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:942725" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:942725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/798546v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:942725" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:942725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/798546v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:942725" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:942725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/798546v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:942725" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:942725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/798546v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:942725" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:942725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Here we present a novel protocol for the construction of saturation single-site—and massive multi-site—mutant libraries of a bacteriophage. We segmented the ΦX174 genome into 14 non-toxic and non-replicative fragments compatible with golden gate assembly. We next used nicking mutagenesis with oligonucleotides prepared from unamplified oligo pools with individual segments as templates to prepare near-comprehensive single-site mutagenesis libraries of genes encoding the F capsid protein (421 amino acids scanned) and G spike protein (172 amino acids scanned). Libraries possessed greater than 99% of all 11,860 programmed mutations. Golden Gate cloning was then used to assemble the complete ΦX174 mutant genome and generate libraries of infective viruses. This protocol will enable reverse genetics experiments for studying viral evolution and, with some modifications, can be applied for engineering of therapeutically relevant bacteriophages with larger genomes.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">Predicting the tempo and trajectory of evolutionary change in the complex environments encountered by viruses and bacteria remains a challenge (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Holmes 2013</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Neher 2013</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lieberman et al. 2014</a>). Understanding such changes are important for fundamental evolutionary studies as well as biotechnology applications like phage therapy for multi-drug resistant bacteria (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dedrick et al. 2019</a>). Two obstacles preventing better predictions are the oversimplification of the environment in experimental evolution studies compared to wild conditions and the vast number of experimentally unexplored mutational combinations that can occur, even in short adaptive walks and small genomes. Advances in DNA sequencing and synthesis have opened new ways to study microbial evolution and overcome some of these obstacles (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrick et al. 2013</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fowler et al. 2014</a>). Unfortunately, suitable methods do not exist for generating comprehensive bacteriophage mutant libraries. This technology gap hinders those seeking to engineer phage for biotechnological applications and for those seeking a deeper understanding of how viruses evolve.</p><p id="p-6">Methods for genetically engineering phages have recently been reviewed (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pires et al. 2016</a>) and commonly involve homologous recombination and recombineering. While such approaches can be improved by clever incorporation of CRISPR-Cas systems (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kiro et al. 2014</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lemay et al. 2017</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schilling et al. 2018</a>), the overall modest efficiencies largely limits mutagenesis to the generation and single-site mutation of chimeric genomes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Doore and Fane 2015</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Doore et al. 2017</a>), gene deletions (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Uchiyama et al. 2009</a>), and limited multi-site mutagenesis (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Stockdale et al. 2015</a>). A recent approach combines mutagenesis by PCR with degenerate (NNK) primers with in-cell recombination to create variation at specific residues in the tail fiber of T3 phage (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yehl et al. 2019</a>). By contrast, human viruses often have reverse genetics systems in place where comprehensive mutant libraries can be prepared for single genes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lee et al. 2018</a>) or regions within a gene (Doud et al. 2018, Wu and Qi 2019) using replicative plasmids that encode whole viruses or viral components. Similarly, deep mutational scanning can be performed on plasmid-encoded phage proteins like the MS2 capsid protein (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hartman et al. 2018</a>). As described by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hartman et al. 2018</a>, such experiments can identify mutations that confer useful characteristics, such as MS2 capsid stability mutations that could improve the targeted delivery of therapeutics to particular cellular compartments.</p><p id="p-7">Advances in the technologies for generating mutant libraries, synthesizing DNA, and for the assembly of large DNA fragments (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">van Dolleweerd et al. 2018</a>) allow for the construction and assembly of user-defined mutagenesis of long nucleic acids. Nicking mutagenesis (NM) can be used to construct comprehensive single-site or other user-defined mutant libraries (Wrenbeck et al. 2016) using dsDNA as a template. In NM, oligos encode the desired mutations by mismatch with the parental template. On-chip ink-jet printed oligo pools have been integrated into NM (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Medina-Cucurella et al. 2019</a>), so one can now construct heterogeneous libraries of oligos that contain tens to hundreds of thousands of high fidelity, unique sequences (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kosuri and Church 2014</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Medina-Cucurella et al. 2019</a>) with a low per base pair cost.</p><p id="p-8">We therefore suspected that large user-defined libraries of phage could be generated by combining the advantages of replicative plasmids with advances in DNA synthesis. The bacteriophage ΦX174 is an excellent system to test this approach. High resolution X-ray crystallography structures of the capsid and spike proteins (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McKenna et al. 1992</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dokland et al. 1997</a>) are available for ΦX174, making it amenable to compare empirical mutagenesis results and molecular modeling predictions, it has a very small genome (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sanger et al. 1978</a>) of 5,386 nucleotides that is readily sequenced, and it is easily grown on a variety of lab <em>E. coli</em> strains. These attractive features of the ΦX174 experimental system have led to proof-of-principle demonstrations from other groups for synthetic genome assembly from oligonucleotides (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Smith et al. 2003</a>) and complete refactoring of the phage (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jaschke et al. 2012</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jaschke et al. 2018</a>).</p><p id="p-9">ΦX174 has also been extensively studied (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aoyama et al. 1981</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hafenstein and Fane 2002</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernal et al. 2004</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rokyta et al. 2006</a>), especially as a model system for experimental evolution (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bull et al. 1997</a>, Crill, Wichman, and Bull 2000, Wichman, Millstein, and Bull 2005) since populations grow rapidly to very large size, mutation rate is high, and the fitness differences between even single mutations can be very large. One of the focal areas of work on ΦX174 and related microvirid phage has been testing models of adaptive evolution and epistasis with empirical data (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rokyta et al. 2005</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Caudle, Miller, and Rokyta 2014</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sackman and Rokyta 2018</a>). The inferential power of these studies has been limited by the modest sizes of the mutational libraries.</p><p id="p-10">Here we have combined Golden Gate assembly, nicking mutagenesis, and oligo pool technology to construct near comprehensive single-site and expansive multisite mutant libraries for the genes encoding the capsid F protein and spike G protein of the bacteriophage ΦX174. To our knowledge this is the first time near comprehensive single site mutant libraries of full capsid and spike proteins have been generated for an infective bacteriophage.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-11">Our strategy for generating large user-defined mutagenesis libraries of bacteriophage ΦX174 is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The circular 5386 nucleotide genome compactly encodes 11 genes where the F and G genes encode the capsid and spike proteins, respectively (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1A</a>). The genome was segmented onto 14 separate plasmids, including 3 plasmids for the F gene and 2 for the G gene (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>). Fragment boundaries were first engineered to divide large genes, separate regulatory regions from protein coding genes, and result in fragments of similar size. Initial attempts to clone some fragments resulted in low plasmid yield, presumably due to gene toxicity. These genes were further divided and tested empirically, resulting in the final set of 14 fragments. User-defined comprehensive mutations on F and G were encoded using nicking mutagenesis with a single unamplified oligo pool containing all 11,860 mutagenic oligonucleotides (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a>). ΦX174 mutant genomes were reconstituted from individual plasmids using Golden Gate cloning and transformation into a host <em>E. coli</em> strain (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1D</a>). Harvested phages contained all possible non-synonymous mutations in the F and G genes depending on the mutagenized segments used for Golden Gate cloning (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1E</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="&#x3A6;X174 mutant library assembly. A. Cartoon structure of the &#x3A6;X174 virion. F and G proteins are organized into pentamers and are the only exposed proteins in the mature virion. B. A linear schematic of the circular &#x3A6;X174 genome showing the size and location of the synthetic genome fragments. The location of mutated residues in genes F and G are indicated. C. An oligo pool containing all mutagenic oligos for genes F and G was generated and used in Nicking Scanning Mutagenesis for the generation of saturation mutant libraries for genes F and G. D. Golden Gate cloning was used to assemble the mutant genomes, which were transformed into XL1-Blue competent cells (Agilent) and plated on susceptible E. coli C cells. The resulting plaques were sequenced. E. Linear schematics depicting some of the possible mutant libraries that can be generated using the workflow and existing libraries." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;&#x3A6;X174 mutant library assembly.&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; Cartoon structure of the &#x3A6;X174 virion. F and G proteins are organized into pentamers and are the only exposed proteins in the mature virion. &lt;strong&gt;B.&lt;/strong&gt; A linear schematic of the circular &#x3A6;X174 genome showing the size and location of the synthetic genome fragments. The location of mutated residues in genes F and G are indicated. &lt;strong&gt;C.&lt;/strong&gt; An oligo pool containing all mutagenic oligos for genes F and G was generated and used in Nicking Scanning Mutagenesis for the generation of saturation mutant libraries for genes F and G. &lt;strong&gt;D.&lt;/strong&gt; Golden Gate cloning was used to assemble the mutant genomes, which were transformed into XL1-Blue competent cells (Agilent) and plated on susceptible &lt;em&gt;E. coli&lt;/em&gt; C cells. The resulting plaques were sequenced. &lt;strong&gt;E.&lt;/strong&gt; Linear schematics depicting some of the possible mutant libraries that can be generated using the workflow and existing libraries.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F1.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F1.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">ΦX174 mutant library assembly.</span><p id="p-12" class="first-child"><strong>A.</strong> Cartoon structure of the ΦX174 virion. F and G proteins are organized into pentamers and are the only exposed proteins in the mature virion. <strong>B.</strong> A linear schematic of the circular ΦX174 genome showing the size and location of the synthetic genome fragments. The location of mutated residues in genes F and G are indicated. <strong>C.</strong> An oligo pool containing all mutagenic oligos for genes F and G was generated and used in Nicking Scanning Mutagenesis for the generation of saturation mutant libraries for genes F and G. <strong>D.</strong> Golden Gate cloning was used to assemble the mutant genomes, which were transformed into XL1-Blue competent cells (Agilent) and plated on susceptible <em>E. coli</em> C cells. The resulting plaques were sequenced. <strong>E.</strong> Linear schematics depicting some of the possible mutant libraries that can be generated using the workflow and existing libraries.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">We sought a reverse genetics system for ΦX174 wherein the virus chromosome was segmented and encoded on individual vector plasmids. Our method of construction closely followed that for the assembly of human coronavirus NL63 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Donaldson et al. 2008</a>), where each plasmid contained unique BsmB1 type IIS restriction endonuclease sites flanking unique five nucleotide overlaps of wild-type (WT) ΦX174. This architecture allows for faithful and easy assembly of the complete genome via Golden Gate cloning. The ΦX174 genome was separated into 14 separate nontoxic fragments by PCR and cloned into vector plasmids where genes were separated from their promoters and larger genes were segmented (full sequences for all plasmid inserts are given in Supplemental Note S1). The segments ranged from 129 to 634 base pairs in length, with necessary additional truncations of genes D and A. ΦX174 phage could be reconstituted by digesting pooled plasmids with BsmB1, ligating inserts overnight, and transforming into electrocompetent <em>E. coli</em> C cells.</p><p id="p-14">The F and G genes were targeted for saturation mutagenesis. We introduced mutations by nicking mutagenesis, which requires the presence of a BbvCI nicking site on the dsDNA plasmid. The F gene plasmid F3 encoding residues 246-427 of the F gene product contained a unique BbvCI sequence, while the remaining four plasmids (F1, F2, G1, G2) required introduction of BbvCI nicking sites in the vector backbone. The presence in each plasmid of the unique BbvCI nicking site was verified with the successful generation of circular ssDNA from the modified plasmids by BbvCI.Nt and exonuclease digestion (<a id="xref-fig-17-1" class="xref-fig" href="#F17">Supplemental Figure S1</a>).</p><p id="p-15">In nicking mutagenesis, desired mutations are introduced by libraries of mutagenic oligonucleotides, which can be sourced from unamplified oligo pools (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Medina-Cucurella et al. 2019</a>). We synthesized a custom pool of 11,860 oligos that encoded nearly every missense and nonsense mutation in the F and G genes. Each missense or nonsense mutation was encoded by a single oligonucleotide. Application of nicking mutagenesis using this oligo pool for different plasmids (F1, F2, F3, G1, G2) resulted in at least 14-fold excess transformants required for 99.9% theoretical coverage of the desired library (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Supplemental Table S3</a>). The diversity of the mutant plasmid libraries was validated using deep sequencing on an Illumina MiSeq platform. All plasmid libraries have &gt;99.7% coverage of all possible single mutations (n = 11,821/11,860, minimum counts for a mutation &gt;5, full library statistics are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Supplemental Table S4</a>). Heatmaps showing mutation-specific frequencies for all segments are shown in <a id="xref-fig-18-1" class="xref-fig" href="#F18">Supplementary Figures S2</a>–<a id="xref-fig-25-1" class="xref-fig" href="#F25">S9</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945480/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945480/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945480" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Mutant library NGS statistics.</span><p id="p-16" class="first-child">Summary table of the of the plasmid libraries prior to viral genome assembly.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">Mutant dsDNA libraries were transformed into non-susceptible hosts (Agilent XL1-Blue), then plated on susceptible hosts (<em>E. coli</em> C) after a 30-minute recovery in SOC. Viral titers increase rapidly after 30 minutes, suggesting that the XL1-Blue cells do not burst until after this time point (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Supplemental Table S5</a>). Accordingly, we reasoned that the number of plaques at 30 minutes represents the number of XL1-Blue cells that were transformed with viable genomes. The number of transformants varied considerably between constructs (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). 19,200 plaques were recovered from the co-transformed G1/G2 libraries, where only 3,480 total variants were expected in the pool. On the contrary, 460 plaques were observed for the F2 library, when 2,280 total variants were in the starting plasmid pool. The underrepresentation of the F2 library at this scale could be remedied by transforming more competent cells. We expect that differences between fragments in the number of recovered viruses is due to in part to the tolerability of fragments to mutation, as inviable mutants would not form plaques. Estimates from deep mutational scanning of model proteins show that approximately 18-38% of all mutations are nonviable (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Firnberg et al. 2014</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Faber et al. 2019</a>), and this percentage may be larger for structural proteins like F and G.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945486/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945486/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945486" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">Characterization of viable virus genotypes in mutant phage library.</span><p id="p-18" class="first-child">Summary table of number of possible viruses, number of viable viruses recovered from one transformation, and the number of mutant and wildtype genotypes recovered. No mutant genotypes were observed more than once.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">To test the validity of the mutagenesis and phage reconstitution method, we picked about 30 plaques generated from each library to Sanger sequence. Of these, 40-50% were wildtype and the remaining contained exactly one mutation (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). No variants were observed more than once, supporting the deep sequencing results showing that the libraries consist of thousands of variants. Interestingly, none of the exact substitutions in these sequences have been previously observed (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Supplementary Table S6</a>), although mutations at the same residues (G106, F29, F45, F84, F185, F204, F205, and F393) have been seen in previous experiments (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wichman and Brown 2010</a>). In all cases the percentage of WT plaques is slightly higher than in the starting plasmid pools (p-values F1: 0.001, F2: 0.088, F3: 0.015, G: 0.001 1-tailed binomial distribution), consistent with purifying selection against mutations (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). In contrast, for the control where all plasmids in the Golden Gate reaction were WT, 0% (0/30) plaques contained mutations in the encoded regions.</p></div><div class="section" id="sec-3"><h2 class="">Discussion</h2><p id="p-20">Here we present a novel method for generating comprehensive single-site and multi-site saturation mutant libraries of the spike and capsid proteins of the bacteriophage ΦX174. This method uses unamplified oligo pools, nicking scanning mutagenesis, and Golden Gate cloning of the fragmented genome.</p><p id="p-21">The fragment-based system provides an ideal way to generate libraries for the study of epistasis and multi-mutational step regions of the neighborhood around wild type. When a gene is in multiple fragments, the number of mutational variants we expect from the assembled mutant fragments is the product of their individual variant number. Hence, since G1 contains 2640 variants and G2 contains 800 variants, they can be combined to generate 2.1 million double mutation variants. Further, nothing requires the fragments to be from the same gene. For example, since the F and G proteins bind to each other to form the capsid of phiX174 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>), libraries combining mutations in F with mutations in G will capture mutations that can readily interact in the protein and have the potential to generate interesting instances of epistasis (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2</a>). The central challenge then becomes how to survey the vast size of these combinatorial plasmid libraries. Right now, our transformational efficiencies vary, but even at the upper end of hundreds of thousands of transformants per reaction, only a fraction of the total diversity in the library can be surveyed.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structural view of potential epistatic interactions between F and G libraries. A. Pentameric complex between F and G shown in surface view. For both F and G, one monomer is shown as a ribbon color-coded according to the saturation mutagenesis library prepared in this work. B. Close-up of interaction surface between the F3 sub-li-brary of F protein (green ribbons) and G2 sub-library of G protein (purple spheres). C. Distance mapping between alpha carbons in F and G crystal structure. Residues within 10 angstrom are colors red, 10-20 are black, and &gt;20 are blue. Euclidian distances calculated from F-G crystal structure. Heatmap shows that many opportunities exist for close proximity (interactions) both within and between fragments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Structural view of potential epistatic interactions between F and G libraries.&lt;/span&gt; &lt;strong&gt;A.&lt;/strong&gt; Pentameric complex between F and G shown in surface view. For both F and G, one monomer is shown as a ribbon color-coded according to the saturation mutagenesis library prepared in this work. &lt;strong&gt;B.&lt;/strong&gt; Close-up of interaction surface between the F3 sub-li-brary of F protein (green ribbons) and G2 sub-library of G protein (purple spheres). &lt;strong&gt;C.&lt;/strong&gt; Distance mapping between alpha carbons in F and G crystal structure. Residues within 10 angstrom are colors red, 10-20 are black, and &gt;20 are blue. Euclidian distances calculated from F-G crystal structure. Heatmap shows that many opportunities exist for close proximity (interactions) both within and between fragments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F2.medium.gif" width="167" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F2.medium.gif" width="167" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Structural view of potential epistatic interactions between F and G libraries.</span><p id="p-22" class="first-child"><strong>A.</strong> Pentameric complex between F and G shown in surface view. For both F and G, one monomer is shown as a ribbon color-coded according to the saturation mutagenesis library prepared in this work. <strong>B.</strong> Close-up of interaction surface between the F3 sub-li-brary of F protein (green ribbons) and G2 sub-library of G protein (purple spheres). <strong>C.</strong> Distance mapping between alpha carbons in F and G crystal structure. Residues within 10 angstrom are colors red, 10-20 are black, and &gt;20 are blue. Euclidian distances calculated from F-G crystal structure. Heatmap shows that many opportunities exist for close proximity (interactions) both within and between fragments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">We are fortunate that our model virus has a small genome, under 6,000 nts, which enabled us to assemble the 14 non-toxic fragments with sufficient transformants for near-complete coverage of the user-defined mutations. This method can be applied with little modification to other phages with small genomes like the microviridae. However, most biotech-relevant phages have larger genomes, and extension of this method to other viruses would encounter the following technical challenges. First, increasing genome sizes decrease transformational efficiencies, and larger genomes have increased susceptibility to DNA shearing (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rogers and Bendich 1988</a>). Both size-dependent effects can disrupt the integrity and coverage of the phage libraries (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Edwards and Rohwer 2005</a>). Second, our method includes restrictions on the DNA sequences that can be used. Golden Gate cloning requires a genome without unique Type IIS restriction sequences, while nicking mutagenesis requires that there be only one orientation of the BbvCI nicking site within the template DNA fragment. Third, ligating 14 fragments is close to the upper limit of Golden Gate cloning. Additionally, increasing the size of individual fragments is difficult as the presented method depends on replicating plasmids in <em>E. coli</em>, and larger fragments are more likely to be toxic in vivo.</p><p id="p-24">Based on the above considerations, adapting this method to larger viruses will require modifications for the genome assembly steps. There are a variety of methods available for attempting to improve genome library assembly and amplification. For improving genome assembly, we speculate that a combination of hierarchical assembly (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lee et al. 2015</a>) with other yeast-based assembly methods (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ando et al. 2015</a>) will allow for large viral genomes to be efficiently assembled. <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Yehl et al. 2019</a> build large libraries at targeted epitope regions by combining methods; plasmid-based mutagenesis and cloning followed by recombination with wild-type phage during infection. Finally, we speculate that the Tx.Tl cell free expression system (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Garamella et al. 2016</a>) will be able to produce the viral libraries with less bias than in vivo amplification in <em>E. coli</em>. In summary, we have presented a method for efficient deep mutational scanning of the bacteriophage ΦX174. We anticipate that this method will find utility in fundamental molecular evolution studies as well as translate to potential medicinal applications.</p></div><div class="section" id="sec-4"><h2 class="">Methods</h2><div id="sec-5" class="subsection"><h3>Reagents</h3><p id="p-25">All purchased enzymes and DNA purification kits were from New England Biolabs, antibiotics were purchased from GoldBio, other chemicals were purchased from Sigma-Aldrich. Individual primers were purchased from Integrated DNA Technologies.</p></div><div id="sec-6" class="subsection"><h3>Segmentation of the ΦX174 genome</h3><p id="p-26">A phage assembly platform for ΦX174 was devised following (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Donaldson et al. 2008</a>). The ΦX174 chromosome was divided into 14 genomic fragments designed to avoid host cell toxicity by separating genes from their promoters and breaking large genes into multiple segments (Supplemental Note S1). Each segment is flanked by unique five nucleotide overlaps of WT ΦX174 sequence so that they can be amplified from the ancestral ΦX174 using PCR primers designed to incorporate terminal BsmB1 restriction sites. Amplicons were cloned into pCR2.1 using the Invitrogen TOPO TA cloning system (Life Technologies, Grand Island, NY) and verified by Sanger sequencing.</p></div><div id="sec-7" class="subsection"><h3>Introduction of nicking site</h3><p id="p-27">The gene fragments – F1, F2, G1, G2 – in the pCR2.1-TOPO plasmid (Supplemental Notes S1 and S2) had the BbvCI nicking site introduced via overhang PCR, type I restriction enzyme cutting, and ligation. First, PCR was performed with overhang primers (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Supplemental Table S1</a>) to introduce the BbvCI site. Standard Phusion Polymerase HF reaction conditions were used with 4 ng of template DNA, cycling conditions as follows: 98°C - 1 min, 25× cycles of: 98°C – 10 seconds, 67°C 15 seconds, 72°C 2.5 minutes, followed by 72°C for 10 minutes. PCR products were run on a 1% agarose gel stained with SYBR™ safe stain (Invitrogen) and the DNA bands at ~4600 bp were extracted using a Monarch<sup>®</sup> DNA Gel Extraction Kit. Next, 1 μg of the PCR product was digested with KpnI (20 U) in NEB Buffer 1.1 at 37°C for two hours. Digested DNA was cleaned and concentrated with a Monarch<sup>®</sup> PCR and DNA Clean and Concentrate Kit and eluted into 20 μL nuclease free H2O. One microliter of the purified and digested DNA was ligated using T4 DNA ligase at ~25°C for 1 hour in standard conditions in a 20 μL reaction. Five microliters of the ligation reaction were transformed into chemically competent XL1-Blue <em>E. coli</em> via standard protocols. Cells were plated on LB agar containing 100 μg/mL carbenicillin and 50 μg/ mL kanamycin and grown at 37°C for ~16 hours. Cells were picked from transformation plates and grown in 50 mL TB with 100 μg/mL carbenicillin and 50 μg/mL kanamycin for ~12 hours, cells were pelleted, and DNA purified using compact midi-preps.</p></div><div id="sec-8" class="subsection"><h3>Construction of single site mutant libraries</h3><p id="p-28">Comprehensive mutant libraries were generated using nicking mutagenesis (NM) as in Wrenbeck et al. 2016 with modifications for using oligo pool mutagenic primers as noted in Medina et al. 2019. A single oligo pool encoding for all possible single missense and nonsense substitutions in F and G was designed using the custom python scripts from Medina et al. 2019 and custom synthesized by Agilent (full sequences of all oligos are given in the associated file “VMA_supplemental_data.csv”). Oligo pools were designed with 20-24 bases of gene overlap flanking the mutated codon. Codons were chosen based on <em>E. coli</em> codon usage frequency, where the highest frequency codon that encoded the mismatch mutation was selected. We printed 14,853 unique oligos, where 14,853 = 12,453 ΦX174 primers– 2,400 other primers. These oligos encode WT (593 primers) and all 20 possible single point mutations (including stop) (11,860 primers). This oligo pool was used directly in NM without further amplification using 2 μg of the relevant golden gate plasmid as a template. Libraries were electroporated into high efficiency electrocompetent XL1-Blue <em>E. coli</em> (Agilent cat #: 200228) using 1 mm electroporation cuvettes at 1200V using a Eppendorf Eporator. Cells were plated on large bioassay plates (245mm × 245mm × 25mm, Sigma-Aldrich) containing LB agar + 100 μg/mL carbenicillin and 50 μg/mL kanamycin and grown at 37°C for ~16 hours. Plates were scraped in 10 mL plain LB, broken into ~1.2 mL aliquots, pelleted, and stored −80°C. DNA was purified from 1× aliquot using a Monarch<sup>®</sup> Plasmid Mini-Prep Kit.</p></div><div id="sec-9" class="subsection"><h3>Illumina sequencing prep and analysis</h3><p id="p-29">Purified library DNA was prepared for deep sequencing as in <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kowalsky et al. (2015)</a> using the primers listed in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Supplemental Table S2</a> and gene tiling as specified in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. DNA was Illumina sequenced on a Mi-Seq platform with 250 BP paired end reads. The University of Colorado BioFrontiers Sequencing Core performed the Illumina sequencing. Data was processed using PACT (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Klesmith and Hackel 2018</a>) to determine the library coverage with the following changes to the default options in the configuration file: fast_filter_translate: qaverage = 20, qlimit = 0; enrichment: ref_count_threshold = 5, sel_ count_threshold = 0, strict_count_threshold = True. Library coverage statistics are shown in <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Supplemental Table S3</a> and <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">S4</a>.</p></div><div id="sec-10" class="subsection"><h3>Assembly of mutant genomes</h3><p id="p-30">We pooled plasmid DNA containing all 14 of the phage DNA fragments (500 ng each) and digested them with BsmB1 (Fermentas Fast Digest, Life Technologies, Grand Island, NY) for 30 minutes to 1 hour at 37°C. The digested plasmids were subjected to agarose gel electrophoresis for 10 to 15 minutes using a 1.2% agarose gel to separate the vector from the inserts. The inserts were excised from the gel, purified using the GeneJET gel extraction kit (Fermentas), ligated overnight at 14°C with T4 DNA ligase (Promega Corporation, Madison, WI), and transformed by electroporation into 50 μl of XL1-Blue electocompetent cells. The transformation mix was resuspended with 950 μl of SOC and either plated immediately or incubated for up to 120 minutes (<a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Supplemental Table S5</a>) at 37°C in the presence of DNase1 (<a id="xref-fig-26-1" class="xref-fig" href="#F26">Supplemental Figure S10</a>) to allow for cell lysis and the removal extracellular DNA, respectively. Aliquots of the cell lysates were added to 3 ml of ΦLB top agar containing 50 μl log-phage E. coli C cells and plated onto a ΦLB agar plate. After four to five hours of incubation at 37°C, recombinant phage plaques were visible and plates were removed from the incubator. To verify that the recombinant phage encoded the intended sequence, we picked about 30 plaques for each intended mutational target and Sanger sequenced the entire targeted gene. We also sequenced the F and G genes for about 30 wildtype plaques to assure that no mutations were naturally accumulating. Briefly, individual plaques were picked with sterile toothpicks and placed in 200 uL ΦLB and gently swirled. 1 uL of this mix was used to PCR amplify approximately ½ of the ΦX174 genome using ΦX-0F (5’-GAGTTTTATCGCTTCCATG-3’) and ΦX-2953R (5’-CCGCCAGCAATAGCACC-3’) primers. Internal sequencing primers ΦX-979F (5’-CGGCCCCTTACTTGAGG-3’) and ΦX-1500R (5’-TTGAGATGGCAGCAACGG-3’) were used to sequence gene F. ΦX-2953R was used to sequence gene G. PCR cleanups and sequencing was done at Eurofins Genomics. PCR reaction conditions were: 5 uL10X Taq buffer, 2.5 uL 10 uM ΦX-0F primer, 10 uM ΦX-2953F primer, 0.8 uL 12.5 uM dNTPs, 0.5 uL Taq polymerase (NEB #M0273), 1 uL template, 37.7 uL H20. Thermocycling conditions were 1 cycle at 95°C for 2 min, 30 cycles at 95°C for 15 sec, 52°C for 30 sec, 68°C for 2 min, 1 cycle at 68°C for 5min.</p></div></div><div class="section" id="sec-11"><h2 class="">Data Availability</h2><p id="p-32">Raw sequencing reads for the truncated mutant libraries have been deposited in the SRA database: SAMN12660301-SAMN12660308. The processed data is available in the supporting information. Additionally, the oligo pool sequences, input files for generating the oligo pool, and an example of the command line input to execute the script from <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Medina-Cucurella et al. (2019)</a> are available in the Supplemental Information associated with this paper.</p></div><div class="section" id="sec-12"><h2 class="">Supplemental Information</h2><div class="section" id="sec-13"><h3>Note S1: DNA sequences of ΦX174 viral capsid protein gene fragments</h3><p id="p-33">Sequences provided span from the 5’ EcoRV to the 3’ BamHI site (bolded sequences). Mutagenic regions are highlighted in pink, non-mutated coding regions are highlighted in blue, BsmBI sites are highlighted in green. Regions with overlapping protein coding regions on different reading frames are colored dark blue.</p><div class="section" id="sec-14"><h4>&gt;Fragment_F1</h4><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F3.medium.gif" width="440" height="195"/><noscript><img class="highwire-fragment fragment-image" alt="Figure3" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F3.medium.gif" width="440" height="195"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-15"><h4>&gt;Fragment_F2</h4><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F4.medium.gif" width="440" height="140"/><noscript><img class="highwire-fragment fragment-image" alt="Figure4" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F4.medium.gif" width="440" height="140"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-16"><h4>&gt;Fragment_F3</h4><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-7.medium.gif" width="440" height="154"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-7.medium.gif" width="440" height="154"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-8.medium.gif" width="440" height="38"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-8.medium.gif" width="440" height="38"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F5/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div></div></div><div class="section" id="sec-17"><h4>&gt;Fragment_G1</h4><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F6.medium.gif" width="440" height="138"/><noscript><img class="highwire-fragment fragment-image" alt="Figure6" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F6.medium.gif" width="440" height="138"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-18"><h4>&gt;Fragment_G2/H1</h4><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F7.medium.gif" width="440" height="151"/><noscript><img class="highwire-fragment fragment-image" alt="Figure7" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F7.medium.gif" width="440" height="151"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-19"><h4>&gt;Fragment_H2</h4><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F8.medium.gif" width="440" height="137"/><noscript><img class="highwire-fragment fragment-image" alt="Figure8" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F8.medium.gif" width="440" height="137"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-20"><h4>&gt;Fragment_H3</h4><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-12.medium.gif" width="440" height="56"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-12.medium.gif" width="440" height="56"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-13.medium.gif" width="440" height="30"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-13.medium.gif" width="440" height="30"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F9/graphic-13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div></div></div><div class="section" id="sec-21"><h4>&gt;Fragment_A1/H</h4><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F10.medium.gif" width="440" height="70"/><noscript><img class="highwire-fragment fragment-image" alt="Figure10" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F10.medium.gif" width="440" height="70"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-22"><h4>&gt;Fragment_A2</h4><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F11.medium.gif" width="440" height="156"/><noscript><img class="highwire-fragment fragment-image" alt="Figure11" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F11.medium.gif" width="440" height="156"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-23"><h4>&gt;Fragment_A3</h4><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F12.medium.gif" width="440" height="111"/><noscript><img class="highwire-fragment fragment-image" alt="Figure12" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F12.medium.gif" width="440" height="111"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure12" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-24"><h4>&gt;Fragment_A4/B</h4><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-17.medium.gif" width="440" height="156"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-17.medium.gif" width="440" height="156"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-18.medium.gif" width="440" height="28"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-18.medium.gif" width="440" height="28"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F13/graphic-18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div></div></div><div class="section" id="sec-25"><h4>&gt;Fragment_K/C/A</h4><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure14" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F14.medium.gif" width="440" height="68"/><noscript><img class="highwire-fragment fragment-image" alt="Figure14" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F14.medium.gif" width="440" height="68"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure14" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-26"><h4>&gt;Fragment_C/K</h4><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure15" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F15.medium.gif" width="440" height="57"/><noscript><img class="highwire-fragment fragment-image" alt="Figure15" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F15.medium.gif" width="440" height="57"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure15" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-27"><h4>&gt;Fragment_D/C/E</h4><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure16" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F16.medium.gif" width="440" height="110"/><noscript><img class="highwire-fragment fragment-image" alt="Figure16" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F16.medium.gif" width="440" height="110"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure16" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div></div><div class="section" id="sec-28"><h3>Note S2: Amino acid sequences of ΦX174 mutagenized viral capsid protein gene fragments</h3><p id="p-34">Only mutagenized regions of gene fragments shown.</p><div class="section" id="sec-29"><h4>&gt;Fragment_F1</h4><p id="p-35">MSNIQTGAERMPHDLSHLGFLAGQIGRLITISTTPVIAGDSFEMDAVGALRLSPLR</p><p id="p-36">RGLAIDSTVDIFTFYVPHRHVYGEQWIKFMKDGVNATPL</p></div><div class="section" id="sec-30"><h4>&gt;Fragment_F2</h4><p id="p-37">NTTGYIDHAAFLGTINPDTNKIPKHLFQGYLNIYNNYFKAPWMPDRTEANPNELN</p><p id="p-38">QDDARYGFRCCHLKNIWTAPLPPETELSRQMTTSTTSIDIMGLQAAYANLHTDQE</p><p id="p-39">RDYFMQRYRDVISSFGGKTSYDADNRPLLVMRSN</p></div><div class="section" id="sec-31"><h4>&gt;Fragment_F3</h4><p id="p-40">SGYDVDGTDQTSLGQFSGRVQQTYKHSVPRFFVPEHGTMFTLALVRFPPTATKEI</p><p id="p-41">QYLNAKGALTYTDIAGDPVLYGNLPPREISMKDVFRSGDSSKKFKIAEGQWYRY</p><p id="p-42">APSYVSPAYHLLEGFPFIQEPPSGDLQERVLIRHHDYDQCFQSVQLLQWNSQVKF</p><p id="p-43">NVTVYRNLPTTRDSIMTS</p></div><div class="section" id="sec-32"><h4>&gt;Fragment_G1</h4><p id="p-44">FQTFISRHNSNFFSDKLVLTSVTPASSAPVLQTPKATSSTLYFDSLTVNAGNGGFL</p><p id="p-45">HCIQMDTSVNAANQVVSVGADIAFDADPKFFACLVRFESSSVPTTLPTAYDVYPL</p><p id="p-46">NGRHDGGYYTVKDCVTIDVLP</p></div><div class="section" id="sec-33"><h4>&gt;Fragment_G2</h4><p id="p-47">PGNNVYVGFMVWSNFTATKCRGLVSLNQVIKEIICLQPLK</p></div></div><div id="sec-34" class="subsection"><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Introduction of nicking sites into shuttle vectors containing viral genes. Verification of the introduction of the BbvCI nicking site into the shuttle vector containing the viral genes is shown by the template prep step in the nicking mutagenesis protocol (generation of ssDNA). Samples were run on a 1% agarose gel with SYBR&#x2122; Safe DNA gel stain (Invitrogen) added before casting, the ladder used is the 1 kb DNA ladder from GoldBio." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Introduction of nicking sites into shuttle vectors containing viral genes.&lt;/span&gt; Verification of the introduction of the BbvCI nicking site into the shuttle vector containing the viral genes is shown by the template prep step in the nicking mutagenesis protocol (generation of ssDNA). Samples were run on a 1% agarose gel with SYBR&#x2122; Safe DNA gel stain (Invitrogen) added before casting, the ladder used is the 1 kb DNA ladder from GoldBio.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F17.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Supplemental Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F17.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplemental Figure S1.</span> <span class="caption-title">Introduction of nicking sites into shuttle vectors containing viral genes.</span><p id="p-48" class="first-child">Verification of the introduction of the BbvCI nicking site into the shuttle vector containing the viral genes is shown by the template prep step in the nicking mutagenesis protocol (generation of ssDNA). Samples were run on a 1% agarose gel with SYBR™ Safe DNA gel stain (Invitrogen) added before casting, the ladder used is the 1 kb DNA ladder from GoldBio.</p><div class="sb-div caption-clear"></div></div></div><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="F1 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;F1 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F18.medium.gif" width="440" height="199"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F18.medium.gif" width="440" height="199"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S2:</span> <span class="caption-title">F1 heatmap of mutational frequency represented by number of sequencing read counts “counts”.</span><div class="sb-div caption-clear"></div></div></div><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="F2 tile 1 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;F2 tile 1 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F19.medium.gif" width="440" height="126"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F19.medium.gif" width="440" height="126"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S3:</span> <span class="caption-title">F2 tile 1 heatmap of mutational frequency represented by number of sequencing read counts “counts”.</span><div class="sb-div caption-clear"></div></div></div><div id="F20" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F20.large.jpg?width=800&amp;height=600&amp;carousel=1" title="F2 tile 2 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;F2 tile 2 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F20.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F20.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S4:</span> <span class="caption-title">F2 tile 2 heatmap of mutational frequency represented by number of sequencing read counts “counts”.</span><div class="sb-div caption-clear"></div></div></div><div id="F21" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F21.large.jpg?width=800&amp;height=600&amp;carousel=1" title="F3 tile 1 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;F3 tile 1 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F21.medium.gif" width="440" height="204"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F21.medium.gif" width="440" height="204"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S5:</span> <span class="caption-title">F3 tile 1 heatmap of mutational frequency represented by number of sequencing read counts “counts”.</span><div class="sb-div caption-clear"></div></div></div><div id="F22" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F22.large.jpg?width=800&amp;height=600&amp;carousel=1" title="F3 tile 2 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;F3 tile 2 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F22.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F22.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S6:</span> <span class="caption-title">F3 tile 2 heatmap of mutational frequency represented by number of sequencing read counts “counts”.</span><div class="sb-div caption-clear"></div></div></div><div id="F23" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F23.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G1 tile 1 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;G1 tile 1 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F23.medium.gif" width="440" height="140"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S7:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F23.medium.gif" width="440" height="140"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S7:</span> <span class="caption-title">G1 tile 1 heatmap of mutational frequency represented by number of sequencing read counts “counts”.</span><div class="sb-div caption-clear"></div></div></div><div id="F24" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F24.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G1 tile 2 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;G1 tile 2 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F24.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F24.medium.gif" width="440" height="206"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S8:</span> <span class="caption-title">G1 tile 2 heatmap of mutational frequency represented by number of sequencing read counts “counts”.</span><div class="sb-div caption-clear"></div></div></div><div id="F25" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F25.large.jpg?width=800&amp;height=600&amp;carousel=1" title="G2 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;G2 heatmap of mutational frequency represented by number of sequencing read counts &#x201C;counts&#x201D;.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F25.medium.gif" width="440" height="162"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S9:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F25.medium.gif" width="440" height="162"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F25.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S9:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F25.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S9:</span> <span class="caption-title">G2 heatmap of mutational frequency represented by number of sequencing read counts “counts”.</span><div class="sb-div caption-clear"></div></div></div><div id="F26" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F26.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Agarose gel image showing effectiveness of DNase1 in &#x3A6;LB. 25 cycles of PCR were performed after a 1-hour 37&#xB0;C incubation of WT ligation mix with DNase1. Primer sequences for phix_2375F and phix_2949R were 5&#x2019;-ACACTGACGACATGGTTCTACAtctgcttaggagtttaatc-3&#x2019; and 5&#x2019;-TACGGTAGCAGAGACTTGGTCTgcaccaaacataaatcacc-3&#x2019;. These primers include annealing sequences for Illumina 2nd round barcoding primers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-661599855" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Agarose gel image showing effectiveness of DNase1 in &#x3A6;LB.&lt;/span&gt; 25 cycles of PCR were performed after a 1-hour 37&#xB0;C incubation of WT ligation mix with DNase1. Primer sequences for phix_2375F and phix_2949R were 5&#x2019;-ACACTGACGACATGGTTCTACAtctgcttaggagtttaatc-3&#x2019; and 5&#x2019;-TACGGTAGCAGAGACTTGGTCTgcaccaaacataaatcacc-3&#x2019;. These primers include annealing sequences for Illumina 2&lt;sup&gt;nd&lt;/sup&gt; round barcoding primers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure S10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F26.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure S10:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F26.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F26.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure S10:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/09/798546/F26.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure S10:</span> <span class="caption-title">Agarose gel image showing effectiveness of DNase1 in ΦLB.</span><p id="p-49" class="first-child">25 cycles of PCR were performed after a 1-hour 37°C incubation of WT ligation mix with DNase1. Primer sequences for phix_2375F and phix_2949R were 5’-ACACTGACGACATGGTTCTACAtctgcttaggagtttaatc-3’ and 5’-TACGGTAGCAGAGACTTGGTCTgcaccaaacataaatcacc-3’. These primers include annealing sequences for Illumina 2<sup>nd</sup> round barcoding primers.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945441/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945441/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945441" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplemental table S1.</span> <span class="caption-title">Primers for incorporating BbvCI nicking sites into the pCR2.1-topo shuttle vector.</span><p id="p-50" class="first-child">Blue text is the overlap region with the shuttle vector, red is the KpnI site, green is the BbvCI site.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945452/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/945452/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplemental table S2.</span> <span class="caption-title">Inner and outer primers for PCR reactions for Illumina sequencing.</span><p id="p-51" class="first-child">Red indicates overhang regions for attaching Illumina adapter primers (inner PCR primers) or overhangs for attaching to inner PCR product (outer PCR primers), black is the overlap region in the gene or the barcode, blue is the Illumina adapter.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945442/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945442/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945442" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplemental Table S3.</span> <span class="caption-title">Mutant library preparation summary.</span><p id="p-52" class="first-child">Summary table of the transformants required for sufficient library coverage, transformants obtained during comprehensive mutant library preparation by nicking scanning mutagenesis, and the fold excess of the number of transformants required for coverage.</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945478/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945478/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945478" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table S4.</span> <span class="caption-title">Mutant plasmid library NGS statistics.</span><p id="p-53" class="first-child">Summary table of the libraries prior to viral genome assembly. Long fragments required separate amplicon sequencing reactions as shown.</p><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945481/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945481/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945481" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplemental Table S5.</span> <span class="caption-title">Viral growth rate during transformation cell recovery.</span><p id="p-54" class="first-child">Viral titers immediately after transformation (0 min) and then for every 30 min after up to 2 hours. As the XL1-Blue cells are not susceptible, the increase in titer is the result of cell bursts, which releases viable phage. While the virus is replicating for the first 30 minutes, they are all held within a cell and will only make one plaque.</p><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945450/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/945450/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplemental Table S6.</span> <span class="caption-title">List of observed mutations in plaque sequencing.</span><p id="p-55" class="first-child">Residues where more than one substitution was observed are bolded. Residues where mutations have been observed in previous studies (but not the exact substitution) are indicated*.</p><div class="sb-div caption-clear"></div></div></div><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/945443/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/945443/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/945443" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplemental Table S7.</span> <span class="caption-title">Testing for the optimal amount of ligation mix to transform.</span><p id="p-56" class="first-child">Different amounts and dilutions of the ligation mix were transformed into XL1-Blue cells to test for effects of DNA and salt concentration on transformation efficiency. The most transformants results from transforming 4 uL of ligated fragments.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-31">We thank the members of the Whitehead, Wichman, and Miller labs for feedback and assistance in composing this work. We especially thank LuAnn Scott for experimental expertise on the ΦX174 system. This work was supported by the National Science Foundation BEACON Centre for the Study of Evolution in Action and an NSF EPSCoR T2 under award numbers DBI-0939454 and OIA-1736253 and the National Institute of General Medical Sciences of the National Institutes of Health under award numbers P20GM104420 and R01GM076040. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-4">
<a href="https://figshare.com/articles/mutagenic_primer_seqs_phiX174/9956600">https://figshare.com/articles/mutagenic_primer_seqs_phiX174/9956600</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ando</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemire</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pires</span>,  <span class="cit-name-given-names">D.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">T.K.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Engineering modular viral scaffolds for targeted bacterial population editing</span>. <abbr class="cit-jnl-abbrev">Cell Systems</abbr> <span class="cit-vol">1</span>:<span class="cit-fpage">187</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D1%26rft.spage%253D187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aoyama</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamatake</span>,  <span class="cit-name-given-names">R.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayashi</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1981</span>) <span class="cit-article-title">Morphogenesis of ΦX174: in vitro synthesis of infectious phage from purified viral components</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Aca. Sci. USA</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">7285</span>–<span class="cit-lpage">7289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAca.%2BSci.%2BUSA%26rft.volume%253D12%26rft.spage%253D7285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.3" data-doi="10.1038/nrg3564"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barrick</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lenski</span>,  <span class="cit-name-given-names">R.E.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Genome dynamics during experimental evolution</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genetics</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">827</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenetics%26rft.volume%253D14%26rft.spage%253D827%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3564%26rft_id%253Dinfo%253Apmid%252F24166031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24166031&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.4" data-doi="10.1016/j.jmb.2004.02.033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernal</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hafenstein</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Esmeralda</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fane</span>,  <span class="cit-name-given-names">B.A.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-auth"><span class="cit-name-surname">Rossmann</span>,  <span class="cit-name-given-names">M.G.</span></span> <span class="cit-article-title">The ΦX174 protein J mediates DNA packaging and viral attachment to host cells</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">337</span>, <span class="cit-fpage">1109</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBernal%26rft.auinit1%253DR.%2BA.%26rft.volume%253D337%26rft.issue%253D5%26rft.spage%253D1109%26rft.epage%253D1122%26rft.atitle%253DThe%2BphiX174%2Bprotein%2BJ%2Bmediates%2BDNA%2Bpackaging%2Band%2Bviral%2Battachment%2Bto%2Bhost%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.02.033%26rft_id%253Dinfo%253Apmid%252F15046981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.02.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15046981&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bull</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Badgett</span>,  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wichman</span>,  <span class="cit-name-given-names">H.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hillis</span>,  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulati</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Molineux</span>,  <span class="cit-name-given-names">I.J.</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Exceptional Convergent Evolution in a Virus</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">147</span>, <span class="cit-fpage">1497</span>–<span class="cit-lpage">1507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBull%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D147%26rft.issue%253D4%26rft.spage%253D1497%26rft.epage%253D1507%26rft.atitle%253DExceptional%2BConvergent%2BEvolution%2Bin%2Ba%2BVirus%26rft_id%253Dinfo%253Apmid%252F9409816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Ny80LzE0OTciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8wOS83OTg1NDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.6" data-doi="10.1534/genetics.113.158154"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caudle</span>,  <span class="cit-name-given-names">S.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">C.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rokyta</span>,  <span class="cit-name-given-names">D.R.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Environment Determines Epistatic Patterns for a ssDNA Virus</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>, <span class="cit-fpage">267</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158154%26rft_id%253Dinfo%253Apmid%252F24214342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMjY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.7" data-doi="10.1038/s41591-019-0437-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dedrick</span>,  <span class="cit-name-given-names">R.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guerrero-Bustamante</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garlena</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ford</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilmour</span>,  <span class="cit-name-given-names">K.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soothill</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobs-Sera</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schooley</span>,  <span class="cit-name-given-names">R.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatfull</span>,  <span class="cit-name-given-names">G.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spencer</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Engineered bacteriophages for treatment of a patient with a disseminated drug-resistant Mycobacterium abscessus</span>. <abbr class="cit-jnl-abbrev">Nature Medicine</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">730</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMedicine%26rft.volume%253D25%26rft.spage%253D730%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41591-019-0437-z%26rft_id%253Dinfo%253Apmid%252F31068712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41591-019-0437-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31068712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.8" data-doi="10.1038/38537"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dokland</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ilag</span>,  <span class="cit-name-given-names">L.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowman</span>,  <span class="cit-name-given-names">B.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Incardona</span>,  <span class="cit-name-given-names">N.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fane</span>,  <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossmann</span>,  <span class="cit-name-given-names">M.G.</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Structure of a viral procapsid with molecular scaffolding</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">389</span>, <span class="cit-fpage">308</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDokland%26rft.auinit1%253DT.%26rft.volume%253D389%26rft.issue%253D6648%26rft.spage%253D308%26rft.epage%253D313%26rft.atitle%253DStructure%2Bof%2Ba%2Bviral%2Bprocapsid%2Bwith%2Bmolecular%2Bscaffolding.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F38537%26rft_id%253Dinfo%253Apmid%252F9305849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/38537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9305849&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XW77200054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.9" data-doi="10.1128/JVI.01804-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>,  <span class="cit-name-given-names">E.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yount</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burkett</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickles</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Systematic Assembly of a Full-Length Infectious Clone of Human Coronavirus NL63</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">82</span>, <span class="cit-fpage">11948</span>–<span class="cit-lpage">11957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01804-08%26rft_id%253Dinfo%253Apmid%252F18818320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Mi8yMy8xMTk0OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzA5Lzc5ODU0Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.10" data-doi="10.1093/molbev/msv130"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doore</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fane</span>,  <span class="cit-name-given-names">B.A.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">The kinetic and thermodynamic aftermath of horizontal gene transfer governs evolutionary recovery</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evo.</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">2571</span>–<span class="cit-lpage">2584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv130%26rft_id%253Dinfo%253Apmid%252F26025979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26025979&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doore</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schweers</span>,  <span class="cit-name-given-names">N.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fane</span>,  <span class="cit-name-given-names">B.A.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Elevating fitness after a horizontal gene exchange in bacteriophage ΦX174</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">501</span>, <span class="cit-fpage">25</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D501%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-798546v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doub</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bloom</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">How single mutations affect viral escape from broad and narrow antibodies to H1 influenza hemagglutinin</span>. <abbr class="cit-jnl-abbrev">Nature Comm.</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BComm.%26rft.volume%253D9%26rft.spage%253D1386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">R.A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohwer</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Viral metagenomics</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Micro.</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">504</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BMicro.%26rft.volume%253D3%26rft.spage%253D504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.14" data-doi="10.1093/molbev/msz184"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faber</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrenbeck</span>,  <span class="cit-name-given-names">E.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Azouz</span>,  <span class="cit-name-given-names">L.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steiner</span>,  <span class="cit-name-given-names">P.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitehead</span>,  <span class="cit-name-given-names">T.A.</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Impact of in vivo protein folding probability on local fitness landscapes</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evo.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msz184</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvo.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsz184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msz184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.15" data-doi="10.1093/molbev/msu081"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firnberg</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Labonte</span>,  <span class="cit-name-given-names">J.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ostermeier</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">A comprehensive, high resolution map of a gene’s fitness landscape</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>:<span class="cit-fpage">1581</span>–<span class="cit-lpage">1592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu081%26rft_id%253Dinfo%253Apmid%252F24567513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567513&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000337067400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.16" data-doi="10.1038/nmeth.3027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fowler</span>,  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fields</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Deep mutational scanning: a new stype of protein science</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">801</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D801%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3027%26rft_id%253Dinfo%253Apmid%252F25075907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25075907&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340075600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.17" data-doi="10.1021/acssynbio.5b00296"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garamella</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marshall</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rustad</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noireaux</span>,  <span class="cit-name-given-names">V.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">The all E. coli Toolbox 2.0: a platform for cell-free synthetic biology</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol.</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">344</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol.%26rft.volume%253D5%26rft.spage%253D344%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facssynbio.5b00296%26rft_id%253Dinfo%253Apmid%252F26818434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acssynbio.5b00296&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26818434&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hafenstein</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fane</span>,  <span class="cit-name-given-names">B.A.</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">ΦX174 genome-capsid interactions influence the biophysical properties of the virion: evidence for a scaffolding-like function for the genome during the final stages of morphogenesis</span>. <abbr class="cit-jnl-abbrev">J. of Viro.</abbr> <span class="cit-vol">76</span>, <span class="cit-fpage">5350</span>–<span class="cit-lpage">5356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2Bof%2BViro.%26rft.volume%253D76%26rft.spage%253D5350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hartman</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jakobson</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Favor</span>,  <span class="cit-name-given-names">A.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lobba</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Álvarez-Benedicto</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Francis</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tullman-Ercek</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Quantitative characterization of all single amino acid variants of a viral capsid based drug delivery vehicle</span>. <abbr class="cit-jnl-abbrev">Nature Comm.</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BComm.%26rft.volume%253D9%26rft.spage%253D1385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">What can we predict about viral evolution and emergence?</span> <abbr class="cit-jnl-abbrev">Curr. Opin. in Vir.</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">180</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2Bin%2BVir.%26rft.volume%253D3%26rft.spage%253D180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.21" data-doi="10.1016/j.virol.2012.09.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jaschke</span>,  <span class="cit-name-given-names">P.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lieberman</span>,  <span class="cit-name-given-names">E.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sierra</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Endy</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A fully decompressed synthetic bacteriophage ΦX174 genome assembled and archived in yeast</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">434</span>:<span class="cit-fpage">278</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D434%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2012.09.020%26rft_id%253Dinfo%253Apmid%252F23079106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2012.09.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23079106&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jaschke</span>,  <span class="cit-name-given-names">P.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dotson</span>,  <span class="cit-name-given-names">G.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hung</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Endy</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Definitive demonstration by synthesis of genome annotation completeness</span>. <abbr class="cit-jnl-abbrev">BioRxiv</abbr> <span class="cit-fpage">455428</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.23" data-doi="10.4161/rna.27766"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kiro</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shitrit</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qimron</span>,  <span class="cit-name-given-names">U.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Efficient engineering of a bacteriophage genome using the type I-E CRISPR-Cas system</span>. <abbr class="cit-jnl-abbrev">RNA Biol.</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">42</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol.%26rft.volume%253D11%26rft.spage%253D42%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.27766%26rft_id%253Dinfo%253Apmid%252F24457913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.27766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24457913&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331332800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klesmith</span>,  <span class="cit-name-given-names">J.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hackel</span>,  <span class="cit-name-given-names">B.J.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Improved mutation function prediction via PACT: Protein Analysis and Classifier Toolkit</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">2707</span>–<span class="cit-lpage">2712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D2707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.25" data-doi="10.1371/journal.pone.0118193"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kowalsky</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klesmith</span>,  <span class="cit-name-given-names">J.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stapleton</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reichkitzer</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitehead</span>  <span class="cit-name-given-names">T.A.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">High-Resolution Sequence-Function Mapping of Full-Length Proteins</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">e0118193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0118193%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0118193%26rft_id%253Dinfo%253Apmid%252F25790064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0118193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25790064&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.26" data-doi="10.1038/nmeth.2918"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kosuri</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Church</span>,  <span class="cit-name-given-names">G.M.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Large-scale de novo DNA synthesis: technologies and applications</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">499</span>–<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D499%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2918%26rft_id%253Dinfo%253Apmid%252F24781323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2918&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24781323&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335873400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.27" data-doi="10.1073/pnas.1806133115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huddleston</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doud</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hooper</span>,  <span class="cit-name-given-names">K.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">N.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bloom</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Deep mutational scanning of hemagglutinin helps predict evolutionary fates of human H3N2 influenza variants</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Aca. Sci. USA</abbr> <span class="cit-vol">115</span>, <span class="cit-fpage">E8276</span>–<span class="cit-lpage">E8285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAca.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1806133115%26rft_id%253Dinfo%253Apmid%252F30104379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzM1L0U4Mjc2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.28" data-doi="10.1021/sb500366v"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeLoache</span>,  <span class="cit-name-given-names">W.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cervantes</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dueber</span>,  <span class="cit-name-given-names">J.E.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">A highly characterized yeast toolkit for modular, multipart assembly</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol.</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">975</span>–<span class="cit-lpage">986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol.%26rft.volume%253D4%26rft.spage%253D975%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fsb500366v%26rft_id%253Dinfo%253Apmid%252F25871405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/sb500366v&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25871405&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.29" data-doi="10.1021/acssynbio.6b00388"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lemay</span>,  <span class="cit-name-given-names">M.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tremblay</span>,  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moineau</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Genome engineering of virulent lactococcal phages using CRIPSR-Cas9</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol.</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">1351</span>–<span class="cit-lpage">1358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol.%26rft.volume%253D6%26rft.spage%253D1351%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facssynbio.6b00388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acssynbio.6b00388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.30" data-doi="10.1038/ng.2848"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lieberman</span>,  <span class="cit-name-given-names">T.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flett</span>,  <span class="cit-name-given-names">K.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yelin</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">T.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McAdam</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Priebe</span>,  <span class="cit-name-given-names">G.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kishony</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Genetic variation of a bacterial pathogen within individuals with cystic fibrosis provides a record of selective pressures</span>. <abbr class="cit-jnl-abbrev">Nature Gen</abbr> <span class="cit-vol">46</span>:<span class="cit-fpage">82</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGen%26rft.volume%253D46%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2848%26rft_id%253Dinfo%253Apmid%252F24316980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2848&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24316980&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.31" data-doi="10.1038/355137a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willingmann</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ilag</span>,  <span class="cit-name-given-names">L.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krishnaswamy</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossmann</span>,  <span class="cit-name-given-names">M.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">N.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">T.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Incardona</span>,  <span class="cit-name-given-names">N.L.</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Atomic structure of single-stranded DNA bacteriophage ΦX174 and its functional implications</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">355</span>, <span class="cit-fpage">137</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcKenna%26rft.auinit1%253DR.%26rft.volume%253D355%26rft.issue%253D6356%26rft.spage%253D137%26rft.epage%253D143%26rft.atitle%253DAtomic%2Bstructure%2Bof%2Bsingle-stranded%2BDNA%2Bbacteriophage%2Bphi%2BX174%2Band%2Bits%2Bfunctional%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F355137a0%26rft_id%253Dinfo%253Apmid%252F1370343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/355137a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1370343&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Medina-Cucurella</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steiner</span>,  <span class="cit-name-given-names">P.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faber</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beltrán</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borelli</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirby</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cutler</span>,  <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitehead</span>,  <span class="cit-name-given-names">T.A.</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">User-defined single pot mutagenesis using unpurified oligo pools</span>. <abbr class="cit-jnl-abbrev">Prot. Eng. Des. Sel.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">1</span>)<span class="cit-fpage">41</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProt.%2BEng.%2BDes.%2BSel.%26rft.volume%253D32%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.33" data-doi="10.1146/annurev-ecolsys-110512-135920"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neher</span>,  <span class="cit-name-given-names">R.A.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Genetic draft, selective interference, and population genetics of rapid adaptation</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">44</span>:<span class="cit-fpage">195</span>–<span class="cit-lpage">215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D44%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110512-135920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110512-135920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pires</span>,  <span class="cit-name-given-names">D.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cleto</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sillankorva</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Azeredo</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">T.K.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Genetically engineered phages: a review of advances over the last decade</span>. <abbr class="cit-jnl-abbrev">Microbiol. and Mol. Biol. Rev.</abbr> <span class="cit-vol">80</span>:<span class="cit-fpage">523</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2Band%2BMol.%2BBiol.%2BRev.%26rft.volume%253D80%26rft.spage%253D523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.35" data-doi="10.1038/ng1535"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rokyta</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caudle</span>,  <span class="cit-name-given-names">S.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wichman</span>,  <span class="cit-name-given-names">H.A.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">An empirical test of the mutational landscape model of adaptation using a single-stranded DNA virus</span>. <abbr class="cit-jnl-abbrev">Nat Genet.</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">441</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRokyta%26rft.auinit1%253DD.%2BR.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D441%26rft.epage%253D444%26rft.atitle%253DAn%2Bempirical%2Btest%2Bof%2Bthe%2Bmutational%2Blandscape%2Bmodel%2Bof%2Badaptation%2Busing%2Ba%2Bsingle-stranded%2BDNA%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1535%26rft_id%253Dinfo%253Apmid%252F15778707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15778707&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228040000030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rokyta</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burch</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caudle</span>,  <span class="cit-name-given-names">S.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wichman</span>,  <span class="cit-name-given-names">H.A.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Horizontal gene transfer and the evolution of microvirid coliphage genomes</span>. <abbr class="cit-jnl-abbrev">J. of Bacter.</abbr> <span class="cit-vol">188</span>, <span class="cit-fpage">1134</span>–<span class="cit-lpage">1142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2Bof%2BBacter.%26rft.volume%253D188%26rft.spage%253D1134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rogers</span>,  <span class="cit-name-given-names">S.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bendich</span>,  <span class="cit-name-given-names">A.J.</span></span> (<span class="cit-pub-date">1988</span>) <span class="cit-article-title">Extraction of DNA from plant tissues</span>. <abbr class="cit-jnl-abbrev">Plant Mole Biol. Manual</abbr> <span class="cit-vol">A6</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMole%2BBiol.%2BManual%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.38" data-doi="10.1534/genetics.117.300451"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sackman</span>,  <span class="cit-name-given-names">A.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rokyta</span>,  <span class="cit-name-given-names">D.R.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Additive Phenotypes Underlie Epistasis of Fitness Effects</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">208</span>, <span class="cit-fpage">339</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300451%26rft_id%253Dinfo%253Apmid%252F29113978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA4LzEvMzM5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.39" data-doi="10.1016/0022-2836(78)90346-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanger</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coulson</span>,  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedmann</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Air</span>,  <span class="cit-name-given-names">G.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrell</span>,  <span class="cit-name-given-names">B.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">N.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fiddes</span>,  <span class="cit-name-given-names">J.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Slocombe</span>,  <span class="cit-name-given-names">P.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1978</span>) <span class="cit-article-title">The nucleotide sequence of bacteriophage phiX174</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">125</span>:<span class="cit-fpage">225</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSanger%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D2%26rft.spage%253D225%26rft.epage%253D246%26rft.atitle%253DThe%2Bnucleotide%2Bsequence%2Bof%2Bbacteriophage%2BphiX174.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252878%252990346-7%26rft_id%253Dinfo%253Apmid%252F731693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(78)90346-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=731693&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978FW60600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schilling</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dietrich</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoppert</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hertel</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">A CRISPR-Cas9-based toolkit for fast and precise in vivo genetic engineering of Bacillus subtilis phages</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253D241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.41" data-doi="10.1073/pnas.2237126100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">H.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>   <span class="cit-name-suffix">III</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfannkoch</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venter</span>,  <span class="cit-name-given-names">J.C.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Generating a synthetic genome by whole genome assembly: ΦX174 bacteriophage from synthetic oligonucleotides</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Aca. Sci. USA</abbr> <span class="cit-vol">100</span>, <span class="cit-fpage">15440</span>–<span class="cit-lpage">15445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAca.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2237126100%26rft_id%253Dinfo%253Apmid%252F14657399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzI2LzE1NDQwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMDkvNzk4NTQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.42" data-doi="10.1371/journal.pone.0131676"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stockdale</span>,  <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silvia</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Douillard</span>,  <span class="cit-name-given-names">F.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mahony</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cambillau</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Sinderen</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Structure and assembly of TP901-1 Virion unveiled by mutagenesis</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e0131676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0131676%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0131676%26rft_id%253Dinfo%253Apmid%252F26147978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0131676&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26147978&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.43" data-doi="10.1016/j.virol.2009.01.030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Uchiyama</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiman</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fane</span>,  <span class="cit-name-given-names">B.A.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">N-terminal deletions of the ΦX174 external scaffolding protein affect the timing and fidelity of assembly</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">386</span>, <span class="cit-fpage">303</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D386%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2009.01.030%26rft_id%253Dinfo%253Apmid%252F19237183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2009.01.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19237183&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F09%2F798546.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267310200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.44" data-doi="10.1021/acssynbio.7b00363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Dolleweerd</span>,  <span class="cit-name-given-names">C.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kessans</span>,  <span class="cit-name-given-names">S.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van de Bittner</span>,  <span class="cit-name-given-names">K.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">L.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bundela</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholson</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">E.J.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">MIDAS: a modular DNA assembly system for synthetic biology</span>. <abbr class="cit-jnl-abbrev">ACS Synth Biol</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">1018</span>–<span class="cit-lpage">1029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth%2BBiol%26rft.volume%253D7%26rft.spage%253D1018%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facssynbio.7b00363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acssynbio.7b00363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wichman</span>,  <span class="cit-name-given-names">H.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">C.J.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Experimental evolution of viruses: Microviridae as a model system</span>. <abbr class="cit-jnl-abbrev">Phil. Trans. of the Royal Society B</abbr>. <span class="cit-vol">365</span>, <span class="cit-fpage">2495</span>–<span class="cit-lpage">2501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2Bof%2Bthe%2BRoyal%2BSociety%2BB%26rft.volume%253D365%26rft.spage%253D2495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-798546v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yehl</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemire</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ando</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mimee</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torres</span>,  <span class="cit-name-given-names">M.D.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de la Fuente-Nunez</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">T.K.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Engineering Phage Host-Range and Suppressing Bacterial Resistance through Phage Tail Fiber Mutagenesis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">179</span>, <span class="cit-fpage">459</span>–<span class="cit-lpage">469.e9</span>. Genetics 170, 19-31.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D179%26rft.spage%253D459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/798546v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F798546v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/798967v1" title="Substrate-mediated regulation of the arginine transporter of Toxoplasma gondii" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 09, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/798546v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/10/09/798546.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/798546v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/798546v1" class="active" data-icon-position="" data-hide-link-title="0">Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/942725" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XC-IypkhpDDN18UmDP32HeNsUYiRn98AwL7NfaGky-I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="942725" id="node942725--2" data-pisa="biorxiv;798546v1" data-pisa-master="biorxiv;798546" data-seqnum="942725" data-apath="/biorxiv/early/2019/10/09/798546.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew S.</span> <span class="nlm-surname">Faber</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">James T.</span> <span class="nlm-surname">Van Leuven</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Martina M.</span> <span class="nlm-surname">Ederer</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yesol</span> <span class="nlm-surname">Sapozhnikov</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Zoë L.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Holly A.</span> <span class="nlm-surname">Wichman</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Timothy A.</span> <span class="nlm-surname">Whitehead</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Craig R.</span> <span class="nlm-surname">Miller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">798546; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/798546 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/798546v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;title=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;title=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;text=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;title=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;t=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;title=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;title=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="942725" id="node942725--3" data-pisa="biorxiv;798546v1" data-pisa-master="biorxiv;798546" data-seqnum="942725" data-apath="/biorxiv/early/2019/10/09/798546.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Saturation mutagenesis genome engineering of infective ΦX174 bacteriophage via unamplified oligo pools and golden gate assembly</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew S.</span> <span class="nlm-surname">Faber</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">James T.</span> <span class="nlm-surname">Van Leuven</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Martina M.</span> <span class="nlm-surname">Ederer</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yesol</span> <span class="nlm-surname">Sapozhnikov</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Zoë L.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Holly A.</span> <span class="nlm-surname">Wichman</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Timothy A.</span> <span class="nlm-surname">Whitehead</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Craig R.</span> <span class="nlm-surname">Miller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">798546; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/798546 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Faber%2C+Matthew+S.&amp;rft.au=Van+Leuven%2C+James+T.&amp;rft.au=Ederer%2C+Martina+M.&amp;rft.au=Sapozhnikov%2C+Yesol&amp;rft.au=Wilson%2C+Zo%C3%AB+L.&amp;rft.au=Wichman%2C+Holly+A.&amp;rft.au=Whitehead%2C+Timothy+A.&amp;rft.au=Miller%2C+Craig+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/942725/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/942725/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/942725/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/942725/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/942725/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/942725/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/942725/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/942725/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/942725/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/942725/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/942725/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/942725/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;count=horizontal&amp;via=&amp;text=Saturation%20mutagenesis%20genome%20engineering%20of%20infective%20%CE%A6X174%20bacteriophage%20via%20unamplified%20oligo%20pools%20and%20golden%20gate%20assembly&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/798546v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/798546v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/798546v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
