<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/24/852657.atom"/>
  <meta name="HW.pisa" content="biorxiv;852657v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="VADR: validation and annotation of virus sequence submissions to GenBank"/>
  <meta name="DC.Identifier" content="10.1101/852657"/>
  <meta name="DC.Date" content="2020-03-24"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This article is a US Government work. It is not subject to copyright under 17 USC 105 and is also made available for use under a CC0 license"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Background GenBank contains over 3 million viral sequences. The National Center for Biotechnology Information (NCBI) previously made available a tool for validating and annotating influenza virus sequences that is used to check submissions to GenBank. Before this project, there was no analogous tool in use for non-influenza viral sequence submissions. Results We developed a system called VADR (Viral Annotation DefineR) that validates and annotates viral sequences in GenBank submissions. The annotation system is based on the analysis of the input nucleotide sequence using models built from curated RefSeqs. Hidden Markov models are used to classify sequences by determining the RefSeq they are most similar to, and feature annotation from the RefSeq is mapped based on a nucleotide alignment of the full sequence to a covariance model. Predicted proteins encoded by the sequence are validated with nucleotide-to-protein alignments using BLAST. The system identifies 43 types of “alerts” that (unlike the previous BLAST-based system) provide deterministic and rigorous feedback to researchers who submit sequences with unexpected characteristics. VADR has been integrated into GenBank’s submission processing pipeline allowing for viral submissions passing all tests to be accepted and annotated automatically, without the need for any human (GenBank indexer) intervention. Unlike the previous submission-checking system, VADR is freely available (&lt;;https://github.com/nawrockie/vadr&gt;;) for local installation and use. VADR has been used for Norovirus submissions since May 2018 and for Dengue virus submissions since January 2019. Other viruses with high numbers of submissions will be added incrementally. Conclusion VADR improves the speed with which non-flu virus submissions to GenBank can be checked and improves the content and quality of the GenBank annotations. The availability and portability of the software allow researchers to run the GenBank checks prior to submitting their viral sequences, and thereby gain confidence that their submissions will be accepted immediately without the need to correspond with GenBank staff. Reciprocally, the adoption of VADR frees GenBank staff to spend more time on services other than checking routine viral sequence submissions. * NCBI : National Center for Biotechnology Information VADR : Viral Annotation DefineR BLAST : Basic Local Alignment Search Tool CM : Covariance Model HMM : Hidden Markov Model nt : nucleotides Kb : kilobase (1000 nucleotides) ncRNA : noncoding RNA mat_peptide : mature peptide NP : Norovirus Partial dataset NC : Norovirus Complete dataset DP : Dengue virus Partial dataset DC : Dengue virus Complete dataset"/>
  <meta name="DC.Contributor" content="Alejandro A Schäffer"/>
  <meta name="DC.Contributor" content="Eneida L Hatcher"/>
  <meta name="DC.Contributor" content="Linda Yankie"/>
  <meta name="DC.Contributor" content="Lara Shonkwiler"/>
  <meta name="DC.Contributor" content="J Rodney Brister"/>
  <meta name="DC.Contributor" content="Ilene Karsch-Mizrachi"/>
  <meta name="DC.Contributor" content="Eric P Nawrocki"/>
  <meta name="article:published_time" content="2020-03-24"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="VADR: validation and annotation of virus sequence submissions to GenBank"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Background: GenBank contains over 3 million viral sequences. The National Center for Biotechnology Information (NCBI) previously made available a tool for validating and annotating influenza virus sequences that is used to check submissions to GenBank. Before this project, there was no analogous tool in use for non-influenza viral sequence submissions. Results: We developed a system called VADR (Viral Annotation DefineR) that validates and annotates viral sequences in GenBank submissions. The annotation system is based on the analysis of the input nucleotide sequence using models built from curated RefSeqs. Hidden Markov models are used to classify sequences by determining the RefSeq they are most similar to, and feature annotation from the RefSeq is mapped based on a nucleotide alignment of the full sequence to a covariance model. Predicted proteins encoded by the sequence are validated with nucleotide-to-protein alignments using BLAST. The system identifies 43 types of &quot;;alerts&quot;; that (unlike the previous BLAST-based system) provide deterministic and rigorous feedback to researchers who submit sequences with unexpected characteristics. VADR has been integrated into GenBank9s submission processing pipeline allowing for viral submissions passing all tests to be accepted and annotated automatically, without the need for any human (GenBank indexer) intervention. Unlike the previous submission-checking system, VADR is freely available (https://github.com/nawrockie/vadr) for local installation and use. VADR has been used for Norovirus submissions since May 2018 and for Dengue virus submissions since January 2019. Other viruses with high numbers of submissions will be added incrementally. Conclusion: VADR improves the speed with which non-flu virus submissions to GenBank can be checked and improves the content and quality of the GenBank annotations. The availability and portability of the software allow researchers to run the GenBank checks prior to submitting their viral sequences, and thereby gain confidence that their submissions will be accepted immediately without the need to correspond with GenBank staff. Reciprocally, the adoption of VADR frees GenBank staff to spend more time on services other than checking routine viral sequence submissions.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;852657v2"/>
  <meta name="citation_id" content="852657v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/852657v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/852657v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/852657v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/852657.full.pdf"/>
  <meta name="citation_doi" content="10.1101/852657"/>
  <meta name="citation_num_pages" content="16"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="852657"/>
  <meta name="citation_author" content="Alejandro A Schäffer"/>
  <meta name="citation_author_institution" content="Cancer Data Science Laboratory, National Cancer Insitute, National Institutes of Health"/>
  <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health"/>
  <meta name="citation_author" content="Eneida L Hatcher"/>
  <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health"/>
  <meta name="citation_author" content="Linda Yankie"/>
  <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health"/>
  <meta name="citation_author" content="Lara Shonkwiler"/>
  <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health"/>
  <meta name="citation_author_institution" content="Massachusetts Institute of Technology Cambridge"/>
  <meta name="citation_author" content="J Rodney Brister"/>
  <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health"/>
  <meta name="citation_author" content="Ilene Karsch-Mizrachi"/>
  <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health"/>
  <meta name="citation_author" content="Eric P Nawrocki"/>
  <meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health"/>
  <meta name="citation_author_email" content="nawrocke@ncbi.nlm.nih.gov"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2497-3427"/>
  <meta name="citation_reference" content="Sayers EW, Cavanaugh M, Clark K, Ostell J, Pruitt KD, Karsch-Mizrachi I. GenBank. Nucleic Acids Res. 2019;47:D94–D99."/>
  <meta name="citation_reference" content="Bao Y, Bolotov P, Dernovoy D, Kiryutin B, Tatusova T. FLAN: a web Server for Influenza Virus Genome Annotation. Nucleic Acids Res. 2007;35:W280–W284."/>
  <meta name="citation_reference" content="Hatcher EL, Zhdanov SA, Bao Y, Blinkova O, Nawrocki EP, Ostapchuck Y, et al. Virus Variation Resource - improved response to emergent viral outbreaks. Nucleic Acids Res. 2017;45:D482–D490."/>
  <meta name="citation_reference" content="Waterman MS. Genomic sequence databases. Genomics. 1990;6:700–701."/>
  <meta name="citation_reference" content="Strasser BJ. The Experimenter’s Museum: GenBank, Natural History, and the Moral Economies of Biomedicine. Isis. 2011;102:60–96."/>
  <meta name="citation_reference" content="Tatusova T, DiCuccio M, Badretdin A, Chetvernin V, Nawrocki EP, Zaslavsky L, et al. NCBI Prokaryotic Genome Annotation Pipeline. Nucleic Acids Res. 2016;44:6614–6624."/>
  <meta name="citation_reference" content="Shean RC, Makhsous N, Stoddard GD, Lin MJ, Greninger AJ. VAPiD: A Lightweight Cross-Platform Viral Annotation Pipeline and Identification Tool to Facilitate Virus Genome Submissions to NCBI GenBank. BMC Bioinformatics. 2019;20:48."/>
  <meta name="citation_reference" content="Wang S, Sundaram JP, Spiro D. VIGOR, an Annotation Program for Small Viral Genomes. BMC Bioinformatics. 2010;11:451."/>
  <meta name="citation_reference" content="Wang S, Sundaram JP, Stockwell TB. VIGOR Extended to Annotate Genomes for Additional 12 Different Viruses. Nucleic Acids Res. 2012;40:W186–W192."/>
  <meta name="citation_reference" content="Zhang KY, Gao YZ, Du MZ, Liu S, Dong C, Guo FB. Vgas: A Viral Genome Annotation System. Front Microbiol. 2019;10:184."/>
  <meta name="citation_reference" content="González-Tortuero E, Sean Sutton TD, Velayudhan V, Shkoporov AN, Draper LA, Stockdale SR, et al. VIGA: a sensitive, precise and automatic de novo VIral Genome Annotator. bioRxiv. 2018;."/>
  <meta name="citation_reference" content="O’Leary NA, Wright MW, Brister JR, Ciufo S, Haddad D, McVeigh R, et al. Reference Sequence (RefSeq) Database at NCBI: Current Status, Taxonomic Expansion, and Functional Annotation. Nucleic Acids Res. 2016;44:D733–D745."/>
  <meta name="citation_reference" content="Pijlman GP, Funk A, Kondratieva N, Leung J, Torres S, van der Aa L, et al. A Highly Structured, Nuclease-Resistant, Noncoding RNA Produced by Flaviviruses is Required for Pathogenicity. Cell Host Microbe. 2008;4:579–591."/>
  <meta name="citation_reference" content="Tycowski KT, Guo YE, Lee N, Moss WN, Vallery TK, Xie M, et al. Viral Noncoding RNAs: More Surprises. Genes Dev. 2015;29:567–584."/>
  <meta name="citation_reference" content="Altschul SF, Madden TL, Schäffer AA, Zhang J, Zhang Z, Miller W, et al. Gapped BLAST and PSI-BLAST: a New Generation of Protein Database Search Programs. Nucleic Acids Res. 1997;25:3389–3402."/>
  <meta name="citation_reference" content="Wernersson R, Pedersen AG. RevTrans: Multiple Alignment of Coding DNA From Aligned Amino Acid Sequences. Nucleic Acids Res. 2003;31:3537–3539."/>
  <meta name="citation_reference" content="Eddy SR. Accelerated Profile HMM Searches. PLoS Comput Biol. 2011;7:e1002195."/>
  <meta name="citation_reference" content="Wheeler TJ, Eddy SR. nhmmer: DNA Homology Search With Profile HMMs. Bioinformatics. 2013;29:2487–2489."/>
  <meta name="citation_reference" content="Nawrocki EP, Eddy SR. Infernal 1.1: 100-fold Faster RNA Homology Searches. Bioinformatics. 2013;29:2933–2935."/>
  <meta name="citation_reference" content="Eddy SR. A Memory-Efficient Dynamic Programming Algorithm for Optimal Alignment of a Sequence to an RNA Secondary Structure. BMC Bioinformatics. 2002;3:18."/>
  <meta name="citation_reference" content="Durbin R, Eddy SR, Krogh A, Mitchison GJ. Biological Sequence Analysis: Probabilistic Models of Proteins and Nucleic Acids. Cambridge UK: Cambridge University Press; 1998."/>
  <meta name="citation_reference" content="Brown MP. Small Subunit Ribosomal RNA Modeling Using Stochastic Context-Free Grammars. ISMB. 2000;8:57–66."/>
  <meta name="citation_reference" content="Nawrocki EP. Structural RNA Homology Search and Alignment Using Covariance Models [Ph.D. thesis]. Washington University School of Medicine; 2009."/>
  <meta name="citation_reference" content="Hulo C, de Castro E, Masson P, Bougueleret L, Bairoch A, Xenarios I, et al. ViralZone: a Knowledge Resource to Understand Virus Diversity. Nucleic Acids Res. 2011;39:D576–D582."/>
  <meta name="citation_reference" content="Jacks T, Power MD, Masiarz FR, Luciw PA, Barr PJ, Varmus HE. Characterization of Ribosomal Frameshifting in HIV-1 gag-pol Expression. Nature. 1988;331:280–283."/>
  <meta name="citation_reference" content="Kang H, Tinoco Jr I. A Mutant RNA Pseudoknot That Promotes Ribosomal Frameshifting in Mouse Mammary Tumor Virus. Nucleic Acids Res. 1997;25:1943–1949."/>
  <meta name="citation_reference" content="Baranov PV, Henderson CM, Anderson CB, Gesteland RF, Atkins JF, Howard MT. Programmed Ribosomal Frameshifting in Decoding the SARS-CoV Genome. Virology. 2005;332:498–510."/>
  <meta name="citation_reference" content="Thurner C, Witwer C, Hofacker IL, Stadler PF. Conserved RNA Secondary Structures in Flaviviridae Genomes. J Gen Virol. 2004;85:1113–1124."/>
  <meta name="citation_reference" content="Smyth RP, Negroni M, Lever AM, Mak J, Kenyon JC. RNA Structure-A Neglected Puppet Master for the Evolution of Virus and Host Immunity. Front Immunol. 2018;9:2097."/>
  <meta name="citation_reference" content="Filomatori CV, Carballeda JM, Villordo SM, Aguirre S, Pallarés HM, Maestre AM, et al. Dengue Virus Genomic Variation Associated With Mosquito Adaptation Defines the Pattern of Viral non-Coding RNAs and Fitness in Human Cells. PLoS Pathog. 2017;13:e1006265."/>
  <meta name="citation_reference" content="MacFadden A, O’Donoghue Z, Silva PAGC, Chapman EG, Olsthoorn RC, Sterken MG, et al. Mechanism and Structural Diversity of Exoribonuclease-Resistant RNA Structures in Flaviviral RNAs. Nat Commun. 2018;9:119."/>
  <meta name="citation_reference" content="Chhabra P, de Graaf M, Parra GI, Chan MC, Green K, Martella V, et al. Updated Classification of Norovirus Genogroups and Genotypes. J Gen Virol. 2019;100:1393–1406."/>
  <meta name="citation_reference" content="Eddy SR. Profile Hidden Markov Models. Bioinformatics. 1998;14:755–763."/>
  <meta name="citation_reference" content="Freyhult EK, Bollback JP, Gardner PP. Exploring Genomic Dark Matter: A Critical Assessment of the Performance of Homology Search Methods on Noncoding RNA. Genome Res. 2007;17:117–125."/>
  <meta name="citation_reference" content="Lagesen K, Hallin P, Rødland EA, Staerfeldt H, Rognes T, Ussery DW. RNAmmer: Consistent and Rapid Annotation of Ribosomal RNA Genes. Nucleic Acids Res. 2007;35:3100–3108."/>
  <meta name="citation_reference" content="Lee JH, Yi H, Chun J. rRNASelector: a Computer Program for Selecting Ribosomal RNA Encoding Sequences From Metagenomic and Metatranscriptomic Shotgun Libraries. J Microbiol. 2011;49:689–691."/>
  <meta name="citation_reference" content="Faggioni G, Pomponi A, Santis RD, Masuelli L, Ciammaruconi A, Monaco F, et al. West Nile Alternative Open Reading Frame (N-NS4B/WARF4) is Produced in Infected West Nile Virus (WNV) Cells and Induces Humoral Response in WNV Infected Individuals. Virol J. 2012;9:283."/>
  <meta name="twitter:title" content="VADR: validation and annotation of virus sequence submissions to GenBank"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Background GenBank contains over 3 million viral sequences. The National Center for Biotechnology Information (NCBI) previously made available a tool for validating and annotating influenza virus sequences that is used to check submissions to GenBank. Before this project, there was no analogous tool in use for non-influenza viral sequence submissions. Results We developed a system called VADR (Viral Annotation DefineR) that validates and annotates viral sequences in GenBank submissions. The annotation system is based on the analysis of the input nucleotide sequence using models built from curated RefSeqs. Hidden Markov models are used to classify sequences by determining the RefSeq they are most similar to, and feature annotation from the RefSeq is mapped based on a nucleotide alignment of the full sequence to a covariance model. Predicted proteins encoded by the sequence are validated with nucleotide-to-protein alignments using BLAST. The system identifies 43 types of “alerts” that (unlike the previous BLAST-based system) provide deterministic and rigorous feedback to researchers who submit sequences with unexpected characteristics. VADR has been integrated into GenBank’s submission processing pipeline allowing for viral submissions passing all tests to be accepted and annotated automatically, without the need for any human (GenBank indexer) intervention. Unlike the previous submission-checking system, VADR is freely available (&lt;;https://github.com/nawrockie/vadr&gt;;) for local installation and use. VADR has been used for Norovirus submissions since May 2018 and for Dengue virus submissions since January 2019. Other viruses with high numbers of submissions will be added incrementally. Conclusion VADR improves the speed with which non-flu virus submissions to GenBank can be checked and improves the content and quality of the GenBank annotations. The availability and portability of the software allow researchers to run the GenBank checks prior to submitting their viral sequences, and thereby gain confidence that their submissions will be accepted immediately without the need to correspond with GenBank staff. Reciprocally, the adoption of VADR frees GenBank staff to spend more time on services other than checking routine viral sequence submissions. * NCBI : National Center for Biotechnology Information VADR : Viral Annotation DefineR BLAST : Basic Local Alignment Search Tool CM : Covariance Model HMM : Hidden Markov Model nt : nucleotides Kb : kilobase (1000 nucleotides) ncRNA : noncoding RNA mat_peptide : mature peptide NP : Norovirus Partial dataset NC : Norovirus Complete dataset DP : Dengue virus Partial dataset DC : Dengue virus Complete dataset"/>
  <meta name="og-title" property="og:title" content="VADR: validation and annotation of virus sequence submissions to GenBank"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/852657v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Background GenBank contains over 3 million viral sequences. The National Center for Biotechnology Information (NCBI) previously made available a tool for validating and annotating influenza virus sequences that is used to check submissions to GenBank. Before this project, there was no analogous tool in use for non-influenza viral sequence submissions. Results We developed a system called VADR (Viral Annotation DefineR) that validates and annotates viral sequences in GenBank submissions. The annotation system is based on the analysis of the input nucleotide sequence using models built from curated RefSeqs. Hidden Markov models are used to classify sequences by determining the RefSeq they are most similar to, and feature annotation from the RefSeq is mapped based on a nucleotide alignment of the full sequence to a covariance model. Predicted proteins encoded by the sequence are validated with nucleotide-to-protein alignments using BLAST. The system identifies 43 types of “alerts” that (unlike the previous BLAST-based system) provide deterministic and rigorous feedback to researchers who submit sequences with unexpected characteristics. VADR has been integrated into GenBank’s submission processing pipeline allowing for viral submissions passing all tests to be accepted and annotated automatically, without the need for any human (GenBank indexer) intervention. Unlike the previous submission-checking system, VADR is freely available (&lt;;https://github.com/nawrockie/vadr&gt;;) for local installation and use. VADR has been used for Norovirus submissions since May 2018 and for Dengue virus submissions since January 2019. Other viruses with high numbers of submissions will be added incrementally. Conclusion VADR improves the speed with which non-flu virus submissions to GenBank can be checked and improves the content and quality of the GenBank annotations. The availability and portability of the software allow researchers to run the GenBank checks prior to submitting their viral sequences, and thereby gain confidence that their submissions will be accepted immediately without the need to correspond with GenBank staff. Reciprocally, the adoption of VADR frees GenBank staff to spend more time on services other than checking routine viral sequence submissions. * NCBI : National Center for Biotechnology Information VADR : Viral Annotation DefineR BLAST : Basic Local Alignment Search Tool CM : Covariance Model HMM : Hidden Markov Model nt : nucleotides Kb : kilobase (1000 nucleotides) ncRNA : noncoding RNA mat_peptide : mature peptide NP : Norovirus Partial dataset NC : Norovirus Complete dataset DP : Dengue virus Partial dataset DC : Dengue virus Complete dataset"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-24"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>VADR: validation and annotation of virus sequence submissions to GenBank | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1210341 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <div id="sec-1" class="subsection">
     <p id="p-2">
      <strong>Background</strong> GenBank contains over 3 million viral sequences. The National Center for Biotechnology Information (NCBI) previously made available a tool for validating and annotating influenza virus sequences that is used to check submissions to GenBank. Before this project, there was no analogous tool in use for non-influenza viral sequence submissions.
     </p>
    </div>
    <div id="sec-2" class="subsection">
     <p id="p-3">
      <strong>Results</strong> We developed a system called VADR (Viral Annotation DefineR) that validates and annotates viral sequences in GenBank submissions. The annotation system is based on the analysis of the input nucleotide sequence using models built from curated RefSeqs. Hidden Markov models are used to classify sequences by determining the RefSeq they are most similar to, and feature annotation from the RefSeq is mapped based on a nucleotide alignment of the full sequence to a covariance model. Predicted proteins encoded by the sequence are validated with nucleotide-to-protein alignments using BLAST. The system identifies 43 types of “alerts” that (unlike the previous BLAST-based system) provide deterministic and rigorous feedback to researchers who submit sequences with unexpected characteristics. VADR has been integrated into GenBank’s submission processing pipeline allowing for viral submissions passing all tests to be accepted and annotated automatically, without the need for any human (GenBank indexer) intervention. Unlike the previous submission-checking system, VADR is freely available (
      <a href="https://github.com/nawrockie/vadr">https://github.com/nawrockie/vadr</a>) for local installation and use. VADR has been used for Norovirus submissions since May 2018 and for Dengue virus submissions since January 2019. Other viruses with high numbers of submissions will be added incrementally.
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <p id="p-4">
      <strong>Conclusion</strong> VADR improves the speed with which non-flu virus submissions to GenBank can be checked and improves the content and quality of the GenBank annotations. The availability and portability of the software allow researchers to run the GenBank checks prior to submitting their viral sequences, and thereby gain confidence that their submissions will be accepted immediately without the need to correspond with GenBank staff. Reciprocally, the adoption of VADR frees GenBank staff to spend more time on services other than checking routine viral sequence submissions.
     </p>
    </div>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Background</h2>
    <p id="p-7">As of September 2019, GenBank [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>] contained more than 3 million viral sequences totaling over 4 billion nucleotides in length and including over 180,000 complete genomes for viruses other than influenza. More than 250,000 of these sequences were submitted in 2018. All sequence submissions are validated prior to deposition in GenBank. Automated validation and annotation methods become increasingly important as sequence submission numbers grow.
    </p>
    <p id="p-8">
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the number of sequences for the 16 virus species with the most sequences in GenBank. Influenza sequences are the second most abundant and the National Center of Biotechnology Information (NCBI), where GenBank is housed, has expended considerable effort to organize flu sequences and streamline the submission of new influenza virus sequences, including a tool to validate and annotate flu submissions called FLAN [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. The influenza virus sequence submission tool (
     <a href="https://www.ncbi.nlm.nih.gov/genome/viruses/variation/help/flu-help-center/submit-flu-sequences/">https://www.ncbi.nlm.nih.gov/genome/viruses/variation/help/flu-help-center/submit-flu-sequences/</a>) is implemented specifically for influenza with many hard-coded features. It has proven impractical to reuse the influenza virus submission code for other viruses.
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1217154/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1217154/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1217154" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1</span>
      <p id="p-9" class="first-child">Viruses with the highest number of sequences in GenBank as of October 10, 2019. The number of sequences for the segmented viruses Influenza and Rotavirus are the sums over all their segments.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-10">NCBI’s Virus Variation Resource [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>] (
     <a href="https://www.ncbi.nlm.nih.gov/genome/viruses/variation">https://www.ncbi.nlm.nih.gov/genome/viruses/variation</a>) includes specialized components that attempt to normalize annotation of previously submitted sequences for rotaviruses, dengue virus, West Nile virus, ebolaviruses, Zika virus, and MERS coronavirus. The resulting standardized annotation supports virus-specific searches using gene and protein names. However, the Virus Variation Resource does not support the submission of new sequences, and its tools are not generalizable, such that creating components for additional virus species is laborious.
    </p>
    <div id="sec-5" class="subsection">
     <h3>Automating validation of sequence submissions</h3>
     <p id="p-11">Early in the history of GenBank, Michael Waterman presciently wrote that “Entering new sequences into the databases requires the database staff to analyze and interpret the sequences and the associated scientific literature [
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>].” Even earlier, Margaret Dayhoff had justified her enormous efforts at sequence database curation reasoning that “a carefully verified collection [of sequences] was more economical in the long run than a quick and dirty collection” [
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. At NCBI, sequence submissions have been checked by indexers since NCBI began operating GenBank, but over the years, limitations to this arrangment have become apparent. First, the number and size of sequence submissions has grown much faster than the NCBI budget and the capability to hire more indexers. Second, although GenBank indexers are trained rigorously and homogeneously and use the same software, there is no formal mechanism to enforce “inter-observer agreement”, meaning that all GenBank indexers would be guaranteed to give the same evaluation of the same submission. Third, researchers who wish to submit their sequences cannot reproduce all the checks that GenBank indexers do. Consequently, problematic sequences cause delays and e-mail interactions between submitters and indexers that might be avoidable using a more deterministic and open system of checks. The importance of transparency in curatorial analysis was emphasized by Walter Goad, one of the founders of GenBank: “It is important that we be perceived by the molecular biology community as offering free and open access to the information 
      <em>and programs</em> we will be collecting” [
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>].
     </p>
     <p id="p-12">For bacterial genomes, these limitations have been addressed by the release of NCBI’s Prokaryotic Genome Annotation Pipeline (PGAP) [
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], which is now available for download as a standalone package (
      <a href="https://github.com/ncbi/pgap/releases">https://github.com/ncbi/pgap/releases</a>). For influenza virus, the limitations were addressed first by FLAN [
      <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>] and then by the influenza virus submission tools provided by GenBank [
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>]. Tools for checking submissions of non-influenza viruses have lagged but in the work described herein we describe new software that begins to close the gap. Importantly, our new software can already be used for many viruses (currently excluding circular genome and segmented viruses) breaking from the paradigm of developing code for one virus at a time that has limited the breadth of the Virus Variation Resource.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Software for viral genome annotation</h3>
     <p id="p-13">Several software packages exist for viral genome annotation. VAPiD [
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>] is designed to simplify the submission of complete viral genome sequences to GenBank by adding metadata, and annotating each input sequence based on comparison with its best-matching reference sequence in a large reference dataset derived from GenBank. VIGOR [
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, 
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>] annotates input sequences by first identifying the most relevant reference database in its collection and then comparing all reference protein and mature peptide sequences in that database to the input sequence to determine its annotation and to identify certain types of errors. Both programs detect and report some types of unexpected errors, such as premature stop codons.
     </p>
     <p id="p-14">Additional programs for viral annotation include VGAS [
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], which incorporates 
      <em>ab initio</em> ORF prediction as well as similarity-based annotation, and VIGA [
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] which can be optimized for speed for huge metagenomics datasets. All of these programs focus primarily on annotating protein-coding regions, although VIGA additionally identifies some types of RNAs and CRISPR repeat elements. Among these four programs, VAPiD and VIGOR provide some capabilities to prepare submissions for GenBank and hence are closer to solving the submission checking problem considered here than are VGAS and VIGA.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>VADR: Viral Annotation DefineR</h3>
     <p id="p-15">In this work, we describe the design and implementation of a new reference-based general software tool called VADR (Viral Annotation DefineR), for the validation and annotation of virus sequences from characterized species that have an open (non-circular) genome less than 25Kb in length and for which a reference (e.g. RefSeq entry) exists. We discuss the use of VADR in evaluating GenBank submissions of norovirus and dengue virus sequences. We compare VADR performance with that of VAPiD and VIGOR and show how it identifies more potential problems with sequences, making it more appropriate for use by GenBank indexers interested in manually reviewing those problems prior to sequence deposition.</p>
     <p id="p-16">VADR compares each input sequence to a library of homology models of viral species built from reference sequences from the RefSeq database [
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], identifies the most similar model, and uses that model to compute an alignment to the RefSeq from which feature annotation boundaries (e.g. coding sequences (denoted CDS), mature peptides, ncRNAs) are derived. Finally, CDS features that encode proteins are validated for protein-coding potential using 
      <em>blastx</em>. Submitted sequences that are confidently aligned and annotated with VADR 
      <em>pass</em> and are cleared for automatic entry into GenBank. In contrast, when a submitted sequence is evaluated by VADR and the comparison to its matching RefSeq reveals the input sequence is divergent in various ways (e.g. early stop codon, regions of low nucleotide similarity), then the sequence 
      <em>fails</em>. Failure means that the sequence is flagged for manual review by an NCBI expert curator, called a “GenBank indexer”, and the sequence is prevented from automatic entry into GenBank. If all sequences in a submission pass, all sequences will automatically be deposited into GenBank. If at least one sequence fails, a report with sequence-specific errors is generated and reported to the submitter or reviewed by the indexer, who can clear sequences for submission or contact the submitter for further investigation of the apparent problems.
     </p>
     <p id="p-17">VADR’s output annotation of each sequence is a fivecolumn feature table conforming to the GenBank feature table syntax rules, and includes all desired features from the RefSeq that could be mapped onto the input sequence via sequence alignment. These features include protein coding regions, the “mature peptide” cleavage products of proteins, genes, noncoding RNAs, and structural RNA features. In the nomenclature of the International Nucleotide Sequence Database Collaboration (INSDC), these are denoted as 
      <em>CDS</em>, 
      <em>mat peptide</em>, 
      <em>gene</em>, 
      <em>ncRNA</em>, and 
      <em>stem loop</em> features, respectively. Submitters rarely annotate RNA features in GenBank virus records even though many viruses have functionally important non-coding RNAs (ncRNAs) and other structural RNA features [
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, 
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. VADR’s capability to annotate these features adds value to the resulting GenBank records.
     </p>
     <p id="p-18">VADR is in production usage for sequence submissions of norovirus and dengue virus, two prominent human pathogens from 
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Dengue virus is a member of family 
      <em>Flaviviridae</em>, which also includes the West Nile Virus and Hepacivirus C (Hepatitis C). (We use upper case Norovirus and Dengue virus when referring to formal names and lower case norovirus and dengue virus when referring to attributes of sequences.) VADR is publicly available, so that researchers who plan to submit norovirus or dengue virus sequences to GenBank may pre-check their submissions with VADR to determine which sequences pass and fail.
     </p>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Implementation</h2>
    <p id="p-19">VADR is written in Perl and is available at 
     <a href="https://github.com/nawrockie/vadr">https://github.com/nawrockie/vadr</a>. The new software uses the existing software packages and libraries listed in 
     <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.
    </p>
    <div id="T2" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1217161/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1217161/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1217161" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 2</span>
      <p id="p-20" class="first-child">Software packages and libraries used within VADR.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-21">VADR contains two major scripts. The 
     <em>v-build.pl</em> script is used to build models of a viral species (e.g. norovirus). The 
     <em>v-annotate.pl</em> script uses those models to annotate sequences from the viral species represented by the models. The main workflow of VADR is depicted in 
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/852657/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="VADR workflow schematic illustrating uses of the two main VADR scripts. v-build.pl can be used once to build a single model or repeatedly to build a library of models. v-annotate.pl can be used with a model or model library to validate and annotate input sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1188439432" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;VADR workflow schematic illustrating uses of the two main VADR scripts.&lt;;/span&gt;; &lt;;em&gt;;v-build.pl&lt;;/em&gt;; can be used once to build a single model or repeatedly to build a library of models. &lt;;em&gt;;v-annotate.pl&lt;;/em&gt;; can be used with a model or model library to validate and annotate input sequences.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/852657/F1.medium.gif" width="440" height="332"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/852657/F1.medium.gif" width="440" height="332"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/852657/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/852657/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1</span>
      <span class="caption-title">VADR workflow schematic illustrating uses of the two main VADR scripts.</span>
      <p id="p-22" class="first-child">
       <em>v-build.pl</em> can be used once to build a single model or repeatedly to build a library of models. 
       <em>v-annotate.pl</em> can be used with a model or model library to validate and annotate input sequences.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="sec-9" class="subsection">
     <h3>The build module: 
      <em>v-build.pl</em>
     </h3>
     <p id="p-23">The 
      <em>v-build.pl</em> script takes as input two arguments: the RefSeq accession to be modeled (e.g. “NC 001959”, for Norovirus genotype GI) and the name of the output directory to create and to populate with output files.
     </p>
     <p id="p-24">The script first retrieves a FASTA-formatted sequence file and five-column feature table files for the RefSeq and any protein sequences associated with CDS features of the RefSeq from NCBI. 
      <em>v-build.pl</em> parses these files and outputs a VADR “model information” file with coordinates of CDS, mat peptide, and gene features and product and exception qualifiers. By default, all other feature types and qualifiers are ignored, although command-line options can be used to specify that additional feature types, such as ncRNA and stem loop, and qualifiers should be included. 
      <em>v-build.pl</em> also outputs a covariance model (CM) file for the input RefSeq created using Infernal v1.1.3’s 
      <em>cmbuild</em> program. If the RefSeq nucleotide sequence being modeled has known secondary structural elements, such as stem loops, then a Stockholm format file with structure annotation can be provided, and the resulting CM file will model the specified structure. This structure will inform the sequence-and-structure based alignment of input sequences by Infernal’s 
      <em>cmalign</em> program in the annotation stage of 
      <em>v-annotate.pl</em>. By default, if no Stockholm file is input to 
      <em>v-build.pl</em>, then the RefSeq sequence is modeled without secondary structure. Additionally, 
      <em>v-build.pl</em> uses the 
      <em>makeblastdb</em> program from BLAST v2.9.0+ [
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>] to create a BLAST database from amino acid translations of the RefSeq CDS features. 
      <em>v-annotate.pl</em> uses this database with 
      <em>blastx</em> to validate its nucleotide-based predictions of CDS features. This design in which VADR annotates nucleotide sequences, but validates with protein alignments allows us to mitigate the known limitation that the 4-letter nucleic acid alphabet is less sensitive in homology searching than the 20-letter amino acid alphabet [
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>], while retaining the capability to annotate ncRNAs and other features that are not translated.
     </p>
     <p id="p-25">
      <em>v-build.pl</em> is designed to allow input alignments instead of single RefSeq accessions. Although models built from alignments have not yet been tested, we plan to use alignment-derived models in the future.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>VADR model library</h3>
     <p id="p-26">VADR includes a library of models generated by 
      <em>v-build.pl</em>, represented by a “vadr.minfo” model info file, a “vadr.cm” CM file, and BLAST database files that include information and models for 194 viral RefSeq genomes (v1.0 VADR library). These 194 RefSeqs comprise 9 norovirus sequences, 4 dengue virus sequences, and an additional 181 RefSeqs from other viruses in the families 
      <em>Caliciviridae</em> and 
      <em>Flaviviridae</em>. The additional 181 RefSeqs allow VADR to recognize when an input sequence purported to be norovirus or dengue virus is in fact a different 
      <em>Caliciviridae</em> or 
      <em>Flaviviridae</em> taxon. This library is currently used by GenBank indexers on submitted sequences to annotate norovirus and dengue virus sequences and to recognize some taxonomically misclassified sequences. Users are able to build their own model libraries with 
      <em>v-build.pl</em> to use instead of this default library. We plan to add additional models to the default set in future versions after testing them internally at NCBI.
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>The annotation module: 
      <em>v-annotate.pl</em>
     </h3>
     <p id="p-27">The 
      <em>v-annotate.pl</em> script takes as input a sequence file in FASTA format and the name of an output directory. It uses the models in the default v1.0 VADR model library to analyze and to annotate the input sequences. 
      <em>v-annotate.pl</em> creates output files including a feature table with annotations in the output directory. The workflow of the 
      <em>v-annotate.pl</em> script consists of four stages: classification, coverage determination, alignment-based annotation and protein validation, each of which is described in more detail below.
     </p>
     <p id="p-28">During each stage, specific possible problems are identified and reported in the output as “alerts”. An alert is meant to inform a VADR user about an unusual, unexpected, or otherwise remarkable characteristic of a sequence. There are 43 types of alerts listed and briefly explained in 
      <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Alerts can pertain to either the entire sequence being evaluated (indicated by “S” in the “S/F” column in 
      <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), or a specific feature of that sequence (indicated by “F” in the “S/F” column). By default, 38 of the 43 alert types are 
      <em>fatal</em> in that they cause a sequence to fail annotation, whereas 5 alerts are reported but not fatal. A sequence passes if and only if it generates zero fatal alerts. Nearly all alert types can be set as fatal or not fatal using the command line options 
      <em>--alt pass</em> and 
      <em>--alt fail</em> to 
      <em>v-annotate.pl</em>, but four alert types are always fatal and cannot be changed: 
      <em>noannotn</em>, 
      <em>revcompl</em>, 
      <em>unexdivg</em>, and 
      <em>noftrann</em>.
     </p>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1217162/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1217162/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1217162" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3</span>
       <p id="p-29" class="first-child">Attributes of the 43 types of VADR alerts organized by the stage in which they are detected. The S/F column indicates whether the alert applies to an entire sequence (S) or to one feature (F) in a sequence. The five non-fatal alerts (four detected in the classification stage, and one in the coverage stage) do not cause a sequence to fail and are not reported in the output feature table. Codes marked with * are always fatal; all other codes can be set to fatal or non-fatal with command-line options to v-annotate.pl.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-30">One motivation for reporting alerts is to identify sequences that should be reviewed by a GenBank indexer as opposed to being entered automatically into the GenBank database without any human inspection. Sequences that pass are automatically entered into GenBank, while sequences that fail are not and must be manually reviewed by an indexer. The alert output contains specific information (e.g. nucleotide positions) when possible to facilitate further inspection by the indexer or by the submitter running VADR locally.</p>
     <div id="sec-12" class="subsection">
      <h4>Classification stage</h4>
      <p id="p-31">Each 
       <em>v-annotate.pl</em> input query sequence is first scored against each model in the library, which by default is the VADR v1.0 library of 194 models. The comparison is made between HMMs from the CM file and each sequence using an abbreviated version of the HM-MER3 [
       <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, 
       <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>] filter pipeline as implemented in Infernal v1.1.3 [
       <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. The full HMMER3 pipeline includes more than five stages, but only the first three stages are carried out during this classification stage of 
       <em>v-annotate.pl</em>, which computes a local forward score of a window of the input sequence. The window boundaries reported at this stage of the filter pipeline are inexact but this imprecision is irrelevant for the classification stage’s goal of identifying the model that scores highest for each input sequence. The window boundaries are more precisely defined at later stages. In practice, the window reported at this stage will often be the entire sequence. For each sequence 
       <em>S</em>, the highest scoring window and associated model 
       <em>M</em> are found, and 
       <em>M</em> (
       <em>S</em>) is defined as the best-scoring model for 
       <em>S</em> to be used in the subsequent stages.
      </p>
     </div>
     <div id="sec-13" class="subsection">
      <h4>Coverage determination stage</h4>
      <p id="p-32">For each sequence 
       <em>S</em>, the model 
       <em>M</em> (
       <em>S</em>) is used to determine the sequence coverage. This is achieved again using the HMMER3 filter pipeline as implemented in Infernal 1.1.3, but now using the full pipeline that reports local matches in each input sequence, with more precise endpoints than the abbreviated pipeline used in the classification stage. For sequences that include internal short stretches of sequence that are dissimilar from the RefSeq, multiple matches may be returned at this stage, with the dissimilar regions not covered by any of the matches. The coverage of 
       <em>S</em> is determined as the fraction of nucleotides in 
       <em>S</em> that occur in any of the alignments on the top strand (+ strand). An alert (
       <em>lowcovrg</em>) is reported for any sequence with coverage below 0.9. Additional alerts can be reported at this stage (
       <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>) based on unexpected characteristics of the set of returned alignments. For example, the 
       <em>indf-strn</em> alert is reported if at least one alignment with a score of 25 bits or more occurs on each strand.
      </p>
     </div>
     <div id="sec-14" class="subsection">
      <h4>Alignment and feature mapping stage</h4>
      <p id="p-33">Each complete sequence 
       <em>S</em> is aligned to the covariance model for 
       <em>M</em> (
       <em>S</em>) using Infernal’s 
       <em>cmalign</em> program generating a Stockholm formatted alignment file. The alignment is then parsed to determine a mapping between sequence positions and model positions, from which per-feature coordinates for 
       <em>S</em> are derived. The 
       <em>cmalign</em> output Stockholm alignment file includes posterior probability values that indicate a level of confidence in each aligned nucleotide. These posterior probability values are used to identify features for which the endpoints are not confidently assigned by the model, which are reported as fatal alerts (
       <em>indf5loc</em> and 
       <em>indf3loc</em>). Different fatal alerts are reported if an input sequence has a gap at a feature endpoint (
       <em>indf5gap</em> or 
       <em>indf3loc</em>). Additional alerts are potentially reported at this stage as well (
       <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>).
      </p>
      <p id="p-34">Standard CM alignment has high computational complexity in time and memory (
       <em>O</em>(
       <em>N</em>
       <sup>4</sup>) in time and 
       <em>O</em>(
       <em>N</em>
       <sup>3</sup>) in memory [
       <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, 
       <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]). Infernal utilizes a constrained alignment technique using bands derived from a first-pass HMM alignment of each sequence to make alignment practical [
       <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, 
       <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. Input sequences that are dissimilar from the model’s RefSeq will have loose constraints, and the required memory for alignment may exceed 8Gb, in which case 
       <em>cmalign</em> will exit in error without aligning the sequence. 
       <em>v-annotate.pl</em> detects for which sequences this occurs, and reports an always fatal 
       <em>unexdivg</em> alert.
      </p>
     </div>
     <div id="sec-15" class="subsection">
      <h4>Protein validation of CDS features</h4>
      <p id="p-35">For each input sequence 
       <em>S</em>, each predicted CDS feature of 30 or more nucleotides, as well as the full sequence 
       <em>S</em>, is then used as a 
       <em>blastx</em> query against the BLAST database of the RefSeq protein sequences created by 
       <em>v-build.pl</em> for model 
       <em>M</em> (
       <em>S</em>). The top 
       <em>blastx</em> match for each RefSeq protein is compared to the CM-based prediction and alerts are generated if specific differences exist. For example, if the endpoints differ by more than five nucleotides on either the 5’ or 3’ ends, a 
       <em>indf5pst</em> or 
       <em>indf3pst</em> alert is reported. The main purpose of this stage is to identify any frameshift mutations in the input sequence, which may not have triggered any upstream alerts. Also at this stage, 
       <em>insertnp</em> or 
       <em>deletinp</em> alerts are reported for sequences with in-frame insertions or deletions longer than an indexer-specified taxon-specific threshold (set at 27 nucleotides by default), so that an indexer can check whether such a large insertion or deletion is plausible.
      </p>
     </div>
    </div>
   </div>
   <div class="section" id="sec-16">
    <h2 class="">Results and discussion</h2>
    <p id="p-36">The VADR software package defines in one place all the checks that GenBank indexers want to do on the nucleotide sequences of incoming virus submissions. We define cautious operational semantics, summarized in 
     <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>, such that any sequence that might have a problem fails, so that an indexer can check the sequence.
    </p>
    <div id="sec-17" class="subsection">
     <h3>Obtaining norovirus and dengue virus sequences</h3>
     <p>To test the joint hypothesis that the VADR operational semantics are more cautious and more rigorous than what was done before its development (pre-2018), we collected a set of test sequences that were last modified no later than December 31, 2017. Specifically, we ran each of the following two Entrez Nucleotide queries on June 24, 2019: </p>
     <ol class="list-ord " id="list-1">
      <li id="list-item-1">
       <p id="p-38">Norovirus NOT chimeric AND 1989:2017[MDAT] AND 50:20000[slen]</p>
      </li>
      <li id="list-item-2">
       <p id="p-39">Dengue NOT chimeric AND 1989:2017[MDAT] AND 50:20000[slen]</p>
      </li>
     </ol>
     <p id="p-40">Distinct “complete genome” and “partial” sequence datasets were created to distinguish behavior on likely complete genomes from behavior on likely partial sequences. To separate the likely complete genomes, we used the empirical cumulative distribution functions of how many sequences returned by the above queries have each length, x, and are (not) annotated as “complete genome”. The selected thresholds were &gt;; 7380nt and &gt;; 10, 371nt for norovirus and dengue virus, respectively because almost all sequences labeled “complete genome” exceed these lengths. However, there were 5 norovirus and 27 dengue virus sequences shorter than these threshold lengths returned by the above Entrez queries that are annotated “complete genome”. Therefore, we revised the queries partitioning each sets of lengths into two disjoint intervals by using the “[slen]” attribute. We refer to the four datasets or selected subsets thereof as “NP”, “NC”, “DP”, and “DC” for Norovirus-Partial, Norovirus-Complete, Dengue-Partial and Dengue-Complete.</p>
     <ol class="list-ord " id="list-2">
      <li id="list-item-3">
       <p id="p-41">Norovirus NOT chimeric AND 1989:2017[MDAT] AND 50:7380[slen]</p>
      </li>
      <li id="list-item-4">
       <p id="p-42">Norovirus NOT chimeric AND 1989:2017[MDAT] AND 7381:20000[slen]</p>
      </li>
      <li id="list-item-5">
       <p id="p-43">Dengue NOT chimeric AND 1989:2017[MDAT] AND 50:10371[slen]</p>
      </li>
      <li id="list-item-6">
       <p id="p-44">Dengue NOT chimeric AND 1989:2017[MDAT] AND 10372:20000[slen]</p>
      </li>
     </ol>
     <p id="p-45">Queries 1 and 3 retrieved lists of 32,190 and 20,973 sequences respectively. Queries 2 and 4 retrieved lists of 1384 and 4580 sequences respectively. The partial sequences far outnumber the complete sequences for these two taxa. For this reason, we avoided design decisions in VADR that would rely on any specific part of the genome being present.</p>
     <p id="p-46">The sets of sequences returned by these queries could shrink over time if sequences are modified in the future. Two choices in our queries may be unexpected and deserve further explanation. First, we put the upper bound “2017” on the date because in 2018 we began using VADR to screen incoming sequences, so it would be unfair to include these more recent sequences in our testing here. In 2018, the versions of VADR were changing rapidly, so testing results from 2018 are omitted in the sequence counts at the beginning of Results, for which we started counting in January 2019. Second, we did not require that the strings Norovirus and Dengue be in the organism field, so as to include some sequences that could plausibly be misclassified taxonomically, and thereby test the behavior of VADR’s classification module. For simplicity and except where evaluating classification errors, we refer to the sequences as “norovirus sequences” and “dengue virus sequences”, even though some of them are known to belong to other taxa.</p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>VADR results for norovirus and dengue virus sequences</h3>
     <p id="p-47">We ran the VADR v1.0 
      <em>v-annotate.pl</em> script with default parameters on the four sequence datasets. The numbers of sequences that pass and fail VADR from each of the four sets are shown in 
      <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. For norovirus, about 92% of partial sequences and about 84% of complete genomes passed VADR. For dengue virus, about 82% of partial sequences and about 91% of complete genome sequences passed. The differences between the VADR pass rates on partial and complete sequences are partly due to idiosyncracies of typical primer pairs used for norovirus and dengue virus sequences and whether they capture portions of the genome that are easier (norovirus) or harder (dengue virus) to align.
     </p>
     <div id="T4" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1217157/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1217157/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1217157" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 4</span>
       <p id="p-48" class="first-child">Summary of pass/fail outcomes for VADR on the full datasets</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-49">
      <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> lists each type of fatal VADR alert observed in one of the four datasets with counts of instances reported and sequences for which one or more instances was reported. The most common alert, 
      <em>peptrans</em>, occurs 6613 times in 1781 of the 59,127 sequences, approximately 3%. This alert does not indicate a unique problem itself, but rather is reported for mature peptides for which the parent CDS that is cleaved to form the mature peptide has a fatal alert, so it is redundant with at least one other alert. The next most common alert, 
      <em>noannotn</em>, occurs for 2753 sequences, 2236 of which are in the DP dataset, indicating that no similar RefSeq was found for these sequences during the classification stage. Other alerts with more than 1000 instances include 
      <em>indf5pst</em> and 
      <em>indf3pst</em> which occur when the 
      <em>blastx</em> protein-based alignment of a predicted CDS translation in the validation stage does not extend to with 5 nucleotides of the 5’ or 3’ ends of the nucleotide-based alignment. Thirty additional fatal alerts occurred for at least one sequence. Four fatal alert types did not occur for any sequence: 
      <em>unexdivg</em> would be reported in the rare case that a sequence was recognized as similar to a RefSeq but too divergent to align within memory requirements; 
      <em>lowsimis</em> would be reported if a dissimilar region occured outside all predicted features, which is unlikely for norovirus and dengue virus which have features nearly along their full length; 
      <em>indfstrp</em> would be reported if 
      <em>blastx</em> reported similarity to a CDS region on the negative strand, when nucleotide similarity is primarily recognized on the positive strand, and 
      <em>incsbgrp</em> would be reported if a sequence is recognized as not belonging to a specified subgroup (e.g. Norovirus genotype), but was never reported in these tests because we did not specify subgroups. The 
      <em>incsbgrp</em> alert was added to fit the design of the NCBI submission interface for norovirus in which submitters are asked to specify the genogroup, and it is triggered only when VADR judges that the sequence belongs to a different genogroup.
     </p>
     <div id="T5" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1217159/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1217159/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1217159" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 5</span>
       <p id="p-50" class="first-child">Counts of fatal VADR alerts reported for the test datasets. The 34 fatal alert codes reported at least once for any test dataset are listed sorted by total number of reports. 4 fatal alert types (unexdivg, lowsimis, incsbgrp and indfstrp.) were not reported for any of the four test sets and are not shown.See 
        <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a> for more information on alerts.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-19" class="subsection">
     <h3>Test datasets for comparison with VAPiD and VIGOR</h3>
     <p id="p-51">Showing that VADR catches problems not caught before is not sufficient to prove that VADR is necessary to check virus submissions to GenBank, as existing software packages may be able to identify these problems as well. We compared the performance of VADR with that of VAPiD and VIGOR on randomly chosen sequences from the sets described above. From each of the four sets of sequences we randomly selected 200 to create four datasets of 200 sequences each for testing, with the condition that any sequence with 10 or more consecutive Ns, or with more than 50% ambiguous nucleotides was rejected from our sets of 200 because those sequences are flagged for manual review by NCBI indexers upon submission, outside the operation of VADR.</p>
     <p id="p-52">VAPiD expects complete genome sequences [
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>] and we verified that it reports spurious errors on the majority of partial sequences. Therefore, here, we report VAPiD tests only on the NC and DC (complete) datasets. The version of VIGOR we tested (VIGOR3) lacks a Dengue virus database, so we tested VIGOR only on the Norovirus NC and NP. Neither VAPiD nor VIGOR is currently designed to handle partial dengue virus sequences and this gives a simple justification for why VADR is needed to check dengue virus submissions.
     </p>
    </div>
    <div id="sec-20" class="subsection">
     <h3>Defining pass/fail criteria for VAPiD and VIGOR</h3>
     <p id="p-53">VAPiD and VIGOR were designed optimistically to prepare submissions to GenBank, while VADR was designed more pessimistically to identify unexpected characteristics in submitted sequences. VAPiD and VIGOR do catch some errors, but neither package has an overall definition of a sequence passing or failing as VADR does. Therefore, as described below, we defined operational semantics of pass/fail in VIGOR and VAPiD to make them more comparable to VADR.</p>
     <p id="p-54">To understand the operational semantics we imposed on VAPiD and VIGOR, it is necessary to begin with the point that they were designed to search the submitted sequence against one or more large databases of known sequences. These databases play a role analogous to the RefSeqs for VADR in that the best match in the database to the input sequence defines what features should be annotated. Because our tests used real sequences already in GenBank and the VAPiD and VIGOR databases also contain sequences already in GenBank, there turns out to be a non-negligible probability that our query sequences are already “known to” VAPiD and/or VIGOR.</p>
     <p id="p-55">For VAPiD, a sequence was considered to fail if: no reference matching it was found in the VAPiD database, the sequence was determined to be reverse complemented relative to the selected reference, or one or more “SEQ_FEAT” errors was listed in the output 
      <em>errorsummary.val</em> file; all other sequences were considered to pass. Errors observed in our testing are “SEQ_FEAT.CdTransFail”, “SEQ_FEAT.NoStop”, “SEQ_FEAT.TransLen”, “SEQ_FEAT.StartCodon”, and “SEQ_FEAT.InternalStop”.
     </p>
     <p id="p-56">For VIGOR, any sequence that has no features annotated in the output 
      <em>.tbl</em> file or for which one or more lines in the output 
      <em>.at</em> file contained a code other than “OK”, “T5” or “T3” in the seventh field was considered to fail and all other sequences were considered to pass. Failure codes observed in our testing are “ES” and “FS” for detected embedded stops and detected frameshifts, respectively.
     </p>
    </div>
    <div id="sec-21" class="subsection">
     <h3>Results of VADR, VAPiD, and VIGOR on test sets</h3>
     <p id="p-57">
      <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> shows the number of sequences that pass and fail with each of the three methods for the datasets on which they were tested. 
      <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a> further compares the results of VADR and VAPiD on the NC and DC datasets, and 
      <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a> further compares the results of VADR and VIGOR on the NC and NP datasets. Because VADR is supposed to identify sequences with unusual properties that should be examined by an expert NCBI indexer before deposition into GenBank, the main goal of the comparison with VAPiD and VIGOR was to ensure that VADR identifies all the problems identified by the other two programs, and possibly more. To determine whether this is the case or not in our test sets, we examine the set of sequences that fail each method in turn below.
     </p>
     <div id="T6" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1217158/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1217158/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1217158" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 6</span>
       <p id="p-58" class="first-child">Summary of pass/fail outcomes for VADR, VAPiD and VIGOR on the 200 sequence test datasets</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T7" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1217155/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1217155/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1217155" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 7</span>
       <p id="p-59" class="first-child">Comparison of pass/fail outcomes for VADR and VAPiD on the 200 sequence Norovirus-Complete (NC) and Dengue-Complete (DC) test datasets.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T8" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1217156/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1217156/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1217156" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 8</span>
       <p id="p-60" class="first-child">Comparison of pass/fail outcomes for VADR and VIGOR on the 200 sequence Norovirus-Complete (NC) and Norovirus-Partial (NP) test datasets.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="sec-22" class="subsection">
      <h4>Test sequences that failed VAPiD</h4>
      <p id="p-61">In our tests of the NC and DC datasets, VAPiD failed 43 total sequences; 12 of these were also failed by VADR and 31 were passed by VADR and VIGOR (
       <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>). Of the 12, nine had an internal stop (SEQ_FEAT.InternalStop) or another CDS translation problem (SEQ_FEAT.CdsTransFail), one had a start codon problem (SEQ_FEAT.StartCodon), one was reverse complemented (FV536857.1; this was the only sequence of the 12 that failed VIGOR as well), and one failed because no similar reference was found for it. All 31 sequences that failed VAPiD but did not fail the other two programs (30 from the NC set and 1 from the DC set) were either 5’ truncated in the first CDS (Norovirus nonstructural polyprotein or Dengue virus polyprotein) or 3’ truncated in the final CDS (Norovirus VP2) or both, according to the VADR and VIGOR results, but were otherwise valid. These all failed VAPiD with at least one of the SEQ_FEAT.NoStop or SEQ_FEAT.CdTransFail errors, which is not surprising because VAPiD was designed for complete genomes and so does not expect truncated CDS sequences. Thus, we conclude that VADR successfully catches all the VAPiD-reported errors that should be caught.
      </p>
     </div>
     <div id="sec-23" class="subsection">
      <h4>Test sequences that failed VIGOR</h4>
      <p id="p-62">In our tests of the NC and NP datasets, VIGOR failed seven total sequences, all seven of which also fail VADR (
       <a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>). Of the seven, three had premature stops, two were reverse complemented (one of these is FV536857.1), one had a frameshift and one failed because no similar reference was found for it. Thus, we conclude that VADR successfully catches all the VIGOR-reported errors that should be caught.
      </p>
     </div>
     <div id="sec-24" class="subsection">
      <h4>Test sequences that failed VADR</h4>
      <p id="p-63">In our tests of the NC, NP, and DC datasets, VADR failed 53 sequences; 18 of these were also failed by VAPiD and/or VIGOR as mentioned above (exactly one sequence, FV536857.1, in the NC dataset failed all three methods), and 35 passed VAPiD and/or VIGOR (
       <a id="xref-table-wrap-8-3" class="xref-table" href="#T8">Table 8</a>). These 35 sequences had issues that were not flagged by either of the other two programs but that indexers would like to manually review before possibly accepting the sequences into GenBank. Those issues include: 16 sequences with early stop codons compared with the closest RefSeq (
       <em>cdsstopn</em> alerts), 12 sequences for which a 
       <em>blastx</em> alignment in the protein validation stage did not extend close enough to the 3’ boundary predicted from the nucleotide-based alignment (
       <em>indf5pst</em> or 
       <em>indf3pst</em> alerts), ten sequences with low similarity to the RefSeq at the 5’ or 3’ end of the sequence or an annotated feature (
       <em>lowsim5f</em>, 
       <em>lowsim3f</em>, or 
       <em>lowsim3s</em> alerts), seven sequences where the 5’ or 3’ boundary of a feature was not aligned with sufficient confidence (
       <em>indf5loc</em>, 
       <em>indf5gap</em>, or 
       <em>indf3loc</em> alerts), five sequences that were expected to be Norovirus but were classified as Sapovirus, another 
       <em>Caliciviridae</em> genus (
       <em>incgroup</em> alerts), five sequences with too large of an insertion or deletion in a 
       <em>blastx</em> alignment (
       <em>insertnp</em> or 
       <em>deletinp</em> alerts), and one sequence not recognized by any of the 
       <em>Caliciviridae</em> or 
       <em>Flaviviridae</em> models (a Salivirus from the 
       <em>Picornaviridae</em> family, 
       <em>noannotn</em> alert). Twelve of the 35 sequences had more than one of the above listed alerts. For the sequences with truncated 
       <em>blastx</em> alignments, we observed that identical or similar truncated alignments were found in VIGOR, but in the current design of VIGOR, truncated nucleotide-to-protein alignments do not trigger an error, at least in these cases.
      </p>
      <p id="p-64">In the DP dataset, for which only VADR was tested, 37 sequences failed. Twenty-eight failed because no similarity to any RefSeq in the reference library was detected (
       <em>noannotn</em> alert). Twenty of these are likely not dengue virus sequences but rather from 
       <em>Aedes aergypti</em> or 
       <em>Aedes albopictus</em>, the mosquito vectors of Dengue virus as they contain one of those species names in the sequence description. Three of the other nine sequences that failed did so because they are reverse complemented (
       <em>revcompl</em> alert). Three others fail because the 
       <em>blastx</em> alignments do not extend far enough relative to the nucleotide-based CDS endpoints (
       <em>indf5pst</em> or 
       <em>indf3pst</em> alerts). One failed because it had an internal stop in the polyprotein CDS (
       <em>cdsstopp</em> alert). One failed because its stem loop feature that is supposed to be found upstream of the CDS lacked similarity to the RefSeq in the first 30 nucleotides (
       <em>lowsim5f</em> alert). The final sequence failed because it was classified as coming from a different species in family 
       <em>Flaviviridae</em>, namely West Nile virus (
       <em>incgroup</em> alert).
      </p>
     </div>
     <div id="sec-25" class="subsection">
      <h4>Definition of false positive and false negative rates for the NC sequences</h4>
      <p id="p-65">For the NC data set, which was tested with all three packages, we attempted to make plausible definitions of false positives (FPs) and false negatives (FNs), so as to compare the three programs by standard measures in statistics. The sequences that are not from the Norovirus taxon are useful tests for quality control, but are implausible submissions, so we exclude the 5 sequences that are not classified as Norovirus in GenBank. Additionally, we exclude the 45 sequences in the NC set that VADR determines are truncated on the 5’ or 3’ end (that either begin 3’ of the start of the nonstructural polyprotein CDS or end 5’ of the end of the VP2 CDS) because VAPiD is not designed to handle incomplete sequences. This leaves 150 of the original 200 sequences in the NC set for defining FP and FN rates. We assert that the 129 sequences that pass all three programs are all positives, which is reasonable for our specific purpose of comparing only these three programs.</p>
      <p id="p-66">We manually examined the remaining 21 sequences and determined that 3 of them have undeniable serious errors and so should be classified as ‘negatives’. FV536857.1, which matches best to a non-norovirus RefSeq in VADR’s model library (NC 006875) and appears to be reverse complemented, is failed by all three programs and so is counted for all three as a true negative (TN). FJ446720.1, which has an early in-frame stop codon 690 nt 5’ of the expected stop site, is appropriately failed by VADR and VIGOR (TN) but passed by VAPiD (FP). KY905331.1 lacks a valid start codon in the nonstructural polyprotein CDS and is failed by VADR and VAPiD (TN) but passed by VIGOR (FP). We cautiously defined the remaining 18 sequences, all of which are failed by VADR but passed by VIGOR and VAPiD, as positives because each sequence is arguably valid but is just too different from the matching RefSeq model to pass VADR. These 18 sequences are TPs for VAPiD and VIGOR and FNs for VADR.</p>
      <p id="p-67">Based on this analysis for the NC results, VADR had a false positive rate of 0 and a false negative rate of 0.12 (18/147). Both VAPiD and VIGOR had a false positive rate of 0.33 (1/3) and a false negative rate of 0. Because one of the design goals of VADR is to prevent problematic sequences from automatic deposition into GenBank, it is more important to minimize the FP rate than the FN rate. While VADR’s FN rate is considerably higher than that of VIGOR and VAPiD, the 18 FN sequences that fail VADR all deviate from the nearest RefSeq in specific ways such that GenBank indexers want to manually examine them. Later, we calculate FP and FN rates for new norovirus and dengue submissions evaluated with VADR which show that the FN rate can be significantly lower in practice.</p>
      <p id="p-68">In summary, to allow for automatic processing of high quality virus sequences, VADR is designed to identify when sequences deviate from the closest RefSeq in various ways and construct informative alerts and output messages about those deviations. VADR catches all the problems that VAPiD and VIGOR catch and numerous other problems. Because of VADR’s usage to check submissions, we can tolerate some false negatives, but the false positive rate should be as close to 0 as possible.</p>
     </div>
     <div id="sec-26" class="subsection">
      <h4>Relevance of reference database size</h4>
      <p id="p-69">The three programs differ in the size of the reference databases they use to map annotation onto input sequences. VADR’s reference library includes nine norovirus RefSeqs and four dengue virus RefSeqs. VAPiD’s reference library contains more than 800 norovirus and more than 4000 dengue virus sequences. VIGOR’s library contains more than 200 total norovirus proteins (polyprotein, VP1 and VP2) and more than 50 norovirus mature peptides. Because of VADR’s role in controlling which sequences are automatically entered into GenBank and how those sequences are annotated, having trustworthy and consistent annotations in the reference library is crucial. We chose RefSeqs as the basis for our library for this reason, after reviewing, updating and in some cases creating new RefSeq records with the intention that they would serve as references to the wider research community beyond the scope of VADR-based annotation.</p>
      <p id="p-70">Larger libraries are more likely to contain mistakes or anomalies that should not be mapped onto incoming sequences that may be automaticlly deposited in GenBank. An example is the sequence FJ446720.1 from the NC dataset, which is in the VAPiD reference library but lacks annotation of the VP1 CDS. Both VADR and VIGOR failed this sequence due to a premature stop in the putative VP1 CDS, but it passed VAPiD because its reference sequence (itself) lacks VP1 annotation and so the VP1 region is not examined for premature stops. This sequence may be biologically valid, but should be reviewed by GenBank indexers as opposed to being automatically deposited into GenBank.</p>
      <p id="p-71">On the other hand, a larger reference database can contain more diversity than a smaller one, and the most common VADR failure in the NC dataset is due to an early stop codon in the nonstructural polyprotein CDS by three nucleotides (11 of the 16 
       <em>cdsstopn</em> alerts in the set of 35 sequences mentioned above). This failure may have been avoided with a larger reference database that included a norovirus sequence with this three nucleotide shorter CDS variant (an example is AB933745.1). We plan to add to VADR’s reference library as we find areas of sequence space that it does not adequately cover; for example, while the manuscript was out for review, 10 norovirus RefSeqs were added for internal testing. We also plan to extend VADR to use profiles built from multiple alignments instead of single sequences which should enhance its ability to analyze and annotate some sequences that are divergent from available RefSeqs.
      </p>
     </div>
     <div id="sec-27" class="subsection">
      <h4>Agreement on passing sequences</h4>
      <p id="p-72">The above discussion focuses on failing sequences, and does not address the validity of passing sequences or the correctness of their annotation. In our testing of VADR since May 2018, indexers have manually reviewed the sequences that pass VADR and have largely agreed that they should indeed pass. Where they have disagreed, we have modified VADR during its development to fail the sequences in question. Additionally, in our testing of the NP and NC datasets, when VADR and VIGOR both pass a sequence they nearly always have the same annotations: 703 of the 705 (99.7%) of the CDS annotations with consistent product names (VP1, VP2 or nonstructural polyprotein) had identical coordinates in the VADR and VIGOR output.</p>
     </div>
    </div>
    <div id="sec-28" class="subsection">
     <h3>Sequences already evaluated with VADR</h3>
     <p id="p-73">To date, VADR has been used to evaluate hundreds of norovirus and dengue virus sequences submitted to GenBank. During intermediate phases of testing, we recorded in detail the fate of every submitted sequence into six categories. Among 922 norovirus sequences submitted in 2019 during this phase, 898 (97%) were accepted automatically, 8 were accepted by an indexer without changes, 8 were accepted by an indexer after making changes, and 8 were sent back to the submitter. Among 2644 dengue virus sequences submitted from the beginning of 2019 until intermediate testing stopped on February 14, 2020, 2520 (95%) were accepted automatically, 15 were accepted by an indexer without changes, 96 were accepted by an indexer after making changes, 9 were sent back to the submitter, and 4 had a more complicated fate. After VADR was put into full production usage for norovirus, the record-keeping was reduced to record only how many sequences are evaluated by VADR and how many sequences pass or fail. Among the first 3143 norovirus sequences evaluated by VADR in production mode, 2809 (89%) passed while 334 (11%) failed. If these data are representative of norovirus and dengue virus submissions, the expected reduction in the number of sequences that need to be manually reviewed by NCBI indexers (which prior to development of VADR was 100%) will be about 10-fold for norovirus submissions and about 20-fold for dengue virus submissions.</p>
     <p id="p-74">In the context of the new submissions, there is virtually no risk that a sequence of the wrong taxon would be submitted via the new GenBank submission portal interfaces for norovirus submissions and dengue virus submission. Therefore, we define that a sequences is a 
      <em>false positive</em> if it passes VADR, but a GenBank indexer deems using other tools that the sequence has an error. Similarly, a sequence is a 
      <em>false negative</em> if it fails VADR, but a GenBank indexer deems that the sequence should pass with no errors. Using these definitions and the counts in the previous paragraph, VADR’s false positive rates are 0 for both norovirus and dengue virus. The false negative rates are 0.009 (8/906) for norovirus and 0.006 (15/2535) for dengue virus.
     </p>
    </div>
    <div id="sec-29" class="subsection">
     <h3>VADR-based annotation of mature peptides and structural RNA features</h3>
     <p id="p-75">The central problems in checking and annotating viral genomes and bacterial genomes differ [
      <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>]. Bacterial genomes are typically larger with hundreds of genes, many of which may be uncharacterized. Hence, assembly and gene prediction are two of the difficult problems in bacterial annotation. Virus genomes are typically smaller with only a handful of genes, but the biological phenomena of cleavage of proteins into mature peptides [
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] and ribosomal slippage [
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, 
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, 
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>] are prominent. Surprisingly, while 97% (31,086 of 32,190) and 85% (17,900 of 20,973) of non-RefSeq norovirus and dengue virus sequences, respectively, last modified between January 1, 1989 and December 31, 2017, had at least one CDS annotated, only 1.4% (443 of 31,086) and 6.4% (1,157 of 17,900) of those those had any mature peptides annotated, even though mature peptide cleavage occurs in all viruses of the corresponding genera. VADR fixes this omission; all norovirus and dengue virus sequences in GenBank annotated with VADR have mature peptides predicted, where possible.
     </p>
     <p id="p-76">As noted above, VAPiD and VIGOR do not attempt to annotate features other than coding sequences: both annotate CDS, and VIGOR annotates mat peptides. VADR has the added capability of annotating any sequence feature that is also annotated in the RefSeq, including conserved structural RNA elements which, though present in many viral genomes [
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, 
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>], are typically not annotated in GenBank. We added annotation of an ncRNA feature, a subgenomic flavivirus RNA (sfRNA), and associated stem loop features, to the four dengue virus RefSeqs in preparation of VADR use for dengue sequence submissions. These structural RNA elements are relevant in pathogenicity and evasion of the host immune system in at least some flaviviruses [
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, 
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, 
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. Incoming dengue virus sequence submissions will now include these RNA annotations because of VADR, which employs covariance models of both the conserved sequence and secondary structure of the RNA elements. In the set of 4171 sequences that passed VADR in the full DC dataset of 4580 sequences, VADR annotated between 4 and 9 stem loop features and exactly 1 ncRNA feature in each sequence for a total of 35,676 stem loop features and 4171 ncRNA features. In the set of 17,276 sequences that passed VADR in the full 20,973 sequence DP dataset, VADR annotated between 1 and 6 stem loop features in 2335 sequences, and exactly one ncRNA feature in 623 of those 2327 sequences for a total of 5157 stem loop features and 623 ncRNA features.
     </p>
     <p id="p-77">VADR can also annotate CDS that involve ribosomal slippage and that capability will become evident when it begins to be used for Hepacivirus, West Nile virus, or ebolaviruses. VADR’s full sequence alignment-based annotation strategy enables annotation of any feature that has nucleotide positional information in the RefSeq annotation. This includes discontiguous features, including multi-segment genes and features as short as a single nucleotide.</p>
    </div>
    <div id="sec-30" class="subsection">
     <h3>Limitations and future directions</h3>
     <p id="p-78">One obvious limitation of VADR at present is that it is used at NCBI only for norovirus and dengue virus sequences, but not yet for other viruses. Near-term plans include to expand the usage to many other members of the family 
      <em>Flaviviridae</em>, including Hepacivirus C, West Nile virus and Zika virus. VADR was not designed for viruses with genomes larger than 25Kb such as eukaryotic DNA viruses such as Epstein-Barr virus (~170Kb). For these larger viruses, VADR would require an impractical amount of memory and time due to the high complexity of the CM algorithms. In future work, we hope to develop new methods to alleviate this restriction. Due to the COVID-19 outbreak beginning in December 2019, we have made 55 
      <em>Coronaviridae</em> models available for download and analysis of coronavirus sequences including SARS-CoV-2, even though VADR was not designed for sequences as large as coronavirus genomes (~30Kb). It is recommended that at least 64Gb of RAM are available when analyzing full length coronavirus genomes. Download and usage instructions for these models can be found on github (
      <a href="https://github.com/nawrockie/vadr/wiki/Available-VADR-model-files">https://github.com/nawrockie/vadr/wiki/Available-VADR-model-files</a>).
     </p>
     <p id="p-79">Users may construct VADR models for any viral species represented by a RefSeq or other GenBank sequence up to 25Kb, but it is important to make sure that all the features for the sequence are properly annotated with the current nomenclature. Users may write to 
      <span class="em-link">
       <span class="em-addr">info{at}ncbi.nlm.nih.gov</span>
      </span> either to inquire as to which viral RefSeqs have been vetted to the standards needed in VADR and required by GenBank indexers, or to suggest corrections or request updates to RefSeq annotations.
     </p>
     <p id="p-80">Another limitation of the current usage of VADR is that each model is built from a single RefSeq sequence. Valid sequences that are sufficiently dissimilar from all RefSeq-based models will fail because alerts designed to identify dissimilar sequences will be reported. The failures of some sequences from the test datasets that pass VAPiD and/or VIGOR can be attributed to this, as discussed in the Results section. During testing with incoming norovirus sequence submissions, three additional RefSeqs, included in the nine norovirus RefSeqs used for the tests described herein, were created to address this issue. Stimulated by some recent norovirus submissions with sequences dissimilar from the nine RefSeqs in the VADR 1.0 library and by a newly published norovirus genotyping paradigm [
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>] that suggested additional candidate RefSeqs more similar to the new sequences, we selected and curated 11 additional norovirus RefSeqs to reach a total of 20. Ten of the additional 11 RefSeqs are being tested on norovirus submissions since early 2020. VADR can be run on such sequences with 19 or 20 models for norovirus instead of nine models. Whichever ones of the additional 11 norovirus RefSeqs prove useful in internal testing will have their models included in a future released version of VADR and be used in automated processing of submissions.
     </p>
     <p id="p-81">An alternative strategy to building models from a single RefSeq is to create and use profile models (profile HMMs and CMs) from trusted sequence alignments of multiple representative sequences that cover the known diversity of the virus species or subspecies being modelled. Profile-based methods are more sensitive at homology detection [
      <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>, 
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>, 
      <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>, 
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>] then single sequence based methods and so this strategy may improve performance. Extending VADR to profiles was envisioned from its design inception and the 
      <em>cmbuild</em> program which creates VADR’s CM files can take as input a multiple alignment. All sequences in the alignment the profile is constructed from should include the same set of features with start and end points aligned, which will limit the phylogenetic breadth of some alignments. For example, among noroviruses, eight of the nine RefSeqs encode three proteins, but murine norovirus (represented by NC 08311) encodes four proteins. A similar problem arises among different subtypes of West Nile virus that may or may not encode WARF4 [
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>].
     </p>
     <p id="p-82">VADR is also flexible as to what types of sequences it models, and it is not restricted to viruses. Any conserved sequence feature, such as a structural RNA gene or region, or part or all of a viroid, can be modelled by VADR. The accuracy of VADR annotations for any sequence depends on how similar it is to the reference sequence(s) from which the model was built.</p>
     <p id="p-83">Although VADR uses Infernal, a program capable of modelling RNA secondary structure, its models will not include structure information unless the user provides it as input to 
      <em>v-build.pl</em>. For models without secondary structure it could be reasonably argued that VADR should use profile HMMs instead of CMs for the alignment stage, because HMMs are more efficient to compute with than CMs and in the absence of secondary structure are essentially equivalent models. However, we opted to always use CMs for the alignment stage because the popular HMM package HM-MER3 is not well-suited to accurately defining alignment endpoints, as it is restricted to local alignment only, whereas Infernal allows global alignment with respect to the sequence, and consequently can identify at least some boundaries more accurately.
     </p>
     <p id="p-84">For some potential users, the requirement to use VADR on either Linux or Mac OS X via the command line is a limitation. Therefore, planning is underway to develop a Dockerized version of VADR that could be used via cloud computing.</p>
     <p id="p-85">The relatively slow speed of VADR is also a limitation. Running as a single execution threads on a 2.93 GHz Intel Xeon processor, the 
      <em>v-build.pl</em> script takes about 15 minutes for a typical calicivirus genome, and about 25 minutes for a typical flavivirus genome, but this step needs to only be run once per RefSeq. The 
      <em>v-annotate.pl</em> script takes about 1.5 seconds or 30 seconds for partial or full-length Norovirus sequences, respectively, and about 9 seconds or 90 seconds for partial or full-length Dengue sequences, respectively. These speeds are about 8 times (NC dataset) or 20 times (DC dataset) slower than VAPiD, and 6 times (NC dataset) or 1.5 times (NP dataset) slower than VIGOR. The two most expensive stages of 
      <em>v-annotate.pl</em> are the classification and alignment stages. To speed up sequence processing, the 
      <em>v-annotate.pl</em> script can be run in parallel without impacting the results, by partitioning the input sequence file, running each partition separately in parallel, and then concatenating the output files. Additionally, the 
      <em>-p</em> option allows users with access to a compute cluster to do this partitioning and parallelization automatically.
     </p>
    </div>
   </div>
   <div class="section" id="sec-31">
    <h2 class="">Conclusions</h2>
    <p id="p-86">VADR is NCBI’s new production software system for checking and annotating non-influenza virus sequence submissions to GenBank. VADR handles both whole genomes and partial genomes. From January 1, 2019 through October 8, 2019, VADR was used to check and annotate 4065 norovirus sequences and 1702 dengue virus sequences. Norovirus and dengue virus researchers can scrutinize the newly annotated sequences and install and use VADR to check sequences that they generate in the future before submitting them to GenBank. VADR implements rigorously documented operational semantics for characterizing sequence problems and other unexpected characteristics, which should simplify the interaction between sequence submitters and GenBank indexers when submitted sequences fail. VADR will be gradually adapted to include more of the viruses in 
     <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and their relatives starting with other members of the family 
     <em>Flaviviridae</em>.
    </p>
   </div>
   <div class="section" id="sec-33">
    <h2 class="">Availability and requirements</h2>
    <p id="p-87">Project name: VADR</p>
    <p id="p-88">Project home page: 
     <a href="https://github.com/nawrockie/vadr">https://github.com/nawrockie/vadr</a>
    </p>
    <p id="p-89">Operating system(s): Linux, Mac OS X</p>
    <p id="p-90">Programming language: Perl</p>
    <p id="p-91">Other requirements: Bio-Easel v0.09, BLAST+ v2.9.0, Infernal v1.1.3, Sequip v0.03 (see 
     <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>)
    </p>
    <p id="p-92">License: public domain</p>
    <p id="p-93">Any restrictions to use by non-academics: none</p>
   </div>
   <div class="section" id="sec-34">
    <h2 class="">Ethics approval and consent to participate</h2>
    <p id="p-107">Not applicable.</p>
   </div>
   <div class="section" id="sec-35">
    <h2 class="">Consent for publication</h2>
    <p id="p-108">Not applicable.</p>
   </div>
   <div class="section" id="sec-36">
    <h2 class="">Availability of data and materials</h2>
    <p id="p-109">All data generated or analyzed during this study are included in this published article, its supplementary material, or NCBI’s GenBank database. Code is available on github (
     <a href="https://github.com/nawrockie/vadr">https://github.com/nawrockie/vadr</a>). The supplementary material includes instructions for reproducing the comparisons of VADR, VAPiD and VIGOR reported in the article.
    </p>
   </div>
   <div class="section COI-statement" id="sec-37">
    <h2 class="">Competing interests</h2>
    <p id="p-110">The authors declare that they have no competing interests.</p>
   </div>
   <div class="section" id="sec-38">
    <h2 class="">Funding</h2>
    <p id="p-111">This research was supported by the Intramural Research of the National Institutes of Health, National Library of Medicine (NLM) and National Cancer Institute.</p>
   </div>
   <div class="section" id="sec-39">
    <h2 class="">Author’s contributions</h2>
    <p id="p-112">AAS wrote the protein validation module and EPN wrote the remainder of the code. ELH updated RefSeq annotation and developed test cases. LY tested the software on sequence submissions and developed test cases. LS wrote an early version of the classification module. JRB and IK-M helped develop the initial specifications for VADR and managed the work of ELH and LY, respectively. AAS and EPN conceived of and designed the software and wrote the paper. All authors helped in the development of the software and read and approved the final manuscript.</p>
   </div>
   <div class="section" id="sec-40">
    <h2 class="">Additional Files</h2>
    <p id="p-114">Additional file 1 — vadr-1.0-paper-supplementary-material.tar.gz</p>
    <p id="p-115">Sequence files, scripts, and instructions for reproducing Norovirus and Dengue virus tests reported in the article. This is a gzipped tarball that can be unpacked with the command ‘tar xf vadr-1.0-paper-supplementary-material.tar.gz’, and includes two README files with instructions on reproducing the tests and reported results.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-113">We thank Estéban Finol and Mariano Garcia-Blanco for providing predicted RNA structures for the Dengue virus RefSeqs, Alex Kotliarov for incorporating VADR into the GenBank submission portal system, Ron Patterson and his team for managing NCBI’s computing resources, and Vince Calhoun for testing VADR and helpful feedback on the manuscript. This research was supported by the Intramural Research Program of the NIH, National Library of Medicine and National Cancer Institute.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-1">
      <p id="p-5">Address reviewers comments; Update statistics of VADR usage on Dengue sequence submissions; Add information on newly available VADR coronavirus models;</p>
     </li>
     <li class="fn-dataset fn-group-external-links" id="fn-2">
      <p id="p-6">
       <a href="https://github.com/nawrockie/vadr">https://github.com/nawrockie/vadr</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section glossary">
    <h2 class="">Abbreviations</h2>
    <dl id="def-list-1">
     <dt id="def-item-1">NCBI
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>National Center for Biotechnology Information
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-2">VADR
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Viral Annotation DefineR
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-3">BLAST
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Basic Local Alignment Search Tool
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-4">CM
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Covariance Model
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-5">HMM
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Hidden Markov Model
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-6">nt
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>nucleotides
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-7">Kb
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>kilobase (1000 nucleotides)
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-8">ncRNA
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>noncoding RNA
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-9">mat_peptide
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>mature peptide
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-10">NP
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Norovirus Partial dataset
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-11">NC
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Norovirus Complete dataset
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-12">DP
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Dengue virus Partial dataset
      <span class="def-item-dd-sep"/>
     </dd>
     <dt id="def-item-13">DC
      <span class="def-item-dt-sep"/>
     </dt>
     <dd>Dengue virus Complete dataset</dd>
    </dl>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.1" data-doi="10.1093/nar/gky989">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sayers</span>
          <span class="cit-name-given-names">EW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cavanaugh</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Clark</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ostell</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pruitt</span>
          <span class="cit-name-given-names">KD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Karsch-Mizrachi</span>
          <span class="cit-name-given-names">I</span>
         </span>. 
         <span class="cit-article-title">GenBank</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">47</span>:
         <span class="cit-fpage">D94</span>–
         <span class="cit-lpage">D99</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DD94%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky989&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.2" data-doi="10.1093/nar/gkm354">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bao</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bolotov</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dernovoy</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kiryutin</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tatusova</span>
          <span class="cit-name-given-names">T</span>
         </span>. 
         <span class="cit-article-title">FLAN: a web Server for Influenza Virus Genome Annotation</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2007</span>;
         <span class="cit-vol">35</span>:
         <span class="cit-fpage">W280</span>–
         <span class="cit-lpage">W284</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm354%26rft_id%253Dinfo%253Apmid%252F17545199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkm354&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17545199&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000255311500052&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.3" data-doi="10.1093/nar/gkw1065">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hatcher</span>
          <span class="cit-name-given-names">EL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhdanov</span>
          <span class="cit-name-given-names">SA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bao</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blinkova</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nawrocki</span>
          <span class="cit-name-given-names">EP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ostapchuck</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Virus Variation Resource - improved response to emergent viral outbreaks</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">45</span>:
         <span class="cit-fpage">D482</span>–
         <span class="cit-lpage">D490</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1065%26rft_id%253Dinfo%253Apmid%252F27899678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkw1065&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27899678&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waterman</span>
          <span class="cit-name-given-names">MS</span>
         </span>. 
         <span class="cit-article-title">Genomic sequence databases</span>. 
         <abbr class="cit-jnl-abbrev">Genomics</abbr>. 
         <span class="cit-pub-date">1990</span>;
         <span class="cit-vol">6</span>:
         <span class="cit-fpage">700</span>–
         <span class="cit-lpage">701</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DWaterman%26rft.auinit1%253DM.%2BS.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D700%26rft.epage%253D701%26rft.atitle%253DGenomic%2Bsequence%2Bdatabases.%26rft_id%253Dinfo%253Apmid%252F2341159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=2341159&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.5" data-doi="10.1086/658657">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Strasser</span>
          <span class="cit-name-given-names">BJ</span>
         </span>. 
         <span class="cit-article-title">The Experimenter’s Museum: GenBank, Natural History, and the Moral Economies of Biomedicine</span>. 
         <abbr class="cit-jnl-abbrev">Isis</abbr>. 
         <span class="cit-pub-date">2011</span>;
         <span class="cit-vol">102</span>:
         <span class="cit-fpage">60</span>–
         <span class="cit-lpage">96</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIsis%26rft.stitle%253DIsis%26rft.aulast%253DStrasser%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D102%26rft.issue%253D1%26rft.spage%253D60%26rft.epage%253D96%26rft.atitle%253DThe%2Bexperimenter%2527s%2Bmuseum%253A%2BGenBank%252C%2Bnatural%2Bhistory%252C%2Band%2Bthe%2Bmoral%2Beconomies%2Bof%2Bbiomedicine.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F658657%26rft_id%253Dinfo%253Apmid%252F21667776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/658657&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21667776&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000289868600003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.6" data-doi="10.1093/nar/gkw569">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tatusova</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">DiCuccio</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Badretdin</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chetvernin</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nawrocki</span>
          <span class="cit-name-given-names">EP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zaslavsky</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">NCBI Prokaryotic Genome Annotation Pipeline</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">44</span>:
         <span class="cit-fpage">6614</span>–
         <span class="cit-lpage">6624</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw569%26rft_id%253Dinfo%253Apmid%252F27342282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkw569&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27342282&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shean</span>
          <span class="cit-name-given-names">RC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Makhsous</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stoddard</span>
          <span class="cit-name-given-names">GD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">MJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Greninger</span>
          <span class="cit-name-given-names">AJ</span>
         </span>. 
         <span class="cit-article-title">VAPiD: A Lightweight Cross-Platform Viral Annotation Pipeline and Identification Tool to Facilitate Virus Genome Submissions to NCBI GenBank</span>. 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. 
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">20</span>:
         <span class="cit-fpage">48</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D20%26rft.spage%253D48%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.8" data-doi="10.1186/1471-2105-11-451">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sundaram</span>
          <span class="cit-name-given-names">JP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Spiro</span>
          <span class="cit-name-given-names">D</span>
         </span>. 
         <span class="cit-article-title">VIGOR, an Annotation Program for Small Viral Genomes</span>. 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. 
         <span class="cit-pub-date">2010</span>;
         <span class="cit-vol">11</span>:
         <span class="cit-fpage">451</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DWang%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.spage%253D451%26rft.epage%253D451%26rft.atitle%253DVIGOR%252C%2Ban%2Bannotation%2Bprogram%2Bfor%2Bsmall%2Bviral%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-451%26rft_id%253Dinfo%253Apmid%252F20822531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-451&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20822531&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.9" data-doi="10.1093/nar/gks528">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sundaram</span>
          <span class="cit-name-given-names">JP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stockwell</span>
          <span class="cit-name-given-names">TB</span>
         </span>. 
         <span class="cit-article-title">VIGOR Extended to Annotate Genomes for Additional 12 Different Viruses</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2012</span>;
         <span class="cit-vol">40</span>:
         <span class="cit-fpage">W186</span>–
         <span class="cit-lpage">W192</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks528%26rft_id%253Dinfo%253Apmid%252F22669909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gks528&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22669909&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000306670900030&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">KY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">YZ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Du</span>
          <span class="cit-name-given-names">MZ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dong</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">FB</span>
         </span>. 
         <span class="cit-article-title">Vgas: A Viral Genome Annotation System</span>. 
         <abbr class="cit-jnl-abbrev">Front Microbiol.</abbr>
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">10</span>:
         <span class="cit-fpage">184</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront%2BMicrobiol.%26rft.volume%253D10%26rft.spage%253D184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">González-Tortuero</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sean Sutton</span>
          <span class="cit-name-given-names">TD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Velayudhan</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shkoporov</span>
          <span class="cit-name-given-names">AN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Draper</span>
          <span class="cit-name-given-names">LA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stockdale</span>
          <span class="cit-name-given-names">SR</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">VIGA: a sensitive, precise and automatic de novo VIral Genome Annotator</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. 
         <span class="cit-pub-date">2018</span>;.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.12" data-doi="10.1093/nar/gkv1189">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">O’Leary</span>
          <span class="cit-name-given-names">NA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wright</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brister</span>
          <span class="cit-name-given-names">JR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ciufo</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haddad</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McVeigh</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Reference Sequence (RefSeq) Database at NCBI: Current Status, Taxonomic Expansion, and Functional Annotation</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">44</span>:
         <span class="cit-fpage">D733</span>–
         <span class="cit-lpage">D745</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1189%26rft_id%253Dinfo%253Apmid%252F26553804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkv1189&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26553804&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.13" data-doi="10.1016/j.chom.2008.10.007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pijlman</span>
          <span class="cit-name-given-names">GP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Funk</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kondratieva</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Torres</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">van der Aa</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Highly Structured, Nuclease-Resistant, Noncoding RNA Produced by Flaviviruses is Required for Pathogenicity</span>. 
         <abbr class="cit-jnl-abbrev">Cell Host Microbe.</abbr>
         <span class="cit-pub-date">2008</span>;
         <span class="cit-vol">4</span>:
         <span class="cit-fpage">579</span>–
         <span class="cit-lpage">591</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DPijlman%26rft.auinit1%253DG.%2BP.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D579%26rft.epage%253D591%26rft.atitle%253DA%2Bhighly%2Bstructured%252C%2Bnuclease-resistant%252C%2Bnoncoding%2BRNA%2Bproduced%2Bby%2Bflaviviruses%2Bis%2Brequired%2Bfor%2Bpathogenicity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2008.10.007%26rft_id%253Dinfo%253Apmid%252F19064258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.chom.2008.10.007&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19064258&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000262007200010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.14" data-doi="10.1101/gad.259077.115">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tycowski</span>
          <span class="cit-name-given-names">KT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">YE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moss</span>
          <span class="cit-name-given-names">WN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vallery</span>
          <span class="cit-name-given-names">TK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Viral Noncoding RNAs: More Surprises</span>. 
         <abbr class="cit-jnl-abbrev">Genes Dev.</abbr>
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">29</span>:
         <span class="cit-fpage">567</span>–
         <span class="cit-lpage">584</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DTycowski%26rft.auinit1%253DK.%2BT.%26rft.volume%253D29%26rft.issue%253D6%26rft.spage%253D567%26rft.epage%253D584%26rft.atitle%253DViral%2Bnoncoding%2BRNAs%253A%2Bmore%2Bsurprises%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.259077.115%26rft_id%253Dinfo%253Apmid%252F25792595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjkvNi81NjciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yNC84NTI2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.15" data-doi="10.1093/nar/25.17.3389">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Altschul</span>
          <span class="cit-name-given-names">SF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Madden</span>
          <span class="cit-name-given-names">TL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schäffer</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a New Generation of Protein Database Search Programs</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">1997</span>;
         <span class="cit-vol">25</span>:
         <span class="cit-fpage">3389</span>–
         <span class="cit-lpage">3402</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9254694&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1997XU79300002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.16" data-doi="10.1093/nar/gkg609">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wernersson</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pedersen</span>
          <span class="cit-name-given-names">AG</span>
         </span>. 
         <span class="cit-article-title">RevTrans: Multiple Alignment of Coding DNA From Aligned Amino Acid Sequences</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2003</span>;
         <span class="cit-vol">31</span>:
         <span class="cit-fpage">3537</span>–
         <span class="cit-lpage">3539</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWernersson%26rft.auinit1%253DR.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3537%26rft.epage%253D3539%26rft.atitle%253DRevTrans%253A%2Bmultiple%2Balignment%2Bof%2Bcoding%2BDNA%2Bfrom%2Baligned%2Bamino%2Bacid%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg609%26rft_id%253Dinfo%253Apmid%252F12824361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkg609&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12824361&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000183832900053&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.17" data-doi="10.1371/journal.pcbi.1002195">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>
          <span class="cit-name-given-names">SR</span>
         </span>. 
         <span class="cit-article-title">Accelerated Profile HMM Searches</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Comput Biol.</abbr>
         <span class="cit-pub-date">2011</span>;
         <span class="cit-vol">7</span>:
         <span class="cit-fpage">e1002195</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002195%26rft.epage%253De1002195%26rft.atitle%253DAccelerated%2BProfile%2BHMM%2BSearches.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002195%26rft_id%253Dinfo%253Apmid%252F22039361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002195&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22039361&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.18" data-doi="10.1093/bioinformatics/btt403">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wheeler</span>
          <span class="cit-name-given-names">TJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>
          <span class="cit-name-given-names">SR</span>
         </span>. 
         <span class="cit-article-title">nhmmer: DNA Homology Search With Profile HMMs</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. 
         <span class="cit-pub-date">2013</span>;
         <span class="cit-vol">29</span>:
         <span class="cit-fpage">2487</span>–
         <span class="cit-lpage">2489</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt403%26rft_id%253Dinfo%253Apmid%252F23842809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt403&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23842809&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000324778500019&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.19" data-doi="10.1093/bioinformatics/btt509">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nawrocki</span>
          <span class="cit-name-given-names">EP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>
          <span class="cit-name-given-names">SR</span>
         </span>. 
         <span class="cit-article-title">Infernal 1.1: 100-fold Faster RNA Homology Searches</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. 
         <span class="cit-pub-date">2013</span>;
         <span class="cit-vol">29</span>:
         <span class="cit-fpage">2933</span>–
         <span class="cit-lpage">2935</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt509%26rft_id%253Dinfo%253Apmid%252F24008419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt509&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24008419&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000326643600018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.20" data-doi="10.1186/1471-2105-3-18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>
          <span class="cit-name-given-names">SR</span>
         </span>. 
         <span class="cit-article-title">A Memory-Efficient Dynamic Programming Algorithm for Optimal Alignment of a Sequence to an RNA Secondary Structure</span>. 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. 
         <span class="cit-pub-date">2002</span>;
         <span class="cit-vol">3</span>:
         <span class="cit-fpage">18</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D3%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DA%2Bmemory-efficient%2Bdynamic%2Bprogramming%2Balgorithm%2Bfor%2Boptimal%2Balignment%2Bof%2Ba%2Bsequence%2Bto%2Ban%2BRNA%2Bsecondary%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-3-18%26rft_id%253Dinfo%253Apmid%252F12095421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-3-18&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12095421&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-book" id="cit-852657v2.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>
          <span class="cit-name-given-names">SR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Krogh</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mitchison</span>
          <span class="cit-name-given-names">GJ</span>
         </span>. 
         <span class="cit-source">Biological Sequence Analysis: Probabilistic Models of Proteins and Nucleic Acids</span>. 
         <span class="cit-publ-loc">Cambridge UK</span>: 
         <span class="cit-publ-name">Cambridge University Press</span>; 
         <span class="cit-pub-date">1998</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Brown</span>
          <span class="cit-name-given-names">MP</span>
         </span>. 
         <span class="cit-article-title">Small Subunit Ribosomal RNA Modeling Using Stochastic Context-Free Grammars</span>. 
         <abbr class="cit-jnl-abbrev">ISMB</abbr>. 
         <span class="cit-pub-date">2000</span>;
         <span class="cit-vol">8</span>:
         <span class="cit-fpage">57</span>–
         <span class="cit-lpage">66</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DISMB%26rft.volume%253D8%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-thesis" id="cit-852657v2.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nawrocki</span>
          <span class="cit-name-given-names">EP</span>
         </span>. 
         <span class="cit-article-title">Structural RNA Homology Search and Alignment Using Covariance Models</span> [
         <span class="named-content ref-degree" id="named-content-1">Ph.D. thesis</span>]. 
         <span class="cit-publ-name">Washington University School of Medicine</span>; 
         <span class="cit-pub-date">2009</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.24" data-doi="10.1093/nar/gkq901">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hulo</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Castro</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Masson</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bougueleret</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bairoch</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xenarios</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">ViralZone: a Knowledge Resource to Understand Virus Diversity</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2011</span>;
         <span class="cit-vol">39</span>:
         <span class="cit-fpage">D576</span>–
         <span class="cit-lpage">D582</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq901%26rft_id%253Dinfo%253Apmid%252F20947564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkq901&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20947564&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000285831700091&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.25" data-doi="10.1038/331280a0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jacks</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Power</span>
          <span class="cit-name-given-names">MD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Masiarz</span>
          <span class="cit-name-given-names">FR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luciw</span>
          <span class="cit-name-given-names">PA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barr</span>
          <span class="cit-name-given-names">PJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Varmus</span>
          <span class="cit-name-given-names">HE</span>
         </span>. 
         <span class="cit-article-title">Characterization of Ribosomal Frameshifting in HIV-1 gag-pol Expression</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">1988</span>;
         <span class="cit-vol">331</span>:
         <span class="cit-fpage">280</span>–
         <span class="cit-lpage">283</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJacks%26rft.auinit1%253DT.%26rft.volume%253D331%26rft.issue%253D6153%26rft.spage%253D280%26rft.epage%253D283%26rft.atitle%253DCharacterization%2Bof%2Bribosomal%2Bframeshifting%2Bin%2BHIV-1%2Bgag-pol%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F331280a0%26rft_id%253Dinfo%253Apmid%252F2447506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/331280a0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=2447506&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1988L714000068&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.26" data-doi="10.1093/nar/25.10.1943">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kang</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tinoco</span>
          <span class="cit-name-suffix">Jr</span>
          <span class="cit-name-given-names">I</span>
         </span>. 
         <span class="cit-article-title">A Mutant RNA Pseudoknot That Promotes Ribosomal Frameshifting in Mouse Mammary Tumor Virus</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">1997</span>;
         <span class="cit-vol">25</span>:
         <span class="cit-fpage">1943</span>–
         <span class="cit-lpage">1949</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.10.1943%26rft_id%253Dinfo%253Apmid%252F9115361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/25.10.1943&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9115361&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1997WZ93900010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.27" data-doi="10.1016/j.virol.2004.11.038">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baranov</span>
          <span class="cit-name-given-names">PV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Henderson</span>
          <span class="cit-name-given-names">CM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>
          <span class="cit-name-given-names">CB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gesteland</span>
          <span class="cit-name-given-names">RF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Atkins</span>
          <span class="cit-name-given-names">JF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Howard</span>
          <span class="cit-name-given-names">MT</span>
         </span>. 
         <span class="cit-article-title">Programmed Ribosomal Frameshifting in Decoding the SARS-CoV Genome</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>. 
         <span class="cit-pub-date">2005</span>;
         <span class="cit-vol">332</span>:
         <span class="cit-fpage">498</span>–
         <span class="cit-lpage">510</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBaranov%26rft.auinit1%253DP.%2BV.%26rft.volume%253D332%26rft.issue%253D2%26rft.spage%253D498%26rft.epage%253D510%26rft.atitle%253DProgrammed%2Bribosomal%2Bframeshifting%2Bin%2Bdecoding%2Bthe%2BSARS-CoV%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2004.11.038%26rft_id%253Dinfo%253Apmid%252F15680415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virol.2004.11.038&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15680415&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000226918200004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.28" data-doi="10.1099/vir.0.19462-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thurner</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Witwer</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hofacker</span>
          <span class="cit-name-given-names">IL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stadler</span>
          <span class="cit-name-given-names">PF</span>
         </span>. 
         <span class="cit-article-title">Conserved RNA Secondary Structures in Flaviviridae Genomes</span>. 
         <abbr class="cit-jnl-abbrev">J Gen Virol.</abbr>
         <span class="cit-pub-date">2004</span>;
         <span class="cit-vol">85</span>:
         <span class="cit-fpage">1113</span>–
         <span class="cit-lpage">1124</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DThurner%26rft.auinit1%253DC.%26rft.volume%253D85%26rft.issue%253D5%26rft.spage%253D1113%26rft.epage%253D1124%26rft.atitle%253DConserved%2BRNA%2Bsecondary%2Bstructures%2Bin%2BFlaviviridae%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19462-0%26rft_id%253Dinfo%253Apmid%252F15105528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1099/vir.0.19462-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15105528&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000221267100005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Smyth</span>
          <span class="cit-name-given-names">RP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Negroni</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lever</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mak</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kenyon</span>
          <span class="cit-name-given-names">JC</span>
         </span>. 
         <span class="cit-article-title">RNA Structure-A Neglected Puppet Master for the Evolution of Virus and Host Immunity</span>. 
         <abbr class="cit-jnl-abbrev">Front Immunol.</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">9</span>:
         <span class="cit-fpage">2097</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront%2BImmunol.%26rft.volume%253D9%26rft.spage%253D2097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.30" data-doi="10.1371/journal.ppat.1006265">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Filomatori</span>
          <span class="cit-name-given-names">CV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carballeda</span>
          <span class="cit-name-given-names">JM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Villordo</span>
          <span class="cit-name-given-names">SM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aguirre</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pallarés</span>
          <span class="cit-name-given-names">HM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Maestre</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Dengue Virus Genomic Variation Associated With Mosquito Adaptation Defines the Pattern of Viral non-Coding RNAs and Fitness in Human Cells</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr>
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">13</span>:
         <span class="cit-fpage">e1006265</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D13%26rft.spage%253De1006265%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006265&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.31" data-doi="10.1038/s41467-017-02604-y">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">MacFadden</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">O’Donoghue</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Silva</span>
          <span class="cit-name-given-names">PAGC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chapman</span>
          <span class="cit-name-given-names">EG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Olsthoorn</span>
          <span class="cit-name-given-names">RC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sterken</span>
          <span class="cit-name-given-names">MG</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Mechanism and Structural Diversity of Exoribonuclease-Resistant RNA Structures in Flaviviral RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun.</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">9</span>:
         <span class="cit-fpage">119</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun.%26rft.volume%253D9%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-02604-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41467-017-02604-y&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.32" data-doi="10.1099/jgv.0.001318">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chhabra</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Graaf</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Parra</span>
          <span class="cit-name-given-names">GI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">MC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Green</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Martella</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Updated Classification of Norovirus Genogroups and Genotypes</span>. 
         <abbr class="cit-jnl-abbrev">J Gen Virol.</abbr>
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">100</span>:
         <span class="cit-fpage">1393</span>–
         <span class="cit-lpage">1406</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol.%26rft.volume%253D100%26rft.spage%253D1393%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fjgv.0.001318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1099/jgv.0.001318&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.33" data-doi="10.1093/bioinformatics/14.9.755">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>
          <span class="cit-name-given-names">SR</span>
         </span>. 
         <span class="cit-article-title">Profile Hidden Markov Models</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. 
         <span class="cit-pub-date">1998</span>;
         <span class="cit-vol">14</span>:
         <span class="cit-fpage">755</span>–
         <span class="cit-lpage">763</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D755%26rft.epage%253D763%26rft.atitle%253DProfile%2Bhidden%2BMarkov%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F14.9.755%26rft_id%253Dinfo%253Apmid%252F9918945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/14.9.755&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9918945&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000077489900002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.34" data-doi="10.1101/gr.5890907">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Freyhult</span>
          <span class="cit-name-given-names">EK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bollback</span>
          <span class="cit-name-given-names">JP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gardner</span>
          <span class="cit-name-given-names">PP</span>
         </span>. 
         <span class="cit-article-title">Exploring Genomic Dark Matter: A Critical Assessment of the Performance of Homology Search Methods on Noncoding RNA</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res.</abbr>
         <span class="cit-pub-date">2007</span>;
         <span class="cit-vol">17</span>:
         <span class="cit-fpage">117</span>–
         <span class="cit-lpage">125</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5890907%26rft_id%253Dinfo%253Apmid%252F17151342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzEvMTE3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjQvODUyNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.35" data-doi="10.1093/nar/gkm160">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lagesen</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hallin</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rødland</span>
          <span class="cit-name-given-names">EA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Staerfeldt</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rognes</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ussery</span>
          <span class="cit-name-given-names">DW</span>
         </span>. 
         <span class="cit-article-title">RNAmmer: Consistent and Rapid Annotation of Ribosomal RNA Genes</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2007</span>;
         <span class="cit-vol">35</span>:
         <span class="cit-fpage">3100</span>–
         <span class="cit-lpage">3108</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm160%26rft_id%253Dinfo%253Apmid%252F17452365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkm160&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17452365&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000247350800027&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.36" data-doi="10.1007/s12275-011-1213-z">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">JH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yi</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chun</span>
          <span class="cit-name-given-names">J</span>
         </span>. 
         <span class="cit-article-title">rRNASelector: a Computer Program for Selecting Ribosomal RNA Encoding Sequences From Metagenomic and Metatranscriptomic Shotgun Libraries</span>. 
         <abbr class="cit-jnl-abbrev">J Microbiol.</abbr>
         <span class="cit-pub-date">2011</span>;
         <span class="cit-vol">49</span>:
         <span class="cit-fpage">689</span>–
         <span class="cit-lpage">691</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmicrobiology%2B%2528Seoul%252C%2BKorea%2529%26rft.stitle%253DJ%2BMicrobiol%26rft.aulast%253DLee%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D689%26rft.epage%253D691%26rft.atitle%253DrRNASelector%253A%2Ba%2Bcomputer%2Bprogram%2Bfor%2Bselecting%2Bribosomal%2BRNA%2Bencoding%2Bsequences%2Bfrom%2Bmetagenomic%2Band%2Bmetatranscriptomic%2Bshotgun%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12275-011-1213-z%26rft_id%253Dinfo%253Apmid%252F21887657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s12275-011-1213-z&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21887657&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000294443100025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-852657v2.37" data-doi="10.1186/1743-422X-9-283">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Faggioni</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pomponi</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Santis</span>
          <span class="cit-name-given-names">RD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Masuelli</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ciammaruconi</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Monaco</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">West Nile Alternative Open Reading Frame (N-NS4B/WARF4) is Produced in Infected West Nile Virus (WNV) Cells and Induces Humoral Response in WNV Infected Individuals</span>. 
         <abbr class="cit-jnl-abbrev">Virol J.</abbr>
         <span class="cit-pub-date">2012</span>;
         <span class="cit-vol">9</span>:
         <span class="cit-fpage">283</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DFaggioni%26rft.auinit1%253DG.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D283%26rft.epage%253D283%26rft.atitle%253DWest%2BNile%2Balternative%2Bopen%2Breading%2Bframe%2B%2528N-NS4B%252FWARF4%2529%2Bis%2Bproduced%2Bin%2Binfected%2BWest%2BNile%2BVirus%2B%2528WNV%2529%2Bcells%2Band%2Binduces%2Bhumoral%2Bresponse%2Bin%2BWNV%2Binfected%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-9-283%26rft_id%253Dinfo%253Apmid%252F23173701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1743-422X-9-283&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23173701&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F852657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
