<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/28/2020.03.27.012013.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.27.012013v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="SARS-CoV-2 and ORF3a: Non-Synonymous Mutations and Polyproline Regions"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.27.012013"/>
  <meta name="DC.Date" content="2020-03-28"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The effect of the rapid accumulation of non-synonymous mutations on the pathogenesis of SARS-CoV-2 is not yet known. To predict the impact of non-synonymous mutations and polyproline regions identified in ORF3a on the formation of B-cell epitopes and their role in evading the immune response, nucleotide and protein sequences of 537 available SARS-CoV-2 genomes were analyzed for the presence of non-synonymous mutations and polyproline regions. Mutations were correlated with changes in epitope formation. A total of 19 different non-synonymous amino acids substitutions were detected in ORF3a among 537 SARS-CoV-2 strains. G251V was the most common and identified in 9.9% (n=53) of the strains and was predicted to lead to the loss of a B-cell like epitope in ORF3a. Polyproline regions were detected in two strains (EPI\_ISL\_410486, France and EPI\_ISL\_407079, Finland) and affected epitopes formation. The accumulation of non-synonymous mutations and detected polyproline regions in ORF3a of SARS-CoV-2 could be driving the evasion of the host immune response thus favoring viral spread. Rapid mutations accumulating in ORF3a should be closely monitored throughout the COVID-19 pandemic. Importance At the surge of the COVID-19 pandemic and after three months of the identification of SARS-CoV-2 as the disease-causing pathogen, nucleic acid changes due to host-pathogen interactions are insightful into the evolution of this virus. In this paper, we have identified a set of non-synonymous mutations in ORF3a and predicted their impact on B-cell like epitope formation. The accumulation of non-synonymous mutations in ORF3a could be driving protein changes that mediate immune evasion and favoring viral spread."/>
  <meta name="DC.Contributor" content="Elio Issa"/>
  <meta name="DC.Contributor" content="Georgi Merhi"/>
  <meta name="DC.Contributor" content="Balig Panossian"/>
  <meta name="DC.Contributor" content="Tamara Salloum"/>
  <meta name="DC.Contributor" content="Sima Tokajian"/>
  <meta name="article:published_time" content="2020-03-28"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="SARS-CoV-2 and ORF3a: Non-Synonymous Mutations and Polyproline Regions"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The effect of the rapid accumulation of non-synonymous mutations on the pathogenesis of SARS-CoV-2 is not yet known. To predict the impact of non-synonymous mutations and polyproline regions identified in ORF3a on the formation of B-cell epitopes and their role in evading the immune response, nucleotide and protein sequences of 537 available SARS-CoV-2 genomes were analyzed for the presence of non-synonymous mutations and polyproline regions. Mutations were correlated with changes in epitope formation. A total of 19 different non-synonymous amino acids substitutions were detected in ORF3a among 537 SARS-CoV-2 strains. G251V was the most common and identified in 9.9% (n=53) of the strains and was predicted to lead to the loss of a B-cell like epitope in ORF3a. Polyproline regions were detected in two strains (EPI_ISL_410486, France and EPI_ISL_407079, Finland) and affected epitopes formation. The accumulation of non-synonymous mutations and detected polyproline regions in ORF3a of SARS-CoV-2 could be driving the evasion of the host immune response thus favoring viral spread. Rapid mutations accumulating in ORF3a should be closely monitored throughout the COVID-19 pandemic.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.27.012013v1"/>
  <meta name="citation_id" content="2020.03.27.012013v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.012013v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.012013v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.012013v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.27.012013"/>
  <meta name="citation_num_pages" content="15"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.27.012013"/>
  <meta name="citation_author" content="Elio Issa"/>
  <meta name="citation_author_institution" content="Department of Natural Sciences, School of Arts and Sciences, Lebanese American University"/>
  <meta name="citation_author" content="Georgi Merhi"/>
  <meta name="citation_author_institution" content="Department of Natural Sciences, School of Arts and Sciences, Lebanese American University"/>
  <meta name="citation_author" content="Balig Panossian"/>
  <meta name="citation_author_institution" content="Department of Natural Sciences, School of Arts and Sciences, Lebanese American University"/>
  <meta name="citation_author" content="Tamara Salloum"/>
  <meta name="citation_author_institution" content="Department of Natural Sciences, School of Arts and Sciences, Lebanese American University"/>
  <meta name="citation_author" content="Sima Tokajian"/>
  <meta name="citation_author_institution" content="Department of Natural Sciences, School of Arts and Sciences, Lebanese American University"/>
  <meta name="citation_author_email" content="stokjian@lau.edu.lb"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3653-8940"/>
  <meta name="citation_reference" content="Zhou P, Yang X-L, Wang X-G, Hu B, Zhang L, Zhang W, Si H-R, Zhu Y, Li B, Huang C-L, Chen H-D, Chen J, Luo Y, Guo H, Jiang R-D, Liu M-Q, Chen Y, Shen X-R, Wang X, Zheng X-S, Zhao K, Chen Q-J, Deng F, Liu L-L, Yan B, Zhan F-X, Wang Y-Y, Xiao G-F, Shi Z-L. 2020. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 1–4."/>
  <meta name="citation_reference" content="Gralinski LE, Menachery VD. 2020. Return of the Coronavirus: 2019-nCoV. 2. Viruses 12:135."/>
  <meta name="citation_reference" content="Andersen KG, Rambaut A, Lipkin WI, Holmes EC, Garry RF. 2020. The proximal origin of SARS-CoV-2. Nat Med 1–3."/>
  <meta name="citation_reference" content="Seemann T. 2014. Prokka: rapid prokaryotic genome annotation. Bioinforma Oxf Engl 30:2068–2069."/>
  <meta name="citation_reference" content="Ding W, Baumdicker F, Neher RA. 2018. panX: pan-genome analysis and exploration. Nucleic Acids Res 46:e5."/>
  <meta name="citation_reference" content="Edgar RC. 2004. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res 32:1792–1797."/>
  <meta name="citation_reference" content="Choi Y, Chan AP. 2015. PROVEAN web server: a tool to predict the functional effect of amino acid substitutions and indels. Bioinformatics 31:2745–2747."/>
  <meta name="citation_reference" content="Artimo P, Jonnalagedda M, Arnold K, Baratin D, Csardi G, de Castro E, Duvaud S, Flegel V, Fortier A, Gasteiger E, Grosdidier A, Hernandez C, Ioannidis V, Kuznetsov D, Liechti R, Moretti S, Mostaguir K, Redaschi N, Rossier G, Xenarios I, Stockinger H. 2012. ExPASy: SIB bioinformatics resource portal. Nucleic Acids Res 40:W597–W603."/>
  <meta name="citation_reference" content="Song J, Li F, Leier A, Marquez-Lago TT, Akutsu T, Haffari G, Chou K-C, Webb GI, Pike RN. 2018. PROSPERous: high-throughput prediction of substrate cleavage sites for 90 proteases with improved accuracy. Bioinformatics 34:684–687."/>
  <meta name="citation_reference" content="Zhang Q, Wang P, Kim Y, Haste-Andersen P, Beaver J, Bourne PE, Bui H-H, Buus S, Frankild S, Greenbaum J, Lund O, Lundegaard C, Nielsen M, Ponomarenko J, Sette A, Zhu Z, Peters B. 2008. Immune epitope database analysis resource (IEDB-AR). Nucleic Acids Res 36:W513–518."/>
  <meta name="citation_reference" content="Minakshi R, Padhan K, Rani M, Khan N, Ahmad F, Jameel S. 2009. The SARS Coronavirus 3a protein causes endoplasmic reticulum stress and induces ligand-independent downregulation of the type 1 interferon receptor. PloS One 4:e8342."/>
  <meta name="citation_reference" content="Park DJ, Dudas G, Wohl S, Goba A, Whitmer SLM, Andersen KG, Sealfon RS, Ladner JT, Kugelman JR, Matranga CB, Winnicki SM, Qu J, Gire SK, Gladden-Young A, Jalloh S, Nosamiefan D, Yozwiak NL, Moses LM, Jiang P-P, Lin AE, Schaffner SF, Bird B, Towner J, Mamoh M, Gbakie M, Kanneh L, Kargbo D, Massally JLB, Kamara FK, Konuwa E, Sellu J, Jalloh AA, Mustapha I, Foday M, Yillah M, Erickson BR, Sealy T, Blau D, Paddock C, Brault A, Amman B, Basile J, Bearden S, Belser J, Bergeron E, Campbell S, Chakrabarti A, Dodd K, Flint M, Gibbons A, Goodman C, Klena J, McMullan L, Morgan L, Russell B, Salzer J, Sanchez A, Wang D, Jungreis I, Tomkins-Tinch C, Kislyuk A, Lin MF, Chapman S, MacInnis B, Matthews A, Bochicchio J, Hensley LE, Kuhn JH, Nusbaum C, Schieffelin JS, Birren BW, Forget M, Nichol ST, Palacios GF, Ndiaye D, Happi C, Gevao SM, Vandi MA, Kargbo B, Holmes EC, Bedford T, Gnirke A, Ströher U, Rambaut A, Garry RF, Sabeti PC. 2015. Ebola Virus Epidemiology, Transmission, and Evolution during Seven Months in Sierra Leone. Cell 161:1516–1526."/>
  <meta name="citation_reference" content="Lhomme S, Abravanel F, Dubois M, Sandres-Saune K, Mansuy J-M, Rostaing L, Kamar N, Izopet J. 2014. Characterization of the polyproline region of the hepatitis E virus in immunocompromised patients. J Virol 88:12017–12025."/>
  <meta name="citation_reference" content="Wessels E, Duijsings D, Notebaart RA, Melchers WJG, Kuppeveld FJM van. 2005. A Proline-Rich Region in the Coxsackievirus 3A Protein Is Required for the Protein To Inhibit Endoplasmic Reticulum-to-Golgi Transport. J Virol 79:5163–5173."/>
  <meta name="citation_reference" content="Purdy MA, Lara J, Khudyakov YE. 2012. The Hepatitis E Virus Polyproline Region Is Involved in Viral Adaptation. PLOS ONE 7:e35974."/>
  <meta name="twitter:title" content="SARS-CoV-2 and ORF3a: Non-Synonymous Mutations and Polyproline Regions"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The effect of the rapid accumulation of non-synonymous mutations on the pathogenesis of SARS-CoV-2 is not yet known. To predict the impact of non-synonymous mutations and polyproline regions identified in ORF3a on the formation of B-cell epitopes and their role in evading the immune response, nucleotide and protein sequences of 537 available SARS-CoV-2 genomes were analyzed for the presence of non-synonymous mutations and polyproline regions. Mutations were correlated with changes in epitope formation. A total of 19 different non-synonymous amino acids substitutions were detected in ORF3a among 537 SARS-CoV-2 strains. G251V was the most common and identified in 9.9% (n=53) of the strains and was predicted to lead to the loss of a B-cell like epitope in ORF3a. Polyproline regions were detected in two strains (EPI\_ISL\_410486, France and EPI\_ISL\_407079, Finland) and affected epitopes formation. The accumulation of non-synonymous mutations and detected polyproline regions in ORF3a of SARS-CoV-2 could be driving the evasion of the host immune response thus favoring viral spread. Rapid mutations accumulating in ORF3a should be closely monitored throughout the COVID-19 pandemic. Importance At the surge of the COVID-19 pandemic and after three months of the identification of SARS-CoV-2 as the disease-causing pathogen, nucleic acid changes due to host-pathogen interactions are insightful into the evolution of this virus. In this paper, we have identified a set of non-synonymous mutations in ORF3a and predicted their impact on B-cell like epitope formation. The accumulation of non-synonymous mutations in ORF3a could be driving protein changes that mediate immune evasion and favoring viral spread."/>
  <meta name="og-title" property="og:title" content="SARS-CoV-2 and ORF3a: Non-Synonymous Mutations and Polyproline Regions"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.27.012013v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The effect of the rapid accumulation of non-synonymous mutations on the pathogenesis of SARS-CoV-2 is not yet known. To predict the impact of non-synonymous mutations and polyproline regions identified in ORF3a on the formation of B-cell epitopes and their role in evading the immune response, nucleotide and protein sequences of 537 available SARS-CoV-2 genomes were analyzed for the presence of non-synonymous mutations and polyproline regions. Mutations were correlated with changes in epitope formation. A total of 19 different non-synonymous amino acids substitutions were detected in ORF3a among 537 SARS-CoV-2 strains. G251V was the most common and identified in 9.9% (n=53) of the strains and was predicted to lead to the loss of a B-cell like epitope in ORF3a. Polyproline regions were detected in two strains (EPI\_ISL\_410486, France and EPI\_ISL\_407079, Finland) and affected epitopes formation. The accumulation of non-synonymous mutations and detected polyproline regions in ORF3a of SARS-CoV-2 could be driving the evasion of the host immune response thus favoring viral spread. Rapid mutations accumulating in ORF3a should be closely monitored throughout the COVID-19 pandemic. Importance At the surge of the COVID-19 pandemic and after three months of the identification of SARS-CoV-2 as the disease-causing pathogen, nucleic acid changes due to host-pathogen interactions are insightful into the evolution of this virus. In this paper, we have identified a set of non-synonymous mutations in ORF3a and predicted their impact on B-cell like epitope formation. The accumulation of non-synonymous mutations in ORF3a could be driving protein changes that mediate immune evasion and favoring viral spread."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-28"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>SARS-CoV-2 and ORF3a: Non-Synonymous Mutations and Polyproline Regions | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1215889 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">The effect of the rapid accumulation of non-synonymous mutations on the pathogenesis of SARS-CoV-2 is not yet known. To predict the impact of non-synonymous mutations and polyproline regions identified in ORF3a on the formation of B-cell epitopes and their role in evading the immune response, nucleotide and protein sequences of 537 available SARS-CoV-2 genomes were analyzed for the presence of non-synonymous mutations and polyproline regions. Mutations were correlated with changes in epitope formation. A total of 19 different non-synonymous amino acids substitutions were detected in ORF3a among 537 SARS-CoV-2 strains. G251V was the most common and identified in 9.9% (n=53) of the strains and was predicted to lead to the loss of a B-cell like epitope in ORF3a. Polyproline regions were detected in two strains (EPI_ISL_410486, France and EPI_ISL_407079, Finland) and affected epitopes formation. The accumulation of non-synonymous mutations and detected polyproline regions in ORF3a of SARS-CoV-2 could be driving the evasion of the host immune response thus favoring viral spread. Rapid mutations accumulating in ORF3a should be closely monitored throughout the COVID-19 pandemic.</p>
    <div id="sec-1" class="subsection">
     <p id="p-4">
      <strong>Importance</strong> At the surge of the COVID-19 pandemic and after three months of the identification of SARS-CoV-2 as the disease-causing pathogen, nucleic acid changes due to host-pathogen interactions are insightful into the evolution of this virus. In this paper, we have identified a set of non-synonymous mutations in ORF3a and predicted their impact on B-cell like epitope formation. The accumulation of non-synonymous mutations in ORF3a could be driving protein changes that mediate immune evasion and favoring viral spread.
     </p>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Introduction</h2>
    <p id="p-5">The rapid spread of the coronavirus disease 2019 (COVID-19) caused by a novel coronavirus, named SARS-CoV-2 due to its symptoms similarity to those induced by the severe acute respiratory syndrome (SARS), is a major global concern (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). The epidemic started in late December 2019 in Wuhan, the capital of Central China’s Hubei Province and since then thousands of cases have been reported in more than 46 countries (
     <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/</a>). Coronaviruses are enveloped non-segmented positive sense RNA viruses belonging to the family Coronaviridae and the order Nidovirales and are broadly distributed in humans and other mammals. The genome of SARS-CoV-2 showed 96.2% sequence similarity to a bat SARS-related coronavirus (SARSr-CoV; RaTG13) collected in Yunnan province, China (
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>) and 79% and 50% similarities to SARS-CoV and MERS-CoV, respectively (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). A transmission from wild-life animals (such as pangolins) to humans has been recently suggested (
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>).
    </p>
    <p id="p-6">With the immediate and continuous release of sequence data, monitoring the rapid evolution of the SARS-CoV-2 genome provides a strong lead towards predicting and potentially mitigating its global spread. ORF3a protein (Accession # YP_009724391.1) is a hypothetical protein showing a 72% sequence similarity to SARS3a protein in SARS-CoV. Here, we investigated the presence of diverse non-synonymous mutations in ORF3a and their effects on the predicted protein structure and its potential implication in the formation of epitopes. Moreover, polyproline regions (PPRs) were detected in two strains. We used this approach to follow and understand the impact of new emerging mutations in the pathogenesis and immune evasion of SARS-CoV-2.</p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Results</h2>
    <div id="sec-4" class="subsection">
     <h3>Micro-clonality within ORF3a</h3>
     <p id="p-7">The clonal diversity of SARS-CoV-2 core genomes was highly similar in tree topology to the gene tree of ORF3a (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Signature mutations within SARS-CoV-2 genomes cluster them into defined phylogenetic clades. Similarly, we observed micro-clonality within the ORF3a gene tree defined by highlighted non-synonymous mutations G251V (green) and Q57H (pink) that are found in conserved phylogenetic micro-clades representing sub-populations of mutants.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic trees of SARS-CoV-2 core genomes and ORF3a. Magnified maximum-likelihood phylogenetic trees of (A) SARS-CoV-2 genomes based on core genome SNP differences in all concatenated ORFs and (B) ORF3a gene tree highlighting G251V mutant clade in green and Q57H mutant clade in pink." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2076937451" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Phylogenetic trees of SARS-CoV-2 core genomes and ORF3a.&lt;;/span&gt;; Magnified maximum-likelihood phylogenetic trees of &lt;;strong&gt;;(A)&lt;;/strong&gt;; SARS-CoV-2 genomes based on core genome SNP differences in all concatenated ORFs and &lt;;strong&gt;;(B)&lt;;/strong&gt;; ORF3a gene tree highlighting G251V mutant clade in green and Q57H mutant clade in pink.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F1.medium.gif" width="440" height="251"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F1.medium.gif" width="440" height="251"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1:</span>
       <span class="caption-title">Phylogenetic trees of SARS-CoV-2 core genomes and ORF3a.</span>
       <p id="p-8" class="first-child">Magnified maximum-likelihood phylogenetic trees of 
        <strong>(A)</strong> SARS-CoV-2 genomes based on core genome SNP differences in all concatenated ORFs and 
        <strong>(B)</strong> ORF3a gene tree highlighting G251V mutant clade in green and Q57H mutant clade in pink.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Non-synonymous Mutations in ORF3a</h3>
     <p id="p-9">ORF3a, encoding a hypothetical protein, showed a 97.82% sequence similarity (100% coverage) to a nonstructural protein NS3 of Bat coronavirus RaTG13 (Accession # QHR63301.1). Moreover, ORF3a has a pro-apoptosis inducing APA3_viroporin conserved domain, also found in SARS-CoV.</p>
     <p id="p-10">Sequence alignment of 537 ORF3a protein sequences revealed a total of 19 non-synonymous amino acids substitutions, of which 52.6% (n=10) had a predicted deleterious functional outcome and 47.4% (n=9) had a neutral functional outcome (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2.A</a>).
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Mutations analysis of ORF3a. (A) Multiple sequence alignment between ORF3a protein of G251V (EPI_ISL_414569, Hong Kong), G254R (EPI_ISL_415627, USA), T176I (EPI_ISL_411950, Jiangsu), PPR-containing proteins (EPI_ISL_410486, France and EPI_ISL_407079, Finland) mutants compared to non-mutant (EPI_ISL_402123, Wuhan) (B) B-cell like epitopes of the non-mutated ORF3a protein (left) and G251V mutant (right). Only values above the threshold (0.5) are included. The mutation lead to the loss of one B cell epitope. (C) B-cell like epitopes of PPR-containing isolates. Additional epitopes are indicated in red. (D) B-cell like epitopes of T176I and G254R mutants with decreased intensity as compared to non-mutant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2076937451" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Mutations analysis of ORF3a.&lt;;/span&gt;; &lt;;strong&gt;;(A)&lt;;/strong&gt;; Multiple sequence alignment between ORF3a protein of G251V (EPI_ISL_414569, Hong Kong), G254R (EPI_ISL_415627, USA), T176I (EPI_ISL_411950, Jiangsu), PPR-containing proteins (EPI_ISL_410486, France and EPI_ISL_407079, Finland) mutants compared to non-mutant (EPI_ISL_402123, Wuhan) &lt;;strong&gt;;(B)&lt;;/strong&gt;; B-cell like epitopes of the non-mutated ORF3a protein (left) and G251V mutant (right). Only values above the threshold (0.5) are included. The mutation lead to the loss of one B cell epitope. &lt;;strong&gt;;(C)&lt;;/strong&gt;; B-cell like epitopes of PPR-containing isolates. Additional epitopes are indicated in red. &lt;;strong&gt;;(D)&lt;;/strong&gt;; B-cell like epitopes of T176I and G254R mutants with decreased intensity as compared to non-mutant.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F2.medium.gif" width="356" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F2.medium.gif" width="356" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.27.012013/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2:</span>
       <span class="caption-title">Mutations analysis of ORF3a.</span>
       <p id="p-11" class="first-child">
        <strong>(A)</strong> Multiple sequence alignment between ORF3a protein of G251V (EPI_ISL_414569, Hong Kong), G254R (EPI_ISL_415627, USA), T176I (EPI_ISL_411950, Jiangsu), PPR-containing proteins (EPI_ISL_410486, France and EPI_ISL_407079, Finland) mutants compared to non-mutant (EPI_ISL_402123, Wuhan) 
        <strong>(B)</strong> B-cell like epitopes of the non-mutated ORF3a protein (left) and G251V mutant (right). Only values above the threshold (0.5) are included. The mutation lead to the loss of one B cell epitope. 
        <strong>(C)</strong> B-cell like epitopes of PPR-containing isolates. Additional epitopes are indicated in red. 
        <strong>(D)</strong> B-cell like epitopes of T176I and G254R mutants with decreased intensity as compared to non-mutant.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-12">G251V was the most frequently detected substitution found in 9.9% (n=53) of the strains followed by Q57H found in 3.9% (n=21) of the strains. Both G251V and Q57H were predicted to be deleterious (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1223657/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1223657/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1223657" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">List of 19 non-synonymous amino acids substitutions in ORF3a among 537 strains.</span>
       <p id="p-13" class="first-child">The G251V substitution is shown in bold.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-6" class="subsection">
     <h3>G251V linked to an Epitope Loss</h3>
     <p id="p-14">The G251V mutations were further investigated. Motif scanning demonstrated that G251V resulted in the loss of a phosphatidylinositol-specific phospholipase X-box domain 
      <em>(</em>PIPI_CX_DOMAIN; 203-275 aa). The G251V substitution created serine protease cleavage site. IEDB analysis revealed the presence of six putative epitopes in the non-mutant ORF3a compared to five epitopes in the mutant ORF3a (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2.B</a>). The G251V substitution in ORF3a was linked to the loss of a putative epitope the impact of which on viral spread and pathogenesis requires further experimental studies. Other T176I and G254R substitutions resulted in a decreased intensity of epitopes number two (blue) and epitope number five (yellow) (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2.D</a>).
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Detection of PPRs</h3>
     <p id="p-15">Notably, we detected PPRs in two SARS-CoV-2 genomes (EPI_ISL_407079, Finland and EPI_ISL_410486, France). PPRs resulted in the joining of epitopes number four (purple) and five (yellow) into one larger epitope(red) of 22 amino acids in size (start:235; end:256; sequence: KIPPPPPPPPPLHTIDGSSGVV) in EPI_ISL_410486 (France) and led the appearance of a new epitope 23 amino acids in size (start 135; end 157; sequence: SKNPPPPPPPPPPPPPPPPHYC) in EPI_ISL_407079 (Finland) (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2.C</a>). Blastn search of a 23 bp DNA stretch from non-mutant strains showed a 100% identity to RaTG13 (Accession # 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN996532.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom">MN996532.1</a>).
     </p>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-16">These combined results suggest that the non-synonymous G251V mutation introduced into ORF3a protein in SARS-CoV-2 could be linked to immune evasion and thus viral spread and pathogenesis. ORF3a is a transmembrane protein that localizes to the plasma membrane especially in the ER-Golgi region and activates the PKR-like ER kinase (PERK) signaling pathway which protects viral proteins against ER-associated degradation. The activation of this pathway leads to apoptosis(
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). A pro-apoptosis inducing APA3_viroporin conserved domain detected in ORF3a of SARS-CoV-2 is also found in SARS-CoV 3A protein (
     <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>).
    </p>
    <p id="p-17">The G251V was detected in ORF3a in 9.9% of the strains (n=53). G251V led to the loss a B cell-like epitope and a PIPLC_X_DOMAIN the eukaryotic homologue of which is involved in signal transduction processes (
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). The accumulation of non-synonymous mutations could be driven by the humoral immunity as reported previously in the mucin-like domain of the Ebola virus glycoprotein (
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>).
    </p>
    <p id="p-18">Of paramount importance is the emergence of PPRs in ORF3a detected in two of the SARS-CoV-2 sequenced genomes (in EPI_ISL_410486, France and EPI_ISL_407079, Finland). PPRs are an open field for recombination that viruses use to adapt based on selective pressure (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). PPRs were previously shown to be indispensable for the activity of the Coxsackievirus B 3A protein which blocks ER-to-Golgi transport affecting protein synthesis (
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). Studies on Hepatitis E virus also highlighted the role of PPRs in host-range adaptation and viral replication (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>).
    </p>
    <p id="p-19">In conclusion, our study reveals and for the first time a common non-synonymous G251V substitution and PPRs in ORF3a which could be respectively linked to the loss of a putative epitope and viral spread and pathogenesis.</p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-10" class="subsection">
     <h3>Pan-genome analysis</h3>
     <p id="p-20">A total of 537 SARS-CoV-2 complete genomes with high quality sequencing downloaded from GISAID were utilized for genome and ORF3a alignments.</p>
     <p id="p-21">All sequences were uniformly annotated using Prokka v 1.1.3 (
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). The annotated Genbank files were edited to have more concise locus tag identifiers. The Genbank annotations of the genomes were used as input in the PanX (
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>) pipeline for pan genome analysis. A core genome threshold of 0.99, MCL inflation parameter of 1.5, and a modified core diversity cutoff for branch lengths above 0.001 were used alongside the default parameters.
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Protein Structure prediction</h3>
     <p id="p-22">Sequences were aligned using MUSCLE v3.8.31 (
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). PROVEAN was used to predict the functional effects of amino acid substitutions (
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). ExPASy and PROSPER were used for motif scanning and protease site prediction, respectively (
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, 
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). The Immune epitope database analysis resource (IEDB-AR) was used for epitopes prediction using a 0.5 threshold and default settings (
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>).
     </p>
    </div>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Author Contributions</h2>
    <p id="p-24">
     <em>Concept and design:</em> S.T. 
     <em>Acquisition, analysis, or interpretation of data:</em> All authors. 
     <em>Drafting of the manuscript:</em> All authors. 
     <em>Critical revision of the manuscript for important intellectual content:</em> Tokajian. 
     <em>Administrative, technical, or material support:</em> E.I, B.P, G.M. 
     <em>Supervision:</em> S.T.
    </p>
   </div>
   <div class="section" id="sec-13">
    <h2 class="">Conflicts of interest</h2>
    <p id="p-25">The authors wish to declare that they do not have any conflict of interests.</p>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">Funding/support</h2>
    <p id="p-26">This work was partially financed by the School of Arts and Sciences Research and Development Council at the Lebanese American University.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-23">We thankfully acknowledge the authors, generating and submitting laboratories of the sequences from GISAID’s EpiCoV™ database. We also acknowledge the authors of all Coronaviridae genome sequences deposited in GenBank. This study does not claim ownership of these sequences, which were used within the analysis workflow to further our understanding of the on-going pandemic of SARS-CoV-2 and the underlying molecular changes that govern the virus’ transmission and infectivity patterns. The authors wish to declare that they do not have any conflict of interests.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">X-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X-G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Si</span>
          <span class="cit-name-given-names">H-R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">C-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">H-D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>
          <span class="cit-name-given-names">R-D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">M-Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shen</span>
          <span class="cit-name-given-names">X-R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>
          <span class="cit-name-given-names">X-S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">Q-J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">L-L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhan</span>
          <span class="cit-name-given-names">F-X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Y-Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>
          <span class="cit-name-given-names">G-F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Z-L.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">4</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.2" data-doi="10.3390/v12020135">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gralinski</span>
          <span class="cit-name-given-names">LE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Menachery</span>
          <span class="cit-name-given-names">VD.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">Return of the Coronavirus: 2019-nCoV. 2</span>. 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>
         <span class="cit-vol">12</span>:
         <span class="cit-fpage">135</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D12%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv12020135%26rft_id%253Dinfo%253Apmid%252F31991541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3390/v12020135&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31991541&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Andersen</span>
          <span class="cit-name-given-names">KG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lipkin</span>
          <span class="cit-name-given-names">WI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">EC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garry</span>
          <span class="cit-name-given-names">RF.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">The proximal origin of SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">Nat Med</abbr>
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">3</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Seemann</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Prokka: rapid prokaryotic genome annotation</span>. 
         <abbr class="cit-jnl-abbrev">Bioinforma Oxf Engl</abbr>
         <span class="cit-vol">30</span>:
         <span class="cit-fpage">2068</span>–
         <span class="cit-lpage">2069</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinforma%2BOxf%2BEngl%26rft.volume%253D30%26rft.spage%253D2068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.5" data-doi="10.1093/nar/gkx977">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ding</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baumdicker</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Neher</span>
          <span class="cit-name-given-names">RA.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">panX: pan-genome analysis and exploration</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">46</span>:
         <span class="cit-fpage">e5</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253De5%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkx977&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.6" data-doi="10.1093/nar/gkh340">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Edgar</span>
          <span class="cit-name-given-names">RC.</span>
         </span>
         <span class="cit-pub-date">2004</span>. 
         <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">32</span>:
         <span class="cit-fpage">1792</span>–
         <span class="cit-lpage">1797</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15034147&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000220487200025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.7" data-doi="10.1093/bioinformatics/btv195">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Choi</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">AP.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">PROVEAN web server: a tool to predict the functional effect of amino acid substitutions and indels</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">31</span>:
         <span class="cit-fpage">2745</span>–
         <span class="cit-lpage">2747</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv195%26rft_id%253Dinfo%253Apmid%252F25851949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv195&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25851949&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.8" data-doi="10.1093/nar/gks400">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Artimo</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jonnalagedda</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Arnold</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baratin</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Csardi</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Castro</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duvaud</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Flegel</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fortier</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gasteiger</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grosdidier</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hernandez</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ioannidis</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuznetsov</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liechti</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moretti</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mostaguir</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Redaschi</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rossier</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xenarios</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stockinger</span>
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">ExPASy: SIB bioinformatics resource portal</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">40</span>:
         <span class="cit-fpage">W597</span>–
         <span class="cit-lpage">W603</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks400%26rft_id%253Dinfo%253Apmid%252F22661580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gks400&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22661580&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000306670900098&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leier</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marquez-Lago</span>
          <span class="cit-name-given-names">TT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Akutsu</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haffari</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chou</span>
          <span class="cit-name-given-names">K-C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Webb</span>
          <span class="cit-name-given-names">GI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pike</span>
          <span class="cit-name-given-names">RN.</span>
         </span>
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-article-title">PROSPERous: high-throughput prediction of substrate cleavage sites for 90 proteases with improved accuracy</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">34</span>:
         <span class="cit-fpage">684</span>–
         <span class="cit-lpage">687</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.10" data-doi="10.1093/nar/gkn254">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haste-Andersen</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Beaver</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bourne</span>
          <span class="cit-name-given-names">PE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bui</span>
          <span class="cit-name-given-names">H-H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Buus</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Frankild</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Greenbaum</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lund</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lundegaard</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nielsen</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ponomarenko</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sette</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peters</span>
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-article-title">Immune epitope database analysis resource (IEDB-AR)</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">36</span>:
         <span class="cit-fpage">W513</span>–
         <span class="cit-lpage">518</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn254%26rft_id%253Dinfo%253Apmid%252F18515843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkn254&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18515843&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000258142300094&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.11" data-doi="10.1371/journal.pone.0008342">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Minakshi</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Padhan</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rani</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Khan</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ahmad</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jameel</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">The SARS Coronavirus 3a protein causes endoplasmic reticulum stress and induces ligand-independent downregulation of the type 1 interferon receptor</span>. 
         <abbr class="cit-jnl-abbrev">PloS One</abbr>
         <span class="cit-vol">4</span>:
         <span class="cit-fpage">e8342</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMinakshi%26rft.auinit1%253DR.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8342%26rft.epage%253De8342%26rft.atitle%253DThe%2BSARS%2BCoronavirus%2B3a%2Bprotein%2Bcauses%2Bendoplasmic%2Breticulum%2Bstress%2Band%2Binduces%2Bligand-independent%2Bdownregulation%2Bof%2Bthe%2Btype%2B1%2Binterferon%2Breceptor.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008342%26rft_id%253Dinfo%253Apmid%252F20020050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008342&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20020050&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.12" data-doi="10.1016/j.cell.2015.06.007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>
          <span class="cit-name-given-names">DJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dudas</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wohl</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Goba</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Whitmer</span>
          <span class="cit-name-given-names">SLM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Andersen</span>
          <span class="cit-name-given-names">KG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sealfon</span>
          <span class="cit-name-given-names">RS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ladner</span>
          <span class="cit-name-given-names">JT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kugelman</span>
          <span class="cit-name-given-names">JR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Matranga</span>
          <span class="cit-name-given-names">CB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Winnicki</span>
          <span class="cit-name-given-names">SM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qu</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gire</span>
          <span class="cit-name-given-names">SK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gladden-Young</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jalloh</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nosamiefan</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yozwiak</span>
          <span class="cit-name-given-names">NL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moses</span>
          <span class="cit-name-given-names">LM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>
          <span class="cit-name-given-names">P-P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">AE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schaffner</span>
          <span class="cit-name-given-names">SF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bird</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Towner</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mamoh</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gbakie</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kanneh</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kargbo</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Massally</span>
          <span class="cit-name-given-names">JLB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kamara</span>
          <span class="cit-name-given-names">FK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Konuwa</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sellu</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jalloh</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mustapha</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Foday</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yillah</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Erickson</span>
          <span class="cit-name-given-names">BR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sealy</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blau</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Paddock</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brault</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Amman</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Basile</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bearden</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Belser</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bergeron</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Campbell</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chakrabarti</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dodd</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Flint</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gibbons</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Goodman</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Klena</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McMullan</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morgan</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Russell</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzer</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sanchez</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jungreis</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tomkins-Tinch</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kislyuk</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">MF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chapman</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">MacInnis</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Matthews</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bochicchio</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hensley</span>
          <span class="cit-name-given-names">LE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuhn</span>
          <span class="cit-name-given-names">JH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nusbaum</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schieffelin</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Birren</span>
          <span class="cit-name-given-names">BW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Forget</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nichol</span>
          <span class="cit-name-given-names">ST</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palacios</span>
          <span class="cit-name-given-names">GF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ndiaye</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Happi</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gevao</span>
          <span class="cit-name-given-names">SM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vandi</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kargbo</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">EC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bedford</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gnirke</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ströher</span>
          <span class="cit-name-given-names">U</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garry</span>
          <span class="cit-name-given-names">RF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sabeti</span>
          <span class="cit-name-given-names">PC.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Ebola Virus Epidemiology, Transmission, and Evolution during Seven Months in Sierra Leone</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-vol">161</span>:
         <span class="cit-fpage">1516</span>–
         <span class="cit-lpage">1526</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D1516%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.06.007%26rft_id%253Dinfo%253Apmid%252F26091036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.06.007&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26091036&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.13" data-doi="10.1128/JVI.01625-14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lhomme</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abravanel</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dubois</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sandres-Saune</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mansuy</span>
          <span class="cit-name-given-names">J-M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rostaing</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kamar</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Izopet</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Characterization of the polyproline region of the hepatitis E virus in immunocompromised patients</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">88</span>:
         <span class="cit-fpage">12017</span>–
         <span class="cit-lpage">12025</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01625-14%26rft_id%253Dinfo%253Apmid%252F25100839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8yMC8xMjAxNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjcuMDEyMDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.14" data-doi="10.1128/JVI.79.8.5163-5173.2005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wessels</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duijsings</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Notebaart</span>
          <span class="cit-name-given-names">RA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Melchers</span>
          <span class="cit-name-given-names">WJG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuppeveld</span>
          <span class="cit-name-given-names">FJM van</span>
         </span>. 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-article-title">A Proline-Rich Region in the Coxsackievirus 3A Protein Is Required for the Protein To Inhibit Endoplasmic Reticulum-to-Golgi Transport</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">79</span>:
         <span class="cit-fpage">5163</span>–
         <span class="cit-lpage">5173</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DWessels%26rft.auinit1%253DE.%26rft.volume%253D79%26rft.issue%253D8%26rft.spage%253D5163%26rft.epage%253D5173%26rft.atitle%253DA%2BProline-Rich%2BRegion%2Bin%2Bthe%2BCoxsackievirus%2B3A%2BProtein%2BIs%2BRequired%2Bfor%2Bthe%2BProtein%2BTo%2BInhibit%2BEndoplasmic%2BReticulum-to-Golgi%2BTransport%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.8.5163-5173.2005%26rft_id%253Dinfo%253Apmid%252F15795300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzgvNTE2MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI4LzIwMjAuMDMuMjcuMDEyMDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.27.012013v1.15" data-doi="10.1371/journal.pone.0035974">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Purdy</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lara</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Khudyakov</span>
          <span class="cit-name-given-names">YE.</span>
         </span>
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">The Hepatitis E Virus Polyproline Region Is Involved in Viral Adaptation</span>. 
         <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>
         <span class="cit-vol">7</span>:
         <span class="cit-fpage">e35974</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPurdy%26rft.auinit1%253DM.%2BA.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De35974%26rft.epage%253De35974%26rft.atitle%253DThe%2Bhepatitis%2BE%2Bvirus%2Bpolyproline%2Bregion%2Bis%2Binvolved%2Bin%2Bviral%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0035974%26rft_id%253Dinfo%253Apmid%252F22545153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0035974&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22545153&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.27.012013.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
