<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/04/24/307678.atom" />
<meta name="HW.pisa" content="biorxiv;307678v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome" />
<meta name="DC.Identifier" content="10.1101/307678" />
<meta name="DC.Date" content="2018-04-24" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="While RNA viruses thrive with massive structural and functional diversity, their genomes size variation is particularly low, ranging only from ~2-to-33 kb. Here, I present the characterization of RNA sequences corresponding to the first virus associated with Aplysia californica . Genome structure and domain architecture suggest that the identified virus is a novel member of Nidovirales . The proposed aplysia californica nido-like virus (AcNV), with a genome sequence of ca.35,906 nt, represents the longest ever recorded RNA virus yet. Phylogenetic insights indicate that AcNV clusters in a major phylloclade of unclassified invertebrate nidoviruses, Roniviridae , and Mesoniviridae . Basal branching in this emerging cluster could indicate that AcNV is a member of a novel divergent clade within Nidovirales . Further, virus RNA detection in multiple independent studies suggests that AcNV is neurotropic with a broad cell/tissue/organ tropism, supported by AcNV occurrence in diverse organs, including the first detection of a Nidovirales in single specific neurons.

Highlights" />
<meta name="DC.Contributor" content="Humberto J Debat" />
<meta name="article:published_time" content="2018-04-24" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;While RNA viruses thrive with massive structural and functional diversity, their genomes size variation is particularly low, ranging only from ~2-to-33 kb. Here, I present the characterization of RNA sequences corresponding to the first virus associated with &lt;i&gt;Aplysia californica&lt;/i&gt;. Genome structure and domain architecture suggest that the identified virus is a novel member of &lt;i&gt;Nidovirales&lt;/i&gt;. The proposed aplysia californica nido-like virus (AcNV), with a genome sequence of ca.35,906 nt, represents the longest ever recorded RNA virus yet. Phylogenetic insights indicate that AcNV clusters in a major phylloclade of unclassified invertebrate nidoviruses, &lt;i&gt;Roniviridae&lt;/i&gt;, and &lt;i&gt;Mesoniviridae&lt;/i&gt;. Basal branching in this emerging cluster could indicate that AcNV is a member of a novel divergent clade within &lt;i&gt;Nidovirales&lt;/i&gt;. Further, virus RNA detection in multiple independent studies suggests that AcNV is neurotropic with a broad cell/tissue/organ tropism, supported by AcNV occurrence in diverse organs, including the first detection of a &lt;i&gt;Nidovirales&lt;/i&gt; in single specific neurons.&lt;/p&gt;&lt;h3&gt;Highlights&lt;/h3&gt;
&lt;p&gt;-RNA virus genomes reported in the literature are limited at ca. 33.4 kb&lt;/p&gt;&lt;p&gt;-A novel nidovirus was identified in the gastropod mollusk &lt;i&gt;Aplysia californica&lt;/i&gt;&lt;/p&gt;&lt;p&gt;-The aplysia californica nido-like virus (AcNV) presents a 35.9 kb RNA genome&lt;/p&gt;&lt;p&gt;-AcNV has a broad tropism, is enriched in the CNS, and accumulates in neurons&lt;/p&gt;&lt;p&gt;-The unique features of &lt;i&gt;A. californica&lt;/i&gt; enables single-neuron virus dynamics of AcNV&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;307678v1" />
<meta name="citation_id" content="307678v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/307678v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/307678v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/307678v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678.full.pdf" />
<meta name="citation_doi" content="10.1101/307678" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="307678" />
<meta name="citation_author" content="Humberto J Debat" />
<meta name="citation_author_institution" content="Instituto de Patología Vegetal, Centro de Investigaciones Agropecuarias, Instituto Nacional de Tecnologia Agropecuaria (IPAVE-CIAP-INTA)" />
<meta name="citation_author_email" content="debat.humberto@inta.gob.ar" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3056-3739" />
<meta name="citation_reference" content="Adams M.J., Lefkowitz E.J., King A.M.Q., Harrach B., Harrison R.L., Knowles N.J., Kropinski A.M., Krupovic M., Kuhn J.H., Mushegian A.R., Nibert M., Sabanadzovic S., Sanfafon H., Siddell S.G., Simmonds P., Varsani A., Zerbini F.M., Gorbalenya A.E., Davison A.J., 2017. Changes to taxonomy and the international code of virus classification and Nomenclature ratified by the international committee on taxonomy of viruses (2017). Arch. Virol. 162, 2505–2538." />
<meta name="citation_reference" content="Anand, K., Palm, G. J., Mesters, J. R., Siddell, S. G., Ziebuhr, J., &amp; Hilgenfeld, R. (2002). Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α◻helical domain. EMBO J., 21(13), 3213–3224." />
<meta name="citation_reference" content="Bodewes, R., Lempp, C., Schurch, A. C., Habierski, A., Hahn, K., Lamers, M., … &amp; Smits, S. L., 2014. Novel divergent nidovirus in a python with pneumonia. J. Gen. Virol., 95 (11), 2480–2485." />
<meta name="citation_reference" content="Brierley, I., Boursnell, M. E., Binns, M. M., Bilimoria, B., Blok, V. C., Brown, T. D., &amp; Inglis, S. C. (1987). An efficient ribosomal frameLshifting signal in the polymeraseLencoding region of the coronavirus IBV. EMBO J., 6(12), 3779–3785." />
<meta name="citation_reference" content="Campillo-Balderas, J. A., Lazcano, A., &amp; Becerra, A., 2015. Viral genome size distribution does not correlate with the antiquity of the host lineages. Front. Ecol. Evol., 3, 143." />
<meta name="citation_reference" content="Carew, T. J., Pinsker, H. M., &amp; Kandel, E. R., 1972. Long-term habituation of a defensive withdrawal reflex in Aplysia. Science, 175(4020), 451–454." />
<meta name="citation_reference" content="Cotten, M., Lam, T. T., Watson, S. J., Palser, A. L., Petrova, V., Grant, P., … &amp; Kellam, P., 2013. Full-genome deep sequencing and phylogenetic analysis of novel human betacoronavirus. Emerging Infect. Dis., 19 (5), 736." />
<meta name="citation_reference" content="citation_title=Virus Taxonomy: Classification and Nomenclature of Viruses: Ninth Report of the International Committee on Taxonomy of Viruses.;citation_pages=784-794;citation_year=2012" />
<meta name="citation_reference" content="Debat, H.J., 2017. An RNA Virome associated to the Golden Orb-weaver Spider Nephila clavipes. Front. Microbiol., 8: 2097." />
<meta name="citation_reference" content="Denison, M. R., Graham, R. L., Donaldson, E. F., Eckerle, L. D., &amp; Baric, R. S., 2011. Coronaviruses: an RNA proofreading machine regulates replication fidelity and diversity. RNA Biol., 8 (2), 270–279." />
<meta name="citation_reference" content="Dolja, V. V., &amp; Koonin, E. V., 2018. Metagenomics reshapes the concepts of RNA virus evolution by revealing extensive horizontal virus transfer. Virus Res., 244, 36–52." />
<meta name="citation_reference" content="Duffy, S., Shackelton, L. A., Holmes, E. C., 2008. Rates of evolutionary change in viruses: patterns and determinants. Nature Rev Genet., 9(4), 267–276." />
<meta name="citation_reference" content="Elya, C., Lok, T. C., Spencer, Q. E., McCausland, H., &amp; Eisen, M. B. (2017). A fungal pathogen that robustly manipulates the behavior of Drosophila melanogaster in the laboratory. bioRxiv, 232140." />
<meta name="citation_reference" content="Fiedler, T. J., Hudder, A., McKay, S. J., Shivkumar, S., Capo, T. R., Schmale, M. C., &amp; Walsh, P. J. (2010). The transcriptome of the early life history stages of the California Sea Hare Aplysia californica. Comp. Biochem. Physiol. Part D Genomics Proteomics, 5(2), 165–170." />
<meta name="citation_reference" content="Gorbalenya A. E. (2001). Big nidovirus genome. In The Nidoviruses (pp. 1–17). Springer, Boston, MA." />
<meta name="citation_reference" content="Gorbalenya, A. E., Koonin, E. V., Donchenko, A. P., &amp; Blinov, V. M. (1989). Coronavirus genome: prediction of putative functional domains in the non-structural polyprotein by comparative amino acid sequence analysis. Nucleic Acids Res., 17(12), 4847–4861." />
<meta name="citation_reference" content="Gorbalenya, A. E., Enjuanes, L., Ziebuhr, J., &amp; Snijder, E. J., 2006. Nidovirales: evolving the largest RNA virus genome. Virus Res., 117 (1), 17–37." />
<meta name="citation_reference" content="Greninger, A. L., 2018. A decade of RNA virus metagenomics is (not) enough. Virus Res., 244, 218–229." />
<meta name="citation_reference" content="Heyland, A., Vue, Z., Voolstra, C. R., Medina, M., &amp; Moroz, L. L., 2011. Developmental transcriptome of Aplysia californica. J. Exp. Zool. B Mol. Dev. Evol., 316 (2), 113–134." />
<meta name="citation_reference" content="Holland, J., Spindler, K., Horodyski, F., Grabau, E., Nichol, S., &amp; VandePol, S., 1982. Rapid evolution of RNA genomes. Science, 215(4540), 1577–1585." />
<meta name="citation_reference" content="Hoon-Hanks, L. L., Layton, M. L., Ossiboff, R. J., Parker, J. S., Dubovi, E. J., &amp; Stenglein, M. D. 2018. Respiratory disease in ball pythons (Python regius) experimentally infected with ball python nidovirus. Virology, doi: 10.1016/j.virol.2017.12.008" />
<meta name="citation_reference" content="Islam, S., Kjallquist, U., Moliner, A., Zajac, P., Fan, J. B., Lonnerberg, P., &amp; Linnarsson, S. (2011). Characterization of the single-cell transcriptional landscape by highly multiplex RNA-seq. Genome Res., 21(7), 1160–1167." />
<meta name="citation_reference" content="Kandel E. R. (1976). Cellular basis of behavior: An introduction to behavioral neurobiology. W.H. Freeman and Company, San Francisco." />
<meta name="citation_reference" content="Katzourakis, A., &amp; Gifford, R. J. 2010. Endogenous viral elements in animal genomes. PLoS Genet., 6 (11), e1001191." />
<meta name="citation_reference" content="Lauber, C., Ziebuhr, J., Junglen, S., Drosten, C., Zirkel, F., Nga, P. T., … &amp; Gorbalenya, A. E., 2012. Mesoniviridae: a proposed new family in the order Nidovirales formed by a single species of mosquito-borne viruses. Arch. Virol., 157 (8), 1623–1628." />
<meta name="citation_reference" content="Marzano, S. Y. L., Nelson, B. D., Ajayi-Oyetunde, O., Bradley, C. A., Hughes, T. J., Hartman, G. L., … &amp; Domier, L. L. (2016). Identification of diverse mycoviruses through metatranscriptomics characterization of the viromes of five major fungal plant pathogens. J. Virol., 90(15), 6846–6863." />
<meta name="citation_reference" content="Moroz, L. L., Edwards, J. R., Puthanveettil, S. V., Kohn, A. B., Ha, T., Heyland, A., … &amp; Kandel, E.R., 2006. Neuronal transcriptome of Aplysia: neuronal compartments and circuitry. Cell, 127 (7), 1453–1467." />
<meta name="citation_reference" content="Moroz, L. L., &amp; Kohn, A. B., 2010. Do different neurons age differently? Direct genome-wide analysis of aging in single identified cholinergic neurons. Front. Aging Neurosci., 2, 6." />
<meta name="citation_reference" content="Moroz, L. L., &amp; Kohn, A. B., 2013. Single-neuron transcriptome and methylome sequencing for epigenomic analysis of aging. Methods Mol. Biol., 1048, 323–352." />
<meta name="citation_reference" content="Nga, P.T., Parquet, M., del, C., Lauber, C., Parida, M., Nabeshima, T., Yu, F., Thuy, N.T., Inoue, S., Ito, T., Okamoto, K., Ichinose, A., Snijder, E.J., Morita, K., Gorbalenya, A.E., 2011. Discovery of the first insect nidovirus, a missing evolutionary link in the emergence of the largest RNA virus genomes. PLoS Pathog., 7, e1002215." />
<meta name="citation_reference" content="Puthanveettil, S. V., Antonov, I., Kalachikov, S., Rajasethupathy, P., Choi, Y. B., Kohn, A. B., … Moroz, L. L. &amp; Kandel, E. R., 2013. A strategy to capture and characterize the synaptic transcriptome. Proc. Natl. Acad. Sci. USA, 110 (18), 7464–7469." />
<meta name="citation_reference" content="Van den Hoecke, S., Verhelst, J., Vuylsteke, M., &amp; Saelens, X., 2015. Analysis of the genetic diversity of influenza A viruses using next-generation DNA sequencing. BMC Genomics, 16 (1), 79." />
<meta name="citation_reference" content="Shi, M., Lin, X. D., Tian, J. H., Chen, L. J., Chen, X., Li, C. X., … &amp; Buchmann, J. (2016). Redefining the invertebrate RNA virosphere. Nature, 540(7634), 539." />
<meta name="citation_reference" content="Shi, M., Zhang, Y. Z., &amp; Holmes, E. C., 2018. Meta-transcriptomics and The Evolutionary Biology of RNA Viruses. Virus Res., 244, 36–52." />
<meta name="citation_reference" content="Stenglein, M.D., Jacobson, E.R., Wozniak, E.J., Wellehan, J.F.X., Kincaid, A., Gordon, M., Porter, B.F., Baumgartner, W., Stahl, S., Kelley, K., Towner, J.S., DeRisi, J.L., 2014. Ball python nidovirus: a candidate etiologic agent for severe respiratory disease in Python regius. mBio, 5, e01484–1414." />
<meta name="twitter:title" content="Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="While RNA viruses thrive with massive structural and functional diversity, their genomes size variation is particularly low, ranging only from ~2-to-33 kb. Here, I present the characterization of RNA sequences corresponding to the first virus associated with Aplysia californica . Genome structure and domain architecture suggest that the identified virus is a novel member of Nidovirales . The proposed aplysia californica nido-like virus (AcNV), with a genome sequence of ca.35,906 nt, represents the longest ever recorded RNA virus yet. Phylogenetic insights indicate that AcNV clusters in a major phylloclade of unclassified invertebrate nidoviruses, Roniviridae , and Mesoniviridae . Basal branching in this emerging cluster could indicate that AcNV is a member of a novel divergent clade within Nidovirales . Further, virus RNA detection in multiple independent studies suggests that AcNV is neurotropic with a broad cell/tissue/organ tropism, supported by AcNV occurrence in diverse organs, including the first detection of a Nidovirales in single specific neurons.

Highlights" />
<meta name="og-title" property="og:title" content="Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/307678v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="While RNA viruses thrive with massive structural and functional diversity, their genomes size variation is particularly low, ranging only from ~2-to-33 kb. Here, I present the characterization of RNA sequences corresponding to the first virus associated with Aplysia californica . Genome structure and domain architecture suggest that the identified virus is a novel member of Nidovirales . The proposed aplysia californica nido-like virus (AcNV), with a genome sequence of ca.35,906 nt, represents the longest ever recorded RNA virus yet. Phylogenetic insights indicate that AcNV clusters in a major phylloclade of unclassified invertebrate nidoviruses, Roniviridae , and Mesoniviridae . Basal branching in this emerging cluster could indicate that AcNV is a member of a novel divergent clade within Nidovirales . Further, virus RNA detection in multiple independent studies suggests that AcNV is neurotropic with a broad cell/tissue/organ tropism, supported by AcNV occurrence in diverse organs, including the first detection of a Nidovirales in single specific neurons.

Highlights" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-04-24" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/307678v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/307678v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/307678v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/307678v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/94482" />
    <title>Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"NeyhZB50GFpDe5Znja4u44uLI3rswsyzpkedJch4qpQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"94482","apath":"\/biorxiv\/early\/2018\/04\/24\/307678.atom","pisa":"biorxiv;307678v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;307678\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;307678\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;307678v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;307678\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;307678\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/307678v1","title":"Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome","identifier":"node\/94482"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224qyIAWh3m8L+4hUNPlzz6whXEV59xD2xDk3VCHAWPO5mdpN4JfSwadmZddtTVbJY11yxoNZDdW5S\\\\\\\/KEfYOK1FZdKpJ+mUhcFm6lsYQUuTYG6WEqo3Y2f2iMwdVlqFP8iZBWdoaIK1ueSo+BHg+uVZPgX7f0aQfWia0srOJGATa6owJrzCB\\\\\\\/cdUDXPnQGQaDY4Fg9Q8hrYJ0YdSCHY8v1lBxkUtiAYsBX2P4pXVcWfnap04S\\\\\\\/R16kdMgAbBAVA\\\\\\\/SfqjooHU5jknSy3Otly401IWotYuyJIYgNMVw4wwMayqvYoRUsgnx8ke82HC95qUsJ6JUgeBBaZq1TR8O47xkUrwmnTRvCdESjtPdEsBmpAvvZOz1cb6J4eFn8S7wnC7CkEGCNKRLhoUhAa20mvek59YwdJ2XfYWNEAVSzDgD+DIFUlLcpS6M7qpjuJkYAHXKYAdETjK6Y\\\\\\\/Cm+DmKazHAdGwJjRg2H7FYqQUde9NqFZOyZWjQZ9usrg0Cd3e+oj9LklUls7eDw2XvyPIiW+rB9S51jmPXKTZqTvw7Z4ugE0nmL9qoQob4IabgwXHbSkNMTDaROrOac+GbJC82F1UTuHK80LSkh8MyGmxo2v9ndSNEh4OSluMgJyAVSmcM5r8NZLtn9Fy\\\\\\\/DwjAmPy7d0IYqjZ68KVBPWX\\\\\\\/ZETP1L6d4Ch+V4hI4kWKQB96pzoOdWrSdF0a\\\\\\\/Wy6sv7kl8\\\\\\\/pV05yLtc4+J2xfoNhi5GBo5TxnyfY1w5JaeC0L8OtkzYIzDPXcGdW\\\\\\\/X3a4G2P9Xfkn+XoPNQ9yR54XYDwD+4Le3c9jLJ9Z\\\\\\\/XO0qeXpdmEby+XUZ1Nw2v54WCEZFUlYWPUjRnBlEdEp4gQouvN2uSMpGyPztxyId0huXmAiKPDQW6I+VznD3zPTuqyoA0wnsAVOK2mlh0JOYY+5+Kisoid+pc5LVtEiEEnLiRebPj2fnCpON3oW\\\\\\\/hd2JQh1j18kuSRwc1hqv2to8v3kWhrw5hiHUzaDCiMx8pCzFsRHQ74pABij1A3zspLXi1sdDNz9kRjGo2iGqswAu7f3Uu\\\\\\\/1xLKFTRqt1CRUGV\\\\\\\/uDywfN4qK3lIf7oOq5iIisjcSObGYv2j9KQdwmQYCIzHt6B4q68nWj\\\\\\\/OCwxf2VteZIMg83\\\\\\\/3+1DQeluO39OylfEA6ePXbVKdKBDhgRON+mda0DrFP\\\\\\\/xIksthgB3xLaTuXXdcS0\\\\\\\/LjkA8kNETOMQL94hvj+84Krhy7qfr3GAIUQsErJL\\\\\\\/FGMO7hc0cDA2E0JYqwtVWzIdp39H1w5LMn6o7LNx+3XvC5T91b+P8TF3G0S5SmTjgB\\\\\\\/IjxpCi7ng=\\\u0022,\\\u0022iv\\\u0022:\\\u0022kRQ1CnuJar1dAybGj6cy4w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00225ecdd64c37b7bc40c75b29eed5ac3d564c13567def27eff7115a687ea014dd30\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/307678v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-94482 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/307678v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-T-ywoc39wIjT6rfnOsFGasrfKlazbiKqaNTCvXMgwaU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="94482" id="node94482" data-pisa="biorxiv;307678v1" data-pisa-master="biorxiv;307678" data-apath="/biorxiv/early/2018/04/24/307678.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3056-3739" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Humberto J</span> <span class="nlm-surname">Debat</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/307678 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node94482" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Humberto J Debat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Instituto de Patología Vegetal, Centro de Investigaciones Agropecuarias, Instituto Nacional de Tecnologia Agropecuaria (IPAVE-CIAP-INTA)</span>, 11 de setiembre 4755, Córdoba, Argentina, X5020ICA.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Humberto%2BJ%2BDebat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Debat%20HJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHumberto%2BJ%2BDebat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3056-3739" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Humberto J Debat</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:debat.humberto@inta.gob.ar" class="" data-icon-position="" data-hide-link-title="0">debat.humberto@inta.gob.ar</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/307678v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:94482" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:94482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/307678v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:94482" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:94482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/307678v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:94482" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:94482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/307678v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:94482" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:94482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/307678v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:94482" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:94482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/307678v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:94482" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:94482/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">While RNA viruses thrive with massive structural and functional diversity, their genomes size variation is particularly low, ranging only from ~2-to-33 kb. Here, I present the characterization of RNA sequences corresponding to the first virus associated with <em>Aplysia californica</em>. Genome structure and domain architecture suggest that the identified virus is a novel member of <em>Nidovirales</em>. The proposed aplysia californica nido-like virus (AcNV), with a genome sequence of ca.35,906 nt, represents the longest ever recorded RNA virus yet. Phylogenetic insights indicate that AcNV clusters in a major phylloclade of unclassified invertebrate nidoviruses, <em>Roniviridae</em>, and <em>Mesoniviridae</em>. Basal branching in this emerging cluster could indicate that AcNV is a member of a novel divergent clade within <em>Nidovirales</em>. Further, virus RNA detection in multiple independent studies suggests that AcNV is neurotropic with a broad cell/tissue/organ tropism, supported by AcNV occurrence in diverse organs, including the first detection of a <em>Nidovirales</em> in single specific neurons.</p><div id="sec-1" class="subsection"><p><strong>Highlights</strong></p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-3">-RNA virus genomes reported in the literature are limited at ca. 33.4 kb</p></li><li id="list-item-2"><p id="p-4">-A novel nidovirus was identified in the gastropod mollusk <em>Aplysia californica</em></p></li><li id="list-item-3"><p id="p-5">-The aplysia californica nido-like virus (AcNV) presents a 35.9 kb RNA genome</p></li><li id="list-item-4"><p id="p-6">-AcNV has a broad tropism, is enriched in the CNS, and accumulates in neurons</p></li><li id="list-item-5"><p id="p-7">-The unique features of <em>A. californica</em> enables single-neuron virus dynamics of AcNV</p></li></ul></div></div><div class="section" id="sec-2"><h2 class="">1. Introduction</h2><p id="p-8">A massive flood of RNA sequence data is becoming accessible as a result of the exponential expansion and democratization of high-throughput sequencing platforms. Metatranscriptomics studies are exploring and elucidating the expression profiles of a global plethora of organisms. In tandem, a growing multifaceted RNA virosphere is emerging, uncovering the viral dark matter associated with every single living organism assessed (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Greninger, 2018</a>). Despite the exponential growth of data generation during the last years and the parallel discovery of thousands of new emergent viruses, one crucial characteristic of RNA virus genomes, which has maintained constant in the metagenomics revolution (perhaps the only constant), is an upper-limit on genome length of &lt;34 kb in RNA viruses (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Shi et al., 2018</a>). Ball python nidovirus (<em>Nidovirales, Coronaviridae</em>) presents a 33.4 kb RNA virus genome (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Stenglein et al., 2014</a>) which still remains to be the largest reported to date and has been associated with respiratory disease in experimental infections of ball pythons (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bodewes et al., 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hoon-Hanks et al, 2018</a>). Nidoviruses are positive-stranded RNA enveloped viruses with a monosegmented large genome. The linear nidoviruses genome is infectious, and its RNA molecule is capped and polyadenylated. The order <em>Nidovirales</em> is comprises four different families: <em>Arteiivindae, Mesoniviridae, Coronaviridae</em> and <em>Roniviridae</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams et al., 2017</a>). While genome size of nidoviruses differs significantly amid ca 12.7-33.4 kb, a general conserved genome organization, their replication strategy and sequence similarity in the replicase protein suggest that nidoviruses share a common ancestor (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gorbalenya et al., 2006</a>). Nidoviruses are widespread in humans and numerous animal species, and have been detected in terrestrial and marine mammals, fish, birds, reptiles, insects and crustaceans (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Groot et al., 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nga et al., 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lauber et al., 2012</a>). Interestingly, despite their evident broad host range, only one nidovirus has been associated with mollusks (Phylum Mollusca), more specifically with a <em>Turritella</em> sea snail (<em>Gastropoda</em>) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Shi et al., 2016</a>). The California sea hare (<em>Aplysia californica</em>) is a sea slug marine opisthobranch gastropod mollusk in the family <em>Aplysiidae</em>. This gigantic gastropod (up to 75 cm and 7 kg of weight) and valuable laboratory animal presents a “simple” nervous system of about 20 thousand neurons, distributed in 10 ganglia of ca. two thousand cells each. These neurons are of massive size and present differential pigmentation, which has been useful for specific individual identification. Targeted studies with individual neurons of <em>A. californica</em> had led to the understanding of specific contributions of concerted neurons to simple and complex behaviors in which they are involved. Hallmarks of behavior neurochemistry have been elucidated in this gastropod such as the classic siphon-withdrawal reflex documented by Eric Kandel’s fundamental early works, demonstrating for the first time that environmental cues may generate structural effects on the nervous system (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carew et al., 1972</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kandel, 1976</a>). Here, I present the genomic characterization of the first virus to be associated with <em>A. californica</em>, a tentative novel member of order <em>Nidovirales</em>, with the largest ever-recorded RNA virus genome.</p></div><div class="section" id="sec-3"><h2 class="">2. Results</h2><div id="sec-4" class="subsection"><h3>2.1 Virus discovery in A. californica</h3><p id="p-9">In order to assess the potential RNA virus landscape of <em>A. californica</em>, every available RNA resource at NCBI was employed for virus discovery. 117 RNA NGS libraries, of diverse studies, target organ/tissue/cell, origin or developmental stage, encompassing over 164.54 gigabases were explored. The raw reads were downloaded and subjected to <em>de novo</em> assembly with Trinity v2.6.6 release or the CLC genomics workbench v8.0.3 assembler into library specific transcriptomes which were further subjected to bulk BLASTX searches (E-value &lt; 1e-5) against a local database of refseq virus proteins. The obtained hits were explored by hand. Interestingly, a large 35.9 kb transcript assembled from a single-cell RNAseq library generated from the interneuron L-29 (SRA = SRR3211539) got a significant best hit (E-value = 7e-22; 24 % identity) to the orf1ab gene product (YP_005352837) of <em>White-eye coronavirus HKU16</em>; (Nidovirales; Coronaviridae; <em>Deltacoronavirus</em>). Iterative mapping of the interneuron L-29 reads were employed to polish the identified transcript which was curated into 35,906 nt, derived of 2,235,790 RNA reads (5.86 % of total library reads) giving an average coverage of 6382,4X. Surprisingly, equivalent or shorter contigs sharing ≥99 % identity to the detected transcript were found in 61 additional <em>A. californica</em> generated transcriptomes. Moreover, raw read mapping of all available libraries using as cut-off &gt;10 reads with a ≥99 % identity suggests that this RNA is present in 71 independent NGS libraries, which is roughly 60.6 % of all available <em>A. californica</em> RNA libraries. Additional analyses were carried out to rule out that this transcript could be derived from the expression of an endogenous <em>A. californica</em> gene (see <a id="xref-sec-6-1" class="xref-sec" href="#sec-6">section 2.3</a>).</p></div><div id="sec-5" class="subsection"><h3>2.2 Structural and functional annotation of Aplisya californica nido-like virus (AcNV)</h3><p id="p-10">Further structural characterization and annotation of the detected transcript indicate the presence of two major ORFs (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1.A</a>). ORF1a (67-17,521 nt coordinates) appears to be followed by a putative pseudoknot (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1.E</a>) that could be associated to translation read-through of a UGA stop codon (opal) at position 17,519-21 nt to generate a larger ORF1ab by suppression of termination (67-25,198 nt coordinates). After a 53 nt spacer a second large ORF2 is predicted at 25,252-34,920 nt coordinates. The encoded ORF1ab and ORF2 are flanked by a 67 nt A/U rich (61.2 %) 5’UTR and a 964 nt long 3’UTR followed by a 22 nt Poly (A) tail. This genome organization, while clearly different from the hallmark configuration of nidoviruses, is shared with the only available nido-like virus associated with a gastropod reported yet: Beihai nido-like virus 1 (BNLV-1; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Shi et al., 2016</a>). BNLV-1 presents two large ORFs encoding two polyproteins, and the first polyprotein encoded in ORF1ab is tentatively generated by read-through of a premature stop codon at 7708-7710 nt, instead of a characteristic −1 ribsomal frameshifting, the standard for nidoviruses (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brierley et al., 1987</a>). In the <em>A. californica</em> detected RNA sequence, the ORF1ab encodes a polyprotein (PP1ab) of 8,375 aa. To my knowledge, besides the recently described Diaphorina citri flavi-like virus and Gamboa mosquito virus polyproteins (8,960 aa and 8,572 aa, respectively) this is the third largest ever-predicted protein for any RNA virus yet. Blastp searches of PP1ab against the non-redundant protein sequences (nr) obtained as best hit the orf1ab polyprotein of <em>Bulbul coronavirus HKU11-796</em> (E-value = 5e-36; 23 % identity). ORF2 (25,252-34,920 nt coordinates) encodes a highly divergent polyprotein (PP2), with no evident virus best hits, but a meager similarity (E-value = 0.035) to the spike glycoprotein of <em>Ball phyton nidovirus 1</em>. Further structural and functional <em>in silico</em>annotation of the PP1ab and PP2 proteins evidenced the presence of several domains in a canonical position and a general architecture typical of nidoviruses (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1.A.<em>i</em></a>). Three major hydrophobic regions comprising multiple trans membrane-spanning regions were predicted with TMHMM v2.0 in ORF1a PP (TM1 <sub>1607</sub>L-Y<sub>1820</sub>, TM2 <sub>4174</sub>F-F<sub>4286</sub>, and TM3 <sub>4704</sub>T-Y<sub>4884</sub>) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1.A.<em>iiiii</em></a>). A putative membrane protease like region (PL, proteolysis associated domain; YdiL; E-value = 1.60e-03) was identified among 1603-1760aa positions. Further, between TM2-TM3 a 3C-like proteinase (E.C.3.4.22.-) region was spotted by HHPred (3CL; PDB structure: 3D23_A of <em>Human coronavirus HKU1</em> main protease; Probability: 96; E-value = 0.045) at 4,293-4,699 aa coordinates. This main protease of nidoviruses, flanked at both sides by TM2-TM3, similar to that of picornavirus 3C proteinases, is responsible for the processing of the most conserved part of the replicase polyprotein (ORF1b) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anand et al., 2002</a>). In the ORF1b polyprotein region, HHPred searches identified at 6496-6911 aa coordinates of ORF1ab a RNA-directed RNA polymerase (RdRP, PDB structure: 1RAJ_A; Probability: 96.82; E-value = 9.6e-6; Pfam = pfam00680) and at 7,259-7596 aa coordinates a Middle East respiratory syndrome coronavirus helicase domain (HEL, PDB structure: 5WWP_A; Probability: 99.76; E-value = 6.4e-23). This peculiar position of the RdRP upstream the HEL domain is distinctive, exclusive of nidovirales among ssRNA(+) viruses (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gorbalenya et al., 1989</a>), and has also been reported in tentative ssRNA(+) hypovirus-like mycoviruses (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Marzano et al., 2016</a>). Additionally, between the RdRP and HEL domains, as expected, a multinuclear coronavirus zinc-binding domain (CV-ZBD) was identified (6,963-7,050 aa; score = 10.893; Prosite = PS51653). The predicted CV-ZBD domain contains 10 His/Cys typical residues at equilocal conserved positions of several nidoviruses (<strong>Supp. Fig 2</strong>). The ZBD domain has been indicated as a genetic marker of order <em>Nidovirales</em>, since it has not been identified in any other viral orders (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gorbalenya et al., 1989</a>). The RdRP-ZBD-HEL predicted region of the ORF1ab protein appears to be the more similar to reported nidoviruses (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig 1.D</a>). Lastly, a divergent methyltransferasa region, similar to that of nsp14 or nsp10, exoribonuclease and transferase (MT; PDB structure: 5C8T_B of Human SARS coronavirus guanine-N7 methyltransferase; Probability: 90.11; E-value = 0.12) was found at the 7,642-7,937 aa coordinates. In sum, the predicted ORF1ab protein presents an array of characteristic domains in an architecture typically associated to nidoviruses (NH2-PL-TM1-TM2-3CLpro-TM3-RdRP-ZBD-HEL-MT-COOH) this specific domain order is essential for diverse aspects of the replicative cycle of nidoviruses (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gorbalenya, 2001</a>). Further characterization of ORF2 encoded protein (PP2) suggests a clear divergence between this polyprotein and ordinary nidoviruses. Nevertheless, divergent but consistent signatures were identified which supports its affinity with BNLV-1. ORF2 encodes a 3,224 polyprotein with an N-terminal signal peptide with a predicted cleavage site between residues Ala<sub>21</sub> and Gly<sub>22</sub> (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig.1.A.<em>i-ii</em></a>). HHPred searches identified at 276-471 aa coordinates of PP2 a Trypsin (TryP; E.C.3.4.21.4) serine protease domain (PDB structure: 5KWM_A; Probability: 98.88; E-value = 4.3e-12). At an equilocal position (128-308 aa coordinates), the ORF2 product of BNLV-1 presents a related domain (Trypsin-like serine protease; Superfamily = SSF50494; E-value = 5.9e-07). While proteases involved in post-translational processing of PP1ab are a hallmark of nidoviruses, evolutionary insights of the detected domain suggest that this peptidase is significantly related to the ones found in the digestive system of many vertebrates, where they hydrolyzes diverse proteins. Future studies should explore the potential source of the detected TryP domain, which might be associated to horizontal transmission from an eventual host. In addition, the C-region of ORF2 of BNLV-1 presents an alphavirus glycoprotein like domain (Superfamily = SSF56983; 1,335-1,546 aa). The <em>A. californica</em> ORF2 product presents an analogous, but highly divergent region with similarity with Torovirinae spike glycoprotein (Sgp; 2,185-3,092 aa; FFAS03 = PF17072.3; 17 % id) at the C-region of the predicted PP2, which could be associated with structural functions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structural characterization and phylogenetic insights of aplysia californica nido-like virus (AcNV) A) (i) Genome graphs depicting architecture and predicted gene products of AcNV. Predicted domains are shown in initialized curved rectangles and indicated at their starting coordinate. Flag signal indicates a putative pseudoknot (E) that could be associated to translation read-through of a UGA stop codon (opal, black rectangle) at position 17,519-21 nt to generate the ORF1ab polyprotein by suppression of termination. Pink bars indicate transmembrane predicted regions. Abbreviations: TM1-3, trans-membrane domains 1-3; 3CL, 3C-like proteinase (3CLpro); CV-ZBD, multinuclear coronavirus zinc-binding domain; PL, proteolysis associated domain; RdRP, RNA-dependent RNA polymerase; HEL, helicase domain; MT, methyltransferase-like domain; SP, signal peptide cleavage site; TryP, Trypsinlike serine protease domain. (ii) SignalP-4.1 prediction of the cleavage site (Ala21/Gly22) of ORF2 polyprotein. C-score, raw cleavage site score; S-score, signal peptide score; Y-score, geometric average of the C-score and the slope of the S-score (iii) Detected polymorphic sites in consensus sequences of diverse A. californica bioprojects generated with a &gt;75% frequency call per-base. Variable sites are indicated by black lines in both the genomic RNA sequence track (top) and/or the predicted gene products (below) of each consensus. (iiii) Non-consensus variant frequency as a percentage of total mapped bases per site determined with 625,367,351 base calls of 5,796,298 AcNV virus derived RNA reads. Detailed polymorphisms are available in Supp. Table 4. (iiiii) Multiple membrane-spanning regions of ORF1ab replicase determined with TMHMM v2.0 (left). Intrinsic disorder regions (right) predicted with the disEMBL tool. Pink lines, Remark-465 predictor; turquoise, hot-loops; yellow, loops or coils. B) Maximum likelihood phylogenetic tree based on MAFFT alignments of predicted replicase protein of AcNV (black star) and related Nidovirales. The Arteiiviridae family was used as tree root. Genera of depicted viruses are indicated by colors following (F). Scale bar represents substitutions per site. Node labels are FastTree support values. Silhouettes indicate the reported host of the respective nidovirus. C) Tentative haplotype network based on PopART using the integer neighbor-joining algorithm. Each blue vertex represents a tentative viral haplotype. Branch bars represent the number of mutations between sequences. D) Similarity levels of AcNV and representative members of each Nidovirales family or sub-family replicases expressed as Circoletto diagrams based on BLASTP searches with an E-value of 1e-1 threshold. RPs are depicted clockwise, and sequence similarity is visualized from blue to red ribbons representing low-to-high sequence identity. F) Unrooted radial phylogenetic tree visualization of (B), genera of depicted viruses are indicated by colors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-346955608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structural characterization and phylogenetic insights of aplysia californica nido-like virus (AcNV) &lt;strong&gt;A&lt;/strong&gt;) (&lt;em&gt;i&lt;/em&gt;) Genome graphs depicting architecture and predicted gene products of AcNV. Predicted domains are shown in initialized curved rectangles and indicated at their starting coordinate. Flag signal indicates a putative pseudoknot (&lt;strong&gt;E&lt;/strong&gt;) that could be associated to translation read-through of a UGA stop codon (opal, black rectangle) at position 17,519-21 nt to generate the ORF1ab polyprotein by suppression of termination. Pink bars indicate transmembrane predicted regions. Abbreviations: TM1-3, trans-membrane domains 1-3; 3CL, 3C-like proteinase (3CL&lt;sup&gt;pro&lt;/sup&gt;); CV-ZBD, multinuclear coronavirus zinc-binding domain; PL, proteolysis associated domain; RdRP, RNA-dependent RNA polymerase; HEL, helicase domain; MT, methyltransferase-like domain; SP, signal peptide cleavage site; TryP, Trypsinlike serine protease domain. (&lt;em&gt;ii&lt;/em&gt;) SignalP-4.1 prediction of the cleavage site (Ala&lt;sub&gt;21&lt;/sub&gt;/Gly&lt;sub&gt;22&lt;/sub&gt;) of ORF2 polyprotein. C-score, raw cleavage site score; S-score, signal peptide score; Y-score, geometric average of the C-score and the slope of the S-score (&lt;em&gt;iii&lt;/em&gt;) Detected polymorphic sites in consensus sequences of diverse &lt;em&gt;A. californica&lt;/em&gt; bioprojects generated with a &gt;75% frequency call per-base. Variable sites are indicated by black lines in both the genomic RNA sequence track (top) and/or the predicted gene products (below) of each consensus. (&lt;em&gt;iiii&lt;/em&gt;) Non-consensus variant frequency as a percentage of total mapped bases per site determined with 625,367,351 base calls of 5,796,298 AcNV virus derived RNA reads. Detailed polymorphisms are available in &lt;strong&gt;Supp. Table 4&lt;/strong&gt;. (&lt;em&gt;iiiii&lt;/em&gt;) Multiple membrane-spanning regions of ORF1ab replicase determined with TMHMM v2.0 (left). Intrinsic disorder regions (right) predicted with the disEMBL tool. Pink lines, Remark-465 predictor; turquoise, hot-loops; yellow, loops or coils. &lt;strong&gt;B&lt;/strong&gt;) Maximum likelihood phylogenetic tree based on MAFFT alignments of predicted replicase protein of AcNV (black star) and related &lt;em&gt;Nidovirales&lt;/em&gt;. The &lt;em&gt;Arteiiviridae&lt;/em&gt; family was used as tree root. Genera of depicted viruses are indicated by colors following (&lt;strong&gt;F&lt;/strong&gt;). Scale bar represents substitutions per site. Node labels are FastTree support values. Silhouettes indicate the reported host of the respective nidovirus. &lt;strong&gt;C&lt;/strong&gt;) Tentative haplotype network based on PopART using the integer neighbor-joining algorithm. Each blue vertex represents a tentative viral haplotype. Branch bars represent the number of mutations between sequences. &lt;strong&gt;D&lt;/strong&gt;) Similarity levels of AcNV and representative members of each &lt;em&gt;Nidovirales&lt;/em&gt; family or sub-family replicases expressed as Circoletto diagrams based on BLASTP searches with an E-value of 1e-1 threshold. RPs are depicted clockwise, and sequence similarity is visualized from blue to red ribbons representing low-to-high sequence identity. &lt;strong&gt;F&lt;/strong&gt;) Unrooted radial phylogenetic tree visualization of (&lt;strong&gt;B&lt;/strong&gt;), genera of depicted viruses are indicated by colors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F1.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F1.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-11" class="first-child">Structural characterization and phylogenetic insights of aplysia californica nido-like virus (AcNV) <strong>A</strong>) (<em>i</em>) Genome graphs depicting architecture and predicted gene products of AcNV. Predicted domains are shown in initialized curved rectangles and indicated at their starting coordinate. Flag signal indicates a putative pseudoknot (<strong>E</strong>) that could be associated to translation read-through of a UGA stop codon (opal, black rectangle) at position 17,519-21 nt to generate the ORF1ab polyprotein by suppression of termination. Pink bars indicate transmembrane predicted regions. Abbreviations: TM1-3, trans-membrane domains 1-3; 3CL, 3C-like proteinase (3CL<sup>pro</sup>); CV-ZBD, multinuclear coronavirus zinc-binding domain; PL, proteolysis associated domain; RdRP, RNA-dependent RNA polymerase; HEL, helicase domain; MT, methyltransferase-like domain; SP, signal peptide cleavage site; TryP, Trypsinlike serine protease domain. (<em>ii</em>) SignalP-4.1 prediction of the cleavage site (Ala<sub>21</sub>/Gly<sub>22</sub>) of ORF2 polyprotein. C-score, raw cleavage site score; S-score, signal peptide score; Y-score, geometric average of the C-score and the slope of the S-score (<em>iii</em>) Detected polymorphic sites in consensus sequences of diverse <em>A. californica</em> bioprojects generated with a &gt;75% frequency call per-base. Variable sites are indicated by black lines in both the genomic RNA sequence track (top) and/or the predicted gene products (below) of each consensus. (<em>iiii</em>) Non-consensus variant frequency as a percentage of total mapped bases per site determined with 625,367,351 base calls of 5,796,298 AcNV virus derived RNA reads. Detailed polymorphisms are available in <strong>Supp. Table 4</strong>. (<em>iiiii</em>) Multiple membrane-spanning regions of ORF1ab replicase determined with TMHMM v2.0 (left). Intrinsic disorder regions (right) predicted with the disEMBL tool. Pink lines, Remark-465 predictor; turquoise, hot-loops; yellow, loops or coils. <strong>B</strong>) Maximum likelihood phylogenetic tree based on MAFFT alignments of predicted replicase protein of AcNV (black star) and related <em>Nidovirales</em>. The <em>Arteiiviridae</em> family was used as tree root. Genera of depicted viruses are indicated by colors following (<strong>F</strong>). Scale bar represents substitutions per site. Node labels are FastTree support values. Silhouettes indicate the reported host of the respective nidovirus. <strong>C</strong>) Tentative haplotype network based on PopART using the integer neighbor-joining algorithm. Each blue vertex represents a tentative viral haplotype. Branch bars represent the number of mutations between sequences. <strong>D</strong>) Similarity levels of AcNV and representative members of each <em>Nidovirales</em> family or sub-family replicases expressed as Circoletto diagrams based on BLASTP searches with an E-value of 1e-1 threshold. RPs are depicted clockwise, and sequence similarity is visualized from blue to red ribbons representing low-to-high sequence identity. <strong>F</strong>) Unrooted radial phylogenetic tree visualization of (<strong>B</strong>), genera of depicted viruses are indicated by colors.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Overall, genetic similarity, genomic organization, functional and structural annotation suggest that the identified transcript could be sequence evidence of a new species of RNA virus with cues of <em>Nidovirales</em>. Thus, I propose the name Aplisya californica nido-like virus (AcNV) to the detected putative virus.</p></div><div id="sec-6" class="subsection"><h3>2.3 Genomic evidence supports AcNV as bona fide extant virus associated to A. californica</h3><p id="p-13">Using as query the predicted gene products of AcNV, multiple TBLASTN searches (E-value &gt;1e-1) targeting the <em>A. californica</em> genome assembly AplCal3.0, NCBI accession number GCF_000002075.1 failed to return any significant hits. This result supports the proposition that the AcNV sequence does not derive from the RNA expression of an integrated DNA element sharing similarities with <em>Nidovirales</em>, or contamination of RNA sequencing libraries with <em>A. californica</em> DNA harboring such element, or potential AcNV like endogenous viral elements EVEs (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Katzourakis &amp; Gifford, 2010</a>). Further, the complete raw DNA reads used to generate the <em>A. californica</em> genome were explored in detail (Bioproject PRJNA13635), to rule out that the AcNV sequence could correspond to an unassembled region of the <em>A. californica</em> genome. In this direction, MEGABLAST of the raw DNA reads (E-value &gt;1e-1) with AcNV resulted in no significant hits to the virus sequence. In sum, these data are in concordance with the proposal that the detected AcNV RNA sequence is, in fact, evidence of a <em>bona fide</em> extant virus associated to <em>A. californica</em>.</p></div><div id="sec-7" class="subsection"><h3>2.4 Phylogenetic insights of AcNV</h3><p id="p-14">To entertain the hypothesis that the proposed AcNV is related to the <em>Nidovirales</em> order, phylogenetic insights were generated based on MAFFT multiple amino acid alignments of predicted ORF1ab proteins of accepted and proposed members of <em>Nidovirales</em> (<strong>Supp. Table 1</strong>), followed by FastTree maximum likelihood phylogenetic trees (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig 1.B, F</a>). Several major lineages could be identified in the obtained phylogenetic tree of <em>Nidovirales</em>: (<em>i</em>) A subfamily <em>Coronavirinae</em> (<em>Coronaviridae</em>) lineage which includes exclusively vertebrate viruses, and can be divided in 4 genera (<em>Alpha-Betta-Gamma-Delta-coronavirus</em>). (<em>ii</em>) A lineage of small nidoviruses (<em>Arteviridae</em>) associated also to vertebrates, which has been recently separated into five genera (<em>Dip-Equ-Nes-Por-Sim-arterivirus</em>). (<em>iii</em>) A lineage of fish (<em>Bafinivirus</em>) and snake and mammals viruses (<em>Torovirus</em>) which are included in a diverse <em>Coronaviridae</em> subfamily (<em>Torovirinae</em>). (<em>iiii</em>) Lastly, a cluster of medium sized nidoviruses, enriched in mosquitoes (<em>Mesoniviridae</em>) and a crustacean group of viruses (<em>Roniviridae</em>) in a concerted lineage with divergent crustacean and a mollusk virus which remain unclassified, could be identified. The obtained tree indicates that AcNV unequivocally clusters within the order, in the pivotal major clade composed of recently proposed crustacean and gastropod nidoviruses, the <em>Roniviridae</em>family and <em>Mesoniviridae</em>. The local topology of the tree and basal branching of AcNV could suggest that this <em>A. californica</em> associated virus might be the first member of a new lineage of nidoviruses (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1.B</a>).</p></div><div id="sec-8" class="subsection"><h3>2.5 AcNV predicted prevalence, tropism and variability</h3><p id="p-15">AcNV could be traced to expression sequence tag (EST) libraries generated more than 10 years ago (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Moroz et al., 2006</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fiedler et al., 2010</a>). Over 97 ESTs were further linked to AcNV, presenting highly supported similarity to the virus sequence (E-value ≤ 9e10-72; identity ≥ 92%). The tentatively AcNV derived ESTs were generated from libraries obtained from the central nervous system (CNS) of <em>A. californica</em>, most hits being linked to a specific pedal and pleural ganglia sample library (<strong>Supp. Table 2</strong>). The suggestive hints towards a CNS enrichment inclined me to further study the potential tropism of AcNV.</p><p id="p-16">In light of the unique <em>A. californica</em> biology of well-established model organism, its amazing central nervous system, and the availability of vast genomic resources, I explored the genomic RNA landscape of AcNV in Californica sea hare. The distinguishing neuroanatomy of <em>A. californica</em> and its well-studied neurons offer an exceptional technical platform for RNA dynamics (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Puthanveettil et al., 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Moroz &amp; Kohn, 2010</a>–<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">2013</a>). To tentatively assess cell/tissue and organ tropism based on virus-derived RNA accumulation in <em>A. californica</em>, available California sea hare RNAseq libraries were mapped to AcNV and normalized as Fragments Per Kilobase of transcript per Million mapped reads (FPKM). Virus RNA transcripts were detected in an outstanding diversity of target tissue and organs, suggesting e pervasive tissue tropism of AcNV (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2</a>; <strong>Supp. Table 3</strong>). At the adult organ level (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2.A, B</a> right panel) AcNV was detected in chemoreceptive areas (rhinophores &amp; tentacles), salivary glands, gills, heart, additional muscles (body wall, buccal and penial), hermaphroditic glands, ovotestis, hepatopancreas and other digestive glands. Interestingly, AcNV appeared to be highly enriched in the CNS. The availability of CNS tissue and single-cell RNAseq data allowed the surveillance of specific tropism at the cellular level of AcNV (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2.B-E</a>). AcNV accumulated at relatively higher levels in the left excitatory interneuron 29 (L29), the left motor neuron 7 (L7), the right serotonergic cerebral ganglion metacerebral neuron (MCC-R) and the right abdominal ganglion cholinergic neuron 2 (R2). The R2 neuron and its axonal pathway is highly conserved and thus can be consistently identified in closely related species of the genus <em>Aplysia</em>. R2 measures up to 1.1 mm in diameter and is the largest neuron in the animal kingdom (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Moroz &amp; Kohn, 2013</a>). AcNV was found at lower relative levels in the left abdominal ganglion motor neuron (LFS), the sensory neuron cluster and the left MCC.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AcNV tentative cell, tissue and organ tropism based on virus-derived RNA accumulation in A. californica. A) Graphs bars expressing virus RNA transcript levels assayed in diverse expression profile libraries derived from distinct A. californica tissues or organs. Values are expressed as Fragments Per Kilobase of transcript per Million mapped reads (FPKM). B) A. californica diagram (right) highlighting the virus RNA positive tissue/organs and a schematic diagram of the central ganglionic nervous system of A. californica (left) indicating specific neurons which presented AcNV virus RNA. RNA levels are visualized as diamonds colored green-yellow-red representing low-to-high relative RNA virus accumulation. Abbreviations: CNS, central nervous system; SNC, sensory neuron cluster; R2-R15, right abdominal ganglion cholinergic 2 and 15 neurons; L7, left motor neuron 7; MCC, serotonergic cerebral ganglion metacerebral neuron; LFS, left abdominal ganglion motor neuron; LE, left excitatory abdominal sensory neuron; L29, left excitatory interneuron 29; SNV-VC, pleural ventrocaudal sensory neuron. C) Graphs bars expressing virus RNA transcript levels assayed in diverse single-cell expression profile libraries derived from specific A. californica individual neurons. Virus FPKM values corresponding to each A. californica sample are available as Supp. Table 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-346955608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;AcNV tentative cell, tissue and organ tropism based on virus-derived RNA accumulation in &lt;em&gt;A. californica&lt;/em&gt;. &lt;strong&gt;A&lt;/strong&gt;) Graphs bars expressing virus RNA transcript levels assayed in diverse expression profile libraries derived from distinct &lt;em&gt;A. californica&lt;/em&gt; tissues or organs. Values are expressed as Fragments Per Kilobase of transcript per Million mapped reads (FPKM). &lt;strong&gt;B&lt;/strong&gt;) &lt;em&gt;A. californica&lt;/em&gt; diagram (right) highlighting the virus RNA positive tissue/organs and a schematic diagram of the central ganglionic nervous system of &lt;em&gt;A. californica&lt;/em&gt; (left) indicating specific neurons which presented AcNV virus RNA. RNA levels are visualized as diamonds colored green-yellow-red representing low-to-high relative RNA virus accumulation. Abbreviations: CNS, central nervous system; SNC, sensory neuron cluster; R2-R15, right abdominal ganglion cholinergic 2 and 15 neurons; L7, left motor neuron 7; MCC, serotonergic cerebral ganglion metacerebral neuron; LFS, left abdominal ganglion motor neuron; LE, left excitatory abdominal sensory neuron; L29, left excitatory interneuron 29; SNV-VC, pleural ventrocaudal sensory neuron. C) Graphs bars expressing virus RNA transcript levels assayed in diverse single-cell expression profile libraries derived from specific &lt;em&gt;A. californica&lt;/em&gt; individual neurons. Virus FPKM values corresponding to each &lt;em&gt;A. californica&lt;/em&gt; sample are available as &lt;strong&gt;Supp. Table 3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F2.medium.gif" width="402" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F2.medium.gif" width="402" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/24/307678/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-17" class="first-child">AcNV tentative cell, tissue and organ tropism based on virus-derived RNA accumulation in <em>A. californica</em>. <strong>A</strong>) Graphs bars expressing virus RNA transcript levels assayed in diverse expression profile libraries derived from distinct <em>A. californica</em> tissues or organs. Values are expressed as Fragments Per Kilobase of transcript per Million mapped reads (FPKM). <strong>B</strong>) <em>A. californica</em> diagram (right) highlighting the virus RNA positive tissue/organs and a schematic diagram of the central ganglionic nervous system of <em>A. californica</em> (left) indicating specific neurons which presented AcNV virus RNA. RNA levels are visualized as diamonds colored green-yellow-red representing low-to-high relative RNA virus accumulation. Abbreviations: CNS, central nervous system; SNC, sensory neuron cluster; R2-R15, right abdominal ganglion cholinergic 2 and 15 neurons; L7, left motor neuron 7; MCC, serotonergic cerebral ganglion metacerebral neuron; LFS, left abdominal ganglion motor neuron; LE, left excitatory abdominal sensory neuron; L29, left excitatory interneuron 29; SNV-VC, pleural ventrocaudal sensory neuron. C) Graphs bars expressing virus RNA transcript levels assayed in diverse single-cell expression profile libraries derived from specific <em>A. californica</em> individual neurons. Virus FPKM values corresponding to each <em>A. californica</em> sample are available as <strong>Supp. Table 3</strong>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">In addition to adult organ tissues and cells, I was able to detect AcNV in libraries generated from individuals at very early developmental stages (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heyland et al., 2011</a>) such as early cleavage (4-7 h after egg masses are laid) and third day individuals (1-4 cells; transition from trochophore to veliger body plan). AcNV virus FPKM values corresponding to each <em>A. californica</em> sample are available as <strong>Supp. Table 3</strong>.</p><p id="p-19">The vast collection of NGS data allowed both the generation of consensus sequences for each bioproject, based on &gt;75 % base calls, and the identification of variable sites at each position (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig 1.A.<em>iiii</em></a>). The analyses of every NGS library allowed insights into the variation within-host and eventual evolutionary hints at a genome wide level. The predicted AcNV <em>bona fide</em> variation was noticeably spotted across the genome with specific regions, such as the 3’ UTR showing greater levels of nonconsensus calls (<strong>Supp. Table 4</strong>). In order to explore evolutionary relationships among potentially distinct virus lineage populations derived from diverse <em>A. californica</em> studies harboring AcNV, haplotype networks were inferred based on multiple MAFFT alignments of complete consensus AcNV virus sequences derived from <em>A. californica</em>Bioprojects employing the PopART (Population Analysis with Reticulate Trees) software. The haplotype networks suggest that the variability among consensu sequences of the diverse AcNV virus positive studies is non-significant (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1.C</a>; <a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig.1.A<em>iii</em></a>).</p></div></div><div class="section" id="sec-9"><h2 class="">3. Discussion</h2><div id="sec-10" class="subsection"><h3>3.1 AcNV as a divergent nido-like virus</h3><p id="p-20">The AcNV genomic plan, predicted gene products, canonical array of conserved domain of nidoviruses in the ORF1ab protein (NH2-PL-TM1-TM2-3CLpro-TM3-RdRP-ZBD-HEL-MT-COOH), and overall sequence similarity with reported nidoviruses, were considered to propose that the detected RNA sequence corresponded to a nido-like <em>bona fide</em> virus. Further, phylogenetic insights indicated that AcNV unequivocally clustered within the order <em>Nidovirales</em>, in the pivotal major clade composed of recently proposed crustacean and gastropod nidoviruses, the <em>Roniviridae</em> family and <em>Mesoniviridae</em>. The basal branching of AcNV could suggest that this <em>A. californica</em> associated virus might be the first member of a new lineage of nidoviruses. The discovery of new related viruses is fundamental to unravel the emergent viral dark matter that would license a better understanding of the evolutionary relations and concomitant taxonomic assignment of AcNV in the context of <em>Nidovirales</em>. Given the evident association that could be traced in terms of evolutionary links between <em>Nidovirales</em> and their hosts, perhaps an interesting follow up strategy would be the (overlooked) exploration of mollusks as source of viral diversity.</p></div><div id="sec-11" class="subsection"><h3>3.2 RNA virus genome size constrains, AcNV and Nidovirales</h3><p id="p-21">AcNV presents the largest RNA virus genome ever recorded (<strong>Supp. Fig 1.A-C</strong>). At ca. 35.9 kb, AcNV is over 2.4 kb longer that the second largest: Ball phyton nidovirus 1 (BPNV-1) RNA genome (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Stenglein et al., 2014</a>). When comparing the 84 refseq genomes of nidovirus available at NCBI and their taxonomic assignment to specific <em>Nidovirales</em> families or sub-families, three major groups became evident: (i) <em>Arteriviridae</em> members, “small nidoviruses”; (ii) <em>Mesoniviridae</em>, “intermediate nidoviruses”, and (iii) <em>Coronaviridae</em> and <em>Ronaviridae</em> “large nidoviruses” (<strong>Supp. Fig 1.A-B</strong>). Within the <em>Coronaviridae</em> group, particular dynamics in size may be observed such as a wider pattern of longitudes of subfamily <em>Torovirinae</em> members presenting a either ca. 20 kb genomes or larger ca. 32-33 kb ones, the latter being the snake-hosted BPNV-1 and Morelia viridis nidovirus. On the other hand, sub-family <em>Coronavirinae</em>members present a more restricted pattern of genome length, which may be grouped by genus, from <em>Deltacoronavirus</em> to <em>Betacoronavirus</em> members (<strong>Supp. Fig 1.A</strong>). It is possible that the discovery of new <em>Ronaviridae</em> like viruses would help complement this emergent grouping by RNA genome size. The placement of AcNV in this context is intriguing; the similar and only other mollusk nido-like virus BNLV-1 was reported to have a much shorter 20.2 kb RNA genome. Nevertheless, there are some hints that may suggest that BNLV-1 could be longer than reported: The described BNLV-1 sequence NC_032496 does not present any stop codon interrupting the ongoing ORF1a, thus the suggested translation start site and the reported 5’UTR could correspond to a truncated coding region of ORF1aPP. In this direction, while nidoviruses ORF1a is typically much longer than ORF1b, ORF1a of BNLV-1 is about the same size as ORF1b. Perhaps future studies should revisit BNLV-1 genome sequence.</p><p id="p-22">Given the estimated large size of AcNV genome, and the apparent constrains in terms of longer RNA viruses, the underlying basis of this discernible bar is under ongoing assessment. Probable causes of this size limit are under discussion, and the most plausible explanation could be associated with the high rate of RNA virus mutation associated with limited mechanisms of proof reading and repair, concomitant with high rates of replication (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Holland, 1982</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Denison et al., 2011</a>). Supporting these assumptions, in parallel, there appears to be a correlation between small genome DNA viruses and high rate of mutation. In evolutionary terms, mutation rates could be associated not only to polymerase fidelity, but also to viral biology, genome architecture and to replication speed (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Duffy et al., 2008</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Campillo-Balderas et al., 2015</a>). The size limit of RNA viruses is not specifically associated to monosegmented viruses. As a matter of fact, the largest members of the <em>Reoviridae</em> family, encompassing 9 to 12 dsRNA segments which added together grasp a total genome of ca. 32 kb, mirror the apparent size limit of <em>Nidovirales</em> and RNA viruses in general (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dolja &amp; Koonin, 2018</a>). From and evolutionary perspective, it was recently suggested that the nido-like clade of animal viruses emerged in early invertebrates and through horizontal virus transmission then colonized vertebrates. This proposition was based on the relative small genome size of deep-rooted arthropod nidoviruses, and the observation that the largest genomes correspond to vertebrate coronaviruses, postulating that gene incorporation and layering (and concomitant genome length growth) succeeded during the co-evolution of nidoviruses and vertebrates (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Dolja &amp; Koonin, 2018</a>). The discovery here of the tentative AcNV associated to a mollusk, with the largest RNA genome of nidoviruses yet, with more affinity to intermediate-long <em>Roniviridae</em> than to larger coronaviruses, implies that the evolutionary history of <em>Nidovirales</em> could be more complex than expected.</p></div><div id="sec-12" class="subsection"><h3>3.3 A non-canonicalgenome organization of AcNV</h3><p id="p-23">One striking difference observed in AcNV and BNLV-1 genomes denotes a fundamental contrast to the predicted genome organization associated to nidoviruses, which consistently harbor at their 5’ region two large partially overlapping ORFs. Translation of ORF1a yields PP1a, and in a third of cases, codon slippage at the ORF1a/b overlap register the −1 reading frame, and continue with the translation of ORF1b (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">De Groot et al., 2012</a>). On the other hand, BNLV-1 and AcNV appear to generate ORF1ab by means of suppression of termination of a premature stop codon after the ORF1a region. Perhaps it is worth mentioning here, that at a final mean coverage of 17,416X obtained of over 62 million viral RNA nt from 71 different <em>A. californica</em> studies, the proposed AcNV genome organization is highly supported and most probably not derived from a misassembly which could have resulted in the misidentification of overlapping ORFs and RFS signals. The identification of more divergent nido-like viruses may help to determine whether this is a conserved characteristic of mollusk associated nidoviruses or just anecdotal.</p></div><div id="sec-13" class="subsection"><h3>3.4 AcNV variability assessment should be complemented with additional data</h3><p id="p-24">Given the considerable number of viral reads, a tentative assessment of AcNV variability was generated. Polymorphic sites were defined with a conservative estimated frequency of at least ≥0.05 and a Phred score of 20 or higher to filter raw reads. Considering that sequencing errors of the diverse platforms reach at most ~1 % (Ion Torrent data), thus a 5% frequency threshold enables the detection of probable true variants in the virus RNA genome with high confidence. <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cotten et al (2013)</a>, with a similar strategy, explored virus variants of human betacoronavirus using as threshold 1 %. A 5% frequency threshold was used by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Van den Hoecke et al (2015)</a> to assess the variation landscape of Influenza virus A. The detected variability among bioprojects appears to be non-significant or within the levels of intra-host variability. While dispersed in time, most NGS libraries were generated from lab reared animals from the same provider. Thus it is tempting to suggest (this is pure speculation) that the low intrinsic variability observed could be linked with a single virus lineage maintained inadvertently through the years at the breeder facilities.</p></div><div id="sec-14" class="subsection"><h3>3.5 AcNV has a broad tropism, is enriched in the CNS, and accumulates in neurons</h3><p id="p-25">The availability of a public collection of NGS RNA libraries of the model organism <em>A. californica</em>, allowed the tentative survey of the genomic RNA landscape of AcNV. Virus RNA transcripts were detected in an outstanding diversity of target tissue and organs, suggesting e pervasive tissue tropism, showing an emergent enrichment of AcNV in the CNS. Single-cell RNAseq data allowed the surveillance of AcNV at the cellular level, including not only specific types of neuron (e.g. abdominal ganglia neurons) but also the detection of AcNV in well-defined individual neurons of <em>A. californica</em> (e.g. R2, L29, R15). Interestingly, AcNV RNA accumulated at high levels in R2, which is the largest known neuron in the animal kingdom, offering an outstanding and unprecedented model for the study of virus dynamics at the single cell level. Besides their accessibility, ease to identify and size, each R2 neuron may yield over 1.9ug of RNA allowing to amplification-free RNA profiling at a single-neuron level (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Moroz &amp; Kohn, 2013</a>). The standard of single-cell RNAseq preparation in any other organism cells, which have picogram levels of nucleic acids, is a robust amplification step, which may lead to irregular coverage, noise and imprecise quantification of sequencing data (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Islam et al., 2011</a>). The potential to conduct expression analyses on individual neurons of <em>A. californica</em> without the need of amplification, in naked-eye visible neurons, provide an elegant platform for genomics and physiological analysis within well-defined cellular networks: an ideal pathosystem to address single cell virus dynamics. It is tempting to suggest that the possibility of amalgamation of the study of this novel nido-like virus and a biological model system such as <em>A. californica</em>with a vast research toolkit, would allow the rapid deployment of modern molecular tools, which could redound in a rapid advancement in the biology of this entity. An impressive example of that is the recent integration of the house flies pathogen <em>Entomophtora muscae</em> into <em>Drosophila melanogaster</em>, which resulted in major advancements in the neurochemistry, genomics and neurobiological aspects of this fungal pathogen (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Elya et al., 2017</a>). In retrospective, it is perhaps worth contemplating the possibility that this hidden entity, present with certainty in samples of more than decade-old studies, might have influenced the outcome of any of the uncountable works published in the last 50 years. It is in this scenario that future studies should eagerly focus on the specific potential effects of this virus and its host. This invaluable information, missing in the literature, would shed light on the evolutionary history of the interaction of this virus and <em>A. californica</em>.</p></div></div><div class="section" id="sec-15"><h2 class="">4. Materials and methods</h2><div id="sec-16" class="subsection"><h3>4.1 Virus discovery</h3><p id="p-26">In order to assess the potential RNA virus landscape of <em>A. californica</em>, every available RNA resource at NCBI was employed for virus discovery. 117 NCBI SRA accessions of raw RNA sequencing reads were explored, obtained by 454 GS FLX and 20, Illumina HiSeq 2000, Illumina Genome Analyzer II and Ion Torrent PGM sequencing platforms, which are accessible at: <a href="https://www.ncbi.nlm.nih.gov/sra/?term=txid6500[Organism:noexp">https://www.ncbi.nlm.nih.gov/sra/?term=txid6500[Organism:noexp</a>]. In addition, nine tissue-specific Transcriptome Shotgun Assemblies (TSA) were examined (GenBank accession numbers: GBCZ01000001, GBDA01000001, GBBV01000001, GBBW01000001, GBBE01000001, GAZL01000001, GBAV01000001, GBBG01000001, GBAQ01000001). Lastly, a collection of 255,605 <em>A. californica</em> expressed sequence tags (ESTs) of diverse developmental stages, tissues and treatments (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Moroz et al, 2006</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Fiedler et al., 2010</a>) were retrieved from <a href="https://www.ncbi.nlm.nih.gov/nucest/?term=txid6500%5bOrganism:noexp]">https://www.ncbi.nlm.nih.gov/nucest/?term=txid6500%5bOrganism:noexp]</a>. The multiplatform RNA sequencing reads were de novo assembled with Trinity (version 2.5.1) <a href="https://github.com/trinityrnasea/trinityrnasea/wiki">https://github.com/trinityrnasea/trinityrnasea/wiki</a> using standard parameters (Illumina data) and/or with the <em>de novo</em> assembly tool of CLC Genomics Workbench (v 8.0.0) with default settings (454 and Ion Torrent data). The obtained metatranscriptomes were assessed by bulk searches on a local server against a refseq virus protein database generated with data available at <a href="http://ftp://ftp.ncbi.nlm.nih.gov/refseq/release/viral/viral.1.protein.faa.gz">ftp://ftp.ncbi.nlm.nih.gov/refseq/release/viral/viral.1.protein.faa.gz</a>. BLASTX with an expected value of 10e-5 was used as threshold, and hits were explored by hand. Tentative virus contigs were curated by iterative mapping of reads using Bowtie2 <a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a> virus Fragments Per Kilobase of transcript per Million mapped reads (FPKM) were estimated with Cufflinks 2.2.1 <a href="http://cole-trapnelllab.github.io/cufflinks/releases/v2.2.1">http://cole-trapnelllab.github.io/cufflinks/releases/v2.2.1/</a>. The robust <em>A. californica</em> genome assembly AplCal3.0 (GenBank accession number GCF_000002075.1) was explored by TBLASTN searches (E-value &gt;1e-1) using as query the predicted protein products of AcNV, in order to identify potential endogenous viral elements EVEs (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Katzourakis &amp; Gifford, 2010</a>) with similarity with AcNV. In addition, raw DNA reads used to generate the <em>A. californica</em> genome (Bioproject PRJNA13635) were explored in detail by MEGABLAST to rule out potential unassembled EVEs missed in AplCal3.0 genome assembly.</p></div><div id="sec-17" class="subsection"><h3>4.2 Virus sequence annotation</h3><p id="p-27">Virus annotation was instrumented as described in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Debat (2017)</a>, briefly, virus ORFs were predicted with ORFfinder (<a href="https://www.ncbi.nlm.nih.gov/orffinder/">https://www.ncbi.nlm.nih.gov/orffinder/</a>) translated gene products were assessed by InterPro (<a href="https://www.ebi.ac.uk/interpro/search/sequence-search">https://www.ebi.ac.uk/interpro/search/sequence-search</a>) and NCBI Conserved domain database v3.16 (<a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi">https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi</a>) to predict domain presence and architecture. Further, HHPred and HHBlits as implemented in (<a href="https://toolkit.tuebingen.mpg.de/#/tools/">https://toolkit.tuebingen.mpg.de/#/tools/</a>) were employed for domain prediction. Additionally, the Structural Classification of Proteins 2 tool available at (<a href="http://scop.mrc-lmb.cam.ac.uk/scop/">http://scop.mrc-lmb.cam.ac.uk/scop/</a>) was used to complement annotation of divergent predicted proteins by hidden Markov models, and the Fold &amp; Function Assignment server (<a href="http://ffas.sanfordburnham.org/ffas-cgi/cgi/document.pl">http://ffas.sanfordburnham.org/ffas-cgi/cgi/document.pl</a>) was used for the detection of remote homologies beyond the reach of other sequence comparison methods such as PSI-Blast. Pseudoknots were predicted with the Dotknot tool available at (<a href="http://dotknot.csse.uwa.edu.au/">http://dotknot.csse.uwa.edu.au/</a>) and visualized with the VARNA 3.93 applet (<a href="http://varna.lri.fr/">http://varna.lri.fr/</a>). Cleavage sites for signal peptides were predicted with the SignalP 4.1 Server (<a href="http://www.cbs.dtu.dk/services/SignalP">http://www.cbs.dtu.dk/services/SignalP</a>). Prediction of transmembrane helices was performed with TMHMM v2.0 server (<a href="http://www.cbs.dtu.dk/services/TMHMM/">http://www.cbs.dtu.dk/services/TMHMM/</a>). Intrinsic Protein Disorder regions were predicted with the disEMBL tool as implemented in <a href="http://dis.embl.de/">http://dis.embl.de/</a> using standard parameters (predictors Remark-465, beta aggregation with Tango, low-complexity with CAST).</p></div><div id="sec-18" class="subsection"><h3>4.3 Phylogenetic insights</h3><p id="p-28">Phylogenetic insights based on predicted ORF1ab proteins were generated by MAFTT 7 <a href="https://mafft.cbrc.jp/alignment/software/">https://mafft.cbrc.jp/alignment/software/</a> multiple amino acid alignments (BLOSUM62 scoring matrix) using as best-fit algorithm E-INS-I which drives local alignment with generalized affine gap costs, and thus it is applicable for large replicases where diverse domains are dispersed among several highly divergent regions. The aligned proteins were subsequently used as input for FastTree 2.1.5 maximum likelihood phylogenetic trees (best-fit model = JTT-Jones-Taylor-Thorton with single rate of evolution for each site = CAT) computing local support values with the Shimodaira-Hasegawa test (SH) and 1,000 tree resamples <a href="http://www.microbesonline.org/fasttree/">http://www.microbesonline.org/fasttree/</a>. The FreeBayes v0.9.18S tool, using as minimum variant frequency ≥0.05 and additional standard parameters was employed for SNPs prediction <a href="https://github.com/ekg/freebayes">https://github.com/ekg/freebayes</a>. Results were integrated and visualized in the Geneious 8.1.9 platform (Biomatters Ltd.). Median haplotype networks were constructed with PopART version 1.7 (<a href="http://popart.otago.ac.nz">http://popart.otago.ac.nz</a>) using the integer neighbor-joining algorithm based on MAFFT alignments of complete AcNV virus genome consensus sequences (&gt;75% frequency call per-base) determined for <em>A. califonica</em> diverse Bioprojects.</p></div><div id="sec-19" class="subsection"><h3>4.4 Data availability</h3><p id="p-29">The AcNV virus (EK) RNA sequence determined in this study is available in GenBank under accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG878985&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom">MG878985</a> [not yet released to the public, is included as supplementary material of this manuscript submission.].</p></div></div><div class="section" id="sec-20"><h2 class="">Funding</h2><p id="p-31">This research did not receive any specific grant from funding agencies in the public, commercial, or not-for-profit sectors.</p></div><div class="section" id="sec-21"><h2 class="">Conflicts of interest</h2><p id="p-32">The author declares no conflict of interest.</p></div><div id="sec-22" class="subsection"><p id="p-33"><strong>Supplementary Figure 1.</strong> Genome size in <em>Nidovirales</em> <strong>A</strong>) Intervals of RNA genome length in kb of families, sub-families, and genera of <em>Nidovirales</em> members. “Small nidoviruses” are represented by the <em>Arteviridae</em> family, “intermediate nidoviruses” by <em>Mesoniviridae</em>, and <em>Roniviridae</em> and <em>Coronaviridae</em> are considered “large nidoviruses”. <strong>B</strong>) Scaled genome diagrams of the 84 Refseq genomes of nidovirus available at NCBI and AcNV on top. Letters indicate taxonomic assignment to specific <em>Nidovirales</em> families or sub-families. The black line indicates the top 10 largest genomes, which are magnified in (<strong>C</strong>) AcNV EK sequence is over 2.4 kb longer than the largest ever reported RNA virus.</p><p id="p-34"><strong>Supplementary Figure 2.</strong> Multinuclear coronavirus zinc-binding domain (CV-ZBD; Prosite =PS51653) ClustalW aa alignment of AcNV and similar nido-like viruses. While divergent, the predicted CV-ZBD domain of AcNV contains 10 His/Cys typical residues (asterisks) at equilocal conserved positions of several nidoviruses.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-30">I would like to express a sincere gratitude to the generators of the amazing underlying data used for this work: Dr. Leonid L. Moroz and Dr. Eric R Kandel. By following open access practices and supporting accessible raw sequence data in public repositories available to the research community, they and their teams have promoted the generation of new knowledge and ideas. Additional thanks to David Karlin for helpful comments. Special thanks to Elba Villanueva for critical reading of the manuscript.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.1" data-doi="10.1007/s00705-017-3358-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefkowitz</span>  <span class="cit-name-given-names">E.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">A.M.Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrach</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">R.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">N.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kropinski</span>  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">J.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mushegian</span>  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabanadzovic</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanfafon</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">S.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Varsani</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zerbini</span>  <span class="cit-name-given-names">F.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Changes to taxonomy and the international code of virus classification and Nomenclature ratified by the international committee on taxonomy of viruses (2017)</span>. <abbr class="cit-jnl-abbrev">Arch. Virol.</abbr> <span class="cit-vol">162</span>, <span class="cit-fpage">2505</span>–<span class="cit-lpage">2538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch.%2BVirol.%26rft.volume%253D162%26rft.spage%253D2505%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-017-3358-5%26rft_id%253Dinfo%253Apmid%252F28434098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-017-3358-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28434098&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.2" data-doi="10.1093/emboj/cdf327"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anand</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palm</span>,  <span class="cit-name-given-names">G. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mesters</span>,  <span class="cit-name-given-names">J. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>,  <span class="cit-name-given-names">S. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α◻helical domain</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">13</span>), <span class="cit-fpage">3213</span>–<span class="cit-lpage">3224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DAnand%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D13%26rft.spage%253D3213%26rft.epage%253D3224%26rft.atitle%253DStructure%2Bof%2Bcoronavirus%2Bmain%2Bproteinase%2Breveals%2Bcombination%2Bof%2Ba%2Bchymotrypsin%2Bfold%2Bwith%2Ban%2Bextra%2B%257Balpha%257D-helical%2Bdomain%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf327%26rft_id%253Dinfo%253Apmid%252F12093723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTMvMzIxMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzI0LzMwNzY3OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.3" data-doi="10.1099/vir.0.068700-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bodewes</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lempp</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schurch</span>,  <span class="cit-name-given-names">A. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Habierski</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamers</span>,  <span class="cit-name-given-names">M.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Smits</span>,  <span class="cit-name-given-names">S. L.</span></span>, <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Novel divergent nidovirus in a python with pneumonia</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr>, <span class="cit-vol">95</span> (<span class="cit-issue">11</span>), <span class="cit-fpage">2480</span>–<span class="cit-lpage">2485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.068700-0%26rft_id%253Dinfo%253Apmid%252F25063552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.068700-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25063552&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boursnell</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Binns</span>,  <span class="cit-name-given-names">M. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bilimoria</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blok</span>,  <span class="cit-name-given-names">V. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">T. D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Inglis</span>,  <span class="cit-name-given-names">S. C.</span></span> (<span class="cit-pub-date">1987</span>). <span class="cit-article-title">An efficient ribosomal frameLshifting signal in the polymeraseLencoding region of the coronavirus IBV</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">3779</span>–<span class="cit-lpage">3785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DBrierley%26rft.auinit1%253DI%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D3779%26rft.epage%253D3785%26rft.atitle%253DAn%2Befficient%2Bribosomal%2Bframe-shifting%2Bsignal%2Bin%2Bthe%2Bpolymerase-encoding%2Bregion%2Bof%2Bthe%2Bcoronavirus%2BIBV%26rft_id%253Dinfo%253Apmid%252F3428275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3428275&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K948600030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Campillo-Balderas</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lazcano</span>,  <span class="cit-name-given-names">A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Becerra</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Viral genome size distribution does not correlate with the antiquity of the host lineages</span>. <abbr class="cit-jnl-abbrev">Front. Ecol. Evol.</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BEcol.%2BEvol.%26rft.volume%253D3%26rft.spage%253D143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.6" data-doi="10.1126/science.175.4020.451"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carew</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pinsker</span>,  <span class="cit-name-given-names">H. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kandel</span>,  <span class="cit-name-given-names">E. R.</span></span>, <span class="cit-pub-date">1972</span>. <span class="cit-article-title">Long-term habituation of a defensive withdrawal reflex in Aplysia</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">175</span>(<span class="cit-issue">4020</span>), <span class="cit-fpage">451</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCarew%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D175%26rft.issue%253D4020%26rft.spage%253D451%26rft.epage%253D454%26rft.atitle%253DLong-Term%2BHabituation%2Bof%2Ba%2BDefensive%2BWithdrawal%2BReflex%2Bin%2BAplysia%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.175.4020.451%26rft_id%253Dinfo%253Apmid%252F17731371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIxNzUvNDAyMC80NTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8yNC8zMDc2NzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.7" data-doi="10.3201/eid1905.130057"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotten</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">T. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palser</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrova</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">P.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Kellam</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Full-genome deep sequencing and phylogenetic analysis of novel human betacoronavirus</span>. <abbr class="cit-jnl-abbrev">Emerging Infect. Dis.</abbr>, <span class="cit-vol">19</span> (<span class="cit-issue">5</span>), <span class="cit-fpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfect.%2BDis.%26rft.volume%253D19%26rft.spage%253D736%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1905.130057%26rft_id%253Dinfo%253Apmid%252F23693015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1905.130057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23693015&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-307678v1.8"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">A.M.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Adams</span>,  <span class="cit-name-given-names">M.J.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Carstens</span>,  <span class="cit-name-given-names">E.B.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Lefkowitz</span>,  <span class="cit-name-given-names">E.J.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">De Groot</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowley</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faaberg</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-pub-date">2012</span>. <span class="cit-chapter-title">Order - Nidovirales</span>. In: <span class="cit-ed"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Adams</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Carstens</span>,  <span class="cit-name-given-names">E.B.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lefkowitz</span>,  <span class="cit-name-given-names">E.J.</span></span> (Eds.), <span class="cit-source">Virus Taxonomy: Classification and Nomenclature of Viruses: Ninth Report of the International Committee on Taxonomy of Viruses.</span> <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">San Diego</span>, pp. <span class="cit-fpage">784</span>–<span class="cit-lpage">794</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.9" data-doi="10.3389/fmicb.2017.02097"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Debat</span>,  <span class="cit-name-given-names">H.J.</span></span>, <span class="cit-pub-date">2017</span>. <span class="cit-article-title">An RNA Virome associated to the Golden Orb-weaver Spider Nephila clavipes</span>. <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr>, <span class="cit-vol">8</span>: <span class="cit-fpage">2097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D8%26rft.spage%253D2097%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2017.02097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2017.02097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.10" data-doi="10.4161/rna.8.2.15013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denison</span>,  <span class="cit-name-given-names">M. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donaldson</span>,  <span class="cit-name-given-names">E. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckerle</span>,  <span class="cit-name-given-names">L. D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S.</span></span>, <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Coronaviruses: an RNA proofreading machine regulates replication fidelity and diversity</span>. <abbr class="cit-jnl-abbrev">RNA Biol.</abbr>, <span class="cit-vol">8</span> (<span class="cit-issue">2</span>), <span class="cit-fpage">270</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRNA%2BBiol%26rft.aulast%253DDenison%26rft.auinit1%253DM.%2BR.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D270%26rft.epage%253D279%26rft.atitle%253DCoronaviruses%253A%2Ban%2BRNA%2Bproofreading%2Bmachine%2Bregulates%2Breplication%2Bfidelity%2Band%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.8.2.15013%26rft_id%253Dinfo%253Apmid%252F21593585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.8.2.15013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21593585&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289934300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.11" data-doi="10.1016/j.virusres.2017.10.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolja</span>,  <span class="cit-name-given-names">V. V.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Koonin</span>,  <span class="cit-name-given-names">E. V.</span></span>, <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Metagenomics reshapes the concepts of RNA virus evolution by revealing extensive horizontal virus transfer</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, <span class="cit-vol">244</span>, <span class="cit-fpage">36</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D244%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.10.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.10.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.12" data-doi="10.1038/nrg2323"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duffy</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shackelton</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Rates of evolutionary change in viruses: patterns and determinants</span>. <abbr class="cit-jnl-abbrev">Nature Rev Genet.</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">267</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDuffy%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D267%26rft.epage%253D276%26rft.atitle%253DRates%2Bof%2Bevolutionary%2Bchange%2Bin%2Bviruses%253A%2Bpatterns%2Band%2Bdeterminants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2323%26rft_id%253Dinfo%253Apmid%252F18319742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319742&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elya</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lok</span>,  <span class="cit-name-given-names">T. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spencer</span>,  <span class="cit-name-given-names">Q. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCausland</span>,  <span class="cit-name-given-names">H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Eisen</span>,  <span class="cit-name-given-names">M. B.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">A fungal pathogen that robustly manipulates the behavior of Drosophila melanogaster in the laboratory</span>. <abbr class="cit-jnl-abbrev">bioRxiv, 232140.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.14" data-doi="10.1016/j.cbd.2010.03.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fiedler</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hudder</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shivkumar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Capo</span>,  <span class="cit-name-given-names">T. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmale</span>,  <span class="cit-name-given-names">M. C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Walsh</span>,  <span class="cit-name-given-names">P. J.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">The transcriptome of the early life history stages of the California Sea Hare Aplysia californica</span>. <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. Part D Genomics Proteomics</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">165</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DComp%2BBiochem%2BPhysiol%2BPart%2BD%2BGenomics%2BProteomics%26rft.aulast%253DFiedler%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D170%26rft.atitle%253DThe%2Btranscriptome%2Bof%2Bthe%2Bearly%2Blife%2Bhistory%2Bstages%2Bof%2Bthe%2BCalifornia%2BSea%2BHare%2BAplysia%2Bcalifornica.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cbd.2010.03.003%26rft_id%253Dinfo%253Apmid%252F20434970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cbd.2010.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20434970&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-307678v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">A. E.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-chapter-title">Big nidovirus genome</span>. In <span class="cit-source">The Nidoviruses</span> (pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>). <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Boston, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.16" data-doi="10.1093/nar/17.12.4847"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>,  <span class="cit-name-given-names">E. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donchenko</span>,  <span class="cit-name-given-names">A. P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Blinov</span>,  <span class="cit-name-given-names">V. M.</span></span> (<span class="cit-pub-date">1989</span>). <span class="cit-article-title">Coronavirus genome: prediction of putative functional domains in the non-structural polyprotein by comparative amino acid sequence analysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">4847</span>–<span class="cit-lpage">4861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D17%26rft.issue%253D12%26rft.spage%253D4847%26rft.epage%253D4861%26rft.atitle%253DCoronavirus%2Bgenome%253A%2Bprediction%2Bof%2Bputative%2Bfunctional%2Bdomains%2Bin%2Bthe%2Bnon-structural%2Bpolyprotein%2Bby%2Bcomparative%2Bamino%2Bacid%2Bsequence%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.12.4847%26rft_id%253Dinfo%253Apmid%252F2526320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/17.12.4847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2526320&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AD05000034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.17" data-doi="10.1016/j.virusres.2006.01.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E. J.</span></span>, <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Nidovirales: evolving the largest RNA virus genome</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, <span class="cit-vol">117</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">17</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D37%26rft.atitle%253DNidovirales%253A%2Bevolving%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.01.017%26rft_id%253Dinfo%253Apmid%252F16503362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16503362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236746000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.18" data-doi="10.1016/j.virusres.2017.10.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greninger</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-pub-date">2018</span>. <span class="cit-article-title">A decade of RNA virus metagenomics is (not) enough</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, <span class="cit-vol">244</span>, <span class="cit-fpage">218</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D244%26rft.spage%253D218%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.10.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.10.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heyland</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vue</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Voolstra</span>,  <span class="cit-name-given-names">C. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Medina</span>,  <span class="cit-name-given-names">M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Moroz</span>,  <span class="cit-name-given-names">L. L.</span></span>, <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Developmental transcriptome of Aplysia californica</span>. <abbr class="cit-jnl-abbrev">J. Exp. Zool. B Mol. Dev. Evol.</abbr>, <span class="cit-vol">316</span> (<span class="cit-issue">2</span>), <span class="cit-fpage">113</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZool.%2BB%2BMol.%2BDev.%2BEvol.%26rft.volume%253D316%26rft.spage%253D113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.20" data-doi="10.1126/science.7041255"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spindler</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horodyski</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grabau</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nichol</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">VandePol</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-pub-date">1982</span>. <span class="cit-article-title">Rapid evolution of RNA genomes</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">215</span>(<span class="cit-issue">4540</span>), <span class="cit-fpage">1577</span>–<span class="cit-lpage">1585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHolland%26rft.auinit1%253DJ%26rft.volume%253D215%26rft.issue%253D4540%26rft.spage%253D1577%26rft.epage%253D1585%26rft.atitle%253DRapid%2Bevolution%2Bof%2BRNA%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7041255%26rft_id%253Dinfo%253Apmid%252F7041255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMTUvNDU0MC8xNTc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMjQvMzA3Njc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.21" data-doi="10.1016/j.virol.2017.12.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoon-Hanks</span>,  <span class="cit-name-given-names">L. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Layton</span>,  <span class="cit-name-given-names">M. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ossiboff</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dubovi</span>,  <span class="cit-name-given-names">E. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Stenglein</span>,  <span class="cit-name-given-names">M. D.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Respiratory disease in ball pythons (Python regius) experimentally infected with ball python nidovirus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2017.12.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D517%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2017.12.008%26rft_id%253Dinfo%253Apmid%252F29329683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2017.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29329683&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.22" data-doi="10.1101/gr.110882.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Islam</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kjallquist</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moliner</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zajac</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">J. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lonnerberg</span>,  <span class="cit-name-given-names">P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Linnarsson</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Characterization of the single-cell transcriptional landscape by highly multiplex RNA-seq</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1160</span>–<span class="cit-lpage">1167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.110882.110%26rft_id%253Dinfo%253Apmid%252F21543516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTE2MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzI0LzMwNzY3OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-307678v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kandel</span>  <span class="cit-name-given-names">E. R.</span></span> (<span class="cit-pub-date">1976</span>). <span class="cit-source">Cellular basis of behavior: An introduction to behavioral neurobiology.</span> <span class="cit-publ-name">W.H. Freeman and Company</span>, <span class="cit-publ-loc">San Francisco</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.24" data-doi="10.1371/journal.pgen.1001191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katzourakis</span>,  <span class="cit-name-given-names">A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gifford</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Endogenous viral elements in animal genomes</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr>, <span class="cit-vol">6</span> (<span class="cit-issue">11</span>), <span class="cit-fpage">e1001191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKatzourakis%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001191%26rft.epage%253De1001191%26rft.atitle%253DEndogenous%2Bviral%2Belements%2Bin%2Banimal%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001191%26rft_id%253Dinfo%253Apmid%252F21124940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21124940&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.25" data-doi="10.1007/s00705-012-1295-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauber</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junglen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zirkel</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nga</span>,  <span class="cit-name-given-names">P. T.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A. E.</span></span>, <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Mesoniviridae: a proposed new family in the order Nidovirales formed by a single species of mosquito-borne viruses</span>. <abbr class="cit-jnl-abbrev">Arch. Virol.</abbr>, <span class="cit-vol">157</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">1623</span>–<span class="cit-lpage">1628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DLauber%26rft.auinit1%253DC.%26rft.volume%253D157%26rft.issue%253D8%26rft.spage%253D1623%26rft.epage%253D1628%26rft.atitle%253DMesoniviridae%253A%2Ba%2Bproposed%2Bnew%2Bfamily%2Bin%2Bthe%2Border%2BNidovirales%2Bformed%2Bby%2Ba%2Bsingle%2Bspecies%2Bof%2Bmosquito-borne%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-012-1295-x%26rft_id%253Dinfo%253Apmid%252F22527862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-012-1295-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22527862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.26" data-doi="10.1128/JVI.00357-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marzano</span>,  <span class="cit-name-given-names">S. Y. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">B. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ajayi-Oyetunde</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradley</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartman</span>,  <span class="cit-name-given-names">G. L.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Domier</span>,  <span class="cit-name-given-names">L. L.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Identification of diverse mycoviruses through metatranscriptomics characterization of the viromes of five major fungal plant pathogens</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>, <span class="cit-vol">90</span>(<span class="cit-issue">15</span>), <span class="cit-fpage">6846</span>–<span class="cit-lpage">6863</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00357-16%26rft_id%253Dinfo%253Apmid%252F27194764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNS82ODQ2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMjQvMzA3Njc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.27" data-doi="10.1016/j.cell.2006.09.052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moroz</span>,  <span class="cit-name-given-names">L. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">J. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Puthanveettil</span>,  <span class="cit-name-given-names">S. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohn</span>,  <span class="cit-name-given-names">A. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ha</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heyland</span>,  <span class="cit-name-given-names">A.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Kandel</span>,  <span class="cit-name-given-names">E.R.</span></span>, <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Neuronal transcriptome of Aplysia: neuronal compartments and circuitry</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">127</span> (<span class="cit-issue">7</span>), <span class="cit-fpage">1453</span>–<span class="cit-lpage">1467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMoroz%26rft.auinit1%253DL.%2BL.%26rft.volume%253D127%26rft.issue%253D7%26rft.spage%253D1453%26rft.epage%253D1467%26rft.atitle%253DNeuronal%2Btranscriptome%2Bof%2BAplysia%253A%2Bneuronal%2Bcompartments%2Band%2Bcircuitry.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.09.052%26rft_id%253Dinfo%253Apmid%252F17190607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.09.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17190607&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243690000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.28" data-doi="10.3389/neuro.24.006.2010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moroz</span>,  <span class="cit-name-given-names">L. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kohn</span>,  <span class="cit-name-given-names">A. B.</span></span>, <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Do different neurons age differently? Direct genome-wide analysis of aging in single identified cholinergic neurons</span>. <abbr class="cit-jnl-abbrev">Front. Aging Neurosci.</abbr>, <span class="cit-vol">2</span>, <span class="cit-fpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BAging%2BNeurosci.%26rft.volume%253D2%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.3389%252Fneuro.24.006.2010%26rft_id%253Dinfo%253Apmid%252F20725513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/neuro.24.006.2010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20725513&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moroz</span>,  <span class="cit-name-given-names">L. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kohn</span>,  <span class="cit-name-given-names">A. B.</span></span>, <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Single-neuron transcriptome and methylome sequencing for epigenomic analysis of aging</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr>, <span class="cit-vol">1048</span>, <span class="cit-fpage">323</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1048%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.30" data-doi="10.1371/journal.ppat.1002215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nga</span>,  <span class="cit-name-given-names">P.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parquet</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">del</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauber</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parida</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nabeshima</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thuy</span>,  <span class="cit-name-given-names">N.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inoue</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ito</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okamoto</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ichinose</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morita</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Discovery of the first insect nidovirus, a missing evolutionary link in the emergence of the largest RNA virus genomes</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">e1002215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DNga%26rft.auinit1%253DP.%2BT.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De1002215%26rft.epage%253De1002215%26rft.atitle%253DDiscovery%2Bof%2Bthe%2Bfirst%2Binsect%2Bnidovirus%252C%2Ba%2Bmissing%2Bevolutionary%2Blink%2Bin%2Bthe%2Bemergence%2Bof%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002215%26rft_id%253Dinfo%253Apmid%252F21931546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21931546&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.31" data-doi="10.1073/pnas.1304422110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Puthanveettil</span>,  <span class="cit-name-given-names">S. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antonov</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalachikov</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajasethupathy</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">Y. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohn</span>,  <span class="cit-name-given-names">A. B.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Moroz</span>,  <span class="cit-name-given-names">L. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kandel</span>,  <span class="cit-name-given-names">E. R.</span></span>, <span class="cit-pub-date">2013</span>. <span class="cit-article-title">A strategy to capture and characterize the synaptic transcriptome</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr>, <span class="cit-vol">110</span> (<span class="cit-issue">18</span>), <span class="cit-fpage">7464</span>–<span class="cit-lpage">7469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1304422110%26rft_id%253Dinfo%253Apmid%252F23589870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE4Lzc0NjQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8yNC8zMDc2NzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.32" data-doi="10.1186/s12864-015-1284-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van den Hoecke</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verhelst</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vuylsteke</span>,  <span class="cit-name-given-names">M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Saelens</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Analysis of the genetic diversity of influenza A viruses using next-generation DNA sequencing</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-vol">16</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1284-z%26rft_id%253Dinfo%253Apmid%252F25758772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1284-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25758772&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.33" data-doi="10.1038/nature20167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">X. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">J. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C. X.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Buchmann</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Redefining the invertebrate RNA virosphere</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">540</span>(<span class="cit-issue">7634</span>), <span class="cit-fpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D540%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature20167%26rft_id%253Dinfo%253Apmid%252F27880757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature20167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27880757&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.34" data-doi="10.1016/j.virusres.2017.10.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y. Z.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Meta-transcriptomics and The Evolutionary Biology of RNA Viruses</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, <span class="cit-vol">244</span>, <span class="cit-fpage">36</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D244%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.10.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.10.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-307678v1.35" data-doi="10.1128/mBio.01484-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stenglein</span>,  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobson</span>,  <span class="cit-name-given-names">E.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wozniak</span>,  <span class="cit-name-given-names">E.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wellehan</span>,  <span class="cit-name-given-names">J.F.X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kincaid</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gordon</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porter</span>,  <span class="cit-name-given-names">B.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baumgartner</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stahl</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelley</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Towner</span>,  <span class="cit-name-given-names">J.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Ball python nidovirus: a candidate etiologic agent for severe respiratory disease in Python regius</span>. <abbr class="cit-jnl-abbrev">mBio</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">e01484</span>–<span class="cit-lpage">1414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D5%26rft.spage%253De01484%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01484-14%26rft_id%253Dinfo%253Apmid%252F25205093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01484-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25205093&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F24%2F307678.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/307678v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F307678v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/307660v1" title="Elementary sensory-motor transformations underlying olfactory navigation in walking fruit-flies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 24, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/307678v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/307678v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/307678v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/307678v1" class="active" data-icon-position="" data-hide-link-title="0">Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/94482" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7PS3NxgsTmEzKlyPzu7zbB80XYHvDoLHDTLLiGwzCYo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="94482" id="node94482--2" data-pisa="biorxiv;307678v1" data-pisa-master="biorxiv;307678" data-seqnum="94482" data-apath="/biorxiv/early/2018/04/24/307678.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Humberto J</span> <span class="nlm-surname">Debat</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">307678; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/307678 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/307678v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;title=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;title=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;text=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;title=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;t=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;title=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;title=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="94482" id="node94482--2" data-pisa="biorxiv;307678v1" data-pisa-master="biorxiv;307678" data-seqnum="94482" data-apath="/biorxiv/early/2018/04/24/307678.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Humberto J</span> <span class="nlm-surname">Debat</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">307678; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/307678 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Debat%2C+Humberto+J"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/94482/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/94482/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/94482/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/94482/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/94482/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/94482/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/94482/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/94482/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/94482/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/94482/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/94482/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/94482/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;count=horizontal&amp;via=&amp;text=Expanding%20the%20size%20limit%20of%20RNA%20viruses%3A%20Evidence%20of%20a%20novel%20divergent%20nidovirus%20in%20California%20sea%20hare%2C%20with%20a%20~35.9%20kb%20virus%20genome&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/307678v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/307678v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/307678v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
