<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">J Med Virol</span><span class="iso-abbrev" title="iso-abbrev">J. Med. Virol</span><span class="doi" title="doi">10.1002/(ISSN)1096-9071</span><span class="publisher-id" title="publisher-id">JMV</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Journal of Medical Virology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0146-6615</span><span class="issn-epub" tagx="issn" title="issn-epub">1096-9071</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7228358">7228358</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/jmv.25754">10.1002/jmv.25754</a></span><span class="publisher-id" title="publisher-id">JMV25754</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Research Articles</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
From SARS and MERS CoVs to SARSâ€�CoVâ€�2: Moving toward more biased codon usage in viral structural and nonstructural genes</div> <div class="alt-title" title="alt-title">
KANDEEL <div class="sc_UNKNOWN">
et al.</div> </div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kandeel</span><span class="given-names" tagx="given-names" title="given-names">Mahmoud</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3668-5147</span><a href="#jmv25754-aff-0001"><sup>1</sup></a><a href="#jmv25754-aff-0002"><sup>2</sup></a><span class="address" title="address"><span class="email" tagx="email" title="email">mkandeel@kfu.edu.sa</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">Abdelazim</span></span><a href="#jmv25754-aff-0003"><sup>3</sup></a><a href="#jmv25754-aff-0004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fayez</span><span class="given-names" tagx="given-names" title="given-names">Mahmoud</span></span><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2145-2714</span><a href="#jmv25754-aff-0005"><sup>5</sup></a><a href="#jmv25754-aff-0006"><sup>6</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alâ€�Nazawi</span><span class="given-names" tagx="given-names" title="given-names">Mohammed</span></span><a href="#jmv25754-aff-0001"><sup>1</sup></a></span></div><span class="citation_author_institution" id="jmv25754-aff-0001">[1], <span class="named-content" title="named-content">Department of Biomedical Sciences, College of Veterinary Medicine</span><span class="institution" title="institution">King Faisal University</span><span class="city" tagx="city" title="city">Alâ€�hofuf</span><span class="country" tagx="country" title="country">Egypt</span></span><span class="citation_author_institution" id="jmv25754-aff-0002">[2], <span class="named-content" title="named-content">Department of Pharmacology, Faculty of Veterinary Medicine</span><span class="institution" title="institution">Kafrelshikh University</span><span class="city" tagx="city" title="city">Kafrelshikh</span><span class="country" tagx="country" title="country">Egypt</span></span><span class="citation_author_institution" id="jmv25754-aff-0003">[3], <span class="named-content" title="named-content">Department of Pathology, College of Veterinary Medicine</span><span class="institution" title="institution">King Faisal University</span><span class="city" tagx="city" title="city">Alâ€�hofuf</span><span class="country" tagx="country" title="country">Saudi Arabia</span></span><span class="citation_author_institution" id="jmv25754-aff-0004">[4], <span class="named-content" title="named-content">Department of Pathology, Faculty of Veterinary Medicine</span><span class="institution" title="institution">Suez Canal University</span><span class="city" tagx="city" title="city">Ismailia</span><span class="country" tagx="country" title="country">Egypt</span></span><span class="citation_author_institution" id="jmv25754-aff-0005">[5], <span class="named-content" title="named-content">Al Ahsa Veterinary Diagnostic Laboratory</span><span class="institution" title="institution">Ministry of Agriculture</span><span class="city" tagx="city" title="city">Alâ€�Ahsa</span><span class="country" tagx="country" title="country">Kingdom of Saudi Arabia</span></span><span class="citation_author_institution" id="jmv25754-aff-0006">[6], <span class="institution" title="institution">Veterinary Serum and Vaccine Institute</span><span class="city" tagx="city" title="city">Cairo</span><span class="country" tagx="country" title="country">Egypt</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Correspondence</b> Mahmoud Kandeel, Department of Biomedical Sciences, College of Veterinary Medicine, King Faisal University, Alâ€�hofuf, Alâ€�ahsa 31982, Saudi Arabia.<br /><br /> Email: <span class="email" tagx="email" title="email">mkandeel@kfu.edu.sa</span><br /><br /></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-3-3</span></span><span class="elocation-id" tagx="elocation-id" title="elocation-id">10.1002/jmv.25754</span><span class="history" title="history"><span class="received" title="received">received: 2020-2-26</span><span class="accepted" title="accepted">accepted: 2020-3-09</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is being made freely available through PubMed Central as part of the COVID-19 public health emergency response. It can be used for unrestricted research re-use and analysis in any form or by any means with acknowledgement of the original source, for the duration of the public health emergency.</span></span></div><a class="self-uri" href="file:JMV-9999-na.pdf" title="self-uri">file:JMV-9999-na.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Severe acute respiratory syndrome coronavirus 2 (SARSâ€�CoVâ€�2) is an emerging disease with fatal outcomes. In this study, a fundamental knowledge gap question is to be resolved by evaluating the differences in biological and pathogenic aspects of SARSâ€�CoVâ€�2 and the changes in SARSâ€�CoVâ€�2 in comparison with the two prior major COV epidemics, SARS and Middle East respiratory syndrome (MERS) coronaviruses.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p>The genome composition, nucleotide analysis, codon usage indices, relative synonymous codons usage, and effective number of codons (ENc) were analyzed in the four structural genes; Spike (S), Envelope (E), membrane (M), and Nucleocapsid (N) genes, and two of the most important nonstructural genes comprising RNAâ€�dependent RNA polymerase and main protease (Mpro) of SARSâ€�CoVâ€�2, Betaâ€�CoV from pangolins, bat SARS, MERS, and SARS CoVs.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>SARSâ€�CoVâ€�2 prefers pyrimidine rich codons to purines. Most highâ€�frequency codons were ending with A or T, while the low frequency and rare codons were ending with G or C. SARSâ€�CoVâ€�2 structural proteins showed 5 to 20 lower ENc values, compared with SARS, bat SARS, and MERS CoVs. This implies higher codon bias and higher gene expression efficiency of SARSâ€�CoVâ€�2 structural proteins. SARSâ€�CoVâ€�2 encoded the highest number of overâ€�biased and negatively biased codons. Pangolin Betaâ€�CoV showed little differences with SARSâ€�CoVâ€�2 ENc values, compared with SARS, bat SARS, and MERS CoV.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>Extreme bias and lower ENc values of SARSâ€�CoVâ€�2, especially in Spike, Envelope, and Mpro genes, are suggestive for higher gene expression efficiency, compared with SARS, bat SARS, and MERS CoVs.</p> </div> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
codon bias</div> <div class="kwd" title="kwd">
COVIDâ€�19</div> <div class="kwd" title="kwd">
MERS CoV</div> <div class="kwd" title="kwd">
nonstructural protein</div> <div class="kwd" title="kwd">
preferred codons</div> <div class="kwd" title="kwd">
SARSâ€�CoVâ€�2</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Deanship of Scientific Research, King Faisal University, Research Groups Track9</div><span class="award-id" tagx="award-id" title="award-id">1811016</span></div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="edited-state" value="corrected-proof" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_JATSPMC version:5.8.0 mode:remove_FC converted:16.04.2020" /> </div> </div> <p content-type="self-citation"><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kandeel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fayez</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alâ€�Nazawi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">From SARS and MERS CoVs to SARSâ€�CoVâ€�2: Moving toward more biased codon usage in viral structural and nonstructural genes</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25754">10.1002/jmv.25754</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31502247">31502247</a></span></span></p> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
INTRODUCTION</div> <p>Severe acute respiratory syndrome coronavirus 2 (SARSâ€�CoVâ€�2) is a new emerging fatal disease emerged in Wuhan, China in December 2019.<a href="#jmv25754-bib-0001"><sup>1</sup></a>, <a href="#jmv25754-bib-0002"><sup>2</sup></a> This is the third CoV epidemic after SARS and Middle East respiratory syndrome (MERS) CoV outbreaks. Initial phylogenetic analysis indicates the formation of a common cluster with bat SARSâ€�like CoV isolated in 2015.<a href="#jmv25754-bib-0003"><sup>3</sup></a> Structural studies showed the close relation of the receptorâ€�binding domain of SARSâ€�CoVâ€�2 with SARS CoV.<a href="#jmv25754-bib-0004"><sup>4</sup></a> Three major CoV epidemics were evolved during the past few decades. The first epidemic was SARS CoV in 2003.<a href="#jmv25754-bib-0005"><sup>5</sup></a> Evidence was provided that SARS CoV was raised from an animal source including an intermediate host animal, which transmitted the virus from a bat carrier to humans.<a href="#jmv25754-bib-0006"><sup>6</sup></a> About a decade later, MERS CoV was first identified in the Arabian Peninsula.<a href="#jmv25754-bib-0007"><sup>7</sup></a> Lastly, in December 2019, the third epidemic emerged in Wuhan, China.</p> <p>CoVs are enveloped, positiveâ€�stranded RNA viruses possessing a comparatively large genome approaching 30â€‰kb and comprising four structural proteins, namely, spike (S), nucleocapsid (N) envelope (E), and membrane (M).<a href="#jmv25754-bib-0008"><sup>8</sup></a> The S protein is responsible for virus attachment to the receptor and fusion with cell membrane.<a href="#jmv25754-bib-0009"><sup>9</sup></a>, <a href="#jmv25754-bib-0010"><sup>10</sup></a> The N protein interacts with the viral RNA to form the ribonucleoprotein.<a href="#jmv25754-bib-0011"><sup>11</sup></a> The E protein helps in virions assembly and comprises ion channel actions<a href="#jmv25754-bib-0012"><sup>12</sup></a>; the M protein shares in the assembly of new virus particles.<a href="#jmv25754-bib-0013"><sup>13</sup></a> CoV genome is organized in 10 openâ€�reading frames (ORFs).<a href="#jmv25754-bib-0014"><sup>14</sup></a> The 5â€² encodes ORF1a and ORF1b, which are translated to give large polyproteins 1a and 1b (polyprotein AB in MERS CoV), which encodes a set of nonstructural protein. The CoV polyprotein is processed by the main protease and papainâ€�like protease to yield the nonstructural proteins.</p> <p>Analysis of genome structure and composition is a part of studies of understanding virus evolution and adaptation to host.<a href="#jmv25754-bib-0015"><sup>15</sup></a>, <a href="#jmv25754-bib-0016"><sup>16</sup></a> Some amino acids are encoded by one codon, while others are encoded by several alternative codons knows as synonymous codons.<a href="#jmv25754-bib-0017"><sup>17</sup></a> Codon bias means a preference for one codon over another during protein translation and affects translation efficiency, which differs from one organism to another.<a href="#jmv25754-bib-0018"><sup>18</sup></a>, <a href="#jmv25754-bib-0019"><sup>19</sup></a></p> <p>In this study, the newly emerged fatal SARSâ€�CoVâ€�2, the four structural genes, and two most important nonstructural genes were evaluated for their nucleotide composition, preferred codons, relative synonymous codons usage (RSCU), and positively and negatively biased codons. This investigation will cover a knowledge gap in our understanding of mechanisms of viral genome evolution, the underlying codon composition, and codon usage preferences in comparison with the most recent CoV epidemic viral infections. The structural genes comprises the S, E, M, and N genes, while the nonstructural genes include the RNAâ€�dependent RNA polymerase (RdRP) and main protease (Mpro) genes.</p> </div> <div class="materials-and-methods" title="materials-and-methods"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
MATERIALS AND METHODS</div> <div class="genedatacollectionandanalyticalprograms" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Gene data collection and analytical programs</div> <p>SARSâ€�CoVâ€�2 and Pangolins Betaâ€�CoV complete genomes sequences were downloaded either from the NCBI GenBank or GISAID (<a href="https://www.gisaid.org/">https://www.gisaid.org/</a>). The sequences of SARS CoV, bat SARS CoV, MERS CoV were retrieved from the gene databases at NCBI.</p> <p>The CLC Genomics Workbench 12.0 (QIAGEN, Aarhus, Denmark) was used to handle the sequences.<a href="#jmv25754-bib-0020"><sup>20</sup></a> The patterns of codon usage were assessed using CodonW 1.4.2.<a href="#jmv25754-bib-0021"><sup>21</sup></a></p> </div> <div class="nucleotidecompositionandcodonusageparameters" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
Nucleotide composition and codon usage parameters</div> <p>The nucleotide composition of the four structure genes, S, E, M, and N, were analyzed to reveal the nucleotides (A, T, G, and C) percentages. A/T, G/C percentage, the percentage of G or C nucleotides at the first position of codons (GC1), the percentage of each nucleotide at the third position of codons, the percentage of G or C nucleotides at the third position of codons (GC3) were calculated.</p> </div> <div class="relativesynonymouscodonsusage" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Relative synonymous codons usage</div> <p>RSCU is calculated by dividing the expected frequencies of synonymous codon against their observed frequencies, according to Equation (<a href="#jmv25754-disp-0001">1</a>)<a href="#jmv25754-bib-0022"><sup>22</sup></a>: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">(1)</span><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
RSCU</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
X</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
ni</div> </div> <div class="msubsup_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
j</div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> </div> <div class="mi_UNKNOWN">
ni</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
,</div> </div></div></div>where <i>X</i><sub><i>ij</i></sub> implies the observed number of codons used and <i>ni</i> stands for the sum of synonymous codons. The raw data for RSCU are provided in the Supporting Information Material.<p /> </div> <div class="theeffectivenumberofcodons" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title">
The effective number of codons</div> <p>ENc values range from 20 to 61. The obtained ENc value can be used to conclude the codon usage bias. An ENc value of &amp;lt;35 indicates strong codon usage bias due to lower number of codons used in protein translation. Higher ENc value indicates low codon usage bias. The raw data for ENc are provided in the Supporting Information Material.</p> </div> </div> <div class="results" title="results"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
RESULTS</div> <div class="nucleotidecompositionsofthenonstructuralproteinsofsarsâ€�covâ€�2,sarscov,batcov,andmerscov" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
Nucleotide compositions of the non structural proteins of SARSâ€�CoVâ€�2, SARS CoV, bat CoV, and MERS CoV</div> <p>In S gene, T nucleotides were the most predominant (32%â€�34%) followed by A (25%â€�29%). SARSâ€�CoVâ€�2 showed the lowest GC% (37%) and the highest AT% (63%), compared with bat SARS, MERS, and SARS CoV. In addition, SARSâ€�CoVâ€�2 has the lowest percentage of G/C nucleotides at the third position of codons followed by pangolins Betaâ€�CoV, which showed the highest A3s. In all of the examined CoVs, the nucleotide percentages in the S gene were in the following order: T&amp;gt;A&amp;gt;C&amp;gt;G. In NT3s, the T3s and G3s were the most and the least frequent nucleotides, respectively (TableÂ <a href="#jmv25754-tbl-0001">1</a>).</p> <div class="table-wrap_UNKNOWN" id="jmv25754-tbl-0001" content-type="Table" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1</span> <p>The codon usage indices of S, E, M, and N genes SARSâ€�CoVâ€�2, pangolins Betaâ€�CoV, SARS CoV, Bat CoV, and MERS CoV</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <thead valign="bottom"> <tr style="border-bottom:solid 1px #000000" valign="bottom"> <th valign="bottom" rowspan="1" colspan="1" /> <th valign="bottom" rowspan="1" colspan="1">Virus</th> <th valign="bottom" rowspan="1" colspan="1">T3s</th> <th valign="bottom" rowspan="1" colspan="1">C3s</th> <th valign="bottom" rowspan="1" colspan="1">A3s</th> <th valign="bottom" rowspan="1" colspan="1">G3s</th> <th valign="bottom" rowspan="1" colspan="1">Nc</th> <th valign="bottom" rowspan="1" colspan="1">GC3s</th> <th valign="bottom" rowspan="1" colspan="1">GC</th> <th valign="bottom" rowspan="1" colspan="1">Gravy</th> <th valign="bottom" rowspan="1" colspan="1">Aromo</th> </tr> </thead> <tbody valign="top"> <tr> <td rowspan="5" valign="top" colspan="1">Spike</td> <td valign="top" rowspan="1" colspan="1">SARSâ€�CoVâ€�2</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.55</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.19</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.13</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">44</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.25</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.08</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Betaâ€�CoV pangolin</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.51</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.21</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.13</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">45</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.27</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.04</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.54</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.22</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.34</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.15</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">46</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.29</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.05</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.12</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Bat SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.52</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.34</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.15</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">48</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.30</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.05</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MERS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.53</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.28</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.19</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">48</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.33</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.41</td> <td valign="top" rowspan="1" colspan="1">0.05</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td rowspan="5" valign="top" colspan="1">Envelope</td> <td valign="top" rowspan="1" colspan="1">SARSâ€�CoVâ€�2</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.49</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.20</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.27</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.21</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">42</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.34</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td valign="top" rowspan="1" colspan="1">1.13</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.12</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Betaâ€�CoV pangolin</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.45</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.28</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.21</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">46</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td valign="top" rowspan="1" colspan="1">1.13</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.12</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.24</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.33</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.21</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">61</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.41</td> <td valign="top" rowspan="1" colspan="1">1.14</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Bat SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.43</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.30</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.21</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">53</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.36</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td valign="top" rowspan="1" colspan="1">1.15</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MERS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.24</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.44</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.18</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">53</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.34</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td valign="top" rowspan="1" colspan="1">0.78</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.14</td> </tr> <tr> <td rowspan="5" valign="top" colspan="1">Membrane</td> <td valign="top" rowspan="1" colspan="1">SARSâ€�CoVâ€�2</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.42</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.26</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.32</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.17</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">54</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.36</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.43</td> <td valign="top" rowspan="1" colspan="1">0.45</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.12</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Betaâ€�CoV pangolin</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.24</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.36</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.28</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">57</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td valign="top" rowspan="1" colspan="1">0.22</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.08</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.36</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.29</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.31</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">60</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.43</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.46</td> <td valign="top" rowspan="1" colspan="1">0.41</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.12</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Bat SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.28</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.30</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.22</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">57</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.41</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.44</td> <td valign="top" rowspan="1" colspan="1">0.43</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.12</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MERS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.43</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.27</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.28</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.19</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">60</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.43</td> <td valign="top" rowspan="1" colspan="1">0.45</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.12</td> </tr> <tr> <td rowspan="5" valign="top" colspan="1">Nucleocapsid</td> <td valign="top" rowspan="1" colspan="1">SARSâ€�CoVâ€�2</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.29</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.16</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">53</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.36</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.47</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.97</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.07</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Betaâ€�CoV pangolin</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.41</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.29</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.15</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">54</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.35</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.47</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.99</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.07</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.31</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.15</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">54</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.48</td> <td valign="top" rowspan="1" colspan="1">âˆ’1.02</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.07</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Bat SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.30</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.16</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">55</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.48</td> <td valign="top" rowspan="1" colspan="1">âˆ’1.00</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.07</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MERS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.45</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.29</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.31</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.17</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">50</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.48</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.87</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.07</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Abbreviations: CoV, coronavirus; MERS, Middle East respiratory syndrome; SARS, severe acute respiratory syndrome.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd.</span>, </span><span class="license" title="license"><span class="license-p" title="license-p">This article is being made freely available through PubMed Central as part of the COVID-19 public health emergency response. It can be used for unrestricted research re-use and analysis in any form or by any means with acknowledgement of the original source, for the duration of the public health emergency.</span></span></div> </div> <p>In the E gene, T nucleotides were the most predominant (34.5%â€�40.4%) followed by A (21.5%â€�25.7%). SARSâ€�CoVâ€�2 showed the lowest GC% (38.2%) and the highest AT% (63%), compared with bat SARS, MERS, and SARS CoV. In addition, SARSâ€�CoVâ€�2 has the lowest frequency of G/C nucleotides. In all of the examined dCoVs, the nucleotide percentages in the E gene were in the following order: T&amp;gt;A&amp;gt;C&amp;gt;G. In NT3s, the T3s and G3s were the most and the least frequent nucleotides, respectively.</p> <p>In the N gene, A nucleotides were the most predominant (29.6%â€�31.7%) followed by C (25%â€�29%). There was little or no differences in GC% and AT% between the CoVs with a conserved tendency for higher AT%. The nucleotide percentages in the N gene were in the following order: A&amp;gt;C&amp;gt;G&amp;gt;T for SARSâ€�CoVâ€�2, SARS, and bat SARS CoVs, while MERS CoV showed a revised order of A&amp;gt;C&amp;gt;T&amp;gt;G. In NT3s, the T3s and G3s were the most and the least frequent nucleotides, respectively.</p> <p>In the M gene, T nucleotides were the most predominant (29.9%â€�31.9%) followed by A (24.4%â€�25.6%). SARSâ€�CoVâ€�2 showed the lowest GC% (42.6%) and the highest AT% (57.4%), compared with bat SARS, MERS, and SARS CoV. In addition, SARSâ€�CoVâ€�2 and pangolins Betaâ€�CoV showed slightly lower G/C nucleotides at the third position of codons. In all of the examined CoVs, the nucleotide percentages in the M gene were in the following order: T&amp;gt;A&amp;gt;C&amp;gt;G. In NT3s, similar to other structural genes, the T3s and G3s were the most and the least frequent nucleotides, respectively.</p> <p>In RdRP, T and A nucleotides were the most predominant nucleotides. In addition, SARSâ€�CoVâ€�2 showed the highest T3s and the lowest G3s (TableÂ <a href="#jmv25754-tbl-0002">2</a>). In contrast, pangolins Betaâ€�CoV and MERS CoV showed the lowest A3s and the highest G3s frequencies. Therefore, similar to structural genes, RdRP contained pyrimidine nucleotides more frequent than purines. For Mpro, there is a conserved profile of general preference for T3s and low frequencies for G3s. Both SARSâ€�CoVâ€�2 and pangolin Betaâ€�CoV showed the lowest G3s frequencies.</p> <div class="table-wrap_UNKNOWN" id="jmv25754-tbl-0002" content-type="Table" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>The codon usage indices of RdRP and Mpro genes SARSâ€�CoVâ€�2, pangolins Betaâ€�CoV, SARS CoV, Bat CoV, and MERS CoV</p>  <table frame="hsides" rules="groups"> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <thead valign="bottom"> <tr style="border-bottom:solid 1px #000000" valign="bottom"> <th valign="bottom" rowspan="1" colspan="1" /> <th valign="bottom" rowspan="1" colspan="1">Virus</th> <th valign="bottom" rowspan="1" colspan="1">T3s</th> <th valign="bottom" rowspan="1" colspan="1">C3s</th> <th valign="bottom" rowspan="1" colspan="1">A3s</th> <th valign="bottom" rowspan="1" colspan="1">G3s</th> <th valign="bottom" rowspan="1" colspan="1">Nc</th> <th valign="bottom" rowspan="1" colspan="1">GC3s</th> <th valign="bottom" rowspan="1" colspan="1">GC</th> <th valign="bottom" rowspan="1" colspan="1">Gravy</th> <th valign="bottom" rowspan="1" colspan="1">Aromo</th> </tr> </thead> <tbody valign="top"> <tr> <td rowspan="5" valign="top" colspan="1">RdRP</td> <td valign="top" rowspan="1" colspan="1">SARSâ€�CoVâ€�2</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.42</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.25</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.20</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">50.91</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.35</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td valign="top" rowspan="1" colspan="1">0.02</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.14</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Betaâ€�CoV pangolin</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.27</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">51.81</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td valign="top" rowspan="1" colspan="1">0.24</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.25</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.36</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.24</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">53.32</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.42</td> <td valign="top" rowspan="1" colspan="1">0.10</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Bat SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.41</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.22</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.36</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.25</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">52.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.41</td> <td valign="top" rowspan="1" colspan="1">0.20</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.09</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MERS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.38</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.25</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.33</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.27</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">55.57</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.41</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.43</td> <td valign="top" rowspan="1" colspan="1">0.37</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.11</td> </tr> <tr> <td rowspan="5" valign="top" colspan="1">Mpro</td> <td valign="top" rowspan="1" colspan="1">SARSâ€�CoVâ€�2</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.52</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.21</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.42</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.13</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">45.68</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.27</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.13</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Betaâ€�CoV pangolin</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.54</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.20</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.13</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">46.65</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.26</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.21</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.13</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.51</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.22</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.18</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">48.91</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.31</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.39</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.20</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.13</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">Bat SARS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.49</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.24</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.37</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.19</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">50.23</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.32</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.20</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.13</td> </tr> <tr> <td valign="top" rowspan="1" colspan="1">MERS CoV</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.54</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.25</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.28</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.21</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">50.95</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.35</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.40</td> <td valign="top" rowspan="1" colspan="1">âˆ’0.18</td> <td align="char" char="." valign="top" rowspan="1" colspan="1">0.14</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Abbreviations: CoV, coronavirus; Mpro, main protease; MERS, Middle East respiratory syndrome; RdRP, RNAâ€�dependent RNA polymerase; SARS, severe acute respiratory syndrome.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd.</span>, </span><span class="license" title="license"><span class="license-p" title="license-p">This article is being made freely available through PubMed Central as part of the COVID-19 public health emergency response. It can be used for unrestricted research re-use and analysis in any form or by any means with acknowledgement of the original source, for the duration of the public health emergency.</span></span></div> </div> </div> <div class="rscuanalysis" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
RSCU analysis</div> <p>In TablesÂ <a href="#jmv25754-tbl-0003">3</a>, <a href="#jmv25754-tbl-0004">4</a>, and Table S1, the RSCU values for codons of CoV structural and nonstructural genes are provided, respectively. The tables are colored by a color scheme to denote the levels of codon usage bias. A value of RSCU =1 means that the observed frequency of codon is equivalent to the predictable frequency and indicating the lack of any codon usage bias. The underrepresented or negatively biased codons denote RSCU &amp;lt;0.6 (blue color), the overexpressed or positively biased codons with RSCU &amp;gt;0.6 (red color). The range between 0.6 and 1.6 conforms to the nonbiased codons.</p> <div class="table-wrap_UNKNOWN" id="jmv25754-tbl-0003" content-type="Table" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 3</span> <p>RSCU values of structural genes (S and E) from SARSâ€�CoVâ€�2, pangolins Betaâ€�CoV, SARS CoV, Bat CoV, and MERS CoV</p>  <table frame="void" rules="groups"> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <tbody valign="top"> <tr> <td valign="top" rowspan="1" colspan="1"><span class="inline-graphic" title="inline-graphic" /></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Abbreviations: CoV, coronavirus; MERS, Middle East respiratory syndrome; RSCU, relative synonymous codons usage; SARS, severe acute respiratory syndrome.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd.</span>, </span><span class="license" title="license"><span class="license-p" title="license-p">This article is being made freely available through PubMed Central as part of the COVID-19 public health emergency response. It can be used for unrestricted research re-use and analysis in any form or by any means with acknowledgement of the original source, for the duration of the public health emergency.</span></span></div> </div> <div class="table-wrap_UNKNOWN" id="jmv25754-tbl-0004" content-type="Table" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 4</span> <p>RSCU values of structural genes (M and N genes) from SARSâ€�CoVâ€�2, pangolins Betaâ€�CoV, SARS CoV, Bat CoV, and MERS CoV</p>  <table frame="void" rules="groups"> <colgroup><col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1" /></colgroup> <tbody valign="top"> <tr> <td valign="top" rowspan="1" colspan="1"><span class="inline-graphic" title="inline-graphic" /></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Abbreviations: CoV, coronavirus; MERS, Middle East respiratory syndrome; RSCU, relative synonymous codons usage; SARS, severe acute respiratory syndrome.</p> </div> </div> <div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd.</span>, </span><span class="license" title="license"><span class="license-p" title="license-p">This article is being made freely available through PubMed Central as part of the COVID-19 public health emergency response. It can be used for unrestricted research re-use and analysis in any form or by any means with acknowledgement of the original source, for the duration of the public health emergency.</span></span></div> </div> <p>In the S gene, the overâ€�biased codons, SARSâ€�CoVâ€�2 showed the highest number of overâ€�biased codons (10 codons), including CTT, ATT, GTT, TCT, CCT, CCA, ACT, GCT, AGA, and GGT. All of these codons contained A3s or T3s. In contrast, pangolin Betaâ€�CoV, SARS CoV, bat SARS CoV, and MERS CoV showed 8, 8, 9, and 8 overâ€�biased codons, respectively (TableÂ <a href="#jmv25754-tbl-0003">3</a>). Therefore, SARSâ€�CoVâ€�2 has the largest number of overâ€�biased codons. The overâ€�biased codons were similar to that provided for SARSâ€�CoVâ€�2 except for CCA and ACT for pangolin Betaâ€�CoV, CCA, and GGT for SARS CoV, CCA for bat SARS CoV and ATT and CCA for MERS CoV.</p> <p>In the N gene, the overâ€�biased codons, SARSâ€�CoVâ€�2 showed the highest number of overâ€�biased codons (six codons), including TTG, CTT, ATT, ACT, GCT, and AGA. In contrast, pangolin Betaâ€�CoV, SARS CoV, bat SARS CoV, and MERS CoV showed 4, 4, 4, and 5 overâ€�biased codons, respectively (TableÂ <a href="#jmv25754-tbl-0004">4</a>). Therefore, SARSâ€�CoVâ€�2 has the largest number of overâ€�biased codons in the N gene. The overâ€�biased codons were similar to that provided for SARSâ€�CoVâ€�2 except TTG for pangolin Betaâ€�CoV, TTG, and CTT for SARS TTG and CTT for bat SARS CoV and TTG for MERS CoV. For MERS CoV, the overâ€�biased codons were slightly different and included CTT, ATT, TCT ACT, GCT, TAC, and AGA.</p> <p>In the M gene, the overâ€�biased codons were CTT, ATT, GTA, GCT, CCA, GAC, GAA, TGT, CGT, and GGA for SARSâ€�CoVâ€�2 (10 codons), CTT, ATT, TCT, TCA, GCT, CCA, and GGA for pangolin Betaâ€�CoV (seven codons), CTT, ATT, GTA, GCT, CCA, GAC, TGT, and CGT for SARS CoV and bat SARS CoV (eight codons), CTT, ATT, GTA, GCT, CCA, GAC, TGT, GGT, and CGT for MERS CoV (nine codons). GGA and GAA codons were nonbiased codons in all coronaviruses except for the SARSâ€�CoVâ€�2 and pangolin Betaâ€�CoV were overâ€�biased. In the E gene, the codon usage could be biased by the short length of the E gene that favors excluding its delivered RSCU values.</p> <p>The frequent negatively biased codons among CoVs include CTG, TCG, AGC, CCG, ACC, ACG, GCG, CGC, and GGG in the S gene, ATA, GTA, TGC, GCG, TGT, AGG and TGC in the N gene and ATA, TGC, and CCC in the M gene.</p> <p>The number of overâ€�biased and negatively biased codons were compared in SARSâ€�CoVâ€�2, pangolin Betaâ€�CoV, SARS CoV, Bat CoV, and MERS CoV. SARSâ€�CoVâ€�2 almost coding the highest number of overâ€�biased and negatively biased codons among all of the structural proteins. In the S gene, SARSâ€�CoVâ€�2 bears 12 and 19 overâ€�biased and negatively biased codons, respectively. The SARSâ€�CoVâ€�2/SARS overâ€�biased codons ratio was 1.2, 1.14, and 1.44 for S, N, and M genes, respectively. In addition, The SARSâ€�CoVâ€�2/SARS negatively biased codons ratio was 1, 1.2, and 1.44 for S, N, and M genes, respectively. Therefore, the SARSâ€�CoVâ€�2 showed the highest number of extreme codon usage patterns of overâ€� or underâ€�biased codons, followed by SARS CoV. The gap between SARSâ€�CoVâ€�2 and SARS CoVs is more tighter than the gap of SARSâ€�CoVâ€�2 or SARS CoV and the bat SARS CoV, which showed a much lower number of biased codons in comparison with the other viruses. MERS CoV showed a higher number of biased codons only in the N gene, compared with SARS and SARSâ€�CoVâ€�2.</p> <p>The structural genes undertook a homogenous profile of codon usage with little differences among the genes. In contrast, NSP as RdRP and Mpro showed larger variations. RdRP showed three overâ€�biased codons and eight common underâ€�biased codons. SARSâ€�CoVâ€�2 and pangolin Betaâ€�CoV showed the highest number of underâ€�biased codons (12 codons). Compared to 10 codons in SARS and bat SARS and five codons in MERS CoV (Table S1).</p> <p>In Mpro, the number of overâ€� and underâ€�biased genes were 11, 10, 8, 8, 6 and 15, 15, 14, 11, and 9 for SARSâ€�CoVâ€�2, pangolin Betaâ€�CoV, SARS, bat SARS, and MERS CoVs, respectively. This agrees with the general predicted highest number of biased codons in SARSâ€�CoVâ€�2.</p> </div> <div class="effectivenumberofcodons" title="sec"><span class="label" tagx="label" title="label">3.3</span><div class="title" tagx="title" title="title">
Effective number of codons</div> <p>ENc implies the effective number of codons and can be used as a measure of codon usage bias. ENc values range from 20 to 61. As the ENc value increases, the codon usage bias is lower. Low ENc value indicates high codon usage bias.</p> <p>SARSâ€�CoVâ€�2 showed the lowest ENc value for all nonstructural and structural genes, compared with pangolins Betaâ€�CoV, SARS, and bat SARS CoVs (FigureÂ <a href="#jmv25754-fig-0001">1</a>). MERS CoV has the lowest ENc value for N and RdRP. The differences between ENc values between SARSâ€�CoVâ€�2 and pangolins Betaâ€�CoV were 0.6, 4.2, 2.9, 0.7, and 0.7 for S, E, M, N, RdRP, and Mpro. These values were the lowest differences compared with the other CoVs.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Effective number of codons values for structural (S, E, M, N) and nonstructural genes (RNAâ€�dependent RNA polymerase and main protease genes) from SARSâ€�CoVâ€�2, pangolins Betaâ€�CoV, SARS CoV, Bat CoV, and MERS CoV. CoV, coronavirus; MERS, Middle East respiratory syndrome; SARS, severe acute respiratory syndrome</p>   </div> </div> </div> <div class="discussion" title="discussion"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
DISCUSSION</div> <p>Codon usage bias is used in the analysis of genes composition and conclusion of the forces controlling evolution and functions.<a href="#jmv25754-bib-0023"><sup>23</sup></a>, <a href="#jmv25754-bib-0024"><sup>24</sup></a> It has been used in the analysis of viral structural<a href="#jmv25754-bib-0025"><sup>25</sup></a>, <a href="#jmv25754-bib-0026"><sup>26</sup></a> and nonstructural genes.<a href="#jmv25754-bib-0025"><sup>25</sup></a>, <a href="#jmv25754-bib-0027"><sup>27</sup></a> In this study, the codon usage bias and genomic composition were compared in structural proteins of the three major CoV epidemicsâ€"SARS, MERS, and SARSâ€�CoVâ€�2.</p> <p>In correlation with the previous knowledge of CoVs genome composition, AT% was higher than GC% in SARSâ€�CoVâ€�2.<a href="#jmv25754-bib-0028"><sup>28</sup></a>, <a href="#jmv25754-bib-0030"><sup>30</sup></a> In all of the structural genes of SARSâ€�CoVâ€�2, either A or T nucleotides were the most predominant nucleotides. In addition, A or T nucleotides were the most predominant nucleotides at the 3rd position of codons. This is in agreement with the previous studies of CoVs.<a href="#jmv25754-bib-0025"><sup>25</sup></a>, <a href="#jmv25754-bib-0031"><sup>31</sup></a></p> <p>RNA viruses had evolved high ENc value (&amp;gt;35), implying low codon bias to adapt a wide range of hosts with various codon usage preferences.<a href="#jmv25754-bib-0029"><sup>29</sup></a> ENc values above 50, in general, mean low codon usage bias. The codon usage data indicated lower number of ENc values of the SARSâ€�CoVâ€�2 compared with SARS, bat SARS, and pangolin CoV. This indicates a higher codon usage bias of SARSâ€�CoVâ€�2. Within these CoVs, pangolin CoV had the least ENc differences.</p> <p>There is a negative correlation between the ENc value and codon usage bias. ENc values indicate higher codon usage bias in SARSâ€�CoVâ€�2 compared with SARS and MERS CoVs, due to lower ENc values, which is mostly observed in S, E, and M genes and to a lesser extent in N and RdRP genes. In SARS, bat SARS, and MERS CoVs E gene, ENc was &amp;gt;60, while in SARSâ€�CoVâ€�2, the ENc value was decreased by an amount of 18 to be no more than 42. Similarly, the M gene ENc value in SARSâ€�CoVâ€�2 was decreased by an amount of 3 to 5. Genes with low expression levels have high ENc values and more rare codons.<a href="#jmv25754-bib-0032"><sup>32</sup></a> The expression of highly biased genes is considered as high.<a href="#jmv25754-bib-0033"><sup>33</sup></a> The relative expression can be concluded from the ENc value, where small ENc value indicates higher bias and a generally higher level of expression.<a href="#jmv25754-bib-0034"><sup>34</sup></a> Thus, the small ENc value is suggesting for higher gene expression. The lower observed ENc, especially for Spike and Envelope genes, values for SARSâ€�CoVâ€�2 structural genes are indicative for higher gene expression potency.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data">  <p>Supporting information</p>  <div class="media" title="media"><a href="JMV-9999-na-s001.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>Supporting information</p>  <div class="media" title="media"><a href="JMV-9999-na-s002.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>Supporting information</p>  <div class="media" title="media"><a href="JMV-9999-na-s003.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>Supporting information</p>  <div class="media" title="media"><a href="JMV-9999-na-s004.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>Supporting information</p>  <div class="media" title="media"><a href="JMV-9999-na-s005.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>Supporting information</p>  <div class="media" title="media"><a href="JMV-9999-na-s006.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data">  <p>Supporting information</p>  <div class="media" title="media"><a href="JMV-9999-na-s007.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGMENT</div> <p>The authors acknowledge the Deanship of Scientific Research at King Faisal University for the financial support under Research Groups track (Grant No. 1811016).</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="jmv25754-bib-0001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Velavan</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span>. <span class="mixed-article-title" title="mixed-article-title">The COVIDâ€�19 epidemic</span>. <span class="source" tagx="source" title="source">Trop Med Int Health</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">278</span>â€�<span class="lpage" tagx="lpage" title="lpage">280</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/tmi.13383">10.1111/tmi.13383</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32052514">32052514</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">Combination of RTâ€�qPCR testing and clinical features for diagnosis of COVIDâ€�19 facilitates management of SARSâ€�CoVâ€�2 outbreak</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2020</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25721">10.1002/jmv.25721</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Benvenuto</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Giovanetti</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ciccozzi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Spoto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Angeletti</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ciccozzi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">The 2019â€�new coronavirus epidemic: Evidence for virus evolution</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">92</span>:<span class="fpage" tagx="fpage" title="fpage">455</span>â€�<span class="lpage" tagx="lpage" title="lpage">459</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.25688">10.1002/jmv.25688</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31994738">31994738</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">395</span>(<span class="issue" tagx="issue" title="issue">10224</span>):<span class="fpage" tagx="fpage" title="fpage">565</span>â€�<span class="lpage" tagx="lpage" title="lpage">574</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32007145">32007145</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Peiris</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Coronavirus as a possible cause of severe acute respiratory syndrome</span>. <span class="source" tagx="source" title="source">The Lancet.</span><span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">361</span>(<span class="issue" tagx="issue" title="issue">9366</span>):<span class="fpage" tagx="fpage" title="fpage">1319</span>â€�<span class="lpage" tagx="lpage" title="lpage">1325</span>.</span></li> <li tag="ref"><a name="jmv25754-bib-0006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Animal origins of the severe acute respiratory syndrome coronavirus: insight from ACE2â€�Sâ€�protein interactions</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">80</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">4211</span>â€�<span class="lpage" tagx="lpage" title="lpage">4219</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16611880">16611880</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zaki</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">van Boheemen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bestebroer</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">367</span>(<span class="issue" tagx="issue" title="issue">19</span>):<span class="fpage" tagx="fpage" title="fpage">1814</span>â€�<span class="lpage" tagx="lpage" title="lpage">1820</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23075143">23075143</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Siddell</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ziebuhr</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Snijder</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Coronaviruses, toroviruses, and arteriviruses</span><span class="source" tagx="source" title="source">Topley and Wilson's microbiology and microbial infections</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley &amp;amp; Sons</span>; <span class="year" tagx="year" title="year">2005</span>.</span></li> <li tag="ref"><a name="jmv25754-bib-0009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Cavanagh</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">The coronavirus surface glycoprotein</span><span class="source" tagx="source" title="source">The coronaviridae</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Germany</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>; <span class="year" tagx="year" title="year">1995</span>:<span class="fpage" tagx="fpage" title="fpage">73</span>â€�<span class="lpage" tagx="lpage" title="lpage">113</span>.</span></li> <li tag="ref"><a name="jmv25754-bib-0010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kandeel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alâ€�Taher</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Schwingenschlogl</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alâ€�Nazawi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular dynamics of Middle East Respiratory Syndrome Coronavirus (MERS CoV) fusion heptad repeat trimers</span>. <span class="source" tagx="source" title="source">Comput Biol Chem</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">75</span>:<span class="fpage" tagx="fpage" title="fpage">205</span>â€�<span class="lpage" tagx="lpage" title="lpage">212</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29803965">29803965</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Risco</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">AntÃ³n</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Enjuanes</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Carrascosa</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>. <span class="mixed-article-title" title="mixed-article-title">The transmissible gastroenteritis coronavirus contains a spherical core shell consisting of M and N proteins</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">1996</span>;<span class="volume" tagx="volume" title="volume">70</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">4773</span>â€�<span class="lpage" tagx="lpage" title="lpage">4777</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8676505">8676505</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ruch</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Machamer</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">The coronavirus E protein: assembly and beyond</span>. <span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">363</span>â€�<span class="lpage" tagx="lpage" title="lpage">382</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22590676">22590676</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Neuman</span><span class="given-names" tagx="given-names" title="given-names">BW</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kiss</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kunding</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">A structural analysis of M protein in coronavirus assembly and morphology</span>. <span class="source" tagx="source" title="source">J Struct Biol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">174</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">11</span>â€�<span class="lpage" tagx="lpage" title="lpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21130884">21130884</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Al Hajjar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Memish</span><span class="given-names" tagx="given-names" title="given-names">ZA</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">McIntoshc</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>. <span class="mixed-article-title" title="mixed-article-title">Middle East respiratory syndrome coronavirus (MERSâ€�CoV): a perpetual challenge</span>. <span class="source" tagx="source" title="source">Ann Saudi Med</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">33</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">427</span>â€�<span class="lpage" tagx="lpage" title="lpage">436</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24188935">24188935</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Bahir</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fromer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Prat</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Linial</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Viral adaptation to host: a proteomeâ€�based analysis of codon usage and amino acid preferences</span>. <span class="source" tagx="source" title="source">Mol Syst Biol</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">311</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19888206">19888206</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">van Hemert</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">van der Kuyl</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Berkhout</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Impact of the biased nucleotide composition of viral RNA genomes on RNA structure and codon usage</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">97</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">2608</span>â€�<span class="lpage" tagx="lpage" title="lpage">2619</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27519195">27519195</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nakamura</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gojobori</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ikemura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Codon usage tabulated from international DNA sequence databases: status for the year 2000</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">292</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592250">10592250</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Chaney</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Roles for synonymous codon usage in protein biogenesis</span>. <span class="source" tagx="source" title="source">Annu Rev Biophys</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">44</span>:<span class="fpage" tagx="fpage" title="fpage">143</span>â€�<span class="lpage" tagx="lpage" title="lpage">166</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25747594">25747594</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Supek</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>. <span class="mixed-article-title" title="mixed-article-title">The code of silence: Widespread associations between synonymous codon biases and gene function</span>. <span class="source" tagx="source" title="source">J Mol Evol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">82</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">65</span>â€�<span class="lpage" tagx="lpage" title="lpage">73</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26538122">26538122</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="mixed-article-title" title="mixed-article-title">CLC Genomics Workbench 12.0 (QIAGEN, Aarhus, Denmark)</span>. <a href="http://www.qiagenbioinformatics.com">http://www.qiagenbioinformatics.com</a>, <span class="year" tagx="year" title="year">2018</span>.</span></li> <li tag="ref"><a name="jmv25754-bib-0021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Peden</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>. <span class="mixed-article-title" title="mixed-article-title"><i>Analysis of codon usage</i></span> (Doctoral dissertation). University of Nottingham; <span class="year" tagx="year" title="year">2000</span>.</span></li> <li tag="ref"><a name="jmv25754-bib-0022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Behura</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Severson</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>. <span class="mixed-article-title" title="mixed-article-title">Codon usage bias: causative factors, quantification methods and genomeâ€�wide patterns: with emphasis on insect genomes</span>. <span class="source" tagx="source" title="source">Biol Rev</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">88</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">49</span>â€�<span class="lpage" tagx="lpage" title="lpage">61</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22889422">22889422</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kandeel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elshazly</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Elâ€�Deeb</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Fayez</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ghonim</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>. <span class="mixed-article-title" title="mixed-article-title">Species specificity and host affinity rather than tissue tropism controls codon usage pattern in respiratory mycoplasmosis</span>. <span class="source" tagx="source" title="source">J Camel Pract Res</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">29</span>â€�<span class="lpage" tagx="lpage" title="lpage">40</span>.</span></li> <li tag="ref"><a name="jmv25754-bib-0024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gumpper</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Constraints of viral RNA synthesis on codon usage of negativeâ€�strand RNA virus</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">93</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">e01775</span>â€�<span class="lpage" tagx="lpage" title="lpage">01718</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30541832">30541832</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sheikh</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alâ€�Taher</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alâ€�Nazawi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alâ€�Mubarak</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kandeel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of preferred codon usage in the coronavirus N genes and their implications for genome evolution and vaccine design</span>. <span class="source" tagx="source" title="source">J Virol Methods</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">277</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">113806</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31911390">31911390</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Makhija</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of synonymous codon usage in spike protein gene of infectious bronchitis virus</span>. <span class="source" tagx="source" title="source">Can J Microbiol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">61</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">983</span>â€�<span class="lpage" tagx="lpage" title="lpage">989</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26452019">26452019</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Alnazawi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Altaher</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kandeel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparative genomic analysis MERS CoV isolated from humans and camels with special reference to virus encoded helicase</span>. <span class="source" tagx="source" title="source">Biol Pharm Bull</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">40</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">1289</span>â€�<span class="lpage" tagx="lpage" title="lpage">1298</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28769010">28769010</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of synonymous codon usage in SARS Coronavirus and other viruses in the Nidovirales</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">101</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">155</span>â€�<span class="lpage" tagx="lpage" title="lpage">161</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15041183">15041183</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Jenkins</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>. <span class="mixed-article-title" title="mixed-article-title">The extent of codon usage bias in human RNA viruses and its evolutionary origin</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>â€�<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12606071">12606071</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of synonymous codon usage in H5N1 virus and other influenza A viruses</span>. <span class="source" tagx="source" title="source">Biosystems</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">81</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">77</span>â€�<span class="lpage" tagx="lpage" title="lpage">86</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15917130">15917130</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Kandeel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Altaher</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Synonymous and biased codon usage by MERS CoV papainâ€�like and 3CLâ€�proteases</span>. <span class="source" tagx="source" title="source">Biol Pharm Bull</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">40</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1086</span>â€�<span class="lpage" tagx="lpage" title="lpage">1091</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28420819">28420819</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Genomeâ€�wide analysis of codon usage bias in four sequenced cotton species</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="elocation-id" tagx="elocation-id" title="elocation-id">e0194372</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29584741">29584741</a></span></span></li> <li tag="ref"><a name="jmv25754-bib-0033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Nair</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Raveendran</span><span class="given-names" tagx="given-names" title="given-names">NT</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Dirisala</span><span class="given-names" tagx="given-names" title="given-names">VR</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Mutational pressure drives evolution of synonymous codon usage in genetically distinct oenothera plastomes</span>. <span class="source" tagx="source" title="source">Iran J Biotechnol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">58</span>â€�<span class="lpage" tagx="lpage" title="lpage">72</span>.</span></li> <li tag="ref"><a name="jmv25754-bib-0034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="string-name" title="string-name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, et al. <span class="mixed-article-title" title="mixed-article-title">Differences in codon usage bias between photosynthesisâ€�related genes and genetic systemâ€�related genes of chloroplast genomes in cultivated and wild solanum species</span>. <span class="source" tagx="source" title="source">Int J Mol Sci</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">3142</span>.</span></li> </ul> </div> </div>  </body></html>