<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Virus Evol</span><span class="iso-abbrev" title="iso-abbrev">Virus Evol</span><span class="publisher-id" title="publisher-id">vevolu</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Virus Evolution</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2057-1577</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5534066">5534066</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28775893">28775893</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/ve/vex014">10.1093/ve/vex014</a></span><span class="publisher-id" title="publisher-id">vex014</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Phylodynamic analysis to inform prevention efforts in mixed HIV epidemics</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">Erik M.</span></span><a href="#vex014-aff1">1</a><a href="#vex014-cor1" /><a href="#vex014-fn1" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ndembi</span><span class="given-names" tagx="given-names" title="given-names">Nicaise</span></span><a href="#vex014-aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nowak</span><span class="given-names" tagx="given-names" title="given-names">Rebecca</span></span><a href="#vex014-aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kijak</span><span class="given-names" tagx="given-names" title="given-names">Gustavo H.</span></span><a href="#vex014-aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Idoko</span><span class="given-names" tagx="given-names" title="given-names">John</span></span><a href="#vex014-aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dakum</span><span class="given-names" tagx="given-names" title="given-names">Patrick</span></span><a href="#vex014-aff2">2</a><a href="#vex014-aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Royal</span><span class="given-names" tagx="given-names" title="given-names">Walter</span></span><a href="#vex014-aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baral</span><span class="given-names" tagx="given-names" title="given-names">Stefan</span></span><a href="#vex014-aff6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dybul</span><span class="given-names" tagx="given-names" title="given-names">Mark</span></span><a href="#vex014-aff7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blattner</span><span class="given-names" tagx="given-names" title="given-names">William A.</span></span><a href="#vex014-aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charurat</span><span class="given-names" tagx="given-names" title="given-names">Man</span></span><a href="#vex014-aff3">3</a></span></div><span class="citation_author_institution" id="vex014-aff1">[1], </span><span class="citation_author_institution" id="vex014-aff2">[2], </span><span class="citation_author_institution" id="vex014-aff3">[3], </span><span class="citation_author_institution" id="vex014-aff4">[4], </span><span class="citation_author_institution" id="vex014-aff5">[5], </span><span class="citation_author_institution" id="vex014-aff6">[6], </span><span class="citation_author_institution" id="vex014-aff7">[7], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Corresponding author: E-mail: <span class="email" tagx="email" title="email">e.volz@imperial.ac.uk</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">‡</span><p><a href="http://orcid.org/0000-0001-6268-8937">http://orcid.org/0000-0001-6268-8937</a></p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2017-7</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-7-7</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2017-7-7</span></span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">vex014</span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="vex014.pdf" title="self-uri">vex014.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>In HIV epidemics of Sub Saharan Africa, the utility of HIV prevention efforts focused on key populations at higher risk of HIV infection and transmission is unclear. We conducted a phylodynamic analysis of HIV-1 <i>pol</i> sequences from four different risk groups in Abuja, Nigeria to estimate transmission patterns between men who have sex with men (MSM) and a representative sample of newly enrolled treatment naive HIV clients without clearly recorded HIV acquisition risks. We develop a realistic dynamical infectious disease model which was fitted to time-scaled phylogenies for subtypes G and CRF02_AG using a structured-coalescent approach. We compare the infectious disease model and structured coalescent to commonly used genetic clustering methods. We estimate HIV incidence among MSM of 7.9% (95%CI, 7.0–10.4) per susceptible person-year, and the population attributable fraction of HIV transmissions from MSM to reproductive age females to be 9.1% (95%CI, 3.8–18.6), and from the reproductive age women to MSM as 0.2% (95%CI, 0.06–0.3). Applying these parameter estimates to evaluate a test-and-treat HIV strategy that target MSM reduces the total HIV infections averted by half with a 2.5-fold saving. These results suggest the importance of addressing the HIV treatment needs of MSM in addition to cost-effectiveness of specific scale-up of treatment for MSM in the context of the mixed HIV epidemic observed in Nigeria.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
phylodynamics</div> <div class="kwd" title="kwd">
HIV</div> <div class="kwd" title="kwd">
coalescent</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Henry M. Jackson Foundation</span><span class="named-content" title="named-content">10.13039/100003896</span></div><span class="award-id" tagx="award-id" title="award-id">W81XWH-11-2-0174</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">National Institute of Allergy and Infectious Diseases</span><span class="named-content" title="named-content">10.13039/100000060</span></div><span class="award-id" tagx="award-id" title="award-id">R01AI120913</span><span class="award-id" tagx="award-id" title="award-id">R01MH086356</span><span class="award-id" tagx="award-id" title="award-id">R01MH099001</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Global Fund to Fight AIDS, Tuberculosis and Malaria</span><span class="named-content" title="named-content">10.13039/100004417</span></div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="named-content" title="named-content">Centers for Disease Control and Prevention</span><span class="named-content" title="named-content">10.13039/100000030</span></div><span class="award-id" tagx="award-id" title="award-id">U2GH000925</span></div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>In countries with hyper-endemic HIV epidemics (e.g. South Africa, Botswana, Lesotho, and Swaziland), the most efficient allocation of prevention activities may be to target all reproductive aged adults, often referred to as the general population in the context of HIV epidemiology. In most of West Africa, such as Nigeria, the opposite may be true. The majority of the HIV epidemics in this region are mixed (<a href="#vex014-B21">Lowndes et al. 2008</a>), such that they are focused and propagated within the highest risk populations yet would be sustained if transmission in either population were interrupted (<a href="#vex014-B4">Boily et al. 2015</a>). The study reported here employs phylodynamic analysis to more precisely inform the contribution of one key population (KP), men who have sex with men (MSM), to the larger Nigerian epidemic and to model the cost-effectiveness of targeted antiretroviral (ART)-based intervention.</p> <p>It is difficult to ascertain HIV transmission patterns between different risk groups using observational data. Pathogen genomes from random samples of infected hosts provide one of the few sources of observational data that can inform precise estimates of transmission dynamics. Recent epidemiological history shapes the genetic diversity of rapidly evolving pathogens like HIV, which has spawned new approaches towards phylodynamic inference with the aim of harnessing pathogen genetic diversity to estimate epidemiological dynamics (<a href="#vex014-B17">Grenfell et al. 2004</a>; <a href="#vex014-B35">Volz et al. 2013)</a>.</p> <p>Most attempts to characterize HIV transmission using HIV sequence data have focused on identifying closely related subsets of sequences or ‘clusters’ in high income countries with high quality molecular surveillance due to routine resistance testing (<a href="#vex014-B20">Lewis et al. 2008</a>; <a href="#vex014-B37">Wertheim et al. 2014</a>; <a href="#vex014-B24">Ratmann et al. 2016</a>). Such clustering may indicate an epidemiological relationship, however, rates of clustering are known to be confounded by incomplete sampling, loss to follow-up, and variables correlated with recency of infection (<a href="#vex014-B36">Volz et al. 2012</a>; <a href="#vex014-B23">Poon 2016</a>). In HIV epidemics of sub-Saharan Africa, sequence samples comprise a small proportion of the large number of total infections, and thus a random sample of sequences will yield few close epidemiological relationships(<a href="#vex014-B33">Volz and Frost 2013</a>). Even when a sample contains no transmission pairs, a wealth of epidemiological information may be inferred from HIV sequence data, since changes in transmission and migration of lineages over long time scales influence HIV genetic diversity and phylogenetic patterns(<a href="#vex014-B26">Rosenberg and Nordborg 2002</a>). In situations with low sampling density, model-based phylodynamic inference methods have become standard. Such modeling approaches have also been useful for estimating the early growth rate of the HIV epidemics (<a href="#vex014-B19">Kretzschmar et al. 2013</a>) and tracing the origins of HIV in central Africa (<a href="#vex014-B14">Faria et al. 2014</a>).</p> <p>Here, we utilized HIV <i>pol</i> sequence data from several populations in Nigeria, a West African country with the second highest absolute HIV burden in the world (<a href="#vex014-B31">UNGASS 2014</a>), to inform our understanding of the phylodynamics of HIV transmission and the potential bridging of infection between concentrated epidemic among MSM and a representative sample of newly enrolled treatment-naïve HIV-positive reproductive-aged adults (i.e. the general population). This report characterizes transmission patterns within the MSM population in Abuja, Nigeria and transmission between risk groups. In addition, HIV test-and-treat as a prevention strategy and its cost-effectiveness in disrupting HIV transmission dynamics is investigated.</p> </div> <div class="2.methods" title="sec"> <div class="title" tagx="title" title="title">
2. Methods</div> <div class="2.1studypopulations" title="sec"> <div class="title" tagx="title" title="title">
2.1 Study populations</div> <p>The Abuja-based <i>TRUST Cohort</i>, an ongoing longitudinal implementation research study of MSM recruited through respondent driven sampling (RDS) has an HIV prevalence observed to be approximately 10 times higher than among all reproductive aged adults (<a href="#vex014-B2">Baral et al. 2015</a>; <a href="#vex014-B6">Charurat et al. 2015</a>). The cohort has high rates of multiple sexually transmitted infections, a high incidence of HIV, and a high rate of bisexuality with 56% of cohort members reporting having female sexual partners in the last 12 months and with 28% of MSM living with HIV reporting having concurrent regular male and female partners (<a href="#sup1">Supplementary Fig. S1</a>).</p> <p>The <i>TRUST</i> Cohort consisted of 806 MSM clients enrolled between March 2013 and December 2014 in a community-based trusted venue in Abuja. At the time of this analysis, the HIV population prevalence in MSM was 45% (10 times that of the general population) and HIV incidence rate was 13.9 per 100 person-years (95%CI, 4.7–33.3) based on detection of incident infections in the longitudinal cohort follow-up. The purpose of the study was to engage and retain MSM in HIV care, treatment, and prevention services and to gain an understanding of social and behavioral approaches that prevent HIV transmission. The median age of the participants was 23 years (interquartile range [IQR], 7). Sixty-six percentage of HIV-infected participants were not on ART at the time of enrollment and the median CD4 count was 318 cells (IQR, 137).</p> <p>The <i>NeuroAIDS</i> Cohort (<a href="#vex014-B1">Akolo et al. 2014</a>) consisted of 77 HIV-infected men and 139 HIV-infected women enrolled between April 2011 and September 2013 from two tertiary facilities in Abuja (University of Abuja Teaching Hospital [UATH] and National Hospital [NHA]). The purpose of the study was to characterize prevalence and incidence of neurocognitive impairment among treatment naïve HIV-infected adults enrolled in HIV care and treatment programs. Individuals were enrolled in an unselected fashion (i.e. not selected a priori because of suspected neurocognitive impairment). Over 90% were ART naïve at enrollment with a median CD4 count of 335 cells (IQR 249) and median log viral load of 4.5 (IQR 1.2). Around 84% were asymptomatic (WHO stage 1). The median age of the participants was 34 years (IQR 11), with 52% married and 15% unemployed (77% female).</p> <p>The <i>TDF</i> Cohort was a retrospective cross-sectional study of archived samples from 175 HIV-infected patients enrolled in HIV treatment and care program between November 2006 and December 2007 and who had virologic failure. The purpose of the study was to determine HIV drug resistance patterns among patients who initiated first-line ART. The mean age was 38.1 (standard deviation [SD], 8.5) and 49% of the participants were female. The median CD4 cell count was 128 cells/μL (IQR, 169) and the median log viral load was 4.7 (IQR, 1.3).</p> <p>The <i>ACTION</i> Cohort is one of the largest PEPFAR-supported programs in Nigeria. Viral load testing was conducted for patients to confirm HIV treatment failure. A total of 191 samples from patients who resided in Abuja between 2011 and 2014 underwent genotypic analysis for drug resistance. The median age was 36 (IQR, 13) and 58% of the patients were female. The median CD4 cell count was 154.</p> </div> <div class="2.2polsequencing" title="sec"> <div class="title" tagx="title" title="title">
2.2 Pol sequencing</div> <p>Plasma viruses with viral load (VL) &amp;gt; 1000 copies/ml were sequenced at the Institute of Human Virology Nigeria, Abuja, Nigeria, using established in-house methods developed and optimized for Nigerian isolates. Characterizing the genetics of 659 HIV-positive subjects was done using RNA extracted from plasma. Detailed sequencing protocols and quality control are described in the <a href="#sup1">Supplementary Material</a>.</p> </div> <div class="2.3phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.3 Phylogenetic analysis</div> <p>Assignment of subtype or circulating recombinant form (CRF) was performed combining tree-phylogenies, sliding-window BLAST, and jumping profile Hidden Markov Model (<a href="#vex014-B28">Schultz et al. 2009</a>). A nucleotide alignment containing subtype reference sequences from the Los Alamos HIV Sequence Database (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a>) was assembled. The overall genetic diversity in the data set was assessed by nucleotide sequence analysis using the Kimura 2-parameter/neighbor-joining method implemented in MEGA (<a href="#vex014-B30">Tamura et al. 2011</a>). Descriptive statistics were obtained using JMP version 10 in SAS. A neighbor-joining phylogeny and subtype assignment for sequences from the different data sources are illustrated in <a href="#vex014-F1">Fig. 1</a> and <a href="#sup1">Supplementary Fig. S2</a>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1.</span></a></div>   <p>Neighbor-joining phylogeny of HIV-1 <i>pol</i> sequences from 659 individuals in Abuja, Nigeria. The arrow depicts a cluster of related sequences belonging to the subtype A1 radiation.</p>   </div> <p>A molecular clock analysis was used to produce dated phylogenies (with calendar time to common ancestry) using BEAST 1.8 (<a href="#vex014-B11">Drummond and Rambaut, 2007</a>). Codons where primary antiretroviral drug resistance mutations were detected (i.e. codons 30, 32, 33, 46, 47, 48, 50, 54, 76, 82, 84, 88 and 90 in protease, and codons 41, 65, 67, 69, 70, 74, 100, 101, 103, 106, 115, 138, 151, 181, 184, 188, 190, 210, 215, 219 and 230 in RT) were first removed from the alignments. Separate phylogenetic analyses were conducted for sequences of subtypes CRF02_AG and G which together comprised the majority of sequences. We used a GTR substitution model with discrete gamma heterogeneity of rates between sites, and a relaxed uncorrelated molecular clock and a Bayesian skyline demographic prior. BEAST Markov chains were run for 300 million iterations. Only data from the TRUST and the NeuroAIDS studies were used for the molecular clock analysis and the phylodynamic analysis. For subsequent phylodynamic analysis, a random sample of 20 trees was drawn from the posterior distributions for subtypes G and CRF02_AG and then combined with a common ancestor in the distant past (code to perform this operation is available at <a href="https://github.com/emvolz/nigeria_hiv_model/misc">https://github.com/emvolz/nigeria_hiv_model/misc</a>).</p> </div> <div class="2.4epidemiologicalmodeling" title="sec"> <div class="title" tagx="title" title="title">
2.4 Epidemiological modeling</div> <p>Epidemiological dynamics were based on standard mathematical models that have previously been fitted to non-genetic surveillance data (<a href="#vex014-B16">Granich et al. 2009</a>; <a href="#vex014-B12">Eaton et al. 2012</a>). The structured coalescent models account for incomplete sampling and differential sampling rates of MSM and the general population. Transmission rates vary depending on stage of infection, sex, risk group, diagnosis status, and treatment status. Patients were classified into five stages of infection based on CD4 including a one-year early HIV infection period (EHI). Stage of infection of each patient at the time of sampling was based on recent testing history (last negative date) and CD4 count. The models account for secular trends in the force of infection, which has generally decreased through time due to behavioral change and increased diagnosis and treatment. The models account for change in diagnosis and treatment rates through time, and separate effects of diagnosis and treatment on reducing transmission rates and on improving longevity. Models of HIV epidemiological dynamics were developed using systems of ordinary differential equations (ODEs) with compartments corresponding to different risk groups, stages of infection, diagnosis and treatment status, and different types of susceptible individuals.</p> <p>Stage of infection was modeled using the CD4-staging formalism developed by <a href="#vex014-B9">Cori et al. (2015)</a>. This includes four CD4-stage categories including AIDS, defined as patients with CD4 &amp;lt; 200. The staging system was modified to include a 1-year early infection (EHI) period instead of a 3-month acute infection period because previous work has shown the former system is more robust to phylodynamic estimation in the presence of phylogenetic error (Volz, et al. 2013a). The model allowed transmission rates to differ between EHI, chronic stages 2-4, and AIDS stage 5. This was parameterized with two parameters <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div> <div class="mn_UNKNOWN">
1</div> </div> </div></div></span>, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div> <div class="mn_UNKNOWN">
2</div> </div> </div></div></span> for the infectiousness of chronic and AIDS relative to EHI. Transmission rates were assumed to be lower in females than males by a factor of 75%.</p> <p>Three risk groups were included in the models: heterosexual males (M), heterosexual females (F), and men who have sex with men (MSM). Transmissions were modeled between M and F, between MSM and other MSM, and between MSM and F. The transmission rates between MSM and F were controlled using two ‘assortativity’ parameters <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> </div></div></span>. With probability <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> </div></div></span> a transmission by a MSM was reserved for the same risk group, with the rest being allocated in proportion to susceptible population sizes of MSM and F. Similarly, a proportion <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> of transmissions by F were reserved for M, with the rest being allocated in proportion to susceptible populations sizes of M and MSM. The transmission rates for MSM and the general population varied through time according to separate logistic functions, the parameters of which were estimated.</p> <p>A summary of model parameters and priors are provided in <a href="#sup1">Supplementary Table S1</a>.</p> <p>The model included diagnosis and treatment. The diagnosis rate was time-dependent and modeled using a logistic function, the parameters of which were estimated. The treatment rate depended on CD4-stage of infection based on prevailing guidelines in force in Nigeria which changed through time. Between 2004 and 2011, patients with CD4 &amp;lt; 200 were treated at a constant rate. After 2011, patients with CD4 &amp;lt; 300 were treated at a constant rate. Patients failed treatment at a fixed rate based on WHO surveillance (one per 4.02 person-years). Transmission rates were assumed to be reduced by 87% for patients on treatment.</p> <p>Based on national census data, the population of the Federal Capital Territory of Nigeria, which includes Abuja and its metropolitan area was 1,405,201 in 2006. We modeled the population size of Abuja through time using an exponential function with growth rate based on census reports (2.53%/year). Phylodynamic estimation also required a compartment for the HIV infected population size outside of the Abuja region, as well as a specification of the rate that lineages are imported from outside of the Abuja region into the Abuja region. This compartment is termed the ‘Source’, since there is a source–sink relationship between the much larger global population of circulating HIV lineages and what was sampled in Abuja. We modeled the size of the source population using an exponential function (estimated rate <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
�?</div></div></span>) and estimated one parameter for the importation rate of lineages from outside of Abuja (<span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
ψ</div></div></span>).</p> <p>The number of compartments for infected individuals was 45 = (five stages of infection) × (undiagnosed+ diagnosed+ treated) × (M + F+MSM). Additionally, there were three compartments for the number susceptible M, F, and MSM and a compartment for the size of the Source population. Results presented in the main text were based on a mathematical model with the following characteristic: An individual was characterized by </p><ul> <li> <p>stage of infection <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
∈</div> </div></div></span>(0–4), with 0 corresponding to EHI and 4 corresponding to AIDS</p> </li> <li> <p>risk group <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
s</div> <div class="mo_UNKNOWN">
∈</div> </div></div></span>(male, female, MSM)</p> </li> <li> <p>diagnosis and treatment status <i>d</i>∈(undiagnosed, diagnosed + not treated, diagnosed and treated)</p> </li> </ul> <p /> <p>The total rate of transmissions from all individuals in risk group <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
s</div></div></span>, stage <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
k</div></div></span> and diagnosis/treatment status <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
d</div></div></span> in an otherwise susceptible population is given by </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
skd</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
s</div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϵ</div> <div class="mi_UNKNOWN">
s</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ω</div> <div class="mi_UNKNOWN">
k</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϵ</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
skd</div> </div> </div> </div></div></div> where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
skd</div> </div> </div> </div></div></span> is the number infected in that category, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϵ</div> <div class="mi_UNKNOWN">
s</div> </div> </div></div></span> controls different baseline infectiousness for males, females, and MSM, <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ϵ</div> <div class="mi_UNKNOWN">
d</div> </div> </div></div></span> may reduce infectiousness for diagnosed individuals and further reduce infectiousness of treated individuals. The function <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mi_UNKNOWN">
s</div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div></div></span> describes secular trends in transmission rates through time using logistic functions. Separate logistic functions were used for MSM and the general population. Transmission rates were further scaled by a density dependent term <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
s</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
s</div> </div> </div></div></span> for the corresponding recipients of transmission; for example, transmission from males to females is scaled by <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span>. Transmissions between MSM and the GP are controlled by the <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> </div></div></span> and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> </div></div></span> parameters as follows: <ul> <li> <p>Transmission rate MSM→F: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mi_UNKNOWN">
F</div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
F</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
F</div> </div> </div></div></div> <p /> </li> <li> <p>Transmission rate MSM→MSM: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
)</div> </div></div></div> <p /> </li> <li> <p>Transmission rate F→MSM: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
)</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> </div> </div> </div></div></div> <p /> </li> <li> <p>Transmission rate F→M: </p><div class="disp-formula" title="disp-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> <div class="mi_UNKNOWN">
M</div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
τ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
F</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
M</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mo_UNKNOWN">
(</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
M</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
m</div> <div class="mi_UNKNOWN">
s</div> <div class="mi_UNKNOWN">
m</div> </div> </div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
f</div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
M</div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
M</div> </div> <div class="mo_UNKNOWN">
)</div> </div></div></div> <p /> </li> </ul> <p /> <p>The model included non-adherence to treatment by including a rate for patients to revert from treated to diagnosed and untreated based on DHS surveillance, which reports 90% retention within one year of starting care. The per capita non-adherence rate was 24.8% per year. Stage progression for treated individuals was slowed to 25% of the baseline rate and was not estimated based on previous modeling work showing insensitivity of model fits to this parameter (<a href="#vex014-B8">Cori et al. 2014</a>).</p> <p>Multiple variations of this model were fitted and compared to this model: </p><ul> <li> <p>A model without diagnosis or treatment, and with a force of infection parameterized using exponential functions as in <a href="#vex014-B16">Granich et al. (2009)</a> Several variations of the force of infection were tried involving polynomials up through order two. For example, with <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
S</div></div></span> susceptibles and <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
I</div></div></span> infected, the transmission rate may be modeled with the density dependent form <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
I</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
b</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
c</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
S</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
I</div> <div class="mo_UNKNOWN">
)</div> </div></div></span>, where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
β</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
b</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> </div></div></span> are estimated. This approach was subsequently abandoned in favor of a logistic function with one fewer parameters. Two variations were tried with symmetric and asymmetric assortativity of females and MSM. Models with asymmetric assortativity were strongly supported.</p> </li> <li> <p>A model which allowed the proportion of men who are MSM to vary through time according to a linear function.</p> </li> <li> <p>A model with linear as opposed to logistic scaling of the force of infection with one fewer parameters.</p> </li> </ul> <p /> <p>The model was implemented in R and C ++ using the Rcpp package (<a href="#vex014-B13">Eddelbuettel et al. 2011</a>), and source files are available at <a href="https://github.com/emvolz/nigeria_hiv_model">https://github.com/emvolz/nigeria_hiv_model</a>, including examples of other model variations.</p> </div> <div class="2.5phylodynamicanalysisandmodelfitting" title="sec"> <div class="title" tagx="title" title="title">
2.5 Phylodynamic analysis and model fitting</div> <p>Structured coalescent models (<a href="#vex014-B32">Volz 2012</a>) were used to infer demographic history and transmission rates in MSM and the general population. Recently, model-based phylogenetic inference has been used to characterize HIV transmission patterns by linking standard mathematical models for HIV epidemic dynamics with population genetic models. An advantage of model-based phylodynamic inference is that a wealth of non-genetic epidemiological surveillance data can be harnessed to calibrate mathematical models and refine parameter estimates. Furthermore, model development can draw on a wealth of expert knowledge that has developed over decades of modeling the HIV epidemic and statistical knowledge gained from fitting similar models to non-genetic surveillance data. Finally, the same models that are used for estimating HIV transmission patterns can be easily adapted for the purpose of predicting epidemic dynamics and to explore the likely impact and cost-effectiveness of different public health interventions.</p> <p>Phylodynamic analysis was carried out using the <i>rcolgem</i> R package, (<a href="http://colgem.r-forge.r-project.org">http://colgem.r-forge.r-project.org</a><u>/</u>) which was used to compute the likelihood of epidemiological parameters given time-scaled HIV phylogenies and the states of sampled individuals. The sample states were based on observed risk group (M, F, MSM, Source) as well as observed CD4 counts at the time of diagnosis where such data were available. Missing data were imputed on the basis of population frequencies of each CD4 stage at the time of sampling of each patient.</p> <p>Fitting the epidemiological models made use of an extension to the general epidemiological structured coalescent model used in <a href="#vex014-B32">Volz (2012)</a>. This approach models the state of a lineage as a continuous time Markov process (CTMC) with time-dependent transition probabilities derived from the epidemiological model. The basic approach is described in <a href="#vex014-B32">Volz (2012)</a>, but we here provide examples of how the state of a lineage may change according to the model defined above. For example, given a lineage in a MSM in stage of infection <i>k</i> &amp;gt; 0, the lineage may revert to stage of infection <i>k</i> − 1 at a rate depending on the progression of disease. According to the coalescent model, this rate is <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> </div> </div></div></span>, where <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
γ</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
1</div> </div> </div> </div></div></span> is the rate of disease progression from stage k-1 to k. Similarly, transmission can change the state of a lineage. Suppose a lineage is in a MSM in the first stage of infection <i>k</i> = 0. The rate that this lineage will jump to a MSM host in stage and diagnosis category (<i>k</i>’,<i>d</i>’) depends on both the transmission rate from the donor deme and the number of infected MSM with EHI (stage zero) according to the following formula: <span class="inline-formula" title="inline-formula"><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> <div class="mo_UNKNOWN">
,</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
k</div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
,</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
d</div> <div class="mo_UNKNOWN">
′</div> </div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> <div class="mo_UNKNOWN">
/</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
I</div> <div class="mrow_UNKNOWN"> <div class="mtext_UNKNOWN">
msm</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
d</div> </div> </div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div></div></span>.</p> <p>Simulation priors were also used to constrain epidemic trajectories within plausible ranges based on WHO surveillance data. Prevalence of infection in the general population was constrained to fall within two standard errors of the estimated means of 0.035 in 2001 and 0.031 in 2012 based on WHO surveillance. Simulated trajectories were docked to the number of patients receiving ARV treatment in 2012 based on UNAIDS data: 491,021 in Nigeria, and estimated 4,706 in Abuja. Similarly, testing rates from simulations were docked to UNAIDS data for 2008 (6.6% per susceptible person year). Diagnosis and treatment rates in MSM were constrained to lie within two standard errors of the estimated values from the Bridging Trust study in 2012 (33% diagnosed, 31% of diagnosed were on treatment). Prevalence of infection in MSM at the end of 2010 and 2014 was constrained to lie within two standard errors of values estimated in the Bridging Trust study (34.9% and 44.5%). Incidence of infection in MSM was constrained to lie within two standard errors of values estimated in the Bridging Trust study in 2014 (13.9% per susceptible person year).</p> <p>Likelihoods of epidemiological parameters were computed using a random sample of 20 phylogenies from the BEAST posterior. The likelihoods were based on combined G and CRF02_AG trees, since both clades have grown in tandem and viral lineages from both clades are in competition for susceptibles. Because coalescent analyses are based on a small sample of trees, we ensured that the particular selection of 20 trees was representative of the posterior by comparing the distribution of distances among and between trees using the metric developed by Kendall and Colijn (<a href="#vex014-B18">Kendall and Colijn, 2016</a>) (<a href="#sup1">Supplementary Material</a>).</p> <p>Models were fitted using a recently developed parallel Bayesian Metropolis-Hastings (pBMH) algorithm (<a href="#vex014-B5">Calderhead, 2014</a>). This algorithm is a modification of traditional MH which makes multiple proposals in parallel and computes the likelihood of each in parallel. Fitting infectious disease models is challenging when the models are complex and likelihood calculations are slow. This approach has the advantage of efficiently utilizing high performance computing clusters when likelihood computations are slow. When computing the likelihood of a new proposal, a phylogeny was sampled uniformly at random from the posterior distribution estimated in BEAST. A Monte Carlo within Metropolis (MCWM) algorithm was used to compute the acceptance probability for each proposal. The pBMH algorithm utilized multivariate normal proposal distributions with variance equal to 0.1% the variance of the prior distributions.</p> <p>In order to reduce the amount of computation required to fit each model, the dimension of the compartmental model was reduced after simulating each trajectory and prior to computing likelihoods with the <i>rcolgem</i> package. This was accomplished by combining chronic stages 2–4, and by combining diagnosed and treated compartments. This reduced the dimension of the system from 46 compartments to 19.</p> <p>Model fitting made use of a computing cluster with 64 cores and more than 30,000 iterations for a total of more than 2 million likelihood calculations. The posterior distribution contained more than 40 thousand unique parameter sets.</p> <p>Model variations were compared heuristically in terms of posterior likelihood and agreement with surveillance data.</p> <p>To assess goodness of fit, we carried out a posterior predictive model simulation procedure (see <a href="#sup1">Supplementary Material</a> and <a href="#sup1">Supplementary Fig. S3</a>). Simulated lineages through time were compared to estimated lineages through time, showing good agreement.</p> </div> <div class="2.6modelingtest-and-treatinterventions" title="sec"> <div class="title" tagx="title" title="title">
2.6 Modeling test-and-treat interventions</div> <p>To evaluate the impact of targeting interventions on key populations such as MSM, we simulated the fitted phylodynamic model for an extended time horizon 2016–2036. We modeled a five-year expansion (2016–2021) in diagnosis, treatment, and retention-in-care rates, such that by 2021 (1) 90% of people living with HIV are diagnosed, (2) 90% of treated individuals are retained in care and virally suppressed after one year of follow-up, and (3) 90% of diagnosed individuals receive ARV irrespective of CD4. We further compared scenarios such that the intervention was targeted to MSM only versus the entire population. Diagnosis, treatment, and retention-in-care rates increased linearly from the 2016 baseline (based on fitted models) to values giving the 90/90/90 diagnosis, treatment, and retention targets.</p> <p>We simulated the fitted models for 2016–2036 and compared three scenarios: </p><ol> <li> <p>No intervention—diagnosis, retention, and treatment rates remain at 2015 levels.</p> </li> <li> <p>Intervention applied only to MSM.</p> </li> <li> <p>Intervention applied to all people living with HIV.</p> </li> </ol> <p /> <p>Scenarios were compared in terms of infections averted and infections averted per person-year of treatment. Costs of the intervention were compared in terms of additional person-years of treatment and Infections averted per unit cost. Projected costs of treatment and new diagnoses in Nigeria were based recent estimates in <a href="#vex014-B22">Mitchell et al. (2015)</a>: </p><ul> <li> <p>$395 per person-year of treatment, which was an approximation obtained by taking an intermediate value between the higher cost for the first year of treatment ($515) and $365 subsequent years.</p> </li> <li> <p>$259 per diagnosis.</p> </li> </ul> <p /> <p><a href="#sup1">Supplementary Fig. S4</a> summarizes the impact and cost of the test-and-treat intervention by comparing targeted versus universal test-and-treat scenarios with a baseline case without any intervention.</p> </div> </div> <div class="3.results" title="sec"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1hivincidenceandprevalencepatternsamongmsmandotherreproductive-agedpopulations" title="sec"> <div class="title" tagx="title" title="title">
3.1 HIV incidence and prevalence patterns among MSM and other reproductive-aged populations</div> <p>There is currently no reliable incidence estimate in Nigeria or its trend over time. We applied phylodynamic analysis to estimate incidence rates for the MSM and reproductive-aged populations as shown in <a href="#vex014-F2">Fig. 2</a> and <a href="#sup1">Supplementary Fig. S5</a>. The incidence rate continues to increase among MSM from the 1980s to 2014 in contrast to the reproductive-aged population where we estimate that incidence is in decline. In 2014, the phylodynamic analysis yields estimates of incidence in MSM of 7.9% per susceptible person-years (95%CI, 7.0–10.4). The phylodynamic estimates substantially refine imprecise estimates obtained directly from follow-up in the <i>TRUST</i> Cohort (green error bar) with incidence rate 13.9 per 100 person-years (95%CI, 7.0–27.8). </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2.</span></a></div>   <p>Estimated prevalence and incidence patterns through time based on phylodynamic analysis. Lines show median posterior estimates and shaded regions show 95% credible intervals. (A) Prevalence in MSM. The black error bar shows estimates from the Nigerian National IBBSS conducted in 2010 which were also used as priors in the phylodynamic analysis. The green error bar shows estimates based on the <i>TRUST</i> study conducted in 2014 which were also used as priors in the phylodynamic analysis. (B) Estimated incidence rate in MSM. The green error bar was estimated in 2014 from the <i>TRUST</i> study and was also used to define priors in the phylodynamic analysis.</p>   </div> <p>The estimated prevalence substantially refines priors based on the HIV Integrated Biological and Behavioral Surveillance (<a href="#vex014-B15">FMOH, 2010</a>) conducted in 2010 and the observed <i>TRUST</i> Cohort estimate in 2014, 44% (green error bars in <a href="#vex014-F2">Fig. 2</a>; 95%CI, 39–48%).</p> </div> <div class="3.2hivtransmissionpatternsbetweenmsmandotherreproductive-agedpopulations" title="sec"> <div class="title" tagx="title" title="title">
3.2 HIV transmission patterns between MSM and other reproductive-aged populations</div> <p>The phylodynamic analysis revealed strong evidence for asymmetric transmission rates between MSM and reproductive-aged females. In 2014, we estimate that 9.1% (95%CI, 3.8–18.6) of new infections in females are attributable to epidemiological interactions with the MSM group (Blue line). In the same year, only 0.2% (95%CI, 0.06–0.3) of new infections in MSM are attributable to females (Red line). <a href="#vex014-F3">Figure 3</a> shows the estimated trends in the population attributable fraction of transmission between MSM and females in the general population. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3.</span></a></div>   <p>Proportion attributable fraction of transmissions based on phylodynamic analysis. Lines show median posterior estimates and shaded regions show the 95% credible interval. Blue denotes the proportion of infections among women in the general population due to the role of MSM. Solid line represents the maximum posterior estimate, and dashed represents the median posterior estimate. Red denotes the proportion of infections among MSM attributable to the role of the general population.</p>   </div> </div> <div class="3.3predictingimpactofuniversalaccesstohivtreatment" title="sec"> <div class="title" tagx="title" title="title">
3.3 Predicting impact of universal access to HIV treatment</div> <p>With the World Health Organization (WHO) call on the need to re-energize HIV programs for key populations (<a href="#vex014-B7">WHO 2015</a>) and recently revised guidelines on ART initiation at any CD4 cell count (<a href="#vex014-B38">World Health Organization 2016</a>), we sought to evaluate the impact of a universal test-and-treat on the modeled HIV transmission dynamics. Universal coverage of ART (UCT) leads to a large reduction in cumulative infections; 26.6% (95% CI, 23.7–29.2%) of infections are averted over five years and 55.4% (95% CI, 53.4–59.2%) of infections are averted over 20 years. Specific treatment approaches for all MSM living with HIV averts approximately half as many infections over 20 years (27.4%, 95% CI, 11.5–43.9%), but at greatly reduced cost. We predict that over five years (2016–2021), the cost of averting a single infection with targeted test-and-treat is $2,929 USD (95%CI, 2,229–3,737), and the same cost for universal test-and-treat is $7,569 USD (95%CI, 5,266–10,766). Over 20 years, the average cost per infection averted comes down to $1,662 USD (95% CI, 1017–2193) for targeted test-and-treat, but is only modestly reduced for universal test-and-treat ($7,343 USD).</p> </div> </div> <div class="4.discussion" title="sec"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>Transmission patterns between different risk groups show the disproportionate epidemiologic role of smaller populations of people living with HIV with unmet treatment needs. In the context of marginalization of key populations enforced by stigmatizing laws (Same Sex Marriage Prohibition Act 2013), these findings provide a public health rationale for governments and health leaders to address the needs of key populations with comprehensive preventative services. Structured coalescent analyses indicates that a substantial proportion of HIV infections in the general female population are linked epidemiologically to the MSM to female bridge which is consistent with reported sexual behaviors and observed incidence trends.</p> <p>The genetic data reveal strong population structure and assortativity within risk-behavior groups. These observations are highly robust to the particular statistical methods used (<a href="#sup1">Supplementary Material</a>). Few transmission clusters marked by small evolutionary distance were detected. Of twelve pairs from subtypes G and CRF02 with evolutionary distance less than 1%, eleven were observed between MSM, and one was observed between MSM and general population. Higher clustering within MSM likely reflects much higher sampling effort in that risk group, and such small numbers of clusters do not allow inference of relative transmission rates between risk groups. Many factors may confound the inference of transmission rates between risk groups, including unequal sampling in different risk groups and changing transmission rates and population sizes through time. The model-based phylodynamic inference is robust to these sources of bias (<a href="#vex014-B3">Beerli and Felsenstein 2001</a>; <a href="#vex014-B10">De Maio et al. 2015</a>), and also reveals many more features of the epidemic history, such as how transmission rates between MSM and reproductive-aged populations have changed over time.</p> <p>Patterns of HIV incidence and prevalence in MSM and other reproductive-aged adults have changed substantially since 1990 in Nigeria, with morbidity in MSM growing dramatically in recent years. There is also not a simple source-sink relationship between MSM and the general population. These findings strongly point to a failure to increase uptake of treatment and prevention services which is further exacerbated by stigmatizing laws (<a href="#vex014-B29">Schwartz et al. 2015</a>). In contrast to the general population, both incidence and prevalence are yet to peak in MSM.</p> <p>Due to increasing incidence and prevalence trends in MSM, the predicted impact of targeted test-and-treat averts many more infections per unit cost than UCT. We previously demonstrated effective implementation of test-and-treat strategy (<a href="#vex014-B6">Charurat et al. 2015</a>) with a program attrition of 10% among individuals engaged in HIV treatment. We show that the gains of this intervention are not limited to MSM, since chains of HIV transmission cross back and forth between the general population and high-risk groups. Based on the estimated transmission rates between risk groups, we have predicted that over twenty years, a targeted test-and-treat strategy would avert about half as many infections as universal test-and-treat, but we estimate that the cost of targeted test-and-treat to only be about 16% of the cost of universal test-and-treat over 20 years. Fear of criminal sanctions as well as of discriminatory or stigmatizing health service provision may preclude the level of ‘test and treat’ services assumed in our model. In cases such as this, comprehensive HIV services should be understood to include legal and other support to MSM organizations to strengthen their capacity to advocate for access to services and changes in discriminatory laws and policies and support for human rights organizations and other allies that can assist MSM organizations in that effort.</p> <p>Our findings should be interpreted with several caveats: The structured coalescent methods are premised on simple random sampling of infected hosts, however MSM patients were enrolled using RDS which yields correlated samples. We examined the data for evidence of correlation between HIV phylogenies and RDS recruitment trees, and we did not find any significant relationships (<a href="#sup1">Supplementary Material</a>). Second, the phylodynamic estimates depend on the accuracy of time-scaled phylogenies estimated using Bayesian relaxed-clock methods, and will be susceptible to factors that would bias such phylogenies, such as recombination or heterotachy. We examined our sequences for evidence of recombination events, and removed any sequences with low quality or evidence of recombination. Third, estimates are based only on sequences of subtypes G and CRF02_AG, and if epidemic dynamics differ substantially for other subtypes, our results may lack representativeness. Fourth, the phylodynamic estimates rely on the approximation that times of common ancestry between lineages correspond to the time of transmission events between hosts, which does not account for within-host evolution of the virus (<a href="#vex014-B34">Volz et al. 2013</a>; <a href="#vex014-B25">Romero-Severson et al. 2014</a>). Finally, it is possible that our model-based estimates would be biased by unmodeled heterogeneities that may influence HIV genetic diversity. While we examine a large range of models and accounted for many variables that may influence transmission rates, it is possible that failure to account for detailed sexual network structure in the epidemiological models may bias results.</p> <p>These analyses have demonstrated the great potential for phylodynamic surveillance of infectious diseases to elucidate hidden transmission patterns which cannot be ascertained from standard surveillance data. As the quality and quantity of HIV sequence data improves, these analyses may be extended to gain a more fine-grained picture of transmission patterns and sexual network structure, as well as the role of other Key Populations and transmission hotspots to provide reliable estimates to guide high impact treatment and prevention interventions.</p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>The research reported in this publication was supported by the Global Fund to Fight AIDS, Tuberculosis and Malaria (GFATM, NGA-H-NACA). The <i>TRUST</i> Cohort is supported by U.S National Institutes of Health under award numbers R01MH099001 and R01AI120913 and by a cooperative agreement (W81XWH-11-2-0174) between the Henry M. Jackson Foundation for the Advancement of Military Medicine, Inc., and the U.S. Department of Defense. The <i>NeuroAIDS</i> Cohort is supported by the U.S. NIH award number R01MH086356. The <i>ACTION</i> Cohort is supported by the HHS/Centers for Disease Control and Prevention (CDC), Global AIDS Program with IHVN (U2GH000925). The content is solely the responsibility of the authors and does not necessarily represent the official views of the GFATM, the U.S. NIH, the U.S. Army, the Department of Defense, or other funders.</p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>EV, WB, SB, and MC designed the study. NN, RN, PD, WR, and MC collected and managed data. EV, NN, and GK conceived the analyses. EV, WB, and MC drafted the manuscript and JI, SB, and MD provided critical review and editing. All authors have seen and approved the paper.</p> </div> <div class="dataavailability" title="sec"> <div class="title" tagx="title" title="title">
Data Availability</div> <p>NeuroAIDS study nucleotide sequences have been deposited with accession numbers: KY989243–KY989389. TDF study sequence accession numbers HQ843507-HQ843681. TRUST/Building study: KY989390–KY989540. Limited anonymized patient meta-data may be available from the authors upon request and conditional upon ethics approval.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary data</div> <p><a href="#sup1">Supplementary data</a> are available at <i>Virus Evolution</i> online.</p> <p><b>Conflict of interest:</b> None declared.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Figure1</span><div class="media" title="media"><a href="vex014_supp_figure1.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Supplementary Data</span><div class="media" title="media"><a href="vex014_supp.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="vex014-B1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akolo</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">‘Neurocognitive impairment associated with predominantly early stage HIV infection in Abuja, Nigeria’</span>, <span class="source" tagx="source" title="source">Journal of Neurovirology</span>, <span class="volume" tagx="volume" title="volume">20</span>/<span class="issue" tagx="issue" title="issue">4</span>: <span class="fpage" tagx="fpage" title="fpage">380</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24927825">24927825</a></span></span></li> <li tag="ref"><a name="vex014-B2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baral</span><span class="given-names" tagx="given-names" title="given-names">S. D.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">‘Evaluating respondent-driven sampling as an implementation tool for universal coverage of antiretroviral studies among men who have sex with men living with HIV’</span>, <span class="source" tagx="source" title="source">Journal of Acquired Immune Deficiency Syndromes</span>, <span class="volume" tagx="volume" title="volume">68/Suppl 2</span>: <span class="fpage" tagx="fpage" title="fpage">S107</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>.</span></li> <li tag="ref"><a name="vex014-B3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beerli</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felsenstein</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">‘Maximum likelihood estimation of a migration matrix and effective population sizes in n subpopulations by using a coalescent approach’</span>, <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">98</span>/<span class="issue" tagx="issue" title="issue">8</span>: <span class="fpage" tagx="fpage" title="fpage">4563</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11287657">11287657</a></span></span></li> <li tag="ref"><a name="vex014-B4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boily</span><span class="given-names" tagx="given-names" title="given-names">M.-C.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">‘What really is a concentrated HIV epidemic and what does it mean for west and central Africa? Insights from mathematical modeling’</span>, <span class="source" tagx="source" title="source">JAIDS Journal of Acquired Immune Deficiency Syndromes</span>, <span class="volume" tagx="volume" title="volume">68</span>: <span class="fpage" tagx="fpage" title="fpage">S74.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25723994">25723994</a></span></span></li> <li tag="ref"><a name="vex014-B5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calderhead</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">‘A general construction for parallelizing Metropolis- Hastings algorithms’</span>, <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span>, <span class="volume" tagx="volume" title="volume">111</span>/<span class="issue" tagx="issue" title="issue">49</span>: <span class="fpage" tagx="fpage" title="fpage">17408</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>.</span></li> <li tag="ref"><a name="vex014-B6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charurat</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">‘Uptake of treatment as prevention for HIV and continuum of care among HIV-positive men who have sex with men in Nigeria’</span>, <span class="source" tagx="source" title="source">Journal of Acquired Immune Deficiency Syndromes</span>, <span class="volume" tagx="volume" title="volume">68 Suppl 2</span>: <span class="fpage" tagx="fpage" title="fpage">S114</span>–<span class="lpage" tagx="lpage" title="lpage">23</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25723975">25723975</a></span></span></li> <li tag="ref"><a name="vex014-B7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: WHO</span>. (<span class="year" tagx="year" title="year">2015</span>) <span class="source" tagx="source" title="source">‘Consolidated Guidelines on HIV Prevention, Diagnosis, Treatment and Care for Key Populations’</span>.. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">World Health Organization</span>.</span></li> <li tag="ref"><a name="vex014-B8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cori</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">‘HPTN 071 (PopART): a cluster-randomized trial of the population impact of an HIV combination prevention intervention including universal testing and treatment: mathematical model’</span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">9</span>/<span class="issue" tagx="issue" title="issue">1</span>: <span class="fpage" tagx="fpage" title="fpage">e84511</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24454728">24454728</a></span></span></li> <li tag="ref"><a name="vex014-B9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cori</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">‘CD4+ cell dynamics in untreated HIV-1 infection: overall rates, and effects of age, viral load, sex and calendar time’</span>, <span class="source" tagx="source" title="source">Aids</span>, <span class="volume" tagx="volume" title="volume">29</span>/<span class="issue" tagx="issue" title="issue">18</span>: <span class="fpage" tagx="fpage" title="fpage">2435</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26558543">26558543</a></span></span></li> <li tag="ref"><a name="vex014-B10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Maio</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">‘New routes to phylogeography: a Bayesian structured coalescent approximation’</span>, <span class="source" tagx="source" title="source">PLoS Genetics</span>, <span class="volume" tagx="volume" title="volume">11</span>/<span class="issue" tagx="issue" title="issue">8</span>: <span class="fpage" tagx="fpage" title="fpage">e1005421.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26267488">26267488</a></span></span></li> <li tag="ref"><a name="vex014-B11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">‘BEAST: Bayesian evolutionary analysis by sampling trees’</span>, <span class="source" tagx="source" title="source">BMC Evolutionary Biology</span>, <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">214.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a></span></span></li> <li tag="ref"><a name="vex014-B12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eaton</span><span class="given-names" tagx="given-names" title="given-names">J. W.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">‘HIV treatment as prevention: systematic comparison of mathematical models of the potential impact of antiretroviral therapy on HIV incidence in South Africa’</span>, <span class="source" tagx="source" title="source">PLoS Medicine</span>, <span class="volume" tagx="volume" title="volume">9</span>/<span class="issue" tagx="issue" title="issue">7</span>: <span class="fpage" tagx="fpage" title="fpage">e1001245.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22802730">22802730</a></span></span></li> <li tag="ref"><a name="vex014-B13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eddelbuettel</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">‘Rcpp: Seamless R and C ++ integration’</span>, <span class="source" tagx="source" title="source">Journal of Statistical Software</span>, <span class="volume" tagx="volume" title="volume">40</span>/<span class="issue" tagx="issue" title="issue">8</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">18</span>.</span></li> <li tag="ref"><a name="vex014-B14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">N. R.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">‘HIV epidemiology. The early spread and epidemic ignition of HIV-1 in human populations’</span>, <span class="source" tagx="source" title="source">Science</span>, <span class="volume" tagx="volume" title="volume">346</span>/<span class="issue" tagx="issue" title="issue">6205</span>: <span class="fpage" tagx="fpage" title="fpage">56</span>–<span class="lpage" tagx="lpage" title="lpage">61</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25278604">25278604</a></span></span></li> <li tag="ref"><a name="vex014-B15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: FMOH</span>. (<span class="year" tagx="year" title="year">2010</span>). <span class="source" tagx="source" title="source">HIV/STI Integrated Biological and Behavioral Surveillance Survey</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Abuja, Nigeria</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nigerian Federal Ministry of Health</span>.</span></li> <li tag="ref"><a name="vex014-B16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Granich</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">‘Universal voluntary HIV testing with immediate antiretroviral therapy as a strategy for elimination of HIV transmission: a mathematical model’</span>, <span class="source" tagx="source" title="source">The Lancet</span>, <span class="volume" tagx="volume" title="volume">373</span>/<span class="issue" tagx="issue" title="issue">9657</span>: <span class="fpage" tagx="fpage" title="fpage">48</span>–<span class="lpage" tagx="lpage" title="lpage">57</span>.</span></li> <li tag="ref"><a name="vex014-B17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">B. T.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">‘Unifying the epidemiological and evolutionary dynamics of pathogens’</span>, <span class="source" tagx="source" title="source">Science</span>, <span class="volume" tagx="volume" title="volume">303</span>/<span class="issue" tagx="issue" title="issue">5656</span>: <span class="fpage" tagx="fpage" title="fpage">327</span>–<span class="lpage" tagx="lpage" title="lpage">32</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14726583">14726583</a></span></span></li> <li tag="ref"><a name="vex014-B18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kendall</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colijn</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">‘Mapping phylogenetic trees to reveal distinct patterns of evolution’</span>, <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>, <span class="volume" tagx="volume" title="volume">33</span>/<span class="issue" tagx="issue" title="issue">10</span>: <span class="fpage" tagx="fpage" title="fpage">2735</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27343287">27343287</a></span></span></li> <li tag="ref"><a name="vex014-B19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kretzschmar</span><span class="given-names" tagx="given-names" title="given-names">M. E.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">‘Prospects of elimination of HIV with test-and-treat strategy’</span>, <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences of the United States of America</span>, <span class="volume" tagx="volume" title="volume">110</span>/<span class="issue" tagx="issue" title="issue">39</span>: <span class="fpage" tagx="fpage" title="fpage">15538</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24009342">24009342</a></span></span></li> <li tag="ref"><a name="vex014-B20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewis</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">‘Episodic sexual transmission of HIV revealed by molecular phylodynamics’</span>, <span class="source" tagx="source" title="source">PLoS Medicine</span>, <span class="volume" tagx="volume" title="volume">5</span>/<span class="issue" tagx="issue" title="issue">3</span>: <span class="fpage" tagx="fpage" title="fpage">e50.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18351795">18351795</a></span></span></li> <li tag="ref"><a name="vex014-B21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowndes</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2008</span>). <span class="source" tagx="source" title="source">West Africa HIV/AIDS epidemiology and response synthesis: implications for prevention</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington, DC</span>, <span class="publisher-name" tagx="publisher-name" title="publisher-name">World Bank</span>.</span></li> <li tag="ref"><a name="vex014-B22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">K. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">‘Modelling the impact and cost-effectiveness of combination prevention amongst HIV serodiscordant couples in Nigeria’</span>, <span class="source" tagx="source" title="source">Aids</span>, <span class="volume" tagx="volume" title="volume">29</span>/<span class="issue" tagx="issue" title="issue">15</span>: <span class="fpage" tagx="fpage" title="fpage">2035</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26355574">26355574</a></span></span></li> <li tag="ref"><a name="vex014-B23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">A. F. Y.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">‘Impacts and shortcomings of genetic clustering methods for infectious disease outbreaks’</span>, <span class="source" tagx="source" title="source">Virus Evolution</span>, <span class="volume" tagx="volume" title="volume">2</span>/<span class="issue" tagx="issue" title="issue">2</span>: <span class="fpage" tagx="fpage" title="fpage">vew031.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28058111">28058111</a></span></span></li> <li tag="ref"><a name="vex014-B24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratmann</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">‘Sources of HIV infection among men having sex with men and implications for prevention’</span>, <span class="source" tagx="source" title="source">Science Translational Medicine</span>, <span class="volume" tagx="volume" title="volume">8</span>/<span class="issue" tagx="issue" title="issue">320</span>: <span class="fpage" tagx="fpage" title="fpage">320ra2</span>.</span></li> <li tag="ref"><a name="vex014-B25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romero-Severson</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">‘Timing and order of transmission events is not directly reflected in a pathogen phylogeny’</span>, <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>, <span class="volume" tagx="volume" title="volume">31</span>/<span class="issue" tagx="issue" title="issue">9</span>: <span class="fpage" tagx="fpage" title="fpage">2472</span>–<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24874208">24874208</a></span></span></li> <li tag="ref"><a name="vex014-B26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenberg</span><span class="given-names" tagx="given-names" title="given-names">N. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nordborg</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">‘Genealogical trees, coalescent theory and the analysis of genetic polymorphisms’</span>, <span class="source" tagx="source" title="source">Nature Reviews Genetics</span>, <span class="volume" tagx="volume" title="volume">3</span>/<span class="issue" tagx="issue" title="issue">5</span>: <span class="fpage" tagx="fpage" title="fpage">380</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>.</span></li> <li tag="ref"><a name="vex014-B27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><i>Nigeria: Same Sex Marriage (Prohibition) Act, 2013</i> [Nigeria], 17 December 2013, available at: <a href="http://www.refworld.org/docid/52f4d9cc4.html">http://www.refworld.org/docid/52f4d9cc4.html</a> [accessed 27 June 2017].</span></li> <li tag="ref"><a name="vex014-B28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schultz</span><span class="given-names" tagx="given-names" title="given-names">A.-K.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">‘jpHMM: improving the reliability of recombination prediction in HIV-1’</span>, <span class="source" tagx="source" title="source">Nucleic Acids Research</span>, <span class="volume" tagx="volume" title="volume">37: (Web Server issue)</span>, <span class="fpage" tagx="fpage" title="fpage">W647</span>–<span class="lpage" tagx="lpage" title="lpage">51</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19443440">19443440</a></span></span></li> <li tag="ref"><a name="vex014-B29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schwartz</span><span class="given-names" tagx="given-names" title="given-names">S. R.</span></span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">‘The immediate eff ect of the Same-Sex Marriage Prohibition Act on stigma, discrimination, and engagement on HIV prevention and treatment services in men who have sex with men in Nigeria: analysis of prospective data from the TRUST cohort’</span>, <span class="source" tagx="source" title="source">The Lancet HIV</span>, <span class="volume" tagx="volume" title="volume">2</span>/<span class="issue" tagx="issue" title="issue">7</span>: <span class="fpage" tagx="fpage" title="fpage">e299</span>–<span class="lpage" tagx="lpage" title="lpage">306</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26125047">26125047</a></span></span></li> <li tag="ref"><a name="vex014-B30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">‘MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods’</span>, <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>, <span class="volume" tagx="volume" title="volume">28</span>/<span class="issue" tagx="issue" title="issue">10</span>: <span class="fpage" tagx="fpage" title="fpage">2731</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353">21546353</a></span></span></li> <li tag="ref"><a name="vex014-B31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: UNGASS</span>. (<span class="year" tagx="year" title="year">2014</span>). <span class="source" tagx="source" title="source">UNGASS Country Progress Report</span><i>.</i><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Nigeria</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">UNGASS</span>.</span></li> <li tag="ref"><a name="vex014-B32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">‘Complex population dynamics and the coalescent under neutrality’</span>, <span class="source" tagx="source" title="source">Genetics</span>, <span class="volume" tagx="volume" title="volume">190</span>/<span class="issue" tagx="issue" title="issue">1</span>: <span class="fpage" tagx="fpage" title="fpage">187</span>–<span class="lpage" tagx="lpage" title="lpage">201</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22042576">22042576</a></span></span></li> <li tag="ref"><a name="vex014-B33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">S. D. W.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">‘Inferring the source of transmission with phylogenetic data’</span>, <span class="source" tagx="source" title="source">PLoS Computational Biology</span>, <span class="volume" tagx="volume" title="volume">9</span>/<span class="issue" tagx="issue" title="issue">12</span>: <span class="fpage" tagx="fpage" title="fpage">e1003397.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24367249">24367249</a></span></span></li> <li tag="ref"><a name="vex014-B34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">‘HIV-1 transmission during early infection in men who have sex with men: a phylodynamic analysis’</span>, <span class="source" tagx="source" title="source">PLoS Medicine</span>, <span class="volume" tagx="volume" title="volume">10</span>/<span class="issue" tagx="issue" title="issue">12</span>: <span class="fpage" tagx="fpage" title="fpage">e1001568; discussion e1001568.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24339751">24339751</a></span></span></li> <li tag="ref"><a name="vex014-B35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koelle</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">‘Viral phylodynamics’</span>, <span class="source" tagx="source" title="source">PLoS Computational Biology</span>, <span class="volume" tagx="volume" title="volume">9</span>/<span class="issue" tagx="issue" title="issue">3</span>: <span class="fpage" tagx="fpage" title="fpage">e1002947.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23555203">23555203</a></span></span></li> <li tag="ref"><a name="vex014-B36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">E. M.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">‘Simple epidemiological dynamics explain phylogenetic clustering of HIV from patients with recent infection’</span>, <span class="source" tagx="source" title="source">PLoS Computational Biology</span>, <span class="volume" tagx="volume" title="volume">8</span>/<span class="issue" tagx="issue" title="issue">6</span>: <span class="fpage" tagx="fpage" title="fpage">e1002552.</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22761556">22761556</a></span></span></li> <li tag="ref"><a name="vex014-B37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wertheim</span><span class="given-names" tagx="given-names" title="given-names">J. O.</span></span></span><span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">‘The global transmission network of HIV-1’</span>, <span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">209</span>/<span class="issue" tagx="issue" title="issue">2</span>: <span class="fpage" tagx="fpage" title="fpage">304</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24151309">24151309</a></span></span></li> <li tag="ref"><a name="vex014-B38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: World Health Organization</span>. (<span class="year" tagx="year" title="year">2016</span>). Guideline on when to start antiretroviral therapy and on pre-exposure prophylaxis for HIV. September 2015. <i>NLM Classification: WHO/WC</i>, 503.</span></li> </ul> </div> </div>  </body></html>