<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/> 
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/29/2020.03.26.010975.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.26.010975v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Comparative performance of four nucleic acid amplification tests for SARS-CoV-2 virus"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.26.010975"/>
  <meta name="DC.Date" content="2020-03-29"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Coronavirus disease 2019 (COVID-19) can be screened and diagnosed through the detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by real-time reverse transcription polymerase chain reaction. SARS-CoV-2 nucleic acid amplification tests (NAATs) have been rapidly developed and quickly applied to clinical testing during the pandemic. However, studies evaluating the performance of these NAAT assays are limited. We evaluated the performance of four NAATs, which were marked by the Conformité Européenne and widely used in China during the pandemic. Results showed that the analytical sensitivity of the four assays was significantly lower than that claimed by the NAAT manufacturers. The limit of detection (LOD) of Daan, Sansure, and Hybribio NAATs was 3000 copies/mL, whereas the LOD of Bioperfectus NAATs was 4000 copies/mL. The results of the consistency test using 46 samples showed that Daan, Sansure, and Hybribio NAATs could detect the samples with a specificity of 100% (30/30) and a sensitivity of 100% (16 /16), whereas Bioperfectus NAAT detected the samples with a specificity of 100% (30/30) and a sensitivity 81.25% (13/16). The sensitivity of Bioperfectus NAAT was lower than that of the three other NAATs; this finding was consistent with the result that Bioperfectus NAAT had a higher LOD than the three other kinds of NAATs. The four above mentioned reagents presented high specificity; however, for the detection of the samples with low virus concentration, Bioperfectus reagent had the risk of missing detection. Therefore, the LOD should be considered in the selection of SARS-CoV-2 NAATs."/>
  <meta name="DC.Contributor" content="Yujuan Xiong"/>
  <meta name="DC.Contributor" content="Zhen-Zhen Li"/>
  <meta name="DC.Contributor" content="Qi-Zhen Zhuang"/>
  <meta name="DC.Contributor" content="Yan Chao"/>
  <meta name="DC.Contributor" content="Fei Li"/>
  <meta name="DC.Contributor" content="Yi-Yuan Ge"/>
  <meta name="DC.Contributor" content="Yi Wang"/>
  <meta name="DC.Contributor" content="Pei-Feng Ke"/>
  <meta name="DC.Contributor" content="Xian-Zhang Huang"/>
  <meta name="article:published_time" content="2020-03-29"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Comparative performance of four nucleic acid amplification tests for SARS-CoV-2 virus"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Coronavirus disease 2019 (COVID-19) can be screened and diagnosed through the detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by real-time reverse transcription polymerase chain reaction. SARS-CoV-2 nucleic acid amplification tests (NAATs) have been rapidly developed and quickly applied to clinical testing during the pandemic. However, studies evaluating the performance of these NAAT assays are limited. We evaluated the performance of four NAATs, which were marked by the Conformité Européenne and widely used in China during the pandemic. Results showed that the analytical sensitivity of the four assays was significantly lower than that claimed by the NAAT manufacturers. The limit of detection (LOD) of Daan, Sansure, and Hybribio NAATs was 3000 copies/mL, whereas the LOD of Bioperfectus NAATs was 4000 copies/mL. The results of the consistency test using 46 samples showed that Daan, Sansure, and Hybribio NAATs could detect the samples with a specificity of 100% (30/30) and a sensitivity of 100% (16 /16), whereas Bioperfectus NAAT detected the samples with a specificity of 100% (30/30) and a sensitivity 81.25% (13/16). The sensitivity of Bioperfectus NAAT was lower than that of the three other NAATs; this finding was consistent with the result that Bioperfectus NAAT had a higher LOD than the three other kinds of NAATs. The four above mentioned reagents presented high specificity; however, for the detection of the samples with low virus concentration, Bioperfectus reagent had the risk of missing detection. Therefore, the LOD should be considered in the selection of SARS-CoV-2 NAATs.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.26.010975v1"/>
  <meta name="citation_id" content="2020.03.26.010975v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010975v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010975v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010975v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.010975.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.26.010975"/>
  <meta name="citation_num_pages" content="15"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.26.010975"/>
  <meta name="citation_author" content="Yujuan Xiong"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, the Second Affiliated Hospital of Guangzhou University of Chinese Medicine, Guangdong Provincial Key Laboratory of Research on Emergency in TCM"/>
  <meta name="citation_author" content="Zhen-Zhen Li"/>
  <meta name="citation_author_institution" content="Second Clinical Medical College, Guangzhou University of Chinese Medicine"/>
  <meta name="citation_author" content="Qi-Zhen Zhuang"/>
  <meta name="citation_author_institution" content="Second Clinical Medical College, Guangzhou University of Chinese Medicine"/>
  <meta name="citation_author" content="Yan Chao"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, the Second Affiliated Hospital of Guangzhou University of Chinese Medicine, Guangdong Provincial Key Laboratory of Research on Emergency in TCM"/>
  <meta name="citation_author" content="Fei Li"/>
  <meta name="citation_author_institution" content="Guangdong Human Papillomavirus (HPV) Molecular Diagnostic Engineering Technology Research Center, Guangdong Hybribio Biotech Co, Ltd"/>
  <meta name="citation_author" content="Yi-Yuan Ge"/>
  <meta name="citation_author_institution" content="Guangdong Human Papillomavirus (HPV) Molecular Diagnostic Engineering Technology Research Center, Guangdong Hybribio Biotech Co, Ltd"/>
  <meta name="citation_author" content="Yi Wang"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, the Second Affiliated Hospital of Guangzhou University of Chinese Medicine, Guangdong Provincial Key Laboratory of Research on Emergency in TCM"/>
  <meta name="citation_author" content="Pei-Feng Ke"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, the Second Affiliated Hospital of Guangzhou University of Chinese Medicine, Guangdong Provincial Key Laboratory of Research on Emergency in TCM"/>
  <meta name="citation_author_email" content="huangxz020@gzucm.edu.cn"/>
  <meta name="citation_author_email" content="Kevinland020@163.com"/>
  <meta name="citation_author" content="Xian-Zhang Huang"/>
  <meta name="citation_author_institution" content="Department of Laboratory Medicine, the Second Affiliated Hospital of Guangzhou University of Chinese Medicine, Guangdong Provincial Key Laboratory of Research on Emergency in TCM"/>
  <meta name="citation_author_email" content="huangxz020@gzucm.edu.cn"/>
  <meta name="citation_author_email" content="Kevinland020@163.com"/>
  <meta name="citation_reference" content="Zhu N, Zhang D, Wang W, Li X, Yang B, Song J, Zhao X, Huang B, Shi W, Lu R, Niu P, Zhan F, Ma X, Wang D, Xu W, Wu G, Gao GF, Tan W, China Novel Coronavirus I, Research T. 2020. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 382:727–733."/>
  <meta name="citation_reference" content="Wang C, Horby PW, Hayden FG, Gao GF. 2020. A novel coronavirus outbreak of global health concern. Lancet 395:470–473."/>
  <meta name="citation_reference" content="Li Q, Guan X, Wu P, Wang X, Zhou L, Tong Y, Ren R, Leung KSM, Lau EHY, Wong JY, Xing X, Xiang N, Wu Y, Li C, Chen Q, Li D, Liu T, Zhao J, Li M, Tu W, Chen C, Jin L, Yang R, Wang Q, Zhou S, Wang R, Liu H, Luo Y, Liu Y, Shao G, Li H, Tao Z, Yang Y, Deng Z, Liu B, Ma Z, Zhang Y, Shi G, Lam TTY, Wu JTK, Gao GF, Cowling BJ, Yang B, Leung GM, Feng Z. 2020. Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia. N Engl J Med doi:10.1056/NEJMoa2001316."/>
  <meta name="citation_reference" content="World Health Organization (2020). Novel Coronavirus(2019-nCoV): Situation Report."/>
  <meta name="citation_reference" content="World Health Organization (2020). Clinical management of severe acute respiratory infection when novel coronavirus (2019-nCoV) infection is suspected: interim guidance (2020-01-28)."/>
  <meta name="citation_reference" content="Li G, De Clercq E. 2020. Therapeutic options for the 2019 novel coronavirus (2019-nCoV). Nat Rev Drug Discov 19:149–150."/>
  <meta name="citation_reference" content="Pang J, Wang MX, Ang IYH, Tan SHX, Lewis RF, Chen JI, Gutierrez RA, Gwee SXW, Chua PEY, Yang Q, Ng XY, Yap RK, Tan HY, Teo YY, Tan CC, Cook AR, Yap JC, Hsu LY. 2020. Potential Rapid Diagnostics, Vaccine and Therapeutics for 2019 Novel Coronavirus (2019-nCoV): A Systematic Review. J Clin Med 9."/>
  <meta name="citation_reference" content="Binnicker MJ. 2020. Emergence of a Novel Coronavirus Disease (COVID-19) and the Importance of Diagnostic Testing: Why Partnership between Clinical Laboratories, Public Health Agencies, and Industry Is Essential to Control the Outbreak. Clin Chem doi:10.1093/clinchem/hvaa071."/>
  <meta name="citation_reference" content="Chu DKW, Pan Y, Cheng SMS, Hui KPY, Krishnan P, Liu Y, Ng DYM, Wan CKC, Yang P, Wang Q, Peiris M, Poon LLM. 2020. Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia. Clin Chem doi:10.1093/clinchem/hvaa029."/>
  <meta name="citation_reference" content="Chan JF, Yip CC, To KK, Tang TH, Wong SC, Leung KH, Fung AY, Ng AC, Zou Z, Tsoi HW, Choi GK, Tam AR, Cheng VC, Chan KH, Tsang OT, Yuen KY. 2020. Improved molecular diagnosis of COVID-19 by the novel, highly sensitive and specific COVID-19-RdRp/Hel real-time reverse transcription-polymerase chain reaction assay validated in vitro and with clinical specimens. J Clin Microbiol doi:10.1128/JCM.00310-20."/>
  <meta name="citation_reference" content="Han Y, Yang H. 2020. The transmission and diagnosis of 2019 novel coronavirus infection disease (COVID-19): A Chinese perspective. J Med Virol doi:10.1002/jmv.25749."/>
  <meta name="citation_reference" content="Zou L, Ruan F, Huang M, Liang L, Huang H, Hong Z, Yu J, Kang M, Song Y, Xia J, Guo Q, Song T, He J, Yen HL, Peiris M, Wu J. 2020. SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients. N Engl J Med 382:1177–1179."/>
  <meta name="citation_reference" content="Xie C, Jiang L, Huang G, Pu H, Gong B, Lin H, Ma S, Chen X, Long B, Si G, Yu H, Jiang L, Yang X, Shi Y, Yang Z. 2020. Comparison of different samples for 2019 novel coronavirus detection by nucleic acid amplification tests. Int J Infect Dis 93:264–267."/>
  <meta name="citation_reference" content="Miller SA, Deak E, Humphries R. 2015. Comparison of the AmpliVue, BD Max System, and illumigene Molecular Assays for Detection of Group B Streptococcus in Antenatal Screening Specimens. J Clin Microbiol 53:1938–41."/>
  <meta name="citation_reference" content="National Health Commission. The guidelines for the diagnosis and treatment of severe acute respiratory syndrome coronavirus 2(SARS-CoV-2)infection (Pilot 7rd version). http://www.nhc.gov.cn/yzygj/s7653p/202003/46c9294a7dfe4cef80dc7f5912eb1989/files/ce3e6945832a438eaae415350a8ce964.pdf."/>
  <meta name="twitter:title" content="Comparative performance of four nucleic acid amplification tests for SARS-CoV-2 virus"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Coronavirus disease 2019 (COVID-19) can be screened and diagnosed through the detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by real-time reverse transcription polymerase chain reaction. SARS-CoV-2 nucleic acid amplification tests (NAATs) have been rapidly developed and quickly applied to clinical testing during the pandemic. However, studies evaluating the performance of these NAAT assays are limited. We evaluated the performance of four NAATs, which were marked by the Conformité Européenne and widely used in China during the pandemic. Results showed that the analytical sensitivity of the four assays was significantly lower than that claimed by the NAAT manufacturers. The limit of detection (LOD) of Daan, Sansure, and Hybribio NAATs was 3000 copies/mL, whereas the LOD of Bioperfectus NAATs was 4000 copies/mL. The results of the consistency test using 46 samples showed that Daan, Sansure, and Hybribio NAATs could detect the samples with a specificity of 100% (30/30) and a sensitivity of 100% (16 /16), whereas Bioperfectus NAAT detected the samples with a specificity of 100% (30/30) and a sensitivity 81.25% (13/16). The sensitivity of Bioperfectus NAAT was lower than that of the three other NAATs; this finding was consistent with the result that Bioperfectus NAAT had a higher LOD than the three other kinds of NAATs. The four above mentioned reagents presented high specificity; however, for the detection of the samples with low virus concentration, Bioperfectus reagent had the risk of missing detection. Therefore, the LOD should be considered in the selection of SARS-CoV-2 NAATs."/>
  <meta name="og-title" property="og:title" content="Comparative performance of four nucleic acid amplification tests for SARS-CoV-2 virus"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.010975v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Coronavirus disease 2019 (COVID-19) can be screened and diagnosed through the detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by real-time reverse transcription polymerase chain reaction. SARS-CoV-2 nucleic acid amplification tests (NAATs) have been rapidly developed and quickly applied to clinical testing during the pandemic. However, studies evaluating the performance of these NAAT assays are limited. We evaluated the performance of four NAATs, which were marked by the Conformité Européenne and widely used in China during the pandemic. Results showed that the analytical sensitivity of the four assays was significantly lower than that claimed by the NAAT manufacturers. The limit of detection (LOD) of Daan, Sansure, and Hybribio NAATs was 3000 copies/mL, whereas the LOD of Bioperfectus NAATs was 4000 copies/mL. The results of the consistency test using 46 samples showed that Daan, Sansure, and Hybribio NAATs could detect the samples with a specificity of 100% (30/30) and a sensitivity of 100% (16 /16), whereas Bioperfectus NAAT detected the samples with a specificity of 100% (30/30) and a sensitivity 81.25% (13/16). The sensitivity of Bioperfectus NAAT was lower than that of the three other NAATs; this finding was consistent with the result that Bioperfectus NAAT had a higher LOD than the three other kinds of NAATs. The four above mentioned reagents presented high specificity; however, for the detection of the samples with low virus concentration, Bioperfectus reagent had the risk of missing detection. Therefore, the LOD should be considered in the selection of SARS-CoV-2 NAATs."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-29"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Comparative performance of four nucleic acid amplification tests for SARS-CoV-2 virus | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1217353 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Coronavirus disease 2019 (COVID-19) can be screened and diagnosed through the detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by real-time reverse transcription polymerase chain reaction. SARS-CoV-2 nucleic acid amplification tests (NAATs) have been rapidly developed and quickly applied to clinical testing during the pandemic. However, studies evaluating the performance of these NAAT assays are limited. We evaluated the performance of four NAATs, which were marked by the Conformité Européenne and widely used in China during the pandemic. Results showed that the analytical sensitivity of the four assays was significantly lower than that claimed by the NAAT manufacturers. The limit of detection (LOD) of Daan, Sansure, and Hybribio NAATs was 3000 copies/mL, whereas the LOD of Bioperfectus NAATs was 4000 copies/mL. The results of the consistency test using 46 samples showed that Daan, Sansure, and Hybribio NAATs could detect the samples with a specificity of 100% (30/30) and a sensitivity of 100% (16 /16), whereas Bioperfectus NAAT detected the samples with a specificity of 100% (30/30) and a sensitivity 81.25% (13/16). The sensitivity of Bioperfectus NAAT was lower than that of the three other NAATs; this finding was consistent with the result that Bioperfectus NAAT had a higher LOD than the three other kinds of NAATs. The four above mentioned reagents presented high specificity; however, for the detection of the samples with low virus concentration, Bioperfectus reagent had the risk of missing detection. Therefore, the LOD should be considered in the selection of SARS-CoV-2 NAATs.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-3">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has spread rapidly since its recent identification in patients with severe pneumonia in Wuhan, China(
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). To date, SARS-CoV-2 has affected more than 372,000 patients worldwide and resulted in more than 16,000 deaths (as of March 24, 2020)(
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). The World Health Organization (WHO) has declared the Chinese outbreak of coronavirus disease 2019 (COVID-19) to be a Public Health Emergency of International Concern and stated that the spread of COVID-19 may be interrupted by early detection, isolation, prompt treatment, and the implementation of a robust system(
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>, 
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). On March 11, at a regular press conference in Geneva, WHO Director-General Tan Desai said that the COVID-19 epidemic can be called a pandemic in terms of its characteristics. Unfortunately, no drug or vaccine has yet been approved to treat human coronaviruses, and new interventions are likely to require months to years to develop(
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). Therefore, early diagnosis of SARS-CoV-2 infection is important to distinguish it from asymptomatic, healthy, and other pathogenic infections and to ensure timely isolation of the infected patients.
    </p>
    <p id="p-4">At present, various detection methods for SARS-CoV-2 infection have been reported, such as real-time reverse transcription–polymerase chain reaction (RT-PCR) assay, sequencing, CRISPR technique, nucleic acid mass spectrometry, and serum immunology(
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). However, real-time RT-PCR remains the primary means for diagnosing COVID-19 among the various diagnostic platforms available. The development of RT-PCR methods to diagnose COVID-19 mainly targets various combinations of the open reading frame (ORF), envelope (E), nucleocapsid (N), spike (S), and RNA-dependent RNA polymerase (RdRp) genes(
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>).
    </p>
    <p id="p-5">Although real-time RT-PCR assay is a highly sensitive technique, false negative results have still been reported in some COVID-19 patients. These results may occur due to insufficient organisms in the specimen resulting from improper collection, transportation, storage, and handling, as well as laboratory test conditions and personnel operation(
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>–
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). Moreover, the quality of the examined regents is related closely to the PCR results for SARS-CoV-2. In response to this outbreak, a number of nucleic acid amplification tests (NAATs) for SARS-CoV-2 were rapidly developed in China and quickly applied to clinical testing. To date, no quality evaluation of SARS-CoV-2 NAATs has been reported.
    </p>
    <p id="p-6">The purpose of this study was to comprehensively evaluate the performance of four SARS-CoV-2 NAATs, which were Conformité Européenne (CE) marked and widely used. The results of this study will be helpful for laboratorians to select the appropriate assay for detecting SARS-CoV-2.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Materials and methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Tested samples</h3>
     <p id="p-7">The kit of performance verification reference materials, which were cell cultures containing pseudoviruses, included 10 positive reference materials, two negative reference materials, 18 analytical specificity reference materials, and three interference reference materials (Guangzhou BDS Biological Technology Company Limited).The positive quality control agents were made with cell cultures containing SARS-CoV-2 pseudoviruses (Guangzhou BDS Biological Technology Company Limited).The pseudoviruses in the positive quality control agents included segments of the ORF1ab gene (the genome coordinates: 900–1500,12200–13500, 18770–18950, and 20560–24453), N gene, and E gene, which were the important characteristic genes of SARS-CoV-2. RNA transcripts, presented by the Chinese Academy of Metrology, were used as standard materials. The RNA transcripts obtained the complete N gene, the complete E gene, and ORF1ab gene segment (14911-15910, GenBank No. NC_O45512) of SARS-CoV-2. The standard value of reference material was obtained by absolute quantitative digital PCR. RNA was extracted from cell cultures with the nucleic acid extraction reagent (Tianlong Technology Company Limited), aliquoted into multiple tubes for testing with each NAAT method, and stored at −80 °C.</p>
     <p id="p-8">Human samples (n = 46), which were RNA leftovers from daily laboratory activity, were provided by the Guangdong Provincial Hospital of Chinese Medicine and Hybribio Medical Laboratory. The selection of clinical samples used in the study was exclusively based on their SARS-CoV-2 virus detection result. The study was approved by the Ethics Committees from the Guangdong Provincial Hospital of Chinese Medicine (ZE2020-027-01).</p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Assay and equipment</h3>
     <p id="p-9">Four sets of systems were used to detect the SARS-CoV-2 virus: Daan SARS-CoV-2 NAAT on LightCycler 480 II PCR instrument (Daan NAAT), Sansure SARS-CoV-2 NAAT on SLAN-96P PCR instrument (Sansure NAAT), Hybribio SARS-CoV-2 virus NAAT on SLAN-96P PCR instrument (Hybribio NAAT), and Bioperfectus SARS-CoV-2 NAAT on SLAN-96P PCR instrument (Bioperfectus NAAT). Real-time PCR was applied to the four systems, which targeted the combinations of the ORF1ab gene, N gene of SARS-CoV-2, and endogenous gene of humans. About 5 μL of RNA was added to the PCR mixture and tested according to the manufacturers’ instructions for each assay. Positive results were defined as the simultaneous detection of the ORF1ab gene and N gene. The four systems had different detection sites for the ORF1ab gene. The PCR primers designed for the four systems were complementary to the sites on the SARS-CoV-2 genomic sequence (GenBank No. 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=NC_045512&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.010975.atom">NC_045512</a>) in the region of 20700–21000 (Daan), 12500–13300 (Sansure), 13400–13550 (Bioperfectus), and 7100–7600 bp (Hybribio). Cycle threshold (Ct) values (i.e., number of cycles required for the fluorescent signal to cross the threshold in RT-PCR) quantified viral load, with low values indicating a high viral load.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Analytical Sensitivity</h3>
     <p id="p-10">Analytical sensitivity was examined in accordance with a modified EP17-A protocol, MM03 and MM19-A guidelines of Clinical Laboratory Standards Institute (CLSI). The analytical sensitivity of each NAAT assay was compared using the same RNA samples obtained from the extraction of mixed clinical samples or positive quality controls. The concentrations of RNA samples were determined by fluorescence quantitative method using standard materials as standard. Serial dilutions were made with a known concentration of the target substance in the analytical range of the expected limit of detection (LOD) and tested in replicates of 20. All panel members were prepared at the same time, aliquoted into individual tubes for each concentration and each assay, stored frozen, and thawed on the day of testing. When 19 or all 20 replicates were positive, the concentration was temporarily defined as LOD, followed by 20 replicates per day for two consecutive days, yielding a total of 60 results. If more than 95% of the results were positive, the concentration was finally determined as the LOD of the reagent. The experiments were spread over 3 days so that the standard deviations reflect the performance of the assay over a range of typical laboratory conditions but without a change in reagent lots.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Precision</h3>
     <p id="p-11">Precision was evaluated in accordance with a modified EP12 protocol, MM09 and MM19-A guideline of CLSI using a patient RNA pool with concentration above 20% of the LOD and the negative sample. Sample pools were divided into five aliquots per level and frozen at −70 °C. One aliquot of each level was thawed daily and analyzed four times per day during a period of five consecutive workdays (n = 20 per level). Precision was evaluated as the coefficient of variation (CV), which was calculated from the data series mean and standard deviation.</p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Accuracy</h3>
     <p id="p-12">Ten positive reference materials and two negative reference materials from the performance verification reference material kit were used to evaluate accuracy.</p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Analytical specificity</h3>
     <p id="p-13">Eighteen pseudovirus samples of analytical specificity reference materials from the performance verification reference material kit were used to determine the analytical specificity of four NAATs, including human coronavirus HCoV-OC43, HCoV-HKU1 RNA, HCoV-229E RNA, HCoV-NL63 RNA, SARS RNA, Middle East respiratory syndrome (MERS) RNA, influenza A HIN1 virus, influenza B INFB virus, respiratory syncytial virus type A and type B, human parainfluenza virus, adenovirus, enterovirus, mycoplasma pneumoniae, Epstein–Barr (EB) virus, human cytomegalovirus, 
      <em>Mycobacterium tuberculosis,</em> and two samples with human genome DNA.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Analytical interferences</h3>
     <p id="p-14">To investigate the analytical interferences, three interference reference materials from the performance verification reference material kit were used, including 6 g/dL hemoglobin, 30 g/dL albumin, and the mix of 100 μg/mL ribavirin and 100 μg/mL azithromycin. All tests were repeated three times.</p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Method comparison</h3>
     <p id="p-15">Forty-six RNA samples from suspected patients with COVID-19 were used in the method comparison. Given that no reference method was used to determine the sensitivity and specificity of NAATs, the consensus result was determined for each sample; samples were categorized as true positive (TP) if positive by two or more assays (
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). The SARS-CoV-2-positive samples were further confirmed by sequencing.
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Statistical analysis</h3>
     <p id="p-16">The significance of the difference in sensitivity between the NAATs was assessed by using Fisher’s exact test in GraphPad Prism (5.0). The Chi-square likelihood ratio test was used to analyze the discordant results. The 95% confidence intervals (CIs) for specificity and sensitivity were calculated by using the Wilson–Score method in GraphPad Prism. Significant differences were considered at 
      <em>P</em> values less than 0.05.
     </p>
    </div>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Result</h2>
    <div id="sec-13" class="subsection">
     <h3>Analytical sensitivity</h3>
     <p id="p-17">We evaluated the analytical sensitivity of each assay to determine differences in the ability of each assay to detect SARS-CoV-2 in mixed clinical samples or in pseudovirus cultures. First, we contrived RNA specimens from SARS-CoV-2 pseudovirus cultures at four separate final concentrations (500, 1000, 2000, and 3000 copies/mL). The ORF1ab gene test results of the serial dilutions were all negative by the Hybribio and the Bioperfectus NAATs. Further investigation revealed that the pseudoviral RNA did not contain the detecting locus of Hybribio and Bioperfectus NAATs targeting the ORF1ab gene. Daan NAAT detected 18/20 replicates at 2000 copies/mL and 20/20 replicates at 3000 copies/mL. Sansure NAAT detected 17/20 at 1000 copies/mL and 20/20 at 2000 copies/mL (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The final results of 60 replicates showed that the LOD of Daan NAAT was 3000 copies/mL for pseudovirus cultures, and the LOD of Sansure NAAT was 2000 copies/mL for pseudovirus cultures.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1228433/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1228433/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1228433" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-18" class="first-child">Comparison of Analytical sensitivity.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-19">To explore the LOD of the NAATs in clinical samples, we further contrived mixed RNA specimens from clinical samples at three separate final concentrations (2000, 3000, and 4000 copies/mL). Daan NAAT detected 12/20 replicates at 2000 copies/mL and 20/20 replicates at 3000 copies/mL (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Sansure NAAT detected 17/20 at 2000 copies/mL and 20/20 at 3000 copies/mL. Hybribio NAAT detected 10/20 replicates at 2000 copies/mL and 20/20 replicates at 3000 copies/mL. Bioperfectus NAAT detected 10/20 at 3000 copies/mL and 20/20 at 4000 copies/mL. The final results of 60 replicates showed that the LOD of clinical samples was 3000 copies/mL by Daan, Sansure, and Hybribio NAATs, whereas that of clinical samples was 4000 copies/mL by Bioperfectus NAAT. The LODs declared in the instructions of Daan, Sansure, Hybribio, and Bioperfectus NAATs were 500, 200,1000, and 1000 copies/mL, respectively.
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Precision</h3>
     <p id="p-20">For the negative sample, the 20 repeated test results by all four NAATs had no amplification curve of target genes but a positive amplification curve of the internal reference gene. For the positive sample, 20 repeated test results by all four NAATs had positive amplification curves of the target genes and internal reference gene. The within-run CVs at the concentrations above the 20% level of the LOD on all four systems were less than 4%, and the total CVs were less than 5% (
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1228430/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1228430/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1228430" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-21" class="first-child">Comparison of precision.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Accuracy</h3>
     <p id="p-22">The results of all four NAATs for accuracy evaluation were in line with expectations by using 10 positive reference materials and two negative reference materials.</p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Analytical specificity</h3>
     <p id="p-23">For the four assays, no amplification curves were obtained for up to 10,000 copies/mL of six coronavirus pseudovirus samples with HCoV-OC43, HCoV-HKU1, HCoV-229E, HCoV-NL63, SARS, and MERS RNA. For the 10 common respiratory pathogens and two human genomic DNA samples, all results of the four assays were also negative.</p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Analytical interferences</h3>
     <p id="p-24">An assessment of interferences is shown in 
      <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The bias of Ct value for the N gene and ORF1ab gene on all four systems was less than 8.5% at the following interfering substance concentrations tested: 6 g/dL hemoglobin, 30 g/dL albumin, 100 μg/mL ribavirin, and 100 μg/mL azithromycin.
     </p>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1228429/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1228429/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1228429" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3.</span>
       <p id="p-25" class="first-child">Sample interferences for common interferents.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Comparison of the four NAATs</h3>
     <p id="p-26">We defined consistency as two or more tests that produce same results and used these results to define true positives and negatives. By this definition, 34.8% (16/30) tested positive and 65.2% (30/46) tested negative. The results for all four NAATs were negative for all 30 negative samples, and the 16 positive samples were reported positive by Daan, Sansure, and Hybribio NAATs. However, three of the 16 positive samples were reported negative by Bioperfectus NAAT (
      <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The Ct values of the N gene and ORF1ab gene in the three positive samples by Daan, Sansure, and Hybribio NAATs are shown in 
      <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>, and the Ct values of the ORF1ab gene were close to the positive threshold of the three NAATs declared in their instructions.
     </p>
     <div id="T4" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1228439/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1228439/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1228439" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 4.</span>
       <p id="p-27" class="first-child">Comparison of NAATs to consensus results for detection of SARS-CoV-2.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T5" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1228428/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1228428/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1228428" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 5.</span>
       <p id="p-28" class="first-child">Ct values of the three clinical samples which was SARS-CoA-2 negative detected by Bioperfectus NAATs.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-29">The sensitivity and specificity for each NAAT were calculated based on consistent analysis. For Daan, Sansure, and Hybribio NAATs, their specificity and sensitivity were 100%. For Bioperfectus NAAT, the sensitivity was 81.25% and specificity was 100%. No significant difference was found in the sensitivity between Bioperfectus NAAT and the three other NAATs (P &gt;; 0.05).</p>
    </div>
   </div>
   <div class="section" id="sec-19">
    <h2 class="">Discussion</h2>
    <p id="p-30">The prevention of the COVID-19 epidemic is very grim. On the basis of China’s experience in preventing the further spread of the disease in the past 2 months, identifying COVID-19 from patients with other diseases as soon as possible has been proven to be particularly important to execute early isolation and early treatment (4, 5, 15). The use of RT-PCR technology to detect SARS-CoV-2 nucleic acid is an effective means to screen SARS-CoV-2 virus-infected patients. The current SARS-CoV-2 NAATs, which are widely used in China, were developed in response to the emergency situation of screening for SARS-CoV-2 infection. The performance of the assays is exactly what all testers want to know. In this study, four NAATs were evaluated. The results showed that the LODs of the four assays were significantly higher than the LODs declared in their instructions, suggesting that each laboratory should re-evaluate the LOD of the reagents depending on the needs of their laboratory. The RNA detection data from pseudovirus cultures showed that the LOD of Sansure was slightly lower than that of Daan NAAT. The RNA detection data from clinical samples showed that the LOD of Bioperfectus NAAT was slightly higher than that of the three other reagents but still reached the detection sensitivity of 20 copies per PCR reaction. There was no significant difference in the LOD of SARS-CoV-2 virus between pseudovirus cultures and clinical samples for Daan and Sansure NAATs. Thus, pseudovirus cultures could be used as an alternative for clinical samples in the performance evaluation of LOD. In addition, the four assays showed high precision, and the CV value was less than 5% when the concentration was 20% higher than the LOD. There were no cross-reactions by the four assays with four common human coronaviruses, SARS, and MERS viruses and no cross-reaction with other common respiratory pathogens, such as respiratory syncytial virus, adenovirus, influenza virus, and 
     <em>Mycoplasma pneumoniae.</em>
    </p>
    <p id="p-31">In the present study, we further compared four NAATs for the detection of SARS-CoV-2 in clinical samples from suspected COVID-19 patients. Results showed that the clinical sensitivity and specificity of Daan, Sansure, and Hybribio NAATs were 100%, while three of the positive samples were missed by Bioperfectus NAAT. The Ct values of the ORF1ab gene of the three samples tested with Daan, Sansure, and Hybribio reagents were all close to 40, indicating that the concentration of the three samples may be lower than the LOD of Bioperfectus. Therefore, the LOD should be included in the reference index when selecting reagents for SARS-CoV-2 in the laboratory. Sequencing verification of the PCR-positive samples in this study suggested that the four assays did not show false positives in the detection of limited clinical samples. Although the small clinical sample size limited the optimal evaluation of the performance of these assays, the results of this study can still help laboratories to select an appropriate SARS-CoV-2 NAAT supplier and interpret their detection results.</p>
    <p id="p-32">Obtaining high-quality viral RNA from original clinical samples is crucial to ensure the accuracy of the detection results by NAATs. Therefore, the poor quality of extraction reagents and the unoptimized extraction process of viral RNA would affect the detection results. In this study, the performance of NAATs was evaluated by using the same viral RNA samples. Therefore, the difference in the experimental results only reflected the difference in the performance of nucleic acid amplification for NAATs but not the diversity in the performance of the whole SARS-CoV-2 detection system, which usually includes the nucleic acid extraction system.</p>
    <p id="p-33">In addition, RNA viruses may show substantial genetic variability. This could result in mismatch between the primer and probes with the target sequence, which can diminish the assay performance or result in false negative results. Therefore, the manufacturer of NAATs should pay attention to the sequence update information of the virus database at any time to ensure no mutation exists in the genome sequences corresponding to the primers, especially for the sequences corresponding to the 3’ end of the primers.</p>
    <p id="p-34">In summary, the four SARS-CoV-2 NAATs commonly used with commercial assays showed good reliability, but the actual LOD was significantly higher than the declared LOD. The analytic sensitivity of Daan, Sansure, and Hybribio NAATs was slightly higher than that of Bioperfectus NAAT.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-35">The authors would like to thank the Chinese Academy of Metrology, Sansure, Bioperfectus, and Hybribio for supplying the test kits.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.1" data-doi="10.1056/NEJMoa2001017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Niu</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhan</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">GF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth cit-collab">China Novel Coronavirus I, Research T</span>. 
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-vol">382</span>:
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">733</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=http://www.n&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.010975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.26.010975v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Horby</span>
          <span class="cit-name-given-names">PW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayden</span>
          <span class="cit-name-given-names">FG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">GF.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">A novel coronavirus outbreak of global health concern</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>:
         <span class="cit-fpage">470</span>–
         <span class="cit-lpage">473</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.3" data-doi="10.1056/NEJMoa2001316">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guan</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tong</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>
          <span class="cit-name-given-names">KSM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lau</span>
          <span class="cit-name-given-names">EHY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wong</span>
          <span class="cit-name-given-names">JY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xing</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiang</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tu</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shao</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tao</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lam</span>
          <span class="cit-name-given-names">TTY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">JTK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">GF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cowling</span>
          <span class="cit-name-given-names">BJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>
          <span class="cit-name-given-names">GM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Feng</span>
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus-Infected Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001316</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001316%26rft_id%253Dinfo%253Apmid%252F31995857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001316&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31995857&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.010975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">World Health Organization</span> (
         <span class="cit-pub-date">2020</span>). 
         <abbr class="cit-jnl-abbrev">Novel Coronavirus(2019-nCoV): Situation Report</abbr>..
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">World Health Organization</span> (
         <span class="cit-pub-date">2020</span>). 
         <abbr class="cit-jnl-abbrev">Clinical management of severe acute respiratory infection when novel coronavirus (2019-nCoV) infection is suspected: interim guidance (2020-01-28)</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">De Clercq</span>
          <span class="cit-name-given-names">E.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">Therapeutic options for the 2019 novel coronavirus (2019-nCoV)</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Drug Discov</abbr>
         <span class="cit-vol">19</span>:
         <span class="cit-fpage">149</span>–
         <span class="cit-lpage">150</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BDrug%2BDiscov%26rft.volume%253D19%26rft.spage%253D149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">MX</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ang</span>
          <span class="cit-name-given-names">IYH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">SHX</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lewis</span>
          <span class="cit-name-given-names">RF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">JI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gutierrez</span>
          <span class="cit-name-given-names">RA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gwee</span>
          <span class="cit-name-given-names">SXW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chua</span>
          <span class="cit-name-given-names">PEY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">XY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yap</span>
          <span class="cit-name-given-names">RK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">HY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Teo</span>
          <span class="cit-name-given-names">YY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cook</span>
          <span class="cit-name-given-names">AR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yap</span>
          <span class="cit-name-given-names">JC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsu</span>
          <span class="cit-name-given-names">LY.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">Potential Rapid Diagnostics, Vaccine and Therapeutics for 2019 Novel Coronavirus (2019-nCoV): A Systematic Review</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Med</abbr>
         <span class="cit-fpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.8" data-doi="10.1093/clinchem/hvaa071">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Binnicker</span>
          <span class="cit-name-given-names">MJ.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">Emergence of a Novel Coronavirus Disease (COVID-19) and the Importance of Diagnostic Testing: Why Partnership between Clinical Laboratories, Public Health Agencies, and Industry Is Essential to Control the Outbreak</span>. 
         <abbr class="cit-jnl-abbrev">Clin Chem</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/clinchem/hvaa071</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClin%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fclinchem%252Fhvaa071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/clinchem/hvaa071&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.26.010975v1.9" data-doi="10.1093/clinchem/hvaa029">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>
          <span class="cit-name-given-names">DKW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>
          <span class="cit-name-given-names">SMS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hui</span>
          <span class="cit-name-given-names">KPY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Krishnan</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">DYM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>
          <span class="cit-name-given-names">CKC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peiris</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Poon</span>
          <span class="cit-name-given-names">LLM.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">Clin Chem</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/clinchem/hvaa029</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClin%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fclinchem%252Fhvaa029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/clinchem/hvaa029&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.10" data-doi="10.1128/JCM.00310-20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">JF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yip</span>
          <span class="cit-name-given-names">CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">To</span>
          <span class="cit-name-given-names">KK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">TH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wong</span>
          <span class="cit-name-given-names">SC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>
          <span class="cit-name-given-names">KH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fung</span>
          <span class="cit-name-given-names">AY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">AC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zou</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tsoi</span>
          <span class="cit-name-given-names">HW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Choi</span>
          <span class="cit-name-given-names">GK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tam</span>
          <span class="cit-name-given-names">AR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>
          <span class="cit-name-given-names">VC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">KH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tsang</span>
          <span class="cit-name-given-names">OT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuen</span>
          <span class="cit-name-given-names">KY.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">Improved molecular diagnosis of COVID-19 by the novel, highly sensitive and specific COVID-19-RdRp/Hel real-time reverse transcription-polymerase chain reaction assay validated in vitro and with clinical specimens</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JCM.00310-20</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00310-20%26rft_id%253Dinfo%253Apmid%252F32132196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjE0OiJKQ00uMDAzMTAtMjB2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI5LzIwMjAuMDMuMjYuMDEwOTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.11" data-doi="10.1002/jmv.25749">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">The transmission and diagnosis of 2019 novel coronavirus infection disease (COVID-19): A Chinese perspective</span>. 
         <abbr class="cit-jnl-abbrev">J Med Virol</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1002/jmv.25749</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.25749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/jmv.25749&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.26.010975v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zou</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruan</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hong</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kang</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">He</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yen</span>
          <span class="cit-name-given-names">HL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peiris</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-vol">382</span>:
         <span class="cit-fpage">1177</span>–
         <span class="cit-lpage">1179</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D1177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pu</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gong</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Long</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Si</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-article-title">Comparison of different samples for 2019 novel coronavirus detection by nucleic acid amplification tests</span>. 
         <abbr class="cit-jnl-abbrev">Int J Infect Dis</abbr>
         <span class="cit-vol">93</span>:
         <span class="cit-fpage">264</span>–
         <span class="cit-lpage">267</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BInfect%2BDis%26rft.volume%253D93%26rft.spage%253D264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.26.010975v1.14" data-doi="10.1128/JCM.00261-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>
          <span class="cit-name-given-names">SA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deak</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Humphries</span>
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Comparison of the AmpliVue, BD Max System, and illumigene Molecular Assays for Detection of Group B Streptococcus in Antenatal Screening Specimens</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>
         <span class="cit-vol">53</span>:
         <span class="cit-fpage">1938</span>–
         <span class="cit-lpage">41</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00261-15%26rft_id%253Dinfo%253Apmid%252F25788551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjUzLzYvMTkzOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzI5LzIwMjAuMDMuMjYuMDEwOTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.03.26.010975v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">National Health Commission</span>. 
         <span class="cit-source">The guidelines for the diagnosis and treatment of severe acute respiratory syndrome coronavirus 2(SARS-CoV-2)infection (Pilot 7rd version)</span>. 
         <a href="http://www.nhc.gov.cn/yzygj/s7653p/202003/46c9294a7dfe4cef80dc7f5912eb1989/files/ce3e6945832a438eaae415350a8ce964.pdf">http://www.nhc.gov.cn/yzygj/s7653p/202003/46c9294a7dfe4cef80dc7f5912eb1989/files/ce3e6945832a438eaae415350a8ce964.pdf</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
