<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/10/23/815480.atom"/>
  <meta name="HW.pisa" content="biorxiv;815480v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="The effect of variant interference on de novo assembly for viral deep sequencing"/>
  <meta name="DC.Identifier" content="10.1101/815480"/>
  <meta name="DC.Date" content="2019-10-23"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Viruses have high mutation rates and generally exist as a mixture of variants in biological samples. Next-generation sequencing (NGS) approach has surpassed Sanger for generating long viral sequences, yet how variants affect NGS de novo assembly remains largely unexplored. Our results from &gt;;15,000 simulated experiments showed that presence of variants can turn an assembly of one genome into tens to thousands of contigs. This “variant interference” (VI) is highly consistent and reproducible by ten most used de novo assemblers, and occurs independent of genome length, read length, and GC content. The main driver of VI is pairwise identities between viral variants. These findings were further supported by in silico simulations, where selective removal of minor variant reads from clinical datasets allow the “rescue” of full viral genomes from fragmented contigs. These results call for careful interpretation of contigs and contig numbers from de novo assembly in viral deep sequencing."/>
  <meta name="DC.Contributor" content="Christina J. Castro"/>
  <meta name="DC.Contributor" content="Rachel L. Marine"/>
  <meta name="DC.Contributor" content="Edward Ramos"/>
  <meta name="DC.Contributor" content="Terry Fei Fan Ng"/>
  <meta name="article:published_time" content="2019-10-23"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="The effect of variant interference on de novo assembly for viral deep sequencing"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Viruses have high mutation rates and generally exist as a mixture of variants in biological samples. Next-generation sequencing (NGS) approach has surpassed Sanger for generating long viral sequences, yet how variants affect NGS &lt;;i&gt;;de novo&lt;;/i&gt;; assembly remains largely unexplored. Our results from &amp;;gt;15,000 simulated experiments showed that presence of variants can turn an assembly of one genome into tens to thousands of contigs. This “variant interference” (VI) is highly consistent and reproducible by ten most used &lt;;i&gt;;de novo&lt;;/i&gt;; assemblers, and occurs independent of genome length, read length, and GC content. The main driver of VI is pairwise identities between viral variants. These findings were further supported by &lt;;i&gt;;in silico&lt;;/i&gt;; simulations, where selective removal of minor variant reads from clinical datasets allow the “rescue” of full viral genomes from fragmented contigs. These results call for careful interpretation of contigs and contig numbers from &lt;;i&gt;;de novo&lt;;/i&gt;; assembly in viral deep sequencing.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;815480v2"/>
  <meta name="citation_id" content="815480v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/815480v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/815480v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/815480v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480.full.pdf"/>
  <meta name="citation_doi" content="10.1101/815480"/>
  <meta name="citation_num_pages" content="39"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="815480"/>
  <meta name="citation_author" content="Christina J. Castro"/>
  <meta name="citation_author_institution" content="Division of Viral Diseases, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention"/>
  <meta name="citation_author_institution" content="Oak Ridge Institute for Science and Education"/>
  <meta name="citation_author" content="Rachel L. Marine"/>
  <meta name="citation_author_institution" content="Division of Viral Diseases, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3875-2959"/>
  <meta name="citation_author" content="Edward Ramos"/>
  <meta name="citation_author_institution" content="General Dynamics Information Technology, Inc., contracting agency to the Office of Informatics, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention"/>
  <meta name="citation_author" content="Terry Fei Fan Ng"/>
  <meta name="citation_author_institution" content="Division of Viral Diseases, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention"/>
  <meta name="citation_author_email" content="ylz9@cdc.gov"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4815-8697"/>
  <meta name="citation_reference" content="Rasmussen AL, Katze MG. Genomic Signatures of Emerging Viruses: A New Era of Systems Epidemiology. Cell Host Microbe 19, 611–618 (2016)."/>
  <meta name="citation_reference" content="Leung P, Eltahla AA, Lloyd AR, Bull RA, Luciani F. Understanding the complex evolution of rapidly mutating viruses with deep sequencing: Beyond the analysis of viral diversity. Virus Res 239, 43–54 (2017)."/>
  <meta name="citation_reference" content="Pierce BG, Keck ZY, Foung SK. Viral evasion and challenges of hepatitis C virus vaccine development. Curr Opin Virol 20, 55–63 (2016)."/>
  <meta name="citation_reference" content="Duffy S, Shackelton LA, Holmes EC. Rates of evolutionary change in viruses: patterns and determinants. Nat Rev Genet 9, 267–276 (2008)."/>
  <meta name="citation_reference" content="Andino R, Domingo E. Viral quasispecies. Virology 479-480, 46–51 (2015)."/>
  <meta name="citation_reference" content="Henn MR, et al. Whole genome deep sequencing of HIV-1 reveals the impact of early minor variants upon immune recognition during acute infection. PLoS pathogens 8, e1002529–e1002529 (2012)."/>
  <meta name="citation_reference" content="Herbeck JT, et al. Demographic processes affect HIV-1 evolution in primary infection before the onset of selective processes. Journal of virology 85, 7523–7534 (2011)."/>
  <meta name="citation_reference" content="Jerzak G, Bernard KA, Kramer LD, Ebel GD. Genetic variation in West Nile virus from naturally infected mosquitoes and birds suggests quasispecies structure and strong purifying selection. The Journal of general virology 86, 2175–2183 (2005)."/>
  <meta name="citation_reference" content="Lauck M, et al. Analysis of hepatitis C virus intrahost diversity across the coding region by ultradeep pyrosequencing. Journal of virology 86, 3952–3960 (2012)."/>
  <meta name="citation_reference" content="Lin S-R, et al. Study of sequence variation of dengue type 3 virus in naturally infected mosquitoes and human hosts: implications for transmission and evolution. Journal of virology 78, 12717–12721 (2004)."/>
  <meta name="citation_reference" content="Murcia PR, et al. Intra- and interhost evolutionary dynamics of equine influenza virus. Journal of virology 84, 6943–6954 (2010)."/>
  <meta name="citation_reference" content="Vignuzzi M, Stone JK, Arnold JJ, Cameron CE, Andino R. Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population. Nature 439, 344–348 (2006)."/>
  <meta name="citation_reference" content="Thai KTD, et al. High-resolution analysis of intrahost genetic diversity in dengue virus serotype 1 infection identifies mixed infections. Journal of virology 86, 835–843 (2012)."/>
  <meta name="citation_reference" content="Yang X, et al. De novo assembly of highly diverse viral populations. BMC Genomics 13, 475 (2012)."/>
  <meta name="citation_reference" content="Deng X, et al. An ensemble strategy that significantly improves de novo assembly of microbial genomes from metagenomic next-generation sequencing data. Nucleic Acids Res 43, e46 (2015)."/>
  <meta name="citation_reference" content="Wommack KE, Bhavsar J, Ravel J. Metagenomics: read length matters. Applied and environmental microbiology 74, 1453–1463 (2008)."/>
  <meta name="citation_reference" content="Castro CJ, Ng TFF. U50: A New Metric for Measuring Assembly Output Based on Non-Overlapping, Target-Specific Contigs. J Comput Biol 24, 1071–1080 (2017)."/>
  <meta name="citation_reference" content="Montmayeur AM, et al. High-throughput next-generation sequencing of polioviruses. J Clin Microbiol 55, 606–615 (2017)."/>
  <meta name="citation_reference" content="Ng TFF, et al. High Variety of Known and New RNA and DNA Viruses of Diverse Origins in Untreated Sewage. Journal of Virology 86, 12161 (2012)."/>
  <meta name="citation_reference" content="Ma S, Du Z, Feng M, Che Y, Li Q. A severe case of co-infection with Enterovirus 71 and vaccine-derived Poliovirus type II. Journal of clinical virology : the official publication of the Pan American Society for Clinical Virology 72, 25–29 (2015)."/>
  <meta name="citation_reference" content="Jorba J, et al. Update on Vaccine-Derived Polioviruses - Worldwide, January 2017-June 2018. MMWR Morbidity and mortality weekly report 67, 1189–1194 (2018)."/>
  <meta name="citation_reference" content="Wick RR, Schultz MB, Zobel J, Holt KE. Bandage: interactive visualization of de novo genome assemblies. Bioinformatics (Oxford, England) 31, 3350–3352 (2015)."/>
  <meta name="citation_reference" content="Lu H, Giordano F, Ning Z. Oxford Nanopore MinION Sequencing and Genome Assembly. Genomics, Proteomics &amp;; Bioinformatics 14, 265–279 (2016)."/>
  <meta name="citation_reference" content="Herath D, Jayasundara D, Ackland D, Saeed I, Tang SL, Halgamuge S. Assessing Species Diversity Using Metavirome Data: Methods and Challenges. Comput Struct Biotechnol J 15, 447–455 (2017)."/>
  <meta name="citation_reference" content="Bunge J, Woodard L, Bohning D, Foster JA, Connolly S, Allen HK. Estimating population diversity with CatchAll. Bioinformatics 28, 1045–1047 (2012)."/>
  <meta name="citation_reference" content="Angly F, et al. PHACCS, an online tool for estimating the structure and diversity of uncultured viral communities using metagenomic information. BMC Bioinformatics 6, 41 (2005)."/>
  <meta name="citation_reference" content="Wang NF, et al. Diversity and Composition of Bacterial Community in Soils and Lake Sediments from an Arctic Lake Area. Frontiers in microbiology 7, 1170–1170 (2016)."/>
  <meta name="citation_reference" content="Rusch DB, et al. The Sorcerer II Global Ocean Sampling expedition: northwest Atlantic through eastern tropical Pacific. PLoS biology 5, e77 (2007)."/>
  <meta name="citation_reference" content="The Human Microbiome Project C, et al. Structure, function and diversity of the healthy human microbiome. Nature 486, 207 (2012)."/>
  <meta name="citation_reference" content="Turnbaugh PJ, Ley RE, Hamady M, Fraser-Liggett CM, Knight R, Gordon JI. The Human Microbiome Project. Nature 449, 804 (2007)."/>
  <meta name="citation_reference" content="Schneider WL, Roossinck MJ. Genetic Diversity in RNA Virus Quasispecies Is Controlled by Host-Virus Interactions. Journal of Virology 75, 6566 (2001)."/>
  <meta name="citation_reference" content="Gregori J, Perales C, Rodriguez-Frias F, Esteban JI, Quer J, Domingo E. Viral quasispecies complexity measures. Virology 493, 227–237 (2016)."/>
  <meta name="citation_reference" content="Huang W, Li L, Myers JR, Marth GT. ART: a next-generation sequencing read simulator. Bioinformatics (Oxford, England) 28, 593–594 (2012)."/>
  <meta name="citation_reference" content="Huang X, Madan A. CAP3: A DNA sequence assembly program. Genome Res 9, 868–877 (1999)."/>
  <meta name="citation_reference" content="Chevreux B, et al. Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs. Genome Res 14, 1147–1159 (2004)."/>
  <meta name="citation_reference" content="Chevreux BW, T.; Suhai, S. Genome sequence assembly using trace signals and additional sequence information. German conference on bioinformatics 99, 45–56 (1999)."/>
  <meta name="citation_reference" content="Jackman SD, et al. ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter. Genome Res 27, 768–777 (2017)."/>
  <meta name="citation_reference" content="citation_title=Research in Computational Molecular Biology;citation_year=2010"/>
  <meta name="citation_reference" content="Nurk S, Meleshko D, Korobeynikov A, Pevzner PA. metaSPAdes: a new versatile metagenomic assembler. Genome Research 27, 824–834 (2017)."/>
  <meta name="citation_reference" content="Luo R, et al. SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler. GigaScience 1, 18 (2012)."/>
  <meta name="citation_reference" content="Bankevich A, et al. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19, 455–477 (2012)."/>
  <meta name="citation_reference" content="Grabherr MG, et al. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nat Biotechnol 29, 644–652 (2011)."/>
  <meta name="citation_reference" content="Kearse M, et al. Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics 28, 1647–1649 (2012)."/>
  <meta name="citation_reference" content="Phillippy AM. New advances in sequence assembly. Genome research 27, xi–xiii (2017)."/>
  <meta name="citation_reference" content="Olivarius S, Plessy C, Carninci P. High-throughput verification of transcriptional starting sites by Deep-RACE. BioTechniques 46, 130–132 (2009)."/>
  <meta name="citation_reference" content="Lagarde J, et al. Extension of human lncRNA transcripts by RACE coupled with long-read high-throughput sequencing (RACE-Seq). Nature Communications 7, 12339 (2016)."/>
  <meta name="twitter:title" content="The effect of variant interference on de novo assembly for viral deep sequencing"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Viruses have high mutation rates and generally exist as a mixture of variants in biological samples. Next-generation sequencing (NGS) approach has surpassed Sanger for generating long viral sequences, yet how variants affect NGS de novo assembly remains largely unexplored. Our results from &gt;;15,000 simulated experiments showed that presence of variants can turn an assembly of one genome into tens to thousands of contigs. This “variant interference” (VI) is highly consistent and reproducible by ten most used de novo assemblers, and occurs independent of genome length, read length, and GC content. The main driver of VI is pairwise identities between viral variants. These findings were further supported by in silico simulations, where selective removal of minor variant reads from clinical datasets allow the “rescue” of full viral genomes from fragmented contigs. These results call for careful interpretation of contigs and contig numbers from de novo assembly in viral deep sequencing."/>
  <meta name="og-title" property="og:title" content="The effect of variant interference on de novo assembly for viral deep sequencing"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/815480v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Viruses have high mutation rates and generally exist as a mixture of variants in biological samples. Next-generation sequencing (NGS) approach has surpassed Sanger for generating long viral sequences, yet how variants affect NGS de novo assembly remains largely unexplored. Our results from &gt;;15,000 simulated experiments showed that presence of variants can turn an assembly of one genome into tens to thousands of contigs. This “variant interference” (VI) is highly consistent and reproducible by ten most used de novo assemblers, and occurs independent of genome length, read length, and GC content. The main driver of VI is pairwise identities between viral variants. These findings were further supported by in silico simulations, where selective removal of minor variant reads from clinical datasets allow the “rescue” of full viral genomes from fragmented contigs. These results call for careful interpretation of contigs and contig numbers from de novo assembly in viral deep sequencing."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-10-23"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>The effect of variant interference on de novo assembly for viral deep sequencing | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-962526 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Viruses have high mutation rates and generally exist as a mixture of variants in biological samples. Next-generation sequencing (NGS) approach has surpassed Sanger for generating long viral sequences, yet how variants affect NGS 
     <em>de novo</em> assembly remains largely unexplored. Our results from &gt;;15,000 simulated experiments showed that presence of variants can turn an assembly of one genome into tens to thousands of contigs. This “variant interference” (VI) is highly consistent and reproducible by ten most used 
     <em>de novo</em> assemblers, and occurs independent of genome length, read length, and GC content. The main driver of VI is pairwise identities between viral variants. These findings were further supported by 
     <em>in silico</em> simulations, where selective removal of minor variant reads from clinical datasets allow the “rescue” of full viral genomes from fragmented contigs. These results call for careful interpretation of contigs and contig numbers from 
     <em>de novo</em> assembly in viral deep sequencing.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-3">For many years, Sanger sequencing has been used to complement classical epidemiological and laboratory methods for investigating viral infections.
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup> As technologies have evolved, the emergence of next-generation sequencing (NGS), which drastically reduced the cost per base to generate sequence data for complete viral genomes, has allowed scientists to apply viral sequencing on a grander scale.
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup> Genomic sequencing is ideal for elucidating viral transmission pathways, characterizing emerging viruses, and locating genomic regions which are functionally important for evading the host immune system or antivirals.
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>
    </p>
    <p id="p-4">Genomic surveillance of viruses is particularly important in light of their rapid rate of evolution. Viruses have higher mutation rates than cellular-based taxa, with RNA viruses having mutation rates as high as 1.5 × 10
     <sup>−3</sup> mutations per nucleotide, per genomic replication cycle.
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup> Due to this high mutation rate, it is well established that most RNA viruses exist as a swarm of quasispecies,
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup> with each quasispecies containing unique single nucleotide polymorphisms (SNPs). The presence of these variants plays a key role in viral adaptation.
    </p>
    <p id="p-5">Due to viruses’ rapid evolution, a single clinical sample often contains a mixture of many closely related viruses. Viral quasispecies are mainly derived from intra-host evolution, with RNA viruses such as poliovirus, human immunodeficiency virus (HIV), hepatitis C (HCV), influenza, dengue, and West Nile viruses maintaining diverse quasispecies populations within a host.
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, 
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, 
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, 
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, 
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, 
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, 
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, 
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </sup> Conversely, the term “viral strains” often refers to different lineages of viruses found in separate hosts, or a co-infection of viruses in the same host due to multiple infection events. As a result, sequence divergence is usually higher when comparing viral strains compared to quasispecies. In this study, we use the term “variant” to encompass both quasispecies and strains regardless of how the variants originated in the biological samples.
    </p>
    <p id="p-6">Since many sequencing technologies produce reads that are significantly shorter than the target genome size, a process to construct contigs, scaffolds, and full-length genomes is needed. Reference-mapping and 
     <em>de novo</em> assembly are the two primary bioinformatic strategies for genome assembly. Reference-mapping requires a closely-related genome as input to align reads, while 
     <em>de novo</em> assembly generates contigs without the use of a reference genome, and therefore is the most suitable strategy for analyzing underexplored taxa
     <sup>
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup> or for viruses with high mutation and/or recombination rates.
    </p>
    <p id="p-7">In this study, we first examined how often NGS and 
     <em>de novo</em> assembly were applied in viral sequencing in GenBank Nucleotide entries (
     <a href="https://www.ncbi.nlm.nih.gov/nucleotide/">www.ncbi.nlm.nih.gov/nucleotide/</a>). Then we investigated how the presence of variants affected assembly results - simulated and clinical NGS datasets were analyzed using multiple assembly programs to explore the effects of genome variant relatedness, read length, and genome length on the resulting contig distribution.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>The rise of NGS and 
      <em>de novo</em> assembler use in GenBank viral sequences
     </h3>
     <p id="p-8">GenBank viral entries from 1982-2017 were collected and analyzed, with extensive analyses performed to evaluate technologies and bioinformatics programs cited in records deposited between 2011 and 2017. Through 2017, there were over 2.3 million viral entries in GenBank; however, over 70% (1.7 million) do not specify a sequencing technology [
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplement Table S1</a>] due to the looser data requirement in earlier years. When looking at recently deposited records (2014-2017), the Illumina sequencing platform was the most common NGS platform used for viral sequencing, with about a 2-fold increase over the next most popular NGS platform [
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1d</a> &amp;; 
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">e</a>]. When long sequences (≥2,000 nt) are considered, NGS technologies surpassed Sanger in 2017 as the dominant strategy for sequencing, comprising 53.8% (14,653/27,217) of entries compared to 46.2% of entries (12,564/27,217) for Sanger [
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1f</a> and 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Supplement Table S2</a>].
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/965855/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/965855/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/965855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Supplement Table S1.</span>
       <span class="caption-title">Total counts from NCBI’s GenBank non-redundant nucleotide database.</span>
       <p id="p-55" class="first-child">
        <strong>†</strong>
        <em>Total count</em> is the combination of all sequencing technologies listed for each entry plus the total number of entries with sequencing technology omitted. This number is higher than the 
        <em>Total # of viral entries in GenBank</em> because it accounts for all entries with multiple sequencing technologies listed.
       </p>
       <p id="p-56">Sequencing Technology, Seq. Tech.; Oxford Nanopore, Oxford NP; Pacific Biosciences, PacBio</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/965853/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/965853/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/965853" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Supplement Table S2.</span>
       <span class="caption-title">Total count of sequencing technologies for sequences &gt;;2000 nt in the NCBI GenBank non-redundant nucleotide database for years 2012–2017.</span>
       <p id="p-57" class="first-child">These numbers were found with the following search criteria: “viruses,” “genomic RNA/DNA,” “GenBank (No RefSeq),” length: 2000 to 2000000, release date: 1/1/201X to 12/31/201X, and “sequencing technology” in any field.</p>
       <p id="p-58">Oxford Nanopore, Oxford NP; Pacific Biosciences, PacBio</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-9">Hybrid sequencing approaches, where researchers use more than one sequencing technology to generate complete viral sequences, have also become more common over the past several years. The most common combination observed was 454 and Sanger (18,002 entries), likely due to the early emergence of the 454 technology compared to other NGS platforms [
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1c</a> and 
      <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Supplement Table S3</a>]. However, combining Illumina with various other sequencing platforms is quite commonplace (&gt;;10,000 entries).
     </p>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/965846/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/965846/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/965846" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Supplement Table S3.</span>
       <span class="caption-title">Total counts from NCBI’s GenBank non-redundant nucleotide database with multiple sequencing technologies listed per entry.</span>
       <p id="p-59" class="first-child">Blank fields indicate absence of entries for the corresponding category.</p>
       <p id="p-60">Sequencing Technologies, Seq. Techs.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Trends and patterns of sequencing technology and assembly methods of viral entries in the GenBank database. (a) Cumulative frequency histogram of all viral entries in GenBank from Jan. 1, 1982 through Dec. 31, 2017 (total=2,338,775 entries). (b) Count of all viral entries with at least one Sequencing Technology documented for the years 1982-2017. For panels (b) and (d), the [[#x201C]]Other[[#x201D]] category denotes entries with the Sequencing Technology field omitted or mis-assigned. (c) Relationship between viral entries listing one or two Sequencing Technologies during 1982[[#x2013]]2017. The number inside the circle indicates viral entries with only one Sequencing Technology listed; the number adjacent to the line indicates entries combining two Sequencing Technologies. The thicker the connection line, the stronger the relationship. (d and e) Percentage ratio graph of all viral entries with Sequencing Technology documented for the years 2010[[#x2013]]2017, with (d) and without (e) the Other category. The majority of entries in earlier years include omissions classified under the Other category, which is detailed in Supplement Table S1. (f) Percentage ratio graph of viral entries with length greater than 2000 nt that have been documented with one of the seven Sequencing Technologies for the years 2012[[#x2013]]2017. The seven technologies includes Sanger (n=1) and NGS technologies (n=6). (g) Percentage ratio graph of viral entries with length greater than 2000 nt and that have been documented with one of the six NGS as the Sequencing Technology for the years 2012[[#x2013]]2017. Compared to panel (f), Sanger is excluded in this graph. (h) Assembly method of viral entries greater than 2000 nt, showing percentage ratio graph of entries with at least one Assembly Method. For (h) and (i), the Other category describes assembly methods outside of the 18 most popular programs investigated. (i) Reclassification of panel (h) by the nature of the assembly methods. The programs can be grouped into de novo assembler, reference-mapping assembler, and software that can perform both." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454539762" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Trends and patterns of sequencing technology and assembly methods of viral entries in the GenBank database.&lt;;/span&gt;; &lt;;strong&gt;;(a)&lt;;/strong&gt;; Cumulative frequency histogram of all viral entries in GenBank from Jan. 1, 1982 through Dec. 31, 2017 (total=2,338,775 entries). &lt;;strong&gt;;(b)&lt;;/strong&gt;; Count of all viral entries with at least one &lt;;em&gt;;Sequencing Technology&lt;;/em&gt;; documented for the years 1982-2017. For panels (b) and (d), the [[#x201C]]Other[[#x201D]] category denotes entries with the &lt;;em&gt;;Sequencing Technology&lt;;/em&gt;; field omitted or mis-assigned. &lt;;strong&gt;;(c)&lt;;/strong&gt;; Relationship between viral entries listing one or two &lt;;em&gt;;Sequencing Technologies&lt;;/em&gt;; during 1982[[#x2013]]2017. The number inside the circle indicates viral entries with only one &lt;;em&gt;;Sequencing Technology&lt;;/em&gt;; listed; the number adjacent to the line indicates entries combining two &lt;;em&gt;;Sequencing Technologies&lt;;/em&gt;;. The thicker the connection line, the stronger the relationship. &lt;;strong&gt;;(d and e)&lt;;/strong&gt;; Percentage ratio graph of all viral entries with &lt;;em&gt;;Sequencing Technology&lt;;/em&gt;; documented for the years 2010[[#x2013]]2017, with (d) and without (e) the &lt;;em&gt;;Other&lt;;/em&gt;; category. The majority of entries in earlier years include omissions classified under the &lt;;em&gt;;Other&lt;;/em&gt;; category, which is detailed in Supplement Table S1. &lt;;strong&gt;;(f)&lt;;/strong&gt;; Percentage ratio graph of viral entries with length greater than 2000 nt that have been documented with one of the seven &lt;;em&gt;;Sequencing Technologies&lt;;/em&gt;; for the years 2012[[#x2013]]2017. The seven technologies includes Sanger (n=1) and NGS technologies (n=6). &lt;;strong&gt;;(g)&lt;;/strong&gt;; Percentage ratio graph of viral entries with length greater than 2000 nt and that have been documented with one of the six NGS as the &lt;;em&gt;;Sequencing Technology&lt;;/em&gt;; for the years 2012[[#x2013]]2017. Compared to panel (f), Sanger is excluded in this graph. &lt;;strong&gt;;(h)&lt;;/strong&gt;; Assembly method of viral entries greater than 2000 nt, showing percentage ratio graph of entries with at least one &lt;;em&gt;;Assembly Method&lt;;/em&gt;;. For (h) and (i), the &lt;;em&gt;;Other&lt;;/em&gt;; category describes assembly methods outside of the 18 most popular programs investigated. &lt;;strong&gt;;(i)&lt;;/strong&gt;; Reclassification of panel (h) by the nature of the assembly methods. The programs can be grouped into &lt;;em&gt;;de novo&lt;;/em&gt;; assembler, reference-mapping assembler, and software that can perform both.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F1.medium.gif" width="440" height="401"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F1.medium.gif" width="440" height="401"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">Trends and patterns of sequencing technology and assembly methods of viral entries in the GenBank database.</span>
       <p id="p-10" class="first-child">
        <strong>(a)</strong> Cumulative frequency histogram of all viral entries in GenBank from Jan. 1, 1982 through Dec. 31, 2017 (total=2,338,775 entries). 
        <strong>(b)</strong> Count of all viral entries with at least one 
        <em>Sequencing Technology</em> documented for the years 1982-2017. For panels (b) and (d), the “Other” category denotes entries with the 
        <em>Sequencing Technology</em> field omitted or mis-assigned. 
        <strong>(c)</strong> Relationship between viral entries listing one or two 
        <em>Sequencing Technologies</em> during 1982–2017. The number inside the circle indicates viral entries with only one 
        <em>Sequencing Technology</em> listed; the number adjacent to the line indicates entries combining two 
        <em>Sequencing Technologies</em>. The thicker the connection line, the stronger the relationship. 
        <strong>(d and e)</strong> Percentage ratio graph of all viral entries with 
        <em>Sequencing Technology</em> documented for the years 2010–2017, with (d) and without (e) the 
        <em>Other</em> category. The majority of entries in earlier years include omissions classified under the 
        <em>Other</em> category, which is detailed in 
        <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Supplement Table S1</a>. 
        <strong>(f)</strong> Percentage ratio graph of viral entries with length greater than 2000 nt that have been documented with one of the seven 
        <em>Sequencing Technologies</em> for the years 2012–2017. The seven technologies includes Sanger (n=1) and NGS technologies (n=6). 
        <strong>(g)</strong> Percentage ratio graph of viral entries with length greater than 2000 nt and that have been documented with one of the six NGS as the 
        <em>Sequencing Technology</em> for the years 2012–2017. Compared to panel (f), Sanger is excluded in this graph. 
        <strong>(h)</strong> Assembly method of viral entries greater than 2000 nt, showing percentage ratio graph of entries with at least one 
        <em>Assembly Method</em>. For (h) and (i), the 
        <em>Other</em> category describes assembly methods outside of the 18 most popular programs investigated. 
        <strong>(i)</strong> Reclassification of panel (h) by the nature of the assembly methods. The programs can be grouped into 
        <em>de novo</em> assembler, reference-mapping assembler, and software that can perform both.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-11">
      <em>De novo</em> assembly programs (ABySS, BWA, Canu, Cap3, IDBA, MIRA, Newbler, SOAPdenovo, SPAdes, Trinity, and Velvet) have increased from less than 1% of viral sequence entries in 2012, to 20% of all viral sequence entries in 2017 [
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1h</a> &amp;; 
      <a id="xref-fig-1-6" class="xref-fig" href="#F1">i</a>]. A similar increase was observed for reference-mapping programs (i.e., Bowtie and Bowtie2), from 0.03% in 2012 to 6.5% in 2017. Multifunctional programs (Suppl. Information) that offer both assembly options were the most common programs cited for the years 2013-2017, but since the exact sequence assembly strategy used for these records is unknown, the contributions of 
      <em>de novo</em> assembly are likely underestimated. An expanded summary of the sequencing technologies and assembly approaches used for viral GenBank records is available in 
      <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Supplement Tables S1</a>–
      <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">S6</a>.
     </p>
     <div id="T4" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/965849/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/965849/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/965849" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Supplement Table S4.</span>
       <span class="caption-title">Total counts from NCBI’s GenBank non-redundant nucleotide database of all entries with three and four sequencing technologies listed</span>
       <p id="p-61" class="first-child">For example, there are a total of 6 entries in GenBank that have the following sequencing technologies listed: 454, Illumina, Ion Torrent, and Sanger for one sequence entry.</p>
       <p id="p-62">Pacific Biosciences, PacBio</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T5" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/965848/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/965848/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/965848" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Supplement Table S5.</span>
       <span class="caption-title">Total count of assembly programs used to generate sequences &gt;;2000 nt in the NCBI GenBank non-redundant nucleotide database.</span>
       <p id="p-63" class="first-child">These numbers were found with the following search criteria: “viruses,” “genomic RNA/DNA,” “GenBank (No RefSeq),” length: 2000 to 2000000, release date: 1/1/201X to 12/31/201X, and ‘”sequencing technology” in any field; the assembly method was then parsed out.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T6" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/965847/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/965847/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/965847" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Supplement Table S6.</span>
       <span class="caption-title">Supplement Table S6. The 10 
        <em>de novo</em> assemblers used for analysis of the simulated data, as categorized by their underlying assembly algorithms.
       </span>
       <p id="p-64" class="first-child">de Bruijn graph, DBG; overlap-layout-739 consensus, OLC.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Effect of variant assembly using popular 
      <em>de novo</em> assemblers
     </h3>
     <p id="p-12">After establishing the growing use of NGS technologies for viral sequencing, we next focused on understanding how the presence of viral variants may influence 
      <em>de novo</em> assembly output. We generated 247 simulated viral NGS datasets representing a continuum of pairwise identity (PID) between two viral variants, from 75% PID (one nucleotide difference every 4 nucleotides), to 99.6% PID (one nucleotide difference every 250 nucleotides) [
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>]. For Experiment 1, these datasets were assembled using 10 of the most used 
      <em>de novo</em> assembly programs [
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and 
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplement Figure S1a</a>] to evaluate their ability to assemble the two variants into their own respective contigs as the PID between the variants increases.
     </p>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Workflow diagrams of simulated data from data creation through de novo assembly. (a) Comparison of assemblers. First, an artificial reference genome and corresponding initial variant reads were created with the following constraints: (1) reference genome length: 100K; (2) GC% of reference genome: 50%; (3) read length: 250 nt; and (4) coverage: 50[[#xD7]]. Second, an artificial mutated variant genome and corresponding mutated variant reads were created 247 times, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants were then combined and used as input for 10 different de novo assemblers with varying underlying algorithms. A total of 2,470 assemblies were performed. (b) Comparison of genome length and GC%. First, 13 artificial reference genomes and corresponding initial variant reads were created for four different genome lengths (2Kb, 10Kb, 100Kb, and 1Mb), each specifying a different GC% ranging from 20%[[#x2013]]80%. In addition, two actual virus reference genomes from NCBI were included, NC_002058 and NC_002645, with genome lengths of 7,440 nt and 27,317 nt, respectively. Read lengths of 250 nt with a coverage of 50X were used for all genomes. Second, an artificial mutated variant genome and corresponding mutated variant reads were created 247 time, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants were then combined for each and used as input for the SPAdes de novo assembler. A total of 13,338 assemblies were performed. (c) Comparison of read length. First, an artificial reference genome and corresponding initial variant reads were created with the following constraints: (1) reference genome length: 100K; (2) GC% of reference genome: 50%; (3) read lengths: 50 nt, 100 nt, 150 nt, or 250 nt; and (4) coverage: 50[[#xD7]]. Second, an artificial mutated variant genome and corresponding mutated variant reads were created, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) up to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants created for each of the four read lengths were then grouped by read length size and used as input for SPAdes de novo assembler. A total of 538 assemblies were performed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454539762" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Workflow diagrams of simulated data from data creation through &lt;;em&gt;;de novo&lt;;/em&gt;; assembly.&lt;;/span&gt;; &lt;;strong&gt;;(a) Comparison of assemblers.&lt;;/strong&gt;; First, an artificial reference genome and corresponding initial variant reads were created with the following constraints: (1) reference genome length: 100K; (2) GC% of reference genome: 50%; (3) read length: 250 nt; and (4) coverage: 50[[#xD7]]. Second, an artificial mutated variant genome and corresponding mutated variant reads were created 247 times, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants were then combined and used as input for 10 different &lt;;em&gt;;de novo&lt;;/em&gt;; assemblers with varying underlying algorithms. A total of 2,470 assemblies were performed. &lt;;strong&gt;;(b) Comparison of genome length and GC%.&lt;;/strong&gt;; First, 13 artificial reference genomes and corresponding initial variant reads were created for four different genome lengths (2Kb, 10Kb, 100Kb, and 1Mb), each specifying a different GC% ranging from 20%[[#x2013]]80%. In addition, two actual virus reference genomes from NCBI were included, NC_002058 and NC_002645, with genome lengths of 7,440 nt and 27,317 nt, respectively. Read lengths of 250 nt with a coverage of 50X were used for all genomes. Second, an artificial mutated variant genome and corresponding mutated variant reads were created 247 time, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants were then combined for each and used as input for the SPAdes &lt;;em&gt;;de novo&lt;;/em&gt;; assembler. A total of 13,338 assemblies were performed. &lt;;strong&gt;;(c) Comparison of read length.&lt;;/strong&gt;; First, an artificial reference genome and corresponding initial variant reads were created with the following constraints: (1) reference genome length: 100K; (2) GC% of reference genome: 50%; (3) read lengths: 50 nt, 100 nt, 150 nt, or 250 nt; and (4) coverage: 50[[#xD7]]. Second, an artificial mutated variant genome and corresponding mutated variant reads were created, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) up to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants created for each of the four read lengths were then grouped by read length size and used as input for SPAdes &lt;;em&gt;;de novo&lt;;/em&gt;; assembler. A total of 538 assemblies were performed.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Supplement Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F7.medium.gif" width="347" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Supplement Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F7.medium.gif" width="347" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplement Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Supplement Figure S1.</span>
       <span class="caption-title">Workflow diagrams of simulated data from data creation through 
        <em>de novo</em> assembly.
       </span>
       <p id="p-53" class="first-child">
        <strong>(a) Comparison of assemblers.</strong> First, an artificial reference genome and corresponding initial variant reads were created with the following constraints: (1) reference genome length: 100K; (2) GC% of reference genome: 50%; (3) read length: 250 nt; and (4) coverage: 50×. Second, an artificial mutated variant genome and corresponding mutated variant reads were created 247 times, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants were then combined and used as input for 10 different 
        <em>de novo</em> assemblers with varying underlying algorithms. A total of 2,470 assemblies were performed. 
        <strong>(b) Comparison of genome length and GC%.</strong> First, 13 artificial reference genomes and corresponding initial variant reads were created for four different genome lengths (2Kb, 10Kb, 100Kb, and 1Mb), each specifying a different GC% ranging from 20%–80%. In addition, two actual virus reference genomes from NCBI were included, NC_002058 and NC_002645, with genome lengths of 7,440 nt and 27,317 nt, respectively. Read lengths of 250 nt with a coverage of 50X were used for all genomes. Second, an artificial mutated variant genome and corresponding mutated variant reads were created 247 time, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants were then combined for each and used as input for the SPAdes 
        <em>de novo</em> assembler. A total of 13,338 assemblies were performed. 
        <strong>(c) Comparison of read length.</strong> First, an artificial reference genome and corresponding initial variant reads were created with the following constraints: (1) reference genome length: 100K; (2) GC% of reference genome: 50%; (3) read lengths: 50 nt, 100 nt, 150 nt, or 250 nt; and (4) coverage: 50×. Second, an artificial mutated variant genome and corresponding mutated variant reads were created, each with a differing pairwise percent identity ranging from 1 mutation every 4 nucleotides (75% PID) up to 1 mutation in every 250 nucleotides (99.6% PID). The initial and mutated variants created for each of the four read lengths were then grouped by read length size and used as input for SPAdes 
        <em>de novo</em> assembler. A total of 538 assemblies were performed.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Workflow diagram of the investigation of variant simulated NGS reads through de novo assembly. First, in step 1, an artificial reference genome and corresponding initial variant reads were created with varying constraints such as genome length, GC content, read length, and assemblers, according to the experiment types as detailed in Supplement Figure S1. In the second step, an artificial mutated variant genome was created. The process is repeated to generate 247 different mutated variants with controlled mutation parameters[[#x2014]] starting with 1 mutation every 4 nucleotides (75% PID) and ending with a mutated variant with 1 mutation in every 250 nucleotides (99.6% PID). Mutated variant reads are also generated for each of the mutation parameters. In the third and fourth steps, the initial and mutated variants were then combined and used as input for de novo assembly for the three experiments, as detailed in Supplement Figure S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454539762" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Workflow diagram of the investigation of variant simulated NGS reads through &lt;;em&gt;;de novo&lt;;/em&gt;; assembly.&lt;;/span&gt;; First, in step 1, an artificial reference genome and corresponding initial variant reads were created with varying constraints such as genome length, GC content, read length, and assemblers, according to the experiment types as detailed in Supplement Figure S1. In the second step, an artificial mutated variant genome was created. The process is repeated to generate 247 different mutated variants with controlled mutation parameters[[#x2014]] starting with 1 mutation every 4 nucleotides (75% PID) and ending with a mutated variant with 1 mutation in every 250 nucleotides (99.6% PID). Mutated variant reads are also generated for each of the mutation parameters. In the third and fourth steps, the initial and mutated variants were then combined and used as input for &lt;;em&gt;;de novo&lt;;/em&gt;; assembly for the three experiments, as detailed in Supplement Figure S1.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F2.medium.gif" width="440" height="154"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F2.medium.gif" width="440" height="154"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Workflow diagram of the investigation of variant simulated NGS reads through 
        <em>de novo</em> assembly.
       </span>
       <p id="p-13" class="first-child">First, in step 1, an artificial reference genome and corresponding initial variant reads were created with varying constraints such as genome length, GC content, read length, and assemblers, according to the experiment types as detailed in 
        <a id="xref-fig-7-2" class="xref-fig" href="#F7">Supplement Figure S1</a>. In the second step, an artificial mutated variant genome was created. The process is repeated to generate 247 different mutated variants with controlled mutation parameters— starting with 1 mutation every 4 nucleotides (75% PID) and ending with a mutated variant with 1 mutation in every 250 nucleotides (99.6% PID). Mutated variant reads are also generated for each of the mutation parameters. In the third and fourth steps, the initial and mutated variants were then combined and used as input for 
        <em>de novo</em> assembly for the three experiments, as detailed in 
        <a id="xref-fig-7-3" class="xref-fig" href="#F7">Supplement Figure S1</a>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-14">One key observation is that the assembly result can change from two (correct) contigs to many (unresolvable) contigs simply by having variant reads; the presence of viral variants affected the contig assembly output of all 10 assemblers tested. The output of the SPAdes, MetaSPAdes, ABySS, Cap3, and IDBA assemblers shared a few commonalities, demonstrated by a conceptual model in 
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. First, below a certain PID, when viral variants have enough distinct nucleotides to resolve the two variant contigs, the 
      <em>de novo</em> assemblers produced two contigs correctly [
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>]. We refer to this as “variant distinction” (VD), with the highest pairwise identity where this occurs as the VD threshold. Above this threshold, the assemblers produced tens to thousands of contigs [
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>], a phenomenon we define as “variant interference” (VI). As PID between the variants continue to increase, the 
      <em>de novo</em> assemblers can no longer distinguish between the variants and assembled all the reads into a single contig, a phenomenon we define as “variant singularity” (VS). [
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>]. The lowest pairwise identity where a single contig is assembled is the VS threshold.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The number of contigs generated by different de novo assemblers using simulated data containing variants differed with a range of percentage identities (PID). Blue denotes de Bruijn graph assemblers (DBG); green denotes overlap-layout-consensus assemblers (OLC); orange denotes commercialized proprietary algorithms. Variant distinction, VD; variant interference, VI; variant singularity, VS. *For SOAPdenovo2, several data points returned zero contigs due to a well-documented segmentation fault error. (a) Schematic diagram depicting concepts of the VD, VI, and VS, and their relationship to PID. (b) Comparison of output from 10 different assemblers. The number of contigs produced by each de novo assemblers at different variant PID ranges (75%[[#x2013]]99.6%) were shown. (c) Close-up of PID ranges where variant interference is the most apparent. The y-axis denotes the number of contigs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454539762" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The number of contigs generated by different &lt;;em&gt;;de novo&lt;;/em&gt;; assemblers using simulated data containing variants differed with a range of percentage identities (PID).&lt;;/span&gt;; Blue denotes de Bruijn graph assemblers (DBG); green denotes overlap-layout-consensus assemblers (OLC); orange denotes commercialized proprietary algorithms. Variant distinction, VD; variant interference, VI; variant singularity, VS. *For SOAPdenovo2, several data points returned zero contigs due to a well-documented segmentation fault error. &lt;;strong&gt;;(a) Schematic diagram depicting concepts of the VD, VI, and VS, and their relationship to PID. (b) Comparison of output from 10 different assemblers.&lt;;/strong&gt;; The number of contigs produced by each &lt;;em&gt;;de novo&lt;;/em&gt;; assemblers at different variant PID ranges (75%[[#x2013]]99.6%) were shown. &lt;;strong&gt;;(c) Close-up of PID ranges where variant interference is the most apparent.&lt;;/strong&gt;; The y-axis denotes the number of contigs.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F3.medium.gif" width="440" height="413"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F3.medium.gif" width="440" height="413"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">The number of contigs generated by different 
        <em>de novo</em> assemblers using simulated data containing variants differed with a range of percentage identities (PID).
       </span>
       <p id="p-15" class="first-child">Blue denotes de Bruijn graph assemblers (DBG); green denotes overlap-layout-consensus assemblers (OLC); orange denotes commercialized proprietary algorithms. Variant distinction, VD; variant interference, VI; variant singularity, VS. *For SOAPdenovo2, several data points returned zero contigs due to a well-documented segmentation fault error. 
        <strong>(a) Schematic diagram depicting concepts of the VD, VI, and VS, and their relationship to PID. (b) Comparison of output from 10 different assemblers.</strong> The number of contigs produced by each 
        <em>de novo</em> assemblers at different variant PID ranges (75%–99.6%) were shown. 
        <strong>(c) Close-up of PID ranges where variant interference is the most apparent.</strong> The y-axis denotes the number of contigs.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-16">Slight differences in the variant interference patterns (relative to the canonical variant interference model) were observed for the 10 assemblers investigated. VD was observed for SPAdes, MetaSPAdes, and ABySS assemblers. While it was not observed with Cap3 and IDBA with the current simulated data parameters, we speculate that VD may occur at a lower PID level for these assemblers than tested in this study. The PID range where VI was observed was distinct for each 
      <em>de novo</em> assembler [
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>]. During VI, SPAdes produced as many as 134 contigs and ABySS produced 3,076 contigs, while MetaSPAdes, Cap3, and IDBA produced up to 10.
     </p>
     <p id="p-17">A different pattern was observed for Mira, Trinity, and SOAPdenovo2 assemblers. The average number of contigs generated by Mira, Trinity, and SOAPdenovo2 was 5, 36, and 283, respectively across all variant PIDs from 75%–99.96%. Specifically, Mira and Trinity generated fewer contigs at low PID, but produced many contigs when the two variants reach 97.1% PID and 96.0% PID, respectively. For SOAPdenovo2, a larger number of contigs were produced regardless of the PID. This indicates that these assemblers generally have major challenges producing a single genome; this has been observed in previous studies comparing assembly performance.
      <sup>
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
      </sup>
     </p>
     <p id="p-18">Finally, Geneious and CLC were the least affected by VI in the simulated datasets tested, returning only 1–5 contigs for all pairwise identities. CLC’s assembly algorithm primarily returned a single contig over the range of PIDs tested (218/247 simulations; 88.3%), thus favoring VS. In comparison, Geneious predominantly distinguished the two variants (234/247 simulations; 94.7%), favoring VD.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Effect of GC content and genome length on variant assembly</h3>
     <p id="p-19">For Experiment 2, we focused our study on evaluating whether VI observed in SPAdes 
      <em>de novo</em> assembly is influenced by the GC content or genome length of the pathogen. Two datasets were used for the evaluation: reads generated from four artificial genomes ranging in length from 2 Kb to 1 Mb, as well as from genome sequences of poliovirus (NC_002058; 7,440 nt in length) and coronavirus (NC_002645; 27,317 nt in length). No discernable correlation was observed between the GC content of variant genomes and the degree of VI for any of the simulated datasets [Supplemental Dataset S2, p &lt;; 0.0001]. Therefore, for subsequent analyses examining the effects of genome length on VI, the number of contigs at each PID level was obtained by averaging the 13 GC simulations.
     </p>
     <p id="p-20">Notably, no matter the genome length, SPAdes produced vastly more contigs (i.e., VI) in a constant, narrow range of PID [99%–99.21% ; 
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a> &amp;; 
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">b</a>]. The effect of variants on assembly was characterized by the three distinct intervals described previously: VD at lower PIDs, VI [
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4b</a>], and VS at higher PIDs for all genome lengths. For example, during VS, a single contig was generated when the two variants shared ≥99.22% PID, but tens to thousands of contigs were generated at a slightly lower PID of 99.21%. This PID threshold, 99.21%, marked the drastic transition from VS to VI, whereas the transition from VI to VD (i.e., the VD threshold) occurred at 98.99% PID [
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4b</a>]. A correlation was observed between genome length and the number of contigs produced during VI, where longer genomes returned proportionally more contigs as expected as total VI occurrence should increase with length [
      <em>r</em>
      <sup>2</sup> = 0.967; p &lt;;0.0001 
      <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4b</a> and 
      <a id="xref-fig-4-6" class="xref-fig" href="#F4">4c</a>].
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The effect of genome length and read length on de novo assembly of simulated variants across a range of percentage identities (PID). (a &amp;; b) Comparison of genome lengths. Six different genome lengths were assembled and the final contig counts were tallied across varying PID thresholds (75%[[#x2013]]99.6%). For the simulated genome lengths of 2Kb, 10kb, 100Kb, and 1Mb, the average of contig number at each PID was plotted. Panel (b) shows the close-up view where interference was the most prominent. For all six genome lengths and each of the 13 iterations, VI consistently occurred in the same range of PID (99.00%[[#x2013]]99.24%). The assembly makes a transition from VD to VI at the threshold of 99.00%, and it makes a transition from VI to VS at the threshold of 99.24%. Also, the longer the genome length, the more contigs produced during VI. (c) The relationship between genome length and the total number of contigs produced. Data from panel (a) were plotted on a logarithmic scale. The total number of contigs produced is significantly dependent on the genome size (r2=0.967; p-value" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454539762" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The effect of genome length and read length on &lt;;em&gt;;de novo&lt;;/em&gt;; assembly of simulated variants across a range of percentage identities (PID). (a &amp;; b) Comparison of genome lengths.&lt;;/span&gt;; Six different genome lengths were assembled and the final contig counts were tallied across varying PID thresholds (75%[[#x2013]]99.6%). For the simulated genome lengths of 2Kb, 10kb, 100Kb, and 1Mb, the average of contig number at each PID was plotted. Panel (b) shows the close-up view where interference was the most prominent. For all six genome lengths and each of the 13 iterations, VI consistently occurred in the same range of PID (99.00%[[#x2013]]99.24%). The assembly makes a transition from VD to VI at the threshold of 99.00%, and it makes a transition from VI to VS at the threshold of 99.24%. Also, the longer the genome length, the more contigs produced during VI. &lt;;strong&gt;;(c) The relationship between genome length and the total number of contigs produced.&lt;;/strong&gt;; Data from panel (a) were plotted on a logarithmic scale. The total number of contigs produced is significantly dependent on the genome size (r&lt;;sup&gt;;2&lt;;/sup&gt;;=0.967; p-value&lt;;0.0001). &lt;;strong&gt;;(d and e) The effect of read length in variant assembly with a genome size of 100K.&lt;;/strong&gt;; Simulated data with four different read lengths were created and assembled, and the final contig counts were tallied across varying PID thresholds (75%[[#x2013]]99.6%). Panel (e) shows the close-up view where interference was the most apparent. When longer read lengths were used, the variant interference PID range was much narrower than when shorter read lengths were used to build contigs.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F4.medium.gif" width="440" height="245"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F4.medium.gif" width="440" height="245"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <span class="caption-title">The effect of genome length and read length on 
        <em>de novo</em> assembly of simulated variants across a range of percentage identities (PID). (a &amp;; b) Comparison of genome lengths.
       </span>
       <p id="p-21" class="first-child">Six different genome lengths were assembled and the final contig counts were tallied across varying PID thresholds (75%–99.6%). For the simulated genome lengths of 2Kb, 10kb, 100Kb, and 1Mb, the average of contig number at each PID was plotted. Panel (b) shows the close-up view where interference was the most prominent. For all six genome lengths and each of the 13 iterations, VI consistently occurred in the same range of PID (99.00%–99.24%). The assembly makes a transition from VD to VI at the threshold of 99.00%, and it makes a transition from VI to VS at the threshold of 99.24%. Also, the longer the genome length, the more contigs produced during VI. 
        <strong>(c) The relationship between genome length and the total number of contigs produced.</strong> Data from panel (a) were plotted on a logarithmic scale. The total number of contigs produced is significantly dependent on the genome size (r
        <sup>2</sup>=0.967; p-value&lt;;0.0001). 
        <strong>(d and e) The effect of read length in variant assembly with a genome size of 100K.</strong> Simulated data with four different read lengths were created and assembled, and the final contig counts were tallied across varying PID thresholds (75%–99.6%). Panel (e) shows the close-up view where interference was the most apparent. When longer read lengths were used, the variant interference PID range was much narrower than when shorter read lengths were used to build contigs.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Effect of read length on variant assembly</h3>
     <p id="p-22">The read length of a given NGS dataset will vary depending on the sequencing platform and kits utilized to generate the data. Since read length is an important factor for 
      <em>de novo</em> assembly success,
      <sup>
       <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
      </sup> we hypothesized that it may also influence the ability to distinguish viral variants. For Experiment 3, using SPAdes we investigated assemblies with four typical read lengths: 50, 100, 150, and 250 nt. At longer read lengths, the VD threshold occurred at higher PIDs [
      <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4d</a> &amp;; 
      <a id="xref-fig-4-8" class="xref-fig" href="#F4">e</a>]. Also, with increasing read length, the width of the PID window where VI occurs gradually decreased from a 1.52% spread to a 0.21% spread [
      <a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4e</a>]. This indicates that longer reads are better for distinguishing viral variants with high PIDs.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>
      <em>In silico</em> experiments examining variant assembly with NGS data derived from clinical samples
     </h3>
     <p id="p-23">For clinical samples, assembly of viral genomes is affected by multiple factors other than the presence of variants, including sequencing error rate, host background reads, depth of genome coverage, and the distribution (i.e., pattern) of genome coverage. We next utilized viral NGS data generated from four picornavirus-positive clinical samples (one coxsackievirus B5, one enterovirus A71, and two parechovirus A3) to explore VI in datasets representative of data that may be encountered during routine NGS. The NGS data for each sample was partitioned into four bins of read data: (1) total reads after quality control (
      <strong>T</strong>); (2) major variants only (
      <strong>M</strong>); (3) major and minor variants only (
      <strong>Mm</strong>); and (4) major variants and background non-viral reads only (
      <strong>MB</strong>) [
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>]. These binned datasets were then assembled separately using three assembly programs: SPAdes, Cap3, and Geneious. By comparing these manipulations, we aimed to test the hypothesis that minor variants directly affect the performance of assembly through VI in real clinical NGS data.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The effect of variant interference in a real dataset from a clinical sample containing enterovirus A71 (EV-A71) and its variants. Fastq reads were partitioned into four components: trimmed reads after quality control (T), major variant (M), minor variant (m), and background (B). These reads were then combined into four different experiments: T, M, Mm, and MB and assembled using SPAdes. The contig representation schematic showing the abundance and length of the generated contigs reveals the impact of variant interference on de novo assembly. The bar graphs show the UG50% metric and the length of the longest contig. UG50% is a percentage-based metric that estimates length of the unique, non-overlapping contigs as proportional to the length of the reference genome.17 Unlike N50, UG50% is suitable for comparisons across different platforms or samples/viruses. More clinical samples and viruses are analyzed similarly in Figure 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454539762" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The effect of variant interference in a real dataset from a clinical sample containing enterovirus A71 (EV-A71) and its variants.&lt;;/span&gt;; Fastq reads were partitioned into four components: trimmed reads after quality control (T), major variant (M), minor variant (m), and background (B). These reads were then combined into four different experiments: T, M, Mm, and MB and assembled using SPAdes. The contig representation schematic showing the abundance and length of the generated contigs reveals the impact of variant interference on &lt;;em&gt;;de novo&lt;;/em&gt;; assembly. The bar graphs show the UG&lt;;sub&gt;;50&lt;;/sub&gt;;% metric and the length of the longest contig. UG&lt;;sub&gt;;50&lt;;/sub&gt;;% is a percentage-based metric that estimates length of the unique, non-overlapping contigs as proportional to the length of the reference genome.&lt;;sup&gt;;17&lt;;/sup&gt;; Unlike N&lt;;sub&gt;;50&lt;;/sub&gt;;, UG&lt;;sub&gt;;50&lt;;/sub&gt;;% is suitable for comparisons across different platforms or samples/viruses. More clinical samples and viruses are analyzed similarly in Figure 6.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F5.medium.gif" width="440" height="216"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F5.medium.gif" width="440" height="216"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <span class="caption-title">The effect of variant interference in a real dataset from a clinical sample containing enterovirus A71 (EV-A71) and its variants.</span>
       <p id="p-24" class="first-child">Fastq reads were partitioned into four components: trimmed reads after quality control (T), major variant (M), minor variant (m), and background (B). These reads were then combined into four different experiments: T, M, Mm, and MB and assembled using SPAdes. The contig representation schematic showing the abundance and length of the generated contigs reveals the impact of variant interference on 
        <em>de novo</em> assembly. The bar graphs show the UG
        <sub>50</sub>% metric and the length of the longest contig. UG
        <sub>50</sub>% is a percentage-based metric that estimates length of the unique, non-overlapping contigs as proportional to the length of the reference genome.
        <sup>
         <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
        </sup> Unlike N
        <sub>50</sub>, UG
        <sub>50</sub>% is suitable for comparisons across different platforms or samples/viruses. More clinical samples and viruses are analyzed similarly in 
        <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-25">Even with an adequate depth of coverage for genome reconstruction, assembly of total reads (
      <strong>T</strong>) in 11/12 experiments resulted in unresolved genome construction – resulting in numerous fragmented viral contigs [
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>]. The only exception was one experiment where one single PeV-A3 (S1) genome was assembled using Cap3. When only reads from the major variant were assembled (
      <strong>M</strong>), full genomes were obtained for all datasets using SPAdes and Cap3, and for the CV-B5 sample using Geneious. Conversely, assembly of the read bins containing major and minor variants (
      <strong>Mm</strong>) resulted in an increased number of contigs for 9 of the 12 sample and assembly software combinations tested [
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>], indicating that VI due to the addition of the minor variant reads likely adversely affected the assembly. The presence of background reads with major variant reads (
      <strong>MB</strong>) did not appear to affect viral genome assembly, as the UG
      <sub>50</sub>% value, a performance metric which only considers unique, non-overlapping contigs for target viruses
      <sup>
       <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>
      </sup>, was similar between 
      <strong>M</strong> and 
      <strong>MB</strong> datasets.
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The effect of variant interference on the assembly of four clinical datasets using three assembly programs. Fastq reads were partitioned into four categories: total reads (T), major variant (M), minor variant (m), and background (B). These reads were then combined into four different categories: T, M, major and minor variants (Mm), and major variant and background (MB). Datasets were assembled using SPAdes, Cap3, and Geneious. The bar graphs show the UG50% metric and the length of the longest contig. Coxsackievirus B5, CV-B5; Enterovirus A71, EV-A71; Parechovirus A3 (Sample 1), PeV-A3 (S1); Parechovirus A3 (Sample 2), PeV-A3 (S2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454539762" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The effect of variant interference on the assembly of four clinical datasets using three assembly programs.&lt;;/span&gt;; Fastq reads were partitioned into four categories: total reads (T), major variant (M), minor variant (m), and background (B). These reads were then combined into four different categories: T, M, major and minor variants (Mm), and major variant and background (MB). Datasets were assembled using SPAdes, Cap3, and Geneious. The bar graphs show the UG&lt;;sub&gt;;50&lt;;/sub&gt;;% metric and the length of the longest contig. Coxsackievirus B5, CV-B5; Enterovirus A71, EV-A71; Parechovirus A3 (Sample 1), PeV-A3 (S1); Parechovirus A3 (Sample 2), PeV-A3 (S2).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F6.medium.gif" width="440" height="148"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F6.medium.gif" width="440" height="148"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6.</span>
       <span class="caption-title">The effect of variant interference on the assembly of four clinical datasets using three assembly programs.</span>
       <p id="p-26" class="first-child">Fastq reads were partitioned into four categories: total reads (T), major variant (M), minor variant (m), and background (B). These reads were then combined into four different categories: T, M, major and minor variants (Mm), and major variant and background (MB). Datasets were assembled using SPAdes, Cap3, and Geneious. The bar graphs show the UG
        <sub>50</sub>% metric and the length of the longest contig. Coxsackievirus B5, CV-B5; Enterovirus A71, EV-A71; Parechovirus A3 (Sample 1), PeV-A3 (S1); Parechovirus A3 (Sample 2), PeV-A3 (S2).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-27">Our analysis of the GenBank quantified the decade-long expansion of NGS technologies and 
     <em>de novo</em> assembly for viral sequencing [
     <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>]. As the number of viral sequences in public databases continues to grow, an important question that naturally arises is how well current 
     <em>de novo</em> assembly programs perform for datasets with viral variants. Viral variants are expected in biological samples, with the number of variants and the extent of the sequence divergence between variants related to the mutation rate of the virus and the types of specimens that are being investigated. For example, samples containing rapidly evolving RNA viruses, such as poliovirus, HIV, and HCV
     <sup>
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, 
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, 
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
     </sup>, environmental samples,
     <sup>
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
     </sup> and clinical samples from immunosuppressed individuals
     <sup>
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, 
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
     </sup> usually harbor many variants. The ability to accurately distinguish variants is imperative to inform treatments (in the case of HIV and HCV), or determine whether a subpopulation of a more virulent variant is present.
    </p>
    <p id="p-28">Several experiments using simulated and clinical sample NGS data were performed to evaluate the ability of genome assembly programs to distinguish genome variants. All assemblers investigated generated fragmented assemblies when the data contained reads from two closely related variants due to “variant interference” (VI). Changes in pairwise identity (PID) as small as 0.01% between the two variants triggered an assembler to change from producing one or two contigs to producing hundreds of contigs. A quintessential example of this phenomenon was the SPAdes assembly of EV-A71 sequences during the 
     <em>in silico</em> experiments with clinical NGS data. Assembly of major variant reads resulted in one full length contig [
     <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>], whereas assembly of datasets containing the major and minor variant reads (
     <strong>Mm</strong> and 
     <strong>T</strong>) were characterized by a number of contigs, resulting in “cobwebs” of contig fragments when visualized using Bandage [
     <a id="xref-fig-8-1" class="xref-fig" href="#F8">Supplement Figure S2</a>].
     <sup>
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
     </sup> Even though the 
     <em>de novo</em> assembly graph linked the different contig fragments, the assembly could not differentiate the multiple routes of possible contig construction. We speculate this is the main reason why VI occurs in the context of de Bruijn graph assemblers.
    </p>
    <div id="F8" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Analysis of the final contig assembly graphs for a clinical sample containing enterovirus A71 (EV-A71) variants using Bandage. Based on the four assemblies in Figure 5, Bandage was used to display the contig graphs from each SPAdes output. The visualizations for T, Mm, and MB show the effects of variant interference, while M shows the ideal assembly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1454539762" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Analysis of the final contig assembly graphs for a clinical sample containing enterovirus A71 (EV-A71) variants using Bandage.&lt;;/span&gt;; Based on the four assemblies in Figure 5, Bandage was used to display the contig graphs from each SPAdes output. The visualizations for T, Mm, and MB show the effects of variant interference, while M shows the ideal assembly.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplement Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F8.medium.gif" width="440" height="408"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplement Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F8.medium.gif" width="440" height="408"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplement Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/23/815480/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplement Figure S2.</span>
      <span class="caption-title">Analysis of the final contig assembly graphs for a clinical sample containing enterovirus A71 (EV-A71) variants using Bandage.</span>
      <p id="p-54" class="first-child">Based on the four assemblies in 
       <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, Bandage was used to display the contig graphs from each SPAdes output. The visualizations for T, Mm, and MB show the effects of variant interference, while M shows the ideal assembly.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-29">The simulated experiments suggested that genome length and read length influence VI; A longer genome length will produce proportionally more contigs during VI, whereas a longer read length decreases the PID range where VI occurs [
     <a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>]. While longer read length improves assembly, unfortunately, platforms that produce long reads such as Oxford Nanopore and PacBio have higher error rates.
     <sup>
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>
     </sup> Until long reads can be produced at high fidelity, researchers must continue to rely on combining long- and short-read NGS datasets, and genome polishing techniques.
     <sup>
      <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>
     </sup>
    </p>
    <p id="p-30">The large number of contigs generated due to VI may be overwhelming for most researchers, and for viral ecology studies, could lead to over-estimation of species richness for methods that use contig spectra to infer richness, such as PHACCS or CatchAll.
     <sup>
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, 
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, 
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>
     </sup> This phenomenon may also impact studies differently depending on the overall goal for generating viral sequence data. For example, some researchers may only be concerned with generating a single major consensus genome, even when variants are detected in the data. This is common during outbreak responses for pathogens such as Ebola virus or Middle East respiratory syndrome coronavirus, where detection of SNPs (indicative of minor variants) is not immediately important. On the other hand, some investigations could favor distinguishing variants, such as for investigating the presence of vaccine-derived poliovirus, where a small number of SNPs may distinguish a vaccine-derived strain from a normal vaccine strain genome.
     <sup>
      <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>
     </sup>
    </p>
    <p id="p-31">The effects of VI could potentially be mitigated by running multiple assembly programs. A previous study testing bioinformatics strategies for assembling viral NGS data found that employing sequential use of de Bruijn graph and overlap-layout-consensus assemblers produced better assemblies.
     <sup>
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>
     </sup> We speculate that this “ensemble strategy”
     <sup>
      <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>
     </sup> may perform better because the multiple assemblers complement one another by having different VI PID thresholds. Future assembly approaches could also consider resolving the VI problem by possibly discriminating the major and minor variant reads first (perhaps by coverage or SNP analysis), and then assembling major and minor variant reads separately.
    </p>
    <p id="p-32">Since we observed VI occurring in simulated data from 2 Kb to 1 Mb genome lengths, we speculate that it may not only affect viral data but also larger draft contigs of bacteria and other microorganisms. Even though bacterial mutation rates are much lower than those of most viruses, bacterial variants are common. For environmental studies, bacterial metagenomes are known to contain many related taxa and variants 
     <sup>
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>
      <span class="xref-sep">,</span>
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, 
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>, 
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>
     </sup>, and in clinical investigations, minor bacterial variants can harbor SNPs that provide resistance against antimicrobials. This warrants future investigation into how the presence of variants may impact the assembly of other microbial datasets.
    </p>
    <p id="p-33">This study aimed to understand how variants affect assembly. As an initial investigation, many confounding factors were simplified for experimentation. Simulated variants studied here only depicted periodic mutations, set at regular intervals. However, in real viral data, SNPs are never evenly distributed across the genome, with zones of divergence and similarity.
     <sup>
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>, 
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>
     </sup> Other important factors which influence genome assembly include sequencing error rates, presence of repetitive regions, and coverage depth. We limited our experiments to keep these factors constant in order to investigate the sole effect of VI. Through this exploration, we demonstrated that reads from related genome variants adversely affect 
     <em>de novo</em> assembly. As NGS and 
     <em>de novo</em> assembly have become essential for generating full-length viral genomes, future studies should investigate the combined effects of the number and relative proportion of minor variants, as well as additional assembly factors (e.g., error rates) to supplement this work.
    </p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Methods</h2>
    <div id="sec-10" class="subsection">
     <h3>Analyzing NGS and assembler usage in the virus nucleotide collection in GenBank</h3>
     <p id="p-34">Viral sequence entries from the GenBank non-redundant nucleotide collection were obtained by downloading all sequences under the virus taxonomy through the end of 2017. A total of 2,338,775 GenBank entries were investigated.</p>
     <p id="p-35">The total number of viral sequences submitted annually in GenBank through December 2017 was calculated by filtering GenBank submissions by “virus,” followed by application of the following additional filtering steps: “genomic DNA/RNA” was selected and a “release date: Jan 1 through Dec 31” was applied to find the total number of viruses for a given year. A custom script was used to filter and count all documented sequencing technologies and assembly methods used for each GenBank entry.</p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Creation of simulated variant genomes and reads</h3>
     <p id="p-36">Simulated genomes were generated using custom scripts that randomly assign each nucleotide over a designated genome length with a weighted distribution dependent on the GC content [
      <a id="xref-fig-7-4" class="xref-fig" href="#F7">Supplement Figure S1</a>]. The random genomes were then screened using NCBI BLAST to insure no similarity/identity existed to any classified organism (i.e., no BLAST hits). These simulated genomes served as the initial variant genome (variant 1). To generate the mutated variant genomes (variant 2), a custom script was used to systematically introduce evenly distributed random mutations at rates from 1 mutation in every 4 nucleotides (75% PID) to 1 mutation in every 250 nucleotides (99.6% PID), incrementing by 1 nucleotide.
     </p>
     <p id="p-37">Following the generation of initial and mutated variant genomes, high-quality fastq reads were generated using ART,
      <sup>
       <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>
      </sup> simulating Illumina MiSeq paired-end runs at 50× coverage with 250 nt reads, DNA/RNA mean fragments size of 500, and quality score of 93. Fastq reads were combined in equal numbers for the initial and mutated variants, and used as input for subsequent 
      <em>de novo</em> assembly experiments [
      <a id="xref-fig-7-5" class="xref-fig" href="#F7">Supplement Figure S1</a>]. The same process was utilized to generate the artificial genomes, initial and mutated variant genomes, and reads for each of the experiments.
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Experiment 1: Analyzing simulated reads from variants using different 
      <em>de novo</em> assembly programs
     </h3>
     <p id="p-38">The simulated datasets containing reads from two variant genomes with nucleotide pairwise identity ranging from 75%–99.6% were analyzed using 10 different genome assembly programs. The 
      <em>de novo</em> assembly algorithms used were either overlap-layout-consensus (OLC) [Cap
      <sup>
       <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>
      </sup> and Mira
      <sup>
       <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>, 
       <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>
      </sup>], de Bruijn graph (DBG) [ABySS
      <sup>
       <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>
      </sup>, IDBA
      <sup>
       <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>
      </sup>, MetaSPAdes
      <sup>
       <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>
      </sup>, SOAPdenovo2
      <sup>
       <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>
      </sup>, SPAdes
      <sup>
       <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>
      </sup>, and Trinity
      <sup>
       <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>
      </sup>], or commercial software packages [CLC (
      <a href="https://www.qiagenbioinformatics.com/">https://www.qiagenbioinformatics.com/</a>) and Geneious
      <sup>
       <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>
      </sup>] whose assembly algorithms are proprietary [
      <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Supplement Table S6</a>]. The simulation settings for the reads were single-end reads, 250 nt read length, and 50× coverage. A total of 2,470 assemblies (247 datasets per genome × 10 assemblers) were analyzed [
      <a id="xref-fig-7-6" class="xref-fig" href="#F7">Supplement Figure S1a</a>].
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Experiment 2: Simulated data by varying genome length and GC content</h3>
     <p id="p-39">Artificial genomes were constructed for four genome lengths: 2 Kb, 10 Kb, 100 Kb, and 1 Mb, with varying GC content from 20%–80%, in 5% increments [
      <a id="xref-fig-7-7" class="xref-fig" href="#F7">Supplement Figure S1b</a>]. Datasets derived using one poliovirus genome (NC_002058) and one coronavirus genome (NC_002645) were also included in this analysis, representing the lower and upper genome length range typical of RNA viruses. The original GC content was kept constant for the poliovirus and coronavirus genomes. For all of these genomes, simulated reads for initial and mutated variants were generated as above.
     </p>
     <p id="p-40">A total of 13,338 SPAdes assemblies were generated, which included 12,844 assemblies for the four artificial genomes (247 datasets per genome × 4 artificial genome lengths × 13 GC content proportions × 1 assembler) and 494 assemblies for the poliovirus and coronavirus datasets (247 datasets per genome × 2 genomes × 1 assembler) [
      <a id="xref-fig-7-8" class="xref-fig" href="#F7">Supplement Figure S1b</a>]. JMP v13.0.0 (
      <a href="https://www.sas.com">www.sas.com</a>) was used to calculate Pearson’s correlation and Spearman’s ρ values to compare the association between percent GC levels and the number of contigs produced at each PID level. Since there was little statistical difference when comparing the contig numbers generated at varying percent GC for each of the four genome length datasets (Spearman’s ρ = 0.8299 to 0.9801, p&lt;;0.001) [Supplement Excel file], the final contig number was averaged across the 13 GC percentages at a given PID. The average contig number was used for plotting the contig assembly results vs percent PID for each simulated genome length [
      <a id="xref-fig-4-11" class="xref-fig" href="#F4">Figures 4a-b</a>].
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Experiment 3: Simulated data by varying read length</h3>
     <p id="p-41">Genome variants were generated as described above (“Creation of simulated variant genomes and reads”) for a genome of size 100 Kb with 50% GC; this was the starting initial variant genome. In this simulation, initial and mutation variant reads at four sequencing read lengths (50, 100, 150, and 250 nt) were created using ART. A total of 538 SPAdes assemblies were generated (47, 97, 147, and 247 datasets for the 50, 100, 150 and 250 nt read lengths, respectively) [
      <a id="xref-fig-7-9" class="xref-fig" href="#F7">Supplement Figure S1c</a>].
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Evaluation of NGS datasets from clinical samples</h3>
     <p id="p-42">Four datasets derived from clinical samples containing picornaviruses (one enterovirus A71 [EV-A71], one coxsackievirus B5 [CV-B5] and two parechovirus A3 [PeV-A3]) were analyzed for this experiment, as previous sequencing analysis using Geneious indicated the presence of genome variants. The datasets were analyzed using an in-house pipeline (VPipe),
      <sup>
       <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>
      </sup> which performs various quality control (QC) steps and 
      <em>de novo</em> assembly using SPAdes. The post-QC reads were considered total reads (
      <strong>T</strong>)and mapped to their respective reference genome in order to determine the major and minor variants present in each sample. Total reads which mapped with high similarity (≥99%) to the major variant were categorized as reads representing the major variant (
      <strong>M</strong>). Unbinned reads from the major variant reference recruitment were used to construct the minor variant consensus using a second round of reference recruitment, and these reads were categorized as the minor variant (
      <strong>m</strong>). Remaining reads from the previous two steps were considered background (
      <strong>B</strong>) reads.
     </p>
     <p id="p-43">
      <em>De novo</em> assembly for each of the four clinical samples was performed for the following binned NGS datasets: (1) total reads only (
      <strong>T</strong>); (2) major variants only (
      <strong>M</strong>); (3) major and minor variants only (
      <strong>Mm</strong>); and (4) major variants and background reads only (
      <strong>MB</strong>). This was repeated with three assembly programs: SPAdes, Cap3, and Geneious. The length of the longest contig produced from each assembly and the performance metric UG
      <sub>50</sub>%.
      <sup>
       <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>
      </sup> were calculated to compare the results for these 48 assemblies (4 experiments × 4 viruses × 3 assemblers).
     </p>
    </div>
   </div>
   <div class="section" id="sec-16">
    <h2 class="">Data Availability</h2>
    <p id="p-44">Sequencing reads for the experiments conducted using clinical specimens are available through the NCBI Sequence Read Archive (SRA) accession PRJNA577924. Reads from simulated datasets (Experiments 1-3) are available upon request.</p>
   </div>
   <div class="section" id="sec-17">
    <h2 class="">Funding Information</h2>
    <p id="p-45">This work was supported in part by Federal appropriations to the Centers for Disease Control and Prevention, through the Advanced Molecular Detection Initiative line item.</p>
   </div>
   <div class="section" id="sec-18">
    <h2 class="">Author contributions</h2>
    <p id="p-47">All authors contributed to the conceptualization, data analysis, preparation, and review of this manuscript. C.J.C, R.L.M., and T.F.F.N. wrote this manuscript.</p>
   </div>
   <div class="section COI-statement" id="sec-19">
    <h2 class="">Competing interests</h2>
    <p id="p-48">The authors declare no competing interests.</p>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">Supplemental Information</h2>
    <div class="section" id="sec-21">
     <h3>Analysis of viral GenBank records</h3>
     <div class="section" id="sec-22">
      <h4>The advent of NGS fuels viral sequencing</h4>
      <p id="p-49">As of December 2017, GenBank’s non-redundant nucleotide database had grown to more than 2.3 million virus sequences, with the annual number of new sequences deposited increasing by 270% between 2007 and 2017 [
       <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1a</a> and 
       <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Supplement Table S1</a>]. GenBank entries started incorporating information on the sequencing technology platform used in 2011. Through 2018, 144,712 viral entries (22%) had documented utilization of NGS sequencing technology, compared to 500,027 entries (77%) utilizing Sanger methods [
       <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1b</a> and 
       <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Supplement Table S1</a>]. Illumina was the most common NGS platform used for viral sequencing, with &gt;;2-fold the number of entries compared to the next most popular NGS platform (31,000 viral entries in 2017 [
       <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1d</a> &amp;; 
       <a id="xref-fig-1-11" class="xref-fig" href="#F1">e</a>]). Although NGS usage has risen tremendously, Sanger sequencing still contributed the majority of all viral sequences. This is likely because Sanger is still attractive for generating short viral sequences over genotyping windows or other informative regions. If only long sequences (≥2000 nt) are considered, NGS technologies surpassed Sanger as the dominant strategy for sequencing in 2017 [
       <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1f</a> and 
       <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Supplement Table S2</a>].
      </p>
      <p id="p-50">A total of 27,217 counts of sequencing technologies were listed for the 25,344 long viral GenBank entries in 2017, as some sequences were generated using two or more sequencing technologies. NGS technologies were listed in 53.8% (14,653/27,217) of entries, versus 46.2% of entries (12,564/27,217) for Sanger. Illumina was identified as the most dominant NGS technology, accounting for 12,615/14,653 entries (86.1%) [
       <a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1g</a> and 
       <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Supplement Table S2</a>].
      </p>
      <p id="p-51">Multiple sequencing technologies may be used to generate viral sequence for one entry. The most common combination observed was 454 and Sanger (18,002 entries), likely due to the early emergence of the 454 technology compared to other NGS platforms [
       <a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1c</a> and 
       <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Supplement Table S3</a>]. This is followed by Illumina and Sanger (3,475), Illumina and 454 (3,299), Illumina and Ion Torrent (2,600), and Illumina and PacBio (997). Interestingly, more recently released longer-read platforms like PacBio and Oxford Nanopore tended to be paired with Illumina more frequently compared to traditional Sanger sequencing. A small number of studies even combined three or four different sequencing technologies (530 and 6 entries, respectively) [
       <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Supplement Table S4</a>]. Some users employed a combined approach to circumvent the inherent flaws of one sequencing platform, particularly for genome finishing.
       <sup>
        <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>
       </sup> For example, after NGS has been used to generate the majority of a RNA virus genome, RACE (Rapid amplification of cDNA ends) is typically performed with Sanger to obtain the 5’ or 3’ termini.
       <sup>
        <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>, 
        <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>
       </sup>
      </p>
     </div>
     <div class="section" id="sec-23">
      <h4>
       <em>De novo</em> assembly plays a major role in analyzing long viral sequences
      </h4>
      <p id="p-52">We analyzed the assembly methods used for GenBank entries of long sequences (≥2000 nt) from 2012 to 2017 when NGS usage become relevant [
       <a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1h</a> &amp;; 
       <a id="xref-fig-1-16" class="xref-fig" href="#F1">i</a> and 
       <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Supplement Table S5</a>]. The number of programs used to assemble viral sequences has steadily increased over time (a &gt;;2-fold increase from 2012-2017). With new sequencing technologies emerging and computational power continually improving, the development of new and better assembly programs always follows suite. The use of specifically-designed 
       <em>de novo</em> assembly programs (ABySS, BWA, Canu, Cap3, IDBA, MIRA, Newbler, SOAPdenovo, SPAdes, Trinity, and Velvet) has increased from less than 1% of viral sequence entries in 2012, to 20% of all viral sequence entries in 2017. A similar increase was observed for reference-mapping software (i.e., Bowtie and Bowtie2), from 0.03% in 2012 to 6.5% in 2017. Multifunctional programs that offer both assembly options, including CLC Genomics Workbench (CLC), DNA Baser, DNASTAR, Geneious, and Sequencher, were by far the most popular option for the years 2013-2017. However, since these commercial software packages can perform both 
       <em>de novo</em> and reference-mapping assembly, the exact sequence assembly strategy used for these records is unknown, and thus the contributions of both 
       <em>de novo</em> assembly and reference recruitment are likely underestimated.
      </p>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-46">We thank Dr. Steve Oberste for thoughtful suggestions on this work.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rasmussen</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Katze</span>
          <span class="cit-name-given-names">MG</span>
         </span>. 
         <span class="cit-article-title">Genomic Signatures of Emerging Viruses: A New Era of Systems Epidemiology</span>. 
         <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">611</span>–
         <span class="cit-lpage">618</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D19%26rft.spage%253D611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.2" data-doi="10.1016/j.virusres.2016.10.014">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eltahla</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lloyd</span>
          <span class="cit-name-given-names">AR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bull</span>
          <span class="cit-name-given-names">RA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luciani</span>
          <span class="cit-name-given-names">F</span>
         </span>. 
         <span class="cit-article-title">Understanding the complex evolution of rapidly mutating viruses with deep sequencing: Beyond the analysis of viral diversity</span>. 
         <abbr class="cit-jnl-abbrev">Virus Res</abbr>
         <span class="cit-vol">239</span>, 
         <span class="cit-fpage">43</span>–
         <span class="cit-lpage">54</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D239%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2016.10.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virusres.2016.10.014&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.3" data-doi="10.1016/j.coviro.2016.09.004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pierce</span>
          <span class="cit-name-given-names">BG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Keck</span>
          <span class="cit-name-given-names">ZY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Foung</span>
          <span class="cit-name-given-names">SK</span>
         </span>. 
         <span class="cit-article-title">Viral evasion and challenges of hepatitis C virus vaccine development</span>. 
         <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">55</span>–
         <span class="cit-lpage">63</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D20%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2016.09.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.coviro.2016.09.004&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.4" data-doi="10.1038/nrg2323">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Duffy</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shackelton</span>
          <span class="cit-name-given-names">LA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">EC</span>
         </span>. 
         <span class="cit-article-title">Rates of evolutionary change in viruses: patterns and determinants</span>. 
         <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">267</span>–
         <span class="cit-lpage">276</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDuffy%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D267%26rft.epage%253D276%26rft.atitle%253DRates%2Bof%2Bevolutionary%2Bchange%2Bin%2Bviruses%253A%2Bpatterns%2Band%2Bdeterminants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2323%26rft_id%253Dinfo%253Apmid%252F18319742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrg2323&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18319742&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000254084100010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Andino</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Domingo</span>
          <span class="cit-name-given-names">E</span>
         </span>. 
         <span class="cit-article-title">Viral quasispecies</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-vol">479-480</span>, 
         <span class="cit-fpage">46</span>–
         <span class="cit-lpage">51</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D479480%26rft.spage%253D46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.6" data-doi="10.1371/journal.ppat.1002529">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Henn</span>
          <span class="cit-name-given-names">MR</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Whole genome deep sequencing of HIV-1 reveals the impact of early minor variants upon immune recognition during acute infection</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">e1002529</span>–
         <span class="cit-lpage">e1002529</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DHenn%26rft.auinit1%253DM.%2BR.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002529%26rft.epage%253De1002529%26rft.atitle%253DWhole%2Bgenome%2Bdeep%2Bsequencing%2Bof%2BHIV-1%2Breveals%2Bthe%2Bimpact%2Bof%2Bearly%2Bminor%2Bvariants%2Bupon%2Bimmune%2Brecognition%2Bduring%2Bacute%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002529%26rft_id%253Dinfo%253Apmid%252F22412369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002529&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22412369&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.7" data-doi="10.1128/JVI.02697-10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Herbeck</span>
          <span class="cit-name-given-names">JT</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Demographic processes affect HIV-1 evolution in primary infection before the onset of selective processes</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">85</span>, 
         <span class="cit-fpage">7523</span>–
         <span class="cit-lpage">7534</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02697-10%26rft_id%253Dinfo%253Apmid%252F21593162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NS8xNS83NTIzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjMvODE1NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.8" data-doi="10.1099/vir.0.81015-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jerzak</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bernard</span>
          <span class="cit-name-given-names">KA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kramer</span>
          <span class="cit-name-given-names">LD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ebel</span>
          <span class="cit-name-given-names">GD</span>
         </span>. 
         <span class="cit-article-title">Genetic variation in West Nile virus from naturally infected mosquitoes and birds suggests quasispecies structure and strong purifying selection</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of general virology</abbr>
         <span class="cit-vol">86</span>, 
         <span class="cit-fpage">2175</span>–
         <span class="cit-lpage">2183</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DJerzak%26rft.auinit1%253DG.%26rft.volume%253D86%26rft.issue%253D8%26rft.spage%253D2175%26rft.epage%253D2183%26rft.atitle%253DGenetic%2Bvariation%2Bin%2BWest%2BNile%2Bvirus%2Bfrom%2Bnaturally%2Binfected%2Bmosquitoes%2Band%2Bbirds%2Bsuggests%2Bquasispecies%2Bstructure%2Band%2Bstrong%2Bpurifying%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.81015-0%26rft_id%253Dinfo%253Apmid%252F16033965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1099/vir.0.81015-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16033965&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.9" data-doi="10.1128/JVI.06627-11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lauck</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Analysis of hepatitis C virus intrahost diversity across the coding region by ultradeep pyrosequencing</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">86</span>, 
         <span class="cit-fpage">3952</span>–
         <span class="cit-lpage">3960</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06627-11%26rft_id%253Dinfo%253Apmid%252F22278255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzcvMzk1MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzIzLzgxNTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.10" data-doi="10.1128/JVI.78.22.12717-12721.2004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">S-R</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Study of sequence variation of dengue type 3 virus in naturally infected mosquitoes and human hosts: implications for transmission and evolution</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">78</span>, 
         <span class="cit-fpage">12717</span>–
         <span class="cit-lpage">12721</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLin%26rft.auinit1%253DS.-R.%26rft.volume%253D78%26rft.issue%253D22%26rft.spage%253D12717%26rft.epage%253D12721%26rft.atitle%253DStudy%2Bof%2BSequence%2BVariation%2Bof%2BDengue%2BType%2B3%2BVirus%2Bin%2BNaturally%2BInfected%2BMosquitoes%2Band%2BHuman%2BHosts%253A%2BImplications%2Bfor%2BTransmission%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.22.12717-12721.2004%26rft_id%253Dinfo%253Apmid%252F15507664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OC8yMi8xMjcxNyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzIzLzgxNTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.11" data-doi="10.1128/JVI.00112-10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Murcia</span>
          <span class="cit-name-given-names">PR</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Intra- and interhost evolutionary dynamics of equine influenza virus</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">84</span>, 
         <span class="cit-fpage">6943</span>–
         <span class="cit-lpage">6954</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00112-10%26rft_id%253Dinfo%253Apmid%252F20444896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xNC82OTQzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjMvODE1NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.12" data-doi="10.1038/nature04388">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vignuzzi</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stone</span>
          <span class="cit-name-given-names">JK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Arnold</span>
          <span class="cit-name-given-names">JJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cameron</span>
          <span class="cit-name-given-names">CE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Andino</span>
          <span class="cit-name-given-names">R</span>
         </span>. 
         <span class="cit-article-title">Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">439</span>, 
         <span class="cit-fpage">344</span>–
         <span class="cit-lpage">348</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVignuzzi%26rft.auinit1%253DM.%26rft.volume%253D439%26rft.issue%253D7074%26rft.spage%253D344%26rft.epage%253D348%26rft.atitle%253DQuasispecies%2Bdiversity%2Bdetermines%2Bpathogenesis%2Bthrough%2Bcooperative%2Binteractions%2Bin%2Ba%2Bviral%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04388%26rft_id%253Dinfo%253Apmid%252F16327776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature04388&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16327776&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000234682100047&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.13" data-doi="10.1128/JVI.05985-11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thai</span>
          <span class="cit-name-given-names">KTD</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">High-resolution analysis of intrahost genetic diversity in dengue virus serotype 1 infection identifies mixed infections</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">86</span>, 
         <span class="cit-fpage">835</span>–
         <span class="cit-lpage">843</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05985-11%26rft_id%253Dinfo%253Apmid%252F22090119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg2LzIvODM1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjMvODE1NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.14" data-doi="10.1186/1471-2164-13-475">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">De novo assembly of highly diverse viral populations</span>. 
         <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">475</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DYang%26rft.auinit1%253DX.%26rft.volume%253D13%26rft.spage%253D475%26rft.epage%253D475%26rft.atitle%253DDe%2Bnovo%2Bassembly%2Bof%2Bhighly%2Bdiverse%2Bviral%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-475%26rft_id%253Dinfo%253Apmid%252F22974120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-475&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22974120&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.15" data-doi="10.1093/nar/gkv002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">An ensemble strategy that significantly improves de novo assembly of microbial genomes from metagenomic next-generation sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">43</span>, 
         <span class="cit-fpage">e46</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv002%26rft_id%253Dinfo%253Apmid%252F25586223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkv002&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25586223&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.16" data-doi="10.1128/AEM.02181-07">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wommack</span>
          <span class="cit-name-given-names">KE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhavsar</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ravel</span>
          <span class="cit-name-given-names">J</span>
         </span>. 
         <span class="cit-article-title">Metagenomics: read length matters</span>. 
         <abbr class="cit-jnl-abbrev">Applied and environmental microbiology</abbr>
         <span class="cit-vol">74</span>, 
         <span class="cit-fpage">1453</span>–
         <span class="cit-lpage">1463</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2Benvironmental%2Bmicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02181-07%26rft_id%253Dinfo%253Apmid%252F18192407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6Ijc0LzUvMTQ1MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzIzLzgxNTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.17" data-doi="10.1089/cmb.2017.0013">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Castro</span>
          <span class="cit-name-given-names">CJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">TFF</span>
         </span>. 
         <span class="cit-article-title">U50: A New Metric for Measuring Assembly Output Based on Non-Overlapping, Target-Specific Contigs</span>. 
         <abbr class="cit-jnl-abbrev">J Comput Biol</abbr>
         <span class="cit-vol">24</span>, 
         <span class="cit-fpage">1071</span>–
         <span class="cit-lpage">1080</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BComput%2BBiol%26rft.volume%253D24%26rft.spage%253D1071%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2017.0013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1089/cmb.2017.0013&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.18" data-doi="10.1128/JCM.02121-16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Montmayeur</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">High-throughput next-generation sequencing of polioviruses</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>
         <span class="cit-vol">55</span>, 
         <span class="cit-fpage">606</span>–
         <span class="cit-lpage">615</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.02121-16%26rft_id%253Dinfo%253Apmid%252F27927929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjU1LzIvNjA2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjMvODE1NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.19" data-doi="10.1128/JVI.00869-12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>
          <span class="cit-name-given-names">TFF</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">High Variety of Known and New RNA and DNA Viruses of Diverse Origins in Untreated Sewage</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>
         <span class="cit-vol">86</span>, 
         <span class="cit-fpage">12161</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00869-12%26rft_id%253Dinfo%253Apmid%252F22933275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yMi8xMjE2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzIzLzgxNTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Du</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Feng</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Che</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Q</span>
         </span>. 
         <span class="cit-article-title">A severe case of co-infection with Enterovirus 71 and vaccine-derived Poliovirus type II</span>. 
         <abbr class="cit-jnl-abbrev">Journal of clinical virology : the official publication of the Pan American Society for Clinical Virology</abbr>
         <span class="cit-vol">72</span>, 
         <span class="cit-fpage">25</span>–
         <span class="cit-lpage">29</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bvirology%2B%253A%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BPan%2BAmerican%2BSociety%2Bfor%2BClinical%2BVirology%26rft.volume%253D72%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jorba</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Update on Vaccine-Derived Polioviruses - Worldwide, January 2017-June 2018</span>. 
         <abbr class="cit-jnl-abbrev">MMWR Morbidity and mortality weekly report</abbr>
         <span class="cit-vol">67</span>, 
         <span class="cit-fpage">1189</span>–
         <span class="cit-lpage">1194</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMMWR%2BMorbidity%2Band%2Bmortality%2Bweekly%2Breport%26rft.volume%253D67%26rft.spage%253D1189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.22" data-doi="10.1093/bioinformatics/btv383">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wick</span>
          <span class="cit-name-given-names">RR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schultz</span>
          <span class="cit-name-given-names">MB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zobel</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holt</span>
          <span class="cit-name-given-names">KE</span>
         </span>. 
         <span class="cit-article-title">Bandage: interactive visualization of de novo genome assemblies</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics (Oxford, England)</abbr>
         <span class="cit-vol">31</span>, 
         <span class="cit-fpage">3350</span>–
         <span class="cit-lpage">3352</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv383%26rft_id%253Dinfo%253Apmid%252F26099265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv383&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26099265&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.23" data-doi="10.1016/j.gpb.2016.05.004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Giordano</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ning</span>
          <span class="cit-name-given-names">Z</span>
         </span>. 
         <span class="cit-article-title">Oxford Nanopore MinION Sequencing and Genome Assembly</span>. 
         <abbr class="cit-jnl-abbrev">Genomics, Proteomics &amp;; Bioinformatics</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">265</span>–
         <span class="cit-lpage">279</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenomics%252C%2BProteomics%2B%2526%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gpb.2016.05.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.gpb.2016.05.004&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Herath</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jayasundara</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ackland</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Saeed</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">SL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Halgamuge</span>
          <span class="cit-name-given-names">S</span>
         </span>. 
         <span class="cit-article-title">Assessing Species Diversity Using Metavirome Data: Methods and Challenges</span>. 
         <abbr class="cit-jnl-abbrev">Comput Struct Biotechnol J</abbr>
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">447</span>–
         <span class="cit-lpage">455</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DComput%2BStruct%2BBiotechnol%2BJ%26rft.volume%253D15%26rft.spage%253D447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.25" data-doi="10.1093/bioinformatics/bts075">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bunge</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Woodard</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bohning</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Foster</span>
          <span class="cit-name-given-names">JA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Connolly</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Allen</span>
          <span class="cit-name-given-names">HK</span>
         </span>. 
         <span class="cit-article-title">Estimating population diversity with CatchAll</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">28</span>, 
         <span class="cit-fpage">1045</span>–
         <span class="cit-lpage">1047</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts075%26rft_id%253Dinfo%253Apmid%252F22333246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts075&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22333246&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000302298900026&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.26" data-doi="10.1186/1471-2105-6-41">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Angly</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">PHACCS, an online tool for estimating the structure and diversity of uncultured viral communities using metagenomic information</span>. 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">41</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DAngly%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.spage%253D41%26rft.epage%253D41%26rft.atitle%253DPHACCS%252C%2Ban%2Bonline%2Btool%2Bfor%2Bestimating%2Bthe%2Bstructure%2Band%2Bdiversity%2Bof%2Buncultured%2Bviral%2Bcommunities%2Busing%2Bmetagenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-41%26rft_id%253Dinfo%253Apmid%252F15743531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-41&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15743531&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">NF</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Diversity and Composition of Bacterial Community in Soils and Lake Sediments from an Arctic Lake Area</span>. 
         <abbr class="cit-jnl-abbrev">Frontiers in microbiology</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">1170</span>–
         <span class="cit-lpage">1170</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bmicrobiology%26rft.volume%253D7%26rft.spage%253D1170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.28" data-doi="10.1371/journal.pbio.0050077">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rusch</span>
          <span class="cit-name-given-names">DB</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The Sorcerer II Global Ocean Sampling expedition: northwest Atlantic through eastern tropical Pacific</span>. 
         <abbr class="cit-jnl-abbrev">PLoS biology</abbr>
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">e77</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DRusch%26rft.auinit1%253DD.%2BB.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De77%26rft.epage%253De77%26rft.atitle%253DThe%2BSorcerer%2BII%2BGlobal%2BOcean%2BSampling%2Bexpedition%253A%2Bnorthwest%2BAtlantic%2Bthrough%2Beastern%2Btropical%2BPacific.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050077%26rft_id%253Dinfo%253Apmid%252F17355176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050077&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17355176&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.29" data-doi="10.1038/nature11234">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">The Human Microbiome</span>
         <span class="cit-auth">
          <span class="cit-name-surname">Project</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure, function and diversity of the healthy human microbiome</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">486</span>, 
         <span class="cit-fpage">207</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D486%26rft.issue%253D7402%26rft.spage%253D207%26rft.epage%253D214%26rft.atitle%253DStructure%252C%2Bfunction%2Band%2Bdiversity%2Bof%2Bthe%2Bhealthy%2Bhuman%2Bmicrobiome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11234%26rft_id%253Dinfo%253Apmid%252F22699609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature11234&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22699609&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000305189000025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.30" data-doi="10.1038/nature06244">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Turnbaugh</span>
          <span class="cit-name-given-names">PJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ley</span>
          <span class="cit-name-given-names">RE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hamady</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fraser-Liggett</span>
          <span class="cit-name-given-names">CM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Knight</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gordon</span>
          <span class="cit-name-given-names">JI</span>
         </span>. 
         <span class="cit-article-title">The Human Microbiome Project</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">449</span>, 
         <span class="cit-fpage">804</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DTurnbaugh%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D804%26rft.epage%253D810%26rft.atitle%253DThe%2Bhuman%2Bmicrobiome%2Bproject.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06244%26rft_id%253Dinfo%253Apmid%252F17943116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature06244&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17943116&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000250230600029&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.31" data-doi="10.1128/JVI.75.14.6566-6571.2001">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schneider</span>
          <span class="cit-name-given-names">WL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roossinck</span>
          <span class="cit-name-given-names">MJ</span>
         </span>. 
         <span class="cit-article-title">Genetic Diversity in RNA Virus Quasispecies Is Controlled by Host-Virus Interactions</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>
         <span class="cit-vol">75</span>, 
         <span class="cit-fpage">6566</span> (
         <span class="cit-pub-date">2001</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSchneider%26rft.auinit1%253DW.%2BL.%26rft.volume%253D75%26rft.issue%253D14%26rft.spage%253D6566%26rft.epage%253D6571%26rft.atitle%253DGenetic%2BDiversity%2Bin%2BRNA%2BVirus%2BQuasispecies%2BIs%2BControlled%2Bby%2BHost-Virus%2BInteractions%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.75.14.6566-6571.2001%26rft_id%253Dinfo%253Apmid%252F11413324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3NS8xNC82NTY2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjMvODE1NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gregori</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Perales</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rodriguez-Frias</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Esteban</span>
          <span class="cit-name-given-names">JI</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Quer</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Domingo</span>
          <span class="cit-name-given-names">E</span>
         </span>. 
         <span class="cit-article-title">Viral quasispecies complexity measures</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-vol">493</span>, 
         <span class="cit-fpage">227</span>–
         <span class="cit-lpage">237</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D493%26rft.spage%253D227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.33" data-doi="10.1093/bioinformatics/btr708">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Myers</span>
          <span class="cit-name-given-names">JR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marth</span>
          <span class="cit-name-given-names">GT</span>
         </span>. 
         <span class="cit-article-title">ART: a next-generation sequencing read simulator</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics (Oxford, England)</abbr>
         <span class="cit-vol">28</span>, 
         <span class="cit-fpage">593</span>–
         <span class="cit-lpage">594</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr708%26rft_id%253Dinfo%253Apmid%252F22199392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr708&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22199392&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000300490500023&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.34" data-doi="10.1101/gr.9.9.868">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Madan</span>
          <span class="cit-name-given-names">A</span>
         </span>. 
         <span class="cit-article-title">CAP3: A DNA sequence assembly program</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">868</span>–
         <span class="cit-lpage">877</span> (
         <span class="cit-pub-date">1999</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D868%26rft.epage%253D877%26rft.atitle%253DCAP3%253A%2BA%2BDNA%2BSequence%2BAssembly%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.868%26rft_id%253Dinfo%253Apmid%252F10508846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NjgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yMy84MTU0ODAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.35" data-doi="10.1101/gr.1917404">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chevreux</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">1147</span>–
         <span class="cit-lpage">1159</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1917404%26rft_id%253Dinfo%253Apmid%252F15140833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE0NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzIzLzgxNTQ4MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chevreux</span>
          <span class="cit-name-given-names">BW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Suhai</span>, 
          <span class="cit-name-given-names">S</span>
         </span>. 
         <span class="cit-article-title">Genome sequence assembly using trace signals and additional sequence information</span>. 
         <abbr class="cit-jnl-abbrev">German conference on bioinformatics</abbr>
         <span class="cit-vol">99</span>, 
         <span class="cit-fpage">45</span>–
         <span class="cit-lpage">56</span> (
         <span class="cit-pub-date">1999</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGerman%2Bconference%2Bon%2Bbioinformatics%26rft.volume%253D99%26rft.spage%253D45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.37" data-doi="10.1101/gr.214346.116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jackman</span>
          <span class="cit-name-given-names">SD</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">768</span>–
         <span class="cit-lpage">777</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.214346.116%26rft_id%253Dinfo%253Apmid%252F28232478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzY4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjMvODE1NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-book" id="cit-815480v2.38">
       <div class="cit-metadata">
        <ol class="duplicate">
         <li>
          <span class="cit-ed">
           <span class="cit-name-surname">Berger</span>
           <span class="cit-name-given-names">B</span>
          </span>
         </li>
        </ol>
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Peng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>
          <span class="cit-name-given-names">HCM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yiu</span>
          <span class="cit-name-given-names">SM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chin</span>
          <span class="cit-name-given-names">FYL</span>
         </span>. 
         <span class="cit-chapter-title">IDBA – A Practical Iterative de Bruijn Graph De Novo Assembler</span>. In: 
         <span class="cit-source">Research in Computational Molecular Biology</span> (ed. 
         <span class="cit-ed">
          <span class="cit-name-surname">Berger</span>
          <span class="cit-name-given-names">B</span>
         </span>). 
         <span class="cit-publ-name">Springer Berlin Heidelberg</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.39" data-doi="10.1101/gr.213959.116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nurk</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meleshko</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Korobeynikov</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pevzner</span>
          <span class="cit-name-given-names">PA</span>
         </span>. 
         <span class="cit-article-title">metaSPAdes: a new versatile metagenomic assembler</span>. 
         <abbr class="cit-jnl-abbrev">Genome Research</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">824</span>–
         <span class="cit-lpage">834</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.213959.116%26rft_id%253Dinfo%253Apmid%252F28298430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvODI0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjMvODE1NDgwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.40" data-doi="10.1186/2047-217X-1-18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</span>. 
         <abbr class="cit-jnl-abbrev">GigaScience</abbr>
         <span class="cit-vol">1</span>, 
         <span class="cit-fpage">18</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23587118&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.41" data-doi="10.1089/cmb.2012.0021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bankevich</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</span>. 
         <abbr class="cit-jnl-abbrev">J Comput Biol</abbr>
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">455</span>–
         <span class="cit-lpage">477</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22506599&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.42" data-doi="10.1038/nbt.1883">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grabherr</span>
          <span class="cit-name-given-names">MG</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span>. 
         <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>
         <span class="cit-vol">29</span>, 
         <span class="cit-fpage">644</span>–
         <span class="cit-lpage">652</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21572440&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.43" data-doi="10.1093/bioinformatics/bts199">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kearse</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">28</span>, 
         <span class="cit-fpage">1647</span>–
         <span class="cit-lpage">1649</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22543367&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000305419800052&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">44.</span>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.44" data-doi="10.1101/gr.223057.117">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Phillippy</span>
          <span class="cit-name-given-names">AM</span>
         </span>. 
         <span class="cit-article-title">New advances in sequence assembly</span>. 
         <abbr class="cit-jnl-abbrev">Genome research</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">xi</span>–
         <span class="cit-lpage">xiii</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.223057.117%26rft_id%253Dinfo%253Apmid%252F28461322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjI3LzUveGkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yMy84MTU0ODAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">45.</span>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.45" data-doi="10.2144/000113066">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Olivarius</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Plessy</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carninci</span>
          <span class="cit-name-given-names">P</span>
         </span>. 
         <span class="cit-article-title">High-throughput verification of transcriptional starting sites by Deep-RACE</span>. 
         <abbr class="cit-jnl-abbrev">BioTechniques</abbr>
         <span class="cit-vol">46</span>, 
         <span class="cit-fpage">130</span>–
         <span class="cit-lpage">132</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DOlivarius%26rft.auinit1%253DS.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D130%26rft.epage%253D132%26rft.atitle%253DHigh-throughput%2Bverification%2Bof%2Btranscriptional%2Bstarting%2Bsites%2Bby%2BDeep-RACE.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000113066%26rft_id%253Dinfo%253Apmid%252F19317658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2144/000113066&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19317658&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F23%2F815480.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000266095700009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">46.</span>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-815480v2.46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lagarde</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Extension of human lncRNA transcripts by RACE coupled with long-read high-throughput sequencing (RACE-Seq)</span>. 
         <abbr class="cit-jnl-abbrev">Nature Communications</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">12339</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D7%26rft.spage%253D12339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
