<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/24/2020.03.09.983064.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.09.983064v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2" />
<meta name="DC.Identifier" content="10.1101/2020.03.09.983064" />
<meta name="DC.Date" content="2020-03-24" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Epidemics of Coronavirus Disease 2019 (COVID-19) now have more than 100,000 confirmed cases worldwide. Diagnosis of COVID-19 is currently performed by RT-qPCR methods, but the capacity of RT-qPCR methods is limited by its requirement of high-level facilities and instruments. Here, we developed and evaluated RT-LAMP assays to detect genomic RNA of SARS-CoV-2, the causative virus of COVID-19. RT-LAMP assays in this study can detect as low as 100 copies of SARS-CoV-2 RNA. Cross-reactivity of RT-LAMP assays to other human Coronaviruses was not observed. We also adapted a colorimetric detection method for our RT-LAMP assay so that the tests potentially performed in higher throughput." />
<meta name="DC.Contributor" content="Gun-Soo Park" />
<meta name="DC.Contributor" content="Keunbon Ku" />
<meta name="DC.Contributor" content="Seung-Hwa Baek" />
<meta name="DC.Contributor" content="Seong Jun Kim" />
<meta name="DC.Contributor" content="Seung Il Kim" />
<meta name="DC.Contributor" content="Bum-Tae Kim" />
<meta name="DC.Contributor" content="Jin-Soo Maeng" />
<meta name="article:published_time" content="2020-03-24" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Epidemics of Coronavirus Disease 2019 (COVID-19) now have more than 100,000 confirmed cases worldwide. Diagnosis of COVID-19 is currently performed by RT-qPCR methods, but the capacity of RT-qPCR methods is limited by its requirement of high-level facilities and instruments. Here, we developed and evaluated RT-LAMP assays to detect genomic RNA of SARS-CoV-2, the causative virus of COVID-19. RT-LAMP assays in this study can detect as low as 100 copies of SARS-CoV-2 RNA. Cross-reactivity of RT-LAMP assays to other human Coronaviruses was not observed. We also adapted a colorimetric detection method for our RT-LAMP assay so that the tests potentially performed in higher throughput.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.09.983064v2" />
<meta name="citation_id" content="2020.03.09.983064v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.09.983064v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.09.983064v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.09.983064v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.09.983064" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.09.983064" />
<meta name="citation_author" content="Gun-Soo Park" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Research Group of Food Processing, Korea Food Research Institute" />
<meta name="citation_author_email" content="pcdhmk@krict.re.kr" />
<meta name="citation_author_email" content="maengjs@krict.re.kr" />
<meta name="citation_author" content="Keunbon Ku" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author" content="Seung-Hwa Baek" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Department of Predictive Toxicology, Korea Institute of Toxicology" />
<meta name="citation_author" content="Seong Jun Kim" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author" content="Seung Il Kim" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Research Center for Bioconvergence Analysis, Korea Basic Science Institute" />
<meta name="citation_author" content="Bum-Tae Kim" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author" content="Jin-Soo Maeng" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Research Group of Food Processing, Korea Food Research Institute" />
<meta name="citation_author_email" content="pcdhmk@krict.re.kr" />
<meta name="citation_author_email" content="maengjs@krict.re.kr" />
<meta name="citation_reference" content="WHO. Coronavirus Disease 2019. 2020; Available from: https://www.who.int/emergencies/diseases/novel-coronavirus-2019." />
<meta name="citation_reference" content="Zhou, P., et al., A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, 2020." />
<meta name="citation_reference" content="Wu, F., et al., A new coronavirus associated with human respiratory disease in China. Nature, 2020." />
<meta name="citation_reference" content="Zhu, N., et al., A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med, 2020." />
<meta name="citation_reference" content="Huang, C., et al., Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet, 2020. 395(10223): p. 497–506." />
<meta name="citation_reference" content="WHO. Coronavirus disease (COVID-19) technical guidance: Laboratory testing for 2019-nCoV in humans. 2020; Available from: https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance." />
<meta name="citation_reference" content="Corman, V.M., et al., Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill, 2020. 25(3)." />
<meta name="citation_reference" content="Chu, D.K.W., et al., Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia. Clin Chem, 2020." />
<meta name="citation_reference" content="Nayak, S., et al., Point-of-Care Diagnostics: Recent Developments in a Connected Age. Anal Chem, 2017. 89(1): p. 102–123." />
<meta name="citation_reference" content="Niemz, A., T.M. Ferguson, and D.S. Boyle, Point-of-care nucleic acid testing for infectious diseases. Trends Biotechnol, 2011. 29(5): p. 240–50." />
<meta name="citation_reference" content="Notomi, T., et al., Loop-mediated isothermal amplification of DNA. Nucleic Acids Res, 2000. 28(12): p. E63." />
<meta name="citation_reference" content="Shirato, K., et al., Detection of Middle East respiratory syndrome coronavirus using reverse transcription loop-mediated isothermal amplification (RT-LAMP). Virol J, 2014. 11: p. 139." />
<meta name="citation_reference" content="Oscorbin, I.P., et al., Comparison of fluorescent intercalating dyes for quantitative loop-mediated isothermal amplification (qLAMP). Biotechniques, 2016. 61(1): p. 20–5." />
<meta name="citation_reference" content="Goto, M., et al., Colorimetric detection of loop-mediated isothermal amplification reaction by using hydroxy naphthol blue. Biotechniques, 2009. 46(3): p. 167–72." />
<meta name="citation_reference" content="Miyamoto, S., et al., Method for colorimetric detection of double-stranded nucleic acid using leuco triphenylmethane dyes. Anal Biochem, 2015. 473: p. 28–33." />
<meta name="citation_reference" content="Tanner, N.A., Y. Zhang, and T.C. Evans, Jr., Visual detection of isothermal nucleic acid amplification using pH-sensitive dyes. Biotechniques, 2015. 58(2): p. 59–68." />
<meta name="citation_reference" content="Nie, K., et al., Evaluation of a direct reverse transcription loop-mediated isothermal amplification method without RNA extraction for the detection of human enterovirus 71 subgenotype C4 in nasopharyngeal swab specimens. PLoS One, 2012. 7(12): p. e52486." />
<meta name="citation_reference" content="Nyan, D.C., et al., Rapid detection of hepatitis B virus in blood plasma by a specific and sensitive loop-mediated isothermal amplification assay. Clin Infect Dis, 2014. 59(1): p. 16–23." />
<meta name="citation_reference" content="Lee, D., et al., Simple and Highly Sensitive Molecular Diagnosis of Zika Virus by Lateral Flow Assays. Anal Chem, 2016. 88(24): p. 12272–12278." />
<meta name="citation_reference" content="Jung, Y.J., et al., Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2. BioRxiv, 2020." />
<meta name="citation_reference" content="Corman, V.M., et al., Detection of a novel human coronavirus by real-time reverse-transcription polymerase chain reaction. Euro Surveill, 2012. 17(39)." />
<meta name="citation_reference" content="Kumar, S., G. Stecher, and K. Tamura, MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. Mol Biol Evol, 2016. 33(7): p. 1870–4." />
<meta name="citation_reference" content="Hiscox, J.A., D. Cavanagh, and P. Britton, Quantification of individual subgenomic mRNA species during replication of the coronavirus transmissible gastroenteritis virus. Virus Res, 1995. 36(2-3): p. 119–30." />
<meta name="citation_reference" content="Khorosheva, E.M., et al., Lack of correlation between reaction speed and analytical sensitivity in isothermal amplification reveals the value of digital methods for optimization: validation using digital real-time RT-LAMP. Nucleic Acids Res, 2016. 44(2): p. e10." />
<meta name="citation_reference" content="Zou, L., et al., SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients. N Engl J Med, 2020." />
<meta name="twitter:title" content="Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Epidemics of Coronavirus Disease 2019 (COVID-19) now have more than 100,000 confirmed cases worldwide. Diagnosis of COVID-19 is currently performed by RT-qPCR methods, but the capacity of RT-qPCR methods is limited by its requirement of high-level facilities and instruments. Here, we developed and evaluated RT-LAMP assays to detect genomic RNA of SARS-CoV-2, the causative virus of COVID-19. RT-LAMP assays in this study can detect as low as 100 copies of SARS-CoV-2 RNA. Cross-reactivity of RT-LAMP assays to other human Coronaviruses was not observed. We also adapted a colorimetric detection method for our RT-LAMP assay so that the tests potentially performed in higher throughput." />
<meta name="og-title" property="og:title" content="Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.09.983064v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Epidemics of Coronavirus Disease 2019 (COVID-19) now have more than 100,000 confirmed cases worldwide. Diagnosis of COVID-19 is currently performed by RT-qPCR methods, but the capacity of RT-qPCR methods is limited by its requirement of high-level facilities and instruments. Here, we developed and evaluated RT-LAMP assays to detect genomic RNA of SARS-CoV-2, the causative virus of COVID-19. RT-LAMP assays in this study can detect as low as 100 copies of SARS-CoV-2 RNA. Cross-reactivity of RT-LAMP assays to other human Coronaviruses was not observed. We also adapted a colorimetric detection method for our RT-LAMP assay so that the tests potentially performed in higher throughput." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-24" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.09.983064v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.09.983064v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.09.983064v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.09.983064v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1210393" />
    <title>Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2 | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"BaXvwE57qZ1kXaQ9n0OB17EJAxuUy9rL7fBWFwdtL9o","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1210393","apath":"\/biorxiv\/early\/2020\/03\/24\/2020.03.09.983064.atom","pisa":"biorxiv;2020.03.09.983064v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.09.983064v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.09.983064v2","title":"Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2","identifier":"node\/1210393"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GfxAB3qpdqL81rDnriZPJzGYcDPy\\\\\\\/P5Q6Kpqs0Glx2jcC+anJWRQXG2ojA8FINUOH\\\\\\\/lteKAcOMc77XfX0zq9bmMm0tqIDewPuvNe2t+wZ1bSuZtcCZ+lCFtVlEMaevKt1YOsP+PSxAFppRmFD1fP2ExfPqUu8teHIEGxsyypIZ64b4gk\\\\\\\/DGaF010lgX7yWN1KZsnqkVLp4f\\\\\\\/lnijlt2IqWI8C\\\\\\\/X7kiD60SvnmvX8don1CEzboYeyu1owFhg0hPS9zvWUV0ddB4qZtE1nfVNUqgzKYxYgYY1jJONaid9LHgzLKE+3VE5TpK8rghfLWVVFJ6cn4LkbAlSNjoxXlQGmogx23SAnZi6YUl+XjH7uS2QsAdV0mOtuFx36Q4zmCwBUfxxo8pu1a25gXi6ktupZXKyi6OReLkRwCQsPIzMUfzjm\\\\\\\/uo9JOh2Yv90BDBEHbmUyH1NcfdKlHr6w9V31VRrtegZO7HissXuipG2gNf02XQ91KwHuPbgDDWnJGgpCDKzirsd1w7twSgJJlYbKmseU\\\\\\\/HvlryoaQCOjcfn6hMyb1A\\\\\\\/yFh\\\\\\\/AYiX3RkI7dvxeEhr7XdDak8x5PjjtpJ6lmtkEHu3F0sVhZHaKHE8kUOKDDeQvSYEUHN97SHUoXsnVWdUr7tnUWCLR5BtBSkSacwy7zsjB+dWDdreBz1epBvBXvDkE2yiNYpOwaauvJxRQSyOjzQaF0EnTWujsmAa0Qwx6lfxSzrnnlM\\\\\\\/6REwcf71rtcox\\\\\\\/w+gqOn9pFE8H0GB0zCX0Nycn+14btYurZ2nGh17p\\\\\\\/LfYWWDO\\\\\\\/EEnYBOhEi7RCfvqRmgzGI78gbLeQD7DrDyxfi\\\\\\\/jioWmDi96etZlfFlbOpwJ6nWagNrHedSwf6lIjgGGUbYlmIvBQAvxeUXGQheLrM83zjPQPax7Au0mEE\\\\\\\/MzrWH2vLEU7fHgkkyHDSaIF5HhbiNbJ+k7ByIe1AxUJibzDUFm8KZb\\\\\\\/8q0GcSLoLpm4nLRKBuUMKXAC8AJg2szCdyED0xexPwcC6X4\\\\\\\/2a5RwbPfMYZVqBpRigy2uhq29ooESNNxRXy\\\\\\\/LXB\\\\\\\/ZLL8jHrc0meybAO\\\\\\\/nfy1\\\\\\\/nBS++oEWYmv5JKo6nKBZogjHPEqhSLqAnFC557LTRdaCJ\\\\\\\/liDcBdRFfSlbhsB4s\\\\\\\/0HYBxNuAU1H9J93OX3ad1RGTVe7vpUmX\\\\\\\/Nhqnxe5ztD9TXbecr1UQTDkoYLzadA+1b9KEM\\\\\\\/RUuwSIhz7jaXeIjjWvdVBcpMu65wW72VuWDqNwFHmHQ8M9UTKqSnKJ7tU03L3yxAq129ccaRluNuWWH78nXr8zaXz41Ovj93bmqMHEdJE68cOxvF0P231M0dsq\\\\\\\/5uCFb\\\u0022,\\\u0022iv\\\u0022:\\\u0022NMd8m6DhbVynqZss\\\\\\\/ROMVQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00223a0aa4ac5229a9772a0c4703f10438880600eef8b6a1f91cf5c875d432b11c10\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.09.983064v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1210393 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.09.983064v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0BcYNaehihk89Js3gj6pa8btUd6_hmly6IjK3uuLg34" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1210393" id="node1210393" data-pisa="biorxiv;2020.03.09.983064v2" data-pisa-master="biorxiv;2020.03.09.983064" data-apath="/biorxiv/early/2020/03/24/2020.03.09.983064.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gun-Soo</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Keunbon</span> <span class="nlm-surname">Ku</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Seung-Hwa</span> <span class="nlm-surname">Baek</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Seong Jun</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Seung Il</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Bum-Tae</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jin-Soo</span> <span class="nlm-surname">Maeng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.09.983064 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1210393" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gun-Soo Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Research Group of Food Processing, Korea Food Research Institute</span>, Wanju-gun, Jeollabuk-do 55365, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gun-Soo%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGun-Soo%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pcdhmk@krict.re.kr" class="" data-icon-position="" data-hide-link-title="0">pcdhmk@krict.re.kr</a>
<a href="mailto:maengjs@krict.re.kr" class="" data-icon-position="" data-hide-link-title="0">maengjs@krict.re.kr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Keunbon Ku </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keunbon%2BKu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ku%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeunbon%2BKu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Seung-Hwa Baek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Predictive Toxicology, Korea Institute of Toxicology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung-Hwa%2BBaek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baek%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeung-Hwa%2BBaek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Seong Jun Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seong%2BJun%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeong%2BJun%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Seung Il Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Research Center for Bioconvergence Analysis, Korea Basic Science Institute</span>, Cheongju 28119, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung%2BIl%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20SI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeung%2BIl%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bum-Tae Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bum-Tae%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABum-Tae%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jin-Soo Maeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Research Group of Food Processing, Korea Food Research Institute</span>, Wanju-gun, Jeollabuk-do 55365, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jin-Soo%2BMaeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maeng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJin-Soo%2BMaeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pcdhmk@krict.re.kr" class="" data-icon-position="" data-hide-link-title="0">pcdhmk@krict.re.kr</a>
<a href="mailto:maengjs@krict.re.kr" class="" data-icon-position="" data-hide-link-title="0">maengjs@krict.re.kr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.09.983064v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1210393" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1210393/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.09.983064v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1210393" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1210393/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.09.983064v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1210393" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1210393/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.09.983064v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1210393" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1210393/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.09.983064v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1210393" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1210393/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Epidemics of Coronavirus Disease 2019 (COVID-19) now have more than 100,000 confirmed cases worldwide. Diagnosis of COVID-19 is currently performed by RT-qPCR methods, but the capacity of RT-qPCR methods is limited by its requirement of high-level facilities and instruments. Here, we developed and evaluated RT-LAMP assays to detect genomic RNA of SARS-CoV-2, the causative virus of COVID-19. RT-LAMP assays in this study can detect as low as 100 copies of SARS-CoV-2 RNA. Cross-reactivity of RT-LAMP assays to other human Coronaviruses was not observed. We also adapted a colorimetric detection method for our RT-LAMP assay so that the tests potentially performed in higher throughput.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">SARS-CoV-2 is the causative viral pathogen of COVID-19 of which the outbreaks resulted in 106,893 confirmed cases involving 3,639 deaths over 103 countries, areas or territories as of 07 March 2020.[<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>] As the name suggests, SARS-CoV-2 is closely related to a group of severe acute respiratory syndrome-related coronaviruses (SARSr-CoV), namely subgenus Sarbecovirus, showing 96% identity to a bat coronavirus [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>].</p><p id="p-5">Diagnosis of COVID-19 can be done through CT scan of suspicious patients and confirmatory lab test is performed using RT-qPCR methods [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>-<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Although RT-qPCR methods are used as the gold-standard for detection of pathogens due to its high sensitivity and specificity, it still have some caveats. Briefly, lab-level facility such as reliable supply of electricity, expensive instruments and trained personnel are required to properly perform RT-qPCR tests. These restrictions hinder to utilize RT-qPCR methods for various point-of-care where pathogen detection might be required [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>].</p><p id="p-6">To overcome such cost-restriction of RT-qPCR and still detect pathogens’ nucleic acids, isothermal amplification methods have been developed [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Among such methods, Loop-mediated isothermal amplification (LAMP) method has some advantages to be applied for point-of-care test (POCT) [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Well optimized LAMP assay shows sensitivity comparable to that of PCR, less than 10 copies per reaction [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Intercalating fluorescent dyes are compatible with LAMP reaction so that we can observe amplification in real-time [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Because amplification efficiency of LAMP reaction is very high, changes in reaction mixture components made it possible to detect the result with colorimetric detection methods [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>-<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Moreover, unpurified sample can be directly used for LAMP [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>-<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. This indicates that high-throughput test is possible when use of unpurified specimen is combined with non-instrumental (<em>e.g</em>. colorimetric) detection.</p><p id="p-7">In this study, we designed and evaluated one-step reverse transcription LAMP (RT-LAMP) methods to detect SARS-CoV-2. We provide a pair of LAMP primer sets that is specific to SARS-CoV-2 and accompanying optimized reaction conditions. To achieve colorimetric detection of LAMP reaction, leuco crystal violet (LCV) method is applied [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>].</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Viral RNA preparation</h3><p id="p-8">SARS-CoV-2 viral RNA was prepared as previously described [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. hCoV-229E and hCoV-OC43 viral RNA were isolated from culture media of infected MRC-5 cells (ATCC<sup>®</sup> CCL-171™). MERS-CoV RNA was isolated from cell pellet lysate of infected Vero cells (ATCC<sup>®</sup> CCL-81™).</p></div><div id="sec-4" class="subsection"><h3>Viral RNA titration</h3><p id="p-9"><em>In vitro</em> transcribed standard RNA for SARS-CoV-2 was prepared as previously described [<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>]. To prepare standard RNA for hCoV-229E, first amplicon of PCR (Forward primer: 5’ – GCTAGTGGATGATCATGCTTTG – 3’, Reverse primer: 5’ – TGGGGCCATAAACTGTTCTATTAC – 3’) was cloned to pBluescript II KS (+) plasmid with <em>BamH</em>I and <em>Xho</em>I. Then, <em>in vitro</em> transcription template was prepared by restriction enzyme cut with <em>Bgl</em>I and <em>Xho</em>I and purification after agarose gel electrophoresis. For hCoV-OC43 and MERS-CoV, amplicons of PCR (OC43 forward primer: 5’ – AGCAACCAGGCTGATGTCAATACC – 3’, OC43 reverse primer: 5’ – AGCAGACCTTCCTGAGCCTTCAAT – 3’, MERS-CoV: UpE region [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]) were synthesized and cloned to pBIC-A plasmid (Bioneer). <em>In vitro</em> transcription template for hCoV-OC43 and MERS-CoV were prepared by restriction enzyme cut with <em>BamH</em>I-<em>Xho</em>I or <em>Ssp</em>I-<em>Xho</em>I, respectively. <em>In vitro</em> transcriptions were done with EZ™ T7 High Yield In Vitro Transcription kit (Enzynomics) as manufacturer’s instructions. RNA products were then purified using Agencourt RNAClean XP (Beckman Coulter). Standard RNA copy numbers were calculated from concentration measured by NanoDrop Lite (Thermo Scientific). All restriction enzymes were purchased from Enzynomics.</p><p id="p-10">To evaluate genomic copy number of viral RNAs, dilutions of standard RNAs and viral RNAs in TE buffer (10 mM Tris-Cl, pH 8.0, 1 mM EDTA) are subjected to one-step RT-qPCR. RT-qPCR reactions were carried out using LightCycler 96 instrument and following reagents: Luna Universal One-Step RT-qPCR Kit (New England Biolabs, NEB) for hCoV-229E and hCoV-OC43, THUNDERBIRD Probe qPCR Master Mix (Toyobo) for MERS-CoV and Luna Universal Probe One-Step RT-qPCR Kit (NEB) for SARS-CoV-2.</p></div><div id="sec-5" class="subsection"><h3>Reverse transcription</h3><p id="p-11">cDNA of SARS-CoV-2 was made using SuperScript IV Reverse Transcriptase (Invitrogen) following manufacturer’s instructions with modifications. Briefly, 10 pmol of random hexamer was used as reverse transcription primer and reaction was performed as follow: 20 minutes at 25°C, 30 minutes at 55°C and 10 minutes at 80°C.</p></div><div id="sec-6" class="subsection"><h3>LAMP and RT-LAMP reaction</h3><p id="p-12">LAMP reaction was performed with reaction mixture containing following components: 1.6 μM FIP/BIP primers, 0.2 μM F3/B3 primers, 0.4 μM LF/LB primers, 1x Isothermal Amplification Buffer II (NEB, 20 mM Tris-HCl pH 8.8, 10 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 150 mM KCl, 2 mM MgSO<sub>4</sub>, 0.1% Tween<sup>®</sup> 20), 6 mM MgSO<sub>4</sub> (NEB, final 8mM Mg<sup>2+</sup>), 1.4 mM each dNTP (Enzynomics), 0.4 μM SYTO-9 (Invitrogen) and 6 U Bst3.0 DNA polymerase (NEB) in total 15 μl reaction volume. For RT-LAMP, 10 U of SuperScript IV Reverse Transcriptase (Invitrogen) was added. For end-point colorimetric detection of LAMP reaction, tweaked version of 5x stock LCV solution[<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>] containing 0.5 mM Crystal Violet (Sigma, C0775), 60 mM Sodium Sulfite (Sigma, S0505) and 5 mM β-Cyclodextrin (Sigma, C4767) was directly added to LAMP reaction mixture to 1x concentration. When using WarmStart Colorimetric LAMP 2X Master Mix (NEB), same concentrations of primers and 0.4 μM SYTO-9 were added. Isothermal incubation and fluorescence signal measurement was performed using LightCycler 96 instrument (Roche) at 69°C with additional heat inactivation (5 minutes at 95°C) and melting curve analysis steps. Fluorescence signals were measured for every minute during 60 (screening and optimization) or 30 (Limit of Detection and cross-reactivity) minutes of incubation. Any changed conditions are specified for each experiment.</p></div></div><div class="section" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>LAMP Primer Design</h3><p id="p-13">Five SARS-CoV-2 sequences (MN908947, MN938384, MN988713, MN985325, and MN975262) and seven SARS-CoV sequences (NC_004718, AY613947, AY313906, AY559094, AY502924, AY278491, and AY502927) were aligned by MEGA7 software.[<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>] SARS-CoV-2 specific regions for LAMP primer design are manually selected: two regions from <em>Nsp3</em> (3055 – 3591, 6172 – 7273), two regions from <em>Spike</em> (S) (21540 – 22549, 22890 – 23779), and one region from <em>Orf8</em> (27824 – 28396). Whole <em>Nucleocapsid</em> (N) gene region is also included as <em>N</em> is usual target of molecular diagnosis due to abundance of its mRNA.[<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>] Two to five basic LAMP primer sets are designed and selected with PrimerExplorer V5 (<a href="http://primerexplorer.jp/lampv5e/index.html">http://primerexplorer.jp/lampv5e/index.html</a>) for each target region. Loop primers are designed by PrimerExplorer V5 or manually selected. 16 LAMP primer sets with proper both of Loop Forward (LF) primer and Loop Backward (LB) primer are selected and subjected to further screening.</p></div><div id="sec-9" class="subsection"><h3>LAMP Primer Screening</h3><p id="p-14">First round of screening was done using WarmStart Colorimetric LAMP 2X Master Mix with cDNA of which corresponding RNA concentration is 8.3 × 10<sup>4</sup> copies/reaction. Nine out of 16 primer sets were selected by threshold time from the results of 40 minutes incubation at 65°C and subjected to further screening.</p><p id="p-15">Second round of screening was done using <em>Bst</em>3.0. The same amount of cDNA as the first round of screening is used. Two primer sets that showed relatively early non-specific amplification are discarded and remaining seven primer sets are subjected to further screening.</p><p id="p-16">Third and fourth rounds of screening were done by checking sensitivity to dilutions of cDNA and RNA, respectively. Five out of seven primer sets showed specific amplification for at least one replicate of duplicate with cDNA concentration corresponding to 1.7 × 10<sup>1</sup> copies of input RNA (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Supplementary Figure 1</a>). Next, we evaluated sensitivity of the five primer sets to RNA dilutions in RT-LAMP using <em>Bst</em>3.0 and SuperScript IV Reverse Transcriptase. Two primer sets, both targeting <em>Nsp3</em>, showed best sensitivity that showed specific amplification at 10<sup>−6</sup> dilution of RNA (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We also added two primer sets, one targeting <em>S</em> and the other targeting <em>N</em>, for reaction optimization experiments as they showed fast threshold time for cDNA and to keep ranges of target genes. Primer sequences are represented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sensitivity of SARS-CoV-2 LAMP assays to cDNA dilutions (A) Real-time amplification fluorescence signal and (B) end-point LCV colorimetric detection results of four SARS-CoV-2 LAMP primer sets that were subjected to reaction optimization. Color of amplification signal curve and LCV detection label for each cDNA dilution are matched. Designated copy number is corresponding reverse transcription input RNA copy number. Some low fluorescence of amplification signal plateaus are due to baseline correction process of LightCycler 96 software." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1595965957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Sensitivity of SARS-CoV-2 LAMP assays to cDNA dilutions&lt;/span&gt; (A) Real-time amplification fluorescence signal and (B) end-point LCV colorimetric detection results of four SARS-CoV-2 LAMP primer sets that were subjected to reaction optimization. Color of amplification signal curve and LCV detection label for each cDNA dilution are matched. Designated copy number is corresponding reverse transcription input RNA copy number. Some low fluorescence of amplification signal plateaus are due to baseline correction process of LightCycler 96 software.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F3.medium.gif" width="425" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F3.medium.gif" width="425" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Supplementary Figure 1.</span> <span class="caption-title">Sensitivity of SARS-CoV-2 LAMP assays to cDNA dilutions</span><p id="p-28" class="first-child">(A) Real-time amplification fluorescence signal and (B) end-point LCV colorimetric detection results of four SARS-CoV-2 LAMP primer sets that were subjected to reaction optimization. Color of amplification signal curve and LCV detection label for each cDNA dilution are matched. Designated copy number is corresponding reverse transcription input RNA copy number. Some low fluorescence of amplification signal plateaus are due to baseline correction process of LightCycler 96 software.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1214563/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1214563/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1214563" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">LAMP primers of which assays optimized and LoD evaluated</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LAMP primer positions on aligned sequences of SARS-CoV-2 and SARS-CoV Primer binding sites of (A) &#x201C;Nsp3_1-61&#x201D; and (B) &#x201C;Nsp3_2-24&#x201D; LAMP primer sets are depicted on aligned sequences of five SARS-CoV-2 (from up, MN908947, MN938384, MN988713, MN985325, MN975262) and seven SARS-CoV (from up, NC_004718, AY613947, AY313906, AY559094, AY502924, AY278491, AY502927). Conserved sites are toggled at 50% level by MEGA7 software so that SARS-CoV-2 specific residues versus SARS-CoV are background colored." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1595965957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;LAMP primer positions on aligned sequences of SARS-CoV-2 and SARS-CoV Primer binding sites of (A) &#x201C;Nsp3_1-61&#x201D; and (B) &#x201C;Nsp3_2-24&#x201D; LAMP primer sets are depicted on aligned sequences of five SARS-CoV-2 (from up, MN908947, MN938384, MN988713, MN985325, MN975262) and seven SARS-CoV (from up, NC_004718, AY613947, AY313906, AY559094, AY502924, AY278491, AY502927). Conserved sites are toggled at 50% level by MEGA7 software so that SARS-CoV-2 specific residues versus SARS-CoV are background colored.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F1.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F1.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1.</span> <p id="p-17" class="first-child">LAMP primer positions on aligned sequences of SARS-CoV-2 and SARS-CoV Primer binding sites of (A) “Nsp3_1-61” and (B) “Nsp3_2-24” LAMP primer sets are depicted on aligned sequences of five SARS-CoV-2 (from up, MN908947, MN938384, MN988713, MN985325, MN975262) and seven SARS-CoV (from up, NC_004718, AY613947, AY313906, AY559094, AY502924, AY278491, AY502927). Conserved sites are toggled at 50% level by MEGA7 software so that SARS-CoV-2 specific residues versus SARS-CoV are background colored.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>LAMP Reaction Optimization</h3><p id="p-18">To optimize RT-LAMP reaction with <em>Bst</em>3.0, we first evaluated optimal concentration of dNTP and Mg<sup>2+</sup> for each primer set. Three dNTP-Mg<sup>2+</sup> concentration combinations are tested at 69°C: dNTP 1.4 mM and Mg<sup>2+</sup> 8 mM, dNTP 1.4 mM and Mg<sup>2+</sup> 6 mM, dNTP 1 mM and Mg<sup>2+</sup> 6 mM. The copy number of RNA template was 1,000 copies per reaction. Best concentration combination was selected for each primer set by whether both of duplicate show specific amplification and by threshold time. Average threshold time (Tt_av) and corresponding dNTP and Mg<sup>2+</sup> concentrations are as follows: dNTP 1 mM and Mg<sup>2+</sup> 6 mM for “Nsp3_1-61” (Tt_av = 11.93), dNTP 1.4 mM and Mg<sup>2+</sup> 8 mM for “Nsp3_2-24” (Tt_av = 7.74), dNTP 1.4mM and Mg<sup>2+</sup> 6mM for “S_1-2-2” and “N_21” (Tt_av = 11.08 and 5.50, respectively).</p><p id="p-19">Next, we checked if amplification is improved at lower temperature (65°C) because optimal reaction temperature of SuperScript IV Reverse Transcriptase is 50-55°C and manufacturer recommend 65-72°C for optimal performance of <em>Bst</em>3.0. Notably, “Nsp3_1-61” and “S_1-2-2” primer sets show improved threshold time (Tt_av = 8.92 and 8.49, respectively).</p></div><div id="sec-11" class="subsection"><h3>Assessing Limit of Detection and Cross-Reactivity</h3><p id="p-20">The Limit of Detection (LoD) of optimized RT-LAMP assays were assessed through 5 to 1000 RNA copies/reaction in triplicate. Among four primer sets subjected to reaction optimization, “S_1-2-2” and “N_21” sets showed relatively poor sensitivity. For “Nsp3_1-61” and “Nsp3_2-24” primer sets, we additionally evaluated LoD with 2-fold SuperScript IV Reverse Transcriptase (20U/reaction). LoD and Tt_av were improved by increasing reverse transcriptase amount. (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Supplementary Figure 2</a>) As a result, both “Nsp3_1-61” and “Nsp3_2-24” RT-LAMP assays could detect as low RNA concentration as 100 copies per reaction.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Colorimetric detection of SARS-CoV-2 RT-LAMP LoD tests for &#x201C;Nsp3_1-61&#x201D; and &#x201C;Nsp3_2-24&#x201D; LCV colorimetric detection results of LoD tests for &#x201C;Nsp3_1-16&#x201D; and &#x201C;Nsp3_2-24&#x201D;. 20U/reaction of reverse transcriptase were used. See main text for detailed reaction condition." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1595965957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Colorimetric detection of SARS-CoV-2 RT-LAMP LoD tests for &#x201C;Nsp3_1-61&#x201D; and &#x201C;Nsp3_2-24&#x201D;&lt;/span&gt; LCV colorimetric detection results of LoD tests for &#x201C;Nsp3_1-16&#x201D; and &#x201C;Nsp3_2-24&#x201D;. 20U/reaction of reverse transcriptase were used. See main text for detailed reaction condition.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F4.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F4.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 2.</span> <span class="caption-title">Colorimetric detection of SARS-CoV-2 RT-LAMP LoD tests for “Nsp3_1-61” and “Nsp3_2-24”</span><p id="p-29" class="first-child">LCV colorimetric detection results of LoD tests for “Nsp3_1-16” and “Nsp3_2-24”. 20U/reaction of reverse transcriptase were used. See main text for detailed reaction condition.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1214567/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1214567/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1214567" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">LoD of RT-LAMP assays targeting SARS-CoV-2</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">The cross-reactivity of two SARS-CoV-2 RT-LAMP assays targeting <em>Nsp3</em> to other human Coronaviruses was not found for hCoV-229E, hCoV-OC43 and MERS-CoV (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cross-reactivity to other Coronaviruses tested for RT-LAMP assay targeting SARS-CoV-2 Real-time amplification fluorescence signal and end-point LCV colorimetric detection results of cross-reactivity test for (A) &#x201C;Nsp3_1-61&#x201D; and (B) &#x201C;Nsp3_2-24&#x201D; LAMP primer sets. Reactions are performed with 20U/reaction of reverse transcriptase and optimized temperature, dNTP concentration and Mg2+ concentration for each primer set. RNA copy number of each viral RNAs are as follow: hCoV-229E &#x2013; 1.6 &#xD7; 106, hCoV-OC43 &#x2013; 1.6&#xD7; 106, MERS-CoV &#x2013; 4.5&#xD7; 106, SARS-CoV-2 &#x2013; 2.5 &#xD7; 103." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1595965957" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cross-reactivity to other Coronaviruses tested for RT-LAMP assay targeting SARS-CoV-2 Real-time amplification fluorescence signal and end-point LCV colorimetric detection results of cross-reactivity test for (A) &#x201C;Nsp3_1-61&#x201D; and (B) &#x201C;Nsp3_2-24&#x201D; LAMP primer sets. Reactions are performed with 20U/reaction of reverse transcriptase and optimized temperature, dNTP concentration and Mg&lt;sup&gt;2+&lt;/sup&gt; concentration for each primer set. RNA copy number of each viral RNAs are as follow: hCoV-229E &#x2013; 1.6 &#xD7; 10&lt;sup&gt;6&lt;/sup&gt;, hCoV-OC43 &#x2013; 1.6&#xD7; 10&lt;sup&gt;6&lt;/sup&gt;, MERS-CoV &#x2013; 4.5&#xD7; 10&lt;sup&gt;6&lt;/sup&gt;, SARS-CoV-2 &#x2013; 2.5 &#xD7; 10&lt;sup&gt;3&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F2.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F2.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/24/2020.03.09.983064/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-22" class="first-child">Cross-reactivity to other Coronaviruses tested for RT-LAMP assay targeting SARS-CoV-2 Real-time amplification fluorescence signal and end-point LCV colorimetric detection results of cross-reactivity test for (A) “Nsp3_1-61” and (B) “Nsp3_2-24” LAMP primer sets. Reactions are performed with 20U/reaction of reverse transcriptase and optimized temperature, dNTP concentration and Mg<sup>2+</sup> concentration for each primer set. RNA copy number of each viral RNAs are as follow: hCoV-229E – 1.6 × 10<sup>6</sup>, hCoV-OC43 – 1.6× 10<sup>6</sup>, MERS-CoV – 4.5× 10<sup>6</sup>, SARS-CoV-2 – 2.5 × 10<sup>3</sup>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-12"><h2 class="">Discussions and Conclusion</h2><p id="p-23">In this study, we designed and screened LAMP primer sets targeting SARS-CoV-2. Reaction optimization was also performed for selected primer sets. In summary, we designed and evaluated a pair of RT-LAMP assays for detection of SARS-CoV-2 with limit of detection of 100 copies per reaction. Our RT-LAMP assays showed specificity to SARS-CoV-2 versus alphacoronavirus (hCoV-229E), betacoronavirus (hCoV-OC43) and MERS-CoV. Although we could not test specificity to SARS-CoV because proper sample was not in our hand, specificity of the RT-LAMP assays are easily expectable from the mismatching bases in primer binding sites (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Especially, both of F1/B1 sites of “Nsp3_1-61” primer set is in SARS-CoV-2 specific region of which aligning SARS-CoV sequence is not exist.</p><p id="p-24">About the sensitivity of LAMP assays, note that average threshold time is not well correlated with limit of detection. In fact, average threshold time of “S_1-2-2” and “N_21” primer sets for 1000 copies were faster than that of “Nsp3_1-61”: 7.33 (RPR: 2/3) and 5.06 (RPR: 1/3) minutes, respectively. This dis-relation is previously reported [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>].One peculiar observation is that both “S_1-2-2” and “N_21” showed good sensitivity to cDNA template (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Supplementary Figure 1</a>) unlike to RNA. Observed difference of sensitivity to RNA and cDNA seems significant even we account slight amplification that might happened during reverse transcription. The reason may include stochastic nature of forming proper LAMP amplification intermediate - “dumbbell” structure [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>], higher stability of DNA:RNA double strand than DNA:DNA double strand and more.</p><p id="p-25">The limit of detection of RT-LAMP assays suggested in this study, 100 copies per reaction, may not enough for sensitive screening of suspicious patients. This relatively high limit of detection would be from the target sequences used for primer design that are selected by specificity versus SARS-CoV. Indeed, target GC percentage and Tm of primers had to be adjusted to get enough number of starting sets for primer screening during LAMP primer design to give less-optimal primer sets. Therefore, there might be better target sequences for LAMP assay of SARS-CoV-2 in the view point of sensitivity. However, expected high specificity of RT-LAMP assays suggested in this study would be a good feature for a confirmatory test. In addition, considering high viral load of SARS-CoV-2 at early stage after symptom onset [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>], suggested RT-LAMP assays may still be usable for screening tests.</p><p id="p-26">In conclusion, we developed highly specific RT-LAMP assays for detection of SARS-CoV-2. The results of these RT-LAMP assays can be detected within 30 minutes after amplification reaction begin. In addition, we provided optimized reaction conditions to which LCV colorimetric detection method is applied that can be used for point-of-care tests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-27">This work was supported by the National Research Council of Science and Technology (NST) grant by the Ministry of Science and ICT (Grant No. CRC-16-01-KRICT).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">Erratum fixed in an author's name.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.09.983064v2.1"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span>. <span class="cit-source">Coronavirus Disease 2019</span>. <span class="cit-pub-date">2020</span>; Available from: <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">https://www.who.int/emergencies/diseases/novel-coronavirus-2019</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.09.983064v2.5" data-doi="10.1016/S0140-6736(20)30183-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>, <span class="cit-pub-date">2020</span>. <span class="cit-vol">395</span>(<span class="cit-issue">10223</span>): p. <span class="cit-fpage">497</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.03.09.983064v2.6"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span>. <span class="cit-source">Coronavirus disease (COVID-19) technical guidance: Laboratory testing for 2019-nCoV in humans</span>. <span class="cit-pub-date">2020</span>; Available from: <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/laboratory-guidance</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.09.983064v2.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>,  <span class="cit-name-given-names">V.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>, <span class="cit-pub-date">2020</span>. <span class="cit-vol">25</span>(<span class="cit-issue">3</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">D.K.W.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</span>. <abbr class="cit-jnl-abbrev">Clin Chem</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.9" data-doi="10.1021/acs.analchem.6b04630"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nayak</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Point-of-Care Diagnostics: Recent Developments in a Connected Age</span>. <abbr class="cit-jnl-abbrev">Anal Chem</abbr>, <span class="cit-pub-date">2017</span>. <span class="cit-vol">89</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">102</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal%2BChem%26rft.volume%253D89%26rft.spage%253D102%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.analchem.6b04630%26rft_id%253Dinfo%253Apmid%252FISI%253A00039134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.analchem.6b04630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=ISI:00039134&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.10" data-doi="10.1016/j.tibtech.2011.01.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Niemz</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.M.</span> <span class="cit-name-surname">Ferguson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.S.</span> <span class="cit-name-surname">Boyle</span></span>, <span class="cit-article-title">Point-of-care nucleic acid testing for infectious diseases</span>. <abbr class="cit-jnl-abbrev">Trends Biotechnol</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">29</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">240</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DNiemz%26rft.auinit1%253DA.%26rft.volume%253D29%26rft.issue%253D5%26rft.spage%253D240%26rft.epage%253D250%26rft.atitle%253DPoint-of-care%2Bnucleic%2Bacid%2Btesting%2Bfor%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2011.01.007%26rft_id%253Dinfo%253Apmid%252F21377748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2011.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21377748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290891900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.11" data-doi="10.1093/nar/28.12.e63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Notomi</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Loop-mediated isothermal amplification of DNA</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2000</span>. <span class="cit-vol">28</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">E63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D28%26rft.spage%253DE63%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.12.e63%26rft_id%253Dinfo%253Apmid%252F10871386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.12.e63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10871386&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.12" data-doi="10.1186/1743-422X-11-139"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shirato</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detection of Middle East respiratory syndrome coronavirus using reverse transcription loop-mediated isothermal amplification (RT-LAMP)</span>. <abbr class="cit-jnl-abbrev">Virol J</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">11</span>: p. <span class="cit-fpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol%2BJ%26rft.volume%253D11%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-11-139%26rft_id%253Dinfo%253Apmid%252F25103205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-11-139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25103205&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.13" data-doi="10.2144/000114432"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oscorbin</span>,  <span class="cit-name-given-names">I.P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparison of fluorescent intercalating dyes for quantitative loop-mediated isothermal amplification (qLAMP)</span>. <abbr class="cit-jnl-abbrev">Biotechniques</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">61</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">20</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D61%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000114432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000114432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.14" data-doi="10.2144/000113072"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goto</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Colorimetric detection of loop-mediated isothermal amplification reaction by using hydroxy naphthol blue</span>. <abbr class="cit-jnl-abbrev">Biotechniques</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">46</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">167</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DGoto%26rft.auinit1%253DM.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D167%26rft.epage%253D172%26rft.atitle%253DColorimetric%2Bdetection%2Bof%2Bloop-mediated%2Bisothermal%2Bamplification%2Breaction%2Bby%2Busing%2Bhydroxy%2Bnaphthol%2Bblue.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000113072%26rft_id%253Dinfo%253Apmid%252F19317660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000113072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19317660&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miyamoto</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Method for colorimetric detection of double-stranded nucleic acid using leuco triphenylmethane dyes</span>. <abbr class="cit-jnl-abbrev">Anal Biochem</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">473</span>: p. <span class="cit-fpage">28</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal%2BBiochem%26rft.volume%253D473%26rft.spage%253D28%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tanner</span>,  <span class="cit-name-given-names">N.A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.C.</span> <span class="cit-name-surname">Evans</span>,   <span class="cit-name-suffix">Jr.</span>.</span>, <span class="cit-article-title">Visual detection of isothermal nucleic acid amplification using pH-sensitive dyes</span>. <abbr class="cit-jnl-abbrev">Biotechniques</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">58</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">59</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D58%26rft.spage%253D59%26rft_id%253Dinfo%253Apmid%252F25652028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25652028&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.17" data-doi="10.1371/journal.pone.0052486"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nie</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Evaluation of a direct reverse transcription loop-mediated isothermal amplification method without RNA extraction for the detection of human enterovirus 71 subgenotype C4 in nasopharyngeal swab specimens</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">7</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">e52486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De52486%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0052486%26rft_id%253Dinfo%253Apmid%252F23272248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0052486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23272248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.09.983064v2.18" data-doi="10.1093/cid/ciu210"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nyan</span>,  <span class="cit-name-given-names">D.C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Rapid detection of hepatitis B virus in blood plasma by a specific and sensitive loop-mediated isothermal amplification assay</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">59</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">16</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciu210%26rft_id%253Dinfo%253Apmid%252F24704724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/ciu210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24704724&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.19" data-doi="10.1021/acs.analchem.6b03460"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Simple and Highly Sensitive Molecular Diagnosis of Zika Virus by Lateral Flow Assays</span>. <abbr class="cit-jnl-abbrev">Anal Chem</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">88</span>(<span class="cit-issue">24</span>): p. <span class="cit-fpage">12272</span>–<span class="cit-lpage">12278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal%2BChem%26rft.volume%253D88%26rft.spage%253D12272%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.analchem.6b03460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.analchem.6b03460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jung</span>,  <span class="cit-name-given-names">Y.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Comparative analysis of primer-probe sets for the laboratory confirmation of SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">BioRxiv</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>,  <span class="cit-name-given-names">V.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Detection of a novel human coronavirus by real-time reverse-transcription polymerase chain reaction</span>. <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">17</span>(<span class="cit-issue">39</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.22" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stecher</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tamura</span></span>, <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">33</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">1870</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.23" data-doi="10.1016/0168-1702(94)00108-O"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hiscox</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Cavanagh</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Britton</span></span>, <span class="cit-article-title">Quantification of individual subgenomic mRNA species during replication of the coronavirus transmissible gastroenteritis virus</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>, <span class="cit-pub-date">1995</span>. <span class="cit-vol">36</span>(<span class="cit-issue">2-3</span>): p. <span class="cit-fpage">119</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DHiscox%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D36%26rft.issue%253D2-3%26rft.spage%253D119%26rft.epage%253D130%26rft.atitle%253DQuantification%2Bof%2Bindividual%2Bsubgenomic%2BmRNA%2Bspecies%2Bduring%2Breplication%2Bof%2Bthe%2Bcoronavirus%2Btransmissible%2Bgastroenteritis%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-1702%252894%252900108-O%26rft_id%253Dinfo%253Apmid%252F7653093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-1702(94)00108-O&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7653093&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QW56200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.24" data-doi="10.1093/nar/gkv877"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khorosheva</span>,  <span class="cit-name-given-names">E.M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Lack of correlation between reaction speed and analytical sensitivity in isothermal amplification reveals the value of digital methods for optimization: validation using digital real-time RT-LAMP</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">44</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">e10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv877%26rft_id%253Dinfo%253Apmid%252F26358811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv877&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26358811&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F24%2F2020.03.09.983064.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.09.983064v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.09.983064v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.09.983064v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.10.899989v2" title="Multi-Object Tracking in Heterogeneous environments (MOTHe) for animal video recordings" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.03.975540v2" title="Globus pallidus dynamics reveal covert strategies for behavioral inhibition" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 24, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.09.983064v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.09.983064v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.09.983064v2" class="active" data-icon-position="" data-hide-link-title="0">Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1210393" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZDO52LUSQoyh8YxiNL-kxU-xOIXCNTkNb22DBmCwM_4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1210393" id="node1210393--2" data-pisa="biorxiv;2020.03.09.983064v2" data-pisa-master="biorxiv;2020.03.09.983064" data-seqnum="1210393" data-apath="/biorxiv/early/2020/03/24/2020.03.09.983064.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gun-Soo</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Keunbon</span> <span class="nlm-surname">Ku</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Seung-Hwa</span> <span class="nlm-surname">Baek</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Seong Jun</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Seung Il</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Bum-Tae</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jin-Soo</span> <span class="nlm-surname">Maeng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.09.983064; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.09.983064 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.09.983064v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;title=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;title=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;text=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;title=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;t=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;title=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;title=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1210393" id="node1210393--3" data-pisa="biorxiv;2020.03.09.983064v2" data-pisa-master="biorxiv;2020.03.09.983064" data-seqnum="1210393" data-apath="/biorxiv/early/2020/03/24/2020.03.09.983064.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Development of Reverse Transcription Loop-mediated Isothermal Amplification (RT-LAMP) Assays Targeting SARS-CoV-2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gun-Soo</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Keunbon</span> <span class="nlm-surname">Ku</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Seung-Hwa</span> <span class="nlm-surname">Baek</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Seong Jun</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Seung Il</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Bum-Tae</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Jin-Soo</span> <span class="nlm-surname">Maeng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.09.983064; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.09.983064 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Park%2C+Gun-Soo&amp;rft.au=Ku%2C+Keunbon&amp;rft.au=Baek%2C+Seung-Hwa&amp;rft.au=Kim%2C+Seong+Jun&amp;rft.au=Kim%2C+Seung+Il&amp;rft.au=Kim%2C+Bum-Tae&amp;rft.au=Maeng%2C+Jin-Soo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1210393/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1210393/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1210393/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1210393/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1210393/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1210393/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1210393/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1210393/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1210393/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1210393/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1210393/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1210393/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;count=horizontal&amp;via=&amp;text=Development%20of%20Reverse%20Transcription%20Loop-mediated%20Isothermal%20Amplification%20%28RT-LAMP%29%20Assays%20Targeting%20SARS-CoV-2&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.09.983064v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.09.983064v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
