<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/08/2020.04.06.027805.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.06.027805v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON" />
<meta name="DC.Identifier" content="10.1101/2020.04.06.027805" />
<meta name="DC.Date" content="2020-04-08" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Background The ongoing COVID-19 pandemic has created an urgency to identify novel vaccine targets for protective immunity against SARS-CoV-2. Consistent with observations for SARS-CoV, a closely related coronavirus responsible for the 2003 SARS outbreak, early reports identify a protective role for both humoral and cell-mediated immunity for SARS CoV-2.

Methods In this study, we leveraged HLA-I and HLA-II T cell epitope prediction tools from RECON® (Real-time Epitope Computation for ONcology), our bioinformatic pipeline that was developed using proteomic profiling of individual HLA-I and HLA-II alleles to predict rules for peptide binding to a diverse set of such alleles. We applied these binding predictors to viral genomes from the Coronaviridae family, and specifically to identify SARS-CoV-2 T cell epitopes.

Results To test the suitability of these tools to identify viral T cell epitopes, we first validated HLA-I and HLA-II predictions on Coronaviridae family epitopes deposited in the Virus Pathogen Database and Analysis Resource (ViPR) database. We then use our HLA-I and HLA-II predictors to identify 11,776 HLA-I and 7,991 HLA-II candidate binding peptides across all 12 open reading frames (ORFs) of SARS-CoV-2. This extensive list of identified candidate peptides is driven by the length of the ORFs and the significant number of HLA-I and HLA-II alleles that we are able to predict (74 and 83, respectively), providing over 99% coverage for the US, European and Asian populations, for both HLA-I and HLA-II. From our SARS-CoV-2 predicted peptide-HLA-I allele pairs, 368 pairs identically matched previously reported pairs in the ViPR database, originating from other forms of coronaviruses. 320 of these pairs (89.1%) had a positive MHC-binding assay result. This analysis reinforces the validity our predictions.

Conclusions Using this bioinformatic platform, we identify multiple putative epitopes for CD4+ and CD8+ T cells whose HLA binding properties cover nearly the entire population and thus may be effective when included in prophylactic vaccines against SARS-CoV-2 to induce broad cellular immunity.

### Competing Interest Statement



*   S
    :   Human Leukocyte Antigen
    MERS-CoV
    :   Middle East Respiratory Syndrome – Coronavirus
    MHC
    :   Major histocompatibility complex
    RECON
    :   Real-time Epitope Computation for ONcology
    SARS-CoV
    :   Severe Acute Respiratory Syndrome – Coronavirus
    SARS-CoV-2
    :   Severe Acute Respiratory Syndrome – Coronavirus – 2
    USA
    :   United States of America
    ViPR
    :   Virus Pathogen Resource
    WHO
    :   World Health Organization" />
<meta name="DC.Contributor" content="Asaf Poran" />
<meta name="DC.Contributor" content="Dewi Harjanto" />
<meta name="DC.Contributor" content="Matthew Malloy" />
<meta name="DC.Contributor" content="Michael S. Rooney" />
<meta name="DC.Contributor" content="Lakshmi Srinivasan" />
<meta name="DC.Contributor" content="Richard B. Gaynor" />
<meta name="article:published_time" content="2020-04-08" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Background: The ongoing COVID-19 pandemic has created an urgency to identify novel vaccine targets for protective immunity against SARS-CoV-2. Consistent with observations for SARS-CoV, a closely related coronavirus responsible for the 2003 SARS outbreak, early reports identify a protective role for both humoral and cell-mediated immunity for SARS CoV-2. 
Methods: In this study, we leveraged HLA-I and HLA-II T cell epitope prediction tools from RECON (R) (Real-time Epitope Computation for ONcology), our bioinformatic pipeline that was developed using proteomic profiling of individual HLA-I and HLA-II alleles to predict rules for peptide binding to a diverse set of such alleles. We applied these binding predictors to viral genomes from the Coronaviridae family, and specifically to identify SARS-CoV-2 T cell epitopes.
Results: To test the suitability of these tools to identify viral T cell epitopes, we first validated HLA-I and HLA-II predictions on Coronaviridae family epitopes deposited in the Virus Pathogen Database and Analysis Resource (ViPR) database. We then use our HLA-I and HLA-II predictors to identify 11,776 HLA-I and 7,991 HLA-II candidate binding peptides across all 12 open reading frames (ORFs) of SARS-CoV-2. This extensive list of identified candidate peptides is driven by the length of the ORFs and the significant number of HLA-I and HLA-II alleles that we are able to predict (74 and 83, respectively), providing over 99% coverage for the US, European and Asian populations, for both HLA-I and HLA-II. From our SARS-CoV-2 predicted peptide-HLA-I allele pairs, 368 pairs identically matched previously reported pairs in the ViPR database, originating from other forms of coronaviruses. 320 of these pairs (89.1%) had a positive MHC-binding assay result. This analysis reinforces the validity our predictions.
Conclusions: Using this bioinformatic platform, we identify multiple putative epitopes for CD4+ and CD8+ T cells whose HLA binding properties cover nearly the entire population and thus may be effective when included in prophylactic vaccines against SARS-CoV-2 to induce broad cellular immunity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.06.027805v1" />
<meta name="citation_id" content="2020.04.06.027805v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.027805v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.027805v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.027805v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.06.027805" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.06.027805" />
<meta name="citation_author" content="Asaf Poran" />
<meta name="citation_author_institution" content="Neon Therapeutics, Inc." />
<meta name="citation_author_email" content="aporan@neontherapeutics.com" />
<meta name="citation_author_email" content="dharjanto@neontherapeutics.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9118-1051" />
<meta name="citation_author" content="Dewi Harjanto" />
<meta name="citation_author_institution" content="Neon Therapeutics, Inc." />
<meta name="citation_author_email" content="aporan@neontherapeutics.com" />
<meta name="citation_author_email" content="dharjanto@neontherapeutics.com" />
<meta name="citation_author" content="Matthew Malloy" />
<meta name="citation_author_institution" content="Neon Therapeutics, Inc." />
<meta name="citation_author" content="Michael S. Rooney" />
<meta name="citation_author_institution" content="Neon Therapeutics, Inc." />
<meta name="citation_author" content="Lakshmi Srinivasan" />
<meta name="citation_author_institution" content="Neon Therapeutics, Inc." />
<meta name="citation_author" content="Richard B. Gaynor" />
<meta name="citation_author_institution" content="Neon Therapeutics, Inc." />
<meta name="citation_reference" content="Cui J, Li F, Shi Z-L. Origin and evolution of pathogenic coronaviruses. Nat Rev Microbiol. 2019;17(3):181–92." />
<meta name="citation_reference" content="Gu J, Gong E, Zhang B, Zheng J, Gao Z, Zhong Y, et al. Multiple organ infection and the pathogenesis of SARS. J Exp Med. 2005 Aug 1;202(3):415–24." />
<meta name="citation_reference" content="Rota PA, Oberste MS, Monroe SS, Nix WA, Campagnoli R, Icenogle JP, et al. Characterization of a novel coronavirus associated with severe acute respiratory syndrome. Science. 2003 May 30;300(5624):1394–9." />
<meta name="citation_reference" content="Ksiazek TG, Erdman D, Goldsmith CS, Zaki SR, Peret T, Emery S, et al. A novel coronavirus associated with severe acute respiratory syndrome. N Engl J Med. 2003 May 15;348(20):1953–66." />
<meta name="citation_reference" content="WHO MERS-CoV Global Summary and Assessment of Risk. World Health Organization; 2018. https://www.who.int/csr/disease/coronavirus_infections/risk-assessment-august-2018.pdf" />
<meta name="citation_reference" content="WHO | Middle East respiratory syndrome coronavirus (MERS-CoV). World Health Organization; [cited 2020 Mar 25]. http://www.who.int/emergencies/mers-cov/en/" />
<meta name="citation_reference" content="WHO | Update 49 - SARS case fatality ratio, incubation period. World Health Organization; [cited 2020 Mar 25]. https://www.who.int/csr/sars/archive/2003_05_07a/en/" />
<meta name="citation_reference" content="WHO | Coronavirus disease (COVID-19) Pandemic. [cited 2020 Mar 25]. https://www.who.int/emergencies/diseases/novel-coronavirus-2019" />
<meta name="citation_reference" content="Chan JF-W, Kok K-H, Zhu Z, Chu H, To KK-W, Yuan S, et al. Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan. Emerg Microbes Infect. 2020 Jan 28;9(1):221–36." />
<meta name="citation_reference" content="Zhou P, Yang X-L, Wang X-G, Hu B, Zhang L, Zhang W, et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature. 2020;579(7798):270–3." />
<meta name="citation_reference" content="Lu R, Zhao X, Li J, Niu P, Yang B, Wu H, et al. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet. 2020 22;395(10224):565–74." />
<meta name="citation_reference" content="Li G, Fan Y, Lai Y, Han T, Li Z, Zhou P, et al. Coronavirus infections and immune responses. J Med Virol. 2020;92(4):424–32." />
<meta name="citation_reference" content="Yang Z-Y, Kong W-P, Huang Y, Roberts A, Murphy BR, Subbarao K, et al. A DNA vaccine induces SARS coronavirus neutralization and protective immunity in mice. Nature. 2004 Apr 1;428(6982):561–4." />
<meta name="citation_reference" content="Graham RL, Becker MM, Eckerle LD, Bolles M, Denison MR, Baric RS. A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease. Nat Med. 2012 Dec;18(12):1820–6." />
<meta name="citation_reference" content="Wang J, Wen J, Li J, Yin J, Zhu Q, Wang H, et al. Assessment of immunoreactive synthetic peptides from the structural proteins of severe acute respiratory syndrome coronavirus. Clin Chem. 2003 Dec;49(12):1989–96." />
<meta name="citation_reference" content="Liu X, Shi Y, Li P, Li L, Yi Y, Ma Q, et al. Profile of antibodies to the nucleocapsid protein of the severe acute respiratory syndrome (SARS)-associated coronavirus in probable SARS patients. Clin Diagn Lab Immunol. 2004 Jan;11(1):227–8." />
<meta name="citation_reference" content="Tang F, Quan Y, Xin Z-T, Wrammert J, Ma M-J, Lv H, et al. Lack of Peripheral Memory B Cell Responses in Recovered Patients with Severe Acute Respiratory Syndrome: A Six-Year Follow-Up Study. The Journal of Immunology. 2011 Jun 15;186(12):7264–8." />
<meta name="citation_reference" content="Cui W, Fan Y, Wu W, Zhang F, Wang J, Ni A. Expression of lymphocytes and lymphocyte subsets in patients with severe acute respiratory syndrome. Clin Infect Dis. 2003 Sep 15;37(6):857–9." />
<meta name="citation_reference" content="Li T, Qiu Z, Zhang L, Han Y, He W, Liu Z, et al. Significant changes of peripheral T lymphocyte subsets in patients with severe acute respiratory syndrome. J Infect Dis. 2004 Feb 15;189(4):648–51." />
<meta name="citation_reference" content="Channappanavar R, Zhao J, Perlman S. T-cell-mediated immune response to respiratory coronaviruses. Immunol Res. 2014 Aug;59(0):118–28." />
<meta name="citation_reference" content="Channappanavar R, Fett C, Zhao J, Meyerholz DK, Perlman S. Virus-specific memory CD8 T cells provide substantial protection from lethal severe acute respiratory syndrome coronavirus infection. J Virol. 2014 Oct;88(19):11034–44." />
<meta name="citation_reference" content="Li CK, Wu H, Yan H, Ma S, Wang L, Zhang M, et al. T Cell Responses to Whole SARS Coronavirus in Humans. The Journal of Immunology. 2008 Oct 15;181(8):5490–500." />
<meta name="citation_reference" content="Thevarajan I, Nguyen THO, Koutsakos M, Druce J, Caly L, van de Sandt CE, et al. Breadth of concomitant immune responses prior to patient recovery: a case report of non-severe COVID-19. Nature Medicine. 2020 Mar 16;1–3." />
<meta name="citation_reference" content="Ng O-W, Chia A, Tan AT, Jadi RS, Leong HN, Bertoletti A, et al. Memory T cell responses targeting the SARS coronavirus persist up to 11 years post-infection. Vaccine. 2016 Apr 12;34(17):2008–14." />
<meta name="citation_reference" content="Zhao J, Zhao J, Perlman S. T Cell Responses Are Required for Protection from Clinical Disease and for Virus Clearance in Severe Acute Respiratory Syndrome Coronavirus-Infected Mice. J Virol. 2010 Sep;84(18):9318–25." />
<meta name="citation_reference" content="Abelin JG, Keskin DB, Sarkizova S, Hartigan CR, Zhang W, Sidney J, et al. Mass Spectrometry Profiling of HLA-Associated Peptidomes in Mono-allelic Cells Enables More Accurate Epitope Prediction. Immunity. 2017 21;46(2):315–26." />
<meta name="citation_reference" content="Abelin JG, Harjanto D, Malloy M, Suri P, Colson T, Goulding SP, et al. Defining HLA-II Ligand Processing and Binding Rules with Mass Spectrometry Enhances Cancer Epitope Prediction. Immunity. 2019 15;51(4):766–779.e17." />
<meta name="citation_reference" content="Archila LLD, Kwok WW. Tetramer-Guided Epitope Mapping: A Rapid Approach to Identify HLA-Restricted T-Cell Epitopes from Composite Allergens. Methods Mol Biol. 2017;1592:199–209." />
<meta name="citation_reference" content="Yang J, James EA, Huston L, Danke NA, Liu AW, Kwok WW. Multiplex mapping of CD4 T cell epitopes using class II tetramers. Clin Immunol. 2006 Jul;120(1):21–32." />
<meta name="citation_reference" content="Pickett BE, Sadat EL, Zhang Y, Noronha JM, Squires RB, Hunt V, et al. ViPR: an open bioinformatics database and analysis resource for virology research. Nucleic Acids Res. 2012 Jan;40(Database issue):D593–8." />
<meta name="citation_reference" content="Grifoni A, Sidney J, Zhang Y, Scheuermann RH, Peters B, Sette A. A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2. Cell Host Microbe. 2020 Mar 12;" />
<meta name="citation_reference" content="Maiers M, Gragert L, Klitz W. High-resolution HLA alleles and haplotypes in the United States population. Hum Immunol. 2007 Sep;68(9):779–88." />
<meta name="citation_reference" content="González-Galarza FF, Takeshita LYC, Santos EJM, Kempson F, Maia MHT, da Silva ALS, et al. Allele frequency net 2015 update: new features for HLA epitopes, KIR and disease and HLA adverse drug reaction associations. Nucleic Acids Res. 2015 Jan;43(Database issue):D784–788." />
<meta name="citation_reference" content="Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, et al. The human genome browser at UCSC. Genome Res. 2002 Jun;12(6):996–1006." />
<meta name="citation_reference" content="Ahmed SF, Quadeer AA, McKay MR. Preliminary Identification of Potential Vaccine Targets for the COVID-19 Coronavirus (SARS-CoV-2) Based on SARS-CoV Immunological Studies. Viruses. 2020 25;12(3)." />
<meta name="citation_reference" content="Zhao J, Zhao J, Mangalam AK, Channappanavar R, Fett C, Meyerholz DK, et al. Airway Memory CD4+ T Cells Mediate Protective Immunity against Emerging Respiratory Coronaviruses. Immunity. 2016 Jun 21;44(6):1379–91." />
<meta name="twitter:title" content="Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Background The ongoing COVID-19 pandemic has created an urgency to identify novel vaccine targets for protective immunity against SARS-CoV-2. Consistent with observations for SARS-CoV, a closely related coronavirus responsible for the 2003 SARS outbreak, early reports identify a protective role for both humoral and cell-mediated immunity for SARS CoV-2.

Methods In this study, we leveraged HLA-I and HLA-II T cell epitope prediction tools from RECON® (Real-time Epitope Computation for ONcology), our bioinformatic pipeline that was developed using proteomic profiling of individual HLA-I and HLA-II alleles to predict rules for peptide binding to a diverse set of such alleles. We applied these binding predictors to viral genomes from the Coronaviridae family, and specifically to identify SARS-CoV-2 T cell epitopes.

Results To test the suitability of these tools to identify viral T cell epitopes, we first validated HLA-I and HLA-II predictions on Coronaviridae family epitopes deposited in the Virus Pathogen Database and Analysis Resource (ViPR) database. We then use our HLA-I and HLA-II predictors to identify 11,776 HLA-I and 7,991 HLA-II candidate binding peptides across all 12 open reading frames (ORFs) of SARS-CoV-2. This extensive list of identified candidate peptides is driven by the length of the ORFs and the significant number of HLA-I and HLA-II alleles that we are able to predict (74 and 83, respectively), providing over 99% coverage for the US, European and Asian populations, for both HLA-I and HLA-II. From our SARS-CoV-2 predicted peptide-HLA-I allele pairs, 368 pairs identically matched previously reported pairs in the ViPR database, originating from other forms of coronaviruses. 320 of these pairs (89.1%) had a positive MHC-binding assay result. This analysis reinforces the validity our predictions.

Conclusions Using this bioinformatic platform, we identify multiple putative epitopes for CD4+ and CD8+ T cells whose HLA binding properties cover nearly the entire population and thus may be effective when included in prophylactic vaccines against SARS-CoV-2 to induce broad cellular immunity.

### Competing Interest Statement



*   S
    :   Human Leukocyte Antigen
    MERS-CoV
    :   Middle East Respiratory Syndrome – Coronavirus
    MHC
    :   Major histocompatibility complex
    RECON
    :   Real-time Epitope Computation for ONcology
    SARS-CoV
    :   Severe Acute Respiratory Syndrome – Coronavirus
    SARS-CoV-2
    :   Severe Acute Respiratory Syndrome – Coronavirus – 2
    USA
    :   United States of America
    ViPR
    :   Virus Pathogen Resource
    WHO
    :   World Health Organization" />
<meta name="og-title" property="og:title" content="Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.027805v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Background The ongoing COVID-19 pandemic has created an urgency to identify novel vaccine targets for protective immunity against SARS-CoV-2. Consistent with observations for SARS-CoV, a closely related coronavirus responsible for the 2003 SARS outbreak, early reports identify a protective role for both humoral and cell-mediated immunity for SARS CoV-2.

Methods In this study, we leveraged HLA-I and HLA-II T cell epitope prediction tools from RECON® (Real-time Epitope Computation for ONcology), our bioinformatic pipeline that was developed using proteomic profiling of individual HLA-I and HLA-II alleles to predict rules for peptide binding to a diverse set of such alleles. We applied these binding predictors to viral genomes from the Coronaviridae family, and specifically to identify SARS-CoV-2 T cell epitopes.

Results To test the suitability of these tools to identify viral T cell epitopes, we first validated HLA-I and HLA-II predictions on Coronaviridae family epitopes deposited in the Virus Pathogen Database and Analysis Resource (ViPR) database. We then use our HLA-I and HLA-II predictors to identify 11,776 HLA-I and 7,991 HLA-II candidate binding peptides across all 12 open reading frames (ORFs) of SARS-CoV-2. This extensive list of identified candidate peptides is driven by the length of the ORFs and the significant number of HLA-I and HLA-II alleles that we are able to predict (74 and 83, respectively), providing over 99% coverage for the US, European and Asian populations, for both HLA-I and HLA-II. From our SARS-CoV-2 predicted peptide-HLA-I allele pairs, 368 pairs identically matched previously reported pairs in the ViPR database, originating from other forms of coronaviruses. 320 of these pairs (89.1%) had a positive MHC-binding assay result. This analysis reinforces the validity our predictions.

Conclusions Using this bioinformatic platform, we identify multiple putative epitopes for CD4+ and CD8+ T cells whose HLA binding properties cover nearly the entire population and thus may be effective when included in prophylactic vaccines against SARS-CoV-2 to induce broad cellular immunity.

### Competing Interest Statement



*   S
    :   Human Leukocyte Antigen
    MERS-CoV
    :   Middle East Respiratory Syndrome – Coronavirus
    MHC
    :   Major histocompatibility complex
    RECON
    :   Real-time Epitope Computation for ONcology
    SARS-CoV
    :   Severe Acute Respiratory Syndrome – Coronavirus
    SARS-CoV-2
    :   Severe Acute Respiratory Syndrome – Coronavirus – 2
    USA
    :   United States of America
    ViPR
    :   Virus Pathogen Resource
    WHO
    :   World Health Organization" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-08" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.06.027805v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.06.027805v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.06.027805v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.06.027805v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1235209" />
    <title>Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6aV1QpmlS_gy_IdmmKWJxDM-RQTea8NtoMc0xg4bInY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1235209","apath":"\/biorxiv\/early\/2020\/04\/08\/2020.04.06.027805.atom","pisa":"biorxiv;2020.04.06.027805v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.06.027805v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.06.027805v1","title":"Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON","identifier":"node\/1235209"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226v\\\\\\\/Ha19p\\\\\\\/XhKOCyH2xYqarYd\\\\\\\/FqigWaU5ejzCWVidupdxRdsrM0x+W2ZW3ynuOA5mRBhwf3EAsjLd3lEXUAem458H1Xw8KJGbdX0LoUUz+bcZ\\\\\\\/PloaQChJ3e++ek7WTihq4BwHUq7BmkADH+a8xxdbrmJv7lZz7R+KAM2YcIBesmX4w16eOu470BLkmlspIAiPDMYUtCvFepGrzP3Mnkuu9wNWgHu+SA83bmCP8v5OUx+KlyBcEVPMklY8aB0QQmVkrW31WnEL+Y2nhkpPx4G1PhWptSZ8IOW1gDzozUj2BAItGDbYBngCrDqsH40sP0HN3BPqWrOqvJphjmo0yzaRT7od2RzsWapI\\\\\\\/pF33zIHA\\\\\\\/vuQu5mnIiqLCFsQf1JWlLTU+KMDHiLi9GOoPSZsBpUXJpHb5n1KYnDzwc3Cn69nUpuuqjF3Gu3m1xS7MGhKo8ft3s\\\\\\\/kdn8anK+7\\\\\\\/xHcQhv4\\\\\\\/KYGaDES0CBmKG1RlD18pk0wGh0KqV9BIL0X3DiPXY8vZ5RMN1k+Es0Di+EBLrE9J0esrPHuNys7seMcO2gZfoaQppTejLQjd9ZUM1I0qfRD8U5gBJ7Jq0FpxavImFZmeQffi62JPf06e5\\\\\\\/tG\\\\\\\/sBY\\\\\\\/Ajicm+SMN0gVcTWAeAqWGNK7PPlqovkL3f7jlj9kRwm4r7hVEa3knaGEmm+xG7wEXlfhWp91xOG2Bv388SjfOSQ+7yddGHCdCPIjgR6LrfDh4d0TLtESkcL4xNNvk0VX9We4vqNc5hLQn86gabnkmho3XdYO79IMAef8us2kfob9d6ysLhlKwmwp8iPa1Ska9ttgftp3QZeNMtW0My1l0KBPq+Jk9XegpWeYJWpgObu05MDOG0oVG5t0hhsaNfCGnVkPkPx2SfN1FTILwelfukbfbk87K5LaschgrdvjzdiNMKpjN9sOn1AKnlwFeYFNcXq8FkYatQ+S8XjHkEARkCsp\\\\\\\/vnYqXP2iIIV76SsXNFkwE+acIWrjrj6nFJy9RZ5yHfrYclhVc4NukvP8MzBhZJVtIOR99rA+4v2W510GBF3trjpH92bIZoGk2nhzUCyjZz6ALA3Nj720Hvj9ITU0Q2oxhFLO7f7fpNhwlKB6eQa30RnChxKvIMuqgHwAT8L2Ph7OJGi0X0R0cIJp1CXxlo8HL5ccKBRhb+WFpZ+RIFDgD\\\\\\\/yUCgqp3gqb4ueGorL\\\\\\\/QpQ5\\\\\\\/yg5sxVDQDGJbp75TzszeRAyLon6u2P\\\\\\\/9\\\\\\\/TyyGm7efrKCAQTGG2LeFvsjkyy+A7GqzWuUjFT77HXmhX\\\\\\\/pVRhqUcX+PQpHZ\\\\\\\/upjOA7IVGC3\\\\\\\/CPxNXYqpbyM3zlvHLsHxOBIpuQkQ+\\\\\\\/MVp+L\\\u0022,\\\u0022iv\\\u0022:\\\u0022XOuJt+jaj10eee+BW7Ka3Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e0677eedf7c3d74cdff8d75e64510346cb11e19106376c292093a6573a18adb3\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.06.027805v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1235209 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.06.027805v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XdffeIy_AFGS3klMtridmPRMbqgqsQqp6eAxTT-O3mU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1235209" id="node1235209" data-pisa="biorxiv;2020.04.06.027805v1" data-pisa-master="biorxiv;2020.04.06.027805" data-apath="/biorxiv/early/2020/04/08/2020.04.06.027805.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9118-1051" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Asaf</span> <span class="nlm-surname">Poran</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dewi</span> <span class="nlm-surname">Harjanto</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Matthew</span> <span class="nlm-surname">Malloy</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Michael S.</span> <span class="nlm-surname">Rooney</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lakshmi</span> <span class="nlm-surname">Srinivasan</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Richard B.</span> <span class="nlm-surname">Gaynor</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.06.027805 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1235209" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Asaf Poran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Neon Therapeutics, Inc.</span>, 40 Erie Street, Suite 110, Cambridge, MA 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asaf%2BPoran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poran%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAsaf%2BPoran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9118-1051" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Asaf Poran</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aporan@neontherapeutics.com" class="" data-icon-position="" data-hide-link-title="0">aporan@neontherapeutics.com</a>
<a href="mailto:dharjanto@neontherapeutics.com" class="" data-icon-position="" data-hide-link-title="0">dharjanto@neontherapeutics.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dewi Harjanto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Neon Therapeutics, Inc.</span>, 40 Erie Street, Suite 110, Cambridge, MA 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dewi%2BHarjanto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harjanto%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADewi%2BHarjanto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aporan@neontherapeutics.com" class="" data-icon-position="" data-hide-link-title="0">aporan@neontherapeutics.com</a>
<a href="mailto:dharjanto@neontherapeutics.com" class="" data-icon-position="" data-hide-link-title="0">dharjanto@neontherapeutics.com</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthew Malloy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Neon Therapeutics, Inc.</span>, 40 Erie Street, Suite 110, Cambridge, MA 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BMalloy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Malloy%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BMalloy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael S. Rooney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Neon Therapeutics, Inc.</span>, 40 Erie Street, Suite 110, Cambridge, MA 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BS.%2BRooney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rooney%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BS.%2BRooney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lakshmi Srinivasan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Neon Therapeutics, Inc.</span>, 40 Erie Street, Suite 110, Cambridge, MA 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lakshmi%2BSrinivasan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srinivasan%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALakshmi%2BSrinivasan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard B. Gaynor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Neon Therapeutics, Inc.</span>, 40 Erie Street, Suite 110, Cambridge, MA 02139</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BB.%2BGaynor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gaynor%20RB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BB.%2BGaynor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.06.027805v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1235209" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1235209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.06.027805v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1235209" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1235209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.06.027805v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1235209" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1235209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.06.027805v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1235209" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1235209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.06.027805v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1235209" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1235209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.06.027805v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1235209" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1235209/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><div id="sec-1" class="subsection"><p id="p-3"><strong>Background</strong> The ongoing COVID-19 pandemic has created an urgency to identify novel vaccine targets for protective immunity against SARS-CoV-2. Consistent with observations for SARS-CoV, a closely related coronavirus responsible for the 2003 SARS outbreak, early reports identify a protective role for both humoral and cell-mediated immunity for SARS CoV-2.</p></div><div id="sec-2" class="subsection"><p id="p-4"><strong>Methods</strong> In this study, we leveraged HLA-I and HLA-II T cell epitope prediction tools from RECON® (Real-time Epitope Computation for ONcology), our bioinformatic pipeline that was developed using proteomic profiling of individual HLA-I and HLA-II alleles to predict rules for peptide binding to a diverse set of such alleles. We applied these binding predictors to viral genomes from the <em>Coronaviridae</em> family, and specifically to identify SARS-CoV-2 T cell epitopes.</p></div><div id="sec-3" class="subsection"><p id="p-5"><strong>Results</strong> To test the suitability of these tools to identify viral T cell epitopes, we first validated HLA-I and HLA-II predictions on <em>Coronaviridae</em> family epitopes deposited in the Virus Pathogen Database and Analysis Resource (ViPR) database. We then use our HLA-I and HLA-II predictors to identify 11,776 HLA-I and 7,991 HLA-II candidate binding peptides across all 12 open reading frames (ORFs) of SARS-CoV-2. This extensive list of identified candidate peptides is driven by the length of the ORFs and the significant number of HLA-I and HLA-II alleles that we are able to predict (74 and 83, respectively), providing over 99% coverage for the US, European and Asian populations, for both HLA-I and HLA-II. From our SARS-CoV-2 predicted peptide-HLA-I allele pairs, 368 pairs identically matched previously reported pairs in the ViPR database, originating from other forms of coronaviruses. 320 of these pairs (89.1%) had a positive MHC-binding assay result. This analysis reinforces the validity our predictions.</p></div><div id="sec-4" class="subsection"><p id="p-6"><strong>Conclusions</strong> Using this bioinformatic platform, we identify multiple putative epitopes for CD4<sup>+</sup> and CD8<sup>+</sup> T cells whose HLA binding properties cover nearly the entire population and thus may be effective when included in prophylactic vaccines against SARS-CoV-2 to induce broad cellular immunity.</p></div></div><div class="section" id="sec-5"><h2 class="">Background</h2><p id="p-8">Coronaviruses are positive-sense single-stranded RNA viruses that have occasionally emerged from zoonotic sources to infect human populations (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). Most of the infections cause mild respiratory symptoms. However, some recent coronavirus infections have resulted in serious morbidity and mortality, including the severe acute respiratory syndrome coronavirus (SARS-CoV) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>–<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>), Middle East respiratory syndrome coronavirus (MERS-CoV) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>) and SARS-CoV-2, which is responsible for the current spreading of COVID-19. These three viruses all belong to the genus <em>Betacoronaviridae</em> (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>). SARS-CoV was identified in South China in 2002 and its global spread led to 8096 cases and 774 deaths (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). The first case of MERS-CoV emerged in 2012 in Saudi Arabia, and since then a total of 2,494 cases and 858 associated deaths have been reported (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>). In contrast to the more limited scope of these other coronavirus infections, SARS-CoV-2, which emerged in Wuhan, China at the end of December 2019, has resulted in 334,981 cases, including 14,652 deaths globally as of March 23, 2020 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). The rapid spread of SARS-CoV-2 has resulted in the World Health Organization declaring a global pandemic. Thus, there is an urgent need for effective vaccines and antiviral treatments against SARS-CoV-2 to deal with this global pandemic.</p><p id="p-9">The genome of SARS-CoV-2 spans 30 kilobases in length and encodes for 12 open-reading frames (ORFs), including four structural proteins. These structural proteins are the spike protein (S), the membrane protein (M), the envelope protein (E), and the nucleocapsid protein (N). In addition, there are over 20 non-structural proteins that account for all the proteins involved in the transcription and replication of the virus (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). All encoded proteins of the virus are potential candidates for developing vaccines to induce robust T cell immunity.</p><p id="p-10">SARS-CoV and SARS-CoV-2 share 76% amino acid identity across the genome (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). This high degree of sequence similarity allows us to leverage the previous research on protective immune responses to SAR-CoV to aid in vaccine development for SARS-CoV-2. Both humoral and cellular immune responses have been shown to be important in host responses to SARS-CoV (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). Antibody responses generated against the S and the N proteins have shown to protect from SARS-CoV infection in mice and have been detected in SARS-CoV infected patients (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>–<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). However, these antibody responses detected against the S protein were short-lived and undetectable in patients six years post-recovery, suggesting that T cell responses may be involved in the long-term control of this virus (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). Indeed, significant changes in the total lymphocyte counts and T cell subset composition have been observed in patients with SARS-CoV; namely, levels of both B cells, and CD4<sup>+</sup> and CD8<sup>+</sup> T cells have been significantly reduced in these patients (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). Similarly, mice infected with SARS-CoV demonstrated that the severity of SARS correlated with the ability to develop a virus-specific T cell response (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>).</p><p id="p-11">Both CD4<sup>+</sup> and CD8<sup>+</sup> T cell responses have been detected in SARS-CoV-infected patients (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>) as well as in SARS-CoV-2 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). Notably, SARS-CoV-specific memory CD8<sup>+</sup> T cells persisted up to 11 years post-infection in patients who recovered from SARS (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). Studies in mice have shown that SARS-CoV-specific memory CD8<sup>+</sup> T cells provided protection against a lethal SARS-CoV infection in aged mice (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>). In addition, adoptive transfer of effector CD4<sup>+</sup> and CD8<sup>+</sup> T cells to immunodeficient or young mice expedited virus clearance and improved clinical results (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>). Immunization with SARS-CoV peptide-pulsed dendritic cells invigorated a T cell response, increasing the number of virus-specific CD8<sup>+</sup> T cells enhancing both virus clearance and overall survival (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). These studies indicate an important role for T cell responses in controlling disease severity, virus clearance and conferring protective immunity to SARS-CoV infections. Given the homology between SARS-CoV and SARS-CoV-2, as well as emerging data on SARS-CoV-2 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>), cellular immune mechanisms might play a critical role in providing protection against SARS-CoV-2.</p><p id="p-12">Here, we used T cell epitope prediction tools from the bioinformatic pipeline RECON® (Real-time Epitope Computation for ONcology) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>) to identify SARS-CoV-2 epitopes recognized by CD4<sup>+</sup> and CD8<sup>+</sup> T cells. RECON was trained on high-quality mono-allelic major histocompatibility complex (MHC) immunopeptidome data generated via mass spectrometry. The use of mass spectrometry allows for the high throughput, and relatively unbiased, collection of MHC binding data compared to traditional binding affinity assays, as well as the inclusion of important chaperone molecules. Additionally, the use of engineered mono-allelic cell lines avoids dependence on in-silico deconvolution techniques and allows for allele coverage to be expanded in a targeted manner.</p><p id="p-13">With this approach, we generated data for 74 human leukocyte antigen (HLA)-I and 83 HLA-II alleles (Supplementary Tables 1 and 2). This mass spectrometry data enabled us to train neural network-based binding predictors that outperform the leading affinity-based predictors for both HLA-I (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>) and HLA-II (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>). Furthermore, we demonstrated in (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>) that this improved binding prediction leads to improved immunogenicity prediction by validating on a data set of tetramer responses to a diverse collection of pathogens and allergens (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>). Although RECON was originally developed to prioritize neoantigens for immunotherapy applications, it is agnostic to the source of peptide sequences evaluated and can be easily applied to peptides derived from pathogens as well. As validation to that end, the binding predictors from RECON were used to score <em>Coronaviridae</em> family peptides that had been assayed for T cell reactivity or MHC binding from the Virus Pathogen Resource (ViPR) database (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). The ViPR database integrates viral pathogen data from internally curated data, researcher submissions and data from various external sources. Our approach provides a significant improvement in both the breadth of predictions, and their validity, compared with a recent study that had a similar aim (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). We used the HLA-I and HLA-II binding predictors from RECON to predict the binding potential of peptide sequences from across the entire SARS-CoV-2 genome for a broad set of HLA-I and HLA-II alleles, covering the vast majority of USA, European, and Asian populations (Supplementary Table 3). Epitopes that were predicted to have a high likelihood of binding for multiple alleles could potentially be included in vaccines to stimulate CD4<sup>+</sup> and CD8<sup>+</sup> immune responses against this virus.</p></div><div class="section" id="sec-6"><h2 class="">METHODS</h2><div id="sec-7" class="subsection"><h3>Retrieval of <em>Coronaviridae</em> family T cell epitopes from ViPR</h3><p id="p-14">Experimentally determined epitopes for the <em>Coronaviridae</em> family for human hosts were retrieved from the Virus Pathogen Database and Analysis Resource (ViPR) (<a href="https://www.viprbrc.org/">https://www.viprbrc.org/</a>; accessed March 5 2020) (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>). To build a validation dataset, both positives and negatives for T cell assays and MHC binding assays were obtained. Only assays associated with alleles identified with at least four-digit resolution and supported by RECON (Supplementary Table 1) were included for this analysis. Positive calls were prioritized – that is, if a given peptide-allele pair was assayed multiple times by a specific assay type and was determined to be positive in any single one of the assays, the peptide-allele pair was classified as positive. Specifically, the priority was given by the following order: Positive-High &gt; Positive-Intermediate &gt; Positive-Low &gt; Positive &gt; Negative (e.g., a peptide allele pairing that was assayed three times with the results Positive-High, Positive, and Negative were assigned a Positive-High result).</p></div><div id="sec-8" class="subsection"><h3>Binding prediction for ViPR <em>Coronaviridae</em> family T cell epitopes</h3><p id="p-15">Peptide-HLA-I allele pairs in the ViPR validation dataset were scored using RECON’s HLA-I binding predictor, a neural network-based model trained on mass spectrometry data (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>).</p><p id="p-16">Similarly, peptide-HLA-II allele pairs in the ViPR validation dataset were scored using RECON’s HLA-II binding predictor, a recently published convolutional neural network-based model trained on mono-allelic mass spectrometry data (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">27</a>). When applying the HLA-II binding predictor, we used the highest score for all 12-20mers within a given assay peptide. This is meant to account for the fact that the predictor is trained on ligands observed via mass spectrometry and may learn processing rules that are irrelevant for assays that do not incorporate processing and presentation.</p></div><div id="sec-9" class="subsection"><h3>Retrieval of SARS-CoV-2 sequence</h3><p id="p-17">The GenBank reference sequence for SARS-CoV-2 (accession: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_045512.2&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom">NC_045512.2</a>) was used for this study. All twelve annotated open-reading frames (orf1a, orf1b, S, ORF3a, E, M, ORF6, ORF7a, ORF7b, ORF8, N, and ORF10) were considered as sources of potential epitopes.</p></div><div id="sec-10" class="subsection"><h3>Identification of HLA-I Epitopes</h3><p id="p-18">To identify candidate HLA-I epitopes, all possible 8-12mer peptide sequences from SARS-CoV-2 were scored with RECON’s HLA-I binding predictor. The HLA-I binding predictor was used to score SARS-CoV-2 peptides binding against 74 alleles, including 21 HLA-A alleles, 35 HLA-B alleles, and 18 HLA-C alleles. Peptide-allele pairs were assigned a percent rank by comparing their binding scores to those of 1,000,000 reference peptides for the same respective allele. Peptide-allele pairs that scored in the top 1% of the scores of these reference peptides were considered strong potential binders.</p><p id="p-19">These top-ranking peptides were then prioritized based on expected USA population coverage (allele frequencies obtained from (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>) – USA frequencies calculated as follows: 0.623*EUR0.133*AFA0.068*APA0.176*HIS), given all the alleles each peptide was expected to bind to (i.e., all the alleles for which the peptide scored in the top 1%). The estimate of population coverage for each peptide was calculated as
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.027805/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/08/2020.04.06.027805/embed/graphic-1.gif"/></noscript></span>

</span>
where f<sub>allele,avg</sub> is the (unweighted) average allele frequency across the USA, European, and Asian Pacific Islander (API) populations and the cumulative product is taken across the three HLA-I loci: (HLA-A, HLA-B, and HLA-C).</p><p id="p-20">The cumulative product itself represents the chance that an individual in the USA does not express any one of the contained alleles; hence, the complement describes the probability that at least one is present. The aim of using USA, European, and API allele frequencies is to cover a diverse population where allele frequency estimates are relatively reliable.</p><p id="p-21">We then construct two ranked lists of HLA-I epitopes by coverage. The first ranks the epitopes by their absolute coverage, such that sequences predicted to bind similar collections of alleles would be ranked similarly (Supplementary Table 4). The second list, referred to as the “disjoint” list, is constructed in an iterative fashion where the sequence with the greatest coverage is selected first, and then the coverage for the remaining epitopes is updated to nullify contributions from any alleles that have already been selected (Supplementary Table 5). This second list was used to generate <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>.</p></div><div id="sec-11" class="subsection"><h3>Identification of HLA-II Epitopes</h3><p id="p-22">To identify HLA-II epitopes, we used RECON’s HLA-II binding predictor to score all 12-20mer sequences in the SARS-CoV-2 proteome to predict both binding potential and the likely binding core within each 12-20mer. Scoring was performed across all supported HLA-II alleles that comprise 46 HLA-DR alleles, 17 HLA-DP alleles, and 20 HLA-DQ alleles (Supplementary Table 2).</p><p id="p-23">Peptide/allele pairs were assigned a percent rank by comparing their binding scores to those of 100,000 reference peptides. Pairs scoring in the top 1% were deemed likely to bind. Additionally, we define the “epitope” of a 12-20mer to be the predicted binding core within the sequence. As such, overlapping 12-20mers with the same predicted binding core for a given allele would constitute a single epitope. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows counts of these epitopes.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242423/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1242423/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1242423" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-24" class="first-child">Summary of the HLA-I and HLA-II epitopes predicted across the 12 SARS-CoV-2 ORFs and their validation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Additionally, we generated two lists of 25mers contained in SARS-CoV-2 protein sequences ranked by population coverage. To do this, we associated each 25mer with all subsequences that were likely binders and calculated the population coverage of the corresponding HLA-II alleles. Given a collection of alleles, we calculated the coverage as described in the previous section, the only difference being the cumulative product is taken across the following four HLA-II loci: HLA-DRB1, HLA-DRB3/4/5, HLA-DP, and HLA-DQ. HLA-II allele frequencies were obtained from (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>) and Allele Frequency Net Database (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>).</p><p id="p-26">As with HLA-I, two sorted lists of predicted binding sequences were generated – one sorted on absolute coverage (Supplementary Table 6), and one sorted on disjoint coverage (Supplementary Table 7), which was used to generate <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> and the observation that it would only require four 25mers to have predicted binders for &gt;99.9% of the USA, European, and API populations.</p></div><div id="sec-12" class="subsection"><h3>Comparison of predicted epitopes to the human proteome</h3><p id="p-27">8-12mer sequences (corresponding to predicted HLA-I epitopes), 9mer sequences (corresponding to predicted HLA-II binding cores), and 25mer sequences (corresponding to predicted HLA-II sequences that bound multiple alleles) from SARS-CoV-2 were compared against sub-sequences of the same length from the human proteome, using UCSC Genome Browser genes with hg19 annotation of the human genome and its protein coding transcripts (63,691 entries) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). Exact matches were identified and flagged in Supplementary Table 4. No exact matches were found for the predicted HLA-II binding cores or 25mer sequences.</p></div></div><div class="section" id="sec-13"><h2 class="">RESULTS</h2><div id="sec-14" class="subsection"><h3>Validating RECON prediction for viral epitopes using ViPR</h3><p id="p-28">We first sought to validate the ability of our predictors to identify epitopes from genomes of the <em>Coronaviridae</em> family. Since SARS-CoV-2 only emerged recently, specific data on SARS-CoV-2 peptide MHC-binding and immunogenic epitopes are currently limited. However, other viruses from the <em>Coronaviridae</em> family have been studied thoroughly, specifically MERS-CoV and SARS-CoV. The latter has significant sequence homology to SARS-CoV-2 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). We therefore sought to leverage previously tested epitopes from across the <em>Coronaviridae</em> family to validate our predictions of viral peptides, with special interest in peptide sequences that incidentally overlapped the novel SARS-CoV-2 virus. To that end, we used the publicly available ViPR database, which lists the results of T cell immunogenicity and MHC peptide-binding assays for both HLA-I and HLA-II alleles for viral pathogen epitopes. We used all assays of <em>Coronaviridae</em> family viruses with human hosts from ViPR as our validation dataset. Assays that did not have an associated four-digit HLA allele or were associated with an allele our models did not support were omitted (see Supplementary Tables 1 and 2 for a list of supported alleles).</p><p id="p-29">For HLA-I, within the validation dataset there were a total of 4,445 unique peptide-HLA allele pairs that were assayed for MHC-binding, using variations of: 1) cellular MHC or purified MHC; 2) a direct or competitive assay; and 3) measured by fluorescence or radioactivity. Two additional peptide-MHC allele pairs were confirmed via X-ray crystallography. Depending on the study from which the data was collected, peptide-MHC allele pairs were either binarily defined in ViPR as “Negative” and “Positive” for binding, or with a more granular scale of positivity: Low, Intermediate, and High. We assigned peptide-MHC allele pairs with multiple measurements with the highest MHC-binding detected across the replicates (see Methods).</p><p id="p-30">We then applied our HLA-I binding predictor from RECON to the peptide-MHC allele pairs in the validation dataset and compared the computed HLA-I percent ranks of these pairs with the reported MHC-binding assay results (Supplementary Table 8). A low percent rank value corresponds to high likelihood of binding (e.g., a peptide with a percent rank of 1% scores amongst the top 1% of the reference peptides). The percent ranks of peptide-MHC allele pairs that had a binary “Positive” result in the MHC-binding assay were significantly lower than pairs with a “Negative” result. Further, in the more granular positive results, stronger assay results (low &lt; intermediate &lt; high) were associated with increasingly lower percent ranks (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). In addition, the two peptide-MHC alleles that were confirmed by X-ray crystallography were predicted as very likely binders, with low percent rank scores of 0.07% and 0.30%. These results demonstrate that our HLA-I binding predictor from RECON can reliably predict the HLA-I binding of peptides from proteins of the <em>Coronaviridae</em> family, to which SARS-CoV-2 belongs.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RECON binding predictors percent rank for both peptide- HLA-I and HLA-II binding pairs from ViPR correlate with their MHC-binding assay results. A) The log10(percent rank) of a predicted peptide-HLA-I allele pair, versus the ViPR reported MHC-binding assay result (either binary Negative/Positive or the scaled Negative/Positive-Low/Positive-Intermediate/Positive-High. In total, there were 4,445 peptide-HLA-I allele pairs. B) The log10(percent rank) of a predicted peptide-HLA-II allele pair, versus the ViPR reported MHC-binding assay result (Negative+Positive-Low/Positive-Intermediate/Positive-High). In total, there were 259 peptide-HLA-II allele pairs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-190749434" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;RECON binding predictors percent rank for both peptide- HLA-I and HLA-II binding pairs from ViPR correlate with their MHC-binding assay results.&lt;/span&gt; &lt;strong&gt;A)&lt;/strong&gt; The log10(percent rank) of a predicted peptide-HLA-I allele pair, versus the ViPR reported MHC-binding assay result (either binary Negative/Positive or the scaled Negative/Positive-Low/Positive-Intermediate/Positive-High. In total, there were 4,445 peptide-HLA-I allele pairs. &lt;strong&gt;B)&lt;/strong&gt; The log10(percent rank) of a predicted peptide-HLA-II allele pair, versus the ViPR reported MHC-binding assay result (Negative&lt;sup&gt;+&lt;/sup&gt;Positive-Low/Positive-Intermediate/Positive-High). In total, there were 259 peptide-HLA-II allele pairs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F1.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F1.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">RECON binding predictors percent rank for both peptide- HLA-I and HLA-II binding pairs from ViPR correlate with their MHC-binding assay results.</span><p id="p-31" class="first-child"><strong>A)</strong> The log10(percent rank) of a predicted peptide-HLA-I allele pair, versus the ViPR reported MHC-binding assay result (either binary Negative/Positive or the scaled Negative/Positive-Low/Positive-Intermediate/Positive-High. In total, there were 4,445 peptide-HLA-I allele pairs. <strong>B)</strong> The log10(percent rank) of a predicted peptide-HLA-II allele pair, versus the ViPR reported MHC-binding assay result (Negative<sup>+</sup>Positive-Low/Positive-Intermediate/Positive-High). In total, there were 259 peptide-HLA-II allele pairs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Assays of T cell reactivity (e.g., interferon-gamma ELISpots, tetramers), which are stricter measures for T cell immunogenicity to epitopes, were performed in significantly lower numbers compared with MHC-binding assays. For HLA-I, the overlap between peptide-MHC allele pairs for which we had a prediction (supported alleles) and pairs with a reported T cell assay consisted of only 32 pairs, of which 23 had a positive result. We did not detect differences in the percent ranks across the positive and negative groups, however sample sizes are extremely small (data not shown). In addition, for HLA-I epitopes, the validation dataset only contained T cell assay results for peptide-MHC allele pairs that had a positive result in a binding assay, suggesting a biased pool of epitopes selected for testing.</p><p id="p-33">In addition to the identification of targets for CD8<sup>+</sup> T cells, we have recently demonstrated and incorporated into RECON the unprecedented ability to predict HLA-II binders (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">27</a>), allowing us to target CD4<sup>+</sup> T cell responses which could be harnessed for SARS-CoV-2 vaccines. These CD4<sup>+</sup> responses can potentially bolster both T cell immunity and enhance humoral immunity (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>).</p><p id="p-34">In a similar fashion to the HLA-I analysis, we scored all <em>Coronaviridae</em> family peptide-MHC allele pairs with supported HLA-II alleles in ViPR, using our HLA-II predictor (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">27</a>) (Supplementary Table 9). There were 259 unique peptide-MHC allele pairs assayed by MHC-binding assays in the ViPR validation dataset for HLA-II. As before, we compared their percent rank with their reported ‘best’ (in the case of multiple measurements) MHC-binding assay result. This comparison could not be performed with the “Negative” pairs as an independent group since there was only one negative result in the validation dataset for HLA-II. The low negative counts may be due to under-reporting of negative assay results or biased selection of the peptides to be assayed. Therefore, we merged the “Negative” and “Positive-Low” groups into one group and compared their percent ranks with either the “Positive-Intermediate” or the “Positive-High” groups (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). This analysis revealed a trend similar to that observed with HLA-I predictions, indicating that stronger MHC-binding assay results are associated with a lower predicted percent rank for HLA-II binders, as we expect for a robust predictor. Similar to the HLA-I T cell assays, there were too few recorded HLA-II T cell assays (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>) in our validation dataset to determine percent rank differences between peptide-HLA II allele pairs testing positive and negative. Together, these findings further corroborate the validity of our epitope predictors, as peptide-MHC allele pairs with positive results in binding assays consistently have lower percent ranks (better scores) by both our HLA-I and HLA-II MHC-binding predictors.</p></div><div id="sec-15" class="subsection"><h3>Epitope Prediction for SARS-CoV-2</h3><p id="p-35">We harnessed RECON’s HLA binding prediction ability to identify the peptides most relevant to the generation of SARS-CoV-2 T cell responses. We first performed the analysis for HLA-I peptide binding; we computed the likelihood of each peptide of lengths 8-12 amino-acid from the 12 SARS-CoV-2 ORFs to bind to any HLA-I allele in our database was computed. We then calculated the percent rank of each peptide-MHC allele pair by comparing their binding scores to those of a set of reference peptides, to generate a list of best-ranking peptide-MHC allele pairs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A-C</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A schematic demonstrating our approach of applying the HLA binding predictors from RECON to identify SARS-CoV-2 T cell epitopes and validate using the ViPR database. A) A diagram of the SARS-CoV-2 virus, listing example proteins. B) Applying our HLA-I and HLA-II binding predictors from RECON to the 12 ORFs of SARS-CoV-2. C) Both HLA-I and HLA-II epitopes are ranked by their likelihood to bind a particular HLA allele (listed sequences were selected randomly and do not represent significant epitopes). D) Epitopes shared between SARS-CoV-2 and other coronaviruses which were previously assayed are used for validation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-190749434" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;A schematic demonstrating our approach of applying the HLA binding predictors from RECON to identify SARS-CoV-2 T cell epitopes and validate using the ViPR database.&lt;/span&gt; &lt;strong&gt;A)&lt;/strong&gt; A diagram of the SARS-CoV-2 virus, listing example proteins. &lt;strong&gt;B)&lt;/strong&gt; Applying our HLA-I and HLA-II binding predictors from RECON to the 12 ORFs of SARS-CoV-2. &lt;strong&gt;C)&lt;/strong&gt; Both HLA-I and HLA-II epitopes are ranked by their likelihood to bind a particular HLA allele (listed sequences were selected randomly and do not represent significant epitopes). &lt;strong&gt;D)&lt;/strong&gt; Epitopes shared between SARS-CoV-2 and other coronaviruses which were previously assayed are used for validation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F2.medium.gif" width="440" height="126"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F2.medium.gif" width="440" height="126"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">A schematic demonstrating our approach of applying the HLA binding predictors from RECON to identify SARS-CoV-2 T cell epitopes and validate using the ViPR database.</span><p id="p-36" class="first-child"><strong>A)</strong> A diagram of the SARS-CoV-2 virus, listing example proteins. <strong>B)</strong> Applying our HLA-I and HLA-II binding predictors from RECON to the 12 ORFs of SARS-CoV-2. <strong>C)</strong> Both HLA-I and HLA-II epitopes are ranked by their likelihood to bind a particular HLA allele (listed sequences were selected randomly and do not represent significant epitopes). <strong>D)</strong> Epitopes shared between SARS-CoV-2 and other coronaviruses which were previously assayed are used for validation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">We detected a total of 11,776 unique SARS-CoV-2 peptides that were predicted to bind at least one HLA-I allele with a percent rank score of 1% or lower (Supplementary Table 4). 14 of these peptides overlapped with a subsequence of the human proteome (see methods, Supplementary Table 4).</p><p id="p-38">Unlike HLA-I, which has a closed binding groove that constrains bound peptide lengths to approximately 8 to 12 amino acids, peptides binding HLA-II have a wider length distribution (up to 30 amino acids or even longer) since the HLA-II binding groove is open at both ends. Peptides bind with a 9 amino acid subsequence (termed the binding core) occupying the HLA-II binding groove, with any flanking sequence overhanging the edges of the molecule. We consider a group of peptides that differ in the flanking regions but share a common binding core as a single epitope. Using the HLA-II predictor we identified 7,207 unique binding-cores that are predicted to bind at least one HLA-II allele with a percent rank score of 1% or lower. The number of high-quality peptide-MHC allele pairs we identify per SARS-CoV-2 gene is listed in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. The majority of predicted peptide-MHC allele pairs are from orf1a and orf1ab, primarily driven by the length of these ORFs. In addition, orf1a and orf1ab have very similar sequences, with over 18,000 identical binding peptide-HLA-I allele pairs predicted for both ORFs. We therefore opted to exclude redundant predictions and only reported unique pairs (see * in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Similarly, all HLA-II predicted epitopes from orf1a were covered by those reported for orf1ab.</p><p id="p-39">To test the validity of the SARS-CoV-2 predicted peptide-HLA pairs, we looked for identical peptide sequences in the <em>Coronaviridae</em> portion of the ViPR database (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>). A total of 368 HLA-I peptide-MHC allele pairs from SARS-CoV-2 had both a percent rank lower than 1% by our predictor and were found in the HLA-I MHC-binding validation dataset. Strikingly, of these HLA-I peptide-MHC allele pairs, 328 (89.1%) had a positive assay result. As a comparison, we also tested for overlap between epitopes predicted to have low likelihood of MHC-binding (percent rank 50% or higher) and the validation dataset. 37 peptide-MHC allele pairs overlapped between these sets, of which 36 (97.2%) had a negative assay result, as predicted. Further, we sought to determine whether our highly predicted SARS-CoV-2 peptide-HLA-I allele pairs (percent rank lower than 1%) would be validated by reported T cell assay results. Despite the significantly smaller number of peptide-MHC allele pairs that were tested for T cell reactivity in the validation dataset, 10 assayed pairs were also highly predicted by our HLA-I binding predictor. Nine out of these 10 (90%) predicted pairs had a positive result to the T cell assay. No low-scoring pairs (percent rank of 50% or above) were reported in the validation dataset. These findings demonstrate the validity of our prediction for peptide-HLA-I allele pairs for SARS-CoV-2 epitopes. Notably, while our algorithms are not trained on T cell reactivity data, and are aimed at peptide-MHC binding, for the few examples for which we had T cell reactivity assay results, we were able to show our highly-scoring peptide-MHC allele pairs are indeed immunogenic in the vast majority of cases.</p><p id="p-40">For HLA-II peptide-MHC allele pairs, only a single HLA-II peptide-MHC allele pair had both a percent rank lower than 1% and was reported in the validation dataset; this single pair (from the envelope protein) had a “Positive-High” assay result.</p></div><div id="sec-16" class="subsection"><h3>A small number of peptides predicted to bind multiple HLA-I and HLA-II alleles can provide broad population coverage</h3><p id="p-41">The concordance between the validation dataset and our highly predicted peptide-MHC allele pairs indicate that the HLA binding predictors from RECON significantly expand the list of predicted MHC binding peptides from the ORFs of SARS-CoV-2. We then sought to estimate the minimal number of HLA-I and HLA-II epitopes that would be required to provide coverage for the USA, European and Asian Pacific Islander populations based on the prevalence of MHC alleles in these populations (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>). We found that a subset of the peptides was predicted to bind a broad set of either HLA-I or HLA-II alleles. We determined that a vaccine containing three of these HLA-I and four HLA-II sequences could provide &gt;99.9% coverage for all of the USA, European, and Asian Pacific Islander populations, for HLA-I and HLA-II, respectively (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Under the assumption that all peptide-MHC allele pairs for which a given peptide scores in the top 1% are indeed immunogenic, this finding could facilitate the design of a parsimonious, broadly effective vaccine.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The USA, European and Asian Pacific Islander (API) populations could achieve &gt;99% coverage with a small number of prioritized multi-allele binding epitopes. A) Cumulative HLA-I coverage for each population versus the number of included prioritized HLA-I epitopes. B) Cumulative HLA-II coverage for each population versus the number of included prioritized HLA-II 25mers. C) The amino-acid sequences predicted to cover over 99.9% of the populations in A and B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-190749434" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The USA, European and Asian Pacific Islander (API) populations could achieve &gt;99% coverage with a small number of prioritized multi-allele binding epitopes.&lt;/span&gt; &lt;strong&gt;A)&lt;/strong&gt; Cumulative HLA-I coverage for each population versus the number of included prioritized HLA-I epitopes. &lt;strong&gt;B)&lt;/strong&gt; Cumulative HLA-II coverage for each population versus the number of included prioritized HLA-II 25mers. &lt;strong&gt;C)&lt;/strong&gt; The amino-acid sequences predicted to cover over 99.9% of the populations in A and B.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F3.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F3.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.027805/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">The USA, European and Asian Pacific Islander (API) populations could achieve &gt;99% coverage with a small number of prioritized multi-allele binding epitopes.</span><p id="p-42" class="first-child"><strong>A)</strong> Cumulative HLA-I coverage for each population versus the number of included prioritized HLA-I epitopes. <strong>B)</strong> Cumulative HLA-II coverage for each population versus the number of included prioritized HLA-II 25mers. <strong>C)</strong> The amino-acid sequences predicted to cover over 99.9% of the populations in A and B.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-17"><h2 class="">Discussion</h2><p id="p-43">In this work, we demonstrated the utility and validity of our HLA-I and HLA-II binding prediction algorithms to the <em>Coronaviridae</em> virus family, and specifically to SARS-CoV-2. By applying these algorithms to previously assayed peptide-MHC allele pairs in ViPR, we were able to show excellent concordance between our binding predictions and the results of the assays for both HLA-I and HLA-II epitopes. We leveraged the homology within the <em>Coronaviridae</em> family to demonstrate that an exceedingly high portion (~90%) of our high-ranking SARS-CoV-2 peptide-MHC allele pairs for which validation was available was indeed confirmed to bind the predicted MHC allele. Likewise, lowly-scoring peptide-MHC allele pairs derived from SARS-CoV-2 that had previously been assayed in ViPR were confirmed as non-binding. We therefore concluded that using RECON’s HLA binding predictors to predict T cell epitopes from the ORFs of SARS-CoV-2 provides a significantly expanded, novel set of high-quality vaccine targets for the virus. These sequences can be exploited for vaccines in various formats, including RNA or peptides.</p><p id="p-44">This application of our prediction algorithms has clearly identified many candidate epitopes that can be included in a vaccine to induce cellular responses against this novel virus. Immune analysis of patients infected with either SARS-CoV or MERS-CoV has identified critical antiviral roles for these cellular responses. Viral specific CD8<sup>+</sup> T cells can be cytotoxic and can kill virally infected cells to reduce disease severity. CD8<sup>+</sup> T cells account for about 80% of the total inflammatory cells in the pulmonary interstitium of SARS-CoV infected patients and play a vital role in eliminating virally infected cells (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>). In addition to having effector functions, CD4<sup>+</sup> T cells can promote the production of virus-specific antibodies by activating T-dependent B cells. With respect to humoral immunity, antibodies are seen primarily to the S and N proteins. Although short lived, antibody responses are essential to control the persistent phase of CoV infection by preventing subsequent viral entry. We thus propose that a combination of B and T cell epitopes could provide long-lasting immunity from SARS-CoV-2 or mitigate the severity of disease when protection is partial.</p><p id="p-45">The strength of our prediction is two-fold: first, we have validated predictors for both HLA-I and HLA-II binders, which potentially could be leveraged to induce both long-term CD4<sup>+</sup> and CD8<sup>+</sup> T cell immunity against the virus. Specifically, our HLA-II predictor, which has also been trained on a large set of mono-allelic mass spectrometry data, has been shown to significantly outperform previously published tools and is used here to identify high-quality CD4<sup>+</sup> epitopes that may contribute to both cellular and humoral immunity (<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">27</a>) (Supplementary Table 6). Second, our expansive database of supported HLA-I and HLA-II alleles provides us with the ability to not only identify many peptide-MHC allele pairs, but to generate a narrow list of peptides with many potential HLA pairings that could be presented by the entire USA, European and Asian Pacific Islander populations. These advantages significantly improve upon previously published findings (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>).</p><p id="p-46">Our algorithms predict peptide-MHC binding, which is necessary but not sufficient to induce a T cell response. Therefore, further experimental work would be needed to refine the list of peptides to strictly immunogenic ones. However, with the breadth of the list we are able to provide, the likelihood of identifying many such epitopes is high. In addition, while the availability of confirmed T cell reactions to SARS2-CoV-2 epitopes is limited, nine out of 10 highly ranking peptide-HLA-I allele pairs that were previously assayed had a positive result in a T cell assay.</p></div><div class="section" id="sec-18"><h2 class="">Conclusions</h2><p id="p-47">In summary, our work provides the most robust set of both CD4<sup>+</sup> and CD8<sup>+</sup> T cells epitopes that are spanning the entire SARS-CoV-2 genome and binding a wide set of HLA-I and HLA-II alleles. Our predicted list of T cell epitopes serves as a resource for the scientific community to generate potent SARS-CoV-2 vaccine epitopes and generate long-lasting T cell immunity. These epitopes are predicted to bind MHC alleles covering over 99.9% of the USA, European and Asian Pacific Islander populations and could complement B cell epitopes that have been shown to be effective but provide short-lived immunity. This expansive data set allows us to identify peptides predicted to bind many alleles and to propose a small set of peptides that are predicted cover over 99.9% of USA, European, and Asian populations and induce broad CD8<sup>+</sup> and CD4<sup>+</sup> immunity.</p></div><div class="section" id="sec-20"><h2 class="">Declarations</h2><div class="section" id="sec-21"><h3>Ethics approval and consent to participate</h3><p id="p-57">Not applicable.</p></div><div class="section" id="sec-22"><h3>Consent for publication</h3><p id="p-58">Not applicable.</p></div><div class="section" id="sec-23"><h3>Availability of data and materials</h3><p id="p-59">All data generated or analyzed during this study are included in this published article and its supplementary information files.</p></div><div class="section COI-statement" id="sec-24"><h3>Competing interests</h3><p id="p-60">AP, DH, MM, MSR, LS, and RBG are all current employees and shareholders of Neon Therapeutics, Inc.</p></div><div class="section" id="sec-25"><h3>Funding</h3><p id="p-61">Neon Therapeutics, Inc.</p></div><div class="section" id="sec-26"><h3>Authors’ contributions</h3><p id="p-62">A.P. – Conceptualization, Formal Analysis, Investigation, Visualization, Writing – Original Draft; D.H. – Conceptualization, Formal Analysis, Investigation, Visualization, Writing – Original Draft; M.M. – Conceptualization, Formal Analysis, Investigation, Visualization, Writing – Original Draft; M.S.R – Supervision, Writing – Review and Editing; L.S. – Supervision, conceptualization, Writing – Original Draft; R.B.G– Supervision, Funding Acquisition, Writing – Review and Editing; All authors read and approved the final manuscript.</p></div></div><div class="section" id="sec-27"><h2 class="">Additional files</h2><p id="p-64"><span class="underline">Supplementary Table 1: HLA-I Alleles Covered by Binding Predictors</span></p><p id="p-65">List of the 74 alleles covered by the HLA-I binding predictor.</p><p id="p-66">Filename: SuppTable_1_RECON_classi_supported_alleles.csv</p><p id="p-67"><span class="underline">Supplementary Table 2: HLA-II Alleles Covered by Binding Predictor</span></p><p id="p-68">List of the 83 alleles covered by the HLA-II binding predictor.</p><p id="p-69">Filename: SuppTable_2_RECON_classii_supported_alleles.csv</p><p id="p-70"><span class="underline">Supplementary Table 3: HLA-I and HLA-II Allele Population Frequencies</span></p><p id="p-71">HLA-I and HLA-II allele frequencies for USA, European (EUR), Asian Pacific Islander (API), African American (AFA), and Hispanic (HIS) populations. The USA population allele frequency is calculated as the following weighted average: 0.623*EUR+0.133*AFA+0.068*APA+0.176*HIS. For alleles where AFA and API population frequencies were not available, the USA population allele frequency values were set to match EUR. Missing API allele frequency values were imputed with 0 for our analyses.</p><p id="p-72">Filename: SuppTable_3_classi_classii_allele_frequencies.xlsx</p><p id="p-73"><span class="underline">Supplementary Table 4: Predicted HLA-I Binders Ranked by Population Coverage</span></p><p id="p-74">Table of all predicted HLA-I binders and their associated allele coverage. The table provides the peptide sequence, the SARS-CoV-2 protein(s) it is derived from, the HLA-I alleles it is predicted to bind to, the corresponding USA, European (EUR), and Asian Pacific Islander (API) population coverage, and a flag to indicate if the peptide sequence overlaps with any sequence in the human proteome.</p><p id="p-75">Filename: SuppTable_4_classi_ranked_by_coverage.csv</p><p id="p-76"><span class="underline">Supplementary Table 5: Broadly Binding HLA-I Peptides</span></p><p id="p-77">The 10 HLA-I predicted binders with the broadest cumulative allele coverage. The table provides the peptide sequence, its rank, the SARS-CoV-2 protein it is derived from, the alleles the peptide is predicted to bind to and the cumulative HLA-I coverage for USA, European (EUR), and Asian Pacific Islander (API) populations for all peptides up to this rank. See the <em>Identification of HLA-I Epitopes</em> section of the Methods for how coverage is calculated. Note that “surface glycoprotein” refers to the spike protein.</p><p id="p-78">Filename: SuppTable_5_classi_best_cumulative_coverage.csv</p><p id="p-79"><span class="underline">Supplementary Table 6: SARS-CoV-2 25mers Ranked by HLA-II Population Coverage</span></p><p id="p-80">Table of all SARS-CoV-2-derived 25mers containing at least 3 predicted HLA-II binders as subsequences. For each 25mer, the table provides the sequence, SARS-CoV-2 protein it is derived from, the alleles associated with the predicted binder subsequences, and their corresponding USA, European (EUR), and Asian Pacific Islander (API) population coverage.</p><p id="p-81">Filename: SuppTable_6_covid_25mers_ranked_by_coverage_AVG.csv</p><p id="p-82"><span class="underline">Supplementary Table 7: Broadly Binding HLA-II 25mers</span></p><p id="p-83">The 10 SARS-CoV-2-derived 25mers with the broadest cumulative predicted HLA-II allele coverage. For each 25mer, the table provides the rank, the peptide sequence, the SARS-CoV-2 protein it is derived from, the cumulative alleles that are covered by all 25mers up to this rank, and the associated USA, European (EUR), and Asian Pacific Islander (API) population coverage. Note that it is not the case that any of these 25mers, or their binding subsequences, are found as subsequences within the human proteome.</p><p id="p-84">Filename: SuppTable_7_covid_25mers_best_cumulative_coverage_AVG.csv</p><p id="p-85"><span class="underline">Supplementary Table 8: RECON binding prediction of ViPR HLA-I epitopes</span></p><p id="p-86">The peptide-HLA alleles pairs from the ViPR database which belong to the <em>Coronaviridae</em> family and have a human host had been scored using RECON’s HLA-I binding predictor. Alleles not reported in a four-digit format or not supported by RECON were excluded.</p><p id="p-87">Filename: SuppTable_8_ViPR_classi_percent-rank.csv</p><p id="p-88"><span class="underline">Supplementary Table 9: RECON binding prediction of ViPR HLA-II epitopes</span></p><p id="p-89">The peptide-HLA alleles pairs from the ViPR database which belong to the <em>Coronaviridae</em> family and have a human host had been scored using RECON’s HLA-II binding predictor. Filename: SuppTable_9_ViPR_classii_percent-rank.csv</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-63">Not applicable.</p></div><div class="section glossary"><h2 class="">List of Abbreviations</h2><dl id="def-list-1"><dt id="def-item-1">S<span class="def-item-dt-sep"></span></dt><dd>Human Leukocyte Antigen<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">MERS-CoV<span class="def-item-dt-sep"></span></dt><dd>Middle East Respiratory Syndrome – Coronavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">MHC<span class="def-item-dt-sep"></span></dt><dd>Major histocompatibility complex<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">RECON<span class="def-item-dt-sep"></span></dt><dd>Real-time Epitope Computation for ONcology<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">SARS-CoV<span class="def-item-dt-sep"></span></dt><dd>Severe Acute Respiratory Syndrome – Coronavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-6">SARS-CoV-2<span class="def-item-dt-sep"></span></dt><dd>Severe Acute Respiratory Syndrome – Coronavirus – 2<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">USA<span class="def-item-dt-sep"></span></dt><dd>United States of America<span class="def-item-dd-sep"></span></dd><dt id="def-item-8">ViPR<span class="def-item-dt-sep"></span></dt><dd>Virus Pathogen Resource<span class="def-item-dd-sep"></span></dd><dt id="def-item-9">WHO<span class="def-item-dt-sep"></span></dt><dd>World Health Organization</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.1" data-doi="10.1038/s41579-018-0118-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Z-L.</span></span> <span class="cit-article-title">Origin and evolution of pathogenic coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">17</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">181</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BMicrobiol%26rft.volume%253D17%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41579-018-0118-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41579-018-0118-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.2" data-doi="10.1084/jem.20050828"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiple organ infection and the pathogenesis of SARS</span>. <abbr class="cit-jnl-abbrev">J Exp Med</abbr>. <span class="cit-pub-date">2005</span> Aug 1;<span class="cit-vol">202</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">415</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1084%252Fjem.20050828%26rft_id%253Dinfo%253Apmid%252F16043521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamVtIjtzOjU6InJlc2lkIjtzOjk6IjIwMi8zLzQxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA4LzIwMjAuMDQuMDYuMDI3ODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.027805v1.3" data-doi="10.1126/science.1085952"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rota</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oberste</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monroe</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nix</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campagnoli</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Icenogle</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterization of a novel coronavirus associated with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2003</span> May 30;<span class="cit-vol">300</span>(<span class="cit-issue">5624</span>):<span class="cit-fpage">1394</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085952%26rft_id%253Dinfo%253Apmid%252F12730500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNC8xMzk0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDgvMjAyMC4wNC4wNi4wMjc4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.4" data-doi="10.1056/NEJMoa030781"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ksiazek</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erdman</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaki</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peret</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A novel coronavirus associated with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-pub-date">2003</span> May 15;<span class="cit-vol">348</span>(<span class="cit-issue">20</span>):<span class="cit-fpage">1953</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030781%26rft_id%253Dinfo%253Apmid%252F12690092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690092&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.06.027805v1.5"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span> <span class="cit-article-title">MERS-CoV Global Summary and Assessment of Risk</span>. <span class="cit-publ-name">World Health Organization</span>; <span class="cit-pub-date">2018</span>. <a href="https://www.who.int/csr/disease/coronavirus_infections/risk-assessment-august-2018.pdf">https://www.who.int/csr/disease/coronavirus_infections/risk-assessment-august-2018.pdf</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.06.027805v1.6"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO | Middle East respiratory syndrome coronavirus (MERS-CoV)</span>. <span class="cit-publ-name">World Health Organization</span>; [cited <span class="cit-date-in-citation">2020 Mar 25</span>]. <a href="http://www.who.int/emergencies/mers-cov/en/">http://www.who.int/emergencies/mers-cov/en/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.06.027805v1.7"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO | Update 49 - SARS case fatality ratio, incubation period</span>. <span class="cit-publ-name">World Health Organization</span>; [cited <span class="cit-date-in-citation">2020 Mar 25</span>]. <a href="https://www.who.int/csr/sars/archive/2003_05_07a/en/">https://www.who.int/csr/sars/archive/2003_05_07a/en/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.06.027805v1.8"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO | Coronavirus disease (COVID-19) Pandemic</span>. [cited <span class="cit-date-in-citation">2020 Mar 25</span>]. <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">https://www.who.int/emergencies/diseases/novel-coronavirus-2019</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.9" data-doi="10.1080/22221751.2020.1719902"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">JF-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kok</span>  <span class="cit-name-given-names">K-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">To</span>  <span class="cit-name-given-names">KK-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr>. <span class="cit-pub-date">2020</span> Jan 28;<span class="cit-vol">9</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">221</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2020.1719902%26rft_id%253Dinfo%253Apmid%252F31987001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/22221751.2020.1719902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31987001&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.10" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X-G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-vol">579</span>(<span class="cit-issue">7798</span>):<span class="cit-fpage">270</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.11" data-doi="10.1016/S0140-6736(20)302518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>. <span class="cit-pub-date">2020</span> 22;<span class="cit-vol">395</span>(<span class="cit-issue">10224</span>):<span class="cit-fpage">565</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%2529302518%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)302518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.12" data-doi="10.1002/jmv.25685"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Coronavirus infections and immune responses</span>. <abbr class="cit-jnl-abbrev">J Med Virol</abbr>. <span class="cit-pub-date">2020</span>;<span class="cit-vol">92</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">424</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BVirol%26rft.volume%253D92%26rft.spage%253D424%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.25685%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.25685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.13" data-doi="10.1038/nature02463"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z-Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">W-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subbarao</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A DNA vaccine induces SARS coronavirus neutralization and protective immunity in mice</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2004</span> Apr 1;<span class="cit-vol">428</span>(<span class="cit-issue">6982</span>):<span class="cit-fpage">561</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYang%26rft.auinit1%253DZ.%2BY.%26rft.volume%253D428%26rft.issue%253D6982%26rft.spage%253D561%26rft.epage%253D564%26rft.atitle%253DA%2BDNA%2Bvaccine%2Binduces%2BSARS%2Bcoronavirus%2Bneutralization%2Band%2Bprotective%2Bimmunity%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02463%26rft_id%253Dinfo%253Apmid%252F15024391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15024391&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.027805v1.14" data-doi="10.1038/nm.2972"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bolles</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-article-title">A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</span>. <abbr class="cit-jnl-abbrev">Nat Med</abbr>. <span class="cit-pub-date">2012</span> Dec;<span class="cit-vol">18</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1820</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMed%26rft.volume%253D18%26rft.spage%253D1820%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.2972%26rft_id%253Dinfo%253Apmid%252F23142821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.2972&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23142821&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.027805v1.15" data-doi="10.1373/clinchem.2003.023184"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Assessment of immunoreactive synthetic peptides from the structural proteins of severe acute respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">Clin Chem</abbr>. <span class="cit-pub-date">2003</span> Dec;<span class="cit-vol">49</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1989</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1373%252Fclinchem.2003.023184%26rft_id%253Dinfo%253Apmid%252F14633869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiY2xpbmNoZW0iO3M6NToicmVzaWQiO3M6MTA6IjQ5LzEyLzE5ODkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOC8yMDIwLjA0LjA2LjAyNzgwNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.16" data-doi="10.1128/CDLI.11.1.227-228.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Profile of antibodies to the nucleocapsid protein of the severe acute respiratory syndrome (SARS)-associated coronavirus in probable SARS patients</span>. <abbr class="cit-jnl-abbrev">Clin Diagn Lab Immunol</abbr>. <span class="cit-pub-date">2004</span> Jan;<span class="cit-vol">11</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">227</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BDiagn%2BLab%2BImmunol%26rft.volume%253D11%26rft.spage%253D227%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCDLI.11.1.227-228.2004%26rft_id%253Dinfo%253Apmid%252F14715574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/CDLI.11.1.227-228.2004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14715574&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">Z-T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrammert</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">M-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Lack of Peripheral Memory B Cell Responses in Recovered Patients with Severe Acute Respiratory Syndrome: A Six-Year Follow-Up Study</span>. <abbr class="cit-jnl-abbrev">The Journal of Immunology</abbr>. <span class="cit-pub-date">2011</span> Jun 15;<span class="cit-vol">186</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">7264</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.volume%253D186%26rft.spage%253D7264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.18" data-doi="10.1086/378587"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Expression of lymphocytes and lymphocyte subsets in patients with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis</abbr>. <span class="cit-pub-date">2003</span> Sep 15;<span class="cit-vol">37</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">857</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1086%252F378587%26rft_id%253Dinfo%253Apmid%252F12955652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/378587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12955652&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185131100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.19" data-doi="10.1086/381535"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Significant changes of peripheral T lymphocyte subsets in patients with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr>. <span class="cit-pub-date">2004</span> Feb 15;<span class="cit-vol">189</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">648</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1086%252F381535%26rft_id%253Dinfo%253Apmid%252F14767818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/381535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14767818&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188832900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Channappanavar</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">T-cell-mediated immune response to respiratory coronaviruses</span>. <abbr class="cit-jnl-abbrev">Immunol Res</abbr>. <span class="cit-pub-date">2014</span> Aug;<span class="cit-vol">59</span>(<span class="cit-issue">0</span>):<span class="cit-fpage">118</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunol%2BRes%26rft.volume%253D59%26rft.spage%253D118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.21" data-doi="10.1128/JVI.01505-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Channappanavar</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fett</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyerholz</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Virus-specific memory CD8 T cells provide substantial protection from lethal severe acute respiratory syndrome coronavirus infection</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2014</span> Oct;<span class="cit-vol">88</span>(<span class="cit-issue">19</span>):<span class="cit-fpage">11034</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01505-14%26rft_id%253Dinfo%253Apmid%252F25056892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8xOS8xMTAzNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA4LzIwMjAuMDQuMDYuMDI3ODA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">CK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">T Cell Responses to Whole SARS Coronavirus in Humans</span>. <abbr class="cit-jnl-abbrev">The Journal of Immunology</abbr>. <span class="cit-pub-date">2008</span> Oct 15;<span class="cit-vol">181</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">5490</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.volume%253D181%26rft.spage%253D5490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thevarajan</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">THO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koutsakos</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Druce</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caly</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">van de Sandt</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Breadth of concomitant immune responses prior to patient recovery: a case report of non-severe COVID-19</span>. <abbr class="cit-jnl-abbrev">Nature Medicine</abbr>. <span class="cit-pub-date">2020</span> Mar 16;<span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">O-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jadi</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">HN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertoletti</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Memory T cell responses targeting the SARS coronavirus persist up to 11 years post-infection</span>. <abbr class="cit-jnl-abbrev">Vaccine</abbr>. <span class="cit-pub-date">2016</span> Apr 12;<span class="cit-vol">34</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">2008</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D34%26rft.spage%253D2008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.25" data-doi="10.1128/JVI.01049-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">T Cell Responses Are Required for Protection from Clinical Disease and for Virus Clearance in Severe Acute Respiratory Syndrome Coronavirus-Infected Mice</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2010</span> Sep;<span class="cit-vol">84</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">9318</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01049-10%26rft_id%253Dinfo%253Apmid%252F20610717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xOC85MzE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDgvMjAyMC4wNC4wNi4wMjc4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.26" data-doi="10.1016/j.immuni.2017.02.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abelin</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keskin</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarkizova</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartigan</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidney</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Mass Spectrometry Profiling of HLA-Associated Peptidomes in Mono-allelic Cells Enables More Accurate Epitope Prediction</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr>. <span class="cit-pub-date">2017</span> 21;<span class="cit-vol">46</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">315</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D46%26rft.spage%253D315%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2017.02.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2017.02.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abelin</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harjanto</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malloy</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suri</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colson</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goulding</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Defining HLA-II Ligand Processing and Binding Rules with Mass Spectrometry Enhances Cancer Epitope Prediction</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr>. <span class="cit-pub-date">2019</span> 15;<span class="cit-vol">51</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">766</span>–<span class="cit-lpage">779</span>.e17.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D51%26rft.spage%253D766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Archila</span>  <span class="cit-name-given-names">LLD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwok</span>  <span class="cit-name-given-names">WW.</span></span> <span class="cit-article-title">Tetramer-Guided Epitope Mapping: A Rapid Approach to Identify HLA-Restricted T-Cell Epitopes from Composite Allergens</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">1592</span>:<span class="cit-fpage">199</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1592%26rft.spage%253D199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.29" data-doi="10.1016/j.clim.2006.03.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huston</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Danke</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwok</span>  <span class="cit-name-given-names">WW.</span></span> <span class="cit-article-title">Multiplex mapping of CD4 T cell epitopes using class II tetramers</span>. <abbr class="cit-jnl-abbrev">Clin Immunol</abbr>. <span class="cit-pub-date">2006</span> Jul;<span class="cit-vol">120</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">21</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bimmunology%2B%2528Orlando%252C%2BFla.%2529%26rft.stitle%253DClin%2BImmunol%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D32%26rft.atitle%253DMultiplex%2Bmapping%2Bof%2BCD4%2BT%2Bcell%2Bepitopes%2Busing%2Bclass%2BII%2Btetramers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.clim.2006.03.008%26rft_id%253Dinfo%253Apmid%252F16677863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.clim.2006.03.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16677863&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238632900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.30" data-doi="10.1093/nar/gkr859"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pickett</span>  <span class="cit-name-given-names">BE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sadat</span>  <span class="cit-name-given-names">EL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noronha</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Squires</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">ViPR: an open bioinformatics database and analysis resource for virology research</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2012</span> Jan;<span class="cit-vol">40</span>(Database issue):<span class="cit-fpage">D593</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr859%26rft_id%253Dinfo%253Apmid%252F22006842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22006842&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300089&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grifoni</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidney</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheuermann</span>  <span class="cit-name-given-names">RH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sette</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>. <span class="cit-pub-date">2020</span> Mar 12;</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.32" data-doi="10.1016/j.humimm.2007.04.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maiers</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gragert</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klitz</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">High-resolution HLA alleles and haplotypes in the United States population</span>. <abbr class="cit-jnl-abbrev">Hum Immunol</abbr>. <span class="cit-pub-date">2007</span> Sep;<span class="cit-vol">68</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">779</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bimmunology%26rft.stitle%253DHum%2BImmunol%26rft.aulast%253DMaiers%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D9%26rft.spage%253D779%26rft.epage%253D788%26rft.atitle%253DHigh-resolution%2BHLA%2Balleles%2Band%2Bhaplotypes%2Bin%2Bthe%2BUnited%2BStates%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.humimm.2007.04.005%26rft_id%253Dinfo%253Apmid%252F17869653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.humimm.2007.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17869653&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249751300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.33" data-doi="10.1093/nar/gku1166"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">González-Galarza</span>  <span class="cit-name-given-names">FF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takeshita</span>  <span class="cit-name-given-names">LYC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">EJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kempson</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maia</span>  <span class="cit-name-given-names">MHT</span></span>, <span class="cit-auth"><span class="cit-name-surname">da Silva</span>  <span class="cit-name-given-names">ALS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Allele frequency net 2015 update: new features for HLA epitopes, KIR and disease and HLA adverse drug reaction associations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2015</span> Jan;<span class="cit-vol">43</span>(Database issue):<span class="cit-fpage">D784</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1166%26rft_id%253Dinfo%253Apmid%252F25414323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25414323&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.027805.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.34" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sugnet</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furey</span>  <span class="cit-name-given-names">TS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roskin</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zahler</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The human genome browser at UCSC</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-pub-date">2002</span> Jun;<span class="cit-vol">12</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">996</span>–<span class="cit-lpage">1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDgvMjAyMC4wNC4wNi4wMjc4MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quadeer</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">MR.</span></span> <span class="cit-article-title">Preliminary Identification of Potential Vaccine Targets for the COVID-19 Coronavirus (SARS-CoV-2) Based on SARS-CoV Immunological Studies</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-pub-date">2020</span> 25;<span class="cit-vol">12</span>(<span class="cit-issue">3</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027805v1.36" data-doi="10.1016/j.immuni.2016.05.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mangalam</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Channappanavar</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fett</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyerholz</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Airway Memory CD4+ T Cells Mediate Protective Immunity against Emerging Respiratory Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr>. <span class="cit-pub-date">2016</span> Jun 21;<span class="cit-vol">44</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1379</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D44%26rft.spage%253D1379%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2016.05.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2016.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.06.027805v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.06.027805v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.07.030478v1" title="Application of singe nuclei RNA sequencing to assess the hepatic effects of 2,3,7,8-tetrachlorodibenzo-p-dioxin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.07.028258v1" title="Reduced somatostatin signalling leads to hypersecretion of glucagon in mice fed a high fat diet" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 08, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.06.027805v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.04.06.027805v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.06.027805v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.06.027805v1" class="active" data-icon-position="" data-hide-link-title="0">Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1235209" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_xnGv9UVJi2Je71AGAfkcno82M2Gzdnk9YJi0BqUIT0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1235209" id="node1235209--2" data-pisa="biorxiv;2020.04.06.027805v1" data-pisa-master="biorxiv;2020.04.06.027805" data-seqnum="1235209" data-apath="/biorxiv/early/2020/04/08/2020.04.06.027805.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Asaf</span> <span class="nlm-surname">Poran</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dewi</span> <span class="nlm-surname">Harjanto</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Matthew</span> <span class="nlm-surname">Malloy</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Michael S.</span> <span class="nlm-surname">Rooney</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lakshmi</span> <span class="nlm-surname">Srinivasan</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Richard B.</span> <span class="nlm-surname">Gaynor</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.06.027805; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.06.027805 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.06.027805v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;title=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;title=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;text=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;title=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;t=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;title=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;title=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1235209" id="node1235209--3" data-pisa="biorxiv;2020.04.06.027805v1" data-pisa-master="biorxiv;2020.04.06.027805" data-seqnum="1235209" data-apath="/biorxiv/early/2020/04/08/2020.04.06.027805.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Sequence-based prediction of vaccine targets for inducing T cell responses to SARS-CoV-2 utilizing the bioinformatics predictor RECON</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Asaf</span> <span class="nlm-surname">Poran</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dewi</span> <span class="nlm-surname">Harjanto</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Matthew</span> <span class="nlm-surname">Malloy</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Michael S.</span> <span class="nlm-surname">Rooney</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lakshmi</span> <span class="nlm-surname">Srinivasan</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Richard B.</span> <span class="nlm-surname">Gaynor</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.06.027805; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.06.027805 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Poran%2C+Asaf&amp;rft.au=Harjanto%2C+Dewi&amp;rft.au=Malloy%2C+Matthew&amp;rft.au=Rooney%2C+Michael+S.&amp;rft.au=Srinivasan%2C+Lakshmi&amp;rft.au=Gaynor%2C+Richard+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1235209/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1235209/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1235209/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1235209/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1235209/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1235209/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1235209/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1235209/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1235209/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1235209/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1235209/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1235209/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;count=horizontal&amp;via=&amp;text=Sequence-based%20prediction%20of%20vaccine%20targets%20for%20inducing%20T%20cell%20responses%20to%20SARS-CoV-2%20utilizing%20the%20bioinformatics%20predictor%20RECON&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027805v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.06.027805v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
