<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2016/06/21/048801.atom" />
<meta name="HW.pisa" content="biorxiv;048801v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension" />
<meta name="DC.Identifier" content="10.1101/048801" />
<meta name="DC.Date" content="2016-06-21" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces ( f s) and monovalent salt concentrations ( C s). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C , we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log C m) α  with α = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation." />
<meta name="DC.Contributor" content="Naoto Hori" />
<meta name="DC.Contributor" content="Natalia A. Denesyuk" />
<meta name="DC.Contributor" content="D. Thirumalai" />
<meta name="article:published_time" content="2016-06-21" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (&lt;i&gt;f&lt;/i&gt;s) and monovalent salt concentrations (&lt;i&gt;C&lt;/i&gt;s). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying &lt;i&gt;f&lt;/i&gt; and &lt;i&gt;C&lt;/i&gt;, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As &lt;i&gt;f&lt;/i&gt; and &lt;i&gt;C&lt;/i&gt; are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log &lt;i&gt;C&lt;/i&gt;&lt;sub&gt;m&lt;/sub&gt;)&lt;sup&gt;&lt;i&gt;α&lt;/i&gt;&lt;/sup&gt; with &lt;i&gt;α&lt;/i&gt; = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="biorxiv;048801v2" />
<meta name="citation_id" content="048801v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/048801v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/048801v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/048801v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801.full.pdf" />
<meta name="citation_doi" content="10.1101/048801" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="048801" />
<meta name="citation_author" content="Naoto Hori" />
<meta name="citation_author_institution" content="Institute for Physical Science and Technology, University of Maryland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0658-8266" />
<meta name="citation_author" content="Natalia A. Denesyuk" />
<meta name="citation_author_institution" content="Institute for Physical Science and Technology, University of Maryland" />
<meta name="citation_author" content="D. Thirumalai" />
<meta name="citation_author_institution" content="Institute for Physical Science and Technology, University of Maryland" />
<meta name="citation_reference" content="Cech, T. R.; Steitz, J. A. The noncoding RNA revolution—trashing old rules to forge new ones. Cell 2014, 157, 77–94, DOI: 10.1016/j.cell.2014.03.008." />
<meta name="citation_reference" content="Tinoco, I.; Bustamante, C. How RNA folds. J. Mol. Biol. 1999, 293, 271–281, DOI: 10.1006/jmbi.1999.3001." />
<meta name="citation_reference" content="Woodson, S. A. Structure and assembly of group I in-trons. Curr. Opin. Struct. Biol. 2005, 15, 324–330, DOI: 10.1016/j.sbi.2005.05.007." />
<meta name="citation_reference" content="Thirumalai, D.; Hyeon, C. RNA and protein folding: common themes and variations. Biochemistry 2005, 44, 4957–70, DOI: 10.1021/bi047314." />
<meta name="citation_reference" content="Chen, S.-J. RNA folding conformational statistics, folding kinetics, and ion electrostatics. Annu. Rev. Biophys. 2008, 37, 197, DOI: 10.1146/an-nurev.biophys.37.032807.125957." />
<meta name="citation_reference" content="Woodson, S. A. RNA folding pathways and the selfassembly of ribosomes. Acc. Chem. Res. 2011, 44, 1312–1319, DOI: 10.1021/ar2000474." />
<meta name="citation_reference" content="Tinoco, I.; TX Li, P.; Bustamante, C. Determination of thermodynamics and kinetics of RNA reactions by force. Q. Rev. Biophys. 2006, 39, 325–360, DOI: 10.1017/S0033583506004446." />
<meta name="citation_reference" content="Theimer, C. A.; Feigon, J. Structure and function of telomerase RNA. Curr. Opin. Struct. Biol. 2006, 16, 307–18, DOI: 10.1016/j.sbi.2006.05.005." />
<meta name="citation_reference" content="Brierley, I.; Digard, P.; Inglis, S. C. Characterization of an efficient coronavirus ribosomal frameshifting signal: requirement for an RNA pseudoknot. Cell 1989, 57, 537–547, DOI: 10.1016/0092-8674(89)90124-4." />
<meta name="citation_reference" content="Powers, T.; Noller, H. F. A functional pseudoknot in 16S ribosomal RNA. EMBO J. 1991, 10, 2203." />
<meta name="citation_reference" content="ten Dam, E.; Pleij, K.; Draper, D. Structural and functional aspects of RNA pseudoknots. Biochemistry 1992, 31, 11665–11676, DOI: 10.1021/bi00162a001." />
<meta name="citation_reference" content="Staple, D. W.; Butcher, S. E. Pseudoknots: RNA structures with diverse functions. PLoS Biol. 2005, 3, e213, DOI: 10.1371/journal.pbio.0030213." />
<meta name="citation_reference" content="Gilley, D.; Blackburn, E. H. The telomerase RNA pseudoknot is critical for the stable assembly of a catalytically active ribonucleoprotein. Proc. Natl. Acad. Sci. USA 1999, 96, 6621–6625, DOI: 10.1073/pnas.96.12.6621." />
<meta name="citation_reference" content="Green, L.; Kim, C.-H.; Bustamante, C.; Tinoco, I. Characterization of the mechanical unfolding of RNA pseudoknots. J. Mol. Biol. 2008, 375, 511–528, DOI: 10.1016/j.jmb.2007.05.058." />
<meta name="citation_reference" content="Ritchie, D. B.; Foster, D. A. N.; Woodside, M. T. Programmed —1 frameshifting efficiency correlates with RNA pseudoknot conformational plasticity, not resistance to mechanical unfolding. Proc. Natl. Acad. Sci. USA 2012, 109, 16167–16172, DOI:10.1073/pnas.1204114109." />
<meta name="citation_reference" content="Kim, H.-K.; Liu, F.; Fei, J.; Bustamante' C.; Gonzalez, R. L.; Tinoco, I. A frameshifting stimulatory stem loop destabilizes the hybrid state and impedes riboso-mal translocation. Proc. Natl. Acad. Sci. USA 2014, 111, 5538–5543, DOI: 10.1073/pnas.1403457111." />
<meta name="citation_reference" content="Chen, G.; Wen, J.-D.; Tinoco, I. Single-molecule mechanical unfolding and folding of a pseudoknot in human telomerase RNA. RNA 2007, 13, 2175–2188, DOI: 10.1261/rna.676707." />
<meta name="citation_reference" content="Giedroc, D. P.; Cornish, P. V. Frameshifting RNA pseudoknots: structure and mechanism. Virus Res. 2009, 139, 193–208, DOI: 10.1016/j.virusres.2008.06.008." />
<meta name="citation_reference" content="White, K. H.; Orzechowski, M.; Fourmy, D.; Visscher, K. Mechanical unfolding of the beet western yellow virus -1 frameshift signal. J. Am. Chem. Soc. 2011, 133, 9775–9782, DOI: 10.1021/ja111281f." />
<meta name="citation_reference" content="Dinman, J. D. Mechanisms and implications of programmed translational frameshifting. WIREs RNA 2012, 3, 661–673, DOI: 10.1002/wrna.1126." />
<meta name="citation_reference" content="Chen, G.; Chang, K.-Y.; Chou, M.-Y.; Bustamante, C.; Tinoco, I. Triplex structures in an RNA pseudoknot enhance mechanical stability and increase efficiency of -1 ri-bosomal frameshifting. Proc. Natl. Acad. Sci. USA 2009, 106, 12706–11, DOI: 10.1073/pnas.0905046106." />
<meta name="citation_reference" content="de Messieres, M.; Chang, J.-C.; Belew, A. T.; Meskauskas, A.; Dinman, J. D.; La Porta, A. Singlemolecule measurements of the CCR5 mRNA unfolding pathways. Biophys. J. 2014, 106, 244–252, DOI: 10.1016/j.bpj.2013.09.036." />
<meta name="citation_reference" content="Napthine, S.; Liphardt, J.; Bloys, A.; Routledge, S.; Brierley, I. The role of RNA pseudoknot stem 1 length in the promotion of efficient -1 ribosomal frameshifting. J. Mol. Biol. 1999, 288, 305–320, DOI: 10.1006/jmbi.1999.2688." />
<meta name="citation_reference" content="Kontos, H.; Napthine, S.; Brierley, I. Ribosomal pausing at a frameshifter RNA pseudoknot is sensitive to reading phase but shows little correlation with frameshift efficiency. Mol. Cell Biol. 2001, 21, 8657–8670, DOI: 10.1128/MCB.21.24.8657-8670.2001." />
<meta name="citation_reference" content="Kim, Y.-G.; Su, L.; Maas, S.; O’Neill, A.; Rich, A. Specific mutations in a viral RNA pseudoknot drastically change ribosomal frameshifting efficiency. Proc. Natl. Acad. Sci. USA 1999, 96, 14234–14239, DOI: 10.1073/pnas.96.25.14234." />
<meta name="citation_reference" content="Mouzakis, K. D.; Lang, A. L.; Vander Meulen, K. A.; Easterday, P. D.; Butcher, S. E. HIV-1 frameshift efficiency is primarily determined by the stability of base pairs positioned at the mRNA entrance channel of the ribosome. Nucleic Acids Res. 2013, 41, 1901–1913, DOI: 10.1093/nar/gks1254." />
<meta name="citation_reference" content="Belew, A. T.; Meskauskas, A.; Musalgaonkar, S.; Advani, V. M.; Sulima, S. O.; Kasprzak, W. K.; Shapiro, B. A.; Dinman, J. D. Ribosomal frameshifting in the CCR5 mRNA is regulated by miRNAs and the NMD pathway. Nature 2014, 512, 265–269, DOI: 10.1038/na-ture13429." />
<meta name="citation_reference" content="Li, Y.; Treffers, E. E.; Napthine, S.; Tas, A.; Zhu, L.; Sun, Z.; Bell, S.; Mark, B. L.; van Veelen, P. A.; van Hemert, M. J.; Firthe, A. E.; Brierleye, I.; Snijderc, E. J.; Fang, Y. Transactivation of programmed ribosomal frameshifting by a viral protein. Proc. Natl. Acad. Sci. USA 2014, 111, E2172–E2181, DOI: 10.1073/pnas.1321930111." />
<meta name="citation_reference" content="Tionoco, I.; Bustamante, C. The effect of force on thermodynamics and kinetics of single molecule reactions. Biophys. Chem. 2002, 101, 513–533, DOI: 10.1016/S0301-4622(02)00177-1." />
<meta name="citation_reference" content="Nixon, P. L.; Giedroc, D. P. Energetics of a strongly pH dependent RNA tertiary structure in a frameshifting pseudoknot. J. Mol. Biol. 2000, 296, 659–71, DOI: 10.1006/jmbi.1999.3464." />
<meta name="citation_reference" content="Soto, A. M.; Misra, V.; Draper, D. E. Tertiary structure of an RNA pseudoknot is stabilized by “diffuse” Mg2+ ions. Biochemistry 2007, 46, 2973–83, DOI: 10.1021/bi0616753." />
<meta name="citation_reference" content="Su, L.; Chen, L.; Egli, M.; Berger, J. M.; Rich, A. Minor groove RNA triplex in the crystal structure of a ribosomal frameshifting viral pseudoknot. Nat. Struct. Mol. Biol. 1999, 6, 285–292, DOI: 10.1038/6722." />
<meta name="citation_reference" content="Tan, Z.-J.; Chen, S.-J. Salt contribution to RNA tertiary structure folding stability. Biophys. J. 2011, 101, 176–87, DOI: 10.1016/j.bpj.2011.05.050." />
<meta name="citation_reference" content="Hyeon, C.; Thirumalai, D. Mechanical unfolding of RNA hairpins. Proc. Natl. Acad. Sci. USA 2005, 102, 6789–94, DOI: 10.1073/pnas.0408314102." />
<meta name="citation_reference" content="Anderson, C. F.; Record Jr, M. T. Salt dependence of oligoion-polyion binding: a thermodynamic description based on preferential interaction coefficients. J. Phys. Chem. 1993, 97, 7116–7126, DOI: 10.1021/j100129a032." />
<meta name="citation_reference" content="Liu, T.; Kaplan, A.; Alexander, L.; Yan, S.; Wen, J.-D.; Lancaster, L.; Wickersham, C. E.; Fredrick, K.; Noller, H.; Tinoco, I.; Bustamante, C. J. Direct measurement of the mechanical work during translocation by the ribosome. eLife 2014, 3, e03406, DOI: 10.7554/eLife.03406." />
<meta name="citation_reference" content="Denesyuk, N. A.; Thirumalai, D. Coarse-grained model for predicting RNA folding thermodynamics. J. Phys. Chem. B 2013, 117, 4901–11, DOI: 10.1021/jp401087x." />
<meta name="citation_reference" content="Cho, S. S.; Pincus, D. L.; Thirumalai, D. Assembly mechanisms of RNA pseudoknots are determined by the stabilities of constituent secondary structures. Proc. Natl. Acad. Sci. USA 2009, 106, 17349–54, DOI: 10.1073/pnas.0906625106." />
<meta name="citation_reference" content="Todd, B. A.; Rau, D. C. Interplay of ion binding and attraction in DNA condensed by multivalent cations. Nucleic Acids Res. 2007, 36, 501–510, DOI: 10.1093/nar/gkm1038." />
<meta name="citation_reference" content="Zhang, H.; Marko, J. F. Maxwell relations for single-DNA experiments: Monitoring protein binding and double-helix torque with force-extension measurements. Phys. Rev. E 2008, 77, 031916–9, DOI: 10.1103/Phys-RevE.77.031916." />
<meta name="citation_reference" content="Dittmore, A.; Landy, J.; Molzon, A. A.; Saleh, O. A. Single-molecule methods for ligand counting: Linking ion uptake to DNA hairpin folding. J. Am. Chem. Soc. 2014, 136, 5974–5980, DOI: 10.1021/ja500094z." />
<meta name="citation_reference" content="Jacobson, D. R.; Saleh, O. A. Measuring the differential stoichiometry and energetics of ligand binding to macromolecules by single-molecule force spectroscopy: an extended theory. J. Phys. Chem. B 2015, 119, 1930–1938, DOI: 10.1021/jp511555g." />
<meta name="citation_reference" content="Jacobson, D. R.; Saleh, O. A. Quantifying the ion atmosphere of unfolded, single-stranded nucleic acids using equilibrium dialysis and single-molecule methods. Nucleic Acids Res. 2016, 44, 3763–3771, DOI: 10.1093/nar/gkw196." />
<meta name="citation_reference" content="Bond, J. P.; Anderson, C. F.; Record Jr, M. T. Conformational transitions of duplex and triplex nucleic acid helices: thermodynamic analysis of effects of salt concentration on stability using preferential interaction coefficients. Biophys. J. 1994, 67, 825, DOI: 10.1016/S0006-3495(94)80542-9." />
<meta name="citation_reference" content="Record Jr, M. T.; Zhang, W.; Anderson, C. F. Analysis of effects of salts and uncharged solutes on protein and nucleic acid equilibria and processes: a practical guide to recognizing and interpreting polyelectrolyte effects, Hofmeister effects, and osmotic effects of salts. Adv. Protein Chem 1998, 51, 281–353." />
<meta name="citation_reference" content="Record Jr, M. T.; Anderson, C. F.; Lohman, T. M. Thermodynamic analysis of ion effects on the binding and conformational equilibria of proteins and nucleic acids: the roles of ion association or release, screening, and ion effects on water activity. Q. Rev. Biophys. 1978, 11, 103–178, DOI: 10.1017/S003358350000202X." />
<meta name="citation_reference" content="Biyun, S.; Cho, S. S.; Thirumalai, D. Folding of human telomerase RNA pseudoknot using ion-jump and temperature-quench simulations. J. Am. Chem. Soc. 2011, 133, 20634–43, DOI: 10.1021/ja2092823." />
<meta name="citation_reference" content="Lin, J.-C.; Thirumalai, D. Kinetics of allosteric transitions in S-adenosylmethionine riboswitch are accurately predicted from the folding landscape. J. Am. Chem. Soc. 2013, 135, 16641–50, DOI: 10.1021/ja408595e." />
<meta name="citation_reference" content="Denesyuk, N. A.; Thirumalai, D. How do metal ions direct ribozyme folding? Nature Chem. 2015, 7, 793–801, DOI: 10.1038/nchem.2330." />
<meta name="citation_reference" content="Malmberg, C. G.; Maryott, A. A. Dielectric constant of water from 0° to 100° C. J. Res. Nat. Bur. Stand. 1956, 56." />
<meta name="citation_reference" content="Heilman-Miller, S. L.; Thirumalai, D.; Woodson, S. A. Role of counterion condensation in folding of the Tetrahymena ribozyme. I. Equilibrium stabilization by cations. J. Mol. Biol. 2001, 306, 1157–1166, DOI: 10.1006/jmbi.2001.4437." />
<meta name="citation_reference" content="Hayes, R. L.; Noel, J. K.; Whitford, P. C.; Mo-hanty, U.; Sanbonmatsu, K. Y.; Onuchic, J. N. Reduced model captures Mg2+-RNA interaction free energy of riboswitches. Biophys. J. 2014, 106, 1508–1519, DOI: 10.1016/j.bpj.2014.01.042." />
<meta name="citation_reference" content="Hayes, R. L.; Noel, J. K.; Mandic, A.; Whitford, P. C.; Sanbonmatsu, K. Y.; Mohanty, U.; Onuchic, J. N. Generalized Manning condensation model captures the RNA ion atmosphere. Phys. Rev. Lett. 2015, 114, 258105, DOI: 10.1103/PhysRevLett.114.258105." />
<meta name="citation_reference" content="Heilman-Miller, S. L.; Pan, J.; Thirumalai, D.; Woodson, S. A. Role of counterion condensation in folding of the Tetrahymena ribozyme II. Counterion-dependence of folding kinetics. J. Mol. Biol. 2001, 309, 57–68, DOI: 10.1006/jmbi.2001.4660." />
<meta name="citation_reference" content="Manning, G. Limiting Laws and Counterion Condensation in Polyelectrolyte Solutions I. Colligative Properties. J. Chem. Phys. 1969, 51, 924, DOI: 10.1063/1.1672157." />
<meta name="citation_reference" content="Honeycutt, J. D.; Thirumalai, D. The nature of folded states of globular proteins. Biopolymers 1992, 32, 695–709, DOI: 10.1002/bip.360320610." />
<meta name="citation_reference" content="Sugita, Y.; Okamoto, Y. Replica-exchange molecular dynamics method for protein folding. Chem. Phys. Lett. 1999, 314, 141–151, DOI: 10.1016/S0009-2614(99)01123-9." />
<meta name="citation_reference" content="Hyeon, C.; Dima, R. I.; Thirumalai, D. Pathways and kinetic barriers in mechanical unfolding and refolding of RNA and proteins. Structure 2006, 14, 1633–1645, DOI: 10.1016/j.str.2006.09.002." />
<meta name="twitter:title" content="Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces ( f s) and monovalent salt concentrations ( C s). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C , we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log C m) α  with α = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation." />
<meta name="og-title" property="og:title" content="Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/048801v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces ( f s) and monovalent salt concentrations ( C s). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying f and C , we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As f and C are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log C m) α  with α = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2016-06-21" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/048801v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/048801v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/048801v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/048801v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/16542" />
    <title>Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"AgO1iUs3KRg6qMptxaIMxbtYLvwDZwo2kiNgnglKzfM","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"16542","apath":"\/biorxiv\/early\/2016\/06\/21\/048801.atom","pisa":"biorxiv;048801v2","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;048801\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;048801\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;048801v2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;048801\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;048801\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/048801v2","title":"Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension","identifier":"node\/16542"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224sQja+PtuECma1UoP4wltF23hucK47GFXKhr1Gunt4xZxKKd0o0mBMMxYqqqIoOcILDWx5AMbjfU4DQDjWhklAgnFso0IVHmtnviDkAS6FSFtOTxEWDodcCBYRS5taqxd1fZnwwaNbs6nWhT8G2xSlS3naxKac2mIdZp0LbEfij45SN+dI1UKyqb8ZGyzRF17AfqLhLtNrBdP\\\\\\\/wA+P4SwZ7JedVbK6q2mBD55Y1PjafDbGdvgzyIETXtYhu8SzQY\\\\\\\/8i5DVW1z6OwWQyqOXOlb6WnpmfBu9JBlRUeL9US1lVDH8WT9AVWF31vHgzN9zM2mGjxAdKUCE7a0WJohAe7gjWdvD5JHqIowXISDRegZ\\\\\\\/s\\\\\\\/dCnAuzrAcrwdTN+I48JWKNKx13S9mUMzy2wVx3pze9TH7MSqgD8JO5wAeSZjmAuGygAucv0Jc44+V4l9rj0HKQPVTGf2xD9NINcWiV+hmTfAPt3YVREexmv6jVATqJlylCNu0BMYAqaZguVhHO6rTbKAkEoRBUr3zkZen6rgcmkdhX5SUDU2q7pgrVBQXlmImdXnaw5ohr97X+f81Zux0iTa0V6ssT9KqZJOCYv1n06UPnEJc9MFeVleLtjvqOwoZ7WRv8ZPfBs\\\\\\\/4YM+Xl33KmckaJRbjAOiNMrAw\\\\\\\/g0fUPzYE9e+DU56NbmOPYkX+G+X3C3K75hPb7v4\\\\\\\/LpFkTpCoNF5L8AJfgVrFNTuTN9vIOFBV61eK0OjXrgyb3HN\\\\\\\/kb7MWpRuVjxAtrKEDsd55vl5T+68rwfliDunSEM1LcArAC+48vLE9IPgM88u7H2NHC5N8AgB7S9uqLEQa6gBSs83y1xM1TTR4VBGoH5uts9K4gY\\\\\\\/hCtyBXdyO1bgxonant99j3Rs6hMEGUdLmSQNT4\\\\\\\/GnG3L12TRv4RRs+LgDyHI\\\\\\\/EtOyWavjqoTe+0NL8KbuvaahWBR6yGn6F6xSW8bAl5ZimM4+e8HnSStCvYbzfDZel5iw\\\\\\\/zezeeRjdsFdwVeCScV4qCBbOYr0ihGMBhu5bWJa1ESPKgJ+wXEJyw8nMhE6brXv4ygi4iiA2s86GoRr7V5D2eomPfAXxSkecyjETNerrwmTWa3RVPXgorwCt0vVJEh6VLwk934FYjOcTmU2ybbbVqG2bGjWfp5fWXfkp0Z5bMgxGc348bwDdYvn8kcs3v\\\\\\\/vUqLBnDhLqW91R+8IqYfD9lZUxGKu2dZ9cyowoL10xRWGfXCyJPBJSSbNAeGeKAaKKDpCLzYkMbWMkc29TTmjmGZ9tkA2m6q7RiIFfVysOwAL7Uh2+rdI5tpXMWqH83m7srOOJn+wdkrn0TCs=\\\u0022,\\\u0022iv\\\u0022:\\\u0022RFlVKYrPLZL84V6xaBA5Fw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022569deeb4dd2648ada5717bc24f55ac282fce6563b4d68fdf122477e177ea257a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/048801v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-16542 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/048801v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cePo-wt6FjDIhSnbrsrgb5p-yhTDsG1WaRZrQjWe_tk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="16542" id="node16542" data-pisa="biorxiv;048801v2" data-pisa-master="biorxiv;048801" data-apath="/biorxiv/early/2016/06/21/048801.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0658-8266" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Naoto</span> <span class="nlm-surname">Hori</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Natalia A.</span> <span class="nlm-surname">Denesyuk</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">D.</span> <span class="nlm-surname">Thirumalai</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/048801 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node16542" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Naoto Hori </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute for Physical Science and Technology, University of Maryland</span>, College Park, MD 20742, <span class='nlm-country'>USA</span><span class='nlm-sup'>†</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Naoto%2BHori%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hori%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANaoto%2BHori%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0658-8266" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Naoto Hori</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Natalia A. Denesyuk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute for Physical Science and Technology, University of Maryland</span>, College Park, MD 20742, <span class='nlm-country'>USA</span><span class='nlm-sup'>†</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BA.%2BDenesyuk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Denesyuk%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2BA.%2BDenesyuk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">D. Thirumalai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute for Physical Science and Technology, University of Maryland</span>, College Park, MD 20742, <span class='nlm-country'>USA</span><span class='nlm-sup'>†</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D.%2BThirumalai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thirumalai%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD.%2BThirumalai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/048801v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:16542" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:16542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/048801v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:16542" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:16542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/048801v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:16542" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:16542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/048801v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:16542" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:16542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/048801v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:16542" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:16542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/048801v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:16542" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:16542/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Because of the potential link between -1 programmed ribosomal frameshifting and response of a pseudoknot (PK) RNA to force, a number of single molecule pulling experiments have been performed on PKs to decipher the mechanism of programmed ribosomal frameshifting. Motivated in part by these experiments, we performed simulations using a coarse-grained model of RNA to describe the response of a PK over a range of mechanical forces (<em>f</em>s) and monovalent salt concentrations (<em>C</em>s). The coarse-grained simulations quantitatively reproduce the multistep thermal melting observed in experiments, thus validating our model. The free energy changes obtained in simulations are in excellent agreement with experiments. By varying <em>f</em> and <em>C</em>, we calculated the phase diagram that shows a sequence of structural transitions, populating distinct intermediate states. As <em>f</em> and <em>C</em> are changed, the stem-loop tertiary interactions rupture first, followed by unfolding of the 3’-end hairpin (I⇌F). Finally, the 5’-end hairpin unravels, producing an extended state (E⇌I). A theoretical analysis of the phase boundaries shows that the critical force for rupture scales as (log <em>C</em><sub>m</sub>)<sup><em>α</em></sup> with <em>α</em> = 1 (0.5) for E⇌I (I⇌F) transition. This relation is used to obtain the preferential ion-RNA interaction coefficient, which can be quantitatively measured in single-molecule experiments, as done previously for DNA hairpins. A by-product of our work is the suggestion that the frameshift efficiency is likely determined by the stability of the 5’ end hairpin that the ribosome first encounters during translation.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-5">The multifarious roles RNA molecules play in controlling a myriad of cellular functions [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>] have made it important to understand in quantitative terms their folding [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>–<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>] and how they respond to external stresses [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. Among them, one of the simplest structural motifs is the RNA pseudoknot (PK), which is involved in many biological functions. The simplest type, referred to as H-type PK, consists of two stems connected by two loops in which one of the stems forms base pairs with the loop of the other. The PK motif, found in many RNA molecules such as telomerase [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>], mRNA [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], ribosomal RNA [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], transfer-messenger RNA, and viral RNA [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>], is functionally important [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. For instance, a PK in the human telomerase RNA is essential for enzyme (a ribonucleoprotein complex) activity [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Many pathogenic mutations in the human telomerase RNA are found in the well-conserved PK region of the RNA, further underscoring the importance of the PK [<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>]. The presence of PKs is also important in ribozyme catalysis and inhibition of ribosome translation.</p><p id="p-6">Recently, there is heightened interest in the biophysics of PK folding because it plays a crucial role in affecting the efficiency of −1 programmed ribosomal frameshifting (−1 PRF) [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Usually, proteins are synthesized when the ribosome reads the mRNA code in three nucleotide steps until the stop codon is reached. In −1 PRF, the open reading frame of mRNA being translated within the ribosome is programmed to be shifted by one nucleotide, and consequently, the mRNA becomes nonfunctional or produces an entirely different protein [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. The downstream PK of the mRNA is a roadblock at the entrance to the mRNA channel on the ribosome, which impedes the translocation of the ribosome. The ribosome must unfold the PK, presumed to occur by exertion of mechanical force, to complete translocation. Because frameshifting efficiency could depend on how the PK responds to force, a number of single-molecule pulling experiments have focused on PKs [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. Several factors could determine −1 PRF, as evidenced by the multitude of proposals based on many experimental studies [<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>–<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. Nevertheless, given that ribosome translocation exerts mechanical force on the downstream mRNA, it is physically reasonable to assume that resistance of PK to force should at least be one factor determining the frameshifting efficiency [<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>].</p><p id="p-7">The considerations given above and the availability of both ensemble and single-molecule pulling experimental data prompted us to focus on the effects of temperature, mechanical force, and salt concentration on a 28nucleotide H-type RNA PK from the beet western yellow virus (BWYV) [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. This PK has been the subject of several ensemble experiments, which have characterized the folding thermodynamics as a function of monovalent and divalent cations [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. The BWYV PK has two stems (S1 and S2) connected by two loops (L1 and L2) as shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1.</a>. The crystal structure reveals that L2 forms triplex-type tertiary interactions with S1[<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. In another context, it has been shown that such interactions and mutations affecting them dramatically change the efficiency of frameshifting [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]. In <em>vitro</em> and <em>in vivo</em> experiments on many mutants of BWYV suggest variations in the frameshift efficiency, which, for some mutants, is attributed to changes in the stem-loop interactions affecting the stability of PK. For others, direct involvement of the ribosome is invoked [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. In addition to mechanical force (<em>f</em>), it is known from several ensemble experiments that the stability of BWYV PK, like other RNA molecules, depends on monovalent and divalent cations [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. Thus, it is important to understand in quantitative detail how this crucial structural motif respond to <em>f</em> and changes in ion concentration.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of the BWYV pseudoknot. (A) Sketch of the crystal structure [32] color-coded according to secondary structures. The black arrows indicate positions at which the external mechanical force is applied. (B) Schematic representation of the secondary structure. Unlike the typical H-type pseudoknot, the two helix stems (S1 and S2) are not coaxially stacked in the BWYV pseudoknot [25]. Tertiary interactions are indicated by red dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Structure of the BWYV pseudoknot. (&lt;strong&gt;A&lt;/strong&gt;) Sketch of the crystal structure [32] color-coded according to secondary structures. The black arrows indicate positions at which the external mechanical force is applied. (&lt;strong&gt;B&lt;/strong&gt;) Schematic representation of the secondary structure. Unlike the typical H-type pseudoknot, the two helix stems (S1 and S2) are not coaxially stacked in the BWYV pseudoknot [25]. Tertiary interactions are indicated by red dashed lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIG. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F1.medium.gif" width="440" height="378"/><noscript><img class="highwire-fragment fragment-image" alt="FIG. 1." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F1.medium.gif" width="440" height="378"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIG. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">FIG. 1.</span> <p id="p-8" class="first-child">Structure of the BWYV pseudoknot. (<strong>A</strong>) Sketch of the crystal structure [<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>] color-coded according to secondary structures. The black arrows indicate positions at which the external mechanical force is applied. (<strong>B</strong>) Schematic representation of the secondary structure. Unlike the typical H-type pseudoknot, the two helix stems (S1 and S2) are not coaxially stacked in the BWYV pseudoknot [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. Tertiary interactions are indicated by red dashed lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Although single-molecule pulling experiments suggest how force affects the stability of RNA [<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>], structural information obtained from such experiments is limited only to changes in molecular extensions. In order to provide a theoretical basis for the changes in the stability and structural transitions as <em>f</em> and monovalent salt concentration (<em>C</em>) are altered, we performed simulations using a coarse-grained three-site-interaction model (TIS model) [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>] of the PK. After demonstrating that our simulations reproduce the temperature-induced structural transitions observed in ensemble experiments, we calculated the global phase diagrams in the [<em>C</em>, <em>f</em>] plane. The simulations, performed in the same force range as used in Laser Optical Tweezer (LOT) experiments [<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>], quantitatively reproduce the experimental observations. By combining our simulations and a theory based on ion-RNA interactions expressed in terms of preferential interaction coefficients [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>], we produce several key predictions: (1) The phase diagram in the [<em>C</em>, <em>f</em>] plane shows that the folded state (F) ruptures in stages populating distinct intermediates as <em>f</em> increases. Structurally, the low force intermediate is similar to the one populated during thermal melting. The sequence of <em>f</em>-induced structural transitions can be described as F⇌I⇌E, where E is an extended state; (2) The critical force, <em>f</em><sub>c</sub>, to rupture the PK scales as (log <em>C</em><sub>m</sub>)<sup><em>α</em></sup> with <em>α</em> = 1 for the I⇌E transition and <em>α</em> = 0.5 for the I⇌F transition. We expect this result to be valid for H-type PKs composed of stems with vastly different stability, such as the BWYV PK. This result is of general validity and is applicable to other PKs as well. The slope of the <em>f</em><sub>c</sub> versus log <em>C</em><sub>m</sub> is proportional to the changes in the preferential interaction coefficient between an intermediate and unfolded states. This prediction can be used in conjunction with single-molecule experiments allowing for direct measurement of a key quantity in ion-RNA interactions; and (3) To the extent the downstream PK is a roadblock for translation, our work suggests that the ribosome should exert a force ~ (10 – 15) pN [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>] for unimpeded translocation along mRNA, although the actual force value could vary depending on factors such as interaction between PK and ribosome. Based in part on our work we propose that there should be a link between the stimulatory effect of frameshifting and the stability of the 5’–end of the PK. Our predictions are amenable to validations using single-molecule experiments.</p></div><div class="section" id="sec-2"><h2 class="">RESULTS AND DISCUSSION</h2><div id="sec-3" class="subsection"><h3>Temperature-induced transitions in BWYV PK</h3><p id="p-10">In order to validate our model, we first performed temperature replica-exchange molecular dynamics (REMD) simulations at <em>C</em> = 500 mM with <em>f</em> = 0. Formations of stems (S1 and S2) and tertiary (stem-loop) interactions (L1 and L2) are characterized by hydrogen bond (HB) energies <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-1.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-2.gif"/></noscript></span></span>, respectively. Changes in the distributions of HB energies as the temperature is increased are shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. S1</a> in the Supporting Information. (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. S1</a> shows that there are two minima corresponding to two distinct states. The minimum that dominates at <em>T</em> = 20°C corresponds to the F state in which both S1 and S2 are formed (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-3.gif"/></noscript></span></span> ≈ −45 kcal/mol). The minimum with <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-4.gif"/></noscript></span></span> ≈ −30 kcal/mol at 80°C corresponds to a state in which S1 is intact (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. S1</a>). The free energy profile of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-5.gif"/></noscript></span></span> shows only one possible minimum around <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-6.gif"/></noscript></span></span> ≈ −20 kcal/mol at <em>T</em> = 20°C, indicating that L1 and L2 fold/unfold cooperatively as temperature changes.</p><p id="p-11">We calculated the free energy profiles <em>G</em>(<em>R<sub><em>α</em></sub></em> = −<em>k</em><sub>B</sub><em>T</em>log<em>P</em>(<em>R</em><sub><em>α</em></sub> (R<sub><em>α</em></sub> = <em>R</em><sub>ee</sub>, end-to-end distance, or <em>R</em><sub>g</sub>, radius of gyration) from the distributions of <em>P</em>(<em>R</em><sub><em>α</em></sub>), which are given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. S2</a> in the Supporting Information. The locations and the number of minima report on the thermodynamic transitions between the states (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). (1) At a low temperature, T = 20°C, the PK is completely folded (F state) including all the structural motifs, S1, S2, L1 and L2. The existence of F state is also confirmed by the presence of a minimum at <em>R</em><sub>ee</sub> ≈ 3 nm (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2A</a>) and at <em>R</em><sub>g</sub> ≈ 1.3 nm (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). These values are similar to those obtained directly from the crystal structure (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-7.gif"/></noscript></span></span>). (2) At <em>T</em> = 60°C, the free energy profile for <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-8.gif"/></noscript></span></span> shows that some of the tertiary interactions, involving L1 and L2, are no longer stably formed (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Fig. S1B</a>). On the other hand, <em>G</em>(<em>R</em><sub>g</sub>) shows two stable minima, indicating that there are two possible states ((<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2B</a>). In one state, both stems S1 and S2 are formed but tertiary interactions are absent, corresponding to the I1 state identified using UV absorbance and differential scanning calorimetry experiments (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>) [<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>]. The other conformation, in which only S1 is formed, has been also observed in experiments and is referred to as the I2 state. It should be noted that the distribution of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-9.gif"/></noscript></span></span> also has two minima at 60°C (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Fig. S1A</a>) whereas <em>G</em>(<em>R</em><sub>ee</sub>) has only one minimum (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2A</a>). (3) At a still higher temperature <em>T</em> = 80°C, <em>G</em>(<em>R</em><sub>ee</sub>) and <em>G</em>(<em>R</em><sub>g</sub>) each have a minimum with S1 stably formed. This is also reflected in (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Fig. S1</a>, which shows a minimum at <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-10.gif"/></noscript></span></span> ≈ −30 kcal/mol and a minimum at <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-11.gif"/></noscript></span></span> ≈ 0. Thus, completely unfolded conformations, U state, and the I2 state are populated. (4) At <em>T</em> = 110°C, both <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-13.gif"/></noscript></span></span> are 0, indicating that only the U state exists. This is also reflected in <em>G</em>(<em>R</em><sub>ee</sub>) and <em>G</em>(<em>R</em><sub>g</sub>), which show that the PK is completely unfolded. In both profiles (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2A</a> and <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2B</a>), the center of minimum is located at larger values than in the F state (<em>R</em><sub>ee</sub> ≈ 5 nm and <em>R</em><sub>g</sub> ≈ 2 nm).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characteristics of the thermally induced folding-unfolding transition of BWYV PK simulated using the temperature-REMD method at C = 500 mM and f = 0.(A and B) Free energy profles of the molecular extension (A;Ree) and radius of gyration (B; Rg) at 20, 60, 80, and 110&#xB0;C.Each curve is normalized to G = 0 at its probability P(R) maximum. (C) Temperature dependence of the fraction of the four prominent states, which are explicitly labeled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Characteristics of the thermally induced folding-unfolding transition of BWYV PK simulated using the temperature-REMD method at C = 500 mM and &lt;em&gt;f&lt;/em&gt; = 0.(A and B) Free energy profles of the molecular extension (A;&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;ee&lt;/sub&gt;) and radius of gyration (B; &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;g&lt;/sub&gt;) at 20, 60, 80, and 110&#xB0;C.Each curve is normalized to G = 0 at its probability &lt;em&gt;P&lt;/em&gt;(&lt;em&gt;R&lt;/em&gt;) maximum. (C) Temperature dependence of the fraction of the four prominent states, which are explicitly labeled.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F2.medium.gif" width="197" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F2.medium.gif" width="197" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <p id="p-12" class="first-child">Characteristics of the thermally induced folding-unfolding transition of BWYV PK simulated using the temperature-REMD method at C = 500 mM and <em>f</em> = 0.(A and B) Free energy profles of the molecular extension (A;<em>R</em><sub>ee</sub>) and radius of gyration (B; <em>R</em><sub>g</sub>) at 20, 60, 80, and 110°C.Each curve is normalized to G = 0 at its probability <em>P</em>(<em>R</em>) maximum. (C) Temperature dependence of the fraction of the four prominent states, which are explicitly labeled.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">The simulation results in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2</a> and <a id="xref-fig-10-4" class="xref-fig" href="#F10">Fig. S1</a> show that melting of BWYV PK undergoes through two intermediate states, F⇌I1⇌I2⇌U where I1 has the topology of the F state but without significant tertiary interactions (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>). Although simulations can be used to distinguish I1 and F states, the intermediate does not have significant population. The identification of I1 in our simulations provides a structural support to experiments in which I1 was inferred from three broad overlapping peaks in the heat capacity curves [<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">31</a>]. It should be stressed that the experimental heat capacity curves exhibit only two discernible peaks, which would normally imply that there is only one intermediate present. The range of melting temperatures for the three transitions can be obtained from the temperature dependence of the various states <em>f</em><sub><em>α</em></sub>(<em>T</em>) (<em>α</em>=F, I1, I2 or U) given in <a id="xref-fig-2-10" class="xref-fig" href="#F2">fig. 2C</a>. The I1 state has a maximum population around <em>T</em> ≈ 50°C. We equate this with the melting temperature <em>T</em><sub>m1</sub> associated with the F⇌I1 transition. Similarly, <em>T</em><sub>m2</sub> for the I1⇌I2 transition is ≈ 70°C. The <em>T</em><sub>m3</sub> for the I2⇌U transition occurs around 90°C (<em>f</em><sub>I2</sub>(<em>T</em><sub>m3</sub>) = <em>f</em><sub>u</sub>(<em>T</em><sub>m3</sub>)). The values of the three melting temperatures obtained in our simulations are in remarkable agreement with experiments <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-14.gif"/></noscript></span></span> taken from results of a differential scanning calorimetry experiment at pH=7 in Table 1 of Nixon and Giedroc [<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">30</a>]).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of the thermodynamic assembly of BWYV pseudoknot inferred from simulations at zero force. Besides folded (F) and unfolded (U) states, there are two intermediate state, I1 and I2. The topology of the I1 state is the same as F, except tertiary interactions are absent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic of the thermodynamic assembly of BWYV pseudoknot inferred from simulations at zero force. Besides folded (F) and unfolded (U) states, there are two intermediate state, I1 and I2. The topology of the I1 state is the same as F, except tertiary interactions are absent.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F3.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F3.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <p id="p-14" class="first-child">Schematic of the thermodynamic assembly of BWYV pseudoknot inferred from simulations at zero force. Besides folded (F) and unfolded (U) states, there are two intermediate state, I1 and I2. The topology of the I1 state is the same as F, except tertiary interactions are absent.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The dependence of the stability of the F state with respect to the U state, Δ<em>G</em><sub>UF</sub>(<em>T</em>) is calculated using a procedure that does not rely on any order parameter [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>].The value of Δ<em>G</em><sub>UF</sub>(T = 37°C) is −14.3 kcal/mol from simulations, which is in excellent agreement with experiments. The experimental value reported by Nixon and Giedroc for Δ<em>G</em><sub>UF</sub>(T = 37°C) = −13.3 kcal/mol, whereas Soto et al. estimated that Δ<em>G</em><sub>UF</sub>(<em>T</em> = 37°C) = −15.1 kcal/mol [<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">31</a>]. We also predict that Δ<em>G</em><sub>UF</sub>(<em>T</em> = 25°C) is −19.0 kcal/mol. The excellent agreement between simulations and experiments for the temperature induced structural transitions and the Δ<em>G</em><sub>UF</sub>(<em>T</em>) validates the model allowing us to predict the effect of <em>f</em> on BWYV PK.</p><p id="p-16"><strong>Diagram of states in the</strong> [<em>C</em>, <em>f</em>] <strong>plane</strong></p><p id="p-17">In order to obtain the phase diagram in the [<em>C</em>, <em>f</em>] plane of the BWYV PK, we performed a series of low friction Langevin dynamics simulations by varying the salt concentration from 5 to 1200 mM and the mechanical force from 0 to 20 pN, at a constant temperature of 50°C. We varied <em>C</em> by changing the Debye length in the simulations, and the mechanical force was externally applied to the terminal nucleotides (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1A</a>).</p><p id="p-18">Determination of the phase diagram in the [<em>C</em>, <em>f</em>] plane requires an appropriate order parameter that can distinguish between the various states. In single-molecule pulling experiments, the variable that can be measured is the molecular extension, <em>R</em><sub>ee</sub>, which is conjugated to the applied force. The overall size of the RNA molecule is assessed using the radius of gyration, <em>R</em><sub>g</sub>. Therefore, we characterized the states of the RNA using both <em>R</em><sub>g</sub> and <em>R</em><sub>ee</sub> as order parameters. The values of the predicted <em>R</em><sub>g</sub> at f = 0 can be measured using scattering experiments. Using these two parameters, we obtained the [<em>C</em>, <em>f</em>] phase diagram ((<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>). Comparison of the diagram of states in (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4D</a> and (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4E</a> reveals common features and some differences. From (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4D</a> and (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4E</a>, we infer that at f &gt; 12.5 pN, extended (E) conformations are dominant at all values of <em>C</em>. As the force decreases, the PK forms compact structures. The boundary separating the extended and relatively compact phases depends on the salt concentration and the value of the mechanical force. The critical force to rupture the compact structures increases linearly as a function of logarithm of salt concentration (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4D</a>; boundary between red and green regions). At low forces (<em>f</em> &lt; 2.5 pN), the diagram of states based on <em>R</em><sub>ee</sub> shows that the extension is relatively small as <em>C</em> changes from a low to a high value. From this finding, one might be tempted to infer that the PK is always folded, which is not physical especially at low (<em>C</em> ≈ 10 mM) ion concentrations. In contrast, (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4E</a> shows that below 5 pN, there is a transition from compact structures (<em>R</em><sub>g</sub> ≈ 1.3 nm in the blue region) at high <em>C</em> to an intermediate state (<em>R</em><sub>g</sub> &gt; 2.2 nm in the green region) at C ≈ 100 mM. The differences between the diagrams of states in the [<em>C</em>, <em>f</em>] plane using <em>R</em><sub>ee</sub> and <em>Rg</em> as order parameters are more vividly illustrated in terms of the free energy profiles <em>G</em>(<em>R</em><sub><em>α</em></sub>) = −<em>k</em><sub>B</sub><em>T</em>log(<em>R</em><sub><em>α</em></sub>) where <em>R</em><sub><em>α</em></sub> is <em>R</em><sub>ee</sub> or <em>R</em><sub>g</sub> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>). The profiles <em>G</em>(<em>R</em><sub>ee</sub>), at three values of <em>C</em> and different <em>f</em> values, show two minima at most. At <em>f</em> = 0, there is only one discernible minimum at <em>R</em><sub>ee</sub> ≈ 4.2 nm at C = 10 mM. The minimum moves to <em>R</em><sub>ee</sub> ≈ 3 nm at <em>C</em> = 1200 mM corresponding to a folded PK. At <em>f</em> = 5 pN there are two minima at <em>C</em> = 10 mM corresponding to a compact structure and a stretched conformation (see the cyan profile in (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a>). As <em>f</em> exceeds 5 pN, there is essentially one minimum whose position increases as <em>f</em> increases. In the force regime (<em>f</em> &gt; 5 pN), only the E state is visible at all <em>C</em>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A&#x2013;C) Representative snapshots corresponding to the three distinct states from simulations. (D) Diagram of states in the [C, f] plane obtained using extension Ree as the order parameter. (E) Same as D, except Rg is used to distinguish the states. The three crosses in D and E correspond to conditions from which the snapshots (A-C) are sampled. The scale for Ree and Rg are given on the right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(&lt;strong&gt;A&lt;/strong&gt;&#x2013;&lt;strong&gt;C&lt;/strong&gt;) Representative snapshots corresponding to the three distinct states from simulations. (&lt;strong&gt;D&lt;/strong&gt;) Diagram of states in the [&lt;em&gt;C&lt;/em&gt;, &lt;em&gt;f&lt;/em&gt;] plane obtained using extension &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;ee&lt;/sub&gt; as the order parameter. (&lt;strong&gt;E&lt;/strong&gt;) Same as D, except &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;g&lt;/sub&gt; is used to distinguish the states. The three crosses in D and E correspond to conditions from which the snapshots (&lt;em&gt;A&lt;/em&gt;-&lt;em&gt;C&lt;/em&gt;) are sampled. The scale for &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;ee&lt;/sub&gt; and &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;g&lt;/sub&gt; are given on the right.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F4.medium.gif" width="316" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F4.medium.gif" width="316" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <p id="p-19" class="first-child">(<strong>A</strong>–<strong>C</strong>) Representative snapshots corresponding to the three distinct states from simulations. (<strong>D</strong>) Diagram of states in the [<em>C</em>, <em>f</em>] plane obtained using extension <em>R</em><sub>ee</sub> as the order parameter. (<strong>E</strong>) Same as D, except <em>R</em><sub>g</sub> is used to distinguish the states. The three crosses in D and E correspond to conditions from which the snapshots (<em>A</em>-<em>C</em>) are sampled. The scale for <em>R</em><sub>ee</sub> and <em>R</em><sub>g</sub> are given on the right.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Free energy profiles as a function of Ree (A-C) and Rg (D-E) at three different salt concentrations and a range of forces. The locations of the minima change as C and f vary, indicating transitions between distinct states of the BWYV PK." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Free energy profiles as a function of &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;ee&lt;/sub&gt; (&lt;strong&gt;A&lt;/strong&gt;-&lt;strong&gt;C&lt;/strong&gt;) and &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;g&lt;/sub&gt; (&lt;strong&gt;D&lt;/strong&gt;-&lt;strong&gt;E&lt;/strong&gt;) at three different salt concentrations and a range of forces. The locations of the minima change as &lt;em&gt;C&lt;/em&gt; and &lt;em&gt;f&lt;/em&gt; vary, indicating transitions between distinct states of the BWYV PK.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F5.medium.gif" width="422" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F5.medium.gif" width="422" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5.</span> <p id="p-20" class="first-child">Free energy profiles as a function of <em>R</em><sub>ee</sub> (<strong>A</strong>-<strong>C</strong>) and <em>R</em><sub>g</sub> (<strong>D</strong>-<strong>E</strong>) at three different salt concentrations and a range of forces. The locations of the minima change as <em>C</em> and <em>f</em> vary, indicating transitions between distinct states of the BWYV PK.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In order to compare the thermal free energy profiles generated at <em>C</em> = 500 mM (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2</a>), we calculated the <em>G</em>(<em>R</em><sub>g</sub>) and <em>G</em>(<em>R</em><sub>ee</sub>) at different forces with <em>C</em> fixed at 500 mM. Comparison of (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Fig. 2</a> and (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Fig. S3</a> shows that at <em>f</em> = 0 the free energy profiles are similar. At <em>f</em> ≠ 0 the folded state is destabilized, and at <em>f</em> = 15 pN, the unfolded state is more stable than the folded PK.</p><p id="p-22">A much richer diagram of states emerges when the <em>G</em>(<em>R</em><sub>g</sub>) = −<em>k</em><sub>B</sub><em>T</em>log<em>P</em>(<em>R</em><sub>g</sub>) is examined. The <em>G</em>(<em>R</em><sub>g</sub>) profile at <em>C</em> = 10 mM is similar to <em>G</em>(<em>R</em><sub>ee</sub>). However, at higher concentrations, there is clear signature of three states (especially at <em>f</em> = 12.5 pN) corresponding to the folded PK, an intermediate state, and the E state. The free energy profiles show that under force, there are minimally three states, which are F, I, and E. Because <em>R</em><sub>ee</sub> cannot capture the complexity of the states populated as <em>C</em> and <em>f</em> are varied, it is important to consider <em>R</em><sub>g</sub> as well to fully capture the phase diagram of the PK. We also computed the free energy profiles as a function of root-mean-square-deviations (RMSD). In terms of the number and relative positions of basins, the profiles based on RMSD are qualitatively similar to one of <em>R</em><sub>g</sub> (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Fig. S4</a>).</p></div><div id="sec-4" class="subsection"><h3>Formation of stems and loops</h3><p id="p-23">In order to highlight the structural changes that occur as <em>C</em> and <em>f</em> are varied, we used, <em>Q</em>, the fraction of native HB interactions as an order parameter. The [<em>C</em>, <em>f</em>] phase diagram (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A</a>) calculated using the average <em>Q</em> is similar to the <em>R</em><sub>g</sub>-phase diagram (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4E</a>), indicating the presence of three states (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6A</a>). Using <em>Q</em>, we can also quantitatively assess the contributions from different structural elements to the stability of the PK and correctly infer the order in which the structural elements of the PK rupture as <em>f</em> is increased. In order to determine the structural details of each state, we calculated the individual <em>Q</em>s for the two stems (<em>Q</em><sub>s1</sub>, <em>Q</em><sub>S2</sub>) and the two loops (<em>Q</em><sub>L1</sub> <em>Q</em><sub>l2</sub>) (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6B</a>). The dependence of (<em>Q</em><sub>S1</sub>) as a function of <em>C</em> and <em>f</em> shows that Stem 1 (S1) is extremely stable and remains intact at all salt concentrations, rupturing only at f ≈ 10 pN (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6B</a>, upper left panel). In contrast, the upper right panel in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6B</a> shows that Stem 2 (S2) is only stably folded above a moderate salt concentration (<em>C</em> ≈ 80 mM) and low <em>f</em>. The stability difference between S1 and S2 can be explained by the number of canonical G-C base pairs; S1 has five G-C base pairs, whereas only three such pairs are in S2. Consequently, S1 cannot be destabilized even at low <em>C</em>, but its rupture to produce extended states requires the use of mechanical force. Above <em>C</em> &gt; 100 mM, the fraction of contacts associated with S2 and the two loops (<em>Q</em><sub>S2</sub>, <em>Q</em><sub>L1</sub> and <em>Q</em><sub>L2</sub>) simultaneously increase. All the interactions are, however, still not completely formed (<em>Q</em> ≈ 0.8) even at the highest salt concentration, <em>C</em> = 1200 mM. In particular, the contacts involving L2, <em>Q</em><sub>L2</sub> ≈ 0.6, implying that tertiary interactions are only marginally stable at <em>T</em> = 50° C.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="[C, f] phase diagram using the fraction of native contacts as the order parameter. (A) The diagram of states determined using an average of the total Q for the PK. The scale is given on the right. (B) Decomposition of Q into stems and loops: S1, stem 1; S2, stem 2; L1, loop 1; L2, loop 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;[&lt;em&gt;C&lt;/em&gt;, &lt;em&gt;f&lt;/em&gt;] phase diagram using the fraction of native contacts as the order parameter. &lt;strong&gt;(A)&lt;/strong&gt; The diagram of states determined using an average of the total &lt;em&gt;Q&lt;/em&gt; for the PK. The scale is given on the right. &lt;strong&gt;(B)&lt;/strong&gt; Decomposition of &lt;em&gt;Q&lt;/em&gt; into stems and loops: S1, stem 1; S2, stem 2; L1, loop 1; L2, loop 2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F6.medium.gif" width="284" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 6." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F6.medium.gif" width="284" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 6.</span> <p id="p-24" class="first-child">[<em>C</em>, <em>f</em>] phase diagram using the fraction of native contacts as the order parameter. <strong>(A)</strong> The diagram of states determined using an average of the total <em>Q</em> for the PK. The scale is given on the right. <strong>(B)</strong> Decomposition of <em>Q</em> into stems and loops: S1, stem 1; S2, stem 2; L1, loop 1; L2, loop 2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The results in <a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6</a> provide a structural explanation of the transitions that take place as <em>C</em> and <em>f</em> are changed. (1) In the high force regime, the two stems are fully ruptured and native interactions are absent creating the E state. (2) Below <em>C</em> ≈ 100 mM and <em>f</em> ≲ 10 pN (left bottom on the [<em>C</em>, <em>f</em>] plane), only S1 is intact, and thus, a hairpin conformation is dominant. This is the intermediate state, I2, identified by Soto et al. who found that S1 remains stable even at salt concentration as less as 20 mM NaCl [<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">31</a>]. (3) Above <em>C</em> ≳ 100 mM and <em>f</em> ≲ 10 pN (right bottom), both S1 and S2 stems are formed, and the tertiary interactions involving the two loops are at least partly formed, ensuring that the topology of the PK is native-like. At <em>f</em> = 0 and temperature as a perturbation, it has been shown that another intermediate state, I1, exists between I2 and folded states [<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">31</a>]. The I1 state may correspond to conformations in which two stems are fully folded with the entire topology of the native conformation intact, but tertiary interactions between stems and loops are incomplete. Although we have accounted for the presence of such a state in our simulations (see <a id="xref-fig-2-13" class="xref-fig" href="#F2">Fig. 2</a>), from here on, we do not distinguish between I1 and the completely folded state in the phase diagram since the increases in <em>Q</em><sub>S2</sub>, <em>Q</em><sub>l1</sub> and <em>Q</em><sub>L2</sub> on the [<em>C</em>, <em>f</em>] plane almost overlap. Therefore, we classify the right bottom region in the [<em>C</em>, <em>f</em>] phase diagram as (I1 + F) state.</p></div><div id="sec-5" class="subsection"><h3>Population analysis of the secondary and tertiary interactions</h3><p id="p-26">The [<em>C</em>, <em>f</em>] phase diagrams provide a thermodynamic basis for predicting the structural changes that occur as <em>f</em> and <em>C</em> are varied. However, they capture only the average property of the ensemble within each state without the benefit of providing a molecular picture of the distinct states, which can only be described using simulations done at force values close to those used in LOT experiments. Therefore, we investigated the distribution of interactions to ascertain the factors that stabilize the three states as <em>C</em> and <em>f</em> are varied. Our analysis is based on the HB energy, <em>U</em><sub>HB</sub>. The energy function for HB captures the formation of base pairs and loop-stem tertiary interactions, taking into account both distance and angular positions of the donor and the acceptor [<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>].</p><p id="p-27"><a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a> shows that the probability densities of the total HB energy have three peaks at <em>U</em><sub>HB</sub> ≈ 0, −30, and −60 kcal/mol. In the high-force regime (<em>f</em> &gt; 12.5 pN), there is only one population at all <em>C</em> around <em>U</em><sub>HB</sub> ≈ 0, which obviously corresponds to the fully extended state. The total HB energy can be decomposed into contributions arising from four distinct structural motifs in the PK (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Fig. S5</a>). The peak at <em>U</em><sub>HB</sub> ≈ −30 kcal/mol (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7</a>) arises due to the formation of S1 (compare <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figs. 7</a> and S5A). The broad peak at <em>U</em><sub>HB</sub> ≈ −60 kcal/mol (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig. 7A</a>) is due to the sum of contributions from S2 and the tertiary interactions.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Probability distributions of the total hydrogen bond energy as a function of C and f, which are labeled. The distinct peaks are fingerprints of the states that are populated at a specified C and f." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Probability distributions of the total hydrogen bond energy as a function of &lt;em&gt;C&lt;/em&gt; and &lt;em&gt;f&lt;/em&gt;, which are labeled. The distinct peaks are fingerprints of the states that are populated at a specified &lt;em&gt;C&lt;/em&gt; and &lt;em&gt;f&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F7.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 7." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F7.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 7.</span> <p id="p-28" class="first-child">Probability distributions of the total hydrogen bond energy as a function of <em>C</em> and <em>f</em>, which are labeled. The distinct peaks are fingerprints of the states that are populated at a specified <em>C</em> and <em>f</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Parsing the <em>U</em><sub>HB</sub> due to contributions from interactions among S1, S2, L1, and L2 produces a picture of thermodynamic ordering as <em>C</em> (<em>f</em>) is increased (decreased) from low (high) value to high (low) value. Low <em>C</em> and high <em>f</em> correspond to the top left region in the phase diagrams (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figs. 4</a> and <a id="xref-fig-6-7" class="xref-fig" href="#F6">6</a>). In all conditions, S1 is the first to form as indicated by the presence of distinct peaks at <em>C</em> &gt; 100 mM and f ≲ 10 pN in <a id="xref-fig-14-2" class="xref-fig" href="#F14">Fig. S5A</a>. Under more favorable conditions (<em>f</em> &lt; 10 pN; <em>C</em> ≈ 300 mM or greater) S2 and tertiary interactions involving L1 are established (<a id="xref-fig-14-3" class="xref-fig" href="#F14">Figs. S5</a> B and C). Only at the highest value of <em>C</em> and <em>f</em> ≲ 5 pN, tertiary interactions involving L2 emerge. Interestingly, the order of formation of the various structural elements en route to the formation of their folded state follows the stabilities of the individual elements with the most stable motif forming first. The PK tertiary interactions are consolidated only after the assembly of the stable structural elements. This finding follows from our earlier work establishing that the hierarchical assembly of RNA PKs is determined by the stabilities of individual structural elements [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>].</p><p id="p-30">We did not find any condition in which S1 and S2 are completely folded, but there are no tertiary interactions, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-15.gif"/></noscript></span></span> ≈ 0 (i.e., isolated I1 state). Our simulations show that the tertiary interactions contribute to the loop energy only after the stems are formed. Thus, the actual population is distributed between I1 and F, and the tertiary interactions are only marginally stable in the conditions examined here. Soto et al. suggested that diffuse Mg<sup>2+</sup> ions play an important role in stabilizing such tertiary interactions [<a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">31</a>]. Because Mg<sup>2+</sup> is not considered here, we cannot determine the precise interactions stabilizing the I1 state, especially at <em>f</em> ≠ 0. In the discussion hereafter, for simplicity, we refer to (I1 + F) state as F.</p></div><div id="sec-6" class="subsection"><h3>Critical rupture force and preferential ion interaction coefficient</h3><p id="p-31">In order to delineate the phase boundaries quantitatively, we calculated the free energy differences between the three major states in the [<em>C</em>, <em>f</em>] plane using <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-16.gif"/></noscript></span></span>, where the classification of the states, E, I (=I2), or F, is based on the HB interaction energies of stems, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-18.gif"/></noscript></span></span> (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 8</a>). A similar equation is used to calculate Δ<em>G</em><sub>IF</sub>. Based on the distribution of <em>U</em><sub>HB</sub>, we determined the threshold values for S1 formation as <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-19.gif"/></noscript></span></span> &lt; −15kcal/mol (<a id="xref-fig-14-4" class="xref-fig" href="#F14">Fig. S5A</a>). Similarly, S2 formation is deemed to occur if <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-20.gif"/></noscript></span></span> −10kcal/mol (<a id="xref-fig-14-5" class="xref-fig" href="#F14">Fig. S5B</a>). We classified each structure depending on whether both stems are formed (F) with both <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-21.gif"/></noscript></span></span> &lt; −15 kcal/mol and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-22.gif"/></noscript></span></span> &lt; −10 kcal/mol, only S1 is formed (I (=I2)), or fully extended (E).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Free energy differences between E and I, and I and F calculated from REMD simulations. The definition of states, E,I, and F, is based on the hydrogen bond interaction energies associated with each structural element (see main text for details).The I state is analogous to I2 in Fig. 3. (A and B) Salt concentration dependences of &#x394;G is plotted for different force values (symbols are explained in A and B, respectively). (C) &#x394;GEI is shown on the [C; f] plane, and phase boundary is quantitatively described (white zone between red and green regions). The blue dashed line is the theoretical prediction where fc &#x221D; log Cm [Equation 4]. Circles (o) indicate conditions with &#x394;GEI = 0 extracted from linear dependence of &#x394;G on log C in (A). It should be noted that the E state is under tension, and hence is different from the unfolded state upon thermal melting at f = 0: (D) Same as (C) except the results describe I&#x21CC;F transition. Here, the critical force describing the phase boundary is a nonlinear function of log Cm [Eq. 6]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Free energy differences between E and I, and I and F calculated from REMD simulations. The definition of states, E,I, and F, is based on the hydrogen bond interaction energies associated with each structural element (see main text for details).The I state is analogous to I2 in Fig. 3. &lt;strong&gt;(A and B)&lt;/strong&gt; Salt concentration dependences of &#x394;&lt;em&gt;G&lt;/em&gt; is plotted for different force values (symbols are explained in A and B, respectively). &lt;strong&gt;(C)&lt;/strong&gt; &#x394;&lt;em&gt;G&lt;/em&gt;&lt;sub&gt;EI&lt;/sub&gt; is shown on the [&lt;em&gt;C&lt;/em&gt;; &lt;em&gt;f&lt;/em&gt;] plane, and phase boundary is quantitatively described (white zone between red and green regions). The blue dashed line is the theoretical prediction where &lt;em&gt;f&lt;/em&gt;&lt;sub&gt;c&lt;/sub&gt; &#x221D; log &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;m&lt;/sub&gt; [Equation 4]. Circles (o) indicate conditions with &#x394;&lt;em&gt;G&lt;/em&gt;&lt;sub&gt;EI&lt;/sub&gt; = 0 extracted from linear dependence of &#x394;&lt;em&gt;G&lt;/em&gt; on log &lt;em&gt;C&lt;/em&gt; in (A). It should be noted that the E state is under tension, and hence is different from the unfolded state upon thermal melting at &lt;em&gt;f&lt;/em&gt; = 0: (D) Same as (C) except the results describe I&#x21CC;F transition. Here, the critical force describing the phase boundary is a nonlinear function of log &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;m&lt;/sub&gt; [Eq. 6].&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F8.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 8." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F8.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 8.</span> <p id="p-32" class="first-child">Free energy differences between E and I, and I and F calculated from REMD simulations. The definition of states, E,I, and F, is based on the hydrogen bond interaction energies associated with each structural element (see main text for details).The I state is analogous to I2 in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3</a>. <strong>(A and B)</strong> Salt concentration dependences of Δ<em>G</em> is plotted for different force values (symbols are explained in A and B, respectively). <strong>(C)</strong> Δ<em>G</em><sub>EI</sub> is shown on the [<em>C</em>; <em>f</em>] plane, and phase boundary is quantitatively described (white zone between red and green regions). The blue dashed line is the theoretical prediction where <em>f</em><sub>c</sub> ∝ log <em>C</em><sub>m</sub> [<a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a>]. Circles (o) indicate conditions with Δ<em>G</em><sub>EI</sub> = 0 extracted from linear dependence of Δ<em>G</em> on log <em>C</em> in (A). It should be noted that the E state is under tension, and hence is different from the unfolded state upon thermal melting at <em>f</em> = 0: (D) Same as (C) except the results describe I⇌F transition. Here, the critical force describing the phase boundary is a nonlinear function of log <em>C</em><sub>m</sub> [<a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Eq. 6</a>].</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The classification of the diagram of states based on the calculation of Δ<em>G</em><sub>EI</sub> and Δ<em>G</em><sub>IF</sub> is consistent with the <em>R</em><sub>g</sub> and <em>Q</em> phase diagrams (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figs. 8C</a> and 8D). At all values of <em>f</em>, Δ<em>G</em> depends linearly on log<em>C</em> over a broad range of <em>C</em> as shown in <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figs. 8A</a> and 8B. The midpoints of salt concentration, <em>C</em><sub>m</sub>s, at each <em>f</em>, determined using <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-24.gif"/></noscript></span></span>, leads to <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-25.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-26.gif"/></noscript></span></span>, respectively, for the E⇌I and I⇌F transitions. The forces <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-28.gif"/></noscript></span></span> are the critical forces needed to rupture the I and F states, respectively. The locus of points <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-30.gif"/></noscript></span></span>(<em>C</em><sub>m</sub> are shown as circles in <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figs. 8C</a> and 8D, respectively. It is clear that <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-31.gif"/></noscript></span></span> is linear in log <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-32.gif"/></noscript></span></span> along the phase boundary. In contrast, the dependence of <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-33.gif"/></noscript></span></span> on log <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-34.gif"/></noscript></span></span> is nonlinear. In what follows, we provide a theoretical interpretation of these results, which automatically shows that the difference in preferential ion interaction coefficients between states can be directly measured in LOT experiments [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>–<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>].</p><p id="p-34">The salt concentration dependence of RNA stability can be determined using the preferential interaction coefficient, defined as <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-35.gif"/></noscript></span></span>, where <em>C</em><sub>RNA</sub> is the concentration of RNA, and the chemical potential of the ions, μ, is a constant [<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. The free energy difference, Δ<em>G</em><sub><em>α</em>β</sub>(=<em>G</em><sub>β</sub>−<em>G</em><sub><em>α</em></sub>), between two states ± and β (such as E and I or I and F) can be expressed as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-9.gif"/></noscript></span></span>where <em>C</em><sub>0</sub> is an arbitrary reference value of the salt concentration [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]. Note that we consider only 1:1 monovalent salt such as KCl or NaCl for which ion activity can be related to salt concentration <em>C</em>. The factor of 2 in (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Eq. 1</a> arises from charge neutrality. The difference, ΔΓ<sub><em>α</em>β</sub>(=Γ−Γ<sub><em>α</em></sub>, is interpreted as an effective number of bound or excluded ions when the RNA molecule changes its conformation between the two states [<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>]. The free energy change upon application of an external force <em>f</em> is <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-36.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-37.gif"/></noscript></span></span>[<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>]. Thus,</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-10.gif"/></noscript></span></span><p id="p-35">In the [<em>C</em>, <em>f</em>] phase diagram, Δ<em>G</em><sub><em>α</em>β</sub> is 0 along the phase boundaries. Since the reference salt concentration <em>C</em><sub>0</sub> is arbitrary, we determined its value using Δ<em>G</em><sub><em>α</em>β</sub>(<em>C</em><sub>o</sub>, <em>f</em> = 0) = 0. Thus, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-38.gif"/></noscript></span></span> is the salt concentration at the midpoint of the transition at zero force. The determination of ΔΓ<sub><em>α</em>β</sub> using this procedure from single-molecule pulling data has been described in several key papers [<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>–<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. By adopting the procedure outlined in these studies and with our choice of <em>C</em><sub>0</sub>, we rearrange (<a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Eq. 2</a> to obtain the phase boundary at f ≠ 0,<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-11.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-39.gif"/></noscript></span></span> is the midpoint of salt concentration, and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-40.gif"/></noscript></span></span> is the critical force associated with the transition between states <em>α</em> and β. Measurement of <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-41.gif"/></noscript></span></span> using data from single-molecule pulling experiments has been used to obtain ΔΓ<sub><em>α</em>β</sub> for few systems that exhibit two-state transitions [<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a>‐<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>]. Here, we have adopted it for a PK exhibiting more complex unfolding transitions. The connection between the relation in <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Eq. 3</a>, derived elsewhere, to Clausius-Clapeyron equation was established recently by Saleh and coworkers [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>].</p><p id="p-36">It follows from <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Eq. 3</a> that the critical force, leading to the prediction that there should be a linear dependence between <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-42.gif"/></noscript></span></span> and the logarithm of the midpoint of the salt concentration if <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-43.gif"/></noscript></span></span> is a constant value along the phase boundary. In accord with the prediction in (<a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">Eq. 4</a>), we find that <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-44.gif"/></noscript></span></span> varies linearly with <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-45.gif"/></noscript></span></span> along the phase boundary separating E and I (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Fig. 8C</a>) where (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-46.gif"/></noscript></span></span>) ≈ 4.9nm over a broad range of <em>f</em><sub>c</sub> (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figs. 9A</a> and 9C).</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-12.gif"/></noscript></span>
</span><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Probability distributions of the extensions for states E and I (A), and I and F (B). All the conditions shown here are near the phase boundaries in the [C, f] plane (&#x394;GEI and &#x394;GIF between the two states are close to 0). Distances between peaks of the two distributions are shown. In the E&#x21CC;I transition (A), the mean distances are almost constant around 4.9 nm. On the other hand, in the I&#x21CC;F transition (B), the mean distance becomes smaller as the mechanical force weakens. (C) Dependence of &#x394;Ree on fc along the phase boundaries. In the I&#x21CC;F transition, the dependence can be fit using , where a = 0.14 nm/pN and b = 0.27 nm (solid line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-505056147" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;(A and B)&lt;/strong&gt; Probability distributions of the extensions for states E and I (A), and I and F (B). All the conditions shown here are near the phase boundaries in the [&lt;em&gt;C&lt;/em&gt;, &lt;em&gt;f&lt;/em&gt;] plane (&#x394;&lt;em&gt;G&lt;/em&gt;&lt;sub&gt;EI&lt;/sub&gt; and &#x394;&lt;em&gt;G&lt;/em&gt;&lt;sub&gt;IF&lt;/sub&gt; between the two states are close to 0). Distances between peaks of the two distributions are shown. In the E&#x21CC;I transition (A), the mean distances are almost constant around 4.9 nm. On the other hand, in the I&#x21CC;F transition (B), the mean distance becomes smaller as the mechanical force weakens. &lt;strong&gt;(C)&lt;/strong&gt; Dependence of &#x394;&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;ee&lt;/sub&gt; on &lt;em&gt;f&lt;/em&gt;&lt;sub&gt;c&lt;/sub&gt; along the phase boundaries. In the I&#x21CC;F transition, the dependence can be fit using , where a = 0.14 nm/pN and b = 0.27 nm (solid line).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F9.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 9." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F9.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/21/048801/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 9.</span> <p id="p-37" class="first-child"><strong>(A and B)</strong> Probability distributions of the extensions for states E and I (A), and I and F (B). All the conditions shown here are near the phase boundaries in the [<em>C</em>, <em>f</em>] plane (Δ<em>G</em><sub>EI</sub> and Δ<em>G</em><sub>IF</sub> between the two states are close to 0). Distances between peaks of the two distributions are shown. In the E⇌I transition (A), the mean distances are almost constant around 4.9 nm. On the other hand, in the I⇌F transition (B), the mean distance becomes smaller as the mechanical force weakens. <strong>(C)</strong> Dependence of Δ<em>R</em><sub>ee</sub> on <em>f</em><sub>c</sub> along the phase boundaries. In the I⇌F transition, the dependence can be fit using <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/F9/embed/inline-graphic-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/F9/embed/inline-graphic-47.gif"/></noscript></span></span>, where a = 0.14 nm/pN and b = 0.27 nm (solid line).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The observed nonlinearity of the phase boundary separating I and F over the entire concentration range (<a id="xref-fig-8-6" class="xref-fig" href="#F8">Fig. 8D</a>) arises because <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-48.gif"/></noscript></span></span> is not a constant but varies linearly with <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-49.gif"/></noscript></span></span> (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figs. 9B</a> and 9C). In giving from F↔I, there is only a rupture of few tertiary interactions and unfolding of the relatively unstable S2. Because this transition is not cooperative (interactions break in a sequential manner), we believe that a linear behavior over a limited force range is not unreasonable. In contrast, the unfolding of S1 is cooperative, like some hairpins studied using pulling experiments, in the E↔I transition, resulting in the constant value for Δ<em>R</em><sub>ee</sub> over the force range probed. Thus, using <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-50.gif"/></noscript></span></span> and (<a id="xref-disp-formula-4-3" class="xref-disp-formula" href="#disp-formula-4">Eq. 4</a>, we find that <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-51.gif"/></noscript></span></span> satisfies, <span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-14.gif"/></noscript></span></span> Note that if <em>a</em> is zero, then <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Eq. 5</a> reduces to <a id="xref-disp-formula-4-4" class="xref-disp-formula" href="#disp-formula-4">Eq. 4</a> with <em>b</em> = <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-52.gif"/></noscript></span></span> = <em>const</em>. Solving <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Eq. 5</a>, the nonlinear dependence of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-53.gif"/></noscript></span></span> is expressed as,<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/graphic-15.gif"/></noscript></span></span> The simulation data can be quantitatively fit using (<a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">Eq. 6</a>) (<a id="xref-fig-8-7" class="xref-fig" href="#F8">Fig. 8D</a>). In general, we expect <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-54.gif"/></noscript></span></span>depends on <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-55.gif"/></noscript></span></span>, and hence, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-56.gif"/></noscript></span></span>. It is worth noting that the estimation of ΔΓ<sub><em>α</em>β</sub> requires only the knowledge of how critical force varies with salt concentrations (Δ<em>G</em> = 0). For any given salt concentration, the best statistics is obtained at the critical force because at <em>f</em><sub>c</sub>, multiple transitions between the two states can be observed.</p><p id="p-39">From the coefficients in the dependence of <em>f</em><sub>c</sub> on log <em>C</em><sub>m</sub>, we can estimate ΔΓ which provides a quantitative estimation of the extent of ion-RNA interactions [<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>]. For the E⇌I transition, (Δ<em>R</em><sub>ee</sub>) ≈ 4.9 nm and the slope of the linear function is 1.7 pN, which leads to ΔΓ<sub>EI</sub> = 0.96 (<a id="xref-fig-8-8" class="xref-fig" href="#F8">Fig. 8C</a>). This indicates that upon the conformational change from unfolded to the intermediate state, one ion pair enters the atmosphere of the PK. For the transition between I and F, fitting the dependence of <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-57.gif"/></noscript></span></span> on <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-58.gif"/></noscript></span></span> with a = 0.14 nm/pN and b = 0.27 nm (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Fig. 9C</a>), we estimate ΔΓ<sub>IF</sub> = 1.8 using <a id="xref-disp-formula-6-3" class="xref-disp-formula" href="#disp-formula-6">Eq. 6</a> (<a id="xref-fig-8-9" class="xref-fig" href="#F8">Fig. 8D</a>). Consequently, the total number of excess cations effectively bound to folded RNA is expected to be around 2.8, a value that is in rough accord with the experimental estimate of 2.2 [<a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">30</a>]. More importantly, ΔΓ<sub><em>α</em>β</sub> can be measured using LOT and magnetic tweezer experiments [<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">39</a>–<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>], thus expanding the scope of such experiments to measure ion-RNA interactions. A recent illustration of the efficacy of single-molecule experiments is the study by Jacobson and Saleh who quantified ΔΓ from force measurements for an RNA hairpin [<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>].</p><p id="p-40">Our analysis is based on the assumption that ΔΓ is a constant independent of the salt concentration. However, it is known experimentally that ΔΓ deviates from a constant value at high salt concentration [<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>].</p><p id="p-41">In <a id="xref-fig-8-10" class="xref-fig" href="#F8">Fig. 8C and D</a>, theoretical lines (blue dotted) show such a deviation from Δ<em>G</em> = 0 of the simulation data (white region) at high salt concentration <em>C</em> ≳ 500mM). Therefore, we believe that our theoretical analysis is most accurate for <em>C</em> ≲ 500 mM. Our coarse-grained simulation reflects the nonlinearity of ΔΓ observed in previous experiments. Understanding the origin of the nonlinearity requires a microscopic theory that should take into account the effects of ion-ion correlations. For complex architectures such as PK or ribozymes, this does not appear straightforward.</p></div></div><div class="section" id="sec-7"><h2 class="">MODEL AND METHODS</h2><div id="sec-8" class="subsection"><h3>The three-interaction-site (TIS) model</h3><p id="p-42">We employed a variant of the three-interaction-site (TIS) model, a coarse-grained model first introduced by Hyeon and Thirumalai for simulating nucleic acids [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. The TIS model has been previously used to make several quantitative predictions for RNA molecules, ranging from hairpins to ribozymes with particular focus on folding and response to <em>f</em> [<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>–<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. More recently, we have incorporated the consequences of counter ion condensation into the TIS model, allowing us to predict thermodynamic properties of RNA hairpins and PK that are in remarkable agreement with experiments [<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a>]. In the TIS model, each nucleotide is represented by three coarse-grained spherical beads corresponding to phosphate (P), ribose sugar (S), and a base (B). Briefly, the effective potential energy (for details, see Ref. [<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">37</a>]) of a given RNA conformation is <em>U</em><sub>TIS</sub> = <em>U</em><sub>L</sub> + <em>U</em><sub>EV</sub> + <em>U</em><sub>ST</sub> + <em>U</em><sub>HB</sub> + <em>U</em><sub>EL</sub>, where <em>U</em><sub>L</sub> accounts for chain connectivity and angular rotation of the polynucleic acids, <em>U</em><sub>EV</sub> accounts for excluded volume interactions of each chemical group, and <em>U</em><sub>ST</sub> and <em>U</em><sub>HB</sub> are the base-stacking and HB interactions, respectively. Finally, the term <em>U</em><sub>EL</sub> corresponds to electrostatic interactions between phosphate groups.</p><p id="p-43">The determination of the parameters for use in simulations of the coarse-grained model of RNA was described in detail in a previous publication [<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">37</a>]. Briefly, we determined the stacking interactions by a learning procedure, which amounts to reproducing the measured melting temperature of dimers. We showed that a single choice of model parameters (stacking interactions, the Debye-HÜckel potential for electrostatic interactions, and structure-specific choice for hydrogen bonds) is sufficient to obtain detailed agreements with available experimental data for three different RNA molecules. Thus, the parameters are transferable and we use them verbatim in the present simulations. The determination of hydrogen bond interaction is predicated on the structure. Most of these are taken from the A-form RNA structure and hence can be readily used for any RNA molecule. Only the interaction lengths and angles in hydrogen-bonding interaction of noncanonical base pairs are based on the specific PDB structure, which in our case is the BWYV pseudoknot in this study. We list all the BYWV-PK-specific values used in our simulations in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table S1</a>.</p><p id="p-44">The repulsive electrostatic interactions between the phosphate groups is taken into account through the Debye-HÜckel theory <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-59.gif"/></noscript></span></span>, where the Debye length is <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-60.gif"/></noscript></span></span>. In the simulations, salt concentration can be varied by changing the ionic strength <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-61.gif"/></noscript></span></span> where q<sub>n</sub> is the charge of ion of type <em>n</em>, and ρ<sub>n</sub> is its number density. Following our earlier study [<a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">37</a>], we use an experimentally fitted function for the temperature-dependent dielectric constant ɛ(<em>T</em>) [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. The renormalized charge on the phosphate group is −q*<em>e</em>(q* &lt; 1). Because of the highly charged nature of the polyanion, counterions condense onto the RNA, thus effectively reducing the effective charge per phosphate. The extent of charge reduction can be calculated using the Oosawa-Manning theory. Charge renormalization, first used in the context of RNA folding by Heilman-Miller et al. [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>] and more recently incorporated into CG models for RNA by us [<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">37</a>] and others [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>], is needed to predict accurately the thermodynamics and kinetics of RNA folding [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]. The renormalized value of the charge on the P group is approximately given by <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-62.gif"/></noscript></span></span>, where the Bjerrum length is <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-63.gif"/></noscript></span></span> and b is the mean distance between the charges on the phosphate groups [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>]. The mean distance (b) between charges on RNA is difficult to estimate (except for rod-like polyelectrolytes) because of its shape fluctuations. Therefore, it should be treated as an adjustable parameter. In our previous study, we showed that b = 4.4 Å provides an excellent fit of the thermodynamics of RNA hairpins and the MMTV pseudoknot [<a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">37</a>]. We use the same value in the present study of BWYV as well. Thus, the force field used in this study is the same as in our previous study attesting to its transferability for simulating small RNA molecules.</p><p id="p-45">The native structure of the BWYV PK is taken from the PDB (437D) [<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>]. The structure contains 28 nucleotides. We added an additional guanosine monophosphate to both the 5’ and 3’ terminus. Thus, the simulated PK has 30 nucleotides. The mechanical force is applied to the ribose sugars of both the 5’– and 3’– terminus (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1A</a>).</p></div></div><div class="section" id="sec-9"><h2 class="">Simulation details</h2><p id="p-46">We performed Langevin dynamics simulations by solving the equation of motion, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-64.gif"/></noscript></span></span>, where m is mass; γ, the friction coefficient, depends on the size of the particle type (P, S, or B); and <strong><em>R</em></strong> is a Gaussian random force, which satisfies the fluctuation-dissipation relation. The [<em>C</em>, <em>f</em>] phase diagram is calculated at a somewhat elevated temperature, <em>T</em>γ50°C, in order to observe multiple folding-unfolding transitions when the monovalent salt concentration is varied over a wide range, from 5 to 1200 mM. The numerical integration is performed using the leap-frog algorithm with time step length δ<em>t</em><sub>L</sub> δ 0.05τ, where τ is the unit of time. We performed low friction Langevin dynamics simulations to enhance the rate of conformational sampling [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. For each condition (<em>C</em> and <em>f</em>), we performed 50 independent simulations for 6 * 10<sup>8</sup> time steps and the last two-third of trajectories are used for data analyses. In addition, we also performed a set of simulations using the replica-exchange molecular dynamics (REMD) to confirm the validity of the results obtained by the low friction Langevin dynamics simulations [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>]. In the REMD simulations, both <em>C</em> and <em>f</em> are treated as replica variables and are exchanged between 256 replicas, by covering <em>C</em> from 1 to 1200 mM and <em>f</em> from 0 to 20 pN. The replicas are distributed over 16 * 16 grid. The combined set of simulations using different protocols assures us that the simulations are converged at all conditions.</p><p id="p-47">Our simulations are done at constant forces under equilibrium conditions in which we observe multiple transitions between the various states. We have made comparison to LOT experiments, which are performed by pulling the RNA at a constant velocity. In general, if RNA is stretched at a constant velocity, then one has to be concerned about non-equilibrium effects. However, in optical tweezer experiments, the pulling speed is low enough so that the tension propagates uniformly throughout the RNA prior to initiation of unfolding [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>]. Thus, the unfolding of PK studied using LOT experiments [<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>] occurs at equilibrium, justifying comparisons to our explicit equilibrium simulations.</p></div><div class="section" id="sec-10"><h2 class="">Analysis of simulation data</h2><p id="p-48">To analyze the structural transitions from which the phase diagrams are determined, we stored snapshots every 10<sup>4</sup> time steps from all the trajectories. We computed the radius of gyration (<em>R</em><sub>g</sub>) and the end-to-end distance (<em>R</em><sub>ee</sub>) using the stored snapshots. The fraction of native contacts, <em>Q</em>, is calculated by counting the number of hydrogen bond (HB) interaction pairs. The assessment of HB formation is based on the instantaneous value of the HB interaction energy, <em>U</em><sub>HB</sub>. In our model, each HB interaction pair contributes up to <em>n</em><span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-65.gif"/></noscript></span></span> toward stability, where <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-66.gif"/></noscript></span></span> is −2.43 kcal/mol, corresponding to the stability of one HB, and <em>n</em> represents number of HB associated in the interaction [<a id="xref-ref-37-9" class="xref-bibr" href="#ref-37">37</a>]. A cutoff value <em>n</em><span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-67.gif"/></noscript></span></span> is defined to pick out contacts that are established. We use a value, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-68.gif"/></noscript></span></span> = −1.15 kcal/mol to obtain the diagram using <em>Q</em> as an order parameter. Modest changes in <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-69.gif"/></noscript></span></span> do not affect the results qualitatively.</p></div><div class="section" id="sec-11"><h2 class="">CONCLUDING REMARKS</h2><p id="p-49">Motivated by the relevance of force-induced transitions in PK to PRF, we have conducted extensive simulations of BWYV PK using a model of RNA, which predicts with near quantitative accuracy the thermodynamics at <em>f</em> = 0. The phase diagram in the [<em>C</em>, <em>f</em>] plane, which can be measured using single-molecule LOT pulling experiments, shows that the thermodynamics of rupture as <em>f</em> is increased, or the assembly of the PK as <em>f</em> is decreased, involves transitions between the extended, intermediate, and the folded states. The predicted linear relationship between <em>f</em><sub>c</sub> and log <em>C</em><sub>m</sub> [<a id="xref-disp-formula-4-5" class="xref-disp-formula" href="#disp-formula-4">Eq. 4</a>] or <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/06/21/048801/embed/inline-graphic-70.gif"/></noscript></span></span> [<a id="xref-disp-formula-6-4" class="xref-disp-formula" href="#disp-formula-6">eq. 6</a>] shows that ΔΓ can be directly measured in LOT experiments. The theoretical arguments leading to <a id="xref-disp-formula-4-6" class="xref-disp-formula" href="#disp-formula-4">Eqs. 4</a> and (<a id="xref-disp-formula-6-5" class="xref-disp-formula" href="#disp-formula-6">6</a>) are general. Thus, <em>f</em> can be used as a perturbant to probe ion-RNA interactions through direct measurement of ΔΓ.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-50">This work was completed when the authors were at the University of Maryland. We are grateful to Jon Din-man, Xin Li, Pavel Zhuravlev, Huong Vu, and Mauro Mugnai for comments on the manuscript. This work was supported in part by a grant from the National Science Foundation (CHE–1636424).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> <span class="em-link"><span class="em-addr">dave.thirumalai{at}gmail.com</span></span></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2>References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.1" data-doi="10.1016/j.cell.2014.03.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cech</span>,  <span class="cit-name-given-names">T. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Steitz</span>,  <span class="cit-name-given-names">J. A.</span></span> <span class="cit-article-title">The noncoding RNA revolution—trashing old rules to forge new ones</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">157</span>, <span class="cit-fpage">77</span>–<span class="cit-lpage">94</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2014.03.008.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.03.008%26rft_id%253Dinfo%253Apmid%252F24679528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.03.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24679528&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333968900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.2" data-doi="10.1006/jmbi.1999.3001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tinoco</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">How RNA folds</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-pub-date">1999</span>, <span class="cit-vol">293</span>, <span class="cit-fpage">271</span>–<span class="cit-lpage">281</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1006/jmbi.1999.3001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DTinoco%26rft.auinit1%253DI.%26rft.volume%253D293%26rft.issue%253D2%26rft.spage%253D271%26rft.epage%253D281%26rft.atitle%253DHow%2BRNA%2Bfolds.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1999.3001%26rft_id%253Dinfo%253Apmid%252F10550208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1999.3001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10550208&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083284300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[3].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-048801v2.3" data-doi="10.1016/j.sbi.2005.05.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woodson</span>,  <span class="cit-name-given-names">S. A.</span></span> <span class="cit-article-title">Structure and assembly of group I in-trons</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol</abbr>. <span class="cit-pub-date">2005</span>, <span class="cit-vol">15</span>, <span class="cit-fpage">324</span>–<span class="cit-lpage">330</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.sbi.2005.05.007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DWoodson%26rft.auinit1%253DS.%2BA.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D324%26rft.epage%253D330%26rft.atitle%253DStructure%2Band%2Bassembly%2Bof%2Bgroup%2BI%2Bintrons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2005.05.007%26rft_id%253Dinfo%253Apmid%252F15922592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sbi.2005.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15922592&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230381600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[4].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-048801v2.4" data-doi="10.1021/bi047314+"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hyeon</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">RNA and protein folding: common themes and variations</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">44</span>, <span class="cit-fpage">4957</span>–<span class="cit-lpage">70</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/bi047314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D44%26rft.spage%253D4957%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi047314%252B%26rft_id%253Dinfo%253Apmid%252F15794634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi047314+&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15794634&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228099000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.5" data-doi="10.1146/annurev.biophys.37.032807.125957"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S.-J.</span></span> <span class="cit-article-title">RNA folding conformational statistics, folding kinetics, and ion electrostatics</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Biophys</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">37</span>, <span class="cit-fpage">197</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1146/an-nurev.biophys.37.032807.125957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnnu%2BRev%2BBiophys%26rft.aulast%253DChen%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D37%26rft.spage%253D197%26rft.epage%253D214%26rft.atitle%253DRNA%2Bfolding%253A%2Bconformational%2Bstatistics%252C%2Bfolding%2Bkinetics%252C%2Band%2Bion%2Belectrostatics.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biophys.37.032807.125957%26rft_id%253Dinfo%253Apmid%252F18573079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biophys.37.032807.125957&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18573079&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258107500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.6" data-doi="10.1021/ar2000474"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woodson</span>,  <span class="cit-name-given-names">S. A.</span></span> <span class="cit-article-title">RNA folding pathways and the selfassembly of ribosomes</span>. <abbr class="cit-jnl-abbrev">Acc. Chem. Res</abbr>. <span class="cit-pub-date">2011</span>, <span class="cit-vol">44</span>, <span class="cit-fpage">1312</span>–<span class="cit-lpage">1319</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/ar2000474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAccounts%2Bof%2BChemical%2BResearch%26rft.stitle%253DAccounts%2Bof%2BChemical%2BResearch%26rft.aulast%253DWoodson%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.issue%253D12%26rft.spage%253D1312%26rft.epage%253D1319%26rft.atitle%253DRNA%2Bfolding%2Bpathways%2Band%2Bthe%2Bself-assembly%2Bof%2Bribosomes.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Far2000474%26rft_id%253Dinfo%253Apmid%252F21714483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ar2000474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21714483&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298575400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.7" data-doi="10.1017/S0033583506004446"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tinoco</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">TX Li</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">Determination of thermodynamics and kinetics of RNA reactions by force</span>. <abbr class="cit-jnl-abbrev">Q. Rev. Biophys</abbr>. <span class="cit-pub-date">2006</span>, <span class="cit-vol">39</span>, <span class="cit-fpage">325</span>–<span class="cit-lpage">360</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1017/S0033583506004446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuarterly%2Breviews%2Bof%2Bbiophysics%26rft.stitle%253DQ%2BRev%2BBiophys%26rft.aulast%253DCogdell%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D325%26rft.epage%253D360%26rft.atitle%253DDetermination%2Bof%2Bthermodynamics%2Band%2Bkinetics%2Bof%2BRNA%2Breactions%2Bby%2Bforce.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0033583506004446%26rft_id%253Dinfo%253Apmid%252F17040613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0033583506004446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17040613&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243367500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.8" data-doi="10.1016/j.sbi.2006.05.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Theimer</span>,  <span class="cit-name-given-names">C. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Feigon</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Structure and function of telomerase RNA</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol</abbr>. <span class="cit-pub-date">2006</span>, <span class="cit-vol">16</span>, <span class="cit-fpage">307</span>–<span class="cit-lpage">18</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.sbi.2006.05.005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DTheimer%26rft.auinit1%253DC.%2BA.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D307%26rft.epage%253D318%26rft.atitle%253DStructure%2Band%2Bfunction%2Bof%2Btelomerase%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2006.05.005%26rft_id%253Dinfo%253Apmid%252F16713250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sbi.2006.05.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16713250&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239082100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.9" data-doi="10.1016/0092-8674(89)90124-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Digard</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Inglis</span>,  <span class="cit-name-given-names">S. C.</span></span> <span class="cit-article-title">Characterization of an efficient coronavirus ribosomal frameshifting signal: requirement for an RNA pseudoknot</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">1989</span>, <span class="cit-vol">57</span>, <span class="cit-fpage">537</span>–<span class="cit-lpage">547</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/0092-8674(89)90124-4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBrierley%26rft.auinit1%253DI.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D537%26rft.epage%253D547%26rft.atitle%253DCharacterization%2Bof%2Ban%2Befficient%2Bcoronavirus%2Bribosomal%2Bframeshifting%2Bsignal%253A%2Brequirement%2Bfor%2Ban%2BRNA%2Bpseudoknot.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990124-4%26rft_id%253Dinfo%253Apmid%252F2720781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90124-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2720781&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U743600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Powers</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Noller</span>,  <span class="cit-name-given-names">H. F.</span></span> <span class="cit-article-title">A functional pseudoknot in 16S ribosomal RNA</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr>. <span class="cit-pub-date">1991</span>, <span class="cit-vol">10</span>, <span class="cit-fpage">2203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ%26rft.volume%253D10%26rft.spage%253D2203%26rft_id%253Dinfo%253Apmid%252F1712293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1712293&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FW47300031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.11" data-doi="10.1021/bi00162a001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">ten Dam</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pleij</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Draper</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Structural and functional aspects of RNA pseudoknots</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-pub-date">1992</span>, <span class="cit-vol">31</span>, <span class="cit-fpage">11665</span>–<span class="cit-lpage">11676</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/bi00162a001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Easton%2529%26rft.stitle%253DBiochemistry%2B%2528Easton%2529%26rft.aulast%253DDam%26rft.auinit1%253DE.%26rft.volume%253D31%26rft.issue%253D47%26rft.spage%253D11665%26rft.epage%253D11676%26rft.atitle%253DStructural%2Band%2Bfunctional%2Baspects%2Bof%2BRNA%2Bpseudoknots.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00162a001%26rft_id%253Dinfo%253Apmid%252F1280160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00162a001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1280160&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.12" data-doi="10.1371/journal.pbio.0030213"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Staple</span>,  <span class="cit-name-given-names">D. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Butcher</span>,  <span class="cit-name-given-names">S. E.</span></span> <span class="cit-article-title">Pseudoknots: RNA structures with diverse functions</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-pub-date">2005</span>, <span class="cit-vol">3</span>, <span class="cit-fpage">e213</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pbio.0030213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DStaple%26rft.auinit1%253DD.%2BW.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De213%26rft.epage%253De213%26rft.atitle%253DPseudoknots%253A%2BRNA%2Bstructures%2Bwith%2Bdiverse%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030213%26rft_id%253Dinfo%253Apmid%252F15941360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15941360&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-confproc" id="cit-048801v2.13" data-doi="10.1073/pnas.96.12.6621"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilley</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Blackburn</span>,  <span class="cit-name-given-names">E. H.</span></span> <span class="cit-article-title">The telomerase RNA pseudoknot is critical for the stable assembly of a catalytically active ribonucleoprotein</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-conf-date">1999</span>, <span class="cit-vol">96</span>, <span class="cit-fpage">6621</span>–<span class="cit-lpage">6625</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.96.12.6621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGilley%26rft.auinit1%253DD.%26rft.volume%253D96%26rft.issue%253D12%26rft.spage%253D6621%26rft.epage%253D6625%26rft.atitle%253DThe%2Btelomerase%2BRNA%2Bpseudoknot%2Bis%2Bcritical%2Bfor%2Bthe%2Bstable%2Bassembly%2Bof%2Ba%2Bcatalytically%2Bactive%2Bribonucleoprotein%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.12.6621%26rft_id%253Dinfo%253Apmid%252F10359761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTYvMTIvNjYyMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE2LzA2LzIxLzA0ODgwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.14" data-doi="10.1016/j.jmb.2007.05.058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">C.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tinoco,</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Characterization of the mechanical unfolding of RNA pseudoknots</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">375</span>, <span class="cit-fpage">511</span>–<span class="cit-lpage">528</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2007.05.058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DGreen%26rft.auinit1%253DL.%26rft.volume%253D375%26rft.issue%253D2%26rft.spage%253D511%26rft.epage%253D528%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Bmechanical%2Bunfolding%2Bof%2BRNA%2Bpseudoknots.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2007.05.058%26rft_id%253Dinfo%253Apmid%252F18021801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2007.05.058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18021801&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252103700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-confproc" id="cit-048801v2.15" data-doi="10.1073/pnas.1204114109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>,  <span class="cit-name-given-names">D. B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Foster</span>,  <span class="cit-name-given-names">D. A. N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Woodside,</span>  <span class="cit-name-given-names">M. T.</span></span> <span class="cit-article-title">Programmed —1 frameshifting efficiency correlates with RNA pseudoknot conformational plasticity, not resistance to mechanical unfolding</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-conf-date">2012</span>, <span class="cit-vol">109</span>, <span class="cit-fpage">16167</span>–<span class="cit-lpage">16172</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1204114109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1204114109%26rft_id%253Dinfo%253Apmid%252F22988073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQwLzE2MTY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDYvMjEvMDQ4ODAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-confproc" id="cit-048801v2.16" data-doi="10.1073/pnas.1403457111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">H.-K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fei,</span>  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bustamante'</span>  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>,  <span class="cit-name-given-names">R. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tinoco</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">A frameshifting stimulatory stem loop destabilizes the hybrid state and impedes riboso-mal translocation</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-conf-date">2014</span>, <span class="cit-vol">111</span>, <span class="cit-fpage">5538</span>–<span class="cit-lpage">5543</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1403457111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1403457111%26rft_id%253Dinfo%253Apmid%252F24706807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE1LzU1MzgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wNi8yMS8wNDg4MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.17" data-doi="10.1261/rna.676707"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wen</span>,  <span class="cit-name-given-names">J.-D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tinoco,</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Single-molecule mechanical unfolding and folding of a pseudoknot in human telomerase RNA</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">13</span>, <span class="cit-fpage">2175</span>–<span class="cit-lpage">2188</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1261/rna.676707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.676707%26rft_id%253Dinfo%253Apmid%252F17959928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMi8yMTc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDYvMjEvMDQ4ODAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[18].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-048801v2.18" data-doi="10.1016/j.virusres.2008.06.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giedroc</span>,  <span class="cit-name-given-names">D. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cornish</span>,  <span class="cit-name-given-names">P. V.</span></span> <span class="cit-article-title">Frameshifting RNA pseudoknots: structure and mechanism</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-pub-date">2009</span>, <span class="cit-vol">139</span>, <span class="cit-fpage">193</span>–<span class="cit-lpage">208</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2008.06.008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGiedroc%26rft.auinit1%253DD.%2BP.%26rft.volume%253D139%26rft.issue%253D2%26rft.spage%253D193%26rft.epage%253D208%26rft.atitle%253DFrameshifting%2BRNA%2Bpseudoknots%253A%2Bstructure%2Band%2Bmechanism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2008.06.008%26rft_id%253Dinfo%253Apmid%252F18621088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2008.06.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18621088&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264329300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.19" data-doi="10.1021/ja111281f"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">K. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Orzechowski</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fourmy</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Visscher</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Mechanical unfolding of the beet western yellow virus -1 frameshift signal</span>. <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc</abbr>. <span class="cit-pub-date">2011</span>, <span class="cit-vol">133</span>, <span class="cit-fpage">9775</span>–<span class="cit-lpage">9782</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/ja111281f</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.aulast%253DWhite%26rft.auinit1%253DK.%2BH.%26rft.volume%253D133%26rft.issue%253D25%26rft.spage%253D9775%26rft.epage%253D9782%26rft.atitle%253DMechanical%2Bunfolding%2Bof%2Bthe%2Bbeet%2Bwestern%2Byellow%2Bvirus%2B-1%2Bframeshift%2Bsignal.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja111281f%26rft_id%253Dinfo%253Apmid%252F21598975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ja111281f&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21598975&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292439700034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.20" data-doi="10.1002/wrna.1126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-article-title">Mechanisms and implications of programmed translational frameshifting</span>. <abbr class="cit-jnl-abbrev">WIREs RNA</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">3</span>, <span class="cit-fpage">661</span>–<span class="cit-lpage">673</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/wrna.1126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DWIREs%2BRNA%26rft.aulast%253DDinman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D661%26rft.epage%253D673%26rft.atitle%253DMechanisms%2Band%2Bimplications%2Bof%2Bprogrammed%2Btranslational%2Bframeshifting.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwrna.1126%26rft_id%253Dinfo%253Apmid%252F22715123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wrna.1126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22715123&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-confproc" id="cit-048801v2.21" data-doi="10.1073/pnas.0905046106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">K.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chou</span>,  <span class="cit-name-given-names">M.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tinoco</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Triplex structures in an RNA pseudoknot enhance mechanical stability and increase efficiency of -1 ri-bosomal frameshifting</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-conf-date">2009</span>, <span class="cit-vol">106</span>, <span class="cit-fpage">12706</span>–<span class="cit-lpage">11</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0905046106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0905046106%26rft_id%253Dinfo%253Apmid%252F19628688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzMxLzEyNzA2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDYvMjEvMDQ4ODAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.22" data-doi="10.1016/j.bpj.2013.09.036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Messieres</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">J.-C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Belew</span>,  <span class="cit-name-given-names">A. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Meskauskas</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">La Porta</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Singlemolecule measurements of the CCR5 mRNA unfolding pathways</span>. <abbr class="cit-jnl-abbrev">Biophys. J</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">106</span>, <span class="cit-fpage">244</span>–<span class="cit-lpage">252</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.bpj.2013.09.036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys.%2BJ%26rft.volume%253D106%26rft.spage%253D244%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2013.09.036%26rft_id%253Dinfo%253Apmid%252F24411256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2013.09.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24411256&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.23" data-doi="10.1006/jmbi.1999.2688"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Napthine</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liphardt</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bloys</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Routledge</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-article-title">Brierley, I. The role of RNA pseudoknot stem 1 length in the promotion of efficient -1 ribosomal frameshifting</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-pub-date">1999</span>, <span class="cit-vol">288</span>, <span class="cit-fpage">305</span>–<span class="cit-lpage">320</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1006/jmbi.1999.2688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNapthine%26rft.auinit1%253DS.%26rft.volume%253D288%26rft.issue%253D3%26rft.spage%253D305%26rft.epage%253D320%26rft.atitle%253DThe%2Brole%2Bof%2BRNA%2Bpseudoknot%2Bstem%2B1%2Blength%2Bin%2Bthe%2Bpromotion%2Bof%2Befficient%2B-1%2Bribosomal%2Bframeshifting.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1999.2688%26rft_id%253Dinfo%253Apmid%252F10329144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1999.2688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10329144&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080204600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[24].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-048801v2.24" data-doi="10.1128/MCB.21.24.8657-8670.2001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kontos</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Napthine</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Ribosomal pausing at a frameshifter RNA pseudoknot is sensitive to reading phase but shows little correlation with frameshift efficiency</span>. <abbr class="cit-jnl-abbrev">Mol. Cell Biol</abbr>. <span class="cit-pub-date">2001</span>, <span class="cit-vol">21</span>, <span class="cit-fpage">8657</span>–<span class="cit-lpage">8670</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/MCB.21.24.8657-8670.2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKontos%26rft.auinit1%253DH.%26rft.volume%253D21%26rft.issue%253D24%26rft.spage%253D8657%26rft.epage%253D8670%26rft.atitle%253DRibosomal%2BPausing%2Bat%2Ba%2BFrameshifter%2BRNA%2BPseudoknot%2BIs%2BSensitive%2Bto%2BReading%2BPhase%2Bbut%2BShows%2BLittle%2BCorrelation%2Bwith%2BFrameshift%2BEfficiency%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.24.8657-8670.2001%26rft_id%253Dinfo%253Apmid%252F11713298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8yNC84NjU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDYvMjEvMDQ4ODAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-confproc" id="cit-048801v2.25" data-doi="10.1073/pnas.96.25.14234"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">Y.-G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Maas</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">O’Neill</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rich</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Specific mutations in a viral RNA pseudoknot drastically change ribosomal frameshifting efficiency</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-conf-date">1999</span>, <span class="cit-vol">96</span>, <span class="cit-fpage">14234</span>–<span class="cit-lpage">14239</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.96.25.14234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKim%26rft.auinit1%253DY.-G.%26rft.volume%253D96%26rft.issue%253D25%26rft.spage%253D14234%26rft.epage%253D14239%26rft.atitle%253DSpecific%2Bmutations%2Bin%2Ba%2Bviral%2BRNA%2Bpseudoknot%2Bdrastically%2Bchange%2Bribosomal%2Bframeshifting%2Befficiency%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.25.14234%26rft_id%253Dinfo%253Apmid%252F10588689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjUvMTQyMzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wNi8yMS8wNDg4MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[26].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-048801v2.26" data-doi="10.1093/nar/gks1254"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mouzakis</span>,  <span class="cit-name-given-names">K. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lang</span>,  <span class="cit-name-given-names">A. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vander Meulen</span>,  <span class="cit-name-given-names">K. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Easterday</span>,  <span class="cit-name-given-names">P. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Butcher</span>,  <span class="cit-name-given-names">S. E.</span></span> <span class="cit-article-title">HIV-1 frameshift efficiency is primarily determined by the stability of base pairs positioned at the mRNA entrance channel of the ribosome</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2013</span>, <span class="cit-vol">41</span>, <span class="cit-fpage">1901</span>–<span class="cit-lpage">1913</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gks1254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1254%26rft_id%253Dinfo%253Apmid%252F23248007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1254&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23248007&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316351800050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[27].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-048801v2.27" data-doi="10.1038/nature13429"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belew</span>,  <span class="cit-name-given-names">A. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Meskauskas</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Musalgaonkar</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Advani</span>,  <span class="cit-name-given-names">V. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sulima</span>,  <span class="cit-name-given-names">S. O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kasprzak</span>,  <span class="cit-name-given-names">W. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>,  <span class="cit-name-given-names">B. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dinman</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-article-title">Ribosomal frameshifting in the CCR5 mRNA is regulated by miRNAs and the NMD pathway</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">512</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/na-ture13429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13429%26rft_id%253Dinfo%253Apmid%252F25043019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25043019&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340508200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-confproc" id="cit-048801v2.28" data-doi="10.1073/pnas.1321930111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Treffers</span>,  <span class="cit-name-given-names">E. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Napthine</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tas</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mark</span>,  <span class="cit-name-given-names">B. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">van Veelen</span>,  <span class="cit-name-given-names">P. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">van Hemert</span>,  <span class="cit-name-given-names">M. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Firthe</span>,  <span class="cit-name-given-names">A. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brierleye</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snijderc</span>,  <span class="cit-name-given-names">E. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Transactivation of programmed ribosomal frameshifting by a viral protein</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-conf-date">2014</span>, <span class="cit-vol">111</span>, <span class="cit-fpage">E2172</span>–<span class="cit-lpage">E2181</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1321930111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1321930111%26rft_id%253Dinfo%253Apmid%252F24825891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzIxL0UyMTcyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDYvMjEvMDQ4ODAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.29" data-doi="10.1016/S0301-4622(02)00177-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tionoco</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">The effect of force on thermodynamics and kinetics of single molecule reactions</span>. <abbr class="cit-jnl-abbrev">Biophys. Chem</abbr>. <span class="cit-pub-date">2002</span>, <span class="cit-vol">101</span>, <span class="cit-fpage">513</span>–<span class="cit-lpage">533</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0301-4622(02)00177-1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys.%2BChem%26rft.volume%253D101%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-4622%252802%252900177-1%26rft_id%253Dinfo%253Apmid%252F12488024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-4622(02)00177-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12488024&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.30" data-doi="10.1006/jmbi.1999.3464"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nixon</span>,  <span class="cit-name-given-names">P. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Giedroc</span>,  <span class="cit-name-given-names">D. P.</span></span> <span class="cit-article-title">Energetics of a strongly pH dependent RNA tertiary structure in a frameshifting pseudoknot</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-pub-date">2000</span>, <span class="cit-vol">296</span>, <span class="cit-fpage">659</span>–<span class="cit-lpage">71</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1006/jmbi.1999.3464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNixon%26rft.auinit1%253DP.%2BL.%26rft.volume%253D296%26rft.issue%253D2%26rft.spage%253D659%26rft.epage%253D671%26rft.atitle%253DEnergetics%2Bof%2Ba%2Bstrongly%2BpH%2Bdependent%2BRNA%2Btertiary%2Bstructure%2Bin%2Ba%2Bframeshifting%2Bpseudoknot.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1999.3464%26rft_id%253Dinfo%253Apmid%252F10669615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1999.3464&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10669615&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085698900026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.31" data-doi="10.1021/bi0616753"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soto</span>,  <span class="cit-name-given-names">A. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Misra</span>,  <span class="cit-name-given-names">V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Draper</span>,  <span class="cit-name-given-names">D. E.</span></span> <span class="cit-article-title">Tertiary structure of an RNA pseudoknot is stabilized by “diffuse” Mg<sup>2+</sup> ions</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">46</span>, <span class="cit-fpage">2973</span>–<span class="cit-lpage">83</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/bi0616753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Easton%2529%26rft.stitle%253DBiochemistry%2B%2528Easton%2529%26rft.aulast%253DSoto%26rft.auinit1%253DA.%2BM.%26rft.volume%253D46%26rft.issue%253D11%26rft.spage%253D2973%26rft.epage%253D2983%26rft.atitle%253DTertiary%2Bstructure%2Bof%2Ban%2BRNA%2Bpseudoknot%2Bis%2Bstabilized%2Bby%2B%2526quot%253Bdiffuse%2526quot%253B%2BMg2%252B%2Bions.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi0616753%26rft_id%253Dinfo%253Apmid%252F17315982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi0616753&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17315982&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.32" data-doi="10.1038/6722"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Egli</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">J. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rich</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Minor groove RNA triplex in the crystal structure of a ribosomal frameshifting viral pseudoknot</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol</abbr>. <span class="cit-pub-date">1999</span>, <span class="cit-vol">6</span>, <span class="cit-fpage">285</span>–<span class="cit-lpage">292</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/6722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2Bbiology%26rft.stitle%253DNat%2BStruct%2BBiol%26rft.aulast%253DSu%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D292%26rft.atitle%253DMinor%2Bgroove%2BRNA%2Btriplex%2Bin%2Bthe%2Bcrystal%2Bstructure%2Bof%2Ba%2Bribosomal%2Bframeshifting%2Bviral%2Bpseudoknot.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F6722%26rft_id%253Dinfo%253Apmid%252F10074948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/6722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10074948&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078970100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.33" data-doi="10.1016/j.bpj.2011.05.050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">Z.-J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S.-J.</span></span> <span class="cit-article-title">Salt contribution to RNA tertiary structure folding stability</span>. <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">101</span>, <span class="cit-fpage">176</span>–<span class="cit-lpage">87</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.bpj.2011.05.050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DTan%26rft.auinit1%253DZ.%2BJ.%26rft.volume%253D101%26rft.issue%253D1%26rft.spage%253D176%26rft.epage%253D187%26rft.atitle%253DSalt%2Bcontribution%2Bto%2BRNA%2Btertiary%2Bstructure%2Bfolding%2Bstability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2011.05.050%26rft_id%253Dinfo%253Apmid%252F21723828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2011.05.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21723828&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292571700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-confproc" id="cit-048801v2.34" data-doi="10.1073/pnas.0408314102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hyeon</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Mechanical unfolding of RNA hairpins</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-conf-date">2005</span>, <span class="cit-vol">102</span>, <span class="cit-fpage">6789</span>–<span class="cit-lpage">94</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0408314102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0408314102%26rft_id%253Dinfo%253Apmid%252F15749822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE5LzY3ODkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wNi8yMS8wNDg4MDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.35" data-doi="10.1021/j100129a032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">C. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Record Jr</span>,  <span class="cit-name-given-names">M. T.</span></span> <span class="cit-article-title">Salt dependence of oligoion-polyion binding: a thermodynamic description based on preferential interaction coefficients</span>. <abbr class="cit-jnl-abbrev">J. Phys. Chem.</abbr> <span class="cit-pub-date">1993</span>, <span class="cit-vol">97</span>, <span class="cit-fpage">7116</span>–<span class="cit-lpage">7126</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/j100129a032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhys.%2BChem.%26rft.volume%253D97%26rft.spage%253D7116%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fj100129a032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/j100129a032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.36" data-doi="10.7554/eLife.03406"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kaplan</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wen</span>,  <span class="cit-name-given-names">J.-D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lancaster</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wickersham</span>,  <span class="cit-name-given-names">C. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fredrick</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Noller</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tinoco</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C. J.</span></span> <span class="cit-article-title">Direct measurement of the mechanical work during translocation by the ribosome</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">3</span>, <span class="cit-fpage">e03406</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.7554/eLife.03406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.03406%26rft_id%253Dinfo%253Apmid%252F25114092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.03406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25114092&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.37" data-doi="10.1021/jp401087x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denesyuk</span>,  <span class="cit-name-given-names">N. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Coarse-grained model for predicting RNA folding thermodynamics</span>. <abbr class="cit-jnl-abbrev">J. Phys. Chem. B</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">117</span>, <span class="cit-fpage">4901</span>–<span class="cit-lpage">11</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/jp401087x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhys.%2BChem.%2BB%26rft.volume%253D117%26rft.spage%253D4901%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjp401087x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jp401087x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-confproc" id="cit-048801v2.38" data-doi="10.1073/pnas.0906625106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cho</span>,  <span class="cit-name-given-names">S. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pincus</span>,  <span class="cit-name-given-names">D. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Assembly mechanisms of RNA pseudoknots are determined by the stabilities of constituent secondary structures</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-conf-date">2009</span>, <span class="cit-vol">106</span>, <span class="cit-fpage">17349</span>–<span class="cit-lpage">54</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0906625106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0906625106%26rft_id%253Dinfo%253Apmid%252F19805055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQxLzE3MzQ5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDYvMjEvMDQ4ODAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.39" data-doi="10.1093/nar/gkm1038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Todd</span>,  <span class="cit-name-given-names">B. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rau</span>,  <span class="cit-name-given-names">D. C.</span></span> <span class="cit-article-title">Interplay of ion binding and attraction in DNA condensed by multivalent cations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2007</span>, <span class="cit-vol">36</span>, <span class="cit-fpage">501</span>–<span class="cit-lpage">510</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkm1038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm1038%26rft_id%253Dinfo%253Apmid%252F18048417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm1038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18048417&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253021600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[40].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-048801v2.40" data-doi="10.1103/Phys-RevE.77.031916"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Marko</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-article-title">Maxwell relations for single-DNA experiments: Monitoring protein binding and double-helix torque with force-extension measurements</span>. <abbr class="cit-jnl-abbrev">Phys. Rev. E</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">77</span>, <span class="cit-fpage">031916</span>–<span class="cit-lpage">9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1103/Phys-RevE.77.031916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BE%26rft.volume%253D77%26rft.spage%253D031916%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhys-RevE.77.031916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/Phys-RevE.77.031916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.41" data-doi="10.1021/ja500094z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dittmore</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Landy</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Molzon</span>,  <span class="cit-name-given-names">A. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Saleh</span>,  <span class="cit-name-given-names">O. A.</span></span> <span class="cit-article-title">Single-molecule methods for ligand counting: Linking ion uptake to DNA hairpin folding</span>. <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">136</span>, <span class="cit-fpage">5974</span>–<span class="cit-lpage">5980</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/ja500094z</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BChem.%2BSoc%26rft.volume%253D136%26rft.spage%253D5974%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja500094z%26rft_id%253Dinfo%253Apmid%252F24694039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ja500094z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24694039&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.42" data-doi="10.1021/jp511555g"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>,  <span class="cit-name-given-names">D. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Saleh</span>,  <span class="cit-name-given-names">O. A.</span></span> <span class="cit-article-title">Measuring the differential stoichiometry and energetics of ligand binding to macromolecules by single-molecule force spectroscopy: an extended theory</span>. <abbr class="cit-jnl-abbrev">J. Phys. Chem. B</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">119</span>, <span class="cit-fpage">1930</span>–<span class="cit-lpage">1938</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/jp511555g</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhys.%2BChem.%2BB%26rft.volume%253D119%26rft.spage%253D1930%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjp511555g%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jp511555g&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.43" data-doi="10.1093/nar/gkw196"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>,  <span class="cit-name-given-names">D. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Saleh</span>,  <span class="cit-name-given-names">O. A.</span></span> <span class="cit-article-title">Quantifying the ion atmosphere of unfolded, single-stranded nucleic acids using equilibrium dialysis and single-molecule methods</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2016</span>, <span class="cit-vol">44</span>, <span class="cit-fpage">3763</span>–<span class="cit-lpage">3771</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw196%26rft_id%253Dinfo%253Apmid%252F27036864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27036864&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.44" data-doi="10.1016/S0006-3495(94)80542-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bond</span>,  <span class="cit-name-given-names">J. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">C. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Record Jr</span>,  <span class="cit-name-given-names">M. T.</span></span> <span class="cit-article-title">Conformational transitions of duplex and triplex nucleic acid helices: thermodynamic analysis of effects of salt concentration on stability using preferential interaction coefficients</span>. <abbr class="cit-jnl-abbrev">Biophys. J</abbr>. <span class="cit-pub-date">1994</span>, <span class="cit-vol">67</span>, <span class="cit-fpage">825</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0006-3495(94)80542-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DBond%26rft.auinit1%253DJ%2BP%26rft.volume%253D67%26rft.issue%253D2%26rft.spage%253D825%26rft.epage%253D836%26rft.atitle%253DConformational%2Btransitions%2Bof%2Bduplex%2Band%2Btriplex%2Bnucleic%2Bacid%2Bhelices%253A%2Bthermodynamic%2Banalysis%2Bof%2Beffects%2Bof%2Bsalt%2Bconcentration%2Bon%2Bstability%2Busing%2Bpreferential%2Binteraction%2Bcoefficients.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0006-3495%252894%252980542-9%26rft_id%253Dinfo%253Apmid%252F7948695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0006-3495(94)80542-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7948695&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NY82300035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.45" data-doi="10.1016/S0065-3233(08)60655-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Record Jr</span>,  <span class="cit-name-given-names">M. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">C. F.</span></span> <span class="cit-article-title">Analysis of effects of salts and uncharged solutes on protein and nucleic acid equilibria and processes: a practical guide to recognizing and interpreting polyelectrolyte effects, Hofmeister effects, and osmotic effects of salts</span>. <abbr class="cit-jnl-abbrev">Adv. Protein Chem</abbr> <span class="cit-pub-date">1998</span>, <span class="cit-vol">51</span>, <span class="cit-fpage">281</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bprotein%2Bchemistry%26rft.stitle%253DAdv%2BProtein%2BChem%26rft.aulast%253DRecord%26rft.auinit1%253DM.%2BT.%26rft.volume%253D51%26rft.spage%253D281%26rft.epage%253D353%26rft.atitle%253DAnalysis%2Bof%2Beffects%2Bof%2Bsalts%2Band%2Buncharged%2Bsolutes%2Bon%2Bprotein%2Band%2Bnucleic%2Bacid%2Bequilibria%2Band%2Bprocesses%253A%2Ba%2Bpractical%2Bguide%2Bto%2Brecognizing%2Band%2Binterpreting%2Bpolyelectrolyte%2Beffects%252C%2BHofmeister%2Beffects%252C%2Band%2Bosmotic%2Beffects%2Bof%2Bsalts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3233%252808%252960655-5%26rft_id%253Dinfo%253Apmid%252F9615173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-3233(08)60655-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9615173&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074625600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.46" data-doi="10.1017/S003358350000202X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Record Jr</span>,  <span class="cit-name-given-names">M. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">C. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lohman</span>,  <span class="cit-name-given-names">T. M.</span></span> <span class="cit-article-title">Thermodynamic analysis of ion effects on the binding and conformational equilibria of proteins and nucleic acids: the roles of ion association or release, screening, and ion effects on water activity</span>. <abbr class="cit-jnl-abbrev">Q. Rev. Biophys</abbr>. <span class="cit-pub-date">1978</span>, <span class="cit-vol">11</span>, <span class="cit-fpage">103</span>–<span class="cit-lpage">178</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1017/S003358350000202X</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuarterly%2Breviews%2Bof%2Bbiophysics%26rft.stitle%253DQ%2BRev%2BBiophys%26rft.aulast%253DRecord%26rft.auinit1%253DM.%2BT.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D103%26rft.epage%253D178%26rft.atitle%253DThermodynamic%2Banalysis%2Bof%2Bion%2Beffects%2Bon%2Bthe%2Bbinding%2Band%2Bconformational%2Bequilibria%2Bof%2Bproteins%2Band%2Bnucleic%2Bacids%253A%2Bthe%2Broles%2Bof%2Bion%2Bassociation%2Bor%2Brelease%252C%2Bscreening%252C%2Band%2Bion%2Beffects%2Bon%2Bwater%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS003358350000202X%26rft_id%253Dinfo%253Apmid%252F353875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S003358350000202X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=353875&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978FJ19600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.47" data-doi="10.1021/ja2092823"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biyun</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cho</span>,  <span class="cit-name-given-names">S. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Folding of human telomerase RNA pseudoknot using ion-jump and temperature-quench simulations</span>. <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc</abbr>. <span class="cit-pub-date">2011</span>, <span class="cit-vol">133</span>, <span class="cit-fpage">20634</span>–<span class="cit-lpage">43</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/ja2092823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.aulast%253DBiyun%26rft.auinit1%253DS.%26rft.volume%253D133%26rft.issue%253D50%26rft.spage%253D20634%26rft.epage%253D20643%26rft.atitle%253DFolding%2Bof%2Bhuman%2Btelomerase%2BRNA%2Bpseudoknot%2Busing%2Bion-jump%2Band%2Btemperature-quench%2Bsimulations.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja2092823%26rft_id%253Dinfo%253Apmid%252F22082261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ja2092823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22082261&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298713600086&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[48].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-048801v2.48" data-doi="10.1021/ja408595e"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">J.-C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Kinetics of allosteric transitions in S-adenosylmethionine riboswitch are accurately predicted from the folding landscape</span>. <abbr class="cit-jnl-abbrev">J. Am. Chem. Soc</abbr>. <span class="cit-pub-date">2013</span>, <span class="cit-vol">135</span>, <span class="cit-fpage">16641</span>–<span class="cit-lpage">50</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1021/ja408595e</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BChem.%2BSoc%26rft.volume%253D135%26rft.spage%253D16641%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja408595e%26rft_id%253Dinfo%253Apmid%252F24087850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ja408595e&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24087850&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.49" data-doi="10.1038/nchem.2330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denesyuk</span>,  <span class="cit-name-given-names">N. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">How do metal ions direct ribozyme folding?</span> <abbr class="cit-jnl-abbrev">Nature Chem</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">7</span>, <span class="cit-fpage">793</span>–<span class="cit-lpage">801</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nchem.2330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BChem%26rft.volume%253D7%26rft.spage%253D793%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnchem.2330%26rft_id%253Dinfo%253Apmid%252F26391078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nchem.2330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26391078&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malmberg</span>,  <span class="cit-name-given-names">C. G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Maryott</span>,  <span class="cit-name-given-names">A. A.</span></span> <span class="cit-article-title">Dielectric constant of water from 0° to 100° C</span>. <abbr class="cit-jnl-abbrev">J. Res. Nat. Bur. Stand</abbr>. <span class="cit-pub-date">1956</span>, <span class="cit-vol">56</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[51].</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.51" data-doi="10.1006/jmbi.2001.4437"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heilman-Miller</span>,  <span class="cit-name-given-names">S. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Woodson</span>,  <span class="cit-name-given-names">S. A.</span></span> <span class="cit-article-title">Role of counterion condensation in folding of the Tetrahymena ribozyme. I. Equilibrium stabilization by cations</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-pub-date">2001</span>, <span class="cit-vol">306</span>, <span class="cit-fpage">1157</span>–<span class="cit-lpage">1166</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1006/jmbi.2001.4437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHeilman-Miller%26rft.auinit1%253DS.%2BL.%26rft.volume%253D306%26rft.issue%253D5%26rft.spage%253D1157%26rft.epage%253D1166%26rft.atitle%253DRole%2Bof%2Bcounterion%2Bcondensation%2Bin%2Bfolding%2Bof%2Bthe%2BTetrahymena%2Bribozyme.%2BI.%2BEquilibrium%2Bstabilization%2Bby%2Bcations.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2001.4437%26rft_id%253Dinfo%253Apmid%252F11237624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2001.4437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237624&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167502500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[52].</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.52" data-doi="10.1016/j.bpj.2014.01.042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">R. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Noel</span>,  <span class="cit-name-given-names">J. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Whitford</span>,  <span class="cit-name-given-names">P. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mo-hanty</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sanbonmatsu</span>,  <span class="cit-name-given-names">K. Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Onuchic</span>,  <span class="cit-name-given-names">J. N.</span></span> <span class="cit-article-title">Reduced model captures Mg<sup>2+</sup>-RNA interaction free energy of riboswitches</span>. <abbr class="cit-jnl-abbrev">Biophys. J</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">106</span>, <span class="cit-fpage">1508</span>–<span class="cit-lpage">1519</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.bpj.2014.01.042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys.%2BJ%26rft.volume%253D106%26rft.spage%253D1508%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2014.01.042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2014.01.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[53].</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.53" data-doi="10.1103/PhysRevLett.114.258105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">R. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Noel</span>,  <span class="cit-name-given-names">J. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mandic</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Whitford</span>,  <span class="cit-name-given-names">P. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sanbonmatsu</span>,  <span class="cit-name-given-names">K. Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mohanty</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Onuchic</span>,  <span class="cit-name-given-names">J. N.</span></span> <span class="cit-article-title">Generalized Manning condensation model captures the RNA ion atmosphere</span>. <abbr class="cit-jnl-abbrev">Phys. Rev. Lett.</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">114</span>, <span class="cit-fpage">258105</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1103/PhysRevLett.114.258105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhys.%2BRev.%2BLett.%26rft.volume%253D114%26rft.spage%253D258105%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevLett.114.258105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevLett.114.258105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[54].</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.54" data-doi="10.1006/jmbi.2001.4660"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heilman-Miller</span>,  <span class="cit-name-given-names">S. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Woodson</span>,  <span class="cit-name-given-names">S. A.</span></span> <span class="cit-article-title">Role of counterion condensation in folding of the Tetrahymena ribozyme II. Counterion-dependence of folding kinetics</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-pub-date">2001</span>, <span class="cit-vol">309</span>, <span class="cit-fpage">57</span>–<span class="cit-lpage">68</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1006/jmbi.2001.4660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHeilman-Miller%26rft.auinit1%253DS.%2BL.%26rft.volume%253D309%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D68%26rft.atitle%253DRole%2Bof%2Bcounterion%2Bcondensation%2Bin%2Bfolding%2Bof%2Bthe%2BTetrahymena%2Bribozyme.%2BII.%2BCounterion-dependence%2Bof%2Bfolding%2Bkinetics.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2001.4660%26rft_id%253Dinfo%253Apmid%252F11491301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2001.4660&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11491301&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169034800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[55].</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.55" data-doi="10.1063/1.1672157"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manning</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Limiting Laws and Counterion Condensation in Polyelectrolyte Solutions I. Colligative Properties</span>. <abbr class="cit-jnl-abbrev">J. Chem. Phys</abbr>. <span class="cit-pub-date">1969</span>, <span class="cit-vol">51</span>, <span class="cit-fpage">924</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1063/1.1672157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BPhys%26rft.volume%253D51%26rft.spage%253D924%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.1672157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1063/1.1672157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[56].</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.56" data-doi="10.1002/bip.360320610"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Honeycutt</span>,  <span class="cit-name-given-names">J. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">The nature of folded states of globular proteins</span>. <abbr class="cit-jnl-abbrev">Biopolymers</abbr> <span class="cit-pub-date">1992</span>, <span class="cit-vol">32</span>, <span class="cit-fpage">695</span>–<span class="cit-lpage">709</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/bip.360320610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiopolymers%26rft.stitle%253DBiopolymers%26rft.aulast%253DHoneycutt%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D32%26rft.issue%253D6%26rft.spage%253D695%26rft.epage%253D709%26rft.atitle%253DThe%2Bnature%2Bof%2Bfolded%2Bstates%2Bof%2Bglobular%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbip.360320610%26rft_id%253Dinfo%253Apmid%252F1643270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bip.360320610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1643270&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HU92400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[57].</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.57" data-doi="10.1016/S0009-2614(99)01123-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sugita</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Okamoto</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Replica-exchange molecular dynamics method for protein folding</span>. <abbr class="cit-jnl-abbrev">Chem. Phys. Lett</abbr>. <span class="cit-pub-date">1999</span>, <span class="cit-vol">314</span>, <span class="cit-fpage">141</span>–<span class="cit-lpage">151</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0009-2614(99)01123-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem.%2BPhys.%2BLett%26rft.volume%253D314%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0009-2614%252899%252901123-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0009-2614(99)01123-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[58].</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-048801v2.58" data-doi="10.1016/j.str.2006.09.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hyeon</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dima</span>,  <span class="cit-name-given-names">R. I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thirumalai</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Pathways and kinetic barriers in mechanical unfolding and refolding of RNA and proteins</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">14</span>, <span class="cit-fpage">1633</span>–<span class="cit-lpage">1645</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.str.2006.09.002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DHyeon%26rft.auinit1%253DC.%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D1633%26rft.epage%253D1645%26rft.atitle%253DPathways%2Band%2Bkinetic%2Bbarriers%2Bin%2Bmechanical%2Bunfolding%2Band%2Brefolding%2Bof%2BRNA%2Band%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2006.09.002%26rft_id%253Dinfo%253Apmid%252F17098189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2006.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17098189&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F21%2F048801.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/048801v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F048801v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/060020v1" title="Mechanistic Modeling and Analysis of the Mammalian Unfolded Protein Response" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/059980v1" title="Epigenetic divergence is sufficient to trigger heterosis in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;June 21, 2016.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/048801v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/048801v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/048801v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/048801v2" class="active" data-icon-position="" data-hide-link-title="0">Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/16542" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-dL24TxEBZy38es61O6xMbClNdyIMZLbQl-j9hr05qeU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="16542" id="node16542--2" data-pisa="biorxiv;048801v2" data-pisa-master="biorxiv;048801" data-seqnum="16542" data-apath="/biorxiv/early/2016/06/21/048801.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Naoto</span> <span class="nlm-surname">Hori</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Natalia A.</span> <span class="nlm-surname">Denesyuk</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">D.</span> <span class="nlm-surname">Thirumalai</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">048801; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/048801 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/048801v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;title=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;title=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;text=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;title=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;t=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;title=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;title=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="16542" id="node16542--3" data-pisa="biorxiv;048801v2" data-pisa-master="biorxiv;048801" data-seqnum="16542" data-apath="/biorxiv/early/2016/06/21/048801.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Salt Effects on the Thermodynamics of a Frameshifting RNA Pseudoknot under Tension</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Naoto</span> <span class="nlm-surname">Hori</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Natalia A.</span> <span class="nlm-surname">Denesyuk</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">D.</span> <span class="nlm-surname">Thirumalai</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">048801; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/048801 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension&amp;rft.volume&amp;rft.issue&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Hori%2C+Naoto&amp;rft.au=Denesyuk%2C+Natalia+A.&amp;rft.au=Thirumalai%2C+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/16542/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/16542/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/16542/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/16542/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/16542/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/16542/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/16542/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/16542/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/16542/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/16542/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/16542/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/16542/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;count=horizontal&amp;via=&amp;text=Salt%20Effects%20on%20the%20Thermodynamics%20of%20a%20Frameshifting%20RNA%20Pseudoknot%20under%20Tension&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/048801v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/048801v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/048801v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biophysics" class="highlight" data-icon-position="" data-hide-link-title="0">Biophysics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
