<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/18/2020.02.19.955484.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.19.955484v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Potential T-cell and B-cell Epitopes of 2019-nCoV"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.19.955484"/>
  <meta name="DC.Date" content="2020-03-18"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data. Significance Statement The novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV."/>
  <meta name="DC.Contributor" content="Ethan Fast"/>
  <meta name="DC.Contributor" content="Russ B. Altman"/>
  <meta name="DC.Contributor" content="Binbin Chen"/>
  <meta name="article:published_time" content="2020-03-18"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Potential T-cell and B-cell Epitopes of 2019-nCoV"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.19.955484v2"/>
  <meta name="citation_id" content="2020.02.19.955484v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.19.955484v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.19.955484v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.19.955484v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.19.955484"/>
  <meta name="citation_num_pages" content="9"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.19.955484"/>
  <meta name="citation_author" content="Ethan Fast"/>
  <meta name="citation_author_institution" content="Nash"/>
  <meta name="citation_author" content="Russ B. Altman"/>
  <meta name="citation_author_institution" content="Department of Bioengineering, Stanford University"/>
  <meta name="citation_author_institution" content="Department of Genetics, Stanford University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3859-2905"/>
  <meta name="citation_author" content="Binbin Chen"/>
  <meta name="citation_author_institution" content="Department of Genetics, Stanford University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2973-2718"/>
  <meta name="citation_reference" content="S Perlman, J Netland, Coronaviruses post-sars: update on replication and pathogenesis. Nat. reviews microbiology 7, 439–450 (2009)."/>
  <meta name="citation_reference" content="WMCR Group,, et al., State of knowledge and data gaps of middle east respiratory syndrome coronavirus (mers-cov) in humans. PLoS currents 5 (2013)."/>
  <meta name="citation_reference" content="C Huang, et al., Clinical features of patients infected with 2019 novel coronavirus in wuhan, china. The Lancet (2020)."/>
  <meta name="citation_reference" content="F Wu, et al., A new coronavirus associated with human respiratory disease in china. Nature, 1–8 (2020)."/>
  <meta name="citation_reference" content="P Zhou, et al., A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, 1–4 (2020)."/>
  <meta name="citation_reference" content="World Health Organization report (https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200216-sitrep-27-covid-19.pdf) (Accessed February 2020)."/>
  <meta name="citation_reference" content="CY Yong, HK Ong, SK Yeap, KL Ho, WS Tan, Recent advances in the vaccine development against middle east respiratory syndrome-coronavirus. Front. Microbiol. 10, 1781 (2019)."/>
  <meta name="citation_reference" content="CM Coleman, et al., Purified coronavirus spike protein nanoparticles induce coronavirus neutralizing antibodies in mice. Vaccine 32, 3169–3174 (2014)."/>
  <meta name="citation_reference" content="CT Tseng, et al., Immunization with sars coronavirus vaccines leads to pulmonary immunopathology on challenge with the sars virus. PloS one 7 (2012)."/>
  <meta name="citation_reference" content="R Rappuoli, S Black, DE Bloom, Vaccines and global health: In search of a sustainable model for vaccine development and delivery. Sci. Transl. Medicine 11, eaaw2888 (2019)."/>
  <meta name="citation_reference" content="SE Olsson, et al., Induction of immune memory following administration of a prophylactic quadrivalent human papillomavirus (hpv) types 6/11/16/18 l1 virus-like particle (vlp) vaccine. Vaccine 25, 4931–4939 (2007)."/>
  <meta name="citation_reference" content="D Suarez, S Schultz-Cherry, Immunology of avian influenza virus: a review. Dev. &amp;; Comp. Immunol. 24, 269–283 (2000)."/>
  <meta name="citation_reference" content="B Briney, et al., Tailored immunogens direct affinity maturation toward hiv neutralizing antibodies. Cell 166, 1459–1470 (2016)."/>
  <meta name="citation_reference" content="SR Pedersen, et al., Immunogenicity of hla class i and ii double restricted influenza a-derived peptides. PloS one 11 (2016)."/>
  <meta name="citation_reference" content="B Chen, et al., Predicting hla class ii antigen presentation through integrated deep learning. Nat. biotechnology 37, 1332–1343 (2019)."/>
  <meta name="citation_reference" content="R Lu, et al., Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. The Lancet (2020)."/>
  <meta name="citation_reference" content="V Jurtz, et al., Netmhcpan-4.0: improved peptide–mhc class i interaction predictions integrating eluted ligand and peptide binding affinity data. The J. Immunol. 199, 3360–3368 (2017)."/>
  <meta name="citation_reference" content="D Wrapp, et al., Cryo-em structure of the 2019-ncov spike in the prefusion conformation. Science (2020)."/>
  <meta name="citation_reference" content="R Yan, Y Zhang, Y Guo, L Xia, Q Zhou, Structural basis for the recognition of the 2019-ncov by human ace2. bioRxiv (2020)."/>
  <meta name="citation_reference" content="J Lan, et al., Crystal structure of the 2019-ncov spike receptor-binding domain bound with the ace2 receptor. bioRxiv (2020)."/>
  <meta name="citation_reference" content="JV Kringelum, C Lundegaard, O Lund, M Nielsen, Reliable b cell epitope predictions: impacts of method development and improved benchmarking. PLoS computational biology 8 (2012)."/>
  <meta name="citation_reference" content="Y Shu, J McCauley, Gisaid: Global initiative on sharing all influenza data–from vision to reality. Eurosurveillance 22 (2017)."/>
  <meta name="citation_reference" content="H Chen, et al., Response of memory cd8+ t cells to severe acute respiratory syndrome (sars) coronavirus in recovered sars patients and healthy individuals. The J. Immunol. 175, 591–598 (2005)."/>
  <meta name="citation_reference" content="M Zhou, et al., Screening and identification of severe acute respiratory syndrome-associated coronavirus-specific ctl epitopes. The J. Immunol. 177, 2138–2145 (2006)."/>
  <meta name="citation_reference" content="YP Tsao, et al., Hla-a* 0201 t-cell epitopes in severe acute respiratory syndrome (sars) coronavirus nucleocapsid and spike proteins. Biochem. biophysical research communications 344, 63–71 (2006)."/>
  <meta name="citation_reference" content="OW Ng, et al., Memory t cell responses targeting the sars coronavirus persist up to 11 years post-infection. Vaccine 34, 2008–2014 (2016)."/>
  <meta name="citation_reference" content="R Channappanavar, C Fett, J Zhao, DK Meyerholz, S Perlman, Virus-specific memory cd8 t cells provide substantial protection from lethal severe acute respiratory syndrome coronavirus infection. J. virology 88, 11034–11044 (2014)."/>
  <meta name="citation_reference" content="R Hua, Y Zhou, Y Wang, Y Hua, G Tong, Identification of two antigenic epitopes on sars-cov spike protein. Biochem. biophysical research communications 319, 929–935 (2004)."/>
  <meta name="citation_reference" content="H Yu, et al., Selection of sars-coronavirus-specific b cell epitopes by phage peptide library screening and evaluation of the immunological effect of epitope-based peptides on mice. Virology 359, 264–274 (2007)."/>
  <meta name="citation_reference" content="JP Guo, M Petric, W Campbell, PL McGeer, Sars corona virus peptides recognized by antibodies in the sera of convalescent cases. Virology 324, 251–256 (2004)."/>
  <meta name="citation_reference" content="W Song, M Gui, X Wang, Y Xiang, Cryo-em structure of the sars coronavirus spike glycoprotein in complex with its host cell receptor ace2. PLoS pathogens 14, e1007236 (2018)."/>
  <meta name="citation_reference" content="TJ Yang, et al., Cryo-em analysis of a feline coronavirus spike protein reveals a unique structure and camouflaging glycans. Proc. Natl. Acad. Sci. (2020)."/>
  <meta name="citation_reference" content="R Vita, et al., The immune epitope database (iedb) 3.0. Nucleic acids research 43, D405–D412 (2015)."/>
  <meta name="citation_reference" content="Y Hu, et al., Immunologic hierarchy, class ii mhc promiscuity, and epitope spreading of a melanoma helper peptide vaccine. Cancer Immunol. Immunother. 63, 779–786 (2014)."/>
  <meta name="citation_reference" content="Y Lv, Z Ruan, L Wang, B Ni, Y Wu, Identification of a novel conserved hla-a* 0201-restricted epitope from the spike protein of sars-cov. BMC immunology 10, 61 (2009)."/>
  <meta name="citation_reference" content="Y He, et al., Identification of immunodominant sites on the spike protein of severe acute respiratory syndrome (sars) coronavirus: implication for developing sars diagnostics and vaccines. The J. Immunol. 173, 4050–4057 (2004)."/>
  <meta name="citation_reference" content="Y He, Y Zhou, P Siddiqui, J Niu, S Jiang, Identification of immunodominant epitopes on the membrane protein of the severe acute respiratory syndrome-associated coronavirus. J. clinical microbiology 43, 3718–3726 (2005)."/>
  <meta name="citation_reference" content="Y He, et al., Mapping of antigenic sites on the nucleocapsid protein of the severe acute respiratory syndrome coronavirus. J. clinical microbiology 42, 5309–5314 (2004)."/>
  <meta name="citation_reference" content="SC Chow, et al., Specific epitopes of the structural and hypothetical proteins elicit variable humoral responses in sars patients. J. clinical pathology 59, 468–476 (2006)."/>
  <meta name="citation_reference" content="M Rasmussen, et al., Uncovering the peptide-binding specificities of hla-c: a general strategy to determine the specificity of any mhc class i molecule. The J. Immunol. 193, 4790–4802 (2014)."/>
  <meta name="citation_reference" content="F Zhou, et al., Deep sequencing of the mhc region in the chinese population contributes to studies of complex disease. Nat. genetics 48, 740–746 (2016)."/>
  <meta name="citation_reference" content="K Tajiri, et al., Analysis of the epitope and neutralizing capacity of human monoclonal antibodies induced by hepatitis b vaccine. Antivir. research 87, 40–49 (2010)."/>
  <meta name="citation_reference" content="P Bettencourt, et al., Identification of antigens presented by mhc for vaccines against tuberculosis. npj Vaccines 5, 1–14 (2020)."/>
  <meta name="citation_reference" content="T Schwede, J Kopp, N Guex, MC Peitsch, Swiss-model: an automated protein homology-modeling server. Nucleic acids research 31, 3381–3385 (2003)."/>
  <meta name="twitter:title" content="Potential T-cell and B-cell Epitopes of 2019-nCoV"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data. Significance Statement The novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV."/>
  <meta name="og-title" property="og:title" content="Potential T-cell and B-cell Epitopes of 2019-nCoV"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.19.955484v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data. Significance Statement The novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-18"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Potential T-cell and B-cell Epitopes of 2019-nCoV | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1201602 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-4">As of early March, 2019-nCoV has infected more than one hundred thousand people and claimed thousands of lives. 2019-nCoV is a novel form of coronavirus that causes COVID-19 and has high similarity with SARS-CoV. No approved vaccine yet exists for any form of coronavirus. Here we use computational tools from structural biology and machine learning to identify 2019-nCoV T-cell and B-cell epitopes based on viral protein antigen presentation and antibody binding properties. These epitopes can be used to develop more effective vaccines and identify neutralizing antibodies. We identified 405 viral peptides with good antigen presentation scores for both human MHC-I and MHC-II alleles, and two potential neutralizing B-cell epitopes near the 2019-nCoV spike protein receptor binding domain (440-460 and 494-506). Analyzing mutation profiles of 68 viral genomes from four continents, we identified 96 coding-change mutations. These mutations are more likely to occur in regions with good MHC-I presentation scores (p=0.02). No mutations are present near the spike protein receptor binding domain. Based on these findings, the spike protein is likely immunogenic and a potential vaccine candidate. We validated our computational pipeline with SARS-CoV experimental data.</p>
    <div id="sec-1" class="subsection">
     <p id="p-5">
      <strong>Significance Statement</strong> The novel coronavirus 2019-nCoV has affected more than 100 countries and continues to spread. There is an immediate need for effective vaccines that contain antigens which trigger responses from human T-cells and B-cells (known as epitopes). Here we identify potential T-cell epitopes through an analysis of human antigen presentation, as well as B-cell epitopes through an analysis of protein structure. We identify a list of top candidates, including an epitope located on 2019-nCoV spike protein that potentially triggers both T-cell and B-cell responses. Analyzing 68 samples, we observe that viral mutations are more likely to happen in regions with strong antigen presentation, a potential form of immune evasion. Our computational pipeline is validated with experimental data from SARS-CoV.
     </p>
    </div>
   </div>
   <div id="sec-2" class="subsection">
    <p id="p-7">
     <strong>C</strong>oronaviruses (CoV) first became widely known after the emergence of severe acute respiratory syndrome (SARS) in 2002. They are are positive, single-strand RNA viruses that often infect birds and a variety of mammals and sometimes migrate to humans (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). While coronaviruses that infect humans usually present mild symptoms (e.g. 15% of cases involving the common cold are caused by a coronavirus), several previous outbreaks, such as SARS and Middle East respiratory syndrome (MERS) (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>), have caused significant fatalities in infected populations (
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>).
    </p>
    <p id="p-8">In December 2019, several patients with a connection in Wuhan, China developed symptoms of viral pneumonia. Sequencing of viral RNA determined that these cases were caused by a novel coronavirus named 2019-nCoV or SARS-CoV-2 (
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, 
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). The virus has infected more than 100,000 people across 100 countries as of March 7th 2020 according to a World Health Organization report (
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). COVID-19, the disease caused by 2019-nCoV, has led to the deaths of thousands of patients and is clearly transmissible between humans (
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>).
    </p>
    <p id="p-9">The deployment of a vaccine against 2019-nCoV would arrest its infection rate and help protect vulnerable populations. However, no vaccine has yet passed beyond clinical trails for any coronavirus, whether in humans or other animals (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, 
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). In animal models, inactivated whole virus vaccines without focused immune epitopes offer incomplete protection (
     <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>). Vaccines for SARS-CoV and MERS-CoV have also resulted in hypersensitivity and immunopathologic responses in mice (
     <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, 
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). The diversity among coronavirus strains presents another challenge (
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, 
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>), as there are at least 6 distinct sub-groups in the coronavirus genus. And while SARS-CoV and 2019-nCoV share the same subgroup (2b), their genomes are only 77% identical (
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>). Understanding the shared and unique epitopes of 2019-nCoV will help the field design better vaccines or diagnostic tests for patients.
    </p>
    <p id="p-10">Both humoral immunity provided by B-cell antibodies and cellular immunity provided by T-cells are essential for effective vaccines (
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, 
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). Though humans can usually mount an antibody response against viruses, only neutralizing antibodies can fully block the entry of viruses into human cells (
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). The location of antibody binding sites on a viral protein strongly affects the body’s ability to produce neutralizing antibodies; for example, HIV has buried surface proteins that protect themselves from human antibodies (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). For this reason, it would be valuable to understand whether 2019-nCoV has potential antibody binding sites (B-cell epitopes) near their interacting surface with its known human entry receptor Angiotensin-Converting Enzyme 2 (ACE2).
    </p>
    <p id="p-11">Beyond neutralizing antibodies, human bodies also rely upon cytotoxic CD8 T-cells and helper CD4 T-cells to fully clear viruses. The presentation of viral peptides by human Major Histocompatibility Complex (MHC or HLA) class I and class II is essential for anti-viral T-cell responses (
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). In contrast to B-cell epitopes, T-cell epitopes can be located anywhere in a viral protein since human cells can process and present both intracellular and extracellular viral peptides (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>).
    </p>
    <p id="p-12">These immunology principles guide the design of our current study (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). First we focus on T-cell epitopes. 2019-nCoV carries 4 major structural proteins—spike (S), membrane (M), envelope (E) and nucleocapsid (N)—and at least 6 other open reading frames (ORFs) (
     <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>, 
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). All protein fragments have the potential to be presented by MHC-I or MHC-II and recognized by T-cells. We apply NetMHCpan4 (
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>) and MARIA (
     <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>), two artificial neural network algorithms, to predict antigen presentation and identify potential T-cell epitopes.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Computational pipeline to identify epitopes in 2019-nCoV. The 2019-nCoV genome codes for Spike (S), Membrane (M), Envelope (E), Nucleocapsid (N) and at least 6 other open reading frames (ORFs). The S protein is a target for antibodies and we model its 3D structure with Discotope2 to identify likely binding sites (B-cell epitopes). We scan peptide sequences from individual viral proteins with NetMHCPan4 and MARIA for MHC-I and MHC-II presentation to identify potential T-cell epitopes. Our MHC-I analysis include common alleles for HLA-A, HLA-B, and HLA-C, and our MHC-II analysis include common alleles of HLA-DR." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675726145" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Computational pipeline to identify epitopes in 2019-nCoV.&lt;;/span&gt;; The 2019-nCoV genome codes for Spike (S), Membrane (M), Envelope (E), Nucleocapsid (N) and at least 6 other open reading frames (ORFs). The S protein is a target for antibodies and we model its 3D structure with Discotope2 to identify likely binding sites (B-cell epitopes). We scan peptide sequences from individual viral proteins with NetMHCPan4 and MARIA for MHC-I and MHC-II presentation to identify potential T-cell epitopes. Our MHC-I analysis include common alleles for HLA-A, HLA-B, and HLA-C, and our MHC-II analysis include common alleles of HLA-DR.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F1.medium.gif" width="432" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F1.medium.gif" width="432" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig. 1.</span>
      <span class="caption-title">Computational pipeline to identify epitopes in 2019-nCoV.</span>
      <p id="p-13" class="first-child">The 2019-nCoV genome codes for Spike (S), Membrane (M), Envelope (E), Nucleocapsid (N) and at least 6 other open reading frames (ORFs). The S protein is a target for antibodies and we model its 3D structure with Discotope2 to identify likely binding sites (B-cell epitopes). We scan peptide sequences from individual viral proteins with NetMHCPan4 and MARIA for MHC-I and MHC-II presentation to identify potential T-cell epitopes. Our MHC-I analysis include common alleles for HLA-A, HLA-B, and HLA-C, and our MHC-II analysis include common alleles of HLA-DR.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-14">Next we focus on B-cell epitopes. The spike (S) protein is the main trans-membrane glycoprotein expressed on the surface of 2019-nCoV and is responsible for receptor binding and virion entry to cells (
     <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>, 
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>). We narrow our B-cell epitope search to S protein since it is the most likely target of human neutralizing antibodies. We first perform homology modeling to estimate the full 2019-nCoV spike protein structure and compare our results to partially solved spike protein structures (
     <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>–
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). Discopte2 uses a combination of amino acid sequences and protein surface properties to predict antibody binding sites given a protein structure (
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>). We use this tool to identify potential 2019-nCoV B-cell epitopes as the co-crystal structure of 2019-nCoV’s spike protein and antibody is not yet available.
    </p>
    <p id="p-15">Finally, we examine the viral mutation pattern. Global sequencing efforts (
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>) have enabled us to obtain a cohort of 68 2019-nCoV genomes from four continents. Based on the resulting viral mutation profile, we examine whether mutations are driven by the immune selection pressure. Highly mutated regions might be excluded from the vaccine candidate pool.
    </p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Results</h2>
    <div id="sec-4" class="subsection">
     <h3>T-cell epitopes of 2019-nCoV based on MHC presentation</h3>
     <p id="p-16">We scanned 11 2019-nCoV genes with NetMHCpan4 and MARIA to identify regions highly presentable by HLA complexes (human MHC). A epitope is labeled positive if presentable by more than one third of common alleles among the Chinese population. We display a summary of our T-cell epitope findings in 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. In 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a> we show regions of strong MHC presentation across HLA-A, HLA-B, HLA-C, and HLA-DR alleles, both at a medium threshold (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2A</a>), and at a high threshold (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2B</a>). Consistent with the similar finding from SARS-CoV, proteins ORF1AB, S, and E contain high numbers of presentable antigen sequences across both MHC-I and MHC-II (
      <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>). These highly presentable peptides on functional viral proteins provide a candidate pool for T-cell epitope identifications and epitope-focused vaccine design. The presentation scores of individual antigen peptides can be found in SI Appendix, Supplementary Tables 5 and 6.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1204680/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1204680/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1204680" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-17" class="first-child">Summary of potential 2019-nCoV T-cell epitope candidates based on HLA antigen presentation scores. The HLA-A, HLA-B, HLA-C, and HLA-DR columns refer to the number of peptide sequences that were predicted to present across more than one third of common alleles among the Chinese population. Length refers to the number of amino acids in a given gene. HLA-C are more likely to present 2019-nCoV antigens across one third of alleles compared to HLA-A or B. Fisher’s exact test P-value levels are indicated with *** (&lt;; 0.001), ** (&lt;; 0.01), and * (&lt;; 0.05).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="MHC presentation regions for 11 2019-nCoV genes across HLA-A, HLA-B, HLA-C, and HLA-DR. We plot results at both medium (A) and high (B) cut-offs for presentation. The X-axis indicates the position of each viral protein, and the y-axis indicates each human HLA gene family. Each blue stripe indicates the fraction of HLA alleles that can present a 9mer or 15mer viral peptide starting from a given position. A peptide being presented by more than one third of common alleles (red) is considered to be high coverage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675726145" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;MHC presentation regions for 11 2019-nCoV genes across HLA-A, HLA-B, HLA-C, and HLA-DR.&lt;;/span&gt;; We plot results at both medium (&lt;;strong&gt;;A&lt;;/strong&gt;;) and high (&lt;;strong&gt;;B&lt;;/strong&gt;;) cut-offs for presentation. The X-axis indicates the position of each viral protein, and the y-axis indicates each human HLA gene family. Each blue stripe indicates the fraction of HLA alleles that can present a 9mer or 15mer viral peptide starting from a given position. A peptide being presented by more than one third of common alleles (red) is considered to be high coverage.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F2.medium.gif" width="440" height="356"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F2.medium.gif" width="440" height="356"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 2.</span>
       <span class="caption-title">MHC presentation regions for 11 2019-nCoV genes across HLA-A, HLA-B, HLA-C, and HLA-DR.</span>
       <p id="p-18" class="first-child">We plot results at both medium (
        <strong>A</strong>) and high (
        <strong>B</strong>) cut-offs for presentation. The X-axis indicates the position of each viral protein, and the y-axis indicates each human HLA gene family. Each blue stripe indicates the fraction of HLA alleles that can present a 9mer or 15mer viral peptide starting from a given position. A peptide being presented by more than one third of common alleles (red) is considered to be high coverage.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-19">NetMHCpan4 predicts that a larger number of 2019-nCoV protein sequences can be presented by HLA-C alleles across all genes (Fisher’s exact test, 
      <em>p</em> = 4.2
      <em>e</em>
      <sup>−59</sup>) compared to HLA-A or HLA-B (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Specifically, ORF1ab (
      <em>p</em> = 6.7
      <em>e</em>
      <sup>−54</sup>), S (
      <em>p</em> = 1.4
      <em>e</em>
      <sup>−9</sup>), ORF3a (
      <em>p</em> = 0.01), and M (
      <em>p</em> = 0.025) can be better presented by HLA-C alleles after Bonferroni correction.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>T-cell epitope validation</h3>
     <p id="p-20">To estimate the ability of antigen presentation scores to identify 2019-nCoV T-cell epitopes, we performed a validation study with known SARS-CoV CD8 and CD4 T-cell epitopes (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). From 4 independent experimental studies (
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>–
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>), we identified known CD8 T-cell epitopes (MHC-I, n=17) and known CD4 T-cell epitopes (MHC-II, n=3). We also obtained known non-epitopes (n=1236 and 246) for CD4, while for CD8 we include all 9mer sliding windows of S protein without a known epitope as non-epitopes. MHC-I presentation (with recommended cut-off 98% (
      <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>)) has a sensitivity of 82.3%, specificity of 97.0% and an AUC of 0.98 (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3A</a>). MHC-II presentation (recommended cut-off 95% (
      <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>)) has a sensitivity of 66.6%, specificity of 91.1% and an AUC of 0.83 (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3B</a>).
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Validation of NetMHCpan and MARIA for T-cell epitope identification with SARS-CoV S protein epitopes. NetMHCpan4 and MARIA presentation percentiles were used to differentiate experimentally identified CD8 T-cell epitopes (MHC-I, n=17)(23[[#x2013]]25) and CD4 T-cell epitopes (MHC-II, n=3)(26) from negative peptides (n=1236 and 246). Negative peptides for CD8 T-cell epitopes are all 9mer sliding windows of S protein without a reported epitope. Negative peptides for CD4 T-cell epitopes are experimentally tested. (A) NetMHCpan4 has a sensitivity of 82.3% and specificity of 97.0% with a 98th presentation percentile cut-off, and an overall AUC of 0.98. (B) MARIA has a sensitivity of 66.6% and specificity of 91.1% with a 95th presentation percentile cut-off, and an overall AUC of 0.83." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675726145" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Validation of NetMHCpan and MARIA for T-cell epitope identification with SARS-CoV S protein epitopes.&lt;;/span&gt;; NetMHCpan4 and MARIA presentation percentiles were used to differentiate experimentally identified CD8 T-cell epitopes (MHC-I, n=17)(23[[#x2013]]25) and CD4 T-cell epitopes (MHC-II, n=3)(26) from negative peptides (n=1236 and 246). Negative peptides for CD8 T-cell epitopes are all 9mer sliding windows of S protein without a reported epitope. Negative peptides for CD4 T-cell epitopes are experimentally tested. (&lt;;strong&gt;;A)&lt;;/strong&gt;; NetMHCpan4 has a sensitivity of 82.3% and specificity of 97.0% with a 98th presentation percentile cut-off, and an overall AUC of 0.98. (&lt;;strong&gt;;B)&lt;;/strong&gt;; MARIA has a sensitivity of 66.6% and specificity of 91.1% with a 95th presentation percentile cut-off, and an overall AUC of 0.83.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F3.medium.gif" width="440" height="243"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F3.medium.gif" width="440" height="243"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 3.</span>
       <span class="caption-title">Validation of NetMHCpan and MARIA for T-cell epitope identification with SARS-CoV S protein epitopes.</span>
       <p id="p-21" class="first-child">NetMHCpan4 and MARIA presentation percentiles were used to differentiate experimentally identified CD8 T-cell epitopes (MHC-I, n=17)(
        <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>–
        <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>) and CD4 T-cell epitopes (MHC-II, n=3)(
        <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>) from negative peptides (n=1236 and 246). Negative peptides for CD8 T-cell epitopes are all 9mer sliding windows of S protein without a reported epitope. Negative peptides for CD4 T-cell epitopes are experimentally tested. (
        <strong>A)</strong> NetMHCpan4 has a sensitivity of 82.3% and specificity of 97.0% with a 98th presentation percentile cut-off, and an overall AUC of 0.98. (
        <strong>B)</strong> MARIA has a sensitivity of 66.6% and specificity of 91.1% with a 95th presentation percentile cut-off, and an overall AUC of 0.83.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-22">Given the high similarity between SARS-CoV and 2019-nCoV proteins, we expect our analysis on 2019-nCoV to perform similarly against future experimentally validated T-cell epitopes. The detailed scores and sequences of this analysis can be found in SI Appendix, Supplementary Tables 7 and 8.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Viral mutation and antigen presentation</h3>
     <p id="p-23">Human immune selection pressure has been shown to drive viral mutations which evade immune surveillance (e.g. low MHC presentation). We hypothesize that a similar phenomenon can occur in 2019-nCoV. We curated a cohort of 68 viral genomes across four continents and identified 93 point mutations, 2 nonsense mutation and 1 deletion mutation compared to the published reference genome (
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>). We plot point mutations against regions of MHC-I or MHC-II presentation in 
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>. The full protein sequence and mutation information can be found in SI Appendix, Dataset S2 and S3.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="2019-nCoV point mutations correlates with MHC-I presentation regions. 93 point mutations at 59 unique positions (red triangles) are present in a cohort of 68 2019-nCoV cohort. Each row represents a viral protein and the shades of blue indicate whether MHC-I or MHC-II or both can present an antigen from this region. A region is labeled as presented by MHC-I or MHC-II if more than one third of the corresponding alleles can present the peptide sequences. Mutations are more likely be located in regions with MHC-I presentation (p = 0.02). No relationship was observed with MHC-II (p = 0.58) or MHC-I and MHC-II double positive region (dark blue, p = 0.14)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675726145" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;2019-nCoV point mutations correlates with MHC-I presentation regions.&lt;;/span&gt;; 93 point mutations at 59 unique positions (red triangles) are present in a cohort of 68 2019-nCoV cohort. Each row represents a viral protein and the shades of blue indicate whether MHC-I or MHC-II or both can present an antigen from this region. A region is labeled as presented by MHC-I or MHC-II if more than one third of the corresponding alleles can present the peptide sequences. Mutations are more likely be located in regions with MHC-I presentation (&lt;;em&gt;;p&lt;;/em&gt;; = 0.02). No relationship was observed with MHC-II (&lt;;em&gt;;p&lt;;/em&gt;; = 0.58) or MHC-I and MHC-II double positive region (dark blue, &lt;;em&gt;;p&lt;;/em&gt;; = 0.14).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F4.medium.gif" width="440" height="261"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F4.medium.gif" width="440" height="261"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 4.</span>
       <span class="caption-title">2019-nCoV point mutations correlates with MHC-I presentation regions.</span>
       <p id="p-24" class="first-child">93 point mutations at 59 unique positions (red triangles) are present in a cohort of 68 2019-nCoV cohort. Each row represents a viral protein and the shades of blue indicate whether MHC-I or MHC-II or both can present an antigen from this region. A region is labeled as presented by MHC-I or MHC-II if more than one third of the corresponding alleles can present the peptide sequences. Mutations are more likely be located in regions with MHC-I presentation (
        <em>p</em> = 0.02). No relationship was observed with MHC-II (
        <em>p</em> = 0.58) or MHC-I and MHC-II double positive region (dark blue, 
        <em>p</em> = 0.14).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-25">Mutations occur in most genes with the exception of E, ORF6, ORF7b, and N. We identify a recurrent mutation L84S in ORF8 in 20 samples. Mutations are more likely to occur in regions with MHC-I presentation (Fischer’s exact test, 
      <em>p</em> = 0.02). No relationship was observed with MHC-II (
      <em>p</em> = 0.58) or an aggregate of MHC-I and MHC-II (
      <em>p</em> = 0.14). This is consistent with the role of CD8 T-cell in clearing infected cells and mutations supporting the evasion of immune surveillance (
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>).
     </p>
     <p id="p-26">We identified two nonsense mutations (ORF1AB T15372A and ORF7A C184T) in two viral samples from Shenzhen, China (EPI_ISL_406592 and EPI_ISL_406594). ORF7A C184U causes an immediate stop gain and a truncation of the last 60 amino acids from the ORF7A gene, which can be presented by multiple MHC alleles as indicated in our heatmap (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>). In a sample from Japan (EPI_ISL_407084), the viral gene ORF1AB loses nucleotides 94-118, which code for GDSVEEVL. An associated antigen FGDSVEEVL is predicted to be good presented by multiple MHC-I alleles (e.g. 99.9% for HLA-C*08:01, 99.2% for HLA-C*01:02, and 99.0% for HLA-B*39:01).
     </p>
     <p id="p-27">In summary, we observed that 2019-nCoV mutations often occur among or delete presentable peptides, a potential form of immune evasion. Patient MHC allele information and T-cell profiling are needed to better assess this phenomenon. However, patients typically carry fewer than 3 mutations, so a vaccine against multiple epitopes will not be nullified by such a small number of mutations.</p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Potential B-cell epitopes of 2019-nCoV</h3>
     <p id="p-28">We obtained the likely 3D structure of the full 2019-nCoV’s spike protein (SI Appendix, Dataset S4 and S5) by homology modeling with SARS-CoV’s spike protein (
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). Given our understanding of SARS (
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>), the spike protein has two major conformations: up and down. The down position is more stable but less accessible to human entry receptors (e.g. ACE2). The receptor-binding domain (RBD) of the protein undergoes hinge-like conformational change to enter the up conformation, which is less stable but more accessible to human receptors (
      <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>). We compared our homology modeled structures to a later solved Cryo-EM structure (
      <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>) and found high similarity (Appendix SI, Fig. S1). Notably, the Cryo-EM solved structure has missing residues in the RBD due to limited electron density. Using modeled structures overcomes this issue.
     </p>
     <p id="p-29">We scanned spike protein structures with Discotope2 to identify potential antibody binding sites on the protein surface (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figs. 5</a> and 
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>). For the SARS-CoV S protein down conformation, we identified a strong cluster of antibody sites (&gt;;30 residues, pink or red) on the receptor binding domain (RBD, the ACE2 interacting surface). Three independent studies (
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>) discovered B-cell epitopes in this region (blue) with a combination of in vitro and ex vivo approaches. Additionally, Discotope2 identifies residue 541-555 as another binding site, which was supported by two independent studies (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a>, blue) (
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>, 
      <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>). This gives us some confidence in the ability of Discotope2 to predict B-cell epitopes if given a unknown protein structure.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Predicted B-cell epitopes on SARS-CoV and 2019-nCoV spike (S) protein [[#x201C]]down[[#x201D]] position. 3D structures of both spike proteins in [[#x201C]]down[[#x201D]] position were scanned with Discotope2 to assess potential antibody binding sites (B-cell epitopes). Red indicates residues with score &gt;; 0.5 (cut-off for specificity of 90%), and pink indicates residues with score &gt;; [[#x2212]]2.5 (cut-off for specificity of 80%). (A) The receptor binding domain (RBD) of SARS spike protein has concentrated high score residues and is comprised of two linear epitopes. One additional binding site is predicted to be around residue 541-555. Three independent experimental studies (28[[#x2013]]30) identify patient antibodies recognize these three linear epitopes (blue). (B) Modeled 2019-nCoV spike protein is predicted to have a similar strong antibody binding site near RBD and an additional site around residue 246-257. (C) The predicted antibody binding site on 2019-nCov RBD potentially overlaps with the interacting surface for the known human entry receptor ACE2 (yellow). (D) Eight 2019-nCoV spike protein point mutations are present in a cohort of 68 viral samples. No mutations are near the ACE2 interacting surface. One mutation (S247R) occurs near one of the predicted antibody binding sites (residue 246-257)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675726145" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Predicted B-cell epitopes on SARS-CoV and 2019-nCoV spike (S) protein [[#x201C]]down[[#x201D]] position.&lt;;/span&gt;; 3D structures of both spike proteins in [[#x201C]]down[[#x201D]] position were scanned with Discotope2 to assess potential antibody binding sites (B-cell epitopes). Red indicates residues with score &gt;; 0.5 (cut-off for specificity of 90%), and pink indicates residues with score &gt;; [[#x2212]]2.5 (cut-off for specificity of 80%). (&lt;;strong&gt;;A&lt;;/strong&gt;;) The receptor binding domain (RBD) of SARS spike protein has concentrated high score residues and is comprised of two linear epitopes. One additional binding site is predicted to be around residue 541-555. Three independent experimental studies (28[[#x2013]]30) identify patient antibodies recognize these three linear epitopes (blue). (&lt;;strong&gt;;B&lt;;/strong&gt;;) Modeled 2019-nCoV spike protein is predicted to have a similar strong antibody binding site near RBD and an additional site around residue 246-257. (&lt;;strong&gt;;C&lt;;/strong&gt;;) The predicted antibody binding site on 2019-nCov RBD potentially overlaps with the interacting surface for the known human entry receptor ACE2 (yellow). (&lt;;strong&gt;;D&lt;;/strong&gt;;) Eight 2019-nCoV spike protein point mutations are present in a cohort of 68 viral samples. No mutations are near the ACE2 interacting surface. One mutation (S247R) occurs near one of the predicted antibody binding sites (residue 246-257).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F5.medium.gif" width="363" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F5.medium.gif" width="363" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 5.</span>
       <span class="caption-title">Predicted B-cell epitopes on SARS-CoV and 2019-nCoV spike (S) protein “down” position.</span>
       <p id="p-30" class="first-child">3D structures of both spike proteins in “down” position were scanned with Discotope2 to assess potential antibody binding sites (B-cell epitopes). Red indicates residues with score &gt;; 0.5 (cut-off for specificity of 90%), and pink indicates residues with score &gt;; −2.5 (cut-off for specificity of 80%). (
        <strong>A</strong>) The receptor binding domain (RBD) of SARS spike protein has concentrated high score residues and is comprised of two linear epitopes. One additional binding site is predicted to be around residue 541-555. Three independent experimental studies (
        <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>–
        <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>) identify patient antibodies recognize these three linear epitopes (blue). (
        <strong>B</strong>) Modeled 2019-nCoV spike protein is predicted to have a similar strong antibody binding site near RBD and an additional site around residue 246-257. (
        <strong>C</strong>) The predicted antibody binding site on 2019-nCov RBD potentially overlaps with the interacting surface for the known human entry receptor ACE2 (yellow). (
        <strong>D</strong>) Eight 2019-nCoV spike protein point mutations are present in a cohort of 68 viral samples. No mutations are near the ACE2 interacting surface. One mutation (S247R) occurs near one of the predicted antibody binding sites (residue 246-257).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Predicted B-cell epitopes on 2019-nCoV spike (S) protein in up conformation. We scanned 3D structures of modeled (A) and experimentally solved (B) 2019-nCoV S protein in the up position with Discotope2 to assess potential antibody binding sites (B-cell epitopes). Red indicates residues with a score greater than 0.5 (with a cut-off for specificity of 90%), and pink indicates residues with score &gt;; [[#x2212]]2.5 (with a cut-off for specificity of 80%). (A) Discotope2 identified multiple B-cell epitopes on the modeled 2019-nCov S protein. The top candidates cluster around the receptor binding domain (RBD). (B) The RBD B-cell epitotope cluster is comprised of two linear epitopes (440-460, 494-506). (C,D) Discotope2 identified highly similar B-cell epitopes with solved 2019-nCoV co-crystal structures from two independent research groups (19, 20). Predicted antibody binding sites overlap with the interacting surface for the known human entry receptor ACE2 (yellow)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1675726145" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Predicted B-cell epitopes on 2019-nCoV spike (S) protein in up conformation.&lt;;/span&gt;; We scanned 3D structures of modeled (A) and experimentally solved (B) 2019-nCoV S protein in the up position with Discotope2 to assess potential antibody binding sites (B-cell epitopes). Red indicates residues with a score greater than 0.5 (with a cut-off for specificity of 90%), and pink indicates residues with score &gt;; [[#x2212]]2.5 (with a cut-off for specificity of 80%). (&lt;;strong&gt;;A&lt;;/strong&gt;;) Discotope2 identified multiple B-cell epitopes on the modeled 2019-nCov S protein. The top candidates cluster around the receptor binding domain (RBD). (&lt;;strong&gt;;B&lt;;/strong&gt;;) The RBD B-cell epitotope cluster is comprised of two linear epitopes (440-460, 494-506). (&lt;;strong&gt;;C&lt;;/strong&gt;;,&lt;;strong&gt;;D&lt;;/strong&gt;;) Discotope2 identified highly similar B-cell epitopes with solved 2019-nCoV co-crystal structures from two independent research groups (19, 20). Predicted antibody binding sites overlap with the interacting surface for the known human entry receptor ACE2 (yellow).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F6.medium.gif" width="440" height="435"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F6.medium.gif" width="440" height="435"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2020.02.19.955484/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 6.</span>
       <span class="caption-title">Predicted B-cell epitopes on 2019-nCoV spike (S) protein in up conformation.</span>
       <p id="p-31" class="first-child">We scanned 3D structures of modeled (A) and experimentally solved (B) 2019-nCoV S protein in the up position with Discotope2 to assess potential antibody binding sites (B-cell epitopes). Red indicates residues with a score greater than 0.5 (with a cut-off for specificity of 90%), and pink indicates residues with score &gt;; −2.5 (with a cut-off for specificity of 80%). (
        <strong>A</strong>) Discotope2 identified multiple B-cell epitopes on the modeled 2019-nCov S protein. The top candidates cluster around the receptor binding domain (RBD). (
        <strong>B</strong>) The RBD B-cell epitotope cluster is comprised of two linear epitopes (440-460, 494-506). (
        <strong>C</strong>,
        <strong>D</strong>) Discotope2 identified highly similar B-cell epitopes with solved 2019-nCoV co-crystal structures from two independent research groups (
        <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, 
        <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>). Predicted antibody binding sites overlap with the interacting surface for the known human entry receptor ACE2 (yellow).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-32">For the 2019-nCoV S protein down conformation, Discotope2 identified a similar antibody binding site on the S protein potential RBD, but with fewer residues (17 residues, 
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>). Residues 440-460 and 494-506 located at the top of the RBD show strong antibody binding properties (&gt;;0.5 raw scores). Discotope2 also identified another antibody binding site (residue 246-257) different from SARS-CoV but with lower scores (&gt;;-2.5 raw scores).
     </p>
     <p id="p-33">For the 2019-nCoV S protein up conformation, Discotope2 identified similar antibody binding sites compared to down conformation (440-460 and 494-506, 
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6</a>). We wanted to explore whether these two linear epitopes are critical for viral interaction with the human entry protein ACE2. After our original analysis, two solved co-crystal structures of 2019-nCoV S protein RBD and human ACE2 become available (
      <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>, 
      <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>). We ran Discotope2 and obtained strikingly similar results (
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6C</a> and 
      <a id="xref-fig-6-4" class="xref-fig" href="#F6">D</a>). The main antibody binding site substantially overlaps with the interacting surface where ACE2 (yellow) binds to S protein, so an antibody binding to this surface is likely to block viral entry into cells.
     </p>
     <p id="p-34">Mutations in key antibody binding sites can help viruses evade the human immune system. We identified 8 point mutation sites on S protein from a cohort of 68 2019-nCoV samples (
      <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5D</a>). All mutations are distant from the S protein RBD, and one mutation (S247R) occurs near one of the predicted minor antibody binding site (residue 246-257).
     </p>
     <p id="p-35">In summary, we observed major structural and B-cell epitope similarity between SARS-CoV and 2019-nCoV spike proteins. RBDs in both proteins seem to be important B-cell epitopes for neutralizing antibodies, however, SARS-CoV appears to have larger attack surface than 2019-nCoV (
      <a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5A, 5B</a>). Fortunately, we have not observed any mutations altering binding sites for this major B-cell epitope in 2019-nCoV.
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Promising candidates for 2019-nCoV vaccines</h3>
     <p id="p-36">After understanding the landscape of T-cell and B-cell epitopes of 2019-nCoV, we summarize our results in 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> for promising candidates for 2019-nCoV vaccines. Ideal antigens should be presented by multiple MHC-I and MHC-II alleles in a general population and contain linear B-cell epitopes associated with neutralizing antibodies. Specifically, we first identified regions with high coverage of MHC-II, then ranked them based on their MHC-I coverage. We further search these top candidates with 90% sequence similarity on IEDB (
      <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>) to assess whether any candidate has been previously reported for being a B-cell epitope or T-cell epitope.
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1204679/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1204679/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1204679" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-37" class="first-child">Top potential epitopes for key 2019-nCoV proteins. We ranked epitopes based on their likely coverage of presentation by MHC-I and MHC-II alleles. S protein 494-508 is highly ranked based on MHC presentation and is also one of the predicted top B-cell epitopes, localized near the S protein receptor binding domain (
        <a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5</a>). MHC-I coverage is calculated by the 9mer with the highest MHC-I coverage for each epitope (highlighted in orange). All candidates are likely to be presented by both MHC-I and MHC-II. “SARS” under the antibody column indicates that one or more SARS homolog of this peptide is a known B-cell epitope.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-38">Around 1000 peptide 15mers can be presented by a wide range of MHC-II (&gt;;33%), which is consistent with our knowledge about promiscuity of MHC-II presentation (
      <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>, 
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). When further requiring a peptide to be presented by more than 33% or 66% of common MHC-I alleles, we narrowed the results to 405 or 44 promising T-cell epitopes, respectively (SI Appendix, Supplementary Tables 5 and 6). The top 13 candidates in key viral proteins can be presented by 52-83% of MHC-I alleles (
      <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Most notably, S protein 494-508 is not only a good MHC presenter but also a predicted B-cell epitope near the S protein receptor binding domain (
      <a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5B</a>), which indicates a good vaccine candidate.
     </p>
     <p id="p-39">When searching these 13 candidates on IEDB, we observed several of their homologous peptides in SARS are T-cell and B-cell epitopes. SARS peptide SIVAYTMSL is similar to 2019-nCoV peptide SQSIIAYMSLGAEN and elicits T-cell responses in chromium cytotoxicity assays (
      <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Previously discovered SARS B-cell epitopes LMSFPQAAPHGVVFLHV (
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>), ASFRLFARTRSMWSF(
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>), KRTATKQYNVTQAF-GRR (
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>), and SRVKNLNSSEGVPDLLV (
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>) share high homology with 2019-nCoV S protein 1056-1060, M 97-111, N 264-278, and E 52-66. These search results support validity of our computational pipeline.
     </p>
    </div>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Discussion</h2>
    <p id="p-40">In summary, we analyzed the 2019-nCoV viral genome for epitope candidates and found 405 likely T-Cell epitopes, with strong MHC-I and MHC-II presentation scores, as well as 2 potential neutralizing B-Cell epitopes on the S protein. We validated our methodology by running a similar analysis on SARS-CoV against known SARS T-cell and B-cell epitopes. Analyzing viral mutations across 68 patient samples we found that they have a tendency to occur in regions with strong MHC-I presentation, a possible form of immune evasion. However, no mutations are present near the spike protein receptor binding domain, which is critical for viral entrance to cells and neutralizing antibody binding.</p>
    <p id="p-41">We observed that significant portions of 2019-nCoV viral protein can be presented by multiple HLA-C alleles. This might be due to relatively conserved HLA-C binding motif on the 9th position (the main anchor residue) where Leucine, Methionine, Phenylalanine, Tyrosine are commonly favored across alleles (
     <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). HLA-C*07:02, HLA-C*01:02, and HLA-C*06:02 all have &gt;;10% allele frequency in the Chinese population (
     <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). This suggests the future epitope-based vaccines should include HLA-C related epitopes beyond the common HLA-A*02:01 and HLA-B*40:01 approach.
    </p>
    <p id="p-42">Similar to SARS-CoV (
     <a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">1</a>, 
     <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>), the 2019-nCoV spike protein is likely to be immunogenic as it carries a number of both T-cell and B-cell epitopes. Recombinant surface protein vaccines have demonstrated strong clinical utilities in hepatitis B and human papillomavirus (HPV) vaccines (
     <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, 
     <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). A recombinant 2019-nCoV spike protein vaccine combined with other top epitopes and an appropriate adjuvant could be a reasonable next step for 2019-nCoV vaccine development. Recently, two biotech companies Moderna and Inovio have announced early clinical trials for spike protein based vaccines.
    </p>
    <p id="p-43">From an economic perspective, the development of coronavirus vaccines faces significant financial hurdles as most such viruses do not cause endemic infections. After an epidemic episode (e.g. SARS) there is little financial incentive for private companies to develop vaccines against a specific strain of virus (
     <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>, 
     <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">8</a>, 
     <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>). To address these incentive issues, governments might provide greater funding support to create vaccines against past and future outbreaks.
    </p>
    <p id="p-44">Our pipeline provides a framework to identify strong epitope-based vaccine candidates and might be applied against any unknown pathogens. We were able to model structures of 2019-nCoV spike protein as soon as the DNA sequence became available, and the modeled structure has high similarity with later solved crystal structures (SI Appendix, Fig. S1). This suggests the researchers can rely on homology remodeling when emerging pathogens first appear if the new pathogen shares high sequence similarity with an existing pathogen.</p>
    <p id="p-45">Previous animal studies show antigens with high MHC presentation scores are more likely to elicit strong T-cell responses, but the correlation between these responses and vaccine efficacy is relatively weak (
     <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>, 
     <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, 
     <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>). When combined with future clinical data, our work can help the field untangle the relationship between antigen presentation and vaccine efficacy.
    </p>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Materials and Methods</h2>
    <p id="p-46">We obtained 2019-nCoV (SARS-CoV-2019) and SARS-CoV reference sequence data from NCBI GeneBank (NC_045512 and NC_004718) (
     <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>, 
     <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>). We obtained viral sequences associated with 68 patients from GISAID on Feb 1st 2020.
    </p>
    <p id="p-47">We broke each gene sequence in 2019-nCoV into sliding windows and used NetMHCpan4 (
     <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>) and MARIA (
     <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">15</a>) to predict MHC-I and MHC-II presentation scores across 32 MHC alleles common in the Chinese population (
     <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>). We marked a peptide sequence as covered by MHC-I and MHC-II if it is presented by more then 33% of constituent alleles. For validation, we applied our methodology to known SARS T-cell epitopes and non-epitopes.
    </p>
    <p id="p-48">We predicted likely human antibody binding sites (B-cell epitopes) on SARS and 2019-nCoV S protein with Disctope2 (
     <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>) after first obtaining an approximate 3D structure of 2019-nCoV S protein by homology modeling SARS S protein (PDB: 6ACC) with SWISS-MODEL (
     <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>, 
     <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). To validate our B-cell epitope predictions, we compared our top 3 B-cell epitopes for SARS S protein with previously experimentally identified epitopes (
     <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a>–
     <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">30</a>).
    </p>
    <p id="p-49">For mutation identification, we translated viral genome sequences from patients into protein sequences and compared them with the reference sequence to identify mutations with edit distance analysis. We compared positions of point mutations with MHC-I or MHC-II presentable regions in the protein with Fisher’s exact test.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGMENTS</h2>
    <p id="p-50">We thank Stanford Schor and Allen Lin for the discussion about coronaviruses. We thank all labs contributing to the global efforts of sequencing 2019-nCoV samples and we attached the full acknowledgment list in SI Appendix, Supplementary Table 1.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-1">
      <p id="p-1">B.C. and E.F. designed the study and implemented the analysis pipeline, and drafted the manuscript. R.A. provided insights into structural biology and reviewed the manuscript.</p>
     </li>
     <li class="fn-conflict" id="fn-2">
      <p id="p-2">Authors have no competing interests.</p>
     </li>
     <li class="fn-update fn-group-summary-of-updates" id="fn-3">
      <p id="p-6">We added the Figure 6 and updated Figure 5 as new structural data of 2019-nCoV becomes available.</p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2>References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S</span>
          <span class="cit-name-surname">Perlman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J</span>
          <span class="cit-name-surname">Netland</span>
         </span>, 
         <span class="cit-article-title">Coronaviruses post-sars: update on replication and pathogenesis</span>. 
         <abbr class="cit-jnl-abbrev">Nat. reviews microbiology</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">439</span>–
         <span class="cit-lpage">450</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2Breviews%2Bmicrobiology%26rft.volume%253D7%26rft.spage%253D439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WMCR Group</span>,, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">State of knowledge and data gaps of middle east respiratory syndrome coronavirus (mers-cov) in humans</span>. 
         <abbr class="cit-jnl-abbrev">PLoS currents</abbr>
         <span class="cit-vol">5</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C</span>
          <span class="cit-name-surname">Huang</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in wuhan, china</span>. 
         <abbr class="cit-jnl-abbrev">The Lancet</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">F</span>
          <span class="cit-name-surname">Wu</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in china</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">8</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P</span>
          <span class="cit-name-surname">Zhou</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">4</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.19.955484v2.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">World Health Organization report</span> (
         <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200216-sitrep-27-covid-19.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200216-sitrep-27-covid-19.pdf</a>) (Accessed 
         <span class="cit-date-in-citation">February 2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.7" data-doi="10.3389/fmicb.2019.01781">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">CY</span>
          <span class="cit-name-surname">Yong</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">HK</span>
          <span class="cit-name-surname">Ong</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">SK</span>
          <span class="cit-name-surname">Yeap</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">KL</span>
          <span class="cit-name-surname">Ho</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">WS</span>
          <span class="cit-name-surname">Tan</span>
         </span>, 
         <span class="cit-article-title">Recent advances in the vaccine development against middle east respiratory syndrome-coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Front. Microbiol</abbr>. 
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">1781</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol%26rft.volume%253D10%26rft.spage%253D1781%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2019.01781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3389/fmicb.2019.01781&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.8" data-doi="10.1016/j.vaccine.2014.04.016">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">CM</span>
          <span class="cit-name-surname">Coleman</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Purified coronavirus spike protein nanoparticles induce coronavirus neutralizing antibodies in mice</span>. 
         <abbr class="cit-jnl-abbrev">Vaccine</abbr>
         <span class="cit-vol">32</span>, 
         <span class="cit-fpage">3169</span>–
         <span class="cit-lpage">3174</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D32%26rft.spage%253D3169%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2014.04.016%26rft_id%253Dinfo%253Apmid%252F24736006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2014.04.016&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24736006&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">CT</span>
          <span class="cit-name-surname">Tseng</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Immunization with sars coronavirus vaccines leads to pulmonary immunopathology on challenge with the sars virus</span>. 
         <abbr class="cit-jnl-abbrev">PloS one</abbr>
         <span class="cit-vol">7</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R</span>
          <span class="cit-name-surname">Rappuoli</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S</span>
          <span class="cit-name-surname">Black</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">DE</span>
          <span class="cit-name-surname">Bloom</span>
         </span>, 
         <span class="cit-article-title">Vaccines and global health: In search of a sustainable model for vaccine development and delivery</span>. 
         <abbr class="cit-jnl-abbrev">Sci. Transl. Medicine</abbr>
         <span class="cit-vol">11</span>, 
         <span class="cit-fpage">eaaw2888</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci.%2BTransl.%2BMedicine%26rft.volume%253D11%26rft.spage%253D2888eaaw%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.11" data-doi="10.1016/j.vaccine.2007.03.049">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">SE</span>
          <span class="cit-name-surname">Olsson</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Induction of immune memory following administration of a prophylactic quadrivalent human papillomavirus (hpv) types 6/11/16/18 l1 virus-like particle (vlp) vaccine</span>. 
         <abbr class="cit-jnl-abbrev">Vaccine</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">4931</span>–
         <span class="cit-lpage">4939</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.stitle%253DVaccine%26rft.aulast%253DOlsson%26rft.auinit1%253DS.%2BE.%26rft.volume%253D25%26rft.issue%253D26%26rft.spage%253D4931%26rft.epage%253D4939%26rft.atitle%253DInduction%2Bof%2Bimmune%2Bmemory%2Bfollowing%2Badministration%2Bof%2Ba%2Bprophylactic%2Bquadrivalent%2Bhuman%2Bpapillomavirus%2B%2528HPV%2529%2Btypes%2B6%252F11%252F16%252F18%2BL1%2Bvirus-like%2Bparticle%2B%2528VLP%2529%2Bvaccine.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2007.03.049%26rft_id%253Dinfo%253Apmid%252F17499406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2007.03.049&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17499406&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000247547400009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.12" data-doi="10.1016/S0145-305X(99)00078-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D</span>
          <span class="cit-name-surname">Suarez</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S</span>
          <span class="cit-name-surname">Schultz-Cherry</span>
         </span>, 
         <span class="cit-article-title">Immunology of avian influenza virus: a review</span>. 
         <abbr class="cit-jnl-abbrev">Dev. &amp;; Comp. Immunol</abbr>. 
         <span class="cit-vol">24</span>, 
         <span class="cit-fpage">269</span>–
         <span class="cit-lpage">283</span> (
         <span class="cit-pub-date">2000</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Band%2Bcomparative%2Bimmunology%26rft.stitle%253DDev%2BComp%2BImmunol%26rft.aulast%253DSuarez%26rft.auinit1%253DD.%2BL.%26rft.volume%253D24%26rft.issue%253D2-3%26rft.spage%253D269%26rft.epage%253D283%26rft.atitle%253DImmunology%2Bof%2Bavian%2Binfluenza%2Bvirus%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0145-305X%252899%252900078-6%26rft_id%253Dinfo%253Apmid%252F10717293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0145-305X(99)00078-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10717293&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000086213800015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.13" data-doi="10.1016/j.cell.2016.08.005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B</span>
          <span class="cit-name-surname">Briney</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Tailored immunogens direct affinity maturation toward hiv neutralizing antibodies</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-vol">166</span>, 
         <span class="cit-fpage">1459</span>–
         <span class="cit-lpage">1470</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D1459%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.005%26rft_id%253Dinfo%253Apmid%252F27610570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.005&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27610570&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">SR</span>
          <span class="cit-name-surname">Pedersen</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Immunogenicity of hla class i and ii double restricted influenza a-derived peptides</span>. 
         <abbr class="cit-jnl-abbrev">PloS one</abbr>
         <span class="cit-vol">11</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Predicting hla class ii antigen presentation through integrated deep learning</span>. 
         <abbr class="cit-jnl-abbrev">Nat. biotechnology</abbr>
         <span class="cit-vol">37</span>, 
         <span class="cit-fpage">1332</span>–
         <span class="cit-lpage">1343</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2Bbiotechnology%26rft.volume%253D37%26rft.spage%253D1332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R</span>
          <span class="cit-name-surname">Lu</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. 
         <abbr class="cit-jnl-abbrev">The Lancet</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">V</span>
          <span class="cit-name-surname">Jurtz</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Netmhcpan-4.0: improved peptide–mhc class i interaction predictions integrating eluted ligand and peptide binding affinity data</span>. 
         <abbr class="cit-jnl-abbrev">The J. Immunol</abbr>. 
         <span class="cit-vol">199</span>, 
         <span class="cit-fpage">3360</span>–
         <span class="cit-lpage">3368</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJ.%2BImmunol%26rft.volume%253D199%26rft.spage%253D3360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D</span>
          <span class="cit-name-surname">Wrapp</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Cryo-em structure of the 2019-ncov spike in the prefusion conformation</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R</span>
          <span class="cit-name-surname">Yan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Guo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L</span>
          <span class="cit-name-surname">Xia</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Q</span>
          <span class="cit-name-surname">Zhou</span>
         </span>, 
         <span class="cit-article-title">Structural basis for the recognition of the 2019-ncov by human ace2</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J</span>
          <span class="cit-name-surname">Lan</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Crystal structure of the 2019-ncov spike receptor-binding domain bound with the ace2 receptor</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">JV</span>
          <span class="cit-name-surname">Kringelum</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C</span>
          <span class="cit-name-surname">Lundegaard</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">O</span>
          <span class="cit-name-surname">Lund</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M</span>
          <span class="cit-name-surname">Nielsen</span>
         </span>, 
         <span class="cit-article-title">Reliable b cell epitope predictions: impacts of method development and improved benchmarking</span>. 
         <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr>
         <span class="cit-vol">8</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Shu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J</span>
          <span class="cit-name-surname">McCauley</span>
         </span>, 
         <span class="cit-article-title">Gisaid: Global initiative on sharing all influenza data–from vision to reality</span>. 
         <abbr class="cit-jnl-abbrev">Eurosurveillance</abbr>
         <span class="cit-vol">22</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Response of memory cd8+ t cells to severe acute respiratory syndrome (sars) coronavirus in recovered sars patients and healthy individuals</span>. 
         <abbr class="cit-jnl-abbrev">The J. Immunol</abbr>. 
         <span class="cit-vol">175</span>, 
         <span class="cit-fpage">591</span>–
         <span class="cit-lpage">598</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJ.%2BImmunol%26rft.volume%253D175%26rft.spage%253D591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.19.955484v2.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M</span>
          <span class="cit-name-surname">Zhou</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Screening and identification of severe acute respiratory syndrome-associated coronavirus-specific ctl epitopes</span>. 
         <abbr class="cit-jnl-abbrev">The J. Immunol</abbr>. 
         <span class="cit-vol">177</span>, 
         <span class="cit-fpage">2138</span>–
         <span class="cit-lpage">2145</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJ.%2BImmunol%26rft.volume%253D177%26rft.spage%253D2138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">YP</span>
          <span class="cit-name-surname">Tsao</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Hla-a* 0201 t-cell epitopes in severe acute respiratory syndrome (sars) coronavirus nucleocapsid and spike proteins</span>. 
         <abbr class="cit-jnl-abbrev">Biochem. biophysical research communications</abbr>
         <span class="cit-vol">344</span>, 
         <span class="cit-fpage">63</span>–
         <span class="cit-lpage">71</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochem.%2Bbiophysical%2Bresearch%2Bcommunications%26rft.volume%253D344%26rft.spage%253D63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">OW</span>
          <span class="cit-name-surname">Ng</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Memory t cell responses targeting the sars coronavirus persist up to 11 years post-infection</span>. 
         <abbr class="cit-jnl-abbrev">Vaccine</abbr>
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">2008</span>–
         <span class="cit-lpage">2014</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D34%26rft.spage%253D2008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.27" data-doi="10.1128/JVI.01505-14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R</span>
          <span class="cit-name-surname">Channappanavar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C</span>
          <span class="cit-name-surname">Fett</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J</span>
          <span class="cit-name-surname">Zhao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">DK</span>
          <span class="cit-name-surname">Meyerholz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S</span>
          <span class="cit-name-surname">Perlman</span>
         </span>, 
         <span class="cit-article-title">Virus-specific memory cd8 t cells provide substantial protection from lethal severe acute respiratory syndrome coronavirus infection</span>. 
         <abbr class="cit-jnl-abbrev">J. virology</abbr>
         <span class="cit-vol">88</span>, 
         <span class="cit-fpage">11034</span>–
         <span class="cit-lpage">11044</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01505-14%26rft_id%253Dinfo%253Apmid%252F25056892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8xOS8xMTAzNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE4LzIwMjAuMDIuMTkuOTU1NDg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R</span>
          <span class="cit-name-surname">Hua</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Zhou</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Hua</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G</span>
          <span class="cit-name-surname">Tong</span>
         </span>, 
         <span class="cit-article-title">Identification of two antigenic epitopes on sars-cov spike protein</span>. 
         <abbr class="cit-jnl-abbrev">Biochem. biophysical research communications</abbr>
         <span class="cit-vol">319</span>, 
         <span class="cit-fpage">929</span>–
         <span class="cit-lpage">935</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochem.%2Bbiophysical%2Bresearch%2Bcommunications%26rft.volume%253D319%26rft.spage%253D929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H</span>
          <span class="cit-name-surname">Yu</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Selection of sars-coronavirus-specific b cell epitopes by phage peptide library screening and evaluation of the immunological effect of epitope-based peptides on mice</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-vol">359</span>, 
         <span class="cit-fpage">264</span>–
         <span class="cit-lpage">274</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DYu%26rft.auinit1%253DH.%26rft.volume%253D359%26rft.issue%253D2%26rft.spage%253D264%26rft.epage%253D274%26rft.atitle%253DSelection%2Bof%2BSARS-coronavirus-specific%2BB%2Bcell%2Bepitopes%2Bby%2Bphage%2Bpeptide%2Blibrary%2Bscreening%2Band%2Bevaluation%2Bof%2Bthe%2Bimmunological%2Beffect%2Bof%2Bepitope-based%2Bpeptides%2Bon%2Bmice.%26rft_id%253Dinfo%253Apmid%252F17055022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=17055022&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.30" data-doi="10.1016/j.virol.2004.04.017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">JP</span>
          <span class="cit-name-surname">Guo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M</span>
          <span class="cit-name-surname">Petric</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W</span>
          <span class="cit-name-surname">Campbell</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">PL</span>
          <span class="cit-name-surname">McGeer</span>
         </span>, 
         <span class="cit-article-title">Sars corona virus peptides recognized by antibodies in the sera of convalescent cases</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-vol">324</span>, 
         <span class="cit-fpage">251</span>–
         <span class="cit-lpage">256</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DGuo%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D324%26rft.issue%253D2%26rft.spage%253D251%26rft.epage%253D256%26rft.atitle%253DSARS%2Bcorona%2Bvirus%2Bpeptides%2Brecognized%2Bby%2Bantibodies%2Bin%2Bthe%2Bsera%2Bof%2Bconvalescent%2Bcases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2004.04.017%26rft_id%253Dinfo%253Apmid%252F15207612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virol.2004.04.017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15207612&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000222376800001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.31" data-doi="10.1371/journal.ppat.1007236">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">W</span>
          <span class="cit-name-surname">Song</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M</span>
          <span class="cit-name-surname">Gui</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Xiang</span>
         </span>, 
         <span class="cit-article-title">Cryo-em structure of the sars coronavirus spike glycoprotein in complex with its host cell receptor ace2</span>. 
         <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">e1007236</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D14%26rft.spage%253De1007236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1007236%26rft_id%253Dinfo%253Apmid%252F30102747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1007236&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30102747&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">TJ</span>
          <span class="cit-name-surname">Yang</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Cryo-em analysis of a feline coronavirus spike protein reveals a unique structure and camouflaging glycans</span>. 
         <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>. (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.33" data-doi="10.1093/nar/gku938">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R</span>
          <span class="cit-name-surname">Vita</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The immune epitope database (iedb) 3.0</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-vol">43</span>, 
         <span class="cit-fpage">D405</span>–
         <span class="cit-lpage">D412</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku938%26rft_id%253Dinfo%253Apmid%252F25300482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gku938&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25300482&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.34" data-doi="10.1007/s00262-014-1551-x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Hu</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Immunologic hierarchy, class ii mhc promiscuity, and epitope spreading of a melanoma helper peptide vaccine</span>. 
         <abbr class="cit-jnl-abbrev">Cancer Immunol. Immunother</abbr>. 
         <span class="cit-vol">63</span>, 
         <span class="cit-fpage">779</span>–
         <span class="cit-lpage">786</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCancer%2BImmunol.%2BImmunother%26rft.volume%253D63%26rft.spage%253D779%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00262-014-1551-x%26rft_id%253Dinfo%253Apmid%252F24756419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s00262-014-1551-x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24756419&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Lv</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z</span>
          <span class="cit-name-surname">Ruan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B</span>
          <span class="cit-name-surname">Ni</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Wu</span>
         </span>, 
         <span class="cit-article-title">Identification of a novel conserved hla-a* 0201-restricted epitope from the spike protein of sars-cov</span>. 
         <abbr class="cit-jnl-abbrev">BMC immunology</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">61</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bimmunology%26rft.volume%253D10%26rft.spage%253D61%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">He</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Identification of immunodominant sites on the spike protein of severe acute respiratory syndrome (sars) coronavirus: implication for developing sars diagnostics and vaccines</span>. 
         <abbr class="cit-jnl-abbrev">The J. Immunol</abbr>. 
         <span class="cit-vol">173</span>, 
         <span class="cit-fpage">4050</span>–
         <span class="cit-lpage">4057</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJ.%2BImmunol%26rft.volume%253D173%26rft.spage%253D4050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.37" data-doi="10.1128/JCM.43.8.3718-3726.2005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">He</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">Zhou</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P</span>
          <span class="cit-name-surname">Siddiqui</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J</span>
          <span class="cit-name-surname">Niu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S</span>
          <span class="cit-name-surname">Jiang</span>
         </span>, 
         <span class="cit-article-title">Identification of immunodominant epitopes on the membrane protein of the severe acute respiratory syndrome-associated coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">J. clinical microbiology</abbr>
         <span class="cit-vol">43</span>, 
         <span class="cit-fpage">3718</span>–
         <span class="cit-lpage">3726</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DHe%26rft.auinit1%253DY.%26rft.volume%253D43%26rft.issue%253D8%26rft.spage%253D3718%26rft.epage%253D3726%26rft.atitle%253DIdentification%2Bof%2BImmunodominant%2BEpitopes%2Bon%2Bthe%2BMembrane%2BProtein%2Bof%2Bthe%2BSevere%2BAcute%2BRespiratory%2BSyndrome-Associated%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.43.8.3718-3726.2005%26rft_id%253Dinfo%253Apmid%252F16081901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjQzLzgvMzcxOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE4LzIwMjAuMDIuMTkuOTU1NDg0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.38" data-doi="10.1128/JCM.42.11.5309-5314.2004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y</span>
          <span class="cit-name-surname">He</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Mapping of antigenic sites on the nucleocapsid protein of the severe acute respiratory syndrome coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">J. clinical microbiology</abbr>
         <span class="cit-vol">42</span>, 
         <span class="cit-fpage">5309</span>–
         <span class="cit-lpage">5314</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DHe%26rft.auinit1%253DY.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D5309%26rft.epage%253D5314%26rft.atitle%253DMapping%2Bof%2BAntigenic%2BSites%2Bon%2Bthe%2BNucleocapsid%2BProtein%2Bof%2Bthe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.42.11.5309-5314.2004%26rft_id%253Dinfo%253Apmid%252F15528730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi8xMS81MzA5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTgvMjAyMC4wMi4xOS45NTU0ODQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.39" data-doi="10.1136/jcp.2005.029868">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">SC</span>
          <span class="cit-name-surname">Chow</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Specific epitopes of the structural and hypothetical proteins elicit variable humoral responses in sars patients</span>. 
         <abbr class="cit-jnl-abbrev">J. clinical pathology</abbr>
         <span class="cit-vol">59</span>, 
         <span class="cit-fpage">468</span>–
         <span class="cit-lpage">476</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2Bclinical%2Bpathology%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fjcp.2005.029868%26rft_id%253Dinfo%253Apmid%252F16461566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToiamNsaW5wYXRoIjtzOjU6InJlc2lkIjtzOjg6IjU5LzUvNDY4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTgvMjAyMC4wMi4xOS45NTU0ODQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.40">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M</span>
          <span class="cit-name-surname">Rasmussen</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Uncovering the peptide-binding specificities of hla-c: a general strategy to determine the specificity of any mhc class i molecule</span>. 
         <abbr class="cit-jnl-abbrev">The J. Immunol</abbr>. 
         <span class="cit-vol">193</span>, 
         <span class="cit-fpage">4790</span>–
         <span class="cit-lpage">4802</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJ.%2BImmunol%26rft.volume%253D193%26rft.spage%253D4790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.41" data-doi="10.1038/ng.3576">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">F</span>
          <span class="cit-name-surname">Zhou</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Deep sequencing of the mhc region in the chinese population contributes to studies of complex disease</span>. 
         <abbr class="cit-jnl-abbrev">Nat. genetics</abbr>
         <span class="cit-vol">48</span>, 
         <span class="cit-fpage">740</span>–
         <span class="cit-lpage">746</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2Bgenetics%26rft.volume%253D48%26rft.spage%253D740%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3576%26rft_id%253Dinfo%253Apmid%252F27213287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ng.3576&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27213287&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.42">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">K</span>
          <span class="cit-name-surname">Tajiri</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Analysis of the epitope and neutralizing capacity of human monoclonal antibodies induced by hepatitis b vaccine</span>. 
         <abbr class="cit-jnl-abbrev">Antivir. research</abbr>
         <span class="cit-vol">87</span>, 
         <span class="cit-fpage">40</span>–
         <span class="cit-lpage">49</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAntivir.%2Bresearch%26rft.volume%253D87%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.43">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P</span>
          <span class="cit-name-surname">Bettencourt</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Identification of antigens presented by mhc for vaccines against tuberculosis</span>. 
         <abbr class="cit-jnl-abbrev">npj Vaccines</abbr>
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">14</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253Dnpj%2BVaccines%26rft.volume%253D5%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">44.</span>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.19.955484v2.44" data-doi="10.1093/nar/gkg520">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">T</span>
          <span class="cit-name-surname">Schwede</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J</span>
          <span class="cit-name-surname">Kopp</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N</span>
          <span class="cit-name-surname">Guex</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">MC</span>
          <span class="cit-name-surname">Peitsch</span>
         </span>, 
         <span class="cit-article-title">Swiss-model: an automated protein homology-modeling server</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>
         <span class="cit-vol">31</span>, 
         <span class="cit-fpage">3381</span>–
         <span class="cit-lpage">3385</span> (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSchwede%26rft.auinit1%253DT.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3381%26rft.epage%253D3385%26rft.atitle%253DSWISS-MODEL%253A%2Ban%2Bautomated%2Bprotein%2Bhomology-modeling%2Bserver%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg520%26rft_id%253Dinfo%253Apmid%252F12824332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkg520&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12824332&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2020.02.19.955484.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000183832900024&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
