<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/22/2020.01.22.915660.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.22.915660v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV" />
<meta name="DC.Identifier" content="10.1101/2020.01.22.915660" />
<meta name="DC.Date" content="2020-01-22" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV." />
<meta name="DC.Contributor" content="Michael Letko" />
<meta name="DC.Contributor" content="Vincent Munster" />
<meta name="article:published_time" content="2020-01-22" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.22.915660v1" />
<meta name="citation_id" content="2020.01.22.915660v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.22.915660" />
<meta name="citation_num_pages" content="26" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.22.915660" />
<meta name="citation_author" content="Michael Letko" />
<meta name="citation_author_institution" content="Laboratory of Virology, Division of Intramural Research, National Institute of Allergy and Infectious Diseases, National Institutes of Health" />
<meta name="citation_author_email" content="vincent.munster@nih.gov" />
<meta name="citation_author_email" content="michael.letko@nih.gov" />
<meta name="citation_author" content="Vincent Munster" />
<meta name="citation_author_institution" content="Laboratory of Virology, Division of Intramural Research, National Institute of Allergy and Infectious Diseases, National Institutes of Health" />
<meta name="citation_author_email" content="vincent.munster@nih.gov" />
<meta name="citation_author_email" content="michael.letko@nih.gov" />
<meta name="citation_reference" content="Wang, M. et al. SARS-CoV infection in a restaurant from palm civet. Emerg Infect Dis 11, 1860–1865, doi:10.3201/eid1112.041293 (2005)." />
<meta name="citation_reference" content="Kuiken, T. et al. Newly discovered coronavirus as the primary cause of severe acute respiratory syndrome. Lancet 362, 263–270, doi:10.1016/S0140-6736(03)13967-0 (2003)." />
<meta name="citation_reference" content="Li, W. et al. Bats are natural reservoirs of SARS-like coronaviruses. Science 310, 676–679, doi:10.1126/science.1118391 (2005)." />
<meta name="citation_reference" content="Ge, X. Y. et al. Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538, doi:10.1038/nature12711 (2013)." />
<meta name="citation_reference" content="Hu, B. et al. Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLoS Pathog 13, e1006698, doi:10.1371/journal.ppat.1006698 (2017)." />
<meta name="citation_reference" content="Lau, S. K. et al. Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats. Proc Natl Acad Sci U S A 102, 14040–14045, doi:10.1073/pnas.0506735102 (2005)." />
<meta name="citation_reference" content="Menachery, V. D. et al. A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence. Nat Med 21, 1508–1513, doi:10.1038/nm.3985 (2015)." />
<meta name="citation_reference" content="Li, F. Structure, Function, and Evolution of Coronavirus Spike Proteins. Annu Rev Virol 3, 237–261, doi:10.1146/annurev-virology-110615-042301 (2016)." />
<meta name="citation_reference" content="Simmons, G., Zmora, P., Gierer, S., Heurich, A. &amp; Pohlmann, S. Proteolytic activation of the SARS-coronavirus spike protein: cutting enzymes at the cutting edge of antiviral research. Antiviral Res 100, 605–614, doi:10.1016/j.antiviral.2013.09.028 (2013)." />
<meta name="citation_reference" content="Matsuyama, S. et al. Efficient activation of the severe acute respiratory syndrome coronavirus spike protein by the transmembrane protease TMPRSS2. J Virol 84, 12658–12664, doi:10.1128/JVI.01542-10 (2010)." />
<meta name="citation_reference" content="Bertram, S. et al. Cleavage and activation of the severe acute respiratory syndrome coronavirus spike protein by human airway trypsin-like protease. J Virol 85, 13363–13372, doi:10.1128/JVI.05300-11 (2011)." />
<meta name="citation_reference" content="Belouzard, S., Chu, V. C. &amp; Whittaker, G. R. Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites. Proc Natl Acad Sci U S A 106, 5871–5876, doi:10.1073/pnas.0809524106 (2009)." />
<meta name="citation_reference" content="Raj, V. S. et al. Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC. Nature 495, 251–254, doi:10.1038/nature12005 (2013)." />
<meta name="citation_reference" content="Li, W. et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454, doi:10.1038/nature02145 (2003)." />
<meta name="citation_reference" content="Li, F., Li, W., Farzan, M. &amp; Harrison, S. C. Structure of SARS coronavirus spike receptor- binding domain complexed with receptor. Science 309, 1864–1868, doi:10.1126/science.1116480 (2005)." />
<meta name="citation_reference" content="Becker, M. M. et al. Synthetic recombinant bat SARS-like coronavirus is infectious in cultured cells and in mice. Proc Natl Acad Sci U S A 105, 19944–19949, doi:10.1073/pnas.0808116105 (2008)." />
<meta name="citation_reference" content="Seifert, S. N. et al. Rousettus aegyptiacus Bats Do Not Support Productive Nipah Virus Replication. J Infect Dis, doi:10.1093/infdis/jiz429 (2019)." />
<meta name="citation_reference" content="Letko, M. et al. Adaptive Evolution of MERS-CoV to Species Variation in DPP4. Cell Rep 24, 1730–1737, doi:10.1016/j.celrep.2018.07.045 (2018)." />
<meta name="citation_reference" content="Takada, A. et al. A system for functional analysis of Ebola virus glycoprotein. Proc Natl Acad Sci U S A 94, 14764–14769, doi:10.1073/pnas.94.26.14764 (1997)." />
<meta name="citation_reference" content="Millet, J. K. &amp; Whittaker, G. R. Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis. Virus Res 202, 120–134, doi:10.1016/j.virusres.2014.11.021 (2015)." />
<meta name="citation_reference" content="Menachery, V. D. et al. Trypsin treatment unlocks barrier for zoonotic coronaviruses infection. bioRxiv (2019)." />
<meta name="citation_reference" content="Yang, Y. et al. Receptor usage and cell entry of bat coronavirus HKU4 provide insight into bat-to-human transmission of MERS coronavirus. Proc Natl Acad Sci U S A 111, 12516–12521, doi:10.1073/pnas.1405889111 (2014)." />
<meta name="citation_reference" content="Matsuyama, S., Ujike, M., Morikawa, S., Tashiro, M. &amp; Taguchi, F. Protease-mediated enhancement of severe acute respiratory syndrome coronavirus infection. Proc Natl Acad Sci U S A 102, 12543–12547, doi:10.1073/pnas.0503203102 (2005)." />
<meta name="citation_reference" content="Kam, Y. W. et al. Cleavage of the SARS coronavirus spike glycoprotein by airway proteases enhances virus entry into human bronchial epithelial cells in vitro. PLoS One 4, e7870, doi:10.1371/journal.pone.0007870 (2009)." />
<meta name="citation_reference" content="Yang, Y. et al. Two Mutations Were Critical for Bat-to-Human Transmission of Middle East Respiratory Syndrome Coronavirus. J Virol 89, 9119–9123, doi:10.1128/JVI.01279- 15 (2015)." />
<meta name="citation_reference" content="Menachery, V. D. et al. Trypsin treatment unlocks barrier for zoonotic bat coronaviruses infection. J Virol, doi:10.1128/JVI.01774-19 (2019)." />
<meta name="citation_reference" content="Beniac, D. R., deVarennes, S. L., Andonov, A., He, R. &amp; Booth, T. F. Conformational reorganization of the SARS coronavirus spike following receptor binding: implications for membrane fusion. PLoS One 2, e1082, doi:10.1371/journal.pone.0001082 (2007)." />
<meta name="citation_reference" content="Woo, P. C. et al. Molecular diversity of coronaviruses in bats. Virology 351, 180–187, doi:10.1016/j.virol.2006.02.041 (2006)." />
<meta name="citation_reference" content="Anthony, S. J. et al. Further Evidence for Bats as the Evolutionary Source of Middle East Respiratory Syndrome Coronavirus. mBio 8, doi:10.1128/mBio.00373-17 (2017)." />
<meta name="citation_reference" content="Shulla, A. et al. A transmembrane serine protease is linked to the severe acute respiratory syndrome coronavirus receptor and activates virus entry. J Virol 85, 873–882, doi:10.1128/JVI.02062-10 (2011)." />
<meta name="citation_reference" content="Adney, D. R. et al. Replication and shedding of MERS-CoV in upper respiratory tract of inoculated dromedary camels. Emerg Infect Dis 20, 1999–2005, doi:10.3201/eid2012.141280 (2014)." />
<meta name="citation_reference" content="Muth, D. et al. Attenuation of replication by a 29 nucleotide deletion in SARS- coronavirus acquired during the early stages of human-to-human transmission. Sci Rep 8, 15177, doi:10.1038/s41598-018-33487-8 (2018)." />
<meta name="citation_reference" content="Hoffmann, M. et al. Differential sensitivity of bat cells to infection by enveloped RNA viruses: coronaviruses, paramyxoviruses, filoviruses, and influenza viruses. PLoS One 8, e72942, doi:10.1371/journal.pone.0072942 (2013)." />
<meta name="citation_reference" content="Liao, C. L. &amp; Lai, M. M. RNA recombination in a coronavirus: recombination between viral genomic RNA and transfected RNA fragments. J Virol 66, 6117–6124 (1992)." />
<meta name="citation_reference" content="Graham, R. L. &amp; Baric, R. S. Recombination, reservoirs, and the modular spike: mechanisms of coronavirus cross-species transmission. J Virol 84, 3134–3146, doi:10.1128/JVI.01394-09 (2010)." />
<meta name="twitter:title" content="Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV." />
<meta name="og-title" property="og:title" content="Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-22" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.22.915660v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.22.915660v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.22.915660v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1117250" />
    <title>Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WPZaSN38M1hjt_QJIK6zrW-AYmA-fHA0ex0boBotLL0","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1117250","apath":"\/biorxiv\/early\/2020\/01\/22\/2020.01.22.915660.atom","pisa":"biorxiv;2020.01.22.915660v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.22.915660v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.22.915660v1","title":"Functional assessment of cell entry and receptor usage for lineage B \u03b2-coronaviruses, including 2019-nCoV","identifier":"node\/1117250"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mXdduK7jNtgGlJ8XPdU5GbkYJBHMKcWS+UIBGkRfq9oH2betqp4778bHOnrixQgMVXU0cnjWJWchd1AuGZ91dvQ\\\\\\\/ccYlBnhhYh1dlnQdDww\\\\\\\/8IPIkPbVuEb+VPvSCZTCxV6DMi5ydRat4M874YpqzoV+gpozP2nl6of3Zz4+CFd3J1h+vjONTZ46Vro+xBhOye71CaUynWPnGeAQRFPgKB\\\\\\\/g4\\\\\\\/+keGkBK\\\\\\\/ADBpxnP5Fz7HpeuvPvAMbhtNlJISlYsQG\\\\\\\/zUUhZuukLg2OQ45ElKEU2TYuJt+IL1l5IfgIwr9rztFnsWHMNLuQV+Tve8H3\\\\\\\/2xHh+Q4ORmubOHIaon3RUGMHUdkL1guX3E+aHYAZfk6Tm11z3Sdu7c6kggX1+pxTlfmTJLzy4aUzQNGAaOILNhSgas26DHPsdWksbwQf6zBP8wTwRKzXPsqRcQ65xJDreI\\\\\\\/RpWgE6SPBH0LFG4x+suEpIZybs+zYL0ZqcBLPy3LXZ3J88IEY9alLzrOmUjvJuQ1XJleFmMc6nNpoVzbdfxlEQJg4Mm5Wlsg+66g9SsWI82z8XScD36N18uSqH4vRzgSDcLEVGDYpBVLRBxmJXRtXfrjzVEr1znN3YpNwGjYmD04kodzS5gcDPMFz6+tWLsaZ0uyecXn+mCMLjms5imnM3lmOESNLZMeHz2eHdQdcQdHIuX5ynoO6ti68QddsY2Tebp0LHm7WQ18c+4BW4NAi9+1X44zAJbp\\\\\\\/abQyesTsn\\\\\\\/ghDnySybSwy0M\\\\\\\/7FcLGG1A7BdrQybXtFpGJG5EPP8TQk36WqqEQ\\\\\\\/P09BLvsy4M29nVPC0q6gTsrJSOXZqxyoCTF3BukZ29LltjD234RYH7aQj3TxcQzl+tSxrooHFICMDm56W06MknwkZg\\\\\\\/G5Cd0z+xykCKMCkPqSC\\\\\\\/MvkrZFzOI3ItL72pxA5x1xUbTNdXLguFoZFihSRGVf+s6cfDHBN7151a1lAbUbBWjCQyUFDZN1ypNhx8idop+ONfE78CoIRqS2d+eg\\\\\\\/mzMQ7bRYeEwrl7yj2l19f3\\\\\\\/UAHk9FSU7Ml1oGDCVZy8REl6K\\\\\\\/t8e62S4a3BNNmqyg8sip2Rcd7I0Ccm9bGggFajvwb+wSNm84uN7t8v31SYC5cBi7AeX+ulf6TgfMfq16uBnT8SPOzqfXzK7yKypSTaSiPdtCsTKY7ds+CDrFtH1tfC3suECQHoQM255e5wOP45GQDASm+cZaZsFS820kRgzJzWAICBmgMK8yIf0dlAbtHRc8kAYs848FpRyW2qV5QM\\\\\\\/pIwGghh9T2ii+qQ\\\\\\\/2e23HuCuE1Tw6jDfDioK\\\\\\\/iafEwx62d2\\\\\\\/v0anpYThjsNCVwaeoVO\\\u0022,\\\u0022iv\\\u0022:\\\u0022lmxbs6nN8bnusvIRH+xpeQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022172934bc5eb7c7daf4c38054ed49a4cd66393e535d4c49aff8272d7c12b18410\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.22.915660v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1117250 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.22.915660v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Iaj-GmCof_2I9T8jI9919Fmj_Wpagbk0X_ltxAs4FjM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1117250" id="node1117250" data-pisa="biorxiv;2020.01.22.915660v1" data-pisa-master="biorxiv;2020.01.22.915660" data-apath="/biorxiv/early/2020/01/22/2020.01.22.915660.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Letko</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Vincent</span> <span class="nlm-surname">Munster</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.22.915660 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1117250" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael Letko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of Virology, Division of Intramural Research, National Institute of Allergy and Infectious Diseases, National Institutes of Health</span>, Hamilton, MT, 59840, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLetko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Letko%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BLetko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vincent.munster@nih.gov" class="" data-icon-position="" data-hide-link-title="0">vincent.munster@nih.gov</a>
<a href="mailto:michael.letko@nih.gov" class="" data-icon-position="" data-hide-link-title="0">michael.letko@nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vincent Munster </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of Virology, Division of Intramural Research, National Institute of Allergy and Infectious Diseases, National Institutes of Health</span>, Hamilton, MT, 59840, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vincent%2BMunster%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munster%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVincent%2BMunster%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vincent.munster@nih.gov" class="" data-icon-position="" data-hide-link-title="0">vincent.munster@nih.gov</a>
<a href="mailto:michael.letko@nih.gov" class="" data-icon-position="" data-hide-link-title="0">michael.letko@nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.22.915660v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1117250" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1117250/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.22.915660v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1117250" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1117250/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.22.915660v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1117250" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1117250/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.22.915660v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1117250" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1117250/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.22.915660v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1117250" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1117250/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.22.915660v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1117250" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1117250/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS- CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these novel viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent 2019-nCoV, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells and confirm that human ACE2 is the receptor for the recently emerging 2019-nCoV.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) first emerged in humans in 2003 after transmitting from animals in open air markets in China<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. Shortly thereafter, several genetically related viruses were identified in Chinese Horseshoes bats (<em>Rhinolophus sinicus</em>)<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. At the same time, improvements in next generation sequencing technology lead to a boom of virus discovery, uncovering thousands of novel virus sequences in wild animal populations around the world. While most of these viruses have never been found in humans, many are genetically similar to known human viruses within the betacoronaviruses (β-CoV) genus. The β-CoVs are further divided into four lineages: lineage B, which includes SARS-CoV and the newly emerging 2019-nCoV, has approximately 200 published virus sequences whereas lineage C, which includes MERS- CoV, has over 500 viral sequences.</p><p id="p-4">Every year, additional novel CoV sequences are discovered. However, there is a massive knowledge gap in the field as very little work is performed after the viral sequences are published. Therefore, it is unknown whether these novel viruses have the potential to emerge in human populations.</p><p id="p-5">Current methods for studying novel β-CoVs are technically demanding. Viral isolation from field samples is rarely successful and reverse genetics recovery of recombinant virus is labor-intensive, and expensive as synthesis of a single genome can cost upwards of $15,000. These limitations are prohibitive to studying novel CoVs at the scale in which they are discovered.</p><p id="p-6">Cell entry is an essential component of cross-species transmission, especially for the β-CoVs. All CoVs encode a surface glycoprotein, spike, which binds to the host-cell receptor and mediates viral entry<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. For β-CoVs, a single region of the spike protein called the receptor binding domain (RBD) mediates the interaction with the host cell receptor. After binding the receptor, a nearby host protease cleaves the spike, which releases the spike fusion peptide, facilitating virus entry<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>–<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. Known host receptors for β-CoVs include angiotensin converting enzyme 2 (ACE2) for SARS-CoV and dipeptidyl peptidase 4 (DPP4) for MERS-CoV<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>.</p><p id="p-7">Structural studies of coronaviruses have shown that the spike RBD is capable of folding independently from the rest of the spike protein and contains all of the structural information for host receptor binding<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Additionally, a previous study showed that replacing the RBD of the lineage B bat virus, Rp3, allowed the virus to enter cells expressing human ACE2 (hACE2)<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. We therefore developed a method to functionally test the RBDs from novel lineage B β-CoVs in place of the SARS-CoV spike RBD (<a id="xref-fig-1-1" class="xref-fig" href="#F1">figure 1</a>). Synthesizing just the RBD of spike is much faster and cost-effective than conventional pseudotyping methods that rely on synthesis of the full ∼4kb spike sequence for novel CoVs: a process that can take weeks and is cost-prohibitive for large panels of spike sequences. The short turnaround time for our approach allowed us to test the receptor usage of all published, unique RBD sequences in lineage B, and also rapidly confirm the ACE2 receptor usage of the 2019-nCoV spike, which emerged in China in January 2020 as our study was ongoing.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="&#x3B2;-Coronavirus lineage B entry with human ACE2 is clade-specific a, &#x3B2;-Coronaviruses, including SARS-CoV, interact with the host cell receptor via the Receptor Binding Domain (RBD) in spike (PDB: 5X5B, 2AJF) . b, SARS-CoV spike was engineered with silent mutations to facilitate cloning novel RBD sequences in place of the SARS spike RBD. SARS spike amino acid numbers are indicated for silent cloning sites and the RBD in grey and orange, respectively. c, outline of experimental workflow. d, Westernblot of producer cell lysates and concentrated reporter particles. e, BHK cells were transfected with either human ACE2 or empty vector, subsequently infected with VSV-reporter particles pseudotyped with chimeric spikes, luciferase was measured and normalized to no pseudotype as a readout for cell entry. Shown are the data for 3 replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1672825997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;&#x3B2;-Coronavirus lineage B entry with human ACE2 is clade-specific&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, &#x3B2;-Coronaviruses, including SARS-CoV, interact with the host cell receptor via the Receptor Binding Domain (RBD) in spike (PDB: 5X5B, 2AJF) . &lt;strong&gt;b&lt;/strong&gt;, SARS-CoV spike was engineered with silent mutations to facilitate cloning novel RBD sequences in place of the SARS spike RBD. SARS spike amino acid numbers are indicated for silent cloning sites and the RBD in grey and orange, respectively. &lt;strong&gt;c&lt;/strong&gt;, outline of experimental workflow. &lt;strong&gt;d&lt;/strong&gt;, Westernblot of producer cell lysates and concentrated reporter particles. &lt;strong&gt;e&lt;/strong&gt;, BHK cells were transfected with either human ACE2 or empty vector, subsequently infected with VSV-reporter particles pseudotyped with chimeric spikes, luciferase was measured and normalized to no pseudotype as a readout for cell entry. Shown are the data for 3 replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F1.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F1.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">β-Coronavirus lineage B entry with human ACE2 is clade-specific</span><p id="p-8" class="first-child"><strong>a</strong>, β-Coronaviruses, including SARS-CoV, interact with the host cell receptor via the Receptor Binding Domain (RBD) in spike (PDB: 5X5B, 2AJF) . <strong>b</strong>, SARS-CoV spike was engineered with silent mutations to facilitate cloning novel RBD sequences in place of the SARS spike RBD. SARS spike amino acid numbers are indicated for silent cloning sites and the RBD in grey and orange, respectively. <strong>c</strong>, outline of experimental workflow. <strong>d</strong>, Westernblot of producer cell lysates and concentrated reporter particles. <strong>e</strong>, BHK cells were transfected with either human ACE2 or empty vector, subsequently infected with VSV-reporter particles pseudotyped with chimeric spikes, luciferase was measured and normalized to no pseudotype as a readout for cell entry. Shown are the data for 3 replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">We show that lineage B RBDs divide into functionally distinct clades and that several previously-unappreciated viruses exhibit compatibility with an unknown receptor on human cells. We also show that these clades are capable of recombining to impart human host-cell entry phenotypes, and that, beyond the RDB-receptor interaction, host protease processing is another species barrier encountered by lineage B β-CoVs during cell entry.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Cells</h3><p id="p-10">293T, A549, BHK, Caco-2, Huh-7.5, PK-15, and Vero cells were maintained in DMEM (Sigma) supplemented with 10% FBS, penicillin/streptomycin, and L-glutamine. RhiNi/40.1, AJ-primary, AJ<em>i</em>, HypNi, RaKSM-2.5i, RhiLu, and RhiNi cells were maintained in DMEM/F12 (Gibco) supplemented with 12% FBS, penicillin/streptomycin, non- essential amino acids, sodium pyruvate and L-glutamine. AJ-primary cells were immortalized with a lentiviral vector expressing SV40 T-antigen following the manufacturer’s instructions to generate AJ<em>i</em> cells (abm; #G203). RaKSM-2.5 primary cells have been previously described and were immortalized in this study similar to AJ<em>i</em> cells<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>.</p></div><div id="sec-4" class="subsection"><h3>Plasmids</h3><p id="p-11">The spike coding sequences for SARS-CoV Urbani, As6526, and BM48-31 were codon optimized for human cells, appended with a 5’ kozak expression sequence (GCCACC) and 3’ tetra-glycine linker followed by nucleotides encoding a FLAG tag sequence (DYKDDDDK). For SARS-CoV spike, silent mutations were introduced around codons 308 and 519 to form KpnI and XhoI digest sites. For As6526 spike, silent mutations were introduced around codons 290 and 501 to form AflII and HindIII digest sites. For BM48-31 spike, silent mutations were introduced around codons 295 and 501 to form AflII and HindIII digest sites. These engineered spike sequences were synthesized and cloned into pcDNA3.1+ (GenScript).</p><p id="p-12">Spike RBDs were first codon-optimized for human cells, appended with regions of the target spike backbone to facilitate Infusion cloning and synthesized as double stranded DNA fragments (IDT DNA). SARS-CoV, As6526 or BM48-31 engineered spike plasmids were digested with their corresponding restriction enzymes and gel purified. RBD inserts were resuspended in water and Infusion cloned into gel purified, digested spike backbone vectors (Takara).</p><p id="p-13">Human ACE2 (Q9BYF1.2), DPP4 (XM_005246371.3), or APN (NP_001141.2) were synthesized and cloned into pcDNA3.1+ (GenScript). All DNA constructs were verified by Sanger sequencing (ACGT Inc.).</p></div><div id="sec-5" class="subsection"><h3>Receptor transfection</h3><p id="p-14">BHK cells were seeded in black 96-well plates and transfected the next day with 100ng plasmid DNA encoding human ACE2, DPP4, APN or empty vector, using polyethyleneimine (Polysciences). All downstream experiments were performed 24 hours post-transfection.</p></div><div id="sec-6" class="subsection"><h3>Pseudotype production</h3><p id="p-15">Pseudotypes were produced as previously described<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>. 293T cells were seeded onto 6-well plates pre-coated with poly-L-lysine (Sigma) and transfected the next day with 1200ng empty plasmid and 400ng of plasmid encoding coronavirus spike or GFP as a no pseudotype control. Twenty-four hours later, transfected cells were infected with VSVΔG particles pseudotyped with VSV-g as previously described<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. After one hour of incubating at 37°C, cells were washed three times and incubated in 2mL DMEM supplemented with 2% FBS, penicillin/streptomycin, and L-glutamine for 48 hours. Supernatants were collected, centrifuged at 500xG for 5 minutes, aliquoted and stored at -80.</p></div><div id="sec-7" class="subsection"><h3>Luciferase-based cell entry assay</h3><p id="p-16">Target cells were seeded in black 96-well plates and inoculated, in triplicate, with equivalent volumes of pseudotype stocks. For trypsin experiments, pseudotype stocks were diluted 1:1 in DMEM without FBS, trypsin was added to a final concentration of 2500μg/mL and samples were incubated at 37°C for 15 minutes. Samples were then diluted again 1:1 in cold DMEM supplemented with 2% FBS and added to cells. Inoculated plates were centrifuged at 1200xG, 4°C, for 1 hour and incubated over night at 37°C. Approximately 18-20 hours post-infection, Bright-Glo luciferase reagent (Promega) was added to each well, 1:1, without removing culture media and luciferase was measured. Relative entry was calculated by normalizing the relative light unit (RLU) for spike pseudotypes to the plate RLU average for the “no pseudotype control.”</p></div><div id="sec-8" class="subsection"><h3>Western blot</h3><p id="p-17">Producer cells (spike-transfected 293T) were lysed in 1%SDS, 150mM NaCl, 50mM Tris-HCl, 5mM EDTA and clarified by centrifugation at 14000xG for 20 minutes. Pseudotyped particles were concentrated from producer cell lysates that were overlaid a 10% OptiPrep cushion in PBS (Sigma) and centrifuged at 20,000× g for 2 hours at 4 °C. Lysates and concentrated particles were analyzed for FLAG, GAPDH and/or VSV-m expression on 10% Bis-Tris PAGE gel (ThermoFisher).</p></div><div id="sec-9" class="subsection"><h3>Accession numbers</h3><p id="p-18">Accession numbers for all spike sequences used here can be found in figure s1b.</p></div></div><div class="section" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>ACE2 entry is lineage B clade 1-specific</h3><p id="p-19">The receptor binding domain (RBD) of lineage B β-CoVs is a single, continuous domain that contains all structural information necessary to interact with the host receptor (<a id="xref-fig-1-2" class="xref-fig" href="#F1">figure 1a, b</a>). We introduced silent mutations in the codon optimized coding sequence for SARS-CoV to facilitate replacing the SARS RBD with the RBD from other lineage B viruses (<a id="xref-fig-1-3" class="xref-fig" href="#F1">figure 1b</a>). All lineage B sequences were downloaded from online repositories and parsed to 29 unique RBD sequences, representing all published variations of the lineage B RBD (supp. fig 1a, b). The panel of 29 RBDs phylogenetically cluster into 3 clades, as previously described<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>, but these RBD clades were not apparent in phylogenetic analysis of other viral sequences, such as the RNA-dependent RNA polymerase (supp. fig 1c). All 29 RBDs were codon optimized, synthesized and cloned in place of the SARS RBD, effectively generating chimeric spike expression constructs. We then generated VSV-luciferase reporter particles pseudotyped with the chimeric spikes (<a id="xref-fig-1-4" class="xref-fig" href="#F1">figure 1c</a>). We chose VSV over lentiviruses as our pseudotype platform because a lentiviral pseudotypes have failed to accurately reflect viral entry with novel bat coronavirus spike protein<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a></sup>. All constructs exhibited similar levels of expression in producer cells and incorporation into VSV pseudotypes, except the chimera with BM48-31 which displayed somewhat reduced expression compared to WT SARS spike (<a id="xref-fig-1-5" class="xref-fig" href="#F1">figure 1d</a>). We then infected BHK cells expressing the receptor for SARS-CoV or empty vector (<a id="xref-fig-1-6" class="xref-fig" href="#F1">figure 1e</a>) and observed only clade 1, which includes SARS-CoV, WIV1 and SHC014, could enter cells transfected with human ACE2 (<a id="xref-fig-1-7" class="xref-fig" href="#F1">figure 1e</a>).</p></div><div id="sec-12" class="subsection"><h3>Protease enhances clade 2 entry</h3><p id="p-20">After binding the host receptor, host-cell protease cleaves spike, releasing the fusion peptide and allowing for host cell entry <sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>. Previous studies have shown that absence of the host protease or incompatibility between the host protease and viral spike can block viral entry <sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>–<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. To circumvent host-cell protease incompatibility or absence, we treated our lineage B pseudotype panel and infected a wide variety of cell types from different host species (<a id="xref-fig-2-1" class="xref-fig" href="#F2">figure 2</a>, supp. fig. 2). In the absence of exogenous protease, only clade 1 infected cells from African green monkey kidney, human gastrointestinal tract, human liver, and porcine kidney, in agreement with previous studies (<a id="xref-fig-2-2" class="xref-fig" href="#F2">figure 2</a>; supp. fig. 2a, b). Surprisingly, exogenous protease enhanced entry of a subset of clade 2 spike chimeras in nonhuman primate, bat and human cells (<a id="xref-fig-2-3" class="xref-fig" href="#F2">figure 2</a>). Importantly, VSV-g pseudotyped particles were able to produce luciferase signal in all cell lines tested in this study (supp. fig. 2c).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trypsin enhances lineage B entry in various cell lines a, Primate cells, b, human cells or c, bat cells were infected with VSV-particles pseudotyped with the lineage B chimeric spike panel. Pseudotypes were either left untreated or incubated with trypsin before addition to the cells. Luciferase was measured and normalized to particles produced without pseudotype. Shown are the data for 3 replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1672825997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Trypsin enhances lineage B entry in various cell lines&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Primate cells, &lt;strong&gt;b&lt;/strong&gt;, human cells or &lt;strong&gt;c&lt;/strong&gt;, bat cells were infected with VSV-particles pseudotyped with the lineage B chimeric spike panel. Pseudotypes were either left untreated or incubated with trypsin before addition to the cells. Luciferase was measured and normalized to particles produced without pseudotype. Shown are the data for 3 replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F2.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F2.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Trypsin enhances lineage B entry in various cell lines</span><p id="p-21" class="first-child"><strong>a</strong>, Primate cells, <strong>b</strong>, human cells or <strong>c</strong>, bat cells were infected with VSV-particles pseudotyped with the lineage B chimeric spike panel. Pseudotypes were either left untreated or incubated with trypsin before addition to the cells. Luciferase was measured and normalized to particles produced without pseudotype. Shown are the data for 3 replicates.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Clade 2 entry is receptor-dependent</h3><p id="p-22">We next tested human variants of known β-CoV receptors for their ability to mediate cell entry of clade 2 and 3 spike chimeras. We also tested human aminopeptidase N (APN), a receptor for alphacoronaviruses, which have been shown to utilize either human ACE2 or human APN for cell entry (<a id="xref-fig-3-1" class="xref-fig" href="#F3">figure 3a</a>). Protease treatment only enhanced entry of clade 1 RBDs on cells expressing human ACE2, but not human DPP4 or APN. No entry was observed with clade 2 or 3 spikes, regardless of receptor or protease addition. Human dipeptidyl peptidase IV (DPP4), the receptor for the lineage C β-CoVs, MERS-CoV, only mediated entry of MERS-CoV (<a id="xref-fig-2-4" class="xref-fig" href="#F2">figure 2b</a>, middle panels). Importantly, in the absence of receptor, no entry was observed for any of the pseudotypes, suggesting that protease-mediated entry is receptor-dependent (<a id="xref-fig-2-5" class="xref-fig" href="#F2">figure 2b</a>, right panels).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lineage B entry into cells with known CoV receptors a, Schematic of known coronavirus RBDs and their receptors. b, Pseudotyped particles were either left untreated and treated with trypsin and subsequently used to infect BHK cells transfected with the coronavirus receptor indicated. Shown are data from 3 replicates. c, Expression and pseudotype incorporation of SARS-S-2019-nCoV RBD chimeras. d, Pseudotypes were used to infect cells expressing hACE2, hDPP4, hAPN, or empty vector, without protease treatment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1672825997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Lineage B entry into cells with known CoV receptors&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Schematic of known coronavirus RBDs and their receptors. &lt;strong&gt;b&lt;/strong&gt;, Pseudotyped particles were either left untreated and treated with trypsin and subsequently used to infect BHK cells transfected with the coronavirus receptor indicated. Shown are data from 3 replicates. &lt;strong&gt;c&lt;/strong&gt;, Expression and pseudotype incorporation of SARS-S-2019-nCoV RBD chimeras. &lt;strong&gt;d&lt;/strong&gt;, Pseudotypes were used to infect cells expressing hACE2, hDPP4, hAPN, or empty vector, without protease treatment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F3.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F3.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">Lineage B entry into cells with known CoV receptors</span><p id="p-23" class="first-child"><strong>a</strong>, Schematic of known coronavirus RBDs and their receptors. <strong>b</strong>, Pseudotyped particles were either left untreated and treated with trypsin and subsequently used to infect BHK cells transfected with the coronavirus receptor indicated. Shown are data from 3 replicates. <strong>c</strong>, Expression and pseudotype incorporation of SARS-S-2019-nCoV RBD chimeras. <strong>d</strong>, Pseudotypes were used to infect cells expressing hACE2, hDPP4, hAPN, or empty vector, without protease treatment.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Receptor usage of 2019-nCoV</h3><p id="p-24">While our study was ongoing, a novel lineage B virus tentatively named 2019- nCoV was identified as the cause of a pneumonia outbreak in Hubei, China. Once the sequence was publicly available, we synthesized, cloned and tested the RBD from 2019- nCoV in our assay with human variants of known coronavirus receptors. The chimeric SARS-2019-nCoV spike protein expressed and was incorporated into particles similarly to other clade 1 chimeric spikes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">figure 3c</a>). The 2019-nCoV RBD was capable of entering cells expressing human ACE2, but not any of the other receptors tested (<a id="xref-fig-3-3" class="xref-fig" href="#F3">figure 3d</a>; s3).</p></div><div id="sec-15" class="subsection"><h3>Clade determinants for ACE2 usage</h3><p id="p-25">Consensus sequences of the three lineage B clades showed several key differences between these groups. Only clade 1 RBDs contain all 14 residues that have been shown through crystallography, to interact with human ACE2 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">figure 4a</a>; s4). The majority of these residues are absent from clades 2 and 3, which contain additional deletions in surface exposed loops that cluster at the interface with ACE2 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">figure 4</a> a, b). We generated a series of clade consensus RBD variants to determine the minimum number of mutations needed to impart ACE2 function on clade 2 and 3 RBDs (<a id="xref-fig-4-3" class="xref-fig" href="#F4">figure 4c</a>). Introducing the two loop deletions from clade 1 in clade 2 results in a reduced spike expression, impaired pseudotype incorporation and loss of cell entry (<a id="xref-fig-4-4" class="xref-fig" href="#F4">figure 4c, d</a>). Restoring these loops in clade 2 and 3 from the loops found in clade 1 did not enhance entry with ACE2 (<a id="xref-fig-4-5" class="xref-fig" href="#F4">figure 4c</a>; 2→1 and 3→1 version 1). Introducing all 14 ACE2 contact points in clade 2 or 3 also failed to restore ACE2 entry (<a id="xref-fig-4-6" class="xref-fig" href="#F4">figure 4c</a>; 2→1 and 3→1 version 2). Only replacing all 14 contact points and the surrounding amino acids (also known as the receptor binding motif, RBM) lead to increased ACE2 entry with clade 2 and 3 RBDs (<a id="xref-fig-4-7" class="xref-fig" href="#F4">figure 4c</a>; 2→1 version 3 = clade 2 residues 322-400 + clade 1 residues 400-501; 3→1 version 3 = clade 3 residues 322-385 + clade 1 residues 386-501). Taken together, these results show that the entire RBM from clade 1 is needed for ACE2 entry.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lineage B clade-specific determinants for human ACE2 usage a, Schematic overview of clade 1, 2 and 3. Highlighted in yellow are the 14 residues that contact human ACE2. Deletions in loops 1 and 2 are indicated for clades 2 and 3. b, Structure of human ACE2 and the SARS-S RBD (PDB: 2AJF), with loops highlighted in gray. c, VSV pseudotypes were generated with the indicated RBD and used to infect BHKs transfected with either human ACE2 or empty vector. Shown are data for 3 replicates. d, Westernblot of producer cell lysates and concentrated pseudotyped particles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1672825997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Lineage B clade-specific determinants for human ACE2 usage&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Schematic overview of clade 1, 2 and 3. Highlighted in yellow are the 14 residues that contact human ACE2. Deletions in loops 1 and 2 are indicated for clades 2 and 3. &lt;strong&gt;b&lt;/strong&gt;, Structure of human ACE2 and the SARS-S RBD (PDB: 2AJF), with loops highlighted in gray. &lt;strong&gt;c&lt;/strong&gt;, VSV pseudotypes were generated with the indicated RBD and used to infect BHKs transfected with either human ACE2 or empty vector. Shown are data for 3 replicates. &lt;strong&gt;d&lt;/strong&gt;, Westernblot of producer cell lysates and concentrated pseudotyped particles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F4.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F4.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">Lineage B clade-specific determinants for human ACE2 usage</span><p id="p-26" class="first-child"><strong>a</strong>, Schematic overview of clade 1, 2 and 3. Highlighted in yellow are the 14 residues that contact human ACE2. Deletions in loops 1 and 2 are indicated for clades 2 and 3. <strong>b</strong>, Structure of human ACE2 and the SARS-S RBD (PDB: 2AJF), with loops highlighted in gray. <strong>c</strong>, VSV pseudotypes were generated with the indicated RBD and used to infect BHKs transfected with either human ACE2 or empty vector. Shown are data for 3 replicates. <strong>d</strong>, Westernblot of producer cell lysates and concentrated pseudotyped particles.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Full-spike and RBD chimeras are comparable</h3><p id="p-27">We next synthesized full-length clade 2 and 3 spikes to compare to our RBD chimeras. We selected the clade 2 spike, As6526, because it consistently gave strong entry signal in human cells following protease-treatment (<a id="xref-fig-2-6" class="xref-fig" href="#F2">figure 2b</a>) as well as BM48-31, the only clade 3 spike in our panel. As we did for SARS-CoV spike, clade 2 and 3 spikes were codon optimized, FLAG-tagged and silent mutations were introduced to facilitate replacing their RBD with the consensus RBD from clade 1 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">figure 5a</a>). All chimeric constructs expressed similarly, with the exception for the SARS-BM48-31 RBD chimera, which exhibited reduced expression and incorporation (<a id="xref-fig-5-2" class="xref-fig" href="#F5">figure 5b</a>). Protease treatment enhanced entry of both the As6526 clade 2 RBD chimera and full-length spike entry into Huh cells (<a id="xref-fig-5-3" class="xref-fig" href="#F5">figure 5c</a>). Protease treatment had no effect on either the BM48-31 clade 3 chimera or full-length spike (<a id="xref-fig-5-4" class="xref-fig" href="#F5">figure 5c</a>). Taken together, these findings show that SARS- lineage B RBD chimeras reflect the entry phenotype of full-length lineage B spikes. Finally, we tested if receptor-binding and protease processing are coupled. We replaced the RBD of full-length clade 2 and 3 spike with the consensus RBD from clade</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of chimeric and full-length clade 2 and 3 spikes a, Full length spike sequences from As6526 (clade 2) and BM48-31 (clade 3) were codon optimized, FLAG tagged and synthesized. Silent mutations flanking the RBD facilitated replacing the native RBD with the clade 1 consensus RBD. b, Westernblot of producer cell lysates and concentrated pseudotypes particles. c, Pseudotypes with indicated spike constructs were left untreated or treated with trypsin and subsequently used to infect Huh-7.5 cells. Shown are data for 3 replicates. d, Pseudotypes with indicated spike constructs were left untreated or treated with trypsin and subsequently used to infect BHK cells transfected with human ACE2. Shown are data for 3 replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1672825997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Comparison of chimeric and full-length clade 2 and 3 spikes&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;, Full length spike sequences from As6526 (clade 2) and BM48-31 (clade 3) were codon optimized, FLAG tagged and synthesized. Silent mutations flanking the RBD facilitated replacing the native RBD with the clade 1 consensus RBD. &lt;strong&gt;b&lt;/strong&gt;, Westernblot of producer cell lysates and concentrated pseudotypes particles. &lt;strong&gt;c&lt;/strong&gt;, Pseudotypes with indicated spike constructs were left untreated or treated with trypsin and subsequently used to infect Huh-7.5 cells. Shown are data for 3 replicates. &lt;strong&gt;d&lt;/strong&gt;, Pseudotypes with indicated spike constructs were left untreated or treated with trypsin and subsequently used to infect BHK cells transfected with human ACE2. Shown are data for 3 replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F5.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F5.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/22/2020.01.22.915660/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <span class="caption-title">Comparison of chimeric and full-length clade 2 and 3 spikes</span><p id="p-28" class="first-child"><strong>a</strong>, Full length spike sequences from As6526 (clade 2) and BM48-31 (clade 3) were codon optimized, FLAG tagged and synthesized. Silent mutations flanking the RBD facilitated replacing the native RBD with the clade 1 consensus RBD. <strong>b</strong>, Westernblot of producer cell lysates and concentrated pseudotypes particles. <strong>c</strong>, Pseudotypes with indicated spike constructs were left untreated or treated with trypsin and subsequently used to infect Huh-7.5 cells. Shown are data for 3 replicates. <strong>d</strong>, Pseudotypes with indicated spike constructs were left untreated or treated with trypsin and subsequently used to infect BHK cells transfected with human ACE2. Shown are data for 3 replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">1 and tested pseudotypes on cells expressing the clade 1 receptor. The clade 1 consensus RBD efficiently facilitated entry of both As6526 and BM48-31 spike only following protease treatment. These findings show that even though BHK-hACE2 cells support full-length clade 1 spike entry, just having the RBD from a clade 1 virus is insufficient to mediate entry. As seen in our previous experiments, protease treatment did not enhance pseudotype entry in the absence of host receptor (<a id="xref-fig-5-5" class="xref-fig" href="#F5">figure 5</a>).</p></div></div><div class="section" id="sec-17"><h2 class="">Discussion</h2><p id="p-30">Despite significant advances in next generation sequencing technologies, which have facilitated the discovery of thousands of novel animal-derived viruses, tools for downstream functional assessment of these novel sequences are lacking. To gain traction on this ever-growing problem, we took a reductionist approach to coronavirus entry and developed a scalable, BSL2-compatible method for testing only the minimal region of the virus essential for interacting with the host receptor (<a id="xref-fig-1-8" class="xref-fig" href="#F1">figure 1</a>, figure s1a). Because most of these viruses have never been isolated, we resorted to synthetic biology and molecular engineering to reduce the burden of gene synthesis to just a small fragment. Thus, the cost and synthesis production time for testing several spikes for entry in our system is dramatically reduced (figure s1d). In theory, this approach to functional viromics should be applicable to a wide variety of virus-host proteins and interactions.</p><p id="p-31">Coronavirus entry is a multi-step process involving multiple, distinct domains in spike that mediate virus attachment to the cell surface, receptor engagement, protease processing and membrane fusion<sup><a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a></sup>. While the RBD:receptor interaction is most studied in this process, recent studies have highlighted the major role host protease processing plays as a species barrier<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>–<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. Lineage C coronaviruses include MERS-CoV as well as distantly related viruses such as HKU4, HKU5 and PDF-2180 <sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. Studies have shown that HKU4 can bind human DPP4 but requires addition of exogenous trypsin to facilitate cell entry and that HKU5 and PDF-2180 spikes can enter human cells through an unidentified receptor with protease treatment<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Analogous to these earlier studies of lineage C CoVs, we observed protease-enhanced entry of lineage B CoVs (<a id="xref-fig-2-7" class="xref-fig" href="#F2">figure 2</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">3</a>, <a id="xref-fig-5-6" class="xref-fig" href="#F5">5</a>). While it has been shown that host proteases cleave spike, allowing for downstream membrane fusion, other evidence suggests that protease may act on the receptor as well to activate it<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. Addition of protease during the course of SARS-CoV infection facilitated entry in cells with low-expression of ACE2 that is normally insufficient to support virus entry<sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup>. Indeed, we saw evidence of residual trypsin activity on the cells after infection in our studies as the cell monolayer was loose compared to the untreated condition. Similarly, Menachery et al. also observed cell rounding during their trypsin infections<sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a></sup>. Therefore, further studies are needed to assess where trypsin is enhancing entry of coronaviruses: at the level of spike, the receptor, or both.</p><p id="p-32">In the absence of exogenous protease, only clade 1 RBDs entered nonhuman primate, human and porcine cell lines (<a id="xref-fig-2-8" class="xref-fig" href="#F2">figure 2a, b</a>, s2a, b). These findings are in strong agreement with previous studies that have either isolated virus (WIV1) or rescued recombinant chimeric viruses (SHC014, Rs4231, Rs7237)<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>. However, with trypsin, a subset of genetically-similar clade 2 RBDs gained entry in these cells, suggesting their barrier is at the level of protease processing (<a id="xref-fig-2-9" class="xref-fig" href="#F2">figure 2a, b</a>). The other spike from clade 2 and 3 did not enter the cells we tested, regardless of protease addition, suggesting an absent or incompatible receptor. Surprisingly, the protease-dependent entry phenotype was consistent in the reverse spike chimeras in which we replaced the RBD in clade 2 or 3 spike with a clade 1 RBD (<a id="xref-fig-5-7" class="xref-fig" href="#F5">figure 5d</a>), suggesting that either the protease site between S1/S2 is not compatible with the chimeric spike backbone or the protease is not expressed in these cells (figure s5). Because clade 1 spikes enter cells expressing human ACE2 without addition of protease but clade2-clade1 chimeras require protease, our data suggests the spike protease cleavage site is adapted to the protease environment of the receptor-bound RBD (figure s5).</p><p id="p-33">None of the spike pseudotypes efficiently entered <em>Rhinolophus</em> cells, which has been observed in previous studies using these cells<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> (<a id="xref-fig-2-10" class="xref-fig" href="#F2">figure 2c</a>). Surprisingly, AJi cells were selectively permissive for only clade 2 entry following protease treatment, which suggests that clade 2 RBDs interact with a receptor that is distinct from clade 1 (<a id="xref-fig-2-11" class="xref-fig" href="#F2">figure 2c</a>).</p><p id="p-34">Our results show that, despite all being classified as the same virus species, most lineage B β-CoVs do not use currently known coronavirus receptors (<a id="xref-fig-1-9" class="xref-fig" href="#F1">figure 1e</a>, <a id="xref-fig-3-5" class="xref-fig" href="#F3">3a, b</a>). Critically, we did not observe any pseudotype entry in the presence of protease and absence of receptor, suggesting that lineage B cell entry is still receptor-dependent following protease treatment (<a id="xref-fig-3-6" class="xref-fig" href="#F3">figure 3b</a>). While our study was ongoing, a novel lineage B β-CoV was identified as the etiological agent behind an outbreak of pneumonia in Wuhan, Hubei, China (2019-nCoV). The RBD for 2019-nCoV has residues and motifs found in all 3 clades but forms a distinct clade, so we tested it for receptor usage and observed entry only with human ACE2 but not other known coronavirus receptors (<a id="xref-fig-3-7" class="xref-fig" href="#F3">figure 3d</a>). Interestingly, within the backbone of SARS-CoV spike, cell entry of 2019-nCoV was similar to the other clade 1 spikes tested, including SARS-CoV. These finding suggests 2019-nCoV is capable of using human ACE2 as efficiently as SARS-CoV, which may help explain the human-to-human transmissibility of this virus. More studies are needed with the full spike sequence and, ideally, a viral isolate.</p><p id="p-35">The receptor binding motif (RBM) is a small region in the C-terminal half of the RBD and contains all the residues that interface with the host receptor (<a id="xref-fig-3-8" class="xref-fig" href="#F3">figure 3a</a>)<sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup>. The 14 contact points in the co-structure of the SARS-RBD bound to human ACE2 are largely absent from clade 2 and 3 RBDs, which also contain deletions compared to clade 1 RBMs (<a id="xref-fig-4-8" class="xref-fig" href="#F4">figure 4a, b</a>, s4a). Simply mutating clade 2 and 3 to have the 14 contact points was insufficient to impart human ACE2 usage (<a id="xref-fig-4-9" class="xref-fig" href="#F4">figure 4c</a>). This is likely because the non- contact residues in the RBM play a supportive and structural role for these contact points, and indeed, these non-contact residues are different between the clades (figure s4).</p><p id="p-36">In contrast to changing individual amino acids, our chimeric RBD constructs show that clade 2 and 3 RBD containing the clade 1 RBM are compatible with human ACE2. Coronaviruses frequently undergo recombination, gaining large swaths of genetic material at once <sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. Taken together with our data, it is possible that recombination with a clade 1 virus will impart compatibility with human ACE2. Interestingly, the 2019-nCoV RBD forms a clade that is distinct from the other 3 clades (figure s1c). However, the 2019- nCoV RBD contains most of the contact points with human ACE2 that are found in clade 1 as well as some amino acid variations that are unique to clade 2 and 3 (figure s4b). Taken together with our receptor assay results, it may be possible that 2019-nCoV arose from recombination between clade 1 and the other clades.</p><p id="p-37">As we saw with the SARS-As6526 RBD (clade 2) spike chimera, full length As6526 spike entered cells following protease treatment, but BM48-31 (clade 3) spike did not (<a id="xref-fig-2-12" class="xref-fig" href="#F2">figure 2</a>, <a id="xref-fig-5-8" class="xref-fig" href="#F5">5c</a>). These data show that the chimeric spikes generally reproduce the entry phenotypes of full-length spikes. Notably, the full length As6526 spike did not enter cells as efficiently as the SARS-As6526 chimera, suggesting that other human-cell adaptations are likely needed in As6526 spike.</p><p id="p-38">The capacity to predict the zoonotic potential of newly detected viruses has been severely hindered by a lack of functional data for these novel animal virus sequences. Here, we have developed a rapid and cost-effective platform to functionally test large groups of related viruses for zoonotic potential. We found that several other lineage B coronaviruses are capable of entering human cells through an unknown receptor and that lineage B spike proteins can recombine to gain entry with a known host-receptor. Taken together with the latest outbreak of 2019-nCoV in humans, these findings underscore the importance of continued surveillance of coronaviruses at the sequence and functional levels in order to better prepare for the next emerging virus.</p></div><div class="section" id="sec-19"><h2 class="">Declarations of Interests</h2><p id="p-40">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-39">We would like to thank Dr. Anthony Schountz for generously sharing primary artibeus cells, as well as Dr. Marcel Muller for providing the <em>Rhinolophus</em> cells used in this study. We would also like to acknowledge Wu,F., Zhao,S., Yu,B., Chen,Y.-M., Wang,W., Hu,Y., Song,Z.-G., Tao,Z.-W., Tian,J.-H., Pei,Y.-Y., Yuan,M.L., Zhang,Y.-L., Dai,F.-H., Liu,Y., Wang,Q.-M., Zheng,J.-J., Xu,L., Holmes,E.C. and Zhang,Y.-Z, the Wuhan Institute of Virology, Chinese Academy of Medical Sciences &amp; Peking Union Medical College, Chinese Center for Disease Control and Prevention, and Wuhan Jinyintan Hospitar for providing the genome sequence for 2019-nCoV and other essential outbreak information to the scientific community. This work was supported by the Intramural Research Program of the National Institute of Allergy and Infectious Diseases.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.1" data-doi="10.3201/eid1112.041293"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-CoV infection in a restaurant from palm civet</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">1860</span>–<span class="cit-lpage">1865</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid1112.041293</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DWang%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D1860%26rft.epage%253D1865%26rft.atitle%253DSARS-CoV%2Binfection%2Bin%2Ba%2Brestaurant%2Bfrom%2Bpalm%2Bcivet.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1112.041293%26rft_id%253Dinfo%253Apmid%252F16485471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1112.041293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16485471&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233768300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.2" data-doi="10.1016/S0140-6736(03)13967-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuiken</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Newly discovered coronavirus as the primary cause of severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">362</span>, <span class="cit-fpage">263</span>–<span class="cit-lpage">270</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(03)13967-0</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DKuiken%26rft.auinit1%253DT.%26rft.volume%253D362%26rft.issue%253D9380%26rft.spage%253D263%26rft.epage%253D270%26rft.atitle%253DNewly%2Bdiscovered%2Bcoronavirus%2Bas%2Bthe%2Bprimary%2Bcause%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252803%252913967-0%26rft_id%253Dinfo%253Apmid%252F12892955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(03)13967-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12892955&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184413400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.3" data-doi="10.1126/science.1118391"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>, <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1118391</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8yMi8yMDIwLjAxLjIyLjkxNTY2MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.22.915660v1.4" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X. Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">503</span>, <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature12711</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.5" data-doi="10.1371/journal.ppat.1006698"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">13</span>, <span class="cit-issue">e1006698</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1006698</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253De1006698%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.22.915660v1.6" data-doi="10.1073/pnas.0506735102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">S. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">14040</span>–<span class="cit-lpage">14045</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0506735102</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0506735102%26rft_id%253Dinfo%253Apmid%252F16169905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzM5LzE0MDQwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjIvMjAyMC4wMS4yMi45MTU2NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.7" data-doi="10.1038/nm.3985"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence</span>. <abbr class="cit-jnl-abbrev">Nat Med</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">1508</span>–<span class="cit-lpage">1513</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nm.3985</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMed%26rft.volume%253D21%26rft.spage%253D1508%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.3985%26rft_id%253Dinfo%253Apmid%252F26552008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.3985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26552008&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.8" data-doi="10.1146/annurev-virology-110615-042301"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F</span></span>. <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">237</span>–<span class="cit-lpage">261</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-virology-110615-042301</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D3%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-110615-042301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-virology-110615-042301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.9" data-doi="10.1016/j.antiviral.2013.09.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simmons</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zmora</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gierer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heurich</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pohlmann</span>,  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">Proteolytic activation of the SARS-coronavirus spike protein: cutting enzymes at the cutting edge of antiviral research</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr> <span class="cit-vol">100</span>, <span class="cit-fpage">605</span>–<span class="cit-lpage">614</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.antiviral.2013.09.028</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2BRes%26rft.volume%253D100%26rft.spage%253D605%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2013.09.028%26rft_id%253Dinfo%253Apmid%252F24121034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2013.09.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24121034&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328434100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.22.915660v1.10" data-doi="10.1128/JVI.01542-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matsuyama</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Efficient activation of the severe acute respiratory syndrome coronavirus spike protein by the transmembrane protease TMPRSS2</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">12658</span>–<span class="cit-lpage">12664</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01542-10</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01542-10%26rft_id%253Dinfo%253Apmid%252F20926566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NC8yNC8xMjY1OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzIyLzIwMjAuMDEuMjIuOTE1NjYwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.22.915660v1.11" data-doi="10.1128/JVI.05300-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bertram</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cleavage and activation of the severe acute respiratory syndrome coronavirus spike protein by human airway trypsin-like protease</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">85</span>, <span class="cit-fpage">13363</span>–<span class="cit-lpage">13372</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.05300-11</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05300-11%26rft_id%253Dinfo%253Apmid%252F21994442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8yNC8xMzM2MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzIyLzIwMjAuMDEuMjIuOTE1NjYwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.12" data-doi="10.1073/pnas.0809524106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belouzard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">V. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R</span></span>. <span class="cit-article-title">Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">106</span>, <span class="cit-fpage">5871</span>–<span class="cit-lpage">5876</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0809524106</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0809524106%26rft_id%253Dinfo%253Apmid%252F19321428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzE0LzU4NzEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8yMi8yMDIwLjAxLjIyLjkxNTY2MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.13" data-doi="10.1038/nature12005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raj</span>,  <span class="cit-name-given-names">V. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>, <span class="cit-fpage">251</span>–<span class="cit-lpage">254</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature12005</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12005%26rft_id%253Dinfo%253Apmid%252F23486063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23486063&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316652300054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.14" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature02145</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.15" data-doi="10.1126/science.1116480"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S. C</span></span>. <span class="cit-article-title">Structure of SARS coronavirus spike receptor- binding domain complexed with receptor</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">309</span>, <span class="cit-fpage">1864</span>–<span class="cit-lpage">1868</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1116480</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjIvMjAyMC4wMS4yMi45MTU2NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.16" data-doi="10.1073/pnas.0808116105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">M. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Synthetic recombinant bat SARS-like coronavirus is infectious in cultured cells and in mice</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">105</span>, <span class="cit-fpage">19944</span>–<span class="cit-lpage">19949</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0808116105</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808116105%26rft_id%253Dinfo%253Apmid%252F19036930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUwLzE5OTQ0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjIvMjAyMC4wMS4yMi45MTU2NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.17" data-doi="10.1093/infdis/jiz429"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seifert</span>,  <span class="cit-name-given-names">S. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Rousettus aegyptiacus Bats Do Not Support Productive Nipah Virus Replication</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/infdis/jiz429</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjiz429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jiz429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.18" data-doi="10.1016/j.celrep.2018.07.045"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Letko</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Adaptive Evolution of MERS-CoV to Species Variation in DPP4</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1730</span>–<span class="cit-lpage">1737</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.celrep.2018.07.045</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D24%26rft.spage%253D1730%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2018.07.045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2018.07.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.19" data-doi="10.1073/pnas.94.26.14764"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takada</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A system for functional analysis of Ebola virus glycoprotein</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">94</span>, <span class="cit-fpage">14764</span>–<span class="cit-lpage">14769</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.94.26.14764</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTakada%26rft.auinit1%253DA.%26rft.volume%253D94%26rft.issue%253D26%26rft.spage%253D14764%26rft.epage%253D14769%26rft.atitle%253DA%2Bsystem%2Bfor%2Bfunctional%2Banalysis%2Bof%2BEbola%2Bvirus%2Bglycoprotein%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.26.14764%26rft_id%253Dinfo%253Apmid%252F9405687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTQvMjYvMTQ3NjQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8yMi8yMDIwLjAxLjIyLjkxNTY2MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.20" data-doi="10.1016/j.virusres.2014.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Millet</span>,  <span class="cit-name-given-names">J. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R</span></span>. <span class="cit-article-title">Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">202</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">134</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2014.11.021</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.11.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Trypsin treatment unlocks barrier for zoonotic coronaviruses infection</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.22" data-doi="10.1073/pnas.1405889111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Receptor usage and cell entry of bat coronavirus HKU4 provide insight into bat-to-human transmission of MERS coronavirus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">12516</span>–<span class="cit-lpage">12521</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1405889111</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1405889111%26rft_id%253Dinfo%253Apmid%252F25114257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM0LzEyNTE2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjIvMjAyMC4wMS4yMi45MTU2NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.22.915660v1.23" data-doi="10.1073/pnas.0503203102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matsuyama</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ujike</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morikawa</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tashiro</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Taguchi</span>,  <span class="cit-name-given-names">F</span></span>. <span class="cit-article-title">Protease-mediated enhancement of severe acute respiratory syndrome coronavirus infection</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">12543</span>–<span class="cit-lpage">12547</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0503203102</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0503203102%26rft_id%253Dinfo%253Apmid%252F16116101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzM1LzEyNTQzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjIvMjAyMC4wMS4yMi45MTU2NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.24" data-doi="10.1371/journal.pone.0007870"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kam</span>,  <span class="cit-name-given-names">Y. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cleavage of the SARS coronavirus spike glycoprotein by airway proteases enhances virus entry into human bronchial epithelial cells in vitro</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>, <span class="cit-issue">e7870</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0007870</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKam%26rft.auinit1%253DY.%2BW.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De7870%26rft.epage%253De7870%26rft.atitle%253DCleavage%2Bof%2Bthe%2BSARS%2Bcoronavirus%2Bspike%2Bglycoprotein%2Bby%2Bairway%2Bproteases%2Benhances%2Bvirus%2Bentry%2Binto%2Bhuman%2Bbronchial%2Bepithelial%2Bcells%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007870%26rft_id%253Dinfo%253Apmid%252F19924243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007870&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19924243&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.25" data-doi="10.1128/JVI.01279-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Two Mutations Were Critical for Bat-to-Human Transmission of Middle East Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">9119</span>–<span class="cit-lpage">9123</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01279-</span> <span class="cit-comment">15 (</span><span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01279-15%26rft_id%253Dinfo%253Apmid%252F26063432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xNy85MTE5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjIvMjAyMC4wMS4yMi45MTU2NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.26" data-doi="10.1128/JVI.01774-19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Trypsin treatment unlocks barrier for zoonotic bat coronaviruses infection</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01774-19</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01774-19%26rft_id%253Dinfo%253Apmid%252F31801868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjE0OiJKVkkuMDE3NzQtMTl2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzIyLzIwMjAuMDEuMjIuOTE1NjYwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.27" data-doi="10.1371/journal.pone.0001082"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beniac</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">deVarennes</span>,  <span class="cit-name-given-names">S. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andonov</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Booth</span>,  <span class="cit-name-given-names">T. F.</span></span> <span class="cit-article-title">Conformational reorganization of the SARS coronavirus spike following receptor binding: implications for membrane fusion</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>, <span class="cit-issue">e1082</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0001082</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D2%26rft.spage%253D1082e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.28" data-doi="10.1016/j.virol.2005.10.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">P. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular diversity of coronaviruses in bats</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">351</span>, <span class="cit-fpage">180</span>–<span class="cit-lpage">187</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2006.02.041</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DWoo%26rft.auinit1%253DP.%2BC.%26rft.volume%253D351%26rft.issue%253D1%26rft.spage%253D180%26rft.epage%253D187%26rft.atitle%253DMolecular%2Bdiversity%2Bof%2Bcoronaviruses%2Bin%2Bbats.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2005.10.032%26rft_id%253Dinfo%253Apmid%252F16647731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2005.10.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16647731&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239319900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.29" data-doi="10.1128/mBio.00373-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anthony</span>,  <span class="cit-name-given-names">S. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Further Evidence for Bats as the Evolutionary Source of Middle East Respiratory Syndrome Coronavirus</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">8</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/mBio.00373-17</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00373-17%26rft_id%253Dinfo%253Apmid%252F28377531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiOC8yL2UwMDM3My0xNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzIyLzIwMjAuMDEuMjIuOTE1NjYwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.30" data-doi="10.1128/JVI.02062-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shulla</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A transmembrane serine protease is linked to the severe acute respiratory syndrome coronavirus receptor and activates virus entry</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">85</span>, <span class="cit-fpage">873</span>–<span class="cit-lpage">882</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02062-10</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02062-10%26rft_id%253Dinfo%253Apmid%252F21068237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg1LzIvODczIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjIvMjAyMC4wMS4yMi45MTU2NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.31" data-doi="10.3201/eid2012.141280"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adney</span>,  <span class="cit-name-given-names">D. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Replication and shedding of MERS-CoV in upper respiratory tract of inoculated dromedary camels</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">1999</span>–<span class="cit-lpage">2005</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid2012.141280</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft.volume%253D20%26rft.spage%253D1999%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2012.141280%26rft_id%253Dinfo%253Apmid%252F25418529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2012.141280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25418529&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.32" data-doi="10.1038/s41598-018-33487-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muth</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Attenuation of replication by a 29 nucleotide deletion in SARS- coronavirus acquired during the early stages of human-to-human transmission</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">8</span>, <span class="cit-issue">15177</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-018-33487-8</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-33487-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-33487-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.33" data-doi="10.1371/journal.pone.0072942"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential sensitivity of bat cells to infection by enveloped RNA viruses: coronaviruses, paramyxoviruses, filoviruses, and influenza viruses</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>, <span class="cit-issue">e72942</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0072942</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De72942%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0072942%26rft_id%253Dinfo%253Apmid%252F24023659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0072942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24023659&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F22%2F2020.01.22.915660.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">C. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">M. M</span></span>. <span class="cit-article-title">RNA recombination in a coronavirus: recombination between viral genomic RNA and transfected RNA fragments</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">66</span>, <span class="cit-fpage">6117</span>–<span class="cit-lpage">6124</span> (<span class="cit-pub-date">1992</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLiao%26rft.auinit1%253DC%2BL%26rft.volume%253D66%26rft.issue%253D10%26rft.spage%253D6117%26rft.epage%253D6124%26rft.atitle%253DRNA%2Brecombination%2Bin%2Ba%2Bcoronavirus%253A%2Brecombination%2Bbetween%2Bviral%2Bgenomic%2BRNA%2Band%2Btransfected%2BRNA%2Bfragments.%26rft_id%253Dinfo%253Apmid%252F1326662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI2Ni8xMC82MTE3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjIvMjAyMC4wMS4yMi45MTU2NjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.22.915660v1.35" data-doi="10.1128/JVI.01394-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S</span></span>. <span class="cit-article-title">Recombination, reservoirs, and the modular spike: mechanisms of coronavirus cross-species transmission</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">3134</span>–<span class="cit-lpage">3146</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01394-09</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01394-09%26rft_id%253Dinfo%253Apmid%252F19906932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzcvMzEzNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzIyLzIwMjAuMDEuMjIuOTE1NjYwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.22.915660v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.21.914002v1" title="Post-prediction Inference" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.01.21.914606v1" title="Prenylation of axonally translated proteins controls NGF-dependent axon growth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 22, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.22.915660v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.01.22.915660v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.22.915660v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.22.915660v1" class="active" data-icon-position="" data-hide-link-title="0">Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1117250" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-dTIo_JRws_Fr9w3ha2ocL9uTF2-ruOl-VVD1TumPj9E" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1117250" id="node1117250--2" data-pisa="biorxiv;2020.01.22.915660v1" data-pisa-master="biorxiv;2020.01.22.915660" data-seqnum="1117250" data-apath="/biorxiv/early/2020/01/22/2020.01.22.915660.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Letko</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Vincent</span> <span class="nlm-surname">Munster</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.22.915660; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.22.915660 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;title=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;title=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;text=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;title=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;t=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;title=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;title=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1117250" id="node1117250--3" data-pisa="biorxiv;2020.01.22.915660v1" data-pisa-master="biorxiv;2020.01.22.915660" data-seqnum="1117250" data-apath="/biorxiv/early/2020/01/22/2020.01.22.915660.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Michael</span> <span class="nlm-surname">Letko</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Vincent</span> <span class="nlm-surname">Munster</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.22.915660; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.22.915660 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Letko%2C+Michael&amp;rft.au=Munster%2C+Vincent"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1117250/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1117250/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1117250/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1117250/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1117250/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1117250/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1117250/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1117250/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1117250/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1117250/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1117250/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1117250/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;count=horizontal&amp;via=&amp;text=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20lineage%20B%20%CE%B2-coronaviruses%2C%20including%202019-nCoV&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.22.915660v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.22.915660v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
