<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/embed/graphic-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2016/10/28/084129.atom" />
<meta name="HW.pisa" content="biorxiv;084129v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accurate prediction of human essential genes using only nucleotide composition and association information" />
<meta name="DC.Identifier" content="10.1101/084129" />
<meta name="DC.Date" content="2016-10-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Three groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a λ-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier’s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (&lt;https://cefg.uestc.edu.cn/Pheg&gt;), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the λ-interval Z curve method could be effectively extended to classification issues of other DNA elements." />
<meta name="DC.Contributor" content="Feng-Biao Biao" />
<meta name="DC.Contributor" content="Chuan Dong" />
<meta name="DC.Contributor" content="Hong-Li Hua" />
<meta name="DC.Contributor" content="Shuo Liu" />
<meta name="DC.Contributor" content="Hao Luo" />
<meta name="DC.Contributor" content="Hong-Wan Zhang" />
<meta name="DC.Contributor" content="Yan-Ting Jin" />
<meta name="DC.Contributor" content="Kai-Yue Zhang" />
<meta name="article:published_time" content="2016-10-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Accurate prediction of human essential genes using only nucleotide composition and association information" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Three groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a λ-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier’s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (https://cefg.uestc.edu.cn/Pheg), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the λ-interval Z curve method could be effectively extended to classification issues of other DNA elements.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="biorxiv;084129v1" />
<meta name="citation_id" content="084129v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/084129v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/084129v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/084129v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129.full.pdf" />
<meta name="citation_doi" content="10.1101/084129" />
<meta name="citation_num_pages" content="32" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="084129" />
<meta name="citation_author" content="Feng-Biao Biao" />
<meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Center for Informational Biology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China" />
<meta name="citation_author_email" content="fbguo@uestc.edu.cn" />
<meta name="citation_author" content="Chuan Dong" />
<meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Center for Informational Biology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China" />
<meta name="citation_author" content="Hong-Li Hua" />
<meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Center for Informational Biology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China" />
<meta name="citation_author" content="Shuo Liu" />
<meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Center for Informational Biology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China" />
<meta name="citation_author" content="Hao Luo" />
<meta name="citation_author_institution" content="Department of Physics, Tianjin University" />
<meta name="citation_author" content="Hong-Wan Zhang" />
<meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Center for Informational Biology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China" />
<meta name="citation_author" content="Yan-Ting Jin" />
<meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Center for Informational Biology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China" />
<meta name="citation_author" content="Kai-Yue Zhang" />
<meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Center for Informational Biology, University of Electronic Science and Technology of China" />
<meta name="citation_author_institution" content="Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China" />
<meta name="citation_reference" content="Amsterdam A, Nissen RM, Sun Z, Swindell EC, Farrington S, Hopkins N. 2004. Identification of 315 genes essential for early zebrafish development. Proc Natl Acad Sci U S A 101: 12792–12797." />
<meta name="citation_reference" content="Blomen VA, Májek P, Jae LT, Bigenzahn JW, Nieuwenhuis J, Staring J, Sacco R, van Diemen FR, Olk N, Stukalov A. 2015. Gene essentiality and synthetic lethality in haploid human cells. Science 350: 1092–1096." />
<meta name="citation_reference" content="Chen LL, Ou HY, Zhang R, Zhang CT. 2003. ZCURVE_CoV: a new system to recognize protein coding genes in coronavirus genomes, and its applications in analyzing SARS-CoV genomes. Biochem Biophys Res Commun 307: 382–388." />
<meta name="citation_reference" content="Chen WH, Minguez P, Lercher MJ, Bork P. 2012. OGEE: an online gene essentiality database. Nucleic Acids Res 40: D901–906." />
<meta name="citation_reference" content="Chen Y, Xu D. 2005. Understanding protein dispensability through machine-learning analysis of high-throughput data. Bioinformatics 21: 575–581." />
<meta name="citation_reference" content="Cheng J, Wu W, Zhang Y, Li X, Jiang X, Wei G, Tao S. 2013. A new computational strategy for predicting essential genes. BMC genomics 14: 910." />
<meta name="citation_reference" content="del Rio G, Koschützki D, Coello G. 2009. How to identify essential genes from molecular networks? BMC Syst Biol 3: 102." />
<meta name="citation_reference" content="Deng J, Deng L, Su S, Zhang M, Lin X, Wei L, Minai AA, Hassett DJ, Lu LJ. 2011. Investigating the predictability of essential genes across distantly related organisms using an integrative approach. Nucleic Acids Res 39: 795–807." />
<meta name="citation_reference" content="Dong C, Yuan YZ, Zhang FZ, Hua HL, Ye YN, Labena AA, Lin H, Chen W, Guo FB. 2016. Combining the pseudo dinucleotide composition with the Z curve method to improve the accuracy of predicting DNA elements: a case study in recombination spots. Mol Biosyst 12: 2893–2900." />
<meta name="citation_reference" content="Fan RE, Chang KW, Hsieh CJ, Wang XR, Lin CJ. 2008. LIBLINEAR: A library for large linear classification. J Mach Learn Res 9: 1871–1874." />
<meta name="citation_reference" content="Fraser A. 2015. Essential Human Genes. Cell Systems 1: 381–382." />
<meta name="citation_reference" content="Fullwood MJ, Liu MH, Pan YF, Liu J, Xu H, Mohamed YB, Orlov YL, Velkov S, Ho A, Mei PH et al. 2009. An oestrogen-receptor-alpha-bound human chromatin interactome. Nature 462: 58–64." />
<meta name="citation_reference" content="Gao F, Zhang CT. 2004. Comparison of various algorithms for recognizing short coding sequences of human genes. Bioinformatics 20: 673–681." />
<meta name="citation_reference" content="Gatto F, Miess H, Schulze A, Nielsen J. 2015. Flux balance analysis predicts essential genes in clear cell renal cell carcinoma metabolism. Sci Rep 5: 10738." />
<meta name="citation_reference" content="Gohlke BO, Nickel J, Otto R, Dunkel M, Preissner R. 2016. CancerResource--updated database of cancer-relevant proteins, mutations and interacting drugs. Nucleic Acids Res 44: D932–937." />
<meta name="citation_reference" content="Guo FB, Ou HY, Zhang CT. 2003. ZCURVE: a new system for recognizing protein-coding genes in bacterial and archaeal genomes. Nucleic Acids Res 31: 1780–1789." />
<meta name="citation_reference" content="Guo FB, Zhang CT. 2006. ZCURVE_V: a new self-training system for recognizing protein-coding genes in viral and phage genomes. BMC bioinformatics 7: 9." />
<meta name="citation_reference" content="Guyon I, Weston J, Barnhill S, Vapnik V. 2002. Gene selection for cancer classification using support vector machines. Machine learning 46: 389–422." />
<meta name="citation_reference" content="Hart T, Chandrashekhar M, Aregger M, Steinhart Z, Brown KR, MacLeod G, Mis M, Zimmermann M, Fradet-Turcotte A, Sun S et al. 2015. High-resolution CRISPR screens reveal fitness genes and genotype-specific cancer liabilities. Cell 163: 1515–1526." />
<meta name="citation_reference" content="Hart T, Moffat J. 2016. BAGEL: a computational framework for identifying essential genes from pooled library screens. BMC bioinformatics 17: 164." />
<meta name="citation_reference" content="Hua ZG, Lin Y, Yuan YZ, Yang DC, Wei W, Guo FB. 2015. ZCURVE 3.0: identify prokaryotic genes with higher accuracy as well as automatically and accurately select essential genes. Nucleic Acids Res 43: W85–90." />
<meta name="citation_reference" content="Juhas M, Eberl L, Glass JI. 2011. Essence of life: essential genes of minimal genomes. Trends Cell Biol 21: 562–568." />
<meta name="citation_reference" content="Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kanapin A, Le Bot N, Moreno S, Sohrmann M. 2003. Systematic functional analysis of the Caenorhabditis elegans genome using RNAi. Nature 421: 231–237." />
<meta name="citation_reference" content="Kim DU, Hayles J, Kim D, Wood V, Park HO, Won M, Yoo HS, Duhig T, Nam M, Palmer G et al. 2010. Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe. Nat Biotechnol 28: 617–623." />
<meta name="citation_reference" content="Kuepfer L, Sauer U, Blank LM. 2005. Metabolic functions of duplicate genes in Saccharomyces cerevisiae. Genome Res 15: 1421–1430." />
<meta name="citation_reference" content="Liao BY, Zhang J. 2007. Mouse duplicate genes are as essential as singletons. Trends Genet 23(8): 378–381." />
<meta name="citation_reference" content="Lloyd JP, Seddon AE, Moghe GD, Simenc MC, Shiu SH. 2015. Characteristics of Plant Essential Genes Allow for within- and between-Species Prediction of Lethal Mutant Phenotypes. Plant cell 27: 2133–2147." />
<meta name="citation_reference" content="Luo H, Lin Y, Gao F, Zhang CT, Zhang R. 2014. DEG 10, an update of the database of essential genes that includes both protein-coding genes and noncoding genomic elements. Nucleic Acids Res 42: D574–D580." />
<meta name="citation_reference" content="Mushegian AR, Koonin EV. 1996. A minimal gene set for cellular life derived by comparison of complete bacterial genomes. Proc Natl Acad Sci U S A 93: 10268–10273." />
<meta name="citation_reference" content="Ou HY, Guo FB, Zhang CT. 2004. GS-Finder: a program to find bacterial gene start sites with a self-training method. Int J Biochem Cell Biol 36: 535–544." />
<meta name="citation_reference" content="Pechter KB, Gallagher L, Pyles H, Manoil CS, Harwood CS. 2016. Essential Genome of the Metabolically Versatile Alphaproteobacterium Rhodopseudomonas palustris. J Bacteriol 198: 867–876." />
<meta name="citation_reference" content="Peng W, Wang J, Wang W, Liu Q, Wu FX, Pan Y. 2012. Iteration method for predicting essential proteins based on orthology and protein-protein interaction networks. BMC Syst Biol 6: 87." />
<meta name="citation_reference" content="Porta-Pardo E, Hrabe T, Godzik A. 2014. Cancer3D: understanding cancer mutations through protein structures. Nucleic Acids Res 43: D968–973." />
<meta name="citation_reference" content="Ruan Y. 2011. Genome-sequencing anniversary. Presenting the human genome: now in 3D! Science 331: 1025–1026." />
<meta name="citation_reference" content="Seringhaus M, Paccanaro A, Borneman A, Snyder M, Gerstein M. 2006. Predicting essential genes in fungal genomes. Genome Res 16: 1126–1135." />
<meta name="citation_reference" content="Wang T, Birsoy K, Hughes NW, Krupczak KM, Post Y, Wei JJ, Lander ES, Sabatini DM. 2015. Identification and characterization of essential genes in the human genome. Science 350: 1096–1101." />
<meta name="citation_reference" content="Wei W, Ning LW, Ye YN, Guo FB. 2013. Geptop: a gene essentiality prediction tool for sequenced bacterial genomes based on orthology and phylogeny. PLoS One 8:e72343." />
<meta name="citation_reference" content="Wu X, Liu H, Liu H, Su J, Lv J, Cui Y, Wang F, Zhang Y. 2013. Z curve theory-based analysis of the dynamic nature of nucleosome positioning in Saccharomyces cerevisiae. Gene 530: 8–18." />
<meta name="citation_reference" content="Yan K, Zhang D. 2015. Feature selection and analysis on correlated gas sensor data with recursive feature elimination. Sens. Actuators. B. Chem. 212: 353–363." />
<meta name="citation_reference" content="Yang JY, Zhou Y, Yu ZG, Anh V, Zhou LQ. 2008. Human Pol II promoter recognition based on primary sequences and free energy of dinucleotides. BMC bioinformatics 9: 113." />
<meta name="citation_reference" content="Yuan Y, Xu Y, Xu J, Ball RL, Liang H. 2012. Predicting the lethal phenotype of the knockout mouse by integrating comprehensive genomic data. Bioinformatics 28: 1246–1252." />
<meta name="citation_reference" content="Zhang CT, Wang J. 2000. Recognition of protein coding genes in the yeast genome at better than 95% accuracy based on the Z curve. Nucleic Acids Res 28: 2804–2814." />
<meta name="citation_reference" content="Zhang CT. 1997. A symmetrical theory of DNA sequences and its applications. J Theor Biol 187: 297–306." />
<meta name="citation_reference" content="Zhang CT, Chou KC. 1994. A graphic approach to analyzing codon usage in 1562 Escherichia coli protein coding sequences. J Mol Biol 238: 1–8." />
<meta name="citation_reference" content="Zhang CT, Zhang R. 1991. Analysis of distribution of bases in the coding sequences by a diagrammatic technique. Nucleic Acids Res 19: 6313–6317." />
<meta name="citation_reference" content="Zhang R, Zhang CT. 1994. Z curves, an intutive tool for visualizing and analyzing the DNA sequences. J Biomol Struct Dyn 11: 767–782." />
<meta name="twitter:title" content="Accurate prediction of human essential genes using only nucleotide composition and association information" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Three groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a λ-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier’s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (&lt;https://cefg.uestc.edu.cn/Pheg&gt;), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the λ-interval Z curve method could be effectively extended to classification issues of other DNA elements." />
<meta name="og-title" property="og:title" content="Accurate prediction of human essential genes using only nucleotide composition and association information" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/084129v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Three groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a λ-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier’s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (&lt;https://cefg.uestc.edu.cn/Pheg&gt;), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the λ-interval Z curve method could be effectively extended to classification issues of other DNA elements." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2016-10-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/084129v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/084129v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/084129v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/084129v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/23431" />
    <title>Accurate prediction of human essential genes using only nucleotide composition and association information | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"YRrLoyJgb4LY_YeF7rcB3UD_XH6Ty-217DqX8JVKhmc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"23431","apath":"\/biorxiv\/early\/2016\/10\/28\/084129.atom","pisa":"biorxiv;084129v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;084129v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/084129v1","title":"Accurate prediction of human essential genes using only nucleotide composition and association information","identifier":"node\/23431"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lJUB5XoGwKCvvsK4C\\\\\\\/qNZerAodbbYOzCD5MxLB+QiPREp5Hu3gDfa1IfoVEs1I+t3rJLyd4B\\\\\\\/VB+YmhEHjd1Cn\\\\\\\/aLE5\\\\\\\/Ob4r7pGd1H93Wh+IkELmJUxlaexrTy3d4KzuMbOOk3Cm36\\\\\\\/3PRibPYbcJYURBEbf\\\\\\\/2vRsZkp8dAsQ6fFrHNSOZ95BfxTg\\\\\\\/sirrUc3fnYWWMO7pppZsDCu8lMfPD977UHO4y6Xn5vvZ8Am7TFilDxLPjbWTQ9\\\\\\\/KQMPrLeuXvCkv4zrHC+ZypZRBD\\\\\\\/ARwLJc\\\\\\\/qh\\\\\\\/9TxlN238V5sZePxnL\\\\\\\/xqpgP2nRxLZkuJEdP5WKJWCLwQDZ3h7zbtCpyDIu0itb32E1wcPeNHLtWgC7yYpBKiPOBxna+A1OfEVRCRyyrnTlJmuC1BgVLATOkKiEfhRqvx4\\\\\\\/cTBX5p8rF9BF97eJSOBkzhG7u0GZX0h0oQ4sYbLDqTuSd1jxaz4XtYJv8xvhMBd0U2DlgDdCLo8eBMZFExytEGdPdzcu24v47niCvi6qTrn5kuMUCTe5v2iY9n3oqUR+RpgjWPPVNpEr7Gru6OUI4KtQkI7tOnE8UnQqNmueLiJFvve7XhxhlOw762xt4hLhGRl8Iom3qmScQMrfggNhIHDxrnmkaFJTR1C+QIet8+suEGYUQoVNdkHIcC9wMhZksOBUokFb+U9PrxRinuNYvbDpq7IDIS0fBJVQj7Op2TJ8FfGwbWMXLrFltCnVFPf9Rs5aorqZmLRWPgyYk\\\\\\\/uUA074Rp4+lYAr1nJJtB42JQplhx0XEQ2+F5C9EXLlz3Xgpy\\\\\\\/A8Oa5As0yURMlIODn1GZlvMIzcd+qVgysfYpu3RGaYe6vg3UDB9AJFIIW2o0cUd4DtboCX29GB\\\\\\\/s9p3A61Jm\\\\\\\/t\\\\\\\/O\\\\\\\/f4E34YweMvL91JuGowDWrDkfaBdsHcKZneigCpyRNETAJAcOD8Ys4xqpjWc9TH+tV5yTQP+znHCDhdujuVd0wH6h73i82CxSmmAnMxTEgLj4k3rxUHAroKSjHcaQdtlmwYdGDo5X6whMXEyn0pdSkGA4CnlCQXApuCHqBUMi61hATl8jvaP3c1VMxNXV6yCrhPXcrOnizJ\\\\\\\/UF7VJSE4s8nNbTcGUjFvAxyTybFKJT1kx90KHLXoekkhLMVctE8naSwO\\\\\\\/OOoZmRO004hbSl0yY6HzaTh8KC5F0Hda+4n9AAihkdV\\\\\\\/uQZhjRtKZNgVjx9MAseTLHLKFZ7qdXzwux3b5Z2G5Y8DPmDj68JTpwCi\\\\\\\/v6X8Uzkyx3Al0KsndrVvzq9jlt3w8iB26wNgu4lnMqQ8JZdduQONhq\\\\\\\/QmcBc25OEIref9g=\\\u0022,\\\u0022iv\\\u0022:\\\u0022SERkWyv3+kzEDoaC1xSt5Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e8f0fce0580568f0661b3d2027af4005121f7a668e0a3cca1b3f9c47be699754\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/084129v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-23431 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/084129v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RD85K5MSarsQUGHvYWh2SaT2kOHEkAomVYwGBQToYPA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="23431" id="node23431" data-pisa="biorxiv;084129v1" data-pisa-master="biorxiv;084129" data-apath="/biorxiv/early/2016/10/28/084129.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Accurate prediction of human essential genes using only nucleotide composition and association information</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Feng-Biao</span> <span class="nlm-surname">Biao</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Chuan</span> <span class="nlm-surname">Dong</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hong-Li</span> <span class="nlm-surname">Hua</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Shuo</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Luo</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Hong-Wan</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Yan-Ting</span> <span class="nlm-surname">Jin</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Kai-Yue</span> <span class="nlm-surname">Zhang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/084129 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node23431" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Feng-Biao Biao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Life Science and Technology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for Informational Biology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Feng-Biao%2BBiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Biao%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFeng-Biao%2BBiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fbguo@uestc.edu.cn" class="" data-icon-position="" data-hide-link-title="0">fbguo@uestc.edu.cn</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chuan Dong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Life Science and Technology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for Informational Biology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuan%2BDong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuan%2BDong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hong-Li Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Life Science and Technology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for Informational Biology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hong-Li%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHong-Li%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shuo Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Life Science and Technology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for Informational Biology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuo%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShuo%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hao Luo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Physics, Tianjin University</span>, Tianjin, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BLuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BLuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hong-Wan Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Life Science and Technology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for Informational Biology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hong-Wan%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHong-Wan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yan-Ting Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Life Science and Technology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for Informational Biology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yan-Ting%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYan-Ting%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kai-Yue Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>School of Life Science and Technology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for Informational Biology, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Key Laboratory for Neuro-information of the Ministry of Education, University of Electronic Science and Technology of China</span>, Chengdu, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai-Yue%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKai-Yue%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/084129v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:23431" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:23431/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/084129v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:23431" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:23431/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/084129v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:23431" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:23431/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/084129v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:23431" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:23431/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/084129v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:23431" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:23431/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Three groups recently identified essential genes in human cancer cell lines using wet experiments, and these genes are of high values. Herein, we improved the widely used Z curve method by creating a λ-interval Z curve, which considered interval association information. With this method and recursive feature elimination technology, a computational model was developed to predict human gene essentiality. The 5-fold cross-validation test based on our benchmark dataset obtained an area under the receiver operating characteristic curve (AUC) of 0.8814. For the rigorous jackknife test, the AUC score was 0.8854. These results demonstrated that the essentiality of human genes could be reliably reflected by only sequence information. However, previous classifiers in three eukaryotes can gave satisfactory prediction only combining sequence with other features. It is also demonstrated that although the information contributed by interval association is less than adjacent nucleotides, this information can still play an independent role. Integrating the interval information into adjacent ones can significantly improve our classifier’s prediction capacity. We re-predicted the benchmark negative dataset by Pheg server (<a href="https://cefg.uestc.edu.cn/Pheg">https://cefg.uestc.edu.cn/Pheg</a>), and 118 genes were additionally predicted as essential. Among them, 21 were found to be homologues in mouse essential genes, indicating that at least a part of the 118 genes were indeed essential, however previous experiments overlooked them. As the first available server, Pheg could predict essentiality for anonymous gene sequences of human. It is also hoped the λ-interval Z curve method could be effectively extended to classification issues of other DNA elements.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Catalogs of essential genes on a whole-genome scale, determined using wet-lab methods, are available for several prokaryotic (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mushegian and Koonin 1996</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Luo et al. 2014</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pechter et al. 2016</a>) and eukaryotic organisms (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kamath et al. 2003</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amsterdam et al. 2004</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liao and Zhang 2007</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kim et al. 2010</a>). Computational methods with high accuracy offer an appealing alternative method for identifying essential genes. Computational methods are broadly divided into three types: machine learning-based methods combining intrinsic and context-dependent features (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Deng et al. 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cheng et al. 2013</a>), flux balance analysis-based methods (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kuepfer et al. 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">del Rio et al. 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gatto et al. 2015</a>), and homology search and evolutionary analysis-based methods (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Peng et al. 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wei et al. 2013</a>). With respect to essential gene prediction in bacteria, we integrated the orthology and phylogenetic information and subsequently developed a universal tool named Geptop (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Wei et al. 2013</a>), which has shown the highest accuracy among all state-of-the-art algorithms.</p><p id="p-4">Some studies have focused on essential gene prediction in eukaryotic genomes. In 2005, Xu et al. investigated protein dispensability in <em>Saccharomyces cerevisiae</em> by combining high-throughput data and machine learning-based methods (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen and Xu 2005</a>). In 2006, Seringhaus et al. reported a machine learning-based method that integrated various intrinsic and predicted features to identify essential genes in yeast <em>S. cerevisiae</em> genomes (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Seringhaus et al. 2006</a>). They for the first time using AUC to measure the machine learning classifier’s capability in identifying essential genes, and they also translated the essentiality from yeast <em>S. cerevisiae</em> to yeast <em>Saccharomyces mikatae.</em> Yuan et al. integrated informative genomic features to perform knockout lethality predictions in mice using three machine learning-based methods (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Yuan et al. 2012</a>). Lloyd et al. analyzed the characteristics of essential genes in the <em>Arabidopsis thaliana</em> genome and used <em>A. thaliana</em> as a machine learning-based model to transform the essentiality annotations to <em>Oryza sativa</em> and <em>S. cerevisiae</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lloyd et al. 2015</a>).</p><p id="p-5">Recently, three research teams approximately identified 2,000 essential genes in human cancer cell lines using CRISPR-Cas9 and gene-trap technology (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blomen et al. 2015</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hart et al. 2015</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Wang et al. 2015</a>). Their results showed high consistency, which further confirmed the accuracy and robustness of the essential gene sets (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fraser 2015</a>). These studies provided an in-depth analysis of tumor-specific essential genes and feasible methods to screen tumor-specific essential genes (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fraser 2015</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hart and Moffat 2016</a>). The essential genes screened by these three teams provided a clear definition of the requirements for sustaining the basic cell activities of individual human tumor cell types. Practically, these genes can be regarded as targets for cancer treatment (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Fraser 2015</a>). The data from these three groups provided a rare opportunity to theoretically study the function, sequence composition, evolution and network topology of human essential genes. One of the most important and interesting theoretical issues in modern biology is whether essential genes and non-essential genes can be accurately classified using computational methods. The models established in the aforementioned three eukaryotic organisms, <em>S. cerevisiae</em> (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chen and Xu 2005</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Seringhaus et al. 2006</a>), <em>Mus musculus</em> (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Yuan et al. 2012</a>), and <em>A. thaliana</em> (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lloyd et al. 2015</a>), involved intrinsic features, or intrinsic and context-dependent features. These context-dependent features included those features extracted from experimental omics data. However, the features derived from experimental data are frequently unavailable; consequently, this type of machine learning model cannot be extended to a wide range of genomes. In the present study, we addressed this problem in humans by using only intrinsic features derived from sequences, from which certain features can be characterized using a λ-interval Z-curve. To facilitate the use of these data by interested researchers, we have provided a user-friendly online web server, Pheg, which can be freely accessed without registration at <a href="http://cefg.uestc.edu.cn/Pheg">http://cefg.uestc.edu.cn/Pheg</a>.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Cross-validation Results</h3><p id="p-6">The final features of this method were described by <em>FV</em><sub><em>w,λ</em></sub>, a value that contains information on the composition of the adjacent w-nucleotides (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gao and Zhang 2004</a>) and λ-interval nucleotides. The association information was also captured by <em>FV</em><sub><em>w,λ</em></sub>. Therefore, this method achieves improved performance compared with using the original Z-curve. The following results solidly confirmed this point.</p><p id="p-7">We performed a 5-fold cross-validation test with w ranging from 2 to 4 and λ ranging from 0 to 5. The detailed results are provided in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, showing that area under the curve (AUC) values gradually increased with increasing λ when w was fixed. An examination of the performance under variable values for w and fixed values for λ revealed that the performance for the classifier improved with increasing values for w.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/260498/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/260498/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/260498" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-8" class="first-child">AUC values at different w, λ and penalty parameters c.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">As shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, we obtained an AUC value of 0.8002 under <em>FV</em><sub><em>2,0</em></sub>. However, after utilizing the λ-interval nucleotide composition, the performance was improved, for example, the best AUC achieved for this model was 0.8449 through the 5-fold cross-validation test under variable <em>FV</em><sub><em>4,5</em></sub>. The AUC was improved 4.47% compared with <em>FV</em><sub><em>2,0</em></sub>. The information redundancy and noise in the original features can influence the predictive power of a classifier, and high-dimensional features also increase the time costs for training and prediction. Feature selection technology can mitigate these disadvantages. The support vector machine (SVM)-recursive feature extraction (RFE)+correlation bias reduction (CBR) method was adopted to rank these features in descending order based on the contribution of each feature. Subsequently,the top 100 features were used to constitute the initial feature subset to train and test the model, and the next 100 features were added into the feature subset, followed by prediction using the same methods. This process was repeated until the top 4,500 features had been added according to the rank order. The test results of each model were evaluated according to the AUC scores via a 5-fold cross-validation test. The AUC values for different top features are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1 (A)</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AUC values obtained under different top n features and the contribution of each group. Figure 1 (A) represents the AUC scores under different top features. Dots with different colors denote different c values. Figure 1 (B) illustrates the selective tendentiousness for every variable type. The red bars denote that the selective tendentiousness has significance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-800651042" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;AUC values obtained under different top n features and the contribution of each group. Figure 1 (A) represents the AUC scores under different top features. Dots with different colors denote different c values. Figure 1 (B) illustrates the selective tendentiousness for every variable type. The red bars denote that the selective tendentiousness has significance.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F1.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F1.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-10" class="first-child">AUC values obtained under different top n features and the contribution of each group. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> (A) represents the AUC scores under different top features. Dots with different colors denote different c values. <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> (B) illustrates the selective tendentiousness for every variable type. The red bars denote that the selective tendentiousness has significance.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Among all 4,545 features examined, the best AUC of 0.8814 was achieved for the top 800 selective features. The final AUC value was 8.12% higher than that for <em>FV</em><sub><em>2,0</em></sub>. To conduct an objective evaluation of this method, we performed a rigorous jackknife test based on the top 800 selected features using the parameters determined via a 5-fold cross-validation test. We obtained an AUC value of 0.8854. As expected, excellent performance was obtained after adopting the λ-interval nucleotide composition and feature selection technology. Those results illustrated that the essentiality of human genes could be well reflected by only sequence information.</p><p id="p-12">As we are extremely interested in the actual essential genes in the predicted results, we used the positive predictive value (PPV) to further refine. This evaluation index can be calculated using the formula TP / (TP+FP), where TP (true positive) and FP (false positive) represent the number of real essential and non-essential genes among the positive predictions. Therefore, the PPV reflects the proportion of actual essential genes among the predicted essential genes. We obtained a PPV of 73.05% (TP=515, FP=190) using the jackknife test based on the top 800 features. One of the simplest cross validation tests is the holdout method. In this procedure, the dataset is separated into two subsets, namely, training and testing datasets. We randomly sampled one-fifth of the positive and negative samples from the benchmark dataset for the training model, and the remaining samples were used as the testing dataset. To comprehensively assess the method used in the present study, we repeated the holdout method 100 times to identify differences in the composition of the training and testing samples. The mean AUC score was used as the final evaluator. A mean AUC score of 0.8537 with a variance of 1.67e-005 was obtained. Additionally, the proportions of samples in the training and predicting datasets were changed for further investigation. One-tenth of the positive and negative samples were randomly sampled as the training dataset, and the remaining samples were used as the testing dataset. This procedure was repeated 100 times. We obtained a mean AUC score of 0.8347 with a variance of 2.77e-005. These results further confirmed that this method was robust and accurate.</p></div><div id="sec-4" class="subsection"><h3>Different contributions of each group of variables to the classification ability</h3><p id="p-13">Features with fixed w and λ values correspond to a specific group of variables. A total of 19 special groups were obtained, namely, <em>fv</em><sub><em>1,0</em></sub>, <em>fv</em><sub><em>1,1</em></sub>, <em>fv</em><sub><em>1,2</em></sub> … <em>fv</em><sub><em>1,5</em></sub>; <em>fv</em><sub><em>2,0</em></sub>, <em>fv</em><sub><em>2,1</em></sub> … <em>fv</em><sub><em>2,5</em></sub>; <em>fv</em><sub><em>3,0</em></sub>, <em>fv</em><sub><em>3,1</em></sub> … <em>fv</em><sub><em>3,5</em></sub>, and <em>fv</em><sub><em>4,0</em></sub>. We calculated the percentage of features in these groups, and the results are provided in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><p id="p-14">For each group, there were two frequencies: P(A), which denotes the actual frequency of features in each group appearing in the top 800 selected features, and P(E), which denotes the expected frequency of the features in each group appearing in the original 4,545 features. Therefore, P(A) was obtained based on the number of selected features in each group divided by 800, and P(E) was calculated by dividing the number of total features in each group by 4,545. P(A) and P(E) are listed in columns 4 and 5, respectively. If P(A) is higher than P(E), then the group makes a higher-than-average contribution to the identification of essential genes. We calculated the tendentiousness using the formula P(A)/P(E)−1 to denote how the features of each group were selected, and the results are listed in column 6 of <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. We further conducted a hypergeometric distribution test for each group, and the p values are listed in column 7. <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1 (B)</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> show that <em>fv</em><sub><em>2,0</em></sub> (p = 1.60E−06), <em>fv</em><sub><em>2,1</em></sub> (p = 0.02407688), and <em>fv</em><sub><em>3,0</em></sub> (p = 7.89E−05) are preferentially selected and are statistically significant. These results demonstrated that there are strong signals for classifying essential and non-essential genes when the character interval is equal to zero or one, but the other groups did not show these strong signals. To further confirm this result, the variables <em>fv</em><sub><em>2,0</em></sub>, <em>fv</em><sub><em>2,1</em></sub>, <em>fv</em><sub><em>2,2</em></sub>, <em>fv</em><sub><em>2,3</em></sub>, <em>fv</em><sub><em>2,4</em></sub>, <em>fv</em><sub><em>2,5</em></sub> and <em>fv</em><sub><em>3,0</em></sub>, <em>fv</em><sub><em>3,1</em></sub>, <em>fv</em><sub><em>3,2</em></sub>, <em>fv</em><sub><em>3,3</em></sub>, <em>fv</em><sub><em>3,4</em></sub>, and <em>fv</em><sub><em>3,5</em></sub> were used as input features. Improved performance was obtained under <em>fv</em><sub><em>2,0</em></sub>, <em>fv</em><sub><em>2,1</em></sub>, and <em>fv</em><sub><em>3,0</em></sub> compared with the other groups (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 1</a>, column 8). Those results demonstrated that the shorter interval association provides more information. However, longer interval association can still play an independent role. Hence, integrating the interval information into adjacent ones could significantly improve our classifier’s capacity of discernment (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/260764/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/260764/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/260764" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-15" class="first-child">Feature details for every variable type in the top 800 selective features. The feature groups with significance are indicated bold font.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>A web server for predicting essential genes in human cancer cell lines</h3><p id="p-16">To facilitate the use of these data by interested researchers, we constructed a user-friendly online web server named Pheg (<span class="underline"><strong>P</strong></span>redictor of <span class="underline"><strong>h</strong></span>uman <span class="underline"><strong>e</strong></span>ssential <span class="underline"><strong>g</strong></span>enes), which is freely accessible at <a href="http://cefg.uestc.edu.cn/Pheg">http://cefg.uestc.edu.cn/Pheg</a>. The Pheg algorithm is based on the λ-interval Z-curve. Additional parameters are not necessary, making this algorithm convenient to use. Pheg can predict whether a query gene is essential using only the CDS region of a gene as input. We integrated logistic regression into the Pheg server to estimate the reliability of the predicted results. Hence, this server can output a probabilistic estimated value as a measurement of gene essentiality for the inputted coding region. Note that this is the first available server for predicting human gene essentiality. Comparatively, some computational models have been proposed for the other eukaryotes however all of them did not provide online prediction service. We re-predicted the genes in the benchmark dataset via Pheg and obtained an AUC = 0.9249 and PPV = 83.84%. A total of 612 genes were identified as essential genes among the 1,516 positive samples, and 118 genes were predicted as essential genes among the 10,499 negative samples. To estimate how many genes among those predictions are real essential genes, we calculated precisions using 5-fold, 10-fold, 15-fold, 20-fold cross-validation, and we obtained precisions with values of 70.43%, 71.63%, 72.48%, 72.22%, which were approximately 70%. Hence, we expect that 82 (118×70%) are correctly predicted essential genes. The information for these 118 genes is provided in Supplemental Information S1.</p><p id="p-17">CancerResource is a comprehensive knowledgebase for drug-target relationships associated with cancer and for supporting information or experimental data (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gohlke et al. 2016</a>). Through the CancerResource database (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gohlke et al. 2016</a>), these 118 genes were searched using the classifier ‘Target search by gene identifiers’ (<a href="https://data-analysis.charite.de/care/index.php?site=target_input">https://data-analysis.charite.de/care/index.php?site=target_input</a>). Among these 118 genes, 17 genes showed interactions with drugs and other chemical molecules, and 13 genes had Cancer3D structures (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Porta-Pardo et al. 2014</a>). The details of these genes are supplemented in Supplemental Information S2. Additionally, we used these 118 gene sequences to conduct a BLAST (Basic Local Alignment Search Tool) search against essential genes in the genome of <em>Mus musculus</em> (mouse). The current mouse essential gene set is accessible in the OGEE database (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen et al. 2012</a>). Considering that no BLAST program is embedded in OGEE, we downloaded the essential gene annotations (gene_essentiality) at (<a href="http://ogee.medgenius.info/downloads/">http://ogee.medgenius.info/downloads/</a>) and extracted the essential gene annotation of <em>M. musculus</em>. We obtained the essential gene sequences according to the annotations (<a href="http://ogee.medgenius.info/downloads/">http://ogee.medgenius.info/downloads/</a>). A BLAST search was performed via ncbi-blast-2.2.30+-win64.exe using the data from OGEE, and homologs for 21 genes were identified (e value &lt; 1e-100) among the 118 predicted essential genes. The sequences of the essential genes in <em>M. musculus</em> are provided in Supplemental Information S3. The details for these 21 genes and their corresponding homologs are provided in Supplemental Information S4. Both CancerResource analyses and BLAST search illustrated that at least a part of these 118 genes have higher probability to be factually essential genes and have been overlooked in the essential gene screening in previous experimental studies. Hence, Pheg sever could be used to predict essentiality for anonymous gene sequences of human and closely related species, and it is also hoped to supplement the essential gene list of human by identifying novel essential gene from the submitted sequences.</p></div></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-18">The Z-curve has been widely used in the field of bioinformatics for tasks such as protein coding gene identification (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zhang and Wang 2000</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chen et al. 2003</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Guo et al. 2003</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Guo and Zhang 2006</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hua et al. 2015</a>), promoter recognition (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Yang et al. 2008</a>), translation start recognition (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ou et al. 2004</a>), recombination spots recognition (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dong et al. 2016</a>), and nucleosome position mapping (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wu et al. 2013</a>). However, correlation and λ-interval nucleotide composition have not been incorporated into the Z-curve method. In the present study, we present a λ-interval Z-curve based on Z-curve theory. The DNA sequence can be understood as an ordinary character sequence; therefore, the method proposed in the present study has the potential for applications in mining characteristics from other character sequences and can be used as a universal feature extraction method for DNA sequences.</p><p id="p-19">Based on the λ-interval Z-curve, we obtained excellent performance in human essential gene identification. This excellent performance might be attributable to the following points: First, we introduced the concept of intervals, reflecting associated information and the λ-interval nucleotide composition. Second, we used feature selection technology in the present study. Thus, noisy and redundant features could be removed from the original features. <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a> shows the improved performance obtained under <em>fv</em><sub><em>2,0</em></sub>, <em>fv</em><sub><em>2,1</em></sub>, and <em>fv</em><sub><em>3,0</em></sub> compared with the other variable groups. Further comparison of these results with other feature groups shown in <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 1</a>, and this comparison shows that the AUC values obtained with -interval variables are smaller than those obtained with shorter interval variables. However, the performance can be improved after adding λ-interval oligonucleotide association information (see <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Hence, the λ-interval Z-curve should reflect additional important information for essential genes that cannot be contained in adjacent nucleotide association information.</p><p id="p-20">In 2005, Chen and Xu used a neural network and SVM to predict the dispensability of proteins in the yeast <em>S. cerevisiae</em> based on the protein evolution rate, protein-interaction connectivity, gene-expression cooperativity and gene-duplication data (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Chen and Xu 2005</a>). The next year, Seringhaus et al. only used 14 features to predict essential genes in <em>S. cerevisiae</em> and obtained a PPV=0.69 (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Seringhaus et al. 2006</a>). Yuan et al. assembled a comprehensive list of 491 candidate genomic features to predict a lethal phenotype in a knockout mouse using three machine learning methods (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Yuan et al. 2012</a>). Among the 491 candidate genomic features, 373 features were derived from genomic sequences, 94 features were derived from mRNA expression, and 24 features were derived from molecular interaction. Moreover, the best AUC value was 0.782. In 2015, Lloyd et al. investigated the relationship between phenotype lethality and gene function, copy number, duplication, expression levels and patterns, rate of evolution, cross-species conservation, and network connectivity, and the random forest-based model used in this study achieved an AUC of 0.81, which is significantly better than that obtained by random guessing. In addition, these authors integrated the features they identified to predict essential genes in <em>A. thaliana</em> (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lloyd et al. 2015</a>). Those previous researches in three eukaryotes illustrated classifiers can gave satisfactory prediction through combining sequence with other features. For human essential gene identification, we only used the sequence composition and interval association information in the present study and still obtained an AUC of 0.8854. Considering that this result is better than the results obtained in previous studies using integrated features, the gene essentiality of the human genome can be accurately reflected based on only the sequence information.</p></div><div class="section" id="sec-7"><h2 class="">Materials and methods</h2><div id="sec-8" class="subsection"><h3>Benchmark dataset</h3><p id="p-21">The human essential genes in cancer cell lines were identified by three different groups (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Blomen et al. 2015</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hart et al. 2015</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Wang et al. 2015</a>). The results showed high consistency (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Fraser 2015</a>), which further confirmed the accuracy and robustness of these data. We extracted the gene essentiality data from the DEG database (<a href="http://tubic.tju.edu.cn/deg/">http://tubic.tju.edu.cn/deg/</a>), the updated version of which contained human gene essentiality information. These essentiality annotations serve as the basis for constructing our benchmark dataset. The flowchart shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> illustrates the construction of the positive and negative dataset.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Description of the construction of the human essential and non-essential gene datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-800651042" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Description of the construction of the human essential and non-essential gene datasets.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F2.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F2.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-22" class="first-child">Description of the construction of the human essential and non-essential gene datasets.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">In the three studies, 13 datasets were provided. As lines HCT116 and KBM7 are represented by two datasets each, 11 cancer cell lines (KBM7, K562, Raji, Jiyoye, A375, HAP1, DLD1, GBM, HCT116, Hela, rpel) are involved in total. Blomen <em>et al.</em> and Wang <em>et al.</em>identified the essential genes in the KBM7 cell line. We combined these two datasets into one gene set, KBM7. A total of 2,073 and 386 essential genes were contained in the two datasets for HCT116. The 386 genes in this dataset were markedly different from those in the datasets for the other cell lines, so this dataset was excluded. Ultimately, 11 essential gene sets were obtained, corresponding to a single cell line. Essential genes, by definition, are indispensable for the survival of organisms under optimized growth conditions and are considered the foundation of life (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Juhas et al. 2011</a>). Therefore, we only retained genes that were identified as lethal genes in more than half of the investigated cell lines. When a gene appeared as essential in more than six cell lines (11/2≈6), it was selected as one sample in the positive dataset. According to this principle, we obtained a total of 1,518 essential gene annotations. We downloaded all of the protein coding gene sequences from the CCDS database (<a href="https://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi">https://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi</a>), and the annotations of protein coding genes were obtained from the HGNC database (<a href="https://www.genenames.org/cgi-bin/statistics">https://www.genenames.org/cgi-bin/statistics</a>, March 1, 2016), which contained 19,003 annotation entries. The essential gene sequences were extracted according to the annotations, and genes with no counterpart in the CCDS database were excluded. According to this criterion, we excluded 2 genes and obtained 1,516 essential genes. We used the essential gene annotation in the DEG dataset, and the gene sequences were extracted from the CCDS because the former did not contain the information for non-essential genes. For human essentiality annotations in the DEG database, a number of scattered annotated essential genes aside from those in the 11 cell lines were identified. A total of 28,166 essential gene annotated entries (including conditional essential gene annotated entries) were obtained. Among these annotations, there were many repeated annotation entries; therefore, there were considerably fewer unique entries. To obtain a more reliable negative dataset, i.e., absolutely non-essential genes, we excluded all of the human essential genes annotated in the DEG database (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Luo et al. 2014</a>) from the list of the protein coding genes. The remaining genes were regarded as the negative dataset, and their gene sequences were extracted from the CCDS database. Genes with no counterpart in the CCDS database were also excluded. A total of 10,499 non-essential genes were obtained using this method. Ultimately, a total of 12,015 gene entries were obtained in the benchmark dataset: 1,516 essential genes and 10,499 non-essential genes. The protein coding gene annotations are provided in Supplemental Information S5, and information for the benchmark dataset is provided in Supplemental Information S6.</p></div><div id="sec-9" class="subsection"><h3>λ-interval Z-curve</h3><p id="p-24">The originally proposed Z-curve variables might reflect the composition of a single nucleotide considering the features derived from phase heterogeneity of a single nucleotide (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhang and Zhang 1991</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhang and Chou 1994</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhang and Zhang 1994</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zhang 1997</a>). Herein, we provided a summary of the Z-curve method used for gene identification (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Zhang and Wang 2000</a>). Let us suppose that the frequencies of bases A, C, G and T occurring in an ORF or a gene fragment at positions 1, 4, 7, …, 2, 5, 8, …, and 3, 6, 9, …, are represented by <em>a</em><sub><em>1</em></sub>, <em>c</em><sub><em>1</em></sub>, <em>g</em><sub><em>1</em></sub>, <em>t</em><sub><em>1</em></sub>; <em>a</em><sub><em>2</em></sub>, <em>c</em><sub><em>2</em></sub>, <em>g</em><sub><em>2</em></sub>, <em>t</em><sub><em>2</em></sub>; <em>a</em><sub><em>3</em></sub>, and <em>c</em><sub><em>3</em></sub>, <em>g</em><sub><em>3</em></sub>, <em>t</em><sub><em>3</em></sub>, respectively. Those 12 symbols represent the frequencies of the bases at the 1<sup>st</sup>, 2<sup>nd</sup> and 3<sup>rd</sup> codon positions, respectively. According to the symbols defined above, the universal Z-curve mathematical expression is as follows (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Zhang and Wang 2000</a>):
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/10/28/084129/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/10/28/084129/embed/graphic-5.gif"/></noscript></span></span></p><p id="p-25">Because composition bias for oligonucleotides in coding DNA sequence (CDS) regions or open reading frames (ORFs) exists, the adjacent w-nucleotides Z-curve method was proposed (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Guo et al. 2003</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gao and Zhang 2004</a>). Let us suppose that w represents the length of the adjacent nucleotide sequence. The Z-curve variables for the phase-specific adjacent w-nucleotides can be calculated as follows:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/10/28/084129/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/10/28/084129/embed/graphic-6.gif"/></noscript></span></span>
where k equals 1, 2, or 3 to indicate that the first oligonucleotide bases are situated at the 1<sup>st</sup>, 2<sup>nd</sup> and 3<sup>rd</sup> codon positions, respectively.</p><p id="p-26">Recent studies demonstrated the existence of long-range associations in chromosomes and showed that these associations are crucial for gene regulation (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fullwood et al. 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ruan 2011</a>). Although the two adjacent nucleotides in the primary structure have no association in some cases, strong associations in terms of tertiary structure might exist. Therefore, we introduced the λ-interval Z-curve to virtually represent the interval range association. The details of this method are described as <em>p<sub>k</sub>(S<sub>w</sub>X)</em>, which represents the frequency of oligonucleotides <em>S</em><sub><em>w</em></sub><em>X</em> in genes or ORFs, where X is one of the four basic bases A, T, G and C. To facilitate this presentation, the length of the oligonucleotide <em>S</em><sub><em>w</em></sub> is represented as w. According to the predetermined characters, we generated the universal equation for the λ-interval Z-curve based on Z-curve theory as follows:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/10/28/084129/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/10/28/084129/embed/graphic-7.gif"/></noscript></span></span>
where x, y, and z represent the accumulation of the three base groups classified according to chemical bond properties. Variable k denotes the phase-specific index of the first base in the nucleotide sequence <em>S</em><sub><em>w</em></sub>, and λ represents the intervals between <em>S</em><sub><em>w</em></sub> and <em>X</em>. The first base in the oligonucleotide <em>S</em><sub><em>w</em></sub> was located at position k. The core part of λ-interval Z-curve forms oligonucleotide <em>S<sub>w</sub>X.</em> A schematic diagram of the formation of these oligonucleotides is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Description of the process of constructing the oligonucleotides using the &#x3BB;-interval Z-curve theory algorithm. A gene or an ORF has three phases, denoted with different colors, i.e., red denotes the base located in the first phase, yellow denotes the base located in the second phase, and blue denotes the base located in the third phase." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-800651042" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Description of the process of constructing the oligonucleotides using the &#x3BB;-interval Z-curve theory algorithm. A gene or an ORF has three phases, denoted with different colors, i.e., red denotes the base located in the first phase, yellow denotes the base located in the second phase, and blue denotes the base located in the third phase.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F3.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F3.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/10/28/084129/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-27" class="first-child">Description of the process of constructing the oligonucleotides using the λ-interval Z-curve theory algorithm. A gene or an ORF has three phases, denoted with different colors, i.e., red denotes the base located in the first phase, yellow denotes the base located in the second phase, and blue denotes the base located in the third phase.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The oligonucleotide window <em>S</em><sub><em>w</em></sub> slides along a DNA molecule sequence according to phase, forming oligonucleotide sets with base X, which is λ intervals away from the last base of <em>S</em><sub><em>w</em></sub>. The periodicity derived from three codons is denoted as <em>S</em><sub><em>w</em></sub><em>X</em>.</p><p id="p-29">When w is equal to 1 and λ is equal to 0, <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a> can be transformed into <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>. When w is more than 1 and λ is equal to 0, <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a> can be transformed into <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>. Thus, the phase-specific single nucleotide Z-curve and phase-specific adjacent w-nucleotide Z-curve are incorporated into the λ-interval Z-curve. Using the λ-interval Z-curve, we can extract more features to characterize DNA sequences. According to Equations (1) and (2), when w is equal to 0, we can obtain 3×3×4<sup>w-1</sup> variables to characterize DNA sequences. When w and λ are greater than 0, we can obtain 3×3×4<sup>w</sup> variables. For convenience, we used <em>fv</em><sub><em>w,λ</em></sub> to represent the variables with the length of w for oligonucleotides <em>S</em><sub><em>w</em></sub>, and the highest interval length between oligonucleotides <em>S</em><sub><em>w</em></sub> and base <em>X</em> is λ. To obtain more information from a DNA sequence, the final variable is described as <em>FV</em><sub><em>w,λ</em></sub>, where w represents the longest oligonucleotides and λ is the highest interval. This variable can be represented as follows:
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/10/28/084129/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2016/10/28/084129/embed/graphic-9.gif"/></noscript></span></span>
where the symbol ‘U’ represents the union set of <em>fv</em><sub><em>w</em>,λ</sub>, i.e., <em>FV</em><sub><em>2,0</em></sub>=[<em>fv</em><sub><em>1,0</em></sub>, <em>fv</em><sub>2,0</sub>]<sup>T</sup>, <em>FV</em><sub><em>2,1</em></sub>=[<em>fv</em><sub><em>1,0</em></sub> ∪ <em>fv</em><sub><em>1,1</em></sub>, <em>fv</em><sub><em>2,0</em></sub>]<sup>T</sup>, <em>FV</em><sub><em>2,2</em></sub>=[<em>fv</em><sub><em>1,0</em></sub> <strong>∪</strong> <em>fv</em><sub><em>1,1</em></sub> <strong>∪</strong> <em>fv</em><sub><em>1,2</em></sub>, <em>fv</em><sub><em>2,0</em></sub>]<sup>T</sup>, …, <em>FV</em><sub><em>3,0</em></sub>=[<em>fv</em><sub><em>1,0</em></sub> <strong>∪</strong> <em>fv</em><sub><em>2,0</em></sub>, <em>fv</em><sub><em>3,0</em></sub>], ….<em>FV</em><sub><em>2,0</em></sub> and <em>FV</em><sub><em>3,0</em></sub> are the combination of adjacent phase-specific w-nucleotide Z-curve variables. We performed this prediction with w ranging from 2 to 4 and λ ranging from 0 to 5. According to the discussion above, we obtained 4,545 variables for <em>FV</em><sub><em>4,5</em></sub>.</p></div><div id="sec-10" class="subsection"><h3>Support vector machine</h3><p id="p-30">SVMs can be classified into linear SVMs and non-linear SVMs according to complexity. SVMs can be used to transform the data into another feature space with more dimensions than the original data and determine the hyper-plane. In this super-dimensional space, the hyper-plane can readily separate the samples. Linear SVMs play a key role in solving ultra-large-scale data, reflecting the effectiveness, rapid speed and splendid generalization of this method in training and prediction. LIBLINEAR, designed by Fan et al. (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fan et al. 2008</a>), is an easy-to-use, freely available software tool to manage large sparse data. The new version of LIBLINEAR (version 2.1-4) supports not only classification, such as L2-loss and L1-loss linear support vector machine, but also regression, such as L2-regularized logistic regression. Given the ultra-high-dimensional feature vectors and large samples contained in the benchmark dataset in the present study, we used the LIBLINEAR software package for prediction. The penalty parameter c was determined using 5-fold cross-validation from 2<sup>−18</sup> to 2<sup>10</sup>. The new version of LIBLINEAR can be downloaded from <a href="https://www.csie.ntu.edu.tw/~cjlin/liblinear/">https://www.csie.ntu.edu.tw/~cjlin/liblinear/</a>.</p></div><div id="sec-11" class="subsection"><h3>Feature extraction technology</h3><p id="p-31">First, the predictive power of a classifier can be influenced by the relevance and noise in the original features. Second, additional time for training and predicting tasks can be increased, reflecting the high-dimensional features. Feature selection (FS) technology is a powerful method for the removal of noise and redundant features from the original features. Hence, the dimension of the features can be reduced. Recursive feature extraction through SVM linear kernels is a powerful FS algorithm (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guyon et al. 2002</a>), but the correlation bias was not considered using this method. Yan and Zhang (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yan and Zhang 2015</a>) proposed an improved method, called SVM−RFE+CBR, which incorporates the CBR. The main concept is that the ranking criterion can be directly derived from the SVM-based model. The feature with the smallest weight is excluded for each run time. The training process was repeated by incorporating CBR until the ranks of all features are obtained. We used SVM-RFE+CBR FS technology to perform feature selection and improve the performance of the classifier.</p></div><div id="sec-12" class="subsection"><h3>Cross-validation and jackknife tests</h3><p id="p-32">Cross-validation test technology is the most popular method for evaluating the performance of classifiers. N-fold cross-validation means that the samples are randomly separated into N sub-samples sets. One of the sub-sample sets is used as the testing dataset, and the remaining sub-sample set is used as the training dataset during each run. The process is executed N times until every sub-dataset is utilized as the testing dataset. If N equals the number of samples in benchmark dataset, then the N-fold cross-validation can also be called jackknife or leave-one-out cross-validation. In the present study, we used the 5-fold cross-validation test to determine the best penalty parameter and adopted the jackknife test to further assess the predictive power of the classifier. The area under the ROC curve, the AUC, is often used to measure the performance quality of a binary classifier. An AUC of 0.5 is equivalent to random prediction, whereas an AUC of 1 represents a perfect prediction. There is no bias for evaluating the performance of the unbalanced dataset through AUC. Therefore, we adopted the AUC as a cross-validation criterion in the present study.</p></div></div><div class="section" id="sec-13"><h2 class="">Data access</h2><p id="p-33">Supplemental data is available at <em>Genome Res.</em> Online:</p><p id="p-34">Supplemental Information S1: The 118 predicted essential genes and their sequences;</p><p id="p-35">Supplemental Information S2: Details for the 17 genes with drug and molecule interactions among the 118 predicted essential genes, distinguished from non-essential genes.</p><p id="p-36">Supplemental Information S3: Sequences of the essential genes in the genome of <em>M. musculus</em>;</p><p id="p-37">Supplemental Information S4: The information for the 21 genes and their corresponding homologs.</p><p id="p-38">Supplemental Information S5: Annotation of human protein-coding genes;</p><p id="p-39">Supplemental Information S6: Benchmark dataset used in the present study. Supplemental files can also be downloaded from the CEFG group website: <a href="http://cefg.cn/Pheg/supplement_info.zip">http://cefg.cn/Pheg/supplement_info.zip</a>.</p></div><div class="section" id="sec-14"><h2 class="">Author contributions</h2><p id="p-41">F. B. Guo conceived, designed and coordinated the present study. C. Dong implemented the main computational analysis. C. Dong, H. L. Hua, S. Liu, H. Luo, H. W. Zhang, Y. T. Jin, and K. Y. Zhang constructed the datasets. C. Dong and H. L. Hua generated the figures. F. B. Guo, C. Dong, and H. L. Hua drafted the manuscript. All authors have read and approved the final manuscript.</p></div><div class="section" id="sec-15"><h2 class="">Conflict of interest statement</h2><p id="p-42">The authors declare that they have no conflicts of interest.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">The authors thank Dr. Ke Yan for providing the open source of the SVM−RFE+CBR script. The authors also thank Prof. Chun-Ting Zhang for inspiring discussions and providing invaluable assistance. In addition, the authors also wish to express thanks to the funding for the open access charge: The National Natural Science Foundation of China [31470068]; Sichuan Youth Science and Technology Foundation of China [2014JQ0051]; and Fundamental Research Funds for the Central Universities of China [ZYGX2015Z006 and ZYGX2015J144].</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.1" data-doi="10.1073/pnas.0403929101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Amsterdam</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nissen</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swindell</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farrington</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">N.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Identification of 315 genes essential for early zebrafish development</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">12792</span>–<span class="cit-lpage">12797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403929101%26rft_id%253Dinfo%253Apmid%252F15256591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM1LzEyNzkyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMTAvMjgvMDg0MTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.2" data-doi="10.1126/science.aac7557"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blomen</span>  <span class="cit-name-given-names">VA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Májek</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jae</span>  <span class="cit-name-given-names">LT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bigenzahn</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nieuwenhuis</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Staring</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sacco</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Diemen</span>  <span class="cit-name-given-names">FR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olk</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stukalov</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Gene essentiality and synthetic lethality in haploid human cells</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">1092</span>–<span class="cit-lpage">1096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac7557%26rft_id%253Dinfo%253Apmid%252F26472760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTAvNjI2NC8xMDkyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMTAvMjgvMDg0MTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.3" data-doi="10.1016/S0006-291X(03)01192-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">HY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">ZCURVE_CoV: a new system to recognize protein coding genes in coronavirus genomes, and its applications in analyzing SARS-CoV genomes</span>. <abbr class="cit-jnl-abbrev">Biochem Biophys Res Commun</abbr> <span class="cit-vol">307</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DChen%26rft.auinit1%253DL.%2BL.%26rft.volume%253D307%26rft.issue%253D2%26rft.spage%253D382%26rft.epage%253D388%26rft.atitle%253DZCURVE_CoV%253A%2Ba%2Bnew%2Bsystem%2Bto%2Brecognize%2Bprotein%2Bcoding%2Bgenes%2Bin%2Bcoronavirus%2Bgenomes%252C%2Band%2Bits%2Bapplications%2Bin%2Banalyzing%2BSARS-CoV%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0006-291X%252803%252901192-6%26rft_id%253Dinfo%253Apmid%252F12859968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0006-291X(03)01192-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12859968&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.4" data-doi="10.1093/nar/gkr986"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">WH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minguez</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lercher</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">OGEE: an online gene essentiality database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr986%26rft_id%253Dinfo%253Apmid%252F22075992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22075992&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300135&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.5" data-doi="10.1093/bioinformatics/bti058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Understanding protein dispensability through machine-learning analysis of high-throughput data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti058%26rft_id%253Dinfo%253Apmid%252F15479713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15479713&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227241200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.6" data-doi="10.1186/1471-2164-14-910"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">A new computational strategy for predicting essential genes</span>. <abbr class="cit-jnl-abbrev">BMC genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%26rft.volume%253D14%26rft.spage%253D910%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.7" data-doi="10.1186/1752-0509-3-102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">del Rio</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koschützki</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coello</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">How to identify essential genes from molecular networks?</span> <abbr class="cit-jnl-abbrev">BMC Syst Biol</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DDel%2BRio%26rft.auinit1%253DG.%26rft.volume%253D3%26rft.spage%253D102%26rft.epage%253D102%26rft.atitle%253DHow%2Bto%2Bidentify%2Bessential%2Bgenes%2Bfrom%2Bmolecular%2Bnetworks%253F%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-3-102%26rft_id%253Dinfo%253Apmid%252F19822021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-3-102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19822021&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.8" data-doi="10.1093/nar/gkq784"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minai</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hassett</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">LJ.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Investigating the predictability of essential genes across distantly related organisms using an integrative approach</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq784%26rft_id%253Dinfo%253Apmid%252F20870748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq784&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20870748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">YZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">FZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">HL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">YN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Labena</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">FB.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Combining the pseudo dinucleotide composition with the Z curve method to improve the accuracy of predicting DNA elements: a case study in recombination spots</span>. <abbr class="cit-jnl-abbrev">Mol Biosyst</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2893</span>–<span class="cit-lpage">2900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiosyst%26rft.volume%253D12%26rft.spage%253D2893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">RE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">KW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">XR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">CJ.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">LIBLINEAR: A library for large linear classification</span>. <abbr class="cit-jnl-abbrev">J Mach Learn Res</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1871</span>–<span class="cit-lpage">1874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMach%2BLearn%2BRes%26rft.volume%253D9%26rft.spage%253D1871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000262636800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Essential Human Genes</span>. <abbr class="cit-jnl-abbrev">Cell Systems</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D1%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.12" data-doi="10.1038/nature08497"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fullwood</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">MH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">YF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mohamed</span>  <span class="cit-name-given-names">YB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orlov</span>  <span class="cit-name-given-names">YL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Velkov</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">PH</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">An oestrogen-receptor-alpha-bound human chromatin interactome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">462</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFullwood%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D462%26rft.issue%253D7269%26rft.spage%253D58%26rft.epage%253D64%26rft.atitle%253DAn%2Boestrogen-receptor-alpha-bound%2Bhuman%2Bchromatin%2Binteractome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08497%26rft_id%253Dinfo%253Apmid%252F19890323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19890323&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271419200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.13" data-doi="10.1093/bioinformatics/btg467"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Comparison of various algorithms for recognizing short coding sequences of human genes</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg467%26rft_id%253Dinfo%253Apmid%252F14764563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14764563&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220485300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gatto</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miess</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Flux balance analysis predicts essential genes in clear cell renal cell carcinoma metabolism</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">10738</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D5%26rft.spage%253D10738%26rft_id%253Dinfo%253Apmid%252F26040780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26040780&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.15" data-doi="10.1093/nar/gkv1283"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gohlke</span>  <span class="cit-name-given-names">BO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nickel</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunkel</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preissner</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">CancerResource--updated database of cancer-relevant proteins, mutations and interacting drugs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D932</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1283%26rft_id%253Dinfo%253Apmid%252F26590406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26590406&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.16" data-doi="10.1093/nar/gkg254"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">FB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">HY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">ZCURVE: a new system for recognizing protein-coding genes in bacterial and archaeal genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1780</span>–<span class="cit-lpage">1789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGuo%26rft.auinit1%253DF.-B.%26rft.volume%253D31%26rft.issue%253D6%26rft.spage%253D1780%26rft.epage%253D1789%26rft.atitle%253DZCURVE%253A%2Ba%2Bnew%2Bsystem%2Bfor%2Brecognizing%2Bprotein-coding%2Bgenes%2Bin%2Bbacterial%2Band%2Barchaeal%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg254%26rft_id%253Dinfo%253Apmid%252F12626720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg254&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12626720&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181546900030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.17" data-doi="10.1186/1471-2105-7-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">FB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">ZCURVE_V: a new self-training system for recognizing protein-coding genes in viral and phage genomes</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVos%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D7%26rft.spage%253D9%26rft.epage%253D9%26rft.atitle%253DZCURVE_V%253A%2Ba%2Bnew%2Bself-training%2Bsystem%2Bfor%2Brecognizing%2Bprotein-coding%2Bgenes%2Bin%2Bviral%2Band%2Bphage%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-9%26rft_id%253Dinfo%253Apmid%252F16401352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16401352&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.18" data-doi="10.1023/A:1012487302797"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guyon</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weston</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barnhill</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vapnik</span>  <span class="cit-name-given-names">V.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Gene selection for cancer classification using support vector machines</span>. <abbr class="cit-jnl-abbrev">Machine learning</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMachine%2Blearning%26rft.volume%253D46%26rft.spage%253D389%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1012487302797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1012487302797&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000171501800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.19" data-doi="10.1016/j.cell.2015.11.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandrashekhar</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aregger</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steinhart</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacLeod</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mis</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fradet-Turcotte</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">High-resolution CRISPR screens reveal fitness genes and genotype-specific cancer liabilities</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1515</span>–<span class="cit-lpage">1526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D1515%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.11.015%26rft_id%253Dinfo%253Apmid%252F26627737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.11.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26627737&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.20" data-doi="10.1186/s12859-016-1015-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moffat</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">BAGEL: a computational framework for identifying essential genes from pooled library screens</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%26rft.volume%253D17%26rft.spage%253D164%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1015-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1015-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.21" data-doi="10.1093/nar/gkv491"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">ZG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">YZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">FB.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">ZCURVE 3.0: identify prokaryotic genes with higher accuracy as well as automatically and accurately select essential genes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">W85</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv491%26rft_id%253Dinfo%253Apmid%252F25977299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25977299&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.22" data-doi="10.1016/j.tcb.2011.07.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Juhas</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eberl</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">JI.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Essence of life: essential genes of minimal genomes</span>. <abbr class="cit-jnl-abbrev">Trends Cell Biol</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DJuhas%26rft.auinit1%253DM.%26rft.volume%253D21%26rft.issue%253D10%26rft.spage%253D562%26rft.epage%253D568%26rft.atitle%253DEssence%2Bof%2Blife%253A%2Bessential%2Bgenes%2Bof%2Bminimal%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2011.07.005%26rft_id%253Dinfo%253Apmid%252F21889892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2011.07.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21889892&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296002600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.23" data-doi="10.1038/nature01278"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kamath</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poulin</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gotta</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanapin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Le Bot</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sohrmann</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Systematic functional analysis of the Caenorhabditis elegans genome using RNAi</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKamath%26rft.auinit1%253DR.%2BS.%26rft.volume%253D421%26rft.issue%253D6920%26rft.spage%253D231%26rft.epage%253D237%26rft.atitle%253DSystematic%2Bfunctional%2Banalysis%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bgenome%2Busing%2BRNAi.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01278%26rft_id%253Dinfo%253Apmid%252F12529635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12529635&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180397600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.24" data-doi="10.1038/nbt.1628"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">DU</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayles</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">HO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Won</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">HS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duhig</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nam</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">G</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKim%26rft.auinit1%253DD.%2BU.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D617%26rft.epage%253D623%26rft.atitle%253DAnalysis%2Bof%2Ba%2Bgenome-wide%2Bset%2Bof%2Bgene%2Bdeletions%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1628%26rft_id%253Dinfo%253Apmid%252F20473289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20473289&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278820200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.25" data-doi="10.1101/gr.3992505"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuepfer</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauer</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blank</span>  <span class="cit-name-given-names">LM.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Metabolic functions of duplicate genes in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1421</span>–<span class="cit-lpage">1430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DKuepfer%26rft.auinit1%253DL.%26rft.volume%253D15%26rft.issue%253D10%26rft.spage%253D1421%26rft.epage%253D1430%26rft.atitle%253DMetabolic%2Bfunctions%2Bof%2Bduplicate%2Bgenes%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3992505%26rft_id%253Dinfo%253Apmid%252F16204195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC8xNDIxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMTAvMjgvMDg0MTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.26" data-doi="10.1016/j.tig.2007.05.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">BY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Mouse duplicate genes are as essential as singletons</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">378</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLiao%26rft.auinit1%253DB.%2BY.%26rft.volume%253D23%26rft.issue%253D8%26rft.spage%253D378%26rft.epage%253D381%26rft.atitle%253DMouse%2Bduplicate%2Bgenes%2Bare%2Bas%2Bessential%2Bas%2Bsingletons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.05.006%26rft_id%253Dinfo%253Apmid%252F17559966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17559966&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248919300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.27" data-doi="10.1105/tpc.15.00051"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seddon</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moghe</span>  <span class="cit-name-given-names">GD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simenc</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">SH.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Characteristics of Plant Essential Genes Allow for within- and between-Species Prediction of Lethal Mutant Phenotypes</span>. <abbr class="cit-jnl-abbrev">Plant cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2133</span>–<span class="cit-lpage">2147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bcell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00051%26rft_id%253Dinfo%253Apmid%252F26286535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMjEzMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE2LzEwLzI4LzA4NDEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.28" data-doi="10.1093/nar/gkt1131"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">DEG 10, an update of the database of essential genes that includes both protein-coding genes and noncoding genomic elements</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D574</span>–<span class="cit-lpage">D580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1131%26rft_id%253Dinfo%253Apmid%252F24243843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24243843&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800085&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.29" data-doi="10.1073/pnas.93.19.10268"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mushegian</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV.</span></span> <span class="cit-pub-date">1996</span>. <span class="cit-article-title">A minimal gene set for cellular life derived by comparison of complete bacterial genomes</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">10268</span>–<span class="cit-lpage">10273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMushegian%26rft.auinit1%253DA.%2BR.%26rft.volume%253D93%26rft.issue%253D19%26rft.spage%253D10268%26rft.epage%253D10273%26rft.atitle%253DA%2Bminimal%2Bgene%2Bset%2Bfor%2Bcellular%2Blife%2Bderived%2Bby%2Bcomparison%2Bof%2Bcomplete%2Bbacterial%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.19.10268%26rft_id%253Dinfo%253Apmid%252F8816789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTMvMTkvMTAyNjgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8xMC8yOC8wODQxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.30" data-doi="10.1016/j.biocel.2003.08.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">HY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">FB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">GS-Finder: a program to find bacterial gene start sites with a self-training method</span>. <abbr class="cit-jnl-abbrev">Int J Biochem Cell Biol</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Binternational%2Bjournal%2Bof%2Bbiochemistry%2B%2526%2Bcell%2Bbiology%26rft.stitle%253DInt%2BJ%2BBiochem%2BCell%2BBiol%26rft.aulast%253DOu%26rft.auinit1%253DH.%2BY.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D535%26rft.epage%253D544%26rft.atitle%253DGS-Finder%253A%2Ba%2Bprogram%2Bto%2Bfind%2Bbacterial%2Bgene%2Bstart%2Bsites%2Bwith%2Ba%2Bself-training%2Bmethod.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocel.2003.08.013%26rft_id%253Dinfo%253Apmid%252F14687930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biocel.2003.08.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14687930&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222537400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.31" data-doi="10.1128/JB.00771-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pechter</span>  <span class="cit-name-given-names">KB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pyles</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manoil</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harwood</span>  <span class="cit-name-given-names">CS.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Essential Genome of the Metabolically Versatile Alphaproteobacterium Rhodopseudomonas palustris</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">867</span>–<span class="cit-lpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBacteriol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00771-15%26rft_id%253Dinfo%253Apmid%252F26712940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTk4LzUvODY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMTAvMjgvMDg0MTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.32" data-doi="10.1186/1752-0509-6-87"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">FX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Iteration method for predicting essential proteins based on orthology and protein-protein interaction networks</span>. <abbr class="cit-jnl-abbrev">BMC Syst Biol</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DPeng%26rft.auinit1%253DW.%26rft.volume%253D6%26rft.spage%253D87%26rft.epage%253D87%26rft.atitle%253DIteration%2Bmethod%2Bfor%2Bpredicting%2Bessential%2Bproteins%2Bbased%2Bon%2Borthology%2Band%2Bprotein-protein%2Binteraction%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-6-87%26rft_id%253Dinfo%253Apmid%252F22808943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-6-87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22808943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Porta-Pardo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hrabe</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Cancer3D: understanding cancer mutations through protein structures</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D968</span>–<span class="cit-lpage">973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D43%26rft.spage%253DD968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.34" data-doi="10.1126/science.1204038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">Y</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Genome-sequencing anniversary. Presenting the human genome: now in 3D!</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">1025</span>–<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBotstein%26rft.auinit1%253DD.%26rft.volume%253D331%26rft.issue%253D6020%26rft.spage%253D1025%26rft.epage%253D1025%26rft.atitle%253DFruits%2Bof%2BGenome%2BSequences%2Bfor%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1204038%26rft_id%253Dinfo%253Apmid%252F21350164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE1OiIzMzEvNjAyMC8xMDI1LWEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8xMC8yOC8wODQxMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.35" data-doi="10.1101/gr.5144106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seringhaus</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paccanaro</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Predicting essential genes in fungal genomes</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1126</span>–<span class="cit-lpage">1135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5144106%26rft_id%253Dinfo%253Apmid%252F16899653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE2LzkvMTEyNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE2LzEwLzI4LzA4NDEyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.36" data-doi="10.1126/science.aac7041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birsoy</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">NW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krupczak</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Post</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">ES</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabatini</span>  <span class="cit-name-given-names">DM.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Identification and characterization of essential genes in the human genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">1096</span>–<span class="cit-lpage">1101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac7041%26rft_id%253Dinfo%253Apmid%252F26472758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTAvNjI2NC8xMDk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMTAvMjgvMDg0MTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.37" data-doi="10.1371/journal.pone.0072343"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">LW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">YN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">FB.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Geptop: a gene essentiality prediction tool for sequenced bacterial genomes based on orthology and phylogeny</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">e72343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De72343%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0072343%26rft_id%253Dinfo%253Apmid%252F23977285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0072343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23977285&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Z curve theory-based analysis of the dynamic nature of nucleosome positioning in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">530</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D530%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Feature selection and analysis on correlated gas sensor data with recursive feature elimination</span>. <abbr class="cit-jnl-abbrev">Sens. Actuators. B. Chem.</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSens.%2BActuators.%2BB.%2BChem.%26rft.volume%253D212%26rft.spage%253D353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.40" data-doi="10.1186/1471-2105-9-113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">ZG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anh</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">LQ.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Human Pol II promoter recognition based on primary sequences and free energy of dinucleotides</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DYang%26rft.auinit1%253DJ.%2BY.%26rft.volume%253D9%26rft.spage%253D113%26rft.epage%253D113%26rft.atitle%253DHuman%2BPol%2BII%2Bpromoter%2Brecognition%2Bbased%2Bon%2Bprimary%2Bsequences%2Band%2Bfree%2Benergy%2Bof%2Bdinucleotides.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-113%26rft_id%253Dinfo%253Apmid%252F18294399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18294399&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.41" data-doi="10.1093/bioinformatics/bts120"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Predicting the lethal phenotype of the knockout mouse by integrating comprehensive genomic data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1246</span>–<span class="cit-lpage">1252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts120%26rft_id%253Dinfo%253Apmid%252F22419784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22419784&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303338200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.42" data-doi="10.1093/nar/28.14.2804"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Recognition of protein coding genes in the yeast genome at better than 95% accuracy based on the Z curve</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2804</span>–<span class="cit-lpage">2814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DC.-T.%26rft.volume%253D28%26rft.issue%253D14%26rft.spage%253D2804%26rft.epage%253D2814%26rft.atitle%253DRecognition%2Bof%2Bprotein%2Bcoding%2Bgenes%2Bin%2Bthe%2Byeast%2Bgenome%2Bat%2Bbetter%2Bthan%2B95%2525%2Baccuracy%2Bbased%2Bon%2Bthe%2BZ%2Bcurve%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.14.2804%26rft_id%253Dinfo%253Apmid%252F10908339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.14.2804&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10908339&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088427500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.43" data-doi="10.1006/jtbi.1997.0401"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT</span></span>. <span class="cit-pub-date">1997</span>. <span class="cit-article-title">A symmetrical theory of DNA sequences and its applications</span>. <abbr class="cit-jnl-abbrev">J Theor Biol</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DZhang%26rft.auinit1%253DC.%2BT.%26rft.volume%253D187%26rft.issue%253D3%26rft.spage%253D297%26rft.epage%253D306%26rft.atitle%253DA%2Bsymmetrical%2Btheory%2Bof%2BDNA%2Bsequences%2Band%2Bits%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.1997.0401%26rft_id%253Dinfo%253Apmid%252F9245572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jtbi.1997.0401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9245572&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XQ20300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.44" data-doi="10.1006/jmbi.1994.1263"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">KC.</span></span> <span class="cit-pub-date">1994</span>. <span class="cit-article-title">A graphic approach to analyzing codon usage in 1562 Escherichia coli protein coding sequences</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">238</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DC.%2BT.%26rft.volume%253D238%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D8%26rft.atitle%253DA%2Bgraphic%2Bapproach%2Bto%2Banalyzing%2Bcodon%2Busage%2Bin%2B1562%2BEscherichia%2Bcoli%2Bprotein%2Bcoding%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1994.1263%26rft_id%253Dinfo%253Apmid%252F8145249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1994.1263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8145249&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NG67900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.45" data-doi="10.1093/nar/19.22.6313"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Analysis of distribution of bases in the coding sequences by a diagrammatic technique</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">6313</span>–<span class="cit-lpage">6317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DC.-T.%26rft.volume%253D19%26rft.issue%253D22%26rft.spage%253D6313%26rft.epage%253D6317%26rft.atitle%253DAnalysis%2Bof%2Bdistribution%2Bof%2Bbases%2Bin%2Bthe%2Bcoding%2Bsequences%2Bby%2Ba%2Bdigrammatic%2Btechnique%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F19.22.6313%26rft_id%253Dinfo%253Apmid%252F1956790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/19.22.6313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1956790&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GR78500036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-084129v1.46" data-doi="10.1080/07391102.1994.10508031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">CT.</span></span> <span class="cit-pub-date">1994</span>. <span class="cit-article-title">Z curves, an intutive tool for visualizing and analyzing the DNA sequences</span>. <abbr class="cit-jnl-abbrev">J Biomol Struct Dyn</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiomolecular%2Bstructure%2B%2526%2Bdynamics%26rft.stitle%253DJ%2BBiomol%2BStruct%2BDyn%26rft.aulast%253DZhang%26rft.auinit1%253DR.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D767%26rft.epage%253D782%26rft.atitle%253DZ%2Bcurves%252C%2Ban%2Bintutive%2Btool%2Bfor%2Bvisualizing%2Band%2Banalyzing%2Bthe%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07391102.1994.10508031%26rft_id%253Dinfo%253Apmid%252F8204213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07391102.1994.10508031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8204213&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F10%2F28%2F084129.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MZ78400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/084129v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F084129v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/084244v1" title="Genome-wide screen for haploinsufficient cell size genes in the opportunistic yeast Candida albicans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/084178v1" title="The need to include phylogeny in trait-based analyses of community composition" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 28, 2016.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/084129v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/084129v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/084129v1" class="active" data-icon-position="" data-hide-link-title="0">Accurate prediction of human essential genes using only nucleotide composition and association information</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/23431" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-E7s2527hEosaC12WV6pxFm_LtvoP9u4jhoD6R23WHTY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="23431" id="node23431--2" data-pisa="biorxiv;084129v1" data-pisa-master="biorxiv;084129" data-seqnum="23431" data-apath="/biorxiv/early/2016/10/28/084129.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Accurate prediction of human essential genes using only nucleotide composition and association information</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Feng-Biao</span> <span class="nlm-surname">Biao</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Chuan</span> <span class="nlm-surname">Dong</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hong-Li</span> <span class="nlm-surname">Hua</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Shuo</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Luo</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Hong-Wan</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Yan-Ting</span> <span class="nlm-surname">Jin</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Kai-Yue</span> <span class="nlm-surname">Zhang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">084129; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/084129 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/084129v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;title=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;title=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;text=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;title=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;t=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;title=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;title=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="23431" id="node23431--3" data-pisa="biorxiv;084129v1" data-pisa-master="biorxiv;084129" data-seqnum="23431" data-apath="/biorxiv/early/2016/10/28/084129.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Accurate prediction of human essential genes using only nucleotide composition and association information</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Feng-Biao</span> <span class="nlm-surname">Biao</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Chuan</span> <span class="nlm-surname">Dong</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hong-Li</span> <span class="nlm-surname">Hua</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Shuo</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hao</span> <span class="nlm-surname">Luo</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Hong-Wan</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Yan-Ting</span> <span class="nlm-surname">Jin</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Kai-Yue</span> <span class="nlm-surname">Zhang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">084129; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/084129 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information&amp;rft.volume&amp;rft.issue&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Biao%2C+Feng-Biao&amp;rft.au=Dong%2C+Chuan&amp;rft.au=Hua%2C+Hong-Li&amp;rft.au=Liu%2C+Shuo&amp;rft.au=Luo%2C+Hao&amp;rft.au=Zhang%2C+Hong-Wan&amp;rft.au=Jin%2C+Yan-Ting&amp;rft.au=Zhang%2C+Kai-Yue"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/23431/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/23431/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/23431/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/23431/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/23431/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/23431/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/23431/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/23431/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/23431/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/23431/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/23431/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/23431/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;count=horizontal&amp;via=&amp;text=Accurate%20prediction%20of%20human%20essential%20genes%20using%20only%20nucleotide%20composition%20and%20association%20information&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/084129v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/084129v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/084129v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
