<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5633171">5633171</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0185866">10.1371/journal.pone.0185866</a></span><span class="publisher-id" title="publisher-id">PONE-D-17-25494</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunodeficiency Viruses</span>: <span class="subject" title="subject">HIV</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunodeficiency Viruses</span>: <span class="subject" title="subject">HIV</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunodeficiency Viruses</span>: <span class="subject" title="subject">HIV</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunodeficiency Viruses</span>: <span class="subject" title="subject">HIV</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Retroviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Lentivirus</span>: <span class="subject" title="subject">HIV</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Retroviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Lentivirus</span>: <span class="subject" title="subject">HIV</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Retroviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Lentivirus</span>: <span class="subject" title="subject">HIV</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Retroviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Lentivirus</span>: <span class="subject" title="subject">HIV</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epidemiology</span>: <span class="subject" title="subject">HIV epidemiology</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Diagnostic medicine</span>: <span class="subject" title="subject">HIV diagnosis and management</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and Places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population Groupings</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Ethnicities</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">European People</span>: <span class="subject" title="subject">Romanian People</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span>: <span class="subject" title="subject">Sequence Analysis</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral diseases</span>: <span class="subject" title="subject">HIV infections</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Transmission and Infection</span></div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Epidemic dispersion of HIV and HCV in a population of co-infected Romanian injecting drug users</div> <div class="alt-title" title="alt-title">
HIV and HCV transmission patterns in Romanian PWID</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6707-3720</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraschiv</span><span class="given-names" tagx="given-names" title="given-names">Simona</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banica</span><span class="given-names" tagx="given-names" title="given-names">Leontina</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicolae</span><span class="given-names" tagx="given-names" title="given-names">Ionelia</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niculescu</span><span class="given-names" tagx="given-names" title="given-names">Iulia</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff002"><sup>2</sup></a><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abagiu</span><span class="given-names" tagx="given-names" title="given-names">Adrian</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jipa</span><span class="given-names" tagx="given-names" title="given-names">Raluca</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pineda-Peña</span><span class="given-names" tagx="given-names" title="given-names">Andrea-Clemencia</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff004"><sup>4</sup></a><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pingarilho</span><span class="given-names" tagx="given-names" title="given-names">Marta</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neaga</span><span class="given-names" tagx="given-names" title="given-names">Emil</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Theys</span><span class="given-names" tagx="given-names" title="given-names">Kristof</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff006"><sup>6</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Libin</span><span class="given-names" tagx="given-names" title="given-names">Pieter</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff006"><sup>6</sup></a><a href="#aff007"><sup>7</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otelea</span><span class="given-names" tagx="given-names" title="given-names">Dan</span></span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abecasis</span><span class="given-names" tagx="given-names" title="given-names">Ana</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a><a href="#aff006"><sup>6</sup></a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Molecular Diagnostics Laboratory, National Institute for Infectious Diseases ‘Matei Bals’, Bucharest, Romania</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Clinical Department, National Institute for Infectious Diseases ‘Matei Bals’, Bucharest, Romania</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">SMZ Süd—Kaiser-Franz-Josef-Spital, 4. Med. Abteilung, Vienna, Austria</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Global Health and Tropical Medicine, Instituto de Higiene e Medicina Tropical, Universidade Nova de Lisboa, Lisbon, Portugal</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Molecular Biology and Immunology Department, Fundación Instituto de Inmunología de Colombia (FIDIC) and Basic Sciences Department, Universidad del Rosario, Bogotá, Colombia</span></span><span class="citation_author_institution" id="aff006">[6], <span class="addr-line" title="addr-line">KU Leuven—University of Leuven, Department of Microbiology and Immunology, Rega Institute for Medical Research, Clinical and Epidemiological Virology, Leuven, Belgium</span></span><span class="citation_author_institution" id="aff007">[7], <span class="addr-line" title="addr-line">Artificial Intelligence lab, Department of computer science, Vrije Universiteit Brussel, Brussels, Belgium</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">Zhefeng</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Fudan University, CHINA</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">mona_manaila@yahoo.com</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-10-10</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span></span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0185866</span><span class="history" title="history"><span class="received" title="received">received: 2017-7-6</span><span class="accepted" title="accepted">accepted: 2017-9-20</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Paraschiv et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pone.0185866.pdf" title="self-uri">pone.0185866.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Co-infections with HIV and HCV are very frequent among people who inject drugs (PWID). However, very few studies comparatively reconstructed the transmission patterns of both viruses in the same population. We have recruited 117 co-infected PWID during a recent HIV outbreak in Romania. Phylogenetic analyses were performed on HIV and HCV sequences in order to characterize and compare transmission dynamics of the two viruses. Three large HIV clusters (2 subtype F1 and one CRF14_BG) and thirteen smaller HCV transmission networks (genotypes 1a, 1b, 3a, 4a and 4d) were identified. Eighty (65%) patients were both in HIV and HCV transmission chains and 70 of those shared the same HIV and HCV cluster with at least one other patient. Molecular clock analysis indicated that all identified HIV clusters originated around 2006, while the origin of the different HCV clusters ranged between 1980 (genotype 1b) and 2011 (genotypes 3a and 4d). HCV infection preceded HIV infection in 80.3% of cases. Coincidental transmission of HIV and HCV was estimated to be rather low (19.65%) and associated with an outbreak among PWID during detention in the same penitentiary. This study has reconstructed and compared the dispersion of these two viruses in a PWID population.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100006595</span><span class="institution" title="institution">Unitatea Executiva pentru Finantarea Invatamantului Superior, a Cercetarii, Dezvoltarii si Inovarii</span></div> </div><span class="award-id" tagx="award-id" title="award-id">BEST-HOPE project 4-003/2012</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otelea</span><span class="given-names" tagx="given-names" title="given-names">Dan</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100006595</span><span class="institution" title="institution">Unitatea Executiva pentru Finantarea Invatamantului Superior, a Cercetarii, Dezvoltarii si Inovarii</span></div> </div><span class="award-id" tagx="award-id" title="award-id">HIV-ID 260/2015</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6707-3720</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraschiv</span><span class="given-names" tagx="given-names" title="given-names">Simona</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100005802</span><span class="institution" title="institution">Autoritatea Natională pentru Cercetare Stiintifică</span></div> </div><span class="award-id" tagx="award-id" title="award-id">GreenVac project, 5SEE/2014</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otelea</span><span class="given-names" tagx="given-names" title="given-names">Dan</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100005802</span><span class="institution" title="institution">Autoritatea Natională pentru Cercetare Stiintifică</span></div> </div><span class="award-id" tagx="award-id" title="award-id">CRCBABI project (642/2014)</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100005802</span><span class="institution" title="institution">Autoritatea Natională pentru Cercetare Stiintifică</span></div> </div><span class="award-id" tagx="award-id" title="award-id">POSDRU/1591/1.5/S/135760-CERO</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neaga</span><span class="given-names" tagx="given-names" title="given-names">Emil</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Fonds Wetenschappelijk Onderzoek – Vlaanderen</span></div><span class="award-id" tagx="award-id" title="award-id">PhD grant</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Libin</span><span class="given-names" tagx="given-names" title="given-names">Pieter</span></span></span></div> <div class="funding-statement" title="funding-statement">
This study was supported by the Executive Agency for Higher Education, Research, Development and Innovation Funding (<a href="http://uefiscdi.gov.ro/">http://uefiscdi.gov.ro/</a>) grant BEST-HOPE 4- 003/2012 (DO) and project HIV-ID 260/2015 (SP). The National Authority for Scientific Research (<a href="http://research.ro/en">http://research.ro/en</a>) partially supported this work, GreenVac project, 5SEE/2014 (DO) and POSCCE program CRCBABI project (642/2014). EN was supported by National Authority for Scientific Research(ANCS), grant POSDRU/1591/1.5/S/135760-CERO. PL was supported by PhD grant of the Fonds Wetenschappelijk Onderzoek – Vlaanderen. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="86 HIV nucleotide sequences reported in this paper were newly deposited in GenBank and have the following accession numbers: KX158994 – KX159079. The remaining 31 HIV sequences were previously published in Genbank [11] and have the following accession numbers: KJ194665, KJ194822, KJ194819, KJ194811, KJ194662, KJ194666, KJ194821, KJ194824, KJ194733, KJ194816, KJ194818, KJ194826, KJ194668, KJ194817, KJ194823, KJ194669, KJ194808, KJ194664, KJ194663, KJ194810, KJ194820, KJ194813, KJ194809, KJ194828, KJ194807, KJ194806, KJ194827, KJ194815, KJ194825, KJ194805, KJ194672. HCV nucleotide sequences used in this analysis were recently deposited in Genbank. HCV-NS5b sequences were deposited with the following accession numbers: KX158877 – KX158993. Accession numbers for HCV – NS3 nucleotide sequences corresponding to PWID are: KX159080 – KX159189 and for the monoinfected patients (control group) are: KX159190 – KX159224." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>86 HIV nucleotide sequences reported in this paper were newly deposited in GenBank and have the following accession numbers: KX158994 – KX159079. The remaining 31 HIV sequences were previously published in Genbank [<a href="#pone.0185866.ref011">11</a>] and have the following accession numbers: KJ194665, KJ194822, KJ194819, KJ194811, KJ194662, KJ194666, KJ194821, KJ194824, KJ194733, KJ194816, KJ194818, KJ194826, KJ194668, KJ194817, KJ194823, KJ194669, KJ194808, KJ194664, KJ194663, KJ194810, KJ194820, KJ194813, KJ194809, KJ194828, KJ194807, KJ194806, KJ194827, KJ194815, KJ194825, KJ194805, KJ194672. HCV nucleotide sequences used in this analysis were recently deposited in Genbank. HCV-NS5b sequences were deposited with the following accession numbers: KX158877 – KX158993. Accession numbers for HCV – NS3 nucleotide sequences corresponding to PWID are: KX159080 – KX159189 and for the monoinfected patients (control group) are: KX159190 – KX159224.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>The use of contaminated needles is an effective way of spreading HIV; it accounts for 10% of infections worldwide and for more than 40% in some regions and countries (Eastern Europe, South America, and East and Southeast Asia) [<a href="#pone.0185866.ref001">1</a>, <a href="#pone.0185866.ref002">2</a>]. Seroprevalence of HCV antibodies is very frequent among people who inject drugs (PWID) rising up to 90%; in the general HIV infected population it raises only to about 10% [<a href="#pone.0185866.ref003">3</a>–<a href="#pone.0185866.ref005">5</a>]. HIV-HCV co-infection affects the natural history of both diseases: increased mortality risk, increased AIDS and liver-related disease mortality, accelerated hepatic fibrosis that leads to cirrhosis and/or hepatocellular carcinoma [<a href="#pone.0185866.ref006">6</a>].</p> <p>Both viruses have a high level of genetic diversity within and between hosts and a high rate of evolution. For HIV-1, several groups, subtypes and circulating recombinant forms (CRFs) are described; for HCV seven different genotypes have been described and each genotype has distinct subtypes [<a href="#pone.0185866.ref007">7</a>, <a href="#pone.0185866.ref008">8</a>]. The high evolutionary rate allows us to study the transmission history of both viruses by analysing their genomic sequences with phylogenetic methods.</p> <p>In recent years, an important HIV outbreak was reported among PWID in Romania, with 30.6% of the total newly diagnosed HIV cases in the country in 2012 [<a href="#pone.0185866.ref009">9</a>]. The most affected region was Bucharest and its suburbs where more than 90% of Romanian PWID are living [<a href="#pone.0185866.ref010">10</a>]. The Romanian PWID infected with HIV are mainly young males, unemployed, often diagnosed in penitentiary, who inject both heroin and new psychoactive drugs ("legal highs�?), and are frequently co-infected with HCV. Two predominant HIV strains were identified in PWID: subtype F1 which had already been circulating in sexual and nosocomially infected patients and the newly endemic CRF14_BG, most probably originating in Spain or Portugal [<a href="#pone.0185866.ref011">11</a>, <a href="#pone.0185866.ref012">12</a>].</p> <p>Previous studies established the role of phylogenetic analysis in assessing and characterizing transmission networks within particular populations, but these were mainly performed in specific populations of HIV patients. However, few studies have been performed that reconstruct simultaneously the transmission history of HIV and HCV viruses in the same population of HIV-HCV co-infected patients [<a href="#pone.0185866.ref013">13</a>, <a href="#pone.0185866.ref014">14</a>] and none analysed the overlap between HIV and HCV transmission networks. Our objective was to reconstruct the history and the transmission patterns of HIV and HCV, to assess the likelihood of simultaneous transmission of these two viruses in a population of co-infected PWID and to analyse the congruence between HIV and HCV transmission networks.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="studypopulation" title="sec"> <div class="title" tagx="title" title="title">
Study population</div> <p>We have included in this study a number of 117 injecting drug users who were diagnosed with HIV infection between 2011 and 2014 in the National Institute for Infectious Diseases ‘Matei Bals’, the reference center for HIV genotyping. All patients were HCV co-infected and naive to antiretroviral treatment. A number of 35 non-PWID, HCV mono-infected patients, were also included in the study.</p> <p>Blood samples and clinical and epidemiological data were collected through a questionnaire survey. Laboratory testing (i.e. HIV viral load, CD4 count, HCV serology, HCV viral load) was performed with <i>in vitro</i> diagnostic (IVD) tests.</p> </div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics statement</div> <p>The study was conducted according to the Declaration of Helsinki and approved by the Ethical Committee of the National Institute for Infectious Diseases ‘Matei Bals’. All the subjects included in this study were adults who provided their written informed consent.</p> </div> <div class="hivgenotyping" title="sec"> <div class="title" tagx="title" title="title">
HIV genotyping</div> <p>Total viral RNA was extracted from 1 ml of plasma using the automated NucliSens EasyMAG nucleic acid extraction system (BioMerieux, Marcy-l'Étoile, France), according to the manufacturer's instructions and eluted in 25μl of water. Ten microliters of RNA were used for HIV genotyping. Reverse transcription and amplification of the HIV-1 <i>pol</i> gene (1301 nucleotides, HXB2: 2253–3554) were performed using the Viroseq HIV-1 Genotyping System (Celera Diagnostics, Alameda, CA) and sequenced with the ABI 3500 Genetic Analyzer (Applied Biosystems, Foster City, CA). The primary data were analysed using Sequencing Analysis Software Version 3.7 (Life Technologies) and the generated sequences were assembled with ViroSeq 2.8 HIV-1 Genotyping System Software (Celera Diagnostics, Alameda, CA). REGA HIV-1&amp;amp;2 automated subtyping tool version 3.0 was used to assign the HIV-1 subtype [<a href="#pone.0185866.ref015">15</a>]. Unclassified samples were further analysed in order to identify possible recombination breakpoints using Simplot v3.5.1 software and the Los Alamos HIV-1 subtype reference dataset (sliding window: 400-nt, T:t ratio = 2.0, model of evolution: Kimura two-parameter, bootstrap: 1000 replicates). All sequences were screened for hypermutation using the Hypermut 2.0 algorithm [<a href="#pone.0185866.ref016">16</a>].</p> </div> <div class="hcvgenotyping" title="sec"> <div class="title" tagx="title" title="title">
HCV genotyping</div> <p>Following RNA extraction as reported above, the NS5b and NS3 regions were sequenced for HCV genotyping. Ten microliters of extracted viral RNA were used for reverse transcription with Transcriptor Reverse Transcriptase (Roche) and random primers (Roche). For amplifying the NS5b region (374 nucleotides, H77: 8256–8630), 1,1X Platinum PCR SuperMix (Life Technologies) and previously published primers were used [<a href="#pone.0185866.ref017">17</a>]. Different sets of primers, specifically designed for genotype 1, 2, 3a, 3b and 4 and 1,1X Platinum PCR SuperMix (Life Technologies) were used to amplify the first part of the NS3 gene (710 nucleotides, H77:3326–4036) [<a href="#pone.0185866.ref018">18</a>].</p> <p>The NS5b and NS3 amplicons were bidirectionally sequenced using BigDye® Terminator system v 1.1 (Life Technologies) and ABI 3500 Genetic Analyzer (Applied Biosystems). Seqscape version 2.7 (Applied Biosystems) was used to assemble and generate consensus sequences. Genotyping was done using the publicly available algorithm Oxford HCV Automated Subtyping Tool (Version 2.0) [<a href="#pone.0185866.ref019">19</a>].</p> </div> <div class="phylogenetic,phylodynamicanalysisanddivergencetimesoftransmissionnetworks" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic, phylodynamic analysis and divergence times of transmission networks</div> <p>Reference sequences were selected from public databases [<a href="#pone.0185866.ref020">20</a>, <a href="#pone.0185866.ref021">21</a>] taking into account genotype / subtype, genomic region and geographic origin. To retrieve control sequences for transmission cluster reconstruction, the most similar HIV sequences were retrieved from our and Los Alamos database using BLAST. Duplicates and clones were discarded. Of 421 selected control sequences, those representative to geographical and time dispersion of the patients sample, were subsequently used for each analysis. Controls for all the HIV-1 subtypes and CRFs identified among Romanian PWID were selected as follows: CRF14_BG (Spain, Portugal, Romania), subtype F1 (Angola, Romania), subtype B (European countries), CRF35_AD (Afghanistan, Iran). CRF14BG epidemics from Spain-Portugal [<a href="#pone.0185866.ref011">11</a>] and Greece [<a href="#pone.0185866.ref012">12</a>] were previously linked to the Romanian HIV epidemic among PWID. Subtype F1 strains from Romanian PWID were previously shown [<a href="#pone.0185866.ref011">11</a>] to originate from strains already circulating in Romania and formerly described as related with strains from Angola [<a href="#pone.0185866.ref022">22</a>, <a href="#pone.0185866.ref023">23</a>].</p> <p>A similar procedure was used for the HCV dataset, where a number of 304 control sequences were used. Multiple alignments were generated using Muscle [<a href="#pone.0185866.ref024">24</a>].When available, concatenate alignments of NS5b and NS3 sequences were used in the Bayesian analyses.</p> <p>Phylogenetic analyses were performed using both maximum-likelihood (ML) and Bayesian approaches, after selecting the best-fit nucleotide substitution model using JModelTest (GTR+4Γ+I for HIV-1 and TN93 for HCV). Datasets for each HIV-1 subtype and HCV genotype identified among Romanian PWID were constructed and analyzed separately. ML trees were inferred using FastTree software [<a href="#pone.0185866.ref025">25</a>] and Bayesian phylogenetic analyses were performed with BEAST version 1.8.1 [<a href="#pone.0185866.ref026">26</a>] using a Bayesian Skyline coalescent tree prior [<a href="#pone.0185866.ref027">27</a>] and the uncorrelated lognormal relaxed clock model [<a href="#pone.0185866.ref028">28</a>]. Two Markov Chain Monte Carlo (MCMC) runs were computed separately for 10<sup>7</sup> generations with a burn-in of 10%. The output of the MCMC analysis was tested for convergence by means of effective sampling size (ESS&amp;gt;100) using the program Tracer v1.6 [<a href="#pone.0185866.ref029">29</a>]. The maximum clade credibility (MCC) tree was summarized with TreeAnnotator program [<a href="#pone.0185866.ref030">30</a>].</p> <p>Transmission clusters were identified as clusters in the ML phylogenetic tree with SH-like support higher or equal to 0.9 and were further confirmed with Bayesian phylogenetic analyses if supported by a Bayesian posterior probability value of 1.0 [<a href="#pone.0185866.ref031">31</a>, <a href="#pone.0185866.ref032">32</a>]. The temporal signal of both datasets was evaluated with TempEst [<a href="#pone.0185866.ref033">33</a>]. Data were statistically evaluated and plotted using GraphPad Prism v6.0. The Mann-Whitney test was performed to identify differences between groups and the Chi-square or Fisher exact test were used to evaluate the distribution in particular subgroups; p-values &amp;lt;0.05 were considered significant.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="characteristicsofthepopulation" title="sec"> <div class="title" tagx="title" title="title">
Characteristics of the population</div> <p>The population of 117 co-infected PWID included in this study were diagnosed with HIV between 2011 and 2014. The diagnosis of HCV and HIV was simultaneous for 71 patients (60.7%), while for the remaining 46 individuals HCV diagnosis was established earlier than the HIV diagnosis. Most of the PWID (64.1%, 75) were asymptomatic at the HIV diagnosis, while 35.9% were diagnosed in more advanced HIV infection stages, classified as stage B (27%, 32) and C (9%, 10) according to CDC and WHO Staging Systems [<a href="#pone.0185866.ref034">34</a>]. The 35 control HCV mono-infected patients were diagnosed during the same period of time, also in the National Institute for Infectious Diseases ‘Matei Bals’.</p> </div> <div class="diseaseseveritydifferswhencomparingpatientsinfectedwithhiv-1subtypef1andwithcrf14_bg-likestrains" title="sec"> <div class="title" tagx="title" title="title">
Disease severity differs when comparing patients infected with HIV-1 subtype F1 and with CRF14_BG-like strains</div> <p>HIV-1 subtype analysis showed that 67.5% (n = 79) of the patients were infected with subtype F1 viruses; recombinant forms present in a significant proportion (26%; n = 30) were CRF14_BG (n = 19, 16.24%) and unique recombinant forms (URF) of CRF_14BG and F1 (n = 11, 9.4%). The PWID infected with these recombinant forms had significantly lower CD4 counts at baseline than patients infected with subtype F1 strains. It is true, though, that an estimation of the time to the infecting event was not possible in either case. No significant differences between these two groups were observed when HIV viral load (VL) or CDC stages were analysed (<a href="#pone.0185866.g001">Fig 1</a>). However, a higher percentage of patients infected with CRF14_BG were diagnosed at stages B and C than the group infected with F1 (47% vs. 34%). Few patients (n = 8, 6.8%) were infected with other types of HIV: subtype B, CRF35_AD and URFs between subtype A1 or B and F1 (<a href="#pone.0185866.t001">Table 1</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0185866.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Clinical and laboratory characteristics of studied patients.</div> <p>A. Differences in HIV-VL among PWID, the two groups infected with F1 subtype and recombinant forms (CRF14_BG, URFs of CRF14_BG and F1) B. Differences in HCV-VL in mono-infected and HIV co-infected patients (PWID) C. CD4 count distribution in PWID at baseline D. Clinical HIV stages at diagnosis in PWID.</p>   </div> <div class="table-wrap_UNKNOWN" id="pone.0185866.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0185866.t001">10.1371/journal.pone.0185866.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Distribution of HIV subtypes and HCV genotypes in Romania co-infected PWID.</div> <p>Chi-square analysis was performed on 2x3 contingency table with pure HIV subtype /recombinant forms and HCV genotype 1a/1b/other and showed no significant difference in HIV subtype and HCV genotype distribution (p = 0.784).</p>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0185866.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="center" rowspan="2" colspan="1">HIV subtypes</th> <th align="center" colspan="5" rowspan="1">HCV genotypes</th> <th align="center" rowspan="1" colspan="1">Total</th> </tr> <tr> <th align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">1a</th> <th align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">1b</th> <th align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">3a</th> <th align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">4a</th> <th align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">4d</th> <th align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1" /> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">Subtype F1</td> <td align="center" rowspan="1" colspan="1">30</td> <td align="center" rowspan="1" colspan="1">29</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1"><b>79</b></td> </tr> <tr> <td align="left" style="background-color:#CCCCCC" rowspan="1" colspan="1">Subtype B</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">3</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">1</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>4</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">CRF14_BG</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">0</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1"><b>19</b></td> </tr> <tr> <td align="left" style="background-color:#CCCCCC" rowspan="1" colspan="1">CRF35_AD</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">1</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>1</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Recombinants CRF14_BG and F1</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1"><b>11</b></td> </tr> <tr> <td align="left" style="background-color:#CCCCCC" rowspan="1" colspan="1">Other recombinants</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">1</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">0</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>3</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>Total</b></td> <td align="center" rowspan="1" colspan="1"><b>48</b></td> <td align="center" rowspan="1" colspan="1"><b>40</b></td> <td align="center" rowspan="1" colspan="1"><b>14</b></td> <td align="center" rowspan="1" colspan="1"><b>7</b></td> <td align="center" rowspan="1" colspan="1"><b>8</b></td> <td align="center" rowspan="1" colspan="1"><b>117</b></td> </tr> </tbody> </table> </div> </div> </div> <div class="hcvviralloadsarehigherinpatientswithhivco-infection" title="sec"> <div class="title" tagx="title" title="title">
HCV viral loads are higher in patients with HIV co-infection</div> <p>All 117 patients were tested to assess the HCV genotype by analysing NS5b and/or NS3 regions of HCV. Both NS5b and NS3 sequences were available for 110 patients, while for the remaining patients only the NS5b region was available. The genotype assessment indicated that the majority of PWID were infected with genotype 1a (n = 47, 40%) and 1b (n = 40, 34%), but genotypes 3a and 4 (a, d) were also identified (<a href="#pone.0185866.t001">Table 1</a>). HCV subtype assignment was concordant in the two genomic regions in the majority of cases (n = 108, 98.2%), with only two exceptions of genotype 1 sequences (2749bh2013 and 2727bh2013) assigned as 1a in NS3 and 1b in NS5b. Unlike HIV-HCV co-infected PWID, the individuals in the HCV mono-infected group were exclusively infected with genotype 1b strains. Significant differences (<a href="#pone.0185866.g001">Fig 1</a>) in HCV VL were observed when comparing mono-infected with co-infected patients, with higher viral loads in the group of co-infected.</p> </div> <div class="allidentifiedhivclustersarelargeandoriginatedaround10yearsago" title="sec"> <div class="title" tagx="title" title="title">
All identified HIV clusters are large and originated around 10 years ago</div> <p>Phylogenetic analysis of Romanian PWID using control sequences from Romania and other parts of the world showed that 90 HIV sequences (76.9%) were part of well-supported phylogenetic clusters, suggesting high levels of transmission networking. Three major HIV local transmission networks were identified by phylogenetic analysis, two within subtype F1 and one with CRF14_BG (<a href="#pone.0185866.g002">Fig 2</a>). One of the subtype F1 clusters included 29 PWID patients, 24 men and 5 women; 17 men (58%) were diagnosed in the same penitentiary. The other subtype F1 cluster identified in PWID is larger and consists of 44 sequences corresponding to 34 men and 10 women living in the same geographical area. The lineages of both F1 clusters seem to originate from strains previously identified in Romanian people living with HIV (PLWH). The transmission network identified in PWID infected with CRF14_BG involved 17 subjects, diagnosed with low CD4 counts (mean 138 cells/mm). The CD4 counts were significantly lower than in patients from the two F1 transmission networks (mean CD4 count of cluster 1–430 cells/mm, cluster 2–510 cells/mm, p&amp;lt;0.001).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0185866.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Phylogenetic analysis of HIV-1 Romanian PWID sequences.</div> <p>Circular tree: Maximum likelihood tree generated as described under Methods section. The sequences corresponding to PWID are colored in red and in black the control sequences. Rectangular trees: The transmission networks identified in PWID were further confirmed with Bayesian phylogenetic analyses. Molecular clock analyses were performed separately for F1 subtype and CRF14_BG sequences. In F1 subtype tree, in red are marked the PWID sequences, in blue are represented sequences of sexually infected patients from Romania and in black are marked F1 sequences from Angola. In CRF14_BG tree, the PWID sequences in red are marked and in green the sequences from Spain and Portugal (PWID). Different clusters were highlighted on the trees and tMRCA was specifically marked for each cluster. The posterior probability support for each transmission cluster is represented at the internal nodes. The scale is in years.</p>   </div> <p>Phylodynamic analysis for all three HIV transmission networks among PWID indicated similar starting time points, dating back to around 10 years ago (2006–2009, widest %95 HPD: 2001–2010). The results are in concordance with the epidemiological and seroconversion data available from these patients.</p> </div> <div class="hcvtransmissionclustersaresmallerandtheiroriginsaremoredispersedintime" title="sec"> <div class="title" tagx="title" title="title">
HCV transmission clusters are smaller and their origins are more dispersed in time</div> <p>Phylogenetic reconstructions uncovered several HCV transmission clusters presented in <a href="#pone.0185866.g003">Fig 3</a>. As opposed to HIV where the virus has spread in only three large clusters, HCV dispersion occurred through multiple transmission networks. In total, 97 HCV sequences (82.9%) clustered in 13 transmission clusters. In comparison, within the HCV mono-infected control group only 8 out of 33 (24.2%) HCV sequences were found to form phylogenetic clusters (χ<sup>2</sup> = 42.18, p&amp;lt;0.001).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0185866.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Phylogenetic analysis of HCV sequences in Romanian HIV co-infected PWID.</div> <p>Circular tree: Maximum likelihood tree generated as described under Methods section. The sequences corresponding to PWID are colored in red and in black the control sequences. Rectangular trees: The transmission networks identified in PWID were further confirmed with Bayesian phylogenetic analyses. Molecular clock analyses were performed separately for HCV genotypes 1a, 1b, 3a, 4a and 4d. Different clusters were specifically indicated on the trees. The posterior probability support and tMRCA for each transmission cluster are represented at the internal nodes. The scale is in years.</p>   </div> <p>Four well supported clusters of different sizes were observed in both genotypes 1a and 1b; the largest consisted of 23 sequences (genotype 1a) and 13 sequences (genotype 1b) (<a href="#pone.0185866.t002">Table 2</a>). Three small genotype 3a clusters were observed, while genotype 4a and 4d sequences identified among PWID formed each one single well supported cluster (<a href="#pone.0185866.g003">Fig 3</a>).</p> <div class="table-wrap_UNKNOWN" id="pone.0185866.t002" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0185866.t002">10.1371/journal.pone.0185866.t002</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
HCV and HIV transmission networks identified within Romanian PWID.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0185866.t002g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="center" rowspan="1" colspan="1">HCV genotype</th> <th align="center" rowspan="1" colspan="1">Cluster</th> <th align="center" rowspan="1" colspan="1">Number of PWID sequences within the cluster</th> <th align="center" rowspan="1" colspan="1">(%)</th> <th align="center" rowspan="1" colspan="1">Sampling period</th> <th align="center" rowspan="1" colspan="1">tMRCA (95% HPD)</th> </tr> </thead> <tbody> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>Genotype 1a (N = 48)</b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">a</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">6</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">12.5</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>2008</b> (2006–2011)</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">b</td> <td align="center" rowspan="1" colspan="1">5</td> <td align="center" rowspan="1" colspan="1">10.42</td> <td align="center" rowspan="1" colspan="1">2013–2014</td> <td align="center" rowspan="1" colspan="1"><b>1995</b> (1983–2002)</td> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">c</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">23</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">47.92</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>1987</b>(1979–1995)</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b> </b></td> <td align="center" rowspan="1" colspan="1">d</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">18.75</td> <td align="center" rowspan="1" colspan="1">2013–2014</td> <td align="center" rowspan="1" colspan="1"><b>2004</b> (2001–2007)</td> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1" /> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>Total</b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>43</b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>89.58</b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1" /> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>Genotype 1b (N = 40)</b></td> <td align="center" rowspan="1" colspan="1">e</td> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">22.50</td> <td align="center" rowspan="1" colspan="1">2013–2014</td> <td align="center" rowspan="1" colspan="1"><b>1992</b> (1982–2002)</td> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">f</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">4</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">10.00</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>1983</b> (1968–1994)</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">g</td> <td align="center" rowspan="1" colspan="1">13</td> <td align="center" rowspan="1" colspan="1">32.5</td> <td align="center" rowspan="1" colspan="1">2013–2014</td> <td align="center" rowspan="1" colspan="1"><b>2007</b> (2002–2010)</td> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">h</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">5</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">12.50</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>1989</b> (1979–1999)</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1"><b>Total</b></td> <td align="center" rowspan="1" colspan="1"><b>31</b></td> <td align="center" rowspan="1" colspan="1"><b>77.5</b></td> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>Genotype 3a (N = 14)</b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">i</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">4</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">28.57</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>2007</b> (2002–2011)</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b> </b></td> <td align="center" rowspan="1" colspan="1">j</td> <td align="center" rowspan="1" colspan="1">2</td> <td align="center" rowspan="1" colspan="1">14.29</td> <td align="center" rowspan="1" colspan="1">2013–2014</td> <td align="center" rowspan="1" colspan="1"><b>2011</b> (2007–2013)</td> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1" /> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">k</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">3</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">21.43</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>2011</b> (2008–2013)</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1"><b>Total</b></td> <td align="center" rowspan="1" colspan="1"><b>9</b></td> <td align="center" rowspan="1" colspan="1"><b>64.28</b></td> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>Genotype 4a (N = 7)</b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">l</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">7</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">100</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>2010</b> (2007–2012)</td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>Genotype 4d (N = 8)</b></td> <td align="center" rowspan="1" colspan="1">m</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">87.5</td> <td align="center" rowspan="1" colspan="1">2013–2014</td> <td align="center" rowspan="1" colspan="1"><b>2011</b> (2007–2013)</td> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>HIV subtype</b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> </tr> <tr> <td align="center" rowspan="1" colspan="1"><b>Subtype F1 (N = 79)</b></td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">29</td> <td align="center" rowspan="1" colspan="1">36.71</td> <td align="center" rowspan="1" colspan="1">2013–2014</td> <td align="center" rowspan="1" colspan="1"><b>2006</b> (2001–2010)</td> </tr> <tr> <td align="left" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b> </b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">44</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">55.70</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>2009</b> (2002–2010)</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1"><b>Total</b></td> <td align="center" rowspan="1" colspan="1"><b>73</b></td> <td align="center" rowspan="1" colspan="1"><b>92.40</b></td> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> </tr> <tr> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>CRF14_BG and recombinants (N = 30)</b></td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">3</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">17</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">56.67</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1">2013–2014</td> <td align="center" style="background-color:#CCCCCC" rowspan="1" colspan="1"><b>2007</b> (2004–2010)</td> </tr> </tbody> </table> </div> </div> <p>Molecular clock analysis of HCV transmission clusters revealed important differences in origin and starting time points. The differences in divergence time were observed within as well as among genotypes. Based on relaxed clock and constant size population model implemented in a Bayesian MCMC framework, the oldest PWID network belongs to genotype 1b and originated around 1983 (95%HPD: 1968–1994), followed by the larger HCV genotype 1a cluster, dated back to 1987 (Figs <a href="#pone.0185866.g003">3</a> and <a href="#pone.0185866.g004">4</a>). However, the time to the most recent common ancestor (tMRCA) estimated for a sub-group of this 1a cluster consisting in 22 out of 23 sequences was more recent (2004, 95%HPD 1999–2006).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0185866.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
The most recent common ancestor (tMRCA) of HIV and HCV transmission clusters in PWID.</div> <p>The mean tMRCA and the high posterior density interval (95%HPD) for each identified transmission cluster are plotted.</p>   </div> <p>Genotype 1b was found to be the first introduced in the studied population: the oldest cluster was estimated to be 30 years old in PWID and more than 50 years in non-PWID. Genotypes 3 and 4 are the most recently introduced in this risk group population, dating back 2–3 years before sampling. The divergence times for all important PWID networks, HIV and HCV, are presented in <a href="#pone.0185866.t002">Table 2</a> and graphically illustrated in <a href="#pone.0185866.g004">Fig 4</a>.</p> </div> <div class="coincidentaltransmissionofhivandhcvisuncommonandoccurredin19.65%ofthepatients" title="sec"> <div class="title" tagx="title" title="title">
Coincidental transmission of HIV and HCV is uncommon and occurred in 19.65% of the patients</div> <p>We have comparatively analysed the patterns of patients distribution in HIV and HCV transmission clusters (<a href="#pone.0185866.g005">Fig 5</a>). Eighty patients were present both in HCV and in HIV transmission clusters (68.4% of all co-infected patients; 82.5% and 88.9% of all HCV and HIV patients present in clusters, respectively). Furthermore, 70 patients (59.8%) were present in the same HIV and HCV cluster with at least one other patient, indicating transmission of HIV and HCV through (at least partly) overlapping transmission chains.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0185866.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Transmission dynamics of HIV and HCV infections.</div> <p>The comparative analysis of the clustering patterns in HIV-HCV co-infected patients is presented. The number of all sequences that were part of both HIV and HCV clusters are shown, in brackets being the simultaneous transmission cases; sequences with a difference of mean divergence times between HCV and HIV (ΔtMRCA) &amp;lt;1.</p>   </div> <p>Phylogenetic analysis and, in particular, measuring the difference in divergence times between viral lineages within a co-infected individual has been proven useful in assessing the frequency of viral coincidental transmission in PWID in the absence of seroconversion data [<a href="#pone.0185866.ref035">35</a>]. We have used the calculated difference of mean divergence times (ΔtMRCA) between HIV and HCV lineages within a patient to estimate the time period lapsed between the acquisition of the two viruses. Low ΔtMRCA indicates a higher likelihood that the two viruses might have been co-transmitted to a susceptible individual, while higher values of ΔtMRCA suggest that the viruses were transmitted at different and distinct time points.</p> <p>For the majority of the patients this analysis suggested that HCV was introduced first, followed by HIV. However, for 5 patients, HIV was introduced first; all of them were infected with subtype F1 and genotype 3a or 4 HCV strains. Twenty three patients had ΔtMRCA between HIV and HCV lineages of less than one year, suggesting potential co-transmission of HIV and HCV. These 23 patients (19.65%) were present in the same HIV and HCV cluster with at least one other patient, suggesting a potential simultaneous transmission (co-transmission) of both infections. We also found that such transmission events are largely associated with particular HCV transmission clusters, specifically clusters c and l (<a href="#pone.0185866.g005">Fig 5</a>). Going back to the records of these patients, we found that twenty-two of those 23 patients were in detention at the moment of study enrolment. This population has a statistically significant higher probability of coincidental transmission (p&amp;lt;0.0001) when compared to the rest of the patients. Five of these patients were infected with CRF14_BG and the rest with subtype F1; the infective HCV genotypes were 1 or 4 (<a href="#pone.0185866.g005">Fig 5</a>).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The PWID represent no more than an estimated 0.2–0.5% of the world’s population, but they account for approximately 5–10% of all people living with HIV and 6.8% of the persons infected with HCV [<a href="#pone.0185866.ref036">36</a>, <a href="#pone.0185866.ref037">37</a>]. Since 2008 injected amphetamine type stimulants ("legal highs�?) became available on the market in Romania and were low-priced. As a result, the number of PWID increased dramatically in recent years so that Romania reported an HIV outbreak among PWID that started in 2011; more than 90% of PWID being concentrated in Bucharest and its surroundings [<a href="#pone.0185866.ref010">10</a>, <a href="#pone.0185866.ref038">38</a>]. Numerous people who were injecting heroin have switched to the new class of drugs due to the lower price and because they were legal at that time. An important behavioral characteristic is that the latter require up to 10 injections every day. This fact, which was coincident with an interruption in needle exchange programs, greatly impacted the needle-sharing behavior in the country [<a href="#pone.0185866.ref039">39</a>]. Preliminary analyses indicated the circulation of CRF14_BG strains in PWID, HIV strains associated with lower CD4 counts and X4 tropism at baseline, as compared with the subtype F1 infected PWID [<a href="#pone.0185866.ref011">11</a>, <a href="#pone.0185866.ref040">40</a>].</p> <p>In this study we have identified, characterized and compared the viral transmission networks of HIV and HCV in a population of HIV-HCV co-infected PWID–using a group of HCV mono-infected patients as controls—and we evaluated the time of introduction of these viruses in this population and explored the probability of co-transmission of the two infections.</p> <p>The clinical and epidemiological impact of HIV-HCV co-infections in PWID has been described by several studies [<a href="#pone.0185866.ref006">6</a>, <a href="#pone.0185866.ref041">41</a>], but only one analysed in parallel the evolutionary history of these two viruses within this particular risk population [<a href="#pone.0185866.ref035">35</a>]. Viral genetic information expressed as nucleotide sequences and analysed with phylogenetic and population genetics has been shown to be very useful in identifying and characterizing transmission clusters of rapidly evolving viruses such as HIV and HCV [<a href="#pone.0185866.ref042">42</a>, <a href="#pone.0185866.ref043">43</a>].</p> <p>Phylogenetic analysis performed in this study showed high levels of transmission networking among Romanian PWIDs: three HIV clusters and thirteen HCV clusters have been identified. The majority of HIV strains fell in two F1 clusters that were nested within Romanian reference strains. This suggests that F1 strains circulating for a long time among Romanian patients, infected sexually and nosocomially were the origin of the subtype F1 outbreaks in PWID and that these strains were restricted to Romania. The CRF14_BG strains circulating in PWID are related with those described earlier in Spain and Portugal in the same risk group population [<a href="#pone.0185866.ref011">11</a>]. However, due to the limited number of CRF14_BG sequences available in public databases, direct filiation between the two epidemiologic events cannot be ascertained. Previous results indicated that the CRF14_BG sequences from Romanian PWID were closely related with the Greek PWID sequences and formed a single monophyletic cluster. Molecular clock analysis revealed that CRF14_BG strains circulating in PWID from Athens originated from one or several strains circulating in Bucharest [<a href="#pone.0185866.ref012">12</a>]. Our phylodynamic analyses indicated that all HIV transmission networks originated at similar time points, dating back 7 to 10 years ago, coincident with the availability of these new injectable amphetamine type stimulants on the market. The patients infected with CRF14_BG had significantly lower CD4 counts at baseline than those infected with F1 subtype strains despite the fact that phylogenetic and seroconversion data suggest their being infected at similar time points. Although we have included in our analysis all the PWID presented in our clinical settings during this period of time, the number of patients infected with CRF14_BG recombinants was smaller than those infected with subtype F1 and this might have impact on the statistical analysis. However, these data are in agreement with previous studies showing an association between CRF14_BG infection, CD4 T cells decline and disease severity, characterized by CXCR4 usage [<a href="#pone.0185866.ref011">11</a>, <a href="#pone.0185866.ref040">40</a>, <a href="#pone.0185866.ref044">44</a>].</p> <p>On the other hand, regardless of the HIV infecting strain, HCV viral loads were significantly higher in co-infected PWID than in HCV mono-infected patients. This difference does not seem to be related to the HCV genotype. Moreover, when comparing HCV mono-infected patients with co-infected PWID carrying HCV genotype 1b, the difference in viral load continued to be statistically significant (<a href="#pone.0185866.s001">S1 Fig</a>). Several studies have shown an accelerated disease progression in HIV-HCV co-infected patients, HIV infection increasing the HCV replication, hepatic inflammation and impairing HCV-specific immune response [<a href="#pone.0185866.ref006">6</a>]. Since CRF14_BG induced a massive depletion of CD4 T cells it was expected to be associated with impaired HCV immune control and thus higher replication and hepatic fibrosis. However, no significant difference in HCV VL was observed between CRF14_BG and subtype F1 infected patients or between HCV VL and CD4 count. Possibly, longer infection periods could be needed to impair the established HCV-specific immune response, since in most of the cases, HCV infection preceded HIV infection.</p> <p>Previous studies have shown the genotype 1b to be mainly acquired through blood transfusion, while genotypes 1a and 3a are associated with intravenous drug use [<a href="#pone.0185866.ref045">45</a>]. In other European countries, mostly HCV genotype 4 has been identified in PWID: subtype 4a in Greece, 4d in Italy and The Netherlands and 4c and 4d in Spain were linked to drug usage [<a href="#pone.0185866.ref046">46</a>–<a href="#pone.0185866.ref049">49</a>]. Our results indicate the circulation of multiple HCV genotypes in co-infected Romanian PWID: 1a, 1b, 3a, 4a and 4d. However, only HCV genotype 1b was found in the HCV mono-infected group, indicating a higher complexity of the HCV epidemic among co-infected PWID.</p> <p>According to phylodynamic analyses, HCV genotype 1b was the first to be introduced in Romania, followed by genotype 1a. The oldest 1b cluster was estimated to be 30 years old in PWID and more than 50 years in non-PWID. Previous studies indicated that the circulation of this genotype in Romania was almost exclusive for a long period of time [<a href="#pone.0185866.ref050">50</a>, <a href="#pone.0185866.ref051">51</a>]. Genotypes 3 and 4 are much more recent and linked to intravenous drug usage; with an estimated origin around 10 years ago, they might also be linked to the availability of injected amphetamine type stimulants [<a href="#pone.0185866.ref052">52</a>].</p> <p>The differences in transmission clustering between HCV and HIV might be explained in part by their estimated acquisition time. Previous observations based on classical epidemiological data indicated that HCV was introduced in PWID much earlier, being associated with heroin use while HIV infection is linked with new psychoactive drugs [<a href="#pone.0185866.ref011">11</a>]. Phylodynamic analysis presented in this study sustains this observation, showing that HIV strains were introduced in PWID in recent years whereas HCV genotypes 1a and 1b had been circulating several decades before. However, HCV transmission networks of genotypes 3 and 4 apparently started later, as they were dated as very recent (2010–2011), thus indicating a potential linkage of these genotypes with new psychoactive drugs.</p> <p>When comparing the transmission chains of HIV and HCV, we evaluated both the congruence between HIV and HCV transmission chains, as well as the likelihood of simultaneous transmission of both infections. We found that most patients (68.4%) were present both in HIV and HCV transmission clusters and that for the majority of the patients’ transmission of HIV and HCV occurred through (at least partly) overlapping transmission chains (<a href="#pone.0185866.g005">Fig 5</a>). We used the method reported by Ng TK and collaborators [<a href="#pone.0185866.ref035">35</a>] to evaluate the frequency of viral coincidental transmission. Applying this method to Romanian co-infected PWID, we found that, for most of the patients, and in accordance with epidemiological data, HCV infection was acquired before HIV.</p> <p>Nevertheless, we identified 23 cases of possible co-transmission (19.65%), corresponding to patients with ΔtMRCA&amp;lt;1 and clustering in partly overlapping transmission chains. Interestingly, these transmission events were associated with particular clusters of genotype 1a and 4a. Further investigation lead us to the finding that 22 of these 23 patients were in detention in the same penitentiary. While largely overlapping transmission chains would be expected, given the fact that this population forcely shares the same contact network while in detention, a higher likelihood of simultaneous transmission of the infections is not necessarily expected in this context. Therefore, our findings call for further investigation about what type of behavioral patterns explain the high rate of co-transmission of HIV and HCV in this penitentiary when compared to the rest of the PWID population. One potential bias in our analysis could be the subevaluation of possible super-infections and co-transmissions: phylogenetic analysis based on population sequencing considers only the predominant viral infective strain. Coincidental transmissions could be underestimated in our analysis if the transmitted strains did not become visible within the viral population.</p> <p>The HIV epidemic among Romanian PWID continues to represent a major problem and HCV co-infection contributes to the worse prognostic within this specific risk population. This study clarifies and compares the patterns of spread and transmission of these two viruses in Romanian PWID, suggesting that a higher rate of transmission of HIV rather than HCV is associated with behavioral changes mainly following the availability of new types of injected drugs (‘legal highs’). While overlapping of HIV and HCV transmission chains is common, the estimated transmission of both viruses within a short time span is less common, with a rate of only 19.65% in this population, and largely associated with drug injection within the same penitentiary. Our results illustrate the need for better prevention policies in the PWID population, in particular and more urgently in the prison population, where an outbreak of simultaneous HIV and HCV transmissions seems to be occurring.</p> <p>One limitation of the study is that the inmate population is overrepresented in the PWID so that extrapolation to the general population should be cautious. It should be though noted that, within this rather small group, no significant clinical or epidemiologic differences were noted according to the inprison vs. outprison status.</p> <div class="accessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Accession numbers</div> <p>86 HIV nucleotide sequences reported in this paper were newly deposited in GenBank and have the following accession numbers: KX158994—KX159079. The remaining 31 HIV sequences were previously published in Genbank [<a href="#pone.0185866.ref011">11</a>] and have the following accession numbers: KJ194665, KJ194822, KJ194819, KJ194811, KJ194662, KJ194666, KJ194821, KJ194824, KJ194733, KJ194816, KJ194818, KJ194826, KJ194668, KJ194817, KJ194823, KJ194669, KJ194808, KJ194664, KJ194663, KJ194810, KJ194820, KJ194813, KJ194809, KJ194828, KJ194807, KJ194806, KJ194827, KJ194815, KJ194825, KJ194805, KJ194672.</p> <p>HCV nucleotide sequences used in this analysis were recently deposited in Genbank. HCV-NS5b sequences were deposited with the following accession numbers: KX158877—KX158993. Accession numbers for HCV–NS3 nucleotide sequences corresponding to PWID are: KX159080—KX159189 and for the monoinfected patients (control group) are: KX159190—KX159224.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
HCV viral load (VL) comparison between HCV monoinfected and co-infected patients with genotype 1b.</div> <p>Statistical analysis was performed with Mann-Whitney test, p value &amp;lt; 0.05 was considered significant.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0185866.s001.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>This study was presented in part as oral presentation at the HIV Virology workshop: Recent advances in Molecular Methods, 11–12 June 2015, Athens.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0185866.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathers</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Degenhardt</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiessing</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hickman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strathdee</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Global epidemiology of injecting drug use and HIV among people who inject drugs: a systematic review</span>. <span class="source" tagx="source" title="source">Lancet</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">372</span>:<span class="fpage" tagx="fpage" title="fpage">1733</span>–<span class="lpage" tagx="lpage" title="lpage">45</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/S0140-6736(08)61311-2">10.1016/S0140-6736(08)61311-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18817968">18817968</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stockman</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strathdee</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>. <span class="mixed-article-title" title="mixed-article-title">HIV among people who use drugs: a global perspective of populations at risk</span>. <span class="source" tagx="source" title="source">J Acquir Immune Defic Syndr</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">S17</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="comment">doi: <a href="https://doi.org/10.1097/QAI.0b013e3181f9c04c">10.1097/QAI.0b013e3181f9c04c</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21045594">21045594</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walsh</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maher</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="mixed-article-title" title="mixed-article-title">HIV and viral hepatitis C coinfection in people who inject drugs: implications of new direct acting antivirals for hepatitis C virus treatment</span>. <span class="source" tagx="source" title="source">Curr Opin HIV AIDS</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">339</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>. <span class="comment">doi: <a href="https://doi.org/10.1097/COH.0b013e328354131e">10.1097/COH.0b013e328354131e</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22498482">22498482</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maher</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jalaludin</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chant</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaldor</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>. <span class="mixed-article-title" title="mixed-article-title">High hepatitis C incidence in new injecting drug users: a policy failure?</span><span class="source" tagx="source" title="source">Aust N Z J Public Health</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">30</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17333606">17333606</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Uusküla</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNutt</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehovitz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heimer</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">High prevalence of blood-borne virus infections and high-risk behaviour among injecting drug users in Tallinn, Estonia</span>. <span class="source" tagx="source" title="source">Int J STD AIDS</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">41</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="comment">doi: <a href="https://doi.org/10.1258/095646207779949907">10.1258/095646207779949907</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17326862">17326862</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feeney</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span>. <span class="mixed-article-title" title="mixed-article-title">HCV and HIV co-infection: mechanisms and management</span>. <span class="source" tagx="source" title="source">Nat Rev Gastroenterol Hepatol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">362</span>–<span class="lpage" tagx="lpage" title="lpage">71</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nrgastro.2014.17">10.1038/nrgastro.2014.17</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24535328">24535328</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butler</span><span class="given-names" tagx="given-names" title="given-names">IF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandrea</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marx</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Apetrei</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">HIV genetic diversity: biological and public health consequences</span>. <span class="source" tagx="source" title="source">Curr HIV Res</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">23</span>–<span class="lpage" tagx="lpage" title="lpage">45</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17266555">17266555</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bukh</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuiken</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muerhoff</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rice</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stapleton</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">Expanded classification of hepatitis C virus into 7 genotypes and 67 subtypes: updated criteria and genotype assignment web resource</span>. <span class="source" tagx="source" title="source">Hepatology</span>, <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">318</span>–<span class="lpage" tagx="lpage" title="lpage">27</span>. <span class="comment">doi: <a href="https://doi.org/10.1002/hep.26744">10.1002/hep.26744</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24115039">24115039</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Romanian HIV/AIDS National Committee Report. 2014; 20–21, available from <a href="http://www.cnlas.ro/images/doc/31122014_eng.pdf">http://www.cnlas.ro/images/doc/31122014_eng.pdf</a>, accessed 7 March 2017</span></li> <li tag="ref"><a name="pone.0185866.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Abagiu A and Botescu A. Update on country situation: Romania; 2nd meeting on detecting and responding to outbreaks of HIV among IDUs. 2012; 11, available from <a href="http://www.emcdda.europa.eu/attachements.cfm/att_195213_EN_02%20Romania.pdf">www.emcdda.europa.eu/attachements.cfm/att_195213_EN_02%20Romania.pdf</a></span></li> <li tag="ref"><a name="pone.0185866.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niculescu</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraschiv</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraskevis</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abagiu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Batan</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banica</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otelea</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="mixed-article-title" title="mixed-article-title">Recent HIV-1 outbreak among Intravenous Drug Users in Romania: Evidence for cocirculation of CRF14_BG and subtype F1 strains</span>. <span class="source" tagx="source" title="source">AIDS Res Hum Retroviruses</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">488</span>–<span class="lpage" tagx="lpage" title="lpage">95</span>. <span class="comment">doi: <a href="https://doi.org/10.1089/aid.2014.0189">10.1089/aid.2014.0189</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25369079">25369079</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraskevis</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraschiv</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sypsa</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikolopoulos</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsiara</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magiorkinis</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Enhanced HIV-1 surveillance using molecular epidemiology to study and monitor HIV-1 outbreaks among intravenous drug users (IDUs) in Athens and Bucharest</span>. <span class="source" tagx="source" title="source">Infect Genet Evol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">109</span>–<span class="lpage" tagx="lpage" title="lpage">21</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.meegid.2015.08.004">10.1016/j.meegid.2015.08.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26247720">26247720</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Asten</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verhaest</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamzira</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez-Aguado</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zangerle</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boufassa</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Spread of hepatitis C virus among European injection drug users infected with HIV: a phylogenetic analysis</span>. <span class="source" tagx="source" title="source">J Infect Dis</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">189</span>: <span class="fpage" tagx="fpage" title="fpage">292</span>–<span class="lpage" tagx="lpage" title="lpage">302</span>. <span class="comment">doi: <a href="https://doi.org/10.1086/380821">10.1086/380821</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14722895">14722895</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matthews</span><span class="given-names" tagx="given-names" title="given-names">GV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pham</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hellard</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grebely</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oon</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Patterns and characteristics of hepatitis C transmission clusters among HIV-positive and HIV-negative individuals in the Australian trial in acute hepatitis C</span>. <span class="source" tagx="source" title="source">Clin Infect Dis</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">52</span>: <span class="fpage" tagx="fpage" title="fpage">803</span>–<span class="lpage" tagx="lpage" title="lpage">11</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/cid/ciq200">10.1093/cid/ciq200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21282185">21282185</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pineda-Peña</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imbrechts</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Libin</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abecasis</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deforche</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Automated subtyping of HIV-1 genetic sequences for clinical and surveillance purposes: performance evaluation of the new REGA version 3 and seven other tools</span>. <span class="source" tagx="source" title="source">Infect Genet Evol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">337</span>–<span class="lpage" tagx="lpage" title="lpage">48</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.meegid.2013.04.032">10.1016/j.meegid.2013.04.032</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23660484">23660484</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korber</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Detecting hypermutations in viral sequences with an emphasis on G—&amp;gt; A hypermutation</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2000</span>; <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">400</span>–<span class="lpage" tagx="lpage" title="lpage">1</span>. Hypermut 2.0 –Analysis &amp;amp; Detection of APOBEC-induced Hypermutation, available from: <a href="http://www.hiv.lanl.gov/content/sequence/HYPERMUT/hypermut.html">http://www.hiv.lanl.gov/content/sequence/HYPERMUT/hypermut.html</a><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10869039">10869039</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandres-Sauné</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deny</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pasquier</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thibaut</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duverlie</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Izopet</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Determining hepatitis C genotype by analyzing the sequence of the NS5b region</span>. <span class="source" tagx="source" title="source">J Virol Methods</span>. <span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">109</span>: <span class="fpage" tagx="fpage" title="fpage">187</span>–<span class="lpage" tagx="lpage" title="lpage">93</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12711062">12711062</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vallet</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viron</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henquell</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Guillou-Guillemette</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lagathu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abravanel</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">NS3 protease polymorphism and natural resistance to protease inhibitors in French patients infected with HCV genotypes 1–5</span>. <span class="source" tagx="source" title="source">Antivir Ther</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">1093</span>–<span class="lpage" tagx="lpage" title="lpage">102</span>. <span class="comment">doi: <a href="https://doi.org/10.3851/IMP1900">10.3851/IMP1900</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22024525">22024525</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alcantara</span><span class="given-names" tagx="given-names" title="given-names">LCJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cassol</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Libin</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deforche</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Ranst</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A standardized framework for accurate, high-throughput genotyping of recombinant and non-recombinant viral sequences</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">37</span>: <span class="fpage" tagx="fpage" title="fpage">634</span>–<span class="lpage" tagx="lpage" title="lpage">642</span>.</span></li> <li tag="ref"><a name="pone.0185866.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Available from <a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a></span></li> <li tag="ref"><a name="pone.0185866.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Available from <a href="http://www.ncbi.nlm.nih.gov/nucleotide/">http://www.ncbi.nlm.nih.gov/nucleotide/</a></span></li> <li tag="ref"><a name="pone.0185866.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guimarães</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vicente</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otsuki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">da Silva</span><span class="given-names" tagx="given-names" title="given-names">RF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francisco</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">da Silva</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Close phylogenetic relationship between Angolan and Romanian HIV-1 subtype F1 isolates</span>. <span class="source" tagx="source" title="source">Retrovirology</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">39</span><span class="comment">doi: <a href="https://doi.org/10.1186/1742-4690-6-39">10.1186/1742-4690-6-39</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19386115">19386115</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bello</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afonso</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morgado</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylodynamics of HIV-1 subtype F1 in Angola, Brazil and Romania</span>. <span class="source" tagx="source" title="source">Infect Genet Evol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">1079</span>–<span class="lpage" tagx="lpage" title="lpage">86</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.meegid.2012.03.014">10.1016/j.meegid.2012.03.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22484759">22484759</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>. <span class="mixed-article-title" title="mixed-article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span>. <span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">113</span><span class="comment">doi: <a href="https://doi.org/10.1186/1471-2105-5-113">10.1186/1471-2105-5-113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15318951">15318951</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehal</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkin</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span>. <span class="mixed-article-title" title="mixed-article-title">FastTree 2—approximately maximum-likelihood trees for large alignments</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e9490</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0009490">10.1371/journal.pone.0009490</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">1969</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/mss075">10.1093/molbev/mss075</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748">22367748</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Bayesian coalescent inference of past population dynamics from molecular sequences</span>. <span class="source" tagx="source" title="source">Mol Biol Evol</span>. <span class="year" tagx="year" title="year">2005</span>; <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">1185</span>–<span class="lpage" tagx="lpage" title="lpage">92</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/molbev/msi103">10.1093/molbev/msi103</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15703244">15703244</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Relaxed phylogenetics and dating with confidence</span>. <span class="source" tagx="source" title="source">PLoS Biol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">e88</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pbio.0040088">10.1371/journal.pbio.0040088</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16683862">16683862</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Rambaut A, Suchard M, Drummond A. Tracer. Available from: <a href="http://tree.bio.ed.ac.uk/software/tracer/">http://tree.bio.ed.ac.uk/software/tracer/</a></span></li> <li tag="ref"><a name="pone.0185866.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <span class="source" tagx="source" title="source">BMC Evol Biol</span>. <span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">214</span><span class="comment">doi: <a href="https://doi.org/10.1186/1471-2148-7-214">10.1186/1471-2148-7-214</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zehender</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ebranati</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santoro</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alteri</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giuliani</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Population dynamics of HIV-1 subtype B in a cohort of men-having-sex-with-men in Rome, Italy</span>. <span class="source" tagx="source" title="source">J Acquir Immune Defic Syndr</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">156</span>–<span class="lpage" tagx="lpage" title="lpage">60</span><span class="comment">doi: <a href="https://doi.org/10.1097/QAI.0b013e3181eb3002">10.1097/QAI.0b013e3181eb3002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20703157">20703157</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Audelin</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cowan</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obel</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jørgensen</span><span class="given-names" tagx="given-names" title="given-names">LB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerstoft</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogenetics of the Danish HIV epidemic: the role of very late presenters in sustaining the epidemic</span>. <span class="source" tagx="source" title="source">J Acquir Immune Defic Syndr</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">62</span>: <span class="fpage" tagx="fpage" title="fpage">102</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="comment">doi: <a href="https://doi.org/10.1097/QAI.0b013e318276becc">10.1097/QAI.0b013e318276becc</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23075917">23075917</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Max Carvalho</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span>. <span class="source" tagx="source" title="source">Virus Evol</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">2</span>(<span class="issue" tagx="issue" title="issue">1</span>): <span class="fpage" tagx="fpage" title="fpage">vew007</span><span class="comment">doi: <a href="https://doi.org/10.1093/ve/vew007">10.1093/ve/vew007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774300">27774300</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Guide for HIV/AIDS clinical care, HIV classification: CDC and WHO staging systems. 2014; available from: <a href="http://aidsetc.org/guide/hiv-classification-cdc-and-who-staging-systems">http://aidsetc.org/guide/hiv-classification-cdc-and-who-staging-systems</a></span></li> <li tag="ref"><a name="pone.0185866.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takebe</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chook</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">WZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abed Al-Darraji</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Co-infections and transmission networks of HCV, HIV-1 and HPgV among people who inject drugs</span>. <span class="source" tagx="source" title="source">Sci Rep</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">15198</span><span class="comment">doi: <a href="https://doi.org/10.1038/srep15198">10.1038/srep15198</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26459957">26459957</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">UNAIDS 2013 Global report, The Epidemic’s Enduring Impact On People Who Inject Drugs. 2013; 30–31, available from: <a href="http://www.unaids.org/sites/default/files/media_asset/UNAIDS_Global_Report_2013_en_1.pdf">http://www.unaids.org/sites/default/files/media_asset/UNAIDS_Global_Report_2013_en_1.pdf</a>.</span></li> <li tag="ref"><a name="pone.0185866.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">United Nations Office on Drugs and Crime, World Drug Report 2015 available from: <a href="https://www.unodc.org/documents/wdr2015/WDR15_Chapter_1.pdf">https://www.unodc.org/documents/wdr2015/WDR15_Chapter_1.pdf</a>, pag 6,</span></li> <li tag="ref"><a name="pone.0185866.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Available from: <a href="http://files.unaids.org/en/dataanalysis/knowyourresponse/countryprogressreports/2014countries/ROU_narrative_report_2014.pdf">http://files.unaids.org/en/dataanalysis/knowyourresponse/countryprogressreports/2014countries/ROU_narrative_report_2014.pdf</a></span></li> <li tag="ref"><a name="pone.0185866.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Botescu A, Abagiu A, Mardarescu M, Ursan M. HIV/AIDS among injecting drug users in Romania. Report of a recent outbreak and initial response policies. 2012; 5–14 available from: <a href="http://www.emcdda.europa.eu/attachements.cfm/att_192024_EN_HIV_outbreak_Romania_2012.pdf">http://www.emcdda.europa.eu/attachements.cfm/att_192024_EN_HIV_outbreak_Romania_2012.pdf</a></span></li> <li tag="ref"><a name="pone.0185866.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bártolo</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abecasis</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borrego</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barroso</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCutchan</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomes</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Origin and epidemiological history of HIV-1 CRF14_BG</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">e24130</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0024130">10.1371/journal.pone.0024130</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21969855">21969855</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weinberg</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">RT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Pathogenesis of accelerated fibrosis in HIV/HCV co-infection</span>. <span class="source" tagx="source" title="source">J Infect Dis</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">207</span>: <span class="fpage" tagx="fpage" title="fpage">S13</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/infdis/jis926">10.1093/infdis/jis926</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23390300">23390300</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Oliveira</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salemi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cassol</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciccozzi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology: HIV-1 and HCV sequences from Libyan outbreak</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">444</span>(<span class="issue" tagx="issue" title="issue">7121</span>): <span class="fpage" tagx="fpage" title="fpage">836</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17171825">17171825</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baele</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ward</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">HIV epidemiology. The early spread and epidemic ignition of HIV-1 in human populations</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">346</span>: <span class="fpage" tagx="fpage" title="fpage">56</span>–<span class="lpage" tagx="lpage" title="lpage">61</span>. <span class="comment">doi: <a href="https://doi.org/10.1126/science.1256739">10.1126/science.1256739</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25278604">25278604</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pérez-�?lvarez</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delgado</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vega</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Montero</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cuevas</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernández-García</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Predominance of CXCR4 tropism in HIV-1 CRF14_BG strains from newly diagnosed infections</span>. <span class="source" tagx="source" title="source">J Antimicrob Chemother</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">69</span>: <span class="fpage" tagx="fpage" title="fpage">246</span>–<span class="lpage" tagx="lpage" title="lpage">53</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/jac/dkt305">10.1093/jac/dkt305</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23900735">23900735</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Esteban</span><span class="given-names" tagx="given-names" title="given-names">JI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauleda</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quer</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">The changing epidemiology of hepatitis C virus infection in Europe</span>. <span class="source" tagx="source" title="source">J Hepatol</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">48</span>: <span class="fpage" tagx="fpage" title="fpage">148</span>–<span class="lpage" tagx="lpage" title="lpage">62</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.jhep.2007.07.033">10.1016/j.jhep.2007.07.033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18022726">18022726</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katsoulidou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sypsa</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tassopoulos</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boletis</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karafoulidou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ketikoglou</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology of hepatitis C virus (HCV) in Greece: temporal trends in HCV genotype-specific incidence and molecular characterization of genotype 4 isolates</span>. <span class="source" tagx="source" title="source">J Viral Hepat</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">19</span>–<span class="lpage" tagx="lpage" title="lpage">27</span>. <span class="comment">doi: <a href="https://doi.org/10.1111/j.1365-2893.2005.00649.x">10.1111/j.1365-2893.2005.00649.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16364078">16364078</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ciccozzi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Equestre</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costantino</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marascio</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quirino</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo Presti</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C virus genotype 4d in Southern Italy: reconstruction of its origin and spread by a phylodynamic analysis</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">84</span>: <span class="fpage" tagx="fpage" title="fpage">1613</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="comment">doi: <a href="https://doi.org/10.1002/jmv.23384">10.1002/jmv.23384</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22930510">22930510</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernández-Arcás</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">López-Siles</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trapero</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferraro</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibáñez</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orihuela</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">High prevalence of hepatitis C virus subtypes 4c and 4d in Malaga (Spain): phylogenetic and epidemiological analyses</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">78</span>: <span class="fpage" tagx="fpage" title="fpage">1429</span>–<span class="lpage" tagx="lpage" title="lpage">35</span>. <span class="comment">doi: <a href="https://doi.org/10.1002/jmv.20706">10.1002/jmv.20706</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16998884">16998884</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Bruijne</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schinkel</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prins</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koekkoek</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aronson</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Ballegooijen</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span><span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Emergence of hepatitis C virus genotype 4: phylogenetic analysis reveals three distinct epidemiological profiles</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">3832</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="comment">doi: <a href="https://doi.org/10.1128/JCM.01146-09">10.1128/JCM.01146-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19794040">19794040</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grigorescu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">HCV genotype 1 is almost exclusively present in Romanian patients with chronic hepatitis C</span>. <span class="source" tagx="source" title="source">J Gastrointestin Liver Dis</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">45</span>–<span class="lpage" tagx="lpage" title="lpage">50</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19337633">19337633</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gheorghe</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Csiki</span><span class="given-names" tagx="given-names" title="given-names">IE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iacob</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gheorghe</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smira</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regep</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="mixed-article-title" title="mixed-article-title">The prevalence and risk factors of hepatitis C virus infection in adult population in Romania: a nationwide survey 2006–2008</span>. <span class="source" tagx="source" title="source">J Gastrointestin Liver Dis</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">373</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21188327">21188327</a></span></span></li> <li tag="ref"><a name="pone.0185866.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sultana</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vagu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Temereanca</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grancea</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slobozeanu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Hepatitis C Virus Genotypes in Injecting Drug Users from Romania</span>. <span class="source" tagx="source" title="source">Cent Eur J Med</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">672</span>–<span class="lpage" tagx="lpage" title="lpage">678</span>. <span class="comment">doi: <a href="https://doi.org/10.2478/s11536-011-0073-6">10.2478/s11536-011-0073-6</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23585824">23585824</a></span></span></li> </ul> </div> </div>  </body></html>