<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5439704">5439704</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0177657">10.1371/journal.pone.0177657</a></span><span class="publisher-id" title="publisher-id">PONE-D-17-05110</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene expression</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene regulation</span>: <span class="subject" title="subject">MicroRNAs</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nucleic acids</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Non-coding RNA</span>: <span class="subject" title="subject">MicroRNAs</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene Expression</span>: <span class="subject" title="subject">Gene Regulation</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene expression</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene regulation</span>: <span class="subject" title="subject">Small nucleolar RNAs</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nucleic acids</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Non-coding RNA</span>: <span class="subject" title="subject">Small nucleolar RNAs</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Diseases</span>: <span class="subject" title="subject">Hand, Foot and Mouth Disease</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing techniques</span>: <span class="subject" title="subject">RNA sequencing</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and analysis methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing techniques</span>: <span class="subject" title="subject">RNA sequencing</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Signal Transduction</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Signaling</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Signaling Cascades</span>: <span class="subject" title="subject">WNT Signaling Cascade</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Anatomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nervous System</span>: <span class="subject" title="subject">Central Nervous System</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Anatomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nervous System</span>: <span class="subject" title="subject">Central Nervous System</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Gene Ontologies</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genome Analysis</span>: <span class="subject" title="subject">Gene Ontologies</span></div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Different microRNA profiles reveal the diverse outcomes induced by EV71 and CA16 infection in human umbilical vein endothelial cells using high-throughput sequencing</div> <div class="alt-title" title="alt-title">
Comparison analysis of microRNAs profiles in human umbilical vein endothelial cells infected by EV71 and CA16</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Yajie</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Jiaqi</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Huiwen</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Jingjing</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Lei</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ning</span><span class="given-names" tagx="given-names" title="given-names">Ruotong</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Hongzhe</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Zening</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">Haitao</span></span><a href="#aff001" /></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-2623-5847</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Longding</span></span><a href="#cor001">*</a><a href="#aff001" /></span></div><span class="citation_author_institution" id="aff001">[], <span class="addr-line" title="addr-line">Yunnan Key Laboratory of Vaccine Research &amp;amp; Development on Severe Infections Disease, Institute of Medical Biology, Chinese Academy of Medical Science and Peking Union Medical College, Kunming, China</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">Yun</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Kunming University of Science and Technology, CHINA</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-con" title="con"> <p> </p><ul> <li> <p><b>Conceptualization:</b> JS YH LL.</p> </li> <li> <p><b>Data curation:</b> JS YH JL HZ JW.</p> </li> <li> <p><b>Formal analysis:</b> LG RN HL ZY HF.</p> </li> <li> <p><b>Funding acquisition:</b> LL.</p> </li> <li> <p><b>Investigation:</b> JW LG RN HL ZY HF LL.</p> </li> <li> <p><b>Methodology:</b> JS YH JL HZ JW.</p> </li> <li> <p><b>Project administration:</b> JS YH.</p> </li> <li> <p><b>Resources:</b> LG RN HL ZY HF.</p> </li> <li> <p><b>Software:</b> JS YH JL.</p> </li> <li> <p><b>Supervision:</b> JW LG LL.</p> </li> <li> <p><b>Validation:</b> JS YH JL HZ JW.</p> </li> <li> <p><b>Visualization:</b> JS YH LL.</p> </li> <li> <p><b>Writing – original draft:</b> JS YH.</p> </li> <li> <p><b>Writing – review &amp;amp; editing:</b> JS YH LL.</p> </li> </ul> <p /> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">longdingl@gmail.com</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2017-5-5</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2017</span></span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0177657</span><span class="history" title="history"><span class="received" title="received">received: 2017-2-8</span><span class="accepted" title="accepted">accepted: 2017-5-1</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Song et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2017</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pone.0177657.pdf" title="self-uri">pone.0177657.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Enterovirus 71 (EV71) and Coxsackievirus A16 (CA16) remain the predominant pathogens in hand, foot, and mouth disease (HFMD), but the factors underlying the pathogenesis of EV71 and CA16 infections have not been elucidated. Recently, the functions of microRNAs (miRNAs) in pathogen-host interactions have been highlighted. In the present study, we performed comprehensive miRNA profiling in EV71- and CA16-infected human umbilical vein endothelial cells (HUVECs) at multiple time points using high-throughput sequencing. The results showed that 135 known miRNAs exhibited remarkable differences in expression. Of these, 30 differentially expressed miRNAs presented opposite trends in EV71- and CA16-infected samples. Subsequently, we mainly focused on the 30 key differentially expressed miRNAs through further screening to predict targets. Gene ontology (GO) and pathway analysis of the predicted targets showed the enrichment of 14 biological processes, 9 molecular functions, 8 cellular components, and 85 pathways. The regulatory networks of these miRNAs with predicted targets, GOs, pathways, and co-expression genes were determined, suggesting that miRNAs display intricate regulatory mechanisms during the infection phase. Consequently, we specifically analyzed the hierarchical GO categories of the predicted targets involved in biological adhesion. The results indicated that the distinct changes induced by EV71 and CA16 infection may be partly linked to the function of the blood-brain barrier. Taken together, this is the first report describing miRNA expression profiles in HUVECs with EV71 and CA16 infections using high-throughput sequencing. Our data provide useful insights that may help to elucidate the different host-pathogen interactions following EV71 and CA16 infection and offer novel therapeutic targets for these infections.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">CAMS Innovation Fund for Medical Sciences</span></div><span class="award-id" tagx="award-id" title="award-id">2016-I2M-1-014</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-2623-5847</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Longding</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">National Natural Sciences Foundations of China</span></div><span class="award-id" tagx="award-id" title="award-id">81373142</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-2623-5847</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Longding</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">CAMS Innovation Fund for Medical Sciences</span></div><span class="award-id" tagx="award-id" title="award-id">2016-I2M-1-019</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span></span></div> <div class="funding-statement" title="funding-statement">
This work was supported CAMS Innovation Fund for Medical Sciences (2016-I2M-1-014 and 2016-I2M-1-019) (<a href="http://www.cams.ac.cn/">http://www.cams.ac.cn/</a>) and National Natural Sciences Foundations of China (81373142) (<a href="http://www.nsfc.gov.cn/">http://www.nsfc.gov.cn/</a>). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the paper and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the paper and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Hand, foot, and mouth disease (HFMD) is a widespread viral illness generally occurring in infants and young children, especially those less than 5 years of age[<a href="#pone.0177657.ref001">1</a>]. Enterovirus 71 (EV71) and coxsakievirus A16 (CA16), members of the human enterovirus A species within the genus <i>Enterovirus</i> of the family Picornaviridae, are the predominant etiological agents of HFMD[<a href="#pone.0177657.ref002">2</a>, <a href="#pone.0177657.ref003">3</a>]. Over the past decade, a series of sporadic cases as well as large epidemics associated with HFMD have arisen in the Asia-Pacific region; therefore HFMD has emerged as a serious threat to public health[<a href="#pone.0177657.ref004">4</a>]. Although EV71 and CA16 share relatively high nucleotide and amino acid homologies (77% and 89% respectively), the clinical manifestations caused by the two viruses are significantly different[<a href="#pone.0177657.ref005">5</a>, <a href="#pone.0177657.ref006">6</a>]. Unlike CA16, which causes milder symptoms that resolve within a few weeks, EV71 infections usually cause severe central nervous system (CNS) complications, such as poliomyelitis-like paralysis, aseptic meningitis, and encephalitis[<a href="#pone.0177657.ref007">7</a>, <a href="#pone.0177657.ref008">8</a>]. Thus, prior development of HFMD vaccines has almost exclusively focused on EV71, and 3 successfully developed inactivated EV71 vaccines have been approved by the China Food and Drug Administration (CFDA) since 2015[<a href="#pone.0177657.ref009">9</a>, <a href="#pone.0177657.ref010">10</a>]. Although these vaccines are effective in providing protection against EV71 infections, their effectiveness may be limited because they cannot induce highly potent and broad cross-protection against various genotypes of other enteroviruses, including CA16[<a href="#pone.0177657.ref003">3</a>, <a href="#pone.0177657.ref008">8</a>]. Furthermore, several severe and fatal cases involving CA16 have recently been reported, and epidemiological surveys indicate that HFMD cases induced by the co-circulation of, co-infection by, or recombination between EV71 and CA16 are increasing[<a href="#pone.0177657.ref003">3</a>, <a href="#pone.0177657.ref011">11</a>]. This trend has led to more serious outcomes than would be expected from infection with a single virus. Thus there is an urgent need for a multivalent vaccine against multiple enteroviruses that would help to control enterovirus-related HFMD epidemics. In our previous study, we found that there were significant anti-CA16 neutralizing antibodies in the healthy junior rhesus macaque immunized with the experimentally inactivated CA16 vaccine, but these neutralizing antibodies failed to protect against CA16 infection[<a href="#pone.0177657.ref012">12</a>]. Hence, it is our hope that further investigations of the molecular mechanisms of EV71 and CA16 will provide a new strategy for the development of a broad-spectrum HFMD vaccine.</p> <p>microRNAs (miRNAs) are small, 18- to 25-nucleotide (nt), long single-stranded noncoding regulatory RNA molecules that regulate gene expression at the posttranscriptional level[<a href="#pone.0177657.ref013">13</a>]. Through complementary pairing with messenger RNA (mRNA), miRNAs exhibit inhibitory effects by inhibiting translation or stimulating mRNA degradation[<a href="#pone.0177657.ref014">14</a>]. A number of recent studies have reported that miRNAs participate in regulating the expression of at least 60% of human genes and play an important role in various physiological processes, including cell proliferation, cell differentiation, hematopoiesis, immune responses, the stress response, and the production of chemokines and cytokines[<a href="#pone.0177657.ref013">13</a>]. Therefore, aberrant miRNA expression seems to play a central role in the pathogenesis and progression of several human diseases, especially cancers, neurodegenerative diseases, and cardiovascular disorders[<a href="#pone.0177657.ref015">15</a>, <a href="#pone.0177657.ref016">16</a>]. Furthermore, the altered expression of specific miRNAs has been explored in a search for novel biomarkers to aid in the diagnosis of various diseases as well as their progression and the efficacy of their treatment[<a href="#pone.0177657.ref017">17</a>]. Additionally, as our knowledge of the role of miRNAs in host-pathogen interactions has advanced, it has been found that viral infections can mediate changes in the expression of cell-specific miRNAs[<a href="#pone.0177657.ref018">18</a>]. Such changes represent not only a potential host defense mechanism against infection by efficiently inhibiting viral invasion with minimal clinical consequences but also establish an environment for productive viral infection and/or latency by promoting evasion of the host antiviral immune response or enhancing viral replication and survival in host cells, with the potentially severe outcomes[<a href="#pone.0177657.ref019">19</a>]. In other words, these changes in cellular miRNAs may directly contribute to viral pathogenesis, suggesting that in-depth research into cell-encoded miRNA functioninduced by EV71 and CA16 may help to explain why diverse clinical features result from infections caused by these viruses.</p> <p>Currently, there are several reports indicating that EV71and CA16 can elicit ectopic host miRNA expression profiles in different infected cells, such as human bronchial epithelial (16HBE) cells[<a href="#pone.0177657.ref020">20</a>], peripheral blood mononuclear cells (PBMCs) in rhesus monkeys[<a href="#pone.0177657.ref021">21</a>], and human rhabdomyosarcoma (RD) cells[<a href="#pone.0177657.ref022">22</a>]. However, miRNA biogenesis has unique cell-specific patterns, suggesting that these different miRNA expression profiles would only partially explain the clinical symptoms and might be not involved in the CNS symptoms. EV71 and CA16 are both highly neurotropic viruses that have induced severe CNS complications in patients, and such CNS complications have been recognized as the main causes of fatal HFMD[<a href="#pone.0177657.ref023">23</a>]. However, the underlying neuropathogenetic mechanisms of EV71 and CA16 infections have yet to be clearly understood. Two likely routes by which these viruses invade the CNS have been considered: it may be transmitted to the CNS from the blood, across the blood-brain barrier (BBB), or it may enter the CNS through retrograde axonal transport from peripheral nerves[<a href="#pone.0177657.ref024">24</a>]. The BBB comprises an interdependent network of brain capillary endothelial cells, endowed with barrier properties, and perivascular cells, containing astrocytes and pericytes that are responsible for maintaining the barrier[<a href="#pone.0177657.ref025">25</a>]. Moreover, many neurotropicviruses from the systemic blood circulation, such as Japanese encephalitis virus (JEV)[<a href="#pone.0177657.ref026">26</a>], West Nile virus (WNV)[<a href="#pone.0177657.ref027">27</a>], can infect the endothelial cells of the BBB, thus disrupting its integrity and eventually entering the CNS. With all this in mind and with the help of high-throughput sequencing technology, we used human umbilical vein endothelial cells (HUVECs), which are often used to build models of the BBB <i>in vitro</i>, to investigate alterations in miRNAs due to EV71 and CA16 infection. It is hoped that the results of this study will provide future perspectives regarding the mechanisms underlying EV71 and CA16 neuropathogenesis.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="virusinfectionandcelllineculture" title="sec"> <div class="title" tagx="title" title="title">
Virus infection and cell line culture</div> <p>HUVECs purchased from Jennino Biological Technology were cultured in Dulbecco’s modified eagle medium (DMEM) (Gibco, USA) supplemented with 10% fetal bovine serum (FBS) (Gibco, USA), 1% L-glutamine (Gibco, USA), and 1% penicillin/streptomycin (Sigma, USA) in a humidified atmosphere containing 95% air and 5% CO<sub>2</sub> at 37°C. For <i>in vitro</i> virus infection, approximately 1×10<sup>5</sup> HUVECs per well were plated in 6-well plates and incubated at 37°C under 5% CO2. When HUVECs completely adhered to the 6-well plates, they were infected EV71 (sub-genotype C4, GenBank: EU812515.1) or CA16-G20 strain (sub-genotype B, GenBank: JN590244.1), which were isolated from an epidemic in Fuyang, China in 2008 and from an HFMD patient in Guangxi, China in 2010, respectively. The inoculation quantity of EV71 and CA16 is strictly controlled in lgCCID50 = 5/100μl, that is, the multiplicity of infection (MOI) of EV71 and CA16 is 1. The virus titers of EV71 and CA16 were previously examined in Vero cells and then stored -80℃ in our lab. The cells were collected at 0, 72, and 96 hours post infection (hpi), resuspended in Trizol (TIANGEN, China) and stored at -80°C until used. Cells infected with EV71 and CA16 for 0 hpi were used as controls. Simultaneously, we defined the following experimental groups: EV71-0h, EV71-72h, EV71-96h, CA16-0h, CA16-72h, and CA16-96h. Additionally, the EV71-0h and CA16-0h groups were used for normalization (the normalization value was set to 1), and they were designated as the control. Each group was processed in triplicate and pooled separately for subsequent total RNA extraction.</p> </div> <div class="rnaextractionandqualitycontrol" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction and quality control</div> <p>Samples from the previously mentioned groups were sent to the National Engineering Center for Biochips in Shanghai. miRNA was isolated from each group using a mirVana<sup>TM</sup> miRNA Isolation Kit (Ambion, USA) in accordance with the manufacturer’s guidelines. RNA purification was carried out after RNA isolation using a miRNeasy Mini kit (Qiagen, Germany) following the manufacturer’s instructions. The quality and integrity of the purified RNA were determined by capillary electrophoresis on an Agilent 2100 Bioanalyzer (Agilent Technologies, USA); only samples with an RNA integrity number (RIN) value greater than 7 were retained for miRNA profile analysis.</p> </div> <div class="smallrnalibraryconstruction,high-throughputsequencing,andmirna-seqdataanalysis" title="sec"> <div class="title" tagx="title" title="title">
SmallRNA library construction, high-throughput sequencing, and miRNA-seq data analysis</div> <p>High-throughput sequencing, which possesses lower background noise and higher sensitivity than microarrays, is a powerful tool for transcriptome and noncoding RNA studies, as it can capture almost all expressed known and novel transcripts[<a href="#pone.0177657.ref028">28</a>]. Sequencing libraries of the previously mentioned samples were generated using a standard TruSeq Small RNA sample preparation kit (Illumina, USA) according to manufacturer recommendations and then prepared for Illumina sequencing. After Illumina sequencing, the sequence reads went through the data cleaning procedure as described earlier: (1) filtering of low-quality sequences, namely, those with undetermined nucleotides (Ns), a quality score (Q-score) less than 10, or reads shorter than 18 nt using fastx_toolkit-0.0.13.2[<a href="#pone.0177657.ref029">29</a>]; (2) removal of the adapter sequences; (3) elimination of noncoding RNAs (e.g., rRNA, tRNA, snoRNA, or snRNA) and repetitive RNAs by mapping clean reads of small RNAs (sRNAs) using Rfam (<a href="http://rfam.janelia.org/">http://rfam.janelia.org/</a>) and piRNA (<a href="http://pirnabank.ibab.ac.in/">http://pirnabank.ibab.ac.in/</a>). The remaining short reads were then mapped to the known miRNA precursors, and the mature miRNAs were deposited in miRBase 19.0 (available online: <a href="http://www.mirbase.org/">http://www.mirbase.org/</a>). It is noteworthy that although high-throughput sequencing found novel sequences of miRNAs, we focused mainly on known miRNAs in this study; novel miRNAs have not recently been identified. Finally, 18- to 35-nt lengths of known and novel miRNAs were selected for further analysis. The sequencing data were deposited in the National Center for Biotechnology Information’s Gene Expression Omnibus (GEO) database (<a href="http://www.ncbi.nlm.nih.gov/geo/">www.ncbi.nlm.nih.gov/geo/</a>) and are accessible at series number GSE94551.</p> </div> <div class="bioinformaticsanalysis" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatics analysis</div> <p>The bioinformatic analysis of the reads aligned to known <i>Homo sapiens</i> pre-miRNA in miRBase 19 was performed as previously described[<a href="#pone.0177657.ref020">20</a>, <a href="#pone.0177657.ref021">21</a>]. In brief, 7 main aspects, including principal component analysis (PCA), differential expression analysis of miRNA sequences, unsupervised hierarchical clustering, trend analysis, target prediction, and functional enrichment of differentially expressed miRNAs, construction of regulatory networks and hierarchical GO category analysis of deregulated miRNA-associated target genes with key GO-biological process categories were systematically analyzed in this study utilizing the corresponding software. The details of this process are shown in <a href="#pone.0177657.g001">Fig 1</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0177657.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
The analysis flow chart of small-RNA sequencing information.</div>   </div> </div> <div class="confirmationofdifferentiallyexpressedmirnasandmirnatargetexpressionbyquantitativert-pcr(qrt-pcr)" title="sec"> <div class="title" tagx="title" title="title">
Confirmation of differentially expressed miRNAs and miRNA target expression by quantitative RT-PCR (qRT-PCR)</div> <p>High-throughput sequencing of differentially expressed miRNAs was further verified for 8 randomly selected miRNAs, including 7 known miRNAs, by qRT-PCR using extracted miRNA samples as templates. Additionally, based on an analysis of the predicted targets of the differentially expressed miRNAs, we also randomly chose 20 targets for indirect corroboration of miRNA expression by qRT-PCR. The protocols for qRT-PCR of miRNAs and mRNAs according to the respective manufacturer’s instructions were used as described in our previous study. Relative miRNA and mRNA abundance were determined by normalizing the miRNA of interest to U6 small nuclear RNA (snRNA) and β-actin mRNA, respectively. Then their relative expression was calculated in terms of threshold cycle value (CT) using the 2<sup>-ΔΔCt</sup> method[<a href="#pone.0177657.ref030">30</a>]. Primers were synthesized by Life Technologies (Life Technologies, USA); the names of the genes and their primers are listed in <a href="#pone.0177657.s001">S1</a> and <a href="#pone.0177657.s002">S2</a> Tables. The experiments for qRT-PCR detection were carried out at least 3 times.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p>For the sequencing data, the raw reads obtained from each library were normalized to the number of transcripts per million clean tags (TPMs). For qRT-PCR, the data are presented as the mean ± standard error of the mean (SEM). Statistical analysis was performed using SPSS 18.0 (IBM SPSS, USA). Differences were considered statistically significant for all analyses if <i>P</i>&amp;lt;0.05.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="comprehensiveoverviewofmirnasequencingdatainhuvecswithinfectionofev71andca16" title="sec"> <div class="title" tagx="title" title="title">
Comprehensive overview of miRNA sequencing data in HUVECs with infection of EV71 and CA16</div> <p>In order to evaluate the effects of EV71 and CA16 infection on miRNA expression in HUVECs, 6 small RNA (sRNA) libraries were constructed and submitted to Illumina sequencing. The details of the resultant sequencing data generation for the subsequent analysis are shown in <a href="#pone.0177657.t001">Table 1</a>. The read classification of different sRNAs types-including miRNA, miscellaneous RNA (misc_RNA), pseudogenes, ribosomal RNA (rRNA), small nucleolar RNA (snoRNA), and snRNA were identified in the samples (<a href="#pone.0177657.s003">S1 Fig</a>), indicating that sRNAs showed dynamic changes in both the EV71- and CA16-infected samples. Additionally, in all libraries, the majority of the sRNAs were 20 to 24nt long, with 22 nt being most abundant, as is typical for Dicer-derived products. Thereafter, the discrete degree of each sample group was examined using PCA (<a href="#pone.0177657.g002">Fig 2A</a>). The results revealed the distribution and clustering of the individual group samples. The Con group and infected groups formed tightly separate clusters, whereas the EV71- and CA16-infected groups were grouped closely together. These results indicate that there was a notable disparity between the EV71-infected and CA16-infected groups.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0177657.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Details of all miRNAs.</div> <p>(A) Principal component analysis (PCA) showing the distribution and clustering of the individual sample groups. Each spot represents a single array. EV71/CA16-infected samples were distinct from the Con sample. (B) Venn diagram representing Known differentially expressed miRNAs.(C) Time- and strain-specific regulation of differential miRNAs during EV71 and CA16 infections. The columns correspond to expression patterns of differentially expressed miRNAs during the EV71 and CA16 infections relative to Con samples at 6 hpi and 12 hpi. Significance was determined using a fold-change threshold of at least 2 and a P value cutoff of 0.05. The intensity of the miRNA expression is indicated in green (lower level of expression) and red (higher level of expression). Dendrograms between samples and between miRNAs are depicted, where the closest branches of the tree represents samples/miRNAs with the most similar expression pattern.</p>   </div> <div class="table-wrap_UNKNOWN" id="pone.0177657.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0177657.t001">10.1371/journal.pone.0177657.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Overview of small-RNA sequencing information and subsequent data analysis.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0177657.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="1" colspan="1">Sample</th> <th align="left" rowspan="1" colspan="1">Raw reads</th> <th align="left" rowspan="1" colspan="1">Clean reads</th> <th align="left" rowspan="1" colspan="1">Adapter-trimmed reads (length &amp;gt; = 18nt)</th> <th align="left" rowspan="1" colspan="1">Reads aligned to known Human pre-miRNA in miRBase 19</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">EV71-0h</td> <td align="right" rowspan="1" colspan="1">19,633,904</td> <td align="right" rowspan="1" colspan="1">19,399,383</td> <td align="left" rowspan="1" colspan="1">19,013,040</td> <td align="left" rowspan="1" colspan="1">381,861</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">EV71-72h</td> <td align="right" rowspan="1" colspan="1">19,306,634</td> <td align="right" rowspan="1" colspan="1">19,179,557</td> <td align="left" rowspan="1" colspan="1">18,852,827</td> <td align="left" rowspan="1" colspan="1">575,475</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">EV71-96h</td> <td align="right" rowspan="1" colspan="1">20,086,058</td> <td align="right" rowspan="1" colspan="1">19,936,799</td> <td align="left" rowspan="1" colspan="1">19,670,311</td> <td align="left" rowspan="1" colspan="1">619,050</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">CA16-0h</td> <td align="right" rowspan="1" colspan="1">19,076,315</td> <td align="right" rowspan="1" colspan="1">18,857,479</td> <td align="left" rowspan="1" colspan="1">17,538,072</td> <td align="left" rowspan="1" colspan="1">324,901</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">CA16-72h</td> <td align="right" rowspan="1" colspan="1">20,946,311</td> <td align="right" rowspan="1" colspan="1">20,786,178</td> <td align="left" rowspan="1" colspan="1">20,585,439</td> <td align="left" rowspan="1" colspan="1">380,393</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">CA16-96h</td> <td align="right" rowspan="1" colspan="1">20,218,869</td> <td align="right" rowspan="1" colspan="1">20,046,884</td> <td align="left" rowspan="1" colspan="1">19,829,142</td> <td align="left" rowspan="1" colspan="1">647,270</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Indicated from left to right are the numbers of reads that raw sequencing data, passed quality filtering (clean reads), the numbers of reads that passed both quality filtering, adapter filtering and length filtering (adapter-trimmed reads ≥ 18nt), and the number of reads that could be aligned to known Homo sapiens pre-miRNA in miRBase 19 with perfect matches, respectively.</p> </div> </div> </div> </div> <div class="differentialmirnaexpressionprofilesassociatedwithev71andca16infectionsinhuvecs" title="sec"> <div class="title" tagx="title" title="title">
Differential miRNA expression profiles associated with EV71 and CA16 infections in HUVECs</div> <p>To investigate the changes in miRNA expression profiles in HUVECs during EV71 and CA16 infection, all the mappable sRNA sequences were first compared with the currently known <i>Homo sapiens</i> miRNAs in the miRBase 19 database; then these miRNA expression patterns following infection were compared with those of the controls. In this study, only those known differentially expressed miRNAs with a threshold <i>P</i> value &amp;lt;0.05 and a minimum absolute fold difference of 2.0 between the control and infected data sets are described. As seen in <a href="#pone.0177657.t002">Table 2</a>, after 72 hpi with EV71 treatment, 34 miRNAs were significantly changed, with 14 being upregulated and 20 downregulated. After 96 hpi with EV71 treatment, 39 miRNAs were remarkably changed, with 14 and 25 being up- and downregulated, respectively. After 72 hpi with CA16 challenge, 64 miRNAs were notably altered; 12 miRNAs were upregulated and 52 miRNAs were downregulated. After 96 hpi with CA16 challenge, 92 miRNAs were obviously altered; 36 miRNAs were upregulated and 56 miRNAs were downregulated.</p> <div class="table-wrap_UNKNOWN" id="pone.0177657.t002" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0177657.t002">10.1371/journal.pone.0177657.t002</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Significantly differentially expressed miRNAs during the course of EV71 and CA16 infection.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0177657.t002g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="2" colspan="1">Comparison</th> <th align="left" colspan="3" rowspan="1">Known differentially expressed miRNAs</th> <th align="left" rowspan="2" colspan="1">Rate of differential expression genes (%)</th> </tr> <tr> <th align="left" rowspan="1" colspan="1">Total</th> <th align="left" rowspan="1" colspan="1">Up</th> <th align="left" rowspan="1" colspan="1">Down</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">EV71-72h vs. EV71-0h</td> <td align="left" rowspan="1" colspan="1">34</td> <td align="left" rowspan="1" colspan="1">14</td> <td align="left" rowspan="1" colspan="1">20</td> <td align="left" rowspan="1" colspan="1">0.0059</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">EV71-96h vs. EV71-0h</td> <td align="left" rowspan="1" colspan="1">39</td> <td align="left" rowspan="1" colspan="1">14</td> <td align="left" rowspan="1" colspan="1">25</td> <td align="left" rowspan="1" colspan="1">0.0062</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">CA16-72h vs. CA16-0h</td> <td align="left" rowspan="1" colspan="1">64</td> <td align="left" rowspan="1" colspan="1">12</td> <td align="left" rowspan="1" colspan="1">52</td> <td align="left" rowspan="1" colspan="1">0.0168</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">CA16-96h vs. CA16-0h</td> <td align="left" rowspan="1" colspan="1">92</td> <td align="left" rowspan="1" colspan="1">36</td> <td align="left" rowspan="1" colspan="1">56</td> <td align="left" rowspan="1" colspan="1">0.0142</td> </tr> </tbody> </table> </div> </div> <p>Subsequently, the data for all differentially expressed miRNAs were graphed into a Venn diagram (<a href="#pone.0177657.g002">Fig 2B</a>). Common and distinct differentially expressed known miRNAs in response to EV71 and CA16 infection across all time points were displayed. Simultaneously, a total of 135 known differentially expressed miRNAs in all groups were submitted to unsupervised hierarchical clustering to construct a heat map based on the differential expression patterns with log 2 values (infected/control) and fold changes (<a href="#pone.0177657.g002">Fig 2C</a>). A positive log 2 value indicated upregulation and a negative log 2 value indicated downregulation. The heat map exhibited marked discrepancies for the same strain at different times as well as for different strains at the same times following infection. Moreover, the hierarchical clusters of groups were in accordance with the result of PCA. Therefore, these results elucidated that known miRNA expression patterns following infection of EV71 and CA16 are strain- and time-specific.</p> <p>Ultimately, to verify the expression patterns of the known miRNAs, 8 differentially expressed miRNAs, namely, miRNA-3196, miRNA-5708, miRNA-4286, miRNA-5095, miRNA-204-5p, miRNA-1972, miRNA-4448 and miRNA-4531 were randomly selected from the high-throughput sequencing for qRT-PCR analysis. Overall, except for miRNA-204-5p, most of the qRT-PCR results of other miRNAs were quite consistent with the results from sequencing data (<a href="#pone.0177657.g003">Fig 3A</a>). Therefore, these results reinforced the reliability of the sequencing data and we concluded that the difference of miRNA-204-5p was primarily due to the limitations associated with the use of qRT-PCR technology.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0177657.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
qRT-PCR validation of differentially expressed miRNAs and partial predicted target genes.</div> <p>(A) Eight significantly differentially expressed miRNAs were chosen to perform qRT-PCR validation. Data from qRT-PCR analysis are shown as mean ± SEM (N = 3). N, Times of the experiment was performed. Expression changes are in the same direction as determined by the miRNAs sequencing. (B) Six predicted target genes were detected. The data were calculated as mean ± SEM from triplicates with normalization by β-actin.</p>   </div> </div> <div class="trendanalysisofthedifferentiallyexpressedmirnas" title="sec"> <div class="title" tagx="title" title="title">
Trend analysis of the differentially expressed miRNAs</div> <p>Based on previous studies which revealed that there were remarkable differences between EV71 and CA16 infections from clinical observation and experiment data of animal models, we hypothesized that the reasons probably depended on the fact that the host miRNAs induced by CA16 infection might be distinct from those that are induced by EV71 infection. Furthermore, although EV71 and CA16 infections stimulate many common miRNAs, some of the induced miRNAs show opposite trends between the viruses. Trend analysis of miRNA expression dynamics was used to profile the differentially expressed miRNAs and to identify the most probable set of clusters generated in the time series. As illustrated in <a href="#pone.0177657.g004">Fig 4</a>, we observed that 30 known differentially expressed miRNAs were all separately and gradually increased and decreased in EV71 and CA16 infection in the time series, indirectly suggesting that they may greatly contribute to the key distinctions induced by the two viruses. To further narrow down the predominant differentially expressed miRNAs, we defined a screening criterion as follows: if the 2 ratios (EV71-72h/CA16-72h and EV71-96h/CA16-96h) of one of the 30 known differentially expressed miRNAs fell between 0.5 and 2, it was eliminated (<a href="#pone.0177657.t003">Table 3</a>). After applying this criterion, only 27 known differentially expressed miRNAs were regarded as significant.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0177657.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
Trend analysis of differentially expressed miRNAs in response to EV71 and CA16 infection over time.</div> <p>The miRNAs that showed opposite expression patterns during the progression of EV71 and CA16 infection are shown in boxes.</p>   </div> <div class="table-wrap_UNKNOWN" id="pone.0177657.t003" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0177657.t003">10.1371/journal.pone.0177657.t003</a><span class="label" tagx="label" title="label">Table 3</span> <div class="title" tagx="title" title="title">
Oppositely expressed miRNAs during the course of EV71 and CA16 infection.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0177657.t003g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="1" colspan="1">miRNAs</th> <th align="left" rowspan="1" colspan="1">EV71-72h (FC)</th> <th align="left" rowspan="1" colspan="1">CA16-96h (FC)</th> <th align="left" rowspan="1" colspan="1">Ratio (EV71-72h/ CA16-72h)</th> <th align="left" rowspan="1" colspan="1">EV71-96h (FC)</th> <th align="left" rowspan="1" colspan="1">CA16-96h (FC)</th> <th align="left" rowspan="1" colspan="1">Ratio (EV71-96h/ CA16-96h)</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">miR-1246</td> <td align="right" rowspan="1" colspan="1">3.38414545</td> <td align="right" rowspan="1" colspan="1">1.50487685</td> <td align="right" rowspan="1" colspan="1">2.248785638</td> <td align="right" rowspan="1" colspan="1">7.95736516</td> <td align="right" rowspan="1" colspan="1">3.63320466</td> <td align="right" rowspan="1" colspan="1">2.190178065</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">let-7e-3p</td> <td align="right" rowspan="1" colspan="1">8.6262531</td> <td align="right" rowspan="1" colspan="1">3.41647717</td> <td align="right" rowspan="1" colspan="1">2.52489704</td> <td align="right" rowspan="1" colspan="1">4.31795009</td> <td align="right" rowspan="1" colspan="1">2.50977954</td> <td align="right" rowspan="1" colspan="1">1.720449953</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-126-5p</td> <td align="right" rowspan="1" colspan="1">2.09121287</td> <td align="right" rowspan="1" colspan="1">0.89907294</td> <td align="right" rowspan="1" colspan="1">2.32596576</td> <td align="right" rowspan="1" colspan="1">2.75713263</td> <td align="right" rowspan="1" colspan="1">1.43982089</td> <td align="right" rowspan="1" colspan="1">1.914913617</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-4531</td> <td align="right" rowspan="1" colspan="1">3.4753899</td> <td align="right" rowspan="1" colspan="1">0.08122525</td> <td align="right" rowspan="1" colspan="1">42.78706307</td> <td align="right" rowspan="1" colspan="1">17.7691624</td> <td align="right" rowspan="1" colspan="1">1.00391181</td> <td align="right" rowspan="1" colspan="1">17.69992362</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-4516</td> <td align="right" rowspan="1" colspan="1">2.62407</td> <td align="right" rowspan="1" colspan="1">0.458006</td> <td align="right" rowspan="1" colspan="1">5.729335457</td> <td align="right" rowspan="1" colspan="1">4.54225918</td> <td align="right" rowspan="1" colspan="1">1.08393377</td> <td align="right" rowspan="1" colspan="1">4.190532029</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-4485-3p</td> <td align="right" rowspan="1" colspan="1">2.70527464</td> <td align="right" rowspan="1" colspan="1">0.3863873</td> <td align="right" rowspan="1" colspan="1">7.0014585</td> <td align="right" rowspan="1" colspan="1">4.22305008</td> <td align="right" rowspan="1" colspan="1">0.68122587</td> <td align="right" rowspan="1" colspan="1">6.199192139</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-542-3p</td> <td align="right" rowspan="1" colspan="1">6.6355793</td> <td align="right" rowspan="1" colspan="1">0.17082386</td> <td align="right" rowspan="1" colspan="1">38.84456997</td> <td align="right" rowspan="1" colspan="1">4.9348001</td> <td align="right" rowspan="1" colspan="1">1.20469418</td> <td align="right" rowspan="1" colspan="1">4.096309413</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-183-3p</td> <td align="right" rowspan="1" colspan="1">4.31312655</td> <td align="right" rowspan="1" colspan="1">1.70823859</td> <td align="right" rowspan="1" colspan="1">2.524897041</td> <td align="right" rowspan="1" colspan="1">3.39267507</td> <td align="right" rowspan="1" colspan="1">4.01564726</td> <td align="right" rowspan="1" colspan="1">0.844863816</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>miR-590-3p</b></td> <td align="right" rowspan="1" colspan="1"><b>3.31778965</b></td> <td align="right" rowspan="1" colspan="1"><b>2.39153402</b></td> <td align="right" rowspan="1" colspan="1"><b>1.387306066</b></td> <td align="right" rowspan="1" colspan="1"><b>4.16373758</b></td> <td align="right" rowspan="1" colspan="1"><b>2.91134426</b></td> <td align="right" rowspan="1" colspan="1"><b>1.430176993</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>miR-340-5p</b></td> <td align="right" rowspan="1" colspan="1"><b>2.23867019</b></td> <td align="right" rowspan="1" colspan="1"><b>1.29194515</b></td> <td align="right" rowspan="1" colspan="1"><b>1.73279043</b></td> <td align="right" rowspan="1" colspan="1"><b>2.41132278</b></td> <td align="right" rowspan="1" colspan="1"><b>1.68724675</b></td> <td align="right" rowspan="1" colspan="1"><b>1.429146495</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-32-5p</td> <td align="right" rowspan="1" colspan="1">6.6355793</td> <td align="right" rowspan="1" colspan="1">0.28470643</td> <td align="right" rowspan="1" colspan="1">23.3067419</td> <td align="right" rowspan="1" colspan="1">6.78535013</td> <td align="right" rowspan="1" colspan="1">1.33854909</td> <td align="right" rowspan="1" colspan="1">5.069182897</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-3196</td> <td align="right" rowspan="1" colspan="1">1.32711586</td> <td align="right" rowspan="1" colspan="1">0.10830033</td> <td align="right" rowspan="1" colspan="1">12.25403306</td> <td align="right" rowspan="1" colspan="1">11.7201502</td> <td align="right" rowspan="1" colspan="1">0.83659318</td> <td align="right" rowspan="1" colspan="1">14.00937818</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-3195</td> <td align="right" rowspan="1" colspan="1">2.32245276</td> <td align="right" rowspan="1" colspan="1">0.24403408</td> <td align="right" rowspan="1" colspan="1">9.516919596</td> <td align="right" rowspan="1" colspan="1">8.01905016</td> <td align="right" rowspan="1" colspan="1">0.43024792</td> <td align="right" rowspan="1" colspan="1">18.63820783</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-7641</td> <td align="right" rowspan="1" colspan="1">3.31778965</td> <td align="right" rowspan="1" colspan="1">0.22476823</td> <td align="right" rowspan="1" colspan="1">14.76093655</td> <td align="right" rowspan="1" colspan="1">6.16850012</td> <td align="right" rowspan="1" colspan="1">0.26418732</td> <td align="right" rowspan="1" colspan="1">23.34896365</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-5708</td> <td align="right" rowspan="1" colspan="1">6.95077979</td> <td align="right" rowspan="1" colspan="1">0.17082386</td> <td align="right" rowspan="1" colspan="1">40.68974821</td> <td align="right" rowspan="1" colspan="1">8.07689201</td> <td align="right" rowspan="1" colspan="1">0.0649802</td> <td align="right" rowspan="1" colspan="1">124.2977401</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-548b-5p</td> <td align="right" rowspan="1" colspan="1">1.65889483</td> <td align="right" rowspan="1" colspan="1">0.85411929</td> <td align="right" rowspan="1" colspan="1">1.942228493</td> <td align="right" rowspan="1" colspan="1">2.15897504</td> <td align="right" rowspan="1" colspan="1">0.25097795</td> <td align="right" rowspan="1" colspan="1">8.602249767</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-4492</td> <td align="right" rowspan="1" colspan="1">1.82478431</td> <td align="right" rowspan="1" colspan="1">0.66431501</td> <td align="right" rowspan="1" colspan="1">2.746866011</td> <td align="right" rowspan="1" colspan="1">2.62161255</td> <td align="right" rowspan="1" colspan="1">0.44618303</td> <td align="right" rowspan="1" colspan="1">5.875643814</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-4286</td> <td align="right" rowspan="1" colspan="1">3.31778965</td> <td align="right" rowspan="1" colspan="1">0.28470643</td> <td align="right" rowspan="1" colspan="1">11.65337096</td> <td align="right" rowspan="1" colspan="1">4.31795008</td> <td align="right" rowspan="1" colspan="1">0.33463727</td> <td align="right" rowspan="1" colspan="1">12.90337465</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-424-3p</td> <td align="right" rowspan="1" colspan="1">1.16122638</td> <td align="right" rowspan="1" colspan="1">0.64058947</td> <td align="right" rowspan="1" colspan="1">1.812746593</td> <td align="right" rowspan="1" colspan="1">1.54212503</td> <td align="right" rowspan="1" colspan="1">0.37646693</td> <td align="right" rowspan="1" colspan="1">4.096309413</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-3182</td> <td align="right" rowspan="1" colspan="1">1.12804848</td> <td align="right" rowspan="1" colspan="1">0.07614242</td> <td align="right" rowspan="1" colspan="1">14.81498108</td> <td align="right" rowspan="1" colspan="1">0.97462302</td> <td align="right" rowspan="1" colspan="1">0.0564214</td> <td align="right" rowspan="1" colspan="1">17.27399554</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-1273f</td> <td align="right" rowspan="1" colspan="1">1.13752788</td> <td align="right" rowspan="1" colspan="1">0.09855223</td> <td align="right" rowspan="1" colspan="1">11.54238648</td> <td align="right" rowspan="1" colspan="1">0.61685001</td> <td align="right" rowspan="1" colspan="1">0.019306</td> <td align="right" rowspan="1" colspan="1">31.95121341</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-1261</td> <td align="right" rowspan="1" colspan="1">1.21652287</td> <td align="right" rowspan="1" colspan="1">0.04380099</td> <td align="right" rowspan="1" colspan="1">27.77386766</td> <td align="right" rowspan="1" colspan="1">1.23370002</td> <td align="right" rowspan="1" colspan="1">0.05148266</td> <td align="right" rowspan="1" colspan="1">23.96341006</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-1244</td> <td align="right" rowspan="1" colspan="1">0.66355793</td> <td align="right" rowspan="1" colspan="1">0.64058947</td> <td align="right" rowspan="1" colspan="1">1.035855196</td> <td align="right" rowspan="1" colspan="1">2.46740005</td> <td align="right" rowspan="1" colspan="1">0.08122525</td> <td align="right" rowspan="1" colspan="1">30.37725397</td> </tr> <tr> <td align="left" rowspan="1" colspan="1"><b>miR-100-5p</b></td> <td align="right" rowspan="1" colspan="1"><b>1.76771833</b></td> <td align="right" rowspan="1" colspan="1"><b>1.48347035</b></td> <td align="right" rowspan="1" colspan="1"><b>1.19161015</b></td> <td align="right" rowspan="1" colspan="1"><b>2.01833324</b></td> <td align="right" rowspan="1" colspan="1"><b>1.50058398</b></td> <td align="right" rowspan="1" colspan="1"><b>1.34503185</b></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-7704</td> <td align="right" rowspan="1" colspan="1">1.14131964</td> <td align="right" rowspan="1" colspan="1">0.353229</td> <td align="right" rowspan="1" colspan="1">3.231104057</td> <td align="right" rowspan="1" colspan="1">1.73951703</td> <td align="right" rowspan="1" colspan="1">0.44240182</td> <td align="right" rowspan="1" colspan="1">3.931984385</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-378i</td> <td align="right" rowspan="1" colspan="1">1.59253903</td> <td align="right" rowspan="1" colspan="1">0.48806817</td> <td align="right" rowspan="1" colspan="1">3.262943869</td> <td align="right" rowspan="1" colspan="1">1.97392004</td> <td align="right" rowspan="1" colspan="1">1.21903577</td> <td align="right" rowspan="1" colspan="1">1.619247016</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">let-7i-3p</td> <td align="right" rowspan="1" colspan="1">2.65423172</td> <td align="right" rowspan="1" colspan="1">0.17082386</td> <td align="right" rowspan="1" colspan="1">15.53782799</td> <td align="right" rowspan="1" colspan="1">2.77582506</td> <td align="right" rowspan="1" colspan="1">1.20469418</td> <td align="right" rowspan="1" colspan="1">2.304174045</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-455-3p</td> <td align="right" rowspan="1" colspan="1">1.76948781</td> <td align="right" rowspan="1" colspan="1">0.59131336</td> <td align="right" rowspan="1" colspan="1">2.992470568</td> <td align="right" rowspan="1" colspan="1">2.46740005</td> <td align="right" rowspan="1" colspan="1">1.35141975</td> <td align="right" rowspan="1" colspan="1">1.825783623</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-374b-3p</td> <td align="right" rowspan="1" colspan="1">1.99067379</td> <td align="right" rowspan="1" colspan="1">0.04641443</td> <td align="right" rowspan="1" colspan="1">42.88911557</td> <td align="right" rowspan="1" colspan="1">0.82246668</td> <td align="right" rowspan="1" colspan="1">0.21512396</td> <td align="right" rowspan="1" colspan="1">3.823222118</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">miR-3159</td> <td align="right" rowspan="1" colspan="1">1.73769495</td> <td align="right" rowspan="1" colspan="1">0.08122525</td> <td align="right" rowspan="1" colspan="1">21.39353154</td> <td align="right" rowspan="1" colspan="1">3.23075681</td> <td align="right" rowspan="1" colspan="1">0.37646693</td> <td align="right" rowspan="1" colspan="1">8.581781146</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Three miRNAs eliminated are highlighted in bold. FC, fold change.</p> </div> </div> </div> </div> <div class="geneontology(go)andpathwayanalysisofpredictedtargetsofdifferentiallyexpressedmirnas" title="sec"> <div class="title" tagx="title" title="title">
Gene ontology (GO) and pathway analysis of predicted targets of differentially expressed miRNAs</div> <p>To gain insight into the possible regulatory roles of the 27 known differentially expressed miRNAs in EV71 and CA16 infections, we used the TargetScan and miRDB programs to predict the potential targets of these miRNAs[<a href="#pone.0177657.ref031">31</a>, <a href="#pone.0177657.ref032">32</a>]. We found that 2,305 genes were predicted to be potential targets with TargetScan, whereas 3,449 genes were predicted to be potential targets with miRDB (<a href="#pone.0177657.s004">S2 Fig</a>). Then GO was introduced into the analysis to reveal functions significantly associated with the predicted target gene candidates of the altered miRNAs. Predicted target functions were classified into 3 core categories: biological process (<a href="#pone.0177657.g005">Fig 5A</a>), molecular function (<a href="#pone.0177657.g005">Fig 5B</a>), and cellular component (<a href="#pone.0177657.g005">Fig 5C</a>). Biological process included ontology rhythmic process, response to stimulus, reproduction, multicellular organismal process, metabolic process, locomotion, localization, immune system process, developmental process, cellular process, cellular component organization or biogenesis, cell killing, biological regulation, and biological adhesion. The molecular function category included transporter activity, translation regulator activity, structural molecule activity, signal transducer activity, receptor activity, channel regulator activity, catalytic activity, binding, and antioxidant activity. The cellular component category included synapse, organelle, membrane, macromolecular complex, extracellular region, extracellular matrix, cell part, and cell junction. Afterward, the predicted targets genes were subjected to the KEGG pathway enrichment analysis using the PANTHER Classification System. As shown in <a href="#pone.0177657.g005">Fig 5D</a>, 85 pathways were identified, suggesting that these signaling pathways are regulated by the differentially expressed miRNAs during viral infection. Among these pathways, many were intimately correlated with immune regulation (e.g., the apoptosis signaling pathway, B-cell activation, interferon-gamma signaling pathway, interleukin signaling pathway, Toll receptor signaling, T-cell activation, etc.), adhesion signaling (e.g., cadherin signaling pathway, cytoskeletal regulation by Rho GTPase, Wnt signaling pathway, etc.) and nervous system function (e.g., axon guidance mediated by netrin, synaptic vesicle trafficking, etc.), which suggested that these alterations involved different immune responses, cell adhesion, and neurological symptoms induced by EV71 or CA16 infection.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0177657.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Gene ontology enrichment terms and KEGG pathway analysis of the predicted targets of differentially expressed miRNAs.</div> <p>The DAVID web-based tool was used to analyze (A) biological process, (B) molecular function and (C) cellular component. GO functional enrichment annotations exhibit in pie charts of the left panel. Bars indicate the number of GOs annotated as unique GO terms and are presented in the right panel. (D) There are 85 pathways in total, as depicted in the pie chart.</p>   </div> </div> <div class="complexregulationnetworkconstructionofmirna-targetsandthemirna-goandmirna-pathways" title="sec"> <div class="title" tagx="title" title="title">
Complex regulation network construction of miRNA-targets and the miRNA-GO and miRNA-pathways</div> <p>To gain a deeper understanding of the regulatory functions of miRNAs, we built miRNA regulatory networks to elaborate the interactions among miRNAs and genes, GOs, and pathways. First, to confirm the key target genes, we looked for intersections of related genes in the results of our GO and pathway analyses. A total of 91 common target genes were determined by Venn diagram; these are listed in <a href="#pone.0177657.s005">S3 Fig</a>. Next, 26 known differentially expressed miRNAs, 11 GOs involved in biological processes, and 81 pathways were identified by refining the correlations among the 91 target genes and the miRNAs, GOs, and pathways. Consequently, by integrating these correlations, interaction networks between miRNAs and targets (<a href="#pone.0177657.g006">Fig 6A</a>), miRNAs and GOs (<a href="#pone.0177657.g006">Fig 6B</a>), and miRNAs and pathways (<a href="#pone.0177657.g006">Fig 6C</a>) were established. Among the 3 networks, 1 miRNA often regulates more than 1 of the target genes, GOs, and pathways, whereas a target gene, GO, or pathway is also often regulated by more than 1 miRNA.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0177657.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
Complexity of the miRNA-regulation networks after infection with EV71 and CA16.</div> <p>miRNAs are depicted as a red colored node at (A), (B) and (C). (A) miRNA-targets network. The targets are displayed by light blue color rounded rectangles. (B) miRNA-GOs network. GO term nodes are depicted as green color rounded rectangles. Edges in the network represent inhibitory effect of miRNAs on GOs. (C) miRNA-pathways network. Purple rectangle-shaped nodes denote pathways. Edges indicate a negative correlation between miRNA and pathways. (D) Co-expression network were constructed by adhesion-related targets. Red nodes are target genes and yellow nodes are co-expression genes. Genes with bigger size are more centralized in the network and have a stronger capacity of modulating adjacent genes. Different color lines mean the different interactions between these genes. (E) Hierarchical GO categories of genes involved in adhesions. A <i>P</i> value cutoff of 0.05 was used to determine significantly enriched GOs. The color gradient of the yellow nodes, from light to dark, displays the P value, from high to low (the lower the P value, the greater the GO significance level). White nodes are no significant.</p>   </div> </div> <div class="co-expressionnetworkanalysis,hierarchicalgocategoryanalysis,anddetectionoftargetgenesinvolvedinkeybiologicalprocesses" title="sec"> <div class="title" tagx="title" title="title">
Co-expression network analysis, hierarchical GO category analysis, and detection of target genes involved in key biological processes</div> <p>In this part, we specifically looked into the hierarchical GO categories of the deregulated miRNA-associated targets with the biological adhesion derived from the previously described biological process analysis. In this biological process category produced by the GO analysis, 8 known differentially expressed miRNAs and 6 targets were found to participate in biological adhesion (<a href="#pone.0177657.t004">Table 4</a>). Moreover, these targets were used to build a co-expression network (<a href="#pone.0177657.g006">Fig 6D</a>), which contained adhesion molecules (e.g., PVRL3, PVRL4, and CTNNB1, etc.), nervous regulation molecules that participated in axon extension and cell migration during neural development (e.g., UNC5C and UNC5A), and cellular structure formation–related molecules involved in the establishment of cell polarity (e.g., PARD3), among others. These targets and co-expression genes may provide new information regarding the underlying distinct mechanisms caused by EV71 and CA16 infection. In addition, in order to investigate the inner functions of the genes involved in our gene network on the ontology level, these targets were also submitted to a hierarchical GO categories analysis. The results of this pointed to statistically significantly enriched GO terms among the given gene list based on their hypergeometric probability. In <a href="#pone.0177657.g006">Fig 6E</a>, the 6 targets are seen to be significantly enriched in 5 key biological processes, including cell proliferation (e.g., epithelial cell proliferation), cell adhesion (e.g., cell-cell adhesion, actin cytoskeleton reorganization, etc.), signal transduction (e.g., small GTPase–mediated signal transduction and Ras protein signal transduction), nervous regulation (e.g., regulation of synapse organization, regulation of nervous system development, etc.), and interplay between host and virus (e.g., initiation of viral infection, entry into host cell, etc.). Hence, according to these findings, we proposed that 8 known differentially expressed miRNAs may indirectly mediate these GOs, which might harbor significance and/or contribute to the pathogenesis and biochemical characteristics of HFMD patients afflicted with EV71 or CA16 infection.</p> <div class="table-wrap_UNKNOWN" id="pone.0177657.t004" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0177657.t004">10.1371/journal.pone.0177657.t004</a><span class="label" tagx="label" title="label">Table 4</span> <div class="title" tagx="title" title="title">
Biological adhesion process related to miRNAs and their target genes.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pone.0177657.t004g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="left" rowspan="1" colspan="1">miRNAs</th> <th align="left" rowspan="1" colspan="1">Target genes</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">let-7i-3p, miR-3195, miR-32-5p, miR-4492<br /><br />miR-4516, miR-548b-5p, miR-5708, miR-7704</td> <td align="left" rowspan="1" colspan="1">RAF1, NTN1, PVRL1<br /><br />COL8A2, RALA, LEF1</td> </tr> </tbody> </table> </div> </div> <p>Subsequently, we chose the 6 targets to further measure and confirm the miRNA sequencing data. In general, the expression levels of these miRNAs and their targets presented an inverse relationship based on the regulatory mechanisms of the miRNAs. Except for NTN1, the expression levels of target genes were inversely correlated with the expression patterns of the miRNAs regulating these target genes (<a href="#pone.0177657.g003">Fig 3B</a>).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>EV71 and CA16, as the main pathogens responsible for worldwide HFMD outbreaks, can cause severe neurological complications and death[<a href="#pone.0177657.ref001">1</a>]. Although 3 inactivated EV71 vaccines have already shown substantial protective effect against EV71-HFMD and EV71-related diseases in clinical trials, the current lack of an effective vaccine to treat HFMD induced by other enteroviruses, including CA16, highlights the importance of developing vaccines to prevent the transmission of the other causative agents[<a href="#pone.0177657.ref009">9</a>]. Thus, further investigations identifying the factors that cause the different manifestations of EV71 and CA16 infection are still urgently needed and may provide new strategies for the development of more efficacious vaccines to prevent HFMD outbreaks. In the current study, we identified a total of 135 differentially expressed miRNAs across different time points following EV71 and CA16 infection; of these, both common and unique miRNAs were identified, indicating that miRNA expression in the context of these viruses has strain- and time-specific tendencies. Moreover, since the 6 common miRNA, namely, miR-7851-3p, miR-619-5p, miR-5585-3p, miR-5096, miR-204-5p, and miR-1285-5p simultaneously participated in both EV71 and CA16 interactions with host cells. It is possible that 1 or 2 of them could serve as potential candidates for the antiviral therapy of HFMD. It has recently been estimated that about 60% of human genes may be subjected to miRNA regulation through the degradation of target mRNAs or the suppression of mRNA translation following specific binding to target mRNA, and the roles of miRNAs in pathogen-host interactions have received increasing attention[<a href="#pone.0177657.ref014">14</a>, <a href="#pone.0177657.ref017">17</a>]. A growing body of reports indicates that viral infections can remodel cellular miRNA expression, which in turn influences cellular microenvironment and metabolism and eventually leads to viral pathogenesis[<a href="#pone.0177657.ref033">33</a>]. For example, the influenza virus notably changes cellular miR-106b, miR-124, and miR-1254 expression in favor of influenza replication by regulating human protease genes (such as ADAMTS7, CPE, DPP3, MST1, and PRSS12), which can cause clinical outcomes ranging from mild upper respiratory infections to severe pneumonia[<a href="#pone.0177657.ref034">34</a>]; respiratory syncytial virus (RSV) obviously enhances let-7f expression in the host, which likely contributes to delayed viral clearance and inhibits let-7i and miR-30b expression, thus promoting viral replication[<a href="#pone.0177657.ref035">35</a>]. Moreover, these altered miRNAs appear to exacerbate the severity of disease. Hence it is no surprise that abnormal miRNA expression is intimately associated with the development and progression of various infectious diseases[<a href="#pone.0177657.ref019">19</a>]. Additionally, evidence from different researchers is accumulating that some miRNAs are involved in the infectious cycle of CA16 or EV71; for instance, 3 key miRNAs (miR-545, miR-324-3p, and miR-143) can be used to distinguish EV71 infection from CA16 infection in patients with HFMD[<a href="#pone.0177657.ref036">36</a>]. Furthermore, elevated expression of the circulating miR-876-5p is a specific response to severe EV71 infections[<a href="#pone.0177657.ref037">37</a>], and miR-432* regulates the replication of CA16 in RD cells[<a href="#pone.0177657.ref038">38</a>]. Another study has reported a dramatic reduction in EV71 replication when DGCR8 (an essential cofactor for miRNA biogenesis) was knocked down prior to EV71 infection, suggesting that EV71 might utilize the host’s miRNAs to enhance viral replication[<a href="#pone.0177657.ref039">39</a>]. Collectively, our discovery of the specific miRNA expression patterns that are produced during EV71 and CA16 infection in HUVECs provides powerful insights into the different mechanisms of viral pathogenesis regulated by miRNAs. Following, these findings further elucidate the key disparities between EV71 and CA16 infection.We further isolated 30 differentially expressed miRNAs that displayed opposite expression trends. The functions of some miRNAs among these 30 have been studied by other researchers. For example, miR-183-3p was validated as a potent prognostic marker for lung adenocarcinoma in female nonsmokers[<a href="#pone.0177657.ref040">40</a>], and declining expression of let-7e-3p was associated with cell cycle arrest and the inhibition of proliferation of cervical cancer cells[<a href="#pone.0177657.ref041">41</a>]. High endogenous and circulating miR-1246 modulated by the direct upstream regulator octamer 4 (Oct4) was shown to be a novel diagnostic marker as well as a tool for therapeutic intervention in hepatocellular carcinoma (HCC) by cooperatively driving Wnt/β-catenin signaling activation[<a href="#pone.0177657.ref042">42</a>]. In addition, miRNA-1246 was found to be a target gene of p53 in the progression of various cancers, including cervical, colorectal, esophageal, hepatic, and pancreatic cancers[<a href="#pone.0177657.ref043">43</a>–<a href="#pone.0177657.ref045">45</a>]. It was also found that the expression of miR-1246, which was increased in EV71-infected cells (in agreement with our findings), could suppress the expression of the disc-large homolog 3 (DLG3), known to contribute to neurological disorders[<a href="#pone.0177657.ref046">46</a>]. In addition, the miR-4492 was significantly upregulated in EV71-infected RD cells, consistent with our results[<a href="#pone.0177657.ref047">47</a>]. However, the specific roles of these differentially expressed miRNAs in our study of EV71 and CA16 infection remain largely unknown and need further exploration.</p> <p>In order to better understand the potential molecular mechanisms of these differentially expressed miRNAs that occur during EV71 and CA16 infection, we carried out GO and pathway analysis of the predicted target genes of the 30 differentially expressed miRNAs identified in trend analysis. Using KEGG pathway analysis and GO annotation for target genes can offer a comprehensive understanding of the target genes at th<b>e</b> pathway, molecular, cellular, and biological levels. The observed changes in biological processes (especially related to immune system processes and biological adhesion), molecular function (especially with reference to binding), cellular components (especially linked to cell junctions) and pathways (especially associated with immune pathways, such as apoptosis signaling pathways, the Toll-receptor signaling pathway, T-cell activation, interleukin signaling, inflammation mediated by chemokine and cytokine signaling, B cell activation, the JAK/STAT signaling pathway, etc., and adhesion pathways, such as the cadherin signaling pathway, cytoskeletal regulation by Rho GTPase, the Wnt signaling pathway, etc.) suggested that these differentially expressed miRNAs might play important roles in immune and adhesion responses in HUVECs infected with EV71 and CA16. Nevertheless, the success of pathogens in entering host cells largely depends on their invasion strategies, including impeding the activation of the host immune system, destroying intercellular junctions, triggering excess apoptosis, and so on[<a href="#pone.0177657.ref048">48</a>]. Furthermore, biochemical and genetic studies have clarified that miRNAs, as key regulators, participate in the regulation of immune function, such as antigen presentation (e.g., miR-155), T-cell receptor signalling (e.g., miR-181a), Toll-like receptor signalling and the ensuing cytokine responses (e.g., miR-146), etc.[<a href="#pone.0177657.ref049">49</a>], and cell-cell contact, such as urothelial permeability (e.g., miR-199a-5p), etc[<a href="#pone.0177657.ref050">50</a>]. Additionally, it has been demonstrated that EV71 infection can affect the p38 MAPK signaling pathway, which may be associated with the secretion of inflammatory cytokines and host-cell apoptosis; these mechanisms may be implicated in inflammation of the CNS and disorders such as encephalitis or meningitis[<a href="#pone.0177657.ref051">51</a>]. Similarly, in our study, the p38 MAPK signaling pathway was also changed by pathway enrichment analysis. Moreover, the predicted target genes were also enriched into specific neuronal cellular components, such as synapses, and neuronal function-related pathways, such as axon guidance mediated by Slit/Robo, netrin, or semaphorins, etc. Thus we hypothesized that the differing pathogeneses of EV71 and CA16 infections are partly attributable to the regulatory roles of the oppositely expressed miRNAs involved in the immune response, intercellular junctions, and neurological symptoms that might lead to the malfunction of host cells or favor the viral life cycle. Next, the associations of the identified miRNAs with target genes, GOs, pathways and co-expression genes were further visualized by creating regulatory networks.</p> <p>It is known that the BBB is composed of microvascular endothelial cells, pericytes, and astrocytes, comprisinga complex physical barrier through tight cell-cell interactions and having a critical role in protecting against invading pathogens and establishing mechanical strength[<a href="#pone.0177657.ref025">25</a>]. Once the integrity of the BBB is disrupted, CNS pathogenesis is facilitated through increasing viral invasion and spread. For example, West Nile virus (WNV) infection of the brain’s microvascular endothelial cells (HBMVEs) significantly increases leukocyte adhesion to the HBMVE monolayer and transmigration across the HBMVE, suggesting that WNV causes destruction of the BBB. This improves leukocyte infiltration into the brain and fosters neuroinflammation[<a href="#pone.0177657.ref027">27</a>]. Dengue virus (DENV) infection, for example, changes the localization of the tight junction proteins Zonula occludens (ZO-1) and Claudin-1 in mouse brain endothelial cells (MBECs), and this process is closely associated with a decrease in transendothelial resistance, an increase in macromolecule permeability, and an increase in the paracellular passing of free virus particles in MBECs, indicating that the elevated permeability of the BBB allowed the entry of virus into the nervous system and caused the neurological manifestations[<a href="#pone.0177657.ref052">52</a>]. EV71 and CA16 are also highly neurotropic viruses, potentially causing varying degrees of neurological complications[<a href="#pone.0177657.ref024">24</a>]. In the present study, therefore, we focused mainly on the biological adhesion results derived from the GO analysis to perform a more in-depth hierarchical GO category analysis. Our results demonstrate that the 8 differentially expressed miRNAs implicated in biological adhesion were consistently upregulated following EV71 infection and downregulated following CA16 infection. Furthermore, owing to the negative regulation that miRNAs exert on their target genes, it was proposed that an inverse correlation exists between the miRNAs and the corresponding GOs of their target genes. Notably, the functions of biological adhesion induced by CA16 infection were significantly activated, whereas they were inhibited by EV71 infection. These results of hierarchical GO category analysis showed that there are 5 significant differences, as follows: cell proliferation (e.g., epithelial cell proliferation), cell adhesion (e.g., cell-cell adhesion, actin cytoskeleton reorganization, etc.), signal transduction (e.g., small GTPase-mediated signal transduction and Ras protein signal transduction), nervous regulation (e.g., regulation of synaptic organization, regulation of nervous system development, etc.) and interplay between host and virus (e.g., initiation of viral infection, entry into the host cell, etc.).These effects as induced by EV71 infection are more susceptible to dysregulation than they are in the context of CA16 infection. Moreover, most of the target genes linked to biological adhesion for the 8 differentially expressed miRNAs described previously were declined during EV71 infection but overexpressed during CA16 infection, as validated by qRT-PCR. These results further demonstrate that the cell-cell contacts of HUVECs may be disturbed during EV71 infection, indicating impairment of the BBB. Taken together, we hypothesize that the neuropathological manifestations of EV17 and CA16 infection may be mediated by the 8 differentially expressed miRNAs. Additionally, in our previous study, miR-4516, involved in biological adhesion, was reduced and elevated in the EV71 and CA16 infection of human bronchial epithelial (16HBE) cells[<a href="#pone.0177657.ref020">20</a>]. According to the previous speculation, miR-4516 may be the key factor causing the destruction of 16HBE cells in CA16 infection and ultimately leading to the repeatedly occurring phenomenon of CA16 infection and its clinical features, which rarely happens following EV71 infection (unpublished data). Nevertheless, in this study, miR-4516, also discovered in biological adhesion, was increased and decreased in the EV71 and CA16 infection of HUVECs, respectively. Therefore it was postulated that the altered miR-4516 generated in the HUVECs subjected to EV71 and CA16 infections may initiate different degree impairment of HUVECs, thus creating a relatively permissive environment for viral invasion into the brain and contributing to the different outcomes of EV71 and CA16 infections in the CNS.</p> <p>In conclusion, for the first time, this study has demonstrated, by small RNA sequencing technology and bioinformatics approaches, that EV71 and CA16 infections result in specific miRNA expression patterns in HUVECs. More studies are needed to dissect the relevant roles of miRNAs in viral pathogenesis. GO and pathway analysis of the predicted targets of the differentially expressed miRNAs were screened from trend analysis and provided comprehensive information about the discrepant biological functions and pathways involved EV71 and CA16 infection. Furthermore, the construction of detailed regulatory networks alsorevealed a global perspective for exploring miRNA-mediated mechanisms. Consequently, it is worth noting that the 8 known differentially expressed miRNAs related to biological adhesion identified in this study are the key miRNAs that modulate cellular junctions in HUVECs and compromise the permeability barrier of HUVECs; these activities may play a role in the dissemination of virus in the CNS.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Primers used for qRT-PCR confirmation of miRNA sequencing.</div> <p>(DOCX)</p>  <div class="media" title="media"><a href="pone.0177657.s001.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
Sequences of target primers used for qRT-PCR assays.</div> <p>(DOCX)</p>  <div class="media" title="media"><a href="pone.0177657.s002.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Bar chart summarized the different classes of sRNAs in the samples.</div> <p>Different types of sRNAs, including miRNA, miscellaneous RNA (misc_RNA), pesudogene, ribosomal RNA (rRNA), small nucleolar RNA (snoRNA) and small nuclear RNA (snRNA), were identified in the samples.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0177657.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Venn diagram of the key target genes number in TargetScan analysis and miRDB analysis.</div> <p>The common putative targets between TargetScan and miRDB analysis are 731.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0177657.s004.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Map of overlapped target genes between GO and pathway analysis.</div> <p>91 target genes were observed and listed on a box.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0177657.s005.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>This work was supported CAMS Innovation Fund for Medical Sciences (2016-I2M-1-014 and 2016-I2M-1-019) and National Natural Sciences Foundations of China (81373142). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0177657.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koh</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bogich</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siegel</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chong</span><span class="given-names" tagx="given-names" title="given-names">EY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Epidemiology of Hand, Foot and Mouth Disease in Asia: A Systematic Review and Analysis</span>. <span class="source" tagx="source" title="source">Pediatr Infect Dis J</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">35</span>: <span class="fpage" tagx="fpage" title="fpage">e285</span>–<span class="lpage" tagx="lpage" title="lpage">300</span>. <span class="comment">doi: <a href="https://doi.org/10.1097/INF.0000000000001242">10.1097/INF.0000000000001242</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27273688">27273688</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Recent Progress towards Novel EV71 Anti-Therapeutics and Vaccines</span>. <span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">6441</span>–<span class="lpage" tagx="lpage" title="lpage">6457</span>. <span class="comment">doi: <a href="https://doi.org/10.3390/v7122949">10.3390/v7122949</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26670245">26670245</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bian</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Coxsackievirus A16: epidemiology, diagnosis, and vaccine</span>. <span class="source" tagx="source" title="source">Hum Vaccin Immunother</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">360</span>–<span class="lpage" tagx="lpage" title="lpage">367</span>. <span class="comment">doi: <a href="https://doi.org/10.4161/hv.27087">10.4161/hv.27087</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24231751">24231751</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lugo</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krogstad</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">Enteroviruses in the early 21st century: new manifestations and challenges</span>. <span class="source" tagx="source" title="source">Curr Opin Pediatr</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">107</span>–<span class="lpage" tagx="lpage" title="lpage">113</span>. <span class="comment">doi: <a href="https://doi.org/10.1097/MOP.0000000000000303">10.1097/MOP.0000000000000303</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26709690">26709690</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oberste</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alexander</span><span class="given-names" tagx="given-names" title="given-names">JP</span><span class="suffix" tagx="suffix" title="suffix">Jr.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kennett</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pallansch</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology and evolution of enterovirus 71 strains isolated from 1970 to 1998</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">1999</span>; <span class="volume" tagx="volume" title="volume">73</span>: <span class="fpage" tagx="fpage" title="fpage">9969</span>–<span class="lpage" tagx="lpage" title="lpage">9975</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10559310">10559310</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seah</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">VT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poh</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">EL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparative proteome analyses of host protein expression in response to Enterovirus 71 and Coxsackievirus A16 infections</span>. <span class="source" tagx="source" title="source">J Proteomics</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">74</span>: <span class="fpage" tagx="fpage" title="fpage">2018</span>–<span class="lpage" tagx="lpage" title="lpage">2024</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.jprot.2011.05.022">10.1016/j.jprot.2011.05.022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21621020">21621020</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pons-Salort</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">EP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grassly</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span>. <span class="mixed-article-title" title="mixed-article-title">The epidemiology of non-polio enteroviruses: recent advances and outstanding questions</span>. <span class="source" tagx="source" title="source">Curr Opin Infect Dis</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">28</span>: <span class="fpage" tagx="fpage" title="fpage">479</span>–<span class="lpage" tagx="lpage" title="lpage">487</span>. <span class="comment">doi: <a href="https://doi.org/10.1097/QCO.0000000000000187">10.1097/QCO.0000000000000187</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26203854">26203854</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bian</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">EV-A71 vaccine licensure: a first step for multivalent enterovirus vaccine to control HFMD and other severe diseases</span>. <span class="source" tagx="source" title="source">Emerg Microbes Infect</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e75</span><span class="comment">doi: <a href="https://doi.org/10.1038/emi.2016.73">10.1038/emi.2016.73</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27436364">27436364</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JX</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mao</span><span class="given-names" tagx="given-names" title="given-names">QY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">ZL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">FC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Development of enterovirus 71 vaccines: from the lab bench to Phase III clinical trials</span>. <span class="source" tagx="source" title="source">Expert Rev Vaccines</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">609</span>–<span class="lpage" tagx="lpage" title="lpage">618</span>. <span class="comment">doi: <a href="https://doi.org/10.1586/14760584.2014.897617">10.1586/14760584.2014.897617</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24621093">24621093</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mo</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">An inactivated enterovirus 71 vaccine in healthy children</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">370</span>: <span class="fpage" tagx="fpage" title="fpage">829</span>–<span class="lpage" tagx="lpage" title="lpage">837</span>. <span class="comment">doi: <a href="https://doi.org/10.1056/NEJMoa1303224">10.1056/NEJMoa1303224</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24571755">24571755</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ku</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leng</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">A combination vaccine comprising of inactivated enterovirus 71 and coxsackievirus A16 elicits balanced protective immunity against both viruses</span>. <span class="source" tagx="source" title="source">Vaccine</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">2406</span>–<span class="lpage" tagx="lpage" title="lpage">2412</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.vaccine.2014.03.012">10.1016/j.vaccine.2014.03.012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24657161">24657161</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Pathologic and immunologic characteristics of coxsackievirus A16 infection in rhesus macaques</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">500</span>: <span class="fpage" tagx="fpage" title="fpage">198</span>–<span class="lpage" tagx="lpage" title="lpage">208</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.virol.2016.10.031">10.1016/j.virol.2016.10.031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27829175">27829175</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winter</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keller</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gregory</span><span class="given-names" tagx="given-names" title="given-names">RI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diederichs</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Many roads to maturity: microRNA biogenesis pathways and their regulation</span>. <span class="source" tagx="source" title="source">Nat Cell Biol</span>. <span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">228</span>–<span class="lpage" tagx="lpage" title="lpage">234</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/ncb0309-228">10.1038/ncb0309-228</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19255566">19255566</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">VN</span></span>. <span class="mixed-article-title" title="mixed-article-title">Regulation of microRNA biogenesis</span>. <span class="source" tagx="source" title="source">Nat Rev Mol Cell Biol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">509</span>–<span class="lpage" tagx="lpage" title="lpage">524</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nrm3838">10.1038/nrm3838</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25027649">25027649</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Connell</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaudhuri</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baltimore</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="mixed-article-title" title="mixed-article-title">Physiological and pathological roles for microRNAs in the immune system</span>. <span class="source" tagx="source" title="source">Nat Rev Immunol</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">111</span>–<span class="lpage" tagx="lpage" title="lpage">122</span>. <span class="comment">doi: <a href="https://doi.org/10.1038/nri2708">10.1038/nri2708</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20098459">20098459</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piva</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spandidos</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gambari</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">From microRNA functions to microRNA therapeutics: novel targets and novel drugs in breast cancer research and treatment (Review)</span>. <span class="source" tagx="source" title="source">Int J Oncol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">985</span>–<span class="lpage" tagx="lpage" title="lpage">994</span>. <span class="comment">doi: <a href="https://doi.org/10.3892/ijo.2013.2059">10.3892/ijo.2013.2059</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23939688">23939688</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Louten</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beach</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palermino</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weeks</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holenstein</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNAs Expressed during Viral Infection: Biomarker Potential and Therapeutic Considerations</span>. <span class="source" tagx="source" title="source">Biomark Insights</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">25</span>–<span class="lpage" tagx="lpage" title="lpage">52</span>.</span></li> <li tag="ref"><a name="pone.0177657.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trobaugh</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimstra</span><span class="given-names" tagx="given-names" title="given-names">WB</span></span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNA Regulation of RNA Virus Replication and Pathogenesis</span>. <span class="source" tagx="source" title="source">Trends Mol Med</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">80</span>–<span class="lpage" tagx="lpage" title="lpage">93</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.molmed.2016.11.003">10.1016/j.molmed.2016.11.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27989642">27989642</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tahamtan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inchley</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marzban</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tavakoli-Yaraki</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teymoori-Rad</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakstad</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The role of microRNAs in respiratory viral infection: friend or foe?</span><span class="source" tagx="source" title="source">Rev Med Virol</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">389</span>–<span class="lpage" tagx="lpage" title="lpage">407</span>. <span class="comment">doi: <a href="https://doi.org/10.1002/rmv.1894">10.1002/rmv.1894</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27373545">27373545</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Comparison analysis of microRNAs in response to EV71 and CA16 infection in human bronchial epithelial cells by high-throughput sequencing to reveal differential infective mechanisms</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">228</span>: <span class="fpage" tagx="fpage" title="fpage">90</span>–<span class="lpage" tagx="lpage" title="lpage">101</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.virusres.2016.11.024">10.1016/j.virusres.2016.11.024</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27890633">27890633</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Different microRNA alterations contribute to diverse outcomes following EV71 and CA16 infections: Insights from high-throughput sequencing in rhesus monkey peripheral blood mononuclear cells</span>. <span class="source" tagx="source" title="source">Int J Biochem Cell Biol</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">81</span>: <span class="fpage" tagx="fpage" title="fpage">20</span>–<span class="lpage" tagx="lpage" title="lpage">31</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.biocel.2016.10.011">10.1016/j.biocel.2016.10.011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27765603">27765603</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">Systematic Identification and Bioinformatic Analysis of MicroRNAs in Response to Infections of Coxsackievirus A16 and Enterovirus 71</span>. <span class="source" tagx="source" title="source">Biomed Res Int</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">2016</span>: <span class="fpage" tagx="fpage" title="fpage">4302470</span><span class="comment">doi: <a href="https://doi.org/10.1155/2016/4302470">10.1155/2016/4302470</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27843944">27843944</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Understanding Enterovirus 71 Neuropathogenesis and Its Impact on Other Neurotropic Enteroviruses</span>. <span class="source" tagx="source" title="source">Brain Pathol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">614</span>–<span class="lpage" tagx="lpage" title="lpage">624</span>. <span class="comment">doi: <a href="https://doi.org/10.1111/bpa.12279">10.1111/bpa.12279</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26276025">26276025</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solomon</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewthwaite</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perera</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardosa</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMinn</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooi</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Virology, epidemiology, pathogenesis, and control of enterovirus 71</span>. <span class="source" tagx="source" title="source">Lancet Infect Dis</span>. <span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">778</span>–<span class="lpage" tagx="lpage" title="lpage">790</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/S1473-3099(10)70194-8">10.1016/S1473-3099(10)70194-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20961813">20961813</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blanchette</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daneman</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">Formation and maintenance of the BBB</span>. <span class="source" tagx="source" title="source">Mech Dev</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">138</span><span class="issue" tagx="issue" title="issue">Pt 1</span>: <span class="fpage" tagx="fpage" title="fpage">8</span>–<span class="lpage" tagx="lpage" title="lpage">16</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26215350">26215350</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">WY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ou</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">CY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Disruption of in vitro endothelial barrier integrity by Japanese encephalitis virus-Infected astrocytes</span>. <span class="source" tagx="source" title="source">Glia</span>. <span class="year" tagx="year" title="year">2015</span>.</span></li> <li tag="ref"><a name="pone.0177657.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roe</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orillo</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verma</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">West Nile virus-induced cell adhesion molecules on human brain microvascular endothelial cells regulate leukocyte adhesion and modulate permeability of the in vitro blood-brain barrier model</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e102598</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0102598">10.1371/journal.pone.0102598</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25036379">25036379</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Overview of available methods for diverse RNA-Seq data analyses</span>. <span class="source" tagx="source" title="source">Sci China Life Sci</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">54</span>: <span class="fpage" tagx="fpage" title="fpage">1121</span>–<span class="lpage" tagx="lpage" title="lpage">1128</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s11427-011-4255-x">10.1007/s11427-011-4255-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22227904">22227904</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">RK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">NGS QC Toolkit: a toolkit for quality control of next generation sequencing data</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e30619</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0030619">10.1371/journal.pone.0030619</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22312429">22312429</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>. <span class="mixed-article-title" title="mixed-article-title">An improvement of the 2^(-delta delta CT) method for quantitative real-time polymerase chain reaction data analysis</span>. <span class="source" tagx="source" title="source">Biostat Bioinforma Biomath</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">71</span>–<span class="lpage" tagx="lpage" title="lpage">85</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25558171">25558171</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agarwal</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nam</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartel</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span>. <span class="mixed-article-title" title="mixed-article-title">Predicting effective microRNA target sites in mammalian mRNAs</span>. <span class="source" tagx="source" title="source">Elife</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">4</span>:<span class="fpage" tagx="fpage" title="fpage">e05005</span>.</span></li> <li tag="ref"><a name="pone.0177657.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>. <span class="mixed-article-title" title="mixed-article-title">miRDB: an online resource for microRNA target prediction and functional annotations</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">43</span>: <span class="fpage" tagx="fpage" title="fpage">D146</span>–<span class="lpage" tagx="lpage" title="lpage">152</span>. <span class="comment">doi: <a href="https://doi.org/10.1093/nar/gku1104">10.1093/nar/gku1104</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25378301">25378301</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Auvinen</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Diagnostic and Prognostic Value of MicroRNA in Viral Diseases</span>. <span class="source" tagx="source" title="source">Mol Diagn Ther</span>. <span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">45</span>–<span class="lpage" tagx="lpage" title="lpage">57</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s40291-016-0236-x">10.1007/s40291-016-0236-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27682074">27682074</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meliopoulos</span><span class="given-names" tagx="given-names" title="given-names">VA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brooks</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakre</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coleman</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">MicroRNA regulation of human protease genes essential for influenza virus replication</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e37169</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0037169">10.1371/journal.pone.0037169</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22606348">22606348</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bakre</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coleman</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">LP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saavedra</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teng</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Respiratory syncytial virus modifies microRNAs regulating host genes that affect virus replication</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">93</span>: <span class="fpage" tagx="fpage" title="fpage">2346</span>–<span class="lpage" tagx="lpage" title="lpage">2356</span>. <span class="comment">doi: <a href="https://doi.org/10.1099/vir.0.044255-0">10.1099/vir.0.044255-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22894925">22894925</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Serum microRNA expression profile distinguishes enterovirus 71 and coxsackievirus 16 infections in patients with hand-foot-and-mouth disease</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">e27071</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0027071">10.1371/journal.pone.0027071</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22087245">22087245</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">RY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weng</span><span class="given-names" tagx="given-names" title="given-names">KF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Elevated expression of circulating miR876-5p is a specific response to severe EV71 infections</span>. <span class="source" tagx="source" title="source">Sci Rep</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">24149</span><span class="comment">doi: <a href="https://doi.org/10.1038/srep24149">10.1038/srep24149</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27052555">27052555</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tien</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. [<span class="mixed-article-title" title="mixed-article-title">MiR432* regulate the replication of coxsackievirus A16 in rhabdomyosarcoma cells</span>]. <span class="source" tagx="source" title="source">Wei Sheng Wu Xue Bao</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">54</span>: <span class="fpage" tagx="fpage" title="fpage">679</span>–<span class="lpage" tagx="lpage" title="lpage">687</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25272817">25272817</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lui</span><span class="given-names" tagx="given-names" title="given-names">YL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">WH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Timms</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hafner</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Enterovirus71 (EV71) utilise host microRNAs to mediate host immune system enhancing survival during infection</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e102997</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0102997">10.1371/journal.pone.0102997</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25047717">25047717</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Up-regulation of microRNA-183-3p is a potent prognostic marker for lung adenocarcinoma of female non-smokers</span>. <span class="source" tagx="source" title="source">Clin Transl Oncol</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">16</span>: <span class="fpage" tagx="fpage" title="fpage">980</span>–<span class="lpage" tagx="lpage" title="lpage">985</span>. <span class="comment">doi: <a href="https://doi.org/10.1007/s12094-014-1183-9">10.1007/s12094-014-1183-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24805982">24805982</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>. [<span class="mixed-article-title" title="mixed-article-title">Expression of miR-let-7e-3p in cervical intraepithelial neoplasm and cervix carcinoma and its clinical significance</span>]. <span class="source" tagx="source" title="source">Zhejiang Da Xue Xue Bao Yi Xue Ban</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">342</span>–<span class="lpage" tagx="lpage" title="lpage">348</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27868406">27868406</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">EY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">KW</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Octamer 4/microRNA-1246 signaling axis drives Wnt/beta-catenin activation in liver cancer stem cells</span>. <span class="source" tagx="source" title="source">Hepatology</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">64</span>: <span class="fpage" tagx="fpage" title="fpage">2062</span>–<span class="lpage" tagx="lpage" title="lpage">2076</span>. <span class="comment">doi: <a href="https://doi.org/10.1002/hep.28821">10.1002/hep.28821</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27639189">27639189</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chin</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nga</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lunny</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">EK</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Tumour-initiating cell-specific miR-1246 and miR-1290 expression converge to promote non-small cell lung cancer progression</span>. <span class="source" tagx="source" title="source">Nat Commun</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">11702</span><span class="comment">doi: <a href="https://doi.org/10.1038/ncomms11702">10.1038/ncomms11702</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27325363">27325363</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Extracellular miR-1246 promotes lung cancer cell proliferation and enhances radioresistance by directly targeting DR5</span>. <span class="source" tagx="source" title="source">Oncotarget</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">32707</span>–<span class="lpage" tagx="lpage" title="lpage">32722</span>. <span class="comment">doi: <a href="https://doi.org/10.18632/oncotarget.9017">10.18632/oncotarget.9017</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27129166">27129166</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muramatsu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ueda</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Inazawa</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Exosomal microRNA miR-1246 induces cell motility and invasion through the regulation of DENND2D in oral squamous cell carcinoma</span>. <span class="source" tagx="source" title="source">Sci Rep</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">38750</span><span class="comment">doi: <a href="https://doi.org/10.1038/srep38750">10.1038/srep38750</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27929118">27929118</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">YQ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Parallel mRNA and microRNA profiling of HEV71-infected human neuroblastoma cells reveal the up-regulation of miR-1246 in association with DLG3 repression</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2014</span>; <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">e95272</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0095272">10.1371/journal.pone.0095272</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24739954">24739954</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xun</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">CF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">QL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Differential expression of miRNAs in enterovirus 71-infected cells</span>. <span class="source" tagx="source" title="source">Virol J</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">56</span><span class="comment">doi: <a href="https://doi.org/10.1186/s12985-015-0288-2">10.1186/s12985-015-0288-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25889836">25889836</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathinayake</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wark</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Innate Immunity and Immune Evasion by Enterovirus 71</span>. <span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">6613</span>–<span class="lpage" tagx="lpage" title="lpage">6630</span>. <span class="comment">doi: <a href="https://doi.org/10.3390/v7122961">10.3390/v7122961</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26694447">26694447</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sonkoly</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stahle</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pivarcsi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">MicroRNAs and immunity: novel players in the regulation of normal immune function and inflammation</span>. <span class="source" tagx="source" title="source">Semin Cancer Biol</span>. <span class="year" tagx="year" title="year">2008</span>; <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">131</span>–<span class="lpage" tagx="lpage" title="lpage">140</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.semcancer.2008.01.005">10.1016/j.semcancer.2008.01.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18291670">18291670</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monastyrskaya</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Freire</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hashemi Gheinani</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klumpp</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babiychuk</span><span class="given-names" tagx="given-names" title="given-names">EB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Draeger</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">miR-199a-5p regulates urothelial permeability and may play a role in bladder pain syndrome</span>. <span class="source" tagx="source" title="source">Am J Pathol</span>. <span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">182</span>: <span class="fpage" tagx="fpage" title="fpage">431</span>–<span class="lpage" tagx="lpage" title="lpage">448</span>. <span class="comment">doi: <a href="https://doi.org/10.1016/j.ajpath.2012.10.020">10.1016/j.ajpath.2012.10.020</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23201090">23201090</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardona</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>. <span class="mixed-article-title" title="mixed-article-title">Intrinsic apoptosis and proinflammatory cytokines regulated in human astrocytes infected with enterovirus 71</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">96</span>: <span class="fpage" tagx="fpage" title="fpage">3010</span>–<span class="lpage" tagx="lpage" title="lpage">3022</span>. <span class="comment">doi: <a href="https://doi.org/10.1099/jgv.0.000235">10.1099/jgv.0.000235</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26296773">26296773</a></span></span></li> <li tag="ref"><a name="pone.0177657.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velandia-Romero</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calderon-Pelaez</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castellanos</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>. <span class="mixed-article-title" title="mixed-article-title">In Vitro Infection with Dengue Virus Induces Changes in the Structure and Function of the Mouse Brain Endothelium</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">e0157786</span><span class="comment">doi: <a href="https://doi.org/10.1371/journal.pone.0157786">10.1371/journal.pone.0157786</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27336851">27336851</a></span></span></li> </ul> </div> </div>  </body></html>